
<html lang="en"     class="pb-page"  data-request-id="0ffa3d38-d09f-40c5-9783-00de9208c295"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-5;article:article:10.1021/acs.jmedchem.9b01339"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="An Agonist Radioligand for the Proinflammatory Lipid-Activated G Protein-Coupled Receptor GPR84 Providing Structural Insights" /></meta><meta name="dc.Creator" content="Meryem  Köse" /></meta><meta name="dc.Creator" content="Thanigaimalai  Pillaiyar" /></meta><meta name="dc.Creator" content="Vigneshwaran  Namasivayam" /></meta><meta name="dc.Creator" content="Elisabetta  De Filippo" /></meta><meta name="dc.Creator" content="Katharina  Sylvester" /></meta><meta name="dc.Creator" content="Trond  Ulven" /></meta><meta name="dc.Creator" content="Ivar  von Kügelgen" /></meta><meta name="dc.Creator" content="Christa E.  Müller" /></meta><meta name="dc.Description" content="The orphan G protein-coupled receptor (GPCR) GPR84 is expressed on immune cells mediating proinflammatory and immunostimulatory effects. In this study, we prepared the fully efficacious, nonbiased ..." /></meta><meta name="Description" content="The orphan G protein-coupled receptor (GPCR) GPR84 is expressed on immune cells mediating proinflammatory and immunostimulatory effects. In this study, we prepared the fully efficacious, nonbiased ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 13, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01339" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01339" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01339" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01339" /></link>
        
    
    

<title>An Agonist Radioligand for the Proinflammatory Lipid-Activated G Protein-Coupled Receptor GPR84 Providing Structural Insights | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01339" /></meta><meta property="og:title" content="An Agonist Radioligand for the Proinflammatory Lipid-Activated G Protein-Coupled Receptor GPR84 Providing Structural Insights" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0022.jpeg" /></meta><meta property="og:description" content="The orphan G protein-coupled receptor (GPCR) GPR84 is expressed on immune cells mediating proinflammatory and immunostimulatory effects. In this study, we prepared the fully efficacious, nonbiased GPR84 agonist 6-hexylamino-2,4(1H,3H)-pyrimidinedione (6) in tritium-labeled form ([3H]PSB-1584) by hydrogenation of a hexenyl-substituted precursor with tritium gas. The radioligand was characterized by kinetic, saturation, and competition assays using membranes of Chinese hamster ovary cells recombinantly expressing the human GPR84. [3H]6 reversibly labeled the receptor with high affinity (KD 2.08 nM). Structurally diverse orthosteric and allosteric ligands, including newly designed and synthesized compounds, were studied in competition binding assays. A homology model of GPR84 was generated to perform docking studies rationalizing the experimental data. The radioligand was additionally used for labeling GPR84 in native cells and tissues. [3H]6 constitutes the first GPR84 agonist radioligand representing a powerful tool for this poorly investigated GPCR, which has potential as a future drug target." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01339"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01339">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01339&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01339&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01339&amp;href=/doi/10.1021/acs.jmedchem.9b01339" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2391-2410</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01287" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01406" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">An Agonist Radioligand for the Proinflammatory Lipid-Activated G Protein-Coupled Receptor GPR84 Providing Structural Insights</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Meryem Köse</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Meryem Köse</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PharmaCenter Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Meryem++K%C3%B6se">Meryem Köse</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3391-3418" title="Orcid link">http://orcid.org/0000-0002-3391-3418</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thanigaimalai Pillaiyar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thanigaimalai Pillaiyar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PharmaCenter Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thanigaimalai++Pillaiyar">Thanigaimalai Pillaiyar</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5575-8896" title="Orcid link">http://orcid.org/0000-0001-5575-8896</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vigneshwaran Namasivayam</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vigneshwaran Namasivayam</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PharmaCenter Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vigneshwaran++Namasivayam">Vigneshwaran Namasivayam</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3031-3377" title="Orcid link">http://orcid.org/0000-0003-3031-3377</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisabetta De Filippo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabetta De Filippo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PharmaCenter Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabetta++De+Filippo">Elisabetta De Filippo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katharina Sylvester</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katharina Sylvester</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PharmaCenter Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katharina++Sylvester">Katharina Sylvester</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Trond Ulven</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Trond Ulven</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Drug Design and Pharmacology, University of Copenhagen, Universitetsparken 2, DK-2100 Copenhagen, Denmark</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Trond++Ulven">Trond Ulven</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8135-1755" title="Orcid link">http://orcid.org/0000-0002-8135-1755</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ivar von Kügelgen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ivar von Kügelgen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, Pharma Center Bonn, Sigmund-Freud-Str. 25, D-53127 Bonn, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ivar++von+K%C3%BCgelgen">Ivar von Kügelgen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Christa E. Müller</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christa E. Müller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">PharmaCenter Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#53303b213a2027327d3e26363f3f362113263d3a7e313c3d3d7d3736"><span class="__cf_email__" data-cfemail="4f2c273d263c3b2e61223a2a23232a3d0f3a2126622d202121612b2a">[email protected]</span></a>. Phone: +49-228-73-2301. Fax: +49-228-73-2567.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christa+E.++M%C3%BCller">Christa E. Müller</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0013-6624" title="Orcid link">http://orcid.org/0000-0002-0013-6624</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01339&amp;href=/doi/10.1021%2Facs.jmedchem.9b01339" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2391–2410</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 13, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 August 2019</li><li><span class="item_label"><b>Published</b> online</span>13 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01339" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01339</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2391%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMeryem%2BK%25C3%25B6se%252C%2BThanigaimalai%2BPillaiyar%252C%2BVigneshwaran%2BNamasivayam%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D5%26contentID%3Dacs.jmedchem.9b01339%26title%3DAn%2BAgonist%2BRadioligand%2Bfor%2Bthe%2BProinflammatory%2BLipid-Activated%2BG%2BProtein-Coupled%2BReceptor%2BGPR84%2BProviding%2BStructural%2BInsights%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2410%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01339"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1116</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01339" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;An Agonist Radioligand for the Proinflammatory Lipid-Activated G Protein-Coupled Receptor GPR84 Providing Structural Insights&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Meryem&quot;,&quot;last_name&quot;:&quot;Köse&quot;},{&quot;first_name&quot;:&quot;Thanigaimalai&quot;,&quot;last_name&quot;:&quot;Pillaiyar&quot;},{&quot;first_name&quot;:&quot;Vigneshwaran&quot;,&quot;last_name&quot;:&quot;Namasivayam&quot;},{&quot;first_name&quot;:&quot;Elisabetta&quot;,&quot;last_name&quot;:&quot;De Filippo&quot;},{&quot;first_name&quot;:&quot;Katharina&quot;,&quot;last_name&quot;:&quot;Sylvester&quot;},{&quot;first_name&quot;:&quot;Trond&quot;,&quot;last_name&quot;:&quot;Ulven&quot;},{&quot;first_name&quot;:&quot;Ivar&quot;,&quot;last_name&quot;:&quot;von Kügelgen&quot;},{&quot;first_name&quot;:&quot;Christa&quot;,&quot;last_name&quot;:&quot;E. Müller&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;2391-2410&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01339&quot;},&quot;abstract&quot;:&quot;The orphan G protein-coupled receptor (GPCR) GPR84 is expressed on immune cells mediating proinflammatory and immunostimulatory effects. In this study, we prepared the fully efficacious, nonbiased GPR84 agonist 6-hexylamino-2,4(1H,3H)-pyrimidinedione (6) in tritium-labeled form ([3H]PSB-1584) by hydrogenation of a hexenyl-substituted precursor with tritium gas. The radioligand was characterized by kinetic, saturation, and competition assays using membranes of Chinese hamster ovary cells recombinantly expressing the human GPR84. [3H]6 reversibly labeled the receptor with high affinity (KD 2.08 nM). Structurally diverse orthosteric and allosteric ligands, including newly designed and synthesized compounds, were studied in competition binding assays. A homology model of GPR84 was generated to perform docking studies rationalizing the experimental data. The radioligand was additionally used for labeling GPR84 in native cells and tissues. [3H]6 constitutes the first GPR84 agonist radioligand representing a powerfu&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01339&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01339" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01339&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01339" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01339&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01339" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01339&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01339&amp;href=/doi/10.1021/acs.jmedchem.9b01339" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01339" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01339" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01339%26sid%3Dliteratum%253Aachs%26pmid%3D31721581%26genre%3Darticle%26aulast%3DK%25C3%25B6se%26date%3D2020%26atitle%3DAn%2BAgonist%2BRadioligand%2Bfor%2Bthe%2BProinflammatory%2BLipid-Activated%2BG%2BProtein-Coupled%2BReceptor%2BGPR84%2BProviding%2BStructural%2BInsights%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D5%26spage%3D2391%26epage%3D2410%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292589" title="Agonists">Agonists</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/jmcmar.2020.63.issue-5/20200312/jmcmar.2020.63.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The orphan G protein-coupled receptor (GPCR) GPR84 is expressed on immune cells mediating proinflammatory and immunostimulatory effects. In this study, we prepared the fully efficacious, nonbiased GPR84 agonist 6-hexylamino-2,4(1<i>H</i>,3<i>H</i>)-pyrimidinedione (<b>6</b>) in tritium-labeled form ([<sup>3</sup>H]PSB-1584) by hydrogenation of a hexenyl-substituted precursor with tritium gas. The radioligand was characterized by kinetic, saturation, and competition assays using membranes of Chinese hamster ovary cells recombinantly expressing the human GPR84. [<sup>3</sup>H]<b>6</b> reversibly labeled the receptor with high affinity (<i>K</i><sub>D</sub> 2.08 nM). Structurally diverse orthosteric and allosteric ligands, including newly designed and synthesized compounds, were studied in competition binding assays. A homology model of GPR84 was generated to perform docking studies rationalizing the experimental data. The radioligand was additionally used for labeling GPR84 in native cells and tissues. [<sup>3</sup>H]<b>6</b> constitutes the first GPR84 agonist radioligand representing a powerful tool for this poorly investigated GPCR, which has potential as a future drug target.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/5"><issue-title>Women in Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34360" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34360" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">GPR84 is a proinflammatory G protein-coupled receptor (GPCR), which is highly expressed on cells of the immune system (e.g., neutrophils, macrophages) and is upregulated under inflammatory conditions.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6 ref7">(1−7)</a> GPR84 functions as an amplifier of inflammatory signaling by enhancing the release of proinflammatory mediators, such as TNFα and various interleukins.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Its activation results in G<sub>i</sub> signaling leading to a reduction in cAMP production via inhibition of adenylate cyclase.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In addition, GPR84 has been described to signal through G<sub>12</sub>/G<sub>13</sub>, linking receptor function to Rho/Rac signaling and modulation of the cytoskeleton. In human macrophages, GPR84 activation was found to mediate G<sub>i</sub> protein-dependent Erk1/2 and Akt phosphorylation as well as G<sub>i</sub>-βγ-dependent PI<sub>3</sub> kinase activation and calcium mobilization.<a onclick="showRef(event, 'ref6 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref8 ref9">(6,8,9)</a> So far, only limited data regarding its functional significance are available. However, recent studies confirm the involvement of GPR84 in metabolic regulation and inflammation and indicate its clinical relevance in various diseases. GPR84 was found to be involved in inflammatory processes relevant to gastroesophageal reflux disease,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> inflammatory bowel disease,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> multiple sclerosis,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> neuropathic pain,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and Alzheimer’s disease.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Moreover, GPR84 has been linked to obesity and diabetes.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14−17)</a> Preliminary evidence indicates that GPR84 might be associated with leukemogenesis,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> osteoclastogenesis,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> as well as organ fibrosis, a pathological outcome of many inflammatory and metabolic diseases.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">GPR84 was identified as a fatty-acid sensing receptor, being activated by micromolar concentrations of medium chain fatty acids (MCFAs) with chain lengths of C9–C14.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Decanoic acid (<b>1</b>) was described as the most potent MCFA with a moderate EC<sub>50</sub> value of 4.5 μM. Several laboratories confirmed GPR84 activation by MCFAs and additionally by hydroxylated MCFAs.<a onclick="showRef(event, 'ref7 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref7 ref21 ref22 ref23 ref24 ref25">(7,21−25)</a> Nevertheless, to date, it is still not fully accepted by the scientific community that the weakly potent MCFAs represent the endogenous agonists of GPR84. Due to the lack of potent ligands, comprehensive studies were hampered. Only recently, potent synthetic agonists have been developed. The naturally occurring benzoquinone embelin (3-undecyl-2,5-dihydroxy-1,4-benzoquinone, <b>2</b>), a lipid-like molecule, also acts as a GPR84 agonist.<a onclick="showRef(event, 'ref21 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref26 ref27">(21,26,27)</a> Subsequent structure–activity relationship study of <b>2</b> led to a new embelin analogue with 45-fold higher potency at human GPR84 than embelin itself.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The synthetic compound 6-octylaminouracil (<b>3</b>) was discovered as another surrogate GPR84 agonist by screening of a compound library.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> High-throughput screening efforts by Zhang et al. resulted in the identification of 2-(hexylthio)-6-hydroxypyrimidin-4(3<i>H</i>)-one (<b>4</b>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Further optimization of lead structure <b>4</b> resulted in the development of 6-nonylpyridine derivative <b>5</b>.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Lead compound <b>3</b> was optimized leading to the development of potent GPR84 agonists (e.g., compound <b>6</b> (6-hexylamino-2,4(1<i>H</i>,3<i>H</i>)-pyrimidinedione) (<b>PSB-1584</b>) and <b>7</b> (6-((<i>p</i>-bromophenylethyl)amino)-2,4(1<i>H</i>,3<i>H</i>)-pyrimidinedione) (<b>PSB-17365</b>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> A structurally very different GPR84 agonist is the natural product-derived 3,3′-diindolylmethane (DIM, <b>8</b>),<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> which had previously been described to activate the estrogen receptor<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and the arylhydrocarbon receptor (AhR)<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> at micromolar concentrations (>5 μM). Subsequent studies confirmed the ability of <b>8</b> to activate human GPR84 with EC<sub>50</sub> values between 0.5 and 5.9 μM.<a onclick="showRef(event, 'ref4 ref22 ref24 ref25 ref28'); return false;" href="javascript:void(0);" class="ref ref4 ref22 ref24 ref25 ref28">(4,22,24,25,28)</a> DIM was proposed to bind to an allosteric site distinct from the decanoic acid binding site and to act as an ago-allosteric modulator activating the receptor by itself and also increasing the effects of lipidic and lipid-like agonists such as decanoic acid.<a onclick="showRef(event, 'ref22 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref25 ref26">(22,25,26)</a> Starting from DIM as a lead structure, potent GPR84 agonists were developed, such as the agonist 3,3′-di(5,7-difluoro-1<i>H</i>-indole-3-yl)methane (<b>9</b>, PSB-16671).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, <b>9</b> appears to additionally interact with (an)other target(s).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a></div><div class="NLM_p">In contrast to agonists, the development of GPR84 antagonists has so far only been described in the patent literature.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In a recent study, antagonists from a patented series were reported as noncompetitive antagonists, binding to a different binding site than the fatty acids and also different from that of 3,3′-diindolylmethane.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Structures and EC<sub>50</sub> values of selected standard GPR84 agonists together with their potencies determined in different functional assay systems are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Potencies of Selected Standard GPR84 Agonists Determined in Different Assay Systems</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0014.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="​" /></col><col align="char" char="​" /></col><col align="char" char="​" /></col><col align="char" char="." /></col><col align="char" char="​" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center" char="​">assay system</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">standard GPR84 agonists</th><th class="colsep0 rowsep0" align="center" char="​">cAMP assay EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="​">[<sup>35</sup>S]GTPγS assay EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="​">calcium assay EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">IP assay</th><th class="colsep0 rowsep0" align="center" char="​">β-arrestin assay EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">decanoic acid (<b>1</b>)</td><td class="colsep0 rowsep0" align="char" char="​">4.5<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></td><td class="colsep0 rowsep0" align="char" char="​">4.6<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></td><td class="colsep0 rowsep0" align="char" char="​">48<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a><sup>,</sup><a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">6.08<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">7.42<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="char" char="​">25<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​">30<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a><sup>,</sup><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​">10<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">20<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">48<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a><sup>,</sup><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">embelin (<b>2</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.089<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.63<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.63<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a><sup>,</sup><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">0.400<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.2<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​">5.0<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.421<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.795<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">6-octylaminouracil (<b>3</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.0167<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.512<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a><sup>,</sup><a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.653<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a><sup>,</sup><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td><td class="colsep0 rowsep0" align="char" char=".">0.105<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a><sup>,</sup><a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.114<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.341<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​">1.25<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​">1.75<a class="ref internalNav" href="#t1fn9" aria-label="i">i</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.438<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">2-hexylthiopyrimidine derivative (<b>4</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.001<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.00079<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.139<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a><sup>,</sup><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">0.597<a class="ref internalNav" href="#t1fn9" aria-label="i">i</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.134<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​">0.213<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.144<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6-nonylpyridine-2,4-diol (<b>5</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.000352<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">0.000189<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a><sup>,</sup><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6-hexylaminouracil (<b>6</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.00493<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">0.00318<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6-(<i>p</i>-bromophenyl-ethylamino)uracil (<b>7</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.00249<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">0.104<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="5" align="left">3,3′-diindolylmethane (<b>8</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.252<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="char" char="​">0.5<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">1.64<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">0.7<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></td><td class="colsep0 rowsep0" align="char" char="​">1.0<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">5.9<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a><sup>,</sup><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></td><td class="colsep0 rowsep0" align="char" char="​">11<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a><sup>,</sup><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">1.11<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="​">1.26<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a><sup>,</sup><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="​"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="​"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3,3′-di-(5,7-difluoro-1<i>H</i>-indol-3-yl)methane (<b>9</b>)</td><td class="colsep0 rowsep0" align="char" char="​">0.0413<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">n.d.</td><td class="colsep0 rowsep0" align="char" char=".">n.d.</td><td class="colsep0 rowsep0" align="char" char="​">5.47<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">CHO/human GPR84 cells.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">CHO/β-arrestin2/human GPR84 cells.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">HEK293/human GPR84 cells.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Sf9 insect/Gαi/human GPR84 cells.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">CHO/aequorin reporter/G<sub>α16</sub>, G<sub>qs5</sub>, G<sub>qo5</sub>, and G<sub>qi9</sub>/human GPR84 cells.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">CHO/G<sub>qi5</sub>/human GPR84 cells.</p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">HEK293/Gα16/human GPR84 cells.</p></div><div class="footnote" id="t1fn8"><sup><sup>h</sup></sup><p class="last">HEK/G<sub>qi5</sub>/human GPR84 cells.</p></div><div class="footnote" id="t1fn9"><sup><sup>i</sup></sup><p class="last">HEK293/β-arrestin2/human GPR84 cells.</p></div><div class="footnote" id="t1fn10"><sup><sup>j</sup></sup><p class="last">Flp-In TREx 293-FLAG-human GPR84-eYFP cells. n.d., not determined.</p></div></div></div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the determined agonist potencies in the different assay systems can vary considerably. For example, determined EC<sub>50</sub> values for agonist <b>3</b> range between 0.0167 and 0.438 μM (26-fold difference). It is well-known that functional assays can be influenced by various factors. Receptor expression levels, ligand bias, activity of signaling cascades in specific cells, or nonspecific, receptor-independent effects can affect the measured EC<sub>50</sub> values. Thus, values obtained in different functional test systems are not comparable.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a></div><div class="NLM_p">However, to date, only functional data are available, and so far, only two studies have been performed attempting to investigate the binding mode of agonists.<a onclick="showRef(event, 'ref22 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref25">(22,25)</a> While Nikaido et al. created a homology model of GPR84 based on the active-state structure of the β<sub>2</sub>-adrenoceptor,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Mahmud et al. generated a homology model using the transmembrane domain architecture of the orexin receptor type 1 (OX<sub>1</sub> receptor).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Moreover, mutagenesis studies were performed, and the mutants were investigated in GTPγS assays investigating G protein activation.<a onclick="showRef(event, 'ref22 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref25">(22,25)</a></div><div class="NLM_p last">In addition to functional assays, radioligand binding studies would be an ideal basis for molecular modeling and docking studies. An antagonist GPR84 radioligand, [<sup>3</sup>H]G9543, whose preparation was described in a patent,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and which had been utilized by Milligan et al., could neither be displaced by decanoic acid nor by DIM.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Consequently, [<sup>3</sup>H]G9543 was supposed to bind to a third binding site distinct from the putative orthosteric binding site for fatty acids, and also different from the allosteric binding site for DIM. Thus, to date, it has not been possible to directly assess the binding affinities of standard GPR84 agonists. To address this issue, we prepared a high-affinity agonist GPR84 radioligand, which binds to the putative orthosteric, fatty acid binding site. In a previous study, we had shown that unlabeled <b>6</b> is a metabolically stable, highly potent, and fully efficacious, nonbiased agonist at human GPR84 as determined in cAMP (EC<sub>50</sub> 4.93 nM) and β-arrestin (EC<sub>50</sub> 3.18 nM) assays.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> We selected this compound for tritium labeling to obtain the first agonist radioligand for GPR84. The radioligand was fully characterized and applied in competition binding assays. The determined binding data served as a basis for analyzing interactions of ligands with GPR84 studied by docking of compounds into a receptor homology model.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56343" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56343" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The radioligand [<sup>3</sup>H]<b>6</b> was prepared as depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. <i>N</i>-Hexenylphthalimide (<b>12</b>) was synthesized by Mitsunobu reaction from commercially available phthalimide (<b>10</b>) and (<i>E</i>)-hex-2-enol (<b>11</b>) in the presence of diethyl azodicarboxylate (DEAD) and triphenylphosphine.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Compound <b>12</b> was converted to the amine <b>13</b> using hydrazine hydrate under reflux conditions. Amine <b>13</b> was subsequently reacted with 6-chlorouracil (<b>14</b>) under reflux<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> to afford the desired radioligand precursor (<i>E</i>)-6-(hex-2-en-1-ylamino)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>15</b>). The catalytic hydrogenation procedure was initially investigated with nonradioactive hydrogen gas to prepare nonradioactive <b>6</b>, before radiolabeling was approached. In procedure A, the reaction was performed at 25 psi (1.7 bar) in methanol (MeOH):tetrahydrofuran (THF) (1:1) in the presence of 10% palladium on activated charcoal for 2 h at room temperature. In procedure B, the reaction was performed at 35 psi (2.4 bar) in <i>N</i>,<i>N</i>′-dimethylformamide (DMF):THF (1:1) as a solvent instead. Under both conditions, <b>15</b> was completely hydrogenated to product <b>6</b>. The alkene <b>15</b> was subsequently custom labeled via hydrogenation of the double bond with tritium gas to obtain the desired radioligand [<sup>3</sup>H]<b>6</b>, whose identity was confirmed by mass spectrometry. [<sup>3</sup>H]<b>6</b> displayed a specific activity of 60 Ci/mmol (2.22 TBq/mmol) and a radiochemical purity of 99.8% as determined by HPLC–UV (254 nm) (see <a class="ref internalNav" href="#notes-3" aria-label="Figures S1–S3">Figures S1–S3</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0012.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of the GPR84 Agonist Radioligand [<sup>3</sup>H]<b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) PPh<sub>3</sub>, EtO<sub>2</sub>CN═NCO<sub>2</sub>Et, THF, 0 °C to rt, 12 h, 95%; (ii) hydrazine hydrate solution (∼80% in H<sub>2</sub>O), 100 °C, 3 h, 75.6%; (iii) 1-butanol, 125 °C, 12 h, 91%; (iv) procedure A: MeOH:THF (1:1), Pd/C (10%), H<sub>2</sub>, 1.7 bar, 2 h, 98%; procedure B: <i>N</i>,<i>N</i>′-dimethylformamide (DMF):THF (1:1), Pd/C (10%), H<sub>2</sub>, 2.4 bar, 2 h, 97.5%; (v) custom labeling in THF as a solvent (for details see <a class="ref internalNav" href="#notes-3" aria-label="SI">SI</a>).</p></p></figure><div class="NLM_p">A large number of previously described GPR84 ligands<a onclick="showRef(event, 'ref4 ref8 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref4 ref8 ref24 ref25 ref26 ref27">(4,8,24−27)</a> were subsequently studied in binding assays using the new radioligand. In addition, we synthesized three series of new lipid-like compounds (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Pyrimidine derivatives <b>4</b>, <b>62</b>–<b>68</b>, <b>73</b>–<b>77</b>, and <b>79</b>–<b>85</b> were prepared from 6-hydroxy-2-thioxo-2,3-dihydropyrimidin-4(1<i>H</i>)-one (<b>61</b>), 4-hydroxy-pyrimidine-2(1<i>H</i>)-thione derivatives (<b>69</b>–<b>71</b>), or 2-thiopyrimidine-4-carboxylic acid (<b>72</b>), respectively, by alkylation with (aryl)alkyl bromides in the presence of potassium carbonate in methanol under reflux conditions. Carboxylic acid ester <b>76</b> was hydrolyzed in the presence of 2-N sodium hydroxide to produce the corresponding carboxylic acid <b>77</b>. Orotic acid ester and amide derivatives were synthesized from orotic acid (<b>78</b>) as depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. For preparing the orotic acid esters <b>79</b> and <b>80</b>, compound <b>78</b> was dissolved in butanol or hexanol and refluxed for 12 h in the presence of catalytic amounts of concentrated sulfuric acid. Orotic acid amides were obtained by condensation of <b>78</b> with various amines in the presence of <i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium-hexafluorophosphate (HATU) and triethylamine (TEA) in DMF.</div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0013.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>4</b>, <b>62</b>–<b>68</b>, <b>73</b>–<b>77</b>, and <b>79</b>–<b>85</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) K<sub>2</sub>CO<sub>3</sub>, methanol, reflux, 12 h; (ii) from <b>76</b>, 2 N NaOH, methanol, 80 °C, 1 h; (iii) for esters (X = O): alcohols, H<sub>2</sub>SO<sub>4</sub>, reflux, 12 h; for amides (X = NH): amines, HATU, TEA, DMF, 12 h, rt.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Biological Experiments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32768" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32768" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Pharmacological Characterization of the New Radioligand</h3><div class="NLM_p">Membrane preparations of a CHO cell line stably transfected with the human GPR84 (CHO-hGPR84) were used to establish a radioligand binding assay. In preliminary experiments, specific binding of 3–10 nM of [<sup>3</sup>H]<b>6</b> to GPR84-expressing cells was detected, while only negligible binding to nontransfected CHO cells was observed under the same conditions (see <a class="ref internalNav" href="#notes-3" aria-label="Figure S4">Figure S4</a>). Subsequently, we investigated the effects of the buffer composition on [<sup>3</sup>H]<b>6</b> binding (1 nM). The addition of 0.05% of fatty acid-free bovine serum albumin (BSA) to tris(hydroxylmethyl)aminomethane (TRIS) as an incubation buffer significantly increased specific binding (8-fold, 0.0024, **). The addition of MgCl<sub>2</sub> (10 mM) also increased specific binding by 3-fold. Therefore, we combined both, MgCl<sub>2</sub> and BSA, which led to a further significant increase in specific binding (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). In contrast, the addition of NaCl led to a significant decrease in specific binding, while the IC<sub>50</sub> value of agonist <b>6</b> was not affected (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a class="ref internalNav" href="#notes-3" aria-label="Figure S5">Figure S5</a>). Sodium ions are known to act as negative allosteric modulators of many GPCRs.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In contrast, Mg<sup>2+</sup> ions may promote the formation of a high-affinity agonist-receptor-G protein complex acting as a positive allosteric modulator. The same effect had been observed in previous studies with agonist radioligands, for example, GPR35 and GPR17.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42">(39−42)</a></div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Effects of BSA and cations on [<sup>3</sup>H]<b>6</b> binding. The experiments were performed at 25 °C (150 min) using 1 nM [<sup>3</sup>H]<b>6</b> and 10 μg of a membrane preparation of CHO cells expressing the human GPR84 (CHO-hGPR84 cells). Data represent means of two independent experiments performed in duplicate; dpm, decays per minute. The unpaired <i>t</i> test was used for statistical comparisons (**, <i>p</i> < 0.01).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, 50 mM TRIS buffer, pH 7.4 containing 10 mM MgCl<sub>2</sub>, and 0.05% fatty acid-free BSA was found to be the optimal assay buffer resulting in low nonspecific binding of only 15%, and these conditions were therefore used in all subsequent binding assays.</div><div class="NLM_p">As a next step, kinetic studies with CHO-hGPR84 cell membranes (10 μg of protein) were performed using 1 nM [<sup>3</sup>H]<b>6</b> in 50 mM TRIS buffer, pH 7.4, supplemented with 10 mM MgCl<sub>2</sub> and 0.05% BSA. Both association and dissociation appeared monophasic (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). In the association experiments (<i>t</i><sub>1/2</sub> = 21 min), an equilibrium was reached within <120 min which remained stable for at least 6 h, indicating that an incubation time of 150 min is sufficient to ensure equilibrium binding of [<sup>3</sup>H]<b>6</b>. The binding was reversible after the addition of a high concentration (100 μM) of the potent GPR84 agonist <b>7</b><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> with a dissociation half-life of 32 min (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). A kinetic <i>K</i><sub>D</sub> value of 2.04 nM was calculated. In saturation experiments using 12 different radioligand concentrations ranging from 0.04 to 10 nM, a <i>K</i><sub>D</sub> value of 2.08 nM and an apparent <i>B</i><sub>max</sub> value of 1.87 pmol/mg protein were determined (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). We subsequently transferred the assay to a 96-well plate format, which allows a higher throughput than the originally used single vials. A radioligand concentration of 2 nM [<sup>3</sup>H]<b>6</b> was found to be suitable for performing competition binding assays in a 96-well plate format. The nonspecific binding at that concentration amounted <7% of total binding. Homologous competition experiments with the unlabeled ligand <b>6</b> were performed using a high concentration of GPR84 ligand <b>7</b> (10 μM) to determine nonspecific binding. Homologous competition assays for agonist <b>6</b> revealed a <i>K</i><sub>i</sub> value of 2.98 nM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). Thus, all three determined affinity values, <i>K</i><sub>i</sub> for the unlabeled agonist <b>6</b>, the kinetic <i>K</i><sub>D</sub> value, as well as the <i>K</i><sub>D</sub> value of [<sup>3</sup>H]<b>6</b> obtained in saturation assays, were virtually identical.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Association and (B) dissociation curve of [<sup>3</sup>H]<b>6</b> binding (1 nM) at 25 °C to membrane preparations of CHO-human GPR84 cells (10 μg protein/vial). Dissociation was achieved by addition of the GPR84 agonist <b>7</b> (100 μM) after 150 min of pre-incubation. A kinetic <i>K</i><sub>D</sub> value of 2.04 nM was determined. (C) Saturation curve of binding of increasing concentrations of [<sup>3</sup>H]<b>6</b>; experiments were performed at 25 °C with 10 μg of protein per vial. For the determination of nonspecific binding, 10 μM of <b>7</b> was used. The following binding parameters were calculated: <i>K</i><sub>D</sub> = 2.08 ± 0.68 nM, <i>B</i><sub>max</sub> = 1.87 ± 0.19 pmol/mg protein. (D) Homologous competition experiments. A <i>K</i><sub>i</sub> value of 2.98 ± 0.38 nM was determined for <b>6</b> (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Mean values ± SEM from 3 to 6 independent experiments performed in duplicates are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Binding Affinity of Standard GPR84 Ligands</h3><div class="NLM_p">With the optimized radioligand-binding assay, we initially investigated the standard GPR84 agonists <b>1</b>–<b>4</b> and <b>6</b>–<b>9</b> in competition experiments (for <i>K</i><sub>i</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). As expected, compounds <b>1</b>–<b>4</b>, <b>6</b>, and <b>7</b> with a lipid-mimetic structure were able to completely displace the radioligand (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The determined <i>K</i><sub>i</sub> values were lower than the EC<sub>50</sub> values previously determined in functional assays (for previously determined EC<sub>50</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Decanoic acid (<b>1</b>) and embelin (<b>2</b>) both showed a higher affinity (<i>K</i><sub>i</sub> 1780 nM and 23.6 nM, respectively) in binding assays compared to the previously determined potencies in functional assays (3- to 14-fold for <b>1</b> and 8- to 212-fold for <b>2</b>). Agonist <b>3</b> displayed a <i>K</i><sub>i</sub> value of 0.626 nM in the binding studies, which is 27- to 2800-fold lower than the respective EC<sub>50</sub> values determined in functional assays (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Similarly, <b>4</b> (<i>K</i><sub>i</sub> 0.219 nM) displayed a 5- to 2730-fold higher affinity in binding studies compared to the potency measured in functional studies. Interestingly, the <i>K</i><sub>i</sub> value for <b>6</b> (2.98 nM) corresponded very well with the EC<sub>50</sub> values previously determined in cAMP and β-arrestin assays (4.93 nM and 3.18 nM, respectively). In contrast, the <i>K</i><sub>i</sub> value of <b>7</b> (0.317 nM) was 8- to 330-fold lower than the published EC<sub>50</sub> values for <b>7</b>.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Competition binding experiments for standard agonists versus the agonist radioligand [<sup>3</sup>H]<b>6</b>. The experiments were performed as described in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>. <i>K</i><sub>i</sub> values for lipid-like agonists were as follows: 1780 ± 285 nM for <b>1</b> (decanoic acid), 23.6 ± 6.8 nM for <b>2</b> (embelin), 0.626 ± 0.205 nM for <b>3</b> (6-octylaminouracil), 0.219 ± 0.054 nM for <b>4</b> (2-hexylthiopyrimidine derivative), and 0.317 ± 0.033 nM for <b>7</b> (6-(<i>p</i>-bromophenylethylamino)uracil). For positive allosteric modulators, EC<sub>50</sub> values were as follows: 368 ± 42 nM for <b>8</b> (diindolylmethane) and 41.5 ± 7.0 nM for <b>9</b> (5,5′,7,7′-tetrafluoro-DIM). Data points represent means ± SEM from 3–6 independent experiments, each performed in duplicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinities and Functional Potencies and Efficacies of (Hydroxy)fatty Acids at the Human GPR84</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0015.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">human GPR84</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">radioligand binding assay <i>K</i><sub>i</sub>  ± SEM (μM) vs [<sup>3</sup>H]<b>6</b> (or percent inhibition ± SEM at 100 μM) (<i>n</i> = 3)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">cAMP assay EC<sub>50</sub>  ± SEM (μM) (or percent receptor activation at 100 μM) [efficacy]<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a> (<i>n</i> = 3)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">β-arrestin assay EC<sub>50</sub>  ± SEM (μM) (or percent receptor activation at 100 μM) [efficacy]<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a> (<i>n</i> = 3)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">embelin (<b>2</b>) (for structure see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>)</td><td class="colsep0 rowsep0"> </td><td class="colsep0 rowsep0">0.0236 ± 0.0068</td><td class="colsep0 rowsep0">0.795 [86 %]<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0">0.400 [100 %]<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5">Structure A: Fatty Acids</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">octanoic acid (<b>16</b>)</td><td class="colsep0 rowsep0">C<sub>6</sub>H<sub>13</sub></td><td class="colsep0 rowsep0">19.6 ± 6.8</td><td class="colsep0 rowsep0">>100 (36%)</td><td class="colsep0 rowsep0">>100 (2%)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">decanoic acid (<b>1</b>)</td><td class="colsep0 rowsep0">C<sub>8</sub>H<sub>17</sub></td><td class="colsep0 rowsep0">1.78 ± 0.29<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0">7.42 [100%]<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0">6.08 [92%]<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">undecanoic acid (<b>17</b>)</td><td class="colsep0 rowsep0">C<sub>9</sub>H<sub>19</sub></td><td class="colsep0 rowsep0">0.108 ± 0.024</td><td class="colsep0 rowsep0">1.60 ± 0.14 [93 %]<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0">4.36 ± 1.51 [82 %]<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">dodecanoic acid (<b>18</b>)</td><td class="colsep0 rowsep0">C<sub>10</sub>H<sub>21</sub></td><td class="colsep0 rowsep0">2.17 ± 0.24</td><td class="colsep0 rowsep0">8.87 ± 0.75 [94 %]</td><td class="colsep0 rowsep0">3.49 ± 0.56 [92 %]<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">tridecanoic acid (<b>19</b>)</td><td class="colsep0 rowsep0">C<sub>11</sub>H<sub>23</sub></td><td class="colsep0 rowsep0">1.93 ± 0.80</td><td class="colsep0 rowsep0">9.61 ± 1.25 [51%]</td><td class="colsep0 rowsep0">1.70 ± 0.64 [77%]<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">tetradecanoic acid (<b>20</b>)</td><td class="colsep0 rowsep0">C<sub>12</sub>H<sub>25</sub></td><td class="colsep0 rowsep0">2.64 ± 0.63</td><td class="colsep0 rowsep0">11.4 ± 1.6 [63%]</td><td class="colsep0 rowsep0">7.01 ± 3.20 [61 %]<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5">Structure B: 2-Hydroxy-fatty Acids</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R,S</i>)-2-hydroxy- octanoic acid <b>(21</b>)</td><td class="colsep0 rowsep0">C<sub>5</sub>H<sub>11</sub></td><td class="colsep0 rowsep0">≈100 (50%)</td><td class="colsep0 rowsep0">>100 (17%)</td><td class="colsep0 rowsep0">>100 (5%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R,S</i>)-2-hydroxy-tetra-decanoic acid (<b>22</b>)</td><td class="colsep0 rowsep0">C<sub>11</sub>H<sub>23</sub></td><td class="colsep0 rowsep0">0.318 ± 0.070</td><td class="colsep0 rowsep0">3.86 ± 1.00 [102%]</td><td class="colsep0 rowsep0">4.36 ± 1.42 [101%]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5">Structure C: 3-Hydroxy-fatty Acids</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R,S</i>)-3-hydroxy- decanoic acid (<b>23</b>)</td><td class="colsep0 rowsep0">C<sub>6</sub>H<sub>13</sub></td><td class="colsep0 rowsep0">3.31 ± 0.81</td><td class="colsep0 rowsep0">31.8 ± 4.67 [97 %]</td><td class="colsep0 rowsep0">4.05 ± 1.35 [54%]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R,S</i>)-3-hydroxy-do-decanoic acid (<b>24</b>)</td><td class="colsep0 rowsep0">C<sub>8</sub>H<sub>17</sub></td><td class="colsep0 rowsep0">0.110 ± 0.021</td><td class="colsep0 rowsep0">1.31 ± 0.15 [112 %]</td><td class="colsep0 rowsep0">3.25 ± 1.34 [125%]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R,S</i>)-3-hydroxy-tetra-decanoic acid (<b>25</b>)</td><td class="colsep0 rowsep0">C<sub>10</sub>H<sub>21</sub></td><td class="colsep0 rowsep0">0.511 ± 0.107<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0">1.57 ± 0.47 [88%]</td><td class="colsep0 rowsep0">0.432 ± 192 [78%]<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R</i>)-3-hydroxy-tetra-decanoic acid (<b>26</b>)</td><td class="colsep0 rowsep0">C<sub>10</sub>H<sub>21</sub></td><td class="colsep0 rowsep0">0.472 ± 0.120</td><td class="colsep0 rowsep0">1.75 ± 0.62 [91%]<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0">0.452 ± 0.158 [77 %]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5">Structure D: 12-Hydroxydodecanoic Acid</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">(<i>R,S</i>)-12-hydroxy-do-decanoic acid (<b>27</b>)</td><td class="colsep0 rowsep0">see struct. D</td><td class="colsep0 rowsep0">>10 (5%)</td><td class="colsep0 rowsep0">>100 (27%)</td><td class="colsep0 rowsep0">>100 (−17%)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Unless otherwise noted.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Efficacy [<i>E</i><sub>max</sub>] relative to the max. effect of decanoic acid (100 μM) (= 100%).</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Efficacy [<i>E</i><sub>max</sub>] relative to the max. effect of embelin (10 μM) (= 100%).</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 2.</p></div><div class="footnote" id="t2fn5"><sup><sup>e</sup></sup><p class="last"><i>n</i> = 4.</p></div><div class="footnote" id="t2fn6"><sup><sup>f</sup></sup><p class="last"><i>n</i> = 7.</p></div></div></div><div class="NLM_p last">DIM (<b>8</b>) did not lead to an inhibition but to a concentration-dependent increase in the specific binding of [<sup>3</sup>H]<b>6</b> by up to 100% compared to the control in the absence of DIM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Although previous studies had suggested that <b>8</b> may be an ago-allosteric modulator,<a onclick="showRef(event, 'ref22 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref25 ref26">(22,25,26)</a> it has not been possible so far to study direct receptor–ligand interaction for DIM. Here we directly show for the first time that <b>8</b> indeed binds to a different site than the lipid mimetic [<sup>3</sup>H]<b>6</b>. The observed increase in specific binding of [<sup>3</sup>H]<b>6</b> further suggests that <b>8</b> not only activates the receptor, but at the same time, it positively modulates the binding of the lipid mimetic <b>6</b>, which is presumed to act as an orthosteric GPR84 agonist. This observation is in agreement with previously reported cAMP data showing an increase in efficacy and potency for decanoic acid with increasing concentrations of <b>8</b>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Binding Affinities of Medium-Chain (Hydroxy)fatty Acids</h3><div class="NLM_p">Next, we investigated several medium-chain (hydroxy)fatty acids (C8–C14, (hydroxy)MCFAs) in binding assays at GPR84 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In parallel, we also evaluated these MCFAs in functional assays (cAMP and β-arrestin assays).</div><div class="NLM_p">Undecanoic acid showed the highest binding affinity among the tested MCFAs with a <i>K</i><sub>i</sub> value of 108 nM (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref internalNav" href="#notes-3" aria-label="Figure S6">Figure S6</a>). It was also the most potent fatty acid in cAMP assays (EC<sub>50</sub> 1.60 μM). Fatty acids with a shorter or a longer chain length displayed lower binding affinities ranging between 2 and 20 μM. We detected a <i>K</i><sub>i</sub> value of 19.6 μM for octanoic acid although no activity in cAMP and β-arrestin assays at concentrations up to 100 μM could be observed for this fatty acid. In previous studies, octanoic acid had also been shown to be inactive in functional assays.<a onclick="showRef(event, 'ref4 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref7">(4,7)</a> In general, we observed a high correlation between <i>K</i><sub>i</sub> values determined in binding studies and the EC<sub>50</sub> values determined in cAMP accumulation studies using the same cell line (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <i>R</i><sup>2</sup> = 0.906), while correlation with EC<sub>50</sub> values obtained from β-arrestin assays appeared to be much lower (<i>R</i><sup>2</sup> = 0.621). Previously, it had been difficult to predict the true binding affinities of GPR84 agonists. The herein determined binding affinity for decanoic acid (1.78 μM), for example, is 100–300 higher than the predicted affinity (<i>K</i><sub>A</sub> 170–530 μM) based on functional data.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> For tetradecanoic acid, a very high EC<sub>50</sub> value of 93.2 μM<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> or even no activity<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> had been reported in previous studies. In contrast, we determined a <i>K</i><sub>i</sub> value of 2.64 μM in binding assays. Using the same CHO cell line that overexpresses the human GPR84, the EC<sub>50</sub> values were 11.4 μM (cAMP assay) and 7.01 μM (β-arrestin assay), respectively. These data illustrate that functional data for agonists are highly dependent on the test system, for example, on receptor expression levels, whereas binding data are largely independent and provide “true” affinities.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of pEC<sub>50</sub> values of test compounds determined in cAMP assays (●) or β-arrestin assays (■) versus their respective p<i>K</i><sub>i</sub> values determined in radioligand binding assays. The same cell line (CHO-β-arrestin-human GPR84 cells) was used for all assays. <i>R</i><sup>2</sup> was determined in a two-tailed Pearson test. (A) Fatty acid derivatives and (B) 6-(ar)alkylamino-substituted uracil derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">All hydroxyfatty acids, except for (<i>R</i>,<i>S</i>)-3-hydroxy-decanoic acid (<b>23</b>), showed higher binding affinities than the respective nonhydroxylated fatty acids confirming previous results from functional assays reported by Suzuki et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The position of the hydroxyl group, at C2 or C3, did not significantly affect the binding affinity (compare <b>22</b> and <b>25</b>).</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Binding affinities of 6-(Ar)alkylamino-Substituted Uracil Derivatives</h3><div class="NLM_p">The binding affinity of the radioligand precursor (E)-6-(hex-2-en-1-ylamino)-pyrimidine-2,4(1H,3H)-dione (<b>15</b>)  was much lower ( 12-fold) when compared to the corresponding saturated compound <b>6</b> (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Next, a series of 6-(ar)alkylamino-substituted uracil derivatives (<b>28</b>–<b>60</b>) previously synthesized in our group<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> were tested for their binding affinity to the human GPR84 (for selected curves see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, for <i>K</i><sub>i</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). As depicted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, the binding affinities correlated with the agonist potencies previously determined in cAMP assays (<i>r</i><sup>2</sup> = 0.775), but much less with pEC<sub>50</sub> values determined in β-arrestin assays (<i>r</i><sup>2</sup> = 0.476) (for the previously obtained functional data see <a class="ref internalNav" href="#notes-3" aria-label="Table S2">Table S2</a>). As seen for the standard agonists <b>1</b>–<b>4</b>, <b>6</b>, and <b>7</b>, binding affinities were up to 57-fold (cAMP assay, on average 17-fold) or up to 803-fold (β-arrestin assay, on average 161-fold) higher compared to their respective potencies in functional assays.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Affinities of Agonists at the Human GPR84 Determined in Radioligand Binding Assays</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0016.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0017.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Affinities were determined in competition binding experiments using 2 nM [<sup>3</sup>H]<b>6</b> and membrane preparations of CHO cells recombinantly expressing hGPR84.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Unless otherwise noted.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 6.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 4.</p></div></div><div></div></div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Competition binding experiments at membrane preparations of recombinant CHO cells expressing human GPR84. (A) Competition binding curves for selected 6-(ar)alkylamino-substituted uracil derivatives. (B) Competition binding curves for synthesized lipid-like compounds <b>62</b>–<b>65</b>, <b>68</b>, <b>74</b>, <b>77</b>, and <b>81</b>–<b>83</b>. Data points represent means ± SEM from three independent experiments, each performed in duplicates. For <i>K</i><sub>i</sub> values, see <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Interestingly, for compound <b>6</b>, previously determined as the most potent unbiased GPR84 agonist of the series,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> no significant difference between the <i>K</i><sub>i</sub> value and the EC<sub>50</sub> values in both functional assays was observed (<i>K</i><sub>i</sub> 2.98 nM and EC<sub>50</sub> 4.93 nM, respectively). In contrast, <b>37</b>, previously determined as the most potent G<sub>i</sub>-biased GPR84 agonist of the series, showed the highest difference (803-fold) between its <i>K</i><sub>i</sub> value from radioligand binding (0.650 nM) and its EC<sub>50</sub> value determined in β-arrestin assays (EC<sub>50</sub> 522 nM).</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Binding Affinities of Thiopyrimidones and Tetrahydropyrimidine Carboxylic Acid Esters and Amides</h3><div class="NLM_p">Taking the potent GPR84 agonist <b>4</b> (2-(hexylthio)-6-hydroxypyrimidin-4(3<i>H</i>)-one)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> as a lead structure, we prepared a small series of derivatives and analogues and evaluated them in radioligand binding assays for their GPR84 affinity (for selected curves see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, for <i>K</i><sub>i</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Agonist <b>4</b> itself displayed a <i>K</i><sub>i</sub> value of 0.219 nM and was actually the compound with the highest affinity of all investigated GPR84 ligands. 6-Nonylpyridine-2,4-diol (<b>5</b>), previously described to be even more potent than <b>4</b>,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> was found to display 7-fold lower affinity as compared to <b>4</b> in binding studies (<i>K</i><sub>i</sub> 1.56 nM). 6-Octylaminouracil (<b>3</b>, 0.626 nM) and 6-<i>p</i>-bromophenethylaminouracil (<b>7</b>, 0.317 nM) belong to the compounds with the highest GPR84 affinity being similarly potent as <b>4</b>, while the affinity of 6-hexylaminouracil (<b>6</b>, 2.98 nM) is slightly lower. The rank order of potency of lipophilic agonists was generally somewhat different in radioligand binding assays as compared to functional assays. One reason could be due to difficulties in handling highly lipophilic, poorly water-soluble molecules. In binding assays employing cell membrane preparations, a higher percentage of DMSO can be used than in cellular systems, and predilutions can all be made in DMSO rather than aqueous buffer. Predilution in buffer could lead to a loss of compound resulting in reduced concentrations in the assay. Therefore, according to our experience, these data obtained in binding studies are most reliably representing true affinities.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Affinities of Agonists at the Human GPR84 Determined in Radioligand Binding Assays</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0018.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0019.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Affinities were determined in competition binding experiments using 2 nM [<sup>3</sup>H]<b>6</b> and membrane preparations of CHO cells recombinantly expressing human GPR84.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Compound <b>4</b> was resynthesized, and the synthetic procedure is described in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last"><i>n</i> = 2.</p></div></div><div></div></div><div class="NLM_p">Introducing aromatic residues into the alkyl chain of lead structure <b>4</b> was well tolerated leading to analogs with similarly potent affinity, for example, 6-phenylethylthiopyrimidine-2,4-diol (<b>62</b>, <i>K</i><sub>i</sub> 0.700 nM), 6-phenylpropylthiopyrimidine-2,4-diol (<b>63</b>, 0.324 nM), and <i>p</i>-chlorobenzylthiopyrimidine-2,4-diol (<b>65</b>, 2.63 nM).</div><div class="NLM_p last">Deletion or replacement of the 4-hydroxy/oxo-function by hydrogen, alkyl, or carboxylate in compounds <b>73</b>–<b>77</b> abolished or dramatically reduced GPR84 affinity. Replacement of the amino group in 6-(ar)alkylaminouracils by an ester or amide linker (compounds <b>79</b>–<b>85</b>) was also not well tolerated (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Binding Affinities of Diindoylmethane Derivatives</h3><div class="NLM_p">We further selected a series of diindolylmethane derivatives (compound <b>86</b>–<b>103</b>) that were previously shown to activate GPR84 in cAMP accumulation and/or β-arrestin recruitment assays.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Like DIM (<b>8</b>) itself (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), most of the substituted DIM derivatives and analogs also showed a concentration-dependent increase in specific binding of [<sup>3</sup>H]<b>6</b> (for selected curves see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, for <i>K</i><sub>i</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Affinity of 3,3′-Diindolylmethane Derivatives at the Human GPR84</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0020.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0021.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Affinities were determined in competition binding experiments using 2 nM [<sup>3</sup>H]<b>6</b> and membrane preparations of CHO cells recombinantly expressing hGPR84. Screenings were performed at a concentration of 10 μM.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values (inhibition of radioligand binding).</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Max. stimulation (%) of radioligand binding relative to the max. binding of [<sup>3</sup>H]<b>6</b> in absence of test compound (DMSO-control (=100%)).</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last"><i>n</i> = 4.</p></div></div><div></div></div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Competition binding experiments at membrane preparations of recombinant CHO cells expressing human GPR84. Allosteric modulation of radioligand binding by 3,3′-diindolylmethane derivatives. Data points represent means ± SEM from three to seven independent experiments, each performed in duplicates. For EC<sub>50</sub> and IC<sub>50</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There was a certain correlation of EC<sub>50</sub> values determined in cAMP accumulation assays with EC<sub>50</sub> values determined in radioligand binding studies (<i>R</i><sup>2</sup> = 0.433, see <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). For example, the most potent GPR84 agonist from this series, the tetrafluoro-DIM <b>9</b>, showed almost the same EC<sub>50</sub> value in cAMP assays as in the binding study (41.3 nM, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, as compared to 41.5 nM, <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Correlation of pEC<sub>50</sub> values of DIM derivatives determined in cAMP assays (●) or β-arrestin assays (■) with their respective p<i>K</i><sub>i</sub> values determined in radioligand binding assays. The same cell line (CHO-β-arrestin-human GPR84 cells) was used for all assays. <i>R</i><sup>2</sup> was determined in a two-tailed Pearson test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Most compounds displayed a difference of <6-fold in both assays. There were a few exceptions: 5,5′-dichloro-DIM (<b>90</b>, 20-fold), 5,5′-dibromo-DIM (<b>91</b>, 58-fold), and 5,5′-difluoro-6,6′-dichloro-DIM (<b>95</b>, 84-fold), which were considerably more potent in the binding as compared to the cAMP assay. If those three outliers were excluded, the correlation was much better (<i>R</i><sup>2</sup> = 0.790).</div><div class="NLM_p last">In contrast, data from binding studies did not at all correlate with EC<sub>50</sub> values determined in β-arrestin assays (see <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). A few DIM derivatives and analogs, compounds <b>93</b>, <b>98</b>, <b>102</b> and <b>103</b>, which had previously been found to be inactive in cAMP assays, but observed to be weak agonists in the β-arrestin assays at GPR84, were found to inhibit rather than increase radioligand binding (see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). These data underscore the complexity of the interactions of these ago-allosteric modulators with GPR84.</div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Molecular Modeling Studies</h3><div class="NLM_p last">Data from radioligand binding studies represent an ideal basis for molecular modeling studies since they are derived from the direct interaction of the compounds with the receptor.</div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Homology Model of the Human GPR84</h3><div class="NLM_p">In order to rationalize the binding modes of the known GPR84 agonists decanoic acid (<b>1</b>), embelin (<b>2</b>), 6-octylaminouracil (<b>3</b>) and its derivatives <b>6</b> and <b>7</b>, and their interactions with the human GPR84, a homology model of the receptor was generated. Among the different GPCR sequences determined by Basic Local Alignment Search Tool (BLAST) search against the Protein Data Bank (PDB), the human dopamine D3 receptor was selected,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> which had been co-crystallized with eticlopride (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Although the amino acid sequence of the human GPR84 and the dopamine D3 receptor displays a low degree of identity (24.6%), the sequence alignment based on the transmembrane motifs both feature a large intracellular loop 3 (ICL3). This supported the selection of the human dopamine D3 receptor as a template for generating the homology model of the human GPR84. The sequence alignment between the human GPR84 and the human dopamine D3 receptor is shown in <a class="ref internalNav" href="#notes-3" aria-label="Figure S18">Figure S18</a>. Previous attempts by Nikaido et al. to generate a homology model for the human GPR84 had been based on the crystal structure of the human β<sub>2</sub>-adrenergic receptor whose X-ray structure was available at that time.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Recently, another homology model of the human GPR84 was generated by Tikhonova et al. using a combination of structures of the human orexin receptor OX1 with the extracellular loop 2 (ECL2) of rhodopsin.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In the present study, we selected the dopamine D3 receptor and aligned it using ClustalO to GPR84 with the transmembrane motifs as well as the loop regions, and manually adjusted it to improve the alignment. From 100 generated models, the presented homology model of the human GPR84 was selected based on the Discrete Optimized Protein Energy (DOPE) score included in Modeller9. Generation of the Ramachandran plot gave seven residues having φ/ψ angles in the disallowed ranges, while the percentage of residues having φ/ψ angles in the most favorable range was around 98%, confirming good stereochemical quality of the selected model (see <a class="ref internalNav" href="#notes-3" aria-label="Figure S19">Figure S19</a>).<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> PROSA II profile analysis verified the sequence structure compatibility of the model with a Z-score of −2.52 (see <a class="ref internalNav" href="#notes-3" aria-label="Figure S20">Figure S20</a>).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><div class="NLM_p">Due to the horizontal binding orientation of eticlopride in the human dopamine D3 receptor crystal structure used as a template, the putative orthosteric binding pocket of the generated human GPR84 model was partially closed. This was mainly due to the predicted conformation of the ECL2 that partially projected into the putative orthosteric binding site of the receptor model. Thus, the best model of the human GPR84 selected based on the DOPE score was further refined by using the loop modeler module implemented in Molecular Operating Environment (MOE 2014.09).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> This led to the identification of the putative orthosteric binding site of the human GPR84 with a volume of 173 Å<sup>3</sup> using the SiteFinder module from MOE 2014.09. The important amino acids in the orthosteric binding site of the human GPR84 are shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. Interestingly, no positively charged residues as in the fatty acid receptors FFA1-3 (Arg185 and Arg258), or in the fatty acid receptor FFA4 (Arg99), were observed in the putative orthosteric binding site of GPR84. This shows that the binding site of the human GPR84 is very different from that of FFAR1–4.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Homology model of the human GPR84. (A) Homology model of the human GPR84 based on the human dopamine D3 receptor crystal structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>). (B) The putative orthosteric binding site of the human GPR84 predicted using SiteFinder implemented in MOE is schematically indicated and depicted as red spheres. (C) The important amino acid residues in the presumed orthosteric binding pocket are shown. Carbon atoms are colored in gray, oxygen atoms in red, nitrogen atoms in blue, and sulfur atoms in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Docking Studies of the Agonists</h3><div class="NLM_p">In order to rationalize the binding affinities of the agonists for GPR84 determined in radioligand binding studies, the binding modes of the potent uracil derivatives <b>3</b>, <b>6</b>, and <b>7</b> were predicted using molecular docking studies. For comparison, the standard GPR84 agonists decanoic acid (<b>1</b>) and embelin (<b>2</b>) were docked into the binding site of the receptor. In an earlier mutagenesis study by Nikaido et al. based on their homology model of the human GPR84, the authors investigated the role of nine amino acids located in the putative orthosteric binding site of the receptor.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Among these, mutation of Leu100 (L100D), Phe101 (F101Y), Asn104 (N104Q), or Asn357 (N357D) resulted in a significant reduction or even a complete loss of the activity of the agonist decanoic acid. In the present docking study, we utilized the published mutagenesis data for selecting the putative binding poses of the selected agonists. The resulting proposed docking poses of <b>1</b>–<b>3</b>, <b>6</b>, and <b>7</b> are shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>.</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Docking studies of selected agonists at the human GPR84 model (based on the human dopamine D3 receptor crystal structure, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>). The docked poses of decanoic acid (<b>1</b>, green), embelin (<b>2</b>, marine blue), and uracil derivatives <b>3</b> (cyan), <b>6</b> (yellow), and <b>7</b> (orange) in the orthosteric binding pocket (surface representation, colored maroon red) of the human GPR84 model are displayed in cartoon representation (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A depicts the amino acid residues in the putative binding site. The proposed interactions for the selected agonists <b>1</b>, <b>2</b>, and <b>6</b> are shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B–D, while those for compounds <b>3</b> and <b>7</b> are depicted in <a class="ref internalNav" href="#notes-3" aria-label="Figure S29A,B">Figure S29A,B</a>. The depicted amino acid residues in the binding pocket of GPR84 are predicted to be important for interactions with all of these lipid-like agonists. The compounds are proposed to be anchored inside the binding cleft by hydrogen-bond interactions with Tyr69, Asn104, and Asn357. The carboxylate function of <b>1</b> likely forms interactions with Tyr69, Asn104, and Asn357. In embelin (<b>2</b>), the C1-keto group is presumed to form interactions with Tyr69 and Asn104, and the 2-hydroxy group likely forms interactions with Asn357. Similar interactions were observed for the uracil derivatives <b>3</b>, <b>6</b>, and <b>7</b> in the model: The C2-carbonyl group is proposed to form interactions with Tyr69 and Asn104 and the N1–H to interact with Asn357. The space between the C4-carbonyl group in uracil derivatives and the nearby amino acids in the binding pocket suggests a water-mediated interaction. Similarly, the OH-group at position 2 of embelin might be deprotonated and form interactions with Asn357 in the binding site. Furthermore, in uracil derivatives <b>3</b>, <b>6</b>, and <b>7</b>, the 6-NH group is proposed to form a strong interaction with Asn357, and the N3–H possibly forms interactions with Asn104 located at a distance of ∼3 Å. These two interactions could not be observed for the GPR84 agonists decanoic acid (<b>1</b>) and embelin (<b>2</b>) in our model. This could explain why higher binding affinities are obtained for the uracil derivatives (e.g., <b>3</b>, <i>K</i><sub>i</sub> 0.626 nM) in comparison to <b>1</b> (<i>K</i><sub>i</sub> 1780 nM) and <b>2</b> (<i>K</i><sub>i</sub> 23.6 nM). The residues Leu100, Phe101, and Trp360 form a hydrophobic subpocket, which likely interacts with the lipophilic GPR84 agonists. Possibly, this hydrophobic interaction is stronger for embelin (<b>2</b>) and the uracil derivatives as compared to the weakly potent decanoic acid (<b>1</b>). The findings of a hydrophobic interaction of Leu100 and Phe101 with <b>1</b> are in agreement with the experimental mutagenesis data obtained by Nikaido et al. because the mutants L100D, L100N, and F101Y resulted in a very weak activity possibly due to impairment of these hydrophobic interactions.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Furthermore, the mutation of N104Q might break the hydrogen-bond interaction between the carboxylate group of <b>1</b> and Asn104. Interestingly, another mutant, N357D, resulted in an almost complete loss of activity of agonist <b>1</b>, possibly due to breaking the hydrogen-bond interaction between the carboxylate and Asn357. The previously published homology model of the human GPR84 that was based on the β<sub>2</sub>-adrenergic receptor focused on positively charged residues that could anchor the carboxylate function of free fatty acids in the receptor.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> However, a negative charge is not required for high GPR84 affinity. Therefore, the current study focused on the published mutagenesis data as well as the structure–activity relationships observed in binding studies, and the proposed model is well in agreement with those data. The long alkyl chains plausibly bind within the cleft that is largely formed by the hydrophobic amino acids Leu84, Leu100, Thr167, Cys168, Phe170, Leu336, Ile340, and Met353. The alkyl chain plays a major role in anchoring the molecules in order to form key interactions with the residues Tyr69, Asn104, and Asn357 in the binding site. For example, we found in the series of uracil derivatives that the binding affinity of the alkyl chain length of five carbon atoms (<b>28</b>, <i>K</i><sub>i</sub> 91.0 nM) is significantly increased with an additional methylene unit in <b>6</b> (<i>K</i><sub>i</sub> 2.98 nM). Further extension of the alkyl chain length in derivatives <b>3</b> and <b>29</b>–<b>31</b> led to a further, moderate increase in binding affinity (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Moreover, introducing an aromatic residue into the alkyl chain in <b>6</b> results in effective occupation of the cleft forming strong hydrophobic interactions with the amino acids in the binding site, resulting in a <i>K</i><sub>i</sub> value of 0.317 nM for agonist <b>7</b>. The large bromo substituent in <b>7</b> may further improve hydrophobic interactions with the amino acid residues of Leu84, Thr167, Cys168, and Phe170. This was confirmed by radioligand binding data showing a reduction in affinity in the following rank order: 4-Br (<b>7</b>, <i>K</i><sub>i</sub> 0.317 nM) ≥ 4-Cl (<b>37</b>, <i>K</i><sub>i</sub> 0.650 nM) > 4-F (<b>36</b>, <i>K</i><sub>i</sub> 11.4 nM).</div><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0010.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Binding site residues of the human GPR84 model (based on the human dopamine D3 receptor crystal structure, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>) and putative binding modes of agonists. (A) Important residues in the binding pocket are shown. The amino acid residues forming hydrogen-bond interactions are labeled in red color, hydrophobic interactions are in blue, and the mutants published by Nikaido et al. are highlighted by red boxes.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Binding poses of agonists in the putative orthosteric binding site: (B) decanoic acid (<b>1</b>, green); (C) embelin (<b>2</b>, marine blue); (D) 6-hexylaminouracil (<b>6</b>, yellow). Red dotted lines indicate hydrogen-bond interactions (for further color code see <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The recent study by Mahmud et al. proposed a different binding site and mode of interaction for agonists <b>1</b>, <b>2</b>, and <b>4</b> on the basis of a hybrid homology model of the human GPR84 generated using the OX1 receptor and the rhodopsin crystal structures.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The authors proposed that the amino acid residue Arg172 in the ECL2 of GPR84 is oriented toward the proposed orthosteric binding site and possibly coordinates the agonists <b>1</b>, <b>2</b>, and <b>4</b>. To confirm this hypothesis, an R172A mutant was generated, and the above-mentioned agonists were examined using [<sup>35</sup>S]GTPγS binding studies.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In that mutant, the affinity of the agonists was lost, but it was also lost for another mutant, R172K, in which Arg172 was replaced by another positively charged residue, a lysine. However, this homologous exchange did not lead to a retention or salvage, not even partially, of the agonists’ potency. Based on the published mutagenesis results, and according to the visualization of our generated homology model, and the performed docking studies, Arg172 does not appear to be directly interacting with the orthosteric agonists. Our current hypothesis is that Arg172 might contribute indirectly to agonist–receptor interaction by inducing a certain conformation in the ECL2 or by disturbing the interaction of Thr167 with agonists.</div></div><div id="sec3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Labeling of GPR84 in Native Cells and Tissues</h3><div class="NLM_p">Finally, we utilized the new radiotracer as a tool for detecting and quantifying GPR84 on the protein level in cells and tissues. It is expected to be advantageous, since antibodies for membrane proteins are often not selective. Therefore, we performed radioligand binding studies in different cell and organ preparations. High specific binding of [<sup>3</sup>H]<b>6</b> was detected in HepG2 (human hepatocarcinoma) cells, Jurkat (human T-lymphocyte) cells, HEK293 (human embryonic kidney) cells, and native human T-lymphocytes, while in MCA-RH7777 (rat buffalo hepatoma) cells and C6 glioma (rat glioblastoma) cells, no specific binding was detectable. Moreover, we could observe specific [<sup>3</sup>H]<b>6</b> binding in liver tissues (calf, mouse, rat) as well as in rat brain cortex and striatum (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). Next, we performed competition experiments with selected tissues and cell lines in order to determine IC<sub>50</sub> values for 6-hexylaminouracil (<b>6</b>) and 6-(<i>p</i>-bromophenylethylamino)uracil (<b>7</b>) (see <a class="ref internalNav" href="#notes-3" aria-label="Figure S30">Figure S30</a>). The GPR84 agonist <b>6</b> displayed higher IC<sub>50</sub> values in all tested tissues and cell lines (8- to 76-fold) compared to the IC<sub>50</sub>/<i>K</i><sub>i</sub> value determined in CHO-human GPR84 cells (<a class="ref internalNav" href="#notes-3" aria-label="Table S3">Table S3</a>). This might be due to the presence of endogenous GPR84 agonists in the preparations. Although MCFAs were shown to activate GPR84, another, perhaps significantly more potent endogenous agonist might exist whose discovery is still awaited. Interestingly, in all tested human tissues and cell lines, <b>7</b> showed higher affinity than <b>6</b>, while in rodent tissues, the opposite was true. Here, <b>6</b> displayed 2 to 5-fold higher affinity than <b>7</b> indicating species differences (<a class="ref internalNav" href="#notes-3" aria-label="Table S3 and Figure S30">Table S3 and Figure S30</a>).</div><figure id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0011.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Specific binding of [<sup>3</sup>H]<b>6</b> to membrane preparations of various native tissues and cell lines. The experiments were performed at 25 °C using 10 nM [<sup>3</sup>H]<b>6</b> and 100 μg protein/vial, except for Jurkat cells and T-lymphocytes: 25 μg protein/vial; HepG2 cell: 2 nM RL. Agonist <b>7</b> was used for the determination of nonspecific binding. Data points represent means ± SEM from three to five independent experiments, each performed in duplicates. At low GPR84 expression levels, the percentage of nonspecific binding is relatively high, and the data are consequently less accurate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37188" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37188" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we developed the first GPR84 agonist radioligand [<sup>3</sup>H]<b>6</b> for studying the binding affinities of receptor ligands. [<sup>3</sup>H]<b>6</b> was found to exhibit high affinity for the human GPR84 along with a low degree of nonspecific binding. Its binding was discovered to be modulated by ions, Mg<sup>2+</sup> increasing, Na<sup>+</sup> reducing its binding affinity. As expected, lipid-like GPR84 agonists inhibited [<sup>3</sup>H]<b>6</b> binding, while DIM and the majority of its derivatives induced an increase in [<sup>3</sup>H]<b>6</b> binding, which provides the ultimate evidence for their positive allosteric modulation of the receptor. For 6-(ar)alkylamino-substituted uracil derivatives, the rank order of the determined affinities correlated mostly with EC<sub>50</sub> values obtained in cAMP assays, while only weak correlation was seen with EC<sub>50</sub> values determined in β-arrestin assays. A series of new lipid-like agonists (<b>62</b>–<b>68</b>, <b>73</b>–<b>77</b>, and <b>79</b>–<b>85</b>) was synthesized and evaluated in binding studies to analyze their SARs. Finally, we generated a homology model of the human GPR84, performed docking studies of selected agonists, and rationalized the observed structure–activity relationships. The predicted binding modes and interactions with the amino acids in the binding site including Leu100, Phe101, Asn104, and Asn357 were well in agreement with previously reported mutagenesis data. Moreover, we could show that [<sup>3</sup>H]<b>6</b> is useful to label human GPR84 in native tissues. The developed GPR84 agonist radioligand represents a powerful pharmacological tool to further explore this yet poorly characterized orphan receptor.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50211" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50211" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Methods</h3><div class="NLM_p last">All commercially available reagents were used as purchased (Acros, Alfa Aesar, Sigma-Aldrich, abcr or TCI). Solvents were used without additional purification or drying except for dichloromethane, which was distilled over calcium hydride. Thin-layer chromatography (TLC) using aluminum sheets with silica gel 60 F254 monitored the reactions (Merck). Column chromatography was performed with silica gel 0.060–0.200 mm, pore diameter ca. 6 nm. All synthesized compounds were finally dried in vacuum at 8–12 Pa (0.08–0.12 mbar) using a sliding vane rotary vacuum pump (Vacuubrand GmbH). <sup>1</sup>H and <sup>13</sup>C NMR data were collected on a Bruker Avance 500 MHz NMR spectrometer at 500 MHz (<sup>1</sup>H), or 126 MHz (<sup>13</sup>C), respectively. If indicated, NMR data were collected on a Bruker Ascend 600 MHz NMR spectrometer at 600 MHz (<sup>1</sup>H) or 151 MHz (<sup>13</sup>C), respectively. DMSO-<i>d</i><sub>6</sub> was employed as a solvent at 303 K, unless otherwise noted. Chemical shifts are reported in parts per million (ppm) relative to the deuterated solvent; that is, DMSO, δ <sup>1</sup>H: 2.49 ppm; <sup>13</sup>C: 39.7 ppm. Coupling constants <i>J</i> are given in Hertz and spin multiplicities are given as s (singlet), d (doublet), t (triplet), q (quartet), sext. (sextet), m (multiplet), br (broad). The purities of isolated products were determined by ESI-mass spectra obtained on an LCMS instrument (Applied Biosystems API 2000 LCMS/MS, HPLC Agilent 1100) using the following procedure: The compounds were dissolved at a concentration of 1.0 mg/mL in acetonitrile containing 2 mM ammonium acetate. Then, 10 μL of the sample was injected into an HPLC column (Macherey-Nagel Nucleodur 3 μ C18, 50 × 2.00 mm). Elution was performed with a gradient of water/acetonitrile (containing 2 mM ammonium acetate) from 90:10 to 0:100 for 20 min at a flow rate of 300 μL/min, starting the gradient after 10 min. UV absorption was detected from 200 to 950 nm using a diode array detector (DAD). Purity of all compounds was determined at 254 nm. Compounds <b>1</b>, <b>2</b>, and <b>16</b>–<b>27</b> were commercially available. Compounds <b>12</b> and <b>13</b> were synthesized according to the reported procedure.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The syntheses of compounds <b>3</b>,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a><b>4</b>,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a><b>5</b>,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> 6,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a><b>7</b>,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a><b>28</b>–<b>60</b>,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and <b>8</b>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><b>9</b>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><b>86</b>–<b>103</b><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> were previously described. Purities of all products were determined by HPLC-UV-MS and proven to be >95%.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Synthesis of (<i>E</i>)-6-(hex-2-en-1-ylamino)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>15</b>)</h3><div class="NLM_p last">To the solution of <b>14</b> (1.0 g, 6.8 mmol) in 1-butanol (20 mL) was added (<i>E</i>)-hex-2-en-1-amine<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> (<b>13</b>, 2.03 g, 20.5 mmol, 3 equiv). The resulting solution was heated up to 125 °C and stirred under reflux for 12 h. After cooling to room temperature, 1-butanol was evaporated under the reduced pressure. The resulting residue was washed with cold 1-butanol (10 mL) and diethyl ether (10 mL), and the solid was dried under vacuum for 6 h.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Synthesis of 6-(hexylamino)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>6</b>)</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Procedure A</h4><div class="NLM_p last">To the solution of compound <b>15</b> (0.035 g, 0.167 mmol) in MeOH:THF (1:1), Pd/C (10%) (0.006 g) was added, and the mixture was thrice evacuated and flushed with hydrogen (H<sub>2</sub>). The reaction was performed at 25 psi (1.7 bar) for 2 h at room temperature. After completion of the reaction, the mixture was filtered through Celite, washed with MeOH:THF (1:1) mixture (20 mL), and evaporated to get the compound <b>6</b>.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Procedure B</h4><div class="NLM_p last">To the solution of compound <b>15</b> (0.040 g, 0.191 mmol) in DMF:THF (1:1), Pd/C (10%) (0.016 g) was added, and the mixture was thrice evacuated and flushed with hydrogen (H2). The reaction was performed at 35 psi (2.4 bar) for 2 h at room temperature. After completion of the reaction, the mixture was filtered through Celite, washed with DMF:THF (1:1) mixture (20 mL), and evaporated to remove THF. The compound in DMF was treated with water (10 mL), and the resulting solid was filtered and dried under vacuum to produce <b>6</b>, which characterized in a previous report.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Synthesis of <b>4</b>, <b>62</b>–<b>68</b>, and <b>73</b>–<b>76</b></h3><div class="NLM_p last">To the solution of 2-mercaptopyrimidine-4,6-diol (<b>61</b>, 1.0 mmol), 2-mercaptopyrimidin-4-ol (<b>69</b>, 1.0 mmol), 2-mercapto-6-methylpyrimidin-4-ol (<b>70</b>, 1.0 mmol), 2-mercapto-6-propylpyrimidin-4-ol (<b>71</b>, 1.0 mmol), or 2-mercaptopyrimidine-4-carboxylic acid (<b>72</b>, 1.0 mmol) and aryl(alkyl)bromides (2 mmol) in methanol, K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) was added. The resulting solution was refluxed under a nitrogen atmosphere for 12 h. The reaction mixture was allowed to cool to room temperature, and methanol was evaporated under reduced pressure to the dryness. The residue was dissolved in ethyl acetate (100 mL) and washed with water (50 mL). The organic layer was dried over MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford a crude product, which was purified by silica gel column chromatography (DCM:MeOH = 99:5) to afford the desired products <b>4</b>, <b>62</b>–<b>68</b>, and <b>73</b>–<b>76</b>.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Synthesis of <b>77</b></h3><div class="NLM_p last">To the solution of <b>76</b> (0.5 mmol) in methanol (20 mL), 2 N NaOH was added (1 mL), and resulting solution was refluxed for 1 h. The reaction mixture was allowed to cool to room temperature, and methanol was evaporated under reduced pressure to the dryness. The residue was dissolved in water (10 mL) and acidified with 2 N HCl (20 mL). The resulting precipitate was filtered, washed with water (20 mL), and dried under open air for 24 h.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Synthesis of <b>79</b> and <b>80</b></h3><div class="NLM_p last">To the solution of <b>78</b> (10 mmol) in appropriated alcohol (20 mL), concentrated H<sub>2</sub>SO<sub>4</sub> (0.1 mmol) was added under ice cooling condition. The solution was warmed up and refluxed for 12 h. The reaction mixture was allowed to cool to room temperature, and ethyl acetate (100 mL) and water (100 mL) were added. The organic layer was separated, dried over MgSO<sub>4</sub>, filtered, and concentrated to afford a crude residue, which was purified by silica gel column chromatography (DCM:MeOH = 98:2) to afford the desired product <b>79</b> or <b>80</b>.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Synthesis of <b>81</b>–<b>85</b></h3><div class="NLM_p last">To the solution of <b>78</b> (10 mmol), appropriate amine (10.5 mmol), HATU (10.5 mmol) in DMF (15 mL), triethylamine (25 mmol) was added slowly. The resulting solution was stirred at room temperature for 12 h. The mixture was poured onto ice water and separated with ethyl acetate (3 × 50 mL). The combined organic layers were washed with brine solution, dried over MgSO4, filtered, and concentrated to afford a crude residue, which was purified by silica gel column chromatography (DCM:MeOH = 95:5) to afford the desired products <b>81</b>–<b>85</b>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 2-(Hexylthio)-6-hydroxypyrimidin-4(3<i>H</i>)-one (<b>4</b>)</h3><div class="NLM_p last">Compound <b>6</b> was synthesized from the reaction of 6-hydroxy-2-mercaptopyrimidin-4(3<i>H</i>)-one (<b>61</b>, 1.0 mmol) with 1-bromohexane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield 37%: colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.65 (s, 2H), 5.10 (s, 1H), 3.07 (t, <i>J</i> = 7.2 Hz, 2H), 1.60 (p, <i>J</i> = 7.3 Hz, 2H), 1.48–1.18 (m, 6H), 1.10–0.47 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 85.70, 30.85, 29.66, 28.81, 27.90, 22.11, 13.98. LC-MS (<i>m</i>/<i>z</i>): positive mode 229 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 98.0%.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (<i>E</i>)-6-(Hex-2-en-1-ylamino)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>15</b>)</h3><div class="NLM_p last">Yield: 91%; pale yellow solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.10 (s, 1H), 9.32 (s, 1H), 6.19 (t, <i>J</i> = 5.7 Hz, 1H), 5.70–5.50 (m, 1H), 5.50–5.24 (m, 1H), 4.39 (s, 1H), 3.59 (t, <i>J</i> = 5.8 Hz, 2H), 1.98 (q, <i>J</i> = 6.9 Hz, 2H), 1.35–1.33 (m, <i>J</i> = 7.3 Hz, 2H), 0.87 (m, 3H). <sup>13</sup>Capt NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.30, 154.06, 150.91, 132.77, 125.60, 73.12, 43.25, 33.76, 21.90, 13.58. LC-MS (<i>m</i>/<i>z</i>): positive mode 210 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 98.5%.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 6-Hydroxy-2-((3-phenylpropyl)thio)pyrimidin-4(3<i>H</i>)-one (<b>62</b>)</h3><div class="NLM_p last">Compound <b>62</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 1-bromo-3-phenylpropane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 56%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.73 (s, 2H), 7.29 (t, <i>J</i> = 7.7 Hz, 2H), 7.20 (dd, <i>J</i> = 17.9, 7.7 Hz, 3H), 3.10 (t, <i>J</i> = 7.1 Hz, 2H), 2.70 (t, <i>J</i> = 7.7 Hz, 2H), 1.95 (p, <i>J</i> = 7.4 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 141.28, 128.52, 128.45, 126.06, 85.78, 34.22, 30.53, 29.37. LC-MS (<i>m</i>/<i>z</i>): positive mode 263 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 98.9%.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 6-Hydroxy-2-((3-phenylbutyl)thio)pyrimidin-4(3<i>H</i>)-one (<b>63</b>)</h3><div class="NLM_p last">Compound <b>63</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 1-bromo-4-phenylbutane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 62%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.61 (s, 2H), 7.38–7.22 (m, 2H), 7.22–6.99 (m, 3H), 5.11 (s, 1H), 3.11 (t, <i>J</i> = 6.6 Hz, 2H), 2.59 (t, <i>J</i> = 7.1 Hz, 2H), 1.65 (pt, <i>J</i> = 5.0, 2.9 Hz, 4H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 142.02, 128.37, 128.36, 125.81, 85.70, 34.67, 30.05, 29.41, 28.53. LC-MS (<i>m</i>/<i>z</i>): positive mode 277 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.3%.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 2-((4-Fluorophenethyl)thio)-6-hydroxypyrimidin-4(3<i>H</i>)-one (<b>64</b>)</h3><div class="NLM_p last">Compound <b>64</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 4-fluorophenethyl bromide (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 72%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-d<sub>6</sub>) δ 7.50–7.22 (m, 2H), 7.22–6.87 (m, 2H), 5.14 (s, 1H), 3.30 (d, <i>J</i> = 6.1 Hz, 2H), 2.92 (dd, <i>J</i> = 8.5, 6.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-d<sub>6</sub>) δ 160.13, 136.25, 136.22, 130.65, 130.59, 115.21, 115.04, 85.77, 34.03, 31.03. LC-MS (<i>m</i>/<i>z</i>): positive mode 267 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 95.7%.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 2-((4-Clorophenethyl)thio)-6-hydroxypyrimidin-4(3<i>H</i>)-one (<b>65</b>)</h3><div class="NLM_p last">Compound <b>65</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 4-chlorophenethyl bromide (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 69%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.39–7.33 (m, 2H), 7.33–7.27 (m, 2H), 5.14 14 (s, 1H), 3.30 (s, 2H merged with H<sub>2</sub>O peak), 2.93 (dd, <i>J</i> = 8.5, 6.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 139.09, 131.13, 130.70, 128.36, 85.76, 34.13, 30.77. LC-MS (<i>m</i>/<i>z</i>): positive mode 283 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 98.7%.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 2-((4-Bromophenethyl)thio)-6-hydroxypyrimidin-4(3<i>H</i>)-one (<b>66</b>)</h3><div class="NLM_p last">Compound <b>66</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 4-bromophenethyl bromide (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 57%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.68–7.31 (m, 2H), 7.39–6.97 (m, 2H), 5.14 (s, 1H), 3.32 (s, 2H), 2.92 (dd, <i>J</i> = 8.5, 6.6 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 139.51, 131.29, 131.10, 119.58, 85.76, 34.18, 30.70. LC-MS (<i>m</i>/<i>z</i>): positive mode 328 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 96.9%.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 2-((4-Methylphenethyl)thio)-6-hydroxypyrimidin-4(3<i>H</i>)-one (<b>67</b>)</h3><div class="NLM_p last">Compound <b>67</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 4-methylphenethyl bromide (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 77%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.16 (d, <i>J</i> = 8.1 Hz, 2H), 7.10 (d, <i>J</i> = 7.7 Hz, 2H), 5.13 (s, 1H), 3.35–3.29 (m, 3H), 2.88 (dd, <i>J</i> = 8.6, 6.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.30, 162.96, 136.98, 135.46, 129.03, 128.63, 85.76, 45.92, 34.43. LC-MS (<i>m</i>/<i>z</i>): positive mode 263 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.0%.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 6-Hydroxy-2-((4-(trifluoromethyl)phenethyl)thio)pyrimidin-4(3<i>H</i>)-one (<b>68</b>)</h3><div class="NLM_p last">Compound <b>68</b> was synthesized from the reaction of <b>61</b> (1.0 mmol) with 4-trifluoromethylphenethyl bromide (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield: 64%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.66 (d, <i>J</i> = 8.0 Hz, 2H), 7.52 (d, <i>J</i> = 7.9 Hz, 2H), 5.15 (s, 1H9; 3.36 (dd, <i>J</i> = 8.5, 6.6 Hz, 4H), 3.05 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 154.53, 144.98, 129.67, 85.77, 34.59, 30.50. LC-MS (<i>m</i>/<i>z</i>): positive mode 317 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 95.6%.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 2-(Hexylthio)pyrimidin-4-ol (<b>73</b>)<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></h3><div class="NLM_p last">Compound <b>73</b> was synthesized from the reaction of 4-hydroxy-pyrimidine-2(1H)-thione (<b>69</b>, 1.0 mmol) with 1-bromohexane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. The compound was characterized in a previous report. LC-MS (<i>m</i>/<i>z</i>): positive mode 213 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 96.6%.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 2-(Hexylthio)-6-methylpyrimidin-4-ol (<b>74</b>)<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></h3><div class="NLM_p last">Compound <b>74</b> was synthesized from the reaction of 4-hydroxy-6-methylpyrimidine-2(1H)-thione (<b>70</b>, 1.0 mmol) with 1-bromohexane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. The compound was characterized in a previous report. LC-MS (<i>m</i>/<i>z</i>): positive mode 227 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.3%.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 2-(Hexylthio)-6-propylpyrimidin-4-ol (<b>75</b>)</h3><div class="NLM_p last">Compound <b>75</b> was synthesized from the reaction of 4-hydroxy-6-propylpyrimidine-2(1H)-thione (<b>71</b>, 1.0 mmol) with 1-bromohexane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. Yield 61%: pale yellow solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.41 (s, 1H), 5.91 (s, 1H), 3.10 (t, <i>J</i> = 7.3 Hz, 2H), 2.39 (t, <i>J</i> = 7.4 Hz, 2H), 1.63 (ddt, <i>J</i> = 14.9, 10.8, 7.1 Hz, 4H), 1.41–1.25 (m, 5H), 0.88 (dt, <i>J</i> = 11.3, 7.2 Hz, 5H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.22, 167.27, 164.07, 106.81, 30.82, 29.67, 29.01, 27.89, 22.05, 20.64, 13.96, 13.55. LC-MS (<i>m</i>/<i>z</i>): positive mode 255 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.7%.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Methyl 2-(Hexylthio)-6-hydroxypyrimidine-4-carboxylate (<b>76</b>)<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></h3><div class="NLM_p last">Compound <b>76</b> was synthesized from the reaction of 6-hydroxy-2-thioxo-2,3-dihydropyrimidine-4-carboxylic acid (<b>72</b>, 1.0 mmol) with 1-bromohexane (2.0 mmol) in the presence of K<sub>2</sub>CO<sub>3</sub> (3.0 mmol) in methanol. The compound was characterized in a previous report. LC-MS (<i>m</i>/<i>z</i>): positive mode 271 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 99.0%.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 2-(Hexylthio)-6-hydroxypyrimidine-4-carboxylic acid (<b>77</b>)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></h3><div class="NLM_p last">Compound <b>77</b> was synthesized by reaction of <b>76</b> with 2 N NaOH. The compound was characterized in a previous report. LC-MS (<i>m</i>/<i>z</i>): positive mode 257 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 98.2%.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Butyl 2,6-Dioxo-1,2,3,6-tetrahydropyrimidine-4-carboxylate (<b>79</b>)</h3><div class="NLM_p last">Compound <b>79</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) and butanol (20 mL) in the presence of concentrated H<sub>2</sub>SO<sub>4</sub> (0.1 mmol). Yield: 64%; brown solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 11.09 (s, 1H), 6.03 (s, 1H), 4.25 (t, <i>J</i> = 6.5 Hz, 2H), 1.75–1.53 (m, 2H), 1.53–1.19 (m, 2H), 0.90 (t, <i>J</i> = 7.4 Hz, 3H, −CH<sub>3</sub>). <sup>13</sup>C NMR (126 MHz, DMSO-d<sub>6</sub>) δ 163.89, 160.44, 150.94, 141.63, 103.66, 66.38, 29.91, 18.65, 13.65. LC-MS (<i>m</i>/<i>z</i>): positive mode 213 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 95.5%.</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Hexyl 2,6-Dioxo-1,2,3,6-tetrahydropyrimidine-4-carboxylate (<b>80</b>)</h3><div class="NLM_p last">Compound <b>80</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) and hexanol (20 mL) in the presence of concentrated H<sub>2</sub>SO<sub>4</sub> (0.1 mmol). Yield: 73%; brown solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 11.09 (s, 1H), 6.02 (s, 1H), 4.24 (t, J = 6.5 Hz, 2H), 1.85–1.58 (m, 2H), 1.45–1.07 (m, 6H), 1.05–0.64 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 63.87, 160.43, 150.93, 141.62, 103.64, 66.66, 30.94, 27.81, 25.03, 22.08, 13.98. LC-MS (<i>m</i>/<i>z</i>): positive mode 241 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 96.0%.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 2,6-Dioxo-<i>N</i>-(3-phenylpropyl)-1,2,3,6-tetrahydropyrimidine-4-carboxamide (<b>81</b>)</h3><div class="NLM_p last">Compound <b>81</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) with 3-phenylpropylamine (10 mmol) in the presence of HATU (10.5 mmol) and TEA (25 mmol) in DMF. Yield: 68%; brown solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.18 (s, 1H), 10.54 (s, 1H), 8.77 (t, <i>J</i> = 5.6 Hz, 1H), 7.30–7.24 (m, 2H), 7.24–7.19 (m, 2H), 7.19–7.13 (m, 1H), 5.99 (s, 1H), 3.20 (td, <i>J</i> = 7.0, 5.5 Hz, 2H), 2.74–2.54 (m, 2H), 1.78 (tt, <i>J</i> = 7.6, 6.5 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.36, 160.23, 145.59, 141.67, 141.67, 128.45, 125.93, 125.93, 99.73, 32.58, 30.40. LC-MS (<i>m</i>/<i>z</i>): positive mode 274 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.5%.</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 2,6-Dioxo-<i>N</i>-pentyl-1,2,3,6-tetrahydropyrimidine-4-carboxamide (<b>82</b>)</h3><div class="NLM_p last">Compound <b>82</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) with pentylamine (10 mmol) in the presence of HATU (10.5 mmol) and TEA (25 mmol) in DMF. Yield: 59%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.20 (s, 1H), 10.55 (s, 1H), 8.93–8.15 (m, 1H), 5.99 (s, 1H), 3.17 (q, <i>J</i> = 6.7 Hz, 2H), 1.47 (p, <i>J</i> = 7.1 Hz, 2H), 1.27 (m, 4H), 0.86 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.31, 160.01, 150.89, 145.40, 99.74, 28.66, 28.35, 21.91, 13.99. LC-MS (<i>m</i>/<i>z</i>): positive mode 226 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 99.0%.</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> <i>N</i>-Hexyl-2,6-dioxo-1,2,3,6-tetrahydropyrimidine-4-carboxamide (<b>83</b>)</h3><div class="NLM_p last">Compound <b>83</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) with hexylamine (10 mmol) in the presence of HATU (10.5 mmol) and TEA (25 mmol) in DMF. Yield: 72%; brown solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.20 (s, 1H), 10.56 (s, 1H), 8.72 (t, <i>J</i> = 5.7 Hz, 1H), 5.98 (s, 1H), 3.17 (td, <i>J</i> = 7.1, 5.6 Hz, 2H), 1.73–1.36 (m, 2H), 1.26 (dd, <i>J</i> = 5.5, 3.4 Hz, 6H), 1.00–0.58 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.34, 160.04, 150.92, 145.45, 99.73, 31.05, 28.64, 26.15, 22.14, 14.03. LC-MS (<i>m</i>/<i>z</i>): positive mode 240 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.3%.</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <i>N</i>-Octyl-2,6-dioxo-1,2,3,6-tetrahydropyrimidine-4-carboxamide (<b>84</b>)</h3><div class="NLM_p last">Compound <b>84</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) with octylamine (10 mmol) in the presence of HATU (10.5 mmol) and TEA (25 mmol) in DMF. Yield: 65%; colorless solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.13 (s, 2H), 6.01 (s, 1H), 2.88–2.65 (m, 2H), 1.54 (tt, <i>J</i> = 7.9, 6.1 Hz, 2H), 1.37–1.05 (m, 10H), 1.03–0.64 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.31, 159.90, 150.88, 145.29, 99.90, 31.35, 31.28, 28.78, 28.73, 28.62, 28.60, 27.03, 26.48, 26.00, 22.18, 14.05. LC-MS (<i>m</i>/<i>z</i>): positive mode 268 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 98.5%.</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> <i>N</i>-Nonyl-2,6-dioxo-1,2,3,6-tetrahydropyrimidine-4-carboxamide (<b>85</b>)</h3><div class="NLM_p last">Compound <b>85</b> was synthesized from the reaction of orotic acid (<b>78</b>, 10 mmol) with nonylamine (10 mmol) in the presence of HATU (10.5 mmol) and TEA (25 mmol) in DMF.Yield: 61%; pale yellow solid; <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.11 (s, 2H), 6.01 (s, 1H); 2.90–2.57 (m, 2H), 1.79–1.41 (m, 2H), 1.41–1.11 (m, 12H), 1.11–0.19 (m, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.41, 160.12, 99.63, 31.38, 29.02, 28.91, 28.82, 28.76, 28.69, 28.66, 27.08, 25.97, 22.20, 14.06. LC-MS (<i>m</i>/<i>z</i>): positive mode 282 [M + H]<sup>1+</sup>; purity by HPLC UV (254 nm)-ESI-MS: 97.0%.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Biological Assays</h3><div class="NLM_p last">The recombinant CHO cell line expressing the human GPR84 (CHO-hGPR84 cells) with a β-galactosidase fragment and arrestin containing the complementary fragment of the enzyme for performing β-arrestin recruitment assays (Pathhunter) was purchased from DiscoverX (Fremont, CA). This cell line was used for the β-arrestin recruitment assays as well as for cAMP accumulation and radioligand binding assays. CHO-hGPR84 cells were cultured in F12 medium supplemented with 10% FCS, 100 units/mL penicillin G, 100 μg/mL streptomycin, 800 μg/mL G 418, 300 μg/mL hygromycin B, and 1% ultraglutamin (Invitrogen, Carlsbad, CA or Sigma-Aldrich, St. Louis, MO). Stock solutions of test compounds and of forskolin were prepared in DMSO. The final DMSO concentration in the assays did not exceed 1%. Data analysis was performed with GraphPad Prism (Version 6.02). Concentration–response data were fitted by nonlinear regression to estimate EC<sub>50</sub> values (Prism 6.02).</div></div><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Membrane Preparations</h3><div class="NLM_p">CHO-β-arrestin-hGPR84 cells were cultured in 150 cm<sup>2</sup> dishes in F12 medium supplemented with 10% FCS, 100 units/mL penicillin G, 100 μg/mL streptomycin, 800 μg/mL G 418, 300 μg/mL hygromycin B, and 1% ultraglutamin. Confluent cells were washed with 5 mL of ice-cold PBS, and the dishes were frozen at 80 °C. Membranes were prepared by adding 1 mL of ice-cold 25 mM Tris buffer, pH 7.4 containing 0.32 M sucrose, 1 mM EDTA, and protease inhibitors (1:100 dilution of the protease inhibitor cocktail, Sigma 8340) and scratching the cells off the previously frozen cell culture dishes. The collected cell suspension was homogenized with an Ultra Turrax and was subsequently centrifuged at 1000<i>g</i> for 10 min at 4 °C. The supernatant was carefully removed before being centrifuged at 48,000<i>g</i> for 30 min at 4 °C. The obtained pellet was resuspended in 50 mM Tris buffer, pH 7.4, homogenized with an Ultra Turrax, and centrifuged once again under the same conditions. The washing step was repeated once more. The final crude membrane pellet was resuspended in 50 mM Tris-HCl buffer (pH 7.4), homogenized in a glass/Teflon homogenizer, and stored at −80 °C in 1 mL aliquots until further use. Protein concentrations were determined according to the method of Lowry.</div><div class="NLM_p">McA-RH7777 cells and C6 glioma cells were provided by Prof. Dr. I. von Kügelgen (Institute of Pharmacology and Toxicology, University of Bonn, Bonn, Germany). For both cell lines, Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% FCS, 100 units/mL penicillin G, 100 μg/mL streptomycin, and 1% ultraglutamin was used. Human embryonic kidney (HEK)293 cells were purchased from the European Tissue Culture Collection (Salisbury, UK) and were cultured using MEM (minimum essential medium eagle) supplemented with 10% FCS, 100 units/mL penicillin G, 100 μg/mL streptomycin, 1% ultraglutamin, and 1% non-essential amino acids solution (NEAA). For HepG2 cells, DMEM, high glucose medium supplemented with 10% FCS, 100 units/mL penicillin G, and 100 μg/mL streptomycin were used. The preparation of the cell membranes from McA-RH7777, C6 glioma, HEK293, and HepG2 cells was performed as described above for CHO-β-arrestin-hGPR84 cells.</div><div class="NLM_p">Bovine brains and livers were obtained from a local abattoir. Mouse and rat tissues were obtained from control animals that had been used for other experiments and were sacrificed.</div><div class="NLM_p last">The following membrane preparations were performed essentially as previously described: lymphocyte isolation and membrane preparations of the lymphocytes,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> preparation of Jurkat-T cell membranes,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> brain cortical membrane preparations, and rat brain striatal membrane preparations,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> membranes of 1321N1 astrocytoma cells.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> GPR84 Radioligand Binding Assays</h3><div class="NLM_p last">The radioligand binding assays were performed in a 96-well format. CHO cell membranes expressing the human GPR84 receptor (10 μg protein per vial) were incubated for 150 min at 25 °C in 0.4 mL of a 50 mM Tris-HCl buffer (pH 7.4) containing 10 mM MgCl<sub>2</sub>, 0.05% fatty acid free BSA, 2 nM [<sup>3</sup>H]<b>6</b> (60 Ci/mmol), and increasing concentrations of test compound. Nonspecific binding (NSB) was determined in the presence of 10 μM of <b>6</b>. Membrane-bound and free radioligand were separated by rapid filtration (GF/B-glass fiber filter) using a Brandel 96-channel cell harvester (Brandel, Gaithersburg, MD, USA) through Packard 96-well GF/B glass fiber filter plates. Filters were rinsed three times, 2 mL each, with ice-cold 50 mM Tris-HCl buffer, pH 7.4. Radioactivity of the 96-well filter plates was counted after 6 h of preincubation with 50 μL of Microscint-20 scintillation cocktail (PerkinElmer, Rodgau, Germany). Data were analyzed using Graph Pad Prism Version 6.0 (San Diego, CA, USA). For the calculation of <i>K</i><sub>i</sub> values, the Cheng–Prusoff equation and a <i>K</i><sub>D</sub> value of 2.08 nM were used. Three independent experiments, each in duplicates were performed. The unpaired <i>t</i> test was used for statistical comparisons (*, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***<i>p</i> < 0.005; ****<i>p</i> < 0.001).</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> GPR84 cAMP Accumulation Assays</h3><div class="NLM_p last">The cAMP assays were performed essentially as previously described.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> In short, CHO-hGPR84 cells were stimulated with test compound dilutions (in DMSO, final concentration: 1%) or with 1% DMSO alone (control) for 5 min at 37 °C. Subsequently, forskolin (10 μM) was added, and the reaction mixtures were incubated for another 15 min at 37 °C. The reaction was stopped with hot (90 °C) lysis solution (4 mM EDTA, 0.01% Triton X-100 in water). Accumulated cAMP levels were quantified by a radioactive assay using [<sup>3</sup>H]cAMP (PerkinElmer, Rodgau, Germany). The forskolin-induced increase in cAMP concentration in the presence of agonist dilutions was expressed as percentage of the response to forskolin in the absence of agonists (% of control). Three independent experiments, each in duplicates, were performed.</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> GPR84 β-Arrestin Recruitment Assays</h3><div class="NLM_p last">β-Arrestin assays were performed essentially as previously described.<a onclick="showRef(event, 'ref24 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref26">(24,26)</a> Briefly, CHO-hGPR84 cells were incubated with compound dilutions (in DMSO, final concentration: 1%) for 90 min before adding the detection reagent (DiscoverX). The reaction mixture was further incubated for 60 min at rt. The luminescence was measured using an NXT plate reader (PerkinElmer, Rodgau, Germany). Three independent experiments were performed, each in duplicates.</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Three-Dimensional Structure of the Human GPR84</h3><div class="NLM_p last">A homology model of the human GPR84 receptor was created using Modeller9.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> The sequence of the human GPR84 was downloaded from the protein database, UniProtKB (Q9NQS5). The human GPR84 sequence comprised of 396 amino acid residues was submitted to Protein Basic Local Alignment Search Tool (BLAST) to search for the closest homologue in the Protein Data Bank (PDB). The templates identified using BLAST and the target sequence, the human GPR84, were aligned using the multiple sequence alignment tool, CLUSTALO.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Among the closest sequences identified, the human dopamine D3 receptor<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> contains a large intracellular loop 3 (ICL3) similar to GPR84. Thus, the human dopamine D3 receptor was selected as a template. The human GPR84 model was generated on the basis of the crystal structure of the human dopamine D3 receptor in complex with eticlopride (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The alignment between the human GPR84 and the dopamine D3 receptor was visually interpreted and manually fixed for further improvement. Each model was optimized using the variable target function method (VTFM) with conjugate gradients (CG), followed by refinement using molecular dynamics (MD) with a simulated annealing (SA) method implemented in Modeler. In the dopamine D3 receptor, the N-terminus and the ICL3 have not been crystallized, and therefore modeling of these regions was not attempted. From the 100 generated models, the best homology model was selected on the basis of the Discrete Optimized Protein Energy (DOPE) score included in Modeler. The extracellular loop 2 (ECL2) of the best model was improved by using the loop modeler module implemented in Molecular Operating Environment (MOE 2014.09).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> The putative orthosteric binding site was predicted by using the SiteFinder module in MOE2014.09. The overall structural quality was confirmed by a Ramachandran Plot, and sequence–structure compatibility of the model was ensured using PROSA II profile analysis.<a onclick="showRef(event, 'ref46 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref48">(46,48)</a> The protonation of the final model was done using the Protonate3D algorithm in MOE2014.09 followed by minimization with a root-mean-square of 0.5 Å.</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Molecular Docking</h3><div class="NLM_p last">The generated homology model of the human GPR84 was used for the docking procedure using AutoDock 4.2.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> The AutoDockTools package was employed to generate the docking input files and to analyze the docking results. Three-dimensional energy scoring grids for a box of 60 × 60 × 60 points with a spacing of 0.375 Å were computed. The grids were centered based on the predicted orthosteric binding site of the receptor. For each ligand, 100 independent docking calculations using the <i>var</i>CPSO-ls algorithm from <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="2e7e7d616e6f5b5a414a414d45">[email protected]</a><a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> implemented in AutoDock4.2 were performed and terminated after 500,000 evaluation steps. Parameters of <i>var</i>CPSO-ls algorithm, the cognitive and social coefficients c1 and c2 were set at 6.05 with 60 individual particles as a swarm size. Default values were used for all the other available parameters for the grid and docking calculations. High-scoring binding poses of lowest energy or more populated poses were selected for the analysis based on visual inspection.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i62"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01339" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01339?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01339</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Mass spectrometric analysis of [<sup>3</sup>H]<b>6</b> and of the unlabeled reference compound <b>6</b>; radiochemical purity of [<sup>3</sup>H]<b>6</b> determined by HPLP-UV (254 nm); product specification sheet of [<sup>3</sup>H]<b>6</b>; specific binding of [<sup>3</sup>H]<b>6</b> to CHO-K1 and CHO-hGPR17 membrane preparations; effects of sodium ions on binding of [<sup>3</sup>H]<b>6</b>. NMR and HPLC-UV/MS data for selected compounds. Overall sequence alignment of the human GPR84 and the human dopamine D3 receptor; Ramachandran diagram of the human GPR84 model; sequence–structure compatibility of the human GPR84 model; docked poses of compounds <b>3</b> and <b>7</b> in the binding site of the GPR84 model; competition binding experiments of compounds <b>6</b> and <b>7</b> at native human and rodent tissues and cell lines. Molecular SMILES strings and EC<sub>50</sub> values (CSV) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">structure of the human dopamine D3 receptor (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_002.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_001.pdf">jm9b01339_si_001.pdf (4.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_002.pdb">jm9b01339_si_002.pdb (348.46 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The authors will release the atomic coordinates of the homology model of the human GPR84 based on the X-ray structure of the human dopamine D3 receptor (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>) upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01339" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53721" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53721" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christa E. Müller</span> - <span class="hlFld-Affiliation affiliation">PharmaCenter
Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal
Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0013-6624" title="Orcid link">http://orcid.org/0000-0002-0013-6624</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#47242f352e343326692a32222b2b22350732292e6a25282929692322"><span class="__cf_email__" data-cfemail="b5d6ddc7dcc6c1d49bd8c0d0d9d9d0c7f5c0dbdc98d7dadbdb9bd1d0">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Meryem Köse</span> - <span class="hlFld-Affiliation affiliation">PharmaCenter
Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal
Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3391-3418" title="Orcid link">http://orcid.org/0000-0002-3391-3418</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thanigaimalai Pillaiyar</span> - <span class="hlFld-Affiliation affiliation">PharmaCenter
Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal
Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5575-8896" title="Orcid link">http://orcid.org/0000-0001-5575-8896</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vigneshwaran Namasivayam</span> - <span class="hlFld-Affiliation affiliation">PharmaCenter
Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal
Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3031-3377" title="Orcid link">http://orcid.org/0000-0003-3031-3377</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabetta De Filippo</span> - <span class="hlFld-Affiliation affiliation">PharmaCenter
Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal
Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katharina Sylvester</span> - <span class="hlFld-Affiliation affiliation">PharmaCenter
Bonn, Pharmaceutical Institute, Department of Pharmaceutical and Medicinal
Chemistry, University of Bonn, An der Immenburg 4, D-53121 Bonn, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Trond Ulven</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Design and Pharmacology, University
of Copenhagen, Universitetsparken
2, DK-2100 Copenhagen, Denmark</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8135-1755" title="Orcid link">http://orcid.org/0000-0002-8135-1755</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ivar von Kügelgen</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, Pharma Center
Bonn, Sigmund-Freud-Str.
25, D-53127 Bonn, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e5032-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71680" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71680" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study was in part supported by the BMBF (Federal Ministery of Education and Research, Germany, BioPharma Neuroallianz). T.P. is grateful to the Alexander von Humboldt (AvH) foundation and to Bayer Pharma for supporting a postdoctoral fellowship. Expert technical contributions by Marion Schneider (LC-MS analyses), Sabine Terhart-Krabbe (NMR), and Annette Reiner (NMR) are gratefully acknowledged.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">cAMP</td><td class="NLM_def"><p class="first last">cyclic adenosine monophosphate</p></td></tr><tr><td class="NLM_term">CHO</td><td class="NLM_def"><p class="first last">Chinese hamster ovary</p></td></tr><tr><td class="NLM_term">DAD</td><td class="NLM_def"><p class="first last">diode array detector</p></td></tr><tr><td class="NLM_term">DIM</td><td class="NLM_def"><p class="first last">diindolylmethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>′-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">FFA/R</td><td class="NLM_def"><p class="first last">free fatty acid/receptor</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">GPR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HEK</td><td class="NLM_def"><p class="first last">human embryonic kidney</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">LC-MS</td><td class="NLM_def"><p class="first last">liquid chromatography-mass spectrometry</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">M/L/SCFA</td><td class="NLM_def"><p class="first last">medium/long/short chain fatty acid</p></td></tr><tr><td class="NLM_term">NFκB</td><td class="NLM_def"><p class="first last">nuclear factor kappa-light-chain-enhancer of activated B cells</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 62 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wittenberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellebrand, S.</span></span> <span> </span><span class="NLM_article-title">An expressed sequence tag (EST) data mining strategy succeeding in the discovery of new Gprotein-coupled receptors</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>307</i></span>,  <span class="NLM_fpage">799</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1006/jmbi.2001.4520</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1006%2Fjmbi.2001.4520" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=11273702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitFymu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2001&pages=799-813&author=T.+Wittenbergerauthor=H.+C.+Schallerauthor=S.+Hellebrand&title=An+expressed+sequence+tag+%28EST%29+data+mining+strategy+succeeding+in+the+discovery+of+new+Gprotein-coupled+receptors&doi=10.1006%2Fjmbi.2001.4520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">An Expressed Sequence Tag (EST) Data Mining Strategy Succeeding in the Discovery of New G-Protein Coupled Receptors</span></div><div class="casAuthors">Wittenberger, Timo; Schaller, H. Chica; Hellebrand, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">799-813</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">We have developed a comprehensive expressed sequence tag database search method and used it for the identification of new members of the G-protein coupled receptor superfamily.  Our approach proved to be esp. useful for the detection of expressed sequence tag sequences that do not encode conserved parts of a protein, making it an ideal tool for the identification of members of divergent protein families or of protein parts without conserved domain structures in the expressed sequence tag database.  At least 14 of the expressed sequence tags found with this strategy are promising candidates for new putative G-protein coupled receptors.  Here, we describe the sequence and expression anal. of five new members of this receptor superfamily, namely GPR84, GPR86, GPR87, GPR90 and GPR91.  We also studied the genomic structure and chromosomal localization of the resp. genes applying in silico methods.  A cluster of six closely related G-protein coupled receptors was found on the human chromosome 3q24-3q25.  It consists of four orphan receptors (GPR86, GPR87, GPR91, and H963), the purinergic receptor P2Y1, and the uridine 5'-diphosphoglucose receptor KIAA0001.  It seems likely that these receptors evolved from a common ancestor and therefore might have related ligands.  In conclusion, we describe a data mining procedure that proved to be useful for the identification and first characterization of new genes and is well applicable for other gene families.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2zESMaLlx3bVg90H21EOLACvtfcHk0ljrGwZTqbDhUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitFymu78%253D&md5=0e8fcf6476a2ff247e9813df05247df3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.4520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.4520%26sid%3Dliteratum%253Aachs%26aulast%3DWittenberger%26aufirst%3DT.%26aulast%3DSchaller%26aufirst%3DH.%2BC.%26aulast%3DHellebrand%26aufirst%3DS.%26atitle%3DAn%2520expressed%2520sequence%2520tag%2520%2528EST%2529%2520data%2520mining%2520strategy%2520succeeding%2520in%2520the%2520discovery%2520of%2520new%2520Gprotein-coupled%2520receptors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D799%26epage%3D813%26doi%3D10.1006%2Fjmbi.2001.4520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yousefi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarai, G.</span></span> <span> </span><span class="NLM_article-title">Cloning and expression analysis of a novel G protein-coupled receptor selectively expressed on granulocytes</span>. <i>J. Leukoc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1045</span>– <span class="NLM_lpage">1052</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=11404393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkslWrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2001&pages=1045-1052&author=S.+Yousefiauthor=P.+R.+Cooperauthor=S.+L.+Potterauthor=B.+Mueckauthor=G.+Jarai&title=Cloning+and+expression+analysis+of+a+novel+G+protein-coupled+receptor+selectively+expressed+on+granulocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and expression analysis of a novel G-protein-coupled receptor selectively expressed on granulocytes</span></div><div class="casAuthors">Yousefi, Shida; Cooper, Paul R.; Potter, Suzanne L.; Mueck, Beatrice; Jarai, Gabor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1045-1052</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">The migration of neutrophils into sites of acute and chronic inflammation is mediated by chemokines.  The authors used degenerate-primer reverse transcriptase-polymerase chain reaction (RT-PCR) to analyze chemokine receptor expression in neutrophils and identify novel receptors.  RNA was isolated from human peripheral blood neutrophils and from neutrophils that had been stimulated for 5 h with granulocyte-macrophage colony-stimulating factor or by coculturing with primary human bronchial epithelial cells.  Amplification products were cloned, and clone redundancy was detd.  Seven known G-protein-coupled receptors were identified among 38 clones-CCR1, CCR4, CXCR1, CXCR2, CXCR4, HM63, and FPR1-as well as a novel gene, EX33.  The full-length EX33 clone was obtained, and an in silico approach was used to identify the putative murine homolog.  The EX33 gene encodes a 396-amino-acid protein with limited sequence identity to known receptors.  Expression studies of several known chemokine receptors and EX33 revealed that resting neutrophils expressed higher levels of CXCRs and EX33 compared with activated neutrophils.  Northern blot expts. revealed that EX33 is expressed mainly in bone marrow, lung, and peripheral blood leukocytes.  Using RT-PCR anal., the authors showed more abundant expression of EX33 in neutrophils and eosinophils, in comparison with that in T- or B-lymphocytes, indicating cell-specific expression among leukocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAMqeieaewbrVg90H21EOLACvtfcHk0ljrGwZTqbDhUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkslWrurs%253D&md5=03d1eb21f619cdd4790ace3c2b571d23</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYousefi%26aufirst%3DS.%26aulast%3DCooper%26aufirst%3DP.%2BR.%26aulast%3DPotter%26aufirst%3DS.%2BL.%26aulast%3DMueck%26aufirst%3DB.%26aulast%3DJarai%26aufirst%3DG.%26atitle%3DCloning%2520and%2520expression%2520analysis%2520of%2520a%2520novel%2520G%2520protein-coupled%2520receptor%2520selectively%2520expressed%2520on%2520granulocytes%26jtitle%3DJ.%2520Leukoc.%2520Biol.%26date%3D2001%26volume%3D69%26spage%3D1045%26epage%3D1052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Venkataraman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, F.</span></span> <span> </span><span class="NLM_article-title">The Gprotein-coupled receptor, GPR84 regulates IL-4 production by T lymphocytes in response to CD3 crosslinking</span>. <i>Immunol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.imlet.2005.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.imlet.2005.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=15993493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKjs77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2005&pages=144-153&author=C.+Venkataramanauthor=F.+Kuo&title=The+Gprotein-coupled+receptor%2C+GPR84+regulates+IL-4+production+by+T+lymphocytes+in+response+to+CD3+crosslinking&doi=10.1016%2Fj.imlet.2005.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The G-protein coupled receptor, GPR84 regulates IL-4 production by T lymphocytes in response to CD3 crosslinking</span></div><div class="casAuthors">Venkataraman, Chandrasekar; Kuo, Frederick</div><div class="citationInfo"><span class="NLM_cas:title">Immunology Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">144-153</span>CODEN:
                <span class="NLM_cas:coden">IMLED6</span>;
        ISSN:<span class="NLM_cas:issn">0165-2478</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The orphan G-protein coupled receptor, GPR84 is highly expressed in the bone marrow, and in splenic T cells and B cells.  In this study, GPR84-deficient mice were generated to understand the biol. function of this orphan receptor.  The proliferation of T and B cells in response to various mitogens was normal in GPR84-deficient mice.  Interestingly, primary stimulation of T cells with anti-CD3 resulted in increased IL-4 but not IL-2 or IFN-γ prodn. in GPR84-/- mice compared to wild-type mice.  Augmented IL-4 prodn. in GPR84-deficient T cells was not related to increased frequency of IL-4-secreting cells in response to anti-CD3 stimulation.  In fact, stimulation with anti-CD3 and anti-CD28 resulted in increased levels of IL-4 but not IFN-γ steady-state mRNA in GPR84-/- T cells.  In addn., Th2 effector cells generated in vitro from GPR84-/- mice produced higher levels of IL-4, IL-5 and IL-13 compared to wild-type mice.  However, there was no detectable difference in the extent of IL-4 and IL-5 prodn. between the two groups of mice in response to antigen stimulation of spleen cells, isolated from mice previously immunized with OVA in alum.  These studies reveal a novel role for GPR84 in regulating early IL-4 gene expression in activated T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoxPqqzrLXAbVg90H21EOLACvtfcHk0ljrGwZTqbDhUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKjs77L&md5=abef6f196008a5055ab99e9213c0d55d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.imlet.2005.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.imlet.2005.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DVenkataraman%26aufirst%3DC.%26aulast%3DKuo%26aufirst%3DF.%26atitle%3DThe%2520Gprotein-coupled%2520receptor%252C%2520GPR84%2520regulates%2520IL-4%2520production%2520by%2520T%2520lymphocytes%2520in%2520response%2520to%2520CD3%2520crosslinking%26jtitle%3DImmunol.%2520Lett.%26date%3D2005%26volume%3D101%26spage%3D144%26epage%3D153%26doi%3D10.1016%2Fj.imlet.2005.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonavicius, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, L.</span></span> <span> </span><span class="NLM_article-title">Medium chain fatty acids as ligands for orphan G protein-coupled receptor GPR84</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">34457</span>– <span class="NLM_lpage">34464</span>, <span class="refDoi"> DOI: 10.1074/jbc.M608019200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1074%2Fjbc.M608019200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=16966319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFKgur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=34457-34464&author=J.+Wangauthor=X.+Wuauthor=N.+Simonaviciusauthor=H.+Tianauthor=L.+Ling&title=Medium+chain+fatty+acids+as+ligands+for+orphan+G+protein-coupled+receptor+GPR84&doi=10.1074%2Fjbc.M608019200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Medium-chain Fatty Acids as Ligands for Orphan G Protein-coupled Receptor GPR84</span></div><div class="casAuthors">Wang, Jinghong; Wu, Xiaosu; Simonavicius, Nicole; Tian, Hui; Ling, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">34457-34464</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Free fatty acids (FFAs) play important physiol. roles in many tissues as an energy source and as signaling mols. in various cellular processes.  Elevated levels of circulating FFAs are assocd. with obesity, dyslipidemia, and diabetes.  Here we show that GPR84, a previously orphan G protein-coupled receptor, functions as a receptor for medium-chain FFAs with carbon chain lengths of 9-14.  Medium-chain FFAs elicit calcium mobilization, inhibit 3',5'-cAMP prodn., and stimulate [35S]guanosine 5'-O-(3-thiotriphosphate) binding in a GPR84-dependent manner.  The activation of GPR84 by medium-chain FFAs couples primarily to a pertussis toxin-sensitive Gi/o pathway.  In addn., we show that GPR84 is selectively expressed in leukocytes and markedly induced in monocytes/macrophages upon activation by lipopolysaccharide.  Furthermore, we demonstrate that medium-chain FFAs amplify lipopolysaccharide-stimulated prodn. of the proinflammatory cytokine interleukin-12 p40 through GPR84.  Our results indicate a role for GPR84 in directly linking fatty acid metab. to immunol. regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-BrABfXO2grVg90H21EOLACvtfcHk0ljCoEctfdRNYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFKgur%252FI&md5=2bbe13a202723f62e59eab0b08a25878</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M608019200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M608019200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSimonavicius%26aufirst%3DN.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DLing%26aufirst%3DL.%26atitle%3DMedium%2520chain%2520fatty%2520acids%2520as%2520ligands%2520for%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR84%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D34457%26epage%3D34464%26doi%3D10.1074%2Fjbc.M608019200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouchard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagé, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bédard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallières, L.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor 84, a microglia-associated protein expressed in neuroinflammatory conditions</span>. <i>Glia</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">790</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1002/glia.20506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fglia.20506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=17390309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BD2s3jtVemsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2007&pages=790-800&author=C.+Bouchardauthor=J.+Pag%C3%A9author=A.+B%C3%A9dardauthor=P.+Tremblayauthor=L.+Valli%C3%A8res&title=G+protein-coupled+receptor+84%2C+a+microglia-associated+protein+expressed+in+neuroinflammatory+conditions&doi=10.1002%2Fglia.20506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor 84, a microglia-associated protein expressed in neuroinflammatory conditions</span></div><div class="casAuthors">Bouchard Caroline; Page Julie; Bedard Andreanne; Tremblay Pierrot; Vallieres Luc</div><div class="citationInfo"><span class="NLM_cas:title">Glia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">790-800</span>
        ISSN:<span class="NLM_cas:issn">0894-1491</span>.
    </div><div class="casAbstract">G protein-coupled receptor 84 (GPR84) is a recently discovered member of the seven transmembrane receptor superfamily whose function and regulation are unknown.  Here, we report that in mice suffering from endotoxemia, microglia express GPR84 in a strong and sustained manner.  This property is shared by subpopulations of peripheral macrophages and, to a much lesser extent, monocytes.  The induction of GPR84 expression by endotoxin is mediated, at least in part, by proinflammatory cytokines, notably tumor necrosis factor (TNF) and interleukin-1 (IL-1), because mice lacking either one or both of these molecules have fewer GPR84-expressing cells in their cerebral cortex than wild-type mice during the early phase of endotoxemia.  Moreover, when injected intracerebrally or added to microglial cultures, recombinant TNF stimulates GPR84 expression through a dexamethasone-insensitive mechanism.  Finally, we show that microglia produce GPR84 not only during endotoxemia, but also during experimental autoimmune encephalomyelitis (EAE), a model of multiple sclerosis.  In conclusion, this study reports the identification of a new sensitive marker of microglial activation, which may play an important regulatory role in neuroimmunological processes, acting downstream to the effects of proinflammatory mediators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR5WBo60SwdsPEeosoOnpOLfW6udTcc2eaOndR94UooYrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3jtVemsw%253D%253D&md5=5bc25664243312ac66f94244ab598c9d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fglia.20506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fglia.20506%26sid%3Dliteratum%253Aachs%26aulast%3DBouchard%26aufirst%3DC.%26aulast%3DPag%25C3%25A9%26aufirst%3DJ.%26aulast%3DB%25C3%25A9dard%26aufirst%3DA.%26aulast%3DTremblay%26aufirst%3DP.%26aulast%3DValli%25C3%25A8res%26aufirst%3DL.%26atitle%3DG%2520protein-coupled%2520receptor%252084%252C%2520a%2520microglia-associated%2520protein%2520expressed%2520in%2520neuroinflammatory%2520conditions%26jtitle%3DGlia%26date%3D2007%26volume%3D55%26spage%3D790%26epage%3D800%26doi%3D10.1002%2Fglia.20506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Recio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purvis, G. S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iveson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeboudj, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Callaghan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griesbach, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dib, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greaves, D. R.</span></span> <span> </span><span class="NLM_article-title">Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1419</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.01419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.3389%2Ffimmu.2018.01419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29973940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1yqtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1419&author=C.+Recioauthor=D.+Lucyauthor=G.+S.+D.+Purvisauthor=P.+Ivesonauthor=L.+Zeboudjauthor=A.+J.+Iqbalauthor=D.+Linauthor=C.+O%E2%80%99Callaghanauthor=L.+Davisonauthor=E.+Griesbachauthor=A.+J.+Russellauthor=G.+M.+Wynneauthor=L.+Dibauthor=C.+Monacoauthor=D.+R.+Greaves&title=Activation+of+the+immune-metabolic+receptor+GPR84+enhances+inflammation+and+phagocytosis+in+macrophages&doi=10.3389%2Ffimmu.2018.01419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages</span></div><div class="casAuthors">Recio, Carlota; Lucy, Daniel; Purvis, Gareth S. D.; Iveson, Poppy; Zeboudj, Lynda; Iqbal, Asif J.; Lin, Daniel; O'Callaghan, Chris; Davison, Lucy; Griesbach, Esther; Russell, Angela J.; Wynne, Graham M.; Dib, Lea; Monaco, Claudia; Greaves, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1419/1-1419/17</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">GPR84 is a member of the metabolic G protein-coupled receptor family, and its expres- sion has been described predominantly in immune cells.  GPR84 activation is involved in the inflammatory response, but the mechanisms by which it modulates inflammation have been incompletely described.  In this study, we investigated GPR84 expression, activation, and function in macrophages to establish the role of the receptor during the inflammatory response.  We obsd. that GPR84 expression in murine tissues is increased by endotoxemia, hyperglycemia, and hypercholesterolemia.  Ex vivo studies revealed that GPR84 mRNA expression is increased by LPS and other pro-inflammatory mols. in different murine and human macrophage populations.  Likewise, high glu- cose concns. and the presence of oxidized LDL increased GPR84 expression in macrophages.  Activation of the GPR84 receptor with a selective agonist, 6-(octylamino) pyrimidine-2,4(1H,3H)-dione (6-n-octylaminouracil, 6-OAU), enhanced the expression of phosphorylated Akt, p-ERK, and p65 nuclear translocation under inflammatory con- ditions and elevated the expression levels of the inflammatory mediators TNFα, IL-6, IL-12B, CCL2, CCL5, and CXCL1.  In addn., GPR84 activation triggered increased bacterial adhesion and phagocytosis in macrophages.  The enhanced inflammatory response mediated by 6-OAU was not obsd. in GPR84/- cells nor in macrophages treated with a selective GPR84 antagonist.  Collectively, our results reveal that GPR84 functions as an enhancer of inflammatory signaling in macrophages once inflammation is established.  Therefore, mols. that antagonize the GPR84 receptor may be potential therapeutic tools in inflammatory and metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXDBYilKMKTrVg90H21EOLACvtfcHk0ljCoEctfdRNYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1yqtr%252FF&md5=0e28917468e38dd31ad0b858b3780f6a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.01419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.01419%26sid%3Dliteratum%253Aachs%26aulast%3DRecio%26aufirst%3DC.%26aulast%3DLucy%26aufirst%3DD.%26aulast%3DPurvis%26aufirst%3DG.%2BS.%2BD.%26aulast%3DIveson%26aufirst%3DP.%26aulast%3DZeboudj%26aufirst%3DL.%26aulast%3DIqbal%26aufirst%3DA.%2BJ.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Callaghan%26aufirst%3DC.%26aulast%3DDavison%26aufirst%3DL.%26aulast%3DGriesbach%26aufirst%3DE.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DWynne%26aufirst%3DG.%2BM.%26aulast%3DDib%26aufirst%3DL.%26aulast%3DMonaco%26aufirst%3DC.%26aulast%3DGreaves%26aufirst%3DD.%2BR.%26atitle%3DActivation%2520of%2520the%2520immune-metabolic%2520receptor%2520GPR84%2520enhances%2520inflammation%2520and%2520phagocytosis%2520in%2520macrophages%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D1419%26doi%3D10.3389%2Ffimmu.2018.01419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaishi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onozawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iino, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, T.</span></span> <span> </span><span class="NLM_article-title">Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">10684</span>– <span class="NLM_lpage">10691</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.420042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1074%2Fjbc.M112.420042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23449982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvFSisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=10684-10691&author=M.+Suzukiauthor=S.+Takaishiauthor=M.+Nagasakiauthor=Y.+Onozawaauthor=I.+Iinoauthor=H.+Maedaauthor=T.+Komaiauthor=T.+Oda&title=Medium-chain+fatty+acid-sensing+receptor%2C+GPR84%2C+is+a+proinflammatory+receptor&doi=10.1074%2Fjbc.M112.420042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Medium-chain Fatty Acid-sensing Receptor, GPR84, Is a Proinflammatory Receptor</span></div><div class="casAuthors">Suzuki, Masakatsu; Takaishi, Sachiko; Nagasaki, Miyuki; Onozawa, Yoshiko; Iino, Ikue; Maeda, Hiroaki; Komai, Tomoaki; Oda, Tomiichiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">10684-10691</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">G protein-coupled receptor 84 (GPR84) is a putative receptor for medium-chain fatty acids (MCFAs), whose pathophysiol. roles have not yet been clarified.  Here, the authors show that GPR84 was activated by MCFAs with the hydroxyl group at the 2- or 3-position more effectively than nonhydroxylated MCFAs.  The authors also identified a surrogate agonist, 6-n-octylaminouracil (6-OAU), for GPR84.  These potential ligands and the surrogate agonist, 6-OAU, stimulated [35S]GTP binding and accumulated phosphoinositides in a GPR84-dependent manner.  The surrogate agonist, 6-OAU, internalized GPR84-EGFP from the cell surface.  Both the potential ligands and 6-OAU elicited chemotaxis of human polymorphonuclear leukocytes (PMNs) and macrophages and amplified LPS-stimulated prodn. of the proinflammatory cytokine IL-8 from PMNs and TNFα from macrophages.  Furthermore, the i.v. injection of 6-OAU raised the blood CXCL1 level in rats, and the inoculation of 6-OAU into the rat air pouch accumulated PMNs and macrophages in the site.  The results indicate a proinflammatory role of GPR84, suggesting that the receptor may be a novel target to treat chronic low grade inflammation assocd.-disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTeAePnQZYFrVg90H21EOLACvtfcHk0liSn0314YYoTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvFSisr8%253D&md5=63a395e01eb2083bfb7d4a1195c88b1a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.420042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.420042%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DTakaishi%26aufirst%3DS.%26aulast%3DNagasaki%26aufirst%3DM.%26aulast%3DOnozawa%26aufirst%3DY.%26aulast%3DIino%26aufirst%3DI.%26aulast%3DMaeda%26aufirst%3DH.%26aulast%3DKomai%26aufirst%3DT.%26aulast%3DOda%26aufirst%3DT.%26atitle%3DMedium-chain%2520fatty%2520acid-sensing%2520receptor%252C%2520GPR84%252C%2520is%2520a%2520proinflammatory%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D10684%26epage%3D10691%26doi%3D10.1074%2Fjbc.M112.420042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatlin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semple, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unett, D. J.</span></span> <span> </span><span class="NLM_article-title">Embelin and its derivatives unravel the signaling, proinflammatory and antiatherogenic properties of GPR84 receptor</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">198</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.phrs.2018.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29471103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Wmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2018&pages=185-198&author=I.+Gaidarovauthor=T.+Anthonyauthor=J.+Gatlinauthor=X.+Chenauthor=D.+Millsauthor=M.+Solomonauthor=S.+Hanauthor=G.+Sempleauthor=D.+J.+Unett&title=Embelin+and+its+derivatives+unravel+the+signaling%2C+proinflammatory+and+antiatherogenic+properties+of+GPR84+receptor&doi=10.1016%2Fj.phrs.2018.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Embelin and its derivatives unravel the signaling, proinflammatory and antiatherogenic properties of GPR84 receptor</span></div><div class="casAuthors">Gaidarov, Ibragim; Anthony, Todd; Gatlin, Joel; Chen, Xiaohua; Mills, David; Solomon, Michelle; Han, Sangdon; Semple, Graeme; Unett, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-198</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">GPR84 is an orphan G-protein coupled receptor, expressed on monocytes, macrophages and neutrophils and is significantly upregulated by inflammatory stimuli.  The physiol. role of GPR84 remains largely unknown.  Medium chain fatty acids (MCFA) activate the receptor and have been proposed to be its endogenous ligands, although the high concns. of MCFAs required for receptor activation generally exceed normal physiol. levels.  We identified the natural product embelin as a highly potent and selective surrogate GPR84 agonist (originally disclosed in patent application WO2007027661A2, 2007) and synthesized close structural analogs with widely varying receptor activities.  These tools were used to perform a comprehensive study of GPR84 signaling and function in recombinant cells and in primary human macrophages and neutrophils.  Activation of recombinant GPR84 by embelin in HEK293 cells results in Gi/o as well as G12/13-Rho signaling.  In human macrophages, GPR84 initiates PTX sensitive Erk1/2 and Akt phosphorylation, PI-3 kinase activation, calcium flux, and release of prostaglandin E2.  In addn., GPR84 signaling in macrophages elicits Gi Gβγ-mediated augmentation of intracellular cAMP, rather than the decrease expected from Giα engagement.  GPR84 activation drives human neutrophil chemotaxis and primes them for amplification of oxidative burst induced by FMLP and C5A.  Loss of GPR84 is assocd. with attenuated LPS-induced release of proinflammatory mediators IL-6, KC-GROα, VEGF, MIP-2 and NGAL from peritoneal exudates.  While initiating numerous proinflammatory activities in macrophages and neutrophils, GPR84 also possesses GPR109A-like antiatherosclerotic properties in macrophages.  Macrophage receptor activation leads to upregulation of cholesterol transporters ABCA1 and ABCG1 and stimulates reverse cholesterol transport.  These data suggest that GPR84 may be a target of therapeutic value and that distinct modes of receptor modulation (inhibition vs. stimulation) may be required for inflammatory and atherosclerotic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAw98Yo2b4GrVg90H21EOLACvtfcHk0liSn0314YYoTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Wmsbo%253D&md5=f4c0ba851ede2aa9739af183423ff9f1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DGaidarov%26aufirst%3DI.%26aulast%3DAnthony%26aufirst%3DT.%26aulast%3DGatlin%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMills%26aufirst%3DD.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DSemple%26aufirst%3DG.%26aulast%3DUnett%26aufirst%3DD.%2BJ.%26atitle%3DEmbelin%2520and%2520its%2520derivatives%2520unravel%2520the%2520signaling%252C%2520proinflammatory%2520and%2520antiatherogenic%2520properties%2520of%2520GPR84%2520receptor%26jtitle%3DPharmacol.%2520Res.%26date%3D2018%26volume%3D131%26spage%3D185%26epage%3D198%26doi%3D10.1016%2Fj.phrs.2018.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundqvist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christenson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdfeldt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mårtensson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Björkman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dieckmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsman, H.</span></span> <span> </span><span class="NLM_article-title">Similarities and differences between the responses induced in human phagocytes through activation of the medium chain fatty acid receptor GPR84 and the short chain fatty acid receptor FFA2R</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1865</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">708</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2018.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bbamcr.2018.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29477577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsFaju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1865&publication_year=2018&pages=695-708&author=M.+Sundqvistauthor=K.+Christensonauthor=A.+Holdfeldtauthor=M.+Gablauthor=J.+M%C3%A5rtenssonauthor=L.+Bj%C3%B6rkmanauthor=R.+Dieckmannauthor=C.+Dahlgrenauthor=H.+Forsman&title=Similarities+and+differences+between+the+responses+induced+in+human+phagocytes+through+activation+of+the+medium+chain+fatty+acid+receptor+GPR84+and+the+short+chain+fatty+acid+receptor+FFA2R&doi=10.1016%2Fj.bbamcr.2018.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Similarities and differences between the responses induced in human phagocytes through activation of the medium chain fatty acid receptor GPR84 and the short chain fatty acid receptor FFA2R</span></div><div class="casAuthors">Sundqvist, Martina; Christenson, Karin; Holdfeldt, Andre; Gabl, Michael; Maartensson, Jonas; Bjoerkman, Lena; Dieckmann, Regis; Dahlgren, Claes; Forsman, Huamei</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1865</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">695-708</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">GPR84 is a recently de-orphanized member of the G-protein coupled receptor (GPCR) family recognizing medium chain fatty acids, and has been suggested to play important roles in inflammation.  Due to the lack of potent and selective GPR84 ligands, the basic knowledge related to GPR84 functions is very limited.  In this study, we have characterized the GPR84 activation profile and regulation mechanism in human phagocytes, using two recently developed small mols. that specifically target GPR84 agonistically (ZQ16) and antagonistically (GLPG1205), resp.  Compared to our earlier characterization of the short chain fatty acid receptor FFA2R which is functionally expressed in neutrophils but not in monocytes, GPR84 is expressed in both cell types and in monocyte-derived macrophages.  In neutrophils, the GPR84 agonist had an activation profile very similar to that of FFA2R.  The GPR84-mediated superoxide release was low in naive cells, but the response could be significantly primed by TNFα and by the actin cytoskeleton disrupting agent Latrunculin A.  Similar to that of FFA2R, a desensitization mechanism bypassing the actin cytoskeleton was utilized by GPR84.  All ZQ16-mediated cellular responses were sensitive to GLPG1205, confirming the GPR84-dependency.  Finally, our data of in vivo transmigrated tissue neutrophils indicate that both GPR84 and FFA2R are involved in neutrophil recruitment processes in vivo.  In summary, we show functional similarities but also some important differences between GPR84 and FFA2R in human phagocytes, thus providing some mechanistic insights into GPR84 regulation in blood neutrophils and cells recruited to an aseptic inflammatory site in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxFh26S2UftbVg90H21EOLACvtfcHk0liSn0314YYoTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsFaju7w%253D&md5=5f80256de9cc62715bf953312937170e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2018.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2018.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DSundqvist%26aufirst%3DM.%26aulast%3DChristenson%26aufirst%3DK.%26aulast%3DHoldfeldt%26aufirst%3DA.%26aulast%3DGabl%26aufirst%3DM.%26aulast%3DM%25C3%25A5rtensson%26aufirst%3DJ.%26aulast%3DBj%25C3%25B6rkman%26aufirst%3DL.%26aulast%3DDieckmann%26aufirst%3DR.%26aulast%3DDahlgren%26aufirst%3DC.%26aulast%3DForsman%26aufirst%3DH.%26atitle%3DSimilarities%2520and%2520differences%2520between%2520the%2520responses%2520induced%2520in%2520human%2520phagocytes%2520through%2520activation%2520of%2520the%2520medium%2520chain%2520fatty%2520acid%2520receptor%2520GPR84%2520and%2520the%2520short%2520chain%2520fatty%2520acid%2520receptor%2520FFA2R%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2018%26volume%3D1865%26spage%3D695%26epage%3D708%26doi%3D10.1016%2Fj.bbamcr.2018.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhuber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassem, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khailah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldeen, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khairy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khayyal, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shcherbakova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Efferth, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich-Merzenich, G.</span></span> <span> </span><span class="NLM_article-title">GPR84 and TREM-1 signaling contribute to the pathogenesis of reflux esophagitis</span>. <i>Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1011</span>– <span class="NLM_lpage">1024</span>, <span class="refDoi"> DOI: 10.2119/molmed.2015.00098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.2119%2Fmolmed.2015.00098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKlsLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1011-1024&author=H.+Abdel-Azizauthor=M.+Schneiderauthor=W.+Neuhuberauthor=A.+M.+Kassemauthor=S.+Khailahauthor=J.+Mullerauthor=H.+G.+Eldeenauthor=A.+Khairyauthor=M.+T.+Khayyalauthor=A.+Shcherbakovaauthor=T.+Efferthauthor=G.+Ulrich-Merzenich&title=GPR84+and+TREM-1+signaling+contribute+to+the+pathogenesis+of+reflux+esophagitis&doi=10.2119%2Fmolmed.2015.00098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">GPR84 and TREM-1 signaling contribute to the pathogenesis of reflux esophagitis</span></div><div class="casAuthors">Abdel-Aziz, Heba; Schneider, Mathias; Neuhuber, Winfried; Kassem, Abdel Meguid; Khailah, Saleem; Mueller, Juergen; Eldeen, Hadeel Gamal; Khairy, Ahmed; Khayyal, Mohamed T.; Shcherbakova, Anastasiia; Efferth, Thomas; Ulrich-Merzenich, Gudrun</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine (Manhasset, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1011-1024</span>CODEN:
                <span class="NLM_cas:coden">MOMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1528-3658</span>.
    
            (<span class="NLM_cas:orgname">Feinstein Institute for Medical Research</span>)
        </div><div class="casAbstract">Gastro-esophageal reflux disease (GERD) is one of the most common disorders in gastroenterol.  Patients present with or without increased acid exposure indicating a nonuniform etiol.  Thus, the common treatment with proton pump inhibitors (PPIs) fails to control symptoms in up to 40% of patients.  To further elucidate the pathophysiol. of the condition and explore new treatment targets, transcriptomics, proteomics and histol. methods were applied to a surgically induced subchronic reflux esophagitis model in Wistar rats after treatment with either omeprazole (PPI) or STW5, a herbal prepn. shown to ameliorate esophagitis without affecting refluxate pH.  The normal human esophageal squamous cell line HET-1A and human endoscopic biopsies were used to confirm our findings to the G-protein-coupled receptor (GPR) 84 in human tissue.  Both treatments reduced reflux-induced macroscopic and microscopic lesions of the esophagi as well as known proinflammatory cytokines.  Proteomic and transcriptomic analyses identified CINC1-3, MIP-1/3α, MIG, RANTES and interleukin (IL)-1β as prominent mediators in GERD.  Most regulated cyto-/chemokines are linked to the TREM-1 signaling pathway.  The fatty acid receptor GPR84 was upregulated in esophagitis but significantly decreased in treated groups, a finding supported by Western blot and immunohistochem. in both rat tissue and HET-1A cells.  GPR84 was also found to be significantly upregulated in patients with grade B reflux esophagitis.  The expression of GPR84 in esophageal tissue and its potential involvement in GERD are reported for the first time.  IL-8 (CINC1-3) and the TREM-1 signaling pathway are proposed, besides GPR84, to play an important role in the pathogenesis of GERD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGv2akUPMqbbVg90H21EOLACvtfcHk0lgtlIqbk6cvPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKlsLbI&md5=d626c0592ea9ac67a3700ec6b2138383</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2119%2Fmolmed.2015.00098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2119%252Fmolmed.2015.00098%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Aziz%26aufirst%3DH.%26aulast%3DSchneider%26aufirst%3DM.%26aulast%3DNeuhuber%26aufirst%3DW.%26aulast%3DKassem%26aufirst%3DA.%2BM.%26aulast%3DKhailah%26aufirst%3DS.%26aulast%3DMuller%26aufirst%3DJ.%26aulast%3DEldeen%26aufirst%3DH.%2BG.%26aulast%3DKhairy%26aufirst%3DA.%26aulast%3DKhayyal%26aufirst%3DM.%2BT.%26aulast%3DShcherbakova%26aufirst%3DA.%26aulast%3DEfferth%26aufirst%3DT.%26aulast%3DUlrich-Merzenich%26aufirst%3DG.%26atitle%3DGPR84%2520and%2520TREM-1%2520signaling%2520contribute%2520to%2520the%2520pathogenesis%2520of%2520reflux%2520esophagitis%26jtitle%3DMol.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D1011%26epage%3D1024%26doi%3D10.2119%2Fmolmed.2015.00098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arijs, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanque, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laukens, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nys, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccotti, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merciris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mate, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parent, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vriendt, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labeguere, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutgeerts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandeghinste, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeire, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brys, R.</span></span> <span> </span><span class="NLM_article-title">GPR84 inhibition as a novel therapeutic approach in IBD: mechanistic and translational studies</span>. <i>J. Crohns Colitis</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">S92</span>– <span class="NLM_lpage">S93</span>, <span class="refDoi"> DOI: 10.1093/ecco-jcc/jju027.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1093%2Fecco-jcc%2Fjju027.149" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=S92-S93&author=S.+Dupontauthor=I.+Arijsauthor=R.+Blanqueauthor=D.+Laukensauthor=K.+Nysauthor=M.+C.+Ceccottiauthor=D.+Mercirisauthor=S.+De+Vosauthor=O.+Mateauthor=I.+Parentauthor=V.+De+Vriendtauthor=F.+Labeguereauthor=R.+Galienauthor=M.+Devosauthor=P.+Rutgeertsauthor=N.+Vandeghinsteauthor=S.+Vermeireauthor=R.+Brys&title=GPR84+inhibition+as+a+novel+therapeutic+approach+in+IBD%3A+mechanistic+and+translational+studies&doi=10.1093%2Fecco-jcc%2Fjju027.149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fecco-jcc%2Fjju027.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fecco-jcc%252Fjju027.149%26sid%3Dliteratum%253Aachs%26aulast%3DDupont%26aufirst%3DS.%26aulast%3DArijs%26aufirst%3DI.%26aulast%3DBlanque%26aufirst%3DR.%26aulast%3DLaukens%26aufirst%3DD.%26aulast%3DNys%26aufirst%3DK.%26aulast%3DCeccotti%26aufirst%3DM.%2BC.%26aulast%3DMerciris%26aufirst%3DD.%26aulast%3DDe%2BVos%26aufirst%3DS.%26aulast%3DMate%26aufirst%3DO.%26aulast%3DParent%26aufirst%3DI.%26aulast%3DDe%2BVriendt%26aufirst%3DV.%26aulast%3DLabeguere%26aufirst%3DF.%26aulast%3DGalien%26aufirst%3DR.%26aulast%3DDevos%26aufirst%3DM.%26aulast%3DRutgeerts%26aufirst%3DP.%26aulast%3DVandeghinste%26aufirst%3DN.%26aulast%3DVermeire%26aufirst%3DS.%26aulast%3DBrys%26aufirst%3DR.%26atitle%3DGPR84%2520inhibition%2520as%2520a%2520novel%2520therapeutic%2520approach%2520in%2520IBD%253A%2520mechanistic%2520and%2520translational%2520studies%26jtitle%3DJ.%2520Crohns%2520Colitis%26date%3D2015%26volume%3D9%26spage%3DS92%26epage%3DS93%26doi%3D10.1093%2Fecco-jcc%2Fjju027.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicol, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Russa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Didangelos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malcangio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, S. B.</span></span> <span> </span><span class="NLM_article-title">The role of Gprotein-receptor 84 in experimental neuropathic pain</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">8959</span>– <span class="NLM_lpage">8969</span>, <span class="refDoi"> DOI: 10.1523/JNEUROSCI.3558-14.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1523%2FJNEUROSCI.3558-14.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=26063927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOqsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=8959-8969&author=L.+S.+Nicolauthor=J.+M.+Dawesauthor=F.+La+Russaauthor=A.+Didangelosauthor=A.+K.+Clarkauthor=C.+Gentryauthor=J.+Gristauthor=J.+B.+Daviesauthor=M.+Malcangioauthor=S.+B.+McMahon&title=The+role+of+Gprotein-receptor+84+in+experimental+neuropathic+pain&doi=10.1523%2FJNEUROSCI.3558-14.2015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The role of G-protein receptor 84 in experimental neuropathic pain</span></div><div class="casAuthors">Nicol, Louise S. C.; Dawes, John M.; La Russa, Federica; Didangelos, Athanasios; Clark, Anna K.; Gentry, Clive; Grist, John; Davies, John B.; Malcangio, Marzia; McMahon, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8959-8969</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">G-protein receptor 84 (GPR84) is an orphan receptor that is induced markedly in monocytes/macrophages and microglia during inflammation, but its pathophysiol. function is unknown.  Here, we investigate the role of GPR84 in a murine model of traumatic nerve injury.  Naive GPR84 knock-out (KO) mice exhibited normal behavioral responses to acute noxious stimuli, but subsequent to partial sciatic nerve ligation (PNL), KOs did not develop mech. or thermal hypersensitivity, in contrast to wild-type (WT) littermates.  Nerve injury increased ionized calcium binding adapter mol. 1 (Iba1) and phosphorylated p38 MAPK immunoreactivity in the dorsal horn and Iba1 and cluster of differentiation 45 expression in the sciatic nerve, with no difference between genotypes.  PCR array anal. revealed that Gpr84 expression was upregulated in the spinal cord and sciatic nerve of WT mice.  In addn., the expression of arginase-1, a marker for anti-inflammatory macrophages, was upregulated in KO sciatic nerve.  Based on this evidence, we investigated whether peripheral macrophages behave differently in the absence of GPR84.  We found that lipopolysaccharide-stimulated KO macrophages exhibited attenuated expression of several proinflammatory mediators, including IL-1β, IL-6, and TNF-α.  Forskolin-stimulated KO macrophages also showed greater cAMP induction, a second messenger assocd. with immunosuppression.  In summary, our results demonstrate that GPR84 is a proinflammatory receptor that contributes to nociceptive signaling via the modulation of macrophages, whereas in its absence the response of these cells to an inflammatory insult is impaired.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEz_UEJb4WPbVg90H21EOLACvtfcHk0lgtlIqbk6cvPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOqsLrJ&md5=4d7a16fa2471aaaa6ed9395ad1215544</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.3558-14.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.3558-14.2015%26sid%3Dliteratum%253Aachs%26aulast%3DNicol%26aufirst%3DL.%2BS.%26aulast%3DDawes%26aufirst%3DJ.%2BM.%26aulast%3DLa%2BRussa%26aufirst%3DF.%26aulast%3DDidangelos%26aufirst%3DA.%26aulast%3DClark%26aufirst%3DA.%2BK.%26aulast%3DGentry%26aufirst%3DC.%26aulast%3DGrist%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DJ.%2BB.%26aulast%3DMalcangio%26aufirst%3DM.%26aulast%3DMcMahon%26aufirst%3DS.%2BB.%26atitle%3DThe%2520role%2520of%2520Gprotein-receptor%252084%2520in%2520experimental%2520neuropathic%2520pain%26jtitle%3DJ.%2520Neurosci.%26date%3D2015%26volume%3D35%26spage%3D8959%26epage%3D8969%26doi%3D10.1523%2FJNEUROSCI.3558-14.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Audoy-Remus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecours, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivest, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallieres, L.</span></span> <span> </span><span class="NLM_article-title">GPR84 deficiency reduces microgliosis, but accelerates dendritic degeneration and cognitive decline in a mouse model of Alzheimer’s disease</span>. <i>Brain, Behav., Immun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1016/j.bbi.2015.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bbi.2015.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25637481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitFCntbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2015&pages=112-120&author=J.+Audoy-Remusauthor=L.+Bozoyanauthor=A.+Dumasauthor=M.+Filaliauthor=C.+Lecoursauthor=S.+Lacroixauthor=S.+Rivestauthor=M.-E.+Tremblayauthor=L.+Vallieres&title=GPR84+deficiency+reduces+microgliosis%2C+but+accelerates+dendritic+degeneration+and+cognitive+decline+in+a+mouse+model+of+Alzheimer%E2%80%99s+disease&doi=10.1016%2Fj.bbi.2015.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">GPR84 deficiency reduces microgliosis, but accelerates dendritic degeneration and cognitive decline in a mouse model of Alzheimer's disease</span></div><div class="casAuthors">Audoy-Remus, Julie; Bozoyan, Lusine; Dumas, Aline; Filali, Mohammed; Lecours, Cynthia; Lacroix, Steve; Rivest, Serge; Tremblay, Marie-Eve; Vallieres, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Brain, Behavior, and Immunity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112-120</span>CODEN:
                <span class="NLM_cas:coden">BBIMEW</span>;
        ISSN:<span class="NLM_cas:issn">0889-1591</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Microglia surrounds the amyloid plaques that form in the brains of patients with Alzheimer's disease (AD), but their role is controversial.  Under inflammatory conditions, these cells can express GPR84, an orphan receptor whose pathophysiol. role is unknown.  Here, we report that GPR84 is upregulated in microglia of APP/PS1 transgenic mice, a model of AD.  Without GPR84, these mice display both accelerated cognitive decline and a reduced no. of microglia, esp. in areas surrounding plaques.  The lack of GPR84 affects neither plaque formation nor hippocampal neurogenesis, but promotes dendritic degeneration.  Furthermore, GPR84 does not influence the clin. progression of other diseases in which its expression has been reported, i.e., exptl. autoimmune encephalomyelitis (EAE) and endotoxic shock.  We conclude that GPR84 plays a beneficial role in amyloid pathol. by acting as a sensor for a yet unknown ligand that promotes microglia recruitment, a response affecting dendritic degeneration and required to prevent further cognitive decline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWBqsDH3-KvrVg90H21EOLACvtfcHk0liJbXi8eVWy-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitFCntbo%253D&md5=4b643287374786cdde611ffbb3002101</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbi.2015.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbi.2015.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DAudoy-Remus%26aufirst%3DJ.%26aulast%3DBozoyan%26aufirst%3DL.%26aulast%3DDumas%26aufirst%3DA.%26aulast%3DFilali%26aufirst%3DM.%26aulast%3DLecours%26aufirst%3DC.%26aulast%3DLacroix%26aufirst%3DS.%26aulast%3DRivest%26aufirst%3DS.%26aulast%3DTremblay%26aufirst%3DM.-E.%26aulast%3DVallieres%26aufirst%3DL.%26atitle%3DGPR84%2520deficiency%2520reduces%2520microgliosis%252C%2520but%2520accelerates%2520dendritic%2520degeneration%2520and%2520cognitive%2520decline%2520in%2520a%2520mouse%2520model%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DBrain%252C%2520Behav.%252C%2520Immun.%26date%3D2015%26volume%3D46%26spage%3D112%26epage%3D120%26doi%3D10.1016%2Fj.bbi.2015.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchigami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ota, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, Y.</span></span> <span> </span><span class="NLM_article-title">Inflammatory changes in adipose tissue enhance expression of GPR84, a medium-chain fatty acid receptor: TNFα enhances GPR84 expression in adipocytes</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.febslet.2012.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=22245676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1yqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=368-372&author=H.+Nagasakiauthor=T.+Kondoauthor=M.+Fuchigamiauthor=H.+Hashimotoauthor=Y.+Sugimuraauthor=N.+Ozakiauthor=H.+Arimaauthor=A.+Otaauthor=Y.+Oisoauthor=Y.+Hamada&title=Inflammatory+changes+in+adipose+tissue+enhance+expression+of+GPR84%2C+a+medium-chain+fatty+acid+receptor%3A+TNF%CE%B1+enhances+GPR84+expression+in+adipocytes&doi=10.1016%2Fj.febslet.2012.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammatory changes in adipose tissue enhance expression of GPR G-protein coupled receptor84, a medium-chain fatty acid receptor</span></div><div class="casAuthors">Nagasaki, Hiroshi; Kondo, Takaaki; Fuchigami, Masahiro; Hashimoto, Hiroyuki; Sugimura, Yoshihisa; Ozaki, Nobuaki; Arima, Hiroshi; Ota, Akira; Oiso, Yutaka; Hamada, Yoji</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this study we aimed to identify the physiol. roles of G protein-coupled receptor 84 (GPR84) in adipose tissue, together with medium-chain fatty acids (MCFAs), the specific ligands for GPR84.  In mice, high-fat diet up-regulated GPR84 expression in fat pads.  In 3T3-L1 adipocytes, coculture with a macrophage cell line, RAW264, or TNFα remarkably enhanced GPR84 expression.  In the presence of TNFα, MCFAs down-regulated adiponectin mRNA expression in 3T3-L1 adipocytes.  Taken together, our results suggest that GPR84 emerges in adipocytes in response to TNFα from infiltrating macrophages and exacerbates the vicious cycle between adiposity and diabesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiwUz5s6M9zLVg90H21EOLACvtfcHk0liJbXi8eVWy-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1yqu7w%253D&md5=8fecdea5b404e472d6f66a9ed22cd88b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DNagasaki%26aufirst%3DH.%26aulast%3DKondo%26aufirst%3DT.%26aulast%3DFuchigami%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DSugimura%26aufirst%3DY.%26aulast%3DOzaki%26aufirst%3DN.%26aulast%3DArima%26aufirst%3DH.%26aulast%3DOta%26aufirst%3DA.%26aulast%3DOiso%26aufirst%3DY.%26aulast%3DHamada%26aufirst%3DY.%26atitle%3DInflammatory%2520changes%2520in%2520adipose%2520tissue%2520enhance%2520expression%2520of%2520GPR84%252C%2520a%2520medium-chain%2520fatty%2520acid%2520receptor%253A%2520TNF%25CE%25B1%2520enhances%2520GPR84%2520expression%2520in%2520adipocytes%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D368%26epage%3D372%26doi%3D10.1016%2Fj.febslet.2012.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellendorph, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräuner-Osborne, H.</span></span> <span> </span><span class="NLM_article-title">Molecular pharmacology of promiscuous seven transmembrane receptors sensing organic nutrients</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1124/mol.109.055244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1124%2Fmol.109.055244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=19487246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVyrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=453-465&author=P.+Wellendorphauthor=L.+D.+Johansenauthor=H.+Br%C3%A4uner-Osborne&title=Molecular+pharmacology+of+promiscuous+seven+transmembrane+receptors+sensing+organic+nutrients&doi=10.1124%2Fmol.109.055244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular pharmacology of promiscuous seven transmembrane receptors sensing organic nutrients</span></div><div class="casAuthors">Wellendorph, Petrine; Dan Johansen, Lars; Brauner-Osborne, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">453-465</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  A no. of highly promiscuous 7 transmembrane (7TM) receptors were cloned and characterized within the last few years.  It is noteworthy that many of these receptors are activated broadly by amino acids, proteolytic degrdn. products, carbohydrates, or free fatty acids and are expressed in taste tissue, the gastrointestinal tract, endocrine glands, adipose tissue, and/or kidney.  These receptors thus hold the potential to act as sensors of food intake, regulating, for example, release of incretin hormones from the gut, insulin/glucagon from the pancreas, and leptin from adipose tissue.  The promiscuous tendency in ligand recognition of these receptors is in contrast to the typical specific interaction with one physiol. agonist seen for most receptors, which challenges the classic "lock-and-key" concept.  We here review the mol. mechanisms of nutrient sensing of the calcium-sensing receptor, the G protein-coupled receptor family C, group 6, subtype A (GPRC6A), and the taste1 receptor T1R1/T1R3, which are sensing L-α-amino acids, the carbohydrate-sensing T1R2/T1R3 receptor, the proteolytic degrdn. product sensor GPR93 (also termed GPR92), and the free fatty acid (FFA) sensing receptors FFA1, FFA2, FFA3, GPR84, and GPR120.  The involvement of the individual receptors in sensing of food intake was validated to different degrees because of limited availability of specific pharmacol. tools and/or receptor knockout mice.  However, as a group, the receptors represent potential drug targets, to treat, for example, type II diabetes by mimicking food intake by potent agonists or pos. allosteric modulators.  The ligand-receptor interactions of the promiscuous receptors of org. nutrients thus remain an interesting subject of emerging functional importance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMz4I0wYJgi7Vg90H21EOLACvtfcHk0liJbXi8eVWy-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVyrs7Y%253D&md5=6dd526aa25e07a85ad9e4e5f45ce0a72</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.055244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.055244%26sid%3Dliteratum%253Aachs%26aulast%3DWellendorph%26aufirst%3DP.%26aulast%3DJohansen%26aufirst%3DL.%2BD.%26aulast%3DBr%25C3%25A4uner-Osborne%26aufirst%3DH.%26atitle%3DMolecular%2520pharmacology%2520of%2520promiscuous%2520seven%2520transmembrane%2520receptors%2520sensing%2520organic%2520nutrients%26jtitle%3DMol.%2520Pharmacol.%26date%3D2009%26volume%3D76%26spage%3D453%26epage%3D465%26doi%3D10.1124%2Fmol.109.055244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yonezawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murayama, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, A.</span></span> <span> </span><span class="NLM_article-title">Free fatty acids-sensing G protein-coupled receptors in drug targeting and therapeutics</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3855</span>– <span class="NLM_lpage">3871</span>, <span class="refDoi"> DOI: 10.2174/09298673113209990168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.2174%2F09298673113209990168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23862620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=3855-3871&author=T.+Yonezawaauthor=R.+Kurataauthor=K.+Yoshidaauthor=M.+A.+Murayamaauthor=X.+Cuiauthor=A.+Hasegawa&title=Free+fatty+acids-sensing+G+protein-coupled+receptors+in+drug+targeting+and+therapeutics&doi=10.2174%2F09298673113209990168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Free Fatty Acids-Sensing G Protein-Coupled Receptors in Drug Targeting and Therapeutics</span></div><div class="casAuthors">Yonezawa, Tomo; Kurata, Riho; Yoshida, Kaori; Murayama, Masanori A.; Cui, Xiaofeng; Hasegawa, Akihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">3855-3871</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptor (GPCR) (also known as seven-transmembrane domain receptor) superfamily represents the largest protein family in the human genome.  These receptors respond to various physiol. ligands such as photons, odors, pheromones, hormones, ions, and small mols. including amines, amino acids to large peptides and steroids.  Thus, GPCRs are involved in many diseases and the target of around half of all conventional drugs.  The physiol. roles of free fatty acids (FFAs), in particular, long-chain FFAs, are important for the development of many metabolic disease including obesity, diabetes, and atherosclerosis.  In the past half decade, deorphanization of several GPCRs has revealed that GPR40, GPR41, GPR43, GPR84 and GPR120 sense concn. of extracellular FFAs with various carbon chain lengths.  GPR40 and GPR120 are activated by medium- and long-chain FFAs.  GPR84 is activated by medium- chain, but not long-chain, FFAs.  GPR41 and GPR43 are activated by short-chain FFAs.  GPR40 is highly expressed in pancreatic beta cells and plays a crucial role in FFAs-induced insulin secretion.  GPR120 is mainly expressed in enteroendocrine cells and plays an important role for FFAs-induced glucagon-like peptide-1.  GPR43 is abundant in leukocytes and adipose tissue, while GPR41 is highly expressed in adipose tissue, the pancreas and leukocytes.  GPR84 is expressed in leukocytes and monocyte/macrophage.  This review aims to shed light on the physiol. roles and development of drugs targeting these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyygc49JEvWLVg90H21EOLACvtfcHk0li92MlIBE7Nqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGnsrjM&md5=6c546f83c25e551b7a557909e713c5e8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.2174%2F09298673113209990168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F09298673113209990168%26sid%3Dliteratum%253Aachs%26aulast%3DYonezawa%26aufirst%3DT.%26aulast%3DKurata%26aufirst%3DR.%26aulast%3DYoshida%26aufirst%3DK.%26aulast%3DMurayama%26aufirst%3DM.%2BA.%26aulast%3DCui%26aufirst%3DX.%26aulast%3DHasegawa%26aufirst%3DA.%26atitle%3DFree%2520fatty%2520acids-sensing%2520G%2520protein-coupled%2520receptors%2520in%2520drug%2520targeting%2520and%2520therapeutics%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2013%26volume%3D20%26spage%3D3855%26epage%3D3871%26doi%3D10.2174%2F09298673113209990168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasubuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, I.</span></span> <span> </span><span class="NLM_article-title">Nutritional signaling via free fatty acid receptors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.3390/ijms17040450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.3390%2Fijms17040450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=27023530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslCmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=450-462&author=J.+Miyamotoauthor=S.+Hasegawaauthor=M.+Kasubuchiauthor=A.+Ichimuraauthor=A.+Nakajimaauthor=I.+Kimura&title=Nutritional+signaling+via+free+fatty+acid+receptors&doi=10.3390%2Fijms17040450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nutritional signaling via free fatty acid receptors</span></div><div class="casAuthors">Miyamoto, Junki; Hasegawa, Sae; Kasubuchi, Mayu; Ichimura, Atsuhiko; Nakajima, Akira; Kimura, Ikuo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450/1-450/12</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Excess energy is stored primarily as triglycerides, which are mobilized when demand for energy arises.  Dysfunction of energy balance by excess food intake leads to metabolic diseases, such as obesity and diabetes.  Free fatty acids (FFAs) provided by dietary fat are not only important nutrients, but also contribute key physiol. functions via FFA receptor (FFAR)-mediated signaling mols., which depend on FFAs' carbon chain length and the ligand specificity of the receptors.  Functional analyses have revealed that FFARs are crit. for metabolic functions, such as peptide hormone secretion and inflammation, and contribute to energy homeostasis.  In particular, recent studies have shown that the administration of selective agonists of G protein-coupled receptor (GPR) 40 and GPR120 improved glucose metab. and systemic metabolic disorders.  Furthermore, the anti-inflammation and energy metab. effects of short chain FAs have been linked to the activation of GPR41 and GPR43.  In this review, we summarize recent progress in research on FFAs and their physiol. roles in the regulation of energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PqYSYQi3q7Vg90H21EOLACvtfcHk0li92MlIBE7Nqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslCmsb4%253D&md5=f78fdb70502dc2a96a156068edcb9637</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3390%2Fijms17040450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17040450%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DJ.%26aulast%3DHasegawa%26aufirst%3DS.%26aulast%3DKasubuchi%26aufirst%3DM.%26aulast%3DIchimura%26aufirst%3DA.%26aulast%3DNakajima%26aufirst%3DA.%26aulast%3DKimura%26aufirst%3DI.%26atitle%3DNutritional%2520signaling%2520via%2520free%2520fatty%2520acid%2520receptors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D450%26epage%3D462%26doi%3D10.3390%2Fijms17040450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, H. H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Y.</span></span> <span> </span><span class="NLM_article-title">GPR84 sustains aberrant beta-catenin signaling in leukemic stem cells for maintenance of MLL leukemogenesis</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">3284</span>– <span class="NLM_lpage">3294</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-10-532523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1182%2Fblood-2013-10-532523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25293777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSgs73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=3284-3294&author=P.+A.+Dietrichauthor=C.+Yangauthor=H.+H.+L.+Leungauthor=J.+R.+Lynchauthor=E.+Gonzalesauthor=B.+Liuauthor=M.+Haberauthor=M.+D.+Norrisauthor=J.+L.+Wangauthor=J.+Y.+Wang&title=GPR84+sustains+aberrant+beta-catenin+signaling+in+leukemic+stem+cells+for+maintenance+of+MLL+leukemogenesis&doi=10.1182%2Fblood-2013-10-532523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">GPR84 sustains aberrant β-catenin signaling in leukemic stem cells for maintenance of MLL leukemogenesis</span></div><div class="casAuthors">Dietrich, Philipp A.; Yang, Chen; Leung, Halina H. L.; Lynch, Jennifer R.; Gonzales, Estrella; Liu, Bing; Haber, Michelle; Norris, Murray D.; Wang, Jianlong; Wang, Jenny Yingzi</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3284-3294</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">β-Catenin is required for establishment of leukemic stem cells (LSCs) in acute myeloid leukemia (AML).  Targeted inhibition of β-catenin signaling has been hampered by the lack of pathway components amenable to pharmacol. manipulation.  Here we identified a novel β-catenin regulator, GPR84, a member of the G protein-coupled receptor family that represents a highly tractable class of drug targets.  High GPR84 expression levels were confirmed in human and mouse AML LSCs compared with hematopoietic stem cells (HSCs).  Suppression of GPR84 significantly inhibited cell growth by inducing G1-phase cell-cycle arrest in pre-LSCs, reduced LSC frequency, and impaired reconstitution of stem cell-derived mixed-lineage leukemia (MLL) AML, which represents an aggressive and drug-resistant subtype of AML.  The GPR84-deficient phenotype in established AML could be rescued by expression of constitutively active β-catenin.  Furthermore, GPR84 conferred a growth advantage to Hoxa9/Meis1a-transduced stem cells.  Microarray anal. demonstrated that GPR84 significantly upregulated a small set of MLL-fusion targets and β-catenin coeffectors, and downregulated a hematopoietic cell-cycle inhibitor.  Altogether, our data reveal a previously unrecognized role of GPR84 in maintaining fully developed AML by sustaining aberrant β-catenin signaling in LSCs, and suggest that targeting the oncogenic GPR84/β-catenin signaling axis may represent a novel therapeutic strategy for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqayOBoeIKDt7Vg90H21EOLACvtfcHk0li92MlIBE7Nqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVSgs73L&md5=3da07da89bd6da728d09aa26b273d659</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-10-532523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-10-532523%26sid%3Dliteratum%253Aachs%26aulast%3DDietrich%26aufirst%3DP.%2BA.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLeung%26aufirst%3DH.%2BH.%2BL.%26aulast%3DLynch%26aufirst%3DJ.%2BR.%26aulast%3DGonzales%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHaber%26aufirst%3DM.%26aulast%3DNorris%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DJ.%2BL.%26aulast%3DWang%26aufirst%3DJ.%2BY.%26atitle%3DGPR84%2520sustains%2520aberrant%2520beta-catenin%2520signaling%2520in%2520leukemic%2520stem%2520cells%2520for%2520maintenance%2520of%2520MLL%2520leukemogenesis%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D3284%26epage%3D3294%26doi%3D10.1182%2Fblood-2013-10-532523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor 84 controls osteoclastogenesis through inhibition of NF-κB and MAPK signaling pathways</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>233</i></span>,  <span class="NLM_fpage">1481</span>– <span class="NLM_lpage">1489</span>, <span class="refDoi"> DOI: 10.1002/jcp.26035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fjcp.26035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=28574596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=2018&pages=1481-1489&author=J.+W.+Parkauthor=H.+J.+Yoonauthor=W.+Y.+Kangauthor=S.+Choauthor=S.+J.+Seongauthor=H.+W.+Leeauthor=Y.+R.+Yoonauthor=H.+J.+Kim&title=G+protein-coupled+receptor+84+controls+osteoclastogenesis+through+inhibition+of+NF-%CE%BAB+and+MAPK+signaling+pathways&doi=10.1002%2Fjcp.26035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor 84 controls osteoclastogenesis through inhibition of NF-κB and MAPK signaling pathways</span></div><div class="casAuthors">Park, Ji-Wan; Yoon, Hye-Jin; Kang, Woo Youl; Cho, Seungil; Seong, Sook Jin; Lee, Hae Won; Yoon, Young-Ran; Kim, Hyun-Ju</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1481-1489</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">GPR84, a member of the G protein-coupled receptor family, is found predominantly in immune cells, such as macrophages, and functions as a pivotal modulator of inflammatory responses.  In this study, we investigated the role of GPR84 in receptor activator of nuclear factor-κB ligand (RANKL)-induced osteoclast differentiation.  Our microarray data showed that GPR84 was significantly downregulated in osteoclasts compared to in their precursors, macrophages.  The overexpression of GPR84 in bone marrow-derived macrophages suppressed the formation of multinucleated osteoclasts without affecting precursor proliferation.  In addn., GPR84 overexpression attenuated the induction of c-Fos and nuclear factor of activated T cells, cytoplasmic 1 (NFATc1), which are transcription factors that are crit. for osteoclastogenesis.  Furthermore, knockdown of GPR84 using a small hairpin RNA promoted RANKL-mediated osteoclast differentiation and gene expression of osteoclastogenic markers.  Mechanistically, GPR84 overexpression blocked RANKL-stimulated phosphorylation of IκBα and three MAPKs, JNK, ERK, and p38.  GPR84 also suppressed NF-κB transcriptional activity mediated by RANKL.  Conversely, GPR84 knockdown enhanced RANKL-induced activation of IκBα and the three MAPKs.  Collectively, our results revealed that GPR84 functions as a neg. regulator of osteoclastogenesis, suggesting that it may be a potential therapeutic target for osteoclast-mediated bone-destructive diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvjBAcWdjpLrVg90H21EOLACvtfcHk0lhqqLS17ZeRXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSmtbvM&md5=fb81b70f841ab32cc66e898a3810405d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fjcp.26035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.26035%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DYoon%26aufirst%3DH.%2BJ.%26aulast%3DKang%26aufirst%3DW.%2BY.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DSeong%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DH.%2BW.%26aulast%3DYoon%26aufirst%3DY.%2BR.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26atitle%3DG%2520protein-coupled%2520receptor%252084%2520controls%2520osteoclastogenesis%2520through%2520inhibition%2520of%2520NF-%25CE%25BAB%2520and%2520MAPK%2520signaling%2520pathways%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D2018%26volume%3D233%26spage%3D1481%26epage%3D1489%26doi%3D10.1002%2Fjcp.26035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibodeau, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grouix, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bournet- Sarra, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hince, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebert, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutsol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holterman, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gervais, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouboudinar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laverdure, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letournau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblond, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbot, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duceppe, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacharie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurin, P.</span></span> <span> </span><span class="NLM_article-title">A newly discovered antifibrotic pathway regulated by two fatty acid receptors</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>188</i></span>,  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1148</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2018.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.ajpath.2018.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29454750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGqs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2018&pages=1132-1148&author=L.+Gagnonauthor=M.+Leducauthor=J.+F.+Thibodeauauthor=M.+Z.+Zhangauthor=B.+Grouixauthor=F.+Bournet-+Sarraauthor=W.+Gagnonauthor=K.+Hinceauthor=M.+Tremblayauthor=L.+Geertsauthor=C.+R.+J.+Kennedyauthor=R.+L.+Hebertauthor=A.+Gutsolauthor=C.+E.+Holtermanauthor=E.+Kamtoauthor=L.+Gervaisauthor=J.+Ouboudinarauthor=J.+Richardauthor=A.+Feltonauthor=A.+Laverdureauthor=J.+C.+Simardauthor=S.+Letournauauthor=M.+P.+Cloutierauthor=F.+A.+Leblondauthor=S.+D.+Abbotauthor=C.+Penneyauthor=J.+S.+Duceppeauthor=B.+Zacharieauthor=J.+Dupuisauthor=A.+Calderoneauthor=Q.+T.+Nguyenauthor=R.+C.+Harrisauthor=P.+Laurin&title=A+newly+discovered+antifibrotic+pathway+regulated+by+two+fatty+acid+receptors&doi=10.1016%2Fj.ajpath.2018.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Newly Discovered Antifibrotic Pathway Regulated by Two Fatty Acid Receptors</span></div><div class="casAuthors">Gagnon, Lyne; Leduc, Martin; Thibodeau, Jean-Francois; Zhang, Ming-Zhi; Grouix, Brigitte; Sarra-Bournet, Francois; Gagnon, William; Hince, Kathy; Tremblay, Mikael; Geerts, Lilianne; Kennedy, Christopher R. J.; Hebert, Richard L.; Gutsol, Alex; Holterman, Chet E.; Kamto, Eldjonai; Gervais, Liette; Ouboudinar, Jugurtha; Richard, Jonathan; Felton, Alexandra; Laverdure, Alexandre; Simard, Jean-Christophe; Letourneau, Sylvie; Cloutier, Marie-Pier; Leblond, Francois A.; Abbott, Shaun D.; Penney, Christopher; Duceppe, Jean-Simon; Zacharie, Boulos; Dupuis, Jocelyn; Calderone, Angelino; Nguyen, Quang T.; Harris, Raymond C.; Laurin, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">188</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1132-1148</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Numerous clin. conditions can lead to organ fibrosis and functional failure.  There is a great need for therapies that could effectively target pathophysiol. pathways involved in fibrosis.  GPR40 and GPR84 are G protein-coupled receptors with free fatty acid ligands and are assocd. with metabolic and inflammatory disorders.  Although GPR40 and GPR84 are involved in diverse physiol. processes, no evidence has demonstrated the relevance of GPR40 and GPR84 in fibrosis pathways.  Using PBI-4050 (3-pentylbenzeneacetic acid sodium salt), a synthetic analog of a medium-chain fatty acid that displays agonist and antagonist ligand affinity toward GPR40 and GPR84, resp., we uncovered an antifibrotic pathway involving these receptors.  In expts. using Gpr40- and Gpr84-knockout mice in models of kidney fibrosis (unilateral ureteral obstruction, long-term post-acute ischemic injury, and adenine-induced chronic kidney disease), we found that GPR40 is protective and GPR84 is deleterious in these diseases.  Moreover, through binding to GPR40 and GPR84, PBI-4050 significantly attenuated fibrosis in many injury contexts, as evidenced by the antifibrotic activity obsd. in kidney, liver, heart, lung, pancreas, and skin fibrosis models.  Therefore, GPR40 and GPR84 may represent promising mol. targets in fibrosis pathways.  We conclude that PBI-4050 is a first-in-class compd. that may be effective for managing inflammatory and fibrosis-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogzjSwncX3zLVg90H21EOLACvtfcHk0lhqqLS17ZeRXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGqs7w%253D&md5=47fff3190f70beafa69897d9d89f206d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2018.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2018.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DGagnon%26aufirst%3DL.%26aulast%3DLeduc%26aufirst%3DM.%26aulast%3DThibodeau%26aufirst%3DJ.%2BF.%26aulast%3DZhang%26aufirst%3DM.%2BZ.%26aulast%3DGrouix%26aufirst%3DB.%26aulast%3DBournet-%2BSarra%26aufirst%3DF.%26aulast%3DGagnon%26aufirst%3DW.%26aulast%3DHince%26aufirst%3DK.%26aulast%3DTremblay%26aufirst%3DM.%26aulast%3DGeerts%26aufirst%3DL.%26aulast%3DKennedy%26aufirst%3DC.%2BR.%2BJ.%26aulast%3DHebert%26aufirst%3DR.%2BL.%26aulast%3DGutsol%26aufirst%3DA.%26aulast%3DHolterman%26aufirst%3DC.%2BE.%26aulast%3DKamto%26aufirst%3DE.%26aulast%3DGervais%26aufirst%3DL.%26aulast%3DOuboudinar%26aufirst%3DJ.%26aulast%3DRichard%26aufirst%3DJ.%26aulast%3DFelton%26aufirst%3DA.%26aulast%3DLaverdure%26aufirst%3DA.%26aulast%3DSimard%26aufirst%3DJ.%2BC.%26aulast%3DLetournau%26aufirst%3DS.%26aulast%3DCloutier%26aufirst%3DM.%2BP.%26aulast%3DLeblond%26aufirst%3DF.%2BA.%26aulast%3DAbbot%26aufirst%3DS.%2BD.%26aulast%3DPenney%26aufirst%3DC.%26aulast%3DDuceppe%26aufirst%3DJ.%2BS.%26aulast%3DZacharie%26aufirst%3DB.%26aulast%3DDupuis%26aufirst%3DJ.%26aulast%3DCalderone%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DQ.%2BT.%26aulast%3DHarris%26aufirst%3DR.%2BC.%26aulast%3DLaurin%26aufirst%3DP.%26atitle%3DA%2520newly%2520discovered%2520antifibrotic%2520pathway%2520regulated%2520by%2520two%2520fatty%2520acid%2520receptors%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2018%26volume%3D188%26spage%3D1132%26epage%3D1148%26doi%3D10.1016%2Fj.ajpath.2018.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Southern, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neetoo-Isseljee, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettleborough, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassoni, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehrman, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, S.</span></span> <span> </span><span class="NLM_article-title">Screening β-arrestin recruitment for the identification of natural ligands for orphan Gprotein-coupled receptors</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1177/1087057113475480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1177%2F1087057113475480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23396314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWisb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=599-609&author=C.+Southernauthor=J.+M.+Cookauthor=Z.+Neetoo-Isseljeeauthor=D.+L.+Taylorauthor=C.+A.+Kettleboroughauthor=A.+Merrittauthor=D.+L.+Bassoniauthor=W.+J.+Raabauthor=E.+Quinnauthor=T.+S.+Wehrmanauthor=A.+P.+Davenportauthor=A.+J.+Brownauthor=A.+Greenauthor=M.+J.+Wigglesworthauthor=S.+Rees&title=Screening+%CE%B2-arrestin+recruitment+for+the+identification+of+natural+ligands+for+orphan+Gprotein-coupled+receptors&doi=10.1177%2F1087057113475480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Screening β-arrestin recruitment for the identification of natural ligands for orphan G-protein-coupled receptors</span></div><div class="casAuthors">Southern, Craig; Cook, Jennifer M.; Neetoo-Isseljee, Zaynab; Taylor, Debra L.; Kettleborough, Catherine A.; Merritt, Andy; Bassoni, Daniel L.; Raab, William J.; Quinn, Elizabeth; Wehrman, Tom S.; Davenport, Anthony P.; Brown, Andrew J.; Green, Andrew; Wigglesworth, Mark J.; Rees, Steve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">599-609, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A variety of G-protein-coupled receptor (GPCR) screening technologies have successfully partnered a no. of GPCRs with their cognate ligands.  GPCR-mediated β-arrestin recruitment is now recognized as a distinct intracellular signaling pathway and ligand-receptor interactions may show a bias toward β-arrestin over classical GPCR signaling pathways.  We hypothesized that the failure to identify native ligands for the remaining orphan GPCRs may be a consequence of biased β-arrestin signaling.  To investigate this, we assembled 10 500 candidate ligands and screened 82 GPCRs using PathHunter β-arrestin recruitment technol.  High-quality screening assays were validated by the inclusion of liganded receptors and the detection and confirmation of these established ligand-receptor pairings.  We describe a candidate endogenous orphan GPCR ligand and a no. of novel surrogate ligands.  However, for the majority of orphan receptors studied, measurement of β-arrestin recruitment did not lead to the identification of cognate ligands from our screening sets. β-Arrestin recruitment represents a robust GPCR screening technol. and ligand-biased signaling is emerging as a therapeutically exploitable feature of GPCR biol.  The identification of cognate ligands for the orphan GPCRs and the extent to which receptors may exist to preferentially signal through β-arrestin in response to their native ligand remain to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRqw2KqMq-GrVg90H21EOLACvtfcHk0ljDam7YwVC7Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWisb%252FP&md5=2d52934eea58a90c12236001e951e107</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1087057113475480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113475480%26sid%3Dliteratum%253Aachs%26aulast%3DSouthern%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DJ.%2BM.%26aulast%3DNeetoo-Isseljee%26aufirst%3DZ.%26aulast%3DTaylor%26aufirst%3DD.%2BL.%26aulast%3DKettleborough%26aufirst%3DC.%2BA.%26aulast%3DMerritt%26aufirst%3DA.%26aulast%3DBassoni%26aufirst%3DD.%2BL.%26aulast%3DRaab%26aufirst%3DW.%2BJ.%26aulast%3DQuinn%26aufirst%3DE.%26aulast%3DWehrman%26aufirst%3DT.%2BS.%26aulast%3DDavenport%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DWigglesworth%26aufirst%3DM.%2BJ.%26aulast%3DRees%26aufirst%3DS.%26atitle%3DScreening%2520%25CE%25B2-arrestin%2520recruitment%2520for%2520the%2520identification%2520of%2520natural%2520ligands%2520for%2520orphan%2520Gprotein-coupled%2520receptors%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D599%26epage%3D609%26doi%3D10.1177%2F1087057113475480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span> <span> </span><span class="NLM_article-title">Mutation analysis and molecular modeling for the investigation of ligand-binding modes of GPR84</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">320</span>, <span class="refDoi"> DOI: 10.1093/jb/mvu075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1093%2Fjb%2Fmvu075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25425658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOgtLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2015&pages=311-320&author=Y.+Nikaidoauthor=Y.+Koyamaauthor=Y.+Yoshikawaauthor=T.+Furuyaauthor=S.+Takeda&title=Mutation+analysis+and+molecular+modeling+for+the+investigation+of+ligand-binding+modes+of+GPR84&doi=10.1093%2Fjb%2Fmvu075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mutation analysis and molecular modeling for the investigation of ligand-binding modes of GPR84</span></div><div class="casAuthors">Nikaido, Yoshiaki; Koyama, Yuuta; Yoshikawa, Yasushi; Furuya, Toshio; Takeda, Shigeki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">311-320</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">GPR84 is a G protein-coupled receptor for medium-chain fatty acids.  Capric acid and 3,3'-diindolylmethane are specific agonists for GPR84.  We built a homol. model of a GPR84-capric acid complex to investigate the ligand-binding mode using the crystal structure of human active-state β2-adrenergic receptor.  We performed site-directed mutagenesis to subject ligand-binding sites to our model using GPR84-Giα fusion proteins and a [35S]GTPγS-binding assay.  We compared the activity of the wild type and mutated forms of GPR84 by [35S]GTPγS binding to capric acid and diindolylmethane.  The mutations L100D 'allesteros-Weinstein numbering: 3.32, F101Y 3.33 and N104Q 3.36 in the transmembrane helix III and N357D 7.39 in the transmembrane helix VII resulted in reduced capric acid activity but maintained the diindolylmethane responses.  Y186F 5.46 and Y186H 5.46 mutations had no characteristic effect on capric acid but with diindolylmethane they significantly affected the G protein activation efficiency.  The L100D 3.32 mutant responded to decylamine, a fatty amine, instead of a natural agonist, the fatty acid capric acid, suggesting that we have identified a mutated G protein-coupled receptor-artificial ligand pairing.  Our mol. model provides an explanation for these results and interactions between GPR84 and capric acid.  Further, from the results of a double stimulation assay, we concluded that diindolylmethane was a pos. allosteric modulator for GPR84.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpnoqt2xo2U7Vg90H21EOLACvtfcHk0ljDam7YwVC7Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOgtLzK&md5=b85955265588602441607c1cbd6a76ef</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvu075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvu075%26sid%3Dliteratum%253Aachs%26aulast%3DNikaido%26aufirst%3DY.%26aulast%3DKoyama%26aufirst%3DY.%26aulast%3DYoshikawa%26aufirst%3DY.%26aulast%3DFuruya%26aufirst%3DT.%26aulast%3DTakeda%26aufirst%3DS.%26atitle%3DMutation%2520analysis%2520and%2520molecular%2520modeling%2520for%2520the%2520investigation%2520of%2520ligand-binding%2520modes%2520of%2520GPR84%26jtitle%3DJ.%2520Biochem.%26date%3D2015%26volume%3D157%26spage%3D311%26epage%3D320%26doi%3D10.1093%2Fjb%2Fmvu075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaspersen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunlop, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulven, T.</span></span> <span> </span><span class="NLM_article-title">Succinct synthesis of saturated hydroxy fatty acids and in vitro evaluation of all hydroxylauric acids on FFA1, FFA4 and GPR84</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1360</span>– <span class="NLM_lpage">1365</span>, <span class="refDoi"> DOI: 10.1039/C7MD00130D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1039%2FC7MD00130D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=30108848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXns1Khtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1360-1365&author=M.+H.+Kaspersenauthor=L.+Jenkinsauthor=J.+Dunlopauthor=G.+Milliganauthor=T.+Ulven&title=Succinct+synthesis+of+saturated+hydroxy+fatty+acids+and+in+vitro+evaluation+of+all+hydroxylauric+acids+on+FFA1%2C+FFA4+and+GPR84&doi=10.1039%2FC7MD00130D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Succinct synthesis of saturated hydroxy fatty acids and in vitro evaluation of all hydroxylauric acids on FFA1, FFA4 and GPR84</span></div><div class="casAuthors">Kaspersen, Mads Holmgaard; Jenkins, Laura; Dunlop, Julia; Milligan, Graeme; Ulven, Trond</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1360-1365</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Satd. hydroxy fatty acids make up a class of underexplored lipids with potentially interesting biol. activities.  We report a succinct and general synthetic route to satd. hydroxy fatty acids hydroxylated at position 6 or higher, and exemplify this with the synthesis of hydroxylauric acids.  All regioisomers of hydroxylauric acids were tested on free fatty acid receptors FFA1, FFA4 and GPR84.  The results show that the introduction of a hydroxy group and its position have a high impact on receptor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmMRJHqZLXrVg90H21EOLACvtfcHk0ljDam7YwVC7Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXns1Khtr4%253D&md5=4515bf7026ba72b2e9abaeb6148ee957</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC7MD00130D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00130D%26sid%3Dliteratum%253Aachs%26aulast%3DKaspersen%26aufirst%3DM.%2BH.%26aulast%3DJenkins%26aufirst%3DL.%26aulast%3DDunlop%26aufirst%3DJ.%26aulast%3DMilligan%26aufirst%3DG.%26aulast%3DUlven%26aufirst%3DT.%26atitle%3DSuccinct%2520synthesis%2520of%2520saturated%2520hydroxy%2520fatty%2520acids%2520and%2520in%2520vitro%2520evaluation%2520of%2520all%2520hydroxylauric%2520acids%2520on%2520FFA1%252C%2520FFA4%2520and%2520GPR84%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1360%26epage%3D1365%26doi%3D10.1039%2FC7MD00130D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borges, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kügelgen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">6-(Ar)Alkylamino-substituted uracil derivatives: lipid mimetics with potent activity at the orphan G protein-coupled receptor 84 (GPR84)</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">3365</span>– <span class="NLM_lpage">3383</span>, <span class="refDoi"> DOI: 10.1021/acsomega.7b02092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b02092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVyqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=3365-3383&author=T.+Pillaiyarauthor=M.+K%C3%B6seauthor=V.+Namasivayamauthor=K.+Sylvesterauthor=G.+Borgesauthor=D.+Thimmauthor=I.+von+K%C3%BCgelgenauthor=C.+E.+M%C3%BCller&title=6-%28Ar%29Alkylamino-substituted+uracil+derivatives%3A+lipid+mimetics+with+potent+activity+at+the+orphan+G+protein-coupled+receptor+84+%28GPR84%29&doi=10.1021%2Facsomega.7b02092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">6-(Ar)Alkylamino-Substituted Uracil Derivatives: Lipid Mimetics with Potent Activity at the Orphan G Protein-Coupled Receptor 84 (GPR84)</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Koese, Meryem; Namasivayam, Vigneshwaran; Sylvester, Katharina; Borges, Gleice; Thimm, Dominik; von Kuegelgen, Ivar; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">3365-3383</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GPR84, a Gi protein-coupled receptor that is activated by medium-chain (hydroxy)fatty acids, appears to play an important role in inflammation, immunity, and cancer.  Recently, 6-octylaminouracil has been reported to act as an agonist at GPR84.  Here, we describe the synthesis of 69 derivs. and analogs which represent new compds.  They were evaluated in (a) cyclic adenosine monophosphate accumulation and (b) β-arrestin assays in human GPR84-expressing cells.  Potent nonbiased as well as G protein-biased agonists were developed, e.g., 6-hexylamino-2,4(1H,3H)-pyrimidinedione (PSB-1584, EC50 5.0 nM (a), 3.2 nM (b), bias factor: 0) and 6-((p-chloro- and p-bromo-phenylethyl)amino)-2,4(1H,3H)-pyrimidinedione (PSB-16434, EC50 7.1 nM (a), 520 nM (b), bias factor: 1.9 = 79-fold Gi pathway-selective; I, PSB-17365, EC50 2.5 nM (a), 100 nM (b), bias factor 1.3 = 20-fold selective), which were selective vs. other free fatty acid-activated receptors.  Compds. II and I were found to be metabolically stable upon incubation with human liver microsomes.  A pharmacophore model was created on the basis of structurally diverse lipidlike GPR84 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc4TDL4U0tF7Vg90H21EOLACvtfcHk0liltCSrMu0pxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVyqu7g%253D&md5=184964bac823f7cd34d4de98da0cef53</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b02092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b02092%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DSylvester%26aufirst%3DK.%26aulast%3DBorges%26aufirst%3DG.%26aulast%3DThimm%26aufirst%3DD.%26aulast%3Dvon%2BK%25C3%25BCgelgen%26aufirst%3DI.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3D6-%2528Ar%2529Alkylamino-substituted%2520uracil%2520derivatives%253A%2520lipid%2520mimetics%2520with%2520potent%2520activity%2520at%2520the%2520orphan%2520G%2520protein-coupled%2520receptor%252084%2520%2528GPR84%2529%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D3365%26epage%3D3383%26doi%3D10.1021%2Facsomega.7b02092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahmud, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulven, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labeguere, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tikhonova, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span> <span> </span><span class="NLM_article-title">Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">17953</span>– <span class="NLM_lpage">17967</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-18159-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fs41598-017-18159-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29263400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC1MzjslSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=17953-17967&author=Z.+A.+Mahmudauthor=L.+Jenkinsauthor=T.+Ulvenauthor=F.+Labeguereauthor=R.+Gosminiauthor=S.+De+Vosauthor=B.+D.+Hudsonauthor=I.+G.+Tikhonovaauthor=G.+Milligan&title=Three+classes+of+ligands+each+bind+to+distinct+sites+on+the+orphan+G+protein-coupled+receptor+GPR84&doi=10.1038%2Fs41598-017-18159-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84</span></div><div class="casAuthors">Mahmud Zobaer Al; Jenkins Laura; Hudson Brian D; Milligan Graeme; Ulven Trond; Labeguere Frederic; Gosmini Romain; Labeguere Frederic; De Vos Steve; Tikhonova Irina G</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17953</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Medium chain fatty acids can activate the pro-inflammatory receptor GPR84 but so also can molecules related to 3,3'-diindolylmethane. 3,3'-Diindolylmethane and decanoic acid acted as strong positive allosteric modulators of the function of each other and analysis showed the affinity of 3,3'-diindolylmethane to be at least 100 fold higher.  Methyl decanoate was not an agonist at GPR84.  This implies a key role in binding for the carboxylic acid of the fatty acid.  Via homology modelling we predicted and confirmed an integral role of arginine(172), located in the 2nd extracellular loop, in the action of decanoic acid but not of 3,3'-diindolylmethane.  Exemplars from a patented series of GPR84 antagonists were able to block agonist actions of both decanoic acid and 3,3'-diindolylmethane at GPR84.  However, although a radiolabelled form of a related antagonist, [(3)H]G9543, was able to bind with high affinity to GPR84, this was not competed for by increasing concentrations of either decanoic acid or 3,3'-diindolylmethane and was not affected adversely by mutation of arginine(172).  These studies identify three separable ligand binding sites within GPR84 and suggest that if medium chain fatty acids are true endogenous regulators then co-binding with a positive allosteric modulator would greatly enhance their function in physiological settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREMjSSm_naSC-bLEOO4dYpfW6udTcc2eaHkhemiZI0lbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzjslSqug%253D%253D&md5=fe3d58f70952bef231505deb9bbb9de0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-18159-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-18159-3%26sid%3Dliteratum%253Aachs%26aulast%3DMahmud%26aufirst%3DZ.%2BA.%26aulast%3DJenkins%26aufirst%3DL.%26aulast%3DUlven%26aufirst%3DT.%26aulast%3DLabeguere%26aufirst%3DF.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DDe%2BVos%26aufirst%3DS.%26aulast%3DHudson%26aufirst%3DB.%2BD.%26aulast%3DTikhonova%26aufirst%3DI.%2BG.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DThree%2520classes%2520of%2520ligands%2520each%2520bind%2520to%2520distinct%2520sites%2520on%2520the%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR84%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D17953%26epage%3D17967%26doi%3D10.1038%2Fs41598-017-18159-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weighardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borges, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Förster, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kügelgen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Diindolylmethane derivatives: potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3636</span>– <span class="NLM_lpage">3655</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01593</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01593" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVWlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3636-3655&author=T.+Pillaiyarauthor=M.+K%C3%B6seauthor=K.+Sylvesterauthor=H.+Weighardtauthor=D.+Thimmauthor=G.+Borgesauthor=I.+F%C3%B6rsterauthor=I.+von+K%C3%BCgelgenauthor=C.+E.+M%C3%BCller&title=Diindolylmethane+derivatives%3A+potent+agonists+of+the+immunostimulatory+orphan+G+protein-coupled+receptor+GPR84&doi=10.1021%2Facs.jmedchem.6b01593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Diindolylmethane Derivatives: Potent Agonists of the Immunostimulatory Orphan G Protein-Coupled Receptor GPR84</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Koese, Meryem; Sylvester, Katharina; Weighardt, Heike; Thimm, Dominik; Borges, Gleice; Foerster, Irmgard; von Kuegelgen, Ivar; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3636-3655</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">3,3'-Diindolylmethanes such as I, 3,3'-biindolylalkanes, and indolylmethylene- and indolylmethyloxindoles were prepd. as agonists of the G-protein-coupled receptor GPR84, an orphan hydroxy fatty acid receptor expressed on immune cells.  Condensation of indoles with formaldehyde or aldehydes in water under microwave irradn. at 100° with or without added sodium dodecyl sulfate yielded the diindolemethane products.  The agonism of products at the human GPR84 was detd. using in cAMP and β-arrestin assays.  Small structural changes in the diindolylmethane products yielded large changes in biol. activity.  3,3'-Diindolylmethanes bearing small lipophilic residues at the 5- and/or 7-position of the indole rings displayed the highest activity in cAMP assays; in β-arrestin assays, the structure-activity relationships differed, indicating biased agonism.  The selectivity of the compds. were detd. against the fatty acid receptors FFAR1 and FFAR4 and the arylhydrocarbon receptor.  Selected compds. were further investigated and displayed an allosteric agonist mechanism of action; the compds. were more stable than 3,3'-diindolylmethane in simulated gastric fluid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhuTbMdKJx77Vg90H21EOLACvtfcHk0liltCSrMu0pxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVWlu7Y%253D&md5=28204d35b027551aceb2d80fcd42200e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01593%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DSylvester%26aufirst%3DK.%26aulast%3DWeighardt%26aufirst%3DH.%26aulast%3DThimm%26aufirst%3DD.%26aulast%3DBorges%26aufirst%3DG.%26aulast%3DF%25C3%25B6rster%26aufirst%3DI.%26aulast%3Dvon%2BK%25C3%25BCgelgen%26aufirst%3DI.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DDiindolylmethane%2520derivatives%253A%2520potent%2520agonists%2520of%2520the%2520immunostimulatory%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR84%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3636%26epage%3D3655%26doi%3D10.1021%2Facs.jmedchem.6b01593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hakak, Y.</span>; <span class="NLM_string-name">Unett, D. J.</span>; <span class="NLM_string-name">Gatlin, J.</span>; <span class="NLM_string-name">Liaw, C. W.</span></span> <span> </span><span class="NLM_article-title">Human G Protein-Coupled Receptor and Modulators Thereof for the Treatment of Atherosclerosis and Atherosclerotic Disease and for the Treatment of Conditions Related to MCP-1 Expression</span>. International Patent application <span class="NLM_patent">WO2007027661</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=Y.+Hakak&author=D.+J.+Unett&author=J.+Gatlin&author=C.+W.+Liaw&title=Human+G+Protein-Coupled+Receptor+and+Modulators+Thereof+for+the+Treatment+of+Atherosclerosis+and+Atherosclerotic+Disease+and+for+the+Treatment+of+Conditions+Related+to+MCP-1+Expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHakak%26aufirst%3DY.%26atitle%3DHuman%2520G%2520Protein-Coupled%2520Receptor%2520and%2520Modulators%2520Thereof%2520for%2520the%2520Treatment%2520of%2520Atherosclerosis%2520and%2520Atherosclerotic%2520Disease%2520and%2520for%2520the%2520Treatment%2520of%2520Conditions%2520Related%2520to%2520MCP-1%2520Expression%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of a novel small molecule agonist for medium chain free fatty acid receptor GPR84</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>357</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1124/jpet.116.232033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1124%2Fjpet.116.232033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=26962172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWhu7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2016&pages=337-344&author=Q.+Zhangauthor=H.+Yangauthor=J.+Liauthor=X.+Xie&title=Discovery+and+characterization+of+a+novel+small+molecule+agonist+for+medium+chain+free+fatty+acid+receptor+GPR84&doi=10.1124%2Fjpet.116.232033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of a novel small-molecule agonist for medium-chain free fatty acid receptor G protein-coupled receptor 84</span></div><div class="casAuthors">Zhang, Qing; Yang, Hui; Li, Jing; Xie, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">G protein-coupled receptor 84 (GPR84) is a free fatty acid receptor activated by medium-chain free fatty acids with 9-14 carbons.  It is expressed mainly in the immune-related tissues, such as spleen, bone marrow, and peripheral blood leukocytes.  GPR84 plays significant roles in inflammatory processes and may represent a novel drug target for the treatment of immune-mediated diseases.  However, the lack of potent and specific ligands for GPR84 hindered the study of its functions and the development of potential clin. applications.  Here, we report the screen of 160,000 small- mol. compds. with a calcium mobilization assay using a human embryonic kidney 293 cell line stably expressing GPR84 and Ga16, and the identification of 2-(hexylthio)pyrimidine-4,6-diol (ZQ-16) as a potent and selective agonist of GPR84 with a novel structure.  ZQ-16 activates several GPR84-mediated signaling pathways, including calcium mobilization, inhibition of cAMP accumulation, phosphorylation of extracellular signal-regulated protein kinase 1/2, receptor desensitization and internalization, and receptor-β-arrestin interaction.  This compd. may be a useful tool to study the functions of GPR84 and a potential candidate for further structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjdl6EN67bhbVg90H21EOLACvtfcHk0lgsa7UbYpYUug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWhu7fE&md5=8a2f4113fddcb35cdd39f99b9470e0c7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1124%2Fjpet.116.232033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.116.232033%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DX.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520novel%2520small%2520molecule%2520agonist%2520for%2520medium%2520chain%2520free%2520fatty%2520acid%2520receptor%2520GPR84%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2016%26volume%3D357%26spage%3D337%26epage%3D344%26doi%3D10.1124%2Fjpet.116.232033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, F. J.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of 2 alkylpyrimidine-4,6-diol and 6 alkylpyridine-2,4-diol as potent GPR84 agonists</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=579-583&author=Y.+Liuauthor=Q.+Zhangauthor=L.+H.+Chenauthor=H.+Yangauthor=W.+Luauthor=X.+Xieauthor=F.+J.+Nan&title=Design+and+synthesis+of+2+alkylpyrimidine-4%2C6-diol+and+6+alkylpyridine-2%2C4-diol+as+potent+GPR84+agonists&doi=10.1021%2Facsmedchemlett.6b00025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of 2-Alkylpyrimidine-4,6-diol and 6-Alkylpyridine-2,4-diol as Potent GPR84 Agonists</span></div><div class="casAuthors">Liu, Yang; Zhang, Qing; Chen, Lin-Hai; Yang, Hui; Lu, Wei; Xie, Xin; Nan, Fa-Jun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">579-583</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of alkylpyrimidine-4,6-diol derivs. were designed and synthesized as novel GPR84 agonists based on a high-throughput screening (HTS) hit I. 6-Nonylpyridine-2,4-diol was identified as the most potent agonist of GPR84 reported so far, with an EC50 of 0.189 nM.  These novel GPR84 agonists will provide valuable tools for the study of the physiol. functions of GPR84.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyEcxh2lX2Q7Vg90H21EOLACvtfcHk0lgsa7UbYpYUug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D&md5=52f50ffae4ab4bbfd79f9eeda0bb063b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00025%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DF.%2BJ.%26atitle%3DDesign%2520and%2520synthesis%2520of%25202%2520alkylpyrimidine-4%252C6-diol%2520and%25206%2520alkylpyridine-2%252C4-diol%2520as%2520potent%2520GPR84%2520agonists%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D579%26epage%3D583%26doi%3D10.1021%2Facsmedchemlett.6b00025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nose, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, T.</span></span> <span> </span><span class="NLM_article-title">Identification of surrogate ligands for orphan G protein-coupled receptors</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2003.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.lfs.2003.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=14607265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos12qtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2003&pages=367-377&author=S.+Takedaauthor=A.+Yamamotoauthor=T.+Okadaauthor=E.+Matsumuraauthor=E.+Noseauthor=K.+Kogureauthor=S.+Kojimaauthor=T.+Haga&title=Identification+of+surrogate+ligands+for+orphan+G+protein-coupled+receptors&doi=10.1016%2Fj.lfs.2003.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of surrogate ligands for orphan G protein-coupled receptors</span></div><div class="casAuthors">Takeda, Shigeki; Yamamoto, Atsushi; Okada, Tomoaki; Matsumura, Emiko; Nose, Emi; Kogure, Katsuya; Kojima, Shigen; Haga, Tatsuya</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">367-377</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">The authors prepd. fusion proteins with an α subunit of G protein Gi (Gi1α) of 26 orphan G protein-coupled receptors (GPCRs) and with Gsα of 10 orphan GPCRs, most of which had been identified from the human genome previously.  Ligands for these fusion proteins were screened from a library consisting of approx. 1000 authentic compds. by measuring their effect on [35S]GTPγS binding to membrane prepns. of insect Sf9 cells expressing these fusion proteins.  Eleven compds. were found to act as surrogate agonists for a GPCR-Gsα and four GPCR-Giα fusion proteins, a compd. as an inverse agonist for two GPCR-Gsα fusion proteins, and a compd. as an endogenous agonist for a GPCR-Giα fusion protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV6jhTXRA457Vg90H21EOLACvtfcHk0lgsa7UbYpYUug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos12qtLY%253D&md5=8869ec4d0d89b5aae40bc8c2090db09f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2003.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2003.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DMatsumura%26aufirst%3DE.%26aulast%3DNose%26aufirst%3DE.%26aulast%3DKogure%26aufirst%3DK.%26aulast%3DKojima%26aufirst%3DS.%26aulast%3DHaga%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520surrogate%2520ligands%2520for%2520orphan%2520G%2520protein-coupled%2520receptors%26jtitle%3DLife%2520Sci.%26date%3D2003%26volume%3D74%26spage%3D367%26epage%3D377%26doi%3D10.1016%2Fj.lfs.2003.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marques, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laflamme, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benassou, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cissokho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemette, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudreau, L.</span></span> <span> </span><span class="NLM_article-title">Low levels of 3,3′-diindolylmethane activate estrogen receptor α and induce proliferation of breast cancer cells in the absence of estradiol</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1186/1471-2407-14-524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1186%2F1471-2407-14-524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25048790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslamtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=524-533&author=M.+Marquesauthor=L.+Laflammeauthor=I.+Benassouauthor=C.+Cissokhoauthor=B.+Guillemetteauthor=L.+Gaudreau&title=Low+levels+of+3%2C3%E2%80%B2-diindolylmethane+activate+estrogen+receptor+%CE%B1+and+induce+proliferation+of+breast+cancer+cells+in+the+absence+of+estradiol&doi=10.1186%2F1471-2407-14-524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Low levels of 3,3'-diindolylmethane activate estrogen receptor α and induce proliferation of breast cancer cells in the absence of estradiol</span></div><div class="casAuthors">Marques, Maud; Laflamme, Liette; Benassou, Ines; Cissokho, Coumba; Guillemette, Benoit; Gaudreau, Luc</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">524/1-524/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: 3,3'-diindolylmethane (DIM) is an acid-catalyzed dimer of idole-3-carbinol (I3C), a phytochem. found in cruciferous vegetables that include broccoli, Brussels sprouts and cabbage.  DIM is an aryl hydrocarbon receptor (AhR) ligand and a potential anticancer agent, namely for the treatment of breast cancer.  It is also advertised as a compd. that regulates sex hormone homeostasis.  Methods: Here we make use of RNA expression assays coupled to Chromatin Immunopptn. (ChIP) in breast cancer cell lines to study the effect of DIM on estrogen signaling.  We further make use of growth assays, as well as fluorescence-activated cell sorting (FACS) assays, to monitor cell growth.  Results: In this study, we report that 'physiol. obtainable' concns. of DIM (10 μM) activate the estrogen receptor α (ERα) signaling pathway in the human breast cancer cell lines MCF7 and T47D, in a 17β-estradiol (E2)-independent manner.  Accordingly, we observe induction of ERα target genes such as GREB1 and TFF1, and an increase in cellular proliferation after treatment with 10 μM DIM in the absence of E2.  By using an ERα specific inhibitor (ICI 182 780), we confirm that the transcriptional and proliferative effects of DIM treatment are mediated by ERα.  We further show that the protein kinase A signaling pathway participates in DIM-mediated activation of ERα.  In contrast, higher concns. of DIM (e.g. 50 μM) have an opposite and expected effect on cells, which is to inhibit proliferation.  Conclusions: We document an unexpected effect of DIM on cell proliferation, which is to stimulate growth by inducing the ERα signaling pathway.  Importantly, this proliferative effect of DIM happens with potentially physiol. concns. that can be provided by the diet or by taking caplet supplements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6MbRkbnyZv7Vg90H21EOLACvtfcHk0lhHtqKrVl_EBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslamtb7L&md5=a47dd52ea7387591cbcc7887e1d9e594</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-524%26sid%3Dliteratum%253Aachs%26aulast%3DMarques%26aufirst%3DM.%26aulast%3DLaflamme%26aufirst%3DL.%26aulast%3DBenassou%26aufirst%3DI.%26aulast%3DCissokho%26aufirst%3DC.%26aulast%3DGuillemette%26aufirst%3DB.%26aulast%3DGaudreau%26aufirst%3DL.%26atitle%3DLow%2520levels%2520of%25203%252C3%25E2%2580%25B2-diindolylmethane%2520activate%2520estrogen%2520receptor%2520%25CE%25B1%2520and%2520induce%2520proliferation%2520of%2520breast%2520cancer%2520cells%2520in%2520the%2520absence%2520of%2520estradiol%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D524%26epage%3D533%26doi%3D10.1186%2F1471-2407-14-524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span> <span> </span><span class="NLM_article-title">A selective aryl hydrocarbon receptor modulator 3,3′-diindolylmethane inhibits gastric cancer cell growth</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">46</span>, <span class="refDoi"> DOI: 10.1186/1756-9966-31-46</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1186%2F1756-9966-31-46" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=22592002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCqsL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=46&author=X.+F.+Yinauthor=J.+Chenauthor=W.+Maoauthor=Y.+H.+Wangauthor=M.+H.+Chen&title=A+selective+aryl+hydrocarbon+receptor+modulator+3%2C3%E2%80%B2-diindolylmethane+inhibits+gastric+cancer+cell+growth&doi=10.1186%2F1756-9966-31-46"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A selective aryl hydrocarbon receptor modulator 3,3'-Diindolylmethane inhibits gastric cancer cell growth</span></div><div class="casAuthors">Yin, Xiao-Fei; Chen, Jie; Mao, Wei; Wang, Yu-Hong; Chen, Min-Hu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Aryl hydrocarbon receptor (AhR) is a ligand-activated transcription factor assocd. with gastric carcinogenesis.  3,3'-Diindolylmethane (DIM) is a relatively non-toxic selective AhR modulator.  This study was to detect the effects of DIM on gastric cancer cell growth.  Methods: Gastric cancer cell SGC7901 was treated with DIM at different concns. (0, 10, 20, 30, 40, 50 μmol/L) with or without an AhR antagonist, resveratrol.  The expression of AhR and Cytochrome P 4501A1 (CYP1A1), a classic target gene of AhR pathway, were detected by RT-PCR and Western blot; cell viability was measured by MTT assay, and the changes in cell cycle and apoptosis were analyzed by flow cytometry.  Results: RT-PCR and western-blot showed that with the increase of the concn. of DIM, AhR protein gradually decreased and CYP1A1 expression increased, suggesting that DIM activated the AhR pathway and caused the translocation of AhR from cytoplasm to nucleus.  MTT assay indicated that the viability of SGC7901 cells was significantly decreased in a concn.- and time-dependent manner after DIM treatment and this could be partially reversed by resveratrol.  Flow cytometry anal. showed that DIM arrested cell cycle in G1 phase and induced cell apoptosis.  Conclusion: Selective aryl hydrocarbon receptor modulator 3,3'-Diindolylmethane inhibits SGC7901 cell proliferation by inducing apoptosis and delaying cell cycle progression.  AhR may be a potential therapeutic target for gastric cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLSqncAyaWjrVg90H21EOLACvtfcHk0lhHtqKrVl_EBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCqsL3E&md5=87cba16be653452c8d9043156f2b800b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1756-9966-31-46&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-9966-31-46%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DX.%2BF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DM.%2BH.%26atitle%3DA%2520selective%2520aryl%2520hydrocarbon%2520receptor%2520modulator%25203%252C3%25E2%2580%25B2-diindolylmethane%2520inhibits%2520gastric%2520cancer%2520cell%2520growth%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D31%26spage%3D46%26doi%3D10.1186%2F1756-9966-31-46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mancini, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmud, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edye, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span> <span> </span><span class="NLM_article-title">On-target and off-target effects of novel orthosteric and allosteric activators of GPR84</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1861</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-38539-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fs41598-019-38539-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=30755705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BB3cfis1amsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1861&author=S.+J.+Manciniauthor=Z.+A.+Mahmudauthor=L.+Jenkinsauthor=D.+Bologniniauthor=R.+Newmanauthor=M.+Barnesauthor=M.+E.+Edyeauthor=S.+B.+McMahonauthor=A.+B.+Tobinauthor=G.+Milligan&title=On-target+and+off-target+effects+of+novel+orthosteric+and+allosteric+activators+of+GPR84&doi=10.1038%2Fs41598-019-38539-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">On-target and off-target effects of novel orthosteric and allosteric activators of GPR84</span></div><div class="casAuthors">Mancini Sarah J; Mahmud Zobaer Al; Jenkins Laura; Bolognini Daniele; Tobin Andrew B; Milligan Graeme; Newman Robert; Barnes Matt; Edye Michelle E; McMahon Stephen B</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1861</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Many members of the G protein-coupled receptor family, including examples with clear therapeutic potential, remain poorly characterised.  This often reflects limited availability of suitable tool ligands with which to interrogate receptor function.  In the case of GPR84, currently a target for the treatment of idiopathic pulmonary fibrosis, recent times have seen the description of novel orthosteric and allosteric agonists.  Using 2-(hexylthiol)pyrimidine-4,6 diol (2-HTP) and di(5,7-difluoro-1H-indole-3-yl)methane (PSB-16671) as exemplars of each class, in cell lines transfected to express either human or mouse GPR84, both ligands acted as effective on-target activators and with high co-operativity in their interactions.  This was also the case in lipopolysaccharide-activated model human and mouse immune cell lines.  However in mouse bone-marrow-derived neutrophils, where expression of GPR84 is particularly high, the capacity of PSB-16671 but not of 2-HTP to promote G protein activation was predominantly off-target because it was not blocked by an antagonist of GPR84 and was preserved in neutrophils isolated from GPR84 deficient mice.  These results illustrate the challenges of attempting to study and define functions of poorly characterised receptors using ligands that have been developed via medicinal chemistry programmes, but where assessed activity has been limited largely to the initially identified target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQc62oWEy_4Oqrn-c4Of3KfW6udTcc2eZf23oITF-Xjrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfis1amsA%253D%253D&md5=1bd20c2443890cb3691d558cb8511c5f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-38539-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-38539-1%26sid%3Dliteratum%253Aachs%26aulast%3DMancini%26aufirst%3DS.%2BJ.%26aulast%3DMahmud%26aufirst%3DZ.%2BA.%26aulast%3DJenkins%26aufirst%3DL.%26aulast%3DBolognini%26aufirst%3DD.%26aulast%3DNewman%26aufirst%3DR.%26aulast%3DBarnes%26aufirst%3DM.%26aulast%3DEdye%26aufirst%3DM.%2BE.%26aulast%3DMcMahon%26aufirst%3DS.%2BB.%26aulast%3DTobin%26aufirst%3DA.%2BB.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DOn-target%2520and%2520off-target%2520effects%2520of%2520novel%2520orthosteric%2520and%2520allosteric%2520activators%2520of%2520GPR84%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D1861%26doi%3D10.1038%2Fs41598-019-38539-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labeguere, F.</span>; <span class="NLM_string-name">Luke, A.</span>; <span class="NLM_string-name">Gregory, N.</span>; <span class="NLM_string-name">Laurent, S.</span>; <span class="NLM_string-name">Stephen, F.</span></span> <span> </span><span class="NLM_article-title">Novel Dihydropyrimidinoisoquinolinones and Pharmaceutical Compositions Thereof for the Treatment of Inflammatory Disorders</span>. <span class="NLM_patent">WO2013092791A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=F.+Labeguere&author=A.+Luke&author=N.+Gregory&author=S.+Laurent&author=F.+Stephen&title=Novel+Dihydropyrimidinoisoquinolinones+and+Pharmaceutical+Compositions+Thereof+for+the+Treatment+of+Inflammatory+Disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLabeguere%26aufirst%3DF.%26atitle%3DNovel%2520Dihydropyrimidinoisoquinolinones%2520and%2520Pharmaceutical%2520Compositions%2520Thereof%2520for%2520the%2520Treatment%2520of%2520Inflammatory%2520Disorders%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niedernberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tunaru, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaukat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span> <span> </span><span class="NLM_article-title">Comparative analysis of functional assays for characterization of agonist ligands at G protein-coupled receptors</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1177/1087057103257555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1177%2F1087057103257555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=14567777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptVCiu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=500-510&author=A.+Niedernbergauthor=S.+Tunaruauthor=A.+Blaukatauthor=B.+Harrisauthor=E.+Kostenis&title=Comparative+analysis+of+functional+assays+for+characterization+of+agonist+ligands+at+G+protein-coupled+receptors&doi=10.1177%2F1087057103257555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative analysis of functional assays for characterization of agonist ligands at G protein-coupled receptors</span></div><div class="casAuthors">Niedernberg, Anke; Tunaru, Sorin; Blaukat, Andree; Harris, Bruce; Kostenis, Evi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">500-510</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A variety of functional assays are available for agonist or antagonist screening of G protein-coupled receptors (GPCRs), but it is a priori not predictable which assay is the most suitable to identify agonists or antagonists of GPCRs with therapeutic value in humans.  More specifically, it is not known how a given set of GPCR agonists compares in different functional assays with respect to potency and efficacy and whether the level of the signaling cascade that is analyzed has any impact on the detection of agonistic responses.  To address this question, the authors used the recently cloned human S1P5 receptor as a model and compared a set of 3 lipid ligands (sphingosine 1-phosphate [SIP], dihydro sphingosine 1-phosphate [dhS1P], and sphingosine) in 5 different functional assays: GTPγS binding, inhibition of adenylyl cyclase activity, mobilization of intracellular Ca2+ via the FLIPR and aequorin technol., and MAP kinase (ERK1/2) activation.  S1P induced agonistic responses in all except the ERK1/2 assays with EC50 values varying by a factor of 10.  Whereas dhS1P was identified as a partial agonist in the GTPγS assay, it behaved as a full agonist in all other settings.  Sphingosine displayed partial agonistic activity exclusively in GTPγS binding assays.  The findings suggest that assays in a given cellular background may vary significantly with respect to suitability for agonist finding and that ligands producing a response may not readily be detectable in all agonist assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvk-TaAKQ0JbVg90H21EOLACvtfcHk0lgSOIlDVxHJIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptVCiu7k%253D&md5=4fdb34d780da43882331e721381fa249</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F1087057103257555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057103257555%26sid%3Dliteratum%253Aachs%26aulast%3DNiedernberg%26aufirst%3DA.%26aulast%3DTunaru%26aufirst%3DS.%26aulast%3DBlaukat%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DB.%26aulast%3DKostenis%26aufirst%3DE.%26atitle%3DComparative%2520analysis%2520of%2520functional%2520assays%2520for%2520characterization%2520of%2520agonist%2520ligands%2520at%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D500%26epage%3D510%26doi%3D10.1177%2F1087057103257555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niedernberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaukat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schöneberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span> <span> </span><span class="NLM_article-title">Regulated and constitutive activation of specific signalling pathways by the human S1P5 receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">493</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0705055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fsj.bjp.0705055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=12569073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvFSrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2003&pages=481-493&author=A.+Niedernbergauthor=A.+Blaukatauthor=T.+Sch%C3%B6nebergauthor=E.+Kostenis&title=Regulated+and+constitutive+activation+of+specific+signalling+pathways+by+the+human+S1P5+receptor&doi=10.1038%2Fsj.bjp.0705055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Regulated and constitutive activation of specific signalling pathways by the human S1P5 receptor</span></div><div class="casAuthors">Niedernberg, Anke; Blaukat, Andree; Schoneberg, Torsten; Kostenis, Evi</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">481-493</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the hypothesis, whether G Protein-coupled receptors (GPCRs) may differentially regulate specific signalling pathways by constitutive and agonist-induced activation using the human sphingosine 1-phosphate receptor S1P5 as a model.  S1P5 receptor-expressing HEK293 cells exhibited a high degree of basal activity for both inhibition of adenylyl cyclase and extracellular signal regulated kinase (ERK) when cultured in serum, which contains high levels of sphingosine 1-phosphate (S1P).  However, basal activity was independent of the presence of S1P:.  (i) constitutive activity remained when cells were cultured in delipidated serum,.  (ii) addn. of S1P to delipidated serum did not increase basal S1P5 receptor signalling.  Conversely, constitutive inhibition of forskolin-stimulated adenylyl cyclase was further enhanced by S1P in S1P5-HEK293 cells.  Transfection of several mammalian cell lines (CHO-K1, HEK293, NIH-3T3, RH7777) with the S1P5 receptor induced cell rounding, which was more pronounced in the presence of S1P-contg. serum.  Rounded cell morphol. did not correlate with apoptotic cell death, but led to detachment of cells.  Cell surface ELISA assays showed that a fraction of plasma membrane S1P5 receptors were dose-dependently internalized with S1P.  These data reveal that intrinsic inhibition of unstimulated adenylyl cyclase or ERK activity by the S1P5 receptor is insensitive to ligand modulation.  Conversely, effects on forskolin-stimulated adenylyl cyclase, cell morphol. and internalization can be further augmented with S1P.  Our results suggest that different signal transduction pathways are not equally activated through constitutively active GPCRs with promiscuous signalling characteristics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouj12-uzY0GrVg90H21EOLACvtfcHk0lgSOIlDVxHJIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvFSrur0%253D&md5=5c9dbb5f868009c8054451c8529cf282</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705055%26sid%3Dliteratum%253Aachs%26aulast%3DNiedernberg%26aufirst%3DA.%26aulast%3DBlaukat%26aufirst%3DA.%26aulast%3DSch%25C3%25B6neberg%26aufirst%3DT.%26aulast%3DKostenis%26aufirst%3DE.%26atitle%3DRegulated%2520and%2520constitutive%2520activation%2520of%2520specific%2520signalling%2520pathways%2520by%2520the%2520human%2520S1P5%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D138%26spage%3D481%26epage%3D493%26doi%3D10.1038%2Fsj.bjp.0705055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitsunobu, O.</span></span> <span> </span><span class="NLM_article-title">The use of diethyl azodicarboxylate and triphenylphosphine in synthesis and transformation of natural products</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>1981</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1055/s-1981-29317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1055%2Fs-1981-29317" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1981&publication_year=1981&pages=1-28&author=O.+Mitsunobu&title=The+use+of+diethyl+azodicarboxylate+and+triphenylphosphine+in+synthesis+and+transformation+of+natural+products&doi=10.1055%2Fs-1981-29317"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1055%2Fs-1981-29317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1981-29317%26sid%3Dliteratum%253Aachs%26aulast%3DMitsunobu%26aufirst%3DO.%26atitle%3DThe%2520use%2520of%2520diethyl%2520azodicarboxylate%2520and%2520triphenylphosphine%2520in%2520synthesis%2520and%2520transformation%2520of%2520natural%2520products%26jtitle%3DSynthesis%26date%3D1981%26volume%3D1981%26spage%3D1%26epage%3D28%26doi%3D10.1055%2Fs-1981-29317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chubukov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherzov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structural basis for allosteric regulation of GPCRs by sodium ions</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>337</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1126/science.1219218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1126%2Fscience.1219218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=22798613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=2012&pages=232-236&author=W.+Liuauthor=E.+Chunauthor=A.+A.+Thompsonauthor=P.+Chubukovauthor=F.+Xuauthor=V.+Katritchauthor=G.+W.+Hanauthor=C.+B.+Rothauthor=L.+H.+Heitmanauthor=A.+P.+IJzermanauthor=V.+Cherzovauthor=R.+C.+Stevens&title=Structural+basis+for+allosteric+regulation+of+GPCRs+by+sodium+ions&doi=10.1126%2Fscience.1219218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Allosteric Regulation of GPCRs by Sodium Ions</span></div><div class="casAuthors">Liu, Wei; Chun, Eugene; Thompson, Aaron A.; Chubukov, Pavel; Xu, Fei; Katritch, Vsevolod; Han, Gye Won; Roth, Christopher B.; Heitman, Laura H.; IJzerman, Adriaan P.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6091</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Pharmacol. responses of G protein-coupled receptors (GPCRs) can be fine-tuned by allosteric modulators.  Structural studies of such effects have been limited due to the medium resoln. of GPCR structures.  We reengineered the human A2A adenosine receptor by replacing its third intracellular loop with apocytochrome b562RIL and solved the structure at 1.8 angstrom resoln.  The high-resoln. structure allowed us to identify 57 ordered water mols. inside the receptor comprising three major clusters.  The central cluster harbors a putative sodium ion bound to the highly conserved aspartate residue Asp2.50.  Addnl., two cholesterols stabilize the conformation of helix VI, and one of 23 ordered lipids intercalates inside the ligand-binding pocket.  These high-resoln. details shed light on the potential role of structured water mols., sodium ions, and lipids/cholesterol in GPCR stabilization and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvcjQimGnIMLVg90H21EOLACvtfcHk0lgSOIlDVxHJIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D&md5=782b2235b76ef77b11b6456f8e4485e1</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1126%2Fscience.1219218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1219218%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DA.%2BA.%26aulast%3DChubukov%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DCherzov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520basis%2520for%2520allosteric%2520regulation%2520of%2520GPCRs%2520by%2520sodium%2520ions%26jtitle%3DScience%26date%3D2012%26volume%3D337%26spage%3D232%26epage%3D236%26doi%3D10.1126%2Fscience.1219218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">6-Bromo-8-(4-[<sup>3</sup>H]methoxy-benzamido)-4-oxo-4H-chromene-2-carboxylic acid ([<sup>3</sup>H]PSB-13253): a powerful tool for studying orphan G protein-coupled receptor GPR35</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7084</span>– <span class="NLM_lpage">7099</span>, <span class="refDoi"> DOI: 10.1021/jm4009373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4009373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOlsbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7084-7099&author=D.+Thimmauthor=M.+Funkeauthor=A.+Meyerauthor=C.+E.+M%C3%BCller&title=6-Bromo-8-%284-%5B3H%5Dmethoxy-benzamido%29-4-oxo-4H-chromene-2-carboxylic+acid+%28%5B3H%5DPSB-13253%29%3A+a+powerful+tool+for+studying+orphan+G+protein-coupled+receptor+GPR35&doi=10.1021%2Fjm4009373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">6-Bromo-8-(4-[3H]methoxybenzamido)-4-oxo-4H-chromene-2-carboxylic Acid: A Powerful Tool for Studying Orphan G Protein-Coupled Receptor GPR35</span></div><div class="casAuthors">Thimm, Dominik; Funke, Mario; Meyer, Anne; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7084-7099</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The potent and selective GPR35 agonist 6-bromo-8-(4-methoxybenzamido)-4-oxo-4H-chromene-2-carboxylic acid was obtained in tritium-labeled form, designated [3H]PSB-13253 (I), with a specific activity of 36 Ci (1.33 TBq)/mmol.  Radiolabeling was achieved by methylation of Et 6-bromo-8-(4-((tert-butyldimethylsilyl)oxy)benzamido)-4-oxo-4H-chromene-2-carboxylate with [3H]methyl tosylate followed by ester hydrolysis.  The radioligand was characterized by kinetic, satn., and competition assays at membrane prepns. of Chinese hamster ovary cells recombinantly expressing the human GPR35.  I labeled the receptor with high affinity (KD = 5.27 nM).  Binding was saturable (Bmax = 12.6 pmol/mg of protein) and reversible.  Affinities of selected std. ligands and a library of amidochromen-4-one-2-carboxylates were detd.  Binding data mostly correlated with potencies detd. in β-arrestin assays.  On the basis of the test results, several new fluorine-substituted 6-bromo-8-benzamidochromen-4-one-2-carboxylic acids were obtained, which represent the most potent GPR35 agonists known to date.  6-Bromo-8-(2,6-difluoro-4-methoxybenzamido)-4-oxo-4H-chromene-2-carboxylic acid (II; Ki = 0.589 nM, EC50 = 5.54 nM) showed the highest affinity with a Ki value in the subnanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodEoburTr-0rVg90H21EOLACvtfcHk0lh4izkrATwxyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOlsbfN&md5=031f866ea6e2c5b40cc2c3977e58dc75</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm4009373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4009373%26sid%3Dliteratum%253Aachs%26aulast%3DThimm%26aufirst%3DD.%26aulast%3DFunke%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3D6-Bromo-8-%25284-%255B3H%255Dmethoxy-benzamido%2529-4-oxo-4H-chromene-2-carboxylic%2520acid%2520%2528%255B3H%255DPSB-13253%2529%253A%2520a%2520powerful%2520tool%2520for%2520studying%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR35%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7084%26epage%3D7099%26doi%3D10.1021%2Fjm4009373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiemke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Development of [(<sup>3</sup>)H]2-carboxy-4,6-dichloro-1<i>H</i>-indole-3-propionic acid ([(<sup>3</sup>)H]PSB-12150): a useful tool for studying GPR17</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1021/ml400399f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400399f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC2cjpvVOjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=326-330&author=M.+K%C3%B6seauthor=K.+Ritterauthor=K.+Thiemkeauthor=M.+Gillardauthor=E.+Kostenisauthor=C.+E.+M%C3%BCller&title=Development+of+%5B%283%29H%5D2-carboxy-4%2C6-dichloro-1H-indole-3-propionic+acid+%28%5B%283%29H%5DPSB-12150%29%3A+a+useful+tool+for+studying+GPR17&doi=10.1021%2Fml400399f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of [(3)H]2-Carboxy-4,6-dichloro-1H-indole-3-propionic Acid ([(3)H]PSB-12150): A Useful Tool for Studying GPR17</span></div><div class="casAuthors">Kose Meryem; Ritter Kirsten; Thiemke Katharina; Muller Christa E; Gillard Michel; Kostenis Evi</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">326-30</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">The recently described synthetic GPR17 agonist 2-carboxy-4,6-dichloro-1H-indole-3-propionic acid (1) was prepared in tritium-labeled form by catalytic hydrogenation of the corresponding propenoic acid derivative 8 with tritium gas.  The radioligand [(3)H]PSB-12150 (9) was obtained with a specific activity of 17 Ci/mmol (629 GBq/mmol).  It showed specific and saturable binding to a single binding site in membrane preparations from Chinese hamster ovary cells recombinantly expressing the human GPR17.  A competition assay procedure was established, which allows the determination of ligand binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS__t-5LBkU-yEJCPKhJPuqfW6udTcc2eYKdS48anwyCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjpvVOjtw%253D%253D&md5=6feadf019982496e6e520e10beeb31db</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fml400399f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400399f%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DRitter%26aufirst%3DK.%26aulast%3DThiemke%26aufirst%3DK.%26aulast%3DGillard%26aufirst%3DM.%26aulast%3DKostenis%26aufirst%3DE.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DDevelopment%2520of%2520%255B%25283%2529H%255D2-carboxy-4%252C6-dichloro-1H-indole-3-propionic%2520acid%2520%2528%255B%25283%2529H%255DPSB-12150%2529%253A%2520a%2520useful%2520tool%2520for%2520studying%2520GPR17%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D326%26epage%3D330%26doi%3D10.1021%2Fml400399f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
der Westhuizen, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Endogenous allosteric modulators of G protein-coupled receptors</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1124/jpet.114.221606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1124%2Fjpet.114.221606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25650376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVWntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2015&pages=246-260&author=E.+T.+Van%0Ader+Westhuizenauthor=C.+Valantauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=Endogenous+allosteric+modulators+of+G+protein-coupled+receptors&doi=10.1124%2Fjpet.114.221606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous allosteric modulators of G protein-coupled receptors</span></div><div class="casAuthors">van der Westhuizen, Emma T.; Valant, Celine; Sexton, Patrick M.; Christopoulos, Christopoulos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">246-260</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) are the largest superfamily of receptors encoded by the human genome, and represent the largest class of current drug targets.  Over the last decade and a half, it has become widely accepted that most, if not all, GPCRs possess spatially distinct allosteric sites that can be targeted by exogenous substances to modulate the receptors' biol. state.  Although many of these allosteric sites are likely to serve other (e.g., structural) roles, they nonetheless possess appropriate properties to be serendipitously targeted by synthetic mols.  However, there are also examples of endogenous substances that can act as allosteric modulators of GPCRs.  These include not only the obvious example, i.e., the G protein, but also a variety of ions, lipids, amino acids, peptides, and accessory proteins that display different degrees of receptor-specific modulatory effects.  This also suggests that some GPCRs may possess true "orphan" allosteric sites for hitherto unappreciated endogenous modulators.  Of note, the increasing identification of allosteric modulator lipids, inflammatory peptides, and GPCR-targeted autoantibodies indicates that disease context plays an important role in the generation of putative endogenous GPCR modulators.  If an endogenous allosteric substance can be shown to play a role in disease, this could also serve as an impetus to pursue synthetic neutral allosteric ligands as novel therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdf28zfijQrVg90H21EOLACvtfcHk0lh4izkrATwxyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVWntb4%253D&md5=4e9eacb10df4523109feba729fbb9409</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1124%2Fjpet.114.221606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.114.221606%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BWesthuizen%26aufirst%3DE.%2BT.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DEndogenous%2520allosteric%2520modulators%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2015%26volume%3D353%26spage%3D246%26epage%3D260%26doi%3D10.1124%2Fjpet.114.221606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shrestha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uemura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, H.</span></span> <span> </span><span class="NLM_article-title">Plasma capric acid concentrations in healthy subjects determined by high-performance liquid chromatography</span>. <i>Ann. Clin. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1177/0004563215569081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1177%2F0004563215569081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25587197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvVKqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2015&pages=588-596&author=R.+Shresthaauthor=S.+P.+Huiauthor=H.+Imaiauthor=S.+Hashimotoauthor=N.+Uemuraauthor=S.+Takedaauthor=H.+Fudaauthor=A.+Suzukiauthor=S.+Yamaguchiauthor=K.+Hiranoauthor=H.+Chiba&title=Plasma+capric+acid+concentrations+in+healthy+subjects+determined+by+high-performance+liquid+chromatography&doi=10.1177%2F0004563215569081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Plasma capric acid concentrations in healthy subjects determined by high-performance liquid chromatography</span></div><div class="casAuthors">Shrestha, Rojeet; Hui, Shu-Ping; Imai, Hiromitsu; Hashimoto, Satoru; Uemura, Naoto; Takeda, Seiji; Fuda, Hirotoshi; Suzuki, Akira; Yamaguchi, Satoshi; Hirano, Ken-ichi; Chiba, Hitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Clinical Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">588-596</span>CODEN:
                <span class="NLM_cas:coden">ACBOBU</span>;
        ISSN:<span class="NLM_cas:issn">0004-5632</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Background: Capric acid (FA10:0, decanoic acid) is a medium-chain fatty acid abundant in tropical oils such as coconut oil, whereas small amts. are present in milk of goat, cow, and human.  Orally ingested FA10:0 is transported to the liver and quickly burnt within it.  Only few reports are available for FA10:0 concns. in human plasma.  Methods: Fasting (n = 5, male/female = 3/2, age 31 ± 9.3 years old) and non-fasting (n = 106, male/female = 44/62, age 21.9 ± 3.2 years old) blood samples were collected from apparently healthy Japanese volunteers.  The total FA10:0 in the plasma were measured by high-performance liq. chromatog. after derivatization with 2-nitrophenylhydrazine followed by UV detection.  Results: Inter and intra-assay coeff. of variation of FA10:0 assay at three different concns. ranged in 1.7-3.9 and 1.3-5.4%, resp., with an anal. recovery of 95.2-104.0%.  FA10:0 concn. was below detection limit (0.1 μmol/L) in each fasting human plasma.  FA10:0 was not detected in 50 (47.2%) of 106 non-fasting blood samples, while 29 (27.4%) plasma samples contained FA10:0 less than or equal to 0.5 μmol/L (0.4 ± 0.1), and 27 (25.5%) contained it at more than 0.5 μmol/L (0.9 ± 0.3).  Conclusion: A half of the non-fasting plasma samples contained detectable FA10:0.  This simple, precise, and accurate high-performance liq. chromatog. method might be useful for monitoring plasma FA10:0 during medium-chain triglycerides therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHytyLwtn8SbVg90H21EOLACvtfcHk0lgv6bLRPCg6Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvVKqtb8%253D&md5=acd27812308c7ea149a50325fb6bf80b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1177%2F0004563215569081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0004563215569081%26sid%3Dliteratum%253Aachs%26aulast%3DShrestha%26aufirst%3DR.%26aulast%3DHui%26aufirst%3DS.%2BP.%26aulast%3DImai%26aufirst%3DH.%26aulast%3DHashimoto%26aufirst%3DS.%26aulast%3DUemura%26aufirst%3DN.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DFuda%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DYamaguchi%26aufirst%3DS.%26aulast%3DHirano%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DH.%26atitle%3DPlasma%2520capric%2520acid%2520concentrations%2520in%2520healthy%2520subjects%2520determined%2520by%2520high-performance%2520liquid%2520chromatography%26jtitle%3DAnn.%2520Clin.%2520Biochem.%26date%3D2015%26volume%3D52%26spage%3D588%26epage%3D596%26doi%3D10.1177%2F0004563215569081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altschul, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gish, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipman, D. J.</span></span> <span> </span><span class="NLM_article-title">Basic local alignment search tool</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">403</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(05)80360-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2FS0022-2836%2805%2980360-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=2231712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=1990&pages=403-410&author=S.+F.+Altschulauthor=W.+Gishauthor=W.+Millerauthor=E.+W.+Myersauthor=D.+J.+Lipman&title=Basic+local+alignment+search+tool&doi=10.1016%2FS0022-2836%2805%2980360-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Basic local alignment search tool</span></div><div class="casAuthors">Altschul, Stephen F.; Gish, Warren; Miller, Webb; Myers, Eugene W.; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">403-10</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A new approach to rapid sequence comparison, basic local alignment search tool (BLAST), directly approximates alignments that optimize a measure of local similarity, the maximal segment pair (MSP) score.  Recent math. results on the stochastic properties of MSP scores allow an anal. of the performance of this method as well as the statistical significance of alignments it generates.  The basic algorithm is simple and robust; it can be implemented in a no. of ways and applied in a variety of contexts including straightforward DNA and protein sequence database searches, motif searches, gene identification searches, and in the anal. of multiple regions of similarity in long DNA sequences.  In addn. to its flexibility and tractability to math. anal., BLAST is an order of magnitude faster than existing sequence comparison tools of comparable sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGEHw-q53MfLVg90H21EOLACvtfcHk0lgv6bLRPCg6Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D&md5=009d2323eb82f0549356880e1101db16</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980360-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980360-2%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DGish%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DMyers%26aufirst%3DE.%2BW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DBasic%2520local%2520alignment%2520search%2520tool%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1990%26volume%3D215%26spage%3D403%26epage%3D410%26doi%3D10.1016%2FS0022-2836%2805%2980360-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure of the human dopamine D3 receptor in complex with a D2/D3 selective antagonist</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span>,  <span class="NLM_fpage">1091</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1126/science.1197410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1126%2Fscience.1197410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=21097933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=2010&pages=1091-1095&author=E.+Y.+T.+Chienauthor=W.+Liuauthor=Q.+Zhaoauthor=V.+Katritchauthor=G.+W.+Hanauthor=M.+A.+Hansonauthor=L.+Shiauthor=A.+H.+Newmanauthor=J.+A.+Javitchauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Structure+of+the+human+dopamine+D3+receptor+in+complex+with+a+D2%2FD3+selective+antagonist&doi=10.1126%2Fscience.1197410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Human Dopamine D3 Receptor in Complex with a D2/D3 Selective Antagonist</span></div><div class="casAuthors">Chien, Ellen Y. T.; Liu, Wei; Zhao, Qiang; Katritch, Vsevolod; Won Han, Gye; Hanson, Michael A.; Shi, Lei; Newman, Amy Hauck; Javitch, Jonathan A.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6007</span>),
    <span class="NLM_cas:pages">1091-1095</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Dopamine modulates movement, cognition, and emotion through activation of dopamine G protein-coupled receptors in the brain.  The crystal structure of the human dopamine D3 receptor (D3R) in complex with the small mol. D2R/D3R-specific antagonist eticlopride reveals important features of the ligand binding pocket and extracellular loops.  On the intracellular side of the receptor, a locked conformation of the ionic lock and two distinctly different conformations of intracellular loop 2 are obsd.  Docking of R-22, a D3R-selective antagonist, reveals an extracellular extension of the eticlopride binding site that comprises a second binding pocket for the aryl amide of R-22, which differs between the highly homologous D2R and D3R.  This difference provides direction to the design of D3R-selective agents for treating drug abuse and other neuropsychiatric indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOEcfSdjMm-7Vg90H21EOLACvtfcHk0lgv6bLRPCg6Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM&md5=12c6f343daea9fb14ca605a092f28533</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fscience.1197410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1197410%26sid%3Dliteratum%253Aachs%26aulast%3DChien%26aufirst%3DE.%2BY.%2BT.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520of%2520the%2520human%2520dopamine%2520D3%2520receptor%2520in%2520complex%2520with%2520a%2520D2%252FD3%2520selective%2520antagonist%26jtitle%3DScience%26date%3D2010%26volume%3D330%26spage%3D1091%26epage%3D1095%26doi%3D10.1126%2Fscience.1197410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tikhonova, I. G.</span></span> <span> </span><span class="NLM_article-title">Application of GPCR structures for modelling of free fatty acid receptors</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1007/164_2016_52</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1007%2F164_2016_52" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=2016&pages=57-77&author=I.+G.+Tikhonova&title=Application+of+GPCR+structures+for+modelling+of+free+fatty+acid+receptors&doi=10.1007%2F164_2016_52"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2F164_2016_52&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F164_2016_52%26sid%3Dliteratum%253Aachs%26aulast%3DTikhonova%26aufirst%3DI.%2BG.%26atitle%3DApplication%2520of%2520GPCR%2520structures%2520for%2520modelling%2520of%2520free%2520fatty%2520acid%2520receptors%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2016%26volume%3D236%26spage%3D57%26epage%3D77%26doi%3D10.1007%2F164_2016_52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramakrishnan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasisekharan, V.</span></span> <span> </span><span class="NLM_article-title">Stereochemistry of polypeptide chain configurations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(63)80023-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2FS0022-2836%2863%2980023-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=13990617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaF3sXks1arsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1963&pages=95-99&author=G.+N.+Ramachandranauthor=C.+Ramakrishnanauthor=V.+Sasisekharan&title=Stereochemistry+of+polypeptide+chain+configurations&doi=10.1016%2FS0022-2836%2863%2980023-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemistry of polypeptide chain configurations</span></div><div class="casAuthors">Ramachandran, G. N.; Ramakrishnan, C.; Sasisekharan, V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-9</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A convenient anal. method for writing the configurations of polypeptide chains is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpggsnuSQqKS7Vg90H21EOLACvtfcHk0liPxYO7mATCEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXks1arsLk%253D&md5=68298929fb718b888df17dff2275a55e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2863%2980023-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252863%252980023-6%26sid%3Dliteratum%253Aachs%26aulast%3DRamachandran%26aufirst%3DG.%2BN.%26aulast%3DRamakrishnan%26aufirst%3DC.%26aulast%3DSasisekharan%26aufirst%3DV.%26atitle%3DStereochemistry%2520of%2520polypeptide%2520chain%2520configurations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1963%26volume%3D7%26spage%3D95%26epage%3D99%26doi%3D10.1016%2FS0022-2836%2863%2980023-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Bakker, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Word, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure validation by Calpha geometry: phi, psi and Cbeta deviation</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">450</span>, <span class="refDoi"> DOI: 10.1002/prot.10286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fprot.10286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=12557186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2003&pages=437-450&author=S.+C.+Lovellauthor=I.+W.+Davisauthor=W.+B.+Arendallauthor=P.+I.+de%0ABakkerauthor=J.+M.+Wordauthor=M.+G.+Prisantauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=Structure+validation+by+Calpha+geometry%3A+phi%2C+psi+and+Cbeta+deviation&doi=10.1002%2Fprot.10286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure validation by Cα geometry: φ, ψ and Cβ deviation</span></div><div class="casAuthors">Lovell, Simon C.; Davis, Ian W.; Arendall, W. Bryan, III; de Bakker, Paul I. W.; Word, J. Michael; Prisant, Michael G.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-450</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Geometrical validation around the Cα is described, with a new Cβ measure and updated Ramachandran plot.  Deviation of the obsd. Cβ atom from ideal position provides a single measure encapsulating the major structure-validation information contained in bond angle distortions.  Cβ deviation is sensitive to incompatibilities between sidechain and backbone caused by misfit conformations or inappropriate refinement restraints.  A new .vphi.,ψ plot using d.-dependent smoothing for 81,234 non-Gly, non-Pro, and non-prePro residues with B < 30 from 500 high-resoln. proteins shows sharp boundaries at crit. edges and clear delineation between large empty areas and regions that are allowed but disfavored.  One such region is the γ-turn conformation near +75°, -60°, counted as forbidden by common structure-validation programs; however, it occurs in well-ordered parts of good structures, it is overrepresented near functional sites, and strain is partly compensated by the γ-turn H-bond.  Favored and allowed .vphi.,ψ regions are also defined for Pro, pre-Pro, and Gly (important because Gly .vphi.,ψ angles are more permissive but less accurately detd.).  Details of these accurate empirical distributions are poorly predicted by previous theor. calcns., including a region left of α-helix, which rates as favorable in energy yet rarely occurs.  A proposed factor explaining this discrepancy is that crowding of the two-peptide NHs permits donating only a single H-bond.  New calcns. by H. Hu et al. (2002) for Ala and Gly dipeptides, using mixed quantum mechanics and mol. mechanics, fit our nonrepetitive data in excellent detail.  To run our geometrical evaluations on a user-uploaded file, see MOLPROBITY (http://kinemage.biochem.duke.edu) or RAMPAGE (http://www-cryst.bioc.cam.ac.uk/rampage).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFFimJUYSPl7Vg90H21EOLACvtfcHk0liPxYO7mATCEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D&md5=e9d41070d9e8832ef9aedf6a523dfd73</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fprot.10286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10286%26sid%3Dliteratum%253Aachs%26aulast%3DLovell%26aufirst%3DS.%2BC.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3Dde%2BBakker%26aufirst%3DP.%2BI.%26aulast%3DWord%26aufirst%3DJ.%2BM.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DStructure%2520validation%2520by%2520Calpha%2520geometry%253A%2520phi%252C%2520psi%2520and%2520Cbeta%2520deviation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D50%26spage%3D437%26epage%3D450%26doi%3D10.1002%2Fprot.10286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiederstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, M. J.</span></span> <span> </span><span class="NLM_article-title">ProSA-web: Interactive web service for the recognition of errors in three-dimensional structures of proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W407</span>– <span class="NLM_lpage">W410</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1093%2Fnar%2Fgkm290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=17517781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W407-W410&author=M.+Wiedersteinauthor=M.+J.+Sippl&title=ProSA-web%3A+Interactive+web+service+for+the+recognition+of+errors+in+three-dimensional+structures+of+proteins&doi=10.1093%2Fnar%2Fgkm290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins</span></div><div class="casAuthors">Wiederstein Markus; Sippl Manfred J</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W407-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A major problem in structural biology is the recognition of errors in experimental and theoretical models of protein structures.  The ProSA program (Protein Structure Analysis) is an established tool which has a large user base and is frequently employed in the refinement and validation of experimental protein structures and in structure prediction and modeling.  The analysis of protein structures is generally a difficult and cumbersome exercise.  The new service presented here is a straightforward and easy to use extension of the classic ProSA program which exploits the advantages of interactive web-based applications for the display of scores and energy plots that highlight potential problems spotted in protein structures.  In particular, the quality scores of a protein are displayed in the context of all known protein structures and problematic parts of a structure are shown and highlighted in a 3D molecule viewer.  The service specifically addresses the needs encountered in the validation of protein structures obtained from X-ray analysis, NMR spectroscopy and theoretical calculations.  ProSA-web is accessible at https://prosa.services.came.sbg.ac.at.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5oR8RchjrrfW6udTcc2eaOgWVpPHTRiLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D&md5=e2aba9fe44147a9048f37c77306edea6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm290%26sid%3Dliteratum%253Aachs%26aulast%3DWiederstein%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DM.%2BJ.%26atitle%3DProSA-web%253A%2520Interactive%2520web%2520service%2520for%2520the%2520recognition%2520of%2520errors%2520in%2520three-dimensional%2520structures%2520of%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW407%26epage%3DW410%26doi%3D10.1093%2Fnar%2Fgkm290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span> <i>Molecular Operating Environment (MOE)</i>, 2014.09, <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>, <span class="NLM_publisher-loc">Montreal,
QC, Canada</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+2014.09%2C+Chemical+Computing+Group+Inc.%2C+Montreal%2C%0AQC%2C+Canada%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Więcek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligneau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schunack, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handzlik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieć-Kononowicz, K.</span></span> <span> </span><span class="NLM_article-title">N-Alkenyl and cycloalkyl carbamates as dual acting histamine H3 and H4 receptor ligands</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2850</span>– <span class="NLM_lpage">2858</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.03.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bmc.2011.03.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=21498080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFOrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=2850-2858&author=M.+Wi%C4%99cekauthor=T.+Kottkeauthor=X.+Ligneauauthor=W.+Schunackauthor=R.+Seifertauthor=H.+Starkauthor=J.+Handzlikauthor=K.+Kie%C4%87-Kononowicz&title=N-Alkenyl+and+cycloalkyl+carbamates+as+dual+acting+histamine+H3+and+H4+receptor+ligands&doi=10.1016%2Fj.bmc.2011.03.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">N-Alkenyl and cycloalkyl carbamates as dual acting histamine H3 and H4 receptor ligands</span></div><div class="casAuthors">Wiecek, Malgorzata; Kottke, Tim; Ligneau, Xavier; Schunack, Walter; Seifert, Roland; Stark, Holger; Handzlik, Jadwiga; Kiec-Kononowicz, Katarzyna</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2850-2858</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Previous studies have shown that several imidazole derivs. posses affinity to histamine H3 and H4 receptors.  Continuing our study on structural requirements responsible for affinity and selectivity for H3/H4 receptor subtypes, two series of 3-(1H-imidazol-4-yl)propyl carbamates were prepd.: a series of unsatd. alkyl derivs. (1-9) and a series possessing a cycloalkyl group different distances to the carbamate moiety (10-13).  The compds. were tested for their affinities at the human histamine H3 receptor, stably expressed in CHO-K1 cells.  Compds. 1, 2, 5-7, 10-13 were investigated for their affinities at the human histamine H4 receptor co-expressed with Gαi2 and Gβ1γ2 subunits in Sf9 cells.  To expand the pharmacol. profile, compds. were further tested for their H3 receptor antagonist activity on guinea pig ileum and in vivo after oral administration to mice.  All tested compds. exhibited good affinity for the human histamine H3 receptor with K i values in the range from 14 to 194 nM.  All compds. were active in vivo after peroral administration (p.o.) to Swiss mice, thus demonstrating their ability to cross the blood-brain barrier.  The most potent H3 receptor ligand of these series was compd. 5, 3-(1H-imidazol-4-yl)propyl pent-4-enylcarbamate with the highest affinity (K i = 14 nM).  Addnl., compd. 3 showed remarkable central nervous system (CNS) H3R activity, increasing the N τ-methylhistamine levels in mice with an ED50 value of 0.55 mg/kg, p.o. evidencing therefore, a twofold increase of inverse agonist/antagonist potency compared to the ref. inverse agonist/antagonist thioperamide.  In this study, the imidazole propyloxy carbamate moiety was kept const.  The different lipophilic moieties connected to the carbamate functionality in the eastern part of the mol. had a range of influences on the human H4 receptor affinity (154-1326 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPAV8EqAjO5rVg90H21EOLACvtfcHk0liFkgwtvxQHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFOrsLw%253D&md5=9b79684afe28bbf9cd62ad96222a8040</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.03.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.03.046%26sid%3Dliteratum%253Aachs%26aulast%3DWi%25C4%2599cek%26aufirst%3DM.%26aulast%3DKottke%26aufirst%3DT.%26aulast%3DLigneau%26aufirst%3DX.%26aulast%3DSchunack%26aufirst%3DW.%26aulast%3DSeifert%26aufirst%3DR.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DHandzlik%26aufirst%3DJ.%26aulast%3DKie%25C4%2587-Kononowicz%26aufirst%3DK.%26atitle%3DN-Alkenyl%2520and%2520cycloalkyl%2520carbamates%2520as%2520dual%2520acting%2520histamine%2520H3%2520and%2520H4%2520receptor%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D2850%26epage%3D2858%26doi%3D10.1016%2Fj.bmc.2011.03.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goulet, J. L.</span>; <span class="NLM_string-name">Holmes, M. A.</span>; <span class="NLM_string-name">Hunt, J. A.</span>; <span class="NLM_string-name">Mills, S. G.</span>; <span class="NLM_string-name">Parsons, W. H.</span>; <span class="NLM_string-name">Sinclair, P. J.</span>; <span class="NLM_string-name">Zaller, D. M.</span></span> <span> </span><span class="NLM_article-title">Src Kinase Inhibitor Compounds</span>. <span class="NLM_patent">WO2001000207A1</span>, <span class="NLM_year">2001</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=J.+L.+Goulet&author=M.+A.+Holmes&author=J.+A.+Hunt&author=S.+G.+Mills&author=W.+H.+Parsons&author=P.+J.+Sinclair&author=D.+M.+Zaller&title=Src+Kinase+Inhibitor+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGoulet%26aufirst%3DJ.%2BL.%26atitle%3DSrc%2520Kinase%2520Inhibitor%2520Compounds%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novotna, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulova, M.</span></span> <span> </span><span class="NLM_article-title">Preparation and characterization of 2-alkylthio-4-(4-bromomethylbenzoyloxy)-6-methylpyrimidines</span>. <i>Chemical Papers</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">656</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaK38XlvFeisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1991&pages=651-656&author=Z.+Novotnaauthor=M.+Koosauthor=M.+Matulova&title=Preparation+and+characterization+of+2-alkylthio-4-%284-bromomethylbenzoyloxy%29-6-methylpyrimidines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation and characterization of 2-alkylthio-4-(4-bromomethylbenzoyloxy)-6-methylpyrimidines</span></div><div class="casAuthors">Novotna, Z.; Koos, M.; Matulova, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Papers</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">651-6</span>CODEN:
                <span class="NLM_cas:coden">CHPAEG</span>;
        ISSN:<span class="NLM_cas:issn">0366-6352</span>.
    </div><div class="casAbstract">By the reaction of alkylthiohydroxymethylpyrimidines I [R = CnH2n+1, n = 1 - 10] with 4-bromomethylbenzoyl chloride, new alkylthio(bromomethylbenzoyloxy)methylpyrimidines II were prepd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqybfruYCU7ULVg90H21EOLACvtfcHk0liFkgwtvxQHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlvFeisw%253D%253D&md5=694f6837d84a5de92b7ed29265accac8</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNovotna%26aufirst%3DZ.%26aulast%3DKoos%26aufirst%3DM.%26aulast%3DMatulova%26aufirst%3DM.%26atitle%3DPreparation%2520and%2520characterization%2520of%25202-alkylthio-4-%25284-bromomethylbenzoyloxy%2529-6-methylpyrimidines%26jtitle%3DChemical%2520Papers%26date%3D1991%26volume%3D45%26spage%3D651%26epage%3D656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span>; <span class="NLM_string-name">Ermert, P.</span>; <span class="NLM_string-name">Luther, A.</span>; <span class="NLM_string-name">Eberle, M.</span>; <span class="NLM_string-name">Bachmann, F.</span></span> <span> </span><span class="NLM_article-title">Preparation of 2,4,6-Trisubstituted Pyrimidine Derivatives as Apoptosis Inducers for the Treatment of Neoplastic and Autoimmune Diseases</span>. <span class="NLM_patent">WO2004087679A1</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=D.+Obrecht&author=P.+Ermert&author=A.+Luther&author=M.+Eberle&author=F.+Bachmann&title=Preparation+of+2%2C4%2C6-Trisubstituted+Pyrimidine+Derivatives+as+Apoptosis+Inducers+for+the+Treatment+of+Neoplastic+and+Autoimmune+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DPreparation%2520of%25202%252C4%252C6-Trisubstituted%2520Pyrimidine%2520Derivatives%2520as%2520Apoptosis%2520Inducers%2520for%2520the%2520Treatment%2520of%2520Neoplastic%2520and%2520Autoimmune%2520Diseases%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacher, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linnemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knolle, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Antiproliferative effects of selective adenosine receptor agonists and antagonists on human lymphocytes: evidence for receptor-independent mechanisms</span>. <i>Purinergic Signalling</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1007/s11302-013-9354-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1007%2Fs11302-013-9354-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23359122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2qsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=351-365&author=A.+C.+Schiedelauthor=S.+K.+Lacherauthor=C.+Linnemannauthor=P.+A.+Knolleauthor=C.+E.+M%C3%BCller&title=Antiproliferative+effects+of+selective+adenosine+receptor+agonists+and+antagonists+on+human+lymphocytes%3A+evidence+for+receptor-independent+mechanisms&doi=10.1007%2Fs11302-013-9354-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Antiproliferative effects of selective adenosine receptor agonists and antagonists on human lymphocytes: evidence for receptor-independent mechanisms</span></div><div class="casAuthors">Schiedel, Anke C.; Lacher, Svenja K.; Linnemann, Carsten; Knolle, Percy A.; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Purinergic Signalling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">351-365</span>CODEN:
                <span class="NLM_cas:coden">PSUIA9</span>;
        ISSN:<span class="NLM_cas:issn">1573-9538</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The effects of std. adenosine receptor (AR) agonists and antagonists on the proliferation of human T lymphocytes, unstimulated and phytohemagglutinin-stimulated human peripheral blood lymphocytes (PBL), and Jurkat T cells were investigated.  Real-time PCR measurements confirmed the presence of all four AR subtypes on the investigated cells, although at different expression levels.  A2A ARs were predominantly expressed in PBL and further upregulated upon stimulation, while malignant Jurkat T cells showed high expression levels of A1, A2A, and A2B ARs.  Cell proliferation was measured by [3H]-thymidine incorporation assays.  Several ligands, including the subtype-selective agonists CPA (A1), BAY60-6583 (A2B), and IB-MECA (A3), and the antagonists PSB-36 (A1), MSX-2 (A2A), and PSB-10 (A3) significantly inhibited cell proliferation at micromolar concns., which were about three orders of magnitude higher than their AR affinities.  In contrast, further investigated AR ligands, including the agonists NECA (nonselective) and CGS21680 (A2A), and the antagonists preladenant (SCH-420814, A2A), PSB-1115 (A2B), and PSB-603 (A2B) showed no or only minor effects on lymphocyte proliferation.  The anti-proliferative effects of the AR agonists could not be blocked by the corresponding antagonists.  The non-selective AR antagonist caffeine stimulated phytohemagglutinin-activated PBL with an EC50 value of 104 μM.  This is the first study to compare a complete set of commonly used AR ligands for all subtypes on lymphocyte proliferation.  Our results strongly suggest that these compds. induce an inhibition of lymphocyte proliferation and cell death through AR-independent mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHmT-RxH31BbVg90H21EOLACvtfcHk0liFkgwtvxQHQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2qsrzE&md5=b454b6b4ad9c9a0c226fc4f40c8bbb14</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs11302-013-9354-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11302-013-9354-7%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DLacher%26aufirst%3DS.%2BK.%26aulast%3DLinnemann%26aufirst%3DC.%26aulast%3DKnolle%26aufirst%3DP.%2BA.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DAntiproliferative%2520effects%2520of%2520selective%2520adenosine%2520receptor%2520agonists%2520and%2520antagonists%2520on%2520human%2520lymphocytes%253A%2520evidence%2520for%2520receptor-independent%2520mechanisms%26jtitle%3DPurinergic%2520Signalling%26date%3D2013%26volume%3D9%26spage%3D351%26epage%3D365%26doi%3D10.1007%2Fs11302-013-9354-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borroto-Escuela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ammon, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Filippo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danish, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacher, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Červinková, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafehi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuxe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Adenosine A<sub>2A</sub> receptor ligand recognition and signaling is blocked by A<sub>2B</sub> receptors</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">13593</span>– <span class="NLM_lpage">13611</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.18632%2Foncotarget.24423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29568380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC1MnktVaqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=13593-13611&author=S.+Hinzauthor=G.+Navarroauthor=D.+Borroto-Escuelaauthor=B.+F.+Seibtauthor=Y.+C.+Ammonauthor=E.+de+Filippoauthor=A.+Danishauthor=S.+K.+Lacherauthor=B.+%C4%8Cervinkov%C3%A1author=M.+Rafehiauthor=K.+Fuxeauthor=A.+C.+Schiedelauthor=R.+Francoauthor=C.+E.+M%C3%BCller&title=Adenosine+A2A+receptor+ligand+recognition+and+signaling+is+blocked+by+A2B+receptors&doi=10.18632%2Foncotarget.24423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine A2A receptor ligand recognition and signaling is blocked by A2B receptors</span></div><div class="casAuthors">Hinz Sonja; Seibt Benjamin F; Ammon York-Christoph; de Filippo Elisabetta; Danish Azeem; Lacher Svenja K; Cervinkova Barbora; Rafehi Muhammad; Schiedel Anke C; Muller Christa E; Navarro Gemma; Franco Rafael; Navarro Gemma; Franco Rafael; Borroto-Escuela Dasiel; Fuxe Kjell</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">13593-13611</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The adenosine receptor (AR) subtypes A2A and A2B are rhodopsin-like Gs protein-coupled receptors whose expression is highly regulated under pathological, e.g. hypoxic, ischemic and inflammatory conditions.  Both receptors play important roles in inflammatory and neurodegenerative diseases, are blocked by caffeine, and have now become major drug targets in immuno-oncology.  By Forster resonance energy transfer (FRET), bioluminescence resonance energy transfer (BRET), bimolecular fluorescence complementation (BiFC) and proximity ligation assays (PLA) we demonstrated A2A-A2BAR heteromeric complex formation.  Moreover we observed a dramatically altered pharmacology of the A2AAR when co-expressed with the A2BAR (A2B ≥ A2A) in recombinant as well as in native cells.  In the presence of A2BARs, A2A-selective ligands lost high affinity binding to A2AARs and displayed strongly reduced potency in cAMP accumulation and dynamic mass redistribution (DMR) assays.  These results have major implications for the use of A2AAR ligands as drugs as they will fail to modulate the receptor in an A2A-A2B heteromer context.  Accordingly, A2A-A2BAR heteromers represent novel pharmacological targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRatiF0vPAX31MGsG_HTXYFfW6udTcc2ea1v37m7AKVDLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnktVaqtg%253D%253D&md5=99d64eaa7ad0a0a688fda90a7d526b7a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24423%26sid%3Dliteratum%253Aachs%26aulast%3DHinz%26aufirst%3DS.%26aulast%3DNavarro%26aufirst%3DG.%26aulast%3DBorroto-Escuela%26aufirst%3DD.%26aulast%3DSeibt%26aufirst%3DB.%2BF.%26aulast%3DAmmon%26aufirst%3DY.%2BC.%26aulast%3Dde%2BFilippo%26aufirst%3DE.%26aulast%3DDanish%26aufirst%3DA.%26aulast%3DLacher%26aufirst%3DS.%2BK.%26aulast%3D%25C4%258Cervinkov%25C3%25A1%26aufirst%3DB.%26aulast%3DRafehi%26aufirst%3DM.%26aulast%3DFuxe%26aufirst%3DK.%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DFranco%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DAdenosine%2520A2A%2520receptor%2520ligand%2520recognition%2520and%2520signaling%2520is%2520blocked%2520by%2520A2B%2520receptors%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D13593%26epage%3D13611%26doi%3D10.18632%2Foncotarget.24423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karcz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiene, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heisig, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrenswerth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieć-Kononowicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Fluorescent-labeled selective adenosine A<sub>2B</sub> receptor antagonist enables competition binding assay by flow cytometry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4301</span>– <span class="NLM_lpage">4316</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjlt1Onsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4301-4316&author=M.+K%C3%B6seauthor=S.+Gollosauthor=T.+Karczauthor=A.+Fieneauthor=F.+Heisigauthor=A.+Behrenswerthauthor=K.+Kie%C4%87-Kononowiczauthor=V.+Namasivayamauthor=C.+E.+M%C3%BCller&title=Fluorescent-labeled+selective+adenosine+A2B+receptor+antagonist+enables+competition+binding+assay+by+flow+cytometry&doi=10.1021%2Facs.jmedchem.7b01627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent-Labeled Selective Adenosine A2B Receptor Antagonist Enables Competition Binding Assay by Flow Cytometry</span></div><div class="casAuthors">Kose Meryem; Gollos Sabrina; Fiene Amelie; Heisig Fabian; Behrenswerth Andrea; Namasivayam Vigneshwaran; Muller Christa E; Karcz Tadeusz; Kiec-Kononowicz Katarzyna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4301-4316</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fluorescent ligands represent powerful tools for biological studies and are considered attractive alternatives to radioligands.  In this study, we developed fluorescent antagonists for A2B adenosine receptors (A2BARs), which are targeted by antiasthmatic xanthines and were proposed as novel targets in immuno-oncology.  Our approach was to merge a small borondipyrromethene (BODIPY) derivative with the pharmacophore of 8-substituted xanthine derivatives.  On the basis of the design, synthesis, and evaluation of model compounds, several fluorescent ligands were synthesized.  Compound 29 (PSB-12105), which displayed high affinity for human, rat, and mouse A2BARs ( Ki = 0.2-2 nM) and high selectivity for this AR subtype, was selected for further studies.  A homology model of the human A2BAR was generated, and docking studies were performed.  Moreover, 29 allowed us to establish a homogeneous receptor-ligand binding assay using flow cytometry.  These compounds constitute the first potent, selective fluorescent A2BAR ligands and are anticipated to be useful for a variety of applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShJSPcTqEx0ZVVjGoY74BafW6udTcc2ea1v37m7AKVDLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjlt1Onsg%253D%253D&md5=9d595306619272562da897096f630b92</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01627%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DGollos%26aufirst%3DS.%26aulast%3DKarcz%26aufirst%3DT.%26aulast%3DFiene%26aufirst%3DA.%26aulast%3DHeisig%26aufirst%3DF.%26aulast%3DBehrenswerth%26aufirst%3DA.%26aulast%3DKie%25C4%2587-Kononowicz%26aufirst%3DK.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DFluorescent-labeled%2520selective%2520adenosine%2520A2B%2520receptor%2520antagonist%2520enables%2520competition%2520binding%2520assay%2520by%2520flow%2520cytometry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4301%26epage%3D4316%26doi%3D10.1021%2Facs.jmedchem.7b01627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdelrahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Tayeb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Ryck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Characterization of P2X4 receptor agonists and antagonists by calcium influx and radioligand binding studies</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2016.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bcp.2016.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=27867013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFSiu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=41-54&author=A.+Abdelrahmanauthor=V.+Namasivayamauthor=S.+Hinzauthor=A.+C.+Schiedelauthor=M.+K%C3%B6seauthor=M.+Burtonauthor=A.+El-Tayebauthor=M.+Gillardauthor=J.+Bajorathauthor=M.+de+Ryckauthor=C.+E.+M%C3%BCller&title=Characterization+of+P2X4+receptor+agonists+and+antagonists+by+calcium+influx+and+radioligand+binding+studies&doi=10.1016%2Fj.bcp.2016.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of P2X4 receptor agonists and antagonists by calcium influx and radioligand binding studies</span></div><div class="casAuthors">Abdelrahman, Aliaa; Namasivayam, Vigneshwaran; Hinz, Sonja; Schiedel, Anke C.; Koese, Meryem; Burton, Maggi; El-Tayeb, Ali; Gillard, Michel; Bajorath, Juergen; de Ryck, Marc; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-54</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antagonists for ATP-activated P2X4 ion channel receptors are currently in the focus as novel drug targets, in particular for the treatment of neuropathic and inflammatory pain.  We stably expressed the human, rat and mouse P2X4 receptors in 1321N1 astrocytoma cells, which is devoid of functional nucleotide receptors, by retroviral transfection, and established monoclonal cell lines.  Calcium flux assay conditions were optimized for high-throughput screening resulting in a Z'-factor of >0.8.  The application of ready-to-use frozen cells did not neg. affect the results of the calcium assays, which is of great advantage for the screening of compd. libraries.  Species differences were obsd., the rat P2X4 receptor being particularly insensitive to many ATP derivs.  Membrane prepns. of the cell lines showed high levels of specific [35S]ATPγS binding with low nonspecific binding (<5% of total binding), while non-transfected cells were devoid of specific binding sites for the radioligand.  Conditions were employed which allow binding studies to be performed at room temp.  While a variety of nucleotide-derived agonists and the antagonist TNP-ATP displaced [35S]ATPγS from its binding site at human P2X4 receptors, the non-nucleotidic antagonists paroxetine and 5-BDBD did not compete with radioligand binding and were therefore characterized as allosteric antagonists.  Homol. modeling was applied to find an explanation for the obsd. species differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2TlVF8aUxbVg90H21EOLACvtfcHk0lhjUYr9wAlDZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFSiu77F&md5=2229ea59f1e799653bd873720675a6b9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DAbdelrahman%26aufirst%3DA.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DHinz%26aufirst%3DS.%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DBurton%26aufirst%3DM.%26aulast%3DEl-Tayeb%26aufirst%3DA.%26aulast%3DGillard%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3Dde%2BRyck%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DCharacterization%2520of%2520P2X4%2520receptor%2520agonists%2520and%2520antagonists%2520by%2520calcium%2520influx%2520and%2520radioligand%2520binding%2520studies%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D125%26spage%3D41%26epage%3D54%26doi%3D10.1016%2Fj.bcp.2016.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Comparative protein modelling by satisfaction of spatial restraints</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1006%2Fjmbi.1993.1626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=8254673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+Saliauthor=T.+L.+Blundell&title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&doi=10.1006%2Fjmbi.1993.1626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein modeling by satisfaction of spatial restraints</span></div><div class="casAuthors">Sali, Andrej; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-815</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe a comparative protein modeling method designed to find the most probable structure for a sequence given its alignment with related structures.  The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability d. functions (pdfs) for the features restrained.  For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equiv. residue in a related protein, and the local similarity between the two sequences.  Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures.  The pdfs restrain Cα-Cα distances, main-chain N-O distances, main-chain and side-chain dihedral angles.  A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database.  The 3D model of a protein is obtained by optimization of the mol. pdf such that the model violates the input restraints as little as possible.  The mol. pdf is derived as a combination of pdfs restraining individual spatial features of the whole mol.  The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms.  The method is automated and is illustrated by the modeling of trypsin from two other serine proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC4wGhwZvib7Vg90H21EOLACvtfcHk0lhge5et2yjv5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D&md5=d4a3c39b2205e36221dc187a3d1a478b</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1626%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modelling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815%26doi%3D10.1006%2Fjmbi.1993.1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Protein structure modeling with MODELLER</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1137</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-0366-5_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1007%2F978-1-4939-0366-5_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=24573470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1137&publication_year=2014&pages=1-15&author=B.+Webbauthor=A.+Sali&title=Protein+structure+modeling+with+MODELLER&doi=10.1007%2F978-1-4939-0366-5_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Structure Modeling with MODELLER</span></div><div class="casAuthors">Webb, Benjamin; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1137</span>
        (<span class="NLM_cas:issue">Protein Structure Prediction</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Genome sequencing projects have resulted in a rapid increase in the no. of known protein sequences.  In contrast, only about one-hundredth of these sequences have been characterized at at. resoln. using exptl. structure detn. methods.  Computational protein structure modeling techniques have the potential to bridge this sequence-structure gap.  In this chapter, we present an example that illustrates the use of MODELLER to construct a comparative model for a protein with unknown structure.  Automation of a similar protocol has resulted in models of useful accuracy for domains in more than half of all known protein sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs6xvMHHyK0bVg90H21EOLACvtfcHk0lhge5et2yjv5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D&md5=19709a3ffefe33fd6e5d8506258bd6f0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-0366-5_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-0366-5_1%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DProtein%2520structure%2520modeling%2520with%2520MODELLER%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2014%26volume%3D1137%26spage%3D1%26epage%3D15%26doi%3D10.1007%2F978-1-4939-0366-5_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remmert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Söding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, D. G.</span></span> <span> </span><span class="NLM_article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">539</span>, <span class="refDoi"> DOI: 10.1038/msb.2011.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fmsb.2011.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=21988835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=539&author=F.+Sieversauthor=A.+Wilmauthor=D.+Dineenauthor=T.+J.+Gibsonauthor=K.+Karplusauthor=W.+Liauthor=R.+Lopezauthor=H.+McWilliamauthor=M.+Remmertauthor=J.+S%C3%B6dingauthor=J.+D.+Thompsonauthor=D.+G.+Higgins&title=Fast%2C+scalable+generation+of+high-quality+protein+multiple+sequence+alignments+using+Clustal+Omega&doi=10.1038%2Fmsb.2011.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span></div><div class="casAuthors">Sievers Fabian; Wilm Andreas; Dineen David; Gibson Toby J; Karplus Kevin; Li Weizhong; Lopez Rodrigo; McWilliam Hamish; Remmert Michael; Soding Johannes; Thompson Julie D; Higgins Desmond G</div><div class="citationInfo"><span class="NLM_cas:title">Molecular systems biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Multiple sequence alignments are fundamental to many sequence analysis methods.  Most alignments are computed using the progressive alignment heuristic.  These methods are starting to become a bottleneck in some analysis pipelines when faced with data sets of the size of many thousands of sequences.  Some methods allow computation of larger data sets while sacrificing quality, and others produce high-quality alignments, but scale badly with the number of sequences.  In this paper, we describe a new program called Clustal Omega, which can align virtually any number of protein sequences quickly and that delivers accurate alignments.  The accuracy of the package on smaller test cases is similar to that of the high-quality aligners.  On larger data sets, Clustal Omega outperforms other packages in terms of execution time and quality.  Clustal Omega also has powerful features for adding sequences to and exploiting information in existing alignments, making use of the vast amount of precomputed information in public databases like Pfam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQD21C9zX5warjnyZkbE8yXfW6udTcc2eZQM5AoDoP4Jbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D&md5=27dd8fac22447528e6d09e5fd1066133</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.75%26sid%3Dliteratum%253Aachs%26aulast%3DSievers%26aufirst%3DF.%26aulast%3DWilm%26aufirst%3DA.%26aulast%3DDineen%26aufirst%3DD.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DKarplus%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DRemmert%26aufirst%3DM.%26aulast%3DS%25C3%25B6ding%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26atitle%3DFast%252C%2520scalable%2520generation%2520of%2520high-quality%2520protein%2520multiple%2520sequence%2520alignments%2520using%2520Clustal%2520Omega%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D539%26doi%3D10.1038%2Fmsb.2011.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0ljlUREQ19f1PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunther, R.</span></span> <span> </span><span class="NLM_article-title"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e7979488a7869293888388848c">[email protected]</a>: a fast flexible molecular docking program based on Swarm intelligence</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00588.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1111%2Fj.1747-0285.2007.00588.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=17986206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGhsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2007&pages=475-484&author=V.+Namasivayamauthor=R.+Gunther&title=pso%40autodock%3A+a+fast+flexible+molecular+docking+program+based+on+Swarm+intelligence&doi=10.1111%2Fj.1747-0285.2007.00588.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0252514d424357564d464d4149">[email protected]</a>: a fast flexible molecular docking program based on Swarm intelligence</span></div><div class="casAuthors">Namasivayam, Vigneshwaran; Guenther, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">475-484</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">On the quest of novel therapeutics, mol. docking methods have proven to be valuable tools for screening large libraries of compds. detg. the interactions of potential drugs with the target proteins.  A widely used docking approach is the simulation of the docking process guided by a binding energy function.  On the basis of the mol. docking program AUTODOCK, we present <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="edbdbea2adacb8b9a2a9a2aea6">[email protected]</a> as a tool for fast flexible mol. docking.  Our novel Particle Swarm Optimization (PSO) algorithms varCPSO and varCPSO-Is are suited for rapid docking of highly flexible ligands.  Thus, a ligand with 23 rotatable bonds was successfully docked within as few as 100 000 computing steps (rmsd = 0.87 Å), which corresponds to only 10% of the computing time demanded by AUTODOCK.  In comparison to other docking techniques as GOLD 3.0, DOCK 6.0, FLEXX 2.2.0, AUTODOCK 3.05, and SODOCK, <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="eebebda1aeafbbbaa1aaa1ada5">[email protected]</a> provides the smallest rmsd values for 12 in 37 protein-ligand complexes.  The av. rmsd value of 1.4 Å is significantly lower then those obtained with the other docking programs, which are all above 2.0 Å.  Thus, <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="b2e2e1fdf2f3e7e6fdf6fdf1f9">[email protected]</a> is suggested as a highly efficient docking program in terms of speed and quality for flexible peptide-protein docking and virtual screening studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryKHisUEtA_LVg90H21EOLACvtfcHk0ljlUREQ19f1PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGhsb3E&md5=a8e0f99d7a51e3b62e42900c05a92b7f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00588.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00588.x%26sid%3Dliteratum%253Aachs%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DGunther%26aufirst%3DR.%26atitle%3Dpso%2540autodock%253A%2520a%2520fast%2520flexible%2520molecular%2520docking%2520program%2520based%2520on%2520Swarm%2520intelligence%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D70%26spage%3D475%26epage%3D484%26doi%3D10.1111%2Fj.1747-0285.2007.00588.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Charikleia S. Batsika, Christiana Mantzourani, Dimitrios Gkikas, Maroula G. Kokotou, Olga G. Mountanea, Christoforos G. Kokotos, Panagiotis K. Politis, <span class="NLM_string-name hlFld-ContribAuthor">George Kokotos</span>. </span><span class="cited-content_cbyCitation_article-title">Saturated Oxo Fatty Acids (SOFAs): A Previously Unrecognized Class of Endogenous Bioactive Lipids Exhibiting a Cell Growth Inhibitory Activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5654-5666. <a href="https://doi.org/10.1021/acs.jmedchem.0c02058" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02058%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSaturated%252BOxo%252BFatty%252BAcids%252B%252528SOFAs%252529%25253A%252BA%252BPreviously%252BUnrecognized%252BClass%252Bof%252BEndogenous%252BBioactive%252BLipids%252BExhibiting%252Ba%252BCell%252BGrowth%252BInhibitory%252BActivity%26aulast%3DBatsika%26aufirst%3DCharikleia%2BS.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D10122020%26date%3D21042021%26volume%3D64%26issue%3D9%26spage%3D5654%26epage%3D5666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lin-Hai Chen, Qing Zhang, Xin Xie, <span class="NLM_string-name hlFld-ContribAuthor">Fa-Jun Nan</span>. </span><span class="cited-content_cbyCitation_article-title">Modulation of the G-Protein-Coupled Receptor 84 (GPR84) by Agonists and Antagonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15399-15409. <a href="https://doi.org/10.1021/acs.jmedchem.0c01378" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01378</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01378%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DModulation%252Bof%252Bthe%252BG-Protein-Coupled%252BReceptor%252B84%252B%252528GPR84%252529%252Bby%252BAgonists%252Band%252BAntagonists%26aulast%3DChen%26aufirst%3DLin-Hai%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06082020%26date%3D03122020%26volume%3D63%26issue%3D24%26spage%3D15399%26epage%3D15409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maroula G. Kokotou, Alexandros C. Kokotos, Dimitrios Gkikas, Olga G. Mountanea, Christiana Mantzourani, Abdulaziz Almutairi, Xiaoyong Lei, Sasanka Ramanadham, Panagiotis K. Politis, <span class="NLM_string-name hlFld-ContribAuthor">George Kokotos</span>. </span><span class="cited-content_cbyCitation_article-title">Saturated Hydroxy Fatty Acids Exhibit a Cell Growth Inhibitory Activity and Suppress the Cytokine-Induced β-Cell Apoptosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12666-12681. <a href="https://doi.org/10.1021/acs.jmedchem.0c00844" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00844</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00844%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSaturated%252BHydroxy%252BFatty%252BAcids%252BExhibit%252Ba%252BCell%252BGrowth%252BInhibitory%252BActivity%252Band%252BSuppress%252Bthe%252BCytokine-Induced%252B%2525CE%2525B2-Cell%252BApoptosis%26aulast%3DKokotou%26aufirst%3DMaroula%2BG.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D04062020%26date%3D30102020%26volume%3D63%26issue%3D21%26spage%3D12666%26epage%3D12681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hang  Zhou</span>, <span class="hlFld-ContribAuthor ">Zhuo  Huang</span>, <span class="hlFld-ContribAuthor ">He  Huang</span>, <span class="hlFld-ContribAuthor ">Chuanjun  Song</span>, <span class="hlFld-ContribAuthor ">Junbiao  Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of bisindolylmethane, bispyrrolylmethane, and indolylpyrrolylmethane derivatives via reductive heteroarylation. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2021,</strong> <em>47 </em>, 132338. <a href="https://doi.org/10.1016/j.tet.2021.132338" title="DOI URL">https://doi.org/10.1016/j.tet.2021.132338</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2021.132338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2021.132338%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DSynthesis%252Bof%252Bbisindolylmethane%25252C%252Bbispyrrolylmethane%25252C%252Band%252Bindolylpyrrolylmethane%252Bderivatives%252Bvia%252Breductive%252Bheteroarylation%26aulast%3DZhou%26aufirst%3DHang%26date%3D2021%26volume%3D47%26spage%3D132338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Grundmann</span>, <span class="hlFld-ContribAuthor ">Eckhard  Bender</span>, <span class="hlFld-ContribAuthor ">Jens  Schamberger</span>, <span class="hlFld-ContribAuthor ">Frank  Eitner</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacology of Free Fatty Acid Receptors and Their Allosteric Modulators. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (4)
                                     , 1763. <a href="https://doi.org/10.3390/ijms22041763" title="DOI URL">https://doi.org/10.3390/ijms22041763</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22041763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22041763%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DPharmacology%252Bof%252BFree%252BFatty%252BAcid%252BReceptors%252Band%252BTheir%252BAllosteric%252BModulators%26aulast%3DGrundmann%26aufirst%3DManuel%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D4%26spage%3D1763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vincent B.  Luscombe</span>, <span class="hlFld-ContribAuthor ">Daniel  Lucy</span>, <span class="hlFld-ContribAuthor ">Carole J.R.  Bataille</span>, <span class="hlFld-ContribAuthor ">Angela J.  Russell</span>, <span class="hlFld-ContribAuthor ">David R.  Greaves</span>. </span><span class="cited-content_cbyCitation_article-title">20 Years an Orphan: Is GPR84 a Plausible Medium-Chain Fatty Acid-Sensing Receptor?. </span><span class="cited-content_cbyCitation_journal-name">DNA and Cell Biology</span><span> <strong>2020,</strong> <em>39 </em>
                                    (11)
                                     , 1926-1937. <a href="https://doi.org/10.1089/dna.2020.5846" title="DOI URL">https://doi.org/10.1089/dna.2020.5846</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/dna.2020.5846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fdna.2020.5846%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520and%2520Cell%2520Biology%26atitle%3D20%252BYears%252Ban%252BOrphan%25253A%252BIs%252BGPR84%252Ba%252BPlausible%252BMedium-Chain%252BFatty%252BAcid-Sensing%252BReceptor%25253F%26aulast%3DLuscombe%26aufirst%3DVincent%2BB.%26date%3D2020%26volume%3D39%26issue%3D11%26spage%3D1926%26epage%3D1937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Marsango</span>, <span class="hlFld-ContribAuthor ">Natasja  Barki</span>, <span class="hlFld-ContribAuthor ">Laura  Jenkins</span>, <span class="hlFld-ContribAuthor ">Andrew B.  Tobin</span>, <span class="hlFld-ContribAuthor ">Graeme  Milligan</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic validation of an orphan G protein‐coupled receptor: The case of GPR84. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2020,</strong> <em>21 </em><a href="https://doi.org/10.1111/bph.15248" title="DOI URL">https://doi.org/10.1111/bph.15248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.15248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.15248%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DTherapeutic%252Bvalidation%252Bof%252Ban%252Borphan%252BG%252Bprotein%2525E2%252580%252590coupled%252Breceptor%25253A%252BThe%252Bcase%252Bof%252BGPR84%26aulast%3DMarsango%26aufirst%3DSara%26date%3D2020%26date%3D2020%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0012.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of the GPR84 Agonist Radioligand [<sup>3</sup>H]<b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) PPh<sub>3</sub>, EtO<sub>2</sub>CN═NCO<sub>2</sub>Et, THF, 0 °C to rt, 12 h, 95%; (ii) hydrazine hydrate solution (∼80% in H<sub>2</sub>O), 100 °C, 3 h, 75.6%; (iii) 1-butanol, 125 °C, 12 h, 91%; (iv) procedure A: MeOH:THF (1:1), Pd/C (10%), H<sub>2</sub>, 1.7 bar, 2 h, 98%; procedure B: <i>N</i>,<i>N</i>′-dimethylformamide (DMF):THF (1:1), Pd/C (10%), H<sub>2</sub>, 2.4 bar, 2 h, 97.5%; (v) custom labeling in THF as a solvent (for details see <a class="ref internalNav" href="#notes-3" aria-label="SI">SI</a>).</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0013.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>4</b>, <b>62</b>–<b>68</b>, <b>73</b>–<b>77</b>, and <b>79</b>–<b>85</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) K<sub>2</sub>CO<sub>3</sub>, methanol, reflux, 12 h; (ii) from <b>76</b>, 2 N NaOH, methanol, 80 °C, 1 h; (iii) for esters (X = O): alcohols, H<sub>2</sub>SO<sub>4</sub>, reflux, 12 h; for amides (X = NH): amines, HATU, TEA, DMF, 12 h, rt.</p></p></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Effects of BSA and cations on [<sup>3</sup>H]<b>6</b> binding. The experiments were performed at 25 °C (150 min) using 1 nM [<sup>3</sup>H]<b>6</b> and 10 μg of a membrane preparation of CHO cells expressing the human GPR84 (CHO-hGPR84 cells). Data represent means of two independent experiments performed in duplicate; dpm, decays per minute. The unpaired <i>t</i> test was used for statistical comparisons (**, <i>p</i> < 0.01).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Association and (B) dissociation curve of [<sup>3</sup>H]<b>6</b> binding (1 nM) at 25 °C to membrane preparations of CHO-human GPR84 cells (10 μg protein/vial). Dissociation was achieved by addition of the GPR84 agonist <b>7</b> (100 μM) after 150 min of pre-incubation. A kinetic <i>K</i><sub>D</sub> value of 2.04 nM was determined. (C) Saturation curve of binding of increasing concentrations of [<sup>3</sup>H]<b>6</b>; experiments were performed at 25 °C with 10 μg of protein per vial. For the determination of nonspecific binding, 10 μM of <b>7</b> was used. The following binding parameters were calculated: <i>K</i><sub>D</sub> = 2.08 ± 0.68 nM, <i>B</i><sub>max</sub> = 1.87 ± 0.19 pmol/mg protein. (D) Homologous competition experiments. A <i>K</i><sub>i</sub> value of 2.98 ± 0.38 nM was determined for <b>6</b> (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Mean values ± SEM from 3 to 6 independent experiments performed in duplicates are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Competition binding experiments for standard agonists versus the agonist radioligand [<sup>3</sup>H]<b>6</b>. The experiments were performed as described in the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>. <i>K</i><sub>i</sub> values for lipid-like agonists were as follows: 1780 ± 285 nM for <b>1</b> (decanoic acid), 23.6 ± 6.8 nM for <b>2</b> (embelin), 0.626 ± 0.205 nM for <b>3</b> (6-octylaminouracil), 0.219 ± 0.054 nM for <b>4</b> (2-hexylthiopyrimidine derivative), and 0.317 ± 0.033 nM for <b>7</b> (6-(<i>p</i>-bromophenylethylamino)uracil). For positive allosteric modulators, EC<sub>50</sub> values were as follows: 368 ± 42 nM for <b>8</b> (diindolylmethane) and 41.5 ± 7.0 nM for <b>9</b> (5,5′,7,7′-tetrafluoro-DIM). Data points represent means ± SEM from 3–6 independent experiments, each performed in duplicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation of pEC<sub>50</sub> values of test compounds determined in cAMP assays (●) or β-arrestin assays (■) versus their respective p<i>K</i><sub>i</sub> values determined in radioligand binding assays. The same cell line (CHO-β-arrestin-human GPR84 cells) was used for all assays. <i>R</i><sup>2</sup> was determined in a two-tailed Pearson test. (A) Fatty acid derivatives and (B) 6-(ar)alkylamino-substituted uracil derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Competition binding experiments at membrane preparations of recombinant CHO cells expressing human GPR84. (A) Competition binding curves for selected 6-(ar)alkylamino-substituted uracil derivatives. (B) Competition binding curves for synthesized lipid-like compounds <b>62</b>–<b>65</b>, <b>68</b>, <b>74</b>, <b>77</b>, and <b>81</b>–<b>83</b>. Data points represent means ± SEM from three independent experiments, each performed in duplicates. For <i>K</i><sub>i</sub> values, see <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Competition binding experiments at membrane preparations of recombinant CHO cells expressing human GPR84. Allosteric modulation of radioligand binding by 3,3′-diindolylmethane derivatives. Data points represent means ± SEM from three to seven independent experiments, each performed in duplicates. For EC<sub>50</sub> and IC<sub>50</sub> values see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Correlation of pEC<sub>50</sub> values of DIM derivatives determined in cAMP assays (●) or β-arrestin assays (■) with their respective p<i>K</i><sub>i</sub> values determined in radioligand binding assays. The same cell line (CHO-β-arrestin-human GPR84 cells) was used for all assays. <i>R</i><sup>2</sup> was determined in a two-tailed Pearson test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Homology model of the human GPR84. (A) Homology model of the human GPR84 based on the human dopamine D3 receptor crystal structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>). (B) The putative orthosteric binding site of the human GPR84 predicted using SiteFinder implemented in MOE is schematically indicated and depicted as red spheres. (C) The important amino acid residues in the presumed orthosteric binding pocket are shown. Carbon atoms are colored in gray, oxygen atoms in red, nitrogen atoms in blue, and sulfur atoms in yellow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Docking studies of selected agonists at the human GPR84 model (based on the human dopamine D3 receptor crystal structure, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>). The docked poses of decanoic acid (<b>1</b>, green), embelin (<b>2</b>, marine blue), and uracil derivatives <b>3</b> (cyan), <b>6</b> (yellow), and <b>7</b> (orange) in the orthosteric binding pocket (surface representation, colored maroon red) of the human GPR84 model are displayed in cartoon representation (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0010.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Binding site residues of the human GPR84 model (based on the human dopamine D3 receptor crystal structure, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>) and putative binding modes of agonists. (A) Important residues in the binding pocket are shown. The amino acid residues forming hydrogen-bond interactions are labeled in red color, hydrophobic interactions are in blue, and the mutants published by Nikaido et al. are highlighted by red boxes.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Binding poses of agonists in the putative orthosteric binding site: (B) decanoic acid (<b>1</b>, green); (C) embelin (<b>2</b>, marine blue); (D) 6-hexylaminouracil (<b>6</b>, yellow). Red dotted lines indicate hydrogen-bond interactions (for further color code see <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/medium/jm9b01339_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0011.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Specific binding of [<sup>3</sup>H]<b>6</b> to membrane preparations of various native tissues and cell lines. The experiments were performed at 25 °C using 10 nM [<sup>3</sup>H]<b>6</b> and 100 μg protein/vial, except for Jurkat cells and T-lymphocytes: 25 μg protein/vial; HepG2 cell: 2 nM RL. Agonist <b>7</b> was used for the determination of nonspecific binding. Data points represent means ± SEM from three to five independent experiments, each performed in duplicates. At low GPR84 expression levels, the percentage of nonspecific binding is relatively high, and the data are consequently less accurate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01339/20200306/images/large/jm9b01339_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01339&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81764" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81764" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 62 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wittenberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellebrand, S.</span></span> <span> </span><span class="NLM_article-title">An expressed sequence tag (EST) data mining strategy succeeding in the discovery of new Gprotein-coupled receptors</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>307</i></span>,  <span class="NLM_fpage">799</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1006/jmbi.2001.4520</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1006%2Fjmbi.2001.4520" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=11273702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitFymu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=307&publication_year=2001&pages=799-813&author=T.+Wittenbergerauthor=H.+C.+Schallerauthor=S.+Hellebrand&title=An+expressed+sequence+tag+%28EST%29+data+mining+strategy+succeeding+in+the+discovery+of+new+Gprotein-coupled+receptors&doi=10.1006%2Fjmbi.2001.4520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">An Expressed Sequence Tag (EST) Data Mining Strategy Succeeding in the Discovery of New G-Protein Coupled Receptors</span></div><div class="casAuthors">Wittenberger, Timo; Schaller, H. Chica; Hellebrand, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">307</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">799-813</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">We have developed a comprehensive expressed sequence tag database search method and used it for the identification of new members of the G-protein coupled receptor superfamily.  Our approach proved to be esp. useful for the detection of expressed sequence tag sequences that do not encode conserved parts of a protein, making it an ideal tool for the identification of members of divergent protein families or of protein parts without conserved domain structures in the expressed sequence tag database.  At least 14 of the expressed sequence tags found with this strategy are promising candidates for new putative G-protein coupled receptors.  Here, we describe the sequence and expression anal. of five new members of this receptor superfamily, namely GPR84, GPR86, GPR87, GPR90 and GPR91.  We also studied the genomic structure and chromosomal localization of the resp. genes applying in silico methods.  A cluster of six closely related G-protein coupled receptors was found on the human chromosome 3q24-3q25.  It consists of four orphan receptors (GPR86, GPR87, GPR91, and H963), the purinergic receptor P2Y1, and the uridine 5'-diphosphoglucose receptor KIAA0001.  It seems likely that these receptors evolved from a common ancestor and therefore might have related ligands.  In conclusion, we describe a data mining procedure that proved to be useful for the identification and first characterization of new genes and is well applicable for other gene families.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2zESMaLlx3bVg90H21EOLACvtfcHk0liUG-0Urmu2lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitFymu78%253D&md5=0e8fcf6476a2ff247e9813df05247df3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.4520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.4520%26sid%3Dliteratum%253Aachs%26aulast%3DWittenberger%26aufirst%3DT.%26aulast%3DSchaller%26aufirst%3DH.%2BC.%26aulast%3DHellebrand%26aufirst%3DS.%26atitle%3DAn%2520expressed%2520sequence%2520tag%2520%2528EST%2529%2520data%2520mining%2520strategy%2520succeeding%2520in%2520the%2520discovery%2520of%2520new%2520Gprotein-coupled%2520receptors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D307%26spage%3D799%26epage%3D813%26doi%3D10.1006%2Fjmbi.2001.4520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yousefi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarai, G.</span></span> <span> </span><span class="NLM_article-title">Cloning and expression analysis of a novel G protein-coupled receptor selectively expressed on granulocytes</span>. <i>J. Leukoc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1045</span>– <span class="NLM_lpage">1052</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=11404393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkslWrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2001&pages=1045-1052&author=S.+Yousefiauthor=P.+R.+Cooperauthor=S.+L.+Potterauthor=B.+Mueckauthor=G.+Jarai&title=Cloning+and+expression+analysis+of+a+novel+G+protein-coupled+receptor+selectively+expressed+on+granulocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cloning and expression analysis of a novel G-protein-coupled receptor selectively expressed on granulocytes</span></div><div class="casAuthors">Yousefi, Shida; Cooper, Paul R.; Potter, Suzanne L.; Mueck, Beatrice; Jarai, Gabor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1045-1052</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">The migration of neutrophils into sites of acute and chronic inflammation is mediated by chemokines.  The authors used degenerate-primer reverse transcriptase-polymerase chain reaction (RT-PCR) to analyze chemokine receptor expression in neutrophils and identify novel receptors.  RNA was isolated from human peripheral blood neutrophils and from neutrophils that had been stimulated for 5 h with granulocyte-macrophage colony-stimulating factor or by coculturing with primary human bronchial epithelial cells.  Amplification products were cloned, and clone redundancy was detd.  Seven known G-protein-coupled receptors were identified among 38 clones-CCR1, CCR4, CXCR1, CXCR2, CXCR4, HM63, and FPR1-as well as a novel gene, EX33.  The full-length EX33 clone was obtained, and an in silico approach was used to identify the putative murine homolog.  The EX33 gene encodes a 396-amino-acid protein with limited sequence identity to known receptors.  Expression studies of several known chemokine receptors and EX33 revealed that resting neutrophils expressed higher levels of CXCRs and EX33 compared with activated neutrophils.  Northern blot expts. revealed that EX33 is expressed mainly in bone marrow, lung, and peripheral blood leukocytes.  Using RT-PCR anal., the authors showed more abundant expression of EX33 in neutrophils and eosinophils, in comparison with that in T- or B-lymphocytes, indicating cell-specific expression among leukocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAMqeieaewbrVg90H21EOLACvtfcHk0ljr_yUlb3cfyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkslWrurs%253D&md5=03d1eb21f619cdd4790ace3c2b571d23</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYousefi%26aufirst%3DS.%26aulast%3DCooper%26aufirst%3DP.%2BR.%26aulast%3DPotter%26aufirst%3DS.%2BL.%26aulast%3DMueck%26aufirst%3DB.%26aulast%3DJarai%26aufirst%3DG.%26atitle%3DCloning%2520and%2520expression%2520analysis%2520of%2520a%2520novel%2520G%2520protein-coupled%2520receptor%2520selectively%2520expressed%2520on%2520granulocytes%26jtitle%3DJ.%2520Leukoc.%2520Biol.%26date%3D2001%26volume%3D69%26spage%3D1045%26epage%3D1052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Venkataraman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, F.</span></span> <span> </span><span class="NLM_article-title">The Gprotein-coupled receptor, GPR84 regulates IL-4 production by T lymphocytes in response to CD3 crosslinking</span>. <i>Immunol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1016/j.imlet.2005.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.imlet.2005.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=15993493" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKjs77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2005&pages=144-153&author=C.+Venkataramanauthor=F.+Kuo&title=The+Gprotein-coupled+receptor%2C+GPR84+regulates+IL-4+production+by+T+lymphocytes+in+response+to+CD3+crosslinking&doi=10.1016%2Fj.imlet.2005.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The G-protein coupled receptor, GPR84 regulates IL-4 production by T lymphocytes in response to CD3 crosslinking</span></div><div class="casAuthors">Venkataraman, Chandrasekar; Kuo, Frederick</div><div class="citationInfo"><span class="NLM_cas:title">Immunology Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">144-153</span>CODEN:
                <span class="NLM_cas:coden">IMLED6</span>;
        ISSN:<span class="NLM_cas:issn">0165-2478</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The orphan G-protein coupled receptor, GPR84 is highly expressed in the bone marrow, and in splenic T cells and B cells.  In this study, GPR84-deficient mice were generated to understand the biol. function of this orphan receptor.  The proliferation of T and B cells in response to various mitogens was normal in GPR84-deficient mice.  Interestingly, primary stimulation of T cells with anti-CD3 resulted in increased IL-4 but not IL-2 or IFN-γ prodn. in GPR84-/- mice compared to wild-type mice.  Augmented IL-4 prodn. in GPR84-deficient T cells was not related to increased frequency of IL-4-secreting cells in response to anti-CD3 stimulation.  In fact, stimulation with anti-CD3 and anti-CD28 resulted in increased levels of IL-4 but not IFN-γ steady-state mRNA in GPR84-/- T cells.  In addn., Th2 effector cells generated in vitro from GPR84-/- mice produced higher levels of IL-4, IL-5 and IL-13 compared to wild-type mice.  However, there was no detectable difference in the extent of IL-4 and IL-5 prodn. between the two groups of mice in response to antigen stimulation of spleen cells, isolated from mice previously immunized with OVA in alum.  These studies reveal a novel role for GPR84 in regulating early IL-4 gene expression in activated T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoxPqqzrLXAbVg90H21EOLACvtfcHk0ljr_yUlb3cfyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKjs77L&md5=abef6f196008a5055ab99e9213c0d55d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.imlet.2005.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.imlet.2005.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DVenkataraman%26aufirst%3DC.%26aulast%3DKuo%26aufirst%3DF.%26atitle%3DThe%2520Gprotein-coupled%2520receptor%252C%2520GPR84%2520regulates%2520IL-4%2520production%2520by%2520T%2520lymphocytes%2520in%2520response%2520to%2520CD3%2520crosslinking%26jtitle%3DImmunol.%2520Lett.%26date%3D2005%26volume%3D101%26spage%3D144%26epage%3D153%26doi%3D10.1016%2Fj.imlet.2005.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonavicius, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, L.</span></span> <span> </span><span class="NLM_article-title">Medium chain fatty acids as ligands for orphan G protein-coupled receptor GPR84</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">34457</span>– <span class="NLM_lpage">34464</span>, <span class="refDoi"> DOI: 10.1074/jbc.M608019200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1074%2Fjbc.M608019200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=16966319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFKgur%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=34457-34464&author=J.+Wangauthor=X.+Wuauthor=N.+Simonaviciusauthor=H.+Tianauthor=L.+Ling&title=Medium+chain+fatty+acids+as+ligands+for+orphan+G+protein-coupled+receptor+GPR84&doi=10.1074%2Fjbc.M608019200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Medium-chain Fatty Acids as Ligands for Orphan G Protein-coupled Receptor GPR84</span></div><div class="casAuthors">Wang, Jinghong; Wu, Xiaosu; Simonavicius, Nicole; Tian, Hui; Ling, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">34457-34464</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Free fatty acids (FFAs) play important physiol. roles in many tissues as an energy source and as signaling mols. in various cellular processes.  Elevated levels of circulating FFAs are assocd. with obesity, dyslipidemia, and diabetes.  Here we show that GPR84, a previously orphan G protein-coupled receptor, functions as a receptor for medium-chain FFAs with carbon chain lengths of 9-14.  Medium-chain FFAs elicit calcium mobilization, inhibit 3',5'-cAMP prodn., and stimulate [35S]guanosine 5'-O-(3-thiotriphosphate) binding in a GPR84-dependent manner.  The activation of GPR84 by medium-chain FFAs couples primarily to a pertussis toxin-sensitive Gi/o pathway.  In addn., we show that GPR84 is selectively expressed in leukocytes and markedly induced in monocytes/macrophages upon activation by lipopolysaccharide.  Furthermore, we demonstrate that medium-chain FFAs amplify lipopolysaccharide-stimulated prodn. of the proinflammatory cytokine interleukin-12 p40 through GPR84.  Our results indicate a role for GPR84 in directly linking fatty acid metab. to immunol. regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-BrABfXO2grVg90H21EOLACvtfcHk0ljr_yUlb3cfyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFKgur%252FI&md5=2bbe13a202723f62e59eab0b08a25878</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M608019200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M608019200%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSimonavicius%26aufirst%3DN.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DLing%26aufirst%3DL.%26atitle%3DMedium%2520chain%2520fatty%2520acids%2520as%2520ligands%2520for%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR84%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D34457%26epage%3D34464%26doi%3D10.1074%2Fjbc.M608019200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouchard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagé, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bédard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallières, L.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor 84, a microglia-associated protein expressed in neuroinflammatory conditions</span>. <i>Glia</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">790</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1002/glia.20506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fglia.20506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=17390309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BD2s3jtVemsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2007&pages=790-800&author=C.+Bouchardauthor=J.+Pag%C3%A9author=A.+B%C3%A9dardauthor=P.+Tremblayauthor=L.+Valli%C3%A8res&title=G+protein-coupled+receptor+84%2C+a+microglia-associated+protein+expressed+in+neuroinflammatory+conditions&doi=10.1002%2Fglia.20506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor 84, a microglia-associated protein expressed in neuroinflammatory conditions</span></div><div class="casAuthors">Bouchard Caroline; Page Julie; Bedard Andreanne; Tremblay Pierrot; Vallieres Luc</div><div class="citationInfo"><span class="NLM_cas:title">Glia</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">790-800</span>
        ISSN:<span class="NLM_cas:issn">0894-1491</span>.
    </div><div class="casAbstract">G protein-coupled receptor 84 (GPR84) is a recently discovered member of the seven transmembrane receptor superfamily whose function and regulation are unknown.  Here, we report that in mice suffering from endotoxemia, microglia express GPR84 in a strong and sustained manner.  This property is shared by subpopulations of peripheral macrophages and, to a much lesser extent, monocytes.  The induction of GPR84 expression by endotoxin is mediated, at least in part, by proinflammatory cytokines, notably tumor necrosis factor (TNF) and interleukin-1 (IL-1), because mice lacking either one or both of these molecules have fewer GPR84-expressing cells in their cerebral cortex than wild-type mice during the early phase of endotoxemia.  Moreover, when injected intracerebrally or added to microglial cultures, recombinant TNF stimulates GPR84 expression through a dexamethasone-insensitive mechanism.  Finally, we show that microglia produce GPR84 not only during endotoxemia, but also during experimental autoimmune encephalomyelitis (EAE), a model of multiple sclerosis.  In conclusion, this study reports the identification of a new sensitive marker of microglial activation, which may play an important regulatory role in neuroimmunological processes, acting downstream to the effects of proinflammatory mediators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR5WBo60SwdsPEeosoOnpOLfW6udTcc2eb-jgtAIxfdw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3jtVemsw%253D%253D&md5=5bc25664243312ac66f94244ab598c9d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fglia.20506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fglia.20506%26sid%3Dliteratum%253Aachs%26aulast%3DBouchard%26aufirst%3DC.%26aulast%3DPag%25C3%25A9%26aufirst%3DJ.%26aulast%3DB%25C3%25A9dard%26aufirst%3DA.%26aulast%3DTremblay%26aufirst%3DP.%26aulast%3DValli%25C3%25A8res%26aufirst%3DL.%26atitle%3DG%2520protein-coupled%2520receptor%252084%252C%2520a%2520microglia-associated%2520protein%2520expressed%2520in%2520neuroinflammatory%2520conditions%26jtitle%3DGlia%26date%3D2007%26volume%3D55%26spage%3D790%26epage%3D800%26doi%3D10.1002%2Fglia.20506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Recio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purvis, G. S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iveson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeboudj, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Callaghan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griesbach, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dib, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaco, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greaves, D. R.</span></span> <span> </span><span class="NLM_article-title">Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1419</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2018.01419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.3389%2Ffimmu.2018.01419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29973940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1yqtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1419&author=C.+Recioauthor=D.+Lucyauthor=G.+S.+D.+Purvisauthor=P.+Ivesonauthor=L.+Zeboudjauthor=A.+J.+Iqbalauthor=D.+Linauthor=C.+O%E2%80%99Callaghanauthor=L.+Davisonauthor=E.+Griesbachauthor=A.+J.+Russellauthor=G.+M.+Wynneauthor=L.+Dibauthor=C.+Monacoauthor=D.+R.+Greaves&title=Activation+of+the+immune-metabolic+receptor+GPR84+enhances+inflammation+and+phagocytosis+in+macrophages&doi=10.3389%2Ffimmu.2018.01419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of the immune-metabolic receptor GPR84 enhances inflammation and phagocytosis in macrophages</span></div><div class="casAuthors">Recio, Carlota; Lucy, Daniel; Purvis, Gareth S. D.; Iveson, Poppy; Zeboudj, Lynda; Iqbal, Asif J.; Lin, Daniel; O'Callaghan, Chris; Davison, Lucy; Griesbach, Esther; Russell, Angela J.; Wynne, Graham M.; Dib, Lea; Monaco, Claudia; Greaves, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Immunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1419/1-1419/17</span>CODEN:
                <span class="NLM_cas:coden">FIRMCW</span>;
        ISSN:<span class="NLM_cas:issn">1664-3224</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">GPR84 is a member of the metabolic G protein-coupled receptor family, and its expres- sion has been described predominantly in immune cells.  GPR84 activation is involved in the inflammatory response, but the mechanisms by which it modulates inflammation have been incompletely described.  In this study, we investigated GPR84 expression, activation, and function in macrophages to establish the role of the receptor during the inflammatory response.  We obsd. that GPR84 expression in murine tissues is increased by endotoxemia, hyperglycemia, and hypercholesterolemia.  Ex vivo studies revealed that GPR84 mRNA expression is increased by LPS and other pro-inflammatory mols. in different murine and human macrophage populations.  Likewise, high glu- cose concns. and the presence of oxidized LDL increased GPR84 expression in macrophages.  Activation of the GPR84 receptor with a selective agonist, 6-(octylamino) pyrimidine-2,4(1H,3H)-dione (6-n-octylaminouracil, 6-OAU), enhanced the expression of phosphorylated Akt, p-ERK, and p65 nuclear translocation under inflammatory con- ditions and elevated the expression levels of the inflammatory mediators TNFα, IL-6, IL-12B, CCL2, CCL5, and CXCL1.  In addn., GPR84 activation triggered increased bacterial adhesion and phagocytosis in macrophages.  The enhanced inflammatory response mediated by 6-OAU was not obsd. in GPR84/- cells nor in macrophages treated with a selective GPR84 antagonist.  Collectively, our results reveal that GPR84 functions as an enhancer of inflammatory signaling in macrophages once inflammation is established.  Therefore, mols. that antagonize the GPR84 receptor may be potential therapeutic tools in inflammatory and metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXDBYilKMKTrVg90H21EOLACvtfcHk0ljC2ys02obCDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1yqtr%252FF&md5=0e28917468e38dd31ad0b858b3780f6a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2018.01419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2018.01419%26sid%3Dliteratum%253Aachs%26aulast%3DRecio%26aufirst%3DC.%26aulast%3DLucy%26aufirst%3DD.%26aulast%3DPurvis%26aufirst%3DG.%2BS.%2BD.%26aulast%3DIveson%26aufirst%3DP.%26aulast%3DZeboudj%26aufirst%3DL.%26aulast%3DIqbal%26aufirst%3DA.%2BJ.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Callaghan%26aufirst%3DC.%26aulast%3DDavison%26aufirst%3DL.%26aulast%3DGriesbach%26aufirst%3DE.%26aulast%3DRussell%26aufirst%3DA.%2BJ.%26aulast%3DWynne%26aufirst%3DG.%2BM.%26aulast%3DDib%26aufirst%3DL.%26aulast%3DMonaco%26aufirst%3DC.%26aulast%3DGreaves%26aufirst%3DD.%2BR.%26atitle%3DActivation%2520of%2520the%2520immune-metabolic%2520receptor%2520GPR84%2520enhances%2520inflammation%2520and%2520phagocytosis%2520in%2520macrophages%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D9%26spage%3D1419%26doi%3D10.3389%2Ffimmu.2018.01419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaishi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onozawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iino, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, T.</span></span> <span> </span><span class="NLM_article-title">Medium-chain fatty acid-sensing receptor, GPR84, is a proinflammatory receptor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">10684</span>– <span class="NLM_lpage">10691</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.420042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1074%2Fjbc.M112.420042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23449982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvFSisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=10684-10691&author=M.+Suzukiauthor=S.+Takaishiauthor=M.+Nagasakiauthor=Y.+Onozawaauthor=I.+Iinoauthor=H.+Maedaauthor=T.+Komaiauthor=T.+Oda&title=Medium-chain+fatty+acid-sensing+receptor%2C+GPR84%2C+is+a+proinflammatory+receptor&doi=10.1074%2Fjbc.M112.420042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Medium-chain Fatty Acid-sensing Receptor, GPR84, Is a Proinflammatory Receptor</span></div><div class="casAuthors">Suzuki, Masakatsu; Takaishi, Sachiko; Nagasaki, Miyuki; Onozawa, Yoshiko; Iino, Ikue; Maeda, Hiroaki; Komai, Tomoaki; Oda, Tomiichiro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">10684-10691</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">G protein-coupled receptor 84 (GPR84) is a putative receptor for medium-chain fatty acids (MCFAs), whose pathophysiol. roles have not yet been clarified.  Here, the authors show that GPR84 was activated by MCFAs with the hydroxyl group at the 2- or 3-position more effectively than nonhydroxylated MCFAs.  The authors also identified a surrogate agonist, 6-n-octylaminouracil (6-OAU), for GPR84.  These potential ligands and the surrogate agonist, 6-OAU, stimulated [35S]GTP binding and accumulated phosphoinositides in a GPR84-dependent manner.  The surrogate agonist, 6-OAU, internalized GPR84-EGFP from the cell surface.  Both the potential ligands and 6-OAU elicited chemotaxis of human polymorphonuclear leukocytes (PMNs) and macrophages and amplified LPS-stimulated prodn. of the proinflammatory cytokine IL-8 from PMNs and TNFα from macrophages.  Furthermore, the i.v. injection of 6-OAU raised the blood CXCL1 level in rats, and the inoculation of 6-OAU into the rat air pouch accumulated PMNs and macrophages in the site.  The results indicate a proinflammatory role of GPR84, suggesting that the receptor may be a novel target to treat chronic low grade inflammation assocd.-disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTeAePnQZYFrVg90H21EOLACvtfcHk0lja-1cE-mZdqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvFSisr8%253D&md5=63a395e01eb2083bfb7d4a1195c88b1a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.420042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.420042%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DTakaishi%26aufirst%3DS.%26aulast%3DNagasaki%26aufirst%3DM.%26aulast%3DOnozawa%26aufirst%3DY.%26aulast%3DIino%26aufirst%3DI.%26aulast%3DMaeda%26aufirst%3DH.%26aulast%3DKomai%26aufirst%3DT.%26aulast%3DOda%26aufirst%3DT.%26atitle%3DMedium-chain%2520fatty%2520acid-sensing%2520receptor%252C%2520GPR84%252C%2520is%2520a%2520proinflammatory%2520receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D10684%26epage%3D10691%26doi%3D10.1074%2Fjbc.M112.420042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anthony, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatlin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solomon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semple, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unett, D. J.</span></span> <span> </span><span class="NLM_article-title">Embelin and its derivatives unravel the signaling, proinflammatory and antiatherogenic properties of GPR84 receptor</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">198</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2018.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.phrs.2018.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29471103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjs1Wmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2018&pages=185-198&author=I.+Gaidarovauthor=T.+Anthonyauthor=J.+Gatlinauthor=X.+Chenauthor=D.+Millsauthor=M.+Solomonauthor=S.+Hanauthor=G.+Sempleauthor=D.+J.+Unett&title=Embelin+and+its+derivatives+unravel+the+signaling%2C+proinflammatory+and+antiatherogenic+properties+of+GPR84+receptor&doi=10.1016%2Fj.phrs.2018.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Embelin and its derivatives unravel the signaling, proinflammatory and antiatherogenic properties of GPR84 receptor</span></div><div class="casAuthors">Gaidarov, Ibragim; Anthony, Todd; Gatlin, Joel; Chen, Xiaohua; Mills, David; Solomon, Michelle; Han, Sangdon; Semple, Graeme; Unett, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">185-198</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">GPR84 is an orphan G-protein coupled receptor, expressed on monocytes, macrophages and neutrophils and is significantly upregulated by inflammatory stimuli.  The physiol. role of GPR84 remains largely unknown.  Medium chain fatty acids (MCFA) activate the receptor and have been proposed to be its endogenous ligands, although the high concns. of MCFAs required for receptor activation generally exceed normal physiol. levels.  We identified the natural product embelin as a highly potent and selective surrogate GPR84 agonist (originally disclosed in patent application WO2007027661A2, 2007) and synthesized close structural analogs with widely varying receptor activities.  These tools were used to perform a comprehensive study of GPR84 signaling and function in recombinant cells and in primary human macrophages and neutrophils.  Activation of recombinant GPR84 by embelin in HEK293 cells results in Gi/o as well as G12/13-Rho signaling.  In human macrophages, GPR84 initiates PTX sensitive Erk1/2 and Akt phosphorylation, PI-3 kinase activation, calcium flux, and release of prostaglandin E2.  In addn., GPR84 signaling in macrophages elicits Gi Gβγ-mediated augmentation of intracellular cAMP, rather than the decrease expected from Giα engagement.  GPR84 activation drives human neutrophil chemotaxis and primes them for amplification of oxidative burst induced by FMLP and C5A.  Loss of GPR84 is assocd. with attenuated LPS-induced release of proinflammatory mediators IL-6, KC-GROα, VEGF, MIP-2 and NGAL from peritoneal exudates.  While initiating numerous proinflammatory activities in macrophages and neutrophils, GPR84 also possesses GPR109A-like antiatherosclerotic properties in macrophages.  Macrophage receptor activation leads to upregulation of cholesterol transporters ABCA1 and ABCG1 and stimulates reverse cholesterol transport.  These data suggest that GPR84 may be a target of therapeutic value and that distinct modes of receptor modulation (inhibition vs. stimulation) may be required for inflammatory and atherosclerotic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAw98Yo2b4GrVg90H21EOLACvtfcHk0lja-1cE-mZdqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Wmsbo%253D&md5=f4c0ba851ede2aa9739af183423ff9f1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2018.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2018.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DGaidarov%26aufirst%3DI.%26aulast%3DAnthony%26aufirst%3DT.%26aulast%3DGatlin%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMills%26aufirst%3DD.%26aulast%3DSolomon%26aufirst%3DM.%26aulast%3DHan%26aufirst%3DS.%26aulast%3DSemple%26aufirst%3DG.%26aulast%3DUnett%26aufirst%3DD.%2BJ.%26atitle%3DEmbelin%2520and%2520its%2520derivatives%2520unravel%2520the%2520signaling%252C%2520proinflammatory%2520and%2520antiatherogenic%2520properties%2520of%2520GPR84%2520receptor%26jtitle%3DPharmacol.%2520Res.%26date%3D2018%26volume%3D131%26spage%3D185%26epage%3D198%26doi%3D10.1016%2Fj.phrs.2018.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundqvist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christenson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdfeldt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mårtensson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Björkman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dieckmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsman, H.</span></span> <span> </span><span class="NLM_article-title">Similarities and differences between the responses induced in human phagocytes through activation of the medium chain fatty acid receptor GPR84 and the short chain fatty acid receptor FFA2R</span>. <i>Biochim. Biophys. Acta, Mol. Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1865</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">708</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2018.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bbamcr.2018.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29477577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsFaju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1865&publication_year=2018&pages=695-708&author=M.+Sundqvistauthor=K.+Christensonauthor=A.+Holdfeldtauthor=M.+Gablauthor=J.+M%C3%A5rtenssonauthor=L.+Bj%C3%B6rkmanauthor=R.+Dieckmannauthor=C.+Dahlgrenauthor=H.+Forsman&title=Similarities+and+differences+between+the+responses+induced+in+human+phagocytes+through+activation+of+the+medium+chain+fatty+acid+receptor+GPR84+and+the+short+chain+fatty+acid+receptor+FFA2R&doi=10.1016%2Fj.bbamcr.2018.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Similarities and differences between the responses induced in human phagocytes through activation of the medium chain fatty acid receptor GPR84 and the short chain fatty acid receptor FFA2R</span></div><div class="casAuthors">Sundqvist, Martina; Christenson, Karin; Holdfeldt, Andre; Gabl, Michael; Maartensson, Jonas; Bjoerkman, Lena; Dieckmann, Regis; Dahlgren, Claes; Forsman, Huamei</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1865</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">695-708</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">GPR84 is a recently de-orphanized member of the G-protein coupled receptor (GPCR) family recognizing medium chain fatty acids, and has been suggested to play important roles in inflammation.  Due to the lack of potent and selective GPR84 ligands, the basic knowledge related to GPR84 functions is very limited.  In this study, we have characterized the GPR84 activation profile and regulation mechanism in human phagocytes, using two recently developed small mols. that specifically target GPR84 agonistically (ZQ16) and antagonistically (GLPG1205), resp.  Compared to our earlier characterization of the short chain fatty acid receptor FFA2R which is functionally expressed in neutrophils but not in monocytes, GPR84 is expressed in both cell types and in monocyte-derived macrophages.  In neutrophils, the GPR84 agonist had an activation profile very similar to that of FFA2R.  The GPR84-mediated superoxide release was low in naive cells, but the response could be significantly primed by TNFα and by the actin cytoskeleton disrupting agent Latrunculin A.  Similar to that of FFA2R, a desensitization mechanism bypassing the actin cytoskeleton was utilized by GPR84.  All ZQ16-mediated cellular responses were sensitive to GLPG1205, confirming the GPR84-dependency.  Finally, our data of in vivo transmigrated tissue neutrophils indicate that both GPR84 and FFA2R are involved in neutrophil recruitment processes in vivo.  In summary, we show functional similarities but also some important differences between GPR84 and FFA2R in human phagocytes, thus providing some mechanistic insights into GPR84 regulation in blood neutrophils and cells recruited to an aseptic inflammatory site in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxFh26S2UftbVg90H21EOLACvtfcHk0lgS5NC8aLailA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsFaju7w%253D&md5=5f80256de9cc62715bf953312937170e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2018.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2018.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DSundqvist%26aufirst%3DM.%26aulast%3DChristenson%26aufirst%3DK.%26aulast%3DHoldfeldt%26aufirst%3DA.%26aulast%3DGabl%26aufirst%3DM.%26aulast%3DM%25C3%25A5rtensson%26aufirst%3DJ.%26aulast%3DBj%25C3%25B6rkman%26aufirst%3DL.%26aulast%3DDieckmann%26aufirst%3DR.%26aulast%3DDahlgren%26aufirst%3DC.%26aulast%3DForsman%26aufirst%3DH.%26atitle%3DSimilarities%2520and%2520differences%2520between%2520the%2520responses%2520induced%2520in%2520human%2520phagocytes%2520through%2520activation%2520of%2520the%2520medium%2520chain%2520fatty%2520acid%2520receptor%2520GPR84%2520and%2520the%2520short%2520chain%2520fatty%2520acid%2520receptor%2520FFA2R%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2018%26volume%3D1865%26spage%3D695%26epage%3D708%26doi%3D10.1016%2Fj.bbamcr.2018.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuhuber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kassem, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khailah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldeen, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khairy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khayyal, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shcherbakova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Efferth, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich-Merzenich, G.</span></span> <span> </span><span class="NLM_article-title">GPR84 and TREM-1 signaling contribute to the pathogenesis of reflux esophagitis</span>. <i>Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1011</span>– <span class="NLM_lpage">1024</span>, <span class="refDoi"> DOI: 10.2119/molmed.2015.00098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.2119%2Fmolmed.2015.00098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKlsLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1011-1024&author=H.+Abdel-Azizauthor=M.+Schneiderauthor=W.+Neuhuberauthor=A.+M.+Kassemauthor=S.+Khailahauthor=J.+Mullerauthor=H.+G.+Eldeenauthor=A.+Khairyauthor=M.+T.+Khayyalauthor=A.+Shcherbakovaauthor=T.+Efferthauthor=G.+Ulrich-Merzenich&title=GPR84+and+TREM-1+signaling+contribute+to+the+pathogenesis+of+reflux+esophagitis&doi=10.2119%2Fmolmed.2015.00098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">GPR84 and TREM-1 signaling contribute to the pathogenesis of reflux esophagitis</span></div><div class="casAuthors">Abdel-Aziz, Heba; Schneider, Mathias; Neuhuber, Winfried; Kassem, Abdel Meguid; Khailah, Saleem; Mueller, Juergen; Eldeen, Hadeel Gamal; Khairy, Ahmed; Khayyal, Mohamed T.; Shcherbakova, Anastasiia; Efferth, Thomas; Ulrich-Merzenich, Gudrun</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Medicine (Manhasset, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1011-1024</span>CODEN:
                <span class="NLM_cas:coden">MOMEF3</span>;
        ISSN:<span class="NLM_cas:issn">1528-3658</span>.
    
            (<span class="NLM_cas:orgname">Feinstein Institute for Medical Research</span>)
        </div><div class="casAbstract">Gastro-esophageal reflux disease (GERD) is one of the most common disorders in gastroenterol.  Patients present with or without increased acid exposure indicating a nonuniform etiol.  Thus, the common treatment with proton pump inhibitors (PPIs) fails to control symptoms in up to 40% of patients.  To further elucidate the pathophysiol. of the condition and explore new treatment targets, transcriptomics, proteomics and histol. methods were applied to a surgically induced subchronic reflux esophagitis model in Wistar rats after treatment with either omeprazole (PPI) or STW5, a herbal prepn. shown to ameliorate esophagitis without affecting refluxate pH.  The normal human esophageal squamous cell line HET-1A and human endoscopic biopsies were used to confirm our findings to the G-protein-coupled receptor (GPR) 84 in human tissue.  Both treatments reduced reflux-induced macroscopic and microscopic lesions of the esophagi as well as known proinflammatory cytokines.  Proteomic and transcriptomic analyses identified CINC1-3, MIP-1/3α, MIG, RANTES and interleukin (IL)-1β as prominent mediators in GERD.  Most regulated cyto-/chemokines are linked to the TREM-1 signaling pathway.  The fatty acid receptor GPR84 was upregulated in esophagitis but significantly decreased in treated groups, a finding supported by Western blot and immunohistochem. in both rat tissue and HET-1A cells.  GPR84 was also found to be significantly upregulated in patients with grade B reflux esophagitis.  The expression of GPR84 in esophageal tissue and its potential involvement in GERD are reported for the first time.  IL-8 (CINC1-3) and the TREM-1 signaling pathway are proposed, besides GPR84, to play an important role in the pathogenesis of GERD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGv2akUPMqbbVg90H21EOLACvtfcHk0lgS5NC8aLailA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKlsLbI&md5=d626c0592ea9ac67a3700ec6b2138383</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2119%2Fmolmed.2015.00098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2119%252Fmolmed.2015.00098%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Aziz%26aufirst%3DH.%26aulast%3DSchneider%26aufirst%3DM.%26aulast%3DNeuhuber%26aufirst%3DW.%26aulast%3DKassem%26aufirst%3DA.%2BM.%26aulast%3DKhailah%26aufirst%3DS.%26aulast%3DMuller%26aufirst%3DJ.%26aulast%3DEldeen%26aufirst%3DH.%2BG.%26aulast%3DKhairy%26aufirst%3DA.%26aulast%3DKhayyal%26aufirst%3DM.%2BT.%26aulast%3DShcherbakova%26aufirst%3DA.%26aulast%3DEfferth%26aufirst%3DT.%26aulast%3DUlrich-Merzenich%26aufirst%3DG.%26atitle%3DGPR84%2520and%2520TREM-1%2520signaling%2520contribute%2520to%2520the%2520pathogenesis%2520of%2520reflux%2520esophagitis%26jtitle%3DMol.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D1011%26epage%3D1024%26doi%3D10.2119%2Fmolmed.2015.00098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arijs, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanque, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laukens, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nys, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccotti, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merciris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mate, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parent, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vriendt, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labeguere, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutgeerts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandeghinste, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeire, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brys, R.</span></span> <span> </span><span class="NLM_article-title">GPR84 inhibition as a novel therapeutic approach in IBD: mechanistic and translational studies</span>. <i>J. Crohns Colitis</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">S92</span>– <span class="NLM_lpage">S93</span>, <span class="refDoi"> DOI: 10.1093/ecco-jcc/jju027.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1093%2Fecco-jcc%2Fjju027.149" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2015&pages=S92-S93&author=S.+Dupontauthor=I.+Arijsauthor=R.+Blanqueauthor=D.+Laukensauthor=K.+Nysauthor=M.+C.+Ceccottiauthor=D.+Mercirisauthor=S.+De+Vosauthor=O.+Mateauthor=I.+Parentauthor=V.+De+Vriendtauthor=F.+Labeguereauthor=R.+Galienauthor=M.+Devosauthor=P.+Rutgeertsauthor=N.+Vandeghinsteauthor=S.+Vermeireauthor=R.+Brys&title=GPR84+inhibition+as+a+novel+therapeutic+approach+in+IBD%3A+mechanistic+and+translational+studies&doi=10.1093%2Fecco-jcc%2Fjju027.149"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1093%2Fecco-jcc%2Fjju027.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fecco-jcc%252Fjju027.149%26sid%3Dliteratum%253Aachs%26aulast%3DDupont%26aufirst%3DS.%26aulast%3DArijs%26aufirst%3DI.%26aulast%3DBlanque%26aufirst%3DR.%26aulast%3DLaukens%26aufirst%3DD.%26aulast%3DNys%26aufirst%3DK.%26aulast%3DCeccotti%26aufirst%3DM.%2BC.%26aulast%3DMerciris%26aufirst%3DD.%26aulast%3DDe%2BVos%26aufirst%3DS.%26aulast%3DMate%26aufirst%3DO.%26aulast%3DParent%26aufirst%3DI.%26aulast%3DDe%2BVriendt%26aufirst%3DV.%26aulast%3DLabeguere%26aufirst%3DF.%26aulast%3DGalien%26aufirst%3DR.%26aulast%3DDevos%26aufirst%3DM.%26aulast%3DRutgeerts%26aufirst%3DP.%26aulast%3DVandeghinste%26aufirst%3DN.%26aulast%3DVermeire%26aufirst%3DS.%26aulast%3DBrys%26aufirst%3DR.%26atitle%3DGPR84%2520inhibition%2520as%2520a%2520novel%2520therapeutic%2520approach%2520in%2520IBD%253A%2520mechanistic%2520and%2520translational%2520studies%26jtitle%3DJ.%2520Crohns%2520Colitis%26date%3D2015%26volume%3D9%26spage%3DS92%26epage%3DS93%26doi%3D10.1093%2Fecco-jcc%2Fjju027.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicol, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Russa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Didangelos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malcangio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, S. B.</span></span> <span> </span><span class="NLM_article-title">The role of Gprotein-receptor 84 in experimental neuropathic pain</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">8959</span>– <span class="NLM_lpage">8969</span>, <span class="refDoi"> DOI: 10.1523/JNEUROSCI.3558-14.2015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1523%2FJNEUROSCI.3558-14.2015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=26063927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOqsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=8959-8969&author=L.+S.+Nicolauthor=J.+M.+Dawesauthor=F.+La+Russaauthor=A.+Didangelosauthor=A.+K.+Clarkauthor=C.+Gentryauthor=J.+Gristauthor=J.+B.+Daviesauthor=M.+Malcangioauthor=S.+B.+McMahon&title=The+role+of+Gprotein-receptor+84+in+experimental+neuropathic+pain&doi=10.1523%2FJNEUROSCI.3558-14.2015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The role of G-protein receptor 84 in experimental neuropathic pain</span></div><div class="casAuthors">Nicol, Louise S. C.; Dawes, John M.; La Russa, Federica; Didangelos, Athanasios; Clark, Anna K.; Gentry, Clive; Grist, John; Davies, John B.; Malcangio, Marzia; McMahon, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8959-8969</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">G-protein receptor 84 (GPR84) is an orphan receptor that is induced markedly in monocytes/macrophages and microglia during inflammation, but its pathophysiol. function is unknown.  Here, we investigate the role of GPR84 in a murine model of traumatic nerve injury.  Naive GPR84 knock-out (KO) mice exhibited normal behavioral responses to acute noxious stimuli, but subsequent to partial sciatic nerve ligation (PNL), KOs did not develop mech. or thermal hypersensitivity, in contrast to wild-type (WT) littermates.  Nerve injury increased ionized calcium binding adapter mol. 1 (Iba1) and phosphorylated p38 MAPK immunoreactivity in the dorsal horn and Iba1 and cluster of differentiation 45 expression in the sciatic nerve, with no difference between genotypes.  PCR array anal. revealed that Gpr84 expression was upregulated in the spinal cord and sciatic nerve of WT mice.  In addn., the expression of arginase-1, a marker for anti-inflammatory macrophages, was upregulated in KO sciatic nerve.  Based on this evidence, we investigated whether peripheral macrophages behave differently in the absence of GPR84.  We found that lipopolysaccharide-stimulated KO macrophages exhibited attenuated expression of several proinflammatory mediators, including IL-1β, IL-6, and TNF-α.  Forskolin-stimulated KO macrophages also showed greater cAMP induction, a second messenger assocd. with immunosuppression.  In summary, our results demonstrate that GPR84 is a proinflammatory receptor that contributes to nociceptive signaling via the modulation of macrophages, whereas in its absence the response of these cells to an inflammatory insult is impaired.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEz_UEJb4WPbVg90H21EOLACvtfcHk0lie0gRoD4nxyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOqsLrJ&md5=4d7a16fa2471aaaa6ed9395ad1215544</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.3558-14.2015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.3558-14.2015%26sid%3Dliteratum%253Aachs%26aulast%3DNicol%26aufirst%3DL.%2BS.%26aulast%3DDawes%26aufirst%3DJ.%2BM.%26aulast%3DLa%2BRussa%26aufirst%3DF.%26aulast%3DDidangelos%26aufirst%3DA.%26aulast%3DClark%26aufirst%3DA.%2BK.%26aulast%3DGentry%26aufirst%3DC.%26aulast%3DGrist%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DJ.%2BB.%26aulast%3DMalcangio%26aufirst%3DM.%26aulast%3DMcMahon%26aufirst%3DS.%2BB.%26atitle%3DThe%2520role%2520of%2520Gprotein-receptor%252084%2520in%2520experimental%2520neuropathic%2520pain%26jtitle%3DJ.%2520Neurosci.%26date%3D2015%26volume%3D35%26spage%3D8959%26epage%3D8969%26doi%3D10.1523%2FJNEUROSCI.3558-14.2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Audoy-Remus, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecours, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivest, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, M.-E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallieres, L.</span></span> <span> </span><span class="NLM_article-title">GPR84 deficiency reduces microgliosis, but accelerates dendritic degeneration and cognitive decline in a mouse model of Alzheimer’s disease</span>. <i>Brain, Behav., Immun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1016/j.bbi.2015.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bbi.2015.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25637481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitFCntbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2015&pages=112-120&author=J.+Audoy-Remusauthor=L.+Bozoyanauthor=A.+Dumasauthor=M.+Filaliauthor=C.+Lecoursauthor=S.+Lacroixauthor=S.+Rivestauthor=M.-E.+Tremblayauthor=L.+Vallieres&title=GPR84+deficiency+reduces+microgliosis%2C+but+accelerates+dendritic+degeneration+and+cognitive+decline+in+a+mouse+model+of+Alzheimer%E2%80%99s+disease&doi=10.1016%2Fj.bbi.2015.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">GPR84 deficiency reduces microgliosis, but accelerates dendritic degeneration and cognitive decline in a mouse model of Alzheimer's disease</span></div><div class="casAuthors">Audoy-Remus, Julie; Bozoyan, Lusine; Dumas, Aline; Filali, Mohammed; Lecours, Cynthia; Lacroix, Steve; Rivest, Serge; Tremblay, Marie-Eve; Vallieres, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Brain, Behavior, and Immunity</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112-120</span>CODEN:
                <span class="NLM_cas:coden">BBIMEW</span>;
        ISSN:<span class="NLM_cas:issn">0889-1591</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Microglia surrounds the amyloid plaques that form in the brains of patients with Alzheimer's disease (AD), but their role is controversial.  Under inflammatory conditions, these cells can express GPR84, an orphan receptor whose pathophysiol. role is unknown.  Here, we report that GPR84 is upregulated in microglia of APP/PS1 transgenic mice, a model of AD.  Without GPR84, these mice display both accelerated cognitive decline and a reduced no. of microglia, esp. in areas surrounding plaques.  The lack of GPR84 affects neither plaque formation nor hippocampal neurogenesis, but promotes dendritic degeneration.  Furthermore, GPR84 does not influence the clin. progression of other diseases in which its expression has been reported, i.e., exptl. autoimmune encephalomyelitis (EAE) and endotoxic shock.  We conclude that GPR84 plays a beneficial role in amyloid pathol. by acting as a sensor for a yet unknown ligand that promotes microglia recruitment, a response affecting dendritic degeneration and required to prevent further cognitive decline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWBqsDH3-KvrVg90H21EOLACvtfcHk0ljqypaxdemEMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitFCntbo%253D&md5=4b643287374786cdde611ffbb3002101</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbi.2015.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbi.2015.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DAudoy-Remus%26aufirst%3DJ.%26aulast%3DBozoyan%26aufirst%3DL.%26aulast%3DDumas%26aufirst%3DA.%26aulast%3DFilali%26aufirst%3DM.%26aulast%3DLecours%26aufirst%3DC.%26aulast%3DLacroix%26aufirst%3DS.%26aulast%3DRivest%26aufirst%3DS.%26aulast%3DTremblay%26aufirst%3DM.-E.%26aulast%3DVallieres%26aufirst%3DL.%26atitle%3DGPR84%2520deficiency%2520reduces%2520microgliosis%252C%2520but%2520accelerates%2520dendritic%2520degeneration%2520and%2520cognitive%2520decline%2520in%2520a%2520mouse%2520model%2520of%2520Alzheimer%25E2%2580%2599s%2520disease%26jtitle%3DBrain%252C%2520Behav.%252C%2520Immun.%26date%3D2015%26volume%3D46%26spage%3D112%26epage%3D120%26doi%3D10.1016%2Fj.bbi.2015.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchigami, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ota, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, Y.</span></span> <span> </span><span class="NLM_article-title">Inflammatory changes in adipose tissue enhance expression of GPR84, a medium-chain fatty acid receptor: TNFα enhances GPR84 expression in adipocytes</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>586</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2012.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.febslet.2012.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=22245676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1yqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=586&publication_year=2012&pages=368-372&author=H.+Nagasakiauthor=T.+Kondoauthor=M.+Fuchigamiauthor=H.+Hashimotoauthor=Y.+Sugimuraauthor=N.+Ozakiauthor=H.+Arimaauthor=A.+Otaauthor=Y.+Oisoauthor=Y.+Hamada&title=Inflammatory+changes+in+adipose+tissue+enhance+expression+of+GPR84%2C+a+medium-chain+fatty+acid+receptor%3A+TNF%CE%B1+enhances+GPR84+expression+in+adipocytes&doi=10.1016%2Fj.febslet.2012.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammatory changes in adipose tissue enhance expression of GPR G-protein coupled receptor84, a medium-chain fatty acid receptor</span></div><div class="casAuthors">Nagasaki, Hiroshi; Kondo, Takaaki; Fuchigami, Masahiro; Hashimoto, Hiroyuki; Sugimura, Yoshihisa; Ozaki, Nobuaki; Arima, Hiroshi; Ota, Akira; Oiso, Yutaka; Hamada, Yoji</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">586</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this study we aimed to identify the physiol. roles of G protein-coupled receptor 84 (GPR84) in adipose tissue, together with medium-chain fatty acids (MCFAs), the specific ligands for GPR84.  In mice, high-fat diet up-regulated GPR84 expression in fat pads.  In 3T3-L1 adipocytes, coculture with a macrophage cell line, RAW264, or TNFα remarkably enhanced GPR84 expression.  In the presence of TNFα, MCFAs down-regulated adiponectin mRNA expression in 3T3-L1 adipocytes.  Taken together, our results suggest that GPR84 emerges in adipocytes in response to TNFα from infiltrating macrophages and exacerbates the vicious cycle between adiposity and diabesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiwUz5s6M9zLVg90H21EOLACvtfcHk0ljqypaxdemEMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1yqu7w%253D&md5=8fecdea5b404e472d6f66a9ed22cd88b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2012.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2012.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DNagasaki%26aufirst%3DH.%26aulast%3DKondo%26aufirst%3DT.%26aulast%3DFuchigami%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DSugimura%26aufirst%3DY.%26aulast%3DOzaki%26aufirst%3DN.%26aulast%3DArima%26aufirst%3DH.%26aulast%3DOta%26aufirst%3DA.%26aulast%3DOiso%26aufirst%3DY.%26aulast%3DHamada%26aufirst%3DY.%26atitle%3DInflammatory%2520changes%2520in%2520adipose%2520tissue%2520enhance%2520expression%2520of%2520GPR84%252C%2520a%2520medium-chain%2520fatty%2520acid%2520receptor%253A%2520TNF%25CE%25B1%2520enhances%2520GPR84%2520expression%2520in%2520adipocytes%26jtitle%3DFEBS%2520Lett.%26date%3D2012%26volume%3D586%26spage%3D368%26epage%3D372%26doi%3D10.1016%2Fj.febslet.2012.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellendorph, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansen, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräuner-Osborne, H.</span></span> <span> </span><span class="NLM_article-title">Molecular pharmacology of promiscuous seven transmembrane receptors sensing organic nutrients</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1124/mol.109.055244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1124%2Fmol.109.055244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=19487246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvVyrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2009&pages=453-465&author=P.+Wellendorphauthor=L.+D.+Johansenauthor=H.+Br%C3%A4uner-Osborne&title=Molecular+pharmacology+of+promiscuous+seven+transmembrane+receptors+sensing+organic+nutrients&doi=10.1124%2Fmol.109.055244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular pharmacology of promiscuous seven transmembrane receptors sensing organic nutrients</span></div><div class="casAuthors">Wellendorph, Petrine; Dan Johansen, Lars; Brauner-Osborne, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">453-465</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  A no. of highly promiscuous 7 transmembrane (7TM) receptors were cloned and characterized within the last few years.  It is noteworthy that many of these receptors are activated broadly by amino acids, proteolytic degrdn. products, carbohydrates, or free fatty acids and are expressed in taste tissue, the gastrointestinal tract, endocrine glands, adipose tissue, and/or kidney.  These receptors thus hold the potential to act as sensors of food intake, regulating, for example, release of incretin hormones from the gut, insulin/glucagon from the pancreas, and leptin from adipose tissue.  The promiscuous tendency in ligand recognition of these receptors is in contrast to the typical specific interaction with one physiol. agonist seen for most receptors, which challenges the classic "lock-and-key" concept.  We here review the mol. mechanisms of nutrient sensing of the calcium-sensing receptor, the G protein-coupled receptor family C, group 6, subtype A (GPRC6A), and the taste1 receptor T1R1/T1R3, which are sensing L-α-amino acids, the carbohydrate-sensing T1R2/T1R3 receptor, the proteolytic degrdn. product sensor GPR93 (also termed GPR92), and the free fatty acid (FFA) sensing receptors FFA1, FFA2, FFA3, GPR84, and GPR120.  The involvement of the individual receptors in sensing of food intake was validated to different degrees because of limited availability of specific pharmacol. tools and/or receptor knockout mice.  However, as a group, the receptors represent potential drug targets, to treat, for example, type II diabetes by mimicking food intake by potent agonists or pos. allosteric modulators.  The ligand-receptor interactions of the promiscuous receptors of org. nutrients thus remain an interesting subject of emerging functional importance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMz4I0wYJgi7Vg90H21EOLACvtfcHk0ljqypaxdemEMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvVyrs7Y%253D&md5=6dd526aa25e07a85ad9e4e5f45ce0a72</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1124%2Fmol.109.055244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.109.055244%26sid%3Dliteratum%253Aachs%26aulast%3DWellendorph%26aufirst%3DP.%26aulast%3DJohansen%26aufirst%3DL.%2BD.%26aulast%3DBr%25C3%25A4uner-Osborne%26aufirst%3DH.%26atitle%3DMolecular%2520pharmacology%2520of%2520promiscuous%2520seven%2520transmembrane%2520receptors%2520sensing%2520organic%2520nutrients%26jtitle%3DMol.%2520Pharmacol.%26date%3D2009%26volume%3D76%26spage%3D453%26epage%3D465%26doi%3D10.1124%2Fmol.109.055244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yonezawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murayama, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, A.</span></span> <span> </span><span class="NLM_article-title">Free fatty acids-sensing G protein-coupled receptors in drug targeting and therapeutics</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3855</span>– <span class="NLM_lpage">3871</span>, <span class="refDoi"> DOI: 10.2174/09298673113209990168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.2174%2F09298673113209990168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23862620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=3855-3871&author=T.+Yonezawaauthor=R.+Kurataauthor=K.+Yoshidaauthor=M.+A.+Murayamaauthor=X.+Cuiauthor=A.+Hasegawa&title=Free+fatty+acids-sensing+G+protein-coupled+receptors+in+drug+targeting+and+therapeutics&doi=10.2174%2F09298673113209990168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Free Fatty Acids-Sensing G Protein-Coupled Receptors in Drug Targeting and Therapeutics</span></div><div class="casAuthors">Yonezawa, Tomo; Kurata, Riho; Yoshida, Kaori; Murayama, Masanori A.; Cui, Xiaofeng; Hasegawa, Akihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">3855-3871</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptor (GPCR) (also known as seven-transmembrane domain receptor) superfamily represents the largest protein family in the human genome.  These receptors respond to various physiol. ligands such as photons, odors, pheromones, hormones, ions, and small mols. including amines, amino acids to large peptides and steroids.  Thus, GPCRs are involved in many diseases and the target of around half of all conventional drugs.  The physiol. roles of free fatty acids (FFAs), in particular, long-chain FFAs, are important for the development of many metabolic disease including obesity, diabetes, and atherosclerosis.  In the past half decade, deorphanization of several GPCRs has revealed that GPR40, GPR41, GPR43, GPR84 and GPR120 sense concn. of extracellular FFAs with various carbon chain lengths.  GPR40 and GPR120 are activated by medium- and long-chain FFAs.  GPR84 is activated by medium- chain, but not long-chain, FFAs.  GPR41 and GPR43 are activated by short-chain FFAs.  GPR40 is highly expressed in pancreatic beta cells and plays a crucial role in FFAs-induced insulin secretion.  GPR120 is mainly expressed in enteroendocrine cells and plays an important role for FFAs-induced glucagon-like peptide-1.  GPR43 is abundant in leukocytes and adipose tissue, while GPR41 is highly expressed in adipose tissue, the pancreas and leukocytes.  GPR84 is expressed in leukocytes and monocyte/macrophage.  This review aims to shed light on the physiol. roles and development of drugs targeting these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyygc49JEvWLVg90H21EOLACvtfcHk0ljPzdolkM44aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGnsrjM&md5=6c546f83c25e551b7a557909e713c5e8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.2174%2F09298673113209990168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F09298673113209990168%26sid%3Dliteratum%253Aachs%26aulast%3DYonezawa%26aufirst%3DT.%26aulast%3DKurata%26aufirst%3DR.%26aulast%3DYoshida%26aufirst%3DK.%26aulast%3DMurayama%26aufirst%3DM.%2BA.%26aulast%3DCui%26aufirst%3DX.%26aulast%3DHasegawa%26aufirst%3DA.%26atitle%3DFree%2520fatty%2520acids-sensing%2520G%2520protein-coupled%2520receptors%2520in%2520drug%2520targeting%2520and%2520therapeutics%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2013%26volume%3D20%26spage%3D3855%26epage%3D3871%26doi%3D10.2174%2F09298673113209990168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasubuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichimura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, I.</span></span> <span> </span><span class="NLM_article-title">Nutritional signaling via free fatty acid receptors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.3390/ijms17040450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.3390%2Fijms17040450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=27023530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslCmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=450-462&author=J.+Miyamotoauthor=S.+Hasegawaauthor=M.+Kasubuchiauthor=A.+Ichimuraauthor=A.+Nakajimaauthor=I.+Kimura&title=Nutritional+signaling+via+free+fatty+acid+receptors&doi=10.3390%2Fijms17040450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nutritional signaling via free fatty acid receptors</span></div><div class="casAuthors">Miyamoto, Junki; Hasegawa, Sae; Kasubuchi, Mayu; Ichimura, Atsuhiko; Nakajima, Akira; Kimura, Ikuo</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">450/1-450/12</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Excess energy is stored primarily as triglycerides, which are mobilized when demand for energy arises.  Dysfunction of energy balance by excess food intake leads to metabolic diseases, such as obesity and diabetes.  Free fatty acids (FFAs) provided by dietary fat are not only important nutrients, but also contribute key physiol. functions via FFA receptor (FFAR)-mediated signaling mols., which depend on FFAs' carbon chain length and the ligand specificity of the receptors.  Functional analyses have revealed that FFARs are crit. for metabolic functions, such as peptide hormone secretion and inflammation, and contribute to energy homeostasis.  In particular, recent studies have shown that the administration of selective agonists of G protein-coupled receptor (GPR) 40 and GPR120 improved glucose metab. and systemic metabolic disorders.  Furthermore, the anti-inflammation and energy metab. effects of short chain FAs have been linked to the activation of GPR41 and GPR43.  In this review, we summarize recent progress in research on FFAs and their physiol. roles in the regulation of energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PqYSYQi3q7Vg90H21EOLACvtfcHk0ljPzdolkM44aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslCmsb4%253D&md5=f78fdb70502dc2a96a156068edcb9637</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3390%2Fijms17040450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17040450%26sid%3Dliteratum%253Aachs%26aulast%3DMiyamoto%26aufirst%3DJ.%26aulast%3DHasegawa%26aufirst%3DS.%26aulast%3DKasubuchi%26aufirst%3DM.%26aulast%3DIchimura%26aufirst%3DA.%26aulast%3DNakajima%26aufirst%3DA.%26aulast%3DKimura%26aufirst%3DI.%26atitle%3DNutritional%2520signaling%2520via%2520free%2520fatty%2520acid%2520receptors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26spage%3D450%26epage%3D462%26doi%3D10.3390%2Fijms17040450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, H. H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haber, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Y.</span></span> <span> </span><span class="NLM_article-title">GPR84 sustains aberrant beta-catenin signaling in leukemic stem cells for maintenance of MLL leukemogenesis</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">3284</span>– <span class="NLM_lpage">3294</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-10-532523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1182%2Fblood-2013-10-532523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25293777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVSgs73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=3284-3294&author=P.+A.+Dietrichauthor=C.+Yangauthor=H.+H.+L.+Leungauthor=J.+R.+Lynchauthor=E.+Gonzalesauthor=B.+Liuauthor=M.+Haberauthor=M.+D.+Norrisauthor=J.+L.+Wangauthor=J.+Y.+Wang&title=GPR84+sustains+aberrant+beta-catenin+signaling+in+leukemic+stem+cells+for+maintenance+of+MLL+leukemogenesis&doi=10.1182%2Fblood-2013-10-532523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">GPR84 sustains aberrant β-catenin signaling in leukemic stem cells for maintenance of MLL leukemogenesis</span></div><div class="casAuthors">Dietrich, Philipp A.; Yang, Chen; Leung, Halina H. L.; Lynch, Jennifer R.; Gonzales, Estrella; Liu, Bing; Haber, Michelle; Norris, Murray D.; Wang, Jianlong; Wang, Jenny Yingzi</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3284-3294</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">β-Catenin is required for establishment of leukemic stem cells (LSCs) in acute myeloid leukemia (AML).  Targeted inhibition of β-catenin signaling has been hampered by the lack of pathway components amenable to pharmacol. manipulation.  Here we identified a novel β-catenin regulator, GPR84, a member of the G protein-coupled receptor family that represents a highly tractable class of drug targets.  High GPR84 expression levels were confirmed in human and mouse AML LSCs compared with hematopoietic stem cells (HSCs).  Suppression of GPR84 significantly inhibited cell growth by inducing G1-phase cell-cycle arrest in pre-LSCs, reduced LSC frequency, and impaired reconstitution of stem cell-derived mixed-lineage leukemia (MLL) AML, which represents an aggressive and drug-resistant subtype of AML.  The GPR84-deficient phenotype in established AML could be rescued by expression of constitutively active β-catenin.  Furthermore, GPR84 conferred a growth advantage to Hoxa9/Meis1a-transduced stem cells.  Microarray anal. demonstrated that GPR84 significantly upregulated a small set of MLL-fusion targets and β-catenin coeffectors, and downregulated a hematopoietic cell-cycle inhibitor.  Altogether, our data reveal a previously unrecognized role of GPR84 in maintaining fully developed AML by sustaining aberrant β-catenin signaling in LSCs, and suggest that targeting the oncogenic GPR84/β-catenin signaling axis may represent a novel therapeutic strategy for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqayOBoeIKDt7Vg90H21EOLACvtfcHk0ljPzdolkM44aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVSgs73L&md5=3da07da89bd6da728d09aa26b273d659</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-10-532523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-10-532523%26sid%3Dliteratum%253Aachs%26aulast%3DDietrich%26aufirst%3DP.%2BA.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLeung%26aufirst%3DH.%2BH.%2BL.%26aulast%3DLynch%26aufirst%3DJ.%2BR.%26aulast%3DGonzales%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHaber%26aufirst%3DM.%26aulast%3DNorris%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DJ.%2BL.%26aulast%3DWang%26aufirst%3DJ.%2BY.%26atitle%3DGPR84%2520sustains%2520aberrant%2520beta-catenin%2520signaling%2520in%2520leukemic%2520stem%2520cells%2520for%2520maintenance%2520of%2520MLL%2520leukemogenesis%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D3284%26epage%3D3294%26doi%3D10.1182%2Fblood-2013-10-532523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seong, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span> <span> </span><span class="NLM_article-title">G protein-coupled receptor 84 controls osteoclastogenesis through inhibition of NF-κB and MAPK signaling pathways</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>233</i></span>,  <span class="NLM_fpage">1481</span>– <span class="NLM_lpage">1489</span>, <span class="refDoi"> DOI: 10.1002/jcp.26035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fjcp.26035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=28574596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFSmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=233&publication_year=2018&pages=1481-1489&author=J.+W.+Parkauthor=H.+J.+Yoonauthor=W.+Y.+Kangauthor=S.+Choauthor=S.+J.+Seongauthor=H.+W.+Leeauthor=Y.+R.+Yoonauthor=H.+J.+Kim&title=G+protein-coupled+receptor+84+controls+osteoclastogenesis+through+inhibition+of+NF-%CE%BAB+and+MAPK+signaling+pathways&doi=10.1002%2Fjcp.26035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">G protein-coupled receptor 84 controls osteoclastogenesis through inhibition of NF-κB and MAPK signaling pathways</span></div><div class="casAuthors">Park, Ji-Wan; Yoon, Hye-Jin; Kang, Woo Youl; Cho, Seungil; Seong, Sook Jin; Lee, Hae Won; Yoon, Young-Ran; Kim, Hyun-Ju</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1481-1489</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">GPR84, a member of the G protein-coupled receptor family, is found predominantly in immune cells, such as macrophages, and functions as a pivotal modulator of inflammatory responses.  In this study, we investigated the role of GPR84 in receptor activator of nuclear factor-κB ligand (RANKL)-induced osteoclast differentiation.  Our microarray data showed that GPR84 was significantly downregulated in osteoclasts compared to in their precursors, macrophages.  The overexpression of GPR84 in bone marrow-derived macrophages suppressed the formation of multinucleated osteoclasts without affecting precursor proliferation.  In addn., GPR84 overexpression attenuated the induction of c-Fos and nuclear factor of activated T cells, cytoplasmic 1 (NFATc1), which are transcription factors that are crit. for osteoclastogenesis.  Furthermore, knockdown of GPR84 using a small hairpin RNA promoted RANKL-mediated osteoclast differentiation and gene expression of osteoclastogenic markers.  Mechanistically, GPR84 overexpression blocked RANKL-stimulated phosphorylation of IκBα and three MAPKs, JNK, ERK, and p38.  GPR84 also suppressed NF-κB transcriptional activity mediated by RANKL.  Conversely, GPR84 knockdown enhanced RANKL-induced activation of IκBα and the three MAPKs.  Collectively, our results revealed that GPR84 functions as a neg. regulator of osteoclastogenesis, suggesting that it may be a potential therapeutic target for osteoclast-mediated bone-destructive diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvjBAcWdjpLrVg90H21EOLACvtfcHk0li3YxkCn8zOew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSmtbvM&md5=fb81b70f841ab32cc66e898a3810405d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fjcp.26035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.26035%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%2BW.%26aulast%3DYoon%26aufirst%3DH.%2BJ.%26aulast%3DKang%26aufirst%3DW.%2BY.%26aulast%3DCho%26aufirst%3DS.%26aulast%3DSeong%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DH.%2BW.%26aulast%3DYoon%26aufirst%3DY.%2BR.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26atitle%3DG%2520protein-coupled%2520receptor%252084%2520controls%2520osteoclastogenesis%2520through%2520inhibition%2520of%2520NF-%25CE%25BAB%2520and%2520MAPK%2520signaling%2520pathways%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D2018%26volume%3D233%26spage%3D1481%26epage%3D1489%26doi%3D10.1002%2Fjcp.26035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibodeau, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grouix, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bournet- Sarra, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hince, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tremblay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerts, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hebert, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutsol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holterman, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gervais, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouboudinar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laverdure, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letournau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblond, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbot, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penney, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duceppe, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacharie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupuis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurin, P.</span></span> <span> </span><span class="NLM_article-title">A newly discovered antifibrotic pathway regulated by two fatty acid receptors</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>188</i></span>,  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1148</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2018.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.ajpath.2018.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29454750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVGqs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2018&pages=1132-1148&author=L.+Gagnonauthor=M.+Leducauthor=J.+F.+Thibodeauauthor=M.+Z.+Zhangauthor=B.+Grouixauthor=F.+Bournet-+Sarraauthor=W.+Gagnonauthor=K.+Hinceauthor=M.+Tremblayauthor=L.+Geertsauthor=C.+R.+J.+Kennedyauthor=R.+L.+Hebertauthor=A.+Gutsolauthor=C.+E.+Holtermanauthor=E.+Kamtoauthor=L.+Gervaisauthor=J.+Ouboudinarauthor=J.+Richardauthor=A.+Feltonauthor=A.+Laverdureauthor=J.+C.+Simardauthor=S.+Letournauauthor=M.+P.+Cloutierauthor=F.+A.+Leblondauthor=S.+D.+Abbotauthor=C.+Penneyauthor=J.+S.+Duceppeauthor=B.+Zacharieauthor=J.+Dupuisauthor=A.+Calderoneauthor=Q.+T.+Nguyenauthor=R.+C.+Harrisauthor=P.+Laurin&title=A+newly+discovered+antifibrotic+pathway+regulated+by+two+fatty+acid+receptors&doi=10.1016%2Fj.ajpath.2018.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Newly Discovered Antifibrotic Pathway Regulated by Two Fatty Acid Receptors</span></div><div class="casAuthors">Gagnon, Lyne; Leduc, Martin; Thibodeau, Jean-Francois; Zhang, Ming-Zhi; Grouix, Brigitte; Sarra-Bournet, Francois; Gagnon, William; Hince, Kathy; Tremblay, Mikael; Geerts, Lilianne; Kennedy, Christopher R. J.; Hebert, Richard L.; Gutsol, Alex; Holterman, Chet E.; Kamto, Eldjonai; Gervais, Liette; Ouboudinar, Jugurtha; Richard, Jonathan; Felton, Alexandra; Laverdure, Alexandre; Simard, Jean-Christophe; Letourneau, Sylvie; Cloutier, Marie-Pier; Leblond, Francois A.; Abbott, Shaun D.; Penney, Christopher; Duceppe, Jean-Simon; Zacharie, Boulos; Dupuis, Jocelyn; Calderone, Angelino; Nguyen, Quang T.; Harris, Raymond C.; Laurin, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">188</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1132-1148</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Numerous clin. conditions can lead to organ fibrosis and functional failure.  There is a great need for therapies that could effectively target pathophysiol. pathways involved in fibrosis.  GPR40 and GPR84 are G protein-coupled receptors with free fatty acid ligands and are assocd. with metabolic and inflammatory disorders.  Although GPR40 and GPR84 are involved in diverse physiol. processes, no evidence has demonstrated the relevance of GPR40 and GPR84 in fibrosis pathways.  Using PBI-4050 (3-pentylbenzeneacetic acid sodium salt), a synthetic analog of a medium-chain fatty acid that displays agonist and antagonist ligand affinity toward GPR40 and GPR84, resp., we uncovered an antifibrotic pathway involving these receptors.  In expts. using Gpr40- and Gpr84-knockout mice in models of kidney fibrosis (unilateral ureteral obstruction, long-term post-acute ischemic injury, and adenine-induced chronic kidney disease), we found that GPR40 is protective and GPR84 is deleterious in these diseases.  Moreover, through binding to GPR40 and GPR84, PBI-4050 significantly attenuated fibrosis in many injury contexts, as evidenced by the antifibrotic activity obsd. in kidney, liver, heart, lung, pancreas, and skin fibrosis models.  Therefore, GPR40 and GPR84 may represent promising mol. targets in fibrosis pathways.  We conclude that PBI-4050 is a first-in-class compd. that may be effective for managing inflammatory and fibrosis-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogzjSwncX3zLVg90H21EOLACvtfcHk0li3YxkCn8zOew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVGqs7w%253D&md5=47fff3190f70beafa69897d9d89f206d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2018.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2018.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DGagnon%26aufirst%3DL.%26aulast%3DLeduc%26aufirst%3DM.%26aulast%3DThibodeau%26aufirst%3DJ.%2BF.%26aulast%3DZhang%26aufirst%3DM.%2BZ.%26aulast%3DGrouix%26aufirst%3DB.%26aulast%3DBournet-%2BSarra%26aufirst%3DF.%26aulast%3DGagnon%26aufirst%3DW.%26aulast%3DHince%26aufirst%3DK.%26aulast%3DTremblay%26aufirst%3DM.%26aulast%3DGeerts%26aufirst%3DL.%26aulast%3DKennedy%26aufirst%3DC.%2BR.%2BJ.%26aulast%3DHebert%26aufirst%3DR.%2BL.%26aulast%3DGutsol%26aufirst%3DA.%26aulast%3DHolterman%26aufirst%3DC.%2BE.%26aulast%3DKamto%26aufirst%3DE.%26aulast%3DGervais%26aufirst%3DL.%26aulast%3DOuboudinar%26aufirst%3DJ.%26aulast%3DRichard%26aufirst%3DJ.%26aulast%3DFelton%26aufirst%3DA.%26aulast%3DLaverdure%26aufirst%3DA.%26aulast%3DSimard%26aufirst%3DJ.%2BC.%26aulast%3DLetournau%26aufirst%3DS.%26aulast%3DCloutier%26aufirst%3DM.%2BP.%26aulast%3DLeblond%26aufirst%3DF.%2BA.%26aulast%3DAbbot%26aufirst%3DS.%2BD.%26aulast%3DPenney%26aufirst%3DC.%26aulast%3DDuceppe%26aufirst%3DJ.%2BS.%26aulast%3DZacharie%26aufirst%3DB.%26aulast%3DDupuis%26aufirst%3DJ.%26aulast%3DCalderone%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DQ.%2BT.%26aulast%3DHarris%26aufirst%3DR.%2BC.%26aulast%3DLaurin%26aufirst%3DP.%26atitle%3DA%2520newly%2520discovered%2520antifibrotic%2520pathway%2520regulated%2520by%2520two%2520fatty%2520acid%2520receptors%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2018%26volume%3D188%26spage%3D1132%26epage%3D1148%26doi%3D10.1016%2Fj.ajpath.2018.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Southern, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neetoo-Isseljee, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettleborough, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassoni, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehrman, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, S.</span></span> <span> </span><span class="NLM_article-title">Screening β-arrestin recruitment for the identification of natural ligands for orphan Gprotein-coupled receptors</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1177/1087057113475480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1177%2F1087057113475480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23396314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWisb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=599-609&author=C.+Southernauthor=J.+M.+Cookauthor=Z.+Neetoo-Isseljeeauthor=D.+L.+Taylorauthor=C.+A.+Kettleboroughauthor=A.+Merrittauthor=D.+L.+Bassoniauthor=W.+J.+Raabauthor=E.+Quinnauthor=T.+S.+Wehrmanauthor=A.+P.+Davenportauthor=A.+J.+Brownauthor=A.+Greenauthor=M.+J.+Wigglesworthauthor=S.+Rees&title=Screening+%CE%B2-arrestin+recruitment+for+the+identification+of+natural+ligands+for+orphan+Gprotein-coupled+receptors&doi=10.1177%2F1087057113475480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Screening β-arrestin recruitment for the identification of natural ligands for orphan G-protein-coupled receptors</span></div><div class="casAuthors">Southern, Craig; Cook, Jennifer M.; Neetoo-Isseljee, Zaynab; Taylor, Debra L.; Kettleborough, Catherine A.; Merritt, Andy; Bassoni, Daniel L.; Raab, William J.; Quinn, Elizabeth; Wehrman, Tom S.; Davenport, Anthony P.; Brown, Andrew J.; Green, Andrew; Wigglesworth, Mark J.; Rees, Steve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">599-609, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A variety of G-protein-coupled receptor (GPCR) screening technologies have successfully partnered a no. of GPCRs with their cognate ligands.  GPCR-mediated β-arrestin recruitment is now recognized as a distinct intracellular signaling pathway and ligand-receptor interactions may show a bias toward β-arrestin over classical GPCR signaling pathways.  We hypothesized that the failure to identify native ligands for the remaining orphan GPCRs may be a consequence of biased β-arrestin signaling.  To investigate this, we assembled 10 500 candidate ligands and screened 82 GPCRs using PathHunter β-arrestin recruitment technol.  High-quality screening assays were validated by the inclusion of liganded receptors and the detection and confirmation of these established ligand-receptor pairings.  We describe a candidate endogenous orphan GPCR ligand and a no. of novel surrogate ligands.  However, for the majority of orphan receptors studied, measurement of β-arrestin recruitment did not lead to the identification of cognate ligands from our screening sets. β-Arrestin recruitment represents a robust GPCR screening technol. and ligand-biased signaling is emerging as a therapeutically exploitable feature of GPCR biol.  The identification of cognate ligands for the orphan GPCRs and the extent to which receptors may exist to preferentially signal through β-arrestin in response to their native ligand remain to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRqw2KqMq-GrVg90H21EOLACvtfcHk0li3YxkCn8zOew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWisb%252FP&md5=2d52934eea58a90c12236001e951e107</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1087057113475480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113475480%26sid%3Dliteratum%253Aachs%26aulast%3DSouthern%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DJ.%2BM.%26aulast%3DNeetoo-Isseljee%26aufirst%3DZ.%26aulast%3DTaylor%26aufirst%3DD.%2BL.%26aulast%3DKettleborough%26aufirst%3DC.%2BA.%26aulast%3DMerritt%26aufirst%3DA.%26aulast%3DBassoni%26aufirst%3DD.%2BL.%26aulast%3DRaab%26aufirst%3DW.%2BJ.%26aulast%3DQuinn%26aufirst%3DE.%26aulast%3DWehrman%26aufirst%3DT.%2BS.%26aulast%3DDavenport%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DWigglesworth%26aufirst%3DM.%2BJ.%26aulast%3DRees%26aufirst%3DS.%26atitle%3DScreening%2520%25CE%25B2-arrestin%2520recruitment%2520for%2520the%2520identification%2520of%2520natural%2520ligands%2520for%2520orphan%2520Gprotein-coupled%2520receptors%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D599%26epage%3D609%26doi%3D10.1177%2F1087057113475480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span> <span> </span><span class="NLM_article-title">Mutation analysis and molecular modeling for the investigation of ligand-binding modes of GPR84</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">320</span>, <span class="refDoi"> DOI: 10.1093/jb/mvu075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1093%2Fjb%2Fmvu075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25425658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOgtLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2015&pages=311-320&author=Y.+Nikaidoauthor=Y.+Koyamaauthor=Y.+Yoshikawaauthor=T.+Furuyaauthor=S.+Takeda&title=Mutation+analysis+and+molecular+modeling+for+the+investigation+of+ligand-binding+modes+of+GPR84&doi=10.1093%2Fjb%2Fmvu075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Mutation analysis and molecular modeling for the investigation of ligand-binding modes of GPR84</span></div><div class="casAuthors">Nikaido, Yoshiaki; Koyama, Yuuta; Yoshikawa, Yasushi; Furuya, Toshio; Takeda, Shigeki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">311-320</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">GPR84 is a G protein-coupled receptor for medium-chain fatty acids.  Capric acid and 3,3'-diindolylmethane are specific agonists for GPR84.  We built a homol. model of a GPR84-capric acid complex to investigate the ligand-binding mode using the crystal structure of human active-state β2-adrenergic receptor.  We performed site-directed mutagenesis to subject ligand-binding sites to our model using GPR84-Giα fusion proteins and a [35S]GTPγS-binding assay.  We compared the activity of the wild type and mutated forms of GPR84 by [35S]GTPγS binding to capric acid and diindolylmethane.  The mutations L100D 'allesteros-Weinstein numbering: 3.32, F101Y 3.33 and N104Q 3.36 in the transmembrane helix III and N357D 7.39 in the transmembrane helix VII resulted in reduced capric acid activity but maintained the diindolylmethane responses.  Y186F 5.46 and Y186H 5.46 mutations had no characteristic effect on capric acid but with diindolylmethane they significantly affected the G protein activation efficiency.  The L100D 3.32 mutant responded to decylamine, a fatty amine, instead of a natural agonist, the fatty acid capric acid, suggesting that we have identified a mutated G protein-coupled receptor-artificial ligand pairing.  Our mol. model provides an explanation for these results and interactions between GPR84 and capric acid.  Further, from the results of a double stimulation assay, we concluded that diindolylmethane was a pos. allosteric modulator for GPR84.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpnoqt2xo2U7Vg90H21EOLACvtfcHk0lgrYdmnm1IBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOgtLzK&md5=b85955265588602441607c1cbd6a76ef</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvu075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvu075%26sid%3Dliteratum%253Aachs%26aulast%3DNikaido%26aufirst%3DY.%26aulast%3DKoyama%26aufirst%3DY.%26aulast%3DYoshikawa%26aufirst%3DY.%26aulast%3DFuruya%26aufirst%3DT.%26aulast%3DTakeda%26aufirst%3DS.%26atitle%3DMutation%2520analysis%2520and%2520molecular%2520modeling%2520for%2520the%2520investigation%2520of%2520ligand-binding%2520modes%2520of%2520GPR84%26jtitle%3DJ.%2520Biochem.%26date%3D2015%26volume%3D157%26spage%3D311%26epage%3D320%26doi%3D10.1093%2Fjb%2Fmvu075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaspersen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunlop, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulven, T.</span></span> <span> </span><span class="NLM_article-title">Succinct synthesis of saturated hydroxy fatty acids and in vitro evaluation of all hydroxylauric acids on FFA1, FFA4 and GPR84</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1360</span>– <span class="NLM_lpage">1365</span>, <span class="refDoi"> DOI: 10.1039/C7MD00130D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1039%2FC7MD00130D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=30108848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXns1Khtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1360-1365&author=M.+H.+Kaspersenauthor=L.+Jenkinsauthor=J.+Dunlopauthor=G.+Milliganauthor=T.+Ulven&title=Succinct+synthesis+of+saturated+hydroxy+fatty+acids+and+in+vitro+evaluation+of+all+hydroxylauric+acids+on+FFA1%2C+FFA4+and+GPR84&doi=10.1039%2FC7MD00130D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Succinct synthesis of saturated hydroxy fatty acids and in vitro evaluation of all hydroxylauric acids on FFA1, FFA4 and GPR84</span></div><div class="casAuthors">Kaspersen, Mads Holmgaard; Jenkins, Laura; Dunlop, Julia; Milligan, Graeme; Ulven, Trond</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1360-1365</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Satd. hydroxy fatty acids make up a class of underexplored lipids with potentially interesting biol. activities.  We report a succinct and general synthetic route to satd. hydroxy fatty acids hydroxylated at position 6 or higher, and exemplify this with the synthesis of hydroxylauric acids.  All regioisomers of hydroxylauric acids were tested on free fatty acid receptors FFA1, FFA4 and GPR84.  The results show that the introduction of a hydroxy group and its position have a high impact on receptor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmMRJHqZLXrVg90H21EOLACvtfcHk0lgrYdmnm1IBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXns1Khtr4%253D&md5=4515bf7026ba72b2e9abaeb6148ee957</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC7MD00130D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00130D%26sid%3Dliteratum%253Aachs%26aulast%3DKaspersen%26aufirst%3DM.%2BH.%26aulast%3DJenkins%26aufirst%3DL.%26aulast%3DDunlop%26aufirst%3DJ.%26aulast%3DMilligan%26aufirst%3DG.%26aulast%3DUlven%26aufirst%3DT.%26atitle%3DSuccinct%2520synthesis%2520of%2520saturated%2520hydroxy%2520fatty%2520acids%2520and%2520in%2520vitro%2520evaluation%2520of%2520all%2520hydroxylauric%2520acids%2520on%2520FFA1%252C%2520FFA4%2520and%2520GPR84%26jtitle%3DMedChemComm%26date%3D2017%26volume%3D8%26spage%3D1360%26epage%3D1365%26doi%3D10.1039%2FC7MD00130D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borges, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kügelgen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">6-(Ar)Alkylamino-substituted uracil derivatives: lipid mimetics with potent activity at the orphan G protein-coupled receptor 84 (GPR84)</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">3365</span>– <span class="NLM_lpage">3383</span>, <span class="refDoi"> DOI: 10.1021/acsomega.7b02092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b02092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVyqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=3365-3383&author=T.+Pillaiyarauthor=M.+K%C3%B6seauthor=V.+Namasivayamauthor=K.+Sylvesterauthor=G.+Borgesauthor=D.+Thimmauthor=I.+von+K%C3%BCgelgenauthor=C.+E.+M%C3%BCller&title=6-%28Ar%29Alkylamino-substituted+uracil+derivatives%3A+lipid+mimetics+with+potent+activity+at+the+orphan+G+protein-coupled+receptor+84+%28GPR84%29&doi=10.1021%2Facsomega.7b02092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">6-(Ar)Alkylamino-Substituted Uracil Derivatives: Lipid Mimetics with Potent Activity at the Orphan G Protein-Coupled Receptor 84 (GPR84)</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Koese, Meryem; Namasivayam, Vigneshwaran; Sylvester, Katharina; Borges, Gleice; Thimm, Dominik; von Kuegelgen, Ivar; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">3365-3383</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">GPR84, a Gi protein-coupled receptor that is activated by medium-chain (hydroxy)fatty acids, appears to play an important role in inflammation, immunity, and cancer.  Recently, 6-octylaminouracil has been reported to act as an agonist at GPR84.  Here, we describe the synthesis of 69 derivs. and analogs which represent new compds.  They were evaluated in (a) cyclic adenosine monophosphate accumulation and (b) β-arrestin assays in human GPR84-expressing cells.  Potent nonbiased as well as G protein-biased agonists were developed, e.g., 6-hexylamino-2,4(1H,3H)-pyrimidinedione (PSB-1584, EC50 5.0 nM (a), 3.2 nM (b), bias factor: 0) and 6-((p-chloro- and p-bromo-phenylethyl)amino)-2,4(1H,3H)-pyrimidinedione (PSB-16434, EC50 7.1 nM (a), 520 nM (b), bias factor: 1.9 = 79-fold Gi pathway-selective; I, PSB-17365, EC50 2.5 nM (a), 100 nM (b), bias factor 1.3 = 20-fold selective), which were selective vs. other free fatty acid-activated receptors.  Compds. II and I were found to be metabolically stable upon incubation with human liver microsomes.  A pharmacophore model was created on the basis of structurally diverse lipidlike GPR84 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc4TDL4U0tF7Vg90H21EOLACvtfcHk0lgrYdmnm1IBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVyqu7g%253D&md5=184964bac823f7cd34d4de98da0cef53</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b02092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b02092%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DSylvester%26aufirst%3DK.%26aulast%3DBorges%26aufirst%3DG.%26aulast%3DThimm%26aufirst%3DD.%26aulast%3Dvon%2BK%25C3%25BCgelgen%26aufirst%3DI.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3D6-%2528Ar%2529Alkylamino-substituted%2520uracil%2520derivatives%253A%2520lipid%2520mimetics%2520with%2520potent%2520activity%2520at%2520the%2520orphan%2520G%2520protein-coupled%2520receptor%252084%2520%2528GPR84%2529%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D3365%26epage%3D3383%26doi%3D10.1021%2Facsomega.7b02092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahmud, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulven, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labeguere, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tikhonova, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span> <span> </span><span class="NLM_article-title">Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">17953</span>– <span class="NLM_lpage">17967</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-18159-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fs41598-017-18159-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29263400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC1MzjslSqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=17953-17967&author=Z.+A.+Mahmudauthor=L.+Jenkinsauthor=T.+Ulvenauthor=F.+Labeguereauthor=R.+Gosminiauthor=S.+De+Vosauthor=B.+D.+Hudsonauthor=I.+G.+Tikhonovaauthor=G.+Milligan&title=Three+classes+of+ligands+each+bind+to+distinct+sites+on+the+orphan+G+protein-coupled+receptor+GPR84&doi=10.1038%2Fs41598-017-18159-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Three classes of ligands each bind to distinct sites on the orphan G protein-coupled receptor GPR84</span></div><div class="casAuthors">Mahmud Zobaer Al; Jenkins Laura; Hudson Brian D; Milligan Graeme; Ulven Trond; Labeguere Frederic; Gosmini Romain; Labeguere Frederic; De Vos Steve; Tikhonova Irina G</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17953</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Medium chain fatty acids can activate the pro-inflammatory receptor GPR84 but so also can molecules related to 3,3'-diindolylmethane. 3,3'-Diindolylmethane and decanoic acid acted as strong positive allosteric modulators of the function of each other and analysis showed the affinity of 3,3'-diindolylmethane to be at least 100 fold higher.  Methyl decanoate was not an agonist at GPR84.  This implies a key role in binding for the carboxylic acid of the fatty acid.  Via homology modelling we predicted and confirmed an integral role of arginine(172), located in the 2nd extracellular loop, in the action of decanoic acid but not of 3,3'-diindolylmethane.  Exemplars from a patented series of GPR84 antagonists were able to block agonist actions of both decanoic acid and 3,3'-diindolylmethane at GPR84.  However, although a radiolabelled form of a related antagonist, [(3)H]G9543, was able to bind with high affinity to GPR84, this was not competed for by increasing concentrations of either decanoic acid or 3,3'-diindolylmethane and was not affected adversely by mutation of arginine(172).  These studies identify three separable ligand binding sites within GPR84 and suggest that if medium chain fatty acids are true endogenous regulators then co-binding with a positive allosteric modulator would greatly enhance their function in physiological settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREMjSSm_naSC-bLEOO4dYpfW6udTcc2eZDP2_xtjmoR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzjslSqug%253D%253D&md5=fe3d58f70952bef231505deb9bbb9de0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-18159-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-18159-3%26sid%3Dliteratum%253Aachs%26aulast%3DMahmud%26aufirst%3DZ.%2BA.%26aulast%3DJenkins%26aufirst%3DL.%26aulast%3DUlven%26aufirst%3DT.%26aulast%3DLabeguere%26aufirst%3DF.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DDe%2BVos%26aufirst%3DS.%26aulast%3DHudson%26aufirst%3DB.%2BD.%26aulast%3DTikhonova%26aufirst%3DI.%2BG.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DThree%2520classes%2520of%2520ligands%2520each%2520bind%2520to%2520distinct%2520sites%2520on%2520the%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR84%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D17953%26epage%3D17967%26doi%3D10.1038%2Fs41598-017-18159-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weighardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borges, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Förster, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kügelgen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Diindolylmethane derivatives: potent agonists of the immunostimulatory orphan G protein-coupled receptor GPR84</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3636</span>– <span class="NLM_lpage">3655</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01593</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01593" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVWlu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3636-3655&author=T.+Pillaiyarauthor=M.+K%C3%B6seauthor=K.+Sylvesterauthor=H.+Weighardtauthor=D.+Thimmauthor=G.+Borgesauthor=I.+F%C3%B6rsterauthor=I.+von+K%C3%BCgelgenauthor=C.+E.+M%C3%BCller&title=Diindolylmethane+derivatives%3A+potent+agonists+of+the+immunostimulatory+orphan+G+protein-coupled+receptor+GPR84&doi=10.1021%2Facs.jmedchem.6b01593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Diindolylmethane Derivatives: Potent Agonists of the Immunostimulatory Orphan G Protein-Coupled Receptor GPR84</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Koese, Meryem; Sylvester, Katharina; Weighardt, Heike; Thimm, Dominik; Borges, Gleice; Foerster, Irmgard; von Kuegelgen, Ivar; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3636-3655</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">3,3'-Diindolylmethanes such as I, 3,3'-biindolylalkanes, and indolylmethylene- and indolylmethyloxindoles were prepd. as agonists of the G-protein-coupled receptor GPR84, an orphan hydroxy fatty acid receptor expressed on immune cells.  Condensation of indoles with formaldehyde or aldehydes in water under microwave irradn. at 100° with or without added sodium dodecyl sulfate yielded the diindolemethane products.  The agonism of products at the human GPR84 was detd. using in cAMP and β-arrestin assays.  Small structural changes in the diindolylmethane products yielded large changes in biol. activity.  3,3'-Diindolylmethanes bearing small lipophilic residues at the 5- and/or 7-position of the indole rings displayed the highest activity in cAMP assays; in β-arrestin assays, the structure-activity relationships differed, indicating biased agonism.  The selectivity of the compds. were detd. against the fatty acid receptors FFAR1 and FFAR4 and the arylhydrocarbon receptor.  Selected compds. were further investigated and displayed an allosteric agonist mechanism of action; the compds. were more stable than 3,3'-diindolylmethane in simulated gastric fluid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhuTbMdKJx77Vg90H21EOLACvtfcHk0lim0-jXAO2X1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVWlu7Y%253D&md5=28204d35b027551aceb2d80fcd42200e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01593%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DSylvester%26aufirst%3DK.%26aulast%3DWeighardt%26aufirst%3DH.%26aulast%3DThimm%26aufirst%3DD.%26aulast%3DBorges%26aufirst%3DG.%26aulast%3DF%25C3%25B6rster%26aufirst%3DI.%26aulast%3Dvon%2BK%25C3%25BCgelgen%26aufirst%3DI.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DDiindolylmethane%2520derivatives%253A%2520potent%2520agonists%2520of%2520the%2520immunostimulatory%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR84%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3636%26epage%3D3655%26doi%3D10.1021%2Facs.jmedchem.6b01593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hakak, Y.</span>; <span class="NLM_string-name">Unett, D. J.</span>; <span class="NLM_string-name">Gatlin, J.</span>; <span class="NLM_string-name">Liaw, C. W.</span></span> <span> </span><span class="NLM_article-title">Human G Protein-Coupled Receptor and Modulators Thereof for the Treatment of Atherosclerosis and Atherosclerotic Disease and for the Treatment of Conditions Related to MCP-1 Expression</span>. International Patent application <span class="NLM_patent">WO2007027661</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=Y.+Hakak&author=D.+J.+Unett&author=J.+Gatlin&author=C.+W.+Liaw&title=Human+G+Protein-Coupled+Receptor+and+Modulators+Thereof+for+the+Treatment+of+Atherosclerosis+and+Atherosclerotic+Disease+and+for+the+Treatment+of+Conditions+Related+to+MCP-1+Expression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHakak%26aufirst%3DY.%26atitle%3DHuman%2520G%2520Protein-Coupled%2520Receptor%2520and%2520Modulators%2520Thereof%2520for%2520the%2520Treatment%2520of%2520Atherosclerosis%2520and%2520Atherosclerotic%2520Disease%2520and%2520for%2520the%2520Treatment%2520of%2520Conditions%2520Related%2520to%2520MCP-1%2520Expression%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of a novel small molecule agonist for medium chain free fatty acid receptor GPR84</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>357</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">344</span>, <span class="refDoi"> DOI: 10.1124/jpet.116.232033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1124%2Fjpet.116.232033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=26962172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWhu7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2016&pages=337-344&author=Q.+Zhangauthor=H.+Yangauthor=J.+Liauthor=X.+Xie&title=Discovery+and+characterization+of+a+novel+small+molecule+agonist+for+medium+chain+free+fatty+acid+receptor+GPR84&doi=10.1124%2Fjpet.116.232033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of a novel small-molecule agonist for medium-chain free fatty acid receptor G protein-coupled receptor 84</span></div><div class="casAuthors">Zhang, Qing; Yang, Hui; Li, Jing; Xie, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-344</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">G protein-coupled receptor 84 (GPR84) is a free fatty acid receptor activated by medium-chain free fatty acids with 9-14 carbons.  It is expressed mainly in the immune-related tissues, such as spleen, bone marrow, and peripheral blood leukocytes.  GPR84 plays significant roles in inflammatory processes and may represent a novel drug target for the treatment of immune-mediated diseases.  However, the lack of potent and specific ligands for GPR84 hindered the study of its functions and the development of potential clin. applications.  Here, we report the screen of 160,000 small- mol. compds. with a calcium mobilization assay using a human embryonic kidney 293 cell line stably expressing GPR84 and Ga16, and the identification of 2-(hexylthio)pyrimidine-4,6-diol (ZQ-16) as a potent and selective agonist of GPR84 with a novel structure.  ZQ-16 activates several GPR84-mediated signaling pathways, including calcium mobilization, inhibition of cAMP accumulation, phosphorylation of extracellular signal-regulated protein kinase 1/2, receptor desensitization and internalization, and receptor-β-arrestin interaction.  This compd. may be a useful tool to study the functions of GPR84 and a potential candidate for further structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjdl6EN67bhbVg90H21EOLACvtfcHk0lim0-jXAO2X1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWhu7fE&md5=8a2f4113fddcb35cdd39f99b9470e0c7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1124%2Fjpet.116.232033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.116.232033%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DX.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520a%2520novel%2520small%2520molecule%2520agonist%2520for%2520medium%2520chain%2520free%2520fatty%2520acid%2520receptor%2520GPR84%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2016%26volume%3D357%26spage%3D337%26epage%3D344%26doi%3D10.1124%2Fjpet.116.232033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, F. J.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of 2 alkylpyrimidine-4,6-diol and 6 alkylpyridine-2,4-diol as potent GPR84 agonists</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=579-583&author=Y.+Liuauthor=Q.+Zhangauthor=L.+H.+Chenauthor=H.+Yangauthor=W.+Luauthor=X.+Xieauthor=F.+J.+Nan&title=Design+and+synthesis+of+2+alkylpyrimidine-4%2C6-diol+and+6+alkylpyridine-2%2C4-diol+as+potent+GPR84+agonists&doi=10.1021%2Facsmedchemlett.6b00025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of 2-Alkylpyrimidine-4,6-diol and 6-Alkylpyridine-2,4-diol as Potent GPR84 Agonists</span></div><div class="casAuthors">Liu, Yang; Zhang, Qing; Chen, Lin-Hai; Yang, Hui; Lu, Wei; Xie, Xin; Nan, Fa-Jun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">579-583</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of alkylpyrimidine-4,6-diol derivs. were designed and synthesized as novel GPR84 agonists based on a high-throughput screening (HTS) hit I. 6-Nonylpyridine-2,4-diol was identified as the most potent agonist of GPR84 reported so far, with an EC50 of 0.189 nM.  These novel GPR84 agonists will provide valuable tools for the study of the physiol. functions of GPR84.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyEcxh2lX2Q7Vg90H21EOLACvtfcHk0liQAJoPaWussw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGht7c%253D&md5=52f50ffae4ab4bbfd79f9eeda0bb063b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00025%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%2BH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DF.%2BJ.%26atitle%3DDesign%2520and%2520synthesis%2520of%25202%2520alkylpyrimidine-4%252C6-diol%2520and%25206%2520alkylpyridine-2%252C4-diol%2520as%2520potent%2520GPR84%2520agonists%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D579%26epage%3D583%26doi%3D10.1021%2Facsmedchemlett.6b00025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nose, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogure, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, T.</span></span> <span> </span><span class="NLM_article-title">Identification of surrogate ligands for orphan G protein-coupled receptors</span>. <i>Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1016/j.lfs.2003.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.lfs.2003.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=14607265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos12qtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2003&pages=367-377&author=S.+Takedaauthor=A.+Yamamotoauthor=T.+Okadaauthor=E.+Matsumuraauthor=E.+Noseauthor=K.+Kogureauthor=S.+Kojimaauthor=T.+Haga&title=Identification+of+surrogate+ligands+for+orphan+G+protein-coupled+receptors&doi=10.1016%2Fj.lfs.2003.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of surrogate ligands for orphan G protein-coupled receptors</span></div><div class="casAuthors">Takeda, Shigeki; Yamamoto, Atsushi; Okada, Tomoaki; Matsumura, Emiko; Nose, Emi; Kogure, Katsuya; Kojima, Shigen; Haga, Tatsuya</div><div class="citationInfo"><span class="NLM_cas:title">Life Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">367-377</span>CODEN:
                <span class="NLM_cas:coden">LIFSAK</span>;
        ISSN:<span class="NLM_cas:issn">0024-3205</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">The authors prepd. fusion proteins with an α subunit of G protein Gi (Gi1α) of 26 orphan G protein-coupled receptors (GPCRs) and with Gsα of 10 orphan GPCRs, most of which had been identified from the human genome previously.  Ligands for these fusion proteins were screened from a library consisting of approx. 1000 authentic compds. by measuring their effect on [35S]GTPγS binding to membrane prepns. of insect Sf9 cells expressing these fusion proteins.  Eleven compds. were found to act as surrogate agonists for a GPCR-Gsα and four GPCR-Giα fusion proteins, a compd. as an inverse agonist for two GPCR-Gsα fusion proteins, and a compd. as an endogenous agonist for a GPCR-Giα fusion protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV6jhTXRA457Vg90H21EOLACvtfcHk0liQAJoPaWussw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos12qtLY%253D&md5=8869ec4d0d89b5aae40bc8c2090db09f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.lfs.2003.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.lfs.2003.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DOkada%26aufirst%3DT.%26aulast%3DMatsumura%26aufirst%3DE.%26aulast%3DNose%26aufirst%3DE.%26aulast%3DKogure%26aufirst%3DK.%26aulast%3DKojima%26aufirst%3DS.%26aulast%3DHaga%26aufirst%3DT.%26atitle%3DIdentification%2520of%2520surrogate%2520ligands%2520for%2520orphan%2520G%2520protein-coupled%2520receptors%26jtitle%3DLife%2520Sci.%26date%3D2003%26volume%3D74%26spage%3D367%26epage%3D377%26doi%3D10.1016%2Fj.lfs.2003.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marques, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laflamme, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benassou, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cissokho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemette, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudreau, L.</span></span> <span> </span><span class="NLM_article-title">Low levels of 3,3′-diindolylmethane activate estrogen receptor α and induce proliferation of breast cancer cells in the absence of estradiol</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1186/1471-2407-14-524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1186%2F1471-2407-14-524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25048790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslamtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=524-533&author=M.+Marquesauthor=L.+Laflammeauthor=I.+Benassouauthor=C.+Cissokhoauthor=B.+Guillemetteauthor=L.+Gaudreau&title=Low+levels+of+3%2C3%E2%80%B2-diindolylmethane+activate+estrogen+receptor+%CE%B1+and+induce+proliferation+of+breast+cancer+cells+in+the+absence+of+estradiol&doi=10.1186%2F1471-2407-14-524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Low levels of 3,3'-diindolylmethane activate estrogen receptor α and induce proliferation of breast cancer cells in the absence of estradiol</span></div><div class="casAuthors">Marques, Maud; Laflamme, Liette; Benassou, Ines; Cissokho, Coumba; Guillemette, Benoit; Gaudreau, Luc</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">524/1-524/10, 10 pp.</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: 3,3'-diindolylmethane (DIM) is an acid-catalyzed dimer of idole-3-carbinol (I3C), a phytochem. found in cruciferous vegetables that include broccoli, Brussels sprouts and cabbage.  DIM is an aryl hydrocarbon receptor (AhR) ligand and a potential anticancer agent, namely for the treatment of breast cancer.  It is also advertised as a compd. that regulates sex hormone homeostasis.  Methods: Here we make use of RNA expression assays coupled to Chromatin Immunopptn. (ChIP) in breast cancer cell lines to study the effect of DIM on estrogen signaling.  We further make use of growth assays, as well as fluorescence-activated cell sorting (FACS) assays, to monitor cell growth.  Results: In this study, we report that 'physiol. obtainable' concns. of DIM (10 μM) activate the estrogen receptor α (ERα) signaling pathway in the human breast cancer cell lines MCF7 and T47D, in a 17β-estradiol (E2)-independent manner.  Accordingly, we observe induction of ERα target genes such as GREB1 and TFF1, and an increase in cellular proliferation after treatment with 10 μM DIM in the absence of E2.  By using an ERα specific inhibitor (ICI 182 780), we confirm that the transcriptional and proliferative effects of DIM treatment are mediated by ERα.  We further show that the protein kinase A signaling pathway participates in DIM-mediated activation of ERα.  In contrast, higher concns. of DIM (e.g. 50 μM) have an opposite and expected effect on cells, which is to inhibit proliferation.  Conclusions: We document an unexpected effect of DIM on cell proliferation, which is to stimulate growth by inducing the ERα signaling pathway.  Importantly, this proliferative effect of DIM happens with potentially physiol. concns. that can be provided by the diet or by taking caplet supplements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6MbRkbnyZv7Vg90H21EOLACvtfcHk0liQAJoPaWussw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslamtb7L&md5=a47dd52ea7387591cbcc7887e1d9e594</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2F1471-2407-14-524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2407-14-524%26sid%3Dliteratum%253Aachs%26aulast%3DMarques%26aufirst%3DM.%26aulast%3DLaflamme%26aufirst%3DL.%26aulast%3DBenassou%26aufirst%3DI.%26aulast%3DCissokho%26aufirst%3DC.%26aulast%3DGuillemette%26aufirst%3DB.%26aulast%3DGaudreau%26aufirst%3DL.%26atitle%3DLow%2520levels%2520of%25203%252C3%25E2%2580%25B2-diindolylmethane%2520activate%2520estrogen%2520receptor%2520%25CE%25B1%2520and%2520induce%2520proliferation%2520of%2520breast%2520cancer%2520cells%2520in%2520the%2520absence%2520of%2520estradiol%26jtitle%3DBMC%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D524%26epage%3D533%26doi%3D10.1186%2F1471-2407-14-524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span> <span> </span><span class="NLM_article-title">A selective aryl hydrocarbon receptor modulator 3,3′-diindolylmethane inhibits gastric cancer cell growth</span>. <i>J. Exp. Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">46</span>, <span class="refDoi"> DOI: 10.1186/1756-9966-31-46</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1186%2F1756-9966-31-46" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=22592002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCqsL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=46&author=X.+F.+Yinauthor=J.+Chenauthor=W.+Maoauthor=Y.+H.+Wangauthor=M.+H.+Chen&title=A+selective+aryl+hydrocarbon+receptor+modulator+3%2C3%E2%80%B2-diindolylmethane+inhibits+gastric+cancer+cell+growth&doi=10.1186%2F1756-9966-31-46"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A selective aryl hydrocarbon receptor modulator 3,3'-Diindolylmethane inhibits gastric cancer cell growth</span></div><div class="casAuthors">Yin, Xiao-Fei; Chen, Jie; Mao, Wei; Wang, Yu-Hong; Chen, Min-Hu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental & Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">46</span>CODEN:
                <span class="NLM_cas:coden">JECRDN</span>;
        ISSN:<span class="NLM_cas:issn">1756-9966</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Aryl hydrocarbon receptor (AhR) is a ligand-activated transcription factor assocd. with gastric carcinogenesis.  3,3'-Diindolylmethane (DIM) is a relatively non-toxic selective AhR modulator.  This study was to detect the effects of DIM on gastric cancer cell growth.  Methods: Gastric cancer cell SGC7901 was treated with DIM at different concns. (0, 10, 20, 30, 40, 50 μmol/L) with or without an AhR antagonist, resveratrol.  The expression of AhR and Cytochrome P 4501A1 (CYP1A1), a classic target gene of AhR pathway, were detected by RT-PCR and Western blot; cell viability was measured by MTT assay, and the changes in cell cycle and apoptosis were analyzed by flow cytometry.  Results: RT-PCR and western-blot showed that with the increase of the concn. of DIM, AhR protein gradually decreased and CYP1A1 expression increased, suggesting that DIM activated the AhR pathway and caused the translocation of AhR from cytoplasm to nucleus.  MTT assay indicated that the viability of SGC7901 cells was significantly decreased in a concn.- and time-dependent manner after DIM treatment and this could be partially reversed by resveratrol.  Flow cytometry anal. showed that DIM arrested cell cycle in G1 phase and induced cell apoptosis.  Conclusion: Selective aryl hydrocarbon receptor modulator 3,3'-Diindolylmethane inhibits SGC7901 cell proliferation by inducing apoptosis and delaying cell cycle progression.  AhR may be a potential therapeutic target for gastric cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLSqncAyaWjrVg90H21EOLACvtfcHk0lhymVcYSllDpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCqsL3E&md5=87cba16be653452c8d9043156f2b800b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1756-9966-31-46&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1756-9966-31-46%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DX.%2BF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DM.%2BH.%26atitle%3DA%2520selective%2520aryl%2520hydrocarbon%2520receptor%2520modulator%25203%252C3%25E2%2580%25B2-diindolylmethane%2520inhibits%2520gastric%2520cancer%2520cell%2520growth%26jtitle%3DJ.%2520Exp.%2520Clin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D31%26spage%3D46%26doi%3D10.1186%2F1756-9966-31-46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mancini, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmud, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edye, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobin, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span> <span> </span><span class="NLM_article-title">On-target and off-target effects of novel orthosteric and allosteric activators of GPR84</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1861</span>, <span class="refDoi"> DOI: 10.1038/s41598-019-38539-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fs41598-019-38539-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=30755705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BB3cfis1amsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1861&author=S.+J.+Manciniauthor=Z.+A.+Mahmudauthor=L.+Jenkinsauthor=D.+Bologniniauthor=R.+Newmanauthor=M.+Barnesauthor=M.+E.+Edyeauthor=S.+B.+McMahonauthor=A.+B.+Tobinauthor=G.+Milligan&title=On-target+and+off-target+effects+of+novel+orthosteric+and+allosteric+activators+of+GPR84&doi=10.1038%2Fs41598-019-38539-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">On-target and off-target effects of novel orthosteric and allosteric activators of GPR84</span></div><div class="casAuthors">Mancini Sarah J; Mahmud Zobaer Al; Jenkins Laura; Bolognini Daniele; Tobin Andrew B; Milligan Graeme; Newman Robert; Barnes Matt; Edye Michelle E; McMahon Stephen B</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1861</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Many members of the G protein-coupled receptor family, including examples with clear therapeutic potential, remain poorly characterised.  This often reflects limited availability of suitable tool ligands with which to interrogate receptor function.  In the case of GPR84, currently a target for the treatment of idiopathic pulmonary fibrosis, recent times have seen the description of novel orthosteric and allosteric agonists.  Using 2-(hexylthiol)pyrimidine-4,6 diol (2-HTP) and di(5,7-difluoro-1H-indole-3-yl)methane (PSB-16671) as exemplars of each class, in cell lines transfected to express either human or mouse GPR84, both ligands acted as effective on-target activators and with high co-operativity in their interactions.  This was also the case in lipopolysaccharide-activated model human and mouse immune cell lines.  However in mouse bone-marrow-derived neutrophils, where expression of GPR84 is particularly high, the capacity of PSB-16671 but not of 2-HTP to promote G protein activation was predominantly off-target because it was not blocked by an antagonist of GPR84 and was preserved in neutrophils isolated from GPR84 deficient mice.  These results illustrate the challenges of attempting to study and define functions of poorly characterised receptors using ligands that have been developed via medicinal chemistry programmes, but where assessed activity has been limited largely to the initially identified target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQc62oWEy_4Oqrn-c4Of3KfW6udTcc2eaOgF8H2wR1iLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfis1amsA%253D%253D&md5=1bd20c2443890cb3691d558cb8511c5f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41598-019-38539-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-019-38539-1%26sid%3Dliteratum%253Aachs%26aulast%3DMancini%26aufirst%3DS.%2BJ.%26aulast%3DMahmud%26aufirst%3DZ.%2BA.%26aulast%3DJenkins%26aufirst%3DL.%26aulast%3DBolognini%26aufirst%3DD.%26aulast%3DNewman%26aufirst%3DR.%26aulast%3DBarnes%26aufirst%3DM.%26aulast%3DEdye%26aufirst%3DM.%2BE.%26aulast%3DMcMahon%26aufirst%3DS.%2BB.%26aulast%3DTobin%26aufirst%3DA.%2BB.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DOn-target%2520and%2520off-target%2520effects%2520of%2520novel%2520orthosteric%2520and%2520allosteric%2520activators%2520of%2520GPR84%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D1861%26doi%3D10.1038%2Fs41598-019-38539-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labeguere, F.</span>; <span class="NLM_string-name">Luke, A.</span>; <span class="NLM_string-name">Gregory, N.</span>; <span class="NLM_string-name">Laurent, S.</span>; <span class="NLM_string-name">Stephen, F.</span></span> <span> </span><span class="NLM_article-title">Novel Dihydropyrimidinoisoquinolinones and Pharmaceutical Compositions Thereof for the Treatment of Inflammatory Disorders</span>. <span class="NLM_patent">WO2013092791A1</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=F.+Labeguere&author=A.+Luke&author=N.+Gregory&author=S.+Laurent&author=F.+Stephen&title=Novel+Dihydropyrimidinoisoquinolinones+and+Pharmaceutical+Compositions+Thereof+for+the+Treatment+of+Inflammatory+Disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLabeguere%26aufirst%3DF.%26atitle%3DNovel%2520Dihydropyrimidinoisoquinolinones%2520and%2520Pharmaceutical%2520Compositions%2520Thereof%2520for%2520the%2520Treatment%2520of%2520Inflammatory%2520Disorders%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niedernberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tunaru, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaukat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span> <span> </span><span class="NLM_article-title">Comparative analysis of functional assays for characterization of agonist ligands at G protein-coupled receptors</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">500</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1177/1087057103257555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1177%2F1087057103257555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=14567777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptVCiu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=500-510&author=A.+Niedernbergauthor=S.+Tunaruauthor=A.+Blaukatauthor=B.+Harrisauthor=E.+Kostenis&title=Comparative+analysis+of+functional+assays+for+characterization+of+agonist+ligands+at+G+protein-coupled+receptors&doi=10.1177%2F1087057103257555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative analysis of functional assays for characterization of agonist ligands at G protein-coupled receptors</span></div><div class="casAuthors">Niedernberg, Anke; Tunaru, Sorin; Blaukat, Andree; Harris, Bruce; Kostenis, Evi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">500-510</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A variety of functional assays are available for agonist or antagonist screening of G protein-coupled receptors (GPCRs), but it is a priori not predictable which assay is the most suitable to identify agonists or antagonists of GPCRs with therapeutic value in humans.  More specifically, it is not known how a given set of GPCR agonists compares in different functional assays with respect to potency and efficacy and whether the level of the signaling cascade that is analyzed has any impact on the detection of agonistic responses.  To address this question, the authors used the recently cloned human S1P5 receptor as a model and compared a set of 3 lipid ligands (sphingosine 1-phosphate [SIP], dihydro sphingosine 1-phosphate [dhS1P], and sphingosine) in 5 different functional assays: GTPγS binding, inhibition of adenylyl cyclase activity, mobilization of intracellular Ca2+ via the FLIPR and aequorin technol., and MAP kinase (ERK1/2) activation.  S1P induced agonistic responses in all except the ERK1/2 assays with EC50 values varying by a factor of 10.  Whereas dhS1P was identified as a partial agonist in the GTPγS assay, it behaved as a full agonist in all other settings.  Sphingosine displayed partial agonistic activity exclusively in GTPγS binding assays.  The findings suggest that assays in a given cellular background may vary significantly with respect to suitability for agonist finding and that ligands producing a response may not readily be detectable in all agonist assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvk-TaAKQ0JbVg90H21EOLACvtfcHk0lj9_Fgc96I9tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptVCiu7k%253D&md5=4fdb34d780da43882331e721381fa249</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F1087057103257555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057103257555%26sid%3Dliteratum%253Aachs%26aulast%3DNiedernberg%26aufirst%3DA.%26aulast%3DTunaru%26aufirst%3DS.%26aulast%3DBlaukat%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DB.%26aulast%3DKostenis%26aufirst%3DE.%26atitle%3DComparative%2520analysis%2520of%2520functional%2520assays%2520for%2520characterization%2520of%2520agonist%2520ligands%2520at%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2003%26volume%3D8%26spage%3D500%26epage%3D510%26doi%3D10.1177%2F1087057103257555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niedernberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaukat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schöneberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span> <span> </span><span class="NLM_article-title">Regulated and constitutive activation of specific signalling pathways by the human S1P5 receptor</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">493</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0705055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fsj.bjp.0705055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=12569073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhvFSrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2003&pages=481-493&author=A.+Niedernbergauthor=A.+Blaukatauthor=T.+Sch%C3%B6nebergauthor=E.+Kostenis&title=Regulated+and+constitutive+activation+of+specific+signalling+pathways+by+the+human+S1P5+receptor&doi=10.1038%2Fsj.bjp.0705055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Regulated and constitutive activation of specific signalling pathways by the human S1P5 receptor</span></div><div class="casAuthors">Niedernberg, Anke; Blaukat, Andree; Schoneberg, Torsten; Kostenis, Evi</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">481-493</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the hypothesis, whether G Protein-coupled receptors (GPCRs) may differentially regulate specific signalling pathways by constitutive and agonist-induced activation using the human sphingosine 1-phosphate receptor S1P5 as a model.  S1P5 receptor-expressing HEK293 cells exhibited a high degree of basal activity for both inhibition of adenylyl cyclase and extracellular signal regulated kinase (ERK) when cultured in serum, which contains high levels of sphingosine 1-phosphate (S1P).  However, basal activity was independent of the presence of S1P:.  (i) constitutive activity remained when cells were cultured in delipidated serum,.  (ii) addn. of S1P to delipidated serum did not increase basal S1P5 receptor signalling.  Conversely, constitutive inhibition of forskolin-stimulated adenylyl cyclase was further enhanced by S1P in S1P5-HEK293 cells.  Transfection of several mammalian cell lines (CHO-K1, HEK293, NIH-3T3, RH7777) with the S1P5 receptor induced cell rounding, which was more pronounced in the presence of S1P-contg. serum.  Rounded cell morphol. did not correlate with apoptotic cell death, but led to detachment of cells.  Cell surface ELISA assays showed that a fraction of plasma membrane S1P5 receptors were dose-dependently internalized with S1P.  These data reveal that intrinsic inhibition of unstimulated adenylyl cyclase or ERK activity by the S1P5 receptor is insensitive to ligand modulation.  Conversely, effects on forskolin-stimulated adenylyl cyclase, cell morphol. and internalization can be further augmented with S1P.  Our results suggest that different signal transduction pathways are not equally activated through constitutively active GPCRs with promiscuous signalling characteristics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouj12-uzY0GrVg90H21EOLACvtfcHk0lj9_Fgc96I9tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhvFSrur0%253D&md5=5c9dbb5f868009c8054451c8529cf282</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705055%26sid%3Dliteratum%253Aachs%26aulast%3DNiedernberg%26aufirst%3DA.%26aulast%3DBlaukat%26aufirst%3DA.%26aulast%3DSch%25C3%25B6neberg%26aufirst%3DT.%26aulast%3DKostenis%26aufirst%3DE.%26atitle%3DRegulated%2520and%2520constitutive%2520activation%2520of%2520specific%2520signalling%2520pathways%2520by%2520the%2520human%2520S1P5%2520receptor%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D138%26spage%3D481%26epage%3D493%26doi%3D10.1038%2Fsj.bjp.0705055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitsunobu, O.</span></span> <span> </span><span class="NLM_article-title">The use of diethyl azodicarboxylate and triphenylphosphine in synthesis and transformation of natural products</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>1981</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1055/s-1981-29317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1055%2Fs-1981-29317" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1981&publication_year=1981&pages=1-28&author=O.+Mitsunobu&title=The+use+of+diethyl+azodicarboxylate+and+triphenylphosphine+in+synthesis+and+transformation+of+natural+products&doi=10.1055%2Fs-1981-29317"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1055%2Fs-1981-29317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1981-29317%26sid%3Dliteratum%253Aachs%26aulast%3DMitsunobu%26aufirst%3DO.%26atitle%3DThe%2520use%2520of%2520diethyl%2520azodicarboxylate%2520and%2520triphenylphosphine%2520in%2520synthesis%2520and%2520transformation%2520of%2520natural%2520products%26jtitle%3DSynthesis%26date%3D1981%26volume%3D1981%26spage%3D1%26epage%3D28%26doi%3D10.1055%2Fs-1981-29317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chun, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chubukov, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherzov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structural basis for allosteric regulation of GPCRs by sodium ions</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>337</i></span>,  <span class="NLM_fpage">232</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1126/science.1219218</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1126%2Fscience.1219218" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=22798613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=2012&pages=232-236&author=W.+Liuauthor=E.+Chunauthor=A.+A.+Thompsonauthor=P.+Chubukovauthor=F.+Xuauthor=V.+Katritchauthor=G.+W.+Hanauthor=C.+B.+Rothauthor=L.+H.+Heitmanauthor=A.+P.+IJzermanauthor=V.+Cherzovauthor=R.+C.+Stevens&title=Structural+basis+for+allosteric+regulation+of+GPCRs+by+sodium+ions&doi=10.1126%2Fscience.1219218"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Allosteric Regulation of GPCRs by Sodium Ions</span></div><div class="casAuthors">Liu, Wei; Chun, Eugene; Thompson, Aaron A.; Chubukov, Pavel; Xu, Fei; Katritch, Vsevolod; Han, Gye Won; Roth, Christopher B.; Heitman, Laura H.; IJzerman, Adriaan P.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">6091</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Pharmacol. responses of G protein-coupled receptors (GPCRs) can be fine-tuned by allosteric modulators.  Structural studies of such effects have been limited due to the medium resoln. of GPCR structures.  We reengineered the human A2A adenosine receptor by replacing its third intracellular loop with apocytochrome b562RIL and solved the structure at 1.8 angstrom resoln.  The high-resoln. structure allowed us to identify 57 ordered water mols. inside the receptor comprising three major clusters.  The central cluster harbors a putative sodium ion bound to the highly conserved aspartate residue Asp2.50.  Addnl., two cholesterols stabilize the conformation of helix VI, and one of 23 ordered lipids intercalates inside the ligand-binding pocket.  These high-resoln. details shed light on the potential role of structured water mols., sodium ions, and lipids/cholesterol in GPCR stabilization and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvcjQimGnIMLVg90H21EOLACvtfcHk0lj9_Fgc96I9tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVKqtbs%253D&md5=782b2235b76ef77b11b6456f8e4485e1</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1126%2Fscience.1219218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1219218%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChun%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DA.%2BA.%26aulast%3DChubukov%26aufirst%3DP.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DRoth%26aufirst%3DC.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DCherzov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520basis%2520for%2520allosteric%2520regulation%2520of%2520GPCRs%2520by%2520sodium%2520ions%26jtitle%3DScience%26date%3D2012%26volume%3D337%26spage%3D232%26epage%3D236%26doi%3D10.1126%2Fscience.1219218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">6-Bromo-8-(4-[<sup>3</sup>H]methoxy-benzamido)-4-oxo-4H-chromene-2-carboxylic acid ([<sup>3</sup>H]PSB-13253): a powerful tool for studying orphan G protein-coupled receptor GPR35</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7084</span>– <span class="NLM_lpage">7099</span>, <span class="refDoi"> DOI: 10.1021/jm4009373</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4009373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOlsbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7084-7099&author=D.+Thimmauthor=M.+Funkeauthor=A.+Meyerauthor=C.+E.+M%C3%BCller&title=6-Bromo-8-%284-%5B3H%5Dmethoxy-benzamido%29-4-oxo-4H-chromene-2-carboxylic+acid+%28%5B3H%5DPSB-13253%29%3A+a+powerful+tool+for+studying+orphan+G+protein-coupled+receptor+GPR35&doi=10.1021%2Fjm4009373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">6-Bromo-8-(4-[3H]methoxybenzamido)-4-oxo-4H-chromene-2-carboxylic Acid: A Powerful Tool for Studying Orphan G Protein-Coupled Receptor GPR35</span></div><div class="casAuthors">Thimm, Dominik; Funke, Mario; Meyer, Anne; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7084-7099</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The potent and selective GPR35 agonist 6-bromo-8-(4-methoxybenzamido)-4-oxo-4H-chromene-2-carboxylic acid was obtained in tritium-labeled form, designated [3H]PSB-13253 (I), with a specific activity of 36 Ci (1.33 TBq)/mmol.  Radiolabeling was achieved by methylation of Et 6-bromo-8-(4-((tert-butyldimethylsilyl)oxy)benzamido)-4-oxo-4H-chromene-2-carboxylate with [3H]methyl tosylate followed by ester hydrolysis.  The radioligand was characterized by kinetic, satn., and competition assays at membrane prepns. of Chinese hamster ovary cells recombinantly expressing the human GPR35.  I labeled the receptor with high affinity (KD = 5.27 nM).  Binding was saturable (Bmax = 12.6 pmol/mg of protein) and reversible.  Affinities of selected std. ligands and a library of amidochromen-4-one-2-carboxylates were detd.  Binding data mostly correlated with potencies detd. in β-arrestin assays.  On the basis of the test results, several new fluorine-substituted 6-bromo-8-benzamidochromen-4-one-2-carboxylic acids were obtained, which represent the most potent GPR35 agonists known to date.  6-Bromo-8-(2,6-difluoro-4-methoxybenzamido)-4-oxo-4H-chromene-2-carboxylic acid (II; Ki = 0.589 nM, EC50 = 5.54 nM) showed the highest affinity with a Ki value in the subnanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodEoburTr-0rVg90H21EOLACvtfcHk0ljsYTD0jzM7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOlsbfN&md5=031f866ea6e2c5b40cc2c3977e58dc75</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm4009373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4009373%26sid%3Dliteratum%253Aachs%26aulast%3DThimm%26aufirst%3DD.%26aulast%3DFunke%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3D6-Bromo-8-%25284-%255B3H%255Dmethoxy-benzamido%2529-4-oxo-4H-chromene-2-carboxylic%2520acid%2520%2528%255B3H%255DPSB-13253%2529%253A%2520a%2520powerful%2520tool%2520for%2520studying%2520orphan%2520G%2520protein-coupled%2520receptor%2520GPR35%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7084%26epage%3D7099%26doi%3D10.1021%2Fjm4009373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiemke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kostenis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Development of [(<sup>3</sup>)H]2-carboxy-4,6-dichloro-1<i>H</i>-indole-3-propionic acid ([(<sup>3</sup>)H]PSB-12150): a useful tool for studying GPR17</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1021/ml400399f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400399f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC2cjpvVOjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=326-330&author=M.+K%C3%B6seauthor=K.+Ritterauthor=K.+Thiemkeauthor=M.+Gillardauthor=E.+Kostenisauthor=C.+E.+M%C3%BCller&title=Development+of+%5B%283%29H%5D2-carboxy-4%2C6-dichloro-1H-indole-3-propionic+acid+%28%5B%283%29H%5DPSB-12150%29%3A+a+useful+tool+for+studying+GPR17&doi=10.1021%2Fml400399f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of [(3)H]2-Carboxy-4,6-dichloro-1H-indole-3-propionic Acid ([(3)H]PSB-12150): A Useful Tool for Studying GPR17</span></div><div class="casAuthors">Kose Meryem; Ritter Kirsten; Thiemke Katharina; Muller Christa E; Gillard Michel; Kostenis Evi</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">326-30</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">The recently described synthetic GPR17 agonist 2-carboxy-4,6-dichloro-1H-indole-3-propionic acid (1) was prepared in tritium-labeled form by catalytic hydrogenation of the corresponding propenoic acid derivative 8 with tritium gas.  The radioligand [(3)H]PSB-12150 (9) was obtained with a specific activity of 17 Ci/mmol (629 GBq/mmol).  It showed specific and saturable binding to a single binding site in membrane preparations from Chinese hamster ovary cells recombinantly expressing the human GPR17.  A competition assay procedure was established, which allows the determination of ligand binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS__t-5LBkU-yEJCPKhJPuqfW6udTcc2eZYZIIBxxf_Jrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjpvVOjtw%253D%253D&md5=6feadf019982496e6e520e10beeb31db</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fml400399f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400399f%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DRitter%26aufirst%3DK.%26aulast%3DThiemke%26aufirst%3DK.%26aulast%3DGillard%26aufirst%3DM.%26aulast%3DKostenis%26aufirst%3DE.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DDevelopment%2520of%2520%255B%25283%2529H%255D2-carboxy-4%252C6-dichloro-1H-indole-3-propionic%2520acid%2520%2528%255B%25283%2529H%255DPSB-12150%2529%253A%2520a%2520useful%2520tool%2520for%2520studying%2520GPR17%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D326%26epage%3D330%26doi%3D10.1021%2Fml400399f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
der Westhuizen, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Endogenous allosteric modulators of G protein-coupled receptors</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1124/jpet.114.221606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1124%2Fjpet.114.221606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25650376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVWntb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2015&pages=246-260&author=E.+T.+Van%0Ader+Westhuizenauthor=C.+Valantauthor=P.+M.+Sextonauthor=A.+Christopoulos&title=Endogenous+allosteric+modulators+of+G+protein-coupled+receptors&doi=10.1124%2Fjpet.114.221606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Endogenous allosteric modulators of G protein-coupled receptors</span></div><div class="casAuthors">van der Westhuizen, Emma T.; Valant, Celine; Sexton, Patrick M.; Christopoulos, Christopoulos</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">246-260</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) are the largest superfamily of receptors encoded by the human genome, and represent the largest class of current drug targets.  Over the last decade and a half, it has become widely accepted that most, if not all, GPCRs possess spatially distinct allosteric sites that can be targeted by exogenous substances to modulate the receptors' biol. state.  Although many of these allosteric sites are likely to serve other (e.g., structural) roles, they nonetheless possess appropriate properties to be serendipitously targeted by synthetic mols.  However, there are also examples of endogenous substances that can act as allosteric modulators of GPCRs.  These include not only the obvious example, i.e., the G protein, but also a variety of ions, lipids, amino acids, peptides, and accessory proteins that display different degrees of receptor-specific modulatory effects.  This also suggests that some GPCRs may possess true "orphan" allosteric sites for hitherto unappreciated endogenous modulators.  Of note, the increasing identification of allosteric modulator lipids, inflammatory peptides, and GPCR-targeted autoantibodies indicates that disease context plays an important role in the generation of putative endogenous GPCR modulators.  If an endogenous allosteric substance can be shown to play a role in disease, this could also serve as an impetus to pursue synthetic neutral allosteric ligands as novel therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptdf28zfijQrVg90H21EOLACvtfcHk0ljsYTD0jzM7IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVWntb4%253D&md5=4e9eacb10df4523109feba729fbb9409</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1124%2Fjpet.114.221606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.114.221606%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bder%2BWesthuizen%26aufirst%3DE.%2BT.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DEndogenous%2520allosteric%2520modulators%2520of%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2015%26volume%3D353%26spage%3D246%26epage%3D260%26doi%3D10.1124%2Fjpet.114.221606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shrestha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uemura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiba, H.</span></span> <span> </span><span class="NLM_article-title">Plasma capric acid concentrations in healthy subjects determined by high-performance liquid chromatography</span>. <i>Ann. Clin. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">596</span>, <span class="refDoi"> DOI: 10.1177/0004563215569081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1177%2F0004563215569081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=25587197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvVKqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2015&pages=588-596&author=R.+Shresthaauthor=S.+P.+Huiauthor=H.+Imaiauthor=S.+Hashimotoauthor=N.+Uemuraauthor=S.+Takedaauthor=H.+Fudaauthor=A.+Suzukiauthor=S.+Yamaguchiauthor=K.+Hiranoauthor=H.+Chiba&title=Plasma+capric+acid+concentrations+in+healthy+subjects+determined+by+high-performance+liquid+chromatography&doi=10.1177%2F0004563215569081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Plasma capric acid concentrations in healthy subjects determined by high-performance liquid chromatography</span></div><div class="casAuthors">Shrestha, Rojeet; Hui, Shu-Ping; Imai, Hiromitsu; Hashimoto, Satoru; Uemura, Naoto; Takeda, Seiji; Fuda, Hirotoshi; Suzuki, Akira; Yamaguchi, Satoshi; Hirano, Ken-ichi; Chiba, Hitoshi</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Clinical Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">588-596</span>CODEN:
                <span class="NLM_cas:coden">ACBOBU</span>;
        ISSN:<span class="NLM_cas:issn">0004-5632</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Background: Capric acid (FA10:0, decanoic acid) is a medium-chain fatty acid abundant in tropical oils such as coconut oil, whereas small amts. are present in milk of goat, cow, and human.  Orally ingested FA10:0 is transported to the liver and quickly burnt within it.  Only few reports are available for FA10:0 concns. in human plasma.  Methods: Fasting (n = 5, male/female = 3/2, age 31 ± 9.3 years old) and non-fasting (n = 106, male/female = 44/62, age 21.9 ± 3.2 years old) blood samples were collected from apparently healthy Japanese volunteers.  The total FA10:0 in the plasma were measured by high-performance liq. chromatog. after derivatization with 2-nitrophenylhydrazine followed by UV detection.  Results: Inter and intra-assay coeff. of variation of FA10:0 assay at three different concns. ranged in 1.7-3.9 and 1.3-5.4%, resp., with an anal. recovery of 95.2-104.0%.  FA10:0 concn. was below detection limit (0.1 μmol/L) in each fasting human plasma.  FA10:0 was not detected in 50 (47.2%) of 106 non-fasting blood samples, while 29 (27.4%) plasma samples contained FA10:0 less than or equal to 0.5 μmol/L (0.4 ± 0.1), and 27 (25.5%) contained it at more than 0.5 μmol/L (0.9 ± 0.3).  Conclusion: A half of the non-fasting plasma samples contained detectable FA10:0.  This simple, precise, and accurate high-performance liq. chromatog. method might be useful for monitoring plasma FA10:0 during medium-chain triglycerides therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHytyLwtn8SbVg90H21EOLACvtfcHk0lipZ3seOxyWgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvVKqtb8%253D&md5=acd27812308c7ea149a50325fb6bf80b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1177%2F0004563215569081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F0004563215569081%26sid%3Dliteratum%253Aachs%26aulast%3DShrestha%26aufirst%3DR.%26aulast%3DHui%26aufirst%3DS.%2BP.%26aulast%3DImai%26aufirst%3DH.%26aulast%3DHashimoto%26aufirst%3DS.%26aulast%3DUemura%26aufirst%3DN.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DFuda%26aufirst%3DH.%26aulast%3DSuzuki%26aufirst%3DA.%26aulast%3DYamaguchi%26aufirst%3DS.%26aulast%3DHirano%26aufirst%3DK.%26aulast%3DChiba%26aufirst%3DH.%26atitle%3DPlasma%2520capric%2520acid%2520concentrations%2520in%2520healthy%2520subjects%2520determined%2520by%2520high-performance%2520liquid%2520chromatography%26jtitle%3DAnn.%2520Clin.%2520Biochem.%26date%3D2015%26volume%3D52%26spage%3D588%26epage%3D596%26doi%3D10.1177%2F0004563215569081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Altschul, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gish, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipman, D. J.</span></span> <span> </span><span class="NLM_article-title">Basic local alignment search tool</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">403</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(05)80360-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2FS0022-2836%2805%2980360-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=2231712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=1990&pages=403-410&author=S.+F.+Altschulauthor=W.+Gishauthor=W.+Millerauthor=E.+W.+Myersauthor=D.+J.+Lipman&title=Basic+local+alignment+search+tool&doi=10.1016%2FS0022-2836%2805%2980360-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Basic local alignment search tool</span></div><div class="casAuthors">Altschul, Stephen F.; Gish, Warren; Miller, Webb; Myers, Eugene W.; Lipman, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">403-10</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A new approach to rapid sequence comparison, basic local alignment search tool (BLAST), directly approximates alignments that optimize a measure of local similarity, the maximal segment pair (MSP) score.  Recent math. results on the stochastic properties of MSP scores allow an anal. of the performance of this method as well as the statistical significance of alignments it generates.  The basic algorithm is simple and robust; it can be implemented in a no. of ways and applied in a variety of contexts including straightforward DNA and protein sequence database searches, motif searches, gene identification searches, and in the anal. of multiple regions of similarity in long DNA sequences.  In addn. to its flexibility and tractability to math. anal., BLAST is an order of magnitude faster than existing sequence comparison tools of comparable sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGEHw-q53MfLVg90H21EOLACvtfcHk0lipZ3seOxyWgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D&md5=009d2323eb82f0549356880e1101db16</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2805%2980360-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252805%252980360-2%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26aulast%3DGish%26aufirst%3DW.%26aulast%3DMiller%26aufirst%3DW.%26aulast%3DMyers%26aufirst%3DE.%2BW.%26aulast%3DLipman%26aufirst%3DD.%2BJ.%26atitle%3DBasic%2520local%2520alignment%2520search%2520tool%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1990%26volume%3D215%26spage%3D403%26epage%3D410%26doi%3D10.1016%2FS0022-2836%2805%2980360-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chien, E. Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Javitch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span> <span> </span><span class="NLM_article-title">Structure of the human dopamine D3 receptor in complex with a D2/D3 selective antagonist</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span>,  <span class="NLM_fpage">1091</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1126/science.1197410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1126%2Fscience.1197410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=21097933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=2010&pages=1091-1095&author=E.+Y.+T.+Chienauthor=W.+Liuauthor=Q.+Zhaoauthor=V.+Katritchauthor=G.+W.+Hanauthor=M.+A.+Hansonauthor=L.+Shiauthor=A.+H.+Newmanauthor=J.+A.+Javitchauthor=V.+Cherezovauthor=R.+C.+Stevens&title=Structure+of+the+human+dopamine+D3+receptor+in+complex+with+a+D2%2FD3+selective+antagonist&doi=10.1126%2Fscience.1197410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Human Dopamine D3 Receptor in Complex with a D2/D3 Selective Antagonist</span></div><div class="casAuthors">Chien, Ellen Y. T.; Liu, Wei; Zhao, Qiang; Katritch, Vsevolod; Won Han, Gye; Hanson, Michael A.; Shi, Lei; Newman, Amy Hauck; Javitch, Jonathan A.; Cherezov, Vadim; Stevens, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6007</span>),
    <span class="NLM_cas:pages">1091-1095</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Dopamine modulates movement, cognition, and emotion through activation of dopamine G protein-coupled receptors in the brain.  The crystal structure of the human dopamine D3 receptor (D3R) in complex with the small mol. D2R/D3R-specific antagonist eticlopride reveals important features of the ligand binding pocket and extracellular loops.  On the intracellular side of the receptor, a locked conformation of the ionic lock and two distinctly different conformations of intracellular loop 2 are obsd.  Docking of R-22, a D3R-selective antagonist, reveals an extracellular extension of the eticlopride binding site that comprises a second binding pocket for the aryl amide of R-22, which differs between the highly homologous D2R and D3R.  This difference provides direction to the design of D3R-selective agents for treating drug abuse and other neuropsychiatric indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOEcfSdjMm-7Vg90H21EOLACvtfcHk0lh5Fcpd49cdng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVahurvM&md5=12c6f343daea9fb14ca605a092f28533</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1126%2Fscience.1197410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1197410%26sid%3Dliteratum%253Aachs%26aulast%3DChien%26aufirst%3DE.%2BY.%2BT.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DHanson%26aufirst%3DM.%2BA.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DJavitch%26aufirst%3DJ.%2BA.%26aulast%3DCherezov%26aufirst%3DV.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26atitle%3DStructure%2520of%2520the%2520human%2520dopamine%2520D3%2520receptor%2520in%2520complex%2520with%2520a%2520D2%252FD3%2520selective%2520antagonist%26jtitle%3DScience%26date%3D2010%26volume%3D330%26spage%3D1091%26epage%3D1095%26doi%3D10.1126%2Fscience.1197410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tikhonova, I. G.</span></span> <span> </span><span class="NLM_article-title">Application of GPCR structures for modelling of free fatty acid receptors</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>236</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1007/164_2016_52</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1007%2F164_2016_52" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=236&publication_year=2016&pages=57-77&author=I.+G.+Tikhonova&title=Application+of+GPCR+structures+for+modelling+of+free+fatty+acid+receptors&doi=10.1007%2F164_2016_52"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2F164_2016_52&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F164_2016_52%26sid%3Dliteratum%253Aachs%26aulast%3DTikhonova%26aufirst%3DI.%2BG.%26atitle%3DApplication%2520of%2520GPCR%2520structures%2520for%2520modelling%2520of%2520free%2520fatty%2520acid%2520receptors%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2016%26volume%3D236%26spage%3D57%26epage%3D77%26doi%3D10.1007%2F164_2016_52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramakrishnan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasisekharan, V.</span></span> <span> </span><span class="NLM_article-title">Stereochemistry of polypeptide chain configurations</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(63)80023-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2FS0022-2836%2863%2980023-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=13990617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaF3sXks1arsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1963&pages=95-99&author=G.+N.+Ramachandranauthor=C.+Ramakrishnanauthor=V.+Sasisekharan&title=Stereochemistry+of+polypeptide+chain+configurations&doi=10.1016%2FS0022-2836%2863%2980023-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Stereochemistry of polypeptide chain configurations</span></div><div class="casAuthors">Ramachandran, G. N.; Ramakrishnan, C.; Sasisekharan, V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-9</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A convenient anal. method for writing the configurations of polypeptide chains is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpggsnuSQqKS7Vg90H21EOLACvtfcHk0lh5Fcpd49cdng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXks1arsLk%253D&md5=68298929fb718b888df17dff2275a55e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2863%2980023-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252863%252980023-6%26sid%3Dliteratum%253Aachs%26aulast%3DRamachandran%26aufirst%3DG.%2BN.%26aulast%3DRamakrishnan%26aufirst%3DC.%26aulast%3DSasisekharan%26aufirst%3DV.%26atitle%3DStereochemistry%2520of%2520polypeptide%2520chain%2520configurations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1963%26volume%3D7%26spage%3D95%26epage%3D99%26doi%3D10.1016%2FS0022-2836%2863%2980023-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Bakker, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Word, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure validation by Calpha geometry: phi, psi and Cbeta deviation</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">450</span>, <span class="refDoi"> DOI: 10.1002/prot.10286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fprot.10286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=12557186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2003&pages=437-450&author=S.+C.+Lovellauthor=I.+W.+Davisauthor=W.+B.+Arendallauthor=P.+I.+de%0ABakkerauthor=J.+M.+Wordauthor=M.+G.+Prisantauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=Structure+validation+by+Calpha+geometry%3A+phi%2C+psi+and+Cbeta+deviation&doi=10.1002%2Fprot.10286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure validation by Cα geometry: φ, ψ and Cβ deviation</span></div><div class="casAuthors">Lovell, Simon C.; Davis, Ian W.; Arendall, W. Bryan, III; de Bakker, Paul I. W.; Word, J. Michael; Prisant, Michael G.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-450</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Geometrical validation around the Cα is described, with a new Cβ measure and updated Ramachandran plot.  Deviation of the obsd. Cβ atom from ideal position provides a single measure encapsulating the major structure-validation information contained in bond angle distortions.  Cβ deviation is sensitive to incompatibilities between sidechain and backbone caused by misfit conformations or inappropriate refinement restraints.  A new .vphi.,ψ plot using d.-dependent smoothing for 81,234 non-Gly, non-Pro, and non-prePro residues with B < 30 from 500 high-resoln. proteins shows sharp boundaries at crit. edges and clear delineation between large empty areas and regions that are allowed but disfavored.  One such region is the γ-turn conformation near +75°, -60°, counted as forbidden by common structure-validation programs; however, it occurs in well-ordered parts of good structures, it is overrepresented near functional sites, and strain is partly compensated by the γ-turn H-bond.  Favored and allowed .vphi.,ψ regions are also defined for Pro, pre-Pro, and Gly (important because Gly .vphi.,ψ angles are more permissive but less accurately detd.).  Details of these accurate empirical distributions are poorly predicted by previous theor. calcns., including a region left of α-helix, which rates as favorable in energy yet rarely occurs.  A proposed factor explaining this discrepancy is that crowding of the two-peptide NHs permits donating only a single H-bond.  New calcns. by H. Hu et al. (2002) for Ala and Gly dipeptides, using mixed quantum mechanics and mol. mechanics, fit our nonrepetitive data in excellent detail.  To run our geometrical evaluations on a user-uploaded file, see MOLPROBITY (http://kinemage.biochem.duke.edu) or RAMPAGE (http://www-cryst.bioc.cam.ac.uk/rampage).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFFimJUYSPl7Vg90H21EOLACvtfcHk0lhnUSioaa14RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpvVKlsA%253D%253D&md5=e9d41070d9e8832ef9aedf6a523dfd73</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fprot.10286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10286%26sid%3Dliteratum%253Aachs%26aulast%3DLovell%26aufirst%3DS.%2BC.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3Dde%2BBakker%26aufirst%3DP.%2BI.%26aulast%3DWord%26aufirst%3DJ.%2BM.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DStructure%2520validation%2520by%2520Calpha%2520geometry%253A%2520phi%252C%2520psi%2520and%2520Cbeta%2520deviation%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D50%26spage%3D437%26epage%3D450%26doi%3D10.1002%2Fprot.10286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiederstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, M. J.</span></span> <span> </span><span class="NLM_article-title">ProSA-web: Interactive web service for the recognition of errors in three-dimensional structures of proteins</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">W407</span>– <span class="NLM_lpage">W410</span>, <span class="refDoi"> DOI: 10.1093/nar/gkm290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1093%2Fnar%2Fgkm290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=17517781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=W407-W410&author=M.+Wiedersteinauthor=M.+J.+Sippl&title=ProSA-web%3A+Interactive+web+service+for+the+recognition+of+errors+in+three-dimensional+structures+of+proteins&doi=10.1093%2Fnar%2Fgkm290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins</span></div><div class="casAuthors">Wiederstein Markus; Sippl Manfred J</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic acids research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">Web Server issue</span>),
    <span class="NLM_cas:pages">W407-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A major problem in structural biology is the recognition of errors in experimental and theoretical models of protein structures.  The ProSA program (Protein Structure Analysis) is an established tool which has a large user base and is frequently employed in the refinement and validation of experimental protein structures and in structure prediction and modeling.  The analysis of protein structures is generally a difficult and cumbersome exercise.  The new service presented here is a straightforward and easy to use extension of the classic ProSA program which exploits the advantages of interactive web-based applications for the display of scores and energy plots that highlight potential problems spotted in protein structures.  In particular, the quality scores of a protein are displayed in the context of all known protein structures and problematic parts of a structure are shown and highlighted in a 3D molecule viewer.  The service specifically addresses the needs encountered in the validation of protein structures obtained from X-ray analysis, NMR spectroscopy and theoretical calculations.  ProSA-web is accessible at https://prosa.services.came.sbg.ac.at.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKn5zO1OOtqL5oR8RchjrrfW6udTcc2eZETuYJ-aD8ubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2svhsVKjug%253D%253D&md5=e2aba9fe44147a9048f37c77306edea6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm290%26sid%3Dliteratum%253Aachs%26aulast%3DWiederstein%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DM.%2BJ.%26atitle%3DProSA-web%253A%2520Interactive%2520web%2520service%2520for%2520the%2520recognition%2520of%2520errors%2520in%2520three-dimensional%2520structures%2520of%2520proteins%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DW407%26epage%3DW410%26doi%3D10.1093%2Fnar%2Fgkm290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span> <i>Molecular Operating Environment (MOE)</i>, 2014.09, <span class="NLM_publisher-name">Chemical Computing Group Inc.</span>, <span class="NLM_publisher-loc">Montreal,
QC, Canada</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+2014.09%2C+Chemical+Computing+Group+Inc.%2C+Montreal%2C%0AQC%2C+Canada%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Więcek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligneau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schunack, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seifert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handzlik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieć-Kononowicz, K.</span></span> <span> </span><span class="NLM_article-title">N-Alkenyl and cycloalkyl carbamates as dual acting histamine H3 and H4 receptor ligands</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2850</span>– <span class="NLM_lpage">2858</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2011.03.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bmc.2011.03.046" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=21498080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFOrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=2850-2858&author=M.+Wi%C4%99cekauthor=T.+Kottkeauthor=X.+Ligneauauthor=W.+Schunackauthor=R.+Seifertauthor=H.+Starkauthor=J.+Handzlikauthor=K.+Kie%C4%87-Kononowicz&title=N-Alkenyl+and+cycloalkyl+carbamates+as+dual+acting+histamine+H3+and+H4+receptor+ligands&doi=10.1016%2Fj.bmc.2011.03.046"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">N-Alkenyl and cycloalkyl carbamates as dual acting histamine H3 and H4 receptor ligands</span></div><div class="casAuthors">Wiecek, Malgorzata; Kottke, Tim; Ligneau, Xavier; Schunack, Walter; Seifert, Roland; Stark, Holger; Handzlik, Jadwiga; Kiec-Kononowicz, Katarzyna</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2850-2858</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Previous studies have shown that several imidazole derivs. posses affinity to histamine H3 and H4 receptors.  Continuing our study on structural requirements responsible for affinity and selectivity for H3/H4 receptor subtypes, two series of 3-(1H-imidazol-4-yl)propyl carbamates were prepd.: a series of unsatd. alkyl derivs. (1-9) and a series possessing a cycloalkyl group different distances to the carbamate moiety (10-13).  The compds. were tested for their affinities at the human histamine H3 receptor, stably expressed in CHO-K1 cells.  Compds. 1, 2, 5-7, 10-13 were investigated for their affinities at the human histamine H4 receptor co-expressed with Gαi2 and Gβ1γ2 subunits in Sf9 cells.  To expand the pharmacol. profile, compds. were further tested for their H3 receptor antagonist activity on guinea pig ileum and in vivo after oral administration to mice.  All tested compds. exhibited good affinity for the human histamine H3 receptor with K i values in the range from 14 to 194 nM.  All compds. were active in vivo after peroral administration (p.o.) to Swiss mice, thus demonstrating their ability to cross the blood-brain barrier.  The most potent H3 receptor ligand of these series was compd. 5, 3-(1H-imidazol-4-yl)propyl pent-4-enylcarbamate with the highest affinity (K i = 14 nM).  Addnl., compd. 3 showed remarkable central nervous system (CNS) H3R activity, increasing the N τ-methylhistamine levels in mice with an ED50 value of 0.55 mg/kg, p.o. evidencing therefore, a twofold increase of inverse agonist/antagonist potency compared to the ref. inverse agonist/antagonist thioperamide.  In this study, the imidazole propyloxy carbamate moiety was kept const.  The different lipophilic moieties connected to the carbamate functionality in the eastern part of the mol. had a range of influences on the human H4 receptor affinity (154-1326 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPAV8EqAjO5rVg90H21EOLACvtfcHk0lgo985_pVhDaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFOrsLw%253D&md5=9b79684afe28bbf9cd62ad96222a8040</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.03.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.03.046%26sid%3Dliteratum%253Aachs%26aulast%3DWi%25C4%2599cek%26aufirst%3DM.%26aulast%3DKottke%26aufirst%3DT.%26aulast%3DLigneau%26aufirst%3DX.%26aulast%3DSchunack%26aufirst%3DW.%26aulast%3DSeifert%26aufirst%3DR.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DHandzlik%26aufirst%3DJ.%26aulast%3DKie%25C4%2587-Kononowicz%26aufirst%3DK.%26atitle%3DN-Alkenyl%2520and%2520cycloalkyl%2520carbamates%2520as%2520dual%2520acting%2520histamine%2520H3%2520and%2520H4%2520receptor%2520ligands%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D2850%26epage%3D2858%26doi%3D10.1016%2Fj.bmc.2011.03.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goulet, J. L.</span>; <span class="NLM_string-name">Holmes, M. A.</span>; <span class="NLM_string-name">Hunt, J. A.</span>; <span class="NLM_string-name">Mills, S. G.</span>; <span class="NLM_string-name">Parsons, W. H.</span>; <span class="NLM_string-name">Sinclair, P. J.</span>; <span class="NLM_string-name">Zaller, D. M.</span></span> <span> </span><span class="NLM_article-title">Src Kinase Inhibitor Compounds</span>. <span class="NLM_patent">WO2001000207A1</span>, <span class="NLM_year">2001</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=J.+L.+Goulet&author=M.+A.+Holmes&author=J.+A.+Hunt&author=S.+G.+Mills&author=W.+H.+Parsons&author=P.+J.+Sinclair&author=D.+M.+Zaller&title=Src+Kinase+Inhibitor+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGoulet%26aufirst%3DJ.%2BL.%26atitle%3DSrc%2520Kinase%2520Inhibitor%2520Compounds%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novotna, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulova, M.</span></span> <span> </span><span class="NLM_article-title">Preparation and characterization of 2-alkylthio-4-(4-bromomethylbenzoyloxy)-6-methylpyrimidines</span>. <i>Chemical Papers</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">656</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaK38XlvFeisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1991&pages=651-656&author=Z.+Novotnaauthor=M.+Koosauthor=M.+Matulova&title=Preparation+and+characterization+of+2-alkylthio-4-%284-bromomethylbenzoyloxy%29-6-methylpyrimidines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation and characterization of 2-alkylthio-4-(4-bromomethylbenzoyloxy)-6-methylpyrimidines</span></div><div class="casAuthors">Novotna, Z.; Koos, M.; Matulova, M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Papers</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">651-6</span>CODEN:
                <span class="NLM_cas:coden">CHPAEG</span>;
        ISSN:<span class="NLM_cas:issn">0366-6352</span>.
    </div><div class="casAbstract">By the reaction of alkylthiohydroxymethylpyrimidines I [R = CnH2n+1, n = 1 - 10] with 4-bromomethylbenzoyl chloride, new alkylthio(bromomethylbenzoyloxy)methylpyrimidines II were prepd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqybfruYCU7ULVg90H21EOLACvtfcHk0lgo985_pVhDaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlvFeisw%253D%253D&md5=694f6837d84a5de92b7ed29265accac8</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNovotna%26aufirst%3DZ.%26aulast%3DKoos%26aufirst%3DM.%26aulast%3DMatulova%26aufirst%3DM.%26atitle%3DPreparation%2520and%2520characterization%2520of%25202-alkylthio-4-%25284-bromomethylbenzoyloxy%2529-6-methylpyrimidines%26jtitle%3DChemical%2520Papers%26date%3D1991%26volume%3D45%26spage%3D651%26epage%3D656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span>; <span class="NLM_string-name">Ermert, P.</span>; <span class="NLM_string-name">Luther, A.</span>; <span class="NLM_string-name">Eberle, M.</span>; <span class="NLM_string-name">Bachmann, F.</span></span> <span> </span><span class="NLM_article-title">Preparation of 2,4,6-Trisubstituted Pyrimidine Derivatives as Apoptosis Inducers for the Treatment of Neoplastic and Autoimmune Diseases</span>. <span class="NLM_patent">WO2004087679A1</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=D.+Obrecht&author=P.+Ermert&author=A.+Luther&author=M.+Eberle&author=F.+Bachmann&title=Preparation+of+2%2C4%2C6-Trisubstituted+Pyrimidine+Derivatives+as+Apoptosis+Inducers+for+the+Treatment+of+Neoplastic+and+Autoimmune+Diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DPreparation%2520of%25202%252C4%252C6-Trisubstituted%2520Pyrimidine%2520Derivatives%2520as%2520Apoptosis%2520Inducers%2520for%2520the%2520Treatment%2520of%2520Neoplastic%2520and%2520Autoimmune%2520Diseases%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacher, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linnemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knolle, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Antiproliferative effects of selective adenosine receptor agonists and antagonists on human lymphocytes: evidence for receptor-independent mechanisms</span>. <i>Purinergic Signalling</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1007/s11302-013-9354-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1007%2Fs11302-013-9354-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=23359122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2qsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=351-365&author=A.+C.+Schiedelauthor=S.+K.+Lacherauthor=C.+Linnemannauthor=P.+A.+Knolleauthor=C.+E.+M%C3%BCller&title=Antiproliferative+effects+of+selective+adenosine+receptor+agonists+and+antagonists+on+human+lymphocytes%3A+evidence+for+receptor-independent+mechanisms&doi=10.1007%2Fs11302-013-9354-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Antiproliferative effects of selective adenosine receptor agonists and antagonists on human lymphocytes: evidence for receptor-independent mechanisms</span></div><div class="casAuthors">Schiedel, Anke C.; Lacher, Svenja K.; Linnemann, Carsten; Knolle, Percy A.; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Purinergic Signalling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">351-365</span>CODEN:
                <span class="NLM_cas:coden">PSUIA9</span>;
        ISSN:<span class="NLM_cas:issn">1573-9538</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The effects of std. adenosine receptor (AR) agonists and antagonists on the proliferation of human T lymphocytes, unstimulated and phytohemagglutinin-stimulated human peripheral blood lymphocytes (PBL), and Jurkat T cells were investigated.  Real-time PCR measurements confirmed the presence of all four AR subtypes on the investigated cells, although at different expression levels.  A2A ARs were predominantly expressed in PBL and further upregulated upon stimulation, while malignant Jurkat T cells showed high expression levels of A1, A2A, and A2B ARs.  Cell proliferation was measured by [3H]-thymidine incorporation assays.  Several ligands, including the subtype-selective agonists CPA (A1), BAY60-6583 (A2B), and IB-MECA (A3), and the antagonists PSB-36 (A1), MSX-2 (A2A), and PSB-10 (A3) significantly inhibited cell proliferation at micromolar concns., which were about three orders of magnitude higher than their AR affinities.  In contrast, further investigated AR ligands, including the agonists NECA (nonselective) and CGS21680 (A2A), and the antagonists preladenant (SCH-420814, A2A), PSB-1115 (A2B), and PSB-603 (A2B) showed no or only minor effects on lymphocyte proliferation.  The anti-proliferative effects of the AR agonists could not be blocked by the corresponding antagonists.  The non-selective AR antagonist caffeine stimulated phytohemagglutinin-activated PBL with an EC50 value of 104 μM.  This is the first study to compare a complete set of commonly used AR ligands for all subtypes on lymphocyte proliferation.  Our results strongly suggest that these compds. induce an inhibition of lymphocyte proliferation and cell death through AR-independent mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHmT-RxH31BbVg90H21EOLACvtfcHk0ljeBoXx-wgUmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2qsrzE&md5=b454b6b4ad9c9a0c226fc4f40c8bbb14</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1007%2Fs11302-013-9354-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11302-013-9354-7%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DLacher%26aufirst%3DS.%2BK.%26aulast%3DLinnemann%26aufirst%3DC.%26aulast%3DKnolle%26aufirst%3DP.%2BA.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DAntiproliferative%2520effects%2520of%2520selective%2520adenosine%2520receptor%2520agonists%2520and%2520antagonists%2520on%2520human%2520lymphocytes%253A%2520evidence%2520for%2520receptor-independent%2520mechanisms%26jtitle%3DPurinergic%2520Signalling%26date%3D2013%26volume%3D9%26spage%3D351%26epage%3D365%26doi%3D10.1007%2Fs11302-013-9354-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borroto-Escuela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ammon, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Filippo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danish, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacher, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Červinková, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafehi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuxe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Adenosine A<sub>2A</sub> receptor ligand recognition and signaling is blocked by A<sub>2B</sub> receptors</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">13593</span>– <span class="NLM_lpage">13611</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.24423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.18632%2Foncotarget.24423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=29568380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC1MnktVaqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=13593-13611&author=S.+Hinzauthor=G.+Navarroauthor=D.+Borroto-Escuelaauthor=B.+F.+Seibtauthor=Y.+C.+Ammonauthor=E.+de+Filippoauthor=A.+Danishauthor=S.+K.+Lacherauthor=B.+%C4%8Cervinkov%C3%A1author=M.+Rafehiauthor=K.+Fuxeauthor=A.+C.+Schiedelauthor=R.+Francoauthor=C.+E.+M%C3%BCller&title=Adenosine+A2A+receptor+ligand+recognition+and+signaling+is+blocked+by+A2B+receptors&doi=10.18632%2Foncotarget.24423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine A2A receptor ligand recognition and signaling is blocked by A2B receptors</span></div><div class="casAuthors">Hinz Sonja; Seibt Benjamin F; Ammon York-Christoph; de Filippo Elisabetta; Danish Azeem; Lacher Svenja K; Cervinkova Barbora; Rafehi Muhammad; Schiedel Anke C; Muller Christa E; Navarro Gemma; Franco Rafael; Navarro Gemma; Franco Rafael; Borroto-Escuela Dasiel; Fuxe Kjell</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">13593-13611</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The adenosine receptor (AR) subtypes A2A and A2B are rhodopsin-like Gs protein-coupled receptors whose expression is highly regulated under pathological, e.g. hypoxic, ischemic and inflammatory conditions.  Both receptors play important roles in inflammatory and neurodegenerative diseases, are blocked by caffeine, and have now become major drug targets in immuno-oncology.  By Forster resonance energy transfer (FRET), bioluminescence resonance energy transfer (BRET), bimolecular fluorescence complementation (BiFC) and proximity ligation assays (PLA) we demonstrated A2A-A2BAR heteromeric complex formation.  Moreover we observed a dramatically altered pharmacology of the A2AAR when co-expressed with the A2BAR (A2B ≥ A2A) in recombinant as well as in native cells.  In the presence of A2BARs, A2A-selective ligands lost high affinity binding to A2AARs and displayed strongly reduced potency in cAMP accumulation and dynamic mass redistribution (DMR) assays.  These results have major implications for the use of A2AAR ligands as drugs as they will fail to modulate the receptor in an A2A-A2B heteromer context.  Accordingly, A2A-A2BAR heteromers represent novel pharmacological targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRatiF0vPAX31MGsG_HTXYFfW6udTcc2eamYITk7YLpM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnktVaqtg%253D%253D&md5=99d64eaa7ad0a0a688fda90a7d526b7a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.24423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.24423%26sid%3Dliteratum%253Aachs%26aulast%3DHinz%26aufirst%3DS.%26aulast%3DNavarro%26aufirst%3DG.%26aulast%3DBorroto-Escuela%26aufirst%3DD.%26aulast%3DSeibt%26aufirst%3DB.%2BF.%26aulast%3DAmmon%26aufirst%3DY.%2BC.%26aulast%3Dde%2BFilippo%26aufirst%3DE.%26aulast%3DDanish%26aufirst%3DA.%26aulast%3DLacher%26aufirst%3DS.%2BK.%26aulast%3D%25C4%258Cervinkov%25C3%25A1%26aufirst%3DB.%26aulast%3DRafehi%26aufirst%3DM.%26aulast%3DFuxe%26aufirst%3DK.%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DFranco%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DAdenosine%2520A2A%2520receptor%2520ligand%2520recognition%2520and%2520signaling%2520is%2520blocked%2520by%2520A2B%2520receptors%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D13593%26epage%3D13611%26doi%3D10.18632%2Foncotarget.24423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gollos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karcz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiene, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heisig, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrenswerth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieć-Kononowicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Fluorescent-labeled selective adenosine A<sub>2B</sub> receptor antagonist enables competition binding assay by flow cytometry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4301</span>– <span class="NLM_lpage">4316</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjlt1Onsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4301-4316&author=M.+K%C3%B6seauthor=S.+Gollosauthor=T.+Karczauthor=A.+Fieneauthor=F.+Heisigauthor=A.+Behrenswerthauthor=K.+Kie%C4%87-Kononowiczauthor=V.+Namasivayamauthor=C.+E.+M%C3%BCller&title=Fluorescent-labeled+selective+adenosine+A2B+receptor+antagonist+enables+competition+binding+assay+by+flow+cytometry&doi=10.1021%2Facs.jmedchem.7b01627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent-Labeled Selective Adenosine A2B Receptor Antagonist Enables Competition Binding Assay by Flow Cytometry</span></div><div class="casAuthors">Kose Meryem; Gollos Sabrina; Fiene Amelie; Heisig Fabian; Behrenswerth Andrea; Namasivayam Vigneshwaran; Muller Christa E; Karcz Tadeusz; Kiec-Kononowicz Katarzyna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4301-4316</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fluorescent ligands represent powerful tools for biological studies and are considered attractive alternatives to radioligands.  In this study, we developed fluorescent antagonists for A2B adenosine receptors (A2BARs), which are targeted by antiasthmatic xanthines and were proposed as novel targets in immuno-oncology.  Our approach was to merge a small borondipyrromethene (BODIPY) derivative with the pharmacophore of 8-substituted xanthine derivatives.  On the basis of the design, synthesis, and evaluation of model compounds, several fluorescent ligands were synthesized.  Compound 29 (PSB-12105), which displayed high affinity for human, rat, and mouse A2BARs ( Ki = 0.2-2 nM) and high selectivity for this AR subtype, was selected for further studies.  A homology model of the human A2BAR was generated, and docking studies were performed.  Moreover, 29 allowed us to establish a homogeneous receptor-ligand binding assay using flow cytometry.  These compounds constitute the first potent, selective fluorescent A2BAR ligands and are anticipated to be useful for a variety of applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShJSPcTqEx0ZVVjGoY74BafW6udTcc2eamYITk7YLpM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjlt1Onsg%253D%253D&md5=9d595306619272562da897096f630b92</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01627%26sid%3Dliteratum%253Aachs%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DGollos%26aufirst%3DS.%26aulast%3DKarcz%26aufirst%3DT.%26aulast%3DFiene%26aufirst%3DA.%26aulast%3DHeisig%26aufirst%3DF.%26aulast%3DBehrenswerth%26aufirst%3DA.%26aulast%3DKie%25C4%2587-Kononowicz%26aufirst%3DK.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DFluorescent-labeled%2520selective%2520adenosine%2520A2B%2520receptor%2520antagonist%2520enables%2520competition%2520binding%2520assay%2520by%2520flow%2520cytometry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4301%26epage%3D4316%26doi%3D10.1021%2Facs.jmedchem.7b01627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdelrahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Tayeb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Ryck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Characterization of P2X4 receptor agonists and antagonists by calcium influx and radioligand binding studies</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2016.11.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1016%2Fj.bcp.2016.11.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=27867013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFSiu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=41-54&author=A.+Abdelrahmanauthor=V.+Namasivayamauthor=S.+Hinzauthor=A.+C.+Schiedelauthor=M.+K%C3%B6seauthor=M.+Burtonauthor=A.+El-Tayebauthor=M.+Gillardauthor=J.+Bajorathauthor=M.+de+Ryckauthor=C.+E.+M%C3%BCller&title=Characterization+of+P2X4+receptor+agonists+and+antagonists+by+calcium+influx+and+radioligand+binding+studies&doi=10.1016%2Fj.bcp.2016.11.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of P2X4 receptor agonists and antagonists by calcium influx and radioligand binding studies</span></div><div class="casAuthors">Abdelrahman, Aliaa; Namasivayam, Vigneshwaran; Hinz, Sonja; Schiedel, Anke C.; Koese, Meryem; Burton, Maggi; El-Tayeb, Ali; Gillard, Michel; Bajorath, Juergen; de Ryck, Marc; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">41-54</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antagonists for ATP-activated P2X4 ion channel receptors are currently in the focus as novel drug targets, in particular for the treatment of neuropathic and inflammatory pain.  We stably expressed the human, rat and mouse P2X4 receptors in 1321N1 astrocytoma cells, which is devoid of functional nucleotide receptors, by retroviral transfection, and established monoclonal cell lines.  Calcium flux assay conditions were optimized for high-throughput screening resulting in a Z'-factor of >0.8.  The application of ready-to-use frozen cells did not neg. affect the results of the calcium assays, which is of great advantage for the screening of compd. libraries.  Species differences were obsd., the rat P2X4 receptor being particularly insensitive to many ATP derivs.  Membrane prepns. of the cell lines showed high levels of specific [35S]ATPγS binding with low nonspecific binding (<5% of total binding), while non-transfected cells were devoid of specific binding sites for the radioligand.  Conditions were employed which allow binding studies to be performed at room temp.  While a variety of nucleotide-derived agonists and the antagonist TNP-ATP displaced [35S]ATPγS from its binding site at human P2X4 receptors, the non-nucleotidic antagonists paroxetine and 5-BDBD did not compete with radioligand binding and were therefore characterized as allosteric antagonists.  Homol. modeling was applied to find an explanation for the obsd. species differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2TlVF8aUxbVg90H21EOLACvtfcHk0lgYwTteJVIijg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFSiu77F&md5=2229ea59f1e799653bd873720675a6b9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2016.11.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2016.11.016%26sid%3Dliteratum%253Aachs%26aulast%3DAbdelrahman%26aufirst%3DA.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DHinz%26aufirst%3DS.%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DBurton%26aufirst%3DM.%26aulast%3DEl-Tayeb%26aufirst%3DA.%26aulast%3DGillard%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3Dde%2BRyck%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DCharacterization%2520of%2520P2X4%2520receptor%2520agonists%2520and%2520antagonists%2520by%2520calcium%2520influx%2520and%2520radioligand%2520binding%2520studies%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D125%26spage%3D41%26epage%3D54%26doi%3D10.1016%2Fj.bcp.2016.11.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. L.</span></span> <span> </span><span class="NLM_article-title">Comparative protein modelling by satisfaction of spatial restraints</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1006%2Fjmbi.1993.1626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=8254673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=779-815&author=A.+Saliauthor=T.+L.+Blundell&title=Comparative+protein+modelling+by+satisfaction+of+spatial+restraints&doi=10.1006%2Fjmbi.1993.1626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative protein modeling by satisfaction of spatial restraints</span></div><div class="casAuthors">Sali, Andrej; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">779-815</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The authors describe a comparative protein modeling method designed to find the most probable structure for a sequence given its alignment with related structures.  The three-dimensional (3D) model is obtained by optimally satisfying spatial restraints derived from the alignment and expressed as probability d. functions (pdfs) for the features restrained.  For example, the probabilities for main-chain conformations of a modelled residue may be restrained by its residue type, main-chain conformation of an equiv. residue in a related protein, and the local similarity between the two sequences.  Several such pdfs are obtained from the correlations between structural features in 17 families of homologous proteins which have been aligned on the basis of their 3D structures.  The pdfs restrain Cα-Cα distances, main-chain N-O distances, main-chain and side-chain dihedral angles.  A smoothing procedure is used in the derivation of these relationships to minimize the problem of a sparse database.  The 3D model of a protein is obtained by optimization of the mol. pdf such that the model violates the input restraints as little as possible.  The mol. pdf is derived as a combination of pdfs restraining individual spatial features of the whole mol.  The optimization procedure is a variable target function method that applies the conjugate gradients algorithm to positions of all non-hydrogen atoms.  The method is automated and is illustrated by the modeling of trypsin from two other serine proteinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC4wGhwZvib7Vg90H21EOLACvtfcHk0lgYwTteJVIijg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ylug%253D%253D&md5=d4a3c39b2205e36221dc187a3d1a478b</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1626%26sid%3Dliteratum%253Aachs%26aulast%3DSali%26aufirst%3DA.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DComparative%2520protein%2520modelling%2520by%2520satisfaction%2520of%2520spatial%2520restraints%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D779%26epage%3D815%26doi%3D10.1006%2Fjmbi.1993.1626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Protein structure modeling with MODELLER</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1137</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-0366-5_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1007%2F978-1-4939-0366-5_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=24573470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1137&publication_year=2014&pages=1-15&author=B.+Webbauthor=A.+Sali&title=Protein+structure+modeling+with+MODELLER&doi=10.1007%2F978-1-4939-0366-5_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Structure Modeling with MODELLER</span></div><div class="casAuthors">Webb, Benjamin; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1137</span>
        (<span class="NLM_cas:issue">Protein Structure Prediction</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Genome sequencing projects have resulted in a rapid increase in the no. of known protein sequences.  In contrast, only about one-hundredth of these sequences have been characterized at at. resoln. using exptl. structure detn. methods.  Computational protein structure modeling techniques have the potential to bridge this sequence-structure gap.  In this chapter, we present an example that illustrates the use of MODELLER to construct a comparative model for a protein with unknown structure.  Automation of a similar protocol has resulted in models of useful accuracy for domains in more than half of all known protein sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs6xvMHHyK0bVg90H21EOLACvtfcHk0lgYwTteJVIijg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns12isrc%253D&md5=19709a3ffefe33fd6e5d8506258bd6f0</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-0366-5_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-0366-5_1%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DB.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DProtein%2520structure%2520modeling%2520with%2520MODELLER%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2014%26volume%3D1137%26spage%3D1%26epage%3D15%26doi%3D10.1007%2F978-1-4939-0366-5_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sievers, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dineen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remmert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Söding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, D. G.</span></span> <span> </span><span class="NLM_article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">539</span>, <span class="refDoi"> DOI: 10.1038/msb.2011.75</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1038%2Fmsb.2011.75" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=21988835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=539&author=F.+Sieversauthor=A.+Wilmauthor=D.+Dineenauthor=T.+J.+Gibsonauthor=K.+Karplusauthor=W.+Liauthor=R.+Lopezauthor=H.+McWilliamauthor=M.+Remmertauthor=J.+S%C3%B6dingauthor=J.+D.+Thompsonauthor=D.+G.+Higgins&title=Fast%2C+scalable+generation+of+high-quality+protein+multiple+sequence+alignments+using+Clustal+Omega&doi=10.1038%2Fmsb.2011.75"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</span></div><div class="casAuthors">Sievers Fabian; Wilm Andreas; Dineen David; Gibson Toby J; Karplus Kevin; Li Weizhong; Lopez Rodrigo; McWilliam Hamish; Remmert Michael; Soding Johannes; Thompson Julie D; Higgins Desmond G</div><div class="citationInfo"><span class="NLM_cas:title">Molecular systems biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">539</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Multiple sequence alignments are fundamental to many sequence analysis methods.  Most alignments are computed using the progressive alignment heuristic.  These methods are starting to become a bottleneck in some analysis pipelines when faced with data sets of the size of many thousands of sequences.  Some methods allow computation of larger data sets while sacrificing quality, and others produce high-quality alignments, but scale badly with the number of sequences.  In this paper, we describe a new program called Clustal Omega, which can align virtually any number of protein sequences quickly and that delivers accurate alignments.  The accuracy of the package on smaller test cases is similar to that of the high-quality aligners.  On larger data sets, Clustal Omega outperforms other packages in terms of execution time and quality.  Clustal Omega also has powerful features for adding sequences to and exploiting information in existing alignments, making use of the vast amount of precomputed information in public databases like Pfam.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQD21C9zX5warjnyZkbE8yXfW6udTcc2ebWUsYeF2IRhrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MbgtlantA%253D%253D&md5=27dd8fac22447528e6d09e5fd1066133</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.75%26sid%3Dliteratum%253Aachs%26aulast%3DSievers%26aufirst%3DF.%26aulast%3DWilm%26aufirst%3DA.%26aulast%3DDineen%26aufirst%3DD.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DKarplus%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DRemmert%26aufirst%3DM.%26aulast%3DS%25C3%25B6ding%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26atitle%3DFast%252C%2520scalable%2520generation%2520of%2520high-quality%2520protein%2520multiple%2520sequence%2520alignments%2520using%2520Clustal%2520Omega%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D539%26doi%3D10.1038%2Fmsb.2011.75" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morris, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindstrom, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanner, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belew, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2785</span>– <span class="NLM_lpage">2791</span>, <span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+M.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+automated+docking+with+selective+receptor+flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0lhAlk2bfO0BMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520automated%2520docking%2520with%2520selective%2520receptor%2520flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunther, R.</span></span> <span> </span><span class="NLM_article-title"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="b8c8cbd7f8d9cdccd7dcd7dbd3">[email protected]</a>: a fast flexible molecular docking program based on Swarm intelligence</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2007.00588.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=10.1111%2Fj.1747-0285.2007.00588.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=17986206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVGhsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2007&pages=475-484&author=V.+Namasivayamauthor=R.+Gunther&title=pso%40autodock%3A+a+fast+flexible+molecular+docking+program+based+on+Swarm+intelligence&doi=10.1111%2Fj.1747-0285.2007.00588.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0757544847465253484348444c">[email protected]</a>: a fast flexible molecular docking program based on Swarm intelligence</span></div><div class="casAuthors">Namasivayam, Vigneshwaran; Guenther, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">475-484</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">On the quest of novel therapeutics, mol. docking methods have proven to be valuable tools for screening large libraries of compds. detg. the interactions of potential drugs with the target proteins.  A widely used docking approach is the simulation of the docking process guided by a binding energy function.  On the basis of the mol. docking program AUTODOCK, we present <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="aafaf9e5eaebfffee5eee5e9e1">[email protected]</a> as a tool for fast flexible mol. docking.  Our novel Particle Swarm Optimization (PSO) algorithms varCPSO and varCPSO-Is are suited for rapid docking of highly flexible ligands.  Thus, a ligand with 23 rotatable bonds was successfully docked within as few as 100 000 computing steps (rmsd = 0.87 Å), which corresponds to only 10% of the computing time demanded by AUTODOCK.  In comparison to other docking techniques as GOLD 3.0, DOCK 6.0, FLEXX 2.2.0, AUTODOCK 3.05, and SODOCK, <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="9fcfccd0dfdecacbd0dbd0dcd4">[email protected]</a> provides the smallest rmsd values for 12 in 37 protein-ligand complexes.  The av. rmsd value of 1.4 Å is significantly lower then those obtained with the other docking programs, which are all above 2.0 Å.  Thus, <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="2575766a656470716a616a666e">[email protected]</a> is suggested as a highly efficient docking program in terms of speed and quality for flexible peptide-protein docking and virtual screening studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryKHisUEtA_LVg90H21EOLACvtfcHk0lhAlk2bfO0BMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGhsb3E&md5=a8e0f99d7a51e3b62e42900c05a92b7f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00588.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00588.x%26sid%3Dliteratum%253Aachs%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DGunther%26aufirst%3DR.%26atitle%3Dpso%2540autodock%253A%2520a%2520fast%2520flexible%2520molecular%2520docking%2520program%2520based%2520on%2520Swarm%2520intelligence%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D70%26spage%3D475%26epage%3D484%26doi%3D10.1111%2Fj.1747-0285.2007.00588.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL','PDB','3PBL'); return false;">PDB: 3PBL</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i62"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01339">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_57457"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01339?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01339</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Mass spectrometric analysis of [<sup>3</sup>H]<b>6</b> and of the unlabeled reference compound <b>6</b>; radiochemical purity of [<sup>3</sup>H]<b>6</b> determined by HPLP-UV (254 nm); product specification sheet of [<sup>3</sup>H]<b>6</b>; specific binding of [<sup>3</sup>H]<b>6</b> to CHO-K1 and CHO-hGPR17 membrane preparations; effects of sodium ions on binding of [<sup>3</sup>H]<b>6</b>. NMR and HPLC-UV/MS data for selected compounds. Overall sequence alignment of the human GPR84 and the human dopamine D3 receptor; Ramachandran diagram of the human GPR84 model; sequence–structure compatibility of the human GPR84 model; docked poses of compounds <b>3</b> and <b>7</b> in the binding site of the GPR84 model; competition binding experiments of compounds <b>6</b> and <b>7</b> at native human and rodent tissues and cell lines. Molecular SMILES strings and EC<sub>50</sub> values (CSV) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_001.pdf">PDF</a>)</p></li><li><p class="inline">structure of the human dopamine D3 receptor (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_002.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_001.pdf">jm9b01339_si_001.pdf (4.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01339/suppl_file/jm9b01339_si_002.pdb">jm9b01339_si_002.pdb (348.46 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The authors will release the atomic coordinates of the homology model of the human GPR84 based on the X-ray structure of the human dopamine D3 receptor (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PBL">3PBL</a>) upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01339&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-5%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01339" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01339" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a5a82cc64196f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
