
<html lang="en"     class="pb-page"  data-request-id="4da62c71-399c-4a28-9a2e-d7adbf5d82cb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2020.63.issue-16;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.9b01129"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Practical High-Quality Electrostatic Potential Surfaces for Drug Discovery Using a Graph-Convolutional Deep Neural Network" /></meta><meta name="dc.Creator" content="Prakash Chandra  Rathi" /></meta><meta name="dc.Creator" content="R. Frederick  Ludlow" /></meta><meta name="dc.Creator" content="Marcel L.  Verdonk" /></meta><meta name="dc.Description" content="Inspecting protein and ligand electrostatic potential (ESP) surfaces in order to optimize electrostatic complementarity is a key activity in drug design. These ESP surfaces need to reflect the true..." /></meta><meta name="Description" content="Inspecting protein and ligand electrostatic potential (ESP) surfaces in order to optimize electrostatic complementarity is a key activity in drug design. These ESP surfaces need to reflect the true..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 25, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01129" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01129" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01129" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01129" /></link>
        
    
    

<title>Practical High-Quality Electrostatic Potential Surfaces for Drug Discovery Using a Graph-Convolutional Deep Neural Network | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01129" /></meta><meta property="og:title" content="Practical High-Quality Electrostatic Potential Surfaces for Drug Discovery Using a Graph-Convolutional Deep Neural Network" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0009.jpeg" /></meta><meta property="og:description" content="Inspecting protein and ligand electrostatic potential (ESP) surfaces in order to optimize electrostatic complementarity is a key activity in drug design. These ESP surfaces need to reflect the true electrostatic nature of the molecules, which typically means time-consuming high-level quantum mechanics (QM) calculations are required. For interactive design much faster alternative methods are required. Here, we present a graph convolutional deep neural network (DNN) model, trained on ESP surfaces derived from high quality QM calculations, that generates ESP surfaces for ligands in a fraction of a second. Additionally, we describe a method for constructing fast QM-trained ESP surfaces for proteins. We show that the DNN model generates ESP surfaces that are in good agreement with QM and that the ESP values correlate well with experimental properties relevant to medicinal chemistry. We believe that these high-quality, interactive ESP surfaces form a powerful tool for driving drug discovery programs forward. The trained model and associated code are available from https://github.com/AstexUK/ESP_DNN." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01129"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01129">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01129&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01129&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01129&amp;href=/doi/10.1021/acs.jmedchem.9b01129" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 8778-8790</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01101" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01919" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Practical High-Quality Electrostatic Potential Surfaces for Drug Discovery Using a Graph-Convolutional Deep Neural Network</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Prakash Chandra Rathi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Prakash Chandra Rathi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Prakash+Chandra++Rathi">Prakash Chandra Rathi</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">R. Frederick Ludlow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">R. Frederick Ludlow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=R.+Frederick++Ludlow">R. Frederick Ludlow</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Marcel L. Verdonk</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcel L. Verdonk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#adc0ccdfcec8c183dbc8dfc9c2c3c6edccded9d583cec2c0"><span class="__cf_email__" data-cfemail="0e636f7c6d6b6220786b7c6a6160654e6f7d7a76206d6163">[emailÂ protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcel+L.++Verdonk">Marcel L. Verdonk</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6484-3328" title="Orcid link">http://orcid.org/0000-0002-6484-3328</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01129&amp;href=/doi/10.1021%2Facs.jmedchem.9b01129" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 8778â8790</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 25, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 July 2019</li><li><span class="item_label"><b>Published</b> online</span>25 September 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01129" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01129</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01129"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7328</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">14</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01129" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Practical High-Quality Electrostatic Potential Surfaces for Drug Discovery Using a Graph-Convolutional Deep Neural Network&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Prakash&quot;,&quot;last_name&quot;:&quot;Chandra Rathi&quot;},{&quot;first_name&quot;:&quot;R.&quot;,&quot;last_name&quot;:&quot;Frederick Ludlow&quot;},{&quot;first_name&quot;:&quot;Marcel&quot;,&quot;last_name&quot;:&quot;L. Verdonk&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8778-8790&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01129&quot;},&quot;abstract&quot;:&quot;Inspecting protein and ligand electrostatic potential (ESP) surfaces in order to optimize electrostatic complementarity is a key activity in drug design. These ESP surfaces need to reflect the true electrostatic nature of the molecules, which typically means time-consuming high-level quantum mechanics (QM) calculations are required. For interactive design much faster alternative methods are required. Here, we present a graph convolutional deep neural network (DNN) model, trained on ESP surfaces derived from high quality QM calculations, that generates ESP surfaces for ligands in a fraction of a second. Additionally, we describe a method for constructing fast QM-trained ESP surfaces for proteins. We show that the DNN model generates ESP surfaces that are in good agreement with QM and that the ESP values correlate well with experimental properties relevant to medicinal chemistry. We believe that these high-quality, interactive ESP surfaces form a powerful tool for driving drug discovery programs forward. The tr&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01129&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01129" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01129&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01129" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01129&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01129" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01129&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01129&amp;href=/doi/10.1021/acs.jmedchem.9b01129" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01129" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01129" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01129%26sid%3Dliteratum%253Aachs%26pmid%3D31553186%26genre%3Darticle%26aulast%3DRathi%26date%3D2020%26atitle%3DPractical%2BHigh-Quality%2BElectrostatic%2BPotential%2BSurfaces%2Bfor%2BDrug%2BDiscovery%2BUsing%2Ba%2BGraph-Convolutional%2BDeep%2BNeural%2BNetwork%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D16%26spage%3D8778%26epage%3D8790%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291202" title="Electrostatics">Electrostatics</a>,</li><li><a href="/action/doSearch?ConceptID=290957" title="Molecular modeling">Molecular modeling</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291787" title="Density functional theory">Density functional theory</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/jmcmar.2020.63.issue-16/20200827/jmcmar.2020.63.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inspecting protein and ligand electrostatic potential (ESP) surfaces in order to optimize electrostatic complementarity is a key activity in drug design. These ESP surfaces need to reflect the true electrostatic nature of the molecules, which typically means time-consuming high-level quantum mechanics (QM) calculations are required. For interactive design much faster alternative methods are required. Here, we present a graph convolutional deep neural network (DNN) model, trained on ESP surfaces derived from high quality QM calculations, that generates ESP surfaces for ligands in a fraction of a second. Additionally, we describe a method for constructing fast QM-trained ESP surfaces for proteins. We show that the DNN model generates ESP surfaces that are in good agreement with QM and that the ESP values correlate well with experimental properties relevant to medicinal chemistry. We believe that these high-quality, interactive ESP surfaces form a powerful tool for driving drug discovery programs forward. The trained model and associated code are available from <a href="https://github.com/AstexUK/ESP_DNN" class="extLink">https://github.com/AstexUK/ESP_DNN</a>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/16"><issue-title>Artificial Intelligence in Drug Discovery</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21077" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21077" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The value of electrostatic calculations for understanding and predicting molecular properties has been recognized for decades.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> In particular, it is well-known that molecular electrostatics can be predictive of a moleculeâs chemical reactivity and its ability to form certain types of interactions. A popular way of visualizing the electrostatic nature of molecules is through electrostatic potential (ESP) surfaces.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Molecular ESP surfaces are generated by systematically moving a unit charge probe around the moleculeâs surface. At each surface point, the ESP energy of the probe is calculated and the molecular surface is then colored accordingly.</div><div class="NLM_p">In drug discovery, it is widely accepted that electrostatic complementarity between protein and ligand is critically important in order to obtain optimal affinity and selectivity. Typically, the strategy is to generate ESP surfaces of different ligands and visually inspect and compare them, where possible in the context of the ESP surface of the target protein.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Additionally, several groups have proposed metrics that assess proteinâligand electrostatic complementarity by computationally comparing the ESP surfaces of protein and ligand in the bound state. A good example of such an electrostatic complementarity measure, as well as a review of this field, was reported by Bauer and Mackey recently.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">The utility to drug discovery of ESP surfaces and any derived metrics strongly relies on the quality of the underlying electrostatic model. However, the electrostatics of proteinâligand interactions is complex for a variety of reasons. First, the types of interactions for which electrostatics play a role are not limited to well-known ones like hydrogen bonds but also include more subtle interactions involving aromatic Ï clouds or Ï holes in halogen and sulfur atoms.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6â8)</a> Also, the local environment within a molecule (e.g., electron withdrawing substituents on an aromatic ring) can have a significant effect on an atomâs electrostatic properties. These effects can only be captured accurately with quantum mechanical (QM) calculations, but these methods are computationally expensive, preventing their use as a tool for interactive drug design. This issue can be partially resolved by using precalculated point charges that have been derived from QM calculations. However, this only addresses the calculation of protein ESP surfaces, as a higher-level (slow) calculation is still required for each new ligand.</div><div class="NLM_p">Recently, Bleiziffer and colleagues used random forest regression models, trained on QM calculations for a large set of compounds, to predict atomic point charges for small molecules.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The aim of this study was to generate partial charges suitable for molecular mechanics (MM) force fields rather than for producing practical high-quality ESP surfaces. Only atom-centered point charges were used, so features like Ï holes and lone pairs were not modeled. As the aim was to derive force field point charges, formally charged groups were modeled without taking into account any shielding of charge in an aqueous environment.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Another recent study used machine learning models, trained on QM calculations, to predict the electrostatic properties of Ï holes.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The derived models accurately predict the maximum ESP value of a Ï hole, but the analysis is limited to halogen atoms.</div><div class="NLM_p">Here, we introduce a method for generating near-QM quality ESP surfaces for ligand molecules in a time frame suitable for interactive drug design; i.e. as a chemist or modeler edits a structure in a molecular visualizer, the ESP surfaces can be updated essentially instantaneously, with minimal loss in precision. We achieve this by training a graph-convolutional deep neural network (DNN) against high-quality QM calculations for over 100â¯000 small molecules. The key features of our model are the following: (1) the electrostatics are described by atom-centered point charges as well as off-centered point charges representing p orbitals, lone pairs, and Ï holes; (2) point charges are trained specifically to optimally reproduce high-quality QM ESP surfaces; (3) a simple empirical correction is applied to formally charged groups, mimicking the shielding of the charge in an aqueous environment. We believe this represents an ideal application for machine learning methods because we are in full control of the training set data. Additionally, unlike more context-dependent properties like proteinâligand binding affinity, the predicted point charges only depend on the chemical environment within a molecule. In particular graph-convolutional DNNs should be ideal for learning the rules that govern the distribution of charge within molecules.</div><div class="NLM_p last">In addition, we describe a method for generating protein ESP surfaces that are fully compatible with the ligand ESP surfaces described above. Using an independent test set, we show that the ligand ESP surfaces predicted by the DNN model are in good agreement with high-quality QM and significantly better than ESP surfaces calculated from AM1-BCC<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> point charges. In addition, we show that there is a good correlation between the DNN ESP values and experimental properties relevant to drug design, illustrating how this method could be successfully employed by medicinal chemists and modelers.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section explains how we have developed methods for generating fast, high-quality ESP surfaces for ligands and proteins. In this study, we focus on the validation of the method for generating ligand ESP surfaces, which uses a graph-convolutional DNN model, trained on high-quality QM calculations for >100â¯000 diverse molecules. Validating the protein ESP surfaces is beyond the scope of the current paper.</div><div class="NLM_p">All QM calculations were run using density functional theory (DFT) at the B3LYP/6-311G** level of theory using B3LYP/6-31G* optimized geometries. This is a popular DFT method<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and also the default QM method used by modelers at Astex for investigating molecular conformation and molecular electrostatics. In the remainder of this manuscript, we will refer to this method simply as âDFTâ. The aim of the DNN model is to produce near-DFT-quality ESP surfaces, using a point-charge model to represent molecular electrostatics.</div><div class="NLM_p">We will discuss four different methods for generating molecular ESP surfaces (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>): (1) DFT ESP surface, derived directly from the DFT electron density map and nuclear positions; this will be used as a benchmark for the other methods; (2) DFT-fp ESP surface, a feature-point (fp) charge model optimized to reproduce the DFT ESP surface of a specific molecule; this shows how well DFT ESP surfaces can be described with our point charge model and represents the best possible quality that the DNN-fp model may achieve; (3) DNN-fp ESP surface, a feature-point-charge model predicted by our graph-convolutional DNN model; (4) AM1-BCC ESP surface, a point charge model based on the AM1 method;<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> AM1 is a semiempirical method and AM1-BCC<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> is a popular model for generating point charges to be used in force fields. The DFT-fp and DNN-fp models use point charges on atoms as well as on lone-pair, p-orbital, and Ï-hole features (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section). The AM1-BCC model only has atomic point charges. To assess the quality of an ESP surface produced by a particular method, we use RMSE<sub>V</sub>, the root-mean-square error (RMSE) in the ESP values across the molecular surface, relative to the DFT ESP surface.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overview of the methods for generating molecular ESP surfaces discussed in this paper, using tadalafil as a representative example of the relative quality of DFT-fp, DNN-fp, and AM1-BCC ESP surfaces, compared to a DFT ESP surface. The DNN-fp ESP surface approaches the quality of the DFT-fp ESP surface, which in turn is a very good approximation of the DFT ESP surface. The AM1-BCC ESP surface, although qualitatively similar to the other ESP surfaces, has significant differences to the DFT ESP surface, particularly around the benzodioxole ring of the tadalafil molecule. RMSE<sub>V</sub> values are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Training and Test Set Construction</h3><div class="NLM_p">In order to derive and validate the graph-convolutional DNN model for the calculation of ESP surfaces for small molecules, we constructed a training set of molecules, as well as an independent test set. For both sets, we ensured broad chemical coverage and relevance to medicinal chemistry (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section). For each molecule in the training and test set we (1) generated the moleculeâs electron density map from the DFT calculation, (2) generated a molecular surface, (3) calculated ESP values on the molecular surface from the DFT electron densities and nuclear positions, and (4) fitted excess point charges, Î<i>q</i> (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section), against the DFT ESP values on the molecular surface to create the DFT-fp model.</div><div class="NLM_p last">For the DFT-fp model, the median RMSE<sub>V</sub> value across all molecules in the training set is 3.15 kcal/mol (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>). We believe this level of precision is more than sufficient for the ESP surfaces to be of utility for interactive drug design, for deriving quantitative structureâactivity relationships (QSAR), or for scoring proteinâligand interactions. For comparison, ESP values on surfaces of neutral molecules range from approximately â60 kcal/mol for strong hydrogen-bond acceptors to +60 kcal/mol for strong donors.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Deep Neural Network Model</h3><div class="NLM_p">A DNN model was trained against the DFT-fp charges generated for the training set. <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A shows a plot of the DFT-fp Î<i>q</i> values against the predicted DNN-fp values for the training set. It is clear that the model is able to describe the point charges of the training set molecules well, with a coefficient of determination, <i>R</i><sup>2</sup> = 0.996 and RMSE in the predicted Î<i>q</i> values, RMSE<sub>Î<i>q</i></sub> = 0.010<i>e</i>. Additionally, with an average RMSE<sub>V</sub> of 3.30 kcal/mol across the training set (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>), the DNN-fp ESP surfaces approximate the quality of the DFT-fp ESP surfaces. The DNN-fp model was then used to calculate point charges for all molecules in the external test set. <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B shows that there is good correlation between DNN-fp and DFT-fp Î<i>q</i> values, with <i>R</i><sup>2</sup> = 0.981 and RMSE<sub>Î<i>q</i></sub> = 0.022<i>e</i>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. DFT-fp Î<i>q</i> values are plotted against the DNN-fp Î<i>q</i> values for (A) the training set and (B) the test set. Color of a data point on the correlation plots indicates the number of observations (see the corresponding color bar). The plots are generated using the hexbin function of the Python matplotlib library<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> with a grid size of 200.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For the current work, the point charges predicted by the model are only of indirect relevance, as the primary objective is to generate high-quality ESP surfaces. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> compares the ESP values produced by the DNN-fp model to those obtained directly from the DFT calculation (AM1-BCC results listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> will be discussed below). The average molecular surface RMSE<sub>V</sub> of 3.27 kcal/mol across all compounds in the test set is comparable to the value obtained for the training set and only slightly above that obtained for DFT-fp point charges. It is clear from the results in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> that all feature types are predicted equally well across the test set. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> compares the CPU timings for the DFT calculations and the DNN-fp model calculations as a function of molecular size. For drug-sized molecules (HAC â 30) the DNN-fp model achieves a speedup of approximately 4 orders of magnitude, compared to the DFT method. The DNN-fp model is fast enough for interactive design, and we believe there is significant scope for speeding up the method further should this be required.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Performance of Point Charge Models in Terms of Reproducing ESP Values on the Molecular Surface across the Compounds in the Test Set<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">DFT-fp</th><th class="colsep0 rowsep0" align="center">DNN-fp</th><th class="colsep0 rowsep0" align="center">AM1-BCC</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">â¨RMSE<sub>V</sub>â©Â molecularÂ surface</td><td class="colsep0 rowsep0" align="left">2.81Â (0.44)</td><td class="colsep0 rowsep0" align="left">3.27Â (0.62)</td><td class="colsep0 rowsep0" align="left">7.86Â (1.04)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSE<sub>V</sub> hydrogen features</td><td class="colsep0 rowsep0" align="left">2.89</td><td class="colsep0 rowsep0" align="left">3.41</td><td class="colsep0 rowsep0" align="left">8.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSE<sub>V</sub> lone-pair features</td><td class="colsep0 rowsep0" align="left">3.23</td><td class="colsep0 rowsep0" align="left">3.97</td><td class="colsep0 rowsep0" align="left">6.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSE<sub>V</sub> p-orbital features</td><td class="colsep0 rowsep0" align="left">2.57</td><td class="colsep0 rowsep0" align="left">3.23</td><td class="colsep0 rowsep0" align="left">9.62</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSE<sub>V</sub> Ï-hole features (all)</td><td class="colsep0 rowsep0" align="left">2.92</td><td class="colsep0 rowsep0" align="left">3.46</td><td class="colsep0 rowsep0" align="left">15.68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSE<sub>V</sub>Â Ï-holeÂ featuresÂ (Cl,Â Br,Â I)</td><td class="colsep0 rowsep0" align="left">2.92</td><td class="colsep0 rowsep0" align="left">3.34</td><td class="colsep0 rowsep0" align="left">18.00</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">The table shows the average RMSE<sub>V</sub> value (kcal/mol) across the entire molecular surface, as well as the RMSE<sub>V</sub> values (kcal/mol) for specific feature types, across the test set. Results are shown for DFT-fp ESP surfaces, DNN-fp ESP surfaces, and AM1-BCC ESP surfaces. In each case the DFT ESP surfaces are used as the reference.</p></div></div></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Timings for DFT, DNN-fp, and AM1-BCC calculations as a function of molecular size. The average and standard deviation at each data point are calculated for 10 molecules randomly selected from our internal compound registry. The DFT timings correspond to the runtime for a single point calculation using 6-311G** basis sets. For AM1, the time required for generating a mol2 file with AM1-BCC charges using ANTECHAMBER<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> is reported, and for DNN-fp, the time required for generating a PQR file with charges on atoms and atomic features is reported (the average DNN-fp timing over all molecules is 0.34 s). All calculations were performed on a single core of an Intel Xeon processor E5 v4 (3.00 GHz).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The DNN-fp model produces Î<i>q</i> values for which the DNN-fp ESP surfaces are calculated. We also tested two much simpler formalisms for generating the Î<i>q</i> values as baseline models in order to further understand the relative performance of the DNN-fp model. In the first of these baseline models, we simply set all Î<i>q</i> values to zero. The â¨RMSE<sub>V</sub>â© value we obtain for this baseline model against the test set is 7.66 (2.81) kcal/mol. In the second baseline model, we derive Î<i>q</i> values for all feature group types as the average Î<i>q</i> value obtained from a constrained point charge fitting procedure across all feature groups of that type in the training set (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section). For this second baseline model, the RMSE<sub>V</sub> value against the test set is 8.43 (3.09) kcal/mol. It is clear that the DNN-fp model significantly outperforms the two baseline models.</div><div class="NLM_p">With any machine-learning model it is important to test for any evidence of overfitting, in this case due to similarity between compounds in the test and training sets. Here we took two separate approaches to this. First, for each compound in the test set we identified the most similar compound in the training set (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>). We then examined the RMSE<sub>V</sub> values as a function of the similarity to the nearest training set compound. This showed that, as expected, the quality of the predicted ESP surfaces does improve slightly with increasing similarity to a compound in the training set (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Figure S1</a>). However, even for test set molecules that are distant from molecules in the training set, the predictions are excellent. Second, we assigned a scaffold to each compound in the training and test set (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Figure S2</a>). We then assessed the quality of the ESP surfaces produced for the molecules containing scaffolds that have a representative in the training set vs those molecules whose scaffolds do not. We saw essentially no difference in RMSE<sub>V</sub> values between these two groups of compounds (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Figure S2</a>). We believe these results show that our model can extrapolate to new molecules and new scaffolds.</div><div class="NLM_p">We are aware of two other studies where researchers have used machine learning methods to derive models predicting partial charges or electrostatic potentials. In their study predicting force field point charges for ligands, Bleiziffer and colleagues reported an RMSE of 0.016<i>e</i> on their external test set.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> However, it is difficult to compare this to our RMSE<sub>Î<i>q</i></sub> value of 0.022<i>e</i> directly for two main reasons. First, the two compound sets are very different; our study uses 4814 diverse medicinal chemistry compounds from ChEMBL, whereas Bleiziffer et al. use a set of 1081 approved drug molecules. Second, whereas Bleiziffer et al. predict point charges for each atom, our study predicts Î<i>q</i> values for feature groups (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section). For comparison, in our study the RMSE error in the point charges placed on individual features is 0.014<i>e</i> for atomic (C, N, O, S, P, F, Cl, Br, I, H) features and 0.0045<i>e</i> for nonatomic (lone-pair, p-orbital, Ï-hole) features. The reason why the errors in the individual feature point charges are smaller is because the error in the feature group Î<i>q</i> values is effectively divided among the feature points in the feature group in the same way Î<i>q</i> is (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section).</div><div class="NLM_p">In another study, Heidrich and colleagues derived a machine learning model specifically to predict the potential of Ï holes of halogen atoms.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The authors obtain RMSE<sub>V</sub> = 3.82 kcal/mol for the potential in the direction of the Ï hole in chloro, bromo, and iodo substituents. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> shows the correlation between the DFT ESP value and the ESP value predicted by the DNN-fp model, in the direction of the Ï hole, for all chloro-, bromo-, and iodo-containing compounds in our test set, with RMSE<sub>V</sub> = 3.34 kcal/mol (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; similar plots are provided for other feature types in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>). Again, we cannot compare these two results directly, in this case because of differences in the compound sets (Heidrich et al. do not make their sets available), the level of theory, and the method for calculating the molecular surface. However, it is clear that our global model is able to predict the potential at halogen Ï holes with a similar precision to the highly specific local models obtained by Heidrich et al. for halogen atoms.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DNN-fp ESP values plotted against the corresponding DFT ESP values for Ï holes in chloro (orange), bromo (pink), and iodo (purple) substituents in the test set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For comparison, we also ran AM1<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> calculations and derived AM1-BCC point charges<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section) for all molecules in the training set and the test set. While AM1 calculations are significantly faster than DFT, they are still too slow to be used interactively, particularly for drug-sized compounds (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). By use of DFT as a benchmark, in terms of generating high-quality ESP surfaces, the point charges produced by our model significantly outperform the AM1-BCC point charges (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The particularly high RMSE<sub>V</sub> values for Ï holes could be due to the fact that AM1-BCC only places point charges on atoms. It should be possible to address this by introducing, within the AM1-BCC protocol, off-center point charges representing Ï holes for halogens and sulfur atoms. It is worth noting that the higher quality DNN-fp ESP surfaces are obtained at a fraction of the computational cost required for AM1-BCC calculations (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>); for drug-sized molecules, the DNN-fp model is approximately 100 times faster than the AM1-BCC method.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Property Predictions</h3><div class="NLM_p">Apart from the ability to deliver high-quality molecular ESP surfaces in a time frame useful for interactive design, the utility of the ESP surfaces is ultimately defined by their ability to provide rational interpretations and correlations for key molecular properties. Here, we will look at four examples where molecular ESP surfaces provide correlations with experimentally determined molecular properties and we will assess the performance of our DNN-fp model in each case.</div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8">A.  Hydrogen-Bond Basicity</h4><div class="NLM_p">The first of these properties is the relative strength of hydrogen bond acceptors. Hydrogen bonding is of significant importance to drug design and understanding the relative abilities of different acceptors to interact with hydrogen bond donors is highly desirable within the context of rational drug discovery. Here we will use the hydrogen bond basicity p<i>K</i><sub>BHX</sub>,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> which provides an experimentally determined scale of the relative strengths of different acceptors in terms of their ability to form hydrogen bonds. It was shown that molecular electrostatic potentials can be predictive of hydrogen bond basicity.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> In particular, Kenny and co-workers published a detailed study on a range of acceptor types,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and we will use data from this paper to test if our DNN-fp ESP surfaces also correlate with hydrogen bond basicity. It is outside the scope of the current study to repeat the work reported by Kenny et al. in its entirety. Instead, we selected a few compounds (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>) from the data sets used in that study, representing four series of compounds: pyridines, acetophenones, nitriles, and ethers. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A shows the p<i>K</i><sub>BHX</sub> values plotted against the lone-pair potentials generated using the DNN-fp model for each of these four series of compounds. It is clear that good correlations are observed between the predicted potential for the lone pair and the p<i>K</i><sub>BHX</sub> values for each of the chemical series. It is interesting to note that we have simply used the ESP value in the direction of the lone pair on the molecular surface rather than identifying the minimum value of the ESP around each acceptor, which was the approach adopted by Kenny et al. Our approach still provides good correlations but may have led to bigger differences between chemical series.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Correlations between DNN-fp ESP values and four experimental properties: (A) hydrogen bond basicity p<i>K</i><sub>BHX</sub> of pyridines (orange), acetophenones (blue), nitriles (gray), and ethers (yellow); (B) acidity constant p<i>K</i><sub>a</sub> for pyridines (blue) and carboxylic acids (orange); (C) fXa affinity p<i>K</i><sub>i</sub>, where the orange arrows indicate the prediction for a compound where the ligand phenyl ring in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A is replaced by a 2-pyridine ringl; (D) XIAP affinity pIC<sub>50</sub>, where only indoline compounds (blue points) were used in the regression and where the orange point represents the chloroazaindoline derivate (compound <b>17</b> in the study by Chessari et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">B.  p<i>K</i><sub>a</sub> Values</h4><div class="NLM_p">The second property we investigate is acidity of functional groups, which can affect key medicinal chemistry parameters like solubility and affinity. It was shown that p<i>K</i><sub>a</sub> values of acidic groups correlate with electrostatic potentials, either at the hydrogen atom of the acidic group or at the lone pair of the conjugate base.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20â22)</a> Here, we used data for a number of 4-substituted pyridines and aliphatic carboxylic acids (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>), extracted from a p<i>K</i><sub>a</sub> database,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> to assess whether we see a similar correlation for ESP values generated using our DNN-fp model. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B shows that there is a clear correlation between the lone pair ESP value of the (conjugate) base and the corresponding p<i>K</i><sub>a</sub> value for both the pyridines and carboxylic acids. Similar correlations are also seen when we use the potential of the hydrogen atom of the protonated species (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). As we observed for the hydrogen bond basicity values, separate regression models are required for each functional group. Although outside the scope of the current work, it would seem that graph-convolutional DNN methods could be ideally suited to train atomic environments directly against p<i>K</i><sub>a</sub> values or p<i>K</i><sub>BHX</sub> values to produce global models for these properties.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Correlation between ESP Values on the Molecular Surface against Hydrogen Bond Basicity, Acidity Constants, and ProteinâLigand Binding Affinity<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="7" align="center">HydrogenÂ BondÂ BasicityÂ (p<i>K</i><sub>BHX</sub>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char="."><i>N</i></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DFT</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DNN</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">AM1</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DNN-DFT</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">AM1-DFT</sub><sup arrange="stagger">2</sup></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pyridines <i>V</i><sub>lp</sub><a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">0.981</td><td class="colsep0 rowsep0" align="char" char=".">0.983</td><td class="colsep0 rowsep0" align="char" char=".">0.930</td><td class="colsep0 rowsep0" align="char" char=".">0.969</td><td class="colsep0 rowsep0" align="char" char=".">0.967</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">acetophenones <i>V</i><sub>lp</sub><a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.896</td><td class="colsep0 rowsep0" align="char" char=".">0.869</td><td class="colsep0 rowsep0" align="char" char=".">0.807</td><td class="colsep0 rowsep0" align="char" char=".">0.976</td><td class="colsep0 rowsep0" align="char" char=".">0.959</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nitriles <i>V</i><sub>lp</sub><a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">0.986</td><td class="colsep0 rowsep0" align="char" char=".">0.978</td><td class="colsep0 rowsep0" align="char" char=".">0.929</td><td class="colsep0 rowsep0" align="char" char=".">0.979</td><td class="colsep0 rowsep0" align="char" char=".">0.962</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ethers <i>V</i><sub>lp</sub><a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0.971</td><td class="colsep0 rowsep0" align="char" char=".">0.941</td><td class="colsep0 rowsep0" align="char" char=".">0.927</td><td class="colsep0 rowsep0" align="char" char=".">0.981</td><td class="colsep0 rowsep0" align="char" char=".">0.866</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="7" align="center">AcidityÂ ConstantÂ (p<i>K</i><sub>a</sub>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char="."><i>N</i></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DFT</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DNN</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">AM1</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DNN-DFT</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">AM1-DFT</sub><sup arrange="stagger">2</sup></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pyridines <i>V</i><sub>lp</sub><a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">0.929</td><td class="colsep0 rowsep0" align="char" char=".">0.931</td><td class="colsep0 rowsep0" align="char" char=".">0.900</td><td class="colsep0 rowsep0" align="char" char=".">0.974</td><td class="colsep0 rowsep0" align="char" char=".">0.967</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pyridines <i>V</i><sub>h</sub><a class="ref internalNav" href="#t2fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">0.898</td><td class="colsep0 rowsep0" align="char" char=".">0.869</td><td class="colsep0 rowsep0" align="char" char=".">0.055</td><td class="colsep0 rowsep0" align="char" char=".">0.982</td><td class="colsep0 rowsep0" align="char" char=".">0.093</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carboxylic acids <i>V</i><sub>lp</sub><a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0.674</td><td class="colsep0 rowsep0" align="char" char=".">0.886</td><td class="colsep0 rowsep0" align="char" char=".">0.770</td><td class="colsep0 rowsep0" align="char" char=".">0.470</td><td class="colsep0 rowsep0" align="char" char=".">0.959</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">carboxylic acids <i>V</i><sub>h</sub><a class="ref internalNav" href="#t2fn2" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0.963</td><td class="colsep0 rowsep0" align="char" char=".">0.941</td><td class="colsep0 rowsep0" align="char" char=".">0.948</td><td class="colsep0 rowsep0" align="char" char=".">0.945</td><td class="colsep0 rowsep0" align="char" char=".">0.975</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="8" align="center">ProteinâLigandÂ BindingÂ AffinityÂ (p<i>K</i><sub><i>i</i></sub>/pIC<sub>50</sub>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char="."><i>N</i></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DFT</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DNN</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">AM1</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">Ï</sub><sup arrange="stagger">2</sup>â¯<a class="ref internalNav" href="#t2fn7" aria-label="h">h</a></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">DNN-DFT</sub><sup arrange="stagger">2</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sub arrange="stagger">AM1-DFT</sub><sup arrange="stagger">2</sup></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">fXa <i>V</i><sub>h</sub><a class="ref internalNav" href="#t2fn3" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">0.720</td><td class="colsep0 rowsep0" align="char" char=".">0.757</td><td class="colsep0 rowsep0" align="char" char=".">0.454</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=".">0.897</td><td class="colsep0 rowsep0" align="char" char=".">0.955</td><td class="colsep0 rowsep0" align="char" char=".">0.835</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">fXa â¨<i>V</i>â©<a class="ref internalNav" href="#t2fn4" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">0.855</td><td class="colsep0 rowsep0" align="char" char=".">0.791</td><td class="colsep0 rowsep0" align="char" char=".">0.655</td><td class="colsep0 rowsep0" align="char" char=".">0.957</td><td class="colsep0 rowsep0" align="char" char=".">0.835</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">XIAP <i>V</i><sub>p</sub><a class="ref internalNav" href="#t2fn5" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.715</td><td class="colsep0 rowsep0" align="char" char=".">0.761</td><td class="colsep0 rowsep0" align="char" char=".">0.707</td><td class="colsep0 rowsep0" rowspan="2" align="char" valign="middle" char=".">0.774</td><td class="colsep0 rowsep0" align="char" char=".">0.985</td><td class="colsep0 rowsep0" align="char" char=".">0.955</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">XIAP <i>V</i><sub>h</sub><a class="ref internalNav" href="#t2fn6" aria-label="g">g</a></td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">0.798</td><td class="colsep0 rowsep0" align="char" char=".">0.789</td><td class="colsep0 rowsep0" align="char" char=".">0.721</td><td class="colsep0 rowsep0" align="char" char=".">0.958</td><td class="colsep0 rowsep0" align="char" char=".">0.949</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup><sup>a</sup></sup><p class="last"><i>R</i><sup>2</sup> values are given between the experimental property and the ESP values for DFT, DNN-fp, and AM1-BCC. Additionally, for the DNN-fp and AM1-BCC ESPs, <i>R</i><sup>2</sup> values are given when plotted against the DFT ESPs. Feature ESP values were calculated as outlined in the <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section.</p></div><div class="footnote" id="t2fn1"><sup><sup>b</sup></sup><p class="last">ESP value in the direction of the lone pair. For carboxylic acids, these calculations were on negatively charged molecules.</p></div><div class="footnote" id="t2fn2"><sup><sup>c</sup></sup><p class="last">ESP value in the direction of the acidic proton. For pyridines, these calculations were on positively charged molecules.</p></div><div class="footnote" id="t2fn3"><sup><sup>d</sup></sup><p class="last">ESP value in the direction of the aromatic proton on the phenyl ring (solid circle in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A).</p></div><div class="footnote" id="t2fn4"><sup><sup>e</sup></sup><p class="last">Average ESP value of the aromatic protons and p orbitals for three atoms on the phenyl ring (dashed ellipsoid in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A).</p></div><div class="footnote" id="t2fn5"><sup><sup>f</sup></sup><p class="last">Average ESP value of the aromatic p orbitals for two atoms in the indoline ring (solid ellipsoid in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).</p></div><div class="footnote" id="t2fn6"><sup><sup>g</sup></sup><p class="last">Average ESP value of two protons on a CH2 group in the indoline ring (dashed circle in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).</p></div><div class="footnote" id="t2fn7"><sup><sup>h</sup></sup><p class="last">Hammett substituent constants (<i>Ï</i><sub>m</sub> for fXa and <i>Ï</i><sub>p</sub> for XIAP). We were unable to find some of the Hammett constants used by Young et al.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> in the provided literature reference,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> so the <i>R</i><sup>2</sup> value for fXa was taken directly from their paper. For XIAP, we used the <i>Ï</i><sub>p</sub> values listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> of Chessari et al.âs paper<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> to calculate <i>R</i><sup>2</sup>.</p></div></div></div></div><div id="sec2_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11">C.  Factor Xa Affinity</h4><div class="NLM_p">The third example involves SAR observed for binding affinities against coagulation factor Xa (fXa). <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A shows the protein ESP surface for the fXa active site. We generate protein ESP surfaces using a completely analogous point charge model, trained on DFT calculations at the same level of theory as the ligand ESP surfaces (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section). As a result, protein ESP surfaces can also be generated interactively and are completely compatible with the ligand surfaces. Our focus here is on the P4 pocket, which is faced by the Ï clouds of three aromatic residues. Apart from being lipophilic in nature, this pocket is also quite electronegative and this is clearly visible from the protein ESP surface. The pocket is well-known to interact favorably with electron-poor or even positively charged lipophilic groups on ligands.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Young and colleagues observed this clearly on a series of closely related compounds where a substituted phenyl is placed in the P4 pocket.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The variable R-substituent on the phenyl ring (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A) is mostly pointing toward solvent, which makes this a good test case to assess the effect of the electrostatic nature of the phenyl ring on affinity. The authors hypothesized that the SAR they observed could be driven by the close contact between the electronegative Ï cloud of Trp215 and the aromatic proton on the ligand in the meta position to the R-substituent (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Hence, using our DNN-fp model, we calculated the ESP values <i>V</i><sub>h</sub> in the direction of the CâH proton on the molecular surface of the ligands investigated by Young et al. and plotted them against the observed fXa affinity of the compounds (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Supporting the authorsâ hypothesis, a good correlation is obtained, with <i>R</i><sup>2</sup> = 0.76 and an RMSE between the fitted line and the experimental p<i>K</i><sub>i</sub> values of 0.59. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A illustrates the effect that the substituents have on the ESP of the CâH proton pointing at Trp215. However, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A also shows that the entire phenyl ring becomes more electropositive as the R-substituent gets more electron withdrawing. Therefore, we also tested the correlation of the average ESP value â¨<i>V</i>â© of the three CâH groups interacting with the electronegative bottom of the S4 pocket (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). Interestingly, this ring-average potential provides a comparable <i>R</i><sup>2</sup> value of 0.80 when plotted against the fXa affinity. Hence, it is difficult to conclude that the effect of the R-substituent on fXa affinity is driven entirely by the electrostatic nature of the meta aromatic proton.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. DFT-trained protein ESP surfaces (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section) for (A) the fXa active site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VH6">2VH6</a>) and (B) the XIAP SMAC binding site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C7A">5C7A</a>). For both targets a ligand is shown and the R-substituent location is indicated. Also, the areas where literature hypotheses are focused in terms of how affinity is affected by electrostatics are highlighted as solid white ellipsoids (<i>V</i><sub>h</sub> for fXa and <i>V</i><sub>p</sub> for XIAP in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The alternative/additional hypotheses tested in the current work are highlighted as dashed white ellipsoids (â¨<i>V</i>â© for fXa and <i>V</i><sub>h</sub> for XIAP in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. ESP surfaces generated by our DNN-fp model for (A) three of the fXa compounds reported by Young et al.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and (B) three compounds reported by Chessari et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Solid line circles/ellipsoids indicate the areas the authors proposed to be linked to the correlation between ESPs and affinity. Dashed line circles/ellipsoids represent areas of alternative/additional hypotheses we test here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12">D.  XIAP Binding Affinity</h4><div class="NLM_p">The final property we investigate is binding affinity for the X-linked inhibitor of apoptosis protein (XIAP). <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B shows the ESP surface of the XIAP SMAC binding site. During a fragment-to-leads campaign against this target, Chessari and co-workers identified the weakly binding (IC<sub>50</sub> â 500 Î¼M) early lead compound shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The authors noted that the Ï system of the indoline ring contacts an electronegative patch on the protein ESP surface, which was considered an electrostatic clash; this negative patch is also clearly visible in our DFT-derived ESP surface for XIAP (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Chessari et al. showed that the XIAP affinity can be greatly improved by optimizing the electrostatics of the indoline ring and that affinity tracked very well with the electron donating/withdrawing nature of the R-substituent (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Hence, using our DNN-fp model, we calculated the average ESP value <i>V</i><sub>p</sub> of the two central carbon atomsâ p orbitals in the indoline ring (which directly contact the negative patch on the protein ESP surface) for all R-substituents reported by Chessari at al. and plotted them against the XIAP affinity (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Again, consistent with the postulated hypothesis, we see a good correlation with <i>R</i><sup>2</sup> = 0.76 (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and an RMSE between the fitted line and the experimental pIC<sub>50</sub> values of 0.37. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B illustrates the effect of the R-substituent on the DNN-fp ESP surface of the ligands. It is interesting to note that, as well as affecting the electronics of the aromatic ring, the R-substituent also significantly modulates the potential around the pyrrolidine CH<sub>2</sub> groups (see <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). It is impressive that the DNN-fp model is able to capture this relatively long-range effect, which is also observed in the DFT ESP surfaces (not shown). One of the pyrrolidine CH<sub>2</sub> groups stacks directly against the Ï cloud of Trp323 (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B), so any electrostatic changes in this part of the ligand may also be reflected in the XIAP affinity of the compounds. Therefore, we also plotted the ESP value <i>V</i><sub>h</sub> of the CH<sub>2</sub> hydrogens against the XIAP affinity and obtained a similar <i>R</i><sup>2</sup> value of 0.79 (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). It is difficult to know whether the interaction with the negative patch or with the Trp323 Ï cloud has a larger effect on the XIAP affinity, but it is likely that both contribute.</div><div class="NLM_p">For all molecules used for the property-ESP hypotheses/correlations listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, we also ran a DFT calculation and repeated all the correlations using DFT ESP values. In nearly all cases, the DFT calculations provide comparable correlation coefficients to the DNN-fp model. The only exception is the p<i>K</i><sub>a</sub> prediction from the lone pair ESP values for carboxylic acids, where DFT performs significantly worse (<i>R</i><sup>2</sup> = 0.67) than the DNN-fp model (<i>R</i><sup>2</sup> = 0.89). We are not sure what the exact reason behind this is, but it might be related to the fact that the DFT calculations were done on formally negatively charged molecules, whereas the DNN-fp model uses an empirical model to correct the partial charges it predicts for the neutral species (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section). Importantly, excellent correlation is observed between the DNN-fp ESP values and the DFT ESP values in all cases except, again, the p<i>K</i><sub>a</sub> prediction from the lone pair ESP values for carboxylic acids.</div><div class="NLM_p">The same exercise was repeated using ESP values calculated using AM1-BCC charges (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). In most cases, AM1-BCC provides useful models with correlation coefficients similar to DFT and DNN-fp. One notable exception to this is the p<i>K</i><sub>a</sub> prediction from the proton ESP values for pyridines, where AM1-BCC performs very poorly. The correlation plot (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Figure S9 in the Supporting Information</a>) shows that there appear to be two separate groups of compounds that display distinct correlations. It is beyond the scope of this work to investigate this further, but it is interesting to note that again we are measuring ESP values on formally charged species. Closer inspection of the data in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> reveals that for 11 out of the 12 hypotheses tested our DNN-fp model provides a higher <i>R</i><sup>2</sup> value than the AM1-BCC method (<i>p</i> = 0.003; exact binomial test). We believe that this is a direct consequence of the higher precision achieved by the DNN-fp model. This is supported by the fact that for 10 out of the 12 cases, the correlation with DFT ESP values is better for the DNN-fp model than it is for the AM1-BCC method (<i>p</i> = 0.02).</div><div class="NLM_p">In order to test to what extent molecular conformation affects the quality of the ESP surfaces produced by our DNN-fp model, we generated 100 conformers for the <i>tert</i>-butyl fXa compound in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A. Next, we generated DFT ESP surfaces as well as DNN-fp ESP surfaces for all conformers and calculated the corresponding RMSE<sub>V</sub> values. Across the 100 conformers, the average RMSE<sub>V</sub> value was 4.03 kcal/mol with a standard deviation of 0.22 kcal/mol. For the two most âfolded upâ conformers, the RMSE<sub>V</sub> values were 4.45 and 4.02 kcal/mol. For comparison, the RMSE<sub>V</sub> value for the Corina geometry is 3.90 kcal/mol. These results indicate that the quality of the ESP surfaces is not particularly dependent on the molecular conformation.</div><div class="NLM_p">Correlation plots such as those in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D can be used to predict the SAR of closely related series of compounds. These examples illustrate how we believe this methodology will be at its most powerful: for the understanding and prediction of local SAR, e.g., modifying a single small substituent or variation of heteroatoms in an aromatic ring. We do not expect the ESP surfaces in isolation to be particularly useful as a general scoring function for predicting proteinâligand binding, because of confounding factors like lipophilic interactions, water molecules and ligand conformational effects.</div><div class="NLM_p">Both Young and colleagues and Chessari et al. reported good correlations between affinity and Hammett substituent constants<a onclick="showRef(event, 'ref26 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref28">(26,28)</a> for the fXa and XIAP examples, respectively.<a onclick="showRef(event, 'ref25 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref27">(25,27)</a> It is clear from <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> that the <i>R</i><sup>2</sup> values obtained for the Hammett constants are comparable to the ones we obtain using ESP values. However, although Hammett constants are clearly very valuable for assessing SAR, there are key advantages to using ESPs. Most importantly, the use of ESPs is not restricted to studying substituents on a single aromatic ring type. For example, if we consider a virtual compound designed against fXa where the ligand phenyl ring in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A is replaced by a 2-pyridine ring, i.e., where the aromatic carbon forming the attachment point for the R-group is replaced by a nitrogen atom, we cannot use standard benzoic acid derived Hammett constants to make predictions about its affinity. However, using the DDN-fp model, we can simply calculate <i>V</i><sub>h</sub> (orange line in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C) and use it to predict that this compound would have p<i>K</i>i(fXa) â 9. Similarly, Chessari et al. also reported the XIAP affinity of the chloroazaindoline derivate (compound <b>17</b> in their study). This compound could not be included in the correlation plot the authors reported against the Hammett constant. However, using the DNN-fp ESPs, we can add the compound to the correlation plot in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D (orange circle) and demonstrate that it fits the trend observed for the other compounds. Another advantage of the use of ESPs over Hammett constants is that the ESP surfaces provide a much more detailed spatial view of the electrostatics of the ligand in the context of those of the protein, allowing more specific hypothesis generation.</div><div class="NLM_p last">The correlations observed for proteinâligand affinity are weaker than they are for hydrogen bond basicity and p<i>K</i><sub>a</sub> values. This is to be expected as the latter two properties depend more directly on the electrostatic properties of the molecules. For proteinâligand binding, other factors will generally play a role. For example, for fXa the R-substituent may form additional interactions with the protein, affect the conformation of the ligand, or change the water structure around that part of the active site, which will perturb the correlation between affinity and the electrostatics of the ligand phenyl ring. In fact, for most proteinâligand affinity SAR examples, these perturbing factors will be more significant, and the relatively clean and direct correlations we observe between affinity and ESP values for fXa and XIAP are probably quite unusual. However, that in no way diminishes the utility to drug discovery of high-quality ESP surfaces for proteins and ligands. Even in a multifactorial SAR scenario, these surfaces provide an excellent indication of the electrostatic complementarity between protein and ligand, and how the ligand may be altered in order to optimize affinity for the protein.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64643" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64643" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have developed methods for generating fast, high-quality ESP surfaces for proteins and ligands. For ligands, we have trained a graph-convolutional DNN model against DFT ESP surfaces for over 100â¯000 diverse molecules. Testing the model against an independent test set of molecules relevant to medicinal chemistry showed that it provides good agreement with DFT ESP surfaces. We showed that in terms of the quality of the ESP surfaces, the DNN-fp model significantly outperforms the AM1-BCC model. The DNN-fp model generates ESP surfaces in a fraction of a second, thus fully enabling their use for interactive drug design. For a drug-sized molecule, the AM1-BCC method is approximately 100-fold slower and DFT ESP surfaces are 4 orders of magnitude slower to generate. For proteins we developed a method for generating fast DFT-trained ESP surfaces that are fully compatible with the ligand DNN-fp ESP surfaces. Next, we assessed the performance of our DNN-fp model in terms of generating ligand ESPs that correlate with experimentally measured molecular properties. These included hydrogen bond basicity, acidity constants, and two examples of proteinâligand binding affinity. In each case, a strong correlation was observed between the DNN-fp ESP values and the experimental property. We showed that correlation coefficients were generally comparable to those obtained with DFT ESP values and better than those observed for AM1-BCC ESP values. We believe this new method, which we make freely available via <a href="https://github.com/AstexUK/ESP_DNN" class="extLink">https://github.com/AstexUK/ESP_DNN</a>, will form a powerful tool for interactive drug design.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Constructing Data Sets</h3><div class="NLM_p">Training set molecules were selected from a superset of commercially available compounds in eMolecules<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and the Sigma-Aldrich<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> catalogue in a way to maximize the coverage of different chemical environments. Molecules with HAC â¤ 17 and consisting exclusively of elements B, Br, C, Cl, F, H, I, N, O, P, and S were considered. All molecules were neutralized, and up to eight tautomers were generated for each molecule using the MolVS<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Python library built upon the RDkit chemoinformatics toolkit (version 2017.09.1).<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> This resulted in a set of 1â¯336â¯480 molecules for further filtering. The local chemical environment of an atom was represented as atom-centered fragments with a varying radius for including neighboring atoms. As such, fragments of increasing radii between zero (only the central atom) and six (the central atom and all atoms that are up to six bonds away) were created for all molecules in the set. All such fragments that occur at least 10 times across the set were considered further. The smallest molecule that contained a particular fragment was set as representative for the fragment type. Next, the smallest set of representative molecules that includes all chemical environments of radius up to 2 was selected. As the number of chemical environments rapidly increases with the radius of the fragment, it was impractical to include all chemical environments with a radius larger than 2, and hence only a subset of chemical environments was included. We selected a subset of molecules that cover 90% of all the atoms in the set in terms of atomic environments with radius 3. Similarly, we selected subsets of molecules covering 85%, 60%, and 50% of all atoms in the set in terms of chemical environments of radii 4, 5, and 6, respectively. This resulted in a training set of 100â¯500 molecules.</div><div class="NLM_p last">An external test set of diverse lead-like molecules was constructed from the ChEMBL database version 20.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Lead-sized ChEMBL molecules (10 â¤ HAC â¤ 25) that have binding affinity of <1 Î¼M against at least one target were candidates for the test set. Molecules were grouped according to HAC, and for each group K-medoids clustering was performed to create a predetermined number of clusters using the pyclust library for Python.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Pairwise Dice-distance between the Morgan fingerprints (radius = 3) of two molecules, calculated using the RDkit toolkit, was used as the distance criteria for K-medoid clustering. For each group of molecules (based on HAC), the number of clusters was set to 2.5% of the total number of molecules in the group with lower and upper bounds on the number of clusters set to 100 and 500, respectively. Cluster representatives from each cluster formed the test set. Similar to the training set, up to eight tautomers for each molecule in the test set were generated. This resulted in a test set of 4815 molecules.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Feature Groups</h3><div class="NLM_p">Our approach describes each non-hydrogen atom in a molecule as a feature group. These feature groups consist of a set of features: the parent atom itself, its bound hydrogen atoms, and additional features representing lone pairs, p-orbitals, or Ï-holes, each of which carries a point charge. Atom typing is achieved via feature group definitions. Each feature group definition uses a SMARTS<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> pattern to assign a feature group type to any atom that matches that pattern. A feature group type provides the rules for turning a matching atom into a feature group; i.e., it specifies which features are to be added and what their relative geometries are.</div><div class="NLM_p">A set of small neutral model compounds (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>) was used to prepare all feature group types and feature group definitions. For each model compound, DFT electron density isosurfaces were generated using the method described below. Each feature group type is defined by a specific model compound. For example, ethane defines the ââCH<sub>3</sub>â feature group type, methanol the ââOHâ feature group type, etc. Parent atom features and hydrogen atom features are placed exactly on the corresponding atoms. For each feature group type, we then decided which additional features to include, as well as the directions they point in, taking into account atomic orbitals, local symmetry, and the location of electrostatic extrema on the DFT ESP isosurface.</div><div class="NLM_p">Having decided on the featuresâ relative directions, we then identified the optimal separation distance between parent atom and nonatomic features. In order to achieve this, we first measured the distance <i>D</i><sub>f</sub> from the parent atom to the ESP surface in the direction of each of the features. Next, we placed all nonatomic features at a distance <i>L</i><sub>f</sub> = <i>C</i>Â·<i>D</i><sub>f</sub> from the parent atom, in the predefined directions. We then systematically varied <i>C</i>, calculating RMSE<sub>V</sub> using unconstrained point charges (see below) for all model compounds and found that <i>C</i> = 0.5 is optimal. Hence, for all results presented here, we used <i>L</i><sub>f</sub> = 0.5<i>D</i><sub>f</sub>. The model compounds were also used to set featuresâ van der Waals radii <i>R</i><sub>f</sub> for all feature group types. For each feature in a feature group type, except the parent atom feature, we used <i>R</i><sub>f</sub> = <i>D</i><sub>f</sub> â <i>L</i><sub>f</sub>. For the parent atom feature itself, we used the distance from the atom to the closest isosurface point that is at least 40Â° away from every nonatomic feature in the feature group.</div><div class="NLM_p">For our DNN-fp representation, point charges of individual features are constrained to the point charges of the other features within the same feature group. This is achieved by defining the charge on feature <i>i</i> in a feature group as follows:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_m001.gif" alt="" id="_i17" /></img></span>where <i>q</i><sub class="stack"><i>i</i></sub><sup class="stack">o</sup> is the reference charge of feature <i>i</i>, Î<i>q</i> is the excess charge on the feature group, and <i>f</i><sub><i>i</i></sub> is the fraction of Î<i>q</i> that is assigned to feature <i>i</i>. For all the work we describe here, we used <i>f</i><sub><i>i</i></sub> = 1/<i>n</i> where <i>n</i> is the number of features in the feature group. This way, the excess charge is distributed uniformly across the features in the feature group. We have found that constraining the featuresâ point charges within a feature group, relative to each other, prevents overfitting and reduces noise levels in the fitted point charges, thus providing the machine learning method the optimal opportunity to detect the true signal in the training set.</div><div class="NLM_p">Reference charges are normalized such that<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_m002.gif" alt="" id="_i18" /></img></span>Reference charges are constants that form part of feature group definitions and were derived from the molecules in the training set. First, unconstrained point charges were generated for all molecules in the training set (see below). Next, for each feature group type, e.g., ââCH<sub>3</sub>â, we calculated the median point charge placed on each feature type (âCâ and âHâ) across the training set. These median charges were then normalized such that they sum up to zero across the feature group.</div><div class="NLM_p last">As the model compounds are all neutral molecules, we still need to assign reference charges to formally charged feature group types. For some feature group types we can simply copy the reference charges from the corresponding neutral feature group. For example, the ââOpâ feature group type (with associated reference charges) is assigned to both neutral carbonyl atoms, as well as negatively charged carboxylate oxygen atoms. However, a number of formally charged feature group types do not have a direct neutral equivalent. For such feature group types, we derived reference charges from the corresponding neutral species. In this simple two-step, highly empirical process, using the ââNH<sub>3</sub>â feature group as an example, we first take the reference charges for the nitrogen (â0.26) and the hydrogens (+0.19) from the ââNH<sub>3</sub>â feature group definition. We cannot use these charges as reference charges for the ââNH<sub>3</sub>â feature group directly because they do not add up to zero but instead to Î<i>q</i><sup>o</sup> = â0.26 + 3Â·0.19 = 0.31. Therefore, in the second step, the reference charges are renormalized by simply subtracting 0.25Â·Î<i>q</i><sup>o</sup> = 0.0775 from each of the feature point reference charges such that the sum over the new reference charges is zero. The newly derived reference charges for the NH<sub>3</sub> feature group are then â0.33 for the nitrogen and +0.11 for the hydrogens.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> ESP Surfaces</h3><div class="NLM_p">3D structures of the model compounds, as well as all molecules in the training set and test set, were generated using the CORINA program (version 4.2.0) with default settings.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Next, 3D DFT ESP maps were generated for all molecules using the Q-Chem software package (version 5.0),<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> using the DFT method at the B3LYP/6-311G** level of theory and the B3LYP/6-31G* optimized geometries. The grid size used for the ESP maps was 0.25 Ã. For all model compounds, we defined the molecular surface directly from the DFT maps as the electron density isosurface at 0.002 au. For the training and test set molecules and all molecules used in the examples we discuss, we generated PQR files<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> containing the positions and radii of all atomic and nonatomic features. Next, we converted the PQR files into Connolly surfaces<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> using the NGL<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> viewerâs âavâ surface mode.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows the four different methods for producing molecular ESP surfaces that are discussed in this work. DFT ESP surfaces were generated by trilinear interpolation of DFT ESP maps for each point of the Connolly surface. The resulting DFT ESP surfaces were considered ground truth for this work.</div><div class="NLM_p">DFT-fp ESP surfaces are based on feature point (fp) charge representations that provide an optimal fit of the DFT ESP surface. As such, a least-squares fitting to the DFT ESP surfaces was used to derive partial charges on all feature positions in a molecule with a constraint that the total charge of the molecule must add up to zero. The partial charges model we use to describe the ESPs uses the classical Coulomb equation:<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_m003.gif" alt="" id="_i20" /></img></span>where Îµ<sub>0</sub> is the relative permittivity (we used a distance dependent relative permittivity Îµ<sub>0</sub> = <i>r</i>) and <i>q</i><sub><i>i</i></sub> is the partial charge on atom <i>i</i>. Our rationale for using a distance dependent relative permittivity here is twofold. First, it provides better fits against the DFT ESP surfaces (for the set of model compounds, RMSE<sub>V</sub> = 2.64 kcal/mol using Îµ<sub>0</sub> = <i>r</i>, whereas RMSEV = 4.73 kcal/mol using Îµ<sub>0</sub> = 1). Second, we want to use a distance-dependent dielectric when the model is used prospectively to generate ESP surfaces for small molecules and proteins (to account for the shielding of charge in an aqueous environment), and the electrostatic function we use to derive the model should be consistent with that.</div><div class="NLM_p">Two distinct types of DFT-fp surfaces were used in this work. In the unconstrained point charges approach, the partial charge on each feature was fitted independently as a free parameter during the least-squares fitting. For the constrained point charges approach, Î<i>q</i> values for each feature group in a molecule were fitted and partial charges on the features were constrained to those of other features in a feature group (see above). The unconstrained point charges approach was used for the model compounds and for deriving reference charges from the training set. The constrained point charges approach was used everywhere else.</div><div class="NLM_p">The DNN-fp ESP surfaces use the constrained point charges model (see above), and Î<i>q</i> values are predicted by our graph convolutional DNN-fp model (see below). Since the DNN-fp model was trained on neutral molecules, we adopted the following scheme for deriving Î<i>q</i> values for a charged molecule. We first neutralize all formally charged atoms in the molecule. Next, we use the DNN-fp model to generate Î<i>q</i> values for all atoms in this neutralized molecule; we will refer to these as Î<i>q</i><sub><i>n</i></sub> values. We then obtain the final Î<i>q</i> values for all atoms in the original charged molecule, as follows:<span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_m004.gif" alt="" id="_i21" /></img></span>where <i>Q</i> is the formal charge on the atom. A protonated primary amine (ââNH<sub>3</sub>â), for example, will have <i>Q</i> = <i>+</i>1, but for groups where the formal charge is delocalized, the partial formal charge will be used. For example, an oxygen in a negatively charged carboxylate group will have <i>Q</i> = â0.5. <i>A</i> is an attenuation factor, mimicking the shielding of formally charged functional groups in an aqueous environment, which was empirically set to 0.4; this highlights charged parts of the ESP surface, without dominating the regions around them.</div><div class="NLM_p">DFT-derived ESP surfaces for proteins are calculated in a completely analogous way to ligand DNN-fp ESP surfaces. The only difference is that for protein atoms, the Î<i>q</i> values are taken directly from a precalculated list that was derived from DFT calculations for protein functional groups (see below).</div><div class="NLM_p last">Finally, AM1-BCC ESP surfaces are calculated from AM1-BCC point charges. AM1-BCC is a charge model wherein charges are calculated using AM1 population analysis, to which the bond charge corrections are added, which were parametrized for reproducing the HF/6-31G* ESPs of a set of molecules.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> For the test set, atomic AM1-BCC partial charges were calculated using the Antechamber<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> module of the AMBER<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> software. ESP values on the NGL surface points were then calculated using the method described above, with Îµ<sub>0</sub> = 1.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Ligand Point Charges from DNN-fp Model</h3><div class="NLM_p">A molecular graph convolutional DNN as shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> was built using the Keras<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> library (version 2.2.4) for Python; TensorFlow<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (version 1.7.0) was used as the back end. The model receives 64 simple atomic properties (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Table S6 in the Supporting Information</a>), and atom connectivities were calculated using RDKit.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Both inputs were arranged as 3D tensors. Given <i>m</i> molecules, the largest of which has <i>n</i> atoms, the property tensor has dimension <i>m</i> Ã <i>n</i> Ã 64. The connectivity is represented by a Boolean tensor of dimension <i>m</i> Ã <i>n</i> Ã <i>n</i>. Both tensors were padded with zeros for molecules with fewer than <i>n</i> atoms. Similarly a 3D tensor of fitted Î<i>q</i> values of dimension <i>m</i> Ã <i>n</i> Ã 1 was used as ground truth, padded with a special ânot a numberâ (NaN) value for molecules with fewer than <i>n</i> atoms; the NaN values were ignored in the loss calculation. A new custom convolutional layer was created, which performs a dot product of the input and a kernel (input width Ã output width) as the first step. For each atom, output from its neighboring atoms (connected by covalent bonds) is added to it: a graph convolution. Finally an activation function is applied before the values are fed into the next layer. The final model consists of two fully connected layers sequentially connected to input property layer, followed by six custom GraphConv layers, each of which gets the connectivity matrix as an input in addition to the preceding layer. The six GraphConv layers are followed by two fully connected layers. All these layers have a width of 160 and use rectified linear unit<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> as an activation function. The last fully connected layer feeds into the fully connected output layer of width = 1 (1 Î<i>q</i> value for each atom) with a linear activation function. The hyperparameters (width of the hidden layers, type of the activation function, and the number of GraphConv layers) described above were identified using grid search algorithm. For hyperparameter optimization, a random subset of the training set (20%) was set aside for validation. The identified best set of hyperparameters were then used for the final model generation, wherein the whole training set was used for learning and the external test set was used for evaluation of the model. For training, a custom loss function was used, which calculates mean absolute error between true and predicted Î<i>q</i> values ignoring pairs of values with NaNs as the true values. The model was trained using the Adam optimizer<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> with learning rate of 0.0025, decay of 0.0005, and minibatches of 128 molecules for 1000 steps.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. DNN model architecture.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Protein Point Charges</h3><div class="NLM_p">Using the constrained point charges model (see above), we precalculated Î<i>q</i> values for feature groups occurring in protein amino acids. For this purpose we used a set of specific model compounds representing backbone and side chain groups in proteins. For simplicity, we have assumed that there is no charge transfer between side chain and main chain or between adjacent residues. We established empirically that this approximation has very little effect on the ESP surface of the protein.</div><div class="NLM_p last">The model compounds used to generate charges are listed in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">Supporting Information</a>. For backbone atoms, we used capped tripeptides, where we constrained the Î<i>q</i> values of the atoms in the central amino acid to add up to zero. Specific tripeptides were used to model glycine and proline. All other amino acid backbone atoms were modeled using an alanine as the central residue and constraining the side-chain methyl to have Î<i>q</i> = 0. Side chains were modeled by capping them with a methyl group representing the CÎ´ atom of the protein backbone and constraining this methyl to have Î<i>q</i> = 0. For example, the serine side chain was modeled as an ethanol molecule, and the Î<i>q</i> value of the methyl group was constrained to be zero.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Calculating Feature ESP Values</h3><div class="NLM_p last">For each feature in a molecule, we measure the corresponding feature ESP value on the molecular surface. For a non-parent-atom feature (hydrogen, lone pair, Ï-hole, and p-orbitals), the feature ESP value was measured as the average ESP value over all surfaces points that are within angular distance of 10Â° from the feature vector (a vector between the parent atom and the feature). For an atomic feature, the ESP value on its nearest of all surface points excluding the surface points that are within 40Â° from any of its nonatomic children features was considered the feature ESP value.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01129" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21071" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21071" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01129" class="ext-link">10.1021/acs.jmedchem.9b01129</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tables providing data on performance on the training set, feature and feature group type definitions, and atomic properties used for the DNN-fp model; figures showing the effect of similarity to training set compounds on the quality of the ESP surfaces produced by the DNN-fp model; plots showing the correlation between predicted ESP values and DFT ESP values for all feature types; plots showing the correlation between predicted and experimental values for four properties, comparing DFT, DNN-fp, and AM1-BCC (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf">jm9b01129_si_001.pdf (1.64 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01129" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70603" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70603" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcel L. Verdonk</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6484-3328" title="Orcid link">http://orcid.org/0000-0002-6484-3328</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#640905160701084a120116000b0a0f240517101c4a070b09"><span class="__cf_email__" data-cfemail="1a777b68797f76346c7f687e7574715a7b696e6234797577">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Prakash Chandra Rathi</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">R. Frederick Ludlow</span> - <span class="hlFld-Affiliation affiliation">Astex Pharmaceuticals, 436 Cambridge Science Park, Milton Road, Cambridge CB4 0QA, United Kingdom</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote"><b>Data and Models:</b> The training and test data (SMILES, atomic properties and connectivity matrices, and fitted Î<i>q</i> values) used for model building are available at <a href="https://doi.org/10.6084/m9.figshare.9768071.v1" class="extLink">https://doi.org/10.6084/m9.figshare.9768071.v1</a>. The trained model and associated code can be found at <a href="https://github.com/AstexUK/ESP_DNN/tree/master/esp_dnn" class="extLink">https://github.com/AstexUK/ESP_DNN/tree/master/esp_dnn</a>.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded by a postdoctoral fellowship awarded to P.C.R. under the âSustaining Innovation Postdoctoral Training Programâ at Astex Pharmaceuticals. We thank Andreas Bender and Richard Lewis of the Chemistry Department at Cambridge University for getting us started with model training and helpful discussions. We thank the members of the Computational Chemistry and Informatics Department for helpful discussions. In particular we thank Gianni Chessari for discussions around XIAP SAR, Paul Mortenson for assisting us by setting up Q-chem calculations, and Chris Murray and Richard Hall for useful feedback and for reading the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ESP</td><td class="NLM_def"><p class="first last">electrostatic potential</p></td></tr><tr><td class="NLM_term">QM</td><td class="NLM_def"><p class="first last">quantum mechanics</p></td></tr><tr><td class="NLM_term">DFT</td><td class="NLM_def"><p class="first last">density functional theory</p></td></tr><tr><td class="NLM_term">MM</td><td class="NLM_def"><p class="first last">molecular mechanics</p></td></tr><tr><td class="NLM_term">DNN</td><td class="NLM_def"><p class="first last">deep neural network</p></td></tr><tr><td class="NLM_term">RMSE</td><td class="NLM_def"><p class="first last">root-mean-square error</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">fXa</td><td class="NLM_def"><p class="first last">factor Xa</p></td></tr><tr><td class="NLM_term">XIAP</td><td class="NLM_def"><p class="first last">X-linked inhibitor of apoptosis protein</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22324" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22324" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scrocco, E.</span>; <span class="NLM_string-name">Tomasi, J.</span></span> <span> </span><span class="NLM_article-title">The Electrostatic Molecular Potential as a Tool for the Interpretation of Molecular Properties</span>. In  <i>New Concepts II</i>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">1973</span>; Vol.  <span class="NLM_volume">42</span>, pp  <span class="NLM_fpage">95</span>â <span class="NLM_lpage">170</span>m <span class="refDoi">Â DOI: 10.1007/3-540-06399-4_6</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1007%2F3-540-06399-4_6" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1973&pages=95-170&author=E.+Scrocco&author=J.+Tomasi&title=New+Concepts+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2F3-540-06399-4_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F3-540-06399-4_6%26sid%3Dliteratum%253Aachs%26aulast%3DScrocco%26aufirst%3DE.%26atitle%3DThe%2520Electrostatic%2520Molecular%2520Potential%2520as%2520a%2520Tool%2520for%2520the%2520Interpretation%2520of%2520Molecular%2520Properties%26btitle%3DNew%2520Concepts%2520II%26pub%3DSpringer%26date%3D1973%26volume%3D42%26spage%3D95%26epage%3D170%26doi%3D10.1007%2F3-540-06399-4_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrolongo, C.</span></span> <span> </span><span class="NLM_article-title">Quantum Chemical Study of Isolated and Interacting Molecules with Biological Activity</span>. <i>Gazz. Chim. Ital.</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">445</span>â <span class="NLM_lpage">478</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=1978&pages=445-478&author=C.+Petrolongo&title=Quantum+Chemical+Study+of+Isolated+and+Interacting+Molecules+with+Biological+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPetrolongo%26aufirst%3DC.%26atitle%3DQuantum%2520Chemical%2520Study%2520of%2520Isolated%2520and%2520Interacting%2520Molecules%2520with%2520Biological%2520Activity%26jtitle%3DGazz.%2520Chim.%2520Ital.%26date%3D1978%26volume%3D108%26spage%3D445%26epage%3D478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonaccorsi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrocco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasi, J.</span></span> <span> </span><span class="NLM_article-title">Molecular SCF Calculations for the Ground State of Some Three Membered Ring Molecules: (CH <sub>2</sub>) <sub>3</sub>, (CH <sub>2</sub>) <sub>2</sub> NH, (CH <sub>2</sub>) <sub>2</sub> NH <sub>2</sub> <sup>+</sup>, (CH <sub>2</sub>) <sub>2</sub> O, (CH <sub>2</sub>) <sub>2</sub> S, (CH) <sub>2</sub> CH <sub>2</sub>, and N <sub>2</sub> CH <sub>2</sub></span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5270</span>â <span class="NLM_lpage">5284</span>, <span class="refDoi">Â DOI: 10.1063/1.1672775</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1063%2F1.1672775" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaE3cXktFeqtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1970&pages=5270-5284&issue=10&author=R.+Bonaccorsiauthor=E.+Scroccoauthor=J.+Tomasi&title=Molecular+SCF+Calculations+for+the+Ground+State+of+Some+Three+Membered+Ring+Molecules%3A+%28CH+2%29+3%2C+%28CH+2%29+2+NH%2C+%28CH+2%29+2+NH+2+%2B%2C+%28CH+2%29+2+O%2C+%28CH+2%29+2+S%2C+%28CH%29+2+CH+2%2C+and+N+2+CH+2&doi=10.1063%2F1.1672775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular SCF calculations for the ground state of some three-membered ring molecules:  (CH2)3, (CH2)2NH, (CH2)2NH+2, (CH2)2O, (CH2)2S, (CH)2CH2, and N2CH2</span></div><div class="casAuthors">Bonaccorsi, Rosanna; Scrocco, Eolo; Tomasi, Jacopo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5270-84</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">LCAO SCF MO calcns. with min. basis sets of Slater-type orbitals are performed for some 3-membered ring compds.  A reinterpretation of the resulting wavefunctions in terms of localized (exclusive) orbitals is presented and discussed.  The bent bonds in the rings are closely evidenced.  The electrostatic potentials produced in the neighboring space by the nuclear and electronic charge distributions are evaluated and employed to show the mol. sites more likely subject to electrophilic attacks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIXfyl5U7vA7Vg90H21EOLACvtfcHk0liYgAKS2qQV-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXktFeqtbs%253D&md5=ed13295f8bcc60f091d78782cfddb415</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1063%2F1.1672775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1672775%26sid%3Dliteratum%253Aachs%26aulast%3DBonaccorsi%26aufirst%3DR.%26aulast%3DScrocco%26aufirst%3DE.%26aulast%3DTomasi%26aufirst%3DJ.%26atitle%3DMolecular%2520SCF%2520Calculations%2520for%2520the%2520Ground%2520State%2520of%2520Some%2520Three%2520Membered%2520Ring%2520Molecules%253A%2520%2528CH%25202%2529%25203%252C%2520%2528CH%25202%2529%25202%2520NH%252C%2520%2528CH%25202%2529%25202%2520NH%25202%2520%252B%252C%2520%2528CH%25202%2529%25202%2520O%252C%2520%2528CH%25202%2529%25202%2520S%252C%2520%2528CH%2529%25202%2520CH%25202%252C%2520and%2520N%25202%2520CH%25202%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1970%26volume%3D52%26issue%3D10%26spage%3D5270%26epage%3D5284%26doi%3D10.1063%2F1.1672775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langridge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaney, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">Electrostatic Potential Molecular Surfaces</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>79</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3754</span>â <span class="NLM_lpage">3758</span>, <span class="refDoi">Â DOI: 10.1073/pnas.79.12.3754</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1073%2Fpnas.79.12.3754" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=6285364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaL38XksV2hsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1982&pages=3754-3758&issue=12&author=P.+K.+Weinerauthor=R.+Langridgeauthor=J.+M.+Blaneyauthor=R.+Schaeferauthor=P.+A.+Kollman&title=Electrostatic+Potential+Molecular+Surfaces&doi=10.1073%2Fpnas.79.12.3754"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic potential molecular surfaces</span></div><div class="casAuthors">Weiner, Paul K.; Langridge, Robert; Blaney, Jeffrey M.; Schaefer, Ronald; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3754-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Color-coded computer graphics representations of the electrostatic potentials of trypsin  [9002-07-7], trypsin inhibitor  [9035-81-8], prealbumin and its thyroxine  [51-48-9] complex, fragments of double-helical DNA, and a netropsin  [1438-30-8]-DNA complex illustrate the electrostatic and topog. complementarity in macromol.-ligand interactions.  This approach is powerful in revealing intermol. specificity and shows promise of having predictive value in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWAcsmv8tYL7Vg90H21EOLACvtfcHk0liYgAKS2qQV-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XksV2hsrs%253D&md5=19afda4a3caf14d58344092eb9bbfd2f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.79.12.3754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.79.12.3754%26sid%3Dliteratum%253Aachs%26aulast%3DWeiner%26aufirst%3DP.%2BK.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DSchaefer%26aufirst%3DR.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DElectrostatic%2520Potential%2520Molecular%2520Surfaces%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1982%26volume%3D79%26issue%3D12%26spage%3D3754%26epage%3D3758%26doi%3D10.1073%2Fpnas.79.12.3754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span> <span> </span><span class="NLM_article-title">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">3036</span>â <span class="NLM_lpage">3050</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3036-3050&issue=6&author=M.+R.+Bauerauthor=M.+D.+Mackey&title=Electrostatic+Complementarity+as+a+Fast+and+Effective+Tool+to+Optimize+Binding+and+Selectivity+of+Protein-Ligand+Complexes&doi=10.1021%2Facs.jmedchem.8b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span></div><div class="casAuthors">Bauer, Matthias R.; Mackey, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3036-3050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrostatic interactions between small mols. and their resp. receptors are essential for mol. recognition and are also key contributors to the binding free energy.  Assessing the electrostatic match of protein-ligand complexes therefore provides important insights into why ligands bind and what can be changed to improve binding.  Ideally, the ligand and protein electrostatic potentials at the protein-ligand interaction interface should maximize their complementarity while minimizing desolvation penalties.  In this work, we present a fast and efficient tool to calc. and visualize the electrostatic complementarity (EC) of protein-ligand complexes.  We compiled benchmark sets demonstrating electrostatically driven structure-activity relationships (SAR) from literature data, including kinase, protein-protein interaction, and GPCR targets, and used these to demonstrate that the EC method can visualize, rationalize, and predict electrostatically driven ligand affinity changes and help to predict compd. selectivity.  The methodol. presented here for the anal. of EC is a powerful and versatile tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pdUzJ9T0trVg90H21EOLACvtfcHk0ljXOUD4z5c7yw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D&md5=a1abb5abe7fa8d73bee264bc94fd2da1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01925%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26atitle%3DElectrostatic%2520Complementarity%2520as%2520a%2520Fast%2520and%2520Effective%2520Tool%2520to%2520Optimize%2520Binding%2520and%2520Selectivity%2520of%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D6%26spage%3D3036%26epage%3D3050%26doi%3D10.1021%2Facs.jmedchem.8b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilcken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckler, F. M.</span></span> <span> </span><span class="NLM_article-title">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1363</span>â <span class="NLM_lpage">1388</span>, <span class="refDoi">Â DOI: 10.1021/jm3012068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3012068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1363-1388&issue=4&author=R.+Wilckenauthor=M.+O.+Zimmermannauthor=A.+Langeauthor=A.+C.+Joergerauthor=F.+M.+Boeckler&title=Principles+and+Applications+of+Halogen+Bonding+in+Medicinal+Chemistry+and+Chemical+Biology&doi=10.1021%2Fjm3012068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Wilcken, Rainer; Zimmermann, Markus O.; Lange, Andreas; Joerger, Andreas C.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1363-1388</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Halogen bonding has been known in material science for decades, but until recently, halogen bonds in protein-ligand interactions were largely the result of serendipitous discovery rather than rational design.  In this Perspective, we provide insights into the phenomenon of halogen bonding, with special focus on its role in drug discovery.  We summarize the theor. background defining its strength and directionality, provide a systematic anal. of its occurrence and interaction geometries in protein-ligand complexes, and give recent examples where halogen bonding has been successfully harnessed for lead identification and optimization.  In light of these data, we discuss the potential and limitations of exploiting halogen bonds for mol. recognition and rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjholHfNZ9kbVg90H21EOLACvtfcHk0ljXOUD4z5c7yw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ&md5=81ba024e4427fac3e7af0a4cd330909c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm3012068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3012068%26sid%3Dliteratum%253Aachs%26aulast%3DWilcken%26aufirst%3DR.%26aulast%3DZimmermann%26aufirst%3DM.%2BO.%26aulast%3DLange%26aufirst%3DA.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPrinciples%2520and%2520Applications%2520of%2520Halogen%2520Bonding%2520in%2520Medicinal%2520Chemistry%2520and%2520Chemical%2520Biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D4%26spage%3D1363%26epage%3D1388%26doi%3D10.1021%2Fjm3012068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, D. A.</span></span> <span> </span><span class="NLM_article-title">The Cation-Ï Interaction</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">885</span>â <span class="NLM_lpage">893</span>, <span class="refDoi">Â DOI: 10.1021/ar300265y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar300265y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSqtrjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2013&pages=885-893&issue=4&author=D.+A.+Dougherty&title=The+Cation-%CF%80+Interaction&doi=10.1021%2Far300265y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Cation-Ï Interaction</span></div><div class="casAuthors">Dougherty, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">885-893</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The chem. community now recognizes the cation-Ï interaction as a major force for mol. recognition, joining the hydrophobic effect, the hydrogen bond, and the ion pair in detg. macromol. structure and drug-receptor interactions.  This Account provides the author's perspective on the intellectual origins and fundamental nature of the cation-Ï interaction.  Early studies on cyclophanes established that water-sol., cationic mols. would forego aq. solvation to enter a hydrophobic cavity if that cavity was lined with Ï systems.  Important gas phase studies established the fundamental nature of the cation-Ï interaction.  The strength of the cation-Ï interaction (Li+ binds to benzene with 38 kcal/mol of binding energy; NH4+ with 19 kcal/mol) distinguishes it from the weaker polar-Ï interactions obsd. in the benzene dimer or water-benzene complexes.  In addn. to the substantial intrinsic strength of the cation-Ï interaction in gas phase studies, the cation-Ï interaction remains energetically significant in aq. media and under biol. conditions.  Many studies have shown that cation-Ï interactions can enhance binding energies by 2-5 kcal/mol, making them competitive with hydrogen bonds and ion pairs in drug-receptor and protein-protein interactions.  As with other noncovalent interactions involving arom. systems, the cation-Ï interaction includes a substantial electrostatic component.  The six (four) CÎ´--HÎ´+ bond dipoles of a mol. like benzene (ethylene) combine to produce a region of neg. electrostatic potential on the face of the Ï system.  Simple electrostatics facilitate a natural attraction of cations to the surface.  The trend for (gas phase) binding energies is Li+ > Na+ > K+ > Rb+: as the ion gets larger the charge is dispersed over a larger sphere and binding interactions weaken, a classical electrostatic effect.  On other hand, polarizability does not define these interactions.  Cyclohexane is more polarizable than benzene but a decidedly poorer cation binder.  Many studies have documented cation-Ï interactions in protein structures, where lysine or arginine side chains interact with phenylalanine, tyrosine, or tryptophan.  In addn., countless studies have established the importance of the cation-Ï interaction in a range of biol. processes.  Our work has focused on mol. neurobiol., and we have shown that neurotransmitters generally use a cation-Ï interaction to bind to their receptors.  We have also shown that many drug-receptor interactions involve cation-Ï interactions.  A cation-Ï interaction plays a crit. role in the binding of nicotine to ACh receptors in the brain, an esp. significant case.  Other researchers have established important cation-Ï interactions in the recognition of the "histone code," in terpene biosynthesis, in chem. catalysis, and in many other systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgQQZnCXWWcrVg90H21EOLACvtfcHk0ljXOUD4z5c7yw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSqtrjL&md5=a4f35bd254c0ecbe695e91f40518e833</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Far300265y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far300265y%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DD.%2BA.%26atitle%3DThe%2520Cation-%25CF%2580%2520Interaction%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2013%26volume%3D46%26issue%3D4%26spage%3D885%26epage%3D893%26doi%3D10.1021%2Far300265y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, F.</span></span> <span> </span><span class="NLM_article-title">Interactions with Aromatic Rings in Chemical and Biological Recognition</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1210</span>â <span class="NLM_lpage">1250</span>, <span class="refDoi">Â DOI: 10.1002/anie.200390319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1002%2Fanie.200390319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=1210-1250&issue=11&author=E.+A.+Meyerauthor=R.+K.+Castellanoauthor=F.+Diederich&title=Interactions+with+Aromatic+Rings+in+Chemical+and+Biological+Recognition&doi=10.1002%2Fanie.200390319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions with aromatic rings in chemical and biological recognition</span></div><div class="casAuthors">Meyer, Emmanuel A.; Castellano, Ronald K.; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1210-1250</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Intermol. interactions involving arom. rings are key processes in both chem. and biol. recognition.  Their understanding is essential for rational drug design and lead optimization in medicinal chem.  Different approaches-biol. studies, mol. recognition studies with artificial receptors, crystallog. database mining, gas-phase studies, and theor. calcns. - are pursued to generate a profound understanding of the structural and energetic parameters of individual recognition modes involving arom. rings.  This review attempts to combine and summarize the knowledge gained from these investigations.  The review focuses mainly on examples with biol. relevance since one of its aims it to enhance the knowledge of mol. recognition forces that is essential for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHUbAhGHcGtLVg90H21EOLACvtfcHk0ljXOUD4z5c7yw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D&md5=e91e6a8d115d56229e1660697a286b04</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.200390319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200390319%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DE.%2BA.%26aulast%3DCastellano%26aufirst%3DR.%2BK.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DInteractions%2520with%2520Aromatic%2520Rings%2520in%2520Chemical%2520and%2520Biological%2520Recognition%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26issue%3D11%26spage%3D1210%26epage%3D1250%26doi%3D10.1002%2Fanie.200390319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bleiziffer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Machine Learning of Partial Charges Derived from High-Quality Quantum-Mechanical Calculations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">579</span>â <span class="NLM_lpage">590</span>, <span class="refDoi">Â DOI: 10.1021/acs.jcim.7b00663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtVemtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=579-590&issue=3&author=P.+Bleizifferauthor=K.+Schallerauthor=S.+Riniker&title=Machine+Learning+of+Partial+Charges+Derived+from+High-Quality+Quantum-Mechanical+Calculations&doi=10.1021%2Facs.jcim.7b00663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Machine Learning of Partial Charges Derived from High-Quality Quantum-Mechanical Calculations</span></div><div class="casAuthors">Bleiziffer, Patrick; Schaller, Kay; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">579-590</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Parametrization of small org. mols. for classical mol. dynamics simulations is not trivial.  The vastness of the chem. space makes approaches using building blocks challenging.  The most common approach is therefore an individual parametrization of each compd. by deriving partial charges from semiempirical or ab initio calcns. and inheriting the bonded and van der Waals (Lennard-Jones) parameters from a (bio)mol. force field.  The quality of the partial charges generated in this fashion depends on the level of the quantum-chem. calcn. as well as on the extn. procedure used.  Here, we present a machine learning (ML) based approach for predicting partial charges extd. from d. functional theory (DFT) electron densities.  The training set was chosen with the goal to provide a broad coverage of the known chem. space of druglike mols.  In addn. to the speed of the approach, the partial charges predicted by ML are not dependent on the three-dimensional conformation in contrast to the ones obtained by fitting to the electrostatic potential (ESP).  To assess the quality and compatibility with std. force fields, we performed benchmark calcns. for the free energy of hydration and liq. properties such as d. and heat of vaporization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmkC7Cjqxr9LVg90H21EOLACvtfcHk0li5IO3ccqytZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtVemtLo%253D&md5=6b70f157ee8f3f9735d2f624de0c3155</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00663%26sid%3Dliteratum%253Aachs%26aulast%3DBleiziffer%26aufirst%3DP.%26aulast%3DSchaller%26aufirst%3DK.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DMachine%2520Learning%2520of%2520Partial%2520Charges%2520Derived%2520from%2520High-Quality%2520Quantum-Mechanical%2520Calculations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26issue%3D3%26spage%3D579%26epage%3D590%26doi%3D10.1021%2Facs.jcim.7b00663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heidrich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Exner, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckler, F. M.</span></span> <span> </span><span class="NLM_article-title">Predicting the Magnitude of Ï-Holes Using VmaxPred, a Fast and Efficient Tool Supporting the Application of Halogen Bonds in Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">636</span>â <span class="NLM_lpage">643</span>, <span class="refDoi">Â DOI: 10.1021/acs.jcim.8b00622</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00622" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFylurnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=636-643&issue=2&author=J.+Heidrichauthor=T.+E.+Exnerauthor=F.+M.+Boeckler&title=Predicting+the+Magnitude+of+%CF%83-Holes+Using+VmaxPred%2C+a+Fast+and+Efficient+Tool+Supporting+the+Application+of+Halogen+Bonds+in+Drug+Discovery&doi=10.1021%2Facs.jcim.8b00622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting the Magnitude of Ï-Holes Using VmaxPred, a Fast and Efficient Tool Supporting the Application of Halogen Bonds in Drug Discovery</span></div><div class="casAuthors">Heidrich, Johannes; Exner, Thomas E.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">636-643</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Halogen bonding (XB) as a modern mol. interaction has received increasing attention, not only in material sciences, but also in biol. systems and drug discovery.  Thus, there is a growing demand for fast, efficient, and easily applicable tailor-made tools supporting the use of halogen bonds in mol. design and medicinal chem.  The potential strength of a halogen bond is dependent on several properties of the Ï-hole donor, e.g. a (hetero)aryl halide, and the Ï-hole acceptor, a nucleophile with n- or Ï-electron d.  Besides the influence of the interaction geometry and the type of acceptor, significant tuning effects on the magnitude of the Ï-hole can be obsd., caused by different (hetero)arom. scaffolds and their substitution patterns.  The most pos. electrostatic potential on the isodensity surface (Vmax), representing the Ï-hole, has been widely used as the std. descriptor for the magnitude of the Ï-hole and the strength of the halogen bond.  Calcn. of Vmax using quantum mech. methods at a reasonable level of theory is time-consuming and, thus, not applicable for larger nos. of compds. in drug discovery projects.  Herein, the authors present a tool for the prediction of this descriptor based on a machine-learned model with a speed up of five to six orders of magnitude compared to MP2 quantum mech. calcns.  According to the test set the squared correlation coeff. is greater than 0.94.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGionFTI8jBbVg90H21EOLACvtfcHk0li5IO3ccqytZA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFylurnE&md5=a1c93e84070b8c6868e9971f6ec39c4a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00622%26sid%3Dliteratum%253Aachs%26aulast%3DHeidrich%26aufirst%3DJ.%26aulast%3DExner%26aufirst%3DT.%2BE.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPredicting%2520the%2520Magnitude%2520of%2520%25CF%2583-Holes%2520Using%2520VmaxPred%252C%2520a%2520Fast%2520and%2520Efficient%2520Tool%2520Supporting%2520the%2520Application%2520of%2520Halogen%2520Bonds%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D2%26spage%3D636%26epage%3D643%26doi%3D10.1021%2Facs.jcim.8b00622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, Efficient Generation of High-Quality Atomic Charges. AM1-BCC Model: II. Parameterization and Validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1623</span>â <span class="NLM_lpage">1641</span>, <span class="refDoi">Â DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&issue=16&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+Efficient+Generation+of+High-Quality+Atomic+Charges.+AM1-BCC+Model%3A+II.+Parameterization+and+Validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lh9SA_-cWmY-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520Efficient%2520Generation%2520of%2520High-Quality%2520Atomic%2520Charges.%2520AM1-BCC%2520Model%253A%2520II.%2520Parameterization%2520and%2520Validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26issue%3D16%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mardirossian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head-Gordon, M.</span></span> <span> </span><span class="NLM_article-title">Thirty Years of Density Functional Theory in Computational Chemistry: An Overview and Extensive Assessment of 200 Density Functionals</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>115</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">2315</span>â <span class="NLM_lpage">2372</span>, <span class="refDoi">Â DOI: 10.1080/00268976.2017.1333644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1080%2F00268976.2017.1333644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCltb3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2017&pages=2315-2372&issue=19&author=N.+Mardirossianauthor=M.+Head-Gordon&title=Thirty+Years+of+Density+Functional+Theory+in+Computational+Chemistry%3A+An+Overview+and+Extensive+Assessment+of+200+Density+Functionals&doi=10.1080%2F00268976.2017.1333644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Thirty years of density functional theory in computational chemistry: an overview and extensive assessment of 200 density functionals</span></div><div class="casAuthors">Mardirossian, Narbe; Head-Gordon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2315-2372</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">In the past 30 years, Kohn-Sham d. functional theory has emerged as the most popular electronic structure method in computational chem.  To assess the ever-increasing no. of approx. exchange-correlation functionals, this review benchmarks a total of 200 d. functionals on a mol. database (MGCDB84) of nearly 5000 data points.  The database employed, provided as Supplemental Data, is comprised of 84 data-sets and contains non-covalent interactions, isomerisation energies, thermochem., and barrier heights.  In addn., the evolution of non-empirical and semi-empirical d. functional design is reviewed, and guidelines are provided for the proper and effective use of d. functionals.  The most promising functional considered is ÏB97M-V, a range-sepd. hybrid meta-GGA with VV10 nonlocal correlation, designed using a combinatorial approach.  From the local GGAs, B97-D3, revPBE-D3, and BLYP-D3 are recommended, while from the local meta-GGAs, B97M-rV is the leading choice, followed by MS1-D3 and M06-L-D3.  The best hybrid GGAs are ÏB97X-V, ÏB97X-D3, and ÏB97X-D, while useful hybrid meta-GGAs (besides ÏB97M-V) include ÏM05-D, M06-2X-D3, and MN15.  Ultimately, today's state-of-the-art functionals are close to achieving the level of accuracy desired for a broad range of chem. applications, and the principal remaining limitations are assocd. with systems that exhibit significant self-interaction/delocalisation errors and/or strong correlation effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNNQ1y9c79UrVg90H21EOLACvtfcHk0lh9SA_-cWmY-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCltb3O&md5=ba27d707ee3f5fcdd949644d3d2cbd5e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1080%2F00268976.2017.1333644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268976.2017.1333644%26sid%3Dliteratum%253Aachs%26aulast%3DMardirossian%26aufirst%3DN.%26aulast%3DHead-Gordon%26aufirst%3DM.%26atitle%3DThirty%2520Years%2520of%2520Density%2520Functional%2520Theory%2520in%2520Computational%2520Chemistry%253A%2520An%2520Overview%2520and%2520Extensive%2520Assessment%2520of%2520200%2520Density%2520Functionals%26jtitle%3DMol.%2520Phys.%26date%3D2017%26volume%3D115%26issue%3D19%26spage%3D2315%26epage%3D2372%26doi%3D10.1080%2F00268976.2017.1333644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dewar, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoebisch, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Healy, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. J. P.</span></span> <span> </span><span class="NLM_article-title">Development and Use of Quantum Mechanical Molecular Models. 76. AM1: A New General Purpose Quantum Mechanical Molecular Model</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>107</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3902</span>â <span class="NLM_lpage">3909</span>, <span class="refDoi">Â DOI: 10.1021/ja00299a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00299a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaL2MXktFWlsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=1985&pages=3902-3909&issue=13&author=M.+J.+S.+Dewarauthor=E.+G.+Zoebischauthor=E.+F.+Healyauthor=J.+J.+P.+Stewart&title=Development+and+Use+of+Quantum+Mechanical+Molecular+Models.+76.+AM1%3A+A+New+General+Purpose+Quantum+Mechanical+Molecular+Model&doi=10.1021%2Fja00299a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Development and use of quantum mechanical molecular models.  76.  AM1:  a new general purpose quantum mechanical molecular model</span></div><div class="casAuthors">Dewar, Michael J. S.; Zoebisch, Eve G.; Healy, Eamonn F.; Stewart, James J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3902-9</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A new parametric quantum mech. mol. model, AM1 (Austin Model 1), based on the NDDO approxn., is described.  In it the major weaknesses of MNDO, in particular failure to reproduce H bonds, are overcome without any increase in computing time.  Results for 167 mols. are reported.  Parameters are currently available for C, H, O, and N.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB-oi6B_HZY7Vg90H21EOLACvtfcHk0lh9SA_-cWmY-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktFWlsLk%253D&md5=5733ca359609184eb3d58fc52c73d2de</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja00299a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00299a024%26sid%3Dliteratum%253Aachs%26aulast%3DDewar%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DZoebisch%26aufirst%3DE.%2BG.%26aulast%3DHealy%26aufirst%3DE.%2BF.%26aulast%3DStewart%26aufirst%3DJ.%2BJ.%2BP.%26atitle%3DDevelopment%2520and%2520Use%2520of%2520Quantum%2520Mechanical%2520Molecular%2520Models.%252076.%2520AM1%253A%2520A%2520New%2520General%2520Purpose%2520Quantum%2520Mechanical%2520Molecular%2520Model%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1985%26volume%3D107%26issue%3D13%26spage%3D3902%26epage%3D3909%26doi%3D10.1021%2Fja00299a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, J. D.</span></span> <span> </span><span class="NLM_article-title">Matplotlib: A 2D Graphics Environment</span>. <i>Comput. Sci. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">90</span>â <span class="NLM_lpage">95</span>, <span class="refDoi">Â DOI: 10.1109/MCSE.2007.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1109%2FMCSE.2007.55" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=90-95&issue=3&author=J.+D.+Hunter&title=Matplotlib%3A+A+2D+Graphics+Environment&doi=10.1109%2FMCSE.2007.55"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1109%2FMCSE.2007.55&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FMCSE.2007.55%26sid%3Dliteratum%253Aachs%26aulast%3DHunter%26aufirst%3DJ.%2BD.%26atitle%3DMatplotlib%253A%2520A%25202D%2520Graphics%2520Environment%26jtitle%3DComput.%2520Sci.%2520Eng.%26date%3D2007%26volume%3D9%26issue%3D3%26spage%3D90%26epage%3D95%26doi%3D10.1109%2FMCSE.2007.55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">247</span>â <span class="NLM_lpage">260</span>, <span class="refDoi">Â DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lhVDp3-o9J1dQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Questel, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renault, E.</span></span> <span> </span><span class="NLM_article-title">The PK(BHX) Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">4073</span>â <span class="NLM_lpage">4086</span>, <span class="refDoi">Â DOI: 10.1021/jm801331y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801331y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4073-4086&issue=14&author=C.+Laurenceauthor=K.+A.+Brameldauthor=J.+Gratonauthor=J.-Y.+Le+Questelauthor=E.+Renault&title=The+PK%28BHX%29+Database%3A+Toward+a+Better+Understanding+of+Hydrogen-Bond+Basicity+for+Medicinal+Chemists&doi=10.1021%2Fjm801331y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The pKBHX Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span></div><div class="casAuthors">Laurence, Christian; Brameld, Ken A.; Graton, Jerome; Le Questel, Jean-Yves; Renault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4073-4086</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The pKBHX database is presented, which corresponds to three main fields of data: hydrogen bond donor identification, thermodn. data and spectroscopic data.  The pKBHX scale of hydrogen donor basicity differs considerable from the pKBH+ scale of proton transfer basicity.  The hydrogen bond basicities of functional groups relevant to medicinal chem. and drug design is reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql9OhmMu2VILVg90H21EOLACvtfcHk0lhVDp3-o9J1dQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D&md5=9ce4bea3e810752baba6ea5784b55ea0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm801331y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801331y%26sid%3Dliteratum%253Aachs%26aulast%3DLaurence%26aufirst%3DC.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DGraton%26aufirst%3DJ.%26aulast%3DLe%2BQuestel%26aufirst%3DJ.-Y.%26aulast%3DRenault%26aufirst%3DE.%26atitle%3DThe%2520PK%2528BHX%2529%2520Database%253A%2520Toward%2520a%2520Better%2520Understanding%2520of%2520Hydrogen-Bond%2520Basicity%2520for%2520Medicinal%2520Chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D14%26spage%3D4073%26epage%3D4086%26doi%3D10.1021%2Fjm801331y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Politzer, P.</span></span> <span> </span><span class="NLM_article-title">Correlations between the Solvent Hydrogen-Bond-Donating Parameter. Alpha. and the Calculated Molecular Surface Electrostatic Potential</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6715</span>â <span class="NLM_lpage">6717</span>, <span class="refDoi">Â DOI: 10.1021/jo00023a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00023a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaK3MXmtlWmu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1991&pages=6715-6717&issue=23&author=J.+S.+Murrayauthor=P.+Politzer&title=Correlations+between+the+Solvent+Hydrogen-Bond-Donating+Parameter.+Alpha.+and+the+Calculated+Molecular+Surface+Electrostatic+Potential&doi=10.1021%2Fjo00023a045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Correlations between the solvent hydrogen-bond-donating parameter Î± and the calculated molecular surface electrostatic potential</span></div><div class="casAuthors">Murray, Jane S.; Politzer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6715-17</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">We use ab initio self-consistent-field calcns. to show that good linear correlations exist between the solvatochromic hydrogen-bond-donating parameters Î± and the most pos. surface electrostatic potentials, VS,max, of two groups of hydrogen bond donors, taken sep.; the correlation coeffs. are 0.96 and 0.98, resp.  These relations confirm that the electrostatic potential in the space around a gas phase mol. is a key (but not sole) factor in detg. its ability to donate a proton in a solvent-to-solute hydrogen bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2RQv6dZFXCLVg90H21EOLACvtfcHk0lhVDp3-o9J1dQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmtlWmu7g%253D&md5=2f72f1aff6e3757592d4cc9526f90976</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjo00023a045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00023a045%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BS.%26aulast%3DPolitzer%26aufirst%3DP.%26atitle%3DCorrelations%2520between%2520the%2520Solvent%2520Hydrogen-Bond-Donating%2520Parameter.%2520Alpha.%2520and%2520the%2520Calculated%2520Molecular%2520Surface%2520Electrostatic%2520Potential%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1991%26volume%3D56%26issue%3D23%26spage%3D6715%26epage%3D6717%26doi%3D10.1021%2Fjo00023a045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Hydrogen Bond Basicity from Computed Molecular Electrostatic Properties: Implications for Comparative Molecular Field Analysis</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">199</span>â <span class="NLM_lpage">202</span>, <span class="refDoi">Â DOI: 10.1039/p29940000199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1039%2Fp29940000199" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1994&pages=199-202&issue=2&author=P.+W.+Kenny&title=Prediction+of+Hydrogen+Bond+Basicity+from+Computed+Molecular+Electrostatic+Properties%3A+Implications+for+Comparative+Molecular+Field+Analysis&doi=10.1039%2Fp29940000199"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2Fp29940000199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29940000199%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DPrediction%2520of%2520Hydrogen%2520Bond%2520Basicity%2520from%2520Computed%2520Molecular%2520Electrostatic%2520Properties%253A%2520Implications%2520for%2520Comparative%2520Molecular%2520Field%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1994%26volume%3D2%26issue%3D2%26spage%3D199%26epage%3D202%26doi%3D10.1039%2Fp29940000199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prokopczyk, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, J. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, G. R.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Bond Basicity Prediction for Medicinal Chemistry Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4278</span>â <span class="NLM_lpage">4288</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01946</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01946" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyjsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4278-4288&issue=9&author=P.+W.+Kennyauthor=C.+A.+Montanariauthor=I.+M.+Prokopczykauthor=J.+F.+R.+Ribeiroauthor=G.+R.+Sartori&title=Hydrogen+Bond+Basicity+Prediction+for+Medicinal+Chemistry+Design&doi=10.1021%2Facs.jmedchem.5b01946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Bond Basicity Prediction for Medicinal Chemistry Design</span></div><div class="casAuthors">Kenny, Peter W.; Montanari, Carlos A.; Prokopczyk, Igor M.; Ribeiro, Jean F. R.; Sartori, Geraldo Rodrigues</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4278-4288</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen bonding is discussed in the context of medicinal chem. design.  Minimized mol. electrostatic potential (Vmin) is shown to be an effective predictor of hydrogen bond basicity (pKBHX), and predictive models are presented for a no. of hydrogen bond acceptor types relevant to medicinal chem.  The problems posed by the presence of nonequivalent hydrogen bond acceptor sites in mol. structures are addressed by using nonlinear regression to fit measured pKBHX to calcd. Vmin.  Predictions are made for hydrogen bond basicity of fluorine in situations where relevant exptl. measurements are not available.  It is shown how predicted pKBHX can be used to provide insight into the nature of bioisosterism and to profile heterocycles.  Examples of pKBHX prediction for mol. structures with multiple, nonequivalent hydrogen bond acceptors are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwaulOTEJQ_rVg90H21EOLACvtfcHk0lgxQqc_1U_Xtg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyjsLo%253D&md5=733e16424c7bb1c254123029ab9fd6f8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01946%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26aulast%3DProkopczyk%26aufirst%3DI.%2BM.%26aulast%3DRibeiro%26aufirst%3DJ.%2BF.%2BR.%26aulast%3DSartori%26aufirst%3DG.%2BR.%26atitle%3DHydrogen%2520Bond%2520Basicity%2520Prediction%2520for%2520Medicinal%2520Chemistry%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4278%26epage%3D4288%26doi%3D10.1021%2Facs.jmedchem.5b01946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollingsworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seybold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span> <span> </span><span class="NLM_article-title">Relationships between Aqueous Acidities and Computed Surface-Electrostatic Potentials and Local Ionization Energies of Substituted Phenols and Benzoic Acids</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">235</span>â <span class="NLM_lpage">239</span>, <span class="refDoi">Â DOI: 10.1007/s00894-004-0185-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1007%2Fs00894-004-0185-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptValtL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=235-239&issue=4&author=Y.+Maauthor=K.+Grossauthor=C.+Hollingsworthauthor=P.+Seyboldauthor=J.+Murray&title=Relationships+between+Aqueous+Acidities+and+Computed+Surface-Electrostatic+Potentials+and+Local+Ionization+Energies+of+Substituted+Phenols+and+Benzoic+Acids&doi=10.1007%2Fs00894-004-0185-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Relationships between aqueous acidities and computed surface-electrostatic potentials and local ionization energies of substituted phenols and benzoic acids</span></div><div class="casAuthors">Ma, Yuguang; Gross, Kevin C.; Hollingsworth, Chad A.; Seybold, Paul G.; Murray, Jane S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-239</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Electrostatic potentials and av. local ionization energies on the mol. surfaces of 19 phenols, 17 benzoic acids and their resp. conjugate bases were computed at the HF/STO-5G(d)//B3LYP/6-311G(d,p) level.  Good correlations were found between pKas and the VS,max values of the neutral acids and the VS,min and â¾IS,min of the conjugate bases for both sets of mols.  VS,max is the most pos. value of the electrostatic potential on the mol. surface and is an indicator of the ease with which the phenols and benzoic acids lose their acidic hydrogens.  VS,min and â¾IS,min are the min. values of the electrostatic potential and the local ionization energy computed on the mol. surface; the VS,min and â¾IS,min of the conjugate bases of the phenoxides and benzoates are indicative, resp., of the tendencies of electrophiles to approach (VS,min) and to react with (â¾IS,min) to reform the original acids.  The correlations obsd. between the computed mol. surface quantities, taken as single parameters, and the exptl. pKa values ranged from R = 0.938 to R = 0.970 for the two classes of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoko4xUSapR6bVg90H21EOLACvtfcHk0lgxQqc_1U_Xtg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptValtL0%253D&md5=df506ded7abd1922d247c0f1d2e50532</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00894-004-0185-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-004-0185-x%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DGross%26aufirst%3DK.%26aulast%3DHollingsworth%26aufirst%3DC.%26aulast%3DSeybold%26aufirst%3DP.%26aulast%3DMurray%26aufirst%3DJ.%26atitle%3DRelationships%2520between%2520Aqueous%2520Acidities%2520and%2520Computed%2520Surface-Electrostatic%2520Potentials%2520and%2520Local%2520Ionization%2520Energies%2520of%2520Substituted%2520Phenols%2520and%2520Benzoic%2520Acids%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2004%26volume%3D10%26issue%3D4%26spage%3D235%26epage%3D239%26doi%3D10.1007%2Fs00894-004-0185-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caballero-GarcÃ­a, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MondragÃ³n-SolÃ³rzano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres-Cadena, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DÃ­az-GarcÃ­a, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval-Lira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barroso-Flores, J.</span></span> <span> </span><span class="NLM_article-title">Calculation of VS, Max and Its Use as a Descriptor for the Theoretical Calculation of PKa Values for Carboxylic Acids</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">79</span>, <span class="refDoi">Â DOI: 10.3390/molecules24010079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.3390%2Fmolecules24010079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlGjtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=79&issue=1&author=G.+Caballero-Garc%C3%ADaauthor=G.+Mondrag%C3%B3n-Sol%C3%B3rzanoauthor=R.+Torres-Cadenaauthor=M.+D%C3%ADaz-Garc%C3%ADaauthor=J.+Sandoval-Liraauthor=J.+Barroso-Flores&title=Calculation+of+VS%2C+Max+and+Its+Use+as+a+Descriptor+for+the+Theoretical+Calculation+of+PKa+Values+for+Carboxylic+Acids&doi=10.3390%2Fmolecules24010079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of VS,max and Its use as a descriptor for the theoretical calculation of pKa values for carboxylic acids</span></div><div class="casAuthors">Caballero-Garcia, Guillermo; Mondragon-Solorzano, Gustavo; Torres-Cadena, Raul; Diaz-Garcia, Marco; Sandoval-Lira, Jacinto; Barroso-Flores, Joaquin</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79/1-79/15</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The theor. calcn. of pKa values for Bronsted acids is a challenging task that involves sophisticated and time-consuming methods.  Therefore, heuristic approaches are efficient and appealing methodologies to approx. these values.  Herein, we used the max. surface electrostatic potential (VS,max) on the acidic hydrogen atoms of carboxylic acids to describe the H-bond interaction with water (the same descriptor that is used to characterize Ï-bonded complexes) and correlate the results with exptl. pKa values to obtain a predictive model for other carboxylic acids.  We benchmarked six different methods, all including an implicit solvation model (water): Five d. functionals and the Moller-Plesset second order perturbation theory in combination with six different basis sets for a total of thirty-six levels of theory.  The ÏB97X-D/cc-pVDZ level of theory stood out as the best one for consistently reproducing the reported pKa values, with a predictive power of 98% correlation in a test set of ten other carboxylic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx0AOGRu5corVg90H21EOLACvtfcHk0lgxQqc_1U_Xtg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlGjtLc%253D&md5=28b6ffb3baefa304cfc062bb61cfb6e9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3390%2Fmolecules24010079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules24010079%26sid%3Dliteratum%253Aachs%26aulast%3DCaballero-Garc%25C3%25ADa%26aufirst%3DG.%26aulast%3DMondrag%25C3%25B3n-Sol%25C3%25B3rzano%26aufirst%3DG.%26aulast%3DTorres-Cadena%26aufirst%3DR.%26aulast%3DD%25C3%25ADaz-Garc%25C3%25ADa%26aufirst%3DM.%26aulast%3DSandoval-Lira%26aufirst%3DJ.%26aulast%3DBarroso-Flores%26aufirst%3DJ.%26atitle%3DCalculation%2520of%2520VS%252C%2520Max%2520and%2520Its%2520Use%2520as%2520a%2520Descriptor%2520for%2520the%2520Theoretical%2520Calculation%2520of%2520PKa%2520Values%2520for%2520Carboxylic%2520Acids%26jtitle%3DMolecules%26date%3D2019%26volume%3D24%26issue%3D1%26spage%3D79%26doi%3D10.3390%2Fmolecules24010079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Virant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DrvariÄ Talian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podlipnik, Ä.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hribar-Lee, B.</span></span> <span> </span><span class="NLM_article-title">Modelling the Correlation Between Molecular Electrostatic Potential and pKa on Sets of Carboxylic Acids, Phenols and Anilines</span>. <i>Acta Chim. Slov.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">560</span>â <span class="NLM_lpage">563</span>, <span class="refDoi">Â DOI: 10.17344/acsi.2016.2962</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.17344%2Facsi.2016.2962" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=28862310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslaqs7rO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=560-563&author=M.+Virantauthor=S.+Drvari%C4%8D+Talianauthor=%C4%8C.+Podlipnikauthor=B.+Hribar-Lee&title=Modelling+the+Correlation+Between+Molecular+Electrostatic+Potential+and+pKa+on+Sets+of+Carboxylic+Acids%2C+Phenols+and+Anilines&doi=10.17344%2Facsi.2016.2962"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Modelling the correlation between molecular electrostatic potential and pKa on sets of carboxylic acids, phenols and anilines</span></div><div class="casAuthors">Virant, Miha; Talian, Sara Drvaric; Podlipnik, Crtomir; Hribar-Lee, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Acta Chimica Slovenica</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">560-563</span>CODEN:
                <span class="NLM_cas:coden">ACSLE7</span>;
        ISSN:<span class="NLM_cas:issn">1318-0207</span>.
    
            (<span class="NLM_cas:orgname">Slovenian Chemical Society</span>)
        </div><div class="casAbstract">Calcns. of mol. electrostatic potential were correlated with exptl. pKa values for different sets of acidic mols. (carboxylic acids, phenols, and anilines) to obtain linear relationships of variable quality.  A single tri-parameter model function was constructed to describe the pKa dependence on MEP maxima together with two automatically generated mol. descriptors, namely the counts of carboxylic acid and amine functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt86JOKj7P27Vg90H21EOLACvtfcHk0ljjXJ0CU0iV6Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslaqs7rO&md5=ceb2888eb8b3c83546779edcde2ba18b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.17344%2Facsi.2016.2962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.17344%252Facsi.2016.2962%26sid%3Dliteratum%253Aachs%26aulast%3DVirant%26aufirst%3DM.%26aulast%3DDrvari%25C4%258D%2BTalian%26aufirst%3DS.%26aulast%3DPodlipnik%26aufirst%3D%25C4%258C.%26aulast%3DHribar-Lee%26aufirst%3DB.%26atitle%3DModelling%2520the%2520Correlation%2520Between%2520Molecular%2520Electrostatic%2520Potential%2520and%2520pKa%2520on%2520Sets%2520of%2520Carboxylic%2520Acids%252C%2520Phenols%2520and%2520Anilines%26jtitle%3DActa%2520Chim.%2520Slov.%26date%3D2017%26spage%3D560%26epage%3D563%26doi%3D10.17344%2Facsi.2016.2962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slater, A. M.</span></span> <span> </span><span class="NLM_article-title">The IUPAC Aqueous and Non-Aqueous Experimental PKa Data Repositories of Organic Acids and Bases</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1031</span>â <span class="NLM_lpage">1034</span>, <span class="refDoi">Â DOI: 10.1007/s10822-014-9764-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1007%2Fs10822-014-9764-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=24952470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaju7nP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1031-1034&issue=10&author=A.+M.+Slater&title=The+IUPAC+Aqueous+and+Non-Aqueous+Experimental+PKa+Data+Repositories+of+Organic+Acids+and+Bases&doi=10.1007%2Fs10822-014-9764-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The IUPAC aqueous and non-aqueous experimental pKa data repositories of organic acids and bases</span></div><div class="casAuthors">Slater, Anthony Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1031-1034</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Accurate and well-curated exptl. pKa data of org. acids and bases in both aq. and non-aq. media are invaluable in many areas of chem. research, including pharmaceutical, agrochem., specialty chem. and property prediction research.  In pharmaceutical research, pKa data are relevant in ligand design, protein binding, absorption, distribution, metab., elimination as well as soly. and dissoln. rate.  The pKa data compilations of the International Union of Pure and Applied Chem., originally in book form, have been carefully converted into computer-readable form, with value being added in the process, in the form of ionisation assignments and tautomer enumeration.  These compilations offer a broad range of chem. in both aq. and non-aq. media and the exptl. conditions and original ref. for all pKa detns. are supplied.  The statistics for these compilations are presented and the utility of the computer-readable form of these compilations is examd. in comparison to other pKa compilations.  Finally, information is provided about how to access these databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFoevnTg4QIrVg90H21EOLACvtfcHk0ljjXJ0CU0iV6Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaju7nP&md5=0f537621a321288255abbb74f8e8bac3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9764-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9764-9%26sid%3Dliteratum%253Aachs%26aulast%3DSlater%26aufirst%3DA.%2BM.%26atitle%3DThe%2520IUPAC%2520Aqueous%2520and%2520Non-Aqueous%2520Experimental%2520PKa%2520Data%2520Repositories%2520of%2520Organic%2520Acids%2520and%2520Bases%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26issue%3D10%26spage%3D1031%26epage%3D1034%26doi%3D10.1007%2Fs10822-014-9764-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, D. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallheer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knabb, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wexler, R. R.</span></span> <span> </span><span class="NLM_article-title">Factor Xa Inhibitors: Next-Generation Antithrombotic Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">6243</span>â <span class="NLM_lpage">6274</span>, <span class="refDoi">Â DOI: 10.1021/jm100146h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100146h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsFOktrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6243-6274&issue=17&author=D.+J.+P.+Pintoauthor=J.+M.+Smallheerauthor=D.+L.+Cheneyauthor=R.+M.+Knabbauthor=R.+R.+Wexler&title=Factor+Xa+Inhibitors%3A+Next-Generation+Antithrombotic+Agents&doi=10.1021%2Fjm100146h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Factor Xa Inhibitors: Next-Generation Antithrombotic Agents</span></div><div class="casAuthors">Pinto, Donald J. P.; Smallheer, Joanne M.; Cheney, Daniel L.; Knabb, Robert M.; Wexler, Ruth R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6243-6274</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjlojOUGw747Vg90H21EOLACvtfcHk0lha3-EQgT86JQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsFOktrY%253D&md5=241f7c0a1fcc789ceed8e0b4beb03beb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm100146h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100146h%26sid%3Dliteratum%253Aachs%26aulast%3DPinto%26aufirst%3DD.%2BJ.%2BP.%26aulast%3DSmallheer%26aufirst%3DJ.%2BM.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DKnabb%26aufirst%3DR.%2BM.%26aulast%3DWexler%26aufirst%3DR.%2BR.%26atitle%3DFactor%2520Xa%2520Inhibitors%253A%2520Next-Generation%2520Antithrombotic%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D17%26spage%3D6243%26epage%3D6274%26doi%3D10.1021%2Fjm100146h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthwick, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns-Kurtis, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charbaut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Convery, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleanthous, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pateman, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patikis, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toomey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, H. E.</span></span> <span> </span><span class="NLM_article-title">Structure and Property Based Design of Factor Xa Inhibitors: Pyrrolidin-2-ones with Biaryl P4 Motifs</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>â <span class="NLM_lpage">27</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2007.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1016%2Fj.bmcl.2007.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=18054228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD1cXis1Sqtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=23-27&issue=1&author=R.+J.+Youngauthor=A.+D.+Borthwickauthor=D.+Brownauthor=C.+L.+Burns-Kurtisauthor=M.+Campbellauthor=C.+Chanauthor=M.+Charbautauthor=C.+Chungauthor=M.+A.+Converyauthor=H.+A.+Kellyauthor=N.+P.+Kingauthor=S.+Kleanthousauthor=A.+M.+Masonauthor=A.+J.+Patemanauthor=A.+N.+Patikisauthor=I.+L.+Pintoauthor=D.+R.+Pollardauthor=S.+Sengerauthor=G.+P.+Shahauthor=J.+R.+Toomeyauthor=N.+S.+Watsonauthor=H.+E.+Weston&title=Structure+and+Property+Based+Design+of+Factor+Xa+Inhibitors%3A+Pyrrolidin-2-ones+with+Biaryl+P4+Motifs&doi=10.1016%2Fj.bmcl.2007.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and property based design of factor Xa inhibitors: pyrrolidin-2-ones with biaryl P4 motifs</span></div><div class="casAuthors">Young, Robert J.; Borthwick, Alan D.; Brown, David; Burns-Kurtis, Cynthia L.; Campbell, Matthew; Chan, Chuen; Charbaut, Marie; Chung, Chun-wa; Convery, Maire A.; Kelly, Henry A.; King, N. Paul; Kleanthous, Savvas; Mason, Andrew M.; Pateman, Anthony J.; Patikis, Angela N.; Pinto, Ivan L.; Pollard, Derek R.; Senger, Stefan; Shah, Gita P.; Toomey, John R.; Watson, Nigel S.; Weston, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-27</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Structure and property based drug design was exploited in the synthesis of sulfonamidopyrrolidin-2-one-based factor Xa (fXa) inhibitors.  By incorporating biaryl P4 groups, a series of highly potent inhibitors, e.g., I, was obtained with encouraging oral pharmacokinetic profiles and significant but sub-optimal anticoagulant activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodg91ZXDFpYrVg90H21EOLACvtfcHk0lha3-EQgT86JQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXis1Sqtg%253D%253D&md5=a5f6a28d12df20bca8578f42d374fc80</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DBorthwick%26aufirst%3DA.%2BD.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DBurns-Kurtis%26aufirst%3DC.%2BL.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DC.%26aulast%3DCharbaut%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DConvery%26aufirst%3DM.%2BA.%26aulast%3DKelly%26aufirst%3DH.%2BA.%26aulast%3DKing%26aufirst%3DN.%2BP.%26aulast%3DKleanthous%26aufirst%3DS.%26aulast%3DMason%26aufirst%3DA.%2BM.%26aulast%3DPateman%26aufirst%3DA.%2BJ.%26aulast%3DPatikis%26aufirst%3DA.%2BN.%26aulast%3DPinto%26aufirst%3DI.%2BL.%26aulast%3DPollard%26aufirst%3DD.%2BR.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DG.%2BP.%26aulast%3DToomey%26aufirst%3DJ.%2BR.%26aulast%3DWatson%26aufirst%3DN.%2BS.%26aulast%3DWeston%26aufirst%3DH.%2BE.%26atitle%3DStructure%2520and%2520Property%2520Based%2520Design%2520of%2520Factor%2520Xa%2520Inhibitors%253A%2520Pyrrolidin-2-ones%2520with%2520Biaryl%2520P4%2520Motifs%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26issue%3D1%26spage%3D23%26epage%3D27%26doi%3D10.1016%2Fj.bmcl.2007.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, R. W.</span></span> <span> </span><span class="NLM_article-title">A Survey of Hammett Substituent Constants and Resonance and Field Parameters</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>91</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">165</span>â <span class="NLM_lpage">195</span>, <span class="refDoi">Â DOI: 10.1021/cr00002a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00002a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaK3MXhs1ehsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1991&pages=165-195&issue=2&author=C.+Hanschauthor=A.+Leoauthor=R.+W.+Taft&title=A+Survey+of+Hammett+Substituent+Constants+and+Resonance+and+Field+Parameters&doi=10.1021%2Fcr00002a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A survey of Hammett substituent constants and resonance and field parameters</span></div><div class="casAuthors">Hansch, Corwin; Leo, A.; Taft, R. W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-95</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">Included in this review is an anal. of newer methods which can supplant this classic procedure for detn. of the title consts., 283 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKOTHJzZT0lbVg90H21EOLACvtfcHk0lheJvVePwfQ6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhs1ehsLo%253D&md5=9fc814cd57c47680a5213f3438037800</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcr00002a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00002a004%26sid%3Dliteratum%253Aachs%26aulast%3DHansch%26aufirst%3DC.%26aulast%3DLeo%26aufirst%3DA.%26aulast%3DTaft%26aufirst%3DR.%2BW.%26atitle%3DA%2520Survey%2520of%2520Hammett%2520Substituent%2520Constants%2520and%2520Resonance%2520and%2520Field%2520Parameters%26jtitle%3DChem.%2520Rev.%26date%3D1991%26volume%3D91%26issue%3D2%26spage%3D165%26epage%3D195%26doi%3D10.1021%2Fcr00002a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reader, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitorino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilsher, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, J.-A.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery Targeting Inhibitor of Apoptosis Proteins: Discovery of a Non-Alanine Lead Series with Dual Activity Against CIAP1 and XIAP</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">6574</span>â <span class="NLM_lpage">6588</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CitLrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6574-6588&issue=16&author=G.+Chessariauthor=I.+M.+Buckauthor=J.+E.+H.+Dayauthor=P.+J.+Dayauthor=A.+Iqbalauthor=C.+N.+Johnsonauthor=E.+J.+Lewisauthor=V.+Martinsauthor=D.+Millerauthor=M.+Readerauthor=D.+C.+Reesauthor=S.+J.+Richauthor=E.+Tamaniniauthor=M.+Vitorinoauthor=G.+A.+Wardauthor=P.+A.+Williamsauthor=G.+Williamsauthor=N.+E.+Wilsherauthor=J.-A.+Woolford&title=Fragment-Based+Drug+Discovery+Targeting+Inhibitor+of+Apoptosis+Proteins%3A+Discovery+of+a+Non-Alanine+Lead+Series+with+Dual+Activity+Against+CIAP1+and+XIAP&doi=10.1021%2Facs.jmedchem.5b00706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery Targeting Inhibitor of Apoptosis Proteins: Discovery of a Non-Alanine Lead Series with Dual Activity Against cIAP1 and XIAP</span></div><div class="casAuthors">Chessari, Gianni; Buck, Ildiko M.; Day, James E. H.; Day, Philip J.; Iqbal, Aman; Johnson, Christopher N.; Lewis, Edward J.; Martins, Vanessa; Miller, Darcey; Reader, Michael; Rees, David C.; Rich, Sharna J.; Tamanini, Emiliano; Vitorino, Marc; Ward, George A.; Williams, Pamela A.; Williams, Glyn; Wilsher, Nicola E.; Woolford, Alison J.-A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6574-6588</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitor of apoptosis proteins (IAPs) are important regulators of apoptosis and pro-survival signaling pathways whose deregulation is often assocd. with tumor genesis and tumor growth.  IAPs have been proposed as targets for anticancer therapy, and a no. of peptidomimetic IAP antagonists have entered clin. trials.  Using the fragment-based screening approach, the authors identified nonpeptidic fragments binding with millimolar affinities to both cellular inhibitor of apoptosis protein 1 (cIAP1) and X-linked inhibitor of apoptosis protein (XIAP).  Structure-based hit optimization together with an anal. of protein-ligand electrostatic potential complementarity allowed us to significantly increase binding affinity of the starting hits.  Subsequent optimization gave a potent nonalanine IAP antagonist I structurally distinct from all IAP antagonists previously reported.  The lead compd. had activity in cell-based assays and in a mouse xenograft efficacy model and represents a highly promising start point for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRpw0ZQNug7Vg90H21EOLACvtfcHk0lgiP4HOVabBHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CitLrL&md5=70a9c10527791ce2cedd0a74e0bbeebb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00706%26sid%3Dliteratum%253Aachs%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DBuck%26aufirst%3DI.%2BM.%26aulast%3DDay%26aufirst%3DJ.%2BE.%2BH.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DIqbal%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DLewis%26aufirst%3DE.%2BJ.%26aulast%3DMartins%26aufirst%3DV.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DReader%26aufirst%3DM.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DVitorino%26aufirst%3DM.%26aulast%3DWard%26aufirst%3DG.%2BA.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DWilsher%26aufirst%3DN.%2BE.%26aulast%3DWoolford%26aufirst%3DJ.-A.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520Targeting%2520Inhibitor%2520of%2520Apoptosis%2520Proteins%253A%2520Discovery%2520of%2520a%2520Non-Alanine%2520Lead%2520Series%2520with%2520Dual%2520Activity%2520Against%2520CIAP1%2520and%2520XIAP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D16%26spage%3D6574%26epage%3D6588%26doi%3D10.1021%2Facs.jmedchem.5b00706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammett, L. P.</span></span> <span> </span><span class="NLM_article-title">The Effect of Structure upon the Reactions of Organic Compounds. Benzene Derivatives</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1937</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">96</span>â <span class="NLM_lpage">103</span>, <span class="refDoi">Â DOI: 10.1021/ja01280a022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja01280a022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaA2sXjtlSqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1937&pages=96-103&issue=1&author=L.+P.+Hammett&title=The+Effect+of+Structure+upon+the+Reactions+of+Organic+Compounds.+Benzene+Derivatives&doi=10.1021%2Fja01280a022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of structure upon the reactions of organic compounds. Benzene derivatives</span></div><div class="casAuthors">Hammett, Louis P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1937</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-103</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A simple formula is proposed to represent the effect of a substituent in the m- or p- position of the benzene ring upon the rate or equil. of a reaction in which the reacting group is in a side chain attached to the ring.  This formula represents the total effect of the substituent and also the influence of changing length of the side chain, of the dielec. const. of the medium, and of temp. with a satisfactory precision in a wide variety of cases.  The theory that a substituent acts by internal electron displacement is in complete agreement with this and accounts also for the large deviations observed in the case of nonrigid structures.  Tables of substituent and reaction consts. are given from which the effect of many substituents upon a large no. of reaction rates and equil may be obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5TaaR_6a9IbVg90H21EOLACvtfcHk0lgiP4HOVabBHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaA2sXjtlSqsw%253D%253D&md5=ec73b07f4ac4872e26e4487a0240d5c2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fja01280a022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja01280a022%26sid%3Dliteratum%253Aachs%26aulast%3DHammett%26aufirst%3DL.%2BP.%26atitle%3DThe%2520Effect%2520of%2520Structure%2520upon%2520the%2520Reactions%2520of%2520Organic%2520Compounds.%2520Benzene%2520Derivatives%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1937%26volume%3D59%26issue%3D1%26spage%3D96%26epage%3D103%26doi%3D10.1021%2Fja01280a022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span>eMolecules database. <a href="https://reaxys.emolecules.com/index.php" class="extLink">https://reaxys.emolecules.com/index.php</a> (accessed Jan 17, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=eMolecules+database.+https%3A%2F%2Freaxys.emolecules.com%2Findex.php+%28accessed+Jan+17%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span>Sigma-Aldrich. <a href="https://www.sigmaaldrich.com/united-kingdom.html" class="extLink">https://www.sigmaaldrich.com/united-kingdom.html</a> (accessed Jun 3, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Sigma-Aldrich.+https%3A%2F%2Fwww.sigmaaldrich.com%2Funited-kingdom.html+%28accessed+Jun+3%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swain, M.</span></span> Molecule Validation and Standardization. <a href="https://github.com/mcs07/MolVS" class="extLink">https://github.com/mcs07/MolVS</a> (accessed Jan 17, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Swain%2C+M.+Molecule+Validation+and+Standardization.+https%3A%2F%2Fgithub.com%2Fmcs07%2FMolVS+%28accessed+Jan+17%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DSwain%26aufirst%3DM.%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span>RDKit: Open-source cheminformatics. <a href="https://www.rdkit.org/" class="extLink">https://www.rdkit.org/</a> (accessed
Jan 17, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A+Open-source+cheminformatics.+https%3A%2F%2Fwww.rdkit.org%2F+%28accessed%0AJan+17%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalovich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: A Large-Scale Bioactivity Database for Drug Discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1100</span>â <span class="NLM_lpage">D1107</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&issue=D1&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+Large-Scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0liyDpjJm6ziLw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520Large-Scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirjalili, V.</span></span> Data Clustering in Python. <a href="https://libraries.io/github/mirjalil/pyclust" class="extLink">https://libraries.io/github/mirjalil/pyclust</a> (accessed <span class="NLM_year">2019)</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mirjalili%2C+V.+Data+Clustering+in+Python.+https%3A%2F%2Flibraries.io%2Fgithub%2Fmirjalil%2Fpyclust+%28accessed+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DMirjalili%26aufirst%3DV.%26date%3D2019%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sayle, R.</span></span> <span> </span><span class="NLM_article-title">1st-class SMARTS patterns</span>. <a href="https://www.daylight.com/meetings/summerschool98/course/basics/ref/sayle/" class="extLink">https://www.daylight.com/meetings/summerschool98/course/basics/ref/sayle/</a> (accessed May 31, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=R.+Sayle&title=1st-class+SMARTS+patterns"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DSayle%26aufirst%3DR.%26atitle%3D1st-class%2520SMARTS%2520patterns%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, J.</span>; <span class="NLM_string-name">Schwab, C.</span>; <span class="NLM_string-name">Gasteiger, J.</span></span> <i>CORINA, 3D Structure Generator</i>, version <span class="NLM_edition">3.1</span>; <span class="NLM_publisher-name">Molecular Networks GmbH</span>: <span class="NLM_publisher-loc">Erlangen, Germany</span>, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=J.+Sadowski&author=C.+Schwab&author=J.+Gasteiger&title=CORINA%2C+3D+Structure+Generator"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSadowski%26aufirst%3DJ.%26jtitle%3DCORINA%252C%25203D%2520Structure%2520Generator%26pub%3DMolecular%2520Networks%2520GmbH%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epifanovsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wormit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kussmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaliman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaliullin, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KuÅ, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proynov, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohrdanz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steele, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, H. L.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuev, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alguire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beran, G. J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berquist, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandhorst, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravaya, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casanova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Closser, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crittenden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diedenhofen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiStasio, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutoi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edgar, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fatehi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fusti-Molnar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghysels, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubeva-Zadorozhnaya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson-Heine, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbach, P. H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohenstein, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagau, T.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaduk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khistyaev, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klunzinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosenkov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczyk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauter, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lao, K. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levchenko, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livshits, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lochan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luenser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manohar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manzer, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardirossian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marenich, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhall, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuscamman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oana, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivares-Amaya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâNeill, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhill, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrine, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peverati, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prociuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehn, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russ, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharada, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">StÃ¼ck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thom, A. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimochi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanovschi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vydrov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeganeh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yost, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, I. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, G. K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, W. A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hehre, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, H. F.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherrill, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truhlar, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warshel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreuw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunietz, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlani, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwaltney, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrecht, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochsenfeld, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rassolov, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slipchenko, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subotnik, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Voorhis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krylov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, P. M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head-Gordon, M.</span></span> <span> </span><span class="NLM_article-title">Advances in Molecular Quantum Chemistry Contained in the Q-Chem 4 Program Package</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">184</span>â <span class="NLM_lpage">215</span>, <span class="refDoi">Â DOI: 10.1080/00268976.2014.952696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1080%2F00268976.2014.952696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2ksbnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2015&pages=184-215&issue=2&author=Y.+Shaoauthor=Z.+Ganauthor=E.+Epifanovskyauthor=A.+T.+B.+Gilbertauthor=M.+Wormitauthor=J.+Kussmannauthor=A.+W.+Langeauthor=A.+Behnauthor=J.+Dengauthor=X.+Fengauthor=D.+Ghoshauthor=M.+Goldeyauthor=P.+R.+Hornauthor=L.+D.+Jacobsonauthor=I.+Kalimanauthor=R.+Z.+Khaliullinauthor=T.+Ku%C5%9Bauthor=A.+Landauauthor=J.+Liuauthor=E.+I.+Proynovauthor=Y.+M.+Rheeauthor=R.+M.+Richardauthor=M.+A.+Rohrdanzauthor=R.+P.+Steeleauthor=E.+J.+Sundstromauthor=H.+L.+Woodcockauthor=P.+M.+Zimmermanauthor=D.+Zuevauthor=B.+Albrechtauthor=E.+Alguireauthor=B.+Austinauthor=G.+J.+O.+Beranauthor=Y.+A.+Bernardauthor=E.+Berquistauthor=K.+Brandhorstauthor=K.+B.+Bravayaauthor=S.+T.+Brownauthor=D.+Casanovaauthor=C.-M.+Changauthor=Y.+Chenauthor=S.+H.+Chienauthor=K.+D.+Closserauthor=D.+L.+Crittendenauthor=M.+Diedenhofenauthor=R.+A.+DiStasioauthor=H.+Doauthor=A.+D.+Dutoiauthor=R.+G.+Edgarauthor=S.+Fatehiauthor=L.+Fusti-Molnarauthor=A.+Ghyselsauthor=A.+Golubeva-Zadorozhnayaauthor=J.+Gomesauthor=M.+W.+D.+Hanson-Heineauthor=P.+H.+P.+Harbachauthor=A.+W.+Hauserauthor=E.+G.+Hohensteinauthor=Z.+C.+Holdenauthor=T.-C.+Jagauauthor=H.+Jiauthor=B.+Kadukauthor=K.+Khistyaevauthor=J.+Kimauthor=J.+Kimauthor=R.+A.+Kingauthor=P.+Klunzingerauthor=D.+Kosenkovauthor=T.+Kowalczykauthor=C.+M.+Krauterauthor=K.+U.+Laoauthor=A.+D.+Laurentauthor=K.+V.+Lawlerauthor=S.+V.+Levchenkoauthor=C.+Y.+Linauthor=F.+Liuauthor=E.+Livshitsauthor=R.+C.+Lochanauthor=A.+Luenserauthor=P.+Manoharauthor=S.+F.+Manzerauthor=S.-P.+Maoauthor=N.+Mardirossianauthor=A.+V.+Marenichauthor=S.+A.+Maurerauthor=N.+J.+Mayhallauthor=E.+Neuscammanauthor=C.+M.+Oanaauthor=R.+Olivares-Amayaauthor=D.+P.+O%E2%80%99Neillauthor=J.+A.+Parkhillauthor=T.+M.+Perrineauthor=R.+Peveratiauthor=A.+Prociukauthor=D.+R.+Rehnauthor=E.+Rostaauthor=N.+J.+Russauthor=S.+M.+Sharadaauthor=S.+Sharmaauthor=D.+W.+Smallauthor=A.+Sodtauthor=T.+Steinauthor=D.+St%C3%BCckauthor=Y.-C.+Suauthor=A.+J.+W.+Thomauthor=T.+Tsuchimochiauthor=V.+Vanovschiauthor=L.+Vogtauthor=O.+Vydrovauthor=T.+Wangauthor=M.+A.+Watsonauthor=J.+Wenzelauthor=A.+Whiteauthor=C.+F.+Williamsauthor=J.+Yangauthor=S.+Yeganehauthor=S.+R.+Yostauthor=Z.-Q.+Youauthor=I.+Y.+Zhangauthor=X.+Zhangauthor=Y.+Zhaoauthor=B.+R.+Brooksauthor=G.+K.+L.+Chanauthor=D.+M.+Chipmanauthor=C.+J.+Cramerauthor=W.+A.+Goddardauthor=M.+S.+Gordonauthor=W.+J.+Hehreauthor=A.+Klamtauthor=H.+F.+Schaeferauthor=M.+W.+Schmidtauthor=C.+D.+Sherrillauthor=D.+G.+Truhlarauthor=A.+Warshelauthor=X.+Xuauthor=A.+Aspuru-Guzikauthor=R.+Baerauthor=A.+T.+Bellauthor=N.+A.+Besleyauthor=J.-D.+Chaiauthor=A.+Dreuwauthor=B.+D.+Dunietzauthor=T.+R.+Furlaniauthor=S.+R.+Gwaltneyauthor=C.-P.+Hsuauthor=Y.+Jungauthor=J.+Kongauthor=D.+S.+Lambrechtauthor=W.+Liangauthor=C.+Ochsenfeldauthor=V.+A.+Rassolovauthor=L.+V.+Slipchenkoauthor=J.+E.+Subotnikauthor=T.+Van+Voorhisauthor=J.+M.+Herbertauthor=A.+I.+Krylovauthor=P.+M.+W.+Gillauthor=M.+Head-Gordon&title=Advances+in+Molecular+Quantum+Chemistry+Contained+in+the+Q-Chem+4+Program+Package&doi=10.1080%2F00268976.2014.952696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in molecular quantum chemistry contained in the Q-Chem 4 program package</span></div><div class="casAuthors">Shao, Yihan; Gan, Zhengting; Epifanovsky, Evgeny; Gilbert, Andrew T. B.; Wormit, Michael; Kussmann, Joerg; Lange, Adrian W.; Behn, Andrew; Deng, Jia; Feng, Xintian; Ghosh, Debashree; Goldey, Matthew; Horn, Paul R.; Jacobson, Leif D.; Kaliman, Ilya; Khaliullin, Rustam Z.; Kus, Tomasz; Landau, Arie; Liu, Jie; Proynov, Emil I.; Rhee, Young Min; Richard, Ryan M.; Rohrdanz, Mary A.; Steele, Ryan P.; Sundstrom, Eric J.; Woodcock, H. Lee, III; Zimmerman, Paul M.; Zuev, Dmitry; Albrecht, Ben; Alguire, Ethan; Austin, Brian; Beran, Gregory J. O.; Bernard, Yves A.; Berquist, Eric; Brandhorst, Kai; Bravaya, Ksenia B.; Brown, Shawn T.; Casanova, David; Chang, Chung-Min; Chen, Yunquing; Chien, Siu Hung; Closser, Kristina D.; Crittenden, Deborah L.; Diedenhofen, Michael; DiStasio, Robert A., Jr.; Do, Hainam; Dutoi, Anthony D.; Edgar, Richard G.; Fatehi, Shervin; Fusti-Molnar, Laszlo; Ghysels, An; Golubeva-Zadorozhnaya, Anna; Gomes, Joseph; Hanson-Heine, Magnus W. D.; Harbach, Philipp H. P.; Hauser, Andreas W.; Hohenstein, Edward G.; Holden, Zachary C.; Jagau, Thomas-C.; Ji, Hyunjun; Kaduk, Ben; Khistyaev, Kirill; Kim, Jaehoon; Kim, Jihan; King, Rollin A.; Klunzinger, Phil; Kosenkov, Dmytro; Kowalczyk, Tim; Krauter, Caroline M.; Lao, Ka Un; Laurent, Adele; Lawler, Keith V.; Levchenko, Sergey V.; Lin, Ching Yeh; Liu, Fenglai; Livshits, Ester; Lochan, Rohini C.; Luenser, Arne; Manohar, Prashant; Manzer, Samuel F.; Mao, Shan-Ping; Mardirossian, Narbe; Marenich, Aleksandr V.; Maurer, Simon A.; Mayhall, Nicholas J.; Neuscamman, Eric; Oana, C. Melania; Olivares-Amaya, Roberto; O'Neill, Darragh P.; Parkhill, John A.; Perrine, Trilisa M.; Peverati, Roberto; Prociuk, Alexander; Rehn, Dirk R.; Rosta, Edina; Russ, Nicholas J.; Sharada, Shaama M.; Sharma, Sandeep; Small, David W.; Sodt, Alexander; Stein, Tamar; Stuck, David; Su, Yu-Chuan; Thom, Alex J. W.; Tsuchimochi, Takashi; Vanovschi, Vitalii; Vogt, Leslie; Vydrov, Oleg; Wang, Tao; Watson, Mark A.; Wenzel, Jan; White, Alec; Williams, Christopher F.; Yang, Jun; Yeganeh, Sina; Yost, Shane R.; You, Zhi-Qiang; Zhang, Igor Ying; Zhang, Xing; Zhao, Yan; Brooks, Bernard R.; Chan, Garnet K. L.; Chipman, Daniel M.; Cramer, Christopher J.; Goddard, William A., III; Gordon, Mark S.; Hehre, Warren J.; Klamt, Andreas; Schaefer, Henry F., III; Schmidt, Michael W.; Sherrill, C. David; Truhlar, Donald G.; Warshel, Arieh; Xu, Xin; Aspuru-Guzik, Alan; Baer, Roi; Bell, Alexis T.; Besley, Nicholas A.; Chai, Jeng-Da; Dreuw, Andreas; Dunietz, Barry D.; Furlani, Thomas R.; Gwaltney, Steven R.; Hsu, Chao-Ping; Jung, Yousung; Kong, Jing; Lambrecht, Daniel S.; Liang, WanZhen; Ochsenfeld, Christian; Rassolov, Vitaly A.; Slipchenko, Lyudmila V.; Subotnik, Joseph E.; Van Voorhis, Troy; Herbert, John M.; Krylov, Anna I.; Gill, Peter M. W.; Head-Gordon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">184-215</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  A summary of the tech. advances that are incorporated in the fourth major release of the Q-Chem quantum chem. program is provided, covering approx. the last seven years.  These include developments in d. functional theory methods and algorithms, NMR (NMR) property evaluation, coupled cluster and perturbation theories, methods for electronically excited and open-shell species, tools for treating extended environments, algorithms for walking on potential surfaces, anal. tools, energy and electron transfer modeling, parallel computing capabilities, and graphical user interfaces.  In addn., a selection of example case studies that illustrate these capabilities is given.  These include extensive benchmarks of the comparative accuracy of modern d. functionals for bonded and non-bonded interactions, tests of attenuated second order Moller-Plesset (MP2) methods for intermol. interactions, a variety of parallel performance benchmarks, and tests of the accuracy of implicit solvation models.  Some specific chem. examples include calcns. on the strongly correlated Cr2 dimer, exploring zeolite-catalyzed ethane dehydrogenation, energy decompn. anal. of a charged ter-mol. complex arising from glycerol photoionisation, and natural transition orbitals for a Frenkel exciton state in a nine-unit model of a self-assembling nanotube.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohxCdZ6JkTu7Vg90H21EOLACvtfcHk0ljNDo2hHW2tkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2ksbnN&md5=a828159693d247dd683f67fe217fb909</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1080%2F00268976.2014.952696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268976.2014.952696%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DY.%26aulast%3DGan%26aufirst%3DZ.%26aulast%3DEpifanovsky%26aufirst%3DE.%26aulast%3DGilbert%26aufirst%3DA.%2BT.%2BB.%26aulast%3DWormit%26aufirst%3DM.%26aulast%3DKussmann%26aufirst%3DJ.%26aulast%3DLange%26aufirst%3DA.%2BW.%26aulast%3DBehn%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DGoldey%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DP.%2BR.%26aulast%3DJacobson%26aufirst%3DL.%2BD.%26aulast%3DKaliman%26aufirst%3DI.%26aulast%3DKhaliullin%26aufirst%3DR.%2BZ.%26aulast%3DKu%25C5%259B%26aufirst%3DT.%26aulast%3DLandau%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DProynov%26aufirst%3DE.%2BI.%26aulast%3DRhee%26aufirst%3DY.%2BM.%26aulast%3DRichard%26aufirst%3DR.%2BM.%26aulast%3DRohrdanz%26aufirst%3DM.%2BA.%26aulast%3DSteele%26aufirst%3DR.%2BP.%26aulast%3DSundstrom%26aufirst%3DE.%2BJ.%26aulast%3DWoodcock%26aufirst%3DH.%2BL.%26aulast%3DZimmerman%26aufirst%3DP.%2BM.%26aulast%3DZuev%26aufirst%3DD.%26aulast%3DAlbrecht%26aufirst%3DB.%26aulast%3DAlguire%26aufirst%3DE.%26aulast%3DAustin%26aufirst%3DB.%26aulast%3DBeran%26aufirst%3DG.%2BJ.%2BO.%26aulast%3DBernard%26aufirst%3DY.%2BA.%26aulast%3DBerquist%26aufirst%3DE.%26aulast%3DBrandhorst%26aufirst%3DK.%26aulast%3DBravaya%26aufirst%3DK.%2BB.%26aulast%3DBrown%26aufirst%3DS.%2BT.%26aulast%3DCasanova%26aufirst%3DD.%26aulast%3DChang%26aufirst%3DC.-M.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChien%26aufirst%3DS.%2BH.%26aulast%3DClosser%26aufirst%3DK.%2BD.%26aulast%3DCrittenden%26aufirst%3DD.%2BL.%26aulast%3DDiedenhofen%26aufirst%3DM.%26aulast%3DDiStasio%26aufirst%3DR.%2BA.%26aulast%3DDo%26aufirst%3DH.%26aulast%3DDutoi%26aufirst%3DA.%2BD.%26aulast%3DEdgar%26aufirst%3DR.%2BG.%26aulast%3DFatehi%26aufirst%3DS.%26aulast%3DFusti-Molnar%26aufirst%3DL.%26aulast%3DGhysels%26aufirst%3DA.%26aulast%3DGolubeva-Zadorozhnaya%26aufirst%3DA.%26aulast%3DGomes%26aufirst%3DJ.%26aulast%3DHanson-Heine%26aufirst%3DM.%2BW.%2BD.%26aulast%3DHarbach%26aufirst%3DP.%2BH.%2BP.%26aulast%3DHauser%26aufirst%3DA.%2BW.%26aulast%3DHohenstein%26aufirst%3DE.%2BG.%26aulast%3DHolden%26aufirst%3DZ.%2BC.%26aulast%3DJagau%26aufirst%3DT.-C.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DKaduk%26aufirst%3DB.%26aulast%3DKhistyaev%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKing%26aufirst%3DR.%2BA.%26aulast%3DKlunzinger%26aufirst%3DP.%26aulast%3DKosenkov%26aufirst%3DD.%26aulast%3DKowalczyk%26aufirst%3DT.%26aulast%3DKrauter%26aufirst%3DC.%2BM.%26aulast%3DLao%26aufirst%3DK.%2BU.%26aulast%3DLaurent%26aufirst%3DA.%2BD.%26aulast%3DLawler%26aufirst%3DK.%2BV.%26aulast%3DLevchenko%26aufirst%3DS.%2BV.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLivshits%26aufirst%3DE.%26aulast%3DLochan%26aufirst%3DR.%2BC.%26aulast%3DLuenser%26aufirst%3DA.%26aulast%3DManohar%26aufirst%3DP.%26aulast%3DManzer%26aufirst%3DS.%2BF.%26aulast%3DMao%26aufirst%3DS.-P.%26aulast%3DMardirossian%26aufirst%3DN.%26aulast%3DMarenich%26aufirst%3DA.%2BV.%26aulast%3DMaurer%26aufirst%3DS.%2BA.%26aulast%3DMayhall%26aufirst%3DN.%2BJ.%26aulast%3DNeuscamman%26aufirst%3DE.%26aulast%3DOana%26aufirst%3DC.%2BM.%26aulast%3DOlivares-Amaya%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DD.%2BP.%26aulast%3DParkhill%26aufirst%3DJ.%2BA.%26aulast%3DPerrine%26aufirst%3DT.%2BM.%26aulast%3DPeverati%26aufirst%3DR.%26aulast%3DProciuk%26aufirst%3DA.%26aulast%3DRehn%26aufirst%3DD.%2BR.%26aulast%3DRosta%26aufirst%3DE.%26aulast%3DRuss%26aufirst%3DN.%2BJ.%26aulast%3DSharada%26aufirst%3DS.%2BM.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSmall%26aufirst%3DD.%2BW.%26aulast%3DSodt%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DT.%26aulast%3DSt%25C3%25BCck%26aufirst%3DD.%26aulast%3DSu%26aufirst%3DY.-C.%26aulast%3DThom%26aufirst%3DA.%2BJ.%2BW.%26aulast%3DTsuchimochi%26aufirst%3DT.%26aulast%3DVanovschi%26aufirst%3DV.%26aulast%3DVogt%26aufirst%3DL.%26aulast%3DVydrov%26aufirst%3DO.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWatson%26aufirst%3DM.%2BA.%26aulast%3DWenzel%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DC.%2BF.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DYeganeh%26aufirst%3DS.%26aulast%3DYost%26aufirst%3DS.%2BR.%26aulast%3DYou%26aufirst%3DZ.-Q.%26aulast%3DZhang%26aufirst%3DI.%2BY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DChan%26aufirst%3DG.%2BK.%2BL.%26aulast%3DChipman%26aufirst%3DD.%2BM.%26aulast%3DCramer%26aufirst%3DC.%2BJ.%26aulast%3DGoddard%26aufirst%3DW.%2BA.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DHehre%26aufirst%3DW.%2BJ.%26aulast%3DKlamt%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DH.%2BF.%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26aulast%3DWarshel%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26aulast%3DBaer%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DA.%2BT.%26aulast%3DBesley%26aufirst%3DN.%2BA.%26aulast%3DChai%26aufirst%3DJ.-D.%26aulast%3DDreuw%26aufirst%3DA.%26aulast%3DDunietz%26aufirst%3DB.%2BD.%26aulast%3DFurlani%26aufirst%3DT.%2BR.%26aulast%3DGwaltney%26aufirst%3DS.%2BR.%26aulast%3DHsu%26aufirst%3DC.-P.%26aulast%3DJung%26aufirst%3DY.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DLambrecht%26aufirst%3DD.%2BS.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DOchsenfeld%26aufirst%3DC.%26aulast%3DRassolov%26aufirst%3DV.%2BA.%26aulast%3DSlipchenko%26aufirst%3DL.%2BV.%26aulast%3DSubotnik%26aufirst%3DJ.%2BE.%26aulast%3DVan%2BVoorhis%26aufirst%3DT.%26aulast%3DHerbert%26aufirst%3DJ.%2BM.%26aulast%3DKrylov%26aufirst%3DA.%2BI.%26aulast%3DGill%26aufirst%3DP.%2BM.%2BW.%26aulast%3DHead-Gordon%26aufirst%3DM.%26atitle%3DAdvances%2520in%2520Molecular%2520Quantum%2520Chemistry%2520Contained%2520in%2520the%2520Q-Chem%25204%2520Program%2520Package%26jtitle%3DMol.%2520Phys.%26date%3D2015%26volume%3D113%26issue%3D2%26spage%3D184%26epage%3D215%26doi%3D10.1080%2F00268976.2014.952696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N.</span></span> APBS-PDB2PQR Documentation. <a href="https://buildmedia.readthedocs.org/media/pdf/apbs-pdb2pqr/latest/apbs-pdb2pqr.pdf" class="extLink">https://buildmedia.readthedocs.org/media/pdf/apbs-pdb2pqr/latest/apbs-pdb2pqr.pdf</a> (accessed <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Baker%2C+N.+APBS-PDB2PQR+Documentation.+https%3A%2F%2Fbuildmedia.readthedocs.org%2Fmedia%2Fpdf%2Fapbs-pdb2pqr%2Flatest%2Fapbs-pdb2pqr.pdf+%28accessed+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, M. L.</span></span> <span> </span><span class="NLM_article-title">Solvent-Accessible Surfaces of Proteins and Nucleic Acids</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>221</i></span> (<span class="NLM_issue">4612</span>),  <span class="NLM_fpage">709</span>â <span class="NLM_lpage">713</span>, <span class="refDoi">Â DOI: 10.1126/science.6879170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1126%2Fscience.6879170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=6879170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaL3sXlsFCkt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=221&publication_year=1983&pages=709-713&issue=4612&author=M.+L.+Connolly&title=Solvent-Accessible+Surfaces+of+Proteins+and+Nucleic+Acids&doi=10.1126%2Fscience.6879170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Solvent-accessible surfaces of proteins and nucleic acids</span></div><div class="casAuthors">Connolly, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">221</span>
        (<span class="NLM_cas:issue">4612</span>),
    <span class="NLM_cas:pages">709-13</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A method is presented for anal. calcg. a smooth, 3-dimensional contour about a mol.  The mol. surface envelope may be drawn on either color raster computer displays or real-time vector computer graphics systems.  Mol. areas and vols. may be computed anal. from this surface representation.  Unlike most previous computer graphics representations of mols., which imitate wire models or space-filling plastic spheres, this surface shows only the atoms that are accessible to solvent.  This anal. method extends the earlier dot surface numerical algorithm, which was applied in enzymol., rational drug design, immunol., and understanding DNA base sequence recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl7dEGpUZzhrVg90H21EOLACvtfcHk0liQpDZ7V5qIJA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXlsFCkt74%253D&md5=371e91867872dcc45ff5d9c5cc142c95</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.6879170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.6879170%26sid%3Dliteratum%253Aachs%26aulast%3DConnolly%26aufirst%3DM.%2BL.%26atitle%3DSolvent-Accessible%2520Surfaces%2520of%2520Proteins%2520and%2520Nucleic%2520Acids%26jtitle%3DScience%26date%3D1983%26volume%3D221%26issue%3D4612%26spage%3D709%26epage%3D713%26doi%3D10.1126%2Fscience.6879170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, P. W.</span></span> <span> </span><span class="NLM_article-title">NGL Viewer: A Web Application for Molecular Visualization</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W576</span>â <span class="NLM_lpage">W579</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkv402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1093%2Fnar%2Fgkv402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=25925569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbvL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W576-W579&issue=W1&author=A.+S.+Roseauthor=P.+W.+Hildebrand&title=NGL+Viewer%3A+A+Web+Application+for+Molecular+Visualization&doi=10.1093%2Fnar%2Fgkv402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">NGL Viewer: a web application for molecular visualization</span></div><div class="casAuthors">Rose, Alexander S.; Hildebrand, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W576-W579</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The NGL Viewer is a web application for the visualization of macromol. structures.  By fully adopting capabilities of modern web browsers, such as WebGL, for mol. graphics, the viewer can interactively display large mol. complexes and is also unaffected by the retirement of third-party plug-ins like Flash and Java Applets.  Generally, the web application offers comprehensive mol. visualization through a graphical user interface so that life scientists can easily access and profit from available structural data.  It supports common structural fileformats (e.g. PDB, mmCIF) and a variety of mol. representations (e.g. 'cartoon, spacefill, licorice').  Moreover, the viewer can be embedded in other web sites to provide specialized visualizations of entries in structural databases or results of structure-related calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq2USM5TOiD7Vg90H21EOLACvtfcHk0lhSKrjAlpyuuA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbvL&md5=c6090488c03049f5dff2386aed09b738</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv402%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DA.%2BS.%26aulast%3DHildebrand%26aufirst%3DP.%2BW.%26atitle%3DNGL%2520Viewer%253A%2520A%2520Web%2520Application%2520for%2520Molecular%2520Visualization%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DW1%26spage%3DW576%26epage%3DW579%26doi%3D10.1093%2Fnar%2Fgkv402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span> <span> </span><span class="NLM_article-title">AstexViewer: A Visualisation Aid for Structure-Based Drug Design</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">871</span>â <span class="NLM_lpage">881</span>, <span class="refDoi">Â DOI: 10.1023/A:1023813504011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1023%2FA%3A1023813504011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=12825620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=871-881&issue=12&author=M.+J.+Hartshorn&title=AstexViewer%3A+A+Visualisation+Aid+for+Structure-Based+Drug+Design&doi=10.1023%2FA%3A1023813504011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">AstexViewer: a visualisation aid for structure-based drug design</span></div><div class="casAuthors">Hartshorn Michael J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">871-81</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">AstexViewer is a Java molecular graphics program that can be used for visualisation in many aspects of structure-based drug design.  This paper describes its functionality, implementation and examples of its use.  The program can run as an Applet in a web browser allowing structures to be displayed without installing additional software.  Applications of its use are described for visualisation and as part of a structure based design platform.  The software is being made freely available to the community and may be downloaded from http://www.astex-technology.com/AstexViewer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPr0A35nMvHjMsgLhIoICsfW6udTcc2eYuC_3xqvXhcbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D&md5=f9396aae9ecf64f256a9f8588f8097bd</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023813504011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023813504011%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26atitle%3DAstexViewer%253A%2520A%2520Visualisation%2520Aid%2520for%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26issue%3D12%26spage%3D871%26epage%3D881%26doi%3D10.1023%2FA%3A1023813504011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, R. J.</span></span> <span> </span><span class="NLM_article-title">The Amber Biomolecular Simulation Programs</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1668</span>â <span class="NLM_lpage">1688</span>, <span class="refDoi">Â DOI: 10.1002/jcc.20290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&issue=16&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+Biomolecular+Simulation+Programs&doi=10.1002%2Fjcc.20290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0lguMS1hfVUu9A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520Biomolecular%2520Simulation%2520Programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26issue%3D16%26spage%3D1668%26epage%3D1688%26doi%3D10.1002%2Fjcc.20290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chollet, F.</span></span>; <span class="NLM_etal"></span>.  <i>Keras</i>; <span class="NLM_publisher-name">GitHub</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=F.+Chollet&title=Keras"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DChollet%26aufirst%3DF.%26jtitle%3DKeras%26pub%3DGitHub%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abadi, M.</span>; <span class="NLM_string-name">Agarwal, A.</span>; <span class="NLM_string-name">Barham, P.</span>; <span class="NLM_string-name">Brevdo, E.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Citro, C.</span>; <span class="NLM_string-name">Corrado, G. S.</span>; <span class="NLM_string-name">Davis, A.</span>; <span class="NLM_string-name">Dean, J.</span>; <span class="NLM_string-name">Devin, M.</span>; <span class="NLM_string-name">Ghemawat, S.</span>; <span class="NLM_string-name">Goodfellow, I.</span>; <span class="NLM_string-name">Harp, A.</span>; <span class="NLM_string-name">Irving, G.</span>; <span class="NLM_string-name">Isard, M.</span>; <span class="NLM_string-name">Jia, Y.</span>; <span class="NLM_string-name">Jozefowicz, R.</span>; <span class="NLM_string-name">Kaiser, L.</span>; <span class="NLM_string-name">Kudlur, M.</span>; <span class="NLM_string-name">Levenberg, J.</span>; <span class="NLM_string-name">ManÃ©, D.</span>; <span class="NLM_string-name">Monga, R.</span>; <span class="NLM_string-name">Moore, S.</span>; <span class="NLM_string-name">Murray, D.</span>; <span class="NLM_string-name">Olah, C.</span>; <span class="NLM_string-name">Schuster, M.</span>; <span class="NLM_string-name">Shlens, J.</span>; <span class="NLM_string-name">Steiner, B.</span>; <span class="NLM_string-name">Sutskever, I.</span>; <span class="NLM_string-name">Talwar, K.</span>; <span class="NLM_string-name">Tucker, P.</span>; <span class="NLM_string-name">Vanhoucke, V.</span>; <span class="NLM_string-name">Vasudevan, V.</span>; <span class="NLM_string-name">ViÃ©gas, F.</span>; <span class="NLM_string-name">Vinyals, O.</span>; <span class="NLM_string-name">Warden, P.</span>; <span class="NLM_string-name">Wattenberg, M.</span>; <span class="NLM_string-name">Wicke, M.</span>; <span class="NLM_string-name">Yu, Y.</span>; <span class="NLM_string-name">Zheng, X.</span></span> <span> </span><span class="NLM_article-title">TensorFlow: Large-Scale Machine Learning on Heterogeneous Systems</span>.  <i>arXiv:1603.044</i>, <span class="NLM_year" style="font-weight: bold;">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=M.+Abadi&author=A.+Agarwal&author=P.+Barham&author=E.+Brevdo&author=Z.+Chen&author=C.+Citro&author=G.+S.+Corrado&author=A.+Davis&author=J.+Dean&author=M.+Devin&author=S.+Ghemawat&author=I.+Goodfellow&author=A.+Harp&author=G.+Irving&author=M.+Isard&author=Y.+Jia&author=R.+Jozefowicz&author=L.+Kaiser&author=M.+Kudlur&author=J.+Levenberg&author=D.+Man%C3%A9&author=R.+Monga&author=S.+Moore&author=D.+Murray&author=C.+Olah&author=M.+Schuster&author=J.+Shlens&author=B.+Steiner&author=I.+Sutskever&author=K.+Talwar&author=P.+Tucker&author=V.+Vanhoucke&author=V.+Vasudevan&author=F.+Vi%C3%A9gas&author=O.+Vinyals&author=P.+Warden&author=M.+Wattenberg&author=M.+Wicke&author=Y.+Yu&author=X.+Zheng&title=TensorFlow%3A+Large-Scale+Machine+Learning+on+Heterogeneous+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbadi%26aufirst%3DM.%26atitle%3DTensorFlow%253A%2520Large-Scale%2520Machine%2520Learning%2520on%2520Heterogeneous%2520Systems%26jtitle%3DarXiv%253A1603.044%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glorot, X.</span>; <span class="NLM_string-name">Bordes, A.</span>; <span class="NLM_string-name">Bengio, Y.</span></span> <span> </span><span class="NLM_article-title">Deep Sparse Rectifier Neural Networks</span>.  <i>Proceedings of the Fourteenth International Conference on Artificial Intelligence and Statistics</i>; <span class="NLM_publisher-name">JMLR</span>, <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">315</span>â <span class="NLM_lpage">323</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=315-323&author=X.+Glorot&author=A.+Bordes&author=Y.+Bengio&title=Proceedings+of+the+Fourteenth+International+Conference+on+Artificial+Intelligence+and+Statistics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGlorot%26aufirst%3DX.%26atitle%3DDeep%2520Sparse%2520Rectifier%2520Neural%2520Networks%26btitle%3DProceedings%2520of%2520the%2520Fourteenth%2520International%2520Conference%2520on%2520Artificial%2520Intelligence%2520and%2520Statistics%26pub%3DJMLR%26date%3D2011%26spage%3D315%26epage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kingma, D. P.</span>; <span class="NLM_string-name">Ba, J.</span></span> <span> </span><span class="NLM_article-title">Adam: A Method for Stochastic Optimization</span>.  <i>ArXiv14126980</i> <span class="NLM_year" style="font-weight: bold;">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+P.+Kingma&author=J.+Ba&title=Adam%3A+A+Method+for+Stochastic+Optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKingma%26aufirst%3DD.%2BP.%26atitle%3DAdam%253A%2520A%2520Method%2520for%2520Stochastic%2520Optimization%26jtitle%3DArXiv14126980%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 14 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xiaoyang Liu, Kathryn E. OâHarra, Jason E. Bara, <span class="NLM_string-name hlFld-ContribAuthor">C. Heath Turner</span>. </span><span class="cited-content_cbyCitation_article-title">Screening Ionic Liquids Based on Ionic Volume and Electrostatic Potential Analyses. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2021,</strong> <em>125 </em>
                                    (14)
                                     , 3653-3664. <a href="https://doi.org/10.1021/acs.jpcb.0c10259" title="DOI URL">https://doi.org/10.1021/acs.jpcb.0c10259</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.0c10259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.0c10259%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DScreening%252BIonic%252BLiquids%252BBased%252Bon%252BIonic%252BVolume%252Band%252BElectrostatic%252BPotential%252BAnalyses%26aulast%3DLiu%26aufirst%3DXiaoyang%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D13112020%26date%3D02022021%26date%3D06042021%26volume%3D125%26issue%3D14%26spage%3D3653%26epage%3D3664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhengdan Zhu, Zhijian Xu, <span class="NLM_string-name hlFld-ContribAuthor">Weiliang Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction Nature and Computational Methods for Halogen Bonding: A Perspective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2683-2696. <a href="https://doi.org/10.1021/acs.jcim.0c00032" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00032%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInteraction%252BNature%252Band%252BComputational%252BMethods%252Bfor%252BHalogen%252BBonding%25253A%252BA%252BPerspective%26aulast%3DZhu%26aufirst%3DZhengdan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14012020%26date%3D08062020%26date%3D26052020%26volume%3D60%26issue%3D6%26spage%3D2683%26epage%3D2696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Agata  Bartyzel</span>, <span class="hlFld-ContribAuthor ">Magda  Kondej</span>, <span class="hlFld-ContribAuthor ">Piotr  StÄpnicki</span>, <span class="hlFld-ContribAuthor ">Tomasz M.  WrÃ³bel</span>, <span class="hlFld-ContribAuthor ">Agnieszka A.  Kaczor</span>. </span><span class="cited-content_cbyCitation_article-title">Experimental and computational structural studies of 5-substituted-3-(1-arylmethyl-1,2,3,6-tetrahydropyridin-4-yl)-1H-indoles. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1245 </em>, 130998. <a href="https://doi.org/10.1016/j.molstruc.2021.130998" title="DOI URL">https://doi.org/10.1016/j.molstruc.2021.130998</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2021.130998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2021.130998%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DExperimental%252Band%252Bcomputational%252Bstructural%252Bstudies%252Bof%252B5-substituted-3-%2525281-arylmethyl-1%25252C2%25252C3%25252C6-tetrahydropyridin-4-yl%252529-1H-indoles%26aulast%3DBartyzel%26aufirst%3DAgata%26date%3D2021%26volume%3D1245%26spage%3D130998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Queen S.  Obu</span>, <span class="hlFld-ContribAuthor ">Hitler  Louis</span>, <span class="hlFld-ContribAuthor ">Joseph O.  Odey</span>, <span class="hlFld-ContribAuthor ">Ishegbe Joyce  Eko</span>, <span class="hlFld-ContribAuthor ">Shuaibu  Abdullahi</span>, <span class="hlFld-ContribAuthor ">Tabe N.  Ntui</span>, <span class="hlFld-ContribAuthor ">Ofiong E.  Offiong</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, spectra (FT-IR, NMR) investigations, DFT study, in silico ADMET and Molecular docking analysis of 2-amino-4-(4-aminophenyl)thiophene-3-carbonitrile as a potential anti-tubercular agent. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1244 </em>, 130880. <a href="https://doi.org/10.1016/j.molstruc.2021.130880" title="DOI URL">https://doi.org/10.1016/j.molstruc.2021.130880</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2021.130880&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2021.130880%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DSynthesis%25252C%252Bspectra%252B%252528FT-IR%25252C%252BNMR%252529%252Binvestigations%25252C%252BDFT%252Bstudy%25252C%252Bin%252Bsilico%252BADMET%252Band%252BMolecular%252Bdocking%252Banalysis%252Bof%252B2-amino-4-%2525284-aminophenyl%252529thiophene-3-carbonitrile%252Bas%252Ba%252Bpotential%252Banti-tubercular%252Bagent%26aulast%3DObu%26aufirst%3DQueen%2BS.%26date%3D2021%26volume%3D1244%26spage%3D130880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Changxing  Sun</span>, <span class="hlFld-ContribAuthor ">Shuai  Zhang</span>, <span class="hlFld-ContribAuthor ">Ping  Qian</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Hao  Deng</span>, <span class="hlFld-ContribAuthor ">Wansheng  Ren</span>, <span class="hlFld-ContribAuthor ">Lin  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and fungicidal activity of novel 2-(2-alkylthio-6-phenylpyrimidin-4-yl)-1H-benzimidazoles. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>47 </em>, 128210. <a href="https://doi.org/10.1016/j.bmcl.2021.128210" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128210%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bfungicidal%252Bactivity%252Bof%252Bnovel%252B2-%2525282-alkylthio-6-phenylpyrimidin-4-yl%252529-1H-benzimidazoles%26aulast%3DSun%26aufirst%3DChangxing%26date%3D2021%26volume%3D47%26spage%3D128210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao-Meng  Zhang</span>, <span class="hlFld-ContribAuthor ">Li  Liang</span>, <span class="hlFld-ContribAuthor ">Lin  Liu</span>, <span class="hlFld-ContribAuthor ">Ming-Jing  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Graph Neural Networks and Their Current Applications in Bioinformatics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fgene.2021.690049" title="DOI URL">https://doi.org/10.3389/fgene.2021.690049</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2021.690049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2021.690049%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DGraph%252BNeural%252BNetworks%252Band%252BTheir%252BCurrent%252BApplications%252Bin%252BBioinformatics%26aulast%3DZhang%26aufirst%3DXiao-Meng%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia Ramos  Faillace</span>, <span class="hlFld-ContribAuthor ">Paula Bacaicoa  Caruso</span>, <span class="hlFld-ContribAuthor ">Luis Fernando Saraiva Macedo  Timmers</span>, <span class="hlFld-ContribAuthor ">DÃ©bora  Favero</span>, <span class="hlFld-ContribAuthor ">Frank Lino  Guzman</span>, <span class="hlFld-ContribAuthor ">Ciliana  Rechenmacher</span>, <span class="hlFld-ContribAuthor ">Luisa Abruzzi  de Oliveira-Busatto</span>, <span class="hlFld-ContribAuthor ">Osmar Norberto  de Souza</span>, <span class="hlFld-ContribAuthor ">Christian  Bredemeier</span>, <span class="hlFld-ContribAuthor ">Maria Helena  Bodanese-Zanettini</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Characterisation of Soybean Osmotins and Their Involvement in Drought Stress Response. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Genetics</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fgene.2021.632685" title="DOI URL">https://doi.org/10.3389/fgene.2021.632685</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fgene.2021.632685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffgene.2021.632685%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Genetics%26atitle%3DMolecular%252BCharacterisation%252Bof%252BSoybean%252BOsmotins%252Band%252BTheir%252BInvolvement%252Bin%252BDrought%252BStress%252BResponse%26aulast%3DFaillace%26aufirst%3DGiulia%2BRamos%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akmaljon G.  Tojiboev</span>, <span class="hlFld-ContribAuthor ">Burkhon Zh.  Elmuradov</span>, <span class="hlFld-ContribAuthor ">Halima  Mouhib</span>, <span class="hlFld-ContribAuthor ">Kambarali K.  Turgunov</span>, <span class="hlFld-ContribAuthor ">Askar Sh.  Abdurazakov</span>, <span class="hlFld-ContribAuthor ">Charos E.  Makhmadiyarova</span>, <span class="hlFld-ContribAuthor ">Bakhodir  Tashkhodjaev</span>, <span class="hlFld-ContribAuthor ">Sirojiddin Z.  Mirzaev</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight from intermolecular interactions and energy framework analyses of 2-substituted 6,7,8,9-tetrahydro-11
              H
              -pyrido[2,1-b]quinazolin-11-ones. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section B Structural Science, Crystal Engineering and Materials</span><span> <strong>2021,</strong> <em>77 </em>
                                    (3)
                                     , 416-426. <a href="https://doi.org/10.1107/S2052520621003498" title="DOI URL">https://doi.org/10.1107/S2052520621003498</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2052520621003498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2052520621003498%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520B%2520Structural%2520Science%252C%2520Crystal%2520Engineering%2520and%2520Materials%26atitle%3DStructural%252Binsight%252Bfrom%252Bintermolecular%252Binteractions%252Band%252Benergy%252Bframework%252Banalyses%252Bof%252B2-substituted%252B6%25252C7%25252C8%25252C9-tetrahydro-11%252BH%252B-pyrido%25255B2%25252C1-b%25255Dquinazolin-11-ones%26aulast%3DTojiboev%26aufirst%3DAkmaljon%2BG.%26date%3D2021%26date%3D2021%26volume%3D77%26issue%3D3%26spage%3D416%26epage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Temitope Isaac  Adelusi</span>, <span class="hlFld-ContribAuthor ">Misbaudeen  Abdul-Hammed</span>, <span class="hlFld-ContribAuthor ">Mukhtar Oluwaseun  Idris</span>, <span class="hlFld-ContribAuthor ">Qudus Kehinde  Oyedele</span>, <span class="hlFld-ContribAuthor ">Ibrahim Olaide  Adedotun</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular dynamics, quantum mechanics and docking studies of some Keap1 inhibitors â An insight into the atomistic mechanisms of their antioxidant potential. </span><span class="cited-content_cbyCitation_journal-name">Heliyon</span><span> <strong>2021,</strong> <em>7 </em>
                                    (6)
                                     , e07317. <a href="https://doi.org/10.1016/j.heliyon.2021.e07317" title="DOI URL">https://doi.org/10.1016/j.heliyon.2021.e07317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.heliyon.2021.e07317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.heliyon.2021.e07317%26sid%3Dliteratum%253Aachs%26jtitle%3DHeliyon%26atitle%3DMolecular%252Bdynamics%25252C%252Bquantum%252Bmechanics%252Band%252Bdocking%252Bstudies%252Bof%252Bsome%252BKeap1%252Binhibitors%252B%2525E2%252580%252593%252BAn%252Binsight%252Binto%252Bthe%252Batomistic%252Bmechanisms%252Bof%252Btheir%252Bantioxidant%252Bpotential%26aulast%3DAdelusi%26aufirst%3DTemitope%2BIsaac%26date%3D2021%26volume%3D7%26issue%3D6%26spage%3De07317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mateus SÃ¡ MagalhÃ£es  Serafim</span>, <span class="hlFld-ContribAuthor ">Valtair Severino  dos Santos JÃºnior</span>, <span class="hlFld-ContribAuthor ">Jadson Castro  Gertrudes</span>, <span class="hlFld-ContribAuthor ">VinÃ­cius GonÃ§alves  Maltarollo</span>, <span class="hlFld-ContribAuthor ">Kathia Maria  Honorio</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning techniques applied to the drug design and discovery of new antivirals: a brief look over the past decade. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>11 </em>, 1-15. <a href="https://doi.org/10.1080/17460441.2021.1918098" title="DOI URL">https://doi.org/10.1080/17460441.2021.1918098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1918098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1918098%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DMachine%252Blearning%252Btechniques%252Bapplied%252Bto%252Bthe%252Bdrug%252Bdesign%252Band%252Bdiscovery%252Bof%252Bnew%252Bantivirals%25253A%252Ba%252Bbrief%252Blook%252Bover%252Bthe%252Bpast%252Bdecade%26aulast%3DSerafim%26aufirst%3DMateus%2BS%25C3%25A1%2BMagalh%25C3%25A3es%26date%3D2021%26date%3D2021%26volume%3D11%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhenxing  Wu</span>, <span class="hlFld-ContribAuthor ">Dejun  Jiang</span>, <span class="hlFld-ContribAuthor ">Chang-Yu  Hsieh</span>, <span class="hlFld-ContribAuthor ">Guangyong  Chen</span>, <span class="hlFld-ContribAuthor ">Ben  Liao</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Cao</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Hyperbolic relational graph convolution networks plus: a simple but highly efficient QSAR-modeling method. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2021,</strong> <em>194 </em><a href="https://doi.org/10.1093/bib/bbab112" title="DOI URL">https://doi.org/10.1093/bib/bbab112</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbab112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbab112%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DHyperbolic%252Brelational%252Bgraph%252Bconvolution%252Bnetworks%252Bplus%25253A%252Ba%252Bsimple%252Bbut%252Bhighly%252Befficient%252BQSAR-modeling%252Bmethod%26aulast%3DWu%26aufirst%3DZhenxing%26date%3D2021%26date%3D2021%26volume%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siphumelele  Majodina</span>, <span class="hlFld-ContribAuthor ">Lubabalo  Ndima</span>, <span class="hlFld-ContribAuthor ">Olufunso O.  Abosede</span>, <span class="hlFld-ContribAuthor ">Eric C.  Hosten</span>, <span class="hlFld-ContribAuthor ">Carolline M. A.  Lorentino</span>, <span class="hlFld-ContribAuthor ">HeloÃ­sa F.  Frota</span>, <span class="hlFld-ContribAuthor ">Leandro S.  Sangenito</span>, <span class="hlFld-ContribAuthor ">Marta H.  Branquinha</span>, <span class="hlFld-ContribAuthor ">AndrÃ© L. S.  Santos</span>, <span class="hlFld-ContribAuthor ">Adeniyi S.  Ogunlaja</span>. </span><span class="cited-content_cbyCitation_article-title">Physical stability enhancement and antimicrobial properties of a sodium ionic cocrystal with theophylline. </span><span class="cited-content_cbyCitation_journal-name">CrystEngComm</span><span> <strong>2021,</strong> <em>23 </em>
                                    (2)
                                     , 335-352. <a href="https://doi.org/10.1039/D0CE01387K" title="DOI URL">https://doi.org/10.1039/D0CE01387K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CE01387K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CE01387K%26sid%3Dliteratum%253Aachs%26jtitle%3DCrystEngComm%26atitle%3DPhysical%252Bstability%252Benhancement%252Band%252Bantimicrobial%252Bproperties%252Bof%252Ba%252Bsodium%252Bionic%252Bcocrystal%252Bwith%252Btheophylline%26aulast%3DMajodina%26aufirst%3DSiphumelele%26date%3D2021%26date%3D2021%26volume%3D23%26issue%3D2%26spage%3D335%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauv  Patel</span>, <span class="hlFld-ContribAuthor ">Tripti  Shukla</span>, <span class="hlFld-ContribAuthor ">Xiuzhen  Huang</span>, <span class="hlFld-ContribAuthor ">David W.  Ussery</span>, <span class="hlFld-ContribAuthor ">Shanzhi  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Machine Learning Methods in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (22)
                                     , 5277. <a href="https://doi.org/10.3390/molecules25225277" title="DOI URL">https://doi.org/10.3390/molecules25225277</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25225277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25225277%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMachine%252BLearning%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DPatel%26aufirst%3DLauv%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D22%26spage%3D5277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyang  Liu</span>, <span class="hlFld-ContribAuthor ">Kathryn E.  OâHarra</span>, <span class="hlFld-ContribAuthor ">Jason E.  Bara</span>, <span class="hlFld-ContribAuthor ">C. Heath  Turner</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular insight into the anion effect and free volume effect of CO
              2
              solubility in multivalent ionic liquids. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2020,</strong> <em>22 </em>
                                    (36)
                                     , 20618-20633. <a href="https://doi.org/10.1039/D0CP03424J" title="DOI URL">https://doi.org/10.1039/D0CP03424J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CP03424J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CP03424J%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DMolecular%252Binsight%252Binto%252Bthe%252Banion%252Beffect%252Band%252Bfree%252Bvolume%252Beffect%252Bof%252BCO%252B2%252Bsolubility%252Bin%252Bmultivalent%252Bionic%252Bliquids%26aulast%3DLiu%26aufirst%3DXiaoyang%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D36%26spage%3D20618%26epage%3D20633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overview of the methods for generating molecular ESP surfaces discussed in this paper, using tadalafil as a representative example of the relative quality of DFT-fp, DNN-fp, and AM1-BCC ESP surfaces, compared to a DFT ESP surface. The DNN-fp ESP surface approaches the quality of the DFT-fp ESP surface, which in turn is a very good approximation of the DFT ESP surface. The AM1-BCC ESP surface, although qualitatively similar to the other ESP surfaces, has significant differences to the DFT ESP surface, particularly around the benzodioxole ring of the tadalafil molecule. RMSE<sub>V</sub> values are in kcal/mol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. DFT-fp Î<i>q</i> values are plotted against the DNN-fp Î<i>q</i> values for (A) the training set and (B) the test set. Color of a data point on the correlation plots indicates the number of observations (see the corresponding color bar). The plots are generated using the hexbin function of the Python matplotlib library<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> with a grid size of 200.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Timings for DFT, DNN-fp, and AM1-BCC calculations as a function of molecular size. The average and standard deviation at each data point are calculated for 10 molecules randomly selected from our internal compound registry. The DFT timings correspond to the runtime for a single point calculation using 6-311G** basis sets. For AM1, the time required for generating a mol2 file with AM1-BCC charges using ANTECHAMBER<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> is reported, and for DNN-fp, the time required for generating a PQR file with charges on atoms and atomic features is reported (the average DNN-fp timing over all molecules is 0.34 s). All calculations were performed on a single core of an Intel Xeon processor E5 v4 (3.00 GHz).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. DNN-fp ESP values plotted against the corresponding DFT ESP values for Ï holes in chloro (orange), bromo (pink), and iodo (purple) substituents in the test set.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Correlations between DNN-fp ESP values and four experimental properties: (A) hydrogen bond basicity p<i>K</i><sub>BHX</sub> of pyridines (orange), acetophenones (blue), nitriles (gray), and ethers (yellow); (B) acidity constant p<i>K</i><sub>a</sub> for pyridines (blue) and carboxylic acids (orange); (C) fXa affinity p<i>K</i><sub>i</sub>, where the orange arrows indicate the prediction for a compound where the ligand phenyl ring in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A is replaced by a 2-pyridine ringl; (D) XIAP affinity pIC<sub>50</sub>, where only indoline compounds (blue points) were used in the regression and where the orange point represents the chloroazaindoline derivate (compound <b>17</b> in the study by Chessari et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. DFT-trained protein ESP surfaces (see <a class="ref internalNav" href="#sec4" aria-label="Methods">Methods</a> section) for (A) the fXa active site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VH6">2VH6</a>) and (B) the XIAP SMAC binding site (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C7A">5C7A</a>). For both targets a ligand is shown and the R-substituent location is indicated. Also, the areas where literature hypotheses are focused in terms of how affinity is affected by electrostatics are highlighted as solid white ellipsoids (<i>V</i><sub>h</sub> for fXa and <i>V</i><sub>p</sub> for XIAP in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The alternative/additional hypotheses tested in the current work are highlighted as dashed white ellipsoids (â¨<i>V</i>â© for fXa and <i>V</i><sub>h</sub> for XIAP in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. ESP surfaces generated by our DNN-fp model for (A) three of the fXa compounds reported by Young et al.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and (B) three compounds reported by Chessari et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Solid line circles/ellipsoids indicate the areas the authors proposed to be linked to the correlation between ESPs and affinity. Dashed line circles/ellipsoids represent areas of alternative/additional hypotheses we test here.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/medium/jm9b01129_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. DNN model architecture.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01129/20200820/images/large/jm9b01129_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01129&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60621" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60621" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scrocco, E.</span>; <span class="NLM_string-name">Tomasi, J.</span></span> <span> </span><span class="NLM_article-title">The Electrostatic Molecular Potential as a Tool for the Interpretation of Molecular Properties</span>. In  <i>New Concepts II</i>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">1973</span>; Vol.  <span class="NLM_volume">42</span>, pp  <span class="NLM_fpage">95</span>â <span class="NLM_lpage">170</span>m <span class="refDoi">Â DOI: 10.1007/3-540-06399-4_6</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1007%2F3-540-06399-4_6" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1973&pages=95-170&author=E.+Scrocco&author=J.+Tomasi&title=New+Concepts+II"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1007%2F3-540-06399-4_6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F3-540-06399-4_6%26sid%3Dliteratum%253Aachs%26aulast%3DScrocco%26aufirst%3DE.%26atitle%3DThe%2520Electrostatic%2520Molecular%2520Potential%2520as%2520a%2520Tool%2520for%2520the%2520Interpretation%2520of%2520Molecular%2520Properties%26btitle%3DNew%2520Concepts%2520II%26pub%3DSpringer%26date%3D1973%26volume%3D42%26spage%3D95%26epage%3D170%26doi%3D10.1007%2F3-540-06399-4_6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrolongo, C.</span></span> <span> </span><span class="NLM_article-title">Quantum Chemical Study of Isolated and Interacting Molecules with Biological Activity</span>. <i>Gazz. Chim. Ital.</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">445</span>â <span class="NLM_lpage">478</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=1978&pages=445-478&author=C.+Petrolongo&title=Quantum+Chemical+Study+of+Isolated+and+Interacting+Molecules+with+Biological+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPetrolongo%26aufirst%3DC.%26atitle%3DQuantum%2520Chemical%2520Study%2520of%2520Isolated%2520and%2520Interacting%2520Molecules%2520with%2520Biological%2520Activity%26jtitle%3DGazz.%2520Chim.%2520Ital.%26date%3D1978%26volume%3D108%26spage%3D445%26epage%3D478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonaccorsi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrocco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomasi, J.</span></span> <span> </span><span class="NLM_article-title">Molecular SCF Calculations for the Ground State of Some Three Membered Ring Molecules: (CH <sub>2</sub>) <sub>3</sub>, (CH <sub>2</sub>) <sub>2</sub> NH, (CH <sub>2</sub>) <sub>2</sub> NH <sub>2</sub> <sup>+</sup>, (CH <sub>2</sub>) <sub>2</sub> O, (CH <sub>2</sub>) <sub>2</sub> S, (CH) <sub>2</sub> CH <sub>2</sub>, and N <sub>2</sub> CH <sub>2</sub></span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">5270</span>â <span class="NLM_lpage">5284</span>, <span class="refDoi">Â DOI: 10.1063/1.1672775</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1063%2F1.1672775" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaE3cXktFeqtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1970&pages=5270-5284&issue=10&author=R.+Bonaccorsiauthor=E.+Scroccoauthor=J.+Tomasi&title=Molecular+SCF+Calculations+for+the+Ground+State+of+Some+Three+Membered+Ring+Molecules%3A+%28CH+2%29+3%2C+%28CH+2%29+2+NH%2C+%28CH+2%29+2+NH+2+%2B%2C+%28CH+2%29+2+O%2C+%28CH+2%29+2+S%2C+%28CH%29+2+CH+2%2C+and+N+2+CH+2&doi=10.1063%2F1.1672775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular SCF calculations for the ground state of some three-membered ring molecules:  (CH2)3, (CH2)2NH, (CH2)2NH+2, (CH2)2O, (CH2)2S, (CH)2CH2, and N2CH2</span></div><div class="casAuthors">Bonaccorsi, Rosanna; Scrocco, Eolo; Tomasi, Jacopo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5270-84</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">LCAO SCF MO calcns. with min. basis sets of Slater-type orbitals are performed for some 3-membered ring compds.  A reinterpretation of the resulting wavefunctions in terms of localized (exclusive) orbitals is presented and discussed.  The bent bonds in the rings are closely evidenced.  The electrostatic potentials produced in the neighboring space by the nuclear and electronic charge distributions are evaluated and employed to show the mol. sites more likely subject to electrophilic attacks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIXfyl5U7vA7Vg90H21EOLACvtfcHk0liuhah0JbK6Yw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXktFeqtbs%253D&md5=ed13295f8bcc60f091d78782cfddb415</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1063%2F1.1672775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1672775%26sid%3Dliteratum%253Aachs%26aulast%3DBonaccorsi%26aufirst%3DR.%26aulast%3DScrocco%26aufirst%3DE.%26aulast%3DTomasi%26aufirst%3DJ.%26atitle%3DMolecular%2520SCF%2520Calculations%2520for%2520the%2520Ground%2520State%2520of%2520Some%2520Three%2520Membered%2520Ring%2520Molecules%253A%2520%2528CH%25202%2529%25203%252C%2520%2528CH%25202%2529%25202%2520NH%252C%2520%2528CH%25202%2529%25202%2520NH%25202%2520%252B%252C%2520%2528CH%25202%2529%25202%2520O%252C%2520%2528CH%25202%2529%25202%2520S%252C%2520%2528CH%2529%25202%2520CH%25202%252C%2520and%2520N%25202%2520CH%25202%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1970%26volume%3D52%26issue%3D10%26spage%3D5270%26epage%3D5284%26doi%3D10.1063%2F1.1672775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langridge, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaney, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">Electrostatic Potential Molecular Surfaces</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>79</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">3754</span>â <span class="NLM_lpage">3758</span>, <span class="refDoi">Â DOI: 10.1073/pnas.79.12.3754</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1073%2Fpnas.79.12.3754" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=6285364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaL38XksV2hsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1982&pages=3754-3758&issue=12&author=P.+K.+Weinerauthor=R.+Langridgeauthor=J.+M.+Blaneyauthor=R.+Schaeferauthor=P.+A.+Kollman&title=Electrostatic+Potential+Molecular+Surfaces&doi=10.1073%2Fpnas.79.12.3754"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic potential molecular surfaces</span></div><div class="casAuthors">Weiner, Paul K.; Langridge, Robert; Blaney, Jeffrey M.; Schaefer, Ronald; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3754-8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Color-coded computer graphics representations of the electrostatic potentials of trypsin  [9002-07-7], trypsin inhibitor  [9035-81-8], prealbumin and its thyroxine  [51-48-9] complex, fragments of double-helical DNA, and a netropsin  [1438-30-8]-DNA complex illustrate the electrostatic and topog. complementarity in macromol.-ligand interactions.  This approach is powerful in revealing intermol. specificity and shows promise of having predictive value in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWAcsmv8tYL7Vg90H21EOLACvtfcHk0liuhah0JbK6Yw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XksV2hsrs%253D&md5=19afda4a3caf14d58344092eb9bbfd2f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.79.12.3754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.79.12.3754%26sid%3Dliteratum%253Aachs%26aulast%3DWeiner%26aufirst%3DP.%2BK.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DSchaefer%26aufirst%3DR.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DElectrostatic%2520Potential%2520Molecular%2520Surfaces%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1982%26volume%3D79%26issue%3D12%26spage%3D3754%26epage%3D3758%26doi%3D10.1073%2Fpnas.79.12.3754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackey, M. D.</span></span> <span> </span><span class="NLM_article-title">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">3036</span>â <span class="NLM_lpage">3050</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b01925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3036-3050&issue=6&author=M.+R.+Bauerauthor=M.+D.+Mackey&title=Electrostatic+Complementarity+as+a+Fast+and+Effective+Tool+to+Optimize+Binding+and+Selectivity+of+Protein-Ligand+Complexes&doi=10.1021%2Facs.jmedchem.8b01925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Electrostatic Complementarity as a Fast and Effective Tool to Optimize Binding and Selectivity of Protein-Ligand Complexes</span></div><div class="casAuthors">Bauer, Matthias R.; Mackey, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3036-3050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Electrostatic interactions between small mols. and their resp. receptors are essential for mol. recognition and are also key contributors to the binding free energy.  Assessing the electrostatic match of protein-ligand complexes therefore provides important insights into why ligands bind and what can be changed to improve binding.  Ideally, the ligand and protein electrostatic potentials at the protein-ligand interaction interface should maximize their complementarity while minimizing desolvation penalties.  In this work, we present a fast and efficient tool to calc. and visualize the electrostatic complementarity (EC) of protein-ligand complexes.  We compiled benchmark sets demonstrating electrostatically driven structure-activity relationships (SAR) from literature data, including kinase, protein-protein interaction, and GPCR targets, and used these to demonstrate that the EC method can visualize, rationalize, and predict electrostatically driven ligand affinity changes and help to predict compd. selectivity.  The methodol. presented here for the anal. of EC is a powerful and versatile tool for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3pdUzJ9T0trVg90H21EOLACvtfcHk0lj4Y-1ZQZ_jeg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1ChsL8%253D&md5=a1abb5abe7fa8d73bee264bc94fd2da1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01925%26sid%3Dliteratum%253Aachs%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DMackey%26aufirst%3DM.%2BD.%26atitle%3DElectrostatic%2520Complementarity%2520as%2520a%2520Fast%2520and%2520Effective%2520Tool%2520to%2520Optimize%2520Binding%2520and%2520Selectivity%2520of%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D6%26spage%3D3036%26epage%3D3050%26doi%3D10.1021%2Facs.jmedchem.8b01925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilcken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joerger, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckler, F. M.</span></span> <span> </span><span class="NLM_article-title">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1363</span>â <span class="NLM_lpage">1388</span>, <span class="refDoi">Â DOI: 10.1021/jm3012068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3012068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1363-1388&issue=4&author=R.+Wilckenauthor=M.+O.+Zimmermannauthor=A.+Langeauthor=A.+C.+Joergerauthor=F.+M.+Boeckler&title=Principles+and+Applications+of+Halogen+Bonding+in+Medicinal+Chemistry+and+Chemical+Biology&doi=10.1021%2Fjm3012068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Wilcken, Rainer; Zimmermann, Markus O.; Lange, Andreas; Joerger, Andreas C.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1363-1388</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Halogen bonding has been known in material science for decades, but until recently, halogen bonds in protein-ligand interactions were largely the result of serendipitous discovery rather than rational design.  In this Perspective, we provide insights into the phenomenon of halogen bonding, with special focus on its role in drug discovery.  We summarize the theor. background defining its strength and directionality, provide a systematic anal. of its occurrence and interaction geometries in protein-ligand complexes, and give recent examples where halogen bonding has been successfully harnessed for lead identification and optimization.  In light of these data, we discuss the potential and limitations of exploiting halogen bonds for mol. recognition and rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjholHfNZ9kbVg90H21EOLACvtfcHk0lj4Y-1ZQZ_jeg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ&md5=81ba024e4427fac3e7af0a4cd330909c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm3012068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3012068%26sid%3Dliteratum%253Aachs%26aulast%3DWilcken%26aufirst%3DR.%26aulast%3DZimmermann%26aufirst%3DM.%2BO.%26aulast%3DLange%26aufirst%3DA.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPrinciples%2520and%2520Applications%2520of%2520Halogen%2520Bonding%2520in%2520Medicinal%2520Chemistry%2520and%2520Chemical%2520Biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26issue%3D4%26spage%3D1363%26epage%3D1388%26doi%3D10.1021%2Fjm3012068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, D. A.</span></span> <span> </span><span class="NLM_article-title">The Cation-Ï Interaction</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">885</span>â <span class="NLM_lpage">893</span>, <span class="refDoi">Â DOI: 10.1021/ar300265y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar300265y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSqtrjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2013&pages=885-893&issue=4&author=D.+A.+Dougherty&title=The+Cation-%CF%80+Interaction&doi=10.1021%2Far300265y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Cation-Ï Interaction</span></div><div class="casAuthors">Dougherty, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">885-893</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The chem. community now recognizes the cation-Ï interaction as a major force for mol. recognition, joining the hydrophobic effect, the hydrogen bond, and the ion pair in detg. macromol. structure and drug-receptor interactions.  This Account provides the author's perspective on the intellectual origins and fundamental nature of the cation-Ï interaction.  Early studies on cyclophanes established that water-sol., cationic mols. would forego aq. solvation to enter a hydrophobic cavity if that cavity was lined with Ï systems.  Important gas phase studies established the fundamental nature of the cation-Ï interaction.  The strength of the cation-Ï interaction (Li+ binds to benzene with 38 kcal/mol of binding energy; NH4+ with 19 kcal/mol) distinguishes it from the weaker polar-Ï interactions obsd. in the benzene dimer or water-benzene complexes.  In addn. to the substantial intrinsic strength of the cation-Ï interaction in gas phase studies, the cation-Ï interaction remains energetically significant in aq. media and under biol. conditions.  Many studies have shown that cation-Ï interactions can enhance binding energies by 2-5 kcal/mol, making them competitive with hydrogen bonds and ion pairs in drug-receptor and protein-protein interactions.  As with other noncovalent interactions involving arom. systems, the cation-Ï interaction includes a substantial electrostatic component.  The six (four) CÎ´--HÎ´+ bond dipoles of a mol. like benzene (ethylene) combine to produce a region of neg. electrostatic potential on the face of the Ï system.  Simple electrostatics facilitate a natural attraction of cations to the surface.  The trend for (gas phase) binding energies is Li+ > Na+ > K+ > Rb+: as the ion gets larger the charge is dispersed over a larger sphere and binding interactions weaken, a classical electrostatic effect.  On other hand, polarizability does not define these interactions.  Cyclohexane is more polarizable than benzene but a decidedly poorer cation binder.  Many studies have documented cation-Ï interactions in protein structures, where lysine or arginine side chains interact with phenylalanine, tyrosine, or tryptophan.  In addn., countless studies have established the importance of the cation-Ï interaction in a range of biol. processes.  Our work has focused on mol. neurobiol., and we have shown that neurotransmitters generally use a cation-Ï interaction to bind to their receptors.  We have also shown that many drug-receptor interactions involve cation-Ï interactions.  A cation-Ï interaction plays a crit. role in the binding of nicotine to ACh receptors in the brain, an esp. significant case.  Other researchers have established important cation-Ï interactions in the recognition of the "histone code," in terpene biosynthesis, in chem. catalysis, and in many other systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgQQZnCXWWcrVg90H21EOLACvtfcHk0lj4Y-1ZQZ_jeg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSqtrjL&md5=a4f35bd254c0ecbe695e91f40518e833</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Far300265y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far300265y%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DD.%2BA.%26atitle%3DThe%2520Cation-%25CF%2580%2520Interaction%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2013%26volume%3D46%26issue%3D4%26spage%3D885%26epage%3D893%26doi%3D10.1021%2Far300265y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederich, F.</span></span> <span> </span><span class="NLM_article-title">Interactions with Aromatic Rings in Chemical and Biological Recognition</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1210</span>â <span class="NLM_lpage">1250</span>, <span class="refDoi">Â DOI: 10.1002/anie.200390319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1002%2Fanie.200390319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=1210-1250&issue=11&author=E.+A.+Meyerauthor=R.+K.+Castellanoauthor=F.+Diederich&title=Interactions+with+Aromatic+Rings+in+Chemical+and+Biological+Recognition&doi=10.1002%2Fanie.200390319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions with aromatic rings in chemical and biological recognition</span></div><div class="casAuthors">Meyer, Emmanuel A.; Castellano, Ronald K.; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1210-1250</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Intermol. interactions involving arom. rings are key processes in both chem. and biol. recognition.  Their understanding is essential for rational drug design and lead optimization in medicinal chem.  Different approaches-biol. studies, mol. recognition studies with artificial receptors, crystallog. database mining, gas-phase studies, and theor. calcns. - are pursued to generate a profound understanding of the structural and energetic parameters of individual recognition modes involving arom. rings.  This review attempts to combine and summarize the knowledge gained from these investigations.  The review focuses mainly on examples with biol. relevance since one of its aims it to enhance the knowledge of mol. recognition forces that is essential for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHUbAhGHcGtLVg90H21EOLACvtfcHk0lj4Y-1ZQZ_jeg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D&md5=e91e6a8d115d56229e1660697a286b04</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.200390319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200390319%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DE.%2BA.%26aulast%3DCastellano%26aufirst%3DR.%2BK.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DInteractions%2520with%2520Aromatic%2520Rings%2520in%2520Chemical%2520and%2520Biological%2520Recognition%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26issue%3D11%26spage%3D1210%26epage%3D1250%26doi%3D10.1002%2Fanie.200390319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bleiziffer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Machine Learning of Partial Charges Derived from High-Quality Quantum-Mechanical Calculations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">579</span>â <span class="NLM_lpage">590</span>, <span class="refDoi">Â DOI: 10.1021/acs.jcim.7b00663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtVemtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=579-590&issue=3&author=P.+Bleizifferauthor=K.+Schallerauthor=S.+Riniker&title=Machine+Learning+of+Partial+Charges+Derived+from+High-Quality+Quantum-Mechanical+Calculations&doi=10.1021%2Facs.jcim.7b00663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Machine Learning of Partial Charges Derived from High-Quality Quantum-Mechanical Calculations</span></div><div class="casAuthors">Bleiziffer, Patrick; Schaller, Kay; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">579-590</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Parametrization of small org. mols. for classical mol. dynamics simulations is not trivial.  The vastness of the chem. space makes approaches using building blocks challenging.  The most common approach is therefore an individual parametrization of each compd. by deriving partial charges from semiempirical or ab initio calcns. and inheriting the bonded and van der Waals (Lennard-Jones) parameters from a (bio)mol. force field.  The quality of the partial charges generated in this fashion depends on the level of the quantum-chem. calcn. as well as on the extn. procedure used.  Here, we present a machine learning (ML) based approach for predicting partial charges extd. from d. functional theory (DFT) electron densities.  The training set was chosen with the goal to provide a broad coverage of the known chem. space of druglike mols.  In addn. to the speed of the approach, the partial charges predicted by ML are not dependent on the three-dimensional conformation in contrast to the ones obtained by fitting to the electrostatic potential (ESP).  To assess the quality and compatibility with std. force fields, we performed benchmark calcns. for the free energy of hydration and liq. properties such as d. and heat of vaporization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmkC7Cjqxr9LVg90H21EOLACvtfcHk0lg_wW6O86FyHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtVemtLo%253D&md5=6b70f157ee8f3f9735d2f624de0c3155</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00663%26sid%3Dliteratum%253Aachs%26aulast%3DBleiziffer%26aufirst%3DP.%26aulast%3DSchaller%26aufirst%3DK.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DMachine%2520Learning%2520of%2520Partial%2520Charges%2520Derived%2520from%2520High-Quality%2520Quantum-Mechanical%2520Calculations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26issue%3D3%26spage%3D579%26epage%3D590%26doi%3D10.1021%2Facs.jcim.7b00663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heidrich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Exner, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckler, F. M.</span></span> <span> </span><span class="NLM_article-title">Predicting the Magnitude of Ï-Holes Using VmaxPred, a Fast and Efficient Tool Supporting the Application of Halogen Bonds in Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">636</span>â <span class="NLM_lpage">643</span>, <span class="refDoi">Â DOI: 10.1021/acs.jcim.8b00622</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00622" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFylurnE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=636-643&issue=2&author=J.+Heidrichauthor=T.+E.+Exnerauthor=F.+M.+Boeckler&title=Predicting+the+Magnitude+of+%CF%83-Holes+Using+VmaxPred%2C+a+Fast+and+Efficient+Tool+Supporting+the+Application+of+Halogen+Bonds+in+Drug+Discovery&doi=10.1021%2Facs.jcim.8b00622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting the Magnitude of Ï-Holes Using VmaxPred, a Fast and Efficient Tool Supporting the Application of Halogen Bonds in Drug Discovery</span></div><div class="casAuthors">Heidrich, Johannes; Exner, Thomas E.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">636-643</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Halogen bonding (XB) as a modern mol. interaction has received increasing attention, not only in material sciences, but also in biol. systems and drug discovery.  Thus, there is a growing demand for fast, efficient, and easily applicable tailor-made tools supporting the use of halogen bonds in mol. design and medicinal chem.  The potential strength of a halogen bond is dependent on several properties of the Ï-hole donor, e.g. a (hetero)aryl halide, and the Ï-hole acceptor, a nucleophile with n- or Ï-electron d.  Besides the influence of the interaction geometry and the type of acceptor, significant tuning effects on the magnitude of the Ï-hole can be obsd., caused by different (hetero)arom. scaffolds and their substitution patterns.  The most pos. electrostatic potential on the isodensity surface (Vmax), representing the Ï-hole, has been widely used as the std. descriptor for the magnitude of the Ï-hole and the strength of the halogen bond.  Calcn. of Vmax using quantum mech. methods at a reasonable level of theory is time-consuming and, thus, not applicable for larger nos. of compds. in drug discovery projects.  Herein, the authors present a tool for the prediction of this descriptor based on a machine-learned model with a speed up of five to six orders of magnitude compared to MP2 quantum mech. calcns.  According to the test set the squared correlation coeff. is greater than 0.94.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGionFTI8jBbVg90H21EOLACvtfcHk0lg_wW6O86FyHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFylurnE&md5=a1c93e84070b8c6868e9971f6ec39c4a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00622%26sid%3Dliteratum%253Aachs%26aulast%3DHeidrich%26aufirst%3DJ.%26aulast%3DExner%26aufirst%3DT.%2BE.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPredicting%2520the%2520Magnitude%2520of%2520%25CF%2583-Holes%2520Using%2520VmaxPred%252C%2520a%2520Fast%2520and%2520Efficient%2520Tool%2520Supporting%2520the%2520Application%2520of%2520Halogen%2520Bonds%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26issue%3D2%26spage%3D636%26epage%3D643%26doi%3D10.1021%2Facs.jcim.8b00622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, Efficient Generation of High-Quality Atomic Charges. AM1-BCC Model: II. Parameterization and Validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1623</span>â <span class="NLM_lpage">1641</span>, <span class="refDoi">Â DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&issue=16&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+Efficient+Generation+of+High-Quality+Atomic+Charges.+AM1-BCC+Model%3A+II.+Parameterization+and+Validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lg_wW6O86FyHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520Efficient%2520Generation%2520of%2520High-Quality%2520Atomic%2520Charges.%2520AM1-BCC%2520Model%253A%2520II.%2520Parameterization%2520and%2520Validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26issue%3D16%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mardirossian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head-Gordon, M.</span></span> <span> </span><span class="NLM_article-title">Thirty Years of Density Functional Theory in Computational Chemistry: An Overview and Extensive Assessment of 200 Density Functionals</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>115</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">2315</span>â <span class="NLM_lpage">2372</span>, <span class="refDoi">Â DOI: 10.1080/00268976.2017.1333644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1080%2F00268976.2017.1333644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCltb3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2017&pages=2315-2372&issue=19&author=N.+Mardirossianauthor=M.+Head-Gordon&title=Thirty+Years+of+Density+Functional+Theory+in+Computational+Chemistry%3A+An+Overview+and+Extensive+Assessment+of+200+Density+Functionals&doi=10.1080%2F00268976.2017.1333644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Thirty years of density functional theory in computational chemistry: an overview and extensive assessment of 200 density functionals</span></div><div class="casAuthors">Mardirossian, Narbe; Head-Gordon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2315-2372</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">In the past 30 years, Kohn-Sham d. functional theory has emerged as the most popular electronic structure method in computational chem.  To assess the ever-increasing no. of approx. exchange-correlation functionals, this review benchmarks a total of 200 d. functionals on a mol. database (MGCDB84) of nearly 5000 data points.  The database employed, provided as Supplemental Data, is comprised of 84 data-sets and contains non-covalent interactions, isomerisation energies, thermochem., and barrier heights.  In addn., the evolution of non-empirical and semi-empirical d. functional design is reviewed, and guidelines are provided for the proper and effective use of d. functionals.  The most promising functional considered is ÏB97M-V, a range-sepd. hybrid meta-GGA with VV10 nonlocal correlation, designed using a combinatorial approach.  From the local GGAs, B97-D3, revPBE-D3, and BLYP-D3 are recommended, while from the local meta-GGAs, B97M-rV is the leading choice, followed by MS1-D3 and M06-L-D3.  The best hybrid GGAs are ÏB97X-V, ÏB97X-D3, and ÏB97X-D, while useful hybrid meta-GGAs (besides ÏB97M-V) include ÏM05-D, M06-2X-D3, and MN15.  Ultimately, today's state-of-the-art functionals are close to achieving the level of accuracy desired for a broad range of chem. applications, and the principal remaining limitations are assocd. with systems that exhibit significant self-interaction/delocalisation errors and/or strong correlation effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNNQ1y9c79UrVg90H21EOLACvtfcHk0lhs7YfRTJf81w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCltb3O&md5=ba27d707ee3f5fcdd949644d3d2cbd5e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1080%2F00268976.2017.1333644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268976.2017.1333644%26sid%3Dliteratum%253Aachs%26aulast%3DMardirossian%26aufirst%3DN.%26aulast%3DHead-Gordon%26aufirst%3DM.%26atitle%3DThirty%2520Years%2520of%2520Density%2520Functional%2520Theory%2520in%2520Computational%2520Chemistry%253A%2520An%2520Overview%2520and%2520Extensive%2520Assessment%2520of%2520200%2520Density%2520Functionals%26jtitle%3DMol.%2520Phys.%26date%3D2017%26volume%3D115%26issue%3D19%26spage%3D2315%26epage%3D2372%26doi%3D10.1080%2F00268976.2017.1333644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dewar, M. J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoebisch, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Healy, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. J. P.</span></span> <span> </span><span class="NLM_article-title">Development and Use of Quantum Mechanical Molecular Models. 76. AM1: A New General Purpose Quantum Mechanical Molecular Model</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>107</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">3902</span>â <span class="NLM_lpage">3909</span>, <span class="refDoi">Â DOI: 10.1021/ja00299a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00299a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaL2MXktFWlsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=1985&pages=3902-3909&issue=13&author=M.+J.+S.+Dewarauthor=E.+G.+Zoebischauthor=E.+F.+Healyauthor=J.+J.+P.+Stewart&title=Development+and+Use+of+Quantum+Mechanical+Molecular+Models.+76.+AM1%3A+A+New+General+Purpose+Quantum+Mechanical+Molecular+Model&doi=10.1021%2Fja00299a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Development and use of quantum mechanical molecular models.  76.  AM1:  a new general purpose quantum mechanical molecular model</span></div><div class="casAuthors">Dewar, Michael J. S.; Zoebisch, Eve G.; Healy, Eamonn F.; Stewart, James J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3902-9</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A new parametric quantum mech. mol. model, AM1 (Austin Model 1), based on the NDDO approxn., is described.  In it the major weaknesses of MNDO, in particular failure to reproduce H bonds, are overcome without any increase in computing time.  Results for 167 mols. are reported.  Parameters are currently available for C, H, O, and N.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB-oi6B_HZY7Vg90H21EOLACvtfcHk0lhs7YfRTJf81w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktFWlsLk%253D&md5=5733ca359609184eb3d58fc52c73d2de</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja00299a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00299a024%26sid%3Dliteratum%253Aachs%26aulast%3DDewar%26aufirst%3DM.%2BJ.%2BS.%26aulast%3DZoebisch%26aufirst%3DE.%2BG.%26aulast%3DHealy%26aufirst%3DE.%2BF.%26aulast%3DStewart%26aufirst%3DJ.%2BJ.%2BP.%26atitle%3DDevelopment%2520and%2520Use%2520of%2520Quantum%2520Mechanical%2520Molecular%2520Models.%252076.%2520AM1%253A%2520A%2520New%2520General%2520Purpose%2520Quantum%2520Mechanical%2520Molecular%2520Model%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1985%26volume%3D107%26issue%3D13%26spage%3D3902%26epage%3D3909%26doi%3D10.1021%2Fja00299a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, J. D.</span></span> <span> </span><span class="NLM_article-title">Matplotlib: A 2D Graphics Environment</span>. <i>Comput. Sci. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">90</span>â <span class="NLM_lpage">95</span>, <span class="refDoi">Â DOI: 10.1109/MCSE.2007.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1109%2FMCSE.2007.55" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=90-95&issue=3&author=J.+D.+Hunter&title=Matplotlib%3A+A+2D+Graphics+Environment&doi=10.1109%2FMCSE.2007.55"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1109%2FMCSE.2007.55&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FMCSE.2007.55%26sid%3Dliteratum%253Aachs%26aulast%3DHunter%26aufirst%3DJ.%2BD.%26atitle%3DMatplotlib%253A%2520A%25202D%2520Graphics%2520Environment%26jtitle%3DComput.%2520Sci.%2520Eng.%26date%3D2007%26volume%3D9%26issue%3D3%26spage%3D90%26epage%3D95%26doi%3D10.1109%2FMCSE.2007.55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">247</span>â <span class="NLM_lpage">260</span>, <span class="refDoi">Â DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lhs7YfRTJf81w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laurence, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Questel, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renault, E.</span></span> <span> </span><span class="NLM_article-title">The PK(BHX) Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">4073</span>â <span class="NLM_lpage">4086</span>, <span class="refDoi">Â DOI: 10.1021/jm801331y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801331y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4073-4086&issue=14&author=C.+Laurenceauthor=K.+A.+Brameldauthor=J.+Gratonauthor=J.-Y.+Le+Questelauthor=E.+Renault&title=The+PK%28BHX%29+Database%3A+Toward+a+Better+Understanding+of+Hydrogen-Bond+Basicity+for+Medicinal+Chemists&doi=10.1021%2Fjm801331y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The pKBHX Database: Toward a Better Understanding of Hydrogen-Bond Basicity for Medicinal Chemists</span></div><div class="casAuthors">Laurence, Christian; Brameld, Ken A.; Graton, Jerome; Le Questel, Jean-Yves; Renault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4073-4086</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The pKBHX database is presented, which corresponds to three main fields of data: hydrogen bond donor identification, thermodn. data and spectroscopic data.  The pKBHX scale of hydrogen donor basicity differs considerable from the pKBH+ scale of proton transfer basicity.  The hydrogen bond basicities of functional groups relevant to medicinal chem. and drug design is reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql9OhmMu2VILVg90H21EOLACvtfcHk0lg6Dg9QxzSV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsFKns7w%253D&md5=9ce4bea3e810752baba6ea5784b55ea0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm801331y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801331y%26sid%3Dliteratum%253Aachs%26aulast%3DLaurence%26aufirst%3DC.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DGraton%26aufirst%3DJ.%26aulast%3DLe%2BQuestel%26aufirst%3DJ.-Y.%26aulast%3DRenault%26aufirst%3DE.%26atitle%3DThe%2520PK%2528BHX%2529%2520Database%253A%2520Toward%2520a%2520Better%2520Understanding%2520of%2520Hydrogen-Bond%2520Basicity%2520for%2520Medicinal%2520Chemists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26issue%3D14%26spage%3D4073%26epage%3D4086%26doi%3D10.1021%2Fjm801331y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Politzer, P.</span></span> <span> </span><span class="NLM_article-title">Correlations between the Solvent Hydrogen-Bond-Donating Parameter. Alpha. and the Calculated Molecular Surface Electrostatic Potential</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6715</span>â <span class="NLM_lpage">6717</span>, <span class="refDoi">Â DOI: 10.1021/jo00023a045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00023a045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaK3MXmtlWmu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1991&pages=6715-6717&issue=23&author=J.+S.+Murrayauthor=P.+Politzer&title=Correlations+between+the+Solvent+Hydrogen-Bond-Donating+Parameter.+Alpha.+and+the+Calculated+Molecular+Surface+Electrostatic+Potential&doi=10.1021%2Fjo00023a045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Correlations between the solvent hydrogen-bond-donating parameter Î± and the calculated molecular surface electrostatic potential</span></div><div class="casAuthors">Murray, Jane S.; Politzer, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6715-17</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">We use ab initio self-consistent-field calcns. to show that good linear correlations exist between the solvatochromic hydrogen-bond-donating parameters Î± and the most pos. surface electrostatic potentials, VS,max, of two groups of hydrogen bond donors, taken sep.; the correlation coeffs. are 0.96 and 0.98, resp.  These relations confirm that the electrostatic potential in the space around a gas phase mol. is a key (but not sole) factor in detg. its ability to donate a proton in a solvent-to-solute hydrogen bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2RQv6dZFXCLVg90H21EOLACvtfcHk0lg6Dg9QxzSV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmtlWmu7g%253D&md5=2f72f1aff6e3757592d4cc9526f90976</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjo00023a045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00023a045%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BS.%26aulast%3DPolitzer%26aufirst%3DP.%26atitle%3DCorrelations%2520between%2520the%2520Solvent%2520Hydrogen-Bond-Donating%2520Parameter.%2520Alpha.%2520and%2520the%2520Calculated%2520Molecular%2520Surface%2520Electrostatic%2520Potential%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1991%26volume%3D56%26issue%3D23%26spage%3D6715%26epage%3D6717%26doi%3D10.1021%2Fjo00023a045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span> <span> </span><span class="NLM_article-title">Prediction of Hydrogen Bond Basicity from Computed Molecular Electrostatic Properties: Implications for Comparative Molecular Field Analysis</span>. <i>J. Chem. Soc., Perkin Trans. 2</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">199</span>â <span class="NLM_lpage">202</span>, <span class="refDoi">Â DOI: 10.1039/p29940000199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1039%2Fp29940000199" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1994&pages=199-202&issue=2&author=P.+W.+Kenny&title=Prediction+of+Hydrogen+Bond+Basicity+from+Computed+Molecular+Electrostatic+Properties%3A+Implications+for+Comparative+Molecular+Field+Analysis&doi=10.1039%2Fp29940000199"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2Fp29940000199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fp29940000199%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26atitle%3DPrediction%2520of%2520Hydrogen%2520Bond%2520Basicity%2520from%2520Computed%2520Molecular%2520Electrostatic%2520Properties%253A%2520Implications%2520for%2520Comparative%2520Molecular%2520Field%2520Analysis%26jtitle%3DJ.%2520Chem.%2520Soc.%252C%2520Perkin%2520Trans.%25202%26date%3D1994%26volume%3D2%26issue%3D2%26spage%3D199%26epage%3D202%26doi%3D10.1039%2Fp29940000199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kenny, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montanari, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prokopczyk, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, J. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, G. R.</span></span> <span> </span><span class="NLM_article-title">Hydrogen Bond Basicity Prediction for Medicinal Chemistry Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4278</span>â <span class="NLM_lpage">4288</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b01946</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01946" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyjsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4278-4288&issue=9&author=P.+W.+Kennyauthor=C.+A.+Montanariauthor=I.+M.+Prokopczykauthor=J.+F.+R.+Ribeiroauthor=G.+R.+Sartori&title=Hydrogen+Bond+Basicity+Prediction+for+Medicinal+Chemistry+Design&doi=10.1021%2Facs.jmedchem.5b01946"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen Bond Basicity Prediction for Medicinal Chemistry Design</span></div><div class="casAuthors">Kenny, Peter W.; Montanari, Carlos A.; Prokopczyk, Igor M.; Ribeiro, Jean F. R.; Sartori, Geraldo Rodrigues</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4278-4288</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen bonding is discussed in the context of medicinal chem. design.  Minimized mol. electrostatic potential (Vmin) is shown to be an effective predictor of hydrogen bond basicity (pKBHX), and predictive models are presented for a no. of hydrogen bond acceptor types relevant to medicinal chem.  The problems posed by the presence of nonequivalent hydrogen bond acceptor sites in mol. structures are addressed by using nonlinear regression to fit measured pKBHX to calcd. Vmin.  Predictions are made for hydrogen bond basicity of fluorine in situations where relevant exptl. measurements are not available.  It is shown how predicted pKBHX can be used to provide insight into the nature of bioisosterism and to profile heterocycles.  Examples of pKBHX prediction for mol. structures with multiple, nonequivalent hydrogen bond acceptors are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwaulOTEJQ_rVg90H21EOLACvtfcHk0lg6Dg9QxzSV6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyjsLo%253D&md5=733e16424c7bb1c254123029ab9fd6f8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01946%26sid%3Dliteratum%253Aachs%26aulast%3DKenny%26aufirst%3DP.%2BW.%26aulast%3DMontanari%26aufirst%3DC.%2BA.%26aulast%3DProkopczyk%26aufirst%3DI.%2BM.%26aulast%3DRibeiro%26aufirst%3DJ.%2BF.%2BR.%26aulast%3DSartori%26aufirst%3DG.%2BR.%26atitle%3DHydrogen%2520Bond%2520Basicity%2520Prediction%2520for%2520Medicinal%2520Chemistry%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4278%26epage%3D4288%26doi%3D10.1021%2Facs.jmedchem.5b01946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollingsworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seybold, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span> <span> </span><span class="NLM_article-title">Relationships between Aqueous Acidities and Computed Surface-Electrostatic Potentials and Local Ionization Energies of Substituted Phenols and Benzoic Acids</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">235</span>â <span class="NLM_lpage">239</span>, <span class="refDoi">Â DOI: 10.1007/s00894-004-0185-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1007%2Fs00894-004-0185-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptValtL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=235-239&issue=4&author=Y.+Maauthor=K.+Grossauthor=C.+Hollingsworthauthor=P.+Seyboldauthor=J.+Murray&title=Relationships+between+Aqueous+Acidities+and+Computed+Surface-Electrostatic+Potentials+and+Local+Ionization+Energies+of+Substituted+Phenols+and+Benzoic+Acids&doi=10.1007%2Fs00894-004-0185-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Relationships between aqueous acidities and computed surface-electrostatic potentials and local ionization energies of substituted phenols and benzoic acids</span></div><div class="casAuthors">Ma, Yuguang; Gross, Kevin C.; Hollingsworth, Chad A.; Seybold, Paul G.; Murray, Jane S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Modeling</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-239</span>CODEN:
                <span class="NLM_cas:coden">JMMOFK</span>;
        ISSN:<span class="NLM_cas:issn">0948-5023</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Electrostatic potentials and av. local ionization energies on the mol. surfaces of 19 phenols, 17 benzoic acids and their resp. conjugate bases were computed at the HF/STO-5G(d)//B3LYP/6-311G(d,p) level.  Good correlations were found between pKas and the VS,max values of the neutral acids and the VS,min and â¾IS,min of the conjugate bases for both sets of mols.  VS,max is the most pos. value of the electrostatic potential on the mol. surface and is an indicator of the ease with which the phenols and benzoic acids lose their acidic hydrogens.  VS,min and â¾IS,min are the min. values of the electrostatic potential and the local ionization energy computed on the mol. surface; the VS,min and â¾IS,min of the conjugate bases of the phenoxides and benzoates are indicative, resp., of the tendencies of electrophiles to approach (VS,min) and to react with (â¾IS,min) to reform the original acids.  The correlations obsd. between the computed mol. surface quantities, taken as single parameters, and the exptl. pKa values ranged from R = 0.938 to R = 0.970 for the two classes of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoko4xUSapR6bVg90H21EOLACvtfcHk0liGw-jk8SU_zw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptValtL0%253D&md5=df506ded7abd1922d247c0f1d2e50532</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00894-004-0185-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-004-0185-x%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DGross%26aufirst%3DK.%26aulast%3DHollingsworth%26aufirst%3DC.%26aulast%3DSeybold%26aufirst%3DP.%26aulast%3DMurray%26aufirst%3DJ.%26atitle%3DRelationships%2520between%2520Aqueous%2520Acidities%2520and%2520Computed%2520Surface-Electrostatic%2520Potentials%2520and%2520Local%2520Ionization%2520Energies%2520of%2520Substituted%2520Phenols%2520and%2520Benzoic%2520Acids%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2004%26volume%3D10%26issue%3D4%26spage%3D235%26epage%3D239%26doi%3D10.1007%2Fs00894-004-0185-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caballero-GarcÃ­a, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MondragÃ³n-SolÃ³rzano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres-Cadena, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DÃ­az-GarcÃ­a, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval-Lira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barroso-Flores, J.</span></span> <span> </span><span class="NLM_article-title">Calculation of VS, Max and Its Use as a Descriptor for the Theoretical Calculation of PKa Values for Carboxylic Acids</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">79</span>, <span class="refDoi">Â DOI: 10.3390/molecules24010079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.3390%2Fmolecules24010079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlGjtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=79&issue=1&author=G.+Caballero-Garc%C3%ADaauthor=G.+Mondrag%C3%B3n-Sol%C3%B3rzanoauthor=R.+Torres-Cadenaauthor=M.+D%C3%ADaz-Garc%C3%ADaauthor=J.+Sandoval-Liraauthor=J.+Barroso-Flores&title=Calculation+of+VS%2C+Max+and+Its+Use+as+a+Descriptor+for+the+Theoretical+Calculation+of+PKa+Values+for+Carboxylic+Acids&doi=10.3390%2Fmolecules24010079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of VS,max and Its use as a descriptor for the theoretical calculation of pKa values for carboxylic acids</span></div><div class="casAuthors">Caballero-Garcia, Guillermo; Mondragon-Solorzano, Gustavo; Torres-Cadena, Raul; Diaz-Garcia, Marco; Sandoval-Lira, Jacinto; Barroso-Flores, Joaquin</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79/1-79/15</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The theor. calcn. of pKa values for Bronsted acids is a challenging task that involves sophisticated and time-consuming methods.  Therefore, heuristic approaches are efficient and appealing methodologies to approx. these values.  Herein, we used the max. surface electrostatic potential (VS,max) on the acidic hydrogen atoms of carboxylic acids to describe the H-bond interaction with water (the same descriptor that is used to characterize Ï-bonded complexes) and correlate the results with exptl. pKa values to obtain a predictive model for other carboxylic acids.  We benchmarked six different methods, all including an implicit solvation model (water): Five d. functionals and the Moller-Plesset second order perturbation theory in combination with six different basis sets for a total of thirty-six levels of theory.  The ÏB97X-D/cc-pVDZ level of theory stood out as the best one for consistently reproducing the reported pKa values, with a predictive power of 98% correlation in a test set of ten other carboxylic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx0AOGRu5corVg90H21EOLACvtfcHk0liGw-jk8SU_zw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlGjtLc%253D&md5=28b6ffb3baefa304cfc062bb61cfb6e9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.3390%2Fmolecules24010079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules24010079%26sid%3Dliteratum%253Aachs%26aulast%3DCaballero-Garc%25C3%25ADa%26aufirst%3DG.%26aulast%3DMondrag%25C3%25B3n-Sol%25C3%25B3rzano%26aufirst%3DG.%26aulast%3DTorres-Cadena%26aufirst%3DR.%26aulast%3DD%25C3%25ADaz-Garc%25C3%25ADa%26aufirst%3DM.%26aulast%3DSandoval-Lira%26aufirst%3DJ.%26aulast%3DBarroso-Flores%26aufirst%3DJ.%26atitle%3DCalculation%2520of%2520VS%252C%2520Max%2520and%2520Its%2520Use%2520as%2520a%2520Descriptor%2520for%2520the%2520Theoretical%2520Calculation%2520of%2520PKa%2520Values%2520for%2520Carboxylic%2520Acids%26jtitle%3DMolecules%26date%3D2019%26volume%3D24%26issue%3D1%26spage%3D79%26doi%3D10.3390%2Fmolecules24010079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Virant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DrvariÄ Talian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podlipnik, Ä.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hribar-Lee, B.</span></span> <span> </span><span class="NLM_article-title">Modelling the Correlation Between Molecular Electrostatic Potential and pKa on Sets of Carboxylic Acids, Phenols and Anilines</span>. <i>Acta Chim. Slov.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">560</span>â <span class="NLM_lpage">563</span>, <span class="refDoi">Â DOI: 10.17344/acsi.2016.2962</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.17344%2Facsi.2016.2962" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=28862310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslaqs7rO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=560-563&author=M.+Virantauthor=S.+Drvari%C4%8D+Talianauthor=%C4%8C.+Podlipnikauthor=B.+Hribar-Lee&title=Modelling+the+Correlation+Between+Molecular+Electrostatic+Potential+and+pKa+on+Sets+of+Carboxylic+Acids%2C+Phenols+and+Anilines&doi=10.17344%2Facsi.2016.2962"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Modelling the correlation between molecular electrostatic potential and pKa on sets of carboxylic acids, phenols and anilines</span></div><div class="casAuthors">Virant, Miha; Talian, Sara Drvaric; Podlipnik, Crtomir; Hribar-Lee, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Acta Chimica Slovenica</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">560-563</span>CODEN:
                <span class="NLM_cas:coden">ACSLE7</span>;
        ISSN:<span class="NLM_cas:issn">1318-0207</span>.
    
            (<span class="NLM_cas:orgname">Slovenian Chemical Society</span>)
        </div><div class="casAbstract">Calcns. of mol. electrostatic potential were correlated with exptl. pKa values for different sets of acidic mols. (carboxylic acids, phenols, and anilines) to obtain linear relationships of variable quality.  A single tri-parameter model function was constructed to describe the pKa dependence on MEP maxima together with two automatically generated mol. descriptors, namely the counts of carboxylic acid and amine functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt86JOKj7P27Vg90H21EOLACvtfcHk0liGw-jk8SU_zw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslaqs7rO&md5=ceb2888eb8b3c83546779edcde2ba18b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.17344%2Facsi.2016.2962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.17344%252Facsi.2016.2962%26sid%3Dliteratum%253Aachs%26aulast%3DVirant%26aufirst%3DM.%26aulast%3DDrvari%25C4%258D%2BTalian%26aufirst%3DS.%26aulast%3DPodlipnik%26aufirst%3D%25C4%258C.%26aulast%3DHribar-Lee%26aufirst%3DB.%26atitle%3DModelling%2520the%2520Correlation%2520Between%2520Molecular%2520Electrostatic%2520Potential%2520and%2520pKa%2520on%2520Sets%2520of%2520Carboxylic%2520Acids%252C%2520Phenols%2520and%2520Anilines%26jtitle%3DActa%2520Chim.%2520Slov.%26date%3D2017%26spage%3D560%26epage%3D563%26doi%3D10.17344%2Facsi.2016.2962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slater, A. M.</span></span> <span> </span><span class="NLM_article-title">The IUPAC Aqueous and Non-Aqueous Experimental PKa Data Repositories of Organic Acids and Bases</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1031</span>â <span class="NLM_lpage">1034</span>, <span class="refDoi">Â DOI: 10.1007/s10822-014-9764-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1007%2Fs10822-014-9764-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=24952470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaju7nP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=1031-1034&issue=10&author=A.+M.+Slater&title=The+IUPAC+Aqueous+and+Non-Aqueous+Experimental+PKa+Data+Repositories+of+Organic+Acids+and+Bases&doi=10.1007%2Fs10822-014-9764-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The IUPAC aqueous and non-aqueous experimental pKa data repositories of organic acids and bases</span></div><div class="casAuthors">Slater, Anthony Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1031-1034</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Accurate and well-curated exptl. pKa data of org. acids and bases in both aq. and non-aq. media are invaluable in many areas of chem. research, including pharmaceutical, agrochem., specialty chem. and property prediction research.  In pharmaceutical research, pKa data are relevant in ligand design, protein binding, absorption, distribution, metab., elimination as well as soly. and dissoln. rate.  The pKa data compilations of the International Union of Pure and Applied Chem., originally in book form, have been carefully converted into computer-readable form, with value being added in the process, in the form of ionisation assignments and tautomer enumeration.  These compilations offer a broad range of chem. in both aq. and non-aq. media and the exptl. conditions and original ref. for all pKa detns. are supplied.  The statistics for these compilations are presented and the utility of the computer-readable form of these compilations is examd. in comparison to other pKa compilations.  Finally, information is provided about how to access these databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFoevnTg4QIrVg90H21EOLACvtfcHk0liGw-jk8SU_zw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaju7nP&md5=0f537621a321288255abbb74f8e8bac3</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9764-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9764-9%26sid%3Dliteratum%253Aachs%26aulast%3DSlater%26aufirst%3DA.%2BM.%26atitle%3DThe%2520IUPAC%2520Aqueous%2520and%2520Non-Aqueous%2520Experimental%2520PKa%2520Data%2520Repositories%2520of%2520Organic%2520Acids%2520and%2520Bases%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26issue%3D10%26spage%3D1031%26epage%3D1034%26doi%3D10.1007%2Fs10822-014-9764-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, D. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smallheer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knabb, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wexler, R. R.</span></span> <span> </span><span class="NLM_article-title">Factor Xa Inhibitors: Next-Generation Antithrombotic Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">6243</span>â <span class="NLM_lpage">6274</span>, <span class="refDoi">Â DOI: 10.1021/jm100146h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100146h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsFOktrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6243-6274&issue=17&author=D.+J.+P.+Pintoauthor=J.+M.+Smallheerauthor=D.+L.+Cheneyauthor=R.+M.+Knabbauthor=R.+R.+Wexler&title=Factor+Xa+Inhibitors%3A+Next-Generation+Antithrombotic+Agents&doi=10.1021%2Fjm100146h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Factor Xa Inhibitors: Next-Generation Antithrombotic Agents</span></div><div class="casAuthors">Pinto, Donald J. P.; Smallheer, Joanne M.; Cheney, Daniel L.; Knabb, Robert M.; Wexler, Ruth R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6243-6274</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjlojOUGw747Vg90H21EOLACvtfcHk0lieAo87pAKFAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsFOktrY%253D&md5=241f7c0a1fcc789ceed8e0b4beb03beb</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm100146h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100146h%26sid%3Dliteratum%253Aachs%26aulast%3DPinto%26aufirst%3DD.%2BJ.%2BP.%26aulast%3DSmallheer%26aufirst%3DJ.%2BM.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DKnabb%26aufirst%3DR.%2BM.%26aulast%3DWexler%26aufirst%3DR.%2BR.%26atitle%3DFactor%2520Xa%2520Inhibitors%253A%2520Next-Generation%2520Antithrombotic%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D17%26spage%3D6243%26epage%3D6274%26doi%3D10.1021%2Fjm100146h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthwick, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns-Kurtis, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charbaut, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Convery, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleanthous, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pateman, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patikis, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, I. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toomey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, H. E.</span></span> <span> </span><span class="NLM_article-title">Structure and Property Based Design of Factor Xa Inhibitors: Pyrrolidin-2-ones with Biaryl P4 Motifs</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>â <span class="NLM_lpage">27</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2007.11.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1016%2Fj.bmcl.2007.11.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=18054228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD1cXis1Sqtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=23-27&issue=1&author=R.+J.+Youngauthor=A.+D.+Borthwickauthor=D.+Brownauthor=C.+L.+Burns-Kurtisauthor=M.+Campbellauthor=C.+Chanauthor=M.+Charbautauthor=C.+Chungauthor=M.+A.+Converyauthor=H.+A.+Kellyauthor=N.+P.+Kingauthor=S.+Kleanthousauthor=A.+M.+Masonauthor=A.+J.+Patemanauthor=A.+N.+Patikisauthor=I.+L.+Pintoauthor=D.+R.+Pollardauthor=S.+Sengerauthor=G.+P.+Shahauthor=J.+R.+Toomeyauthor=N.+S.+Watsonauthor=H.+E.+Weston&title=Structure+and+Property+Based+Design+of+Factor+Xa+Inhibitors%3A+Pyrrolidin-2-ones+with+Biaryl+P4+Motifs&doi=10.1016%2Fj.bmcl.2007.11.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and property based design of factor Xa inhibitors: pyrrolidin-2-ones with biaryl P4 motifs</span></div><div class="casAuthors">Young, Robert J.; Borthwick, Alan D.; Brown, David; Burns-Kurtis, Cynthia L.; Campbell, Matthew; Chan, Chuen; Charbaut, Marie; Chung, Chun-wa; Convery, Maire A.; Kelly, Henry A.; King, N. Paul; Kleanthous, Savvas; Mason, Andrew M.; Pateman, Anthony J.; Patikis, Angela N.; Pinto, Ivan L.; Pollard, Derek R.; Senger, Stefan; Shah, Gita P.; Toomey, John R.; Watson, Nigel S.; Weston, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-27</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Structure and property based drug design was exploited in the synthesis of sulfonamidopyrrolidin-2-one-based factor Xa (fXa) inhibitors.  By incorporating biaryl P4 groups, a series of highly potent inhibitors, e.g., I, was obtained with encouraging oral pharmacokinetic profiles and significant but sub-optimal anticoagulant activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodg91ZXDFpYrVg90H21EOLACvtfcHk0lieAo87pAKFAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXis1Sqtg%253D%253D&md5=a5f6a28d12df20bca8578f42d374fc80</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.11.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.11.023%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DBorthwick%26aufirst%3DA.%2BD.%26aulast%3DBrown%26aufirst%3DD.%26aulast%3DBurns-Kurtis%26aufirst%3DC.%2BL.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DC.%26aulast%3DCharbaut%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DConvery%26aufirst%3DM.%2BA.%26aulast%3DKelly%26aufirst%3DH.%2BA.%26aulast%3DKing%26aufirst%3DN.%2BP.%26aulast%3DKleanthous%26aufirst%3DS.%26aulast%3DMason%26aufirst%3DA.%2BM.%26aulast%3DPateman%26aufirst%3DA.%2BJ.%26aulast%3DPatikis%26aufirst%3DA.%2BN.%26aulast%3DPinto%26aufirst%3DI.%2BL.%26aulast%3DPollard%26aufirst%3DD.%2BR.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DG.%2BP.%26aulast%3DToomey%26aufirst%3DJ.%2BR.%26aulast%3DWatson%26aufirst%3DN.%2BS.%26aulast%3DWeston%26aufirst%3DH.%2BE.%26atitle%3DStructure%2520and%2520Property%2520Based%2520Design%2520of%2520Factor%2520Xa%2520Inhibitors%253A%2520Pyrrolidin-2-ones%2520with%2520Biaryl%2520P4%2520Motifs%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26issue%3D1%26spage%3D23%26epage%3D27%26doi%3D10.1016%2Fj.bmcl.2007.11.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, R. W.</span></span> <span> </span><span class="NLM_article-title">A Survey of Hammett Substituent Constants and Resonance and Field Parameters</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>91</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">165</span>â <span class="NLM_lpage">195</span>, <span class="refDoi">Â DOI: 10.1021/cr00002a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00002a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaK3MXhs1ehsLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1991&pages=165-195&issue=2&author=C.+Hanschauthor=A.+Leoauthor=R.+W.+Taft&title=A+Survey+of+Hammett+Substituent+Constants+and+Resonance+and+Field+Parameters&doi=10.1021%2Fcr00002a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A survey of Hammett substituent constants and resonance and field parameters</span></div><div class="casAuthors">Hansch, Corwin; Leo, A.; Taft, R. W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-95</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">Included in this review is an anal. of newer methods which can supplant this classic procedure for detn. of the title consts., 283 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKOTHJzZT0lbVg90H21EOLACvtfcHk0lieAo87pAKFAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhs1ehsLo%253D&md5=9fc814cd57c47680a5213f3438037800</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fcr00002a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00002a004%26sid%3Dliteratum%253Aachs%26aulast%3DHansch%26aufirst%3DC.%26aulast%3DLeo%26aufirst%3DA.%26aulast%3DTaft%26aufirst%3DR.%2BW.%26atitle%3DA%2520Survey%2520of%2520Hammett%2520Substituent%2520Constants%2520and%2520Resonance%2520and%2520Field%2520Parameters%26jtitle%3DChem.%2520Rev.%26date%3D1991%26volume%3D91%26issue%3D2%26spage%3D165%26epage%3D195%26doi%3D10.1021%2Fcr00002a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reader, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitorino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilsher, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolford, J.-A.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery Targeting Inhibitor of Apoptosis Proteins: Discovery of a Non-Alanine Lead Series with Dual Activity Against CIAP1 and XIAP</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">6574</span>â <span class="NLM_lpage">6588</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CitLrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6574-6588&issue=16&author=G.+Chessariauthor=I.+M.+Buckauthor=J.+E.+H.+Dayauthor=P.+J.+Dayauthor=A.+Iqbalauthor=C.+N.+Johnsonauthor=E.+J.+Lewisauthor=V.+Martinsauthor=D.+Millerauthor=M.+Readerauthor=D.+C.+Reesauthor=S.+J.+Richauthor=E.+Tamaniniauthor=M.+Vitorinoauthor=G.+A.+Wardauthor=P.+A.+Williamsauthor=G.+Williamsauthor=N.+E.+Wilsherauthor=J.-A.+Woolford&title=Fragment-Based+Drug+Discovery+Targeting+Inhibitor+of+Apoptosis+Proteins%3A+Discovery+of+a+Non-Alanine+Lead+Series+with+Dual+Activity+Against+CIAP1+and+XIAP&doi=10.1021%2Facs.jmedchem.5b00706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery Targeting Inhibitor of Apoptosis Proteins: Discovery of a Non-Alanine Lead Series with Dual Activity Against cIAP1 and XIAP</span></div><div class="casAuthors">Chessari, Gianni; Buck, Ildiko M.; Day, James E. H.; Day, Philip J.; Iqbal, Aman; Johnson, Christopher N.; Lewis, Edward J.; Martins, Vanessa; Miller, Darcey; Reader, Michael; Rees, David C.; Rich, Sharna J.; Tamanini, Emiliano; Vitorino, Marc; Ward, George A.; Williams, Pamela A.; Williams, Glyn; Wilsher, Nicola E.; Woolford, Alison J.-A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6574-6588</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitor of apoptosis proteins (IAPs) are important regulators of apoptosis and pro-survival signaling pathways whose deregulation is often assocd. with tumor genesis and tumor growth.  IAPs have been proposed as targets for anticancer therapy, and a no. of peptidomimetic IAP antagonists have entered clin. trials.  Using the fragment-based screening approach, the authors identified nonpeptidic fragments binding with millimolar affinities to both cellular inhibitor of apoptosis protein 1 (cIAP1) and X-linked inhibitor of apoptosis protein (XIAP).  Structure-based hit optimization together with an anal. of protein-ligand electrostatic potential complementarity allowed us to significantly increase binding affinity of the starting hits.  Subsequent optimization gave a potent nonalanine IAP antagonist I structurally distinct from all IAP antagonists previously reported.  The lead compd. had activity in cell-based assays and in a mouse xenograft efficacy model and represents a highly promising start point for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRpw0ZQNug7Vg90H21EOLACvtfcHk0ljzMl4HviDWEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CitLrL&md5=70a9c10527791ce2cedd0a74e0bbeebb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00706%26sid%3Dliteratum%253Aachs%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DBuck%26aufirst%3DI.%2BM.%26aulast%3DDay%26aufirst%3DJ.%2BE.%2BH.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DIqbal%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DLewis%26aufirst%3DE.%2BJ.%26aulast%3DMartins%26aufirst%3DV.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DReader%26aufirst%3DM.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DVitorino%26aufirst%3DM.%26aulast%3DWard%26aufirst%3DG.%2BA.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DWilsher%26aufirst%3DN.%2BE.%26aulast%3DWoolford%26aufirst%3DJ.-A.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520Targeting%2520Inhibitor%2520of%2520Apoptosis%2520Proteins%253A%2520Discovery%2520of%2520a%2520Non-Alanine%2520Lead%2520Series%2520with%2520Dual%2520Activity%2520Against%2520CIAP1%2520and%2520XIAP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D16%26spage%3D6574%26epage%3D6588%26doi%3D10.1021%2Facs.jmedchem.5b00706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammett, L. P.</span></span> <span> </span><span class="NLM_article-title">The Effect of Structure upon the Reactions of Organic Compounds. Benzene Derivatives</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1937</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">96</span>â <span class="NLM_lpage">103</span>, <span class="refDoi">Â DOI: 10.1021/ja01280a022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja01280a022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaA2sXjtlSqsw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1937&pages=96-103&issue=1&author=L.+P.+Hammett&title=The+Effect+of+Structure+upon+the+Reactions+of+Organic+Compounds.+Benzene+Derivatives&doi=10.1021%2Fja01280a022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of structure upon the reactions of organic compounds. Benzene derivatives</span></div><div class="casAuthors">Hammett, Louis P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1937</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">96-103</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A simple formula is proposed to represent the effect of a substituent in the m- or p- position of the benzene ring upon the rate or equil. of a reaction in which the reacting group is in a side chain attached to the ring.  This formula represents the total effect of the substituent and also the influence of changing length of the side chain, of the dielec. const. of the medium, and of temp. with a satisfactory precision in a wide variety of cases.  The theory that a substituent acts by internal electron displacement is in complete agreement with this and accounts also for the large deviations observed in the case of nonrigid structures.  Tables of substituent and reaction consts. are given from which the effect of many substituents upon a large no. of reaction rates and equil may be obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5TaaR_6a9IbVg90H21EOLACvtfcHk0ljzMl4HviDWEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaA2sXjtlSqsw%253D%253D&md5=ec73b07f4ac4872e26e4487a0240d5c2</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fja01280a022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja01280a022%26sid%3Dliteratum%253Aachs%26aulast%3DHammett%26aufirst%3DL.%2BP.%26atitle%3DThe%2520Effect%2520of%2520Structure%2520upon%2520the%2520Reactions%2520of%2520Organic%2520Compounds.%2520Benzene%2520Derivatives%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1937%26volume%3D59%26issue%3D1%26spage%3D96%26epage%3D103%26doi%3D10.1021%2Fja01280a022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span>eMolecules database. <a href="https://reaxys.emolecules.com/index.php" class="extLink">https://reaxys.emolecules.com/index.php</a> (accessed Jan 17, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=eMolecules+database.+https%3A%2F%2Freaxys.emolecules.com%2Findex.php+%28accessed+Jan+17%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span>Sigma-Aldrich. <a href="https://www.sigmaaldrich.com/united-kingdom.html" class="extLink">https://www.sigmaaldrich.com/united-kingdom.html</a> (accessed Jun 3, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Sigma-Aldrich.+https%3A%2F%2Fwww.sigmaaldrich.com%2Funited-kingdom.html+%28accessed+Jun+3%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swain, M.</span></span> Molecule Validation and Standardization. <a href="https://github.com/mcs07/MolVS" class="extLink">https://github.com/mcs07/MolVS</a> (accessed Jan 17, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Swain%2C+M.+Molecule+Validation+and+Standardization.+https%3A%2F%2Fgithub.com%2Fmcs07%2FMolVS+%28accessed+Jan+17%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DSwain%26aufirst%3DM.%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span>RDKit: Open-source cheminformatics. <a href="https://www.rdkit.org/" class="extLink">https://www.rdkit.org/</a> (accessed
Jan 17, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=RDKit%3A+Open-source+cheminformatics.+https%3A%2F%2Fwww.rdkit.org%2F+%28accessed%0AJan+17%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalovich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: A Large-Scale Bioactivity Database for Drug Discovery</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1100</span>â <span class="NLM_lpage">D1107</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkr777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-D1107&issue=D1&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+A+Large-Scale+Bioactivity+Database+for+Drug+Discovery&doi=10.1093%2Fnar%2Fgkr777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lgyZFLnPxWAEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520A%2520Large-Scale%2520Bioactivity%2520Database%2520for%2520Drug%2520Discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26issue%3DD1%26spage%3DD1100%26epage%3DD1107%26doi%3D10.1093%2Fnar%2Fgkr777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirjalili, V.</span></span> Data Clustering in Python. <a href="https://libraries.io/github/mirjalil/pyclust" class="extLink">https://libraries.io/github/mirjalil/pyclust</a> (accessed <span class="NLM_year">2019)</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mirjalili%2C+V.+Data+Clustering+in+Python.+https%3A%2F%2Flibraries.io%2Fgithub%2Fmirjalil%2Fpyclust+%28accessed+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DMirjalili%26aufirst%3DV.%26date%3D2019%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sayle, R.</span></span> <span> </span><span class="NLM_article-title">1st-class SMARTS patterns</span>. <a href="https://www.daylight.com/meetings/summerschool98/course/basics/ref/sayle/" class="extLink">https://www.daylight.com/meetings/summerschool98/course/basics/ref/sayle/</a> (accessed May 31, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=R.+Sayle&title=1st-class+SMARTS+patterns"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DSayle%26aufirst%3DR.%26atitle%3D1st-class%2520SMARTS%2520patterns%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, J.</span>; <span class="NLM_string-name">Schwab, C.</span>; <span class="NLM_string-name">Gasteiger, J.</span></span> <i>CORINA, 3D Structure Generator</i>, version <span class="NLM_edition">3.1</span>; <span class="NLM_publisher-name">Molecular Networks GmbH</span>: <span class="NLM_publisher-loc">Erlangen, Germany</span>, <span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=J.+Sadowski&author=C.+Schwab&author=J.+Gasteiger&title=CORINA%2C+3D+Structure+Generator"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSadowski%26aufirst%3DJ.%26jtitle%3DCORINA%252C%25203D%2520Structure%2520Generator%26pub%3DMolecular%2520Networks%2520GmbH%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epifanovsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wormit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kussmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horn, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaliman, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khaliullin, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">KuÅ, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proynov, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohrdanz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steele, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodcock, H. L.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuev, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alguire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beran, G. J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berquist, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandhorst, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravaya, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casanova, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Closser, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crittenden, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diedenhofen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiStasio, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Do, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutoi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edgar, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fatehi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fusti-Molnar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghysels, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubeva-Zadorozhnaya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson-Heine, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbach, P. H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohenstein, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagau, T.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaduk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khistyaev, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klunzinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosenkov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczyk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauter, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lao, K. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levchenko, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livshits, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lochan, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luenser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manohar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manzer, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardirossian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marenich, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhall, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuscamman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oana, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivares-Amaya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâNeill, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkhill, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrine, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peverati, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prociuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehn, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russ, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharada, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">StÃ¼ck, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thom, A. J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchimochi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanovschi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vydrov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeganeh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yost, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Z.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, I. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, G. K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, W. A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hehre, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klamt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, H. F.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherrill, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truhlar, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warshel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspuru-Guzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besley, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreuw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunietz, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlani, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwaltney, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrecht, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochsenfeld, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rassolov, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slipchenko, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subotnik, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Voorhis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krylov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, P. M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Head-Gordon, M.</span></span> <span> </span><span class="NLM_article-title">Advances in Molecular Quantum Chemistry Contained in the Q-Chem 4 Program Package</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>113</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">184</span>â <span class="NLM_lpage">215</span>, <span class="refDoi">Â DOI: 10.1080/00268976.2014.952696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1080%2F00268976.2014.952696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2ksbnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2015&pages=184-215&issue=2&author=Y.+Shaoauthor=Z.+Ganauthor=E.+Epifanovskyauthor=A.+T.+B.+Gilbertauthor=M.+Wormitauthor=J.+Kussmannauthor=A.+W.+Langeauthor=A.+Behnauthor=J.+Dengauthor=X.+Fengauthor=D.+Ghoshauthor=M.+Goldeyauthor=P.+R.+Hornauthor=L.+D.+Jacobsonauthor=I.+Kalimanauthor=R.+Z.+Khaliullinauthor=T.+Ku%C5%9Bauthor=A.+Landauauthor=J.+Liuauthor=E.+I.+Proynovauthor=Y.+M.+Rheeauthor=R.+M.+Richardauthor=M.+A.+Rohrdanzauthor=R.+P.+Steeleauthor=E.+J.+Sundstromauthor=H.+L.+Woodcockauthor=P.+M.+Zimmermanauthor=D.+Zuevauthor=B.+Albrechtauthor=E.+Alguireauthor=B.+Austinauthor=G.+J.+O.+Beranauthor=Y.+A.+Bernardauthor=E.+Berquistauthor=K.+Brandhorstauthor=K.+B.+Bravayaauthor=S.+T.+Brownauthor=D.+Casanovaauthor=C.-M.+Changauthor=Y.+Chenauthor=S.+H.+Chienauthor=K.+D.+Closserauthor=D.+L.+Crittendenauthor=M.+Diedenhofenauthor=R.+A.+DiStasioauthor=H.+Doauthor=A.+D.+Dutoiauthor=R.+G.+Edgarauthor=S.+Fatehiauthor=L.+Fusti-Molnarauthor=A.+Ghyselsauthor=A.+Golubeva-Zadorozhnayaauthor=J.+Gomesauthor=M.+W.+D.+Hanson-Heineauthor=P.+H.+P.+Harbachauthor=A.+W.+Hauserauthor=E.+G.+Hohensteinauthor=Z.+C.+Holdenauthor=T.-C.+Jagauauthor=H.+Jiauthor=B.+Kadukauthor=K.+Khistyaevauthor=J.+Kimauthor=J.+Kimauthor=R.+A.+Kingauthor=P.+Klunzingerauthor=D.+Kosenkovauthor=T.+Kowalczykauthor=C.+M.+Krauterauthor=K.+U.+Laoauthor=A.+D.+Laurentauthor=K.+V.+Lawlerauthor=S.+V.+Levchenkoauthor=C.+Y.+Linauthor=F.+Liuauthor=E.+Livshitsauthor=R.+C.+Lochanauthor=A.+Luenserauthor=P.+Manoharauthor=S.+F.+Manzerauthor=S.-P.+Maoauthor=N.+Mardirossianauthor=A.+V.+Marenichauthor=S.+A.+Maurerauthor=N.+J.+Mayhallauthor=E.+Neuscammanauthor=C.+M.+Oanaauthor=R.+Olivares-Amayaauthor=D.+P.+O%E2%80%99Neillauthor=J.+A.+Parkhillauthor=T.+M.+Perrineauthor=R.+Peveratiauthor=A.+Prociukauthor=D.+R.+Rehnauthor=E.+Rostaauthor=N.+J.+Russauthor=S.+M.+Sharadaauthor=S.+Sharmaauthor=D.+W.+Smallauthor=A.+Sodtauthor=T.+Steinauthor=D.+St%C3%BCckauthor=Y.-C.+Suauthor=A.+J.+W.+Thomauthor=T.+Tsuchimochiauthor=V.+Vanovschiauthor=L.+Vogtauthor=O.+Vydrovauthor=T.+Wangauthor=M.+A.+Watsonauthor=J.+Wenzelauthor=A.+Whiteauthor=C.+F.+Williamsauthor=J.+Yangauthor=S.+Yeganehauthor=S.+R.+Yostauthor=Z.-Q.+Youauthor=I.+Y.+Zhangauthor=X.+Zhangauthor=Y.+Zhaoauthor=B.+R.+Brooksauthor=G.+K.+L.+Chanauthor=D.+M.+Chipmanauthor=C.+J.+Cramerauthor=W.+A.+Goddardauthor=M.+S.+Gordonauthor=W.+J.+Hehreauthor=A.+Klamtauthor=H.+F.+Schaeferauthor=M.+W.+Schmidtauthor=C.+D.+Sherrillauthor=D.+G.+Truhlarauthor=A.+Warshelauthor=X.+Xuauthor=A.+Aspuru-Guzikauthor=R.+Baerauthor=A.+T.+Bellauthor=N.+A.+Besleyauthor=J.-D.+Chaiauthor=A.+Dreuwauthor=B.+D.+Dunietzauthor=T.+R.+Furlaniauthor=S.+R.+Gwaltneyauthor=C.-P.+Hsuauthor=Y.+Jungauthor=J.+Kongauthor=D.+S.+Lambrechtauthor=W.+Liangauthor=C.+Ochsenfeldauthor=V.+A.+Rassolovauthor=L.+V.+Slipchenkoauthor=J.+E.+Subotnikauthor=T.+Van+Voorhisauthor=J.+M.+Herbertauthor=A.+I.+Krylovauthor=P.+M.+W.+Gillauthor=M.+Head-Gordon&title=Advances+in+Molecular+Quantum+Chemistry+Contained+in+the+Q-Chem+4+Program+Package&doi=10.1080%2F00268976.2014.952696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in molecular quantum chemistry contained in the Q-Chem 4 program package</span></div><div class="casAuthors">Shao, Yihan; Gan, Zhengting; Epifanovsky, Evgeny; Gilbert, Andrew T. B.; Wormit, Michael; Kussmann, Joerg; Lange, Adrian W.; Behn, Andrew; Deng, Jia; Feng, Xintian; Ghosh, Debashree; Goldey, Matthew; Horn, Paul R.; Jacobson, Leif D.; Kaliman, Ilya; Khaliullin, Rustam Z.; Kus, Tomasz; Landau, Arie; Liu, Jie; Proynov, Emil I.; Rhee, Young Min; Richard, Ryan M.; Rohrdanz, Mary A.; Steele, Ryan P.; Sundstrom, Eric J.; Woodcock, H. Lee, III; Zimmerman, Paul M.; Zuev, Dmitry; Albrecht, Ben; Alguire, Ethan; Austin, Brian; Beran, Gregory J. O.; Bernard, Yves A.; Berquist, Eric; Brandhorst, Kai; Bravaya, Ksenia B.; Brown, Shawn T.; Casanova, David; Chang, Chung-Min; Chen, Yunquing; Chien, Siu Hung; Closser, Kristina D.; Crittenden, Deborah L.; Diedenhofen, Michael; DiStasio, Robert A., Jr.; Do, Hainam; Dutoi, Anthony D.; Edgar, Richard G.; Fatehi, Shervin; Fusti-Molnar, Laszlo; Ghysels, An; Golubeva-Zadorozhnaya, Anna; Gomes, Joseph; Hanson-Heine, Magnus W. D.; Harbach, Philipp H. P.; Hauser, Andreas W.; Hohenstein, Edward G.; Holden, Zachary C.; Jagau, Thomas-C.; Ji, Hyunjun; Kaduk, Ben; Khistyaev, Kirill; Kim, Jaehoon; Kim, Jihan; King, Rollin A.; Klunzinger, Phil; Kosenkov, Dmytro; Kowalczyk, Tim; Krauter, Caroline M.; Lao, Ka Un; Laurent, Adele; Lawler, Keith V.; Levchenko, Sergey V.; Lin, Ching Yeh; Liu, Fenglai; Livshits, Ester; Lochan, Rohini C.; Luenser, Arne; Manohar, Prashant; Manzer, Samuel F.; Mao, Shan-Ping; Mardirossian, Narbe; Marenich, Aleksandr V.; Maurer, Simon A.; Mayhall, Nicholas J.; Neuscamman, Eric; Oana, C. Melania; Olivares-Amaya, Roberto; O'Neill, Darragh P.; Parkhill, John A.; Perrine, Trilisa M.; Peverati, Roberto; Prociuk, Alexander; Rehn, Dirk R.; Rosta, Edina; Russ, Nicholas J.; Sharada, Shaama M.; Sharma, Sandeep; Small, David W.; Sodt, Alexander; Stein, Tamar; Stuck, David; Su, Yu-Chuan; Thom, Alex J. W.; Tsuchimochi, Takashi; Vanovschi, Vitalii; Vogt, Leslie; Vydrov, Oleg; Wang, Tao; Watson, Mark A.; Wenzel, Jan; White, Alec; Williams, Christopher F.; Yang, Jun; Yeganeh, Sina; Yost, Shane R.; You, Zhi-Qiang; Zhang, Igor Ying; Zhang, Xing; Zhao, Yan; Brooks, Bernard R.; Chan, Garnet K. L.; Chipman, Daniel M.; Cramer, Christopher J.; Goddard, William A., III; Gordon, Mark S.; Hehre, Warren J.; Klamt, Andreas; Schaefer, Henry F., III; Schmidt, Michael W.; Sherrill, C. David; Truhlar, Donald G.; Warshel, Arieh; Xu, Xin; Aspuru-Guzik, Alan; Baer, Roi; Bell, Alexis T.; Besley, Nicholas A.; Chai, Jeng-Da; Dreuw, Andreas; Dunietz, Barry D.; Furlani, Thomas R.; Gwaltney, Steven R.; Hsu, Chao-Ping; Jung, Yousung; Kong, Jing; Lambrecht, Daniel S.; Liang, WanZhen; Ochsenfeld, Christian; Rassolov, Vitaly A.; Slipchenko, Lyudmila V.; Subotnik, Joseph E.; Van Voorhis, Troy; Herbert, John M.; Krylov, Anna I.; Gill, Peter M. W.; Head-Gordon, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">184-215</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  A summary of the tech. advances that are incorporated in the fourth major release of the Q-Chem quantum chem. program is provided, covering approx. the last seven years.  These include developments in d. functional theory methods and algorithms, NMR (NMR) property evaluation, coupled cluster and perturbation theories, methods for electronically excited and open-shell species, tools for treating extended environments, algorithms for walking on potential surfaces, anal. tools, energy and electron transfer modeling, parallel computing capabilities, and graphical user interfaces.  In addn., a selection of example case studies that illustrate these capabilities is given.  These include extensive benchmarks of the comparative accuracy of modern d. functionals for bonded and non-bonded interactions, tests of attenuated second order Moller-Plesset (MP2) methods for intermol. interactions, a variety of parallel performance benchmarks, and tests of the accuracy of implicit solvation models.  Some specific chem. examples include calcns. on the strongly correlated Cr2 dimer, exploring zeolite-catalyzed ethane dehydrogenation, energy decompn. anal. of a charged ter-mol. complex arising from glycerol photoionisation, and natural transition orbitals for a Frenkel exciton state in a nine-unit model of a self-assembling nanotube.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohxCdZ6JkTu7Vg90H21EOLACvtfcHk0ljaU7BPpPq8jw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2ksbnN&md5=a828159693d247dd683f67fe217fb909</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1080%2F00268976.2014.952696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268976.2014.952696%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DY.%26aulast%3DGan%26aufirst%3DZ.%26aulast%3DEpifanovsky%26aufirst%3DE.%26aulast%3DGilbert%26aufirst%3DA.%2BT.%2BB.%26aulast%3DWormit%26aufirst%3DM.%26aulast%3DKussmann%26aufirst%3DJ.%26aulast%3DLange%26aufirst%3DA.%2BW.%26aulast%3DBehn%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DGoldey%26aufirst%3DM.%26aulast%3DHorn%26aufirst%3DP.%2BR.%26aulast%3DJacobson%26aufirst%3DL.%2BD.%26aulast%3DKaliman%26aufirst%3DI.%26aulast%3DKhaliullin%26aufirst%3DR.%2BZ.%26aulast%3DKu%25C5%259B%26aufirst%3DT.%26aulast%3DLandau%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DProynov%26aufirst%3DE.%2BI.%26aulast%3DRhee%26aufirst%3DY.%2BM.%26aulast%3DRichard%26aufirst%3DR.%2BM.%26aulast%3DRohrdanz%26aufirst%3DM.%2BA.%26aulast%3DSteele%26aufirst%3DR.%2BP.%26aulast%3DSundstrom%26aufirst%3DE.%2BJ.%26aulast%3DWoodcock%26aufirst%3DH.%2BL.%26aulast%3DZimmerman%26aufirst%3DP.%2BM.%26aulast%3DZuev%26aufirst%3DD.%26aulast%3DAlbrecht%26aufirst%3DB.%26aulast%3DAlguire%26aufirst%3DE.%26aulast%3DAustin%26aufirst%3DB.%26aulast%3DBeran%26aufirst%3DG.%2BJ.%2BO.%26aulast%3DBernard%26aufirst%3DY.%2BA.%26aulast%3DBerquist%26aufirst%3DE.%26aulast%3DBrandhorst%26aufirst%3DK.%26aulast%3DBravaya%26aufirst%3DK.%2BB.%26aulast%3DBrown%26aufirst%3DS.%2BT.%26aulast%3DCasanova%26aufirst%3DD.%26aulast%3DChang%26aufirst%3DC.-M.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DChien%26aufirst%3DS.%2BH.%26aulast%3DClosser%26aufirst%3DK.%2BD.%26aulast%3DCrittenden%26aufirst%3DD.%2BL.%26aulast%3DDiedenhofen%26aufirst%3DM.%26aulast%3DDiStasio%26aufirst%3DR.%2BA.%26aulast%3DDo%26aufirst%3DH.%26aulast%3DDutoi%26aufirst%3DA.%2BD.%26aulast%3DEdgar%26aufirst%3DR.%2BG.%26aulast%3DFatehi%26aufirst%3DS.%26aulast%3DFusti-Molnar%26aufirst%3DL.%26aulast%3DGhysels%26aufirst%3DA.%26aulast%3DGolubeva-Zadorozhnaya%26aufirst%3DA.%26aulast%3DGomes%26aufirst%3DJ.%26aulast%3DHanson-Heine%26aufirst%3DM.%2BW.%2BD.%26aulast%3DHarbach%26aufirst%3DP.%2BH.%2BP.%26aulast%3DHauser%26aufirst%3DA.%2BW.%26aulast%3DHohenstein%26aufirst%3DE.%2BG.%26aulast%3DHolden%26aufirst%3DZ.%2BC.%26aulast%3DJagau%26aufirst%3DT.-C.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DKaduk%26aufirst%3DB.%26aulast%3DKhistyaev%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DKing%26aufirst%3DR.%2BA.%26aulast%3DKlunzinger%26aufirst%3DP.%26aulast%3DKosenkov%26aufirst%3DD.%26aulast%3DKowalczyk%26aufirst%3DT.%26aulast%3DKrauter%26aufirst%3DC.%2BM.%26aulast%3DLao%26aufirst%3DK.%2BU.%26aulast%3DLaurent%26aufirst%3DA.%2BD.%26aulast%3DLawler%26aufirst%3DK.%2BV.%26aulast%3DLevchenko%26aufirst%3DS.%2BV.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLivshits%26aufirst%3DE.%26aulast%3DLochan%26aufirst%3DR.%2BC.%26aulast%3DLuenser%26aufirst%3DA.%26aulast%3DManohar%26aufirst%3DP.%26aulast%3DManzer%26aufirst%3DS.%2BF.%26aulast%3DMao%26aufirst%3DS.-P.%26aulast%3DMardirossian%26aufirst%3DN.%26aulast%3DMarenich%26aufirst%3DA.%2BV.%26aulast%3DMaurer%26aufirst%3DS.%2BA.%26aulast%3DMayhall%26aufirst%3DN.%2BJ.%26aulast%3DNeuscamman%26aufirst%3DE.%26aulast%3DOana%26aufirst%3DC.%2BM.%26aulast%3DOlivares-Amaya%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DD.%2BP.%26aulast%3DParkhill%26aufirst%3DJ.%2BA.%26aulast%3DPerrine%26aufirst%3DT.%2BM.%26aulast%3DPeverati%26aufirst%3DR.%26aulast%3DProciuk%26aufirst%3DA.%26aulast%3DRehn%26aufirst%3DD.%2BR.%26aulast%3DRosta%26aufirst%3DE.%26aulast%3DRuss%26aufirst%3DN.%2BJ.%26aulast%3DSharada%26aufirst%3DS.%2BM.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSmall%26aufirst%3DD.%2BW.%26aulast%3DSodt%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DT.%26aulast%3DSt%25C3%25BCck%26aufirst%3DD.%26aulast%3DSu%26aufirst%3DY.-C.%26aulast%3DThom%26aufirst%3DA.%2BJ.%2BW.%26aulast%3DTsuchimochi%26aufirst%3DT.%26aulast%3DVanovschi%26aufirst%3DV.%26aulast%3DVogt%26aufirst%3DL.%26aulast%3DVydrov%26aufirst%3DO.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWatson%26aufirst%3DM.%2BA.%26aulast%3DWenzel%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DA.%26aulast%3DWilliams%26aufirst%3DC.%2BF.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DYeganeh%26aufirst%3DS.%26aulast%3DYost%26aufirst%3DS.%2BR.%26aulast%3DYou%26aufirst%3DZ.-Q.%26aulast%3DZhang%26aufirst%3DI.%2BY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DBrooks%26aufirst%3DB.%2BR.%26aulast%3DChan%26aufirst%3DG.%2BK.%2BL.%26aulast%3DChipman%26aufirst%3DD.%2BM.%26aulast%3DCramer%26aufirst%3DC.%2BJ.%26aulast%3DGoddard%26aufirst%3DW.%2BA.%26aulast%3DGordon%26aufirst%3DM.%2BS.%26aulast%3DHehre%26aufirst%3DW.%2BJ.%26aulast%3DKlamt%26aufirst%3DA.%26aulast%3DSchaefer%26aufirst%3DH.%2BF.%26aulast%3DSchmidt%26aufirst%3DM.%2BW.%26aulast%3DSherrill%26aufirst%3DC.%2BD.%26aulast%3DTruhlar%26aufirst%3DD.%2BG.%26aulast%3DWarshel%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DAspuru-Guzik%26aufirst%3DA.%26aulast%3DBaer%26aufirst%3DR.%26aulast%3DBell%26aufirst%3DA.%2BT.%26aulast%3DBesley%26aufirst%3DN.%2BA.%26aulast%3DChai%26aufirst%3DJ.-D.%26aulast%3DDreuw%26aufirst%3DA.%26aulast%3DDunietz%26aufirst%3DB.%2BD.%26aulast%3DFurlani%26aufirst%3DT.%2BR.%26aulast%3DGwaltney%26aufirst%3DS.%2BR.%26aulast%3DHsu%26aufirst%3DC.-P.%26aulast%3DJung%26aufirst%3DY.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DLambrecht%26aufirst%3DD.%2BS.%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DOchsenfeld%26aufirst%3DC.%26aulast%3DRassolov%26aufirst%3DV.%2BA.%26aulast%3DSlipchenko%26aufirst%3DL.%2BV.%26aulast%3DSubotnik%26aufirst%3DJ.%2BE.%26aulast%3DVan%2BVoorhis%26aufirst%3DT.%26aulast%3DHerbert%26aufirst%3DJ.%2BM.%26aulast%3DKrylov%26aufirst%3DA.%2BI.%26aulast%3DGill%26aufirst%3DP.%2BM.%2BW.%26aulast%3DHead-Gordon%26aufirst%3DM.%26atitle%3DAdvances%2520in%2520Molecular%2520Quantum%2520Chemistry%2520Contained%2520in%2520the%2520Q-Chem%25204%2520Program%2520Package%26jtitle%3DMol.%2520Phys.%26date%3D2015%26volume%3D113%26issue%3D2%26spage%3D184%26epage%3D215%26doi%3D10.1080%2F00268976.2014.952696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baker, N.</span></span> APBS-PDB2PQR Documentation. <a href="https://buildmedia.readthedocs.org/media/pdf/apbs-pdb2pqr/latest/apbs-pdb2pqr.pdf" class="extLink">https://buildmedia.readthedocs.org/media/pdf/apbs-pdb2pqr/latest/apbs-pdb2pqr.pdf</a> (accessed <span class="NLM_year">2018</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Baker%2C+N.+APBS-PDB2PQR+Documentation.+https%3A%2F%2Fbuildmedia.readthedocs.org%2Fmedia%2Fpdf%2Fapbs-pdb2pqr%2Flatest%2Fapbs-pdb2pqr.pdf+%28accessed+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DN.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, M. L.</span></span> <span> </span><span class="NLM_article-title">Solvent-Accessible Surfaces of Proteins and Nucleic Acids</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>221</i></span> (<span class="NLM_issue">4612</span>),  <span class="NLM_fpage">709</span>â <span class="NLM_lpage">713</span>, <span class="refDoi">Â DOI: 10.1126/science.6879170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1126%2Fscience.6879170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=6879170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADyaL3sXlsFCkt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=221&publication_year=1983&pages=709-713&issue=4612&author=M.+L.+Connolly&title=Solvent-Accessible+Surfaces+of+Proteins+and+Nucleic+Acids&doi=10.1126%2Fscience.6879170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Solvent-accessible surfaces of proteins and nucleic acids</span></div><div class="casAuthors">Connolly, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">221</span>
        (<span class="NLM_cas:issue">4612</span>),
    <span class="NLM_cas:pages">709-13</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A method is presented for anal. calcg. a smooth, 3-dimensional contour about a mol.  The mol. surface envelope may be drawn on either color raster computer displays or real-time vector computer graphics systems.  Mol. areas and vols. may be computed anal. from this surface representation.  Unlike most previous computer graphics representations of mols., which imitate wire models or space-filling plastic spheres, this surface shows only the atoms that are accessible to solvent.  This anal. method extends the earlier dot surface numerical algorithm, which was applied in enzymol., rational drug design, immunol., and understanding DNA base sequence recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl7dEGpUZzhrVg90H21EOLACvtfcHk0lgz1wuivt9JbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXlsFCkt74%253D&md5=371e91867872dcc45ff5d9c5cc142c95</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1126%2Fscience.6879170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.6879170%26sid%3Dliteratum%253Aachs%26aulast%3DConnolly%26aufirst%3DM.%2BL.%26atitle%3DSolvent-Accessible%2520Surfaces%2520of%2520Proteins%2520and%2520Nucleic%2520Acids%26jtitle%3DScience%26date%3D1983%26volume%3D221%26issue%3D4612%26spage%3D709%26epage%3D713%26doi%3D10.1126%2Fscience.6879170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, P. W.</span></span> <span> </span><span class="NLM_article-title">NGL Viewer: A Web Application for Molecular Visualization</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W576</span>â <span class="NLM_lpage">W579</span>, <span class="refDoi">Â DOI: 10.1093/nar/gkv402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1093%2Fnar%2Fgkv402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=25925569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbvL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=W576-W579&issue=W1&author=A.+S.+Roseauthor=P.+W.+Hildebrand&title=NGL+Viewer%3A+A+Web+Application+for+Molecular+Visualization&doi=10.1093%2Fnar%2Fgkv402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">NGL Viewer: a web application for molecular visualization</span></div><div class="casAuthors">Rose, Alexander S.; Hildebrand, Peter W.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W576-W579</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The NGL Viewer is a web application for the visualization of macromol. structures.  By fully adopting capabilities of modern web browsers, such as WebGL, for mol. graphics, the viewer can interactively display large mol. complexes and is also unaffected by the retirement of third-party plug-ins like Flash and Java Applets.  Generally, the web application offers comprehensive mol. visualization through a graphical user interface so that life scientists can easily access and profit from available structural data.  It supports common structural fileformats (e.g. PDB, mmCIF) and a variety of mol. representations (e.g. 'cartoon, spacefill, licorice').  Moreover, the viewer can be embedded in other web sites to provide specialized visualizations of entries in structural databases or results of structure-related calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq2USM5TOiD7Vg90H21EOLACvtfcHk0lgz1wuivt9JbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtbvL&md5=c6090488c03049f5dff2386aed09b738</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv402%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DA.%2BS.%26aulast%3DHildebrand%26aufirst%3DP.%2BW.%26atitle%3DNGL%2520Viewer%253A%2520A%2520Web%2520Application%2520for%2520Molecular%2520Visualization%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DW1%26spage%3DW576%26epage%3DW579%26doi%3D10.1093%2Fnar%2Fgkv402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span> <span> </span><span class="NLM_article-title">AstexViewer: A Visualisation Aid for Structure-Based Drug Design</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">871</span>â <span class="NLM_lpage">881</span>, <span class="refDoi">Â DOI: 10.1023/A:1023813504011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1023%2FA%3A1023813504011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=12825620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=871-881&issue=12&author=M.+J.+Hartshorn&title=AstexViewer%3A+A+Visualisation+Aid+for+Structure-Based+Drug+Design&doi=10.1023%2FA%3A1023813504011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">AstexViewer: a visualisation aid for structure-based drug design</span></div><div class="casAuthors">Hartshorn Michael J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">871-81</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">AstexViewer is a Java molecular graphics program that can be used for visualisation in many aspects of structure-based drug design.  This paper describes its functionality, implementation and examples of its use.  The program can run as an Applet in a web browser allowing structures to be displayed without installing additional software.  Applications of its use are described for visualisation and as part of a structure based design platform.  The software is being made freely available to the community and may be downloaded from http://www.astex-technology.com/AstexViewer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSPr0A35nMvHjMsgLhIoICsfW6udTcc2eYk-wp5R8-sVbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3psl2nsA%253D%253D&md5=f9396aae9ecf64f256a9f8588f8097bd</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023813504011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023813504011%26sid%3Dliteratum%253Aachs%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26atitle%3DAstexViewer%253A%2520A%2520Visualisation%2520Aid%2520for%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26issue%3D12%26spage%3D871%26epage%3D881%26doi%3D10.1023%2FA%3A1023813504011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, R. J.</span></span> <span> </span><span class="NLM_article-title">The Amber Biomolecular Simulation Programs</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">1668</span>â <span class="NLM_lpage">1688</span>, <span class="refDoi">Â DOI: 10.1002/jcc.20290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&issue=16&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+Biomolecular+Simulation+Programs&doi=10.1002%2Fjcc.20290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0lhxY9AHAfblxg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520Biomolecular%2520Simulation%2520Programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26issue%3D16%26spage%3D1668%26epage%3D1688%26doi%3D10.1002%2Fjcc.20290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chollet, F.</span></span>; <span class="NLM_etal"></span>.  <i>Keras</i>; <span class="NLM_publisher-name">GitHub</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=F.+Chollet&title=Keras"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DChollet%26aufirst%3DF.%26jtitle%3DKeras%26pub%3DGitHub%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abadi, M.</span>; <span class="NLM_string-name">Agarwal, A.</span>; <span class="NLM_string-name">Barham, P.</span>; <span class="NLM_string-name">Brevdo, E.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Citro, C.</span>; <span class="NLM_string-name">Corrado, G. S.</span>; <span class="NLM_string-name">Davis, A.</span>; <span class="NLM_string-name">Dean, J.</span>; <span class="NLM_string-name">Devin, M.</span>; <span class="NLM_string-name">Ghemawat, S.</span>; <span class="NLM_string-name">Goodfellow, I.</span>; <span class="NLM_string-name">Harp, A.</span>; <span class="NLM_string-name">Irving, G.</span>; <span class="NLM_string-name">Isard, M.</span>; <span class="NLM_string-name">Jia, Y.</span>; <span class="NLM_string-name">Jozefowicz, R.</span>; <span class="NLM_string-name">Kaiser, L.</span>; <span class="NLM_string-name">Kudlur, M.</span>; <span class="NLM_string-name">Levenberg, J.</span>; <span class="NLM_string-name">ManÃ©, D.</span>; <span class="NLM_string-name">Monga, R.</span>; <span class="NLM_string-name">Moore, S.</span>; <span class="NLM_string-name">Murray, D.</span>; <span class="NLM_string-name">Olah, C.</span>; <span class="NLM_string-name">Schuster, M.</span>; <span class="NLM_string-name">Shlens, J.</span>; <span class="NLM_string-name">Steiner, B.</span>; <span class="NLM_string-name">Sutskever, I.</span>; <span class="NLM_string-name">Talwar, K.</span>; <span class="NLM_string-name">Tucker, P.</span>; <span class="NLM_string-name">Vanhoucke, V.</span>; <span class="NLM_string-name">Vasudevan, V.</span>; <span class="NLM_string-name">ViÃ©gas, F.</span>; <span class="NLM_string-name">Vinyals, O.</span>; <span class="NLM_string-name">Warden, P.</span>; <span class="NLM_string-name">Wattenberg, M.</span>; <span class="NLM_string-name">Wicke, M.</span>; <span class="NLM_string-name">Yu, Y.</span>; <span class="NLM_string-name">Zheng, X.</span></span> <span> </span><span class="NLM_article-title">TensorFlow: Large-Scale Machine Learning on Heterogeneous Systems</span>.  <i>arXiv:1603.044</i>, <span class="NLM_year" style="font-weight: bold;">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=M.+Abadi&author=A.+Agarwal&author=P.+Barham&author=E.+Brevdo&author=Z.+Chen&author=C.+Citro&author=G.+S.+Corrado&author=A.+Davis&author=J.+Dean&author=M.+Devin&author=S.+Ghemawat&author=I.+Goodfellow&author=A.+Harp&author=G.+Irving&author=M.+Isard&author=Y.+Jia&author=R.+Jozefowicz&author=L.+Kaiser&author=M.+Kudlur&author=J.+Levenberg&author=D.+Man%C3%A9&author=R.+Monga&author=S.+Moore&author=D.+Murray&author=C.+Olah&author=M.+Schuster&author=J.+Shlens&author=B.+Steiner&author=I.+Sutskever&author=K.+Talwar&author=P.+Tucker&author=V.+Vanhoucke&author=V.+Vasudevan&author=F.+Vi%C3%A9gas&author=O.+Vinyals&author=P.+Warden&author=M.+Wattenberg&author=M.+Wicke&author=Y.+Yu&author=X.+Zheng&title=TensorFlow%3A+Large-Scale+Machine+Learning+on+Heterogeneous+Systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbadi%26aufirst%3DM.%26atitle%3DTensorFlow%253A%2520Large-Scale%2520Machine%2520Learning%2520on%2520Heterogeneous%2520Systems%26jtitle%3DarXiv%253A1603.044%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glorot, X.</span>; <span class="NLM_string-name">Bordes, A.</span>; <span class="NLM_string-name">Bengio, Y.</span></span> <span> </span><span class="NLM_article-title">Deep Sparse Rectifier Neural Networks</span>.  <i>Proceedings of the Fourteenth International Conference on Artificial Intelligence and Statistics</i>; <span class="NLM_publisher-name">JMLR</span>, <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">315</span>â <span class="NLM_lpage">323</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=315-323&author=X.+Glorot&author=A.+Bordes&author=Y.+Bengio&title=Proceedings+of+the+Fourteenth+International+Conference+on+Artificial+Intelligence+and+Statistics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGlorot%26aufirst%3DX.%26atitle%3DDeep%2520Sparse%2520Rectifier%2520Neural%2520Networks%26btitle%3DProceedings%2520of%2520the%2520Fourteenth%2520International%2520Conference%2520on%2520Artificial%2520Intelligence%2520and%2520Statistics%26pub%3DJMLR%26date%3D2011%26spage%3D315%26epage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kingma, D. P.</span>; <span class="NLM_string-name">Ba, J.</span></span> <span> </span><span class="NLM_article-title">Adam: A Method for Stochastic Optimization</span>.  <i>ArXiv14126980</i> <span class="NLM_year" style="font-weight: bold;">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+P.+Kingma&author=J.+Ba&title=Adam%3A+A+Method+for+Stochastic+Optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKingma%26aufirst%3DD.%2BP.%26atitle%3DAdam%253A%2520A%2520Method%2520for%2520Stochastic%2520Optimization%26jtitle%3DArXiv14126980%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VH6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VH6','PDB','2VH6'); return false;">PDB: 2VH6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C7A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5C7A','PDB','5C7A'); return false;">PDB: 5C7A</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i25"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01129">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_04104"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01129">10.1021/acs.jmedchem.9b01129</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tables providing data on performance on the training set, feature and feature group type definitions, and atomic properties used for the DNN-fp model; figures showing the effect of similarity to training set compounds on the quality of the ESP surfaces produced by the DNN-fp model; plots showing the correlation between predicted ESP values and DFT ESP values for all feature types; plots showing the correlation between predicted and experimental values for four properties, comparing DFT, DNN-fp, and AM1-BCC (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01129/suppl_file/jm9b01129_si_001.pdf">jm9b01129_si_001.pdf (1.64 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01129&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-16%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01129" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01129" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac47518e1192e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
