
<html lang="en"     class="pb-page"  data-request-id="718bb545-f8bf-472e-b20d-c40ce45f8cc3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.1c00665;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="3CL Protease Inhibitors with an Electrophilic Arylketone Moiety as Anti-SARS-CoV-2 Agents" /></meta><meta name="dc.Creator" content="Sho  Konno" /></meta><meta name="dc.Creator" content="Kiyotaka  Kobayashi" /></meta><meta name="dc.Creator" content="Miki  Senda" /></meta><meta name="dc.Creator" content="Yuta  Funai" /></meta><meta name="dc.Creator" content="Yuta  Seki" /></meta><meta name="dc.Creator" content="Ikumi  Tamai" /></meta><meta name="dc.Creator" content="Laura  Schäkel" /></meta><meta name="dc.Creator" content="Kyousuke  Sakata" /></meta><meta name="dc.Creator" content="Thanigaimalai  Pillaiyar" /></meta><meta name="dc.Creator" content="Akihiro  Taguchi" /></meta><meta name="dc.Creator" content="Atsuhiko  Taniguchi" /></meta><meta name="dc.Creator" content="Michael  Gütschow" /></meta><meta name="dc.Creator" content="Christa E.  Müller" /></meta><meta name="dc.Creator" content="Koh  Takeuchi" /></meta><meta name="dc.Creator" content="Mikako  Hirohama" /></meta><meta name="dc.Creator" content="Atsushi  Kawaguchi" /></meta><meta name="dc.Creator" content="Masaki  Kojima" /></meta><meta name="dc.Creator" content="Toshiya  Senda" /></meta><meta name="dc.Creator" content="Yoshiyuki  Shirasaka" /></meta><meta name="dc.Creator" content="Wataru  Kamitani" /></meta><meta name="dc.Creator" content="Yoshio  Hayashi" /></meta><meta name="dc.Description" content="The novel coronavirus, SARS-CoV-2, has been identified as the causative agent for the current coronavirus disease (COVID-19) pandemic. 3CL protease (3CLpro) plays a pivotal role in the processing o..." /></meta><meta name="Description" content="The novel coronavirus, SARS-CoV-2, has been identified as the causative agent for the current coronavirus disease (COVID-19) pandemic. 3CL protease (3CLpro) plays a pivotal role in the processing o..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 27, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00665" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00665" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00665" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00665" /></link>
        
    
    

<title>3CL Protease Inhibitors with an Electrophilic Arylketone Moiety as Anti-SARS-CoV-2 Agents | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00665" /></meta><meta property="og:title" content="3CL Protease Inhibitors with an Electrophilic Arylketone Moiety as Anti-SARS-CoV-2 Agents" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0006.jpeg" /></meta><meta property="og:description" content="The novel coronavirus, SARS-CoV-2, has been identified as the causative agent for the current coronavirus disease (COVID-19) pandemic. 3CL protease (3CLpro) plays a pivotal role in the processing of viral polyproteins. We report peptidomimetic compounds with a unique benzothiazolyl ketone as a warhead group, which display potent activity against SARS-CoV-2 3CLpro. The most potent inhibitor YH-53 can strongly block the SARS-CoV-2 replication. X-ray structural analysis revealed that YH-53 establishes multiple hydrogen bond interactions with backbone amino acids and a covalent bond with the active site of 3CLpro. Further results from computational and experimental studies, including an in vitro absorption, distribution, metabolism, and excretion profile, in vivo pharmacokinetics, and metabolic analysis of YH-53 suggest that it has a high potential as a lead candidate to compete with COVID-19." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00665"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00665">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00665&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00665&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00665&amp;href=/doi/10.1021/acs.jmedchem.1c00665" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00917" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00683" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">3CL Protease Inhibitors with an Electrophilic Arylketone Moiety as Anti-SARS-CoV-2 Agents</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sho Konno</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sho Konno</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sho++Konno">Sho Konno</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kiyotaka Kobayashi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kiyotaka Kobayashi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kiyotaka++Kobayashi">Kiyotaka Kobayashi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Miki Senda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Miki Senda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Biology Research Center, Institute of Materials Structure Science, High Energy Accelerator Research Organization (KEK), Tsukuba 305-0801, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Miki++Senda">Miki Senda</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3863-4503" title="Orcid link">https://orcid.org/0000-0003-3863-4503</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuta Funai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuta Funai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuta++Funai">Yuta Funai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yuta Seki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuta Seki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuta++Seki">Yuta Seki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ikumi Tamai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ikumi Tamai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ikumi++Tamai">Ikumi Tamai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4388-083X" title="Orcid link">https://orcid.org/0000-0003-4388-083X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura Schäkel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura Schäkel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Institute, Pharmaceutical & Medicinal Chemistry, University of Bonn, Bonn 53121, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Sch%C3%A4kel">Laura Schäkel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kyousuke Sakata</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kyousuke Sakata</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo 192-0392, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kyousuke++Sakata">Kyousuke Sakata</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thanigaimalai Pillaiyar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thanigaimalai Pillaiyar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Institute, Pharmaceutical/Medicinal Chemistry, University of Tübingen, Tübingen 72076, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thanigaimalai++Pillaiyar">Thanigaimalai Pillaiyar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Akihiro Taguchi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Akihiro Taguchi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Akihiro++Taguchi">Akihiro Taguchi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Atsuhiko Taniguchi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Atsuhiko Taniguchi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Atsuhiko++Taniguchi">Atsuhiko Taniguchi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Gütschow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Gütschow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Institute, Pharmaceutical & Medicinal Chemistry, University of Bonn, Bonn 53121, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++G%C3%BCtschow">Michael Gütschow</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9376-7897" title="Orcid link">https://orcid.org/0000-0002-9376-7897</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christa E. Müller</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christa E. Müller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaceutical Institute, Pharmaceutical & Medicinal Chemistry, University of Bonn, Bonn 53121, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christa+E.++M%C3%BCller">Christa E. Müller</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0013-6624" title="Orcid link">https://orcid.org/0000-0002-0013-6624</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Koh Takeuchi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Koh Takeuchi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cellular and Molecular Biotechnology Research Institute, National Institute of Advanced Industrial Science and Technology, Koto, Tokyo 135-0064, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Koh++Takeuchi">Koh Takeuchi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-6227-4627" title="Orcid link">https://orcid.org/0000-0002-6227-4627</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mikako Hirohama</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mikako Hirohama</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Medicine, Transborder Medical Research Center, University of Tsukuba, Tsukuba 305-8575, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mikako++Hirohama">Mikako Hirohama</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Atsushi Kawaguchi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Atsushi Kawaguchi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Medicine, Transborder Medical Research Center, University of Tsukuba, Tsukuba 305-8575, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Atsushi++Kawaguchi">Atsushi Kawaguchi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masaki Kojima</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masaki Kojima</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo 192-0392, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masaki++Kojima">Masaki Kojima</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Toshiya Senda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Toshiya Senda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Biology Research Center, Institute of Materials Structure Science, High Energy Accelerator Research Organization (KEK), Tsukuba 305-0801, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Toshiya++Senda">Toshiya Senda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yoshiyuki Shirasaka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yoshiyuki Shirasaka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yoshiyuki++Shirasaka">Yoshiyuki Shirasaka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wataru Kamitani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wataru Kamitani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Infectious Diseases and Host Defense, Gunma University Graduate School of Medicine, Maebashi 371-8511, Japan</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wataru++Kamitani">Wataru Kamitani</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Yoshio Hayashi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yoshio Hayashi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#51283930283022393811253e28303a247f30327f3b21"><span class="__cf_email__" data-cfemail="85fcede4fce4f6edecc5f1eafce4eef0abe4e6abeff5">[email protected]</span></a>. Phone: (+81)42-676-3279.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yoshio++Hayashi">Yoshio Hayashi</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7010-6914" title="Orcid link">https://orcid.org/0000-0002-7010-6914</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00665&amp;href=/doi/10.1021%2Facs.jmedchem.1c00665" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 27, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 April 2021</li><li><span class="item_label"><b>Published</b> online</span>27 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00665" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00665</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSho%2BKonno%252C%2BKiyotaka%2BKobayashi%252C%2BMiki%2BSenda%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00665%26title%3D3CL%2BProtease%2BInhibitors%2Bwith%2Ban%2BElectrophilic%2BArylketone%2BMoiety%2Bas%2BAnti-SARS-CoV-2%2BAgents%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00665"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">686</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00665" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;3CL Protease Inhibitors with an Electrophilic Arylketone Moiety as Anti-SARS-CoV-2 Agents&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sho&quot;,&quot;last_name&quot;:&quot;Konno&quot;},{&quot;first_name&quot;:&quot;Kiyotaka&quot;,&quot;last_name&quot;:&quot;Kobayashi&quot;},{&quot;first_name&quot;:&quot;Miki&quot;,&quot;last_name&quot;:&quot;Senda&quot;},{&quot;first_name&quot;:&quot;Yuta&quot;,&quot;last_name&quot;:&quot;Funai&quot;},{&quot;first_name&quot;:&quot;Yuta&quot;,&quot;last_name&quot;:&quot;Seki&quot;},{&quot;first_name&quot;:&quot;Ikumi&quot;,&quot;last_name&quot;:&quot;Tamai&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Schäkel&quot;},{&quot;first_name&quot;:&quot;Kyousuke&quot;,&quot;last_name&quot;:&quot;Sakata&quot;},{&quot;first_name&quot;:&quot;Thanigaimalai&quot;,&quot;last_name&quot;:&quot;Pillaiyar&quot;},{&quot;first_name&quot;:&quot;Akihiro&quot;,&quot;last_name&quot;:&quot;Taguchi&quot;},{&quot;first_name&quot;:&quot;Atsuhiko&quot;,&quot;last_name&quot;:&quot;Taniguchi&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Gütschow&quot;},{&quot;first_name&quot;:&quot;Christa&quot;,&quot;last_name&quot;:&quot;E. Müller&quot;},{&quot;first_name&quot;:&quot;Koh&quot;,&quot;last_name&quot;:&quot;Takeuchi&quot;},{&quot;first_name&quot;:&quot;Mikako&quot;,&quot;last_name&quot;:&quot;Hirohama&quot;},{&quot;first_name&quot;:&quot;Atsushi&quot;,&quot;last_name&quot;:&quot;Kawaguchi&quot;},{&quot;first_name&quot;:&quot;Masaki&quot;,&quot;last_name&quot;:&quot;Kojima&quot;},{&quot;first_name&quot;:&quot;Toshiya&quot;,&quot;last_name&quot;:&quot;Senda&quot;},{&quot;first_name&quot;:&quot;Yoshiyuki&quot;,&quot;last_name&quot;:&quot;Shirasaka&quot;},{&quot;first_name&quot;:&quot;Wataru&quot;,&quot;last_name&quot;:&quot;Kamitani&quot;},{&quot;first_name&quot;:&quot;Yoshio&quot;,&quot;last_name&quot;:&quot;Hayashi&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00665&quot;},&quot;abstract&quot;:&quot;The novel coronavirus, SARS-CoV-2, has been identified as the causative agent for the current coronavirus disease (COVID-19) pandemic. 3CL protease (3CLpro) plays a pivotal role in the processing of viral polyproteins. We report peptidomimetic compounds with a unique benzothiazolyl ketone as a warhead group, which display potent activity against SARS-CoV-2 3CLpro. The most potent inhibitor YH-53 can strongly block the SARS-CoV-2 replication. X-ray structural analysis revealed that YH-53 establishes multiple hydrogen bond interactions with backbone amino acids and a covalent bond with the active site of 3CLpro. Further results from computational and experimental studies, including an in vitro absorption, distribution, metabolism, and excretion profile, in vivo pharmacokinetics, and metabolic analysis of YH-53 suggest that it has a high potential as a lead candidate to compete with COVID-19.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00665&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00665" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00665&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00665" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00665&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00665" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00665&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00665&amp;href=/doi/10.1021/acs.jmedchem.1c00665" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00665" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00665" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00665%26sid%3Dliteratum%253Aachs%26pmid%3D34313428%26genre%3Darticle%26aulast%3DKonno%26date%3D2021%26atitle%3D3CL%2BProtease%2BInhibitors%2Bwith%2Ban%2BElectrophilic%2BArylketone%2BMoiety%2Bas%2BAnti-SARS-CoV-2%2BAgents%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292644" title="SARS-CoV-2">SARS-CoV-2</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291089" title="Plasma">Plasma</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The novel coronavirus, SARS-CoV-2, has been identified as the causative agent for the current coronavirus disease (COVID-19) pandemic. 3CL protease (3CL<sup>pro</sup>) plays a pivotal role in the processing of viral polyproteins. We report peptidomimetic compounds with a unique benzothiazolyl ketone as a warhead group, which display potent activity against SARS-CoV-2 3CL<sup>pro</sup>. The most potent inhibitor YH-53 can strongly block the SARS-CoV-2 replication. X-ray structural analysis revealed that YH-53 establishes multiple hydrogen bond interactions with backbone amino acids and a covalent bond with the active site of 3CL<sup>pro</sup>. Further results from computational and experimental studies, including an <i>in vitro</i> absorption, distribution, metabolism, and excretion profile, <i>in vivo</i> pharmacokinetics, and metabolic analysis of YH-53 suggest that it has a high potential as a lead candidate to compete with COVID-19.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar"><issue-title>COVID-19</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In December 2019, an outbreak of coronavirus disease-19 (COVID-19) emerged in Wuhan, China and evolved into a pandemic with major effects on the public health care system and the global economy. Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been identified as an etiological agent<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> and shares a very high RNA genome (∼80%) similarity with that of SARS-CoV-1,<a onclick="showRef(event, 'ref2 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref5">(2,5)</a> a virus that caused the SARS outbreak in 2003.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> As of March 5, 2021, more than 110 million cases of COVID-19 and over 2.5 million deaths have been reported. Recently, several vaccines protecting against SARS-CoV-2 were approved in many countries. With respect to the treatment of SARS-CoV-2, however, drug repositioning is a promising approach to discovery of new indications from existing clinical drugs since they can become available within a short development period. Considering the seriousness and the pandemic of SARS-CoV-2 infection, a huge effort has been made to reposition approved drugs that are known to act against related pathogens.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Remdesivir, an inhibitor of RNA-dependent RNA polymerase (RdRp), which was originally developed for use against ebolavirus,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> has been approved for the treatment of patients with severe COVID-19 as a directly acting antiviral agent.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The approved vaccines and drugs for treatment of COVID-19 give us hope to cope with the pandemic, but the variety of specific inhibitors which fight SARS-CoV-2 infection with a different mechanism of action are urgently needed to reduce the cases and also the severity and fatality rates. Examples include α-ketoamide<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and aldehyde inhibitors<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> with potent anti-SARS-CoV-2 activity through their protease inhibition. These compounds have good pharmacokinetics and no significant toxicity.</div><div class="NLM_p">Coronavirus is an enveloped, single-stranded, positive-sense RNA virus that is manipulated directly by a host’s translational system. Its replicase gene encodes for two large overlapping polyproteins, pp1a and pp1ab, which are required for viral replication and transcription.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The functional proteins are released from these polyproteins by extensive proteolytic processing, predominantly by the 3C-like protease (3CL<sup>pro</sup>), also referred to as the main protease (M<sup>pro</sup>), and the papain-like protease (PL<sup>pro</sup>).<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19">(15−19)</a> 3CL<sup>pro</sup>, a cysteine protease that is active at 11 conserved sites within the large viral polyproteins pp1a and pp1ab, is essential for viral replication and multiplication.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The indispensable role of 3CL<sup>pro</sup> in the viral life cycle and the absence of closely related human homologues establish 3CL<sup>pro</sup> as an ideal target for the development of antiviral agents for coronavirus diseases.<a onclick="showRef(event, 'ref7 ref15 ref21'); return false;" href="javascript:void(0);" class="ref ref7 ref15 ref21">(7,15,21)</a> A large part of the drug discovery program for the control of SARS in 2003 and MERS in 2012 has therefore been dedicated to the development of potent 3CL<sup>pro</sup> inhibitors.<a onclick="showRef(event, 'ref7 ref15 ref21'); return false;" href="javascript:void(0);" class="ref ref7 ref15 ref21">(7,15,21)</a> 3CL<sup>pro</sup> of SARS-CoV-2 shows 99% homology and 96% identity to that of SARS-CoV-1.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">We have been developing a series of substrate-derived inhibitors of SARS-CoV-1 3CL<sup>pro</sup> since the outbreak of SARS in 2003.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26 ref27">(22−27)</a> Inhibitors SH-5, YH-53, and YH-71 (previously designated as <b>2i</b>, <b>5h</b>, and <b>5n</b>, respectively)<a onclick="showRef(event, 'ref25 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref27">(25,27)</a> that contain a unique benzothiazolyl ketone as the P1′-directed warhead were developed from a tetrapeptide inhibitor (<b>1</b>) by incorporation of a trifluoromethylketone moiety. They are of particular interest because they display potent SARS-CoV-1 3CL<sup>pro</sup> inhibitory activities with <i>K</i><sub>i</sub> values of 4.1,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> 6.3<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and 22<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The electrophilic ketone warhead forms a reversible covalent bond with the nucleophilic thiolate of the active site Cys145 in SARS-CoV-1 3CL<sup>pro</sup>, resulting in a hemithioketal intermediate that transiently inactivates the enzyme (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). The reversibility of this inhibition reduces the chances of the nonspecific irreversible reaction of numerous mammalian thiols and also the risk of clinically undesirable side effects or immune reactions. Given the highly structural identity of 3CL<sup>pro</sup> between SARS-CoV-1 and SARS-CoV-2, it is expected that these inhibitors would show anti-SARS-CoV-2 activity through 3CL<sup>pro</sup> inhibition.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In the present study, we repositioned and investigated the potential of our SARS-CoV-1 inhibitors as anti-SARS-CoV-2 drugs. In particular, inhibitors containing an indole moiety at the P3-position and the benzothiazolyl ketone as the reactive warhead as in YH-53 showed an extremely potent inhibitory activity against 3CL<sup>pro</sup> of SARS-CoV-2 and completely blocked viral replication in Vero cells. To understand the binding mode of these inhibitors, cocrystal structures of SARS-CoV-2 3CL<sup>pro</sup> in complexes with YH-53 and SH-5 were determined, providing useful insights into the precise interaction between the inhibitors’ structures and the relevant substrate pockets of the viral protease. Additional comprehensive characterization such as <i>in vitro</i> absorption, distribution, metabolism, and excretion (ADME), toxicity, pharmacokinetics, computational principal component analysis, and metabolic analysis have shown YH-53 to be an attractive lead compound in the development of anti-SARS-CoV-2 agents.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. 3CL protease inhibitors for SARS-CoV-1 with electrophilic arylketone warhead. (A) Structures of 3CL<sup>pro</sup> inhibitors <b>1</b>, SH-5, YH-53, and YH-71. SH-5 contains a tripeptide scaffold with a warhead and a carbamoyl unit at P4. YH-53 and YH-71 consist of a dipeptide scaffold with a warhead and a heteroaromatic unit at P3. (B) Proposed mechanism of inhibition by SH-5. Once SH-5 is bound to the enzyme, the active site Cys145 of 3CL<sup>pro</sup> attacks the ketone of SH-5 to afford a reversible covalent bond. The hemithioketal intermediate would then be stabilized by an oxyanion hole.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Inhibitory Activities for 3CL Protease</h3><div class="NLM_p">The inhibitors SH-5, YH-53 and YH-71 were prepared using our previously published synthetic methods.<a onclick="showRef(event, 'ref25 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref27">(25,27)</a> Using our recently developed biochemical assay, we first examined the inhibitory activity of SH-5, YH-53, and YH-71 against SARS-CoV-2 3CL<sup>pro</sup> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and C). The full length 3CL<sup>pro</sup> of SARS-CoV-2, His-tagged at the C-terminus, was expressed in <i>Escherichia coli</i>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The three inhibitors, SH-5, YH-53, and YH-71 were shown to be potent inhibitors of SARS-CoV-2 3CL<sup>pro</sup>, with <i>K</i><sub>i</sub> values in the nanomolar range (SH-5: <i>K</i><sub>i</sub> = 14.5 ± 1.7 nM; YH-53: <i>K</i><sub>i</sub> = 34.7 ± 2.1 nM; YH-71: <i>K</i><sub>i</sub> = 32.1 ± 3.3 nM). The inhibition mode of the most potent inhibitor, SH-5, was determined and was shown to be competitive (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). This confirms the predicted mechanism of action, a tight and reversible binding, and interaction with the active site in the S4-S1′-pockets. Hattori et al. recently reported the SARS-CoV-2 3CL<sup>pro</sup> inhibitory activity of YH-53 with <i>K</i><sub>i</sub> = 17.6 nM, which is consistent with our data.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Peptidomimetic benzothiazolyl ketone inhibitors have already been studied as inhibitors of several serine and cysteine proteases and the reversible formation of hemi(thio)ketal adducts has been demonstrated.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33">(30−33)</a> The reversible mode of action is supported by the analysis of the progress curves which did not show time-dependent inhibition (<a class="ref internalNav" href="#notes1" aria-label="Figure S1">Figure S1</a>). We also carried out an NMR titration experiment study, which confirmed that YH-53 bind to SARS-CoV-2 3CL<sup>pro</sup> in 1:1 stoichiometry and with slow exchange kinetics, as expected from the tight interaction between YH-53 and the target protein (<a class="ref internalNav" href="#notes1" aria-label="Figure S2">Figure S2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. SARS-CoV-2 3CL<sup>pro</sup> inhibitory assay. (A) Concentration-dependent inhibition of SARS-CoV-2 3CL<sup>pro</sup> by SH-5, YH-53 and YH-71. Reactions were monitored for 10 min. Data points represent mean values ± SEM from three independent experiments. <i>K</i><sub>i</sub> values were calculated using the Cheng–Prusoff equation and are noted in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The <i>K</i><sub>m</sub> value of the substrate was 48.4 μM. (B) Lineweaver–Burk plot for SH-5 inhibition of SARS-CoV-2 3CL<sup>pro</sup> at 15, 30, 45, 60, and 75 μM of fluorogenic substrate and 0, 10, 20, 30, 40, and 50 nM of the inhibitor SH-5. Three independent experiments were performed, and reactions were monitored for 10 min. The common intercept on the ordinate indicates a competitive inhibition type. (C) Global fit of kinetic data from (B) for competitive enzyme inhibition. The best fit value of <i>K</i><sub><i>i</i></sub> 19.8 ± 2.3 nM was in the same range as the value calculated by the Cheng–Prusoff equation (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> X-ray Crystallography of SARS-CoV-2 3CL Protease and Comparison of YH-53 and SH-5</h3><div class="NLM_p">We next determined crystal structures of the 3CL<sup>pro</sup>-YH-53 and -SH-5 complexes at 1.65 and 2.15 Å resolutions, respectively, to reveal the binding mode of such peptide-type inhibitors (<a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and B). The crystal structures showed that both inhibitors have extended conformations which interact with 3CL<sup>pro</sup> and the main chains of the inhibitors interact with the 12th β-strand of the 3CL<sup>pro</sup> in an antiparallel manner. Moreover, Cys145 forms a tetrahedral hemithioketal bond with the carbonyl carbon at the P1 position (<a class="ref internalNav" href="#notes1" aria-label="Figure S3">Figure S3</a>). Since the chemical structures of the P1’ (benzothiazole), P1 (pyrrolidin-2-one), and P2 (isobutyl) moieties of the two inhibitors are identical, the C-terminal portion of the inhibitors interacts with the 3CL<sup>pro</sup> in the same manner (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). While the benzothiazole, pyrrolidine-2-one, and isobutyl groups are well accommodated in the S1′, S1, and S2 pockets, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and E), only the pyrrolidin-2-one group is completely buried in the protein. The carbonyl and amine group of pyrrolidine-2-one at P1 position interacts with the side chain of His163 and Glu166, respectively. Interestingly, nitrogen and sulfur atoms of the benzothiazole form a hydrogen bond network with water and His41 in the active site, respectively. Since the chemical structures of the N-terminal part of YH-53 and SH-5 are different, the inhibitors provoke different interactions with the S3 and S4 specificity pockets of the target. In the SH-5 complex, the isopropyl group at the P3 position is exposed to the bulk solvent (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). The benzyloxy group at the P4 position interacts with Gln189, Thr190, and Ala191, and its one face is exposed to the solvent. At the P3 and P4 residues, only small conformational changes were observed when the structure was compared with the inhibitor-free form (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M2Q">6M2Q</a>). On the other hand, YH-53 binding induced a larger conformational change in a loop region (residues 188–194) of 3CL<sup>pro</sup>, and the residues of the loop region were shifted toward the inhibitor by approximately 2.5 Å. As a result, Thr190, Gln189, and the backbone Glu166 cover the 4-methoxy-indole group at the P3 position of YH-53, which is important for the enhanced inhibitory activity. The side chain carbonyl of Gln189 forms a hydrogen bond with the main chain amide group at the P2 position. As a result, the active site in the YH-53 complex has a more closed conformation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Since the methoxy group on the P3 indole group is not involved in strong interactions with the protein, YH-71, an inhibitor with P3-benzimidazole, can be expected to bind 3CL<sup>pro</sup> similarly to YH-53 with a conformational change of the loop region. Notably, the observed conformational change of the loop region is rather rare among the 3CL<sup>pro</sup> structures that have been characterized to date. Of the 345 3CL<sup>pro</sup> coordinates that were selected with RMSD values less than 2.0 Å for the YH-53 complex, only 12 structures have similar conformations of the loop region (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of SARS-CoV-2 3CL<sup>pro</sup> with its inhibitors. (A, B) Inhibitor binding sites of 3CL<sup>pro</sup> for YH-53 (A) and SH-5 (B). Protein carbons for YH-53 and SH-5 complexes are shown in green and yellow, respectively. YH-53 (A) and SH-5 (B) are in cyan and orange, respectively. Water molecules are labeled with “w”. (C) LSQ superposition of the YH-53 and SH-5 complexes using 281 CA atoms (RMSD = 0.594 Å). Cyan arrows show the shifts of loop residues. Atom color scheme is the same as that in panels (A) and (B). (D, E) Surface representations of SH-5 and YH-53 binding sites. Atom color scheme is the same as that in panels (A) and (B), but water molecules are in salmon pink. (F) Distribution of averaged Cα-atom displacements of residues 188–194 in 3CL<sup>pro</sup>. We performed least-squares fittings of 3CL<sup>pro</sup> structures obtained from the PDB onto the present 3CL<sup>pro</sup>-YH53 structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E18">7E18</a>) with residues 10–290 (Cα atoms) using program LSQKAB in the CCP4 program suite,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and 345 3CL<sup>pro</sup> structures with the RMSD values less than 2.0 Å were selected for the following calculation. The averaged Cα-atom displacement of residues 188–194 was calculated for each superposed structure using the LSQKAB output that contains Cα-atom displacements of the superposed residues, and the histogram was prepared. Twelve structures indicated with the red bracket have loop structures similar to the YH-53 complex. The others have loop structures similar to the inhibitor-free form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Antiviral Activity of SH-5, YH-53, and YH-71</h3><div class="NLM_p">The antiviral activities of the inhibitors SH-5, YH-53, and YH-71 against SARS-CoV-2 were examined by a cytopathic effect (CPE) assay in Vero cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, the viral proliferation was completely blocked by the addition of 25 μM SH-5, YH-53, or YH-71. Unlike the recombinant 3CL<sup>pro</sup> inhibition, YH-53 showed better antiviral activity than SH-5. Of note, YH-53 completely blocked the viral proliferation even at a concentration of 10 μM. No cytotoxicity of YH-53 was observed with a CC<sub>50</sub> value of >100 μM (<a class="ref internalNav" href="#notes1" aria-label="Figure S4">Figure S4</a>). To further validate the antiviral effect of YH-53, we used a quantitative real-time polymerase chain reaction (qRT-PCR) to monitor the total amount of viral RNA (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Consistent with the CPE assay, copies of total RNA were efficiently reduced with increased concentrations of YH-53. The antiviral activity of YH-53 was enhanced by adding CP-100356, which is a potent inhibitor of MDR-1 efflux transporter (<a class="ref internalNav" href="#notes1" aria-label="Figure S5">Figure S5</a>). These results indicate that YH-53 is a potent antiviral agent against SARS-CoV-2.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Anti-SARS-CoV-2 activities of SH-5, YH-53 and YH-71 in Vero cells. (A) Antiviral activity of SH-5, YH-53, and YH-71. Vero cells were infected with SARS-CoV-2 using a multiplicity of infection (MOI) of 0.01. The inhibitors SH-5, YH-53, and YH-71 (1–30 μM) were treated to the infected cells then incubated for 48 h at 37 °C. The viral/serum mixtures were transferred to the flesh Vero cell and incubated for 24 h at 37 °C, followed by subject to the plaque assay. Data points represent mean values ± SEM from three independent experiments. (B) Total viral RNA of infected cells treated with YH-53 was analyzed by qPCR. Data points represent mean values ± SEM from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Computational and <i>In Vitro</i> Safety and Toxicity Profiling of YH-53</h3><div class="NLM_p last">Encouraged by these results, we turned our attention to the possibility of YH-53 having drug development potential. First, we studied the behavior of YH-53 in plasma. YH-53 was found to be significantly stable in plasma, and no degradation was observed at 37 °C for 6 h in rat plasma. It was observed that 99% of YH-53 bound to the plasma proteins in both human and rat plasma. Next, we investigated the <i>in vitro</i> safety and toxicity profile of YH-53. To assess CYP inhibition by YH-53, human microsomes were incubated with YH-53 and the activity of CYP1A2, CYP2C8, CYP2C9, CYP2D6, and CYP3A4 was measured. As shown in <a class="ref internalNav" href="#notes1" aria-label="Table S2">Table S2</a>, no CYP2C9 and CYP3A4 inhibition of YH-53 was observed, while CYP1A2, CYP2D6, and CYP2C8 were moderately inhibited at 10 μM YH-53, which is considered tolerable. To examine the mutagenicity of YH-53, the Ames test and the MicroFlow assay were conducted. YH-53 showed no mutagenetic activities for both <i>Salmonella typhimurium</i> TA98 and TA100 strains and did not induce formation of micronuclei. We also tested YH-53 in a hERG inhibition assay to assess potential cardiotoxicity. No inhibition of the hERG current was observed at >100 μM of YH-53. In another approach to validation of the feasibility of a possible drug based on YH-53, we analyzed its pharmaceutical properties using an informatics method.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Construction of chemical space and a subsequent principal component analysis (PCA) revealed that YH-53 was mapped onto the same region occupied by other FDA-approved small molecule drugs (<a class="ref internalNav" href="#notes1" aria-label="Figure S6">Figure S6</a>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Specifically, the location of YH-53 was within the distribution range of antiviral protease inhibitors, known as the -navir group. Taken together, these results suggest that YH-53 possesses a favorable safety profile as a potential drug.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Preclinical <i>In Vitro</i> and <i>In Vivo</i> Assessment of ADME/PK Properties of YH-53</h3><div class="NLM_p">To characterize and understand the ADME properties of YH-53, its oral bioavailability and pharmacokinetics in rats were assessed with various <i>in vitro</i> and <i>in vivo</i> experiments. When YH-53 (0.1 mg/kg) was intravenously administered to rats, the area under the plasma concentration–time curve (AUC<sub>0–∞</sub>) of YH-53 was evaluated as 19.7 ± 8.2 ng·h/mL (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). After oral administration at 0.5 mg/kg, the AUC<sub>0–∞</sub> of YH-53 was 3.49 ± 1.00 ng·h/mL in rats (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Based on these pharmacokinetic parameters, the absolute bioavailability (BA) of YH-53 in rats was estimated to be 3.55% (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The maximum plasma concentration (<i>C</i><sub>max</sub>) and the time to reach maximum plasma concentration (<i>t</i><sub>max</sub>) of YH-53 were 1.08 ± 0.30 ng/mL and 0.350 ± 0.137 h, respectively. Interestingly, a higher apparent distribution volume of YH-53 (<i>V</i><sub>d</sub> = 3.51 ± 2.83 L/kg) subsequent to intravenous infusion was observed, suggesting a high peripheral distribution into body tissues. This is supported by the results from compartmental analysis demonstrating that the pharmacokinetic data could be best fitted to the three-compartment model based on the lowest value of the Akaike Information Criterion (18.2, 13.2, −30.2, and −26.2 for the one, two, three, and four compartment models, respectively) (<a class="ref internalNav" href="#notes1" aria-label="Figure S7 and Table S3">Figure S7 and Table S3</a>). On the other hand, the terminal half-life (<i>t</i><sub>1/2</sub>) of YH-53 after intravenous administration was 2.97 ± 1.02 h, while <i>t</i><sub>1/2</sub> of YH-53 after oral administration was 9.64 ± 6.11 h, results that appear to be inconsistent. In <a class="ref internalNav" href="#notes1" aria-label="Table S4">Table S4</a>, the permeability of YH-53 (1 μM) was assessed using monolayers of Caco-2 cells and exhibited high permeability values (18.9 ± 1.1 × 10<sup>–6</sup> cm/s). Moreover, <i>in vitro</i> metabolism study of YH-53 using cryopreserved hepatocytes in human and rat led to the identification of seven metabolites, and the main metabolic pathway appeared to be amide hydrolysis, producing HC-5, HC-6, and HC-7, with lesser contributions from cytochrome P450 (CYP)-catalyzed oxidative pathways (<a class="ref internalNav" href="#notes1" aria-label="Figure S8 and Table S5">Figure S8 and Table S5</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mean plasma concentration–time profile of YH-53 in rats. (A) Mean plasma concentration–time profile of YH-53 in rats after i.v. administration. YH-53 (0.1 mg/kg; 4 mL/kg) was administered by an i.v. bolus injection in rats. Inset depicts the same data on a semilogarithmic scale. Data are shown as the mean ± SD (<i>n</i> = 5). (B) Mean plasma concentration–time profile of YH-53 in rats after oral administration. YH-53 (0.5 mg/kg; 4 mL/kg) was administered orally in rats. Inset depicts the same data on a semilogarithmic scale. Data are shown as the mean ± SD (<i>n</i> = 5).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Pharmacokinetic Parameters of YH-53 in Rats after Intravenous and Oral Administration<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">pharmacokinetic parameters<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">i.v.<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">oral<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dose</td><td class="colsep0 rowsep0" align="left">(mg/kg)</td><td class="colsep0 rowsep0" align="left">0.100</td><td class="colsep0 rowsep0" align="left">0.500</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub></td><td class="colsep0 rowsep0" align="left">(ng/mL)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1.08 ± 0.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>max</sub></td><td class="colsep0 rowsep0" align="left">(h)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.350 ± 0.137</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–7</sub></td><td class="colsep0 rowsep0" align="left">(ng·h/mL)</td><td class="colsep0 rowsep0" align="left">19.3 ± 8.2</td><td class="colsep0 rowsep0" align="left">1.59 ± 0.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–∞</sub></td><td class="colsep0 rowsep0" align="left">(ng·h/mL)</td><td class="colsep0 rowsep0" align="left">19.7 ± 8.2</td><td class="colsep0 rowsep0" align="left">3.49 ± 1.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub></td><td class="colsep0 rowsep0" align="left">(h)</td><td class="colsep0 rowsep0" align="left">2.97 ± 1.02</td><td class="colsep0 rowsep0" align="left">9.64 ± 6.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MRT<sub>tot</sub></td><td class="colsep0 rowsep0" align="left">(h)</td><td class="colsep0 rowsep0" align="left">0.494 ± 0.147</td><td class="colsep0 rowsep0" align="left">13.2 ± 7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>tot</sub></td><td class="colsep0 rowsep0" align="left">(L/h/kg)</td><td class="colsep0 rowsep0" align="left">6.43 ± 3.50</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>tot</sub>/<i>F</i></td><td class="colsep0 rowsep0" align="left">(L/h/kg)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">159 ± 70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>d</sub></td><td class="colsep0 rowsep0" align="left">(L/kg)</td><td class="colsep0 rowsep0" align="left">3.51 ± 2.83</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>d</sub>/<i>F</i></td><td class="colsep0 rowsep0" align="left">(L/kg)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1800 ± 773</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i></td><td class="colsep0 rowsep0" align="left">(%)</td><td class="colsep0 rowsep0" colspan="2" align="center">3.55</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">YH-53 was intravenously (0.1 mg/kg) and orally (0.5 mg/kg) administered to rats.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Definitions of disposition parameters: AUC<sub>0-∞</sub>, area under plasma concentration–time curve from 0 to infinity; <i>C</i><sub>max</sub>, peak plasma drug concentration; <i>t</i><sub>max</sub>, time to reach maximum plasma concentration; CL<sub>tot</sub>, total clearance; <i>F</i>, oral bioavailability; <i>V</i><sub>d</sub>, volume of distribution; <i>t</i><sub>1/2</sub>, elimination half-life.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Pharmacokinetic parameters of YH-53 after a single intravenous dose were obtained from <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Pharmacokinetic parameters of YH-53 after a single oral dose were obtained from <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Data are shown as means ± SD (<i>n</i> = 5).</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45865" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45865" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protease inhibitors are major therapeutic agents for viral infections involving HIV and hepatitis C virus. In general, the design of these inhibitors is based on the natural substrate structure and introducing the appropriate inhibitory machinery that adapts to the substrate degradation mechanism of the protease. Similarly, drug development targeting the 3CL<sup>pro</sup> of SARS-CoV-2, a cysteine protease, is a promising strategy to fight COVID-19. In this report, we investigated the inhibitory effect of our original substrate-derived SARS-CoV-1 3CL<sup>pro</sup> inhibitors SH-5, YH-53, and YH-71 against SARS-CoV-2. All three inhibitors containing benzothiazolylketone, a unique P1′-arylketone moiety, showed potent inhibition against SARS-CoV-2 3CL<sup>pro</sup> and completely blocked the viral proliferation at 25 μM in Vero cells (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A, B). The TCID<sub>50</sub> value of the most potent YH-53 was ∼5 μM. Our data suggests that aryl ketone-type 3CL<sup>pro</sup> inhibitors developed for SARS-CoV-1 are effective against SARS-CoV-2 with a clear mechanism of action in a competitive manner. The results are largely attributed to the quite high similarity in the entire amino acid sequence and three-dimensional structure, particularly 100% identity at the active site in 3CL<sup>pro</sup> between SARS-CoV-1 and SARS-CoV-2.<a onclick="showRef(event, 'ref2 ref29'); return false;" href="javascript:void(0);" class="ref ref2 ref29">(2,29)</a> NMR data also indicated that this specific binding of YH-53 with 3CL<sup>pro</sup> is 1:1 stoichiometry with slow exchange kinetics, suggesting that the binding is tightly maintained (<a class="ref internalNav" href="#notes1" aria-label="Figure S2">Figure S2</a>). The potent anti-SARS-CoV-2 activity of YH-53 is consistent with a recent report by Hattori et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> It should be noted, however, that a slight gap was observed between the enzyme inhibitory and antiviral activities. This may be due to the ineffectiveness of the inhibitors to either penetrate into the cell or achieve high concentrations owing to the decomposition, consumption, or exclusion from the cell. In the MDR1 substrate screening assay, YH-53 acts as a substrate of the p-glycoprotein efflux pump (basal to apical/apical to basal ratio > 130) that is also expressed in Vero cells,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and this could be a valid reason for consumption of high concentrations of inhibitors to realize the effective <i>in vitro</i> antiviral activity (<a class="ref internalNav" href="#notes1" aria-label="Figure S4">Figure S4</a>). It has been reported that the expression level of the MDR1 gene increased by inflammatory responses in COVID-19 patients with mild or severe symptoms.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> However, the upregulation of MDR1 expression is limited to a small subset of respiratory cells and most respiratory cells remain MDR1 negative.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Further in vivo studies on the antiviral activity of YH-53 need to be pursued.</div><div class="NLM_p">In an effort to understand the precise recognition of arylketone-type 3CL<sup>pro</sup> inhibitors, high resolution X-ray structures of SARS-CoV-2 3CL<sup>pro</sup>-YH-53 and -SH-5 complexes were established (<a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and B). In addition to the key interactions between P1 pyrrolidine-2-one and Glu166 and His163, heteroatoms of benzothiazole at P1’ form further hydrogen bonds with water and His41, which would contribute the tight binding of the inhibitors. It has been observed that each side chain moiety on the extended conformation of YH-53 and SH-5 backbones fits into each substrate pocket in the active site of 3CL<sup>pro</sup>, and it was proven that the tight binding of these high-affinity inhibitors leads to the potent enzyme inhibitory activity with <i>K</i><sub>i</sub> values in the low nanomolar range. Interestingly, at the N-terminal part where the structure of two inhibitors is different, the binding mode to the protease was different (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and E). Compared to the tripeptide-type SH-5 containing the P4 part, the smaller dipeptide-type YH-53 resulted in a significant conformational change of the loop region of 3CL<sup>pro</sup>, drawing it closer to the inhibitor and making the active site more closed. This explains the strong interaction of YH-53 with 3CL<sup>pro</sup> even in the absence of the P4 moiety. It has also been revealed that this characteristic conformational change of the loop region is rare, as shown by the comparison with known PDB coordinates of 3CL<sup>pro</sup> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Other inhibitor-3CL<sup>pro</sup> complexes keep the loop structures similar to the inhibitor free form. Therefore, the conformational change may be a useful indicator for assessing the properties of new inhibitors developed in the future.</div><div class="NLM_p">Safety and toxicological evaluations suggested that the most potent antiviral 3CL<sup>pro</sup> inhibitor YH-53 has a high safety index with low cytotoxicity, no or moderate but tolerable CYP inhibition, no mutagenicity and no cardiotoxicity (<a class="ref internalNav" href="#notes1" aria-label="Table S2">Table S2</a>). As mentioned above, YH-53 was identified as the best SARS-CoV-2 3CL<sup>pro</sup> inhibitor in the screening of a compound library of 400 potential inhibitors which found YH-53 to be an excellent lead compound with a favorable safety profile.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Physicochemical properties including values of molecular weight of 575, cLogP of 2.37, LogD of 3.37, kinetic solubility of 7.7 μg/mL, and PAMPA of 213 nm/s are in the acceptable range for lead candidates. Moreover, the computational analysis reveals that YH-53 has strong potential for drug development, endowed with similar pharmaceutical properties to existing clinically used antiviral protease inhibitors (<a class="ref internalNav" href="#notes1" aria-label="Figure S6">Figure S6</a>).</div><div class="NLM_p">By means of the simultaneous analysis of intravenous and oral pharmacokinetic data, the bioavailability of YH-53 in rats was estimated to be approximately 3.6% (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). This low bioavailability is caused by poor absorption and/or significant first-pass metabolism. Interestingly, <i>in vitro</i> assessment of intestinal permeability of YH-53 using a Caco-2 cell monolayer system exhibited excellent permeability, close to that of metoprolol, a completely absorbed drug (Fa ≥ 90%) (<a class="ref internalNav" href="#notes1" aria-label="Table S4">Table S4</a>). Moreover, an <i>in vitro</i> metabolism study revealed that YH-53 is mainly metabolized by hydrolytic enzymes, with minor contributions from CYP enzymes (<a class="ref internalNav" href="#notes1" aria-label="Figure S8 and Table S5">Figure S8 and Table S5</a>). Taking these results into account, it is reasonable to consider that the low bioavailability of YH-53 in rats may be mainly due to the first-pass metabolism catalyzed by hydrolytic enzymes such as carboxylesterases or proteases in the intestine and/or liver. Meanwhile, a higher apparent distribution volume of YH-53 than the total body water (0.668 L/kg) of rats was observed, suggesting a high peripheral distribution into body tissues.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> This is emphasized by the fact that the pharmacokinetic data were best fitted to the three-compartment model. Theoretically, a three-compartment model raises the possibility that there is an initial fast distribution phase (fast-redistributing tissues) and slow distribution phase (slow-redistributing tissues), followed by an elimination phase based on redistribution from drug stored in the tissues. Further investigation would be necessary to examine the distribution of YH-53 to potential target organs such as lung, bronchial airway, intestine, heart, liver, and kidney which express angiotensin-converting enzyme 2 (ACE2), a functional receptor for SARS-CoV-2 infection. On the other hand, the <i>t</i><sub>1/2</sub> value of YH-53 after intravenous administration was approximately 3 h, indicating its relatively fast elimination from the body; that is, the mean residence time (MRT) in plasma is limited. However, after oral administration, the <i>t</i><sub>1/2</sub> value of YH-53 was approximately 10 h, implying a prolonged elimination of YH-53. These inconsistent observations of intravenous and oral administrations may be explained by apparent flip-flop kinetics due to irregular absorption of YH-53. Indeed, the plasma concentration–time profile after oral administration shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B seems to exhibit the unexpected double peaks, a moderate extra peak. At present, the precise mechanism expressing the double peak in the plasma concentration–time profile after oral administration of YH-53 is not totally clear, but speculation is that this phenomenon may be caused by irregular absorption due to the enterohepatic recirculation, site-specific absorption, variable gastric emptying, or precipitation/resolubilization in the gastrointestinal tract. Further studies will be needed to clarify the mechanisms of a large difference in the <i>t</i><sub>1/2</sub> value of YH-53 between intravenous and oral administrations. Our finding will help to design and develop the more potent inhibitors with excellent pharmacokinetic profiles.</div><div class="NLM_p last">PF-00835231, which is one of the most advanced 3CL<sup>pro</sup> inhibitors, independently developed by the Pfizer group, has a similar chemical structure as YH-53<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> While YH-53 contains a benzothiazolylketone group as the warhead, PF-00835231 has a hydroxymethylketone at the corresponding position. Both compounds showed similar antiviral activities and low cytotoxicity (CC<sub>50</sub> > 100 μM) and are recognized by MDR-1. Hence, the antiviral activity of both compounds is enhanced under the presence of an MDR-1 inhibitor. Compared with the oral bioavailability of PF-00835231 (1.4%), that of YH-53 (3.6%) is in the same range.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The higher cLogP value of YH-53 (2.37) compared with PF-00835231 (0.75) would be due to the increased hydrophobicity of the benzothiazole unit, which may contribute to the relatively long <i>t</i><sub>1/2</sub> (9.6 h) of YH-53 compared with the <i>t</i><sub>1/2</sub> of PF-00835231 (0.72 h) after oral administration. This relatively high hydrophobicity is also contributing to the high cell penetration propensity of YH-53. These results indicate that YH-53 can be a valuable lead compound with a different warhead group</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, YH-53, our original 3CL<sup>pro</sup> inhibitor of SARS-CoV-1, is a potential lead compound for the development of an anti-COVID-19 drug. YH-53 exhibited not only potent SARS-CoV-2 inhibition in 3CL<sup>pro</sup> and cellular antiviral assays but also a most favorable <i>in vitro</i> ADME profile. <i>In vivo</i> pharmacokinetics and metabolic analyses of YH-53 indicated that the first pass effect with hydrolysis between the P1–P2 amide bond in the liver would be the main reason of the low bioavailability of YH-53 in rats. We believe that these data will strongly facilitate the drug development targeting SARS-CoV-2 3CL<sup>pro</sup>.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> General Part</h3><div class="NLM_p last">The purity of SH-5, YH-53, and YH-71 exceeded 95% by HPLC analysis. The analytical HPLC charts are disclosed in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Preparation of SH-5, YH-53, and YH-71</h3><div class="NLM_p last">SH-5,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> YH-53,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and YH-71<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> were synthesized and purified according to published procedures.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Expression and Purification of SARS-CoV-2 3CL<sup>pro</sup></h3><div class="NLM_p last">For the enzyme inhibition assay, the full-length SARS-CoV-2 3CL<sup>pro</sup>, His-tagged at the C-terminus, was expressed and purified as described previously.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> For crystallization, the SARS-CoV-2 3CL<sup>pro</sup>-encoding pGEX-6P-1 vector was kindly provided by Dr. Rolf Hilgenfeld.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Briefly, <i>E. coli</i> BL21 (DE3) transformed with the plasmid was precultured in 5 mL of LB medium supplied with ampicillin (100 μg/mL) at 37 °C overnight. The incubated culture was inoculated into 1 L of LB medium supplied with 100 μg/mL ampicillin. The cells were induced with 0.5 mM isopropyl-<span class="smallcaps smallerCapital">d</span>-thiogalactoside (IPTG) until the OD<sub>600</sub> reached 0.6–0.8. After incubation for 5 h at 30 °C, the cells were harvested by centrifugation at 2000<i>g</i>, 4 °C for 30 min. The pellets were resuspended in 30 mL of the lysis buffer (20 mM Tris, 150 mM NaCl, pH 7.8) and then lysed by sonication on ice. The lysate was clarified by centrifugation at 10 000<i>g</i> at 4 °C for 1 h. The supernatant was loaded onto a HisPurTM Ni-NTA resin (Thermo Scientific) which was equilibrated with the lysis buffer. The resin was washed with the lysis buffer with 10 mM imidazole to remove nonspecifically binding proteins, followed by elution with imidazole ranging from 20 to 400 mM. The fractions containing the target protein were combined and mixed with PreScission protease (Cytiva) at a molar ratio of 5:1 and dialyzed into the dialysis buffer (20 mM Tris, 150 mM NaCl, 1 mM DTT, pH 7.8) at 4 °C overnight, resulting in the target protein with authentic N- and C-termini. The PreScission-treated 3CL<sup>pro</sup> was applied to connected GST-Accept (Nacalai Tesque Inc.) and HisPurTM Ni-NTA resin to remove the GST-tagged PreScission protease, the His-tag, and the protein with uncleaved His-tag. The His-tag-free 3CL<sup>pro</sup> in the flow-through was subjected to buffer exchange with the stock buffer (20 mM Tris, 150 mM NaCl, 1 mM EDTA, 1 mM DTT, pH 7.8) by using Amicon Ultra 15 centrifugal filters (10 kDa, Merck Millipore) at 3000<i>g</i> at 4 °C. The purified 3CL<sup>pro</sup> (10 mg/mL) was stored at −80 °C and used for crystallization.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> SARS-CoV-2 3CL<sup>pro</sup> Inhibition Assay</h3><div class="NLM_p last">Recombinant His-tagged SARS-CoV-2 3CL<sup>pro</sup> was defrosted and used immediately in the enzyme assays which were performed on a Pherastar FSX plate reader (BMG Labtech, Offenburg, Germany) at 37 °C with an excitation wavelength of 360 nm and an emission wavelength of 460 nm. Black half area 96-well plates with a flat bottom were obtained from Greiner Bio-One (Kremsmünster, Austria). The assay buffer was 50 mM 3-(<i>N</i>-morpholino)propanesulfonic acid (MOPS), pH 7.2 containing 10 mM NaCl, 1 mM EDTA, and 0.01% Triton X-100. A 2.5 mM stock solution of the substrate (Boc-Abu-Tle-Leu-Gln-AMC)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> was prepared in DMSO. Test compounds were dissolved in DMSO to obtain 1.0 mM stock solutions. The substrate was diluted 33-fold with assay buffer and pipetted into a well containing 1 μL of inhibitor solution. This mixture was incubated at 37 °C for 5 min. Upon addition of 1 μL of enzyme solution (0.4 μg/μL His-tagged 3CL<sup>pro</sup> diluted in 14 μL of assay buffer), the measurement was started and recorded for 10 min. The final enzyme concentration was 8 ng/μL of His-tagged 3CL<sup>pro</sup>. The final substrate concentration was 50 μM which approximately corresponds to its <i>K</i><sub>m</sub> value, and the final DMSO concentration was 4%. The product formation rate of the uninhibited control was set at 100%. At least 11 different inhibitor concentrations were tested to determine IC<sub>50</sub> values. The product formation rate during the first 10 min was evaluated and plotted versus the corresponding inhibitor concentrations. IC<sub>50</sub> values were calculated by nonlinear regression. The corresponding <i>K</i><sub>i</sub> values were calculated using the Cheng–Prusoff equation.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> The inhibition type of SH-5 was determined by monitoring the effects of five different inhibitor and substrate concentrations on the product formation rate for 10 min. The inhibition type of SH-5 was determined by monitoring the reactions for 10 min in the presence of five different inhibitor and substrate concentrations. The Lineweaver–Burk plot and the global data fitting for competitive inhibition were performed with GraphPad Prism 8.0.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Crystallization of SARS-CoV-2 3CL<sup>pro</sup>-YH-53 and -SH-5</h3><div class="NLM_p last">The crystallization conditions of the 3CL<sup>pro</sup>-YH-53 complex were initially screened using Crystal Screen 1 and 2 (Hampton Research) and PEGsII (Qiagen) with a Protein Crystallization System 2 (PXS2) at the Structural Biology Research Center, KEK in Japan.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Screening was performed by the sitting-drop vapor-diffusion method with crystallization drops consisting of 0.2 μL protein solutions (4.6 mg/mL 3CL<sup>pro</sup> containing 0.9 mM YH-53) and 0.2 μL screening solutions at 277 K. Crystals of the 3CL<sup>pro</sup>-YH-53 complex were observed after 3 days under the conditions of Crystal Screen 1 #10 (30%(w/v) PEG4000, 0.2 M ammonium acetate, 0.1 M sodium acetate pH 4.6) at 277 K. Crystallization conditions were further optimized, and the final crystallization conditions were 25% (w/v) PEG4000, 0.2 M ammonium acetate, and 0.1 M sodium acetate at pH 4.6. Before diffraction data collection, crystals were cryoprotected in a solution containing 30% (v/v) ethylene glycol, 25% (w/v) PEG4000, 0.14 M ammonium acetate, and 70 mM sodium acetate at pH 4.6 for 20 s. Crystallization conditions of the 3CL<sup>pro</sup>-SH-5 complex were optimized based on the crystallization conditions of the 3CL<sup>pro</sup>-YH-53 complex by the sitting-drop vapor-diffusion method with crystallization drops consisting of 1.2 μL protein solutions (4.6 mg/mL containing 0.9 mM SH-5) and 1.2 μL screening solutions at 277 K by varying the concentration of PEG4000. The final crystallization conditions were 17.5% (w/v) PEG4000, 0.2 M ammonium acetate, and 0.1 M sodium acetate at pH 4.6. Before diffraction data collection, crystals were cryoprotected in a solution containing 30% (v/v) ethylene glycol, 25% (w/v) PEG4000, 0.14 M ammonium acetate, and 70 mM sodium acetate at pH 4.6 for 20 s. X-ray diffraction data were collected at 95 K, using an Eiger X16 M detector on BL-17A of the Photon Factory, KEK (Tsukuba, Japan). Diffraction data were processed and scaled by XDS and XSCALE, respectively.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The phases were determined by the molecular replacement method<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> using the program Phaser with the previously reported model of 3CL<sup>pro</sup> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Y2F">6Y2F</a>) as a search model. Refinement of the crystallographic was performed using PHENIX.refine<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and Coot.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> NMR Interaction Study between SARS-CoV-2 3CL<sup>pro</sup> and YH-53</h3><div class="NLM_p last">For the NMR study, a double-stranded DNA fragment encoding N-terminal NH-tagged SARS-CoV-2 3CL<sup>pro</sup> was synthesized (Thermo Fisher Scientific) with a PreScission protease sequence located between NH-tag and the target protein. The DNA fragment was subjected to a 16 h <i>in vitro</i> translation with stable isotope labeled amino acids for Met-ε1 <sup>1</sup>H,<sup>13</sup>C-labeling under a <sup>2</sup>H-labeling background (Musaibou-kun, Taiyo Nippon Sanso). The resultant reaction solution containing NH-tagged SARS-CoV-2 3CL<sup>pro</sup> was purified for crystallization in the same way as the <i>E. coli</i> expressed SARS-CoV-2 3CL<sup>pro</sup>. YH-53 titration experiments were performed using a Bruker Avance-600 MHz spectrometer equipped with cryogenic triple resonance probes (Bruker Biospin). NMR experiments were performed at 298 K and TOPSPIN (Bruker Biospin) was used to process NMR spectra.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> SARS-CoV-2 Antiviral Assay</h3><div class="NLM_p last">Vero cells were seeded onto 96-well plates at 1.0 × 10<sup>4</sup> cells/well and cultured at 37 °C overnight. Viruses (KNG strain) were used to inoculate cell line at MOI = 0.01. After adsorption at 37 °C for 1 h, the cells were washed with DMEM and with fresh DMEM containing 10% FBS with different concentrations of SH-5, YH-53, and YH-71. Infected cells were cultured at 37 °C for 72 h, and culture supernatants were collected to determine infectious virus titers using TCID<sub>50</sub>. For TCID<sub>50</sub>, viruses were diluted by 10-fold serial dilution from 10 times dilution using DMEM without FBS. The diluted viruses were inoculated into Vero cells and incubated at 37 °C for 4 days. After incubation, the infected cells were fixed with buffered formaldehyde (Nacalai Tesque) and stained with crystal violet. The TCID<sub>50</sub> was calculated by the Spearman and Karber algorithm.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cytotoxic Assay</h3><div class="NLM_p last">Vero cells were seeded onto 96-well plates at 1.0 × 10<sup>4</sup> cells/well and cultured at 37 °C overnight. SH-5, YH-53, and YH-71 were individually added to the cultured Vero cells. After incubation for 24 h, MTT assays were carried out using the MTT Cell Count Kit (Nacalai Tesque) according to manufacturer’s instructions. Absorbance at 570 nm was then measured with the iMark microplate absorbance reader (BioRad).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> SARS-CoV-2 Antiviral Assay with a MDR-1 inhibitor</h3><div class="NLM_p last">VeroE6/TMPRSS2 cells were seeded onto 12-well plates at 1.0 × 10<sup>5</sup> cells/well and cultured at 37 °C overnight. Cells were infected with SARS-CoV-2 virus (JPN/TY/WK-521 strain) at MOI = 0.1. After adsorption at 37 °C for 1 h, the cells were washed with DMEM and fresh DMEM containing 10% FBS with different concentrations of YH-53 and CP-100356. At 24 h postinfection, the culture supernatants were collected and the viral genomes were purified using the MagMax Vrial/Pathogen Nucleic Acid Isolation Kit (Thermo Fisher Scientific). The level of viral genomes was determined by real time RT-PCR with SuperScript III (Thermo Fisher Scientific) and FastStart SYBR Green Master (Roche) using primers specific for <i>S</i> gene (5′-CAATGGTTTAACAGGCACAGG-3′ and 5′-CTCAAGTGTCTGTGGATCACG-3′).</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Rat Plasma Stability</h3><div class="NLM_p last">YH-53 (1 μM) was added into the plasma and incubated at 37 °C for 0, 1, 2, and 6 h. After incubation, the compound concentrations in the plasma were determined by LC/MS/MS (UFLC system coupled to an API 4000 triple quadrupole mass spectrometer and Unison UK-C18 HT column (3.0 μm, 2.0 × 20 mm)).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Plasma Protein Binding</h3><div class="NLM_p last">Human donor-pooled plasma was purchased from Cosmo Bio Co. Ltd. (Tokyo, Japan). Rat plasma used in the study was obtained from Crl:CD(SD) (IGS) male rats (Charles River Laboratories Japan, Inc., Kanagawa, Japan). The plasma protein binding of each compound was determined by the equilibrium dialysis method with HTDialysis Teflon dialysis chambers and cellulose membranes (MWCO 6–8 kDa). Human and rat plasma were mixed with YH-53 (1 μM). Dialysis was conducted against PBS in an 8% CO<sub>2</sub>-supplemented atmosphere at 37 °C for 16–20 h. The concentrations of the compound in both plasma and PBS were determined by LC/MS/MS (UFLC system coupled to a Triple QUAD 5500 (SCIEX, MA) and Unison UK-C18 HT column (3.0 μm, 2.0 × 20 mm)). The unbound fraction in plasma (fu, p) was calculated as the ratio of the peak area of compounds from PBS side to those from the plasma side of the dialysis apparatus.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Kinetic Solubility</h3><div class="NLM_p last">YH-53 solution dissolved in DMSO was added to the second fluid for the disintegration test of the Japanese Pharmacopoeia (JP2nd). After incubation, precipitates were separated by filtration. The solubility was determined by using a UV plate reader (SpectraMax 384plus, Molecure device, CA).</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> CYP Inhibition Assay</h3><div class="NLM_p last">Human liver microsomes were purchased from Sekisui XenoTech, LLC. The microsomes (0.1 mg protein/mL), substrates (tacrine, paclitaxel, tolbutamide, dextromethorphan, and midazolam, respectively), and YH-53 (10 μM) were mixed in phosphate buffer (pH 7.4). The reactions were initiated by adding an NADPH generating system (a mixture of MgCl<sub>2</sub>, β-NADP<sup>+</sup>, glucose-6-phosphate, and glucose-6-phosphate dehydrogenase) to the mixtures before incubation. The mixture was incubated at 37 °C for 10 min, and the reaction was terminated by adding acetonitrile. The activities of CYP1A2, CYP2C8, CYP2C9, CYP2D6, and CYP3A4 were determined by the peaks of 1-hydroxytacrine, 6α-hydroxypaclitaxel, 4-hydroxytolbutamide, dextrorphan, and 1′-hydroxymidazolam, respectively. The analysis was conducted by LC/MS/MS (UFLC system coupled to an API 5000 triple quadrupole mass spectrometer and Unison UK-C18 (3.0 μm, 2.0 × 10 mm)). The activities of test samples were expressed as the percentage of activity remaining compared with a control sample containing no inhibitor. The inhibition ratio (% inhibition) was calculated to the following:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_m001.gif" alt="" id="_i26" /></img></span></div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Principal Component Analysis</h3><div class="NLM_p last">To construct the chemical space, SD (Structure-Data) files of 2034 FDA-approved drugs were downloaded from DrugBank<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (assessed July 2, 2020). In order to distinguish various kinds of protease inhibitors from the other drugs, 15 gliptin group drugs (alogliptin, anagliptin, bisegliptin, carmegliptin, evogliptin, gemigliptin, gosogliptin, linagliptin, omarigliptin, retagliptin, saxagliptin, sitagliptin, teneligriptin, trelagliptin and vildagliptin), 26 pril group (benazepril, benazeprilat, captopril, cilazapril, cilazaprilat, delapril, enalapril, enalaprilat, epicaptopril, fosinoprilat, imidapril, lisinopril, moexipril, moexiprilat, perindopril, perindoprilat, quinapril, quinaprilat, ramipril, ramiprilat, spirapril, tempcapril, trandolapril, trandolaprilat, zofenopril and zofenoprilat), one previr group (bioceprevir) and 8 navir group drugs (amprenavir, darunavir, denavir, fosamprenavir, indinavir, mozenavir, nelfinavir and tipranavir) were extracted. An SD file of YH-53 was also produced by Avogadro.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Fourteen physicochemical properties used as chemistry informer descriptors<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> were calculated from the SD files by RDKit (Open-Source Cheminformatics Software). PCA was carried out by scikit-learn (Open-Source Machine Learning Libraries). The resulting chemical space was presented on the projective plane composed of the first (PC1) and the second (PC2) principal component axes for evaluation.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Ames Test</h3><div class="NLM_p last"><i>Salmonella typhimurium</i> TA98 and TA100 were originally transferred from National Institute of Technology and Evaluation (Tokyo, Japan) and were subsequently maintained in-house. Two Ames Salmonella tester strains were used: TA98, which detects frameshift mutations, and TA100, which detects base pair substitutions. Briefly, each tester strain was preincubated for 20 min both with and without a phenobarbital and 5,6-benzoflavone-induced rat liver S9 fraction (IEDA Trading Corporation, Tokyo, Japan) in duplicate in 96-well plates with six concentrations (1.4–333.3 μg equiv/well) of each compound. Six concentrations of each positive control mutagen (in duplicate) and 12 replicates of vehicle controls (DMSO, FUJIFILM Wako Pure Chemical Corporation, Osaka, Japan) were also used. The positive control without S9 was 2-(2-furyl)-3-(5-nitro-2-furyl) acrylamide (Oriental Yeast Co., Ltd., Tokyo, Japan). The positive control with S9 was 2-aminoanthracene (Oriental Yeast Co., Ltd.). After preincubation, semienriched top agar was added to the 96-well plate and mixed sufficiently, and then the mixtures were overlaid on minimal glucose agar 24-well plates. After the overlaid agar solidified, these plates were stored at 37 °C in an incubator (MIR-553, Sanyo Electric Biomedical Co., Ltd., Osaka, Japan) for 48 h. Cultures were examined for signs of cytotoxicity (decreased background lawn), compound precipitation, and number of mutant colonies. Revertant colonies in each well were counted with an automatic colony counter (Gelcounttm, Oxford Optronix Ltd., Abingdon, United Kingdom). A positive response was defined as the appearance of ≥2-fold revertant colonies for TA100 and ≥3-fold for TA98 over the control revertant colonies. As an additional criterion used, an average of at least nine colonies for TA100 or six colonies for TA98, and concentration–response relationships were also taken into account to consider a compound positive in this assay.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> hERG Assay</h3><div class="NLM_p last">Whole-cell patch-clamp voltage clamp experiments were performed on Chinese hamster ovary cells lines stably expressing the human ether-a-go-go related gene (CHO hERG Duo, B’SYS GmbH, Witterswil, Switzerland). Measurements were performed using a SyncroPatch 384PE (Nanion Technologies, Munich, Germany) incorporated into a Biomek FX pipetting robot (Beckman Coulter, CA), and ×4 hole chips with high resistance (Nanion, NPC R-384 patchclamp 4 × High) were used. The external solution consisted of 10 mM HEPES, 140 mM NaCl, 4 mM KCl, 1 mM MgCl<sub>2</sub>, 2 mM CaCl<sub>2</sub>, and <span class="smallcaps smallerCapital">d</span>-glucose 5 mM and was adjusted to pH 7.4 with NaOH. The internal solution contained 10 mM HEPES, 120 mM KF, 20 mM KCl, 10 mM EGTA and was adjusted to pH 7.2 with KOH. hERG current was measured using a pulse pattern, repeated every 5 s, consisting of a depolarization to 40 mV for 500 ms, followed by a ramp (−1.2 V/s) to −80 mV for 100 ms (holding potential = −80 mV). Peak tail current was measured during the repolarizing ramp. All experiments were performed at room temperature (rt), while the cell suspension was rotated at 4 °C on a Peltier shaker (CellHotel) until use. Concentration response curves for each compound were investigated independently, <i>i.e.</i>, each cell received only one compound, typically increasing concentrations of 1.1, 3.3, 10, and 30 μM in a cumulative fashion with a 5 min interval. To monitor the current stability during the compound period, some cells received vehicle control (0.3∼0.4% (v/v) DMSO) instead. Positive control (0.11, 0.33, 1, and 3 μM of verapamil) and negative control (0.3, 0.3, 0.3, and 0.4% (v/v) DMSO) values were measured in each plate. Valid whole-cell recordings had to meet the following criteria: (i) a seal resistance ≥ 50 MΩ for a multihole (4 cells) patch clamp experiment, (ii) peak current ≥ 0.5 nA, and (iii) run-down or -up during before drug application: within ±10%. or in the DMSO control (whole experimental period): within ±15%.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>In Vitro</i> MicroFlow Assay</h3><div class="NLM_p">The precultured human lymphoblastoid TK-6 cells (DS Pharma Promo Co., Ltd., Osaka, Japan) resuspended in assay culture medium were plated on 96-well plates. Then, for the short-term treatment group with metabolic activation, the cells were treated with test compound and 1% (v/v) + S9 mixture for 3 h followed by a further 21 h incubation in a humid atmosphere at 37 °C with 5% CO<sub>2</sub> (PHC Corporation, Tokyo, Japan) without test compound and S9 mixture. For the continuous treatment group without metabolic activation, the cells were incubated with test compound without S9 mixture for a continuous 24 h in the humid atmosphere at 37 °C with 5% CO<sub>2</sub>. Cyclophosphamide (CP, for treatment with metabolic activation, Shionogi & Co. Ltd. Osaka, Japan) and Mitomycin C (MMC, for treatment without metabolic activation, Kyowa Hakko Kirin Co. Ltd., Tokyo, Japan) were used as concurrent positive controls to ensure the performance of the assay. After treatment, the cells were washed and stained according to the package insert of In Vitro MicroFlow Kit and measured with the BD LSR Fortessa (Becton, Dickinson and Company). Briefly, cells were first stained with a photoactivated dye, Dye A (EMA) and then washed, lysed, and stained with lysis solutions containing RNase, nucleic acid Dye B (SYTOX Green), and counting beads. DNA from apoptotic/necrotic cells with compromised cell membranes was labeled with both EMA and SYTOX Green, which can be distinguished from EMA-negative and SYTOX Green-positive MN. Measurements were performed on 20 000 main nuclei per concentration. If there were less than 10 000 main nuclei per concentration, this was recorded in the “remarks” column. The micronucleus was set at 1/100 to 1/10 of the 2<i>n</i> nuclear fluorescence intensity. The lowest concentration at which precipitation was observed at the end of the treatment was adopted as the highest evaluation concentration. As the same amount of counting beads was added to each well, the nuclei to beads ratio was used to calculate relative population doubling (RPD) as a cytotoxicity measurement. The relative population doubling (RPD) at each concentration were calculated by the following formulas.<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_m002.gif" alt="" id="_i31" /></img></span><span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_m003.gif" alt="" id="_i32" /></img></span></div><div class="NLM_p">Data acquisition and analysis were accomplished with FACSDiva 6.2. The strategy for acquiring flow cytometry data, such as region configuration and gating logic, was as described in the 96-well In Vitro MicroFlow Kits manual. The following parameters were calculated and a test compound was deemed positive if all of the following criteria are met:</div><div class="NLM_p">Parameters:</div><div class="NLM_p">(i) Frequency of micronuclei relative to main nucleus (%MN/%MicroNuclei)</div><div class="NLM_p">(ii) Ratio of micronucleus frequency to simultaneous negative controls (fold MN)</div><div class="NLM_p last">(iii) Cytotoxicity indicators include the EMA positive ratio (fold EMA/fold ethidium monoazide) and the RPD (relative population doubling) for concurrent negative controls after 24 h culture.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> MDR1 Substrate Screening Assay</h3><div class="NLM_p last">Human MDR1-expressing MDCK cells were cultured, and the transcellular transport study was performed. The cells were grown in a transwell 96-well permeable support (pore size 0.4 μm, 0.143 cm<sup>2</sup> surface area) with a polycarbonate membrane (Corning Life Sciences, Lowell, MA). The cells were preincubated with HBSS for 10 min at 37 °C. Subsequently, transcellular transport was initiated by the addition of HBSS either to apical compartments (75 μL) or to basolateral compartments (250 μL) containing 10 μM YH-53. The assay was terminated by separating each assay plate after 1 h. Aliquost (25 μL) from the opposite compartments were mixed with acetonitrile. After centrifugation, the compound concentrations in the supernatant were measured by LC/MS/MS, using a UFLC system coupled to a Triple QUAD 5500 and an Unison UK-C18 HT column (3.0 μm, 2.0 × 20 mm). The apparent permeability of YH-53 in the receiver wells was determined, and the efflux ratio (ER) for the membrane permeability test was calculated using the following equation:<span class="NLM_disp-formula" id="ueq6"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_m004.gif" alt="" id="_i34" /></img></span>where Papp,AtoB and Papp,BtoA represent the apparent permeability in the apical-to-basal direction and the basal-to-apical direction, respectively.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>In Vivo</i> Pharmacokinetic Study of YH-53 in Rats</h3><div class="NLM_p last">An <i>in vivo</i> pharmacokinetic study in rats was carried out as described previously.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> Male Wistar rats were housed three per cage with free access to commercial chow and tap water and were maintained on a 12 h dark/light cycle (08:00–20:00 light) in an air-controlled room (temperature, 24.0 ± 1 °C; humidity, 55 ± 5%). All animal experimental protocols were reviewed and approved by the Committee of Animal Care and Welfare of Kanazawa University. Male Wistar rats (230–250 g, body weight) that fasted overnight were anesthetized by isoflurane. The right jugular veins of rats were cannulated with silicone tubing (100–00N; 0.5 mm I.D., 1.0 mm O.D., Kaneka Medical Products). YH-53 was administrated via the left jugular veins (i.v. administration, 0.1 mg/kg) or orally administrated by gavage (p.o. administration, 0.5 mg/kg) with YH-53 in saline. The rats were moving freely and not anesthetized during the experiment. Blood samples (500 μL) were collected from the cannula into heparinized tubes at designated times up to 7 h. Each blood sample was replaced with an equal volume of saline, and heparinized saline was used to maintain the patency of the catheter. Blood samples were centrifuged at 3000 rpm for 10 min. The resultant plasma was stored at −30 °C until analysis. The plasma concentration of YH-53 was measured using a LC-MS/MS system consisting of an AB-Sciex API 5000 triple quadrupole mass spectrometer (AB SCIEX, Foster City, CA) coupled with a LC-20AD ultrafast liquid chromatography (UFLC) system (Shimadzu Company, Kyoto, Japan). An Agilent ZORBAX Eclipse Plus (C18, 50 × 2.1 mm, 5 μm) was used as the analytical column. A mobile phase composed of 0.1% formic acid (A) and acetonitrile (B) was used at a flow rate of 0.4 mL/min with a gradient elution. The mass transitions (Q1/Q3) of <i>m</i>/<i>z</i> 576.2/290 and 260.2/116.0 were used for YH-53 and propranolol (as an internal standard), respectively. Analyst software version 1.7.2 (AB SCIEX) was used for data analysis. Plasma concentration–time curves of YH-53 were plotted and analyzed. The area under the plasma concentration–time curve from 0 to 7 h (AUC<sub>0–7</sub>) and from 0 to ∞ (AUC<sub>0–∞</sub>) was calculated by the trapezoidal method using noncompartmental analysis with the MOMENT program.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The maximum plasma concentration (<i>C</i><sub>max</sub>) and the time to reach <i>C</i><sub>max</sub> (<i>t</i><sub>max</sub>) were obtained directly from the experimental data. The elimination half-life (<i>t</i><sub>1/2</sub>) was calculated based on the terminal elimination rate constant determined by log–linear regression of the final data points (at least 3). Also, the compartment model analysis was calculated using the numerical analysis program Napp.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Transport Experiments of YH-53 Using Caco-2 Cells</h3><div class="NLM_p last">An <i>in vitro</i> permeability assay using Caco-2 cells was carried out as described previously.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The Caco-2 cell line was obtained from American Type Culture Collection (Rockville, MD). Caco-2 cells were cultured at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub> in air using DMEM supplemented with 10% FBS, 1% <span class="smallcaps smallerCapital">l</span>-glutamine, 1% NEAA, and 5% antibiotic–antimycotic solution. Cells were routinely subcultured at 90% confluency with trypsin-EDTA. For transport studies, Caco-2 cells were plated onto cell culture inserts (3.0 μm pores, 0.9 cm<sup>2</sup> growth area) at a density of 6.4 × 10<sup>4</sup> cells/insert. The culture medium (0.8 mL in the inset and 2.0 mL in the well) was replaced with fresh medium at 5 days after initiation of cell culture and every 48 h thereafter. After 21 days in culture, the Caco-2 monolayer was utilized for the transport experiments. The cell monolayers were preincubated in transport medium (TM, Hanks’ balanced salt solution with 10 mM HEPES, pH 7.4) for 30 min at 37 °C. After preincubation, transepithelial electrical resistance (TEER) of the monolayers was measured routinely before and after each experiment. Then the study was initiated by adding the test drugs (YH-53, metoprolol, or atenolol) to the donor (apical, AP) side and TM to the receiver (basal, BL) side. Metoprolol and atenolol were used as high- and low-permeability model drugs, respectively. Samples were obtained from the receiver side at 30, 60, 90, and 120 min. All experiments were performed at 37 °C. The amount of YH-53 was measured using a LC-MS/MS system. The permeability (apparent permeability coefficient, cm/s) of each drug was calculated according to the following equation:<span class="NLM_disp-formula" id="ueq7"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_m005.gif" alt="" id="_i37" /></img></span>where <i>Q</i> is the amount of compound transported over time <i>t</i> (therefore, d<i>Q</i>/d<i>t</i> is the amount of compound transported within a given time period [μmol/s]). <i>C</i><sub>D</sub> is the initial concentration of compound in the donor compartment (μM), and <i>A</i> is the membrane surface area (cm<sup>2</sup>).</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Metabolic Analysis</h3><div class="NLM_p last">Cryopreserved human hepatocytes were purchased from Thermo Fisher Scientific (Waltham, MA). Cryopreserved rat hepatocytes were obtained from BioIVT (Westbury, NY). The hepatocytes were suspended with InVitroGRO KHB buffer (BioIVT) to prepare 6 × 10<sup>5</sup> viable cells/mL. YH-53 (10 μM) was added into the hepatocytes and incubated for 0 and 2 h at 37 °C in a humidified atmosphere containing 5% CO<sub>2</sub>. Reactions were terminated by the addition of acetonitrile. After centrifugation, aliquots of the supernatant were injected into a LC/UV/MS system for metabolite analysis. The metabolites formed during the incubation were detected at a UV wavelength of 295 nm. The metabolite structure was presumed based on the protonated or deprotonated molecule and the product ion spectrum data.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00665" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28030" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28030" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00665?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00665</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Kinetics, NMR titration, and structure of 3CL<sup>pro</sup>; cytotoxicity and antiviral activity of tested compounds with a MDR-1 inhibitor; computational, in vitro metabolic and in vivo pharmacokinetic data of YH-53 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula string table (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_001.pdf">jm1c00665_si_001.pdf (1.62 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_002.csv">jm1c00665_si_002.csv (0.38 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00665" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13665" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13665" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yoshio Hayashi</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7010-6914" title="Orcid link">https://orcid.org/0000-0002-7010-6914</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#037a6b627a62706b6a43776c7a6268762d62602d6973"><span class="__cf_email__" data-cfemail="077e6f667e66746f6e4773687e666c72296664296d77">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sho Konno</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kiyotaka Kobayashi</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miki Senda</span> - <span class="hlFld-Affiliation affiliation">Structural
Biology Research Center, Institute of Materials Structure Science, High Energy Accelerator Research Organization (KEK), Tsukuba 305-0801, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3863-4503" title="Orcid link">https://orcid.org/0000-0003-3863-4503</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuta Funai</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuta Seki</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ikumi Tamai</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4388-083X" title="Orcid link">https://orcid.org/0000-0003-4388-083X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Schäkel</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Institute, Pharmaceutical & Medicinal Chemistry, University of Bonn, Bonn 53121, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kyousuke Sakata</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, Tokyo University of Pharmacy
and Life Sciences, Hachioji,
Tokyo 192-0392, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thanigaimalai Pillaiyar</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Institute, Pharmaceutical/Medicinal Chemistry, University of Tübingen, Tübingen 72076, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akihiro Taguchi</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atsuhiko Taniguchi</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Department of Medicinal Chemistry, Tokyo University of Pharmacy and Life Sciences, Hachioji, Tokyo, 192-0392, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Gütschow</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Institute, Pharmaceutical & Medicinal Chemistry, University of Bonn, Bonn 53121, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9376-7897" title="Orcid link">https://orcid.org/0000-0002-9376-7897</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christa E. Müller</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Institute, Pharmaceutical & Medicinal Chemistry, University of Bonn, Bonn 53121, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-0013-6624" title="Orcid link">https://orcid.org/0000-0002-0013-6624</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Koh Takeuchi</span> - <span class="hlFld-Affiliation affiliation">Cellular
and Molecular Biotechnology Research Institute, National Institute of Advanced Industrial Science and Technology, Koto, Tokyo 135-0064, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-6227-4627" title="Orcid link">https://orcid.org/0000-0002-6227-4627</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mikako Hirohama</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Medicine, Transborder Medical Research Center, University of Tsukuba, Tsukuba 305-8575, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atsushi Kawaguchi</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Medicine, Transborder Medical Research Center, University of Tsukuba, Tsukuba 305-8575, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masaki Kojima</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, Tokyo University of Pharmacy
and Life Sciences, Hachioji,
Tokyo 192-0392, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toshiya Senda</span> - <span class="hlFld-Affiliation affiliation">Structural
Biology Research Center, Institute of Materials Structure Science, High Energy Accelerator Research Organization (KEK), Tsukuba 305-0801, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yoshiyuki Shirasaka</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Pharmacy, Institute of Medical, Pharmaceutical and Health Sciences, Kanazawa University, Kanazawa 920-1192, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wataru Kamitani</span> - <span class="hlFld-Affiliation affiliation">Department
of Infectious Diseases and Host Defense, Gunma University Graduate School of Medicine, Maebashi 371-8511, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by Japan Agency for Medical Research and Development (AMED) (20fk0108267s0101 to M.K., Y.S., W.K. and Y.H.) This work was partly supported by CREST, Japan Science and Technology Agency (JST) (JP20356709 to T.S., K.T., A.K. and Y.H.) and Grant-in-Aid for Scientific Research (B) (19H03356 to Y.H.) and Grant-in-Aid for Research Activity Start-up (20K22723 to S.K.) from the Japan Society of the Promotion of Science (JSPS), and Basis for Supporting Innovative Drug Discovery and Life Science Research (BINDS) from AMED under Grant Number JP20am0101071 to T.S. (support number 2688).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">PDB codes for SARS-CoV-2 3CL<sup>pro</sup> with bound YH-53 and SH-5 are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E18">7E18</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E19">7E19</a>, respectively.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38117" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38117" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We greatly appreciate Dr. Rolf Hilgenfeld (Univ. of Lübeck) who kindly gave us SARS-CoV-2 3CL<sup>pro</sup>-encoding vector. We are grateful for contributions (enzyme expression and purification, assay establishment, and synthesis of substrate, respectively) by Dominik Thimm, Anke Schiedel, Lan Phuong Vu, Ghazl Al Hamwi, Vigneshwaran Namasivayam, Robin Gedschold, Carina Lemke, Vittoria Lopez, Salahuddin Mirza, Miriam Diett, and Katharina Sylvester, University of Bonn.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AMC</td><td class="NLM_def"><p class="first last">7-amino-4-methylcoumarin</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">3CL<sup>pro</sup></td><td class="NLM_def"><p class="first last">three chymotrypsin-like protease</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiothreitol</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">EDTA</td><td class="NLM_def"><p class="first last">ethylenediaminetetraacetic acid</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">U.S. Food and Drug Administration</p></td></tr><tr><td class="NLM_term">HBSS</td><td class="NLM_def"><p class="first last">Hank’s balanced salt solution</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go related gene</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">MDR</td><td class="NLM_def"><p class="first last">multidrug resistance</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">NADPH</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide phosphate</p></td></tr><tr><td class="NLM_term">NTA</td><td class="NLM_def"><p class="first last">nitrilotriacetic acid</p></td></tr><tr><td class="NLM_term">NEAA</td><td class="NLM_def"><p class="first last">nonessential amino acids</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">polyethylene glycol</p></td></tr><tr><td class="NLM_term">KF</td><td class="NLM_def"><p class="first last">potassium fluoride</p></td></tr><tr><td class="NLM_term">RNase</td><td class="NLM_def"><p class="first last">ribonuclease</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">TCID</td><td class="NLM_def"><p class="first last">tissue culture infectious dose</p></td></tr><tr><td class="NLM_term">TMPRSS2</td><td class="NLM_def"><p class="first last">transmembrane protease, serine 2</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21678" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21678" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span> <span> </span><span class="NLM_article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>382</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2001017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1056%2FNEJMoa2001017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=31978945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslGmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2020&pages=727-733&issue=8&author=N.+Zhuauthor=D.+Zhangauthor=W.+Wangauthor=X.+Liauthor=B.+Yangauthor=J.+Songauthor=X.+Zhaoauthor=B.+Huangauthor=W.+Shiauthor=R.+Luauthor=P.+Niuauthor=F.+Zhanauthor=X.+Maauthor=D.+Wangauthor=W.+Xuauthor=G.+Wuauthor=G.+F.+Gaoauthor=W.+Tan&title=A+Novel+Coronavirus+from+Patients+with+Pneumonia+in+China%2C+2019&doi=10.1056%2FNEJMoa2001017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A novel coronavirus from patients with pneumonia in China, 2019</span></div><div class="casAuthors">Zhu, Na; Zhang, Dingyu; Wang, Wenling; Li, Xingwang; Yang, Bo; Song, Jingdong; Zhao, Xiang; Huang, Baoying; Shi, Weifeng; Lu, Roujian; Niu, Peihua; Zhan, Faxian; Ma, Xuejun; Wang, Dayan; Xu, Wenbo; Wu, Guizhen; Gao, George F.; Tan, Wenjie</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">727-733</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">In Dec. 2019, a cluster of patients with pneumonia of unknown cause was linked to a seafood wholesale market in Wuhan, China.  A previously unknown betacoronavirus was discovered through the use of unbiased sequencing in samples from patients with pneumonia.  Human airway epithelial cells were used to isolate a novel coronavirus, named 2019-nCoV, which formed a clade within the subgenus sarbecovirus, Orthocoronavirinae subfamily.  Different from both MERS-CoV and SARS-CoV, 2019-nCoV is the seventh member of the family of coronaviruses that infect humans.  Enhanced surveillance and further investigation are ongoing.  Complete genome sequences of the three novel coronaviruses were submitted to GISAID (BetaCoV/Wuhan/ IVDC-HB-01/2019, accession ID: EPI_ISL_402119; BetaCoV/Wuhan/IVDC-HB-04/2020, accession ID: EPI_ISL_402120; BetaCoV/Wuhan/IVDC-HB-05/2019, accession ID: EPI_ISL_402121).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibbAn_VqfXbVg90H21EOLACvtfcHk0likvUttjsvI3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslGmsrc%253D&md5=73cc5c839e1e934da69b39537063b7b3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2001017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2001017%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DR.%26aulast%3DNiu%26aufirst%3DP.%26aulast%3DZhan%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DA%2520Novel%2520Coronavirus%2520from%2520Patients%2520with%2520Pneumonia%2520in%2520China%252C%25202019%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D382%26issue%3D8%26spage%3D727%26epage%3D733%26doi%3D10.1056%2FNEJMoa2001017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z. L.</span></span> <span> </span><span class="NLM_article-title">A Pneumonia Outbreak Associated with a New Coronavirus of Probable Bat Origin.</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span> (<span class="NLM_issue">7798</span>),  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2012-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1038%2Fs41586-020-2012-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32015507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=270-273&issue=7798&author=P.+Zhouauthor=X.+L.+Yangauthor=X.+G.+Wangauthor=B.+Huauthor=L.+Zhangauthor=W.+Zhangauthor=H.+R.+Siauthor=Y.+Zhuauthor=B.+Liauthor=C.+L.+Huangauthor=H.+D.+Chenauthor=J.+Chenauthor=Y.+Luoauthor=H.+Guoauthor=R.+Di+Jiangauthor=M.+Q.+Liuauthor=Y.+Chenauthor=X.+R.+Shenauthor=X.+Wangauthor=X.+S.+Zhengauthor=K.+Zhaoauthor=Q.+J.+Chenauthor=F.+Dengauthor=L.+L.+Liuauthor=B.+Yanauthor=F.+X.+Zhanauthor=Y.+Y.+Wangauthor=G.+F.+Xiaoauthor=Z.+L.+Shi&title=A+Pneumonia+Outbreak+Associated+with+a+New+Coronavirus+of+Probable+Bat+Origin.&doi=10.1038%2Fs41586-020-2012-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span></div><div class="casAuthors">Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">270-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Since the outbreak of severe acute respiratory syndrome (SARS) 18 years ago, a large no. of SARS-related coronaviruses (SARSr-CoVs) have been discovered in their natural reservoir host, bats1-4.  Previous studies have shown that some bat SARSr-CoVs have the potential to infect humans5-7.  Here we report the identification and characterization of a new coronavirus (2019-nCoV), which caused an epidemic of acute respiratory syndrome in humans in Wuhan, China.  The epidemic, which started on 12 Dec. 2019, had caused 2,794 lab.-confirmed infections including 80 deaths by 26 Jan. 2020.  Full-length genome sequences were obtained from five patients at an early stage of the outbreak.  The sequences are almost identical and share 79.6% sequence identity to SARS-CoV.  Furthermore, we show that 2019-nCoV is 96% identical at the whole-genome level to a bat coronavirus.  Pairwise protein sequence anal. of seven conserved non-structural proteins domains show that this virus belongs to the species of SARSr-CoV.  In addn., 2019-nCoV virus isolated from the bronchoalveolar lavage fluid of a critically ill patient could be neutralized by sera from several patients.  Notably, we confirmed that 2019-nCoV uses the same cell entry receptor-angiotensin converting enzyme II (ACE2)-as SARS-CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_Wb_iUsbFrVg90H21EOLACvtfcHk0likvUttjsvI3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D&md5=236f17d4d3c7978d72513e5e0258f1b3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2012-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2012-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DX.%2BG.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSi%26aufirst%3DH.%2BR.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.%2BL.%26aulast%3DChen%26aufirst%3DH.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DDi%2BJiang%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DM.%2BQ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DX.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DQ.%2BJ.%26aulast%3DDeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DL.%2BL.%26aulast%3DYan%26aufirst%3DB.%26aulast%3DZhan%26aufirst%3DF.%2BX.%26aulast%3DWang%26aufirst%3DY.%2BY.%26aulast%3DXiao%26aufirst%3DG.%2BF.%26aulast%3DShi%26aufirst%3DZ.%2BL.%26atitle%3DA%2520Pneumonia%2520Outbreak%2520Associated%2520with%2520a%2520New%2520Coronavirus%2520of%2520Probable%2520Bat%2520Origin.%26jtitle%3DNature%26date%3D2020%26volume%3D579%26issue%3D7798%26spage%3D270%26epage%3D273%26doi%3D10.1038%2Fs41586-020-2012-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, E. H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T. T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowling, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span> <span> </span><span class="NLM_article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>382</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1199</span>– <span class="NLM_lpage">1207</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2001316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1056%2FNEJMoa2001316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=31995857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1Whtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2020&pages=1199-1207&issue=13&author=Q.+Liauthor=X.+Guanauthor=P.+Wuauthor=X.+Wangauthor=L.+Zhouauthor=Y.+Tongauthor=R.+Renauthor=K.+S.+M.+Leungauthor=E.+H.+Y.+Lauauthor=J.+Y.+Wongauthor=X.+Xingauthor=N.+Xiangauthor=Y.+Wuauthor=C.+Liauthor=Q.+Chenauthor=D.+Liauthor=T.+Liuauthor=J.+Zhaoauthor=M.+Liuauthor=W.+Tuauthor=C.+Chenauthor=L.+Jinauthor=R.+Yangauthor=Q.+Wangauthor=S.+Zhouauthor=R.+Wangauthor=H.+Liuauthor=Y.+Luoauthor=Y.+Liuauthor=G.+Shaoauthor=H.+Liauthor=Z.+Taoauthor=Y.+Yangauthor=Z.+Dengauthor=B.+Liuauthor=Z.+Maauthor=Y.+Zhangauthor=G.+Shiauthor=T.+T.+Y.+Lamauthor=J.+T.+Wuauthor=G.+F.+Gaoauthor=B.+J.+Cowlingauthor=B.+Yangauthor=G.+M.+Leungauthor=Z.+Feng&title=Early+Transmission+Dynamics+in+Wuhan%2C+China%2C+of+Novel+Coronavirus%E2%80%93Infected+Pneumonia&doi=10.1056%2FNEJMoa2001316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Early transmission dynamics in Wuhan, China, of novel coronavirus-infected pneumonia</span></div><div class="casAuthors">Li, Qun; Guan, Xuhua; Wu, Peng; Wang, Xiaoye; Zhou, Lei; Tong, Yeqing; Ren, Ruiqi; Leung, Kathy S. M.; Lau, Eric H. Y.; Wong, Jessica Y.; Xing, Xuesen; Xiang, Nijuan; Wu, Yang; Li, Chao; Chen, Qi; Li, Dan; Liu, Tian; Zhao, Jing; Liu, Man; Tu, Wenxiao; Chen, Chuding; Jin, Lianmei; Yang, Rui; Wang, Qi; Zhou, Suhua; Wang, Rui; Liu, Hui; Luo, Yinbo; Liu, Yuan; Shao, Ge; Li, Huan; Tao, Zhongfa; Yang, Yang; Deng, Zhiqiang; Liu, Boxi; Ma, Zhitao; Zhang, Yanping; Shi, Guoqing; Lam, Tommy T. Y.; Wu, Joseph T.; Gao, George F.; Cowling, Benjamin J.; Yang, Bo; Leung, Gabriel M.; Feng, Zijian</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1199-1207</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">The initial cases of novel coronavirus (2019-nCoV)-infected pneumonia (NCIP) occurred in Wuhan, Hubei Province, China, in Dec. 2019 and Jan. 2020.  We analyzed data on the 1st 425 confirmed cases in Wuhan to det. the epidemiol. characteristics of NCIP.  We collected information on demog. characteristics, exposure history, and illness timelines of lab.-confirmed cases of NCIP that had been reported by Jan. 22, 2020.  We described characteristics of the cases and estd. the key epidemiol. time-delay distributions.  In the early period of exponential growth, we estd. the epidemic doubling time and the basic reproductive no.  Among the 1st 425 patients with confirmed NCIP, the median age was 59 yr and 56% were male.  The majority of cases (55%) with onset before Jan. 1, 2020, were linked to the Huanan Seafood Wholesale Market, as compared with 8.6% of the subsequent cases.  The mean incubation period was 5.2 days, with the 95th percentile of the distribution at 12.5 days.  In its early stages, the epidemic doubled in size every 7.4 days.  With a mean serial interval of 7.5 days, the basic reproductive no. was estd. to be 2.2.  On the basis of this information, there is evidence that human-to-human transmission has occurred among close contacts since the middle of Dec. 2019.  Considerable efforts to reduce transmission will be required to control outbreaks if similar dynamics apply elsewhere.  Measures to prevent or reduce transmission should be implemented in populations at risk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD4bpM4aGXN7Vg90H21EOLACvtfcHk0ljGP8fS0Gs93Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1Whtrw%253D&md5=d82c8e8a173f6ab879d332f0abb7228e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2001316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2001316%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DGuan%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DLeung%26aufirst%3DK.%2BS.%2BM.%26aulast%3DLau%26aufirst%3DE.%2BH.%2BY.%26aulast%3DWong%26aufirst%3DJ.%2BY.%26aulast%3DXing%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DTu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DG.%26aulast%3DLam%26aufirst%3DT.%2BT.%2BY.%26aulast%3DWu%26aufirst%3DJ.%2BT.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DCowling%26aufirst%3DB.%2BJ.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DLeung%26aufirst%3DG.%2BM.%26aulast%3DFeng%26aufirst%3DZ.%26atitle%3DEarly%2520Transmission%2520Dynamics%2520in%2520Wuhan%252C%2520China%252C%2520of%2520Novel%2520Coronavirus%25E2%2580%2593Infected%2520Pneumonia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D382%26issue%3D13%26spage%3D1199%26epage%3D1207%26doi%3D10.1056%2FNEJMoa2001316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokudo, N.</span></span> <span> </span><span class="NLM_article-title">Sustaining Containment of COVID-19: Global Sharing for Pandemic Response</span>. <i>Glob. Heal. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.35772/ghm.2020.01040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.35772%2Fghm.2020.01040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33330776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BB3szosl2lsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2020&pages=53-55&author=H.+Mitsuyaauthor=N.+Kokudo&title=Sustaining+Containment+of+COVID-19%3A+Global+Sharing+for+Pandemic+Response&doi=10.35772%2Fghm.2020.01040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Sustaining containment of COVID-19: global sharing for pandemic response</span></div><div class="casAuthors">Mitsuya Hiroaki; Kokudo Norihiro</div><div class="citationInfo"><span class="NLM_cas:title">Global health & medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">53-55</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) outbreak, caused by SARS-CoV-2, has rapidly escalated into a global pandemic.  One of our significant concerns is that we have no data as to whether people, who acquired immunity against this deadly virus and recovered from COVID-19, are protected from further infections with the same virus.  Moreover, we have no data as to whether this pandemic will persist in our societies and continue vexing us for long periods of time.  Implementing science-based response strategy is essential to sustain containment of COVID-19 globally.  Rapidly sharing scientific information means providing real-time research data and relevant findings.  As an international academic journal, Global Health & Medicine publishes this special issue entitled "GHM Special Topic: COVID-19".  It includes a range of articles describing COVID-19 based on frontline data from Japan, China, the United States, Italy, the United Kingdom, and other countries and areas worldwide.  Our hope is that the rapid publication and sharing of information contribute, in whichever possible way, to this global fight against COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUcmPsNTJyj0HdcurJClFcfW6udTcc2eayfN8JxCKOArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3szosl2lsg%253D%253D&md5=ebe9eeb276bf878b93ac2cf54f284007</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.35772%2Fghm.2020.01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.35772%252Fghm.2020.01040%26sid%3Dliteratum%253Aachs%26aulast%3DMitsuya%26aufirst%3DH.%26aulast%3DKokudo%26aufirst%3DN.%26atitle%3DSustaining%2520Containment%2520of%2520COVID-19%253A%2520Global%2520Sharing%2520for%2520Pandemic%2520Response%26jtitle%3DGlob.%2520Heal.%2520Med.%26date%3D2020%26volume%3D2%26spage%3D53%26epage%3D55%26doi%3D10.35772%2Fghm.2020.01040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">A New Coronavirus Associated with Human Respiratory Disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.+M.+Chenauthor=W.+Wangauthor=Z.+G.+Songauthor=Y.+Huauthor=Z.+W.+Taoauthor=J.+H.+Tianauthor=Y.+Y.+Peiauthor=M.+L.+Yuanauthor=Y.+L.+Zhangauthor=F.+H.+Daiauthor=Y.+Liuauthor=Q.+M.+Wangauthor=J.+J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.+Z.+Zhang&title=A+New+Coronavirus+Associated+with+Human+Respiratory+Disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0ljGP8fS0Gs93Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.%2BM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.%2BG.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.%2BW.%26aulast%3DTian%26aufirst%3DJ.%2BH.%26aulast%3DPei%26aufirst%3DY.%2BY.%26aulast%3DYuan%26aufirst%3DM.%2BL.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DDai%26aufirst%3DF.%2BH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%2BM.%26aulast%3DZheng%26aufirst%3DJ.%2BJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.%2BZ.%26atitle%3DA%2520New%2520Coronavirus%2520Associated%2520with%2520Human%2520Respiratory%2520Disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. H.</span></span> <span> </span><span class="NLM_article-title">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6595</span>– <span class="NLM_lpage">6628</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01461</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01461" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6595-6628&author=T.+Pillaiyarauthor=M.+Manickamauthor=V.+Namasivayamauthor=Y.+Hayashiauthor=S.+H.+Jung&title=An+Overview+of+Severe+Acute+Respiratory+Syndrome-Coronavirus+%28SARS-CoV%29+3CL+Protease+Inhibitors%3A+Peptidomimetics+and+Small+Molecule+Chemotherapy&doi=10.1021%2Facs.jmedchem.5b01461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Manickam, Manoj; Namasivayam, Vigneshwaran; Hayashi, Yoshio; Jung, Sang-Hun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6595-6628</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome (SARS) is caused by a newly emerged coronavirus that infected more than 8000 individuals and resulted in more than 800 (10-15%) fatalities in 2003.  The causative agent of SARS has been identified as a novel human coronavirus (SARS-CoV), and its viral protease, SARS-CoV 3CLpro, has been shown to be essential for replication and has hence been recognized as a potent drug target for SARS infection.  Currently, there is no effective treatment for this epidemic despite the intensive research that has been undertaken since 2003 (over 3500 publications).  This perspective focuses on the status of various efficacious anti-SARS-CoV 3CLpro chemotherapies discovered during the last 12 years (2003-2015) from all sources, including lab. synthetic methods, natural products, and virtual screening.  We describe here mainly peptidomimetic and small mol. inhibitors of SARS-CoV 3CLpro.  Attempts have been made to provide a complete description of the structural features and binding modes of these inhibitors under many conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB_JCXe-m0RLVg90H21EOLACvtfcHk0lirSy0uUwZWHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D&md5=6ce37076204280f64a7f362633fcee1a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01461%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DManickam%26aufirst%3DM.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DJung%26aufirst%3DS.%2BH.%26atitle%3DAn%2520Overview%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome-Coronavirus%2520%2528SARS-CoV%2529%25203CL%2520Protease%2520Inhibitors%253A%2520Peptidomimetics%2520and%2520Small%2520Molecule%2520Chemotherapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6595%26epage%3D6628%26doi%3D10.1021%2Facs.jmedchem.5b01461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preiser, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Werf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodt, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabenau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panning, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolesnikova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouchier, R. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burguière, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinatl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escriou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grywna, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramme, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuguerra, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickerts, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klenk, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterhaus, A. D. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, H. W.</span></span> <span> </span><span class="NLM_article-title">Identification of a Novel Coronavirus in Patients with Severe Acute Respiratory Syndrome</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1967</span>– <span class="NLM_lpage">1976</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa030747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1056%2FNEJMoa030747" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=12690091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslajurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2003&pages=1967-1976&author=C.+Drostenauthor=S.+G%C3%BCntherauthor=W.+Preiserauthor=S.+van+der%0AWerfauthor=H.-R.+Brodtauthor=S.+Beckerauthor=H.+Rabenauauthor=M.+Panningauthor=L.+Kolesnikovaauthor=R.+A.+M.+Fouchierauthor=A.+Bergerauthor=A.-M.+Burgui%C3%A8reauthor=J.+Cinatlauthor=M.+Eickmannauthor=N.+Escriouauthor=K.+Grywnaauthor=S.+Krammeauthor=J.-C.+Manuguerraauthor=S.+M%C3%BCllerauthor=V.+Rickertsauthor=M.+St%C3%BCrmerauthor=S.+Viethauthor=H.-D.+Klenkauthor=A.+D.+M.+E.+Osterhausauthor=H.+Schmitzauthor=H.+W.+Doerr&title=Identification+of+a+Novel+Coronavirus+in+Patients+with+Severe+Acute+Respiratory+Syndrome&doi=10.1056%2FNEJMoa030747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span></div><div class="casAuthors">Drosten, Christian; Guenther, Stephan; Preiser, Wolfgang; van der Werf, Sylvie; Brodt, Hans-Reinhard; Becker, Stephan; Rabenau, Holger; Panning, Marcus; Kolesnikova, Larissa; Fouchier, Ron A. M.; Berger, Annemarie; Burguiere, Ana-Maria; Cinatl, Jindrich; Eickmann, Markus; Escriou, Nicolas; Grywna, Klaus; Kramme, Stefanie; Manuguerra, Jean-Claude; Mueller, Stefanie; Rickerts, Volker; Stuermer, Martin; Vieth, Simon; Klenk, Hans-Dieter; Osterhaus, Albert D. M. E.; Schmitz, Herbert; Doerr, Hans Wilheim</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1967-1976</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">The severe acute respiratory syndrome (SARS) has recently been identified as a new clin. entity.  SARS is thought to be caused by an unknown infectious agent.  Clin. specimens from patients with SARS were searched for unknown viruses with the use of cell cultures and mol. techniques.  A novel coronavirus was identified in patients with SARS.  The virus was isolated in cell culture, and a sequence 300 nucleotides in length was obtained by a PCR (PCR)-based random-amplification procedure.  Genetic characterization indicated that the virus is only distantly related to known coronaviruses (identical in 50 to 60% of the nucleotide sequence).  On the basis of the obtained sequence, conventional and real-time PCR assays for specific and sensitive detection of the novel virus were established.  Virus was detected in a variety of clin. specimens from patients with SARS but not in controls.  High concns. of viral RNA of up to 100 million mols. per mL were found in sputum.  Viral RNA was also detected at extremely low concns. in plasma during the acute phase and in feces during the late convalescent phase.  Infected patients showed seroconversion on the Vero cells in which the virus was isolated.  The novel coronavirus might have a role in causing SARS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqN8uHpwOG07Vg90H21EOLACvtfcHk0lirSy0uUwZWHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslajurw%253D&md5=3838a7a01273f688e53dc86088bfa9db</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa030747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa030747%26sid%3Dliteratum%253Aachs%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DPreiser%26aufirst%3DW.%26aulast%3Dvan%2Bder%2BWerf%26aufirst%3DS.%26aulast%3DBrodt%26aufirst%3DH.-R.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRabenau%26aufirst%3DH.%26aulast%3DPanning%26aufirst%3DM.%26aulast%3DKolesnikova%26aufirst%3DL.%26aulast%3DFouchier%26aufirst%3DR.%2BA.%2BM.%26aulast%3DBerger%26aufirst%3DA.%26aulast%3DBurgui%25C3%25A8re%26aufirst%3DA.-M.%26aulast%3DCinatl%26aufirst%3DJ.%26aulast%3DEickmann%26aufirst%3DM.%26aulast%3DEscriou%26aufirst%3DN.%26aulast%3DGrywna%26aufirst%3DK.%26aulast%3DKramme%26aufirst%3DS.%26aulast%3DManuguerra%26aufirst%3DJ.-C.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DRickerts%26aufirst%3DV.%26aulast%3DSt%25C3%25BCrmer%26aufirst%3DM.%26aulast%3DVieth%26aufirst%3DS.%26aulast%3DKlenk%26aufirst%3DH.-D.%26aulast%3DOsterhaus%26aufirst%3DA.%2BD.%2BM.%2BE.%26aulast%3DSchmitz%26aufirst%3DH.%26aulast%3DDoerr%26aufirst%3DH.%2BW.%26atitle%3DIdentification%2520of%2520a%2520Novel%2520Coronavirus%2520in%2520Patients%2520with%2520Severe%2520Acute%2520Respiratory%2520Syndrome%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2003%26volume%3D348%26spage%3D1967%26epage%3D1976%26doi%3D10.1056%2FNEJMoa030747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byun, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Identification of Antiviral Drug Candidates against SARS-CoV-2 from FDA-Approved Drugs</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">e00819</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1128/AAC.00819-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FAAC.00819-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32366720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslGgsLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&pages=e00819-20&author=S.+Jeonauthor=M.+Koauthor=J.+Leeauthor=I.+Choiauthor=S.+Y.+Byunauthor=S.+Parkauthor=D.+Shumauthor=S.+Kim&title=Identification+of+Antiviral+Drug+Candidates+against+SARS-CoV-2+from+FDA-Approved+Drugs&doi=10.1128%2FAAC.00819-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</span></div><div class="casAuthors">Jeon, Sangeun; Ko, Meehyun; Lee, Jihye; Choi, Inhee; Byun, Soo Young; Park, Soonju; Shum, David; Kim, Seungtaek</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e00819</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Drug repositioning is the only feasible option to immediately address the COVID-19 global challenge.  We screened a panel of 48 FDA-approved drugs against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which were preselected by an assay of SARS-CoV.  We identified 24 potential antiviral drug candidates against SARS-CoV-2 infection.  Some drug candidates showed very low 50% inhibitory concns. (IC50s), and in particular, two FDA-approved drugs-niclosamide and ciclesonide-were notable in some respects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI441p4IDYQLVg90H21EOLACvtfcHk0ljNYIx25gJ-Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslGgsLvN&md5=f3e0bb95ed274876838945ef5952cc54</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FAAC.00819-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00819-20%26sid%3Dliteratum%253Aachs%26aulast%3DJeon%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DI.%26aulast%3DByun%26aufirst%3DS.%2BY.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520Antiviral%2520Drug%2520Candidates%2520against%2520SARS-CoV-2%2520from%2520FDA-Approved%2520Drugs%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26spage%3De00819%26epage%3D20%26doi%3D10.1128%2FAAC.00819-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jan, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juang, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shie, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chein, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span> <span> </span><span class="NLM_article-title">Identification of Existing Pharmaceuticals and Herbal Medicines as Inhibitors of SARS-CoV-2 Infection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">e2021579118</span>, <span class="refDoi"> DOI: 10.1073/pnas.2021579118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1073%2Fpnas.2021579118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33452205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjtV2lsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2021&pages=e2021579118&author=J.+T.+Janauthor=T.+J.+R.+Chengauthor=Y.+P.+Juangauthor=H.+H.+Maauthor=Y.+T.+Wuauthor=W.+B.+Yangauthor=C.+W.+Chengauthor=X.+Chenauthor=T.+H.+Chouauthor=J.+J.+Shieauthor=W.+C.+Chengauthor=R.+J.+Cheinauthor=S.+S.+Maoauthor=P.+H.+Liangauthor=C.+Maauthor=S.+C.+Hungauthor=C.+H.+Wong&title=Identification+of+Existing+Pharmaceuticals+and+Herbal+Medicines+as+Inhibitors+of+SARS-CoV-2+Infection&doi=10.1073%2Fpnas.2021579118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of existing pharmaceuticals and herbal medicines as inhibitors of SARS-CoV-2 infection</span></div><div class="casAuthors">Jan, Jia-Tsrong; Cheng, Ting-Jen Rachel; Juang, Yu-Pu; Ma, Hsiu-Hua; Wu, Ying-Ta; Yang, Wen-Bin; Cheng, Cheng-Wei; Chen, Xiaorui; Chou, Ting-Hung; Shie, Jiun-Jie; Cheng, Wei-Chieh; Chein, Rong-Jie; Mao, Shi-Shan; Liang, Pi-Hui; Ma, Che; Hung, Shang-Cheng; Wong, Chi-Huey</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e2021579118</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The outbreak of COVID-19 caused by SARS-CoV-2 has resulted in more than 50 million confirmed cases and over 1 million deaths worldwide as of Nov. 2020.  Currently, there are no effective antivirals approved by the Food and Drug Administration to contain this pandemic except the antiviral agent remdesivir.  In addn., the trimeric spike protein on the viral surface is highly glycosylated and almost 200,000 variants with mutations at more than 1,000 positions in its 1,273 amino acid sequence were reported, posing a major challenge in the development of antibodies and vaccines.  It is therefore urgently needed to have alternative and timely treatments for the disease.  In this study, we used a cell-based infection assay to screen more than 3,000 agents used in humans and animals, including 2,855 small mols. and 190 traditional herbal medicines, and identified 15 active small mols. in concns. ranging from 0.1 nM to 50 μM.  Two enzymic assays, along with mol. modeling, were then developed to confirm those targeting the virus 3CL protease and the RNA-dependent RNA polymerase.  Several water exts. of herbal medicines were active in the cell-based assay and could be further developed as plant-derived anti-SARS-CoV-2 agents.  Some of the active compds. identified in the screen were further tested in vivo, and it was found that mefloquine, nelfinavir, and exts. of Ganoderma lucidum (RF3), Perilla frutescens, and Mentha haplocalyx were effective in a challenge study using hamsters as disease model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDZ_HLt-yZjLVg90H21EOLACvtfcHk0ljNYIx25gJ-Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjtV2lsb8%253D&md5=c8a34583707cea0f0fc984c9a173d5f4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2021579118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2021579118%26sid%3Dliteratum%253Aachs%26aulast%3DJan%26aufirst%3DJ.%2BT.%26aulast%3DCheng%26aufirst%3DT.%2BJ.%2BR.%26aulast%3DJuang%26aufirst%3DY.%2BP.%26aulast%3DMa%26aufirst%3DH.%2BH.%26aulast%3DWu%26aufirst%3DY.%2BT.%26aulast%3DYang%26aufirst%3DW.%2BB.%26aulast%3DCheng%26aufirst%3DC.%2BW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChou%26aufirst%3DT.%2BH.%26aulast%3DShie%26aufirst%3DJ.%2BJ.%26aulast%3DCheng%26aufirst%3DW.%2BC.%26aulast%3DChein%26aufirst%3DR.%2BJ.%26aulast%3DMao%26aufirst%3DS.%2BS.%26aulast%3DLiang%26aufirst%3DP.%2BH.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHung%26aufirst%3DS.%2BC.%26aulast%3DWong%26aufirst%3DC.%2BH.%26atitle%3DIdentification%2520of%2520Existing%2520Pharmaceuticals%2520and%2520Herbal%2520Medicines%2520as%2520Inhibitors%2520of%2520SARS-CoV-2%2520Infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2021%26volume%3D118%26spage%3De2021579118%26doi%3D10.1073%2Fpnas.2021579118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamarthi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupte, A.</span></span> <span> </span><span class="NLM_article-title">The Journey of Remdesivir: From Ebola to COVID-19</span>. <i>Drugs Context</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.7573/dic.2020-4-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.7573%2Fdic.2020-4-14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1-9&author=J.+Pardoauthor=A.+M.+Shuklaauthor=G.+Chamarthiauthor=A.+Gupte&title=The+Journey+of+Remdesivir%3A+From+Ebola+to+COVID-19&doi=10.7573%2Fdic.2020-4-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.7573%2Fdic.2020-4-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7573%252Fdic.2020-4-14%26sid%3Dliteratum%253Aachs%26aulast%3DPardo%26aufirst%3DJ.%26aulast%3DShukla%26aufirst%3DA.%2BM.%26aulast%3DChamarthi%26aufirst%3DG.%26aulast%3DGupte%26aufirst%3DA.%26atitle%3DThe%2520Journey%2520of%2520Remdesivir%253A%2520From%2520Ebola%2520to%2520COVID-19%26jtitle%3DDrugs%2520Context%26date%3D2020%26volume%3D9%26spage%3D1%26epage%3D9%26doi%3D10.7573%2Fdic.2020-4-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">U.S. Food & Drug Administration</span>.  <i>FDA approves
first treatment for COVID-19</i>. <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19" class="extLink">https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19</a> (accessed 2021-01-21).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+%26+Drug+Administration.+FDA+approves%0Afirst+treatment+for+COVID-19.+https%3A%2F%2Fwww.fda.gov%2Fnews-events%2Fpress-announcements%2Ffda-approves-first-treatment-covid-19+%28accessed+2021-01-21%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DFDA%2520approves%250Afirst%2520treatment%2520for%2520COVID-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of SARS-CoV-2 Main Protease Provides a Basis for Design of Improved a-Ketoamide Inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+Structure+of+SARS-CoV-2+Main+Protease+Provides+a+Basis+for+Design+of+Improved+a-Ketoamide+Inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0ljNYIx25gJ-Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520SARS-CoV-2%2520Main%2520Protease%2520Provides%2520a%2520Basis%2520for%2520Design%2520of%2520Improved%2520a-Ketoamide%2520Inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Antiviral Drug Candidates Targeting the SARS-CoV-2 Main Protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.+M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.+K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-Based+Design+of+Antiviral+Drug+Candidates+Targeting+the+SARS-CoV-2+Main+Protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0lhdCKWTF75Jxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.%2BM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%2BK.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-Based%2520Design%2520of%2520Antiviral%2520Drug%2520Candidates%2520Targeting%2520the%2520SARS-CoV-2%2520Main%2520Protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meenakshisundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span> <span> </span><span class="NLM_article-title">Recent Discovery and Development of Inhibitors Targeting Coronaviruses</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">688</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.drudis.2020.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32006468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFKgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=668-688&author=T.+Pillaiyarauthor=S.+Meenakshisundaramauthor=M.+Manickam&title=Recent+Discovery+and+Development+of+Inhibitors+Targeting+Coronaviruses&doi=10.1016%2Fj.drudis.2020.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Recent discovery and development of inhibitors targeting coronaviruses</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Meenakshisundaram, Sangeetha; Manickam, Manoj</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">668-688</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Human coronaviruses (CoVs) are enveloped viruses with a pos.-sense single-stranded RNA genome.  Currently, six human CoVs have been reported including human coronavirus 229E (HCoV-229E), OC43 (HCoV-OC43), NL63 (HCoV-NL63), HKU1 (HCoV-HKU1), severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV), and MiddleEast respiratory syndrome (MERS) coronavirus (MERS-CoV).  They cause moderate to severe respiratory and intestinal infections in humans.  In this review, we focus on recent advances in the research and development of small-mol. anti-human coronavirus therapies targeting different stages of the CoV life cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGaJIxrfUjnLVg90H21EOLACvtfcHk0lhdCKWTF75Jxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFKgtro%253D&md5=08faec7e1ec7df3a3f6d4c4e34fe0807</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DMeenakshisundaram%26aufirst%3DS.%26aulast%3DManickam%26aufirst%3DM.%26atitle%3DRecent%2520Discovery%2520and%2520Development%2520of%2520Inhibitors%2520Targeting%2520Coronaviruses%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D668%26epage%3D688%26doi%3D10.1016%2Fj.drudis.2020.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">From SARS to MERS: Crystallographic Studies on Coronaviral Proteases Enable Antiviral Drug Design</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">4085</span>– <span class="NLM_lpage">4096</span>, <span class="refDoi"> DOI: 10.1111/febs.12936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1111%2Ffebs.12936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=25039866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2014&pages=4085-4096&author=R.+Hilgenfeld&title=From+SARS+to+MERS%3A+Crystallographic+Studies+on+Coronaviral+Proteases+Enable+Antiviral+Drug+Design&doi=10.1111%2Ffebs.12936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">From SARS to MERS: crystallographic studies on coronaviral proteases enable antiviral drug design</span></div><div class="casAuthors">Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4085-4096</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Here, the author focuses on the important contributions that macromol. crystallog. has made over the past 12 yr to elucidating structures and mechanisms of the essential proteases of coronaviruses, the main protease (Mpro) and the papain-like protease (PLpro).  The role of x-ray crystallog. in structure-assisted drug discovery against these targets is discussed.  Aspects dealt with in this review include the emergence of the SARS coronavirus in 2002-2003 and of the MERS coronavirus 10 yr later and the origins of these viruses.  The crystal structure of the free SARS coronavirus Mpro and its dependence on pH is discussed, as are efforts to design inhibitors on the basis of these structures.  The mechanism of maturation of the enzyme from the viral polyprotein is still a matter of debate.  The crystal structure of the SARS coronavirus PLpro and its complex with ubiquitin is also discussed, as is its ortholog from MERS coronavirus.  Efforts at predictive structure-based inhibitor development for bat coronavirus Mpros to increase the preparedness against zoonotic transmission to man are described as well.  The paper closes with a brief discussion of structure-based discovery of antivirals in an academic setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy_Y5ir6KZ8LVg90H21EOLACvtfcHk0lhdCKWTF75Jxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbI&md5=6a9330976a5c3a13dd4bf7bed14665ce</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12936%26sid%3Dliteratum%253Aachs%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DFrom%2520SARS%2520to%2520MERS%253A%2520Crystallographic%2520Studies%2520on%2520Coronaviral%2520Proteases%2520Enable%2520Antiviral%2520Drug%2520Design%26jtitle%3DFEBS%2520J.%26date%3D2014%26volume%3D281%26spage%3D4085%26epage%3D4096%26doi%3D10.1111%2Ffebs.12936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishii, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Autoprocessing Mechanism of Severe Acute Respiratory Syndrome Coronavirus 3C-like Protease (SARS-CoV 3CLpro) from Its Polyproteins</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">2002</span>– <span class="NLM_lpage">2013</span>, <span class="refDoi"> DOI: 10.1111/febs.12222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1111%2Ffebs.12222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23452147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsFWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=2002-2013&author=T.+Muramatsuauthor=Y.+T.+Kimauthor=W.+Nishiiauthor=T.+Teradaauthor=M.+Shirouzuauthor=S.+Yokoyama&title=Autoprocessing+Mechanism+of+Severe+Acute+Respiratory+Syndrome+Coronavirus+3C-like+Protease+%28SARS-CoV+3CLpro%29+from+Its+Polyproteins&doi=10.1111%2Ffebs.12222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Autoprocessing mechanism of severe acute respiratory syndrome coronavirus 3C-like protease (SARS-CoV 3CLpro) from its polyproteins</span></div><div class="casAuthors">Muramatsu, Tomonari; Kim, Yong-Tae; Nishii, Wataru; Terada, Takaho; Shirouzu, Mikako; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2002-2013</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Like many other RNA viruses, severe acute respiratory syndrome coronavirus (SARS-CoV) produces polyproteins contg. several non-structural proteins, which are then processed by the viral proteases.  These proteases often exist within the polyproteins, and are excised by their own proteolytic activity (autoprocessing').  It is important to investigate the autoprocessing mechanism of these proteases from the point of view of anti-SARS-CoV drug design.  In this paper, we describe a new method for investigating the autoprocessing mechanism of the main protease (Mpro), which is also called the 3C-like protease (3CLpro).  Using our method, we measured the activities, under the same conditions, of the mature form and pro-forms with the N-terminal pro-sequence, the C-terminal pro-sequence or both pro-sequences, toward the pro-form with both N- and C-terminal pro-sequences.  The data indicate that the pro-forms of the enzyme have proteolytic activity, and are stimulated by the same proteolytic activity.  The stimulation occurs in two steps, with approx. eightfold stimulation by N-terminal cleavage, approx. fourfold stimulation by C-terminal cleavage, and 23-fold stimulation by the cleavage of both termini, compared to the pro-form with both the N- and C-terminal pro-sequences.  Such cleavage mainly occurs in a trans manner; i.e. the pro-form dimer cleaves the monomeric form.  The stimulation by N-terminal pro-sequence removal is due to the cis (intra-dimer and inter-protomer) effect of formation of the new N-terminus, whereas that by C-terminal cleavage is due to removal of its trans (inter-dimer) inhibitory effect.  A numerical simulation of the maturation pathway is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2_Wx5y0FDxLVg90H21EOLACvtfcHk0ljs2BeGuwytlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsFWguro%253D&md5=20a698c60419d73fdb9921ec371371f0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12222%26sid%3Dliteratum%253Aachs%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DY.%2BT.%26aulast%3DNishii%26aufirst%3DW.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DAutoprocessing%2520Mechanism%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520Coronavirus%25203C-like%2520Protease%2520%2528SARS-CoV%25203CLpro%2529%2520from%2520Its%2520Polyproteins%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D2002%26epage%3D2013%26doi%3D10.1111%2Ffebs.12222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">28134</span>– <span class="NLM_lpage">28140</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.095851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1074%2Fjbc.M109.095851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=20489209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVOnsLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=28134-28140&author=C.+Liauthor=Y.+Qiauthor=X.+Tengauthor=Z.+Yangauthor=P.+Weiauthor=C.+Zhangauthor=L.+Tanauthor=L.+Zhouauthor=Y.+Liuauthor=L.+Lai&title=Maturation+Mechanism+of+Severe+Acute+Respiratory+Syndrome+%28SARS%29+Coronavirus+3C-like+Proteinase&doi=10.1074%2Fjbc.M109.095851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase</span></div><div class="casAuthors">Li, Chunmei; Qi, Yifei; Teng, Xin; Yang, Zongchang; Wei, Ping; Zhang, Changsheng; Tan, Lei; Zhou, Lu; Liu, Ying; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">28134-28140</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The 3C-like proteinase (3CLpro) of the severe acute respiratory syndrome (SARS) coronavirus plays a vital role in virus maturation and is proposed to be a key target for drug design against SARS.  Various in vitro studies revealed that only the dimer of the matured 3CLpro is active.  However, as the internally encoded 3CLpro gets matured from the replicase polyprotein by autolytic cleavage at both the N-terminal and the C-terminal flanking sites, it is unclear whether the polyprotein also needs to dimerize first for its autocleavage reaction.  We constructed a large protein contg. the cyan fluorescent protein (C), the N-terminal flanking substrate peptide of SARS 3CLpro (XX), SARS 3CLpro (3CLP), and the yellow fluorescent protein (Y) to study the autoprocessing of 3CLpro using fluorescence resonance energy transfer.  In contrast to the matured 3CLpro, the polyprotein, as well as the one-step digested product, 3CLP-Y-His, were shown to be monomeric in gel filtration and analytic ultracentrifuge anal.  However, dimers can still be induced and detected when incubating these large proteins with a substrate analog compd. in both chem. crosslinking expts. and analytic ultracentrifuge anal.  We also measured enzyme activity under different enzyme concns. and found a clear tendency of substrate-induced dimer formation.  Based on these discoveries, we conclude that substrate-induced dimerization is essential for the activity of SARS-3CLpro in the polyprotein, and a modified model for the 3CLpro maturation process was proposed.  As many viral proteases undergo a similar maturation process, this model might be generally applicable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos2gN4o9DHBrVg90H21EOLACvtfcHk0ljs2BeGuwytlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVOnsLjM&md5=669fba32d04912c7e084570f98d7c1c7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.095851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.095851%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DMaturation%2520Mechanism%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520%2528SARS%2529%2520Coronavirus%25203C-like%2520Proteinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D28134%26epage%3D28140%26doi%3D10.1074%2Fjbc.M109.095851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotouhi-Ardakani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lytvyn, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachance, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulea, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménard, R.</span></span> <span> </span><span class="NLM_article-title">The Papain-Like Protease from the Severe Acute Respiratory Syndrome Coronavirus Is a Deubiquitinating Enzyme</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">15199</span>– <span class="NLM_lpage">15208</span>, <span class="refDoi"> DOI: 10.1128/JVI.79.24.15199-15208.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FJVI.79.24.15199-15208.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=16306591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XhslOitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=15199-15208&author=H.+A.+Lindnerauthor=N.+Fotouhi-Ardakaniauthor=V.+Lytvynauthor=P.+Lachanceauthor=T.+Suleaauthor=R.+M%C3%A9nard&title=The+Papain-Like+Protease+from+the+Severe+Acute+Respiratory+Syndrome+Coronavirus+Is+a+Deubiquitinating+Enzyme&doi=10.1128%2FJVI.79.24.15199-15208.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The papain-like protease from the severe acute respiratory syndrome coronavirus is a deubiquitinating enzyme</span></div><div class="casAuthors">Lindner, Holger A.; Fotouhi-Ardakani, Nasser; Lytvyn, Viktoria; Lachance, Paule; Sulea, Traian; Menard, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15199-15208</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The severe acute respiratory syndrome coronavirus papain-like protease (SARS-CoV PLpro) is involved in the processing of the viral polyprotein and, thereby, contributes to the biogenesis of the virus replication complex.  Structural bioinformatics has revealed a relationship for the SARS-CoV PLpro to herpesvirus-assocd. ubiquitin-specific protease (HAUSP), a ubiquitin-specific protease, indicating potential deubiquitinating activity in addn. to its function in polyprotein processing.  In order to confirm this prediction, we overexpressed and purified SARS-CoV PLpro (amino acids [aa]1507 to 1858) from Escherichia coli.  The purified enzyme hydrolyzed ubiquitin-7-amino-4-methylcoumarin (Ub-AMC), a general deubiquitinating enzyme substrate, with a catalytic efficiency of 13,100 M-1 s-1, 220-fold more efficiently than the small synthetic peptide substrate Z-LRGG-AMC, which incorporates the C-terminal four residues of ubiquitin.  In addn., SARS-CoV PLpro was inhibited by the specific deubiquitinating enzyme inhibitor ubiquitin aldehyde, with an inhibition const. of 210 nM.  The purified SARS-CoV PLpro disassembles branched polyubiquitin chains with lengths of two to seven (Ub2-7) or four (Ub4) units, which involves isopeptide bond cleavage.  SARS-CoV PLpro processing activity was also detected against a protein fused to the C terminus of the ubiquitin-like modifier ISG15, both in vitro using the purified enzyme and in HeLa cells by coexpression with SARS-CoV PLpro (aa 1198 to 2009).  These results clearly establish that SARS-CoV PLpro is a deubiquitinating enzyme, thereby confirming our earlier prediction.  This unexpected activity for a coronavirus papain-like protease suggests a novel viral strategy to modulate the host cell ubiquitination machinery to its advantage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1m1bqW9GpybVg90H21EOLACvtfcHk0ljs2BeGuwytlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhslOitw%253D%253D&md5=dff5e5e3d3b2fc8838413a8551480383</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.24.15199-15208.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.24.15199-15208.2005%26sid%3Dliteratum%253Aachs%26aulast%3DLindner%26aufirst%3DH.%2BA.%26aulast%3DFotouhi-Ardakani%26aufirst%3DN.%26aulast%3DLytvyn%26aufirst%3DV.%26aulast%3DLachance%26aufirst%3DP.%26aulast%3DSulea%26aufirst%3DT.%26aulast%3DM%25C3%25A9nard%26aufirst%3DR.%26atitle%3DThe%2520Papain-Like%2520Protease%2520from%2520the%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520Coronavirus%2520Is%2520a%2520Deubiquitinating%2520Enzyme%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D15199%26epage%3D15208%26doi%3D10.1128%2FJVI.79.24.15199-15208.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barretto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jukneliene, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">The Papain-Like Protease of Severe Acute Respiratory Syndrome Coronavirus Has Deubiquitinating Activity</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">15189</span>– <span class="NLM_lpage">15198</span>, <span class="refDoi"> DOI: 10.1128/JVI.79.24.15189-15198.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FJVI.79.24.15189-15198.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=16306590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XhslOitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=15189-15198&author=N.+Barrettoauthor=D.+Juknelieneauthor=K.+Ratiaauthor=Z.+Chenauthor=A.+D.+Mesecarauthor=S.+C.+Baker&title=The+Papain-Like+Protease+of+Severe+Acute+Respiratory+Syndrome+Coronavirus+Has+Deubiquitinating+Activity&doi=10.1128%2FJVI.79.24.15189-15198.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The papain-like protease of severe acute respiratory syndrome coronavirus has deubiquitinating activity</span></div><div class="casAuthors">Barretto, Naina; Jukneliene, Dalia; Ratia, Kiira; Chen, Zhongbin; Mesecar, Andrew D.; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15189-15198</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Replication of the genomic RNA of severe acute respiratory syndrome coronavirus (SARS-CoV) is mediated by replicase polyproteins that are processed by two viral proteases, papain-like protease (PLpro) and 3C-like protease (3CLpro).  Previously, we showed that SARS-CoV PLpro processes the replicase polyprotein at three conserved cleavage sites.  Here, we report the identification and characterization of a 316-amino-acid catalytic core domain of PLpro that can efficiently cleave replicase substrates in trans-cleavage assays and peptide substrates in fluorescent resonance energy transfer-based protease assays.  We performed bioinformatics anal. on 16 papain-like protease domains from nine different coronaviruses and identified a putative catalytic triad (Cys1651-His1812-Asp1826) and zinc-binding site.  Mutagenesis studies revealed that Asp1826 and the four cysteine residues involved in zinc binding are essential for SARS-CoV PLpro activity.  Mol. modeling of SARS-CoV PLpro suggested that this catalytic core may also have deubiquitinating activity.  We tested this hypothesis by measuring the deubiquitinating activity of PLpro by two independent assays.  SARS CoV-PLpro hydrolyzed both diubiquitin and ubiquitin-7-amino-4-methylcoumarin (AMC) substrates, and hydrolysis of ubiquitin-AMC is approx. 180-fold more efficient than hydrolysis of a peptide substrate that mimics the PLpro replicase recognition sequence.  To investigate the crit. determinants recognized by PLpro, we performed site-directed mutagenesis on the P6 to P2' residues at each of the three PLpro cleavage sites.  We found that PLpro recognizes the consensus cleavage sequence LXGG, which is also the consensus sequence recognized by cellular deubiquitinating enzymes.  This similarity in the substrate recognition sites should be considered during the development of SARS-CoV PLpro inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiQGj9bzOcILVg90H21EOLACvtfcHk0li9oDTp86InYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhslOitg%253D%253D&md5=d9e028b3addd30d53050758db977d207</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.24.15189-15198.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.24.15189-15198.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBarretto%26aufirst%3DN.%26aulast%3DJukneliene%26aufirst%3DD.%26aulast%3DRatia%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DThe%2520Papain-Like%2520Protease%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520Coronavirus%2520Has%2520Deubiquitinating%2520Activity%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D15189%26epage%3D15198%26doi%3D10.1128%2FJVI.79.24.15189-15198.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hegyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span> <span> </span><span class="NLM_article-title">Conservation of Substrate Specificities among Coronavirus Main Proteases</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1099/0022-1317-83-3-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1099%2F0022-1317-83-3-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=11842254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BD387ktV2ntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2002&pages=595-599&author=A.+Hegyiauthor=J.+Ziebuhr&title=Conservation+of+Substrate+Specificities+among+Coronavirus+Main+Proteases&doi=10.1099%2F0022-1317-83-3-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation of substrate specificities among coronavirus main proteases</span></div><div class="casAuthors">Hegyi Annette; Ziebuhr John</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of general virology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">Pt 3</span>),
    <span class="NLM_cas:pages">595-599</span>
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    </div><div class="casAbstract">The key enzyme in coronavirus replicase polyprotein processing is the coronavirus main protease, 3CL(pro).  The substrate specificities of five coronavirus main proteases, including the prototypic enzymes from the coronavirus groups I, II and III, were characterized.  Recombinant main proteases of human coronavirus (HCoV), transmissible gastroenteritis virus (TGEV), feline infectious peritonitis virus, avian infectious bronchitis virus and mouse hepatitis virus (MHV) were tested in peptide-based trans-cleavage assays.  The determination of relative rate constants for a set of corresponding HCoV, TGEV and MHV 3CL(pro) cleavage sites revealed a conserved ranking of these sites.  Furthermore, a synthetic peptide representing the N-terminal HCoV 3CL(pro) cleavage site was shown to be effectively hydrolysed by noncognate main proteases.  The data show that the differential cleavage kinetics of sites within pp1a/pp1ab are a conserved feature of coronavirus main proteases and lead us to predict similar processing kinetics for the replicase polyproteins of all coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTF5bW4pZ-I-QHqQV89nF-2fW6udTcc2eZLDfEjebY3Obntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD387ktV2ntg%253D%253D&md5=7c6aa15224a074bca51ea82d12f2f827</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1099%2F0022-1317-83-3-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F0022-1317-83-3-595%26sid%3Dliteratum%253Aachs%26aulast%3DHegyi%26aufirst%3DA.%26aulast%3DZiebuhr%26aufirst%3DJ.%26atitle%3DConservation%2520of%2520Substrate%2520Specificities%2520among%2520Coronavirus%2520Main%2520Proteases%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2002%26volume%3D83%26spage%3D595%26epage%3D599%26doi%3D10.1099%2F0022-1317-83-3-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easwaran, M.</span></span> <span> </span><span class="NLM_article-title">The Recent Outbreaks of Human Coronaviruses: A Medicinal Chemistry Perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1002/med.21724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fmed.21724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32852058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVagt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=72-135&author=T.+Pillaiyarauthor=L.+L.+Wendtauthor=M.+Manickamauthor=M.+Easwaran&title=The+Recent+Outbreaks+of+Human+Coronaviruses%3A+A+Medicinal+Chemistry+Perspective&doi=10.1002%2Fmed.21724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The recent outbreaks of human coronaviruses: A medicinal chemistry perspective</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Wendt, Lukas L.; Manickam, Manoj; Easwaran, Maheswaran</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-135</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Coronaviruses (CoVs) infect both humans and animals.  In humans, CoVs can cause respiratory, kidney, heart, brain, and intestinal infections that can range from mild to lethal.  Since the start of the 21st century, three β-coronaviruses have crossed the species barrier to infect humans: severe-acute respiratory syndrome (SARS)-CoV-1, Middle East respiratory syndrome (MERS)-CoV, and SARS-CoV-2 (2019-nCoV).  These viruses are dangerous and can easily be transmitted from human to human.  Therefore, the development of anticoronaviral therapies is urgently needed.  However, to date, no approved vaccines or drugs against CoV infections are available.  In this review, we focus on the medicinal chem. efforts toward the development of antiviral agents against SARS-CoV-1, MERS-CoV, SARS-CoV-2, targeting biochem. events important for viral replication and its life cycle.  These targets include the spike glycoprotein and its host-receptors for viral entry, proteases that are essential for cleaving polyproteins to produce functional proteins, and RNA-dependent RNA polymerase for viral RNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowH3Dpqna3Q7Vg90H21EOLACvtfcHk0li9oDTp86InYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVagt7Y%253D&md5=efe56354f1182ce8c16cede75e04842b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fmed.21724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21724%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DWendt%26aufirst%3DL.%2BL.%26aulast%3DManickam%26aufirst%3DM.%26aulast%3DEaswaran%26aufirst%3DM.%26atitle%3DThe%2520Recent%2520Outbreaks%2520of%2520Human%2520Coronaviruses%253A%2520A%2520Medicinal%2520Chemistry%2520Perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D41%26spage%3D72%26epage%3D135%26doi%3D10.1002%2Fmed.21724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sydnes, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Glutamic Acid and Glutamine Peptides Possessing a Trifluoromethyl Ketone Group as SARS-CoV 3CL Protease Inhibitors</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8601</span>– <span class="NLM_lpage">8609</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2006.06.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.tet.2006.06.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32287416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVSmtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=8601-8609&author=M.+O.+Sydnesauthor=Y.+Hayashiauthor=V.+K.+Sharmaauthor=T.+Hamadaauthor=U.+Bachaauthor=J.+Barrilaauthor=E.+Freireauthor=Y.+Kiso&title=Synthesis+of+Glutamic+Acid+and+Glutamine+Peptides+Possessing+a+Trifluoromethyl+Ketone+Group+as+SARS-CoV+3CL+Protease+Inhibitors&doi=10.1016%2Fj.tet.2006.06.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of glutamic acid and glutamine peptides possessing a trifluoromethyl ketone group as SARS-CoV 3CL protease inhibitors</span></div><div class="casAuthors">Sydnes, Magne O.; Hayashi, Yoshio; Sharma, Vinay K.; Hamada, Takashi; Bacha, Usman; Barrila, Jennifer; Freire, Ernesto; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">8601-8609</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Trifluoromethyl-β-amino alc. I was synthesized in five steps starting from Cbz-L-Glu-OH where the key step involved the introduction of the trifluoromethyl (CF3) group to oxazolidinone II (Cbz = benzyloxycarbonyl), resulting in the formation of silyl ether III.  Compd. I was then converted into four tri- and tetraglutamic acid and glutamine peptides possessing a CF3-ketone group that exhibited inhibitory activity against severe acute respiratory syndrome coronavirus protease (SARS-CoV 3CLpro).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZcF2qvPRx9rVg90H21EOLACvtfcHk0li9oDTp86InYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVSmtbY%253D&md5=fcab218b800eac06f32827f6fdac23ce</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2006.06.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2006.06.052%26sid%3Dliteratum%253Aachs%26aulast%3DSydnes%26aufirst%3DM.%2BO.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DSharma%26aufirst%3DV.%2BK.%26aulast%3DHamada%26aufirst%3DT.%26aulast%3DBacha%26aufirst%3DU.%26aulast%3DBarrila%26aufirst%3DJ.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DSynthesis%2520of%2520Glutamic%2520Acid%2520and%2520Glutamine%2520Peptides%2520Possessing%2520a%2520Trifluoromethyl%2520Ketone%2520Group%2520as%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%26jtitle%3DTetrahedron%26date%3D2006%26volume%3D62%26spage%3D8601%26epage%3D8609%26doi%3D10.1016%2Fj.tet.2006.06.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bacha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelli, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mario Amzel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Development of Broad-Spectrum Halomethyl Ketone Inhibitors against Coronavirus Main Protease 3CLpro</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2008.00679.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1111%2Fj.1747-0285.2008.00679.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=18611220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslymu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2008&pages=34-49&author=U.+Bachaauthor=J.+Barrilaauthor=S.+B.+Gabelliauthor=Y.+Kisoauthor=L.+Mario+Amzelauthor=E.+Freire&title=Development+of+Broad-Spectrum+Halomethyl+Ketone+Inhibitors+against+Coronavirus+Main+Protease+3CLpro&doi=10.1111%2Fj.1747-0285.2008.00679.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Development of broad-spectrum halomethyl ketone inhibitors against coronavirus main protease 3CLpro</span></div><div class="casAuthors">Bacha, Usman; Barrila, Jennifer; Gabelli, Sandra B.; Kiso, Yoshiaki; Amzel, L. Mario; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-49</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Coronaviruses comprise a large group of RNA viruses with diverse host specificity.  The emergence of highly pathogenic strains like the SARS coronavirus (SARS-Co-V), and the discovery of two new coronaviruses, NL-63 and HKU1, corroborates the high rate of mutation and recombination that have enabled them to cross species barriers and infect novel hosts.  For that reason, the development of broad-spectrum antivirals that are effective against several members of this family is highly desirable.  This goal can be accomplished by designing inhibitors against a target, such as the main protease 3CLpro (Mpro), which is highly conserved among all coronaviruses.  Here 3CLpro derived from the SARS-Co-V was used as the primary target to identify a new class of inhibitors contg. a halomethyl ketone warhead.  The compds. are highly potent against SARS 3CLpro with Ki's as low as 300 nM.  The crystal structure of the complex of one of the compds. with 3CLpro indicates that this inhibitor forms a thioether linkage between the halomethyl carbon of the warhead and the catalytic Cys 145.  Furthermore, Structure Activity Relationship (SAR) studies of these compds. have led to the identification of a pharmacophore that accurately defines the essential mol. features required for the high affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIBzjnE5XmbVg90H21EOLACvtfcHk0li7tmoLRTR84Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslymu7s%253D&md5=db8382f357a5be0fffe340b113664fd5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2008.00679.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2008.00679.x%26sid%3Dliteratum%253Aachs%26aulast%3DBacha%26aufirst%3DU.%26aulast%3DBarrila%26aufirst%3DJ.%26aulast%3DGabelli%26aufirst%3DS.%2BB.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DMario%2BAmzel%26aufirst%3DL.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DDevelopment%2520of%2520Broad-Spectrum%2520Halomethyl%2520Ketone%2520Inhibitors%2520against%2520Coronavirus%2520Main%2520Protease%25203CLpro%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2008%26volume%3D72%26spage%3D34%26epage%3D49%26doi%3D10.1111%2Fj.1747-0285.2008.00679.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Regnier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">New Developments for the Design, Synthesis and Biological Evaluation of Potent SARS-CoV 3CLpro Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2727</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.bmcl.2009.03.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=19362479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltl2rs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2722-2727&author=T.+Regnierauthor=D.+Sarmaauthor=K.+Hidakaauthor=U.+Bachaauthor=E.+Freireauthor=Y.+Hayashiauthor=Y.+Kiso&title=New+Developments+for+the+Design%2C+Synthesis+and+Biological+Evaluation+of+Potent+SARS-CoV+3CLpro+Inhibitors&doi=10.1016%2Fj.bmcl.2009.03.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">New developments for the design, synthesis and biological evaluation of potent SARS-CoV 3CLpro inhibitors</span></div><div class="casAuthors">Regnier, Thomas; Sarma, Diganta; Hidaka, Koushi; Bacha, Usman; Freire, Ernesto; Hayashi, Yoshio; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2722-2727</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of trifluoromethyl, benzothiazolyl or thiazolyl ketone-contg. peptidic compds. as SARS-CoV 3CLpro protease inhibitors were developed and their potency was evaluated by in vitro protease inhibitory assays.  Three candidates had encouraging results for the development of new anti-SARS compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9aRtN4lboCrVg90H21EOLACvtfcHk0li7tmoLRTR84Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltl2rs7o%253D&md5=37d3aac3723a2a8e6417600f24c62e8b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.118%26sid%3Dliteratum%253Aachs%26aulast%3DRegnier%26aufirst%3DT.%26aulast%3DSarma%26aufirst%3DD.%26aulast%3DHidaka%26aufirst%3DK.%26aulast%3DBacha%26aufirst%3DU.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DNew%2520Developments%2520for%2520the%2520Design%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Potent%2520SARS-CoV%25203CLpro%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2722%26epage%3D2727%26doi%3D10.1016%2Fj.bmcl.2009.03.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanigaimalai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakiuchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of New Tripeptide-Type SARS-CoV 3CL Protease Inhibitors Containing an Electrophilic Arylketone Moiety</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.bmc.2012.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23245752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCkurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=412-424&author=S.+Konnoauthor=P.+Thanigaimalaiauthor=T.+Yamamotoauthor=K.+Nakadaauthor=R.+Kakiuchiauthor=K.+Takayamaauthor=Y.+Yamazakiauthor=F.+Yakushijiauthor=K.+Akajiauthor=Y.+Kisoauthor=Y.+Kawasakiauthor=S.+E.+Chenauthor=E.+Freireauthor=Y.+Hayashi&title=Design+and+Synthesis+of+New+Tripeptide-Type+SARS-CoV+3CL+Protease+Inhibitors+Containing+an+Electrophilic+Arylketone+Moiety&doi=10.1016%2Fj.bmc.2012.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of new tripeptide-type SARS-CoV 3CL protease inhibitors containing an electrophilic arylketone moiety</span></div><div class="casAuthors">Konno, Sho; Thanigaimalai, Pillaiyar; Yamamoto, Takehito; Nakada, Kiyohiko; Kakiuchi, Rie; Takayama, Kentaro; Yamazaki, Yuri; Yakushiji, Fumika; Akaji, Kenichi; Kiso, Yoshiaki; Kawasaki, Yuko; Chen, Shen-En; Freire, Ernesto; Hayashi, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">412-424</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We describe here the design, synthesis and biol. evaluation of a series of mols. toward the development of novel peptidomimetic inhibitors of SARS-CoV 3CLpro.  A docking study involving binding between the initial lead compd. (I) and the SARS-CoV 3CLpro motivated the replacement of a thiazole with a benzothiazole unit as a warhead moiety at the P1' site.  This modification led to the identification of more potent derivs., including (II) (R1 = R2 = H; R1 = OMe, R2 = H; R1 = H, R2 = OMe; R1 = NMe2, H2 = H; R1 = H, R2 = NMe2), with IC50 or Ki values in the submicromolar to nanomolar range.  In particular, compds. II (R1 = R2 = H; R1 = H, R2 = NMe2) exhibited the most potent inhibitory activities, with Ki values of 4.1 and 3.1 nM, resp.  The peptidomimetic compds. identified through this process are attractive leads for the development of potential therapeutic agents against SARS.  The structural requirements of the peptidomimetics with potent inhibitory activities against SARS-CoV 3CLpro may be summarized as follows: (i) the presence of a benzothiazole warhead at the S1'-position; (ii) hydrogen bonding capabilities at the cyclic lactam of the S1-site; (iii) appropriate stereochem. and hydrophobic moiety size at the S2-site and (iv) a unique folding conformation assumed by the phenoxyacetyl moiety at the S4-site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlvN_nxqEIZLVg90H21EOLACvtfcHk0li7tmoLRTR84Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCkurvN&md5=9aee15f25e1a6276b8973da835472e65</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DKonno%26aufirst%3DS.%26aulast%3DThanigaimalai%26aufirst%3DP.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DNakada%26aufirst%3DK.%26aulast%3DKakiuchi%26aufirst%3DR.%26aulast%3DTakayama%26aufirst%3DK.%26aulast%3DYamazaki%26aufirst%3DY.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%2BE.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520New%2520Tripeptide-Type%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%2520Containing%2520an%2520Electrophilic%2520Arylketone%2520Moiety%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D412%26epage%3D424%26doi%3D10.1016%2Fj.bmc.2012.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thanigaimalai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koiwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naser-Tavakolian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schön, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of Novel Dipeptide-Type SARS-CoV 3CL Protease Inhibitors: Structure-Activity Relationship Study</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2013.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.ejmech.2013.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23747811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFemt7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2013&pages=436-447&author=P.+Thanigaimalaiauthor=S.+Konnoauthor=T.+Yamamotoauthor=Y.+Koiwaiauthor=A.+Taguchiauthor=K.+Takayamaauthor=F.+Yakushijiauthor=K.+Akajiauthor=Y.+Kisoauthor=Y.+Kawasakiauthor=S.+E.+Chenauthor=A.+Naser-Tavakolianauthor=A.+Sch%C3%B6nauthor=E.+Freireauthor=Y.+Hayashi&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+Novel+Dipeptide-Type+SARS-CoV+3CL+Protease+Inhibitors%3A+Structure-Activity+Relationship+Study&doi=10.1016%2Fj.ejmech.2013.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and biological evaluation of novel dipeptide-type SARS-CoV 3CL protease inhibitors: Structure-activity relationship study</span></div><div class="casAuthors">Thanigaimalai, Pillaiyar; Konno, Sho; Yamamoto, Takehito; Koiwai, Yuji; Taguchi, Akihiro; Takayama, Kentaro; Yakushiji, Fumika; Akaji, Kenichi; Kiso, Yoshiaki; Kawasaki, Yuko; Chen, Shen-En; Naser-Tavakolian, Aurash; Schon, Arne; Freire, Ernesto; Hayashi, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">436-447</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">This work describes the design, synthesis, and evaluation of low-mol. wt. peptidic SARS-CoV 3CL protease inhibitors.  The inhibitors were designed based on the potent tripeptidic Z-Val-Leu-Ala(pyrrolidone-3-yl)-2-benzothiazole (I) (Ki = 4.1 nM), in which the P3 valine unit was substituted with a variety of distinct moieties.  The resulting series of dipeptide-type inhibitors displayed moderate to good inhibitory activities against 3CLpro.  In particular, compds. (II) (R1 = OMe, R2 = H and R1 = H, R2 = OMe) exhibited good inhibitory activities with Ki values of 0.39 and 0.33 μM, resp.  These low-mol. wt. compds. are attractive leads for the further development of potent peptidomimetic inhibitors with pharmaceutical profiles.  Docking studies were performed to model the binding interaction of the compd. II (R1 = OMe, R2 = H) with the SARS-CoV 3CL protease.  The preliminary SAR study of the peptidomimetic compds. with potent inhibitory activities revealed several structural features that boosted the inhibitory activity: (i) a benzothiazole warhead at the S1' position, (ii) a γ-lactam unit at the S1-position, (iii) an appropriately hydrophobic leucine moiety at the S2-position, and (iv) a hydrogen bond between the N-arylglycine unit and a backbone hydrogen bond donor at the S3-position.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-wamiD_-5LVg90H21EOLACvtfcHk0ljikDObD8SavQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFemt7nP&md5=622466d52d5a7def121ee16b9ac4423b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2013.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2013.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DThanigaimalai%26aufirst%3DP.%26aulast%3DKonno%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DKoiwai%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DA.%26aulast%3DTakayama%26aufirst%3DK.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%2BE.%26aulast%3DNaser-Tavakolian%26aufirst%3DA.%26aulast%3DSch%25C3%25B6n%26aufirst%3DA.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520Novel%2520Dipeptide-Type%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%253A%2520Structure-Activity%2520Relationship%2520Study%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D65%26spage%3D436%26epage%3D447%26doi%3D10.1016%2Fj.ejmech.2013.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thanigaimalai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koiwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naser-Tavakolian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schön, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span> <span> </span><span class="NLM_article-title">Development of Potent Dipeptide-Type SARS-CoV 3CL Protease Inhibitors with Novel P3 Scaffolds: Design, Synthesis, Biological Evaluation, and Docking Studies</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">372</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2013.07.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.ejmech.2013.07.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23994330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ShsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2013&pages=372-384&author=P.+Thanigaimalaiauthor=S.+Konnoauthor=T.+Yamamotoauthor=Y.+Koiwaiauthor=A.+Taguchiauthor=K.+Takayamaauthor=F.+Yakushijiauthor=K.+Akajiauthor=S.+E.+Chenauthor=A.+Naser-Tavakolianauthor=A.+Sch%C3%B6nauthor=E.+Freireauthor=Y.+Hayashi&title=Development+of+Potent+Dipeptide-Type+SARS-CoV+3CL+Protease+Inhibitors+with+Novel+P3+Scaffolds%3A+Design%2C+Synthesis%2C+Biological+Evaluation%2C+and+Docking+Studies&doi=10.1016%2Fj.ejmech.2013.07.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of potent dipeptide-type SARS-CoV 3CL protease inhibitors with novel P3 scaffolds: Design, synthesis, biological evaluation, and docking studies</span></div><div class="casAuthors">Thanigaimalai, Pillaiyar; Konno, Sho; Yamamoto, Takehito; Koiwai, Yuji; Taguchi, Akihiro; Takayama, Kentaro; Yakushiji, Fumika; Akaji, Kenichi; Chen, Shen-En; Naser-Tavakolian, Aurash; Schon, Arne; Freire, Ernesto; Hayashi, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">372-384</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">We report the design and synthesis of a series of dipeptide-type inhibitors with novel P3 scaffolds that display potent inhibitory activity against SARS-CoV 3CLpro.  A docking study involving binding between the dipeptidic lead compd. (I) and 3CLpro suggested the modification of a structurally flexible P3 N-(3-methoxyphenyl)glycine with various rigid P3 moieties in I.  The modifications led to the identification of several potent derivs. with the inhibitory activities (Ki or IC50) in the submicromolar to nanomolar range.  Compd. (II) , in particular, displayed the most potent inhibitory activity, with a Ki value of 0.006 μM.  This potency was 65-fold higher than the potency of the lead compd. I (Ki = 0.39 μM).  In addn., the Ki value of II was in very good agreement with the binding affinity (16 nM) obsd. in isothermal titrn. calorimetry (ITC).  A SAR study around the P3 group in the lead 4 led to the identification of a rigid indole-2-carbonyl unit as one of the best P3 moieties (III).  Further optimization showed that a methoxy substitution at the 4-position on the indole unit was highly favorable for enhancing the inhibitory potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeJOG03zeobLVg90H21EOLACvtfcHk0ljikDObD8SavQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ShsrzL&md5=28055f45719f5220358fac55d015d2ff</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2013.07.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2013.07.037%26sid%3Dliteratum%253Aachs%26aulast%3DThanigaimalai%26aufirst%3DP.%26aulast%3DKonno%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DKoiwai%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DA.%26aulast%3DTakayama%26aufirst%3DK.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%2BE.%26aulast%3DNaser-Tavakolian%26aufirst%3DA.%26aulast%3DSch%25C3%25B6n%26aufirst%3DA.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520Potent%2520Dipeptide-Type%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%2520with%2520Novel%2520P3%2520Scaffolds%253A%2520Design%252C%2520Synthesis%252C%2520Biological%2520Evaluation%252C%2520and%2520Docking%2520Studies%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D68%26spage%3D372%26epage%3D384%26doi%3D10.1016%2Fj.ejmech.2013.07.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breidenbach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäkel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Hamwi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gedschold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vielmuth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zyulina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodem, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gütschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Targeting the Main Protease of SARS-CoV-2: From the Establishment of High Throughput Screening to the Design of Tailored Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10423</span>– <span class="NLM_lpage">10429</span>, <span class="refDoi"> DOI: 10.1002/anie.202016961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fanie.202016961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnt1Oht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2021&pages=10423-10429&author=J.+Breidenbachauthor=C.+Lemkeauthor=T.+Pillaiyarauthor=L.+Sch%C3%A4kelauthor=G.+Al+Hamwiauthor=M.+Diettauthor=R.+Gedscholdauthor=N.+Geigerauthor=V.+Lopezauthor=S.+Mirzaauthor=V.+Namasivayamauthor=A.+C.+Schiedelauthor=K.+Sylvesterauthor=D.+Thimmauthor=C.+Vielmuthauthor=L.+P.+Vuauthor=M.+Zyulinaauthor=J.+Bodemauthor=M.+G%C3%BCtschowauthor=C.+E.+M%C3%BCller&title=Targeting+the+Main+Protease+of+SARS-CoV-2%3A+From+the+Establishment+of+High+Throughput+Screening+to+the+Design+of+Tailored+Inhibitors&doi=10.1002%2Fanie.202016961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the main protease of SARS-CoV-2: from the establishment of high throughput screening to the design of tailored inhibitors</span></div><div class="casAuthors">Breidenbach, Julian; Lemke, Carina; Pillaiyar, Thanigaimalai; Schaekel, Laura; Al Hamwi, Ghazl; Diett, Miriam; Gedschold, Robin; Geiger, Nina; Lopez, Vittoria; Mirza, Salahuddin; Namasivayam, Vigneshwaran; Schiedel, Anke C.; Sylvester, Katharina; Thimm, Dominik; Vielmuth, Christin; Phuong Vu, Lan; Zyulina, Maria; Bodem, Jochen; Guetschow, Michael; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10423-10429</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The main protease of SARS-CoV-2 (Mpro), the causative agent of COVID-19, constitutes a significant drug target.  A new fluorogenic substrate was kinetically compared to an internally quenched fluorescent peptide and shown to be ideally suitable for high throughput screening with recombinantly expressed Mpro.  Two classes of protease inhibitors, azanitriles and pyridyl esters, were identified, optimized and subjected to in-depth biochem. characterization.  Tailored peptides equipped with the unique azanitrile warhead exhibited concomitant inhibition of Mpro and cathepsin L, a protease relevant for viral cell entry.  Pyridyl indole esters were analyzed by a positional scanning.  Our focused approach towards Mpro inhibitors proved to be superior to virtual screening.  With two irreversible inhibitors, azanitrile 8 (kinac/Ki=37 500 M-1 s-1, Ki=24.0 nM) and pyridyl ester 17 (kinac/Ki=29 100 M-1 s-1, Ki=10.0 nM), promising drug candidates for further development have been discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3x8Ey5-EjoLVg90H21EOLACvtfcHk0lhUmIOaFXlT6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnt1Oht7w%253D&md5=4687ae7d3e266627db5e5a03d6e87f06</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.202016961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202016961%26sid%3Dliteratum%253Aachs%26aulast%3DBreidenbach%26aufirst%3DJ.%26aulast%3DLemke%26aufirst%3DC.%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DSch%25C3%25A4kel%26aufirst%3DL.%26aulast%3DAl%2BHamwi%26aufirst%3DG.%26aulast%3DDiett%26aufirst%3DM.%26aulast%3DGedschold%26aufirst%3DR.%26aulast%3DGeiger%26aufirst%3DN.%26aulast%3DLopez%26aufirst%3DV.%26aulast%3DMirza%26aufirst%3DS.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DSylvester%26aufirst%3DK.%26aulast%3DThimm%26aufirst%3DD.%26aulast%3DVielmuth%26aufirst%3DC.%26aulast%3DVu%26aufirst%3DL.%2BP.%26aulast%3DZyulina%26aufirst%3DM.%26aulast%3DBodem%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DTargeting%2520the%2520Main%2520Protease%2520of%2520SARS-CoV-2%253A%2520From%2520the%2520Establishment%2520of%2520High%2520Throughput%2520Screening%2520to%2520the%2520Design%2520of%2520Tailored%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2021%26volume%3D60%26spage%3D10423%26epage%3D10429%26doi%3D10.1002%2Fanie.202016961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi-Kuwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghavaiah, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lendy, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamatsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamune, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarchoan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodawer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misumi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule Compound with an Indole Moiety Inhibits the Main Protease of SARS-CoV-2 and Blocks Virus Replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">668</span>, <span class="refDoi"> DOI: 10.1038/s41467-021-20900-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1038%2Fs41467-021-20900-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33510133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXivVWiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=668&author=S.+Hattoriauthor=N.+Higashi-Kuwataauthor=H.+Hayashiauthor=S.+R.+Alluauthor=J.+Raghavaiahauthor=H.+Bulutauthor=D.+Dasauthor=B.+J.+Ansonauthor=E.+K.+Lendyauthor=Y.+Takamatsuauthor=N.+Takamuneauthor=N.+Kishimotoauthor=K.+Murayamaauthor=K.+Hasegawaauthor=M.+Liauthor=D.+A.+Davisauthor=E.+N.+Kodamaauthor=R.+Yarchoanauthor=A.+Wlodawerauthor=S.+Misumiauthor=A.+D.+Mesecarauthor=A.+K.+Ghoshauthor=H.+Mitsuya&title=A+Small+Molecule+Compound+with+an+Indole+Moiety+Inhibits+the+Main+Protease+of+SARS-CoV-2+and+Blocks+Virus+Replication&doi=10.1038%2Fs41467-021-20900-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule compound with an indole moiety inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Hattori, Shin-ichiro; Higashi-Kuwata, Nobuyo; Hayashi, Hironori; Allu, Srinivasa Rao; Raghavaiah, Jakka; Bulut, Haydar; Das, Debananda; Anson, Brandon J.; Lendy, Emma K.; Takamatsu, Yuki; Takamune, Nobutoki; Kishimoto, Naoki; Murayama, Kazutaka; Hasegawa, Kazuya; Li, Mi; Davis, David A.; Kodama, Eiichi N.; Yarchoan, Robert; Wlodawer, Alexander; Misumi, Shogo; Mesecar, Andrew D.; Ghosh, Arun K.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">668</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Except remdesivir, no specific antivirals for SARS-CoV-2 infection are currently available.  Here, we characterize two small-mol.-compds., named GRL-1720 and 5h, contg. an indoline and indole moiety, resp., which target the SARS-CoV-2 main protease (Mpro).  We use VeroE6 cell-based assays with RNA-qPCR, cytopathic assays, and immunocytochem. and show both compds. to block the infectivity of SARS-CoV-2 with EC50 values of 15 ± 4 and 4.2 ± 0.7 μM for GRL-1720 and 5h, resp.  Remdesivir permitted viral breakthrough at high concns.; however, compd. 5h completely blocks SARS-CoV-2 infection in vitro without viral breakthrough or detectable cytotoxicity.  Combination of 5h and remdesivir exhibits synergism against SARS-CoV-2.  Addnl. X-ray structural anal. show that 5h forms a covalent bond with Mpro and makes polar interactions with multiple active site amino acid residues.  The present data suggest that 5h might serve as a lead Mpro inhibitor for the development of therapeutics for SARS-CoV-2 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb8PrKVV9DWbVg90H21EOLACvtfcHk0lhUmIOaFXlT6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXivVWiurs%253D&md5=200388ac214c25024fb542da4a7cb6bf</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41467-021-20900-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-021-20900-6%26sid%3Dliteratum%253Aachs%26aulast%3DHattori%26aufirst%3DS.%26aulast%3DHigashi-Kuwata%26aufirst%3DN.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DAllu%26aufirst%3DS.%2BR.%26aulast%3DRaghavaiah%26aufirst%3DJ.%26aulast%3DBulut%26aufirst%3DH.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DAnson%26aufirst%3DB.%2BJ.%26aulast%3DLendy%26aufirst%3DE.%2BK.%26aulast%3DTakamatsu%26aufirst%3DY.%26aulast%3DTakamune%26aufirst%3DN.%26aulast%3DKishimoto%26aufirst%3DN.%26aulast%3DMurayama%26aufirst%3DK.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DD.%2BA.%26aulast%3DKodama%26aufirst%3DE.%2BN.%26aulast%3DYarchoan%26aufirst%3DR.%26aulast%3DWlodawer%26aufirst%3DA.%26aulast%3DMisumi%26aufirst%3DS.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DA%2520Small%2520Molecule%2520Compound%2520with%2520an%2520Indole%2520Moiety%2520Inhibits%2520the%2520Main%2520Protease%2520of%2520SARS-CoV-2%2520and%2520Blocks%2520Virus%2520Replication%26jtitle%3DNat.%2520Commun.%26date%3D2021%26volume%3D12%26spage%3D668%26doi%3D10.1038%2Fs41467-021-20900-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, É.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Désilets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchêne, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chagnon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, É.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent, selective inhibitors of matriptase</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">530</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1021/ml3000534</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3000534" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1OntrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=530-534&author=%C3%89.+Colomboauthor=A.+D%C3%A9siletsauthor=D.+Duch%C3%AAneauthor=F.+Chagnonauthor=R.+Najmanovichauthor=R.+Leducauthor=%C3%89.+Marsault&title=Design+and+synthesis+of+potent%2C+selective+inhibitors+of+matriptase&doi=10.1021%2Fml3000534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Potent, Selective Inhibitors of Matriptase</span></div><div class="casAuthors">Colombo, Eloic; Desilets, Antoine; Duchene, Dominic; Chagnon, Felix; Najmanovich, Rafael; Leduc, Richard; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">530-534</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Matriptase is a member of the type II transmembrane serine protease family.  Several studies have reported deregulated matriptase expression in several types of epithelial cancers, suggesting that matriptase constitutes a potential target for cancer therapy.  We report herein a new series of slow, tight-binding inhibitors of matriptase, which mimic the P1-P4 substrate recognition sequence of the enzyme.  Preliminary structure-activity relationships indicate that this benzothiazole-contg. RQAR-peptidomimetic is a very potent inhibitor and possesses a good selectivity for matriptase vs. other serine proteases.  A mol. model was generated to elucidate the key contacts between inhibitor (I) and matriptase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlqkegeO0GdbVg90H21EOLACvtfcHk0lhUmIOaFXlT6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1OntrY%253D&md5=560a72862038f32c57c829618fdd28aa</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fml3000534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3000534%26sid%3Dliteratum%253Aachs%26aulast%3DColombo%26aufirst%3D%25C3%2589.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DDuch%25C3%25AAne%26aufirst%3DD.%26aulast%3DChagnon%26aufirst%3DF.%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3D%25C3%2589.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%252C%2520selective%2520inhibitors%2520of%2520matriptase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D530%26epage%3D534%26doi%3D10.1021%2Fml3000534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karmakar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owusu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klampfer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janetka, J. W.</span></span> <span> </span><span class="NLM_article-title">α-Ketobenzothiazole Serine Protease Inhibitors of Aberrant HGF/c-MET and MSP/RON Kinase Pathway Signaling in Cancer</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fcmdc.201500600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=26889658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC28Xis1Ghtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=585-599&author=Z.+Hanauthor=P.+K.+W.+Harrisauthor=P.+Karmakarauthor=T.+Kimauthor=B.+Y.+Owusuauthor=S.+A.+Wildmanauthor=L.+Klampferauthor=J.+W.+Janetka&title=%CE%B1-Ketobenzothiazole+Serine+Protease+Inhibitors+of+Aberrant+HGF%2Fc-MET+and+MSP%2FRON+Kinase+Pathway+Signaling+in+Cancer&doi=10.1002%2Fcmdc.201500600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketobenzothiazole Serine Protease Inhibitors of Aberrant HGF/c-MET and MSP/RON Kinase Pathway Signaling in Cancer</span></div><div class="casAuthors">Han, Zhenfu; Harris, Peter K. W.; Karmakar, Partha; Kim, Tommy; Owusu, Ben Y.; Wildman, Scott A.; Klampfer, Lidija; Janetka, James W.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">585-599</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Upregulation of the HGF and MSP growth-factor processing serine endopeptidases HGFA, matriptase and hepsin is correlated with increased metastasis in multiple tumor types driven by c-MET or RON kinase signaling.  The authors rationally designed P1' α-ketobenzothiazole mechanism-based inhibitors of these proteases.  Structure-activity studies are presented, which resulted in the identification of potent inhibitors with differential selectivity.  The tetrapeptide inhibitors span the P1-P1' substrate cleavage site via a P1' amide linker off the benzothiazole, occupying the S3' pocket.  Optimized inhibitors display sub-nanomolar enzyme inhibition against one, two, or all three of HGFA, matriptase, and hepsin.  Several compds. also have good selectivity against the related trypsin-like proteases, thrombin and Factor Xa.  Finally, the authors show that inhibitors block the fibroblast (HGF)-mediated migration of invasive DU145 prostate cancer cells.  In addn. to prostate cancer, breast, colon, lung, pancreas, gliomas, and multiple myeloma tumors all depend on HGF and MSP for tumor survival and progression.  Therefore, these unique inhibitors have potential as new therapeutics for a diverse set of tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo67oTbWfCgKrVg90H21EOLACvtfcHk0ljMr8UCQP25Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xis1Ghtb8%253D&md5=6fd98f5e020a8de9089995cc15b883eb</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500600%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DHarris%26aufirst%3DP.%2BK.%2BW.%26aulast%3DKarmakar%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DOwusu%26aufirst%3DB.%2BY.%26aulast%3DWildman%26aufirst%3DS.%2BA.%26aulast%3DKlampfer%26aufirst%3DL.%26aulast%3DJanetka%26aufirst%3DJ.%2BW.%26atitle%3D%25CE%25B1-Ketobenzothiazole%2520Serine%2520Protease%2520Inhibitors%2520of%2520Aberrant%2520HGF%252Fc-MET%2520and%2520MSP%252FRON%2520Kinase%2520Pathway%2520Signaling%2520in%2520Cancer%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D585%26epage%3D599%26doi%3D10.1002%2Fcmdc.201500600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">St-Georges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Désilets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Béliveau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghinet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dion, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, É.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, É.</span></span> <span> </span><span class="NLM_article-title">Modulating the selectivity of matriptase-2 inhibitors with unnatural amino acids</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">123</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.ejmech.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=28219045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVeksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=110-123&author=C.+St-Georgesauthor=A.+D%C3%A9siletsauthor=F.+B%C3%A9liveauauthor=M.+Ghinetauthor=S.+P.+Dionauthor=%C3%89.+Colomboauthor=P.+Boudreaultauthor=R.+J.+Najmanovichauthor=R.+Leducauthor=%C3%89.+Marsault&title=Modulating+the+selectivity+of+matriptase-2+inhibitors+with+unnatural+amino+acids&doi=10.1016%2Fj.ejmech.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating the selectivity of matriptase-2 inhibitors with unnatural amino acids</span></div><div class="casAuthors">St-Georges, Catherine; Desilets, Antoine; Beliveau, Francois; Ghinet, Mariana; Dion, Sebastien P.; Colombo, Eloic; Boudreault, Pierre-Luc; Najmanovich, Rafael J.; Leduc, Richard; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110-123</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Matriptase-2, a type II transmembrane serine protease (TTSP), is expressed in the liver and regulates iron homeostasis via the cleavage of hemojuvelin.  Matriptase-2 emerges as an attractive target for the treatment of conditions assocd. with iron overload, such as hemochromatosis or beta-thalassemia.  Starting from the crystal structure of its closest homolog matriptase, we constructed a homol. model of matriptase-2 in order to further optimize the selectivity of serine trap peptidomimetic inhibitors for matriptase-2 vs matriptase.  Careful modifications of the P4, P3 and P2 positions with the help of unnatural amino acids led to a thorough understanding of Structure-Activity Relationship and a >60-fold increase in selectivity for matriptase-2 vs. matriptase.  Addnl., the introduction of unnatural amino acids led to significant increases in plasma stability.  Such compds. represent useful pharmacol. tools to test matriptase-2 inhibition in a context of iron overload.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd_FHXeAyV_rVg90H21EOLACvtfcHk0ljMr8UCQP25Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVeksrc%253D&md5=c93364406e2b1d4ba75ce5736bd28035</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DSt-Georges%26aufirst%3DC.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DB%25C3%25A9liveau%26aufirst%3DF.%26aulast%3DGhinet%26aufirst%3DM.%26aulast%3DDion%26aufirst%3DS.%2BP.%26aulast%3DColombo%26aufirst%3D%25C3%2589.%26aulast%3DBoudreault%26aufirst%3DP.%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3D%25C3%2589.%26atitle%3DModulating%2520the%2520selectivity%2520of%2520matriptase-2%2520inhibitors%2520with%2520unnatural%2520amino%2520acids%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D129%26spage%3D110%26epage%3D123%26doi%3D10.1016%2Fj.ejmech.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, L. W.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of ketone-containing, tripeptidyl human rhinovirus 3C protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(99)00587-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2FS0960-894X%2899%2900587-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10636240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjtVyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=45-48&author=P.+S.+Dragovichauthor=R.+Zhouauthor=S.+E.+Webberauthor=T.+J.+Prinsauthor=A.+K.+Kwokauthor=K.+Okanoauthor=S.+A.+Fuhrmanauthor=L.+S.+Zalmanauthor=F.+Z.+Maldonadoauthor=E.+L.+Brownauthor=L.+W.+Meadorauthor=A.+K.+Patickauthor=C.+E.+Fordauthor=M.+A.+Brothersauthor=L.+S.+Binfordauthor=D.+A.+Matthewsauthor=R.+A.+Ferreauthor=S.+T.+Worland&title=Structure-based+design+of+ketone-containing%2C+tripeptidyl+human+rhinovirus+3C+protease+inhibitors&doi=10.1016%2FS0960-894X%2899%2900587-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of ketone-containing, tripeptidyl human rhinovirus 3C protease inhibitors</span></div><div class="casAuthors">Dragovich, Peter S.; Zhou, Ru; Webber, Stephen E.; Prins, Thomas J.; Kwok, Annette K.; Okano, Koji; Fuhrman, Shella A.; Zalman, Leora S.; Maldonado, Fausto C.; Brown, Edward L.; Meador, James W., III; Patick, Amy K.; Ford, Clifford E.; Brothers, Mary A.; Binford, Susan L.; Matthews, David A.; Ferre, Rose Ann; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-48</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Tripeptide-derived mols. incorporating C-terminal ketone electrophiles were evaluated as reversible inhibitors of the cysteine-contg. human rhinovirus 3C protease (3CP).  For example, tripeptide ketone I displayed potent 3CP inhibition activity (Ki = 0.0045 μM) and in vitro antiviral properties (EC50 = 0.34 μM) when tested against HRV serotype-14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxw_91k2flKrVg90H21EOLACvtfcHk0ljMr8UCQP25Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjtVyhtA%253D%253D&md5=60b8bd8d56ccd09ae8e60ce341926271</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2899%2900587-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252899%252900587-9%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DKwok%26aufirst%3DA.%2BK.%26aulast%3DOkano%26aufirst%3DK.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMaldonado%26aufirst%3DF.%2BZ.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DL.%2BW.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DBinford%26aufirst%3DL.%2BS.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-based%2520design%2520of%2520ketone-containing%252C%2520tripeptidyl%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D45%26epage%3D48%26doi%3D10.1016%2FS0960-894X%2899%2900587-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lianFiub_tuig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dropinski, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykstra, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirocco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streckfuss, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cernak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vachal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krska, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreher, S. D.</span></span> <span> </span><span class="NLM_article-title">Chemistry Informer Libraries: A Chemoinformatics Enabled Approach to Evaluate and Advance Synthetic Methods</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2604</span>– <span class="NLM_lpage">2613</span>, <span class="refDoi"> DOI: 10.1039/C5SC04751J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1039%2FC5SC04751J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=28660032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2604-2613&author=P.+S.+Kutchukianauthor=J.+F.+Dropinskiauthor=K.+D.+Dykstraauthor=B.+Liauthor=D.+A.+Diroccoauthor=E.+C.+Streckfussauthor=L.+C.+Campeauauthor=T.+Cernakauthor=P.+Vachalauthor=I.+W.+Daviesauthor=S.+W.+Krskaauthor=S.+D.+Dreher&title=Chemistry+Informer+Libraries%3A+A+Chemoinformatics+Enabled+Approach+to+Evaluate+and+Advance+Synthetic+Methods&doi=10.1039%2FC5SC04751J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry informer libraries: a chemoinformatics enabled approach to evaluate and advance synthetic methods</span></div><div class="casAuthors">Kutchukian, Peter S.; Dropinski, James F.; Dykstra, Kevin D.; Li, Bing; DiRocco, Daniel A.; Streckfuss, Eric C.; Campeau, Louis-Charles; Cernak, Tim; Vachal, Petr; Davies, Ian W.; Krska, Shane W.; Dreher, Spencer D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2604-2613</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Major new advances in synthetic chem. methods are typically reported using simple, non-standardized reaction substrates, and reaction failures are rarely documented.  This makes the evaluation and choice of a synthetic method difficult.  We report a standardized complex mol. diagnostic approach using collections of relevant drug-like mols. which we call chem. informer libraries.  With this approach, all chem. results, successes and failures, can be documented to compare and evolve synthetic methods.  To aid in the visualization of chem. results in drug-like physicochem. space we have used an informatics methodol. termed principal component anal.  We have validated this method using palladium- and copper-catalyzed reactions, including Suzuki-Miyaura, cyanation and Buchwald-Hartwig amination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDsNJ0zmf9DrVg90H21EOLACvtfcHk0lianFiub_tuig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSju7o%253D&md5=d7169f9ae6f74e43df0e44d2fa00e5af</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2FC5SC04751J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC04751J%26sid%3Dliteratum%253Aachs%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DDropinski%26aufirst%3DJ.%2BF.%26aulast%3DDykstra%26aufirst%3DK.%2BD.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DDirocco%26aufirst%3DD.%2BA.%26aulast%3DStreckfuss%26aufirst%3DE.%2BC.%26aulast%3DCampeau%26aufirst%3DL.%2BC.%26aulast%3DCernak%26aufirst%3DT.%26aulast%3DVachal%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DI.%2BW.%26aulast%3DKrska%26aufirst%3DS.%2BW.%26aulast%3DDreher%26aufirst%3DS.%2BD.%26atitle%3DChemistry%2520Informer%2520Libraries%253A%2520A%2520Chemoinformatics%2520Enabled%2520Approach%2520to%2520Evaluate%2520and%2520Advance%2520Synthetic%2520Methods%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D2604%26epage%3D2613%26doi%3D10.1039%2FC5SC04751J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrivastava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassanali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stothard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolsey, J.</span></span> <span> </span><span class="NLM_article-title">DrugBank: A Comprehensive Resource for in Silico Drug Discovery and Exploration</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1093/nar/gkj067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1093%2Fnar%2Fgkj067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=16381955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFOrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=668-672&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=S.+Shrivastavaauthor=M.+Hassanaliauthor=P.+Stothardauthor=Z.+Changauthor=J.+Woolsey&title=DrugBank%3A+A+Comprehensive+Resource+for+in+Silico+Drug+Discovery+and+Exploration&doi=10.1093%2Fnar%2Fgkj067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank: a comprehensive resource for in silico drug discovery and exploration</span></div><div class="casAuthors">Wishart, David S.; Knox, Craig; Guo, An Chi; Shrivastava, Savita; Hassanali, Murtaza; Stothard, Paul; Chang, Zhan; Woolsey, Jennifer</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Database</span>),
    <span class="NLM_cas:pages">D668-D672</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a unique bioinformatics/cheminformatics resource that combines detailed drug (i.e. chem.) data with comprehensive drug target (i.e. protein) information.  The database contains >4100 drug entries including >800 FDA approved small mol. and biotech drugs as well as >3200 exptl. drugs.  Addnl., >14 000 protein or drug target sequences are linked to these drug entries.  Each DrugCard entry contains >80 data fields with half of the information being devoted to drug/chem. data and the other half devoted to drug target or protein data.  Many data fields are hyperlinked to other databases (KEGG, PubChem, ChEBI, PDB, Swiss-Prot and GenBank) and a variety of structure viewing applets.  The database is fully searchable supporting extensive text, sequence, chem. structure and relational query searches.  Potential applications of DrugBank include in silico drug target discovery, drug design, drug docking or screening, drug metab. prediction, drug interaction prediction and general pharmaceutical education.  DrugBank is available at http://redpoll.pharmacy.ualberta.ca/drugbank/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaNy9WiVtlu7Vg90H21EOLACvtfcHk0lianFiub_tuig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFOrsw%253D%253D&md5=c3f4ad2fc489440be5b16f91bc9cf7db</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkj067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkj067%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DHassanali%26aufirst%3DM.%26aulast%3DStothard%26aufirst%3DP.%26aulast%3DChang%26aufirst%3DZ.%26aulast%3DWoolsey%26aufirst%3DJ.%26atitle%3DDrugBank%253A%2520A%2520Comprehensive%2520Resource%2520for%2520in%2520Silico%2520Drug%2520Discovery%2520and%2520Exploration%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D668%26epage%3D672%26doi%3D10.1093%2Fnar%2Fgkj067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boras, B.</span>; <span class="NLM_string-name">Jones, R. M.</span>; <span class="NLM_string-name">Anson, B. J.</span>; <span class="NLM_string-name">Arenson, D.</span>; <span class="NLM_string-name">Aschenbrenner, L.</span>; <span class="NLM_string-name">Bakowski, M. A.</span>; <span class="NLM_string-name">Beutler, N.</span>; <span class="NLM_string-name">Binder, J.</span>; <span class="NLM_string-name">Chen, E.</span>; <span class="NLM_string-name">Eng, H.</span>; <span class="NLM_string-name">Hammond, H.</span>; <span class="NLM_string-name">Hammond, J.</span>; <span class="NLM_string-name">Haupt, R. E.</span>; <span class="NLM_string-name">Hoffman, R.</span>; <span class="NLM_string-name">Kadar, E. P.</span>; <span class="NLM_string-name">Kania, R.</span>; <span class="NLM_string-name">Kimoto, E.</span>; <span class="NLM_string-name">Kirkpatrick, M. G.</span>; <span class="NLM_string-name">Lanyon, L.</span>; <span class="NLM_string-name">Lendy, E. K.</span>; <span class="NLM_string-name">Lillis, J. R.</span>; <span class="NLM_string-name">Logue, J.</span>; <span class="NLM_string-name">Luthra, S. A.</span>; <span class="NLM_string-name">Ma, C.</span>; <span class="NLM_string-name">Mason, S. W.</span>; <span class="NLM_string-name">McGrath, M. E.</span>; <span class="NLM_string-name">Noell, S.</span>; <span class="NLM_string-name">Obach, R. S.</span>; <span class="NLM_string-name">O’Brien, M. N.</span>; <span class="NLM_string-name">O’Connor, R.</span>; <span class="NLM_string-name">Ogilvie, K.</span>; <span class="NLM_string-name">Owen, D.</span>; <span class="NLM_string-name">Pettersson, M.</span>; <span class="NLM_string-name">Reese, M. R.</span>; <span class="NLM_string-name">Rogers, T. F.</span>; <span class="NLM_string-name">Rossulek, M. I.</span>; <span class="NLM_string-name">Sathish, J. G.</span>; <span class="NLM_string-name">Shirai, N.</span>; <span class="NLM_string-name">Steppan, C.</span>; <span class="NLM_string-name">Ticehurst, M.</span>; <span class="NLM_string-name">Updyke, L. W.</span>; <span class="NLM_string-name">Weston, S.</span>; <span class="NLM_string-name">Zhu, Y.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Chatterjee, A. K.</span>; <span class="NLM_string-name">Mesecar, A. D.</span>; <span class="NLM_string-name">Frieman, M. B.</span>; <span class="NLM_string-name">Anderson, A. S.</span>; <span class="NLM_string-name">Allerton, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel Inhibitor of Coronavirus 3CL Protease for the Potential Treatment of COVID-19</span>.  <i>bioRxiv</i>, September 30, <span class="NLM_year" style="font-weight: bold;">2020</span>. <span class="refDoi"> DOI: 10.1101/2020.09.12.293498</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1101%2F2020.09.12.293498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32935104" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=B.+Boras&author=R.+M.+Jones&author=B.+J.+Anson&author=D.+Arenson&author=L.+Aschenbrenner&author=M.+A.+Bakowski&author=N.+Beutler&author=J.+Binder&author=E.+Chen&author=H.+Eng&author=H.+Hammond&author=J.+Hammond&author=R.+E.+Haupt&author=R.+Hoffman&author=E.+P.+Kadar&author=R.+Kania&author=E.+Kimoto&author=M.+G.+Kirkpatrick&author=L.+Lanyon&author=E.+K.+Lendy&author=J.+R.+Lillis&author=J.+Logue&author=S.+A.+Luthra&author=C.+Ma&author=S.+W.+Mason&author=M.+E.+McGrath&author=S.+Noell&author=R.+S.+Obach&author=M.+N.+O%E2%80%99Brien&author=R.+O%E2%80%99Connor&author=K.+Ogilvie&author=D.+Owen&author=M.+Pettersson&author=M.+R.+Reese&author=T.+F.+Rogers&author=M.+I.+Rossulek&author=J.+G.+Sathish&author=N.+Shirai&author=C.+Steppan&author=M.+Ticehurst&author=L.+W.+Updyke&author=S.+Weston&author=Y.+Zhu&author=J.+Wang&author=A.+K.+Chatterjee&author=A.+D.+Mesecar&author=M.+B.+Frieman&author=A.+S.+Anderson&author=C.+Allerton&title=Discovery+of+a+Novel+Inhibitor+of+Coronavirus+3CL+Protease+for+the+Potential+Treatment+of+COVID-19&doi=10.1101%2F2020.09.12.293498"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1101%2F2020.09.12.293498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F2020.09.12.293498%26sid%3Dliteratum%253Aachs%26aulast%3DBoras%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Inhibitor%2520of%2520Coronavirus%25203CL%2520Protease%2520for%2520the%2520Potential%2520Treatment%2520of%2520COVID-19%26jtitle%3DbioRxiv%26date%3D2020%26doi%3D10.1101%2F2020.09.12.293498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vries, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valero-Jimenez, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desvignes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devlin, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinlever, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggles, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koralov, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, M.</span></span> <span> </span><span class="NLM_article-title">A Comparative Analysis of SARS-CoV-2 Antivirals Characterizes 3CLpro Inhibitor PF-00835231 as a Potential New Treatment for COVID-19</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">e01819</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1128/JVI.01819-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FJVI.01819-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXht1ensrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2021&pages=e01819-20&author=M.+de%0AVriesauthor=A.+S.+Mohamedauthor=R.+A.+Prescottauthor=A.+M.+Valero-Jimenezauthor=L.+Desvignesauthor=R.+O%E2%80%99Connorauthor=C.+Steppanauthor=J.+C.+Devlinauthor=E.+Ivanovaauthor=A.+Herreraauthor=A.+Schinleverauthor=P.+Looseauthor=K.+Rugglesauthor=S.+B.+Koralovauthor=A.+S.+Andersonauthor=J.+Binderauthor=M.+Dittmann&title=A+Comparative+Analysis+of+SARS-CoV-2+Antivirals+Characterizes+3CLpro+Inhibitor+PF-00835231+as+a+Potential+New+Treatment+for+COVID-19&doi=10.1128%2FJVI.01819-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A comparative analysis of SARS-CoV-2 antivirals characterizes 3CLpro inhibitor PF-00835231 as a potential new treatment for COVID-19</span></div><div class="casAuthors">de Vries, Maren; Mohamed, Adil S.; Prescott, Rachel A.; Valero-Jimenez, Ana M.; Desvignes, Ludovic; O'Connor, Rebecca; Steppan, Claire; Devlin, Joseph C.; Ivanova, Ellie; Herrera, Alberto; Schinlever, Austin; Loose, Paige; Ruggles, Kelly; Koralov, Sergei B.; Anderson, Annaliesa S.; Binder, Joseph; Dittmann, Meike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e01819</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the etiol. agent of coronavirus disease 2019 (COVID-19).  There is a dire need for novel effective antivirals to treat COVID-19, as the only approved direct-acting antiviral to date is remdesivir, targeting the viral polymerase complex.  A potential alternate target in the viral life cycle is the main SARS-CoV-2 protease 3CLpro (Mpro).  The drug candidate PF-00835231 is the active compd. of the first anti-3CLpro regimen in clin. trials.  Here, we perform a comparative anal. of PF-00835231, the preclin. 3CLpro inhibitor GC-376, and the polymerase inhibitor remdesivir, in alveolar basal epithelial cells modified to express ACE2 (A549+ACE2 cells).  We find PF-00835231 with at least similar or higher potency than remdesivir or GC-376.  A time-of-drug-addn. approach delineates the timing of early SARS-CoV-2 life cycle steps in A549+ACE2 cells and validates PF-00835231's early time of action.  In a model of the human polarized airway epithelium, both PF-00835231 and remdesivir potently inhibit SARS-CoV-2 at low micromolar concns.  Finally, we show that the efflux transporter P-glycoprotein, which was previously suggested to diminish PF-00835231's efficacy based on expts. in monkey kidney Vero E6 cells, does not neg. impact PF-00835231 efficacy in either A549+ACE2 cells or human polarized airway epithelial cultures.  Thus, our study provides in vitro evidence for the potential of PF-00835231 as an effective SARS-CoV-2 antiviral and addresses concerns that emerged based on prior studies in nonhuman in vitro models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR9dKQsVRU9bVg90H21EOLACvtfcHk0ljtRwVxdcluwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXht1ensrjI&md5=d67cdf0d1ddc1c14f01f55d25fd48bd6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FJVI.01819-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01819-20%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVries%26aufirst%3DM.%26aulast%3DMohamed%26aufirst%3DA.%2BS.%26aulast%3DPrescott%26aufirst%3DR.%2BA.%26aulast%3DValero-Jimenez%26aufirst%3DA.%2BM.%26aulast%3DDesvignes%26aufirst%3DL.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DDevlin%26aufirst%3DJ.%2BC.%26aulast%3DIvanova%26aufirst%3DE.%26aulast%3DHerrera%26aufirst%3DA.%26aulast%3DSchinlever%26aufirst%3DA.%26aulast%3DLoose%26aufirst%3DP.%26aulast%3DRuggles%26aufirst%3DK.%26aulast%3DKoralov%26aufirst%3DS.%2BB.%26aulast%3DAnderson%26aufirst%3DA.%2BS.%26aulast%3DBinder%26aufirst%3DJ.%26aulast%3DDittmann%26aufirst%3DM.%26atitle%3DA%2520Comparative%2520Analysis%2520of%2520SARS-CoV-2%2520Antivirals%2520Characterizes%25203CLpro%2520Inhibitor%2520PF-00835231%2520as%2520a%2520Potential%2520New%2520Treatment%2520for%2520COVID-19%26jtitle%3DJ.%2520Virol.%26date%3D2021%26volume%3D95%26spage%3De01819%26epage%3D20%26doi%3D10.1128%2FJVI.01819-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, T.</span></span> <span> </span><span class="NLM_article-title">Physiological Parameters in Laboratory Animals and Humans</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1093</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1023/A:1018943613122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1023%2FA%3A1018943613122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=8378254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADyaK3szptFaqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1993&pages=1093-1095&author=B.+Daviesauthor=T.+Morris&title=Physiological+Parameters+in+Laboratory+Animals+and+Humans&doi=10.1023%2FA%3A1018943613122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Physiological parameters in laboratory animals and humans</span></div><div class="casAuthors">Davies B; Morris T</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1093-5</span>
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDDwka3A6hEYYPsCkJIw00fW6udTcc2eaRexculXVl1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3szptFaqsg%253D%253D&md5=094cdd17306b58cc17c8cf4fd5db9927</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1023%2FA%3A1018943613122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1018943613122%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DB.%26aulast%3DMorris%26aufirst%3DT.%26atitle%3DPhysiological%2520Parameters%2520in%2520Laboratory%2520Animals%2520and%2520Humans%26jtitle%3DPharm.%2520Res.%26date%3D1993%26volume%3D10%26spage%3D1093%26epage%3D1095%26doi%3D10.1023%2FA%3A1018943613122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=J.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+Ketone-Based+Covalent+Inhibitors+of+Coronavirus+3CL+Proteases+for+the+Potential+Therapeutic+Treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lh0Zb6wUtDh9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Ketone-Based%2520Covalent%2520Inhibitors%2520of%2520Coronavirus%25203CL%2520Proteases%2520for%2520the%2520Potential%2520Therapeutic%2520Treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the Inhibition Constant (KI) and the Concentration of Inhibitor Which Causes 50 per Cent Inhibition (I50) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+Inhibition+Constant+%28KI%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520Inhibition%2520Constant%2520%2528KI%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520Which%2520Causes%252050%2520per%2520Cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senda, T.</span></span> <span> </span><span class="NLM_article-title">A Fully Automated Crystallization Apparatus for Small Protein Quantities</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1107/S2053230X20015514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS2053230X20015514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33439153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtFSgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2021&pages=29-36&author=R.+Katoauthor=M.+Hirakiauthor=Y.+Yamadaauthor=M.+Tanabeauthor=T.+Senda&title=A+Fully+Automated+Crystallization+Apparatus+for+Small+Protein+Quantities&doi=10.1107%2FS2053230X20015514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A fully automated crystallization apparatus for small protein quantities</span></div><div class="casAuthors">Kato, Ryuichi; Hiraki, Masahiko; Yamada, Yusuke; Tanabe, Mikio; Senda, Toshiya</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology Communications</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-36</span>CODEN:
                <span class="NLM_cas:coden">ACSFEN</span>;
        ISSN:<span class="NLM_cas:issn">2053-230X</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">In 2003, a fully automated protein crystn. and monitoring system (PXS) was developed to support the structural genomics projects that were initiated in the early 2000s.  In PXS, crystn. plates were automatically set up using the vapor-diffusion method, transferred to incubators and automatically obsd. according to a pre-set schedule.  The captured images of each crystn. drop could be monitored through the internet using a web browser.  While the screening throughput of PXS was very high, the demands of users have gradually changed over the ensuing years.  To study difficult proteins, it has become important to screen crystn. conditions using small amts. of proteins.  Moreover, membrane proteins have become one of the main targets for X-ray crystallog.  Therefore, to meet the evolving demands of users, PXS was upgraded to PXS2.  In PXS2, the min. vol. of the dispenser is reduced to 0.1μl to minimize the amt. of sample, and the resoln. of the captured images is increased to five million pixels in order to observe small crystn. drops in detail.  In addn. to the 20°C incubators, a 4°C incubator was installed in PXS2 because crystn. results may vary with temp.  To support membrane-protein crystn., PXS2 includes a procedure for the bicelle method.  In addn., the system supports a lipidic cubic phase (LCP) method that uses a film sandwich plate and that was specifically designed for PXS2.  These improvements expand the applicability of PXS2, reducing the bottleneck of X-ray protein crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt0SjXkr_IqrVg90H21EOLACvtfcHk0lh0Zb6wUtDh9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtFSgtr8%253D&md5=99a0aed5d2b74bfd00d54b2841d99eb9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS2053230X20015514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053230X20015514%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DR.%26aulast%3DHiraki%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DY.%26aulast%3DTanabe%26aufirst%3DM.%26aulast%3DSenda%26aufirst%3DT.%26atitle%3DA%2520Fully%2520Automated%2520Crystallization%2520Apparatus%2520for%2520Small%2520Protein%2520Quantities%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Commun.%26date%3D2021%26volume%3D77%26spage%3D29%26epage%3D36%26doi%3D10.1107%2FS2053230X20015514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljZstm__Vo_vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljZstm__Vo_vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.Refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.Refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0ljZstm__Vo_vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.Refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljZstm__Vo_vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanwell, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonie, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeersch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, G. R.</span></span> <span> </span><span class="NLM_article-title">Avogadro: An Advanced Semantic Chemical Editor, Visualization, and Analysis Platform</span>. <i>J. Cheminform.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-4-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1186%2F1758-2946-4-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=22889332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BC38fns1yntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=17&author=M.+D.+Hanwellauthor=D.+E.+Curtisauthor=D.+C.+Lonieauthor=T.+Vandermeerschauthor=E.+Zurekauthor=G.+R.+Hutchison&title=Avogadro%3A+An+Advanced+Semantic+Chemical+Editor%2C+Visualization%2C+and+Analysis+Platform&doi=10.1186%2F1758-2946-4-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Avogadro: an advanced semantic chemical editor, visualization, and analysis platform</span></div><div class="casAuthors">Hanwell Marcus D; Curtis Donald E; Lonie David C; Vandermeersch Tim; Zurek Eva; Hutchison Geoffrey R</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">UNLABELLED:  BACKGROUND:  The Avogadro project has developed an advanced molecule editor and visualizer designed for cross-platform use in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.  It offers flexible, high quality rendering, and a powerful plugin architecture.  Typical uses include building molecular structures, formatting input files, and analyzing output of a wide variety of computational chemistry packages.  By using the CML file format as its native document type, Avogadro seeks to enhance the semantic accessibility of chemical data types.  RESULTS:  The work presented here details the Avogadro library, which is a framework providing a code library and application programming interface (API) with three-dimensional visualization capabilities; and has direct applications to research and education in the fields of chemistry, physics, materials science, and biology.  The Avogadro application provides a rich graphical interface using dynamically loaded plugins through the library itself.  The application and library can each be extended by implementing a plugin module in C++ or Python to explore different visualization techniques, build/manipulate molecular structures, and interact with other programs.  We describe some example extensions, one which uses a genetic algorithm to find stable crystal structures, and one which interfaces with the PackMol program to create packed, solvated structures for molecular dynamics simulations.  The 1.0 release series of Avogadro is the main focus of the results discussed here.  CONCLUSIONS:  Avogadro offers a semantic chemical builder and platform for visualization and analysis.  For users, it offers an easy-to-use builder, integrated support for downloading from common databases such as PubChem and the Protein Data Bank, extracting chemical data from a wide variety of formats, including computational chemistry output, and native, semantic support for the CML file format.  For developers, it can be easily extended via a powerful plugin mechanism to support new features in organic chemistry, inorganic complexes, drug design, materials, biomolecules, and simulations.  Avogadro is freely available under an open-source license from        http://avogadro.openmolecules.net.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTitVKXb07oIcuCWE5mhx1RfW6udTcc2eaucZZfuStr6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fns1yntQ%253D%253D&md5=91fe7fe658064a132273004aa5ee0776</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-4-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-4-17%26sid%3Dliteratum%253Aachs%26aulast%3DHanwell%26aufirst%3DM.%2BD.%26aulast%3DCurtis%26aufirst%3DD.%2BE.%26aulast%3DLonie%26aufirst%3DD.%2BC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DZurek%26aufirst%3DE.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DAvogadro%253A%2520An%2520Advanced%2520Semantic%2520Chemical%2520Editor%252C%2520Visualization%252C%2520and%2520Analysis%2520Platform%26jtitle%3DJ.%2520Cheminform.%26date%3D2012%26volume%3D4%26spage%3D17%26doi%3D10.1186%2F1758-2946-4-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamai, I.</span></span> <span> </span><span class="NLM_article-title">Differential Effect of Grapefruit Juice on Intestinal Absorption of Statins Due to Inhibition of Organic Anion Transporting Polypeptide and/or P-Glycoprotein</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">3843</span>– <span class="NLM_lpage">3853</span>, <span class="refDoi"> DOI: 10.1002/jps.22586</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fjps.22586" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=21520088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFemt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2011&pages=3843-3853&author=Y.+Shirasakaauthor=K.+Suzukiauthor=T.+Nakanishiauthor=I.+Tamai&title=Differential+Effect+of+Grapefruit+Juice+on+Intestinal+Absorption+of+Statins+Due+to+Inhibition+of+Organic+Anion+Transporting+Polypeptide+and%2For+P-Glycoprotein&doi=10.1002%2Fjps.22586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Differential effect of grapefruit juice on intestinal absorption of statins due to inhibition of organic anion transporting polypeptide and/or P-glycoprotein</span></div><div class="casAuthors">Shirasaka, Yoshiyuki; Suzuki, Kensuke; Nakanishi, Takeo; Tamai, Ikumi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3843-3853</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The purpose of this study is to examine the contributions of org. anion transporting polypeptide (Oatp) and/or P-glycoprotein (P-gp) to grapefruit juice (GFJ) interaction with two statins, pravastatin and pitavastatin, which undergo negligible metab. in rats.  The two statins were found to be substrates of both Oatp1a5 and Oatp2b1, whereas pitavastatin, but not pravastatin, was a substrate of P-gp.  The plasma concn. of pravastatin after oral administration was significantly decreased by GFJ and naringin, whereas that of pitavastatin was significantly increased.  Naringin inhibited Oatp1a5- and Oatp2b1-mediated uptake of pravastatin and Oatp1a5-mediated, but not Oatp2b1-mediated, uptake of pitavastatin.  Naringin also inhibited P-gp-mediated transport of pitavastatin.  These results suggested that the decrease of pravastatin absorption in the presence of GFJ is due to the inhibitory effect of naringin on Oatp, whereas the increase of pitavastatin is due to the inhibition of P-gp.  These observations are consistent with the results of in situ absorption studies.  In conclusion, Oatp and/or P-gp contribute to the intestinal absorption of statins, and the differential effect of GFJ on pravastatin and pitavastatin absorption is at least partly accounted for by the different inhibitory effects of naringin on these transporters. © 2011 Wiley-Liss, Inc. and the American Pharmacists Assocn. J Pharm Sci 100:3843-3853, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKsxXgBVv6LVg90H21EOLACvtfcHk0ljtNGa8dP7cxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFemt74%253D&md5=5190b9c2020a6768aea2e376d2d0a989</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fjps.22586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.22586%26sid%3Dliteratum%253Aachs%26aulast%3DShirasaka%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DNakanishi%26aufirst%3DT.%26aulast%3DTamai%26aufirst%3DI.%26atitle%3DDifferential%2520Effect%2520of%2520Grapefruit%2520Juice%2520on%2520Intestinal%2520Absorption%2520of%2520Statins%2520Due%2520to%2520Inhibition%2520of%2520Organic%2520Anion%2520Transporting%2520Polypeptide%2520and%252For%2520P-Glycoprotein%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2011%26volume%3D100%26spage%3D3843%26epage%3D3853%26doi%3D10.1002%2Fjps.22586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Funai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichijo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, K.</span></span> <span> </span><span class="NLM_article-title">Effect of Osmolality on the Pharmacokinetic Interaction between Apple Juice and Atenolol in Rats</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">386</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1124/dmd.118.084483</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1124%2Fdmd.118.084483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=30622163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFOisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=386-391&author=Y.+Funaiauthor=Y.+Shirasakaauthor=M.+Ishiharaauthor=M.+Takemuraauthor=K.+Ichijoauthor=H.+Kishimotoauthor=K.+Inoue&title=Effect+of+Osmolality+on+the+Pharmacokinetic+Interaction+between+Apple+Juice+and+Atenolol+in+Rats&doi=10.1124%2Fdmd.118.084483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of osmolality on the pharmacokinetic interaction between apple juice and atenolol in rats</span></div><div class="casAuthors">Funai, Yuta; Shirasaka, Yoshiyuki; Ishihara, Marika; Takemura, Miyuki; Ichijo, Kazuki; Kishimoto, Hisanao; Inoue, Katsuhisa</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">386-391</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A recent clin. study reported that the ingestion of apple juice (AJ) markedly reduced the plasma concn. of atenolol; however, our in vitro study showed that atenolol may not be a substrate of org. anion transporting polypeptide 2B1 (OATP2B1), so this AJ-atenolol interaction cannot be explained by inhibition of OATP2B1.  On the other hand, we more recently showed that the soln. osmolality influences gastrointestinal (GI) water vol., and this may indirectly affect intestinal drug absorption.  In this study, we examd. whether the osmolality dependence of water dynamics can account for AJ-atenolol interactions by evaluating the GI water vol. and the atenolol aborption in the presence of AJ in rats.  Water absorption was highest in purified water, followed by saline and isosmotic mannitol soln., and the lowest in AJ, confirming that water absorption is indeed osmolality-dependent.  Interestingly, AJ showed apparent water secretion into the intestinal lumen.  Further, the fraction of atenolol absorbed in intestine was significantly lower in AJ or hyperosmotic mannitol soln. (adjusted to the osmolality of AJ) than after administration in purified water.  Comparable results were obsd. in an in vivo pharmacokinetic study in rats.  Our results indicate that orally administered AJ has a capacity to modulate luminal water vol. depending on the osmolality, and this effect may result in significant AJ-atenolol interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTYwR4fC653LVg90H21EOLACvtfcHk0ljtNGa8dP7cxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFOisr4%253D&md5=3666bb01ddde21fe98acf9cf0cbd4591</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1124%2Fdmd.118.084483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.118.084483%26sid%3Dliteratum%253Aachs%26aulast%3DFunai%26aufirst%3DY.%26aulast%3DShirasaka%26aufirst%3DY.%26aulast%3DIshihara%26aufirst%3DM.%26aulast%3DTakemura%26aufirst%3DM.%26aulast%3DIchijo%26aufirst%3DK.%26aulast%3DKishimoto%26aufirst%3DH.%26aulast%3DInoue%26aufirst%3DK.%26atitle%3DEffect%2520of%2520Osmolality%2520on%2520the%2520Pharmacokinetic%2520Interaction%2520between%2520Apple%2520Juice%2520and%2520Atenolol%2520in%2520Rats%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2019%26volume%3D47%26spage%3D386%26epage%3D391%26doi%3D10.1124%2Fdmd.118.084483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uno, T.</span></span> <span> </span><span class="NLM_article-title">Statistical Moments in Pharmacokinetics</span>. <i>J. Pharmacokinet. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1007/BF01062109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1007%2FBF01062109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=731417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADyaE1M%252FptVKqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1978&pages=547-558&author=K.+Yamaokaauthor=T.+Nakagawaauthor=T.+Uno&title=Statistical+Moments+in+Pharmacokinetics&doi=10.1007%2FBF01062109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical moments in pharmacokinetics</span></div><div class="casAuthors">Yamaoka K; Nakagawa T; Uno T</div><div class="citationInfo"><span class="NLM_cas:title">Journal of pharmacokinetics and biopharmaceutics</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">547-58</span>
        ISSN:<span class="NLM_cas:issn">0090-466X</span>.
    </div><div class="casAbstract">Statistical moments are parameters that describe the characteristics of the time courses of plasma concentration (area, mean residence time, and variance of residence time) and of the urinary excretion rate that follow administration of a single dose of a drug.  The relationship between the moments of a time-course curve and pharmacokinetic profiles of drug disposition, i.e., absorption, distribution, metabolism, and excretion, is described.  The moments are related to the extent and rate of bioavailability, and it is shown that they can be effectively applied to the deconvolution operation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9G3T-afn-o_csfMJQC1ZdfW6udTcc2eYKpZiVs0ZDUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE1M%252FptVKqtQ%253D%253D&md5=22a76390b1ea71d8f306d598dc01b040</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2FBF01062109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01062109%26sid%3Dliteratum%253Aachs%26aulast%3DYamaoka%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DT.%26aulast%3DUno%26aufirst%3DT.%26atitle%3DStatistical%2520Moments%2520in%2520Pharmacokinetics%26jtitle%3DJ.%2520Pharmacokinet.%2520Biopharm.%26date%3D1978%26volume%3D6%26spage%3D547%26epage%3D558%26doi%3D10.1007%2FBF01062109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hisaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, Y.</span></span> <span> </span><span class="NLM_article-title">Analysis of Nonlinear and Nonsteady State Hepatic Extraction with the Dispersion Model Using the Finite Difference Method</span>. <i>J. Pharmacokinet. Pharmacodyn.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1023/A:1023294632129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1023%2FA%3A1023294632129" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1998&pages=495-519&author=A.+Hisakaauthor=Y.+Sugiyama&title=Analysis+of+Nonlinear+and+Nonsteady+State+Hepatic+Extraction+with+the+Dispersion+Model+Using+the+Finite+Difference+Method&doi=10.1023%2FA%3A1023294632129"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023294632129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023294632129%26sid%3Dliteratum%253Aachs%26aulast%3DHisaka%26aufirst%3DA.%26aulast%3DSugiyama%26aufirst%3DY.%26atitle%3DAnalysis%2520of%2520Nonlinear%2520and%2520Nonsteady%2520State%2520Hepatic%2520Extraction%2520with%2520the%2520Dispersion%2520Model%2520Using%2520the%2520Finite%2520Difference%2520Method%26jtitle%3DJ.%2520Pharmacokinet.%2520Pharmacodyn.%26date%3D1998%26volume%3D26%26spage%3D495%26epage%3D519%26doi%3D10.1023%2FA%3A1023294632129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakane, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, S.</span></span> <span> </span><span class="NLM_article-title">Effect of P-Glycoprotein Expression Levels on the Concentration-Dependent Permeability of Drugs to the Cell Membrane</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">565</span>, <span class="refDoi"> DOI: 10.1002/jps.21114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fjps.21114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=17828734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BD2sjhs1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2008&pages=553-565&author=Y.+Shirasakaauthor=T.+Sakaneauthor=S.+Yamashita&title=Effect+of+P-Glycoprotein+Expression+Levels+on+the+Concentration-Dependent+Permeability+of+Drugs+to+the+Cell+Membrane&doi=10.1002%2Fjps.21114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of P-glycoprotein expression levels on the concentration-dependent permeability of drugs to the cell membrane</span></div><div class="casAuthors">Shirasaka Yoshiyuki; Sakane Toshiyasu; Yamashita Shinji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of pharmaceutical sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">553-65</span>
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    </div><div class="casAbstract">The purpose of this study is to develop a kinetic model that can predict the in vivo absorption of P-glycoprotein (P-gp) substrates from in vitro data.  Apical (AP) to basal (BL) absorptive permeability of typical P-gp substrate drugs including quinidine, verapamil, vinblastine, and digoxin, were measured in several cell monolayers with different levels of P-gp expression, normal, P-gp induced, P-gp highly induced and MDR1-knockdown Caco-2 cells and MDR1-MDCKII cells.  In all cell monolayers, AP to BL permeability of P-gp substrates increased when their AP concentration was increased, showing a sigmoid-type relationship to donor (AP) concentrations.  At the higher concentration range, permeability reached a maximum value, suggesting saturation of P-gp-mediated efflux, and at the lower concentration range, permeability decreased depending on P-gp expression level.  A simple kinetic model was applied to the permeability-concentration curve of each drug to obtain the fundamental parameters for P-gp-mediated transport, K(m(app)) and V(max).  Both K(m(app)) and V(max) of each drug were found to show linear correlations with expression level of P-gp.  This study clearly demonstrated the possibility to estimate the permeability of P-gp substrate drugs in human intestine from the expression level of P-gp, and thus the possibility to predict oral absorption of those drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpkUtoBnpEpz0O18WR3iAofW6udTcc2eYKpZiVs0ZDUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjhs1emsA%253D%253D&md5=e208a5a449a6456bac9027791df93bd4</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjps.21114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21114%26sid%3Dliteratum%253Aachs%26aulast%3DShirasaka%26aufirst%3DY.%26aulast%3DSakane%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DS.%26atitle%3DEffect%2520of%2520P-Glycoprotein%2520Expression%2520Levels%2520on%2520the%2520Concentration-Dependent%2520Permeability%2520of%2520Drugs%2520to%2520the%2520Cell%2520Membrane%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2008%26volume%3D97%26spage%3D553%26epage%3D565%26doi%3D10.1002%2Fjps.21114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. 3CL protease inhibitors for SARS-CoV-1 with electrophilic arylketone warhead. (A) Structures of 3CL<sup>pro</sup> inhibitors <b>1</b>, SH-5, YH-53, and YH-71. SH-5 contains a tripeptide scaffold with a warhead and a carbamoyl unit at P4. YH-53 and YH-71 consist of a dipeptide scaffold with a warhead and a heteroaromatic unit at P3. (B) Proposed mechanism of inhibition by SH-5. Once SH-5 is bound to the enzyme, the active site Cys145 of 3CL<sup>pro</sup> attacks the ketone of SH-5 to afford a reversible covalent bond. The hemithioketal intermediate would then be stabilized by an oxyanion hole.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. SARS-CoV-2 3CL<sup>pro</sup> inhibitory assay. (A) Concentration-dependent inhibition of SARS-CoV-2 3CL<sup>pro</sup> by SH-5, YH-53 and YH-71. Reactions were monitored for 10 min. Data points represent mean values ± SEM from three independent experiments. <i>K</i><sub>i</sub> values were calculated using the Cheng–Prusoff equation and are noted in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The <i>K</i><sub>m</sub> value of the substrate was 48.4 μM. (B) Lineweaver–Burk plot for SH-5 inhibition of SARS-CoV-2 3CL<sup>pro</sup> at 15, 30, 45, 60, and 75 μM of fluorogenic substrate and 0, 10, 20, 30, 40, and 50 nM of the inhibitor SH-5. Three independent experiments were performed, and reactions were monitored for 10 min. The common intercept on the ordinate indicates a competitive inhibition type. (C) Global fit of kinetic data from (B) for competitive enzyme inhibition. The best fit value of <i>K</i><sub><i>i</i></sub> 19.8 ± 2.3 nM was in the same range as the value calculated by the Cheng–Prusoff equation (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of SARS-CoV-2 3CL<sup>pro</sup> with its inhibitors. (A, B) Inhibitor binding sites of 3CL<sup>pro</sup> for YH-53 (A) and SH-5 (B). Protein carbons for YH-53 and SH-5 complexes are shown in green and yellow, respectively. YH-53 (A) and SH-5 (B) are in cyan and orange, respectively. Water molecules are labeled with “w”. (C) LSQ superposition of the YH-53 and SH-5 complexes using 281 CA atoms (RMSD = 0.594 Å). Cyan arrows show the shifts of loop residues. Atom color scheme is the same as that in panels (A) and (B). (D, E) Surface representations of SH-5 and YH-53 binding sites. Atom color scheme is the same as that in panels (A) and (B), but water molecules are in salmon pink. (F) Distribution of averaged Cα-atom displacements of residues 188–194 in 3CL<sup>pro</sup>. We performed least-squares fittings of 3CL<sup>pro</sup> structures obtained from the PDB onto the present 3CL<sup>pro</sup>-YH53 structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E18">7E18</a>) with residues 10–290 (Cα atoms) using program LSQKAB in the CCP4 program suite,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and 345 3CL<sup>pro</sup> structures with the RMSD values less than 2.0 Å were selected for the following calculation. The averaged Cα-atom displacement of residues 188–194 was calculated for each superposed structure using the LSQKAB output that contains Cα-atom displacements of the superposed residues, and the histogram was prepared. Twelve structures indicated with the red bracket have loop structures similar to the YH-53 complex. The others have loop structures similar to the inhibitor-free form.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Anti-SARS-CoV-2 activities of SH-5, YH-53 and YH-71 in Vero cells. (A) Antiviral activity of SH-5, YH-53, and YH-71. Vero cells were infected with SARS-CoV-2 using a multiplicity of infection (MOI) of 0.01. The inhibitors SH-5, YH-53, and YH-71 (1–30 μM) were treated to the infected cells then incubated for 48 h at 37 °C. The viral/serum mixtures were transferred to the flesh Vero cell and incubated for 24 h at 37 °C, followed by subject to the plaque assay. Data points represent mean values ± SEM from three independent experiments. (B) Total viral RNA of infected cells treated with YH-53 was analyzed by qPCR. Data points represent mean values ± SEM from three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/medium/jm1c00665_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mean plasma concentration–time profile of YH-53 in rats. (A) Mean plasma concentration–time profile of YH-53 in rats after i.v. administration. YH-53 (0.1 mg/kg; 4 mL/kg) was administered by an i.v. bolus injection in rats. Inset depicts the same data on a semilogarithmic scale. Data are shown as the mean ± SD (<i>n</i> = 5). (B) Mean plasma concentration–time profile of YH-53 in rats after oral administration. YH-53 (0.5 mg/kg; 4 mL/kg) was administered orally in rats. Inset depicts the same data on a semilogarithmic scale. Data are shown as the mean ± SD (<i>n</i> = 5).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00665/20210727/images/large/jm1c00665_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00665&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13118" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13118" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span> <span> </span><span class="NLM_article-title">A Novel Coronavirus from Patients with Pneumonia in China, 2019</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>382</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2001017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1056%2FNEJMoa2001017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=31978945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjslGmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2020&pages=727-733&issue=8&author=N.+Zhuauthor=D.+Zhangauthor=W.+Wangauthor=X.+Liauthor=B.+Yangauthor=J.+Songauthor=X.+Zhaoauthor=B.+Huangauthor=W.+Shiauthor=R.+Luauthor=P.+Niuauthor=F.+Zhanauthor=X.+Maauthor=D.+Wangauthor=W.+Xuauthor=G.+Wuauthor=G.+F.+Gaoauthor=W.+Tan&title=A+Novel+Coronavirus+from+Patients+with+Pneumonia+in+China%2C+2019&doi=10.1056%2FNEJMoa2001017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A novel coronavirus from patients with pneumonia in China, 2019</span></div><div class="casAuthors">Zhu, Na; Zhang, Dingyu; Wang, Wenling; Li, Xingwang; Yang, Bo; Song, Jingdong; Zhao, Xiang; Huang, Baoying; Shi, Weifeng; Lu, Roujian; Niu, Peihua; Zhan, Faxian; Ma, Xuejun; Wang, Dayan; Xu, Wenbo; Wu, Guizhen; Gao, George F.; Tan, Wenjie</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">727-733</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">In Dec. 2019, a cluster of patients with pneumonia of unknown cause was linked to a seafood wholesale market in Wuhan, China.  A previously unknown betacoronavirus was discovered through the use of unbiased sequencing in samples from patients with pneumonia.  Human airway epithelial cells were used to isolate a novel coronavirus, named 2019-nCoV, which formed a clade within the subgenus sarbecovirus, Orthocoronavirinae subfamily.  Different from both MERS-CoV and SARS-CoV, 2019-nCoV is the seventh member of the family of coronaviruses that infect humans.  Enhanced surveillance and further investigation are ongoing.  Complete genome sequences of the three novel coronaviruses were submitted to GISAID (BetaCoV/Wuhan/ IVDC-HB-01/2019, accession ID: EPI_ISL_402119; BetaCoV/Wuhan/IVDC-HB-04/2020, accession ID: EPI_ISL_402120; BetaCoV/Wuhan/IVDC-HB-05/2019, accession ID: EPI_ISL_402121).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibbAn_VqfXbVg90H21EOLACvtfcHk0lgP2kWAvgyJxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjslGmsrc%253D&md5=73cc5c839e1e934da69b39537063b7b3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2001017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2001017%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DR.%26aulast%3DNiu%26aufirst%3DP.%26aulast%3DZhan%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DTan%26aufirst%3DW.%26atitle%3DA%2520Novel%2520Coronavirus%2520from%2520Patients%2520with%2520Pneumonia%2520in%2520China%252C%25202019%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D382%26issue%3D8%26spage%3D727%26epage%3D733%26doi%3D10.1056%2FNEJMoa2001017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z. L.</span></span> <span> </span><span class="NLM_article-title">A Pneumonia Outbreak Associated with a New Coronavirus of Probable Bat Origin.</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span> (<span class="NLM_issue">7798</span>),  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2012-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1038%2Fs41586-020-2012-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32015507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=270-273&issue=7798&author=P.+Zhouauthor=X.+L.+Yangauthor=X.+G.+Wangauthor=B.+Huauthor=L.+Zhangauthor=W.+Zhangauthor=H.+R.+Siauthor=Y.+Zhuauthor=B.+Liauthor=C.+L.+Huangauthor=H.+D.+Chenauthor=J.+Chenauthor=Y.+Luoauthor=H.+Guoauthor=R.+Di+Jiangauthor=M.+Q.+Liuauthor=Y.+Chenauthor=X.+R.+Shenauthor=X.+Wangauthor=X.+S.+Zhengauthor=K.+Zhaoauthor=Q.+J.+Chenauthor=F.+Dengauthor=L.+L.+Liuauthor=B.+Yanauthor=F.+X.+Zhanauthor=Y.+Y.+Wangauthor=G.+F.+Xiaoauthor=Z.+L.+Shi&title=A+Pneumonia+Outbreak+Associated+with+a+New+Coronavirus+of+Probable+Bat+Origin.&doi=10.1038%2Fs41586-020-2012-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span></div><div class="casAuthors">Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">270-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Since the outbreak of severe acute respiratory syndrome (SARS) 18 years ago, a large no. of SARS-related coronaviruses (SARSr-CoVs) have been discovered in their natural reservoir host, bats1-4.  Previous studies have shown that some bat SARSr-CoVs have the potential to infect humans5-7.  Here we report the identification and characterization of a new coronavirus (2019-nCoV), which caused an epidemic of acute respiratory syndrome in humans in Wuhan, China.  The epidemic, which started on 12 Dec. 2019, had caused 2,794 lab.-confirmed infections including 80 deaths by 26 Jan. 2020.  Full-length genome sequences were obtained from five patients at an early stage of the outbreak.  The sequences are almost identical and share 79.6% sequence identity to SARS-CoV.  Furthermore, we show that 2019-nCoV is 96% identical at the whole-genome level to a bat coronavirus.  Pairwise protein sequence anal. of seven conserved non-structural proteins domains show that this virus belongs to the species of SARSr-CoV.  In addn., 2019-nCoV virus isolated from the bronchoalveolar lavage fluid of a critically ill patient could be neutralized by sera from several patients.  Notably, we confirmed that 2019-nCoV uses the same cell entry receptor-angiotensin converting enzyme II (ACE2)-as SARS-CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_Wb_iUsbFrVg90H21EOLACvtfcHk0lgicDTLo5Nr6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D&md5=236f17d4d3c7978d72513e5e0258f1b3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2012-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2012-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DX.%2BG.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSi%26aufirst%3DH.%2BR.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.%2BL.%26aulast%3DChen%26aufirst%3DH.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DDi%2BJiang%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DM.%2BQ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DX.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DQ.%2BJ.%26aulast%3DDeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DL.%2BL.%26aulast%3DYan%26aufirst%3DB.%26aulast%3DZhan%26aufirst%3DF.%2BX.%26aulast%3DWang%26aufirst%3DY.%2BY.%26aulast%3DXiao%26aufirst%3DG.%2BF.%26aulast%3DShi%26aufirst%3DZ.%2BL.%26atitle%3DA%2520Pneumonia%2520Outbreak%2520Associated%2520with%2520a%2520New%2520Coronavirus%2520of%2520Probable%2520Bat%2520Origin.%26jtitle%3DNature%26date%3D2020%26volume%3D579%26issue%3D7798%26spage%3D270%26epage%3D273%26doi%3D10.1038%2Fs41586-020-2012-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, E. H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T. T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowling, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span> <span> </span><span class="NLM_article-title">Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>382</i></span> (<span class="NLM_issue">13</span>),  <span class="NLM_fpage">1199</span>– <span class="NLM_lpage">1207</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2001316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1056%2FNEJMoa2001316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=31995857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmt1Whtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2020&pages=1199-1207&issue=13&author=Q.+Liauthor=X.+Guanauthor=P.+Wuauthor=X.+Wangauthor=L.+Zhouauthor=Y.+Tongauthor=R.+Renauthor=K.+S.+M.+Leungauthor=E.+H.+Y.+Lauauthor=J.+Y.+Wongauthor=X.+Xingauthor=N.+Xiangauthor=Y.+Wuauthor=C.+Liauthor=Q.+Chenauthor=D.+Liauthor=T.+Liuauthor=J.+Zhaoauthor=M.+Liuauthor=W.+Tuauthor=C.+Chenauthor=L.+Jinauthor=R.+Yangauthor=Q.+Wangauthor=S.+Zhouauthor=R.+Wangauthor=H.+Liuauthor=Y.+Luoauthor=Y.+Liuauthor=G.+Shaoauthor=H.+Liauthor=Z.+Taoauthor=Y.+Yangauthor=Z.+Dengauthor=B.+Liuauthor=Z.+Maauthor=Y.+Zhangauthor=G.+Shiauthor=T.+T.+Y.+Lamauthor=J.+T.+Wuauthor=G.+F.+Gaoauthor=B.+J.+Cowlingauthor=B.+Yangauthor=G.+M.+Leungauthor=Z.+Feng&title=Early+Transmission+Dynamics+in+Wuhan%2C+China%2C+of+Novel+Coronavirus%E2%80%93Infected+Pneumonia&doi=10.1056%2FNEJMoa2001316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Early transmission dynamics in Wuhan, China, of novel coronavirus-infected pneumonia</span></div><div class="casAuthors">Li, Qun; Guan, Xuhua; Wu, Peng; Wang, Xiaoye; Zhou, Lei; Tong, Yeqing; Ren, Ruiqi; Leung, Kathy S. M.; Lau, Eric H. Y.; Wong, Jessica Y.; Xing, Xuesen; Xiang, Nijuan; Wu, Yang; Li, Chao; Chen, Qi; Li, Dan; Liu, Tian; Zhao, Jing; Liu, Man; Tu, Wenxiao; Chen, Chuding; Jin, Lianmei; Yang, Rui; Wang, Qi; Zhou, Suhua; Wang, Rui; Liu, Hui; Luo, Yinbo; Liu, Yuan; Shao, Ge; Li, Huan; Tao, Zhongfa; Yang, Yang; Deng, Zhiqiang; Liu, Boxi; Ma, Zhitao; Zhang, Yanping; Shi, Guoqing; Lam, Tommy T. Y.; Wu, Joseph T.; Gao, George F.; Cowling, Benjamin J.; Yang, Bo; Leung, Gabriel M.; Feng, Zijian</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">382</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1199-1207</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">The initial cases of novel coronavirus (2019-nCoV)-infected pneumonia (NCIP) occurred in Wuhan, Hubei Province, China, in Dec. 2019 and Jan. 2020.  We analyzed data on the 1st 425 confirmed cases in Wuhan to det. the epidemiol. characteristics of NCIP.  We collected information on demog. characteristics, exposure history, and illness timelines of lab.-confirmed cases of NCIP that had been reported by Jan. 22, 2020.  We described characteristics of the cases and estd. the key epidemiol. time-delay distributions.  In the early period of exponential growth, we estd. the epidemic doubling time and the basic reproductive no.  Among the 1st 425 patients with confirmed NCIP, the median age was 59 yr and 56% were male.  The majority of cases (55%) with onset before Jan. 1, 2020, were linked to the Huanan Seafood Wholesale Market, as compared with 8.6% of the subsequent cases.  The mean incubation period was 5.2 days, with the 95th percentile of the distribution at 12.5 days.  In its early stages, the epidemic doubled in size every 7.4 days.  With a mean serial interval of 7.5 days, the basic reproductive no. was estd. to be 2.2.  On the basis of this information, there is evidence that human-to-human transmission has occurred among close contacts since the middle of Dec. 2019.  Considerable efforts to reduce transmission will be required to control outbreaks if similar dynamics apply elsewhere.  Measures to prevent or reduce transmission should be implemented in populations at risk.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD4bpM4aGXN7Vg90H21EOLACvtfcHk0lgicDTLo5Nr6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmt1Whtrw%253D&md5=d82c8e8a173f6ab879d332f0abb7228e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2001316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2001316%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DGuan%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DR.%26aulast%3DLeung%26aufirst%3DK.%2BS.%2BM.%26aulast%3DLau%26aufirst%3DE.%2BH.%2BY.%26aulast%3DWong%26aufirst%3DJ.%2BY.%26aulast%3DXing%26aufirst%3DX.%26aulast%3DXiang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DTu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DG.%26aulast%3DLam%26aufirst%3DT.%2BT.%2BY.%26aulast%3DWu%26aufirst%3DJ.%2BT.%26aulast%3DGao%26aufirst%3DG.%2BF.%26aulast%3DCowling%26aufirst%3DB.%2BJ.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DLeung%26aufirst%3DG.%2BM.%26aulast%3DFeng%26aufirst%3DZ.%26atitle%3DEarly%2520Transmission%2520Dynamics%2520in%2520Wuhan%252C%2520China%252C%2520of%2520Novel%2520Coronavirus%25E2%2580%2593Infected%2520Pneumonia%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D382%26issue%3D13%26spage%3D1199%26epage%3D1207%26doi%3D10.1056%2FNEJMoa2001316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokudo, N.</span></span> <span> </span><span class="NLM_article-title">Sustaining Containment of COVID-19: Global Sharing for Pandemic Response</span>. <i>Glob. Heal. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.35772/ghm.2020.01040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.35772%2Fghm.2020.01040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33330776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BB3szosl2lsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2020&pages=53-55&author=H.+Mitsuyaauthor=N.+Kokudo&title=Sustaining+Containment+of+COVID-19%3A+Global+Sharing+for+Pandemic+Response&doi=10.35772%2Fghm.2020.01040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Sustaining containment of COVID-19: global sharing for pandemic response</span></div><div class="casAuthors">Mitsuya Hiroaki; Kokudo Norihiro</div><div class="citationInfo"><span class="NLM_cas:title">Global health & medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">53-55</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) outbreak, caused by SARS-CoV-2, has rapidly escalated into a global pandemic.  One of our significant concerns is that we have no data as to whether people, who acquired immunity against this deadly virus and recovered from COVID-19, are protected from further infections with the same virus.  Moreover, we have no data as to whether this pandemic will persist in our societies and continue vexing us for long periods of time.  Implementing science-based response strategy is essential to sustain containment of COVID-19 globally.  Rapidly sharing scientific information means providing real-time research data and relevant findings.  As an international academic journal, Global Health & Medicine publishes this special issue entitled "GHM Special Topic: COVID-19".  It includes a range of articles describing COVID-19 based on frontline data from Japan, China, the United States, Italy, the United Kingdom, and other countries and areas worldwide.  Our hope is that the rapid publication and sharing of information contribute, in whichever possible way, to this global fight against COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRUcmPsNTJyj0HdcurJClFcfW6udTcc2ebA-bgHgq4H6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3szosl2lsg%253D%253D&md5=ebe9eeb276bf878b93ac2cf54f284007</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.35772%2Fghm.2020.01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.35772%252Fghm.2020.01040%26sid%3Dliteratum%253Aachs%26aulast%3DMitsuya%26aufirst%3DH.%26aulast%3DKokudo%26aufirst%3DN.%26atitle%3DSustaining%2520Containment%2520of%2520COVID-19%253A%2520Global%2520Sharing%2520for%2520Pandemic%2520Response%26jtitle%3DGlob.%2520Heal.%2520Med.%26date%3D2020%26volume%3D2%26spage%3D53%26epage%3D55%26doi%3D10.35772%2Fghm.2020.01040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">A New Coronavirus Associated with Human Respiratory Disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.+M.+Chenauthor=W.+Wangauthor=Z.+G.+Songauthor=Y.+Huauthor=Z.+W.+Taoauthor=J.+H.+Tianauthor=Y.+Y.+Peiauthor=M.+L.+Yuanauthor=Y.+L.+Zhangauthor=F.+H.+Daiauthor=Y.+Liuauthor=Q.+M.+Wangauthor=J.+J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.+Z.+Zhang&title=A+New+Coronavirus+Associated+with+Human+Respiratory+Disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0lhv5VCtjWXFig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.%2BM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.%2BG.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.%2BW.%26aulast%3DTian%26aufirst%3DJ.%2BH.%26aulast%3DPei%26aufirst%3DY.%2BY.%26aulast%3DYuan%26aufirst%3DM.%2BL.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DDai%26aufirst%3DF.%2BH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%2BM.%26aulast%3DZheng%26aufirst%3DJ.%2BJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.%2BZ.%26atitle%3DA%2520New%2520Coronavirus%2520Associated%2520with%2520Human%2520Respiratory%2520Disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. H.</span></span> <span> </span><span class="NLM_article-title">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">6595</span>– <span class="NLM_lpage">6628</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01461</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01461" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6595-6628&author=T.+Pillaiyarauthor=M.+Manickamauthor=V.+Namasivayamauthor=Y.+Hayashiauthor=S.+H.+Jung&title=An+Overview+of+Severe+Acute+Respiratory+Syndrome-Coronavirus+%28SARS-CoV%29+3CL+Protease+Inhibitors%3A+Peptidomimetics+and+Small+Molecule+Chemotherapy&doi=10.1021%2Facs.jmedchem.5b01461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Severe Acute Respiratory Syndrome-Coronavirus (SARS-CoV) 3CL Protease Inhibitors: Peptidomimetics and Small Molecule Chemotherapy</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Manickam, Manoj; Namasivayam, Vigneshwaran; Hayashi, Yoshio; Jung, Sang-Hun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6595-6628</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome (SARS) is caused by a newly emerged coronavirus that infected more than 8000 individuals and resulted in more than 800 (10-15%) fatalities in 2003.  The causative agent of SARS has been identified as a novel human coronavirus (SARS-CoV), and its viral protease, SARS-CoV 3CLpro, has been shown to be essential for replication and has hence been recognized as a potent drug target for SARS infection.  Currently, there is no effective treatment for this epidemic despite the intensive research that has been undertaken since 2003 (over 3500 publications).  This perspective focuses on the status of various efficacious anti-SARS-CoV 3CLpro chemotherapies discovered during the last 12 years (2003-2015) from all sources, including lab. synthetic methods, natural products, and virtual screening.  We describe here mainly peptidomimetic and small mol. inhibitors of SARS-CoV 3CLpro.  Attempts have been made to provide a complete description of the structural features and binding modes of these inhibitors under many conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB_JCXe-m0RLVg90H21EOLACvtfcHk0lhv5VCtjWXFig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVyns7c%253D&md5=6ce37076204280f64a7f362633fcee1a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01461%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DManickam%26aufirst%3DM.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DJung%26aufirst%3DS.%2BH.%26atitle%3DAn%2520Overview%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome-Coronavirus%2520%2528SARS-CoV%2529%25203CL%2520Protease%2520Inhibitors%253A%2520Peptidomimetics%2520and%2520Small%2520Molecule%2520Chemotherapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6595%26epage%3D6628%26doi%3D10.1021%2Facs.jmedchem.5b01461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preiser, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Werf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodt, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabenau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panning, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolesnikova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouchier, R. A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burguière, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinatl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eickmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escriou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grywna, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramme, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuguerra, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickerts, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klenk, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterhaus, A. D. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, H. W.</span></span> <span> </span><span class="NLM_article-title">Identification of a Novel Coronavirus in Patients with Severe Acute Respiratory Syndrome</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1967</span>– <span class="NLM_lpage">1976</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa030747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1056%2FNEJMoa030747" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=12690091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslajurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2003&pages=1967-1976&author=C.+Drostenauthor=S.+G%C3%BCntherauthor=W.+Preiserauthor=S.+van+der%0AWerfauthor=H.-R.+Brodtauthor=S.+Beckerauthor=H.+Rabenauauthor=M.+Panningauthor=L.+Kolesnikovaauthor=R.+A.+M.+Fouchierauthor=A.+Bergerauthor=A.-M.+Burgui%C3%A8reauthor=J.+Cinatlauthor=M.+Eickmannauthor=N.+Escriouauthor=K.+Grywnaauthor=S.+Krammeauthor=J.-C.+Manuguerraauthor=S.+M%C3%BCllerauthor=V.+Rickertsauthor=M.+St%C3%BCrmerauthor=S.+Viethauthor=H.-D.+Klenkauthor=A.+D.+M.+E.+Osterhausauthor=H.+Schmitzauthor=H.+W.+Doerr&title=Identification+of+a+Novel+Coronavirus+in+Patients+with+Severe+Acute+Respiratory+Syndrome&doi=10.1056%2FNEJMoa030747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel coronavirus in patients with severe acute respiratory syndrome</span></div><div class="casAuthors">Drosten, Christian; Guenther, Stephan; Preiser, Wolfgang; van der Werf, Sylvie; Brodt, Hans-Reinhard; Becker, Stephan; Rabenau, Holger; Panning, Marcus; Kolesnikova, Larissa; Fouchier, Ron A. M.; Berger, Annemarie; Burguiere, Ana-Maria; Cinatl, Jindrich; Eickmann, Markus; Escriou, Nicolas; Grywna, Klaus; Kramme, Stefanie; Manuguerra, Jean-Claude; Mueller, Stefanie; Rickerts, Volker; Stuermer, Martin; Vieth, Simon; Klenk, Hans-Dieter; Osterhaus, Albert D. M. E.; Schmitz, Herbert; Doerr, Hans Wilheim</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">1967-1976</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">The severe acute respiratory syndrome (SARS) has recently been identified as a new clin. entity.  SARS is thought to be caused by an unknown infectious agent.  Clin. specimens from patients with SARS were searched for unknown viruses with the use of cell cultures and mol. techniques.  A novel coronavirus was identified in patients with SARS.  The virus was isolated in cell culture, and a sequence 300 nucleotides in length was obtained by a PCR (PCR)-based random-amplification procedure.  Genetic characterization indicated that the virus is only distantly related to known coronaviruses (identical in 50 to 60% of the nucleotide sequence).  On the basis of the obtained sequence, conventional and real-time PCR assays for specific and sensitive detection of the novel virus were established.  Virus was detected in a variety of clin. specimens from patients with SARS but not in controls.  High concns. of viral RNA of up to 100 million mols. per mL were found in sputum.  Viral RNA was also detected at extremely low concns. in plasma during the acute phase and in feces during the late convalescent phase.  Infected patients showed seroconversion on the Vero cells in which the virus was isolated.  The novel coronavirus might have a role in causing SARS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqN8uHpwOG07Vg90H21EOLACvtfcHk0liLyqXUrM8jbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslajurw%253D&md5=3838a7a01273f688e53dc86088bfa9db</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa030747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa030747%26sid%3Dliteratum%253Aachs%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DG%25C3%25BCnther%26aufirst%3DS.%26aulast%3DPreiser%26aufirst%3DW.%26aulast%3Dvan%2Bder%2BWerf%26aufirst%3DS.%26aulast%3DBrodt%26aufirst%3DH.-R.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRabenau%26aufirst%3DH.%26aulast%3DPanning%26aufirst%3DM.%26aulast%3DKolesnikova%26aufirst%3DL.%26aulast%3DFouchier%26aufirst%3DR.%2BA.%2BM.%26aulast%3DBerger%26aufirst%3DA.%26aulast%3DBurgui%25C3%25A8re%26aufirst%3DA.-M.%26aulast%3DCinatl%26aufirst%3DJ.%26aulast%3DEickmann%26aufirst%3DM.%26aulast%3DEscriou%26aufirst%3DN.%26aulast%3DGrywna%26aufirst%3DK.%26aulast%3DKramme%26aufirst%3DS.%26aulast%3DManuguerra%26aufirst%3DJ.-C.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DRickerts%26aufirst%3DV.%26aulast%3DSt%25C3%25BCrmer%26aufirst%3DM.%26aulast%3DVieth%26aufirst%3DS.%26aulast%3DKlenk%26aufirst%3DH.-D.%26aulast%3DOsterhaus%26aufirst%3DA.%2BD.%2BM.%2BE.%26aulast%3DSchmitz%26aufirst%3DH.%26aulast%3DDoerr%26aufirst%3DH.%2BW.%26atitle%3DIdentification%2520of%2520a%2520Novel%2520Coronavirus%2520in%2520Patients%2520with%2520Severe%2520Acute%2520Respiratory%2520Syndrome%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2003%26volume%3D348%26spage%3D1967%26epage%3D1976%26doi%3D10.1056%2FNEJMoa030747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byun, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Identification of Antiviral Drug Candidates against SARS-CoV-2 from FDA-Approved Drugs</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">e00819</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1128/AAC.00819-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FAAC.00819-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32366720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslGgsLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2020&pages=e00819-20&author=S.+Jeonauthor=M.+Koauthor=J.+Leeauthor=I.+Choiauthor=S.+Y.+Byunauthor=S.+Parkauthor=D.+Shumauthor=S.+Kim&title=Identification+of+Antiviral+Drug+Candidates+against+SARS-CoV-2+from+FDA-Approved+Drugs&doi=10.1128%2FAAC.00819-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of antiviral drug candidates against SARS-CoV-2 from FDA-approved drugs</span></div><div class="casAuthors">Jeon, Sangeun; Ko, Meehyun; Lee, Jihye; Choi, Inhee; Byun, Soo Young; Park, Soonju; Shum, David; Kim, Seungtaek</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e00819</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Drug repositioning is the only feasible option to immediately address the COVID-19 global challenge.  We screened a panel of 48 FDA-approved drugs against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which were preselected by an assay of SARS-CoV.  We identified 24 potential antiviral drug candidates against SARS-CoV-2 infection.  Some drug candidates showed very low 50% inhibitory concns. (IC50s), and in particular, two FDA-approved drugs-niclosamide and ciclesonide-were notable in some respects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI441p4IDYQLVg90H21EOLACvtfcHk0liLyqXUrM8jbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslGgsLvN&md5=f3e0bb95ed274876838945ef5952cc54</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2FAAC.00819-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00819-20%26sid%3Dliteratum%253Aachs%26aulast%3DJeon%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DI.%26aulast%3DByun%26aufirst%3DS.%2BY.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520Antiviral%2520Drug%2520Candidates%2520against%2520SARS-CoV-2%2520from%2520FDA-Approved%2520Drugs%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2020%26volume%3D64%26spage%3De00819%26epage%3D20%26doi%3D10.1128%2FAAC.00819-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jan, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T. J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juang, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shie, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chein, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span> <span> </span><span class="NLM_article-title">Identification of Existing Pharmaceuticals and Herbal Medicines as Inhibitors of SARS-CoV-2 Infection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">e2021579118</span>, <span class="refDoi"> DOI: 10.1073/pnas.2021579118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1073%2Fpnas.2021579118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33452205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXjtV2lsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2021&pages=e2021579118&author=J.+T.+Janauthor=T.+J.+R.+Chengauthor=Y.+P.+Juangauthor=H.+H.+Maauthor=Y.+T.+Wuauthor=W.+B.+Yangauthor=C.+W.+Chengauthor=X.+Chenauthor=T.+H.+Chouauthor=J.+J.+Shieauthor=W.+C.+Chengauthor=R.+J.+Cheinauthor=S.+S.+Maoauthor=P.+H.+Liangauthor=C.+Maauthor=S.+C.+Hungauthor=C.+H.+Wong&title=Identification+of+Existing+Pharmaceuticals+and+Herbal+Medicines+as+Inhibitors+of+SARS-CoV-2+Infection&doi=10.1073%2Fpnas.2021579118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of existing pharmaceuticals and herbal medicines as inhibitors of SARS-CoV-2 infection</span></div><div class="casAuthors">Jan, Jia-Tsrong; Cheng, Ting-Jen Rachel; Juang, Yu-Pu; Ma, Hsiu-Hua; Wu, Ying-Ta; Yang, Wen-Bin; Cheng, Cheng-Wei; Chen, Xiaorui; Chou, Ting-Hung; Shie, Jiun-Jie; Cheng, Wei-Chieh; Chein, Rong-Jie; Mao, Shi-Shan; Liang, Pi-Hui; Ma, Che; Hung, Shang-Cheng; Wong, Chi-Huey</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e2021579118</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The outbreak of COVID-19 caused by SARS-CoV-2 has resulted in more than 50 million confirmed cases and over 1 million deaths worldwide as of Nov. 2020.  Currently, there are no effective antivirals approved by the Food and Drug Administration to contain this pandemic except the antiviral agent remdesivir.  In addn., the trimeric spike protein on the viral surface is highly glycosylated and almost 200,000 variants with mutations at more than 1,000 positions in its 1,273 amino acid sequence were reported, posing a major challenge in the development of antibodies and vaccines.  It is therefore urgently needed to have alternative and timely treatments for the disease.  In this study, we used a cell-based infection assay to screen more than 3,000 agents used in humans and animals, including 2,855 small mols. and 190 traditional herbal medicines, and identified 15 active small mols. in concns. ranging from 0.1 nM to 50 μM.  Two enzymic assays, along with mol. modeling, were then developed to confirm those targeting the virus 3CL protease and the RNA-dependent RNA polymerase.  Several water exts. of herbal medicines were active in the cell-based assay and could be further developed as plant-derived anti-SARS-CoV-2 agents.  Some of the active compds. identified in the screen were further tested in vivo, and it was found that mefloquine, nelfinavir, and exts. of Ganoderma lucidum (RF3), Perilla frutescens, and Mentha haplocalyx were effective in a challenge study using hamsters as disease model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDZ_HLt-yZjLVg90H21EOLACvtfcHk0liLyqXUrM8jbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXjtV2lsb8%253D&md5=c8a34583707cea0f0fc984c9a173d5f4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.2021579118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.2021579118%26sid%3Dliteratum%253Aachs%26aulast%3DJan%26aufirst%3DJ.%2BT.%26aulast%3DCheng%26aufirst%3DT.%2BJ.%2BR.%26aulast%3DJuang%26aufirst%3DY.%2BP.%26aulast%3DMa%26aufirst%3DH.%2BH.%26aulast%3DWu%26aufirst%3DY.%2BT.%26aulast%3DYang%26aufirst%3DW.%2BB.%26aulast%3DCheng%26aufirst%3DC.%2BW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChou%26aufirst%3DT.%2BH.%26aulast%3DShie%26aufirst%3DJ.%2BJ.%26aulast%3DCheng%26aufirst%3DW.%2BC.%26aulast%3DChein%26aufirst%3DR.%2BJ.%26aulast%3DMao%26aufirst%3DS.%2BS.%26aulast%3DLiang%26aufirst%3DP.%2BH.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DHung%26aufirst%3DS.%2BC.%26aulast%3DWong%26aufirst%3DC.%2BH.%26atitle%3DIdentification%2520of%2520Existing%2520Pharmaceuticals%2520and%2520Herbal%2520Medicines%2520as%2520Inhibitors%2520of%2520SARS-CoV-2%2520Infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2021%26volume%3D118%26spage%3De2021579118%26doi%3D10.1073%2Fpnas.2021579118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamarthi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupte, A.</span></span> <span> </span><span class="NLM_article-title">The Journey of Remdesivir: From Ebola to COVID-19</span>. <i>Drugs Context</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.7573/dic.2020-4-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.7573%2Fdic.2020-4-14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1-9&author=J.+Pardoauthor=A.+M.+Shuklaauthor=G.+Chamarthiauthor=A.+Gupte&title=The+Journey+of+Remdesivir%3A+From+Ebola+to+COVID-19&doi=10.7573%2Fdic.2020-4-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.7573%2Fdic.2020-4-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7573%252Fdic.2020-4-14%26sid%3Dliteratum%253Aachs%26aulast%3DPardo%26aufirst%3DJ.%26aulast%3DShukla%26aufirst%3DA.%2BM.%26aulast%3DChamarthi%26aufirst%3DG.%26aulast%3DGupte%26aufirst%3DA.%26atitle%3DThe%2520Journey%2520of%2520Remdesivir%253A%2520From%2520Ebola%2520to%2520COVID-19%26jtitle%3DDrugs%2520Context%26date%3D2020%26volume%3D9%26spage%3D1%26epage%3D9%26doi%3D10.7573%2Fdic.2020-4-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">U.S. Food & Drug Administration</span>.  <i>FDA approves
first treatment for COVID-19</i>. <a href="https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19" class="extLink">https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19</a> (accessed 2021-01-21).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Food+%26+Drug+Administration.+FDA+approves%0Afirst+treatment+for+COVID-19.+https%3A%2F%2Fwww.fda.gov%2Fnews-events%2Fpress-announcements%2Ffda-approves-first-treatment-covid-19+%28accessed+2021-01-21%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DFDA%2520approves%250Afirst%2520treatment%2520for%2520COVID-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of SARS-CoV-2 Main Protease Provides a Basis for Design of Improved a-Ketoamide Inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+Structure+of+SARS-CoV-2+Main+Protease+Provides+a+Basis+for+Design+of+Improved+a-Ketoamide+Inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0lhLBFut4z7MwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520Structure%2520of%2520SARS-CoV-2%2520Main%2520Protease%2520Provides%2520a%2520Basis%2520for%2520Design%2520of%2520Improved%2520a-Ketoamide%2520Inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Antiviral Drug Candidates Targeting the SARS-CoV-2 Main Protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.+M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.+K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-Based+Design+of+Antiviral+Drug+Candidates+Targeting+the+SARS-CoV-2+Main+Protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0lhLBFut4z7MwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.%2BM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%2BK.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-Based%2520Design%2520of%2520Antiviral%2520Drug%2520Candidates%2520Targeting%2520the%2520SARS-CoV-2%2520Main%2520Protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meenakshisundaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span> <span> </span><span class="NLM_article-title">Recent Discovery and Development of Inhibitors Targeting Coronaviruses</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">688</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.drudis.2020.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32006468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFKgtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2020&pages=668-688&author=T.+Pillaiyarauthor=S.+Meenakshisundaramauthor=M.+Manickam&title=Recent+Discovery+and+Development+of+Inhibitors+Targeting+Coronaviruses&doi=10.1016%2Fj.drudis.2020.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Recent discovery and development of inhibitors targeting coronaviruses</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Meenakshisundaram, Sangeetha; Manickam, Manoj</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">668-688</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Human coronaviruses (CoVs) are enveloped viruses with a pos.-sense single-stranded RNA genome.  Currently, six human CoVs have been reported including human coronavirus 229E (HCoV-229E), OC43 (HCoV-OC43), NL63 (HCoV-NL63), HKU1 (HCoV-HKU1), severe acute respiratory syndrome (SARS) coronavirus (SARS-CoV), and MiddleEast respiratory syndrome (MERS) coronavirus (MERS-CoV).  They cause moderate to severe respiratory and intestinal infections in humans.  In this review, we focus on recent advances in the research and development of small-mol. anti-human coronavirus therapies targeting different stages of the CoV life cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGaJIxrfUjnLVg90H21EOLACvtfcHk0lhLBFut4z7MwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFKgtro%253D&md5=08faec7e1ec7df3a3f6d4c4e34fe0807</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DMeenakshisundaram%26aufirst%3DS.%26aulast%3DManickam%26aufirst%3DM.%26atitle%3DRecent%2520Discovery%2520and%2520Development%2520of%2520Inhibitors%2520Targeting%2520Coronaviruses%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26volume%3D25%26spage%3D668%26epage%3D688%26doi%3D10.1016%2Fj.drudis.2020.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">From SARS to MERS: Crystallographic Studies on Coronaviral Proteases Enable Antiviral Drug Design</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">4085</span>– <span class="NLM_lpage">4096</span>, <span class="refDoi"> DOI: 10.1111/febs.12936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1111%2Ffebs.12936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=25039866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2014&pages=4085-4096&author=R.+Hilgenfeld&title=From+SARS+to+MERS%3A+Crystallographic+Studies+on+Coronaviral+Proteases+Enable+Antiviral+Drug+Design&doi=10.1111%2Ffebs.12936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">From SARS to MERS: crystallographic studies on coronaviral proteases enable antiviral drug design</span></div><div class="casAuthors">Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4085-4096</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Here, the author focuses on the important contributions that macromol. crystallog. has made over the past 12 yr to elucidating structures and mechanisms of the essential proteases of coronaviruses, the main protease (Mpro) and the papain-like protease (PLpro).  The role of x-ray crystallog. in structure-assisted drug discovery against these targets is discussed.  Aspects dealt with in this review include the emergence of the SARS coronavirus in 2002-2003 and of the MERS coronavirus 10 yr later and the origins of these viruses.  The crystal structure of the free SARS coronavirus Mpro and its dependence on pH is discussed, as are efforts to design inhibitors on the basis of these structures.  The mechanism of maturation of the enzyme from the viral polyprotein is still a matter of debate.  The crystal structure of the SARS coronavirus PLpro and its complex with ubiquitin is also discussed, as is its ortholog from MERS coronavirus.  Efforts at predictive structure-based inhibitor development for bat coronavirus Mpros to increase the preparedness against zoonotic transmission to man are described as well.  The paper closes with a brief discussion of structure-based discovery of antivirals in an academic setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy_Y5ir6KZ8LVg90H21EOLACvtfcHk0ljAT2X7y7SOag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbI&md5=6a9330976a5c3a13dd4bf7bed14665ce</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12936%26sid%3Dliteratum%253Aachs%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DFrom%2520SARS%2520to%2520MERS%253A%2520Crystallographic%2520Studies%2520on%2520Coronaviral%2520Proteases%2520Enable%2520Antiviral%2520Drug%2520Design%26jtitle%3DFEBS%2520J.%26date%3D2014%26volume%3D281%26spage%3D4085%26epage%3D4096%26doi%3D10.1111%2Ffebs.12936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishii, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Autoprocessing Mechanism of Severe Acute Respiratory Syndrome Coronavirus 3C-like Protease (SARS-CoV 3CLpro) from Its Polyproteins</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">2002</span>– <span class="NLM_lpage">2013</span>, <span class="refDoi"> DOI: 10.1111/febs.12222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1111%2Ffebs.12222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23452147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsFWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=2002-2013&author=T.+Muramatsuauthor=Y.+T.+Kimauthor=W.+Nishiiauthor=T.+Teradaauthor=M.+Shirouzuauthor=S.+Yokoyama&title=Autoprocessing+Mechanism+of+Severe+Acute+Respiratory+Syndrome+Coronavirus+3C-like+Protease+%28SARS-CoV+3CLpro%29+from+Its+Polyproteins&doi=10.1111%2Ffebs.12222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Autoprocessing mechanism of severe acute respiratory syndrome coronavirus 3C-like protease (SARS-CoV 3CLpro) from its polyproteins</span></div><div class="casAuthors">Muramatsu, Tomonari; Kim, Yong-Tae; Nishii, Wataru; Terada, Takaho; Shirouzu, Mikako; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2002-2013</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Like many other RNA viruses, severe acute respiratory syndrome coronavirus (SARS-CoV) produces polyproteins contg. several non-structural proteins, which are then processed by the viral proteases.  These proteases often exist within the polyproteins, and are excised by their own proteolytic activity (autoprocessing').  It is important to investigate the autoprocessing mechanism of these proteases from the point of view of anti-SARS-CoV drug design.  In this paper, we describe a new method for investigating the autoprocessing mechanism of the main protease (Mpro), which is also called the 3C-like protease (3CLpro).  Using our method, we measured the activities, under the same conditions, of the mature form and pro-forms with the N-terminal pro-sequence, the C-terminal pro-sequence or both pro-sequences, toward the pro-form with both N- and C-terminal pro-sequences.  The data indicate that the pro-forms of the enzyme have proteolytic activity, and are stimulated by the same proteolytic activity.  The stimulation occurs in two steps, with approx. eightfold stimulation by N-terminal cleavage, approx. fourfold stimulation by C-terminal cleavage, and 23-fold stimulation by the cleavage of both termini, compared to the pro-form with both the N- and C-terminal pro-sequences.  Such cleavage mainly occurs in a trans manner; i.e. the pro-form dimer cleaves the monomeric form.  The stimulation by N-terminal pro-sequence removal is due to the cis (intra-dimer and inter-protomer) effect of formation of the new N-terminus, whereas that by C-terminal cleavage is due to removal of its trans (inter-dimer) inhibitory effect.  A numerical simulation of the maturation pathway is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2_Wx5y0FDxLVg90H21EOLACvtfcHk0ljAT2X7y7SOag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsFWguro%253D&md5=20a698c60419d73fdb9921ec371371f0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12222%26sid%3Dliteratum%253Aachs%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DY.%2BT.%26aulast%3DNishii%26aufirst%3DW.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DAutoprocessing%2520Mechanism%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520Coronavirus%25203C-like%2520Protease%2520%2528SARS-CoV%25203CLpro%2529%2520from%2520Its%2520Polyproteins%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D2002%26epage%3D2013%26doi%3D10.1111%2Ffebs.12222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">28134</span>– <span class="NLM_lpage">28140</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.095851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1074%2Fjbc.M109.095851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=20489209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVOnsLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=28134-28140&author=C.+Liauthor=Y.+Qiauthor=X.+Tengauthor=Z.+Yangauthor=P.+Weiauthor=C.+Zhangauthor=L.+Tanauthor=L.+Zhouauthor=Y.+Liuauthor=L.+Lai&title=Maturation+Mechanism+of+Severe+Acute+Respiratory+Syndrome+%28SARS%29+Coronavirus+3C-like+Proteinase&doi=10.1074%2Fjbc.M109.095851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Maturation Mechanism of Severe Acute Respiratory Syndrome (SARS) Coronavirus 3C-like Proteinase</span></div><div class="casAuthors">Li, Chunmei; Qi, Yifei; Teng, Xin; Yang, Zongchang; Wei, Ping; Zhang, Changsheng; Tan, Lei; Zhou, Lu; Liu, Ying; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">28134-28140</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The 3C-like proteinase (3CLpro) of the severe acute respiratory syndrome (SARS) coronavirus plays a vital role in virus maturation and is proposed to be a key target for drug design against SARS.  Various in vitro studies revealed that only the dimer of the matured 3CLpro is active.  However, as the internally encoded 3CLpro gets matured from the replicase polyprotein by autolytic cleavage at both the N-terminal and the C-terminal flanking sites, it is unclear whether the polyprotein also needs to dimerize first for its autocleavage reaction.  We constructed a large protein contg. the cyan fluorescent protein (C), the N-terminal flanking substrate peptide of SARS 3CLpro (XX), SARS 3CLpro (3CLP), and the yellow fluorescent protein (Y) to study the autoprocessing of 3CLpro using fluorescence resonance energy transfer.  In contrast to the matured 3CLpro, the polyprotein, as well as the one-step digested product, 3CLP-Y-His, were shown to be monomeric in gel filtration and analytic ultracentrifuge anal.  However, dimers can still be induced and detected when incubating these large proteins with a substrate analog compd. in both chem. crosslinking expts. and analytic ultracentrifuge anal.  We also measured enzyme activity under different enzyme concns. and found a clear tendency of substrate-induced dimer formation.  Based on these discoveries, we conclude that substrate-induced dimerization is essential for the activity of SARS-3CLpro in the polyprotein, and a modified model for the 3CLpro maturation process was proposed.  As many viral proteases undergo a similar maturation process, this model might be generally applicable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos2gN4o9DHBrVg90H21EOLACvtfcHk0ljZ_ePnvNl4gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVOnsLjM&md5=669fba32d04912c7e084570f98d7c1c7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.095851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.095851%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DMaturation%2520Mechanism%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520%2528SARS%2529%2520Coronavirus%25203C-like%2520Proteinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D28134%26epage%3D28140%26doi%3D10.1074%2Fjbc.M109.095851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindner, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotouhi-Ardakani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lytvyn, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachance, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulea, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménard, R.</span></span> <span> </span><span class="NLM_article-title">The Papain-Like Protease from the Severe Acute Respiratory Syndrome Coronavirus Is a Deubiquitinating Enzyme</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">15199</span>– <span class="NLM_lpage">15208</span>, <span class="refDoi"> DOI: 10.1128/JVI.79.24.15199-15208.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FJVI.79.24.15199-15208.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=16306591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XhslOitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=15199-15208&author=H.+A.+Lindnerauthor=N.+Fotouhi-Ardakaniauthor=V.+Lytvynauthor=P.+Lachanceauthor=T.+Suleaauthor=R.+M%C3%A9nard&title=The+Papain-Like+Protease+from+the+Severe+Acute+Respiratory+Syndrome+Coronavirus+Is+a+Deubiquitinating+Enzyme&doi=10.1128%2FJVI.79.24.15199-15208.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The papain-like protease from the severe acute respiratory syndrome coronavirus is a deubiquitinating enzyme</span></div><div class="casAuthors">Lindner, Holger A.; Fotouhi-Ardakani, Nasser; Lytvyn, Viktoria; Lachance, Paule; Sulea, Traian; Menard, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15199-15208</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The severe acute respiratory syndrome coronavirus papain-like protease (SARS-CoV PLpro) is involved in the processing of the viral polyprotein and, thereby, contributes to the biogenesis of the virus replication complex.  Structural bioinformatics has revealed a relationship for the SARS-CoV PLpro to herpesvirus-assocd. ubiquitin-specific protease (HAUSP), a ubiquitin-specific protease, indicating potential deubiquitinating activity in addn. to its function in polyprotein processing.  In order to confirm this prediction, we overexpressed and purified SARS-CoV PLpro (amino acids [aa]1507 to 1858) from Escherichia coli.  The purified enzyme hydrolyzed ubiquitin-7-amino-4-methylcoumarin (Ub-AMC), a general deubiquitinating enzyme substrate, with a catalytic efficiency of 13,100 M-1 s-1, 220-fold more efficiently than the small synthetic peptide substrate Z-LRGG-AMC, which incorporates the C-terminal four residues of ubiquitin.  In addn., SARS-CoV PLpro was inhibited by the specific deubiquitinating enzyme inhibitor ubiquitin aldehyde, with an inhibition const. of 210 nM.  The purified SARS-CoV PLpro disassembles branched polyubiquitin chains with lengths of two to seven (Ub2-7) or four (Ub4) units, which involves isopeptide bond cleavage.  SARS-CoV PLpro processing activity was also detected against a protein fused to the C terminus of the ubiquitin-like modifier ISG15, both in vitro using the purified enzyme and in HeLa cells by coexpression with SARS-CoV PLpro (aa 1198 to 2009).  These results clearly establish that SARS-CoV PLpro is a deubiquitinating enzyme, thereby confirming our earlier prediction.  This unexpected activity for a coronavirus papain-like protease suggests a novel viral strategy to modulate the host cell ubiquitination machinery to its advantage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1m1bqW9GpybVg90H21EOLACvtfcHk0ljZ_ePnvNl4gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhslOitw%253D%253D&md5=dff5e5e3d3b2fc8838413a8551480383</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.24.15199-15208.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.24.15199-15208.2005%26sid%3Dliteratum%253Aachs%26aulast%3DLindner%26aufirst%3DH.%2BA.%26aulast%3DFotouhi-Ardakani%26aufirst%3DN.%26aulast%3DLytvyn%26aufirst%3DV.%26aulast%3DLachance%26aufirst%3DP.%26aulast%3DSulea%26aufirst%3DT.%26aulast%3DM%25C3%25A9nard%26aufirst%3DR.%26atitle%3DThe%2520Papain-Like%2520Protease%2520from%2520the%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520Coronavirus%2520Is%2520a%2520Deubiquitinating%2520Enzyme%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D15199%26epage%3D15208%26doi%3D10.1128%2FJVI.79.24.15199-15208.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barretto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jukneliene, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">The Papain-Like Protease of Severe Acute Respiratory Syndrome Coronavirus Has Deubiquitinating Activity</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">15189</span>– <span class="NLM_lpage">15198</span>, <span class="refDoi"> DOI: 10.1128/JVI.79.24.15189-15198.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FJVI.79.24.15189-15198.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=16306590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XhslOitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2005&pages=15189-15198&author=N.+Barrettoauthor=D.+Juknelieneauthor=K.+Ratiaauthor=Z.+Chenauthor=A.+D.+Mesecarauthor=S.+C.+Baker&title=The+Papain-Like+Protease+of+Severe+Acute+Respiratory+Syndrome+Coronavirus+Has+Deubiquitinating+Activity&doi=10.1128%2FJVI.79.24.15189-15198.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The papain-like protease of severe acute respiratory syndrome coronavirus has deubiquitinating activity</span></div><div class="casAuthors">Barretto, Naina; Jukneliene, Dalia; Ratia, Kiira; Chen, Zhongbin; Mesecar, Andrew D.; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15189-15198</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Replication of the genomic RNA of severe acute respiratory syndrome coronavirus (SARS-CoV) is mediated by replicase polyproteins that are processed by two viral proteases, papain-like protease (PLpro) and 3C-like protease (3CLpro).  Previously, we showed that SARS-CoV PLpro processes the replicase polyprotein at three conserved cleavage sites.  Here, we report the identification and characterization of a 316-amino-acid catalytic core domain of PLpro that can efficiently cleave replicase substrates in trans-cleavage assays and peptide substrates in fluorescent resonance energy transfer-based protease assays.  We performed bioinformatics anal. on 16 papain-like protease domains from nine different coronaviruses and identified a putative catalytic triad (Cys1651-His1812-Asp1826) and zinc-binding site.  Mutagenesis studies revealed that Asp1826 and the four cysteine residues involved in zinc binding are essential for SARS-CoV PLpro activity.  Mol. modeling of SARS-CoV PLpro suggested that this catalytic core may also have deubiquitinating activity.  We tested this hypothesis by measuring the deubiquitinating activity of PLpro by two independent assays.  SARS CoV-PLpro hydrolyzed both diubiquitin and ubiquitin-7-amino-4-methylcoumarin (AMC) substrates, and hydrolysis of ubiquitin-AMC is approx. 180-fold more efficient than hydrolysis of a peptide substrate that mimics the PLpro replicase recognition sequence.  To investigate the crit. determinants recognized by PLpro, we performed site-directed mutagenesis on the P6 to P2' residues at each of the three PLpro cleavage sites.  We found that PLpro recognizes the consensus cleavage sequence LXGG, which is also the consensus sequence recognized by cellular deubiquitinating enzymes.  This similarity in the substrate recognition sites should be considered during the development of SARS-CoV PLpro inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiQGj9bzOcILVg90H21EOLACvtfcHk0ljZ_ePnvNl4gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhslOitg%253D%253D&md5=d9e028b3addd30d53050758db977d207</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FJVI.79.24.15189-15198.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.79.24.15189-15198.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBarretto%26aufirst%3DN.%26aulast%3DJukneliene%26aufirst%3DD.%26aulast%3DRatia%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DThe%2520Papain-Like%2520Protease%2520of%2520Severe%2520Acute%2520Respiratory%2520Syndrome%2520Coronavirus%2520Has%2520Deubiquitinating%2520Activity%26jtitle%3DJ.%2520Virol.%26date%3D2005%26volume%3D79%26spage%3D15189%26epage%3D15198%26doi%3D10.1128%2FJVI.79.24.15189-15198.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hegyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span> <span> </span><span class="NLM_article-title">Conservation of Substrate Specificities among Coronavirus Main Proteases</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1099/0022-1317-83-3-595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1099%2F0022-1317-83-3-595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=11842254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BD387ktV2ntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2002&pages=595-599&author=A.+Hegyiauthor=J.+Ziebuhr&title=Conservation+of+Substrate+Specificities+among+Coronavirus+Main+Proteases&doi=10.1099%2F0022-1317-83-3-595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation of substrate specificities among coronavirus main proteases</span></div><div class="casAuthors">Hegyi Annette; Ziebuhr John</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of general virology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">Pt 3</span>),
    <span class="NLM_cas:pages">595-599</span>
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    </div><div class="casAbstract">The key enzyme in coronavirus replicase polyprotein processing is the coronavirus main protease, 3CL(pro).  The substrate specificities of five coronavirus main proteases, including the prototypic enzymes from the coronavirus groups I, II and III, were characterized.  Recombinant main proteases of human coronavirus (HCoV), transmissible gastroenteritis virus (TGEV), feline infectious peritonitis virus, avian infectious bronchitis virus and mouse hepatitis virus (MHV) were tested in peptide-based trans-cleavage assays.  The determination of relative rate constants for a set of corresponding HCoV, TGEV and MHV 3CL(pro) cleavage sites revealed a conserved ranking of these sites.  Furthermore, a synthetic peptide representing the N-terminal HCoV 3CL(pro) cleavage site was shown to be effectively hydrolysed by noncognate main proteases.  The data show that the differential cleavage kinetics of sites within pp1a/pp1ab are a conserved feature of coronavirus main proteases and lead us to predict similar processing kinetics for the replicase polyproteins of all coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTF5bW4pZ-I-QHqQV89nF-2fW6udTcc2eblN7ZScnHNtrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD387ktV2ntg%253D%253D&md5=7c6aa15224a074bca51ea82d12f2f827</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1099%2F0022-1317-83-3-595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F0022-1317-83-3-595%26sid%3Dliteratum%253Aachs%26aulast%3DHegyi%26aufirst%3DA.%26aulast%3DZiebuhr%26aufirst%3DJ.%26atitle%3DConservation%2520of%2520Substrate%2520Specificities%2520among%2520Coronavirus%2520Main%2520Proteases%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2002%26volume%3D83%26spage%3D595%26epage%3D599%26doi%3D10.1099%2F0022-1317-83-3-595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendt, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manickam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easwaran, M.</span></span> <span> </span><span class="NLM_article-title">The Recent Outbreaks of Human Coronaviruses: A Medicinal Chemistry Perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1002/med.21724</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fmed.21724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32852058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVagt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=72-135&author=T.+Pillaiyarauthor=L.+L.+Wendtauthor=M.+Manickamauthor=M.+Easwaran&title=The+Recent+Outbreaks+of+Human+Coronaviruses%3A+A+Medicinal+Chemistry+Perspective&doi=10.1002%2Fmed.21724"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The recent outbreaks of human coronaviruses: A medicinal chemistry perspective</span></div><div class="casAuthors">Pillaiyar, Thanigaimalai; Wendt, Lukas L.; Manickam, Manoj; Easwaran, Maheswaran</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-135</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Coronaviruses (CoVs) infect both humans and animals.  In humans, CoVs can cause respiratory, kidney, heart, brain, and intestinal infections that can range from mild to lethal.  Since the start of the 21st century, three β-coronaviruses have crossed the species barrier to infect humans: severe-acute respiratory syndrome (SARS)-CoV-1, Middle East respiratory syndrome (MERS)-CoV, and SARS-CoV-2 (2019-nCoV).  These viruses are dangerous and can easily be transmitted from human to human.  Therefore, the development of anticoronaviral therapies is urgently needed.  However, to date, no approved vaccines or drugs against CoV infections are available.  In this review, we focus on the medicinal chem. efforts toward the development of antiviral agents against SARS-CoV-1, MERS-CoV, SARS-CoV-2, targeting biochem. events important for viral replication and its life cycle.  These targets include the spike glycoprotein and its host-receptors for viral entry, proteases that are essential for cleaving polyproteins to produce functional proteins, and RNA-dependent RNA polymerase for viral RNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowH3Dpqna3Q7Vg90H21EOLACvtfcHk0liiXvV8WI8-NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVagt7Y%253D&md5=efe56354f1182ce8c16cede75e04842b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fmed.21724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21724%26sid%3Dliteratum%253Aachs%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DWendt%26aufirst%3DL.%2BL.%26aulast%3DManickam%26aufirst%3DM.%26aulast%3DEaswaran%26aufirst%3DM.%26atitle%3DThe%2520Recent%2520Outbreaks%2520of%2520Human%2520Coronaviruses%253A%2520A%2520Medicinal%2520Chemistry%2520Perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D41%26spage%3D72%26epage%3D135%26doi%3D10.1002%2Fmed.21724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sydnes, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Glutamic Acid and Glutamine Peptides Possessing a Trifluoromethyl Ketone Group as SARS-CoV 3CL Protease Inhibitors</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">8601</span>– <span class="NLM_lpage">8609</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2006.06.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.tet.2006.06.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32287416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvVSmtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=8601-8609&author=M.+O.+Sydnesauthor=Y.+Hayashiauthor=V.+K.+Sharmaauthor=T.+Hamadaauthor=U.+Bachaauthor=J.+Barrilaauthor=E.+Freireauthor=Y.+Kiso&title=Synthesis+of+Glutamic+Acid+and+Glutamine+Peptides+Possessing+a+Trifluoromethyl+Ketone+Group+as+SARS-CoV+3CL+Protease+Inhibitors&doi=10.1016%2Fj.tet.2006.06.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of glutamic acid and glutamine peptides possessing a trifluoromethyl ketone group as SARS-CoV 3CL protease inhibitors</span></div><div class="casAuthors">Sydnes, Magne O.; Hayashi, Yoshio; Sharma, Vinay K.; Hamada, Takashi; Bacha, Usman; Barrila, Jennifer; Freire, Ernesto; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">8601-8609</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Trifluoromethyl-β-amino alc. I was synthesized in five steps starting from Cbz-L-Glu-OH where the key step involved the introduction of the trifluoromethyl (CF3) group to oxazolidinone II (Cbz = benzyloxycarbonyl), resulting in the formation of silyl ether III.  Compd. I was then converted into four tri- and tetraglutamic acid and glutamine peptides possessing a CF3-ketone group that exhibited inhibitory activity against severe acute respiratory syndrome coronavirus protease (SARS-CoV 3CLpro).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZcF2qvPRx9rVg90H21EOLACvtfcHk0liiXvV8WI8-NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvVSmtbY%253D&md5=fcab218b800eac06f32827f6fdac23ce</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2006.06.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2006.06.052%26sid%3Dliteratum%253Aachs%26aulast%3DSydnes%26aufirst%3DM.%2BO.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DSharma%26aufirst%3DV.%2BK.%26aulast%3DHamada%26aufirst%3DT.%26aulast%3DBacha%26aufirst%3DU.%26aulast%3DBarrila%26aufirst%3DJ.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DSynthesis%2520of%2520Glutamic%2520Acid%2520and%2520Glutamine%2520Peptides%2520Possessing%2520a%2520Trifluoromethyl%2520Ketone%2520Group%2520as%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%26jtitle%3DTetrahedron%26date%3D2006%26volume%3D62%26spage%3D8601%26epage%3D8609%26doi%3D10.1016%2Fj.tet.2006.06.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bacha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrila, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelli, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mario Amzel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Development of Broad-Spectrum Halomethyl Ketone Inhibitors against Coronavirus Main Protease 3CLpro</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1111/j.1747-0285.2008.00679.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1111%2Fj.1747-0285.2008.00679.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=18611220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslymu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2008&pages=34-49&author=U.+Bachaauthor=J.+Barrilaauthor=S.+B.+Gabelliauthor=Y.+Kisoauthor=L.+Mario+Amzelauthor=E.+Freire&title=Development+of+Broad-Spectrum+Halomethyl+Ketone+Inhibitors+against+Coronavirus+Main+Protease+3CLpro&doi=10.1111%2Fj.1747-0285.2008.00679.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Development of broad-spectrum halomethyl ketone inhibitors against coronavirus main protease 3CLpro</span></div><div class="casAuthors">Bacha, Usman; Barrila, Jennifer; Gabelli, Sandra B.; Kiso, Yoshiaki; Amzel, L. Mario; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">34-49</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Coronaviruses comprise a large group of RNA viruses with diverse host specificity.  The emergence of highly pathogenic strains like the SARS coronavirus (SARS-Co-V), and the discovery of two new coronaviruses, NL-63 and HKU1, corroborates the high rate of mutation and recombination that have enabled them to cross species barriers and infect novel hosts.  For that reason, the development of broad-spectrum antivirals that are effective against several members of this family is highly desirable.  This goal can be accomplished by designing inhibitors against a target, such as the main protease 3CLpro (Mpro), which is highly conserved among all coronaviruses.  Here 3CLpro derived from the SARS-Co-V was used as the primary target to identify a new class of inhibitors contg. a halomethyl ketone warhead.  The compds. are highly potent against SARS 3CLpro with Ki's as low as 300 nM.  The crystal structure of the complex of one of the compds. with 3CLpro indicates that this inhibitor forms a thioether linkage between the halomethyl carbon of the warhead and the catalytic Cys 145.  Furthermore, Structure Activity Relationship (SAR) studies of these compds. have led to the identification of a pharmacophore that accurately defines the essential mol. features required for the high affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIBzjnE5XmbVg90H21EOLACvtfcHk0likc94Qp83kWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslymu7s%253D&md5=db8382f357a5be0fffe340b113664fd5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2008.00679.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2008.00679.x%26sid%3Dliteratum%253Aachs%26aulast%3DBacha%26aufirst%3DU.%26aulast%3DBarrila%26aufirst%3DJ.%26aulast%3DGabelli%26aufirst%3DS.%2BB.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DMario%2BAmzel%26aufirst%3DL.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DDevelopment%2520of%2520Broad-Spectrum%2520Halomethyl%2520Ketone%2520Inhibitors%2520against%2520Coronavirus%2520Main%2520Protease%25203CLpro%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2008%26volume%3D72%26spage%3D34%26epage%3D49%26doi%3D10.1111%2Fj.1747-0285.2008.00679.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Regnier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacha, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span> <span> </span><span class="NLM_article-title">New Developments for the Design, Synthesis and Biological Evaluation of Potent SARS-CoV 3CLpro Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2722</span>– <span class="NLM_lpage">2727</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.bmcl.2009.03.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=19362479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltl2rs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2722-2727&author=T.+Regnierauthor=D.+Sarmaauthor=K.+Hidakaauthor=U.+Bachaauthor=E.+Freireauthor=Y.+Hayashiauthor=Y.+Kiso&title=New+Developments+for+the+Design%2C+Synthesis+and+Biological+Evaluation+of+Potent+SARS-CoV+3CLpro+Inhibitors&doi=10.1016%2Fj.bmcl.2009.03.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">New developments for the design, synthesis and biological evaluation of potent SARS-CoV 3CLpro inhibitors</span></div><div class="casAuthors">Regnier, Thomas; Sarma, Diganta; Hidaka, Koushi; Bacha, Usman; Freire, Ernesto; Hayashi, Yoshio; Kiso, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2722-2727</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of trifluoromethyl, benzothiazolyl or thiazolyl ketone-contg. peptidic compds. as SARS-CoV 3CLpro protease inhibitors were developed and their potency was evaluated by in vitro protease inhibitory assays.  Three candidates had encouraging results for the development of new anti-SARS compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9aRtN4lboCrVg90H21EOLACvtfcHk0likc94Qp83kWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltl2rs7o%253D&md5=37d3aac3723a2a8e6417600f24c62e8b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.118%26sid%3Dliteratum%253Aachs%26aulast%3DRegnier%26aufirst%3DT.%26aulast%3DSarma%26aufirst%3DD.%26aulast%3DHidaka%26aufirst%3DK.%26aulast%3DBacha%26aufirst%3DU.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DKiso%26aufirst%3DY.%26atitle%3DNew%2520Developments%2520for%2520the%2520Design%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%2520Potent%2520SARS-CoV%25203CLpro%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2722%26epage%3D2727%26doi%3D10.1016%2Fj.bmcl.2009.03.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanigaimalai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kakiuchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamazaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of New Tripeptide-Type SARS-CoV 3CL Protease Inhibitors Containing an Electrophilic Arylketone Moiety</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">412</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2012.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.bmc.2012.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23245752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCkurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=412-424&author=S.+Konnoauthor=P.+Thanigaimalaiauthor=T.+Yamamotoauthor=K.+Nakadaauthor=R.+Kakiuchiauthor=K.+Takayamaauthor=Y.+Yamazakiauthor=F.+Yakushijiauthor=K.+Akajiauthor=Y.+Kisoauthor=Y.+Kawasakiauthor=S.+E.+Chenauthor=E.+Freireauthor=Y.+Hayashi&title=Design+and+Synthesis+of+New+Tripeptide-Type+SARS-CoV+3CL+Protease+Inhibitors+Containing+an+Electrophilic+Arylketone+Moiety&doi=10.1016%2Fj.bmc.2012.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of new tripeptide-type SARS-CoV 3CL protease inhibitors containing an electrophilic arylketone moiety</span></div><div class="casAuthors">Konno, Sho; Thanigaimalai, Pillaiyar; Yamamoto, Takehito; Nakada, Kiyohiko; Kakiuchi, Rie; Takayama, Kentaro; Yamazaki, Yuri; Yakushiji, Fumika; Akaji, Kenichi; Kiso, Yoshiaki; Kawasaki, Yuko; Chen, Shen-En; Freire, Ernesto; Hayashi, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">412-424</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We describe here the design, synthesis and biol. evaluation of a series of mols. toward the development of novel peptidomimetic inhibitors of SARS-CoV 3CLpro.  A docking study involving binding between the initial lead compd. (I) and the SARS-CoV 3CLpro motivated the replacement of a thiazole with a benzothiazole unit as a warhead moiety at the P1' site.  This modification led to the identification of more potent derivs., including (II) (R1 = R2 = H; R1 = OMe, R2 = H; R1 = H, R2 = OMe; R1 = NMe2, H2 = H; R1 = H, R2 = NMe2), with IC50 or Ki values in the submicromolar to nanomolar range.  In particular, compds. II (R1 = R2 = H; R1 = H, R2 = NMe2) exhibited the most potent inhibitory activities, with Ki values of 4.1 and 3.1 nM, resp.  The peptidomimetic compds. identified through this process are attractive leads for the development of potential therapeutic agents against SARS.  The structural requirements of the peptidomimetics with potent inhibitory activities against SARS-CoV 3CLpro may be summarized as follows: (i) the presence of a benzothiazole warhead at the S1'-position; (ii) hydrogen bonding capabilities at the cyclic lactam of the S1-site; (iii) appropriate stereochem. and hydrophobic moiety size at the S2-site and (iv) a unique folding conformation assumed by the phenoxyacetyl moiety at the S4-site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlvN_nxqEIZLVg90H21EOLACvtfcHk0likc94Qp83kWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCkurvN&md5=9aee15f25e1a6276b8973da835472e65</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DKonno%26aufirst%3DS.%26aulast%3DThanigaimalai%26aufirst%3DP.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DNakada%26aufirst%3DK.%26aulast%3DKakiuchi%26aufirst%3DR.%26aulast%3DTakayama%26aufirst%3DK.%26aulast%3DYamazaki%26aufirst%3DY.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%2BE.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520New%2520Tripeptide-Type%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%2520Containing%2520an%2520Electrophilic%2520Arylketone%2520Moiety%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D412%26epage%3D424%26doi%3D10.1016%2Fj.bmc.2012.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thanigaimalai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koiwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiso, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naser-Tavakolian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schön, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of Novel Dipeptide-Type SARS-CoV 3CL Protease Inhibitors: Structure-Activity Relationship Study</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2013.05.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.ejmech.2013.05.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23747811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFemt7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2013&pages=436-447&author=P.+Thanigaimalaiauthor=S.+Konnoauthor=T.+Yamamotoauthor=Y.+Koiwaiauthor=A.+Taguchiauthor=K.+Takayamaauthor=F.+Yakushijiauthor=K.+Akajiauthor=Y.+Kisoauthor=Y.+Kawasakiauthor=S.+E.+Chenauthor=A.+Naser-Tavakolianauthor=A.+Sch%C3%B6nauthor=E.+Freireauthor=Y.+Hayashi&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+Novel+Dipeptide-Type+SARS-CoV+3CL+Protease+Inhibitors%3A+Structure-Activity+Relationship+Study&doi=10.1016%2Fj.ejmech.2013.05.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and biological evaluation of novel dipeptide-type SARS-CoV 3CL protease inhibitors: Structure-activity relationship study</span></div><div class="casAuthors">Thanigaimalai, Pillaiyar; Konno, Sho; Yamamoto, Takehito; Koiwai, Yuji; Taguchi, Akihiro; Takayama, Kentaro; Yakushiji, Fumika; Akaji, Kenichi; Kiso, Yoshiaki; Kawasaki, Yuko; Chen, Shen-En; Naser-Tavakolian, Aurash; Schon, Arne; Freire, Ernesto; Hayashi, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">436-447</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">This work describes the design, synthesis, and evaluation of low-mol. wt. peptidic SARS-CoV 3CL protease inhibitors.  The inhibitors were designed based on the potent tripeptidic Z-Val-Leu-Ala(pyrrolidone-3-yl)-2-benzothiazole (I) (Ki = 4.1 nM), in which the P3 valine unit was substituted with a variety of distinct moieties.  The resulting series of dipeptide-type inhibitors displayed moderate to good inhibitory activities against 3CLpro.  In particular, compds. (II) (R1 = OMe, R2 = H and R1 = H, R2 = OMe) exhibited good inhibitory activities with Ki values of 0.39 and 0.33 μM, resp.  These low-mol. wt. compds. are attractive leads for the further development of potent peptidomimetic inhibitors with pharmaceutical profiles.  Docking studies were performed to model the binding interaction of the compd. II (R1 = OMe, R2 = H) with the SARS-CoV 3CL protease.  The preliminary SAR study of the peptidomimetic compds. with potent inhibitory activities revealed several structural features that boosted the inhibitory activity: (i) a benzothiazole warhead at the S1' position, (ii) a γ-lactam unit at the S1-position, (iii) an appropriately hydrophobic leucine moiety at the S2-position, and (iv) a hydrogen bond between the N-arylglycine unit and a backbone hydrogen bond donor at the S3-position.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3-wamiD_-5LVg90H21EOLACvtfcHk0lhB-MZnS4-jvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFemt7nP&md5=622466d52d5a7def121ee16b9ac4423b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2013.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2013.05.005%26sid%3Dliteratum%253Aachs%26aulast%3DThanigaimalai%26aufirst%3DP.%26aulast%3DKonno%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DKoiwai%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DA.%26aulast%3DTakayama%26aufirst%3DK.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DKiso%26aufirst%3DY.%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DS.%2BE.%26aulast%3DNaser-Tavakolian%26aufirst%3DA.%26aulast%3DSch%25C3%25B6n%26aufirst%3DA.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%2520Novel%2520Dipeptide-Type%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%253A%2520Structure-Activity%2520Relationship%2520Study%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D65%26spage%3D436%26epage%3D447%26doi%3D10.1016%2Fj.ejmech.2013.05.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thanigaimalai, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koiwai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taguchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakushiji, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaji, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naser-Tavakolian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schön, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span> <span> </span><span class="NLM_article-title">Development of Potent Dipeptide-Type SARS-CoV 3CL Protease Inhibitors with Novel P3 Scaffolds: Design, Synthesis, Biological Evaluation, and Docking Studies</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">372</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2013.07.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.ejmech.2013.07.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=23994330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ShsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2013&pages=372-384&author=P.+Thanigaimalaiauthor=S.+Konnoauthor=T.+Yamamotoauthor=Y.+Koiwaiauthor=A.+Taguchiauthor=K.+Takayamaauthor=F.+Yakushijiauthor=K.+Akajiauthor=S.+E.+Chenauthor=A.+Naser-Tavakolianauthor=A.+Sch%C3%B6nauthor=E.+Freireauthor=Y.+Hayashi&title=Development+of+Potent+Dipeptide-Type+SARS-CoV+3CL+Protease+Inhibitors+with+Novel+P3+Scaffolds%3A+Design%2C+Synthesis%2C+Biological+Evaluation%2C+and+Docking+Studies&doi=10.1016%2Fj.ejmech.2013.07.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Development of potent dipeptide-type SARS-CoV 3CL protease inhibitors with novel P3 scaffolds: Design, synthesis, biological evaluation, and docking studies</span></div><div class="casAuthors">Thanigaimalai, Pillaiyar; Konno, Sho; Yamamoto, Takehito; Koiwai, Yuji; Taguchi, Akihiro; Takayama, Kentaro; Yakushiji, Fumika; Akaji, Kenichi; Chen, Shen-En; Naser-Tavakolian, Aurash; Schon, Arne; Freire, Ernesto; Hayashi, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">372-384</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">We report the design and synthesis of a series of dipeptide-type inhibitors with novel P3 scaffolds that display potent inhibitory activity against SARS-CoV 3CLpro.  A docking study involving binding between the dipeptidic lead compd. (I) and 3CLpro suggested the modification of a structurally flexible P3 N-(3-methoxyphenyl)glycine with various rigid P3 moieties in I.  The modifications led to the identification of several potent derivs. with the inhibitory activities (Ki or IC50) in the submicromolar to nanomolar range.  Compd. (II) , in particular, displayed the most potent inhibitory activity, with a Ki value of 0.006 μM.  This potency was 65-fold higher than the potency of the lead compd. I (Ki = 0.39 μM).  In addn., the Ki value of II was in very good agreement with the binding affinity (16 nM) obsd. in isothermal titrn. calorimetry (ITC).  A SAR study around the P3 group in the lead 4 led to the identification of a rigid indole-2-carbonyl unit as one of the best P3 moieties (III).  Further optimization showed that a methoxy substitution at the 4-position on the indole unit was highly favorable for enhancing the inhibitory potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeJOG03zeobLVg90H21EOLACvtfcHk0lhB-MZnS4-jvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ShsrzL&md5=28055f45719f5220358fac55d015d2ff</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2013.07.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2013.07.037%26sid%3Dliteratum%253Aachs%26aulast%3DThanigaimalai%26aufirst%3DP.%26aulast%3DKonno%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DKoiwai%26aufirst%3DY.%26aulast%3DTaguchi%26aufirst%3DA.%26aulast%3DTakayama%26aufirst%3DK.%26aulast%3DYakushiji%26aufirst%3DF.%26aulast%3DAkaji%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DS.%2BE.%26aulast%3DNaser-Tavakolian%26aufirst%3DA.%26aulast%3DSch%25C3%25B6n%26aufirst%3DA.%26aulast%3DFreire%26aufirst%3DE.%26aulast%3DHayashi%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520Potent%2520Dipeptide-Type%2520SARS-CoV%25203CL%2520Protease%2520Inhibitors%2520with%2520Novel%2520P3%2520Scaffolds%253A%2520Design%252C%2520Synthesis%252C%2520Biological%2520Evaluation%252C%2520and%2520Docking%2520Studies%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D68%26spage%3D372%26epage%3D384%26doi%3D10.1016%2Fj.ejmech.2013.07.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breidenbach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillaiyar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäkel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Hamwi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gedschold, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiger, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namasivayam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sylvester, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thimm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vielmuth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zyulina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodem, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gütschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Targeting the Main Protease of SARS-CoV-2: From the Establishment of High Throughput Screening to the Design of Tailored Inhibitors</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10423</span>– <span class="NLM_lpage">10429</span>, <span class="refDoi"> DOI: 10.1002/anie.202016961</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fanie.202016961" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnt1Oht7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2021&pages=10423-10429&author=J.+Breidenbachauthor=C.+Lemkeauthor=T.+Pillaiyarauthor=L.+Sch%C3%A4kelauthor=G.+Al+Hamwiauthor=M.+Diettauthor=R.+Gedscholdauthor=N.+Geigerauthor=V.+Lopezauthor=S.+Mirzaauthor=V.+Namasivayamauthor=A.+C.+Schiedelauthor=K.+Sylvesterauthor=D.+Thimmauthor=C.+Vielmuthauthor=L.+P.+Vuauthor=M.+Zyulinaauthor=J.+Bodemauthor=M.+G%C3%BCtschowauthor=C.+E.+M%C3%BCller&title=Targeting+the+Main+Protease+of+SARS-CoV-2%3A+From+the+Establishment+of+High+Throughput+Screening+to+the+Design+of+Tailored+Inhibitors&doi=10.1002%2Fanie.202016961"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the main protease of SARS-CoV-2: from the establishment of high throughput screening to the design of tailored inhibitors</span></div><div class="casAuthors">Breidenbach, Julian; Lemke, Carina; Pillaiyar, Thanigaimalai; Schaekel, Laura; Al Hamwi, Ghazl; Diett, Miriam; Gedschold, Robin; Geiger, Nina; Lopez, Vittoria; Mirza, Salahuddin; Namasivayam, Vigneshwaran; Schiedel, Anke C.; Sylvester, Katharina; Thimm, Dominik; Vielmuth, Christin; Phuong Vu, Lan; Zyulina, Maria; Bodem, Jochen; Guetschow, Michael; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10423-10429</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The main protease of SARS-CoV-2 (Mpro), the causative agent of COVID-19, constitutes a significant drug target.  A new fluorogenic substrate was kinetically compared to an internally quenched fluorescent peptide and shown to be ideally suitable for high throughput screening with recombinantly expressed Mpro.  Two classes of protease inhibitors, azanitriles and pyridyl esters, were identified, optimized and subjected to in-depth biochem. characterization.  Tailored peptides equipped with the unique azanitrile warhead exhibited concomitant inhibition of Mpro and cathepsin L, a protease relevant for viral cell entry.  Pyridyl indole esters were analyzed by a positional scanning.  Our focused approach towards Mpro inhibitors proved to be superior to virtual screening.  With two irreversible inhibitors, azanitrile 8 (kinac/Ki=37 500 M-1 s-1, Ki=24.0 nM) and pyridyl ester 17 (kinac/Ki=29 100 M-1 s-1, Ki=10.0 nM), promising drug candidates for further development have been discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3x8Ey5-EjoLVg90H21EOLACvtfcHk0lhB-MZnS4-jvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnt1Oht7w%253D&md5=4687ae7d3e266627db5e5a03d6e87f06</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fanie.202016961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202016961%26sid%3Dliteratum%253Aachs%26aulast%3DBreidenbach%26aufirst%3DJ.%26aulast%3DLemke%26aufirst%3DC.%26aulast%3DPillaiyar%26aufirst%3DT.%26aulast%3DSch%25C3%25A4kel%26aufirst%3DL.%26aulast%3DAl%2BHamwi%26aufirst%3DG.%26aulast%3DDiett%26aufirst%3DM.%26aulast%3DGedschold%26aufirst%3DR.%26aulast%3DGeiger%26aufirst%3DN.%26aulast%3DLopez%26aufirst%3DV.%26aulast%3DMirza%26aufirst%3DS.%26aulast%3DNamasivayam%26aufirst%3DV.%26aulast%3DSchiedel%26aufirst%3DA.%2BC.%26aulast%3DSylvester%26aufirst%3DK.%26aulast%3DThimm%26aufirst%3DD.%26aulast%3DVielmuth%26aufirst%3DC.%26aulast%3DVu%26aufirst%3DL.%2BP.%26aulast%3DZyulina%26aufirst%3DM.%26aulast%3DBodem%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DTargeting%2520the%2520Main%2520Protease%2520of%2520SARS-CoV-2%253A%2520From%2520the%2520Establishment%2520of%2520High%2520Throughput%2520Screening%2520to%2520the%2520Design%2520of%2520Tailored%2520Inhibitors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2021%26volume%3D60%26spage%3D10423%26epage%3D10429%26doi%3D10.1002%2Fanie.202016961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashi-Kuwata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghavaiah, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulut, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anson, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lendy, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamatsu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takamune, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murayama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarchoan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodawer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misumi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuya, H.</span></span> <span> </span><span class="NLM_article-title">A Small Molecule Compound with an Indole Moiety Inhibits the Main Protease of SARS-CoV-2 and Blocks Virus Replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">668</span>, <span class="refDoi"> DOI: 10.1038/s41467-021-20900-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1038%2Fs41467-021-20900-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33510133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXivVWiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=668&author=S.+Hattoriauthor=N.+Higashi-Kuwataauthor=H.+Hayashiauthor=S.+R.+Alluauthor=J.+Raghavaiahauthor=H.+Bulutauthor=D.+Dasauthor=B.+J.+Ansonauthor=E.+K.+Lendyauthor=Y.+Takamatsuauthor=N.+Takamuneauthor=N.+Kishimotoauthor=K.+Murayamaauthor=K.+Hasegawaauthor=M.+Liauthor=D.+A.+Davisauthor=E.+N.+Kodamaauthor=R.+Yarchoanauthor=A.+Wlodawerauthor=S.+Misumiauthor=A.+D.+Mesecarauthor=A.+K.+Ghoshauthor=H.+Mitsuya&title=A+Small+Molecule+Compound+with+an+Indole+Moiety+Inhibits+the+Main+Protease+of+SARS-CoV-2+and+Blocks+Virus+Replication&doi=10.1038%2Fs41467-021-20900-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule compound with an indole moiety inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Hattori, Shin-ichiro; Higashi-Kuwata, Nobuyo; Hayashi, Hironori; Allu, Srinivasa Rao; Raghavaiah, Jakka; Bulut, Haydar; Das, Debananda; Anson, Brandon J.; Lendy, Emma K.; Takamatsu, Yuki; Takamune, Nobutoki; Kishimoto, Naoki; Murayama, Kazutaka; Hasegawa, Kazuya; Li, Mi; Davis, David A.; Kodama, Eiichi N.; Yarchoan, Robert; Wlodawer, Alexander; Misumi, Shogo; Mesecar, Andrew D.; Ghosh, Arun K.; Mitsuya, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">668</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Except remdesivir, no specific antivirals for SARS-CoV-2 infection are currently available.  Here, we characterize two small-mol.-compds., named GRL-1720 and 5h, contg. an indoline and indole moiety, resp., which target the SARS-CoV-2 main protease (Mpro).  We use VeroE6 cell-based assays with RNA-qPCR, cytopathic assays, and immunocytochem. and show both compds. to block the infectivity of SARS-CoV-2 with EC50 values of 15 ± 4 and 4.2 ± 0.7 μM for GRL-1720 and 5h, resp.  Remdesivir permitted viral breakthrough at high concns.; however, compd. 5h completely blocks SARS-CoV-2 infection in vitro without viral breakthrough or detectable cytotoxicity.  Combination of 5h and remdesivir exhibits synergism against SARS-CoV-2.  Addnl. X-ray structural anal. show that 5h forms a covalent bond with Mpro and makes polar interactions with multiple active site amino acid residues.  The present data suggest that 5h might serve as a lead Mpro inhibitor for the development of therapeutics for SARS-CoV-2 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb8PrKVV9DWbVg90H21EOLACvtfcHk0lgJ3VrbYZeoJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXivVWiurs%253D&md5=200388ac214c25024fb542da4a7cb6bf</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fs41467-021-20900-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-021-20900-6%26sid%3Dliteratum%253Aachs%26aulast%3DHattori%26aufirst%3DS.%26aulast%3DHigashi-Kuwata%26aufirst%3DN.%26aulast%3DHayashi%26aufirst%3DH.%26aulast%3DAllu%26aufirst%3DS.%2BR.%26aulast%3DRaghavaiah%26aufirst%3DJ.%26aulast%3DBulut%26aufirst%3DH.%26aulast%3DDas%26aufirst%3DD.%26aulast%3DAnson%26aufirst%3DB.%2BJ.%26aulast%3DLendy%26aufirst%3DE.%2BK.%26aulast%3DTakamatsu%26aufirst%3DY.%26aulast%3DTakamune%26aufirst%3DN.%26aulast%3DKishimoto%26aufirst%3DN.%26aulast%3DMurayama%26aufirst%3DK.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DD.%2BA.%26aulast%3DKodama%26aufirst%3DE.%2BN.%26aulast%3DYarchoan%26aufirst%3DR.%26aulast%3DWlodawer%26aufirst%3DA.%26aulast%3DMisumi%26aufirst%3DS.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DMitsuya%26aufirst%3DH.%26atitle%3DA%2520Small%2520Molecule%2520Compound%2520with%2520an%2520Indole%2520Moiety%2520Inhibits%2520the%2520Main%2520Protease%2520of%2520SARS-CoV-2%2520and%2520Blocks%2520Virus%2520Replication%26jtitle%3DNat.%2520Commun.%26date%3D2021%26volume%3D12%26spage%3D668%26doi%3D10.1038%2Fs41467-021-20900-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, É.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Désilets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchêne, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chagnon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, É.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of potent, selective inhibitors of matriptase</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">530</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1021/ml3000534</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3000534" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1OntrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=530-534&author=%C3%89.+Colomboauthor=A.+D%C3%A9siletsauthor=D.+Duch%C3%AAneauthor=F.+Chagnonauthor=R.+Najmanovichauthor=R.+Leducauthor=%C3%89.+Marsault&title=Design+and+synthesis+of+potent%2C+selective+inhibitors+of+matriptase&doi=10.1021%2Fml3000534"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Potent, Selective Inhibitors of Matriptase</span></div><div class="casAuthors">Colombo, Eloic; Desilets, Antoine; Duchene, Dominic; Chagnon, Felix; Najmanovich, Rafael; Leduc, Richard; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">530-534</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Matriptase is a member of the type II transmembrane serine protease family.  Several studies have reported deregulated matriptase expression in several types of epithelial cancers, suggesting that matriptase constitutes a potential target for cancer therapy.  We report herein a new series of slow, tight-binding inhibitors of matriptase, which mimic the P1-P4 substrate recognition sequence of the enzyme.  Preliminary structure-activity relationships indicate that this benzothiazole-contg. RQAR-peptidomimetic is a very potent inhibitor and possesses a good selectivity for matriptase vs. other serine proteases.  A mol. model was generated to elucidate the key contacts between inhibitor (I) and matriptase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlqkegeO0GdbVg90H21EOLACvtfcHk0lgJ3VrbYZeoJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1OntrY%253D&md5=560a72862038f32c57c829618fdd28aa</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fml3000534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3000534%26sid%3Dliteratum%253Aachs%26aulast%3DColombo%26aufirst%3D%25C3%2589.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DDuch%25C3%25AAne%26aufirst%3DD.%26aulast%3DChagnon%26aufirst%3DF.%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3D%25C3%2589.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520potent%252C%2520selective%2520inhibitors%2520of%2520matriptase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D530%26epage%3D534%26doi%3D10.1021%2Fml3000534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karmakar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owusu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klampfer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janetka, J. W.</span></span> <span> </span><span class="NLM_article-title">α-Ketobenzothiazole Serine Protease Inhibitors of Aberrant HGF/c-MET and MSP/RON Kinase Pathway Signaling in Cancer</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">599</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fcmdc.201500600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=26889658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC28Xis1Ghtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=585-599&author=Z.+Hanauthor=P.+K.+W.+Harrisauthor=P.+Karmakarauthor=T.+Kimauthor=B.+Y.+Owusuauthor=S.+A.+Wildmanauthor=L.+Klampferauthor=J.+W.+Janetka&title=%CE%B1-Ketobenzothiazole+Serine+Protease+Inhibitors+of+Aberrant+HGF%2Fc-MET+and+MSP%2FRON+Kinase+Pathway+Signaling+in+Cancer&doi=10.1002%2Fcmdc.201500600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketobenzothiazole Serine Protease Inhibitors of Aberrant HGF/c-MET and MSP/RON Kinase Pathway Signaling in Cancer</span></div><div class="casAuthors">Han, Zhenfu; Harris, Peter K. W.; Karmakar, Partha; Kim, Tommy; Owusu, Ben Y.; Wildman, Scott A.; Klampfer, Lidija; Janetka, James W.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">585-599</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Upregulation of the HGF and MSP growth-factor processing serine endopeptidases HGFA, matriptase and hepsin is correlated with increased metastasis in multiple tumor types driven by c-MET or RON kinase signaling.  The authors rationally designed P1' α-ketobenzothiazole mechanism-based inhibitors of these proteases.  Structure-activity studies are presented, which resulted in the identification of potent inhibitors with differential selectivity.  The tetrapeptide inhibitors span the P1-P1' substrate cleavage site via a P1' amide linker off the benzothiazole, occupying the S3' pocket.  Optimized inhibitors display sub-nanomolar enzyme inhibition against one, two, or all three of HGFA, matriptase, and hepsin.  Several compds. also have good selectivity against the related trypsin-like proteases, thrombin and Factor Xa.  Finally, the authors show that inhibitors block the fibroblast (HGF)-mediated migration of invasive DU145 prostate cancer cells.  In addn. to prostate cancer, breast, colon, lung, pancreas, gliomas, and multiple myeloma tumors all depend on HGF and MSP for tumor survival and progression.  Therefore, these unique inhibitors have potential as new therapeutics for a diverse set of tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo67oTbWfCgKrVg90H21EOLACvtfcHk0lgJ3VrbYZeoJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xis1Ghtb8%253D&md5=6fd98f5e020a8de9089995cc15b883eb</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500600%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DHarris%26aufirst%3DP.%2BK.%2BW.%26aulast%3DKarmakar%26aufirst%3DP.%26aulast%3DKim%26aufirst%3DT.%26aulast%3DOwusu%26aufirst%3DB.%2BY.%26aulast%3DWildman%26aufirst%3DS.%2BA.%26aulast%3DKlampfer%26aufirst%3DL.%26aulast%3DJanetka%26aufirst%3DJ.%2BW.%26atitle%3D%25CE%25B1-Ketobenzothiazole%2520Serine%2520Protease%2520Inhibitors%2520of%2520Aberrant%2520HGF%252Fc-MET%2520and%2520MSP%252FRON%2520Kinase%2520Pathway%2520Signaling%2520in%2520Cancer%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D585%26epage%3D599%26doi%3D10.1002%2Fcmdc.201500600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">St-Georges, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Désilets, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Béliveau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghinet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dion, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, É.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najmanovich, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, É.</span></span> <span> </span><span class="NLM_article-title">Modulating the selectivity of matriptase-2 inhibitors with unnatural amino acids</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">110</span>– <span class="NLM_lpage">123</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2Fj.ejmech.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=28219045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVeksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2017&pages=110-123&author=C.+St-Georgesauthor=A.+D%C3%A9siletsauthor=F.+B%C3%A9liveauauthor=M.+Ghinetauthor=S.+P.+Dionauthor=%C3%89.+Colomboauthor=P.+Boudreaultauthor=R.+J.+Najmanovichauthor=R.+Leducauthor=%C3%89.+Marsault&title=Modulating+the+selectivity+of+matriptase-2+inhibitors+with+unnatural+amino+acids&doi=10.1016%2Fj.ejmech.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Modulating the selectivity of matriptase-2 inhibitors with unnatural amino acids</span></div><div class="casAuthors">St-Georges, Catherine; Desilets, Antoine; Beliveau, Francois; Ghinet, Mariana; Dion, Sebastien P.; Colombo, Eloic; Boudreault, Pierre-Luc; Najmanovich, Rafael J.; Leduc, Richard; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110-123</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Matriptase-2, a type II transmembrane serine protease (TTSP), is expressed in the liver and regulates iron homeostasis via the cleavage of hemojuvelin.  Matriptase-2 emerges as an attractive target for the treatment of conditions assocd. with iron overload, such as hemochromatosis or beta-thalassemia.  Starting from the crystal structure of its closest homolog matriptase, we constructed a homol. model of matriptase-2 in order to further optimize the selectivity of serine trap peptidomimetic inhibitors for matriptase-2 vs matriptase.  Careful modifications of the P4, P3 and P2 positions with the help of unnatural amino acids led to a thorough understanding of Structure-Activity Relationship and a >60-fold increase in selectivity for matriptase-2 vs. matriptase.  Addnl., the introduction of unnatural amino acids led to significant increases in plasma stability.  Such compds. represent useful pharmacol. tools to test matriptase-2 inhibition in a context of iron overload.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd_FHXeAyV_rVg90H21EOLACvtfcHk0lgznaiGfQ-1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVeksrc%253D&md5=c93364406e2b1d4ba75ce5736bd28035</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DSt-Georges%26aufirst%3DC.%26aulast%3DD%25C3%25A9silets%26aufirst%3DA.%26aulast%3DB%25C3%25A9liveau%26aufirst%3DF.%26aulast%3DGhinet%26aufirst%3DM.%26aulast%3DDion%26aufirst%3DS.%2BP.%26aulast%3DColombo%26aufirst%3D%25C3%2589.%26aulast%3DBoudreault%26aufirst%3DP.%26aulast%3DNajmanovich%26aufirst%3DR.%2BJ.%26aulast%3DLeduc%26aufirst%3DR.%26aulast%3DMarsault%26aufirst%3D%25C3%2589.%26atitle%3DModulating%2520the%2520selectivity%2520of%2520matriptase-2%2520inhibitors%2520with%2520unnatural%2520amino%2520acids%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D129%26spage%3D110%26epage%3D123%26doi%3D10.1016%2Fj.ejmech.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, L. W.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of ketone-containing, tripeptidyl human rhinovirus 3C protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(99)00587-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2FS0960-894X%2899%2900587-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10636240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjtVyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=45-48&author=P.+S.+Dragovichauthor=R.+Zhouauthor=S.+E.+Webberauthor=T.+J.+Prinsauthor=A.+K.+Kwokauthor=K.+Okanoauthor=S.+A.+Fuhrmanauthor=L.+S.+Zalmanauthor=F.+Z.+Maldonadoauthor=E.+L.+Brownauthor=L.+W.+Meadorauthor=A.+K.+Patickauthor=C.+E.+Fordauthor=M.+A.+Brothersauthor=L.+S.+Binfordauthor=D.+A.+Matthewsauthor=R.+A.+Ferreauthor=S.+T.+Worland&title=Structure-based+design+of+ketone-containing%2C+tripeptidyl+human+rhinovirus+3C+protease+inhibitors&doi=10.1016%2FS0960-894X%2899%2900587-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of ketone-containing, tripeptidyl human rhinovirus 3C protease inhibitors</span></div><div class="casAuthors">Dragovich, Peter S.; Zhou, Ru; Webber, Stephen E.; Prins, Thomas J.; Kwok, Annette K.; Okano, Koji; Fuhrman, Shella A.; Zalman, Leora S.; Maldonado, Fausto C.; Brown, Edward L.; Meador, James W., III; Patick, Amy K.; Ford, Clifford E.; Brothers, Mary A.; Binford, Susan L.; Matthews, David A.; Ferre, Rose Ann; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-48</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Tripeptide-derived mols. incorporating C-terminal ketone electrophiles were evaluated as reversible inhibitors of the cysteine-contg. human rhinovirus 3C protease (3CP).  For example, tripeptide ketone I displayed potent 3CP inhibition activity (Ki = 0.0045 μM) and in vitro antiviral properties (EC50 = 0.34 μM) when tested against HRV serotype-14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxw_91k2flKrVg90H21EOLACvtfcHk0lgznaiGfQ-1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjtVyhtA%253D%253D&md5=60b8bd8d56ccd09ae8e60ce341926271</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2899%2900587-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252899%252900587-9%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DKwok%26aufirst%3DA.%2BK.%26aulast%3DOkano%26aufirst%3DK.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMaldonado%26aufirst%3DF.%2BZ.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DL.%2BW.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DBinford%26aufirst%3DL.%2BS.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-based%2520design%2520of%2520ketone-containing%252C%2520tripeptidyl%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D45%26epage%3D48%26doi%3D10.1016%2FS0960-894X%2899%2900587-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lgznaiGfQ-1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dropinski, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dykstra, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirocco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Streckfuss, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cernak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vachal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krska, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dreher, S. D.</span></span> <span> </span><span class="NLM_article-title">Chemistry Informer Libraries: A Chemoinformatics Enabled Approach to Evaluate and Advance Synthetic Methods</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2604</span>– <span class="NLM_lpage">2613</span>, <span class="refDoi"> DOI: 10.1039/C5SC04751J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1039%2FC5SC04751J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=28660032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2604-2613&author=P.+S.+Kutchukianauthor=J.+F.+Dropinskiauthor=K.+D.+Dykstraauthor=B.+Liauthor=D.+A.+Diroccoauthor=E.+C.+Streckfussauthor=L.+C.+Campeauauthor=T.+Cernakauthor=P.+Vachalauthor=I.+W.+Daviesauthor=S.+W.+Krskaauthor=S.+D.+Dreher&title=Chemistry+Informer+Libraries%3A+A+Chemoinformatics+Enabled+Approach+to+Evaluate+and+Advance+Synthetic+Methods&doi=10.1039%2FC5SC04751J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry informer libraries: a chemoinformatics enabled approach to evaluate and advance synthetic methods</span></div><div class="casAuthors">Kutchukian, Peter S.; Dropinski, James F.; Dykstra, Kevin D.; Li, Bing; DiRocco, Daniel A.; Streckfuss, Eric C.; Campeau, Louis-Charles; Cernak, Tim; Vachal, Petr; Davies, Ian W.; Krska, Shane W.; Dreher, Spencer D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2604-2613</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Major new advances in synthetic chem. methods are typically reported using simple, non-standardized reaction substrates, and reaction failures are rarely documented.  This makes the evaluation and choice of a synthetic method difficult.  We report a standardized complex mol. diagnostic approach using collections of relevant drug-like mols. which we call chem. informer libraries.  With this approach, all chem. results, successes and failures, can be documented to compare and evolve synthetic methods.  To aid in the visualization of chem. results in drug-like physicochem. space we have used an informatics methodol. termed principal component anal.  We have validated this method using palladium- and copper-catalyzed reactions, including Suzuki-Miyaura, cyanation and Buchwald-Hartwig amination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDsNJ0zmf9DrVg90H21EOLACvtfcHk0lhYXsrVxjIY1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSju7o%253D&md5=d7169f9ae6f74e43df0e44d2fa00e5af</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2FC5SC04751J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC04751J%26sid%3Dliteratum%253Aachs%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DDropinski%26aufirst%3DJ.%2BF.%26aulast%3DDykstra%26aufirst%3DK.%2BD.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DDirocco%26aufirst%3DD.%2BA.%26aulast%3DStreckfuss%26aufirst%3DE.%2BC.%26aulast%3DCampeau%26aufirst%3DL.%2BC.%26aulast%3DCernak%26aufirst%3DT.%26aulast%3DVachal%26aufirst%3DP.%26aulast%3DDavies%26aufirst%3DI.%2BW.%26aulast%3DKrska%26aufirst%3DS.%2BW.%26aulast%3DDreher%26aufirst%3DS.%2BD.%26atitle%3DChemistry%2520Informer%2520Libraries%253A%2520A%2520Chemoinformatics%2520Enabled%2520Approach%2520to%2520Evaluate%2520and%2520Advance%2520Synthetic%2520Methods%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D2604%26epage%3D2613%26doi%3D10.1039%2FC5SC04751J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrivastava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassanali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stothard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolsey, J.</span></span> <span> </span><span class="NLM_article-title">DrugBank: A Comprehensive Resource for in Silico Drug Discovery and Exploration</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1093/nar/gkj067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1093%2Fnar%2Fgkj067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=16381955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFOrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=668-672&author=D.+S.+Wishartauthor=C.+Knoxauthor=A.+C.+Guoauthor=S.+Shrivastavaauthor=M.+Hassanaliauthor=P.+Stothardauthor=Z.+Changauthor=J.+Woolsey&title=DrugBank%3A+A+Comprehensive+Resource+for+in+Silico+Drug+Discovery+and+Exploration&doi=10.1093%2Fnar%2Fgkj067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank: a comprehensive resource for in silico drug discovery and exploration</span></div><div class="casAuthors">Wishart, David S.; Knox, Craig; Guo, An Chi; Shrivastava, Savita; Hassanali, Murtaza; Stothard, Paul; Chang, Zhan; Woolsey, Jennifer</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">Database</span>),
    <span class="NLM_cas:pages">D668-D672</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a unique bioinformatics/cheminformatics resource that combines detailed drug (i.e. chem.) data with comprehensive drug target (i.e. protein) information.  The database contains >4100 drug entries including >800 FDA approved small mol. and biotech drugs as well as >3200 exptl. drugs.  Addnl., >14 000 protein or drug target sequences are linked to these drug entries.  Each DrugCard entry contains >80 data fields with half of the information being devoted to drug/chem. data and the other half devoted to drug target or protein data.  Many data fields are hyperlinked to other databases (KEGG, PubChem, ChEBI, PDB, Swiss-Prot and GenBank) and a variety of structure viewing applets.  The database is fully searchable supporting extensive text, sequence, chem. structure and relational query searches.  Potential applications of DrugBank include in silico drug target discovery, drug design, drug docking or screening, drug metab. prediction, drug interaction prediction and general pharmaceutical education.  DrugBank is available at http://redpoll.pharmacy.ualberta.ca/drugbank/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaNy9WiVtlu7Vg90H21EOLACvtfcHk0lhYXsrVxjIY1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFOrsw%253D%253D&md5=c3f4ad2fc489440be5b16f91bc9cf7db</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkj067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkj067%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DShrivastava%26aufirst%3DS.%26aulast%3DHassanali%26aufirst%3DM.%26aulast%3DStothard%26aufirst%3DP.%26aulast%3DChang%26aufirst%3DZ.%26aulast%3DWoolsey%26aufirst%3DJ.%26atitle%3DDrugBank%253A%2520A%2520Comprehensive%2520Resource%2520for%2520in%2520Silico%2520Drug%2520Discovery%2520and%2520Exploration%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D668%26epage%3D672%26doi%3D10.1093%2Fnar%2Fgkj067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boras, B.</span>; <span class="NLM_string-name">Jones, R. M.</span>; <span class="NLM_string-name">Anson, B. J.</span>; <span class="NLM_string-name">Arenson, D.</span>; <span class="NLM_string-name">Aschenbrenner, L.</span>; <span class="NLM_string-name">Bakowski, M. A.</span>; <span class="NLM_string-name">Beutler, N.</span>; <span class="NLM_string-name">Binder, J.</span>; <span class="NLM_string-name">Chen, E.</span>; <span class="NLM_string-name">Eng, H.</span>; <span class="NLM_string-name">Hammond, H.</span>; <span class="NLM_string-name">Hammond, J.</span>; <span class="NLM_string-name">Haupt, R. E.</span>; <span class="NLM_string-name">Hoffman, R.</span>; <span class="NLM_string-name">Kadar, E. P.</span>; <span class="NLM_string-name">Kania, R.</span>; <span class="NLM_string-name">Kimoto, E.</span>; <span class="NLM_string-name">Kirkpatrick, M. G.</span>; <span class="NLM_string-name">Lanyon, L.</span>; <span class="NLM_string-name">Lendy, E. K.</span>; <span class="NLM_string-name">Lillis, J. R.</span>; <span class="NLM_string-name">Logue, J.</span>; <span class="NLM_string-name">Luthra, S. A.</span>; <span class="NLM_string-name">Ma, C.</span>; <span class="NLM_string-name">Mason, S. W.</span>; <span class="NLM_string-name">McGrath, M. E.</span>; <span class="NLM_string-name">Noell, S.</span>; <span class="NLM_string-name">Obach, R. S.</span>; <span class="NLM_string-name">O’Brien, M. N.</span>; <span class="NLM_string-name">O’Connor, R.</span>; <span class="NLM_string-name">Ogilvie, K.</span>; <span class="NLM_string-name">Owen, D.</span>; <span class="NLM_string-name">Pettersson, M.</span>; <span class="NLM_string-name">Reese, M. R.</span>; <span class="NLM_string-name">Rogers, T. F.</span>; <span class="NLM_string-name">Rossulek, M. I.</span>; <span class="NLM_string-name">Sathish, J. G.</span>; <span class="NLM_string-name">Shirai, N.</span>; <span class="NLM_string-name">Steppan, C.</span>; <span class="NLM_string-name">Ticehurst, M.</span>; <span class="NLM_string-name">Updyke, L. W.</span>; <span class="NLM_string-name">Weston, S.</span>; <span class="NLM_string-name">Zhu, Y.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Chatterjee, A. K.</span>; <span class="NLM_string-name">Mesecar, A. D.</span>; <span class="NLM_string-name">Frieman, M. B.</span>; <span class="NLM_string-name">Anderson, A. S.</span>; <span class="NLM_string-name">Allerton, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel Inhibitor of Coronavirus 3CL Protease for the Potential Treatment of COVID-19</span>.  <i>bioRxiv</i>, September 30, <span class="NLM_year" style="font-weight: bold;">2020</span>. <span class="refDoi"> DOI: 10.1101/2020.09.12.293498</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1101%2F2020.09.12.293498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=32935104" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=B.+Boras&author=R.+M.+Jones&author=B.+J.+Anson&author=D.+Arenson&author=L.+Aschenbrenner&author=M.+A.+Bakowski&author=N.+Beutler&author=J.+Binder&author=E.+Chen&author=H.+Eng&author=H.+Hammond&author=J.+Hammond&author=R.+E.+Haupt&author=R.+Hoffman&author=E.+P.+Kadar&author=R.+Kania&author=E.+Kimoto&author=M.+G.+Kirkpatrick&author=L.+Lanyon&author=E.+K.+Lendy&author=J.+R.+Lillis&author=J.+Logue&author=S.+A.+Luthra&author=C.+Ma&author=S.+W.+Mason&author=M.+E.+McGrath&author=S.+Noell&author=R.+S.+Obach&author=M.+N.+O%E2%80%99Brien&author=R.+O%E2%80%99Connor&author=K.+Ogilvie&author=D.+Owen&author=M.+Pettersson&author=M.+R.+Reese&author=T.+F.+Rogers&author=M.+I.+Rossulek&author=J.+G.+Sathish&author=N.+Shirai&author=C.+Steppan&author=M.+Ticehurst&author=L.+W.+Updyke&author=S.+Weston&author=Y.+Zhu&author=J.+Wang&author=A.+K.+Chatterjee&author=A.+D.+Mesecar&author=M.+B.+Frieman&author=A.+S.+Anderson&author=C.+Allerton&title=Discovery+of+a+Novel+Inhibitor+of+Coronavirus+3CL+Protease+for+the+Potential+Treatment+of+COVID-19&doi=10.1101%2F2020.09.12.293498"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1101%2F2020.09.12.293498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F2020.09.12.293498%26sid%3Dliteratum%253Aachs%26aulast%3DBoras%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Inhibitor%2520of%2520Coronavirus%25203CL%2520Protease%2520for%2520the%2520Potential%2520Treatment%2520of%2520COVID-19%26jtitle%3DbioRxiv%26date%3D2020%26doi%3D10.1101%2F2020.09.12.293498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Vries, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prescott, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valero-Jimenez, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desvignes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devlin, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinlever, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggles, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koralov, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, M.</span></span> <span> </span><span class="NLM_article-title">A Comparative Analysis of SARS-CoV-2 Antivirals Characterizes 3CLpro Inhibitor PF-00835231 as a Potential New Treatment for COVID-19</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">e01819</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1128/JVI.01819-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1128%2FJVI.01819-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXht1ensrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2021&pages=e01819-20&author=M.+de%0AVriesauthor=A.+S.+Mohamedauthor=R.+A.+Prescottauthor=A.+M.+Valero-Jimenezauthor=L.+Desvignesauthor=R.+O%E2%80%99Connorauthor=C.+Steppanauthor=J.+C.+Devlinauthor=E.+Ivanovaauthor=A.+Herreraauthor=A.+Schinleverauthor=P.+Looseauthor=K.+Rugglesauthor=S.+B.+Koralovauthor=A.+S.+Andersonauthor=J.+Binderauthor=M.+Dittmann&title=A+Comparative+Analysis+of+SARS-CoV-2+Antivirals+Characterizes+3CLpro+Inhibitor+PF-00835231+as+a+Potential+New+Treatment+for+COVID-19&doi=10.1128%2FJVI.01819-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A comparative analysis of SARS-CoV-2 antivirals characterizes 3CLpro inhibitor PF-00835231 as a potential new treatment for COVID-19</span></div><div class="casAuthors">de Vries, Maren; Mohamed, Adil S.; Prescott, Rachel A.; Valero-Jimenez, Ana M.; Desvignes, Ludovic; O'Connor, Rebecca; Steppan, Claire; Devlin, Joseph C.; Ivanova, Ellie; Herrera, Alberto; Schinlever, Austin; Loose, Paige; Ruggles, Kelly; Koralov, Sergei B.; Anderson, Annaliesa S.; Binder, Joseph; Dittmann, Meike</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e01819</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is the etiol. agent of coronavirus disease 2019 (COVID-19).  There is a dire need for novel effective antivirals to treat COVID-19, as the only approved direct-acting antiviral to date is remdesivir, targeting the viral polymerase complex.  A potential alternate target in the viral life cycle is the main SARS-CoV-2 protease 3CLpro (Mpro).  The drug candidate PF-00835231 is the active compd. of the first anti-3CLpro regimen in clin. trials.  Here, we perform a comparative anal. of PF-00835231, the preclin. 3CLpro inhibitor GC-376, and the polymerase inhibitor remdesivir, in alveolar basal epithelial cells modified to express ACE2 (A549+ACE2 cells).  We find PF-00835231 with at least similar or higher potency than remdesivir or GC-376.  A time-of-drug-addn. approach delineates the timing of early SARS-CoV-2 life cycle steps in A549+ACE2 cells and validates PF-00835231's early time of action.  In a model of the human polarized airway epithelium, both PF-00835231 and remdesivir potently inhibit SARS-CoV-2 at low micromolar concns.  Finally, we show that the efflux transporter P-glycoprotein, which was previously suggested to diminish PF-00835231's efficacy based on expts. in monkey kidney Vero E6 cells, does not neg. impact PF-00835231 efficacy in either A549+ACE2 cells or human polarized airway epithelial cultures.  Thus, our study provides in vitro evidence for the potential of PF-00835231 as an effective SARS-CoV-2 antiviral and addresses concerns that emerged based on prior studies in nonhuman in vitro models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR9dKQsVRU9bVg90H21EOLACvtfcHk0ljZHOjOc--4EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXht1ensrjI&md5=d67cdf0d1ddc1c14f01f55d25fd48bd6</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FJVI.01819-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01819-20%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVries%26aufirst%3DM.%26aulast%3DMohamed%26aufirst%3DA.%2BS.%26aulast%3DPrescott%26aufirst%3DR.%2BA.%26aulast%3DValero-Jimenez%26aufirst%3DA.%2BM.%26aulast%3DDesvignes%26aufirst%3DL.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DR.%26aulast%3DSteppan%26aufirst%3DC.%26aulast%3DDevlin%26aufirst%3DJ.%2BC.%26aulast%3DIvanova%26aufirst%3DE.%26aulast%3DHerrera%26aufirst%3DA.%26aulast%3DSchinlever%26aufirst%3DA.%26aulast%3DLoose%26aufirst%3DP.%26aulast%3DRuggles%26aufirst%3DK.%26aulast%3DKoralov%26aufirst%3DS.%2BB.%26aulast%3DAnderson%26aufirst%3DA.%2BS.%26aulast%3DBinder%26aufirst%3DJ.%26aulast%3DDittmann%26aufirst%3DM.%26atitle%3DA%2520Comparative%2520Analysis%2520of%2520SARS-CoV-2%2520Antivirals%2520Characterizes%25203CLpro%2520Inhibitor%2520PF-00835231%2520as%2520a%2520Potential%2520New%2520Treatment%2520for%2520COVID-19%26jtitle%3DJ.%2520Virol.%26date%3D2021%26volume%3D95%26spage%3De01819%26epage%3D20%26doi%3D10.1128%2FJVI.01819-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, T.</span></span> <span> </span><span class="NLM_article-title">Physiological Parameters in Laboratory Animals and Humans</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1093</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1023/A:1018943613122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1023%2FA%3A1018943613122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=8378254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADyaK3szptFaqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1993&pages=1093-1095&author=B.+Daviesauthor=T.+Morris&title=Physiological+Parameters+in+Laboratory+Animals+and+Humans&doi=10.1023%2FA%3A1018943613122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Physiological parameters in laboratory animals and humans</span></div><div class="casAuthors">Davies B; Morris T</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical research</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1093-5</span>
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDDwka3A6hEYYPsCkJIw00fW6udTcc2eaXVf2Xpi-D-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3szptFaqsg%253D%253D&md5=094cdd17306b58cc17c8cf4fd5db9927</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1023%2FA%3A1018943613122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1018943613122%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DB.%26aulast%3DMorris%26aufirst%3DT.%26atitle%3DPhysiological%2520Parameters%2520in%2520Laboratory%2520Animals%2520and%2520Humans%26jtitle%3DPharm.%2520Res.%26date%3D1993%26volume%3D10%26spage%3D1093%26epage%3D1095%26doi%3D10.1023%2FA%3A1018943613122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=J.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+Ketone-Based+Covalent+Inhibitors+of+Coronavirus+3CL+Proteases+for+the+Potential+Therapeutic+Treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0ljZHOjOc--4EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Ketone-Based%2520Covalent%2520Inhibitors%2520of%2520Coronavirus%25203CL%2520Proteases%2520for%2520the%2520Potential%2520Therapeutic%2520Treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the Inhibition Constant (KI) and the Concentration of Inhibitor Which Causes 50 per Cent Inhibition (I50) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+Inhibition+Constant+%28KI%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520Inhibition%2520Constant%2520%2528KI%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520Which%2520Causes%252050%2520per%2520Cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiraki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senda, T.</span></span> <span> </span><span class="NLM_article-title">A Fully Automated Crystallization Apparatus for Small Protein Quantities</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">36</span>, <span class="refDoi"> DOI: 10.1107/S2053230X20015514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS2053230X20015514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=33439153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtFSgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2021&pages=29-36&author=R.+Katoauthor=M.+Hirakiauthor=Y.+Yamadaauthor=M.+Tanabeauthor=T.+Senda&title=A+Fully+Automated+Crystallization+Apparatus+for+Small+Protein+Quantities&doi=10.1107%2FS2053230X20015514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A fully automated crystallization apparatus for small protein quantities</span></div><div class="casAuthors">Kato, Ryuichi; Hiraki, Masahiko; Yamada, Yusuke; Tanabe, Mikio; Senda, Toshiya</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology Communications</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-36</span>CODEN:
                <span class="NLM_cas:coden">ACSFEN</span>;
        ISSN:<span class="NLM_cas:issn">2053-230X</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">In 2003, a fully automated protein crystn. and monitoring system (PXS) was developed to support the structural genomics projects that were initiated in the early 2000s.  In PXS, crystn. plates were automatically set up using the vapor-diffusion method, transferred to incubators and automatically obsd. according to a pre-set schedule.  The captured images of each crystn. drop could be monitored through the internet using a web browser.  While the screening throughput of PXS was very high, the demands of users have gradually changed over the ensuing years.  To study difficult proteins, it has become important to screen crystn. conditions using small amts. of proteins.  Moreover, membrane proteins have become one of the main targets for X-ray crystallog.  Therefore, to meet the evolving demands of users, PXS was upgraded to PXS2.  In PXS2, the min. vol. of the dispenser is reduced to 0.1μl to minimize the amt. of sample, and the resoln. of the captured images is increased to five million pixels in order to observe small crystn. drops in detail.  In addn. to the 20°C incubators, a 4°C incubator was installed in PXS2 because crystn. results may vary with temp.  To support membrane-protein crystn., PXS2 includes a procedure for the bicelle method.  In addn., the system supports a lipidic cubic phase (LCP) method that uses a film sandwich plate and that was specifically designed for PXS2.  These improvements expand the applicability of PXS2, reducing the bottleneck of X-ray protein crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt0SjXkr_IqrVg90H21EOLACvtfcHk0li794Rzcqaaug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtFSgtr8%253D&md5=99a0aed5d2b74bfd00d54b2841d99eb9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS2053230X20015514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053230X20015514%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DR.%26aulast%3DHiraki%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DY.%26aulast%3DTanabe%26aufirst%3DM.%26aulast%3DSenda%26aufirst%3DT.%26atitle%3DA%2520Fully%2520Automated%2520Crystallization%2520Apparatus%2520for%2520Small%2520Protein%2520Quantities%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Commun.%26date%3D2021%26volume%3D77%26spage%3D29%26epage%3D36%26doi%3D10.1107%2FS2053230X20015514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0li794Rzcqaaug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li794Rzcqaaug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.Refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.Refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lg2XleSYDhumw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.Refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lg2XleSYDhumw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanwell, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonie, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandermeersch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zurek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchison, G. R.</span></span> <span> </span><span class="NLM_article-title">Avogadro: An Advanced Semantic Chemical Editor, Visualization, and Analysis Platform</span>. <i>J. Cheminform.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-4-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1186%2F1758-2946-4-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=22889332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BC38fns1yntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=17&author=M.+D.+Hanwellauthor=D.+E.+Curtisauthor=D.+C.+Lonieauthor=T.+Vandermeerschauthor=E.+Zurekauthor=G.+R.+Hutchison&title=Avogadro%3A+An+Advanced+Semantic+Chemical+Editor%2C+Visualization%2C+and+Analysis+Platform&doi=10.1186%2F1758-2946-4-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Avogadro: an advanced semantic chemical editor, visualization, and analysis platform</span></div><div class="casAuthors">Hanwell Marcus D; Curtis Donald E; Lonie David C; Vandermeersch Tim; Zurek Eva; Hutchison Geoffrey R</div><div class="citationInfo"><span class="NLM_cas:title">Journal of cheminformatics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">UNLABELLED:  BACKGROUND:  The Avogadro project has developed an advanced molecule editor and visualizer designed for cross-platform use in computational chemistry, molecular modeling, bioinformatics, materials science, and related areas.  It offers flexible, high quality rendering, and a powerful plugin architecture.  Typical uses include building molecular structures, formatting input files, and analyzing output of a wide variety of computational chemistry packages.  By using the CML file format as its native document type, Avogadro seeks to enhance the semantic accessibility of chemical data types.  RESULTS:  The work presented here details the Avogadro library, which is a framework providing a code library and application programming interface (API) with three-dimensional visualization capabilities; and has direct applications to research and education in the fields of chemistry, physics, materials science, and biology.  The Avogadro application provides a rich graphical interface using dynamically loaded plugins through the library itself.  The application and library can each be extended by implementing a plugin module in C++ or Python to explore different visualization techniques, build/manipulate molecular structures, and interact with other programs.  We describe some example extensions, one which uses a genetic algorithm to find stable crystal structures, and one which interfaces with the PackMol program to create packed, solvated structures for molecular dynamics simulations.  The 1.0 release series of Avogadro is the main focus of the results discussed here.  CONCLUSIONS:  Avogadro offers a semantic chemical builder and platform for visualization and analysis.  For users, it offers an easy-to-use builder, integrated support for downloading from common databases such as PubChem and the Protein Data Bank, extracting chemical data from a wide variety of formats, including computational chemistry output, and native, semantic support for the CML file format.  For developers, it can be easily extended via a powerful plugin mechanism to support new features in organic chemistry, inorganic complexes, drug design, materials, biomolecules, and simulations.  Avogadro is freely available under an open-source license from        http://avogadro.openmolecules.net.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTitVKXb07oIcuCWE5mhx1RfW6udTcc2ebDfFDCUPQaJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fns1yntQ%253D%253D&md5=91fe7fe658064a132273004aa5ee0776</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-4-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-4-17%26sid%3Dliteratum%253Aachs%26aulast%3DHanwell%26aufirst%3DM.%2BD.%26aulast%3DCurtis%26aufirst%3DD.%2BE.%26aulast%3DLonie%26aufirst%3DD.%2BC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DZurek%26aufirst%3DE.%26aulast%3DHutchison%26aufirst%3DG.%2BR.%26atitle%3DAvogadro%253A%2520An%2520Advanced%2520Semantic%2520Chemical%2520Editor%252C%2520Visualization%252C%2520and%2520Analysis%2520Platform%26jtitle%3DJ.%2520Cheminform.%26date%3D2012%26volume%3D4%26spage%3D17%26doi%3D10.1186%2F1758-2946-4-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakanishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamai, I.</span></span> <span> </span><span class="NLM_article-title">Differential Effect of Grapefruit Juice on Intestinal Absorption of Statins Due to Inhibition of Organic Anion Transporting Polypeptide and/or P-Glycoprotein</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">3843</span>– <span class="NLM_lpage">3853</span>, <span class="refDoi"> DOI: 10.1002/jps.22586</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fjps.22586" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=21520088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC3MXosFemt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2011&pages=3843-3853&author=Y.+Shirasakaauthor=K.+Suzukiauthor=T.+Nakanishiauthor=I.+Tamai&title=Differential+Effect+of+Grapefruit+Juice+on+Intestinal+Absorption+of+Statins+Due+to+Inhibition+of+Organic+Anion+Transporting+Polypeptide+and%2For+P-Glycoprotein&doi=10.1002%2Fjps.22586"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Differential effect of grapefruit juice on intestinal absorption of statins due to inhibition of organic anion transporting polypeptide and/or P-glycoprotein</span></div><div class="casAuthors">Shirasaka, Yoshiyuki; Suzuki, Kensuke; Nakanishi, Takeo; Tamai, Ikumi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3843-3853</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The purpose of this study is to examine the contributions of org. anion transporting polypeptide (Oatp) and/or P-glycoprotein (P-gp) to grapefruit juice (GFJ) interaction with two statins, pravastatin and pitavastatin, which undergo negligible metab. in rats.  The two statins were found to be substrates of both Oatp1a5 and Oatp2b1, whereas pitavastatin, but not pravastatin, was a substrate of P-gp.  The plasma concn. of pravastatin after oral administration was significantly decreased by GFJ and naringin, whereas that of pitavastatin was significantly increased.  Naringin inhibited Oatp1a5- and Oatp2b1-mediated uptake of pravastatin and Oatp1a5-mediated, but not Oatp2b1-mediated, uptake of pitavastatin.  Naringin also inhibited P-gp-mediated transport of pitavastatin.  These results suggested that the decrease of pravastatin absorption in the presence of GFJ is due to the inhibitory effect of naringin on Oatp, whereas the increase of pitavastatin is due to the inhibition of P-gp.  These observations are consistent with the results of in situ absorption studies.  In conclusion, Oatp and/or P-gp contribute to the intestinal absorption of statins, and the differential effect of GFJ on pravastatin and pitavastatin absorption is at least partly accounted for by the different inhibitory effects of naringin on these transporters. © 2011 Wiley-Liss, Inc. and the American Pharmacists Assocn. J Pharm Sci 100:3843-3853, 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNKsxXgBVv6LVg90H21EOLACvtfcHk0lg2XleSYDhumw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXosFemt74%253D&md5=5190b9c2020a6768aea2e376d2d0a989</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fjps.22586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.22586%26sid%3Dliteratum%253Aachs%26aulast%3DShirasaka%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DNakanishi%26aufirst%3DT.%26aulast%3DTamai%26aufirst%3DI.%26atitle%3DDifferential%2520Effect%2520of%2520Grapefruit%2520Juice%2520on%2520Intestinal%2520Absorption%2520of%2520Statins%2520Due%2520to%2520Inhibition%2520of%2520Organic%2520Anion%2520Transporting%2520Polypeptide%2520and%252For%2520P-Glycoprotein%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2011%26volume%3D100%26spage%3D3843%26epage%3D3853%26doi%3D10.1002%2Fjps.22586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Funai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishihara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichijo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, K.</span></span> <span> </span><span class="NLM_article-title">Effect of Osmolality on the Pharmacokinetic Interaction between Apple Juice and Atenolol in Rats</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">386</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1124/dmd.118.084483</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1124%2Fdmd.118.084483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=30622163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFOisr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=386-391&author=Y.+Funaiauthor=Y.+Shirasakaauthor=M.+Ishiharaauthor=M.+Takemuraauthor=K.+Ichijoauthor=H.+Kishimotoauthor=K.+Inoue&title=Effect+of+Osmolality+on+the+Pharmacokinetic+Interaction+between+Apple+Juice+and+Atenolol+in+Rats&doi=10.1124%2Fdmd.118.084483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of osmolality on the pharmacokinetic interaction between apple juice and atenolol in rats</span></div><div class="casAuthors">Funai, Yuta; Shirasaka, Yoshiyuki; Ishihara, Marika; Takemura, Miyuki; Ichijo, Kazuki; Kishimoto, Hisanao; Inoue, Katsuhisa</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">386-391</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A recent clin. study reported that the ingestion of apple juice (AJ) markedly reduced the plasma concn. of atenolol; however, our in vitro study showed that atenolol may not be a substrate of org. anion transporting polypeptide 2B1 (OATP2B1), so this AJ-atenolol interaction cannot be explained by inhibition of OATP2B1.  On the other hand, we more recently showed that the soln. osmolality influences gastrointestinal (GI) water vol., and this may indirectly affect intestinal drug absorption.  In this study, we examd. whether the osmolality dependence of water dynamics can account for AJ-atenolol interactions by evaluating the GI water vol. and the atenolol aborption in the presence of AJ in rats.  Water absorption was highest in purified water, followed by saline and isosmotic mannitol soln., and the lowest in AJ, confirming that water absorption is indeed osmolality-dependent.  Interestingly, AJ showed apparent water secretion into the intestinal lumen.  Further, the fraction of atenolol absorbed in intestine was significantly lower in AJ or hyperosmotic mannitol soln. (adjusted to the osmolality of AJ) than after administration in purified water.  Comparable results were obsd. in an in vivo pharmacokinetic study in rats.  Our results indicate that orally administered AJ has a capacity to modulate luminal water vol. depending on the osmolality, and this effect may result in significant AJ-atenolol interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTYwR4fC653LVg90H21EOLACvtfcHk0lj7vcuLwnuj5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFOisr4%253D&md5=3666bb01ddde21fe98acf9cf0cbd4591</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1124%2Fdmd.118.084483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.118.084483%26sid%3Dliteratum%253Aachs%26aulast%3DFunai%26aufirst%3DY.%26aulast%3DShirasaka%26aufirst%3DY.%26aulast%3DIshihara%26aufirst%3DM.%26aulast%3DTakemura%26aufirst%3DM.%26aulast%3DIchijo%26aufirst%3DK.%26aulast%3DKishimoto%26aufirst%3DH.%26aulast%3DInoue%26aufirst%3DK.%26atitle%3DEffect%2520of%2520Osmolality%2520on%2520the%2520Pharmacokinetic%2520Interaction%2520between%2520Apple%2520Juice%2520and%2520Atenolol%2520in%2520Rats%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2019%26volume%3D47%26spage%3D386%26epage%3D391%26doi%3D10.1124%2Fdmd.118.084483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uno, T.</span></span> <span> </span><span class="NLM_article-title">Statistical Moments in Pharmacokinetics</span>. <i>J. Pharmacokinet. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1007/BF01062109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1007%2FBF01062109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=731417" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADyaE1M%252FptVKqtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1978&pages=547-558&author=K.+Yamaokaauthor=T.+Nakagawaauthor=T.+Uno&title=Statistical+Moments+in+Pharmacokinetics&doi=10.1007%2FBF01062109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical moments in pharmacokinetics</span></div><div class="casAuthors">Yamaoka K; Nakagawa T; Uno T</div><div class="citationInfo"><span class="NLM_cas:title">Journal of pharmacokinetics and biopharmaceutics</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">547-58</span>
        ISSN:<span class="NLM_cas:issn">0090-466X</span>.
    </div><div class="casAbstract">Statistical moments are parameters that describe the characteristics of the time courses of plasma concentration (area, mean residence time, and variance of residence time) and of the urinary excretion rate that follow administration of a single dose of a drug.  The relationship between the moments of a time-course curve and pharmacokinetic profiles of drug disposition, i.e., absorption, distribution, metabolism, and excretion, is described.  The moments are related to the extent and rate of bioavailability, and it is shown that they can be effectively applied to the deconvolution operation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9G3T-afn-o_csfMJQC1ZdfW6udTcc2eYFIr8C-25AALntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE1M%252FptVKqtQ%253D%253D&md5=22a76390b1ea71d8f306d598dc01b040</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2FBF01062109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01062109%26sid%3Dliteratum%253Aachs%26aulast%3DYamaoka%26aufirst%3DK.%26aulast%3DNakagawa%26aufirst%3DT.%26aulast%3DUno%26aufirst%3DT.%26atitle%3DStatistical%2520Moments%2520in%2520Pharmacokinetics%26jtitle%3DJ.%2520Pharmacokinet.%2520Biopharm.%26date%3D1978%26volume%3D6%26spage%3D547%26epage%3D558%26doi%3D10.1007%2FBF01062109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hisaka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, Y.</span></span> <span> </span><span class="NLM_article-title">Analysis of Nonlinear and Nonsteady State Hepatic Extraction with the Dispersion Model Using the Finite Difference Method</span>. <i>J. Pharmacokinet. Pharmacodyn.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1023/A:1023294632129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1023%2FA%3A1023294632129" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1998&pages=495-519&author=A.+Hisakaauthor=Y.+Sugiyama&title=Analysis+of+Nonlinear+and+Nonsteady+State+Hepatic+Extraction+with+the+Dispersion+Model+Using+the+Finite+Difference+Method&doi=10.1023%2FA%3A1023294632129"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023294632129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023294632129%26sid%3Dliteratum%253Aachs%26aulast%3DHisaka%26aufirst%3DA.%26aulast%3DSugiyama%26aufirst%3DY.%26atitle%3DAnalysis%2520of%2520Nonlinear%2520and%2520Nonsteady%2520State%2520Hepatic%2520Extraction%2520with%2520the%2520Dispersion%2520Model%2520Using%2520the%2520Finite%2520Difference%2520Method%26jtitle%3DJ.%2520Pharmacokinet.%2520Pharmacodyn.%26date%3D1998%26volume%3D26%26spage%3D495%26epage%3D519%26doi%3D10.1023%2FA%3A1023294632129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakane, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, S.</span></span> <span> </span><span class="NLM_article-title">Effect of P-Glycoprotein Expression Levels on the Concentration-Dependent Permeability of Drugs to the Cell Membrane</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">565</span>, <span class="refDoi"> DOI: 10.1002/jps.21114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=10.1002%2Fjps.21114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=17828734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;key=1%3ACAS%3A280%3ADC%252BD2sjhs1emsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2008&pages=553-565&author=Y.+Shirasakaauthor=T.+Sakaneauthor=S.+Yamashita&title=Effect+of+P-Glycoprotein+Expression+Levels+on+the+Concentration-Dependent+Permeability+of+Drugs+to+the+Cell+Membrane&doi=10.1002%2Fjps.21114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of P-glycoprotein expression levels on the concentration-dependent permeability of drugs to the cell membrane</span></div><div class="casAuthors">Shirasaka Yoshiyuki; Sakane Toshiyasu; Yamashita Shinji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of pharmaceutical sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">553-65</span>
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    </div><div class="casAbstract">The purpose of this study is to develop a kinetic model that can predict the in vivo absorption of P-glycoprotein (P-gp) substrates from in vitro data.  Apical (AP) to basal (BL) absorptive permeability of typical P-gp substrate drugs including quinidine, verapamil, vinblastine, and digoxin, were measured in several cell monolayers with different levels of P-gp expression, normal, P-gp induced, P-gp highly induced and MDR1-knockdown Caco-2 cells and MDR1-MDCKII cells.  In all cell monolayers, AP to BL permeability of P-gp substrates increased when their AP concentration was increased, showing a sigmoid-type relationship to donor (AP) concentrations.  At the higher concentration range, permeability reached a maximum value, suggesting saturation of P-gp-mediated efflux, and at the lower concentration range, permeability decreased depending on P-gp expression level.  A simple kinetic model was applied to the permeability-concentration curve of each drug to obtain the fundamental parameters for P-gp-mediated transport, K(m(app)) and V(max).  Both K(m(app)) and V(max) of each drug were found to show linear correlations with expression level of P-gp.  This study clearly demonstrated the possibility to estimate the permeability of P-gp substrate drugs in human intestine from the expression level of P-gp, and thus the possibility to predict oral absorption of those drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpkUtoBnpEpz0O18WR3iAofW6udTcc2eYFIr8C-25AALntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2sjhs1emsA%253D%253D&md5=e208a5a449a6456bac9027791df93bd4</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fjps.21114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21114%26sid%3Dliteratum%253Aachs%26aulast%3DShirasaka%26aufirst%3DY.%26aulast%3DSakane%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DS.%26atitle%3DEffect%2520of%2520P-Glycoprotein%2520Expression%2520Levels%2520on%2520the%2520Concentration-Dependent%2520Permeability%2520of%2520Drugs%2520to%2520the%2520Cell%2520Membrane%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2008%26volume%3D97%26spage%3D553%26epage%3D565%26doi%3D10.1002%2Fjps.21114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M2Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6M2Q','PDB','6M2Q'); return false;">PDB: 6M2Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E18" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E18','PDB','7E18'); return false;">PDB: 7E18</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Y2F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6Y2F','PDB','6Y2F'); return false;">PDB: 6Y2F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E19" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7E19','PDB','7E19'); return false;">PDB: 7E19</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i39"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00665">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_16342"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00665?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00665</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Kinetics, NMR titration, and structure of 3CL<sup>pro</sup>; cytotoxicity and antiviral activity of tested compounds with a MDR-1 inhibitor; computational, in vitro metabolic and in vivo pharmacokinetic data of YH-53 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula string table (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_001.pdf">jm1c00665_si_001.pdf (1.62 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00665/suppl_file/jm1c00665_si_002.csv">jm1c00665_si_002.csv (0.38 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00665&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00665%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00665" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac6e9cbbfd1af","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
