
<html lang="en"     class="pb-page"  data-request-id="c0f5ba08-0741-4cac-8310-53146f857781"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.1c00879;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2021.64.issue-14"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="PEGylation of the Antimicrobial Peptide PG-1: A Link between Propensity for Nanostructuring and Capacity of the Antitrypsin Hydrolytic Ability" /></meta><meta name="dc.Creator" content="Weikang  Yu" /></meta><meta name="dc.Creator" content="Jiajun  Wang" /></meta><meta name="dc.Creator" content="Zihang  Wang" /></meta><meta name="dc.Creator" content="Lingxue  Li" /></meta><meta name="dc.Creator" content="Wenyu  Li" /></meta><meta name="dc.Creator" content="Jing  Song" /></meta><meta name="dc.Creator" content="Shanshan  Zhang" /></meta><meta name="dc.Creator" content="Anshan  Shan" /></meta><meta name="dc.Description" content="The increasing prevalence of antibacterial resistance globally underscores the urgent need for updated antimicrobial peptides (AMPs). Here, we describe a strategy for inducing the self-assembly of ..." /></meta><meta name="Description" content="The increasing prevalence of antibacterial resistance globally underscores the urgent need for updated antimicrobial peptides (AMPs). Here, we describe a strategy for inducing the self-assembly of ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 1, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00879" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00879" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00879" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00879" /></link>
        
    
    

<title>PEGylation of the Antimicrobial Peptide PG-1: A Link between Propensity for Nanostructuring and Capacity of the Antitrypsin Hydrolytic Ability | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00879" /></meta><meta property="og:title" content="PEGylation of the Antimicrobial Peptide PG-1: A Link between Propensity for Nanostructuring and Capacity of the Antitrypsin Hydrolytic Ability" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0011.jpeg" /></meta><meta property="og:description" content="The increasing prevalence of antibacterial resistance globally underscores the urgent need for updated antimicrobial peptides (AMPs). Here, we describe a strategy for inducing the self-assembly of protegrin-1 (PG-1) into nanostructured antimicrobial agents with significantly improved pharmacological properties. Our strategy involves PEGylation in the terminals of PG-1 and subsequent self-assembly in aqueous media in the absence of exogenous excipients. Compared with the parent PG-1, the therapeutic index (TI) of NPG750(TIGram-negative bacteria = 17.07) and CPG2000(TIAll = 26.02) was increased. Importantly, NPG750 and CPG2000 offered higher stability toward trypsin degradation. Mechanistically, NPG750 and CPG2000 exerted their bactericidal activity by membrane-active mechanisms due to which microbes were not prone to develop resistance. Our findings proved PEGylation as a simple yet versatile strategy for generating AMP-derived bioactive drugs with excellent antitrypsin hydrolytic ability and lower cytotoxicity. This provides a theoretical basis for the further clinical application of AMPs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00879"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00879">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00879&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00879&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00879&amp;href=/doi/10.1021/acs.jmedchem.1c00879" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 10469-10481</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00863" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00883" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">PEGylation of the Antimicrobial Peptide PG-1: A Link between Propensity for Nanostructuring and Capacity of the Antitrypsin Hydrolytic Ability</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Weikang Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weikang Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weikang++Yu">Weikang Yu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-9181-4226" title="Orcid link">https://orcid.org/0000-0001-9181-4226</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiajun Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiajun Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiajun++Wang">Jiajun Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2086-2105" title="Orcid link">https://orcid.org/0000-0002-2086-2105</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zihang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zihang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zihang++Wang">Zihang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lingxue Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lingxue Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lingxue++Li">Lingxue Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenyu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenyu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenyu++Li">Wenyu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Song</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Song</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Song">Jing Song</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shanshan Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shanshan Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shanshan++Zhang">Shanshan Zhang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Anshan Shan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anshan Shan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang District, Harbin, Heilongjiang 150030, P. R. China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#97f6e4e4fff6f9d7f9f2f6e2b9f2f3e2b9f4f9"><span class="__cf_email__" data-cfemail="28495b5b40494668464d495d064d4c5d064b46">[email protected]</span></a>. Tel: +86 451 55190685. Fax: +86 451 55103336.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anshan++Shan">Anshan Shan</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00879&amp;href=/doi/10.1021%2Facs.jmedchem.1c00879" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 10469–10481</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 1, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 May 2021</li><li><span class="item_label"><b>Published</b> online</span>1 July 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00879" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00879</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10469%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DWeikang%2BYu%252C%2BJiajun%2BWang%252C%2BZihang%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D14%26contentID%3Dacs.jmedchem.1c00879%26title%3DPEGylation%2Bof%2Bthe%2BAntimicrobial%2BPeptide%2BPG-1%253A%2BA%2BLink%2Bbetween%2BPropensity%2Bfor%2BNanostructuring%2Band%2BCapacity%2Bof%2Bthe%2BAntitrypsin%2BHydrolytic%2BAbility%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10481%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00879"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">195</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00879" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;PEGylation of the Antimicrobial Peptide PG-1: A Link between Propensity for Nanostructuring and Capacity of the Antitrypsin Hydrolytic Ability&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Weikang&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Jiajun&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Zihang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Lingxue&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Wenyu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Shanshan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Anshan&quot;,&quot;last_name&quot;:&quot;Shan&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;10469-10481&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00879&quot;},&quot;abstract&quot;:&quot;The increasing prevalence of antibacterial resistance globally underscores the urgent need for updated antimicrobial peptides (AMPs). Here, we describe a strategy for inducing the self-assembly of protegrin-1 (PG-1) into nanostructured antimicrobial agents with significantly improved pharmacological properties. Our strategy involves PEGylation in the terminals of PG-1 and subsequent self-assembly in aqueous media in the absence of exogenous excipients. Compared with the parent PG-1, the therapeutic index (TI) of NPG750(TIGram-negative bacteria = 17.07) and CPG2000(TIAll = 26.02) was increased. Importantly, NPG750 and CPG2000 offered higher stability toward trypsin degradation. Mechanistically, NPG750 and CPG2000 exerted their bactericidal activity by membrane-active mechanisms due to which microbes were not prone to develop resistance. Our findings proved PEGylation as a simple yet versatile strategy for generating AMP-derived bioactive drugs with excellent antitrypsin hydrolytic ability and lower cytotoxicit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00879&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00879" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00879&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00879" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00879&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00879" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00879&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00879&amp;href=/doi/10.1021/acs.jmedchem.1c00879" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00879" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00879" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00879%26sid%3Dliteratum%253Aachs%26pmid%3D34196552%26genre%3Darticle%26aulast%3DYu%26date%3D2021%26atitle%3DPEGylation%2Bof%2Bthe%2BAntimicrobial%2BPeptide%2BPG-1%253A%2BA%2BLink%2Bbetween%2BPropensity%2Bfor%2BNanostructuring%2Band%2BCapacity%2Bof%2Bthe%2BAntitrypsin%2BHydrolytic%2BAbility%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D14%26spage%3D10469%26epage%3D10481%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292224" title="Serum">Serum</a>,</li><li><a href="/action/doSearch?ConceptID=291650" title="Alcohols">Alcohols</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291214" title="Bacteria">Bacteria</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The increasing prevalence of antibacterial resistance globally underscores the urgent need for updated antimicrobial peptides (AMPs). Here, we describe a strategy for inducing the self-assembly of protegrin-1 (PG-1) into nanostructured antimicrobial agents with significantly improved pharmacological properties. Our strategy involves PEGylation in the terminals of PG-1 and subsequent self-assembly in aqueous media in the absence of exogenous excipients. Compared with the parent PG-1, the therapeutic index (TI) of NPG<sub>750</sub>(TI<sub>Gram-negative bacteria</sub> = 17.07) and CPG<sub>2000</sub>(TI<sub>All</sub> = 26.02) was increased. Importantly, NPG<sub>750</sub> and CPG<sub>2000</sub> offered higher stability toward trypsin degradation. Mechanistically, NPG<sub>750</sub> and CPG<sub>2000</sub> exerted their bactericidal activity by membrane-active mechanisms due to which microbes were not prone to develop resistance. Our findings proved PEGylation as a simple yet versatile strategy for generating AMP-derived bioactive drugs with excellent antitrypsin hydrolytic ability and lower cytotoxicity. This provides a theoretical basis for the further clinical application of AMPs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As the first defensive line for an organism’s immune system against the threat of antibiotic-resistant strains,<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> antimicrobial peptides (AMPs) have a broad antibacterial spectrum, fast bactericidal rate, and a lower likelihood of resistance development, which make them a favorable alternative to antibiotics. Protegrin-1 (PG-1) is a pig-derived AMP with four cysteine residues stabilized by two disulfide bonds (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). These two disulfide bonds would promote PG-1 to form an antiparallel β-sheet and U-shaped rigid rotation domain (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">Figure S1</a>), which endow PG-1 with the ability to create pores in the membrane.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Recently, the development and utilization of PG-1 as antimicrobials have made progress, but there are still some shortcomings, such as weak cell selectivity,<a onclick="showRef(event, 'ref4 ref8'); return false;" href="javascript:void(0);" class="ref ref4 ref8">(4,8)</a> inactivation in serum, and protease degradation.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> In order to solve these problems, in previous studies, the derivatives of PG-1 were synthesized by preserving the corner region of PG-1 and substituting low polar amino acids,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> or hybridizing the fragments of Cecropin A with low toxicity and the PG-1 corner region, to reduce the cytotoxicity and improve the antibacterial activity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> However, the problem of protease degradation has not been solved.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Molecular structural formula of PEG and its derivatives: (A) PEG, (B) mPEG-NH<sub>2</sub>, and (C) mPEG-COOH; design of the engineered peptides: (D) sequence and schematic structure of PG-1; (E) NPG<i><sub>n</sub></i>: X = mPEG<sub>750</sub>-COOH, mPEG<sub>1000</sub>-COOH, and mPEG<sub>2000</sub>-COOH, <i>n</i> = 750 Da, 1000 Da, and 2000 Da; and (F) CPG<i><sub>n</sub></i>: Y = mPEG<sub>750</sub>-NH<sub>2</sub>, mPEG<sub>1000</sub>-NH<sub>2</sub>, and mPEG<sub>2000</sub>-NH<sub>2</sub>, <i>n</i> = 750 Da, 1000 Da, and 2000 Da.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Self-assembly is considered an efficient strategy to improve the proteolytic resistance and cell selectivity of peptides.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Some polymeric matrix materials, such as aliphatic acids, palmitoyl groups, and cholesterol, have been introduced into peptides to facilitate the formation of diverse polyplex nanostructures.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19">(16−19)</a> In these peptide-conjugated polymers, the core peptides would be hidden inside the nanostructures, thus reducing degradation by proteolytic enzymes and recognition by the immune system.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">The covalent modification of poly(ethylene glycol) (PEG), which consisted of repeated units of ethylene glycol (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), is a successful self-assembly strategy for peptides and proteins.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> For example, chitosan and peptide conjugated with PEG could self-assemble into a neutrally charged nanosphere structure with a size of ∼100 nm in aqueous solution, which is advantageous to combat <i>Pseudomonas aeruginosa (P. aeruginosa)</i> biofilm formation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Hong et al. conjugated the N-terminal of bacitracin A with PEGylated PLGA copolymers to construct PEGylated self-assembled Nano-BAs for the treatment of invasive infections.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> However, in these previous studies, PEG played more of an adjuvant role for the self-aggregation of nanopeptides; few studies used PEG as an independent carrier to study its self-assembly driving force.</div><div class="NLM_p">Herein, we utilized PEG alone as a carrier to explore the ability of PEG to promote the self-assembly formation of PG-1 to resist protease hydrolysis. The length and attachment site of the PEG moiety play an important role in determining some properties of peptide drugs and maximizing the retention of biological functions in peptide drugs.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The low-molecular-mass polyethylene glycol monomethyl ether (mPEG) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,C) was coupled to the N-terminal of PG-1, named as NPG<i><sub>n</sub></i>. Moreover, Manhart et al. found that the N- and C-terminal modifications have different effects on the activation of the gastric inhibitory polypeptide (GIP) receptor, suggesting that an appropriate relative orientation of N- and C-terminal parts was essential for the bioactivity of peptides.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Thus, the corresponding derivatives of PG-1 with C-terminal PEGylation, named as CPG<i><sub>n</sub></i>, were also designed (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E,F).</div><div class="NLM_p last">The antibacterial activities and cytotoxicities were first determined to assess the biological function of PEGylated peptides. Subsequently, the nanocharacterization using innovative PEGylated peptides was investigated. The behavior of peptides with trypsin was further analyzed to determine the stability of peptides and to elucidate the underlying mechanism of PEGylated peptides against Gram-negative bacteria. This study modified the end of PG-1 with PEG as a “booster” to construct self-assembled peptides to achieve the purpose of improving its proteolytic resistance and reducing its cytotoxicity while maintaining maximum biological activity, which may support further clinical applications of peptide-based supramolecular nanoparticles in the treatment of drug-resistant infections.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Construction and Secondary Structure Analysis of the PEGylated Peptides</h3><div class="NLM_p">To explore the effects of different molecular weight mPEG on the physicochemical properties of PG-1, we coupled different molecular weight (750, 1000, and 2000 Da) mPEG-COOH and mPEG-NH<sub>2</sub> to its N- or C-terminal via an amide bond, named as NPG<i><sub>n</sub></i> and CPG<i><sub>n</sub></i> (<i>n</i> represents the molecular weight of mPEG), respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). All PEGylated peptides were purified to >95% purity by reversed-phase high-performance liquid chromatography (RP-HPLC) and verified by matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Sequence of Engineered Peptides and Its Purity and Net Charge</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center">sequence</th><th class="colsep0 rowsep0" align="center">purity(%)</th><th class="colsep0 rowsep0" align="center">net charge</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PG-1</td><td class="colsep0 rowsep0" align="left">RGGRLCYCRRRFCVCVGR</td><td class="colsep0 rowsep0" align="left">95.02</td><td class="colsep0 rowsep0" align="left">+6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">mPEG<sub>750</sub>-RGGRLCYCRRRFCVCVGR</td><td class="colsep0 rowsep0" align="left">95.49</td><td class="colsep0 rowsep0" align="left">+6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">mPEG<sub>1000</sub>-RGGRLCYCRRRFCVCVGR</td><td class="colsep0 rowsep0" align="left">96.31</td><td class="colsep0 rowsep0" align="left">+6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">mPEG<sub>2000</sub>-RGGRLCYCRRRFCVCVGR</td><td class="colsep0 rowsep0" align="left">95.61</td><td class="colsep0 rowsep0" align="left">+6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">RGGRLCYCRRRFCVCVGR-mPEG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">95.52</td><td class="colsep0 rowsep0" align="left">+6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">RGGRLCYCRRRFCVCVGR-mPEG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">95.24</td><td class="colsep0 rowsep0" align="left">+6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">RGGRLCYCRRRFCVCVGR-mPEG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">95.73</td><td class="colsep0 rowsep0" align="left">+6</td></tr></tbody></table></div></div><div class="NLM_p">Subsequently, circular dichroism (CD) analysis was performed to analyze the secondary structure transformation of the PEGylated derivatives compared to PG-1 in 1.5 mM phosphate-buffered saline (PBS) and 1.5 mM sodium dodecyl sulfate (SDS). As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, PG-1 showed a mixed structure of a disordered conformation and β-sheets in PBS; this result was the same as that reported by Lindsey<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> because there was a negative peak around 210 nm and a positive peak around 230 nm, indicating the emergence of a mixed structure. This same trend was reported in CPG<sub>1000</sub>. Notably, CPG<sub>750</sub> and CPG<sub>2000</sub> mainly presented disordered structures in PBS, and the CD spectra of the two peptides were characterized by a strong negative peak around 205 nm and no clear positive peak.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. CD spectra of all peptides. All peptides were dissolved in 1.5 mM PBS (pH 7.4) and 1.5 mM SDS and the mean residual ellipticity was plotted against wavelength. The values from three scans were averaged per sample, and the peptide concentrations were fixed at 25 μg/mL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Instead, each spectrum of NPG<i><sub>n</sub></i> was dominated by a minimum near 216–220 nm and a positive maximum near 202–205 nm in PBS, which displayed a typical β-sheet structure. However, in SDS, all the engineered peptides showed sheet-like structures with positive peaks around 200 and 205 nm and negative peaks around 214 and 222 nm. We concluded that although mPEG did not cause gross alterations in the secondary structure, but small changes in mPEG may affect the intrinsic stability of each peptide.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> In Vitro Biological Activity Determination of the PEGylated Peptides</h3><div class="NLM_p">High biocompatibility is an important criterion for the clinical application of AMPs; as many previous studies have proved that PG-1 has high cytotoxicity against mammalian cells,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> a new strategy is required to address this problem. Thus, we selected human red blood cells and human embryonic kidney (HEK) 293T cells as model cells to assess whether PEGylation would improve the cell selectivity of PG-1. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, all tested PEGylated peptides showed cytotoxic effects in a dose-dependent manner. Interestingly, at the highest concentration, the cytotoxicity of NPG<i><sub>n</sub></i> was significantly lower than that of PG-1, showing good biocompatibility. However, a totally different situation was observed for CPG<i><sub>n</sub></i>, PEG coupling to the C-terminal could not significantly reduce the cytotoxicity of the high concentrations of PG-1, or even worse.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Biocompatibility of the engineered peptides: (A) hemolytic activities of the peptides against human red blood cells. (B) HEK 293T cells. The data were derived from three independent experiments and presented as the mean ± standard error (SE). *<i>P</i> < 0.01, **<i>P</i> < 0.001, and ***<i>P</i> < 0.0001, Student’s t-test, significant difference from the control group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To quantify the cytotoxicity of PEGylated peptides, HC<sub>10</sub> and IC<sub>50</sub> values of the PEGylated peptides were calculated. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, the HC<sub>10</sub> values of all PEGylated peptides were improved compared to PG-1, indicating that PEG modification could reduce the hemolytic activity of PG-1, and the PEG modification at the N-terminal had a better effect. Similarly, the IC<sub>50</sub> values of NPG<i><sub>n</sub></i> for HEK 293T cells were approximately twice higher than that of PG-1, but the IC<sub>50</sub> values of CPG<i><sub>n</sub></i> were similar to that of PG-1(<a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">Figure S4</a>). Overall, the PEG-coupled position had a great influence on the cytotoxicity of peptides, and the N-terminal modification could reduce their cytotoxicity. Subsequently, the antibacterial activity of the PEGylated peptides against Gram-negative, Gram-positive bacteria, and some drug-resistant strains was expressed with the minimum inhibitory/bactericidal concentrations (MICs/MBCs) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">Table S1</a>), and the geometric mean (GM) values of the MICs of the PEGylated peptides across all the tested bacteria is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, which represents the average antibacterial level of a peptide. As shown in the results, the PEG-coupled N-terminal obviously weakened the antimicrobial activity of PG-1 by 3.76–10.29-fold. Particularly for Gram-positive bacteria, NPG<i><sub>n</sub></i> showed almost no antibacterial activity. By contrast, CPG<i><sub>n</sub></i> were substantially more potent than either PG-1or NPG<i><sub>n</sub></i> against all tested strains. Among them, CPG<sub>2000</sub> exhibited the highest average antibacterial activity (GM<sub>all</sub> = 3.88), and its GM<sub>all</sub> value was 2.88 times lower than that of PG-1.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. MICs (MBCs) (μg/mL) of the Engineered Peptides against Gram-Negative Bacteria</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptides</th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>E. coli</i></th><th class="colsep0 rowsep0" align="center"><i>S. typhimurium</i></th><th class="colsep0 rowsep0" align="center"><i>S. typhimurium</i></th><th class="colsep0 rowsep0" align="center"><i>Salmonella pullorum</i></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">25922</th><th class="colsep0 rowsep0" align="center">987P</th><th class="colsep0 rowsep0" align="center">1515</th><th class="colsep0 rowsep0" align="center">21530</th><th class="colsep0 rowsep0" align="center">UB1005</th><th class="colsep0 rowsep0" align="center">078</th><th class="colsep0 rowsep0" align="center">K88</th><th class="colsep0 rowsep0" align="center">K99</th><th class="colsep0 rowsep0" align="center">14028</th><th class="colsep0 rowsep0" align="center">7731</th><th class="colsep0 rowsep0" align="center"><i>C7913</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PG-1</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">2 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">melittin</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">1 (1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">16 (32)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">32 (128)</td><td class="colsep0 rowsep0" align="left">64 (128)</td><td class="colsep0 rowsep0" align="left">8 (8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">>128<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">128(>128)</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">16 (16)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32 (64)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">32 (64)</td><td class="colsep0 rowsep0" align="left">128 (128)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">64 (64)</td><td class="colsep0 rowsep0" align="left">64 (128)</td><td class="colsep0 rowsep0" align="left">128 (128)</td><td class="colsep0 rowsep0" align="left">4 (4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">1 (1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">2 (2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">1 (1)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">>128 indicates the loss of antimicrobial activity.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. MICs (MBCs) (μg/mL) of the Engineered Peptides against Gram-Positive Bacteria</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptides</th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i></th><th class="colsep0 rowsep0" align="center"><i>E. faecalis</i></th><th class="colsep0 rowsep0" align="center"><i>S. epidermidis</i></th><th class="colsep0 rowsep0" align="center"><i>S. aureus</i></th><th class="colsep0 rowsep0" align="center"><i>Listeria monocytogenes</i></th><th class="colsep0 rowsep0" align="center"><i>Bacillus subtilis</i></th><th class="colsep0 rowsep0" align="center"><i>MRSA</i><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">29213</th><th class="colsep0 rowsep0" align="center">29212</th><th class="colsep0 rowsep0" align="center">12228</th><th class="colsep0 rowsep0" align="center">25923</th><th class="colsep0 rowsep0" align="center">1.1073</th><th class="colsep0 rowsep0" align="center">63501</th><th class="colsep0 rowsep0" align="center">43300</th><th class="colsep0 rowsep0" align="center"> </th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PG-1</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">melittin</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">2 (4)</td><td class="colsep0 rowsep0" align="left">1 (1)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">8 (16)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">1 (1)</td><td class="colsep0 rowsep0" align="left">8 (8)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">32 (32)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">2 (2)</td><td class="colsep0 rowsep0" align="left">16 (16)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (8)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left">1 (2)</td><td class="colsep0 rowsep0" align="left">4 (4)</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Methicillin-resistant <i>Staphylococcus aureus</i>.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. MHC, GM<sub>MIC</sub>, and TI Values of the Peptides</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" rowspan="3" align="center">peptide</th><th class="colsep0 rowsep0" rowspan="3" align="center">HC<sub>10</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a>(μg/mL)</th><th class="colsep0 rowsep0" rowspan="3" align="center">IC<sub>50</sub>(μg/mL)</th><th class="colsep0 rowsep0" colspan="4" align="center">GM (μg/mL)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" colspan="3" align="center">therapeutic index (TI)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" rowspan="2" align="center">GM<sub>HC10&IC50</sub></th><th class="colsep0 rowsep0" align="center">Gram-negative</th><th class="colsep0 rowsep0" align="center">Gram-positive</th><th class="colsep0 rowsep0" rowspan="2" align="center">all</th><th class="colsep0 rowsep0" rowspan="2" align="center">Gram-negative bacteria</th><th class="colsep0 rowsep0" rowspan="2" align="center">Gram-positive bacteria</th><th class="colsep0 rowsep0" rowspan="2" align="center">all</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">bacteria</th><th class="colsep0 rowsep0" align="center">bacteria</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PG-1</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">103.3</td><td class="colsep0 rowsep0" align="left">57.49</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">9.75</td><td class="colsep0 rowsep0" align="left">8.83</td><td class="colsep0 rowsep0" align="left">7.19</td><td class="colsep0 rowsep0" align="left">5.90</td><td class="colsep0 rowsep0" align="left">6.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">melittin</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">10.12</td><td class="colsep0 rowsep0" align="left">9.00</td><td class="colsep0 rowsep0" align="left">3.32</td><td class="colsep0 rowsep0" align="left">3.65</td><td class="colsep0 rowsep0" align="left">3.47</td><td class="colsep0 rowsep0" align="left">2.71</td><td class="colsep0 rowsep0" align="left">2.47</td><td class="colsep0 rowsep0" align="left">2.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">>512</td><td class="colsep0 rowsep0" align="left">256.9</td><td class="colsep0 rowsep0" align="left">512.90</td><td class="colsep0 rowsep0" align="left">30.05</td><td class="colsep0 rowsep0" align="left">141.32</td><td class="colsep0 rowsep0" align="left">65.16</td><td class="colsep0 rowsep0" align="left">17.07</td><td class="colsep0 rowsep0" align="left">3.63</td><td class="colsep0 rowsep0" align="left">7.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">>512</td><td class="colsep0 rowsep0" align="left">273.5</td><td class="colsep0 rowsep0" align="left">529.21</td><td class="colsep0 rowsep0" align="left">82.35</td><td class="colsep0 rowsep0" align="left">156.03</td><td class="colsep0 rowsep0" align="left">113.35</td><td class="colsep0 rowsep0" align="left">6.43</td><td class="colsep0 rowsep0" align="left">3.39</td><td class="colsep0 rowsep0" align="left">4.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">>512</td><td class="colsep0 rowsep0" align="left">248.2</td><td class="colsep0 rowsep0" align="left">504.14</td><td class="colsep0 rowsep0" align="left">49.74</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">79.79</td><td class="colsep0 rowsep0" align="left">10.14</td><td class="colsep0 rowsep0" align="left">3.94</td><td class="colsep0 rowsep0" align="left">6.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">88.1</td><td class="colsep0 rowsep0" align="left">106.19</td><td class="colsep0 rowsep0" align="left">4.83</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">6.22</td><td class="colsep0 rowsep0" align="left">21.99</td><td class="colsep0 rowsep0" align="left">13.27</td><td class="colsep0 rowsep0" align="left">17.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">109.4</td><td class="colsep0 rowsep0" align="left">118.34</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">14.5</td><td class="colsep0 rowsep0" align="left">10.77</td><td class="colsep0 rowsep0" align="left">14.79</td><td class="colsep0 rowsep0" align="left">8.16</td><td class="colsep0 rowsep0" align="left">10.99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">79.63</td><td class="colsep0 rowsep0" align="left">100.96</td><td class="colsep0 rowsep0" align="left">3.75</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">3.88</td><td class="colsep0 rowsep0" align="left">26.92</td><td class="colsep0 rowsep0" align="left">25.24</td><td class="colsep0 rowsep0" align="left">26.02</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">HC<sub>10</sub> is the minimum hemolytic concentration that caused 10% hemolysis of hRBCs. When no detectable hemolytic activity was observed at 512 μg/mL, a value of 1024 μg/mL was used to calculate the TI.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">The GM of the peptide MICs against bacteria was calculated. When no detectable antimicrobial activity was observed at 128 μg/mL, a value of 256 μg/mL was used to calculate the GM.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">TI is calculated as GM<sub>HC10IC50</sub>/GM<sub>MIC</sub>. Larger values indicate greater therapeutic potential.</p></div></div></div><div class="NLM_p">Maintaining high stability in a complex physiological environment is another prerequisite for the clinical application of AMPs. Thus, we evaluated the antibacterial activity of the PEGylated peptides against <i>Escherichia coli (E. coli)</i> in the presence of serum and salts to reflect their stability in a physiological environment. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, compared with the respective controls, no or only twofold decrease in the antibacterial activities of the PEGylated peptides was observed in the presence of most of the cations, while Ca<sup>2+</sup> obviously compromised the antibacterial activity of all PEGylated peptides by up to fourfold, indicating that except for Ca<sup>2+</sup>, the PEGylated peptides showed strong resistance to most salt ions similar to PG-1. Furthermore, it was worth noting that the addition of serum had a dose-dependent inhibitory effect on the antimicrobial activity of PEGylated peptides. With 50% serum, the MIC of all PEGylated peptides increased by more than 4 times, while 25 and 12.5% serum had relatively little effect on the antimicrobial activity of the PEGylated peptides. A comprehensive analysis of the GM<sub>MIC</sub> values of the PEGylated peptides was conducted across all tested serum concentrations, and we found that the GM<sub>MIC</sub> values of NPG<sub>750</sub> and CPG<sub>750</sub> had the lowest amount of variability, indicating that low-molecular-weight PEG coupling could improve the stability of PG-1 in serum. Comparatively, the GM<sub>MIC</sub> values of CPG<sub>2000</sub> had the same amount of variability as PG-1 across all tested serum concentrations. However, among all tested PEGylated peptides, CPG<sub>2000</sub> still had the highest antibacterial activity in the serum environment, which meant that CPG<sub>2000</sub> possessed more clinical application potential.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. MIC Values (μg/mL) of the Engineered Peptides against <i>E. Coil</i> in the Presence of Physiological Salts and Serum</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" rowspan="2" align="center">peptide</th><th class="rowsep1 colsep0" rowspan="2" align="center">control<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="9" align="center">physiological salt concentration<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="rowsep1 colsep0" colspan="5" align="center">serum concentration (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">NaCl</th><th class="colsep0 rowsep0" align="center">KCl</th><th class="colsep0 rowsep0" align="center">NH<sub>4</sub>Cl</th><th class="colsep0 rowsep0" align="center">MgCl<sub>2</sub></th><th class="colsep0 rowsep0" align="center">CaCl<sub>2</sub></th><th class="colsep0 rowsep0" align="center">ZnCl<sub>2</sub></th><th class="colsep0 rowsep0" align="center">FeCl<sub>3</sub></th><th class="colsep0 rowsep0" align="center">GM</th><th class="colsep0 rowsep0" align="center">fold<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">50</th><th class="colsep0 rowsep0" align="center">25</th><th class="colsep0 rowsep0" align="center">12.5</th><th class="colsep0 rowsep0" align="center">GM</th><th class="colsep0 rowsep0" align="center">fold</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PG-1</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">11.89</td><td class="colsep0 rowsep0" align="left">1.49</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32.00</td><td class="colsep0 rowsep0" align="left">4.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">26.25</td><td class="colsep0 rowsep0" align="left">1.64</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">40.32</td><td class="colsep0 rowsep0" align="left">2.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">45.25</td><td class="colsep0 rowsep0" align="left">1.41</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">128.00</td><td class="colsep0 rowsep0" align="left">4.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32.00</td><td class="colsep0 rowsep0" align="left">2.00</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">80.63</td><td class="colsep0 rowsep0" align="left">5.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>750</sub></td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">5.38</td><td class="colsep0 rowsep0" align="left">1.35</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">10.08</td><td class="colsep0 rowsep0" align="left">2.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>1000</sub></td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">11.89</td><td class="colsep0 rowsep0" align="left">1.49</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">25.40</td><td class="colsep0 rowsep0" align="left">3.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CPG<sub>2000</sub></td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2.97</td><td class="colsep0 rowsep0" align="left">1.49</td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">8.00</td><td class="colsep0 rowsep0" align="left">4.00</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">The control MIC values were determined in the absence of physiological salts and serum.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">The final concentrations of NaCl, KCl, NH<sub>4</sub>Cl, MgCl<sub>2</sub>, CaCl<sub>2</sub>, ZnCl<sub>2</sub>, and FeCl<sub>3</sub> were 150 mM, 4.5 mM, 6 μM, 1 mM, 2 mM, 8 μM, and 4 μM, respectively.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Fold change is calculated as GM/control. Lower values indicate a smaller fold change in the MIC value.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Screening of the PEGylated Peptides with High Clinical Therapeutic Potency</h3><div class="NLM_p">To evaluate and quantify the therapeutic potency of the PEGylated peptides, the TI (GM<sub>HC10&IC50</sub>/GM<sub>MIC</sub>) of the PEGylated peptides was further calculated and is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. The data showed that NPG<sub>750</sub> (TI<sub>all</sub> = 7.87) and CPG<sub>2000</sub> (TI<sub>all</sub> = 26.02) possessed the highest TI value within their respective series, which was1.21 and 4 times that of PG-1(TI<sub>all</sub> = 6.51), respectively. Considering the different advantages, both NPG<sub>750</sub> (with high stability) and CPG<sub>2000</sub> (with the best antibacterial activity) showed great potential for applications in the clinic.</div><div class="NLM_p">Furthermore, similar to PG-1, both NPG<sub>750</sub> and CPG<sub>2000</sub> eliminated more than 99.99% of <i>E. coli</i> cells within 30 s at 1× MIC, showing that NPG<sub>750</sub> and CPG<sub>2000</sub> had rapid sterilization ability in the clinic (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Time–kill curves of peptides at 1× MIC against <i>E. coli</i>. The data were derived from three independent experiments and presented as mean ± SE. (B) Resistance development of peptides. Exposure of peptides and ampicillin-sensitive strain of <i>E. coli</i> to sub-MIC dosing for 30 sequential passages. The test was carried out three times independently with analogous results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Currently, resistance to AMPs has evolved in some resistant microorganisms.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Thus, we also assessed the resistance development potential of <i>E. coli</i> to NPG<sub>750</sub> and CPG<sub>2000</sub> by conducting a drug resistance study. As the control, the ampicillin treatment began to develop resistance as early as passage 16, and its MIC value raised by 32-fold after 30 passages (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). On the contrary, the antimicrobial potency of NPG<sub>750</sub> and CPG<sub>2000</sub> was unaltered for the duration of the test, which suggested that NPG<sub>750</sub> and CPG<sub>2000</sub> had a low likelihood of resistance development.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Nanocharacterization and Trypsin Stability of NPG<sub>750</sub> and CPG<sub>2000</sub></h3><div class="NLM_p">In order to verify whether the PEGylated peptides could form self-assembled nanostructures, the critical aggregation concentrations (CACs) of NPG<sub>750</sub> and CPG<sub>2000</sub> were first determined using the 1-anilino-8-naphthalene sulfonate (ANS) fluorescence assay, which is an anionic dye that produces high fluorescence intensity when it binds to biological macromolecules. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, the ANS fluorescence intensity of NPG<sub>750</sub> and CPG<sub>2000</sub> increased with the increase of peptide concentrations and aggregation strength, and the CAC values of NPG<sub>750</sub> and CPG<sub>2000</sub> were 32.64 and 59.25 μg/mL, respectively, indicating that the PEGylated peptide molecules could aggregate to form a nanostructure. Instead, the fluorescence of PG-1 showed no significant increase in ultrapure water (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">Figure S5</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) ANS fluorescence binding assay for NPG<sub>750</sub> and CPG<sub>2000</sub> at different concentrations in deionized water (pH 7.4) and their hydrodynamic size at 25 °C. (B) TEM images and SEM images of engineered peptides. (C) Fold changes of MIC values of peptides in the presence of 8, 4, 2, and 1 mg/mL trypsin. The test was carried out three times independently. (D) 8 mg/mL trypsin and peptides were incubated at 37 °C for 1, 2, 4, and 8 h, respectively. The molecular weight of the marker from top to bottom corresponds to 31, 20.1, 14.4, 6.5, and 3.3 KDa, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The micromorphologies of nanostructured NPG<sub>750</sub> and CPG<sub>2000</sub> were further analyzed by transmission electron microscopy (TEM) and scanning electron microscopy (SEM). The combined results showed that both NPG<sub>750</sub> and CPG<sub>2000</sub> could form spherical and fibrous nanostructures above the CAC. Intriguingly, NPG<sub>750</sub> tended to form a dense nanofiber reticular structure, while CPG<sub>2000</sub> tended to form spherical micelles with diameters of around 20 nm (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The dynamic light scattering (DLS) data confirmed the reliability of the above results.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29−31)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, both NPG<sub>750</sub> and CPG<sub>2000</sub> exhibited two average hydrodynamic diameters, suggesting that NPG<sub>750</sub> and CPG<sub>2000</sub> presented two micromorphologies in aqueous solution. The above results confirmed that PEG coupling could facilitate the self-assembly of PG-1. Subsequently, we focused on the proteolytic resistance of the nanostructured NPG<sub>750</sub> and CPG<sub>2000</sub>. To study the bactericidal activities, PEGylated peptides (512 μg/mL) were incubated with different concentrations of trypsin (the highest concentration up to 8 mg/mL) for a continuous time of 1–8 h and the bactericidal activities were determined to preliminarily reflect their antitrypsin hydrolytic ability. The results are shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C. PG-1 and melittin, which were chosen as controls, were completely inactivated after incubation with 1 mg/mL trypsin for 1 h. However, CPG<sub>2000</sub> was not completely inactivated after incubation with high concentrations of trypsin (2–8 mg/mL), but its MIC value against <i>E. coli</i> increased, indicating that the antitrypsin hydrolytic ability of CPG<sub>2000</sub> was stronger than that of PG-1. It is worth noting that NPG<sub>750</sub> possessed extremely high proteolytic resistance and it still maintained the same antibacterial activity as the untreated NPG<sub>750</sub> after 8 h of incubation with different concentrations of trypsin.</div><div class="NLM_p">In order to ensure that the high proteolytic resistance of NPG<sub>750</sub> and CPG<sub>2000</sub> was caused by self-aggregation rather than their own characteristics, we also determined the bactericidal activities of monomers NPG<sub>750</sub> and CPG<sub>2000</sub> after incubation with trypsin. The data showed that the antibacterial activity of monomers NPG<sub>750</sub> and CPG<sub>2000</sub> was lost (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">Figure S6</a>), meaning that the antitrypsin hydrolytic ability of NPG<sub>750</sub> and CPG<sub>2000</sub> came from self-aggregation.</div><div class="NLM_p last">The antitrypsin hydrolytic abilities of the PEGylated peptides were further verified using the 16.5% Tricine–SDS-PAGE assay. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D shows that NPG<sub>750</sub> showed great resistance to hydrolysis of trypsin, even after 8 h of incubation, as similar bands to those of the controls were still present. However, CPG<sub>2000</sub> could be degraded after incubation with trypsin, but the bands corresponding to CPG<sub>2000</sub> were still visible, indicating that trypsin worked incompletely on CPG<sub>2000</sub>. In comparison, melittin and PG-1 were completely degraded after incubation with trypsin for 1 h. In a word, the above results confirmed that PEG-coupling modification could facilitate the self-aggregation of PG-1 to form nanostructures and improved the proteolytic resistance of PG-1.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Membrane-Active Mechanism Analysis of NPG<sub>750</sub> and CPG<sub>2000</sub></h3><div class="NLM_p">Many studies believe that the AMPs mainly exert antibacterial activity through inducing the formation of transmembrane pores.<a onclick="showRef(event, 'ref13 ref32'); return false;" href="javascript:void(0);" class="ref ref13 ref32">(13,32)</a> Thus, NPG<sub>750</sub> and CPG<sub>2000</sub> were further studied for their membrane-active mechanism to kill the model Gram-negative bacteria <i>E. coli</i>.</div><div class="NLM_p">Lipopolysaccharides (LPS), as a major component of the outer membrane of almost all Gram-negative bacteria, are the initial site of interaction between AMPs and the bacterial outer membrane. Thus, the LPS binding activities of NPG<sub>750</sub> and CPG<sub>2000</sub> were tested and the results revealed that NPG<sub>750</sub> and CPG<sub>2000</sub> showed dose-dependent binding with LPS (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Subsequently, a competitive inhibition assay between LPS and the PEGylated peptides was conducted to further clarify the initial sites of action of the PEGylated peptides. <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B shows that the coincubation of LPS with the PEGylated peptides had a dose-dependent inhibition effect on the bactericidal activity of the PEGylated peptides, and the antibacterial activities of the PEGylated peptides were completely blocked when the concentration of LPS was 1024 μg/mL, which further suggests that NPG<sub>750</sub> and CPG<sub>2000</sub> initially bind to LPS.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Peptide-binding affinity to LPS (<i>E. coli</i> O111:B4). The excitation wavelength was 580 nm and the emission wavelength was 620 nm. The graphs were derived from the average of three independent trials. All the data are expressed as mean ± SD. (B) Cell mortality of <i>E. coli</i> cells as a function of LPS concentration. The concentration of peptides used was fixed at 1× MIC. (C) The OM permeability induced by tested peptides. The fluorescence intensity was monitored at an excitation wavelength of 350 nm and an emission wavelength of 420 nm. (D) Membrane potential osmotic dye DISC<sub>3</sub>-5 was used to detect the effects of (D-l) NPG<sub>750</sub> and (D-2) CPG<sub>2000</sub> on the membrane potential of <i>E. coli</i> cells. The excitation wavelength was 622 nm, the emission wavelength was 670 nm, and the duration was 2000 s. (E) Inner membrane (IM) permeability of peptides. The hydrolysis of <i>o</i>-nitrophenyl-β-<span class="smallcaps smallerCapital">d</span>-galactopyranoside (ONPG) due to the release of cytoplasmic β-galactosidase of <i>E. coli</i> treated by different concentrations of (E-1) NPG<sub>750</sub> and (E-2) CPG<sub>2000</sub>. The duration was 5000 s. (F) Mimic bacterial phospholipid bilayer liposomes. POPC:POPG = 3:1(molar:molar). *<i>P</i> < 0.01, **<i>P</i> < 0.001, and ***<i>P</i> < 0.0001, Student’s t-test, significant difference from the control group. (G) Simulated eukaryotic membrane liposomes. POPC:POPG = 10:1(molar:molar).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The outer membrane (OM) of Gram-negative bacteria is the first barrier that AMPs need to penetrate. The hydrophobic fluorescent probe NPN was applied to evaluate the OM permeability of the PEGylated peptides. NPG<sub>750</sub> and CPG<sub>2000</sub> permeabilized the OM of <i>E. coli</i> in a concentration-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). At high concentrations (64–128 μg/mL), the OM permeability induced by NPG<sub>750</sub> was stronger than those of PG-1 and CPG<sub>2000</sub>. However, CPG<sub>2000</sub> showed the strongest OM permeability when the concentration was below 32 μg/mL. Thus, the results demonstrate that NPG<sub>750</sub> and CPG<sub>2000</sub> could permeabilize the OM of Gram-negative bacteria.</div><div class="NLM_p">Destroying the cytoplasmic membrane (CM) of Gram-negative bacteria is the key step for AMPs to exert their antimicrobial activity. The CM depolarization assay results showed that NPG<sub>750</sub> and CPG<sub>2000</sub> caused a dose- and time-dependent fluorescence increase for <i>E. coli</i> within 2000 s, and even at low concentrations, NPG<sub>750</sub> and CPG<sub>2000</sub> also could cause rapid CM depolarization of <i>E. coli</i> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). The CM permeability of the PEGylated peptides was investigated using the probe of <i>o</i>-nitrophenyl-β-<span class="smallcaps smallerCapital">d</span>-galactopyranoside (ONPG). As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">Figure S7</a>, CPG<sub>2000</sub> and melittin induced the rapid release of β-galactosidase at 4 μg/mL, while the release rates of PG-1 and NPG<sub>750</sub> were slightly slower, but they also increased with the increase of the peptide concentration. We also utilized calcein-loaded large unilamellar vesicles (LUVs) (POPC/POPG = 3:1 (molar/molar)) as a model of the CM of Gram-negative bacteria to monitor the pore formation induced by NPG<sub>750</sub> and CPG<sub>2000</sub>. The results showed that both NPG<sub>750</sub> and CPG<sub>2000</sub> could induce good leakage activity similar to PG-1 at 1× MIC, confirming the pore formation induced by NPG<sub>750</sub> and CPG<sub>2000</sub> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>F). Interestingly, with the increase of the concentration to 2× CAC, the leakage rate of calcein caused by NPG<sub>750</sub> obviously decreased, suggesting that the membrane-active mechanism of NPG<sub>750</sub> was reduced. For the LUVs composed of phosphatidylcholine (PC) with cholesterol that mimic the mammalian cell membrane, NPG<sub>750</sub> showed similar results (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>G). These results showed that above the CAC, the formation of nanostructures would reduce the ability of NPG<sub>750</sub> to disrupt the membrane integrity, while reducing its cytotoxicity.</div><div class="NLM_p">To better understand the membrane-active mechanisms of NPG<sub>750</sub> and CPG<sub>2000</sub>, TEM and SEM were used to visually investigate the surface and internal morphology changes of <i>E. coli</i> treated with the PEGylated peptides. After treatment with NPG<sub>750</sub> and CPG<sub>2000</sub>, the SEM clearly demonstrated that the bacterial cells showed obvious morphological shrinkage, cytoplasmic leakage, and pore formation, while the formation of blank areas and the separation of the plasma membrane and cell wall were also observed by TEM. These micrograph observations further demonstrated that NPG<sub>750</sub> and CPG<sub>2000</sub> exert antibacterial activity through membrane-active mechanisms (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. SEM and TEM micrographs of <i>E. coli</i> K88 treated with peptides at 1× MIC. SEM images of (A) control <i>E. coli</i> K88, (B) <i>E. coli</i> K88 treated with PG-1, (C) NPG<sub>750</sub>, and (D) CPG<sub>2000</sub>. (E) TEM image of control <i>E. coli</i> K88; (F) <i>E. coli</i> K88 treated with PG-1 showed that the cytoplasm leaked to form a blank area. (G) NPG<sub>750</sub> caused a marked contraction of the membrane of <i>E. coli</i> K88. (H) CPG<sub>2000</sub> caused bacterial rupture and separation of the plasma membrane and cell wall.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A variety of self-assembled peptide nanodrugs has been widely used in biotechnology. These peptides are self-assembled to form well-defined nanostructures such as tubes, bands, fibers, spheres, and micelles in the past few decades. The self-assembly of peptides is essential for their clinical application with the advantages of good biocompatibility, versatility, and specific recognition.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a></div><div class="NLM_p">In the present study, PG-1 was coupled with PEG to form nanostructures, reducing the toxicity and solving the problem of lower trypsin stability. To retain the effect of PG-1 on induced transmembrane migration and bacterial leakage, two types of PEG binding sites were set: terminal carboxyl group (CPG<i><sub>n</sub></i>) and an amino group (NPG<i><sub>n</sub></i>). A series of engineered peptides NPG<i><sub>n</sub></i> and CPG<i><sub>n</sub></i> was designed for developing new antibacterial nanodrugs, which have low cytotoxicity and high trypsin stability. The results displayed some differences between N-terminal and C-terminal functions. The CD analysis showed that PEG modification did not change the secondary structure of PG-1, but NPG<i><sub>n</sub></i> promoted the rapid formation of folded antiparallel β-sheets and an angular domain of PG-1. All peptides formed stable β-sheets in SDS. At the same time, NPG<i><sub>n</sub></i> mainly existed in the antiparallel β-sheet form in PBS solution, which may be related to the formation of a nanofiber network. The increase of free energy driven by the rise in the β-sheet content promotes the construction of fiber structures with intermolecular hydrogen bonding. NPG<sub>750</sub>, a kind of NPG<i><sub>n</sub></i>, was mainly present as β-sheets in PBS, which was induced by the signal of a higher aggregate tendency. However, CPG<sub>750</sub> and CPG<sub>2000</sub>, two kinds of CPG<i><sub>n</sub></i>, showed a more disordered conformation in PBS, indicating that the C-terminal modification of PG-1 was flexible and tolerates structural and functional changes. Although the C-terminal conversion is flexible in theory, some of the functions obtained are persuasive.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><div class="NLM_p">The antibacterial effect played a major function in remolding PG-1. The antibacterial activity was significantly reduced in NPG<i><sub>n</sub></i> but improved with varying degrees in CPG<i><sub>n</sub></i>. As known to all, the antibacterial mechanism of AMPs is not well elucidated clearly. A widely accepted theory is that the positive charge and hydrophobic amino acids of AMPs bind to the lipid bilayer receptors on bacteria through electrostatic gravity and hydrophobic forces, resulting in bacterial death with membrane leakage.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> PEG could break the delicate balance between hydrophilicity and hydrophobicity of AMPs itself. Furthermore, PEGylation has no effect on the physical properties of the membrane surface. For recognization, the N-terminal of PG-1 preferentially inserts into the phospholipid bilayer,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and PEG closed the positive charge of arginine and hydrophobicity leading to the decrease or even loss of their activity in NPG<i><sub>n</sub></i>. In contrast, the amino terminal of CPG<sub>2000</sub> was not affected by the mPEG coupling at the C-terminal, which could also insert into the cell membrane. This transformation increased the surface area of CPG<sub>2000</sub> acting on lipids, improving the antibacterial activity. It is worth noting that the MIC values of NPG<sub>750</sub> and CPG<sub>2000</sub> for <i>E. coli</i> K88 were 16 and 2 μg/mL, respectively, which were lower than their CAC values of 32.64 and 59.25 μg/mL; these results indicated that the killing of bacteria by NPG<sub>750</sub> and CPG<sub>2000</sub> was still by their monomer molecules.</div><div class="NLM_p">The toxicity of AMPs is an important risk factor in clinical applications. NPG<sub>750</sub> and CPG<sub>2000</sub> showed dose-dependent toxicity on human blood cells and HEK 293T cells. For NPG<sub>750</sub>, the charge shielding effect may be one of the reasons for reducing toxicity.<a onclick="showRef(event, 'ref20 ref38'); return false;" href="javascript:void(0);" class="ref ref20 ref38">(20,38)</a> The fibrosis of N-terminal mPEG produces a sizeable steric hindrance to form a shell to wrap charged amino acids and hydrophobic amino acids, which reduces the binding force of NPG<sub>750</sub> to the eukaryotic cell membrane. For PG-1, the C-terminal is flexible, and the N-terminal of AMPs preferentially inserts into the cell membrane,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> which retains the spatial site of CPG<sub>2000</sub> acting on the cell membrane. Because of this flexibility, the trend of CPG<sub>2000</sub> aggregation is weak. The nanoaggregates do not encapsulate CPG<sub>2000</sub> completely, and the positive charge exposes as a hydrophilic group. When the nanoparticles were close to the cell membrane, the larger sphere volume induced an increase of cytotoxicity with large-scale tearing of the cell membrane. The surface of bacterial cells and eukaryotic cells is composed of different lipid compositions. The OM of mammalian cells is mainly composed of PC, sphingomyelin, and some cholesterol molecules, which are electrically neutral under physiological pH. Furthermore, the extracellular lobules of bacteria are composed of phosphatidylglycerol (PG) and negatively charged phospholipids, which provides the relative specificity of peptides for cell selection.<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39−41)</a></div><div class="NLM_p">Further study of the antibacterial mechanism revealed that all the engineered peptides retained the membrane dissolution effects as PG-1 in a concentration-dependent trend. The results showed that the OM permeability of NPG<sub>750</sub> was more vital than its IM permeability and LPS binding ability compared with PG-1. The stability and membrane breaking ability of CPG<sub>2000</sub> were improved. The creation of nanofibers reduced the ability of binding to negatively charged LPS by the encapsulated positively charged and hydrophobic amino acids with the PEG shell at high peptide concentrations.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The penetration of NPG<sub>750</sub> may be accomplished by aggregation on the OM of bacteria. The covering of the fiber network increases the surface area of the engineered peptide contact membrane and causes instability to the OM, and a large number of fibers inserting into the membrane resulted in higher OM permeability of NPG<sub>750</sub>. For example, some surfactant-like self-assembly peptides form nanorods to complete the sterilization process by inserting into the OM to increase the surface tension of the membrane.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> CPG<sub>2000</sub> caused a significant increase of IM permeability and binding ability than PG-1 at low concentrations (4–16 μg/mL). This was not caused by spherical micelles because the MIC value of CPG<sub>2000</sub> against <i>E. coli</i> K88 (2 μg/mL) was lower than that of PG-1 (8 μg/mL). This may be determined by the antibacterial ability of the peptide itself. In the LUVs of the simulated bacterial membrane (POPC/POPG = 3:1) and eukaryotic cell membrane (POPC/POPG = 10:1), the binding rate of engineered peptides to liposome membrane decreased after self-assembly. When the hydrophilic shell wraps the core peptide, the local density of PG-1 itself decreases. Therefore, it showed a low binding rate of the bacterial and eukaryotic cell membranes. This indicates that the toxicity of the self-assembled engineered peptides to bacteria and eukaryotic cells will be reduced. A schematic model of the antibacterial mechanism is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Possible mechanism of NPG<sub>750</sub> and CPG<sub>2000</sub> against <i>E. coli</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>, trypsin mainly acts on the peptide bond formed by lysine and arginine carboxyl. Arginine and lysine of melittin and PG-1 were cut off, making them lose their ability to adhere to the bacterial membrane.<a onclick="showRef(event, 'ref27 ref43'); return false;" href="javascript:void(0);" class="ref ref27 ref43">(27,43)</a> The fiber network, the high aggregation trend of NPG<sub>750</sub>, composed of a hydrophilic shell to avoid the enzyme cutting site of trypsin, and mPEG acted as the carrier; CPG<sub>2000</sub> showed an incomplete aggregation ability, and the active center was not encapsulated in the spherical micelle of CPG<sub>2000</sub> aggregation. Because there were still a small number of fibers in CPG<sub>2000</sub> at high concentrations, it was difficult for the spherical micelle structure to resist the cleavage of trypsin cleavage. The small number of fibers may be the reason for the higher antitrypsin ability of CPG<sub>2000</sub> than PG-1.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Schematic diagram of the antitrypsin mechanism of engineered peptides. Engineered peptides form nanostructures to resist the hydrolysis of trypsin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The previous studies had shown that the decrease and loss of activity were common with the introduction of PEG at the N-terminal of peptides or proteins,<a onclick="showRef(event, 'ref19 ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref19 ref44 ref45 ref46">(19,44−46)</a> but few studies reported the modification of mPEG at the C-terminal. A comparative report study had reported the N-terminal and C-terminal coupling mPEG of GIP. The N-terminal and C-terminal residues of GIP (1–30) were modified with PEG (long chain, 40 KDa), and the N-terminal modification of GIP directly abolished the functional activity. On the contrary, the C-terminal-modified GIP maintained the original mobility and potency of receptor binding and produced tolerance to dipeptidyl peptidase IV (DPPIV). It was also found that the resistance of C-terminal-modified GIP to DPPIV was not complete. This resistance was stabilized by palmitoyl introduced into the N-terminal. It seemed that the functional change of the C-terminal was regulated by the N-terminal.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The present study showed a different result from the previous studies. First, mPEG has a low molecular weight (750–2000 Da), and this modification will not completely inactivate NPG<i><sub>n</sub></i>. Second, additional modifiers on the N-terminal of CPG<sub>2000</sub> were not needed to stabilize the C-terminal function. The study indicates that the modification sites of PEG were different and a non-natural cysteine thiol group at the end of GIP for PEG binding. Both NPG<i><sub>n</sub></i> and CPG<i><sub>n</sub></i> showed that the length of low molecular weight mPEG had no significant correlation with the antibacterial activity, toxicity, and trypsin stability. Previous studies show a negative correlation between the molecular weight of mPEG and the activity of peptides.<a onclick="showRef(event, 'ref19 ref46 ref48'); return false;" href="javascript:void(0);" class="ref ref19 ref46 ref48">(19,46,48)</a> Combined with our research, the molecular weight of mPEG was too small to reach the threshold.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this study, by introducing PEG-coupling modification, we successfully converted the porcine AMP PG-1 into an efficacious nanoantibacterial agent, which greatly improved the proteolytic resistance of PG-1. Meanwhile, we found that PEG coupled to the N- or C-terminal showed different effects on the biological activity of PG-1. PEG coupled to the N-terminal would weaken the antibacterial activity of PG-1 while reducing its cytotoxicity, which was the opposite of PEG coupled to the C-terminal. Among all designed PEGylated peptides, NPG<sub>750</sub> and CPG<sub>2000</sub> possessed high cell selectivity, great stability in the physiological environment, and good antitrypsin hydrolytic activity. Additionally, NPG<sub>750</sub> and CPG<sub>2000</sub> exerted their bactericidal activity by membrane-active mechanisms, which suggested that microbes were less likely to develop drug resistance against them. In a word, the construction of nanoparticles modified by PEG is an effective strategy to improve the clinical application potential of AMPs, and this also opened up a new route to the research and development of nanodrugs.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71258" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71258" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis and Characterization of Peptides</h3><div class="NLM_p last">mPEG-COOH (750, 1000, and 2000 Da) and mPEG-NH<sub>2</sub> (750, 1000, and 2000 Da) were purchased from Pengsheng Biochemistry Co., Ltd. (Shanghai, China). Engineered peptides were synthesized by GL Biochem Corporation (Shanghai, China). The fidelity was measured by MALDI-TOF MS (Linear Science Co., Ltd.). The purity (>95%) and retention time of peptides were determined by RP-HPLC. The chromatographic column was GS-12 0-5-C18-BION 4.6 × 250 mm<sup>2</sup>, 220 nm, 10 μL, and the nonlinear water/acetonitrile gradient elution was used.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Bacterial Strains</h3><div class="NLM_p last">The bacteria strains <i>E. coli</i> ATCC 25922, <i>E. coli</i> K88, <i>E. coli</i> K99, <i>E. coli</i> 987P, <i>E. coli</i> CVCC 1515, <i>E. coli</i> CICC 21530, <i>E. coli</i> 078, <i>Salmonella pullorum</i> C7913, and <i>Salmonella typhimurium</i> ATCC14028 and C7731 were obtained from the College of Veterinary Medicine, Northeast Agricultural University (Harbin, China), and <i>E. coli</i> UB1005 was kindly provided by the State Key Laboratory of Microbial Technology (Shandong University, China). <i>Staphylococcus aureus (S. aureus)</i> ATCC 29213, <i>S. aureus</i> ATCC 25923, <i>S. aureus</i> 43300, <i>Staphylococcus epidermidis</i> ATCC 12228, <i>Bacillus subtilis</i> 63501, <i>Enterococcus faecalis</i> ATCC 29212, <i>P. aeruginosa</i> ATCC 27853, <i>P. aeruginosa</i> PAO1, and clinical isolates of <i>Acinetobacter baumannii</i> were provided by the 2nd Affiliated Hospital of Harbin Medical University (Harbin, China). <i>Listeria monocytogenes</i> CGMCC1.10753 were purchased from China General Microbiological Culture Collection Center (CGMCC).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> CD Spectroscopy</h3><div class="NLM_p last">The secondary structures of each peptide (25 μg/mL) were measured in 1.5 mM PBS (pH 7.4) and 1.5 mM SDS as previously described.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Biocompatibility Assays</h3><div class="NLM_p last">The biocompatibility of the engineered peptides was tested as previously described by assessing the cytotoxicity of the peptides to human red blood cells and HEK 293T cells.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Antimicrobial Activity Assays</h3><div class="NLM_p last">The MICs/MBCs of the engineered peptides against a series of bacterial strains were determined using the modified broth microdilution method of the National Committee for Clinical Laboratory Standards (NCCLS) as previously described.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The time–kill curve of the peptide was further plotted. A bacterial suspension (5 mL) (0.5–1 × 10<sup>5</sup> CFU/mL) was incubated with or without the peptide at 1× MIC, and at various exposure times, 100 μL of the samples was taken out and spread on Mueller–Hinton agar (MHA) plates to count the number of viable colonies and calculate the bacterial survival rate. The experiment was repeated three times independently.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Salt and Serum Sensitivity Assays</h3><div class="NLM_p last">The salt and serum sensitivity of a peptide was evaluated as previously described by determining the antibacterial activity of the peptide in the presence of physiological salts (150 mM NaCl, 4.5 mM KCl, 6 μM NH<sub>4</sub>Cl, 8 μM ZnCl<sub>2</sub>, 1 mM MgCl<sub>2</sub>, 2 mM CaCl<sub>2</sub>, and 4 μM FeCl<sub>3</sub>) or serum (12.5, 25, and 50%).<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The microorganism was <i>E. coli</i> K88. The experiment was repeated three times independently.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Drug Resistance Assay</h3><div class="NLM_p last">The drug resistance of <i>E. coli</i> K88 to the engineered peptides was evaluated using the sequential passaging method.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Ampicillin was employed as the model antibiotic for the drug resistance assay. First, the MICs of the peptide and ampicillin were determined as described previously. Next, each sub-MIC concentration of bacteria was diluted to 1 × 10<sup>5</sup> CFU/mL with Mueller–Hinton Broth (MHB, AOBOX) and was used for the next passage MIC determination. The above procedure was repeated for 30 passages.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> CAC Assay</h3><div class="NLM_p last">The CAC of the peptide was determined using the ANS fluorescence assay. ANS was dissolved in dimethylformamide to a concentration of 1 mM, and then 1 μL of this solution was added to 100 μL of different concentrations of peptide. The mixed solution was transferred to a quartz cuvette and tested using an F-4500 fluorescence spectrophotometer (Hitachi, Japan), and the fluorescence spectrum was collected from 420 to 550 nm with excitation at 360 nm. The CAC was determined from the matched curve, which was obtained by the ANS fluorescence intensities at 475 nm plotted against different concentrations of the peptide.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> DLS Assay</h3><div class="NLM_p last">The hydrodynamic diameter of the particles present in the peptide–ellipticine formulation was determined using a Zetasizer Nano Z90 (Malvern Instruments, Worcestershire, UK). Formulated solutions (1 mL) were placed in a disposable plastic cuvette. The reflective index and viscosity of pure water were used as the solvent settings, while the reflective index of the solute was set to be the same as that of the proteins. The temperature was maintained at 25 °C, and the solution was equilibrated for 2 min before data acquisition. Three measurements were performed for each sample. The data were automatically analyzed with the software to generate the intensity-based size distribution of the complexes.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Trypsin Stability Assay</h3><div class="NLM_p last">The trypsin stability of the peptides was evaluated using the modified MIC method and 16.5% Tricine–SDS-PAGE method.<a onclick="showRef(event, 'ref43 ref49'); return false;" href="javascript:void(0);" class="ref ref43 ref49">(43,49)</a> To determine the impact of trypsin concentration on the antimicrobial activities of the peptides, the engineered peptides were incubated with different concentrations of trypsin (1, 2, 4, and 8 mg/mL) for 1, 2, 4, and 8 h at 37 °C. <i>E. coli</i> K88 was used in the assays, and the subsequent method was the MIC test method. After the engineered peptides were incubated in the presence of 8 mg/mL trypsin for 1, 2, 4, and 8 h, the effect of incubation time on the engineered peptide was determined using the 16.5% Tricine–SDS-PAGE method. Finally, the engineered peptide was diluted to the concentration of MIC and incubated with trypsin, and then trypsin was removed in a water bath at 100 °C, and 20 μL was dropped into the MHA plate. The number of colonies was calculated every other day. To determine that the engineered peptide avoids the restriction site by formed a self-assembly structure.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> LPS Binding Assay</h3><div class="NLM_p last">The LPS binding of the peptides was determined using the BODIPY-TR-cadaverine (BC, Sigma, USA) displacement assay as previously described.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> LPS came from <i>E. coli</i> O111:B4 (Sigma, USA).</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> LPS Competitive Inhibition Assay</h3><div class="NLM_p last">LPS (50 μL) from <i>E. coli</i> O111:B4 and peptides (50 μL) were incubated for 1 h to achieve a final LPS concentration of 4–1024 μg/mL. Then, 100 μL of bacterial solution (0.5 × 10<sup>5</sup> CFU/mL) was added to the peptide–LPS mixture and incubated at 37 °C for 90 min. The final concentration of peptide was maintained at 1 × MIC. Then, a volume of 20 μL of the mixture was taken out from each well, diluted with an appropriate amount of PBS, spread on MHA agar plates, and cultured overnight. The number of colonies was counted. The experiment was repeated three times independently.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> OM Permeability Assays</h3><div class="NLM_p last">The OM permeabilization of the peptides against <i>E. coli</i> K88 was monitored using the fluorescent dye NPN uptake assay. The detailed protocol was consistent with the method reported by Wang et al.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> CM Depolarization Assays</h3><div class="NLM_p last">The change of the CM potential of <i>E. coli</i> K88 with peptide treatment was monitored using a membrane potential-dependent probe DiSC<sub>3</sub>-5 as previously described.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The fluorescence density was recorded from 0 to 2000 s.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> IM Permeability Assay</h3><div class="NLM_p last"><i>E. coli</i> K88 cells were incubated to mid-log phase in MHB, which contained 2% lactose at 37 °C, and then harvested and diluted to an OD<sub>600</sub> of 0.05 in 5 mM HEPES buffer (pH 7.4, containing 20 mM glucose and 1.5 mM ONPG). Then, cell suspensions and equal volumes of peptide solution (4–32 μg/mL) were added to 96-well plates and incubated at 37 °C. Images were recorded at time 0 and every 180 s until 5000 s at OD<sub>420</sub>.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Liposome Leakage</h3><div class="NLM_p last">Calcein was dissolved in 4 mol/L NaOH solution, a dye buffer (30 mM Tris, 20 mM NaCl, and 0.1 mM ethylenediaminetetraacetic acid, pH 7.4) was added until calcein was completely dissolved to 70 mM, and the solution was adjusted to 7.4 with HCl. Calcein-entrapped LUVs were made as described previously.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The unencapsulated calcein was eluted on a Sephadex G-50 column at a flow rate of 0.5 mL/min. The running buffer was Tris–HCl (50 mm, pH value 7.4). LUVs were diluted to 100 μM in the buffer. Then, 50 μL of liposome solution and 50 μL of peptide solution were incubated for15 min at room temperature. The calcein leakage rate was determined as mentioned earlier.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> SEM Characterization of Bacterial Samples</h3><div class="NLM_p last">Mid-log phase <i>E. coli</i> K88 cells (OD<sub>600nm</sub> = 0.02) were incubated with the peptides at 1× MIC for 1 h and then harvested and washed three times with PBS by centrifugation at 5000 g for 5 min.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Peptide samples: NPG<sub>750</sub> and CPG<sub>2000</sub> were smeared on the cover slide (10 μL), dried naturally in an aseptic operating table, and observed on the sample. The final concentrations of peptides were 512 μg/mL and the CAC value.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> TEM Characterization</h3><div id="sec5_18_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Bacterial Samples</h4><div class="NLM_p last">Bacterial samples were prepared according to the method reported by Wang et al.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Peptide samples: To characterize the shape of peptides in neutral and acidic Milli-Q water, NPG<sub>750</sub> and CPG<sub>2000</sub> were visualized using a TEM (Hitachi HT7800 TEM). The peptide solution (10 μL) was plated on copper-coated grids (400 square mesh, Electron Microscopy Sciences) for 5 min, and the grids were stained with 0.1% phosphotungstic acid for 30 s and dried for 15 min before imaging, operating at 100 kV.</div></div><div id="sec5_18_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Statistical Analysis</h4><div class="NLM_p last">The data were analyzed by ANOVA and Student’s t-test using Social Sciences (SPSS) version 20.0 (Chicago, Illinois, USA). Quantitative data were expressed as the mean ± SE/SD, and <i>P</i> < 0.01 was considered statistically significant.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00879" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44743" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44743" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00879?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00879</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">The 3D model and helical wheel projections of PG-1; RP-HPLC spectra of the engineered peptides; mass spectra of the engineered peptides; MICs (MBCs) (μg/mL) of the engineered peptides against a drug-resistant susceptible strain; IC<sub>50</sub> curves of engineered peptides on HEK 293T cells; ANS fluorescence binding assay for PG-1; resistance of engineered peptides (1× MIC) to trypsin; and CM depolarization assays and IM permeability assays of PG-1 and melittin (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf">jm1c00879_si_001.pdf (1.57 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00879" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55121" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anshan Shan</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d9b8aaaab1b8b799b7bcb8acf7bcbdacf7bab7"><span class="__cf_email__" data-cfemail="88e9fbfbe0e9e6c8e6ede9fda6edecfda6ebe6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weikang Yu</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-9181-4226" title="Orcid link">https://orcid.org/0000-0001-9181-4226</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiajun Wang</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-2086-2105" title="Orcid link">https://orcid.org/0000-0002-2086-2105</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zihang Wang</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lingxue Li</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenyu Li</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Song</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shanshan Zhang</span> - <span class="hlFld-Affiliation affiliation">Institute of Animal Nutrition, Northeast Agricultural University, No. 600 Changjiang Road, Xiangfang
District, Harbin, Heilongjiang 150030, P. R. China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>W.K.Y. and J.J.W. conducted the main experiment assay and contributed equally to this work; Z.H.W. and L.X.L. conducted the SEM assay. W.Y.L. conducted the TEM assay. W.K.Y. and J.J.W. wrote the main manuscript text. A.S.S. supervised the work and revised the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (32030101, 31872368, and 32002215), by China Agriculture Research System of MOF and MARA, and the Natural Science Foundation of Heilongjiang Province (TD2019C001).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMPs</td><td class="NLM_def"><p class="first last">antimicrobial peptides</p></td></tr><tr><td class="NLM_term">PG-1</td><td class="NLM_def"><p class="first last">protegrin-1</p></td></tr><tr><td class="NLM_term">TI</td><td class="NLM_def"><p class="first last">therapeutic index</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">mPEG</td><td class="NLM_def"><p class="first last">polyethylene glycol monomethyl</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reversed-phase high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">MALDI-TOF MS</td><td class="NLM_def"><p class="first last">matrix-assisted laser desorption/ionization time-of-flight mass spectrometry</p></td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">circular dichroism</p></td></tr><tr><td class="NLM_term">HEK</td><td class="NLM_def"><p class="first last">human embryonic kidney</p></td></tr><tr><td class="NLM_term">GM</td><td class="NLM_def"><p class="first last">geometric mean</p></td></tr><tr><td class="NLM_term">CAC</td><td class="NLM_def"><p class="first last">critical aggregation concentrations</p></td></tr><tr><td class="NLM_term">ANS</td><td class="NLM_def"><p class="first last">1-anilino-8-naphthalene sulfonate</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">scanning electron microscopy</p></td></tr><tr><td class="NLM_term">TEM</td><td class="NLM_def"><p class="first last">transmission electron microscopy</p></td></tr><tr><td class="NLM_term">OM</td><td class="NLM_def"><p class="first last">outer membrane</p></td></tr><tr><td class="NLM_term">IM</td><td class="NLM_def"><p class="first last">inner membrane</p></td></tr><tr><td class="NLM_term">ONPG</td><td class="NLM_def"><p class="first last"><i>o</i>-nitrophenyl-β-<span class="smallcaps smallerCapital">d</span>-galactopyranoside</p></td></tr><tr><td class="NLM_term">LUVs</td><td class="NLM_def"><p class="first last">large unilamellar vesicles</p></td></tr><tr><td class="NLM_term">PC</td><td class="NLM_def"><p class="first last">phosphatidylcholine</p></td></tr><tr><td class="NLM_term">PG</td><td class="NLM_def"><p class="first last">phosphatidylglycerol</p></td></tr><tr><td class="NLM_term">DPPIV</td><td class="NLM_def"><p class="first last">dipeptidyl peptidase IV</p></td></tr><tr><td class="NLM_term">NCCLS</td><td class="NLM_def"><p class="first last">National Committee for Clinical Laboratory Standards</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">NPN</td><td class="NLM_def"><p class="first last"><i>N</i>-phenyl-1-naphthylamine</p></td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">BODIPY-TR-cadaverine</p></td></tr><tr><td class="NLM_term">DiSC<sub>3</sub>-5</td><td class="NLM_def"><p class="first last">dipropylthiadicarbocyanine iodide</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last"><i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid</p></td></tr><tr><td class="NLM_term">MHB</td><td class="NLM_def"><p class="first last">Mueller–Hinton Broth.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06161" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06161" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thulshan
Jayathilaka, E. H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liyanage, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapaksha, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dananjaya, S. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikapitiya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Zoysa, M.</span></span> <span> </span><span class="NLM_article-title">Octominin: An antibacterial and anti-biofilm peptide for controlling the multidrug resistance and pathogenic streptococcus parauberis</span>. <i>Fish Shellfish Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.fsi.2020.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.fsi.2020.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=33378697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3MXitlSisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2021&pages=23-34&author=E.+H.+T.+Thulshan%0AJayathilakaauthor=T.+D.+Liyanageauthor=D.+C.+Rajapakshaauthor=S.+H.+S.+Dananjayaauthor=C.+Nikapitiyaauthor=I.+Whangauthor=M.+De+Zoysa&title=Octominin%3A+An+antibacterial+and+anti-biofilm+peptide+for+controlling+the+multidrug+resistance+and+pathogenic+streptococcus+parauberis&doi=10.1016%2Fj.fsi.2020.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Octominin: An antibacterial and anti-biofilm peptide for controlling the multidrug resistance and pathogenic Streptococcus parauberis</span></div><div class="casAuthors">Thulshan Jayathilaka, E. H. T.; Liyanage, T. D.; Rajapaksha, D. C.; Dananjaya, S. H. S.; Nikapitiya, Chamilani; Whang, Ilson; De Zoysa, Mahanama</div><div class="citationInfo"><span class="NLM_cas:title">Fish & Shellfish Immunology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">FSIMEP</span>;
        ISSN:<span class="NLM_cas:issn">1050-4648</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Streptococcus parauberis is a pathogenic gram-pos. bacterium that causes streptococcosis infection in fish.  Since S. parauberis is becoming resistant to multiple antibiotics, the development of alternatives, such as antimicrobial peptides, has gained great attention.  Octominin, derived from the defense protein of Octopus minor, showed a significant antimicrobial activity against multidrug resistance S. parauberis, with a min. inhibitory concn. (MIC) and a min. bactericidal concn. (MBC) of 50 and 100μg/mL, resp.  Furthermore, time-kill kinetics, agar diffusion, and bacterial viability assays confirmed the concn.-dependent antibacterial activity of Octominin against S. parauberis.  Field emission SEM anal. showed morphol. and ultra-structural changes in S. parauberis upon Octominin treatment.  Moreover, Octominin treatment demonstrated changes in membrane permeability, induced reactive oxygen species (ROS), and its binding ability to genomic DNA, suggesting its strong bactericidal activity with multiple modes of action.  We confirmed the inhibition of biofilm formation and the eradication of existing biofilms in a concn.-dependent manner.  Addnl., Octominin on S. parauberis at transcriptional level exhibited downregulation of membrane formation (pgsA and cds1), DNA repairing (recF), biofilm formation (pgaB and epsF) genes, while upregulation of ROS detoxification (sodA) and DNA protecting (ahpF) related genes.  An in vivo study confirmed a significantly (P < 0.05) higher relative percentage survival in Octominin-treated larval zebrafish exposed to S. parauberis (93.3%) compared to the control group (20.0%).  Collectively, our results confirm that Octominin could be a potential antibacterial and anti-biofilm agent against S. parauberis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA9JweWgCoAbVg90H21EOLACvtfcHk0ljT09H-oJQHsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXitlSisrc%253D&md5=f548c79f12e9c5a6217461d14cc88bc1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.fsi.2020.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fsi.2020.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DThulshan%2BJayathilaka%26aufirst%3DE.%2BH.%2BT.%26aulast%3DLiyanage%26aufirst%3DT.%2BD.%26aulast%3DRajapaksha%26aufirst%3DD.%2BC.%26aulast%3DDananjaya%26aufirst%3DS.%2BH.%2BS.%26aulast%3DNikapitiya%26aufirst%3DC.%26aulast%3DWhang%26aufirst%3DI.%26aulast%3DDe%2BZoysa%26aufirst%3DM.%26atitle%3DOctominin%253A%2520An%2520antibacterial%2520and%2520anti-biofilm%2520peptide%2520for%2520controlling%2520the%2520multidrug%2520resistance%2520and%2520pathogenic%2520streptococcus%2520parauberis%26jtitle%3DFish%2520Shellfish%2520Immunol.%26date%3D2021%26volume%3D110%26spage%3D23%26epage%3D34%26doi%3D10.1016%2Fj.fsi.2020.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Sousa, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assane, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos-Filho, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cilli, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jesus, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarski, F.</span></span> <span> </span><span class="NLM_article-title">Haematological, biochemical and immunological biomarkers, antibacterial activity, and survival in nile tilapia oreochromis niloticus after treatment using antimicrobial peptide LL-37 against streptococcus agalactiae</span>. <i>Aquaculture</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>533</i></span>, <span class="NLM_elocation-id">736181</span> <span class="refDoi"> DOI: 10.1016/j.aquaculture.2020.736181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.aquaculture.2020.736181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3MXmsVemug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2021&author=E.+L.+de%0ASousaauthor=I.+M.+Assaneauthor=N.+A.+Santos-Filhoauthor=E.+M.+Cilliauthor=R.+B.+de+Jesusauthor=F.+Pilarski&title=Haematological%2C+biochemical+and+immunological+biomarkers%2C+antibacterial+activity%2C+and+survival+in+nile+tilapia+oreochromis+niloticus+after+treatment+using+antimicrobial+peptide+LL-37+against+streptococcus+agalactiae&doi=10.1016%2Fj.aquaculture.2020.736181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Haematological, biochemical and immunological biomarkers, antibacterial activity, and survival in Nile tilapia Oreochromis niloticus after treatment using antimicrobial peptide LL-37 against Streptococcus agalactiae</span></div><div class="casAuthors">de Sousa, Elielma Lima; Assane, Inacio Mateus; Santos-Filho, Norival Alves; Cilli, Eduardo Maffud; de Jesus, Raphael Barbetta; Pilarski, Fabiana</div><div class="citationInfo"><span class="NLM_cas:title">Aquaculture</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">736181</span>CODEN:
                <span class="NLM_cas:coden">AQCLAL</span>;
        ISSN:<span class="NLM_cas:issn">0044-8486</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This study aimed to evaluate the antimicrobial activity and therapeutic effectiveness of peptide LL-37 against streptococcosis caused by Streptococcus agalactiae in Nile tilapia and to assess some of its potential adverse effects (in vivo haematol., biochem., and immunol. profiles) in the fish.  After in vitro cytotoxicity studies of LL-37 against Nile tilapia red blood cells (RBCs) and antimicrobial susceptibility testing (min. inhibitory concn. and min. bactericidal concn., MIC and MBC, resp.) against Aeromonas hydrophila and S. agalactiae, fish were divided into six exptl. groups.  G1 and G2 were challenged by i.p. injection with S. agalactiae and received a single oral dose of LL-37 (15 mg/kg bw) and florfenicol (FFC) (15 mg/kg bw), resp.  G3 was challenged with S. agalactiae and received a single i.p. dose of LL-37 (15 mg/kg bw).  The other groups were controls for infection (G4) and antimicrobial administration (G5 and G6).  LL-37 showed weak hemolytic activity against Nile tilapia RBCs (24.4% lysis at 625μg/mL) and strong in vitro antimicrobial activity against S. agalactiae (MIC and MBC = 31.25μg/mL).  No significant effects were obsd. in the haematol., biochem., and immunol. profiles of fish medicated with LL-37 or FFC, compared to the control fish.  Twenty days after the challenge, survival rates were 33.38% (G1), 38.51% (G2), 23.12% (G3), 25% (G4), and 100% (G5 and G6).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmkmE-ERNBwrVg90H21EOLACvtfcHk0ljT09H-oJQHsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXmsVemug%253D%253D&md5=72acdcce2cb1c998151a9cd333d34c21</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.aquaculture.2020.736181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aquaculture.2020.736181%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSousa%26aufirst%3DE.%2BL.%26aulast%3DAssane%26aufirst%3DI.%2BM.%26aulast%3DSantos-Filho%26aufirst%3DN.%2BA.%26aulast%3DCilli%26aufirst%3DE.%2BM.%26aulast%3Dde%2BJesus%26aufirst%3DR.%2BB.%26aulast%3DPilarski%26aufirst%3DF.%26atitle%3DHaematological%252C%2520biochemical%2520and%2520immunological%2520biomarkers%252C%2520antibacterial%2520activity%252C%2520and%2520survival%2520in%2520nile%2520tilapia%2520oreochromis%2520niloticus%2520after%2520treatment%2520using%2520antimicrobial%2520peptide%2520LL-37%2520against%2520streptococcus%2520agalactiae%26jtitle%3DAquaculture%26date%3D2021%26volume%3D533%26doi%3D10.1016%2Fj.aquaculture.2020.736181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Cross-strand interaction, central bending, and sequence pattern act as biomodulators of simplified beta-hairpin antimicrobial amphiphiles</span>. <i>Small</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>, <span class="NLM_elocation-id">2003899</span> <span class="refDoi"> DOI: 10.1002/smll.202003899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1002%2Fsmll.202003899" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&author=C.+Shaoauthor=Y.+Zhuauthor=Q.+Jianauthor=Z.+Laiauthor=P.+Tanauthor=G.+Liauthor=A.+Shan&title=Cross-strand+interaction%2C+central+bending%2C+and+sequence+pattern+act+as+biomodulators+of+simplified+beta-hairpin+antimicrobial+amphiphiles&doi=10.1002%2Fsmll.202003899"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fsmll.202003899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fsmll.202003899%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DJian%26aufirst%3DQ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DCross-strand%2520interaction%252C%2520central%2520bending%252C%2520and%2520sequence%2520pattern%2520act%2520as%2520biomodulators%2520of%2520simplified%2520beta-hairpin%2520antimicrobial%2520amphiphiles%26jtitle%3DSmall%26date%3D2020%26volume%3D17%26doi%3D10.1002%2Fsmll.202003899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides: promising alternatives in the post feeding antibiotic era</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">859</span>, <span class="refDoi"> DOI: 10.1002/med.21542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1002%2Fmed.21542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=30353555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=831-859&author=J.+Wangauthor=X.+Douauthor=J.+Songauthor=Y.+Lyuauthor=X.+Zhuauthor=L.+Xuauthor=W.+Liauthor=A.+Shan&title=Antimicrobial+peptides%3A+promising+alternatives+in+the+post+feeding+antibiotic+era&doi=10.1002%2Fmed.21542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial peptides: Promising alternatives in the post feeding antibiotic era</span></div><div class="casAuthors">Wang, Jiajun; Dou, Xiujing; Song, Jing; Lyu, Yinfeng; Zhu, Xin; Xu, Lin; Li, Weizhong; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">831-859</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Antimicrobial peptides (AMPs), crit. components of the innate immune system, are widely distributed throughout the animal and plant kingdoms.  They can protect against a broad array of infection-causing agents, such as bacteria, fungi, parasites, viruses, and tumor cells, and also exhibit immunomodulatory activity.  AMPs exert antimicrobial activities primarily through mechanisms involving membrane disruption, so they have a lower likelihood of inducing drug resistance.  Extensive studies on the structure-activity relationship have revealed that net charge, hydrophobicity, and amphipathicity are the most important physicochem. and structural determinants endowing AMPs with antimicrobial potency and cell selectivity.  This review summarizes the recent advances in AMPs development with respect to characteristics, structure-activity relationships, functions, antimicrobial mechanisms, expression regulation, and applications in food, medicine, and animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkakUqHPXXh7Vg90H21EOLACvtfcHk0lgmQb9CkRq3yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlGksbY%253D&md5=4ddae625caf29942522874d247a7b52c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fmed.21542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21542%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DLyu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DAntimicrobial%2520peptides%253A%2520promising%2520alternatives%2520in%2520the%2520post%2520feeding%2520antibiotic%2520era%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26volume%3D39%26spage%3D831%26epage%3D859%26doi%3D10.1002%2Fmed.21542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span> <span> </span><span class="NLM_article-title">Avian host defense cathelicidins: structure, expression, biological functions, and potential therapeutic applications</span>. <i>Poultry Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">6434</span>– <span class="NLM_lpage">6445</span>, <span class="refDoi"> DOI: 10.1016/j.psj.2020.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.psj.2020.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=33248558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtl2qs7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2020&pages=6434-6445&author=Y.+Wangauthor=M.+Wangauthor=A.+Shanauthor=X.+Feng&title=Avian+host+defense+cathelicidins%3A+structure%2C+expression%2C+biological+functions%2C+and+potential+therapeutic+applications&doi=10.1016%2Fj.psj.2020.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Avian host defense cathelicidins: structure, expression, biological functions, and potential therapeutic applications</span></div><div class="casAuthors">Wang, Yingjie; Wang, Min; Shan, Anshan; Feng, Xingjun</div><div class="citationInfo"><span class="NLM_cas:title">Poultry Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6434-6445</span>CODEN:
                <span class="NLM_cas:coden">POSCAL</span>;
        ISSN:<span class="NLM_cas:issn">0032-5791</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Host defense peptides (HDP) are multifunctional effectors of the innate immune system, which has antimicrobial and pleiotropic immunomodulatory functions.  Although there is a very sophisticated superposition of adaptive immune systems in vertebrates, this system is still essential.  As an important family of HDP, cathelicidins are also known for their broad-spectrum antibacterial activity against bacteria, fungi, and enveloped viruses.  It has been found in humans and other species, including cattle, pigs, sheep, goats, chickens, rabbits, and some kind of fish.  Among them, cathelicidins in birds were described for the first time in 2005.  This review focuses on the structure, biol. activities, expression, and regulation of avian cathelicidin, esp. main effects of host defense cathelicidin on potential therapeutic applications.  According to the results obtained both in vitro and in vivo, good perspectives have been opened for cathelicidin.  Nevertheless, further studies are needed to better characterize the mechanisms of action underlying the beneficial effects of cathelicidin as novel therapeutic alternatives to antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWbAO8_7giKrVg90H21EOLACvtfcHk0lgmQb9CkRq3yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtl2qs7bN&md5=b6fea1dfff833898a2c3784690952795</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.psj.2020.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.psj.2020.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DX.%26atitle%3DAvian%2520host%2520defense%2520cathelicidins%253A%2520structure%252C%2520expression%252C%2520biological%2520functions%252C%2520and%2520potential%2520therapeutic%2520applications%26jtitle%3DPoultry%2520Sci.%26date%3D2020%26volume%3D99%26spage%3D6434%26epage%3D6445%26doi%3D10.1016%2Fj.psj.2020.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ranade, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramalingam, R.</span></span> <span> </span><span class="NLM_article-title">A review on bioactive porcine peptide, protegrin-1</span>. <i>Int. J. Pept. Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1493</span>– <span class="NLM_lpage">1501</span>, <span class="refDoi"> DOI: 10.1007/s10989-019-09955-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1007%2Fs10989-019-09955-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFSjsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=1493-1501&author=S.+S.+Ranadeauthor=R.+Ramalingam&title=A+review+on+bioactive+porcine+peptide%2C+protegrin-1&doi=10.1007%2Fs10989-019-09955-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Review on Bioactive Porcine Peptide, Protegrin-1</span></div><div class="casAuthors">Ranade, Shruti Sunil; Ramalingam, Rajasekaran</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide Research and Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1493-1501</span>CODEN:
                <span class="NLM_cas:coden">IJPRFC</span>;
        ISSN:<span class="NLM_cas:issn">1573-3149</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Multi drug resistance is a major problem of the twenty first century.  In order to combat this issue, there is an urgent need in the pharmaceutical industry, for novel therapeutic agents.  Antimicrobial peptides such as protegrins which exhibit non-specific membranolytic action can be viewed as probable therapeutic agents and replace conventional antibiotics.  Protegrin-1 (PG-1) is a peptide isolated from porcine leukocytes.  Its primary role is its antimicrobial activity against a broad-spectrum of gram-pos. as well as gram-neg. bacteria and fungi.  Its antagonistic activity can be accounted by its pore formation mechanism in microbial membranes.  In addn., PG-1 has multiple roles viz., anticancer and antiviral activity, immunomodulatory functions and numerous applications which increase its suitability as a potential therapeutic agent.  This review paper presents a comprehensive overview of biol. roles, lytic mechanism of action and applications of PG-1, thus providing a thorough understanding of this β-sheet peptide, which structurally resembles defensin peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_qXbg9tfiHLVg90H21EOLACvtfcHk0lhagHmDVANPFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFSjsrbN&md5=f33a4216483c000a44f6b0a24770dbe8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2Fs10989-019-09955-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10989-019-09955-8%26sid%3Dliteratum%253Aachs%26aulast%3DRanade%26aufirst%3DS.%2BS.%26aulast%3DRamalingam%26aufirst%3DR.%26atitle%3DA%2520review%2520on%2520bioactive%2520porcine%2520peptide%252C%2520protegrin-1%26jtitle%3DInt.%2520J.%2520Pept.%2520Res.%2520Ther.%26date%3D2020%26volume%3D26%26spage%3D1493%26epage%3D1501%26doi%3D10.1007%2Fs10989-019-09955-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottler, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Salud Bea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelburne, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramamoorthy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, E. N. G.</span></span> <span> </span><span class="NLM_article-title">Using fluorous amino acids to probe the effects of changing hydrophobicity on the physical and biological properties of the beta-hairpin antimicrobial peptide protegrin-1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">9243</span>– <span class="NLM_lpage">9250</span>, <span class="refDoi"> DOI: 10.1021/bi801045n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi801045n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpslOhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=9243-9250&author=L.+M.+Gottlerauthor=R.+de+la+Salud+Beaauthor=C.+E.+Shelburneauthor=A.+Ramamoorthyauthor=E.+N.+G.+Marsh&title=Using+fluorous+amino+acids+to+probe+the+effects+of+changing+hydrophobicity+on+the+physical+and+biological+properties+of+the+beta-hairpin+antimicrobial+peptide+protegrin-1&doi=10.1021%2Fbi801045n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Using Fluorous Amino Acids To Probe the Effects of Changing Hydrophobicity on the Physical and Biological Properties of the β-Hairpin Antimicrobial Peptide Protegrin-1</span></div><div class="casAuthors">Gottler, Lindsey M.; de la Salud Bea, Roberto; Shelburne, Charles E.; Ramamoorthy, Ayyalusamy; Marsh, E. Neil G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">9243-9250</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protegrins are potent members of the β-hairpin-forming class of antimicrobial peptides.  Key to their antimicrobial activity is their assembly into oligomeric structures upon binding to the bacterial membrane.  To examine the relationship between the physicochem. properties of the peptide and its biol. activity, we have synthesized variants of protegrin-1 in which key residues in the hydrophobic core, valine-14 and -16, are changed to leucine and to the extensively fluorinated analog hexafluoroleucine.  These substitutions have the effect of making the peptide progressively more hydrophobic while minimally perturbing the secondary structure.  The leucine-contg. peptide was significantly more active than wild-type protegrin against several common pathogenic bacterial strains, whereas the hexafluoroleucine-substituted peptide, in contrast, showed significantly diminished activity against several bacterial strains.  Isothermal titrn. calorimetry measurements revealed significant changes in the interaction of the peptides binding to small unilamelar vesicles that mimic the lipid compn. of the bacterial membrane.  The binding isotherms for wild-type and leucine-substituted protegrins indicate that electrostatic interactions dominate the membrane-peptide interaction, whereas the isotherm for the hexafluoroleucine-substituted protegrin suggests a diminished electrostatic component to binding.  Notably both of these substitutions appear to alter the stoichiometry of the lipid-peptide interaction, suggesting that these substitutions may stabilize oligomerized forms of protegrin that are postulated to be intermediates in the assembly of the β-barrel membrane pore structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMIUlmWDuc6bVg90H21EOLACvtfcHk0lhagHmDVANPFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpslOhu7s%253D&md5=bd0c2c90f5c3aa620ad3e23956c75945</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbi801045n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi801045n%26sid%3Dliteratum%253Aachs%26aulast%3DGottler%26aufirst%3DL.%2BM.%26aulast%3Dde%2Bla%2BSalud%2BBea%26aufirst%3DR.%26aulast%3DShelburne%26aufirst%3DC.%2BE.%26aulast%3DRamamoorthy%26aufirst%3DA.%26aulast%3DMarsh%26aufirst%3DE.%2BN.%2BG.%26atitle%3DUsing%2520fluorous%2520amino%2520acids%2520to%2520probe%2520the%2520effects%2520of%2520changing%2520hydrophobicity%2520on%2520the%2520physical%2520and%2520biological%2520properties%2520of%2520the%2520beta-hairpin%2520antimicrobial%2520peptide%2520protegrin-1%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D9243%26epage%3D9250%26doi%3D10.1021%2Fbi801045n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span> <span> </span><span class="NLM_article-title">De novo generation of short antimicrobial peptides with enhanced stability and cell specificity</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1093/jac/dkt322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1093%2Fjac%2Fdkt322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=23946320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegtrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=121-132&author=H.+Kimauthor=J.+H.+Jangauthor=S.+C.+Kimauthor=J.+H.+Cho&title=De+novo+generation+of+short+antimicrobial+peptides+with+enhanced+stability+and+cell+specificity&doi=10.1093%2Fjac%2Fdkt322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">De novo generation of short antimicrobial peptides with enhanced stability and cell specificity</span></div><div class="casAuthors">Kim, Hyun; Jang, Ju Hye; Kim, Sun Chang; Cho, Ju Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-132</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives Though antimicrobial peptides (AMPs) show great potential as novel antibiotics, therapeutic applications are hindered by their low stability, toxicity and high manufg. cost.  Various chem. modification strategies are employed to overcome these problems.  However, chem. modifications often significantly increase the manufg. cost of AMPs with only limited pharmacokinetic advantages.  Therefore, we developed AMPs with enhanced stability and cell specificity that can be economically produced.  Methods Peptides were designed by systematic amino acid arrangement without the incorporation of both non-natural amino acids and peptidomimetics.  Antimicrobial activities were measured against Gram-pos. bacteria, Gram-neg. bacteria and fungi by MIC evaluation under both std. and physiol. relevant conditions.  Cytotoxicity towards human cells was evaluated to verify selective antimicrobial activity.  The antibacterial mechanism of the peptides was elucidated by β-galactosidase assay and SEM.  Results Among the designed peptides, GNU6 and GNU7 showed potent antimicrobial activity against bacteria and fungi and maintained their activity in the presence of 150 mM NaCl and 10% serum.  These peptides were not digested by exposure to trypsin, chymotrypsin and aureolysin for up to 12 h and showed potent antimicrobial activity against methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci.  Moreover, they did not affect the viability of erythrocytes, keratinocytes and fibroblasts up to 128 mg/L.  A membrane permeabilization assay and SEM anal. showed that GNU6 and GNU7 compromised membrane integrity and function in microorganisms.  Conclusions This study suggests that GNU6 and GNU7 might overcome serious problems that currently prevent the clin. use of AMPs and be developed as novel antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5zPEe3xO5SrVg90H21EOLACvtfcHk0lhSfzx-hk5MiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegtrbN&md5=5238110fd2cdb45a7915c0c81b3cdfbc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkt322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkt322%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.%26aulast%3DJang%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DS.%2BC.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26atitle%3DDe%2520novo%2520generation%2520of%2520short%2520antimicrobial%2520peptides%2520with%2520enhanced%2520stability%2520and%2520cell%2520specificity%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D121%26epage%3D132%26doi%3D10.1093%2Fjac%2Fdkt322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Legowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukajtis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sztabkowska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizeria, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wysocka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolka, K.</span></span> <span> </span><span class="NLM_article-title">Introduction of pro and its analogues in the conserved p1’ position of trypsin inhibitor SFTI-1 retains its inhibitory activity</span>. <i>Protein Pept. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1158</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.2174/092986611797201002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.2174%2F092986611797201002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=21605056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1158-1167&author=A.+Legowskaauthor=D.+Debowskiauthor=R.+Lukajtisauthor=E.+Sztabkowskaauthor=A.+Mizeriaauthor=K.+Brzozowskiauthor=M.+Wysockaauthor=A.+Lesnerauthor=K.+Rolka&title=Introduction+of+pro+and+its+analogues+in+the+conserved+p1%E2%80%99+position+of+trypsin+inhibitor+SFTI-1+retains+its+inhibitory+activity&doi=10.2174%2F092986611797201002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction of Pro and its analogues in the conserved P1' position of trypsin inhibitor SFTI-1 retains its inhibitory activity</span></div><div class="casAuthors">Legowska, Anna; Debowski, Dawid; Lukajtis, Rafal; Sztabkowska, Emilia; Mizeria, Aneta; Brzozowski, Krzysztof; Wysocka, Magdalena; Lesner, Adam; Rolka, Krzysztof</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Peptide Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1158-1167</span>CODEN:
                <span class="NLM_cas:coden">PPELEN</span>;
        ISSN:<span class="NLM_cas:issn">0929-8665</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A no. of monocyclic SFTI-1 analogs modified in the conserved inhibitor P1' position by Pro, its L-hydroxyproline (Hyp) deriv. as well as mimetics with different ring size were synthesized by the solid-phase method.  Replacement of Ser6 by Pro, Hyp, and a four-member ring, L-azetidine-2-carboxylic acid (Aze), retained trypsin or chymotrypsin inhibitory activity.  The detd. assocn. equil. consts. of these analogs with a cognate enzyme were about two orders of magnitude lower than those obtained for ones with conserved Ser6.  In all analogs, with the exception of one, [Phe5,Aze6]SFTI-1, the P1-P1' reactive site remained intact.  The results provide first evidence that the conserved Ser in the P1' position of Bowman-Birk inhibitors can be successfully replaced by an amino acid with a secondary amine group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL1VFIjxzcIbVg90H21EOLACvtfcHk0lhSfzx-hk5MiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrO&md5=502cd50f8fa894e1251ec675eccd55c9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F092986611797201002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986611797201002%26sid%3Dliteratum%253Aachs%26aulast%3DLegowska%26aufirst%3DA.%26aulast%3DDebowski%26aufirst%3DD.%26aulast%3DLukajtis%26aufirst%3DR.%26aulast%3DSztabkowska%26aufirst%3DE.%26aulast%3DMizeria%26aufirst%3DA.%26aulast%3DBrzozowski%26aufirst%3DK.%26aulast%3DWysocka%26aufirst%3DM.%26aulast%3DLesner%26aufirst%3DA.%26aulast%3DRolka%26aufirst%3DK.%26atitle%3DIntroduction%2520of%2520pro%2520and%2520its%2520analogues%2520in%2520the%2520conserved%2520p1%25E2%2580%2599%2520position%2520of%2520trypsin%2520inhibitor%2520SFTI-1%2520retains%2520its%2520inhibitory%2520activity%26jtitle%3DProtein%2520Pept.%2520Lett.%26date%3D2011%26volume%3D18%26spage%3D1158%26epage%3D1167%26doi%3D10.2174%2F092986611797201002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">D-amino acid substitution enhances the stability of antimicrobial peptide polybia-CP</span>. <i>Acta Biochim. Biophys. Sin. (Shanghai)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">925</span>, <span class="refDoi"> DOI: 10.1093/abbs/gmx091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1093%2Fabbs%2Fgmx091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=28981608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252Fls1Kntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=916-925&author=F.+Jiaauthor=J.+Wangauthor=J.+Pengauthor=P.+Zhaoauthor=Z.+Kongauthor=K.+Wangauthor=W.+Yanauthor=R.+Wang&title=D-amino+acid+substitution+enhances+the+stability+of+antimicrobial+peptide+polybia-CP&doi=10.1093%2Fabbs%2Fgmx091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">D-amino acid substitution enhances the stability of antimicrobial peptide polybia-CP</span></div><div class="casAuthors">Jia Fengjing; Wang Jiayi; Peng Jinxiu; Zhao Ping; Wang Kairong; Yan Wenjin; Wang Rui; Kong Ziqing</div><div class="citationInfo"><span class="NLM_cas:title">Acta biochimica et biophysica Sinica</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">916-925</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">With the increasing emergence of resistant microbes toward conventional antimicrobial agents, there is an urgent need for the development of antimicrobial agents with novel action mode.  Antimicrobial peptides (AMPs) are believed to be one kind of ideal alternatives.  However, AMPs can be easily degraded by protease, which limited their therapeutic use.  In the present study, D-amino acid substitution strategy was employed to enhance the stability of polybia-CP.  We investigated the stability of peptides against the degradation of trypsin and chymotrypsin by determining the antimicrobial activity or determining the HPLC profile of peptides after incubation with proteases.  Our results showed that both the all D-amino acid derivative (D-CP) and partial D-lysine substitution derivative (D-lys-CP) have an improved stability against trypsin and chymotrypsin.  Although D-CP takes left-hand α-helical conformation and D-lys-CP loses some α-helical content, both of the D-amino acid-substituted derivatives maintain their parental peptides' membrane active action mode.  In addition, D-lys-CP showed a slight weaker antimicrobial activity than polybia-CP, but the hemolytic activity decreased greatly.  These results suggest that D-CP and D-lys-CP can offer strategy to improve the property of AMPs and may be leading compounds for the development of novel antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6borawicmlUGy1nwu7K2mfW6udTcc2eY7tYLa07nSnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252Fls1Kntw%253D%253D&md5=9cadf0a1a38f230c8b1dc9d521d6dfb8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmx091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmx091%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DKong%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DD-amino%2520acid%2520substitution%2520enhances%2520the%2520stability%2520of%2520antimicrobial%2520peptide%2520polybia-CP%26jtitle%3DActa%2520Biochim.%2520Biophys.%2520Sin.%2520%2528Shanghai%2529%26date%3D2017%26volume%3D49%26spage%3D916%26epage%3D925%26doi%3D10.1093%2Fabbs%2Fgmx091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorris, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röckendorf, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fránek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, A.</span></span> <span> </span><span class="NLM_article-title">Rapid profiling of peptide stability in proteolytic environments</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">1580</span>– <span class="NLM_lpage">1586</span>, <span class="refDoi"> DOI: 10.1021/ac802324f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac802324f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotlGltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=1580-1586&author=H.+H.+Gorrisauthor=S.+Badeauthor=N.+R%C3%B6ckendorfauthor=E.+Albersauthor=M.+A.+Schmidtauthor=M.+Fr%C3%A1nekauthor=A.+Frey&title=Rapid+profiling+of+peptide+stability+in+proteolytic+environments&doi=10.1021%2Fac802324f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Profiling of Peptide Stability in Proteolytic Environments</span></div><div class="casAuthors">Gorris, Hans H.; Bade, Steffen; Rockendorf, Niels; Albers, Eike; Schmidt, M. Alexander; Franek, Milan; Frey, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1580-1586</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The notorious degrdn. susceptibility of peptides is a major obstacle to their use as medicinal drugs.  Assays with which the stability of peptides in complex proteolytic environments can be detd. are thus indispensable for peptide drug development.  Herein, we describe a new peptide proteolysis assay that meets that demand.  It unites the high-throughput capacity of heterogeneous with the well-defined kinetic characteristics of homogeneous assay formats and operates on the cleavage-caused loss of a detection handle.  We have confirmed throughput screening the assay's accuracy with well-defined model interactions and proved its high versatility and robustness with a representative application where we detd. the half-lives of 375 different peptides in a crude intestinal protease prepn.  With this reliable, reproducible, and efficient assay the enzyme kinetics of any peptide-protease interaction is accessible even for complex protease solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPi_HqtCxog7Vg90H21EOLACvtfcHk0ljaFj9lgD7szw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotlGltQ%253D%253D&md5=54bf211252901e687a7c01b1f201ac2d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fac802324f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac802324f%26sid%3Dliteratum%253Aachs%26aulast%3DGorris%26aufirst%3DH.%2BH.%26aulast%3DBade%26aufirst%3DS.%26aulast%3DR%25C3%25B6ckendorf%26aufirst%3DN.%26aulast%3DAlbers%26aufirst%3DE.%26aulast%3DSchmidt%26aufirst%3DM.%2BA.%26aulast%3DFr%25C3%25A1nek%26aufirst%3DM.%26aulast%3DFrey%26aufirst%3DA.%26atitle%3DRapid%2520profiling%2520of%2520peptide%2520stability%2520in%2520proteolytic%2520environments%26jtitle%3DAnal.%2520Chem.%26date%3D2009%26volume%3D81%26spage%3D1580%26epage%3D1586%26doi%3D10.1021%2Fac802324f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial potency and selectivity of simplified symmetric-end peptides</span>. <i>Biomaterials</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8039</span>, <span class="refDoi"> DOI: 10.1016/j.biomaterials.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.biomaterials.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=24952979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWqs7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=8028-8039&author=N.+Dongauthor=X.+Zhuauthor=S.+Chouauthor=A.+Shanauthor=W.+Liauthor=J.+Jiang&title=Antimicrobial+potency+and+selectivity+of+simplified+symmetric-end+peptides&doi=10.1016%2Fj.biomaterials.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial potency and selectivity of simplified symmetric-end peptides</span></div><div class="casAuthors">Dong, Na; Zhu, Xin; Chou, Shuli; Shan, Anshan; Li, Weizhong; Jiang, Junguang</div><div class="citationInfo"><span class="NLM_cas:title">Biomaterials</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8028-8039</span>CODEN:
                <span class="NLM_cas:coden">BIMADU</span>;
        ISSN:<span class="NLM_cas:issn">0142-9612</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because antimicrobial peptides (AMPs) are potentially useful for the treatment of multidrug-resistant infections, more attention is being paid to the structural modification and structure-function relationship of both naturally occurring and synthetic AMPs.  Previous studies indicated that Protegrin-1 (PG-1), isolated from porcine leukocytes, exhibited considerable antimicrobial activity and cytotoxicity.  The β-turn of PG-1 floated on the surface of bacterial membrane, while its β-strand inserted into the bacterial membrane and formed pores that were dedicated to producing cytotoxicity.  For reducing cytotoxicity and improving cells selectivity, we designed a series of simplified sym.-end peptides by combining the β-turn of PG-1 with simple amino acid repeat sequences.  The sequence of designed sym.-end peptides is (XR)nH(RX)n, (n = 1,2; X represents I, F, W and P; H represents CRRRFC).  The sym.-end peptides displayed antimicrobial activity against both gram-pos. and gram-neg. bacteria.  In particular, (XR)2H(RX)2 (X here is I, F and W) showed greater antimicrobial potency than PG-1.  Hemolysis activity and cytotoxicity, detected by using human red blood cells (RBCs) and human embryonic lung fibroblasts MRC-5 cells, were observably lower than the native peptide PG-1. (IR)2H(RI)2 (IR2), folded into β-sheet structures, displayed the highest therapeutic index, suggesting its great cell selectivity.  The fluorescence spectroscopy, flow cytometry, and electron microscopy observation indicated that IR2 exhibited great membrane penetration potential by inducing membrane blebbing, disruption and lysis.  Collectively, generating sym.-end β-sheet peptides is a promising strategy for designing effective AMPs with great antimicrobial activities and cell selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZGe2XjNdNN7Vg90H21EOLACvtfcHk0ljaFj9lgD7szw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWqs7nM&md5=58acfdd2b050d20c77f42dc2d34c742f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.biomaterials.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biomaterials.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DChou%26aufirst%3DS.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DJ.%26atitle%3DAntimicrobial%2520potency%2520and%2520selectivity%2520of%2520simplified%2520symmetric-end%2520peptides%26jtitle%3DBiomaterials%26date%3D2014%26volume%3D35%26spage%3D8028%26epage%3D8039%26doi%3D10.1016%2Fj.biomaterials.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">Design of hybrid beta-hairpin peptides with enhanced cell specificity and potent anti-inflammatory activity</span>. <i>Biomaterials</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1016/j.biomaterials.2012.09.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.biomaterials.2012.09.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=23046754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKksL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=237-250&author=Y.+F.+Liuauthor=X.+Xiaauthor=L.+Xuauthor=Y.+Z.+Wang&title=Design+of+hybrid+beta-hairpin+peptides+with+enhanced+cell+specificity+and+potent+anti-inflammatory+activity&doi=10.1016%2Fj.biomaterials.2012.09.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design of hybrid β-hairpin peptides with enhanced cell specificity and potent anti-inflammatory activity</span></div><div class="casAuthors">Liu, Yi Fan; Xia, Xi; Xu, Liang; Wang, Yi Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Biomaterials</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">237-250</span>CODEN:
                <span class="NLM_cas:coden">BIMADU</span>;
        ISSN:<span class="NLM_cas:issn">0142-9612</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs) have attracted considerable attention for their broad-spectrum antimicrobial activity and reduced tendency to cause bacterial resistance.  Emerging concerns over the host cytotoxicity of AMPs, however, may ultimately compromise their development as pharmaceuticals.  In order to optimize AMPs with potent cell specificity and anti-inflammatory activity, we designed β-hairpin hybrid peptides based upon progetrin-1, bovine lactoferricin and cecropin A.  The synthetic hybrid peptides LB-PG and CA-PG demonstrated high selectivity over a wide range of microbes from Gram-pos. and Gram-neg. bacteria in porcine red blood cells.  SEM (SEM) and transmission electron microscopy (TEM) show that these peptides kill microbial cells by penetrating the cell membrane and damaging the membrane envelope.  Gel retardation demonstrates that the peptides have a high affinity for DNA, indicating an addnl. possible intracellular bactericidal mechanism.  Moreover, the hybrid peptides inhibit the expression of LPS-induced proinflammatory cytokines and chemokines, such as tumor necrosis factor-α (TNF-α), inducible nitric oxide synthase (iNOS), macrophage inflammatory protein-1α (MIP-1α) and monocyte chemoattractant protein 1(MCP-1), following LPS stimulation in RAW264.7 cells.  Our results indicate that these hybrid peptides have considerable potential for future development as antimicrobial and anti-inflammatory agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRe2RvSF4SjbVg90H21EOLACvtfcHk0ljaFj9lgD7szw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKksL7J&md5=25edbb8639a1a96dd35e013a4fb961fa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.biomaterials.2012.09.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biomaterials.2012.09.032%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%2BF.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%2BZ.%26atitle%3DDesign%2520of%2520hybrid%2520beta-hairpin%2520peptides%2520with%2520enhanced%2520cell%2520specificity%2520and%2520potent%2520anti-inflammatory%2520activity%26jtitle%3DBiomaterials%26date%3D2013%26volume%3D34%26spage%3D237%26epage%3D250%26doi%3D10.1016%2Fj.biomaterials.2012.09.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides with protease stability: progress and perspective</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2047</span>– <span class="NLM_lpage">2050</span>, <span class="refDoi"> DOI: 10.4155/fmc-2019-0167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.4155%2Ffmc-2019-0167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=31538527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=2047-2050&author=C.+Shaoauthor=Y.+Zhuauthor=Z.+Laiauthor=P.+Tanauthor=A.+Shan&title=Antimicrobial+peptides+with+protease+stability%3A+progress+and+perspective&doi=10.4155%2Ffmc-2019-0167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial peptides with protease stability: progress and perspective</span></div><div class="casAuthors">Shao, Changxuan; Zhu, Yongjie; Lai, Zhenheng; Tan, Peng; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2047-2050</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Antimicrobial peptides have attracted decades of attention as potential candidates for addressing multidrug-resistant infections; however, their dismal record of immature stability in clin. trials suggests a likely frustrating ending.  A revolution of novel antimicrobial peptides against protease degrdn. is already on the road. The drug delivery routes of i.m. or s.c. injections may not require too much stability of the peptide itself, but i.v. or oral administration will certainly expose these peptides to the ordeal of the proteases in serum, digestive tract or even the proteases secreted by the bacteria. Thus, with the deepening of research on AMPs, more attention has been focused on their lability to proteolytic degrdn. and the prodn.  Cost closely related to this. Therefore, the enzyme stability study of antimicrobial peptides is a long and arduous process, just like the initial exploration of the structural and functional relationship many years ago. With the advances of premature stability, the peptide-based therapeutics will likely usher in renewed innovations in pharmaceuticals, healthcare nutraceutical, food preservative, cosmetic or animal husbandry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSBFJ1FEOufLVg90H21EOLACvtfcHk0lgtNpAB0frStw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7bF&md5=f987c4538cd723eb5d1722caa84f2f59</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2019-0167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2019-0167%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DAntimicrobial%2520peptides%2520with%2520protease%2520stability%253A%2520progress%2520and%2520perspective%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D2047%26epage%3D2050%26doi%3D10.4155%2Ffmc-2019-0167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augusto, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felício, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, N. C.</span></span> <span> </span><span class="NLM_article-title">Designing improved active peptides for therapeutic approaches against infectious diseases</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2018.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.biotechadv.2018.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=29330093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGhu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=415-429&author=B.+Gomesauthor=M.+T.+Augustoauthor=M.+R.+Fel%C3%ADcioauthor=A.+Hollmannauthor=O.+L.+Francoauthor=S.+Gon%C3%A7alvesauthor=N.+C.+Santos&title=Designing+improved+active+peptides+for+therapeutic+approaches+against+infectious+diseases&doi=10.1016%2Fj.biotechadv.2018.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Designing improved active peptides for therapeutic approaches against infectious diseases</span></div><div class="casAuthors">Gomes, Barbara; Augusto, Marcelo T.; Felicio, Mario R.; Hollmann, Axel; Franco, Octavio L.; Goncalves, Sonia; Santos, Nuno C.</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">415-429</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Infectious diseases are one of the main causes of human morbidity and mortality.  In the last few decades, pathogenic microorganisms' resistance to conventional drugs has been increasing, and it is now pinpointed as a major worldwide health concern.  The need to search for new therapeutic options, as well as improved treatment outcomes, has therefore increased significantly, with biol. active peptides representing a new alternative.  A substantial research effort is being dedicated towards their development, esp. due to improved biocompatibility and target selectivity.  However, the inherent limitations of peptide drugs are restricting their application.  In this review, we summarize the current status of peptide drug development, focusing on antiviral and antimicrobial peptide activities, highlighting the design improvements needed, and those already being used, to overcome the drawbacks of the therapeutic application of biol. active peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIlcI4labulbVg90H21EOLACvtfcHk0lgtNpAB0frStw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGhu7c%253D&md5=98b3e01c6f8dbef08f30767ea2eb1865</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2018.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2018.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DGomes%26aufirst%3DB.%26aulast%3DAugusto%26aufirst%3DM.%2BT.%26aulast%3DFel%25C3%25ADcio%26aufirst%3DM.%2BR.%26aulast%3DHollmann%26aufirst%3DA.%26aulast%3DFranco%26aufirst%3DO.%2BL.%26aulast%3DGon%25C3%25A7alves%26aufirst%3DS.%26aulast%3DSantos%26aufirst%3DN.%2BC.%26atitle%3DDesigning%2520improved%2520active%2520peptides%2520for%2520therapeutic%2520approaches%2520against%2520infectious%2520diseases%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2018%26volume%3D36%26spage%3D415%26epage%3D429%26doi%3D10.1016%2Fj.biotechadv.2018.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avrahami, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shai, Y.</span></span> <span> </span><span class="NLM_article-title">Conjugation of a magainin analogue with lipophilic acids controls hydrophobicity, solution assembly, and cell selectivity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2254</span>– <span class="NLM_lpage">2263</span>, <span class="refDoi"> DOI: 10.1021/bi011549t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi011549t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD38Xnt1GlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2254-2263&author=D.+Avrahamiauthor=Y.+Shai&title=Conjugation+of+a+magainin+analogue+with+lipophilic+acids+controls+hydrophobicity%2C+solution+assembly%2C+and+cell+selectivity&doi=10.1021%2Fbi011549t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conjugation of a magainin analogue with lipophilic acids controls hydrophobicity, solution assembly, and cell selectivity</span></div><div class="casAuthors">Avrahami, Dorit; Shai, Yechiel</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2254-2263</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Our basic understanding of how to combat fungal infections has not kept pace with the recent sharp rise in life-threatening cases found, particularly among immuno-compromised individuals.  Current investigations for new potential antifungal agents have focused on antimicrobial peptides, which are used as a cell-free defense mechanism in all organisms.  Unfortunately, despite their high antibacterial activity, most of them are not active toward fungi, the reason of which is not clear.  Here, we present a new approach to modify an antibacterial peptide, a magainin analog, to display antifungal activity by its conjugation with lipophilic acids.  This approach has the advantage of producing well-defined changes in hydrophobicity, secondary structure, and self-assocn.  These modifications were characterized in soln. at physiol. concns. using CD spectroscopy, tryptophan fluorescence, and anal. ultracentrifugation.  In order of increasing hydrophobicity, the attachment to the magainin-2 analog of (i) heptanoic acid results in a monomeric, unordered structure, (ii) undecanoic acid yields concn.-dependent oligomers of α-helixes, and (iii) palmitic acid yields concn.-independent α-helical monomers, a novel lipopeptide structure which is resistant to proteolytic digestion.  Membrane-lipopeptide interactions and the membrane-bound structures were studied using fluorescence and ATR-FTIR in PC/PE/PI/ergosterol (5/2.5/2.5/1, wt./wt.) SUV, which constitute the major components of Candida albicans bilayers.  A direct correlation was found between oligomerization of the lipopeptides in soln. and potent antifungal activity.  These results provide insight to a new approach of modulating hydrophobicity and self-assembly of antimicrobial peptides in soln., without altering the sequence of the peptide chain.  These studies also provide a general means of developing a new group of lipopeptide candidates as therapeutic agents against fungal infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoajv-S8sGnybVg90H21EOLACvtfcHk0lgtNpAB0frStw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xnt1GlsA%253D%253D&md5=5281b939bbe93ddd9f62bf5fe7302c00</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fbi011549t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011549t%26sid%3Dliteratum%253Aachs%26aulast%3DAvrahami%26aufirst%3DD.%26aulast%3DShai%26aufirst%3DY.%26atitle%3DConjugation%2520of%2520a%2520magainin%2520analogue%2520with%2520lipophilic%2520acids%2520controls%2520hydrophobicity%252C%2520solution%2520assembly%252C%2520and%2520cell%2520selectivity%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D2254%26epage%3D2263%26doi%3D10.1021%2Fbi011549t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avrahami, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shai, Y.</span></span> <span> </span><span class="NLM_article-title">A new group of antifungal and antibacterial lipopeptides derived from non-membrane active peptides conjugated to palmitic acid</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">12277</span>– <span class="NLM_lpage">12285</span>, <span class="refDoi"> DOI: 10.1074/jbc.M312260200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1074%2Fjbc.M312260200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=14709550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitl2gsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=12277-12285&author=D.+Avrahamiauthor=Y.+Shai&title=A+new+group+of+antifungal+and+antibacterial+lipopeptides+derived+from+non-membrane+active+peptides+conjugated+to+palmitic+acid&doi=10.1074%2Fjbc.M312260200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A new group of antifungal and antibacterial lipopeptides derived from non-membrane active peptides conjugated to palmitic acid</span></div><div class="casAuthors">Avrahami, Dorit; Shai, Yechiel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">12277-12285</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We report on the synthesis, biol. function, and a plausible mode of action of a new group of lipopeptides with potent antifungal and antibacterial activities.  These lipopeptides are derived from pos. charged peptides contg. D- and L-amino acids (diastereomers) that are palmitoylated (PA) at their N terminus.  The peptides investigated have the sequence K4X7W, where X designates Gly, Ala, Val, or Leu (designated D-X peptides).  The data revealed that PA-D-G and PA-D-A gained potent antibacterial and antifungal activity despite the fact that both parental peptides were completely devoid of any activity toward microorganisms and model phospholipid membranes.  In contrast, PA-D-L lost the potent antibacterial activity of the parental peptide but gained and preserved partial antifungal activity.  Interestingly, both D-V and its palmitoylated analog were inactive toward bacteria, and only the palmitoylated peptide was highly potent toward yeast.  Both PA-D-L and PA-D-V lipopeptides were also endowed with hemolytic activity.  Mode of action studies were performed by using tryptophan fluorescence and attenuated total reflectance Fourier transform IR and CD spectroscopy as well as transmembrane depolarization assays with bacteria and fungi.  The data suggest that the lipopeptides act by increasing the permeability of the cell membrane and that differences in their potency and target specificity are the result of differences in their oligomeric state and ability to dissoc. and insert into the cytoplasmic membrane.  These results provide insight regarding a new approach of modulating hydrophobicity and the self-assembly of nonmembrane interacting peptides in order to endow them with both antibacterial and antifungal activities urgently needed to combat bacterial and fungal infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAv3oC-E2ZFLVg90H21EOLACvtfcHk0lh9Epdn79OLpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitl2gsr0%253D&md5=9b8a883fa41e416addd7de08b5d6ebdc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M312260200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M312260200%26sid%3Dliteratum%253Aachs%26aulast%3DAvrahami%26aufirst%3DD.%26aulast%3DShai%26aufirst%3DY.%26atitle%3DA%2520new%2520group%2520of%2520antifungal%2520and%2520antibacterial%2520lipopeptides%2520derived%2520from%2520non-membrane%2520active%2520peptides%2520conjugated%2520to%2520palmitic%2520acid%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D12277%26epage%3D12285%26doi%3D10.1074%2Fjbc.M312260200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, J. K.</span></span> <span> </span><span class="NLM_article-title">Water-soluble lipopolymer as a gene carrier to corpus cavernosum</span>. <i>Int. J. Impot. Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1038/sj.ijir.3901314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1038%2Fsj.ijir.3901314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=15800653" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslGnsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=326-334&author=M.+Leeauthor=J.+K.+Ryuauthor=S.+M.+Ohauthor=E.+Leeauthor=H.+Y.+Shinauthor=S.+U.+Songauthor=S.+W.+Kimauthor=J.+K.+Suh&title=Water-soluble+lipopolymer+as+a+gene+carrier+to+corpus+cavernosum&doi=10.1038%2Fsj.ijir.3901314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Water-soluble lipopolymer as a gene carrier to corpus cavernosum</span></div><div class="casAuthors">Lee, M.; Ryu, J-K.; Oh, S-M.; Lee, E.; Shin, H-Y.; Song, S. U.; Kim, S. W.; Suh, J-K.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Impotence Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">326-334</span>CODEN:
                <span class="NLM_cas:coden">IJIRFB</span>;
        ISSN:<span class="NLM_cas:issn">0955-9930</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Adenovirus or naked plasmid DNA (pDNA) has been used to deliver the therapeutic gene into corpus cavernosum.  However, the potential risks of viral vector and inefficiency of naked pDNA have limited their clin. application.  In this study, water-sol. lipopolymer (WSLP) was evaluated as a gene carrier to corpus cavernosum.  The WSLP/pDNA complex was transfected to smooth muscle cells in vitro.  WSLP had high transfection efficiency, which was comparable to poly(ethylenimine) (PEI).  In addn., WSLP had much less cytotoxicity than PEI, suggesting that WSLP is a safer carrier than PEI.  To evaluate the transfection efficiency to corpus cavernosum, the WSLP/pDNA complex was injected into the rat corpus cavernosum.  As a result, the WSLP/pDNA complex showed higher transfection efficiency than naked pDNA.  In addn., the gene expression was dependent upon the dose of the complex.  The results suggest that WSLP may be useful for gene therapy of erectile dysfunction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwSv4YaVQIKrVg90H21EOLACvtfcHk0lh9Epdn79OLpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslGnsL8%253D&md5=670da61b629e366784b1a5426e28fd62</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsj.ijir.3901314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.ijir.3901314%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%26aulast%3DRyu%26aufirst%3DJ.%2BK.%26aulast%3DOh%26aufirst%3DS.%2BM.%26aulast%3DLee%26aufirst%3DE.%26aulast%3DShin%26aufirst%3DH.%2BY.%26aulast%3DSong%26aufirst%3DS.%2BU.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DSuh%26aufirst%3DJ.%2BK.%26atitle%3DWater-soluble%2520lipopolymer%2520as%2520a%2520gene%2520carrier%2520to%2520corpus%2520cavernosum%26jtitle%3DInt.%2520J.%2520Impot.%2520Res.%26date%3D2005%26volume%3D17%26spage%3D326%26epage%3D334%26doi%3D10.1038%2Fsj.ijir.3901314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span> <span> </span><span class="NLM_article-title">Modification of antimicrobial peptide with low molar mass poly(ethylene glycol)</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">788</span>, <span class="refDoi"> DOI: 10.1093/jb/mvn134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1093%2Fjb%2Fmvn134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=18845567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1MXis1Cmug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2008&pages=781-788&author=G.+Zhangauthor=B.+Hanauthor=X.+Linauthor=X.+Wuauthor=H.+Yan&title=Modification+of+antimicrobial+peptide+with+low+molar+mass+poly%28ethylene+glycol%29&doi=10.1093%2Fjb%2Fmvn134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modification of antimicrobial peptide with low molar mass poly(ethylene glycol)</span></div><div class="casAuthors">Zhang, Genghui; Han, Baozhong; Lin, Xiaoyan; Wu, Xin; Yan, Husheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">PEGylation of peptide drugs prolongs their circulating lifetimes in plasma.  However, PEGylation can produce a decrease in the in vitro bioactivity.  Longer poly(ethylene glycol) (PEG) chains are favorable for circulating lifetimes but unfavorable for in vitro bioactivities.  In order to circumvent the conflicting effects of PEG length, a hydrophobic peptide, using an antimicrobial peptide as a model, was PEGylated with short PEG chains.  The PEGylated peptides self-assembled in aq. soln. into micelles with PEG shell and peptide core.  In these micelles, the core peptides were protected by the shell, thus reducing proteolytic degrdn.  Meanwhile, most of the in vitro antimicrobial activities still remained due to the short PEG chain attached.  The stabilities of the PEGylated peptides were much higher than that of the unPEGylated peptides in the presence of chymotrypsin and serum.  The antimicrobial activities of the PEGylated peptides in the presence of serum, an ex vivo assay, were much higher than that of the unPEGylated peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgB-hNz3efbbVg90H21EOLACvtfcHk0lh9Epdn79OLpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXis1Cmug%253D%253D&md5=12648ad61504661fe49b0a435a43dfcb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvn134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvn134%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DH.%26atitle%3DModification%2520of%2520antimicrobial%2520peptide%2520with%2520low%2520molar%2520mass%2520poly%2528ethylene%2520glycol%2529%26jtitle%3DJ.%2520Biochem.%26date%3D2008%26volume%3D144%26spage%3D781%26epage%3D788%26doi%3D10.1093%2Fjb%2Fmvn134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benincasa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahariev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelillo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gennaro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scocchi, M.</span></span> <span> </span><span class="NLM_article-title">PEGylation of the peptide Bac7(1–35) reduces renal clearance while retaining antibacterial activity and bacterial cell penetration capacity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.03.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.ejmech.2015.03.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=25817771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFWjsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2015&pages=210-219&author=M.+Benincasaauthor=S.+Zaharievauthor=C.+Pelilloauthor=A.+Milanauthor=R.+Gennaroauthor=M.+Scocchi&title=PEGylation+of+the+peptide+Bac7%281%E2%80%9335%29+reduces+renal+clearance+while+retaining+antibacterial+activity+and+bacterial+cell+penetration+capacity&doi=10.1016%2Fj.ejmech.2015.03.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylation of the peptide Bac7(1-35) reduces renal clearance while retaining antibacterial activity and bacterial cell penetration capacity</span></div><div class="casAuthors">Benincasa, Monica; Zahariev, Sotir; Pelillo, Chiara; Milan, Annalisa; Gennaro, Renato; Scocchi, Marco</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">210-219</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The proline-rich antibacterial peptide Bac7(1-35) protects mice against Salmonella typhimurium infection, despite its rapid clearance.  To overcome this problem the peptide was linked to a polyethylene glycol (PEG) mol. either via a cleavable ester bond or via a non-hydrolysable amide bond.  Both the PEGylated conjugates retained most of the in vitro activity against S. typhimurium.  In addn., the ester bond was cleaved in human serum or plasma, releasing a carboxymethyl deriv. of Bac7(1-35) which accounts for a higher activity of this peptide with relative to the other, non-hydrolysable form.  Both PEGylated peptides maintained the capacity of the unconjugated form to kill bacteria without permeabilizing the bacterial membranes, by penetrating into cells.  They exploited the same transporter as unmodified Bac7(1-35), suggesting it has the capacity to internalize quite sizeable cargo if this is linked to Bac7 fragment.  PEGylation allows the peptide to have a wide distribution in mice, and a slow renal clearance, indicating that this strategy would improve the bioavailability of Bac7, and in principle of other antimicrobial peptides.  This can be an equally important issue to reducing cytotoxicity for therapeutic use of these antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjn03eW3Tzq7Vg90H21EOLACvtfcHk0lh4XY3fZYJ1Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFWjsL0%253D&md5=3e764baf5637f63b12fcebe9b431f690</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.03.028%26sid%3Dliteratum%253Aachs%26aulast%3DBenincasa%26aufirst%3DM.%26aulast%3DZahariev%26aufirst%3DS.%26aulast%3DPelillo%26aufirst%3DC.%26aulast%3DMilan%26aufirst%3DA.%26aulast%3DGennaro%26aufirst%3DR.%26aulast%3DScocchi%26aufirst%3DM.%26atitle%3DPEGylation%2520of%2520the%2520peptide%2520Bac7%25281%25E2%2580%259335%2529%2520reduces%2520renal%2520clearance%2520while%2520retaining%2520antibacterial%2520activity%2520and%2520bacterial%2520cell%2520penetration%2520capacity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D95%26spage%3D210%26epage%3D219%26doi%3D10.1016%2Fj.ejmech.2015.03.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zetterberg, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reijmar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pränting, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engström, Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, K.</span></span> <span> </span><span class="NLM_article-title">PEG-stabilized lipid disks as carriers for amphiphilic antimicrobial peptides</span>. <i>J. Controlled Release</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1016/j.jconrel.2011.08.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.jconrel.2011.08.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=21903146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFGlt7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2011&pages=323-328&author=M.+M.+Zetterbergauthor=K.+Reijmarauthor=M.+Pr%C3%A4ntingauthor=%C3%85.+Engstr%C3%B6mauthor=D.+I.+Anderssonauthor=K.+Edwards&title=PEG-stabilized+lipid+disks+as+carriers+for+amphiphilic+antimicrobial+peptides&doi=10.1016%2Fj.jconrel.2011.08.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PEG-stabilized lipid disks as carriers for amphiphilic antimicrobial peptides</span></div><div class="casAuthors">Zetterberg, Malin M.; Reijmar, Karin; Praenting, Maria; Engstroem, Aake; Andersson, Dan I.; Edwards, Katarina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Controlled Release</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">323-328</span>CODEN:
                <span class="NLM_cas:coden">JCREEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-3659</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antimicrobial peptides hold potential as a possible alternative, or complement, to conventional antibiotics but new, safe and efficient means are needed for formulation and administration of the peptides.  In this study we have investigated the utility of a novel type of lipid particles, the polyethylene glycol-stabilized lipid disks, as carriers for the model peptide melittin.  The structural integrity of the carrier particle when loaded with the peptide was investigated using cryo-transmission electron microscopy.  Liposome leakage upon addn. of the peptide-lipid disks was monitored as a means to verify the membrane lytic effect of the formulation.  The susceptibility of melittin to tryptic digestion was studied and compared in the absence and presence of lipid disks.  Finally, the antibacterial effect of the peptide-lipid disk formulation was compared to that of free melittin after both single and repeated exposure to Escherichia coli.  The results show that melittin can redistribute from the disk into a new host membrane and that formulation in the disks does not compromise melittin's membrane permeabilizing ability.  Further, the peptide was found to be fully protected against degrdn. when bound to the disks.  Time-kill expts. revealed that all the antibacterial effect of melittin administered in free form was gone after a single exposure to E. coli.  In contrast, the disk formulation showed significant cell-killing effect also upon a second exposure to bacteria, indicating an extended release of peptide from the lipid disks.  These results suggest that the lipid disks constitute a new class of promising carriers for peptide antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdg7EJ-_P-0LVg90H21EOLACvtfcHk0lh4XY3fZYJ1Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFGlt7%252FJ&md5=ed1374826be651618d7d5af3f0def605</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jconrel.2011.08.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jconrel.2011.08.029%26sid%3Dliteratum%253Aachs%26aulast%3DZetterberg%26aufirst%3DM.%2BM.%26aulast%3DReijmar%26aufirst%3DK.%26aulast%3DPr%25C3%25A4nting%26aufirst%3DM.%26aulast%3DEngstr%25C3%25B6m%26aufirst%3D%25C3%2585.%26aulast%3DAndersson%26aufirst%3DD.%2BI.%26aulast%3DEdwards%26aufirst%3DK.%26atitle%3DPEG-stabilized%2520lipid%2520disks%2520as%2520carriers%2520for%2520amphiphilic%2520antimicrobial%2520peptides%26jtitle%3DJ.%2520Controlled%2520Release%26date%3D2011%26volume%3D156%26spage%3D323%26epage%3D328%26doi%3D10.1016%2Fj.jconrel.2011.08.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ju, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span> <span> </span><span class="NLM_article-title">Combating pseudomonas aeruginosa biofilms by a chitosan-peg-peptide conjugate via changes in assembled structure</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">13731</span>– <span class="NLM_lpage">13738</span>, <span class="refDoi"> DOI: 10.1021/acsami.0c02034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.0c02034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkslWlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=13731-13738&author=X.+Juauthor=J.+Chenauthor=M.+Zhouauthor=M.+Zhuauthor=Z.+Liauthor=S.+Gaoauthor=J.+Ouauthor=D.+Xuauthor=M.+Wuauthor=S.+Jiangauthor=Y.+Huauthor=Y.+Tianauthor=Z.+Niu&title=Combating+pseudomonas+aeruginosa+biofilms+by+a+chitosan-peg-peptide+conjugate+via+changes+in+assembled+structure&doi=10.1021%2Facsami.0c02034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Pseudomonas aeruginosa Biofilms by a Chitosan-PEG-Peptide Conjugate via Changes in Assembled Structure</span></div><div class="casAuthors">Ju, Xiaoyan; Chen, Jun; Zhou, Mengxue; Zhu, Meng; Li, Zhuang; Gao, Sijia; Ou, Jinzhao; Xu, Dandan; Wu, Man; Jiang, Shidong; Hu, Yi; Tian, Ye; Niu, Zhongwei</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">13731-13738</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa (P. aeruginosa) biofilms are assocd. with a wide range of infections, from chronic tissue diseases to implanted medical devices.  In a biofilm, the extracellular polymeric substance (EPS) causes an inhibited penetration of antibacterial agents, leading to a 100-1000 times tolerance of the bacteria.  In view of the water-filled channels in biofilms and the highly neg. charge of EPS, we design a chitosan-polyethylene glycol-peptide conjugate (CS-PEG-LK13) in this study.  The CS-PEG-LK13 prefers a neutrally charged assembly at a size of ~ 100 nm in aq. environment, while undergoes disassembly to expose the α-helical peptide at the bacterial cell membrane.  This behavior provides CS-PEG-LK13 superiorities in both penetrating the biofilms and inactivating the bacteria.  At a concn. of 8 times the min. inhibitory concn., CS-PEG-LK13 has a much higher antibacterial efficiency (72.70%) than LK13 peptide (15.24%) and tobramycin (33.57%) in an in vitro P. aeruginosa biofilm.  Moreover, CS-PEG-LK13 behaves comparable capability of combating an implanted P. aeruginosa biofilm to highly excess tobramycin.  This work has implications for the design of new antibacterial agents in biofilm combating.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG0WyU1O-ZCLVg90H21EOLACvtfcHk0libfCEppkTAEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkslWlsb4%253D&md5=fb50d9278b3513a86c97fbe405b3a229</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsami.0c02034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.0c02034%26sid%3Dliteratum%253Aachs%26aulast%3DJu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DOu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DNiu%26aufirst%3DZ.%26atitle%3DCombating%2520pseudomonas%2520aeruginosa%2520biofilms%2520by%2520a%2520chitosan-peg-peptide%2520conjugate%2520via%2520changes%2520in%2520assembled%2520structure%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2020%26volume%3D12%26spage%3D13731%26epage%3D13738%26doi%3D10.1021%2Facsami.0c02034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span> <span> </span><span class="NLM_article-title">PEGylated self-assembled nano-bacitracin A: probing the antibacterial mechanism and real-time tracing of target delivery in vivo</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10688</span>– <span class="NLM_lpage">10705</span>, <span class="refDoi"> DOI: 10.1021/acsami.8b00135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.8b00135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFSqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=10688-10705&author=W.+Hongauthor=Y.+Zhaoauthor=Y.+Guoauthor=C.+Huangauthor=P.+Qiuauthor=J.+Zhuauthor=C.+Chuauthor=H.+Shiauthor=M.+Liu&title=PEGylated+self-assembled+nano-bacitracin+A%3A+probing+the+antibacterial+mechanism+and+real-time+tracing+of+target+delivery+in+vivo&doi=10.1021%2Facsami.8b00135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylated self-assembled nano-bacitracin A: Probing the antibacterial mechanism and real-time tracing of target delivery in vivo</span></div><div class="casAuthors">Hong, Wei; Zhao, Yining; Guo, Yuru; Huang, Chengcheng; Qiu, Peng; Zhu, Jia; Chu, Chun; Shi, Hong; Liu, Mingchun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">10688-10705</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although nano-self-assemblies of hydrophobic-modified bacitracin A with poly(D,L-lactic-co-glycolic acid) (PLGA) (nano-BAPLGA) have demonstrated promising antibacterial activities, the application of nano-BAPLGA was severely compromised by low water soly.  In this study, a series of PEGylated PLGA copolymers were selected to conjugate with the N-terminus of bacitracin A to construct PEGylated self-assembled nano-BAs and to further develop nano-self-assemblies of bacitracin A with strong antibacterial potency and high soly.  Compared with nano-BAPLGA, all PEGylated nano-BAs, except nano-BA5k, exhibited strong antibacterial efficiency against both Gram-pos. and Gram-neg. bacteria by inducing loss of cytoplasmic membrane potential, membrane permeabilization, and leakage of calcein from artificial cell membranes.  Studies elucidating the underlying mechanism of PEGylated nano-BAs against Gram-neg. bacteria indicated that the strong hydrophobic and van der Waals interactions between PLGA and lipopolysaccharide (LPS) could bind, neutralize, and disassoc. LPS, facilitating cellular uptake of the nanoparticles, which could destabilize the membrane, resulting in cell death.  Moreover, PEGylated nano-BAs (nano-BA12k) with a longer PLGA block were expected to occupy a higher local d. of BA mass on the surface and result in stronger hydrophobic and van der Waals interactions with LPS, which were responsible for the enhanced antibacterial activity against Gram-pos. and emerging antibacterial activity against Gram-neg. bacteria, resp.  In vivo imaging verified that PEGylated nano-BAs exhibited higher inflammatory tissue distribution and longer circulation time than nano-BAPLGA.  Therefore, although PEGylation did not affect antibacterial activity, it is necessary for target delivery and resistance to clearance of the obsd. PEGylated nano-BAs.  In vivo, nano-BA12k also showed the highest therapeutic index against infection burden in a mouse thigh infection model among the tested formulations, which showed good correlation with the in vitro results.  In conclusion, nano-BA12k showed high efficacy in the treatment of invasive infections.  This new approach of constructing nanoantibiotics by modification of com. available antibiotics with PEGylated copolymers is safe, cost-effective, and environmentally friendly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuxkfEosKprVg90H21EOLACvtfcHk0libfCEppkTAEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFSqurg%253D&md5=468e3e10bcae106b7a132f425e76991c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsami.8b00135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.8b00135%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DChu%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DM.%26atitle%3DPEGylated%2520self-assembled%2520nano-bacitracin%2520A%253A%2520probing%2520the%2520antibacterial%2520mechanism%2520and%2520real-time%2520tracing%2520of%2520target%2520delivery%2520in%2520vivo%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2018%26volume%3D10%26spage%3D10688%26epage%3D10705%26doi%3D10.1021%2Facsami.8b00135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, M.</span></span> <span> </span><span class="NLM_article-title">Pegylation: a novel process for modifying pharmacokinetics</span>. <i>Clin. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.2165/00003088-200140070-00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.2165%2F00003088-200140070-00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=11510630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1ars70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=539-551&author=J.+M.+Harrisauthor=N.+E.+Martinauthor=M.+Modi&title=Pegylation%3A+a+novel+process+for+modifying+pharmacokinetics&doi=10.2165%2F00003088-200140070-00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pegylation. A novel process for modifying pharmacokinetics</span></div><div class="casAuthors">Harris, J. Milton; Martin, Nancy E.; Modi, Marlene</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacokinetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">539-551</span>CODEN:
                <span class="NLM_cas:coden">CPKNDH</span>;
        ISSN:<span class="NLM_cas:issn">0312-5963</span>.
    
            (<span class="NLM_cas:orgname">Adis International Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of liposomal carriers and the modification of therapeutic mols. through the attachment of poly(ethylene glycol) [PEG] moieties ('pegylation') are the most common approaches for enhancing the delivery of parenteral agents.  Although 'classical' liposomes (i.e. phospholipid bilayer vehicles) have been effective in decreasing the clearance of encapsulated agents and in passively targeting specific tissues, they are assocd. with considerable limitations.  Pegylation may be an effective method of delivering therapeutic proteins and modifying their pharmacokinetic properties, in turn modifying pharmacodynamics, via a mechanism dependent on altered binding properties of the native protein.  Pegylation reduces renal clearance and, for some products, results in a more sustained absorption after s.c. administration as well as restricted distribution.  These pharmacokinetic changes may result in more const. and sustained plasma concns., which can lead to increases in clin. effectiveness when the desired effects are concn.-dependent.  Maintaining drug concns. at or near a target concn. for an extended period of time is often clin. advantageous, and is particularly useful in antiviral therapy, since const. antiviral pressure should prevent replication and may thereby suppress the emergence of resistant variants.  Addnl., PEG modification may decrease adverse effects caused by the large variations in peak-to-trough plasma drug concns. assocd. with frequent administration and by the immunogenicity of unmodified proteins.  Pegylated proteins may have reduced immunogenicity because PEG-induced steric hindrance can prevent immune recognition.  Two PEG-modified proteins are currently approved by the US Food and Drug Administration; several others, including cytokines such as interferon-α (IFNα), growth factors and free radical scavengers, are under development.  Careful assessment of various pegylated IFNα products suggests that pegylated mols. can be differentiated on the basis of their pharmacokinetic properties and related changes in pharmacodynamics.  Because the size, geometry and attachment site of the PEG moiety play a crucial role in detg. these properties, therapeutically optimized agents must be designed on a protein-by-protein basis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnVR6JmxSu-bVg90H21EOLACvtfcHk0ljCPwhae6I9xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1ars70%253D&md5=58c186af44c9cac1b7d3a90db977383b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.2165%2F00003088-200140070-00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00003088-200140070-00005%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DJ.%2BM.%26aulast%3DMartin%26aufirst%3DN.%2BE.%26aulast%3DModi%26aufirst%3DM.%26atitle%3DPegylation%253A%2520a%2520novel%2520process%2520for%2520modifying%2520pharmacokinetics%26jtitle%3DClin.%2520Pharmacokinet.%26date%3D2001%26volume%3D40%26spage%3D539%26epage%3D551%26doi%3D10.2165%2F00003088-200140070-00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manhart, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinke, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntosh, C. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pederson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demuth, H. U.</span></span> <span> </span><span class="NLM_article-title">Structure-function analysis of a series of novel GIP analogues containing different helical length linkers</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">3081</span>– <span class="NLM_lpage">3088</span>, <span class="refDoi"> DOI: 10.1021/bi026868e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi026868e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1Oiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=3081-3088&author=S.+Manhartauthor=S.+A.+Hinkeauthor=C.+H.+S.+McIntoshauthor=R.+A.+Pedersonauthor=H.+U.+Demuth&title=Structure-function+analysis+of+a+series+of+novel+GIP+analogues+containing+different+helical+length+linkers&doi=10.1021%2Fbi026868e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-function analysis of a series of novel GIP analogues containing different helical length linkers</span></div><div class="casAuthors">Manhart, Susanne; Hinke, Simon A.; McIntosh, Christopher H. S.; Pederson, Raymond A.; Demuth, Hans-Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3081-3088</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucose-dependent insulinotropic polypeptide (GIP1-42) is a potent glucose-lowering intestinal peptide hormone.  The equipotent GIP1-30NH2 was structurally modified by linking N- and C-terminal fragments with several different linkers.  Substitution of the middle region of GIP by a flexible aminohexanoic linker resulted in greatly reduced binding affinity and redn. or complete loss of bioactivity.  Connection of the bioactive domains GIP1-14 and GIP19-30NH2 by EKEK or AAAA linkers resulted in peptide agonists with approx. 3-4-fold increased bioactivity as compared to GIP1-30NH2.  Conformational anal. by CD spectroscopy of GIP fragments and analogs suggests a helical region in the C-terminal (19-30) portion of GIP.  It was demonstrated that stabilization of this C-terminal helical region by the introduction of helical linkers favored binding and activation of the GIP receptor.  The authors' results suggest an important contribution of a direct interaction of the first 14 amino acids with the GIP receptor, an appropriate relative orientation of N- and C-terminal parts of GIP, and the presence of helical linkers to be essential for bioactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEznrTXFLdMbVg90H21EOLACvtfcHk0ljCPwhae6I9xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1Oiu7w%253D&md5=90262367e1b4c16303bae79ecf7bc72f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi026868e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi026868e%26sid%3Dliteratum%253Aachs%26aulast%3DManhart%26aufirst%3DS.%26aulast%3DHinke%26aufirst%3DS.%2BA.%26aulast%3DMcIntosh%26aufirst%3DC.%2BH.%2BS.%26aulast%3DPederson%26aufirst%3DR.%2BA.%26aulast%3DDemuth%26aufirst%3DH.%2BU.%26atitle%3DStructure-function%2520analysis%2520of%2520a%2520series%2520of%2520novel%2520GIP%2520analogues%2520containing%2520different%2520helical%2520length%2520linkers%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D3081%26epage%3D3088%26doi%3D10.1021%2Fbi026868e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suwatthee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. Y. C.</span></span> <span> </span><span class="NLM_article-title">Beyond electrostatics: antimicrobial peptide selectivity and the influence of cholesterol-mediated fluidity and lipid chain length on protegrin-1 activity</span>. <i>Biochim. Biophys. Acta-Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1861</i></span>,  <span class="NLM_fpage">182977</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2019.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2019.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=31077677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtF2nsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1861&publication_year=2019&pages=182977&author=J.+M.+Hendersonauthor=N.+S.+Iyengarauthor=K.+L.+H.+Lamauthor=E.+Maldonadoauthor=T.+Suwattheeauthor=I.+Royauthor=A.+J.+Waringauthor=K.+Y.+C.+Lee&title=Beyond+electrostatics%3A+antimicrobial+peptide+selectivity+and+the+influence+of+cholesterol-mediated+fluidity+and+lipid+chain+length+on+protegrin-1+activity&doi=10.1016%2Fj.bbamem.2019.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond electrostatics: antimicrobial peptide selectivity and the influence of cholesterol-mediated fluidity and lipid chain length on protegrin-1 activity</span></div><div class="casAuthors">Henderson, J. Michael; Iyengar, Nishanth S.; Lam, Kin Lok H.; Maldonado, Eddie; Suwatthee, Tiffany; Roy, Indroneil; Waring, Alan J.; Lee, Ka Yee C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1861</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">182977</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs) are a promising class of innate host defense mols. for next-generation antibiotics, as they uniquely target and permeabilize membranes of pathogens.  This selectivity has been explained by the electrostatic attraction between these predominantly cationic peptides and the bacterial membrane, which is heavily populated with anionic lipids.  However, AMP-resistant bacteria have non-electrostatic countermeasures that modulate membrane rigidity and thickness.  We explore how variations in phys. properties affect the membrane affinity and disruption process of protegrin-1 (PG-1) in phosphatidylcholine (PC) membranes with altered lipid packing densities and thicknesses.  From isothermal titrn. calorimetry and at. force microscopy, our results showed that PG-1 could no longer insert into membranes of increasing cholesterol amts. nor into monounsatd. PC membranes of increasing thicknesses with similar fluidities.  Prevention of PG-1's incorporation consequently made the membranes more resistant to peptide-induced structural transformations like pore formation.  Our study provides evidence that AMP affinity and activity are strongly correlated with the fluidity and thickness of the membrane.  A basic understanding of how phys. mechanisms can regulate cell selectivity and resistance towards AMPs will aid in the development of new antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS8H_GzFo-vbVg90H21EOLACvtfcHk0ljCPwhae6I9xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtF2nsLbL&md5=6d577884896e9306890dbd4db1dad6a9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2019.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2019.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DJ.%2BM.%26aulast%3DIyengar%26aufirst%3DN.%2BS.%26aulast%3DLam%26aufirst%3DK.%2BL.%2BH.%26aulast%3DMaldonado%26aufirst%3DE.%26aulast%3DSuwatthee%26aufirst%3DT.%26aulast%3DRoy%26aufirst%3DI.%26aulast%3DWaring%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DK.%2BY.%2BC.%26atitle%3DBeyond%2520electrostatics%253A%2520antimicrobial%2520peptide%2520selectivity%2520and%2520the%2520influence%2520of%2520cholesterol-mediated%2520fluidity%2520and%2520lipid%2520chain%2520length%2520on%2520protegrin-1%2520activity%26jtitle%3DBiochim.%2520Biophys.%2520Acta-Biomembr.%26date%3D2019%26volume%3D1861%26spage%3D182977%26doi%3D10.1016%2Fj.bbamem.2019.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maystrenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">The addition of a synthetic LPS-targeting domain improves serum stability while maintaining antimicrobial, antibiofilm, and cell stimulating properties of an antimicrobial peptide</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1014</span>, <span class="refDoi"> DOI: 10.3390/biom10071014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.3390%2Fbiom10071014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1ChsLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=1014&author=A.+Maystrenkoauthor=Y.+Fengauthor=N.+Akhtarauthor=J.+Li&title=The+addition+of+a+synthetic+LPS-targeting+domain+improves+serum+stability+while+maintaining+antimicrobial%2C+antibiofilm%2C+and+cell+stimulating+properties+of+an+antimicrobial+peptide&doi=10.3390%2Fbiom10071014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The addition of a synthetic LPS-targeting domain improves serum stability while maintaining antimicrobial, antibiofilm, and cell stimulating properties of an antimicrobial peptide</span></div><div class="casAuthors">Maystrenko, Anna; Feng, Yulong; Akhtar, Nadeem; Li, Julang</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1014</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Multi-drug resistant (MDR) bacteria and their biofilms are a concern in veterinary and human medicine.  Protegrin-1 (PG-1), a potent antimicrobial peptide (AMP) with antimicrobial and immunomodulatory properties, is considered a potential substitute for conventional antibiotics.  To improve stability of PG-1, a hybrid peptide, SynPG-1, was designed.  The antimicrobial and antibiofilm properties of PG-1 and the PG-1 hybrid against MDR pathogens was analyzed, and activity after incubation with physiol. fluids was compared.  The effects of these peptides on the IPEC-J2 cell line was also investigated.  While PG-1 maintained some activity in 25% serum for 2 h, SynPG-1 was able to retain activity in the same condition for up to 24 h, representing a 12-fold increase in stability.  Both peptides had some antibiofilm activity against Escherichia coli and Salmonella typhimurium.  While both peptides prevented biofilm formation of methicillin-resistant Staphylococcus aureus (MRSA), neither could destroy MRSA's pre-formed biofilms.  Both peptides maintained activity after incubation with trypsin and porcine gastric fluid, but not intestinal fluid, and stimulated IPEC-J2 cell migration.  These findings suggest that SynPG-1 has much better serum stability while maintaining the same antimicrobial potency as PG-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrib7YXFKncyrVg90H21EOLACvtfcHk0liVf6Oyla7n_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1ChsLfJ&md5=0f8593beb414571b3ea239fc7c0276bd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3390%2Fbiom10071014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom10071014%26sid%3Dliteratum%253Aachs%26aulast%3DMaystrenko%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DAkhtar%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DThe%2520addition%2520of%2520a%2520synthetic%2520LPS-targeting%2520domain%2520improves%2520serum%2520stability%2520while%2520maintaining%2520antimicrobial%252C%2520antibiofilm%252C%2520and%2520cell%2520stimulating%2520properties%2520of%2520an%2520antimicrobial%2520peptide%26jtitle%3DBiomolecules%26date%3D2020%26volume%3D10%26spage%3D1014%26doi%3D10.3390%2Fbiom10071014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moravej, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moravej, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yazdanparast, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heiat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirhosseini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosazadeh Moghaddam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirnejad, R.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides: features, action, and their resistance mechanisms in bacteria</span>. <i>Microb. Drug Resist.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">747</span>– <span class="NLM_lpage">767</span>, <span class="refDoi"> DOI: 10.1089/mdr.2017.0392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1089%2Fmdr.2017.0392" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=29957118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht12mtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=747-767&author=H.+Moravejauthor=Z.+Moravejauthor=M.+Yazdanparastauthor=M.+Heiatauthor=A.+Mirhosseiniauthor=M.+Moosazadeh+Moghaddamauthor=R.+Mirnejad&title=Antimicrobial+peptides%3A+features%2C+action%2C+and+their+resistance+mechanisms+in+bacteria&doi=10.1089%2Fmdr.2017.0392"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial Peptides: Features, Action, and Their Resistance Mechanisms in Bacteria</span></div><div class="casAuthors">Moravej, Hoda; Moravej, Zahra; Yazdanparast, Maryam; Heiat, Mohammad; Mirhosseini, Ali; Moosazadeh Moghaddam, Mehrdad; Mirnejad, Reza</div><div class="citationInfo"><span class="NLM_cas:title">Microbial Drug Resistance (New Rochelle, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">747-767</span>CODEN:
                <span class="NLM_cas:coden">MDREFJ</span>;
        ISSN:<span class="NLM_cas:issn">1076-6294</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  In recent years, because of increased resistance to conventional antimicrobials, many researchers have started to study the synthesis of new antibiotics to control the disease-causing effects of infectious pathogens.  Antimicrobial peptides (AMPs) are among the newest antibiotics; these peptides are integral compds. in all kinds of organisms and play a significant role in microbial ecol., and critically contribute to the innate immunity of organisms by destroying invading microorganisms.  Moreover, AMPs may encourage cells to produce chemokines, stimulate angiogenesis, accelerate wound healing, and influence programmed cell death in multicellular organisms.  Bacteria differ in their inherent susceptibility and resistance mechanisms to these peptides when responding to the antimicrobial effects of AMPs.  Generally, the development of AMP resistance mechanisms is driven by direct competition between bacterial species, and host and pathogen interactions.  Several studies have shown diverse mechanisms of bacterial resistance to AMPs, for example, some bacteria produce proteases and trapping proteins; some modify cell surface charge, change membrane fluidity, and activate efflux pumps; and some species make use of biofilms and exopolymers, and develop sensing systems by selective gene expression.  A closer understanding of bacterial resistance mechanisms may help in developing novel therapeutic approaches for the treatment of infections caused by pathogenic organisms that are successful in developing extensive resistance to AMPs.  Based on these observations, this review discusses the properties of AMPs, their targeting mechanisms, and bacterial resistance mechanisms against AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpRMQ0V4yVbVg90H21EOLACvtfcHk0liVf6Oyla7n_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht12mtrfF&md5=176913027ea9b055f92550095dde962f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1089%2Fmdr.2017.0392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fmdr.2017.0392%26sid%3Dliteratum%253Aachs%26aulast%3DMoravej%26aufirst%3DH.%26aulast%3DMoravej%26aufirst%3DZ.%26aulast%3DYazdanparast%26aufirst%3DM.%26aulast%3DHeiat%26aufirst%3DM.%26aulast%3DMirhosseini%26aufirst%3DA.%26aulast%3DMoosazadeh%2BMoghaddam%26aufirst%3DM.%26aulast%3DMirnejad%26aufirst%3DR.%26atitle%3DAntimicrobial%2520peptides%253A%2520features%252C%2520action%252C%2520and%2520their%2520resistance%2520mechanisms%2520in%2520bacteria%26jtitle%3DMicrob.%2520Drug%2520Resist.%26date%3D2018%26volume%3D24%26spage%3D747%26epage%3D767%26doi%3D10.1089%2Fmdr.2017.0392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, B.</span></span> <span> </span><span class="NLM_article-title">Self-assembling surfactant-like peptide A(6)K as potential delivery system for hydrophobic drugs</span>. <i>Int. J. Nanomed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">858</span>, <span class="refDoi"> DOI: 10.2147/IJN.S71696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.2147%2FIJN.S71696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=25670898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislaiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=847-858&author=F.+Qiuauthor=Y.+Chenauthor=C.+Tangauthor=J.+Zhangauthor=M.+Gongauthor=B.+Su&title=Self-assembling+surfactant-like+peptide+A%286%29K+as+potential+delivery+system+for+hydrophobic+drugs&doi=10.2147%2FIJN.S71696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Self-assembling surfactant-like peptide A6K as potential delivery system for hydrophobic drugs</span></div><div class="casAuthors">Chen, Yongzhu; Tang, Chengkang; Zhang, Jie; Gong, Meng; Su, Bo; Qiu, Feng</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Nanomedicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">847-858/1-847-858/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJNNHQ</span>;
        ISSN:<span class="NLM_cas:issn">1178-2013</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Background: Finding a suitable delivery system to improve the water soly. of hydrophobic drugs is a crit. challenge in the development of effective formulations.  In this study, we used A6K, a self-assembling surfactant-like peptide, as a carrier to encapsulate and deliver hydrophobic pyrene.  Methods: Pyrene was mixed with A6K by magnetic stirring to form a suspension.  Confocal laser scanning microscopy, TEM, dynamic light scattering, at. force microscopy, fluorescence, and cell uptake measurements were carried out to study the features and stability of the nanostructures, the state and content of pyrene, as well as the pyrene release profile.  Results: The suspension formed contained pyrene monomers trapped in the hydrophobic cores of the micellar nanofibers formed by A6K, as well as nanosized pyrene crystals wrapped up and stabilized by the nanofibers.  The two different encapsulation methods greatly increased the concn. of pyrene in the suspension, and formation of pyrene crystals wrapped up by A6K nanofibers might be the major contributor to this effect.  Furthermore, the suspension system could readily release and transfer pyrene into living cells.  Conclusion: A6K could be further exploited as a promising delivery system for hydrophobic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxCLW4xvdgnrVg90H21EOLACvtfcHk0lhsmm527sWtPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislaiu74%253D&md5=9bb78ad7c7acc4d5523f35f8353c97d8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2147%2FIJN.S71696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FIJN.S71696%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGong%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DB.%26atitle%3DSelf-assembling%2520surfactant-like%2520peptide%2520A%25286%2529K%2520as%2520potential%2520delivery%2520system%2520for%2520hydrophobic%2520drugs%26jtitle%3DInt.%2520J.%2520Nanomed.%26date%3D2015%26volume%3D10%26spage%3D847%26epage%3D858%26doi%3D10.2147%2FIJN.S71696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, M. F.</span></span> <span> </span><span class="NLM_article-title">Self-assembly of surfactant-like peptides</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">12729</span>– <span class="NLM_lpage">12736</span>, <span class="refDoi"> DOI: 10.1021/la7011183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/la7011183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Ohsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=12729-12736&author=D.+J.+Adamsauthor=K.+Holtzmannauthor=C.+Schneiderauthor=M.+F.+Butler&title=Self-assembly+of+surfactant-like+peptides&doi=10.1021%2Fla7011183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Assembly of Surfactant-like Peptides</span></div><div class="casAuthors">Adams, Dave J.; Holtzmann, Kathrin; Schneider, Christian; Butler, Michael F.</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">12729-12736</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inspired by recent work describing surfactant-like peptides, we have carried out a systematic study on peptides with the underlying compn. of V6D2, altering the abs. sequence to det. the importance of the surfactant-like structure.  All of the peptides examd. here formed self-assembled structures in water.  However, in contrast to other reports, we have found a surprising diversity of structures including fibers, tapes, and twisted ribbons but an absence of the vesicles and nanotubes described previously.  Further investigations demonstrated that peptide purity plays a significant role in the outcome of the self-assembly.  Different batches behave very differently, which can be linked to the compns. of these batches.  This work shows that there is a need for not only rational design but also ease of synthesis of the building blocks for self-assembled structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZV4rvg4WlhrVg90H21EOLACvtfcHk0lhsmm527sWtPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Ohsb%252FM&md5=4b128d1ff6f9d8e4df05cd6e71e64401</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fla7011183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fla7011183%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DHoltzmann%26aufirst%3DK.%26aulast%3DSchneider%26aufirst%3DC.%26aulast%3DButler%26aufirst%3DM.%2BF.%26atitle%3DSelf-assembly%2520of%2520surfactant-like%2520peptides%26jtitle%3DLangmuir%26date%3D2007%26volume%3D23%26spage%3D12729%26epage%3D12736%26doi%3D10.1021%2Fla7011183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perinelli, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vllasaliu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doutch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casettari, L.</span></span> <span> </span><span class="NLM_article-title">PEGylation affects the self-assembling behaviour of amphiphilic octapeptides</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>571</i></span>, <span class="NLM_elocation-id">118752</span> <span class="refDoi"> DOI: 10.1016/j.ijpharm.2019.118752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.ijpharm.2019.118752" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=31606529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVGlsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2019&author=D.+R.+Perinelliauthor=M.+Campanaauthor=I.+Singhauthor=D.+Vllasaliuauthor=J.+Doutchauthor=G.+F.+Palmieriauthor=L.+Casettari&title=PEGylation+affects+the+self-assembling+behaviour+of+amphiphilic+octapeptides&doi=10.1016%2Fj.ijpharm.2019.118752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylation affects the self-assembling behaviour of amphiphilic octapeptides</span></div><div class="casAuthors">Perinelli, Diego Romano; Campana, Mario; Singh, Ishwar; Vllasaliu, Driton; Doutch, James; Palmieri, Giovanni Filippo; Casettari, Luca</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">571</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">118752</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Surfactant-like peptides are a class of amphiphilic macromols., which are able to self-assemble in water forming different supramol. structures.  Among them, octapeptides composed of six hydrophobic and two hydrophilic residues have attracted interest since they have a length similar to those of natural phospholipids.  Supramol. structures of different amphiphilic octapeptides have been widely reported, but no study has been performed aimed at investigating the effect of PEGylation on their self-assembling behavior.  The aim of the present work was to synthesize and characterize the self-assembling behavior of PEGylated alanine- or valine based amphiphilic octapeptides (mPEG1.9kDa-DDAAAAAA and mPEG1.9kDa-DDVVVVVV) in comparison to the non-PEGylated ones (DDAAAAAA and DDVVVVVV).  The self-aggregation process in ultrapure water was investigated by fluorescence spectroscopy, small angle neutron scattering (SANS), dynamic light scattering (DLS), while the secondary structure was assessed by CD.  PEGylation markedly affects the self-assembling behavior of these amphiphilic octapeptides in terms of both crit. aggregation concn. (CAC) and shape of the formed supramol. aggregates.  Indeed, PEGylation increases CAC and prevents the self-aggregation into fibrillary supramol. aggregates (as obsd. for non-PEGylated peptides), by promoting the formation of micelle-like structures (as demonstrated for valine-based octapeptide).  On the other side, the secondary structure of peptides seems not to be affected by PEGylation.  Overall, these results suggest that self-assembling behavior of amphiphilic octapeptides can be modified by PEGylation, with a great potential impact for the future applications of these nanomaterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotvNRv-0fPXrVg90H21EOLACvtfcHk0lhA0YAus4AnfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVGlsL%252FN&md5=091bf55552cd3ca00eaaba8e0b69700a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2019.118752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2019.118752%26sid%3Dliteratum%253Aachs%26aulast%3DPerinelli%26aufirst%3DD.%2BR.%26aulast%3DCampana%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DI.%26aulast%3DVllasaliu%26aufirst%3DD.%26aulast%3DDoutch%26aufirst%3DJ.%26aulast%3DPalmieri%26aufirst%3DG.%2BF.%26aulast%3DCasettari%26aufirst%3DL.%26atitle%3DPEGylation%2520affects%2520the%2520self-assembling%2520behaviour%2520of%2520amphiphilic%2520octapeptides%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2019%26volume%3D571%26doi%3D10.1016%2Fj.ijpharm.2019.118752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Combined systems approaches reveal a multistage mode of action of a marine antimicrobial peptide against pathogenic escherichia coli and its protective effect against bacterial peritonitis and endotoxemia</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e01056-16</span> <span class="refDoi"> DOI: 10.1128/AAC.01056-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1128%2FAAC.01056-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=27795369" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=X.+Wangauthor=D.+Tengauthor=R.+Maoauthor=N.+Yangauthor=Y.+Haoauthor=J.+Wang&title=Combined+systems+approaches+reveal+a+multistage+mode+of+action+of+a+marine+antimicrobial+peptide+against+pathogenic+escherichia+coli+and+its+protective+effect+against+bacterial+peritonitis+and+endotoxemia&doi=10.1128%2FAAC.01056-16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1128%2FAAC.01056-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01056-16%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DTeng%26aufirst%3DD.%26aulast%3DMao%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DN.%26aulast%3DHao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DCombined%2520systems%2520approaches%2520reveal%2520a%2520multistage%2520mode%2520of%2520action%2520of%2520a%2520marine%2520antimicrobial%2520peptide%2520against%2520pathogenic%2520escherichia%2520coli%2520and%2520its%2520protective%2520effect%2520against%2520bacterial%2520peritonitis%2520and%2520endotoxemia%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.01056-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroll, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span> <span> </span><span class="NLM_article-title">Self-assembled peptide nanofibers display natural antimicrobial peptides to selectively kill bacteria without compromising cytocompatibility</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">28681</span>– <span class="NLM_lpage">28689</span>, <span class="refDoi"> DOI: 10.1021/acsami.9b09583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.9b09583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVSqtr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=28681-28689&author=W.+Chenauthor=S.+Yangauthor=S.+Liauthor=J.+C.+Langauthor=C.+Maoauthor=P.+Krollauthor=L.+Tangauthor=H.+Dong&title=Self-assembled+peptide+nanofibers+display+natural+antimicrobial+peptides+to+selectively+kill+bacteria+without+compromising+cytocompatibility&doi=10.1021%2Facsami.9b09583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Assembled Peptide Nanofibers Display Natural Antimicrobial Peptides to Selectively Kill Bacteria without Compromising Cytocompatibility</span></div><div class="casAuthors">Chen, Weike; Yang, Su; Li, Shuxin; Lang, John C.; Mao, Chuanbin; Kroll, Peter; Tang, Liping; Dong, He</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">28681-28689</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the major hurdles in the development of antimicrobial peptide (AMP)-based materials is their poor capacity in selectively killing bacteria without harming nearby mammalian cells.  Namely, they are antimicrobial but cytotoxic.  Current methods of nanoparticle-encapsulated AMPs to target bacteria selectively still have not yet overcome this hurdle.  Here we demonstrate a simple yet effective method to address this daunting challenge by assocg. a natural AMP with a β-sheet-forming synthetic peptide.  The integrated peptides self-assembled to form a supramol. nanofiber, resulting in the presentation of the AMP at the nanofiber-solvent interface in a precisely controlled manner.  Using melittin as a model natural AMP, we found that the conformation of melittin changed dramatically when presented on the nanofiber surface, which in turn modulated the induced membrane permeability of the bacterial and mammalian cell membranes.  Specifically, the presentation of melittin on the nanofiber restricted its hydrophobic residues, leading to a redn. of the hydrophobic interaction with lipids in the cell membranes.  Compellingly, the reduced hydrophobic interaction led to a considerably decrease of melittin's induced permeability of the mammalian cell membrane than that of the bacterial cell membrane.  As a result, the AMP-displaying nanofiber preferentially permeabilized and disrupted the membrane of the bacteria without compromising the mammalian cells.  Such improved membrane selectivity and cytocompatibility were confirmed in a cell-based membrane localization and live-dead assay.  Our new strategy holds great promise for fabricating cytocompatible antimicrobial assemblies that offer safer and more effective administration of therapeutic AMPs.  These assemblies, with intrinsic antimicrobial activity and cytocompatibility, can also serve as building blocks for the construction of higher-ordered scaffolds for other biomedical applications such as tissue engineering and regenerative medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojN6nRcrufsbVg90H21EOLACvtfcHk0lhA0YAus4AnfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVSqtr%252FI&md5=c32908aa80f1e5511bfea1be9cf602a3</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facsami.9b09583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.9b09583%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLang%26aufirst%3DJ.%2BC.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DKroll%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DH.%26atitle%3DSelf-assembled%2520peptide%2520nanofibers%2520display%2520natural%2520antimicrobial%2520peptides%2520to%2520selectively%2520kill%2520bacteria%2520without%2520compromising%2520cytocompatibility%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2019%26volume%3D11%26spage%3D28681%26epage%3D28689%26doi%3D10.1021%2Facsami.9b09583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dehsorkhi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamley, I. W.</span></span> <span> </span><span class="NLM_article-title">Self-assembling amphiphilic peptides</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1002/psc.2633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1002%2Fpsc.2633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=24729276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVCltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=453-467&author=A.+Dehsorkhiauthor=V.+Castellettoauthor=I.+W.+Hamley&title=Self-assembling+amphiphilic+peptides&doi=10.1002%2Fpsc.2633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Self-assembling amphiphilic peptides</span></div><div class="casAuthors">Dehsorkhi, Ashkan; Castelletto, Valeria; Hamley, Ian W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">453-467</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  The self-assembly of several classes of amphiphilic peptides is reviewed, and selected applications are discussed.  The authors discuss recent work on the self-assembly of lipopeptides, surfactant-like peptides, and amyloid peptides derived from the amyloid-β peptide.  The influence of environmental variables such as pH and temp. on aggregate nanostructure is discussed.  Enzyme-induced remodeling due to peptide cleavage and nanostructure control through photocleavage or photocrosslinking are also considered.  Lastly, selected applications of amphiphilic peptides in biomedicine and materials science are outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobEYsURgvLhbVg90H21EOLACvtfcHk0lhA0YAus4AnfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVCltLw%253D&md5=cb940bade3d9a1b06b36d0f76e8b5964</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2633%26sid%3Dliteratum%253Aachs%26aulast%3DDehsorkhi%26aufirst%3DA.%26aulast%3DCastelletto%26aufirst%3DV.%26aulast%3DHamley%26aufirst%3DI.%2BW.%26atitle%3DSelf-assembling%2520amphiphilic%2520peptides%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2014%26volume%3D20%26spage%3D453%26epage%3D467%26doi%3D10.1002%2Fpsc.2633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guiotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzobon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manganelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, F. M.</span></span> <span> </span><span class="NLM_article-title">PEGylation of the antimicrobial peptide nisin A: problems and perspectives</span>. <i>Farmaco</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/S0014-827X(02)01301-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2FS0014-827X%2802%2901301-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=12595036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1KrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2003&pages=45-50&author=A.+Guiottoauthor=M.+Pozzobonauthor=M.+Canevariauthor=R.+Manganelliauthor=M.+Scarinauthor=F.+M.+Veronese&title=PEGylation+of+the+antimicrobial+peptide+nisin+A%3A+problems+and+perspectives&doi=10.1016%2FS0014-827X%2802%2901301-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylation of the antimicrobial peptide nisin A: problems and perspectives</span></div><div class="casAuthors">Guiotto, Andrea; Pozzobon, Michela; Canevari, Mirta; Manganelli, Riccardo; Scarin, Marino; Veronese, Francesco M.</div><div class="citationInfo"><span class="NLM_cas:title">Farmaco</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-50</span>CODEN:
                <span class="NLM_cas:coden">FRMCE8</span>;
        ISSN:<span class="NLM_cas:issn">0014-827X</span>.
    
            (<span class="NLM_cas:orgname">Editions Scientifiques et Medicales Elsevier</span>)
        </div><div class="casAbstract">Nisin is a natural antimicrobial peptide produced by Lactococcus lactis and widely employed as food preservative.  Its low soly. in neutral aq. solns., its instability at physiol. pH and its rapid breakdown by proteolytic enzymes has limited its use for processed foods (processed cheese, milk and derivs., canned vegetables).  The conjugation to poly(ethylene glycol) (PEG) could improve its soly. and protect it towards enzymes present in non optimally processed food.  The authors report the synthesis of a PEG-nisin conjugate, and the microbiol. assays against some bacterial cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroZP0JJBl3TbVg90H21EOLACvtfcHk0lg2zd3iCiDHdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1KrsLw%253D&md5=879c1320913de7a8c4dd91977e8ac7ed</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0014-827X%2802%2901301-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-827X%252802%252901301-0%26sid%3Dliteratum%253Aachs%26aulast%3DGuiotto%26aufirst%3DA.%26aulast%3DPozzobon%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DM.%26aulast%3DManganelli%26aufirst%3DR.%26aulast%3DScarin%26aufirst%3DM.%26aulast%3DVeronese%26aufirst%3DF.%2BM.%26atitle%3DPEGylation%2520of%2520the%2520antimicrobial%2520peptide%2520nisin%2520A%253A%2520problems%2520and%2520perspectives%26jtitle%3DFarmaco%26date%3D2003%26volume%3D58%26spage%3D45%26epage%3D50%26doi%3D10.1016%2FS0014-827X%2802%2901301-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciumac, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. R.</span></span> <span> </span><span class="NLM_article-title">Membrane targeting cationic antimicrobial peptides</span>. <i>J. Colloid Interface Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/j.jcis.2018.10.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.jcis.2018.10.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=30439615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKlsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2019&pages=163-185&author=D.+Ciumacauthor=H.+Gongauthor=X.+Huauthor=J.+R.+Lu&title=Membrane+targeting+cationic+antimicrobial+peptides&doi=10.1016%2Fj.jcis.2018.10.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane targeting cationic antimicrobial peptides</span></div><div class="casAuthors">Ciumac, Daniela; Gong, Haoning; Hu, Xuzhi; Lu, Jian Ren</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Colloid and Interface Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">163-185</span>CODEN:
                <span class="NLM_cas:coden">JCISA5</span>;
        ISSN:<span class="NLM_cas:issn">0021-9797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many short cationic peptides are amphiphilic and are often termed antimicrobial peptides (AMPs) as they can kill various microorganisms.  These AMPs have largely been discovered from nature, but over the past two decades many biomimetic and de novo designed AMPs have been reported, offering a huge variety of attractive properties for further exploitation.  Under the current global endeavour of fighting against antimicrobial resistance, it is useful to introduce AMPs to the biointerface research community and compare their modes of action with conventional antibiotics.  Because natural AMPs often have long sequences and other biol. functions implicated, they can't be used as antimicrobial agents.  However, rational AMP design helps eliminate their shortcomings and more importantly, optimize their structure-function relationship.  This review will first introduce the key approaches recently utilized in structural design of AMPs and then introduce the main lipid membrane models such as spread lipid monolayers and vesicles together with the characterization techniques adopted in early AMP design and development.  These studies are crucial towards understanding key factors affecting their efficacy and toxicity.  Thus, various interfacial measurements facilitated by different forms of lipid monolayers and bilayers provide valuable support to the selective responses of AMPs to different cell types used in bactericidal assays and cytotoxicity tests, emphasizing the link between mol. models and cell models.  A no. of clin. trials of AMPs have been either under way or completed, demonstrating the huge potential of AMPs in a range of applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpjYgn14ge7rVg90H21EOLACvtfcHk0liGJhLITfBKtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKlsL%252FL&md5=5608475b4023dc6b74c3c0e4d19aeb25</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jcis.2018.10.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jcis.2018.10.103%26sid%3Dliteratum%253Aachs%26aulast%3DCiumac%26aufirst%3DD.%26aulast%3DGong%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DJ.%2BR.%26atitle%3DMembrane%2520targeting%2520cationic%2520antimicrobial%2520peptides%26jtitle%3DJ.%2520Colloid%2520Interface%2520Sci.%26date%3D2019%26volume%3D537%26spage%3D163%26epage%3D185%26doi%3D10.1016%2Fj.jcis.2018.10.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walkenhorst, W. F.</span></span> <span> </span><span class="NLM_article-title">Using adjuvants and environmental factors to modulate the activity of antimicrobial peptides</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1858</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2015.12.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2015.12.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=26751595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvFyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1858&publication_year=2016&pages=926-935&author=W.+F.+Walkenhorst&title=Using+adjuvants+and+environmental+factors+to+modulate+the+activity+of+antimicrobial+peptides&doi=10.1016%2Fj.bbamem.2015.12.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Using adjuvants and environmental factors to modulate the activity of antimicrobial peptides</span></div><div class="casAuthors">Walkenhorst, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1858</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">926-935</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The increase in antibiotic resistant and multi-drug resistant bacterial infections has serious implications for the future of health care.  The difficulty in finding both new microbial targets and new drugs against existing targets adds to the concern.  The use of combination and adjuvant therapies are potential strategies to counter this threat.  Antimicrobial peptides (AMPs) are a promising class of antibiotics (ABs), particularly for topical and surface applications.  Efforts have been directed toward a no. of strategies, including the use of conventional ABs combined with AMPs, and the use of potentiating agents to increase the performance of AMPs.  This review focuses on combination strategies such as adjuvants and the manipulation of environmental variables to improve the efficacy of AMPs as potential therapeutic agents.  This article is part of a Special Issue entitled: Antimicrobial peptides edited by Karl Lohner and Kai Hilpert.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZKfoFuXLBrVg90H21EOLACvtfcHk0lhnBgpSBDYE4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvFyhtw%253D%253D&md5=efa08e283dd66737a3dc2bfe635e2b5a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2015.12.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2015.12.034%26sid%3Dliteratum%253Aachs%26aulast%3DWalkenhorst%26aufirst%3DW.%2BF.%26atitle%3DUsing%2520adjuvants%2520and%2520environmental%2520factors%2520to%2520modulate%2520the%2520activity%2520of%2520antimicrobial%2520peptides%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2016%26volume%3D1858%26spage%3D926%26epage%3D935%26doi%3D10.1016%2Fj.bbamem.2015.12.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroux, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauthier, M. A.</span></span> <span> </span><span class="NLM_article-title">Releasable and traceless pegylation of arginine-rich antimicrobial peptides</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4082</span>– <span class="NLM_lpage">4086</span>, <span class="refDoi"> DOI: 10.1039/C7SC00770A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1039%2FC7SC00770A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=30155213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVakur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4082-4086&author=Y.+Gongauthor=D.+Andinaauthor=S.+Naharauthor=J.+C.+Lerouxauthor=M.+A.+Gauthier&title=Releasable+and+traceless+pegylation+of+arginine-rich+antimicrobial+peptides&doi=10.1039%2FC7SC00770A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Releasable and traceless PEGylation of arginine-rich antimicrobial peptides</span></div><div class="casAuthors">Gong, Y.; Andina, D.; Nahar, S.; Leroux, J.-C.; Gauthier, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4082-4086</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Arginine-rich antimicrobial peptides (AMPs) are emerging therapeutics of interest.  However, their applicability is limited by their short circulation half-life, caused in part by their small size and digestion by blood proteases.  This study reports a strategy to temporarily mask arginine residues within AMPs with methoxy poly(ethylene glycol).  Based on the reagent used, release of AMPs occurred in hours to days in a completely traceless fashion.  In vitro, conjugates were insensitive to serum proteases, and released native AMP with full in vitro bioactivity.  This strategy is thus highly relevant and should be adaptable to the entire family of arginine-rich AMPs.  It may potentially be used to improve AMP-therapies by providing a more steady concn. of AMP in the blood after a single injection, avoiding toxic effects at high AMP doses, and reducing the no. of doses required over the treatment duration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjON13iKApULVg90H21EOLACvtfcHk0lj875QkZusk0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVakur4%253D&md5=ce26e313c6d8779e5c48a86d80eb47ef</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2FC7SC00770A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC00770A%26sid%3Dliteratum%253Aachs%26aulast%3DGong%26aufirst%3DY.%26aulast%3DAndina%26aufirst%3DD.%26aulast%3DNahar%26aufirst%3DS.%26aulast%3DLeroux%26aufirst%3DJ.%2BC.%26aulast%3DGauthier%26aufirst%3DM.%2BA.%26atitle%3DReleasable%2520and%2520traceless%2520pegylation%2520of%2520arginine-rich%2520antimicrobial%2520peptides%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D4082%26epage%3D4086%26doi%3D10.1039%2FC7SC00770A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, P. K. J.</span></span> <span> </span><span class="NLM_article-title">Cholesterol, lanosterol, and ergosterol attenuate the membrane association of LL-37(W27F) and temporin L</span>. <i>Biochim. Biophys. Acta-Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1778</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1466</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2008.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2008.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=18358828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVaqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1778&publication_year=2008&pages=1460-1466&author=R.+Soodauthor=P.+K.+J.+Kinnunen&title=Cholesterol%2C+lanosterol%2C+and+ergosterol+attenuate+the+membrane+association+of+LL-37%28W27F%29+and+temporin+L&doi=10.1016%2Fj.bbamem.2008.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Cholesterol, lanosterol, and ergosterol attenuate the membrane association of LL-37(W27F) and temporin L</span></div><div class="casAuthors">Sood, Rohit; Kinnunen, Paavo K. J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1778</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1460-1466</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sterols impart significant changes to the biophys. properties of lipid bilayers.  In this regard the impact of cholesterol on membrane organization and dynamics is particularly well documented and serves for comparison with other sterols.  However, the factors underlying the mol. evolution of cholesterol remain enigmatic.  To this end, cholesterol attenuates membrane perturbation by the so-called antimicrobial peptides (AMPs), produced ubiquitously by eukaryotic cells to combat bacterial infections by compromising the permeability barrier function of the microbial target membranes.  In the present study, we addressed the effects of cholesterol, ergosterol, and lanosterol on the membrane assocn. of two structurally and functionally diverse AMPs viz.  LL-37(F27W) and temporin L (TemL) using fluorescence spectroscopy.  Interestingly, sterol concn. dependent effects on the membrane assocn. of these peptides were obsd.  At XSterol = 0.5 cholesterol was most effective in reducing the membrane intercalation of both LL-37(F27W) and TemL, the corresponding efficiencies of the three sterols decreasing as cholesterol > lanosterol ≥ ergosterol, and cholesterol > lanosterol > ergosterol.  It is conceivable that part of the selection pressure for the chem. evolution of cholesterol may have derived from the ability to protect the AMP-secreting host cell from the membrane damaging action of the antimicrobial peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJqHhuljTss7Vg90H21EOLACvtfcHk0liGX4zggpOpZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVaqurw%253D&md5=b108232047a4afe0e02c667960163641</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2008.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2008.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DSood%26aufirst%3DR.%26aulast%3DKinnunen%26aufirst%3DP.%2BK.%2BJ.%26atitle%3DCholesterol%252C%2520lanosterol%252C%2520and%2520ergosterol%2520attenuate%2520the%2520membrane%2520association%2520of%2520LL-37%2528W27F%2529%2520and%2520temporin%2520L%26jtitle%3DBiochim.%2520Biophys.%2520Acta-Biomembr.%26date%3D2008%26volume%3D1778%26spage%3D1460%26epage%3D1466%26doi%3D10.1016%2Fj.bbamem.2008.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillies, E. R.</span></span> <span> </span><span class="NLM_article-title">Amphipathic beta-strand mimics as potential membrane disruptive antibiotics</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">5953</span>– <span class="NLM_lpage">5960</span>, <span class="refDoi"> DOI: 10.1021/jo900933r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo900933r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2009&pages=5953-5960&author=J.+L.+Watsonauthor=E.+R.+Gillies&title=Amphipathic+beta-strand+mimics+as+potential+membrane+disruptive+antibiotics&doi=10.1021%2Fjo900933r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Amphipathic β-strand mimics as potential membrane disruptive antibiotics</span></div><div class="casAuthors">Watson, Jessica L.; Gillies, Elizabeth R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5953-5960</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, there have been increasing nos. of bacterial strains emerging that are resistant to the currently available antibiotics.  In the search for new antibiotics, attention has been focused on natural antimicrobial peptides that act by selectively disrupting the membranes of bacterial cells, a mechanism that is thought to be nonconducive to the development of resistance.  It is desirable to mimic the structures and activities of these peptides while introducing properties such as resistance to proteolytic degrdn., which make mols. more ideal for development as drugs.  Described here is the design and synthesis of β-strand mimetic oligomers based on alternating α-amino acids and azacyclohexenone units that segregate cationic lysine and hydrophobic valine side chains on opposite faces of the β-strand.  1H NMR diln. studies demonstrated that despite the incorporation of alternating D- and L-amino acids in order to obtain facial amphiphilicity, these oligomers are capable of dimerizing to β-sheet mimics in a manner similar to the oligomers contg. all L-amino acids.  The ability of the mols. to disrupt phospholipid vesicles mimicking the membranes of both bacterial and mammalian cells was investigated using a fluorescent dye leakage assay.  Several of the oligomers were found to exhibit activity and selectivity for the bacterial over mammalian membranes.  Overall, these studies demonstrate the promise of this class of mols. for the development of new potential antibiotics and provide information on the structural features that are important for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow49wha4sQNbVg90H21EOLACvtfcHk0liGX4zggpOpZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXoslCntb0%253D&md5=b561a4ad3b63ff3c4f403f81f2c59a37</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjo900933r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo900933r%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DJ.%2BL.%26aulast%3DGillies%26aufirst%3DE.%2BR.%26atitle%3DAmphipathic%2520beta-strand%2520mimics%2520as%2520potential%2520membrane%2520disruptive%2520antibiotics%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2009%26volume%3D74%26spage%3D5953%26epage%3D5960%26doi%3D10.1021%2Fjo900933r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Design of imperfectly amphipathic alpha-helical antimicrobial peptides with enhanced cell selectivity</span>. <i>Acta Biomater.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">244</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1016/j.actbio.2013.08.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.actbio.2013.08.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=24021230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCitrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=244-257&author=X.+Zhuauthor=N.+Dongauthor=Z.+Wangauthor=Z.+Maauthor=L.+Zhangauthor=Q.+Maauthor=A.+Shan&title=Design+of+imperfectly+amphipathic+alpha-helical+antimicrobial+peptides+with+enhanced+cell+selectivity&doi=10.1016%2Fj.actbio.2013.08.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design of imperfectly amphipathic α-helical antimicrobial peptides with enhanced cell selectivity</span></div><div class="casAuthors">Zhu, Xin; Dong, Na; Wang, Zeyun; Ma, Zhi; Zhang, Licong; Ma, Qingquan; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biomaterialia</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">244-257</span>CODEN:
                <span class="NLM_cas:coden">ABCICB</span>;
        ISSN:<span class="NLM_cas:issn">1742-7061</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs), which are produced by multicellular organisms as a defense mechanism against competing pathogenic microbes, appear to be excellent candidates for the development of novel antimicrobial agents.  Amphipathicity is traditionally believed to be crucial to the de novo design or systematic optimization of AMPs.  In this study, we designed a series of short α-helical AMPs with imperfect amphipathicity to augment the arsenal of strategies and to gain further insights into their antimicrobial and hemolytic activity.  These imperfectly amphipathic α-helical AMPs were designed by replacing the paired charged amino acid residues on the polar face of an amphipathic peptide with tryptophan residues on the basis of α-helical protein folding principles.  PRW4, an imperfectly amphipathic α-helical AMP with hydrogen bonds formed by paired tryptophan residues, was obsd. to be more selective towards bacterial cells than toward human red blood cells.  PRW4 was also effective against Gram-neg. and Gram-pos. bacteria, and fluorescence spectroscopy, flow cytometry, SEM and transmission electron microscopy indicated that PRW4 killed microbial cells by permeabilizing the cell membrane and damaging their membrane integrity.  Therefore, disruptive amphipathicity has excellent potential for the rational design and optimization of AMPs with promising antimicrobial activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphnDG1okHKT7Vg90H21EOLACvtfcHk0ljn_CZI_qklKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCitrrM&md5=9c175868f253e06a4fca1c415dcbaba2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.actbio.2013.08.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.actbio.2013.08.043%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DDesign%2520of%2520imperfectly%2520amphipathic%2520alpha-helical%2520antimicrobial%2520peptides%2520with%2520enhanced%2520cell%2520selectivity%26jtitle%3DActa%2520Biomater.%26date%3D2014%26volume%3D10%26spage%3D244%26epage%3D257%26doi%3D10.1016%2Fj.actbio.2013.08.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span> <span> </span><span class="NLM_article-title">Antibacterial activities of short designer peptides: a link between propensity for nanostructuring and capacity for membrane destabilization</span>. <i>Biomacromolecules</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">402</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1021/bm901130u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bm901130u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFWiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=402-411&author=C.+Chenauthor=F.+Panauthor=S.+Zhangauthor=J.+Huauthor=M.+Caoauthor=J.+Wangauthor=H.+Xuauthor=X.+Zhaoauthor=J.+Lu&title=Antibacterial+activities+of+short+designer+peptides%3A+a+link+between+propensity+for+nanostructuring+and+capacity+for+membrane+destabilization&doi=10.1021%2Fbm901130u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial Activities of Short Designer Peptides: a Link between Propensity for Nanostructuring and Capacity for Membrane Destabilization</span></div><div class="casAuthors">Chen, Cuixia; Pan, Fang; Zhang, Shengzhong; Hu, Jing; Cao, Meiwen; Wang, Jing; Xu, Hai; Zhao, Xiubo; Lu, Jian R.</div><div class="citationInfo"><span class="NLM_cas:title">Biomacromolecules</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">402-411</span>CODEN:
                <span class="NLM_cas:coden">BOMAF6</span>;
        ISSN:<span class="NLM_cas:issn">1525-7797</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Amphiphilic peptides A3K, A6K, and A9K displayed an increasing propensity for nanoaggregation with increasing the size of hydrophobic alanine moiety, and the size and shape of the aggregates showed a steady transition from loose peptide stacks formed by A3K, long nanofibers by A6K, to short and narrow nanorods by A9K.  This size and shape transition was broadly consistent with the trend predicted from interfacial packing and curvature change if these peptide surfactants were treated as conventional surfactants.  The antibacterial capacity, defined by the killing of percentage of bacteria in a given time and peptide concn., showed a strong correlation to peptide hydrophobicity, evident from both microscopic and fluorescence imaging studies.  For A9K, the power for membrane permeation and bacterial clustering intensified with peptide concn. and incubation time.  These results thus depict a pos. correlation between the propensity for self-assembly of the peptides, their membrane penetration power, and bactericidal capacity.  Although the exposure of A9K to a preformed DPPC membrane bilayer showed little structural disturbance, the same treatment to the preformed DPPG membrane bilayer led to substantial disruption of model membrane structure, a trend entirely consistent with the high selectivity obsd. from membrane hemolytic studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGods7GafznUrrVg90H21EOLACvtfcHk0lgTdljHc3hnfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFWiuw%253D%253D&md5=d20b33143f944bf25c592a76c1b4a310</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fbm901130u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbm901130u%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%26aulast%3DPan%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DAntibacterial%2520activities%2520of%2520short%2520designer%2520peptides%253A%2520a%2520link%2520between%2520propensity%2520for%2520nanostructuring%2520and%2520capacity%2520for%2520membrane%2520destabilization%26jtitle%3DBiomacromolecules%26date%3D2010%26volume%3D11%26spage%3D402%26epage%3D411%26doi%3D10.1021%2Fbm901130u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides with high proteolytic resistance for combating gram-negative bacteria</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2286</span>– <span class="NLM_lpage">2304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01348</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01348" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2lsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2286-2304&author=J.+Wangauthor=J.+Songauthor=Z.+Yangauthor=S.+Heauthor=Y.+Yangauthor=X.+Fengauthor=X.+Douauthor=A.+Shan&title=Antimicrobial+peptides+with+high+proteolytic+resistance+for+combating+gram-negative+bacteria&doi=10.1021%2Facs.jmedchem.8b01348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial Peptides with High Proteolytic Resistance for Combating Gram-Negative Bacteria</span></div><div class="casAuthors">Wang, Jiajun; Song, Jing; Yang, Zhanyi; He, Shiqi; Yang, Yi; Feng, Xingjun; Dou, Xiujing; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2286-2304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poor proteolytic resistance is an urgent problem to be solved in the clin. application of antimicrobial peptides (AMPs), yet common solns., such as complicated chem. modifications and utilization of D-amino acids, greatly increase the difficulty and cost of producing AMPs.  In this work, a set of novel peptides was synthesized based on an antitrypsin/antichymotrypsin hydrolytic peptide structure unit (XYPX)n (X represents I, L, and V; Y represents R and K), which was designed using a systematic natural amino acid arrangement.  Of these peptides, 1 of them with 7 repeat units had the highest av. selectivity index (GMSI = 99.07) for all of the Gram-neg. bacteria tested and remained highly effective in combating Escherichia coli infection in vivo.  Importantly, the same peptide also had dramatic resistance to a high concn. of trypsin/chymotrypsin hydrolysis and exerted bactericidal activity through a membrane-disruptive mechanism.  Overall, these findings provide new approaches for the development of antiprotease hydrolytic peptides that target Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5SNyY4JzKxLVg90H21EOLACvtfcHk0lhZ5rSnt0BidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2lsrs%253D&md5=0a5b9a7741b7b397b9f15e1d9ac30970</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01348%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DDou%26aufirst%3DX.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DAntimicrobial%2520peptides%2520with%2520high%2520proteolytic%2520resistance%2520for%2520combating%2520gram-negative%2520bacteria%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2286%26epage%3D2304%26doi%3D10.1021%2Facs.jmedchem.8b01348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuzaki, K.</span></span> <span> </span><span class="NLM_article-title">Action mechanism of pegylated magainin 2 analogue peptide</span>. <i>Biochim. Biophys. Acta-Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>1768</i></span>,  <span class="NLM_fpage">2578</span>– <span class="NLM_lpage">2585</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2007.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2007.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=17662233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGrsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1768&publication_year=2007&pages=2578-2585&author=Y.+Imuraauthor=M.+Nishidaauthor=K.+Matsuzaki&title=Action+mechanism+of+pegylated+magainin+2+analogue+peptide&doi=10.1016%2Fj.bbamem.2007.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Action mechanism of PEGylated magainin 2 analogue peptide</span></div><div class="casAuthors">Imura, Yuichi; Nishida, Minoru; Matsuzaki, Katsumi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1768</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2578-2585</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PEGylation is frequently used to improve the efficacy of protein and peptide drugs.  Recently, we investigated its effects on the action mechanism of the cyclic β-sheet antimicrobial peptide tachyplesin I isolated from Tachypleus tridentatus [Y. Imura, M. Nishida, Y. Ogawa, Y. Takakura, K. Matsuzaki, Action Mechanism of Tachyplesin I and Effects of PEGylation, Biochim. Biophys. Acta 1768 (2007) 1160-1169].  PEGylation did not change the basic mechanism behind the membrane-permeabilizing effect of the peptide on liposomes, however, it decreased the antimicrobial activity and cytotoxicity.  To obtain further information on the effects of PEGylation on the activities of antimicrobial peptides, we designed another structurally different PEGylated antimicrobial peptide (PEG-F5W, E19Q-magainin 2-amide) based on the α-helical peptide magainin 2 isolated from the African clawed frog Xenopus laevis.  The PEGylated peptide induced the leakage of calcein from egg yolk L-α-phosphatidylglycerol/egg yolk L-α-phosphatidylcholine large unilamellar vesicles, however, the activity was weaker than that of the control peptides.  The PEGylated peptide induced lipid flip-flop coupled to the leakage and was translocated into the inner leaflet of the bilayer, indicating that PEGylation did not alter the basic mechanism of membrane permeabilization of the parent peptide.  The cytotoxicity of the non-PEGylated peptides was nullified by PEGylation.  At the same time, the antimicrobial activity was weakened only by 4 fold.  The effects of PEGylation on the activity of magainin were compared with those for tachyplesin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorfObvUcCooLVg90H21EOLACvtfcHk0lhZ5rSnt0BidA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGrsrrF&md5=1a494f20cb6d22c7df1e87ef16adcef3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2007.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2007.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DImura%26aufirst%3DY.%26aulast%3DNishida%26aufirst%3DM.%26aulast%3DMatsuzaki%26aufirst%3DK.%26atitle%3DAction%2520mechanism%2520of%2520pegylated%2520magainin%25202%2520analogue%2520peptide%26jtitle%3DBiochim.%2520Biophys.%2520Acta-Biomembr.%26date%3D2007%26volume%3D1768%26spage%3D2578%26epage%3D2585%26doi%3D10.1016%2Fj.bbamem.2007.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heersema, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peppas, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, H. D. C.</span></span> <span> </span><span class="NLM_article-title">Optimization of cationic nanogel pegylation to achieve mammalian cytocompatibility with limited loss of gram-negative bactericidal activity</span>. <i>Biomacromolecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1528</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1021/acs.biomac.0c00081</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biomac.0c00081" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltl2ntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=1528-1538&author=J.+Guauthor=J.+R.+Cleggauthor=L.+A.+Heersemaauthor=N.+A.+Peppasauthor=H.+D.+C.+Smyth&title=Optimization+of+cationic+nanogel+pegylation+to+achieve+mammalian+cytocompatibility+with+limited+loss+of+gram-negative+bactericidal+activity&doi=10.1021%2Facs.biomac.0c00081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Cationic Nanogel PEGylation to Achieve Mammalian Cytocompatibility with Limited Loss of Gram-Negative Bactericidal Activity</span></div><div class="casAuthors">Gu, Joann; Clegg, John R.; Heersema, Lara A.; Peppas, Nicholas A.; Smyth, Hugh D. C.</div><div class="citationInfo"><span class="NLM_cas:title">Biomacromolecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1528-1538</span>CODEN:
                <span class="NLM_cas:coden">BOMAF6</span>;
        ISSN:<span class="NLM_cas:issn">1525-7797</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tuning the compn. of antimicrobial nanogels can significantly alter both nanogel cytotoxicity and antibacterial activity.  This project investigated the extent to which PEGylation of cationic, hydrophobic nanogels altered their cytotoxicity and bactericidal activity.  These biodegradable, cationic nanogels were synthesized by activators regenerated by electron transfer atom transfer radical polymn. (ARGET ATRP) emulsion polymn. with up to 13.9 wt % PEG (MW = 2000) MA, as verified by 1H NMR.  Nanogel bactericidal activity was assessed against Gram-neg. E. coli and P. aeruginosa and Gram-pos. S. mutans and S. aureus by measuring membrane lysis with a LIVE/DEAD assay.  E. coli and S. mutans viability was further validated by measuring metabolic activity with a PrestoBlue assay and imaging bacteria stained with a LIVE/DEAD probe.  All tested nanogels decreased the membrane integrity (0.5 mg/mL dose) for Gram-neg. E. coli and P. aeruginosa, irresp. of the extent of PEGylation.  PEGylation (13.9 wt %) increased the cytocompatibility of cationic nanogels toward RAW 264.7 murine macrophages and L929 murine fibroblasts by over 100-fold, relative to control nanogels.  PEGylation (42.8 wt %) reduced nanogel uptake by 43% for macrophages and 63% for fibroblasts.  Therefore, PEGylation reduced nanogel toxicity to mammalian cells without significantly compromising their bactericidal activity.  These results facilitate future nanogel design for perturbing the growth of Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0i7kZSpsc17Vg90H21EOLACvtfcHk0li9mhv8jwv7SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltl2ntLc%253D&md5=a5cfee9799d97194604eaf1b2efaf859</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.biomac.0c00081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biomac.0c00081%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DClegg%26aufirst%3DJ.%2BR.%26aulast%3DHeersema%26aufirst%3DL.%2BA.%26aulast%3DPeppas%26aufirst%3DN.%2BA.%26aulast%3DSmyth%26aufirst%3DH.%2BD.%2BC.%26atitle%3DOptimization%2520of%2520cationic%2520nanogel%2520pegylation%2520to%2520achieve%2520mammalian%2520cytocompatibility%2520with%2520limited%2520loss%2520of%2520gram-negative%2520bactericidal%2520activity%26jtitle%3DBiomacromolecules%26date%3D2020%26volume%3D21%26spage%3D1528%26epage%3D1538%26doi%3D10.1021%2Facs.biomac.0c00081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chess, R. B.</span></span> <span> </span><span class="NLM_article-title">Effect of pegylation on pharmaceuticals</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nrd1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1038%2Fnrd1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=12612647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=214-221&author=J.+M.+Harrisauthor=R.+B.+Chess&title=Effect+of+pegylation+on+pharmaceuticals&doi=10.1038%2Fnrd1033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of PEGylation on pharmaceuticals</span></div><div class="casAuthors">Harris, J. Milton; Chess, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">214-221</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein and peptide drugs hold great promise as therapeutic agents.  However, many are degraded by proteolytic enzymes, can be rapidly cleared by the kidneys, generate neutralizing antibodies and have a short circulating half-life.  PEGylation, the process by which polyethylene glycol chains are attached to protein and peptide drugs, can overcome these and other shortcomings.  By increasing the mol. mass of proteins and peptides and shielding them from proteolytic enzymes, PEGylation improves pharmacokinetics.  This article will review how PEGylation can result in drugs that are often more effective and safer, and which show improved patient convenience and compliance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLhPAuz8WjMLVg90H21EOLACvtfcHk0li9mhv8jwv7SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLs%253D&md5=d6bb387e746740b10e088e1f55af37db</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnrd1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1033%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DJ.%2BM.%26aulast%3DChess%26aufirst%3DR.%2BB.%26atitle%3DEffect%2520of%2520pegylation%2520on%2520pharmaceuticals%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D214%26epage%3D221%26doi%3D10.1038%2Fnrd1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salhanick, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clairmont, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckholz, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span> <span> </span><span class="NLM_article-title">Contribution of site-specific pegylation to the dipeptidyl peptidase IV stability of glucose-dependent insulinotropic polypeptide</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">4114</span>– <span class="NLM_lpage">4117</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bmcl.2005.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=15993590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2MXns1SgtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=4114-4117&author=A.+I.+Salhanickauthor=K.+B.+Clairmontauthor=T.+M.+Buckholzauthor=C.+M.+Pellegrinoauthor=S.+Haauthor=K.+J.+Lumb&title=Contribution+of+site-specific+pegylation+to+the+dipeptidyl+peptidase+IV+stability+of+glucose-dependent+insulinotropic+polypeptide&doi=10.1016%2Fj.bmcl.2005.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of site-specific PEGylation to the dipeptidyl peptidase IV stability of glucose-dependent insulinotropic polypeptide</span></div><div class="casAuthors">Salhanick, Arthur I.; Clairmont, Kevin B.; Buckholz, Thomas M.; Pellegrino, Carla M.; Ha, Sha; Lumb, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4114-4117</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The effects of PEGylation of glucose-dependent insulinotropic polypeptide (GIP) on potency and dipeptidyl peptidase IV (DPPIV) stability are reported.  N-terminal modification of GIP(1-30) with 40 kDa polyethylene glycol (PEG) abrogates functional activity.  In contrast, C-terminal PEGylation of GIP(1-30) maintains full agonism and reasonable potency at the GIP receptor and confers a high level of DPPIV resistance.  Moreover, the dual modification of N-terminal palmitoylation and C-terminal PEGylation results in a full agonist of comparable potency to native GIP that is stable to DPPIV cleavage.  The results provide the basis for the development of long acting, PEGylated GIP, GIP variants, or GIP-based hybrid peptide therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsENy8kGz-IrVg90H21EOLACvtfcHk0li9mhv8jwv7SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXns1SgtL0%253D&md5=8123e84e7bf70de96ef27311ad284afa</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DSalhanick%26aufirst%3DA.%2BI.%26aulast%3DClairmont%26aufirst%3DK.%2BB.%26aulast%3DBuckholz%26aufirst%3DT.%2BM.%26aulast%3DPellegrino%26aufirst%3DC.%2BM.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26atitle%3DContribution%2520of%2520site-specific%2520pegylation%2520to%2520the%2520dipeptidyl%2520peptidase%2520IV%2520stability%2520of%2520glucose-dependent%2520insulinotropic%2520polypeptide%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D4114%26epage%3D4117%26doi%3D10.1016%2Fj.bmcl.2005.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pasut, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, F. M.</span></span> <span> </span><span class="NLM_article-title">Polymer-drug conjugation, recent achievements and general strategies</span>. <i>Prog. Polym. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1016/j.progpolymsci.2007.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.progpolymsci.2007.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFeksLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2007&pages=933-961&author=G.+Pasutauthor=F.+M.+Veronese&title=Polymer-drug+conjugation%2C+recent+achievements+and+general+strategies&doi=10.1016%2Fj.progpolymsci.2007.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Polymer-drug conjugation, recent achievements and general strategies</span></div><div class="casAuthors">Pasut, G.; Veronese, F. M.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Polymer Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">933-961</span>CODEN:
                <span class="NLM_cas:coden">PRPSB8</span>;
        ISSN:<span class="NLM_cas:issn">0079-6700</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The field of drug delivery is fast expanding and its potentials have already been proved by the many products in the market.  Among all approaches, polymer conjugation is a well known and widely exploited technique useful to improve therapeutic properties of peptides, proteins, small mols. or oligonucleotides.  Polymer-conjugated drugs generally exhibit prolonged half-life, higher stability, water soly., lower immunogenicity and antigenicity and often also specific targeting to tissues or cells.  This technol., exploited for the first time in the 1950s and 1960s, received a great development both for the introduction and study of new and different polymers and for the progresses in the chem. strategies of coupling.  Polymer-drug conjugates are already in the market for the treatment of different diseases, demonstrating the potentials of the technol.  Furthermore, new polymers, in addn. to the most known N-(2-hydroxypropyl)methacrylamide copolymer (HPMA), polyglutamic acid (PGA) and poly(ethylene glycol) (PEG), are continuously investigated and proposed.  The review will discuss the most recent achievements in polymer conjugation with special emphasis on PEG application strategies and approved products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowAFlEG0CRKbVg90H21EOLACvtfcHk0lhVvVo7DLVHDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFeksLc%253D&md5=263b82b9257683f2a922d9b8014aeb55</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.progpolymsci.2007.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.progpolymsci.2007.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DPasut%26aufirst%3DG.%26aulast%3DVeronese%26aufirst%3DF.%2BM.%26atitle%3DPolymer-drug%2520conjugation%252C%2520recent%2520achievements%2520and%2520general%2520strategies%26jtitle%3DProg.%2520Polym.%2520Sci.%26date%3D2007%26volume%3D32%26spage%3D933%26epage%3D961%26doi%3D10.1016%2Fj.progpolymsci.2007.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Rational avoidance of protease cleavage sites and symmetrical end-tagging significantly enhances the stability and therapeutic potential of antimicrobial peptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9421</span>– <span class="NLM_lpage">9435</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9421-9435&author=Y.+Zhuauthor=C.+Shaoauthor=G.+Liauthor=Z.+Laiauthor=P.+Tanauthor=Q.+Jianauthor=B.+Chengauthor=A.+Shan&title=Rational+avoidance+of+protease+cleavage+sites+and+symmetrical+end-tagging+significantly+enhances+the+stability+and+therapeutic+potential+of+antimicrobial+peptides&doi=10.1021%2Facs.jmedchem.0c00583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Avoidance of Protease Cleavage Sites and Symmetrical End-Tagging Significantly Enhances the Stability and Therapeutic Potential of Antimicrobial Peptides</span></div><div class="casAuthors">Zhu, Yongjie; Shao, Changxuan; Li, Guoyu; Lai, Zhenheng; Tan, Peng; Jian, Qiao; Cheng, Baojing; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9421-9435</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs) are expected to solve problem of antibiotic resistance because of their distinctive nonspecific membrane-disruptive mechanism.  However, clin. applications of AMPs have been precluded by their poor stability, although various complex chem. strategies have been employed to solve this problem, and this undoubtedly greatly increases the manufg. cost.  Herein, a series of novel peptides with high stability were developed based on protease-specific cleavage sites and sym. end-tagging.  Among these peptides, II-I4-II exhibited the best antibacterial activity and the highest therapeutic index.  More importantly, II-I4-II showed extremely high stability in the presence of various proteases, physiol. salts and serum, and under acid, alkali, and heat conditions, and it exhibited excellent therapeutic potential in vivo.  Addnl., II-I4-II exhibited a membrane-disruptive mechanism and a low propensity to induce resistance.  In general, these findings contribute to the design of AMPs with high stability and might accelerate clin. applications of AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0eoqZgwqxnbVg90H21EOLACvtfcHk0lhVvVo7DLVHDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmtbvI&md5=1eaf97e703bd9b16a562cc860b3d39e3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00583%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DJian%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DB.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DRational%2520avoidance%2520of%2520protease%2520cleavage%2520sites%2520and%2520symmetrical%2520end-tagging%2520significantly%2520enhances%2520the%2520stability%2520and%2520therapeutic%2520potential%2520of%2520antimicrobial%2520peptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9421%26epage%3D9435%26doi%3D10.1021%2Facs.jmedchem.0c00583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Combating drug-resistant fungi with novel imperfectly amphipathic palindromic peptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3889</span>– <span class="NLM_lpage">3907</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01729</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01729" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFShu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3889-3907&author=J.+Wangauthor=S.+Chouauthor=Z.+Yangauthor=Y.+Yangauthor=Z.+Wangauthor=J.+Songauthor=X.+Douauthor=A.+Shan&title=Combating+drug-resistant+fungi+with+novel+imperfectly+amphipathic+palindromic+peptides&doi=10.1021%2Facs.jmedchem.7b01729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Drug-Resistant Fungi with Novel Imperfectly Amphipathic Palindromic Peptides</span></div><div class="casAuthors">Wang, Jiajun; Chou, Shuli; Yang, Zhanyi; Yang, Yang; Wang, Zhihua; Song, Jing; Dou, Xiujing; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3889-3907</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antimicrobial peptides are an important weapon against invading pathogens and are potential candidates as novel antibacterial agents, but their antifungal activities are not fully developed.  In this study, a set of imperfectly amphipathic peptides was developed based on the imperfectly amphipathic palindromic structure Rn(XRXXXRX)Rn (n = 1,2; X represents L, I, F, or W), and the engineered peptides exhibited high antimicrobial activities against all fungi and bacteria tested (including fluconazole-resistant Candida albicans), with geometric mean (GM) MICs ranging from 2.2 to 6.62μM.  Of such peptides, 13 (I6) (RRIRIIIRIRR-NH2) that was Ile rich in its hydrophobic face had the highest antifungal activity (GMfungi =1.64μM), meanwhile showing low toxicity and high salt and serum tolerance.  It also had dramatic LPS-neutralizing propensity and a potent membrane-disruptive mechanism against microbial cells.  In summary, these findings were useful for short AMPs design to combat the growing threat of drug-resistant fungal and bacterial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXoGkyENHFN7Vg90H21EOLACvtfcHk0lhVvVo7DLVHDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFShu7k%253D&md5=c5806d09b3beb6deb2c806c45333e973</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01729%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChou%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DX.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DCombating%2520drug-resistant%2520fungi%2520with%2520novel%2520imperfectly%2520amphipathic%2520palindromic%2520peptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3889%26epage%3D3907%26doi%3D10.1021%2Facs.jmedchem.7b01729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of trp-rich engineered amphiphiles by single hydrophobic amino acid end-tagging</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">43820</span>– <span class="NLM_lpage">43834</span>, <span class="refDoi"> DOI: 10.1021/acsami.9b12706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.9b12706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFSqtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=43820-43834&author=J.+Songauthor=J.+Wangauthor=N.+Zhanauthor=T.+Sunauthor=W.+Yuauthor=L.+Zhangauthor=A.+Shanauthor=A.+Zhang&title=Therapeutic+potential+of+trp-rich+engineered+amphiphiles+by+single+hydrophobic+amino+acid+end-tagging&doi=10.1021%2Facsami.9b12706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Potential of Trp-Rich Engineered Amphiphiles by Single Hydrophobic Amino Acid End-Tagging</span></div><div class="casAuthors">Song, Jing; Wang, Jiajun; Zhan, Na; Sun, Taotao; Yu, Weikang; Zhang, Licong; Shan, Anshan; Zhang, Aizhong</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">43820-43834</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">End-tagging with a single hydrophobic residue contributes to improve the cell selectivity of antimicrobial peptides (AMPs), but systematic studies have been lacking.  Thus, this study aimed to systematically investigate how end-tagging with hydrophobic residues at the C-terminus and Gly capped at the N-terminus of W4 (RWRWWWRWR) affects the bioactivity of W4 variants.  Among all the hydrophobic residues, only Ala end-tagging improved the antibacterial activity of W4.  Meanwhile, Gly capped at the N-terminus could promote the helical propensity of the end-tagged peptides in dodecylphosphocholine micelles, increasing their antimicrobial activities.  Of these peptides, GW4A (GRWRWWWRWRA) showed the best antibacterial activity against the 19 species of bacteria tested (GMMIC = 1.86 μM) with low toxicity, thus possessing the highest cell selectivity (TIall = 137.63).  It also had rapid sterilization, good salt and serum resistance, and LPS-neutralizing activity.  Antibacterial mechanism studies showed that the short peptide GW4A killed bacteria by destroying cell membrane integrity and causing cytoplasmic leakage.  Overall, these findings suggested that systematic studies on terminal modifications promoted the development of peptide design theory and provided a potential method for optimization of effective AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6d5P_0TzCoLVg90H21EOLACvtfcHk0lhv9vI1Kc6bWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFSqtL%252FM&md5=f71c5d1ce4892d24299ab1783199df99</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facsami.9b12706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.9b12706%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhan%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DTherapeutic%2520potential%2520of%2520trp-rich%2520engineered%2520amphiphiles%2520by%2520single%2520hydrophobic%2520amino%2520acid%2520end-tagging%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2019%26volume%3D11%26spage%3D43820%26epage%3D43834%26doi%3D10.1021%2Facsami.9b12706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">Highly stabilized alpha-helical coiled coils kill gram-negative bacteria by multicomplementary mechanisms under acidic condition</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">22113</span>– <span class="NLM_lpage">22128</span>, <span class="refDoi"> DOI: 10.1021/acsami.9b04654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.9b04654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2lsb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=22113-22128&author=Z.+Laiauthor=P.+Tanauthor=Y.+Zhuauthor=C.+Shaoauthor=A.+Shanauthor=L.+Li&title=Highly+stabilized+alpha-helical+coiled+coils+kill+gram-negative+bacteria+by+multicomplementary+mechanisms+under+acidic+condition&doi=10.1021%2Facsami.9b04654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Stabilized α-Helical Coiled Coils Kill Gram-Negative Bacteria by Multicomplementary Mechanisms under Acidic Condition</span></div><div class="casAuthors">Lai, Zhenheng; Tan, Peng; Zhu, Yongjie; Shao, Changxuan; Shan, Anshan; Li, Lu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">22113-22128</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although antimicrobial peptides (AMPs) hold tremendous promise in overcoming the threats of multidrug resistance, the main obstacle to successful therapeutic applications is their poor stability.  Various synthetic strategies such as unnatural amino acids and chem. modifications have made advances for improving this problem.  However, this complicated synthesis often greatly increases the cost of prodn.  Here, a series of novel peptides, designed by combining an α-helical coiled coil model, knowledge of the specificity of proteolysis and major parameters of AMPs, exhibited efficient activity against all tested Gram-neg. bacteria under acidic condition and demonstrate low toxicity.  Of these α-helical coiled coil peptides, 3IH3 displayed the highest av. therapeutic index (GMTI = 294.25) with high stability toward salts, serum, extreme pH, heat, and proteases.  Electron microscopy and biol. anal. technique analyses showed that 3IH3 killed bacterial cells via a multicomplementary mechanism at pH 6.0, with phys. membrane disruption as the dominant bactericidal mechanism.  These results suggest that 3IH3 shows great stability as an inexpensive and effective antimicrobial activity agent and has the potential for clin. application in the treatment of infections occurring in body sites with acidic pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDuifidWFC6LVg90H21EOLACvtfcHk0lhv9vI1Kc6bWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2lsb%252FK&md5=584757d80cec28bd24369137f49fb64b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facsami.9b04654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.9b04654%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DHighly%2520stabilized%2520alpha-helical%2520coiled%2520coils%2520kill%2520gram-negative%2520bacteria%2520by%2520multicomplementary%2520mechanisms%2520under%2520acidic%2520condition%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2019%26volume%3D11%26spage%3D22113%26epage%3D22128%26doi%3D10.1021%2Facsami.9b04654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Molecular structural formula of PEG and its derivatives: (A) PEG, (B) mPEG-NH<sub>2</sub>, and (C) mPEG-COOH; design of the engineered peptides: (D) sequence and schematic structure of PG-1; (E) NPG<i><sub>n</sub></i>: X = mPEG<sub>750</sub>-COOH, mPEG<sub>1000</sub>-COOH, and mPEG<sub>2000</sub>-COOH, <i>n</i> = 750 Da, 1000 Da, and 2000 Da; and (F) CPG<i><sub>n</sub></i>: Y = mPEG<sub>750</sub>-NH<sub>2</sub>, mPEG<sub>1000</sub>-NH<sub>2</sub>, and mPEG<sub>2000</sub>-NH<sub>2</sub>, <i>n</i> = 750 Da, 1000 Da, and 2000 Da.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. CD spectra of all peptides. All peptides were dissolved in 1.5 mM PBS (pH 7.4) and 1.5 mM SDS and the mean residual ellipticity was plotted against wavelength. The values from three scans were averaged per sample, and the peptide concentrations were fixed at 25 μg/mL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Biocompatibility of the engineered peptides: (A) hemolytic activities of the peptides against human red blood cells. (B) HEK 293T cells. The data were derived from three independent experiments and presented as the mean ± standard error (SE). *<i>P</i> < 0.01, **<i>P</i> < 0.001, and ***<i>P</i> < 0.0001, Student’s t-test, significant difference from the control group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Time–kill curves of peptides at 1× MIC against <i>E. coli</i>. The data were derived from three independent experiments and presented as mean ± SE. (B) Resistance development of peptides. Exposure of peptides and ampicillin-sensitive strain of <i>E. coli</i> to sub-MIC dosing for 30 sequential passages. The test was carried out three times independently with analogous results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) ANS fluorescence binding assay for NPG<sub>750</sub> and CPG<sub>2000</sub> at different concentrations in deionized water (pH 7.4) and their hydrodynamic size at 25 °C. (B) TEM images and SEM images of engineered peptides. (C) Fold changes of MIC values of peptides in the presence of 8, 4, 2, and 1 mg/mL trypsin. The test was carried out three times independently. (D) 8 mg/mL trypsin and peptides were incubated at 37 °C for 1, 2, 4, and 8 h, respectively. The molecular weight of the marker from top to bottom corresponds to 31, 20.1, 14.4, 6.5, and 3.3 KDa, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Peptide-binding affinity to LPS (<i>E. coli</i> O111:B4). The excitation wavelength was 580 nm and the emission wavelength was 620 nm. The graphs were derived from the average of three independent trials. All the data are expressed as mean ± SD. (B) Cell mortality of <i>E. coli</i> cells as a function of LPS concentration. The concentration of peptides used was fixed at 1× MIC. (C) The OM permeability induced by tested peptides. The fluorescence intensity was monitored at an excitation wavelength of 350 nm and an emission wavelength of 420 nm. (D) Membrane potential osmotic dye DISC<sub>3</sub>-5 was used to detect the effects of (D-l) NPG<sub>750</sub> and (D-2) CPG<sub>2000</sub> on the membrane potential of <i>E. coli</i> cells. The excitation wavelength was 622 nm, the emission wavelength was 670 nm, and the duration was 2000 s. (E) Inner membrane (IM) permeability of peptides. The hydrolysis of <i>o</i>-nitrophenyl-β-<span class="smallcaps smallerCapital">d</span>-galactopyranoside (ONPG) due to the release of cytoplasmic β-galactosidase of <i>E. coli</i> treated by different concentrations of (E-1) NPG<sub>750</sub> and (E-2) CPG<sub>2000</sub>. The duration was 5000 s. (F) Mimic bacterial phospholipid bilayer liposomes. POPC:POPG = 3:1(molar:molar). *<i>P</i> < 0.01, **<i>P</i> < 0.001, and ***<i>P</i> < 0.0001, Student’s t-test, significant difference from the control group. (G) Simulated eukaryotic membrane liposomes. POPC:POPG = 10:1(molar:molar).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. SEM and TEM micrographs of <i>E. coli</i> K88 treated with peptides at 1× MIC. SEM images of (A) control <i>E. coli</i> K88, (B) <i>E. coli</i> K88 treated with PG-1, (C) NPG<sub>750</sub>, and (D) CPG<sub>2000</sub>. (E) TEM image of control <i>E. coli</i> K88; (F) <i>E. coli</i> K88 treated with PG-1 showed that the cytoplasm leaked to form a blank area. (G) NPG<sub>750</sub> caused a marked contraction of the membrane of <i>E. coli</i> K88. (H) CPG<sub>2000</sub> caused bacterial rupture and separation of the plasma membrane and cell wall.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Possible mechanism of NPG<sub>750</sub> and CPG<sub>2000</sub> against <i>E. coli</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/medium/jm1c00879_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Schematic diagram of the antitrypsin mechanism of engineered peptides. Engineered peptides form nanostructures to resist the hydrolysis of trypsin.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00879/20210716/images/large/jm1c00879_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00879&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05001" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05001" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thulshan
Jayathilaka, E. H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liyanage, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapaksha, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dananjaya, S. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikapitiya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Zoysa, M.</span></span> <span> </span><span class="NLM_article-title">Octominin: An antibacterial and anti-biofilm peptide for controlling the multidrug resistance and pathogenic streptococcus parauberis</span>. <i>Fish Shellfish Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1016/j.fsi.2020.12.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.fsi.2020.12.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=33378697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3MXitlSisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2021&pages=23-34&author=E.+H.+T.+Thulshan%0AJayathilakaauthor=T.+D.+Liyanageauthor=D.+C.+Rajapakshaauthor=S.+H.+S.+Dananjayaauthor=C.+Nikapitiyaauthor=I.+Whangauthor=M.+De+Zoysa&title=Octominin%3A+An+antibacterial+and+anti-biofilm+peptide+for+controlling+the+multidrug+resistance+and+pathogenic+streptococcus+parauberis&doi=10.1016%2Fj.fsi.2020.12.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Octominin: An antibacterial and anti-biofilm peptide for controlling the multidrug resistance and pathogenic Streptococcus parauberis</span></div><div class="casAuthors">Thulshan Jayathilaka, E. H. T.; Liyanage, T. D.; Rajapaksha, D. C.; Dananjaya, S. H. S.; Nikapitiya, Chamilani; Whang, Ilson; De Zoysa, Mahanama</div><div class="citationInfo"><span class="NLM_cas:title">Fish & Shellfish Immunology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">FSIMEP</span>;
        ISSN:<span class="NLM_cas:issn">1050-4648</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Streptococcus parauberis is a pathogenic gram-pos. bacterium that causes streptococcosis infection in fish.  Since S. parauberis is becoming resistant to multiple antibiotics, the development of alternatives, such as antimicrobial peptides, has gained great attention.  Octominin, derived from the defense protein of Octopus minor, showed a significant antimicrobial activity against multidrug resistance S. parauberis, with a min. inhibitory concn. (MIC) and a min. bactericidal concn. (MBC) of 50 and 100μg/mL, resp.  Furthermore, time-kill kinetics, agar diffusion, and bacterial viability assays confirmed the concn.-dependent antibacterial activity of Octominin against S. parauberis.  Field emission SEM anal. showed morphol. and ultra-structural changes in S. parauberis upon Octominin treatment.  Moreover, Octominin treatment demonstrated changes in membrane permeability, induced reactive oxygen species (ROS), and its binding ability to genomic DNA, suggesting its strong bactericidal activity with multiple modes of action.  We confirmed the inhibition of biofilm formation and the eradication of existing biofilms in a concn.-dependent manner.  Addnl., Octominin on S. parauberis at transcriptional level exhibited downregulation of membrane formation (pgsA and cds1), DNA repairing (recF), biofilm formation (pgaB and epsF) genes, while upregulation of ROS detoxification (sodA) and DNA protecting (ahpF) related genes.  An in vivo study confirmed a significantly (P < 0.05) higher relative percentage survival in Octominin-treated larval zebrafish exposed to S. parauberis (93.3%) compared to the control group (20.0%).  Collectively, our results confirm that Octominin could be a potential antibacterial and anti-biofilm agent against S. parauberis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA9JweWgCoAbVg90H21EOLACvtfcHk0lgKq1BA0H5NnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXitlSisrc%253D&md5=f548c79f12e9c5a6217461d14cc88bc1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.fsi.2020.12.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fsi.2020.12.017%26sid%3Dliteratum%253Aachs%26aulast%3DThulshan%2BJayathilaka%26aufirst%3DE.%2BH.%2BT.%26aulast%3DLiyanage%26aufirst%3DT.%2BD.%26aulast%3DRajapaksha%26aufirst%3DD.%2BC.%26aulast%3DDananjaya%26aufirst%3DS.%2BH.%2BS.%26aulast%3DNikapitiya%26aufirst%3DC.%26aulast%3DWhang%26aufirst%3DI.%26aulast%3DDe%2BZoysa%26aufirst%3DM.%26atitle%3DOctominin%253A%2520An%2520antibacterial%2520and%2520anti-biofilm%2520peptide%2520for%2520controlling%2520the%2520multidrug%2520resistance%2520and%2520pathogenic%2520streptococcus%2520parauberis%26jtitle%3DFish%2520Shellfish%2520Immunol.%26date%3D2021%26volume%3D110%26spage%3D23%26epage%3D34%26doi%3D10.1016%2Fj.fsi.2020.12.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Sousa, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assane, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos-Filho, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cilli, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jesus, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarski, F.</span></span> <span> </span><span class="NLM_article-title">Haematological, biochemical and immunological biomarkers, antibacterial activity, and survival in nile tilapia oreochromis niloticus after treatment using antimicrobial peptide LL-37 against streptococcus agalactiae</span>. <i>Aquaculture</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>533</i></span>, <span class="NLM_elocation-id">736181</span> <span class="refDoi"> DOI: 10.1016/j.aquaculture.2020.736181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.aquaculture.2020.736181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3MXmsVemug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2021&author=E.+L.+de%0ASousaauthor=I.+M.+Assaneauthor=N.+A.+Santos-Filhoauthor=E.+M.+Cilliauthor=R.+B.+de+Jesusauthor=F.+Pilarski&title=Haematological%2C+biochemical+and+immunological+biomarkers%2C+antibacterial+activity%2C+and+survival+in+nile+tilapia+oreochromis+niloticus+after+treatment+using+antimicrobial+peptide+LL-37+against+streptococcus+agalactiae&doi=10.1016%2Fj.aquaculture.2020.736181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Haematological, biochemical and immunological biomarkers, antibacterial activity, and survival in Nile tilapia Oreochromis niloticus after treatment using antimicrobial peptide LL-37 against Streptococcus agalactiae</span></div><div class="casAuthors">de Sousa, Elielma Lima; Assane, Inacio Mateus; Santos-Filho, Norival Alves; Cilli, Eduardo Maffud; de Jesus, Raphael Barbetta; Pilarski, Fabiana</div><div class="citationInfo"><span class="NLM_cas:title">Aquaculture</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">736181</span>CODEN:
                <span class="NLM_cas:coden">AQCLAL</span>;
        ISSN:<span class="NLM_cas:issn">0044-8486</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">This study aimed to evaluate the antimicrobial activity and therapeutic effectiveness of peptide LL-37 against streptococcosis caused by Streptococcus agalactiae in Nile tilapia and to assess some of its potential adverse effects (in vivo haematol., biochem., and immunol. profiles) in the fish.  After in vitro cytotoxicity studies of LL-37 against Nile tilapia red blood cells (RBCs) and antimicrobial susceptibility testing (min. inhibitory concn. and min. bactericidal concn., MIC and MBC, resp.) against Aeromonas hydrophila and S. agalactiae, fish were divided into six exptl. groups.  G1 and G2 were challenged by i.p. injection with S. agalactiae and received a single oral dose of LL-37 (15 mg/kg bw) and florfenicol (FFC) (15 mg/kg bw), resp.  G3 was challenged with S. agalactiae and received a single i.p. dose of LL-37 (15 mg/kg bw).  The other groups were controls for infection (G4) and antimicrobial administration (G5 and G6).  LL-37 showed weak hemolytic activity against Nile tilapia RBCs (24.4% lysis at 625μg/mL) and strong in vitro antimicrobial activity against S. agalactiae (MIC and MBC = 31.25μg/mL).  No significant effects were obsd. in the haematol., biochem., and immunol. profiles of fish medicated with LL-37 or FFC, compared to the control fish.  Twenty days after the challenge, survival rates were 33.38% (G1), 38.51% (G2), 23.12% (G3), 25% (G4), and 100% (G5 and G6).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmkmE-ERNBwrVg90H21EOLACvtfcHk0ljdN8hPiNaWdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXmsVemug%253D%253D&md5=72acdcce2cb1c998151a9cd333d34c21</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.aquaculture.2020.736181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aquaculture.2020.736181%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BSousa%26aufirst%3DE.%2BL.%26aulast%3DAssane%26aufirst%3DI.%2BM.%26aulast%3DSantos-Filho%26aufirst%3DN.%2BA.%26aulast%3DCilli%26aufirst%3DE.%2BM.%26aulast%3Dde%2BJesus%26aufirst%3DR.%2BB.%26aulast%3DPilarski%26aufirst%3DF.%26atitle%3DHaematological%252C%2520biochemical%2520and%2520immunological%2520biomarkers%252C%2520antibacterial%2520activity%252C%2520and%2520survival%2520in%2520nile%2520tilapia%2520oreochromis%2520niloticus%2520after%2520treatment%2520using%2520antimicrobial%2520peptide%2520LL-37%2520against%2520streptococcus%2520agalactiae%26jtitle%3DAquaculture%26date%3D2021%26volume%3D533%26doi%3D10.1016%2Fj.aquaculture.2020.736181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Cross-strand interaction, central bending, and sequence pattern act as biomodulators of simplified beta-hairpin antimicrobial amphiphiles</span>. <i>Small</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>, <span class="NLM_elocation-id">2003899</span> <span class="refDoi"> DOI: 10.1002/smll.202003899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1002%2Fsmll.202003899" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&author=C.+Shaoauthor=Y.+Zhuauthor=Q.+Jianauthor=Z.+Laiauthor=P.+Tanauthor=G.+Liauthor=A.+Shan&title=Cross-strand+interaction%2C+central+bending%2C+and+sequence+pattern+act+as+biomodulators+of+simplified+beta-hairpin+antimicrobial+amphiphiles&doi=10.1002%2Fsmll.202003899"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fsmll.202003899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fsmll.202003899%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DJian%26aufirst%3DQ.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DCross-strand%2520interaction%252C%2520central%2520bending%252C%2520and%2520sequence%2520pattern%2520act%2520as%2520biomodulators%2520of%2520simplified%2520beta-hairpin%2520antimicrobial%2520amphiphiles%26jtitle%3DSmall%26date%3D2020%26volume%3D17%26doi%3D10.1002%2Fsmll.202003899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides: promising alternatives in the post feeding antibiotic era</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">859</span>, <span class="refDoi"> DOI: 10.1002/med.21542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1002%2Fmed.21542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=30353555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXntlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=831-859&author=J.+Wangauthor=X.+Douauthor=J.+Songauthor=Y.+Lyuauthor=X.+Zhuauthor=L.+Xuauthor=W.+Liauthor=A.+Shan&title=Antimicrobial+peptides%3A+promising+alternatives+in+the+post+feeding+antibiotic+era&doi=10.1002%2Fmed.21542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial peptides: Promising alternatives in the post feeding antibiotic era</span></div><div class="casAuthors">Wang, Jiajun; Dou, Xiujing; Song, Jing; Lyu, Yinfeng; Zhu, Xin; Xu, Lin; Li, Weizhong; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">831-859</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Antimicrobial peptides (AMPs), crit. components of the innate immune system, are widely distributed throughout the animal and plant kingdoms.  They can protect against a broad array of infection-causing agents, such as bacteria, fungi, parasites, viruses, and tumor cells, and also exhibit immunomodulatory activity.  AMPs exert antimicrobial activities primarily through mechanisms involving membrane disruption, so they have a lower likelihood of inducing drug resistance.  Extensive studies on the structure-activity relationship have revealed that net charge, hydrophobicity, and amphipathicity are the most important physicochem. and structural determinants endowing AMPs with antimicrobial potency and cell selectivity.  This review summarizes the recent advances in AMPs development with respect to characteristics, structure-activity relationships, functions, antimicrobial mechanisms, expression regulation, and applications in food, medicine, and animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkakUqHPXXh7Vg90H21EOLACvtfcHk0ljdN8hPiNaWdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXntlGksbY%253D&md5=4ddae625caf29942522874d247a7b52c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fmed.21542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21542%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DLyu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DAntimicrobial%2520peptides%253A%2520promising%2520alternatives%2520in%2520the%2520post%2520feeding%2520antibiotic%2520era%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26volume%3D39%26spage%3D831%26epage%3D859%26doi%3D10.1002%2Fmed.21542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span> <span> </span><span class="NLM_article-title">Avian host defense cathelicidins: structure, expression, biological functions, and potential therapeutic applications</span>. <i>Poultry Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">6434</span>– <span class="NLM_lpage">6445</span>, <span class="refDoi"> DOI: 10.1016/j.psj.2020.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.psj.2020.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=33248558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhtl2qs7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2020&pages=6434-6445&author=Y.+Wangauthor=M.+Wangauthor=A.+Shanauthor=X.+Feng&title=Avian+host+defense+cathelicidins%3A+structure%2C+expression%2C+biological+functions%2C+and+potential+therapeutic+applications&doi=10.1016%2Fj.psj.2020.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Avian host defense cathelicidins: structure, expression, biological functions, and potential therapeutic applications</span></div><div class="casAuthors">Wang, Yingjie; Wang, Min; Shan, Anshan; Feng, Xingjun</div><div class="citationInfo"><span class="NLM_cas:title">Poultry Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6434-6445</span>CODEN:
                <span class="NLM_cas:coden">POSCAL</span>;
        ISSN:<span class="NLM_cas:issn">0032-5791</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Host defense peptides (HDP) are multifunctional effectors of the innate immune system, which has antimicrobial and pleiotropic immunomodulatory functions.  Although there is a very sophisticated superposition of adaptive immune systems in vertebrates, this system is still essential.  As an important family of HDP, cathelicidins are also known for their broad-spectrum antibacterial activity against bacteria, fungi, and enveloped viruses.  It has been found in humans and other species, including cattle, pigs, sheep, goats, chickens, rabbits, and some kind of fish.  Among them, cathelicidins in birds were described for the first time in 2005.  This review focuses on the structure, biol. activities, expression, and regulation of avian cathelicidin, esp. main effects of host defense cathelicidin on potential therapeutic applications.  According to the results obtained both in vitro and in vivo, good perspectives have been opened for cathelicidin.  Nevertheless, further studies are needed to better characterize the mechanisms of action underlying the beneficial effects of cathelicidin as novel therapeutic alternatives to antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWbAO8_7giKrVg90H21EOLACvtfcHk0li8Y1hccnrpiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhtl2qs7bN&md5=b6fea1dfff833898a2c3784690952795</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.psj.2020.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.psj.2020.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DX.%26atitle%3DAvian%2520host%2520defense%2520cathelicidins%253A%2520structure%252C%2520expression%252C%2520biological%2520functions%252C%2520and%2520potential%2520therapeutic%2520applications%26jtitle%3DPoultry%2520Sci.%26date%3D2020%26volume%3D99%26spage%3D6434%26epage%3D6445%26doi%3D10.1016%2Fj.psj.2020.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ranade, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramalingam, R.</span></span> <span> </span><span class="NLM_article-title">A review on bioactive porcine peptide, protegrin-1</span>. <i>Int. J. Pept. Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1493</span>– <span class="NLM_lpage">1501</span>, <span class="refDoi"> DOI: 10.1007/s10989-019-09955-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1007%2Fs10989-019-09955-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFSjsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=1493-1501&author=S.+S.+Ranadeauthor=R.+Ramalingam&title=A+review+on+bioactive+porcine+peptide%2C+protegrin-1&doi=10.1007%2Fs10989-019-09955-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Review on Bioactive Porcine Peptide, Protegrin-1</span></div><div class="casAuthors">Ranade, Shruti Sunil; Ramalingam, Rajasekaran</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide Research and Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1493-1501</span>CODEN:
                <span class="NLM_cas:coden">IJPRFC</span>;
        ISSN:<span class="NLM_cas:issn">1573-3149</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Multi drug resistance is a major problem of the twenty first century.  In order to combat this issue, there is an urgent need in the pharmaceutical industry, for novel therapeutic agents.  Antimicrobial peptides such as protegrins which exhibit non-specific membranolytic action can be viewed as probable therapeutic agents and replace conventional antibiotics.  Protegrin-1 (PG-1) is a peptide isolated from porcine leukocytes.  Its primary role is its antimicrobial activity against a broad-spectrum of gram-pos. as well as gram-neg. bacteria and fungi.  Its antagonistic activity can be accounted by its pore formation mechanism in microbial membranes.  In addn., PG-1 has multiple roles viz., anticancer and antiviral activity, immunomodulatory functions and numerous applications which increase its suitability as a potential therapeutic agent.  This review paper presents a comprehensive overview of biol. roles, lytic mechanism of action and applications of PG-1, thus providing a thorough understanding of this β-sheet peptide, which structurally resembles defensin peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_qXbg9tfiHLVg90H21EOLACvtfcHk0li8Y1hccnrpiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFSjsrbN&md5=f33a4216483c000a44f6b0a24770dbe8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2Fs10989-019-09955-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10989-019-09955-8%26sid%3Dliteratum%253Aachs%26aulast%3DRanade%26aufirst%3DS.%2BS.%26aulast%3DRamalingam%26aufirst%3DR.%26atitle%3DA%2520review%2520on%2520bioactive%2520porcine%2520peptide%252C%2520protegrin-1%26jtitle%3DInt.%2520J.%2520Pept.%2520Res.%2520Ther.%26date%3D2020%26volume%3D26%26spage%3D1493%26epage%3D1501%26doi%3D10.1007%2Fs10989-019-09955-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottler, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Salud Bea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelburne, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramamoorthy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsh, E. N. G.</span></span> <span> </span><span class="NLM_article-title">Using fluorous amino acids to probe the effects of changing hydrophobicity on the physical and biological properties of the beta-hairpin antimicrobial peptide protegrin-1</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">9243</span>– <span class="NLM_lpage">9250</span>, <span class="refDoi"> DOI: 10.1021/bi801045n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi801045n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpslOhu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=9243-9250&author=L.+M.+Gottlerauthor=R.+de+la+Salud+Beaauthor=C.+E.+Shelburneauthor=A.+Ramamoorthyauthor=E.+N.+G.+Marsh&title=Using+fluorous+amino+acids+to+probe+the+effects+of+changing+hydrophobicity+on+the+physical+and+biological+properties+of+the+beta-hairpin+antimicrobial+peptide+protegrin-1&doi=10.1021%2Fbi801045n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Using Fluorous Amino Acids To Probe the Effects of Changing Hydrophobicity on the Physical and Biological Properties of the β-Hairpin Antimicrobial Peptide Protegrin-1</span></div><div class="casAuthors">Gottler, Lindsey M.; de la Salud Bea, Roberto; Shelburne, Charles E.; Ramamoorthy, Ayyalusamy; Marsh, E. Neil G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">9243-9250</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protegrins are potent members of the β-hairpin-forming class of antimicrobial peptides.  Key to their antimicrobial activity is their assembly into oligomeric structures upon binding to the bacterial membrane.  To examine the relationship between the physicochem. properties of the peptide and its biol. activity, we have synthesized variants of protegrin-1 in which key residues in the hydrophobic core, valine-14 and -16, are changed to leucine and to the extensively fluorinated analog hexafluoroleucine.  These substitutions have the effect of making the peptide progressively more hydrophobic while minimally perturbing the secondary structure.  The leucine-contg. peptide was significantly more active than wild-type protegrin against several common pathogenic bacterial strains, whereas the hexafluoroleucine-substituted peptide, in contrast, showed significantly diminished activity against several bacterial strains.  Isothermal titrn. calorimetry measurements revealed significant changes in the interaction of the peptides binding to small unilamelar vesicles that mimic the lipid compn. of the bacterial membrane.  The binding isotherms for wild-type and leucine-substituted protegrins indicate that electrostatic interactions dominate the membrane-peptide interaction, whereas the isotherm for the hexafluoroleucine-substituted protegrin suggests a diminished electrostatic component to binding.  Notably both of these substitutions appear to alter the stoichiometry of the lipid-peptide interaction, suggesting that these substitutions may stabilize oligomerized forms of protegrin that are postulated to be intermediates in the assembly of the β-barrel membrane pore structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMIUlmWDuc6bVg90H21EOLACvtfcHk0lgFcLC4RLmS8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpslOhu7s%253D&md5=bd0c2c90f5c3aa620ad3e23956c75945</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbi801045n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi801045n%26sid%3Dliteratum%253Aachs%26aulast%3DGottler%26aufirst%3DL.%2BM.%26aulast%3Dde%2Bla%2BSalud%2BBea%26aufirst%3DR.%26aulast%3DShelburne%26aufirst%3DC.%2BE.%26aulast%3DRamamoorthy%26aufirst%3DA.%26aulast%3DMarsh%26aufirst%3DE.%2BN.%2BG.%26atitle%3DUsing%2520fluorous%2520amino%2520acids%2520to%2520probe%2520the%2520effects%2520of%2520changing%2520hydrophobicity%2520on%2520the%2520physical%2520and%2520biological%2520properties%2520of%2520the%2520beta-hairpin%2520antimicrobial%2520peptide%2520protegrin-1%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D9243%26epage%3D9250%26doi%3D10.1021%2Fbi801045n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span> <span> </span><span class="NLM_article-title">De novo generation of short antimicrobial peptides with enhanced stability and cell specificity</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1093/jac/dkt322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1093%2Fjac%2Fdkt322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=23946320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegtrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=121-132&author=H.+Kimauthor=J.+H.+Jangauthor=S.+C.+Kimauthor=J.+H.+Cho&title=De+novo+generation+of+short+antimicrobial+peptides+with+enhanced+stability+and+cell+specificity&doi=10.1093%2Fjac%2Fdkt322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">De novo generation of short antimicrobial peptides with enhanced stability and cell specificity</span></div><div class="casAuthors">Kim, Hyun; Jang, Ju Hye; Kim, Sun Chang; Cho, Ju Hyun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-132</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Objectives Though antimicrobial peptides (AMPs) show great potential as novel antibiotics, therapeutic applications are hindered by their low stability, toxicity and high manufg. cost.  Various chem. modification strategies are employed to overcome these problems.  However, chem. modifications often significantly increase the manufg. cost of AMPs with only limited pharmacokinetic advantages.  Therefore, we developed AMPs with enhanced stability and cell specificity that can be economically produced.  Methods Peptides were designed by systematic amino acid arrangement without the incorporation of both non-natural amino acids and peptidomimetics.  Antimicrobial activities were measured against Gram-pos. bacteria, Gram-neg. bacteria and fungi by MIC evaluation under both std. and physiol. relevant conditions.  Cytotoxicity towards human cells was evaluated to verify selective antimicrobial activity.  The antibacterial mechanism of the peptides was elucidated by β-galactosidase assay and SEM.  Results Among the designed peptides, GNU6 and GNU7 showed potent antimicrobial activity against bacteria and fungi and maintained their activity in the presence of 150 mM NaCl and 10% serum.  These peptides were not digested by exposure to trypsin, chymotrypsin and aureolysin for up to 12 h and showed potent antimicrobial activity against methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci.  Moreover, they did not affect the viability of erythrocytes, keratinocytes and fibroblasts up to 128 mg/L.  A membrane permeabilization assay and SEM anal. showed that GNU6 and GNU7 compromised membrane integrity and function in microorganisms.  Conclusions This study suggests that GNU6 and GNU7 might overcome serious problems that currently prevent the clin. use of AMPs and be developed as novel antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5zPEe3xO5SrVg90H21EOLACvtfcHk0lgFcLC4RLmS8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegtrbN&md5=5238110fd2cdb45a7915c0c81b3cdfbc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkt322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkt322%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DH.%26aulast%3DJang%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DS.%2BC.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26atitle%3DDe%2520novo%2520generation%2520of%2520short%2520antimicrobial%2520peptides%2520with%2520enhanced%2520stability%2520and%2520cell%2520specificity%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D121%26epage%3D132%26doi%3D10.1093%2Fjac%2Fdkt322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Legowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukajtis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sztabkowska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizeria, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wysocka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolka, K.</span></span> <span> </span><span class="NLM_article-title">Introduction of pro and its analogues in the conserved p1’ position of trypsin inhibitor SFTI-1 retains its inhibitory activity</span>. <i>Protein Pept. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1158</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.2174/092986611797201002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.2174%2F092986611797201002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=21605056" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1158-1167&author=A.+Legowskaauthor=D.+Debowskiauthor=R.+Lukajtisauthor=E.+Sztabkowskaauthor=A.+Mizeriaauthor=K.+Brzozowskiauthor=M.+Wysockaauthor=A.+Lesnerauthor=K.+Rolka&title=Introduction+of+pro+and+its+analogues+in+the+conserved+p1%E2%80%99+position+of+trypsin+inhibitor+SFTI-1+retains+its+inhibitory+activity&doi=10.2174%2F092986611797201002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Introduction of Pro and its analogues in the conserved P1' position of trypsin inhibitor SFTI-1 retains its inhibitory activity</span></div><div class="casAuthors">Legowska, Anna; Debowski, Dawid; Lukajtis, Rafal; Sztabkowska, Emilia; Mizeria, Aneta; Brzozowski, Krzysztof; Wysocka, Magdalena; Lesner, Adam; Rolka, Krzysztof</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Peptide Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1158-1167</span>CODEN:
                <span class="NLM_cas:coden">PPELEN</span>;
        ISSN:<span class="NLM_cas:issn">0929-8665</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A no. of monocyclic SFTI-1 analogs modified in the conserved inhibitor P1' position by Pro, its L-hydroxyproline (Hyp) deriv. as well as mimetics with different ring size were synthesized by the solid-phase method.  Replacement of Ser6 by Pro, Hyp, and a four-member ring, L-azetidine-2-carboxylic acid (Aze), retained trypsin or chymotrypsin inhibitory activity.  The detd. assocn. equil. consts. of these analogs with a cognate enzyme were about two orders of magnitude lower than those obtained for ones with conserved Ser6.  In all analogs, with the exception of one, [Phe5,Aze6]SFTI-1, the P1-P1' reactive site remained intact.  The results provide first evidence that the conserved Ser in the P1' position of Bowman-Birk inhibitors can be successfully replaced by an amino acid with a secondary amine group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL1VFIjxzcIbVg90H21EOLACvtfcHk0lgFcLC4RLmS8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrO&md5=502cd50f8fa894e1251ec675eccd55c9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.2174%2F092986611797201002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986611797201002%26sid%3Dliteratum%253Aachs%26aulast%3DLegowska%26aufirst%3DA.%26aulast%3DDebowski%26aufirst%3DD.%26aulast%3DLukajtis%26aufirst%3DR.%26aulast%3DSztabkowska%26aufirst%3DE.%26aulast%3DMizeria%26aufirst%3DA.%26aulast%3DBrzozowski%26aufirst%3DK.%26aulast%3DWysocka%26aufirst%3DM.%26aulast%3DLesner%26aufirst%3DA.%26aulast%3DRolka%26aufirst%3DK.%26atitle%3DIntroduction%2520of%2520pro%2520and%2520its%2520analogues%2520in%2520the%2520conserved%2520p1%25E2%2580%2599%2520position%2520of%2520trypsin%2520inhibitor%2520SFTI-1%2520retains%2520its%2520inhibitory%2520activity%26jtitle%3DProtein%2520Pept.%2520Lett.%26date%3D2011%26volume%3D18%26spage%3D1158%26epage%3D1167%26doi%3D10.2174%2F092986611797201002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">D-amino acid substitution enhances the stability of antimicrobial peptide polybia-CP</span>. <i>Acta Biochim. Biophys. Sin. (Shanghai)</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">916</span>– <span class="NLM_lpage">925</span>, <span class="refDoi"> DOI: 10.1093/abbs/gmx091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1093%2Fabbs%2Fgmx091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=28981608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252Fls1Kntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=916-925&author=F.+Jiaauthor=J.+Wangauthor=J.+Pengauthor=P.+Zhaoauthor=Z.+Kongauthor=K.+Wangauthor=W.+Yanauthor=R.+Wang&title=D-amino+acid+substitution+enhances+the+stability+of+antimicrobial+peptide+polybia-CP&doi=10.1093%2Fabbs%2Fgmx091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">D-amino acid substitution enhances the stability of antimicrobial peptide polybia-CP</span></div><div class="casAuthors">Jia Fengjing; Wang Jiayi; Peng Jinxiu; Zhao Ping; Wang Kairong; Yan Wenjin; Wang Rui; Kong Ziqing</div><div class="citationInfo"><span class="NLM_cas:title">Acta biochimica et biophysica Sinica</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">916-925</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">With the increasing emergence of resistant microbes toward conventional antimicrobial agents, there is an urgent need for the development of antimicrobial agents with novel action mode.  Antimicrobial peptides (AMPs) are believed to be one kind of ideal alternatives.  However, AMPs can be easily degraded by protease, which limited their therapeutic use.  In the present study, D-amino acid substitution strategy was employed to enhance the stability of polybia-CP.  We investigated the stability of peptides against the degradation of trypsin and chymotrypsin by determining the antimicrobial activity or determining the HPLC profile of peptides after incubation with proteases.  Our results showed that both the all D-amino acid derivative (D-CP) and partial D-lysine substitution derivative (D-lys-CP) have an improved stability against trypsin and chymotrypsin.  Although D-CP takes left-hand α-helical conformation and D-lys-CP loses some α-helical content, both of the D-amino acid-substituted derivatives maintain their parental peptides' membrane active action mode.  In addition, D-lys-CP showed a slight weaker antimicrobial activity than polybia-CP, but the hemolytic activity decreased greatly.  These results suggest that D-CP and D-lys-CP can offer strategy to improve the property of AMPs and may be leading compounds for the development of novel antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6borawicmlUGy1nwu7K2mfW6udTcc2eYA6dZdB8k_Nbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252Fls1Kntw%253D%253D&md5=9cadf0a1a38f230c8b1dc9d521d6dfb8</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Fabbs%2Fgmx091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabbs%252Fgmx091%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DKong%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DD-amino%2520acid%2520substitution%2520enhances%2520the%2520stability%2520of%2520antimicrobial%2520peptide%2520polybia-CP%26jtitle%3DActa%2520Biochim.%2520Biophys.%2520Sin.%2520%2528Shanghai%2529%26date%3D2017%26volume%3D49%26spage%3D916%26epage%3D925%26doi%3D10.1093%2Fabbs%2Fgmx091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gorris, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bade, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röckendorf, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fránek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frey, A.</span></span> <span> </span><span class="NLM_article-title">Rapid profiling of peptide stability in proteolytic environments</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">1580</span>– <span class="NLM_lpage">1586</span>, <span class="refDoi"> DOI: 10.1021/ac802324f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac802324f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotlGltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=1580-1586&author=H.+H.+Gorrisauthor=S.+Badeauthor=N.+R%C3%B6ckendorfauthor=E.+Albersauthor=M.+A.+Schmidtauthor=M.+Fr%C3%A1nekauthor=A.+Frey&title=Rapid+profiling+of+peptide+stability+in+proteolytic+environments&doi=10.1021%2Fac802324f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Profiling of Peptide Stability in Proteolytic Environments</span></div><div class="casAuthors">Gorris, Hans H.; Bade, Steffen; Rockendorf, Niels; Albers, Eike; Schmidt, M. Alexander; Franek, Milan; Frey, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1580-1586</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The notorious degrdn. susceptibility of peptides is a major obstacle to their use as medicinal drugs.  Assays with which the stability of peptides in complex proteolytic environments can be detd. are thus indispensable for peptide drug development.  Herein, we describe a new peptide proteolysis assay that meets that demand.  It unites the high-throughput capacity of heterogeneous with the well-defined kinetic characteristics of homogeneous assay formats and operates on the cleavage-caused loss of a detection handle.  We have confirmed throughput screening the assay's accuracy with well-defined model interactions and proved its high versatility and robustness with a representative application where we detd. the half-lives of 375 different peptides in a crude intestinal protease prepn.  With this reliable, reproducible, and efficient assay the enzyme kinetics of any peptide-protease interaction is accessible even for complex protease solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPi_HqtCxog7Vg90H21EOLACvtfcHk0lhX3f-Anf-WVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotlGltQ%253D%253D&md5=54bf211252901e687a7c01b1f201ac2d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fac802324f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac802324f%26sid%3Dliteratum%253Aachs%26aulast%3DGorris%26aufirst%3DH.%2BH.%26aulast%3DBade%26aufirst%3DS.%26aulast%3DR%25C3%25B6ckendorf%26aufirst%3DN.%26aulast%3DAlbers%26aufirst%3DE.%26aulast%3DSchmidt%26aufirst%3DM.%2BA.%26aulast%3DFr%25C3%25A1nek%26aufirst%3DM.%26aulast%3DFrey%26aufirst%3DA.%26atitle%3DRapid%2520profiling%2520of%2520peptide%2520stability%2520in%2520proteolytic%2520environments%26jtitle%3DAnal.%2520Chem.%26date%3D2009%26volume%3D81%26spage%3D1580%26epage%3D1586%26doi%3D10.1021%2Fac802324f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial potency and selectivity of simplified symmetric-end peptides</span>. <i>Biomaterials</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">8028</span>– <span class="NLM_lpage">8039</span>, <span class="refDoi"> DOI: 10.1016/j.biomaterials.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.biomaterials.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=24952979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWqs7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=8028-8039&author=N.+Dongauthor=X.+Zhuauthor=S.+Chouauthor=A.+Shanauthor=W.+Liauthor=J.+Jiang&title=Antimicrobial+potency+and+selectivity+of+simplified+symmetric-end+peptides&doi=10.1016%2Fj.biomaterials.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial potency and selectivity of simplified symmetric-end peptides</span></div><div class="casAuthors">Dong, Na; Zhu, Xin; Chou, Shuli; Shan, Anshan; Li, Weizhong; Jiang, Junguang</div><div class="citationInfo"><span class="NLM_cas:title">Biomaterials</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">8028-8039</span>CODEN:
                <span class="NLM_cas:coden">BIMADU</span>;
        ISSN:<span class="NLM_cas:issn">0142-9612</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because antimicrobial peptides (AMPs) are potentially useful for the treatment of multidrug-resistant infections, more attention is being paid to the structural modification and structure-function relationship of both naturally occurring and synthetic AMPs.  Previous studies indicated that Protegrin-1 (PG-1), isolated from porcine leukocytes, exhibited considerable antimicrobial activity and cytotoxicity.  The β-turn of PG-1 floated on the surface of bacterial membrane, while its β-strand inserted into the bacterial membrane and formed pores that were dedicated to producing cytotoxicity.  For reducing cytotoxicity and improving cells selectivity, we designed a series of simplified sym.-end peptides by combining the β-turn of PG-1 with simple amino acid repeat sequences.  The sequence of designed sym.-end peptides is (XR)nH(RX)n, (n = 1,2; X represents I, F, W and P; H represents CRRRFC).  The sym.-end peptides displayed antimicrobial activity against both gram-pos. and gram-neg. bacteria.  In particular, (XR)2H(RX)2 (X here is I, F and W) showed greater antimicrobial potency than PG-1.  Hemolysis activity and cytotoxicity, detected by using human red blood cells (RBCs) and human embryonic lung fibroblasts MRC-5 cells, were observably lower than the native peptide PG-1. (IR)2H(RI)2 (IR2), folded into β-sheet structures, displayed the highest therapeutic index, suggesting its great cell selectivity.  The fluorescence spectroscopy, flow cytometry, and electron microscopy observation indicated that IR2 exhibited great membrane penetration potential by inducing membrane blebbing, disruption and lysis.  Collectively, generating sym.-end β-sheet peptides is a promising strategy for designing effective AMPs with great antimicrobial activities and cell selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZGe2XjNdNN7Vg90H21EOLACvtfcHk0lhX3f-Anf-WVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWqs7nM&md5=58acfdd2b050d20c77f42dc2d34c742f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.biomaterials.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biomaterials.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DChou%26aufirst%3DS.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DJ.%26atitle%3DAntimicrobial%2520potency%2520and%2520selectivity%2520of%2520simplified%2520symmetric-end%2520peptides%26jtitle%3DBiomaterials%26date%3D2014%26volume%3D35%26spage%3D8028%26epage%3D8039%26doi%3D10.1016%2Fj.biomaterials.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">Design of hybrid beta-hairpin peptides with enhanced cell specificity and potent anti-inflammatory activity</span>. <i>Biomaterials</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1016/j.biomaterials.2012.09.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.biomaterials.2012.09.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=23046754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKksL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=237-250&author=Y.+F.+Liuauthor=X.+Xiaauthor=L.+Xuauthor=Y.+Z.+Wang&title=Design+of+hybrid+beta-hairpin+peptides+with+enhanced+cell+specificity+and+potent+anti-inflammatory+activity&doi=10.1016%2Fj.biomaterials.2012.09.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design of hybrid β-hairpin peptides with enhanced cell specificity and potent anti-inflammatory activity</span></div><div class="casAuthors">Liu, Yi Fan; Xia, Xi; Xu, Liang; Wang, Yi Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Biomaterials</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">237-250</span>CODEN:
                <span class="NLM_cas:coden">BIMADU</span>;
        ISSN:<span class="NLM_cas:issn">0142-9612</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs) have attracted considerable attention for their broad-spectrum antimicrobial activity and reduced tendency to cause bacterial resistance.  Emerging concerns over the host cytotoxicity of AMPs, however, may ultimately compromise their development as pharmaceuticals.  In order to optimize AMPs with potent cell specificity and anti-inflammatory activity, we designed β-hairpin hybrid peptides based upon progetrin-1, bovine lactoferricin and cecropin A.  The synthetic hybrid peptides LB-PG and CA-PG demonstrated high selectivity over a wide range of microbes from Gram-pos. and Gram-neg. bacteria in porcine red blood cells.  SEM (SEM) and transmission electron microscopy (TEM) show that these peptides kill microbial cells by penetrating the cell membrane and damaging the membrane envelope.  Gel retardation demonstrates that the peptides have a high affinity for DNA, indicating an addnl. possible intracellular bactericidal mechanism.  Moreover, the hybrid peptides inhibit the expression of LPS-induced proinflammatory cytokines and chemokines, such as tumor necrosis factor-α (TNF-α), inducible nitric oxide synthase (iNOS), macrophage inflammatory protein-1α (MIP-1α) and monocyte chemoattractant protein 1(MCP-1), following LPS stimulation in RAW264.7 cells.  Our results indicate that these hybrid peptides have considerable potential for future development as antimicrobial and anti-inflammatory agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRe2RvSF4SjbVg90H21EOLACvtfcHk0lhRR8Yh6RUSEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKksL7J&md5=25edbb8639a1a96dd35e013a4fb961fa</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.biomaterials.2012.09.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biomaterials.2012.09.032%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%2BF.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%2BZ.%26atitle%3DDesign%2520of%2520hybrid%2520beta-hairpin%2520peptides%2520with%2520enhanced%2520cell%2520specificity%2520and%2520potent%2520anti-inflammatory%2520activity%26jtitle%3DBiomaterials%26date%3D2013%26volume%3D34%26spage%3D237%26epage%3D250%26doi%3D10.1016%2Fj.biomaterials.2012.09.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides with protease stability: progress and perspective</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2047</span>– <span class="NLM_lpage">2050</span>, <span class="refDoi"> DOI: 10.4155/fmc-2019-0167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.4155%2Ffmc-2019-0167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=31538527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=2047-2050&author=C.+Shaoauthor=Y.+Zhuauthor=Z.+Laiauthor=P.+Tanauthor=A.+Shan&title=Antimicrobial+peptides+with+protease+stability%3A+progress+and+perspective&doi=10.4155%2Ffmc-2019-0167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial peptides with protease stability: progress and perspective</span></div><div class="casAuthors">Shao, Changxuan; Zhu, Yongjie; Lai, Zhenheng; Tan, Peng; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2047-2050</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Antimicrobial peptides have attracted decades of attention as potential candidates for addressing multidrug-resistant infections; however, their dismal record of immature stability in clin. trials suggests a likely frustrating ending.  A revolution of novel antimicrobial peptides against protease degrdn. is already on the road. The drug delivery routes of i.m. or s.c. injections may not require too much stability of the peptide itself, but i.v. or oral administration will certainly expose these peptides to the ordeal of the proteases in serum, digestive tract or even the proteases secreted by the bacteria. Thus, with the deepening of research on AMPs, more attention has been focused on their lability to proteolytic degrdn. and the prodn.  Cost closely related to this. Therefore, the enzyme stability study of antimicrobial peptides is a long and arduous process, just like the initial exploration of the structural and functional relationship many years ago. With the advances of premature stability, the peptide-based therapeutics will likely usher in renewed innovations in pharmaceuticals, healthcare nutraceutical, food preservative, cosmetic or animal husbandry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSBFJ1FEOufLVg90H21EOLACvtfcHk0lhRR8Yh6RUSEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7bF&md5=f987c4538cd723eb5d1722caa84f2f59</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2019-0167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2019-0167%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DAntimicrobial%2520peptides%2520with%2520protease%2520stability%253A%2520progress%2520and%2520perspective%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D2047%26epage%3D2050%26doi%3D10.4155%2Ffmc-2019-0167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augusto, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felício, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, N. C.</span></span> <span> </span><span class="NLM_article-title">Designing improved active peptides for therapeutic approaches against infectious diseases</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2018.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.biotechadv.2018.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=29330093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGhu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=415-429&author=B.+Gomesauthor=M.+T.+Augustoauthor=M.+R.+Fel%C3%ADcioauthor=A.+Hollmannauthor=O.+L.+Francoauthor=S.+Gon%C3%A7alvesauthor=N.+C.+Santos&title=Designing+improved+active+peptides+for+therapeutic+approaches+against+infectious+diseases&doi=10.1016%2Fj.biotechadv.2018.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Designing improved active peptides for therapeutic approaches against infectious diseases</span></div><div class="casAuthors">Gomes, Barbara; Augusto, Marcelo T.; Felicio, Mario R.; Hollmann, Axel; Franco, Octavio L.; Goncalves, Sonia; Santos, Nuno C.</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">415-429</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Infectious diseases are one of the main causes of human morbidity and mortality.  In the last few decades, pathogenic microorganisms' resistance to conventional drugs has been increasing, and it is now pinpointed as a major worldwide health concern.  The need to search for new therapeutic options, as well as improved treatment outcomes, has therefore increased significantly, with biol. active peptides representing a new alternative.  A substantial research effort is being dedicated towards their development, esp. due to improved biocompatibility and target selectivity.  However, the inherent limitations of peptide drugs are restricting their application.  In this review, we summarize the current status of peptide drug development, focusing on antiviral and antimicrobial peptide activities, highlighting the design improvements needed, and those already being used, to overcome the drawbacks of the therapeutic application of biol. active peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIlcI4labulbVg90H21EOLACvtfcHk0lhRR8Yh6RUSEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGhu7c%253D&md5=98b3e01c6f8dbef08f30767ea2eb1865</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2018.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2018.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DGomes%26aufirst%3DB.%26aulast%3DAugusto%26aufirst%3DM.%2BT.%26aulast%3DFel%25C3%25ADcio%26aufirst%3DM.%2BR.%26aulast%3DHollmann%26aufirst%3DA.%26aulast%3DFranco%26aufirst%3DO.%2BL.%26aulast%3DGon%25C3%25A7alves%26aufirst%3DS.%26aulast%3DSantos%26aufirst%3DN.%2BC.%26atitle%3DDesigning%2520improved%2520active%2520peptides%2520for%2520therapeutic%2520approaches%2520against%2520infectious%2520diseases%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2018%26volume%3D36%26spage%3D415%26epage%3D429%26doi%3D10.1016%2Fj.biotechadv.2018.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avrahami, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shai, Y.</span></span> <span> </span><span class="NLM_article-title">Conjugation of a magainin analogue with lipophilic acids controls hydrophobicity, solution assembly, and cell selectivity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2254</span>– <span class="NLM_lpage">2263</span>, <span class="refDoi"> DOI: 10.1021/bi011549t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi011549t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD38Xnt1GlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2254-2263&author=D.+Avrahamiauthor=Y.+Shai&title=Conjugation+of+a+magainin+analogue+with+lipophilic+acids+controls+hydrophobicity%2C+solution+assembly%2C+and+cell+selectivity&doi=10.1021%2Fbi011549t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conjugation of a magainin analogue with lipophilic acids controls hydrophobicity, solution assembly, and cell selectivity</span></div><div class="casAuthors">Avrahami, Dorit; Shai, Yechiel</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2254-2263</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Our basic understanding of how to combat fungal infections has not kept pace with the recent sharp rise in life-threatening cases found, particularly among immuno-compromised individuals.  Current investigations for new potential antifungal agents have focused on antimicrobial peptides, which are used as a cell-free defense mechanism in all organisms.  Unfortunately, despite their high antibacterial activity, most of them are not active toward fungi, the reason of which is not clear.  Here, we present a new approach to modify an antibacterial peptide, a magainin analog, to display antifungal activity by its conjugation with lipophilic acids.  This approach has the advantage of producing well-defined changes in hydrophobicity, secondary structure, and self-assocn.  These modifications were characterized in soln. at physiol. concns. using CD spectroscopy, tryptophan fluorescence, and anal. ultracentrifugation.  In order of increasing hydrophobicity, the attachment to the magainin-2 analog of (i) heptanoic acid results in a monomeric, unordered structure, (ii) undecanoic acid yields concn.-dependent oligomers of α-helixes, and (iii) palmitic acid yields concn.-independent α-helical monomers, a novel lipopeptide structure which is resistant to proteolytic digestion.  Membrane-lipopeptide interactions and the membrane-bound structures were studied using fluorescence and ATR-FTIR in PC/PE/PI/ergosterol (5/2.5/2.5/1, wt./wt.) SUV, which constitute the major components of Candida albicans bilayers.  A direct correlation was found between oligomerization of the lipopeptides in soln. and potent antifungal activity.  These results provide insight to a new approach of modulating hydrophobicity and self-assembly of antimicrobial peptides in soln., without altering the sequence of the peptide chain.  These studies also provide a general means of developing a new group of lipopeptide candidates as therapeutic agents against fungal infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoajv-S8sGnybVg90H21EOLACvtfcHk0lg6cM-4g5Ofeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xnt1GlsA%253D%253D&md5=5281b939bbe93ddd9f62bf5fe7302c00</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fbi011549t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi011549t%26sid%3Dliteratum%253Aachs%26aulast%3DAvrahami%26aufirst%3DD.%26aulast%3DShai%26aufirst%3DY.%26atitle%3DConjugation%2520of%2520a%2520magainin%2520analogue%2520with%2520lipophilic%2520acids%2520controls%2520hydrophobicity%252C%2520solution%2520assembly%252C%2520and%2520cell%2520selectivity%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D2254%26epage%3D2263%26doi%3D10.1021%2Fbi011549t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avrahami, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shai, Y.</span></span> <span> </span><span class="NLM_article-title">A new group of antifungal and antibacterial lipopeptides derived from non-membrane active peptides conjugated to palmitic acid</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">12277</span>– <span class="NLM_lpage">12285</span>, <span class="refDoi"> DOI: 10.1074/jbc.M312260200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1074%2Fjbc.M312260200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=14709550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitl2gsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=12277-12285&author=D.+Avrahamiauthor=Y.+Shai&title=A+new+group+of+antifungal+and+antibacterial+lipopeptides+derived+from+non-membrane+active+peptides+conjugated+to+palmitic+acid&doi=10.1074%2Fjbc.M312260200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A new group of antifungal and antibacterial lipopeptides derived from non-membrane active peptides conjugated to palmitic acid</span></div><div class="casAuthors">Avrahami, Dorit; Shai, Yechiel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">12277-12285</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">We report on the synthesis, biol. function, and a plausible mode of action of a new group of lipopeptides with potent antifungal and antibacterial activities.  These lipopeptides are derived from pos. charged peptides contg. D- and L-amino acids (diastereomers) that are palmitoylated (PA) at their N terminus.  The peptides investigated have the sequence K4X7W, where X designates Gly, Ala, Val, or Leu (designated D-X peptides).  The data revealed that PA-D-G and PA-D-A gained potent antibacterial and antifungal activity despite the fact that both parental peptides were completely devoid of any activity toward microorganisms and model phospholipid membranes.  In contrast, PA-D-L lost the potent antibacterial activity of the parental peptide but gained and preserved partial antifungal activity.  Interestingly, both D-V and its palmitoylated analog were inactive toward bacteria, and only the palmitoylated peptide was highly potent toward yeast.  Both PA-D-L and PA-D-V lipopeptides were also endowed with hemolytic activity.  Mode of action studies were performed by using tryptophan fluorescence and attenuated total reflectance Fourier transform IR and CD spectroscopy as well as transmembrane depolarization assays with bacteria and fungi.  The data suggest that the lipopeptides act by increasing the permeability of the cell membrane and that differences in their potency and target specificity are the result of differences in their oligomeric state and ability to dissoc. and insert into the cytoplasmic membrane.  These results provide insight regarding a new approach of modulating hydrophobicity and the self-assembly of nonmembrane interacting peptides in order to endow them with both antibacterial and antifungal activities urgently needed to combat bacterial and fungal infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAv3oC-E2ZFLVg90H21EOLACvtfcHk0lg6cM-4g5Ofeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitl2gsr0%253D&md5=9b8a883fa41e416addd7de08b5d6ebdc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M312260200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M312260200%26sid%3Dliteratum%253Aachs%26aulast%3DAvrahami%26aufirst%3DD.%26aulast%3DShai%26aufirst%3DY.%26atitle%3DA%2520new%2520group%2520of%2520antifungal%2520and%2520antibacterial%2520lipopeptides%2520derived%2520from%2520non-membrane%2520active%2520peptides%2520conjugated%2520to%2520palmitic%2520acid%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D12277%26epage%3D12285%26doi%3D10.1074%2Fjbc.M312260200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, J. K.</span></span> <span> </span><span class="NLM_article-title">Water-soluble lipopolymer as a gene carrier to corpus cavernosum</span>. <i>Int. J. Impot. Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1038/sj.ijir.3901314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1038%2Fsj.ijir.3901314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=15800653" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlslGnsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=326-334&author=M.+Leeauthor=J.+K.+Ryuauthor=S.+M.+Ohauthor=E.+Leeauthor=H.+Y.+Shinauthor=S.+U.+Songauthor=S.+W.+Kimauthor=J.+K.+Suh&title=Water-soluble+lipopolymer+as+a+gene+carrier+to+corpus+cavernosum&doi=10.1038%2Fsj.ijir.3901314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Water-soluble lipopolymer as a gene carrier to corpus cavernosum</span></div><div class="casAuthors">Lee, M.; Ryu, J-K.; Oh, S-M.; Lee, E.; Shin, H-Y.; Song, S. U.; Kim, S. W.; Suh, J-K.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Impotence Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">326-334</span>CODEN:
                <span class="NLM_cas:coden">IJIRFB</span>;
        ISSN:<span class="NLM_cas:issn">0955-9930</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Adenovirus or naked plasmid DNA (pDNA) has been used to deliver the therapeutic gene into corpus cavernosum.  However, the potential risks of viral vector and inefficiency of naked pDNA have limited their clin. application.  In this study, water-sol. lipopolymer (WSLP) was evaluated as a gene carrier to corpus cavernosum.  The WSLP/pDNA complex was transfected to smooth muscle cells in vitro.  WSLP had high transfection efficiency, which was comparable to poly(ethylenimine) (PEI).  In addn., WSLP had much less cytotoxicity than PEI, suggesting that WSLP is a safer carrier than PEI.  To evaluate the transfection efficiency to corpus cavernosum, the WSLP/pDNA complex was injected into the rat corpus cavernosum.  As a result, the WSLP/pDNA complex showed higher transfection efficiency than naked pDNA.  In addn., the gene expression was dependent upon the dose of the complex.  The results suggest that WSLP may be useful for gene therapy of erectile dysfunction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwSv4YaVQIKrVg90H21EOLACvtfcHk0lhLIuV7ey1cKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlslGnsL8%253D&md5=670da61b629e366784b1a5426e28fd62</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsj.ijir.3901314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.ijir.3901314%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%26aulast%3DRyu%26aufirst%3DJ.%2BK.%26aulast%3DOh%26aufirst%3DS.%2BM.%26aulast%3DLee%26aufirst%3DE.%26aulast%3DShin%26aufirst%3DH.%2BY.%26aulast%3DSong%26aufirst%3DS.%2BU.%26aulast%3DKim%26aufirst%3DS.%2BW.%26aulast%3DSuh%26aufirst%3DJ.%2BK.%26atitle%3DWater-soluble%2520lipopolymer%2520as%2520a%2520gene%2520carrier%2520to%2520corpus%2520cavernosum%26jtitle%3DInt.%2520J.%2520Impot.%2520Res.%26date%3D2005%26volume%3D17%26spage%3D326%26epage%3D334%26doi%3D10.1038%2Fsj.ijir.3901314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span> <span> </span><span class="NLM_article-title">Modification of antimicrobial peptide with low molar mass poly(ethylene glycol)</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">788</span>, <span class="refDoi"> DOI: 10.1093/jb/mvn134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1093%2Fjb%2Fmvn134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=18845567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1MXis1Cmug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2008&pages=781-788&author=G.+Zhangauthor=B.+Hanauthor=X.+Linauthor=X.+Wuauthor=H.+Yan&title=Modification+of+antimicrobial+peptide+with+low+molar+mass+poly%28ethylene+glycol%29&doi=10.1093%2Fjb%2Fmvn134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Modification of antimicrobial peptide with low molar mass poly(ethylene glycol)</span></div><div class="casAuthors">Zhang, Genghui; Han, Baozhong; Lin, Xiaoyan; Wu, Xin; Yan, Husheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    
            (<span class="NLM_cas:orgname">Japanese Biochemical Society</span>)
        </div><div class="casAbstract">PEGylation of peptide drugs prolongs their circulating lifetimes in plasma.  However, PEGylation can produce a decrease in the in vitro bioactivity.  Longer poly(ethylene glycol) (PEG) chains are favorable for circulating lifetimes but unfavorable for in vitro bioactivities.  In order to circumvent the conflicting effects of PEG length, a hydrophobic peptide, using an antimicrobial peptide as a model, was PEGylated with short PEG chains.  The PEGylated peptides self-assembled in aq. soln. into micelles with PEG shell and peptide core.  In these micelles, the core peptides were protected by the shell, thus reducing proteolytic degrdn.  Meanwhile, most of the in vitro antimicrobial activities still remained due to the short PEG chain attached.  The stabilities of the PEGylated peptides were much higher than that of the unPEGylated peptides in the presence of chymotrypsin and serum.  The antimicrobial activities of the PEGylated peptides in the presence of serum, an ex vivo assay, were much higher than that of the unPEGylated peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgB-hNz3efbbVg90H21EOLACvtfcHk0liYCpgV3ZetCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXis1Cmug%253D%253D&md5=12648ad61504661fe49b0a435a43dfcb</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fjb%2Fmvn134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjb%252Fmvn134%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DH.%26atitle%3DModification%2520of%2520antimicrobial%2520peptide%2520with%2520low%2520molar%2520mass%2520poly%2528ethylene%2520glycol%2529%26jtitle%3DJ.%2520Biochem.%26date%3D2008%26volume%3D144%26spage%3D781%26epage%3D788%26doi%3D10.1093%2Fjb%2Fmvn134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benincasa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahariev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelillo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gennaro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scocchi, M.</span></span> <span> </span><span class="NLM_article-title">PEGylation of the peptide Bac7(1–35) reduces renal clearance while retaining antibacterial activity and bacterial cell penetration capacity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2015.03.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.ejmech.2015.03.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=25817771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltFWjsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2015&pages=210-219&author=M.+Benincasaauthor=S.+Zaharievauthor=C.+Pelilloauthor=A.+Milanauthor=R.+Gennaroauthor=M.+Scocchi&title=PEGylation+of+the+peptide+Bac7%281%E2%80%9335%29+reduces+renal+clearance+while+retaining+antibacterial+activity+and+bacterial+cell+penetration+capacity&doi=10.1016%2Fj.ejmech.2015.03.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylation of the peptide Bac7(1-35) reduces renal clearance while retaining antibacterial activity and bacterial cell penetration capacity</span></div><div class="casAuthors">Benincasa, Monica; Zahariev, Sotir; Pelillo, Chiara; Milan, Annalisa; Gennaro, Renato; Scocchi, Marco</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">210-219</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The proline-rich antibacterial peptide Bac7(1-35) protects mice against Salmonella typhimurium infection, despite its rapid clearance.  To overcome this problem the peptide was linked to a polyethylene glycol (PEG) mol. either via a cleavable ester bond or via a non-hydrolysable amide bond.  Both the PEGylated conjugates retained most of the in vitro activity against S. typhimurium.  In addn., the ester bond was cleaved in human serum or plasma, releasing a carboxymethyl deriv. of Bac7(1-35) which accounts for a higher activity of this peptide with relative to the other, non-hydrolysable form.  Both PEGylated peptides maintained the capacity of the unconjugated form to kill bacteria without permeabilizing the bacterial membranes, by penetrating into cells.  They exploited the same transporter as unmodified Bac7(1-35), suggesting it has the capacity to internalize quite sizeable cargo if this is linked to Bac7 fragment.  PEGylation allows the peptide to have a wide distribution in mice, and a slow renal clearance, indicating that this strategy would improve the bioavailability of Bac7, and in principle of other antimicrobial peptides.  This can be an equally important issue to reducing cytotoxicity for therapeutic use of these antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjn03eW3Tzq7Vg90H21EOLACvtfcHk0li-tjlR2KLS_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltFWjsL0%253D&md5=3e764baf5637f63b12fcebe9b431f690</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2015.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2015.03.028%26sid%3Dliteratum%253Aachs%26aulast%3DBenincasa%26aufirst%3DM.%26aulast%3DZahariev%26aufirst%3DS.%26aulast%3DPelillo%26aufirst%3DC.%26aulast%3DMilan%26aufirst%3DA.%26aulast%3DGennaro%26aufirst%3DR.%26aulast%3DScocchi%26aufirst%3DM.%26atitle%3DPEGylation%2520of%2520the%2520peptide%2520Bac7%25281%25E2%2580%259335%2529%2520reduces%2520renal%2520clearance%2520while%2520retaining%2520antibacterial%2520activity%2520and%2520bacterial%2520cell%2520penetration%2520capacity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D95%26spage%3D210%26epage%3D219%26doi%3D10.1016%2Fj.ejmech.2015.03.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zetterberg, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reijmar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pränting, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engström, Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, K.</span></span> <span> </span><span class="NLM_article-title">PEG-stabilized lipid disks as carriers for amphiphilic antimicrobial peptides</span>. <i>J. Controlled Release</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">323</span>– <span class="NLM_lpage">328</span>, <span class="refDoi"> DOI: 10.1016/j.jconrel.2011.08.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.jconrel.2011.08.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=21903146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFGlt7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2011&pages=323-328&author=M.+M.+Zetterbergauthor=K.+Reijmarauthor=M.+Pr%C3%A4ntingauthor=%C3%85.+Engstr%C3%B6mauthor=D.+I.+Anderssonauthor=K.+Edwards&title=PEG-stabilized+lipid+disks+as+carriers+for+amphiphilic+antimicrobial+peptides&doi=10.1016%2Fj.jconrel.2011.08.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">PEG-stabilized lipid disks as carriers for amphiphilic antimicrobial peptides</span></div><div class="casAuthors">Zetterberg, Malin M.; Reijmar, Karin; Praenting, Maria; Engstroem, Aake; Andersson, Dan I.; Edwards, Katarina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Controlled Release</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">323-328</span>CODEN:
                <span class="NLM_cas:coden">JCREEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-3659</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antimicrobial peptides hold potential as a possible alternative, or complement, to conventional antibiotics but new, safe and efficient means are needed for formulation and administration of the peptides.  In this study we have investigated the utility of a novel type of lipid particles, the polyethylene glycol-stabilized lipid disks, as carriers for the model peptide melittin.  The structural integrity of the carrier particle when loaded with the peptide was investigated using cryo-transmission electron microscopy.  Liposome leakage upon addn. of the peptide-lipid disks was monitored as a means to verify the membrane lytic effect of the formulation.  The susceptibility of melittin to tryptic digestion was studied and compared in the absence and presence of lipid disks.  Finally, the antibacterial effect of the peptide-lipid disk formulation was compared to that of free melittin after both single and repeated exposure to Escherichia coli.  The results show that melittin can redistribute from the disk into a new host membrane and that formulation in the disks does not compromise melittin's membrane permeabilizing ability.  Further, the peptide was found to be fully protected against degrdn. when bound to the disks.  Time-kill expts. revealed that all the antibacterial effect of melittin administered in free form was gone after a single exposure to E. coli.  In contrast, the disk formulation showed significant cell-killing effect also upon a second exposure to bacteria, indicating an extended release of peptide from the lipid disks.  These results suggest that the lipid disks constitute a new class of promising carriers for peptide antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdg7EJ-_P-0LVg90H21EOLACvtfcHk0li-tjlR2KLS_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFGlt7%252FJ&md5=ed1374826be651618d7d5af3f0def605</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jconrel.2011.08.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jconrel.2011.08.029%26sid%3Dliteratum%253Aachs%26aulast%3DZetterberg%26aufirst%3DM.%2BM.%26aulast%3DReijmar%26aufirst%3DK.%26aulast%3DPr%25C3%25A4nting%26aufirst%3DM.%26aulast%3DEngstr%25C3%25B6m%26aufirst%3D%25C3%2585.%26aulast%3DAndersson%26aufirst%3DD.%2BI.%26aulast%3DEdwards%26aufirst%3DK.%26atitle%3DPEG-stabilized%2520lipid%2520disks%2520as%2520carriers%2520for%2520amphiphilic%2520antimicrobial%2520peptides%26jtitle%3DJ.%2520Controlled%2520Release%26date%3D2011%26volume%3D156%26spage%3D323%26epage%3D328%26doi%3D10.1016%2Fj.jconrel.2011.08.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ju, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span> <span> </span><span class="NLM_article-title">Combating pseudomonas aeruginosa biofilms by a chitosan-peg-peptide conjugate via changes in assembled structure</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">13731</span>– <span class="NLM_lpage">13738</span>, <span class="refDoi"> DOI: 10.1021/acsami.0c02034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.0c02034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkslWlsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&pages=13731-13738&author=X.+Juauthor=J.+Chenauthor=M.+Zhouauthor=M.+Zhuauthor=Z.+Liauthor=S.+Gaoauthor=J.+Ouauthor=D.+Xuauthor=M.+Wuauthor=S.+Jiangauthor=Y.+Huauthor=Y.+Tianauthor=Z.+Niu&title=Combating+pseudomonas+aeruginosa+biofilms+by+a+chitosan-peg-peptide+conjugate+via+changes+in+assembled+structure&doi=10.1021%2Facsami.0c02034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Pseudomonas aeruginosa Biofilms by a Chitosan-PEG-Peptide Conjugate via Changes in Assembled Structure</span></div><div class="casAuthors">Ju, Xiaoyan; Chen, Jun; Zhou, Mengxue; Zhu, Meng; Li, Zhuang; Gao, Sijia; Ou, Jinzhao; Xu, Dandan; Wu, Man; Jiang, Shidong; Hu, Yi; Tian, Ye; Niu, Zhongwei</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">13731-13738</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pseudomonas aeruginosa (P. aeruginosa) biofilms are assocd. with a wide range of infections, from chronic tissue diseases to implanted medical devices.  In a biofilm, the extracellular polymeric substance (EPS) causes an inhibited penetration of antibacterial agents, leading to a 100-1000 times tolerance of the bacteria.  In view of the water-filled channels in biofilms and the highly neg. charge of EPS, we design a chitosan-polyethylene glycol-peptide conjugate (CS-PEG-LK13) in this study.  The CS-PEG-LK13 prefers a neutrally charged assembly at a size of ~ 100 nm in aq. environment, while undergoes disassembly to expose the α-helical peptide at the bacterial cell membrane.  This behavior provides CS-PEG-LK13 superiorities in both penetrating the biofilms and inactivating the bacteria.  At a concn. of 8 times the min. inhibitory concn., CS-PEG-LK13 has a much higher antibacterial efficiency (72.70%) than LK13 peptide (15.24%) and tobramycin (33.57%) in an in vitro P. aeruginosa biofilm.  Moreover, CS-PEG-LK13 behaves comparable capability of combating an implanted P. aeruginosa biofilm to highly excess tobramycin.  This work has implications for the design of new antibacterial agents in biofilm combating.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG0WyU1O-ZCLVg90H21EOLACvtfcHk0li-tjlR2KLS_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkslWlsb4%253D&md5=fb50d9278b3513a86c97fbe405b3a229</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsami.0c02034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.0c02034%26sid%3Dliteratum%253Aachs%26aulast%3DJu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DOu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DNiu%26aufirst%3DZ.%26atitle%3DCombating%2520pseudomonas%2520aeruginosa%2520biofilms%2520by%2520a%2520chitosan-peg-peptide%2520conjugate%2520via%2520changes%2520in%2520assembled%2520structure%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2020%26volume%3D12%26spage%3D13731%26epage%3D13738%26doi%3D10.1021%2Facsami.0c02034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span> <span> </span><span class="NLM_article-title">PEGylated self-assembled nano-bacitracin A: probing the antibacterial mechanism and real-time tracing of target delivery in vivo</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10688</span>– <span class="NLM_lpage">10705</span>, <span class="refDoi"> DOI: 10.1021/acsami.8b00135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.8b00135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFSqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=10688-10705&author=W.+Hongauthor=Y.+Zhaoauthor=Y.+Guoauthor=C.+Huangauthor=P.+Qiuauthor=J.+Zhuauthor=C.+Chuauthor=H.+Shiauthor=M.+Liu&title=PEGylated+self-assembled+nano-bacitracin+A%3A+probing+the+antibacterial+mechanism+and+real-time+tracing+of+target+delivery+in+vivo&doi=10.1021%2Facsami.8b00135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylated self-assembled nano-bacitracin A: Probing the antibacterial mechanism and real-time tracing of target delivery in vivo</span></div><div class="casAuthors">Hong, Wei; Zhao, Yining; Guo, Yuru; Huang, Chengcheng; Qiu, Peng; Zhu, Jia; Chu, Chun; Shi, Hong; Liu, Mingchun</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">10688-10705</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although nano-self-assemblies of hydrophobic-modified bacitracin A with poly(D,L-lactic-co-glycolic acid) (PLGA) (nano-BAPLGA) have demonstrated promising antibacterial activities, the application of nano-BAPLGA was severely compromised by low water soly.  In this study, a series of PEGylated PLGA copolymers were selected to conjugate with the N-terminus of bacitracin A to construct PEGylated self-assembled nano-BAs and to further develop nano-self-assemblies of bacitracin A with strong antibacterial potency and high soly.  Compared with nano-BAPLGA, all PEGylated nano-BAs, except nano-BA5k, exhibited strong antibacterial efficiency against both Gram-pos. and Gram-neg. bacteria by inducing loss of cytoplasmic membrane potential, membrane permeabilization, and leakage of calcein from artificial cell membranes.  Studies elucidating the underlying mechanism of PEGylated nano-BAs against Gram-neg. bacteria indicated that the strong hydrophobic and van der Waals interactions between PLGA and lipopolysaccharide (LPS) could bind, neutralize, and disassoc. LPS, facilitating cellular uptake of the nanoparticles, which could destabilize the membrane, resulting in cell death.  Moreover, PEGylated nano-BAs (nano-BA12k) with a longer PLGA block were expected to occupy a higher local d. of BA mass on the surface and result in stronger hydrophobic and van der Waals interactions with LPS, which were responsible for the enhanced antibacterial activity against Gram-pos. and emerging antibacterial activity against Gram-neg. bacteria, resp.  In vivo imaging verified that PEGylated nano-BAs exhibited higher inflammatory tissue distribution and longer circulation time than nano-BAPLGA.  Therefore, although PEGylation did not affect antibacterial activity, it is necessary for target delivery and resistance to clearance of the obsd. PEGylated nano-BAs.  In vivo, nano-BA12k also showed the highest therapeutic index against infection burden in a mouse thigh infection model among the tested formulations, which showed good correlation with the in vitro results.  In conclusion, nano-BA12k showed high efficacy in the treatment of invasive infections.  This new approach of constructing nanoantibiotics by modification of com. available antibiotics with PEGylated copolymers is safe, cost-effective, and environmentally friendly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbuxkfEosKprVg90H21EOLACvtfcHk0lhOmui3bkh-RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFSqurg%253D&md5=468e3e10bcae106b7a132f425e76991c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsami.8b00135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.8b00135%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DChu%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DM.%26atitle%3DPEGylated%2520self-assembled%2520nano-bacitracin%2520A%253A%2520probing%2520the%2520antibacterial%2520mechanism%2520and%2520real-time%2520tracing%2520of%2520target%2520delivery%2520in%2520vivo%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2018%26volume%3D10%26spage%3D10688%26epage%3D10705%26doi%3D10.1021%2Facsami.8b00135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, M.</span></span> <span> </span><span class="NLM_article-title">Pegylation: a novel process for modifying pharmacokinetics</span>. <i>Clin. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.2165/00003088-200140070-00005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.2165%2F00003088-200140070-00005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=11510630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3MXms1ars70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=539-551&author=J.+M.+Harrisauthor=N.+E.+Martinauthor=M.+Modi&title=Pegylation%3A+a+novel+process+for+modifying+pharmacokinetics&doi=10.2165%2F00003088-200140070-00005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Pegylation. A novel process for modifying pharmacokinetics</span></div><div class="casAuthors">Harris, J. Milton; Martin, Nancy E.; Modi, Marlene</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Pharmacokinetics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">539-551</span>CODEN:
                <span class="NLM_cas:coden">CPKNDH</span>;
        ISSN:<span class="NLM_cas:issn">0312-5963</span>.
    
            (<span class="NLM_cas:orgname">Adis International Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of liposomal carriers and the modification of therapeutic mols. through the attachment of poly(ethylene glycol) [PEG] moieties ('pegylation') are the most common approaches for enhancing the delivery of parenteral agents.  Although 'classical' liposomes (i.e. phospholipid bilayer vehicles) have been effective in decreasing the clearance of encapsulated agents and in passively targeting specific tissues, they are assocd. with considerable limitations.  Pegylation may be an effective method of delivering therapeutic proteins and modifying their pharmacokinetic properties, in turn modifying pharmacodynamics, via a mechanism dependent on altered binding properties of the native protein.  Pegylation reduces renal clearance and, for some products, results in a more sustained absorption after s.c. administration as well as restricted distribution.  These pharmacokinetic changes may result in more const. and sustained plasma concns., which can lead to increases in clin. effectiveness when the desired effects are concn.-dependent.  Maintaining drug concns. at or near a target concn. for an extended period of time is often clin. advantageous, and is particularly useful in antiviral therapy, since const. antiviral pressure should prevent replication and may thereby suppress the emergence of resistant variants.  Addnl., PEG modification may decrease adverse effects caused by the large variations in peak-to-trough plasma drug concns. assocd. with frequent administration and by the immunogenicity of unmodified proteins.  Pegylated proteins may have reduced immunogenicity because PEG-induced steric hindrance can prevent immune recognition.  Two PEG-modified proteins are currently approved by the US Food and Drug Administration; several others, including cytokines such as interferon-α (IFNα), growth factors and free radical scavengers, are under development.  Careful assessment of various pegylated IFNα products suggests that pegylated mols. can be differentiated on the basis of their pharmacokinetic properties and related changes in pharmacodynamics.  Because the size, geometry and attachment site of the PEG moiety play a crucial role in detg. these properties, therapeutically optimized agents must be designed on a protein-by-protein basis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnVR6JmxSu-bVg90H21EOLACvtfcHk0lhOmui3bkh-RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXms1ars70%253D&md5=58c186af44c9cac1b7d3a90db977383b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.2165%2F00003088-200140070-00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00003088-200140070-00005%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DJ.%2BM.%26aulast%3DMartin%26aufirst%3DN.%2BE.%26aulast%3DModi%26aufirst%3DM.%26atitle%3DPegylation%253A%2520a%2520novel%2520process%2520for%2520modifying%2520pharmacokinetics%26jtitle%3DClin.%2520Pharmacokinet.%26date%3D2001%26volume%3D40%26spage%3D539%26epage%3D551%26doi%3D10.2165%2F00003088-200140070-00005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manhart, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinke, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIntosh, C. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pederson, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demuth, H. U.</span></span> <span> </span><span class="NLM_article-title">Structure-function analysis of a series of novel GIP analogues containing different helical length linkers</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">3081</span>– <span class="NLM_lpage">3088</span>, <span class="refDoi"> DOI: 10.1021/bi026868e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi026868e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1Oiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=3081-3088&author=S.+Manhartauthor=S.+A.+Hinkeauthor=C.+H.+S.+McIntoshauthor=R.+A.+Pedersonauthor=H.+U.+Demuth&title=Structure-function+analysis+of+a+series+of+novel+GIP+analogues+containing+different+helical+length+linkers&doi=10.1021%2Fbi026868e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-function analysis of a series of novel GIP analogues containing different helical length linkers</span></div><div class="casAuthors">Manhart, Susanne; Hinke, Simon A.; McIntosh, Christopher H. S.; Pederson, Raymond A.; Demuth, Hans-Ulrich</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3081-3088</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucose-dependent insulinotropic polypeptide (GIP1-42) is a potent glucose-lowering intestinal peptide hormone.  The equipotent GIP1-30NH2 was structurally modified by linking N- and C-terminal fragments with several different linkers.  Substitution of the middle region of GIP by a flexible aminohexanoic linker resulted in greatly reduced binding affinity and redn. or complete loss of bioactivity.  Connection of the bioactive domains GIP1-14 and GIP19-30NH2 by EKEK or AAAA linkers resulted in peptide agonists with approx. 3-4-fold increased bioactivity as compared to GIP1-30NH2.  Conformational anal. by CD spectroscopy of GIP fragments and analogs suggests a helical region in the C-terminal (19-30) portion of GIP.  It was demonstrated that stabilization of this C-terminal helical region by the introduction of helical linkers favored binding and activation of the GIP receptor.  The authors' results suggest an important contribution of a direct interaction of the first 14 amino acids with the GIP receptor, an appropriate relative orientation of N- and C-terminal parts of GIP, and the presence of helical linkers to be essential for bioactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEznrTXFLdMbVg90H21EOLACvtfcHk0lg15UGBj9DmTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1Oiu7w%253D&md5=90262367e1b4c16303bae79ecf7bc72f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi026868e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi026868e%26sid%3Dliteratum%253Aachs%26aulast%3DManhart%26aufirst%3DS.%26aulast%3DHinke%26aufirst%3DS.%2BA.%26aulast%3DMcIntosh%26aufirst%3DC.%2BH.%2BS.%26aulast%3DPederson%26aufirst%3DR.%2BA.%26aulast%3DDemuth%26aufirst%3DH.%2BU.%26atitle%3DStructure-function%2520analysis%2520of%2520a%2520series%2520of%2520novel%2520GIP%2520analogues%2520containing%2520different%2520helical%2520length%2520linkers%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D3081%26epage%3D3088%26doi%3D10.1021%2Fbi026868e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyengar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K. L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suwatthee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. Y. C.</span></span> <span> </span><span class="NLM_article-title">Beyond electrostatics: antimicrobial peptide selectivity and the influence of cholesterol-mediated fluidity and lipid chain length on protegrin-1 activity</span>. <i>Biochim. Biophys. Acta-Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1861</i></span>,  <span class="NLM_fpage">182977</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2019.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2019.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=31077677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtF2nsLbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1861&publication_year=2019&pages=182977&author=J.+M.+Hendersonauthor=N.+S.+Iyengarauthor=K.+L.+H.+Lamauthor=E.+Maldonadoauthor=T.+Suwattheeauthor=I.+Royauthor=A.+J.+Waringauthor=K.+Y.+C.+Lee&title=Beyond+electrostatics%3A+antimicrobial+peptide+selectivity+and+the+influence+of+cholesterol-mediated+fluidity+and+lipid+chain+length+on+protegrin-1+activity&doi=10.1016%2Fj.bbamem.2019.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond electrostatics: antimicrobial peptide selectivity and the influence of cholesterol-mediated fluidity and lipid chain length on protegrin-1 activity</span></div><div class="casAuthors">Henderson, J. Michael; Iyengar, Nishanth S.; Lam, Kin Lok H.; Maldonado, Eddie; Suwatthee, Tiffany; Roy, Indroneil; Waring, Alan J.; Lee, Ka Yee C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1861</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">182977</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs) are a promising class of innate host defense mols. for next-generation antibiotics, as they uniquely target and permeabilize membranes of pathogens.  This selectivity has been explained by the electrostatic attraction between these predominantly cationic peptides and the bacterial membrane, which is heavily populated with anionic lipids.  However, AMP-resistant bacteria have non-electrostatic countermeasures that modulate membrane rigidity and thickness.  We explore how variations in phys. properties affect the membrane affinity and disruption process of protegrin-1 (PG-1) in phosphatidylcholine (PC) membranes with altered lipid packing densities and thicknesses.  From isothermal titrn. calorimetry and at. force microscopy, our results showed that PG-1 could no longer insert into membranes of increasing cholesterol amts. nor into monounsatd. PC membranes of increasing thicknesses with similar fluidities.  Prevention of PG-1's incorporation consequently made the membranes more resistant to peptide-induced structural transformations like pore formation.  Our study provides evidence that AMP affinity and activity are strongly correlated with the fluidity and thickness of the membrane.  A basic understanding of how phys. mechanisms can regulate cell selectivity and resistance towards AMPs will aid in the development of new antimicrobial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS8H_GzFo-vbVg90H21EOLACvtfcHk0lg15UGBj9DmTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtF2nsLbL&md5=6d577884896e9306890dbd4db1dad6a9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2019.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2019.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DJ.%2BM.%26aulast%3DIyengar%26aufirst%3DN.%2BS.%26aulast%3DLam%26aufirst%3DK.%2BL.%2BH.%26aulast%3DMaldonado%26aufirst%3DE.%26aulast%3DSuwatthee%26aufirst%3DT.%26aulast%3DRoy%26aufirst%3DI.%26aulast%3DWaring%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DK.%2BY.%2BC.%26atitle%3DBeyond%2520electrostatics%253A%2520antimicrobial%2520peptide%2520selectivity%2520and%2520the%2520influence%2520of%2520cholesterol-mediated%2520fluidity%2520and%2520lipid%2520chain%2520length%2520on%2520protegrin-1%2520activity%26jtitle%3DBiochim.%2520Biophys.%2520Acta-Biomembr.%26date%3D2019%26volume%3D1861%26spage%3D182977%26doi%3D10.1016%2Fj.bbamem.2019.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maystrenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">The addition of a synthetic LPS-targeting domain improves serum stability while maintaining antimicrobial, antibiofilm, and cell stimulating properties of an antimicrobial peptide</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1014</span>, <span class="refDoi"> DOI: 10.3390/biom10071014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.3390%2Fbiom10071014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1ChsLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=1014&author=A.+Maystrenkoauthor=Y.+Fengauthor=N.+Akhtarauthor=J.+Li&title=The+addition+of+a+synthetic+LPS-targeting+domain+improves+serum+stability+while+maintaining+antimicrobial%2C+antibiofilm%2C+and+cell+stimulating+properties+of+an+antimicrobial+peptide&doi=10.3390%2Fbiom10071014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The addition of a synthetic LPS-targeting domain improves serum stability while maintaining antimicrobial, antibiofilm, and cell stimulating properties of an antimicrobial peptide</span></div><div class="casAuthors">Maystrenko, Anna; Feng, Yulong; Akhtar, Nadeem; Li, Julang</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1014</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Multi-drug resistant (MDR) bacteria and their biofilms are a concern in veterinary and human medicine.  Protegrin-1 (PG-1), a potent antimicrobial peptide (AMP) with antimicrobial and immunomodulatory properties, is considered a potential substitute for conventional antibiotics.  To improve stability of PG-1, a hybrid peptide, SynPG-1, was designed.  The antimicrobial and antibiofilm properties of PG-1 and the PG-1 hybrid against MDR pathogens was analyzed, and activity after incubation with physiol. fluids was compared.  The effects of these peptides on the IPEC-J2 cell line was also investigated.  While PG-1 maintained some activity in 25% serum for 2 h, SynPG-1 was able to retain activity in the same condition for up to 24 h, representing a 12-fold increase in stability.  Both peptides had some antibiofilm activity against Escherichia coli and Salmonella typhimurium.  While both peptides prevented biofilm formation of methicillin-resistant Staphylococcus aureus (MRSA), neither could destroy MRSA's pre-formed biofilms.  Both peptides maintained activity after incubation with trypsin and porcine gastric fluid, but not intestinal fluid, and stimulated IPEC-J2 cell migration.  These findings suggest that SynPG-1 has much better serum stability while maintaining the same antimicrobial potency as PG-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrib7YXFKncyrVg90H21EOLACvtfcHk0lg15UGBj9DmTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1ChsLfJ&md5=0f8593beb414571b3ea239fc7c0276bd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3390%2Fbiom10071014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom10071014%26sid%3Dliteratum%253Aachs%26aulast%3DMaystrenko%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DAkhtar%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DThe%2520addition%2520of%2520a%2520synthetic%2520LPS-targeting%2520domain%2520improves%2520serum%2520stability%2520while%2520maintaining%2520antimicrobial%252C%2520antibiofilm%252C%2520and%2520cell%2520stimulating%2520properties%2520of%2520an%2520antimicrobial%2520peptide%26jtitle%3DBiomolecules%26date%3D2020%26volume%3D10%26spage%3D1014%26doi%3D10.3390%2Fbiom10071014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moravej, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moravej, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yazdanparast, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heiat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirhosseini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosazadeh Moghaddam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirnejad, R.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides: features, action, and their resistance mechanisms in bacteria</span>. <i>Microb. Drug Resist.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">747</span>– <span class="NLM_lpage">767</span>, <span class="refDoi"> DOI: 10.1089/mdr.2017.0392</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1089%2Fmdr.2017.0392" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=29957118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht12mtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=747-767&author=H.+Moravejauthor=Z.+Moravejauthor=M.+Yazdanparastauthor=M.+Heiatauthor=A.+Mirhosseiniauthor=M.+Moosazadeh+Moghaddamauthor=R.+Mirnejad&title=Antimicrobial+peptides%3A+features%2C+action%2C+and+their+resistance+mechanisms+in+bacteria&doi=10.1089%2Fmdr.2017.0392"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial Peptides: Features, Action, and Their Resistance Mechanisms in Bacteria</span></div><div class="casAuthors">Moravej, Hoda; Moravej, Zahra; Yazdanparast, Maryam; Heiat, Mohammad; Mirhosseini, Ali; Moosazadeh Moghaddam, Mehrdad; Mirnejad, Reza</div><div class="citationInfo"><span class="NLM_cas:title">Microbial Drug Resistance (New Rochelle, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">747-767</span>CODEN:
                <span class="NLM_cas:coden">MDREFJ</span>;
        ISSN:<span class="NLM_cas:issn">1076-6294</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">A review.  In recent years, because of increased resistance to conventional antimicrobials, many researchers have started to study the synthesis of new antibiotics to control the disease-causing effects of infectious pathogens.  Antimicrobial peptides (AMPs) are among the newest antibiotics; these peptides are integral compds. in all kinds of organisms and play a significant role in microbial ecol., and critically contribute to the innate immunity of organisms by destroying invading microorganisms.  Moreover, AMPs may encourage cells to produce chemokines, stimulate angiogenesis, accelerate wound healing, and influence programmed cell death in multicellular organisms.  Bacteria differ in their inherent susceptibility and resistance mechanisms to these peptides when responding to the antimicrobial effects of AMPs.  Generally, the development of AMP resistance mechanisms is driven by direct competition between bacterial species, and host and pathogen interactions.  Several studies have shown diverse mechanisms of bacterial resistance to AMPs, for example, some bacteria produce proteases and trapping proteins; some modify cell surface charge, change membrane fluidity, and activate efflux pumps; and some species make use of biofilms and exopolymers, and develop sensing systems by selective gene expression.  A closer understanding of bacterial resistance mechanisms may help in developing novel therapeutic approaches for the treatment of infections caused by pathogenic organisms that are successful in developing extensive resistance to AMPs.  Based on these observations, this review discusses the properties of AMPs, their targeting mechanisms, and bacterial resistance mechanisms against AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpRMQ0V4yVbVg90H21EOLACvtfcHk0lifFMLpH-4mzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht12mtrfF&md5=176913027ea9b055f92550095dde962f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1089%2Fmdr.2017.0392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fmdr.2017.0392%26sid%3Dliteratum%253Aachs%26aulast%3DMoravej%26aufirst%3DH.%26aulast%3DMoravej%26aufirst%3DZ.%26aulast%3DYazdanparast%26aufirst%3DM.%26aulast%3DHeiat%26aufirst%3DM.%26aulast%3DMirhosseini%26aufirst%3DA.%26aulast%3DMoosazadeh%2BMoghaddam%26aufirst%3DM.%26aulast%3DMirnejad%26aufirst%3DR.%26atitle%3DAntimicrobial%2520peptides%253A%2520features%252C%2520action%252C%2520and%2520their%2520resistance%2520mechanisms%2520in%2520bacteria%26jtitle%3DMicrob.%2520Drug%2520Resist.%26date%3D2018%26volume%3D24%26spage%3D747%26epage%3D767%26doi%3D10.1089%2Fmdr.2017.0392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, B.</span></span> <span> </span><span class="NLM_article-title">Self-assembling surfactant-like peptide A(6)K as potential delivery system for hydrophobic drugs</span>. <i>Int. J. Nanomed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">858</span>, <span class="refDoi"> DOI: 10.2147/IJN.S71696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.2147%2FIJN.S71696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=25670898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislaiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=847-858&author=F.+Qiuauthor=Y.+Chenauthor=C.+Tangauthor=J.+Zhangauthor=M.+Gongauthor=B.+Su&title=Self-assembling+surfactant-like+peptide+A%286%29K+as+potential+delivery+system+for+hydrophobic+drugs&doi=10.2147%2FIJN.S71696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Self-assembling surfactant-like peptide A6K as potential delivery system for hydrophobic drugs</span></div><div class="casAuthors">Chen, Yongzhu; Tang, Chengkang; Zhang, Jie; Gong, Meng; Su, Bo; Qiu, Feng</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Nanomedicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">847-858/1-847-858/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">IJNNHQ</span>;
        ISSN:<span class="NLM_cas:issn">1178-2013</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">Background: Finding a suitable delivery system to improve the water soly. of hydrophobic drugs is a crit. challenge in the development of effective formulations.  In this study, we used A6K, a self-assembling surfactant-like peptide, as a carrier to encapsulate and deliver hydrophobic pyrene.  Methods: Pyrene was mixed with A6K by magnetic stirring to form a suspension.  Confocal laser scanning microscopy, TEM, dynamic light scattering, at. force microscopy, fluorescence, and cell uptake measurements were carried out to study the features and stability of the nanostructures, the state and content of pyrene, as well as the pyrene release profile.  Results: The suspension formed contained pyrene monomers trapped in the hydrophobic cores of the micellar nanofibers formed by A6K, as well as nanosized pyrene crystals wrapped up and stabilized by the nanofibers.  The two different encapsulation methods greatly increased the concn. of pyrene in the suspension, and formation of pyrene crystals wrapped up by A6K nanofibers might be the major contributor to this effect.  Furthermore, the suspension system could readily release and transfer pyrene into living cells.  Conclusion: A6K could be further exploited as a promising delivery system for hydrophobic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxCLW4xvdgnrVg90H21EOLACvtfcHk0lifFMLpH-4mzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislaiu74%253D&md5=9bb78ad7c7acc4d5523f35f8353c97d8</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2147%2FIJN.S71696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FIJN.S71696%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGong%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DB.%26atitle%3DSelf-assembling%2520surfactant-like%2520peptide%2520A%25286%2529K%2520as%2520potential%2520delivery%2520system%2520for%2520hydrophobic%2520drugs%26jtitle%3DInt.%2520J.%2520Nanomed.%26date%3D2015%26volume%3D10%26spage%3D847%26epage%3D858%26doi%3D10.2147%2FIJN.S71696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holtzmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, M. F.</span></span> <span> </span><span class="NLM_article-title">Self-assembly of surfactant-like peptides</span>. <i>Langmuir</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">12729</span>– <span class="NLM_lpage">12736</span>, <span class="refDoi"> DOI: 10.1021/la7011183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/la7011183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Ohsb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=12729-12736&author=D.+J.+Adamsauthor=K.+Holtzmannauthor=C.+Schneiderauthor=M.+F.+Butler&title=Self-assembly+of+surfactant-like+peptides&doi=10.1021%2Fla7011183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Assembly of Surfactant-like Peptides</span></div><div class="casAuthors">Adams, Dave J.; Holtzmann, Kathrin; Schneider, Christian; Butler, Michael F.</div><div class="citationInfo"><span class="NLM_cas:title">Langmuir</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">12729-12736</span>CODEN:
                <span class="NLM_cas:coden">LANGD5</span>;
        ISSN:<span class="NLM_cas:issn">0743-7463</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inspired by recent work describing surfactant-like peptides, we have carried out a systematic study on peptides with the underlying compn. of V6D2, altering the abs. sequence to det. the importance of the surfactant-like structure.  All of the peptides examd. here formed self-assembled structures in water.  However, in contrast to other reports, we have found a surprising diversity of structures including fibers, tapes, and twisted ribbons but an absence of the vesicles and nanotubes described previously.  Further investigations demonstrated that peptide purity plays a significant role in the outcome of the self-assembly.  Different batches behave very differently, which can be linked to the compns. of these batches.  This work shows that there is a need for not only rational design but also ease of synthesis of the building blocks for self-assembled structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZV4rvg4WlhrVg90H21EOLACvtfcHk0lifFMLpH-4mzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Ohsb%252FM&md5=4b128d1ff6f9d8e4df05cd6e71e64401</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fla7011183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fla7011183%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DD.%2BJ.%26aulast%3DHoltzmann%26aufirst%3DK.%26aulast%3DSchneider%26aufirst%3DC.%26aulast%3DButler%26aufirst%3DM.%2BF.%26atitle%3DSelf-assembly%2520of%2520surfactant-like%2520peptides%26jtitle%3DLangmuir%26date%3D2007%26volume%3D23%26spage%3D12729%26epage%3D12736%26doi%3D10.1021%2Fla7011183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perinelli, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vllasaliu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doutch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmieri, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casettari, L.</span></span> <span> </span><span class="NLM_article-title">PEGylation affects the self-assembling behaviour of amphiphilic octapeptides</span>. <i>Int. J. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>571</i></span>, <span class="NLM_elocation-id">118752</span> <span class="refDoi"> DOI: 10.1016/j.ijpharm.2019.118752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.ijpharm.2019.118752" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=31606529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVGlsL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2019&author=D.+R.+Perinelliauthor=M.+Campanaauthor=I.+Singhauthor=D.+Vllasaliuauthor=J.+Doutchauthor=G.+F.+Palmieriauthor=L.+Casettari&title=PEGylation+affects+the+self-assembling+behaviour+of+amphiphilic+octapeptides&doi=10.1016%2Fj.ijpharm.2019.118752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylation affects the self-assembling behaviour of amphiphilic octapeptides</span></div><div class="casAuthors">Perinelli, Diego Romano; Campana, Mario; Singh, Ishwar; Vllasaliu, Driton; Doutch, James; Palmieri, Giovanni Filippo; Casettari, Luca</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Pharmaceutics (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">571</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">118752</span>CODEN:
                <span class="NLM_cas:coden">IJPHDE</span>;
        ISSN:<span class="NLM_cas:issn">0378-5173</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Surfactant-like peptides are a class of amphiphilic macromols., which are able to self-assemble in water forming different supramol. structures.  Among them, octapeptides composed of six hydrophobic and two hydrophilic residues have attracted interest since they have a length similar to those of natural phospholipids.  Supramol. structures of different amphiphilic octapeptides have been widely reported, but no study has been performed aimed at investigating the effect of PEGylation on their self-assembling behavior.  The aim of the present work was to synthesize and characterize the self-assembling behavior of PEGylated alanine- or valine based amphiphilic octapeptides (mPEG1.9kDa-DDAAAAAA and mPEG1.9kDa-DDVVVVVV) in comparison to the non-PEGylated ones (DDAAAAAA and DDVVVVVV).  The self-aggregation process in ultrapure water was investigated by fluorescence spectroscopy, small angle neutron scattering (SANS), dynamic light scattering (DLS), while the secondary structure was assessed by CD.  PEGylation markedly affects the self-assembling behavior of these amphiphilic octapeptides in terms of both crit. aggregation concn. (CAC) and shape of the formed supramol. aggregates.  Indeed, PEGylation increases CAC and prevents the self-aggregation into fibrillary supramol. aggregates (as obsd. for non-PEGylated peptides), by promoting the formation of micelle-like structures (as demonstrated for valine-based octapeptide).  On the other side, the secondary structure of peptides seems not to be affected by PEGylation.  Overall, these results suggest that self-assembling behavior of amphiphilic octapeptides can be modified by PEGylation, with a great potential impact for the future applications of these nanomaterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotvNRv-0fPXrVg90H21EOLACvtfcHk0lghJMPlg0oGHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVGlsL%252FN&md5=091bf55552cd3ca00eaaba8e0b69700a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ijpharm.2019.118752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijpharm.2019.118752%26sid%3Dliteratum%253Aachs%26aulast%3DPerinelli%26aufirst%3DD.%2BR.%26aulast%3DCampana%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DI.%26aulast%3DVllasaliu%26aufirst%3DD.%26aulast%3DDoutch%26aufirst%3DJ.%26aulast%3DPalmieri%26aufirst%3DG.%2BF.%26aulast%3DCasettari%26aufirst%3DL.%26atitle%3DPEGylation%2520affects%2520the%2520self-assembling%2520behaviour%2520of%2520amphiphilic%2520octapeptides%26jtitle%3DInt.%2520J.%2520Pharm.%26date%3D2019%26volume%3D571%26doi%3D10.1016%2Fj.ijpharm.2019.118752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Combined systems approaches reveal a multistage mode of action of a marine antimicrobial peptide against pathogenic escherichia coli and its protective effect against bacterial peritonitis and endotoxemia</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>, <span class="NLM_elocation-id">e01056-16</span> <span class="refDoi"> DOI: 10.1128/AAC.01056-16</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1128%2FAAC.01056-16" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=27795369" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&author=X.+Wangauthor=D.+Tengauthor=R.+Maoauthor=N.+Yangauthor=Y.+Haoauthor=J.+Wang&title=Combined+systems+approaches+reveal+a+multistage+mode+of+action+of+a+marine+antimicrobial+peptide+against+pathogenic+escherichia+coli+and+its+protective+effect+against+bacterial+peritonitis+and+endotoxemia&doi=10.1128%2FAAC.01056-16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1128%2FAAC.01056-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01056-16%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DTeng%26aufirst%3DD.%26aulast%3DMao%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DN.%26aulast%3DHao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DCombined%2520systems%2520approaches%2520reveal%2520a%2520multistage%2520mode%2520of%2520action%2520of%2520a%2520marine%2520antimicrobial%2520peptide%2520against%2520pathogenic%2520escherichia%2520coli%2520and%2520its%2520protective%2520effect%2520against%2520bacterial%2520peritonitis%2520and%2520endotoxemia%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26doi%3D10.1128%2FAAC.01056-16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroll, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span> <span> </span><span class="NLM_article-title">Self-assembled peptide nanofibers display natural antimicrobial peptides to selectively kill bacteria without compromising cytocompatibility</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">28681</span>– <span class="NLM_lpage">28689</span>, <span class="refDoi"> DOI: 10.1021/acsami.9b09583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.9b09583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVSqtr%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=28681-28689&author=W.+Chenauthor=S.+Yangauthor=S.+Liauthor=J.+C.+Langauthor=C.+Maoauthor=P.+Krollauthor=L.+Tangauthor=H.+Dong&title=Self-assembled+peptide+nanofibers+display+natural+antimicrobial+peptides+to+selectively+kill+bacteria+without+compromising+cytocompatibility&doi=10.1021%2Facsami.9b09583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Assembled Peptide Nanofibers Display Natural Antimicrobial Peptides to Selectively Kill Bacteria without Compromising Cytocompatibility</span></div><div class="casAuthors">Chen, Weike; Yang, Su; Li, Shuxin; Lang, John C.; Mao, Chuanbin; Kroll, Peter; Tang, Liping; Dong, He</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">28681-28689</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">One of the major hurdles in the development of antimicrobial peptide (AMP)-based materials is their poor capacity in selectively killing bacteria without harming nearby mammalian cells.  Namely, they are antimicrobial but cytotoxic.  Current methods of nanoparticle-encapsulated AMPs to target bacteria selectively still have not yet overcome this hurdle.  Here we demonstrate a simple yet effective method to address this daunting challenge by assocg. a natural AMP with a β-sheet-forming synthetic peptide.  The integrated peptides self-assembled to form a supramol. nanofiber, resulting in the presentation of the AMP at the nanofiber-solvent interface in a precisely controlled manner.  Using melittin as a model natural AMP, we found that the conformation of melittin changed dramatically when presented on the nanofiber surface, which in turn modulated the induced membrane permeability of the bacterial and mammalian cell membranes.  Specifically, the presentation of melittin on the nanofiber restricted its hydrophobic residues, leading to a redn. of the hydrophobic interaction with lipids in the cell membranes.  Compellingly, the reduced hydrophobic interaction led to a considerably decrease of melittin's induced permeability of the mammalian cell membrane than that of the bacterial cell membrane.  As a result, the AMP-displaying nanofiber preferentially permeabilized and disrupted the membrane of the bacteria without compromising the mammalian cells.  Such improved membrane selectivity and cytocompatibility were confirmed in a cell-based membrane localization and live-dead assay.  Our new strategy holds great promise for fabricating cytocompatible antimicrobial assemblies that offer safer and more effective administration of therapeutic AMPs.  These assemblies, with intrinsic antimicrobial activity and cytocompatibility, can also serve as building blocks for the construction of higher-ordered scaffolds for other biomedical applications such as tissue engineering and regenerative medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojN6nRcrufsbVg90H21EOLACvtfcHk0lghJMPlg0oGHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVSqtr%252FI&md5=c32908aa80f1e5511bfea1be9cf602a3</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facsami.9b09583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.9b09583%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLang%26aufirst%3DJ.%2BC.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DKroll%26aufirst%3DP.%26aulast%3DTang%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DH.%26atitle%3DSelf-assembled%2520peptide%2520nanofibers%2520display%2520natural%2520antimicrobial%2520peptides%2520to%2520selectively%2520kill%2520bacteria%2520without%2520compromising%2520cytocompatibility%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2019%26volume%3D11%26spage%3D28681%26epage%3D28689%26doi%3D10.1021%2Facsami.9b09583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dehsorkhi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletto, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamley, I. W.</span></span> <span> </span><span class="NLM_article-title">Self-assembling amphiphilic peptides</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1002/psc.2633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1002%2Fpsc.2633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=24729276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVCltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=453-467&author=A.+Dehsorkhiauthor=V.+Castellettoauthor=I.+W.+Hamley&title=Self-assembling+amphiphilic+peptides&doi=10.1002%2Fpsc.2633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Self-assembling amphiphilic peptides</span></div><div class="casAuthors">Dehsorkhi, Ashkan; Castelletto, Valeria; Hamley, Ian W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">453-467</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">A review.  The self-assembly of several classes of amphiphilic peptides is reviewed, and selected applications are discussed.  The authors discuss recent work on the self-assembly of lipopeptides, surfactant-like peptides, and amyloid peptides derived from the amyloid-β peptide.  The influence of environmental variables such as pH and temp. on aggregate nanostructure is discussed.  Enzyme-induced remodeling due to peptide cleavage and nanostructure control through photocleavage or photocrosslinking are also considered.  Lastly, selected applications of amphiphilic peptides in biomedicine and materials science are outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobEYsURgvLhbVg90H21EOLACvtfcHk0lgdBbBvNQVMkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVCltLw%253D&md5=cb940bade3d9a1b06b36d0f76e8b5964</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2633%26sid%3Dliteratum%253Aachs%26aulast%3DDehsorkhi%26aufirst%3DA.%26aulast%3DCastelletto%26aufirst%3DV.%26aulast%3DHamley%26aufirst%3DI.%2BW.%26atitle%3DSelf-assembling%2520amphiphilic%2520peptides%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2014%26volume%3D20%26spage%3D453%26epage%3D467%26doi%3D10.1002%2Fpsc.2633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guiotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzobon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manganelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, F. M.</span></span> <span> </span><span class="NLM_article-title">PEGylation of the antimicrobial peptide nisin A: problems and perspectives</span>. <i>Farmaco</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/S0014-827X(02)01301-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2FS0014-827X%2802%2901301-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=12595036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3sXht1KrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2003&pages=45-50&author=A.+Guiottoauthor=M.+Pozzobonauthor=M.+Canevariauthor=R.+Manganelliauthor=M.+Scarinauthor=F.+M.+Veronese&title=PEGylation+of+the+antimicrobial+peptide+nisin+A%3A+problems+and+perspectives&doi=10.1016%2FS0014-827X%2802%2901301-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PEGylation of the antimicrobial peptide nisin A: problems and perspectives</span></div><div class="casAuthors">Guiotto, Andrea; Pozzobon, Michela; Canevari, Mirta; Manganelli, Riccardo; Scarin, Marino; Veronese, Francesco M.</div><div class="citationInfo"><span class="NLM_cas:title">Farmaco</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">45-50</span>CODEN:
                <span class="NLM_cas:coden">FRMCE8</span>;
        ISSN:<span class="NLM_cas:issn">0014-827X</span>.
    
            (<span class="NLM_cas:orgname">Editions Scientifiques et Medicales Elsevier</span>)
        </div><div class="casAbstract">Nisin is a natural antimicrobial peptide produced by Lactococcus lactis and widely employed as food preservative.  Its low soly. in neutral aq. solns., its instability at physiol. pH and its rapid breakdown by proteolytic enzymes has limited its use for processed foods (processed cheese, milk and derivs., canned vegetables).  The conjugation to poly(ethylene glycol) (PEG) could improve its soly. and protect it towards enzymes present in non optimally processed food.  The authors report the synthesis of a PEG-nisin conjugate, and the microbiol. assays against some bacterial cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroZP0JJBl3TbVg90H21EOLACvtfcHk0lgdBbBvNQVMkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXht1KrsLw%253D&md5=879c1320913de7a8c4dd91977e8ac7ed</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0014-827X%2802%2901301-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-827X%252802%252901301-0%26sid%3Dliteratum%253Aachs%26aulast%3DGuiotto%26aufirst%3DA.%26aulast%3DPozzobon%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DM.%26aulast%3DManganelli%26aufirst%3DR.%26aulast%3DScarin%26aufirst%3DM.%26aulast%3DVeronese%26aufirst%3DF.%2BM.%26atitle%3DPEGylation%2520of%2520the%2520antimicrobial%2520peptide%2520nisin%2520A%253A%2520problems%2520and%2520perspectives%26jtitle%3DFarmaco%26date%3D2003%26volume%3D58%26spage%3D45%26epage%3D50%26doi%3D10.1016%2FS0014-827X%2802%2901301-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciumac, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. R.</span></span> <span> </span><span class="NLM_article-title">Membrane targeting cationic antimicrobial peptides</span>. <i>J. Colloid Interface Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1016/j.jcis.2018.10.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.jcis.2018.10.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=30439615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKlsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2019&pages=163-185&author=D.+Ciumacauthor=H.+Gongauthor=X.+Huauthor=J.+R.+Lu&title=Membrane+targeting+cationic+antimicrobial+peptides&doi=10.1016%2Fj.jcis.2018.10.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane targeting cationic antimicrobial peptides</span></div><div class="casAuthors">Ciumac, Daniela; Gong, Haoning; Hu, Xuzhi; Lu, Jian Ren</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Colloid and Interface Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">163-185</span>CODEN:
                <span class="NLM_cas:coden">JCISA5</span>;
        ISSN:<span class="NLM_cas:issn">0021-9797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Many short cationic peptides are amphiphilic and are often termed antimicrobial peptides (AMPs) as they can kill various microorganisms.  These AMPs have largely been discovered from nature, but over the past two decades many biomimetic and de novo designed AMPs have been reported, offering a huge variety of attractive properties for further exploitation.  Under the current global endeavour of fighting against antimicrobial resistance, it is useful to introduce AMPs to the biointerface research community and compare their modes of action with conventional antibiotics.  Because natural AMPs often have long sequences and other biol. functions implicated, they can't be used as antimicrobial agents.  However, rational AMP design helps eliminate their shortcomings and more importantly, optimize their structure-function relationship.  This review will first introduce the key approaches recently utilized in structural design of AMPs and then introduce the main lipid membrane models such as spread lipid monolayers and vesicles together with the characterization techniques adopted in early AMP design and development.  These studies are crucial towards understanding key factors affecting their efficacy and toxicity.  Thus, various interfacial measurements facilitated by different forms of lipid monolayers and bilayers provide valuable support to the selective responses of AMPs to different cell types used in bactericidal assays and cytotoxicity tests, emphasizing the link between mol. models and cell models.  A no. of clin. trials of AMPs have been either under way or completed, demonstrating the huge potential of AMPs in a range of applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpjYgn14ge7rVg90H21EOLACvtfcHk0lgdBbBvNQVMkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKlsL%252FL&md5=5608475b4023dc6b74c3c0e4d19aeb25</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jcis.2018.10.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jcis.2018.10.103%26sid%3Dliteratum%253Aachs%26aulast%3DCiumac%26aufirst%3DD.%26aulast%3DGong%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DJ.%2BR.%26atitle%3DMembrane%2520targeting%2520cationic%2520antimicrobial%2520peptides%26jtitle%3DJ.%2520Colloid%2520Interface%2520Sci.%26date%3D2019%26volume%3D537%26spage%3D163%26epage%3D185%26doi%3D10.1016%2Fj.jcis.2018.10.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walkenhorst, W. F.</span></span> <span> </span><span class="NLM_article-title">Using adjuvants and environmental factors to modulate the activity of antimicrobial peptides</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1858</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2015.12.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2015.12.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=26751595" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvFyhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1858&publication_year=2016&pages=926-935&author=W.+F.+Walkenhorst&title=Using+adjuvants+and+environmental+factors+to+modulate+the+activity+of+antimicrobial+peptides&doi=10.1016%2Fj.bbamem.2015.12.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Using adjuvants and environmental factors to modulate the activity of antimicrobial peptides</span></div><div class="casAuthors">Walkenhorst, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1858</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">926-935</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The increase in antibiotic resistant and multi-drug resistant bacterial infections has serious implications for the future of health care.  The difficulty in finding both new microbial targets and new drugs against existing targets adds to the concern.  The use of combination and adjuvant therapies are potential strategies to counter this threat.  Antimicrobial peptides (AMPs) are a promising class of antibiotics (ABs), particularly for topical and surface applications.  Efforts have been directed toward a no. of strategies, including the use of conventional ABs combined with AMPs, and the use of potentiating agents to increase the performance of AMPs.  This review focuses on combination strategies such as adjuvants and the manipulation of environmental variables to improve the efficacy of AMPs as potential therapeutic agents.  This article is part of a Special Issue entitled: Antimicrobial peptides edited by Karl Lohner and Kai Hilpert.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNZKfoFuXLBrVg90H21EOLACvtfcHk0ljbJ6GkHKROmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvFyhtw%253D%253D&md5=efa08e283dd66737a3dc2bfe635e2b5a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2015.12.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2015.12.034%26sid%3Dliteratum%253Aachs%26aulast%3DWalkenhorst%26aufirst%3DW.%2BF.%26atitle%3DUsing%2520adjuvants%2520and%2520environmental%2520factors%2520to%2520modulate%2520the%2520activity%2520of%2520antimicrobial%2520peptides%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2016%26volume%3D1858%26spage%3D926%26epage%3D935%26doi%3D10.1016%2Fj.bbamem.2015.12.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leroux, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauthier, M. A.</span></span> <span> </span><span class="NLM_article-title">Releasable and traceless pegylation of arginine-rich antimicrobial peptides</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4082</span>– <span class="NLM_lpage">4086</span>, <span class="refDoi"> DOI: 10.1039/C7SC00770A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1039%2FC7SC00770A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=30155213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsVakur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4082-4086&author=Y.+Gongauthor=D.+Andinaauthor=S.+Naharauthor=J.+C.+Lerouxauthor=M.+A.+Gauthier&title=Releasable+and+traceless+pegylation+of+arginine-rich+antimicrobial+peptides&doi=10.1039%2FC7SC00770A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Releasable and traceless PEGylation of arginine-rich antimicrobial peptides</span></div><div class="casAuthors">Gong, Y.; Andina, D.; Nahar, S.; Leroux, J.-C.; Gauthier, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4082-4086</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Arginine-rich antimicrobial peptides (AMPs) are emerging therapeutics of interest.  However, their applicability is limited by their short circulation half-life, caused in part by their small size and digestion by blood proteases.  This study reports a strategy to temporarily mask arginine residues within AMPs with methoxy poly(ethylene glycol).  Based on the reagent used, release of AMPs occurred in hours to days in a completely traceless fashion.  In vitro, conjugates were insensitive to serum proteases, and released native AMP with full in vitro bioactivity.  This strategy is thus highly relevant and should be adaptable to the entire family of arginine-rich AMPs.  It may potentially be used to improve AMP-therapies by providing a more steady concn. of AMP in the blood after a single injection, avoiding toxic effects at high AMP doses, and reducing the no. of doses required over the treatment duration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjON13iKApULVg90H21EOLACvtfcHk0ljbJ6GkHKROmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsVakur4%253D&md5=ce26e313c6d8779e5c48a86d80eb47ef</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2FC7SC00770A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC00770A%26sid%3Dliteratum%253Aachs%26aulast%3DGong%26aufirst%3DY.%26aulast%3DAndina%26aufirst%3DD.%26aulast%3DNahar%26aufirst%3DS.%26aulast%3DLeroux%26aufirst%3DJ.%2BC.%26aulast%3DGauthier%26aufirst%3DM.%2BA.%26atitle%3DReleasable%2520and%2520traceless%2520pegylation%2520of%2520arginine-rich%2520antimicrobial%2520peptides%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D4082%26epage%3D4086%26doi%3D10.1039%2FC7SC00770A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinnunen, P. K. J.</span></span> <span> </span><span class="NLM_article-title">Cholesterol, lanosterol, and ergosterol attenuate the membrane association of LL-37(W27F) and temporin L</span>. <i>Biochim. Biophys. Acta-Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>1778</i></span>,  <span class="NLM_fpage">1460</span>– <span class="NLM_lpage">1466</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2008.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2008.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=18358828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVaqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1778&publication_year=2008&pages=1460-1466&author=R.+Soodauthor=P.+K.+J.+Kinnunen&title=Cholesterol%2C+lanosterol%2C+and+ergosterol+attenuate+the+membrane+association+of+LL-37%28W27F%29+and+temporin+L&doi=10.1016%2Fj.bbamem.2008.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Cholesterol, lanosterol, and ergosterol attenuate the membrane association of LL-37(W27F) and temporin L</span></div><div class="casAuthors">Sood, Rohit; Kinnunen, Paavo K. J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">1778</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1460-1466</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Sterols impart significant changes to the biophys. properties of lipid bilayers.  In this regard the impact of cholesterol on membrane organization and dynamics is particularly well documented and serves for comparison with other sterols.  However, the factors underlying the mol. evolution of cholesterol remain enigmatic.  To this end, cholesterol attenuates membrane perturbation by the so-called antimicrobial peptides (AMPs), produced ubiquitously by eukaryotic cells to combat bacterial infections by compromising the permeability barrier function of the microbial target membranes.  In the present study, we addressed the effects of cholesterol, ergosterol, and lanosterol on the membrane assocn. of two structurally and functionally diverse AMPs viz.  LL-37(F27W) and temporin L (TemL) using fluorescence spectroscopy.  Interestingly, sterol concn. dependent effects on the membrane assocn. of these peptides were obsd.  At XSterol = 0.5 cholesterol was most effective in reducing the membrane intercalation of both LL-37(F27W) and TemL, the corresponding efficiencies of the three sterols decreasing as cholesterol > lanosterol ≥ ergosterol, and cholesterol > lanosterol > ergosterol.  It is conceivable that part of the selection pressure for the chem. evolution of cholesterol may have derived from the ability to protect the AMP-secreting host cell from the membrane damaging action of the antimicrobial peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJqHhuljTss7Vg90H21EOLACvtfcHk0ljbJ6GkHKROmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVaqurw%253D&md5=b108232047a4afe0e02c667960163641</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2008.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2008.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DSood%26aufirst%3DR.%26aulast%3DKinnunen%26aufirst%3DP.%2BK.%2BJ.%26atitle%3DCholesterol%252C%2520lanosterol%252C%2520and%2520ergosterol%2520attenuate%2520the%2520membrane%2520association%2520of%2520LL-37%2528W27F%2529%2520and%2520temporin%2520L%26jtitle%3DBiochim.%2520Biophys.%2520Acta-Biomembr.%26date%3D2008%26volume%3D1778%26spage%3D1460%26epage%3D1466%26doi%3D10.1016%2Fj.bbamem.2008.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillies, E. R.</span></span> <span> </span><span class="NLM_article-title">Amphipathic beta-strand mimics as potential membrane disruptive antibiotics</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">5953</span>– <span class="NLM_lpage">5960</span>, <span class="refDoi"> DOI: 10.1021/jo900933r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo900933r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD1MXoslCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2009&pages=5953-5960&author=J.+L.+Watsonauthor=E.+R.+Gillies&title=Amphipathic+beta-strand+mimics+as+potential+membrane+disruptive+antibiotics&doi=10.1021%2Fjo900933r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Amphipathic β-strand mimics as potential membrane disruptive antibiotics</span></div><div class="casAuthors">Watson, Jessica L.; Gillies, Elizabeth R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5953-5960</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, there have been increasing nos. of bacterial strains emerging that are resistant to the currently available antibiotics.  In the search for new antibiotics, attention has been focused on natural antimicrobial peptides that act by selectively disrupting the membranes of bacterial cells, a mechanism that is thought to be nonconducive to the development of resistance.  It is desirable to mimic the structures and activities of these peptides while introducing properties such as resistance to proteolytic degrdn., which make mols. more ideal for development as drugs.  Described here is the design and synthesis of β-strand mimetic oligomers based on alternating α-amino acids and azacyclohexenone units that segregate cationic lysine and hydrophobic valine side chains on opposite faces of the β-strand.  1H NMR diln. studies demonstrated that despite the incorporation of alternating D- and L-amino acids in order to obtain facial amphiphilicity, these oligomers are capable of dimerizing to β-sheet mimics in a manner similar to the oligomers contg. all L-amino acids.  The ability of the mols. to disrupt phospholipid vesicles mimicking the membranes of both bacterial and mammalian cells was investigated using a fluorescent dye leakage assay.  Several of the oligomers were found to exhibit activity and selectivity for the bacterial over mammalian membranes.  Overall, these studies demonstrate the promise of this class of mols. for the development of new potential antibiotics and provide information on the structural features that are important for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow49wha4sQNbVg90H21EOLACvtfcHk0li2RKNVnkk5AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXoslCntb0%253D&md5=b561a4ad3b63ff3c4f403f81f2c59a37</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjo900933r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo900933r%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DJ.%2BL.%26aulast%3DGillies%26aufirst%3DE.%2BR.%26atitle%3DAmphipathic%2520beta-strand%2520mimics%2520as%2520potential%2520membrane%2520disruptive%2520antibiotics%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2009%26volume%3D74%26spage%3D5953%26epage%3D5960%26doi%3D10.1021%2Fjo900933r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Design of imperfectly amphipathic alpha-helical antimicrobial peptides with enhanced cell selectivity</span>. <i>Acta Biomater.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">244</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1016/j.actbio.2013.08.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.actbio.2013.08.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=24021230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFCitrrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=244-257&author=X.+Zhuauthor=N.+Dongauthor=Z.+Wangauthor=Z.+Maauthor=L.+Zhangauthor=Q.+Maauthor=A.+Shan&title=Design+of+imperfectly+amphipathic+alpha-helical+antimicrobial+peptides+with+enhanced+cell+selectivity&doi=10.1016%2Fj.actbio.2013.08.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Design of imperfectly amphipathic α-helical antimicrobial peptides with enhanced cell selectivity</span></div><div class="casAuthors">Zhu, Xin; Dong, Na; Wang, Zeyun; Ma, Zhi; Zhang, Licong; Ma, Qingquan; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Acta Biomaterialia</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">244-257</span>CODEN:
                <span class="NLM_cas:coden">ABCICB</span>;
        ISSN:<span class="NLM_cas:issn">1742-7061</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs), which are produced by multicellular organisms as a defense mechanism against competing pathogenic microbes, appear to be excellent candidates for the development of novel antimicrobial agents.  Amphipathicity is traditionally believed to be crucial to the de novo design or systematic optimization of AMPs.  In this study, we designed a series of short α-helical AMPs with imperfect amphipathicity to augment the arsenal of strategies and to gain further insights into their antimicrobial and hemolytic activity.  These imperfectly amphipathic α-helical AMPs were designed by replacing the paired charged amino acid residues on the polar face of an amphipathic peptide with tryptophan residues on the basis of α-helical protein folding principles.  PRW4, an imperfectly amphipathic α-helical AMP with hydrogen bonds formed by paired tryptophan residues, was obsd. to be more selective towards bacterial cells than toward human red blood cells.  PRW4 was also effective against Gram-neg. and Gram-pos. bacteria, and fluorescence spectroscopy, flow cytometry, SEM and transmission electron microscopy indicated that PRW4 killed microbial cells by permeabilizing the cell membrane and damaging their membrane integrity.  Therefore, disruptive amphipathicity has excellent potential for the rational design and optimization of AMPs with promising antimicrobial activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphnDG1okHKT7Vg90H21EOLACvtfcHk0li2RKNVnkk5AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFCitrrM&md5=9c175868f253e06a4fca1c415dcbaba2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.actbio.2013.08.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.actbio.2013.08.043%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DDesign%2520of%2520imperfectly%2520amphipathic%2520alpha-helical%2520antimicrobial%2520peptides%2520with%2520enhanced%2520cell%2520selectivity%26jtitle%3DActa%2520Biomater.%26date%3D2014%26volume%3D10%26spage%3D244%26epage%3D257%26doi%3D10.1016%2Fj.actbio.2013.08.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span> <span> </span><span class="NLM_article-title">Antibacterial activities of short designer peptides: a link between propensity for nanostructuring and capacity for membrane destabilization</span>. <i>Biomacromolecules</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">402</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1021/bm901130u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bm901130u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFWiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=402-411&author=C.+Chenauthor=F.+Panauthor=S.+Zhangauthor=J.+Huauthor=M.+Caoauthor=J.+Wangauthor=H.+Xuauthor=X.+Zhaoauthor=J.+Lu&title=Antibacterial+activities+of+short+designer+peptides%3A+a+link+between+propensity+for+nanostructuring+and+capacity+for+membrane+destabilization&doi=10.1021%2Fbm901130u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial Activities of Short Designer Peptides: a Link between Propensity for Nanostructuring and Capacity for Membrane Destabilization</span></div><div class="casAuthors">Chen, Cuixia; Pan, Fang; Zhang, Shengzhong; Hu, Jing; Cao, Meiwen; Wang, Jing; Xu, Hai; Zhao, Xiubo; Lu, Jian R.</div><div class="citationInfo"><span class="NLM_cas:title">Biomacromolecules</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">402-411</span>CODEN:
                <span class="NLM_cas:coden">BOMAF6</span>;
        ISSN:<span class="NLM_cas:issn">1525-7797</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Amphiphilic peptides A3K, A6K, and A9K displayed an increasing propensity for nanoaggregation with increasing the size of hydrophobic alanine moiety, and the size and shape of the aggregates showed a steady transition from loose peptide stacks formed by A3K, long nanofibers by A6K, to short and narrow nanorods by A9K.  This size and shape transition was broadly consistent with the trend predicted from interfacial packing and curvature change if these peptide surfactants were treated as conventional surfactants.  The antibacterial capacity, defined by the killing of percentage of bacteria in a given time and peptide concn., showed a strong correlation to peptide hydrophobicity, evident from both microscopic and fluorescence imaging studies.  For A9K, the power for membrane permeation and bacterial clustering intensified with peptide concn. and incubation time.  These results thus depict a pos. correlation between the propensity for self-assembly of the peptides, their membrane penetration power, and bactericidal capacity.  Although the exposure of A9K to a preformed DPPC membrane bilayer showed little structural disturbance, the same treatment to the preformed DPPG membrane bilayer led to substantial disruption of model membrane structure, a trend entirely consistent with the high selectivity obsd. from membrane hemolytic studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGods7GafznUrrVg90H21EOLACvtfcHk0li2RKNVnkk5AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFWiuw%253D%253D&md5=d20b33143f944bf25c592a76c1b4a310</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fbm901130u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbm901130u%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%26aulast%3DPan%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DAntibacterial%2520activities%2520of%2520short%2520designer%2520peptides%253A%2520a%2520link%2520between%2520propensity%2520for%2520nanostructuring%2520and%2520capacity%2520for%2520membrane%2520destabilization%26jtitle%3DBiomacromolecules%26date%3D2010%26volume%3D11%26spage%3D402%26epage%3D411%26doi%3D10.1021%2Fbm901130u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial peptides with high proteolytic resistance for combating gram-negative bacteria</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2286</span>– <span class="NLM_lpage">2304</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01348</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01348" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisl2lsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2286-2304&author=J.+Wangauthor=J.+Songauthor=Z.+Yangauthor=S.+Heauthor=Y.+Yangauthor=X.+Fengauthor=X.+Douauthor=A.+Shan&title=Antimicrobial+peptides+with+high+proteolytic+resistance+for+combating+gram-negative+bacteria&doi=10.1021%2Facs.jmedchem.8b01348"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial Peptides with High Proteolytic Resistance for Combating Gram-Negative Bacteria</span></div><div class="casAuthors">Wang, Jiajun; Song, Jing; Yang, Zhanyi; He, Shiqi; Yang, Yi; Feng, Xingjun; Dou, Xiujing; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2286-2304</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poor proteolytic resistance is an urgent problem to be solved in the clin. application of antimicrobial peptides (AMPs), yet common solns., such as complicated chem. modifications and utilization of D-amino acids, greatly increase the difficulty and cost of producing AMPs.  In this work, a set of novel peptides was synthesized based on an antitrypsin/antichymotrypsin hydrolytic peptide structure unit (XYPX)n (X represents I, L, and V; Y represents R and K), which was designed using a systematic natural amino acid arrangement.  Of these peptides, 1 of them with 7 repeat units had the highest av. selectivity index (GMSI = 99.07) for all of the Gram-neg. bacteria tested and remained highly effective in combating Escherichia coli infection in vivo.  Importantly, the same peptide also had dramatic resistance to a high concn. of trypsin/chymotrypsin hydrolysis and exerted bactericidal activity through a membrane-disruptive mechanism.  Overall, these findings provide new approaches for the development of antiprotease hydrolytic peptides that target Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5SNyY4JzKxLVg90H21EOLACvtfcHk0lhoBVMdXUh-WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisl2lsrs%253D&md5=0a5b9a7741b7b397b9f15e1d9ac30970</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01348&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01348%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DDou%26aufirst%3DX.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DAntimicrobial%2520peptides%2520with%2520high%2520proteolytic%2520resistance%2520for%2520combating%2520gram-negative%2520bacteria%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2286%26epage%3D2304%26doi%3D10.1021%2Facs.jmedchem.8b01348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuzaki, K.</span></span> <span> </span><span class="NLM_article-title">Action mechanism of pegylated magainin 2 analogue peptide</span>. <i>Biochim. Biophys. Acta-Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>1768</i></span>,  <span class="NLM_fpage">2578</span>– <span class="NLM_lpage">2585</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2007.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bbamem.2007.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=17662233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGrsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1768&publication_year=2007&pages=2578-2585&author=Y.+Imuraauthor=M.+Nishidaauthor=K.+Matsuzaki&title=Action+mechanism+of+pegylated+magainin+2+analogue+peptide&doi=10.1016%2Fj.bbamem.2007.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Action mechanism of PEGylated magainin 2 analogue peptide</span></div><div class="casAuthors">Imura, Yuichi; Nishida, Minoru; Matsuzaki, Katsumi</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1768</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2578-2585</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">PEGylation is frequently used to improve the efficacy of protein and peptide drugs.  Recently, we investigated its effects on the action mechanism of the cyclic β-sheet antimicrobial peptide tachyplesin I isolated from Tachypleus tridentatus [Y. Imura, M. Nishida, Y. Ogawa, Y. Takakura, K. Matsuzaki, Action Mechanism of Tachyplesin I and Effects of PEGylation, Biochim. Biophys. Acta 1768 (2007) 1160-1169].  PEGylation did not change the basic mechanism behind the membrane-permeabilizing effect of the peptide on liposomes, however, it decreased the antimicrobial activity and cytotoxicity.  To obtain further information on the effects of PEGylation on the activities of antimicrobial peptides, we designed another structurally different PEGylated antimicrobial peptide (PEG-F5W, E19Q-magainin 2-amide) based on the α-helical peptide magainin 2 isolated from the African clawed frog Xenopus laevis.  The PEGylated peptide induced the leakage of calcein from egg yolk L-α-phosphatidylglycerol/egg yolk L-α-phosphatidylcholine large unilamellar vesicles, however, the activity was weaker than that of the control peptides.  The PEGylated peptide induced lipid flip-flop coupled to the leakage and was translocated into the inner leaflet of the bilayer, indicating that PEGylation did not alter the basic mechanism of membrane permeabilization of the parent peptide.  The cytotoxicity of the non-PEGylated peptides was nullified by PEGylation.  At the same time, the antimicrobial activity was weakened only by 4 fold.  The effects of PEGylation on the activity of magainin were compared with those for tachyplesin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorfObvUcCooLVg90H21EOLACvtfcHk0lhoBVMdXUh-WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGrsrrF&md5=1a494f20cb6d22c7df1e87ef16adcef3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2007.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2007.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DImura%26aufirst%3DY.%26aulast%3DNishida%26aufirst%3DM.%26aulast%3DMatsuzaki%26aufirst%3DK.%26atitle%3DAction%2520mechanism%2520of%2520pegylated%2520magainin%25202%2520analogue%2520peptide%26jtitle%3DBiochim.%2520Biophys.%2520Acta-Biomembr.%26date%3D2007%26volume%3D1768%26spage%3D2578%26epage%3D2585%26doi%3D10.1016%2Fj.bbamem.2007.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heersema, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peppas, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, H. D. C.</span></span> <span> </span><span class="NLM_article-title">Optimization of cationic nanogel pegylation to achieve mammalian cytocompatibility with limited loss of gram-negative bactericidal activity</span>. <i>Biomacromolecules</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1528</span>– <span class="NLM_lpage">1538</span>, <span class="refDoi"> DOI: 10.1021/acs.biomac.0c00081</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biomac.0c00081" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltl2ntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2020&pages=1528-1538&author=J.+Guauthor=J.+R.+Cleggauthor=L.+A.+Heersemaauthor=N.+A.+Peppasauthor=H.+D.+C.+Smyth&title=Optimization+of+cationic+nanogel+pegylation+to+achieve+mammalian+cytocompatibility+with+limited+loss+of+gram-negative+bactericidal+activity&doi=10.1021%2Facs.biomac.0c00081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Cationic Nanogel PEGylation to Achieve Mammalian Cytocompatibility with Limited Loss of Gram-Negative Bactericidal Activity</span></div><div class="casAuthors">Gu, Joann; Clegg, John R.; Heersema, Lara A.; Peppas, Nicholas A.; Smyth, Hugh D. C.</div><div class="citationInfo"><span class="NLM_cas:title">Biomacromolecules</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1528-1538</span>CODEN:
                <span class="NLM_cas:coden">BOMAF6</span>;
        ISSN:<span class="NLM_cas:issn">1525-7797</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tuning the compn. of antimicrobial nanogels can significantly alter both nanogel cytotoxicity and antibacterial activity.  This project investigated the extent to which PEGylation of cationic, hydrophobic nanogels altered their cytotoxicity and bactericidal activity.  These biodegradable, cationic nanogels were synthesized by activators regenerated by electron transfer atom transfer radical polymn. (ARGET ATRP) emulsion polymn. with up to 13.9 wt % PEG (MW = 2000) MA, as verified by 1H NMR.  Nanogel bactericidal activity was assessed against Gram-neg. E. coli and P. aeruginosa and Gram-pos. S. mutans and S. aureus by measuring membrane lysis with a LIVE/DEAD assay.  E. coli and S. mutans viability was further validated by measuring metabolic activity with a PrestoBlue assay and imaging bacteria stained with a LIVE/DEAD probe.  All tested nanogels decreased the membrane integrity (0.5 mg/mL dose) for Gram-neg. E. coli and P. aeruginosa, irresp. of the extent of PEGylation.  PEGylation (13.9 wt %) increased the cytocompatibility of cationic nanogels toward RAW 264.7 murine macrophages and L929 murine fibroblasts by over 100-fold, relative to control nanogels.  PEGylation (42.8 wt %) reduced nanogel uptake by 43% for macrophages and 63% for fibroblasts.  Therefore, PEGylation reduced nanogel toxicity to mammalian cells without significantly compromising their bactericidal activity.  These results facilitate future nanogel design for perturbing the growth of Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0i7kZSpsc17Vg90H21EOLACvtfcHk0lhoBVMdXUh-WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltl2ntLc%253D&md5=a5cfee9799d97194604eaf1b2efaf859</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.biomac.0c00081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biomac.0c00081%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DClegg%26aufirst%3DJ.%2BR.%26aulast%3DHeersema%26aufirst%3DL.%2BA.%26aulast%3DPeppas%26aufirst%3DN.%2BA.%26aulast%3DSmyth%26aufirst%3DH.%2BD.%2BC.%26atitle%3DOptimization%2520of%2520cationic%2520nanogel%2520pegylation%2520to%2520achieve%2520mammalian%2520cytocompatibility%2520with%2520limited%2520loss%2520of%2520gram-negative%2520bactericidal%2520activity%26jtitle%3DBiomacromolecules%26date%3D2020%26volume%3D21%26spage%3D1528%26epage%3D1538%26doi%3D10.1021%2Facs.biomac.0c00081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chess, R. B.</span></span> <span> </span><span class="NLM_article-title">Effect of pegylation on pharmaceuticals</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nrd1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1038%2Fnrd1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=12612647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=214-221&author=J.+M.+Harrisauthor=R.+B.+Chess&title=Effect+of+pegylation+on+pharmaceuticals&doi=10.1038%2Fnrd1033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of PEGylation on pharmaceuticals</span></div><div class="casAuthors">Harris, J. Milton; Chess, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">214-221</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein and peptide drugs hold great promise as therapeutic agents.  However, many are degraded by proteolytic enzymes, can be rapidly cleared by the kidneys, generate neutralizing antibodies and have a short circulating half-life.  PEGylation, the process by which polyethylene glycol chains are attached to protein and peptide drugs, can overcome these and other shortcomings.  By increasing the mol. mass of proteins and peptides and shielding them from proteolytic enzymes, PEGylation improves pharmacokinetics.  This article will review how PEGylation can result in drugs that are often more effective and safer, and which show improved patient convenience and compliance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLhPAuz8WjMLVg90H21EOLACvtfcHk0lgiiLItuxxowA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtLs%253D&md5=d6bb387e746740b10e088e1f55af37db</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnrd1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1033%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DJ.%2BM.%26aulast%3DChess%26aufirst%3DR.%2BB.%26atitle%3DEffect%2520of%2520pegylation%2520on%2520pharmaceuticals%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D214%26epage%3D221%26doi%3D10.1038%2Fnrd1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salhanick, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clairmont, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckholz, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumb, K. J.</span></span> <span> </span><span class="NLM_article-title">Contribution of site-specific pegylation to the dipeptidyl peptidase IV stability of glucose-dependent insulinotropic polypeptide</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">4114</span>– <span class="NLM_lpage">4117</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2005.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.bmcl.2005.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=15993590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2MXns1SgtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=4114-4117&author=A.+I.+Salhanickauthor=K.+B.+Clairmontauthor=T.+M.+Buckholzauthor=C.+M.+Pellegrinoauthor=S.+Haauthor=K.+J.+Lumb&title=Contribution+of+site-specific+pegylation+to+the+dipeptidyl+peptidase+IV+stability+of+glucose-dependent+insulinotropic+polypeptide&doi=10.1016%2Fj.bmcl.2005.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of site-specific PEGylation to the dipeptidyl peptidase IV stability of glucose-dependent insulinotropic polypeptide</span></div><div class="casAuthors">Salhanick, Arthur I.; Clairmont, Kevin B.; Buckholz, Thomas M.; Pellegrino, Carla M.; Ha, Sha; Lumb, Kevin J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4114-4117</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The effects of PEGylation of glucose-dependent insulinotropic polypeptide (GIP) on potency and dipeptidyl peptidase IV (DPPIV) stability are reported.  N-terminal modification of GIP(1-30) with 40 kDa polyethylene glycol (PEG) abrogates functional activity.  In contrast, C-terminal PEGylation of GIP(1-30) maintains full agonism and reasonable potency at the GIP receptor and confers a high level of DPPIV resistance.  Moreover, the dual modification of N-terminal palmitoylation and C-terminal PEGylation results in a full agonist of comparable potency to native GIP that is stable to DPPIV cleavage.  The results provide the basis for the development of long acting, PEGylated GIP, GIP variants, or GIP-based hybrid peptide therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsENy8kGz-IrVg90H21EOLACvtfcHk0lgiiLItuxxowA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXns1SgtL0%253D&md5=8123e84e7bf70de96ef27311ad284afa</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2005.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2005.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DSalhanick%26aufirst%3DA.%2BI.%26aulast%3DClairmont%26aufirst%3DK.%2BB.%26aulast%3DBuckholz%26aufirst%3DT.%2BM.%26aulast%3DPellegrino%26aufirst%3DC.%2BM.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DLumb%26aufirst%3DK.%2BJ.%26atitle%3DContribution%2520of%2520site-specific%2520pegylation%2520to%2520the%2520dipeptidyl%2520peptidase%2520IV%2520stability%2520of%2520glucose-dependent%2520insulinotropic%2520polypeptide%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2005%26volume%3D15%26spage%3D4114%26epage%3D4117%26doi%3D10.1016%2Fj.bmcl.2005.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pasut, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, F. M.</span></span> <span> </span><span class="NLM_article-title">Polymer-drug conjugation, recent achievements and general strategies</span>. <i>Prog. Polym. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">933</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1016/j.progpolymsci.2007.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=10.1016%2Fj.progpolymsci.2007.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFeksLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2007&pages=933-961&author=G.+Pasutauthor=F.+M.+Veronese&title=Polymer-drug+conjugation%2C+recent+achievements+and+general+strategies&doi=10.1016%2Fj.progpolymsci.2007.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Polymer-drug conjugation, recent achievements and general strategies</span></div><div class="casAuthors">Pasut, G.; Veronese, F. M.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Polymer Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">933-961</span>CODEN:
                <span class="NLM_cas:coden">PRPSB8</span>;
        ISSN:<span class="NLM_cas:issn">0079-6700</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The field of drug delivery is fast expanding and its potentials have already been proved by the many products in the market.  Among all approaches, polymer conjugation is a well known and widely exploited technique useful to improve therapeutic properties of peptides, proteins, small mols. or oligonucleotides.  Polymer-conjugated drugs generally exhibit prolonged half-life, higher stability, water soly., lower immunogenicity and antigenicity and often also specific targeting to tissues or cells.  This technol., exploited for the first time in the 1950s and 1960s, received a great development both for the introduction and study of new and different polymers and for the progresses in the chem. strategies of coupling.  Polymer-drug conjugates are already in the market for the treatment of different diseases, demonstrating the potentials of the technol.  Furthermore, new polymers, in addn. to the most known N-(2-hydroxypropyl)methacrylamide copolymer (HPMA), polyglutamic acid (PGA) and poly(ethylene glycol) (PEG), are continuously investigated and proposed.  The review will discuss the most recent achievements in polymer conjugation with special emphasis on PEG application strategies and approved products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowAFlEG0CRKbVg90H21EOLACvtfcHk0lguFJtCkJ-V-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFeksLc%253D&md5=263b82b9257683f2a922d9b8014aeb55</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.progpolymsci.2007.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.progpolymsci.2007.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DPasut%26aufirst%3DG.%26aulast%3DVeronese%26aufirst%3DF.%2BM.%26atitle%3DPolymer-drug%2520conjugation%252C%2520recent%2520achievements%2520and%2520general%2520strategies%26jtitle%3DProg.%2520Polym.%2520Sci.%26date%3D2007%26volume%3D32%26spage%3D933%26epage%3D961%26doi%3D10.1016%2Fj.progpolymsci.2007.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Rational avoidance of protease cleavage sites and symmetrical end-tagging significantly enhances the stability and therapeutic potential of antimicrobial peptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9421</span>– <span class="NLM_lpage">9435</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9421-9435&author=Y.+Zhuauthor=C.+Shaoauthor=G.+Liauthor=Z.+Laiauthor=P.+Tanauthor=Q.+Jianauthor=B.+Chengauthor=A.+Shan&title=Rational+avoidance+of+protease+cleavage+sites+and+symmetrical+end-tagging+significantly+enhances+the+stability+and+therapeutic+potential+of+antimicrobial+peptides&doi=10.1021%2Facs.jmedchem.0c00583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Avoidance of Protease Cleavage Sites and Symmetrical End-Tagging Significantly Enhances the Stability and Therapeutic Potential of Antimicrobial Peptides</span></div><div class="casAuthors">Zhu, Yongjie; Shao, Changxuan; Li, Guoyu; Lai, Zhenheng; Tan, Peng; Jian, Qiao; Cheng, Baojing; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9421-9435</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antimicrobial peptides (AMPs) are expected to solve problem of antibiotic resistance because of their distinctive nonspecific membrane-disruptive mechanism.  However, clin. applications of AMPs have been precluded by their poor stability, although various complex chem. strategies have been employed to solve this problem, and this undoubtedly greatly increases the manufg. cost.  Herein, a series of novel peptides with high stability were developed based on protease-specific cleavage sites and sym. end-tagging.  Among these peptides, II-I4-II exhibited the best antibacterial activity and the highest therapeutic index.  More importantly, II-I4-II showed extremely high stability in the presence of various proteases, physiol. salts and serum, and under acid, alkali, and heat conditions, and it exhibited excellent therapeutic potential in vivo.  Addnl., II-I4-II exhibited a membrane-disruptive mechanism and a low propensity to induce resistance.  In general, these findings contribute to the design of AMPs with high stability and might accelerate clin. applications of AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0eoqZgwqxnbVg90H21EOLACvtfcHk0lguFJtCkJ-V-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmtbvI&md5=1eaf97e703bd9b16a562cc860b3d39e3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00583%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DJian%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DB.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DRational%2520avoidance%2520of%2520protease%2520cleavage%2520sites%2520and%2520symmetrical%2520end-tagging%2520significantly%2520enhances%2520the%2520stability%2520and%2520therapeutic%2520potential%2520of%2520antimicrobial%2520peptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9421%26epage%3D9435%26doi%3D10.1021%2Facs.jmedchem.0c00583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span> <span> </span><span class="NLM_article-title">Combating drug-resistant fungi with novel imperfectly amphipathic palindromic peptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3889</span>– <span class="NLM_lpage">3907</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01729</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01729" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFShu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3889-3907&author=J.+Wangauthor=S.+Chouauthor=Z.+Yangauthor=Y.+Yangauthor=Z.+Wangauthor=J.+Songauthor=X.+Douauthor=A.+Shan&title=Combating+drug-resistant+fungi+with+novel+imperfectly+amphipathic+palindromic+peptides&doi=10.1021%2Facs.jmedchem.7b01729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Drug-Resistant Fungi with Novel Imperfectly Amphipathic Palindromic Peptides</span></div><div class="casAuthors">Wang, Jiajun; Chou, Shuli; Yang, Zhanyi; Yang, Yang; Wang, Zhihua; Song, Jing; Dou, Xiujing; Shan, Anshan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3889-3907</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Antimicrobial peptides are an important weapon against invading pathogens and are potential candidates as novel antibacterial agents, but their antifungal activities are not fully developed.  In this study, a set of imperfectly amphipathic peptides was developed based on the imperfectly amphipathic palindromic structure Rn(XRXXXRX)Rn (n = 1,2; X represents L, I, F, or W), and the engineered peptides exhibited high antimicrobial activities against all fungi and bacteria tested (including fluconazole-resistant Candida albicans), with geometric mean (GM) MICs ranging from 2.2 to 6.62μM.  Of such peptides, 13 (I6) (RRIRIIIRIRR-NH2) that was Ile rich in its hydrophobic face had the highest antifungal activity (GMfungi =1.64μM), meanwhile showing low toxicity and high salt and serum tolerance.  It also had dramatic LPS-neutralizing propensity and a potent membrane-disruptive mechanism against microbial cells.  In summary, these findings were useful for short AMPs design to combat the growing threat of drug-resistant fungal and bacterial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXoGkyENHFN7Vg90H21EOLACvtfcHk0lguFJtCkJ-V-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFShu7k%253D&md5=c5806d09b3beb6deb2c806c45333e973</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01729%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChou%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DX.%26aulast%3DShan%26aufirst%3DA.%26atitle%3DCombating%2520drug-resistant%2520fungi%2520with%2520novel%2520imperfectly%2520amphipathic%2520palindromic%2520peptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3889%26epage%3D3907%26doi%3D10.1021%2Facs.jmedchem.7b01729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Therapeutic potential of trp-rich engineered amphiphiles by single hydrophobic amino acid end-tagging</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">43820</span>– <span class="NLM_lpage">43834</span>, <span class="refDoi"> DOI: 10.1021/acsami.9b12706</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.9b12706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFSqtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=43820-43834&author=J.+Songauthor=J.+Wangauthor=N.+Zhanauthor=T.+Sunauthor=W.+Yuauthor=L.+Zhangauthor=A.+Shanauthor=A.+Zhang&title=Therapeutic+potential+of+trp-rich+engineered+amphiphiles+by+single+hydrophobic+amino+acid+end-tagging&doi=10.1021%2Facsami.9b12706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Potential of Trp-Rich Engineered Amphiphiles by Single Hydrophobic Amino Acid End-Tagging</span></div><div class="casAuthors">Song, Jing; Wang, Jiajun; Zhan, Na; Sun, Taotao; Yu, Weikang; Zhang, Licong; Shan, Anshan; Zhang, Aizhong</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">43820-43834</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">End-tagging with a single hydrophobic residue contributes to improve the cell selectivity of antimicrobial peptides (AMPs), but systematic studies have been lacking.  Thus, this study aimed to systematically investigate how end-tagging with hydrophobic residues at the C-terminus and Gly capped at the N-terminus of W4 (RWRWWWRWR) affects the bioactivity of W4 variants.  Among all the hydrophobic residues, only Ala end-tagging improved the antibacterial activity of W4.  Meanwhile, Gly capped at the N-terminus could promote the helical propensity of the end-tagged peptides in dodecylphosphocholine micelles, increasing their antimicrobial activities.  Of these peptides, GW4A (GRWRWWWRWRA) showed the best antibacterial activity against the 19 species of bacteria tested (GMMIC = 1.86 μM) with low toxicity, thus possessing the highest cell selectivity (TIall = 137.63).  It also had rapid sterilization, good salt and serum resistance, and LPS-neutralizing activity.  Antibacterial mechanism studies showed that the short peptide GW4A killed bacteria by destroying cell membrane integrity and causing cytoplasmic leakage.  Overall, these findings suggested that systematic studies on terminal modifications promoted the development of peptide design theory and provided a potential method for optimization of effective AMPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6d5P_0TzCoLVg90H21EOLACvtfcHk0ljNdVuyDX7wXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFSqtL%252FM&md5=f71c5d1ce4892d24299ab1783199df99</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facsami.9b12706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.9b12706%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhan%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DTherapeutic%2520potential%2520of%2520trp-rich%2520engineered%2520amphiphiles%2520by%2520single%2520hydrophobic%2520amino%2520acid%2520end-tagging%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2019%26volume%3D11%26spage%3D43820%26epage%3D43834%26doi%3D10.1021%2Facsami.9b12706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span> <span> </span><span class="NLM_article-title">Highly stabilized alpha-helical coiled coils kill gram-negative bacteria by multicomplementary mechanisms under acidic condition</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">22113</span>– <span class="NLM_lpage">22128</span>, <span class="refDoi"> DOI: 10.1021/acsami.9b04654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.9b04654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2lsb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=22113-22128&author=Z.+Laiauthor=P.+Tanauthor=Y.+Zhuauthor=C.+Shaoauthor=A.+Shanauthor=L.+Li&title=Highly+stabilized+alpha-helical+coiled+coils+kill+gram-negative+bacteria+by+multicomplementary+mechanisms+under+acidic+condition&doi=10.1021%2Facsami.9b04654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Stabilized α-Helical Coiled Coils Kill Gram-Negative Bacteria by Multicomplementary Mechanisms under Acidic Condition</span></div><div class="casAuthors">Lai, Zhenheng; Tan, Peng; Zhu, Yongjie; Shao, Changxuan; Shan, Anshan; Li, Lu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">22113-22128</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although antimicrobial peptides (AMPs) hold tremendous promise in overcoming the threats of multidrug resistance, the main obstacle to successful therapeutic applications is their poor stability.  Various synthetic strategies such as unnatural amino acids and chem. modifications have made advances for improving this problem.  However, this complicated synthesis often greatly increases the cost of prodn.  Here, a series of novel peptides, designed by combining an α-helical coiled coil model, knowledge of the specificity of proteolysis and major parameters of AMPs, exhibited efficient activity against all tested Gram-neg. bacteria under acidic condition and demonstrate low toxicity.  Of these α-helical coiled coil peptides, 3IH3 displayed the highest av. therapeutic index (GMTI = 294.25) with high stability toward salts, serum, extreme pH, heat, and proteases.  Electron microscopy and biol. anal. technique analyses showed that 3IH3 killed bacterial cells via a multicomplementary mechanism at pH 6.0, with phys. membrane disruption as the dominant bactericidal mechanism.  These results suggest that 3IH3 shows great stability as an inexpensive and effective antimicrobial activity agent and has the potential for clin. application in the treatment of infections occurring in body sites with acidic pH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDuifidWFC6LVg90H21EOLACvtfcHk0ljNdVuyDX7wXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2lsb%252FK&md5=584757d80cec28bd24369137f49fb64b</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facsami.9b04654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.9b04654%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DTan%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DShan%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DL.%26atitle%3DHighly%2520stabilized%2520alpha-helical%2520coiled%2520coils%2520kill%2520gram-negative%2520bacteria%2520by%2520multicomplementary%2520mechanisms%2520under%2520acidic%2520condition%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2019%26volume%3D11%26spage%3D22113%26epage%3D22128%26doi%3D10.1021%2Facsami.9b04654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i37"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00879">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_82582"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00879?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00879</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">The 3D model and helical wheel projections of PG-1; RP-HPLC spectra of the engineered peptides; mass spectra of the engineered peptides; MICs (MBCs) (μg/mL) of the engineered peptides against a drug-resistant susceptible strain; IC<sub>50</sub> curves of engineered peptides on HEK 293T cells; ANS fluorescence binding assay for PG-1; resistance of engineered peptides (1× MIC) to trypsin; and CM depolarization assays and IM permeability assays of PG-1 and melittin (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00879/suppl_file/jm1c00879_si_001.pdf">jm1c00879_si_001.pdf (1.57 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00879&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00879%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-14" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00879" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a50479a54d1a7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
