
<html lang="en"     class="pb-page"  data-request-id="7b320f7d-518b-4849-b5a8-55bd02462454"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c02258;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Biological Evaluation of Peptidomimetic Aldehydes as Broad-Spectrum Inhibitors against Enterovirus and SARS-CoV-2" /></meta><meta name="dc.Creator" content="Wenhao  Dai" /></meta><meta name="dc.Creator" content="Dirk  Jochmans" /></meta><meta name="dc.Creator" content="Hang  Xie" /></meta><meta name="dc.Creator" content="Hang  Yang" /></meta><meta name="dc.Creator" content="Jian  Li" /></meta><meta name="dc.Creator" content="Haixia  Su" /></meta><meta name="dc.Creator" content="Di  Chang" /></meta><meta name="dc.Creator" content="Jiang  Wang" /></meta><meta name="dc.Creator" content="Jingjing  Peng" /></meta><meta name="dc.Creator" content="Lili  Zhu" /></meta><meta name="dc.Creator" content="Yong  Nian" /></meta><meta name="dc.Creator" content="Rolf  Hilgenfeld" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Kaixian  Chen" /></meta><meta name="dc.Creator" content="Leike  Zhang" /></meta><meta name="dc.Creator" content="Yechun  Xu" /></meta><meta name="dc.Creator" content="Johan  Neyts" /></meta><meta name="dc.Creator" content="Hong  Liu" /></meta><meta name="dc.Description" content="A novel series of peptidomimetic aldehydes was designed and synthesized to target 3C protease (3Cpro) of enterovirus 71 (EV71). Most of the compounds exhibited high antiviral activity, and among th..." /></meta><meta name="Description" content="A novel series of peptidomimetic aldehydes was designed and synthesized to target 3C protease (3Cpro) of enterovirus 71 (EV71). Most of the compounds exhibited high antiviral activity, and among th..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 19, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02258" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02258" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02258" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02258" /></link>
        
    
    

<title>Design, Synthesis, and Biological Evaluation of Peptidomimetic Aldehydes as Broad-Spectrum Inhibitors against Enterovirus and SARS-CoV-2 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02258" /></meta><meta property="og:title" content="Design, Synthesis, and Biological Evaluation of Peptidomimetic Aldehydes as Broad-Spectrum Inhibitors against Enterovirus and SARS-CoV-2" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0010.jpeg" /></meta><meta property="og:description" content="A novel series of peptidomimetic aldehydes was designed and synthesized to target 3C protease (3Cpro) of enterovirus 71 (EV71). Most of the compounds exhibited high antiviral activity, and among them, compound 18p demonstrated potent enzyme inhibitory activity and broad-spectrum antiviral activity on a panel of enteroviruses and rhinoviruses. The crystal structure of EV71 3Cpro in complex with 18p determined at a resolution of 1.2 Å revealed that 18p covalently linked to the catalytic Cys147 with an aldehyde group. In addition, these compounds also exhibited good inhibitory activity against the 3CLpro and the replication of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), especially compound 18p (IC50 = 0.034 μM, EC50 = 0.29 μM). According to our previous work, these compounds have no reasons for concern regarding acute toxicity. Compared with AG7088, compound 18p also exhibited good pharmacokinetic properties and more potent anticoronavirus activity, making it an excellent lead for further development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02258"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02258">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02258&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02258&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02258&amp;href=/doi/10.1021/acs.jmedchem.0c02258" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00509" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00311" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Biological Evaluation of Peptidomimetic Aldehydes as Broad-Spectrum Inhibitors against Enterovirus and SARS-CoV-2</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Wenhao Dai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenhao Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenhao++Dai">Wenhao Dai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dirk Jochmans</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dirk Jochmans</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology and Immunology, Rega Institute for Medical Research, Laboratory of Virology and Chemotherapy, Leuven B-3000, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dirk++Jochmans">Dirk Jochmans</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hang Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hang Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hang++Xie">Hang Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hang Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hang Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Virology, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences, Wuhan, Hubei 430071, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hang++Yang">Hang Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jian Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing, 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Li">Jian Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haixia Su</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haixia Su</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haixia++Su">Haixia Su</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Di Chang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Di Chang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, 130 Meilong Road, Shanghai 200237, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Di++Chang">Di Chang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, University of Chinese Academy of Sciences, Hangzhou 310024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiang++Wang">Jiang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jingjing Peng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jingjing Peng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jingjing++Peng">Jingjing Peng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lili Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lili Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, 130 Meilong Road, Shanghai 200237, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lili++Zhu">Lili Zhu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yong Nian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Nian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing, 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Nian">Yong Nian</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rolf Hilgenfeld</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rolf Hilgenfeld</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Molecular Medicine, University of Lübeck, 23562 Lübeck, Germany</div><div class="loa-info-affiliations-info">German Center for Infection Research (DZIF), University of Lübeck, 23562 Lübeck, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rolf++Hilgenfeld">Rolf Hilgenfeld</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8850-2977" title="Orcid link">http://orcid.org/0000-0001-8850-2977</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hualiang Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hualiang Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, University of Chinese Academy of Sciences, Hangzhou 310024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kaixian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaixian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaixian++Chen">Kaixian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leike Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leike Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Virology, Wuhan Institute of Virology, Center for Biosafety Mega-Science, Chinese Academy of Sciences, Wuhan, Hubei 430071, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#057f6d646b6269606c6e6045726d2b6c6a732b666b"><span class="__cf_email__" data-cfemail="87fdefe6e9e0ebe2eeece2c7f0efa9eee8f1a9e4e9">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leike++Zhang">Leike Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2593-2571" title="Orcid link">http://orcid.org/0000-0002-2593-2571</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yechun Xu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yechun Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, University of Chinese Academy of Sciences, Hangzhou 310024, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#8af3e9f2ffcaf9e3e7e7a4ebe9a4e9e4"><span class="__cf_email__" data-cfemail="0d746e75784d7e646060236c6e236e63">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yechun++Xu">Yechun Xu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1581-6155" title="Orcid link">http://orcid.org/0000-0002-1581-6155</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johan Neyts</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johan Neyts</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">KU Leuven, Department of Microbiology and Immunology, Rega Institute for Medical Research, Laboratory of Virology and Chemotherapy, Leuven B-3000, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#2a4045424b4404444f535e596a415f464f5f5c4f4404484f"><span class="__cf_email__" data-cfemail="a6ccc9cec7c888c8c3dfd2d5e6cdd3cac3d3d0c3c888c4c3">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johan++Neyts">Johan Neyts</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Hong Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing, 210023, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, Beijing 100049, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, University of Chinese Academy of Sciences, Hangzhou 310024, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d9b1b5b0ac99aab0b4b4f7b8baf7bab7"><span class="__cf_email__" data-cfemail="90f8fcf9e5d0e3f9fdfdbef1f3bef3fe">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Liu">Hong Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3685-6268" title="Orcid link">http://orcid.org/0000-0003-3685-6268</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02258&amp;href=/doi/10.1021%2Facs.jmedchem.0c02258" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 19, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 December 2020</li><li><span class="item_label"><b>Published</b> online</span>19 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02258" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02258</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DWenhao%2BDai%252C%2BDirk%2BJochmans%252C%2BHang%2BXie%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.0c02258%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BEvaluation%2Bof%2BPeptidomimetic%2BAldehydes%2Bas%2BBroad-Spectrum%2BInhibitors%2Bagainst%2BEnterovirus%2Band%2BSARS-CoV-2%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D15%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02258"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2851</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02258" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Biological Evaluation of Peptidomimetic Aldehydes as Broad-Spectrum Inhibitors against Enterovirus and SARS-CoV-2&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Wenhao&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Dirk&quot;,&quot;last_name&quot;:&quot;Jochmans&quot;},{&quot;first_name&quot;:&quot;Hang&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Hang&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Haixia&quot;,&quot;last_name&quot;:&quot;Su&quot;},{&quot;first_name&quot;:&quot;Di&quot;,&quot;last_name&quot;:&quot;Chang&quot;},{&quot;first_name&quot;:&quot;Jiang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Jingjing&quot;,&quot;last_name&quot;:&quot;Peng&quot;},{&quot;first_name&quot;:&quot;Lili&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Nian&quot;},{&quot;first_name&quot;:&quot;Rolf&quot;,&quot;last_name&quot;:&quot;Hilgenfeld&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Kaixian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Leike&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yechun&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Johan&quot;,&quot;last_name&quot;:&quot;Neyts&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02258&quot;},&quot;abstract&quot;:&quot;A novel series of peptidomimetic aldehydes was designed and synthesized to target 3C protease (3Cpro) of enterovirus 71 (EV71). Most of the compounds exhibited high antiviral activity, and among them, compound 18p demonstrated potent enzyme inhibitory activity and broad-spectrum antiviral activity on a panel of enteroviruses and rhinoviruses. The crystal structure of EV71 3Cpro in complex with 18p determined at a resolution of 1.2 Å revealed that 18p covalently linked to the catalytic Cys147 with an aldehyde group. In addition, these compounds also exhibited good inhibitory activity against the 3CLpro and the replication of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), especially compound 18p (IC50 = 0.034 μM, EC50 = 0.29 μM). According to our previous work, these compounds have no reasons for concern regarding acute toxicity. Compared with AG7088, compound 18p also exhibited good pharmacokinetic properties and more potent anticoronavirus activity, making it an excellent lead for further devel&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02258&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02258" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02258&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02258" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02258&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02258" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02258&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02258&amp;href=/doi/10.1021/acs.jmedchem.0c02258" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02258" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02258" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02258%26sid%3Dliteratum%253Aachs%26pmid%3D33872498%26genre%3Darticle%26aulast%3DDai%26date%3D2021%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BEvaluation%2Bof%2BPeptidomimetic%2BAldehydes%2Bas%2BBroad-Spectrum%2BInhibitors%2Bagainst%2BEnterovirus%2Band%2BSARS-CoV-2%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292236" title="Aldehydes">Aldehydes</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A novel series of peptidomimetic aldehydes was designed and synthesized to target 3C protease (3C<sup>pro</sup>) of enterovirus 71 (EV71). Most of the compounds exhibited high antiviral activity, and among them, compound <b>18p</b> demonstrated potent enzyme inhibitory activity and broad-spectrum antiviral activity on a panel of enteroviruses and rhinoviruses. The crystal structure of EV71 3C<sup>pro</sup> in complex with <b>18p</b> determined at a resolution of 1.2 Å revealed that <b>18p</b> covalently linked to the catalytic Cys147 with an aldehyde group. In addition, these compounds also exhibited good inhibitory activity against the 3CL<sup>pro</sup> and the replication of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), especially compound <b>18p</b> (IC<sub>50</sub> = 0.034 μM, EC<sub>50</sub> = 0.29 μM). According to our previous work, these compounds have no reasons for concern regarding acute toxicity. Compared with <b>AG7088</b>, compound <b>18p</b> also exhibited good pharmacokinetic properties and more potent anticoronavirus activity, making it an excellent lead for further development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar"><issue-title>COVID-19</issue-title></a> special issue.</p></div><div class="extra-info-sec articleNote"><h4><span></span>  Note</h4><p class="first last">This article is made available via the <a href="/page/vi/chemistry_coronavirus_research" class="ext-link">ACS COVID-19 subset</a> for unrestricted RESEARCH re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24543" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24543" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Enterovirus 71 (EV71) belongs to the enterovirus genus of the picornaviridae family, and the genome of EV71 is composed of a single-stranded positive-sense RNA. To the best of our knowledge, EV71 is not only the primary pathogen of hand, foot, and mouth disease (HFMD) but also closely associated with neurological syndromes such as severe encephalitis and aseptic meningitis, and these diseases caused by the EV71 have become a worldwide health problem. Especially, young children are more susceptible to be infected by the enterovirus.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> However, to date, there are no approved drugs to prevent or treat the associated diseases.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Considering the detriment of HFMD and the central neurological syndromes in children, the development of effective antiviral drugs is urgently needed.</div><div class="NLM_p">The genome of enterovirus 71 encodes a polyprotein precursor, which is cleaved by the 3C protease (3C<sup>pro</sup>) and 2A protease (2A<sup>pro</sup>) into structural proteins and nonstructural proteins, of which the 3C<sup>pro</sup> is responsible for most of the cleavages. The 3C<sup>pro</sup> of enteroviruses and rhinoviruses (RV) is essential for viral replication, and it not only shared a high degree of homology at an amino acid level but also contained a Cys-His-Asp/Glu catalytic triad. In addition, the 3C<sup>pro</sup> is an exceptional cysteine protease with unique folding and catalytic mechanism, and a Gln is almost required in the P1 position of the substrates. As far as we know, none of the known human proteases possessed a similar cleavage specificity, which makes 3C<sup>pro</sup> become a highly prospective target for developing broad-spectrum drugs.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Recent efforts in drug discovery also furnished several inhibitors against the EV71 3C<sup>pro</sup> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Those peptidomimetic compounds with a warhead in P1′ and a lactam ring in P1 could be briefly divided into reversible and irreversible inhibitors according to their binding modes. Compounds <b>1</b>–<b>6</b> are reversible inhibitors, among which compounds <b>1</b>, <b>2</b>, <b>4</b>, and <b>5</b> are covalent reversible inhibitors, while <b>3</b> and <b>6</b> are noncovalent inhibitors. Those compounds exhibited good anti-EV71 activity with EC<sub>50</sub> values in the range of 0.009–3.7 μM, while the broad-spectrum antiviral activity and the drug-like properties were rarely evaluated.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Rupintrivir (<b>AG7088</b>)<a onclick="showRef(event, 'cit5a'); return false;" href="javascript:void(0);" class="ref cit5a">(5a)</a> was reported to have the excellent antivirus activity against EV71 with an EC<sub>50</sub> value of 0.009 μM and has entered clinical trials as a protease inhibitor targeting rhinovirus 3C<sup>pro</sup>, but its activity against coronaviruses is weak. Compounds <b>1</b><a onclick="showRef(event, 'cit5b'); return false;" href="javascript:void(0);" class="ref cit5b">(5b)</a> and <b>2</b><a onclick="showRef(event, 'cit5c cit5d'); return false;" href="javascript:void(0);" class="ref cit5c cit5d">(5c,d)</a> showed good anti-EV71 activity, and the selectivity of compounds <b>3</b><a onclick="showRef(event, 'cit5e'); return false;" href="javascript:void(0);" class="ref cit5e">(5e)</a> and <b>4</b><a onclick="showRef(event, 'cit5f'); return false;" href="javascript:void(0);" class="ref cit5f">(5f)</a> toward other common mammalian proteases was high, but their pharmaceutical properties were not satisfied; no more research progresses have been reported. Our compound (<b>5</b><a onclick="showRef(event, 'cit5g'); return false;" href="javascript:void(0);" class="ref cit5g">(5g)</a>) exhibited broad-spectrum antiviral activity, but its activity against EV71 was weak. As reported, the plasma stability of compounds <b>6</b><a onclick="showRef(event, 'cit5h'); return false;" href="javascript:void(0);" class="ref cit5h">(5h)</a> and <b>7</b><a onclick="showRef(event, 'cit5i'); return false;" href="javascript:void(0);" class="ref cit5i">(5i)</a> was impressive, while their pharmacokinetic properties needed to be further improved. Although many inhibitors with different warheads had been reported, there was still no effective drug on the market. With those in mind, novel broad-spectrum antiviral inhibitors with good pharmacokinetic properties and safety need to be designed.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representatives of reported EV71 3C protease inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Compound Design</h3><div class="NLM_p"><b>AG7088</b> has potent anti-EV71 activity (EC<sub>50</sub> = 0.009 μM), so we analyzed the crystal structure of EV71 3C<sup>pro</sup> with <b>AG7088</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) for our compound’s design.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The results demonstrate that α,β-unsaturated ester of <b>AG7088</b> forms a covalent linkage with the Cys147 residue in the S1′ subsite of the EV71 3C<sup>pro</sup>, which is the key point for maintaining the antivirus activity. The (<i>S</i>)-γ-lactam ring at P1 position forms hydrogen bonds with the crucial residues including  Thr142 and  His161 in the S1 subsite, and the substituted phenyl group at the P2 position occupies the S2 subsite well. The complex also revealed that the isopropyl in the P3 moiety is solvent-exposed, and the P4 moiety forms hydrogen bonds with Gly164, Asn145, and Ser128.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a><sup>h</sup> Unfortunately, the α,β-unsaturated ester is easily hydrolyzed, the plasma stability of the <b>AG7088</b> is poor, and its half-life in rat plasma is less than 2 min; in addition, <b>AG7088</b> was inactive to the SARS-CoV 3CL protease (IC<sub>50</sub> > 100 μM).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> We want to overcome this shortcoming of <b>AG7088</b> and obtain some novel compounds.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structure of the surface representation of EV71 3C<sup>pro</sup> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GHT">4GHT</a>) complexed with the <b>AG7088</b> (yellow).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In our previous work, peptidomimetic α-ketoamides (compound <b>5</b>) exhibited broad-spectrum antivirus activity.<a onclick="showRef(event, 'cit5g'); return false;" href="javascript:void(0);" class="ref cit5g">(5g)</a> Former reports showed that compound <b>1</b> also had good anti-EV71 activity (EC<sub>50</sub> = 0.096 μM). After analyzing the crystal structure of <b>AG7088</b> with EV71 3C<sup>pro</sup> and comparing the structure of compound <b>1</b>, compound <b>5</b>, and <b>AG7088</b>, we found that they shared similar important key pharmacophore fragments with a warhead and (<i>S</i>)-γ-lactam ring. In this core structure, the warheads could be covalently linked to Cys147, and the lactam ring could interact with the crucial residues, and we envisioned <b>AG7088</b> showed better antivirus activity might benefit from the additional interaction at the P3 and P4 position. When we embarked on designing the novel compounds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), the key pharmacophore fragments in compounds <b>1</b>, <b>5</b>, and <b>AG7088</b> were extracted, and then some common warheads were investigated to get compounds <b>19a</b> and <b>19b</b>. Subsequently, we introduced some heterocyclic moieties into the P3 position to obtain the corresponding peptidomimetic aldehydes <b>18b</b>–<b>18p</b>. In a second approach, a valine moiety was introduced in the P3 position to keep the chain length similar to <b>AG7088</b>, and different heterocycles were investigated in the P4 position to get compounds <b>26a</b>–<b>26e</b>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of novel EV71 3C protease inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The synthetic routes and chemical structures of the compounds (<b>19a</b>, <b>19b</b>, and <b>18b</b>–<b>18p</b>) are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The starting material <b>8</b> was obtained from commercial suppliers and used without further purification, and the key intermediate <b>11</b><a onclick="showRef(event, 'cit5e'); return false;" href="javascript:void(0);" class="ref cit5e">(5e)</a> was synthesized by following the literature. The intermediate <b>13</b> was synthesized starting from the amino <b>11</b> and <i>N</i>-(<i>t</i>-butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine <b>12</b>. After the <i>t</i>-butoxycarbonyl group was removed from <b>13</b> by 4 M HCl in dioxane, the intermediate <b>14</b> was obtained. A subsequent coupling reaction between compound <b>14</b> and the corresponding acids <b>15</b> resulted in esters <b>16a</b>–<b>16p</b>. The peptidomimetic aldehydes <b>1</b> and <b>18b</b>–<b>18p</b> were approached via a two-step route, in which the ester derivatives <b>16a</b>–<b>16p</b> were first reduced with NaBH<sub>4</sub> to generate the primary alcohols <b>17a</b>–<b>17p</b>, and subsequently, <b>17a</b>–<b>17p</b> were oxidized into aldehydes <b>1</b> and <b>18b</b>–<b>18p</b> with Dess–Martin periodinane (DMP). Finally, compounds <b>19a</b> and <b>19b</b> were obtained from <b>1</b> by the Wittig reaction.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Procedure of Target Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiHMDS, THF, −78 °C; (b) NaBH<sub>4</sub>, CoCl<sub>2</sub>, 0 °C; (c) 4 M HCl, 12 h; (d) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12 h; (e) 4 M HCl, 12 h; (f) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12 h; (g) NaBH<sub>4</sub>, CH<sub>3</sub>OH; (h) Dess–Martin periodinane, NaHCO<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>; (i) Ph<sub>3</sub>PCH<sub>2</sub>COOR<sub>4</sub>, Et<sub>3</sub>N, DCM.</p></p></figure><div class="NLM_p">The synthetic procedure of compounds <b>26a</b>–<b>26e</b> was shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Compound <b>21</b> was obtained via a condensation reaction between the same intermediate <b>14</b> with <i>N</i>-(<i>t</i>-butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-valine <b>20</b>, and then, the <i>t</i>-butoxycarbonyl group of <b>21</b> was removed by 4 M HCl in dioxane to obtain <b>22</b>. Subsequently, compound <b>22</b> was coupled with the corresponding acid <b>23</b> to afford esters <b>24a</b>–<b>24e</b>. After completion of a reduction of the esters <b>24a</b>–<b>24e</b>, the desired products (<b>25a-25e</b>) were reoxidized by DMP to obtain the final products <b>26a</b>–<b>26e</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis Procedure of Target Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12 h; (b) 4 M HCl, 12 h; (c) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12h; (d) NaBH<sub>4</sub>, CH<sub>3</sub>OH; (e) Dess–Martin Periodinane, NaHCO<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationship of the Compounds</h3><div class="NLM_p">All the synthesized compounds were tested for inhibitory activity of EV71 3C<sup>pro</sup> and antiviral activity of EV71, and results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Enzyme Inhibitory Activity and Anti-EV71 Activities of Peptidomimetic Aldehydes with R<sub>1</sub> and R<sub>2</sub> Modifications<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0008.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0009.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Each value represented the average results from three independent experiments.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Enzyme Inhibitory Activity and anti-EV71 Activities of Peptidomimetic Aldehydes with R<sub>3</sub> Modifications<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0013.gif" alt="" id="GRAPHIC-d7e1121-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0014.gif" alt="" id="GRAPHIC-d7e1122-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Each value represented the average results from three independent experiments.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Modification on R<sub>1</sub> and R<sub>2</sub></h3><div class="NLM_p last">Structural modifications on R<sub>1</sub> and R<sub>2</sub> were first conducted, and a number of derivatives have been designed, synthesized, and biologically evaluated. The results were summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and two compounds (compound <b>1</b> and <b>AG7088</b>) were used as references in this work. IC<sub>50</sub> values of compound <b>1</b> were inconsistent with previous reports, which might be caused by a difference in enzyme concentration.<a onclick="showRef(event, 'cit5b'); return false;" href="javascript:void(0);" class="ref cit5b">(5b)</a> These data indicated that the enzyme inhibitory activity of peptidomimetic aldehyde (compound <b>1</b>, IC<sub>50</sub> = 4.57 ± 0.27 μM) was weaker than the compounds, in which α,β-unsaturated ester was defined as the warhead (compounds <b>19a</b> and <b>19b</b>), while compound <b>1</b> displayed better anti-EV71 activity (EC<sub>50</sub> = 0.10 ± 0.01 μM) than α,β-unsaturated methyl ester <b>19a</b> (EC<sub>50</sub> = 1.21 ± 0.14 μM) and α,β-unsaturated benzyl ester <b>19b</b> (EC<sub>50</sub> = 3.10 ± 0.09 μM), and the antiviral results showed that a small group might be more suitable in P1′. Based on the antiviral activity, an aldehyde was selected as a new warhead for further optimization. When aldehyde on R<sub>2</sub> was incorporated and the R<sub>1</sub> moiety was replaced with heterocyclic motifs, most of the target compounds (<b>18b</b>–<b>18l</b>, <b>18n</b>, and <b>18p</b>) showed better 3C<sup>pro</sup> inhibitory activity (IC<sub>50</sub> < 4.0 μM) than compound <b>1</b> (IC<sub>50</sub> = 4.57 μM), indicating that the introduction of the heterocyclic ring in P3 might be able to form additional interactions with the S4 subsite to improve the inhibitory activity of 3C<sup>pro</sup>. The inhibitory activity of compound <b>11m</b> was decreased, which might be due to the fact that the nitrogen atom in this compound could not form an additional interaction. As we know, the antiviral activity is a result caused by multiple factors, which was related not only to enzyme inhibitory activity but also the other properties such as permeability. So the EC<sub>50</sub> values deserved further discussion. The enzyme inhibitory activity of monocyclic moiety-substituted derivatives (<b>18b</b>, <b>18c</b>) was increased, while a lower antiviral activity was observed when comparing to compound <b>1</b>. Those results indicated that monocyclic moieties might not be a good choice on R<sub>1</sub>, and then bicyclic heterocycles were investigated. The anti-EV71 activities of compounds with heterocyclic acene moieties <b>18d</b>–<b>18h</b> were decreased, and the introduction of 7-bromoimidazo[1,2-<i>a</i>]pyridine (compound <b>18i</b>) also reduced the anti-EV71 activity. Subsequently, benzoheterocycles were introduced on R<sub>1</sub> (<b>18j</b>–<b>18p</b>), and compounds <b>18j</b>, <b>18k</b>, and <b>18m</b> had similar activities compared to compound <b>1</b>. The introduction of methyl on quinoxaline (<b>18l</b>) reduced the antiviral activity; when R<sub>1</sub> was replaced with a substituted quinoline (<b>18n</b>), the anti-EV71 activity was also increased. It was noteworthy that compound <b>18o</b> (EC<sub>50</sub> = 0.07 ± 0.01 μM) showed better anti-EV71 activity than compound <b>1</b>, albeit the cytotoxicity of these compounds was slightly increased (CC<sub>50</sub> = 16.4 μM). The 2-indole scaffold was incorporated, and the activity of <b>18p</b> against EV71 (EC<sub>50</sub> = 0.030 ± 0.002 μM) was increased. The discrepancy of the activities between compounds including <b>18h</b> and <b>18j</b> with <b>18p</b> indicated that the NH moiety in the 2-indole scaffold was vital for maintaining the anti-EV71 activity.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Modification on the R<sub>3</sub> Position</h3><div class="NLM_p last">On the other hand, as showed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, structural modifications had been continued by introducing a valine motif at the P3 position, and most of the target compounds (<b>26a</b>–<b>26e</b>) showed good enzyme inhibitory activities (ranging from 2.43 μM to 7.49 μM), while the inhibitory activities of those compounds were weaker than <b>AG7088</b> (IC<sub>50</sub> = 1.89 μM). We proposed the reason might be that, after replacing the carbon atom in <b>AG7088</b> with a nitrogen atom, the conformation of those compounds (<b>26a</b>–<b>26e</b>) changes a lot, which made those compounds unable to occupy the binding pocket very well, and α,β-unsaturated ester showed more potent inhibitory activity against 3C<sup>pro</sup> than aldehyde. The results of antiviral activities showed that the introduction of 5-methyl-1,2-oxazole and benzoheterocycles moiety on R<sub>3</sub> made the anti-EV71 activity of the corresponding compounds (<b>26a</b>–<b>26d</b>) comparable to compound <b>1</b>. Especially when the quinoline group was introduced into the R<sub>3</sub> position, compound <b>26d</b> showed good activity (EC<sub>50</sub> = 0.043 ± 0.010 μM), while the activities of these tripeptide compounds were decreased compared with compound <b>18p</b> and <b>AG7088</b>. In addition, compounds <b>26d</b> and <b>18p</b> exhibited safety cytotoxicity (CC<sub>50</sub> = 82.8 μM and CC<sub>50</sub> > 100 μM, respectively).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystal Structure of  EV71 3C<sup>pro</sup> in Complex with <b>18p</b></h3><div class="NLM_p">To understand the binding mode of these inhibitors with the protease, the complex structure of EV71 3C<sup>pro</sup> bound with <b>18p</b> was determined at a resolution of 1.2 Å. Compound <b>18p</b> bound into the substrate-binding site at the surface of the protease and occupied S1, S2, and S4 subsites (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). At the S1′ subsite, the aldehyde group of <b>18p</b> covalently linked to the catalytic Cys147 of the protease (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B,C). In addition to this covalent bond, the oxygen atom of the aldehyde group established H-bonds with the side chain of catalytic His40 directly and the NH of the Gly145 main chain (part of the oxyanion hole) through a water molecule (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The (<i>S</i>)-γ-lactam ring of <b>18p</b> perfectly engaged within the S1 subsite. The oxygen atom of the (<i>S</i>)-γ-lactam ring formed H-bonds with the side chains of His161 and Thr142, while the nitrogen atom of the (<i>S</i>)-γ-lactam ring formed a H-bond with the main chain of Thr142. The benzyl ring of <b>18p</b> occupied the S2 subsite by forming hydrophobic interactions with His40, Glu71, and Leu127. The indole group of <b>18p</b> fitted into the S4 subsite and donated a H-bond to the main chain of Gly164. In addition, the amide bonds of <b>18p</b> also participated in binding by establishing H-bonds with Ile162 and Gly164 directly and with Ser128 through a water molecule (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structure of the EV71 3C<sup>pro</sup> in complex with <b>18p</b>. (A) The binding mode of <b>18p</b> at the substrate-binding site of the EV71 3C<sup>pro</sup> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7DNC">7DNC</a>). The EV71 3C<sup>pro</sup> was shown as a molecular surface, and <b>18p</b> was shown by light orange sticks. (B) 2<i>F</i>o-<i>F</i>c density maps contoured at 2.0 σ are shown for <b>18p</b> and C147. (C) Interactions of <b>18p</b> with the surrounding residues revealed by the crystal structure. Residues are shown as light blue sticks, and H-bonds are represented by black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A structure overlay of the EV71 3C<sup>pro</sup>-<b>18p</b> and the EV71 3C<sup>pro</sup>-<b>AG7088</b> complexes revealed that these two compounds adopted similar binding modes (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1A">Figure S1A</a>). The difference mainly lied in the interactions with the S1′ and S4 subsites. <b>AG7088</b> covalently bound to the catalytic Cys147 with its α,β-unsaturated ketone, while the aldehyde group of <b>18p</b> was used to covalently link to the catalytic Cys147. Simultaneously, the oxygen atom of the aldehyde group participated in forming multiple H-bonds with the surrounding residues including the catalytic His40. At the S4 subsite, the indole ring of <b>18p</b> flipped up and established a shorter H-bond (2.9 Å) with Gly164 compared to the H-bond (3.1 Å) between <b>AG7088</b> and Gly164. In addition, a new H-bond was formed between the amide bond of <b>18p</b> and Ser128, mediated by a water molecule (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1B, C">Figure S1B, C</a>).</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Activity on a Panel of Enteroviruses and Rhinoviruses</h3><div class="NLM_p">Considering that the 3C proteases are conserved between different viruses, four compounds (<b>1</b>, <b>18p</b>, <b>19a</b>, and <b>26b</b>) were further tested against a panel of relevant enteroviruses and rhinoviruses, and results were summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. The results demonstrated that the derivatives with aldehyde warhead had potent broad-spectrum antiviral activities. Compound <b>1</b> showed better broad-spectrum antiviral activity than the α, β-unsaturated ester <b>19a</b>. Additionally, the dipeptide compound <b>18p</b> was proved to have more efficacy than the tripeptide compound <b>26b</b>. Besides, compounds <b>18p</b> also exhibited excellent anti-EV68 activity (EC<sub>50</sub> = 0.03 ± 0.01 μM), and compounds <b>18p</b> and <b>26b</b> also exhibited high anti-CoxA21 activity (EC<sub>50</sub> = 0.43 ± 0.11 μM and EC<sub>50</sub> = 0.51 ± 0.01 μM, respectively), while its antiviral activities against CoxB3 and HRV were weak, which might be caused by the difference in substrate binding pockets (<a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>). According to these results, compound <b>18p</b> has a lot of potential to be accessed in a broad-spectrum antiviral drug.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Activity of Inhibitors against a Panel of Enteroviruses and Rhinoviruses<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">EV71 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">EV68 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">CoxA21 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">CoxB3 EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">RV-A02-WT EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">RV-B14-WT EC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">0.10 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.08 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">1.73 ± 0.82</td><td class="colsep0 rowsep0" align="char" char=".">15.87</td><td class="colsep0 rowsep0" align="char" char=".">6.60</td><td class="colsep0 rowsep0" align="char" char=".">1.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="char" char="±">0.030 ± 0.002</td><td class="colsep0 rowsep0" align="char" char="±">0.03 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.43 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">4.19</td><td class="colsep0 rowsep0" align="char" char=".">1.62</td><td class="colsep0 rowsep0" align="char" char=".">0.81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19a</b></td><td class="colsep0 rowsep0" align="char" char="±">1.21 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.10 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">3.62 ± 1.19</td><td class="colsep0 rowsep0" align="char" char=".">77.67</td><td class="colsep0 rowsep0" align="char" char=".">1.68</td><td class="colsep0 rowsep0" align="char" char=".">1.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.12 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.26 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">0.51 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">9.15</td><td class="colsep0 rowsep0" align="char" char=".">1.02</td><td class="colsep0 rowsep0" align="char" char=".">0.98</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The value of EV71, EV68, and CoxA21 represented the average results from three independent experiments.</p></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Antiviral Activity on SARS-CoV-2</h3><div class="NLM_p">In late December 2019, an emerged coronavirus called SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2), causes the pandemic COVID-19 (coronavirus disease 2019). There are no specific antiviral drugs approved by the FDA except Remdesivir, so the development of more effective antiviral drugs is of great significance.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> SARS-CoV-2 is an enveloped, positive-sense, single-stranded RNA virus that belongs to the β-lineage of the coronavirus. The genome is translated into two polyproteins, and then the polyproteins were cleaved by 3C-like protease (3CL<sup>pro</sup>, also named the main protease) and papain-like proteinase (PL<sup>pro</sup>). The majority of this proteolytic processing utilizes the 3CL<sup>pro</sup>, which plays a vital role in SARS-CoV-2′s replication. Unlike enterovirus 3C<sup>pro</sup>, the active form of the SARS-CoV-2 3CL<sup>pro</sup> is a dimer, and it has a catalytic dyad containing cysteine and histidine.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> As we know, the 3C<sup>pro</sup> of EV71 and 3CL<sup>pro</sup> of SARS-CoV-2 share some similarities; for example, both the catalytic sites contain cysteine and histidine, and a glutarnine (Gln) is almost always required in the P1 position of their substrates. In our previous work, we found the active sites of 3C<sup>pro</sup> and 3CL<sup>pro</sup> also are similar and usually composed of four sites: S1′, S1, S2, and S4.<a onclick="showRef(event, 'cit5g cit9d'); return false;" href="javascript:void(0);" class="ref cit5g cit9d">(5g,9d)</a> Recently, many inhibitors targeting SARS-CoV-2 3CL<sup>pro</sup> have been reported, and most of them shared similar key pharmacophore fragments (warhead and lactam ring) and exhibited good inhibitor activity against 3CL<sup>pro</sup> and replication of SARS-CoV-2 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'cit9b cit9c ref11'); return false;" href="javascript:void(0);" class="ref cit9b cit9c ref11">(9b,c,11)</a> Among them, two peptide inhibitors developed by Pfizer (<b>PF-007304814</b> is a phosphate prodrug of <b>PF-00835231</b>)<a onclick="showRef(event, 'cit11b'); return false;" href="javascript:void(0);" class="ref cit11b">(11b)</a> and our group (compound <b>29</b>)<a onclick="showRef(event, 'cit9d'); return false;" href="javascript:void(0);" class="ref cit9d">(9d)</a> show potent antiviral activity and good safety, and both have entered phase I clinical trials. Our antienterovirus compounds show some similarity to compound <b>29</b>, so some 3C<sup>pro</sup> inhibitors were selected to be further tested against 3CL<sup>pro</sup> as well as the replication of SARS-CoV-2.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representatives of reported SARS-CoV-2 3CL protease inhibitors</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The results of enzyme inhibitory activity and anti-SARS-CoV-2 activity were summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Most of the compounds showed excellent inhibitory activity of 3CL<sup>pro</sup> (IC<sub>50</sub> < 0.10 μM), especially when the IC<sub>50</sub> value of <b>18p</b> was 0.034 μM. However, when the quinoline group was incorporated, the enzyme inhibitory activities were decreased (<b>18m</b>, <b>18n</b>, and <b>26d</b>). In our previous work, the 2-indole moiety could form an additional H-bond with Glu166 in the S4 subsite.<a onclick="showRef(event, 'cit9d'); return false;" href="javascript:void(0);" class="ref cit9d">(9d)</a> When 2-quinoline and the derivative (<b>18m</b>, <b>18n</b>, and <b>18o</b>) were introduced, the H-bond could not be formed, so this might be the reason for the decrease of inhibitory activity against 3CL<sup>pro</sup>. Then, the antiviral activity against SARS-CoV-2 was evaluated, of which six benzoheterocyclic dipeptide compounds exhibited excellent inhibitory activity (EC<sub>50</sub> < 0.5 μM), and the EC<sub>50</sub> values of <b>18o</b> and <b>18p</b> were 0.25 μM and 0.29 μM, respectively. In addition, the selection index of <b>18p</b> (SI = 2786) is better than <b>18o</b> (SI = 1192). The results showed the inhibitory activities of the tripeptide compounds (<b>26a</b>–<b>26d</b>) were greater than 1 μM, and the reason might be that the tripeptide compounds showed poor membrane permeability.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Those results indicate the compound <b>18p</b> also is a good starting point for further optimization as a SARS-CoV-2 inhibitor.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Enzyme Inhibitory Activities and Anti-SARS-CoV-2 Activities of Peptidomimetic Aldehydes<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.078 ± 0.016</td><td class="colsep0 rowsep0" align="char" char="±">1.35 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18e</b></td><td class="colsep0 rowsep0" align="char" char="±">0.059 ± 0.005</td><td class="colsep0 rowsep0" align="char" char="±">0.76 ± 0.26</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18g</b></td><td class="colsep0 rowsep0" align="char" char="±">0.097 ± 0.017</td><td class="colsep0 rowsep0" align="char" char="±">8.21 ± 0.68</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18j</b></td><td class="colsep0 rowsep0" align="char" char="±">0.080 ± 0.002</td><td class="colsep0 rowsep0" align="char" char="±">0.49 ± 0.001</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18l</b></td><td class="colsep0 rowsep0" align="char" char="±">0.065 ± 0.011</td><td class="colsep0 rowsep0" align="char" char="±">0.44 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">602.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18m</b></td><td class="colsep0 rowsep0" align="char" char="±">0.140 ± 0.012</td><td class="colsep0 rowsep0" align="char" char="±">0.43 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">627.2 ± 16.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18n</b></td><td class="colsep0 rowsep0" align="char" char="±">0.240 ± 0.042</td><td class="colsep0 rowsep0" align="char" char="±">0.35 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">823.1 ± 32.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18o</b></td><td class="colsep0 rowsep0" align="char" char="±">0.120± 0.003</td><td class="colsep0 rowsep0" align="char" char="±">0.25 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">298.7 ± 4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="char" char="±">0.034 ± 0.004</td><td class="colsep0 rowsep0" align="char" char="±">0.29 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">808.7 ± 20.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.067 ± 0.014</td><td class="colsep0 rowsep0" align="char" char="±">>2</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.068 ± 0.012</td><td class="colsep0 rowsep0" align="char" char="±">4.22 ± 0.25</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26c</b></td><td class="colsep0 rowsep0" align="char" char="±">0.067 ± 0.014</td><td class="colsep0 rowsep0" align="char" char="±">5.62 ± 1.26</td><td class="colsep0 rowsep0" align="char" char="±">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.18 ± 0.029</td><td class="colsep0 rowsep0" align="char" char="±">4.12 ± 0.52</td><td class="colsep0 rowsep0" align="char" char="±">604.1 ± 5.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Each value represented the average results from three independent experiments</p></div></div></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Preliminary Pharmacokinetic (PK) Evaluation of Compounds <b>18p</b> and <b>26d</b></h3><div class="NLM_p">To explore the further druggability of the novel peptide aldehydes, compounds <b>18p</b> and <b>26d</b> were evaluated for their pharmacokinetic properties in mice after intraperitoneal (20 mg/kg), subcutaneous (5 mg/kg), and intravenous (5 mg/kg) administration. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, compound <b>18p</b> given intraperitoneally and subcutaneously displayed a much higher area under the curve (AUC) value than that of <b>26d</b>. Compound <b>18p</b> also displayed a longer half-life (<i>T</i><sub>1/2</sub>) of 5.85 h when administrated intravenously. Those results indicate that compound <b>18p</b> has better PK properties than <b>26d</b> to warrant further study. The reason why the bioavailability of compound <b>18p</b> is greater than 100% may be due to changes in the clearance rate in different routes of administration, and the clearance rate of <b>18p</b> is fast by intravenous injection.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Preliminary Pharmacokinetic (PK) Evaluation of Compounds <b>18p</b> and <b>26d</b><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±"><i>T</i><sub><sub><b>1/2</b></sub></sub></th><th class="colsep0 rowsep0" align="center" char="±"><i>T</i><sub><sub>max</sub></sub></th><th class="colsep0 rowsep0" align="center" char="±"><i>C</i><sub><sub>max</sub></sub></th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub><sub>last</sub></sub></th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub><sub>INF_obs</sub></sub></th><th class="colsep0 rowsep0" align="center" char="±">CL</th><th class="colsep0 rowsep0" align="center" char="±">MRT</th><th class="colsep0 rowsep0" align="center" char="±">Vss<sub><sub>_obs</sub></sub></th><th class="colsep0 rowsep0" align="center" char="."><i>F</i></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">admin</th><th class="colsep0 rowsep0" align="center" char="±">(h)</th><th class="colsep0 rowsep0" align="center" char="±">(h)</th><th class="colsep0 rowsep0" align="center" char="±">(ng/mL)</th><th class="colsep0 rowsep0" align="center" char="±">(h ng/mL)</th><th class="colsep0 rowsep0" align="center" char="±">(h ng/mL)</th><th class="colsep0 rowsep0" align="center" char="±">(mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="±">(h)</th><th class="colsep0 rowsep0" align="center" char="±">(mL/kg)</th><th class="colsep0 rowsep0" align="center" char=".">(%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="left">ip</td><td class="colsep0 rowsep0" align="char" char="±">5.36 ± 1.12</td><td class="colsep0 rowsep0" align="char" char="±">0.25</td><td class="colsep0 rowsep0" align="char" char="±">14572 ± 3105</td><td class="colsep0 rowsep0" align="char" char="±">15952 ± 3468</td><td class="colsep0 rowsep0" align="char" char="±">16080 ± 3559</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1.69 ± 0.60</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">230</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sc</td><td class="colsep0 rowsep0" align="char" char="±">4.96 ± 0.71</td><td class="colsep0 rowsep0" align="char" char="±">0.58 ± 0.29</td><td class="colsep0 rowsep0" align="char" char="±">2762 ± 689</td><td class="colsep0 rowsep0" align="char" char="±">3568 ± 490</td><td class="colsep0 rowsep0" align="char" char="±">3579 ± 489</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1.34 ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">206</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iv</td><td class="colsep0 rowsep0" align="char" char="±">5.85 ± 0.75</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1732 ± 161</td><td class="colsep0 rowsep0" align="char" char="±">1745 ± 163</td><td class="colsep0 rowsep0" align="char" char="±">48.0 ± 4.7</td><td class="colsep0 rowsep0" align="char" char="±">1.42 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">4119 ± 676</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left"><b>26d</b></td><td class="colsep0 rowsep0" align="left">ip</td><td class="colsep0 rowsep0" align="char" char="±">5.38 ± 0.28</td><td class="colsep0 rowsep0" align="char" char="±">0.25</td><td class="colsep0 rowsep0" align="char" char="±">4542 ± 457</td><td class="colsep0 rowsep0" align="char" char="±">4637 ± 472</td><td class="colsep0 rowsep0" align="char" char="±">4651 ± 472</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1.11 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">84.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sc</td><td class="colsep0 rowsep0" align="char" char="±">5.03 ± 4.30</td><td class="colsep0 rowsep0" align="char" char="±">0.25</td><td class="colsep0 rowsep0" align="char" char="±">1159 ± 46</td><td class="colsep0 rowsep0" align="char" char="±">1321 ± 222</td><td class="colsep0 rowsep0" align="char" char="±">1338 ± 231</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1.82 ± 0.89</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char=".">96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iv</td><td class="colsep0 rowsep0" align="char" char="±">2.98 ± 2.50</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">1378 ± 116</td><td class="colsep0 rowsep0" align="char" char="±">1390 ± 114</td><td class="colsep0 rowsep0" align="char" char="±">60.2 ± 4.7</td><td class="colsep0 rowsep0" align="char" char="±">0.98 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="±">3536 ± 298</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">The value represented the average results from three independent experiments.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30546" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30546" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, a series of novel protease inhibitors with an aldehyde warhead was designed, synthesized, and biologically evaluated on EV71 by analyzing the crystal structure of EV71 3C protease with <b>AG7088</b>. Most of the compounds have potent inhibitory activity of 3C<sup>pro</sup> and EV71. The SAR study indicated that the introduction of aldehyde at P1′ position presented better antiviral activities than the α, β-unsaturated ester. Heteroaromatic scaffolds were introduced at the R<sub>1</sub> group, and most of the compounds displayed good antiviral activities, especially when R<sub>1</sub> is an indole moiety (<b>18p</b>). Compound <b>18p</b> not only has high inhibitory activity of 3C<sup>pro</sup> and EV71 (3C<sup>pro</sup>: IC<sub>50</sub> = 2.36 μM, EV71: EC<sub>50</sub> = 0.030 μM), but also against EV68 (EC<sub>50</sub> = 0.03 μM), CoxA21 (EC<sub>50</sub> = 0.43 μM) and RV-B14-WT (EC<sub>50</sub> = 0.81 μM). It also showed moderate activity against CoxB3 (EC<sub>50</sub> = 4.19 μM) and RV-A02-WT (EC<sub>50</sub> = 1.62 μM) and low toxicity (CC<sub>50</sub> > 100 μM). The crystal structure of EV71 3C<sup>pro</sup> in complex with <b>18p</b> was also determined at a resolution of 1.2 Å. It showed that <b>18p</b> fitted into the S1′, S1, S2, and S4 sites perfectly and established multiple H-bonds with the surrounding residues including the catalytic His40. In addition, the aldehyde group of <b>18p</b> covalently bound to the catalytic Cys147, which was essential for maintaining the potency of these newly designed inhibitors, and the NH in the indole group formed a hydrogen bond with the main chain of Gly164. Some compounds were further evaluated against 3CL<sup>pro</sup> and SARS-CoV-2, and the <b>18p</b> also showed excellent inhibitory activity (3CL<sup>pro</sup>: IC<sub>50</sub> = 0.034 μM, SARS-CoV-2: EC<sub>50</sub> = 0.29 μM). In addition, this class of compounds also has no reasons for concern regarding acute toxicity.<a onclick="showRef(event, 'cit9d'); return false;" href="javascript:void(0);" class="ref cit9d">(9d)</a> Compared with <b>AG7088</b>, compound <b>18p</b> exhibited good PK properties and more potent anticoronavirus activity, and it is an excellent starting point for further optimization toward a broad-spectrum antiviral drug.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77388" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77388" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Methods</h3><div class="NLM_p last">The materials and solvents were purchased from commercial sources and used without further purification. All products were characterized by their NMR and MS spectra. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a 400, 500, or 600 MHz instrument. Compounds were purified by chromatography with silica gel (300–400 mesh). Analytical thin-layer chromatography (TLC) was HSGF 254 (0.15–0.2 mm thickness). Preparative thin-layer chromatography (PTLC) was HSGF 254 (0.4–0.5 mm thickness). High-resolution mass spectra (HRMS) were measured on a Micromass Ultra Q-TOF spectrometer. HPLC analysis of all final compounds was performed on Agilent-1100 HPLC with a binary pump and photodiode array detector (DAD), using an Agilent Extend-C18 column (150 mm × 4.6 mm, 5 μm). All final compounds were analyzed using MeOH/H<sub>2</sub>O = 70:30 (v/v) (0.8 mL/min), and all of them had an at least 95% purity.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Synthesis Procedure of Compounds</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Synthetic Procedure of Compounds <b>9</b>–<b>11</b></h4><div class="NLM_p">The solution of lithium bis(trimethylsilyl)amide (LHMDS) (94 mL, 1 M in THF) was added dropwise to a solution of <i>N</i>-Boc-<span class="smallcaps smallerCapital">l</span>-glutamic acid dimethyl ester (<b>8</b>) (12.0 g, 43.6 mmol) in THF (100 mL) at −78 °C; then, the mixture was stirred at −78 °C for 1 h. Subsequently, bromoacetonitrile (3.24 mL, 46.6 mmol) was added dropwise to the mixture under the temperature of −78 °C, and the reaction was kept at −78 °C for an additional 4 h. After the reactant was consumed, the reaction was quenched by NH<sub>4</sub>Cl (40 mL). The reaction mixture was warm up to room temperature and extracted with ethyl acetate (50 mL × 3). The organic layers were concentrated and purified by flash column chromatography (petroleum ether/ethyl acetate = 4:1) to give product <b>9</b> (7.58 g, 55%) as a colorless oil. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 5.11 (d, <i>J</i> = 7.5 Hz, 1H), 4.38 (s, 1H), 3.77 (s, 3H), 3.75 (s, 3H), 2.92–2.82 (m, 1H), 2.81–2.71 (m, 2H), 2.24–2.08 (m, 2H), 1.44 (s, 9H).</div><div class="NLM_p">Then, in a round-bottomed flask, compound <b>9</b> (6.0 g, 19.09 mmol) was dissolved in anhydrous MeOH (100 mL) before CoCl<sub>2</sub>·6H<sub>2</sub>O (2.72 g, 11.45 mmol) was added at 0 °C. Subsequently, NaBH<sub>4</sub> (4.35 g, 114.78 mmol) was added porionwise, and the reaction mixture was warmed to room temperature and stirred for 12 h. After the reactant was consumed, the reaction was quenched by NH<sub>4</sub>Cl (30 mL). MeOH in the mixture was evaporated, and the residual mixture was extracted with ethyl acetate (50 mL × 3). The organic layers were washed by saturated NH<sub>4</sub>Cl solution (100 mL × 3) and brine (100 mL × 3); then, the organic phase was dried (MgSO<sub>4</sub>) and concentrated. The residue was purified by flash column chromatography (petroleum ether/ethyl acetate = 2:1) to give product <b>10</b> (2.18g, 40%) as a white solid. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.64 (s, 1H), 5.56 (s, 1H), 4.29 (d, <i>J</i> = 9.1 Hz, 1H), 3.71 (s, 3H), 3.37–3.26 (m, 2H), 2.47–2.42 (m, 2H), 2.13–2.08 (m, 1H), 1.84–1.81 (m, 2H), 1.41 (s, 9H).</div><div class="NLM_p last">Compound <b>10</b> (1.0 g, 3.5 mmol) was dissolved in 10 mL of DCM; then, the HCl (9 mL, 4 M in dioxane) was added. The reaction mixture was stirred at 20 °C for 12 h, and the mixture was concentrated in vacuo to get a white solid <b>11</b>, which could be used for the following step without purification.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Synthesis Procedure of Compound <b>13</b></h4><div class="NLM_p last">To a solution of Boc-<span class="smallcaps smallerCapital">l</span>-Phe-OH <b>12</b> (1.1 g, 3.5 mmol) in DCM (40 mL) was added HATU (1.9 g, 4.9 mmol) sequentially at −20 °C, and then the residue concentrated crude product <b>11</b> (0.77g 3.5 mmol) was added. After 30 min later, DIPEA (1.7 mL, 10.5 mmol) was added dropwise. Then, the reaction mixture was stirred at −20 °C for 12 h. The resulting mixture was washed by saturated ammonium chloride solution (100 mL × 3), saturated NaHCO<sub>3</sub> solution (100 mL × 3), and brine (100 mL × 3). The organic phase layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. The resulting mixture residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 40:1 v/v) to afford the pure product <b>13</b> (1.26 g, 83%) as a light solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.61 (d, <i>J</i> = 7.0 Hz, 1H), 7.21 (m, 5H), 6.82 (s, 1H), 5.26 (s, 1H), 4.50 (d, <i>J</i> = 6.8 Hz, 2H), 3.69 (s, 3H), 3.36–3.25 (m, 2H), 3.12 (m, 1H), 2.99 (dd, <i>J</i> = 13.4, 7.3 Hz, 1H), 2.34 (s, 2H), 2.19–2.10 (m, 1H), 1.81 (m, 2H), 1.35 (s, 9H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Synthesis Procedure of Compounds <b>16a</b>–<b>16p</b></h4><div class="NLM_p last">To a dry 100 mL flask in which <b>13</b> (1.5 g, 3.5 mmol) was dissolved with dry DCM was added 4 M HCl (9 mL, 35 mmol) slowly at 20 °C, and the resulting mixture was stirred at an ambient temperature for 12 h. The solvent was removed in vacuo, and the crude product <b>14</b> was directly used in the next step without further purification. Then, <i>trans</i>-3-phenylacrylic acid <b>15a</b> (0.49 g, 1.5 mmol) was dissolved in a dry 100 mL flask with CH<sub>2</sub>Cl<sub>2</sub>, HATU (0.68g, 1.8 mmol) was added sequentially at −20 °C, and then compound <b>14</b> (0.55g 1.5 mmol) was added. DIPEA (0.73 mL, 4.5 mmol) was added dropwise after 30 min. Then, the reaction mixture was stirred at −20 °C for 12 h, followed by washing with a saturated ammonium chloride solution (100 mL × 3), saturated NaHCO<sub>3</sub> solution (100 mL × 3), and brine (100 mL × 3). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated, and the residue was purified by column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/CH<sub>3</sub>OH, 30:1 v/v) to afford the pure product <b>16a</b> (0.55 g, 80%) as a light solid. <sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.45 (d, <i>J</i> = 7.4 Hz, 1H), 7.82 (d, <i>J</i> = 8.4 Hz, 1H), 7.54–7.47 (m, 3H), 7.37–7.31 (m, 5H), 7.26 (t, <i>J</i> = 7.6 Hz, 2H), 7.21–7.16 (m, 2H), 6.75 (d, <i>J</i> = 15.7 Hz, 1H), 5.01 (td, <i>J</i> = 8.4, 5.1 Hz, 1H), 4.53 (m, 1H), 3.69 (s, 3H), 3.33–3.24 (m, 3H), 3.04 (dd, <i>J</i> = 13.9, 8.5 Hz, 1H), 2.46 (dd, <i>J</i> = 9.8, 4.7 Hz, 1H), 2.31 (dd, <i>J</i> = 7.4, 4.8 Hz, 1H), 2.24–2.15 (m, 1H), 1.79 (m, 2H).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Synthesis Procedure of Compounds <b>17a</b>–<b>17p</b></h4><div class="NLM_p last">In a dry 100 mL flask was dissolved <b>16a</b> (0.92 g, 2.0 mmol) in dry THF, NaBH<sub>4</sub> (0.6 g, 16 mmol) was added slowly at 0 °C, and then, the reaction mixture was stirred at rt for 3 h. The completion of the reaction was confirmed by TLC; then the reaction was quenched and concentrated to get a crude residue. The residue was dissolved in DCM and washed with saturated ammonium chloride solution (50 mL × 3), saturated NaHCO<sub>3</sub> solution (50 mL × 3), and brine (50 mL × 3). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated, and the residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 20:1 v/v) to afford the pure product <b>17a</b> (0.77 g, 90%) as a light solid. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.97 (d, <i>J</i> = 8.8 Hz, 1H), 7.55–7.48 (m, 3H), 7.40–7.34 (m, 3H), 7.32–7.28 (m, 4H), 7.23 (td, <i>J</i> = 6.0, 3.2 Hz, 2H), 6.67 (d, <i>J</i> = 15.8 Hz, 1H), 4.74 (dd, <i>J</i> = 8.1, 6.7 Hz, 1H), 3.98–3.91 (m, 1H), 3.44 (dd, <i>J</i> = 11.0, 5.0 Hz, 1H), 3.34–3.22 (m, 4H), 3.17 (dd, <i>J</i> = 13.7, 6.6 Hz, 1H), 3.05 (dd, <i>J</i> = 13.7, 8.3 Hz, 1H), 2.46 (dd, <i>J</i> = 9.8, 2.5 Hz, 1H), 2.37–2.27 (m, 1H), 1.98–1.89 (m, 1H), 1.81–1.69 (m, 1H), 1.55 (m, 1H).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> General Synthesis Procedure of Compounds <b>1</b> and <b>18b</b>–<b>18p</b></h4><div class="NLM_p last">To a solution of the <b>17a</b> (0.87 g, 2.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> was added DMP (1.01 g, 2.4 mmol) slowly, and the reaction mixture was stirred at room temperature for 5 h. The completion of the reaction was confirmed by TLC, the reaction was quenched and concentrated, and the reaction was filtered and washed with saturated NaHCO<sub>3</sub> solution (50 mL × 3) and brine (50 mL × 3). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated, and the residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 20:1 v/v) to afford the pure product <b>1</b> (0.65 g, 76%) as a light solid. <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 7.57 (dt, <i>J</i> = 10.6, 8.4 Hz, 4H), 7.42–7.36 (m, 3H), 7.34–7.25 (m, 4H), 7.22 (d, <i>J</i> = 6.7 Hz, 1H), 6.84 (s, 1H), 6.80–6.72 (m, 1H), 4.90 (dd, <i>J</i> = 7.9, 5.7 Hz, 1H), 4.54–4.19 (m, 1H), 4.06–3.80 (m, 1H), 3.29–3.16 (m, 3H), 3.11 (m, 1H), 2.40–2.17 (m, 2H), 1.97 (dd, <i>J</i> = 14.1, 3.8 Hz, 1H), 1.82–1.48 (m, 2H).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> General Synthesis Procedure of Compounds <b>19a</b> and <b>19b</b></h4><div class="NLM_p last">To a solution of the <b>1</b> (0.86 g, 2.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> were added methyl-(triphenylphosphoranylidene) acetate (0.80 g, 2.4 mmol) and Et<sub>3</sub>N (0.56 mL, 4 mmol); then the reaction mixture was stirred at room temperature for 12 h. The completion of the reaction was confirmed by TLC; then the reaction was washed with saturated ammonium chloride solution (50 mL × 3), saturated NaHCO<sub>3</sub> solution (50 mL × 3), and brine (50 mL × 3). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated, and the residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 40:1 v/v) to afford the pure product <b>19a</b> (0.78 g, 78%) as a light solid.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Synthesis Procedure of Compound <b>21</b></h4><div class="NLM_p last">To a solution of Boc-<span class="smallcaps smallerCapital">l</span>-Val-OH <b>20</b> (0.76 g, 3.5 mmol) in DCM (40 mL) was added HATU (1.9 g, 4.9 mmol) sequentially at −20 °C, and then the residue concentrated crude product <b>14</b> (1.28g 3.5 mmol) was added. After 30 min later, DIPEA (1.7 mL, 10.5 mmol) was added dropwise. Then, the reaction mixture was stirred at −20 °C for 12 h. The resulting mixture was washed by saturated ammonium chloride solution (100 mL × 3), saturated NaHCO<sub>3</sub> solution (100 mL × 3), and brine (100 mL × 3). The organic phase layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. The resulting mixture residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 40:1 v/v) to afford the pure product <b>21</b> (1.58 g, 85%) as a light solid. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.67 (d, <i>J</i> = 7.7 Hz, 1H), 7.49 (d, <i>J</i> = 8.5 Hz, 1H), 7.24–7.14 (m, 6H), 7.01 (s, 1H), 5.11 (d, <i>J</i> = 9.1 Hz, 1H), 4.97–4.91 (m, 1H), 4.52 (t, <i>J</i> = 8.1 Hz, 1H), 3.84 (t, <i>J</i> = 8.1 Hz, 1H), 3.66 (s, 3H), 3.31 (m, 2H), 3.07 (d, <i>J</i> = 5.9 Hz, 2H), 2.39–2.27 (m, 1H), 2.18–2.06 (m, 2H), 1.99 (dd, <i>J</i> = 13.6, 6.8 Hz, 1H), 1.80–1.71 (m, 2H), 1.43 (s, 9H), 0.87 (d, <i>J</i> = 6.6 Hz, 3H), 0.82 (d, <i>J</i> = 6.6 Hz, 3H).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> General Synthesis Procedure of Compounds <b>24a</b>–<b>24e</b></h4><div class="NLM_p last">To a dry 100 mL flask in which <b>21</b> (1.86 g, 3.5 mmol) was dissolved in dry DCM was added 4 M HCl (9 mL, 35 mmol) slowly at 20 °C, and the resulting mixture was stirred at an ambient temperature for 12 h. The solvent was removed in vacuo, and the crude product <b>22</b> was directly used in next step without further purification. Then compound <b>22</b> was coupled with quinaldic acid <b>23d</b> to get the esters <b>24d</b> (0.96 g, 80%), and the synthesis procedure is similar to <b>17a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.60 (d, <i>J</i> = 9.5 Hz, 1H), 8.31 (d, <i>J</i> = 8.5 Hz, 1H), 8.25 (d, <i>J</i> = 8.5 Hz, 1H), 8.11 (d, <i>J</i> = 8.5 Hz, 1H), 7.97–7.86 (m, 2H), 7.81–7.75 (m, 1H), 7.72 (d, <i>J</i> = 8.4 Hz, 1H), 7.67–7.59 (m, 1H), 7.27 (s, 1H), 7.16 (d, <i>J</i> = 7.3 Hz, 2H), 7.02 (t, <i>J</i> = 7.6 Hz, 2H), 6.81 (t, <i>J</i> = 7.4 Hz, 1H), 5.00 (td, <i>J</i> = 8.3, 5.1 Hz, 1H), 4.64 (ddd, <i>J</i> = 11.6, 8.4, 3.2 Hz, 1H), 4.39 (dd, <i>J</i> = 9.2, 8.2 Hz, 1H), 3.69 (s, 3H), 3.40 (t, <i>J</i> = 8.6 Hz, 2H), 3.18 (dd, <i>J</i> = 13.8, 5.0 Hz, 1H), 3.01 (dd, <i>J</i> = 13.8, 8.0 Hz, 1H), 2.49–2.40 (m, 1H), 2.23 (m, 3H), 1.92–1.77 (m, 2H), 1.00 (d, <i>J</i> = 6.7 Hz, 3H), 0.95 (d, <i>J</i> = 6.7 Hz, 3H).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> General Synthesis Procedure of Compounds <b>25a</b>–<b>25e</b></h4><div class="NLM_p last">In a dry 100 mL flask was dissolved the <b>24d</b> (0.88 g, 1.5 mmol) in dry THF, NaBH<sub>4</sub> (0.45 g, 12 mmol) was added slowly at 0 °C, and then the reaction mixture was stirred at rt for 3 h. The completion of the reaction was confirmed by TLC; then the reaction was quenched and concentrated to get a crude residue. The residue was dissolved in DCM and washed with saturated ammonium chloride solution (50 mL × 3), saturated NaHCO<sub>3</sub> solution (50 mL × 3), and brine (50 mL × 3). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated, and the residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 20:1 v/v) to afford the pure product <b>25d</b> (0.71 g, 85%) as a light solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.60 (d, <i>J</i> = 8.3 Hz, 1H), 8.18 (d, <i>J</i> = 8.5 Hz, 1H), 8.09 (d, <i>J</i> = 8.5 Hz, 1H), 7.93 (t, <i>J</i> = 8.1 Hz, 2H), 7.84 (d, <i>J</i> = 7.7 Hz, 1H), 7.65 (t, <i>J</i> = 7.1 Hz, 1H), 7.62–7.52 (m, 2H), 7.19 (d, <i>J</i> = 7.3 Hz, 2H), 7.09 (t, <i>J</i> = 7.6 Hz, 2H), 7.02–6.87 (m, 2H), 4.91 (dd, <i>J</i> = 14.4, 8.4 Hz, 1H), 4.45 (dd, <i>J</i> = 8.0, 6.7 Hz, 1H), 4.10–3.98 (m, 1H), 3.57 (ddd, <i>J</i> = 33.4, 11.4, 4.6 Hz, 2H), 3.43–3.23 (m, 3H), 3.17 (dd, <i>J</i> = 13.7, 5.7 Hz, 1H), 3.02 (dd, <i>J</i> = 13.7, 8.4 Hz, 1H), 2.35 (m, 3H), 2.16–2.00 (m, 1H), 1.78 (dd, <i>J</i> = 11.5, 9.3 Hz, 1H), 1.57 (m, 1H), 0.97 (d, <i>J</i> = 6.7 Hz, 3H), 0.92 (d, <i>J</i> = 6.8 Hz, 3H).</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> General Synthesis Procedure of Compounds <b>26a</b>–<b>26e</b></h4><div class="NLM_p">To a solution of the <b>25d</b> (0.56 g, 1.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> was added DMP (0.51 g, 1.2 mmol) slowly, and the reaction mixture was stirred at room temperature. The completion of the reaction was confirmed by TLC; then, the reaction was quenched and concentrated, and the reaction was filtered and washed with saturated NaHCO<sub>3</sub> solution (50 mL × 3) and brine (50 mL × 3). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated, and the residue was purified by column chromatography (DCM/CH<sub>3</sub>OH, 20:1 v/v) to afford the pure product <b>26d</b> (0.42 g, 76%) as a light solid.</div><div id="sec4_2_10_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Methyl (<i>S</i>,<i>E</i>)-4-((<i>S</i>)-2-Cinnamamido-3-phenylpropanamido)-5-((<i>S</i>)-2-oxopyrrolidin-3-yl)pent-2-enoate (<b>19a</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.15 (d, <i>J</i> = 8.2 Hz, 1H), 7.99 (d, <i>J</i> = 8.1 Hz, 1H), 7.77–7.61 (m, 1H), 7.55 (s, 1H), 7.52 (dd, <i>J</i> = 6.7, 2.6 Hz, 2H), 7.39–7.34 (m, 3H), 7.31 (d, <i>J</i> = 7.2 Hz, 2H), 7.26 (t, <i>J</i> = 7.6 Hz, 3H), 7.21–7.18 (m, 1H), 7.15 (s, 1H), 6.86 (dd, <i>J</i> = 15.7, 5.3 Hz, 1H), 6.80 (d, <i>J</i> = 15.8 Hz, 1H), 5.83 (dd, <i>J</i> = 15.7, 1.6 Hz, 1H), 4.96 (dd, <i>J</i> = 14.8, 7.6 Hz, 1H), 4.75–4.68 (m, 1H), 3.69 (s, 3H), 3.26–3.22 (m, 1H), 3.11 (dd, <i>J</i> = 13.6, 7.6 Hz, 1H), 2.30–2.23 (m, 1H), 2.05–1.97 (m, 2H), 1.80–1.73 (m, 1H), 1.61 (td, <i>J</i> = 9.7, 4.9 Hz, 1H); <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub><i>6</i></sub>): δ 179.4, 171.4, 166.2, 165.4, 148.5, 140.0, 137.5, 135.2, 132.0, 131.9, 131.8, 129.4, 128.8, 128.7, 128.6, 128.3, 127.7, 126.6, 121.6, 120.1, 55.2, 50.8, 48.5, 40.0, 38.0, 35.2. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>28</sub>H<sub>30</sub>N<sub>3</sub>O<sub>5</sub>, 488.2191; found, 488.2182. Purity: 98.6%.</div></div><div id="sec4_2_10_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Benzyl (<i>S</i>,<i>E</i>)-4-((<i>S</i>)-2-Cinnamamido-3-phenylpropanamido)-5-((<i>S</i>)-2-oxopyrrolidin-3-yl)pent-2-enoate (<b>19b</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.90 (s, 1H), 7.55 (dd, <i>J</i> = 15.6, 2.7 Hz, 1H), 7.47–7.41 (m, 2H), 7.40–7.29 (m, 9H), 7.19 (d, <i>J</i> = 4.1 Hz, 4H), 7.10 (m, 1H), 6.92 (s, 1H), 6.72 (dd, <i>J</i> = 15.6, 5.4 Hz, 2H), 6.47 (d, <i>J</i> = 15.6 Hz, 1H), 5.76 (d, <i>J</i> = 15.7 Hz, 1H), 5.15 (s, 2H), 5.08 (d, <i>J</i> = 6.7 Hz, 1H), 4.49 (d, <i>J</i> = 5.3 Hz, 1H), 3.25 (m, 2H), 3.19–3.13 (m, 1H), 3.07 (dd, <i>J</i> = 13.5, 7.2 Hz, 1H), 2.27 (s, 2H), 1.95 (t, <i>J</i> = 10.8 Hz, 1H), 1.73–1.62 (m, 1H), 1.54–1.47 (m, 1H), 1.26 (d, <i>J</i> = 1.3 Hz, 1H). <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 179.7, 170.9, 165.6, 147.1, 141.2, 135.9, 135.5, 134.2, 129.4, 129.1, 128.4, 128.2, 128.1, 127.5, 126.6, 120.5, 119.9, 66.0, 54.1, 48.8, 48.7, 40.3, 38.7, 37.9, 34.4, 28.0. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>34</sub>H<sub>34</sub>N<sub>3</sub>O<sub>5</sub>, 564.2504; found, 564.2508. Purity: 98.1%.</div></div><div id="sec4_2_10_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 5-Methyl-<i>N</i>-((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)isoxazole-3-carboxamide (<b>18b</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.14 (d, <i>J</i> = 8.4 Hz, 1H), 9.41 (s, 1H), 8.28–8.23 (m, 1H), 7.78 (dd, <i>J</i> = 12.5, 8.8 Hz, 1H), 7.58 (d, <i>J</i> = 1.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.33 (m, 2H), 7.23 (d, <i>J</i> = 7.7 Hz, 2H), 7.16 (dd, <i>J</i> = 6.3, 1.9 Hz, 4H), 5.44 (dd, <i>J</i> = 70.6, 8.0 Hz, 1H), 5.06 (dd, <i>J</i> = 7.1, 4.5 Hz, 1H), 4.51 (m, 1H), 4.13–3.99 (m, 1H), 3.32 (d, <i>J</i> = 2.0 Hz, 1H), 3.20 (m, 2H), 2.44–2.34 (m, 1H), 2.33–2.25 (m, 1H), 1.79–1.55 (m, 2H).<sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.6, 179.0, 171.0, 170.8, 158.3, 136.8, 129.0, 127.9, 126.2, 100.7, 57.0, 54.0, 39.6, 37.4, 29.3, 27.6, 10.8. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>21</sub>H<sub>23</sub>N<sub>4</sub>O<sub>5</sub>, 411.1674; found, 411.1682. Purity: 95.0%.</div></div><div id="sec4_2_10_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 5-Fluoro-<i>N</i>-((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)picolinamide (<b>18c</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.42 (s, 1H), 8.87 (d, <i>J</i> = 12.0 Hz, 1H), 8.75 (d, <i>J</i> = 6.0 Hz, 1H), 8.63 (d, <i>J</i> = 2.7 Hz, 1H), 8.24 (d, <i>J</i> = 8.1 Hz, 1H), 7.98–7.94 (m, 1H), 7.37 (d, <i>J</i> = 7.3 Hz, 2H), 7.30 (dd, <i>J</i> = 10.4, 4.7 Hz, 2H), 7.22 (m, 1H), 5.01 (m, 1H), 4.32–4.27 (m, 1H), 3.38–3.34 (m, 1H), 3.31–3.27 (m, 1H), 3.19 (dd, <i>J</i> = 13.9, 8.9 Hz, 1H), 2.51–2.40 (m, 1H), 2.39–2.33 (m, 1H), 1.97–1.88 (m, 1H), 1.82 (m, 1H), 1.39 (s, 1H), 1.31 (d, <i>J</i> = 2.5 Hz, 2H). <sup>13</sup>C NMR (150 MHz, acetone-<i>d</i><sub>6</sub>): δ 200.1, 179.3, 171.7, 163.9, 160.0, 158.3, 144.7, 140.3, 137.7, 129.4, 128.3, 126.6, 121.7, 58.0, 55.3, 40.1, 38.2, 37.6, 30.8. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>22</sub>H<sub>22</sub>FN<sub>4</sub>O<sub>4</sub>, 425.1631; found, 425.1631. Purity: 95.7%.</div></div><div id="sec4_2_10_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl) propan-2-yl)amino)-3-phenylpropan-2-yl)benzo[<i>d</i>][1,3]dioxole-5-carboxamide. (<b>18d</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.25 (s, 1H), 8.47 (d, <i>J</i> = 6.3 Hz, 1H), 7.32–7.20 (m, 8H), 6.89 (s, 1H), 6.78–6.73 (m, 1H), 5.97 (s, 2H), 5.16–4.92 (m, 1H), 4.44–4.24 (m, 1H), 3.31–3.21 (m, 3H), 2.33 (m, 2H), 2.03–1.45 (m, 3H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ 199.9, 180.0, 172.4, 166.5, 150.5, 147.8, 136.6, 129.5, 128.5, 127.9, 126.9, 122.1, 107.9, 107.7, 101.7, 57.7, 54.6, 40.6, 38.8, 38.0, 29.7, 28.4. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>24</sub>H<sub>24</sub>N<sub>3</sub>O<sub>6</sub>, 450.1671; found, 450.1663. Purity: 96.7%.</div></div><div id="sec4_2_10_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)benzofuran-5-carboxamide (<b>18e</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.43 (s, 1H), 8.59 (d, <i>J</i> = 6.8 Hz, 1H), 8.18 (dd, <i>J</i> = 15.6, 1.8 Hz, 1H), 8.08–7.99 (m, 1H), 7.92–7.82 (m, 2H), 7.53 (t, <i>J</i> = 8.1 Hz, 1H), 7.38 (d, <i>J</i> = 7.3 Hz, 2H), 7.28 (t, <i>J</i> = 7.5 Hz, 2H), 7.19 (t, <i>J</i> = 7.3 Hz, 1H), 7.13 (s, 1H), 6.97–6.90 (m, 1H), 5.07 (m, 1H), 4.37 (m, 1H), 3.37–3.33 (m, 1H), 3.29–3.20 (m, 3H), 2.52–2.24 (m, 2H), 2.05–1.98 (m, 1H), 1.87–1.62 (m, 2H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 200.1, 179.3, 172.3, 166.9, 156.5, 146.7, 137.9, 129.5, 128.3, 127.4, 126.5, 124.1, 121.1, 110.8, 107.1, 57.6, 55.3, 40.0, 37.9, 37.8. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>24</sub>N<sub>3</sub>O<sub>5</sub>, 446.1721; found, 446.1715. Purity: 95.2%.</div></div><div id="sec4_2_10_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (<b>18f</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.22 (s, 1H), 8.36 (d, <i>J</i> = 6.3 Hz, 1H), 7.32 (d, <i>J</i> = 2.0 Hz, 1H), 7.23 (s, 5H), 7.02 (d, <i>J</i> = 8.1 Hz, 1H), 6.83 (d, <i>J</i> = 8.4 Hz, 1H), 6.55 (s, 1H), 5.07 (m, 1H), 4.27–4.21 (m, 6H), 3.31–3.24 (m, 2H), 3.20 (dd, <i>J</i> = 14.4, 6.7 Hz, 2H), 2.35–2.28 (m, 2H), 1.90 (dd, <i>J</i> = 10.2, 6.4 Hz, 1H), 1.80 (m, 2H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ 200.0, 180.0, 172.3, 166.4, 146.7, 143.3, 136.5, 129.5, 128.5, 127.0, 120.7, 117.2, 116.7, 64.5, 64.2, 57.7, 54.5, 40.6, 38.8, 38.0, 29.5, 28.5. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>26</sub>N<sub>3</sub>O<sub>6</sub>, 464.1827; found, 464.1824. Purity: 97.5%.</div></div><div id="sec4_2_10_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-5-carboxamide (<b>18g</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.14 (d, <i>J</i> = 8.8 Hz, 1H), 9.41 (s, 1H), 8.28–8.23 (m, 1H), 7.78 (m, 1H), 7.58 (d, <i>J</i> = 1.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.33 (m, 2H), 7.23 (d, <i>J</i> = 7.7 Hz, 2H), 7.16 (dd, <i>J</i> = 6.3, 1.9 Hz, 4H), 5.44 m, 1H), 5.06 (dd, <i>J</i> = 7.1, 4.5 Hz, 1H), 4.51 (m, 1H), 4.13–3.99 (m, 1H), 3.32 (d, <i>J</i> = 2.0 Hz, 1H), 3.20 (m, 2H), 2.44–2.34 (m, 1H), 2.33–2.25 (m, 1H), 1.79–1.55 (m, 2H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.8, 178.9, 171.4, 163.7, 143.5, 142.2, 136.8, 129.2, 127.9, 126.3, 122.8, 121.6, 120.3, 97.9, 64.5, 63.3, 57.1, 54.5, 53.4, 39.6, 37.4, 27.7. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>26</sub>N<sub>3</sub>O<sub>6</sub>, 464.1827; found, 464.183. Purity: 97.6%.</div></div><div id="sec4_2_10_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> <i>N</i>-((<i>R</i>)-1-Oxo-1-(((<i>R</i>)-1-oxo-3-((<i>R</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)-1<i>H</i>-indole-5-carboxamide (<b>18h</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 10.53 (s, 1H), 9.39 (s, 1H), 8.19–8.16 (m, 1H), 7.73 (d, <i>J</i> = 8.0 Hz, 1H), 7.66 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 7.43 (d, <i>J</i> = 8.5 Hz, 1H), 7.40–7.38 (m, 1H), 7.37–7.33 (m, 2H), 7.26 (m, 2H), 7.19–7.15 (m, 1H), 6.96 (s, 1H), 6.54 (m, 1H), 5.01 (dt, <i>J</i> = 8.2, 4.0 Hz, 1H), 4.31–4.26 (m, 1H), 3.31 (dd, <i>J</i> = 8.2, 5.6 Hz, 1H), 3.26–3.18 (m, 3H), 2.45–2.36 (m, 1H), 2.32–2.21 (m, 1H), 1.98–1.95 (m, 1H), 1.81–1.68 (m, 2H). 13C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.7, 178.6, 171.9, 167.3, 137.6, 137.5, 129.0, 127.8, 127.2, 126.0, 125.8, 125.1, 120.4, 119.9, 110.4, 102.1, 57.1, 54.6, 39.5, 37.4. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>, 445.1881; found, 445.1873. Purity: 95.1%.</div></div><div id="sec4_2_10_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 6-Bromo-<i>N</i>-((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)imidazo[1,2-a]pyridine-2-carboxamide (<b>18i</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.40 (s, 1H), 8.79 (s, 1H), 8.53 (d, <i>J</i> = 6.2 Hz, 1H), 8.27 (d, <i>J</i> = 4.9 Hz, 1H), 8.04 (d, <i>J</i> = 7.3 Hz, 1H), 7.58–7.52 (m, 1H), 7.47–7.40 (m, 1H), 7.32 (t, <i>J</i> = 6.4 Hz, 2H), 7.28–7.22 (m, 2H), 7.18 (t, <i>J</i> = 7.2 Hz, 1H), 6.91 (s, 1H), 5.03–4.93 (m, 1H), 4.44 (m, 1H), 3.32 (t, <i>J</i> = 6.9 Hz, 1H), 3.27–3.23 (m, 1H), 2.45–2.26 (m, 2H), 1.99–1.94 (m, 1H), 1.80–1.75 (m, 1H), 1.37 (s, 1H), 1.29 (d, <i>J</i> = 3.3 Hz, 2H).<sup>13</sup>C NMR (150 MHz, acetone-<i>d</i><sub>6</sub>): δ 200.0, 178.9, 171.4, 161.5, 142.8, 137.3, 129.5, 129.3, 128.3, 128.2, 127.6, 126.6, 118.7, 114.8, 107.2, 98.7, 57.6, 54.0, 39.9, 38.4, 37.8, 30.8. HRMS (ESI) <i>m</i>/<i>z</i>: [M + H] <sup>+</sup> calcd for C<sub>24</sub>H<sub>25</sub>BrN<sub>5</sub>O<sub>4</sub>, 526.1084; found, 526.1093. Purity: 98.8%.</div></div><div id="sec4_2_10_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)benzofuran-2-carboxamide (<b>18j</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.43 (s, 1H), 8.64 (d, <i>J</i> = 6.8 Hz, 1H), 8.11 (d, <i>J</i> = 8.0 Hz, 1H), 7.71 (d, <i>J</i> = 7.5 Hz, 1H), 7.56–7.50 (m, 1H), 7.49–7.41 (m, 2H), 7.37 (dd, <i>J</i> = 9.2, 2.3 Hz, 2H), 7.32–7.22 (m, 3H), 7.19 (d, <i>J</i> = 7.5 Hz, 1H), 7.12 (s, 1H), 5.10–5.01 (m, 1H), 4.58–4.32 (m, 1H), 3.38 (m, 1H), 3.25 (dd, <i>J</i> = 9.0, 2.5 Hz, 3H), 2.53–2.27 (m, 2H), 2.04 (m, 1H), 1.86–1.62 (m, 2H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.6, 178.8, 171.1, 157.8, 154.3, 148.5, 137.0, 129.0, 127.9, 127.1, 126.4, 126.1, 123.2, 122.2, 111.3, 109.6, 97.9, 57.1, 54.0, 39.6, 37.4. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>24</sub>N<sub>3</sub>O<sub>5</sub>, 446.1721; found, 446.172. Purity: 95.6%.</div></div><div id="sec4_2_10_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)quinoxaline-2-carboxamide (<b>18k</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.47 (s, 1H), 8.69 (d, <i>J</i> = 8.0 Hz, 1H), 8.15 (m, 2H), 7.95 (m, 2H), 7.80 (d, <i>J</i> = 8.0 Hz, 1H), 7.34 (d, <i>J</i> = 7.3 Hz, 2H), 7.25 (t, <i>J</i> = 7.3 Hz, 2H), 7.18 (t, <i>J</i> = 7.3 Hz, 1H), 6.83 (s, 1H), 5.03–4.95 (m, 1H), 3.99 (dd, <i>J</i> = 7.3, 4.3 Hz, 1H), 3.50 (m, 2H), 3.31–3.20 (m, 3H), 2.39–2.28 (m, 2H), 1.95–1.87 (m, 1H), 1.78–1.68 (m, 1H), 1.62–1.51 (m, 1H). <sup>13</sup>C NMR (151 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.5, 179.0, 170.7, 162.3, 143.1, 139.7, 136.7, 131.3, 130.6, 129.2, 129.1, 128.9, 127.9, 1277, 126.2, 97.9, 57.4, 53.9, 53.6, 39.5, 37.6. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>24</sub>N<sub>5</sub>O<sub>4</sub>, 458.1834; found, 458.1823. Purity: 98.3%.</div></div><div id="sec4_2_10_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 3-Methyl-<i>N</i>-((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)quinoxaline-2-carboxamide (<b>18l</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.30 (s, 1H), 8.63 (d, <i>J</i> = 8.5 Hz, 1H), 8.33 (d, <i>J</i> = 6.2 Hz, 1H), 8.06 (d, <i>J</i> = 7.5 Hz, 1H), 8.02 (d, <i>J</i> = 8.5 Hz, 1H), 7.80 (m, 1H), 7.75 (dd, <i>J</i> = 11.0, 4.0 Hz, 1H), 7.39–7.27 (m, 5H), 7.25 (d, <i>J</i> = 7.1 Hz, 1H), 6.56 (s, 1H), 5.12 (q, <i>J</i> = 7.1 Hz, 1H), 4.32 (m, 1H), 3.31 (dd, <i>J</i> = 12.8, 6.2 Hz, 3H), 3.02 (s, 3H), 2.40–2.30 (m, 2H), 1.99–1.93 (m, 1H), 1.88–1.75 (m, 2H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ 199.8, 179.9, 171.8, 164.5, 153.9, 142.9, 139.1, 136.4, 131.7, 129.7, 129.6, 129.4, 128.6, 128.4, 127.1, 57.7, 54.5, 40.5, 39.1, 37.9, 29.7, 28.4, 24.5. HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>28</sub>N<sub>5</sub>O<sub>4</sub>, 474.2136; found, 474.215. Purity: 98.8%.</div></div><div id="sec4_2_10_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)quinoline-2-carboxamide (<b>18m</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.22 (s, 1H), 8.84 (d, <i>J</i> = 8.5 Hz, 1H), 8.28 (d, <i>J</i> = 8.5 Hz, 1H), 8.22 (dd, <i>J</i> = 10.1, 6.0 Hz, 1H), 8.11 (m, 2H), 7.86 (d, <i>J</i> = 8.1 Hz, 1H), 7.77 (dd, <i>J</i> = 11.2, 4.1 Hz, 1H), 7.62 (m, 1H), 7.36–7.27 (m, 4H), 7.24 (t, J = 7.2 Hz, 1H), 5.08 (m, 1H), 4.33–4.28 (m, 1H), 3.35–3.29 (m, 2H), 3.25 (t, <i>J</i> = 6.8 Hz, 2H), 2.38–2.30 (m, 2H), 1.93–1.84 (m, 2H), 1.79–1.71 (m, 2H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>): δ 199.8, 179.8, 171.7, 164.4, 149.1, 146.6, 137.4, 136.5, 129.6, 129.4, 128.6, 127.1, 118.7, 57.7, 54.7, 40.5, 39.0, 37.8, 29.6, 28.6. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>26</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>, 457.1881; found, 457.1888. Purity: 95.2%.</div></div><div id="sec4_2_10_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 7-Bromo-<i>N</i>-((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl) amino)-3-phenylpropan-2-yl)quinoline-2-carboxamide (<b>18n</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.14 (d, <i>J</i> = 8.4 Hz, 1H), 9.41 (s, 1H), 8.28–8.23 (m, 1H), 7.78 (dd, <i>J</i> = 12.5, 8.8 Hz, 1H), 7.58 (d, <i>J</i> = 1.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.33 (m, 2H), 7.23 (d, <i>J</i> = 7.7 Hz, 2H), 7.16 (dd, <i>J</i> = 6.3, 1.9 Hz, 4H), 5.44 (m, 1H), 5.06 (dd, <i>J</i> = 7.1, 4.5 Hz, 1H), 4.51 (m, 1H), 4.13–3.99 (m, 1H), 3.32 (d, <i>J</i> = 2.0 Hz, 1H), 3.20 (m, 2H), 2.44–2.34 (m, 1H), 2.33–2.25 (m, 1H), 1.79–1.55 (m, 2H). <sup>13</sup>C NMR (150 MHz, acetone-<i>d</i><sub>6</sub>): δ 200.0, 178.9, 171.3, 163.3, 150.7, 147.0, 138.0, 137.2, 131.6, 131.3, 129.8, 129.7, 129.6, 128.4, 128.1, 126.7, 123.8, 119.1, 57.8, 54.6, 54.1, 39.9, 38.3, 37.9. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>26</sub>H<sub>24</sub>BrN<sub>4</sub>O<sub>4</sub>, 535.0986; found, 535.0992. Purity: 95.0%.</div></div><div id="sec4_2_10_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 6-Chloro-<i>N</i>-((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)ami no)-3-phenylpropan-2-yl)-2<i>H</i>-chromene-3-carboxamide (<b>18o</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.24 (s, 1H), 8.55 (d, <i>J</i> = 6.0 Hz, 1H), 7.29–7.26 (m, 2H), 7.22 (m, 5H), 7.10 (m, 1H), 6.99 (d, <i>J</i> = 2.6 Hz, 1H), 6.95 (s, 1H), 6.72 (d, <i>J</i> = 8.6 Hz, 1H), 6.58 (s, 1H), 5.01 (dd, <i>J</i> = 14.9, 6.9 Hz, 1H), 4.91–4.85 (m, 2H), 4.25–4.20 (m, 1H), 3.32–3.26 (m, 2H), 3.18 (m, 2H), 2.34–2.29 (m, 2H), 1.94–1.87 (m, 1H), 1.82–1.77 (m, 1H). <sup>13</sup>C NMR (150 MHz, CDCl<sub>3</sub>): δ 201.6, 181.9, 174.3, 166.5, 155.1, 138.2, 132.7, 131.4, 130.5, 129.6, 129.1, 129.0, 128.9, 128.3, 124.2, 119.2, 66.8, 59.9, 56.1, 42.6, 40.6, 40.1, 31.4, 30.5. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>26</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>5</sub>, 494.1488; found, 494.1481. Purity: 96.2%.</div></div><div id="sec4_2_10_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> <i>N</i>-((<i>S</i>)-1-Oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)-1<i>H</i>-indole-2-carboxamide (<b>18p</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, acetone-<i>d</i><sub>6</sub>): δ 10.52 (s, 1H), 9.40 (s, 1H), 8.16 (d, <i>J</i> = 13.6 Hz, 1H), 7.73–7.62 (m, 2H), 7.45–7.40 (m, 1H), 7.39 (d, <i>J</i> = 2.4 Hz, 1H), 7.35 (t, <i>J</i> = 7.6 Hz, 2H), 7.28–7.21 (m, 2H), 7.18 (t, <i>J</i> = 6.6 Hz, 1H), 6.94–6.78 (m, 1H), 6.54 (s, 1H), 5.45–4.98 (m, 1H), 4.97–4.39 (m, 1H), 4.34–3.82 (m, 1H), 3.26–3.16 (m, 3H), 2.45–2.22 (m, 2H), 2.00–1.86 (m, 1H), 1.81–1.43 (m, 2H).. <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 200.0, 179.6, 172.1, 161.6, 137.6, 137.0, 131.0, 129.5, 129.4, 128.3, 128.2, 127.7, 126.5, 123.9, 121.7, 120.0, 112.3, 103.5, 57.6, 54.9, 40.1, 37.9, 37.8. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>25</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>, 445.1881; found, 445.1881. Purity: 97.0%.</div></div><div id="sec4_2_10_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 5-Methyl-<i>N</i>-((<i>S</i>)-3-methyl-1-oxo-1-(((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)amino)butan-2-yl)isoxazole-3-carboxamide (<b>26a</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.35 (s, 1H), 8.30 (d, <i>J</i> = 6.8 Hz, 1H), 7.81–7.69 (m, 1H), 7.58 (d, <i>J</i> = 8.4 Hz, 1H), 7.31–7.22 (m, 4H), 7.17 (t, <i>J</i> = 7.1 Hz, 1H), 6.94 (s, 1H), 6.52 (d, <i>J</i> = 0.7 Hz, 1H), 4.79 (m, 1H), 4.54–4.43 (m, 1H), 4.27 (m, 1H), 3.32–3.18 (m, 3H), 3.04 (dd, <i>J</i> = 13.8, 8.1 Hz, 1H), 2.44–2.28 (m, 2H), 2.22 (dd, <i>J</i> = 13.4, 6.7 Hz, 1H), 1.93 (d, <i>J</i> = 5.6 Hz, 1H), 1.81–1.72 (m, 2H), 1.31 (s, 2H), 0.93 (dd, <i>J</i> = 10.3, 6.8 Hz, 6H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.4, 178.5, 171.1, 171.0, 169.8, 158.2, 136.9, 128.9, 127.7, 126.0, 100.7, 57.7, 56.9, 54.2, 39.4, 37.23,37.2, 29.1, 27.7, 18.3, 16.9, 10.7. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>26</sub>H<sub>33</sub>N<sub>5</sub>O<sub>6</sub>, 510.2358; found, 510.2352. Purity: 96.7%.</div></div><div id="sec4_2_10_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> <i>N</i>-((<i>S</i>)-3-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)amino)butan-2-yl)-1<i>H</i>-indole-2-carboxamide (<b>26b</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 11.20 (s, 1H), 9.46 (s, 1H), 8.22 (d, <i>J</i> = 7.4 Hz, 1H), 7.96–7.88 (m, 1H), 7.75 (dd, <i>J</i> = 11.9, 4.0 Hz, 1H), 7.62 (m, 2H), 7.33–7.29 (m, 1H), 7.26–7.20 (m, 3H), 7.19–7.06 (m, 4H), 7.03 (dd, <i>J</i> = 16.8, 9.5 Hz, 1H), 4.88–4.80 (m, 1H), 4.54–4.32 (m, 2H), 3.38 (dd, <i>J</i> = 19.0, 10.1 Hz, 1H), 3.32–3.24 (m, 2H), 3.00 (m, 1H), 2.43 (m, 1H), 2.32 (m, 2H), 2.23 (tt, <i>J</i> = 9.0, 4.5 Hz, 1H), 1.87–1.81 (m, 1H), 1.31 (d, <i>J</i> = 1.7 Hz, 1H), 0.96 (dd, <i>J</i> = 10.7, 6.9 Hz, 6H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.8, 179.2, 171.5, 170.2, 162.5, 137.2, 130.2, 128.6, 127.7, 125.8, 123.4, 121.1, 119.6, 112.2, 103.2, 60.0, 56.7, 54.2, 39.5, 37.3, 36.5, 30.3, 29.5, 29.1, 27.0, 18.1, 17.5. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>30</sub>H<sub>34</sub>N<sub>5</sub>O<sub>5</sub>, 544.2565; found, 544.2568. Purity: 97.6%.</div></div><div id="sec4_2_10_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> <i>N</i>-((<i>S</i>)-3-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)amino)butan-2-yl)benzofuran-2-carboxamide (<b>26c</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.37 (s, 1H), 8.36 (d, <i>J</i> = 7.1 Hz, 1H), 8.02 (d, <i>J</i> = 8.1 Hz, 1H), 7.77 (d, <i>J</i> = 7.8 Hz, 1H), 7.60 (t, <i>J</i> = 6.2 Hz, 2H), 7.48 (dd, <i>J</i> = 8.4, 7.5 Hz, 1H), 7.34 (t, <i>J</i> = 7.5 Hz, 1H), 7.30–7.26 (m, 2H), 7.21–7.03 (m, 4H), 4.86 (m, 1H), 4.63–4.55 (m, 1H), 4.51–4.30 (m, 1H), 4.07 (d, <i>J</i> = 2.6 Hz, 1H), 3.30–3.16 (m, 3H), 3.05 (dd, <i>J</i> = 13.9, 8.2 Hz, 1H), 2.43–2.18 (m, 3H), 2.05–1.97 (m, 1H), 1.84–1.75 (m, 1H), 1.60 (m, 1H), 0.98 (dd, <i>J</i> = 6.6, 2.2 Hz, 6H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.5, 179.5, 178.8, 171.2, 170.2, 158.0, 154.3, 148.5, 136.9, 128.9, 127.7, 126.5, 125.9, 123.2, 122.2, 111.3, 109.8, 58.0, 56.8, 54.3, 53.6, 39.5, 37.3, 30.5, 18.4, 17.4. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>30</sub>H<sub>33</sub>N<sub>4</sub>O<sub>6</sub>, 545.2406; found, 545.2407. Purity: 98.7%.</div></div><div id="sec4_2_10_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> <i>N</i>-((<i>S</i>)-3-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)amino)butan-2-yl)quinoline-2-carboxamide (<b>26d</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.40 (s, 1H), 8.74 (d, <i>J</i> = 8.5 Hz, 1H), 8.52 (d, <i>J</i> = 8.5 Hz, 1H), 8.27 (d, <i>J</i> = 8.5 Hz, 1H), 8.14 (d, <i>J</i> = 8.5 Hz, 1H), 8.06 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (d, <i>J</i> = 8.0 Hz, 1H), 7.88–7.82 (m, 1H), 7.72 (dd, <i>J</i> = 11.2, 4.0 Hz, 1H), 7.29 (d, <i>J</i> = 7.4 Hz, 2H), 7.19 (t, <i>J</i> = 7.6 Hz, 2H), 7.09 (d, <i>J</i> = 7.5 Hz, 1H), 4.85 (dd, <i>J</i> = 11.2, 5.0 Hz, 1H), 4.62–4.55 (m, 1H), 4.40–4.30 (m, 1H), 3.33–3.20 (m, 3H), 3.06–3.00 (m, 1H), 2.50–2.41 (m, 1H), 2.38–2.27 (m, 2H), 2.06–2.00 (m, 1H), 1.80 (dt, <i>J</i> = 12.9, 9.6 Hz, 2H), 1.30 (d, <i>J</i> = 5.1 Hz, 1H), 0.99 (dd, <i>J</i> = 19.0, 6.8 Hz, 6H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.5, 178.7, 171.2, 170.3, 163.7, 149.2, 145.9, 137.4, 137.0, 129.9, 129.1, 128.8, 127.7, 127.6, 127.5, 125.9, 118.2, 58.0, 56.8, 54.3, 39.5, 37.3, 37.2, 30.8, 18.5, 17.0. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>31</sub>H<sub>35</sub>N<sub>5</sub>O<sub>5</sub>, 556.2565; found, 556.2562. Purity: 95.4%.</div></div><div id="sec4_2_10_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> <i>N</i>-((<i>S</i>)-3-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-3-phenylpropan-2-yl)amino)butan-2-yl)-2,3-dihydrobenzo[<i>b</i>][1,4]dioxine-6-carboxamide (<b>26e</b>)</h5><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.35 (s, 1H), 8.32 (d, <i>J</i> = 7.1 Hz, 1H), 7.88 (d, <i>J</i> = 8.1 Hz, 1H), 7.61 (s, 1H), 7.46 (dd, <i>J</i> = 6.5, 2.1 Hz, 2H), 7.26 (d, <i>J</i> = 7.1 Hz, 2H), 7.20 (t, <i>J</i> = 7.4 Hz, 2H), 7.14 (s, 1H), 6.89 (d, <i>J</i> = 9.0 Hz, 1H), 4.81 (m 1H), 4.45 (dd, <i>J</i> = 15.3, 7.4 Hz, 1H), 4.36–4.27 (m, 5H), 3.30–3.23 (m, 2H), 3.18–3.03 (m, 2H), 2.44–2.28 (m, 2H), 2.21 (d, <i>J</i> = 7.0 Hz, 1H), 1.98–1.92 (m, 1H), 1.77 (dd, <i>J</i> = 9.0, 3.1 Hz, 2H), 1.31 (s, 1H), 0.95 (t, <i>J</i> = 7.3 Hz, 6H). <sup>13</sup>C NMR (125 MHz, acetone-<i>d</i><sub>6</sub>): δ 199.5, 171.2, 170.8, 165.9, 146.2, 142.8, 136.9, 128.9, 127.7, 127.0, 125.9, 120.4, 116.3, 64.1, 63.7, 59.0, 54.1, 37.3, 37.2, 29.2, 18.5, 17.6. HRMS (ESI) <i>m</i>/<i>z</i>: [M – H]<sup>−</sup> calcd for C<sub>30</sub>H<sub>35</sub>N<sub>4</sub>O<sub>7</sub>, 563.2511; found, 563.2501. Purity: 95.1%.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Materials and Methods</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Protein Expression and Purification</h4><div class="NLM_p last">The full-length gene encoding the EV71 3C<sup>pro</sup> and the SARS-CoV-2 3CL<sup>pro</sup> with an <i>N</i>-terminal 6 × His-SUMO2 fusion tag was cloned into the pET-15b vector. The resulting plasmids were transformed into BL21 (DE3) cells for protein expression. The expressed proteins were purified by a Ni-NTA column (GE) and transformed into the cleavage buffer (25 mM Tris, pH 7.5, 300 mM NaCl, 2 mM DTT) containing SUMO Specific Peptidase 2 (SENP2) for removing the 6 × His-SUMO2 fusion tag. The resulting protein samples were further purified by Q-Sepharose (GE Healthcare) and Superdex200 (GE Healthcare). The eluted EV71 3C<sup>pro</sup> and SARS-CoV-2 3CL<sup>pro</sup> were stored in a solution containing 25 mM Tris (pH 8.0), 500 mM NaCl, and 2 mM DTT, and in a solution containing 10 mM Tris (pH 7.5), respectively.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Protein Crystallization and Structure Determination</h4><div class="NLM_p last">The purified EV71 3C<sup>pro</sup> was concentrated to 10 mg/mL and incubated 1 h with 2 mM <b>18p</b> before crystallization condition screening. Crystallization was performed at 20 °C using a hanging drop vapor-diffusion method, by mixing equal volumes (1:1 μL) of the protein and crystallization solution. Crystals were finally yielded in a solution containing 0.1 M MES monohydrate (pH 6.0), 20% 2-propanol, and 20% polyethylene glycol monomethyl ether 2000. Then, crystals were flash-frozen in liquid nitrogen in the presence of the reservoir solution supplemented with 20% glycerol. X-ray diffraction data were collected at beamline BL18U1 at the Shanghai Synchrotron Radiation Facility.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The data were processed with HKL3000 software packages.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The complex structure was solved by molecular replacement using the program PHASER<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> with a search model of PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GHT">4GHT</a>. The model was built using Coot<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and refined with a simulated-annealing protocol implemented in the program PHENIX.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The refined structure was deposited to the Protein Data Bank with an accession code, 7DNC.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Inhibition Assays of the EV71 3C<sup>pro</sup> and the SARS-CoV-2 3CL<sup>pro</sup></h4><div class="NLM_p">A fluorescence resonance energy transfer (FRET) protease assay was applied to measure the inhibitory activity of compounds against the EV71 3C<sup>pro</sup>. The fluorogenic substrate Dacyl-KTSAVLQSGFRKME-Edans was synthesized by GenScript (Nanjing, China). The assay was performed in a total volume of 120 μL. The recombinant EV71 3C<sup>pro</sup> at a final concentration of 5 μM was mixed with serial dilutions of each compound in 80 μL of assay buffer (25 mM Tris, pH 8.0, 150 mM NaCl, and 10% glycerol) and incubated for 10 min. The reaction was initiated by adding 40 μL of a fluorogenic substrate at a final concentration of 25 μM. The reaction solution was then incubated at 30 °C for 3 h. After that, the fluorescence signal at 340 nm (excitation)/490 nm (emission) was measured immediately with a Bio-Tek Synergy4 plate reader.</div><div class="NLM_p last">The inhibition assay of the SARS-CoV-2 3CL<sup>pro</sup> has been described previously.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In brief, the recombinant SARS-CoV-2 3CL<sup>pro</sup> at a concentration of 30 nM was mixed with serial dilutions of each compound in 80 μL of assay buffer (50 mM Tris–HCl, pH 7.3, 1 mM EDTA) and incubated for 10 min. The reaction was initiated by adding 40 μL of a fluorogenic substrate (MCA-AVLQSGFR-Lys(Dnp)-Lys-NH2) at a final concentration of 20 μM. After that, the fluorescence signal at 320 nm (excitation)/405 nm (emission) was measured immediately every 35 s for 3.5 min with a Bio-Tek Synergy4 plate reader. The velocities of reactions with compounds added at various concentrations compared to the reaction added with DMSO were calculated and used to generate inhibition profiles. For each compound, at least three independent experiments were performed for the determination of IC<sub>50</sub> values. The IC<sub>50</sub> values were expressed as the mean ± SD and determined via nonlinear regression analysis using GraphPad Prism software 8.0 (GraphPad Software, Inc., San Diego, CA, USA).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Cells</h4><div class="NLM_p">RD cells were maintained in MEM Rega-3 medium supplemented with 2% FBS, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, and 0.075% NaHCO<sub>3</sub>, all supplied by Gibco, Life Technologies. Cells were maintained at 37 °C in a humidified atmosphere of 5% CO<sub>2</sub>.</div><div class="NLM_p last">The Vero E6 cell line was obtained from American Type Culture Collection (ATCC, no. 1586) and maintained in Dulbecco’s modified Eagle medium (DMEM; Gibco Invitrogen) supplemented with 10% fetal bovine serum (FBS; Gibco Invitrogen), 1% antibiotic/antimycotic (Gibco Invitrogen), at 37 °C in a humidified 5% CO<sub>2</sub> incubator.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Viruses</h4><div class="NLM_p">Enterovirus 71 strain BrCr (EV71 BrCr), kindly provided by Prof. Dr. F. van Kuppeveld (Universiteit Utrecht, The Netherlands), was grown on RD cells. When a full cytopathic effect (CPE) was observed, the virus was harvested from the supernatants after centrifugation (10 min, 3000 rpm) and stored at −80 °C. The viral titer was determined by end point titration.</div><div class="NLM_p last">A clinical isolate of SARS-CoV-2 (nCoV-2019BetaCoV/Wuhan/WIV04/2019) was propagated in Vero E6 cells, and a viral titer was determined by TCID50.<a onclick="showRef(event, 'cit8a'); return false;" href="javascript:void(0);" class="ref cit8a">(8a)</a> All infection experiments were performed at biosafety level-3 (BSL-3).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Antiviral Assay</h4><div class="NLM_p">The anti-EV71 activity of selected compounds was tested in RD cells and seeded in a 96-well plate (2.5 × 104 cells/well). Cells were allowed to adhere overnight, after which cells were infected with EV71; a serial dilution of selected compounds was added and incubated for 4 days, i.e., until complete CPE was observed in the untreated and infected virus control conditions. CPE was subsequently quantified using an MTS-reduction assay (MTS = 3f-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2<i>H</i>-tetrazolium, inner salt). For this, a MTS/phenazine methosulfate (PMS) stock solution (2 mg/mL MTS (Promega, Leiden, The Netherlands) and 46 μg/mL PMS (Sigma-Aldrich, Bornem, Belgium) in PBS at pH 6–6.5) was diluted 1:20 in MEM (Life Technologies, Gent, Belgium cat. no. 21090-022). The medium was aspirated from wells, and 75 μL of MTS/PMS solution was added. After 1–2 h of incubation at 37 °C, the absorbance was measured at 498 nm. The % inhibition for each well is then calculated by normalization of the absorbance to the condition of untreated-infected cells (=0% inhibition) and the condition of untreated–uninfected cells (=100% inhibition). From the obtained dose–response curve, the EC<sub>50</sub> is calculated by curve fitting using Dotmatics software.</div><div class="NLM_p">To assess the antiviral activity of compounds against SARS-CoV-2, preseeded Vero E6 cells (5 × 10<sup>4</sup> cells/well) were treated with different concentrations of the indicated compounds for 1 h and then were infected with SARS-CoV-2 at an MOI of 0.01. At 24 h pi, the cell supernatant was collected, and a viral RNA copy number in the cell supernatant was measured using real-time PCR, as described previously.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">The antiviral activity of selected compounds against EV68, CoxA21, CoxB3, RV-A02, and RV-B14 was tested as described previously.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p last">To assess the cytotoxicity of the test compounds, Vero E6 cells preseeded in a 96-well dish (2 × 10<sup>4</sup> cells/well) were treated with different concentrations of the indicated compounds, and 24 h later, the relative numbers of surviving cells were measured with cell counting kit-8 (GK10001, GLPBIO) according to the manufacturer’s instructions.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Pharmacokinetic Profiles in CD-1 Mice</h4><div class="NLM_p">Male CD-1 mice (<i>n</i> = 3 per group) were treated with a solution of compounds <b>18p</b> and <b>26d</b> (DMSO/EtOH/PEG300/NaCl (5:5:40:50, v/v/v/v)) at doses of 20 mg/kg, 5 mg/kg, and 5 mg/kg via intraperitoneal (ip), subcutaneous (sc), and intravenous (iv), respectively. Blood samples were collected at 0.05, 0.25, 0.75, 2, 4, 8, and 24 h after administration. Serum samples were obtained through common procedures, and the concentrations of the compound in the supernatant were analyzed by LC-MS/MS.</div><div class="NLM_p last">All procedures relating to animal handling, care, and treatment were performed according to the guidelines approved by the Institutional Animal Care and Use Committee of the contract research organizations performing the study.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02258" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08682" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08682" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02258?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02258</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">An overlay of EV71 3C<sup>pro</sup>-<b>18p</b> and <b>AG7088</b> complex structures; the binding pocket of 3C<sup>pro</sup> of CoXB3, RV-A02, and EV71; the anti-SARS-CoV-2 activities of compounds; the <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of target compounds; the purity of target compounds; and the crystallography data collection and refinement statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula stings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_001.pdf">jm0c02258_si_001.pdf (3.41 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_002.csv">jm0c02258_si_002.csv (3.23 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors have been deposited into the Protein Data Bank with accession code 7DNC (EV71 3C<sup>pro</sup>-<b>18p</b>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02258" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70535" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70535" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leike Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Virology, Wuhan Institute
of Virology, Center for Biosafety Mega-Science, Chinese Academy of
Sciences, Wuhan, Hubei 430071, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2593-2571" title="Orcid link">http://orcid.org/0000-0002-2593-2571</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86fceee7e8e1eae3efede3c6f1eea8efe9f0a8e5e8"><span class="__cf_email__" data-cfemail="97edfff6f9f0fbf2fefcf2d7e0ffb9fef8e1b9f4f9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yechun Xu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, University of Chinese Academy of
Sciences, Hangzhou 310024, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1581-6155" title="Orcid link">http://orcid.org/0000-0002-1581-6155</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5b2238232e1b28323636753a38753835"><span class="__cf_email__" data-cfemail="79001a010c390a10141457181a571a17">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johan Neyts</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology and Immunology, Rega Institute
for Medical Research, Laboratory of Virology
and Chemotherapy, Leuven B-3000, Belgium</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d6bcb9beb7b8f8b8b3afa2a596bda3bab3a3a0b3b8f8b4b3"><span class="__cf_email__" data-cfemail="254f4a4d444b0b4b405c5156654e5049405053404b0b4740">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Liu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing, 210023, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, University of Chinese Academy of
Sciences, Hangzhou 310024, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3685-6268" title="Orcid link">http://orcid.org/0000-0003-3685-6268</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0a6266637f4a79636767246b69246964"><span class="__cf_email__" data-cfemail="442c282d3104372d29296a25276a272a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenhao Dai</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dirk Jochmans</span> - <span class="hlFld-Affiliation affiliation">KU
Leuven, Department of Microbiology and Immunology, Rega Institute
for Medical Research, Laboratory of Virology
and Chemotherapy, Leuven B-3000, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hang Xie</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hang Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Virology, Wuhan Institute
of Virology, Center for Biosafety Mega-Science, Chinese Academy of
Sciences, Wuhan, Hubei 430071, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing, 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haixia Su</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Di Chang</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, 130 Meilong Road, Shanghai 200237, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiang Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, University of Chinese Academy of
Sciences, Hangzhou 310024, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingjing Peng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lili Zhu</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, 130 Meilong Road, Shanghai 200237, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Nian</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing, 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rolf Hilgenfeld</span> - <span class="hlFld-Affiliation affiliation">Institute
of Molecular Medicine, University of Lübeck, 23562 Lübeck, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German Center
for Infection Research (DZIF), University
of Lübeck, 23562 Lübeck, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8850-2977" title="Orcid link">http://orcid.org/0000-0001-8850-2977</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, University of Chinese Academy of
Sciences, Hangzhou 310024, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaixian Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>W.D., D.J., H.X., and H.Y. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e5139-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34413" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34413" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful to the National Natural Science Foundation of China (21632008, 21672231, 21877122, 21877118, and 81620108027), the Strategic Priority Research Program of the Chinese Academy of Sciences (XDA12040107 and XDA12040201), and the Youth Innovation Promotion Association CAS (grants 2018367 to L.-K.Z.) for financial support</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations used</td></tr><tr><td class="NLM_term">EV71</td><td class="NLM_def"><p class="first last">enterovirus 71</p></td></tr><tr><td class="NLM_term">HFMD</td><td class="NLM_def"><p class="first last">hand, foot, and mouth disease</p></td></tr><tr><td class="NLM_term">SARS-CoV-2</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome coronavirus 2</p></td></tr><tr><td class="NLM_term">COVID-19</td><td class="NLM_def"><p class="first last">coronavirus disease 2019</p></td></tr><tr><td class="NLM_term">3C<sup>pro</sup></td><td class="NLM_def"><p class="first last">3C protease</p></td></tr><tr><td class="NLM_term">3CL<sup>pro</sup></td><td class="NLM_def"><p class="first last">3C-like protease</p></td></tr><tr><td class="NLM_term">HRV</td><td class="NLM_def"><p class="first last">human rhinoviruses</p></td></tr><tr><td class="NLM_term">EV68</td><td class="NLM_def"><p class="first last">enterovirus 68</p></td></tr><tr><td class="NLM_term">CoxA21</td><td class="NLM_def"><p class="first last">coxsackievirus A21</p></td></tr><tr><td class="NLM_term">DMP</td><td class="NLM_def"><p class="first last">Dess–Martin periodinane</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein data bank</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last"><i>O</i>-(7-; 2-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropyl-ethyl-amine</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectra</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85262" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85262" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 20 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennette, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, H. H.</span></span> <span> </span><span class="NLM_article-title">An apparently new enterovirus isolated from patients with disease of the central nervous system</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1093/infdis/129.3.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1093%2Finfdis%2F129.3.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=4361245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADyaE2c7hsl2lsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=1974&pages=304-309&author=N.+J.+Schmidtauthor=E.+H.+Lennetteauthor=H.+H.+Ho&title=An+apparently+new+enterovirus+isolated+from+patients+with+disease+of+the+central+nervous+system&doi=10.1093%2Finfdis%2F129.3.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">An apparently new enterovirus isolated from patients with disease of the central nervous system</span></div><div class="casAuthors">Schmidt N J; Lennette E H; Ho H H</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of infectious diseases</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">304-9</span>
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9Z6tSn96A0ftos8R3JUhXfW6udTcc2ebWWWj6WT7AI7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE2c7hsl2lsw%253D%253D&md5=2b47e2de0621e8ab2d4f99e40d0e02d5</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2F129.3.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252F129.3.304%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DN.%2BJ.%26aulast%3DLennette%26aufirst%3DE.%2BH.%26aulast%3DHo%26aufirst%3DH.%2BH.%26atitle%3DAn%2520apparently%2520new%2520enterovirus%2520isolated%2520from%2520patients%2520with%2520disease%2520of%2520the%2520central%2520nervous%2520system%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D1974%26volume%3D129%26spage%3D304%26epage%3D309%26doi%3D10.1093%2Finfdis%2F129.3.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. C.</span></span> <span> </span><span class="NLM_article-title">Acute flaccid paralysis in infants and young children with enterovirus 71 infection: MR imaging findings and clinical correlates</span>. <i>Am. J. Neuroradiol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">205</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=11158910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BD3M7nslygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=200-205&author=C.+Y.+Chenauthor=Y.+C.+Changauthor=C.+C.+Huangauthor=C.+C.+Luiauthor=K.+W.+Leeauthor=S.+C.+Huang&title=Acute+flaccid+paralysis+in+infants+and+young+children+with+enterovirus+71+infection%3A+MR+imaging+findings+and+clinical+correlates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Acute flaccid paralysis in infants and young children with enterovirus 71 infection: MR imaging findings and clinical correlates</span></div><div class="casAuthors">Chen C Y; Chang Y C; Huang C C; Lui C C; Lee K W; Huang S C</div><div class="citationInfo"><span class="NLM_cas:title">AJNR. American journal of neuroradiology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">200-5</span>
        ISSN:<span class="NLM_cas:issn">0195-6108</span>.
    </div><div class="casAbstract">BACKGROUND AND PURPOSE:  Enterovirus 71 (EV71) infection is now considered an important cause of childhood acute flaccid paralysis.  The purpose of our study was to determine whether EV71-infection-related acute flaccid paralysis in infants and young children has characteristic MR imaging patterns.  METHODS:  Seven infants and young children with acute paralysis of the upper or lower extremities and positive EV71 cultures underwent spinal MR studies during an outbreak of hand-foot-and-mouth disease in Taiwan in 1998.  RESULTS:  Acute paralysis was observed in one upper extremity in two patients, in one lower extremity in three patients, and in both lower extremities in two patients.  None of the patients had sensory impairment or bulbar palsy.  MR studies showed unilateral or bilateral hyperintense lesions in the anterior horn regions of the cord on T2-weighted images in six patients.  No abnormal signal was present in one patient.  Two of three patients who received intravenous injections of contrast material had ventral root enhancement on T1-weighted images.  One of them also had enhancement of the unilateral anterior horn cells.  At clinical follow-up, both patients with bilateral anterior horn abnormalities had residual motor weakness, whereas only one of the five patients with unilateral involvement had residual weakness.  CONCLUSION:  EV71 radiculomyelitis tends to be unilateral and to specifically involve both the anterior horn cells of the cord and the ventral roots.  MR imaging allows early detection of spinal cord and root lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSSyQVHwRse6f6BottYdIAWfW6udTcc2eaz26EMGItnSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3M7nslygtw%253D%253D&md5=fcfb26746adb20a36a5e4dd3c9dbc77b</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BY.%26aulast%3DChang%26aufirst%3DY.%2BC.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DLui%26aufirst%3DC.%2BC.%26aulast%3DLee%26aufirst%3DK.%2BW.%26aulast%3DHuang%26aufirst%3DS.%2BC.%26atitle%3DAcute%2520flaccid%2520paralysis%2520in%2520infants%2520and%2520young%2520children%2520with%2520enterovirus%252071%2520infection%253A%2520MR%2520imaging%2520findings%2520and%2520clinical%2520correlates%26jtitle%3DAm.%2520J.%2520Neuroradiol.%26date%3D2001%26volume%3D22%26spage%3D200%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNutt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span> <span> </span><span class="NLM_article-title">Neuropathology in 2 cases of fatal enterovirus type 71 infection from a recent epidemic in the People’s Republic of China: a histopathologic, immunohistochemical, and reverse transcription polymerase chain reaction study</span>. <i>Hum. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1288</span>– <span class="NLM_lpage">1295</span>, <span class="refDoi"> DOI: 10.1016/j.humpath.2009.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.humpath.2009.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19386354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSitbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2009&pages=1288-1295&author=Y.+Yangauthor=H.+Wangauthor=E.+Gongauthor=J.+Duauthor=X.+Zhaoauthor=M.+A.+McNuttauthor=S.+Wangauthor=Y.+Zhongauthor=Z.+Gaoauthor=J.+Zheng&title=Neuropathology+in+2+cases+of+fatal+enterovirus+type+71+infection+from+a+recent+epidemic+in+the+People%E2%80%99s+Republic+of+China%3A+a+histopathologic%2C+immunohistochemical%2C+and+reverse+transcription+polymerase+chain+reaction+study&doi=10.1016%2Fj.humpath.2009.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1cR"><div class="casContent"><span class="casTitleNuber">1c</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropathology in 2 cases of fatal enterovirus type 71 infection from a recent epidemic in the People's Republic of China: a histopathologic, immunohistochemical, and reverse transcription polymerase chain reaction study</span></div><div class="casAuthors">Yang, Yu; Wang, Hua; Gong, Encong; Du, Juan; Zhao, Xishun; McNutt, Michael A.; Wang, Shenglan; Zhong, Yanfeng; Gao, Zifen; Zheng, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1288-1295</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Between late March and June of 2008, an outbreak of hand, foot, and mouth disease caused by enterovirus 71 occurred in China.  In this outbreak, more than 176,000 cases occurred, and at least 40 people died.  However, there has been no description of the neuropathol. features of fatal enterovirus 71 infection cases during this outbreak.  We report postmortem studies in 2 fatal cases of enterovirus 71 infection with examn. of the central nervous system using histopathol., immunohistochem., and reverse transcription polymerase chain reaction.  Characteristic features of acute encephalitis were found predominantly in the brain stem and spinal cord.  Abundant nos. of inflammatory cells with unusual irregularly rod-shaped and lobulated nuclei, which mimicked neutrophil morphol., were found both in abscess-like clusters assocd. with necrosis, as well as in inflammatory infiltrates.  Immunohistochem. showed that most of these cells were CD68 pos. and CD15 neg.  Viral antigens were found in the cytoplasm of neurons, neuronal processes, and inflammatory cells, most often assocd. with glial nodules.  The presence of enterovirus 71 was confirmed by reverse transcription polymerase chain reaction and sequencing.  Results indicate that immunohistochem. with CD68 and CD15 may offer some help in the differential diagnosis of brain stem encephalitis caused by enterovirus 71.  Postmortem reverse transcription polymerase chain reaction from brain stem tissues, which have already undergone fixation and histol. processing, can be an important diagnostic tool, which may be of particular value in patients who may have been misdiagnosed clin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAVuQnvSrkvbVg90H21EOLACvtfcHk0lgfN6esAD8QjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSitbrF&md5=da609281c171d96e82a6138d2388c55e</span></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2009.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2009.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGong%26aufirst%3DE.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMcNutt%26aufirst%3DM.%2BA.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DJ.%26atitle%3DNeuropathology%2520in%25202%2520cases%2520of%2520fatal%2520enterovirus%2520type%252071%2520infection%2520from%2520a%2520recent%2520epidemic%2520in%2520the%2520People%25E2%2580%2599s%2520Republic%2520of%2520China%253A%2520a%2520histopathologic%252C%2520immunohistochemical%252C%2520and%2520reverse%2520transcription%2520polymerase%2520chain%2520reaction%2520study%26jtitle%3DHum.%2520Pathol.%26date%3D2009%26volume%3D40%26spage%3D1288%26epage%3D1295%26doi%3D10.1016%2Fj.humpath.2009.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">McMinn, P. C.</span></span> <span> </span><span class="NLM_article-title">An overview of the evolution of enterovirus 71 and its clinical and public health significance</span>. <i>FEMS Microbiol Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2002.tb00601.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1111%2Fj.1574-6976.2002.tb00601.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=12007645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjtl2itLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2002&pages=91-107&author=P.+C.+McMinn&title=An+overview+of+the+evolution+of+enterovirus+71+and+its+clinical+and+public+health+significance&doi=10.1111%2Fj.1574-6976.2002.tb00601.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1dR"><div class="casContent"><span class="casTitleNuber">1d</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the evolution of enterovirus 71 and its clinical and public health significance</span></div><div class="casAuthors">McMinn, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-107</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Since its discovery in 1969, enterovirus 71 (EV71) has been recognized as a frequent cause of epidemics of hand-foot-and-mouth disease (HFMD) assocd. with severe neurol. sequelae in a small proportion of cases.  There has been a significant increase in EV71 epidemic activity throughout the Asia-Pacific region since 1997.  Recent HFMD epidemics in this region have been assocd. with a severe form of brainstem encephalitis assocd. with pulmonary edema and high case-fatality rates.  The emergence of large-scale epidemic activity in the Asia-Pacific region has been assocd. with the circulation of three genetic lineages that appear to be undergoing rapid evolutionary change.  Two of these lineages (B3 and B4) have not been described previously and appear to have arisen from an endemic focus in equatorial Asia, which has served as a source of virus for HFMD epidemics in Malaysia, Singapore and Australia.  The third lineage (C2) has previously been identified and was primarily responsible for the large HFMD epidemic in Taiwan during 1998.  As EV71 appears not to be susceptible to newly developed antiviral agents and a vaccine is not currently available, control of EV71 epidemics through high-level surveillance and public health intervention needs to be maintained and extended throughout the Asia-Pacific region.  Future research should focus on (1) understanding the mol. genetics of EV71 virulence, (2) identification of the receptor(s) for EV71, (3) development of antiviral agents to ameliorate the severity of neurol. disease and (4) vaccine development to control epidemics.  Following the successful experience of the poliomyelitis control program, it may be possible to control EV71 epidemics if an effective live-attenuated vaccine is developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6kmGHJuTlArVg90H21EOLACvtfcHk0lgfN6esAD8QjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjtl2itLw%253D&md5=649be3c04036cf0f6f3534a955271318</span></div><a href="/servlet/linkout?suffix=cit1d&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2002.tb00601.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2002.tb00601.x%26sid%3Dliteratum%253Aachs%26aulast%3DMcMinn%26aufirst%3DP.%2BC.%26atitle%3DAn%2520overview%2520of%2520the%2520evolution%2520of%2520enterovirus%252071%2520and%2520its%2520clinical%2520and%2520public%2520health%2520significance%26jtitle%3DFEMS%2520Microbiol%2520Rev.%26date%3D2002%26volume%3D26%26spage%3D91%26epage%3D107%26doi%3D10.1111%2Fj.1574-6976.2002.tb00601.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span> <span> </span><span class="NLM_article-title">Viral and host proteins involved in picornavirus life cycle</span>. <i>J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1186/1423-0127-16-103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1186%2F1423-0127-16-103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19925687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BD1MjptFSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=103-117&author=J.+Y.+Linauthor=T.+C.+Chenauthor=K.+F.+Wengauthor=S.+C.+Changauthor=L.+L.+Chenauthor=S.+R.+Shih&title=Viral+and+host+proteins+involved+in+picornavirus+life+cycle&doi=10.1186%2F1423-0127-16-103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1eR"><div class="casContent"><span class="casTitleNuber">1e</span><div class="casTitle"><span class="NLM_cas:atitle">Viral and host proteins involved in picornavirus life cycle</span></div><div class="casAuthors">Lin Jing-Yi; Chen Tzu-Chun; Weng Kuo-Feng; Chang Shih-Cheng; Chen Li-Lien; Shih Shin-Ru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomedical science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Picornaviruses cause several diseases, not only in humans but also in various animal hosts.  For instance, human enteroviruses can cause hand-foot-and-mouth disease, herpangina, myocarditis, acute flaccid paralysis, acute hemorrhagic conjunctivitis, severe neurological complications, including brainstem encephalitis, meningitis and poliomyelitis, and even death.  The interaction between the virus and the host is important for viral replication, virulence and pathogenicity.  This article reviews studies of the functions of viral and host factors that are involved in the life cycle of picornavirus.  The interactions of viral capsid proteins with host cell receptors is discussed first, and the mechanisms by which the viral and host cell factors are involved in viral replication, viral translation and the switch from translation to RNA replication are then addressed.  Understanding how cellular proteins interact with viral RNA or viral proteins, as well as the roles of each in viral infection, will provide insights for the design of novel antiviral agents based on these interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3vmkIwk_wJedfST00W0BcfW6udTcc2ea-ExsA0MOKQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjptFSqsg%253D%253D&md5=ab1a69bd6d303ff40fc25b1e49ad100a</span></div><a href="/servlet/linkout?suffix=cit1e&amp;dbid=16384&amp;doi=10.1186%2F1423-0127-16-103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1423-0127-16-103%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%2BY.%26aulast%3DChen%26aufirst%3DT.%2BC.%26aulast%3DWeng%26aufirst%3DK.%2BF.%26aulast%3DChang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DL.%2BL.%26aulast%3DShih%26aufirst%3DS.%2BR.%26atitle%3DViral%2520and%2520host%2520proteins%2520involved%2520in%2520picornavirus%2520life%2520cycle%26jtitle%3DJ.%2520Biomed.%2520Sci.%26date%3D2009%26volume%3D16%26spage%3D103%26epage%3D117%26doi%3D10.1186%2F1423-0127-16-103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit1f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. H.</span></span> <span> </span><span class="NLM_article-title">Enterovirus 71 infection of human dendritic cells</span>. <i>Exp. Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">1166</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.3181/0903-RM-116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.3181%2F0903-RM-116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ams7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2009&pages=1166-1173&author=Y.+W.+Linauthor=S.+W.+Wangauthor=Y.+Y.+Tungauthor=S.+H.+Chen&title=Enterovirus+71+infection+of+human+dendritic+cells&doi=10.3181%2F0903-RM-116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1fR"><div class="casContent"><span class="casTitleNuber">1f</span><div class="casTitle"><span class="NLM_cas:atitle">Enterovirus 71 infection of human dendritic cells</span></div><div class="casAuthors">Lin, Yu-Wen; Wang, Shainn-Wei; Tung, Yuk-Ying; Chen, Shun-Hua</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Biology and Medicine (Maywood, NJ, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1166-1173</span>CODEN:
                <span class="NLM_cas:coden">EBMMBE</span>;
        ISSN:<span class="NLM_cas:issn">1535-3702</span>.
    
            (<span class="NLM_cas:orgname">Society for Experimental Biology and Medicine</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) causes death and long-term neurol. sequelae in hundreds of thousands of young children, but its pathogenesis remains elusive.  Dendritic cells (DCs) play a crucial role in antiviral immunity by functioning as professional antigen-presenting cells to prime T cells and by secreting cytokines to modulate immune responses.  Here, we show that EV71 productively infected human immature DCs and expressed viral antigen in DCs.  EV71 entry into DCs was partially mediated by DC-SIGN.  Further analyses revealed that EV71 increased the viability, activation, release of cytokines, interleukin-6, interleukin-12, and tumor necrosis factor-α in DCs.  Moreover, EV71 enabled DCs to stimulate T-cell proliferation.  Collectively, these findings suggest that EV71 infection of human DCs in vivo is very likely to elicit protective immunity, because in infected mice, both T cells and IL-6 function to reduce mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBCp6P0rqREbVg90H21EOLACvtfcHk0ljB0a2bcqVnfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ams7fI&md5=248cc1c49295514a284bb2e1f716f3d3</span></div><a href="/servlet/linkout?suffix=cit1f&amp;dbid=16384&amp;doi=10.3181%2F0903-RM-116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3181%252F0903-RM-116%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DY.%2BW.%26aulast%3DWang%26aufirst%3DS.%2BW.%26aulast%3DTung%26aufirst%3DY.%2BY.%26aulast%3DChen%26aufirst%3DS.%2BH.%26atitle%3DEnterovirus%252071%2520infection%2520of%2520human%2520dendritic%2520cells%26jtitle%3DExp.%2520Biol.%2520Med.%26date%3D2009%26volume%3D234%26spage%3D1166%26epage%3D1173%26doi%3D10.3181%2F0903-RM-116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A. Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, P.</span></span> <span> </span><span class="NLM_article-title">Pilot scale production of highly efficacious and stable enterovirus 71 vaccine candidates</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e34834</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0034834</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1371%2Fjournal.pone.0034834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=22529942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmtlyqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e34834&author=A.+H.+Chouauthor=C.+C.+Liuauthor=C.+P.+Changauthor=M.+S.+Guoauthor=S.+Y.+Hsiehauthor=W.+H.+Yangauthor=H.+J.+Chaoauthor=C.+L.+Wuauthor=J.+L.+Huangauthor=M.+S.+Leeauthor=A.+Y.+C.+Huauthor=S.+C.+Linauthor=Y.+Y.+Huangauthor=M.+H.+Huauthor=Y.+H.+Chowauthor=J.+R.+Chiangauthor=J.+Y.+Changauthor=P.+Chong&title=Pilot+scale+production+of+highly+efficacious+and+stable+enterovirus+71+vaccine+candidates&doi=10.1371%2Fjournal.pone.0034834"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Pilot scale production of highly efficacious and stable enterovirus 71 vaccine candidates</span></div><div class="casAuthors">Chou, Ai-Hsiang; Liu, Chia-Chyi; Chang, Cheng-Peng; Guo, Meng-Shin; Hsieh, Shih-Yang; Yang, Wen-Hsueh; Chao, Hsin-Ju; Wu, Chien-Long; Huang, Ju-Lan; Lee, Min-Shi; Hu, Alan Yung-Chi; Lin, Sue-Chen; Huang, Yu-Yun; Hu, Mei-Hua; Chow, Yen-Hung; Chiang, Jen-Ron; Chang, Jui-Yuan; Chong, Pele</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e34834</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Enterovirus 71 (EV71) has caused several epidemics of hand, foot and mouth diseases (HFMD) in Asia and now is being recognized as an important neurotropic virus.  Effective medications and prophylactic vaccine against EV71 infection are urgently needed.  Based on the success of inactivated poliovirus vaccine, a prototype chem. inactivated EV71 vaccine candidate has been developed and currently in human phase 1 clin. trial.  Principal Finding: In this report, we present the development of a serum-free cell-based EV71 vaccine.  The optimization at each step of the manufg. process was investigated, characterized and quantified.  In the up-stream process development, different com. available cell culture media either contg. serum or serum-free was screened for cell growth and virus yield using the roller-bottle technol.  VP-SFM serum-free medium was selected based on the Vero cell growth profile and EV71 virus prodn.  After the up-stream processes (virus harvest, diafiltration and concn.), a combination of gel-filtration liq. chromatog. and/or sucrose-gradient ultracentrifugation down-stream purifn. processes were investigated at a pilot scale of 40 L each.  Although the combination of chromatog. and sucrose-gradient ultracentrifugation produced extremely pure EV71 infectious virus particles, the overall yield of vaccine was 7-10 % as detd. by a VP2-based quant. ELISA.  Using chromatog. as the downstream purifn., the virus yield was 30-43 %.  To retain the integrity of virus neutralization epitopes and the stability of the vaccine product, the best virus inactivation was found to be 0.025% formalin-treatment at 37°C for 3 to 6 days.  Furthermore, the formalin-inactivated virion vaccine candidate was found to be stable for >18 mo at 4°C and a microgram of viral proteins formulated with alum adjuvant could induce strong virus-neutralizing antibody responses in mice, rats, rabbits, and non-human primates.  Conclusion: These results provide valuable information supporting the current cell-based serum-free EV71 vaccine candidate going into human Phase I clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPI5iYNnTzpbVg90H21EOLACvtfcHk0ljB0a2bcqVnfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmtlyqtr8%253D&md5=843537d2d8f55429a4e64e8eb512d38b</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0034834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0034834%26sid%3Dliteratum%253Aachs%26aulast%3DChou%26aufirst%3DA.%2BH.%26aulast%3DLiu%26aufirst%3DC.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BP.%26aulast%3DGuo%26aufirst%3DM.%2BS.%26aulast%3DHsieh%26aufirst%3DS.%2BY.%26aulast%3DYang%26aufirst%3DW.%2BH.%26aulast%3DChao%26aufirst%3DH.%2BJ.%26aulast%3DWu%26aufirst%3DC.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%2BL.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DHu%26aufirst%3DA.%2BY.%2BC.%26aulast%3DLin%26aufirst%3DS.%2BC.%26aulast%3DHuang%26aufirst%3DY.%2BY.%26aulast%3DHu%26aufirst%3DM.%2BH.%26aulast%3DChow%26aufirst%3DY.%2BH.%26aulast%3DChiang%26aufirst%3DJ.%2BR.%26aulast%3DChang%26aufirst%3DJ.%2BY.%26aulast%3DChong%26aufirst%3DP.%26atitle%3DPilot%2520scale%2520production%2520of%2520highly%2520efficacious%2520and%2520stable%2520enterovirus%252071%2520vaccine%2520candidates%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De34834%26doi%3D10.1371%2Fjournal.pone.0034834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Z.</span></span> <span> </span><span class="NLM_article-title">Immunogenicity and safety of an enterovirus 71 vaccine in healthy Chinese children and infants: a randomised, double-blind, placebo-controlled phase 2 clinical trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">1037</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(12)61764-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2FS0140-6736%2812%2961764-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23352749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVymsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2013&pages=1037-1045&author=F.+C.+Zhuauthor=Z.+L.+Liangauthor=X.+L.+Liauthor=H.+M.+Geauthor=F.+Y.+Mengauthor=Q.+Y.+Maoauthor=Y.+T.+Zhangauthor=Y.+M.+Huauthor=Z.+Y.+Zhangauthor=J.+X.+Liauthor=F.+Gaoauthor=Q.+H.+Chenauthor=Q.+Y.+Zhuauthor=K.+Chuauthor=X.+Wuauthor=X.+Yaoauthor=H.+J.+Guoauthor=X.+Q.+Chenauthor=P.+Liuauthor=Y.+Y.+Dongauthor=F.+X.+Liauthor=X.+L.+Shenauthor=J.+Z.+Wang&title=Immunogenicity+and+safety+of+an+enterovirus+71+vaccine+in+healthy+Chinese+children+and+infants%3A+a+randomised%2C+double-blind%2C+placebo-controlled+phase+2+clinical+trial&doi=10.1016%2FS0140-6736%2812%2961764-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Immunogenicity and safety of an enterovirus 71 vaccine in healthy Chinese children and infants: a randomised, double-blind, placebo-controlled phase 2 clinical trial</span></div><div class="casAuthors">Zhu, Feng-Cai; Liang, Zheng-Lun; Li, Xiu-Ling; Ge, Heng-Ming; Meng, Fan-Yue; Mao, Qun-Ying; Zhang, Yun-Tao; Hu, Yue-Mei; Zhang, Zhen-Yu; Li, Jing-Xin; Gao, Fan; Chen, Qing-Hua; Zhu, Qi-Yan; Chu, Kai; Wu, Xing; Yao, Xin; Guo, Hui-Jie; Chen, Xiao-Qin; Liu, Pei; Dong, Yu-Ying; Li, Feng-Xiang; Shen, Xin-Liang; Wang, Jun-Zhi</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9871</span>),
    <span class="NLM_cas:pages">1037-1045</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Enterovirus 71 (EV71) outbreaks are a socioeconomic burden, esp. in the western Pacific region.  Results of phase 1 clin. trials suggest an EV71 vaccine has a clin. acceptable safety profile and immunogenicity.  We aimed to assess the best possible dose and formulation, immunogenicity, and safety profile of this EV71 vaccine in healthy Chinese children.  Methods: This randomized, double-blind, placebo-controlled, phase 2 trial was undertaken at one site in Donghai County, Jiangsu Province, China.  Eligible participants were healthy boys or girls aged 6-36 mo.  Participants were randomly assigned (1:1:1:1:1) to receive either 160 U, 320 U, or 640 U alum-adjuvant EV71 vaccine, 640 U adjuvant-free EV71 vaccine, or a placebo (contg. alum adjuvant only), according to a blocked randomization list generated by SAS 9.1.  Participants and investigators were masked to the assignment.  The primary endpoint was anti-EV71 neutralizing antibody geometric mean titers (GMTs) at day 56, analyzed according to protocol.  The study is registered with ClinicalTrials.gov, no. NCT01399853.  Findings: We randomly assigned 1200 participants, 240 (120 aged 6-11 mo [infants] and 120 aged 12-36 mo [children]) of whom were assigned to each dose. 1106 participants completed the study and were included in the according-to-protocol anal.  The main reasons for dropout were withdrawal of consent and refusal to donate a blood sample.  Infants who received the 640 U adjuvant vaccine had the highest GMTs on day 56 (742·2 [95% CI 577·3-954·3]), followed by those who received the 320 U formulation (497·9 [383·1-647·0]).  For children, those who received the 320 U formulation had the highest GMTs on day 56 (1383·2 [1037·3-1844·5]).  Participants who received the vaccine had significantly higher GMTs than did who received placebo (p<0·0001).  For the subgroup of participants who were seroneg. at baseline, both infants and children who received the 640 U adjuvant vaccine had the highest GMTs on day 56 (522·8 [403·9-676·6] in infants and 708·4 [524·1-957·6] in children), followed by those who received the 320 U adjuvant vaccine (358·2 [280·5-457·5] in infants and 498·0 [383·4-646·9] in children).  549 (45·8%) Of 1200 participants (95 CI 42·9-48·6%) reported at least one injection-site or systemic adverse reaction, but the incidence of adverse reactions did not differ significantly between groups (p=0·36).  The 640 U alum-adjuvant vaccine group had a significantly higher incidence of induration than did the 640 U adjuvant-free group (p=0·001).  Interpretation: Taking immunogenicity, safety, and prodn. capacity into account, the 320 U alum-adjuvant formulation of the EV71 vaccine is probably the best possible formulation for phase 3 trials.  Funding: The National Science and Technol. Major Project (2011ZX10004-902) of the Chinese Ministry of Science and Technol., China's 12-5 National Major Infectious Disease Program (2012ZX10002-001), and Beijing Vigoo Biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_OLrmt_3gtrVg90H21EOLACvtfcHk0lhc7aPaO7DtHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVymsb0%253D&md5=d6d11a5249ed8ae3a8d4407dd0e7478a</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2812%2961764-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252812%252961764-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DF.%2BC.%26aulast%3DLiang%26aufirst%3DZ.%2BL.%26aulast%3DLi%26aufirst%3DX.%2BL.%26aulast%3DGe%26aufirst%3DH.%2BM.%26aulast%3DMeng%26aufirst%3DF.%2BY.%26aulast%3DMao%26aufirst%3DQ.%2BY.%26aulast%3DZhang%26aufirst%3DY.%2BT.%26aulast%3DHu%26aufirst%3DY.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%2BY.%26aulast%3DLi%26aufirst%3DJ.%2BX.%26aulast%3DGao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DQ.%2BH.%26aulast%3DZhu%26aufirst%3DQ.%2BY.%26aulast%3DChu%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DX.%2BQ.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DDong%26aufirst%3DY.%2BY.%26aulast%3DLi%26aufirst%3DF.%2BX.%26aulast%3DShen%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DJ.%2BZ.%26atitle%3DImmunogenicity%2520and%2520safety%2520of%2520an%2520enterovirus%252071%2520vaccine%2520in%2520healthy%2520Chinese%2520children%2520and%2520infants%253A%2520a%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520phase%25202%2520clinical%2520trial%26jtitle%3DLancet%26date%3D2013%26volume%3D381%26spage%3D1037%26epage%3D1045%26doi%3D10.1016%2FS0140-6736%2812%2961764-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Stanway, G.</span></span> <span> </span><span class="NLM_article-title">Structure, function and evolution of picornaviruses</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2483</span>– <span class="NLM_lpage">2501</span>, <span class="refDoi"> DOI: 10.1099/0022-1317-71-11-2483</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1099%2F0022-1317-71-11-2483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=2254747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK3MXjtVKi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1990&pages=2483-2501&author=G.+Stanway&title=Structure%2C+function+and+evolution+of+picornaviruses&doi=10.1099%2F0022-1317-71-11-2483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3aR"><div class="casContent"><span class="casTitleNuber">3a</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function and evolution of picornaviruses</span></div><div class="casAuthors">Stanway, Glyn</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2483-501</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    </div><div class="casAbstract">A review with 172 refs., illustrating how mol. and structural analyses have revealed the biol. diversity seen in picornaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDuBkYasUPF7Vg90H21EOLACvtfcHk0lhc7aPaO7DtHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXjtVKi&md5=7cd20db9e42c972c15ed6b85d0c37e22</span></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1099%2F0022-1317-71-11-2483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F0022-1317-71-11-2483%26sid%3Dliteratum%253Aachs%26aulast%3DStanway%26aufirst%3DG.%26atitle%3DStructure%252C%2520function%2520and%2520evolution%2520of%2520picornaviruses%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D1990%26volume%3D71%26spage%3D2483%26epage%3D2501%26doi%3D10.1099%2F0022-1317-71-11-2483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit3b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weady, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span> <span> </span><span class="NLM_article-title">Conservation of amino acids in human rhinovirus 3C protease correlates with broad-spectrum antiviral activity of Rupintrivir, a novel human rhinovirus 3C protease inhibitor</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1128/AAC.49.2.619-626.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.49.2.619-626.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15673742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht12nt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=619-626&author=S.+L.+Binfordauthor=F.+Maldonadoauthor=M.+A.+Brothersauthor=P.+T.+Weadyauthor=L.+S.+Zalmanauthor=J.+W.+Meadorauthor=D.+A.+Matthewsauthor=A.+K.+Patick&title=Conservation+of+amino+acids+in+human+rhinovirus+3C+protease+correlates+with+broad-spectrum+antiviral+activity+of+Rupintrivir%2C+a+novel+human+rhinovirus+3C+protease+inhibitor&doi=10.1128%2FAAC.49.2.619-626.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation of amino acids in human rhinovirus 3C protease correlates with broad-spectrum antiviral activity of rupintrivir, a novel human rhinovirus 3C protease inhibitor</span></div><div class="casAuthors">Binford, S. L.; Maldonado, F.; Brothers, M. A.; Weady, P. T.; Zalman, L. S.; Meador, J. W., III; Matthews, D. A.; Patick, A. K.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">619-626</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The picornavirus 3C protease is required for the majority of proteolytic cleavages that occur during the viral life cycle.  Comparisons of published amino acid sequences from 6 human rhinoviruses (HRV) and 20 human enteroviruses (HEV) show considerable variability in the 3C protease-coding region but strict conservation of the catalytic triad residues.  Rupintrivir (formerly AG7088) is an irreversible inhibitor of HRV 3C protease with potent in vitro activity against all HRV serotypes (48 of 48), HEV strains (4 of 4), and untyped HRV field isolates (46 of 46) tested.  To better understand the relationship between in vitro antiviral activity and 3C protease-rupintrivir binding interactions, the authors performed nucleotide sequence analyses on an addnl. 21 HRV serotypes and 11 HRV clin. isolates.  Antiviral activity was also detd. for 23 HRV clin. isolates and four addnl. HEV strains.  Sequence comparison of 3C proteases (n = 58) show that 13 and 11 of the 14 amino acids that are involved in side chain interactions with rupintrivir are strictly conserved among HRV and HEV, resp.  These sequence analyses are consistent with the comparable in vitro antiviral potencies of rupintrivir against all HRV serotypes, HRV isolates, and HEV strains tested (50% effective concn. range, 3 to 183 nM; n = 125).  In summary, the conservation of crit. amino acid residues in 3C protease and the observation of potent, broad-spectrum antipicornavirus activity of rupintrivir highlight the advantages of 3C protease as an antiviral target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4MFFGS5_xkLVg90H21EOLACvtfcHk0lhc7aPaO7DtHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht12nt7Y%253D&md5=936a058a019545e186f5a6e5b4cec298</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.1128%2FAAC.49.2.619-626.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.49.2.619-626.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DMaldonado%26aufirst%3DF.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DWeady%26aufirst%3DP.%2BT.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26atitle%3DConservation%2520of%2520amino%2520acids%2520in%2520human%2520rhinovirus%25203C%2520protease%2520correlates%2520with%2520broad-spectrum%2520antiviral%2520activity%2520of%2520Rupintrivir%252C%2520a%2520novel%2520human%2520rhinovirus%25203C%2520protease%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2005%26volume%3D49%26spage%3D619%26epage%3D626%26doi%3D10.1128%2FAAC.49.2.619-626.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit3c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Condon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budahazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slsson, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafranca, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribskov, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S.</span></span> <span> </span><span class="NLM_article-title">Structure of human rhinovirus 3C protease reveals a trypsin-like polypeptide fold, RNA-binding site, and means for cleaving precursor polyprotein</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">761</span>– <span class="NLM_lpage">771</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(94)90059-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2F0092-8674%2894%2990059-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=7515772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK2cXksFejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1994&pages=761-771&author=D.+A.+Matthewsauthor=W.+W.+Smithauthor=R.+A.+Ferreauthor=B.+Condonauthor=G.+Budahaziauthor=W.+Slssonauthor=J.+E.+Villafrancaauthor=C.+A.+Jansonauthor=H.+E.+McElroyauthor=C.+L.+Gribskovauthor=S.+Worland&title=Structure+of+human+rhinovirus+3C+protease+reveals+a+trypsin-like+polypeptide+fold%2C+RNA-binding+site%2C+and+means+for+cleaving+precursor+polyprotein&doi=10.1016%2F0092-8674%2894%2990059-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3cR"><div class="casContent"><span class="casTitleNuber">3c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human rhinovirus 3C protease reveals a trypsin-like polypeptide folds, RNA-binding site, and means for cleaving precursor polyprotein</span></div><div class="casAuthors">Matthews, David A.; Smith, Ward W.; Ferre, Rose Ann; Condon, Brad; Budahal, Georgg; Sisson, Wes; Villafranca, J. E.; Janson, Cheryl A.; McElroy, H. E.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">761-71</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The structure of human rhinovirus-14 3C protease (3Cpro) has been detd. at 2.3 Å resoln. and refined to an R factor of 0.22.  This cysteine protease folds into two topol. equiv. six-stranded β barrels in this sense is similar to trypsin-like serine proteases.  However, there are differences in the lengths and positioning of individual β strands as well as in loops connecting elements of secondary structure.  The catalytic residues Cys-146, His-40, and Glu-71 are positioned as in serine proteases, but the oxyanion hole is moved 1-1.2 Å away.  Residues that bind to the 5' noncoding region of rhinovirus genomic RNA are located on the opposite side of the mol. from the active site.  Interactions between individual 3Cpro mols. in the crystal lattice suggest a model for intermol. proteolytic cleavage of the 3CD polyprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonBQyOVEP8krVg90H21EOLACvtfcHk0lj9N6oOUAEY5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXksFejtL8%253D&md5=7815a9833147d6b1aa7bb7067073415e</span></div><a href="/servlet/linkout?suffix=cit3c&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2894%2990059-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252894%252990059-0%26sid%3Dliteratum%253Aachs%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DSmith%26aufirst%3DW.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DCondon%26aufirst%3DB.%26aulast%3DBudahazi%26aufirst%3DG.%26aulast%3DSlsson%26aufirst%3DW.%26aulast%3DVillafranca%26aufirst%3DJ.%2BE.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DMcElroy%26aufirst%3DH.%2BE.%26aulast%3DGribskov%26aufirst%3DC.%2BL.%26aulast%3DWorland%26aufirst%3DS.%26atitle%3DStructure%2520of%2520human%2520rhinovirus%25203C%2520protease%2520reveals%2520a%2520trypsin-like%2520polypeptide%2520fold%252C%2520RNA-binding%2520site%252C%2520and%2520means%2520for%2520cleaving%2520precursor%2520polyprotein%26jtitle%3DCell%26date%3D1994%26volume%3D77%26spage%3D761%26epage%3D771%26doi%3D10.1016%2F0092-8674%2894%2990059-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit3d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M. S.</span></span> <span> </span><span class="NLM_article-title">Mutations at KFRDI and VGK domains of enterovirus 71 3C protease affect its RNA binding and proteolytic activities</span>. <i>J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">248</span>, <span class="refDoi"> DOI: 10.1007/BF02256567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1007%2FBF02256567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=14966374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXht1Cnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=239-248&author=S.+R.+Shihauthor=C.+Chiangauthor=T.+C.+Chenauthor=C.+N.+Wuauthor=J.+T.+A.+Hsuauthor=J.+C.+Leeauthor=M.+J.+Hwangauthor=M.+L.+Liauthor=G.+W.+Chenauthor=M.+S.+Ho&title=Mutations+at+KFRDI+and+VGK+domains+of+enterovirus+71+3C+protease+affect+its+RNA+binding+and+proteolytic+activities&doi=10.1007%2FBF02256567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3dR"><div class="casContent"><span class="casTitleNuber">3d</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations at KFRDI and VGK Domains of Enterovirus 71 3C Protease Affect Its RNA Binding and Proteolytic Activities</span></div><div class="casAuthors">Shih, Shin-Ru; Chiang, Chiayn; Chen, Tzu-Chun; Wu, Cheng-Nan; Hsu, John Tsu-An; Lee, Jin-Ching; Hwang, Ming-Jing; Li, Mei-Ling; Chen, Guang-Wu; Ho, Mei-Shan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomedical Science (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-248</span>CODEN:
                <span class="NLM_cas:coden">JBCIEA</span>;
        ISSN:<span class="NLM_cas:issn">1021-7770</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">The 3C protease (3Cpro) of enterovirus 71 (EV71) is a good mol. target for drug discovery.  Notably, this protease was found to possess RNA-binding activity.  The regions responsible for RNA binding were classified as KFRDI' (positions 82-86) and 'VGK' (positions 154-156) in 3Cpro by mutagenesis study.  Although the RNA-binding regions are structurally distinct from the catalytic site of EV71 3Cpro, mutations in the RNA-binding regions influenced 3Cpro proteolytic activity.  In contrast, mutations at the catalytic site had almost no influence on RNA binding ability.  We identified certain mutations within 3Cpro which abrogated both the RNA-binding activity of the expressed, recombinant, protease and the ability to rescue virus from an infectious full-length clone of EV71 (pEV71).  Interestingly, mutation at position 84 from Arg(R) to Lys(K) was found to retain good RNA binding and proteolytic activity for the recombinant 3Cpro; however, no virus could be rescued when pEV71 with the R84K mutation was introduced into the infectious copy.  Together, these results may provide useful information for using 3Cpro as the mol. target to develop anti-EV71 agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr3_Su1I1FU7Vg90H21EOLACvtfcHk0lj9N6oOUAEY5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXht1Cnu7o%253D&md5=08d151fea6fc47f692f264fbe0357381</span></div><a href="/servlet/linkout?suffix=cit3d&amp;dbid=16384&amp;doi=10.1007%2FBF02256567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02256567%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DS.%2BR.%26aulast%3DChiang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DT.%2BC.%26aulast%3DWu%26aufirst%3DC.%2BN.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DHwang%26aufirst%3DM.%2BJ.%26aulast%3DLi%26aufirst%3DM.%2BL.%26aulast%3DChen%26aufirst%3DG.%2BW.%26aulast%3DHo%26aufirst%3DM.%2BS.%26atitle%3DMutations%2520at%2520KFRDI%2520and%2520VGK%2520domains%2520of%2520enterovirus%252071%25203C%2520protease%2520affect%2520its%2520RNA%2520binding%2520and%2520proteolytic%2520activities%26jtitle%3DJ.%2520Biomed.%2520Sci.%26date%3D2004%26volume%3D11%26spage%3D239%26epage%3D248%26doi%3D10.1007%2FBF02256567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit3e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Structures of EV71 RNA-dependent RNA polymerase in complex with substrate and analogue provide a drug target against the hand-foot-and-mouth disease pandemic in China</span>. <i>Protein Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1007/s13238-010-0061-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1007%2Fs13238-010-0061-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21203964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3cXoslelurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=491-500&author=Y.+Wuauthor=Z.+Louauthor=Y.+Miaoauthor=Y.+Yuauthor=H.+Dongauthor=W.+Pengauthor=M.+Bartlamauthor=X.+Liauthor=Z.+Rao&title=Structures+of+EV71+RNA-dependent+RNA+polymerase+in+complex+with+substrate+and+analogue+provide+a+drug+target+against+the+hand-foot-and-mouth+disease+pandemic+in+China&doi=10.1007%2Fs13238-010-0061-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3eR"><div class="casContent"><span class="casTitleNuber">3e</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of EV71 RNA-dependent RNA polymerase in complex with substrate and analogue provide a drug target against the hand-foot-and-mouth disease pandemic in China</span></div><div class="casAuthors">Wu, Yang; Lou, Zhiyong; Miao, Yi; Yu, Yue; Dong, Hui; Peng, Wei; Bartlam, Mark; Li, Xuemei; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">491-500</span>CODEN:
                <span class="NLM_cas:coden">PCREFB</span>;
        ISSN:<span class="NLM_cas:issn">1674-800X</span>.
    
            (<span class="NLM_cas:orgname">Higher Education Press</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71), one of the major causative agents for hand-foot-and-mouth disease (HFMD), has caused more than 100 deaths among Chinese children since March 2008.  The EV71 genome encodes an RNA-dependent RNA polymerase (RdRp), denoted 3Dpol, which is central for viral genome replication and is a key target for the discovery of specific antiviral therapeutics.  Here we report the crystal structures of EV71 RdRp (3Dpol) and in complex with substrate guanosine-5'-triphosphate and analog 5-bromouridine-5'-triphosphate best to 2.4 Å resoln.  The structure of EV71 RdRp (3Dpol) has a wider open thumb domain compared with the most closely related crystal structure of poliovirus RdRp.  And the EV71 RdRp (3Dpol) complex with GTP or Br-UTP bounded shows two distinct movements of the polymerase by substrate or analog binding.  The model of the complex with the template:primer derived by superimposition with foot-and-mouth disease virus (FMDV) 3D/RNA complex reveals the likely recognition and binding of template:primer RNA by the polymerase.  These results together provide a mol. basis for EV71 RNA replication and reveal a potential target for anti-EV71 drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQj64xy57fGLVg90H21EOLACvtfcHk0lj9N6oOUAEY5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXoslelurw%253D&md5=44d2d2b455afdb16db34f748ebe1297a</span></div><a href="/servlet/linkout?suffix=cit3e&amp;dbid=16384&amp;doi=10.1007%2Fs13238-010-0061-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13238-010-0061-7%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DStructures%2520of%2520EV71%2520RNA-dependent%2520RNA%2520polymerase%2520in%2520complex%2520with%2520substrate%2520and%2520analogue%2520provide%2520a%2520drug%2520target%2520against%2520the%2520hand-foot-and-mouth%2520disease%2520pandemic%2520in%2520China%26jtitle%3DProtein%2520Cell%26date%3D2010%26volume%3D1%26spage%3D491%26epage%3D500%26doi%3D10.1007%2Fs13238-010-0061-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit3f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Antiviral drug discovery for the treatment of enterovirus 71 infections</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">194</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.antiviral.2012.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23261847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFSku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=183-194&author=L.+Shangauthor=M.+Xuauthor=Z.+Yin&title=Antiviral+drug+discovery+for+the+treatment+of+enterovirus+71+infections&doi=10.1016%2Fj.antiviral.2012.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3fR"><div class="casContent"><span class="casTitleNuber">3f</span><div class="casTitle"><span class="NLM_cas:atitle">Antiviral drug discovery for the treatment of enterovirus 71 infections</span></div><div class="casAuthors">Shang, Luqing; Xu, Mengying; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-194</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Enterovirus 71 (EV71) is a small, pos.-sense, single-stranded RNA virus in the genus Enterovirus, family Picornavirus.  It causes hand, foot, and mouth disease in infants and children, which in a small percentage of cases progresses to central nervous system infection, ranging from aseptic meningitis to fatal encephalitis.  Sporadic cases of EV71 infection occur throughout the world, but large epidemics have occurred recently in Southeast Asia and China.  There are currently no approved vaccines or antiviral therapies for the prevention or treatment of EV71 infection.  This paper reviews efforts to develop antiviral therapies against EV71.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjzLwxTUQnX7Vg90H21EOLACvtfcHk0li1HJRx-KSiyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFSku7c%253D&md5=ad739907255ef3e637e5160fd79bf4f6</span></div><a href="/servlet/linkout?suffix=cit3f&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DAntiviral%2520drug%2520discovery%2520for%2520the%2520treatment%2520of%2520enterovirus%252071%2520infections%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D97%26spage%3D183%26epage%3D194%26doi%3D10.1016%2Fj.antiviral.2012.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span> <span> </span><span class="NLM_article-title">Enterovirus 71 3C protease cleaves a novel target Cstf-64 and inhibits cellular polyadenylation</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e1000593</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1000593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1371%2Fjournal.ppat.1000593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19779565" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000593&author=K.+F.+Wengauthor=M.+L.+Liauthor=C.+T.+Hungauthor=S.+R.+Shih&title=Enterovirus+71+3C+protease+cleaves+a+novel+target+Cstf-64+and+inhibits+cellular+polyadenylation&doi=10.1371%2Fjournal.ppat.1000593"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000593%26sid%3Dliteratum%253Aachs%26aulast%3DWeng%26aufirst%3DK.%2BF.%26aulast%3DLi%26aufirst%3DM.%2BL.%26aulast%3DHung%26aufirst%3DC.%2BT.%26aulast%3DShih%26aufirst%3DS.%2BR.%26atitle%3DEnterovirus%252071%25203C%2520protease%2520cleaves%2520a%2520novel%2520target%2520Cstf-64%2520and%2520inhibits%2520cellular%2520polyadenylation%26jtitle%3DPLoS%2520Pathog.%26date%3D2009%26volume%3D5%26spage%3De1000593%26doi%3D10.1371%2Fjournal.ppat.1000593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiew, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">11754</span>– <span class="NLM_lpage">11762</span>, <span class="refDoi"> DOI: 10.1128/JVI.01348-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.01348-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=22915796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=11754-11762&author=Y.+Kimauthor=S.+Lovellauthor=K.+C.+Tiewauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=K.+P.+Battaileauthor=W.+C.+Groutasauthor=K.+O.+Chang&title=Broad-spectrum+antivirals+against+3C+or+3C-like+proteases+of+picornaviruses%2C+noroviruses%2C+and+coronaviruses&doi=10.1128%2FJVI.01348-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span></div><div class="casAuthors">Kim, Yunjeong; Lovell, Scott; Tiew, Kok-Chuan; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Battaile, Kevin P.; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11754-11762</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phylogenetic anal. has demonstrated that some pos.-sense RNA viruses can be classified into the picornavirus-like super-cluster, which includes picornaviruses, caliciviruses, and coronaviruses.  These viruses possess 3C or 3C-like proteases (3Cpro or 3CLpro, resp.), which contain a typical chymotrypsin-like fold and a catalytic triad (or dyad) with a Cys residue as a nucleophile.  The conserved key sites of 3Cpro or 3CLpro may serve as attractive targets for the design of broad-spectrum antivirals for multiple viruses in the supercluster.  We previously reported the structure-based design and synthesis of potent protease inhibitors of Norwalk virus (NV), a member of the Caliciviridae family.  We report herein the broad-spectrum antiviral activities of three compds. possessing a common dipeptidyl residue with different warheads, i.e., an aldehyde (GC373), a bisulfite adduct (GC376), and an α-ketoamide (GC375), against viruses that belong to the supercluster.  All compds. were highly effective against the majority of tested viruses, with half-maximal inhibitory concns. in the high nanomolar or low micromolar range in enzyme- and/or cell-based assays and with high therapeutic indexes.  We also report the high-resoln. X-ray cocrystal structures of NV 3CLpro-, poliovirus 3Cpro-, and transmissible gastroenteritis virus 3CLpro- GC376 inhibitor complexes, which show the compd. covalently bound to a nucleophilic Cys residue in the catalytic site of the corresponding protease.  We conclude that these compds. have the potential to be developed as antiviral therapeutics aimed at a single virus or multiple viruses in the picornavirus-like supercluster by targeting 3Cpro or 3CLpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDn4MUH7bIlLVg90H21EOLACvtfcHk0li1HJRx-KSiyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M&md5=c7fe984b52490199a2c2ef333c22fb98</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1128%2FJVI.01348-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01348-12%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DTiew%26aufirst%3DK.%2BC.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3DBroad-spectrum%2520antivirals%2520against%25203C%2520or%25203C-like%2520proteases%2520of%2520picornaviruses%252C%2520noroviruses%252C%2520and%2520coronaviruses%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D11754%26epage%3D11762%26doi%3D10.1128%2FJVI.01348-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit4c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">What coronavirus3C-like protease tells us: From structure, substrate selectivity, to inhibitor design</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1002/med.21783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fmed.21783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2021&pages=1-34&author=M.+Xiongauthor=H.+Suauthor=W.+Zhaoauthor=H.+Xieauthor=Q.+Shaoauthor=Y.+Xu&title=What+coronavirus3C-like+protease+tells+us%3A+From+structure%2C+substrate+selectivity%2C+to+inhibitor+design&doi=10.1002%2Fmed.21783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.1002%2Fmed.21783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21783%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DWhat%2520coronavirus3C-like%2520protease%2520tells%2520us%253A%2520From%2520structure%252C%2520substrate%2520selectivity%252C%2520to%2520inhibitor%2520design%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D3%26spage%3D1%26epage%3D34%26doi%3D10.1002%2Fmed.21783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwaltney, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi-Burris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span> <span> </span><span class="NLM_article-title">Phase II, randomized, double-blind, placebo-controlled studies of Ruprintrivir nasal spray 2-percent suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3907</span>– <span class="NLM_lpage">3916</span>, <span class="refDoi"> DOI: 10.1128/AAC.47.12.3907-3916.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.47.12.3907-3916.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=14638501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2003&pages=3907-3916&author=F.+G.+Haydenauthor=R.+B.+Turnerauthor=J.+M.+Gwaltneyauthor=K.+Chi-Burrisauthor=M.+Gerstenauthor=P.+Hsyuauthor=A.+K.+Patickauthor=G.+J.+Smithauthor=L.+S.+Zalman&title=Phase+II%2C+randomized%2C+double-blind%2C+placebo-controlled+studies+of+Ruprintrivir+nasal+spray+2-percent+suspension+for+prevention+and+treatment+of+experimentally+induced+rhinovirus+colds+in+healthy+volunteers&doi=10.1128%2FAAC.47.12.3907-3916.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">Phase II, randomized, double-blind, placebo-controlled studies of ruprintrivir nasal spray 2% suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span></div><div class="casAuthors">Hayden, Frederick G.; Turner, Ronald B.; Gwaltney, Jack M.; Chi-burris, Kathy; Gersten, Merril; Hsyu, Poe; Patick, Amy K.; Smith, George J., III; Zalman, Leora S.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3907-3916</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Human rhinovirus (HRV) infections are usually self-limited but may be assocd. with serious consequences, particularly in those with asthma and chronic respiratory disease.  Effective antiviral agents are needed for preventing and treating HRV illnesses.  Ruprintrivir (Agouron Pharmaceuticals, Inc., San Diego, Calif.) selectively inhibits HRV 3C protease and shows potent, broad-spectrum anti-HRV activity in vitro.  We conducted three double-blind, placebo-controlled clin. trials in 202 healthy volunteers to assess the activity of ruprintrivir in exptl. HRV infection.  Subjects were randomized to receive intranasal ruprintrivir (8 mg) or placebo sprays as prophylaxis (two or five times daily [2×/day or 5×/day] for 5 days) starting 6 h before infection or as treatment (5×/day for 4 days) starting 24 h after infection.  Ruprintrivir prophylaxis reduced the proportion of subjects with pos. viral cultures (for 5×/day dosing groups, 44% for ruprintrivir treatment group vs. 70% for placebo treatment group [P = 0.03]; for 2×/day dosing groups, 60% for ruprintrivir group vs. 92% for placebo group [P = 0.004]) and viral titers but did not decrease the frequency of colds.  Ruprintrivir treatment reduced the mean total daily symptom score (2.2 for ruprintrivir treatment group and 3.3 for the placebo treatment group [P = 0.014]) by 33%.  Secondary endpoints, including viral titers, individual symptom scores, and nasal discharge wts., were also reduced by ruprintrivir treatment.  Overall, ruprintrivir was well tolerated; blood-tinged mucus and nasal passage irritation were the most common adverse effects reported.  Pharmacokinetic anal. of plasma and nasal ruprintrivir concns. revealed intranasal drug residence with minimal systemic absorption.  Results from these studies in exptl. rhinoviral infection support continued investigation of intranasal ruprintrivir in the setting of natural HRV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoudaxRiBJLg7Vg90H21EOLACvtfcHk0liCdy84f8pJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D&md5=6d6c33f56af0c4f45145b291ace2dba1</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1128%2FAAC.47.12.3907-3916.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.47.12.3907-3916.2003%26sid%3Dliteratum%253Aachs%26aulast%3DHayden%26aufirst%3DF.%2BG.%26aulast%3DTurner%26aufirst%3DR.%2BB.%26aulast%3DGwaltney%26aufirst%3DJ.%2BM.%26aulast%3DChi-Burris%26aufirst%3DK.%26aulast%3DGersten%26aufirst%3DM.%26aulast%3DHsyu%26aufirst%3DP.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26atitle%3DPhase%2520II%252C%2520randomized%252C%2520double-blind%252C%2520placebo-controlled%2520studies%2520of%2520Ruprintrivir%2520nasal%2520spray%25202-percent%2520suspension%2520for%2520prevention%2520and%2520treatment%2520of%2520experimentally%2520induced%2520rhinovirus%2520colds%2520in%2520healthy%2520volunteers%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2003%26volume%3D47%26spage%3D3907%26epage%3D3916%26doi%3D10.1128%2FAAC.47.12.3907-3916.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shie, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P. H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of 3C protease inhibitors as anti-enterovirus 71 agents</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">7388</span>– <span class="NLM_lpage">7398</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmc.2008.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=18583140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps12jtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=7388-7398&author=C.+J.+Kuoauthor=J.+J.+Shieauthor=J.+M.+Fangauthor=G.+R.+Yenauthor=J.+T.+A.+Hsuauthor=H.+G.+Liuauthor=S.+N.+Tsengauthor=S.+C.+Changauthor=C.+Y.+Leeauthor=S.+R.+Shihauthor=P.+H.+Liang&title=Design%2C+synthesis%2C+and+evaluation+of+3C+protease+inhibitors+as+anti-enterovirus+71+agents&doi=10.1016%2Fj.bmc.2008.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and evaluation of 3C protease inhibitors as anti-enterovirus 71 agents</span></div><div class="casAuthors">Kuo, Chih-Jung; Shie, Jiun-Jie; Fang, Jim-Min; Yen, Guei-Rung; Hsu, John T.-A.; Liu, Hun-Ge; Tseng, Sung-Nain; Chang, Shih-Cheng; Lee, Ching-Yin; Shih, Shin-Ru; Liang, Po-Huang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7388-7398</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human enterovirus (EV) belongs to the picornavirus family, which consists of over 200 medically relevant viruses.  A peptidomimetic inhibitor AG7088 was developed to inhibit the 3C protease of rhinovirus (a member of the family), a chymotrypsin-like protease required for viral replication, by forming a covalent bond with the active site Cys residue.  In this study, we have prepd. the recombinant 3C protease from EV71 (TW/2231/98), a particular strain which causes severe outbreaks in Asia, and developed inhibitors against the protease and the viral replication.  For inhibitor design, the P3 group of AG7088, which is not interacting with the rhinovirus protease, was replaced with a series of cinnamoyl derivs. directly linked to P2 group through an amide bond to simplify the synthesis.  While the replacement caused decreased potency, the activity can be largely improved by substituting the α,β-unsatd. ester with an aldehyde at the P1' position.  The best inhibitor 10b showed EC50 of 18 nM without apparent toxicity (CC50 > 25 μM).  Our study provides potent inhibitors of the EV71 3C protease as anti-EV71 agents and facilitates the combinatorial synthesis of derivs. for further improving the inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0kBp35E-n6bVg90H21EOLACvtfcHk0liCdy84f8pJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps12jtbg%253D&md5=a319b7e1253e6f14592c9317469c4d74</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DC.%2BJ.%26aulast%3DShie%26aufirst%3DJ.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%2BM.%26aulast%3DYen%26aufirst%3DG.%2BR.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%2BA.%26aulast%3DLiu%26aufirst%3DH.%2BG.%26aulast%3DTseng%26aufirst%3DS.%2BN.%26aulast%3DChang%26aufirst%3DS.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BY.%26aulast%3DShih%26aufirst%3DS.%2BR.%26aulast%3DLiang%26aufirst%3DP.%2BH.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%25203C%2520protease%2520inhibitors%2520as%2520anti-enterovirus%252071%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D7388%26epage%3D7398%26doi%3D10.1016%2Fj.bmc.2008.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship study of peptidomimetic aldehydes as enterovirus 71 3C protease inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.ejmech.2016.08.064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=27614190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2jurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=559-573&author=Y.+Zhaiauthor=Y.+Maauthor=F.+Maauthor=Q.+Nieauthor=X.+Renauthor=Y.+Wangauthor=L.+Shangauthor=Z.+Yin&title=Structure-activity+relationship+study+of+peptidomimetic+aldehydes+as+enterovirus+71+3C+protease+inhibitors&doi=10.1016%2Fj.ejmech.2016.08.064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5cR"><div class="casContent"><span class="casTitleNuber">5c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationship study of peptidomimetic aldehydes as enterovirus 71 3C protease inhibitors</span></div><div class="casAuthors">Zhai, Yangyang; Ma, Yuying; Ma, Fei; Nie, Quandeng; Ren, Xuejiao; Wang, Yaxin; Shang, Luqing; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">559-573</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of peptidomimetic aldehydes were designed, synthesized, and evaluated for their biochem. activity against 3C protease (3Cpro) and anti-enterovirus 71 (EV71) activity in vitro.  Mol. docking revealed that 5s (IC50 = 0.22 ± 0.07 μM, EC50 = 0.18 ± 0.05 μM) could bind well to the active site of EV71 3Cpro, which was consistent with the biol. data compared to ref. 5a (IC50 = 0.54 ± 0.02 μM, EC50 = 0.26 ± 0.07 μM).  Structure and relationship study led to the discovery of aldehyde 5x (IC50 = 0.10 ± 0.02 μM, EC50 = 0.11 ± 0.07 μM), which exhibited the most potent 3Cpro inhibitory and antiviral activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMBUXTkD4dyLVg90H21EOLACvtfcHk0liCdy84f8pJlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2jurnN&md5=41cabd5d56ab0164e48cbd7806dbf2c7</span></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.064%26sid%3Dliteratum%253Aachs%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DF.%26aulast%3DNie%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DStructure-activity%2520relationship%2520study%2520of%2520peptidomimetic%2520aldehydes%2520as%2520enterovirus%252071%25203C%2520protease%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D559%26epage%3D573%26doi%3D10.1016%2Fj.ejmech.2016.08.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">A peptidyl aldehyde NK-1.8k suppresses EV71 and EV68 infection by targeting at 3C<sup>pro</sup></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1128/AAC.00049-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.00049-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=25691647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms12qsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=2636-2646&author=Y.+Wangauthor=B.+Yangauthor=Y.+Zhaiauthor=Z.+Yinauthor=Y.+Sunauthor=Z.+Rao&title=A+peptidyl+aldehyde+NK-1.8k+suppresses+EV71+and+EV68+infection+by+targeting+at+3Cpro&doi=10.1128%2FAAC.00049-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5dR"><div class="casContent"><span class="casTitleNuber">5d</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl aldehyde NK-1.8k suppresses enterovirus 71 and enterovirus 68 infection by targeting protease 3C</span></div><div class="casAuthors">Wang, Yaxin; Yang, Ben; Zhai, Yangyang; Yin, Zheng; Sun, Yuna; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2636-2646</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus (EV) is one of the major causative agents of hand, foot, and mouth disease in the Pacific-Asia region.  In particular, EV71 causes severe central nervous system infections, and the fatality rates from EV71 infection are high.  Moreover, an outbreak of respiratory illnesses caused by an emerging EV, EV68, recently occurred among over 1,000 young children in the United States and was also assocd. with neurol. infections.  Although enterovirus has emerged as a considerable global public health threat, no antiviral drug for clin. use is available.  In the present work, we screened our compd. library for agents targeting viral protease and identified a peptidyl aldehyde, NK-1.8k, that inhibits the proliferation of different EV71 strains and one EV68 strain and that had a 50% effective concn. of 90 nM.  Low cytotoxicity (50% cytotoxic concn., >200 μM) indicated a high selective index of over 2,000.  We further characterized a single amino acid substitution inside protease 3C (3Cpro), N69S, which conferred EV71 resistance to NK-1.8k, possibly by increasing the flexibility of the substrate binding pocket of 3Cpro.  The combination of NK-1.8k and an EV71 RNA-dependent RNA polymerase inhibitor or entry inhibitor exhibited a strong synergistic anti-EV71 effect.  Our findings suggest that NK-1.8k could potentially be developed for anti-EV therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMAtO795xx4LVg90H21EOLACvtfcHk0lhzURE2stHh6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms12qsr4%253D&md5=197d1a7a3af9eb32db0bef6c2dc741de</span></div><a href="/servlet/linkout?suffix=cit5d&amp;dbid=16384&amp;doi=10.1128%2FAAC.00049-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00049-15%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DA%2520peptidyl%2520aldehyde%2520NK-1.8k%2520suppresses%2520EV71%2520and%2520EV68%2520infection%2520by%2520targeting%2520at%25203Cpro%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D2636%26epage%3D2646%26doi%3D10.1128%2FAAC.00049-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9414</span>– <span class="NLM_lpage">9420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9414-9420&author=Y.+Zhaiauthor=X.+Zhaoauthor=Z.+Cuiauthor=M.+Wangauthor=Y.+Wangauthor=L.+Liauthor=Q.+Sunauthor=X.+Yangauthor=D.+Zengauthor=Y.+Liuauthor=Y.+Sunauthor=Z.+Louauthor=L.+Shangauthor=Z.+Yin&title=Cyanohydrin+as+an+Anchoring+Group+for+Potent+and+Selective+Inhibitors+of+Enterovirus+71+3C+Protease&doi=10.1021%2Facs.jmedchem.5b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5eR"><div class="casContent"><span class="casTitleNuber">5e</span><div class="casTitle"><span class="NLM_cas:atitle">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span></div><div class="casAuthors">Zhai, Yangyang; Zhao, Xiangshuai; Cui, Zhengjie; Wang, Man; Wang, Yaxin; Li, Linfeng; Sun, Qi; Yang, Xi; Zeng, Debin; Liu, Ying; Sun, Yuna; Lou, Zhiyong; Shang, Luqing; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9414-9420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyanohydrin derivs. as enterovirus 71 (EV71) 3C protease (3Cpro) inhibitors have been synthesized and assayed for their biochem. and antiviral activities.  Compared with the reported inhibitors, cyanohydrins (1S,2S,2'S,5S)-I and (1R,2S,2'S,5S)-I exhibited significantly improved activity and attractive selectivity profiles against other proteases, which were a result of the specific interactions between the cyanohydrin moiety and the catalytic site of 3Cpro.  Cyanohydrin as an anchoring group with high selectivity and excellent inhibitory activity represents a useful choice for cysteine protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxejr9UxByVLVg90H21EOLACvtfcHk0lhzURE2stHh6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK&md5=e43c6b3002e7c61cf8419bc83782be79</span></div><a href="/servlet/linkout?suffix=cit5e&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01013%26sid%3Dliteratum%253Aachs%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DCui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DCyanohydrin%2520as%2520an%2520Anchoring%2520Group%2520for%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Enterovirus%252071%25203C%2520Protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9414%26epage%3D9420%26doi%3D10.1021%2Facs.jmedchem.5b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">Application of dually activated Michael acceptor to the rational design of reversible covalent inhibitor for enterovirus 71 3C protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6146</span>– <span class="NLM_lpage">6162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6146-6162&author=Y.+Maauthor=L.+Liauthor=S.+Heauthor=C.+Shangauthor=Y.+Sunauthor=N.+Liuauthor=T.+D.+Meekauthor=Y.+Wangauthor=L.+Shang&title=Application+of+dually+activated+Michael+acceptor+to+the+rational+design+of+reversible+covalent+inhibitor+for+enterovirus+71+3C+protease&doi=10.1021%2Facs.jmedchem.9b00387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5fR"><div class="casContent"><span class="casTitleNuber">5f</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Dually Activated Michael Acceptor to the Rational Design of Reversible Covalent Inhibitor for Enterovirus 71 3C Protease</span></div><div class="casAuthors">Ma, Yuying; Li, Linfeng; He, Shuai; Shang, Chengyou; Sun, Yang; Liu, Ning; Meek, Thomas D.; Wang, Yaxin; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6146-6162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors (TCIs) have attracted growing attention from the pharmaceutical industry in recent decades because they have potential advantages in terms of efficacy, selectivity, and safety.  TCIs have recently evolved into a new version with reversibility that can be systematically modulated.  This feature may diminish the risk of haptenization and help optimize the drug-target residence time as needed.  The enteroviral 3C protease (3Cpro) is a valuable therapeutic target, but the development of 3Cpro inhibitors is far from satisfactory.  Therefore, we aimed to apply a reversible TCI approach to the design of novel 3Cpro inhibitors.  The introduction of various substituents onto the α-carbon of classical Michael acceptors yielded inhibitors bearing several classes of warheads.  Using steady-state kinetics and biomol. mass spectrometry, we confirmed the mode of reversible covalent inhibition and elucidated the mechanism by which the potency and reversibility were affected by electronic and steric factors.  This research produced several potent inhibitors with good selectivity and suitable reversibility; moreover, it validated the reversible TCI approach in the field of viral infection, suggesting broader applications in the design of reversible covalent inhibitors for other proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkpF34VsXG8bVg90H21EOLACvtfcHk0lg1i4C4X04f2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI&md5=2de757026f2542bff8092060a72107a5</span></div><a href="/servlet/linkout?suffix=cit5f&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00387%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DL.%26atitle%3DApplication%2520of%2520dually%2520activated%2520Michael%2520acceptor%2520to%2520the%2520rational%2520design%2520of%2520reversible%2520covalent%2520inhibitor%2520for%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6146%26epage%3D6162%26doi%3D10.1021%2Facs.jmedchem.9b00387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunn, A von.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, A de.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: structure-based design, synthesis, and activity assessment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4562</span>– <span class="NLM_lpage">4578</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01828</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01828" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4562-4578&author=L.+Zhangauthor=D.+Linauthor=Y.+Kusovauthor=Y.+Nianauthor=Q.+Maauthor=J.+Wangauthor=A+von.+Brunnauthor=P.+Leyssenauthor=K.+Lankoauthor=J.+Neytsauthor=A+de.+Wildeauthor=E.+J.+Snijderauthor=H.+Liuauthor=R.+Hilgenfeld&title=%CE%B1-Ketoamides+as+broad-spectrum+inhibitors+of+coronavirus+and+enterovirus+replication%3A+structure-based+design%2C+synthesis%2C+and+activity+assessment&doi=10.1021%2Facs.jmedchem.9b01828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5gR"><div class="casContent"><span class="casTitleNuber">5g</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Kusov, Yuri; Nian, Yong; Ma, Qingjun; Wang, Jiang; von Brunn, Albrecht; Leyssen, Pieter; Lanko, Kristina; Neyts, Johan; de Wilde, Adriaan; Snijder, Eric J.; Liu, Hong; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4562-4578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate.  Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs.  In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases.  Six crystal structures of protease:inhibitor complexes were detd. as part of this study.  Compds. synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic.  Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera.  The best near-equipotent inhibitors, (I) [R1 = cyclopentylmethyl (P2) and R1 = cyclohexylmethyl (P2)], display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures.  In Huh7 cells, I [R1 = cyclohexylmethyl (P2)] exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWpcfl4wTRLLVg90H21EOLACvtfcHk0lg1i4C4X04f2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D&md5=60d91a8ecf916856a844d835ecf84d5f</span></div><a href="/servlet/linkout?suffix=cit5g&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01828%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DNian%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBrunn%26aufirst%3DA%2Bvon.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DLanko%26aufirst%3DK.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DWilde%26aufirst%3DA%2Bde.%26aulast%3DSnijder%26aufirst%3DE.%2BJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketoamides%2520as%2520broad-spectrum%2520inhibitors%2520of%2520coronavirus%2520and%2520enterovirus%2520replication%253A%2520structure-based%2520design%252C%2520synthesis%252C%2520and%2520activity%2520assessment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4562%26epage%3D4578%26doi%3D10.1021%2Facs.jmedchem.9b01828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5h"><span><span class="NLM_label">(h) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">4-Iminooxazolidin-2-one as a bioisostere of the cyanohydrin moiety: Inhibitors of enterovirus 71 3C protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10333</span>– <span class="NLM_lpage">10339</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01335</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01335" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10333-10339&author=Y.+Maauthor=C.+Shangauthor=P.+Yangauthor=L.+Liauthor=Y.+Zhaiauthor=Z.+Yinauthor=B.+Wangauthor=L.+Shang&title=4-Iminooxazolidin-2-one+as+a+bioisostere+of+the+cyanohydrin+moiety%3A+Inhibitors+of+enterovirus+71+3C+protease&doi=10.1021%2Facs.jmedchem.8b01335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5hR"><div class="casContent"><span class="casTitleNuber">5h</span><div class="casTitle"><span class="NLM_cas:atitle">4-Iminooxazolidin-2-one as a Bioisostere of the Cyanohydrin Moiety: Inhibitors of Enterovirus 71 3C Protease</span></div><div class="casAuthors">Ma, Yuying; Shang, Chengyou; Yang, Peng; Li, Linfeng; Zhai, Yangyang; Yin, Zheng; Wang, Binghe; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10333-10339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A recently reported potent inhibitor of enterovirus 71 3C protease, (R)-1, was found to have stability and potential toxicity issues due to the presence of a cyanohydrin moiety.  Modifying the labile cyanohydrin moiety, by serendipity, led to the discovery of 4-iminooxazolidin-2-one-based inhibitors I and II with potent inhibitory activity and significantly improved stability.  In vivo pharmacokinetic studies of I also demonstrated high plasma exposure and moderate half-life.  These compds. have shown potential of becoming anti-EV71 drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7c_GeTuqpN7Vg90H21EOLACvtfcHk0liZd9wadWEazQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7I&md5=b180c88252dd62093c79ba660549454e</span></div><a href="/servlet/linkout?suffix=cit5h&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01335%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DShang%26aufirst%3DL.%26atitle%3D4-Iminooxazolidin-2-one%2520as%2520a%2520bioisostere%2520of%2520the%2520cyanohydrin%2520moiety%253A%2520Inhibitors%2520of%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10333%26epage%3D10339%26doi%3D10.1021%2Facs.jmedchem.8b01335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5i"><span><span class="NLM_label">(i) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of anti-EV71 agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3346</span>– <span class="NLM_lpage">3350</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.05.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5i&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmcl.2016.05.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5i&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=27234148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5i&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos12kt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=3346-3350&author=P.+Liauthor=B.+Yangauthor=F.+Haoauthor=P.+Wangauthor=H.+Heauthor=L.+Huangauthor=X.+Zhangauthor=S.+Zhangauthor=X.+Pengauthor=K.+Yinauthor=J.+Huauthor=X.+Chenauthor=Z.+Guauthor=L.+Wangauthor=L.+Shenauthor=G.+Huauthor=N.+Liauthor=J.+Liauthor=S.+Chenauthor=W.+Xiaoauthor=Z.+Wangauthor=Q.+Guoauthor=X.+Changauthor=L.+Zhangauthor=Q.+Caiauthor=T.+Lin&title=Design%2C+synthesis%2C+and+biological+evaluation+of+anti-EV71+agents&doi=10.1016%2Fj.bmcl.2016.05.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5iR"><div class="casContent"><span class="casTitleNuber">5i</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and biological evaluation of anti-EV71 agents</span></div><div class="casAuthors">Li, Peng; Yang, Bailing; Hao, Fei; Wang, Ping; He, Haiying; Huang, Lei; Zhang, Xuan; Zhang, Shengbin; Peng, Xuanjia; Yin, Ke; Hu, Jiao; Chen, Xinsheng; Gu, Zhengxian; Wang, Li; Shen, Liang; Hu, Guoping; Li, Ning; Li, Jian; Chen, Shuhui; Xiao, Wei; Wang, Zhenzhong; Guo, Qingming; Chang, Xiujuan; Zhang, Lanjun; Cai, Qixu; Lin, Tianwei</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3346-3350</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) is a major causative agent of hand, foot and mouth disease (HFMD), which can spread its infections to the central nervous and other systems with severe consequences.  In this article, design, chem. synthesis, and biol. evaluation of various anti-EV71 agents which incorporate Michael acceptors are described.  Further SAR study demonstrated that lactone type of Michael acceptor provided a new lead of anti-EV71 drug candidates with high anti-EV71 activity in cell-based assay and enhanced mouse plasma stability.  One of the most potent compds. [N-((3S)-6-(4-fluorobenzyl)-2-methyl-4,7-dioxo-7-(((S,E)-1-(2-oxodihydrofuran-3(2H)-ylidene)-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)heptan-3-yl)-5-methylisoxazole-3-carboxamide, cell-based anti-EV71 EC50 = 0.028 μM] showed acceptable stability profile towards mouse plasma, which resulted in promising pharmacokinetics in mouse via IP administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWJtv6RNsjvbVg90H21EOLACvtfcHk0liZd9wadWEazQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos12kt74%253D&md5=2813fbe22c4f4ceff01180d3f1fbddd8</span></div><a href="/servlet/linkout?suffix=cit5i&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.05.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.05.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DHao%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DYin%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DChang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DT.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520anti-EV71%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D3346%26epage%3D3350%26doi%3D10.1016%2Fj.bmcl.2016.05.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5j"><span><span class="NLM_label">(j) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shivanna, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kankanamalage, A. C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum inhibitors against 3C-like proteases of feline coronaviruses and feline caliciviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">4942</span>– <span class="NLM_lpage">4950</span>, <span class="refDoi"> DOI: 10.1128/JVI.03688-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5j&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.03688-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5j&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=25694593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5j&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFajsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=4942-4950&author=Y.+Kimauthor=V.+Shivannaauthor=S.+Narayananauthor=A.+M.+Priorauthor=S.+Weerasekaraauthor=D.+H.+Huaauthor=A.+C.+G.+Kankanamalageauthor=W.+C.+Groutasauthor=K.-O.+Chang&title=Broad-spectrum+inhibitors+against+3C-like+proteases+of+feline+coronaviruses+and+feline+caliciviruses&doi=10.1128%2FJVI.03688-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5jR"><div class="casContent"><span class="casTitleNuber">5j</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum inhibitors against 3C-like proteases of feline coronaviruses and feline caliciviruses</span></div><div class="casAuthors">Kim, Yunjeong; Shivanna, Vinay; Narayanan, Sanjeev; Prior, Allan M.; Weerasekara, Sahani; Hua, Duy H.; Kankanamalage, Anushka C. Galasiti; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4942-4950</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Feline infectious peritonitis and virulent, systemic calicivirus infection are caused by certain types of feline coronaviruses (FCoVs) and feline caliciviruses (FCVs), resp., and are important infectious diseases with high fatality rates in members of the Felidae family.  While FCoV and FCV belong to two distinct virus families, the Coronaviridae and the Caliciviridae, resp., they share a dependence on viral 3C-like protease (3CLpro) for their replication.  Since 3CLpro is functionally and structurally conserved among these viruses and essential for viral replication, 3CLpro is considered a potential target for the design of antiviral drugs with broad-spectrum activities against these distinct and highly important viral infections.  However, small-mol. inhibitors against the 3CLpro enzymes of FCoV and FCV have not been previously identified.  In this study, derivs. of peptidyl compds. targeting 3CLpro were synthesized and evaluated for their activities against FCoV and FCV.  The structures of compds. that showed potent dual antiviral activities with a wide margin of safety were identified and are discussed.  Furthermore, the in vivo efficacy of 3CLpro inhibitors was evaluated using a mouse model of coronavirus infection.  I.p. administration of two 3CLpro inhibitors in mice infected with murine hepatitis virus A59, a hepatotropic coronavirus, resulted in significant redns. in virus titers and pathol. lesions in the liver compared to the findings for the controls.  These results suggest that the series of 3CLpro inhibitors described here may have the potential to be further developed as therapeutic agents against these important viruses in domestic and wild cats.  This study provides important insights into the structure and function relationships of 3CLpro for the design of antiviral drugs with broader antiviral activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIjU5Ipe2fLVg90H21EOLACvtfcHk0liZd9wadWEazQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFajsb0%253D&md5=b57a99f8574d04454f0236f62da168f7</span></div><a href="/servlet/linkout?suffix=cit5j&amp;dbid=16384&amp;doi=10.1128%2FJVI.03688-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03688-14%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DShivanna%26aufirst%3DV.%26aulast%3DNarayanan%26aufirst%3DS.%26aulast%3DPrior%26aufirst%3DA.%2BM.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DKankanamalage%26aufirst%3DA.%2BC.%2BG.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3DBroad-spectrum%2520inhibitors%2520against%25203C-like%2520proteases%2520of%2520feline%2520coronaviruses%2520and%2520feline%2520caliciviruses%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D4942%26epage%3D4950%26doi%3D10.1128%2FJVI.03688-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5k"><span><span class="NLM_label">(k) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroze, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and bioevaluation of viral 3C and 3C-like protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6317</span>– <span class="NLM_lpage">6320</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5k&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmcl.2013.09.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5k&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=24125888" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5k&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1WjurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6317-6320&author=A.+M.+Priorauthor=Y.+Kimauthor=S.+Weerasekaraauthor=M.+Morozeauthor=K.+R.+Allistonauthor=R.+A.+Uyauthor=W.+C.+Groutasauthor=K.+O.+Changauthor=D.+H.+Hua&title=Design%2C+synthesis%2C+and+bioevaluation+of+viral+3C+and+3C-like+protease+inhibitors&doi=10.1016%2Fj.bmcl.2013.09.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5kR"><div class="casContent"><span class="casTitleNuber">5k</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and bioevaluation of viral 3C and 3C-like protease inhibitors</span></div><div class="casAuthors">Prior, Allan M.; Kim, Yunjeong; Weerasekara, Sahani; Moroze, Meghan; Alliston, Kevin R.; Uy, Roxanne Adeline Z.; Groutas, William C.; Chang, Kyeong-Ok; Hua, Duy H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6317-6320</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A class of tripeptidyl transition state inhibitors contg. a P1 glutamine surrogate, a P2 leucine, and a P3 arylalanines, was found to potently inhibit Norwalk virus replication in enzyme and cell based assays.  An array of warheads, including aldehyde, α-ketoamide, bisulfite adduct, and α-hydroxyphosphonate transition state mimic, was also investigated.  Tripeptidyls 2 and 6 possess antiviral activities against noroviruses, human rhinovirus, severe acute respiratory syndrome coronavirus, and coronavirus 229E, suggesting a broad range of antiviral activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtxJZJuCaVpLVg90H21EOLACvtfcHk0lir_BigNBNasw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1WjurfI&md5=4e71ce4a328d813f0b53ca153296f55b</span></div><a href="/servlet/linkout?suffix=cit5k&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.070%26sid%3Dliteratum%253Aachs%26aulast%3DPrior%26aufirst%3DA.%2BM.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DMoroze%26aufirst%3DM.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DUy%26aufirst%3DR.%2BA.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26aulast%3DHua%26aufirst%3DD.%2BH.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520bioevaluation%2520of%2520viral%25203C%2520and%25203C-like%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6317%26epage%3D6320%26doi%3D10.1016%2Fj.bmcl.2013.09.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit5l"><span><span class="NLM_label">(l) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerawarna, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, R. A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravapalli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lushington, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic inhibitors of 3C and 3C-like proteases of picornavirus, norovirus, and coronavirus</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3709</span>– <span class="NLM_lpage">3712</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5l&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmcl.2013.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5l&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23727045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5l&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFWqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3709-3712&author=S.+R.+Mandadapuauthor=P.+M.+Weerawarnaauthor=A.+M.+Priorauthor=R.+A.+Z.+Uyauthor=S.+Aravapalliauthor=K.+R.+Allistonauthor=G.+H.+Lushingtonauthor=Y.+j.+Kimauthor=D.+H.+Huaauthor=K.+O.+Changauthor=W.+C.+Groutas&title=Macrocyclic+inhibitors+of+3C+and+3C-like+proteases+of+picornavirus%2C+norovirus%2C+and+coronavirus&doi=10.1016%2Fj.bmcl.2013.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5lR"><div class="casContent"><span class="casTitleNuber">5l</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic inhibitors of 3C and 3C-like proteases of picornavirus, norovirus, and coronavirus</span></div><div class="casAuthors">Mandadapu, Sivakoteswara Rao; Weerawarna, Pathum M.; Prior, Allan M.; Uy, Roxanne Adeline Z.; Aravapalli, Sridhar; Alliston, Kevin R.; Lushington, Gerald H.; Kim, Yunjeong; Hua, Duy H.; Chang, Kyeong-Ok; Groutas, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3709-3712</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The design, synthesis, and in vitro evaluation of the first macrocyclic inhibitor I of 3C and 3C-like proteases of picornavirus, norovirus, and coronavirus are reported.  The in vitro inhibitory activity (50% effective concn.) of the macrocyclic inhibitor toward enterovirus 3C protease (CVB3 Nancy strain), and coronavirus (SARS-CoV) and norovirus 3C-like proteases, was detd. to be 1.8, 15.5 and 5.1 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9gBpCKI1MrVg90H21EOLACvtfcHk0lir_BigNBNasw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFWqsbc%253D&md5=c0498882427ca6d13bd5cbbc452bac7d</span></div><a href="/servlet/linkout?suffix=cit5l&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DWeerawarna%26aufirst%3DP.%2BM.%26aulast%3DPrior%26aufirst%3DA.%2BM.%26aulast%3DUy%26aufirst%3DR.%2BA.%2BZ.%26aulast%3DAravapalli%26aufirst%3DS.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DLushington%26aufirst%3DG.%2BH.%26aulast%3DKim%26aufirst%3DY.%2Bj.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DChang%26aufirst%3DK.%2BO.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26atitle%3DMacrocyclic%2520inhibitors%2520of%25203C%2520and%25203C-like%2520proteases%2520of%2520picornavirus%252C%2520norovirus%252C%2520and%2520coronavirus%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3709%26epage%3D3712%26doi%3D10.1016%2Fj.bmcl.2013.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batugo, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 6. Structure-activity studies of orally bioavailable, 2-Pyridone-containing peptidomimetics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1607</span>– <span class="NLM_lpage">1623</span>, <span class="refDoi"> DOI: 10.1021/jm010469k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010469k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhslegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1607-1623&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=E.+L.+Brownauthor=F.+C.+Maldonadoauthor=S.+A.+Fuhrmanauthor=L.+S.+Zalmanauthor=T.+Tuntlandauthor=C.+A.+Leeauthor=A.+K.+Patickauthor=D.+A.+Matthewsauthor=T.+F.+Hendricksonauthor=M.+B.+Kosaauthor=B.+Liuauthor=M.+R.+Batugoauthor=J.+R.+Gleesonauthor=S.+K.+Sakataauthor=L.+J.+Chenauthor=M.+C.+Guzmanauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=S.+T.+Worland&title=Structure-based+design%2C+synthesis%2C+and+biological+evaluation+of+irreversible+human+rhinovirus+3C+protease+inhibitors.+6.+Structure-activity+studies+of+orally+bioavailable%2C+2-Pyridone-containing+peptidomimetics&doi=10.1021%2Fjm010469k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 6. Structure-Activity Studies of Orally Bioavailable, 2-Pyridone-Containing Peptidomimetics</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Brown, Edward L.; Maldonado, Fausto C.; Fuhrman, Shella A.; Zalman, Leora S.; Tuntland, Tove; Lee, Caroline A.; Patick, Amy K.; Matthews, David A.; Hendrickson, Thomas F.; Kosa, Maha B.; Liu, Bo; Batugo, Minerva R.; Gleeson, Jean-Paul R.; Sakata, Sylvie K.; Chen, Lijian; Guzman, Mark C.; Meador, James W., III; Ferre, Rose Ann; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1607-1623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various 2-pyridone-contg. human rhinovirus (HRV) 3C protease (3CP) inhibitors are described.  These compds. are comprised of a peptidomimetic binding determinant and a Michael acceptor moiety, which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The 2-pyridone-contg. inhibitors typically display improved 3CP inhibition properties relative to related peptide-derived mols. along with more favorable antiviral properties.  The cocrystal structure of one pyridone-derived 3CP inhibitor, peptidomimetic I, complexed with HRV-2 3CP is also described along with certain ab initio conformation analyses.  Optimization of the 2-pyridone-contg. compds. is shown to provide several highly active 3CP inhibitors (kobs/[I] > 500 000 M-1 s-1) that function as potent antirhinoviral agents (EC50 = <0.05 μM) against multiple virus serotypes in cell culture.  One 2-pyridone-contg. 3CP inhibitor, isoxazolyl deriv. II, is shown to be bioavailable in the dog after oral dosing (F = 48%).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj9kk44U_ydLVg90H21EOLACvtfcHk0lir_BigNBNasw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhslegs7c%253D&md5=e15f157fa1af06cac037fc39f2a36d5b</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1021%2Fjm010469k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010469k%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMaldonado%26aufirst%3DF.%2BC.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBatugo%26aufirst%3DM.%2BR.%26aulast%3DGleeson%26aufirst%3DJ.%2BR.%26aulast%3DSakata%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DL.%2BJ.%26aulast%3DGuzman%26aufirst%3DM.%2BC.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520irreversible%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors.%25206.%2520Structure-activity%2520studies%2520of%2520orally%2520bioavailable%252C%25202-Pyridone-containing%2520peptidomimetics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1607%26epage%3D1623%26doi%3D10.1021%2Fjm010469k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span> <span> </span><span class="NLM_article-title">Rupintrivir is a promising candidate for treating severe cases of enterovirus-71 infection: Evaluation of antiviral efficacy in a murine infection model</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.antiviral.2012.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23295352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlKgt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=264-269&author=X.+Zhangauthor=Z.+Songauthor=B.+Qinauthor=X.+Zhangauthor=L.+Chenauthor=Y.+Huauthor=Z.+Yuan&title=Rupintrivir+is+a+promising+candidate+for+treating+severe+cases+of+enterovirus-71+infection%3A+Evaluation+of+antiviral+efficacy+in+a+murine+infection+model&doi=10.1016%2Fj.antiviral.2012.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Rupintrivir is a promising candidate for treating severe cases of enterovirus-71 infection: Evaluation of antiviral efficacy in a murine infection model</span></div><div class="casAuthors">Zhang, Xiaonan; Song, Zhigang; Qin, Boyin; Zhang, Xiaoling; Chen, Lixiang; Hu, Yunwen; Yuan, Zhenghong</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">264-269</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Enterovirus-71 (EV71) infections can cause life-threatening diseases with neurol. symptoms.  Currently, no direct targeting antivirals are available to combat severe EV71 infection.  Rupintrivir (AG7088) is a compd. originally designed for Rhinovirus 3C protease.  Previous computational analyses by us and crystallog. studies by others suggested that rupintrivir is also a high affinity inhibitor to EV71 3C.  Thus, we aimed to further evaluate its anti-EV71 activity in vivo at clin. acceptable doses.  It was obsd. that administration of rupintrivir in suckling mice largely protected them from limb paralysis and dramatically improved survival (38.5% DMSO vs. 90.9% at 0.1 mg/kg, p = 0.006).  Histol., immunohistochem. and quant. RT-PCR analyses confirmed that rupintrivir profoundly alleviated virus induced necrotizing myositis, suppressed viral RNA and blocked EV71 VP1 expression in various tissues.  In conclusion, we established that rupintrivir can strongly contain the spread of EV71 infection in vivo at a clin. acceptable dose (as low as 0.1 mg/kg).  As its safety has been fully tested in previous clin. trials, rupintrivir is suitable for immediate evaluation of potential benefits in EV71-infected individuals with life-threatening neurol. symptoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgPcXAts672rVg90H21EOLACvtfcHk0lgZ-LK3AdrNBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlKgt7k%253D&md5=53409a7532cd1692ed426e59f5b24534</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DQin%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DZ.%26atitle%3DRupintrivir%2520is%2520a%2520promising%2520candidate%2520for%2520treating%2520severe%2520cases%2520of%2520enterovirus-71%2520infection%253A%2520Evaluation%2520of%2520antiviral%2520efficacy%2520in%2520a%2520murine%2520infection%2520model%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D97%26spage%3D264%26epage%3D269%26doi%3D10.1016%2Fj.antiviral.2012.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diem, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2444</span>– <span class="NLM_lpage">2450</span>, <span class="refDoi"> DOI: 10.1128/AAC.43.10.2444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.43.10.2444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10508022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=2444-2450&author=A.+K.+Patickauthor=S.+L.+Binfordauthor=M.+A.+Brothersauthor=R.+L.+Jacksonauthor=C.+E.+Fordauthor=M.+D.+Diemauthor=F.+Maldonadoauthor=P.+S.+Dragovichauthor=R.+Zhouauthor=T.+J.+Prinsauthor=S.+A.+Fuhrmanauthor=J.+W.+Meadorauthor=L.+S.+Zalmanauthor=D.+A.+Matthewsauthor=S.+T.+Worland&title=In+vitro+antiviral+activity+of+AG7088%2C+a+potent+inhibitor+of+human+rhinovirus+3C+protease&doi=10.1128%2FAAC.43.10.2444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span></div><div class="casAuthors">Patick, A. K.; Binford, S. L.; Brothers, M. A.; Jackson, R. L.; Ford, C. E.; Diem, M. D.; Maldonado, F.; Dragovich, P. S.; Zhou, R.; Prins, T. J.; Fuhrman, S. A.; Meador, J. W.; Zalman, L. S.; Matthews, D. A.; Worland, S. T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2444-2450</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">AG-7088 is a potent, irreversible inhibitor of human rhinovirus (HRV) 3C protease (inactivation rate const. kobs/[I] = 1,470,000±440,000 M-1 s-1 for HRV 14) that was discovered by protein structure-based drug design methodologies.  In H1-HeLa and MRC-5 cell protection assays, AG-7088 inhibited the replication of all HRV serotypes (48 of 48) tested with a mean 50% EC50 of 0.023 μM (range 0.003-0.081 μM) and a mean EC90 of 0.082 μM (range 0.018-0.261 μM), as well as that of related picornaviruses including coxsackieviruses A21 and B3, enterovirus 70, and echovirus 11.  No decreases in the antiviral activity of AG-7088 were obsd. when assays were performed in the presence of α1-acid glycoprotein or mucin proteins found in nasal secretions.  The 50% cytotoxic concn. of AG-7088 was >1000 μM, yielding a therapeutic index of >12,346 to >333,333.  In a single-cycle, time-of-addn. assay, AG-7088 had antiviral activity when added up to 6 h after infection.  A compd. targeting viral attachment and/or uncoating (WIN-51711) was effective only when added at the initiation of virus infection.  Direct inhibition of 3C proteolytic activity in infected cells treated with AG-7088 was demonstrated by SDS-polyacrylamide gel electrophoresis anal. of radiolabeled proteins, which showed a dose-dependent accumulation of viral precursor polyproteins and a decrease of processed protein products.  The broad spectrum of antiviral activity of AG-7088, combined with its efficacy even when added late in the virus life cycle, highlights the advantages of 3C protease as a target and suggests that AG7088 is a promising clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojpMqAUz2M3LVg90H21EOLACvtfcHk0lgZ-LK3AdrNBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D&md5=70e98480d38e133c96866005b0578f75</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.10.2444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.10.2444%26sid%3Dliteratum%253Aachs%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DJackson%26aufirst%3DR.%2BL.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DDiem%26aufirst%3DM.%2BD.%26aulast%3DMaldonado%26aufirst%3DF.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520of%2520AG7088%252C%2520a%2520potent%2520inhibitor%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D2444%26epage%3D2450%26doi%3D10.1128%2FAAC.43.10.2444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of enterovirus 71 3C protease complexed with rupintrivir reveal the roles of catalytically important residues</span>. <i>J. Virology.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">10021</span>– <span class="NLM_lpage">10030</span>, <span class="refDoi"> DOI: 10.1128/JVI.05107-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.05107-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21813612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=10021-10030&author=J.+Wangauthor=T.+Fanauthor=X.+Yaoauthor=Z.+Wuauthor=L.+Guoauthor=L.+Leiauthor=J.+Wangauthor=M.+Wangauthor=Q.+Jinauthor=S.+Cui&title=Crystal+structures+of+enterovirus+71+3C+protease+complexed+with+rupintrivir+reveal+the+roles+of+catalytically+important+residues&doi=10.1128%2FJVI.05107-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6dR"><div class="casContent"><span class="casTitleNuber">6d</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of enterovirus 71 3C protease complexed with rupintrivir reveal the roles of catalytically important residues</span></div><div class="casAuthors">Wang, Jing; Fan, Tingting; Yao, Xue; Wu, Zhiqiang; Guo, Li; Lei, Xiaobo; Wang, Jianwei; Wang, Meitian; Jin, Qi; Cui, Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10021-10030</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) is the primary pathogenic cause of hand-foot-mouth disease (HFMD), but an effective antiviral drug currently is unavailable.  Rupintrivir, an inhibitor against human rhinovirus (HRV), has potent antiviral activities against EV71.  We detd. the high-resoln. crystal structures of the EV71 3Cpro/rupintrivir complex, showing that although rupintrivir interacts with EV71 3Cpro similarly to HRV 3Cpro, the C-terminus of the inhibitor cannot accommodate the leaving-group pockets of EV71 3Cpro.  Our structures reveal that EV71 3Cpro possesses a surface-recessive S2' pocket that is not present in HRV 3Cpro that contributes to the addnl. substrate binding affinity.  Combined with mutagenic studies, we demonstrated that catalytic Glu71 is irreplaceable for maintaining the overall architecture of the active site and, most importantly, the productive conformation of catalytic His40.  We discovered the role of a previously uncharacterized residue, Arg39 of EV71 3Cpro, that can neutralize the neg. charge of Glu71, which may subsequently assist deprotonation of His40 during proteolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfsf-EHIhPLVg90H21EOLACvtfcHk0li6bM13Wz-s0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurzI&md5=1893b96430b7f6f3d0992916bb61b8a5</span></div><a href="/servlet/linkout?suffix=cit6d&amp;dbid=16384&amp;doi=10.1128%2FJVI.05107-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.05107-11%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DCui%26aufirst%3DS.%26atitle%3DCrystal%2520structures%2520of%2520enterovirus%252071%25203C%2520protease%2520complexed%2520with%2520rupintrivir%2520reveal%2520the%2520roles%2520of%2520catalytically%2520important%2520residues%26jtitle%3DJ.%2520Virology.%26date%3D2011%26volume%3D85%26spage%3D10021%26epage%3D10030%26doi%3D10.1128%2FJVI.05107-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">Enterovirus 71 and Coxsackievirus A16 3C proteases: binding to Rupintrivir and their substrates and anti-hand, foot, and mouth disease virus drug design</span>. <i>J. Virology.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">10319</span>– <span class="NLM_lpage">10331</span>, <span class="refDoi"> DOI: 10.1128/JVI.00787-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.00787-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21795339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=10319-10331&author=G.+Luauthor=J.+Qiauthor=Z.+Chenauthor=X.+Xuauthor=F.+Gaoauthor=D.+Linauthor=W.+Qianauthor=H.+Liuauthor=H.+Jiangauthor=J.+Yanauthor=G.+F.+Gao&title=Enterovirus+71+and+Coxsackievirus+A16+3C+proteases%3A+binding+to+Rupintrivir+and+their+substrates+and+anti-hand%2C+foot%2C+and+mouth+disease+virus+drug+design&doi=10.1128%2FJVI.00787-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6eR"><div class="casContent"><span class="casTitleNuber">6e</span><div class="casTitle"><span class="NLM_cas:atitle">Enterovirus 71 and coxsackievirus A16 3C proteases: binding to rupintrivir and their substrates and anti-hand, foot, and mouth disease virus drug design</span></div><div class="casAuthors">Lu, Guangwen; Qi, Jianxun; Chen, Zhujun; Xu, Xiang; Gao, Feng; Lin, Daizong; Qian, Wangke; Liu, Hong; Jiang, Hualiang; Yan, Jinghua; Gao, George F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10319-10331</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) and coxsackievirus A16 (CVA16) are the major causative agents of hand, foot, and mouth disease (HFMD), which is prevalent in Asia.  Thus far, there are no prophylactic or therapeutic measures against HFMD.  The 3C proteases from EV71 and CVA16 play important roles in viral replication and are therefore ideal drug targets.  By using biochem., mutational, and structural approaches, we broadly characterized both proteases.  A series of high-resoln. structures of the free or substrate-bound enzymes were solved.  These structures, together with our cleavage specificity assay, well explain the marked substrate preferences of both proteases for particular P4, P1, and P1' residue types, as well as the relative malleability of the P2 amino acid.  More importantly, the complex structures of EV71 and CVA16 3Cs with rupintrivir, a specific human rhinovirus (HRV) 3C protease inhibitor, were solved.  These structures reveal a half-closed S2 subsite and a size-reduced S1' subsite that limit the access of the P1' group of rupintrivir to both enzymes, explaining the reported low inhibition activity of the compd. toward EV71 and CVA16.  In conclusion, the detailed characterization of both proteases in this study could direct us to a proposal for rational design of EV71/CVA16 3C inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD7JDuc7TN2rVg90H21EOLACvtfcHk0li6bM13Wz-s0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurvI&md5=9e9cee092cfbc470f57ecaddd32064e4</span></div><a href="/servlet/linkout?suffix=cit6e&amp;dbid=16384&amp;doi=10.1128%2FJVI.00787-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00787-11%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DQian%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DEnterovirus%252071%2520and%2520Coxsackievirus%2520A16%25203C%2520proteases%253A%2520binding%2520to%2520Rupintrivir%2520and%2520their%2520substrates%2520and%2520anti-hand%252C%2520foot%252C%2520and%2520mouth%2520disease%2520virus%2520drug%2520design%26jtitle%3DJ.%2520Virology.%26date%3D2011%26volume%3D85%26spage%3D10319%26epage%3D10331%26doi%3D10.1128%2FJVI.00787-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span> <span> </span><span class="NLM_article-title">Structures of Enterovirus 71 3C proteinase (strain E2004104-TW-CDC) and its complex with rupintrivir</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1107/S0907444913002862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444913002862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23633597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvVSnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=866-871&author=C.+Wuauthor=Q.+Caiauthor=C.+Chenauthor=N.+Liauthor=X.+Pengauthor=Y.+Caiauthor=K.+Yinauthor=X.+Chenauthor=X.+Wangauthor=R.+Zhangauthor=L.+Liuauthor=S.+Chenauthor=J.+Liauthor=T.+Lin&title=Structures+of+Enterovirus+71+3C+proteinase+%28strain+E2004104-TW-CDC%29+and+its+complex+with+rupintrivir&doi=10.1107%2FS0907444913002862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6fR"><div class="casContent"><span class="casTitleNuber">6f</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of Enterovirus 71 3C proteinase (strain E2004104-TW-CDC) and its complex with rupintrivir</span></div><div class="casAuthors">Wu, Caiming; Cai, Qixu; Chen, Chen; Li, Ning; Peng, Xuanjia; Cai, Yaxian; Yin, Ke; Chen, Xinsheng; Wang, Xiaolong; Zhang, Rongfu; Liu, Lijie; Chen, Shuhui; Li, Jian; Lin, Tianwei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">866-871</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The crystal structure of 3C proteinase (3Cpro) from Enterovirus 71 (EV71) was detd. in space group C2221 to 2.2 Å resoln.  The fold was similar to that of 3Cpro from other picornaviruses, but the difference in the β-ribbon reported in a previous structure was not obsd.  This β-ribbon was folded over the substrate-binding cleft and constituted part of the essential binding sites for interaction with the substrate.  The structure of its complex with rupintrivir (AG7088), a peptidomimetic inhibitor, was also characterized in space group P212121 to 1.96 Å resoln.  The inhibitor was accommodated without any spatial hindrance despite the more constricted binding site; this was confirmed by functional assays, in which the inhibitor showed comparable potency towards EV71 3Cpro and human rhinovirus 3Cpro, which is the target that rupintrivir was designed against.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6W5v6O6LT-bVg90H21EOLACvtfcHk0li6bM13Wz-s0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvVSnt7Y%253D&md5=53be6c2cf82e5abcd9dbbe11c2ef370d</span></div><a href="/servlet/linkout?suffix=cit6f&amp;dbid=16384&amp;doi=10.1107%2FS0907444913002862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913002862%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DT.%26atitle%3DStructures%2520of%2520Enterovirus%252071%25203C%2520proteinase%2520%2528strain%2520E2004104-TW-CDC%2529%2520and%2520its%2520complex%2520with%2520rupintrivir%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D866%26epage%3D871%26doi%3D10.1107%2FS0907444913002862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit6g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human enterovirus 71 3C protease</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>408</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.jmb.2011.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21396941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKgsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=408&publication_year=2011&pages=449-461&author=S.+Cuiauthor=J.+Wangauthor=T.+Fanauthor=B.+Qinauthor=L.+Guoauthor=X.+Leiauthor=J.+Wangauthor=M.+Wangauthor=Q.+Jin&title=Crystal+structure+of+human+enterovirus+71+3C+protease&doi=10.1016%2Fj.jmb.2011.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6gR"><div class="casContent"><span class="casTitleNuber">6g</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Human Enterovirus 71 3C Protease</span></div><div class="casAuthors">Cui, Sheng; Wang, Jing; Fan, Tingting; Qin, Bo; Guo, Li; Lei, Xiaobo; Wang, Jianwei; Wang, Meitian; Jin, Qi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">408</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">449-461</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human enterovirus 71 (EV71) is the major pathogen that causes hand, foot and mouth disease that particularly affects young children.  Growing hand, foot and mouth disease outbreaks were obsd. worldwide in recent years and caused devastating losses both economically and politically.  However, vaccines or effective drugs are unavailable to date.  The genome of EV71 consists of a pos. sense, single-stranded RNA of ∼ 7400 bp, encoding a large precursor polyprotein that requires proteolytic processing to generate mature viral proteins.  The proteolytic processing mainly depends on EV71 3C protease (3Cpro) that possesses both proteolysis and RNA binding activities, which enable the protease to perform multiple tasks in viral replication and pathogen-host interactions.  The central roles played by EV71 3Cpro make it an appealing target for antiviral drug development.  We detd. the first crystal structure of EV71 3Cpro and analyzed its enzymic activity.  The crystal structure shows that EV71 3Cpro has a typical chymotrypsin-like fold that is common in picornaviral 3Cpro.  Strikingly, we found an important surface loop, also denoted as β-ribbon, which adopts a novel open conformation in EV71 3Cpro.  We identified two important residues located at the base of the β-ribbon, Gly123 and His133, which form hinges that govern the intrinsic flexibility of the ribbon.  Structure-guided mutagenesis studies revealed that the hinge residues are important to EV71 3Cpro proteolytic activities.  In summary, our work provides the first structural insight into EV71 3Cpro, including a mobile β-ribbon, which is relevant to the proteolytic mechanism.  Our data also provides a framework for structure-guided inhibitor design against EV71 3Cpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQpc2_sfZjJrVg90H21EOLACvtfcHk0lgrlF2YzZVOUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKgsr0%253D&md5=9b9c91734cbf576138a680635bc23708</span></div><a href="/servlet/linkout?suffix=cit6g&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DT.%26aulast%3DQin%26aufirst%3DB.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DQ.%26atitle%3DCrystal%2520structure%2520of%2520human%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D408%26spage%3D449%26epage%3D461%26doi%3D10.1016%2Fj.jmb.2011.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potts, B. C. M.</span></span> <span> </span><span class="NLM_article-title">Liquid Chromatography-Mass spectrometry and liquid Chromatography-NMR characterization of in vitro metabolites of a potent and irreversible peptidomimetic inhibitor of rhinovirus 3C protease</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">734</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=11302940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtF2jur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=729-734&author=K.+E.+Zhangauthor=B.+Heeauthor=C.+A.+Leeauthor=B.+Liangauthor=B.+C.+M.+Potts&title=Liquid+Chromatography-Mass+spectrometry+and+liquid+Chromatography-NMR+characterization+of+in+vitro+metabolites+of+a+potent+and+irreversible+peptidomimetic+inhibitor+of+rhinovirus+3C+protease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Liquid chromatography-mass spectrometry and liquid chromatography-NMR characterization of in vitro metabolites of a potent and irreversible peptidomimetic inhibitor of rhinovirus 3C protease</span></div><div class="casAuthors">Zhang, Kanyin E.; Hee, Brian; Lee, Caroline A.; Liang, Baihong; Potts, Barbara C. M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">729-734</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In vitro metab. of AG7088 [trans-(4S,2'R,5'S,3"'S)-4-{2'-4-(4-fluorobenzyl)-6'-methyl-5'-[(5"-methylisoxazole-3"-carbonylamino]-4-oxoheptanoylamino}-5-(2"'-oxopyrrolidin-3-"'-yl)pent-2-enoic acid Et ester] was studied in liver microsomes isolated from mice, rats, rabbits, dogs, monkeys, and humans.  The structures of the metabolites were characterized by liq. chromatog. (LC)-tandem mass spectrometry and LC-NMR methods.  Hydrolysis of the Et ester to produce metabolite M4 (AG7185) is the predominant pathway in all species, with the greatest activity obsd. in rodents and rabbits, followed by monkeys, dogs, and humans.  Several hydroxylation products were identified as minor metabolites, including diastereomers M1 and M2, with a hydroxy group at the P1-lactam moiety, and M3, with a hydroxy group at the Me position of the methylisoxazole ring.  Rodent and rabbit liver microsomes formed almost exclusively the acid metabolite M4 (AG7185), with very little hydroxylated metabolites, whereas monkey liver microsomes formed more secondary metabolites (i.e., acid analogs of the hydroxylated metabolites).  The overall metabolic profile of AG7088 formed in dog liver microsomes closely resembled that of human liver microsomes; therefore, this species may be the most appropriate animal model relative to humans for exposure to AG7088 and its metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlwGOmiSAbBLVg90H21EOLACvtfcHk0lgrlF2YzZVOUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtF2jur4%253D&md5=263c089ccac01ee70fdb383fa958903d</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%2BE.%26aulast%3DHee%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DLiang%26aufirst%3DB.%26aulast%3DPotts%26aufirst%3DB.%2BC.%2BM.%26atitle%3DLiquid%2520Chromatography-Mass%2520spectrometry%2520and%2520liquid%2520Chromatography-NMR%2520characterization%2520of%2520in%2520vitro%2520metabolites%2520of%2520a%2520potent%2520and%2520irreversible%2520peptidomimetic%2520inhibitor%2520of%2520rhinovirus%25203C%2520protease%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2001%26volume%3D29%26spage%3D729%26epage%3D734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marakovits, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejto, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harr, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 4. incorporation of P1 lactam moieties as L-glutamine replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1224</span>, <span class="refDoi"> DOI: 10.1021/jm9805384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9805384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1213-1224&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=S.+E.+Webberauthor=J.+T.+Marakovitsauthor=S.+A.+Fuhrmanauthor=A.+K.+Patickauthor=D.+A.+Matthewsauthor=C.+A.+Leeauthor=C.+E.+Fordauthor=B.+J.+Burkeauthor=P.+A.+Rejtoauthor=T.+F.+Hendricksonauthor=T.+Tuntlandauthor=E.+L.+Brownauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=J.+E.+Harrauthor=M.+B.+Kosaauthor=S.+T.+Worland&title=Structure-based+design%2C+synthesis%2C+and+biological+evaluation+of+irreversible+human+rhinovirus+3C+protease+inhibitors.+4.+incorporation+of+P1+lactam+moieties+as+L-glutamine+replacements&doi=10.1021%2Fjm9805384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Webber, Stephen E.; Marakovits, Joseph T.; Fuhrman, Shella A.; Patick, Amy K.; Matthews, David A.; Lee, Caroline A.; Ford, Clifford E.; Burke, Benjamin J.; Rejto, Paul A.; Hendrickson, Thomas F.; Tuntland, Tove; Brown, Edward L.; Meador, James W., III; Ferre, Rose Ann; Harr, James E. V.; Kosa, Maha B.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1224</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various human rhinovirus (HRV) 3C protease (3CP) inhibitors, e.g. I [R = CH2CHMe2, CHMe2; X = (S)-CH, (R)-CH, N; n = 1, 2] and II (R = CHMe2, CMe3; Y = NH, CH2) which incorporate P1 lactam moieties in lieu of an L-glutamine residue are described.  These compds. are comprised of a tripeptidyl or peptidomimetic binding determinant and an Et propenoate Michael acceptor moiety which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The P1-lactam-contg. inhibitors display significantly increased 3CP inhibition activity along with improved antirhinoviral properties relative to corresponding L-glutamine-derived mols.  In addn., several lactam-contg. compds. exhibit excellent selectivity for HRV 3CP over several other serine and cysteine proteases and are not appreciably degraded by a variety of biol. agents.  Lactam II (R = CHMe2, Y = CH2, n = 1) (AG7088) was one of the most potent inhibitors (mean antirhinoviral EC90 ≈ 0.10 μM, n = 46 serotypes), and is shown to warrant addnl. preclin. development to explore its potential for use as an antirhinoviral agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlBuxFPwMr97Vg90H21EOLACvtfcHk0lgrlF2YzZVOUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D&md5=0e2bf5a03d17ea371663a252e7b97b20</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1021%2Fjm9805384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9805384%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%2BT.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBurke%26aufirst%3DB.%2BJ.%26aulast%3DRejto%26aufirst%3DP.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DHarr%26aufirst%3DJ.%2BE.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520irreversible%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors.%25204.%2520incorporation%2520of%2520P1%2520lactam%2520moieties%2520as%2520L-glutamine%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1213%26epage%3D1224%26doi%3D10.1021%2Fjm9805384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit7c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pithavala, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penning, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, B. M.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetics and safety of an antirhinoviral agent, Ruprintrivir, in healthy volunteers</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">392</span>– <span class="NLM_lpage">397</span>, <span class="refDoi"> DOI: 10.1128/AAC.46.2.392-397.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.46.2.392-397.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=11796347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFyisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=392-397&author=P.+H.+Hsyuauthor=Y.+K.+Pithavalaauthor=M.+Gerstenauthor=C.+A.+Penningauthor=B.+M.+Kerr&title=Pharmacokinetics+and+safety+of+an+antirhinoviral+agent%2C+Ruprintrivir%2C+in+healthy+volunteers&doi=10.1128%2FAAC.46.2.392-397.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7cR"><div class="casContent"><span class="casTitleNuber">7c</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetics and safety of an antirhinoviral agent, ruprintrivir, in healthy volunteers</span></div><div class="casAuthors">Hsyu, Poe-Hirr; Pithavala, Yazdi K.; Gersten, Merril; Penning, Carol A.; Kerr, Bradley M.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">392-397</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A single-dose study and a multiple-dose study of the safety and pharmacokinetics of ruprintrivir, a new selective irreversible inhibitor of human rhinovirus 3C protease, were conducted with healthy adult volunteers.  Both studies were double-blind, randomized, placebo-controlled, parallel-group investigations of ruprintrivir administered intranasally at two dose levels.  The parent drug and its acid metabolite, AG7185, were measured in plasma samples and nasal washings, and the safety of the treatments was monitored.  Intranasal ruprintrivir, administered as single doses of 4 and 8 mg or every 3 h, six times per day, for 7 days was safe and well tolerated.  Adverse events were mild, short-lived, and confined to the upper respiratory tract (i.e., nose and throat, taste and smell perceptions).  Adverse events were similar after placebo and after single or multiple doses of active drug.  Systemic exposure to ruprintrivir was rarely detectable with the highest measured concn. of ≤0.52 ng/mL; the assay had a lower limit of quantification of 0.2 ng/mL.  Systemic exposure to the metabolite was also low, with a highest measured concn. of 3.25 ng/mL.  Concns. of AG7185 obsd. during multiple dosing were higher than those obsd. after the first dose but were no more than predicted from the single-dose study.  Substantial amts. of ruprintrivir were obsd. intranasally for at least 9 h after multiple doses of ruprintrivir.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou7tDU2Prh-bVg90H21EOLACvtfcHk0lgE7zu4WdZ49A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFyisw%253D%253D&md5=dffd842fdf9f88226661fbc96ae77b1c</span></div><a href="/servlet/linkout?suffix=cit7c&amp;dbid=16384&amp;doi=10.1128%2FAAC.46.2.392-397.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.46.2.392-397.2002%26sid%3Dliteratum%253Aachs%26aulast%3DHsyu%26aufirst%3DP.%2BH.%26aulast%3DPithavala%26aufirst%3DY.%2BK.%26aulast%3DGersten%26aufirst%3DM.%26aulast%3DPenning%26aufirst%3DC.%2BA.%26aulast%3DKerr%26aufirst%3DB.%2BM.%26atitle%3DPharmacokinetics%2520and%2520safety%2520of%2520an%2520antirhinoviral%2520agent%252C%2520Ruprintrivir%252C%2520in%2520healthy%2520volunteers%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2002%26volume%3D46%26spage%3D392%26epage%3D397%26doi%3D10.1128%2FAAC.46.2.392-397.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit7d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shie, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jan, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic α, β-unsaturated esters</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">5240</span>– <span class="NLM_lpage">5252</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2005.05.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmc.2005.05.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15994085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVKhtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=5240-5252&author=J.+J.+Shieauthor=J.+M.+Fangauthor=T.+H.+Kuoauthor=C.+J.+Kuoauthor=P.+H.+Liangauthor=H.+J.+Huangauthor=Y.+T.+Wuauthor=J.+T.+Janauthor=Y.+S.+E.+Chengauthor=C.+H.+Wong&title=Inhibition+of+the+severe+acute+respiratory+syndrome+3CL+protease+by+peptidomimetic+%CE%B1%2C+%CE%B2-unsaturated+esters&doi=10.1016%2Fj.bmc.2005.05.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7dR"><div class="casContent"><span class="casTitleNuber">7d</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic α,β-unsaturated esters</span></div><div class="casAuthors">Shie, Jiun-Jie; Fang, Jim-Min; Kuo, Tun-Hsun; Kuo, Chih-Jung; Liang, Po-Huang; Huang, Hung-Jyun; Wu, Yin-Ta; Jan, Jia-Tsrong; Cheng, Yih-Shyun E.; Wong, Chi-Huey</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5240-5252</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The proteolytic processing of polyproteins by the 3CL protease of severe acute respiratory syndrome coronavirus is essential for the viral propagation.  A series of tripeptide α,β-unsatd. esters and ketomethylene isosteres, including AG7088, are synthesized and assayed to target the 3CL protease.  Though AG7088 is inactive (IC50 > 100 μM), the ketomethylene isosteres and tripeptide α,β-unsatd. esters contg. both P1 and P2 phenylalanine residues show modest inhibitory activity (IC50 = 11-39 μM).  The Phe-Phe dipeptide inhibitors are designed on the basis of computer modeling of the enzyme-inhibitor complex.  The most potent inhibitor (I) with an inhibition const. of 0.52 μM is obtained by condensation of the Phe-Phe dipeptide α,β-unsatd. ester with 4-(dimethylamino)cinnamic acid.  The cell-based assays also indicate that I is a nontoxic anti-SARS agent with an EC50 value of 0.18 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWHJTRZOlTybVg90H21EOLACvtfcHk0lgE7zu4WdZ49A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVKhtb4%253D&md5=c2dea03aff1915bda613a491663ab58d</span></div><a href="/servlet/linkout?suffix=cit7d&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2005.05.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2005.05.065%26sid%3Dliteratum%253Aachs%26aulast%3DShie%26aufirst%3DJ.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%2BM.%26aulast%3DKuo%26aufirst%3DT.%2BH.%26aulast%3DKuo%26aufirst%3DC.%2BJ.%26aulast%3DLiang%26aufirst%3DP.%2BH.%26aulast%3DHuang%26aufirst%3DH.%2BJ.%26aulast%3DWu%26aufirst%3DY.%2BT.%26aulast%3DJan%26aufirst%3DJ.%2BT.%26aulast%3DCheng%26aufirst%3DY.%2BS.%2BE.%26aulast%3DWong%26aufirst%3DC.%2BH.%26atitle%3DInhibition%2520of%2520the%2520severe%2520acute%2520respiratory%2520syndrome%25203CL%2520protease%2520by%2520peptidomimetic%2520%25CE%25B1%252C%2520%25CE%25B2-unsaturated%2520esters%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2005%26volume%3D13%26spage%3D5240%26epage%3D5252%26doi%3D10.1016%2Fj.bmc.2005.05.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z. L.</span></span> <span> </span><span class="NLM_article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2012-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41586-020-2012-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32015507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=270-273&author=P.+Zhouauthor=X.+L.+Yangauthor=X.+G.+Wangauthor=B.+Huauthor=L.+Zhangauthor=W.+Zhangauthor=H.+R.+Siauthor=Y.+Zhuauthor=B.+Liauthor=C.+L.+Huangauthor=H.+D.+Chenauthor=J.+Chenauthor=Y.+Luoauthor=H.+Guoauthor=R.+D.+Jiangauthor=M.+Q.+Liuauthor=Y.+Chenauthor=X.+R.+Shenauthor=X.+Wangauthor=X.+S.+Zhengauthor=K.+Zhaoauthor=Q.+J.+Chenauthor=F.+Dengauthor=L.+L.+Liuauthor=B.+Yanauthor=F.+X.+Zhanauthor=Y.+Y.+Wangauthor=G.+F.+Xiaoauthor=Z.+L.+Shi&title=A+pneumonia+outbreak+associated+with+a+new+coronavirus+of+probable+bat+origin&doi=10.1038%2Fs41586-020-2012-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span></div><div class="casAuthors">Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">270-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Since the outbreak of severe acute respiratory syndrome (SARS) 18 years ago, a large no. of SARS-related coronaviruses (SARSr-CoVs) have been discovered in their natural reservoir host, bats1-4.  Previous studies have shown that some bat SARSr-CoVs have the potential to infect humans5-7.  Here we report the identification and characterization of a new coronavirus (2019-nCoV), which caused an epidemic of acute respiratory syndrome in humans in Wuhan, China.  The epidemic, which started on 12 Dec. 2019, had caused 2,794 lab.-confirmed infections including 80 deaths by 26 Jan. 2020.  Full-length genome sequences were obtained from five patients at an early stage of the outbreak.  The sequences are almost identical and share 79.6% sequence identity to SARS-CoV.  Furthermore, we show that 2019-nCoV is 96% identical at the whole-genome level to a bat coronavirus.  Pairwise protein sequence anal. of seven conserved non-structural proteins domains show that this virus belongs to the species of SARSr-CoV.  In addn., 2019-nCoV virus isolated from the bronchoalveolar lavage fluid of a critically ill patient could be neutralized by sera from several patients.  Notably, we confirmed that 2019-nCoV uses the same cell entry receptor-angiotensin converting enzyme II (ACE2)-as SARS-CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_Wb_iUsbFrVg90H21EOLACvtfcHk0lgE7zu4WdZ49A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D&md5=236f17d4d3c7978d72513e5e0258f1b3</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2012-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2012-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DX.%2BG.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSi%26aufirst%3DH.%2BR.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.%2BL.%26aulast%3DChen%26aufirst%3DH.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DR.%2BD.%26aulast%3DLiu%26aufirst%3DM.%2BQ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DX.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DQ.%2BJ.%26aulast%3DDeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DL.%2BL.%26aulast%3DYan%26aufirst%3DB.%26aulast%3DZhan%26aufirst%3DF.%2BX.%26aulast%3DWang%26aufirst%3DY.%2BY.%26aulast%3DXiao%26aufirst%3DG.%2BF.%26aulast%3DShi%26aufirst%3DZ.%2BL.%26atitle%3DA%2520pneumonia%2520outbreak%2520associated%2520with%2520a%2520new%2520coronavirus%2520of%2520probable%2520bat%2520origin%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D270%26epage%3D273%26doi%3D10.1038%2Fs41586-020-2012-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">A new coronavirus associated with human respiratory disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.+M.+Chenauthor=W.+Wangauthor=Z.+G.+Songauthor=Y.+Huauthor=Z.+W.+Taoauthor=J.+H.+Tianauthor=Y.+Y.+Peiauthor=M.+L.+Yuanauthor=Y.+L.+Zhangauthor=F.+H.+Daiauthor=Y.+Liuauthor=Q.+M.+Wangauthor=J.+J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.+Z.+Zhang&title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0liZ6m-Clb0skg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.%2BM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.%2BG.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.%2BW.%26aulast%3DTian%26aufirst%3DJ.%2BH.%26aulast%3DPei%26aufirst%3DY.%2BY.%26aulast%3DYuan%26aufirst%3DM.%2BL.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DDai%26aufirst%3DF.%2BH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%2BM.%26aulast%3DZheng%26aufirst%3DJ.%2BJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.%2BZ.%26atitle%3DA%2520new%2520coronavirus%2520associated%2520with%2520human%2520respiratory%2520disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chinazzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioannini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litvinova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastore y Piontti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viboud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halloran, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longini, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vespignani, A.</span></span> <span> </span><span class="NLM_article-title">The effect of travel restrictions on the spread of the 2019 novel coronavirus (COVID-19) outbreak</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1126/science.aba9757</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.aba9757" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32144116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=395-400&author=M.+Chinazziauthor=J.+T.+Davisauthor=M.+Ajelliauthor=C.+Gioanniniauthor=M.+Litvinovaauthor=S.+Merlerauthor=A.+Pastore+y+Pionttiauthor=K.+Muauthor=L.+Rossiauthor=K.+Sunauthor=C.+Viboudauthor=X.+Xiongauthor=H.+Yuauthor=M.+E.+Halloranauthor=I.+M.+Longiniauthor=A.+Vespignani&title=The+effect+of+travel+restrictions+on+the+spread+of+the+2019+novel+coronavirus+%28COVID-19%29+outbreak&doi=10.1126%2Fscience.aba9757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8cR"><div class="casContent"><span class="casTitleNuber">8c</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of travel restrictions on the spread of the 2019 novel coronavirus (COVID-19) outbreak</span></div><div class="casAuthors">Chinazzi, Matteo; Davis, Jessica T.; Ajelli, Marco; Gioannini, Corrado; Litvinova, Maria; Merler, Stefano; Pastore y Piontti, Ana; Mu, Kunpeng; Rossi, Luca; Sun, Kaiyuan; Viboud, Cecile; Xiong, Xinyue; Yu, Hongjie; Halloran, M. Elizabeth; Longini, Ira M., Jr.; Vespignani, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">395-400</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Motivated by the rapid spread of coronavirus disease 2019 (COVID-19) in mainland China, we use a global metapopulation disease transmission model to project the impact of travel limitations on the national and international spread of the epidemic.  The model is calibrated on the basis of internationally reported cases and shows that, at the start of the travel ban from Wuhan on 23 Jan. 2020, most Chinese cities had already received many infected travelers.  The travel quarantine of Wuhan delayed the overall epidemic progression by only 3 to 5 days in mainland China but had a more marked effect on the international scale, where case importations were reduced by nearly 80% until mid-Feb.  Modeling results also indicate that sustained 90% travel restrictions to and from mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher redn. of transmission in the community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhnsJ8Ikq42rVg90H21EOLACvtfcHk0liZ6m-Clb0skg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtLw%253D&md5=3d9c6e65c976c694e84837bb6f06f6ae</span></div><a href="/servlet/linkout?suffix=cit8c&amp;dbid=16384&amp;doi=10.1126%2Fscience.aba9757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aba9757%26sid%3Dliteratum%253Aachs%26aulast%3DChinazzi%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DJ.%2BT.%26aulast%3DAjelli%26aufirst%3DM.%26aulast%3DGioannini%26aufirst%3DC.%26aulast%3DLitvinova%26aufirst%3DM.%26aulast%3DMerler%26aufirst%3DS.%26aulast%3DPastore%2By%2BPiontti%26aufirst%3DA.%26aulast%3DMu%26aufirst%3DK.%26aulast%3DRossi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DK.%26aulast%3DViboud%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DHalloran%26aufirst%3DM.%2BE.%26aulast%3DLongini%26aufirst%3DI.%2BM.%26aulast%3DVespignani%26aufirst%3DA.%26atitle%3DThe%2520effect%2520of%2520travel%2520restrictions%2520on%2520the%2520spread%2520of%2520the%25202019%2520novel%2520coronavirus%2520%2528COVID-19%2529%2520outbreak%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D395%26epage%3D400%26doi%3D10.1126%2Fscience.aba9757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ledford, H.</span></span> <span> </span><span class="NLM_article-title">Coronavirus breakthrough: dexamethasone is first drug shown to save lives</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">469</span>, <span class="refDoi"> DOI: 10.1038/d41586-020-01824-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fd41586-020-01824-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32546811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1ejtrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=469&author=H.+Ledford&title=Coronavirus+breakthrough%3A+dexamethasone+is+first+drug+shown+to+save+lives&doi=10.1038%2Fd41586-020-01824-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8dR"><div class="casContent"><span class="casTitleNuber">8d</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus breakthrough: dexamethasone is first drug shown to save lives</span></div><div class="casAuthors">Ledford, Heidi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7813</span>),
    <span class="NLM_cas:pages">469</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In a large trial, a cheap and widely available steroid cut deaths by one-third among patients critically ill with COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBJB_W2-H5D7Vg90H21EOLACvtfcHk0liZ6m-Clb0skg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1ejtrbE&md5=4437eb1c5d52bde519fb55f07625f404</span></div><a href="/servlet/linkout?suffix=cit8d&amp;dbid=16384&amp;doi=10.1038%2Fd41586-020-01824-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41586-020-01824-5%26sid%3Dliteratum%253Aachs%26aulast%3DLedford%26aufirst%3DH.%26atitle%3DCoronavirus%2520breakthrough%253A%2520dexamethasone%2520is%2520first%2520drug%2520shown%2520to%2520save%2520lives%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D469%26doi%3D10.1038%2Fd41586-020-01824-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit8e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kupferschmidt, K.</span></span> <span> </span><span class="NLM_article-title">Global trial eliminates drugs, pivots to new ones</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>370</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1126/science.370.6515.388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.370.6515.388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=33093086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BB3s7ltFSjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2020&pages=388-389&author=K.+Kupferschmidt&title=Global+trial+eliminates+drugs%2C+pivots+to+new+ones&doi=10.1126%2Fscience.370.6515.388"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8eR"><div class="casContent"><span class="casTitleNuber">8e</span><div class="casTitle"><span class="NLM_cas:atitle">Global trial eliminates drugs, pivots to new ones</span></div><div class="casAuthors">Kupferschmidt Kai</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">6515</span>),
    <span class="NLM_cas:pages">388-389</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRf1iuBtpFC0OkdrW1FkXG-fW6udTcc2eZLTWYVg8s6dbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3s7ltFSjtg%253D%253D&md5=64a9d2d5006a1b3a09cb83b50e7341e9</span></div><a href="/servlet/linkout?suffix=cit8e&amp;dbid=16384&amp;doi=10.1126%2Fscience.370.6515.388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.370.6515.388%26sid%3Dliteratum%253Aachs%26aulast%3DKupferschmidt%26aufirst%3DK.%26atitle%3DGlobal%2520trial%2520eliminates%2520drugs%252C%2520pivots%2520to%2520new%2520ones%26jtitle%3DScience%26date%3D2020%26volume%3D370%26spage%3D388%26epage%3D389%26doi%3D10.1126%2Fscience.370.6515.388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahabi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Drug development and medicinal chemistry efforts toward SARS-coronavirus and Covid-19 therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fcmdc.202000223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32324951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=907-932&author=A.+K.+Ghoshauthor=M.+Brindisiauthor=D.+Shahabiauthor=M.+E.+Chapmanauthor=A.+D.+Mesecar&title=Drug+development+and+medicinal+chemistry+efforts+toward+SARS-coronavirus+and+Covid-19+therapeutics&doi=10.1002%2Fcmdc.202000223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span></div><div class="casAuthors">Ghosh, Arun K.; Brindisi, Margherita; Shahabi, Dana; Chapman, Mackenzie E.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">907-932</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 infection is spreading at an alarming rate and has created an unprecedented health emergency around the globe.  There is no effective vaccine or approved drug treatment against COVID-19 and other pathogenic coronaviruses.  The development of antiviral agents is an urgent priority.  Biochem. events crit. to the coronavirus replication cycle provided a no. of attractive targets for drug development.  These include, spike protein for binding to host cell-surface receptors, proteolytic enzymes that are essential for processing polyproteins into mature viruses, and RNA-dependent RNA polymerase for RNA replication.  There has been a lot of ground work for drug discovery and development against these targets.  Also, high-throughput screening efforts have led to the identification of diverse lead structures, including natural product-derived mols.  This review highlights past and present drug discovery and medicinal-chem. approaches against SARS-CoV, MERS-CoV and COVID-19 targets.  The review hopes to stimulate further research and will be a useful guide to the development of effective therapies against COVID-19 and other pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcgVQF9N3zf7Vg90H21EOLACvtfcHk0lhzEXiHNUM5fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D&md5=8b4c3c83c05a7b64f39ba7d803ea1709</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000223%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DShahabi%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DM.%2BE.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DDrug%2520development%2520and%2520medicinal%2520chemistry%2520efforts%2520toward%2520SARS-coronavirus%2520and%2520Covid-19%2520therapeutics%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D907%26epage%3D932%26doi%3D10.1002%2Fcmdc.202000223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved alpha-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+alpha-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0lhzEXiHNUM5fQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520alpha-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of M(pro) from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+M%28pro%29+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0liJNWmtJY-D4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520M%2528pro%2529%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit9d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.+M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.+K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9dR"><div class="casContent"><span class="casTitleNuber">9d</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0liJNWmtJY-D4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit9d&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.%2BM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%2BK.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.</span></span> <span> </span><span class="NLM_article-title">Enterovirus A71 proteins: structure and function</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">286</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2018.00286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.3389%2Ffmicb.2018.00286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=29515559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BC1MroslOhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=286&author=J.+Yuanauthor=L.+Shenauthor=J.+Wuauthor=X.+Zouauthor=J.+Guauthor=J.+Chenauthor=L.+Mao&title=Enterovirus+A71+proteins%3A+structure+and+function&doi=10.3389%2Ffmicb.2018.00286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Enterovirus A71 Proteins: Structure and Function</span></div><div class="casAuthors">Yuan Jingjing; Chen Jianguo; Mao Lingxiang; Yuan Jingjing; Shen Li; Wu Jing; Zou Xinran; Gu Jiaqi</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">286</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">Enterovirus A71 (EV-A71) infection has grown to become a serious threat to global public health.  It is one of the major causes of hand, foot, and mouth disease (HFMD) in infants and young children.  EV-A71 can also infect the central nervous system (CNS) and induce diverse neurological complications, such as brainstem encephalitis, aseptic meningitis, and acute flaccid paralysis, or even death.  Viral proteins play a crucial role in EV-A71 infection.  Many recent studies have discussed the structure and function of EV-A71 proteins, and the findings reported will definitely aid the development of vaccines and therapeutic approaches.  This article reviews the progress in the research on the structure and function of EV-A71 proteins.  Available literature can provide a basis for studying the pathogenesis of EV-A71 infection in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ6VkEBjsea1QMthz5_Rx5rfW6udTcc2eawfgsEtshYCrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MroslOhuw%253D%253D&md5=642c8ff42f4db800c1b0bd327e939b56</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2018.00286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2018.00286%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DL.%26atitle%3DEnterovirus%2520A71%2520proteins%253A%2520structure%2520and%2520function%26jtitle%3DFront.%2520Microbiol.%26date%3D2018%26volume%3D9%26spage%3D286%26doi%3D10.3389%2Ffmicb.2018.00286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit10b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, R. S.</span></span> <span> </span><span class="NLM_article-title">A concise review of baseline facts of SARS-CoV-2 for interdisciplinary research</span>. <i>ChemistrySelect.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10923</span>, <span class="refDoi"> DOI: 10.1002/slct.202002420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fslct.202002420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvV2nurzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=10897-10923&author=A.+Tyagiauthor=S.+Nigamauthor=R.+S.+Chauhan&title=A+concise+review+of+baseline+facts+of+SARS-CoV-2+for+interdisciplinary+research&doi=10.1002%2Fslct.202002420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">A Concise Review of Baseline Facts of SARS-CoV-2 for Interdisciplinary Research</span></div><div class="casAuthors">Tyagi, Adish; Nigam, Sandeep; Chauhan, Rohit Singh</div><div class="citationInfo"><span class="NLM_cas:title">ChemistrySelect</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10897-10923</span>CODEN:
                <span class="NLM_cas:coden">CHEMUD</span>;
        ISSN:<span class="NLM_cas:issn">2365-6549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 is a global health emergency.  Investigation covering various aspects of diagnosis, treatment, and vaccine development for COVID-19 are being attempted globally.  Common goal of reducing loss of life requires accelerated and integrated effort from researchers of different fields i. e. chem., biol., clin./medical science etc.  Integrated yet timely effort requires the understanding of interdisciplinary key concepts as well as knowledge of major challenges, present status and strategies.  Hence, in this article, we present an overview of baseline/key facts of SARS-CoV-2 mingled with current status and strategies in a simple, straight forward and coherent manner.  In particular, the review attempts to integrate and arrange the key facts at the interface of chem. and biol.  Overview of epidemiol. of SARS-CoV-2 virus infection, structural characteristics of virus, detection methods and therapeutic agents based on antibodies, therapeutic agents based on traditional drug mol., drug development based on specific catalytic enzyme inhibitors, vaccine design approaches have been provided.  Since enzyme site and catalytic action is the key for biol. process, they have been lucidly explained at mol. level.  It is believed that accredited information amalgamated in this report will provide a strong rational and lucid groundwork for the coherent interdisciplinary research to countermeasure covid-19 crisis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiCFiihiV-QrVg90H21EOLACvtfcHk0lip0Xr4NQoHyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvV2nurzF&md5=1faf94b647286fd085fb0b13b648c217</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1002%2Fslct.202002420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fslct.202002420%26sid%3Dliteratum%253Aachs%26aulast%3DTyagi%26aufirst%3DA.%26aulast%3DNigam%26aufirst%3DS.%26aulast%3DChauhan%26aufirst%3DR.%2BS.%26atitle%3DA%2520concise%2520review%2520of%2520baseline%2520facts%2520of%2520SARS-CoV-2%2520for%2520interdisciplinary%2520research%26jtitle%3DChemistrySelect.%26date%3D2020%26volume%3D5%26spage%3D10897%26epage%3D10923%26doi%3D10.1002%2Fslct.202002420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit10c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palm, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesters, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddell, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3213</span>– <span class="NLM_lpage">3224</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdf327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1093%2Femboj%2Fcdf327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=12093723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsFCgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=3213-3224&author=K.+Anandauthor=G.+J.+Palmauthor=J.+R.+Mestersauthor=S.+G.+Siddellauthor=J.+Ziebuhrauthor=R.+Hilgenfeld&title=Structure+of+coronavirus+main+proteinase+reveals+combination+of+a+chymotrypsin+fold+with+an+extra+%CE%B1-helical+domain&doi=10.1093%2Femboj%2Fcdf327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10cR"><div class="casContent"><span class="casTitleNuber">10c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain</span></div><div class="casAuthors">Anand, Kanchan; Palm, Gottfried J.; Mesters, Jeroen R.; Siddell, Stuart G.; Ziebuhr, John; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3213-3224</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The key enzyme in coronavirus polyprotein processing is the viral main proteinase, Mpro, a protein with extremely low sequence similarity to other viral and cellular proteinases.  Here, the crystal structure of the 33.1 kDa transmissible gastroenteritis (corona)virus Mpro is reported.  The structure was refined to 1.96 Å resoln. and revealed three dimers in the asym. unit.  The mutual arrangement of the protomers in each of the dimers suggests that Mpro self-processing occurs in trans.  The active site, comprised of Cys144 and His41, is part of a chymotrypsin-like fold that is connected by a 16 residue loop to an extra domain featuring a novel α-helical fold.  Mol. modeling and mutagenesis data implicate the loop in substrate binding and elucidate S1 and S2 subsites suitable to accommodate the side chains of the P1 glutamine and P2 leucine residues of Mpro substrates.  Interactions involving the N-terminus and the α-helical domain stabilize the loop in the orientation required for trans-cleavage activity.  The study illustrates that RNA viruses have evolved unprecedented variations of the classical chymotrypsin fold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-3mPwalwvErVg90H21EOLACvtfcHk0lip0Xr4NQoHyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsFCgt78%253D&md5=5bb1ad2d3b1416639968f26ea46ebf03</span></div><a href="/servlet/linkout?suffix=cit10c&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdf327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdf327%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DK.%26aulast%3DPalm%26aufirst%3DG.%2BJ.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DSiddell%26aufirst%3DS.%2BG.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DStructure%2520of%2520coronavirus%2520main%2520proteinase%2520reveals%2520combination%2520of%2520a%2520chymotrypsin%2520fold%2520with%2520an%2520extra%2520%25CE%25B1-helical%2520domain%26jtitle%3DEMBO%2520J.%26date%3D2002%26volume%3D21%26spage%3D3213%26epage%3D3224%26doi%3D10.1093%2Femboj%2Fcdf327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alugubelli, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vatansever, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drelich, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blankenship, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratch, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayatshahi, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierke, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">A quick route to multiple highly potent SARS-CoV-2 main protease inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fcmdc.202000924" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2021&pages=1-8&author=K.+S.+Yangauthor=X.+R.+Maauthor=Y.+Maauthor=Y.+R.+Alugubelliauthor=D.+A.+Scottauthor=E.+C.+Vatanseverauthor=A.+K.+Drelichauthor=B.+Sankaranauthor=Z.+Z.+Gengauthor=L.+R.+Blankenshipauthor=H.+E.+Wardauthor=Y.+J.+Shengauthor=J.+C.+Hsuauthor=K.+C.+Kratchauthor=B.+Zhaoauthor=H.+S.+Hayatshahiauthor=J.+Liuauthor=P.+Liauthor=C.+A.+Fierkeauthor=C.+K.+Tsengauthor=S.+Xuauthor=W.+R.+Liu&title=A+quick+route+to+multiple+highly+potent+SARS-CoV-2+main+protease+inhibitors&doi=10.1002%2Fcmdc.202000924"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11a&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000924%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DMa%26aufirst%3DX.%2BR.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DAlugubelli%26aufirst%3DY.%2BR.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DVatansever%26aufirst%3DE.%2BC.%26aulast%3DDrelich%26aufirst%3DA.%2BK.%26aulast%3DSankaran%26aufirst%3DB.%26aulast%3DGeng%26aufirst%3DZ.%2BZ.%26aulast%3DBlankenship%26aufirst%3DL.%2BR.%26aulast%3DWard%26aufirst%3DH.%2BE.%26aulast%3DSheng%26aufirst%3DY.%2BJ.%26aulast%3DHsu%26aufirst%3DJ.%2BC.%26aulast%3DKratch%26aufirst%3DK.%2BC.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DHayatshahi%26aufirst%3DH.%2BS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DFierke%26aufirst%3DC.%2BA.%26aulast%3DTseng%26aufirst%3DC.%2BK.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DA%2520quick%2520route%2520to%2520multiple%2520highly%2520potent%2520SARS-CoV-2%2520main%2520protease%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2021%26volume%3D15%26spage%3D1%26epage%3D8%26doi%3D10.1002%2Fcmdc.202000924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit11b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=J.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+Ketone-Based+Covalent+Inhibitors+of+Coronavirus+3CL+Proteases+for+the+Potential+Therapeutic+Treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11bR"><div class="casContent"><span class="casTitleNuber">11b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lifT4xOc24H9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit11b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Ketone-Based%2520Covalent%2520Inhibitors%2520of%2520Coronavirus%25203CL%2520Proteases%2520for%2520the%2520Potential%2520Therapeutic%2520Treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit11c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4282</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18096-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41467-020-18096-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32855413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4282&author=W.+Vuongauthor=M.+B.+Khanauthor=C.+Fischerauthor=E.+Arutyunovaauthor=T.+Lamerauthor=J.+Shieldsauthor=H.+A.+Saffranauthor=R.+T.+McKayauthor=M.+J.+van+Belkumauthor=M.+A.+Joyceauthor=H.+S.+Youngauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=M.+J.+Lemieux&title=Feline+coronavirus+drug+inhibits+the+main+protease+of+SARS-CoV-2+and+blocks+virus+replication&doi=10.1038%2Fs41467-020-18096-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11cR"><div class="casContent"><span class="casTitleNuber">11c</span><div class="casTitle"><span class="NLM_cas:atitle">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Vuong, Wayne; Khan, Muhammad Bashir; Fischer, Conrad; Arutyunova, Elena; Lamer, Tess; Shields, Justin; Saffran, Holly A.; McKay, Ryan T.; van Belkum, Marco J.; Joyce, Michael A.; Young, Howard S.; Tyrrell, D. Lorne; Vederas, John C.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4282</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The main protease, Mpro (or 3CLpro) in SARS-CoV-2 is a viable drug target because of its essential role in the cleavage of the virus polypeptide.  Feline infectious peritonitis, a fatal coronavirus infection in cats, was successfully treated previously with a prodrug GC376, a dipeptide-based protease inhibitor.  We show the prodrug and its parent GC373, are effective inhibitors of the Mpro from both SARS-CoV and SARS-CoV-2 with IC50 values in the nanomolar range.  Crystal structures of SARS-CoV-2 Mpro with these inhibitors have a covalent modification of the nucleophilic Cys145.  NMR anal. reveals that inhibition proceeds via reversible formation of a hemithioacetal.  GC373 and GC376 are potent inhibitors of SARS-CoV-2 replication in cell culture.  They are strong drug candidates for the treatment of human coronavirus infections because they have already been successful in animals.  The work here lays the framework for their use in human trials for the treatment of COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMA7W4XtRQrbVg90H21EOLACvtfcHk0lifT4xOc24H9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN&md5=c06d2dc9a65ae289505fe120d41d8885</span></div><a href="/servlet/linkout?suffix=cit11c&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18096-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18096-2%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DShields%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DFeline%2520coronavirus%2520drug%2520inhibits%2520the%2520main%2520protease%2520of%2520SARS-CoV-2%2520and%2520blocks%2520virus%2520replication%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4282%26doi%3D10.1038%2Fs41467-020-18096-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzarano, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span> <span> </span><span class="NLM_article-title">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. I. Investigation of potential experimental and mechanistic explanations</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1080/0049825042000205540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1080%2F0049825042000205540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15268980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2004&pages=353-366&author=K.+W.+Wardauthor=L.+M.+Azzaranoauthor=C.+A.+Evansauthor=B.+R.+Smith&title=Apparent+absolute+oral+bioavailability+in+excess+of+100%25+for+a+vitronectin+receptor+antagonist+%28SB-265123%29+in+rat.+I.+Investigation+of+potential+experimental+and+mechanistic+explanations&doi=10.1080%2F0049825042000205540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. I. Investigation of potential experimental and mechanistic explanations</span></div><div class="casAuthors">Ward, K. W.; Azzarano, L. M.; Evans, C. A.; Smith, B. R.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">353-366</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">SB-265123 is a novel αvβ3 (the vitronectin receptor) antagonist.  Previous rat studies with it revealed an apparent abs. oral bioavailability (Fapp) of > 100%.  The present studies were conducted to investigate the potential causes for this observation.  Of 49 SB-265123 analogs evaluated in rat using an identical exptl. design, Fapp>100% was obsd. for 22 of them, suggesting that the obsd. Fapp>100% with SB-265123 was not anomalous.  All 22 compds. had clearances<15 mL min-1 kg-1.  However, Fapp>100% were not recorded for all low-clearance analogs.  Using SB-265123 as a model to investigate potential artifacts, it was demonstrated that using a chiral assay did not decrease Fapp.  Addnl., qual. sample anal. demonstrated that no metabolites were present in the blood plasma that could interfere with the liq. chromatog. coupled with tandem mass spectrometry detection assay.  The high Fapp was also dose-order-, delivery system- and vehicle-independent, and was not affected by the feeding status of the animals.  Furthermore, a linearity expt. and an absorption study indicated that oral administration of SB-265123 does not result in hepatic portal vein concns. that exceed the pharmacokinetic linearity of SB-265123.  These observations suggest that the obsd. Fapp>100% for SB-265123 is not due to an exptl. artifact or an obvious pharmacokinetic non-linearity.  The mechanism(s) for this phenomenon is explored further in the 2nd part of the present paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp288gThfUWArVg90H21EOLACvtfcHk0lifT4xOc24H9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurk%253D&md5=50c8721ab52880039deaaa2ee53b9644</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1080%2F0049825042000205540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F0049825042000205540%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DAzzarano%26aufirst%3DL.%2BM.%26aulast%3DEvans%26aufirst%3DC.%2BA.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DApparent%2520absolute%2520oral%2520bioavailability%2520in%2520excess%2520of%2520100%2525%2520for%2520a%2520vitronectin%2520receptor%2520antagonist%2520%2528SB-265123%2529%2520in%2520rat.%2520I.%2520Investigation%2520of%2520potential%2520experimental%2520and%2520mechanistic%2520explanations%26jtitle%3DXenobiotica%26date%3D2004%26volume%3D34%26spage%3D353%26epage%3D366%26doi%3D10.1080%2F0049825042000205540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kehler, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzarano, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span> <span> </span><span class="NLM_article-title">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. II. Studies implicating transporter-mediated intestinal secretion</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1080/0049825042000205540a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1080%2F0049825042000205540a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15268981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2004&pages=367-377&author=K.+W.+Wardauthor=L.+B.+Hardyauthor=J.+R.+Kehlerauthor=L.+M.+Azzaranoauthor=B.+R.+Smith&title=Apparent+absolute+oral+bioavailability+in+excess+of+100%25+for+a+vitronectin+receptor+antagonist+%28SB-265123%29+in+rat.+II.+Studies+implicating+transporter-mediated+intestinal+secretion&doi=10.1080%2F0049825042000205540a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. II. Studies implicating transporter-mediated intestinal secretion</span></div><div class="casAuthors">Ward, K. W.; Hardy, L. B.; Kehler, J. R.; Azzarano, L. M.; Smith, B. R.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">367-377</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Transporters were increasingly identified as a factor in limiting the oral bioavailability of certain drugs.  Previously, the present authors investigated a compd. (SB-265123) with an apparent abs. oral bioavailability (Fapp) consistently >100%, and excluded likely artifactual causes for this observation, as well as std. considerations of non-stationary or non-linear pharmacokinetics.  The data led the authors to believe that SB-265123 might be a transporter substrate in the rat, and it was hypothesized that transporter interactions might be responsible for the obsd. Fapp>100%.  In the present study, a model was proposed incorporating rapid and complete absorption and elimination by a saturable intestinal secretory pathway.  Intestinal secretion was demonstrated for SB-265123 using a rat single-pass intestinal perfusion technique.  In addn., in a study employing both independent and simultaneous i.v. and oral administration of SB-265123, exposure to SB-265123 was greater than additive on joint i.v. and oral administration, lending further support to the hypothesis of a saturable transporter.  Furthermore, in a study with co-administration of GF120918A, a transporter inhibitor, the obsd. Fapp for SB-265123 was only 84%, providing addnl. evidence for transporter involvement in the >100% Fapp phenomenon.  Experience with SB-265123 illustrates a counterintuitive impact of transporters on oral bioavailability and highlights the importance of considering transporter interactions in the systemic disposition of xenobiotics, even those not demonstrating low oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQlI6oO-2rRLVg90H21EOLACvtfcHk0lhUcJ-DoWE85A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurY%253D&md5=33d6c532955bbca6540152adbf512f97</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1080%2F0049825042000205540a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F0049825042000205540a%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DHardy%26aufirst%3DL.%2BB.%26aulast%3DKehler%26aufirst%3DJ.%2BR.%26aulast%3DAzzarano%26aufirst%3DL.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DApparent%2520absolute%2520oral%2520bioavailability%2520in%2520excess%2520of%2520100%2525%2520for%2520a%2520vitronectin%2520receptor%2520antagonist%2520%2528SB-265123%2529%2520in%2520rat.%2520II.%2520Studies%2520implicating%2520transporter-mediated%2520intestinal%2520secretion%26jtitle%3DXenobiotica%26date%3D2004%26volume%3D34%26spage%3D367%26epage%3D377%26doi%3D10.1080%2F0049825042000205540a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J. H.</span></span> <span> </span><span class="NLM_article-title">Upgrade of macromolecular crystallography beamline BL17U1 at SSRF</span>. <i>Nucl. Sci. Tech.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">687</span>, <span class="refDoi"> DOI: 10.1007/s41365-018-0398-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1007%2Fs41365-018-0398-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=681-687&author=Q.+S.+Wangauthor=K.+H.+Zhangauthor=Y.+Cuiauthor=Z.+J.+Wangauthor=Q.+Y.+Panauthor=K.+Liuauthor=B.+Sunauthor=H.+Zhouauthor=M.+J.+Liauthor=Q.+Xuauthor=C.+Y.+Xuauthor=F.+Yuauthor=J.+H.+He&title=Upgrade+of+macromolecular+crystallography+beamline+BL17U1+at+SSRF&doi=10.1007%2Fs41365-018-0398-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs41365-018-0398-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs41365-018-0398-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%2BS.%26aulast%3DZhang%26aufirst%3DK.%2BH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%2BJ.%26aulast%3DPan%26aufirst%3DQ.%2BY.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DC.%2BY.%26aulast%3DYu%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DJ.%2BH.%26atitle%3DUpgrade%2520of%2520macromolecular%2520crystallography%2520beamline%2520BL17U1%2520at%2520SSRF%26jtitle%3DNucl.%2520Sci.%2520Tech.%26date%3D2018%26volume%3D29%26spage%3D681%26epage%3D687%26doi%3D10.1007%2Fs41365-018-0398-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution-from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution-from+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lhUcJ-DoWE85A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution-from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhUcJ-DoWE85A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljxJUh8Ykd2PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0ljxJUh8Ykd2PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. C.</span></span> <span> </span><span class="NLM_article-title">Anti-SARS-CoV-2 activities in vitro of Shuanghuanglian preparations and bioactive ingredients</span>. <i>Acta Pharmacol. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1177</span>, <span class="refDoi"> DOI: 10.1038/s41401-020-0483-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41401-020-0483-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32737471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFShu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=1167-1177&author=H.+X.+Suauthor=S.+Yaoauthor=W.+F.+Zhaoauthor=M.+J.+Liauthor=J.+Liuauthor=W.+J.+Shangauthor=H.+Xieauthor=C.+Q.+Keauthor=H.+C.+Huauthor=M.+N.+Gaoauthor=K.+Q.+Yuauthor=H.+Liuauthor=J.+S.+Shenauthor=W.+Tangauthor=L.+K.+Zhangauthor=G.+F.+Xiaoauthor=L.+Niauthor=D.+W.+Wangauthor=J.+P.+Zuoauthor=H.+L.+Jiangauthor=F.+Baiauthor=Y.+Wuauthor=Y.+Yeauthor=Y.+C.+Xu&title=Anti-SARS-CoV-2+activities+in+vitro+of+Shuanghuanglian+preparations+and+bioactive+ingredients&doi=10.1038%2Fs41401-020-0483-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-SARS-CoV-2 activities in vitro of Shuanghuanglian preparations and bioactive ingredients</span></div><div class="casAuthors">Su, Hai-xia; Yao, Sheng; Zhao, Wen-feng; Li, Min-jun; Liu, Jia; Shang, Wei-juan; Xie, Hang; Ke, Chang-qiang; Hu, Hang-chen; Gao, Mei-na; Yu, Kun-qian; Liu, Hong; Shen, Jing-shan; Tang, Wei; Zhang, Lei-ke; Xiao, Geng-fu; Ni, Li; Wang, Dao-wen; Zuo, Jian-ping; Jiang, Hua-liang; Bai, Fang; Wu, Yan; Ye, Yang; Xu, Ye-chun</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmacologica Sinica</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1167-1177</span>CODEN:
                <span class="NLM_cas:coden">APSCG5</span>;
        ISSN:<span class="NLM_cas:issn">1671-4083</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Human infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes coronavirus disease 2019 (COVID-19) and there is no cure currently.  The 3CL protease (3CLpro) is a highly conserved protease which is indispensable for CoVs replication, and is a promising target for development of broad-spectrum antiviral drugs.  In this study we investigated the anti-SARS-CoV-2 potential of Shuanghuanglian prepn., a Chinese traditional patent medicine with a long history for treating respiratory tract infection in China.  We showed that either the oral liq. of Shuanghuanglian, the lyophilized powder of Shuanghuanglian for injection or their bioactive components dose-dependently inhibited SARS-CoV-2 3CLpro as well as the replication of SARS-CoV-2 in Vero E6 cells.  Baicalin and baicalein, two ingredients of Shuanghuanglian, were characterized as the first noncovalent, nonpeptidomimetic inhibitors of SARS-CoV-2 3CLpro and exhibited potent antiviral activities in a cell-based system.  Remarkably, the binding mode of baicalein with SARS-CoV-2 3CLpro detd. by X-ray protein crystallog. was distinctly different from those of known 3CLpro inhibitors.  Baicalein was productively ensconced in the core of the substrate-binding pocket by interacting with two catalytic residues, the crucial S1/S2 subsites and the oxyanion loop, acting as a "shield" in front of the catalytic dyad to effectively prevent substrate access to the catalytic dyad within the active site.  Overall, this study provides an example for exploring the in vitro potency of Chinese traditional patent medicines and effectively identifying bioactive ingredients toward a specific target, and gains evidence supporting the in vivo studies of Shuanghuanglian oral liq. as well as two natural products for COVID-19 treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE-aZrrHwqZrVg90H21EOLACvtfcHk0ljxJUh8Ykd2PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFShu7vI&md5=8560cc7a979d67e0fa0cf17eee71ef00</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41401-020-0483-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41401-020-0483-6%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DH.%2BX.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DW.%2BF.%26aulast%3DLi%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DShang%26aufirst%3DW.%2BJ.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DKe%26aufirst%3DC.%2BQ.%26aulast%3DHu%26aufirst%3DH.%2BC.%26aulast%3DGao%26aufirst%3DM.%2BN.%26aulast%3DYu%26aufirst%3DK.%2BQ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DJ.%2BS.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%2BK.%26aulast%3DXiao%26aufirst%3DG.%2BF.%26aulast%3DNi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DD.%2BW.%26aulast%3DZuo%26aufirst%3DJ.%2BP.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%2BC.%26atitle%3DAnti-SARS-CoV-2%2520activities%2520in%2520vitro%2520of%2520Shuanghuanglian%2520preparations%2520and%2520bioactive%2520ingredients%26jtitle%3DActa%2520Pharmacol.%2520Sin.%26date%3D2020%26volume%3D41%26spage%3D1167%26epage%3D1177%26doi%3D10.1038%2Fs41401-020-0483-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span> <span> </span><span class="NLM_article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">271</span>, <span class="refDoi"> DOI: 10.1038/s41422-020-0282-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41422-020-0282-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32020029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkt1Ciu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=269-271&author=M.+Wangauthor=R.+Caoauthor=L.+Zhangauthor=X.+Yangauthor=J.+Liuauthor=M.+Xuauthor=Z.+Shiauthor=Z.+Huauthor=W.+Zhongauthor=G.+Xiao&title=Remdesivir+and+chloroquine+effectively+inhibit+the+recently+emerged+novel+coronavirus+%282019-nCoV%29+in+vitro&doi=10.1038%2Fs41422-020-0282-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span></div><div class="casAuthors">Wang, Manli; Cao, Ruiyuan; Zhang, Leike; Yang, Xinglou; Liu, Jia; Xu, Mingyue; Shi, Zhengli; Hu, Zhihong; Zhong, Wu; Xiao, Gengfu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-271</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In Dec. 2019, a novel pneumonia caused by a previously unknown pathogen emerged in Wuhan, a city of 11 million people in central China.  The initial cases were linked to exposures in a seafood market in Wuhan.  The pathogen was soon identified as a novel coronavirus (2019-nCoV), which is closely related to severe acute respiratory syndrome CoV (SARS-CoV).  Currently, there is no specific treatment against the new virus.  Therefore, identifying effective antiviral agents to combat the disease is urgently needed.  In this study, we evaluated the antiviral efficiency of FAD-approved drugs including ribavirin, penciclovir, nitazoxanide, nafamostat, chloroquine and two well-known broad-spectrum antiviral drugs remdesivir (GS-5734) and favipiravir (T-705) against a clin. isolate of 2019-nCoV in vitro.  Our findings reveal that remdesivir and chloroquine are highly effective in the control of 2019-nCoV infection in vitro.  Since these compds. have been used in human patients with a safety track record and shown to be effective against various ailments, we suggest that they should be assessed in human patients suffering from the novel coronavirus disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCnj2tW4TeD7Vg90H21EOLACvtfcHk0liTfIN-T2dTXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkt1Ciu7k%253D&md5=bdf16b352ebc06bc148ed5c513b7d414</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41422-020-0282-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-020-0282-0%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DZhong%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26atitle%3DRemdesivir%2520and%2520chloroquine%2520effectively%2520inhibit%2520the%2520recently%2520emerged%2520novel%2520coronavirus%2520%25282019-nCoV%2529%2520in%2520vitro%26jtitle%3DCell%2520Res.%26date%3D2020%26volume%3D30%26spage%3D269%26epage%3D271%26doi%3D10.1038%2Fs41422-020-0282-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibaut, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernachio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Kuppeveld, F. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delang, L.</span></span> <span> </span><span class="NLM_article-title">Antiviral activity of broad-spectrum and enterovirus-specific inhibitors against clinical isolates of enterovirus D68</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7782</span>– <span class="NLM_lpage">7785</span>, <span class="refDoi"> DOI: 10.1128/AAC.01375-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.01375-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=26369972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVGgtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=7782-7785&author=L.+Sunauthor=A.+Meijerauthor=M.+Froeyenauthor=L.+Zhangauthor=H.+J.+Thibautauthor=J.+Baggenauthor=S.+Georgeauthor=J.+Vernachioauthor=F.+J.+M.+van+Kuppeveldauthor=P.+Leyssenauthor=R.+Hilgenfeldauthor=J.+Neytsauthor=L.+Delang&title=Antiviral+activity+of+broad-spectrum+and+enterovirus-specific+inhibitors+against+clinical+isolates+of+enterovirus+D68&doi=10.1128%2FAAC.01375-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20aR"><div class="casContent"><span class="casTitleNuber">20a</span><div class="casTitle"><span class="NLM_cas:atitle">Antiviral activity of broad-spectrum and enterovirus specific inhibitors against clinical isolates of enterovirus D68</span></div><div class="casAuthors">Sun, Liang; Meijer, Adam; Froeyen, Mathy; Zhang, Linlin; Thibaut, Hendrik Jan; Baggen, Jim; George, Shyla; Vernachio, John; van Kuppeveld, Frank J. M.; Leyssen, Pieter; Hilgenfeld, Rolf; Neyts, Johan; Delang, Leen</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7782-7785</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We investigated the susceptibility of 10 enterovirus D68 (EV-D68) isolates (belonging to clusters A, B, and C) to (entero)virus inhibitors with different mechanisms of action.  The 3C-protease inhibitors proved to be more efficient than enviroxime and pleconaril, which in turn were more effective than vapendavir and pirodavir.  Favipiravir proved to be a weak inhibitor.  Resistance to pleconaril maps to V69A in the VP1 protein, and resistance to rupintrivir maps to V104I in the 3C protease.  A structural explanation of why both substitutions may cause resistance is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1u7CM0ymnsLVg90H21EOLACvtfcHk0liTfIN-T2dTXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVGgtLo%253D&md5=4e58b350187aeedecec35db48ba3ea7b</span></div><a href="/servlet/linkout?suffix=cit20a&amp;dbid=16384&amp;doi=10.1128%2FAAC.01375-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01375-15%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DMeijer%26aufirst%3DA.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3DBaggen%26aufirst%3DJ.%26aulast%3DGeorge%26aufirst%3DS.%26aulast%3DVernachio%26aufirst%3DJ.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DDelang%26aufirst%3DL.%26atitle%3DAntiviral%2520activity%2520of%2520broad-spectrum%2520and%2520enterovirus-specific%2520inhibitors%2520against%2520clinical%2520isolates%2520of%2520enterovirus%2520D68%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D7782%26epage%3D7785%26doi%3D10.1128%2FAAC.01375-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit20b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Aguado, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibaut, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priego, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimeno, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarasa, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Pérez, M.-J.</span></span> <span> </span><span class="NLM_article-title">9-Arylpurines as a novel class of enterovirus inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1021/jm901240p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901240p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVeku7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=316-324&author=L.+Aguadoauthor=H.+J.+Thibautauthor=E.-M.+Priegoauthor=M.-L.+Jimenoauthor=M.-J.+Camarasaauthor=J.+Neytsauthor=M.-J.+P%C3%A9rez-P%C3%A9rez&title=9-Arylpurines+as+a+novel+class+of+enterovirus+inhibitors&doi=10.1021%2Fjm901240p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20bR"><div class="casContent"><span class="casTitleNuber">20b</span><div class="casTitle"><span class="NLM_cas:atitle">9-Arylpurines as a Novel Class of Enterovirus Inhibitors</span></div><div class="casAuthors">Aguado, Leire; Thibaut, Hendrik Jan; Priego, Eva-Maria; Jimeno, Maria-Luisa; Camarasa, Maria-Jose; Neyts, Johan; Perez-Perez, Maria-Jesus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">316-324</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we report on a novel class of enterovirus inhibitors that can be structurally described as 9-arylpurines.  These compds. elicit activity against a variety of enteroviruses in the low μM range including Coxsackie virus A16, A21, A24, Coxsackie virus B3, and echovirus 9.  Structure-activity relationship (SAR) studies indicate that a chlorine or bromine atom is required at position 6 of the purine ring for antiviral activity.  The most selective compds. in this series inhibited Coxsackie virus B3 replication in a dose-dependent manner with EC50 values around 5-8 μM.  No toxicity on different cell lines was obsd. at concns. up to 250 μM.  Moreover, no cross-resistance to TBZE-029 and TTP-8307 CVB3 resistant strains was detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZK6ihSR_g27Vg90H21EOLACvtfcHk0lhGypIU8Cc9tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVeku7jI&md5=605fb1225f0efebe3a9eeae22d9be2c6</span></div><a href="/servlet/linkout?suffix=cit20b&amp;dbid=16384&amp;doi=10.1021%2Fjm901240p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901240p%26sid%3Dliteratum%253Aachs%26aulast%3DAguado%26aufirst%3DL.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3DPriego%26aufirst%3DE.-M.%26aulast%3DJimeno%26aufirst%3DM.-L.%26aulast%3DCamarasa%26aufirst%3DM.-J.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DP%25C3%25A9rez-P%25C3%25A9rez%26aufirst%3DM.-J.%26atitle%3D9-Arylpurines%2520as%2520a%2520novel%2520class%2520of%2520enterovirus%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D316%26epage%3D324%26doi%3D10.1021%2Fjm901240p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit20c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coluccia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giorgio, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terme, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirilli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regina, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvestri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirabelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanelle, P.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Drug Design of Potent Pyrazole Derivatives against Rhinovirus Replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8402</span>– <span class="NLM_lpage">8416</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00931</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00931" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8402-8416&author=L.+D.+Costaauthor=E.+Scheersauthor=A.+Colucciaauthor=A.+Casulliauthor=M.+Rocheauthor=C.+D.+Giorgioauthor=J.+Neytsauthor=T.+Termeauthor=R.+Cirilliauthor=G.+L.+Reginaauthor=R.+Silvestriauthor=C.+Mirabelliauthor=P.+Vanelle&title=Structure-Based+Drug+Design+of+Potent+Pyrazole+Derivatives+against+Rhinovirus+Replication&doi=10.1021%2Facs.jmedchem.8b00931"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00931%26sid%3Dliteratum%253Aachs%26aulast%3DCosta%26aufirst%3DL.%2BD.%26aulast%3DScheers%26aufirst%3DE.%26aulast%3DColuccia%26aufirst%3DA.%26aulast%3DCasulli%26aufirst%3DA.%26aulast%3DRoche%26aufirst%3DM.%26aulast%3DGiorgio%26aufirst%3DC.%2BD.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DTerme%26aufirst%3DT.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DRegina%26aufirst%3DG.%2BL.%26aulast%3DSilvestri%26aufirst%3DR.%26aulast%3DMirabelli%26aufirst%3DC.%26aulast%3DVanelle%26aufirst%3DP.%26atitle%3DStructure-Based%2520Drug%2520Design%2520of%2520Potent%2520Pyrazole%2520Derivatives%2520against%2520Rhinovirus%2520Replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8402%26epage%3D8416%26doi%3D10.1021%2Facs.jmedchem.8b00931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b','cit1c','cit1d','cit1e','cit1f'],'ref2':['cit2a','cit2b'],'ref3':['cit3a','cit3b','cit3c','cit3d','cit3e','cit3f'],'ref4':['cit4a','cit4b','cit4c'],'ref5':['cit5a','cit5b','cit5c','cit5d','cit5e','cit5f','cit5g','cit5h','cit5i','cit5j','cit5k','cit5l'],'ref6':['cit6a','cit6b','cit6c','cit6d','cit6e','cit6f','cit6g'],'ref7':['cit7a','cit7b','cit7c','cit7d'],'ref8':['cit8a','cit8b','cit8c','cit8d','cit8e'],'ref9':['cit9a','cit9b','cit9c','cit9d'],'ref10':['cit10a','cit10b','cit10c'],'ref11':['cit11a','cit11b','cit11c'],'ref12':['cit12a','cit12b'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20a','cit20b','cit20c']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representatives of reported EV71 3C protease inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. X-ray structure of the surface representation of EV71 3C<sup>pro</sup> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GHT">4GHT</a>) complexed with the <b>AG7088</b> (yellow).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Design of novel EV71 3C protease inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis Procedure of Target Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) LiHMDS, THF, −78 °C; (b) NaBH<sub>4</sub>, CoCl<sub>2</sub>, 0 °C; (c) 4 M HCl, 12 h; (d) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12 h; (e) 4 M HCl, 12 h; (f) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12 h; (g) NaBH<sub>4</sub>, CH<sub>3</sub>OH; (h) Dess–Martin periodinane, NaHCO<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>; (i) Ph<sub>3</sub>PCH<sub>2</sub>COOR<sub>4</sub>, Et<sub>3</sub>N, DCM.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis Procedure of Target Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12 h; (b) 4 M HCl, 12 h; (c) HATU, DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, −20 °C, 12h; (d) NaBH<sub>4</sub>, CH<sub>3</sub>OH; (e) Dess–Martin Periodinane, NaHCO<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structure of the EV71 3C<sup>pro</sup> in complex with <b>18p</b>. (A) The binding mode of <b>18p</b> at the substrate-binding site of the EV71 3C<sup>pro</sup> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7DNC">7DNC</a>). The EV71 3C<sup>pro</sup> was shown as a molecular surface, and <b>18p</b> was shown by light orange sticks. (B) 2<i>F</i>o-<i>F</i>c density maps contoured at 2.0 σ are shown for <b>18p</b> and C147. (C) Interactions of <b>18p</b> with the surrounding residues revealed by the crystal structure. Residues are shown as light blue sticks, and H-bonds are represented by black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/medium/jm0c02258_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representatives of reported SARS-CoV-2 3CL protease inhibitors</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.0c02258/20210427/images/large/jm0c02258_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02258&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 20 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennette, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, H. H.</span></span> <span> </span><span class="NLM_article-title">An apparently new enterovirus isolated from patients with disease of the central nervous system</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">1974</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1093/infdis/129.3.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1093%2Finfdis%2F129.3.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=4361245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADyaE2c7hsl2lsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=1974&pages=304-309&author=N.+J.+Schmidtauthor=E.+H.+Lennetteauthor=H.+H.+Ho&title=An+apparently+new+enterovirus+isolated+from+patients+with+disease+of+the+central+nervous+system&doi=10.1093%2Finfdis%2F129.3.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">An apparently new enterovirus isolated from patients with disease of the central nervous system</span></div><div class="casAuthors">Schmidt N J; Lennette E H; Ho H H</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of infectious diseases</span>
        (<span class="NLM_cas:date">1974</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">304-9</span>
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ9Z6tSn96A0ftos8R3JUhXfW6udTcc2eY4-lEWRz35xbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE2c7hsl2lsw%253D%253D&md5=2b47e2de0621e8ab2d4f99e40d0e02d5</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2F129.3.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252F129.3.304%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DN.%2BJ.%26aulast%3DLennette%26aufirst%3DE.%2BH.%26aulast%3DHo%26aufirst%3DH.%2BH.%26atitle%3DAn%2520apparently%2520new%2520enterovirus%2520isolated%2520from%2520patients%2520with%2520disease%2520of%2520the%2520central%2520nervous%2520system%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D1974%26volume%3D129%26spage%3D304%26epage%3D309%26doi%3D10.1093%2Finfdis%2F129.3.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. C.</span></span> <span> </span><span class="NLM_article-title">Acute flaccid paralysis in infants and young children with enterovirus 71 infection: MR imaging findings and clinical correlates</span>. <i>Am. J. Neuroradiol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">205</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=11158910" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BD3M7nslygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=200-205&author=C.+Y.+Chenauthor=Y.+C.+Changauthor=C.+C.+Huangauthor=C.+C.+Luiauthor=K.+W.+Leeauthor=S.+C.+Huang&title=Acute+flaccid+paralysis+in+infants+and+young+children+with+enterovirus+71+infection%3A+MR+imaging+findings+and+clinical+correlates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Acute flaccid paralysis in infants and young children with enterovirus 71 infection: MR imaging findings and clinical correlates</span></div><div class="casAuthors">Chen C Y; Chang Y C; Huang C C; Lui C C; Lee K W; Huang S C</div><div class="citationInfo"><span class="NLM_cas:title">AJNR. American journal of neuroradiology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">200-5</span>
        ISSN:<span class="NLM_cas:issn">0195-6108</span>.
    </div><div class="casAbstract">BACKGROUND AND PURPOSE:  Enterovirus 71 (EV71) infection is now considered an important cause of childhood acute flaccid paralysis.  The purpose of our study was to determine whether EV71-infection-related acute flaccid paralysis in infants and young children has characteristic MR imaging patterns.  METHODS:  Seven infants and young children with acute paralysis of the upper or lower extremities and positive EV71 cultures underwent spinal MR studies during an outbreak of hand-foot-and-mouth disease in Taiwan in 1998.  RESULTS:  Acute paralysis was observed in one upper extremity in two patients, in one lower extremity in three patients, and in both lower extremities in two patients.  None of the patients had sensory impairment or bulbar palsy.  MR studies showed unilateral or bilateral hyperintense lesions in the anterior horn regions of the cord on T2-weighted images in six patients.  No abnormal signal was present in one patient.  Two of three patients who received intravenous injections of contrast material had ventral root enhancement on T1-weighted images.  One of them also had enhancement of the unilateral anterior horn cells.  At clinical follow-up, both patients with bilateral anterior horn abnormalities had residual motor weakness, whereas only one of the five patients with unilateral involvement had residual weakness.  CONCLUSION:  EV71 radiculomyelitis tends to be unilateral and to specifically involve both the anterior horn cells of the cord and the ventral roots.  MR imaging allows early detection of spinal cord and root lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSSyQVHwRse6f6BottYdIAWfW6udTcc2eY4-lEWRz35xbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3M7nslygtw%253D%253D&md5=fcfb26746adb20a36a5e4dd3c9dbc77b</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BY.%26aulast%3DChang%26aufirst%3DY.%2BC.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DLui%26aufirst%3DC.%2BC.%26aulast%3DLee%26aufirst%3DK.%2BW.%26aulast%3DHuang%26aufirst%3DS.%2BC.%26atitle%3DAcute%2520flaccid%2520paralysis%2520in%2520infants%2520and%2520young%2520children%2520with%2520enterovirus%252071%2520infection%253A%2520MR%2520imaging%2520findings%2520and%2520clinical%2520correlates%26jtitle%3DAm.%2520J.%2520Neuroradiol.%26date%3D2001%26volume%3D22%26spage%3D200%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNutt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span> <span> </span><span class="NLM_article-title">Neuropathology in 2 cases of fatal enterovirus type 71 infection from a recent epidemic in the People’s Republic of China: a histopathologic, immunohistochemical, and reverse transcription polymerase chain reaction study</span>. <i>Hum. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1288</span>– <span class="NLM_lpage">1295</span>, <span class="refDoi"> DOI: 10.1016/j.humpath.2009.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.humpath.2009.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19386354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSitbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2009&pages=1288-1295&author=Y.+Yangauthor=H.+Wangauthor=E.+Gongauthor=J.+Duauthor=X.+Zhaoauthor=M.+A.+McNuttauthor=S.+Wangauthor=Y.+Zhongauthor=Z.+Gaoauthor=J.+Zheng&title=Neuropathology+in+2+cases+of+fatal+enterovirus+type+71+infection+from+a+recent+epidemic+in+the+People%E2%80%99s+Republic+of+China%3A+a+histopathologic%2C+immunohistochemical%2C+and+reverse+transcription+polymerase+chain+reaction+study&doi=10.1016%2Fj.humpath.2009.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1cR"><div class="casContent"><span class="casTitleNuber">1c</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropathology in 2 cases of fatal enterovirus type 71 infection from a recent epidemic in the People's Republic of China: a histopathologic, immunohistochemical, and reverse transcription polymerase chain reaction study</span></div><div class="casAuthors">Yang, Yu; Wang, Hua; Gong, Encong; Du, Juan; Zhao, Xishun; McNutt, Michael A.; Wang, Shenglan; Zhong, Yanfeng; Gao, Zifen; Zheng, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Human Pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1288-1295</span>CODEN:
                <span class="NLM_cas:coden">HPCQA4</span>;
        ISSN:<span class="NLM_cas:issn">0046-8177</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Between late March and June of 2008, an outbreak of hand, foot, and mouth disease caused by enterovirus 71 occurred in China.  In this outbreak, more than 176,000 cases occurred, and at least 40 people died.  However, there has been no description of the neuropathol. features of fatal enterovirus 71 infection cases during this outbreak.  We report postmortem studies in 2 fatal cases of enterovirus 71 infection with examn. of the central nervous system using histopathol., immunohistochem., and reverse transcription polymerase chain reaction.  Characteristic features of acute encephalitis were found predominantly in the brain stem and spinal cord.  Abundant nos. of inflammatory cells with unusual irregularly rod-shaped and lobulated nuclei, which mimicked neutrophil morphol., were found both in abscess-like clusters assocd. with necrosis, as well as in inflammatory infiltrates.  Immunohistochem. showed that most of these cells were CD68 pos. and CD15 neg.  Viral antigens were found in the cytoplasm of neurons, neuronal processes, and inflammatory cells, most often assocd. with glial nodules.  The presence of enterovirus 71 was confirmed by reverse transcription polymerase chain reaction and sequencing.  Results indicate that immunohistochem. with CD68 and CD15 may offer some help in the differential diagnosis of brain stem encephalitis caused by enterovirus 71.  Postmortem reverse transcription polymerase chain reaction from brain stem tissues, which have already undergone fixation and histol. processing, can be an important diagnostic tool, which may be of particular value in patients who may have been misdiagnosed clin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAVuQnvSrkvbVg90H21EOLACvtfcHk0ljSvBJvOI_UJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSitbrF&md5=da609281c171d96e82a6138d2388c55e</span></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1016%2Fj.humpath.2009.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.humpath.2009.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGong%26aufirst%3DE.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMcNutt%26aufirst%3DM.%2BA.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DZheng%26aufirst%3DJ.%26atitle%3DNeuropathology%2520in%25202%2520cases%2520of%2520fatal%2520enterovirus%2520type%252071%2520infection%2520from%2520a%2520recent%2520epidemic%2520in%2520the%2520People%25E2%2580%2599s%2520Republic%2520of%2520China%253A%2520a%2520histopathologic%252C%2520immunohistochemical%252C%2520and%2520reverse%2520transcription%2520polymerase%2520chain%2520reaction%2520study%26jtitle%3DHum.%2520Pathol.%26date%3D2009%26volume%3D40%26spage%3D1288%26epage%3D1295%26doi%3D10.1016%2Fj.humpath.2009.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">McMinn, P. C.</span></span> <span> </span><span class="NLM_article-title">An overview of the evolution of enterovirus 71 and its clinical and public health significance</span>. <i>FEMS Microbiol Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">107</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2002.tb00601.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1111%2Fj.1574-6976.2002.tb00601.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=12007645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjtl2itLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2002&pages=91-107&author=P.+C.+McMinn&title=An+overview+of+the+evolution+of+enterovirus+71+and+its+clinical+and+public+health+significance&doi=10.1111%2Fj.1574-6976.2002.tb00601.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1dR"><div class="casContent"><span class="casTitleNuber">1d</span><div class="casTitle"><span class="NLM_cas:atitle">An overview of the evolution of enterovirus 71 and its clinical and public health significance</span></div><div class="casAuthors">McMinn, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-107</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Since its discovery in 1969, enterovirus 71 (EV71) has been recognized as a frequent cause of epidemics of hand-foot-and-mouth disease (HFMD) assocd. with severe neurol. sequelae in a small proportion of cases.  There has been a significant increase in EV71 epidemic activity throughout the Asia-Pacific region since 1997.  Recent HFMD epidemics in this region have been assocd. with a severe form of brainstem encephalitis assocd. with pulmonary edema and high case-fatality rates.  The emergence of large-scale epidemic activity in the Asia-Pacific region has been assocd. with the circulation of three genetic lineages that appear to be undergoing rapid evolutionary change.  Two of these lineages (B3 and B4) have not been described previously and appear to have arisen from an endemic focus in equatorial Asia, which has served as a source of virus for HFMD epidemics in Malaysia, Singapore and Australia.  The third lineage (C2) has previously been identified and was primarily responsible for the large HFMD epidemic in Taiwan during 1998.  As EV71 appears not to be susceptible to newly developed antiviral agents and a vaccine is not currently available, control of EV71 epidemics through high-level surveillance and public health intervention needs to be maintained and extended throughout the Asia-Pacific region.  Future research should focus on (1) understanding the mol. genetics of EV71 virulence, (2) identification of the receptor(s) for EV71, (3) development of antiviral agents to ameliorate the severity of neurol. disease and (4) vaccine development to control epidemics.  Following the successful experience of the poliomyelitis control program, it may be possible to control EV71 epidemics if an effective live-attenuated vaccine is developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6kmGHJuTlArVg90H21EOLACvtfcHk0ljSvBJvOI_UJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjtl2itLw%253D&md5=649be3c04036cf0f6f3534a955271318</span></div><a href="/servlet/linkout?suffix=cit1d&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2002.tb00601.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2002.tb00601.x%26sid%3Dliteratum%253Aachs%26aulast%3DMcMinn%26aufirst%3DP.%2BC.%26atitle%3DAn%2520overview%2520of%2520the%2520evolution%2520of%2520enterovirus%252071%2520and%2520its%2520clinical%2520and%2520public%2520health%2520significance%26jtitle%3DFEMS%2520Microbiol%2520Rev.%26date%3D2002%26volume%3D26%26spage%3D91%26epage%3D107%26doi%3D10.1111%2Fj.1574-6976.2002.tb00601.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span> <span> </span><span class="NLM_article-title">Viral and host proteins involved in picornavirus life cycle</span>. <i>J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1186/1423-0127-16-103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1186%2F1423-0127-16-103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19925687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BD1MjptFSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=103-117&author=J.+Y.+Linauthor=T.+C.+Chenauthor=K.+F.+Wengauthor=S.+C.+Changauthor=L.+L.+Chenauthor=S.+R.+Shih&title=Viral+and+host+proteins+involved+in+picornavirus+life+cycle&doi=10.1186%2F1423-0127-16-103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1eR"><div class="casContent"><span class="casTitleNuber">1e</span><div class="casTitle"><span class="NLM_cas:atitle">Viral and host proteins involved in picornavirus life cycle</span></div><div class="casAuthors">Lin Jing-Yi; Chen Tzu-Chun; Weng Kuo-Feng; Chang Shih-Cheng; Chen Li-Lien; Shih Shin-Ru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomedical science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Picornaviruses cause several diseases, not only in humans but also in various animal hosts.  For instance, human enteroviruses can cause hand-foot-and-mouth disease, herpangina, myocarditis, acute flaccid paralysis, acute hemorrhagic conjunctivitis, severe neurological complications, including brainstem encephalitis, meningitis and poliomyelitis, and even death.  The interaction between the virus and the host is important for viral replication, virulence and pathogenicity.  This article reviews studies of the functions of viral and host factors that are involved in the life cycle of picornavirus.  The interactions of viral capsid proteins with host cell receptors is discussed first, and the mechanisms by which the viral and host cell factors are involved in viral replication, viral translation and the switch from translation to RNA replication are then addressed.  Understanding how cellular proteins interact with viral RNA or viral proteins, as well as the roles of each in viral infection, will provide insights for the design of novel antiviral agents based on these interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3vmkIwk_wJedfST00W0BcfW6udTcc2eaqdAFLMlSnv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MjptFSqsg%253D%253D&md5=ab1a69bd6d303ff40fc25b1e49ad100a</span></div><a href="/servlet/linkout?suffix=cit1e&amp;dbid=16384&amp;doi=10.1186%2F1423-0127-16-103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1423-0127-16-103%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%2BY.%26aulast%3DChen%26aufirst%3DT.%2BC.%26aulast%3DWeng%26aufirst%3DK.%2BF.%26aulast%3DChang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DL.%2BL.%26aulast%3DShih%26aufirst%3DS.%2BR.%26atitle%3DViral%2520and%2520host%2520proteins%2520involved%2520in%2520picornavirus%2520life%2520cycle%26jtitle%3DJ.%2520Biomed.%2520Sci.%26date%3D2009%26volume%3D16%26spage%3D103%26epage%3D117%26doi%3D10.1186%2F1423-0127-16-103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit1f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. H.</span></span> <span> </span><span class="NLM_article-title">Enterovirus 71 infection of human dendritic cells</span>. <i>Exp. Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">1166</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.3181/0903-RM-116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.3181%2F0903-RM-116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ams7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2009&pages=1166-1173&author=Y.+W.+Linauthor=S.+W.+Wangauthor=Y.+Y.+Tungauthor=S.+H.+Chen&title=Enterovirus+71+infection+of+human+dendritic+cells&doi=10.3181%2F0903-RM-116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1fR"><div class="casContent"><span class="casTitleNuber">1f</span><div class="casTitle"><span class="NLM_cas:atitle">Enterovirus 71 infection of human dendritic cells</span></div><div class="casAuthors">Lin, Yu-Wen; Wang, Shainn-Wei; Tung, Yuk-Ying; Chen, Shun-Hua</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Biology and Medicine (Maywood, NJ, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1166-1173</span>CODEN:
                <span class="NLM_cas:coden">EBMMBE</span>;
        ISSN:<span class="NLM_cas:issn">1535-3702</span>.
    
            (<span class="NLM_cas:orgname">Society for Experimental Biology and Medicine</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) causes death and long-term neurol. sequelae in hundreds of thousands of young children, but its pathogenesis remains elusive.  Dendritic cells (DCs) play a crucial role in antiviral immunity by functioning as professional antigen-presenting cells to prime T cells and by secreting cytokines to modulate immune responses.  Here, we show that EV71 productively infected human immature DCs and expressed viral antigen in DCs.  EV71 entry into DCs was partially mediated by DC-SIGN.  Further analyses revealed that EV71 increased the viability, activation, release of cytokines, interleukin-6, interleukin-12, and tumor necrosis factor-α in DCs.  Moreover, EV71 enabled DCs to stimulate T-cell proliferation.  Collectively, these findings suggest that EV71 infection of human DCs in vivo is very likely to elicit protective immunity, because in infected mice, both T cells and IL-6 function to reduce mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBCp6P0rqREbVg90H21EOLACvtfcHk0ljlKZsJh9JiYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ams7fI&md5=248cc1c49295514a284bb2e1f716f3d3</span></div><a href="/servlet/linkout?suffix=cit1f&amp;dbid=16384&amp;doi=10.3181%2F0903-RM-116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3181%252F0903-RM-116%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DY.%2BW.%26aulast%3DWang%26aufirst%3DS.%2BW.%26aulast%3DTung%26aufirst%3DY.%2BY.%26aulast%3DChen%26aufirst%3DS.%2BH.%26atitle%3DEnterovirus%252071%2520infection%2520of%2520human%2520dendritic%2520cells%26jtitle%3DExp.%2520Biol.%2520Med.%26date%3D2009%26volume%3D234%26spage%3D1166%26epage%3D1173%26doi%3D10.3181%2F0903-RM-116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, A. Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, P.</span></span> <span> </span><span class="NLM_article-title">Pilot scale production of highly efficacious and stable enterovirus 71 vaccine candidates</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e34834</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0034834</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1371%2Fjournal.pone.0034834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=22529942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmtlyqtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e34834&author=A.+H.+Chouauthor=C.+C.+Liuauthor=C.+P.+Changauthor=M.+S.+Guoauthor=S.+Y.+Hsiehauthor=W.+H.+Yangauthor=H.+J.+Chaoauthor=C.+L.+Wuauthor=J.+L.+Huangauthor=M.+S.+Leeauthor=A.+Y.+C.+Huauthor=S.+C.+Linauthor=Y.+Y.+Huangauthor=M.+H.+Huauthor=Y.+H.+Chowauthor=J.+R.+Chiangauthor=J.+Y.+Changauthor=P.+Chong&title=Pilot+scale+production+of+highly+efficacious+and+stable+enterovirus+71+vaccine+candidates&doi=10.1371%2Fjournal.pone.0034834"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Pilot scale production of highly efficacious and stable enterovirus 71 vaccine candidates</span></div><div class="casAuthors">Chou, Ai-Hsiang; Liu, Chia-Chyi; Chang, Cheng-Peng; Guo, Meng-Shin; Hsieh, Shih-Yang; Yang, Wen-Hsueh; Chao, Hsin-Ju; Wu, Chien-Long; Huang, Ju-Lan; Lee, Min-Shi; Hu, Alan Yung-Chi; Lin, Sue-Chen; Huang, Yu-Yun; Hu, Mei-Hua; Chow, Yen-Hung; Chiang, Jen-Ron; Chang, Jui-Yuan; Chong, Pele</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e34834</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Enterovirus 71 (EV71) has caused several epidemics of hand, foot and mouth diseases (HFMD) in Asia and now is being recognized as an important neurotropic virus.  Effective medications and prophylactic vaccine against EV71 infection are urgently needed.  Based on the success of inactivated poliovirus vaccine, a prototype chem. inactivated EV71 vaccine candidate has been developed and currently in human phase 1 clin. trial.  Principal Finding: In this report, we present the development of a serum-free cell-based EV71 vaccine.  The optimization at each step of the manufg. process was investigated, characterized and quantified.  In the up-stream process development, different com. available cell culture media either contg. serum or serum-free was screened for cell growth and virus yield using the roller-bottle technol.  VP-SFM serum-free medium was selected based on the Vero cell growth profile and EV71 virus prodn.  After the up-stream processes (virus harvest, diafiltration and concn.), a combination of gel-filtration liq. chromatog. and/or sucrose-gradient ultracentrifugation down-stream purifn. processes were investigated at a pilot scale of 40 L each.  Although the combination of chromatog. and sucrose-gradient ultracentrifugation produced extremely pure EV71 infectious virus particles, the overall yield of vaccine was 7-10 % as detd. by a VP2-based quant. ELISA.  Using chromatog. as the downstream purifn., the virus yield was 30-43 %.  To retain the integrity of virus neutralization epitopes and the stability of the vaccine product, the best virus inactivation was found to be 0.025% formalin-treatment at 37°C for 3 to 6 days.  Furthermore, the formalin-inactivated virion vaccine candidate was found to be stable for >18 mo at 4°C and a microgram of viral proteins formulated with alum adjuvant could induce strong virus-neutralizing antibody responses in mice, rats, rabbits, and non-human primates.  Conclusion: These results provide valuable information supporting the current cell-based serum-free EV71 vaccine candidate going into human Phase I clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPI5iYNnTzpbVg90H21EOLACvtfcHk0ljlKZsJh9JiYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmtlyqtr8%253D&md5=843537d2d8f55429a4e64e8eb512d38b</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0034834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0034834%26sid%3Dliteratum%253Aachs%26aulast%3DChou%26aufirst%3DA.%2BH.%26aulast%3DLiu%26aufirst%3DC.%2BC.%26aulast%3DChang%26aufirst%3DC.%2BP.%26aulast%3DGuo%26aufirst%3DM.%2BS.%26aulast%3DHsieh%26aufirst%3DS.%2BY.%26aulast%3DYang%26aufirst%3DW.%2BH.%26aulast%3DChao%26aufirst%3DH.%2BJ.%26aulast%3DWu%26aufirst%3DC.%2BL.%26aulast%3DHuang%26aufirst%3DJ.%2BL.%26aulast%3DLee%26aufirst%3DM.%2BS.%26aulast%3DHu%26aufirst%3DA.%2BY.%2BC.%26aulast%3DLin%26aufirst%3DS.%2BC.%26aulast%3DHuang%26aufirst%3DY.%2BY.%26aulast%3DHu%26aufirst%3DM.%2BH.%26aulast%3DChow%26aufirst%3DY.%2BH.%26aulast%3DChiang%26aufirst%3DJ.%2BR.%26aulast%3DChang%26aufirst%3DJ.%2BY.%26aulast%3DChong%26aufirst%3DP.%26atitle%3DPilot%2520scale%2520production%2520of%2520highly%2520efficacious%2520and%2520stable%2520enterovirus%252071%2520vaccine%2520candidates%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De34834%26doi%3D10.1371%2Fjournal.pone.0034834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Z.</span></span> <span> </span><span class="NLM_article-title">Immunogenicity and safety of an enterovirus 71 vaccine in healthy Chinese children and infants: a randomised, double-blind, placebo-controlled phase 2 clinical trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">1037</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(12)61764-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2FS0140-6736%2812%2961764-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23352749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVymsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2013&pages=1037-1045&author=F.+C.+Zhuauthor=Z.+L.+Liangauthor=X.+L.+Liauthor=H.+M.+Geauthor=F.+Y.+Mengauthor=Q.+Y.+Maoauthor=Y.+T.+Zhangauthor=Y.+M.+Huauthor=Z.+Y.+Zhangauthor=J.+X.+Liauthor=F.+Gaoauthor=Q.+H.+Chenauthor=Q.+Y.+Zhuauthor=K.+Chuauthor=X.+Wuauthor=X.+Yaoauthor=H.+J.+Guoauthor=X.+Q.+Chenauthor=P.+Liuauthor=Y.+Y.+Dongauthor=F.+X.+Liauthor=X.+L.+Shenauthor=J.+Z.+Wang&title=Immunogenicity+and+safety+of+an+enterovirus+71+vaccine+in+healthy+Chinese+children+and+infants%3A+a+randomised%2C+double-blind%2C+placebo-controlled+phase+2+clinical+trial&doi=10.1016%2FS0140-6736%2812%2961764-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Immunogenicity and safety of an enterovirus 71 vaccine in healthy Chinese children and infants: a randomised, double-blind, placebo-controlled phase 2 clinical trial</span></div><div class="casAuthors">Zhu, Feng-Cai; Liang, Zheng-Lun; Li, Xiu-Ling; Ge, Heng-Ming; Meng, Fan-Yue; Mao, Qun-Ying; Zhang, Yun-Tao; Hu, Yue-Mei; Zhang, Zhen-Yu; Li, Jing-Xin; Gao, Fan; Chen, Qing-Hua; Zhu, Qi-Yan; Chu, Kai; Wu, Xing; Yao, Xin; Guo, Hui-Jie; Chen, Xiao-Qin; Liu, Pei; Dong, Yu-Ying; Li, Feng-Xiang; Shen, Xin-Liang; Wang, Jun-Zhi</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9871</span>),
    <span class="NLM_cas:pages">1037-1045</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Background: Enterovirus 71 (EV71) outbreaks are a socioeconomic burden, esp. in the western Pacific region.  Results of phase 1 clin. trials suggest an EV71 vaccine has a clin. acceptable safety profile and immunogenicity.  We aimed to assess the best possible dose and formulation, immunogenicity, and safety profile of this EV71 vaccine in healthy Chinese children.  Methods: This randomized, double-blind, placebo-controlled, phase 2 trial was undertaken at one site in Donghai County, Jiangsu Province, China.  Eligible participants were healthy boys or girls aged 6-36 mo.  Participants were randomly assigned (1:1:1:1:1) to receive either 160 U, 320 U, or 640 U alum-adjuvant EV71 vaccine, 640 U adjuvant-free EV71 vaccine, or a placebo (contg. alum adjuvant only), according to a blocked randomization list generated by SAS 9.1.  Participants and investigators were masked to the assignment.  The primary endpoint was anti-EV71 neutralizing antibody geometric mean titers (GMTs) at day 56, analyzed according to protocol.  The study is registered with ClinicalTrials.gov, no. NCT01399853.  Findings: We randomly assigned 1200 participants, 240 (120 aged 6-11 mo [infants] and 120 aged 12-36 mo [children]) of whom were assigned to each dose. 1106 participants completed the study and were included in the according-to-protocol anal.  The main reasons for dropout were withdrawal of consent and refusal to donate a blood sample.  Infants who received the 640 U adjuvant vaccine had the highest GMTs on day 56 (742·2 [95% CI 577·3-954·3]), followed by those who received the 320 U formulation (497·9 [383·1-647·0]).  For children, those who received the 320 U formulation had the highest GMTs on day 56 (1383·2 [1037·3-1844·5]).  Participants who received the vaccine had significantly higher GMTs than did who received placebo (p<0·0001).  For the subgroup of participants who were seroneg. at baseline, both infants and children who received the 640 U adjuvant vaccine had the highest GMTs on day 56 (522·8 [403·9-676·6] in infants and 708·4 [524·1-957·6] in children), followed by those who received the 320 U adjuvant vaccine (358·2 [280·5-457·5] in infants and 498·0 [383·4-646·9] in children).  549 (45·8%) Of 1200 participants (95 CI 42·9-48·6%) reported at least one injection-site or systemic adverse reaction, but the incidence of adverse reactions did not differ significantly between groups (p=0·36).  The 640 U alum-adjuvant vaccine group had a significantly higher incidence of induration than did the 640 U adjuvant-free group (p=0·001).  Interpretation: Taking immunogenicity, safety, and prodn. capacity into account, the 320 U alum-adjuvant formulation of the EV71 vaccine is probably the best possible formulation for phase 3 trials.  Funding: The National Science and Technol. Major Project (2011ZX10004-902) of the Chinese Ministry of Science and Technol., China's 12-5 National Major Infectious Disease Program (2012ZX10002-001), and Beijing Vigoo Biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_OLrmt_3gtrVg90H21EOLACvtfcHk0ljlKZsJh9JiYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVymsb0%253D&md5=d6d11a5249ed8ae3a8d4407dd0e7478a</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2812%2961764-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252812%252961764-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DF.%2BC.%26aulast%3DLiang%26aufirst%3DZ.%2BL.%26aulast%3DLi%26aufirst%3DX.%2BL.%26aulast%3DGe%26aufirst%3DH.%2BM.%26aulast%3DMeng%26aufirst%3DF.%2BY.%26aulast%3DMao%26aufirst%3DQ.%2BY.%26aulast%3DZhang%26aufirst%3DY.%2BT.%26aulast%3DHu%26aufirst%3DY.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%2BY.%26aulast%3DLi%26aufirst%3DJ.%2BX.%26aulast%3DGao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DQ.%2BH.%26aulast%3DZhu%26aufirst%3DQ.%2BY.%26aulast%3DChu%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DH.%2BJ.%26aulast%3DChen%26aufirst%3DX.%2BQ.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DDong%26aufirst%3DY.%2BY.%26aulast%3DLi%26aufirst%3DF.%2BX.%26aulast%3DShen%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DJ.%2BZ.%26atitle%3DImmunogenicity%2520and%2520safety%2520of%2520an%2520enterovirus%252071%2520vaccine%2520in%2520healthy%2520Chinese%2520children%2520and%2520infants%253A%2520a%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%2520phase%25202%2520clinical%2520trial%26jtitle%3DLancet%26date%3D2013%26volume%3D381%26spage%3D1037%26epage%3D1045%26doi%3D10.1016%2FS0140-6736%2812%2961764-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Stanway, G.</span></span> <span> </span><span class="NLM_article-title">Structure, function and evolution of picornaviruses</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">2483</span>– <span class="NLM_lpage">2501</span>, <span class="refDoi"> DOI: 10.1099/0022-1317-71-11-2483</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1099%2F0022-1317-71-11-2483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=2254747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK3MXjtVKi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1990&pages=2483-2501&author=G.+Stanway&title=Structure%2C+function+and+evolution+of+picornaviruses&doi=10.1099%2F0022-1317-71-11-2483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3aR"><div class="casContent"><span class="casTitleNuber">3a</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function and evolution of picornaviruses</span></div><div class="casAuthors">Stanway, Glyn</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2483-501</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    </div><div class="casAbstract">A review with 172 refs., illustrating how mol. and structural analyses have revealed the biol. diversity seen in picornaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDuBkYasUPF7Vg90H21EOLACvtfcHk0lgJ02FcFUA3Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXjtVKi&md5=7cd20db9e42c972c15ed6b85d0c37e22</span></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1099%2F0022-1317-71-11-2483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F0022-1317-71-11-2483%26sid%3Dliteratum%253Aachs%26aulast%3DStanway%26aufirst%3DG.%26atitle%3DStructure%252C%2520function%2520and%2520evolution%2520of%2520picornaviruses%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D1990%26volume%3D71%26spage%3D2483%26epage%3D2501%26doi%3D10.1099%2F0022-1317-71-11-2483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit3b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weady, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span> <span> </span><span class="NLM_article-title">Conservation of amino acids in human rhinovirus 3C protease correlates with broad-spectrum antiviral activity of Rupintrivir, a novel human rhinovirus 3C protease inhibitor</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">626</span>, <span class="refDoi"> DOI: 10.1128/AAC.49.2.619-626.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.49.2.619-626.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15673742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht12nt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2005&pages=619-626&author=S.+L.+Binfordauthor=F.+Maldonadoauthor=M.+A.+Brothersauthor=P.+T.+Weadyauthor=L.+S.+Zalmanauthor=J.+W.+Meadorauthor=D.+A.+Matthewsauthor=A.+K.+Patick&title=Conservation+of+amino+acids+in+human+rhinovirus+3C+protease+correlates+with+broad-spectrum+antiviral+activity+of+Rupintrivir%2C+a+novel+human+rhinovirus+3C+protease+inhibitor&doi=10.1128%2FAAC.49.2.619-626.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Conservation of amino acids in human rhinovirus 3C protease correlates with broad-spectrum antiviral activity of rupintrivir, a novel human rhinovirus 3C protease inhibitor</span></div><div class="casAuthors">Binford, S. L.; Maldonado, F.; Brothers, M. A.; Weady, P. T.; Zalman, L. S.; Meador, J. W., III; Matthews, D. A.; Patick, A. K.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">619-626</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The picornavirus 3C protease is required for the majority of proteolytic cleavages that occur during the viral life cycle.  Comparisons of published amino acid sequences from 6 human rhinoviruses (HRV) and 20 human enteroviruses (HEV) show considerable variability in the 3C protease-coding region but strict conservation of the catalytic triad residues.  Rupintrivir (formerly AG7088) is an irreversible inhibitor of HRV 3C protease with potent in vitro activity against all HRV serotypes (48 of 48), HEV strains (4 of 4), and untyped HRV field isolates (46 of 46) tested.  To better understand the relationship between in vitro antiviral activity and 3C protease-rupintrivir binding interactions, the authors performed nucleotide sequence analyses on an addnl. 21 HRV serotypes and 11 HRV clin. isolates.  Antiviral activity was also detd. for 23 HRV clin. isolates and four addnl. HEV strains.  Sequence comparison of 3C proteases (n = 58) show that 13 and 11 of the 14 amino acids that are involved in side chain interactions with rupintrivir are strictly conserved among HRV and HEV, resp.  These sequence analyses are consistent with the comparable in vitro antiviral potencies of rupintrivir against all HRV serotypes, HRV isolates, and HEV strains tested (50% effective concn. range, 3 to 183 nM; n = 125).  In summary, the conservation of crit. amino acid residues in 3C protease and the observation of potent, broad-spectrum antipicornavirus activity of rupintrivir highlight the advantages of 3C protease as an antiviral target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4MFFGS5_xkLVg90H21EOLACvtfcHk0lgJ02FcFUA3Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht12nt7Y%253D&md5=936a058a019545e186f5a6e5b4cec298</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.1128%2FAAC.49.2.619-626.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.49.2.619-626.2005%26sid%3Dliteratum%253Aachs%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DMaldonado%26aufirst%3DF.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DWeady%26aufirst%3DP.%2BT.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26atitle%3DConservation%2520of%2520amino%2520acids%2520in%2520human%2520rhinovirus%25203C%2520protease%2520correlates%2520with%2520broad-spectrum%2520antiviral%2520activity%2520of%2520Rupintrivir%252C%2520a%2520novel%2520human%2520rhinovirus%25203C%2520protease%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2005%26volume%3D49%26spage%3D619%26epage%3D626%26doi%3D10.1128%2FAAC.49.2.619-626.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit3c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Condon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Budahazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slsson, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafranca, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McElroy, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribskov, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S.</span></span> <span> </span><span class="NLM_article-title">Structure of human rhinovirus 3C protease reveals a trypsin-like polypeptide fold, RNA-binding site, and means for cleaving precursor polyprotein</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">761</span>– <span class="NLM_lpage">771</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(94)90059-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2F0092-8674%2894%2990059-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=7515772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK2cXksFejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1994&pages=761-771&author=D.+A.+Matthewsauthor=W.+W.+Smithauthor=R.+A.+Ferreauthor=B.+Condonauthor=G.+Budahaziauthor=W.+Slssonauthor=J.+E.+Villafrancaauthor=C.+A.+Jansonauthor=H.+E.+McElroyauthor=C.+L.+Gribskovauthor=S.+Worland&title=Structure+of+human+rhinovirus+3C+protease+reveals+a+trypsin-like+polypeptide+fold%2C+RNA-binding+site%2C+and+means+for+cleaving+precursor+polyprotein&doi=10.1016%2F0092-8674%2894%2990059-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3cR"><div class="casContent"><span class="casTitleNuber">3c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human rhinovirus 3C protease reveals a trypsin-like polypeptide folds, RNA-binding site, and means for cleaving precursor polyprotein</span></div><div class="casAuthors">Matthews, David A.; Smith, Ward W.; Ferre, Rose Ann; Condon, Brad; Budahal, Georgg; Sisson, Wes; Villafranca, J. E.; Janson, Cheryl A.; McElroy, H. E.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">761-71</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The structure of human rhinovirus-14 3C protease (3Cpro) has been detd. at 2.3 Å resoln. and refined to an R factor of 0.22.  This cysteine protease folds into two topol. equiv. six-stranded β barrels in this sense is similar to trypsin-like serine proteases.  However, there are differences in the lengths and positioning of individual β strands as well as in loops connecting elements of secondary structure.  The catalytic residues Cys-146, His-40, and Glu-71 are positioned as in serine proteases, but the oxyanion hole is moved 1-1.2 Å away.  Residues that bind to the 5' noncoding region of rhinovirus genomic RNA are located on the opposite side of the mol. from the active site.  Interactions between individual 3Cpro mols. in the crystal lattice suggest a model for intermol. proteolytic cleavage of the 3CD polyprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonBQyOVEP8krVg90H21EOLACvtfcHk0lgJ02FcFUA3Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXksFejtL8%253D&md5=7815a9833147d6b1aa7bb7067073415e</span></div><a href="/servlet/linkout?suffix=cit3c&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2894%2990059-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252894%252990059-0%26sid%3Dliteratum%253Aachs%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DSmith%26aufirst%3DW.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DCondon%26aufirst%3DB.%26aulast%3DBudahazi%26aufirst%3DG.%26aulast%3DSlsson%26aufirst%3DW.%26aulast%3DVillafranca%26aufirst%3DJ.%2BE.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DMcElroy%26aufirst%3DH.%2BE.%26aulast%3DGribskov%26aufirst%3DC.%2BL.%26aulast%3DWorland%26aufirst%3DS.%26atitle%3DStructure%2520of%2520human%2520rhinovirus%25203C%2520protease%2520reveals%2520a%2520trypsin-like%2520polypeptide%2520fold%252C%2520RNA-binding%2520site%252C%2520and%2520means%2520for%2520cleaving%2520precursor%2520polyprotein%26jtitle%3DCell%26date%3D1994%26volume%3D77%26spage%3D761%26epage%3D771%26doi%3D10.1016%2F0092-8674%2894%2990059-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit3d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M. S.</span></span> <span> </span><span class="NLM_article-title">Mutations at KFRDI and VGK domains of enterovirus 71 3C protease affect its RNA binding and proteolytic activities</span>. <i>J. Biomed. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">248</span>, <span class="refDoi"> DOI: 10.1007/BF02256567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1007%2FBF02256567" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=14966374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXht1Cnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=239-248&author=S.+R.+Shihauthor=C.+Chiangauthor=T.+C.+Chenauthor=C.+N.+Wuauthor=J.+T.+A.+Hsuauthor=J.+C.+Leeauthor=M.+J.+Hwangauthor=M.+L.+Liauthor=G.+W.+Chenauthor=M.+S.+Ho&title=Mutations+at+KFRDI+and+VGK+domains+of+enterovirus+71+3C+protease+affect+its+RNA+binding+and+proteolytic+activities&doi=10.1007%2FBF02256567"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3dR"><div class="casContent"><span class="casTitleNuber">3d</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations at KFRDI and VGK Domains of Enterovirus 71 3C Protease Affect Its RNA Binding and Proteolytic Activities</span></div><div class="casAuthors">Shih, Shin-Ru; Chiang, Chiayn; Chen, Tzu-Chun; Wu, Cheng-Nan; Hsu, John Tsu-An; Lee, Jin-Ching; Hwang, Ming-Jing; Li, Mei-Ling; Chen, Guang-Wu; Ho, Mei-Shan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomedical Science (Basel, Switzerland)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-248</span>CODEN:
                <span class="NLM_cas:coden">JBCIEA</span>;
        ISSN:<span class="NLM_cas:issn">1021-7770</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">The 3C protease (3Cpro) of enterovirus 71 (EV71) is a good mol. target for drug discovery.  Notably, this protease was found to possess RNA-binding activity.  The regions responsible for RNA binding were classified as KFRDI' (positions 82-86) and 'VGK' (positions 154-156) in 3Cpro by mutagenesis study.  Although the RNA-binding regions are structurally distinct from the catalytic site of EV71 3Cpro, mutations in the RNA-binding regions influenced 3Cpro proteolytic activity.  In contrast, mutations at the catalytic site had almost no influence on RNA binding ability.  We identified certain mutations within 3Cpro which abrogated both the RNA-binding activity of the expressed, recombinant, protease and the ability to rescue virus from an infectious full-length clone of EV71 (pEV71).  Interestingly, mutation at position 84 from Arg(R) to Lys(K) was found to retain good RNA binding and proteolytic activity for the recombinant 3Cpro; however, no virus could be rescued when pEV71 with the R84K mutation was introduced into the infectious copy.  Together, these results may provide useful information for using 3Cpro as the mol. target to develop anti-EV71 agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr3_Su1I1FU7Vg90H21EOLACvtfcHk0liT8_aV1Y5Gbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXht1Cnu7o%253D&md5=08d151fea6fc47f692f264fbe0357381</span></div><a href="/servlet/linkout?suffix=cit3d&amp;dbid=16384&amp;doi=10.1007%2FBF02256567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02256567%26sid%3Dliteratum%253Aachs%26aulast%3DShih%26aufirst%3DS.%2BR.%26aulast%3DChiang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DT.%2BC.%26aulast%3DWu%26aufirst%3DC.%2BN.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DHwang%26aufirst%3DM.%2BJ.%26aulast%3DLi%26aufirst%3DM.%2BL.%26aulast%3DChen%26aufirst%3DG.%2BW.%26aulast%3DHo%26aufirst%3DM.%2BS.%26atitle%3DMutations%2520at%2520KFRDI%2520and%2520VGK%2520domains%2520of%2520enterovirus%252071%25203C%2520protease%2520affect%2520its%2520RNA%2520binding%2520and%2520proteolytic%2520activities%26jtitle%3DJ.%2520Biomed.%2520Sci.%26date%3D2004%26volume%3D11%26spage%3D239%26epage%3D248%26doi%3D10.1007%2FBF02256567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit3e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Structures of EV71 RNA-dependent RNA polymerase in complex with substrate and analogue provide a drug target against the hand-foot-and-mouth disease pandemic in China</span>. <i>Protein Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1007/s13238-010-0061-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1007%2Fs13238-010-0061-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21203964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3cXoslelurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=491-500&author=Y.+Wuauthor=Z.+Louauthor=Y.+Miaoauthor=Y.+Yuauthor=H.+Dongauthor=W.+Pengauthor=M.+Bartlamauthor=X.+Liauthor=Z.+Rao&title=Structures+of+EV71+RNA-dependent+RNA+polymerase+in+complex+with+substrate+and+analogue+provide+a+drug+target+against+the+hand-foot-and-mouth+disease+pandemic+in+China&doi=10.1007%2Fs13238-010-0061-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3eR"><div class="casContent"><span class="casTitleNuber">3e</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of EV71 RNA-dependent RNA polymerase in complex with substrate and analogue provide a drug target against the hand-foot-and-mouth disease pandemic in China</span></div><div class="casAuthors">Wu, Yang; Lou, Zhiyong; Miao, Yi; Yu, Yue; Dong, Hui; Peng, Wei; Bartlam, Mark; Li, Xuemei; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">491-500</span>CODEN:
                <span class="NLM_cas:coden">PCREFB</span>;
        ISSN:<span class="NLM_cas:issn">1674-800X</span>.
    
            (<span class="NLM_cas:orgname">Higher Education Press</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71), one of the major causative agents for hand-foot-and-mouth disease (HFMD), has caused more than 100 deaths among Chinese children since March 2008.  The EV71 genome encodes an RNA-dependent RNA polymerase (RdRp), denoted 3Dpol, which is central for viral genome replication and is a key target for the discovery of specific antiviral therapeutics.  Here we report the crystal structures of EV71 RdRp (3Dpol) and in complex with substrate guanosine-5'-triphosphate and analog 5-bromouridine-5'-triphosphate best to 2.4 Å resoln.  The structure of EV71 RdRp (3Dpol) has a wider open thumb domain compared with the most closely related crystal structure of poliovirus RdRp.  And the EV71 RdRp (3Dpol) complex with GTP or Br-UTP bounded shows two distinct movements of the polymerase by substrate or analog binding.  The model of the complex with the template:primer derived by superimposition with foot-and-mouth disease virus (FMDV) 3D/RNA complex reveals the likely recognition and binding of template:primer RNA by the polymerase.  These results together provide a mol. basis for EV71 RNA replication and reveal a potential target for anti-EV71 drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQj64xy57fGLVg90H21EOLACvtfcHk0liT8_aV1Y5Gbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXoslelurw%253D&md5=44d2d2b455afdb16db34f748ebe1297a</span></div><a href="/servlet/linkout?suffix=cit3e&amp;dbid=16384&amp;doi=10.1007%2Fs13238-010-0061-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13238-010-0061-7%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DMiao%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DStructures%2520of%2520EV71%2520RNA-dependent%2520RNA%2520polymerase%2520in%2520complex%2520with%2520substrate%2520and%2520analogue%2520provide%2520a%2520drug%2520target%2520against%2520the%2520hand-foot-and-mouth%2520disease%2520pandemic%2520in%2520China%26jtitle%3DProtein%2520Cell%26date%3D2010%26volume%3D1%26spage%3D491%26epage%3D500%26doi%3D10.1007%2Fs13238-010-0061-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit3f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Antiviral drug discovery for the treatment of enterovirus 71 infections</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">194</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.antiviral.2012.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23261847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFSku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=183-194&author=L.+Shangauthor=M.+Xuauthor=Z.+Yin&title=Antiviral+drug+discovery+for+the+treatment+of+enterovirus+71+infections&doi=10.1016%2Fj.antiviral.2012.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3fR"><div class="casContent"><span class="casTitleNuber">3f</span><div class="casTitle"><span class="NLM_cas:atitle">Antiviral drug discovery for the treatment of enterovirus 71 infections</span></div><div class="casAuthors">Shang, Luqing; Xu, Mengying; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-194</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Enterovirus 71 (EV71) is a small, pos.-sense, single-stranded RNA virus in the genus Enterovirus, family Picornavirus.  It causes hand, foot, and mouth disease in infants and children, which in a small percentage of cases progresses to central nervous system infection, ranging from aseptic meningitis to fatal encephalitis.  Sporadic cases of EV71 infection occur throughout the world, but large epidemics have occurred recently in Southeast Asia and China.  There are currently no approved vaccines or antiviral therapies for the prevention or treatment of EV71 infection.  This paper reviews efforts to develop antiviral therapies against EV71.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjzLwxTUQnX7Vg90H21EOLACvtfcHk0liT8_aV1Y5Gbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFSku7c%253D&md5=ad739907255ef3e637e5160fd79bf4f6</span></div><a href="/servlet/linkout?suffix=cit3f&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DAntiviral%2520drug%2520discovery%2520for%2520the%2520treatment%2520of%2520enterovirus%252071%2520infections%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D97%26spage%3D183%26epage%3D194%26doi%3D10.1016%2Fj.antiviral.2012.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span> <span> </span><span class="NLM_article-title">Enterovirus 71 3C protease cleaves a novel target Cstf-64 and inhibits cellular polyadenylation</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">e1000593</span>, <span class="refDoi"> DOI: 10.1371/journal.ppat.1000593</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1371%2Fjournal.ppat.1000593" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19779565" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000593&author=K.+F.+Wengauthor=M.+L.+Liauthor=C.+T.+Hungauthor=S.+R.+Shih&title=Enterovirus+71+3C+protease+cleaves+a+novel+target+Cstf-64+and+inhibits+cellular+polyadenylation&doi=10.1371%2Fjournal.ppat.1000593"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000593%26sid%3Dliteratum%253Aachs%26aulast%3DWeng%26aufirst%3DK.%2BF.%26aulast%3DLi%26aufirst%3DM.%2BL.%26aulast%3DHung%26aufirst%3DC.%2BT.%26aulast%3DShih%26aufirst%3DS.%2BR.%26atitle%3DEnterovirus%252071%25203C%2520protease%2520cleaves%2520a%2520novel%2520target%2520Cstf-64%2520and%2520inhibits%2520cellular%2520polyadenylation%26jtitle%3DPLoS%2520Pathog.%26date%3D2009%26volume%3D5%26spage%3De1000593%26doi%3D10.1371%2Fjournal.ppat.1000593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiew, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">11754</span>– <span class="NLM_lpage">11762</span>, <span class="refDoi"> DOI: 10.1128/JVI.01348-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.01348-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=22915796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=11754-11762&author=Y.+Kimauthor=S.+Lovellauthor=K.+C.+Tiewauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=K.+P.+Battaileauthor=W.+C.+Groutasauthor=K.+O.+Chang&title=Broad-spectrum+antivirals+against+3C+or+3C-like+proteases+of+picornaviruses%2C+noroviruses%2C+and+coronaviruses&doi=10.1128%2FJVI.01348-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span></div><div class="casAuthors">Kim, Yunjeong; Lovell, Scott; Tiew, Kok-Chuan; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Battaile, Kevin P.; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11754-11762</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phylogenetic anal. has demonstrated that some pos.-sense RNA viruses can be classified into the picornavirus-like super-cluster, which includes picornaviruses, caliciviruses, and coronaviruses.  These viruses possess 3C or 3C-like proteases (3Cpro or 3CLpro, resp.), which contain a typical chymotrypsin-like fold and a catalytic triad (or dyad) with a Cys residue as a nucleophile.  The conserved key sites of 3Cpro or 3CLpro may serve as attractive targets for the design of broad-spectrum antivirals for multiple viruses in the supercluster.  We previously reported the structure-based design and synthesis of potent protease inhibitors of Norwalk virus (NV), a member of the Caliciviridae family.  We report herein the broad-spectrum antiviral activities of three compds. possessing a common dipeptidyl residue with different warheads, i.e., an aldehyde (GC373), a bisulfite adduct (GC376), and an α-ketoamide (GC375), against viruses that belong to the supercluster.  All compds. were highly effective against the majority of tested viruses, with half-maximal inhibitory concns. in the high nanomolar or low micromolar range in enzyme- and/or cell-based assays and with high therapeutic indexes.  We also report the high-resoln. X-ray cocrystal structures of NV 3CLpro-, poliovirus 3Cpro-, and transmissible gastroenteritis virus 3CLpro- GC376 inhibitor complexes, which show the compd. covalently bound to a nucleophilic Cys residue in the catalytic site of the corresponding protease.  We conclude that these compds. have the potential to be developed as antiviral therapeutics aimed at a single virus or multiple viruses in the picornavirus-like supercluster by targeting 3Cpro or 3CLpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDn4MUH7bIlLVg90H21EOLACvtfcHk0liW-k35SYC_7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M&md5=c7fe984b52490199a2c2ef333c22fb98</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1128%2FJVI.01348-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01348-12%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DTiew%26aufirst%3DK.%2BC.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3DBroad-spectrum%2520antivirals%2520against%25203C%2520or%25203C-like%2520proteases%2520of%2520picornaviruses%252C%2520noroviruses%252C%2520and%2520coronaviruses%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D11754%26epage%3D11762%26doi%3D10.1128%2FJVI.01348-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit4c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span> <span> </span><span class="NLM_article-title">What coronavirus3C-like protease tells us: From structure, substrate selectivity, to inhibitor design</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1002/med.21783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fmed.21783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2021&pages=1-34&author=M.+Xiongauthor=H.+Suauthor=W.+Zhaoauthor=H.+Xieauthor=Q.+Shaoauthor=Y.+Xu&title=What+coronavirus3C-like+protease+tells+us%3A+From+structure%2C+substrate+selectivity%2C+to+inhibitor+design&doi=10.1002%2Fmed.21783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.1002%2Fmed.21783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21783%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DShao%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DY.%26atitle%3DWhat%2520coronavirus3C-like%2520protease%2520tells%2520us%253A%2520From%2520structure%252C%2520substrate%2520selectivity%252C%2520to%2520inhibitor%2520design%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D3%26spage%3D1%26epage%3D34%26doi%3D10.1002%2Fmed.21783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gwaltney, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi-Burris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span> <span> </span><span class="NLM_article-title">Phase II, randomized, double-blind, placebo-controlled studies of Ruprintrivir nasal spray 2-percent suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3907</span>– <span class="NLM_lpage">3916</span>, <span class="refDoi"> DOI: 10.1128/AAC.47.12.3907-3916.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.47.12.3907-3916.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=14638501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2003&pages=3907-3916&author=F.+G.+Haydenauthor=R.+B.+Turnerauthor=J.+M.+Gwaltneyauthor=K.+Chi-Burrisauthor=M.+Gerstenauthor=P.+Hsyuauthor=A.+K.+Patickauthor=G.+J.+Smithauthor=L.+S.+Zalman&title=Phase+II%2C+randomized%2C+double-blind%2C+placebo-controlled+studies+of+Ruprintrivir+nasal+spray+2-percent+suspension+for+prevention+and+treatment+of+experimentally+induced+rhinovirus+colds+in+healthy+volunteers&doi=10.1128%2FAAC.47.12.3907-3916.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">Phase II, randomized, double-blind, placebo-controlled studies of ruprintrivir nasal spray 2% suspension for prevention and treatment of experimentally induced rhinovirus colds in healthy volunteers</span></div><div class="casAuthors">Hayden, Frederick G.; Turner, Ronald B.; Gwaltney, Jack M.; Chi-burris, Kathy; Gersten, Merril; Hsyu, Poe; Patick, Amy K.; Smith, George J., III; Zalman, Leora S.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3907-3916</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Human rhinovirus (HRV) infections are usually self-limited but may be assocd. with serious consequences, particularly in those with asthma and chronic respiratory disease.  Effective antiviral agents are needed for preventing and treating HRV illnesses.  Ruprintrivir (Agouron Pharmaceuticals, Inc., San Diego, Calif.) selectively inhibits HRV 3C protease and shows potent, broad-spectrum anti-HRV activity in vitro.  We conducted three double-blind, placebo-controlled clin. trials in 202 healthy volunteers to assess the activity of ruprintrivir in exptl. HRV infection.  Subjects were randomized to receive intranasal ruprintrivir (8 mg) or placebo sprays as prophylaxis (two or five times daily [2×/day or 5×/day] for 5 days) starting 6 h before infection or as treatment (5×/day for 4 days) starting 24 h after infection.  Ruprintrivir prophylaxis reduced the proportion of subjects with pos. viral cultures (for 5×/day dosing groups, 44% for ruprintrivir treatment group vs. 70% for placebo treatment group [P = 0.03]; for 2×/day dosing groups, 60% for ruprintrivir group vs. 92% for placebo group [P = 0.004]) and viral titers but did not decrease the frequency of colds.  Ruprintrivir treatment reduced the mean total daily symptom score (2.2 for ruprintrivir treatment group and 3.3 for the placebo treatment group [P = 0.014]) by 33%.  Secondary endpoints, including viral titers, individual symptom scores, and nasal discharge wts., were also reduced by ruprintrivir treatment.  Overall, ruprintrivir was well tolerated; blood-tinged mucus and nasal passage irritation were the most common adverse effects reported.  Pharmacokinetic anal. of plasma and nasal ruprintrivir concns. revealed intranasal drug residence with minimal systemic absorption.  Results from these studies in exptl. rhinoviral infection support continued investigation of intranasal ruprintrivir in the setting of natural HRV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoudaxRiBJLg7Vg90H21EOLACvtfcHk0liW-k35SYC_7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsV2nu7w%253D&md5=6d6c33f56af0c4f45145b291ace2dba1</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1128%2FAAC.47.12.3907-3916.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.47.12.3907-3916.2003%26sid%3Dliteratum%253Aachs%26aulast%3DHayden%26aufirst%3DF.%2BG.%26aulast%3DTurner%26aufirst%3DR.%2BB.%26aulast%3DGwaltney%26aufirst%3DJ.%2BM.%26aulast%3DChi-Burris%26aufirst%3DK.%26aulast%3DGersten%26aufirst%3DM.%26aulast%3DHsyu%26aufirst%3DP.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26atitle%3DPhase%2520II%252C%2520randomized%252C%2520double-blind%252C%2520placebo-controlled%2520studies%2520of%2520Ruprintrivir%2520nasal%2520spray%25202-percent%2520suspension%2520for%2520prevention%2520and%2520treatment%2520of%2520experimentally%2520induced%2520rhinovirus%2520colds%2520in%2520healthy%2520volunteers%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2003%26volume%3D47%26spage%3D3907%26epage%3D3916%26doi%3D10.1128%2FAAC.47.12.3907-3916.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shie, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P. H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of 3C protease inhibitors as anti-enterovirus 71 agents</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">7388</span>– <span class="NLM_lpage">7398</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmc.2008.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=18583140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps12jtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=7388-7398&author=C.+J.+Kuoauthor=J.+J.+Shieauthor=J.+M.+Fangauthor=G.+R.+Yenauthor=J.+T.+A.+Hsuauthor=H.+G.+Liuauthor=S.+N.+Tsengauthor=S.+C.+Changauthor=C.+Y.+Leeauthor=S.+R.+Shihauthor=P.+H.+Liang&title=Design%2C+synthesis%2C+and+evaluation+of+3C+protease+inhibitors+as+anti-enterovirus+71+agents&doi=10.1016%2Fj.bmc.2008.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and evaluation of 3C protease inhibitors as anti-enterovirus 71 agents</span></div><div class="casAuthors">Kuo, Chih-Jung; Shie, Jiun-Jie; Fang, Jim-Min; Yen, Guei-Rung; Hsu, John T.-A.; Liu, Hun-Ge; Tseng, Sung-Nain; Chang, Shih-Cheng; Lee, Ching-Yin; Shih, Shin-Ru; Liang, Po-Huang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">7388-7398</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human enterovirus (EV) belongs to the picornavirus family, which consists of over 200 medically relevant viruses.  A peptidomimetic inhibitor AG7088 was developed to inhibit the 3C protease of rhinovirus (a member of the family), a chymotrypsin-like protease required for viral replication, by forming a covalent bond with the active site Cys residue.  In this study, we have prepd. the recombinant 3C protease from EV71 (TW/2231/98), a particular strain which causes severe outbreaks in Asia, and developed inhibitors against the protease and the viral replication.  For inhibitor design, the P3 group of AG7088, which is not interacting with the rhinovirus protease, was replaced with a series of cinnamoyl derivs. directly linked to P2 group through an amide bond to simplify the synthesis.  While the replacement caused decreased potency, the activity can be largely improved by substituting the α,β-unsatd. ester with an aldehyde at the P1' position.  The best inhibitor 10b showed EC50 of 18 nM without apparent toxicity (CC50 > 25 μM).  Our study provides potent inhibitors of the EV71 3C protease as anti-EV71 agents and facilitates the combinatorial synthesis of derivs. for further improving the inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0kBp35E-n6bVg90H21EOLACvtfcHk0lhnE8NwejlXkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps12jtbg%253D&md5=a319b7e1253e6f14592c9317469c4d74</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DC.%2BJ.%26aulast%3DShie%26aufirst%3DJ.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%2BM.%26aulast%3DYen%26aufirst%3DG.%2BR.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%2BA.%26aulast%3DLiu%26aufirst%3DH.%2BG.%26aulast%3DTseng%26aufirst%3DS.%2BN.%26aulast%3DChang%26aufirst%3DS.%2BC.%26aulast%3DLee%26aufirst%3DC.%2BY.%26aulast%3DShih%26aufirst%3DS.%2BR.%26aulast%3DLiang%26aufirst%3DP.%2BH.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%25203C%2520protease%2520inhibitors%2520as%2520anti-enterovirus%252071%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D7388%26epage%3D7398%26doi%3D10.1016%2Fj.bmc.2008.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship study of peptidomimetic aldehydes as enterovirus 71 3C protease inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">573</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.08.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.ejmech.2016.08.064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=27614190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2jurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2016&pages=559-573&author=Y.+Zhaiauthor=Y.+Maauthor=F.+Maauthor=Q.+Nieauthor=X.+Renauthor=Y.+Wangauthor=L.+Shangauthor=Z.+Yin&title=Structure-activity+relationship+study+of+peptidomimetic+aldehydes+as+enterovirus+71+3C+protease+inhibitors&doi=10.1016%2Fj.ejmech.2016.08.064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5cR"><div class="casContent"><span class="casTitleNuber">5c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationship study of peptidomimetic aldehydes as enterovirus 71 3C protease inhibitors</span></div><div class="casAuthors">Zhai, Yangyang; Ma, Yuying; Ma, Fei; Nie, Quandeng; Ren, Xuejiao; Wang, Yaxin; Shang, Luqing; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">559-573</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of peptidomimetic aldehydes were designed, synthesized, and evaluated for their biochem. activity against 3C protease (3Cpro) and anti-enterovirus 71 (EV71) activity in vitro.  Mol. docking revealed that 5s (IC50 = 0.22 ± 0.07 μM, EC50 = 0.18 ± 0.05 μM) could bind well to the active site of EV71 3Cpro, which was consistent with the biol. data compared to ref. 5a (IC50 = 0.54 ± 0.02 μM, EC50 = 0.26 ± 0.07 μM).  Structure and relationship study led to the discovery of aldehyde 5x (IC50 = 0.10 ± 0.02 μM, EC50 = 0.11 ± 0.07 μM), which exhibited the most potent 3Cpro inhibitory and antiviral activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMBUXTkD4dyLVg90H21EOLACvtfcHk0lhnE8NwejlXkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2jurnN&md5=41cabd5d56ab0164e48cbd7806dbf2c7</span></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.08.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.08.064%26sid%3Dliteratum%253Aachs%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DF.%26aulast%3DNie%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DStructure-activity%2520relationship%2520study%2520of%2520peptidomimetic%2520aldehydes%2520as%2520enterovirus%252071%25203C%2520protease%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D124%26spage%3D559%26epage%3D573%26doi%3D10.1016%2Fj.ejmech.2016.08.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">A peptidyl aldehyde NK-1.8k suppresses EV71 and EV68 infection by targeting at 3C<sup>pro</sup></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2636</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1128/AAC.00049-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.00049-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=25691647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms12qsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=2636-2646&author=Y.+Wangauthor=B.+Yangauthor=Y.+Zhaiauthor=Z.+Yinauthor=Y.+Sunauthor=Z.+Rao&title=A+peptidyl+aldehyde+NK-1.8k+suppresses+EV71+and+EV68+infection+by+targeting+at+3Cpro&doi=10.1128%2FAAC.00049-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5dR"><div class="casContent"><span class="casTitleNuber">5d</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl aldehyde NK-1.8k suppresses enterovirus 71 and enterovirus 68 infection by targeting protease 3C</span></div><div class="casAuthors">Wang, Yaxin; Yang, Ben; Zhai, Yangyang; Yin, Zheng; Sun, Yuna; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2636-2646</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus (EV) is one of the major causative agents of hand, foot, and mouth disease in the Pacific-Asia region.  In particular, EV71 causes severe central nervous system infections, and the fatality rates from EV71 infection are high.  Moreover, an outbreak of respiratory illnesses caused by an emerging EV, EV68, recently occurred among over 1,000 young children in the United States and was also assocd. with neurol. infections.  Although enterovirus has emerged as a considerable global public health threat, no antiviral drug for clin. use is available.  In the present work, we screened our compd. library for agents targeting viral protease and identified a peptidyl aldehyde, NK-1.8k, that inhibits the proliferation of different EV71 strains and one EV68 strain and that had a 50% effective concn. of 90 nM.  Low cytotoxicity (50% cytotoxic concn., >200 μM) indicated a high selective index of over 2,000.  We further characterized a single amino acid substitution inside protease 3C (3Cpro), N69S, which conferred EV71 resistance to NK-1.8k, possibly by increasing the flexibility of the substrate binding pocket of 3Cpro.  The combination of NK-1.8k and an EV71 RNA-dependent RNA polymerase inhibitor or entry inhibitor exhibited a strong synergistic anti-EV71 effect.  Our findings suggest that NK-1.8k could potentially be developed for anti-EV therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMAtO795xx4LVg90H21EOLACvtfcHk0lhnE8NwejlXkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms12qsr4%253D&md5=197d1a7a3af9eb32db0bef6c2dc741de</span></div><a href="/servlet/linkout?suffix=cit5d&amp;dbid=16384&amp;doi=10.1128%2FAAC.00049-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00049-15%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DA%2520peptidyl%2520aldehyde%2520NK-1.8k%2520suppresses%2520EV71%2520and%2520EV68%2520infection%2520by%2520targeting%2520at%25203Cpro%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D2636%26epage%3D2646%26doi%3D10.1128%2FAAC.00049-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span> <span> </span><span class="NLM_article-title">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9414</span>– <span class="NLM_lpage">9420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9414-9420&author=Y.+Zhaiauthor=X.+Zhaoauthor=Z.+Cuiauthor=M.+Wangauthor=Y.+Wangauthor=L.+Liauthor=Q.+Sunauthor=X.+Yangauthor=D.+Zengauthor=Y.+Liuauthor=Y.+Sunauthor=Z.+Louauthor=L.+Shangauthor=Z.+Yin&title=Cyanohydrin+as+an+Anchoring+Group+for+Potent+and+Selective+Inhibitors+of+Enterovirus+71+3C+Protease&doi=10.1021%2Facs.jmedchem.5b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5eR"><div class="casContent"><span class="casTitleNuber">5e</span><div class="casTitle"><span class="NLM_cas:atitle">Cyanohydrin as an Anchoring Group for Potent and Selective Inhibitors of Enterovirus 71 3C Protease</span></div><div class="casAuthors">Zhai, Yangyang; Zhao, Xiangshuai; Cui, Zhengjie; Wang, Man; Wang, Yaxin; Li, Linfeng; Sun, Qi; Yang, Xi; Zeng, Debin; Liu, Ying; Sun, Yuna; Lou, Zhiyong; Shang, Luqing; Yin, Zheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9414-9420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyanohydrin derivs. as enterovirus 71 (EV71) 3C protease (3Cpro) inhibitors have been synthesized and assayed for their biochem. and antiviral activities.  Compared with the reported inhibitors, cyanohydrins (1S,2S,2'S,5S)-I and (1R,2S,2'S,5S)-I exhibited significantly improved activity and attractive selectivity profiles against other proteases, which were a result of the specific interactions between the cyanohydrin moiety and the catalytic site of 3Cpro.  Cyanohydrin as an anchoring group with high selectivity and excellent inhibitory activity represents a useful choice for cysteine protease inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxejr9UxByVLVg90H21EOLACvtfcHk0lh5eIxjJ4HOIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVeqsr%252FK&md5=e43c6b3002e7c61cf8419bc83782be79</span></div><a href="/servlet/linkout?suffix=cit5e&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01013%26sid%3Dliteratum%253Aachs%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DCui%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DShang%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DZ.%26atitle%3DCyanohydrin%2520as%2520an%2520Anchoring%2520Group%2520for%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Enterovirus%252071%25203C%2520Protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9414%26epage%3D9420%26doi%3D10.1021%2Facs.jmedchem.5b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">Application of dually activated Michael acceptor to the rational design of reversible covalent inhibitor for enterovirus 71 3C protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6146</span>– <span class="NLM_lpage">6162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00387</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00387" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6146-6162&author=Y.+Maauthor=L.+Liauthor=S.+Heauthor=C.+Shangauthor=Y.+Sunauthor=N.+Liuauthor=T.+D.+Meekauthor=Y.+Wangauthor=L.+Shang&title=Application+of+dually+activated+Michael+acceptor+to+the+rational+design+of+reversible+covalent+inhibitor+for+enterovirus+71+3C+protease&doi=10.1021%2Facs.jmedchem.9b00387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5fR"><div class="casContent"><span class="casTitleNuber">5f</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Dually Activated Michael Acceptor to the Rational Design of Reversible Covalent Inhibitor for Enterovirus 71 3C Protease</span></div><div class="casAuthors">Ma, Yuying; Li, Linfeng; He, Shuai; Shang, Chengyou; Sun, Yang; Liu, Ning; Meek, Thomas D.; Wang, Yaxin; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6146-6162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted covalent inhibitors (TCIs) have attracted growing attention from the pharmaceutical industry in recent decades because they have potential advantages in terms of efficacy, selectivity, and safety.  TCIs have recently evolved into a new version with reversibility that can be systematically modulated.  This feature may diminish the risk of haptenization and help optimize the drug-target residence time as needed.  The enteroviral 3C protease (3Cpro) is a valuable therapeutic target, but the development of 3Cpro inhibitors is far from satisfactory.  Therefore, we aimed to apply a reversible TCI approach to the design of novel 3Cpro inhibitors.  The introduction of various substituents onto the α-carbon of classical Michael acceptors yielded inhibitors bearing several classes of warheads.  Using steady-state kinetics and biomol. mass spectrometry, we confirmed the mode of reversible covalent inhibition and elucidated the mechanism by which the potency and reversibility were affected by electronic and steric factors.  This research produced several potent inhibitors with good selectivity and suitable reversibility; moreover, it validated the reversible TCI approach in the field of viral infection, suggesting broader applications in the design of reversible covalent inhibitors for other proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkpF34VsXG8bVg90H21EOLACvtfcHk0lh5eIxjJ4HOIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGqsLjI&md5=2de757026f2542bff8092060a72107a5</span></div><a href="/servlet/linkout?suffix=cit5f&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00387%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DL.%26atitle%3DApplication%2520of%2520dually%2520activated%2520Michael%2520acceptor%2520to%2520the%2520rational%2520design%2520of%2520reversible%2520covalent%2520inhibitor%2520for%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6146%26epage%3D6162%26doi%3D10.1021%2Facs.jmedchem.9b00387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunn, A von.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, A de.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: structure-based design, synthesis, and activity assessment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4562</span>– <span class="NLM_lpage">4578</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01828</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01828" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4562-4578&author=L.+Zhangauthor=D.+Linauthor=Y.+Kusovauthor=Y.+Nianauthor=Q.+Maauthor=J.+Wangauthor=A+von.+Brunnauthor=P.+Leyssenauthor=K.+Lankoauthor=J.+Neytsauthor=A+de.+Wildeauthor=E.+J.+Snijderauthor=H.+Liuauthor=R.+Hilgenfeld&title=%CE%B1-Ketoamides+as+broad-spectrum+inhibitors+of+coronavirus+and+enterovirus+replication%3A+structure-based+design%2C+synthesis%2C+and+activity+assessment&doi=10.1021%2Facs.jmedchem.9b01828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5gR"><div class="casContent"><span class="casTitleNuber">5g</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Kusov, Yuri; Nian, Yong; Ma, Qingjun; Wang, Jiang; von Brunn, Albrecht; Leyssen, Pieter; Lanko, Kristina; Neyts, Johan; de Wilde, Adriaan; Snijder, Eric J.; Liu, Hong; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4562-4578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate.  Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs.  In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases.  Six crystal structures of protease:inhibitor complexes were detd. as part of this study.  Compds. synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic.  Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera.  The best near-equipotent inhibitors, (I) [R1 = cyclopentylmethyl (P2) and R1 = cyclohexylmethyl (P2)], display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures.  In Huh7 cells, I [R1 = cyclohexylmethyl (P2)] exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWpcfl4wTRLLVg90H21EOLACvtfcHk0liha-Ee4eXepg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D&md5=60d91a8ecf916856a844d835ecf84d5f</span></div><a href="/servlet/linkout?suffix=cit5g&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01828%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DNian%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBrunn%26aufirst%3DA%2Bvon.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DLanko%26aufirst%3DK.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DWilde%26aufirst%3DA%2Bde.%26aulast%3DSnijder%26aufirst%3DE.%2BJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketoamides%2520as%2520broad-spectrum%2520inhibitors%2520of%2520coronavirus%2520and%2520enterovirus%2520replication%253A%2520structure-based%2520design%252C%2520synthesis%252C%2520and%2520activity%2520assessment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4562%26epage%3D4578%26doi%3D10.1021%2Facs.jmedchem.9b01828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5h"><span><span class="NLM_label">(h) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, L.</span></span> <span> </span><span class="NLM_article-title">4-Iminooxazolidin-2-one as a bioisostere of the cyanohydrin moiety: Inhibitors of enterovirus 71 3C protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10333</span>– <span class="NLM_lpage">10339</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01335</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01335" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10333-10339&author=Y.+Maauthor=C.+Shangauthor=P.+Yangauthor=L.+Liauthor=Y.+Zhaiauthor=Z.+Yinauthor=B.+Wangauthor=L.+Shang&title=4-Iminooxazolidin-2-one+as+a+bioisostere+of+the+cyanohydrin+moiety%3A+Inhibitors+of+enterovirus+71+3C+protease&doi=10.1021%2Facs.jmedchem.8b01335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5hR"><div class="casContent"><span class="casTitleNuber">5h</span><div class="casTitle"><span class="NLM_cas:atitle">4-Iminooxazolidin-2-one as a Bioisostere of the Cyanohydrin Moiety: Inhibitors of Enterovirus 71 3C Protease</span></div><div class="casAuthors">Ma, Yuying; Shang, Chengyou; Yang, Peng; Li, Linfeng; Zhai, Yangyang; Yin, Zheng; Wang, Binghe; Shang, Luqing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10333-10339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A recently reported potent inhibitor of enterovirus 71 3C protease, (R)-1, was found to have stability and potential toxicity issues due to the presence of a cyanohydrin moiety.  Modifying the labile cyanohydrin moiety, by serendipity, led to the discovery of 4-iminooxazolidin-2-one-based inhibitors I and II with potent inhibitory activity and significantly improved stability.  In vivo pharmacokinetic studies of I also demonstrated high plasma exposure and moderate half-life.  These compds. have shown potential of becoming anti-EV71 drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7c_GeTuqpN7Vg90H21EOLACvtfcHk0liha-Ee4eXepg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7I&md5=b180c88252dd62093c79ba660549454e</span></div><a href="/servlet/linkout?suffix=cit5h&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01335%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DShang%26aufirst%3DL.%26atitle%3D4-Iminooxazolidin-2-one%2520as%2520a%2520bioisostere%2520of%2520the%2520cyanohydrin%2520moiety%253A%2520Inhibitors%2520of%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10333%26epage%3D10339%26doi%3D10.1021%2Facs.jmedchem.8b01335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5i"><span><span class="NLM_label">(i) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of anti-EV71 agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3346</span>– <span class="NLM_lpage">3350</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.05.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5i&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmcl.2016.05.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5i&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=27234148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5i&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos12kt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=3346-3350&author=P.+Liauthor=B.+Yangauthor=F.+Haoauthor=P.+Wangauthor=H.+Heauthor=L.+Huangauthor=X.+Zhangauthor=S.+Zhangauthor=X.+Pengauthor=K.+Yinauthor=J.+Huauthor=X.+Chenauthor=Z.+Guauthor=L.+Wangauthor=L.+Shenauthor=G.+Huauthor=N.+Liauthor=J.+Liauthor=S.+Chenauthor=W.+Xiaoauthor=Z.+Wangauthor=Q.+Guoauthor=X.+Changauthor=L.+Zhangauthor=Q.+Caiauthor=T.+Lin&title=Design%2C+synthesis%2C+and+biological+evaluation+of+anti-EV71+agents&doi=10.1016%2Fj.bmcl.2016.05.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5iR"><div class="casContent"><span class="casTitleNuber">5i</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and biological evaluation of anti-EV71 agents</span></div><div class="casAuthors">Li, Peng; Yang, Bailing; Hao, Fei; Wang, Ping; He, Haiying; Huang, Lei; Zhang, Xuan; Zhang, Shengbin; Peng, Xuanjia; Yin, Ke; Hu, Jiao; Chen, Xinsheng; Gu, Zhengxian; Wang, Li; Shen, Liang; Hu, Guoping; Li, Ning; Li, Jian; Chen, Shuhui; Xiao, Wei; Wang, Zhenzhong; Guo, Qingming; Chang, Xiujuan; Zhang, Lanjun; Cai, Qixu; Lin, Tianwei</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3346-3350</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) is a major causative agent of hand, foot and mouth disease (HFMD), which can spread its infections to the central nervous and other systems with severe consequences.  In this article, design, chem. synthesis, and biol. evaluation of various anti-EV71 agents which incorporate Michael acceptors are described.  Further SAR study demonstrated that lactone type of Michael acceptor provided a new lead of anti-EV71 drug candidates with high anti-EV71 activity in cell-based assay and enhanced mouse plasma stability.  One of the most potent compds. [N-((3S)-6-(4-fluorobenzyl)-2-methyl-4,7-dioxo-7-(((S,E)-1-(2-oxodihydrofuran-3(2H)-ylidene)-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)heptan-3-yl)-5-methylisoxazole-3-carboxamide, cell-based anti-EV71 EC50 = 0.028 μM] showed acceptable stability profile towards mouse plasma, which resulted in promising pharmacokinetics in mouse via IP administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWJtv6RNsjvbVg90H21EOLACvtfcHk0liha-Ee4eXepg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos12kt74%253D&md5=2813fbe22c4f4ceff01180d3f1fbddd8</span></div><a href="/servlet/linkout?suffix=cit5i&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.05.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.05.036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DHao%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DYin%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DChang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DT.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520anti-EV71%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D3346%26epage%3D3350%26doi%3D10.1016%2Fj.bmcl.2016.05.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5j"><span><span class="NLM_label">(j) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shivanna, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kankanamalage, A. C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum inhibitors against 3C-like proteases of feline coronaviruses and feline caliciviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">4942</span>– <span class="NLM_lpage">4950</span>, <span class="refDoi"> DOI: 10.1128/JVI.03688-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5j&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.03688-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5j&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=25694593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5j&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFajsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=4942-4950&author=Y.+Kimauthor=V.+Shivannaauthor=S.+Narayananauthor=A.+M.+Priorauthor=S.+Weerasekaraauthor=D.+H.+Huaauthor=A.+C.+G.+Kankanamalageauthor=W.+C.+Groutasauthor=K.-O.+Chang&title=Broad-spectrum+inhibitors+against+3C-like+proteases+of+feline+coronaviruses+and+feline+caliciviruses&doi=10.1128%2FJVI.03688-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5jR"><div class="casContent"><span class="casTitleNuber">5j</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum inhibitors against 3C-like proteases of feline coronaviruses and feline caliciviruses</span></div><div class="casAuthors">Kim, Yunjeong; Shivanna, Vinay; Narayanan, Sanjeev; Prior, Allan M.; Weerasekara, Sahani; Hua, Duy H.; Kankanamalage, Anushka C. Galasiti; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4942-4950</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Feline infectious peritonitis and virulent, systemic calicivirus infection are caused by certain types of feline coronaviruses (FCoVs) and feline caliciviruses (FCVs), resp., and are important infectious diseases with high fatality rates in members of the Felidae family.  While FCoV and FCV belong to two distinct virus families, the Coronaviridae and the Caliciviridae, resp., they share a dependence on viral 3C-like protease (3CLpro) for their replication.  Since 3CLpro is functionally and structurally conserved among these viruses and essential for viral replication, 3CLpro is considered a potential target for the design of antiviral drugs with broad-spectrum activities against these distinct and highly important viral infections.  However, small-mol. inhibitors against the 3CLpro enzymes of FCoV and FCV have not been previously identified.  In this study, derivs. of peptidyl compds. targeting 3CLpro were synthesized and evaluated for their activities against FCoV and FCV.  The structures of compds. that showed potent dual antiviral activities with a wide margin of safety were identified and are discussed.  Furthermore, the in vivo efficacy of 3CLpro inhibitors was evaluated using a mouse model of coronavirus infection.  I.p. administration of two 3CLpro inhibitors in mice infected with murine hepatitis virus A59, a hepatotropic coronavirus, resulted in significant redns. in virus titers and pathol. lesions in the liver compared to the findings for the controls.  These results suggest that the series of 3CLpro inhibitors described here may have the potential to be further developed as therapeutic agents against these important viruses in domestic and wild cats.  This study provides important insights into the structure and function relationships of 3CLpro for the design of antiviral drugs with broader antiviral activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIjU5Ipe2fLVg90H21EOLACvtfcHk0lhMbKhcxxymaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFajsb0%253D&md5=b57a99f8574d04454f0236f62da168f7</span></div><a href="/servlet/linkout?suffix=cit5j&amp;dbid=16384&amp;doi=10.1128%2FJVI.03688-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03688-14%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DShivanna%26aufirst%3DV.%26aulast%3DNarayanan%26aufirst%3DS.%26aulast%3DPrior%26aufirst%3DA.%2BM.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DKankanamalage%26aufirst%3DA.%2BC.%2BG.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3DBroad-spectrum%2520inhibitors%2520against%25203C-like%2520proteases%2520of%2520feline%2520coronaviruses%2520and%2520feline%2520caliciviruses%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D4942%26epage%3D4950%26doi%3D10.1128%2FJVI.03688-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5k"><span><span class="NLM_label">(k) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moroze, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and bioevaluation of viral 3C and 3C-like protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6317</span>– <span class="NLM_lpage">6320</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5k&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmcl.2013.09.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5k&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=24125888" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5k&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1WjurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6317-6320&author=A.+M.+Priorauthor=Y.+Kimauthor=S.+Weerasekaraauthor=M.+Morozeauthor=K.+R.+Allistonauthor=R.+A.+Uyauthor=W.+C.+Groutasauthor=K.+O.+Changauthor=D.+H.+Hua&title=Design%2C+synthesis%2C+and+bioevaluation+of+viral+3C+and+3C-like+protease+inhibitors&doi=10.1016%2Fj.bmcl.2013.09.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5kR"><div class="casContent"><span class="casTitleNuber">5k</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and bioevaluation of viral 3C and 3C-like protease inhibitors</span></div><div class="casAuthors">Prior, Allan M.; Kim, Yunjeong; Weerasekara, Sahani; Moroze, Meghan; Alliston, Kevin R.; Uy, Roxanne Adeline Z.; Groutas, William C.; Chang, Kyeong-Ok; Hua, Duy H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6317-6320</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A class of tripeptidyl transition state inhibitors contg. a P1 glutamine surrogate, a P2 leucine, and a P3 arylalanines, was found to potently inhibit Norwalk virus replication in enzyme and cell based assays.  An array of warheads, including aldehyde, α-ketoamide, bisulfite adduct, and α-hydroxyphosphonate transition state mimic, was also investigated.  Tripeptidyls 2 and 6 possess antiviral activities against noroviruses, human rhinovirus, severe acute respiratory syndrome coronavirus, and coronavirus 229E, suggesting a broad range of antiviral activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtxJZJuCaVpLVg90H21EOLACvtfcHk0lhMbKhcxxymaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1WjurfI&md5=4e71ce4a328d813f0b53ca153296f55b</span></div><a href="/servlet/linkout?suffix=cit5k&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.070%26sid%3Dliteratum%253Aachs%26aulast%3DPrior%26aufirst%3DA.%2BM.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DMoroze%26aufirst%3DM.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DUy%26aufirst%3DR.%2BA.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26aulast%3DHua%26aufirst%3DD.%2BH.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520bioevaluation%2520of%2520viral%25203C%2520and%25203C-like%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6317%26epage%3D6320%26doi%3D10.1016%2Fj.bmcl.2013.09.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit5l"><span><span class="NLM_label">(l) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerawarna, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prior, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uy, R. A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aravapalli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lushington, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. j.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic inhibitors of 3C and 3C-like proteases of picornavirus, norovirus, and coronavirus</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3709</span>– <span class="NLM_lpage">3712</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5l&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmcl.2013.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5l&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23727045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5l&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosFWqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=3709-3712&author=S.+R.+Mandadapuauthor=P.+M.+Weerawarnaauthor=A.+M.+Priorauthor=R.+A.+Z.+Uyauthor=S.+Aravapalliauthor=K.+R.+Allistonauthor=G.+H.+Lushingtonauthor=Y.+j.+Kimauthor=D.+H.+Huaauthor=K.+O.+Changauthor=W.+C.+Groutas&title=Macrocyclic+inhibitors+of+3C+and+3C-like+proteases+of+picornavirus%2C+norovirus%2C+and+coronavirus&doi=10.1016%2Fj.bmcl.2013.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5lR"><div class="casContent"><span class="casTitleNuber">5l</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic inhibitors of 3C and 3C-like proteases of picornavirus, norovirus, and coronavirus</span></div><div class="casAuthors">Mandadapu, Sivakoteswara Rao; Weerawarna, Pathum M.; Prior, Allan M.; Uy, Roxanne Adeline Z.; Aravapalli, Sridhar; Alliston, Kevin R.; Lushington, Gerald H.; Kim, Yunjeong; Hua, Duy H.; Chang, Kyeong-Ok; Groutas, William C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3709-3712</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The design, synthesis, and in vitro evaluation of the first macrocyclic inhibitor I of 3C and 3C-like proteases of picornavirus, norovirus, and coronavirus are reported.  The in vitro inhibitory activity (50% effective concn.) of the macrocyclic inhibitor toward enterovirus 3C protease (CVB3 Nancy strain), and coronavirus (SARS-CoV) and norovirus 3C-like proteases, was detd. to be 1.8, 15.5 and 5.1 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9gBpCKI1MrVg90H21EOLACvtfcHk0lhMbKhcxxymaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosFWqsbc%253D&md5=c0498882427ca6d13bd5cbbc452bac7d</span></div><a href="/servlet/linkout?suffix=cit5l&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DWeerawarna%26aufirst%3DP.%2BM.%26aulast%3DPrior%26aufirst%3DA.%2BM.%26aulast%3DUy%26aufirst%3DR.%2BA.%2BZ.%26aulast%3DAravapalli%26aufirst%3DS.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DLushington%26aufirst%3DG.%2BH.%26aulast%3DKim%26aufirst%3DY.%2Bj.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DChang%26aufirst%3DK.%2BO.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26atitle%3DMacrocyclic%2520inhibitors%2520of%25203C%2520and%25203C-like%2520proteases%2520of%2520picornavirus%252C%2520norovirus%252C%2520and%2520coronavirus%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D3709%26epage%3D3712%26doi%3D10.1016%2Fj.bmcl.2013.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batugo, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 6. Structure-activity studies of orally bioavailable, 2-Pyridone-containing peptidomimetics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1607</span>– <span class="NLM_lpage">1623</span>, <span class="refDoi"> DOI: 10.1021/jm010469k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010469k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhslegs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1607-1623&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=E.+L.+Brownauthor=F.+C.+Maldonadoauthor=S.+A.+Fuhrmanauthor=L.+S.+Zalmanauthor=T.+Tuntlandauthor=C.+A.+Leeauthor=A.+K.+Patickauthor=D.+A.+Matthewsauthor=T.+F.+Hendricksonauthor=M.+B.+Kosaauthor=B.+Liuauthor=M.+R.+Batugoauthor=J.+R.+Gleesonauthor=S.+K.+Sakataauthor=L.+J.+Chenauthor=M.+C.+Guzmanauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=S.+T.+Worland&title=Structure-based+design%2C+synthesis%2C+and+biological+evaluation+of+irreversible+human+rhinovirus+3C+protease+inhibitors.+6.+Structure-activity+studies+of+orally+bioavailable%2C+2-Pyridone-containing+peptidomimetics&doi=10.1021%2Fjm010469k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 6. Structure-Activity Studies of Orally Bioavailable, 2-Pyridone-Containing Peptidomimetics</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Brown, Edward L.; Maldonado, Fausto C.; Fuhrman, Shella A.; Zalman, Leora S.; Tuntland, Tove; Lee, Caroline A.; Patick, Amy K.; Matthews, David A.; Hendrickson, Thomas F.; Kosa, Maha B.; Liu, Bo; Batugo, Minerva R.; Gleeson, Jean-Paul R.; Sakata, Sylvie K.; Chen, Lijian; Guzman, Mark C.; Meador, James W., III; Ferre, Rose Ann; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1607-1623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various 2-pyridone-contg. human rhinovirus (HRV) 3C protease (3CP) inhibitors are described.  These compds. are comprised of a peptidomimetic binding determinant and a Michael acceptor moiety, which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The 2-pyridone-contg. inhibitors typically display improved 3CP inhibition properties relative to related peptide-derived mols. along with more favorable antiviral properties.  The cocrystal structure of one pyridone-derived 3CP inhibitor, peptidomimetic I, complexed with HRV-2 3CP is also described along with certain ab initio conformation analyses.  Optimization of the 2-pyridone-contg. compds. is shown to provide several highly active 3CP inhibitors (kobs/[I] > 500 000 M-1 s-1) that function as potent antirhinoviral agents (EC50 = <0.05 μM) against multiple virus serotypes in cell culture.  One 2-pyridone-contg. 3CP inhibitor, isoxazolyl deriv. II, is shown to be bioavailable in the dog after oral dosing (F = 48%).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj9kk44U_ydLVg90H21EOLACvtfcHk0lhUfA0omJjLtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhslegs7c%253D&md5=e15f157fa1af06cac037fc39f2a36d5b</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1021%2Fjm010469k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010469k%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMaldonado%26aufirst%3DF.%2BC.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBatugo%26aufirst%3DM.%2BR.%26aulast%3DGleeson%26aufirst%3DJ.%2BR.%26aulast%3DSakata%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DL.%2BJ.%26aulast%3DGuzman%26aufirst%3DM.%2BC.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520irreversible%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors.%25206.%2520Structure-activity%2520studies%2520of%2520orally%2520bioavailable%252C%25202-Pyridone-containing%2520peptidomimetics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1607%26epage%3D1623%26doi%3D10.1021%2Fjm010469k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Z.</span></span> <span> </span><span class="NLM_article-title">Rupintrivir is a promising candidate for treating severe cases of enterovirus-71 infection: Evaluation of antiviral efficacy in a murine infection model</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2012.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.antiviral.2012.12.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23295352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlKgt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2013&pages=264-269&author=X.+Zhangauthor=Z.+Songauthor=B.+Qinauthor=X.+Zhangauthor=L.+Chenauthor=Y.+Huauthor=Z.+Yuan&title=Rupintrivir+is+a+promising+candidate+for+treating+severe+cases+of+enterovirus-71+infection%3A+Evaluation+of+antiviral+efficacy+in+a+murine+infection+model&doi=10.1016%2Fj.antiviral.2012.12.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Rupintrivir is a promising candidate for treating severe cases of enterovirus-71 infection: Evaluation of antiviral efficacy in a murine infection model</span></div><div class="casAuthors">Zhang, Xiaonan; Song, Zhigang; Qin, Boyin; Zhang, Xiaoling; Chen, Lixiang; Hu, Yunwen; Yuan, Zhenghong</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">264-269</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Enterovirus-71 (EV71) infections can cause life-threatening diseases with neurol. symptoms.  Currently, no direct targeting antivirals are available to combat severe EV71 infection.  Rupintrivir (AG7088) is a compd. originally designed for Rhinovirus 3C protease.  Previous computational analyses by us and crystallog. studies by others suggested that rupintrivir is also a high affinity inhibitor to EV71 3C.  Thus, we aimed to further evaluate its anti-EV71 activity in vivo at clin. acceptable doses.  It was obsd. that administration of rupintrivir in suckling mice largely protected them from limb paralysis and dramatically improved survival (38.5% DMSO vs. 90.9% at 0.1 mg/kg, p = 0.006).  Histol., immunohistochem. and quant. RT-PCR analyses confirmed that rupintrivir profoundly alleviated virus induced necrotizing myositis, suppressed viral RNA and blocked EV71 VP1 expression in various tissues.  In conclusion, we established that rupintrivir can strongly contain the spread of EV71 infection in vivo at a clin. acceptable dose (as low as 0.1 mg/kg).  As its safety has been fully tested in previous clin. trials, rupintrivir is suitable for immediate evaluation of potential benefits in EV71-infected individuals with life-threatening neurol. symptoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgPcXAts672rVg90H21EOLACvtfcHk0lhUfA0omJjLtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlKgt7k%253D&md5=53409a7532cd1692ed426e59f5b24534</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2012.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2012.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DQin%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DZ.%26atitle%3DRupintrivir%2520is%2520a%2520promising%2520candidate%2520for%2520treating%2520severe%2520cases%2520of%2520enterovirus-71%2520infection%253A%2520Evaluation%2520of%2520antiviral%2520efficacy%2520in%2520a%2520murine%2520infection%2520model%26jtitle%3DAntiviral%2520Res.%26date%3D2013%26volume%3D97%26spage%3D264%26epage%3D269%26doi%3D10.1016%2Fj.antiviral.2012.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diem, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2444</span>– <span class="NLM_lpage">2450</span>, <span class="refDoi"> DOI: 10.1128/AAC.43.10.2444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.43.10.2444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10508022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=2444-2450&author=A.+K.+Patickauthor=S.+L.+Binfordauthor=M.+A.+Brothersauthor=R.+L.+Jacksonauthor=C.+E.+Fordauthor=M.+D.+Diemauthor=F.+Maldonadoauthor=P.+S.+Dragovichauthor=R.+Zhouauthor=T.+J.+Prinsauthor=S.+A.+Fuhrmanauthor=J.+W.+Meadorauthor=L.+S.+Zalmanauthor=D.+A.+Matthewsauthor=S.+T.+Worland&title=In+vitro+antiviral+activity+of+AG7088%2C+a+potent+inhibitor+of+human+rhinovirus+3C+protease&doi=10.1128%2FAAC.43.10.2444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span></div><div class="casAuthors">Patick, A. K.; Binford, S. L.; Brothers, M. A.; Jackson, R. L.; Ford, C. E.; Diem, M. D.; Maldonado, F.; Dragovich, P. S.; Zhou, R.; Prins, T. J.; Fuhrman, S. A.; Meador, J. W.; Zalman, L. S.; Matthews, D. A.; Worland, S. T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2444-2450</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">AG-7088 is a potent, irreversible inhibitor of human rhinovirus (HRV) 3C protease (inactivation rate const. kobs/[I] = 1,470,000±440,000 M-1 s-1 for HRV 14) that was discovered by protein structure-based drug design methodologies.  In H1-HeLa and MRC-5 cell protection assays, AG-7088 inhibited the replication of all HRV serotypes (48 of 48) tested with a mean 50% EC50 of 0.023 μM (range 0.003-0.081 μM) and a mean EC90 of 0.082 μM (range 0.018-0.261 μM), as well as that of related picornaviruses including coxsackieviruses A21 and B3, enterovirus 70, and echovirus 11.  No decreases in the antiviral activity of AG-7088 were obsd. when assays were performed in the presence of α1-acid glycoprotein or mucin proteins found in nasal secretions.  The 50% cytotoxic concn. of AG-7088 was >1000 μM, yielding a therapeutic index of >12,346 to >333,333.  In a single-cycle, time-of-addn. assay, AG-7088 had antiviral activity when added up to 6 h after infection.  A compd. targeting viral attachment and/or uncoating (WIN-51711) was effective only when added at the initiation of virus infection.  Direct inhibition of 3C proteolytic activity in infected cells treated with AG-7088 was demonstrated by SDS-polyacrylamide gel electrophoresis anal. of radiolabeled proteins, which showed a dose-dependent accumulation of viral precursor polyproteins and a decrease of processed protein products.  The broad spectrum of antiviral activity of AG-7088, combined with its efficacy even when added late in the virus life cycle, highlights the advantages of 3C protease as a target and suggests that AG7088 is a promising clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojpMqAUz2M3LVg90H21EOLACvtfcHk0lhUfA0omJjLtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D&md5=70e98480d38e133c96866005b0578f75</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.10.2444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.10.2444%26sid%3Dliteratum%253Aachs%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DJackson%26aufirst%3DR.%2BL.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DDiem%26aufirst%3DM.%2BD.%26aulast%3DMaldonado%26aufirst%3DF.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520of%2520AG7088%252C%2520a%2520potent%2520inhibitor%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D2444%26epage%3D2450%26doi%3D10.1128%2FAAC.43.10.2444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of enterovirus 71 3C protease complexed with rupintrivir reveal the roles of catalytically important residues</span>. <i>J. Virology.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">10021</span>– <span class="NLM_lpage">10030</span>, <span class="refDoi"> DOI: 10.1128/JVI.05107-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.05107-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21813612" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=10021-10030&author=J.+Wangauthor=T.+Fanauthor=X.+Yaoauthor=Z.+Wuauthor=L.+Guoauthor=L.+Leiauthor=J.+Wangauthor=M.+Wangauthor=Q.+Jinauthor=S.+Cui&title=Crystal+structures+of+enterovirus+71+3C+protease+complexed+with+rupintrivir+reveal+the+roles+of+catalytically+important+residues&doi=10.1128%2FJVI.05107-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6dR"><div class="casContent"><span class="casTitleNuber">6d</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of enterovirus 71 3C protease complexed with rupintrivir reveal the roles of catalytically important residues</span></div><div class="casAuthors">Wang, Jing; Fan, Tingting; Yao, Xue; Wu, Zhiqiang; Guo, Li; Lei, Xiaobo; Wang, Jianwei; Wang, Meitian; Jin, Qi; Cui, Sheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10021-10030</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) is the primary pathogenic cause of hand-foot-mouth disease (HFMD), but an effective antiviral drug currently is unavailable.  Rupintrivir, an inhibitor against human rhinovirus (HRV), has potent antiviral activities against EV71.  We detd. the high-resoln. crystal structures of the EV71 3Cpro/rupintrivir complex, showing that although rupintrivir interacts with EV71 3Cpro similarly to HRV 3Cpro, the C-terminus of the inhibitor cannot accommodate the leaving-group pockets of EV71 3Cpro.  Our structures reveal that EV71 3Cpro possesses a surface-recessive S2' pocket that is not present in HRV 3Cpro that contributes to the addnl. substrate binding affinity.  Combined with mutagenic studies, we demonstrated that catalytic Glu71 is irreplaceable for maintaining the overall architecture of the active site and, most importantly, the productive conformation of catalytic His40.  We discovered the role of a previously uncharacterized residue, Arg39 of EV71 3Cpro, that can neutralize the neg. charge of Glu71, which may subsequently assist deprotonation of His40 during proteolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplfsf-EHIhPLVg90H21EOLACvtfcHk0liZ-wws4_uXFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurzI&md5=1893b96430b7f6f3d0992916bb61b8a5</span></div><a href="/servlet/linkout?suffix=cit6d&amp;dbid=16384&amp;doi=10.1128%2FJVI.05107-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.05107-11%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DT.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DCui%26aufirst%3DS.%26atitle%3DCrystal%2520structures%2520of%2520enterovirus%252071%25203C%2520protease%2520complexed%2520with%2520rupintrivir%2520reveal%2520the%2520roles%2520of%2520catalytically%2520important%2520residues%26jtitle%3DJ.%2520Virology.%26date%3D2011%26volume%3D85%26spage%3D10021%26epage%3D10030%26doi%3D10.1128%2FJVI.05107-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">Enterovirus 71 and Coxsackievirus A16 3C proteases: binding to Rupintrivir and their substrates and anti-hand, foot, and mouth disease virus drug design</span>. <i>J. Virology.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">10319</span>– <span class="NLM_lpage">10331</span>, <span class="refDoi"> DOI: 10.1128/JVI.00787-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FJVI.00787-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21795339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2011&pages=10319-10331&author=G.+Luauthor=J.+Qiauthor=Z.+Chenauthor=X.+Xuauthor=F.+Gaoauthor=D.+Linauthor=W.+Qianauthor=H.+Liuauthor=H.+Jiangauthor=J.+Yanauthor=G.+F.+Gao&title=Enterovirus+71+and+Coxsackievirus+A16+3C+proteases%3A+binding+to+Rupintrivir+and+their+substrates+and+anti-hand%2C+foot%2C+and+mouth+disease+virus+drug+design&doi=10.1128%2FJVI.00787-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6eR"><div class="casContent"><span class="casTitleNuber">6e</span><div class="casTitle"><span class="NLM_cas:atitle">Enterovirus 71 and coxsackievirus A16 3C proteases: binding to rupintrivir and their substrates and anti-hand, foot, and mouth disease virus drug design</span></div><div class="casAuthors">Lu, Guangwen; Qi, Jianxun; Chen, Zhujun; Xu, Xiang; Gao, Feng; Lin, Daizong; Qian, Wangke; Liu, Hong; Jiang, Hualiang; Yan, Jinghua; Gao, George F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">10319-10331</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Enterovirus 71 (EV71) and coxsackievirus A16 (CVA16) are the major causative agents of hand, foot, and mouth disease (HFMD), which is prevalent in Asia.  Thus far, there are no prophylactic or therapeutic measures against HFMD.  The 3C proteases from EV71 and CVA16 play important roles in viral replication and are therefore ideal drug targets.  By using biochem., mutational, and structural approaches, we broadly characterized both proteases.  A series of high-resoln. structures of the free or substrate-bound enzymes were solved.  These structures, together with our cleavage specificity assay, well explain the marked substrate preferences of both proteases for particular P4, P1, and P1' residue types, as well as the relative malleability of the P2 amino acid.  More importantly, the complex structures of EV71 and CVA16 3Cs with rupintrivir, a specific human rhinovirus (HRV) 3C protease inhibitor, were solved.  These structures reveal a half-closed S2 subsite and a size-reduced S1' subsite that limit the access of the P1' group of rupintrivir to both enzymes, explaining the reported low inhibition activity of the compd. toward EV71 and CVA16.  In conclusion, the detailed characterization of both proteases in this study could direct us to a proposal for rational design of EV71/CVA16 3C inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD7JDuc7TN2rVg90H21EOLACvtfcHk0liZ-wws4_uXFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlWiurvI&md5=9e9cee092cfbc470f57ecaddd32064e4</span></div><a href="/servlet/linkout?suffix=cit6e&amp;dbid=16384&amp;doi=10.1128%2FJVI.00787-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00787-11%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DQian%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DEnterovirus%252071%2520and%2520Coxsackievirus%2520A16%25203C%2520proteases%253A%2520binding%2520to%2520Rupintrivir%2520and%2520their%2520substrates%2520and%2520anti-hand%252C%2520foot%252C%2520and%2520mouth%2520disease%2520virus%2520drug%2520design%26jtitle%3DJ.%2520Virology.%26date%3D2011%26volume%3D85%26spage%3D10319%26epage%3D10331%26doi%3D10.1128%2FJVI.00787-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6f"><span><span class="NLM_label">(f) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, T.</span></span> <span> </span><span class="NLM_article-title">Structures of Enterovirus 71 3C proteinase (strain E2004104-TW-CDC) and its complex with rupintrivir</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1107/S0907444913002862</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444913002862" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=23633597" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvVSnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=866-871&author=C.+Wuauthor=Q.+Caiauthor=C.+Chenauthor=N.+Liauthor=X.+Pengauthor=Y.+Caiauthor=K.+Yinauthor=X.+Chenauthor=X.+Wangauthor=R.+Zhangauthor=L.+Liuauthor=S.+Chenauthor=J.+Liauthor=T.+Lin&title=Structures+of+Enterovirus+71+3C+proteinase+%28strain+E2004104-TW-CDC%29+and+its+complex+with+rupintrivir&doi=10.1107%2FS0907444913002862"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6fR"><div class="casContent"><span class="casTitleNuber">6f</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of Enterovirus 71 3C proteinase (strain E2004104-TW-CDC) and its complex with rupintrivir</span></div><div class="casAuthors">Wu, Caiming; Cai, Qixu; Chen, Chen; Li, Ning; Peng, Xuanjia; Cai, Yaxian; Yin, Ke; Chen, Xinsheng; Wang, Xiaolong; Zhang, Rongfu; Liu, Lijie; Chen, Shuhui; Li, Jian; Lin, Tianwei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">866-871</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The crystal structure of 3C proteinase (3Cpro) from Enterovirus 71 (EV71) was detd. in space group C2221 to 2.2 Å resoln.  The fold was similar to that of 3Cpro from other picornaviruses, but the difference in the β-ribbon reported in a previous structure was not obsd.  This β-ribbon was folded over the substrate-binding cleft and constituted part of the essential binding sites for interaction with the substrate.  The structure of its complex with rupintrivir (AG7088), a peptidomimetic inhibitor, was also characterized in space group P212121 to 1.96 Å resoln.  The inhibitor was accommodated without any spatial hindrance despite the more constricted binding site; this was confirmed by functional assays, in which the inhibitor showed comparable potency towards EV71 3Cpro and human rhinovirus 3Cpro, which is the target that rupintrivir was designed against.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6W5v6O6LT-bVg90H21EOLACvtfcHk0libonlyEANLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvVSnt7Y%253D&md5=53be6c2cf82e5abcd9dbbe11c2ef370d</span></div><a href="/servlet/linkout?suffix=cit6f&amp;dbid=16384&amp;doi=10.1107%2FS0907444913002862&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913002862%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DT.%26atitle%3DStructures%2520of%2520Enterovirus%252071%25203C%2520proteinase%2520%2528strain%2520E2004104-TW-CDC%2529%2520and%2520its%2520complex%2520with%2520rupintrivir%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D866%26epage%3D871%26doi%3D10.1107%2FS0907444913002862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit6g"><span><span class="NLM_label">(g) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human enterovirus 71 3C protease</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>408</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2011.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.jmb.2011.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=21396941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKgsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=408&publication_year=2011&pages=449-461&author=S.+Cuiauthor=J.+Wangauthor=T.+Fanauthor=B.+Qinauthor=L.+Guoauthor=X.+Leiauthor=J.+Wangauthor=M.+Wangauthor=Q.+Jin&title=Crystal+structure+of+human+enterovirus+71+3C+protease&doi=10.1016%2Fj.jmb.2011.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6gR"><div class="casContent"><span class="casTitleNuber">6g</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Human Enterovirus 71 3C Protease</span></div><div class="casAuthors">Cui, Sheng; Wang, Jing; Fan, Tingting; Qin, Bo; Guo, Li; Lei, Xiaobo; Wang, Jianwei; Wang, Meitian; Jin, Qi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">408</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">449-461</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human enterovirus 71 (EV71) is the major pathogen that causes hand, foot and mouth disease that particularly affects young children.  Growing hand, foot and mouth disease outbreaks were obsd. worldwide in recent years and caused devastating losses both economically and politically.  However, vaccines or effective drugs are unavailable to date.  The genome of EV71 consists of a pos. sense, single-stranded RNA of ∼ 7400 bp, encoding a large precursor polyprotein that requires proteolytic processing to generate mature viral proteins.  The proteolytic processing mainly depends on EV71 3C protease (3Cpro) that possesses both proteolysis and RNA binding activities, which enable the protease to perform multiple tasks in viral replication and pathogen-host interactions.  The central roles played by EV71 3Cpro make it an appealing target for antiviral drug development.  We detd. the first crystal structure of EV71 3Cpro and analyzed its enzymic activity.  The crystal structure shows that EV71 3Cpro has a typical chymotrypsin-like fold that is common in picornaviral 3Cpro.  Strikingly, we found an important surface loop, also denoted as β-ribbon, which adopts a novel open conformation in EV71 3Cpro.  We identified two important residues located at the base of the β-ribbon, Gly123 and His133, which form hinges that govern the intrinsic flexibility of the ribbon.  Structure-guided mutagenesis studies revealed that the hinge residues are important to EV71 3Cpro proteolytic activities.  In summary, our work provides the first structural insight into EV71 3Cpro, including a mobile β-ribbon, which is relevant to the proteolytic mechanism.  Our data also provides a framework for structure-guided inhibitor design against EV71 3Cpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQpc2_sfZjJrVg90H21EOLACvtfcHk0libonlyEANLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKgsr0%253D&md5=9b9c91734cbf576138a680635bc23708</span></div><a href="/servlet/linkout?suffix=cit6g&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DFan%26aufirst%3DT.%26aulast%3DQin%26aufirst%3DB.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DQ.%26atitle%3DCrystal%2520structure%2520of%2520human%2520enterovirus%252071%25203C%2520protease%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D408%26spage%3D449%26epage%3D461%26doi%3D10.1016%2Fj.jmb.2011.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potts, B. C. M.</span></span> <span> </span><span class="NLM_article-title">Liquid Chromatography-Mass spectrometry and liquid Chromatography-NMR characterization of in vitro metabolites of a potent and irreversible peptidomimetic inhibitor of rhinovirus 3C protease</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">734</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=11302940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtF2jur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2001&pages=729-734&author=K.+E.+Zhangauthor=B.+Heeauthor=C.+A.+Leeauthor=B.+Liangauthor=B.+C.+M.+Potts&title=Liquid+Chromatography-Mass+spectrometry+and+liquid+Chromatography-NMR+characterization+of+in+vitro+metabolites+of+a+potent+and+irreversible+peptidomimetic+inhibitor+of+rhinovirus+3C+protease"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7aR"><div class="casContent"><span class="casTitleNuber">7a</span><div class="casTitle"><span class="NLM_cas:atitle">Liquid chromatography-mass spectrometry and liquid chromatography-NMR characterization of in vitro metabolites of a potent and irreversible peptidomimetic inhibitor of rhinovirus 3C protease</span></div><div class="casAuthors">Zhang, Kanyin E.; Hee, Brian; Lee, Caroline A.; Liang, Baihong; Potts, Barbara C. M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">729-734</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In vitro metab. of AG7088 [trans-(4S,2'R,5'S,3"'S)-4-{2'-4-(4-fluorobenzyl)-6'-methyl-5'-[(5"-methylisoxazole-3"-carbonylamino]-4-oxoheptanoylamino}-5-(2"'-oxopyrrolidin-3-"'-yl)pent-2-enoic acid Et ester] was studied in liver microsomes isolated from mice, rats, rabbits, dogs, monkeys, and humans.  The structures of the metabolites were characterized by liq. chromatog. (LC)-tandem mass spectrometry and LC-NMR methods.  Hydrolysis of the Et ester to produce metabolite M4 (AG7185) is the predominant pathway in all species, with the greatest activity obsd. in rodents and rabbits, followed by monkeys, dogs, and humans.  Several hydroxylation products were identified as minor metabolites, including diastereomers M1 and M2, with a hydroxy group at the P1-lactam moiety, and M3, with a hydroxy group at the Me position of the methylisoxazole ring.  Rodent and rabbit liver microsomes formed almost exclusively the acid metabolite M4 (AG7185), with very little hydroxylated metabolites, whereas monkey liver microsomes formed more secondary metabolites (i.e., acid analogs of the hydroxylated metabolites).  The overall metabolic profile of AG7088 formed in dog liver microsomes closely resembled that of human liver microsomes; therefore, this species may be the most appropriate animal model relative to humans for exposure to AG7088 and its metabolites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlwGOmiSAbBLVg90H21EOLACvtfcHk0libonlyEANLmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtF2jur4%253D&md5=263c089ccac01ee70fdb383fa958903d</span></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%2BE.%26aulast%3DHee%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DLiang%26aufirst%3DB.%26aulast%3DPotts%26aufirst%3DB.%2BC.%2BM.%26atitle%3DLiquid%2520Chromatography-Mass%2520spectrometry%2520and%2520liquid%2520Chromatography-NMR%2520characterization%2520of%2520in%2520vitro%2520metabolites%2520of%2520a%2520potent%2520and%2520irreversible%2520peptidomimetic%2520inhibitor%2520of%2520rhinovirus%25203C%2520protease%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2001%26volume%3D29%26spage%3D729%26epage%3D734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marakovits, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejto, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harr, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 4. incorporation of P1 lactam moieties as L-glutamine replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1213</span>– <span class="NLM_lpage">1224</span>, <span class="refDoi"> DOI: 10.1021/jm9805384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9805384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1213-1224&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=S.+E.+Webberauthor=J.+T.+Marakovitsauthor=S.+A.+Fuhrmanauthor=A.+K.+Patickauthor=D.+A.+Matthewsauthor=C.+A.+Leeauthor=C.+E.+Fordauthor=B.+J.+Burkeauthor=P.+A.+Rejtoauthor=T.+F.+Hendricksonauthor=T.+Tuntlandauthor=E.+L.+Brownauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=J.+E.+Harrauthor=M.+B.+Kosaauthor=S.+T.+Worland&title=Structure-based+design%2C+synthesis%2C+and+biological+evaluation+of+irreversible+human+rhinovirus+3C+protease+inhibitors.+4.+incorporation+of+P1+lactam+moieties+as+L-glutamine+replacements&doi=10.1021%2Fjm9805384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Webber, Stephen E.; Marakovits, Joseph T.; Fuhrman, Shella A.; Patick, Amy K.; Matthews, David A.; Lee, Caroline A.; Ford, Clifford E.; Burke, Benjamin J.; Rejto, Paul A.; Hendrickson, Thomas F.; Tuntland, Tove; Brown, Edward L.; Meador, James W., III; Ferre, Rose Ann; Harr, James E. V.; Kosa, Maha B.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1224</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various human rhinovirus (HRV) 3C protease (3CP) inhibitors, e.g. I [R = CH2CHMe2, CHMe2; X = (S)-CH, (R)-CH, N; n = 1, 2] and II (R = CHMe2, CMe3; Y = NH, CH2) which incorporate P1 lactam moieties in lieu of an L-glutamine residue are described.  These compds. are comprised of a tripeptidyl or peptidomimetic binding determinant and an Et propenoate Michael acceptor moiety which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The P1-lactam-contg. inhibitors display significantly increased 3CP inhibition activity along with improved antirhinoviral properties relative to corresponding L-glutamine-derived mols.  In addn., several lactam-contg. compds. exhibit excellent selectivity for HRV 3CP over several other serine and cysteine proteases and are not appreciably degraded by a variety of biol. agents.  Lactam II (R = CHMe2, Y = CH2, n = 1) (AG7088) was one of the most potent inhibitors (mean antirhinoviral EC90 ≈ 0.10 μM, n = 46 serotypes), and is shown to warrant addnl. preclin. development to explore its potential for use as an antirhinoviral agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlBuxFPwMr97Vg90H21EOLACvtfcHk0lha8XR-e83OYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D&md5=0e2bf5a03d17ea371663a252e7b97b20</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1021%2Fjm9805384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9805384%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%2BT.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBurke%26aufirst%3DB.%2BJ.%26aulast%3DRejto%26aufirst%3DP.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DHarr%26aufirst%3DJ.%2BE.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520irreversible%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors.%25204.%2520incorporation%2520of%2520P1%2520lactam%2520moieties%2520as%2520L-glutamine%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1213%26epage%3D1224%26doi%3D10.1021%2Fjm9805384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit7c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pithavala, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gersten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penning, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, B. M.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetics and safety of an antirhinoviral agent, Ruprintrivir, in healthy volunteers</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">392</span>– <span class="NLM_lpage">397</span>, <span class="refDoi"> DOI: 10.1128/AAC.46.2.392-397.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.46.2.392-397.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=11796347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFyisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=392-397&author=P.+H.+Hsyuauthor=Y.+K.+Pithavalaauthor=M.+Gerstenauthor=C.+A.+Penningauthor=B.+M.+Kerr&title=Pharmacokinetics+and+safety+of+an+antirhinoviral+agent%2C+Ruprintrivir%2C+in+healthy+volunteers&doi=10.1128%2FAAC.46.2.392-397.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7cR"><div class="casContent"><span class="casTitleNuber">7c</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetics and safety of an antirhinoviral agent, ruprintrivir, in healthy volunteers</span></div><div class="casAuthors">Hsyu, Poe-Hirr; Pithavala, Yazdi K.; Gersten, Merril; Penning, Carol A.; Kerr, Bradley M.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">392-397</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A single-dose study and a multiple-dose study of the safety and pharmacokinetics of ruprintrivir, a new selective irreversible inhibitor of human rhinovirus 3C protease, were conducted with healthy adult volunteers.  Both studies were double-blind, randomized, placebo-controlled, parallel-group investigations of ruprintrivir administered intranasally at two dose levels.  The parent drug and its acid metabolite, AG7185, were measured in plasma samples and nasal washings, and the safety of the treatments was monitored.  Intranasal ruprintrivir, administered as single doses of 4 and 8 mg or every 3 h, six times per day, for 7 days was safe and well tolerated.  Adverse events were mild, short-lived, and confined to the upper respiratory tract (i.e., nose and throat, taste and smell perceptions).  Adverse events were similar after placebo and after single or multiple doses of active drug.  Systemic exposure to ruprintrivir was rarely detectable with the highest measured concn. of ≤0.52 ng/mL; the assay had a lower limit of quantification of 0.2 ng/mL.  Systemic exposure to the metabolite was also low, with a highest measured concn. of 3.25 ng/mL.  Concns. of AG7185 obsd. during multiple dosing were higher than those obsd. after the first dose but were no more than predicted from the single-dose study.  Substantial amts. of ruprintrivir were obsd. intranasally for at least 9 h after multiple doses of ruprintrivir.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGou7tDU2Prh-bVg90H21EOLACvtfcHk0lha8XR-e83OYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFyisw%253D%253D&md5=dffd842fdf9f88226661fbc96ae77b1c</span></div><a href="/servlet/linkout?suffix=cit7c&amp;dbid=16384&amp;doi=10.1128%2FAAC.46.2.392-397.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.46.2.392-397.2002%26sid%3Dliteratum%253Aachs%26aulast%3DHsyu%26aufirst%3DP.%2BH.%26aulast%3DPithavala%26aufirst%3DY.%2BK.%26aulast%3DGersten%26aufirst%3DM.%26aulast%3DPenning%26aufirst%3DC.%2BA.%26aulast%3DKerr%26aufirst%3DB.%2BM.%26atitle%3DPharmacokinetics%2520and%2520safety%2520of%2520an%2520antirhinoviral%2520agent%252C%2520Ruprintrivir%252C%2520in%2520healthy%2520volunteers%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2002%26volume%3D46%26spage%3D392%26epage%3D397%26doi%3D10.1128%2FAAC.46.2.392-397.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit7d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Shie, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jan, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y. S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic α, β-unsaturated esters</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">5240</span>– <span class="NLM_lpage">5252</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2005.05.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1016%2Fj.bmc.2005.05.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15994085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVKhtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2005&pages=5240-5252&author=J.+J.+Shieauthor=J.+M.+Fangauthor=T.+H.+Kuoauthor=C.+J.+Kuoauthor=P.+H.+Liangauthor=H.+J.+Huangauthor=Y.+T.+Wuauthor=J.+T.+Janauthor=Y.+S.+E.+Chengauthor=C.+H.+Wong&title=Inhibition+of+the+severe+acute+respiratory+syndrome+3CL+protease+by+peptidomimetic+%CE%B1%2C+%CE%B2-unsaturated+esters&doi=10.1016%2Fj.bmc.2005.05.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7dR"><div class="casContent"><span class="casTitleNuber">7d</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the severe acute respiratory syndrome 3CL protease by peptidomimetic α,β-unsaturated esters</span></div><div class="casAuthors">Shie, Jiun-Jie; Fang, Jim-Min; Kuo, Tun-Hsun; Kuo, Chih-Jung; Liang, Po-Huang; Huang, Hung-Jyun; Wu, Yin-Ta; Jan, Jia-Tsrong; Cheng, Yih-Shyun E.; Wong, Chi-Huey</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5240-5252</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The proteolytic processing of polyproteins by the 3CL protease of severe acute respiratory syndrome coronavirus is essential for the viral propagation.  A series of tripeptide α,β-unsatd. esters and ketomethylene isosteres, including AG7088, are synthesized and assayed to target the 3CL protease.  Though AG7088 is inactive (IC50 > 100 μM), the ketomethylene isosteres and tripeptide α,β-unsatd. esters contg. both P1 and P2 phenylalanine residues show modest inhibitory activity (IC50 = 11-39 μM).  The Phe-Phe dipeptide inhibitors are designed on the basis of computer modeling of the enzyme-inhibitor complex.  The most potent inhibitor (I) with an inhibition const. of 0.52 μM is obtained by condensation of the Phe-Phe dipeptide α,β-unsatd. ester with 4-(dimethylamino)cinnamic acid.  The cell-based assays also indicate that I is a nontoxic anti-SARS agent with an EC50 value of 0.18 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWHJTRZOlTybVg90H21EOLACvtfcHk0lha8XR-e83OYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVKhtb4%253D&md5=c2dea03aff1915bda613a491663ab58d</span></div><a href="/servlet/linkout?suffix=cit7d&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2005.05.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2005.05.065%26sid%3Dliteratum%253Aachs%26aulast%3DShie%26aufirst%3DJ.%2BJ.%26aulast%3DFang%26aufirst%3DJ.%2BM.%26aulast%3DKuo%26aufirst%3DT.%2BH.%26aulast%3DKuo%26aufirst%3DC.%2BJ.%26aulast%3DLiang%26aufirst%3DP.%2BH.%26aulast%3DHuang%26aufirst%3DH.%2BJ.%26aulast%3DWu%26aufirst%3DY.%2BT.%26aulast%3DJan%26aufirst%3DJ.%2BT.%26aulast%3DCheng%26aufirst%3DY.%2BS.%2BE.%26aulast%3DWong%26aufirst%3DC.%2BH.%26atitle%3DInhibition%2520of%2520the%2520severe%2520acute%2520respiratory%2520syndrome%25203CL%2520protease%2520by%2520peptidomimetic%2520%25CE%25B1%252C%2520%25CE%25B2-unsaturated%2520esters%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2005%26volume%3D13%26spage%3D5240%26epage%3D5252%26doi%3D10.1016%2Fj.bmc.2005.05.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z. L.</span></span> <span> </span><span class="NLM_article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2012-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41586-020-2012-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32015507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=270-273&author=P.+Zhouauthor=X.+L.+Yangauthor=X.+G.+Wangauthor=B.+Huauthor=L.+Zhangauthor=W.+Zhangauthor=H.+R.+Siauthor=Y.+Zhuauthor=B.+Liauthor=C.+L.+Huangauthor=H.+D.+Chenauthor=J.+Chenauthor=Y.+Luoauthor=H.+Guoauthor=R.+D.+Jiangauthor=M.+Q.+Liuauthor=Y.+Chenauthor=X.+R.+Shenauthor=X.+Wangauthor=X.+S.+Zhengauthor=K.+Zhaoauthor=Q.+J.+Chenauthor=F.+Dengauthor=L.+L.+Liuauthor=B.+Yanauthor=F.+X.+Zhanauthor=Y.+Y.+Wangauthor=G.+F.+Xiaoauthor=Z.+L.+Shi&title=A+pneumonia+outbreak+associated+with+a+new+coronavirus+of+probable+bat+origin&doi=10.1038%2Fs41586-020-2012-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8aR"><div class="casContent"><span class="casTitleNuber">8a</span><div class="casTitle"><span class="NLM_cas:atitle">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span></div><div class="casAuthors">Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">270-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Since the outbreak of severe acute respiratory syndrome (SARS) 18 years ago, a large no. of SARS-related coronaviruses (SARSr-CoVs) have been discovered in their natural reservoir host, bats1-4.  Previous studies have shown that some bat SARSr-CoVs have the potential to infect humans5-7.  Here we report the identification and characterization of a new coronavirus (2019-nCoV), which caused an epidemic of acute respiratory syndrome in humans in Wuhan, China.  The epidemic, which started on 12 Dec. 2019, had caused 2,794 lab.-confirmed infections including 80 deaths by 26 Jan. 2020.  Full-length genome sequences were obtained from five patients at an early stage of the outbreak.  The sequences are almost identical and share 79.6% sequence identity to SARS-CoV.  Furthermore, we show that 2019-nCoV is 96% identical at the whole-genome level to a bat coronavirus.  Pairwise protein sequence anal. of seven conserved non-structural proteins domains show that this virus belongs to the species of SARSr-CoV.  In addn., 2019-nCoV virus isolated from the bronchoalveolar lavage fluid of a critically ill patient could be neutralized by sera from several patients.  Notably, we confirmed that 2019-nCoV uses the same cell entry receptor-angiotensin converting enzyme II (ACE2)-as SARS-CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_Wb_iUsbFrVg90H21EOLACvtfcHk0lgdZ2B9n5gmVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D&md5=236f17d4d3c7978d72513e5e0258f1b3</span></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2012-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2012-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DX.%2BG.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSi%26aufirst%3DH.%2BR.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.%2BL.%26aulast%3DChen%26aufirst%3DH.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DR.%2BD.%26aulast%3DLiu%26aufirst%3DM.%2BQ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DX.%2BR.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DQ.%2BJ.%26aulast%3DDeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DL.%2BL.%26aulast%3DYan%26aufirst%3DB.%26aulast%3DZhan%26aufirst%3DF.%2BX.%26aulast%3DWang%26aufirst%3DY.%2BY.%26aulast%3DXiao%26aufirst%3DG.%2BF.%26aulast%3DShi%26aufirst%3DZ.%2BL.%26atitle%3DA%2520pneumonia%2520outbreak%2520associated%2520with%2520a%2520new%2520coronavirus%2520of%2520probable%2520bat%2520origin%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D270%26epage%3D273%26doi%3D10.1038%2Fs41586-020-2012-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">A new coronavirus associated with human respiratory disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.+M.+Chenauthor=W.+Wangauthor=Z.+G.+Songauthor=Y.+Huauthor=Z.+W.+Taoauthor=J.+H.+Tianauthor=Y.+Y.+Peiauthor=M.+L.+Yuanauthor=Y.+L.+Zhangauthor=F.+H.+Daiauthor=Y.+Liuauthor=Q.+M.+Wangauthor=J.+J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.+Z.+Zhang&title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8bR"><div class="casContent"><span class="casTitleNuber">8b</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0lgdZ2B9n5gmVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.%2BM.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.%2BG.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.%2BW.%26aulast%3DTian%26aufirst%3DJ.%2BH.%26aulast%3DPei%26aufirst%3DY.%2BY.%26aulast%3DYuan%26aufirst%3DM.%2BL.%26aulast%3DZhang%26aufirst%3DY.%2BL.%26aulast%3DDai%26aufirst%3DF.%2BH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%2BM.%26aulast%3DZheng%26aufirst%3DJ.%2BJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.%2BZ.%26atitle%3DA%2520new%2520coronavirus%2520associated%2520with%2520human%2520respiratory%2520disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Chinazzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gioannini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litvinova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastore y Piontti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viboud, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halloran, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longini, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vespignani, A.</span></span> <span> </span><span class="NLM_article-title">The effect of travel restrictions on the spread of the 2019 novel coronavirus (COVID-19) outbreak</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1126/science.aba9757</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.aba9757" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32144116" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=395-400&author=M.+Chinazziauthor=J.+T.+Davisauthor=M.+Ajelliauthor=C.+Gioanniniauthor=M.+Litvinovaauthor=S.+Merlerauthor=A.+Pastore+y+Pionttiauthor=K.+Muauthor=L.+Rossiauthor=K.+Sunauthor=C.+Viboudauthor=X.+Xiongauthor=H.+Yuauthor=M.+E.+Halloranauthor=I.+M.+Longiniauthor=A.+Vespignani&title=The+effect+of+travel+restrictions+on+the+spread+of+the+2019+novel+coronavirus+%28COVID-19%29+outbreak&doi=10.1126%2Fscience.aba9757"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8cR"><div class="casContent"><span class="casTitleNuber">8c</span><div class="casTitle"><span class="NLM_cas:atitle">The effect of travel restrictions on the spread of the 2019 novel coronavirus (COVID-19) outbreak</span></div><div class="casAuthors">Chinazzi, Matteo; Davis, Jessica T.; Ajelli, Marco; Gioannini, Corrado; Litvinova, Maria; Merler, Stefano; Pastore y Piontti, Ana; Mu, Kunpeng; Rossi, Luca; Sun, Kaiyuan; Viboud, Cecile; Xiong, Xinyue; Yu, Hongjie; Halloran, M. Elizabeth; Longini, Ira M., Jr.; Vespignani, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">395-400</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Motivated by the rapid spread of coronavirus disease 2019 (COVID-19) in mainland China, we use a global metapopulation disease transmission model to project the impact of travel limitations on the national and international spread of the epidemic.  The model is calibrated on the basis of internationally reported cases and shows that, at the start of the travel ban from Wuhan on 23 Jan. 2020, most Chinese cities had already received many infected travelers.  The travel quarantine of Wuhan delayed the overall epidemic progression by only 3 to 5 days in mainland China but had a more marked effect on the international scale, where case importations were reduced by nearly 80% until mid-Feb.  Modeling results also indicate that sustained 90% travel restrictions to and from mainland China only modestly affect the epidemic trajectory unless combined with a 50% or higher redn. of transmission in the community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhnsJ8Ikq42rVg90H21EOLACvtfcHk0ljr3Abgi4KL0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtLw%253D&md5=3d9c6e65c976c694e84837bb6f06f6ae</span></div><a href="/servlet/linkout?suffix=cit8c&amp;dbid=16384&amp;doi=10.1126%2Fscience.aba9757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aba9757%26sid%3Dliteratum%253Aachs%26aulast%3DChinazzi%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DJ.%2BT.%26aulast%3DAjelli%26aufirst%3DM.%26aulast%3DGioannini%26aufirst%3DC.%26aulast%3DLitvinova%26aufirst%3DM.%26aulast%3DMerler%26aufirst%3DS.%26aulast%3DPastore%2By%2BPiontti%26aufirst%3DA.%26aulast%3DMu%26aufirst%3DK.%26aulast%3DRossi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DK.%26aulast%3DViboud%26aufirst%3DC.%26aulast%3DXiong%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DHalloran%26aufirst%3DM.%2BE.%26aulast%3DLongini%26aufirst%3DI.%2BM.%26aulast%3DVespignani%26aufirst%3DA.%26atitle%3DThe%2520effect%2520of%2520travel%2520restrictions%2520on%2520the%2520spread%2520of%2520the%25202019%2520novel%2520coronavirus%2520%2528COVID-19%2529%2520outbreak%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D395%26epage%3D400%26doi%3D10.1126%2Fscience.aba9757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ledford, H.</span></span> <span> </span><span class="NLM_article-title">Coronavirus breakthrough: dexamethasone is first drug shown to save lives</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">469</span>, <span class="refDoi"> DOI: 10.1038/d41586-020-01824-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fd41586-020-01824-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32546811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1ejtrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=469&author=H.+Ledford&title=Coronavirus+breakthrough%3A+dexamethasone+is+first+drug+shown+to+save+lives&doi=10.1038%2Fd41586-020-01824-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8dR"><div class="casContent"><span class="casTitleNuber">8d</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus breakthrough: dexamethasone is first drug shown to save lives</span></div><div class="casAuthors">Ledford, Heidi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7813</span>),
    <span class="NLM_cas:pages">469</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In a large trial, a cheap and widely available steroid cut deaths by one-third among patients critically ill with COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBJB_W2-H5D7Vg90H21EOLACvtfcHk0ljr3Abgi4KL0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1ejtrbE&md5=4437eb1c5d52bde519fb55f07625f404</span></div><a href="/servlet/linkout?suffix=cit8d&amp;dbid=16384&amp;doi=10.1038%2Fd41586-020-01824-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fd41586-020-01824-5%26sid%3Dliteratum%253Aachs%26aulast%3DLedford%26aufirst%3DH.%26atitle%3DCoronavirus%2520breakthrough%253A%2520dexamethasone%2520is%2520first%2520drug%2520shown%2520to%2520save%2520lives%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D469%26doi%3D10.1038%2Fd41586-020-01824-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit8e"><span><span class="NLM_label">(e) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Kupferschmidt, K.</span></span> <span> </span><span class="NLM_article-title">Global trial eliminates drugs, pivots to new ones</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>370</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1126/science.370.6515.388</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.370.6515.388" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=33093086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BB3s7ltFSjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=370&publication_year=2020&pages=388-389&author=K.+Kupferschmidt&title=Global+trial+eliminates+drugs%2C+pivots+to+new+ones&doi=10.1126%2Fscience.370.6515.388"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8eR"><div class="casContent"><span class="casTitleNuber">8e</span><div class="casTitle"><span class="NLM_cas:atitle">Global trial eliminates drugs, pivots to new ones</span></div><div class="casAuthors">Kupferschmidt Kai</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">370</span>
        (<span class="NLM_cas:issue">6515</span>),
    <span class="NLM_cas:pages">388-389</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRf1iuBtpFC0OkdrW1FkXG-fW6udTcc2ebUcby90hpKILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3s7ltFSjtg%253D%253D&md5=64a9d2d5006a1b3a09cb83b50e7341e9</span></div><a href="/servlet/linkout?suffix=cit8e&amp;dbid=16384&amp;doi=10.1126%2Fscience.370.6515.388&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.370.6515.388%26sid%3Dliteratum%253Aachs%26aulast%3DKupferschmidt%26aufirst%3DK.%26atitle%3DGlobal%2520trial%2520eliminates%2520drugs%252C%2520pivots%2520to%2520new%2520ones%26jtitle%3DScience%26date%3D2020%26volume%3D370%26spage%3D388%26epage%3D389%26doi%3D10.1126%2Fscience.370.6515.388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahabi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Drug development and medicinal chemistry efforts toward SARS-coronavirus and Covid-19 therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fcmdc.202000223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32324951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=907-932&author=A.+K.+Ghoshauthor=M.+Brindisiauthor=D.+Shahabiauthor=M.+E.+Chapmanauthor=A.+D.+Mesecar&title=Drug+development+and+medicinal+chemistry+efforts+toward+SARS-coronavirus+and+Covid-19+therapeutics&doi=10.1002%2Fcmdc.202000223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9aR"><div class="casContent"><span class="casTitleNuber">9a</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span></div><div class="casAuthors">Ghosh, Arun K.; Brindisi, Margherita; Shahabi, Dana; Chapman, Mackenzie E.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">907-932</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 infection is spreading at an alarming rate and has created an unprecedented health emergency around the globe.  There is no effective vaccine or approved drug treatment against COVID-19 and other pathogenic coronaviruses.  The development of antiviral agents is an urgent priority.  Biochem. events crit. to the coronavirus replication cycle provided a no. of attractive targets for drug development.  These include, spike protein for binding to host cell-surface receptors, proteolytic enzymes that are essential for processing polyproteins into mature viruses, and RNA-dependent RNA polymerase for RNA replication.  There has been a lot of ground work for drug discovery and development against these targets.  Also, high-throughput screening efforts have led to the identification of diverse lead structures, including natural product-derived mols.  This review highlights past and present drug discovery and medicinal-chem. approaches against SARS-CoV, MERS-CoV and COVID-19 targets.  The review hopes to stimulate further research and will be a useful guide to the development of effective therapies against COVID-19 and other pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcgVQF9N3zf7Vg90H21EOLACvtfcHk0lhT_pIFtdx88Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D&md5=8b4c3c83c05a7b64f39ba7d803ea1709</span></div><a href="/servlet/linkout?suffix=cit9a&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000223%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DShahabi%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DM.%2BE.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DDrug%2520development%2520and%2520medicinal%2520chemistry%2520efforts%2520toward%2520SARS-coronavirus%2520and%2520Covid-19%2520therapeutics%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D907%26epage%3D932%26doi%3D10.1002%2Fcmdc.202000223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved alpha-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+alpha-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9bR"><div class="casContent"><span class="casTitleNuber">9b</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0lhT_pIFtdx88Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit9b&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520alpha-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of M(pro) from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+M%28pro%29+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9cR"><div class="casContent"><span class="casTitleNuber">9c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0lhT_pIFtdx88Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit9c&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520M%2528pro%2529%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit9d"><span><span class="NLM_label">(d) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.+M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.+K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9dR"><div class="casContent"><span class="casTitleNuber">9d</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0lipM68HJhMd3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit9d&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.%2BM.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%2BK.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.</span></span> <span> </span><span class="NLM_article-title">Enterovirus A71 proteins: structure and function</span>. <i>Front. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">286</span>, <span class="refDoi"> DOI: 10.3389/fmicb.2018.00286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.3389%2Ffmicb.2018.00286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=29515559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A280%3ADC%252BC1MroslOhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=286&author=J.+Yuanauthor=L.+Shenauthor=J.+Wuauthor=X.+Zouauthor=J.+Guauthor=J.+Chenauthor=L.+Mao&title=Enterovirus+A71+proteins%3A+structure+and+function&doi=10.3389%2Ffmicb.2018.00286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10aR"><div class="casContent"><span class="casTitleNuber">10a</span><div class="casTitle"><span class="NLM_cas:atitle">Enterovirus A71 Proteins: Structure and Function</span></div><div class="casAuthors">Yuan Jingjing; Chen Jianguo; Mao Lingxiang; Yuan Jingjing; Shen Li; Wu Jing; Zou Xinran; Gu Jiaqi</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in microbiology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">286</span>
        ISSN:<span class="NLM_cas:issn">1664-302X</span>.
    </div><div class="casAbstract">Enterovirus A71 (EV-A71) infection has grown to become a serious threat to global public health.  It is one of the major causes of hand, foot, and mouth disease (HFMD) in infants and young children.  EV-A71 can also infect the central nervous system (CNS) and induce diverse neurological complications, such as brainstem encephalitis, aseptic meningitis, and acute flaccid paralysis, or even death.  Viral proteins play a crucial role in EV-A71 infection.  Many recent studies have discussed the structure and function of EV-A71 proteins, and the findings reported will definitely aid the development of vaccines and therapeutic approaches.  This article reviews the progress in the research on the structure and function of EV-A71 proteins.  Available literature can provide a basis for studying the pathogenesis of EV-A71 infection in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ6VkEBjsea1QMthz5_Rx5rfW6udTcc2ea5aSjAnXDbh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MroslOhuw%253D%253D&md5=642c8ff42f4db800c1b0bd327e939b56</span></div><a href="/servlet/linkout?suffix=cit10a&amp;dbid=16384&amp;doi=10.3389%2Ffmicb.2018.00286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmicb.2018.00286%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DL.%26atitle%3DEnterovirus%2520A71%2520proteins%253A%2520structure%2520and%2520function%26jtitle%3DFront.%2520Microbiol.%26date%3D2018%26volume%3D9%26spage%3D286%26doi%3D10.3389%2Ffmicb.2018.00286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit10b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauhan, R. S.</span></span> <span> </span><span class="NLM_article-title">A concise review of baseline facts of SARS-CoV-2 for interdisciplinary research</span>. <i>ChemistrySelect.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10923</span>, <span class="refDoi"> DOI: 10.1002/slct.202002420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fslct.202002420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvV2nurzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=10897-10923&author=A.+Tyagiauthor=S.+Nigamauthor=R.+S.+Chauhan&title=A+concise+review+of+baseline+facts+of+SARS-CoV-2+for+interdisciplinary+research&doi=10.1002%2Fslct.202002420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10bR"><div class="casContent"><span class="casTitleNuber">10b</span><div class="casTitle"><span class="NLM_cas:atitle">A Concise Review of Baseline Facts of SARS-CoV-2 for Interdisciplinary Research</span></div><div class="casAuthors">Tyagi, Adish; Nigam, Sandeep; Chauhan, Rohit Singh</div><div class="citationInfo"><span class="NLM_cas:title">ChemistrySelect</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10897-10923</span>CODEN:
                <span class="NLM_cas:coden">CHEMUD</span>;
        ISSN:<span class="NLM_cas:issn">2365-6549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 is a global health emergency.  Investigation covering various aspects of diagnosis, treatment, and vaccine development for COVID-19 are being attempted globally.  Common goal of reducing loss of life requires accelerated and integrated effort from researchers of different fields i. e. chem., biol., clin./medical science etc.  Integrated yet timely effort requires the understanding of interdisciplinary key concepts as well as knowledge of major challenges, present status and strategies.  Hence, in this article, we present an overview of baseline/key facts of SARS-CoV-2 mingled with current status and strategies in a simple, straight forward and coherent manner.  In particular, the review attempts to integrate and arrange the key facts at the interface of chem. and biol.  Overview of epidemiol. of SARS-CoV-2 virus infection, structural characteristics of virus, detection methods and therapeutic agents based on antibodies, therapeutic agents based on traditional drug mol., drug development based on specific catalytic enzyme inhibitors, vaccine design approaches have been provided.  Since enzyme site and catalytic action is the key for biol. process, they have been lucidly explained at mol. level.  It is believed that accredited information amalgamated in this report will provide a strong rational and lucid groundwork for the coherent interdisciplinary research to countermeasure covid-19 crisis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiCFiihiV-QrVg90H21EOLACvtfcHk0lipM68HJhMd3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvV2nurzF&md5=1faf94b647286fd085fb0b13b648c217</span></div><a href="/servlet/linkout?suffix=cit10b&amp;dbid=16384&amp;doi=10.1002%2Fslct.202002420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fslct.202002420%26sid%3Dliteratum%253Aachs%26aulast%3DTyagi%26aufirst%3DA.%26aulast%3DNigam%26aufirst%3DS.%26aulast%3DChauhan%26aufirst%3DR.%2BS.%26atitle%3DA%2520concise%2520review%2520of%2520baseline%2520facts%2520of%2520SARS-CoV-2%2520for%2520interdisciplinary%2520research%26jtitle%3DChemistrySelect.%26date%3D2020%26volume%3D5%26spage%3D10897%26epage%3D10923%26doi%3D10.1002%2Fslct.202002420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit10c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palm, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesters, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddell, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3213</span>– <span class="NLM_lpage">3224</span>, <span class="refDoi"> DOI: 10.1093/emboj/cdf327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1093%2Femboj%2Fcdf327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=12093723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsFCgt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=3213-3224&author=K.+Anandauthor=G.+J.+Palmauthor=J.+R.+Mestersauthor=S.+G.+Siddellauthor=J.+Ziebuhrauthor=R.+Hilgenfeld&title=Structure+of+coronavirus+main+proteinase+reveals+combination+of+a+chymotrypsin+fold+with+an+extra+%CE%B1-helical+domain&doi=10.1093%2Femboj%2Fcdf327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10cR"><div class="casContent"><span class="casTitleNuber">10c</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of coronavirus main proteinase reveals combination of a chymotrypsin fold with an extra α-helical domain</span></div><div class="casAuthors">Anand, Kanchan; Palm, Gottfried J.; Mesters, Jeroen R.; Siddell, Stuart G.; Ziebuhr, John; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3213-3224</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The key enzyme in coronavirus polyprotein processing is the viral main proteinase, Mpro, a protein with extremely low sequence similarity to other viral and cellular proteinases.  Here, the crystal structure of the 33.1 kDa transmissible gastroenteritis (corona)virus Mpro is reported.  The structure was refined to 1.96 Å resoln. and revealed three dimers in the asym. unit.  The mutual arrangement of the protomers in each of the dimers suggests that Mpro self-processing occurs in trans.  The active site, comprised of Cys144 and His41, is part of a chymotrypsin-like fold that is connected by a 16 residue loop to an extra domain featuring a novel α-helical fold.  Mol. modeling and mutagenesis data implicate the loop in substrate binding and elucidate S1 and S2 subsites suitable to accommodate the side chains of the P1 glutamine and P2 leucine residues of Mpro substrates.  Interactions involving the N-terminus and the α-helical domain stabilize the loop in the orientation required for trans-cleavage activity.  The study illustrates that RNA viruses have evolved unprecedented variations of the classical chymotrypsin fold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-3mPwalwvErVg90H21EOLACvtfcHk0lhoD7LVP6NFNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsFCgt78%253D&md5=5bb1ad2d3b1416639968f26ea46ebf03</span></div><a href="/servlet/linkout?suffix=cit10c&amp;dbid=16384&amp;doi=10.1093%2Femboj%2Fcdf327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252Fcdf327%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DK.%26aulast%3DPalm%26aufirst%3DG.%2BJ.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DSiddell%26aufirst%3DS.%2BG.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DStructure%2520of%2520coronavirus%2520main%2520proteinase%2520reveals%2520combination%2520of%2520a%2520chymotrypsin%2520fold%2520with%2520an%2520extra%2520%25CE%25B1-helical%2520domain%26jtitle%3DEMBO%2520J.%26date%3D2002%26volume%3D21%26spage%3D3213%26epage%3D3224%26doi%3D10.1093%2Femboj%2Fcdf327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alugubelli, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vatansever, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drelich, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blankenship, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratch, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayatshahi, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fierke, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. R.</span></span> <span> </span><span class="NLM_article-title">A quick route to multiple highly potent SARS-CoV-2 main protease inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1002%2Fcmdc.202000924" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2021&pages=1-8&author=K.+S.+Yangauthor=X.+R.+Maauthor=Y.+Maauthor=Y.+R.+Alugubelliauthor=D.+A.+Scottauthor=E.+C.+Vatanseverauthor=A.+K.+Drelichauthor=B.+Sankaranauthor=Z.+Z.+Gengauthor=L.+R.+Blankenshipauthor=H.+E.+Wardauthor=Y.+J.+Shengauthor=J.+C.+Hsuauthor=K.+C.+Kratchauthor=B.+Zhaoauthor=H.+S.+Hayatshahiauthor=J.+Liuauthor=P.+Liauthor=C.+A.+Fierkeauthor=C.+K.+Tsengauthor=S.+Xuauthor=W.+R.+Liu&title=A+quick+route+to+multiple+highly+potent+SARS-CoV-2+main+protease+inhibitors&doi=10.1002%2Fcmdc.202000924"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11a&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000924%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DK.%2BS.%26aulast%3DMa%26aufirst%3DX.%2BR.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DAlugubelli%26aufirst%3DY.%2BR.%26aulast%3DScott%26aufirst%3DD.%2BA.%26aulast%3DVatansever%26aufirst%3DE.%2BC.%26aulast%3DDrelich%26aufirst%3DA.%2BK.%26aulast%3DSankaran%26aufirst%3DB.%26aulast%3DGeng%26aufirst%3DZ.%2BZ.%26aulast%3DBlankenship%26aufirst%3DL.%2BR.%26aulast%3DWard%26aufirst%3DH.%2BE.%26aulast%3DSheng%26aufirst%3DY.%2BJ.%26aulast%3DHsu%26aufirst%3DJ.%2BC.%26aulast%3DKratch%26aufirst%3DK.%2BC.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DHayatshahi%26aufirst%3DH.%2BS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DFierke%26aufirst%3DC.%2BA.%26aulast%3DTseng%26aufirst%3DC.%2BK.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DW.%2BR.%26atitle%3DA%2520quick%2520route%2520to%2520multiple%2520highly%2520potent%2520SARS-CoV-2%2520main%2520protease%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2021%26volume%3D15%26spage%3D1%26epage%3D8%26doi%3D10.1002%2Fcmdc.202000924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit11b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=J.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+Ketone-Based+Covalent+Inhibitors+of+Coronavirus+3CL+Proteases+for+the+Potential+Therapeutic+Treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11bR"><div class="casContent"><span class="casTitleNuber">11b</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lhoD7LVP6NFNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit11b&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520Ketone-Based%2520Covalent%2520Inhibitors%2520of%2520Coronavirus%25203CL%2520Proteases%2520for%2520the%2520Potential%2520Therapeutic%2520Treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit11c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4282</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18096-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41467-020-18096-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32855413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4282&author=W.+Vuongauthor=M.+B.+Khanauthor=C.+Fischerauthor=E.+Arutyunovaauthor=T.+Lamerauthor=J.+Shieldsauthor=H.+A.+Saffranauthor=R.+T.+McKayauthor=M.+J.+van+Belkumauthor=M.+A.+Joyceauthor=H.+S.+Youngauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=M.+J.+Lemieux&title=Feline+coronavirus+drug+inhibits+the+main+protease+of+SARS-CoV-2+and+blocks+virus+replication&doi=10.1038%2Fs41467-020-18096-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11cR"><div class="casContent"><span class="casTitleNuber">11c</span><div class="casTitle"><span class="NLM_cas:atitle">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Vuong, Wayne; Khan, Muhammad Bashir; Fischer, Conrad; Arutyunova, Elena; Lamer, Tess; Shields, Justin; Saffran, Holly A.; McKay, Ryan T.; van Belkum, Marco J.; Joyce, Michael A.; Young, Howard S.; Tyrrell, D. Lorne; Vederas, John C.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4282</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The main protease, Mpro (or 3CLpro) in SARS-CoV-2 is a viable drug target because of its essential role in the cleavage of the virus polypeptide.  Feline infectious peritonitis, a fatal coronavirus infection in cats, was successfully treated previously with a prodrug GC376, a dipeptide-based protease inhibitor.  We show the prodrug and its parent GC373, are effective inhibitors of the Mpro from both SARS-CoV and SARS-CoV-2 with IC50 values in the nanomolar range.  Crystal structures of SARS-CoV-2 Mpro with these inhibitors have a covalent modification of the nucleophilic Cys145.  NMR anal. reveals that inhibition proceeds via reversible formation of a hemithioacetal.  GC373 and GC376 are potent inhibitors of SARS-CoV-2 replication in cell culture.  They are strong drug candidates for the treatment of human coronavirus infections because they have already been successful in animals.  The work here lays the framework for their use in human trials for the treatment of COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMA7W4XtRQrbVg90H21EOLACvtfcHk0lhkL2eeW9TiQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN&md5=c06d2dc9a65ae289505fe120d41d8885</span></div><a href="/servlet/linkout?suffix=cit11c&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18096-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18096-2%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DShields%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DFeline%2520coronavirus%2520drug%2520inhibits%2520the%2520main%2520protease%2520of%2520SARS-CoV-2%2520and%2520blocks%2520virus%2520replication%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4282%26doi%3D10.1038%2Fs41467-020-18096-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzarano, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span> <span> </span><span class="NLM_article-title">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. I. Investigation of potential experimental and mechanistic explanations</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1080/0049825042000205540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1080%2F0049825042000205540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15268980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2004&pages=353-366&author=K.+W.+Wardauthor=L.+M.+Azzaranoauthor=C.+A.+Evansauthor=B.+R.+Smith&title=Apparent+absolute+oral+bioavailability+in+excess+of+100%25+for+a+vitronectin+receptor+antagonist+%28SB-265123%29+in+rat.+I.+Investigation+of+potential+experimental+and+mechanistic+explanations&doi=10.1080%2F0049825042000205540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. I. Investigation of potential experimental and mechanistic explanations</span></div><div class="casAuthors">Ward, K. W.; Azzarano, L. M.; Evans, C. A.; Smith, B. R.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">353-366</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">SB-265123 is a novel αvβ3 (the vitronectin receptor) antagonist.  Previous rat studies with it revealed an apparent abs. oral bioavailability (Fapp) of > 100%.  The present studies were conducted to investigate the potential causes for this observation.  Of 49 SB-265123 analogs evaluated in rat using an identical exptl. design, Fapp>100% was obsd. for 22 of them, suggesting that the obsd. Fapp>100% with SB-265123 was not anomalous.  All 22 compds. had clearances<15 mL min-1 kg-1.  However, Fapp>100% were not recorded for all low-clearance analogs.  Using SB-265123 as a model to investigate potential artifacts, it was demonstrated that using a chiral assay did not decrease Fapp.  Addnl., qual. sample anal. demonstrated that no metabolites were present in the blood plasma that could interfere with the liq. chromatog. coupled with tandem mass spectrometry detection assay.  The high Fapp was also dose-order-, delivery system- and vehicle-independent, and was not affected by the feeding status of the animals.  Furthermore, a linearity expt. and an absorption study indicated that oral administration of SB-265123 does not result in hepatic portal vein concns. that exceed the pharmacokinetic linearity of SB-265123.  These observations suggest that the obsd. Fapp>100% for SB-265123 is not due to an exptl. artifact or an obvious pharmacokinetic non-linearity.  The mechanism(s) for this phenomenon is explored further in the 2nd part of the present paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp288gThfUWArVg90H21EOLACvtfcHk0lhkL2eeW9TiQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurk%253D&md5=50c8721ab52880039deaaa2ee53b9644</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1080%2F0049825042000205540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F0049825042000205540%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DAzzarano%26aufirst%3DL.%2BM.%26aulast%3DEvans%26aufirst%3DC.%2BA.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DApparent%2520absolute%2520oral%2520bioavailability%2520in%2520excess%2520of%2520100%2525%2520for%2520a%2520vitronectin%2520receptor%2520antagonist%2520%2528SB-265123%2529%2520in%2520rat.%2520I.%2520Investigation%2520of%2520potential%2520experimental%2520and%2520mechanistic%2520explanations%26jtitle%3DXenobiotica%26date%3D2004%26volume%3D34%26spage%3D353%26epage%3D366%26doi%3D10.1080%2F0049825042000205540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kehler, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzarano, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. R.</span></span> <span> </span><span class="NLM_article-title">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. II. Studies implicating transporter-mediated intestinal secretion</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">377</span>, <span class="refDoi"> DOI: 10.1080/0049825042000205540a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1080%2F0049825042000205540a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15268981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2004&pages=367-377&author=K.+W.+Wardauthor=L.+B.+Hardyauthor=J.+R.+Kehlerauthor=L.+M.+Azzaranoauthor=B.+R.+Smith&title=Apparent+absolute+oral+bioavailability+in+excess+of+100%25+for+a+vitronectin+receptor+antagonist+%28SB-265123%29+in+rat.+II.+Studies+implicating+transporter-mediated+intestinal+secretion&doi=10.1080%2F0049825042000205540a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Apparent absolute oral bioavailability in excess of 100% for a vitronectin receptor antagonist (SB-265123) in rat. II. Studies implicating transporter-mediated intestinal secretion</span></div><div class="casAuthors">Ward, K. W.; Hardy, L. B.; Kehler, J. R.; Azzarano, L. M.; Smith, B. R.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">367-377</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Transporters were increasingly identified as a factor in limiting the oral bioavailability of certain drugs.  Previously, the present authors investigated a compd. (SB-265123) with an apparent abs. oral bioavailability (Fapp) consistently >100%, and excluded likely artifactual causes for this observation, as well as std. considerations of non-stationary or non-linear pharmacokinetics.  The data led the authors to believe that SB-265123 might be a transporter substrate in the rat, and it was hypothesized that transporter interactions might be responsible for the obsd. Fapp>100%.  In the present study, a model was proposed incorporating rapid and complete absorption and elimination by a saturable intestinal secretory pathway.  Intestinal secretion was demonstrated for SB-265123 using a rat single-pass intestinal perfusion technique.  In addn., in a study employing both independent and simultaneous i.v. and oral administration of SB-265123, exposure to SB-265123 was greater than additive on joint i.v. and oral administration, lending further support to the hypothesis of a saturable transporter.  Furthermore, in a study with co-administration of GF120918A, a transporter inhibitor, the obsd. Fapp for SB-265123 was only 84%, providing addnl. evidence for transporter involvement in the >100% Fapp phenomenon.  Experience with SB-265123 illustrates a counterintuitive impact of transporters on oral bioavailability and highlights the importance of considering transporter interactions in the systemic disposition of xenobiotics, even those not demonstrating low oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQlI6oO-2rRLVg90H21EOLACvtfcHk0lhkL2eeW9TiQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsF2jurY%253D&md5=33d6c532955bbca6540152adbf512f97</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1080%2F0049825042000205540a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F0049825042000205540a%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DHardy%26aufirst%3DL.%2BB.%26aulast%3DKehler%26aufirst%3DJ.%2BR.%26aulast%3DAzzarano%26aufirst%3DL.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26atitle%3DApparent%2520absolute%2520oral%2520bioavailability%2520in%2520excess%2520of%2520100%2525%2520for%2520a%2520vitronectin%2520receptor%2520antagonist%2520%2528SB-265123%2529%2520in%2520rat.%2520II.%2520Studies%2520implicating%2520transporter-mediated%2520intestinal%2520secretion%26jtitle%3DXenobiotica%26date%3D2004%26volume%3D34%26spage%3D367%26epage%3D377%26doi%3D10.1080%2F0049825042000205540a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J. H.</span></span> <span> </span><span class="NLM_article-title">Upgrade of macromolecular crystallography beamline BL17U1 at SSRF</span>. <i>Nucl. Sci. Tech.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">687</span>, <span class="refDoi"> DOI: 10.1007/s41365-018-0398-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1007%2Fs41365-018-0398-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=681-687&author=Q.+S.+Wangauthor=K.+H.+Zhangauthor=Y.+Cuiauthor=Z.+J.+Wangauthor=Q.+Y.+Panauthor=K.+Liuauthor=B.+Sunauthor=H.+Zhouauthor=M.+J.+Liauthor=Q.+Xuauthor=C.+Y.+Xuauthor=F.+Yuauthor=J.+H.+He&title=Upgrade+of+macromolecular+crystallography+beamline+BL17U1+at+SSRF&doi=10.1007%2Fs41365-018-0398-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs41365-018-0398-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs41365-018-0398-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%2BS.%26aulast%3DZhang%26aufirst%3DK.%2BH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DZ.%2BJ.%26aulast%3DPan%26aufirst%3DQ.%2BY.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DM.%2BJ.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DC.%2BY.%26aulast%3DYu%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DJ.%2BH.%26atitle%3DUpgrade%2520of%2520macromolecular%2520crystallography%2520beamline%2520BL17U1%2520at%2520SSRF%26jtitle%3DNucl.%2520Sci.%2520Tech.%26date%3D2018%26volume%3D29%26spage%3D681%26epage%3D687%26doi%3D10.1007%2Fs41365-018-0398-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cymborowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span> <span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution-from diffraction images to an initial model in minutes</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">866</span>, <span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution-from+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lis8KaZRbklmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution-from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lis8KaZRbklmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lis8KaZRbklmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.+W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lggWAdtoMWA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ke, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. C.</span></span> <span> </span><span class="NLM_article-title">Anti-SARS-CoV-2 activities in vitro of Shuanghuanglian preparations and bioactive ingredients</span>. <i>Acta Pharmacol. Sin.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1167</span>– <span class="NLM_lpage">1177</span>, <span class="refDoi"> DOI: 10.1038/s41401-020-0483-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41401-020-0483-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32737471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFShu7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=1167-1177&author=H.+X.+Suauthor=S.+Yaoauthor=W.+F.+Zhaoauthor=M.+J.+Liauthor=J.+Liuauthor=W.+J.+Shangauthor=H.+Xieauthor=C.+Q.+Keauthor=H.+C.+Huauthor=M.+N.+Gaoauthor=K.+Q.+Yuauthor=H.+Liuauthor=J.+S.+Shenauthor=W.+Tangauthor=L.+K.+Zhangauthor=G.+F.+Xiaoauthor=L.+Niauthor=D.+W.+Wangauthor=J.+P.+Zuoauthor=H.+L.+Jiangauthor=F.+Baiauthor=Y.+Wuauthor=Y.+Yeauthor=Y.+C.+Xu&title=Anti-SARS-CoV-2+activities+in+vitro+of+Shuanghuanglian+preparations+and+bioactive+ingredients&doi=10.1038%2Fs41401-020-0483-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-SARS-CoV-2 activities in vitro of Shuanghuanglian preparations and bioactive ingredients</span></div><div class="casAuthors">Su, Hai-xia; Yao, Sheng; Zhao, Wen-feng; Li, Min-jun; Liu, Jia; Shang, Wei-juan; Xie, Hang; Ke, Chang-qiang; Hu, Hang-chen; Gao, Mei-na; Yu, Kun-qian; Liu, Hong; Shen, Jing-shan; Tang, Wei; Zhang, Lei-ke; Xiao, Geng-fu; Ni, Li; Wang, Dao-wen; Zuo, Jian-ping; Jiang, Hua-liang; Bai, Fang; Wu, Yan; Ye, Yang; Xu, Ye-chun</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmacologica Sinica</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1167-1177</span>CODEN:
                <span class="NLM_cas:coden">APSCG5</span>;
        ISSN:<span class="NLM_cas:issn">1671-4083</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Human infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes coronavirus disease 2019 (COVID-19) and there is no cure currently.  The 3CL protease (3CLpro) is a highly conserved protease which is indispensable for CoVs replication, and is a promising target for development of broad-spectrum antiviral drugs.  In this study we investigated the anti-SARS-CoV-2 potential of Shuanghuanglian prepn., a Chinese traditional patent medicine with a long history for treating respiratory tract infection in China.  We showed that either the oral liq. of Shuanghuanglian, the lyophilized powder of Shuanghuanglian for injection or their bioactive components dose-dependently inhibited SARS-CoV-2 3CLpro as well as the replication of SARS-CoV-2 in Vero E6 cells.  Baicalin and baicalein, two ingredients of Shuanghuanglian, were characterized as the first noncovalent, nonpeptidomimetic inhibitors of SARS-CoV-2 3CLpro and exhibited potent antiviral activities in a cell-based system.  Remarkably, the binding mode of baicalein with SARS-CoV-2 3CLpro detd. by X-ray protein crystallog. was distinctly different from those of known 3CLpro inhibitors.  Baicalein was productively ensconced in the core of the substrate-binding pocket by interacting with two catalytic residues, the crucial S1/S2 subsites and the oxyanion loop, acting as a "shield" in front of the catalytic dyad to effectively prevent substrate access to the catalytic dyad within the active site.  Overall, this study provides an example for exploring the in vitro potency of Chinese traditional patent medicines and effectively identifying bioactive ingredients toward a specific target, and gains evidence supporting the in vivo studies of Shuanghuanglian oral liq. as well as two natural products for COVID-19 treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoE-aZrrHwqZrVg90H21EOLACvtfcHk0lggWAdtoMWA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFShu7vI&md5=8560cc7a979d67e0fa0cf17eee71ef00</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41401-020-0483-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41401-020-0483-6%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DH.%2BX.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DW.%2BF.%26aulast%3DLi%26aufirst%3DM.%2BJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DShang%26aufirst%3DW.%2BJ.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DKe%26aufirst%3DC.%2BQ.%26aulast%3DHu%26aufirst%3DH.%2BC.%26aulast%3DGao%26aufirst%3DM.%2BN.%26aulast%3DYu%26aufirst%3DK.%2BQ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DJ.%2BS.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%2BK.%26aulast%3DXiao%26aufirst%3DG.%2BF.%26aulast%3DNi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DD.%2BW.%26aulast%3DZuo%26aufirst%3DJ.%2BP.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%2BC.%26atitle%3DAnti-SARS-CoV-2%2520activities%2520in%2520vitro%2520of%2520Shuanghuanglian%2520preparations%2520and%2520bioactive%2520ingredients%26jtitle%3DActa%2520Pharmacol.%2520Sin.%26date%3D2020%26volume%3D41%26spage%3D1167%26epage%3D1177%26doi%3D10.1038%2Fs41401-020-0483-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span> <span> </span><span class="NLM_article-title">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">271</span>, <span class="refDoi"> DOI: 10.1038/s41422-020-0282-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1038%2Fs41422-020-0282-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=32020029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkt1Ciu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=269-271&author=M.+Wangauthor=R.+Caoauthor=L.+Zhangauthor=X.+Yangauthor=J.+Liuauthor=M.+Xuauthor=Z.+Shiauthor=Z.+Huauthor=W.+Zhongauthor=G.+Xiao&title=Remdesivir+and+chloroquine+effectively+inhibit+the+recently+emerged+novel+coronavirus+%282019-nCoV%29+in+vitro&doi=10.1038%2Fs41422-020-0282-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Remdesivir and chloroquine effectively inhibit the recently emerged novel coronavirus (2019-nCoV) in vitro</span></div><div class="casAuthors">Wang, Manli; Cao, Ruiyuan; Zhang, Leike; Yang, Xinglou; Liu, Jia; Xu, Mingyue; Shi, Zhengli; Hu, Zhihong; Zhong, Wu; Xiao, Gengfu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-271</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In Dec. 2019, a novel pneumonia caused by a previously unknown pathogen emerged in Wuhan, a city of 11 million people in central China.  The initial cases were linked to exposures in a seafood market in Wuhan.  The pathogen was soon identified as a novel coronavirus (2019-nCoV), which is closely related to severe acute respiratory syndrome CoV (SARS-CoV).  Currently, there is no specific treatment against the new virus.  Therefore, identifying effective antiviral agents to combat the disease is urgently needed.  In this study, we evaluated the antiviral efficiency of FAD-approved drugs including ribavirin, penciclovir, nitazoxanide, nafamostat, chloroquine and two well-known broad-spectrum antiviral drugs remdesivir (GS-5734) and favipiravir (T-705) against a clin. isolate of 2019-nCoV in vitro.  Our findings reveal that remdesivir and chloroquine are highly effective in the control of 2019-nCoV infection in vitro.  Since these compds. have been used in human patients with a safety track record and shown to be effective against various ailments, we suggest that they should be assessed in human patients suffering from the novel coronavirus disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCnj2tW4TeD7Vg90H21EOLACvtfcHk0lggWAdtoMWA0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkt1Ciu7k%253D&md5=bdf16b352ebc06bc148ed5c513b7d414</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41422-020-0282-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-020-0282-0%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DZhong%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26atitle%3DRemdesivir%2520and%2520chloroquine%2520effectively%2520inhibit%2520the%2520recently%2520emerged%2520novel%2520coronavirus%2520%25282019-nCoV%2529%2520in%2520vitro%26jtitle%3DCell%2520Res.%26date%3D2020%26volume%3D30%26spage%3D269%26epage%3D271%26doi%3D10.1038%2Fs41422-020-0282-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibaut, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernachio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Kuppeveld, F. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delang, L.</span></span> <span> </span><span class="NLM_article-title">Antiviral activity of broad-spectrum and enterovirus-specific inhibitors against clinical isolates of enterovirus D68</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7782</span>– <span class="NLM_lpage">7785</span>, <span class="refDoi"> DOI: 10.1128/AAC.01375-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=10.1128%2FAAC.01375-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=26369972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVGgtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=7782-7785&author=L.+Sunauthor=A.+Meijerauthor=M.+Froeyenauthor=L.+Zhangauthor=H.+J.+Thibautauthor=J.+Baggenauthor=S.+Georgeauthor=J.+Vernachioauthor=F.+J.+M.+van+Kuppeveldauthor=P.+Leyssenauthor=R.+Hilgenfeldauthor=J.+Neytsauthor=L.+Delang&title=Antiviral+activity+of+broad-spectrum+and+enterovirus-specific+inhibitors+against+clinical+isolates+of+enterovirus+D68&doi=10.1128%2FAAC.01375-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20aR"><div class="casContent"><span class="casTitleNuber">20a</span><div class="casTitle"><span class="NLM_cas:atitle">Antiviral activity of broad-spectrum and enterovirus specific inhibitors against clinical isolates of enterovirus D68</span></div><div class="casAuthors">Sun, Liang; Meijer, Adam; Froeyen, Mathy; Zhang, Linlin; Thibaut, Hendrik Jan; Baggen, Jim; George, Shyla; Vernachio, John; van Kuppeveld, Frank J. M.; Leyssen, Pieter; Hilgenfeld, Rolf; Neyts, Johan; Delang, Leen</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7782-7785</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We investigated the susceptibility of 10 enterovirus D68 (EV-D68) isolates (belonging to clusters A, B, and C) to (entero)virus inhibitors with different mechanisms of action.  The 3C-protease inhibitors proved to be more efficient than enviroxime and pleconaril, which in turn were more effective than vapendavir and pirodavir.  Favipiravir proved to be a weak inhibitor.  Resistance to pleconaril maps to V69A in the VP1 protein, and resistance to rupintrivir maps to V104I in the 3C protease.  A structural explanation of why both substitutions may cause resistance is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1u7CM0ymnsLVg90H21EOLACvtfcHk0lgeY0j8E2Joqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVGgtLo%253D&md5=4e58b350187aeedecec35db48ba3ea7b</span></div><a href="/servlet/linkout?suffix=cit20a&amp;dbid=16384&amp;doi=10.1128%2FAAC.01375-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01375-15%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DL.%26aulast%3DMeijer%26aufirst%3DA.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3DBaggen%26aufirst%3DJ.%26aulast%3DGeorge%26aufirst%3DS.%26aulast%3DVernachio%26aufirst%3DJ.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%2BM.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DHilgenfeld%26aufirst%3DR.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DDelang%26aufirst%3DL.%26atitle%3DAntiviral%2520activity%2520of%2520broad-spectrum%2520and%2520enterovirus-specific%2520inhibitors%2520against%2520clinical%2520isolates%2520of%2520enterovirus%2520D68%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D7782%26epage%3D7785%26doi%3D10.1128%2FAAC.01375-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit20b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Aguado, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibaut, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Priego, E.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimeno, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarasa, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Pérez, M.-J.</span></span> <span> </span><span class="NLM_article-title">9-Arylpurines as a novel class of enterovirus inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">324</span>, <span class="refDoi"> DOI: 10.1021/jm901240p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901240p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVeku7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=316-324&author=L.+Aguadoauthor=H.+J.+Thibautauthor=E.-M.+Priegoauthor=M.-L.+Jimenoauthor=M.-J.+Camarasaauthor=J.+Neytsauthor=M.-J.+P%C3%A9rez-P%C3%A9rez&title=9-Arylpurines+as+a+novel+class+of+enterovirus+inhibitors&doi=10.1021%2Fjm901240p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20bR"><div class="casContent"><span class="casTitleNuber">20b</span><div class="casTitle"><span class="NLM_cas:atitle">9-Arylpurines as a Novel Class of Enterovirus Inhibitors</span></div><div class="casAuthors">Aguado, Leire; Thibaut, Hendrik Jan; Priego, Eva-Maria; Jimeno, Maria-Luisa; Camarasa, Maria-Jose; Neyts, Johan; Perez-Perez, Maria-Jesus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">316-324</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we report on a novel class of enterovirus inhibitors that can be structurally described as 9-arylpurines.  These compds. elicit activity against a variety of enteroviruses in the low μM range including Coxsackie virus A16, A21, A24, Coxsackie virus B3, and echovirus 9.  Structure-activity relationship (SAR) studies indicate that a chlorine or bromine atom is required at position 6 of the purine ring for antiviral activity.  The most selective compds. in this series inhibited Coxsackie virus B3 replication in a dose-dependent manner with EC50 values around 5-8 μM.  No toxicity on different cell lines was obsd. at concns. up to 250 μM.  Moreover, no cross-resistance to TBZE-029 and TTP-8307 CVB3 resistant strains was detected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZK6ihSR_g27Vg90H21EOLACvtfcHk0lgeY0j8E2Joqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVeku7jI&md5=605fb1225f0efebe3a9eeae22d9be2c6</span></div><a href="/servlet/linkout?suffix=cit20b&amp;dbid=16384&amp;doi=10.1021%2Fjm901240p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901240p%26sid%3Dliteratum%253Aachs%26aulast%3DAguado%26aufirst%3DL.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3DPriego%26aufirst%3DE.-M.%26aulast%3DJimeno%26aufirst%3DM.-L.%26aulast%3DCamarasa%26aufirst%3DM.-J.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DP%25C3%25A9rez-P%25C3%25A9rez%26aufirst%3DM.-J.%26atitle%3D9-Arylpurines%2520as%2520a%2520novel%2520class%2520of%2520enterovirus%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D316%26epage%3D324%26doi%3D10.1021%2Fjm901240p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit20c"><span><span class="NLM_label">(c) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coluccia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giorgio, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terme, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirilli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regina, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvestri, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirabelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanelle, P.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Drug Design of Potent Pyrazole Derivatives against Rhinovirus Replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8402</span>– <span class="NLM_lpage">8416</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00931</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00931" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8402-8416&author=L.+D.+Costaauthor=E.+Scheersauthor=A.+Colucciaauthor=A.+Casulliauthor=M.+Rocheauthor=C.+D.+Giorgioauthor=J.+Neytsauthor=T.+Termeauthor=R.+Cirilliauthor=G.+L.+Reginaauthor=R.+Silvestriauthor=C.+Mirabelliauthor=P.+Vanelle&title=Structure-Based+Drug+Design+of+Potent+Pyrazole+Derivatives+against+Rhinovirus+Replication&doi=10.1021%2Facs.jmedchem.8b00931"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20c&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00931%26sid%3Dliteratum%253Aachs%26aulast%3DCosta%26aufirst%3DL.%2BD.%26aulast%3DScheers%26aufirst%3DE.%26aulast%3DColuccia%26aufirst%3DA.%26aulast%3DCasulli%26aufirst%3DA.%26aulast%3DRoche%26aufirst%3DM.%26aulast%3DGiorgio%26aufirst%3DC.%2BD.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DTerme%26aufirst%3DT.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DRegina%26aufirst%3DG.%2BL.%26aulast%3DSilvestri%26aufirst%3DR.%26aulast%3DMirabelli%26aufirst%3DC.%26aulast%3DVanelle%26aufirst%3DP.%26atitle%3DStructure-Based%2520Drug%2520Design%2520of%2520Potent%2520Pyrazole%2520Derivatives%2520against%2520Rhinovirus%2520Replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8402%26epage%3D8416%26doi%3D10.1021%2Facs.jmedchem.8b00931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GHT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GHT','PDB','4GHT'); return false;">PDB: 4GHT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7DNC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7DNC','PDB','7DNC'); return false;">PDB: 7DNC</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02258">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_43613"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02258?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02258</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">An overlay of EV71 3C<sup>pro</sup>-<b>18p</b> and <b>AG7088</b> complex structures; the binding pocket of 3C<sup>pro</sup> of CoXB3, RV-A02, and EV71; the anti-SARS-CoV-2 activities of compounds; the <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of target compounds; the purity of target compounds; and the crystallography data collection and refinement statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula stings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_001.pdf">jm0c02258_si_001.pdf (3.41 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02258/suppl_file/jm0c02258_si_002.csv">jm0c02258_si_002.csv (3.23 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors have been deposited into the Protein Data Bank with accession code 7DNC (EV71 3C<sup>pro</sup>-<b>18p</b>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02258&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02258%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02258" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799949bf8213dc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
