
<html lang="en"     class="pb-page"  data-request-id="7c19757b-e4a7-43e9-8d5b-7fa0bdc35ef8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2005.48.issue-22;article:article:10.1021/jm0493360;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="PostDOCK:  A Structural, Empirical Approach to Scoring Protein Ligand Complexes" /></meta><meta name="dc.Creator" content="Clayton  Springer,*" /></meta><meta name="dc.Creator" content="Helgi  Adalsteinsson," /></meta><meta name="dc.Creator" content="Malin M.  Young," /></meta><meta name="dc.Creator" content="Philip W.  Kegelmeyer, and" /></meta><meta name="dc.Creator" content="Diana C.  Roe" /></meta><meta name="dc.Description" content="In this work we introduce a postprocessing filter (PostDOCK) that distinguishes true binding ligand−protein complexes from docking artifacts (that are created by DOCK 4.0.1). PostDOCK is a pattern ..." /></meta><meta name="Description" content="In this work we introduce a postprocessing filter (PostDOCK) that distinguishes true binding ligand−protein complexes from docking artifacts (that are created by DOCK 4.0.1). PostDOCK is a pattern ..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="September 30, 2005" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm0493360" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-48-22-6821" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(04)09336-7" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2005 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm0493360" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm0493360" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm0493360" /></link>
        
    
    

<title>PostDOCK:  A Structural, Empirical Approach to Scoring Protein Ligand Complexes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm0493360" /></meta><meta property="og:title" content="PostDOCK:  A Structural, Empirical Approach to Scoring Protein Ligand Complexes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360n00001.jpeg" /></meta><meta property="og:description" content=" In this work we introduce a postprocessing filter (PostDOCK) that distinguishes true binding ligand−protein complexes from docking artifacts (that are created by DOCK 4.0.1). PostDOCK is a pattern recognition system that relies on (1) a database of complexes, (2) biochemical descriptors of those complexes, and (3) machine learning tools. We use the protein databank (PDB) as the structural database of complexes and create diverse training and validation sets from it based on the “families of structurally similar proteins” (FSSP) hierarchy. For the biochemical descriptors, we consider terms from the DOCK score, empirical scoring, and buried solvent accessible surface area. For the machine-learners, we use a random forest classifier and logistic regression. Our results were obtained on a test set of 44 structurally diverse protein targets. Our highest performing descriptor combinations obtained ∼19-fold enrichment (39 of 44 binding complexes were correctly identified, while only allowing 2 of 44 decoy complexes), and our best overall accuracy was 92%.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm0493360"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm0493360">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm0493360&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm0493360&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm0493360&amp;href=/doi/10.1021/jm0493360" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2005</span><span class="cit-fg-volume">, 48</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 6821-6831</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/48/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm049418o" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm0501782" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">PostDOCK:  A Structural, Empirical Approach to Scoring Protein Ligand Complexes</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Clayton++Springer">Clayton Springer</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Helgi++Adalsteinsson">Helgi Adalsteinsson</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Malin+M.++Young">Malin M. Young</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Philip+W.++Kegelmeyer">Philip W. Kegelmeyer</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Diana+C.++Roe">Diana C. Roe</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Sandia National Labs, P.O. Box 969, MS 9951, Livermore, California 94551 </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm0493360&amp;href=/doi/10.1021%2Fjm0493360" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2005</span></span><span class="cit-volume">, 48</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 6821–6831</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 30, 2005</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 August 2004</li><li><span class="item_label"><b>Published</b> online</span>30 September 2005</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 November 2005</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm0493360" title="DOI URL">https://doi.org/10.1021/jm0493360</a></div><div class="article_header-article-copyright"><strong>Copyright © 2005 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6821%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DClayton%2BSpringer%252C%2BHelgi%2BAdalsteinsson%252C%2BMalin%2BM.%2BYoung%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D48%26issueNum%3D22%26contentID%3Djm0493360%26title%3DPostDOCK%253A%25E2%2580%2589%2BA%2BStructural%252C%2BEmpirical%2BApproach%2Bto%2BScoring%2BProtein%2BLigand%2BComplexes%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6831%26publicationDate%3DNovember%2B2005">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm0493360"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">399</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">26</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm0493360" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;PostDOCK:  A Structural, Empirical Approach to Scoring Protein Ligand Complexes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Clayton&quot;,&quot;last_name&quot;:&quot;Springer&quot;},{&quot;first_name&quot;:&quot;Helgi&quot;,&quot;last_name&quot;:&quot;Adalsteinsson&quot;},{&quot;first_name&quot;:&quot;Malin&quot;,&quot;last_name&quot;:&quot;M. Young&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;W. Kegelmeyer&quot;},{&quot;first_name&quot;:&quot;Diana&quot;,&quot;last_name&quot;:&quot;C. Roe&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2005&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;48&quot;,&quot;pages&quot;:&quot;6821-6831&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm0493360&quot;},&quot;abstract&quot;:&quot;In this work we introduce a postprocessing filter (PostDOCK) that distinguishes true binding ligand−protein complexes from docking artifacts (that are created by DOCK 4.0.1). PostDOCK is a pattern recognition system that relies on (1) a database of complexes, (2) biochemical descriptors of those complexes, and (3) machine learning tools. We use the protein databank (PDB) as the structural database of complexes and create diverse training and validation sets from it based on the “families of structurally similar proteins” (FSSP) hierarchy. For the biochemical descriptors, we consider terms from the DOCK score, empirical scoring, and buried solvent accessible surface area. For the machine-learners, we use a random forest classifier and logistic regression. Our results were obtained on a test set of 44 structurally diverse protein targets. Our highest performing descriptor combinations obtained ∼19-fold enrichment (39 of 44 binding complexes were correctly identified, while only allowing 2 of 44 decoy complexes)&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm0493360&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0493360" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm0493360&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0493360" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm0493360&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0493360" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm0493360&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0493360&amp;href=/doi/10.1021/jm0493360" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm0493360" /></input><a href="/doi/pdf/10.1021/jm0493360" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (722 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm0493360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm0493360%26sid%3Dliteratum%253Aachs%26pmid%3D16250641%26genre%3Darticle%26aulast%3DSpringer%26date%3D2005%26atitle%3DPostDOCK%253A%25E2%2580%2589%2BA%2BStructural%252C%2BEmpirical%2BApproach%2Bto%2BScoring%2BProtein%2BLigand%2BComplexes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D48%26issue%3D22%26spage%3D6821%26epage%3D6831%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=291901" title="Chemical calculations">Chemical calculations</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/48/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360n00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360n00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In this work we introduce a postprocessing filter (PostDOCK) that distinguishes true binding ligand−protein complexes from docking artifacts (that are created by DOCK 4.0.1). PostDOCK is a pattern recognition system that relies on (1) a database of complexes, (2) biochemical descriptors of those complexes, and (3) machine learning tools. We use the protein databank (PDB) as the structural database of complexes and create diverse training and validation sets from it based on the “families of structurally similar proteins” (FSSP) hierarchy. For the biochemical descriptors, we consider terms from the DOCK score, empirical scoring, and buried solvent accessible surface area. For the machine-learners, we use a random forest classifier and logistic regression. Our results were obtained on a test set of 44 structurally diverse protein targets. Our highest performing descriptor combinations obtained ∼19-fold enrichment (39 of 44 binding complexes were correctly identified, while only allowing 2 of 44 decoy complexes), and our best overall accuracy was 92%. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm0493360AF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  Current address:  Novartis Institutes for Biomedical Research, 100 Technology Square, Cambridge, MA 02139. Phone (617) 871-7588. <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="c2a1aea3bbb6adacecb1b2b0abaca5a7b082acadb4a3b0b6abb1eca1adafec">[email protected]</a></p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76062" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76062" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Molecular docking calculations are a well-established tool in drug discovery. Docking is used to screen large databases of small molecules against a given target receptor and to identify those that can reasonably be expected to bind to the receptor. The core function of all docking programs is to identify ligand poses (i.e. orientations and conformations) that match the structural and chemical characteristics of the target receptor. Typically, docking calculations are run on large sets of small molecules (10<sup>6</sup>−10<sup>9</sup> molecules or larger); thus, execution speed is increased at the expense of accuracy. The top ranking ligands are then considered for further development. </div><div class="NLM_p">The most accurate calculations of the free energy of binding for a protein−ligand complex use explicit waters in full molecular dynamics/Monte Carlo simulation.<sup><a onclick="showRef(event, 'jm0493360b00001 jm0493360b00002 jm0493360b00003'); return false;" href="javascript:void(0);" class="ref jm0493360b00001 jm0493360b00002 jm0493360b00003">1-3</a></sup> These calculations are still at least 6 orders of magnitude too slow to be used in large-scale screening of putative ligands.<sup><a onclick="showRef(event, 'jm0493360b00004 jm0493360b00005 jm0493360b00006 jm0493360b00007'); return false;" href="javascript:void(0);" class="ref jm0493360b00004 jm0493360b00005 jm0493360b00006 jm0493360b00007">4-7</a></sup> Although docking does not include explicit representations of solvent water molecules, implicit solvation models, such as generalized Born solvation, have been used successfully in docking simulations,<sup><a onclick="showRef(event, 'jm0493360b00008'); return false;" href="javascript:void(0);" class="ref jm0493360b00008">8</a></sup> but the added computational cost of those methods restricts them to much smaller datasets than occur in high throughput screening. Faster docking methods make further approximations to these models.<sup><a onclick="showRef(event, 'jm0493360b00009 jm0493360b00010'); return false;" href="javascript:void(0);" class="ref jm0493360b00009 jm0493360b00010">9,10</a></sup></div><div class="NLM_p">Due to the compromises between accuracy and speed that have to be made for large scale docking simulations, it is common to refer to the binding value assigned to a docked pose as a “score” rather than as free energy and to refer to the simplified physics models used to arrive at these scores as “scoring functions”. The most common fast scoring functions used for screening large datasetsare:  force field based scoring schemes<sup><a onclick="showRef(event, 'jm0493360b00011 jm0493360b00012 jm0493360b00013 jm0493360b00014'); return false;" href="javascript:void(0);" class="ref jm0493360b00011 jm0493360b00012 jm0493360b00013 jm0493360b00014">11-14</a></sup> in which the nonbonded electrostatic and Lennard−Jones terms are used from a molecular mechanics force field to estimate intermolecular interactions (e.g., DOCK), knowledge-based scoring schemes<sup><a onclick="showRef(event, 'jm0493360b00015 jm0493360b00016'); return false;" href="javascript:void(0);" class="ref jm0493360b00015 jm0493360b00016">15,16</a></sup> in which binding potentials are derived from statistical analysis of cocrystallized protein−ligand pairs (e.g., Pmf, DrugScore), and empirically derived scoring schemes<sup><a onclick="showRef(event, 'jm0493360b00017 jm0493360b00018'); return false;" href="javascript:void(0);" class="ref jm0493360b00017 jm0493360b00018">17,18</a></sup> which begin by assuming a chemically intuitive form for the descriptors (e.g., hydrogen bonding and contact surface area) and then fit coefficients to create a potential function (e.g., LUDI and ChemScore). These scoring functions are useful singly or in consensus scoring.<sup><a onclick="showRef(event, 'jm0493360b00019'); return false;" href="javascript:void(0);" class="ref jm0493360b00019">19</a></sup> Bissantz et al.<sup><a onclick="showRef(event, 'jm0493360b00020'); return false;" href="javascript:void(0);" class="ref jm0493360b00020">20</a></sup> have shown that none of several popular scoring functions is best for both hydrophilic (thymadine kinase) and hydrophobic (estrogen receptor) targets. The functional form of empirically derived scoring schemes can be chosen to optimize computational speed and chemical interpretability, but their predictive accuracy usually does not necessarily extend beyond the target receptor families on which they were trained. </div><div class="NLM_p">In this paper, we describe the development of a new approach to evaluating and ranking ligand−receptor complexes. The goal is to improve the results obtained by the fast, but inexact, scoring methods in docking by adding a postprocessing filter (the “postDOCK” filter) that can divide the output poses into binders and decoys. Each postDOCK filter uses molecular descriptors to describe the characteristics of the ligand−receptor complex, and pattern recognition to separate out the binding from the nonbinding ligands. The molecular descriptors include ones similar to those used in existing empirical scoring schemes, as well as novel descriptors. We use a random forest classifier.<sup><a onclick="showRef(event, 'jm0493360b00021'); return false;" href="javascript:void(0);" class="ref jm0493360b00021">21</a></sup> Random forest is an extension of the classification and regression tree (CART) algorithm. CARTs have previously been used in chemical applications,<sup><a onclick="showRef(event, 'jm0493360b00022'); return false;" href="javascript:void(0);" class="ref jm0493360b00022">22</a></sup> are nonparametric, and train quickly,<a class="ref showTableEvent internalNav" data-ID="jm0493360t00001" href="#jm0493360t00001"></a><a class="ref showTableEvent internalNav" data-ID="jm0493360t00001a" href="#jm0493360t00001a"></a> and their output is a readily interpretable tree.<div class="NLM_table-wrap" id="jm0493360t00001"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 1.  Selected Examples of Protein−Ligands and Decoys</p></div></div><div class="scrollable-table-wrap"><table class="table single-image " width="100%"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360u00001a.gif" alt="" id="_i4" /></img></td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="jm0493360t00001a"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table I (Continued)</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360u00001b.gif" alt="" id="_i5" /></img></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td></tr></tbody></table></div></div></div><div class="NLM_p last"><a name="jm004802206824" id="jm004802206824"></a>To address the high dimensional space created by using multiple descriptors, described below, we used ensembles of learners and feature set selection techniques. To provide broad applicability, postDOCK was trained on 152 protein families<sup><a onclick="showRef(event, 'jm0493360b00023 jm0493360b00024'); return false;" href="javascript:void(0);" class="ref jm0493360b00023 jm0493360b00024">23,24</a></sup> that represent a sequentially diverse set of protein targets. Using a test set of 44 structurally diverse protein targets, the postDOCK filter exhibits a 19-fold enrichment for identifying correctly docked vs decoy complexes over random filtering. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Computational Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08348" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08348" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Construction of the Database of Binders and Decoys.</b> To construct the binding complexes for the data sets, all protein−ligand complexes from the protein data bank (PDB) were scanned. We eliminated ligands that had missing heavy atom coordinates, were common crystallographic solvents, that were monomeric sugars containing less than 10 atoms, that had more than 100 heavy atoms, or that had greater than 30 rotatable bonds. Missing side chains and hydrogen atoms of the proteins were added to all complexes in the final data sets using Sybyl. Ligands were atom typed and hydrogens added by hand. To remove steric clashes, the position of the ligand in the receptor site was optimized using 100 steps of the simplex optimizer in DOCK4.0.1.<sup><a onclick="showRef(event, 'jm0493360b00025'); return false;" href="javascript:void(0);" class="ref jm0493360b00025">25</a></sup> Removing steric clashes is also important because we do not want steric clashes to be associated with true binding poses, even if they are found frequently in crystallographic poses. </div><div class="NLM_p">To determine the performance of pattern recognition algorithms on the relevant problem requires proper selection of the training set. Since the postDOCK filter is intended as a general docking filter, we constructed the training and test sets using the broadest range of protein targets available. The 152 training complexes and 44 test complexes were selected from this set of PDB complexes using the Fold classification based on Structure-Structure alignment of Proteins (FSSP) fold tree<sup>23,24</sup> to achieve diversity across all known proteins. The training set was a sequentially diverse set generated by selecting one protein target from each cluster in the lowest tier of the FSSP hierarchy. From the ∼2400 clusters in FSSP, most of which had no ligands, there were a total of 152 ligand−protein complexes for the training set (Table S1). The validation set was drawn from the FSSP fold tree's structurally diverse top tier. The top tier of FSSP has ∼600 structurally diverse protein folds. If we limit ourselves to no more than one protein−ligand complex from each fold that is not already used in the training set we are left with 44 complexes for the validation set (Table S2). Some selected examples of protein targets, their true binders, and the decoy used are provided in Table 1. </div><div class="NLM_p">We also created training and test sets of “decoy” ligands from the other ligands in the PDB. Separate decoys were generated for each protein in the training and test sets. The DOCKing process began with a DOCK sphere-set generated using the coordinates of the native ligand to identify the binding site. All ligands were DOCKed into each protein target (500 orientations with a distance tolerance of 0.25 Å, a distance minimum of 2.0 Å, and a node matching minimum of 3 and maximum of 10 nodes). Flexible docking was used with simultaneous search of torsions and default flexible parameters. The protein and ligand were assigned AMBER all atom charges and Gasteiger-Marsili partial charges respectively by Sybyl, followed by simplex optimization with DOCK's default force field parameters. Those ligands that were successfully DOCKed and had a daylight fingerprint Tanimoto coefficient<sup>31</sup> of <0.5 with the native binder were candidate decoys. From this pool, the decoy ligand was selected at random. At this Tanimoto threshold we hoped to reduce the false negative rate in the decoy dataset no more frequently than a binder is found through random screening. Although not the focus of their paper, Martin et al.<sup>32</sup> show that the probability of compounds being active steadily decreases away from an active compound. For the MAO inhibitors, 0.5 is the lowest Tanimito distance shown, and it appears to have a slightly higher probability of being active than the average in the set. Table 1 shows selected pairs of binders and decoys. The diversity of the ligands with respect to hydrogen bonding acceptors and donors and CLogP<sup>33</sup> for the training and test sets are shown in Figures <a class="ref showTableEvent internalNav" data-ID="jm0493360t00002" href="#jm0493360t00002"></a><a class="ref showTableEvent internalNav" data-ID="jm0493360t00003" href="#jm0493360t00003"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00001" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00002" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00003" id="" class=" internalNav"></a>1 and 2. <div id="_i7"><figure id="jm0493360f00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00001.jpeg" id="_i8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 The test set's hydrophobic (CLogp) and hydrophilic (H-bond donors plus acceptors, calculated by Daylight) properties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00002.jpeg" id="_i9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 The training set's hydrophobic (CLogp) and hydrophilic (H-bond donors plus acceptors, calculated by Daylight) properties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00003.jpeg" id="_i10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 A schematic set of ROC curves showing perfect performance, high performance, low performance, and random performance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm0493360t00002"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 2.  Descriptors Used</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">name of descriptor</td><td class="colsep2 rowsep2">type of descriptors</td><td class="colsep2 rowsep2">number of descriptors </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DOCK </td><td class="colsep2 rowsep2">DOCK direct interaction:  vdw and electrostatics </td><td class="colsep2 rowsep2">2+1<i><sup>a</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SASA </td><td class="colsep2 rowsep2">solvent accessible surface area (1.4 Å probe) calculated by SURF </td><td class="colsep2 rowsep2">11+1<i><sup>a</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SASA-L </td><td class="colsep2 rowsep2">solvent accessible surface area (6 Å probe) calculated by SURF </td><td class="colsep2 rowsep2">11+1<i><sup>a</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ChemScore </td><td class="colsep2 rowsep2">hydrogen bonding, metal binding, lipophilic, rotatable bonds </td><td class="colsep2 rowsep2">4+1<i><sup>a</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">32</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> The additional (+1) descriptor is from treating the sum of the descriptors as an additional descriptor. This was included because decision trees do not automatically detect linear combinations.</p></div></div><div class="NLM_table-wrap" id="jm0493360t00003"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 3.  Test Set Performance of Random Forest (each row shows the performance for a subset when used to build an ensemble of decision trees)</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DOCK<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">SASA<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">SASA-L<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">ChemScore<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">Percent Binders correctly identified at best performance<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">Percent decoys incorrectly identified at best performance<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">Enrichment identified  at 2 FP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2" colspan="4">all<sup><i>c</i></sup></td><td class="colsep2 rowsep2">86.4 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">18.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">es </td><td class="colsep2 rowsep2">Total and O_BYL<sup><i>e</i></sup></td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb<i><sup>f</sup></i></td><td class="colsep2 rowsep2">86.4 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total and C_BYL </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb and lipo </td><td class="colsep2 rowsep2">86.4 </td><td class="colsep2 rowsep2">4.5 </td><td class="colsep2 rowsep2">19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb and rot </td><td class="colsep2 rowsep2">84.1 </td><td class="colsep2 rowsep2">4.5 </td><td class="colsep2 rowsep2">18.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">es </td><td class="colsep2 rowsep2">Total and C_ALI </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">84.1 </td><td class="colsep2 rowsep2">4.5 </td><td class="colsep2 rowsep2">18.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">C_ALI </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">84.1 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">18.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">79.5 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">18 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">C_ALI </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">81.8 </td><td class="colsep2 rowsep2">4.5 </td><td class="colsep2 rowsep2">18 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total and C_ARO </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">79.5 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">17.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">81.8 </td><td class="colsep2 rowsep2">4.5 </td><td class="colsep2 rowsep2">18 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">75.0 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">16.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">O_BYL </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">68.2 </td><td class="colsep2 rowsep2">4.5 </td><td class="colsep2 rowsep2">15 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">C_ALI </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">hb </td><td class="colsep2 rowsep2">72.7 </td><td class="colsep2 rowsep2">6.81 </td><td class="colsep2 rowsep2">14.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">84.1 </td><td class="colsep2 rowsep2">40.9 </td><td class="colsep2 rowsep2">7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">50.0 </td><td class="colsep2 rowsep2">13.6 </td><td class="colsep2 rowsep2">7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">C_ALI </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">47.7 </td><td class="colsep2 rowsep2">18.2 </td><td class="colsep2 rowsep2">7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">es </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">27.3 </td><td class="colsep2 rowsep2">6.8 </td><td class="colsep2 rowsep2">5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">29.5 </td><td class="colsep2 rowsep2">6.8 </td><td class="colsep2 rowsep2">3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">vdW </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">45.4 </td><td class="colsep2 rowsep2">27.3 </td><td class="colsep2 rowsep2">3.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> The leftmost four columns describe which descriptors are in each subset. <i><sup>b</sup></i> 50% threshold is the max performance of the ROC curve. The error bar for these columns if about 3.5%. <i><sup>c</sup></i> “all” means that each component was included. <i><sup>d</sup></i><sup></sup> “Total” means the summed descriptor values. <i><sup>e</sup></i><sup></sup> “C_BYL”, “O_BYL”, and “C_ALI” are components of the SASA. <i><sup>f</sup></i><sup></sup> “hb”, “lipo”, and “rot” are components of ChemScore.</p></div></div></div><div class="NLM_p"><b>Structural Descriptors. </b>Structural descriptors were used to characterize the protein−ligand complexes. First the structures were minimized using the DOCK total score to remove any steric clashes from both the binders and decoys. This ensured that only valid DOCK poses were evaluated. The set of feature descriptors used to characterize the ligand−protein complexes are summarized in Table 2. The DOCK total score provided the nonbonded van der Waals and electrostatic interaction force field terms, but the other descriptors were calculated in a postprocessing step. These included descriptors from our in-house implementation of the Eldridge ChemScore function.<sup>18</sup> The Eldridge function consists of terms for hydrogen bonding, metal bonding, lipophilicity, and the number of bonds whose rotation is blocked by the presence of the receptor. In addition, we calculated a series of buried solvent accessible surface area (SASA) descriptors, using SURF<sup>34</sup> with a 1.4 Å probe, as a computationally inexpensive way of gauging solvation effects. We then calculated the surface attributable to each of a set of atom types, in an atomic solvation parameter<sup>35</sup> inspired approach, and used each as a separate descriptor. The atom types were based on SYBYL MOL2 atom types (see Table S3). As there is evidence that more than one length scale is needed to describe the self-association of hydrogen bonding water in aqueous solvation,<sup>6</sup> we also created a novel descriptor called SASA-L. Our goal with this descriptor was to capture the clathrate ring length scale effects of the hydrogen bonding-associated network, so we selected a 6 Å probe (as compared to 1.4 Å for a water-sized SASA probe) to calculate the SASA-L. Again we calculated how much surface was attributable to each surface atom type (see Table S3) to generate a set of descriptors. Finally, because trees consider each descriptor individually, rather than in a linear combination, decision trees can benefit from including totals of our DOCK, SASA, SASA-L, and ChemScore descriptors as well as the 28 component descriptors (for a total of 32 descriptors considered). The correlation matrix of the selected descriptor set is provided in the Supporting Information. </div><div class="NLM_p"><b>The Pattern Recognition Algorithm.</b> We used R's package of random forest (called randomForest).<sup>21</sup> Important individual features were selected by variable importance,<sup>21</sup> and subsets were selected from high performing individual features. To select which feature sets to use, we began with the the variables selected by random forest's variable selection procedure. From the top 20 of these all pairs, triplets and quartets were assessed, and those with the highest performance are noted. For a single learner (either tree or linear <a name="jm004802206825" id="jm004802206825"></a> model), reducing the number of descriptors improves performance because the relatively small number of complexes in the PDB-based training set is spread over fewer dimensions.<sup><a onclick="showRef(event, 'jm0493360b00026 jm0493360b00027 jm0493360b00028 jm0493360b00029 jm0493360b00030 jm0493360b00031 jm0493360b00032 jm0493360b00033 jm0493360b00034 jm0493360b00035 jm0493360b00036 jm0493360b00037 jm0493360b00038'); return false;" href="javascript:void(0);" class="ref jm0493360b00026 jm0493360b00027 jm0493360b00028 jm0493360b00029 jm0493360b00030 jm0493360b00031 jm0493360b00032 jm0493360b00033 jm0493360b00034 jm0493360b00035 jm0493360b00036 jm0493360b00037 jm0493360b00038">36-38</a></sup> Finally, the most important and appealing aspect of using fewer descriptors is to produce a simpler, more physically interpretable model. </div><div class="NLM_p">Once optimal descriptor subsets were determined, we developed linear models to complement the ensemble of decision trees for these sets. Our linear models are logistic regression (LR) <sup>39,40</sup> with the glm package of R.<sup>41</sup></div><div class="NLM_p last"><b>Creating a ROC Curve.</b> To calculate the performance of our models over the full range of specificity and sensitivity, we have calculated receiver operator characteristic (ROC) curves.<sup>42</sup> To generate an ROC curve, we adjust the threshold at which the ensemble of learners determines that a test example is a binder (or decoy). At one extreme, we begin at the bottom, left corner where the threshold is set so high that all the examples are declared decoys (see Figure 1). As the threshold is decreased incrementally, more examples are declared binders. On the ROC curve a true binder being declared a binder moves the curve up, and a decoy being declared a binder moves the curve to the right. In a perfect ROC curve all of the binders are found first and the curve goes to the top, left corner. As the threshold is increased, all the examples are declared binders and the curve goes from the bottom, left to the top, right corner. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Searching for Optimal Descriptor Subsets.</b> To capture more of the essential biochemistry of complex stability, we considered a large number of descriptors (see Table 2). With 32 descriptors, there are 2<sup>32</sup> − 1 (or <a name="jm004802206826" id="jm004802206826"></a> 4.29 × 10<sup>9</sup>) possible descriptor subsets. To reduce the choice of subsets to a computationally tractable problem, we applied feature set selection methods. This data set is fairly straightforward so forward feature set selection and variable selection all identify very similar descriptor subsets as promising. Selected high-performing subsets are shown in Table 3. This table shows the number of binders and<a class="ref showTableEvent internalNav" data-ID="jm0493360t00004" href="#jm0493360t00004"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00004" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00005" id="" class=" internalNav"></a> decoys correctly and incorrectly identified at the best performing point on the ROC curve (i.e., closest to the top, left corner). We expect statistical fluctuations on the order of sqrt((<i>N</i>)<i>p</i>(1 <i>−</i><i> p</i>)) ∼ ±9 for the enrichment factor in the top performing sets. All top subsets include ChemScore hydrogen bonding (H-bond) and a buried surface area (SASA) term. Many of these subsets also include terms from the DOCK force-field score and the large sphere SASA-L. <div id="_i12"><figure id="jm0493360f00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00004.jpeg" id="_i13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Full ROC curves for selected descriptor subsets using a random forest learner (see also Table 3). Overall performance is determined by proximity to the top, left corner. The single descriptor subsets make up the lowest performers. Selected points are presented numerically in Table 3. Figure 5 gives a detailed view of the high performing region.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00005.jpeg" id="_i14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 Detail of the high-performing region of Figure 4 (top, left). All the best subsets contain SASA<sub>TOTAL</sub> and ChemScore<sub>H</sub><sub>-</sub><sub>BOND</sub>, and all the lowest performing subsets contain only a single descriptor. Those without a subscript are using all the descriptor from that source (Table 2). (This figure uses the same line colors and symbols that are used in Figure 4.)</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm0493360t00004"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 4.  Test Set Performance from Selected Logistic Regression Subsets</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DOCK<i><sup>a</sup></i></td><td class="colsep2 rowsep2">SASA<i><sup>a</sup></i></td><td class="colsep2 rowsep2">ChemScore<i><sup>a</sup></i></td><td class="colsep2 rowsep2">percent of binders correctly identified at best performance<i><sup>b</sup></i></td><td class="colsep2 rowsep2">percent of decoys incorrectly identified at best performance<i><sup>b</sup></i></td><td class="colsep2 rowsep2">enrichment factor at 5% FP </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Hb </td><td class="colsep2 rowsep2">84.1 </td><td class="colsep2 rowsep2">6.8 </td><td class="colsep2 rowsep2">16.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Hb </td><td class="colsep2 rowsep2">70.5 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">15.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Hb </td><td class="colsep2 rowsep2">68.2 </td><td class="colsep2 rowsep2">2.3 </td><td class="colsep2 rowsep2">15 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">63.6 </td><td class="colsep2 rowsep2">13.6 </td><td class="colsep2 rowsep2">9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Hb </td><td class="colsep2 rowsep2">70.5 </td><td class="colsep2 rowsep2">6.8 </td><td class="colsep2 rowsep2">13.5 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">63.6 </td><td class="colsep2 rowsep2">13.6 </td><td class="colsep2 rowsep2">9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">Total </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">61.3 </td><td class="colsep2 rowsep2">11.3 </td><td class="colsep2 rowsep2">10.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> The four left-most columns describe which descriptors are in each subset.<i><sup>b</sup></i><sup></sup> The error bar for this column is about 7.5%.</p></div></div></div><div class="NLM_p"><b>Performance of Descriptor Subsets.</b> To evaluate the full performance of the descriptor subsets on our test set, we examined each over its entire range of sensitivity and specificity using ROC curves (see Figures 3 and 4). We see that the lowest performing subsets plotted in Figure 4 are composed of single descriptors; conversely, all of the top performing feature sets rely on feature synergies. Table 3 shows the midpoint on the ROC curve for each descriptor subset. To estimate enrichment performance, we also note the number of true positives discovered when 2 (of 44) false positives are allowed for each descriptor subset. We note that although the existing scoring schemes DOCK and ChemScore, which are each summarized in a single descriptor, eliminate many decoys (5× and 7× enrichment are seen, respectively), by adding additional descriptors we can obtain up to 19× enrichment (Table 3). In screening, the goal is enrichment, so while getting the binding complexes correct is important, it is critical to correctly identify (and then discard) as many decoys <a name="jm004802206827" id="jm004802206827"></a> as possible. In this case, enrichment factor is the best metric to evaluate. Note that ROC curves are asymptotically equivalent to enrichment curves (true positives vs all positives) when the number of false positives becomes very large (e.g., when screening a large, diverse library<a class="ref showTableEvent internalNav" data-ID="jm0493360t00005" href="#jm0493360t00005"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00006" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00007" id="" class=" internalNav"></a> for hits). For other applications (e.g., obtaining docking poses), a balanced consideration of binding and nonbinding interactions may be desired, and overall performance is the appropriate metric. One of our key results is to identify the best performing descriptor set(s) for the postDOCK filter for each of these two applications. Apparently, synergy between hydrophobic (either SASA<sub>total</sub> or ChemScore<sub>lipophilic</sub>) and hydrophilic (as ChemScore<sub>H</sub><sub>-</sub><sub>bonds</sub>) descriptors is essential for both enrichment and overall performance:  these descriptors appear in all our best subsets (Figures 3 and 4, and Table 3).<a data-tab="pane-pcw-Figures" href="#jm0493360f00008" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00009" id="" class=" internalNav"></a><div id="_i15"><figure id="jm0493360f00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00006.jpeg" id="_i16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Full ROC curves for selected descriptor subsets using logistic regression. Descriptor sets with a subscript are for a single descriptor. Overall performance is measured by proximity to the top, left corner. The best subset contains SASA<sub>TOTAL</sub> and ChemScore<sub>H</sub><sub>-</sub><sub>BOND</sub>. Lowest performing subsets are composed of a single descriptor. (Table 2). Selected points from this figure are presented numerically in Table 4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00007.jpeg" id="_i17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Detail of high performers from Figure 6 and using the same colors and symbols. Overall performance is by proximity to the top, left corner. The best subset contains SASA<sub>total</sub> and CSCORE<sub>H</sub><sub>-</sub><sub>bond</sub>. Lowest performing subsets are composed of a single descriptor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00008.jpeg" id="_i18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 The points show the values of ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub> and total SASA<sub>total</sub> descriptors for the test set for both binders and decoys. The blue lines show the partitioning of the space as given by a single tree on the training set into rectangular regions which are labeled 1 through 5. The regions 1, 2, and 4 contain a majority of binders, and regions 3 and 5 contain a majority of decoys. Eight points (discussed in the text) are labeled by the pdb id of the target, four targets with a binder and decoy each. The black type labels and black diamond points are the binding examples and the red code labels are the decoys. The poses and structures for these points are shown in Figures 9−16.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00009" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00009.jpeg" id="_i19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 The decoy gdn (from 1hnc) docked into HIV protease (1hpx) falls in region 3 of Figure 8. It does not bury enough surface area or have enough H-bonding to be mistaken for a binder.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm0493360t00005"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 5.  Coefficients from Selected Logistic Regression Models<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">models</td><td class="colsep2 rowsep2">intercept</td><td class="colsep2 rowsep2">ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub></td><td class="colsep2 rowsep2">SASA<sub>total</sub></td><td class="colsep2 rowsep2">DOCK<sub>total</sub></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SASA<sub>total </sub>+ ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub></td><td class="colsep2 rowsep2">−2.64 ± 0.37 </td><td class="colsep2 rowsep2">−0.314 ± 0.043 </td><td class="colsep2 rowsep2">−0.00406 ± 0.00097 </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DOCK<sub>total</sub> + ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub></td><td class="colsep2 rowsep2">−0.02 ± 0.37 </td><td class="colsep2 rowsep2">−0.380 ± 0.048 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">+0.066 ± 0.016 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DOCK<sub>total</sub> + SASA<sub>total</sub> + ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub></td><td class="colsep2 rowsep2">−1.29 ± 0.46 </td><td class="colsep2 rowsep2">−0.367 ± 0.049 </td><td class="colsep2 rowsep2">−0.0053 ± 0.0011 </td><td class="colsep2 rowsep2">+0.076 ± 0.017 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DOCK<sub>total</sub></td><td class="colsep2 rowsep2">+0.10 ± 0.26 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">+0.0036 ± 0.0086 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SASA<sub>total</sub></td><td class="colsep2 rowsep2">−1.66 ± 0.29 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">−0.00541 ± 0.00090 </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ChemScore<sub>H-bond</sub></td><td class="colsep2 rowsep2">−1.44 ± 0.20 </td><td class="colsep2 rowsep2">−0.316 ± 0.041 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> There is one subset per row and one column for each descriptor. Empty cells indicate that the descriptor is not included in that subset. The “std dev” are as reported from R's glm package.</p></div></div></div><div class="NLM_p"><b>Enrichment Results.</b> The ROC curve's initial slope (the enrichment factor) is the relevant performance metric in virtual screening where the goal is to minimize the number of false positives for each true positive discovered. Random screening has an enrichment factor of 1-fold; useful procedures have higher enrichment factors. Existing single descriptor scoring schemes DOCK and ChemScore each have 5-fold or 7-fold enrichments. The highest enrichment is found in subsets that contain the features SASA<sub>Total</sub> and ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub>, which is ∼20-fold enrichment with 5% false positives. With logistic regression the subset of (DOCK<sub>total</sub> + buried SASA<sub>total</sub>, + ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub>) has an enrichment factor of ∼16-fold. </div><div class="NLM_p"><b>Overall Performance.</b> For applications where eliminating decoys and discovering binders are of equal importance (for example, lead optimization and pose generation), the best overall performance is the closest that the ROC curve gets to the top, left corner; that is <a name="jm004802206828" id="jm004802206828"></a> when (precent true positives found) <i>−</i> (percent false positives allowed) is at its highest. The overall performance of ChemScore and DOCK alone are finding 37/44 true binders while allowing 18/44 decoys and finding 13/44 true binders while allowing 3/44 decoys, respectively. By contrast, the highest performance found in any subset is finding 38/44 true binders while only allowing 2/44 decoys. To estimate the effects stochastic elements of the random forest we grew 300 random forests. In these 300 the mean enrichment was 18.96 and<a data-tab="pane-pcw-Figures" href="#jm0493360f00010" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00011" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00012" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00013" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00014" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00015" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0493360f00016" id="" class=" internalNav"></a> 25th, 50th (median), and 75th percentiles were 18.5, 19, and 19.5, respectively. The best performance is (percentage of binders found) − (percentage of decoys allowed). The mean best performance is 83% and the 25th, 50th, 75th percentiles were 81.8, 81.8, 84.1%, respectively. <div id="_i20"><figure id="jm0493360f00010" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00010.jpeg" id="_i21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 The binding mode of KNI in HIV protease (1hpx) as found in the pdb. It has strong enough interactions to be correctly identified as a binder and falls in region 1 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00011" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00011.jpeg" id="_i22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 11 Thrombin (1uvu) with decoy (1okl_mns) ligand DOCKed in. It is correctly identified as a nonbinder and falls in region 3 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00011"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00012" data-index="12" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00012.jpeg" id="_i23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 12 This is thrombin with binder from the pdb. It is correctly identified as a binder and falls in region 4 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00012"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00013" data-index="13" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00013.jpeg" id="_i24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 13 The decoy pcp (1a96) docked into cytidine deaminase (1ctt). This decoy has enough buried surface area and H-bonding to fall in region 4, a majority binding region, and therefore is a false positive in our single tree, two-descriptor learner. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00013"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00014" data-index="14" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00014.jpeg" id="_i25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 14 This is native ligand (dhz) docked into cytidine deaminase (1ctt). It has enough interactions to be correctly considered a binder and can be found in region 1 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00014"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00015" data-index="15" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00015.jpeg" id="_i26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 15 The decoy ligand (phb) in carbonic anhydrase (1cnw). Because of its lack of interactions it falls in region 3 of Figure 8 and is correctly excluded as a decoy by a single tree learner. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00015"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0493360f00016" data-index="16" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00016.jpeg" id="_i27" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 16 The native ligand (eg1) in carbonic anhydrase (1cnw). This complex does not have enough interactions to be considered a binder. Therefore for this two-descriptor single tree this would be false negative. Metal binding interactions must be considered to properly evaluate this complex.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00016"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Decoys are Easier to Identify Than Binders.</b> Traditional scoring functions focus on correctly predicting how well a ligand will bind. Enrichment emphasizes eliminating decoys. Almost all the descriptor combinations we consider find eliminating decoys easier than finding binders. For example, the DOCK total force field score gets most of its enrichment from eliminating decoys (from Table 3:  DOCK<sub>total</sub> identifies 13 of 44 binders while eliminating 41 of 44 decoys), rather than positively identifying those that do bind. Indeed, for many descriptor subsets, the percentage of decoys correctly identified is roughly 25% higher than the percentage of binders correctly identified at the midpoint on the ROC curve (Figures 3 and 4). Another manifestation of this is that most of the overall performance maxima occurred with fewer false positives than false negatives. Logistic regression also finds decoys easier to detect than binders (Figures 5 and 6). </div><div class="NLM_p"><b>Logistic Regression.</b> We can connect this work with other empirical scoring functions by making a linear model of our descriptors. Visual inspection of Figure 8 suggests that a linear decision surface would be effective for this training set. We use logistic regression (LR) as our linear method.<sup>40</sup> The performance of the descriptors in this linear model is a little lower than that of the random forest model (see Table 4). Similar features and feature subsets are important for the two techniques. In particular, SASA<sub>total</sub>, ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub>, and DOCK<sub>total</sub> are part of the high performing descriptor subset, and once again the single descriptors still greatly underperform the feature combinations; thus, synergies between descriptors are still important. Note the case ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub> which is the best individual feature and is better on its own than the DOCK<sub>total</sub> and SASA<sub>total</sub> subset. Together they still form the best triplet and subset we found. </div><div class="NLM_p">The coefficients with their error bars for the LR models are shown in Table 5. For all descriptors, more negative values indicate stronger interactions:  those descriptors with positive coefficients are associated with improved binding. (DOCK<sub>total</sub> + SASA<sub>total</sub> + ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub>) is the best performing subset. LR performs better on descriptor subsets than all the features because unlike the ensemble of trees it is susceptible to noisy and/or low information features included in the entire set. </div><div class="NLM_p"><b>A Single Decision Tree Example.</b> One advantage of using subsets of descriptors is their ease of physical interpretation. To illustrate this point we show a simple example using only two descriptors. Figure 8 shows the recursive partitioning of the training set using the top performing descriptor pair:  SASA<sub>total</sub> and ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub>. Again for simplicity, Figure 8 shows results for a single decision tree instead of an ensemble of trees. Each partition is labeled in the sequential order in which it is derived. Partition 1 is the first partition dividing binders and nonbinders, and this partition (ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub> < ∼-7 Kcal) has a majority of binders. Partition 2 divides the remaining compounds, and again complexes scoring in this partition (ChemScore <sub>H-bond</sub> > ∼−7 kcal but SASA<sub>total</sub> < ∼−500 Å<sup>2</sup>) are considered binders. Partition 3 divides the remainder, and all compounds scoring in this partition (ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub> > ∼4.8 and SASA<sub>total</sub> > ∼−500 Å<sup>2</sup>) are considered nonbinders. Similarly, compounds scoring in partitions 4 and 5 are binders and nonbinders, respectively. It is physically reasonable that ligands that have the highest <a name="jm004802206829" id="jm004802206829"></a> number of hydrogen bonds and/or most buried surface area are partitioned into binding regions. </div><div class="NLM_p">To show how this simple descriptor set performs on particular examples, we plotted and labeled the compounds from the test set onto this partitioning scheme (see Figure 8). Inaccurate predictions occur when a decoy complex buries a large amount of surface area or makes many hydrogen bonds. Conversely, inaccurate predictions also occur when some true binders do not bury surface area or have enough hydrogen bonds in this simple partitioning to be labeled binders. Although this learner uses only two descriptors, it does a fairly good job of correctly partitioning the test set. With more descriptors, we can better distinguish binders from nonbinders, such as in our top-performing descriptor set (SASA-L<sub>carbon</sub> + SASA<sub>total</sub> + ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub>). </div><div class="NLM_p last">We consider four specific cases from the test set in the example from Figure 8. Two of them are treated correctly, and the other two either misclassify a decoy or a binder. In our first example, HIV-1 protease (PDB entry 1hpx), both the binder and decoy are properly scored. The crystallographic ligand has a large amount of buried surface area, placing it clearly in a binding <a name="jm004802206830" id="jm004802206830"></a> region. The HIV-1 protease decoy is clearly in the nonbinding partition 3, due to both the lower number of hydrogen bonds and the smaller amount of buried surface area (see Figures 9 and 10). Thrombin (PDB entry 1uvu) is also classified correctly. Again the thrombin PDB complex has much greater H-bonding and buried surface area relative to the decoy (see Figures 11 and 12). The PDB structure of carbonic anhydrase (PDB entry 1cnw) is incorrectly perceived to be a decoy. This is because the H-bonding and SASA terms miss the metal binding interaction that is critical for this target (see Figures 13 and 14). When metal binding is included, this complex is correctly recognized. The cytidine deaminase binder is correctly recognized as such, but the decoy has enough H-bonding that it is predicted to be a binder (see Figures 15 and 16). Inclusion of the DOCK total allows the correct identification of this decoy complex. Using many descriptors and complex pattern recognition methods achieves higher performance; having fairly good performance from a simple model achieves interpretability. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50226" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50226" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the virtual screening of large libraries, the vast majority of compounds are of little interest, and limited computation time can be spent on each compound. The task is to eliminate as many nonbinders as possible while retaining the binders. The approach we developed combines fast docking (frozen protein, no explicit solvent) with a pattern recognition algorithm to create a set of postDOCK filters that can be used to triage the implausible binding poses, leaving a greatly reduced set of compounds for further consideration. For almost all descriptor sets, decoys were much easier to detect than binders, and therefore high enrichments were possible without having high accuracy on binders. Existing scoring schemes (DOCK and ChemScore) eliminate many decoys (with 5-fold and 7-fold enrichments). Our best model (constructed using all of our descriptors with random forest learner) has a 19-fold enrichment (recovering 39 of 44 binders while allowing only 2 of 44 of the decoys). </div><div class="NLM_p">We used machine learning techniques to systematically improve performance by testing new descriptors in combination with existing descriptors. Our results clearly show that the DOCK force field could benefit from buried surface area and H-bonding terms. Further improvements in performance will come from better descriptors. One source of improvement could come from using partial charge calculation methods that are more accurate than the Gasteiger−Marsili procedure that we used and replacing the distance dependent dielectric with a better implicit solvation method. In this paper, we have tried large sphere SASA (which to our knowledge is a novel descriptor in characterizing protein−ligand complexes) in addition to the more traditional descriptors of protein-ligand complexes. Although it does not contribute to enrichment, the large sphere buried surface area descriptor may improve performance in the mid part of the ROC curve. This effect is not statistically significant and needs to be repeated before we can say that we are seeing aqueous self-association effects. </div><div class="NLM_p last">Because our examples were trained and tested on a diverse set of proteins, we expect the postDOCK filters we listed to be generally applicable to a variety of targets when there is no other a priori information. For cases where there is additional information, such as binding data to a homologous protein, the methods described in this paper could be used to train a dataset specific to the target of interest. Additional descriptors may be added to further tune the postDOCK filter to that target, such as adding metal binding descriptors when the target is a zinc binding protein complex (as in our carbonic anhydrase example). Similarly, this methodology can be applied to tune target specific postDOCK filters using binding assay data. </div></div><div class="NLM_back"><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm0493360/suppl_file/jm0493360_s.pdf">jm0493360_s.pdf (191.9 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm0493360" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34878" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34878" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clayton Springer</span> - <span class="hlFld-Affiliation affiliation">Sandia National Labs, P.O. Box 969, MS 9951, Livermore, California 94551 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helgi Adalsteinsson</span> - <span class="hlFld-Affiliation affiliation">Sandia National Labs, P.O. Box 969, MS 9951, Livermore, California 94551 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Malin M. Young</span> - <span class="hlFld-Affiliation affiliation">Sandia National Labs, P.O. Box 969, MS 9951, Livermore, California 94551 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip W. Kegelmeyer</span> - <span class="hlFld-Affiliation affiliation">Sandia National Labs, P.O. Box 969, MS 9951, Livermore, California 94551 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Diana C. Roe</span> - <span class="hlFld-Affiliation affiliation">Sandia National Labs, P.O. Box 969, MS 9951, Livermore, California 94551 </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">C.S. and H.A. acknowledge postdoctoral funding support from the Sandia National Labs LDRD program. </p></div><div class="refs-header-label"><a name="jm004802206831" id="jm004802206831"></a></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="jm0493360b00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm0493360b00001_1"><span><span class="NLM_string-ref">Orozco, M.; Tirado-Rives, J.; Jorgensen, W. L. Mechanism for the rotamase activity of FK506 binding protein from molecular dynamics simulations. <i>  Biochemistry </i><b>1993</b>,<i> 32</i>, 12864−12874.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1993&pages=12874&author=M.+Orozco&author=J.+Tirado-Rives&author=W.+L+Jorgensen&title=Mechanism+for+the+rotamase+activity+of+FK506+binding+protein+from+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DMechanism%2520for%2520the%2520rotamase%2520activity%2520of%2520FK506%2520binding%2520protein%2520from%2520molecular%2520dynamics%2520simulations%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D12874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm0493360b00002_1"><span><span class="NLM_string-ref">Sham, Y. Y.; Chu, Z. T.; Tao, H.; Warshel, A. Examining methods for calculations of binding free energies:  LRA; LIE; PDLD-LRA; and PDLD/S-LRA calculations of ligands binding to an HIV protease. <i>  Proteins-Struct. </i><i>Funct. Genet. </i><b>2000</b>,<i> 39</i>, 393−407.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=407&author=Y.+Y.+Sham&author=Z.+T.+Chu&author=H.+Tao&author=A+Warshel&title=Examining+methods+for+calculations+of+binding+free+energies%3A%E2%80%89+LRA%3B+LIE%3B+PDLD-LRA%3B+and+PDLD%2FS-LRA+calculations+of+ligands+binding+to+an+HIV+protease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSham%26aufirst%3DY.%2BY.%26atitle%3DExamining%2520methods%2520for%2520calculations%2520of%2520binding%2520free%2520energies%253A%25E2%2580%2589%2520LRA%253B%2520LIE%253B%2520PDLD-LRA%253B%2520and%2520PDLD%252FS-LRA%2520calculations%2520of%2520ligands%2520binding%2520to%2520an%2520HIV%2520protease%26jtitle%3DProteins-Struct.%2520Funct.%2520Genet.%26date%3D2000%26volume%3D39%26spage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm0493360b00003_1"><span><span class="NLM_string-ref">Zhang, L. Y.; Gallicchio, E.; Friesner, R. A.; Levy, R. M.; Guermeur, Y. et al. Solvent models for protein−ligand binding:  Comparison of implicit solvent Poisson and surface generalized born models with explicit solvent simulations. <i>  J. Comput. Chem. </i><b>2001</b>,<i> 22</i>, 591−607.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=607&author=L.+Y.+Zhang&author=E.+Gallicchio&author=R.+A.+Friesner&author=R.+M.+Levy&author=Y+Guermeur&title=et+al.+Solvent+models+for+protein%E2%88%92ligand+binding%3A%E2%80%89+Comparison+of+implicit+solvent+Poisson+and+surface+generalized+born+models+with+explicit+solvent+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%2BY.%26atitle%3Det%2520al.%2520Solvent%2520models%2520for%2520protein%25E2%2588%2592ligand%2520binding%253A%25E2%2580%2589%2520Comparison%2520of%2520implicit%2520solvent%2520Poisson%2520and%2520surface%2520generalized%2520born%2520models%2520with%2520explicit%2520solvent%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm0493360b00004_1"><span><span class="NLM_string-ref">Robinson, G. W.; Cho, C. H.; Urquidi, J. Isosbestic points in liquid water:  Further strong evidence for the two-state mixture model. <i>  J. Chem. Phys. </i><b>1999</b>,<i> 111</i>, 698−702.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1999&pages=702&author=G.+W.+Robinson&author=C.+H.+Cho&author=J+Urquidi&title=Isosbestic+points+in+liquid+water%3A%E2%80%89+Further+strong+evidence+for+the+two-state+mixture+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DG.%2BW.%26atitle%3DIsosbestic%2520points%2520in%2520liquid%2520water%253A%25E2%2580%2589%2520Further%2520strong%2520evidence%2520for%2520the%2520two-state%2520mixture%2520model%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1999%26volume%3D111%26spage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm0493360b00005_1"><span><span class="NLM_string-ref">BenNaim, A. A measure of the average cooperativity of a binding system. <i>  J. </i><i>Chem. Phys. </i><b>1998</b>,<i> 109</i>, 7443−7449.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=1998&pages=7449&author=A+BenNaim&title=A+measure+of+the+average+cooperativity+of+a+binding+system"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBenNaim%26aufirst%3DA%26atitle%3DA%2520measure%2520of%2520the%2520average%2520cooperativity%2520of%2520a%2520binding%2520system%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1998%26volume%3D109%26spage%3D7449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm0493360b00006_1"><span><span class="NLM_string-ref">Lum, K.; Chandler, D.; Weeks, J. D. Hydrophobicity at small and large length scales. <i>  J. Phys. Chem. B </i><b>1999</b>,<i> 103</i>, 4570−4577.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=4577&author=K.+Lum&author=D.+Chandler&author=J.+D+Weeks&title=Hydrophobicity+at+small+and+large+length+scales"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLum%26aufirst%3DK.%26atitle%3DHydrophobicity%2520at%2520small%2520and%2520large%2520length%2520scales%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D4577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm0493360b00007_1"><span><span class="NLM_string-ref">Wallqvist, A.; Covell, D. G. Cooperativity of Water-Solute Interactions at a Hydrophilic Surface. <i>  J. Phys. Chem. </i><b>1995</b>,<i> 99</i>, 5705−5712.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=5712&author=A.+Wallqvist&author=D.+G+Covell&title=Cooperativity+of+Water-Solute+Interactions+at+a+Hydrophilic+Surface"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DCooperativity%2520of%2520Water-Solute%2520Interactions%2520at%2520a%2520Hydrophilic%2520Surface%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D5712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm0493360b00008_1"><span><span class="NLM_string-ref">Zou, X. Q.; Sun, Y. X.; Kuntz, I. D. Inclusion of solvation in ligand binding free energy calculations using the generalized-born model. <i>  J. Am. Chem. Soc. </i><b>1999</b>,<i> 121</i>, 8033−8043.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=8043&author=X.+Q.+Zou&author=Y.+X.+Sun&author=I.+D+Kuntz&title=Inclusion+of+solvation+in+ligand+binding+free+energy+calculations+using+the+generalized-born+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DX.%2BQ.%26atitle%3DInclusion%2520of%2520solvation%2520in%2520ligand%2520binding%2520free%2520energy%2520calculations%2520using%2520the%2520generalized-born%2520model%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D8043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm0493360b00009_1"><span><span class="NLM_string-ref">Majeux, N.; Scarsi, M.; Caflisch, A. Efficient electrostatic solvation model for protein-fragment docking. <i>  Proteins-Struct. Funct. Genet. </i><b>2001</b>,<i> 42</i>, 256−268.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2001&pages=268&author=N.+Majeux&author=M.+Scarsi&author=A+Caflisch&title=Efficient+electrostatic+solvation+model+for+protein-fragment+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMajeux%26aufirst%3DN.%26atitle%3DEfficient%2520electrostatic%2520solvation%2520model%2520for%2520protein-fragment%2520docking%26jtitle%3DProteins-Struct.%2520Funct.%2520Genet.%26date%3D2001%26volume%3D42%26spage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00010"><div class="reference"><div class="NLM_citation" id="jm0493360b00010_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Leach, A. R.; Kuntz, I. D. Ligand solvation in molecular docking. <i>  Proteins-Struct. Funct. Genet. </i><b>1999</b>,<i> 34</i>, 4−16.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=16&author=B.+K.+Shoichet&author=A.+R.+Leach&author=I.+D+Kuntz&title=Ligand+solvation+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520solvation%2520in%2520molecular%2520docking%26jtitle%3DProteins-Struct.%2520Funct.%2520Genet.%26date%3D1999%26volume%3D34%26spage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm0493360b00011_1"><span><span class="NLM_string-ref">Meng, E. C.; Shoichet, B. K.; Kuntz, I. D. Automated docking with grid-based energy evaluation. <i>  J. </i><i>Comput.</i><i> Chem. </i><b>1992</b>,<i> 13</i>, 505−524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=524&author=E.+C.+Meng&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Automated+docking+with+grid-based+energy+evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26atitle%3DAutomated%2520docking%2520with%2520grid-based%2520energy%2520evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm0493360b00012_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Leach, A. R.; Kuntz, I. D. Ligand solvation in molecular docking. <i>  Proteins </i><b>1999</b>,<i> 34</i>, 4−16.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=16&author=B.+K.+Shoichet&author=A.+R.+Leach&author=I.+D+Kuntz&title=Ligand+solvation+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520solvation%2520in%2520molecular%2520docking%26jtitle%3DProteins%26date%3D1999%26volume%3D34%26spage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm0493360b00013_1"><span><span class="NLM_string-ref">Jones, G.; Willett, P.; Glen, R. C.; Leach, A. R.; Taylor, R. Development and validation of a genetic algorithm for flexible docking. <i>  J. Mol. Biol. </i><b>1997</b>, <i>267</i>, 727−748.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=748&author=G.+Jones&author=P.+Willett&author=R.+C.+Glen&author=A.+R.+Leach&author=R+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm0493360b00014_1"><span><span class="NLM_string-ref">Pang, Y. P.; Perola, E.; Xu, K.; Prendergast, F. G. EUDOC:  A computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases. <i>  J. Comput. Chem. </i><b>2001</b>,<i> 22</i>, 1750−1771.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=1771&author=Y.+P.+Pang&author=E.+Perola&author=K.+Xu&author=F.+G.+Prendergast&title=computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DY.%2BP.%26atitle%3Dcomputer%2520program%2520for%2520identification%2520of%2520drug%2520interaction%2520sites%2520in%2520macromolecules%2520and%2520drug%2520leads%2520from%2520chemical%2520databases%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D1771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm0493360b00015_1"><span><span class="NLM_string-ref">Muegge, I.; Martin, Y. A general and fast scoring function for protein−ligand interactions:  A simplified potential approach. <i>  J. Med. Chem. </i><b>1999</b>,<i> 42</i>, 791−804.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=804&author=I.+Muegge&author=Y+Martin&title=A+general+and+fast+scoring+function+for+protein%E2%88%92ligand+interactions%3A%E2%80%89+A+simplified+potential+approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26atitle%3DA%2520general%2520and%2520fast%2520scoring%2520function%2520for%2520protein%25E2%2588%2592ligand%2520interactions%253A%25E2%2580%2589%2520A%2520simplified%2520potential%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm0493360b00016_1"><span><span class="NLM_string-ref">Gohlke, H.; Hendlich, M.; Klebe, G. Knowledge-based scoring function to predict protein−ligand interactions. <i>  J. Mol. Biol. </i><b>2000</b>,<i> 295</i>, 337−356.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2000&pages=356&author=H.+Gohlke&author=M.+Hendlich&author=G+Klebe&title=Knowledge-based+scoring+function+to+predict+protein%E2%88%92ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DKnowledge-based%2520scoring%2520function%2520to%2520predict%2520protein%25E2%2588%2592ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D295%26spage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm0493360b00017_1"><span><span class="NLM_string-ref">Bohm, H. J. The development of a simple empirical scoring function to estimate the binding constant for a protein−ligand complex of known three-dimensional structure. <b>1994</b>,<i> 8</i>, 243−256.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=H.+J.+Bohm&title=The+development+of+a+simple+empirical+scoring+function+to+estimate+the+binding+constant+for+a+protein%E2%88%92ligand+complex+of+known+three-dimensional+structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBohm%26aufirst%3DH.%2BJ.%26btitle%3DThe%2520development%2520of%2520a%2520simple%2520empirical%2520scoring%2520function%2520to%2520estimate%2520the%2520binding%2520constant%2520for%2520a%2520protein%25E2%2588%2592ligand%2520complex%2520of%2520known%2520three-dimensional%2520structure%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm0493360b00018_1"><span><span class="NLM_string-ref">Eldridge, M. D.; Murray, C. W.; Auton, T. R.; Paolini, G. V.; Mee, R. P. Empirical scoring functions 0.1. The development of a fast empirical scoring function to estimate the binding affinity of ligands in receptor complexes. <i>  J. Comput.-Aided Mol. Des. </i><b>1997</b>,<i> 11</i>, 425−445.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=445&author=M.+D.+Eldridge&author=C.+W.+Murray&author=T.+R.+Auton&author=G.+V.+Paolini&author=R.+P+Mee&title=Empirical+scoring+functions+0.1.+The+development+of+a+fast+empirical+scoring+function+to+estimate+the+binding+affinity+of+ligands+in+receptor+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEldridge%26aufirst%3DM.%2BD.%26atitle%3DEmpirical%2520scoring%2520functions%25200.1.%2520The%2520development%2520of%2520a%2520fast%2520empirical%2520scoring%2520function%2520to%2520estimate%2520the%2520binding%2520affinity%2520of%2520ligands%2520in%2520receptor%2520complexes%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm0493360b00019_1"><span><span class="NLM_string-ref">Charifson, P. S.; Corkery, J. J.; Murcko, M. A.; Walters, W. P. Consensus scoring:  A method for obtaining improved hit rates from docking databases of three-dimensional structures into proteins. <i>  J. Med. Chem. </i><b>1999</b>,<i> 42</i>, 5100−5109.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5109&author=P.+S.+Charifson&author=J.+J.+Corkery&author=M.+A.+Murcko&author=W.+P.+Walters&author=A+Consensus&title=method+for+obtaining+improved+hit+rates+from+docking+databases+of+three-dimensional+structures+into+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3Dmethod%2520for%2520obtaining%2520improved%2520hit%2520rates%2520from%2520docking%2520databases%2520of%2520three-dimensional%2520structures%2520into%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00020"><div class="reference"><div class="NLM_citation" id="jm0493360b00020_1"><span><span class="NLM_string-ref">Bissantz, C.; Folkers, G.; Rognan, D. Protein-Based Virtual Screening of Chemical Databases. 1. Evaluation of Different Docking/Scoring Combinations. <i>  J. Med. Chem. </i><b>2000</b>,<i> 43</i>, 4759−4767.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4767&author=C.+Bissantz&author=G.+Folkers&author=D+Rognan&title=Protein-Based+Virtual+Screening+of+Chemical+Databases.+1.+Evaluation+of+Different+Docking%2FScoring+Combinations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26atitle%3DProtein-Based%2520Virtual%2520Screening%2520of%2520Chemical%2520Databases.%25201.%2520Evaluation%2520of%2520Different%2520Docking%252FScoring%2520Combinations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm0493360b00021_1"><span><span class="NLM_string-ref">Breiman, L. Random Forests. <i>  Machine Learning</i><b>2001</b>, <i>45</i>, 5−32.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Breiman%2C+L.+Random+Forests.+Machine+Learning+2001%2C+45%2C+5%E2%88%9232.+Breiman%2C+L.+Random+Forests.+Machine+Learning2001%2C+45%2C+5%E2%88%9232."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26jtitle%3DMachine%2520Learning%26date%3D2001%26volume%3D45%26spage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm0493360b00022_1"><span><span class="NLM_string-ref">Rusinko, A.; MW, F.; Lambert, C.; Brown, P.; Young, S. Analysis of a large structure/biological activity data set using rescursive partitioning. <i>  J. Chem. </i><i>Inf. Comput. Sci. </i><b>1999</b>,<i> 39</i>, 1017−1026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=1026&author=A.+Rusinko&author=C.+Lambert&author=P.+Brown&author=S+Young&title=Analysis+of+a+large+structure%2Fbiological+activity+data+set+using+rescursive+partitioning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRusinko%26aufirst%3DA.%26atitle%3DAnalysis%2520of%2520a%2520large%2520structure%252Fbiological%2520activity%2520data%2520set%2520using%2520rescursive%2520partitioning%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm0493360b00023_1"><span><span class="NLM_string-ref">Holm, L.; Sander, C. The FSSP Database of Structurally Aligned Protein Fold Families. <i>  Nucleic Acids Res. </i><b>1994</b>,<i> 22</i>, 3600−3609.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=3609&author=L.+Holm&author=C+Sander&title=The+FSSP+Database+of+Structurally+Aligned+Protein+Fold+Families"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26atitle%3DThe%2520FSSP%2520Database%2520of%2520Structurally%2520Aligned%2520Protein%2520Fold%2520Families%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26spage%3D3609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm0493360b00024_1"><span><span class="NLM_string-ref">Holm, L.; Sander, C. Dali/FSSP classification of three-dimensional protein folds. <i>  Nucleic Acids Res. </i><b>1997</b>,<i> 25</i>, 231−234.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=234&author=L.+Holm&author=C.+Sander&author=FSSP+Dali&title=classification+of+three-dimensional+protein+folds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26atitle%3Dclassification%2520of%2520three-dimensional%2520protein%2520folds%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm0493360b00025_1"><span><span class="NLM_string-ref">Ewing, T. J. A.; Makino, S.; Skillman, A. G.; Kuntz, I. D. DOCK 4.0:  Search strategies for automated molecular docking of flexible molecule databases. <i>  J. Comput.-Aided Mol. Des. </i><b>2001</b>,<i> 15</i>, 411−428.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=428&author=T.+J.+A.+Ewing&author=S.+Makino&author=A.+G.+Skillman&author=I.+D.+Kuntz&title=Search+strategies+for+automated+molecular+docking+of+flexible+molecule+databases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEwing%26aufirst%3DT.%2BJ.%2BA.%26atitle%3DSearch%2520strategies%2520for%2520automated%2520molecular%2520docking%2520of%2520flexible%2520molecule%2520databases%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm0493360b00026_1"><span><span class="NLM_string-ref">Meng, E. C.; Shoichet, B. K.; Kuntz, I. D. Automated Docking with Grid-Based Energy Evaluation. <i>  J. Comput. Chem. </i><b>1992</b>,<i> 13</i>, 505−524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=524&author=E.+C.+Meng&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Automated+Docking+with+Grid-Based+Energy+Evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26atitle%3DAutomated%2520Docking%2520with%2520Grid-Based%2520Energy%2520Evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00027"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00027'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="jm0493360b00027_1"><span><span class="NLM_string-ref">Weiner, S. J.; Kollman, P. A.; Case, D. A.; Singh, U. C.; Ghio, C. et al. A New Force Field for Molecular Mechanical Simulation of Nucleic Acids and Proteins. <i>  J. Am. Chem. Soc. </i><b>1984</b>,<i> 106</i>, 765−784.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1984&pages=784&author=S.+J.+Weiner&author=P.+A.+Kollman&author=D.+A.+Case&author=U.+C.+Singh&author=C+Ghio&title=et+al.+A+New+Force+Field+for+Molecular+Mechanical+Simulation+of+Nucleic+Acids+and+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWeiner%26aufirst%3DS.%2BJ.%26atitle%3Det%2520al.%2520A%2520New%2520Force%2520Field%2520for%2520Molecular%2520Mechanical%2520Simulation%2520of%2520Nucleic%2520Acids%2520and%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1984%26volume%3D106%26spage%3D784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00028"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00028'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="jm0493360b00028_1"><span><span class="NLM_string-ref">Gasteiger, J.; Marsili, M. <i>  Organ. Magn. Reson. </i><b>1981</b>,<i> 15</i>, 353−360.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Gasteiger%2C+J.%3B+Marsili%2C+M.+Organ.+Magn.+Reson.+1981%2C+15%2C+353%E2%88%92360.+Gasteiger%2C+J.%3B+Marsili%2C+M.+Organ.+Magn.+Reson.+1981%2C+15%2C+353%E2%88%92360."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26jtitle%3DOrgan.%2520Magn.%2520Reson.%26date%3D1981%26volume%3D15%26spage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00029"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00029'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="jm0493360b00029_1"><span><span class="NLM_string-ref">Gasteiger, J.; Marsili, M. <i>  Tetrahedron </i><b>1980</b>,<i> 36</i>, 3219−3228.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Gasteiger%2C+J.%3B+Marsili%2C+M.+Tetrahedron+1980%2C+36%2C+3219%E2%88%923228.+Gasteiger%2C+J.%3B+Marsili%2C+M.+Tetrahedron+1980%2C+36%2C+3219%E2%88%923228."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00030"><div class="reference"><div class="NLM_citation" id="jm0493360b00030_1"><span><span class="NLM_string-ref">Marsili, M.; Gasteiger, J. <i>  Croat. Chem. Acta </i><b>1980</b>,<i> 53</i>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Marsili%2C+M.%3B+Gasteiger%2C+J.+Croat.+Chem.+Acta+1980%2C+53.+Marsili%2C+M.%3B+Gasteiger%2C+J.+Croat.+Chem.+Acta+1980%2C+53."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarsili%26aufirst%3DM.%26jtitle%3DCroat.%2520Chem.%2520Acta%26date%3D1980%26spage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00031"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00031'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="jm0493360b00031_1"><span><span class="NLM_string-ref">Willett, P. <i>  Similarity and Clustering in Chemical Information Space</i>; Research Studies Press:  Letchworth.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Willett%2C+P.+Similarity+and+Clustering+in+Chemical+Information+Space%3B+Research+Studies+Press%3A%E2%80%89+Letchworth.+Willett%2C+P.+Similarity+and+Clustering+in+Chemical+Information+Space%3B+Research+Studies+Press%3A%E2%80%89+Letchworth."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26btitle%3DSimilarity%2520and%2520Clustering%2520in%2520Chemical%2520Information%2520Space%26pub%3DResearch%2520Studies%2520Press" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00032"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00032'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="jm0493360b00032_1"><span><span class="NLM_string-ref">Martin, Y. C.; Kofron, J. L.; Traphagen, L. M. Do Structurally Similar Molecular Have Similar Biological Activity. <i>  J. Med. Chem. </i><b>2002</b>,<i> 45</i>, 4350−4358.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4358&author=Y.+C.+Martin&author=J.+L.+Kofron&author=L.+M+Traphagen&title=Do+Structurally+Similar+Molecular+Have+Similar+Biological+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DDo%2520Structurally%2520Similar%2520Molecular%2520Have%2520Similar%2520Biological%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00033"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00033'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="jm0493360b00033_1"><span><span class="NLM_string-ref">Leo, A. <i>  Programs CLogP and CMR</i>; BioByte Corp.:  Claremont, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Leo%2C+A.+Programs+CLogP+and+CMR%3B+BioByte+Corp.%3A%E2%80%89+Claremont%2C+CA.+Leo%2C+A.+Programs+CLogP+and+CMR%3B+BioByte+Corp.%3A%E2%80%89+Claremont%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLeo%26aufirst%3DA.%26btitle%3DPrograms%2520CLogP%2520and%2520CMR%26pub%3DCA" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00034"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00034'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="jm0493360b00034_1"><span><span class="NLM_string-ref">Varshney, A.; Brooks, F. J. Fast Analytical Computation of Richard's Smooth Molecular Surface; UNC at Chapel Hill:  Chapel Hill, NC, 1993.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=A.+Varshney&author=F.+J.+Brooks&title=Fast+Analytical+Computation+of+Richard%27s+Smooth+Molecular+Surface"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVarshney%26aufirst%3DA.%26btitle%3DFast%2520Analytical%2520Computation%2520of%2520Richard%2527s%2520Smooth%2520Molecular%2520Surface%26pub%3DNC%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00035"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00035'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="jm0493360b00035_1"><span><span class="NLM_string-ref">Eisenberg, D.; McLachlan, A. Solvation Energy in Protein Folding and Binding. <i>  Nature </i><b>1986</b>,<i> 319</i>, 199−203.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=1986&pages=203&author=D.+Eisenberg&author=A+McLachlan&title=Solvation+Energy+in+Protein+Folding+and+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DSolvation%2520Energy%2520in%2520Protein%2520Folding%2520and%2520Binding%26jtitle%3DNature%26date%3D1986%26volume%3D319%26spage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00036"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00036'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="jm0493360b00036_1"><span><span class="NLM_string-ref">Trunk, G. V. A problem of Dimensionality:  A Simple Example. <i>  IEEE Trans. </i><i>Pattern Anal. Machine Intell. </i><b>1979</b>,<i> 1</i>, 306−307.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1979&pages=307&author=G.+V+Trunk&title=A+problem+of+Dimensionality%3A%E2%80%89+A+Simple+Example"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrunk%26aufirst%3DG.%2BV%26atitle%3DA%2520problem%2520of%2520Dimensionality%253A%25E2%2580%2589%2520A%2520Simple%2520Example%26jtitle%3DIEEE%2520Trans.%2520Pattern%2520Anal.%2520Machine%2520Intell.%26date%3D1979%26volume%3D1%26spage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00037"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00037'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="jm0493360b00037_1"><span><span class="NLM_string-ref">Brunzell, H.; Eriksson, J. Feature reduction of classification of multidimensional data. <i>  Pattern Recog. </i><b>2000</b>,<i> 33</i>, 1741−1748.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=1748&author=H.+Brunzell&author=J+Eriksson&title=Feature+reduction+of+classification+of+multidimensional+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrunzell%26aufirst%3DH.%26atitle%3DFeature%2520reduction%2520of%2520classification%2520of%2520multidimensional%2520data%26jtitle%3DPattern%2520Recog.%26date%3D2000%26volume%3D33%26spage%3D1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00038"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00038'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="jm0493360b00038_1"><span><span class="NLM_string-ref">Cover, T. M.; Van Campenhout, J. M. On the possible ordering in the measurement selection problem. <i>  IEEE Trans. Syst., Man, Cybernetics </i><b>1977</b>,<i> 7</i>, 657−661.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1977&pages=661&author=T.+M.+Cover&author=J.+M+Van+Campenhout&title=On+the+possible+ordering+in+the+measurement+selection+problem"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCover%26aufirst%3DT.%2BM.%26atitle%3DOn%2520the%2520possible%2520ordering%2520in%2520the%2520measurement%2520selection%2520problem%26jtitle%3DIEEE%2520Trans.%2520Syst.%252C%2520Man%252C%2520Cybernetics%26date%3D1977%26volume%3D7%26spage%3D661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00039"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00039'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="jm0493360b00039_1"><span><span class="NLM_string-ref">Venables, W. N.; Ripley, B. D. <i>  Modern Applied Statistics with S-plus</i>; Springer:  New York, 1994.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=W.+N.+Venables&author=B.+D.+Ripley&title=Modern+Applied+Statistics+with+S-plus"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVenables%26aufirst%3DW.%2BN.%26btitle%3DModern%2520Applied%2520Statistics%2520with%2520S-plus%26pub%3DSpringer%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00040"><div class="reference"><div class="NLM_citation" id="jm0493360b00040_1"><span><span class="NLM_string-ref">McCullagh, P.; J. A., N. <i>  Generalized Linear Models</i>; Chapman and Hall:  London, 1989.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm0493360b00040/jm0493360b00040_1&amp;dbid=16&amp;doi=10.1021%2Fjm0493360&amp;key=10.1007%2F978-1-4899-3242-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&author=P.+McCullagh&title=Generalized+Linear+Models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00040_1&amp;dbid=16384&amp;doi=10.1007%2F978-1-4899-3242-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4899-3242-6%26sid%3Dliteratum%253Aachs%26aulast%3DMcCullagh%26aufirst%3DP.%26btitle%3DGeneralized%2520Linear%2520Models%26pub%3DChapman%2520and%2520Hall%26date%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00041"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00041'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="jm0493360b00041_1"><span><span class="NLM_string-ref">CRAN. The Comprehensive R Archive Network.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CRAN.+The+Comprehensive+R+Archive+Network.+CRAN.+The+Comprehensive+R+Archive+Network."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0493360b00042"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0493360b00042'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="jm0493360b00042_1"><span><span class="NLM_string-ref">Duda, R.; Hart; Stark, P. <i>  Pattern Classification</i>; John Wiley and Sons:  New York, 2001.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=R.+Duda&author=+Hart&title=Pattern+Classification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDuda%26aufirst%3DR.%26btitle%3DPattern%2520Classification%26pub%3DJohn%2520Wiley%2520and%2520Sons%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm0493360b00001':['jm0493360b00001_1'],'jm0493360b00002':['jm0493360b00002_1'],'jm0493360b00003':['jm0493360b00003_1'],'jm0493360b00004':['jm0493360b00004_1'],'jm0493360b00005':['jm0493360b00005_1'],'jm0493360b00006':['jm0493360b00006_1'],'jm0493360b00007':['jm0493360b00007_1'],'jm0493360b00008':['jm0493360b00008_1'],'jm0493360b00009':['jm0493360b00009_1'],'jm0493360b00010':['jm0493360b00010_1'],'jm0493360b00011':['jm0493360b00011_1'],'jm0493360b00012':['jm0493360b00012_1'],'jm0493360b00013':['jm0493360b00013_1'],'jm0493360b00014':['jm0493360b00014_1'],'jm0493360b00015':['jm0493360b00015_1'],'jm0493360b00016':['jm0493360b00016_1'],'jm0493360b00017':['jm0493360b00017_1'],'jm0493360b00018':['jm0493360b00018_1'],'jm0493360b00019':['jm0493360b00019_1'],'jm0493360b00020':['jm0493360b00020_1'],'jm0493360b00021':['jm0493360b00021_1'],'jm0493360b00022':['jm0493360b00022_1'],'jm0493360b00023':['jm0493360b00023_1'],'jm0493360b00024':['jm0493360b00024_1'],'jm0493360b00025':['jm0493360b00025_1'],'jm0493360b00026':['jm0493360b00026_1'],'jm0493360b00027':['jm0493360b00027_1'],'jm0493360b00028':['jm0493360b00028_1'],'jm0493360b00029':['jm0493360b00029_1'],'jm0493360b00030':['jm0493360b00030_1'],'jm0493360b00031':['jm0493360b00031_1'],'jm0493360b00032':['jm0493360b00032_1'],'jm0493360b00033':['jm0493360b00033_1'],'jm0493360b00034':['jm0493360b00034_1'],'jm0493360b00035':['jm0493360b00035_1'],'jm0493360b00036':['jm0493360b00036_1'],'jm0493360b00037':['jm0493360b00037_1'],'jm0493360b00038':['jm0493360b00038_1'],'jm0493360b00039':['jm0493360b00039_1'],'jm0493360b00040':['jm0493360b00040_1'],'jm0493360b00041':['jm0493360b00041_1'],'jm0493360b00042':['jm0493360b00042_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 26 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuna  Yan</span>, <span class="hlFld-ContribAuthor ">Weijun  Wang</span>, <span class="hlFld-ContribAuthor ">Zhaoxi  Sun</span>, <span class="hlFld-ContribAuthor ">John Z. H.  Zhang</span>, and <span class="hlFld-ContribAuthor ">Changge  Ji</span>  . </span><span class="cited-content_cbyCitation_article-title">Protein–Ligand Empirical Interaction Components for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (8)
                                     , 1793-1806. <a href="https://doi.org/10.1021/acs.jcim.7b00017" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00017%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProtein%2525E2%252580%252593Ligand%252BEmpirical%252BInteraction%252BComponents%252Bfor%252BVirtual%252BScreening%26aulast%3DYan%26aufirst%3DYuna%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D10012017%26date%3D18072017%26date%3D28082017%26date%3D05072017%26volume%3D57%26issue%3D8%26spage%3D1793%26epage%3D1806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Ramsbeck</span>, <span class="hlFld-ContribAuthor ">Mirko  Buchholz</span>, <span class="hlFld-ContribAuthor ">Birgit  Koch</span>, <span class="hlFld-ContribAuthor ">Livia  Böhme</span>, <span class="hlFld-ContribAuthor ">Torsten  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Hans-Ulrich  Demuth</span>, and <span class="hlFld-ContribAuthor ">Ulrich  Heiser</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationships of Benzimidazole-Based Glutaminyl Cyclase Inhibitors Featuring a Heteroaryl Scaffold. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (17)
                                     , 6613-6625. <a href="https://doi.org/10.1021/jm4001709" title="DOI URL">https://doi.org/10.1021/jm4001709</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm4001709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm4001709%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationships%252Bof%252BBenzimidazole-Based%252BGlutaminyl%252BCyclase%252BInhibitors%252BFeaturing%252Ba%252BHeteroaryl%252BScaffold%26aulast%3DRamsbeck%26aufirst%3DDaniel%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D01022013%26date%3D19082013%26date%3D12092013%26date%3D25072013%26volume%3D56%26issue%3D17%26spage%3D6613%26epage%3D6625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Fan</span>, <span class="hlFld-ContribAuthor ">Dina  Schneidman-Duhovny</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Guangqiang  Dong</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, and <span class="hlFld-ContribAuthor ">Andrej  Sali</span>  . </span><span class="cited-content_cbyCitation_article-title">Statistical Potential for Modeling and Ranking of Protein–Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (12)
                                     , 3078-3092. <a href="https://doi.org/10.1021/ci200377u" title="DOI URL">https://doi.org/10.1021/ci200377u</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200377u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200377u%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStatistical%252BPotential%252Bfor%252BModeling%252Band%252BRanking%252Bof%252BProtein%2525E2%252580%252593Ligand%252BInteractions%26aulast%3DFan%26aufirst%3DHao%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D08082011%26date%3D21112011%26date%3D27122011%26date%3D20102011%26volume%3D51%26issue%3D12%26spage%3D3078%26epage%3D3092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah L.  Kinnings</span>, <span class="hlFld-ContribAuthor ">Nina  Liu</span>, <span class="hlFld-ContribAuthor ">Peter J.  Tonge</span>, <span class="hlFld-ContribAuthor ">Richard M.  Jackson</span>, <span class="hlFld-ContribAuthor ">Lei  Xie</span>, and <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>  . </span><span class="cited-content_cbyCitation_article-title">A Machine Learning-Based Method To Improve Docking Scoring Functions and Its Application to Drug Repurposing. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (2)
                                     , 408-419. <a href="https://doi.org/10.1021/ci100369f" title="DOI URL">https://doi.org/10.1021/ci100369f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100369f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100369f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DA%252BMachine%252BLearning-Based%252BMethod%252BTo%252BImprove%252BDocking%252BScoring%252BFunctions%252Band%252BIts%252BApplication%252Bto%252BDrug%252BRepurposing%26aulast%3DKinnings%26aufirst%3DSarah%2BL.%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D17092010%26date%3D03022011%26date%3D28022011%26volume%3D51%26issue%3D2%26spage%3D408%26epage%3D419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dengguo  Wei</span>, <span class="hlFld-ContribAuthor ">Hao  Zheng</span>, <span class="hlFld-ContribAuthor ">Naifang  Su</span>, <span class="hlFld-ContribAuthor ">Minghua  Deng</span>, and <span class="hlFld-ContribAuthor ">Luhua  Lai</span>  . </span><span class="cited-content_cbyCitation_article-title">Binding Energy Landscape Analysis Helps to Discriminate True Hits from High-Scoring Decoys in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (10)
                                     , 1855-1864. <a href="https://doi.org/10.1021/ci900463u" title="DOI URL">https://doi.org/10.1021/ci900463u</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900463u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900463u%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBinding%252BEnergy%252BLandscape%252BAnalysis%252BHelps%252Bto%252BDiscriminate%252BTrue%252BHits%252Bfrom%252BHigh-Scoring%252BDecoys%252Bin%252BVirtual%252BScreening%26aulast%3DWei%26aufirst%3DDengguo%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D26%252011%25202009%26date%3D27092010%26date%3D25102010%26volume%3D50%26issue%3D10%26spage%3D1855%26epage%3D1864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Teodora  Djikic</span>, <span class="hlFld-ContribAuthor ">Zarko  Gagic</span>, <span class="hlFld-ContribAuthor ">Katarina  Nikolic</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Discovery of Kinase Inhibitors Using Docking Studies. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 337-365. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00009-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00009-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00009-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00009-6%26sid%3Dliteratum%253Aachs%26atitle%3DDesign%252Band%252BDiscovery%252Bof%252BKinase%252BInhibitors%252BUsing%252BDocking%252BStudies%26aulast%3DDjikic%26aufirst%3DTeodora%26date%3D2021%26spage%3D337%26epage%3D365%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jin  Li</span>, <span class="hlFld-ContribAuthor ">WeiChao  Liu</span>, <span class="hlFld-ContribAuthor ">Yongping  Song</span>, <span class="hlFld-ContribAuthor ">JiYi  Xia</span>. </span><span class="cited-content_cbyCitation_article-title">Improved method of structure-based virtual screening based on ensemble learning. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (13)
                                     , 7609-7618. <a href="https://doi.org/10.1039/C9RA09211K" title="DOI URL">https://doi.org/10.1039/C9RA09211K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA09211K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA09211K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DImproved%252Bmethod%252Bof%252Bstructure-based%252Bvirtual%252Bscreening%252Bbased%252Bon%252Bensemble%252Blearning%26aulast%3DLi%26aufirst%3DJin%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D13%26spage%3D7609%26epage%3D7618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro H. M.  Torres</span>, <span class="hlFld-ContribAuthor ">Ana C. R.  Sodero</span>, <span class="hlFld-ContribAuthor ">Paula  Jofily</span>, <span class="hlFld-ContribAuthor ">Floriano P.  Silva-Jr</span>. </span><span class="cited-content_cbyCitation_article-title">Key Topics in Molecular Docking for Drug Design. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (18)
                                     , 4574. <a href="https://doi.org/10.3390/ijms20184574" title="DOI URL">https://doi.org/10.3390/ijms20184574</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20184574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20184574%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DKey%252BTopics%252Bin%252BMolecular%252BDocking%252Bfor%252BDrug%252BDesign%26aulast%3DTorres%26aufirst%3DPedro%2BH.%2BM.%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D18%26spage%3D4574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olga  Tarasova</span>, <span class="hlFld-ContribAuthor ">Vladimir  Poroikov</span>, <span class="hlFld-ContribAuthor ">Alexander  Veselovsky</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking Studies of HIV-1 Resistance to Reverse Transcriptase Inhibitors: Mini-Review. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (5)
                                     , 1233. <a href="https://doi.org/10.3390/molecules23051233" title="DOI URL">https://doi.org/10.3390/molecules23051233</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23051233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23051233%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMolecular%252BDocking%252BStudies%252Bof%252BHIV-1%252BResistance%252Bto%252BReverse%252BTranscriptase%252BInhibitors%25253A%252BMini-Review%26aulast%3DTarasova%26aufirst%3DOlga%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D5%26spage%3D1233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qurrat Ul  Ain</span>, <span class="hlFld-ContribAuthor ">Antoniya  Aleksandrova</span>, <span class="hlFld-ContribAuthor ">Florian D.  Roessler</span>, <span class="hlFld-ContribAuthor ">Pedro J.  Ballester</span>. </span><span class="cited-content_cbyCitation_article-title">Machine-learning scoring functions to improve structure-based binding affinity prediction and virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: Computational Molecular Science</span><span> <strong>2015,</strong> <em>5 </em>
                                    (6)
                                     , 405-424. <a href="https://doi.org/10.1002/wcms.1225" title="DOI URL">https://doi.org/10.1002/wcms.1225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1225%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520Computational%2520Molecular%2520Science%26atitle%3DMachine-learning%252Bscoring%252Bfunctions%252Bto%252Bimprove%252Bstructure-based%252Bbinding%252Baffinity%252Bprediction%252Band%252Bvirtual%252Bscreening%26aulast%3DAin%26aufirst%3DQurrat%2BUl%26date%3D2015%26date%3D2015%26volume%3D5%26issue%3D6%26spage%3D405%26epage%3D424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  Lavecchia</span>. </span><span class="cited-content_cbyCitation_article-title">Machine-learning approaches in drug discovery: methods and applications. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2015,</strong> <em>20 </em>
                                    (3)
                                     , 318-331. <a href="https://doi.org/10.1016/j.drudis.2014.10.012" title="DOI URL">https://doi.org/10.1016/j.drudis.2014.10.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2014.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2014.10.012%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DMachine-learning%252Bapproaches%252Bin%252Bdrug%252Bdiscovery%25253A%252Bmethods%252Band%252Bapplications%26aulast%3DLavecchia%26aufirst%3DAntonio%26date%3D2015%26volume%3D20%26issue%3D3%26spage%3D318%26epage%3D331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongjian  Li</span>, <span class="hlFld-ContribAuthor ">Kwong-Sak  Leung</span>, <span class="hlFld-ContribAuthor ">Man-Hon  Wong</span>, <span class="hlFld-ContribAuthor ">Pedro J.  Ballester</span>. </span><span class="cited-content_cbyCitation_article-title">Improving AutoDock Vina Using Random Forest: The Growing Accuracy of Binding Affinity Prediction by the Effective Exploitation of Larger Data Sets. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2015,</strong> <em>34 </em>
                                    (2-3)
                                     , 115-126. <a href="https://doi.org/10.1002/minf.201400132" title="DOI URL">https://doi.org/10.1002/minf.201400132</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201400132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201400132%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DImproving%252BAutoDock%252BVina%252BUsing%252BRandom%252BForest%25253A%252BThe%252BGrowing%252BAccuracy%252Bof%252BBinding%252BAffinity%252BPrediction%252Bby%252Bthe%252BEffective%252BExploitation%252Bof%252BLarger%252BData%252BSets%26aulast%3DLi%26aufirst%3DHongjian%26date%3D2015%26date%3D2015%26volume%3D34%26issue%3D2-3%26spage%3D115%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John B. O.  Mitchell</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning methods in chemoinformatics. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2014,</strong> <em>4 </em>
                                    (5)
                                     , 468-481. <a href="https://doi.org/10.1002/wcms.1183" title="DOI URL">https://doi.org/10.1002/wcms.1183</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1183%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DMachine%252Blearning%252Bmethods%252Bin%252Bchemoinformatics%26aulast%3DMitchell%26aufirst%3DJohn%2BB.%2BO.%26date%3D2014%26date%3D2014%26volume%3D4%26issue%3D5%26spage%3D468%26epage%3D481" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">W. G.  Touw</span>, <span class="hlFld-ContribAuthor ">J. R.  Bayjanov</span>, <span class="hlFld-ContribAuthor ">L.  Overmars</span>, <span class="hlFld-ContribAuthor ">L.  Backus</span>, <span class="hlFld-ContribAuthor ">J.  Boekhorst</span>, <span class="hlFld-ContribAuthor ">M.  Wels</span>, <span class="hlFld-ContribAuthor ">S. A. F. T.  van Hijum</span>. </span><span class="cited-content_cbyCitation_article-title">Data mining in the Life Sciences with Random Forest: a walk in the park or lost in the jungle?. </span><span class="cited-content_cbyCitation_journal-name">Briefings in Bioinformatics</span><span> <strong>2013,</strong> <em>14 </em>
                                    (3)
                                     , 315-326. <a href="https://doi.org/10.1093/bib/bbs034" title="DOI URL">https://doi.org/10.1093/bib/bbs034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bib/bbs034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbib%2Fbbs034%26sid%3Dliteratum%253Aachs%26jtitle%3DBriefings%2520in%2520Bioinformatics%26atitle%3DData%252Bmining%252Bin%252Bthe%252BLife%252BSciences%252Bwith%252BRandom%252BForest%25253A%252Ba%252Bwalk%252Bin%252Bthe%252Bpark%252Bor%252Blost%252Bin%252Bthe%252Bjungle%25253F%26aulast%3DTouw%26aufirst%3DW.%2BG.%26date%3D2013%26date%3D2012%26volume%3D14%26issue%3D3%26spage%3D315%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eunkyoung  Jung</span>, <span class="hlFld-ContribAuthor ">Seung-Hoon  Choi</span>, <span class="hlFld-ContribAuthor ">Nam Kyung  Lee</span>, <span class="hlFld-ContribAuthor ">Sang-Kee  Kang</span>, <span class="hlFld-ContribAuthor ">Yun-Jaie  Choi</span>, <span class="hlFld-ContribAuthor ">Jae-Min  Shin</span>, <span class="hlFld-ContribAuthor ">Kihang  Choi</span>, <span class="hlFld-ContribAuthor ">Dong Hyun  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning study for the prediction of transdermal peptide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2011,</strong> <em>25 </em>
                                    (4)
                                     , 339-347. <a href="https://doi.org/10.1007/s10822-011-9424-2" title="DOI URL">https://doi.org/10.1007/s10822-011-9424-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-011-9424-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-011-9424-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DMachine%252Blearning%252Bstudy%252Bfor%252Bthe%252Bprediction%252Bof%252Btransdermal%252Bpeptide%26aulast%3DJung%26aufirst%3DEunkyoung%26date%3D2011%26date%3D2011%26volume%3D25%26issue%3D4%26spage%3D339%26epage%3D347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bohdan  Waszkowycz</span>, <span class="hlFld-ContribAuthor ">David E.  Clark</span>, <span class="hlFld-ContribAuthor ">Emanuela  Gancia</span>. </span><span class="cited-content_cbyCitation_article-title">Outstanding challenges in protein–ligand docking and structure‐based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2011,</strong> <em>1 </em>
                                    (2)
                                     , 229-259. <a href="https://doi.org/10.1002/wcms.18" title="DOI URL">https://doi.org/10.1002/wcms.18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.18%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DOutstanding%252Bchallenges%252Bin%252Bprotein%2525E2%252580%252593ligand%252Bdocking%252Band%252Bstructure%2525E2%252580%252590based%252Bvirtual%252Bscreening%26aulast%3DWaszkowycz%26aufirst%3DBohdan%26date%3D2011%26date%3D2011%26volume%3D1%26issue%3D2%26spage%3D229%26epage%3D259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tianxiao  Huan</span>, <span class="hlFld-ContribAuthor ">Xiaogang  Wu</span>, <span class="hlFld-ContribAuthor ">Jake Y  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Systems biology visualization tools for drug target discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2010,</strong> <em>5 </em>
                                    (5)
                                     , 425-439. <a href="https://doi.org/10.1517/17460441003725102" title="DOI URL">https://doi.org/10.1517/17460441003725102</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441003725102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441003725102%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DSystems%252Bbiology%252Bvisualization%252Btools%252Bfor%252Bdrug%252Btarget%252Bdiscovery%26aulast%3DHuan%26aufirst%3DTianxiao%26date%3D2010%26date%3D2010%26volume%3D5%26issue%3D5%26spage%3D425%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irene  Barrett</span>, <span class="hlFld-ContribAuthor ">Miriam  Carr</span>, <span class="hlFld-ContribAuthor ">Niamh  O’Boyle</span>, <span class="hlFld-ContribAuthor ">Lisa M.  Greene</span>, <span class="hlFld-ContribAuthor ">Andrew  J. S. Knox</span>, <span class="hlFld-ContribAuthor ">David G.  Lloyd</span>, <span class="hlFld-ContribAuthor ">Daniela M.  Zisterer</span>, <span class="hlFld-ContribAuthor ">Mary J.  Meegan</span>. </span><span class="cited-content_cbyCitation_article-title">Lead identification of conformationally restricted benzoxepin type combretastatin analogs: synthesis, antiproliferative activity, and tubulin effects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2010,</strong> <em>25 </em>
                                    (2)
                                     , 180-194. <a href="https://doi.org/10.3109/14756360903169659" title="DOI URL">https://doi.org/10.3109/14756360903169659</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3109/14756360903169659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3109%2F14756360903169659%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DLead%252Bidentification%252Bof%252Bconformationally%252Brestricted%252Bbenzoxepin%252Btype%252Bcombretastatin%252Banalogs%25253A%252Bsynthesis%25252C%252Bantiproliferative%252Bactivity%25252C%252Band%252Btubulin%252Beffects%26aulast%3DBarrett%26aufirst%3DIrene%26date%3D2010%26date%3D2010%26volume%3D25%26issue%3D2%26spage%3D180%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eunkyoung  Jung</span>, <span class="hlFld-ContribAuthor ">Junhyoung  Kim</span>, <span class="hlFld-ContribAuthor ">Seung-Hoon  Choi</span>, <span class="hlFld-ContribAuthor ">Minkyoung  Kim</span>, <span class="hlFld-ContribAuthor ">Hokyoung  Rhee</span>, <span class="hlFld-ContribAuthor ">Jae-Min  Shin</span>, <span class="hlFld-ContribAuthor ">Kihang  Choi</span>, <span class="hlFld-ContribAuthor ">Sang-Kee  Kang</span>, <span class="hlFld-ContribAuthor ">Nam Kyung  Lee</span>, <span class="hlFld-ContribAuthor ">Yun-Jaie  Choi</span>, <span class="hlFld-ContribAuthor ">Dong Hyun  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Artificial neural network study on organ-targeting peptides. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2010,</strong> <em>24 </em>
                                    (1)
                                     , 49-56. <a href="https://doi.org/10.1007/s10822-009-9313-0" title="DOI URL">https://doi.org/10.1007/s10822-009-9313-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-009-9313-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-009-9313-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DArtificial%252Bneural%252Bnetwork%252Bstudy%252Bon%252Borgan-targeting%252Bpeptides%26aulast%3DJung%26aufirst%3DEunkyoung%26date%3D2010%26date%3D2009%26volume%3D24%26issue%3D1%26spage%3D49%26epage%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">E Besray  Unal</span>, <span class="hlFld-ContribAuthor ">Attila  Gursoy</span>, <span class="hlFld-ContribAuthor ">Burak  Erman</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational energies and entropies of peptides, and the peptide–protein binding problem. </span><span class="cited-content_cbyCitation_journal-name">Physical Biology</span><span> <strong>2009,</strong> <em>6 </em>
                                    (3)
                                     , 036014. <a href="https://doi.org/10.1088/1478-3975/6/3/036014" title="DOI URL">https://doi.org/10.1088/1478-3975/6/3/036014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1088/1478-3975/6/3/036014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1088%2F1478-3975%2F6%2F3%2F036014%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Biology%26atitle%3DConformational%252Benergies%252Band%252Bentropies%252Bof%252Bpeptides%25252C%252Band%252Bthe%252Bpeptide%2525E2%252580%252593protein%252Bbinding%252Bproblem%26aulast%3DUnal%26aufirst%3DE%2BBesray%26date%3D2009%26date%3D2009%26volume%3D6%26issue%3D3%26spage%3D036014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew J. S.  Knox</span>, <span class="hlFld-ContribAuthor ">Mary J.  Meegan</span>, <span class="hlFld-ContribAuthor ">Vladimir  Sobolev</span>, <span class="hlFld-ContribAuthor ">Dermot  Frost</span>, <span class="hlFld-ContribAuthor ">Daniela M.  Zisterer</span>, <span class="hlFld-ContribAuthor ">D. Clive  Williams</span>, <span class="hlFld-ContribAuthor ">David G.  Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">Target Specific Virtual Screening:  Optimization of an Estrogen Receptor Screening Platform. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2007,</strong> <em>50 </em>
                                    (22)
                                     , 5301-5310. <a href="https://doi.org/10.1021/jm0700262" title="DOI URL">https://doi.org/10.1021/jm0700262</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm0700262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm0700262%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTarget%252BSpecific%252BVirtual%252BScreening%25253A%2525E2%252580%252589%252BOptimization%252Bof%252Ban%252BEstrogen%252BReceptor%252BScreening%252BPlatform%26aulast%3DKnox%26aufirst%3DAndrew%2BJ.%2BS.%26date%3D2007%26date%3D2007%26volume%3D50%26issue%3D22%26spage%3D5301%26epage%3D5310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew J.  Bordner</span>, <span class="hlFld-ContribAuthor ">Andrey A.  Gorin</span>. </span><span class="cited-content_cbyCitation_article-title">Protein docking using surface matching and supervised machine learning. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2007,</strong> <em>68 </em>
                                    (2)
                                     , 488-502. <a href="https://doi.org/10.1002/prot.21406" title="DOI URL">https://doi.org/10.1002/prot.21406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.21406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.21406%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DProtein%252Bdocking%252Busing%252Bsurface%252Bmatching%252Band%252Bsupervised%252Bmachine%252Blearning%26aulast%3DBordner%26aufirst%3DAndrew%2BJ.%26date%3D2007%26date%3D2007%26volume%3D68%26issue%3D2%26spage%3D488%26epage%3D502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregory A.  Landrum</span>, <span class="hlFld-ContribAuthor ">Julie E.  Penzotti</span>, <span class="hlFld-ContribAuthor ">Santosh  Putta</span>. </span><span class="cited-content_cbyCitation_article-title">Feature-map vectors: a new class of informative descriptors for computational drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2007,</strong> <em>20 </em>
                                    (12)
                                     , 751-762. <a href="https://doi.org/10.1007/s10822-006-9085-8" title="DOI URL">https://doi.org/10.1007/s10822-006-9085-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-006-9085-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-006-9085-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DFeature-map%252Bvectors%25253A%252Ba%252Bnew%252Bclass%252Bof%252Binformative%252Bdescriptors%252Bfor%252Bcomputational%252Bdrug%252Bdiscovery%26aulast%3DLandrum%26aufirst%3DGregory%2BA.%26date%3D2007%26date%3D2007%26volume%3D20%26issue%3D12%26spage%3D751%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gilles  Marcou</span>, <span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (1)
                                     , 195-207. <a href="https://doi.org/10.1021/ci600342e" title="DOI URL">https://doi.org/10.1021/ci600342e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci600342e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DOptimizing%252BFragment%252Band%252BScaffold%252BDocking%252Bby%252BUse%252Bof%252BMolecular%252BInteraction%252BFingerprints%26aulast%3DMarcou%26aufirst%3DGilles%26date%3D2007%26date%3D2006%26volume%3D47%26issue%3D1%26spage%3D195%26epage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Bender</span>, <span class="hlFld-ContribAuthor ">Jeremy L.  Jenkins</span>, <span class="hlFld-ContribAuthor ">Qingliang  Li</span>, <span class="hlFld-ContribAuthor ">Sam E.  Adams</span>, <span class="hlFld-ContribAuthor ">Edward O.  Cannon</span>, <span class="hlFld-ContribAuthor ">Robert C.  Glen</span>. </span><span class="cited-content_cbyCitation_article-title">Chapter 9 Molecular Similarity: Advances in Methods, Applications and Validations in Virtual Screening and QSAR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2006,</strong>,, 141-168. <a href="https://doi.org/10.1016/S1574-1400(06)02009-3" title="DOI URL">https://doi.org/10.1016/S1574-1400(06)02009-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1574-1400(06)02009-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1574-1400%2806%2902009-3%26sid%3Dliteratum%253Aachs%26atitle%3DChapter%252B9%252BMolecular%252BSimilarity%25253A%252BAdvances%252Bin%252BMethods%25252C%252BApplications%252Band%252BValidations%252Bin%252BVirtual%252BScreening%252Band%252BQSAR%26aulast%3DBender%26aufirst%3DAndreas%26date%3D2006%26spage%3D141%26epage%3D168%26pub%3DElsevier%26date%3D2006%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wendy D.  Cornell</span>. </span><span class="cited-content_cbyCitation_article-title">Chapter 16 Recent Evaluations of High Throughput Docking Methods for Pharmaceutical Lead Finding – Consensus and Caveats. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2006,</strong>,, 297-323. <a href="https://doi.org/10.1016/S1574-1400(06)02016-0" title="DOI URL">https://doi.org/10.1016/S1574-1400(06)02016-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1574-1400(06)02016-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1574-1400%2806%2902016-0%26sid%3Dliteratum%253Aachs%26atitle%3DChapter%252B16%252BRecent%252BEvaluations%252Bof%252BHigh%252BThroughput%252BDocking%252BMethods%252Bfor%252BPharmaceutical%252BLead%252BFinding%252B%2525E2%252580%252593%252BConsensus%252Band%252BCaveats%26aulast%3DCornell%26aufirst%3DWendy%2BD.%26date%3D2006%26spage%3D297%26epage%3D323%26pub%3DElsevier%26date%3D2006%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360n00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360n00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00001.jpeg" id="rightTab-_i8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 The test set's hydrophobic (CLogp) and hydrophilic (H-bond donors plus acceptors, calculated by Daylight) properties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00002.jpeg" id="rightTab-_i9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 The training set's hydrophobic (CLogp) and hydrophilic (H-bond donors plus acceptors, calculated by Daylight) properties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00003.jpeg" id="rightTab-_i10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 A schematic set of ROC curves showing perfect performance, high performance, low performance, and random performance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00004.jpeg" id="rightTab-_i13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Full ROC curves for selected descriptor subsets using a random forest learner (see also Table 3). Overall performance is determined by proximity to the top, left corner. The single descriptor subsets make up the lowest performers. Selected points are presented numerically in Table 3. Figure 5 gives a detailed view of the high performing region.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00005.jpeg" id="rightTab-_i14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 Detail of the high-performing region of Figure 4 (top, left). All the best subsets contain SASA<sub>TOTAL</sub> and ChemScore<sub>H</sub><sub>-</sub><sub>BOND</sub>, and all the lowest performing subsets contain only a single descriptor. Those without a subscript are using all the descriptor from that source (Table 2). (This figure uses the same line colors and symbols that are used in Figure 4.)</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00006.jpeg" id="rightTab-_i16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Full ROC curves for selected descriptor subsets using logistic regression. Descriptor sets with a subscript are for a single descriptor. Overall performance is measured by proximity to the top, left corner. The best subset contains SASA<sub>TOTAL</sub> and ChemScore<sub>H</sub><sub>-</sub><sub>BOND</sub>. Lowest performing subsets are composed of a single descriptor. (Table 2). Selected points from this figure are presented numerically in Table 4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00007.jpeg" id="rightTab-_i17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Detail of high performers from Figure 6 and using the same colors and symbols. Overall performance is by proximity to the top, left corner. The best subset contains SASA<sub>total</sub> and CSCORE<sub>H</sub><sub>-</sub><sub>bond</sub>. Lowest performing subsets are composed of a single descriptor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00008.jpeg" id="rightTab-_i18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 The points show the values of ChemScore<sub>H</sub><sub>-</sub><sub>bond</sub> and total SASA<sub>total</sub> descriptors for the test set for both binders and decoys. The blue lines show the partitioning of the space as given by a single tree on the training set into rectangular regions which are labeled 1 through 5. The regions 1, 2, and 4 contain a majority of binders, and regions 3 and 5 contain a majority of decoys. Eight points (discussed in the text) are labeled by the pdb id of the target, four targets with a binder and decoy each. The black type labels and black diamond points are the binding examples and the red code labels are the decoys. The poses and structures for these points are shown in Figures 9−16.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00009" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00009.jpeg" id="rightTab-_i19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 The decoy gdn (from 1hnc) docked into HIV protease (1hpx) falls in region 3 of Figure 8. It does not bury enough surface area or have enough H-bonding to be mistaken for a binder.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00010" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00010.jpeg" id="rightTab-_i21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 The binding mode of KNI in HIV protease (1hpx) as found in the pdb. It has strong enough interactions to be correctly identified as a binder and falls in region 1 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00011" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00011.jpeg" id="rightTab-_i22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 11 Thrombin (1uvu) with decoy (1okl_mns) ligand DOCKed in. It is correctly identified as a nonbinder and falls in region 3 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00011"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00012" data-index="12" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00012.jpeg" id="rightTab-_i23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 12 This is thrombin with binder from the pdb. It is correctly identified as a binder and falls in region 4 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00012"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00013" data-index="13" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00013.jpeg" id="rightTab-_i24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 13 The decoy pcp (1a96) docked into cytidine deaminase (1ctt). This decoy has enough buried surface area and H-bonding to fall in region 4, a majority binding region, and therefore is a false positive in our single tree, two-descriptor learner. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00013"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00014" data-index="14" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00014.jpeg" id="rightTab-_i25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 14 This is native ligand (dhz) docked into cytidine deaminase (1ctt). It has enough interactions to be correctly considered a binder and can be found in region 1 of Figure 8. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00014"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00015" data-index="15" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00015.jpeg" id="rightTab-_i26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 15 The decoy ligand (phb) in carbonic anhydrase (1cnw). Because of its lack of interactions it falls in region 3 of Figure 8 and is correctly excluded as a decoy by a single tree learner. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00015"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0493360f00016" data-index="16" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/medium/jm0493360f00016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00016.jpeg" id="rightTab-_i27" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 16 The native ligand (eg1) in carbonic anhydrase (1cnw). This complex does not have enough interactions to be considered a binder. Therefore for this two-descriptor single tree this would be false negative. Metal binding interactions must be considered to properly evaluate this complex.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-22/jm0493360/production/images/large/jm0493360f00016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0493360&amp;id=jm0493360f00016"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><a name="jm004802206831" id="jm004802206831"></a></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm0493360b00001"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00001_1"><span><span class="NLM_string-ref">Orozco, M.; Tirado-Rives, J.; Jorgensen, W. L. Mechanism for the rotamase activity of FK506 binding protein from molecular dynamics simulations. <i>  Biochemistry </i><b>1993</b>,<i> 32</i>, 12864−12874.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1993&pages=12874&author=M.+Orozco&author=J.+Tirado-Rives&author=W.+L+Jorgensen&title=Mechanism+for+the+rotamase+activity+of+FK506+binding+protein+from+molecular+dynamics+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOrozco%26aufirst%3DM.%26atitle%3DMechanism%2520for%2520the%2520rotamase%2520activity%2520of%2520FK506%2520binding%2520protein%2520from%2520molecular%2520dynamics%2520simulations%26jtitle%3DBiochemistry%26date%3D1993%26volume%3D32%26spage%3D12874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00002"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00002_1"><span><span class="NLM_string-ref">Sham, Y. Y.; Chu, Z. T.; Tao, H.; Warshel, A. Examining methods for calculations of binding free energies:  LRA; LIE; PDLD-LRA; and PDLD/S-LRA calculations of ligands binding to an HIV protease. <i>  Proteins-Struct. </i><i>Funct. Genet. </i><b>2000</b>,<i> 39</i>, 393−407.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=407&author=Y.+Y.+Sham&author=Z.+T.+Chu&author=H.+Tao&author=A+Warshel&title=Examining+methods+for+calculations+of+binding+free+energies%3A%E2%80%89+LRA%3B+LIE%3B+PDLD-LRA%3B+and+PDLD%2FS-LRA+calculations+of+ligands+binding+to+an+HIV+protease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSham%26aufirst%3DY.%2BY.%26atitle%3DExamining%2520methods%2520for%2520calculations%2520of%2520binding%2520free%2520energies%253A%25E2%2580%2589%2520LRA%253B%2520LIE%253B%2520PDLD-LRA%253B%2520and%2520PDLD%252FS-LRA%2520calculations%2520of%2520ligands%2520binding%2520to%2520an%2520HIV%2520protease%26jtitle%3DProteins-Struct.%2520Funct.%2520Genet.%26date%3D2000%26volume%3D39%26spage%3D407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00003"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00003_1"><span><span class="NLM_string-ref">Zhang, L. Y.; Gallicchio, E.; Friesner, R. A.; Levy, R. M.; Guermeur, Y. et al. Solvent models for protein−ligand binding:  Comparison of implicit solvent Poisson and surface generalized born models with explicit solvent simulations. <i>  J. Comput. Chem. </i><b>2001</b>,<i> 22</i>, 591−607.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=607&author=L.+Y.+Zhang&author=E.+Gallicchio&author=R.+A.+Friesner&author=R.+M.+Levy&author=Y+Guermeur&title=et+al.+Solvent+models+for+protein%E2%88%92ligand+binding%3A%E2%80%89+Comparison+of+implicit+solvent+Poisson+and+surface+generalized+born+models+with+explicit+solvent+simulations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%2BY.%26atitle%3Det%2520al.%2520Solvent%2520models%2520for%2520protein%25E2%2588%2592ligand%2520binding%253A%25E2%2580%2589%2520Comparison%2520of%2520implicit%2520solvent%2520Poisson%2520and%2520surface%2520generalized%2520born%2520models%2520with%2520explicit%2520solvent%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00004"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00004_1"><span><span class="NLM_string-ref">Robinson, G. W.; Cho, C. H.; Urquidi, J. Isosbestic points in liquid water:  Further strong evidence for the two-state mixture model. <i>  J. Chem. Phys. </i><b>1999</b>,<i> 111</i>, 698−702.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1999&pages=702&author=G.+W.+Robinson&author=C.+H.+Cho&author=J+Urquidi&title=Isosbestic+points+in+liquid+water%3A%E2%80%89+Further+strong+evidence+for+the+two-state+mixture+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DG.%2BW.%26atitle%3DIsosbestic%2520points%2520in%2520liquid%2520water%253A%25E2%2580%2589%2520Further%2520strong%2520evidence%2520for%2520the%2520two-state%2520mixture%2520model%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1999%26volume%3D111%26spage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00005"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00005_1"><span><span class="NLM_string-ref">BenNaim, A. A measure of the average cooperativity of a binding system. <i>  J. </i><i>Chem. Phys. </i><b>1998</b>,<i> 109</i>, 7443−7449.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=1998&pages=7449&author=A+BenNaim&title=A+measure+of+the+average+cooperativity+of+a+binding+system"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBenNaim%26aufirst%3DA%26atitle%3DA%2520measure%2520of%2520the%2520average%2520cooperativity%2520of%2520a%2520binding%2520system%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1998%26volume%3D109%26spage%3D7449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00006"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00006_1"><span><span class="NLM_string-ref">Lum, K.; Chandler, D.; Weeks, J. D. Hydrophobicity at small and large length scales. <i>  J. Phys. Chem. B </i><b>1999</b>,<i> 103</i>, 4570−4577.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1999&pages=4577&author=K.+Lum&author=D.+Chandler&author=J.+D+Weeks&title=Hydrophobicity+at+small+and+large+length+scales"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLum%26aufirst%3DK.%26atitle%3DHydrophobicity%2520at%2520small%2520and%2520large%2520length%2520scales%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D1999%26volume%3D103%26spage%3D4577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00007"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00007_1"><span><span class="NLM_string-ref">Wallqvist, A.; Covell, D. G. Cooperativity of Water-Solute Interactions at a Hydrophilic Surface. <i>  J. Phys. Chem. </i><b>1995</b>,<i> 99</i>, 5705−5712.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=1995&pages=5712&author=A.+Wallqvist&author=D.+G+Covell&title=Cooperativity+of+Water-Solute+Interactions+at+a+Hydrophilic+Surface"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26atitle%3DCooperativity%2520of%2520Water-Solute%2520Interactions%2520at%2520a%2520Hydrophilic%2520Surface%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1995%26volume%3D99%26spage%3D5712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00008"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00008_1"><span><span class="NLM_string-ref">Zou, X. Q.; Sun, Y. X.; Kuntz, I. D. Inclusion of solvation in ligand binding free energy calculations using the generalized-born model. <i>  J. Am. Chem. Soc. </i><b>1999</b>,<i> 121</i>, 8033−8043.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=8043&author=X.+Q.+Zou&author=Y.+X.+Sun&author=I.+D+Kuntz&title=Inclusion+of+solvation+in+ligand+binding+free+energy+calculations+using+the+generalized-born+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DX.%2BQ.%26atitle%3DInclusion%2520of%2520solvation%2520in%2520ligand%2520binding%2520free%2520energy%2520calculations%2520using%2520the%2520generalized-born%2520model%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D8043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00009"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00009_1"><span><span class="NLM_string-ref">Majeux, N.; Scarsi, M.; Caflisch, A. Efficient electrostatic solvation model for protein-fragment docking. <i>  Proteins-Struct. Funct. Genet. </i><b>2001</b>,<i> 42</i>, 256−268.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2001&pages=268&author=N.+Majeux&author=M.+Scarsi&author=A+Caflisch&title=Efficient+electrostatic+solvation+model+for+protein-fragment+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMajeux%26aufirst%3DN.%26atitle%3DEfficient%2520electrostatic%2520solvation%2520model%2520for%2520protein-fragment%2520docking%26jtitle%3DProteins-Struct.%2520Funct.%2520Genet.%26date%3D2001%26volume%3D42%26spage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm0493360b00010_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Leach, A. R.; Kuntz, I. D. Ligand solvation in molecular docking. <i>  Proteins-Struct. Funct. Genet. </i><b>1999</b>,<i> 34</i>, 4−16.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=16&author=B.+K.+Shoichet&author=A.+R.+Leach&author=I.+D+Kuntz&title=Ligand+solvation+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520solvation%2520in%2520molecular%2520docking%26jtitle%3DProteins-Struct.%2520Funct.%2520Genet.%26date%3D1999%26volume%3D34%26spage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00011"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00011_1"><span><span class="NLM_string-ref">Meng, E. C.; Shoichet, B. K.; Kuntz, I. D. Automated docking with grid-based energy evaluation. <i>  J. </i><i>Comput.</i><i> Chem. </i><b>1992</b>,<i> 13</i>, 505−524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=524&author=E.+C.+Meng&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Automated+docking+with+grid-based+energy+evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26atitle%3DAutomated%2520docking%2520with%2520grid-based%2520energy%2520evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00012"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00012_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Leach, A. R.; Kuntz, I. D. Ligand solvation in molecular docking. <i>  Proteins </i><b>1999</b>,<i> 34</i>, 4−16.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=16&author=B.+K.+Shoichet&author=A.+R.+Leach&author=I.+D+Kuntz&title=Ligand+solvation+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520solvation%2520in%2520molecular%2520docking%26jtitle%3DProteins%26date%3D1999%26volume%3D34%26spage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00013"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00013_1"><span><span class="NLM_string-ref">Jones, G.; Willett, P.; Glen, R. C.; Leach, A. R.; Taylor, R. Development and validation of a genetic algorithm for flexible docking. <i>  J. Mol. Biol. </i><b>1997</b>, <i>267</i>, 727−748.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=748&author=G.+Jones&author=P.+Willett&author=R.+C.+Glen&author=A.+R.+Leach&author=R+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00014"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00014_1"><span><span class="NLM_string-ref">Pang, Y. P.; Perola, E.; Xu, K.; Prendergast, F. G. EUDOC:  A computer program for identification of drug interaction sites in macromolecules and drug leads from chemical databases. <i>  J. Comput. Chem. </i><b>2001</b>,<i> 22</i>, 1750−1771.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=1771&author=Y.+P.+Pang&author=E.+Perola&author=K.+Xu&author=F.+G.+Prendergast&title=computer+program+for+identification+of+drug+interaction+sites+in+macromolecules+and+drug+leads+from+chemical+databases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPang%26aufirst%3DY.%2BP.%26atitle%3Dcomputer%2520program%2520for%2520identification%2520of%2520drug%2520interaction%2520sites%2520in%2520macromolecules%2520and%2520drug%2520leads%2520from%2520chemical%2520databases%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D1771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00015"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00015_1"><span><span class="NLM_string-ref">Muegge, I.; Martin, Y. A general and fast scoring function for protein−ligand interactions:  A simplified potential approach. <i>  J. Med. Chem. </i><b>1999</b>,<i> 42</i>, 791−804.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=804&author=I.+Muegge&author=Y+Martin&title=A+general+and+fast+scoring+function+for+protein%E2%88%92ligand+interactions%3A%E2%80%89+A+simplified+potential+approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMuegge%26aufirst%3DI.%26atitle%3DA%2520general%2520and%2520fast%2520scoring%2520function%2520for%2520protein%25E2%2588%2592ligand%2520interactions%253A%25E2%2580%2589%2520A%2520simplified%2520potential%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00016"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00016_1"><span><span class="NLM_string-ref">Gohlke, H.; Hendlich, M.; Klebe, G. Knowledge-based scoring function to predict protein−ligand interactions. <i>  J. Mol. Biol. </i><b>2000</b>,<i> 295</i>, 337−356.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=295&publication_year=2000&pages=356&author=H.+Gohlke&author=M.+Hendlich&author=G+Klebe&title=Knowledge-based+scoring+function+to+predict+protein%E2%88%92ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGohlke%26aufirst%3DH.%26atitle%3DKnowledge-based%2520scoring%2520function%2520to%2520predict%2520protein%25E2%2588%2592ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D295%26spage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00017"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00017_1"><span><span class="NLM_string-ref">Bohm, H. J. The development of a simple empirical scoring function to estimate the binding constant for a protein−ligand complex of known three-dimensional structure. <b>1994</b>,<i> 8</i>, 243−256.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=H.+J.+Bohm&title=The+development+of+a+simple+empirical+scoring+function+to+estimate+the+binding+constant+for+a+protein%E2%88%92ligand+complex+of+known+three-dimensional+structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBohm%26aufirst%3DH.%2BJ.%26btitle%3DThe%2520development%2520of%2520a%2520simple%2520empirical%2520scoring%2520function%2520to%2520estimate%2520the%2520binding%2520constant%2520for%2520a%2520protein%25E2%2588%2592ligand%2520complex%2520of%2520known%2520three-dimensional%2520structure%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00018"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00018_1"><span><span class="NLM_string-ref">Eldridge, M. D.; Murray, C. W.; Auton, T. R.; Paolini, G. V.; Mee, R. P. Empirical scoring functions 0.1. The development of a fast empirical scoring function to estimate the binding affinity of ligands in receptor complexes. <i>  J. Comput.-Aided Mol. Des. </i><b>1997</b>,<i> 11</i>, 425−445.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=445&author=M.+D.+Eldridge&author=C.+W.+Murray&author=T.+R.+Auton&author=G.+V.+Paolini&author=R.+P+Mee&title=Empirical+scoring+functions+0.1.+The+development+of+a+fast+empirical+scoring+function+to+estimate+the+binding+affinity+of+ligands+in+receptor+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEldridge%26aufirst%3DM.%2BD.%26atitle%3DEmpirical%2520scoring%2520functions%25200.1.%2520The%2520development%2520of%2520a%2520fast%2520empirical%2520scoring%2520function%2520to%2520estimate%2520the%2520binding%2520affinity%2520of%2520ligands%2520in%2520receptor%2520complexes%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1997%26volume%3D11%26spage%3D445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00019"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00019_1"><span><span class="NLM_string-ref">Charifson, P. S.; Corkery, J. J.; Murcko, M. A.; Walters, W. P. Consensus scoring:  A method for obtaining improved hit rates from docking databases of three-dimensional structures into proteins. <i>  J. Med. Chem. </i><b>1999</b>,<i> 42</i>, 5100−5109.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5109&author=P.+S.+Charifson&author=J.+J.+Corkery&author=M.+A.+Murcko&author=W.+P.+Walters&author=A+Consensus&title=method+for+obtaining+improved+hit+rates+from+docking+databases+of+three-dimensional+structures+into+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3Dmethod%2520for%2520obtaining%2520improved%2520hit%2520rates%2520from%2520docking%2520databases%2520of%2520three-dimensional%2520structures%2520into%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm0493360b00020_1"><span><span class="NLM_string-ref">Bissantz, C.; Folkers, G.; Rognan, D. Protein-Based Virtual Screening of Chemical Databases. 1. Evaluation of Different Docking/Scoring Combinations. <i>  J. Med. Chem. </i><b>2000</b>,<i> 43</i>, 4759−4767.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4767&author=C.+Bissantz&author=G.+Folkers&author=D+Rognan&title=Protein-Based+Virtual+Screening+of+Chemical+Databases.+1.+Evaluation+of+Different+Docking%2FScoring+Combinations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26atitle%3DProtein-Based%2520Virtual%2520Screening%2520of%2520Chemical%2520Databases.%25201.%2520Evaluation%2520of%2520Different%2520Docking%252FScoring%2520Combinations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00021"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00021_1"><span><span class="NLM_string-ref">Breiman, L. Random Forests. <i>  Machine Learning</i><b>2001</b>, <i>45</i>, 5−32.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Breiman%2C+L.+Random+Forests.+Machine+Learning+2001%2C+45%2C+5%E2%88%9232.+Breiman%2C+L.+Random+Forests.+Machine+Learning2001%2C+45%2C+5%E2%88%9232."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBreiman%26aufirst%3DL.%26jtitle%3DMachine%2520Learning%26date%3D2001%26volume%3D45%26spage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00022"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00022_1"><span><span class="NLM_string-ref">Rusinko, A.; MW, F.; Lambert, C.; Brown, P.; Young, S. Analysis of a large structure/biological activity data set using rescursive partitioning. <i>  J. Chem. </i><i>Inf. Comput. Sci. </i><b>1999</b>,<i> 39</i>, 1017−1026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=1026&author=A.+Rusinko&author=C.+Lambert&author=P.+Brown&author=S+Young&title=Analysis+of+a+large+structure%2Fbiological+activity+data+set+using+rescursive+partitioning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRusinko%26aufirst%3DA.%26atitle%3DAnalysis%2520of%2520a%2520large%2520structure%252Fbiological%2520activity%2520data%2520set%2520using%2520rescursive%2520partitioning%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00023"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00023_1"><span><span class="NLM_string-ref">Holm, L.; Sander, C. The FSSP Database of Structurally Aligned Protein Fold Families. <i>  Nucleic Acids Res. </i><b>1994</b>,<i> 22</i>, 3600−3609.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1994&pages=3609&author=L.+Holm&author=C+Sander&title=The+FSSP+Database+of+Structurally+Aligned+Protein+Fold+Families"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26atitle%3DThe%2520FSSP%2520Database%2520of%2520Structurally%2520Aligned%2520Protein%2520Fold%2520Families%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1994%26volume%3D22%26spage%3D3609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00024"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00024_1"><span><span class="NLM_string-ref">Holm, L.; Sander, C. Dali/FSSP classification of three-dimensional protein folds. <i>  Nucleic Acids Res. </i><b>1997</b>,<i> 25</i>, 231−234.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=234&author=L.+Holm&author=C.+Sander&author=FSSP+Dali&title=classification+of+three-dimensional+protein+folds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26atitle%3Dclassification%2520of%2520three-dimensional%2520protein%2520folds%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00025"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00025_1"><span><span class="NLM_string-ref">Ewing, T. J. A.; Makino, S.; Skillman, A. G.; Kuntz, I. D. DOCK 4.0:  Search strategies for automated molecular docking of flexible molecule databases. <i>  J. Comput.-Aided Mol. Des. </i><b>2001</b>,<i> 15</i>, 411−428.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=428&author=T.+J.+A.+Ewing&author=S.+Makino&author=A.+G.+Skillman&author=I.+D.+Kuntz&title=Search+strategies+for+automated+molecular+docking+of+flexible+molecule+databases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEwing%26aufirst%3DT.%2BJ.%2BA.%26atitle%3DSearch%2520strategies%2520for%2520automated%2520molecular%2520docking%2520of%2520flexible%2520molecule%2520databases%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00026"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00026_1"><span><span class="NLM_string-ref">Meng, E. C.; Shoichet, B. K.; Kuntz, I. D. Automated Docking with Grid-Based Energy Evaluation. <i>  J. Comput. Chem. </i><b>1992</b>,<i> 13</i>, 505−524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=524&author=E.+C.+Meng&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Automated+Docking+with+Grid-Based+Energy+Evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26atitle%3DAutomated%2520Docking%2520with%2520Grid-Based%2520Energy%2520Evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00027"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00027" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00027_1"><span><span class="NLM_string-ref">Weiner, S. J.; Kollman, P. A.; Case, D. A.; Singh, U. C.; Ghio, C. et al. A New Force Field for Molecular Mechanical Simulation of Nucleic Acids and Proteins. <i>  J. Am. Chem. Soc. </i><b>1984</b>,<i> 106</i>, 765−784.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1984&pages=784&author=S.+J.+Weiner&author=P.+A.+Kollman&author=D.+A.+Case&author=U.+C.+Singh&author=C+Ghio&title=et+al.+A+New+Force+Field+for+Molecular+Mechanical+Simulation+of+Nucleic+Acids+and+Proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWeiner%26aufirst%3DS.%2BJ.%26atitle%3Det%2520al.%2520A%2520New%2520Force%2520Field%2520for%2520Molecular%2520Mechanical%2520Simulation%2520of%2520Nucleic%2520Acids%2520and%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1984%26volume%3D106%26spage%3D784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00028"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00028" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00028_1"><span><span class="NLM_string-ref">Gasteiger, J.; Marsili, M. <i>  Organ. Magn. Reson. </i><b>1981</b>,<i> 15</i>, 353−360.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Gasteiger%2C+J.%3B+Marsili%2C+M.+Organ.+Magn.+Reson.+1981%2C+15%2C+353%E2%88%92360.+Gasteiger%2C+J.%3B+Marsili%2C+M.+Organ.+Magn.+Reson.+1981%2C+15%2C+353%E2%88%92360."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26jtitle%3DOrgan.%2520Magn.%2520Reson.%26date%3D1981%26volume%3D15%26spage%3D360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00029"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00029" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00029_1"><span><span class="NLM_string-ref">Gasteiger, J.; Marsili, M. <i>  Tetrahedron </i><b>1980</b>,<i> 36</i>, 3219−3228.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Gasteiger%2C+J.%3B+Marsili%2C+M.+Tetrahedron+1980%2C+36%2C+3219%E2%88%923228.+Gasteiger%2C+J.%3B+Marsili%2C+M.+Tetrahedron+1980%2C+36%2C+3219%E2%88%923228."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00030"><div class="reference"><div class="NLM_citation" id="rightTab-jm0493360b00030_1"><span><span class="NLM_string-ref">Marsili, M.; Gasteiger, J. <i>  Croat. Chem. Acta </i><b>1980</b>,<i> 53</i>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Marsili%2C+M.%3B+Gasteiger%2C+J.+Croat.+Chem.+Acta+1980%2C+53.+Marsili%2C+M.%3B+Gasteiger%2C+J.+Croat.+Chem.+Acta+1980%2C+53."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarsili%26aufirst%3DM.%26jtitle%3DCroat.%2520Chem.%2520Acta%26date%3D1980%26spage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00031"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00031" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00031_1"><span><span class="NLM_string-ref">Willett, P. <i>  Similarity and Clustering in Chemical Information Space</i>; Research Studies Press:  Letchworth.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Willett%2C+P.+Similarity+and+Clustering+in+Chemical+Information+Space%3B+Research+Studies+Press%3A%E2%80%89+Letchworth.+Willett%2C+P.+Similarity+and+Clustering+in+Chemical+Information+Space%3B+Research+Studies+Press%3A%E2%80%89+Letchworth."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26btitle%3DSimilarity%2520and%2520Clustering%2520in%2520Chemical%2520Information%2520Space%26pub%3DResearch%2520Studies%2520Press" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00032"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00032" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00032_1"><span><span class="NLM_string-ref">Martin, Y. C.; Kofron, J. L.; Traphagen, L. M. Do Structurally Similar Molecular Have Similar Biological Activity. <i>  J. Med. Chem. </i><b>2002</b>,<i> 45</i>, 4350−4358.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=4358&author=Y.+C.+Martin&author=J.+L.+Kofron&author=L.+M+Traphagen&title=Do+Structurally+Similar+Molecular+Have+Similar+Biological+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DDo%2520Structurally%2520Similar%2520Molecular%2520Have%2520Similar%2520Biological%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D4358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00033"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00033" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00033_1"><span><span class="NLM_string-ref">Leo, A. <i>  Programs CLogP and CMR</i>; BioByte Corp.:  Claremont, CA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Leo%2C+A.+Programs+CLogP+and+CMR%3B+BioByte+Corp.%3A%E2%80%89+Claremont%2C+CA.+Leo%2C+A.+Programs+CLogP+and+CMR%3B+BioByte+Corp.%3A%E2%80%89+Claremont%2C+CA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLeo%26aufirst%3DA.%26btitle%3DPrograms%2520CLogP%2520and%2520CMR%26pub%3DCA" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00034"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00034" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00034_1"><span><span class="NLM_string-ref">Varshney, A.; Brooks, F. J. Fast Analytical Computation of Richard's Smooth Molecular Surface; UNC at Chapel Hill:  Chapel Hill, NC, 1993.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1993&author=A.+Varshney&author=F.+J.+Brooks&title=Fast+Analytical+Computation+of+Richard%27s+Smooth+Molecular+Surface"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVarshney%26aufirst%3DA.%26btitle%3DFast%2520Analytical%2520Computation%2520of%2520Richard%2527s%2520Smooth%2520Molecular%2520Surface%26pub%3DNC%26date%3D1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00035"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00035" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00035_1"><span><span class="NLM_string-ref">Eisenberg, D.; McLachlan, A. Solvation Energy in Protein Folding and Binding. <i>  Nature </i><b>1986</b>,<i> 319</i>, 199−203.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=1986&pages=203&author=D.+Eisenberg&author=A+McLachlan&title=Solvation+Energy+in+Protein+Folding+and+Binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEisenberg%26aufirst%3DD.%26atitle%3DSolvation%2520Energy%2520in%2520Protein%2520Folding%2520and%2520Binding%26jtitle%3DNature%26date%3D1986%26volume%3D319%26spage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00036"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00036" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00036_1"><span><span class="NLM_string-ref">Trunk, G. V. A problem of Dimensionality:  A Simple Example. <i>  IEEE Trans. </i><i>Pattern Anal. Machine Intell. </i><b>1979</b>,<i> 1</i>, 306−307.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1979&pages=307&author=G.+V+Trunk&title=A+problem+of+Dimensionality%3A%E2%80%89+A+Simple+Example"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTrunk%26aufirst%3DG.%2BV%26atitle%3DA%2520problem%2520of%2520Dimensionality%253A%25E2%2580%2589%2520A%2520Simple%2520Example%26jtitle%3DIEEE%2520Trans.%2520Pattern%2520Anal.%2520Machine%2520Intell.%26date%3D1979%26volume%3D1%26spage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00037"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00037" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00037_1"><span><span class="NLM_string-ref">Brunzell, H.; Eriksson, J. Feature reduction of classification of multidimensional data. <i>  Pattern Recog. </i><b>2000</b>,<i> 33</i>, 1741−1748.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=1748&author=H.+Brunzell&author=J+Eriksson&title=Feature+reduction+of+classification+of+multidimensional+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrunzell%26aufirst%3DH.%26atitle%3DFeature%2520reduction%2520of%2520classification%2520of%2520multidimensional%2520data%26jtitle%3DPattern%2520Recog.%26date%3D2000%26volume%3D33%26spage%3D1748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00038"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00038" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00038_1"><span><span class="NLM_string-ref">Cover, T. M.; Van Campenhout, J. M. On the possible ordering in the measurement selection problem. <i>  IEEE Trans. Syst., Man, Cybernetics </i><b>1977</b>,<i> 7</i>, 657−661.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1977&pages=661&author=T.+M.+Cover&author=J.+M+Van+Campenhout&title=On+the+possible+ordering+in+the+measurement+selection+problem"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCover%26aufirst%3DT.%2BM.%26atitle%3DOn%2520the%2520possible%2520ordering%2520in%2520the%2520measurement%2520selection%2520problem%26jtitle%3DIEEE%2520Trans.%2520Syst.%252C%2520Man%252C%2520Cybernetics%26date%3D1977%26volume%3D7%26spage%3D661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00039"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00039" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00039_1"><span><span class="NLM_string-ref">Venables, W. N.; Ripley, B. D. <i>  Modern Applied Statistics with S-plus</i>; Springer:  New York, 1994.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=W.+N.+Venables&author=B.+D.+Ripley&title=Modern+Applied+Statistics+with+S-plus"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DVenables%26aufirst%3DW.%2BN.%26btitle%3DModern%2520Applied%2520Statistics%2520with%2520S-plus%26pub%3DSpringer%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00040"><div class="reference"><div class="NLM_citation" id="rightTab-jm0493360b00040_1"><span><span class="NLM_string-ref">McCullagh, P.; J. A., N. <i>  Generalized Linear Models</i>; Chapman and Hall:  London, 1989.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm0493360b00040/jm0493360b00040_1&amp;dbid=16&amp;doi=10.1021%2Fjm0493360&amp;key=10.1007%2F978-1-4899-3242-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&author=P.+McCullagh&title=Generalized+Linear+Models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00040_1&amp;dbid=16384&amp;doi=10.1007%2F978-1-4899-3242-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4899-3242-6%26sid%3Dliteratum%253Aachs%26aulast%3DMcCullagh%26aufirst%3DP.%26btitle%3DGeneralized%2520Linear%2520Models%26pub%3DChapman%2520and%2520Hall%26date%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00041"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00041" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00041_1"><span><span class="NLM_string-ref">CRAN. The Comprehensive R Archive Network.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CRAN.+The+Comprehensive+R+Archive+Network.+CRAN.+The+Comprehensive+R+Archive+Network."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0493360b00042"><div class="reference"><strong class="refLabel"><a href="#jm0493360b00042" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-jm0493360b00042_1"><span><span class="NLM_string-ref">Duda, R.; Hart; Stark, P. <i>  Pattern Classification</i>; John Wiley and Sons:  New York, 2001.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=R.+Duda&author=+Hart&title=Pattern+Classification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0493360b00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDuda%26aufirst%3DR.%26btitle%3DPattern%2520Classification%26pub%3DJohn%2520Wiley%2520and%2520Sons%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="NLM_notes" id="rightTab-"><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30"><a href="/doi/suppl/10.1021%2Fjm0493360">Supporting Information Available</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The pdb id codes of the proteins and ligands that form the training and test sets; the atom typing scheme used for the SASA calculations; pairwise correlation between descriptors in the training set. This material is available free of charge via the Internet at <a class="ext-link" href="http://pubs.acs.org" target="_blank">http://pubs.acs.org</a>. </div></div></div><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm0493360/suppl_file/jm0493360_s.pdf">jm0493360_s.pdf (191.9 kb)</a></li></ul></div></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm0493360&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2005.48.issue-22%3Barticle%3Aarticle%3A10.1021%2Fjm0493360%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm0493360" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                722KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a28718f723c6a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
