
<html lang="en"     class="pb-page"  data-request-id="ce8619c0-eee6-4ad9-be44-36fb3401b116"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-12;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01895;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Revisiting Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: An Improved Computational Model" /></meta><meta name="dc.Creator" content="Jihui  Zhao" /></meta><meta name="dc.Creator" content="Rongrong  Cui" /></meta><meta name="dc.Creator" content="Lihao  Wang" /></meta><meta name="dc.Creator" content="Yingjia  Chen" /></meta><meta name="dc.Creator" content="Zunyun  Fu" /></meta><meta name="dc.Creator" content="Xiaoyu  Ding" /></meta><meta name="dc.Creator" content="Chen  Cui" /></meta><meta name="dc.Creator" content="Tianbiao  Yang" /></meta><meta name="dc.Creator" content="Xutong  Li" /></meta><meta name="dc.Creator" content="Yuan  Xu" /></meta><meta name="dc.Creator" content="Kaixian  Chen" /></meta><meta name="dc.Creator" content="Xiaomin  Luo" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Mingyue  Zheng" /></meta><meta name="dc.Description" content="Aldehyde oxidase (AOX) is a drug metabolizing molybdo-flavoenzyme that has gained increasing attention because of contribution to the biotransformation in phase I metabolism of xenobiotics. Unfortu..." /></meta><meta name="Description" content="Aldehyde oxidase (AOX) is a drug metabolizing molybdo-flavoenzyme that has gained increasing attention because of contribution to the biotransformation in phase I metabolism of xenobiotics. Unfortu..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 19, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01895" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01895" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01895" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01895" /></link>
        
    
    

<title>Revisiting Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: An Improved Computational Model | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01895" /></meta><meta property="og:title" content="Revisiting Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: An Improved Computational Model" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0013.jpeg" /></meta><meta property="og:description" content="Aldehyde oxidase (AOX) is a drug metabolizing molybdo-flavoenzyme that has gained increasing attention because of contribution to the biotransformation in phase I metabolism of xenobiotics. Unfortunately, the intra- and interspecies variations in AOX activity and lack of reliable and predictive animal models make evaluation of AOX-catalyzed metabolism prone to be misleading. In this study, we developed an improved computational model integrating both atom-level and molecule-level features to predict whether a drug-like molecule is a potential human AOX (hAOX) substrate and to identify the corresponding sites of metabolism. Additionally, we combined the proposed computational strategy and in vitro experiments for evaluating the metabolic property of a series of epigenetic-related drug candidates still in the early stage of development. In summary, this study provides an improved strategy to evaluate the liability of molecules toward hAOX and offers useful information for accelerating the drug design and optimization stage." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01895"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01895">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01895&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01895&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01895&amp;href=/doi/10.1021/acs.jmedchem.9b01895" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 6523-6537</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01849" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01483" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Revisiting Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: An Improved Computational Model</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jihui Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jihui Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan District, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jihui++Zhao">Jihui Zhao</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Rongrong Cui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rongrong Cui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rongrong++Cui">Rongrong Cui</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lihao Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lihao Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Gillings School of Public Health, University of North Carolina at Chapel Hill, 135 Dauer Drive, Chapel Hill, North Carolina 27599, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lihao++Wang">Lihao Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yingjia Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yingjia Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yingjia++Chen">Yingjia Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Zunyun Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zunyun Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing 210023, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zunyun++Fu">Zunyun Fu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xiaoyu Ding</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoyu Ding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyu++Ding">Xiaoyu Ding</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Chen Cui</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chen Cui</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chen++Cui">Chen Cui</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Tianbiao Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tianbiao Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tianbiao++Yang">Tianbiao Yang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xutong Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xutong Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xutong++Li">Xutong Li</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yuan Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuan Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai EnnovaBio Pharmaceuticals Co., Ltd., Room 404, Building 2, Lane 720, Cailun Road, Pudong New Area, Shanghai 200120, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuan++Xu">Yuan Xu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Kaixian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaixian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing 210023, China</div><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, and School of Life Science and Technology, ShanghaiTech University, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaixian++Chen">Kaixian Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xiaomin Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaomin Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaomin++Luo">Xiaomin Luo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0426-3417" title="Orcid link">http://orcid.org/0000-0003-0426-3417</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Hualiang Jiang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hualiang Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">College of Pharmacy, Nanjing University of Chinese Medicine, 138 Xianlin Avenue, Qixia District, Nanjing 210023, China</div><div class="loa-info-affiliations-info">Shanghai Institute for Advanced Immunochemical Studies, and School of Life Science and Technology, ShanghaiTech University, Shanghai 200031, China</div></div><span class="conrtib-corresp"><strong>*</strong>H.J.: phone, 86-21-50806600-1303; email, <a href="/cdn-cgi/l/email-protection#b5ddd9dfdcd4dbd2f5c6dcd8d89bd4d69bd6db"><span class="__cf_email__" data-cfemail="ed858187848c838aad9e848080c38c8ec38e83">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Mingyue Zheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingyue Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan District, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>M.Z.: phone, 86-21-50806600-1308; email, <a href="/cdn-cgi/l/email-protection#1f726665777a71785f6c767272317e7c317c71"><span class="__cf_email__" data-cfemail="99f4e0e3f1fcf7fed9eaf0f4f4b7f8fab7faf7">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingyue++Zheng">Mingyue Zheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3323-3092" title="Orcid link">http://orcid.org/0000-0002-3323-3092</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01895&amp;href=/doi/10.1021%2Facs.jmedchem.9b01895" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 6523â6537</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 19, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 November 2019</li><li><span class="item_label"><b>Published</b> online</span>19 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 June 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01895" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01895</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6523%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJihui%2BZhao%252C%2BRongrong%2BCui%252C%2BLihao%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D12%26contentID%3Dacs.jmedchem.9b01895%26title%3DRevisiting%2BAldehyde%2BOxidase%2BMediated%2BMetabolism%2Bin%2BDrug-like%2BMolecules%253A%2BAn%2BImproved%2BComputational%2BModel%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6537%26publicationDate%3DJune%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01895"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1334</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01895" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Revisiting Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: An Improved Computational Model&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jihui&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Rongrong&quot;,&quot;last_name&quot;:&quot;Cui&quot;},{&quot;first_name&quot;:&quot;Lihao&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yingjia&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Zunyun&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Xiaoyu&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Chen&quot;,&quot;last_name&quot;:&quot;Cui&quot;},{&quot;first_name&quot;:&quot;Tianbiao&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Xutong&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yuan&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Kaixian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Xiaomin&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Mingyue&quot;,&quot;last_name&quot;:&quot;Zheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;6523-6537&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01895&quot;},&quot;abstract&quot;:&quot;Aldehyde oxidase (AOX) is a drug metabolizing molybdo-flavoenzyme that has gained increasing attention because of contribution to the biotransformation in phase I metabolism of xenobiotics. Unfortunately, the intra- and interspecies variations in AOX activity and lack of reliable and predictive animal models make evaluation of AOX-catalyzed metabolism prone to be misleading. In this study, we developed an improved computational model integrating both atom-level and molecule-level features to predict whether a drug-like molecule is a potential human AOX (hAOX) substrate and to identify the corresponding sites of metabolism. Additionally, we combined the proposed computational strategy and in vitro experiments for evaluating the metabolic property of a series of epigenetic-related drug candidates still in the early stage of development. In summary, this study provides an improved strategy to evaluate the liability of molecules toward hAOX and offers useful information for accelerating the drug design and optimi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01895&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01895" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01895&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01895" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01895&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01895" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01895&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01895&amp;href=/doi/10.1021/acs.jmedchem.9b01895" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01895" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01895" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01895%26sid%3Dliteratum%253Aachs%26pmid%3D32191458%26genre%3Darticle%26aulast%3DZhao%26date%3D2020%26atitle%3DRevisiting%2BAldehyde%2BOxidase%2BMediated%2BMetabolism%2Bin%2BDrug-like%2BMolecules%253A%2BAn%2BImproved%2BComputational%2BModel%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D12%26spage%3D6523%26epage%3D6537%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291537" title="Electrical energy">Electrical energy</a>,</li><li><a href="/action/doSearch?ConceptID=290957" title="Molecular modeling">Molecular modeling</a>,</li><li><a href="/action/doSearch?ConceptID=292143" title="Metabolism">Metabolism</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/jmcmar.2020.63.issue-12/20200625/jmcmar.2020.63.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Aldehyde oxidase (AOX) is a drug metabolizing molybdo-flavoenzyme that has gained increasing attention because of contribution to the biotransformation in phase I metabolism of xenobiotics. Unfortunately, the intra- and interspecies variations in AOX activity and lack of reliable and predictive animal models make evaluation of AOX-catalyzed metabolism prone to be misleading. In this study, we developed an improved computational model integrating both atom-level and molecule-level features to predict whether a drug-like molecule is a potential human AOX (hAOX) substrate and to identify the corresponding sites of metabolism. Additionally, we combined the proposed computational strategy and in vitro experiments for evaluating the metabolic property of a series of epigenetic-related drug candidates still in the early stage of development. In summary, this study provides an improved strategy to evaluate the liability of molecules toward hAOX and offers useful information for accelerating the drug design and optimization stage.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/12"><issue-title>Drug Metabolism and Toxicology</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aldehyde oxidase (AOX), a drug metabolizing molybdo-flavoenzyme distributed mainly in mammal cytoplasm, has broad substrate specificity and capability to catalyze the metabolism of various endogenous and exogenous transformations, including oxidation of aldehydes and nitrogen-containing heterocycles,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> hydrolysis of amides,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and reduction of <i>N</i>-oxides and <i>S</i>-oxides.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> AOX widely distributes with profound species differences in vivo and plays an important role in phase I metabolism of xenobiotics. It has been reported that AOX contributes to clinical failures by metabolizing molecules with N-containing heterocyclic aromatic ring, in which the aromatic N is introduced to stabilize the metabolism by CYP-450s and consequently increase the susceptibility to hAOX.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Ever-growing small molecule candidates have failed due to hAOX-mediated oxidative metabolism.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> For example, INC280 (capmatinib), PF-04217903, SGX-523, and others have been terminated during the early phase of clinical trials due to AOX-catalyzed biotransformation.<a onclick="showRef(event, 'ref3 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref3 ref6 ref7">(3,6,7)</a> However, the different experimental medium or poor in vitroâin vivo extrapolation may significantly influence the estimation of hAOX-mediated metabolism, not to mention the uncertainty caused by species differences. Under these circumstances, computational modeling can be considered as a complementary strategy to predict hAOX metabolic profiles with homologous experimental data.</div><div class="NLM_p">Recently, Manevski et al. discussed thoroughly the biological functions, catalytic mechanism, experimental assessment, and the rational drug design approaches to modulate AOX-mediated metabolism in drug design and discovery.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The structure of hAOX contains three domains: (1) the flavin adenine dinucleotide (FAD) domain, (2) two [2Fe-2S] redox domain, and (3) the molybdenum cofactor (MoCo) and substrate binding domain.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The three domains are essential for hAOX-mediated biotransformation of substrate.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Several studies have been reported about the mechanism of hAOX-catalyzed oxidation of azaheterocycle molecules,<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10â12)</a> which can be briefly summarized as follows: (1) The nucleophilic attack at the electron deficient C of the substrate is initialized by the hydroxyl of the MoCo. (2) The H attached to the electron deficient C transfers to the S of MoCo simultaneously, and then the substrate binds to the MoCo by forming the CâOâMo bond.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (3) The electron deficient C is nucleophilically attacked by one water molecule from solvent, and the substrate is released as an oxidation product.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The proposed mechanism of AOX-mediated oxidation was included in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Scheme showing the proposed mechanism of AOX-mediated oxidation. PDB structure used for illustration is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UHX">4UHX</a>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and the substrate is phthalazine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Inspired by the catalytic mechanism of the hAOX mediated metabolism, several in silico models have been reported to predict whether a molecule can be metabolized by hAOX and where its potential sites of metabolism (SOMs) locate in the structure. Torres et al. used the energy of the tetrahedral intermediate formation to predict the regioselectivity in hAOX-catalyzed metabolisms.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Dalvie et al. employed a similar strategy to predict the structureâmetabolism relationship (SMR) for zoniporide analogues.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> They proposed that the energies of formation of tetrahedral intermediates could establish a possible relationship with the occurrence of a nucleophilic attack, as well as the electrostatic potential (ESP) charge of the carbon adjacent to the nitrogen atom and the energy of lowest unoccupied molecular orbital (<i>E</i><sub>LUMO</sub>). Besides, Jones et al. used the energy of the tetrahedral intermediate and steric hindrance as two descriptors to predict the intrinsic clearance for eight drugs metabolized by hAOX.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> On the basis of similar descriptors, we constructed a decision tree model to predict the susceptibility of molecules toward hAOX.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> More recently, Lepri et al. experimentally evaluated a large amount of small molecules containing hAOX-substrate-like motifs and summarized the general SMR rules for hAOX mediated metabolism.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Their work proposed that the electronic and exposure effects were two important features and mutually influenced, in which the electronic effects were calculated using density functional theory and exposure effects were assessed by molecular docking analysis to consider the ligand exposure in hAOX catalytic site. Montefiori et al. explored the regioselectivity of a series of 6-substituted 4-quinazolinones using the concerted mechanism as a reactive model.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Their results indicated that the stability of the hydroxylated tetrahedral intermediate could be used to determine whether a site in the AOX substrate was a likely SOM. In their study, the ESP charge of the electron deficient C showed good correlation with the stability of the hydroxylated tetrahedral intermediate, which could be used to separate the SOMs and non-SOMs.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Cruciani et al. developed a probability function for hAOX-metabolized oxidation with 88% prediction accuracy based on a large set of homogeneous experimental data, in which both exposure and electrophilic effects were taken into account.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Furthermore, Montefiori et al. collected a data set of 78 substrates with known SOMs information and constructed models to predict the AOX-mediated SOMs, in which the importance of the chemical shift and ESP charge of the deficient C of SOMs were highlighted.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">In our previous work, the reaction energy barrier (Î<i>E</i><sub>inter</sub>) and steric hindrance (Steric) were adopted to predict the potential SOMs of the molecules toward hAOX-mediated metabolism. As the results illuminated, the DT<sub>AOX</sub> model exhibited good predictive performance and gave a clear explanation of the relationship between the exposure, reaction energy barrier effects, and the hAOX-mediated metabolism.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Consequently, the recently published hAOX-mediated oxidative metabolism data were used to evaluate the previously published DT<sub>AOX</sub> model including substrate/nonsubstrate data.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> The sample distribution was calculated, and more than half of these recently published data fall outside the applicability domain (AD) defined by DT<sub>AOX</sub>.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> As a result, the model yielded a lower recall rate (alternatively named as the sensitivity score, SE) of 0.74 and a specificity score (SP) of 0.64. Clearly, DT<sub>AOX</sub> needs to be improved to predict whether a molecule was a true substrate and to identify the potential metabolic site toward hAOX-mediated oxidative metabolism. Checking the incorrect predictions, we further noticed that DT<sub>AOX</sub> also failed to recognize the potential SOM with similar scaffolds. For example, the tendency toward hAOX of position 2 in quinoline analogues was indistinguishable to the model because these sites showed similar reaction energy barrier and steric hindrance, as illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. Among these quinoline analogues, only MOL038 was a hAOX substrate, and the other three were all mistaken by DT<sub>AOX</sub> for substrates. Apparently, more features are needed to capture the substituent differences in these similar molecules. Similar observations were also reported by Cruciani et al.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Although the 5-nitrophthalazine, 5-aminophthalazine, and phthalazine were analogues with only different substituents at same position of phthalazine, these compounds exhibited different relationship with the empirical SMR concluded by Lepri et al.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> In addition, DT<sub>AOX</sub> was also evaluated toward the newly published data with known hAOX-catalyzed oxidative sites within the applicability domain.<a onclick="showRef(event, 'ref7 ref19'); return false;" href="javascript:void(0);" class="ref ref7 ref19">(7,19)</a> As summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S1</a>, the low sensitivity score suggested that many SOMs were mispredicted to be negative by the model.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Site of metabolism prediction results of four quinoline analogues by DT<sub>AOX</sub> model. Position 2 in these quinoline analogues is marked as black asterisk. Pred.: prediction results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The above retrospective analyses revealed that our previous computational model DT<sub>AOX</sub> trained using small data set has narrow applicability domain and insufficient generalization performance. In this context, more data and more global and local features need to be considered to improve the model. Several works have been published to predict the metabolizing enzyme mediated metabolic selectivity by constructing machine learning models with impressive predictive performance. For example, Peng et al. predicted UGT-catalyzed metabolism by constructing SVM classification model and identified 84% of experimentally observed sites of metabolisms for an external test.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> A multitask deep autoencoder neural network model was developed by Li et al. to predict the inhibitors of five major isozymes of CYP450 and outperformed traditional machine learning methods.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The recently published homogeneously hAOX-mediated experimental data and structurally diverse molecules have laid a solid foundation for constructing a more accurate hAOX-mediated drug metabolism model.</div><div class="NLM_p last">There are two objectives of this study: (1) to accurately predict whether the molecule is hAOX substrate and (2) to predict where the SOMs are located, i.e., to identify the specific site(s) within substrates that is metabolized by hAOX. First, the structurally diverse, experimental homogeneous data set was collected from the latest published literature. Second, the molecular and atomic properties including topological, physicochemical, and electronic descriptors were calculated, and the descriptors were combined to construct the substrate/nonsubstrate classification model. Third, the selected key descriptors were combined to optimize the SOMs classification model. Fourth, external test sets were used to verify the generalization ability of the substrate/nonsubstrate and SOMs classification model. In the end, the epigenetics-related inhibitors containing hAOX-substrate-like motifs were predicted using the combined model and in vitro experiments were carried out for further validation.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78776" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78776" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> hAOX-Catalyzed Oxidation: A View from Molecular Level to Atomic Level</h3><div class="NLM_p">Usually, some physicalâchemical properties such as logÂ <i>P</i> and logÂ <i>D</i> are the important factors to discriminate whether a molecule is a cytochrome P450 substrate or not.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> However, for hAOX substrates and nonsubstrates, their physicalâchemical properties are of less discriminatory power.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> As mentioned above, a variety of computational models have been developed to predict the hAOX-catalyzed metabolism. On the basis of homogeneously hAOX-mediated experimental data with structurally diverse skeletons recently published, we herein studied whether there were individual topological and electronic features that could determine the regioselectivity of hAOX-mediated metabolism.</div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the hAOX substrates have broad substrate specificity with widely different molecular size and geometry. Taking the distribution of molecular volume as an example, we may find from <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A that the interquartile range of hAOX substrates is from 230 to 345. This value is wider than that of a few major isoforms of CYP450s with substrate specificity, such as CYP1A2, CYP2D6, and CYP2C19.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Besides, the molecular weights showed similar distribution among substrates and nonsubstrates, as well as the molecular volume and molecular surface area (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Consequently, it is difficult to estimate whether a molecule is a hAOX substrate according to the molecular weight or molecular surface area. Since the hAOX-catalyzed oxidation of azaheterocycles involves the nucleophilic attack initialized by the MoCo to the molecule, the lower molecular electrophilicity might decrease the liability of a molecule toward the hAOX metabolism. In other words, the substrate might have higher electrophilicity than nonsubstrate, which might promote the nucleophilic attack by MoCo. In particular, the interaction of the highest occupied molecular orbital (HOMO) of the nucleophile with the lowest unoccupied molecular orbital (LUMO) of the electrophile is essential for bond formation.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> For hAOX-catalyzed reaction, the reactant with lower <i>E</i><sub>LUMO</sub> could be more electrophilic and thus more susceptible to hAOX. Dalvie et al. have verified the correlation between <i>E</i><sub>LUMO</sub> and hAOX-mediated susceptibility of zoniporide analogues.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> In addition to <i>E</i><sub>LUMO</sub>, some other properties like electronegativity,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> hardness,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and ESP properties have also been used to explore the nucleophilic attack reaction.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Accordingly, we calculated these electronic descriptors of reactivity or properties that determined the electrophilicity of the molecules, including <i>E</i><sub>LUMO</sub>, electronegativity, hardness, and the maximum and minimum ESP energies. For the electronic properties like <i>E</i><sub>LUMO</sub>, electronegativity, and hardness, there were no significant difference between substrates and nonsubstrates (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B,D). As for the ESP properties, several studies demonstrated that the partial charges such as MK charge or ESP partial charge of the metabolic sites, namely, the electron-deficient carbon adjacent to the nitrogen, were more positive than non-SOMs.<a onclick="showRef(event, 'ref12 ref16 ref18'); return false;" href="javascript:void(0);" class="ref ref12 ref16 ref18">(12,16,18)</a> Because the ESP partial charge of the atom was the integral of electrostatic potential surface on the atom, and for the nitrogen-containing heterocycle rings, the more positive charge of the SOMs might implicate the higher value of the ESP energy. Thus, we speculated that the substrate might have higher maximum values of the ESP energies (Max_ElPot) on the electrostatic potential maps or the nonsubstrates might have lower minimum values of the ESP energies (MIN_ElPot) on the electrostatic potential maps. However, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D, for these properties there was no significant difference between the substrates and nonsubstrates either.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Distribution of topological and electronic descriptors between substrates and nonsubstrates. (A) Distribution of molecular weight (unit, amu), molecular volume (unit, Ã<sup>3</sup>), and molecular surface area (unit, Ã<sup>2</sup>). (B) Distribution of the energy of lowest unoccupied molecular orbital (<i>E</i><sub>LUMO</sub>) (unit, eV), molecular electronegativity (unit, eV) and molecular hardness (unit: eV). (C) Distribution of the maximum and minimum values of the ESP, energies (kJ/mol) as mapped onto the electrostatic potential maps, denoted as Max_ElPot and Min_ElPot, respectively. Blue and orange colors represent the substrates and nonsubstrates, respectively. (D) <i>p</i> value of corresponding descriptors in parts AâC.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For these topological or electronic features, we found that there was no significant distribution between hAOX substrates and nonsubstrates, highlighting the following two points: (1) The hAOX possessed a broad range of substrate specificity and regioselectivity for oxidation of nitrogen-containing heterocycles. (2) For some electronic features like the <i>E</i><sub>LUMO</sub>, the correlation might only exist in a specific scaffold, but it might diminish when given a large set of structurally diverse molecules. Similar situations also appeared in the studies of Dalvie et al. and Torres et al.<a onclick="showRef(event, 'ref3 ref10 ref14'); return false;" href="javascript:void(0);" class="ref ref3 ref10 ref14">(3,10,14)</a> Therefore, these results indicated that we need to further explore the correlation of other features between published SOMs and non-SOMs.</div><div class="NLM_p">Next, we compared the distribution of the local property related to the SOMs and non-SOMs. Overall, the descriptors explored three types of effects, including the reactive energy barrier (Î<i>E</i><sub>inter</sub>), hindrance effects (Steric, exposed area of reactive atoms), and electronic effects (ESP charge and chemical shift of reactive atoms). Unlike the properties of molecular level, the distribution differences of atomic descriptors were significant in most instances (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>). First of all, the Î<i>E</i><sub>inter</sub> representing the reaction energy barrier had the highest significance with <i>p</i> value of â¼10<sup>â36</sup> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>). Chemical shift, ESP charge, and exposed area properties on the reactive carbon were more discriminating than those on the hydrogens attached to the carbon. In contrast, the distribution of chemical shift and ESP charge of the electron-rich nitrogen showed less significant difference except for the exposed area (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>). The statistical data of the distribution of these calculated descriptors are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Distribution of some energy-related, hindrance effect related, and electronic descriptors between SOMs and non-SOMs. The notation of these descriptors is listed following the order from left to right: Î<i>E</i><sub>inter</sub>, the energy changes between prototype and the tetrahedral intermediate; Steric, steric hindrance; ESP, electrostatic potential charge; CS, chemical shift; EA, exposed area. The prefix N_, C_, or H_ represent the electron-rich nitrogen, electron-deficient carbon, and hydrogen attached to the electron-deficient carbon atoms, respectively. The <i>p</i> values of the descriptors between SOMs and non-SOMs were calculated using <i>t</i> test statistical hypothesis. The corresponding significance of each descriptor is depicted as black asterisk within the underlying range: (â) 10<sup>â5</sup> < <i>p</i> < 0.05; (ââ) 10<sup>â15</sup> < <i>p</i> < 10<sup>â5</sup>; (âââ) <i>p</i> < 10<sup>â15</sup>. Blue and orange colors represent the SOMs and non-SOMs, respectively. Note that all these descriptors were normalized using StandardScaler implemented in Scikit-learn version 0.21.3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The reaction energy barrier that determines whether a reaction occurs or not has been adopted to discriminate the hAOX-mediated metabolism by many researchers.<a onclick="showRef(event, 'ref6 ref7 ref10'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref10">(6,7,10)</a> From the above analysis, we might find that the relative energy between the substrate and the intermediate (Î<i>E</i><sub>inter</sub>) indeed showed significant differences on the two classes. In addition, the chemical shift and ESP charge of the electron-deficient carbon of the SOMs also showed different distributions. Montefiori et al. have proposed that the ESP charge and chemical shift of the reactive carbon had certain correlation with the energies of the transition states and were good discriminators to predict the SOMs.<a onclick="showRef(event, 'ref12 ref18'); return false;" href="javascript:void(0);" class="ref ref12 ref18">(12,18)</a> Here, we also explored the relationship of ESP charge and chemical shift of the reactive carbon with the Î<i>E</i><sub>inter</sub>, but the results were not in concert with Montefiori et al.âs (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Figure S2</a>). Additionally, we also discovered that the properties of the hydrogen adjacent to the aromatic carbon showed significant differences (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>). There have been reports demonstrating that hydride displacement for a substrate might be a rate-limiting step.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Attention has been drawn to investigate the thermodynamics of the hydride extraction step, but the exposed effect of this hydrogen has not yet been explored.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p last">Overall, from molecular level to atomic level, these statistical results highlighted that the hAOX had quite wide substrate selectivity and diversity, and the common electrophilic descriptors like <i>E</i><sub>LUMO</sub>, electronegativity, and others showed less significant relationship with the tendency toward hAOX-mediated metabolism. As for the discrimination between SOMs and non-SOMs, the local features such as the reaction energy between substrate and tetrahedral intermediate, properties of aromatic carbon and adjacent hydrogen were important. On the basis of these analyses, we next proposed a new scheme to predict whether the molecule was a hAOX substrate and its regioselectivity of metabolism.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Construction of the Substrate/Nonsubstrate Classification Model</h3><div class="NLM_p">As mentioned above, the decision-tree-based model needed to be refined to incorporate more knowledge of AOX substrates and nonsubstrates. Here, a substrate/nonsubstrate classification model was first constructed. The maximum, minimum, and average values of local features of all potential SOMs within each molecule were calculated (described in <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Then, all descriptors were used to construct the substrate/nonsubstrate classification model using training data set of Cruciani et al.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> During the data set training stage, for every hyperparameter combination, 5-fold cross-validation was performed and the F1-score was used as model performance evaluation metric. The final selected hyperparameters were the following: layer size, [400,200]; learning rate, 0.001; batch size, 32; dropout rate, 0.3. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> summarized the model performance of the deep neural network (DNN) classification model. For comparison, the methods recently proposed by Montefiori et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> were benchmarked here with the same test and external data sets. We may find that the DNN model demonstrated more accurate prediction results on these tests.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Performance of DNN Substrate/Nonsubstrate Classification Model</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" align="center" char=".">train</th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">test</th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">external test</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">performance score</th><th class="colsep0 rowsep0" align="center" char=".">DNN</th><th class="colsep0 rowsep0" align="center" char=".">DNN</th><th class="colsep0 rowsep0" align="center" char=".">NMR shielding<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">ESP charge<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">chemcial shift<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">DNN</th><th class="colsep0 rowsep0" align="center" char=".">NMR shielding</th><th class="colsep0 rowsep0" align="center" char=".">ESP charge</th><th class="colsep0 rowsep0" align="center" char=".">chemcial shift</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SE</td><td class="colsep0 rowsep0" align="char" char=".">0.88</td><td class="colsep0 rowsep0" align="char" char=".">0.80</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td><td class="colsep0 rowsep0" align="char" char=".">0.86</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td><td class="colsep0 rowsep0" align="char" char=".">0.85</td><td class="colsep0 rowsep0" align="char" char=".">0.64</td><td class="colsep0 rowsep0" align="char" char=".">0.64</td><td class="colsep0 rowsep0" align="char" char=".">0.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SP</td><td class="colsep0 rowsep0" align="char" char=".">0.95</td><td class="colsep0 rowsep0" align="char" char=".">1.00</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">0.94</td><td class="colsep0 rowsep0" align="char" char=".">0.58</td><td class="colsep0 rowsep0" align="char" char=".">0.58</td><td class="colsep0 rowsep0" align="char" char=".">0.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Acc</td><td class="colsep0 rowsep0" align="char" char=".">0.92</td><td class="colsep0 rowsep0" align="char" char=".">0.93</td><td class="colsep0 rowsep0" align="char" char=".">0.57</td><td class="colsep0 rowsep0" align="char" char=".">0.70</td><td class="colsep0 rowsep0" align="char" char=".">0.57</td><td class="colsep0 rowsep0" align="char" char=".">0.90</td><td class="colsep0 rowsep0" align="char" char=".">0.60</td><td class="colsep0 rowsep0" align="char" char=".">0.60</td><td class="colsep0 rowsep0" align="char" char=".">0.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">F1</td><td class="colsep0 rowsep0" align="char" char=".">0.90</td><td class="colsep0 rowsep0" align="char" char=".">0.89</td><td class="colsep0 rowsep0" align="char" char=".">0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.73</td><td class="colsep0 rowsep0" align="char" char=".">0.63</td><td class="colsep0 rowsep0" align="char" char=".">0.88</td><td class="colsep0 rowsep0" align="char" char=".">0.54</td><td class="colsep0 rowsep0" align="char" char=".">0.54</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Descriptors for predicting AOX-mediated SOMs proposed by Montefiori et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></p></div></div></div><div class="NLM_p">As stated, it was difficult for the DT<sub>AOX</sub> model to predict the metabolic tendency accurately within similar analogues and to recognize the nonsubstrate accurately. Thus, we analyzed the classification ability of DNN model toward a series of similar molecules in the test data set. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the MOL486, MOL487, and MOL488 were mistaken as substrates by the DT<sub>AOX</sub> model, which were stable toward hAOX-mediated metabolism. In the DT<sub>AOX</sub> model, the position 2 in purine ring within MOL486 and in imidazole ring within MOL487 and MOL488 were predicted to be metabolic sites. In contrast, the DNN substrate/nonsubstrate classification model could accurately recognize the metabolic susceptibility of these molecules. (For the molecules in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, the DNN substrate/nonsubstrate classification model also yielded sound prediction results, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Figure S3 of Supporting Information</a>.)</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Examples of prediction ability of DNN substrate/nonsubstrate classification model on test data set. The positive and negative sites predicted by DT<sub>AOX</sub> are colored in red and blue circles, respectively. Pred.: prediction results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also investigated the model performance on external data set containing different molecules with diverse skeletons to evaluate the general applicability and robustness of this model. Furthermore, the DNN classification model was verified toward the external data set in which the molecules were collected from different resources with multiple experimental conditions. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the classification performance on external data set was slightly poorer than test data set, with the exception of the higher SE score. These results on both test data set and external data set revealed the generalization ability of the DNN classification model.</div><div class="NLM_p last">For several models introduced before, the focus was placed on determining which aromatic carbon could be metabolized by hAOX. In these studies, thermokinetic, electronic, and steric hindrance effects were three main factors that were taken into consideration. The developed models yielded good prediction results toward molecules with similar scaffolds but showed less generalization ability to molecules with different scaffolds.<a onclick="showRef(event, 'ref6 ref14'); return false;" href="javascript:void(0);" class="ref ref6 ref14">(6,14)</a> One possible reason might be that the metabolism of different scaffolds was influenced by different factors; thus the consensus and limited descriptors were unable to differentiate the metabolic property of structurally diverse molecules. Here, both global and local descriptors were combined to construct the substrate/nonsubstrate classification model, which yielded much improved predictive ability to structurally diverse molecules. On the basis of this model, we next explored where the SOMs were located and then optimized our previously reported SOMs prediction model DT<sub>AOX</sub>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Optimization of SOMs Classification Model for Predicting SOMs toward hAOX</h3><div class="NLM_p">We have evaluated the DT<sub>AOX</sub> toward recently published data with known SOMs/non-SOMs information, and the DT<sub>AOX</sub> showed insufficient generalization ability to these compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S1</a>). After analyzing the predicted results of every branch of the decision tree, we observed that the Î<i>E</i><sub>inter</sub> could differentiate the true SOMs and false SOMs accurately, while the branch derived from the Steric was unsatisfactory. This result was also reflected by the statistical analysis described in previous section, where the Î<i>E</i><sub>inter</sub> was the most discriminating descriptor among all the local properties between SOMs and non-SOMs (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>), but Steric was less significant. Here, the decision tree based method was applied to optimize the previous DT<sub>AOX</sub> model (methods described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). In decision tree models, a nodeâs impurity is a function that measures the homogeneity of classes in data samples reaching the node. The smaller the degree of impurity, the more skewed the class distribution. By minimizing the gini impurity of every branch,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> we observed that combining the local exposed area of the reactive atom H adjacent to the reactive center C (H_EA) with Steric could significantly improve the performance derived from the middle range of Î<i>E</i><sub>inter</sub>. We therefore optimized the classification tree model, which will be referred to as DTN<sub>AOX</sub> hereafter (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Decision tree model DTN<sub>AOX</sub> (A) for predicting hAOX-mediated SOMs and its performance on (B) the test data set and (C) the external data set, compared with our previous DT<sub>AOX</sub> model<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and three descriptors proposed by Montefiori et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Taking H_EA into consideration, we noticed that the sensitivity score was improved significantly toward the test data set (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). For further validation, the DTN<sub>AOX</sub> was evaluated toward an external data set extracted from several different literature studies with diverse scaffolds and different experimental conditions. For this external data set, the DTN<sub>AOX</sub> model demonstrated decent performance (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) and significantly exceeded DT<sub>AOX</sub>, especially in terms of SE score. Prediction results of the methods proposed by Montefiori et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> were also benchmarked here for reference.</div><div class="NLM_p last">As shown, the SE was improved significantly with the descriptor H_EA introduced. Though the significance of the properties of the aromatic C had been demonstrated before (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S2</a>), after further analysis of these descriptors, we found that the Î<i>E</i><sub>inter</sub>, H_EA, and Steric yielded the best performance toward test data set. First of all, the thermokinetic effect was of high importance for the metabolic reaction. According to the proposed reaction mechanism aforementioned, the H atom attached to the potential metabolic site C of the compound needed to be seized by MoCo; thus the exposed area of H also had a great influence on the occurrence of the reaction. Actually, the descriptor Steric describes the global steric hindrance, which indicated whether the MoCo could access to the reaction site, and the H_EA represents the local steric influence when two reactive atoms access each other for bond breakage and formation. With reaction barrier and global and local hindrance effects considered, we might find that our optimized model has not only good generalization ability but also good interpretation ability for the mechanism of AOX-mediated reaction.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Fusing Model for Prediction of hAOX Substrate and Site of Metabolism</h3><div class="NLM_p">In the actual drug-discovery-and-optimization program, the researchers were concerned with whether the molecule is metabolized and then recognition and optimization of the SOMs came next. Thus, the DNN substrate/nonsubstrate and DTN<sub>AOX</sub> classification model were combined here to predict whether the molecule was a hAOX substrate and the corresponding SOM or SOMs for practicable purpose. Overall, the fusing model could efficiently and accurately identify the molecular liability toward hAOX-mediated biotransformation and predict the SOMs, providing more comprehensive information on the hAOX-catalyzed metabolism. The proposed fusing model flowchart is shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Flowchart of fusing model. The data sets 1 and data sets 2 contain the data with substrate/nonsubstrate and SOMs/non-SOMs labels, respectively. The red and blue circles indicate the predicted metabolic and nonmetabolic sites, respectively. The arrows indicate the metabolic sites identified in human liver cytosols.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Screen AOX Substrates in Epigenetics-Related Inhibitors Using Fusing Model</h3><div class="NLM_p">The compounds regulating epigenetic modification are currently in the early stage of clinical development. Most of their inhibitors are similar to kinase inhibitors that contain nitrogen-containing aromatic heterocyclic fragments, which may be potential hAOX substrates. In order to further verify the practicability of our model, virtual screening of hAOX substrates was conducted for compounds related to epigenetic regulation that have not been reported on hAOX metabolism. 102 epigenetics-related inhibitors containing AOX-substrate-like motif were obtained from Selleck (<a href="https://www.selleck.cn/pharmacological_epigenetics.html" class="extLink">https://www.selleck.cn/pharmacological_epigenetics.html</a>). All potential AOX metabolic sites of these epigenetics-related inhibitors were labeled by matching predefined SMARTS rules, then the fusing model was used to predict whether the molecule was a hAOX substrate and the corresponding SOMs. To validate the in silico prediction results, 15 compounds were incubated with human liver cytosol separately, and ultraperformance liquid chromatography (UPLC)/quadrupole time-of-flight mass spectrometry (Q-TOF MS) was used to analyze the incubation mixtures to assess their potential to turnover by hAOX (the 15 compoundsâ information is provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Supporting Information, Table S6</a>). A compound metabolized would display an additional peak in the total ion chromatogram upon mass spectrometric analysis of the incubation mixture, meaning there would be addition of an oxygen atom and occurrence of oxidative metabolism if the molecular ion (MH<sup>+</sup>) corresponding to the peak of the metabolite was 16 Da greater than the molecular ion of parent compound. The relationship of the observed oxidative metabolite with the parent was confirmed from their respective mass spectral fragment ions. Furthermore, the contribution of hAOX to the oxidative metabolism of a given compound incubated with human liver cytosol was confirmed by chemical inhibition studies. The effects of various molybdenum hydroxylase inhibitors on the formation of oxidative metabolite were examined. The chemical inhibitors raloxifene and hydralazine were used to selectively inhibit cytosolic AOX, whereas allopurinol was used to selectively inhibit cytosolic xanthine oxidase (XO).<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29â31)</a> If metabolism by hAOX was involved, a substantial decrease of the liquid chromatographyâmass spectrometry (LCâMS) peak area would be observed for an oxidative metabolite after adding raloxifene or hydralazine to the incubation system. In this way, the different inhibitory potential of AOX and XO selective inhibitors to the formation of the oxidative metabolite would suggest which enzyme was mainly responsible for the oxidation metabolism.</div><div class="NLM_p">On the basis of the fusing model, compounds S7276, S7577, S7832, and S2686 were predicted as substrates by DNN substrate/nonsubstrate classification model and the rest of 11 compounds were predicted as nonsubstrates for hAOX (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Then, the corresponding SOMs of four predicted substrates were predicted by using DTN<sub>AOX</sub> SOMs classification model highlighted in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, and the descriptor values are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S3</a>. To validate our in vitro experimental method, known hAOX substrates SGX523 and JNJ-38877605 and amodiaquine were evaluated with the above-mentioned human liver cytosol assay.<a onclick="showRef(event, 'ref17 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref17 ref32 ref33">(17,32,33)</a> All these positive controls displayed significant oxidative metabolite formation and high inhibition ratio of the selective AOX inhibitor, which verified the reliability of the assay (data and spectrum are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Supporting Information Figures S4 and S5 and Table S7</a>). For the 15 epigenetics-related inhibitors, only compounds S7276, S7577, S7832, S7373, S1129, and S2686 had mono-oxidized metabolites detected by LCâMS (ion extraction chromatographs of parent and their corresponding mono-oxidized metabolites are shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>). The metabolic region of these six compounds was identified based on the fragment ion information from Q-TOF MS and marked using dotted box (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). No further attempt was taken to confirm the oxidative metabolites structures. Then, to identify the molybdenum hydroxylase (AOX or XO) mainly contributing to the oxidation metabolism of these six compounds, chemical inhibitors were added to the incubation system. AOX selective inhibitors raloxifene and hydralazine inhibited over 93.9% of the oxidation metabolite formation of these compounds, while XO selective inhibitor allopurinol exhibited limited inhibition (less than 12.0%) toward them (data for inhibition study are listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and the LCâMS peak area data are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Supporting Information Tables S8 and S9</a>). In summary, these data demonstrated that AOX and not XO was the enzyme responsible for the formation of the oxidative metabolite. And accordingly, S7276, S7577, S7832, S7373, S1129, and S2686 were confirmed to be hAOX substrates.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Predicted hAOX-catalyzed SOMs for 15 epigenetics-related inhibitors using the fusing model. Red and blue circles indicate the metabolic and nonmetabolic sites, respectively, predicted by DTN<sub>AOX</sub>. The experimentally proposed metabolic region is marked using dotted box. Pred.: prediction results. Exp.: experimental results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ion extraction chromatographs of the parent compounds (left) and their corresponding mono-oxidized metabolites (right) formed in human liver cytosol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Positive product ion spectra of the parent compounds (left) and their corresponding mono-oxidized metabolites (right) formed in human liver cytosol. The <i>m</i>/<i>z</i> of parent ion was colored in purple, and the <i>m</i>/<i>z</i> of relevant product ions which were applied to identify the oxidized regions was colored in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibition Study of AOX/XO Selective Inhibitor on the Formation of Oxidative Metabolite in Human Liver Cytosol<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char=".">raloxifene (AOXÂ inhibitor)</th><th class="colsep0 rowsep0" align="center" char=".">hydralazine (AOXÂ inhibitor)</th><th class="colsep0 rowsep0" align="center" char=".">allopurinol (XOÂ inhibitor)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">inhibition ratio (%)</th><th class="colsep0 rowsep0" align="center" char=".">inhibition ratio (%)</th><th class="colsep0 rowsep0" align="center" char=".">inhibition ratio (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S7276</td><td class="colsep0 rowsep0" align="char" char=".">98.3</td><td class="colsep0 rowsep0" align="char" char=".">99.7</td><td class="colsep0 rowsep0" align="char" char=".">7.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S7577</td><td class="colsep0 rowsep0" align="char" char=".">93.9</td><td class="colsep0 rowsep0" align="char" char=".">97.3</td><td class="colsep0 rowsep0" align="char" char=".">4.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S7832</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">2.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S7373</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">7.50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S1129</td><td class="colsep0 rowsep0" align="char" char=".">99.7</td><td class="colsep0 rowsep0" align="char" char=".">99.7</td><td class="colsep0 rowsep0" align="char" char=".">9.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">S2686</td><td class="colsep0 rowsep0" align="char" char=".">99.6</td><td class="colsep0 rowsep0" align="char" char=".">99.4</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Inhibition ratio = 1 â [(peak area of oxidative metabolite in human liver cytosol with inhibitor)/(peak area of oxidative metabolite in human liver cytosol without inhibitor)].</p></div></div></div><div class="NLM_p">The molecules experimented included miscellaneous scaffolds like quinoline-, isoquinoline-, quinazoline-, quinoxaline-, and pyridine-substituted azaheterocycle ring compounds as well as bicyclic skeletons. Unlike the molecules experimented by Lepri et al., electron-donating group (EDG) and electron-withdrawing group (EWG) appeared more than once for most types of aforementioned scaffolds within our experimented molecules. The fusing model yielded an overall accuracy of 0.87, with SE = 0.67 and SP = 1.0. For the substrates predicted by DNN classification model, the predicted SOMs were in accordance with the proposed metabolic region (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Despite two wrong predictions for substrates, the fusing model showed satisfactory performance on the experimental validation.</div><div class="NLM_p">For compounds containing quinoline fragments or isoquinoline fragments such as S7276, S7577, S7832, and S1007, the model gave sound prediction results, and the corresponding SOMs of substrates were in accordance with the proposed metabolic region. As we have discussed, for quinoline-containing substrates, the Î<i>E</i><sub>inter</sub> alone could not determine the susceptibility of potential SOMs toward oxidation and the Steric always gave a higher false positive rate. Like S7276, the same instances were also discovered in our experimented molecules that Î<i>E</i><sub>inter</sub> alone was not decisive. Like site 1 in S7276, after combining H_EA with Steric, the DTN<sub>AOX</sub> model could predict the true SOMs accurately while Steric alone gave the wrong prediction. Lepri et al. have concluded that the EDG at positions 4, 6, and 7 of quinolines or isoquinolines might be substrate, and the EWG vice versa.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The same instance was also discovered in our experimental molecules S7276 and S7832 with EDG substituents, which were all susceptible to hAOX. Besides, regarding the metabolism of S7577, the EDG at position 8 also increased the susceptibility toward hAOX, which was not discovered in existing studies.</div><div class="NLM_p">Regarding compounds containing quinazoline-, quinoxaline-, and bicyclic-skeletons substituted, the metabolism was more complicated. For quinazoline-substituted molecules like S2867 and S5904, there were EDG groups at position 4 and two EWG groups at positions 6 and 7, and this kind of substitution was beyond the SMR scope summarized by Lepri et al.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The same situation was discovered in quinoxaline- and bicyclic-skeletons substituted compounds that were not monosubstituted, namely, with only one EDG or EWG substituted. Except for S1129, the fusing model still gave concerted predictive results for these heterogeneously substituted scaffolds. Lepri et al. proposed that pyridine-containing compounds are always stable to hAOX-mediated biotransformation,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> while the in vitro experimental results demonstrated that some pyridine substituted molecules are also susceptible toward hAOX. For example, S7373 with pyridine skeleton was metabolited by hAOX while the S8776 was stable, and both molecules have EWG substituted. For the last two kinds of skeletons, molecules were all correctly predicted except S7373.</div><div class="NLM_p last">Overall, these results highlighted the practicality of our model in recognizing the regioselectivity of molecules and metabolic SOMs toward hAOX. With a clear SMARTS-pattern definition of the potential SOMs, the fusing model can be used to prioritize compounds on the basis of their susceptibility toward hAOX.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52236" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52236" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">As issues caused by hAOX-catalyzed metabolism of new drug molecules continue to emerge, it is critical that medicinal scientists have reliable and efficient methodologies to identify whether a molecule is a hAOX substrate and where its metabolic site is. Unfortunately, the hAOX has quite wide substrate selectivity and diversity, which poses challenges for the predictive models. In this study, a combined model was developed based on structurally diverse molecules with homogeneously experimental data, and it yielded improved performance on predicting the metabolic propensity and explaining the metabolic regioselectivity of hAOX mediated reaction. Moreover, the improved computational model was used for screening structurally diverse drug candidates containing potential hAOX metabolic sites. The results were then experimentally validated with in vitro assays. Overall, this study provides a practical scheme to optimize hAOX metabolic profile of chemicals, which is of essence for the hit-to-lead optimization in drug development.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60337" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60337" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Potential SOMs and Tetrahedral Intermediate Defining</h3><div class="NLM_p">According to the catalytic mechanism and previously reported hAOX metabolites, we optimized the SMILES arbitrary target specification (SMARTS) strings of the two types of potential SOMs (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The code for extracting potential SOMs defined by SMARTS was implemented using Python 3.6 with RDKit version 2018.09.1.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Two Types of Potential Sites of Metabolism by hAOX</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0012.gif" alt="" id="gr13" /></img><div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Data Set Preparation and Construction</h3><div class="NLM_p">Existing metabolic data sets may show variance in the quality of chemical/biological samples, measurement methods, experimental conditions, etc. To address this problem, (1) we only used experimentally determined metabolism data, and in silico data were not considered; (2) we have carefully chosen the data validated in human liver cytosol as our main data set to reduce the inconsistency of experimental conditions; (3) we further excluded the data of the same molecule collected from different sources but with inconsistent metabolic profiles. These procedures help us increase the reliability of the data set on merging data from different sources. The hAOX-oxidated experimental data from the work published by Lepri et al. and Cruciani et al. were collected.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> The homogeneous data were used for model construction, and the number of molecules used to construct model are listed in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. For the substrate/nonsubstrate classification model, the training data set was extracted from Cruciani et al.âs work and the test data set was from Lepri et al.âs work, and the inconsistent data and molecule that failed for descriptor calculation were excluded (For detailed molecules that were excluded, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Supporting Information</a>). For SOM classification model, our previous training data set was enriched with recently published data.<a onclick="showRef(event, 'ref7 ref19'); return false;" href="javascript:void(0);" class="ref ref7 ref19">(7,19)</a> The same test data set in ref <a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> was used here for model comparison.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Data Sets Used in the Model Construction and Validation</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">data set distribution</th><th class="colsep0 rowsep0" align="center" char=".">substrate/nonsubstrate classification model (moleculesÂ perÂ dataÂ set)</th><th class="colsep0 rowsep0" align="center" char=".">SOMs classification model (sitesÂ perÂ dataÂ set)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">train data set</td><td class="colsep0 rowsep0" align="left">positive</td><td class="colsep0 rowsep0" align="char" char=".">202</td><td class="colsep0 rowsep0" align="char" char=".">45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">negative</td><td class="colsep0 rowsep0" align="char" char=".">299</td><td class="colsep0 rowsep0" align="char" char=".">128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">test data set</td><td class="colsep0 rowsep0" align="left">positive</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">negative</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">externalÂ dataÂ set</td><td class="colsep0 rowsep0" align="left">positive</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">negative</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">70</td></tr></tbody></table></div></div><div class="NLM_p last">Furthermore, we assessed the predictive ability of the classification model against an external data sets collected from the literature published recently.<a onclick="showRef(event, 'ref7 ref19 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref7 ref19 ref34 ref35">(7,19,34,35)</a> Note that there were no overlapping molecules among the training data set, test data set, and external data set for substrate/nonsubstrate classification model construction, which was the same as the construction of SOMs/non-SOMs classification model. The detailed description of the data set used for substrate/nonsubstrate model construction is provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Supporting Information</a>. And the structures with SOM information collected from newly published work is provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S4</a>. The structures of nonsubstrates collected from newly published work are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">Table S5</a>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Calculation of Descriptors</h3><div class="NLM_p">A total of 48 descriptors were calculated here. The energy changes between hAOX substrate and the tetrahedral intermediate (Î<i>E</i><sub>inter</sub>) and steric hindrance (Steric) were calculated based on the methods described by Xu et al.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The other descriptors were computed using Spartanâ18 version 1.2.0, which could be categorized as topological, physicochemical, and electronic features. <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> provides a brief description of these descriptors.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Condensed List of Descriptors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">atom-levelÂ descriptors</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ExposedÂ area</td><td class="colsep0 rowsep0" align="left">Exposed area of the reaction related atoms, describing the steric exposure of the reaction related atom</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bond order</td><td class="colsep0 rowsep0" align="left">BondÂ orderÂ ofÂ theÂ CâHÂ andÂ CâN</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Atom charge</td><td class="colsep0 rowsep0" align="left">Electrostatic potential charge of the reaction related atoms</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChemicalÂ shift</td><td class="colsep0 rowsep0" align="left">Chemical shift of the reaction related atoms</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Isotropic</td><td class="colsep0 rowsep0" align="left">Average three diagonal elements of the magnetic shielding tensor</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">molecule-levelÂ descriptors</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Electronegativity</td><td class="colsep0 rowsep0" align="left">Electronegativity of the molecule, defined as â(HOMOÂ +Â LUMO)/2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Hardness</td><td class="colsep0 rowsep0" align="left">Hardness of the molecule, defined as â(HOMOÂ âÂ LUMO)/2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">QSAR_ACC.Area</td><td class="colsep0 rowsep0" align="left">AccessibleÂ areaÂ ofÂ theÂ molecularÂ surface</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">QSAR_MIN_Elpot</td><td class="colsep0 rowsep0" align="left">Min local values of the electrostatic potential</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Dipole</td><td class="colsep0 rowsep0" align="left">Dipole moment of the molecule</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Solvation</td><td class="colsep0 rowsep0" align="left">Solvated energies of the molecule</td></tr></tbody></table></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Model Construction and Hyperparameter Optimization</h3><div class="NLM_p">Deep neural networks (DNNs) have been successfully applied in addressing drug metabolism related prediction tasks,<a onclick="showRef(event, 'ref21 ref36'); return false;" href="javascript:void(0);" class="ref ref21 ref36">(21,36)</a> in which the algorithm can be considered as a function that maps the input vector, which is the representation of a compound by its predefined features, to an output vector that represents the prediction task. For substrate/nonsubstrate classification, the deep neural network algorithm was used to predict whether the molecule was a hAOX substrate. The maximum, minimum, and average values of descriptors of all potential SOMs within each molecule were calculated, which were fed into the model as molecular level descriptors. The descriptor mapping and model construction procedures are illustrated in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Descriptor mapping and model construction flowchart. The quinoline and DNN model structure were used to elucidate the flowchart.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To optimize the performance of DNN model, the hyperparameter optimization was carried out via a grid search algorithm.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The searched hyperparameter included network architecture (layer size), learning rate, batch size, and dropout rate. The detailed hyperparameter setting is listed in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Hyperparameter Setting</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">hyperparameter</th><th class="colsep0 rowsep0" align="center">setting</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">layer size</td><td class="colsep0 rowsep0" align="left">([100,200,300,400,500,600,700,800,900,1000])</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">([100,50], [200,100], [300,150], [400,200], [500,250], [600,300], [700,350], [800,400], [900,450], [1000,500])</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">([400,200,100], [600,300,150], [800,400,200])</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">learningÂ rate</td><td class="colsep0 rowsep0" align="left">(0.00001,Â 0.00005,Â 0.0001,Â 0.0005,Â 0.001,Â 0.005,Â 0.01)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">batch size</td><td class="colsep0 rowsep0" align="left">(8, 16, 32, 64)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dropout rate</td><td class="colsep0 rowsep0" align="left">(0.1, 0.2, 0.3, 0.4, 0.5)</td></tr></tbody></table></div></div><div class="NLM_p last">For SOMs classification model, the decision-tree-based classification model was constructed with the Scikit-learn version 0.21.3 using the Classification and Regression Trees (CART) algorithm.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> CART is a nonparametric approach that can deal with both categorical and numeric dependent variables, and it constructs binary trees using the feature and threshold that yield the largest information gain at each node.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> During the model training phase, 5-fold cross validation was implemented on the training data set with F1-score as validation criterion, and the obtained models were evaluated by the following metrics:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_m001.gif" alt="" id="_i21" /></img></span><span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_m002.gif" alt="" id="_i22" /></img></span><span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_m003.gif" alt="" id="_i23" /></img></span><span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_m004.gif" alt="" id="_i24" /></img></span>TP is the number of true positives predicted by the model. TN is the number of true negatives predicted by the model. FP is the number of false positives predicted by the model. FN is the number of false negatives predicted by the model.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Human Liver Cytosol Incubations of Epigenetics Related Inhibitors</h3><div class="NLM_p last">Human liver cytosol for assays (mixed sex; pool of 50 donors; catalog no. H0610.C, lot no. 1610027; 30 males and 20 females) was purchased from Sekisui XenoTech. The incubations were carried out in 1.5 mL microcentrifuge tubes. Briefly, the incubation mixture was composed of human liver cytosol (2 mg/mL final protein concentration), 2 mM MgCl<sub>2</sub>, 10 Î¼M test compound, and 100 mM potassium phosphate buffer, pH 7.4. The total volume of each incubation was 200 Î¼L, and the final concentration of dimethyl sulfoxide (DMSO) used in the assay was 0.1% (v/v). Reactions were commenced with the addition of test compound after prewarming at 37 Â°C for 3 min in a low-speed shaking thermomixer and terminated by adding an equal volume of ice-cold acetonitrile after 3 h of incubation. Each sample was immediately mixed and subsequently centrifuged at 12â¯000 rpm for 10 min at room temperature. The supernatant was transferred into a 96-well plate for analysis by UPLC/Q-TOF MS. The control samples were prepared with no test compound or with inactivated enzymes. All samples for the experiment were performed in triplicate.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> UPLC/Q-TOF MS Analysis</h3><div class="NLM_p last">The UPLC/Q-TOF MS system consisted of a Waters Acquity UPLC system coupled to a Waters Synapt G2 Q-TOF MS. Chromatography was performed on a Waters Acquity UPLC HSS T3 column (2.1 mm Ã 100 mm, 1.8 Î¼m; maintained at 45 Â°C) using a mobile phase that consisted of solvent A (2.5 mM aqueous ammonium acetate containing 0.1% formic acid) and solvent B (methanol or acetonitrile/methanol, 9:1, v/v). The mobile phase was delivered at 0.4 mL/min in a gradient manner, e.g., 0â1 min, at 10% solvent B; 1â6.5 min, from 10% to 55% solvent B; 6.5â7.5 min, from 55% to 95% solvent B; 7.5â9.0 min, at 95% solvent B; and 9.0â10.0 min, at 10% solvent B. A 7 Î¼L aliquot of the sample in the 96-well plate was injected into the system. The Synapt G2 Q-TOF MS was operated in resolution mode with electrospray ionization. The capillary and cone voltages were set at 3 kV and 30 V. The source and desolvation temperatures were set at 120 and 400 Â°C, respectively. Nitrogen was applied as the cone gas (50 L/h) and desolvation gas (800 L/h). All data were acquired in positive ionization mode. Mass was externally calibrated with a 0.5 mM sodium formate solution at 25 Î¼L/min over a range of <i>m</i>/<i>z</i> 50â1000 and corrected during acquisition using an external reference (LockSpray) consisting of a 40 ng/mL solution of leucine enkephalin infused at a flow rate of 10 Î¼L/min via a LockSpray interface, generating a reference ion for positive ion mode (<i>m</i>/<i>z</i> = 556.2771) to ensure accuracy during the analysis. Metabolites were characterized by MS full scan and product ion scan that both ranged from <i>m</i>/<i>z</i> 50 to 1000. Product ion scan was obtained from either the MS<sup>E</sup> approach or the dedicated MS/MS experiment. For the MS<sup>E</sup> approach, accurate mass data were acquired by two separate scan functions programmed with independent collision energies (e.g., trap collision energy in function 1 was 2 V, and an energy ramp of 15â25 V was used in function 2). With the MS/MS experiment, for example, the trap collision energy was ramped from 10 to 16 V, whereas the transfer collision energy was ramped from 14 to 22 V. All the instrument control and data acquisition were achieved using Waters MassLynx software (version 4.1).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Molybdenum Hydroxylase Inhibition Study</h3><div class="NLM_p last">Aldehyde oxidase selective inhibitors raloxifene, hydralazine, and xanthine oxidase selective inhibitor allopurinol were used in the study to further investigate the involvement of human aldehyde oxidase in the oxidative metabolism of the test compounds. The incubation sample contained human liver cytosol (2 mg/mL final protein concentration), 2 mM MgCl<sub>2</sub>, 10 Î¼M test compound, 100 Î¼M inhibitor, and 100 mM potassium phosphate buffer, pH 7.4. The test compounds and inhibitors were dissolved in DMSO, respectively, and the final concentration of DMSO in each incubation (200 Î¼L total volume) was 1.1% (v/v). The mixtures were preincubated for 3 min at 37 Â°C, and each reaction was initiated by adding test compound. After incubation at 37 Â°C for 3 h, reactions were terminated by adding 200 Î¼L of ice-cold acetonitrile. The mixtures were centrifuged, and 7 Î¼L of the supernatant was injected into the UPLC/Q-TOF MS. Controls without inhibitors were also prepared. Experiments were performed in triplicate.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01895" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21543" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21543" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01895?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01895</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures and tables illustrating the sample distribution according to the applicability domain defined by DT<sub>AOX</sub> (Figure S1); relationship of Î<i>E</i><sub>inter</sub> versus the ESP charge and the chemical shift of the reactive carbon (Figure S2); prediction results of molecules in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> by DNN substrate/nonsubstrate classification model (Figure S3); ion extraction chromatographs of the positive control (Figure S4); positive product ion spectra of the positive control (Figure S5); predictive performance of DT<sub>AOX</sub> on newly published data (Table S1); distributions of descriptors between SOMs and nonSOMs (Table S2); descriptor values of four substrates predicted by fusing model (Table S3); substrates with SOM information collected from newly published work (Table S4); nonsubstratesâ structures collected from newly published work (Table S5); structures used for experimental validation (Table S6); inhibition study of oxidative metabolites for positive control incubated in human liver cytosol with or without human AOX chemical inhibitors (Table S7); LCâMS peak areas of oxidative metabolites for compounds incubated in human liver cytosol with or without human AOX chemical inhibitors (Table S8); LCâMS peak areas of oxidative metabolites for compounds incubated in human liver cytosol with or without human XO chemical inhibitor (Table S9) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and other information used in model construction (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf">jm9b01895_si_001.pdf (633.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_002.csv">jm9b01895_si_002.csv (66.47 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01895" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06847" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06847" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing 210023, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, and School of Life
Science and Technology, ShanghaiTech University, Shanghai 200031, China</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fd959197949c939abd8e949090d39c9ed39e93"><span class="__cf_email__" data-cfemail="197175737078777e596a70747437787a377a77">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingyue Zheng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan District, Beijing 100049, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3323-3092" title="Orcid link">http://orcid.org/0000-0002-3323-3092</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#036e7a796b666d6443706a6e6e2d62602d606d"><span class="__cf_email__" data-cfemail="c5a8bcbfada0aba285b6aca8a8eba4a6eba6ab">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jihui Zhao</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No.19(A) Yuquan Road, Shijingshan District, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rongrong Cui</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lihao Wang</span> - <span class="hlFld-Affiliation affiliation">Gillings
School of Public Health, University of North
Carolina at Chapel Hill, 135 Dauer Drive, Chapel Hill, North Carolina 27599, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingjia Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zunyun Fu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing 210023, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyu Ding</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chen Cui</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tianbiao Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xutong Li</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan Xu</span> - <span class="hlFld-Affiliation affiliation">Shanghai
EnnovaBio Pharmaceuticals Co., Ltd.,
Room 404, Building 2, Lane 720, Cailun Road, Pudong New Area, Shanghai 200120, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaixian Chen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Nanjing University of Chinese
Medicine, 138 Xianlin
Avenue, Qixia District, Nanjing 210023, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">Shanghai
Institute for Advanced Immunochemical Studies, and School of Life
Science and Technology, ShanghaiTech University, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaomin Luo</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zu Chong Zhi Road, Shanghai 201203, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0426-3417" title="Orcid link">http://orcid.org/0000-0003-0426-3417</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.Z. and R.C. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The DNN model is available at github (<a href="https://github.com/AOXpredict/" class="extLink">https://github.com/AOXpredict/</a>).</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21037" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21037" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We gratefully acknowledge financial support from the National Natural Science Foundation of China (Grant 81773634 to M.Z), National Science & Technology Major Project âKey New Drug Creation and Manufacturing Programâ, China (Grant 2018ZX09711002 to H.J.), and âPersonalized MedicinesâMolecular Signature-Based Drug Discovery and Developmentâ, Strategic Priority Research Program of the Chinese Academy of Sciences (Grant XDA12050201 to M.Z.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AOX</td><td class="NLM_def"><p class="first last">aldehyde oxidase</p></td></tr><tr><td class="NLM_term">hAOX</td><td class="NLM_def"><p class="first last">human AOX</p></td></tr><tr><td class="NLM_term">SOM</td><td class="NLM_def"><p class="first last">site of metabolism</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">MoCo</td><td class="NLM_def"><p class="first last">molybdenum cofactor</p></td></tr><tr><td class="NLM_term">SMR</td><td class="NLM_def"><p class="first last">structureâmetabolism relationship</p></td></tr><tr><td class="NLM_term">ESP</td><td class="NLM_def"><p class="first last">electrostatic potential</p></td></tr><tr><td class="NLM_term">LUMO</td><td class="NLM_def"><p class="first last">lowest unoccupied molecular orbital</p></td></tr><tr><td class="NLM_term"><i>E</i><sub>LUMO</sub></td><td class="NLM_def"><p class="first last">energy of LUMO</p></td></tr><tr><td class="NLM_term">AD</td><td class="NLM_def"><p class="first last">applicability domain</p></td></tr><tr><td class="NLM_term">SE</td><td class="NLM_def"><p class="first last">sensitivity score</p></td></tr><tr><td class="NLM_term">SP</td><td class="NLM_def"><p class="first last">specificity score</p></td></tr><tr><td class="NLM_term">Acc</td><td class="NLM_def"><p class="first last">accuracy</p></td></tr><tr><td class="NLM_term">HOMO</td><td class="NLM_def"><p class="first last">highest occupied molecular orbital</p></td></tr><tr><td class="NLM_term">DNN</td><td class="NLM_def"><p class="first last">deep neural network</p></td></tr><tr><td class="NLM_term">DT</td><td class="NLM_def"><p class="first last">decision tree</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultraperformance liquid chromatography</p></td></tr><tr><td class="NLM_term">Q-TOF MS</td><td class="NLM_def"><p class="first last">quadrupole time-of-flight mass spectrometry</p></td></tr><tr><td class="NLM_term">LCâMS</td><td class="NLM_def"><p class="first last">liquid chromatographyâmass spectrometry</p></td></tr><tr><td class="NLM_term">XO</td><td class="NLM_def"><p class="first last">xanthine oxidase</p></td></tr><tr><td class="NLM_term">EDG</td><td class="NLM_def"><p class="first last">electron-donating group</p></td></tr><tr><td class="NLM_term">EWG</td><td class="NLM_def"><p class="first last">electron-withdrawing group</p></td></tr><tr><td class="NLM_term">SMARTS</td><td class="NLM_def"><p class="first last">SMILES arbitrary target specification</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40063" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40063" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckworth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span> <span> </span><span class="NLM_article-title">Medicinal Chemistry Approaches to Avoid Aldehyde Oxidase Metabolism</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2856</span>â <span class="NLM_lpage">2860</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2012.02.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1016%2Fj.bmcl.2012.02.069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22429467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFSitbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2856-2860&issue=8&author=D.+C.+Prydeauthor=T.+D.+Tranauthor=P.+Jonesauthor=J.+Duckworthauthor=M.+Howardauthor=I.+Gardnerauthor=R.+Hylandauthor=R.+Websterauthor=T.+Wenhamauthor=S.+Bagalauthor=K.+Omotoauthor=R.+P.+Schneiderauthor=J.+Lin&title=Medicinal+Chemistry+Approaches+to+Avoid+Aldehyde+Oxidase+Metabolism&doi=10.1016%2Fj.bmcl.2012.02.069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry approaches to avoid aldehyde oxidase metabolism</span></div><div class="casAuthors">Pryde, David C.; Tran, Thien-Duc; Jones, Peter; Duckworth, Jonathan; Howard, Martin; Gardner, Iain; Hyland, Ruth; Webster, Rob; Wenham, Tracey; Bagal, Sharan; Omoto, Kiyoyuki; Schneider, Richard P.; Lin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2856-2860</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AO) is a molybdenum-contg. enzyme distributed throughout the animal kingdom and capable of metabolising a wide range of aldehydes and N-heterocyclic compds.  Although metab. by this enzyme in man is recognized to have significant clin. impact where human AO activity was not predicted by screening in preclin. species, there is very little reported literature offering real examples where drug discoverers have successfully designed away from AO oxidn.  This article reports on some strategies adopted in the Pfizer TLR7 agonist program to successfully switch off AO metab. that was seen principally in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZKVEe9ZUzMbVg90H21EOLACvtfcHk0lguQ_AYLj2C6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFSitbs%253D&md5=4c4c68b64f1c7013ad978408add0b09a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.069%26sid%3Dliteratum%253Aachs%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BD.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DDuckworth%26aufirst%3DJ.%26aulast%3DHoward%26aufirst%3DM.%26aulast%3DGardner%26aufirst%3DI.%26aulast%3DHyland%26aufirst%3DR.%26aulast%3DWebster%26aufirst%3DR.%26aulast%3DWenham%26aufirst%3DT.%26aulast%3DBagal%26aufirst%3DS.%26aulast%3DOmoto%26aufirst%3DK.%26aulast%3DSchneider%26aufirst%3DR.%2BP.%26aulast%3DLin%26aufirst%3DJ.%26atitle%3DMedicinal%2520Chemistry%2520Approaches%2520to%2520Avoid%2520Aldehyde%2520Oxidase%2520Metabolism%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26issue%3D8%26spage%3D2856%26epage%3D2860%26doi%3D10.1016%2Fj.bmcl.2012.02.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sodhi, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halladay, J. S.</span></span> <span> </span><span class="NLM_article-title">A Novel Reaction Mediated by Human Aldehyde Oxidase: Amide Hydrolysis of GDC-0834</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">908</span>â <span class="NLM_lpage">915</span>, <span class="refDoi">Â DOI: 10.1124/dmd.114.061804</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.114.061804" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=25845827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlCrtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=908-915&issue=6&author=J.+K.+Sodhiauthor=S.+Wongauthor=D.+S.+Kirkpatrickauthor=L.+Liuauthor=S.+C.+Khojastehauthor=C.+E.+Hopauthor=J.+T.+Barrauthor=J.+P.+Jonesauthor=J.+S.+Halladay&title=A+Novel+Reaction+Mediated+by+Human+Aldehyde+Oxidase%3A+Amide+Hydrolysis+of+GDC-0834&doi=10.1124%2Fdmd.114.061804"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A novel reaction mediated by human aldehyde oxidase: amide hydrolysis of GDC-0834</span></div><div class="casAuthors">Sodhi, Jasleen K.; Wong, Susan; Kirkpatrick, Donald S.; Liu, Lichuan; Khojasteh, S. Cyrus; Hop, Cornelis E. C. A.; Barr, John T.; Jones, Jeffrey P.; Halladay, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">908-915</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">GDC-0834, a Bruton's tyrosine kinase inhibitor investigated as a potential treatment of rheumatoid arthritis, was previously reported to be extensively metabolized by amide hydrolysis such that no measurable levels of this compd. were detected in human circulation after oral administration.  In vitro studies in human liver cytosol detd. that GDC-0834 (R)-N-(3-(6-(4-(1,4-dimethyl-3-oxopiperazin-2-yl)phenylamino)-4-methyl-5-oxo-4,5-dihydropyrazin-2-yl)-2-methylphenyl)-4,5,6,7-tetrahydrobenzo[b]thiophene-2-carboxamide was rapidly hydrolyzed with a CLint of 0.511 mL/min per mg of protein.  Aldehyde oxidase (AO) and carboxylesterase (CES) were putatively identified as the enzymes responsible after cytosolic fractionation and mass spectrometry-proteomics anal. of the enzymically active fractions.  Results were confirmed by a series of kinetic expts. with inhibitors of AO, CES, and xanthine oxidase (XO), which implicated AO and CES, but not XO, as mediating GDC-0834 amide hydrolysis.  Further supporting the interaction between GDC-0834 and AO, GDC-0834 was shown to be a potent reversible inhibitor of six known AO substrates with IC50 values ranging from 0.86 to 1.87 Î¼M.  Addnl., in silico modeling studies suggest that GDC-0834 is capable of binding in the active site of AO with the amide bond of GDC-0834 near the molybdenum cofactor (MoCo), orientated in such a way to enable potential nucleophilic attack on the carbonyl of the amide bond by the hydroxyl of MoCo.  Together, the in vitro and in silico results suggest the involvement of AO in the amide hydrolysis of GDC-0834.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWAO_iWKyUVLVg90H21EOLACvtfcHk0lguQ_AYLj2C6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlCrtrw%253D&md5=4cf67605bc094945de8229be5c9cacae</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1124%2Fdmd.114.061804&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.114.061804%26sid%3Dliteratum%253Aachs%26aulast%3DSodhi%26aufirst%3DJ.%2BK.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DKirkpatrick%26aufirst%3DD.%2BS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DHop%26aufirst%3DC.%2BE.%26aulast%3DBarr%26aufirst%3DJ.%2BT.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26aulast%3DHalladay%26aufirst%3DJ.%2BS.%26atitle%3DA%2520Novel%2520Reaction%2520Mediated%2520by%2520Human%2520Aldehyde%2520Oxidase%253A%2520Amide%2520Hydrolysis%2520of%2520GDC-0834%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26issue%3D6%26spage%3D908%26epage%3D915%26doi%3D10.1124%2Fdmd.114.061804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manevski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecomte, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toselli, F.</span></span> <span> </span><span class="NLM_article-title">Metabolism by Aldehyde Oxidase: Drug Design and Complementary Approaches to Challenges in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">10955</span>â <span class="NLM_lpage">10994</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b00875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10955-10994&issue=24&author=N.+Manevskiauthor=L.+Kingauthor=W.+R.+Pittauthor=F.+Lecomteauthor=F.+Toselli&title=Metabolism+by+Aldehyde+Oxidase%3A+Drug+Design+and+Complementary+Approaches+to+Challenges+in+Drug+Discovery&doi=10.1021%2Facs.jmedchem.9b00875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism by Aldehyde Oxidase: Drug Design and Complementary Approaches to Challenges in Drug Discovery</span></div><div class="casAuthors">Manevski, Nenad; King, Lloyd; Pitt, William R.; Lecomte, Fabien; Toselli, Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10955-10994</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Aldehyde oxidase (AO) catalyzes oxidns. of azaheterocycles and aldehydes, amide hydrolysis, and diverse redns.  AO substrates are rare among marketed drugs and many candidates failed due to poor pharmacokinetics, interspecies differences, and adverse effects.  As most issues arise from complex and poorly understood AO biol., an effective soln. is to stop or decrease AO metab.  This perspective focuses on rational drug design approaches to modulate AO-mediated metab. in drug discovery.  AO biol. aspects are also covered, as they are complementary to chem. design and important when selecting the exptl. system for risk assessment.  The authors' recommendation is an early consideration of AO-mediated metab. supported by computational and in vitro exptl. methods, but not an automatic avoidance of AO structural flags, many of which are versatile and valuable building blocks.  Preferably, consideration of AO mediated metab. should be part of the multiparametric drug-optimization process, with the goal to improve overall drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMW2Yo9GpLObVg90H21EOLACvtfcHk0lhgmJYNYgFy6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrbP&md5=7fd30c75ea2190c0fc74c816feef9ba9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00875%26sid%3Dliteratum%253Aachs%26aulast%3DManevski%26aufirst%3DN.%26aulast%3DKing%26aufirst%3DL.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DLecomte%26aufirst%3DF.%26aulast%3DToselli%26aufirst%3DF.%26atitle%3DMetabolism%2520by%2520Aldehyde%2520Oxidase%253A%2520Drug%2520Design%2520and%2520Complementary%2520Approaches%2520to%2520Challenges%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D24%26spage%3D10955%26epage%3D10994%26doi%3D10.1021%2Facs.jmedchem.9b00875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paragas, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joswig-Jones, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">The Two Faces of Aldehyde Oxidase: Oxidative and Reductive Transformations of 5-Nitroquinoline</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">210</span>â <span class="NLM_lpage">217</span>, <span class="refDoi">Â DOI: 10.1016/j.bcp.2017.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1016%2Fj.bcp.2017.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28888950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKltbfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2017&pages=210-217&author=E.+M.+Paragasauthor=S.+C.+Humphreysauthor=J.+Minauthor=C.+A.+Joswig-Jonesauthor=J.+P.+Jones&title=The+Two+Faces+of+Aldehyde+Oxidase%3A+Oxidative+and+Reductive+Transformations+of+5-Nitroquinoline&doi=10.1016%2Fj.bcp.2017.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The two faces of aldehyde oxidase: Oxidative and reductive transformations of 5-nitroquinoline</span></div><div class="casAuthors">Paragas, Erickson M.; Humphreys, Sara C.; Min, Joshua; Joswig-Jones, Carolyn A.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">210-217</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a cytosolic enzyme responsible for the metab. of some drugs and drug candidates.  AOX catalyzes the oxidative hydroxylation of substrates including several aliph. and arom. aldehydes, and nitrogen-contg. heterocyclic compds.  AOX is also reported to catalyze the reductive metab. of nitro-compds., N-oxides, sulfoxides, isoxazoles, isothiazoles, nitrite and hydroxamic acids.  These reductive transformations are not well understood and are generally believed to only occur at low oxygen concns.  In this study, we used 5-nitroquinoline (5NQ) as a substrate to further understand both the oxidative and the reductive transformations catalyzed by AOX.  In vitro reaction of 5NQ with AOX under aerobic conditions generated the oxidized (2-oxo-5-nitroquinoline, 2-oxo-5NQ), the reduced (5-aminoquinoline, 5AQ) and the oxidized/reduced (2-oxo-5-aminoquinoline, 2-oxo-5AQ) metabolites.  Interestingly, in human liver cytosol, co-incubation of 5NQ and known AOX oxidative substrates DACA and phthalazine significantly increased the yield of the reduced metabolite, while oxidized metabolites prodn. decreased.  These data indicate that 5NQ can be reduced at atm. oxygen concns. and that the reductive transformation occurs at a second site that is kinetically distinct from the oxidative site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEP7SDjFQWZ7Vg90H21EOLACvtfcHk0lhgmJYNYgFy6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKltbfE&md5=4a2acff32f835e365bdb5a079c291eab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DParagas%26aufirst%3DE.%2BM.%26aulast%3DHumphreys%26aufirst%3DS.%2BC.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DJoswig-Jones%26aufirst%3DC.%2BA.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520Two%2520Faces%2520of%2520Aldehyde%2520Oxidase%253A%2520Oxidative%2520and%2520Reductive%2520Transformations%2520of%25205-Nitroquinoline%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D145%26spage%3D210%26epage%3D217%26doi%3D10.1016%2Fj.bcp.2017.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerny, M. A.</span></span> <span> </span><span class="NLM_article-title">Prevalence of Non-Cytochrome P450-Mediated Metabolism in Food and Drug Administration-Approved Oral and Intravenous Drugs: 2006â2015</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1246</span>â <span class="NLM_lpage">1252</span>, <span class="refDoi">Â DOI: 10.1124/dmd.116.070763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.116.070763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=27084892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2gurrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1246-1252&issue=8&author=M.+A.+Cerny&title=Prevalence+of+Non-Cytochrome+P450-Mediated+Metabolism+in+Food+and+Drug+Administration-Approved+Oral+and+Intravenous+Drugs%3A+2006%E2%80%932015&doi=10.1124%2Fdmd.116.070763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of non-cytochrome P450-mediated metabolism in food and drug administration-approved oral and intravenous drugs: 2006-2015</span></div><div class="casAuthors">Cerny, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1246-1252</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In recent years, claims of increased involvement of non-cytochrome P 450 (non-P 450) enzymes in the metab. of drugs have appeared in the literature.  However, no temporal summaries of the contribution of non-P 450 enzymes to the metab. of drugs have been published.  Using data from human radiolabeled absorption, distribution, metab., and excretion studies available for a set of 125 orally or i.v. administered small-mol. drugs approved by the United States Food and Drug Administration from 2006 to 2015, the contributions of P 450 and non-P 450 enzymes to the formation of major metabolites (â¥10% of dose) were assessed and tabulated.  Over this time frame, the involvement of P 450 vs. non-P 450 enzymes in the formation of major metabolites is compared, and the individual non-P 450 enzymes responsible are described.  This anal. indicates that non-P 450 enzymes contribute significantly to the metab. of the 125 drugs analyzed.  Approx. 30% of the metab. of these drugs is carried out by non-P 450 enzymes, with the predominant non-P 450 enzymes identified being glucuronosyltransferases (11.7%), hydrolases (10.8%), carbonyl reductases (2.4%), and aldehyde oxidase (1.1%).  Although significant, the relative contribution of non-P 450 enzymes to drug metab. does not appear to have increased dramatically over the last 10 years.  As the current evaluation involves drugs which emerged from the discovery phase >10 years ago, this evaluation may not reflect the current or evolving situation in some research organizations; therefore, addnl. monitoring and assessment of the involvement of non-P 450 enzymes in the metab. of drugs will be conducted in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0bkJkoG1VbVg90H21EOLACvtfcHk0lhgmJYNYgFy6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2gurrI&md5=7d59ddc9f2467feef2e9a46350c47df9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.070763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.070763%26sid%3Dliteratum%253Aachs%26aulast%3DCerny%26aufirst%3DM.%2BA.%26atitle%3DPrevalence%2520of%2520Non-Cytochrome%2520P450-Mediated%2520Metabolism%2520in%2520Food%2520and%2520Drug%2520Administration-Approved%2520Oral%2520and%2520Intravenous%2520Drugs%253A%25202006%25E2%2580%25932015%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26issue%3D8%26spage%3D1246%26epage%3D1252%26doi%3D10.1124%2Fdmd.116.070763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: A Combined Computational and Experimental Study</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2973</span>â <span class="NLM_lpage">2982</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjslyitLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2973-2982&issue=7&author=Y.+Xuauthor=L.+Liauthor=Y.+Wangauthor=J.+Xingauthor=L.+Zhouauthor=D.+Zhongauthor=X.+Luoauthor=H.+Jiangauthor=K.+Chenauthor=M.+Zhengauthor=P.+Dengauthor=X.+Chen&title=Aldehyde+Oxidase+Mediated+Metabolism+in+Drug-like+Molecules%3A+A+Combined+Computational+and+Experimental+Study&doi=10.1021%2Facs.jmedchem.7b00019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: A Combined Computational and Experimental Study</span></div><div class="casAuthors">Xu, Yuan; Li, Liang; Wang, Yulan; Xing, Jing; Zhou, Lei; Zhong, Dafang; Luo, Xiaomin; Jiang, Hualiang; Chen, Kaixian; Zheng, Mingyue; Deng, Pan; Chen, Xiaoyan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2973-2982</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is an important drug-metabolizing enzyme.  However, the current in vitro models for evaluating AOX metab. are sometimes misleading, and preclin. animal models generally fail to predict human AOX-mediated metab.  In this study, the authors report a combined computational and exptl. investigation of drug-like mols. that are potential aldehyde oxidase substrates, of which multiple sites of metab. (SOMs) mediated by AOX and their preferences for the reaction can be unambiguously identified.  In addn., the proposed strategy was used to evaluate the metab. of newly designed c-Met inhibitors, and a success switch-off of AOX metab. was obsd.  Overall, this study provide useful information to guide lead optimization and drug discovery based on AOX-mediated metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPvn1B_ge-grVg90H21EOLACvtfcHk0lhgmJYNYgFy6A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjslyitLc%253D&md5=2cb27d8a575dacef529f20cfe32cf8dc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00019%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DAldehyde%2520Oxidase%2520Mediated%2520Metabolism%2520in%2520Drug-like%2520Molecules%253A%2520A%2520Combined%2520Computational%2520and%2520Experimental%2520Study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D7%26spage%3D2973%26epage%3D2982%26doi%3D10.1021%2Facs.jmedchem.7b00019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y. J.</span></span> <span> </span><span class="NLM_article-title">Metabolism of c-Met Kinase Inhibitors Containing Quinoline by Aldehyde Oxidase, Electron Donating, and Steric Hindrance Effect</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1847</span>â <span class="NLM_lpage">1855</span>, <span class="refDoi">Â DOI: 10.1124/dmd.118.081919</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.118.081919" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=30209037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1Onsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=1847-1855&issue=12&author=J.+W.+Zhangauthor=W.+Xiaoauthor=Z.+T.+Gaoauthor=Z.+T.+Yuauthor=J.+Y.+J.+Zhang&title=Metabolism+of+c-Met+Kinase+Inhibitors+Containing+Quinoline+by+Aldehyde+Oxidase%2C+Electron+Donating%2C+and+Steric+Hindrance+Effect&doi=10.1124%2Fdmd.118.081919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of c-Met kinase inhibitors containing quinoline by aldehyde oxidase, electron donating, and steric hindrance effect</span></div><div class="casAuthors">Zhang, Jiang Wei; Xiao, Wen; Gao, Zhen Ting; Yu, Zheng Tian; Zhang, Ji Yue Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1847-1855</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Some quinoline-contg. c-Met kinase inhibitors are aldehyde oxidase (AO) substrates. 3-Substituted quinoline triazolopyridine analogs were synthesized to understand the electron-donating and steric hindrance effects on AO-mediated metab.  Metabolic stability studies for these quinoline analogs were carried out in liver cytosol from mice, rats, cynomolgus monkeys, and humans.  Several 3-N-substituted analogs were found to be unstable in monkey liver cytosolic incubations (half-life, <10 min), and five of them (63, 53, 51, 11, and 71) were chosen for addnl. mechanistic studies.  Mono-oxygenation on the quinoline ring was identified by liq. chromatog. tandem mass spectrometry.  Metabolite formation was inhibited by the AO inhibitors menadione and raloxifene, but not by the xanthine oxidase inhibitor allopurinol.  It was found that small electron-donating groups at the 3-quinoline moiety made the analogs more susceptible to AO metab., whereas large 3-substituents could reverse the trend.  Although species differences were obsd., this trend was applicable to all species tested.  Small electron-donating substituents at the 3-quinoline moiety increased both affinity (decreased Michaelis const.) and Vmax max. velocity toward AO in kinetic studies, whereas large substituents decreased both parameters probably as a result of steric hindrance.  Based on our anal., a common structural feature with high AO liability was proposed.  Our finding could provide useful information for chemists to minimize potential AO liability when designing quinoline analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkGiQQGWoG1bVg90H21EOLACvtfcHk0lgKuB0iVwPXXw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1Onsbk%253D&md5=87b1321378c2b70c2e2490e23263b3fa</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1124%2Fdmd.118.081919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.118.081919%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%2BW.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DZ.%2BT.%26aulast%3DYu%26aufirst%3DZ.%2BT.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%2BJ.%26atitle%3DMetabolism%2520of%2520c-Met%2520Kinase%2520Inhibitors%2520Containing%2520Quinoline%2520by%2520Aldehyde%2520Oxidase%252C%2520Electron%2520Donating%252C%2520and%2520Steric%2520Hindrance%2520Effect%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2018%26volume%3D46%26issue%3D12%26spage%3D1847%26epage%3D1855%26doi%3D10.1124%2Fdmd.118.081919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trincao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvalho, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garattini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimkuhler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romao, M. J.</span></span> <span> </span><span class="NLM_article-title">The First Mammalian Aldehyde Oxidase Crystal Structure: Insights into Substrate Specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">40690</span>â <span class="NLM_lpage">40702</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M112.390419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1074%2Fjbc.M112.390419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=23019336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslantr7L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=40690-40702&issue=48&author=C.+Coelhoauthor=M.+Mahroauthor=J.+Trincaoauthor=A.+T.+Carvalhoauthor=M.+J.+Ramosauthor=M.+Teraoauthor=E.+Garattiniauthor=S.+Leimkuhlerauthor=M.+J.+Romao&title=The+First+Mammalian+Aldehyde+Oxidase+Crystal+Structure%3A+Insights+into+Substrate+Specificity&doi=10.1074%2Fjbc.M112.390419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The First Mammalian Aldehyde Oxidase Crystal Structure</span></div><div class="casAuthors">Coelho, Catarina; Mahro, Martin; Trincao, Jose; Carvalho, Alexandra T. P.; Ramos, Maria Joao; Terao, Mineko; Garattini, Enrico; Leimkuehler, Silke; Romao, Maria Joao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">40690-40702</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aldehyde oxidases (AOXs) are homodimeric proteins belonging to the xanthine oxidase family of molybdenum-contg. enzymes.  Each 150-kDa monomer contains a FAD redox cofactor, two spectroscopically distinct [2Fe-2S] clusters, and a molybdenum cofactor located within the protein active site.  AOXs are characterized by broad range substrate specificity, oxidizing different aldehydes and arom. N-heterocycles.  Despite increasing recognition of its role in the metab. of drugs and xenobiotics, the physiol. function of the protein is still largely unknown.  We have crystd. and solved the crystal structure of mouse liver aldehyde oxidase 3 to 2.9 Ã.  This is the first mammalian AOX whose structure has been solved.  The structure provides important insights into the protein active center and further evidence on the catalytic differences characterizing AOX and xanthine oxidoreductase.  The mouse liver aldehyde oxidase 3 three-dimensional structure combined with kinetic, mutagenesis data, mol. docking, and mol. dynamics studies make a decisive contribution to understand the mol. basis of its rather broad substrate specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJhbcCiD1gB7Vg90H21EOLACvtfcHk0lgKuB0iVwPXXw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslantr7L&md5=df014384e84736af9a937fd383f24523</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.390419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.390419%26sid%3Dliteratum%253Aachs%26aulast%3DCoelho%26aufirst%3DC.%26aulast%3DMahro%26aufirst%3DM.%26aulast%3DTrincao%26aufirst%3DJ.%26aulast%3DCarvalho%26aufirst%3DA.%2BT.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26aulast%3DTerao%26aufirst%3DM.%26aulast%3DGarattini%26aufirst%3DE.%26aulast%3DLeimkuhler%26aufirst%3DS.%26aulast%3DRomao%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520First%2520Mammalian%2520Aldehyde%2520Oxidase%2520Crystal%2520Structure%253A%2520Insights%2520into%2520Substrate%2520Specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26issue%3D48%26spage%3D40690%26epage%3D40702%26doi%3D10.1074%2Fjbc.M112.390419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos-Silva, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimkuhler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romao, M. J.</span></span> <span> </span><span class="NLM_article-title">Structural Insights Into Xenobiotic and Inhibitor Binding to Human Aldehyde Oxidase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">779</span>â <span class="NLM_lpage">783</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1038%2Fnchembio.1895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=26322824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlylsrnJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=779-783&issue=10&author=C.+Coelhoauthor=A.+Fotiauthor=T.+Hartmannauthor=T.+Santos-Silvaauthor=S.+Leimkuhlerauthor=M.+J.+Romao&title=Structural+Insights+Into+Xenobiotic+and+Inhibitor+Binding+to+Human+Aldehyde+Oxidase&doi=10.1038%2Fnchembio.1895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into xenobiotic and inhibitor binding to human aldehyde oxidase</span></div><div class="casAuthors">Coelho, Catarina; Foti, Alessandro; Hartmann, Tobias; Santos-Silva, Teresa; Leimkuhler, Silke; Romao, Maria Joao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">779-783</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a xanthine oxidase (XO)-related enzyme with emerging importance due to its role in the metab. of drugs and xenobiotics.  We report the first crystal structures of human AOX1, substrate free (2.6-Ã resoln.) and in complex with the substrate phthalazine and the inhibitor thioridazine (2.7-Ã resoln.).  Anal. of the protein active site combined with steady-state kinetic studies highlight the unique features, including binding and substrate orientation at the active site, that characterize human AOX1 as an important drug-metabolizing enzyme.  Structural anal. of the complex with the noncompetitive inhibitor thioridazine revealed a new, unexpected and fully occupied inhibitor-binding site that is structurally conserved among mammalian AOXs and XO.  The new structural insights into the catalytic and inhibition mechanisms of human AOX that we now report will be of great value for the rational anal. of clin. drug interactions involving inhibition of AOX1 and for the prediction and design of AOX-stable putative drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbH6qUJJ5AMbVg90H21EOLACvtfcHk0lgKuB0iVwPXXw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlylsrnJ&md5=2053b7ec132f3e4ed7a82d011a46072a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1895%26sid%3Dliteratum%253Aachs%26aulast%3DCoelho%26aufirst%3DC.%26aulast%3DFoti%26aufirst%3DA.%26aulast%3DHartmann%26aufirst%3DT.%26aulast%3DSantos-Silva%26aufirst%3DT.%26aulast%3DLeimkuhler%26aufirst%3DS.%26aulast%3DRomao%26aufirst%3DM.%2BJ.%26atitle%3DStructural%2520Insights%2520Into%2520Xenobiotic%2520and%2520Inhibitor%2520Binding%2520to%2520Human%2520Aldehyde%2520Oxidase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D10%26spage%3D779%26epage%3D783%26doi%3D10.1038%2Fnchembio.1895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Torres, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzekwa, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMasters, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fandozzi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">Use of Density Functional Calculations to Predict the Regioselectivity of Drugs and Molecules Metabolized by Aldehyde Oxidase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">4642</span>â <span class="NLM_lpage">4647</span>, <span class="refDoi">Â DOI: 10.1021/jm0703690</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0703690" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsVKnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=4642-4647&issue=19&author=R.+A.+Torresauthor=K.+R.+Korzekwaauthor=D.+R.+McMastersauthor=C.+M.+Fandozziauthor=J.+P.+Jones&title=Use+of+Density+Functional+Calculations+to+Predict+the+Regioselectivity+of+Drugs+and+Molecules+Metabolized+by+Aldehyde+Oxidase&doi=10.1021%2Fjm0703690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Density Functional Calculations To Predict the Regioselectivity of Drugs and Molecules Metabolized by Aldehyde Oxidase</span></div><div class="casAuthors">Torres, Rhonda A.; Korzekwa, Kenneth R.; McMasters, Daniel R.; Fandozzi, Christine M.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4642-4647</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase is a molybdenum hydroxylase that catalyzes the oxidn. of aldehydes and nitrogen-contg. heterocycles.  The enzyme plays a dual role in the metab. of physiol. important endogenous compds. and the biotransformation of xenobiotics.  Using d. functional theory methods, geometry optimization of tetrahedral intermediates of drugs and druglike compds. was examd. to predict the likely metabolites of aldehyde oxidase.  The calcns. suggest that the lowest energy tetrahedral intermediate resulting from the initial substrate corresponds to the obsd. metabolite â¥90% of the time.  Addnl. calcns. were performed on a series of heterocyclic compds. where the products resulting from metab. by xanthine oxidase and aldehyde oxidase differ in many instances.  Again, the lowest energy tetrahedral intermediate corresponded to the obsd. product of aldehyde oxidase metab. â¥90% for the compds. examd., while the obsd. products of xanthine oxidase were not well predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIR2Ha4_oLVg90H21EOLACvtfcHk0lgDc28hLk9v4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsVKnsb4%253D&md5=514ee5703f306287a3b7315fc5cea434</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm0703690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0703690%26sid%3Dliteratum%253Aachs%26aulast%3DTorres%26aufirst%3DR.%2BA.%26aulast%3DKorzekwa%26aufirst%3DK.%2BR.%26aulast%3DMcMasters%26aufirst%3DD.%2BR.%26aulast%3DFandozzi%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DUse%2520of%2520Density%2520Functional%2520Calculations%2520to%2520Predict%2520the%2520Regioselectivity%2520of%2520Drugs%2520and%2520Molecules%2520Metabolized%2520by%2520Aldehyde%2520Oxidase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26issue%3D19%26spage%3D4642%26epage%3D4647%26doi%3D10.1021%2Fjm0703690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alfaro, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">Studies On the Mechanism of Aldehyde Oxidase and Xanthine Oxidase</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">9469</span>â <span class="NLM_lpage">9472</span>, <span class="refDoi">Â DOI: 10.1021/jo801053u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo801053u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGku7vI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=9469-9472&issue=23&author=J.+F.+Alfaroauthor=J.+P.+Jones&title=Studies+On+the+Mechanism+of+Aldehyde+Oxidase+and+Xanthine+Oxidase&doi=10.1021%2Fjo801053u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Studies on the Mechanism of Aldehyde Oxidase and Xanthine Oxidase</span></div><div class="casAuthors">Alfaro, Joshua F.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9469-9472</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DFT calcns. support a concerted mechanism for xanthine oxidase and aldehyde oxidase hydride displacement from the sp2 carbon of 6-substituted 4-quinazolinones.  The variations in transition state structure show that C-O bond formation is nearly complete in the transition state and the transition state changes are anti-Hammond with the C-H and C-O bond lengths being more product-like for the faster reactions.  The C-O bond length in the transition state is around 90% formed.  However, the C-H bond is only about 80% broken.  This leads to a very tetrahedral transition state with an O-C-N angle of 109Â°.  Thus, while the mechanism is concerted, the antibonding orbital of the C-H bond that is broken is not directly attacked by the nucleophile and instead hydride displacement occurs after almost complete tetrahedral transition state formation.  In support of this the C=N bond is lengthened in the transition state indicating that attack on the electrophilic carbon occurs by addn. to the C=N bond with neg. charge increasing on the nitrogen.  Differences in exptl. reaction rates are accurately reproduced by these calcns. and tend to support this mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrK9tmdy1IwLVg90H21EOLACvtfcHk0lgDc28hLk9v4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlGku7vI&md5=188f70b1454b3e6bfd19301aafae2908</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjo801053u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo801053u%26sid%3Dliteratum%253Aachs%26aulast%3DAlfaro%26aufirst%3DJ.%2BF.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DStudies%2520On%2520the%2520Mechanism%2520of%2520Aldehyde%2520Oxidase%2520and%2520Xanthine%2520Oxidase%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26issue%3D23%26spage%3D9469%26epage%3D9472%26doi%3D10.1021%2Fjo801053u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montefiori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, L.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase: Reaction Mechanism and Prediction of Site of Metabolism</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">4237</span>â <span class="NLM_lpage">4244</span>, <span class="refDoi">Â DOI: 10.1021/acsomega.7b00658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b00658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohtr3J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=4237-4244&issue=8&author=M.+Montefioriauthor=F.+S.+Jorgensenauthor=L.+Olsen&title=Aldehyde+Oxidase%3A+Reaction+Mechanism+and+Prediction+of+Site+of+Metabolism&doi=10.1021%2Facsomega.7b00658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Oxidase: Reaction Mechanism and Prediction of Site of Metabolism</span></div><div class="casAuthors">Montefiori, Marco; Joergensen, Flemming Steen; Olsen, Lars</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4237-4244</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AO) is a Mo-contg. enzyme involved in the clearance of drug compds. contg. aldehydes and N-contg. heterocyclic fragments.  AO has gained considerable interest in recent years because of examples of too fast clearance of drug compds. in development.  Thus, it is important to be able to predict AO-mediated drug metab.  Therefore, we characterized the structural and energetic aspects of different mechanisms with DFT using the Mo cofactor (MoCo) as model for the reactive part of the enzyme.  For a series of 6-substituted 4-quinazolinones, the trend in activation energies was the same for 3 tested reaction mechanisms.  Using the concerted mechanism as model for the enzymic reaction, the transition states for formation of all possible metabolites for a series of known AO substrates were detd.  The lowest activation energies corresponded in all cases to the exptl. obsd. sites of metab. (SOMs).  Various mol. properties were calcd. and investigated as more easily determinable markers for reactivity.  Both the stabilities of the intermediates and products correlated to some extent with the transition state energies, and may be used to predict the SOM.  The electrostatic potential (ESP)-derived charges were also good markers for prediction of the exptl. SOMs for this set of compds. and may pave the way for the development of fast methods for prediction of SOMs for AO substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWMl-me1-SlbVg90H21EOLACvtfcHk0lgDc28hLk9v4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohtr3J&md5=5aa9f790b1ff0f28cb270322618de06c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b00658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b00658%26sid%3Dliteratum%253Aachs%26aulast%3DMontefiori%26aufirst%3DM.%26aulast%3DJorgensen%26aufirst%3DF.%2BS.%26aulast%3DOlsen%26aufirst%3DL.%26atitle%3DAldehyde%2520Oxidase%253A%2520Reaction%2520Mechanism%2520and%2520Prediction%2520of%2520Site%2520of%2520Metabolism%26jtitle%3DACS%2520Omega%26date%3D2017%26volume%3D2%26issue%3D8%26spage%3D4237%26epage%3D4244%26doi%3D10.1021%2Facsomega.7b00658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. D.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase: An Enzyme of Emerging Importance in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">8441</span>â <span class="NLM_lpage">8460</span>, <span class="refDoi">Â DOI: 10.1021/jm100888d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100888d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFygt73N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8441-8460&issue=24&author=D.+C.+Prydeauthor=D.+Dalvieauthor=Q.+Huauthor=P.+Jonesauthor=R.+S.+Obachauthor=T.+D.+Tran&title=Aldehyde+Oxidase%3A+An+Enzyme+of+Emerging+Importance+in+Drug+Discovery&doi=10.1021%2Fjm100888d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Oxidase: An Enzyme of Emerging Importance in Drug Discovery</span></div><div class="casAuthors">Pryde, David C.; Dalvie, Deepak; Hu, Qiyue; Jones, Peter; Obach, R. Scott; Tran, Thien-Duc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8441-8460</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Aldehyde oxidase has for many years been recognized as a metabolizing enzyme contained within the cytosolic compartment of many tissues and in many species.  The first literature ref. to AO dates from the 1930s, but literature citations to this enzyme have been steadily growing ever since, esp. in the past decade.  It has been increasingly recognized in this past decade that AO, through its unique structure, distribution, and substrate recognition, has an important role to play in the metab. of drugs.  In this timely Perspective, we present the current knowledge of the enzyme's expression, its structure, and its distribution across species.  We offer an anal. of substrates recognized by AO and highlight clin. examples where metab. by this enzyme has had significant clin. impact or led to the termination of a drug development program.  Finally, through anal. of the structure of known substrates, we present data to suggest the significant impact AO could play on future drug discovery programs along with strategies that could be employed to mitigate this.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqALwFS70G6tLVg90H21EOLACvtfcHk0lgDc28hLk9v4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFygt73N&md5=972eb9862a7dc7bff024b530ab40de7f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm100888d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100888d%26sid%3Dliteratum%253Aachs%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DTran%26aufirst%3DT.%2BD.%26atitle%3DAldehyde%2520Oxidase%253A%2520An%2520Enzyme%2520of%2520Emerging%2520Importance%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D24%26spage%3D8441%26epage%3D8460%26doi%3D10.1021%2Fjm100888d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, P.</span></span> <span> </span><span class="NLM_article-title">Effect of Structural Variation on Aldehyde Oxidase-Catalyzed Oxidation of Zoniporide</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1575</span>â <span class="NLM_lpage">1587</span>, <span class="refDoi">Â DOI: 10.1124/dmd.112.045823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.112.045823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22587988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2iurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1575-1587&issue=8&author=D.+Dalvieauthor=H.+Sunauthor=C.+Xiangauthor=Q.+Huauthor=Y.+Jiangauthor=P.+Kang&title=Effect+of+Structural+Variation+on+Aldehyde+Oxidase-Catalyzed+Oxidation+of+Zoniporide&doi=10.1124%2Fdmd.112.045823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of structural variation on aldehyde oxidase-catalyzed oxidation of zoniporide</span></div><div class="casAuthors">Dalvie, Deepak; Sun, Hao; Xiang, Cathie; Hu, Qiyue; Jiang, Ying; Kang, Ping</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1575-1587</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Current studies explored the effect of structural changes on the aldehyde oxidase (AO)-mediated metab. of zoniporide (1).  Zoniporide analogs with modifications of the acylguanidine moiety, the cyclopropyl group on the pyrazole ring, and the quinoline ring were studied for their AO-catalyzed metab. using the human S9 fraction.  Anal. of the half-lives suggested that subtle changes in the structure of 1 influenced its metab. and that the guanidine and the quinoline moieties were prerequisites for AO-catalyzed oxidn. to 2-oxozoniporide (M1).  In contrast, replacement of the cyclopropyl group with other alkyl groups was tolerated.  The effect of structural variation on AO properties was rationalized by docking 1 and its analogs into the human AO homol. model.  These studies indicated the importance of electrostatic, Ï-Ï stacking and hydrophobic interactions of the three motifs with residues in the active site.  Differences in substrate properties were also rationalized by comparing their half-lives with cLogD, electrophilicity parameters [electrostatic potential (ESP) charges and energy of lowest unoccupied MOs (ELUMO)], and the energies of formation of tetrahedral intermediates.  Whereas the success of energetics in predicting the AO substrate properties of analogs was 87%, the predictive ability of other descriptors was none (cLogD) to 60% (ESP charges and ELUMO).  Overall, the structure-metab. relationship could be rationalized using a combination of both the energy calcns. and docking studies.  This combination method can be incorporated into a strategy for mitigating AO liabilities obsd. in the lead candidate or studying structure-metab. relationships of other AO substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCkVhA55F__bVg90H21EOLACvtfcHk0lgijhkJXT8owg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2iurbK&md5=b3ec4f1916d516092e72156dd63304a5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.045823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.045823%26sid%3Dliteratum%253Aachs%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXiang%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DP.%26atitle%3DEffect%2520of%2520Structural%2520Variation%2520on%2520Aldehyde%2520Oxidase-Catalyzed%2520Oxidation%2520of%2520Zoniporide%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26issue%3D8%26spage%3D1575%26epage%3D1587%26doi%3D10.1124%2Fdmd.112.045823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzekwa, K. R.</span></span> <span> </span><span class="NLM_article-title">Predicting Intrinsic Clearance for Drugs and Drug Candidates Metabolized by Aldehyde Oxidase</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1262</span>â <span class="NLM_lpage">1268</span>, <span class="refDoi">Â DOI: 10.1021/mp300568r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300568r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ajtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1262-1268&issue=4&author=J.+P.+Jonesauthor=K.+R.+Korzekwa&title=Predicting+Intrinsic+Clearance+for+Drugs+and+Drug+Candidates+Metabolized+by+Aldehyde+Oxidase&doi=10.1021%2Fmp300568r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Intrinsic Clearance for Drugs and Drug Candidates Metabolized by Aldehyde Oxidase</span></div><div class="casAuthors">Jones, Jeffrey P.; Korzekwa, Kenneth R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1262-1268</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metab. by aldehyde oxidase (AO) has been responsible for a no. of drug failures in clin. trials.  The main reason is the clearance values for drugs metabolized by AO are underestimated by allometric scaling from preclin. species.  Furthermore, in vitro human data also underestimates clearance.  We have developed the first in silico models to predict both in vitro and in vivo human intrinsic clearance for 8 drugs with just two chem. descriptors.  These models explain a large amt. of the variance in the data using two computational ests. of the electronic and steric features of the reaction.  The in vivo computational models for human metab. are better than in vitro preclin. animal testing at predicting human intrinsic clearance.  Thus, it appears that AO is amenable to computational prediction of rates, which may be used to guide drug discovery, and predict pharmacokinetics for clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMe4imNUVqorVg90H21EOLACvtfcHk0lgijhkJXT8owg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ajtbc%253D&md5=b91f8487291a445d04cccd20fd9227a3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fmp300568r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300568r%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DJ.%2BP.%26aulast%3DKorzekwa%26aufirst%3DK.%2BR.%26atitle%3DPredicting%2520Intrinsic%2520Clearance%2520for%2520Drugs%2520and%2520Drug%2520Candidates%2520Metabolized%2520by%2520Aldehyde%2520Oxidase%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26issue%3D4%26spage%3D1262%26epage%3D1268%26doi%3D10.1021%2Fmp300568r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lepri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cucco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goracci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruciani, G.</span></span> <span> </span><span class="NLM_article-title">Structure-Metabolism Relationships in Human-AOX: Chemical Insights From a Large Database of Aza-Aromatic and Amide Compounds</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">E3178</span>â <span class="NLM_lpage">E3187</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1618881114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1073%2Fpnas.1618881114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28373537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsV2nsb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E3178-E3187&issue=16&author=S.+Lepriauthor=M.+Ceccarelliauthor=N.+Milaniauthor=S.+Tortorellaauthor=A.+Cuccoauthor=A.+Valeriauthor=L.+Goracciauthor=A.+Brinkauthor=G.+Cruciani&title=Structure-Metabolism+Relationships+in+Human-AOX%3A+Chemical+Insights+From+a+Large+Database+of+Aza-Aromatic+and+Amide+Compounds&doi=10.1073%2Fpnas.1618881114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-metabolism relationships in human-AOX: Chemical insights from a large database of aza-aromatic and amide compounds.</span></div><div class="casAuthors">Lepri, Susan; Ceccarelli, Martina; Milani, Nicolo; Tortorella, Sara; Cucco, Andrea; Valeri, Aurora; Goracci, Laura; Brink, Andreas; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">E3178-E3187</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a metabolic enzyme catalyzing the oxidn. of aldehyde and aza-arom. compds. and the hydrolysis of amides, moieties frequently shared by the majority of drugs.  Despite its key role in human metab., to date only fragmentary information about the chem. features responsible for AOX susceptibility are reported and only "very local" structure-metab. relationships based on a small no. of similar compds. have been developed.  This study reports a more comprehensive coverage of the chem. space of structures with a high risk of AOX phase I metab. in humans.  More than 270 compds. were studied to identify the site of metab. and the metabolite(s).  Both electronic [supported by d. functional theory (DFT) calcns.] and exposure effects were considered when rationalizing the structure-metab. relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotA9C1TVn-9bVg90H21EOLACvtfcHk0lgijhkJXT8owg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsV2nsb8%253D&md5=81cd013f7b63399e10d950e30377dbc0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1618881114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1618881114%26sid%3Dliteratum%253Aachs%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DCeccarelli%26aufirst%3DM.%26aulast%3DMilani%26aufirst%3DN.%26aulast%3DTortorella%26aufirst%3DS.%26aulast%3DCucco%26aufirst%3DA.%26aulast%3DValeri%26aufirst%3DA.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DBrink%26aufirst%3DA.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DStructure-Metabolism%2520Relationships%2520in%2520Human-AOX%253A%2520Chemical%2520Insights%2520From%2520a%2520Large%2520Database%2520of%2520Aza-Aromatic%2520and%2520Amide%2520Compounds%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D16%26spage%3DE3178%26epage%3DE3187%26doi%3D10.1073%2Fpnas.1618881114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cruciani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesarini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spyrakis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goracci, L.</span></span> <span> </span><span class="NLM_article-title">From Experiments to a Fast Easy-to-Use Computational Methodology to Predict Human Aldehyde Oxidase Selectivity and Metabolic Reactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">360</span>â <span class="NLM_lpage">371</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01552</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01552" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOru7fK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=360-371&issue=1&author=G.+Crucianiauthor=N.+Milaniauthor=P.+Benedettiauthor=S.+Lepriauthor=L.+Cesariniauthor=M.+Baroniauthor=F.+Spyrakisauthor=S.+Tortorellaauthor=E.+Mosconiauthor=L.+Goracci&title=From+Experiments+to+a+Fast+Easy-to-Use+Computational+Methodology+to+Predict+Human+Aldehyde+Oxidase+Selectivity+and+Metabolic+Reactions&doi=10.1021%2Facs.jmedchem.7b01552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">From Experiments to a Fast Easy-to-Use Computational Methodology to Predict Human Aldehyde Oxidase Selectivity and Metabolic Reactions</span></div><div class="casAuthors">Cruciani, Gabriele; Milani, Nicolo; Benedetti, Paolo; Lepri, Susan; Cesarini, Lucia; Baroni, Massimo; Spyrakis, Francesca; Tortorella, Sara; Mosconi, Edoardo; Goracci, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">360-371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a molibdo-flavoenzyme that raised great interest in recent years, since its contribution in xenobiotics metab. has not always been identified before clin. trials, with consequent neg. effects on the fate of new potential drugs.  The fundamental role of AOX in metabolizing xenobiotics is also due to the attempt of medicinal chemists to stabilize candidates toward cytochrome P 450 activity, which increases the risk for new compds. to be susceptible to AOX nucleophile attack.  Therefore, novel strategies to predict the potential liability of new entities towards AOX enzyme are urgently needed to increase effectiveness, reduce costs, and prioritize exptl. studies.  In the present work, we present the most up-to-date computational method to predict liability towards human AOX, for applications in drug design and pharmacokinetic optimization.  The method was developed using a large dataset of homogeneous exptl. data, which is also disclosed as supplementary material.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoi_Tnrnus317Vg90H21EOLACvtfcHk0lgp4qzIx_Iaig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOru7fK&md5=23c7fd8a5abcba87e7fc065d5e6dafc6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01552%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DMilani%26aufirst%3DN.%26aulast%3DBenedetti%26aufirst%3DP.%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DCesarini%26aufirst%3DL.%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DTortorella%26aufirst%3DS.%26aulast%3DMosconi%26aufirst%3DE.%26aulast%3DGoracci%26aufirst%3DL.%26atitle%3DFrom%2520Experiments%2520to%2520a%2520Fast%2520Easy-to-Use%2520Computational%2520Methodology%2520to%2520Predict%2520Human%2520Aldehyde%2520Oxidase%2520Selectivity%2520and%2520Metabolic%2520Reactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D1%26spage%3D360%26epage%3D371%26doi%3D10.1021%2Facs.jmedchem.7b01552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montefiori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyngholm-Kjaerby, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, F. S.</span></span> <span> </span><span class="NLM_article-title">Fast Methods for Prediction of Aldehyde Oxidase-Mediated Site-of-Metabolism</span>. <i>Comput. Struct. Biotechnol. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">345</span>â <span class="NLM_lpage">351</span>, <span class="refDoi">Â DOI: 10.1016/j.csbj.2019.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1016%2Fj.csbj.2019.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=30949305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlt12jsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=345-351&author=M.+Montefioriauthor=C.+Lyngholm-Kjaerbyauthor=A.+Longauthor=L.+Olsenauthor=F.+S.+Jorgensen&title=Fast+Methods+for+Prediction+of+Aldehyde+Oxidase-Mediated+Site-of-Metabolism&doi=10.1016%2Fj.csbj.2019.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Methods for Prediction of Aldehyde Oxidase-Mediated Site-of-Metabolism</span></div><div class="casAuthors">Montefiori, Marco; Lyngholm-Kjaerby, Casper; Long, Anthony; Olsen, Lars; Joergensen, Flemming Steen</div><div class="citationInfo"><span class="NLM_cas:title">Computational and Structural Biotechnology Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">345-351</span>CODEN:
                <span class="NLM_cas:coden">CSBJAC</span>;
        ISSN:<span class="NLM_cas:issn">2001-0370</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aldehyde Oxidase (AO) is an enzyme involved in the metab. of aldehydes and N-contg. heterocyclic compds.  Many drug compds. contain heterocyclic moieties, and AO metab. has lead to failure of several late-stage drug candidates.  Therefore, it is important to take AO-mediated metab. into account early in the drug discovery process, and thus, to have fast and reliable models to predict the site of metab. (SOM).  We have collected a dataset of 78 substrates of human AO with a total of 89 SOMs and 347 non-SOMs and detd. at. descriptors for each compd.  The descriptors comprise NMR shielding and ESP charges from d. functional theory (DFT), NMR chem. shift from ChemBioDraw, and Gasteiger charges from RDKit.  Addnl., at. accessibility was considered using 2D-SASA and relative span descriptors from SMARTCyp.  Finally, stability of the product, the metabolite, was detd. with DFT and also used as a descriptor.  All descriptors have AUC larger than 0.75.  In particular, descriptors related to the chem. shielding and chem. shift (AUC = 0.96) and ESP charges (AUC = 0.96) proved to be good descriptors.  We recommend two simple methods to identify the SOM for a given mol.: (1) use ChemBioDraw to calc. the chem. shift or (2) calc. ESP charges or chem. shift using DFT.  The first approach is fast but somewhat difficult to automate, while the second is more time-consuming, but can easily be automated.  The two methods predict correctly 93% and 91%, resp., of the 89 exptl. obsd. SOMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8QYgsDr8OrLVg90H21EOLACvtfcHk0lgp4qzIx_Iaig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlt12jsLw%253D&md5=8a01ea8667b1bfcccc7ab59570df6739</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.csbj.2019.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.csbj.2019.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DMontefiori%26aufirst%3DM.%26aulast%3DLyngholm-Kjaerby%26aufirst%3DC.%26aulast%3DLong%26aufirst%3DA.%26aulast%3DOlsen%26aufirst%3DL.%26aulast%3DJorgensen%26aufirst%3DF.%2BS.%26atitle%3DFast%2520Methods%2520for%2520Prediction%2520of%2520Aldehyde%2520Oxidase-Mediated%2520Site-of-Metabolism%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2019%26volume%3D17%26spage%3D345%26epage%3D351%26doi%3D10.1016%2Fj.csbj.2019.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dick, R. A.</span></span> <span> </span><span class="NLM_article-title">Refinement of In Vitro Methods for Identification of Aldehyde Oxidase Substrates Reveals Metabolites of Kinase Inhibitors</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">846</span>â <span class="NLM_lpage">859</span>, <span class="refDoi">Â DOI: 10.1124/dmd.118.080960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.118.080960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=29615437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWlsb7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=846-859&issue=6&author=R.+A.+Dick&title=Refinement+of+In+Vitro+Methods+for+Identification+of+Aldehyde+Oxidase+Substrates+Reveals+Metabolites+of+Kinase+Inhibitors&doi=10.1124%2Fdmd.118.080960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of in vitro methods for identification of aldehyde oxidase substrates reveals metabolites of kinase inhibitors</span></div><div class="casAuthors">Dick, Ryan A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">846-859</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">To identify aldehyde oxidase (AO) substrates, an assay procedure was developed that leverages the capabilities of high-resoln. mass spectrometry to simultaneously monitor parent loss and formation of hydroxylated metabolite over time in incubations with liver cytosol.  By incorporating metabolite monitoring, false positives resulting from metab. by other cytosolic enzymes or processes were decreased.  A diverse set of 34 kinase inhibitors contg. AO-substrate motifs was screened, and 35% of the compds. were identified as human AO substrates.  Confirmation was obtained through detn. of the site of metab.  Human AO substrates identified contained unsubstituted diazanaphthalene moieties (A77-01, INCB 28060, ML-347, LDN-193189, and SB-525334), 4-aminoquinazoline cores (lapatinib, lapatinib M1, and CL-387785), and terminal pyridine and pyrimidine groups (imatinib, bafetinib, and AMG 900).  Rat and cynomolgus monkey AO displayed substrate specificities that overlapped moderately with human; rates of metab. were often higher and lower for cynomolgus monkey and rat, resp., compared with human.  A subset of novel AO substrates identified in this study was also subjected to two other methods for AO substrate detn.: comparison of human liver microsome and hepatocyte stability, and the effect of hydralazine, an AO-specific inhibitor, on hepatocyte stability.  These methods appeared to correlate and be capable of identifying AO substrates when more than one-third of metab. in hepatocytes was AO-mediated; however, significant limitations exist.  Considering the sensitivity, efficiency, and definitiveness of the cytosol assay with metabolite monitoring, its use is recommended as a primary screen for AO substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlGGJzSPQLDbVg90H21EOLACvtfcHk0lgp4qzIx_Iaig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWlsb7M&md5=0b08ae43f93b6a9874f10b40422d93b2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fdmd.118.080960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.118.080960%26sid%3Dliteratum%253Aachs%26aulast%3DDick%26aufirst%3DR.%2BA.%26atitle%3DRefinement%2520of%2520In%2520Vitro%2520Methods%2520for%2520Identification%2520of%2520Aldehyde%2520Oxidase%2520Substrates%2520Reveals%2520Metabolites%2520of%2520Kinase%2520Inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2018%26volume%3D46%26issue%3D6%26spage%3D846%26epage%3D859%26doi%3D10.1124%2Fdmd.118.080960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span> <span> </span><span class="NLM_article-title">In Silico Site of Metabolism Prediction for Human UGT-Catalyzed Reactions</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">398</span>â <span class="NLM_lpage">405</span>, <span class="refDoi">Â DOI: 10.1093/bioinformatics/btt681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1093%2Fbioinformatics%2Fbtt681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=24273240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=398-405&issue=3&author=J.+Pengauthor=J.+Luauthor=Q.+Shenauthor=M.+Zhengauthor=X.+Luoauthor=W.+Zhuauthor=H.+Jiangauthor=K.+Chen&title=In+Silico+Site+of+Metabolism+Prediction+for+Human+UGT-Catalyzed+Reactions&doi=10.1093%2Fbioinformatics%2Fbtt681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">In silico site of metabolism prediction for human UGT-catalyzed reactions</span></div><div class="casAuthors">Peng, Jianlong; Lu, Jing; Shen, Qiancheng; Zheng, Mingyue; Luo, Xiaomin; Zhu, Weiliang; Jiang, Hualiang; Chen, Kaixian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">398-405</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The human uridine diphosphate-glucuronosyltransferase enzyme family catalyzes the glucuronidation of the glycosyl group of a nucleotide sugar to an acceptor compd. (substrate), which is the most common conjugation pathway that serves to protect the organism from the potential toxicity of xenobiotics.  Moreover, it could affect the pharmacol. profile of a drug.  Therefore, it is important to identify the metabolically labile sites for glucuronidation.  Results: In the present study, we developed four in silico models to predict sites of glucuronidation, for four major sites of metab. functional groups, i.e. aliph. hydroxyl, arom. hydroxyl, carboxylic acid or amino nitrogen, resp.  According to the mechanism of glucuronidation, a series of 'local' and 'global' mol. descriptors characterizing the at. reactivity, bonding strength and phys.-chem. properties were calcd. and selected with a genetic algorithm-based feature selection approach.  The constructed support vector machine classification models show good prediction performance, with the balanced accuracy ranging from 0.88 to 0.96 on test set.  For further validation, our models can successfully identify 84% of exptl. obsd. sites of metabs. for an external test set contg. 54 mols.  Availability and implementation: The software somugt based on our models is available at www.dddc.ac.cn/adme/jlpeng/somugt_win32.zip.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="f1899c9d849eb1989c9cdf9092df929f">[emailÂ protected]</a> or <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="3e534744565b50597e5f5752104d565d505d105f5d105d50">[emailÂ protected]</a> Supplementary information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTnU5JiJGKPrVg90H21EOLACvtfcHk0li2YD3A0MeAUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D&md5=c99109a95f1bb99af1c93a3946a3dd44</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt681%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26atitle%3DIn%2520Silico%2520Site%2520of%2520Metabolism%2520Prediction%2520for%2520Human%2520UGT-Catalyzed%2520Reactions%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26issue%3D3%26spage%3D398%26epage%3D405%26doi%3D10.1093%2Fbioinformatics%2Fbtt681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span> <span> </span><span class="NLM_article-title">Prediction of Human Cytochrome P450 Inhibition Using a Multitask Deep Autoencoder Neural Network</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">4336</span>â <span class="NLM_lpage">4345</span>, <span class="refDoi">Â DOI: 10.1021/acs.molpharmaceut.8b00110</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.8b00110" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps12nt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=4336-4345&author=X.+Liauthor=Y.+Xuauthor=L.+Laiauthor=J.+Pei&title=Prediction+of+Human+Cytochrome+P450+Inhibition+Using+a+Multitask+Deep+Autoencoder+Neural+Network&doi=10.1021%2Facs.molpharmaceut.8b00110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Human Cytochrome P450 Inhibition Using a Multitask Deep Autoencoder Neural Network</span></div><div class="casAuthors">Li, Xiang; Xu, Youjun; Lai, Luhua; Pei, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4336-4345</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adverse side effects of drug-drug interactions induced by human cytochrome P 450 (CYP450) inhibition is an important consideration in drug discovery.  It is highly desirable to develop computational models that can predict the inhibitive effect of a compd. against a specific CYP450 isoform.  In this study, we developed a multitask model for concurrent inhibition prediction of five major CYP450 isoforms, namely, 1A2, 2C9, 2C19, 2D6, and 3A4.  The model was built by training a multitask autoencoder deep neural network (DNN) on a large dataset contg. more than 13,000 compds., extd. from the PubChem BioAssay Database.  We demonstrate that the multitask model gave better prediction results than that of single-task models, previous reported classifiers, and traditional machine learning methods on an av. of five prediction tasks.  Our multitask DNN model gave av. prediction accuracies of 86.4% for the 10-fold cross-validation and 88.7% for the external test datasets.  In addn., we built linear regression models to quantify how the other tasks contributed to the prediction difference of a given task between single-task and multitask models, and we explained under what conditions the multitask model will outperform the single-task model, which suggested how to use multitask DNN models more effectively.  We applied sensitivity anal. to ext. useful knowledge about CYP450 inhibition, which may shed light on the structural features of these isoforms and give hints about how to avoid side effects during drug development.  Our models are freely available at http://repharma.pku.edu.cn/deepcyp/home.php or http://www.pkumdl.cn/deepcyp/home.php.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWZzh0LeCx9rVg90H21EOLACvtfcHk0li2YD3A0MeAUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps12nt7s%253D&md5=2c08dbac3a65242c61ef70c3147f4a85</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.8b00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.8b00110%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26atitle%3DPrediction%2520of%2520Human%2520Cytochrome%2520P450%2520Inhibition%2520Using%2520a%2520Multitask%2520Deep%2520Autoencoder%2520Neural%2520Network%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2018%26volume%3D15%26spage%3D4336%26epage%3D4345%26doi%3D10.1021%2Facs.molpharmaceut.8b00110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. A.</span></span> <span> </span><span class="NLM_article-title">Discovery and ADMET: Where Are We Now</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">467</span>â <span class="NLM_lpage">481</span>, <span class="refDoi">Â DOI: 10.2174/156802611794480909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.2174%2F156802611794480909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=21320070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslSnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=467-481&issue=4&author=D.+A.+Smith&title=Discovery+and+ADMET%3A+Where+Are+We+Now&doi=10.2174%2F156802611794480909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and ADMET: where are we now</span></div><div class="casAuthors">Smith, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-481</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The gradual alignment with all of drug metab. with all aspects of drug discovery and development has led to a complete realignment of the way the work is conducted.  From a background of conducting bespoke in vivo studies much of the work is now in a high throughput screening mode.  Large technol. advances were made, but the nature of drug metab. processes, being multi-system and promiscuous means that much of the help provided to the medicinal chem. is reactive rather than based on fundamental disposition structure-activity relationships.  Lessons learned around the chem. and physicochem. properties more often assocd. with successful discovery and development projects are only moderately helpful when the high value pharmacol. targets of today only yield potent ligands outside of the boundaries these properties describe.  Pivotal to the impact of these properties is the intrinsic permeability of a mol., something not as widely recognized as perhaps it should be.  Metabolic lability is still a problem and the tactics employed are unchanged in 20 years: attempt to lower lipophilicity, if it is too high overall or introduce blocking groups, particularly halogens, after identifying the sites of metab.  Perhaps the greatest success drug metab. science has had over the last fifteen years is it's pivotal role in characterizing drug-drug interactions and providing screening systems and computational models to investigate them.  It still has many undeveloped areas of the science such as the role of metabolites in drug activity and why compds. vary in their extent of biliary excretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEw_2VMNc--bVg90H21EOLACvtfcHk0li2YD3A0MeAUw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslSnurs%253D&md5=b4e1b937f2f048a3ec9b97df98b339e6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2174%2F156802611794480909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794480909%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26atitle%3DDiscovery%2520and%2520ADMET%253A%2520Where%2520Are%2520We%2520Now%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26issue%3D4%26spage%3D467%26epage%3D481%26doi%3D10.2174%2F156802611794480909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shoombuatong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prathipati, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prachayasittikul, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaduangrat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratiwi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanwimolruk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikberg, J. E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spjuth, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nantasenamat, C.</span></span> <span> </span><span class="NLM_article-title">Towards Predicting the Cytochrome P450 Modulation: From QSAR to Proteochemometric Modeling</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">540</span>â <span class="NLM_lpage">555</span>, <span class="refDoi">Â DOI: 10.2174/1389200218666170320121932</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.2174%2F1389200218666170320121932" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28322159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Okur%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=540-555&issue=6&author=W.+Shoombuatongauthor=P.+Prathipatiauthor=V.+Prachayasittikulauthor=N.+Schaduangratauthor=A.+A.+Malikauthor=R.+Pratiwiauthor=S.+Wanwimolrukauthor=J.+E.+S.+Wikbergauthor=M.+P.+Gleesonauthor=O.+Spjuthauthor=C.+Nantasenamat&title=Towards+Predicting+the+Cytochrome+P450+Modulation%3A+From+QSAR+to+Proteochemometric+Modeling&doi=10.2174%2F1389200218666170320121932"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Towards Predicting the Cytochrome P450 Modulation: From QSAR to Proteochemometric Modeling</span></div><div class="casAuthors">Shoombuatong, Watshara; Prathipati, Philip; Prachayasittikul, Veda; Schaduangrat, Nalini; Malik, Aijaz Ahmad; Pratiwi, Reny; Wanwimolruk, Sompon; Wikberg, Jarl E. S.; Gleeson, Matthew Paul; Spjuth, Ola; Nantasenamat, Chanin</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-555</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Drug metab. dets. the fate of a drug when it enters the human body and is a crit. factor in defining their absorption, distribution, metab., excretion and toxicity (ADMET) characteristics.  Among the various drug metabolizing enzymes, cytochrome P450s (CYP450) constitute an important protein family that aside from functioning in xenobiotic metab., is also responsible for a diverse array of other roles encompassing steroid and cholesterol biosynthesis, fatty acid metab., calcium homeostasis, neuroendocrine functions and growth regulation.  Although CYP450 typically converts xenobiotics into safe metabolites, there are some situations whereby the metabolite is more toxic than its parent mol.  Computational modeling has been instrumental in CYP450 research by rationalizing the nature of the binding event (i.e. inhibit or induce CYP450s) or metabolic stability of query compds. of interest.  A plethora of computational approaches encompassing ligand, structure and systems based approaches have been utilized to model CYP450-ligand interactions.  This review provides a brief background on the CYP450 family (i.e. its roles, advantages and disadvantages as well as its modulators) and then discusses the various computational approaches that have been used to model CYP450-ligand interaction.  Particular focus was given to the use of quant. structure-activity relationship (QSAR) and more recent proteochemometric modeling studies.  Finally, a perspective on the current state of the art and future trends of the field is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUBK5dISfQ2LVg90H21EOLACvtfcHk0li5P7tBNERSlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Okur%252FN&md5=5dea9e43349371555af5030e3a145f02</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.2174%2F1389200218666170320121932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389200218666170320121932%26sid%3Dliteratum%253Aachs%26aulast%3DShoombuatong%26aufirst%3DW.%26aulast%3DPrathipati%26aufirst%3DP.%26aulast%3DPrachayasittikul%26aufirst%3DV.%26aulast%3DSchaduangrat%26aufirst%3DN.%26aulast%3DMalik%26aufirst%3DA.%2BA.%26aulast%3DPratiwi%26aufirst%3DR.%26aulast%3DWanwimolruk%26aufirst%3DS.%26aulast%3DWikberg%26aufirst%3DJ.%2BE.%2BS.%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26aulast%3DSpjuth%26aufirst%3DO.%26aulast%3DNantasenamat%26aufirst%3DC.%26atitle%3DTowards%2520Predicting%2520the%2520Cytochrome%2520P450%2520Modulation%253A%2520From%2520QSAR%2520to%2520Proteochemometric%2520Modeling%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2017%26volume%3D18%26issue%3D6%26spage%3D540%26epage%3D555%26doi%3D10.2174%2F1389200218666170320121932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwobel, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleva, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enoch, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajot, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M. T.</span></span> <span> </span><span class="NLM_article-title">Measurement and Estimation of Electrophilic Reactivity for Predictive Toxicology</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>111</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">2562</span>â <span class="NLM_lpage">2596</span>, <span class="refDoi">Â DOI: 10.1021/cr100098n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr100098n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFeku7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2011&pages=2562-2596&issue=4&author=J.+A.+Schwobelauthor=Y.+K.+Kolevaauthor=S.+J.+Enochauthor=F.+Bajotauthor=M.+Hewittauthor=J.+C.+Maddenauthor=D.+W.+Robertsauthor=T.+W.+Schultzauthor=M.+T.+Cronin&title=Measurement+and+Estimation+of+Electrophilic+Reactivity+for+Predictive+Toxicology&doi=10.1021%2Fcr100098n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement and Estimation of Electrophilic Reactivity for Predictive Toxicology</span></div><div class="casAuthors">Schwobel, Johannes A. H.; Koleva, Yana K.; Enoch, Steven J.; Bajot, Fania; Hewitt, Mark; Madden, Judith C.; Roberts, David W.; Schultz, Terry W.; Cronin, Mark T. D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2562-2596</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A database of chem. reactivity data from the past 80 years is compiled for measurement and estn. of electrophilic reactivity for predictive toxicol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgDcG9HOFicLVg90H21EOLACvtfcHk0li5P7tBNERSlw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFeku7c%253D&md5=62a4386ab81bb50a91bdef8bb029e74b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fcr100098n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr100098n%26sid%3Dliteratum%253Aachs%26aulast%3DSchwobel%26aufirst%3DJ.%2BA.%26aulast%3DKoleva%26aufirst%3DY.%2BK.%26aulast%3DEnoch%26aufirst%3DS.%2BJ.%26aulast%3DBajot%26aufirst%3DF.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DRoberts%26aufirst%3DD.%2BW.%26aulast%3DSchultz%26aufirst%3DT.%2BW.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26atitle%3DMeasurement%2520and%2520Estimation%2520of%2520Electrophilic%2520Reactivity%2520for%2520Predictive%2520Toxicology%26jtitle%3DChem.%2520Rev.%26date%3D2011%26volume%3D111%26issue%3D4%26spage%3D2562%26epage%3D2596%26doi%3D10.1021%2Fcr100098n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarbrough, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. L.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships for Reactivity of Carbonyl-Containing Compounds with Glutathione</span>. <i>SAR and QSAR in Environmental Research</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">313</span>â <span class="NLM_lpage">322</span>, <span class="refDoi">Â DOI: 10.1080/10659360500204152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1080%2F10659360500204152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=16234173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKqtr%252FE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2005&pages=313-322&issue=4&author=T.+W.+Schultzauthor=J.+W.+Yarbroughauthor=E.+L.+Johnson&title=Structure-Activity+Relationships+for+Reactivity+of+Carbonyl-Containing+Compounds+with+Glutathione&doi=10.1080%2F10659360500204152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships for reactivity of carbonyl-containing compounds with glutathione</span></div><div class="casAuthors">Schultz, T. W.; Yarbrough, J. W.; Johnson, E. L.</div><div class="citationInfo"><span class="NLM_cas:title">SAR and QSAR in Environmental Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">SQERED</span>;
        ISSN:<span class="NLM_cas:issn">1062-936X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">For toxicol.-based structure-activity relationships to advance, will require a better understanding of mol. reactivity.  A rapid and inexpensive spectrophotometric assay for detg. the reactive to glutathione (GSH) was developed and used to det. GSH reactivity (reactGSH) data for 21 aliph. derivs. of esters, ketones and aldehydes.  From these data, a series of structure-activity relationships were evaluated.  The structure feature assocd. with reactGSH was an acetylenic or olefinic moiety conjugated to a carbonyl group (i.e. polarized Î±,Î²-unsatn.).  This structure conveys the capacity to undergo a covalent interaction with the thiol group of cysteine (i.e. Michael- addn.).  Quant. reactGSH of the Î±,Î²-unsatd. carbonyl compds. is reliant upon the specific mol. structure with several tendencies obsd.  Specifically, it was noted that for Î±,Î²-unsatd. carbonyl compds.: (1) the acetylenic-substituted derivs. were more reactive than the corresponding olefinic-substituted ones; (2) terminal vinyl-substituted derivs. was more reactive than the internal vinylene-substituted ones; (3) Me substitution on the vinyl carbon atoms diminishes reactivity and methyl-substitution on the carbon atom farthest from the carbonyl group causes a larger redn.; (4) derivs. with carbon-carbon double bond on the end of the mol. (i.e. vinyl ketone) were more reactive than one with the carbon-oxygen bond at the end of the mol. (i.e. aldehyde) and (5) the ester with an addnl. unsatd. vinyl groups were more reactive than the deriv. having an unsatd. Et group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0zcWMFpuabVg90H21EOLACvtfcHk0lhiYI2TB7XCAw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKqtr%252FE&md5=0586fe55450d43235b05155559874647</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F10659360500204152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10659360500204152%26sid%3Dliteratum%253Aachs%26aulast%3DSchultz%26aufirst%3DT.%2BW.%26aulast%3DYarbrough%26aufirst%3DJ.%2BW.%26aulast%3DJohnson%26aufirst%3DE.%2BL.%26atitle%3DStructure-Activity%2520Relationships%2520for%2520Reactivity%2520of%2520Carbonyl-Containing%2520Compounds%2520with%2520Glutathione%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26date%3D2005%26volume%3D16%26issue%3D4%26spage%3D313%26epage%3D322%26doi%3D10.1080%2F10659360500204152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerlings, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span> <span> </span><span class="NLM_article-title">Local Softness and Hardness Based Reactivity Descriptors for Predicting Intra- and Intermolecular Reactivity Sequences: Carbonyl Compounds</span>. <i>J. Phys. Chem. A</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>102</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">3746</span>â <span class="NLM_lpage">3755</span>, <span class="refDoi">Â DOI: 10.1021/jp973450v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp973450v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADyaK1cXis12nt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3746-3755&issue=21&author=R.+K.+Royauthor=S.+Krishnamurtiauthor=P.+Geerlingsauthor=S.+Pal&title=Local+Softness+and+Hardness+Based+Reactivity+Descriptors+for+Predicting+Intra-+and+Intermolecular+Reactivity+Sequences%3A+Carbonyl+Compounds&doi=10.1021%2Fjp973450v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Local softness and hardness based reactivity descriptors for predicting intra- and intermolecular reactivity sequences: carbonyl compounds</span></div><div class="casAuthors">Roy, R. K.; Krishnamurti, S.; Geerlings, P.; Pal, S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3746-3755</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DFT-based reactivity descriptors local softness and local hardness are used as reactivity indexes to predict the reactivity sequences (both intramol. and intermol.) of carbonyl compds. toward nucleophilic attack.  The finite-difference approxn. is used to calc. local softness, whereas local hardness is approximated by -Vel/2N, where Vel is the electronic part of the mol. electrostatic potential.  Both aldehydes and ketones, aliph. and arom., were selected as systems.  Crit. cases, e.g., PhCH:CHCHO, MeCH:CHCHO, and CH2:CHCHO, where a C:C double bond is conjugated with the C:O group, are also considered.  Two new reactivity descriptors are proposed, relative electrophilicity (sk+/sk-) and relative nucleophilicity (sk-/sk+), which will help to locate the preferable reactive sites.  Local hardness can be used as a guiding parameter when constructing intermol. reactivity sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriGb2mLcYtX7Vg90H21EOLACvtfcHk0lhiYI2TB7XCAw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis12nt7k%253D&md5=c6d1906eee251453d99bc094f25d038b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjp973450v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp973450v%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DR.%2BK.%26aulast%3DKrishnamurti%26aufirst%3DS.%26aulast%3DGeerlings%26aufirst%3DP.%26aulast%3DPal%26aufirst%3DS.%26atitle%3DLocal%2520Softness%2520and%2520Hardness%2520Based%2520Reactivity%2520Descriptors%2520for%2520Predicting%2520Intra-%2520and%2520Intermolecular%2520Reactivity%2520Sequences%253A%2520Carbonyl%2520Compounds%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D1998%26volume%3D102%26issue%3D21%26spage%3D3746%26epage%3D3755%26doi%3D10.1021%2Fjp973450v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, W.</span></span> <span> </span><span class="NLM_article-title">A Combined QM/MM Study on the Reductive Half-Reaction of Xanthine Oxidase: Substrate Orientation and Mechanism</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">14885</span>â <span class="NLM_lpage">14902</span>, <span class="refDoi">Â DOI: 10.1021/ja9045394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9045394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtF2ltLjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=14885-14902&issue=41&author=S.+Metzauthor=W.+Thiel&title=A+Combined+QM%2FMM+Study+on+the+Reductive+Half-Reaction+of+Xanthine+Oxidase%3A+Substrate+Orientation+and+Mechanism&doi=10.1021%2Fja9045394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Combined QM/MM Study on the Reductive Half-Reaction of Xanthine Oxidase: Substrate Orientation and Mechanism</span></div><div class="casAuthors">Metz, Sebastian; Thiel, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">14885-14902</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Quantum mech./mol. mech. (QM/MM) methods were used to investigate the conversion of xanthine to uric acid in xanthine oxidase.  Seven mechanistic variants were considered with different tautomeric forms of xanthine, different protonation states of the active-site residues, and different substrate orientations.  The most favorable pathway (setup G) has a B3LYP/MM barrier of about 14 kcal mol-1, consistent with the available exptl. data.  This multistep mechanism starts with Glu1261 deprotonating the xanthine at the N3 position followed by a proton transfer from the cofactor to the N9 atom of xanthine; the thus activated cofactor and substrate then react to form a tetrahedral intermediate, and a subsequent rate-limiting hydride transfer generates the product.  The substrate orientation that has commonly been assumed in the literature leads to the most stable reactant complex, but the opposite orientation ("upside down") is computed to be the most favorable one during the reaction (setup G).  In the "upside down" conformation, the Arg880 residue can best stabilize the reactive xanthine species with the neg. charged N3 atom, esp. the tetrahedral intermediate and the following transition state for hydride transfer which is generally the highest point on the computed energy profiles.  QM-only calcns. for a minimal gas-phase model and for larger cluster models are performed for comparison, in particular for establishing intrinsic reactivities and a common energy scale.  An anal. of the computational results provides detailed insight into the essential mechanistic role of the active-site residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-z1Egy9O7xLVg90H21EOLACvtfcHk0lhiYI2TB7XCAw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtF2ltLjM&md5=55c9e18fc825b1b019b7a65ab8040a6a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja9045394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9045394%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DS.%26aulast%3DThiel%26aufirst%3DW.%26atitle%3DA%2520Combined%2520QM%252FMM%2520Study%2520on%2520the%2520Reductive%2520Half-Reaction%2520of%2520Xanthine%2520Oxidase%253A%2520Substrate%2520Orientation%2520and%2520Mechanism%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26issue%3D41%26spage%3D14885%26epage%3D14902%26doi%3D10.1021%2Fja9045394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raileanu, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, K.</span></span> <span> </span><span class="NLM_article-title">Theoretical Comparison Between the Gini Index and Information Gain Criteria</span>. <i>Annals of Mathematics and Artificial Intelligence</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">77</span>â <span class="NLM_lpage">93</span>, <span class="refDoi">Â DOI: 10.1023/B:AMAI.0000018580.96245.c6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1023%2FB%3AAMAI.0000018580.96245.c6" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2004&pages=77-93&issue=1&author=L.+E.+Raileanuauthor=K.+Stoffel&title=Theoretical+Comparison+Between+the+Gini+Index+and+Information+Gain+Criteria&doi=10.1023%2FB%3AAMAI.0000018580.96245.c6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1023%2FB%3AAMAI.0000018580.96245.c6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253AAMAI.0000018580.96245.c6%26sid%3Dliteratum%253Aachs%26aulast%3DRaileanu%26aufirst%3DL.%2BE.%26aulast%3DStoffel%26aufirst%3DK.%26atitle%3DTheoretical%2520Comparison%2520Between%2520the%2520Gini%2520Index%2520and%2520Information%2520Gain%2520Criteria%26jtitle%3DAnnals%2520of%2520Mathematics%2520and%2520Artificial%2520Intelligence%26date%3D2004%26volume%3D41%26issue%3D1%26spage%3D77%26epage%3D93%26doi%3D10.1023%2FB%3AAMAI.0000018580.96245.c6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Potent Inhibition of Human Liver Aldehyde Oxidase by Raloxifene</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">89</span>â <span class="NLM_lpage">97</span>, <span class="refDoi">Â DOI: 10.1124/dmd.32.1.89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.32.1.89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=14709625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFWqs7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=89-97&issue=1&author=R.+S.+Obach&title=Potent+Inhibition+of+Human+Liver+Aldehyde+Oxidase+by+Raloxifene&doi=10.1124%2Fdmd.32.1.89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibition of human liver aldehyde oxidase by raloxifene</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-97</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The selective estrogen receptor modulator, raloxifene, has been demonstrated as a potent uncompetitive inhibitor of human liver aldehyde oxidase-catalyzed oxidn. of phthalazine, vanillin, and nicotine-Î1'(5')-iminium ion, with Ki values of 0.87 to 1.4 nM.  Inhibition was not time-dependent.  Raloxifene has also been shown to be a noncompetitive inhibitor of an aldehyde oxidase-catalyzed redn. reaction of a hydroxamic acid-contg. compd., with a Ki of 51 nM.  However, raloxifene had only small effects on xanthine oxidase, an enzyme related to aldehyde oxidase.  In addn., several other compds. of the same therapeutic class as raloxifene were examd. for their potential to inhibit aldehyde oxidase.  However, none were as potent as raloxifene, since IC50 values were orders of magnitude higher and ranged from 0.29 to 57 Î¼M.  In an examn. of analogs of raloxifene, it was shown that the bisphenol structure with a hydrophobic group on the 3-position of the benzothiophene ring system was the most important element that imparts inhibitory potency.  The relevance of these data to the mechanistic understanding of aldehyde oxidase catalysis, as well as to the potential for raloxifene to cause drug interactions with agents for which aldehyde oxidase-mediated metab. is important, such as zaleplon or famciclovir, is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6iDLIEl8GgrVg90H21EOLACvtfcHk0lheQ6dTI33VzQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFWqs7w%253D&md5=a693f57077e2d685eea25a998f4603d7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.1.89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.1.89%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPotent%2520Inhibition%2520of%2520Human%2520Liver%2520Aldehyde%2520Oxidase%2520by%2520Raloxifene%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26issue%3D1%26spage%3D89%26epage%3D97%26doi%3D10.1124%2Fdmd.32.1.89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelevitz, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Hydralazine as a Selective Probe Inactivator of Aldehyde Oxidase in Human Hepatocytes: Estimation of the Contribution of Aldehyde Oxidase to Metabolic Clearance</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1441</span>â <span class="NLM_lpage">1448</span>, <span class="refDoi">Â DOI: 10.1124/dmd.112.045195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.112.045195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22522748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1441-1448&issue=7&author=T.+J.+Strelevitzauthor=C.+C.+Orozcoauthor=R.+S.+Obach&title=Hydralazine+as+a+Selective+Probe+Inactivator+of+Aldehyde+Oxidase+in+Human+Hepatocytes%3A+Estimation+of+the+Contribution+of+Aldehyde+Oxidase+to+Metabolic+Clearance&doi=10.1124%2Fdmd.112.045195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Hydralazine as a selective probe inactivator of aldehyde oxidase in human hepatocytes: estimation of the contribution of aldehyde oxidase to metabolic clearance</span></div><div class="casAuthors">Strelevitz, Timothy J.; Orozco, Christine C.; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1441-1448</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AO) metab. could lead to significant underestimation of clearance in prediction of human pharmacokinetics as well as unanticipated exposure to AO-generated metabolites, if not accounted for early in drug research.  We report a method using cryopreserved human hepatocytes and the time-dependent AO inhibitor hydralazine (KI = 83 Â± 27 Î¼M, kintact = 0.063 Â± 0.007 min-1), which ests. the contribution of AO metab. relative to total hepatic clearance.  Using zaleplon as a probe substrate and simultaneously monitoring the AO-catalyzed formation of oxozaleplon and the CYP3A-catalyzed formation of desethyzaleplon in the presence of a range of hydralazine concns., it was detd. that >90% inhibition of the AO activity with minimal effect on the CYP3A activity could be achieved with 25 to 50 Î¼M hydralazine.  This method was used to est. the fraction metabolized due to AO [fm(AO)] for six compds. with clearance attributed to AO along with four other drugs not metabolized by AO.  The fm(AO) values for the AO substrates ranged between 0.49 and 0.83.  Differences in estd. fm(AO) between two batches of pooled human hepatocytes suggest that sensitivity to hydralazine varies slightly with hepatocyte prepns.  Substrates with a CYP2D6 contribution to clearance were affected by hydralazine to a minor extent, because of weak inhibition of this enzyme.  Overall, these findings demonstrate that hydralazine, at a concn. of 25 to 50 Î¼M, can be used in human hepatocyte incubations to est. the contribution of AO to the hepatic clearance of drugs and other compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQBKznnCiG-LVg90H21EOLACvtfcHk0lheQ6dTI33VzQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vJ&md5=6237313df0fe4ded2d2f37e55c3d4b03</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.045195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.045195%26sid%3Dliteratum%253Aachs%26aulast%3DStrelevitz%26aufirst%3DT.%2BJ.%26aulast%3DOrozco%26aufirst%3DC.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DHydralazine%2520as%2520a%2520Selective%2520Probe%2520Inactivator%2520of%2520Aldehyde%2520Oxidase%2520in%2520Human%2520Hepatocytes%253A%2520Estimation%2520of%2520the%2520Contribution%2520of%2520Aldehyde%2520Oxidase%2520to%2520Metabolic%2520Clearance%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26issue%3D7%26spage%3D1441%26epage%3D1448%26doi%3D10.1124%2Fdmd.112.045195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elion, G. B.</span></span> <span> </span><span class="NLM_article-title">On the Mechanism of Inactivation of Xanthine Oxidase by Allopurinol and Other Pyrazolo[3,4-d]pyrimidines</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>245</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2837</span>â <span class="NLM_lpage">2844</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=5467924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADyaE3cXkt1Wlsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=245&publication_year=1970&pages=2837-2844&issue=11&author=V.+Masseyauthor=H.+Komaiauthor=G.+Palmerauthor=G.+B.+Elion&title=On+the+Mechanism+of+Inactivation+of+Xanthine+Oxidase+by+Allopurinol+and+Other+Pyrazolo%5B3%2C4-d%5Dpyrimidines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of inactivation of xanthine oxidase by allopurinol and other pyrazolo[3,4-d] pyrimidines</span></div><div class="casAuthors">Massey, Vincent; Komai, Hirochika; Palmer, Graham; Elion, Gertrude B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">245</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2837-44</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Allopurinol and other 6-unsubstituted pyrazolo[3,4-d]pyrimidines have been shown to reduce all of the oxidn.-redn. reactive groups of milk xanthine oxidase.  The great inhibitory action of these compds. has been shown to be due to a sort of suicide reaction, in which the product formed by enzymic hydroxylation in position 6 is joined in complex with a partially reduced form of the enzyme.  Several lines of evidence are presented indicating that complex formation is with the enzyme-bound molybdenum in the Mo(IV) state.  Studies on the stoichiometry of binding of the product of allopurinol reaction, alloxanthine(4,6-dihydroxypyrazolo[3,4 -d]pyrimidine), revealed that the amt. of alloxanthine bound was proportional to the percentage of functional active sites in the enzyme prepn.; i.e. it serves as an active site label.  From these results it is concluded that the best prepns. of xanthine oxidase so far obtained contain only 75-80% functional active sites, even though they possess their full complement of flavine, Mo, and Fe-S components.  Evidence is presented that the presence of nonfunctional active sites is responsible for several of the slow reactions previously observed with the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopDeldmbam1rVg90H21EOLACvtfcHk0lheQ6dTI33VzQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXkt1Wlsbc%253D&md5=f766d55ced0d40a1c95700bdd14cf1c0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMassey%26aufirst%3DV.%26aulast%3DKomai%26aufirst%3DH.%26aulast%3DPalmer%26aufirst%3DG.%26aulast%3DElion%26aufirst%3DG.%2BB.%26atitle%3DOn%2520the%2520Mechanism%2520of%2520Inactivation%2520of%2520Xanthine%2520Oxidase%2520by%2520Allopurinol%2520and%2520Other%2520Pyrazolo%255B3%252C4-d%255Dpyrimidines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1970%26volume%3D245%26issue%3D11%26spage%3D2837%26epage%3D2844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maduskuie, T. P.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falahatpisheh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeleswaram, S.</span></span> <span> </span><span class="NLM_article-title">Species-Specific Metabolism of SGX523 by Aldehyde Oxidase and the Toxicological Implications</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1277</span>â <span class="NLM_lpage">1285</span>, <span class="refDoi">Â DOI: 10.1124/dmd.110.032375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.110.032375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=20421447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVWktrjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=1277-1285&issue=8&author=S.+Diamondauthor=J.+Boerauthor=T.+P.+Maduskuieauthor=N.+Falahatpishehauthor=Y.+Liauthor=S.+Yeleswaram&title=Species-Specific+Metabolism+of+SGX523+by+Aldehyde+Oxidase+and+the+Toxicological+Implications&doi=10.1124%2Fdmd.110.032375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Species-specific metabolism of SGX523 by aldehyde oxidase and the toxicological implications</span></div><div class="casAuthors">Diamond, Sharon; Boer, Jason; Maduskuie, Thomas P., Jr.; Falahatpisheh, Nikoo; Li, Yu; Yeleswaram, Swamy</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1277-1285</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">An investigation was conducted to follow up on the apparent species-dependent toxicity reported for 6-(6-(1-methyl-1H-pyrazol-4-yl)-[1,2,4]triazolo[4,3-b]pyridazin-3-ylthio]quinoline (SGX523), a mesenchymal-epithelial transition factor (c-MET) inhibitor that entered clin. development for the treatment of solid tumors.  Patients treated with SGX523 exhibited compromised renal function presumably resulting from crystal deposits in renal tubules.  Our independent metabolite profiling of SGX523 indicates that a major NADPH-independent, late-eluting metabolite [6-(6-(1-methyl-1H-pyrazol-4-yl)-[1,2,4]triazolo[4,3-b]pyridazin-3-ylthio)quinolin-2(1H)-one (M11)] was generated by monkey and human liver S-9, and to a lesser extent by rat S-9, whereas M11 was absent in dog S-9 incubations.  We confirmed the identity of M11 as 2-quinolinone-SGX523.  Expts. with various molybdenum hydroxylase inhibitors showed that aldehyde oxidase (AO), and not xanthine oxidase, metabolized SGX523 to M11 in monkey and human liver cytosol.  In addn., the oxygen incorporated into M11 was derived from water rather than atm. oxygen, corroborating M11 formation via AO.  After oral dosing in monkeys, metabolite profiling of plasma and urine showed that SGX523 was indeed metabolized to M11 and its N-demethyl analog (M8).  In urine, M11 levels were â¼70-fold greater than that of SGX523, and the soly. of M11 in urine was only 3% of that of SGX523.  In summary, SGX523 is metabolized by AO in a species-specific manner to a markedly less-sol. metabolite, M11.  We propose that M11 is likely involved in the obsd. obstructive nephropathy reported in clin. studies.  Moreover, this study illustrates the need to conduct thorough metabolic evaluations early in drug development to select the most relevant nonclin. species for toxicol. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryx8buDjc41LVg90H21EOLACvtfcHk0lheQ6dTI33VzQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVWktrjM&md5=27450436ad7a69d72d4cc8ae9860ccd9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1124%2Fdmd.110.032375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.110.032375%26sid%3Dliteratum%253Aachs%26aulast%3DDiamond%26aufirst%3DS.%26aulast%3DBoer%26aufirst%3DJ.%26aulast%3DMaduskuie%26aufirst%3DT.%2BP.%26aulast%3DFalahatpisheh%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYeleswaram%26aufirst%3DS.%26atitle%3DSpecies-Specific%2520Metabolism%2520of%2520SGX523%2520by%2520Aldehyde%2520Oxidase%2520and%2520the%2520Toxicological%2520Implications%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2010%26volume%3D38%26issue%3D8%26spage%3D1277%26epage%3D1285%26doi%3D10.1124%2Fdmd.110.032375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lolkema, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohets, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkenau, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lampo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, M. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Doorn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellemans, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eskens, F.</span></span> <span> </span><span class="NLM_article-title">The c-Met Tyrosine Kinase Inhibitor JNJ-38877605 Causes Renal Toxicity Through Species-Specific Insoluble Metabolite Formation</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2297</span>â <span class="NLM_lpage">2304</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-14-3258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1158%2F1078-0432.CCR-14-3258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=25745036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslOktbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2297-2304&issue=10&author=M.+P.+Lolkemaauthor=H.+H.+Bohetsauthor=H.+T.+Arkenauauthor=A.+Lampoauthor=E.+Baraleauthor=M.+J.+A.+de+Jongeauthor=L.+van+Doornauthor=P.+Hellemansauthor=J.+S.+de+Bonoauthor=F.+Eskens&title=The+c-Met+Tyrosine+Kinase+Inhibitor+JNJ-38877605+Causes+Renal+Toxicity+Through+Species-Specific+Insoluble+Metabolite+Formation&doi=10.1158%2F1078-0432.CCR-14-3258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The c-Met Tyrosine Kinase Inhibitor JNJ-38877605 Causes Renal Toxicity through Species-Specific Insoluble Metabolite Formation</span></div><div class="casAuthors">Lolkema, Martijn P.; Bohets, Hilde H.; Arkenau, Hendrik-Tobias; Lampo, Ann; Barale, Erio; de Jonge, Maja J. A.; van Doorn, Leni; Hellemans, Peter; de Bono, Johann S.; Eskens, Ferry A. L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2297-2304</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The receptor tyrosine kinase c-Met plays an important role in tumorigenesis and is a novel target for anticancer treatment.  This phase I, first-in-human trial, explored safety, pharmacokinetics, pharmacodynamics, and initial antitumor activity of JNJ-38877605, a potent and selective c-Met inhibitor.  Exptl. Design: We performed a phase I dose-escalation study according to the std. 3+3 design.  Results: Even at subtherapeutic doses, mild though recurrent renal toxicity was obsd. in virtually all patients.  Renal toxicity had not been obsd. in preclin. studies in rats and dogs.  Addnl. preclin. studies pointed toward the rabbit as a suitable toxicol. model, as the formation of the M10 metabolite of JNJ-38877605 specifically occurred in rabbits and humans.  Addnl. toxicol. studies in rabbits clearly demonstrated that JNJ-38877605 induced species-specific renal toxicity.  Histopathol. evaluation in rabbits revealed renal crystal formation with degenerative and inflammatory changes.  Identification of the components of these renal crystals revealed M1/3 and M5/6 metabolites.  Accordingly, it was found that humans and rabbits showed significantly increased systemic exposure to these metabolites relative to other species.  These main culprit insol. metabolites were generated by aldehyde oxidase activity.  Alternative dosing schedules of JNJ-3877605 and concomitant probenecid administration in rabbits failed to prevent renal toxicity at dose levels that could be pharmacol. active.  Conclusions: Combined clin. and correlative preclin. studies suggest that renal toxicity of JNJ-38877605 is caused by the formation of species-specific insol. metabolites.  These observations preclude further clin. development of JNJ-38877605.  Clin Cancer Res; 21(10); 2297-304. Â©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro1cpkCcM_IbVg90H21EOLACvtfcHk0liA3GwF5dk8aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslOktbc%253D&md5=9900b077fbb0766d9c26b3908e7ee1c4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-3258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-3258%26sid%3Dliteratum%253Aachs%26aulast%3DLolkema%26aufirst%3DM.%2BP.%26aulast%3DBohets%26aufirst%3DH.%2BH.%26aulast%3DArkenau%26aufirst%3DH.%2BT.%26aulast%3DLampo%26aufirst%3DA.%26aulast%3DBarale%26aufirst%3DE.%26aulast%3Dde%2BJonge%26aufirst%3DM.%2BJ.%2BA.%26aulast%3Dvan%2BDoorn%26aufirst%3DL.%26aulast%3DHellemans%26aufirst%3DP.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26aulast%3DEskens%26aufirst%3DF.%26atitle%3DThe%2520c-Met%2520Tyrosine%2520Kinase%2520Inhibitor%2520JNJ-38877605%2520Causes%2520Renal%2520Toxicity%2520Through%2520Species-Specific%2520Insoluble%2520Metabolite%2520Formation%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26issue%3D10%26spage%3D2297%26epage%3D2304%26doi%3D10.1158%2F1078-0432.CCR-14-3258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of (S)-1-(1-(Imidazo[1,2-a]pyridin-6-yl)ethyl)-6-(1-methyl-1H-pyrazol-4-yl)-1H-[1,2, 3]triazolo[4,5-b]pyrazine (Volitinib) as a Highly Potent and Selective Mesenchymal-Epithelial Transition Factor (c-Met) Inhibitor in Clinical Development for Treatment of Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">7577</span>â <span class="NLM_lpage">7589</span>, <span class="refDoi">Â DOI: 10.1021/jm500510f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500510f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSns7jN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7577-7589&issue=18&author=H.+Jiaauthor=G.+Daiauthor=J.+Wengauthor=Z.+Zhangauthor=Q.+Wangauthor=F.+Zhouauthor=L.+Jiaoauthor=Y.+Cuiauthor=Y.+Renauthor=S.+Fanauthor=J.+Zhouauthor=W.+Qingauthor=Y.+Guauthor=J.+Wangauthor=Y.+Saiauthor=W.+Su&title=Discovery+of+%28S%29-1-%281-%28Imidazo%5B1%2C2-a%5Dpyridin-6-yl%29ethyl%29-6-%281-methyl-1H-pyrazol-4-yl%29-1H-%5B1%2C2%2C+3%5Dtriazolo%5B4%2C5-b%5Dpyrazine+%28Volitinib%29+as+a+Highly+Potent+and+Selective+Mesenchymal-Epithelial+Transition+Factor+%28c-Met%29+Inhibitor+in+Clinical+Development+for+Treatment+of+Cancer&doi=10.1021%2Fjm500510f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of (S)-1-(1-(Imidazo[1,2-a]pyridin-6-yl)ethyl)-6-(1-methyl-1H-pyrazol-4-yl)-1H-[1,2,3]triazolo[4,5-b]pyrazine (Volitinib) as a Highly Potent and Selective Mesenchymal-Epithelial Transition Factor (c-Met) Inhibitor in Clinical Development for Treatment of Cancer</span></div><div class="casAuthors">Jia, Hong; Dai, Guangxiu; Weng, Jianyang; Zhang, Zhulin; Wang, Qing; Zhou, Feng; Jiao, Longxian; Cui, Yumin; Ren, Yongxin; Fan, Shiming; Zhou, Jinghong; Qing, Weiguo; Gu, Yi; Wang, Jian; Sai, Yang; Su, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7577-7589</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HGF/c-Met signaling has been implicated in human cancers.  Herein we describe the invention of a series of novel triazolopyrazine c-Met inhibitors.  The structure-activity relation of these compds. was investigated, leading to the identification of Volitinib (I), which demonstrated favorable pharmacokinetic properties in mice and good antitumor activities in the human glioma xenograft model in athymic nude mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJmuzIGrPGZbVg90H21EOLACvtfcHk0liA3GwF5dk8aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSns7jN&md5=aced38bbd7431f5ac01b6698e8657261</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm500510f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500510f%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DH.%26aulast%3DDai%26aufirst%3DG.%26aulast%3DWeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DJiao%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DQing%26aufirst%3DW.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSai%26aufirst%3DY.%26aulast%3DSu%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520%2528S%2529-1-%25281-%2528Imidazo%255B1%252C2-a%255Dpyridin-6-yl%2529ethyl%2529-6-%25281-methyl-1H-pyrazol-4-yl%2529-1H-%255B1%252C2%252C%25203%255Dtriazolo%255B4%252C5-b%255Dpyrazine%2520%2528Volitinib%2529%2520as%2520a%2520Highly%2520Potent%2520and%2520Selective%2520Mesenchymal-Epithelial%2520Transition%2520Factor%2520%2528c-Met%2529%2520Inhibitor%2520in%2520Clinical%2520Development%2520for%2520Treatment%2520of%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D18%26spage%3D7577%26epage%3D7589%26doi%3D10.1021%2Fjm500510f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, X.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase-Dependent Species Difference in Hepatic Metabolism of Fasudil to Hydroxyfasudil</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">170</span>â <span class="NLM_lpage">177</span>, <span class="refDoi">Â DOI: 10.1080/00498254.2017.1292016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1080%2F00498254.2017.1292016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28166443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Gksbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=170-177&issue=2&author=Z.+Maoauthor=Y.+Wuauthor=Q.+Liauthor=X.+Wangauthor=Y.+Liuauthor=X.+Di&title=Aldehyde+Oxidase-Dependent+Species+Difference+in+Hepatic+Metabolism+of+Fasudil+to+Hydroxyfasudil&doi=10.1080%2F00498254.2017.1292016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde oxidase-dependent species difference in hepatic metabolism of fasudil to hydroxyfasudil</span></div><div class="casAuthors">Mao, Zhengsheng; Wu, Yali; Li, Qiuying; Wang, Xin; Liu, Youping; Di, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">170-177</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">An investigation on the metabolic mechanism of fasudil to hydroxyfasudil was conducted in vitro using liver subcellular fractions of different species.  Hydroxyfasudil was generated in large amts. by rat liver S9 and to a similar extent by human liver S9 but was not detected in dog liver S9 incubations.  Studies with various molybdenum hydroxylase inhibitors demonstrated that aldehyde oxidase (AO), but not xanthine oxidase (XO), selectively catalyzed fasudil to hydroxyfasudil in both rat and human liver cytosol.  In addn., the oxygen atom incorporated into hydroxyfasudil was derived from water rather than atm. oxygen, which further corroborated AO involvement.  Enzyme kinetics expts. revealed that fasudil had a higher affinity to human hepatic AO than to rat hepatic AO.  Besides, significantly different in vivo pharmacokinetic parameters obsd. between male and female rats indicated that the AO activity in rats was gender-dependent.  The present study provided first evidences that AO causes differences in fasudil metab. between species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLweM3FIR00rVg90H21EOLACvtfcHk0lgMJZ086zb19w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Gksbk%253D&md5=03b6d2441e74bc60045e1fdcb192a0ec</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F00498254.2017.1292016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00498254.2017.1292016%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDi%26aufirst%3DX.%26atitle%3DAldehyde%2520Oxidase-Dependent%2520Species%2520Difference%2520in%2520Hepatic%2520Metabolism%2520of%2520Fasudil%2520to%2520Hydroxyfasudil%26jtitle%3DXenobiotica%26date%3D2018%26volume%3D48%26issue%3D2%26spage%3D170%26epage%3D177%26doi%3D10.1080%2F00498254.2017.1292016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamidass, S. J.</span></span> <span> </span><span class="NLM_article-title">Modeling Reactivity to Biological Macromolecules With a Deep Multitask Network</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">529</span>â <span class="NLM_lpage">537</span>, <span class="refDoi">Â DOI: 10.1021/acscentsci.6b00162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1CktL%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=529-537&issue=8&author=T.+B.+Hughesauthor=N.+L.+Dangauthor=G.+P.+Millerauthor=S.+J.+Swamidass&title=Modeling+Reactivity+to+Biological+Macromolecules+With+a+Deep+Multitask+Network&doi=10.1021%2Facscentsci.6b00162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Reactivity to Biological Macromolecules with a Deep Multitask Network</span></div><div class="casAuthors">Hughes, Tyler B.; Dang, Na Le; Miller, Grover P.; Swamidass, S. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">529-537</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most small-mol. drug candidates fail before entering the market, frequently because of unexpected toxicity.  Often, toxicity is detected only late in drug development, because many types of toxicities, esp. idiosyncratic adverse drug reactions (IADRs), are particularly hard to predict and detect.  Moreover, drug-induced liver injury (DILI) is the most frequent reason drugs are withdrawn from the market and causes 50% of acute liver failure cases in the United States.  A common mechanism often underlies many types of drug toxicities, including both DILI and IADRs.  Drugs are bioactivated by drug-metabolizing enzymes into reactive metabolites, which then conjugate to sites in proteins or DNA to form adducts.  DNA adducts are often mutagenic and may alter the reading and copying of genes and their regulatory elements, causing gene dysregulation and even triggering cancer.  Similarly, protein adducts can disrupt their normal biol. functions and induce harmful immune responses.  Unfortunately, reactive metabolites are not reliably detected by expts., and it is also expensive to test drug candidates for potential to form DNA or protein adducts during the early stages of drug development.  In contrast, computational methods have the potential to quickly screen for covalent binding potential, thereby flagging problematic mols. and reducing the total no. of necessary expts.  Here, the authors train a deep convolution neural network-the XenoSite reactivity model-using literature data to accurately predict both sites and probability of reactivity for mols. with glutathione, cyanide, protein, and DNA.  On the site level, cross-validated predictions had area under the curve (AUC) performances of 89.8% for DNA and 94.4% for protein.  Furthermore, the model sepd. mols. electrophilically reactive with DNA and protein from nonreactive mols. with cross-validated AUC performances of 78.7% and 79.8%, resp.  On both the site- and mol.-level, the model's performances significantly outperformed reactivity indexes derived from quantum simulations that are reported in the literature.  Moreover, the authors developed and applied a selectivity score to assess preferential reactions with the macromols. as opposed to the common screening traps.  For the entire data set of 2803 mols., this approach yielded totals of 257 (9.2%) and 227 (8.1%) mols. predicted to be reactive only with DNA and protein, resp., and hence those that would be missed by std. reactivity screening expts.  Site of reactivity data is an underutilized resource that can be used to not only predict if mols. are reactive, but also show where they might be modified to reduce toxicity while retaining efficacy.  The XenoSite reactivity model is available at http://swami.wustl.edu/xenosite/p/reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC4SVmokCrNbVg90H21EOLACvtfcHk0lgMJZ086zb19w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1CktL%252FN&md5=33cbff16248c5fa113a00539d9d9ecd5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00162%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DT.%2BB.%26aulast%3DDang%26aufirst%3DN.%2BL.%26aulast%3DMiller%26aufirst%3DG.%2BP.%26aulast%3DSwamidass%26aufirst%3DS.%2BJ.%26atitle%3DModeling%2520Reactivity%2520to%2520Biological%2520Macromolecules%2520With%2520a%2520Deep%2520Multitask%2520Network%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26issue%3D8%26spage%3D529%26epage%3D537%26doi%3D10.1021%2Facscentsci.6b00162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G. E.</span></span> In  <i>Neural Networks: Tricks of the Trade</i>; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">599</span>â <span class="NLM_lpage">619</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1007%2F978-3-642-35289-8_32" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=599-619&author=G.+E.+Hinton&title=Neural+Networks%3A+Tricks+of+the+Trade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-35289-8_32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-35289-8_32%26sid%3Dliteratum%253Aachs%26aulast%3DHinton%26aufirst%3DG.%2BE.%26btitle%3DNeural%2520Networks%253A%2520Tricks%2520of%2520the%2520Trade%26pub%3DSpringer%26date%3D2012%26spage%3D599%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speybroeck, N.</span></span> <span> </span><span class="NLM_article-title">Classification and Regression Trees</span>. <i>International Journal of Public Health</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">243</span>â <span class="NLM_lpage">246</span>, <span class="refDoi">Â DOI: 10.1007/s00038-011-0315-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1007%2Fs00038-011-0315-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22015650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A280%3ADC%252BC383ksVynug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=243-246&issue=1&author=N.+Speybroeck&title=Classification+and+Regression+Trees&doi=10.1007%2Fs00038-011-0315-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Classification and regression trees</span></div><div class="casAuthors">Speybroeck N</div><div class="citationInfo"><span class="NLM_cas:title">International journal of public health</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2T0B6WFwfoUodMGTatQ7LfW6udTcc2eYZ6Sb_TMzBirntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC383ksVynug%253D%253D&md5=d0a601b734fb50f9291317ec45ccfacc</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs00038-011-0315-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00038-011-0315-z%26sid%3Dliteratum%253Aachs%26aulast%3DSpeybroeck%26aufirst%3DN.%26atitle%3DClassification%2520and%2520Regression%2520Trees%26jtitle%3DInternational%2520Journal%2520of%2520Public%2520Health%26date%3D2012%26volume%3D57%26issue%3D1%26spage%3D243%26epage%3D246%26doi%3D10.1007%2Fs00038-011-0315-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henrard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speybroeck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, C.</span></span> <span> </span><span class="NLM_article-title">Classification and Regression Tree Analysis vs. Multivariable Linear and Logistic Regression Methods as Statistical Tools for Studying Haemophilia</span>. <i>Haemophilia</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">715</span>â <span class="NLM_lpage">722</span>, <span class="refDoi">Â DOI: 10.1111/hae.12778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1111%2Fhae.12778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=26248714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A280%3ADC%252BC287gsVaqsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=715-722&issue=6&author=S.+Henrardauthor=N.+Speybroeckauthor=C.+Hermans&title=Classification+and+Regression+Tree+Analysis+vs.+Multivariable+Linear+and+Logistic+Regression+Methods+as+Statistical+Tools+for+Studying+Haemophilia&doi=10.1111%2Fhae.12778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Classification and regression tree analysis vs. multivariable linear and logistic regression methods as statistical tools for studying haemophilia</span></div><div class="casAuthors">Henrard S; Speybroeck N; Henrard S; Hermans C</div><div class="citationInfo"><span class="NLM_cas:title">Haemophilia : the official journal of the World Federation of Hemophilia</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">715-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Haemophilia is a rare genetic haemorrhagic disease characterized by partial or complete deficiency of coagulation factor VIII, for haemophilia A, or IX, for haemophilia B.  As in any other medical research domain, the field of haemophilia research is increasingly concerned with finding factors associated with binary or continuous outcomes through multivariable models.  Traditional models include multiple logistic regressions, for binary outcomes, and multiple linear regressions for continuous outcomes.  Yet these regression models are at times difficult to implement, especially for non-statisticians, and can be difficult to interpret.  AIMS:  The present paper sought to didactically explain how, why, and when to use classification and regression tree (CART) analysis for haemophilia research.  MATERIALS & METHODS:  The CART method is non-parametric and non-linear, based on the repeated partitioning of a sample into subgroups based on a certain criterion.  Breiman developed this method in 1984.  Classification trees (CTs) are used to analyse categorical outcomes and regression trees (RTs) to analyse continuous ones.  RESULTS:  The CART methodology has become increasingly popular in the medical field, yet only a few examples of studies using this methodology specifically in haemophilia have to date been published.  Two examples using CART analysis and previously published in this field are didactically explained in details.  CONCLUSION:  There is increasing interest in using CART analysis in the health domain, primarily due to its ease of implementation, use, and interpretation, thus facilitating medical decision-making.  This method should be promoted for analysing continuous or categorical outcomes in haemophilia, when applicable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcScPq4eJV7QaEh5eUSa2oe_fW6udTcc2eZnjbwXecJpe7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287gsVaqsQ%253D%253D&md5=04ddd829a199f7144b3ee6c768160fb1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1111%2Fhae.12778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fhae.12778%26sid%3Dliteratum%253Aachs%26aulast%3DHenrard%26aufirst%3DS.%26aulast%3DSpeybroeck%26aufirst%3DN.%26aulast%3DHermans%26aufirst%3DC.%26atitle%3DClassification%2520and%2520Regression%2520Tree%2520Analysis%2520vs.%2520Multivariable%2520Linear%2520and%2520Logistic%2520Regression%2520Methods%2520as%2520Statistical%2520Tools%2520for%2520Studying%2520Haemophilia%26jtitle%3DHaemophilia%26date%3D2015%26volume%3D21%26issue%3D6%26spage%3D715%26epage%3D722%26doi%3D10.1111%2Fhae.12778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nikhilesh V.  Dhuria</span>, <span class="hlFld-ContribAuthor ">Bianka  Haro</span>, <span class="hlFld-ContribAuthor ">Amit  Kapadia</span>, <span class="hlFld-ContribAuthor ">Khadjia A.  Lobo</span>, <span class="hlFld-ContribAuthor ">Bernice  Matusow</span>, <span class="hlFld-ContribAuthor ">Mary A.  Schleiff</span>, <span class="hlFld-ContribAuthor ">Christina  Tantoy</span>, <span class="hlFld-ContribAuthor ">Jasleen K.  Sodhi</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in predicting CYP-independent metabolism. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2021,</strong> <em>53 </em>
                                    (2)
                                     , 188-206. <a href="https://doi.org/10.1080/03602532.2021.1923728" title="DOI URL">https://doi.org/10.1080/03602532.2021.1923728</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2021.1923728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2021.1923728%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DRecent%252Bdevelopments%252Bin%252Bpredicting%252BCYP-independent%252Bmetabolism%26aulast%3DDhuria%26aufirst%3DNikhilesh%2BV.%26date%3D2021%26date%3D2021%26volume%3D53%26issue%3D2%26spage%3D188%26epage%3D206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Scheme showing the proposed mechanism of AOX-mediated oxidation. PDB structure used for illustration is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UHX">4UHX</a>,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and the substrate is phthalazine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Site of metabolism prediction results of four quinoline analogues by DT<sub>AOX</sub> model. Position 2 in these quinoline analogues is marked as black asterisk. Pred.: prediction results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Distribution of topological and electronic descriptors between substrates and nonsubstrates. (A) Distribution of molecular weight (unit, amu), molecular volume (unit, Ã<sup>3</sup>), and molecular surface area (unit, Ã<sup>2</sup>). (B) Distribution of the energy of lowest unoccupied molecular orbital (<i>E</i><sub>LUMO</sub>) (unit, eV), molecular electronegativity (unit, eV) and molecular hardness (unit: eV). (C) Distribution of the maximum and minimum values of the ESP, energies (kJ/mol) as mapped onto the electrostatic potential maps, denoted as Max_ElPot and Min_ElPot, respectively. Blue and orange colors represent the substrates and nonsubstrates, respectively. (D) <i>p</i> value of corresponding descriptors in parts AâC.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Distribution of some energy-related, hindrance effect related, and electronic descriptors between SOMs and non-SOMs. The notation of these descriptors is listed following the order from left to right: Î<i>E</i><sub>inter</sub>, the energy changes between prototype and the tetrahedral intermediate; Steric, steric hindrance; ESP, electrostatic potential charge; CS, chemical shift; EA, exposed area. The prefix N_, C_, or H_ represent the electron-rich nitrogen, electron-deficient carbon, and hydrogen attached to the electron-deficient carbon atoms, respectively. The <i>p</i> values of the descriptors between SOMs and non-SOMs were calculated using <i>t</i> test statistical hypothesis. The corresponding significance of each descriptor is depicted as black asterisk within the underlying range: (â) 10<sup>â5</sup> < <i>p</i> < 0.05; (ââ) 10<sup>â15</sup> < <i>p</i> < 10<sup>â5</sup>; (âââ) <i>p</i> < 10<sup>â15</sup>. Blue and orange colors represent the SOMs and non-SOMs, respectively. Note that all these descriptors were normalized using StandardScaler implemented in Scikit-learn version 0.21.3.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Examples of prediction ability of DNN substrate/nonsubstrate classification model on test data set. The positive and negative sites predicted by DT<sub>AOX</sub> are colored in red and blue circles, respectively. Pred.: prediction results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Decision tree model DTN<sub>AOX</sub> (A) for predicting hAOX-mediated SOMs and its performance on (B) the test data set and (C) the external data set, compared with our previous DT<sub>AOX</sub> model<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and three descriptors proposed by Montefiori et al.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Flowchart of fusing model. The data sets 1 and data sets 2 contain the data with substrate/nonsubstrate and SOMs/non-SOMs labels, respectively. The red and blue circles indicate the predicted metabolic and nonmetabolic sites, respectively. The arrows indicate the metabolic sites identified in human liver cytosols.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Predicted hAOX-catalyzed SOMs for 15 epigenetics-related inhibitors using the fusing model. Red and blue circles indicate the metabolic and nonmetabolic sites, respectively, predicted by DTN<sub>AOX</sub>. The experimentally proposed metabolic region is marked using dotted box. Pred.: prediction results. Exp.: experimental results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Ion extraction chromatographs of the parent compounds (left) and their corresponding mono-oxidized metabolites (right) formed in human liver cytosol.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Positive product ion spectra of the parent compounds (left) and their corresponding mono-oxidized metabolites (right) formed in human liver cytosol. The <i>m</i>/<i>z</i> of parent ion was colored in purple, and the <i>m</i>/<i>z</i> of relevant product ions which were applied to identify the oxidized regions was colored in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/medium/jm9b01895_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Descriptor mapping and model construction flowchart. The quinoline and DNN model structure were used to elucidate the flowchart.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-12/acs.jmedchem.9b01895/20200618/images/large/jm9b01895_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01895&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30042" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30042" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duckworth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span> <span> </span><span class="NLM_article-title">Medicinal Chemistry Approaches to Avoid Aldehyde Oxidase Metabolism</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2856</span>â <span class="NLM_lpage">2860</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2012.02.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1016%2Fj.bmcl.2012.02.069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22429467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFSitbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2856-2860&issue=8&author=D.+C.+Prydeauthor=T.+D.+Tranauthor=P.+Jonesauthor=J.+Duckworthauthor=M.+Howardauthor=I.+Gardnerauthor=R.+Hylandauthor=R.+Websterauthor=T.+Wenhamauthor=S.+Bagalauthor=K.+Omotoauthor=R.+P.+Schneiderauthor=J.+Lin&title=Medicinal+Chemistry+Approaches+to+Avoid+Aldehyde+Oxidase+Metabolism&doi=10.1016%2Fj.bmcl.2012.02.069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry approaches to avoid aldehyde oxidase metabolism</span></div><div class="casAuthors">Pryde, David C.; Tran, Thien-Duc; Jones, Peter; Duckworth, Jonathan; Howard, Martin; Gardner, Iain; Hyland, Ruth; Webster, Rob; Wenham, Tracey; Bagal, Sharan; Omoto, Kiyoyuki; Schneider, Richard P.; Lin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2856-2860</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AO) is a molybdenum-contg. enzyme distributed throughout the animal kingdom and capable of metabolising a wide range of aldehydes and N-heterocyclic compds.  Although metab. by this enzyme in man is recognized to have significant clin. impact where human AO activity was not predicted by screening in preclin. species, there is very little reported literature offering real examples where drug discoverers have successfully designed away from AO oxidn.  This article reports on some strategies adopted in the Pfizer TLR7 agonist program to successfully switch off AO metab. that was seen principally in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZKVEe9ZUzMbVg90H21EOLACvtfcHk0lig4QOj84c-WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFSitbs%253D&md5=4c4c68b64f1c7013ad978408add0b09a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.069%26sid%3Dliteratum%253Aachs%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DTran%26aufirst%3DT.%2BD.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DDuckworth%26aufirst%3DJ.%26aulast%3DHoward%26aufirst%3DM.%26aulast%3DGardner%26aufirst%3DI.%26aulast%3DHyland%26aufirst%3DR.%26aulast%3DWebster%26aufirst%3DR.%26aulast%3DWenham%26aufirst%3DT.%26aulast%3DBagal%26aufirst%3DS.%26aulast%3DOmoto%26aufirst%3DK.%26aulast%3DSchneider%26aufirst%3DR.%2BP.%26aulast%3DLin%26aufirst%3DJ.%26atitle%3DMedicinal%2520Chemistry%2520Approaches%2520to%2520Avoid%2520Aldehyde%2520Oxidase%2520Metabolism%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26issue%3D8%26spage%3D2856%26epage%3D2860%26doi%3D10.1016%2Fj.bmcl.2012.02.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sodhi, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barr, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halladay, J. S.</span></span> <span> </span><span class="NLM_article-title">A Novel Reaction Mediated by Human Aldehyde Oxidase: Amide Hydrolysis of GDC-0834</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">908</span>â <span class="NLM_lpage">915</span>, <span class="refDoi">Â DOI: 10.1124/dmd.114.061804</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.114.061804" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=25845827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlCrtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=908-915&issue=6&author=J.+K.+Sodhiauthor=S.+Wongauthor=D.+S.+Kirkpatrickauthor=L.+Liuauthor=S.+C.+Khojastehauthor=C.+E.+Hopauthor=J.+T.+Barrauthor=J.+P.+Jonesauthor=J.+S.+Halladay&title=A+Novel+Reaction+Mediated+by+Human+Aldehyde+Oxidase%3A+Amide+Hydrolysis+of+GDC-0834&doi=10.1124%2Fdmd.114.061804"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A novel reaction mediated by human aldehyde oxidase: amide hydrolysis of GDC-0834</span></div><div class="casAuthors">Sodhi, Jasleen K.; Wong, Susan; Kirkpatrick, Donald S.; Liu, Lichuan; Khojasteh, S. Cyrus; Hop, Cornelis E. C. A.; Barr, John T.; Jones, Jeffrey P.; Halladay, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">908-915</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">GDC-0834, a Bruton's tyrosine kinase inhibitor investigated as a potential treatment of rheumatoid arthritis, was previously reported to be extensively metabolized by amide hydrolysis such that no measurable levels of this compd. were detected in human circulation after oral administration.  In vitro studies in human liver cytosol detd. that GDC-0834 (R)-N-(3-(6-(4-(1,4-dimethyl-3-oxopiperazin-2-yl)phenylamino)-4-methyl-5-oxo-4,5-dihydropyrazin-2-yl)-2-methylphenyl)-4,5,6,7-tetrahydrobenzo[b]thiophene-2-carboxamide was rapidly hydrolyzed with a CLint of 0.511 mL/min per mg of protein.  Aldehyde oxidase (AO) and carboxylesterase (CES) were putatively identified as the enzymes responsible after cytosolic fractionation and mass spectrometry-proteomics anal. of the enzymically active fractions.  Results were confirmed by a series of kinetic expts. with inhibitors of AO, CES, and xanthine oxidase (XO), which implicated AO and CES, but not XO, as mediating GDC-0834 amide hydrolysis.  Further supporting the interaction between GDC-0834 and AO, GDC-0834 was shown to be a potent reversible inhibitor of six known AO substrates with IC50 values ranging from 0.86 to 1.87 Î¼M.  Addnl., in silico modeling studies suggest that GDC-0834 is capable of binding in the active site of AO with the amide bond of GDC-0834 near the molybdenum cofactor (MoCo), orientated in such a way to enable potential nucleophilic attack on the carbonyl of the amide bond by the hydroxyl of MoCo.  Together, the in vitro and in silico results suggest the involvement of AO in the amide hydrolysis of GDC-0834.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWAO_iWKyUVLVg90H21EOLACvtfcHk0lig4QOj84c-WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlCrtrw%253D&md5=4cf67605bc094945de8229be5c9cacae</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1124%2Fdmd.114.061804&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.114.061804%26sid%3Dliteratum%253Aachs%26aulast%3DSodhi%26aufirst%3DJ.%2BK.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DKirkpatrick%26aufirst%3DD.%2BS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DHop%26aufirst%3DC.%2BE.%26aulast%3DBarr%26aufirst%3DJ.%2BT.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26aulast%3DHalladay%26aufirst%3DJ.%2BS.%26atitle%3DA%2520Novel%2520Reaction%2520Mediated%2520by%2520Human%2520Aldehyde%2520Oxidase%253A%2520Amide%2520Hydrolysis%2520of%2520GDC-0834%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2015%26volume%3D43%26issue%3D6%26spage%3D908%26epage%3D915%26doi%3D10.1124%2Fdmd.114.061804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manevski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecomte, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toselli, F.</span></span> <span> </span><span class="NLM_article-title">Metabolism by Aldehyde Oxidase: Drug Design and Complementary Approaches to Challenges in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">10955</span>â <span class="NLM_lpage">10994</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.9b00875</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00875" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrbP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10955-10994&issue=24&author=N.+Manevskiauthor=L.+Kingauthor=W.+R.+Pittauthor=F.+Lecomteauthor=F.+Toselli&title=Metabolism+by+Aldehyde+Oxidase%3A+Drug+Design+and+Complementary+Approaches+to+Challenges+in+Drug+Discovery&doi=10.1021%2Facs.jmedchem.9b00875"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism by Aldehyde Oxidase: Drug Design and Complementary Approaches to Challenges in Drug Discovery</span></div><div class="casAuthors">Manevski, Nenad; King, Lloyd; Pitt, William R.; Lecomte, Fabien; Toselli, Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10955-10994</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Aldehyde oxidase (AO) catalyzes oxidns. of azaheterocycles and aldehydes, amide hydrolysis, and diverse redns.  AO substrates are rare among marketed drugs and many candidates failed due to poor pharmacokinetics, interspecies differences, and adverse effects.  As most issues arise from complex and poorly understood AO biol., an effective soln. is to stop or decrease AO metab.  This perspective focuses on rational drug design approaches to modulate AO-mediated metab. in drug discovery.  AO biol. aspects are also covered, as they are complementary to chem. design and important when selecting the exptl. system for risk assessment.  The authors' recommendation is an early consideration of AO-mediated metab. supported by computational and in vitro exptl. methods, but not an automatic avoidance of AO structural flags, many of which are versatile and valuable building blocks.  Preferably, consideration of AO mediated metab. should be part of the multiparametric drug-optimization process, with the goal to improve overall drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMW2Yo9GpLObVg90H21EOLACvtfcHk0lig4QOj84c-WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFamsrbP&md5=7fd30c75ea2190c0fc74c816feef9ba9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00875%26sid%3Dliteratum%253Aachs%26aulast%3DManevski%26aufirst%3DN.%26aulast%3DKing%26aufirst%3DL.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DLecomte%26aufirst%3DF.%26aulast%3DToselli%26aufirst%3DF.%26atitle%3DMetabolism%2520by%2520Aldehyde%2520Oxidase%253A%2520Drug%2520Design%2520and%2520Complementary%2520Approaches%2520to%2520Challenges%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D24%26spage%3D10955%26epage%3D10994%26doi%3D10.1021%2Facs.jmedchem.9b00875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paragas, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joswig-Jones, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">The Two Faces of Aldehyde Oxidase: Oxidative and Reductive Transformations of 5-Nitroquinoline</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>145</i></span>,  <span class="NLM_fpage">210</span>â <span class="NLM_lpage">217</span>, <span class="refDoi">Â DOI: 10.1016/j.bcp.2017.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1016%2Fj.bcp.2017.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28888950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKltbfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2017&pages=210-217&author=E.+M.+Paragasauthor=S.+C.+Humphreysauthor=J.+Minauthor=C.+A.+Joswig-Jonesauthor=J.+P.+Jones&title=The+Two+Faces+of+Aldehyde+Oxidase%3A+Oxidative+and+Reductive+Transformations+of+5-Nitroquinoline&doi=10.1016%2Fj.bcp.2017.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The two faces of aldehyde oxidase: Oxidative and reductive transformations of 5-nitroquinoline</span></div><div class="casAuthors">Paragas, Erickson M.; Humphreys, Sara C.; Min, Joshua; Joswig-Jones, Carolyn A.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">145</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">210-217</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a cytosolic enzyme responsible for the metab. of some drugs and drug candidates.  AOX catalyzes the oxidative hydroxylation of substrates including several aliph. and arom. aldehydes, and nitrogen-contg. heterocyclic compds.  AOX is also reported to catalyze the reductive metab. of nitro-compds., N-oxides, sulfoxides, isoxazoles, isothiazoles, nitrite and hydroxamic acids.  These reductive transformations are not well understood and are generally believed to only occur at low oxygen concns.  In this study, we used 5-nitroquinoline (5NQ) as a substrate to further understand both the oxidative and the reductive transformations catalyzed by AOX.  In vitro reaction of 5NQ with AOX under aerobic conditions generated the oxidized (2-oxo-5-nitroquinoline, 2-oxo-5NQ), the reduced (5-aminoquinoline, 5AQ) and the oxidized/reduced (2-oxo-5-aminoquinoline, 2-oxo-5AQ) metabolites.  Interestingly, in human liver cytosol, co-incubation of 5NQ and known AOX oxidative substrates DACA and phthalazine significantly increased the yield of the reduced metabolite, while oxidized metabolites prodn. decreased.  These data indicate that 5NQ can be reduced at atm. oxygen concns. and that the reductive transformation occurs at a second site that is kinetically distinct from the oxidative site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEP7SDjFQWZ7Vg90H21EOLACvtfcHk0lg1BbVmGtMhaw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKltbfE&md5=4a2acff32f835e365bdb5a079c291eab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DParagas%26aufirst%3DE.%2BM.%26aulast%3DHumphreys%26aufirst%3DS.%2BC.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DJoswig-Jones%26aufirst%3DC.%2BA.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520Two%2520Faces%2520of%2520Aldehyde%2520Oxidase%253A%2520Oxidative%2520and%2520Reductive%2520Transformations%2520of%25205-Nitroquinoline%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2017%26volume%3D145%26spage%3D210%26epage%3D217%26doi%3D10.1016%2Fj.bcp.2017.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerny, M. A.</span></span> <span> </span><span class="NLM_article-title">Prevalence of Non-Cytochrome P450-Mediated Metabolism in Food and Drug Administration-Approved Oral and Intravenous Drugs: 2006â2015</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1246</span>â <span class="NLM_lpage">1252</span>, <span class="refDoi">Â DOI: 10.1124/dmd.116.070763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.116.070763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=27084892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhsl2gurrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1246-1252&issue=8&author=M.+A.+Cerny&title=Prevalence+of+Non-Cytochrome+P450-Mediated+Metabolism+in+Food+and+Drug+Administration-Approved+Oral+and+Intravenous+Drugs%3A+2006%E2%80%932015&doi=10.1124%2Fdmd.116.070763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of non-cytochrome P450-mediated metabolism in food and drug administration-approved oral and intravenous drugs: 2006-2015</span></div><div class="casAuthors">Cerny, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1246-1252</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">In recent years, claims of increased involvement of non-cytochrome P 450 (non-P 450) enzymes in the metab. of drugs have appeared in the literature.  However, no temporal summaries of the contribution of non-P 450 enzymes to the metab. of drugs have been published.  Using data from human radiolabeled absorption, distribution, metab., and excretion studies available for a set of 125 orally or i.v. administered small-mol. drugs approved by the United States Food and Drug Administration from 2006 to 2015, the contributions of P 450 and non-P 450 enzymes to the formation of major metabolites (â¥10% of dose) were assessed and tabulated.  Over this time frame, the involvement of P 450 vs. non-P 450 enzymes in the formation of major metabolites is compared, and the individual non-P 450 enzymes responsible are described.  This anal. indicates that non-P 450 enzymes contribute significantly to the metab. of the 125 drugs analyzed.  Approx. 30% of the metab. of these drugs is carried out by non-P 450 enzymes, with the predominant non-P 450 enzymes identified being glucuronosyltransferases (11.7%), hydrolases (10.8%), carbonyl reductases (2.4%), and aldehyde oxidase (1.1%).  Although significant, the relative contribution of non-P 450 enzymes to drug metab. does not appear to have increased dramatically over the last 10 years.  As the current evaluation involves drugs which emerged from the discovery phase >10 years ago, this evaluation may not reflect the current or evolving situation in some research organizations; therefore, addnl. monitoring and assessment of the involvement of non-P 450 enzymes in the metab. of drugs will be conducted in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0bkJkoG1VbVg90H21EOLACvtfcHk0lg1BbVmGtMhaw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2gurrI&md5=7d59ddc9f2467feef2e9a46350c47df9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.070763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.070763%26sid%3Dliteratum%253Aachs%26aulast%3DCerny%26aufirst%3DM.%2BA.%26atitle%3DPrevalence%2520of%2520Non-Cytochrome%2520P450-Mediated%2520Metabolism%2520in%2520Food%2520and%2520Drug%2520Administration-Approved%2520Oral%2520and%2520Intravenous%2520Drugs%253A%25202006%25E2%2580%25932015%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26issue%3D8%26spage%3D1246%26epage%3D1252%26doi%3D10.1124%2Fdmd.116.070763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: A Combined Computational and Experimental Study</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">2973</span>â <span class="NLM_lpage">2982</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjslyitLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2973-2982&issue=7&author=Y.+Xuauthor=L.+Liauthor=Y.+Wangauthor=J.+Xingauthor=L.+Zhouauthor=D.+Zhongauthor=X.+Luoauthor=H.+Jiangauthor=K.+Chenauthor=M.+Zhengauthor=P.+Dengauthor=X.+Chen&title=Aldehyde+Oxidase+Mediated+Metabolism+in+Drug-like+Molecules%3A+A+Combined+Computational+and+Experimental+Study&doi=10.1021%2Facs.jmedchem.7b00019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Oxidase Mediated Metabolism in Drug-like Molecules: A Combined Computational and Experimental Study</span></div><div class="casAuthors">Xu, Yuan; Li, Liang; Wang, Yulan; Xing, Jing; Zhou, Lei; Zhong, Dafang; Luo, Xiaomin; Jiang, Hualiang; Chen, Kaixian; Zheng, Mingyue; Deng, Pan; Chen, Xiaoyan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2973-2982</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is an important drug-metabolizing enzyme.  However, the current in vitro models for evaluating AOX metab. are sometimes misleading, and preclin. animal models generally fail to predict human AOX-mediated metab.  In this study, the authors report a combined computational and exptl. investigation of drug-like mols. that are potential aldehyde oxidase substrates, of which multiple sites of metab. (SOMs) mediated by AOX and their preferences for the reaction can be unambiguously identified.  In addn., the proposed strategy was used to evaluate the metab. of newly designed c-Met inhibitors, and a success switch-off of AOX metab. was obsd.  Overall, this study provide useful information to guide lead optimization and drug discovery based on AOX-mediated metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPvn1B_ge-grVg90H21EOLACvtfcHk0lg1BbVmGtMhaw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjslyitLc%253D&md5=2cb27d8a575dacef529f20cfe32cf8dc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00019%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DAldehyde%2520Oxidase%2520Mediated%2520Metabolism%2520in%2520Drug-like%2520Molecules%253A%2520A%2520Combined%2520Computational%2520and%2520Experimental%2520Study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D7%26spage%3D2973%26epage%3D2982%26doi%3D10.1021%2Facs.jmedchem.7b00019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Y. J.</span></span> <span> </span><span class="NLM_article-title">Metabolism of c-Met Kinase Inhibitors Containing Quinoline by Aldehyde Oxidase, Electron Donating, and Steric Hindrance Effect</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1847</span>â <span class="NLM_lpage">1855</span>, <span class="refDoi">Â DOI: 10.1124/dmd.118.081919</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.118.081919" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=30209037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1Onsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=1847-1855&issue=12&author=J.+W.+Zhangauthor=W.+Xiaoauthor=Z.+T.+Gaoauthor=Z.+T.+Yuauthor=J.+Y.+J.+Zhang&title=Metabolism+of+c-Met+Kinase+Inhibitors+Containing+Quinoline+by+Aldehyde+Oxidase%2C+Electron+Donating%2C+and+Steric+Hindrance+Effect&doi=10.1124%2Fdmd.118.081919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of c-Met kinase inhibitors containing quinoline by aldehyde oxidase, electron donating, and steric hindrance effect</span></div><div class="casAuthors">Zhang, Jiang Wei; Xiao, Wen; Gao, Zhen Ting; Yu, Zheng Tian; Zhang, Ji Yue Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1847-1855</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Some quinoline-contg. c-Met kinase inhibitors are aldehyde oxidase (AO) substrates. 3-Substituted quinoline triazolopyridine analogs were synthesized to understand the electron-donating and steric hindrance effects on AO-mediated metab.  Metabolic stability studies for these quinoline analogs were carried out in liver cytosol from mice, rats, cynomolgus monkeys, and humans.  Several 3-N-substituted analogs were found to be unstable in monkey liver cytosolic incubations (half-life, <10 min), and five of them (63, 53, 51, 11, and 71) were chosen for addnl. mechanistic studies.  Mono-oxygenation on the quinoline ring was identified by liq. chromatog. tandem mass spectrometry.  Metabolite formation was inhibited by the AO inhibitors menadione and raloxifene, but not by the xanthine oxidase inhibitor allopurinol.  It was found that small electron-donating groups at the 3-quinoline moiety made the analogs more susceptible to AO metab., whereas large 3-substituents could reverse the trend.  Although species differences were obsd., this trend was applicable to all species tested.  Small electron-donating substituents at the 3-quinoline moiety increased both affinity (decreased Michaelis const.) and Vmax max. velocity toward AO in kinetic studies, whereas large substituents decreased both parameters probably as a result of steric hindrance.  Based on our anal., a common structural feature with high AO liability was proposed.  Our finding could provide useful information for chemists to minimize potential AO liability when designing quinoline analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkGiQQGWoG1bVg90H21EOLACvtfcHk0lipQtpKjviQwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1Onsbk%253D&md5=87b1321378c2b70c2e2490e23263b3fa</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1124%2Fdmd.118.081919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.118.081919%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%2BW.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DZ.%2BT.%26aulast%3DYu%26aufirst%3DZ.%2BT.%26aulast%3DZhang%26aufirst%3DJ.%2BY.%2BJ.%26atitle%3DMetabolism%2520of%2520c-Met%2520Kinase%2520Inhibitors%2520Containing%2520Quinoline%2520by%2520Aldehyde%2520Oxidase%252C%2520Electron%2520Donating%252C%2520and%2520Steric%2520Hindrance%2520Effect%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2018%26volume%3D46%26issue%3D12%26spage%3D1847%26epage%3D1855%26doi%3D10.1124%2Fdmd.118.081919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trincao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvalho, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramos, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garattini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimkuhler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romao, M. J.</span></span> <span> </span><span class="NLM_article-title">The First Mammalian Aldehyde Oxidase Crystal Structure: Insights into Substrate Specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span> (<span class="NLM_issue">48</span>),  <span class="NLM_fpage">40690</span>â <span class="NLM_lpage">40702</span>, <span class="refDoi">Â DOI: 10.1074/jbc.M112.390419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1074%2Fjbc.M112.390419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=23019336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslantr7L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=40690-40702&issue=48&author=C.+Coelhoauthor=M.+Mahroauthor=J.+Trincaoauthor=A.+T.+Carvalhoauthor=M.+J.+Ramosauthor=M.+Teraoauthor=E.+Garattiniauthor=S.+Leimkuhlerauthor=M.+J.+Romao&title=The+First+Mammalian+Aldehyde+Oxidase+Crystal+Structure%3A+Insights+into+Substrate+Specificity&doi=10.1074%2Fjbc.M112.390419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The First Mammalian Aldehyde Oxidase Crystal Structure</span></div><div class="casAuthors">Coelho, Catarina; Mahro, Martin; Trincao, Jose; Carvalho, Alexandra T. P.; Ramos, Maria Joao; Terao, Mineko; Garattini, Enrico; Leimkuehler, Silke; Romao, Maria Joao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">40690-40702</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aldehyde oxidases (AOXs) are homodimeric proteins belonging to the xanthine oxidase family of molybdenum-contg. enzymes.  Each 150-kDa monomer contains a FAD redox cofactor, two spectroscopically distinct [2Fe-2S] clusters, and a molybdenum cofactor located within the protein active site.  AOXs are characterized by broad range substrate specificity, oxidizing different aldehydes and arom. N-heterocycles.  Despite increasing recognition of its role in the metab. of drugs and xenobiotics, the physiol. function of the protein is still largely unknown.  We have crystd. and solved the crystal structure of mouse liver aldehyde oxidase 3 to 2.9 Ã.  This is the first mammalian AOX whose structure has been solved.  The structure provides important insights into the protein active center and further evidence on the catalytic differences characterizing AOX and xanthine oxidoreductase.  The mouse liver aldehyde oxidase 3 three-dimensional structure combined with kinetic, mutagenesis data, mol. docking, and mol. dynamics studies make a decisive contribution to understand the mol. basis of its rather broad substrate specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJhbcCiD1gB7Vg90H21EOLACvtfcHk0lipQtpKjviQwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslantr7L&md5=df014384e84736af9a937fd383f24523</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.390419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.390419%26sid%3Dliteratum%253Aachs%26aulast%3DCoelho%26aufirst%3DC.%26aulast%3DMahro%26aufirst%3DM.%26aulast%3DTrincao%26aufirst%3DJ.%26aulast%3DCarvalho%26aufirst%3DA.%2BT.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26aulast%3DTerao%26aufirst%3DM.%26aulast%3DGarattini%26aufirst%3DE.%26aulast%3DLeimkuhler%26aufirst%3DS.%26aulast%3DRomao%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520First%2520Mammalian%2520Aldehyde%2520Oxidase%2520Crystal%2520Structure%253A%2520Insights%2520into%2520Substrate%2520Specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26issue%3D48%26spage%3D40690%26epage%3D40702%26doi%3D10.1074%2Fjbc.M112.390419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coelho, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos-Silva, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leimkuhler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romao, M. J.</span></span> <span> </span><span class="NLM_article-title">Structural Insights Into Xenobiotic and Inhibitor Binding to Human Aldehyde Oxidase</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">779</span>â <span class="NLM_lpage">783</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1895</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1038%2Fnchembio.1895" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=26322824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlylsrnJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=779-783&issue=10&author=C.+Coelhoauthor=A.+Fotiauthor=T.+Hartmannauthor=T.+Santos-Silvaauthor=S.+Leimkuhlerauthor=M.+J.+Romao&title=Structural+Insights+Into+Xenobiotic+and+Inhibitor+Binding+to+Human+Aldehyde+Oxidase&doi=10.1038%2Fnchembio.1895"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into xenobiotic and inhibitor binding to human aldehyde oxidase</span></div><div class="casAuthors">Coelho, Catarina; Foti, Alessandro; Hartmann, Tobias; Santos-Silva, Teresa; Leimkuhler, Silke; Romao, Maria Joao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">779-783</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a xanthine oxidase (XO)-related enzyme with emerging importance due to its role in the metab. of drugs and xenobiotics.  We report the first crystal structures of human AOX1, substrate free (2.6-Ã resoln.) and in complex with the substrate phthalazine and the inhibitor thioridazine (2.7-Ã resoln.).  Anal. of the protein active site combined with steady-state kinetic studies highlight the unique features, including binding and substrate orientation at the active site, that characterize human AOX1 as an important drug-metabolizing enzyme.  Structural anal. of the complex with the noncompetitive inhibitor thioridazine revealed a new, unexpected and fully occupied inhibitor-binding site that is structurally conserved among mammalian AOXs and XO.  The new structural insights into the catalytic and inhibition mechanisms of human AOX that we now report will be of great value for the rational anal. of clin. drug interactions involving inhibition of AOX1 and for the prediction and design of AOX-stable putative drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbH6qUJJ5AMbVg90H21EOLACvtfcHk0lipQtpKjviQwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlylsrnJ&md5=2053b7ec132f3e4ed7a82d011a46072a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1895%26sid%3Dliteratum%253Aachs%26aulast%3DCoelho%26aufirst%3DC.%26aulast%3DFoti%26aufirst%3DA.%26aulast%3DHartmann%26aufirst%3DT.%26aulast%3DSantos-Silva%26aufirst%3DT.%26aulast%3DLeimkuhler%26aufirst%3DS.%26aulast%3DRomao%26aufirst%3DM.%2BJ.%26atitle%3DStructural%2520Insights%2520Into%2520Xenobiotic%2520and%2520Inhibitor%2520Binding%2520to%2520Human%2520Aldehyde%2520Oxidase%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D10%26spage%3D779%26epage%3D783%26doi%3D10.1038%2Fnchembio.1895" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Torres, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzekwa, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMasters, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fandozzi, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">Use of Density Functional Calculations to Predict the Regioselectivity of Drugs and Molecules Metabolized by Aldehyde Oxidase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">4642</span>â <span class="NLM_lpage">4647</span>, <span class="refDoi">Â DOI: 10.1021/jm0703690</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0703690" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsVKnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=4642-4647&issue=19&author=R.+A.+Torresauthor=K.+R.+Korzekwaauthor=D.+R.+McMastersauthor=C.+M.+Fandozziauthor=J.+P.+Jones&title=Use+of+Density+Functional+Calculations+to+Predict+the+Regioselectivity+of+Drugs+and+Molecules+Metabolized+by+Aldehyde+Oxidase&doi=10.1021%2Fjm0703690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Density Functional Calculations To Predict the Regioselectivity of Drugs and Molecules Metabolized by Aldehyde Oxidase</span></div><div class="casAuthors">Torres, Rhonda A.; Korzekwa, Kenneth R.; McMasters, Daniel R.; Fandozzi, Christine M.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4642-4647</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase is a molybdenum hydroxylase that catalyzes the oxidn. of aldehydes and nitrogen-contg. heterocycles.  The enzyme plays a dual role in the metab. of physiol. important endogenous compds. and the biotransformation of xenobiotics.  Using d. functional theory methods, geometry optimization of tetrahedral intermediates of drugs and druglike compds. was examd. to predict the likely metabolites of aldehyde oxidase.  The calcns. suggest that the lowest energy tetrahedral intermediate resulting from the initial substrate corresponds to the obsd. metabolite â¥90% of the time.  Addnl. calcns. were performed on a series of heterocyclic compds. where the products resulting from metab. by xanthine oxidase and aldehyde oxidase differ in many instances.  Again, the lowest energy tetrahedral intermediate corresponded to the obsd. product of aldehyde oxidase metab. â¥90% for the compds. examd., while the obsd. products of xanthine oxidase were not well predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmnIR2Ha4_oLVg90H21EOLACvtfcHk0lipQtpKjviQwA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsVKnsb4%253D&md5=514ee5703f306287a3b7315fc5cea434</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm0703690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0703690%26sid%3Dliteratum%253Aachs%26aulast%3DTorres%26aufirst%3DR.%2BA.%26aulast%3DKorzekwa%26aufirst%3DK.%2BR.%26aulast%3DMcMasters%26aufirst%3DD.%2BR.%26aulast%3DFandozzi%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DUse%2520of%2520Density%2520Functional%2520Calculations%2520to%2520Predict%2520the%2520Regioselectivity%2520of%2520Drugs%2520and%2520Molecules%2520Metabolized%2520by%2520Aldehyde%2520Oxidase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26issue%3D19%26spage%3D4642%26epage%3D4647%26doi%3D10.1021%2Fjm0703690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alfaro, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span> <span> </span><span class="NLM_article-title">Studies On the Mechanism of Aldehyde Oxidase and Xanthine Oxidase</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">9469</span>â <span class="NLM_lpage">9472</span>, <span class="refDoi">Â DOI: 10.1021/jo801053u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo801053u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGku7vI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=9469-9472&issue=23&author=J.+F.+Alfaroauthor=J.+P.+Jones&title=Studies+On+the+Mechanism+of+Aldehyde+Oxidase+and+Xanthine+Oxidase&doi=10.1021%2Fjo801053u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Studies on the Mechanism of Aldehyde Oxidase and Xanthine Oxidase</span></div><div class="casAuthors">Alfaro, Joshua F.; Jones, Jeffrey P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9469-9472</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DFT calcns. support a concerted mechanism for xanthine oxidase and aldehyde oxidase hydride displacement from the sp2 carbon of 6-substituted 4-quinazolinones.  The variations in transition state structure show that C-O bond formation is nearly complete in the transition state and the transition state changes are anti-Hammond with the C-H and C-O bond lengths being more product-like for the faster reactions.  The C-O bond length in the transition state is around 90% formed.  However, the C-H bond is only about 80% broken.  This leads to a very tetrahedral transition state with an O-C-N angle of 109Â°.  Thus, while the mechanism is concerted, the antibonding orbital of the C-H bond that is broken is not directly attacked by the nucleophile and instead hydride displacement occurs after almost complete tetrahedral transition state formation.  In support of this the C=N bond is lengthened in the transition state indicating that attack on the electrophilic carbon occurs by addn. to the C=N bond with neg. charge increasing on the nitrogen.  Differences in exptl. reaction rates are accurately reproduced by these calcns. and tend to support this mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrK9tmdy1IwLVg90H21EOLACvtfcHk0lg9Yx5FDYzvkA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlGku7vI&md5=188f70b1454b3e6bfd19301aafae2908</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjo801053u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo801053u%26sid%3Dliteratum%253Aachs%26aulast%3DAlfaro%26aufirst%3DJ.%2BF.%26aulast%3DJones%26aufirst%3DJ.%2BP.%26atitle%3DStudies%2520On%2520the%2520Mechanism%2520of%2520Aldehyde%2520Oxidase%2520and%2520Xanthine%2520Oxidase%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26issue%3D23%26spage%3D9469%26epage%3D9472%26doi%3D10.1021%2Fjo801053u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montefiori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, L.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase: Reaction Mechanism and Prediction of Site of Metabolism</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">4237</span>â <span class="NLM_lpage">4244</span>, <span class="refDoi">Â DOI: 10.1021/acsomega.7b00658</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.7b00658" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohtr3J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=4237-4244&issue=8&author=M.+Montefioriauthor=F.+S.+Jorgensenauthor=L.+Olsen&title=Aldehyde+Oxidase%3A+Reaction+Mechanism+and+Prediction+of+Site+of+Metabolism&doi=10.1021%2Facsomega.7b00658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Oxidase: Reaction Mechanism and Prediction of Site of Metabolism</span></div><div class="casAuthors">Montefiori, Marco; Joergensen, Flemming Steen; Olsen, Lars</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4237-4244</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AO) is a Mo-contg. enzyme involved in the clearance of drug compds. contg. aldehydes and N-contg. heterocyclic fragments.  AO has gained considerable interest in recent years because of examples of too fast clearance of drug compds. in development.  Thus, it is important to be able to predict AO-mediated drug metab.  Therefore, we characterized the structural and energetic aspects of different mechanisms with DFT using the Mo cofactor (MoCo) as model for the reactive part of the enzyme.  For a series of 6-substituted 4-quinazolinones, the trend in activation energies was the same for 3 tested reaction mechanisms.  Using the concerted mechanism as model for the enzymic reaction, the transition states for formation of all possible metabolites for a series of known AO substrates were detd.  The lowest activation energies corresponded in all cases to the exptl. obsd. sites of metab. (SOMs).  Various mol. properties were calcd. and investigated as more easily determinable markers for reactivity.  Both the stabilities of the intermediates and products correlated to some extent with the transition state energies, and may be used to predict the SOM.  The electrostatic potential (ESP)-derived charges were also good markers for prediction of the exptl. SOMs for this set of compds. and may pave the way for the development of fast methods for prediction of SOMs for AO substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWMl-me1-SlbVg90H21EOLACvtfcHk0lg9Yx5FDYzvkA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohtr3J&md5=5aa9f790b1ff0f28cb270322618de06c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsomega.7b00658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.7b00658%26sid%3Dliteratum%253Aachs%26aulast%3DMontefiori%26aufirst%3DM.%26aulast%3DJorgensen%26aufirst%3DF.%2BS.%26aulast%3DOlsen%26aufirst%3DL.%26atitle%3DAldehyde%2520Oxidase%253A%2520Reaction%2520Mechanism%2520and%2520Prediction%2520of%2520Site%2520of%2520Metabolism%26jtitle%3DACS%2520Omega%26date%3D2017%26volume%3D2%26issue%3D8%26spage%3D4237%26epage%3D4244%26doi%3D10.1021%2Facsomega.7b00658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T. D.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase: An Enzyme of Emerging Importance in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">24</span>),  <span class="NLM_fpage">8441</span>â <span class="NLM_lpage">8460</span>, <span class="refDoi">Â DOI: 10.1021/jm100888d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100888d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFygt73N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8441-8460&issue=24&author=D.+C.+Prydeauthor=D.+Dalvieauthor=Q.+Huauthor=P.+Jonesauthor=R.+S.+Obachauthor=T.+D.+Tran&title=Aldehyde+Oxidase%3A+An+Enzyme+of+Emerging+Importance+in+Drug+Discovery&doi=10.1021%2Fjm100888d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde Oxidase: An Enzyme of Emerging Importance in Drug Discovery</span></div><div class="casAuthors">Pryde, David C.; Dalvie, Deepak; Hu, Qiyue; Jones, Peter; Obach, R. Scott; Tran, Thien-Duc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8441-8460</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Aldehyde oxidase has for many years been recognized as a metabolizing enzyme contained within the cytosolic compartment of many tissues and in many species.  The first literature ref. to AO dates from the 1930s, but literature citations to this enzyme have been steadily growing ever since, esp. in the past decade.  It has been increasingly recognized in this past decade that AO, through its unique structure, distribution, and substrate recognition, has an important role to play in the metab. of drugs.  In this timely Perspective, we present the current knowledge of the enzyme's expression, its structure, and its distribution across species.  We offer an anal. of substrates recognized by AO and highlight clin. examples where metab. by this enzyme has had significant clin. impact or led to the termination of a drug development program.  Finally, through anal. of the structure of known substrates, we present data to suggest the significant impact AO could play on future drug discovery programs along with strategies that could be employed to mitigate this.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqALwFS70G6tLVg90H21EOLACvtfcHk0lg9Yx5FDYzvkA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFygt73N&md5=972eb9862a7dc7bff024b530ab40de7f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm100888d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100888d%26sid%3Dliteratum%253Aachs%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DTran%26aufirst%3DT.%2BD.%26atitle%3DAldehyde%2520Oxidase%253A%2520An%2520Enzyme%2520of%2520Emerging%2520Importance%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26issue%3D24%26spage%3D8441%26epage%3D8460%26doi%3D10.1021%2Fjm100888d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, P.</span></span> <span> </span><span class="NLM_article-title">Effect of Structural Variation on Aldehyde Oxidase-Catalyzed Oxidation of Zoniporide</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1575</span>â <span class="NLM_lpage">1587</span>, <span class="refDoi">Â DOI: 10.1124/dmd.112.045823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.112.045823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22587988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2iurbK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1575-1587&issue=8&author=D.+Dalvieauthor=H.+Sunauthor=C.+Xiangauthor=Q.+Huauthor=Y.+Jiangauthor=P.+Kang&title=Effect+of+Structural+Variation+on+Aldehyde+Oxidase-Catalyzed+Oxidation+of+Zoniporide&doi=10.1124%2Fdmd.112.045823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Effect of structural variation on aldehyde oxidase-catalyzed oxidation of zoniporide</span></div><div class="casAuthors">Dalvie, Deepak; Sun, Hao; Xiang, Cathie; Hu, Qiyue; Jiang, Ying; Kang, Ping</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1575-1587</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Current studies explored the effect of structural changes on the aldehyde oxidase (AO)-mediated metab. of zoniporide (1).  Zoniporide analogs with modifications of the acylguanidine moiety, the cyclopropyl group on the pyrazole ring, and the quinoline ring were studied for their AO-catalyzed metab. using the human S9 fraction.  Anal. of the half-lives suggested that subtle changes in the structure of 1 influenced its metab. and that the guanidine and the quinoline moieties were prerequisites for AO-catalyzed oxidn. to 2-oxozoniporide (M1).  In contrast, replacement of the cyclopropyl group with other alkyl groups was tolerated.  The effect of structural variation on AO properties was rationalized by docking 1 and its analogs into the human AO homol. model.  These studies indicated the importance of electrostatic, Ï-Ï stacking and hydrophobic interactions of the three motifs with residues in the active site.  Differences in substrate properties were also rationalized by comparing their half-lives with cLogD, electrophilicity parameters [electrostatic potential (ESP) charges and energy of lowest unoccupied MOs (ELUMO)], and the energies of formation of tetrahedral intermediates.  Whereas the success of energetics in predicting the AO substrate properties of analogs was 87%, the predictive ability of other descriptors was none (cLogD) to 60% (ESP charges and ELUMO).  Overall, the structure-metab. relationship could be rationalized using a combination of both the energy calcns. and docking studies.  This combination method can be incorporated into a strategy for mitigating AO liabilities obsd. in the lead candidate or studying structure-metab. relationships of other AO substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCkVhA55F__bVg90H21EOLACvtfcHk0lgOAv5KgblALA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2iurbK&md5=b3ec4f1916d516092e72156dd63304a5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.045823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.045823%26sid%3Dliteratum%253Aachs%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXiang%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKang%26aufirst%3DP.%26atitle%3DEffect%2520of%2520Structural%2520Variation%2520on%2520Aldehyde%2520Oxidase-Catalyzed%2520Oxidation%2520of%2520Zoniporide%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26issue%3D8%26spage%3D1575%26epage%3D1587%26doi%3D10.1124%2Fdmd.112.045823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korzekwa, K. R.</span></span> <span> </span><span class="NLM_article-title">Predicting Intrinsic Clearance for Drugs and Drug Candidates Metabolized by Aldehyde Oxidase</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1262</span>â <span class="NLM_lpage">1268</span>, <span class="refDoi">Â DOI: 10.1021/mp300568r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/mp300568r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ajtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1262-1268&issue=4&author=J.+P.+Jonesauthor=K.+R.+Korzekwa&title=Predicting+Intrinsic+Clearance+for+Drugs+and+Drug+Candidates+Metabolized+by+Aldehyde+Oxidase&doi=10.1021%2Fmp300568r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Intrinsic Clearance for Drugs and Drug Candidates Metabolized by Aldehyde Oxidase</span></div><div class="casAuthors">Jones, Jeffrey P.; Korzekwa, Kenneth R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1262-1268</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metab. by aldehyde oxidase (AO) has been responsible for a no. of drug failures in clin. trials.  The main reason is the clearance values for drugs metabolized by AO are underestimated by allometric scaling from preclin. species.  Furthermore, in vitro human data also underestimates clearance.  We have developed the first in silico models to predict both in vitro and in vivo human intrinsic clearance for 8 drugs with just two chem. descriptors.  These models explain a large amt. of the variance in the data using two computational ests. of the electronic and steric features of the reaction.  The in vivo computational models for human metab. are better than in vitro preclin. animal testing at predicting human intrinsic clearance.  Thus, it appears that AO is amenable to computational prediction of rates, which may be used to guide drug discovery, and predict pharmacokinetics for clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMe4imNUVqorVg90H21EOLACvtfcHk0lgOAv5KgblALA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ajtbc%253D&md5=b91f8487291a445d04cccd20fd9227a3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fmp300568r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300568r%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DJ.%2BP.%26aulast%3DKorzekwa%26aufirst%3DK.%2BR.%26atitle%3DPredicting%2520Intrinsic%2520Clearance%2520for%2520Drugs%2520and%2520Drug%2520Candidates%2520Metabolized%2520by%2520Aldehyde%2520Oxidase%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26issue%3D4%26spage%3D1262%26epage%3D1268%26doi%3D10.1021%2Fmp300568r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lepri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceccarelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cucco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goracci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruciani, G.</span></span> <span> </span><span class="NLM_article-title">Structure-Metabolism Relationships in Human-AOX: Chemical Insights From a Large Database of Aza-Aromatic and Amide Compounds</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">E3178</span>â <span class="NLM_lpage">E3187</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1618881114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1073%2Fpnas.1618881114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28373537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlsV2nsb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E3178-E3187&issue=16&author=S.+Lepriauthor=M.+Ceccarelliauthor=N.+Milaniauthor=S.+Tortorellaauthor=A.+Cuccoauthor=A.+Valeriauthor=L.+Goracciauthor=A.+Brinkauthor=G.+Cruciani&title=Structure-Metabolism+Relationships+in+Human-AOX%3A+Chemical+Insights+From+a+Large+Database+of+Aza-Aromatic+and+Amide+Compounds&doi=10.1073%2Fpnas.1618881114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-metabolism relationships in human-AOX: Chemical insights from a large database of aza-aromatic and amide compounds.</span></div><div class="casAuthors">Lepri, Susan; Ceccarelli, Martina; Milani, Nicolo; Tortorella, Sara; Cucco, Andrea; Valeri, Aurora; Goracci, Laura; Brink, Andreas; Cruciani, Gabriele</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">E3178-E3187</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a metabolic enzyme catalyzing the oxidn. of aldehyde and aza-arom. compds. and the hydrolysis of amides, moieties frequently shared by the majority of drugs.  Despite its key role in human metab., to date only fragmentary information about the chem. features responsible for AOX susceptibility are reported and only "very local" structure-metab. relationships based on a small no. of similar compds. have been developed.  This study reports a more comprehensive coverage of the chem. space of structures with a high risk of AOX phase I metab. in humans.  More than 270 compds. were studied to identify the site of metab. and the metabolite(s).  Both electronic [supported by d. functional theory (DFT) calcns.] and exposure effects were considered when rationalizing the structure-metab. relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotA9C1TVn-9bVg90H21EOLACvtfcHk0lgOAv5KgblALA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlsV2nsb8%253D&md5=81cd013f7b63399e10d950e30377dbc0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1618881114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1618881114%26sid%3Dliteratum%253Aachs%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DCeccarelli%26aufirst%3DM.%26aulast%3DMilani%26aufirst%3DN.%26aulast%3DTortorella%26aufirst%3DS.%26aulast%3DCucco%26aufirst%3DA.%26aulast%3DValeri%26aufirst%3DA.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DBrink%26aufirst%3DA.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DStructure-Metabolism%2520Relationships%2520in%2520Human-AOX%253A%2520Chemical%2520Insights%2520From%2520a%2520Large%2520Database%2520of%2520Aza-Aromatic%2520and%2520Amide%2520Compounds%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26issue%3D16%26spage%3DE3178%26epage%3DE3187%26doi%3D10.1073%2Fpnas.1618881114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cruciani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesarini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spyrakis, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortorella, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goracci, L.</span></span> <span> </span><span class="NLM_article-title">From Experiments to a Fast Easy-to-Use Computational Methodology to Predict Human Aldehyde Oxidase Selectivity and Metabolic Reactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">360</span>â <span class="NLM_lpage">371</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01552</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01552" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOru7fK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=360-371&issue=1&author=G.+Crucianiauthor=N.+Milaniauthor=P.+Benedettiauthor=S.+Lepriauthor=L.+Cesariniauthor=M.+Baroniauthor=F.+Spyrakisauthor=S.+Tortorellaauthor=E.+Mosconiauthor=L.+Goracci&title=From+Experiments+to+a+Fast+Easy-to-Use+Computational+Methodology+to+Predict+Human+Aldehyde+Oxidase+Selectivity+and+Metabolic+Reactions&doi=10.1021%2Facs.jmedchem.7b01552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">From Experiments to a Fast Easy-to-Use Computational Methodology to Predict Human Aldehyde Oxidase Selectivity and Metabolic Reactions</span></div><div class="casAuthors">Cruciani, Gabriele; Milani, Nicolo; Benedetti, Paolo; Lepri, Susan; Cesarini, Lucia; Baroni, Massimo; Spyrakis, Francesca; Tortorella, Sara; Mosconi, Edoardo; Goracci, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">360-371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AOX) is a molibdo-flavoenzyme that raised great interest in recent years, since its contribution in xenobiotics metab. has not always been identified before clin. trials, with consequent neg. effects on the fate of new potential drugs.  The fundamental role of AOX in metabolizing xenobiotics is also due to the attempt of medicinal chemists to stabilize candidates toward cytochrome P 450 activity, which increases the risk for new compds. to be susceptible to AOX nucleophile attack.  Therefore, novel strategies to predict the potential liability of new entities towards AOX enzyme are urgently needed to increase effectiveness, reduce costs, and prioritize exptl. studies.  In the present work, we present the most up-to-date computational method to predict liability towards human AOX, for applications in drug design and pharmacokinetic optimization.  The method was developed using a large dataset of homogeneous exptl. data, which is also disclosed as supplementary material.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoi_Tnrnus317Vg90H21EOLACvtfcHk0lgOAv5KgblALA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOru7fK&md5=23c7fd8a5abcba87e7fc065d5e6dafc6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01552%26sid%3Dliteratum%253Aachs%26aulast%3DCruciani%26aufirst%3DG.%26aulast%3DMilani%26aufirst%3DN.%26aulast%3DBenedetti%26aufirst%3DP.%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DCesarini%26aufirst%3DL.%26aulast%3DBaroni%26aufirst%3DM.%26aulast%3DSpyrakis%26aufirst%3DF.%26aulast%3DTortorella%26aufirst%3DS.%26aulast%3DMosconi%26aufirst%3DE.%26aulast%3DGoracci%26aufirst%3DL.%26atitle%3DFrom%2520Experiments%2520to%2520a%2520Fast%2520Easy-to-Use%2520Computational%2520Methodology%2520to%2520Predict%2520Human%2520Aldehyde%2520Oxidase%2520Selectivity%2520and%2520Metabolic%2520Reactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D1%26spage%3D360%26epage%3D371%26doi%3D10.1021%2Facs.jmedchem.7b01552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montefiori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyngholm-Kjaerby, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, F. S.</span></span> <span> </span><span class="NLM_article-title">Fast Methods for Prediction of Aldehyde Oxidase-Mediated Site-of-Metabolism</span>. <i>Comput. Struct. Biotechnol. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">345</span>â <span class="NLM_lpage">351</span>, <span class="refDoi">Â DOI: 10.1016/j.csbj.2019.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1016%2Fj.csbj.2019.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=30949305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlt12jsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=345-351&author=M.+Montefioriauthor=C.+Lyngholm-Kjaerbyauthor=A.+Longauthor=L.+Olsenauthor=F.+S.+Jorgensen&title=Fast+Methods+for+Prediction+of+Aldehyde+Oxidase-Mediated+Site-of-Metabolism&doi=10.1016%2Fj.csbj.2019.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Methods for Prediction of Aldehyde Oxidase-Mediated Site-of-Metabolism</span></div><div class="casAuthors">Montefiori, Marco; Lyngholm-Kjaerby, Casper; Long, Anthony; Olsen, Lars; Joergensen, Flemming Steen</div><div class="citationInfo"><span class="NLM_cas:title">Computational and Structural Biotechnology Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">345-351</span>CODEN:
                <span class="NLM_cas:coden">CSBJAC</span>;
        ISSN:<span class="NLM_cas:issn">2001-0370</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Aldehyde Oxidase (AO) is an enzyme involved in the metab. of aldehydes and N-contg. heterocyclic compds.  Many drug compds. contain heterocyclic moieties, and AO metab. has lead to failure of several late-stage drug candidates.  Therefore, it is important to take AO-mediated metab. into account early in the drug discovery process, and thus, to have fast and reliable models to predict the site of metab. (SOM).  We have collected a dataset of 78 substrates of human AO with a total of 89 SOMs and 347 non-SOMs and detd. at. descriptors for each compd.  The descriptors comprise NMR shielding and ESP charges from d. functional theory (DFT), NMR chem. shift from ChemBioDraw, and Gasteiger charges from RDKit.  Addnl., at. accessibility was considered using 2D-SASA and relative span descriptors from SMARTCyp.  Finally, stability of the product, the metabolite, was detd. with DFT and also used as a descriptor.  All descriptors have AUC larger than 0.75.  In particular, descriptors related to the chem. shielding and chem. shift (AUC = 0.96) and ESP charges (AUC = 0.96) proved to be good descriptors.  We recommend two simple methods to identify the SOM for a given mol.: (1) use ChemBioDraw to calc. the chem. shift or (2) calc. ESP charges or chem. shift using DFT.  The first approach is fast but somewhat difficult to automate, while the second is more time-consuming, but can easily be automated.  The two methods predict correctly 93% and 91%, resp., of the 89 exptl. obsd. SOMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8QYgsDr8OrLVg90H21EOLACvtfcHk0lin-JOIdNGXnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlt12jsLw%253D&md5=8a01ea8667b1bfcccc7ab59570df6739</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.csbj.2019.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.csbj.2019.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DMontefiori%26aufirst%3DM.%26aulast%3DLyngholm-Kjaerby%26aufirst%3DC.%26aulast%3DLong%26aufirst%3DA.%26aulast%3DOlsen%26aufirst%3DL.%26aulast%3DJorgensen%26aufirst%3DF.%2BS.%26atitle%3DFast%2520Methods%2520for%2520Prediction%2520of%2520Aldehyde%2520Oxidase-Mediated%2520Site-of-Metabolism%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2019%26volume%3D17%26spage%3D345%26epage%3D351%26doi%3D10.1016%2Fj.csbj.2019.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dick, R. A.</span></span> <span> </span><span class="NLM_article-title">Refinement of In Vitro Methods for Identification of Aldehyde Oxidase Substrates Reveals Metabolites of Kinase Inhibitors</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">846</span>â <span class="NLM_lpage">859</span>, <span class="refDoi">Â DOI: 10.1124/dmd.118.080960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.118.080960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=29615437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWlsb7M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=846-859&issue=6&author=R.+A.+Dick&title=Refinement+of+In+Vitro+Methods+for+Identification+of+Aldehyde+Oxidase+Substrates+Reveals+Metabolites+of+Kinase+Inhibitors&doi=10.1124%2Fdmd.118.080960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of in vitro methods for identification of aldehyde oxidase substrates reveals metabolites of kinase inhibitors</span></div><div class="casAuthors">Dick, Ryan A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">846-859</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">To identify aldehyde oxidase (AO) substrates, an assay procedure was developed that leverages the capabilities of high-resoln. mass spectrometry to simultaneously monitor parent loss and formation of hydroxylated metabolite over time in incubations with liver cytosol.  By incorporating metabolite monitoring, false positives resulting from metab. by other cytosolic enzymes or processes were decreased.  A diverse set of 34 kinase inhibitors contg. AO-substrate motifs was screened, and 35% of the compds. were identified as human AO substrates.  Confirmation was obtained through detn. of the site of metab.  Human AO substrates identified contained unsubstituted diazanaphthalene moieties (A77-01, INCB 28060, ML-347, LDN-193189, and SB-525334), 4-aminoquinazoline cores (lapatinib, lapatinib M1, and CL-387785), and terminal pyridine and pyrimidine groups (imatinib, bafetinib, and AMG 900).  Rat and cynomolgus monkey AO displayed substrate specificities that overlapped moderately with human; rates of metab. were often higher and lower for cynomolgus monkey and rat, resp., compared with human.  A subset of novel AO substrates identified in this study was also subjected to two other methods for AO substrate detn.: comparison of human liver microsome and hepatocyte stability, and the effect of hydralazine, an AO-specific inhibitor, on hepatocyte stability.  These methods appeared to correlate and be capable of identifying AO substrates when more than one-third of metab. in hepatocytes was AO-mediated; however, significant limitations exist.  Considering the sensitivity, efficiency, and definitiveness of the cytosol assay with metabolite monitoring, its use is recommended as a primary screen for AO substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlGGJzSPQLDbVg90H21EOLACvtfcHk0lin-JOIdNGXnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWlsb7M&md5=0b08ae43f93b6a9874f10b40422d93b2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1124%2Fdmd.118.080960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.118.080960%26sid%3Dliteratum%253Aachs%26aulast%3DDick%26aufirst%3DR.%2BA.%26atitle%3DRefinement%2520of%2520In%2520Vitro%2520Methods%2520for%2520Identification%2520of%2520Aldehyde%2520Oxidase%2520Substrates%2520Reveals%2520Metabolites%2520of%2520Kinase%2520Inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2018%26volume%3D46%26issue%3D6%26spage%3D846%26epage%3D859%26doi%3D10.1124%2Fdmd.118.080960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span> <span> </span><span class="NLM_article-title">In Silico Site of Metabolism Prediction for Human UGT-Catalyzed Reactions</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">398</span>â <span class="NLM_lpage">405</span>, <span class="refDoi">Â DOI: 10.1093/bioinformatics/btt681</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1093%2Fbioinformatics%2Fbtt681" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=24273240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=398-405&issue=3&author=J.+Pengauthor=J.+Luauthor=Q.+Shenauthor=M.+Zhengauthor=X.+Luoauthor=W.+Zhuauthor=H.+Jiangauthor=K.+Chen&title=In+Silico+Site+of+Metabolism+Prediction+for+Human+UGT-Catalyzed+Reactions&doi=10.1093%2Fbioinformatics%2Fbtt681"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">In silico site of metabolism prediction for human UGT-catalyzed reactions</span></div><div class="casAuthors">Peng, Jianlong; Lu, Jing; Shen, Qiancheng; Zheng, Mingyue; Luo, Xiaomin; Zhu, Weiliang; Jiang, Hualiang; Chen, Kaixian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">398-405</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The human uridine diphosphate-glucuronosyltransferase enzyme family catalyzes the glucuronidation of the glycosyl group of a nucleotide sugar to an acceptor compd. (substrate), which is the most common conjugation pathway that serves to protect the organism from the potential toxicity of xenobiotics.  Moreover, it could affect the pharmacol. profile of a drug.  Therefore, it is important to identify the metabolically labile sites for glucuronidation.  Results: In the present study, we developed four in silico models to predict sites of glucuronidation, for four major sites of metab. functional groups, i.e. aliph. hydroxyl, arom. hydroxyl, carboxylic acid or amino nitrogen, resp.  According to the mechanism of glucuronidation, a series of 'local' and 'global' mol. descriptors characterizing the at. reactivity, bonding strength and phys.-chem. properties were calcd. and selected with a genetic algorithm-based feature selection approach.  The constructed support vector machine classification models show good prediction performance, with the balanced accuracy ranging from 0.88 to 0.96 on test set.  For further validation, our models can successfully identify 84% of exptl. obsd. sites of metabs. for an external test set contg. 54 mols.  Availability and implementation: The software somugt based on our models is available at www.dddc.ac.cn/adme/jlpeng/somugt_win32.zip.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="bec6d3d2cbd1fed7d3d390dfdd90ddd0">[emailÂ protected]</a> or <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="79140003111c171e39181015570a111a171a57181a571a17">[emailÂ protected]</a> Supplementary information: Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTnU5JiJGKPrVg90H21EOLACvtfcHk0lin-JOIdNGXnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Cjtb8%253D&md5=c99109a95f1bb99af1c93a3946a3dd44</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt681%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26atitle%3DIn%2520Silico%2520Site%2520of%2520Metabolism%2520Prediction%2520for%2520Human%2520UGT-Catalyzed%2520Reactions%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26issue%3D3%26spage%3D398%26epage%3D405%26doi%3D10.1093%2Fbioinformatics%2Fbtt681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span> <span> </span><span class="NLM_article-title">Prediction of Human Cytochrome P450 Inhibition Using a Multitask Deep Autoencoder Neural Network</span>. <i>Mol. Pharmaceutics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">4336</span>â <span class="NLM_lpage">4345</span>, <span class="refDoi">Â DOI: 10.1021/acs.molpharmaceut.8b00110</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.8b00110" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC1cXps12nt7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=4336-4345&author=X.+Liauthor=Y.+Xuauthor=L.+Laiauthor=J.+Pei&title=Prediction+of+Human+Cytochrome+P450+Inhibition+Using+a+Multitask+Deep+Autoencoder+Neural+Network&doi=10.1021%2Facs.molpharmaceut.8b00110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Human Cytochrome P450 Inhibition Using a Multitask Deep Autoencoder Neural Network</span></div><div class="casAuthors">Li, Xiang; Xu, Youjun; Lai, Luhua; Pei, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4336-4345</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adverse side effects of drug-drug interactions induced by human cytochrome P 450 (CYP450) inhibition is an important consideration in drug discovery.  It is highly desirable to develop computational models that can predict the inhibitive effect of a compd. against a specific CYP450 isoform.  In this study, we developed a multitask model for concurrent inhibition prediction of five major CYP450 isoforms, namely, 1A2, 2C9, 2C19, 2D6, and 3A4.  The model was built by training a multitask autoencoder deep neural network (DNN) on a large dataset contg. more than 13,000 compds., extd. from the PubChem BioAssay Database.  We demonstrate that the multitask model gave better prediction results than that of single-task models, previous reported classifiers, and traditional machine learning methods on an av. of five prediction tasks.  Our multitask DNN model gave av. prediction accuracies of 86.4% for the 10-fold cross-validation and 88.7% for the external test datasets.  In addn., we built linear regression models to quantify how the other tasks contributed to the prediction difference of a given task between single-task and multitask models, and we explained under what conditions the multitask model will outperform the single-task model, which suggested how to use multitask DNN models more effectively.  We applied sensitivity anal. to ext. useful knowledge about CYP450 inhibition, which may shed light on the structural features of these isoforms and give hints about how to avoid side effects during drug development.  Our models are freely available at http://repharma.pku.edu.cn/deepcyp/home.php or http://www.pkumdl.cn/deepcyp/home.php.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWZzh0LeCx9rVg90H21EOLACvtfcHk0lin-JOIdNGXnQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXps12nt7s%253D&md5=2c08dbac3a65242c61ef70c3147f4a85</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.8b00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.8b00110%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26atitle%3DPrediction%2520of%2520Human%2520Cytochrome%2520P450%2520Inhibition%2520Using%2520a%2520Multitask%2520Deep%2520Autoencoder%2520Neural%2520Network%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2018%26volume%3D15%26spage%3D4336%26epage%3D4345%26doi%3D10.1021%2Facs.molpharmaceut.8b00110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D. A.</span></span> <span> </span><span class="NLM_article-title">Discovery and ADMET: Where Are We Now</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">467</span>â <span class="NLM_lpage">481</span>, <span class="refDoi">Â DOI: 10.2174/156802611794480909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.2174%2F156802611794480909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=21320070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkslSnurs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=467-481&issue=4&author=D.+A.+Smith&title=Discovery+and+ADMET%3A+Where+Are+We+Now&doi=10.2174%2F156802611794480909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and ADMET: where are we now</span></div><div class="casAuthors">Smith, Dennis A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">467-481</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The gradual alignment with all of drug metab. with all aspects of drug discovery and development has led to a complete realignment of the way the work is conducted.  From a background of conducting bespoke in vivo studies much of the work is now in a high throughput screening mode.  Large technol. advances were made, but the nature of drug metab. processes, being multi-system and promiscuous means that much of the help provided to the medicinal chem. is reactive rather than based on fundamental disposition structure-activity relationships.  Lessons learned around the chem. and physicochem. properties more often assocd. with successful discovery and development projects are only moderately helpful when the high value pharmacol. targets of today only yield potent ligands outside of the boundaries these properties describe.  Pivotal to the impact of these properties is the intrinsic permeability of a mol., something not as widely recognized as perhaps it should be.  Metabolic lability is still a problem and the tactics employed are unchanged in 20 years: attempt to lower lipophilicity, if it is too high overall or introduce blocking groups, particularly halogens, after identifying the sites of metab.  Perhaps the greatest success drug metab. science has had over the last fifteen years is it's pivotal role in characterizing drug-drug interactions and providing screening systems and computational models to investigate them.  It still has many undeveloped areas of the science such as the role of metabolites in drug activity and why compds. vary in their extent of biliary excretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEw_2VMNc--bVg90H21EOLACvtfcHk0ljoYF3NY_HiwQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkslSnurs%253D&md5=b4e1b937f2f048a3ec9b97df98b339e6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.2174%2F156802611794480909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611794480909%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DD.%2BA.%26atitle%3DDiscovery%2520and%2520ADMET%253A%2520Where%2520Are%2520We%2520Now%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26issue%3D4%26spage%3D467%26epage%3D481%26doi%3D10.2174%2F156802611794480909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shoombuatong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prathipati, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prachayasittikul, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaduangrat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratiwi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wanwimolruk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikberg, J. E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleeson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spjuth, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nantasenamat, C.</span></span> <span> </span><span class="NLM_article-title">Towards Predicting the Cytochrome P450 Modulation: From QSAR to Proteochemometric Modeling</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">540</span>â <span class="NLM_lpage">555</span>, <span class="refDoi">Â DOI: 10.2174/1389200218666170320121932</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.2174%2F1389200218666170320121932" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28322159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Okur%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=540-555&issue=6&author=W.+Shoombuatongauthor=P.+Prathipatiauthor=V.+Prachayasittikulauthor=N.+Schaduangratauthor=A.+A.+Malikauthor=R.+Pratiwiauthor=S.+Wanwimolrukauthor=J.+E.+S.+Wikbergauthor=M.+P.+Gleesonauthor=O.+Spjuthauthor=C.+Nantasenamat&title=Towards+Predicting+the+Cytochrome+P450+Modulation%3A+From+QSAR+to+Proteochemometric+Modeling&doi=10.2174%2F1389200218666170320121932"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Towards Predicting the Cytochrome P450 Modulation: From QSAR to Proteochemometric Modeling</span></div><div class="casAuthors">Shoombuatong, Watshara; Prathipati, Philip; Prachayasittikul, Veda; Schaduangrat, Nalini; Malik, Aijaz Ahmad; Pratiwi, Reny; Wanwimolruk, Sompon; Wikberg, Jarl E. S.; Gleeson, Matthew Paul; Spjuth, Ola; Nantasenamat, Chanin</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-555</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Drug metab. dets. the fate of a drug when it enters the human body and is a crit. factor in defining their absorption, distribution, metab., excretion and toxicity (ADMET) characteristics.  Among the various drug metabolizing enzymes, cytochrome P450s (CYP450) constitute an important protein family that aside from functioning in xenobiotic metab., is also responsible for a diverse array of other roles encompassing steroid and cholesterol biosynthesis, fatty acid metab., calcium homeostasis, neuroendocrine functions and growth regulation.  Although CYP450 typically converts xenobiotics into safe metabolites, there are some situations whereby the metabolite is more toxic than its parent mol.  Computational modeling has been instrumental in CYP450 research by rationalizing the nature of the binding event (i.e. inhibit or induce CYP450s) or metabolic stability of query compds. of interest.  A plethora of computational approaches encompassing ligand, structure and systems based approaches have been utilized to model CYP450-ligand interactions.  This review provides a brief background on the CYP450 family (i.e. its roles, advantages and disadvantages as well as its modulators) and then discusses the various computational approaches that have been used to model CYP450-ligand interaction.  Particular focus was given to the use of quant. structure-activity relationship (QSAR) and more recent proteochemometric modeling studies.  Finally, a perspective on the current state of the art and future trends of the field is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUBK5dISfQ2LVg90H21EOLACvtfcHk0ljoYF3NY_HiwQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Okur%252FN&md5=5dea9e43349371555af5030e3a145f02</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.2174%2F1389200218666170320121932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389200218666170320121932%26sid%3Dliteratum%253Aachs%26aulast%3DShoombuatong%26aufirst%3DW.%26aulast%3DPrathipati%26aufirst%3DP.%26aulast%3DPrachayasittikul%26aufirst%3DV.%26aulast%3DSchaduangrat%26aufirst%3DN.%26aulast%3DMalik%26aufirst%3DA.%2BA.%26aulast%3DPratiwi%26aufirst%3DR.%26aulast%3DWanwimolruk%26aufirst%3DS.%26aulast%3DWikberg%26aufirst%3DJ.%2BE.%2BS.%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26aulast%3DSpjuth%26aufirst%3DO.%26aulast%3DNantasenamat%26aufirst%3DC.%26atitle%3DTowards%2520Predicting%2520the%2520Cytochrome%2520P450%2520Modulation%253A%2520From%2520QSAR%2520to%2520Proteochemometric%2520Modeling%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2017%26volume%3D18%26issue%3D6%26spage%3D540%26epage%3D555%26doi%3D10.2174%2F1389200218666170320121932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwobel, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleva, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enoch, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajot, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, M. T.</span></span> <span> </span><span class="NLM_article-title">Measurement and Estimation of Electrophilic Reactivity for Predictive Toxicology</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>111</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">2562</span>â <span class="NLM_lpage">2596</span>, <span class="refDoi">Â DOI: 10.1021/cr100098n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr100098n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFeku7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2011&pages=2562-2596&issue=4&author=J.+A.+Schwobelauthor=Y.+K.+Kolevaauthor=S.+J.+Enochauthor=F.+Bajotauthor=M.+Hewittauthor=J.+C.+Maddenauthor=D.+W.+Robertsauthor=T.+W.+Schultzauthor=M.+T.+Cronin&title=Measurement+and+Estimation+of+Electrophilic+Reactivity+for+Predictive+Toxicology&doi=10.1021%2Fcr100098n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement and Estimation of Electrophilic Reactivity for Predictive Toxicology</span></div><div class="casAuthors">Schwobel, Johannes A. H.; Koleva, Yana K.; Enoch, Steven J.; Bajot, Fania; Hewitt, Mark; Madden, Judith C.; Roberts, David W.; Schultz, Terry W.; Cronin, Mark T. D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2562-2596</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A database of chem. reactivity data from the past 80 years is compiled for measurement and estn. of electrophilic reactivity for predictive toxicol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgDcG9HOFicLVg90H21EOLACvtfcHk0ljoYF3NY_HiwQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFeku7c%253D&md5=62a4386ab81bb50a91bdef8bb029e74b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fcr100098n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr100098n%26sid%3Dliteratum%253Aachs%26aulast%3DSchwobel%26aufirst%3DJ.%2BA.%26aulast%3DKoleva%26aufirst%3DY.%2BK.%26aulast%3DEnoch%26aufirst%3DS.%2BJ.%26aulast%3DBajot%26aufirst%3DF.%26aulast%3DHewitt%26aufirst%3DM.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DRoberts%26aufirst%3DD.%2BW.%26aulast%3DSchultz%26aufirst%3DT.%2BW.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26atitle%3DMeasurement%2520and%2520Estimation%2520of%2520Electrophilic%2520Reactivity%2520for%2520Predictive%2520Toxicology%26jtitle%3DChem.%2520Rev.%26date%3D2011%26volume%3D111%26issue%3D4%26spage%3D2562%26epage%3D2596%26doi%3D10.1021%2Fcr100098n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarbrough, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. L.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships for Reactivity of Carbonyl-Containing Compounds with Glutathione</span>. <i>SAR and QSAR in Environmental Research</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">313</span>â <span class="NLM_lpage">322</span>, <span class="refDoi">Â DOI: 10.1080/10659360500204152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1080%2F10659360500204152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=16234173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKqtr%252FE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2005&pages=313-322&issue=4&author=T.+W.+Schultzauthor=J.+W.+Yarbroughauthor=E.+L.+Johnson&title=Structure-Activity+Relationships+for+Reactivity+of+Carbonyl-Containing+Compounds+with+Glutathione&doi=10.1080%2F10659360500204152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships for reactivity of carbonyl-containing compounds with glutathione</span></div><div class="casAuthors">Schultz, T. W.; Yarbrough, J. W.; Johnson, E. L.</div><div class="citationInfo"><span class="NLM_cas:title">SAR and QSAR in Environmental Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">313-322</span>CODEN:
                <span class="NLM_cas:coden">SQERED</span>;
        ISSN:<span class="NLM_cas:issn">1062-936X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">For toxicol.-based structure-activity relationships to advance, will require a better understanding of mol. reactivity.  A rapid and inexpensive spectrophotometric assay for detg. the reactive to glutathione (GSH) was developed and used to det. GSH reactivity (reactGSH) data for 21 aliph. derivs. of esters, ketones and aldehydes.  From these data, a series of structure-activity relationships were evaluated.  The structure feature assocd. with reactGSH was an acetylenic or olefinic moiety conjugated to a carbonyl group (i.e. polarized Î±,Î²-unsatn.).  This structure conveys the capacity to undergo a covalent interaction with the thiol group of cysteine (i.e. Michael- addn.).  Quant. reactGSH of the Î±,Î²-unsatd. carbonyl compds. is reliant upon the specific mol. structure with several tendencies obsd.  Specifically, it was noted that for Î±,Î²-unsatd. carbonyl compds.: (1) the acetylenic-substituted derivs. were more reactive than the corresponding olefinic-substituted ones; (2) terminal vinyl-substituted derivs. was more reactive than the internal vinylene-substituted ones; (3) Me substitution on the vinyl carbon atoms diminishes reactivity and methyl-substitution on the carbon atom farthest from the carbonyl group causes a larger redn.; (4) derivs. with carbon-carbon double bond on the end of the mol. (i.e. vinyl ketone) were more reactive than one with the carbon-oxygen bond at the end of the mol. (i.e. aldehyde) and (5) the ester with an addnl. unsatd. vinyl groups were more reactive than the deriv. having an unsatd. Et group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf0zcWMFpuabVg90H21EOLACvtfcHk0ljN7bYzMvNLkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKqtr%252FE&md5=0586fe55450d43235b05155559874647</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1080%2F10659360500204152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10659360500204152%26sid%3Dliteratum%253Aachs%26aulast%3DSchultz%26aufirst%3DT.%2BW.%26aulast%3DYarbrough%26aufirst%3DJ.%2BW.%26aulast%3DJohnson%26aufirst%3DE.%2BL.%26atitle%3DStructure-Activity%2520Relationships%2520for%2520Reactivity%2520of%2520Carbonyl-Containing%2520Compounds%2520with%2520Glutathione%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26date%3D2005%26volume%3D16%26issue%3D4%26spage%3D313%26epage%3D322%26doi%3D10.1080%2F10659360500204152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerlings, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, S.</span></span> <span> </span><span class="NLM_article-title">Local Softness and Hardness Based Reactivity Descriptors for Predicting Intra- and Intermolecular Reactivity Sequences: Carbonyl Compounds</span>. <i>J. Phys. Chem. A</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>102</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">3746</span>â <span class="NLM_lpage">3755</span>, <span class="refDoi">Â DOI: 10.1021/jp973450v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp973450v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADyaK1cXis12nt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1998&pages=3746-3755&issue=21&author=R.+K.+Royauthor=S.+Krishnamurtiauthor=P.+Geerlingsauthor=S.+Pal&title=Local+Softness+and+Hardness+Based+Reactivity+Descriptors+for+Predicting+Intra-+and+Intermolecular+Reactivity+Sequences%3A+Carbonyl+Compounds&doi=10.1021%2Fjp973450v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Local softness and hardness based reactivity descriptors for predicting intra- and intermolecular reactivity sequences: carbonyl compounds</span></div><div class="casAuthors">Roy, R. K.; Krishnamurti, S.; Geerlings, P.; Pal, S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry A</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3746-3755</span>CODEN:
                <span class="NLM_cas:coden">JPCAFH</span>;
        ISSN:<span class="NLM_cas:issn">1089-5639</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DFT-based reactivity descriptors local softness and local hardness are used as reactivity indexes to predict the reactivity sequences (both intramol. and intermol.) of carbonyl compds. toward nucleophilic attack.  The finite-difference approxn. is used to calc. local softness, whereas local hardness is approximated by -Vel/2N, where Vel is the electronic part of the mol. electrostatic potential.  Both aldehydes and ketones, aliph. and arom., were selected as systems.  Crit. cases, e.g., PhCH:CHCHO, MeCH:CHCHO, and CH2:CHCHO, where a C:C double bond is conjugated with the C:O group, are also considered.  Two new reactivity descriptors are proposed, relative electrophilicity (sk+/sk-) and relative nucleophilicity (sk-/sk+), which will help to locate the preferable reactive sites.  Local hardness can be used as a guiding parameter when constructing intermol. reactivity sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriGb2mLcYtX7Vg90H21EOLACvtfcHk0ljN7bYzMvNLkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis12nt7k%253D&md5=c6d1906eee251453d99bc094f25d038b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjp973450v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp973450v%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DR.%2BK.%26aulast%3DKrishnamurti%26aufirst%3DS.%26aulast%3DGeerlings%26aufirst%3DP.%26aulast%3DPal%26aufirst%3DS.%26atitle%3DLocal%2520Softness%2520and%2520Hardness%2520Based%2520Reactivity%2520Descriptors%2520for%2520Predicting%2520Intra-%2520and%2520Intermolecular%2520Reactivity%2520Sequences%253A%2520Carbonyl%2520Compounds%26jtitle%3DJ.%2520Phys.%2520Chem.%2520A%26date%3D1998%26volume%3D102%26issue%3D21%26spage%3D3746%26epage%3D3755%26doi%3D10.1021%2Fjp973450v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, W.</span></span> <span> </span><span class="NLM_article-title">A Combined QM/MM Study on the Reductive Half-Reaction of Xanthine Oxidase: Substrate Orientation and Mechanism</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span> (<span class="NLM_issue">41</span>),  <span class="NLM_fpage">14885</span>â <span class="NLM_lpage">14902</span>, <span class="refDoi">Â DOI: 10.1021/ja9045394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9045394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtF2ltLjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=14885-14902&issue=41&author=S.+Metzauthor=W.+Thiel&title=A+Combined+QM%2FMM+Study+on+the+Reductive+Half-Reaction+of+Xanthine+Oxidase%3A+Substrate+Orientation+and+Mechanism&doi=10.1021%2Fja9045394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Combined QM/MM Study on the Reductive Half-Reaction of Xanthine Oxidase: Substrate Orientation and Mechanism</span></div><div class="casAuthors">Metz, Sebastian; Thiel, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">14885-14902</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Quantum mech./mol. mech. (QM/MM) methods were used to investigate the conversion of xanthine to uric acid in xanthine oxidase.  Seven mechanistic variants were considered with different tautomeric forms of xanthine, different protonation states of the active-site residues, and different substrate orientations.  The most favorable pathway (setup G) has a B3LYP/MM barrier of about 14 kcal mol-1, consistent with the available exptl. data.  This multistep mechanism starts with Glu1261 deprotonating the xanthine at the N3 position followed by a proton transfer from the cofactor to the N9 atom of xanthine; the thus activated cofactor and substrate then react to form a tetrahedral intermediate, and a subsequent rate-limiting hydride transfer generates the product.  The substrate orientation that has commonly been assumed in the literature leads to the most stable reactant complex, but the opposite orientation ("upside down") is computed to be the most favorable one during the reaction (setup G).  In the "upside down" conformation, the Arg880 residue can best stabilize the reactive xanthine species with the neg. charged N3 atom, esp. the tetrahedral intermediate and the following transition state for hydride transfer which is generally the highest point on the computed energy profiles.  QM-only calcns. for a minimal gas-phase model and for larger cluster models are performed for comparison, in particular for establishing intrinsic reactivities and a common energy scale.  An anal. of the computational results provides detailed insight into the essential mechanistic role of the active-site residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-z1Egy9O7xLVg90H21EOLACvtfcHk0ljN7bYzMvNLkw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtF2ltLjM&md5=55c9e18fc825b1b019b7a65ab8040a6a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja9045394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9045394%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DS.%26aulast%3DThiel%26aufirst%3DW.%26atitle%3DA%2520Combined%2520QM%252FMM%2520Study%2520on%2520the%2520Reductive%2520Half-Reaction%2520of%2520Xanthine%2520Oxidase%253A%2520Substrate%2520Orientation%2520and%2520Mechanism%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26issue%3D41%26spage%3D14885%26epage%3D14902%26doi%3D10.1021%2Fja9045394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raileanu, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoffel, K.</span></span> <span> </span><span class="NLM_article-title">Theoretical Comparison Between the Gini Index and Information Gain Criteria</span>. <i>Annals of Mathematics and Artificial Intelligence</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">77</span>â <span class="NLM_lpage">93</span>, <span class="refDoi">Â DOI: 10.1023/B:AMAI.0000018580.96245.c6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1023%2FB%3AAMAI.0000018580.96245.c6" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2004&pages=77-93&issue=1&author=L.+E.+Raileanuauthor=K.+Stoffel&title=Theoretical+Comparison+Between+the+Gini+Index+and+Information+Gain+Criteria&doi=10.1023%2FB%3AAMAI.0000018580.96245.c6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1023%2FB%3AAMAI.0000018580.96245.c6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253AAMAI.0000018580.96245.c6%26sid%3Dliteratum%253Aachs%26aulast%3DRaileanu%26aufirst%3DL.%2BE.%26aulast%3DStoffel%26aufirst%3DK.%26atitle%3DTheoretical%2520Comparison%2520Between%2520the%2520Gini%2520Index%2520and%2520Information%2520Gain%2520Criteria%26jtitle%3DAnnals%2520of%2520Mathematics%2520and%2520Artificial%2520Intelligence%26date%3D2004%26volume%3D41%26issue%3D1%26spage%3D77%26epage%3D93%26doi%3D10.1023%2FB%3AAMAI.0000018580.96245.c6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Potent Inhibition of Human Liver Aldehyde Oxidase by Raloxifene</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">89</span>â <span class="NLM_lpage">97</span>, <span class="refDoi">Â DOI: 10.1124/dmd.32.1.89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.32.1.89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=14709625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFWqs7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=89-97&issue=1&author=R.+S.+Obach&title=Potent+Inhibition+of+Human+Liver+Aldehyde+Oxidase+by+Raloxifene&doi=10.1124%2Fdmd.32.1.89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibition of human liver aldehyde oxidase by raloxifene</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-97</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The selective estrogen receptor modulator, raloxifene, has been demonstrated as a potent uncompetitive inhibitor of human liver aldehyde oxidase-catalyzed oxidn. of phthalazine, vanillin, and nicotine-Î1'(5')-iminium ion, with Ki values of 0.87 to 1.4 nM.  Inhibition was not time-dependent.  Raloxifene has also been shown to be a noncompetitive inhibitor of an aldehyde oxidase-catalyzed redn. reaction of a hydroxamic acid-contg. compd., with a Ki of 51 nM.  However, raloxifene had only small effects on xanthine oxidase, an enzyme related to aldehyde oxidase.  In addn., several other compds. of the same therapeutic class as raloxifene were examd. for their potential to inhibit aldehyde oxidase.  However, none were as potent as raloxifene, since IC50 values were orders of magnitude higher and ranged from 0.29 to 57 Î¼M.  In an examn. of analogs of raloxifene, it was shown that the bisphenol structure with a hydrophobic group on the 3-position of the benzothiophene ring system was the most important element that imparts inhibitory potency.  The relevance of these data to the mechanistic understanding of aldehyde oxidase catalysis, as well as to the potential for raloxifene to cause drug interactions with agents for which aldehyde oxidase-mediated metab. is important, such as zaleplon or famciclovir, is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6iDLIEl8GgrVg90H21EOLACvtfcHk0lgrzhmK2c2Pug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFWqs7w%253D&md5=a693f57077e2d685eea25a998f4603d7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1124%2Fdmd.32.1.89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.32.1.89%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPotent%2520Inhibition%2520of%2520Human%2520Liver%2520Aldehyde%2520Oxidase%2520by%2520Raloxifene%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2004%26volume%3D32%26issue%3D1%26spage%3D89%26epage%3D97%26doi%3D10.1124%2Fdmd.32.1.89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Strelevitz, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orozco, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Hydralazine as a Selective Probe Inactivator of Aldehyde Oxidase in Human Hepatocytes: Estimation of the Contribution of Aldehyde Oxidase to Metabolic Clearance</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>40</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1441</span>â <span class="NLM_lpage">1448</span>, <span class="refDoi">Â DOI: 10.1124/dmd.112.045195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.112.045195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22522748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1441-1448&issue=7&author=T.+J.+Strelevitzauthor=C.+C.+Orozcoauthor=R.+S.+Obach&title=Hydralazine+as+a+Selective+Probe+Inactivator+of+Aldehyde+Oxidase+in+Human+Hepatocytes%3A+Estimation+of+the+Contribution+of+Aldehyde+Oxidase+to+Metabolic+Clearance&doi=10.1124%2Fdmd.112.045195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Hydralazine as a selective probe inactivator of aldehyde oxidase in human hepatocytes: estimation of the contribution of aldehyde oxidase to metabolic clearance</span></div><div class="casAuthors">Strelevitz, Timothy J.; Orozco, Christine C.; Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1441-1448</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Aldehyde oxidase (AO) metab. could lead to significant underestimation of clearance in prediction of human pharmacokinetics as well as unanticipated exposure to AO-generated metabolites, if not accounted for early in drug research.  We report a method using cryopreserved human hepatocytes and the time-dependent AO inhibitor hydralazine (KI = 83 Â± 27 Î¼M, kintact = 0.063 Â± 0.007 min-1), which ests. the contribution of AO metab. relative to total hepatic clearance.  Using zaleplon as a probe substrate and simultaneously monitoring the AO-catalyzed formation of oxozaleplon and the CYP3A-catalyzed formation of desethyzaleplon in the presence of a range of hydralazine concns., it was detd. that >90% inhibition of the AO activity with minimal effect on the CYP3A activity could be achieved with 25 to 50 Î¼M hydralazine.  This method was used to est. the fraction metabolized due to AO [fm(AO)] for six compds. with clearance attributed to AO along with four other drugs not metabolized by AO.  The fm(AO) values for the AO substrates ranged between 0.49 and 0.83.  Differences in estd. fm(AO) between two batches of pooled human hepatocytes suggest that sensitivity to hydralazine varies slightly with hepatocyte prepns.  Substrates with a CYP2D6 contribution to clearance were affected by hydralazine to a minor extent, because of weak inhibition of this enzyme.  Overall, these findings demonstrate that hydralazine, at a concn. of 25 to 50 Î¼M, can be used in human hepatocyte incubations to est. the contribution of AO to the hepatic clearance of drugs and other compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQBKznnCiG-LVg90H21EOLACvtfcHk0lgrzhmK2c2Pug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOrs7vJ&md5=6237313df0fe4ded2d2f37e55c3d4b03</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.045195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.045195%26sid%3Dliteratum%253Aachs%26aulast%3DStrelevitz%26aufirst%3DT.%2BJ.%26aulast%3DOrozco%26aufirst%3DC.%2BC.%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DHydralazine%2520as%2520a%2520Selective%2520Probe%2520Inactivator%2520of%2520Aldehyde%2520Oxidase%2520in%2520Human%2520Hepatocytes%253A%2520Estimation%2520of%2520the%2520Contribution%2520of%2520Aldehyde%2520Oxidase%2520to%2520Metabolic%2520Clearance%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D40%26issue%3D7%26spage%3D1441%26epage%3D1448%26doi%3D10.1124%2Fdmd.112.045195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elion, G. B.</span></span> <span> </span><span class="NLM_article-title">On the Mechanism of Inactivation of Xanthine Oxidase by Allopurinol and Other Pyrazolo[3,4-d]pyrimidines</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>245</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2837</span>â <span class="NLM_lpage">2844</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=5467924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADyaE3cXkt1Wlsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=245&publication_year=1970&pages=2837-2844&issue=11&author=V.+Masseyauthor=H.+Komaiauthor=G.+Palmerauthor=G.+B.+Elion&title=On+the+Mechanism+of+Inactivation+of+Xanthine+Oxidase+by+Allopurinol+and+Other+Pyrazolo%5B3%2C4-d%5Dpyrimidines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism of inactivation of xanthine oxidase by allopurinol and other pyrazolo[3,4-d] pyrimidines</span></div><div class="casAuthors">Massey, Vincent; Komai, Hirochika; Palmer, Graham; Elion, Gertrude B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">245</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2837-44</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Allopurinol and other 6-unsubstituted pyrazolo[3,4-d]pyrimidines have been shown to reduce all of the oxidn.-redn. reactive groups of milk xanthine oxidase.  The great inhibitory action of these compds. has been shown to be due to a sort of suicide reaction, in which the product formed by enzymic hydroxylation in position 6 is joined in complex with a partially reduced form of the enzyme.  Several lines of evidence are presented indicating that complex formation is with the enzyme-bound molybdenum in the Mo(IV) state.  Studies on the stoichiometry of binding of the product of allopurinol reaction, alloxanthine(4,6-dihydroxypyrazolo[3,4 -d]pyrimidine), revealed that the amt. of alloxanthine bound was proportional to the percentage of functional active sites in the enzyme prepn.; i.e. it serves as an active site label.  From these results it is concluded that the best prepns. of xanthine oxidase so far obtained contain only 75-80% functional active sites, even though they possess their full complement of flavine, Mo, and Fe-S components.  Evidence is presented that the presence of nonfunctional active sites is responsible for several of the slow reactions previously observed with the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopDeldmbam1rVg90H21EOLACvtfcHk0lgrzhmK2c2Pug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXkt1Wlsbc%253D&md5=f766d55ced0d40a1c95700bdd14cf1c0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMassey%26aufirst%3DV.%26aulast%3DKomai%26aufirst%3DH.%26aulast%3DPalmer%26aufirst%3DG.%26aulast%3DElion%26aufirst%3DG.%2BB.%26atitle%3DOn%2520the%2520Mechanism%2520of%2520Inactivation%2520of%2520Xanthine%2520Oxidase%2520by%2520Allopurinol%2520and%2520Other%2520Pyrazolo%255B3%252C4-d%255Dpyrimidines%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1970%26volume%3D245%26issue%3D11%26spage%3D2837%26epage%3D2844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maduskuie, T. P.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falahatpisheh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeleswaram, S.</span></span> <span> </span><span class="NLM_article-title">Species-Specific Metabolism of SGX523 by Aldehyde Oxidase and the Toxicological Implications</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1277</span>â <span class="NLM_lpage">1285</span>, <span class="refDoi">Â DOI: 10.1124/dmd.110.032375</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1124%2Fdmd.110.032375" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=20421447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVWktrjM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=1277-1285&issue=8&author=S.+Diamondauthor=J.+Boerauthor=T.+P.+Maduskuieauthor=N.+Falahatpishehauthor=Y.+Liauthor=S.+Yeleswaram&title=Species-Specific+Metabolism+of+SGX523+by+Aldehyde+Oxidase+and+the+Toxicological+Implications&doi=10.1124%2Fdmd.110.032375"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Species-specific metabolism of SGX523 by aldehyde oxidase and the toxicological implications</span></div><div class="casAuthors">Diamond, Sharon; Boer, Jason; Maduskuie, Thomas P., Jr.; Falahatpisheh, Nikoo; Li, Yu; Yeleswaram, Swamy</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1277-1285</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">An investigation was conducted to follow up on the apparent species-dependent toxicity reported for 6-(6-(1-methyl-1H-pyrazol-4-yl)-[1,2,4]triazolo[4,3-b]pyridazin-3-ylthio]quinoline (SGX523), a mesenchymal-epithelial transition factor (c-MET) inhibitor that entered clin. development for the treatment of solid tumors.  Patients treated with SGX523 exhibited compromised renal function presumably resulting from crystal deposits in renal tubules.  Our independent metabolite profiling of SGX523 indicates that a major NADPH-independent, late-eluting metabolite [6-(6-(1-methyl-1H-pyrazol-4-yl)-[1,2,4]triazolo[4,3-b]pyridazin-3-ylthio)quinolin-2(1H)-one (M11)] was generated by monkey and human liver S-9, and to a lesser extent by rat S-9, whereas M11 was absent in dog S-9 incubations.  We confirmed the identity of M11 as 2-quinolinone-SGX523.  Expts. with various molybdenum hydroxylase inhibitors showed that aldehyde oxidase (AO), and not xanthine oxidase, metabolized SGX523 to M11 in monkey and human liver cytosol.  In addn., the oxygen incorporated into M11 was derived from water rather than atm. oxygen, corroborating M11 formation via AO.  After oral dosing in monkeys, metabolite profiling of plasma and urine showed that SGX523 was indeed metabolized to M11 and its N-demethyl analog (M8).  In urine, M11 levels were â¼70-fold greater than that of SGX523, and the soly. of M11 in urine was only 3% of that of SGX523.  In summary, SGX523 is metabolized by AO in a species-specific manner to a markedly less-sol. metabolite, M11.  We propose that M11 is likely involved in the obsd. obstructive nephropathy reported in clin. studies.  Moreover, this study illustrates the need to conduct thorough metabolic evaluations early in drug development to select the most relevant nonclin. species for toxicol. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryx8buDjc41LVg90H21EOLACvtfcHk0lgrzhmK2c2Pug"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVWktrjM&md5=27450436ad7a69d72d4cc8ae9860ccd9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1124%2Fdmd.110.032375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.110.032375%26sid%3Dliteratum%253Aachs%26aulast%3DDiamond%26aufirst%3DS.%26aulast%3DBoer%26aufirst%3DJ.%26aulast%3DMaduskuie%26aufirst%3DT.%2BP.%26aulast%3DFalahatpisheh%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYeleswaram%26aufirst%3DS.%26atitle%3DSpecies-Specific%2520Metabolism%2520of%2520SGX523%2520by%2520Aldehyde%2520Oxidase%2520and%2520the%2520Toxicological%2520Implications%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2010%26volume%3D38%26issue%3D8%26spage%3D1277%26epage%3D1285%26doi%3D10.1124%2Fdmd.110.032375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lolkema, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohets, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkenau, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lampo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, M. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Doorn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellemans, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eskens, F.</span></span> <span> </span><span class="NLM_article-title">The c-Met Tyrosine Kinase Inhibitor JNJ-38877605 Causes Renal Toxicity Through Species-Specific Insoluble Metabolite Formation</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2297</span>â <span class="NLM_lpage">2304</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.CCR-14-3258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1158%2F1078-0432.CCR-14-3258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=25745036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2MXoslOktbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2297-2304&issue=10&author=M.+P.+Lolkemaauthor=H.+H.+Bohetsauthor=H.+T.+Arkenauauthor=A.+Lampoauthor=E.+Baraleauthor=M.+J.+A.+de+Jongeauthor=L.+van+Doornauthor=P.+Hellemansauthor=J.+S.+de+Bonoauthor=F.+Eskens&title=The+c-Met+Tyrosine+Kinase+Inhibitor+JNJ-38877605+Causes+Renal+Toxicity+Through+Species-Specific+Insoluble+Metabolite+Formation&doi=10.1158%2F1078-0432.CCR-14-3258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The c-Met Tyrosine Kinase Inhibitor JNJ-38877605 Causes Renal Toxicity through Species-Specific Insoluble Metabolite Formation</span></div><div class="casAuthors">Lolkema, Martijn P.; Bohets, Hilde H.; Arkenau, Hendrik-Tobias; Lampo, Ann; Barale, Erio; de Jonge, Maja J. A.; van Doorn, Leni; Hellemans, Peter; de Bono, Johann S.; Eskens, Ferry A. L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2297-2304</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The receptor tyrosine kinase c-Met plays an important role in tumorigenesis and is a novel target for anticancer treatment.  This phase I, first-in-human trial, explored safety, pharmacokinetics, pharmacodynamics, and initial antitumor activity of JNJ-38877605, a potent and selective c-Met inhibitor.  Exptl. Design: We performed a phase I dose-escalation study according to the std. 3+3 design.  Results: Even at subtherapeutic doses, mild though recurrent renal toxicity was obsd. in virtually all patients.  Renal toxicity had not been obsd. in preclin. studies in rats and dogs.  Addnl. preclin. studies pointed toward the rabbit as a suitable toxicol. model, as the formation of the M10 metabolite of JNJ-38877605 specifically occurred in rabbits and humans.  Addnl. toxicol. studies in rabbits clearly demonstrated that JNJ-38877605 induced species-specific renal toxicity.  Histopathol. evaluation in rabbits revealed renal crystal formation with degenerative and inflammatory changes.  Identification of the components of these renal crystals revealed M1/3 and M5/6 metabolites.  Accordingly, it was found that humans and rabbits showed significantly increased systemic exposure to these metabolites relative to other species.  These main culprit insol. metabolites were generated by aldehyde oxidase activity.  Alternative dosing schedules of JNJ-3877605 and concomitant probenecid administration in rabbits failed to prevent renal toxicity at dose levels that could be pharmacol. active.  Conclusions: Combined clin. and correlative preclin. studies suggest that renal toxicity of JNJ-38877605 is caused by the formation of species-specific insol. metabolites.  These observations preclude further clin. development of JNJ-38877605.  Clin Cancer Res; 21(10); 2297-304. Â©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro1cpkCcM_IbVg90H21EOLACvtfcHk0lgsgXNw-P_S6Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXoslOktbc%253D&md5=9900b077fbb0766d9c26b3908e7ee1c4</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-3258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-3258%26sid%3Dliteratum%253Aachs%26aulast%3DLolkema%26aufirst%3DM.%2BP.%26aulast%3DBohets%26aufirst%3DH.%2BH.%26aulast%3DArkenau%26aufirst%3DH.%2BT.%26aulast%3DLampo%26aufirst%3DA.%26aulast%3DBarale%26aufirst%3DE.%26aulast%3Dde%2BJonge%26aufirst%3DM.%2BJ.%2BA.%26aulast%3Dvan%2BDoorn%26aufirst%3DL.%26aulast%3DHellemans%26aufirst%3DP.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26aulast%3DEskens%26aufirst%3DF.%26atitle%3DThe%2520c-Met%2520Tyrosine%2520Kinase%2520Inhibitor%2520JNJ-38877605%2520Causes%2520Renal%2520Toxicity%2520Through%2520Species-Specific%2520Insoluble%2520Metabolite%2520Formation%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26issue%3D10%26spage%3D2297%26epage%3D2304%26doi%3D10.1158%2F1078-0432.CCR-14-3258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qing, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, W.</span></span> <span> </span><span class="NLM_article-title">Discovery of (S)-1-(1-(Imidazo[1,2-a]pyridin-6-yl)ethyl)-6-(1-methyl-1H-pyrazol-4-yl)-1H-[1,2, 3]triazolo[4,5-b]pyrazine (Volitinib) as a Highly Potent and Selective Mesenchymal-Epithelial Transition Factor (c-Met) Inhibitor in Clinical Development for Treatment of Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">7577</span>â <span class="NLM_lpage">7589</span>, <span class="refDoi">Â DOI: 10.1021/jm500510f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500510f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSns7jN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7577-7589&issue=18&author=H.+Jiaauthor=G.+Daiauthor=J.+Wengauthor=Z.+Zhangauthor=Q.+Wangauthor=F.+Zhouauthor=L.+Jiaoauthor=Y.+Cuiauthor=Y.+Renauthor=S.+Fanauthor=J.+Zhouauthor=W.+Qingauthor=Y.+Guauthor=J.+Wangauthor=Y.+Saiauthor=W.+Su&title=Discovery+of+%28S%29-1-%281-%28Imidazo%5B1%2C2-a%5Dpyridin-6-yl%29ethyl%29-6-%281-methyl-1H-pyrazol-4-yl%29-1H-%5B1%2C2%2C+3%5Dtriazolo%5B4%2C5-b%5Dpyrazine+%28Volitinib%29+as+a+Highly+Potent+and+Selective+Mesenchymal-Epithelial+Transition+Factor+%28c-Met%29+Inhibitor+in+Clinical+Development+for+Treatment+of+Cancer&doi=10.1021%2Fjm500510f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of (S)-1-(1-(Imidazo[1,2-a]pyridin-6-yl)ethyl)-6-(1-methyl-1H-pyrazol-4-yl)-1H-[1,2,3]triazolo[4,5-b]pyrazine (Volitinib) as a Highly Potent and Selective Mesenchymal-Epithelial Transition Factor (c-Met) Inhibitor in Clinical Development for Treatment of Cancer</span></div><div class="casAuthors">Jia, Hong; Dai, Guangxiu; Weng, Jianyang; Zhang, Zhulin; Wang, Qing; Zhou, Feng; Jiao, Longxian; Cui, Yumin; Ren, Yongxin; Fan, Shiming; Zhou, Jinghong; Qing, Weiguo; Gu, Yi; Wang, Jian; Sai, Yang; Su, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7577-7589</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HGF/c-Met signaling has been implicated in human cancers.  Herein we describe the invention of a series of novel triazolopyrazine c-Met inhibitors.  The structure-activity relation of these compds. was investigated, leading to the identification of Volitinib (I), which demonstrated favorable pharmacokinetic properties in mice and good antitumor activities in the human glioma xenograft model in athymic nude mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJmuzIGrPGZbVg90H21EOLACvtfcHk0lixTWLciiqQQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSns7jN&md5=aced38bbd7431f5ac01b6698e8657261</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm500510f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500510f%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DH.%26aulast%3DDai%26aufirst%3DG.%26aulast%3DWeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DJiao%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DQing%26aufirst%3DW.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSai%26aufirst%3DY.%26aulast%3DSu%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520%2528S%2529-1-%25281-%2528Imidazo%255B1%252C2-a%255Dpyridin-6-yl%2529ethyl%2529-6-%25281-methyl-1H-pyrazol-4-yl%2529-1H-%255B1%252C2%252C%25203%255Dtriazolo%255B4%252C5-b%255Dpyrazine%2520%2528Volitinib%2529%2520as%2520a%2520Highly%2520Potent%2520and%2520Selective%2520Mesenchymal-Epithelial%2520Transition%2520Factor%2520%2528c-Met%2529%2520Inhibitor%2520in%2520Clinical%2520Development%2520for%2520Treatment%2520of%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26issue%3D18%26spage%3D7577%26epage%3D7589%26doi%3D10.1021%2Fjm500510f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, X.</span></span> <span> </span><span class="NLM_article-title">Aldehyde Oxidase-Dependent Species Difference in Hepatic Metabolism of Fasudil to Hydroxyfasudil</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">170</span>â <span class="NLM_lpage">177</span>, <span class="refDoi">Â DOI: 10.1080/00498254.2017.1292016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1080%2F00498254.2017.1292016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=28166443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1Gksbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=170-177&issue=2&author=Z.+Maoauthor=Y.+Wuauthor=Q.+Liauthor=X.+Wangauthor=Y.+Liuauthor=X.+Di&title=Aldehyde+Oxidase-Dependent+Species+Difference+in+Hepatic+Metabolism+of+Fasudil+to+Hydroxyfasudil&doi=10.1080%2F00498254.2017.1292016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Aldehyde oxidase-dependent species difference in hepatic metabolism of fasudil to hydroxyfasudil</span></div><div class="casAuthors">Mao, Zhengsheng; Wu, Yali; Li, Qiuying; Wang, Xin; Liu, Youping; Di, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">170-177</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">An investigation on the metabolic mechanism of fasudil to hydroxyfasudil was conducted in vitro using liver subcellular fractions of different species.  Hydroxyfasudil was generated in large amts. by rat liver S9 and to a similar extent by human liver S9 but was not detected in dog liver S9 incubations.  Studies with various molybdenum hydroxylase inhibitors demonstrated that aldehyde oxidase (AO), but not xanthine oxidase (XO), selectively catalyzed fasudil to hydroxyfasudil in both rat and human liver cytosol.  In addn., the oxygen atom incorporated into hydroxyfasudil was derived from water rather than atm. oxygen, which further corroborated AO involvement.  Enzyme kinetics expts. revealed that fasudil had a higher affinity to human hepatic AO than to rat hepatic AO.  Besides, significantly different in vivo pharmacokinetic parameters obsd. between male and female rats indicated that the AO activity in rats was gender-dependent.  The present study provided first evidences that AO causes differences in fasudil metab. between species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLweM3FIR00rVg90H21EOLACvtfcHk0lixTWLciiqQQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1Gksbk%253D&md5=03b6d2441e74bc60045e1fdcb192a0ec</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F00498254.2017.1292016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00498254.2017.1292016%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDi%26aufirst%3DX.%26atitle%3DAldehyde%2520Oxidase-Dependent%2520Species%2520Difference%2520in%2520Hepatic%2520Metabolism%2520of%2520Fasudil%2520to%2520Hydroxyfasudil%26jtitle%3DXenobiotica%26date%3D2018%26volume%3D48%26issue%3D2%26spage%3D170%26epage%3D177%26doi%3D10.1080%2F00498254.2017.1292016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamidass, S. J.</span></span> <span> </span><span class="NLM_article-title">Modeling Reactivity to Biological Macromolecules With a Deep Multitask Network</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">529</span>â <span class="NLM_lpage">537</span>, <span class="refDoi">Â DOI: 10.1021/acscentsci.6b00162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.6b00162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1CktL%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&pages=529-537&issue=8&author=T.+B.+Hughesauthor=N.+L.+Dangauthor=G.+P.+Millerauthor=S.+J.+Swamidass&title=Modeling+Reactivity+to+Biological+Macromolecules+With+a+Deep+Multitask+Network&doi=10.1021%2Facscentsci.6b00162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Reactivity to Biological Macromolecules with a Deep Multitask Network</span></div><div class="casAuthors">Hughes, Tyler B.; Dang, Na Le; Miller, Grover P.; Swamidass, S. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">529-537</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most small-mol. drug candidates fail before entering the market, frequently because of unexpected toxicity.  Often, toxicity is detected only late in drug development, because many types of toxicities, esp. idiosyncratic adverse drug reactions (IADRs), are particularly hard to predict and detect.  Moreover, drug-induced liver injury (DILI) is the most frequent reason drugs are withdrawn from the market and causes 50% of acute liver failure cases in the United States.  A common mechanism often underlies many types of drug toxicities, including both DILI and IADRs.  Drugs are bioactivated by drug-metabolizing enzymes into reactive metabolites, which then conjugate to sites in proteins or DNA to form adducts.  DNA adducts are often mutagenic and may alter the reading and copying of genes and their regulatory elements, causing gene dysregulation and even triggering cancer.  Similarly, protein adducts can disrupt their normal biol. functions and induce harmful immune responses.  Unfortunately, reactive metabolites are not reliably detected by expts., and it is also expensive to test drug candidates for potential to form DNA or protein adducts during the early stages of drug development.  In contrast, computational methods have the potential to quickly screen for covalent binding potential, thereby flagging problematic mols. and reducing the total no. of necessary expts.  Here, the authors train a deep convolution neural network-the XenoSite reactivity model-using literature data to accurately predict both sites and probability of reactivity for mols. with glutathione, cyanide, protein, and DNA.  On the site level, cross-validated predictions had area under the curve (AUC) performances of 89.8% for DNA and 94.4% for protein.  Furthermore, the model sepd. mols. electrophilically reactive with DNA and protein from nonreactive mols. with cross-validated AUC performances of 78.7% and 79.8%, resp.  On both the site- and mol.-level, the model's performances significantly outperformed reactivity indexes derived from quantum simulations that are reported in the literature.  Moreover, the authors developed and applied a selectivity score to assess preferential reactions with the macromols. as opposed to the common screening traps.  For the entire data set of 2803 mols., this approach yielded totals of 257 (9.2%) and 227 (8.1%) mols. predicted to be reactive only with DNA and protein, resp., and hence those that would be missed by std. reactivity screening expts.  Site of reactivity data is an underutilized resource that can be used to not only predict if mols. are reactive, but also show where they might be modified to reduce toxicity while retaining efficacy.  The XenoSite reactivity model is available at http://swami.wustl.edu/xenosite/p/reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC4SVmokCrNbVg90H21EOLACvtfcHk0lixTWLciiqQQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1CktL%252FN&md5=33cbff16248c5fa113a00539d9d9ecd5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.6b00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.6b00162%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DT.%2BB.%26aulast%3DDang%26aufirst%3DN.%2BL.%26aulast%3DMiller%26aufirst%3DG.%2BP.%26aulast%3DSwamidass%26aufirst%3DS.%2BJ.%26atitle%3DModeling%2520Reactivity%2520to%2520Biological%2520Macromolecules%2520With%2520a%2520Deep%2520Multitask%2520Network%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2016%26volume%3D2%26issue%3D8%26spage%3D529%26epage%3D537%26doi%3D10.1021%2Facscentsci.6b00162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G. E.</span></span> In  <i>Neural Networks: Tricks of the Trade</i>; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">599</span>â <span class="NLM_lpage">619</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1007%2F978-3-642-35289-8_32" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=599-619&author=G.+E.+Hinton&title=Neural+Networks%3A+Tricks+of+the+Trade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-35289-8_32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-35289-8_32%26sid%3Dliteratum%253Aachs%26aulast%3DHinton%26aufirst%3DG.%2BE.%26btitle%3DNeural%2520Networks%253A%2520Tricks%2520of%2520the%2520Trade%26pub%3DSpringer%26date%3D2012%26spage%3D599%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speybroeck, N.</span></span> <span> </span><span class="NLM_article-title">Classification and Regression Trees</span>. <i>International Journal of Public Health</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">243</span>â <span class="NLM_lpage">246</span>, <span class="refDoi">Â DOI: 10.1007/s00038-011-0315-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1007%2Fs00038-011-0315-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=22015650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A280%3ADC%252BC383ksVynug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=243-246&issue=1&author=N.+Speybroeck&title=Classification+and+Regression+Trees&doi=10.1007%2Fs00038-011-0315-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Classification and regression trees</span></div><div class="casAuthors">Speybroeck N</div><div class="citationInfo"><span class="NLM_cas:title">International journal of public health</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-6</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS2T0B6WFwfoUodMGTatQ7LfW6udTcc2ebx6WBV3dUgJrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC383ksVynug%253D%253D&md5=d0a601b734fb50f9291317ec45ccfacc</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs00038-011-0315-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00038-011-0315-z%26sid%3Dliteratum%253Aachs%26aulast%3DSpeybroeck%26aufirst%3DN.%26atitle%3DClassification%2520and%2520Regression%2520Trees%26jtitle%3DInternational%2520Journal%2520of%2520Public%2520Health%26date%3D2012%26volume%3D57%26issue%3D1%26spage%3D243%26epage%3D246%26doi%3D10.1007%2Fs00038-011-0315-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henrard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speybroeck, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermans, C.</span></span> <span> </span><span class="NLM_article-title">Classification and Regression Tree Analysis vs. Multivariable Linear and Logistic Regression Methods as Statistical Tools for Studying Haemophilia</span>. <i>Haemophilia</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">715</span>â <span class="NLM_lpage">722</span>, <span class="refDoi">Â DOI: 10.1111/hae.12778</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=10.1111%2Fhae.12778" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=26248714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;key=1%3ACAS%3A280%3ADC%252BC287gsVaqsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=715-722&issue=6&author=S.+Henrardauthor=N.+Speybroeckauthor=C.+Hermans&title=Classification+and+Regression+Tree+Analysis+vs.+Multivariable+Linear+and+Logistic+Regression+Methods+as+Statistical+Tools+for+Studying+Haemophilia&doi=10.1111%2Fhae.12778"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Classification and regression tree analysis vs. multivariable linear and logistic regression methods as statistical tools for studying haemophilia</span></div><div class="casAuthors">Henrard S; Speybroeck N; Henrard S; Hermans C</div><div class="citationInfo"><span class="NLM_cas:title">Haemophilia : the official journal of the World Federation of Hemophilia</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">715-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Haemophilia is a rare genetic haemorrhagic disease characterized by partial or complete deficiency of coagulation factor VIII, for haemophilia A, or IX, for haemophilia B.  As in any other medical research domain, the field of haemophilia research is increasingly concerned with finding factors associated with binary or continuous outcomes through multivariable models.  Traditional models include multiple logistic regressions, for binary outcomes, and multiple linear regressions for continuous outcomes.  Yet these regression models are at times difficult to implement, especially for non-statisticians, and can be difficult to interpret.  AIMS:  The present paper sought to didactically explain how, why, and when to use classification and regression tree (CART) analysis for haemophilia research.  MATERIALS & METHODS:  The CART method is non-parametric and non-linear, based on the repeated partitioning of a sample into subgroups based on a certain criterion.  Breiman developed this method in 1984.  Classification trees (CTs) are used to analyse categorical outcomes and regression trees (RTs) to analyse continuous ones.  RESULTS:  The CART methodology has become increasingly popular in the medical field, yet only a few examples of studies using this methodology specifically in haemophilia have to date been published.  Two examples using CART analysis and previously published in this field are didactically explained in details.  CONCLUSION:  There is increasing interest in using CART analysis in the health domain, primarily due to its ease of implementation, use, and interpretation, thus facilitating medical decision-making.  This method should be promoted for analysing continuous or categorical outcomes in haemophilia, when applicable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcScPq4eJV7QaEh5eUSa2oe_fW6udTcc2ebx6WBV3dUgJrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287gsVaqsQ%253D%253D&md5=04ddd829a199f7144b3ee6c768160fb1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1111%2Fhae.12778&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fhae.12778%26sid%3Dliteratum%253Aachs%26aulast%3DHenrard%26aufirst%3DS.%26aulast%3DSpeybroeck%26aufirst%3DN.%26aulast%3DHermans%26aufirst%3DC.%26atitle%3DClassification%2520and%2520Regression%2520Tree%2520Analysis%2520vs.%2520Multivariable%2520Linear%2520and%2520Logistic%2520Regression%2520Methods%2520as%2520Statistical%2520Tools%2520for%2520Studying%2520Haemophilia%26jtitle%3DHaemophilia%26date%3D2015%26volume%3D21%26issue%3D6%26spage%3D715%26epage%3D722%26doi%3D10.1111%2Fhae.12778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UHX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UHX','PDB','4UHX'); return false;">PDB: 4UHX</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01895">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_12735"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01895?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01895</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures and tables illustrating the sample distribution according to the applicability domain defined by DT<sub>AOX</sub> (Figure S1); relationship of Î<i>E</i><sub>inter</sub> versus the ESP charge and the chemical shift of the reactive carbon (Figure S2); prediction results of molecules in <a id="rightTab-" class=" internalNav" href="#fig2">Figure <a id="rightTab-" class=" internalNav" href="#fig2">2</a></a> by DNN substrate/nonsubstrate classification model (Figure S3); ion extraction chromatographs of the positive control (Figure S4); positive product ion spectra of the positive control (Figure S5); predictive performance of DT<sub>AOX</sub> on newly published data (Table S1); distributions of descriptors between SOMs and nonSOMs (Table S2); descriptor values of four substrates predicted by fusing model (Table S3); substrates with SOM information collected from newly published work (Table S4); nonsubstratesâ structures collected from newly published work (Table S5); structures used for experimental validation (Table S6); inhibition study of oxidative metabolites for positive control incubated in human liver cytosol with or without human AOX chemical inhibitors (Table S7); LCâMS peak areas of oxidative metabolites for compounds incubated in human liver cytosol with or without human AOX chemical inhibitors (Table S8); LCâMS peak areas of oxidative metabolites for compounds incubated in human liver cytosol with or without human XO chemical inhibitor (Table S9) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and other information used in model construction (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_001.pdf">jm9b01895_si_001.pdf (633.5 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01895/suppl_file/jm9b01895_si_002.csv">jm9b01895_si_002.csv (66.47 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01895&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-12%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01895%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01895" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679abeb24f423cd6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
