
<html lang="en"     class="pb-page"  data-request-id="53e59dc9-4258-49b6-9f17-a707e34e0bf6"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02176;issue:issue:10.1021/jmcmar.2021.64.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1" /></meta><meta name="dc.Creator" content="Xing-Jie  Dai" /></meta><meta name="dc.Creator" content="Ying  Liu" /></meta><meta name="dc.Creator" content="Lei-Peng  Xue" /></meta><meta name="dc.Creator" content="Xiao-Peng  Xiong" /></meta><meta name="dc.Creator" content="Ying  Zhou" /></meta><meta name="dc.Creator" content="Yi-Chao  Zheng" /></meta><meta name="dc.Creator" content="Hong-Min  Liu" /></meta><meta name="dc.Description" content="As a flavin adenine dinucleotide (FAD)-dependent monoamine oxidase, lysine specific demethylase 1 (LSD1/KDM1A) functions as a transcription coactivator or corepressor to regulate the methylation of..." /></meta><meta name="Description" content="As a flavin adenine dinucleotide (FAD)-dependent monoamine oxidase, lysine specific demethylase 1 (LSD1/KDM1A) functions as a transcription coactivator or corepressor to regulate the methylation of..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 23, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02176" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02176" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02176" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02176" /></link>
        
    
    

<title>Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02176" /></meta><meta property="og:title" content="Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0018.jpeg" /></meta><meta property="og:description" content="As a flavin adenine dinucleotide (FAD)-dependent monoamine oxidase, lysine specific demethylase 1 (LSD1/KDM1A) functions as a transcription coactivator or corepressor to regulate the methylation of histone 3 lysine 4 and 9 (H3K4/9), and it has emerged as a promising epigenetic target for anticancer treatment. To date, numerous inhibitors targeting LSD1 have been developed, some of which are undergoing clinical trials for cancer therapy. Although only two reversible LSD1 inhibitors CC-90011 and SP-2577 are in the clinical stage, the past decade has seen remarkable advances in the development of reversible LSD1 inhibitors. Herein, we provide a comprehensive review about structures, biological evaluation, and structure–activity relationship (SAR) of reversible LSD1 inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02176"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02176">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02176&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02176&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02176&amp;href=/doi/10.1021/acs.jmedchem.0c02176" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2466-2488</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01746" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c02227" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.1c00568"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xing-Jie Dai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xing-Jie Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xing-Jie++Dai">Xing-Jie Dai</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Liu">Ying Liu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lei-Peng Xue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lei-Peng Xue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lei-Peng++Xue">Lei-Peng Xue</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiao-Peng Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiao-Peng Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiao-Peng++Xiong">Xiao-Peng Xiong</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Zhou">Ying Zhou</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi-Chao Zheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi-Chao Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +8637167781908. E-mail: <a href="/cdn-cgi/l/email-protection#9ae3f3f9f2fbf5e0f2fff4fddae0e0efb4fffeefb4f9f4"><span class="__cf_email__" data-cfemail="ed94848e858c82978588838aad979798c3888998c38e83">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi-Chao++Zheng">Yi-Chao Zheng</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath6">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2662-3770" title="Orcid link">http://orcid.org/0000-0002-2662-3770</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Hong-Min Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong-Min Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, State Key Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory of Henan Province for Drug Quality and Evaluation, Institute of Drug Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +8637167781739. E-mail: <a href="/cdn-cgi/l/email-protection#6a06031f02072a10101f440f0e1f440904"><span class="__cf_email__" data-cfemail="533f3a263b3e132929267d3637267d303d">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong-Min++Liu">Hong-Min Liu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath7">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6771-9421" title="Orcid link">http://orcid.org/0000-0001-6771-9421</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02176&amp;href=/doi/10.1021%2Facs.jmedchem.0c02176" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2466–2488</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 23, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 December 2020</li><li><span class="item_label"><b>Published</b> online</span>23 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2466%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXing-Jie%2BDai%252C%2BYing%2BLiu%252C%2BLei-Peng%2BXue%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D5%26contentID%3Dacs.jmedchem.0c02176%26title%3DReversible%2BLysine%2BSpecific%2BDemethylase%2B1%2B%2528LSD1%2529%2BInhibitors%253A%2BA%2BPromising%2BWrench%2Bto%2BImpair%2BLSD1%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2488%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02176"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1642</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02176" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xing-Jie&quot;,&quot;last_name&quot;:&quot;Dai&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Lei-Peng&quot;,&quot;last_name&quot;:&quot;Xue&quot;},{&quot;first_name&quot;:&quot;Xiao-Peng&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Yi-Chao&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Hong-Min&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;2466-2488&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02176&quot;},&quot;abstract&quot;:&quot;As a flavin adenine dinucleotide (FAD)-dependent monoamine oxidase, lysine specific demethylase 1 (LSD1/KDM1A) functions as a transcription coactivator or corepressor to regulate the methylation of histone 3 lysine 4 and 9 (H3K4/9), and it has emerged as a promising epigenetic target for anticancer treatment. To date, numerous inhibitors targeting LSD1 have been developed, some of which are undergoing clinical trials for cancer therapy. Although only two reversible LSD1 inhibitors CC-90011 and SP-2577 are in the clinical stage, the past decade has seen remarkable advances in the development of reversible LSD1 inhibitors. Herein, we provide a comprehensive review about structures, biological evaluation, and structure–activity relationship (SAR) of reversible LSD1 inhibitors.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02176&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02176" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02176&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02176" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02176&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02176" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02176&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02176&amp;href=/doi/10.1021/acs.jmedchem.0c02176" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02176" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02176" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26pmid%3D33619958%26genre%3Darticle%26aulast%3DDai%26date%3D2021%26atitle%3DReversible%2BLysine%2BSpecific%2BDemethylase%2B1%2B%2528LSD1%2529%2BInhibitors%253A%2BA%2BPromising%2BWrench%2Bto%2BImpair%2BLSD1%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/jmcmar.2021.64.issue-5/20210311/jmcmar.2021.64.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">As a flavin adenine dinucleotide (FAD)-dependent monoamine oxidase, lysine specific demethylase 1 (LSD1/KDM1A) functions as a transcription coactivator or corepressor to regulate the methylation of histone 3 lysine 4 and 9 (H3K4/9), and it has emerged as a promising epigenetic target for anticancer treatment. To date, numerous inhibitors targeting LSD1 have been developed, some of which are undergoing clinical trials for cancer therapy. Although only two reversible LSD1 inhibitors CC-90011 and SP-2577 are in the clinical stage, the past decade has seen remarkable advances in the development of reversible LSD1 inhibitors. Herein, we provide a comprehensive review about structures, biological evaluation, and structure–activity relationship (SAR) of reversible LSD1 inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68723" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68723" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone methylation is one of the most extensively investigated epigenetic modifications, which was initially considered to be irreversible until lysine specific demethylase 1 (LSD1) was characterized by Shi in 2004 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Since this landmark discovery, research on the role of LSD1 in cancers and other diseases has grown dramatically in the past decade. LSD1 consists of 852 amino acids, forming the N-terminal Swi3p/Rsc8p/Moira (SWIRM) α-helical domain, the substrate-binding site and FAD-binding site containing the amine oxidase domain, and the TOWER domain (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure and function of LSD1. (A) Cocrystal structure of LSD1/CoREST complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>). (B) FAD-dependent LSD1 mediated demethylation of methylated lysine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">LSD1 is a FAD-dependent oxidase that demethylates mono- and dimethylated H3K4 when it associates with RE1 silencing transcription factor corepressor (CoREST) and functions as a transcription repressor,<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> and demethylates mono- and dimethylated H3K9 when it colocalizes with the androgen receptor (AR) and acts as a transcription activator (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> As LSD1 can remove transcriptional activating marks, it has the potential to repress gene expression, and inhibitor targeting LSD1 can be used to induce the re-expression of aberrantly silenced tumor suppressor genes for anticancer therapy.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> LSD1 was also reported to demethylate nonhistone substrates, such as protein p53 and DNA methyltransferase 1 (DNMT1).<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Besides, it shares a similar sequence and structure with monoamine oxidases (MAOs)<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and lysine specific demethylase 2 (LSD2/KDM1B, another FAD-dependent demethylase discovered by Karytinos et al. in 2009<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a>). Increasing evidence has implicated the significant role of LSD1 in various cancers (such as small cell lung cancer (SCLC), bladder cancer, gastric cancer, prostate cancer, and acute myeloid leukemia (AML)) as well as several noncancer diseases (such as neurodegeneration, viral infections, blood disorders, and cardiovascular), making it a promising therapeutic target for the treatment of diverse diseases, especially for cancer therapy.<a onclick="showRef(event, 'ref2 ref14 ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref2 ref14 ref15 ref16 ref17 ref18 ref19">(2,14−19)</a></div><div class="NLM_p">Significant advances have been made toward the discovery of effective LSD1 inhibitors over the past decade.<a onclick="showRef(event, 'ref15 ref20 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref15 ref20 ref21 ref22 ref23 ref24 ref25">(15,20−25)</a> Particularly, six tranylcypromine (TCP)-based irreversible LSD1 inhibitors are currently undergoing clinical trials alone or in combination with other drugs for the treatment of SCLC and AML, including TCP, GSK2879552, IMG-7289, ORY-1001, INCB059872, and ORY-2001 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> Very recently, we have provided an extensive review of the development of TCP-based irreversible LSD1 inhibitors.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. LSD1 inhibitors in clinical trials. (A) Structure of TCP-based irreversible LSD1 inhibitors in clinical trials. (B) Structure of reversible LSD1 inhibitors in clinical trials.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">However, TCP-based LSD1 inhibitors are often accompanied by side effects due to the covalent binding to FAD and high affinity to a variety of targets.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Apart from TCP-based inhibitors, phenelzine- and pargyline-based irreversible LSD1 inhibitors exhibited insufficient activities and poor selectivity.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> In recent years, in order to investigate whether the efficacy/safety profile can be improved by noncovalent LSD1 inhibitors, research on reversible LSD1 inhibitors with diverse structures is in full swing by many researchers, including us.<a onclick="showRef(event, 'ref20 ref22 ref23 ref31'); return false;" href="javascript:void(0);" class="ref ref20 ref22 ref23 ref31">(20,22,23,31)</a> In the development of LSD1 inhibitors, compared to irreversible inhibitors, only a few reversible inhibitors warrant further optimization. Thus, the discovery of novel reversible LSD1 inhibitors with diverse chemical structures and remarkable bioactivities has become a promising strategy for cancer therapy. Despite the prominent progress on reversible LSD1 inhibitors, in particular, CC-90011 and SP-2577 are in clinical trials (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), no comprehensive reviews on reversible LSD1 inhibitors have been published to date. Therefore, it is important and necessary to review the progress and shed light on further trends in this promising field. This review provides a comprehensive overview of molecular structures, activities, and SAR of reversible LSD1 inhibitors that have been reported over the past decade. These inhibitors can be classified into nine categories depending upon their structural characteristics and nature of the functional groups, including polyamine derivatives, polypeptide derivatives, six-membered heterocyclic compounds, five-membered heterocyclic compounds, fused heterocyclic compounds, styrene derivatives, 3-oxoamino-benzenesulfonamide derivatives, metal complexes, and others. Comparison of these classes of reversible LSD1 inhibitors is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Comparison of Nine Classes of Reversible LSD1 Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0016.gif" alt="" id="gr16" /></img><div></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">2.  Reversible LSD1 Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.1.  Polyamine Derivatives</h3><div class="NLM_p">Polyamine derivatives were among the first developed reversible LSD1 inhibitors. As reported, LSD1 shares close sequence homology with spermine oxidase (SMOX),<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and guanidines have been reported to inhibit SMOX.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Inspired by this finding, in 2007, Huang et al. identified biguanide and bisguanidine polyamine derivatives as novel LSD1 inhibitors (>50% inhibition at 1 μM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Double reciprocal plots indicated that the two most potent compounds <b>1</b> and <b>2</b> inhibited LSD1 in a noncompetitive manner, and they played a role by inducing the activation of silenced secreted frizzled-related protein 4/5 (SFRP4/5) and GATA binding protein 5 (GATA5) in colon carcinoma cell line HCT116, and the repression of specific genes was concurrent with increased H3K4me2 and acetyl-H3K9 levels as well as decreased H3K9me1/2. In 2012, the same group further reported that compound <b>1</b> and PG11144 (compound <b>3</b>) could significantly increase global H3K4me1/2 in human breast cancer cell line MDA-MB-231 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Polyamine derivatives. (A) Structure and properties of amidinoguanidinium derivatives as LSD1 inhibitors. (B) Structure and properties of thiourea derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2011, Wang et al. reported several specific nonpeptide LSD1 inhibitors, including CBB1003 (compound <b>4</b>) and CBB1007 (compound <b>5</b>), based on the structural features of the acidic surface in LSD1 active cavity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Their guanidinium groups formed hydrogen bond interactions with surrounding negatively charged residues. By using these molecules as probes, inhibition of LSD1 blocked multiple pluripotent cancer cell growth. Further investigations revealed that CBB1003 could inhibit colorectal cancer cell (CRC) growth.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div><div class="NLM_p last">In 2015, Nowotarski et al. reported (bis)urea and (bis)thiourea-based LSD1 inhibitors.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Among them, compounds <b>6</b>–<b>8</b> exhibited moderate LSD1 inhibitory activity <i>in vitro</i> with IC<sub>50</sub> values ranging between 5 and 8 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Besides, at the cellular level, compound <b>8</b> showed moderate inhibitory activity against lung adenocarcinoma cell line Calu-6 (IC<sub>50</sub> = 4.2 μM) and breast cancer cell line MCF-7 (IC<sub>50</sub> = 4.8 μM). In addition, compounds <b>6</b>–<b>8</b> markedly promoted the mRNA expression for secreted frizzled-related protein 2 (SFRP2), H-cadherin (HCAD), and protein p16 in Calu-6 cells.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.2.  Polypeptide Derivatives</h3><div class="NLM_p">N-terminal 21 amino acids of snail family transcriptional repressor 1 (Snail1) bind to the active site of LSD1 in a conformation like that of H3, and thus Snail1 can act as an endogenous LSD1 inhibitor by competing with the H3 substrate for binding to LSD1. In 2013, Tortorici et al. characterized a series of short peptides that can inhibit LSD1 reversibly (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Snail1 1–6 peptide (compound <b>9</b>), 1–9 peptide (compound <b>10</b>), and 1–20 (compound <b>11</b>) peptide inhibited the LSD1/CoREST complex moderately with <i>K</i><sub>i</sub> values of 28.4, 0.14, and 0.21 μM, respectively. Although longer peptides have strong binding affinities for LSD1, their ligand efficiency is poor. Therefore, the Snail1 1–6 peptide was selected for further optimization. Enzymatic, crystallographic (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZMT">3ZMT</a>), spectroscopic, and computational studies revealed that Snail1 1–6 peptide bound to LSD1/CoREST by recognizing the positively charged α-helical turn. Several additional hydrogen bonds were formed between the polar groups (carbonyl, hydroxyl, and amine groups) of Snail1 1–6 and the side chains of Asn535, Ala539, Asp553, Asp556, Glu559, and His564. Besides, these short peptide-based inhibitors exhibited promising antiproliferative activities against human promyelocytic leukemia cell line HL-60.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Polypeptide derivatives. (A) Structure and properties of linear peptides based LSD1 inhibitors. (B) Structure and properties of cyclic peptides based LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Besides, macrocyclic peptide provides an additional choice for peptide-based LSD1 inhibitors.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> In 2007, Forneris et al. reported a 21-mer peptide analogous to the H3K4, where Lys4 was mutated to a methionine (compound <b>12</b>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Linear peptide <b>12</b> was an effective inhibitor of LSD1 (<i>K</i><sub>i</sub> = 40 nM) and inhibited the LSD1/CoREST complex potently (<i>K</i><sub>i</sub> = 50 nM). Structure analysis of peptide <b>12</b> in complex with LSD/CoREST revealed that the side chains of some paired amino acid residues (including Arg2 and Gln5, Arg2 and Ser10, Arg2 and Lys14, and Gln5 and Ser10) were close to each other. In 2013, Kumarasinghe et al. developed a series of cyclic peptides as novel potent LSD1 inhibitors based on the 21 amino acid of H3K4.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> To mimic the binding conformation of peptide <b>12</b>, these new cyclic peptides were constructed by substituting one lysine and one glutamic acid at selected positions and, these residues were cyclized to form a lactam bridge. Among them, cyclic peptide <b>13</b> produced the greatest LSD1 inhibition (IC<sub>50</sub> = 2.1 μM, <i>K</i><sub>i</sub> = 385 nM) and showed modest antitumor activity against MCF-7 and Calu-6 cell lines. In 2018, the same group used alanine scanning mutagenesis to identify critical residues in cyclic peptide <b>13</b> for binding to the active site, and a few more active mutated peptides against recombinant LSD1/CoREST were developed.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Among them, cyclic mutant peptides <b>14</b> and <b>15</b> showed the most potent LSD1 inhibitory activity with IC<sub>50</sub> values of 136 nM and 107 nM, respectively. Also, cyclic peptides <b>14</b> and <b>15</b> could elevate H3K4me2 levels in human erythroleukemia K562 cells.</div><div class="NLM_p">In 2020, Yang et al. reported macrocyclic peptide-based reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The linear 11-mer compound <b>16</b> where Lys4 was replaced by a methionine was characterized as the shortest peptide-based LSD1 inhibitor. Compared to linear 11-mer compound <b>16</b>, macrocycle <b>17</b> exhibited higher LSD1 inhibitory activity (<i>K</i><sub>i</sub> = 2.3 μM). Cocrystal structure of compound <b>17</b> in complex with LSD1/CoREST1 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S35">6S35</a>) revealed that it was located around the outer rim of the LSD1 active site with no direct interactions with FAD. The α-amino group of Ala1 of compound <b>17</b> was situated in the LSD1 cationic pocket formed by Asp375 and Glu379, while it formed a salt-bridge with Asp375. The carbonyl and amino groups of Ala1 of compound <b>17</b> formed hydrogen bond interactions with Cys360, while the α-amide of Arg2 also shared a hydrogen bond with the carboxylate group of Glu379. Furthermore, the carbonyl group of Met4 formed a hydrogen bond with Asn535, and two additional hydrogen bonds were formed between His564 and the carbonyl groups of Ala7 and Glu6. Additionally, as illustrated by nuclear magnetic resonance (NMR) spectra, a turn at Gln5 was found in the LSD1-bound conformation of compound <b>17</b>.</div><div class="NLM_p last">In 2016, Speranzini et al. identified antibiotics polymyxins B (compound <b>18</b>) and E (compound <b>19</b>) as natural cyclic peptide based LSD1 inhibitors from their in-house compound library (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> A further enzymatic assay indicated that polymyxins B (<i>K</i><sub>i</sub> = 157 nM) and E (<i>K</i><sub>i</sub> = 193 nM) competed with H3K4 to inhibit LSD1/CoREST potently. Cocrystal structures of LSD1/CoREST in complex with polymyxins B (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3F">5L3F</a>) and E (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3G">5L3G</a>) indicated that they both interacted with negatively charged residues Glu559, Asp555, Asp556, Asp553, and Glu379. However, they could not penetrate deeply into the catalytic pocket and are far from the flavin (>5 Å).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.3.  Six-Membered Heterocyclic Compounds</h3><div class="NLM_p">GSK-690 (compound <b>20</b>, also known as GSK-354) is a pyridine-based reversible LSD1 inhibitor disclosed in 2013 ACCR meeting (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In the same year, Hitchin et al. reported that GSK-690 inhibited LSD1 potently (IC<sub>50</sub> = 90 nM) with excellent selectivity for LSD1 over MAO-A (IC<sub>50</sub> > 200 μM).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In THP-1 cells, GSK-690 induced the expression of the surrogate cellular biomarker CD86 and showed moderate cellular LSD1 inhibitory activity (IC<sub>50</sub> = 1.4 μM) with non-cytotoxicity at 20 μM.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Six-membered heterocyclic compounds. (A) Structure and properties of pyridine derivatives as LSD1 inhibitors. (B) Structure and properties of pyrimidine derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2015, Wu et al. identified several 3-(piperidin-4-ylmethoxy)-pyridine derivatives as novel LSD1 inhibitors sharing a similar structure with GSK-690 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, compound <b>22</b> resulting from attaching a piperidin-4-ylmethyl group to the amino group of TCP (compound <b>21</b>) significantly increased LSD1 inhibitory potency and selectivity. To design competitive drug-like LSD1 inhibitors, compound <b>23</b> retained the phenyl group and the piperidin-4-ylmethyl group in compound <b>22</b>, and the cyclopropylamine moiety was replaced by a pyridine ring. A docking study of compound <b>23</b> revealed that the pyridine ring had hydrophobic and electrostatic interactions with FAD and Tyr761. The piperidine amino group made a hydrogen bond with Asp555. The introduction of the benzene ring enhanced the binding affinity of rationally designed compound <b>24</b> to LSD1, which was identified as a weak LSD1 inhibitor (<i>K</i><sub>i</sub> = 47.8 μM). SAR studies based on compound <b>24</b> identified compound <b>25</b> (<i>K</i><sub>i</sub> = 29 nM) as the most potent LSD1 inhibitor that can induce the accumulation of H3K4me2 in leukemia cell line MV4-11. Furthermore, compound <b>25</b> also strongly inhibited proliferation of leukemia cell lines MV4–11 (EC<sub>50</sub> = 0.36 μM) and Molm-13 (EC<sub>50</sub> = 3.4 μM) as well as breast cancer cell lines MDA-MB-231 (EC<sub>50</sub> = 5.6 μM) and MCF-7 (EC<sub>50</sub> = 3.6 μM). In addition, compound <b>25</b> suppressed normal fibroblast cells WI-38 growth weakly (EC<sub>50</sub> = 26.6 μM), indicating its acceptable cellular selectivity. In 2018, Niwa et al. determined the cocrystal structure of compound <b>25</b> complexed with LSD1/CoREST (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5YJB">5YJB</a>).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> This crystal structure showed that the piperidine ring penetrated into the negatively charged cavity and interacted with Asn540 and Asp555 alternately. The 4-cyanophenyl group stood deeply in the LSD1 substrate-binding pocket and interacted with Lys661. Furthermore, the 4-methylphenyl group bound to a hydrophobic cavity.</div><div class="NLM_p">In 2015, our group developed pyrimidine-thiourea hybrids as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> A terminal alkyne appendage containing compound <b>26</b> was shown to be one of the most active and selective LSD1 inhibitors, and it performed inhibitory activity against LSD1 in a time-dependent manner. At the biochemical level, the false-positive result was excluded by the coupled Amplex Red/horseradish peroxidase (HRP) assay. Further, ultrafiltration and biolayer interferometry (BLI) confirmed compound <b>26</b> bound to LSD1 tightly but reversibly. Additionally, compound <b>26</b> was found to exert strong cytotoxic effects against gastric cancer cells overexpressing LSD1. Besides, compound <b>26</b> also exhibited significant inhibitory effects on cell invasion and migration, as well as tumor-suppressing and antimetastasis effects <i>in vivo</i> without obvious toxicity.</div><div class="NLM_p">Osimertinib (compound <b>27</b>, also known as AZD9291) is a tyrosine kinase inhibitor and anti-neoplastic agent for treating epidermal growth factor receptor (EGFR) mutant nonsmall cell lung cancer (NSCLC). As both LSD1 and EGFR are important drug targets for NSCLC therapy, in 2019, our group first discovered osimertinib as a promising LSD1 inhibitor with an IC<sub>50</sub> value of 3.98 μM and then performed cellular LSD1 inhibitory activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Lineweaver–Burk plots and dilution assay characterized osimertinib as a FAD-competitive and reversible LSD1 inhibitor. In addition, osimertinib could inhibit the proliferation and migration of NCI-H1975 cells. Docking studies showed that osimertinib occupied the LSD1 active pocket away from FAD. The carbonyl group formed a hydrogen bond with Thr335, while the <i>N</i>,<i>N</i>-dimethyl group had an ionic interaction with Asp556.</div><div class="NLM_p">CC-90011 (compound <b>31</b>), developed by Celgene, is a reversible LSD1 inhibitor in clinical trials (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> In 2020, Kanouni et al. reported the discovery of CC-90011.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> A high-throughput screening (HTS) of a library containing 300 000 compounds and a structure-based design library was performed in this study, and compound <b>28</b> was finally identified as an attractive hit compound for further SAR exploration. Binding modes predicted that the nitrogen atom in the benzonitrile formed a hydrogen bond interaction with Lys661. Compound <b>28</b> could also form a salt-bridge with pyrrolidine and Asp555. Besides, the chlorine atom projected into a hydrophobic cavity formed by Trp695, Ile356, Leu677, Leu692, and Phe358. Compound <b>29</b> with excellent inhibitory potency against both LSD1 (IC<sub>50</sub> = 3.0 nM) and the cellular differentiation marker CD11b (IC<sub>50</sub> = 28 nM) was later obtained by introducing a lipophilic group <i>p</i>-tolyl and substituting aminopyridine of compound <b>28</b> with pyrrolidine. In addition, adding a fluorine atom to the 3-position of the benzonitrile and replacing the pyrimidine ring with the pyrazinone ring (compound <b>30</b>) further improved the inhibitory activity against LSD1 (IC<sub>50</sub> = 0.5 nM) and CD11b (IC<sub>50</sub> = 18 nM). Pyrimidinone exhibited better activity than pyrazinone, and replacing <i>p</i>-Me with <i>p</i>-OMe reduced the inhibitory activity against the human ether-a-go-go-related gene (hERG) channel. Further adding a fluorine atom to the 3-position of the methoxyphenyl group and 3-position of the benzonitrile respectively provided CC-90011. CC-90011 is a potent LSD1 inhibitor with an IC<sub>50</sub> value of 0.25 nM and can induce the expression of CD11b in human leukemia cell line THP-1 with an EC<sub>50</sub> value of 7 nM. In addition, it showed potent antiproliferative activity against AML cell line kasumi-1 (EC<sub>50</sub> = 2 nM) without any effect on IMR-90 human lung fibroblasts. CC-90011 had no enzymatic inhibition against MAO-A/B and LSD2 even at 10 μM. Furthermore, CC-90011 could induce SCLC cell differentiation. Structure analysis of the complex of CC-90011 with LSD1 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K">6W4K</a>) showed that the compound can form a Y-shaped configuration, while the aminopiperidine ring formed a salt-bridge with Asp555. Meanwhile, the benzonitrile occupied a hydrophobic cavity surrounded by FAD and several amino acid side chains. The nitrile group was found to form a hydrogen bond with Lys661, while the 2-F-anisole moiety formed a hydrophobic interaction in a shallow cavity neighboring the FAD-binding pocket. Until now, a phase I trial of CC-90011 for the treatment of advanced solid tumors and relapsed/refractory non-Hodgkin lymphoma (R/R NHL) has been completed (NCT02875223, EudraCT 2015-005243-13). A phase I/II clinical trial of CC-90011 combined with cisplatin and etoposide to treat extensive-stage SCLC is also ongoing (NCT03850067). Very recently, a phase I study of CC-90011 was initiated by Celgene to test its safety and efficacy for the treatment of advanced cancers, in combination with nivolumab (NCT04350463).</div><div class="NLM_p last">As LSD1 uses FAD as a cofactor,<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55,56)</a> compounds with similar structural features or scaffolds to FAD may compete with FAD for LSD1 binding, presenting a new and promising strategy to develop highly active LSD1 inhibitors with novel chemical structures. In 2020, our group reported a series of 5-cyano-6-phenyl pyrimidine analogues containing the 1,2,3-triazole moiety to be FAD competitive LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> SARs studies identified compound <b>32</b> as the most potent candidate (IC<sub>50</sub> = 183 nM). Compound <b>32</b> was shown to inhibit LSD1 reversibly and time-dependently. Cellular experiments revealed that compound <b>32</b> may promote the increasing in H3K4/9 mono- and dimethylation. Besides, by reversing the epithelial-mesenchymal transition (EMT), compound <b>32</b> inhibited tumor cell migration and invasion. Docking analysis suggested that the triazole ring made a hydrogen bond with Val288, while the cyano group attached to the pyrimidine ring formed hydrogen bond interactions with Lys661 and Met332. The oxygen atom and nitrogen atom of bisamide formed hydrogen bond interactions with Arg316 and Thr624 respectively, while the benzene ring had π–π interactions with Trp751 and Tyr761. In addition, compound <b>32</b> was found to engage in extensive hydrophobic interactions and van der Waals interactions.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.4.  Five-Membered Heterocyclic Compounds</h3><div class="NLM_p">In 2014, Kutz et al. identified a series of 3,5-diamino-1,2,4-triazole analogues as reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A).<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Among them, the two most potent compounds <b>33</b> (IC<sub>50</sub> = 1.19 μM) and <b>34</b> (IC<sub>50</sub> = 2.22 μM) performed high selectivity for LSD1 over MAO-A/B (IC<sub>50</sub> > 200 μM) and caused H3K4me2 accumulation in Calu-6 cells. Compound <b>33</b> was also shown to be a reversible and competitive LSD1 inhibitor. Furthermore, compounds <b>33</b> and <b>34</b> showed low toxicity toward mammalian cells, indicating that the 1,2,4-triazole scaffold may be used to treat epigenetically based diseases. Additional docking data showed that there are several key interactions between LSD1 and compound <b>33</b>, including two hydrogen bonds with Asp555 and Ala539, respectively. Additionally, this compound participated in a π–π interaction with the flavin ring. In 2019, the same group further revealed a novel class of 3,5-diamino-1,2,4-triazole as LSD1 inhibitors.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Among these newly synthesized entities, several representative compounds <b>35</b>–<b>37</b> inhibited LSD1 and SMOX <i>in vitro</i> simultaneously.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Triazole derivatives. (A) Structure and properties of 3,5-diamino-1,2,4-triazole derivatives as LSD1 inhibitors. (B) Structure and properties of triazole-dithiocarbamate derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2013, our group developed several 1,2,3-triazole-dithiocarbamate analogues as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Among these compounds, compound <b>38</b> exhibited promising LSD1 inhibitory activity (IC<sub>50</sub> = 2.1 μM). Both dilution assay and dialysis experiments supported the reversible inhibitory activity of compound <b>38</b>, and Lineweaver–Burk plots characterized compound <b>38</b> as a noncompetitive inhibitor over the LSD1 substrate H3K4me2. Moreover, compound <b>38</b> significantly inhibited cell migration and invasion of human gastric cancer cell lines HGC-27 and MGC-803. Furthermore, compound <b>38</b> effectively inhibited tumor growth <i>in vivo</i> without significant toxicity. Docking studies of compound <b>38</b> indicated that the carbonyl oxygen was favorable to form hydrogen bonds with Val333, Met332, and Ala331. In addition, the triazole ring could also form a hydrogen bond with Arg316. During the molecular dynamics simulations, the two binding modes were well maintained, which was consistent with the experimental results, indicating compound <b>38</b> may be embedded in the LSD1 cavity where FAD stands.</div><div class="NLM_p">Inspired by the strong inhibitory activities of coumarins against MAO-A/B,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> in 2014, our group reported coumarin-1,2,3-triazole-dithiocarbamate hybrids as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Among these compounds, the most active compound <b>39</b> inhibited LSD1 potently (IC<sub>50</sub> = 0.39 μM). Compound <b>39</b> had no inhibitory effects on MAO-A/B, showing excellent selectivity for LSD1. Dilution assay and dialysis experiments indicated that compound <b>39</b> was a reversible LSD1 inhibitor. In addition, compound <b>39</b> upregulated the H3K4me1/2 and H3K9me2 levels in MGC-803 cells.</div><div class="NLM_p">In 2013, Dulla et al. designed and synthesized several 3-amino/guanidine substituted phenyl oxazole derivatives as new LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A).<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Among them, compounds <b>40</b>–<b>42</b> showed promising activities against LSD1 and cancer cells <i>in vitro</i>. Besides, compounds <b>41</b> and <b>42</b> could induce apoptosis in zebrafish embryos. Furthermore, docking studies revealed that all these compounds performed good binding interactions in the LSD1 active sites. Take compound <b>42</b>, for example: its guanidine group made hydrogen bonds with Glu308 and Arg310, while the phenyl group formed a π–π stacking interaction with Arg316.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Other five-membered heterocyclic compounds. (A–E) Structure and properties of oxazole derivatives (A), aminothiazole derivatives (B), pyrazole derivatives (C), imidazole derivatives (D), and pyrrolidine derivatives (E) as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the same year, Hitchin et al. developed two series of aminothiazole-based reversible LSD1 inhibitors with the aid of biochemical fragment-based screening (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The most active compound <b>43</b> showed moderate inhibitory potency against LSD1 (IC<sub>50</sub> = 7.5 μM) and displayed a weak inhibitory effect on MAO-A (IC<sub>50</sub> = 40.3 μM). However, a cellular assay indicated that compound <b>43</b> had no cellular LSD1 inhibitory activity even at 50 μM.</div><div class="NLM_p">A patent in 2015 optimized the structure of GSK-690 and synthesized several LSD1 inhibitors with similar structures.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> This patent disclosed three pyridyl methyl isomers, among which the preferred compound <b>44</b> showed the best LSD1 inhibitory activity <i>in vitro</i>. In 2017, Mold et al. further developed several reversible LSD1 inhibitors bearing a 5-hydroxypyrazole scaffold from compound <b>44</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C).<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Compound <b>44</b> was shown to be an effective and reversible LSD1 inhibitor (IC<sub>50</sub> = 0.23 μM). Docking results of compound <b>44</b> indicated that the nitrile displaced bridging water between FAD and Lys661, and the basic center was directed toward Asp555 and Asp556. In addition, the pyridyl moiety made a hydrogen bond with Asn535. Further optimization of compound <b>44</b> yielded a series of novel potent compounds, of which compound <b>45</b> was the most potent LSD1 inhibitor with an IC<sub>50</sub> value of 79 nM <i>in vitro</i>. Besides, compound <b>45</b> showed a reasonable half-life and moderate oral bioavailability in mice.</div><div class="NLM_p">In 2019, Nie et al. reported imidazole-based reversible LSD1 inhibitors based on the structural features of GSK-690 toward the active cavity of LSD1, resulting in the identification of imidazole analogues (such as compound <b>46</b>) with enhanced potency (IC<sub>50</sub> = 8 nM) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Molecular modeling of compound <b>46</b> indicated that the cyanophenyl moiety closely approached FAD, while the cyano group was predicated to make a hydrogen bond with Lys661. The primary amino group formed polar interactions with Asp555, while the tolyl moiety was embedded into a hydrophobic cavity formed by Phe538, Leu677 and Trp695. Moreover, the carboxamide group limited the free rotation of cyclic amine, making it more chemically accessible to explore the negative electrostatic region. Whereafter, the hERG profile and pharmacokinetic (PK) properties were further optimized by probing the hydrophobic pocket and increasing interactions with polar residues. Upon these optimizations, compound <b>47</b> showed the strongest LSD1 inhibitory potency (IC<sub>50</sub> = 0.7 nM) with reduced hERG inhibition. Furthermore, in THP-1 cells, compound <b>47</b> induced the expression of CD11b with an EC<sub>50</sub> value of 14 nM.</div><div class="NLM_p last">In 2017, Mold et al. developed a series of 4-(pyrrolidin-3-yl)benzonitrile derivatives as novel reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E).<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Among them, the most active compound <b>48</b> exhibited an IC<sub>50</sub> value of 57 nM and a <i>K</i><sub>d</sub> value of 22 nM. In addition, compound <b>48</b> showed good selectivity over MAO-A/B (IC<sub>50</sub> > 25 μM), and higher hERG selectivity than GSK690. Besides, compound <b>48</b> increased the expression of CD86 in THP-1 cells. Molecular docking results of compound <b>48</b> indicated that the nitrile group formed a hydrogen bond with Lys661, while the <i>p</i>-tolyl groups were embedded into a hydrophobic cavity formed by Trp695, Leu693, Leu677, and Ile356.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.5.  Fused Heterocyclic Compounds</h3><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10">2.5.1.  6,6-Fused Heterocyclic Compounds</h4><div class="NLM_p">In 2012, Willmann et al. identified namoline (compound <b>49</b>) as a novel LSD1 inhibitor (IC<sub>50</sub> = 51 μM) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Dilution assay indicated that namoline is a reversible LSD1 inhibitor with no inhibitory effects on MAO-A/B. The false-positive result was excluded by the coupled Amplex Red/HRP assay. As namoline impaired the demethylase activity of LSD1, AR-regulated gene expression was silenced by namoline. Besides, namoline also inhibited androgen-dependent cell proliferation and tumor growth in a xenograft tumor model.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. 6,6-Fused heterocyclic compounds. (A–C) Structure and properties of flavones derivatives (A), quinazoline derivatives (B), and tetrahydroquinoline derivatives (C) as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Baicalin (compound <b>50</b>) is an active ingredient in the skullcap, which possesses various pharmacological properties, including antitumor, antiinflammation, antibacterial, etc. In 2016, our group discovered baicalin as the first flavonoid-based reversible LSD1 inhibitor (IC<sub>50</sub> = 3.01 μM) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Dilution assay revealed that baicalin reversibly inhibited LSD1, while in MGC-803 cells, baicalin increased H3K4me2 and CD86 mRNA levels dose-dependently. Furthermore, baicalin significantly suppressed cell proliferation and migration of MGC-803 cells.</div><div class="NLM_p">In 2018, Han et al. successfully obtained several natural LSD1 inhibitors from <i>S. baicalensis Georgi</i> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> Among them, wogonoside (compound <b>51</b>) showed the best LSD1 inhibitory potency (IC<sub>50</sub> = 2.98 μM). Dilution assay suggested that wogonoside inhibited LSD1 reversibly. Wogonoside could also remarkably reduce the cell viability of MDA-MB-231 cells dose-dependently (IC<sub>50</sub> = 14.94 μM). Docking results of wogonoside revealed that the hydroxyl group of the sugar moiety formed hydrogen bonds with Ala809 and Val811. The 8-position hydroxyl group and the 1-position oxygen atom of the flavonoid moiety were predicted to make hydrogen bond interactions with Arg316, while the 6-position hydroxyl group and 5-position carbonyl group formed three hydrogen bonds with Glu801, Ser289, and Thr624. The benzene ring attached to the basic flavonoid structure may have extensive hydrophobic interactions with Ala814 and Pro626.</div><div class="NLM_p">In 2019, Xu et al. evaluated the LSD1 inhibitory potency of 12 natural flavones (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Compared with aglycones without sugar moiety, flavonoid monoglycosides had greater inhibitory effects on LSD1. Among them, isoquercitrin (compound <b>52</b>) exhibited the most potent LSD1 inhibitory potency (IC<sub>50</sub> = 0.95 μM). Furthermore, isoquercitrin could induce the expression of the key proteins in mitochondrial apoptosis and cause MDA-MB-231 cell apoptosis. The docking results revealed that isoquercitrin could be docked well into the active pocket of LSD1. In the sugar moiety, the hydroxyl group formed two hydrogen bonds with Met332 and Val333. In the flavonoid moiety, the phenol hydroxyl formed extensive hydrogen bond interactions with Leu659, Leu329, Gly330, and Ala809, while the benzene ring established a π–π stacking interaction with Trp751.</div><div class="NLM_p">In 2020, Wang et al. reported biochanin A (compound <b>53</b>) to be a novel LSD1 inhibitor by screening a commercial natural product library containing about 500 compounds (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Biochanin A inhibited LSD1 potently (IC<sub>50</sub> = 2.95 μM) and showed excellent selectivity over MAO-A/B. The false-positive result was excluded by the coupled Amplex Red/HRP assay. Further, dilution assay suggested that biochanin A was a reversible LSD1 inhibitor. Biochanin A could also induce accumulation of H3K4me1/2 in MGC-803 cells and moderately suppress cell growth (IC<sub>50</sub> = 6.77 μM) with less toxicity than that in normal gastric cell line GES-1 (IC<sub>50</sub> > 32 μM). Furthermore, biochanin A was found to suppress colony formation, migration, and induce apoptosis of MGC-803 cells dose-dependently.</div><div class="NLM_p">In 2016, Speranzini et al. reported quinazoline-based compounds as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Among these compounds, compound <b>54</b> was identified as the most effective LSD1 inhibitor (IC<sub>50</sub> = 0.243 μM). Compound <b>54</b> stood in LSD1/CoREST (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3E">5L3E</a>) occupied the same site of LSD1 as polymyxins and formed a pile of five molecules that fully obstructed the entrance to the active cavity. In addition, this complex interacted with negatively charged residues Glu559, Asp555, Asp557, Asp556, Asp553, and Glu387 extensively.</div><div class="NLM_p last">In 2020, Wang et al. designed and synthesized tetrahydroquinoline-based novel reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C).<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Among these compounds, the two most active compounds <b>55</b> and <b>56</b> exhibited excellent LSD1 inhibitory potency with IC<sub>50</sub> values of 50 nM and 540 nM, respectively. Compounds <b>55</b> and <b>56</b> also displayed excellent selectivity over MAO-A/B. Compound <b>55</b> was proven to bind LSD1 reversibly in a noncompetitive manner. Additionally, compounds <b>55</b> and <b>56</b> remarkably inhibited MGC-803 cell proliferation with IC<sub>50</sub> values of 1.13 and 1.15 μM, respectively, as well as induced MGC-803 cell apoptosis. It is noteworthy that oral administration of compound <b>56</b> suppressed the growth of MGC-803 xenograft tumors without apparent side effects. Docking studies of compound <b>55</b> showed that it formed hydrogen bond interactions with Asp555 and Asp556.</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11">2.5.2.  6,5-Fused Heterocyclic Compounds</h4><div class="NLM_p">In 2017, our group reported [1,2,3]triazolo[4,5-<i>d</i>]pyrimidine analogues as new LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A).<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> Compound <b>57</b> showed the strong LSD1 inhibitory activity (IC<sub>50</sub> = 0.564 μM) with moderate selectivity over MAO-A/B. Dilution assay indicated that compound <b>57</b> inhibited LSD1 reversibly. In addition, compound <b>57</b> could significantly inhibit LSD1 in MGC-803 cells as well as suppress the migration of MGC-803 cells. Molecular docking results indicated that the pyridine nitrogen atom of compound <b>57</b> formed a hydrogen bond with Met332, which led to the improved inhibitory activity of the 2-thiopyridine series.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Triazole-fused pyrimidines. (A) Structure and properties of [1,2,3]triazolo[4,5-<i>d</i>]pyrimidine derivatives as LSD1 inhibitors. (B) Structure and properties of [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2019, our group designed and synthesized [1,2,3]triazolo[4,5-<i>d</i>]pyrimidine analogues as novel reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A).<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> The most potent compound <b>58</b> inhibited LSD1 reversibly (IC<sub>50</sub> = 49 nM, <i>K</i><sub>i</sub> = 16 nM) and competitively with H3K4me2, and exhibited high selectivity over MAO-A/B (<10% inhibition at 1 μM). Compared with compound <b>57</b>, the introduction of the electronegative tetrazole ring played a key role in improving the inhibitory activity of compound <b>58</b> against LSD1. Compound <b>58</b> showed marked antiproliferative effects on lymphoma cell line Raji as well as leukemia cell lines OCL-AML3, K562, THP-1, and U937. In THP-1 cells, compound <b>58</b> remarkably suppressed colony formation and induced significant morphological change concentration-dependently, and induced the expression of CD86 and CD11b. Docking analysis indicated that the tetrazole ring formed two hydrogen bonds with Asn535 and Gln358. The benzene ring connected to the tetrazole moiety and formed hydrophobic interaction with Trp695, Leu677, and Ile356. The triazole ring formed an electrostatic interaction and formed a hydrogen bond with His564. Meanwhile, the pyrimidine ring formed a hydrogen bond with Asn535. The 2-Cl phenyl group was embedded into a hydrophobic pocket formed by Tyr761, Trp695, Val333, and Phe538, as well as formed π–π stacking interactions with the flavin ring, Tyr761, Trp695, and Phe538. Additionally, the propargyl group formed hydrophobic interaction with Trp552, Leu536, and Phe382.</div><div class="NLM_p">In 2020, our group developed [1,2,3]triazolo[4,5-<i>d</i>]pyrimidine analogues bearing a (thio)urea moiety as new LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A).<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Among these compounds, compound <b>59</b> exhibited moderate LSD1 inhibitory activity (IC<sub>50</sub> = 9.75 μM) and induced the cellular level of H3K4me2. Furthermore, compound <b>59</b> showed excellent selectivity over MAO-A/B, BTK, and CDK1/2/4/6. Additionally, compound <b>59</b> suppressed the proliferation of several LSD1 overexpressed cancers, including MGC-803, EC109 (human esophageal cancer cell line), PC-3 (human prostate cancer line), and B16-F10 cells (murine melanoma cell line). Docking results revealed that compound <b>59</b> could be docked well into the LSD1 active pocket. The nitrogen atom at the 3-position of the triazole ring and the carbonyl of the pyrimidine ring can form a hydrogen bond with Glu801. A hydrogen-bond formation was also observed between Arg316 and naphthylamine. Additionally, the naphthyl ring occupied a hydrophobic cavity formed by Tyr571, Tyr761, and Trp751.</div><div class="NLM_p">In 2017, our group developed novel [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine-thiosemicarbazide hybrids as reversible LSD1 inhibitors and some of them exhibited potent inhibitory effects on LSD1 and suppressed A549 and PC-9 cells growth selectively (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B).<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Among them, the most active compound <b>60</b> was found to inhibit LSD1 potently (IC<sub>50</sub> = 0.154 μM). Molecular docking resulted predicted that compound <b>60</b> formed hydrogen bond interactions with Ala331, Met332, and Ala539, respectively, and it could also form an arene-H stacking interaction with Val333.</div><div class="NLM_p">In 2019, our group reported a novel series of 4-(<i>N</i>-methylpiperazin-1-yl)aniline substituted [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine analogues as promising LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B).<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> The most active compound <b>61</b> inhibited LSD1 reversibly with moderate activity (IC<sub>50</sub> = 1.72 μM) and exhibited certain selectivity over MAO-A/B. It is worth mentioning that compound <b>61</b> did not inhibit HRP or quench H<sub>2</sub>O<sub>2</sub>, excluding false-positive results. Compared with compound <b>60</b>, replacing the thiosemicarbazide group with a 4-(<i>N</i>-methylpiperazine) aniline group caused the decrease of LSD1 inhibitory activity. Further, a dilution assay revealed that compound <b>61</b> inhibited LSD1 reversibly. Treating LSD1 overexpressed lung cancer cell line A549 with compound <b>61</b> led to an increasing of H3K4me1/2 and H3K9me2 concentration-dependently. In addition, compound <b>61</b> concentration-dependently inhibited A549 cell migration. Docking results revealed that compound <b>61</b> fit well into the hydrophobic cavity and overlapped with the position of FAD in the LSD1 active cavity, indicating that it may be an FAD-competitive LSD1 inhibitor. Besides, hydrogen bonds were formed between the NH attached to the triazolo[1,5-<i>a</i>] pyrimidine moiety and Arg316, the NH of benzimidazole and Glu308, as well as N-Me piperazine and Met4.</div><div class="NLM_p">In 2019, our group designed hydrazine substituted [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine analogues as novel reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B).<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> The most active compound <b>62</b> inhibited LSD1 potently and reversibly (IC<sub>50</sub> = 882.3 nM) with good selectivity over MAO-A/B. Compared with compound <b>60</b>, replacing the thiosemicarbazide group with a hydrazine group resulted in decreased LSD1 inhibitory activity. In addition, compound <b>62</b> induced the accumulation of H3K4me1/2 and H3K9me1/2 dose-dependently in A549 cells and remarkably inhibited cell migration and evasion concentration-dependently. Docking studies of compound <b>62</b> in LSD1/CoRERST showed that the phenyl group of compound <b>62</b> penetrated into a hydrophobic cavity formed by Leu693, Leu677, Trp695, Ile356, and Val333 and established π–π interactions with Trp695 and His564. The bicyclic N-heterocyclic ring of compound <b>62</b> was located very close to the FAD in which the NH attached to the aromatic ring formed a hydrogen bond with the flavin ring. In addition, ArNHNH<sub>2</sub> formed a hydrogen bond with Tyr761, and the N-heterocyclic ring displayed a π–π stacking effect with Phe538.</div><div class="NLM_p">In addition to the aforementioned pyrimidine-fused triazole LSD1 inhibitors, other 6,5-fused heterocyclic-based LSD1 inhibitors have also been developed in recent years. Melatonin is an indole hormone widely presented in plants, animals, and microbes, which is effective in many types of cancers.<a onclick="showRef(event, 'ref80 ref81 ref82 ref83 ref84 ref85 ref86 ref87'); return false;" href="javascript:void(0);" class="ref ref80 ref81 ref82 ref83 ref84 ref85 ref86 ref87">(80−87)</a> In 2017, Yang et al. reported that melatonin (compound <b>63)</b> had potential value for treating oral cancer patient-derived tumor xenograft (PDX) models by inhibiting LSD1 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A).<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> Additionally, melatonin significantly suppressed tumor growth and decreased tumor weight with no associated toxicity.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Other 6,5-fused heterocyclic compounds. (A–C) Structure and properties of indole derivatives (A), xanthine derivatives (B), and 2,3-benzofuran derivatives (C) as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2018, Xi et al. designed and synthesized several 4-(4-benzyloxy)phenoxypiperidines containing an indole moiety as novel reversible LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A).<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> Among them, compound <b>64</b> exhibited potent and reversible LSD1 inhibitory activity (IC<sub>50</sub> = 4 μM). Furthermore, a wound healing assay showed that compound <b>64</b> inhibited the migration of A549 and HCT-116 cells (colon cancer cell line). Further, docking studies showed that compound <b>64</b> adopted a U-shaped conformation (strand-turn-strand) within the catalytic pocket of LSD1 forced by an H-bond interaction with Asp555, and the indole moiety formed a π–π interaction with FAD.</div><div class="NLM_p">In 2019, our group reported the xanthine analogues as novel FAD-based LSD1 inhibitors. Among these compounds, the most active compound <b>65</b> inhibited LSD1 moderately (IC<sub>50</sub> = 6.45 μM) (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B).<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> Molecular docking studies revealed that compound <b>65</b> may locate in a hydrophilic cavity where the SH group formed a hydrogen bond with Gly314, the pyrimidinedione NH formed a hydrogen bond with Thr624, and the imidazole nitrogen atom formed a hydrogen bond with Arg316. In addition, the pyrimidinedione ring formed a hydrogen bond interaction with Gly287. Additionally, compounds <b>66</b> and <b>67</b> (<i>S</i>-benzylated products of compound <b>65</b>) were found to inhibit multiple targets, including bromodomain-containing protein 4 (BRD4), indoleamine 2,3-dioxygenase 1 (IDO1), LSD1, and MAO-A/B. Compounds <b>66</b> and <b>67</b> could also significantly inhibit the growth of MGC-803 cells.</div><div class="NLM_p last">Compound <b>68</b> was previously reported to be an elicitor for plants to combat insects.<a onclick="showRef(event, 'ref91 ref92'); return false;" href="javascript:void(0);" class="ref ref91 ref92">(91,92)</a> In 2020, He et al. reported a series of benzofuran acylhydrazones <b>69</b> via moving the acylhydrazone portion from the 5-position to 2-position on the benzofuran scaffold of compound <b>68</b> as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>c).<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Among these compounds, compounds <b>69a</b>–<b>c</b> are the three most active LSD inhibitors with IC<sub>50</sub> values of 14.4, 7.2, and 14.78 nM, respectively. At the cellular level, compounds <b>69a</b>–<b>c</b> suppressed the proliferation of six cancer cell lines, including PC-3 and MCG-803.</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12">2.5.3.  5,5-Fused Heterocyclic Compounds</h4><div class="NLM_p">In 2017, Sartori et al. performed HTS of a bioactive compound library containing 34 000 compounds to identify novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A).<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> This screening identified 115 hit compounds inhibiting LSD1 at a low micromolar concentration, of which <i>N</i>-phenyl-4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide-based compounds were preferred. Following cocrystal structure analysis of compound <b>70</b> in complex with LSD1/CoREST (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN">5LGN</a>) suggested that compound <b>70</b> was located in the LSD1 active site with an approximately planar conformation due to the electronic delocalization between the two rings bridged by the amide group. The thienopyrrole moiety of compound <b>70</b> was caged among residues Thr814, Tyr765, Leu663, Thr335, and Val333. Besides, the aryl group was located among residues Trp699, Phe542, and Val333. Further preliminary structural optimization resulted in compound <b>71</b> with significantly improved LSD1 inhibitory potency (IC<sub>50</sub> = 0.162 μM). In the same year, based on multiple ligand/LSD1/CoREST cocrystal structures, this group further reported the structure-guided optimization of thieno[3,3-<i>b</i>]pyrrole based potent LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A).<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> Among these compounds, compounds <b>72</b>–<b>74</b> sharing similar scaffolds with compound <b>64</b> showed nanomolar IC<sub>50</sub> values against LSD1 <i>in vitro</i> with excellent selectivity over LSD2 and MAO-A/B. Additionally, in THP-1 cells, compounds <b>73</b> and <b>74</b> inhibited colony formation remarkably. X-ray structure of compound <b>74</b> in complex with LSD1/CoREST (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI">5LHI</a>) confirmed that the additional interaction between the ethyloxymethyl chain and Gln358 was associated with the enhanced potency of disubstituted compounds.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. 5,5-Fused heterocyclic compounds. (A) Structure and properties of thieno[3,2-<i>b</i>]pyrrole-5-carboxamides derivatives as LSD1 inhibitors. (B) Structure and properties of 5-imidazolyl-thieno[3,2-<i>b</i>]pyrroles derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In 2020, the same group further discovered 5-imidazolyl-thieno[3,2-<i>b</i>]pyrroles as novel reversible LSD1 inhibitors.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> Using compound <b>75</b> (IC<sub>50</sub> = 25.7 μM) as a starting compound, the initial SAR exploration resulted in inhibitors with picomolar inhibitory activity <i>in vitro</i> (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B). Further SAR exploration identified compound <b>76</b> as the most potent LSD1 inhibitor with an IC<sub>50</sub> value of 0.1 nM. Moreover, a safe dose of compound <b>76</b> had remarkable <i>in vivo</i> efficacy. The complex of compound <b>77</b> (one of the most potent compounds) adopted a U-shaped conformation in the active cavity of LSD1. The positively charged amino tail formed a salt-bridge with Asp555. Besides, the thienopyrrole moiety formed hydrophobic contacts with Tyr761, Leu659, Thr335, and Val333, as well as a π–π stacking interaction with FAD.</div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">2.6.  Styrene Derivatives</h3><div class="NLM_p">In 2012, Hazeldine et al. reported small-molecule amidoximes analogues as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A).<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> Among them, compound <b>78</b> displayed the best LSD1 inhibitory activity with an IC<sub>50</sub> value of 16.8 μM and induced dramatic changes in methylation at H3K4 in Calu-6 cells (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A). In addition, it increased cellular levels of SFRP2, HCAD and the transcription factor GATA binding protein 4 (GATA4). In 2013, Abdulla et al. reported resveratrol (compound <b>79</b>) as a natural LSD1 inhibitor with an IC<sub>50</sub> value of 10.2 μM (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A), but further SAR studies were not conducted by this group.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> In 2017, our group developed novel resveratrol derivatives containing an amidoxime group as novel potent LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A).<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> Among them, compounds <b>80</b> and <b>81</b> showed the best inhibitory effect against LSD1 with IC<sub>50</sub> values of 0.121 μM and 0.123 μM, respectively. A dilution assay indicated that compounds <b>80</b> and <b>81</b> bound to LSD1 reversibly. Cellular analysis suggested that compounds <b>80</b> and <b>81</b> significantly elevated the levels of H3K4me2 without an impact on LSD1 expression and enhanced the mRNA level of CD86 in MGC-803 cells. Docking experiments indicated that compound <b>80</b> was situated in front of FAD, hindering the approach of the substrate to FAD. The amine group formed a hydrogen bond with Asp555, while the hydroxyl group formed a hydrogen bond with Ser762. The phenyl group attached to amidoxime formed hydrophobic interactions with Pro808, Thr810, Ala809, Ala539, and Trp552. The dioxybenzene occupied a hydrophobic pocket formed by the flavin ring, Trp695, Ala539, Phe538, Met332, Val333, and Tyr761, while the hydroxyl group formed a hydrogen bond with Ala809.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Styrene derivatives. (A) Structure and properties of stilbene-amidoxime hybrids as LSD1 inhibitors. (B) Structure and properties of cinnamic acid derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2018, our group reported stilbene derivatives containing an amidoxime scaffold as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B).<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> The most potent compound <b>82</b> (IC<sub>50</sub> = 0.283 μM) was shown to be a FAD-competitive and reversible LSD1 inhibitor, which was further supported by the docking results. The amine group formed a hydrogen bond with Thr624 and an electrostatic interaction with Glu801, while the imine nitrogen atom formed a hydrogen bond with Glu801. In addition, the phenyl group attached to the pyridine ring occupied a hydrophobic pocket formed by Trp810, Leu659, Lys661, Trp751, Leu329, Tyr761, and Gly330, and formed an arene-H stacking interaction with Trp751. Moreover, the benzene ring attached to the amidoxime also formed an arene-H stacking interaction with Val288. All these interactions made compound <b>82</b> bind tightly to LSD1. In THP-1 cells, compound <b>82</b> was shown to upregulate the expression of CD86. In addition, compound <b>82</b> exhibited excellent antiproliferative effects on two human leukemia cell lines THP-1 and MOLM-13 with IC<sub>50</sub> values of 5.76 μM and 8.34 μM, respectively. Besides, compound <b>82</b> remarkably suppressed colony formation of THP-1 cells concentration-dependently.</div><div class="NLM_p">In 2013, Abdulla et al. first identified curcumin (compound <b>83)</b> as a natural LSD1 inhibitor that can efficiently suppress myogenic expression and differentiation of the mouse myoblast cell line C2C12.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> In 2019, Wang et al. evaluated the inhibitory activity of curcumin against LSD1(IC<sub>50</sub> = 9.6 μM) and developed a series of cinnamamide-based LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B).<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> Among these compounds, compound <b>84</b> displayed the strongest LSD1 inhibitory potency (IC<sub>50</sub> = 0.8 μM) and exhibited excellent selectivity over MAO-A/B (IC<sub>50</sub> > 50 μM). Moreover, compounds <b>84</b> and <b>85</b> showed anticlonogenic effects in A549 cells with an IC<sub>50</sub> value of 9.333 μM and 4.419 μM, respectively. Molecular docking results showed that they bound to the active cavity of LSD1 surrounded by key residues Asp555 and Asp556.</div><div class="NLM_p">In 2018, Xu et al. developed a new class of reversible LSD1 inhibitors bearing a 5-arylidene barbiturate skeleton (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B).<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> The most active compound <b>86</b> was shown to inhibit LSD1 potently (IC<sub>50</sub> = 0.41 μM) and reversibly, as well as displayed high selectivity over MAO-A/B. Besides, compound <b>86</b> induced differentiation of acute promyelocytic leukemia cell line NB4 and remarkably escalated the H3K4 methylation level.</div><div class="NLM_p last">In 2020, Li et al. developed a series of novel chalcone-dithiocarbamate hybrids as potent LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B).<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> Among them, compound <b>88</b> displayed the most potent inhibitory activity against LSD1 (IC<sub>50</sub>= 0.14 μM) in a reversible and time-dependent manner. Furthermore, compound <b>88</b> also induced the accumulation of H3K9me1/2 and inhibited cell proliferation against several leukemia cell lines, including HAL-01, KE-37, P30-OHK, SUP-B15, MOLT-4, and LC4–1 cells. Besides, compound <b>88</b> could significantly suppress MOLT-4 xenograft tumor growth. Docking studies showed that compound <b>88</b> may be located in a similar pocket as compound <b>54</b> and forms hydrogen bond interactions with Phe538, Phe 382, and Leu386.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">2.7.  3-Oxoamino-benzenesulfonamide Derivatives</h3><div class="NLM_p">In 2013, Sorna et al. carried out a structure-based virtual screening of a library containing about 2 million compounds resulted in a series of <i>N</i>′-(1-phenylethylidene)-benzohydrazide-based LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>).<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> Among them, compound <b>89</b> bearing <i>N</i>′-(1-phenylethylidene)-benzohydrazide core had the highest LSD1 inhibitory activity (IC<sub>50</sub> = 291 nM). Further, adding a sulfonyl group to the arylhydrazide moiety resulted in compound <b>90</b> with a significant increase in LSD1 inhibitory activity (IC<sub>50</sub> = 19 nM). Docking results showed the protonated morpholine nitrogen atom of compound <b>90</b> formed an ionic interaction, and its 2-hydroxyphenyl moiety was inserted more deeply into the pocket compared to compound <b>89</b>. The follow-up hit-to-lead optimization and SAR study finally identified SP-2577 (compound <b>91</b>, also known as seclidemstat) as the optimal candidate compound (<i>K</i><sub>i</sub> = 31 nM, IC<sub>50</sub> = 13 nM) (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). In addition, SP-2577 had no inhibitory effects on MAO-A/B. Furthermore, SP-2577 exhibited minimal inhibitory activity against cytochrome P450 monooxygenases (CYPs) and hERG, as well as suppressed the proliferation of several cancer cells, such as colorectal and breast cancer. Besides, SP-2577 could promote antitumor immunity in switch/sucrose nonfermentable (SWI/SNF) complex mutated ovarian cancer,<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> as well as inhibit virus production, viral DNA replication, and late gene expression.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> SP-2577 could also offer more efficacy for patients with Ewing sarcoma (ES). Hence, SP-2577 was promoted to further clinical studies. A phase I/II clinical study of SP-2577 in subjects with relapsed or refractory ES has been conducted (NCT03600649),<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> and a phase I clinical trial was also initiated to treat advanced solid tumors (NCT03895684).</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. 3-Oxoamino-benzenesulfonamide derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2015, Zhou et al. characterized a series of (<i>E</i>)-<i>N</i>′-(2, 3-dihydro-1<i>H</i>-inden-1-ylidene) benzohydrazide derivatives as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>).<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> Among them, compounds <b>92</b> and <b>93</b> exhibited the strongest inhibitory potency against LSD1 with IC<sub>50</sub> values of 1.4 and 1.7 nM, respectively. Additionally, compounds <b>92</b> and <b>93</b> exhibited significant antiproliferative activities against LSD1 overexpressed human cancer cells, including lung cancer, hepatocellular carcinoma, ovarian carcinoma, colorectal cancer, breast cancer, and prostate cancer cells. The binding mode of compound <b>93</b> in complex with LSD1 revealed that the hydroxyl group formed hydrogen bond interactions with Gly314 and Val590, while the acyl group formed a hydrogen bond with Arg310.</div><div class="NLM_p">In 2017, Xi et al. reported a series of 3-oxoamino-benzenesulfonamide derivatives as potential LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>).<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> Compounds <b>94</b> and <b>95</b> exhibited the most inhibitory potency and selectivity against LSD1 with IC<sub>50</sub> values of 9.5 and 6.9 μM, respectively. Docking studies of compounds <b>94</b> and <b>95</b> showed that they can interact with key residues Trp756, Thr624, Arg316, and Ala309 via weak hydrogen bond interactions.</div><div class="NLM_p last">In 2016, Zha et al. identified <i>N</i>-(3-substituted-phenyl) benzenesulfonamides as novel reversible LSD1 inhibitors with IC<sub>50</sub> values ranging between 7.5 and 68 μM (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>).<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> Among these compounds, the three most active compounds <b>96</b>–<b>98</b> showed excellent selectivity over MAO-A/B. Furthermore, molecular docking revealed that compound <b>96</b> completely occupied the FAD binding region in LSD1. The sulfonamide oxygen atom formed a hydrogen bond with Arg316, while the oxygen atom connected to the sulfur atom formed a hydrogen bond with Ser289.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">2.8.  Metal Complexes</h3><div class="NLM_p">In 2017, Yang et al. reported a rhodium(III) complex <b>99</b> as the first metal-based LSD1 inhibitor (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>A) that may occupy part of the LSD1 active site, interacting with the residues around Asp556, which was illustrated by molecular docking.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> Complex <b>99</b> exhibited excellent LSD1 inhibitory activity and selectivity (<i>K</i><sub>i</sub> = 0.57 μM, IC<sub>50</sub> = 0.04 μM). At the cellular level, complex <b>99</b> showed antiproliferative activity against PC-3 cells (IC<sub>50</sub> = 2.83 μM). A mechanism study suggested that complex <b>99</b> disrupted the interaction between LSD1 and H3K4me2 in PC-3 cells, induced the activation of LSD1 regulated gene promoters, including protein p21, forkhead box A2 (FOXA2), and bone morphogenetic protein 2 (BMP2). Besides, complex <b>99</b> decreased glucose transporter 1 (GLUT1) expression without affecting the expression of LSD1 in PC-3 cells.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Metal complexes. (A) Structure and properties of rhodium(III) complex as LSD1 inhibitor. (B) Structure and properties of vanadium complexes as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In the same year, Lu et al. identified a small-size block of Schiff base vanadium complexes as novel LSD1 inhibitors (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>B).<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Among these synthesized complexes, the most active complex <b>100</b> exhibited promising inhibitory activity against LSD1 (IC<sub>50</sub> = 19.0 μM). Besides, complex <b>100</b> exhibited very weak inhibitory activities against MAO-A/B, representing excellent selectivity for LSD1.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">2.9.  Other Reversible LSD1 Inhibitors</h3><div class="NLM_p">In 2017, Mold reported several (4-cyanophenyl)-glycine-based LSD1 inhibitors from an HTS and <i>in silico</i> molecular modeling approaches (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>).<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Initial optimization from a weakly hit compound <b>101</b> afforded the 4-aminopiperidine derivative <b>102</b>. Whereafter, the sulfonyl group was replaced with methylene to obtain a (4-cyanophenyl) glycinamide skeleton, which increased cell membrane permeability and bioavailability. Introducing a fluorine atom and incorporating a spirocyclic scaffold provided the most potent compound <b>103</b> with an IC<sub>50</sub> value of 83 nM against LSD1. Besides, compound <b>103</b> shared a <i>K</i><sub>d</sub> value of 32 nM against LSD1 and an EC<sub>50</sub> value of 0.67 μM in the CD86 assay.</div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Other reversible LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In 2020, He et al. developed a series of dual inhibitors with an <i>exo</i>-5,6-bis(4-hydroxyphenyl)-7-oxabicyclo[2.2.1]hept-5-ene-2-sulfonic acid phenyl ester pharmacophore scaffold targeting both estrogen receptor α (ERα) and LSD1 (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>).<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> Among them, compound <b>104</b> showed strong ERα binding affinity (relative binding affinity (RBA) = 9.67%, RBA value of E2 as 100%), and high selectivity (α/β = 7.11). Meanwhile, compound <b>104</b> exhibited the highest inhibitory activity against LSD1 (IC<sub>50</sub> = 1.55 μM) and MCF-7 cells (IC<sub>50</sub> = 8.79 μM). Molecular docking results of compound <b>104</b> with LSD1/CoREST complex showed that the oxygen atom of the phenolic hydroxyl group formed hydrogen bonds with Try571 and His812, respectively, while the pyridine ring of compound <b>104</b> formed a π–π stacking interaction with His564.</div><div class="NLM_p">In 2018, Han et al. identified α-mangostin (compound <b>105</b>) as the first xanthone-based LSD1 inhibitor with promising LSD1 inhibitory potency (IC<sub>50</sub> = 2.81 μM) (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>) in a reversible manner.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> In MDA-MB-231 cells, α-mangostin induced a dramatic increase in the accumulation of H3K4me2 dose-dependently, while the LSD1 expression was not affected. In addition, α-mangostin caused a remarkable increment in CD86 expression and suppressed MDA-MB-231 cell migration concentration-dependently. Furthermore, α-mangostin increased the expression of E-cadherin and decreased the expression of N-cadherin. Molecular docking suggested that α-mangostin could be docked well into the active pocket of LSD1 as the <i>meta</i>-position hydroxyl group of the xanthone scaffold formed hydrogen bond interactions with Arg316 and Thr810. The phenyl group formed extensive π–π stacking interactions with Arg316, Val811, Ala331, Glu801, and Tyr761. Moreover, two isopentane groups formed extensive hydrophobic interactions with Ala814, Trp751, and Leu659.</div><div class="NLM_p">In 2020, our group first discovered protoberberine alkaloids as natural LSD1 inhibitors by screening our in-house compound library (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>).<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> Among them, epiberberine (compound <b>106</b>) displayed the most potent LSD1 inhibitory activity (IC<sub>50</sub> = 0.14 μM) in a reversible manner and high selectivity over MAO-A/B. SAR indicated that the isoquinoline-based tetracyclic scaffold is the active skeleton, and variation of the substituents connected to the core moiety caused significant changes in activity. Besides, epiberberine inhibited cell differentiation of AML cell lines THP-1 and HL-60. Furthermore, epiberberine could prolong the survival time of THP-1 cells in mice and suppress AML cell growth without significant toxicity.</div><div class="NLM_p last">In the same year, Jia et al. reported capsaicin (compound <b>107</b>) as a novel potent LSD1 inhibitor (IC<sub>50</sub> = 0.6 μM) by screening a library of 1971 compounds (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>).<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> Capsaicin showed very weak inhibitory effects on MAO-A/B, lysine demethylase 5A (KDM5A), and lysine demethylase 5B (KDM5B) even at 10 μM. Lineweaver–Burk plot analysis and a dilution assay identified capsaicin as an FAD-competitive and reversible LSD1 inhibitor. Capsaicin could also inhibit BGC-823 cell migration and invasion by reversing EMT. Besides, capsaicin significantly induced the expression of an epithelial cell marker cadherin-1 and suppressed the expression of the two mesenchymal cell markers N-cadherin and vimentin. It was also reported to inhibit LSD1 and induce the accumulation of H3K4me1/2 in BGC-823 cells. Further docking analysis revealed that capsaicin may occupy the FAD-binding sites and overlap with the position of FAD in LSD1 active cavity, confirming capsaicin as a FAD-competitive LSD1 inhibitor.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">3.  Summary and Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45088" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45088" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The biological functions of LSD1 have been widely investigated since its discovery in 2004. LSD1 is involved in multiple biological processes, and its dysfunction may contribute to various cancers, such as AML and SCLC. Abrogation of LSD1 genetically or pharmacologically has been proven to suppress tumor cell proliferation, invasion, and migration. Thus, LSD1 has been considered as a promising epigenetic target for anticancer treatment.</div><div class="NLM_p">In the past decade, numerous LSD1 inhibitors with two chemotypes, irreversible inhibitors and reversible inhibitors, have been reported. Most irreversible LSD1 inhibitors share TCP, propargylamine, or hydrazine structure as pharmacophores. Mechanically, irreversible LSD1 inhibitors interact with FAD to form a covalent bond to inhibit LSD1 in an irreversible manner.<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> To date, six TCP-based LSD1 inhibitors, including TCP, GSK2879552, IMG-7289, ORY-1001, INCB059872, and ORY-2001 have entered clinical trials for the treatment of AML and SCLC, etc.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> Although irreversible inhibitors exert long-lasting effects on the target of interest, their potential off-target reactivity and other side effects are of great concern. As a result, in recent years, many reversible LSD1 inhibitors have been reported, and most of them were discovered by adopting the basic drug discovery strategies such as HTS and hit-to-lead optimization. Up to now, only two reversible LSD1 inhibitors (CC-90011 and SP-2577) have entered the clinical trials. Reversible LSD1 inhibitors are preferred in terms of clinical utility since they may provide a safer metabolic profile. The phenotypes of reversible and irreversible LSD1 inhibitors have not been fully established clinically to date, and more studies are required to explore their potential differences and side effects.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">Natural products are a rich source with large-scale chemical diversities, diverse biological activities, and relatively mild toxicity for drug discovery. Several natural products, including cyclic peptides, flavones, melatonin, baicalin, resveratrol, curcumin, α-mangostin, and epiberberine have been found with potent LSD1 inhibitory activity.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Although most natural compounds have been reported to exhibit moderate-to-poor inhibitory activity against LSD1, the abundance of natural products with diverse skeletons will provide a promising prospect for developing novel potent reversible LSD1 inhibitors. Additionally, the discovery of rhodium(III)- and vanadium-based LSD1 inhibitors indicates that metal-based complexes can be used to develop more potent LSD1 inhibitors.</div><div class="NLM_p last">Molecular hybridization, which covalently combines different drug pharmacophoric moieties, could produce novel hybrid compounds with improved affinity and efficacy. In addition, these hybrids may exhibit different and/or synergistic effects, increased selectivity, and reduced adverse effects. Molecular hybridization has also been used as an important tool for designing new prototypes of innovative drugs [104].<a onclick="showRef(event, 'ref119 ref120'); return false;" href="javascript:void(0);" class="ref ref119 ref120">(119,120)</a> As mentioned in the main text, our group has successfully applied molecular hybridization strategies to obtain many highly active and selective pyrimidine-triazole hybrid compounds.<a onclick="showRef(event, 'ref56 ref73 ref74 ref75 ref76 ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref56 ref73 ref74 ref75 ref76 ref77 ref78">(56,73−78)</a> Moreover, the development of dual inhibitors and the combination strategy of LSD1 inhibitors with other therapeutics may have great potential for cancer therapy.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02176" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86165" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86165" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi-Chao Zheng</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2662-3770" title="Orcid link">http://orcid.org/0000-0002-2662-3770</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#156c7c767d747a6f7d707b72556f6f603b7071603b767b"><span class="__cf_email__" data-cfemail="aad3c3c9c2cbc5d0c2cfc4cdead0d0df84cfcedf84c9c4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong-Min Liu</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6771-9421" title="Orcid link">http://orcid.org/0000-0001-6771-9421</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9bf7f2eef3f6dbe1e1eeb5feffeeb5f8f5"><span class="__cf_email__" data-cfemail="701c1905181d300a0a055e1514055e131e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xing-Jie Dai</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Liu</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei-Peng Xue</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao-Peng Xiong</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Zhou</span> - <span class="hlFld-Affiliation affiliation">Key Lab of Advanced Drug
Preparation Technologies, Ministry of Education of China, State Key
Laboratory of Esophageal Cancer Prevention & Treatment, Key Laboratory
of Henan Province for Drug Quality and Evaluation, Institute of Drug
Discovery and Development, School of Pharmaceutical Sciences, Zhengzhou University, 100 Kexue Avenue, Zhengzhou, Henan 450001, China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81276" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81276" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Xing-Jie Dai</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2333-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Xing-Jie Dai obtained his M.S. degree in medicinal chemistry from Zhengzhou University in 2016 under the supervision of Prof. Jingchao Tao and his Ph.D. degree in Organic Chemistry from University of Chinese Academy of Science in 2019 under the supervision of Prof. Xiao-Mei Zhang. In 2019, He joined the group of Prof. Hong-Min Liu as a postdoctoral researcher in the School of Pharmaceutical Sciences at Zhengzhou University. His research is dedicated to the design, synthesis, and evaluation of novel small-molecule epigenetic inhibitors.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Ying Liu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2336-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Ying Liu obtained his B.S. degree in Pharmaceutical Science from Henan University in 2014 and his M.S in pharmaceutical chemistry from Zhengzhou University in 2017 under the supervision of Prof. Yu Ke. He is currently doing his fourth year Ph.D. studies at Zhengzhou University under the supervision of Prof. Dequan Yu and Prof. Hong-Min Liu. Now he works on the design, synthesis, and biological evaluation of natural product extracts used as anticancer agents.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Lei-Peng Xue</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2339-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Lei-Peng Xue obtained his B.S. degree in pharmacy at Nanchang University in 2019. He is currently undergoing Master’s studies in medicinal chemistry at Zhengzhou University.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Xiao-Peng Xiong</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2342-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Xiao-Peng Xiong obtained his B.S degree in Clinical Pharmacy at Jilin University in 2019. He is currently studying for a Master’s degree in medicinal chemistry at Zhengzhou University under the direction of Associate Prof. Yichao Zheng.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Ying Zhou</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2345-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Ying Zhou obtained her B.S degree in Pharmacy at Wannan Medical College in 2020. She is currently studying for a Master’s degree in medicinal chemistry at Zhengzhou University under the direction of Associate Prof. Yichao Zheng.</p></figure></div><div class="bio" rid="ath6"><figure id="ath6" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Yi-Chao Zheng</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2348-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Yi-Chao Zheng got his B.S. degree in Pharmaceutical Science from China Pharmaceutical University in 2008, his M.S in Pharmacy from Katholieke Universiteit Leuven in 2010, and his Ph.D. in Medicinal Chemistry from Zhengzhou University in 2014. He is currently an associate professor in the School of Pharmaceutical Sciences at Zhengzhou University. His current research interest resides in the identification of epigenetic targeted drug discovery and their biological study.</p></figure></div><div class="bio" rid="ath7"><figure id="ath7" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Hong-Min Liu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=BIO-d7e2351-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last">Hong-Min Liu received his M.S. in Pharmaceutical Sciences from Department of Pharmaceutical Sciences, Kanazawa University, Japan, and his Ph.D. in Division of Life Sciences, Department of Bioactive and Related Substances Chemistry, Kanazawa University, Japan. He was appointed as a professor in 1995 and has been the Dean of Key Lab of Advanced Drug Preparation Technologies, Ministry of Education of China, at Zhengzhou University since 2005. His work focuses on the identification of epigenetic targeted drug discovery.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38270" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38270" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study was supported by National Key Research Program of Proteins (Nos. 2018YFE0195100 and 2016YFA0501800 for H.-M.L.); National Natural Science Foundation of China (No. 81602961 for Y.-C.Z., Nos. 81430085, 82020108030, and 81773562 for H.-M.L.); and Science and Technology Innovation Talents of Henan Provincial Education Department (No. 19IRTSTHN001 for Y.-C.Z.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbeviations Used</td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">BLI</td><td class="NLM_def"><p class="first last">biolayer interferometry</p></td></tr><tr><td class="NLM_term">BMP2</td><td class="NLM_def"><p class="first last">bone morphogenetic protein 2</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">CCC</td><td class="NLM_def"><p class="first last">countercurrent chromatography</p></td></tr><tr><td class="NLM_term">CoREST</td><td class="NLM_def"><p class="first last">RE1-silencing transcription factor corepressor</p></td></tr><tr><td class="NLM_term">CR</td><td class="NLM_def"><p class="first last">complete response</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancer cells</p></td></tr><tr><td class="NLM_term">CYPs</td><td class="NLM_def"><p class="first last">cytochrome P450 monooxygenases</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA methyltransferase 1</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">EMT</td><td class="NLM_def"><p class="first last">epithelial-mesenchymal transition</p></td></tr><tr><td class="NLM_term">Erα</td><td class="NLM_def"><p class="first last">estrogen receptor α</p></td></tr><tr><td class="NLM_term">ES</td><td class="NLM_def"><p class="first last">Ewing sarcoma</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">FOXA2</td><td class="NLM_def"><p class="first last">forkhead box A2</p></td></tr><tr><td class="NLM_term">GATA4</td><td class="NLM_def"><p class="first last">GATA binding protein 4</p></td></tr><tr><td class="NLM_term">GATA5</td><td class="NLM_def"><p class="first last">GATA binding protein 5</p></td></tr><tr><td class="NLM_term">GLUT1</td><td class="NLM_def"><p class="first last">glucose transporter 1</p></td></tr><tr><td class="NLM_term">HCAD</td><td class="NLM_def"><p class="first last">H-cadherin</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go-related gene</p></td></tr><tr><td class="NLM_term">HRP</td><td class="NLM_def"><p class="first last">horseradish peroxidase</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">IDO1</td><td class="NLM_def"><p class="first last">indoleamine 2,3-dioxygenase 1</p></td></tr><tr><td class="NLM_term">KDM5A</td><td class="NLM_def"><p class="first last">lysine demethylase 5A</p></td></tr><tr><td class="NLM_term">KDM5B</td><td class="NLM_def"><p class="first last">lysine demethylase 5B</p></td></tr><tr><td class="NLM_term">LSD1/KDM1A</td><td class="NLM_def"><p class="first last">histone lysine specific demethylase 1</p></td></tr><tr><td class="NLM_term">LSD2/KDM1B</td><td class="NLM_def"><p class="first last">lysine specific demethylase 2</p></td></tr><tr><td class="NLM_term">MAOs</td><td class="NLM_def"><p class="first last">monoamine oxidases</p></td></tr><tr><td class="NLM_term">NENs</td><td class="NLM_def"><p class="first last">neuroendocrine neoplasms</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">nonsmall cell lung cancer</p></td></tr><tr><td class="NLM_term">PDX</td><td class="NLM_def"><p class="first last">patient-derived xenograft</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">RBA</td><td class="NLM_def"><p class="first last">relative binding affinity</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SCLC</td><td class="NLM_def"><p class="first last">small cell lung cancer</p></td></tr><tr><td class="NLM_term">SFRP2</td><td class="NLM_def"><p class="first last">secreted frizzled-related protein 2</p></td></tr><tr><td class="NLM_term">SFRP4/5</td><td class="NLM_def"><p class="first last">secreted frizzled-related protein 4/5</p></td></tr><tr><td class="NLM_term">SMOX</td><td class="NLM_def"><p class="first last">spermine oxidase</p></td></tr><tr><td class="NLM_term">Snail1</td><td class="NLM_def"><p class="first last">Snail family transcriptional repressor 1</p></td></tr><tr><td class="NLM_term">SWI/SNF</td><td class="NLM_def"><p class="first last">switch/sucrose nonfermentable</p></td></tr><tr><td class="NLM_term">TCP</td><td class="NLM_def"><p class="first last">tranylcypromine</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 120 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liG1usQFLx87w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">Advances toward LSD1 inhibitors for cancer therapy</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1242</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.4155%2Ffmc-2017-0068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28722477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1227-1242&author=X.+L.+Fuauthor=P.+Zhangauthor=B.+Yu&title=Advances+toward+LSD1+inhibitors+for+cancer+therapy&doi=10.4155%2Ffmc-2017-0068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Advances toward LSD1 inhibitors for cancer therapy</span></div><div class="casAuthors">Fu, Xiaoli; Zhang, Peng; Yu, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1227-1242</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  LSD1 has become an important biol. validated epigenetic target for cancer therapy since its identification in 2004.  LSD1 mediates many cellular signaling pathways and is involved in the initiation and development of cancers.  Aberrant overexpression of LSD1 has been obsd. in different types of cancers, and inactivation by small mols. suppresses cancer cell differentiation, proliferation, invasion and migration.  To date, a large no. of LSD1 inhibitors have been reported, RG6016, GSK-2879552, INCB059872, IMG-7289 and CC-90011 are currently undergoing clin. assessment for the treatment of acute myeloid leukemia, small-cell lung cancer, etc.  In this review, we briefly highlight recent advances of LSD1 inhibitors mainly covering the literatures from 2015 to 2017 and tentatively propose our perspectives on the design of new LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlJAzt51S4crVg90H21EOLACvtfcHk0lgHRfyx4NL0-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK&md5=182f10eb7a2aea7b5c13f50f84f44c20</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DAdvances%2520toward%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1227%26epage%3D1242%26doi%3D10.4155%2Ffmc-2017-0068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stavropoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelz, A.</span></span> <span> </span><span class="NLM_article-title">Crystal structure and mechanism of human lysine-specific demethylase-1</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">632</span>, <span class="refDoi"> DOI: 10.1038/nsmb1113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnsmb1113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=16799558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=626-632&author=P.+Stavropoulosauthor=G.+Blobelauthor=A.+Hoelz&title=Crystal+structure+and+mechanism+of+human+lysine-specific+demethylase-1&doi=10.1038%2Fnsmb1113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and mechanism of human lysine-specific demethylase-1</span></div><div class="casAuthors">Stavropoulos, Pete; Blobel, Guenter; Hoelz, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">626-632</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The reversible methylation of specific Lys residues in histone tails is crucial in epigenetic gene regulation.  LSD1, the 1st known lysine-specific demethylase, selectively removes mono-Me and di-Me, but not tri-Me modifications of Lys-4 or Lys-9 in histone-3.  Here, the authors present the crystal structure of LSD1 at 2.9 Å resoln.  LSD1 was found to form a highly asym., closely packed domain structure from which a long helical 'tower' domain protruded.  The active site cavity was spacious enough to accommodate several residues of the histone tail substrate, but did not appear capable of recognizing the different methylation states of the substrate, lysine.  This supported the hypothesis that trimethylated lysine was chem. rather than sterically discriminated.  The authors presented a biochem. anal. of LSD1 mutants that identified crucial residues in the active site cavity and showed the importance of the SWIRM and tower domains for catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG8Vp2AlsWQLVg90H21EOLACvtfcHk0lgHRfyx4NL0-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D&md5=34cbc6f0936b642a8668d1d76abd1e0e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1113%26sid%3Dliteratum%253Aachs%26aulast%3DStavropoulos%26aufirst%3DP.%26aulast%3DBlobel%26aufirst%3DG.%26aulast%3DHoelz%26aufirst%3DA.%26atitle%3DCrystal%2520structure%2520and%2520mechanism%2520of%2520human%2520lysine-specific%2520demethylase-1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D626%26epage%3D632%26doi%3D10.1038%2Fnsmb1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yennawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the LSD1/CoREST histone demethylase bound to its nucleosome substrate</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">914</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.molcel.2020.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32396821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptlegsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=903-914&author=S.+A.+Kimauthor=J.+Zhuauthor=N.+Yennawarauthor=P.+Eekauthor=S.+Tan&title=Crystal+structure+of+the+LSD1%2FCoREST+histone+demethylase+bound+to+its+nucleosome+substrate&doi=10.1016%2Fj.molcel.2020.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the LSD1/CoREST Histone Demethylase Bound to Its Nucleosome Substrate</span></div><div class="casAuthors">Kim, Sang-Ah; Zhu, Jiang; Yennawar, Neela; Eek, Priit; Tan, Song</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">903-914.e4</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">LSD1 (lysine specific demethylase; also known as KDM1A), the first histone demethylase discovered, regulates cell-fate detn. and is overexpressed in multiple cancers.  LSD1 demethylates histone H3 Lys4, an epigenetic mark for active genes, but requires the CoREST repressor to act on nucleosome substrates.  To understand how an accessory subunit (CoREST) enables a chromatin enzyme (LSD1) to function on a nucleosome and not just histones, we have detd. the crystal structure of the LSD1/CoREST complex bound to a 191-bp nucleosome.  We find that the LSD1 catalytic domain binds extranucleosomal DNA and is unexpectedly positioned 100 Å away from the nucleosome core.  CoREST makes crit. contacts with both histone and DNA components of the nucleosome, explaining its essential function in demethylating nucleosome substrates.  Our studies also show that the LSD1(K661A) frequently used as a catalytically inactive mutant in vivo (based on in vitro peptide studies) actually retains substantial H3K4 demethylase activity on nucleosome substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH-RwffJwsB7Vg90H21EOLACvtfcHk0lgHRfyx4NL0-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptlegsr0%253D&md5=6604db71a7ad1eb0ca25ecddd96ca8f5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BA.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DYennawar%26aufirst%3DN.%26aulast%3DEek%26aufirst%3DP.%26aulast%3DTan%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520LSD1%252FCoREST%2520histone%2520demethylase%2520bound%2520to%2520its%2520nucleosome%2520substrate%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D903%26epage%3D914%26doi%3D10.1016%2Fj.molcel.2020.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span> <span> </span><span class="NLM_article-title">Development and crystallographic evaluation of histone H3 peptide with N-terminal serine substitution as a potent inhibitor of lysine-specific demethylase 1</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2617</span>– <span class="NLM_lpage">2624</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2017.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28336409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVSltbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2617-2624&author=Y.+Amanoauthor=M.+Kikuchiauthor=S.+Satoauthor=S.+Yokoyamaauthor=T.+Umeharaauthor=N.+Umezawaauthor=T.+Higuchi&title=Development+and+crystallographic+evaluation+of+histone+H3+peptide+with+N-terminal+serine+substitution+as+a+potent+inhibitor+of+lysine-specific+demethylase+1&doi=10.1016%2Fj.bmc.2017.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Development and crystallographic evaluation of histone H3 peptide with N-terminal serine substitution as a potent inhibitor of lysine-specific demethylase 1</span></div><div class="casAuthors">Amano, Yuichi; Kikuchi, Masaki; Sato, Shin; Yokoyama, Shigeyuki; Umehara, Takashi; Umezawa, Naoki; Higuchi, Tsunehiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2617-2624</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1A) is a flavoenzyme demethylase, which removes mono- and di-Me groups from histone H3 Lys4 (H3K4) or Lys9 (H3K9) in complexes with several nuclear proteins.  Since LSD1 is implicated in the tumorigenesis and progression of various cancers, LSD1-specific inhibitors are considered as potential anti-cancer agents.  A modified H3 peptide with substitution of Lys4 to Met [H3K4M] is already known to be a potent competitive inhibitor of LSD1.  In this study, we synthesized a series of H3K4M peptide derivs. and evaluated their LSD1-inhibitory activities in vitro.  We found that substitutions of the N-terminal amino acid with amino acids having a larger side chain were generally not tolerated, but substitution of Ala1 to Ser unexpectedly resulted in more potent inhibitory activity toward LSD1.  X-ray crystallog. anal. of H3K4M derivs. bound to the LSD1·CoREST complex revealed the presence of addnl. hydrogen bonding between the N-terminal Ser residue of the H3 peptide deriv. and LSD1.  The present structural and biochem. findings will be helpful for obtaining more potent peptidic inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSLYWQH97b27Vg90H21EOLACvtfcHk0ljnkiPJv8Ym8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVSltbc%253D&md5=c988cd6c93b3c0dc4bd371374cd2d2df</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DAmano%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DHiguchi%26aufirst%3DT.%26atitle%3DDevelopment%2520and%2520crystallographic%2520evaluation%2520of%2520histone%2520H3%2520peptide%2520with%2520N-terminal%2520serine%2520substitution%2520as%2520a%2520potent%2520inhibitor%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2617%26epage%3D2624%26doi%3D10.1016%2Fj.bmc.2017.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+J.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljnkiPJv8Ym8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BJ.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, A. H. F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span> <span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>437</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0ljnkiPJv8Ym8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray-Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The re-expression of the epigenetically silenced e-cadherin gene by a polyamine analogue lysine-specific demethylase-1 (LSD1) inhibitor in human acute myeloid leukemia cell lines</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">594</span>, <span class="refDoi"> DOI: 10.1007/s00726-013-1485-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00726-013-1485-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23508577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1Srt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=585-594&author=T.+Murray-Stewartauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=The+re-expression+of+the+epigenetically+silenced+e-cadherin+gene+by+a+polyamine+analogue+lysine-specific+demethylase-1+%28LSD1%29+inhibitor+in+human+acute+myeloid+leukemia+cell+lines&doi=10.1007%2Fs00726-013-1485-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The re-expression of the epigenetically silenced e-cadherin gene by a polyamine analogue lysine-specific demethylase-1 (LSD1) inhibitor in human acute myeloid leukemia cell lines</span></div><div class="casAuthors">Murray-Stewart, Tracy; Woster, Patrick M.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">585-594</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag GmbH</span>)
        </div><div class="casAbstract">Aberrant epigenetic silencing of tumor suppressor genes is a common feature obsd. during the transformation process of many cancers, including those of hematol. origin.  Histone modifications, including acetylation, phosphorylation, and methylation, collaborate with DNA CpG island methylation to regulate gene expression.  The dynamic process of histone methylation is the latest of these epigenetic modifications to be described, and the identification and characterization of LSD1 as a demethylase of lysine 4 of histone H3 (H3K4) has confirmed that both the enzyme and the modified histone play important roles as regulators of gene expression.  LSD1 activity contributes to the suppression of gene expression by demethylating promoter-region mono- and dimethyl-H3K4 histone marks that are assocd. with active gene expression.  As most post-translational modifications are reversible, the enzymes involved in the modification of histones have become targets for chemotherapeutic intervention.  In this study, we examd. the effects of the polyamine analog LSD1 inhibitor 2d (1,15-bis{N5-[3,3-(diphenyl)propyl]-N1-biguanido}-4,12-diazapentadecane) in human acute myeloid leukemia (AML) cell lines.  In each line studied, 2d evoked cytotoxicity and inhibited LSD1 activity, as evidenced by increases in the global levels of mono- and di-methylated H3K4 proteins.  Global increases in other chromatin modifications were also obsd. following exposure to 2d, suggesting a broad response to this compd. with respect to chromatin regulation.  On a gene-specific level, treatment with 2d resulted in the re-expression of e-cadherin, a tumor suppressor gene frequently silenced by epigenetic modification in AML.  Quant. chromatin immunopptn. anal. of the e-cadherin promoter further confirmed that this re-expression was concurrent with changes in both active and repressive histone marks that were consistent with LSD1 inhibition.  As hematol. malignancies have demonstrated promising clin. responses to agents targeting epigenetic silencing, this polyamine analog LSD1 inhibitor presents an exciting new avenue for the development of novel therapeutic agents for the treatment of AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpENhX8f3UG3bVg90H21EOLACvtfcHk0lht5sUV37GrVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1Srt7s%253D&md5=01861b27c9f87be5e1915cb3dc3a9978</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs00726-013-1485-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-013-1485-1%26sid%3Dliteratum%253Aachs%26aulast%3DMurray-Stewart%26aufirst%3DT.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DThe%2520re-expression%2520of%2520the%2520epigenetically%2520silenced%2520e-cadherin%2520gene%2520by%2520a%2520polyamine%2520analogue%2520lysine-specific%2520demethylase-1%2520%2528LSD1%2529%2520inhibitor%2520in%2520human%2520acute%2520myeloid%2520leukemia%2520cell%2520lines%26jtitle%3DAmino%2520Acids%26date%3D2014%26volume%3D46%26spage%3D585%26epage%3D594%26doi%3D10.1007%2Fs00726-013-1485-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">8023</span>– <span class="NLM_lpage">8028</span>, <span class="refDoi"> DOI: 10.1073/pnas.0700720104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1073%2Fpnas.0700720104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17463086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+Murray+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes&doi=10.1073%2Fpnas.0700720104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span></div><div class="casAuthors">Huang, Yi; Greene, Eriko; Stewart, Tracy Murray; Goodwin, Andrew C.; Baylin, Stephen B.; Woster, Patrick M.; Casero, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8023-8028</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Epigenetic chromatin modification is a major regulator of eukaryotic gene expression, and aberrant epigenetic silencing of gene expression contributes to tumorigenesis.  Histone modifications include acetylation, phosphorylation, and methylation, resulting in a combination of histone marks known collectively as the histone code.  The chromatin marks at a given promoter det., in part, whether specific promoters are in an open/active conformation or closed/repressed conformation.  Dimethyl-lysine 4 histone H3 (H3K4me2) is a transcription-activating chromatin mark at gene promoters, and demethylation of this mark by the lysine-specific demethylase 1 (LSD1), a homolog of polyamine oxidases, may broadly repress gene expression.  We now report that novel biguanide and bisguanidine polyamine analogs are potent inhibitors of LSD1.  These analogs inhibit LSD1 in human colon carcinoma cells and affect a reexpression of multiple, aberrantly silenced genes important in the development of colon cancer, including members of the secreted frizzle-related proteins (SFRPs) and the GATA family of transcription factors.  Furthermore, we demonstrate by chromatin immunopptn. anal. that the reexpression is concurrent with increased H3K4me2 and acetyl-H3K9 marks, decreased H3K9me1 and H3K9me2 repressive marks.  We thus define important new agents for reversing aberrant repression of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC96zJoYEK67Vg90H21EOLACvtfcHk0lht5sUV37GrVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D&md5=4c7863ecb038e3e10c5774cba2c28a45</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700720104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700720104%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMurray%2BStewart%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028%26doi%3D10.1073%2Fpnas.0700720104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. P.</span></span> <span> </span><span class="NLM_article-title">LSD1 demethylates histone and non-histone proteins</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.4161/epi.4.3.8443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.4161%2Fepi.4.3.8443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=19395867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlWntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=129-132&author=T.+B.+Nicholsonauthor=T.+P.+Chen&title=LSD1+demethylates+histone+and+non-histone+proteins&doi=10.4161%2Fepi.4.3.8443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates histone and non-histone proteins</span></div><div class="casAuthors">Nicholson, Thomas B.; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">EPIGB6</span>;
        ISSN:<span class="NLM_cas:issn">1559-2294</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  One of the key breakthroughs in the epigenetics/chromatin field in the last several years was the identification of enzymes capable of removing the Me group from methylated lysines in histone proteins.  Lysine-specific demethylase 1 (LSD1) was the first such enzyme identified, which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  LSD1 is essential for mammalian development and likely involved in many biol. processes.  Recent studies show that LSD1 demethylates p53 and DNA methyltransferase 1 (Dnmt1) and regulates their cellular functions, indicating that LSD1 fulfills its biol. functions by directly acting on both histone and non-histone proteins.  LSD1 contains several defined domains and assocs. with a no. of protein complexes.  Interacting partners of LSD1 may play key roles in detg./modulating the activity and specificity of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3kUnCrwJ2H7Vg90H21EOLACvtfcHk0lht5sUV37GrVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlWntbs%253D&md5=3bb2c45af1a5b8f1e0149d518409faba</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4161%2Fepi.4.3.8443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.4.3.8443%26sid%3Dliteratum%253Aachs%26aulast%3DNicholson%26aufirst%3DT.%2BB.%26aulast%3DChen%26aufirst%3DT.%2BP.%26atitle%3DLSD1%2520demethylates%2520histone%2520and%2520non-histone%2520proteins%26jtitle%3DEpigenetics%26date%3D2009%26volume%3D4%26spage%3D129%26epage%3D132%26doi%3D10.4161%2Fepi.4.3.8443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hevi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurash, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. P.</span></span> <span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+T.+Sunauthor=H.+Changauthor=G.+L.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+P.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0liShE5RvSGQNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%2BT.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%2BL.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%2BP.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubalek, F.</span></span> <span> </span><span class="NLM_article-title">Structure and mechanism of monoamine oxidase</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1983</span>– <span class="NLM_lpage">1993</span>, <span class="refDoi"> DOI: 10.2174/0929867043364784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2174%2F0929867043364784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=15279562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslWmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1983-1993&author=D.+E.+Edmondsonauthor=A.+Matteviauthor=C.+Bindaauthor=M.+Liauthor=F.+Hubalek&title=Structure+and+mechanism+of+monoamine+oxidase&doi=10.2174%2F0929867043364784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of monoamine oxidase</span></div><div class="casAuthors">Edmondson, D. E.; Mattevi, A.; Binda, C.; Li, M.; Hubalek, F.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1983-1993</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Monoamine oxidases A and B (MAO A and MAO B) are mitochondrial outer membrane-bound flavoproteins that catalyze the oxidative deamination of neurotransmitters and biogenic amines.  A no. of mechanism-based inhibitors have been developed for clin. use as antidepressants and as neuroprotective drugs.  To facilitate the development of more effective and specific inhibitors, a detailed understanding of the structures and catalytic mechanisms of these enzymes is required.  The recent development of high level expression systems for producing recombinant human liver MAO A and MAO B in Pichia pastoris has facilitated the detn. of the 3-dimensional crystal structures of MAO B (up to 1.7 Å resoln.) in complex with different reversible (isatin, 1,4-diphenyl-2-butene) and irreversible inhibitors (pargyline, N-(2-aminoethyl)-p-chlorobenzamide, and trans-2-phenylcyclopropylamine).  The binding of substrates or inhibitors to MAO B involves an initial negotiation of a protein loop occurring near the surface of the membrane and 2 hydrophobic cavities, an "entrance" cavity, and an "active site" cavity.  These 2 cavities can either be sep. or in a fused state depending on the conformation of the Ile-199 side-chain, which appears to function as a gate.  The amine function of the bound substrate approaches the re face of the bent and "puckered" covalent FAD through an "arom. cage" formed by 2 Tyr residues that are perpendicular to the plane of the flavin ring.  No amino acid residues that could function as acids or bases are found near the catalytic site.  The existing structural data on MAO B support previous QSAR results and are also supportive of a proposed polar nucleophilic mechanism for MAO A and B catalysis rather than the alternatively proposed single electron transfer mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaQv9eLs73u7Vg90H21EOLACvtfcHk0liShE5RvSGQNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslWmu7s%253D&md5=270a408e70f5a136f49489ff7b5fd482</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2174%2F0929867043364784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867043364784%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DHubalek%26aufirst%3DF.%26atitle%3DStructure%2520and%2520mechanism%2520of%2520monoamine%2520oxidase%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2004%26volume%3D11%26spage%3D1983%26epage%3D1993%26doi%3D10.2174%2F0929867043364784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karytinos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Profumo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaglioli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">A novel mammalian flavin-dependent histone demethylase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">17775</span>– <span class="NLM_lpage">17782</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1074%2Fjbc.M109.003087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=19407342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17775-17782&author=A.+Karytinosauthor=F.+Fornerisauthor=A.+Profumoauthor=G.+Ciossaniauthor=E.+Battaglioliauthor=C.+Bindaauthor=A.+Mattevi&title=A+novel+mammalian+flavin-dependent+histone+demethylase&doi=10.1074%2Fjbc.M109.003087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mammalian Flavin-dependent Histone Demethylase</span></div><div class="casAuthors">Karytinos, Aristotele; Forneris, Federico; Profumo, Antonella; Ciossani, Giuseppe; Battaglioli, Elena; Binda, Claudia; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17775-17782</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of Lys residues on histone proteins is a well known and extensively characterized epigenetic mark.  The recent discovery of lysine-specific demethylase 1 (LSD1) demonstrated that lysine methylation can be dynamically controlled.  Among the histone demethylases so far identified, LSD1 has the unique feature of functioning through a flavin-dependent amine oxidn. reaction.  Data base anal. reveals that mammalian genomes contain a gene (AOF1, for amine-oxidase flavin-contg. domain 1) that is homologous to the LSD1-coding gene.  Here, we demonstrate that the protein encoded by AOF1 represents a second mammalian flavin-dependent histone demethylase, named LSD2.  The new demethylase is strictly specific for mono- and dimethylated Lys4 of histone H3, recognizes a long stretch of the H3 N-terminal tail, senses the presence of addnl. epigenetic marks on the histone substrate, and is covalently inhibited by tranylcypromine.  As opposed to LSD1, LSD2 does not form a biochem. stable complex with the C-terminal domain of the corepressor protein CoREST.  Furthermore, LSD2 contains a CW-type zinc finger motif with potential zinc-binding sites that are not present in LSD1.  We conclude that mammalian LSD2 represents a new flavin-dependent H3-Lys4 demethylase that features substrate specificity properties highly similar to those of LSD1 but is very likely to be part of chromatin-remodeling complexes that are distinct from those involving LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSPhtflZQU7Vg90H21EOLACvtfcHk0liShE5RvSGQNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D&md5=4fbe32f2274de75915e2308781a214d1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.003087%26sid%3Dliteratum%253Aachs%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DProfumo%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520novel%2520mammalian%2520flavin-dependent%2520histone%2520demethylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17775%26epage%3D17782%26doi%3D10.1074%2Fjbc.M109.003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of LSD1 for cancer treatment</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3194</span>– <span class="NLM_lpage">3213</span>, <span class="refDoi"> DOI: 10.3390/molecules23123194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.3390%2Fmolecules23123194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1yrtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=3194-3213&author=G.+J.+Yangauthor=P.+M.+Leiauthor=S.+Y.+Wongauthor=D.+L.+Maauthor=C.+H.+Leung&title=Pharmacological+inhibition+of+LSD1+for+cancer+treatment&doi=10.3390%2Fmolecules23123194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of LSD1 for cancer treatment</span></div><div class="casAuthors">Yang, Guan-Jun; Lei, Pui-Man; Wong, Suk-Yu; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3194/1-3194/20</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Lysine-specific demethylase 1A (LSD1, also named KDM1A) is a demethylase that can remove Me groups from histones H3K4me1/2 and H3K9me1/2.  It is aberrantly expressed in many cancers, where it impedes differentiation and contributes to cancer cell proliferation, cell metastasis and invasiveness, and is assocd. with inferior prognosis.  Pharmacol. inhibition of LSD1 has been reported to significantly attenuate tumor progression in vitro and in vivo in a range of solid tumors and acute myeloid leukemia.  This review will present the structural aspects of LSD1, its role in carcinogenesis, a comparison of currently available approaches for screening LSD1 inhibitors, a classification of LSD1 inhibitors, and its potential as a drug target in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSEBy-HsQHlrVg90H21EOLACvtfcHk0lgt-365HiyEAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1yrtb8%253D&md5=87fe2ac11e363e75ac0ef22ff6412d24</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23123194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23123194%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DG.%2BJ.%26aulast%3DLei%26aufirst%3DP.%2BM.%26aulast%3DWong%26aufirst%3DS.%2BY.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DPharmacological%2520inhibition%2520of%2520LSD1%2520for%2520cancer%2520treatment%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D3194%26epage%3D3213%26doi%3D10.3390%2Fmolecules23123194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Pharmacoepigenetics of LSD1 inhibitors in cancer</span>. <i>Pharmacoepigenetics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">530</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-813939-4.00014-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2FB978-0-12-813939-4.00014-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=523-530&author=B.+Yuauthor=H.+M.+Liu&title=Pharmacoepigenetics+of+LSD1+inhibitors+in+cancer&doi=10.1016%2FB978-0-12-813939-4.00014-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-813939-4.00014-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-813939-4.00014-0%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DPharmacoepigenetics%2520of%2520LSD1%2520inhibitors%2520in%2520cancer%26jtitle%3DPharmacoepigenetics%26date%3D2019%26volume%3D10%26spage%3D523%26epage%3D530%26doi%3D10.1016%2FB978-0-12-813939-4.00014-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Majello, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacca, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amente, S.</span></span> <span> </span><span class="NLM_article-title">Expanding the role of the histone lysine-specific demethylase LSD1 in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">324</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.3390/cancers11030324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.3390%2Fcancers11030324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlKntLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=324-338&author=B.+Majelloauthor=F.+Goriniauthor=C.+D.+Saccaauthor=S.+Amente&title=Expanding+the+role+of+the+histone+lysine-specific+demethylase+LSD1+in+cancer&doi=10.3390%2Fcancers11030324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the role of the histone Lysine-specific demethylase LSD1 in cancer</span></div><div class="casAuthors">Majello, Barbara; Gorini, Francesca; Sacca, Carmen Daniela; Amente, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">324</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Studies of alterations in histone methylation in cancer have led to the identification of histone methyltransferases and demethylases as novel targets for therapy.  Lysine-specific demethylase 1 (LSD1, also known as KDM1A), demethylates H3K4me1/2, or H3K9me1/2 in a context-dependent manner.  In addn. to the well-studied role of LSD1 in the epigenetic regulation of histone methylation changes, LSD1 regulates the methylation dynamic of several non-histone proteins and participates in the assembly of different long noncoding RNA (lncRNA_ complexes.  LSD1 is highly expressed in various cancers, playing a pivotal role in different cancer-related processes.  Here, we summarized recent findings on the role of LSD1 in the regulation of different biol. processes in cancer cells through dynamic methylation of non-histone proteins and phys. assocn. with dedicated lncRNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHjEJqn28EirVg90H21EOLACvtfcHk0lgt-365HiyEAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlKntLjF&md5=d3aede81a2c84173b2efeb5a997a8e6d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Fcancers11030324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11030324%26sid%3Dliteratum%253Aachs%26aulast%3DMajello%26aufirst%3DB.%26aulast%3DGorini%26aufirst%3DF.%26aulast%3DSacca%26aufirst%3DC.%2BD.%26aulast%3DAmente%26aufirst%3DS.%26atitle%3DExpanding%2520the%2520role%2520of%2520the%2520histone%2520lysine-specific%2520demethylase%2520LSD1%2520in%2520cancer%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D324%26epage%3D338%26doi%3D10.3390%2Fcancers11030324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Targeting LSD1 for acute myeloid leukemia (AML) treatment</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">105335</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2020.105335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.phrs.2020.105335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=33285227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3MXit12qsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2021&pages=105335&author=S.+J.+Zhangauthor=M.+H.+Liuauthor=Y.+F.+Yaoauthor=B.+Yuauthor=H.+M.+Liu&title=Targeting+LSD1+for+acute+myeloid+leukemia+%28AML%29+treatment&doi=10.1016%2Fj.phrs.2020.105335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting LSD1 for acute myeloid leukemia (AML) treatment</span></div><div class="casAuthors">Zhang, Shujing; Liu, Menghan; Yao, Yongfang; Yu, Bin; Liu, Hongmin</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105335</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeted therapy for acute myeloid leukemia (AML) is an effective strategy, but currently there are very limited therapeutic targets for AML treatment.  Histone lysine specific demethylase 1 (LSD1) is highly expressed in many cancers, impedes the differentiation of cancer cells, promotes the proliferation, metastasis and invasion of cancer cells, and is assocd. with poor prognosis.  Targeting LSD1 has been recognized as a promising strategy for AML treatment in recent years.  Based on these features, in the , we discussed the main epigenetic drugs targeting LSD1 for AML therapy.  Thus, this focuses on the progress of LSD1 inhibitors in AML treatment, particularly those such as tranylcypromine (TCP), ORY-1001, GSK2879552, and IMG-7289 in clin. trials.  These inhibitors provide novel scaffolds for designing new LSD1 inhibitors.  Besides, combined therapies of LSD1 inhibitors with other drugs for AML treatment are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbIlySiOvBHLVg90H21EOLACvtfcHk0lgt-365HiyEAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXit12qsr8%253D&md5=ce58f5ef8cd0e2cc605d0f18a1ab30d0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2020.105335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2020.105335%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%2BJ.%26aulast%3DLiu%26aufirst%3DM.%2BH.%26aulast%3DYao%26aufirst%3DY.%2BF.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTargeting%2520LSD1%2520for%2520acute%2520myeloid%2520leukemia%2520%2528AML%2529%2520treatment%26jtitle%3DPharmacol.%2520Res.%26date%3D2021%26volume%3D164%26spage%3D105335%26doi%3D10.1016%2Fj.phrs.2020.105335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwergel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">LSD1 inhibitors: a patent review (2010–2015)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1517/13543776.2016.1165209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1517%2F13543776.2016.1165209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27019002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvFGqs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=565-580&author=G.+Staziauthor=C.+Zwergelauthor=S.+Valenteauthor=A.+Mai&title=LSD1+inhibitors%3A+a+patent+review+%282010%E2%80%932015%29&doi=10.1517%2F13543776.2016.1165209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibitors: a patent review (2010-2015)</span></div><div class="casAuthors">Stazi, Giulia; Zwergel, Clemens; Valente, Sergio; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">565-580</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Lysine demethylase 1 (LSD1) plays an important role in mediating the expression of genes involved in cancer and non-cancer diseases such as viral infections, cardiovascular and neurodegenerative disorders.  It is involved in a no. of processes from adipogenesis to cell adhesion to viral latency, regulating several cell pathways related with proliferation, development, and cell cycle control.  Numerous chem. entities have been studied in recent years and some of them entered the clin. arena.  This review summarizes recent efforts in the drug development of LSD1 inhibitors reported in the patent literature covering the 2010-2015 period, including their potential use as therapeutics in cancerous, neurol., inflammatory, cardiovascular, and viral diseases.  The development of novel potent and selective LSD1 inhibitors is ongoing, in order to improve their potency and selectivity against specific types of cancer or non-cancer diseases.  More in-depth studies are required to assess the role of LSD1 inhibitors in the expression of LSD1 target genes, for a better assessment of the biochem. underlying their efficacy, and to provide evidence for any possible side effects.  Furthermore, an interesting therapeutic approach is the use of LSD1 inhibitors in conjunction with other epidrugs to combine their therapeutic potential, leading to innovative, personalized treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCf5uleaXD27Vg90H21EOLACvtfcHk0lgWbXCWsyskog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvFGqs7w%253D&md5=557242114797bc59b35e9f66cec893bf</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1517%2F13543776.2016.1165209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2016.1165209%26sid%3Dliteratum%253Aachs%26aulast%3DStazi%26aufirst%3DG.%26aulast%3DZwergel%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DLSD1%2520inhibitors%253A%2520a%2520patent%2520review%2520%25282010%25E2%2580%25932015%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2016%26volume%3D26%26spage%3D565%26epage%3D580%26doi%3D10.1517%2F13543776.2016.1165209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, O.</span></span> <span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1038/nm.3101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnm.3101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23416702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=291-294&author=L.+Shiauthor=S.+Cuiauthor=J.+D.+Engelauthor=O.+Tanabe&title=Lysine-specific+demethylase+1+is+a+therapeutic+target+for+fetal+hemoglobin+induction&doi=10.1038%2Fnm.3101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span></div><div class="casAuthors">Shi, Lihong; Cui, Shuaiying; Engel, James D.; Tanabe, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">291-294</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Enhanced fetal γ-globin synthesis alleviates symptoms of β-globinopathies such as sickle cell disease and β-thalassemia, but current γ-globin-inducing drugs offer limited beneficial effects.  We show here that lysine-specific demethylase 1 (LSD1) inhibition by RNAi in human erythroid cells or by the monoamine oxidase inhibitor tranylcypromine in human erythroid cells or β-type globin-transgenic mice enhances γ-globin expression.  LSD1 is thus a promising therapeutic target for γ-globin induction, and tranylcypromine may serve as a lead compd. for the development of a new γ-globin inducer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPuLk6KrTQ-rVg90H21EOLACvtfcHk0lgWbXCWsyskog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D&md5=3747256331d40eb16b2ce54a85df1761</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnm.3101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3101%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DS.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DTanabe%26aufirst%3DO.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520a%2520therapeutic%2520target%2520for%2520fetal%2520hemoglobin%2520induction%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D291%26epage%3D294%26doi%3D10.1038%2Fnm.3101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry insights in the discovery of novel LSD1 inhibitors</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1396</span>, <span class="refDoi"> DOI: 10.2217/epi.15.86</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2217%2Fepi.15.86" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=26646727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVejtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1379-1396&author=X.+Wangauthor=B.+Huangauthor=T.+Suzukiauthor=X.+Liuauthor=P.+Zhan&title=Medicinal+chemistry+insights+in+the+discovery+of+novel+LSD1+inhibitors&doi=10.2217%2Fepi.15.86"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry insights in the discovery of novel LSD1 inhibitors</span></div><div class="casAuthors">Wang, Xueshun; Huang, Boshi; Suzuki, Takayoshi; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1379-1396</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">LSD1 is an epigenetic modulator assocd. with transcriptional regulation of genes involved in a broad spectrum of key cellular processes, and its activity is often altered under pathol. conditions.  LSD1 inhibitors are considered to be candidates for therapy of cancer, viral diseases and neurodegeneration.  Many LSD1 inhibitors with various scaffolds have been disclosed, and a few potent mols. are in different stages of clin. development.  In this review, we summarize recent biol. findings on the roles of LSD1 and the current understanding of the clin. significance of LSD1, and focus on the medicinal chem. strategies used in the design and development of LSD1 inhibitors as drug-like epigenetic modulators since 2012, including a brief consideration of structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDkOV2H-u3CrVg90H21EOLACvtfcHk0lgWbXCWsyskog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVejtbrL&md5=da13cfc3e8ec09e86a00d5e87e700f6f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.86&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.86%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DMedicinal%2520chemistry%2520insights%2520in%2520the%2520discovery%2520of%2520novel%2520LSD1%2520inhibitors%26jtitle%3DEpigenomics%26date%3D2015%26volume%3D7%26spage%3D1379%26epage%3D1396%26doi%3D10.2217%2Fepi.15.86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span> <span> </span><span class="NLM_article-title">Drug design concepts for LSD1-selective inhibitors</span>. <i>Chem. Rec.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1782</span>– <span class="NLM_lpage">1791</span>, <span class="refDoi"> DOI: 10.1002/tcr.201810031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Ftcr.201810031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30277644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSntrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=1782-1791&author=Y.+Otaauthor=T.+Suzuki&title=Drug+design+concepts+for+LSD1-selective+inhibitors&doi=10.1002%2Ftcr.201810031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Design Concepts for LSD1-Selective Inhibitors</span></div><div class="casAuthors">Ota, Yosuke; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Record</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1782-1791</span>CODEN:
                <span class="NLM_cas:coden">CRHEAK</span>;
        ISSN:<span class="NLM_cas:issn">1528-0691</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is one of the flavin-dependent oxidases and is involved in many cellular processes by controlling the methylation of histone H3.  Recently, it has been reported that LSD1 is assocd. with several diseases such as cancer, metabolic disorders, and psychiatric diseases.  Thus, LSD1 is an attractive mol. target for the treatment of these diseases, and its inhibitors are predicted as therapeutic agents.  Although a variety of LSD1 inhibitors have been reported to date, many of them show insufficient activities and selectivity toward LSD1.  Meanwhile, we identified several LSD1-selective inhibitors using target-guided synthesis strategies based on our original ideas.  Our LSD1 inhibitors show not only potent LSD1-selective inhibitory activities, but also unique bioactivities both in vitro and in vivo.  This account highlights our drug design concepts for and identification of LSD1-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVzDiBBp_k5bVg90H21EOLACvtfcHk0likiHjKdDJY8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSntrbO&md5=c87dd1d25218fb7f06e302087c61834a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Ftcr.201810031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Ftcr.201810031%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DDrug%2520design%2520concepts%2520for%2520LSD1-selective%2520inhibitors%26jtitle%3DChem.%2520Rec.%26date%3D2018%26volume%3D18%26spage%3D1782%26epage%3D1791%26doi%3D10.1002%2Ftcr.201810031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+U.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0likiHjKdDJY8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, Q. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">Natural products as LSD1 inhibitors for cancer therapy</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.apsb.2020.06.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=Y.+Fangauthor=C.+Yangauthor=Z.+Q.+Yuauthor=X.+C.+Liauthor=Q.+C.+Muauthor=G.+C.+Liaoauthor=B.+Yu&title=Natural+products+as+LSD1+inhibitors+for+cancer+therapy&doi=10.1016%2Fj.apsb.2020.06.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DZ.%2BQ.%26aulast%3DLi%26aufirst%3DX.%2BC.%26aulast%3DMu%26aufirst%3DQ.%2BC.%26aulast%3DLiao%26aufirst%3DG.%2BC.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DNatural%2520products%2520as%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26doi%3D10.1016%2Fj.apsb.2020.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">LSD1/KDM1A inhibitors in clinical trials: advances and prospects</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1186/s13045-019-0811-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1186%2Fs13045-019-0811-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31801559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=129-142&author=Y.+Fangauthor=G.+C.+Liaoauthor=B.+Yu&title=LSD1%2FKDM1A+inhibitors+in+clinical+trials%3A+advances+and+prospects&doi=10.1186%2Fs13045-019-0811-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1/KDM1A inhibitors in clinical trials: advances and prospects</span></div><div class="casAuthors">Fang Yuan; Liao Guochao; Yu Bin; Yu Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone demethylase LSD1 plays key roles during carcinogenesis, targeting LSD1 is becoming an emerging option for the treatment of cancers.  Numerous LSD1 inhibitors have been reported to date, some of them such as TCP, ORY-1001, GSK-2879552, IMG-7289, INCB059872, CC-90011, and ORY-2001 currently undergo clinical assessment for cancer therapy, particularly for small lung cancer cells (SCLC) and acute myeloid leukemia (AML).  This review is to provide a comprehensive overview of LSD1 inhibitors in clinical trials including molecular mechanistic studies, clinical efficacy, adverse drug reactions, and PD/PK studies and offer prospects in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbk3-5OqEjmWlSCSgBhwJFfW6udTcc2eY1gq2jY1jiOLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D&md5=112299f15281667341a55dea57f6c83b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DG.%2BC.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DLSD1%252FKDM1A%2520inhibitors%2520in%2520clinical%2520trials%253A%2520advances%2520and%2520prospects%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2019%26volume%3D12%26spage%3D129%26epage%3D142%26doi%3D10.1186%2Fs13045-019-0811-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Tranylcypromine based LSD1 inhibitor: summary and prospective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14197</span>– <span class="NLM_lpage">14215</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14197-14215&author=X.+J.+Daiauthor=Y.+Liuauthor=X.+P.+Xiongauthor=L.+P.+Xueauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Tranylcypromine+based+LSD1+inhibitor%3A+summary+and+prospective&doi=10.1021%2Facs.jmedchem.0c00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective</span></div><div class="casAuthors">Dai, Xing-Jie; Liu, Ying; Xiong, Xiao-Peng; Xue, Lei-Peng; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14197-14215</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1/KDM1A) has become an important and promising anticancer target since it was first identified in 2004 and specially demethylates lysine residues of histone H3K4me1/2 and H3K9me1/2.  LSD1 is ubiquitously overexpressed in diverse cancers, and abrogation of LSD1 results in inhibition of proliferation, invasion, and migration in cancer cells.  Over the past decade, a no. of biol. active small-mol. LSD1 inhibitors have been developed.  To date, six trans-2-phenylcyclopropylamine (TCP)-based LSD1 inhibitors (including TCP, ORY-1001, GSK-2879552, INCB059872, IMG-7289, and ORY-2001) that covalently bind to the FAD (FAD) within the LSD1 catalytic cavity have already entered into clin. trials.  Here, we provide an overview about the structures, activities, and structure-activity relationship (SAR) of TCP-based LSD1 inhibitors that mainly covers the literature from 2008 to date.  The opportunities, challenges, and future research directions in this emerging and promising field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqasvgaOVZu7Vg90H21EOLACvtfcHk0ljmPoBJe3BgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO&md5=88638bc10c8c090b3df3a0d2f2c020a8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DX.%2BP.%26aulast%3DXue%26aufirst%3DL.%2BP.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTranylcypromine%2520based%2520LSD1%2520inhibitor%253A%2520summary%2520and%2520prospective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14197%26epage%3D14215%26doi%3D10.1021%2Facs.jmedchem.0c00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span> </span><span class="NLM_article-title">The LSD1 inhibitor iadademstat is active in acute
myeloid leukemia</span>.  <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume">10</span>,  <span class="NLM_fpage">OF4</span>.<span class="refDoi"> DOI: 10.1158/2159-8290.CD-RW2020-155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1158%2F2159-8290.CD-RW2020-155" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+LSD1+inhibitor+iadademstat+is+active+in+acute%0Amyeloid+leukemia.+Cancer+Discovery+2020%2C+10%2C+OF4.+10.1158%2F2159-8290.CD-RW2020-155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-RW2020-155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-RW2020-155%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520LSD1%2520inhibitor%2520iadademstat%2520is%2520active%2520in%2520acute%250Amyeloid%2520leukemia%26jtitle%3DCancer%2520Discovery%26date%3D2020%26volume%3D10%26spage%3DOF4%26doi%3D10.1158%2F2159-8290.CD-RW2020-155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirapu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lufino, M. M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willekens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valls, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fyfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro-Palomino, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-1001, a potent and selective covalent KDM1A inhibitor, for the treatment of acute leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">511</span>, e12 <span class="refDoi"> DOI: 10.1016/j.ccell.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ccell.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29502954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=495-511&author=T.+Maesauthor=C.+Mascaroauthor=I.+Tirapuauthor=A.+Estiarteauthor=F.+Ciceriauthor=S.+Lunardiauthor=N.+Guibourtauthor=A.+Perdonesauthor=M.+M.+P.+Lufinoauthor=T.+C.+P.+Somervailleauthor=D.+H.+Wisemanauthor=C.+Duyauthor=A.+Melnickauthor=C.+Willekensauthor=A.+Ortegaauthor=M.+Martinellauthor=N.+Vallsauthor=G.+Kurzauthor=M.+Fyfeauthor=J.+C.+Castro-Palominoauthor=C.+Buesa&title=ORY-1001%2C+a+potent+and+selective+covalent+KDM1A+inhibitor%2C+for+the+treatment+of+acute+leukemia&doi=10.1016%2Fj.ccell.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ORY-1001, a Potent and Selective Covalent KDM1A Inhibitor, for the Treatment of Acute Leukemia</span></div><div class="casAuthors">Maes, Tamara; Mascaro, Cristina; Tirapu, Inigo; Estiarte, Angels; Ciceri, Filippo; Lunardi, Serena; Guibourt, Nathalie; Perdones, Alvaro; Lufino, Michele M. P.; Somervaille, Tim C. P.; Wiseman, Dan H.; Duy, Cihangir; Melnick, Ari; Willekens, Christophe; Ortega, Alberto; Martinell, Marc; Valls, Nuria; Kurz, Guido; Fyfe, Matthew; Castro-Palomino, Julio Cesar; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-511.e12</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lysine-specific demethylase KDM1A is a key regulator of stem cell potential in acute myeloid leukemia (AML).  ORY-1001 is a highly potent and selective KDM1A inhibitor that induces H3K4me2 accumulation on KDM1A target genes, blast differentiation, and redn. of leukemic stem cell capacity in AML.  ORY-1001 exhibits potent synergy with std.-of-care drugs and selective epigenetic inhibitors, reduces growth of an AML xenograft model, and extends survival in a mouse PDX (patient-derived xenograft) model of T cell acute leukemia.  Surrogate pharmacodynamic biomarkers developed based on expression changes in leukemia cell lines were translated to samples from patients treated with ORY-1001.  ORY-1001 is a selective KDM1A inhibitor in clin. trials and is currently being evaluated in patients with leukemia and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2QFk3s8-eLVg90H21EOLACvtfcHk0ljmPoBJe3BgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D&md5=68f69ade1b0110f3e0adf0173326e9b7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascaro%26aufirst%3DC.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DLunardi%26aufirst%3DS.%26aulast%3DGuibourt%26aufirst%3DN.%26aulast%3DPerdones%26aufirst%3DA.%26aulast%3DLufino%26aufirst%3DM.%2BM.%2BP.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DWillekens%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMartinell%26aufirst%3DM.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DKurz%26aufirst%3DG.%26aulast%3DFyfe%26aufirst%3DM.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%2BC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-1001%252C%2520a%2520potent%2520and%2520selective%2520covalent%2520KDM1A%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520acute%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D495%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levant, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keck, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Tranylcypromine substituted <i>cis</i>-hydroxycyclobutylnaphthamides as potent and selective dopamine D<sub>3</sub> receptor antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4962</span>– <span class="NLM_lpage">4968</span>, <span class="refDoi"> DOI: 10.1021/jm401798r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401798r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4962-4968&author=J.+Chenauthor=B.+Levantauthor=C.+Jiangauthor=T.+M.+Keckauthor=A.+H.+Newmanauthor=S.+M.+Wang&title=Tranylcypromine+substituted+cis-hydroxycyclobutylnaphthamides+as+potent+and+selective+dopamine+D3+receptor+antagonists&doi=10.1021%2Fjm401798r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Substituted cis-Hydroxycyclobutylnaphthamides as Potent and Selective Dopamine D3 Receptor Antagonists</span></div><div class="casAuthors">Chen, Jianyong; Levant, Beth; Jiang, Cheng; Keck, Thomas M.; Newman, Amy Hauck; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4962-4968</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a class of potent and selective dopamine D3 receptor antagonists based upon tranylcypromine.  Although tranylcypromine has a low affinity for the rat D3 receptor (Ki = 12.8 μM), our efforts have yielded (1R,2S)-11 (CJ-1882), which has Ki values of 2.7 and 2.8 nM at the rat and human dopamine D3 receptors, resp., and displays resp. selectivities of >10000-fold and 223-fold over the rat and human D2 receptors.  Evaluation in a β-arrestin functional assay showed that (1R,2S)-11 is a potent and competitive antagonist at the human D3 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowWudWdAfWgLVg90H21EOLACvtfcHk0libePjUUIqXAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D&md5=e2ddf0711d00103be281cb043b19ec3a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401798r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401798r%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLevant%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DTranylcypromine%2520substituted%2520cis-hydroxycyclobutylnaphthamides%2520as%2520potent%2520and%2520selective%2520dopamine%2520D3%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4962%26epage%3D4968%26doi%3D10.1021%2Fjm401798r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prusevich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieniewicz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adejola, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taverna, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tackett, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratan, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, O. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">A selective phenelzine analogue inhibitor of histone demethylase LSD1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1284</span>– <span class="NLM_lpage">1293</span>, <span class="refDoi"> DOI: 10.1021/cb500018s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500018s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1284-1293&author=P.+Prusevichauthor=J.+H.+Kalinauthor=S.+A.+Mingauthor=M.+Bassoauthor=J.+Givensauthor=X.+Liauthor=J.+Huauthor=M.+S.+Taylorauthor=A.+M.+Cieniewiczauthor=P.+Y.+Hsiaoauthor=R.+Huangauthor=H.+Robersonauthor=N.+Adejolaauthor=L.+B.+Averyauthor=R.+A.+Caseroauthor=S.+D.+Tavernaauthor=J.+Qianauthor=A.+J.+Tackettauthor=R.+R.+Ratanauthor=O.+G.+McDonaldauthor=A.+P.+Feinbergauthor=P.+A.+Cole&title=A+selective+phenelzine+analogue+inhibitor+of+histone+demethylase+LSD1&doi=10.1021%2Fcb500018s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span></div><div class="casAuthors">Prusevich, Polina; Kalin, Jay H.; Ming, Shonoi A.; Basso, Manuela; Givens, Jeffrey; Li, Xin; Hu, Jianfei; Taylor, Martin S.; Cieniewicz, Anne M.; Hsiao, Po-Yuan; Huang, Rong; Roberson, Heather; Adejola, Nkosi; Avery, Lindsay B.; Casero, Robert A., Jr.; Taverna, Sean D.; Qian, Jiang; Tackett, Alan J.; Ratan, Rajiv R.; McDonald, Oliver G.; Feinberg, Andrew P.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1284-1293</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme that oxidatively cleaves Me groups from monomethyl and di-Me Lys 4 of histone H3 (H3K4Me1, H3K4Me2) and can contribute to gene silencing.  This study describes the design and synthesis of analogs of a monoamine oxidase antidepressant, phenelzine, and their LSD1 inhibitory properties.  A novel phenelzine analog (bizine) contg. a phenyl-butyrylamide appendage was shown to be a potent LSD1 inhibitor in vitro and was selective vs. monoamine oxidases A/B and the LSD1 homolog, LSD2.  Bizine was found to be effective at modulating bulk histone methylation in cancer cells, and ChIP-seq expts. revealed a statistically significant overlap in the H3K4 methylation pattern of genes affected by bizine and those altered in LSD1-/- cells.  Treatment of two cancer cell lines, LNCaP and H460, with bizine conferred a redn. in proliferation rate, and bizine showed additive to synergistic effects on cell growth when used in combination with two out of five HDAC inhibitors tested.  Moreover, neurons exposed to oxidative stress were protected by the presence of bizine, suggesting potential applications in neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Z1w5uKT7qbVg90H21EOLACvtfcHk0libePjUUIqXAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D&md5=36c9f3fb9b28579fa3cad41052cf335a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcb500018s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500018s%26sid%3Dliteratum%253Aachs%26aulast%3DPrusevich%26aufirst%3DP.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DMing%26aufirst%3DS.%2BA.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DGivens%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCieniewicz%26aufirst%3DA.%2BM.%26aulast%3DHsiao%26aufirst%3DP.%2BY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DRoberson%26aufirst%3DH.%26aulast%3DAdejola%26aufirst%3DN.%26aulast%3DAvery%26aufirst%3DL.%2BB.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DMcDonald%26aufirst%3DO.%2BG.%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520selective%2520phenelzine%2520analogue%2520inhibitor%2520of%2520histone%2520demethylase%2520LSD1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1284%26epage%3D1293%26doi%3D10.1021%2Fcb500018s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pippel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Fincke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yiu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Nonpeptidic propargylamines as inhibitors of lysine specific demethylase 1 (LSD1) with cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7334</span>– <span class="NLM_lpage">7342</span>, <span class="refDoi"> DOI: 10.1021/jm400792m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400792m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7334-7342&author=M.+L.+Schmittauthor=A.+T.+Hauserauthor=L.+Carlinoauthor=M.+Pippelauthor=J.+Schulz-Finckeauthor=E.+Metzgerauthor=D.+Willmannauthor=T.+Yiuauthor=M.+Bartonauthor=R.+Schuleauthor=W.+Sipplauthor=M.+Jung&title=Nonpeptidic+propargylamines+as+inhibitors+of+lysine+specific+demethylase+1+%28LSD1%29+with+cellular+activity&doi=10.1021%2Fjm400792m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Nonpeptidic Propargylamines as Inhibitors of Lysine Specific Demethylase 1 (LSD1) with Cellular Activity</span></div><div class="casAuthors">Schmitt, Martin L.; Hauser, Alexander-Thomas; Carlino, Luca; Pippel, Martin; Schulz-Fincke, Johannes; Metzger, Eric; Willmann, Dominica; Yiu, Teresa; Barton, Michelle; Schuele, Roland; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7334-7342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine demethylases play an important role in epigenetic regulation and thus in the development of diseases like cancer or neurodegenerative disorders.  As the lysine specific demethylase 1 (LSD1/KDM1) has been strongly connected to androgen and estrogen dependent gene expression, it serves as a promising target for the therapy of hormone dependent cancer.  Here, we report on the discovery of new small mol. inhibitors of LSD1 contg. a propargylamine warhead, starting out from lysine contg. substrate analogs.  On the basis of these substrate mimicking inhibitors, we were able to increase potency by a combination of similarity-based virtual screening and subsequent synthetic optimization resulting in more druglike LSD1 inhibitors that led to histone hypermethylation in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGRr4rUPfsLVg90H21EOLACvtfcHk0liav5NfvL7QLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO&md5=d4de914dec37f011a7b9488ec539a395</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm400792m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400792m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSchulz-Fincke%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DYiu%26aufirst%3DT.%26aulast%3DBarton%26aufirst%3DM.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DNonpeptidic%2520propargylamines%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520with%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7334%26epage%3D7342%26doi%3D10.1021%2Fjm400792m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span> <span> </span><span class="NLM_article-title">Reversible inhibitors of LSD1 as therapeutic agents in acute myeloid leukemia: clinical significance and progress to date</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">586</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+inhibitors+of+LSD1+as+therapeutic+agents+in+acute+myeloid+leukemia%3A+clinical+significance+and+progress+to+date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0liav5NfvL7QLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520inhibitors%2520of%2520LSD1%2520as%2520therapeutic%2520agents%2520in%2520acute%2520myeloid%2520leukemia%253A%2520clinical%2520significance%2520and%2520progress%2520to%2520date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bianchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polticelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascenzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federico, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariottini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cona, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of polyamine and spermine oxidases by polyamine analogues</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1111/j.1742-4658.2006.05137.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fj.1742-4658.2006.05137.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=16519678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsVCmt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=2006&pages=1115-1123&author=M.+Bianchiauthor=F.+Polticelliauthor=P.+Ascenziauthor=M.+Bottaauthor=R.+Federicoauthor=P.+Mariottiniauthor=A.+Cona&title=Inhibition+of+polyamine+and+spermine+oxidases+by+polyamine+analogues&doi=10.1111%2Fj.1742-4658.2006.05137.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of polyamine and spermine oxidases by polyamine analogues</span></div><div class="casAuthors">Bianchi, Marzia; Polticelli, Fabio; Ascenzi, Paolo; Botta, Maurizio; Federico, Rodolfo; Mariottini, Paolo; Cona, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1115-1123</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Polyamine oxidase (PAO) and spermine oxidase (SMO) are involved in the catabolism of polyamines - basic regulators of cell growth and proliferation.  The discovery of selective inhibitors of PAO and SMO represents an important tool in studying the involvement of these enzymes in polyamine homeostasis and a starting point for the development of novel antineoplastic drugs.  Here, a comparative study on murine PAO (mPAO) and SMO (mSMO) inhibition by the polyamine analogs 1,8-diaminooctane, 1,12-diaminododecane, N-prenylagmatine (G3), guazatine and N,N1-bis(2,3-butadienyl)-1,4-butanediamine (MDL72527) is reported.  Interestingly, 1,12-Diaminododecane and G3 behave as specific inhibitors of mPAO, values of Ki for mPAO inhibition being lower than those for mSMO inactivation by several orders of magnitude.  The anal. of mol. models of mPAO and mSMO indicates a significant redn. of the hydrophobic pocket located in maize PAO (MPAO) at the wider catalytic tunnel opening.  This observation provides a rationale to explain the lower affinity displayed by G3, guazatine and MDL72527 for mPAO and mSMO as compared to MPAO.  The different behavior displayed by 1,12-diaminododecane towards mPAO and mSMO reveals the occurrence of basic differences in the ligand binding mode of the two enzymes, the first enzyme interacting mainly with substrate secondary amino groups and the second one with substrate primary amino groups.  Thus, the data reported here provide the basis for the development of novel and selective inhibitors able to discriminate between mammalian SMO and PAO activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqrXK8OAU1drVg90H21EOLACvtfcHk0lhX_w3WJggL5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsVCmt7w%253D&md5=370a7eff98ab45bec57d14c1839cad93</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2006.05137.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2006.05137.x%26sid%3Dliteratum%253Aachs%26aulast%3DBianchi%26aufirst%3DM.%26aulast%3DPolticelli%26aufirst%3DF.%26aulast%3DAscenzi%26aufirst%3DP.%26aulast%3DBotta%26aufirst%3DM.%26aulast%3DFederico%26aufirst%3DR.%26aulast%3DMariottini%26aufirst%3DP.%26aulast%3DCona%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520polyamine%2520and%2520spermine%2520oxidases%2520by%2520polyamine%2520analogues%26jtitle%3DFEBS%2520J.%26date%3D2006%26volume%3D273%26spage%3D1115%26epage%3D1123%26doi%3D10.1111%2Fj.1742-4658.2006.05137.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">8023</span>– <span class="NLM_lpage">8028</span>, <span class="refDoi"> DOI: 10.1073/pnas.0700720104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1073%2Fpnas.0700720104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17463086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+Murray+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes&doi=10.1073%2Fpnas.0700720104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span></div><div class="casAuthors">Huang, Yi; Greene, Eriko; Stewart, Tracy Murray; Goodwin, Andrew C.; Baylin, Stephen B.; Woster, Patrick M.; Casero, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8023-8028</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Epigenetic chromatin modification is a major regulator of eukaryotic gene expression, and aberrant epigenetic silencing of gene expression contributes to tumorigenesis.  Histone modifications include acetylation, phosphorylation, and methylation, resulting in a combination of histone marks known collectively as the histone code.  The chromatin marks at a given promoter det., in part, whether specific promoters are in an open/active conformation or closed/repressed conformation.  Dimethyl-lysine 4 histone H3 (H3K4me2) is a transcription-activating chromatin mark at gene promoters, and demethylation of this mark by the lysine-specific demethylase 1 (LSD1), a homolog of polyamine oxidases, may broadly repress gene expression.  We now report that novel biguanide and bisguanidine polyamine analogs are potent inhibitors of LSD1.  These analogs inhibit LSD1 in human colon carcinoma cells and affect a reexpression of multiple, aberrantly silenced genes important in the development of colon cancer, including members of the secreted frizzle-related proteins (SFRPs) and the GATA family of transcription factors.  Furthermore, we demonstrate by chromatin immunopptn. anal. that the reexpression is concurrent with increased H3K4me2 and acetyl-H3K9 marks, decreased H3K9me1 and H3K9me2 repressive marks.  We thus define important new agents for reversing aberrant repression of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC96zJoYEK67Vg90H21EOLACvtfcHk0lhX_w3WJggL5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D&md5=4c7863ecb038e3e10c5774cba2c28a45</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700720104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700720104%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMurray%2BStewart%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028%26doi%3D10.1073%2Fpnas.0700720104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marton, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Polyamine analogs modulate gene expression by inhibiting lysine-specific demethylase 1 (LSD1) and altering chromatin structure in human breast cancer cells</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">887</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1007/s00726-011-1004-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00726-011-1004-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=21805138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlyqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2012&pages=887-898&author=Q.+Zhuauthor=Y.+Huangauthor=L.+J.+Martonauthor=P.+M.+Wosterauthor=N.+E.+Davidsonauthor=R.+A.+Casero&title=Polyamine+analogs+modulate+gene+expression+by+inhibiting+lysine-specific+demethylase+1+%28LSD1%29+and+altering+chromatin+structure+in+human+breast+cancer+cells&doi=10.1007%2Fs00726-011-1004-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Polyamine analogs modulate gene expression by inhibiting lysine-specific demethylase 1 (LSD1) and altering chromatin structure in human breast cancer cells</span></div><div class="casAuthors">Zhu, Qingsong; Huang, Yi; Marton, Laurence J.; Woster, Patrick M.; Davidson, Nancy E.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">887-898</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">SpringerWienNewYork</span>)
        </div><div class="casAbstract">Aberrant epigenetic repression of gene expression has been implicated in most cancers, including breast cancer.  The nuclear amine oxidase, lysine-specific demethylase 1 (LSD1) has the ability to broadly repress gene expression by removing the activating mono- and di-methylation marks at the lysine 4 residue of histone 3 (H3K4me1 and me2).  Addnl., LSD1 is highly expressed in estrogen receptor α neg. (ER-) breast cancer cells.  Since epigenetic marks are reversible, they make attractive therapeutic targets.  Here we examine the effects of polyamine analog inhibitors of LSD1 on gene expression, with the goal of targeting LSD1 as a therapeutic modality in the treatment of breast cancer.  Exposure of the ER-neg. human breast cancer cells, MDA-MB-231 to the LSD1 inhibitors, 2d or PG11144, significantly increases global H3K4me1 and H3K4me2, and alters gene expression.  Array anal. indicated that 98 (75 up and 23 down) and 477 (237 up and 240 down) genes changed expression by at least 1.5-fold or greater after treatment with 2d and PG11144, resp.  The expression of 12 up-regulated genes by 2d and 14 up-regulated genes by PG11144 was validated by quant. RT-PCR.  Quant. chromatin immunopptn. (ChIP) anal. demonstrated that up-regulated gene expression by polyamine analogs is assocd. with increase of the active histone marks H3K4me1, H3K4me2 and H3K9act, and decrease of the repressive histone marks H3K9me2 and H3K27me3, in the promoter regions of the relevant target genes.  These data indicate that the pharmacol. inhibition of LSD1 can effectively alter gene expression and that this therapeutic strategy has potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3fWUWScc-bVg90H21EOLACvtfcHk0lhUXvcDWMmIuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlyqtbo%253D&md5=3d99339a2158d2a16e270250c4b44426</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2Fs00726-011-1004-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-011-1004-1%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DMarton%26aufirst%3DL.%2BJ.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DDavidson%26aufirst%3DN.%2BE.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DPolyamine%2520analogs%2520modulate%2520gene%2520expression%2520by%2520inhibiting%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520and%2520altering%2520chromatin%2520structure%2520in%2520human%2520breast%2520cancer%2520cells%26jtitle%3DAmino%2520Acids%26date%3D2012%26volume%3D42%26spage%3D887%26epage%3D898%26doi%3D10.1007%2Fs00726-011-1004-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Novel histone demethylase LSD1 inhibitors selectively target cancer cells with pluripotent stem cell properties</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">7238</span>– <span class="NLM_lpage">7249</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-0896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1158%2F0008-5472.CAN-11-0896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=21975933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCmsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=7238-7249&author=J.+Wangauthor=F.+Luauthor=Q.+Renauthor=H.+Sunauthor=Z.+S.+Xuauthor=R.+F.+Lanauthor=Y.+Q.+Liuauthor=D.+Wardauthor=J.+M.+Quanauthor=T.+Yeauthor=H.+Zhang&title=Novel+histone+demethylase+LSD1+inhibitors+selectively+target+cancer+cells+with+pluripotent+stem+cell+properties&doi=10.1158%2F0008-5472.CAN-11-0896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Histone Demethylase LSD1 Inhibitors Selectively Target Cancer Cells with Pluripotent Stem Cell Properties</span></div><div class="casAuthors">Wang, Jing; Lu, Fei; Ren, Qi; Sun, Hong; Xu, Zhengshuang; Lan, Rongfeng; Liu, Yuqing; Ward, David; Quan, Junmin; Ye, Tao; Zhang, Hui</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7238-7249</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone modification dets. epigenetic patterns of gene expression with methylation of histone H3 at lysine 4 (H3K4) often assocd. with active promoters.  LSD1/KDM1 is a histone demethylase that suppresses gene expression by converting dimethylated H3K4 to mono- and unmethylated H3K4.  LSD1 is essential for metazoan development, but its pathophysiol. functions in cancer remain mainly uncharacterized.  In this study, we developed specific bioactive small inhibitors of LSD1 that enhance H3K4 methylation and derepress epigenetically suppressed genes in vivo.  Strikingly, these compds. inhibited the proliferation of pluripotent cancer cells including teratocarcinoma, embryonic carcinoma, and seminoma or embryonic stem cells that express the stem cell markers Oct4 and Sox2 while displaying min. growth-inhibitory effects on non-pluripotent cancer or normal somatic cells.  RNA interference-mediated knockdown of LSD1 expression phenocopied these effects, confirming the specificity of small mols. and further establishing the high degree of sensitivity and selectivity of pluripotent cancer cells to LSD1 ablation.  In support of these results, we found that LSD1 protein level is highly elevated in pluripotent cancer cells and in human testicular seminoma tissues that express Oct4.  Using these novel chem. inhibitors as probes, our findings establish LSD1 and histone H3K4 methylation as essential cancer-selective epigenetic targets in cancer cells that have pluripotent stem cell properties.  Cancer Res; 71(23); 7238-49.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojMPbxSB7ahbVg90H21EOLACvtfcHk0lhUXvcDWMmIuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCmsb%252FF&md5=742f5d8239623f8ebe17f968f152a2a0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-0896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-0896%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DF.%26aulast%3DRen%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%2BS.%26aulast%3DLan%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BQ.%26aulast%3DWard%26aufirst%3DD.%26aulast%3DQuan%26aufirst%3DJ.%2BM.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DNovel%2520histone%2520demethylase%2520LSD1%2520inhibitors%2520selectively%2520target%2520cancer%2520cells%2520with%2520pluripotent%2520stem%2520cell%2520properties%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D7238%26epage%3D7249%26doi%3D10.1158%2F0008-5472.CAN-11-0896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. C.</span></span> <span> </span><span class="NLM_article-title">CBB1003, a lysine-specific demethylase 1 inhibitor, suppresses colorectal cancer cells growth through down-regulation of leucine-rich repeat-containing G-protein-coupled receptor 5 expression</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1007/s00432-014-1782-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00432-014-1782-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25060070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1aitrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2015&pages=11-21&author=H.+C.+Hsuauthor=Y.+S.+Liuauthor=K.+C.+Tsengauthor=T.+S.+Yangauthor=C.+Y.+Yehauthor=J.+F.+Youauthor=H.+Y.+Hungauthor=S.+J.+Chenauthor=H.+C.+Chen&title=CBB1003%2C+a+lysine-specific+demethylase+1+inhibitor%2C+suppresses+colorectal+cancer+cells+growth+through+down-regulation+of+leucine-rich+repeat-containing+G-protein-coupled+receptor+5+expression&doi=10.1007%2Fs00432-014-1782-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">CBB1003, a lysine-specific demethylase 1 inhibitor, suppresses colorectal cancer cells growth through down-regulation of leucine-rich repeat-containing G-protein-coupled receptor 5 expression</span></div><div class="casAuthors">Hsu, Hung-Chih; Liu, Yi-Shiuan; Tseng, Kai-Chi; Yang, Tsai-Sheng; Yeh, Chien-Yuh; You, Jeng-Fu; Hung, Hsin-Yuan; Chen, Shu-Jen; Chen, Hua-Chien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer Research and Clinical Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-21</span>CODEN:
                <span class="NLM_cas:coden">JCROD7</span>;
        ISSN:<span class="NLM_cas:issn">0171-5216</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Lysine-specific demethylase 1 (LSD1) was highly expressed in several malignancies and had been implicated in pathol. processes of cancer cells.  However, the role of LSD1 in colorectal cancer (CRC) carcinogenesis, prognosis and treatment remains uncharacterized.  Methods: In this study, we examd. LSD1 expression in paraffin-embedded CRC specimens from 295 patients, including 65 patients with paired samples of colorectal carcinoma, adjacent adenoma and normal colorectal tissues.  Using an LSD1 inhibitor, CBB1003, as a probe, we studied the assocn. between LSD1 and leucine-rich repeat-contg. G-protein-coupled receptor 5 (LGR5), a CRC stem cell marker involved in carcinogenesis.  The anti-tumor effects of CBB1003 on CRC cells were also examd.  Results: LSD1 expression was significantly elevated in colorectal tumor tissues compared with adjacent adenoma and normal colorectal tissues (P < 0.001), and LSD1 levels were significantly correlated with an advanced AJCC T stage (P = 0.012) and distant metastasis (P = 0.004).  CBB1003 inhibited CRC cell proliferation and colony formation.  In cultured CRC cells, inhibiting LSD1 activity by CBB1003 caused a decrease in LGR5 levels while overexpression of LGR5 reduced CBB1003-induced cell death.  We also obsd. the inactivation of β-catenin/TCF signaling after CBB1003 treatment, consistent with the pos. correlations among LSD1, LGR5, β-catenin and c-Myc expression in human colon tumor and adenoma tissues.  Conclusion: Our result suggested that LSD1 overexpression promotes CRC development and that the LSD1 inhibitor inhibits CRC cell growth by down-regulating LGR5 levels and inactivates the Wnt/β-catenin pathway.  Thus, LSD1 and its inhibitor might provide a new target or a useful strategy for therapy of CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ21LenCyhXbVg90H21EOLACvtfcHk0lhUXvcDWMmIuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1aitrzM&md5=75c69d5e807d2d587103b9a8761077b7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs00432-014-1782-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-014-1782-4%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DH.%2BC.%26aulast%3DLiu%26aufirst%3DY.%2BS.%26aulast%3DTseng%26aufirst%3DK.%2BC.%26aulast%3DYang%26aufirst%3DT.%2BS.%26aulast%3DYeh%26aufirst%3DC.%2BY.%26aulast%3DYou%26aufirst%3DJ.%2BF.%26aulast%3DHung%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BC.%26atitle%3DCBB1003%252C%2520a%2520lysine-specific%2520demethylase%25201%2520inhibitor%252C%2520suppresses%2520colorectal%2520cancer%2520cells%2520growth%2520through%2520down-regulation%2520of%2520leucine-rich%2520repeat-containing%2520G-protein-coupled%2520receptor%25205%2520expression%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D141%26spage%3D11%26epage%3D21%26doi%3D10.1007%2Fs00432-014-1782-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowotarski, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachaiyappan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holshouser, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutz, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Structure-activity study for (bis)ureidopropyl- and (bis)thioureidopropyldiamine LSD1 inhibitors with 3–5-3 and 3–6-3 carbon backbone architectures</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.01.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2015.01.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25725609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=1601-1612&author=S.+L.+Nowotarskiauthor=B.+Pachaiyappanauthor=S.+L.+Holshouserauthor=C.+J.+Kutzauthor=Y.+Liauthor=Y.+Huangauthor=S.+K.+Sharmaauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=Structure-activity+study+for+%28bis%29ureidopropyl-+and+%28bis%29thioureidopropyldiamine+LSD1+inhibitors+with+3%E2%80%935-3+and+3%E2%80%936-3+carbon+backbone+architectures&doi=10.1016%2Fj.bmc.2015.01.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity study for (bis)ureidopropyl- and (bis)thioureidopropyldiamine LSD1 inhibitors with 3-5-3 and 3-6-3 carbon backbone architectures</span></div><div class="casAuthors">Nowotarski, Shannon L.; Pachaiyappan, Boobalan; Holshouser, Steven L.; Kutz, Craig J.; Li, Youxuan; Huang, Yi; Sharma, Shiv K.; Casero, Robert A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1601-1612</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Methylation at specific histone lysine residues is a crit. posttranslational modification that alters chromatin architecture, and dysregulated lysine methylation/demethylation is assocd. with the silencing of tumor suppressor genes.  The enzyme lysine-specific demethylase 1 (LSD1) complexed to specific transcription factors catalyzes the oxidative demethylation of mono- and dimethyllysine 4 of histone H3 (H3K4me and H3K4me2, resp.).  The authors have previously reported potent (bis)urea and (bis)thiourea LSD1 inhibitors that increase cellular levels of H3K4me and H3K4me2, promote the reexpression of silenced tumor suppressor genes and suppress tumor growth in vitro.  Here the authors report the design addnl. (bis)urea and (bis)thiourea LSD1 inhibitors that feature 3-5-3 or 3-6-3 carbon backbone architectures.  Three of these compds. displayed single-digit IC50 values in a recombinant LSD1 assay.  In addn., compd. I exhibited an IC50 of 4.2 μM against the Calu-6 human lung adenocarcinoma line, and 4.8 μM against the MCF7 breast tumor cell line, in an MTS cell viability assay.  Following treatment with I and its analogs, Calu-6 cells exhibited a significant increase in the mRNA expression for the silenced tumor suppressor genes SFRP2, HCAD and p16, and modest increases in GATA4 message.  The compds. described in this paper represent the most potent epigenetic modulators in this series, and have potential for use as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc-GvXmyP2JbVg90H21EOLACvtfcHk0ljUvF1bPm_Ubg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrY%253D&md5=84aca2f083c3f4df2d66818c08129152</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.01.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.01.049%26sid%3Dliteratum%253Aachs%26aulast%3DNowotarski%26aufirst%3DS.%2BL.%26aulast%3DPachaiyappan%26aufirst%3DB.%26aulast%3DHolshouser%26aufirst%3DS.%2BL.%26aulast%3DKutz%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DSharma%26aufirst%3DS.%2BK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DStructure-activity%2520study%2520for%2520%2528bis%2529ureidopropyl-%2520and%2520%2528bis%2529thioureidopropyldiamine%2520LSD1%2520inhibitors%2520with%25203%25E2%2580%25935-3%2520and%25203%25E2%2580%25936-3%2520carbon%2520backbone%2520architectures%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D1601%26epage%3D1612%26doi%3D10.1016%2Fj.bmc.2015.01.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tortorici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrello, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarelli, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellore, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabb, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packham, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1677</span>– <span class="NLM_lpage">1682</span>, <span class="refDoi"> DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged α-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0ljUvF1bPm_Ubg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljUvF1bPm_Ubg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0lj_Vs9LSCny6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaglioli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of LSD1-CoREST selectivity in histone H3 recognition</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">20070</span>– <span class="NLM_lpage">20074</span>, <span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+basis+of+LSD1-CoREST+selectivity+in+histone+H3+recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0lj_Vs9LSCny6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520LSD1-CoREST%2520selectivity%2520in%2520histone%2520H3%2520recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasinghe, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel cyclic peptide inhibitors of lysine-specific demethylase 1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1021/ml4002997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4002997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCrsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=29-33&author=I.+R.+Kumarasingheauthor=P.+M.+Woster&title=Synthesis+and+evaluation+of+novel+cyclic+peptide+inhibitors+of+lysine-specific+demethylase+1&doi=10.1021%2Fml4002997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Novel Cyclic Peptide Inhibitors of Lysine-Specific Demethylase 1</span></div><div class="casAuthors">Kumarasinghe, Isuru R.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-33</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) selectively removes Me groups from mono- and dimethylated histone 3 lysine 4 (H3K4), resulting in gene silencing.  LSD1 is overexpressed in many human cancers, resulting in aberrant silencing of tumor suppressor genes.  Thus, LSD1 is a validated target for the discovery of antitumor agents.  Using a ligand-based approach, the authors designed and synthesized a series of cyclic and linear peptides that are effective inhibitors of LSD1.  Linear peptide 7 and cyclic peptide 9 inhibited LSD1 in vitro by 91 and 94%, resp., at a concn. of 10 μM.  Compd. 9 was a potent LSD1 inhibitor (IC50 = 2.1 μM; Ki = 385 nM) and had moderate antitumor activity in the MCF-7 and Calu-6 cell lines in vitro.  Importantly, 9 is significantly more stable to hydrolysis in rat plasma than the linear analog 7.  The cyclic peptides described herein represent important lead structures in the search for inhibitors of flavin-dependent histone demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGZKg9n5W93bVg90H21EOLACvtfcHk0lj_Vs9LSCny6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCrsr3F&md5=65810b4100bf4fb4fb10c204e77f88a0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml4002997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4002997%26sid%3Dliteratum%253Aachs%26aulast%3DKumarasinghe%26aufirst%3DI.%2BR.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520cyclic%2520peptide%2520inhibitors%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D29%26epage%3D33%26doi%3D10.1021%2Fml4002997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasinghe, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Cyclic peptide inhibitors of lysine-specific demethylase 1 with improved potency identified by alanine scanning mutagenesis</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2018.01.098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29459279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1yrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2018&pages=210-220&author=I.+R.+Kumarasingheauthor=P.+M.+Woster&title=Cyclic+peptide+inhibitors+of+lysine-specific+demethylase+1+with+improved+potency+identified+by+alanine+scanning+mutagenesis&doi=10.1016%2Fj.ejmech.2018.01.098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic by peptide inhibitors of lysine-specific demethylase 1 with improved potency identified by alanine scanning mutagenesis</span></div><div class="casAuthors">Kumarasinghe, Isuru R.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">210-220</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is a chromatin-remodeling enzyme that plays an important role in cancer.  Over-expression of LSD1 decreases methylation at histone 3 lysine 4, and aberrantly silences tumor suppressor genes.  Inhibitors of LSD1 have been designed as chem. probes and potential antitumor agents.  The authors recently reported the cyclic peptide 9 (H-ARTM-c[K5TARK9E10]TGGK14APRKQLA-NH2), which potently and reversibly inhibits LSD1 (IC50 2.1 μM; Ki 385nM).  Systematic alanine mutagenesis of 9 revealed residues that are crit. for LSD1 inhibition, and these mutated peptides were evaluated as LSD1 inhibitors.  Alanine substitution at positions 2, 3, 4, 6 and 11-17 preserved inhibition, while substitution of alanine at positions 8 and 9 resulted in complete loss of activity.  Cyclic mutant peptides 11 (H-ARAM-c[K5TARKE10]TGGK14APRKQLA-OH) and 16 (H-ARTM-c[K5TARKE10]AGG-KAPRKQLA-OH) produced the greatest LSD1 inhibition, and 11, 16, 27 and 28 increased global H3K4me2 in K562cells.  In addn., 16, 27 and 28 promoted significant increases in H3K4me2 levels at the promoter sites of the genes IGFBP2 and FEZ1.  Data from these LSD1 inhibitors will aid in the design of peptidomimetics with improved stability and pharmacokinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMbcw-dQpAy7Vg90H21EOLACvtfcHk0lgPEdI5wKzMWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1yrsLg%253D&md5=931ec70bf2eb602bc64fbdc3b8b68ae3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.098%26sid%3Dliteratum%253Aachs%26aulast%3DKumarasinghe%26aufirst%3DI.%2BR.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DCyclic%2520peptide%2520inhibitors%2520of%2520lysine-specific%2520demethylase%25201%2520with%2520improved%2520potency%2520identified%2520by%2520alanine%2520scanning%2520mutagenesis%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D148%26spage%3D210%26epage%3D220%26doi%3D10.1016%2Fj.ejmech.2018.01.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talibov, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastiano, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic peptides uncover a novel binding mode for reversible inhibitors of LSD1</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3979</span>– <span class="NLM_lpage">3995</span>, <span class="refDoi"> DOI: 10.1021/acsomega.9b03493</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.9b03493" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFansLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=3979-3995&author=J.+Yangauthor=V.+O.+Talibovauthor=S.+Peintnerauthor=C.+Rheeauthor=V.+Poongavanamauthor=M.+Geitmannauthor=M.+R.+Sebastianoauthor=B.+Simonauthor=J.+Hennigauthor=D.+Dobritzschauthor=U.+H.+Danielsonauthor=J.+Kihlberg&title=Macrocyclic+peptides+uncover+a+novel+binding+mode+for+reversible+inhibitors+of+LSD1&doi=10.1021%2Facsomega.9b03493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides Uncover a Novel Binding Mode for Reversible Inhibitors of LSD1</span></div><div class="casAuthors">Yang, Jie; Talibov, Vladimir O.; Peintner, Stefan; Rhee, Claire; Poongavanam, Vasanthanathan; Geitmann, Matthis; Sebastiano, Matteo Rossi; Simon, Bernd; Hennig, Janosch; Dobritzsch, Doreen; Danielson, U. Helena; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3979-3995</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme which regulates the methylation of Lys4 of histone 3 (H3) and is overexpressed in certain cancers.  We used structures of H3 substrate analogs bound to LSD1 to design macrocyclic peptide inhibitors of LSD1.  A linear, Lys4 to Met-substituted, 11-mer (4) was identified as the shortest peptide distinctly interacting with LSD1.  It was evolved into macrocycle 31, which was >40 fold more potent (Ki = 2.3μM) than 4.  Linear and macrocyclic peptides exhibited unexpected differences in structure-activity relationships for interactions with LSD1, indicating that they bind LSD1 differently.  This was confirmed by the crystal structure of 31 in complex with LSD1-CoREST1, which revealed a novel binding mode at the outer rim of the LSD1 active site and without a direct interaction with FAD.  NMR spectroscopy of 31 suggests that macrocyclization restricts its soln. ensemble to conformations that include the one in the cryst. complex.  Our results provide a solid basis for the design of optimized reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSVg23mFeHAbVg90H21EOLACvtfcHk0lgPEdI5wKzMWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFansLc%253D&md5=34987d78346262625897765eefc198cf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facsomega.9b03493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.9b03493%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTalibov%26aufirst%3DV.%2BO.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DRhee%26aufirst%3DC.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DSebastiano%26aufirst%3DM.%2BR.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DHennig%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520peptides%2520uncover%2520a%2520novel%2520binding%2520mode%2520for%2520reversible%2520inhibitors%2520of%2520LSD1%26jtitle%3DACS%2520Omega%26date%3D2020%26volume%3D5%26spage%3D3979%26epage%3D3995%26doi%3D10.1021%2Facsomega.9b03493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speranzini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrocco, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forgione, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehdipour, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Polymyxins and quinazolines are LSD1/KDM1A inhibitors with unusual structural features</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1601017</span> <span class="refDoi"> DOI: 10.1126/sciadv.1601017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1126%2Fsciadv.1601017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27626075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVWisL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=V.+Speranziniauthor=D.+Rotiliauthor=G.+Ciossaniauthor=S.+Pilottoauthor=B.+Marroccoauthor=M.+Forgioneauthor=A.+Lucidiauthor=F.+Fornerisauthor=P.+Mehdipourauthor=S.+Velankarauthor=A.+Maiauthor=A.+Mattevi&title=Polymyxins+and+quinazolines+are+LSD1%2FKDM1A+inhibitors+with+unusual+structural+features&doi=10.1126%2Fsciadv.1601017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Polymyxins and quinazolines are LSD1/KDM1A inhibitors with unusual structural features</span></div><div class="casAuthors">Speranzini, Valentina; Rotili, Dante; Ciossani, Giuseppe; Pilotto, Simona; Marrocco, Biagina; Forgione, Mariantonietta; Lucidi, Alessia; Forneris, Federico; Mehdipour, Parinaz; Velankar, Sameer; Mai, Antonello; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1601017/1-e1601017/8</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Because of its involvement in the progression of several malignant tumors, the histone lysine-specific demethylase1 (LSD1) has become a prominent drug target in modern medicinal chem. research.  We report on the discoveryof two classes of noncovalent inhibitors displaying unique structural features.  The antibiotics polymyxins bind atthe entrance of the substrate cleft, where their highly charged cyclic moiety interacts with a cluster of pos. charged amino acids.  The same site is occupied by quinazoline-based compds., which were found to inhibit theenzyme through a most peculiar mode because they form a pile of five to seven mols. that obstruct accessto the active center.  These data significantly indicate unpredictable strategies for the development of epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs4C4ugZSy5bVg90H21EOLACvtfcHk0lgPEdI5wKzMWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVWisL8%253D&md5=1bb63c0077a5f83a3d270933314b7d9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1601017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1601017%26sid%3Dliteratum%253Aachs%26aulast%3DSperanzini%26aufirst%3DV.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DForgione%26aufirst%3DM.%26aulast%3DLucidi%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DMehdipour%26aufirst%3DP.%26aulast%3DVelankar%26aufirst%3DS.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DPolymyxins%2520and%2520quinazolines%2520are%2520LSD1%252FKDM1A%2520inhibitors%2520with%2520unusual%2520structural%2520features%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1601017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span> <span> </span><span class="NLM_article-title">Drugging the cancer epigenome</span>.  <i>Proceedings of the 104th Annual Meeting of the American Association for Cancer Research</i>; <span class="NLM_publisher-name">American Association for Cancer Research</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=D.+Dhanak&title=Drugging+the+cancer+epigenome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DDrugging%2520the%2520cancer%2520epigenome%26jtitle%3DProceedings%2520of%2520the%2520104th%2520Annual%2520Meeting%2520of%2520the%2520American%2520Association%2520for%2520Cancer%2520Research%26pub%3DAmerican%2520Association%2520for%2520Cancer%2520Research%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitchin, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockerill, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairweather, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1513</span>– <span class="NLM_lpage">1522</span>, <span class="refDoi"> DOI: 10.1039/c3md00226h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2Fc3md00226h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1513-1522&author=J.+R.+Hitchinauthor=J.+Blaggauthor=R.+Burkeauthor=S.+Burnsauthor=M.+J.+Cockerillauthor=E.+E.+Fairweatherauthor=C.+Huttonauthor=A.+M.+Jordanauthor=C.+McAndrewauthor=A.+Mirzaauthor=D.+Mouldauthor=G.+J.+Thomsonauthor=I.+Waddellauthor=D.+J.+Ogilvie&title=Development+and+evaluation+of+selective%2C+reversible+LSD1+inhibitors+derived+from+fragments&doi=10.1039%2Fc3md00226h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span></div><div class="casAuthors">Hitchin, James R.; Blagg, Julian; Burke, Rosemary; Burns, Samantha; Cockerill, Mark J.; Fairweather, Emma E.; Hutton, Colin; Jordan, Allan M.; McAndrew, Craig; Mirza, Amin; Mould, Daniel; Thomson, Graeme J.; Waddell, Ian; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1513-1522</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of aminothiazoles have been developed as reversible inhibitors of lysine specific demethylase 1 (LSD1) through the expansion of a hit derived from a high concn. biochem. fragment based screen of 2466 compds.  The potency of the initial fragment hit was increased 32-fold through synthesis, with one series of compds. showing clear structure-activity relationships and inhibitory activities in the range of 7 to 187 μM in a biochem. assay.  This series also showed selectivity against the related FAD-dependent enzyme mono-amine oxidase A (MAO-A).  Although a wide range of irreversible inhibitors of LSD1 have been reported with activities in the low nanomolar range, this work represents one of the first reported examples of a reversible small mol. inhibitor of LSD1 with clear SAR and selectivity against MAO-A, and could provide a platform for the development of more potent reversible inhibitors.  Herein, we also report the use of a recently developed cell-based assay for profiling LSD1 inhibitors, and present results on our own compds. as well as a selection of recently described reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUgrL8DOIuDbVg90H21EOLACvtfcHk0lhkruGXu2GBIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK&md5=bec441bbb94f970eab94b96026db7fbe</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2Fc3md00226h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00226h%26sid%3Dliteratum%253Aachs%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DBurns%26aufirst%3DS.%26aulast%3DCockerill%26aufirst%3DM.%2BJ.%26aulast%3DFairweather%26aufirst%3DE.%2BE.%26aulast%3DHutton%26aufirst%3DC.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DMould%26aufirst%3DD.%26aulast%3DThomson%26aufirst%3DG.%2BJ.%26aulast%3DWaddell%26aufirst%3DI.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520selective%252C%2520reversible%2520LSD1%2520inhibitors%2520derived%2520from%2520fragments%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D1513%26epage%3D1522%26doi%3D10.1039%2Fc3md00226h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y. C.</span></span> <span> </span><span class="NLM_article-title">3-(Piperidin-4-ylmethoxy)pyridine containing compounds are potent inhibitors of lysine specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=253-263&author=F.+R.+Wuauthor=C.+Zhouauthor=Y.+Yaoauthor=L.+P.+Weiauthor=Z.+Z.+Fengauthor=L.+S.+Dengauthor=Y.+C.+Song&title=3-%28Piperidin-4-ylmethoxy%29pyridine+containing+compounds+are+potent+inhibitors+of+lysine+specific+demethylase+1&doi=10.1021%2Facs.jmedchem.5b01361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Wu, Fangrui; Zhou, Chao; Yao, Yuan; Wei, Liping; Feng, Zizhen; Deng, Lisheng; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-263</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation.  Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4).  LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression.  We report the design, synthesis, and structure-activity relationships of 3-(piperidin-4-ylmethoxy)pyridine contg. compds. as potent LSD1 inhibitors with Ki values as low as 29 nM.  These compds. exhibited high selectivity (>160×) against related monoamine oxidase A and B.  Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode.  The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKAqQdCeodaLVg90H21EOLACvtfcHk0lhkruGXu2GBIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N&md5=2c74e06ed3c755826d25917d02cb845f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%2BR.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%2BP.%26aulast%3DFeng%26aufirst%3DZ.%2BZ.%26aulast%3DDeng%26aufirst%3DL.%2BS.%26aulast%3DSong%26aufirst%3DY.%2BC.%26atitle%3D3-%2528Piperidin-4-ylmethoxy%2529pyridine%2520containing%2520compounds%2520are%2520potent%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D253%26epage%3D263%26doi%3D10.1021%2Facs.jmedchem.5b01361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of LSD1 in complex with 4-[5-(Piperidin-4-ylmethoxy)-2-(<i>p</i>-tolyl)pyridin-3-yl]benzonitrile</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1538</span>– <span class="NLM_lpage">1545</span>, <span class="refDoi"> DOI: 10.3390/molecules23071538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.3390%2Fmolecules23071538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGht7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1538-1545&author=H.+Niwaauthor=S.+Satoauthor=T.+Hashimotoauthor=K.+Matsunoauthor=T.+Umehara&title=Crystal+structure+of+LSD1+in+complex+with+4-%5B5-%28Piperidin-4-ylmethoxy%29-2-%28p-tolyl%29pyridin-3-yl%5Dbenzonitrile&doi=10.3390%2Fmolecules23071538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of LSD1 in complex with 4-[5-(piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile</span></div><div class="casAuthors">Niwa, Hideaki; Sato, Shin; Hashimoto, Tomoko; Matsuno, Kenji; Umehara, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1538/1-1538/9</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Because lysine-specific demethylase 1 (LSD1) regulates the maintenance of cancer stem cell properties, small-mol. inhibitors of LSD1 are expected to be useful for the treatment of several cancers.  Reversible inhibitors of LSD1 with submicromolar inhibitory potency have recently been reported, but their exact binding modes are poorly understood.  In this study, we synthesized a recently reported reversible inhibitor, 4-[5-(piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile, which bears a 4-piperidinylmethoxy group, a 4-methylphenyl group, and a 4-cyanophenyl group on a pyridine ring, and detd. the crystal structure of LSD1 in complex with this inhibitor at 2.96 Å.  We obsd. strong electron d. for the compd., showing that its cyano group forms a hydrogen bond with Lys661, which is a crit. residue in the lysine demethylation reaction located deep in the catalytic center of LSD1.  The piperidine ring interacts with the side chains of Asp555 and Asn540 in two conformations, and the 4-methylphenyl group is bound in a hydrophobic pocket in the catalytic center.  Our elucidation of the binding mode of this compd. can be expected to facilitate the rational design of more-potent reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5WNfouAObSLVg90H21EOLACvtfcHk0lhkruGXu2GBIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGht7zF&md5=774ef15eacf2facce6aa485a877b31a9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23071538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23071538%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DMatsuno%26aufirst%3DK.%26aulast%3DUmehara%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520LSD1%2520in%2520complex%2520with%25204-%255B5-%2528Piperidin-4-ylmethoxy%2529-2-%2528p-tolyl%2529pyridin-3-yl%255Dbenzonitrile%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1538%26epage%3D1545%26doi%3D10.3390%2Fmolecules23071538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and structure-activity relationship of novel LSD1 inhibitors based on pyrimidine-thiourea hybrids as potent, orally active antitumor agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1705</span>– <span class="NLM_lpage">1716</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1705-1716&author=L.+Y.+Maauthor=Y.+C.+Zhengauthor=S.+Q.+Wangauthor=B.+Wangauthor=Z.+R.+Wangauthor=L.+P.+Pangauthor=M.+Zhangauthor=J.+W.+Wangauthor=L.+Dingauthor=J.+Liauthor=C.+Wangauthor=B.+Huauthor=Y.+Liuauthor=X.+D.+Zhangauthor=J.+J.+Wangauthor=Z.+J.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=Design%2C+synthesis%2C+and+structure-activity+relationship+of+novel+LSD1+inhibitors+based+on+pyrimidine-thiourea+hybrids+as+potent%2C+orally+active+antitumor+agents&doi=10.1021%2Facs.jmedchem.5b00037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Structure-Activity Relationship of Novel LSD1 Inhibitors Based on Pyrimidine-Thiourea Hybrids As Potent, Orally Active Antitumor Agents</span></div><div class="casAuthors">Ma, Li-Ying; Zheng, Yi-Chao; Wang, Sai-Qi; Wang, Bo; Wang, Zhi-Ru; Pang, Lu-Ping; Zhang, Miao; Wang, Jun-Wei; Ding, Lina; Li, Juan; Wang, Cong; Hu, Biao; Liu, Ying; Zhang, Xiao-Dan; Wang, Jia-Jia; Wang, Zhi-Jian; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1705-1716</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) was reported to be overexpressed in several human cancers and recognized as a promising anticancer drug target.  In the current study, we designed and synthesized a novel series of pyrimidine-thiourea hybrids and evaluated their potential LSD1 inhibitory effect.  One of the compds., 6b, contg. a terminal alkyne appendage, was shown to be the most potent and selective LSD1 inhibitor in vitro and exhibited strong cytotoxicity against LSD1 overexpressed gastric cancer cells.  Compd. 6b also showed marked inhibition of cell migration and invasion as well as significant in vivo tumor suppressing and antimetastasis role, without significant side effects by oral administration.  Our findings indicate that the pyrimidine-thiourea-based LSD1 inactivator may serve as a leading compd. targeting LSD1 overexpressed cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy8SIAhdqGGbVg90H21EOLACvtfcHk0lh4K2mffmxhgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyksbw%253D&md5=7256792e23b75e5f26bac47dfa773b11</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00037%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DS.%2BQ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DPang%26aufirst%3DL.%2BP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%2BD.%26aulast%3DWang%26aufirst%3DJ.%2BJ.%26aulast%3DWang%26aufirst%3DZ.%2BJ.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520structure-activity%2520relationship%2520of%2520novel%2520LSD1%2520inhibitors%2520based%2520on%2520pyrimidine-thiourea%2520hybrids%2520as%2520potent%252C%2520orally%2520active%2520antitumor%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1705%26epage%3D1716%26doi%3D10.1021%2Facs.jmedchem.5b00037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Identification of osimertinib (AZD9291) as a lysine specific demethylase 1 inhibitor</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2018.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30502627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGjsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2019&pages=164-169&author=Z.+R.+Liauthor=F.+Z.+Suoauthor=B.+Huauthor=Y.+J.+Guoauthor=D.+J.+Fuauthor=B.+Yuauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Identification+of+osimertinib+%28AZD9291%29+as+a+lysine+specific+demethylase+1+inhibitor&doi=10.1016%2Fj.bioorg.2018.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of osimertinib (AZD9291) as a lysine specific demethylase 1 inhibitor</span></div><div class="casAuthors">Li, Zhong-Rui; Suo, Feng-Zhi; Hu, Bo; Guo, Yan-Jia; Fu, Dong-Jun; Yu, Bin; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">164-169</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Osimertinib (AZD9291) is a third-generation epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) that has been approved for the treatment of EGFR-mutated non-small cell lung cancer (NSCLC).  In this study, osimertinib was characterized as a LSD1 inhibitor for the first time with an IC50 of 3.98 ± 0.3 μM and showed LSD1 inhibitory effect at cellular level.  These findings provide new mol. skeleton for dual inhibitor for LSD1 and EGFR.  Osimertinib could serve as a lead compd. for further development for anti-NSCLC drug discovery with dual targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxIcxf303_PLVg90H21EOLACvtfcHk0lh4K2mffmxhgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGjsLzK&md5=88eae871919618a7a03a57d01096fd87</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DGuo%26aufirst%3DY.%2BJ.%26aulast%3DFu%26aufirst%3DD.%2BJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DIdentification%2520of%2520osimertinib%2520%2528AZD9291%2529%2520as%2520a%2520lysine%2520specific%2520demethylase%25201%2520inhibitor%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D84%26spage%3D164%26epage%3D169%26doi%3D10.1016%2Fj.bioorg.2018.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonanno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zulato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attili, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Bianco, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zago, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frega, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarneri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conte, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indraccolo, S.</span></span> <span> </span><span class="NLM_article-title">Liquid biopsy as tool to monitor and predict clinical benefit from chemotherapy (CT) and immunotherapy (IT) in advanced non-small cell lung cancer (aNSCLC): A prospective study</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">669</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdy303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1093%2Fannonc%2Fmdy303" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=649-669&author=L.+Bonannoauthor=E.+Zulatoauthor=I.+Attiliauthor=A.+Pavanauthor=P.+Del+Biancoauthor=G.+Nardoauthor=M.+Verzaauthor=L.+Pasqualiniauthor=G.+Paselloauthor=G.+Zagoauthor=S.+Fregaauthor=M.+Fassanauthor=F.+Calabreseauthor=A.+Amadoriauthor=V.+Guarneriauthor=P.+F.+Conteauthor=S.+Indraccolo&title=Liquid+biopsy+as+tool+to+monitor+and+predict+clinical+benefit+from+chemotherapy+%28CT%29+and+immunotherapy+%28IT%29+in+advanced+non-small+cell+lung+cancer+%28aNSCLC%29%3A+A+prospective+study&doi=10.1093%2Fannonc%2Fmdy303"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdy303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdy303%26sid%3Dliteratum%253Aachs%26aulast%3DBonanno%26aufirst%3DL.%26aulast%3DZulato%26aufirst%3DE.%26aulast%3DAttili%26aufirst%3DI.%26aulast%3DPavan%26aufirst%3DA.%26aulast%3DDel%2BBianco%26aufirst%3DP.%26aulast%3DNardo%26aufirst%3DG.%26aulast%3DVerza%26aufirst%3DM.%26aulast%3DPasqualini%26aufirst%3DL.%26aulast%3DPasello%26aufirst%3DG.%26aulast%3DZago%26aufirst%3DG.%26aulast%3DFrega%26aufirst%3DS.%26aulast%3DFassan%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DF.%26aulast%3DAmadori%26aufirst%3DA.%26aulast%3DGuarneri%26aufirst%3DV.%26aulast%3DConte%26aufirst%3DP.%2BF.%26aulast%3DIndraccolo%26aufirst%3DS.%26atitle%3DLiquid%2520biopsy%2520as%2520tool%2520to%2520monitor%2520and%2520predict%2520clinical%2520benefit%2520from%2520chemotherapy%2520%2528CT%2529%2520and%2520immunotherapy%2520%2528IT%2529%2520in%2520advanced%2520non-small%2520cell%2520lung%2520cancer%2520%2528aNSCLC%2529%253A%2520A%2520prospective%2520study%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2018%26volume%3D29%26spage%3D649%26epage%3D669%26doi%3D10.1093%2Fannonc%2Fmdy303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isambert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Romano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filvaroff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolova, Z.</span></span> <span> </span><span class="NLM_article-title">Phase I study of CC-90011 in patients with advanced solid tumors and relapsed/refractory non-Hodgkin lymphoma (R/R NHL)</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">I4</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdz029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1093%2Fannonc%2Fmdz029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=I4&author=A.+Hollebecqueauthor=J.+S.+de+Bonoauthor=R.+Plummerauthor=N.+Isambertauthor=P.+Martin-Romanoauthor=E.+Baudinauthor=S.+Moraauthor=E.+Filvaroffauthor=M.+Lambaauthor=Z.+Nikolova&title=Phase+I+study+of+CC-90011+in+patients+with+advanced+solid+tumors+and+relapsed%2Frefractory+non-Hodgkin+lymphoma+%28R%2FR+NHL%29&doi=10.1093%2Fannonc%2Fmdz029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdz029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdz029%26sid%3Dliteratum%253Aachs%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26aulast%3DPlummer%26aufirst%3DR.%26aulast%3DIsambert%26aufirst%3DN.%26aulast%3DMartin-Romano%26aufirst%3DP.%26aulast%3DBaudin%26aufirst%3DE.%26aulast%3DMora%26aufirst%3DS.%26aulast%3DFilvaroff%26aufirst%3DE.%26aulast%3DLamba%26aufirst%3DM.%26aulast%3DNikolova%26aufirst%3DZ.%26atitle%3DPhase%2520I%2520study%2520of%2520CC-90011%2520in%2520patients%2520with%2520advanced%2520solid%2520tumors%2520and%2520relapsed%252Frefractory%2520non-Hodgkin%2520lymphoma%2520%2528R%252FR%2520NHL%2529%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2019%26volume%3D30%26spage%3DI4%26doi%3D10.1093%2Fannonc%2Fmdz029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, N. Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stansfield, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawton, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosfield, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreilein, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares-Greco, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veal, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of CC-90011: a potent and selective reversible inhibitor of lysine specific demethylase 1 (LSD1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14522</span>– <span class="NLM_lpage">15529</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00978</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00978" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVSqsr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14522-15529&author=T.+Kanouniauthor=C.+Severinauthor=R.+W.+Choauthor=N.+Y.+Y.+Yuenauthor=J.+Xuauthor=L.+Shiauthor=C.+Laiauthor=J.+R.+Del+Rosarioauthor=R.+K.+Stansfieldauthor=L.+N.+Lawtonauthor=D.+Hosfieldauthor=S.+O%E2%80%99Connellauthor=M.+M.+Kreileinauthor=P.+Tavares-Grecoauthor=Z.+Nieauthor=S.+W.+Kaldorauthor=J.+M.+Vealauthor=J.+A.+Staffordauthor=Y.+K.+Chen&title=Discovery+of+CC-90011%3A+a+potent+and+selective+reversible+inhibitor+of+lysine+specific+demethylase+1+%28LSD1%29&doi=10.1021%2Facs.jmedchem.0c00978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1)</span></div><div class="casAuthors">Kanouni, Toufike; Severin, Christophe; Cho, Robert W.; Yuen, Natalie Y.-Y.; Xu, Jiangchun; Shi, Lihong; Lai, Chon; Del Rosario, Joselyn R.; Stansfield, Ryan K.; Lawton, Lee N.; Hosfield, David; O'Connell, Shawn; Kreilein, Matt M.; Tavares-Greco, Paula; Nie, Zhe; Kaldor, Stephen W.; Veal, James M.; Stafford, Jeffrey A.; Chen, Young K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14522-14529</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone demethylase LSDl (KDMlA) belongs to the FAD (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biol.  Dysregulation and overexpression of LSD1 are hallmarks of a no. of human diseases, particularly cancers that are characterized as morphol. poorly differentiated.  As such, inhibitors of LSD1 have potential to be beneficial as a cancer therapy.  The most clin. advanced inhibitors of LSDl are covalent inhibitors derived from tranylcypromine (TCP).  Herein, we report the discovery of a novel series of reversible and selective LSDl inhibitors.  Exploration of structure-activity relationships (SARs) and optimization of ADME properties resulted in the identification of clin. candidate CC-90011.  CC-90011 exhibits potent on-target induction of cellular differentiation in acute myeloid leukemia (AML) and small cell lung cancer (SCLC) cell lines, and antitumor efficacy in patient-derived xenograft (PDX) SCLC models.  CC-90011 is currently in phase 2 trials in patients with first line, extensive stage SCLC (ClinicalTrials.gov identifier: NCT03850067).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6pM7y3zzSNbVg90H21EOLACvtfcHk0lg8MHwpMpyc8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVSqsr%252FF&md5=cd3fdea62cf5448237fa341747c59884</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00978%26sid%3Dliteratum%253Aachs%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DSeverin%26aufirst%3DC.%26aulast%3DCho%26aufirst%3DR.%2BW.%26aulast%3DYuen%26aufirst%3DN.%2BY.%2BY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DDel%2BRosario%26aufirst%3DJ.%2BR.%26aulast%3DStansfield%26aufirst%3DR.%2BK.%26aulast%3DLawton%26aufirst%3DL.%2BN.%26aulast%3DHosfield%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DS.%26aulast%3DKreilein%26aufirst%3DM.%2BM.%26aulast%3DTavares-Greco%26aufirst%3DP.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DKaldor%26aufirst%3DS.%2BW.%26aulast%3DVeal%26aufirst%3DJ.%2BM.%26aulast%3DStafford%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DY.%2BK.%26atitle%3DDiscovery%2520of%2520CC-90011%253A%2520a%2520potent%2520and%2520selective%2520reversible%2520inhibitor%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14522%26epage%3D15529%26doi%3D10.1021%2Facs.jmedchem.0c00978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span> <span> </span><span class="NLM_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1080/15592294.2017.1290032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1080%2F15592294.2017.1290032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28277979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=340-352&author=H.+Niwaauthor=T.+Umehara&title=Structural+insight+into+inhibitors+of+flavin+adenine+dinucleotide-dependent+lysine+demethylases&doi=10.1080%2F15592294.2017.1290032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span></div><div class="casAuthors">Niwa Hideaki; Umehara Takashi; Umehara Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-352</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Until 2004, many researchers believed that protein methylation in eukaryotic cells was an irreversible reaction.  However, the discovery of lysine-specific demethylase 1 in 2004 drastically changed this view and the concept of chromatin regulation.  Since then, the enzymes responsible for lysine demethylation and their cellular substrates, biological significance, and selective regulation have become major research topics in epigenetics and chromatin biology.  Many cell-permeable inhibitors for lysine demethylases have been developed, including both target-specific and nonspecific inhibitors.  Structural understanding of how these inhibitors bind to lysine demethylases is crucial both for validation of the inhibitors as chemical probes and for the rational design of more potent, target-specific inhibitors.  This review focuses on published small-molecule inhibitors targeted at the two flavin adenine dinucleotide-dependent lysine demethylases, lysine-specific demethylases 1 and 2, and how the inhibitors interact with the tertiary structures of the enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyKG85Hfra9xjv2lLSLOxFfW6udTcc2ebW_rlV8sdDBLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D&md5=31f93c2cc7909c4197f5892f7f04af9d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1080%2F15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DUmehara%26aufirst%3DT.%26atitle%3DStructural%2520insight%2520into%2520inhibitors%2520of%2520flavin%2520adenine%2520dinucleotide-dependent%2520lysine%2520demethylases%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D340%26epage%3D352%26doi%3D10.1080%2F15592294.2017.1290032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e25444</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0025444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1371%2Fjournal.pone.0025444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=21984927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=X.+Kongauthor=S.+Ouyangauthor=Z.+Liangauthor=J.+Luauthor=L.+Chenauthor=B.+Shenauthor=D.+Liauthor=M.+Zhengauthor=K.+K.+Liauthor=C.+Luoauthor=H.+Jiang&title=Catalytic+mechanism+investigation+of+lysine-specific+demethylase+1+%28LSD1%29%3A+a+computational+study&doi=10.1371%2Fjournal.pone.0025444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span></div><div class="casAuthors">Kong, Xiangqian; Ouyang, Sisheng; Liang, Zhongjie; Lu, Junyan; Chen, Liang; Shen, Bairong; Li, Donghai; Zheng, Mingyue; Li, Keqin Kathy; Luo, Cheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25444</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), the first identified histone demethylase, is a flavin-dependent amine oxidase which specifically demethylates mono- or di-methylated H3K4 and H3K9 via a redox process.  It participates in a broad spectrum of biol. processes and is of high importance in cell proliferation, adipogenesis, spermatogenesis, chromosome segregation and embryonic development.  To date, as a potential drug target for discovering anti-tumor drugs, the medical significance of LSD1 has been greatly appreciated.  However, the catalytic mechanism for the rate-limiting reductive half-reaction in demethylation remains controversial.  By employing a combined computational approach including mol. modeling, mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns., the catalytic mechanism of dimethylated H3K4 demethylation by LSD1 was characterized in details.  The three-dimensional (3D) model of the complex was composed of LSD1, CoREST, and histone substrate.  A 30-ns MD simulation of the model highlights the pivotal role of the conserved Tyr761 and lysine-water-flavin motif in properly orienting FAD with respect to substrate.  The synergy of the two factors effectively stabilizes the catalytic environment and facilitated the demethylation reaction.  On the basis of the reasonable consistence between simulation results and available mutagenesis data, QM/MM strategy was further employed to probe the catalytic mechanism of the reductive half-reaction in demethylation.  The characteristics of the demethylation pathway detd. by the potential energy surface and charge distribution anal. indicates that this reaction belongs to the direct hydride transfer mechanism.  Our study provides insights into the LSD1 mechanism of reductive half-reaction in demethylation and has important implications for the discovery of regulators against LSD1 enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIm_wJ2koFrVg90H21EOLACvtfcHk0lizZuvKA_-CbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP&md5=7ccdcdcf6e8770ea0d77c1cc80f79da8</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025444%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCatalytic%2520mechanism%2520investigation%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%253A%2520a%2520computational%2520study%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0025444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Exploration of 5-cyano-6-phenylpyrimidin derivatives containing an 1,2,3-triazole moiety as potent FAD-based LSD1 inhibitors</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1658</span>– <span class="NLM_lpage">1668</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.apsb.2020.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=33088686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnslSrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=1658-1668&author=L.+Y.+Maauthor=H.+J.+Wangauthor=Y.+H.+Youauthor=C.+Y.+Maauthor=Y.+J.+Liuauthor=F.+F.+Yangauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Exploration+of+5-cyano-6-phenylpyrimidin+derivatives+containing+an+1%2C2%2C3-triazole+moiety+as+potent+FAD-based+LSD1+inhibitors&doi=10.1016%2Fj.apsb.2020.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of 5-cyano-6-phenylpyrimidin derivatives containing an 1,2,3-triazole moiety as potent FAD-based LSD1 inhibitors</span></div><div class="casAuthors">Ma, Liying; Wang, Haojie; You, Yinghua; Ma, Chaoya; Liu, Yuejiao; Yang, Feifei; Zheng, Yichao; Liu, Hongmin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1658-1668</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) has become a potential therapeutic target for the treatment of cancer.  Discovery and develop novel and potent LSD1 inhibitors is a challenge, although several of them have already entered into clin. trials.  Herein, for the first time, we reported the discovery of a series of 5-cyano-6-phenylpyrimidine derivs. as LSD1 inhibitors using FAD (FAD) similarity-based designing strategy, of which compd. 14q was finally identified to repress LSD1 with IC50 = 183 nmol/L.  Docking anal. suggested that compd. 14q fitted well into the FAD-binding pocket.  Further mechanism studies showed that compd. 14q may inhibit LSD1 activity competitively by occupying the FAD binding sites of LSD1 and inhibit cell migration and invasion by reversing epithelial to mesenchymal transition (EMT).  Overall, these findings showed that compd. 14q is a suitable candidate for further development of novel FAD similarity-based LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGalgCfnOafrVg90H21EOLACvtfcHk0lizZuvKA_-CbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnslSrsg%253D%253D&md5=fb61dc6d62dd0920a3dfa005ddfdf7b2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DWang%26aufirst%3DH.%2BJ.%26aulast%3DYou%26aufirst%3DY.%2BH.%26aulast%3DMa%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DY.%2BJ.%26aulast%3DYang%26aufirst%3DF.%2BF.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DExploration%2520of%25205-cyano-6-phenylpyrimidin%2520derivatives%2520containing%2520an%25201%252C2%252C3-triazole%2520moiety%2520as%2520potent%2520FAD-based%2520LSD1%2520inhibitors%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26volume%3D10%26spage%3D1658%26epage%3D1668%26doi%3D10.1016%2Fj.apsb.2020.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutz, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holshouser, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrow, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1863</span>– <span class="NLM_lpage">1870</span>, <span class="refDoi"> DOI: 10.1039/C4MD00283K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2FC4MD00283K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25580204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1863-1870&author=C.+J.+Kutzauthor=S.+L.+Holshouserauthor=E.+A.+Marrowauthor=P.+M.+Woster&title=3%2C5-Diamino-1%2C2%2C4-triazoles+as+a+novel+scaffold+for+potent%2C+reversible+LSD1+%28KDM1A%29+inhibitors&doi=10.1039%2FC4MD00283K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span></div><div class="casAuthors">Kutz, Craig J.; Holshouser, Steven L.; Marrow, Ethan A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1863-1870</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The chromatin remodeling amine oxidase lysine-specific demethylase 1 (LSD1) has become an attractive target for the design of specific inhibitors with therapeutic potential.  We, and others, have described LSD1 inhibitors that have potential as antitumor agents.  Many of the currently known LSD1 inhibitors are poor drug candidates, or are structurally based on the tranylcypromine backbone, thus increasing the potential for off-target effects mediated by other amine oxidases.  We now describe a series of potent LSD1 inhibitors based on a novel 1,2,4-triazole scaffold; these inhibitors show a high degree of specificity for LSD1 in vitro, and cause increases in cellular histone 3 dimethyllysine 4 (H3K4me2), a gene transcription activating mark.  Importantly, these inhibitors exhibit low toxicity towards mammalian cells in vitro, and thus they may show utility in the treatment of epigenetically-based diseases where cell death is not a desired endpoint.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXpwx29ujiWbVg90H21EOLACvtfcHk0lizZuvKA_-CbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N&md5=3a6314468bdb665cb30ae86505e5eb15</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2FC4MD00283K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00283K%26sid%3Dliteratum%253Aachs%26aulast%3DKutz%26aufirst%3DC.%2BJ.%26aulast%3DHolshouser%26aufirst%3DS.%2BL.%26aulast%3DMarrow%26aufirst%3DE.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D3%252C5-Diamino-1%252C2%252C4-triazoles%2520as%2520a%2520novel%2520scaffold%2520for%2520potent%252C%2520reversible%2520LSD1%2520%2528KDM1A%2529%2520inhibitors%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1863%26epage%3D1870%26doi%3D10.1039%2FC4MD00283K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holshouser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunworth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray-Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Dual inhibitors of LSD1 and spermine oxidase</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">778</span>– <span class="NLM_lpage">790</span>, <span class="refDoi"> DOI: 10.1039/C8MD00610E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2FC8MD00610E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31191868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXis1Gqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=778-790&author=S.+Holshouserauthor=M.+Dunworthauthor=T.+Murray-Stewartauthor=Y.+K.+Petersonauthor=P.+Burgerauthor=J.+Kirkpatrickauthor=H.+H.+Chenauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=Dual+inhibitors+of+LSD1+and+spermine+oxidase&doi=10.1039%2FC8MD00610E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Dual inhibitors of LSD1 and spermine oxidase</span></div><div class="casAuthors">Holshouser, Steven; Dunworth, Matthew; Murray-Stewart, Tracy; Peterson, Yuri K.; Burger, Pieter; Kirkpatrick, Joy; Chen, Huan-Huan; Casero, Robert A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">778-790</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We have previously described the synthesis and evaluation of 3,5-diamino-1,2,4-triazole analogs as inhibitors of the flavin-dependent histone demethylase LSD1.  These compds. are potent inhibitors of LSD1 without activity against monoamine oxidases A and B, and promote the elevation of H3K4me2 levels in tumor cells in vitro.  We now report that the cytotoxicity of these analogs in pancreatic tumor cells correlates with the overexpression of LSD1 in each tumor type.  In addn., we show that a subset of these 3,5-diamino-1,2,4-triazole analogs inhibit a related flavin-dependent oxidase, the polyamine catabolic enzyme spermine oxidase (SMOX) in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCgjp3cbY2rVg90H21EOLACvtfcHk0lizZuvKA_-CbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXis1Gqs70%253D&md5=b81307aef3440a1dc7ad431a33b6ca89</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2FC8MD00610E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8MD00610E%26sid%3Dliteratum%253Aachs%26aulast%3DHolshouser%26aufirst%3DS.%26aulast%3DDunworth%26aufirst%3DM.%26aulast%3DMurray-Stewart%26aufirst%3DT.%26aulast%3DPeterson%26aufirst%3DY.%2BK.%26aulast%3DBurger%26aufirst%3DP.%26aulast%3DKirkpatrick%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%2BH.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DDual%2520inhibitors%2520of%2520LSD1%2520and%2520spermine%2520oxidase%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D778%26epage%3D790%26doi%3D10.1039%2FC8MD00610E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Triazole-dithiocarbamate based selective lysine specific demethylase 1 (LSD1) inactivators inhibit gastric cancer cell growth, invasion, and migration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8543</span>– <span class="NLM_lpage">8560</span>, <span class="refDoi"> DOI: 10.1021/jm401002r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401002r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8543-8560&author=Y.+C.+Zhengauthor=Y.+C.+Duanauthor=J.+L.+Maauthor=R.+M.+Xuauthor=X.+Ziauthor=W.+L.+Lvauthor=M.+M.+Wangauthor=X.+W.+Yeauthor=S.+Zhuauthor=D.+Mobleyauthor=Y.+Y.+Zhuauthor=J.+W.+Wangauthor=J.+F.+Liauthor=Z.+R.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=Triazole-dithiocarbamate+based+selective+lysine+specific+demethylase+1+%28LSD1%29+inactivators+inhibit+gastric+cancer+cell+growth%2C+invasion%2C+and+migration&doi=10.1021%2Fjm401002r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Triazole-Dithiocarbamate Based Selective Lysine Specific Demethylase 1 (LSD1) Inactivators Inhibit Gastric Cancer Cell Growth, Invasion, and Migration</span></div><div class="casAuthors">Zheng, Yi-Chao; Duan, Ying-Chao; Ma, Jin-Lian; Xu, Rui-Min; Zi, Xiaolin; Lv, Wen-Lei; Wang, Meng-Meng; Ye, Xian-Wei; Zhu, Shun; Mobley, David; Zhu, Yan-Yan; Wang, Jun-Wei; Li, Jin-Feng; Wang, Zhi-Ru; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8543-8560</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1), the first identified histone demethylase, plays an important role in epigenetic regulation of gene activation and repression.  The up-regulated LSD1's expression has been reported in several malignant tumors.  In the current study, we designed and synthesized five series of 1,2,3-triazole-dithiocarbamate hybrids and screened their inhibitory activity toward LSD1.  We found that some of these compds., esp. compd. (I), exhibited the most specific and robust inhibition of LSD1.  Interestingly, compd. I also showed potent and selective cytotoxicity against LSD1 overexpressing gastric cancer cell lines MGC-803 and HGC-27, as well as marked inhibition of cell migration and invasion, compared to 2-PCPA.  Furthermore, compd. I effectively reduced the tumor growth bared by human gastric cancer cells in vivo with no signs of adverse side effects.  These findings suggested that compd. I deserves further investigation as a lead compd. in the treatment of LSD1 overexpressing gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjGdOvultbebVg90H21EOLACvtfcHk0ljS8vxljie9YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL&md5=4e4594271ff6b62237ff17a87d6f04d8</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm401002r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401002r%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DZi%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DM.%2BM.%26aulast%3DYe%26aufirst%3DX.%2BW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DMobley%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%2BY.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DLi%26aufirst%3DJ.%2BF.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTriazole-dithiocarbamate%2520based%2520selective%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520inactivators%2520inhibit%2520gastric%2520cancer%2520cell%2520growth%252C%2520invasion%252C%2520and%2520migration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8543%26epage%3D8560%26doi%3D10.1021%2Fjm401002r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrancosta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laras, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbette, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maigret, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quéléver, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatsubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, J. L.</span></span> <span> </span><span class="NLM_article-title">Naphthyl and coumarinyl biarylpiperazine derivatives as highly potent human beta-secretase inhibitors. Design, synthesis, and enzymatic BACE-1 and cell assays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4275</span>– <span class="NLM_lpage">4285</span>, <span class="refDoi"> DOI: 10.1021/jm0602864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0602864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFent70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4275-4285&author=C.+Garinoauthor=T.+Tomitaauthor=N.+Pietrancostaauthor=Y.+Larasauthor=R.+Rosasauthor=G.+Herbetteauthor=B.+Maigretauthor=G.+Qu%C3%A9l%C3%A9verauthor=T.+Iwatsuboauthor=J.+L.+Kraus&title=Naphthyl+and+coumarinyl+biarylpiperazine+derivatives+as+highly+potent+human+beta-secretase+inhibitors.+Design%2C+synthesis%2C+and+enzymatic+BACE-1+and+cell+assays&doi=10.1021%2Fjm0602864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Naphthyl and coumarinyl biarylpiperazine derivatives as highly potent human β-secretase inhibitors. design, synthesis, and enzymatic BACE-1 and cell assays</span></div><div class="casAuthors">Garino, Cedrik; Tomita, Taisuke; Pietrancosta, Nicolas; Laras, Younes; Rosas, Roselyne; Herbette, Gaetan; Maigret, Bernard; Quelever, Gilles; Iwatsubo, Takeshi; Kraus, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4275-4285</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Twenty novel β-secretase inhibitors contg. biarylpiperazine moieties belonging to naphthyl and coumarinyl series were designed for their potential use in Alzheimer's disease therapy.  Enzymic and cell-based assays have been carried out.  The biol. results clearly demonstrate that specific substituents located at the N4-position of the piperazine ring result in excellent in vitro inhibitory potency (IC50 values ranging between 40 and 70 nM).  Variable temp. NMR and modeling studies are consistent with the obtained biol. data, since these studies confirmed that introduction at the N4-position of the piperazine ring allows productive interactions within the BACE-1 active site, which appear to be determinative for high BACE-1 inhibitory activity.  These results are of particular interest since some of the new analogs belonging to the naphthyl series are almost one log more active than the best inhibitor of the similar family recently reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUbFH1mOoKlrVg90H21EOLACvtfcHk0ljS8vxljie9YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFent70%253D&md5=8a29e41e8ae98ef9dab01d4e94bef291</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm0602864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0602864%26sid%3Dliteratum%253Aachs%26aulast%3DGarino%26aufirst%3DC.%26aulast%3DTomita%26aufirst%3DT.%26aulast%3DPietrancosta%26aufirst%3DN.%26aulast%3DLaras%26aufirst%3DY.%26aulast%3DRosas%26aufirst%3DR.%26aulast%3DHerbette%26aufirst%3DG.%26aulast%3DMaigret%26aufirst%3DB.%26aulast%3DQu%25C3%25A9l%25C3%25A9ver%26aufirst%3DG.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DKraus%26aufirst%3DJ.%2BL.%26atitle%3DNaphthyl%2520and%2520coumarinyl%2520biarylpiperazine%2520derivatives%2520as%2520highly%2520potent%2520human%2520beta-secretase%2520inhibitors.%2520Design%252C%2520synthesis%252C%2520and%2520enzymatic%2520BACE-1%2520and%2520cell%2520assays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4275%26epage%3D4285%26doi%3D10.1021%2Fjm0602864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of coumarin-1,2,3-triazole–dithiocarbamate hybrids as potent LSD1 inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1039/C4MD00031E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2FC4MD00031E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXms1Wqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=650-654&author=X.+W.+Yeauthor=Y.+C.+Zhengauthor=Y.+C.+Duanauthor=M.+M.+Wangauthor=B.+Yuauthor=J.+L.+Renauthor=J.+L.+Maauthor=E.+Zhangauthor=H.+M.+Liu&title=Synthesis+and+biological+evaluation+of+coumarin-1%2C2%2C3-triazole%E2%80%93dithiocarbamate+hybrids+as+potent+LSD1+inhibitors&doi=10.1039%2FC4MD00031E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of coumarin-1,2,3-triazole-dithiocarbamate hybrids as potent LSD1 inhibitors</span></div><div class="casAuthors">Ye, Xian-Wei; Zheng, Yi-Chao; Duan, Ying-Chao; Wang, Meng-Meng; Yu, Bin; Ren, Jing-Li; Ma, Jin-Lian; Zhang, En; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">650-654</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of coumarin-1,2,3-triazole-dithiocarbamate hybrids I [R1 = 7-Cl, 6-Cl, 7-NH2, 5,7-(HO)2, etc.] and II (R2 = H, 7-OH) were designed, synthesized and evaluated for their inhibitory activity towards lysine specific demethylase 1 (LSD1).  Compds. I [R1 = 7-Cl, H, 7-NH2, 7-CH3, 5,7-(OH)2, 7,8-(OH)2, 7-OH, 7-OCH3] presented potent activity against lysine specific demethylase 1.  Among them, compd. I (R1 = 7-OH) (III) showed potent and reversible inhibition against lysine specific demethylase 1 with an IC50 value of 0.39 μM, which was 74-fold more potent than that of tranylcypromine (2-PCPA).  Besides, compd. III displayed excellent selectivity against lysine specific demethylase 1 without inhibition against monoamine oxidases (MAOs) A and B.  Further investigation revealed that compd. III was active at both recombinant and cell levels by upregulating the expression of H3K4me1, H3K4me2 and H3K9me2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4HUPEXzC-R7Vg90H21EOLACvtfcHk0ljWe787smJKLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXms1Wqs78%253D&md5=c29af9f1f1c649032d4bec605bd92e8e</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1039%2FC4MD00031E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00031E%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DX.%2BW.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DRen%26aufirst%3DJ.%2BL.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520coumarin-1%252C2%252C3-triazole%25E2%2580%2593dithiocarbamate%2520hybrids%2520as%2520potent%2520LSD1%2520inhibitors%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D650%26epage%3D654%26doi%3D10.1039%2FC4MD00031E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dulla, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirla, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathore, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deora, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddika, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of 3-amino/guanidine substituted phenyl oxazoles as a novel class of LSD1 inhibitors with anti-proliferative properties</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3103</span>– <span class="NLM_lpage">3107</span>, <span class="refDoi"> DOI: 10.1039/c3ob40217g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2Fc3ob40217g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23575971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtlCgtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=3103-3107&author=B.+Dullaauthor=K.+T.+Kirlaauthor=V.+Rathoreauthor=G.+S.+Deoraauthor=S.+Kavelaauthor=S.+Maddikaauthor=K.+Chattiauthor=O.+Reiserauthor=J.+Iqbalauthor=M.+Pal&title=Synthesis+and+evaluation+of+3-amino%2Fguanidine+substituted+phenyl+oxazoles+as+a+novel+class+of+LSD1+inhibitors+with+anti-proliferative+properties&doi=10.1039%2Fc3ob40217g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of 3-amino/guanidine substituted phenyl oxazoles as a novel class of LSD1 inhibitors with anti-proliferative properties</span></div><div class="casAuthors">Dulla, Balakrishna; Kirla, Krishna Tulasi; Rathore, Vandana; Deora, Girdhar Singh; Kavela, Sridhar; Maddika, Subbareddy; Chatti, Kiranam; Reiser, Oliver; Iqbal, Javed; Pal, Manojit</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3103-3107</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of functionalized Ph oxazole derivs. was designed, synthesized and screened in vitro for their activities against LSD1 and for effects on viability of cervical and breast cancer cells, and in vivo for effects using zebrafish embryos.  These compds. are likely to act via multiple epigenetic mechanisms specific to cancer cells including LSD1 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNO6bAB1J-vLVg90H21EOLACvtfcHk0ljWe787smJKLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtlCgtLo%253D&md5=25274d1a41ee948934915c9ca004a884</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2Fc3ob40217g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob40217g%26sid%3Dliteratum%253Aachs%26aulast%3DDulla%26aufirst%3DB.%26aulast%3DKirla%26aufirst%3DK.%2BT.%26aulast%3DRathore%26aufirst%3DV.%26aulast%3DDeora%26aufirst%3DG.%2BS.%26aulast%3DKavela%26aufirst%3DS.%26aulast%3DMaddika%26aufirst%3DS.%26aulast%3DChatti%26aufirst%3DK.%26aulast%3DReiser%26aufirst%3DO.%26aulast%3DIqbal%26aufirst%3DJ.%26aulast%3DPal%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%25203-amino%252Fguanidine%2520substituted%2520phenyl%2520oxazoles%2520as%2520a%2520novel%2520class%2520of%2520LSD1%2520inhibitors%2520with%2520anti-proliferative%2520properties%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D3103%26epage%3D3107%26doi%3D10.1039%2Fc3ob40217g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. K.</span>; <span class="NLM_string-name">Kanouni, T.</span>; <span class="NLM_string-name">Kaldor, S. W.</span>; <span class="NLM_string-name">Stafford, J. A.</span>; <span class="NLM_string-name">Veal, J. M.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of lysine specific demethylase-1</span>. WO <span class="NLM_patent">2015168466 A1</span>, November 5, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Y.+K.+Chen&author=T.+Kanouni&author=S.+W.+Kaldor&author=J.+A.+Stafford&author=J.+M.+Veal&title=Inhibitors+of+lysine+specific+demethylase-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BK.%26atitle%3DInhibitors%2520of%2520lysine%2520specific%2520demethylase-1%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremberg, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D.</span></span> <span> </span><span class="NLM_article-title">Development of 5-hydroxypyrazole derivatives as reversible inhibitors of lysine specific demethylase 1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3190</span>– <span class="NLM_lpage">3195</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2017.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28545974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlWmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3190-3195&author=D.+P.+Mouldauthor=U.+Brembergauthor=A.+M.+Jordanauthor=M.+Geitmannauthor=A.+Maiques-Diazauthor=A.+E.+McGonagleauthor=H.+F.+Smallauthor=T.+C.+P.+Somervailleauthor=D.+Ogilvie&title=Development+of+5-hydroxypyrazole+derivatives+as+reversible+inhibitors+of+lysine+specific+demethylase+1&doi=10.1016%2Fj.bmcl.2017.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Development of 5-hydroxypyrazole derivatives as reversible inhibitors of lysine specific demethylase 1</span></div><div class="casAuthors">Mould, Daniel P.; Bremberg, Ulf; Jordan, Allan M.; Geitmann, Matthis; Maiques-Diaz, Alba; McGonagle, Alison E.; Small, Helen F.; Somervaille, Tim C. P.; Ogilvie, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3190-3195</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of reversible inhibitors of lysine specific demethylase 1 (LSD1) with a 5-hydroxypyrazole scaffold have been developed from compd. 7, which was identified from the patent literature.  Surface plasmon resonance (SPR) and biochem. anal. showed it to be a reversible LSD1 inhibitor with an IC50 value of 0.23 μM.  Optimization of this compd. by rational design afforded compds. with Kd values of <10 nM.  In human THP-1 cells, these compds. were found to upregulate the expression of the surrogate cellular biomarker CD86.  Compd. 11p was found to have moderate oral bioavailability in mice suggesting its potential for use as an in vivo tool compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6uEfijQ4G3rVg90H21EOLACvtfcHk0ljWe787smJKLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlWmt7k%253D&md5=2b4d825ef43c843b1436137a64d6caaf</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DBremberg%26aufirst%3DU.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSmall%26aufirst%3DH.%2BF.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DOgilvie%26aufirst%3DD.%26atitle%3DDevelopment%2520of%25205-hydroxypyrazole%2520derivatives%2520as%2520reversible%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3190%26epage%3D3195%26doi%3D10.1016%2Fj.bmcl.2017.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stansfield, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veal, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design and discovery of potent and selective lysine-specific demethylase 1 (LSD1) inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2018.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30409536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFanur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=103-106&author=Z.+Nieauthor=L.+Shiauthor=C.+Laiauthor=C.+Severinauthor=J.+Xuauthor=J.+R.+Del+Rosarioauthor=R.+K.+Stansfieldauthor=R.+W.+Choauthor=T.+Kanouniauthor=J.+M.+Vealauthor=J.+A.+Staffordauthor=Y.+K.+Chen&title=Structure-based+design+and+discovery+of+potent+and+selective+lysine-specific+demethylase+1+%28LSD1%29+inhibitors&doi=10.1016%2Fj.bmcl.2018.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and discovery of potent and selective lysine-specific demethylase 1 (LSD1) inhibitors</span></div><div class="casAuthors">Nie, Zhe; Shi, Lihong; Lai, Chon; Severin, Christophe; Xu, Jiangchun; Del Rosario, Joselyn R.; Stansfield, Ryan K.; Cho, Robert W.; Kanouni, Toufike; Veal, James M.; Stafford, Jeffrey A.; Chen, Young K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-106</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The histone demethylase LSD1 is a key enzyme in the epigenetic regulation of gene transcription.  Here we present our efforts to discover small mol. reversible inhibitors of LSD1 as an attractive approach to treat hematol. malignancies and certain solid tumors.  Using structure-based drug design, we designed and synthesized a novel series of heteroarom. imidazole inhibitors that demonstrate potent inhibition of the demethylase activity and low nanomolar cell-based activity.  This novel LSD1 inhibitor series was further optimized by attenuating the hERG inhibition and improving oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJSb7MdQzvUrVg90H21EOLACvtfcHk0liXm9nN3vel8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFanur3M&md5=5185217eb3df25eebda16800bccf4419</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DSeverin%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDel%2BRosario%26aufirst%3DJ.%2BR.%26aulast%3DStansfield%26aufirst%3DR.%2BK.%26aulast%3DCho%26aufirst%3DR.%2BW.%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DVeal%26aufirst%3DJ.%2BM.%26aulast%3DStafford%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DY.%2BK.%26atitle%3DStructure-based%2520design%2520and%2520discovery%2520of%2520potent%2520and%2520selective%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D103%26epage%3D106%26doi%3D10.1016%2Fj.bmcl.2018.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremberg, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of 4-(pyrrolidin-3-yl)benzonitrile derivatives as inhibitors of lysine specific demethylase 1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4755</span>– <span class="NLM_lpage">4759</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2017.08.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28927796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWgurjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=4755-4759&author=D.+P.+Mouldauthor=U.+Brembergauthor=A.+M.+Jordanauthor=M.+Geitmannauthor=A.+E.+McGonagleauthor=T.+C.+P.+Somervailleauthor=G.+J.+Spencerauthor=D.+J.+Ogilvie&title=Development+and+evaluation+of+4-%28pyrrolidin-3-yl%29benzonitrile+derivatives+as+inhibitors+of+lysine+specific+demethylase+1&doi=10.1016%2Fj.bmcl.2017.08.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of 4-(pyrrolidin-3-yl)benzonitrile derivatives as inhibitors of lysine specific demethylase 1</span></div><div class="casAuthors">Mould, Daniel P.; Bremberg, Ulf; Jordan, Allan M.; Geitmann, Matthis; McGonagle, Alison E.; Somervaille, Tim C. P.; Spencer, Gary J.; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4755-4759</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As part of our ongoing efforts to develop reversible inhibitors of LSD1, we identified a series of 4-(pyrrolidin-3-yl)benzonitrile derivs. that act as successful scaffold-hops of the literature inhibitor GSK-690.  The most active compd., I, demonstrated a Kd value of 22 nM and a biochem. IC50 of 57 nM.  In addn., this compd. displayed improved selectivity over the hERG ion channel compared to GSK-690, and no activity against the related enzymes MAO-A and B.  In human THP-1 acute myeloid leukemia cells, I was found to increase the expression of the surrogate cellular biomarker CD86.  This work further demonstrates the versatility of scaffold-hopping as a method to develop structurally diverse, potent inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriYqPorIJKgLVg90H21EOLACvtfcHk0liXm9nN3vel8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWgurjF&md5=a2cf57c70017d8b08dd9d6c23fd5b0c4</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DBremberg%26aufirst%3DU.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%25204-%2528pyrrolidin-3-yl%2529benzonitrile%2520derivatives%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D4755%26epage%3D4759%26doi%3D10.1016%2Fj.bmcl.2017.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jandausch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forne, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imhof, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirfel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span> <span> </span><span class="NLM_article-title">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">2704</span>– <span class="NLM_lpage">2709</span>, <span class="refDoi"> DOI: 10.1002/ijc.27555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fijc.27555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=22447389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2704-2709&author=D.+Willmannauthor=S.+Limauthor=S.+Wetzelauthor=E.+Metzgerauthor=A.+Jandauschauthor=W.+Wilkauthor=M.+Jungauthor=I.+Forneauthor=A.+Imhofauthor=A.+Janzerauthor=J.+Kirfelauthor=H.+Waldmannauthor=R.+Schuleauthor=R.+Buettner&title=Impairment+of+prostate+cancer+cell+growth+by+a+selective+and+reversible+lysine-specific+demethylase+1+inhibitor&doi=10.1002%2Fijc.27555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span></div><div class="casAuthors">Willmann, Dominica; Lim, Soyoung; Wetzel, Stefan; Metzger, Eric; Jandausch, Anett; Wilk, Wolfram; Jung, Manfred; Forne, Ignasi; Imhof, Axel; Janzer, Andreas; Kirfel, Jutta; Waldmann, Herbert; Schuele, Roland; Buettner, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2704-2709</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Post-translational modifications of histones by chromatin modifying enzymes regulate chromatin structure and gene expression.  As deregulation of histone modifications contributes to cancer progression, inhibition of chromatin modifying enzymes such as histone demethylases is an attractive therapeutic strategy to impair cancer growth.  Lysine-specific demethylase 1 (LSD1) removes mono- and di-Me marks from lysine 4 or 9 of histone H3.  LSD1 in assocn. with the androgen receptor (AR) controls androgen-dependent gene expression and prostate tumor cell proliferation, thus highlighting LSD1 as a drug target.  By combining protein structure similarity clustering and in vitro screening, we identified Namoline, a γ-pyrone, as a novel, selective and reversible LSD1 inhibitor.  Namoline blocks LSD1 demethylase activity in vitro and in vivo.  Inhibition of LSD1 by Namoline leads to silencing of AR-regulated gene expression and severely impairs androgen-dependent proliferation in vitro and in vivo.  Thus, Namoline is a novel promising starting compd. for the development of therapeutics to treat androgen-dependent prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKxYi0f1zoOrVg90H21EOLACvtfcHk0liXm9nN3vel8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D&md5=3104057823357b199b635e1a22071b2a</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fijc.27555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27555%26sid%3Dliteratum%253Aachs%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DJandausch%26aufirst%3DA.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DForne%26aufirst%3DI.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DImpairment%2520of%2520prostate%2520cancer%2520cell%2520growth%2520by%2520a%2520selective%2520and%2520reversible%2520lysine-specific%2520demethylase%25201%2520inhibitor%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26spage%3D2704%26epage%3D2709%26doi%3D10.1002%2Fijc.27555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Baicalin, a natural LSD1 inhibitor</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2016.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2016.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27814566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOqsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=129-131&author=Y.+C.+Zhengauthor=D.+D.+Shenauthor=M.+Renauthor=X.+Q.+Liuauthor=Z.+R.+Wangauthor=Y.+Liuauthor=Q.+N.+Zhangauthor=L.+J.+Zhaoauthor=L.+J.+Zhaoauthor=J.+L.+Maauthor=B.+Yuauthor=H.+M.+Liu&title=Baicalin%2C+a+natural+LSD1+inhibitor&doi=10.1016%2Fj.bioorg.2016.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Baicalin, a natural LSD1 inhibitor</span></div><div class="casAuthors">Zheng, Yi-Chao; Shen, Dan-Dan; Ren, Meng; Liu, Xue-Qi; Wang, Zhi-Ru; Liu, Ying; Zhang, Qian-Na; Zhao, Li-Juan; Zhao, Li-Jie; Ma, Jin-Lian; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">129-131</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Baicalin is one of the active ingredients in the skullcap, with a variety of pharmacol. effects, such as blood pressure redn., sedation, liver-protection, gallbladder-protection, antibacteria, and anti-inflammation.  In the study, baicalin was first characterized as a LSD1 inhibitor with an IC50 of 3.01 μM and showed strong LSD1 inhibitory effect in cells.  Baicalin may serve as a template for designing flavone-based LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsRTkjAeKcWbVg90H21EOLACvtfcHk0lhey-ulOmF2Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOqsbnK&md5=0fef2e4a12b491f203d24f4a23823b80</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2016.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2016.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DRen%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BQ.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%2BN.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DBaicalin%252C%2520a%2520natural%2520LSD1%2520inhibitor%26jtitle%3DBioorg.%2520Chem.%26date%3D2016%26volume%3D69%26spage%3D129%26epage%3D131%26doi%3D10.1016%2Fj.bioorg.2016.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span> <span> </span><span class="NLM_article-title">Bioactivity-guided cut countercurrent chromatography for isolation of lysine-specific demethylase 1 inhibitors from Scutellaria baicalensis Georgi</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1016</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.aca.2018.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.aca.2018.01.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29534805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1016&publication_year=2018&pages=59-68&author=C.+Hanauthor=S.+Wangauthor=Z.+Liauthor=C.+Chenauthor=J.+Houauthor=D.+Xuauthor=R.+Wangauthor=Y.+Linauthor=J.+Luoauthor=L.+Kong&title=Bioactivity-guided+cut+countercurrent+chromatography+for+isolation+of+lysine-specific+demethylase+1+inhibitors+from+Scutellaria+baicalensis+Georgi&doi=10.1016%2Fj.aca.2018.01.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivity-guided cut countercurrent chromatography for isolation of lysine-specific demethylase 1 inhibitors from Scutellaria baicalensis Georgi</span></div><div class="casAuthors">Han, Chao; Wang, Shanshan; Li, Zhongrui; Chen, Chen; Hou, Jiqin; Xu, Dingqiao; Wang, Ruizhi; Lin, Yaolan; Luo, Jianguang; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1016</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Countercurrent chromatog. (CCC) has gradually become a widely used method for preparative sepn. of bioactive natural mols.  These mols. generally contain distinct scaffolds and characteristics that cannot be readily isolated from plants.  While one-dimensional CCC is typically used for the initial purifn. with insufficiently resolved peaks after locating bioactive components, two-dimensional (2D) or multi-dimensional CCC strategies are employed to improve the resoln. of peaks.  However, these methods usually present certain disadvantages, such as complicated procedures and increased time consumption, exptl. costs, and equipment requirements.  Here, a bioactivity-guided cut CCC strategy was established to isolate lysine-specific demethylase 1 (LSD1) inhibitors from Scutellaria baicalensis Georgi.  Gradient-elution CCC coupled with real-time detection of LSD1 inhibition by the collected fractions was developed.  Next, an online-storage recycling CCC mode was designed to enable the active fractions to be stored in coils, and these active fractions were further sepd. to obtain pure compds. by using sequential recycling elution.  In this strategy, active fractions are first identified, and then pure LSD1 inhibitors are isolated in the 2D CCC mode through continuous sepn. on a single instrument.  By using our bioactivity-guided cut CCC strategy, we successfully isolated six natural LSD1 inhibitors from S. baicalensis Georgi, five of which were identified for the first time as natural LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHMApE4g-b17Vg90H21EOLACvtfcHk0lhey-ulOmF2Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaqsbs%253D&md5=862344f018da2cf3a708e3476e5e5d6b</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2018.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2018.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DL.%26atitle%3DBioactivity-guided%2520cut%2520countercurrent%2520chromatography%2520for%2520isolation%2520of%2520lysine-specific%2520demethylase%25201%2520inhibitors%2520from%2520Scutellaria%2520baicalensis%2520Georgi%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2018%26volume%3D1016%26spage%3D59%26epage%3D68%26doi%3D10.1016%2Fj.aca.2018.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span> <span> </span><span class="NLM_article-title">Flavone-based natural product agents as new lysine-specific demethylase 1 inhibitors exhibiting cytotoxicity against breast cancer cells in vitro</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">370</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2018.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30552007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyksL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=370-374&author=X.+Xuauthor=W.+Pengauthor=C.+Liuauthor=S.+Liauthor=J.+Leiauthor=Z.+Wangauthor=L.+Kongauthor=C.+Han&title=Flavone-based+natural+product+agents+as+new+lysine-specific+demethylase+1+inhibitors+exhibiting+cytotoxicity+against+breast+cancer+cells+in+vitro&doi=10.1016%2Fj.bmc.2018.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Flavone-based natural product agents as new lysine-specific demethylase 1 inhibitors exhibiting cytotoxicity against breast cancer cells in vitro</span></div><div class="casAuthors">Xu, Xiao; Peng, Wenhui; Liu, Cuiyun; Li, Sixuan; Lei, Jiali; Wang, Zhen; Kong, Lingyi; Han, Chao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">370-374</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) has recently emerged as a therapeutic target for cancer.  However, almost all LSD1 inhibitors developed to date are chemo-synthesized mols.  In this study, the LSD1 inhibitory activity of 12 natural flavones, including four aglycons and their corresponding monoglycosides and diglucosides, was evaluated.  Based on the structure-activity relationships, LSD1 inhibition activity was greater for flavonoid monoglycosides than their aglycons lacking the sugar moiety.  The effects of isoquercitrin, which exhibited optimal LSD1 inhibitory activity, on cancer cell properties were evaluated.  Isoquercitrin induced the expression of key proteins in the mitochondrial-mediated apoptosis pathway and caused apoptosis in LSD1-overexpressing MDA-MB-231 cells via the inhibition of LSD1.  These findings suggest that natural LSD1 inhibitors, and particularly isoquercitrin, are promising for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoslK-6yseLw7Vg90H21EOLACvtfcHk0lgCfz77xi69sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyksL7M&md5=2b21cf4cc599cd1a8ae85d7460754cf5</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DKong%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DC.%26atitle%3DFlavone-based%2520natural%2520product%2520agents%2520as%2520new%2520lysine-specific%2520demethylase%25201%2520inhibitors%2520exhibiting%2520cytotoxicity%2520against%2520breast%2520cancer%2520cells%2520in%2520vitro%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D370%26epage%3D374%26doi%3D10.1016%2Fj.bmc.2018.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Q. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Q.</span></span> <span> </span><span class="NLM_article-title">Identification and biological evaluation of natural product Biochanin A</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">103674</span>– <span class="NLM_lpage">103679</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.103674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2020.103674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32097796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjvVKjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2020&pages=103674-103679&author=L.+Wangauthor=L.+Z.+Liauthor=Q.+X.+Hanauthor=X.+F.+Wangauthor=D.+Zhaoauthor=J.+Q.+Liu&title=Identification+and+biological+evaluation+of+natural+product+Biochanin+A&doi=10.1016%2Fj.bioorg.2020.103674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and biological evaluation of natural product Biochanin A</span></div><div class="casAuthors">Wang, Lei; Li, Lingzhao; Han, Quanxiang; Wang, Xiaofang; Zhao, Di; Liu, Junqi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103674</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Natural products have shown promise for epigenetic modulations and thus are therapeutically potential for cancer prevention and treatment.  In this work, we report the identification of natural product Biochanin A as a new LSD1 inhibitor and further biol. evaluation in gastric MGC-803 cells.  Biochanin A effectively and reversibly inhibited LSD1 (IC50 = 2.95μM) and showed selective inhibition to LSD1 over MAO-A/B.  In gastric MGC-803 cells, Biochanin A induced accumulation of H3K4me1/2, inhibited cell growth moderately (IC50 = 6.77μM), but was less toxic to normal GES-1 (IC50 > 32μM).  Mechanistic studies showed that Biochanin A suppressed colony formation, cell apoptosis, and migration of MGC-803 cells.  This study may help to elucidate the anticancer mechanisms of Biochanin A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnWmRkIH8HF7Vg90H21EOLACvtfcHk0lgCfz77xi69sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjvVKjs7c%253D&md5=b0ff6f75858691d6b21d6a4b6d6f3a4e</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.103674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.103674%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%2BZ.%26aulast%3DHan%26aufirst%3DQ.%2BX.%26aulast%3DWang%26aufirst%3DX.%2BF.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26atitle%3DIdentification%2520and%2520biological%2520evaluation%2520of%2520natural%2520product%2520Biochanin%2520A%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D97%26spage%3D103674%26epage%3D103679%26doi%3D10.1016%2Fj.bioorg.2020.103674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M. S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">112243</span>– <span class="NLM_lpage">112262</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2020.112243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32229389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtFOmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2020&pages=112243-112262&author=X.+R.+Wangauthor=C.+Zhangauthor=X.+Y.+Zhangauthor=J.+K.+Yanauthor=J.+M.+Wangauthor=Q.+W.+Jiangauthor=L.+Y.+Zhaoauthor=D.+M.+Zhaoauthor=M.+S.+Cheng&title=Design%2C+synthesis+and+biological+evaluation+of+tetrahydroquinoline-based+reversible+LSD1+inhibitors&doi=10.1016%2Fj.ejmech.2020.112243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors</span></div><div class="casAuthors">Wang, Xinran; Zhang, Cai; Zhang, Xiangyu; Yan, Jiangkun; Wang, Jiming; Jiang, Qinwen; Zhao, Liyu; Zhao, Dongmei; Cheng, Maosheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112243</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The targeted regulation of LSD1, which is highly expressed in a variety of tumor cells, is a promising cancer therapy strategy.  Several LSD1 inhibitors are currently under clin. evaluation, and most of these inhibitors are irreversible.  Here, the design, synthesis and biochem. evaluation of novel tetrahydroquinoline-based reversible LSD1 inhibitors has been reported.  Compds. (I) and (II), which are selective to LSD1 over MAO-A/B, exhibit excellent LSD1 inhibition at the mol. levels with IC50= 55 nM and 540 nM, resp.  The classic Lineweaver-Burk plots revealed that compd. I could reversibly bind the LSD1 enzyme in a noncompetitive manner.  Mol. docking was used to reveal the potential binding-mode of the compds. and interpret the structure-activity relationships.  Furthermore, compds. I and II significantly inhibited proliferation (IC50= 1.13μM and 1.15μM, resp.) and induced apoptosis in MGC-803 cells with high expression of LSD1.  Compd. IIshowed acceptable liver microsomal stability.  Meanwhile, II did not appear to inhibit CYPs at 10μM in vitro.  Remarkably, the oral administration of compd. II can inhibit the growth of MGC-803 xenograft tumors without significant side effects.  Author findings suggest that tetrahydroquinoline-based LSD1 inhibitors deserve further investigation for the treatment of LSD1 overexpressing cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouUACUV9d8o7Vg90H21EOLACvtfcHk0lguX3qfXYDKUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtFOmurg%253D&md5=83fc46cdb95e5833d0c17fd6b0709ebd</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112243%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%2BR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%2BY.%26aulast%3DYan%26aufirst%3DJ.%2BK.%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DJiang%26aufirst%3DQ.%2BW.%26aulast%3DZhao%26aufirst%3DL.%2BY.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520tetrahydroquinoline-based%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D194%26spage%3D112243%26epage%3D112262%26doi%3D10.1016%2Fj.ejmech.2020.112243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of [1,2,3]triazolo[4,5-d]pyrimidine derivatives as novel LSD1 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00423</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00423" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjslyju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=384-389&author=Z.+H.+Liauthor=X.+Q.+Liuauthor=P.+F.+Gengauthor=F.+Z.+Suoauthor=J.+L.+Maauthor=B.+Yuauthor=T.+Q.+Zhaoauthor=Z.+Q.+Zhouauthor=C.+X.+Huangauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Discovery+of+%5B1%2C2%2C3%5Dtriazolo%5B4%2C5-d%5Dpyrimidine+derivatives+as+novel+LSD1+inhibitors&doi=10.1021%2Facsmedchemlett.6b00423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of [1,2,3]Triazolo[4,5-d]pyrimidine Derivatives as Novel LSD1 Inhibitors</span></div><div class="casAuthors">Li, Zhong-Hua; Liu, Xue-Qi; Geng, Peng-Fei; Suo, Feng-Zhi; Ma, Jin-Lian; Yu, Bin; Zhao, Tao-Qian; Zhou, Zhao-Qing; Huang, Chen-Xi; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">384-389</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays a pivotal role in regulating the lysine methylation.  The aberrant overexpression of LSD1 was reported to be involved in the progression of certain human malignant tumors.  Abrogation of LSD1 with RNAi or small mol. inhibitors may lead to the inhibition of cancer proliferation and migration.  Herein, a series of [1,2,3]triazolo[4,5-d]pyrimidine derivs. were synthesized and evaluated for their LSD1 inhibitory effects.  The structure-activity relation studies (SARs) were conducted by exploring three regions of this scaffold, leading to the discovery of compd. 27 as potent LSD1 inhibitor (IC50 = 0.564 μM).  Compd. 27 was identified as a reversible LSD1 inhibitor and showed certain selectivity to LSD1 over monoamine oxidase A/B (MAO-A/B).  When MGC-803 cells were treated with compd. 27, the activity of LSD1 can be significantly inhibited, and the cell migration ability was also suppressed.  Docking studies indicated that the hydrogen interaction between the nitrogen atom in the pyridine ring and Met332 could be responsible for the improved activity of 2-thiopyridine series.  The [1,2,3]triazolo[4,5-d]pyrimidine scaffold can be used as the template for designing new LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeTW4tbCivwLVg90H21EOLACvtfcHk0liJpNTsQKxvWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjslyju74%253D&md5=dc8bb933eebb850e165663d6de2b8270</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00423%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BH.%26aulast%3DLiu%26aufirst%3DX.%2BQ.%26aulast%3DGeng%26aufirst%3DP.%2BF.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%2BQ.%26aulast%3DZhou%26aufirst%3DZ.%2BQ.%26aulast%3DHuang%26aufirst%3DC.%2BX.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520%255B1%252C2%252C3%255Dtriazolo%255B4%252C5-d%255Dpyrimidine%2520derivatives%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D384%26epage%3D389%26doi%3D10.1021%2Facsmedchemlett.6b00423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Development of the triazole-fused pyrimidine derivatives as highly potent and reversible inhibitors of histone lysine specific demethylase 1 (LSD1/KDM1A)</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2019.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.apsb.2019.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31384539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BB3MvltFyitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=794-808&author=Z.+H.+Liauthor=L.+N.+Dingauthor=Z.+R.+Liauthor=Z.+Z.+Wangauthor=F.+Z.+Suoauthor=D.+D.+Shenauthor=T.+Q.+Zhaoauthor=X.+D.+Sunauthor=J.+W.+Wangauthor=Y.+Liuauthor=L.+Y.+Maauthor=B.+Zhaoauthor=P.+F.+Gengauthor=B.+Yuauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Development+of+the+triazole-fused+pyrimidine+derivatives+as+highly+potent+and+reversible+inhibitors+of+histone+lysine+specific+demethylase+1+%28LSD1%2FKDM1A%29&doi=10.1016%2Fj.apsb.2019.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the triazole-fused pyrimidine derivatives as highly potent and reversible inhibitors of histone lysine specific demethylase 1 (LSD1/KDM1A)</span></div><div class="casAuthors">Li Zhonghua; Ding Lina; Li Zhongrui; Wang Zhizheng; Suo Fengzhi; Shen Dandan; Zhao Taoqian; Sun Xudong; Wang Junwei; Liu Ying; Ma Liying; Zhao Bing; Geng Pengfei; Yu Bin; Zheng Yichao; Liu Hongmin; Li Zhonghua; Ding Lina; Li Zhongrui; Wang Zhizheng; Suo Fengzhi; Shen Dandan; Zhao Taoqian; Sun Xudong; Wang Junwei; Liu Ying; Ma Liying; Zhao Bing; Geng Pengfei; Yu Bin; Zheng Yichao; Liu Hongmin; Li Zhonghua; Ding Lina; Li Zhongrui; Wang Zhizheng; Suo Fengzhi; Shen Dandan; Zhao Taoqian; Sun Xudong; Wang Junwei; Liu Ying; Ma Liying; Zhao Bing; Geng Pengfei; Yu Bin; Zheng Yichao; Liu Hongmin; Yu Bin</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">794-808</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) has been recognized as an important modulator in post-translational process in epigenetics.  Dysregulation of LSD1 has been implicated in the development of various cancers.  Herein, we report the discovery of the hit compound 8a (IC50 = 3.93 μmol/L) and further medicinal chemistry efforts, leading to the generation of compound 15u (IC50 = 49 nmol/L, and Ki = 16 nmol/L), which inhibited LSD1 reversibly and competitively with H3K4me2, and was selective to LSD1 over MAO-A/B.  Docking studies were performed to rationalize the potency of compound 15u.  Compound 15u also showed strong antiproliferative activity against four leukemia cell lines (OCL-AML3, K562, THP-1 and U937) as well as the lymphoma cell line Raji with the IC50 values of 1.79, 1.30, 0.45, 1.22 and 1.40 μmol/L, respectively.  In THP-1 cell line, 15u significantly inhibited colony formation and caused remarkable morphological changes.  Compound 15u induced expression of CD86 and CD11b in THP-1 cells, confirming its cellular activity and ability of inducing differentiation.  The findings further indicate that targeting LSD1 is a promising strategy for AML treatment, the triazole-fused pyrimidine derivatives are new scaffolds for the development of LSD1/KDM1A inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT87t7BYwXnlAqdek-gHSrIfW6udTcc2eaKsQGpSq_e6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvltFyitA%253D%253D&md5=9c4f8acdb5e0d95edc99b6e0cab947f2</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2019.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BH.%26aulast%3DDing%26aufirst%3DL.%2BN.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DWang%26aufirst%3DZ.%2BZ.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DZhao%26aufirst%3DT.%2BQ.%26aulast%3DSun%26aufirst%3DX.%2BD.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DGeng%26aufirst%3DP.%2BF.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDevelopment%2520of%2520the%2520triazole-fused%2520pyrimidine%2520derivatives%2520as%2520highly%2520potent%2520and%2520reversible%2520inhibitors%2520of%2520histone%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2019%26volume%3D9%26spage%3D794%26epage%3D808%26doi%3D10.1016%2Fj.apsb.2019.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. Q.</span></span> <span> </span><span class="NLM_article-title">[1,2,3]Triazolo[4,5-d]pyrimidine derivatives incorporating (thio)urea moiety as a novel scaffold for LSD1 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">111989</span>– <span class="NLM_lpage">111997</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2019.111989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31881456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2020&pages=111989-111997&author=Z.+H.+Liauthor=J.+L.+Maauthor=G.+Z.+Liuauthor=X.+H.+Zhangauthor=T.+T.+Qinauthor=W.+H.+Renauthor=T.+Q.+Zhaoauthor=X.+H.+Chenauthor=Z.+Q.+Zhang&title=%5B1%2C2%2C3%5DTriazolo%5B4%2C5-d%5Dpyrimidine+derivatives+incorporating+%28thio%29urea+moiety+as+a+novel+scaffold+for+LSD1+inhibitors&doi=10.1016%2Fj.ejmech.2019.111989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">[1,2,3]Triazolo[4,5-d]pyrimidine derivatives incorporating (thio)urea moiety as a novel scaffold for LSD1 inhibitors</span></div><div class="casAuthors">Li, Zhong-Hua; Ma, Jin-Lian; Liu, Gai-Zhi; Zhang, Xin-Hui; Qin, Ting-Ting; Ren, Wei-Hong; Zhao, Tao-Qian; Chen, Xiao-Hui; Zhang, Zhen-Qiang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111989</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an essential role in maintaining a balanced methylation status at histone tails.  Overexpression of LSD1 has been involved in the development of a variety of human diseases, including cancers.  Herein, on the basis of our previously developed LSD1 inhibitors, two series of new [1,2,3]triazolo[4,5-d]pyrimidine derivs. incorporating (thio)urea moiety were designed and evaluated for their LSD1 inhibitory abilities, leading to a novel chem. class of LSD1 inhibitors.  Among them, compd. 31 was found to moderately inhibit LSD1 activity, as well as increase the expression of H3K4me2 at the cellular level.  This compd. also showed good selectivity against MAO-A/-B, and a panel of kinases such as CDK and BTK.  Besides, the MTT assay suggested that the selected compds. could inhibit the proliferation of LSD1-overexpressed cancer cells.  Although this class of compds. only showed moderate anti-LSD1 activity in the micromolar range, this work presents a novel chemotype of LSD1 inhibitors with good enzyme selectivity as well as cellular LSD1 inhibitory activity, and could provide a useful template for the development of more potent LSD1 inhibitors for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ2CKPL-inPrVg90H21EOLACvtfcHk0lgtgeV1YlgcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfM&md5=a62bf474d109b25ad9a517cccb6d0db2</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111989%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BH.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DLiu%26aufirst%3DG.%2BZ.%26aulast%3DZhang%26aufirst%3DX.%2BH.%26aulast%3DQin%26aufirst%3DT.%2BT.%26aulast%3DRen%26aufirst%3DW.%2BH.%26aulast%3DZhao%26aufirst%3DT.%2BQ.%26aulast%3DChen%26aufirst%3DX.%2BH.%26aulast%3DZhang%26aufirst%3DZ.%2BQ.%26atitle%3D%255B1%252C2%252C3%255DTriazolo%255B4%252C5-d%255Dpyrimidine%2520derivatives%2520incorporating%2520%2528thio%2529urea%2520moiety%2520as%2520a%2520novel%2520scaffold%2520for%2520LSD1%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D187%26spage%3D111989%26epage%3D111997%26doi%3D10.1016%2Fj.ejmech.2019.111989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-a]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2016.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27769034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=940-951&author=S.+Wangauthor=L.+J.+Zhaoauthor=Y.+C.+Zhengauthor=D.+D.+Shenauthor=E.+F.+Miaoauthor=X.+P.+Qiaoauthor=L.+J.+Zhaoauthor=Y.+Liuauthor=R.+Huangauthor=B.+Yuauthor=H.+M.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+%5B1%2C2%2C4%5Dtriazolo%5B1%2C5-a%5Dpyrimidines+as+potent+lysine+specific+demethylase+1+%28LSD1%2FKDM1A%29+inhibitors&doi=10.1016%2Fj.ejmech.2016.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-a]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span></div><div class="casAuthors">Wang, Shuai; Zhao, Li-Jie; Zheng, Yi-Chao; Shen, Dan-Dan; Miao, Er-Fei; Qiao, Xue-Peng; Zhao, Li-Juan; Liu, Ying; Huang, Ruilei; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">940-951</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A new series of [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1 inhibitors was designed, synthesized, and further evaluated for cytotoxicity against MGC-803, EC109, A549 and PC-9 cells as well as the ability of inhibiting LSD1.  Some of these compds. showed potent inhibition toward LSD1 and selectively inhibited growth of A549 and PC-9 cells.  Compd. I potently inhibited growth of PC-9 cells (IC50 = 0.59 μM), about 4-fold more potent than 5-FU.  Further SAR studies led to the identification of compds. I and II, which had good growth inhibition against all the tested cancer cell lines and were much more potent than 5-FU and GSK2879552.  Besides, compds. III-V inhibited LSD1 potently (IC50 = 0.154, 1.19 and 0.557 μM, resp.).  Docking studies revealed that compd. III formed arene-H interactions with Val333 and hydrogen bonds with surrounding Ala331, Met332, and Ala539 residues.  Compd. III significantly inhibited migration of A549 and PC-9 cells in a concn.-dependent manner, but had different effect on the expression of E-cadherin and N-cadherin.  The [1,2,4]triazolo[1,5-a]pyrimidine scaffold may serve as a starting point for developing potent LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0jqW-eLBQObVg90H21EOLACvtfcHk0lgtgeV1YlgcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ&md5=d022678613486890faacfef9c163492b</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DMiao%26aufirst%3DE.%2BF.%26aulast%3DQiao%26aufirst%3DX.%2BP.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520%255B1%252C2%252C4%255Dtriazolo%255B1%252C5-a%255Dpyrimidines%2520as%2520potent%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D125%26spage%3D940%26epage%3D951%26doi%3D10.1016%2Fj.ejmech.2016.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.02.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2019.02.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30780087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Wkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=388-401&author=S.+Wangauthor=Z.+R.+Liauthor=F.+Z.+Suoauthor=X.+H.+Yuanauthor=B.+Yuauthor=H.+M.+Liu&title=Synthesis%2C+structure-activity+relationship+studies+and+biological+characterization+of+new+%5B1%2C2%2C4%5Dtriazolo%5B1%2C5-a%5Dpyrimidine-based+LSD1%2FKDM1A+inhibitors&doi=10.1016%2Fj.ejmech.2019.02.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors</span></div><div class="casAuthors">Wang, Shuai; Li, Zhong-Rui; Suo, Feng-Zhi; Yuan, Xiao-Han; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">388-401</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The design, synthesis and biochem. characterization of [1,2,4]triazolo[1,5-a]pyrimidine derivs. I [R1 = H, [(2-bromophenyl)methyl]sulfanyl, prop-2-en-1-ylsulfanyl, [(1H-1,3-benzodiazol-2-ylmethyl)sulfanyl], etc.; R2 = H, Me, (CH2)4CH3; R3 = Me, Et, C6H5; R2, R3 = -(CH2)3-; R4 = H, C6H5, [4-(4-methylpiperazin-1-yl)phenyl], etc.] as new LSD1 inhibitors have been reported.  Of these compds., compd. I [R1 = (1H-1,3-benzodiazol-2-ylsulfanyl)methyl; R2 = H; R3 = Me; R4 = [4-(4-methylpiperazin-1-yl)phenyl]] (II) inhibited LSD1 in a reversible manner (IC50 = 1.72 μM) and showed selectivity to LSD1 over MAO-A/B.  Besides, compd. II displayed FAD-competitive binding to LSD1.  Interestingly, compd. II did not inhibit horseradish peroxidase (HRP) and quench H2O2, thus excluding the possibility that LSD1 inhibition by compd. II was due to the HRP inhibition and consumption of H2O2.  In LSD1 overexpressed A549 cells, compd. II concn.-dependently induced accumulation of H3K4me1/me2 and H3K9me2 and showed cellular target engagement to LSD1.  Addnl., compd. II significantly inhibited migration of A549 cells in a concn.-dependent manner, further western blot anal. showed that compd. II increased expression levels of epithelial cell markers E-Cadherin and Claudin-1, down-regulated mesenchymal cell marker N-Cadherin and the upstream transcription factors Snail and Slug.  Docking studies were also performed to rationalize the potency of compd. II toward LSD1.  To conclude, the [1,2,4]triazolo[1,5-a]pyrimidine I could serve as a promising scaffold for the development of new LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFGjGReI1_PLVg90H21EOLACvtfcHk0lgtgeV1YlgcJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Wkurk%253D&md5=a5ce8c3d2239535b47bf93cf20c9c5c3</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DYuan%26aufirst%3DX.%2BH.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DSynthesis%252C%2520structure-activity%2520relationship%2520studies%2520and%2520biological%2520characterization%2520of%2520new%2520%255B1%252C2%252C4%255Dtriazolo%255B1%252C5-a%255Dpyrimidine-based%2520LSD1%252FKDM1A%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401%26doi%3D10.1016%2Fj.ejmech.2019.02.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Experience-based discovery (EBD) of aryl hydrazines as new scaffolds for the development of LSD1/KDM1A inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2019.01.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30739825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivFygtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2019&pages=432-444&author=Z.+R.+Liauthor=S.+Wangauthor=L.+Yangauthor=X.+H.+Yuanauthor=F.+Z.+Suoauthor=B.+Yuauthor=H.+M.+Liu&title=Experience-based+discovery+%28EBD%29+of+aryl+hydrazines+as+new+scaffolds+for+the+development+of+LSD1%2FKDM1A+inhibitors&doi=10.1016%2Fj.ejmech.2019.01.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Experience-based discovery (EBD) of aryl hydrazines as new scaffolds for the development of LSD1/KDM1A inhibitors</span></div><div class="casAuthors">Li, Zhong-Rui; Wang, Shuai; Yang, Linlin; Yuan, Xiao-Han; Suo, Feng-Zhi; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">432-444</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Phenelzine was first employed to design new aryl hydrazine-based LSD1 inhibitors based on the experience-based discovery (EBD) strategy.  Among these compds., D8 potently inhibited LSD1 (IC50 = 882.30 nM) in a reversible manner.  Compd. D8 was selective to LSD1 over MAO-A/B and showed H3K4me2 competitive binding to LSD1.  The interaction between H3K4me2 and LSD1 was also confirmed by the Co-IP assay.  In LSD1 overexpressed A549 cells, compd. D8 dose-dependently induced accumulation of LSD1 substrates H3K4me1/2 and H3K9me1/2, showed cellular target engagement to LSD1 and significantly inhibited cell migration of A549 cells.  Docking studies suggested that compd. D8 occupied the peptide binding region and therefore blocked the access of the peptide substrate to the FAD, finally leading to the demethylase activity inhibition of LSD1.  The findings indicate that aryl hydrazines are new scaffolds for the design of LSD1 inhibitors, the identification of D8 provides further evidence for our previously proposed general principle that fused heterocycles with an amine group are potentially active toward LSD1 by competitive binding to LSD1 with H3 peptide substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGH1kwCayTurVg90H21EOLACvtfcHk0ljG3iHGP27iQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivFygtL4%253D&md5=18d5b4e7e51f6a911f0e07bb5ed955be</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.075%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYuan%26aufirst%3DX.%2BH.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DExperience-based%2520discovery%2520%2528EBD%2529%2520of%2520aryl%2520hydrazines%2520as%2520new%2520scaffolds%2520for%2520the%2520development%2520of%2520LSD1%252FKDM1A%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D166%26spage%3D432%26epage%3D444%26doi%3D10.1016%2Fj.ejmech.2019.01.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coto-Montes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boga, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vriend, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span> <span> </span><span class="NLM_article-title">Melatonin: an ancient molecule that makes oxygen metabolically tolerable</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">403</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.1111/jpi.12267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=26272235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyju7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=403-419&author=L.+C.+Manchesterauthor=A.+Coto-Montesauthor=J.+A.+Bogaauthor=L.+P.+Andersenauthor=Z.+Zhouauthor=A.+Galanoauthor=J.+Vriendauthor=D.+X.+Tanauthor=R.+J.+Reiter&title=Melatonin%3A+an+ancient+molecule+that+makes+oxygen+metabolically+tolerable&doi=10.1111%2Fjpi.12267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin: an ancient molecule that makes oxygen metabolically tolerable</span></div><div class="casAuthors">Manchester, Lucien C.; Coto-Montes, Ana; Boga, Jose Antonio; Andersen, Lars Peter H.; Zhou, Zhou; Galano, Annia; Vriend, Jerry; Tan, Dun-Xian; Reiter, Russel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">403-419</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Melatonin is remarkably functionally diverse with actions as a free radical scavenger and antioxidant, circadian rhythm regulator, anti-inflammatory and immunoregulating mol., and as an oncostatic agent.  We hypothesize that the initial and primary function of melatonin in photosynthetic cyanobacteria, which appeared on Earth 3.5-3.2 billion years ago, was as an antioxidant.  The evolution of melatonin as an antioxidant by this organism was necessary as photosynthesis is assocd. with the generation of toxic-free radicals.  The other secondary functions of melatonin came about much later in evolution.  We also surmise that mitochondria and chloroplasts may be primary sites of melatonin synthesis in all eukaryotic cells that possess these organelles.  This prediction is made on the basis that mitochondria and chloroplasts of eukaryotes developed from purple nonsulfur bacteria (which also produce melatonin) and cyanobacteria when they were engulfed by early eukaryotes.  Thus, we speculate that the melatonin-synthesizing actions of the engulfed bacteria were retained when these organelles became mitochondria and chloroplasts, resp.  That mitochondria are likely sites of melatonin formation is supported by the observation that this organelle contains high levels of melatonin that are not impacted by blood melatonin concns.  Melatonin has a remarkable array of means by which it thwarts oxidative damage.  It, as well as its metabolites, is differentially effective in scavenging a variety of reactive oxygen and reactive nitrogen species.  Moreover, melatonin and its metabolites modulate a large no. of antioxidative and pro-oxidative enzymes, leading to a redn. in oxidative damage.  The actions of melatonin on radical metabolizing/producing enzymes may be mediated by the Keap1-Nrf2-ARE pathway.  Beyond its direct free radical scavenging and indirect antioxidant effects, melatonin has a variety of physiol. and metabolic advantages that may enhance its ability to limit oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosyCdaNdqJYLVg90H21EOLACvtfcHk0ljG3iHGP27iQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyju7fI&md5=b086220e459573991456ef35b91d7cc6</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12267%26sid%3Dliteratum%253Aachs%26aulast%3DManchester%26aufirst%3DL.%2BC.%26aulast%3DCoto-Montes%26aufirst%3DA.%26aulast%3DBoga%26aufirst%3DJ.%2BA.%26aulast%3DAndersen%26aufirst%3DL.%2BP.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DGalano%26aufirst%3DA.%26aulast%3DVriend%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DD.%2BX.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26atitle%3DMelatonin%253A%2520an%2520ancient%2520molecule%2520that%2520makes%2520oxygen%2520metabolically%2520tolerable%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D59%26spage%3D403%26epage%3D419%26doi%3D10.1111%2Fjpi.12267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span> <span> </span><span class="NLM_article-title">Melatonin induces apoptosis of colorectal cancer cells through HDAC4 nuclear import mediated by CaMKII inactivation</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">438</span>, <span class="refDoi"> DOI: 10.1111/jpi.12226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25752481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=429-438&author=J.+Y.+Weiauthor=W.+M.+Liauthor=L.+L.+Zhouauthor=Q.+N.+Luauthor=W.+He&title=Melatonin+induces+apoptosis+of+colorectal+cancer+cells+through+HDAC4+nuclear+import+mediated+by+CaMKII+inactivation&doi=10.1111%2Fjpi.12226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin induces apoptosis of colorectal cancer cells through HDAC4 nuclear import mediated by CaMKII inactivation</span></div><div class="casAuthors">Wei, Jia-Yi; Li, Wan-Ming; Zhou, Lin-Lin; Lu, Qiu-Nan; He, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-438</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Melatonin induces apoptosis in many different cancer cell lines, including colorectal cancer.  However, the precise mechanisms involved remain largely unresolved.  In this study, we provide evidence to reveal a new mechanism by which melatonin induces apoptosis of colorectal cancer LoVo cells.  Melatonin at pharmacol. concns. significantly suppressed cell proliferation and induced apoptosis in a dose-dependent manner.  The obsd. apoptosis was accompanied by the melatonin-induced dephosphorylation and nuclear import of histone deacetylase 4 (HDAC4).  Pretreatment with a HDAC4-specific siRNA effectively attenuated the melatonin-induced apoptosis, indicating that nuclear localization of HDAC4 is required for melatonin-induced apoptosis.  Moreover, constitutively active Ca2+/calmodulin-dependent protein kinase II alpha (CaMKIIα) abrogated the melatonin-induced HDAC4 nuclear import and apoptosis of LoVo cells.  Furthermore, melatonin decreased H3 acetylation on bcl-2 promoter, leading to a redn. of bcl-2 expression, whereas constitutively active CaMKIIα(T286D) or HDAC4-specific siRNA abrogated the effect of melatonin.  In conclusion, the present study provides evidence that melatonin-induced apoptosis in colorectal cancer LoVo cells largely depends on the nuclear import of HDAC4 and subsequent H3 deacetylation via the inactivation of CaMKIIα.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqZKAF9ddSo7Vg90H21EOLACvtfcHk0ljG3iHGP27iQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb4%253D&md5=90bcaac43d977d9e8203437697b72c1b</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12226%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%2BY.%26aulast%3DLi%26aufirst%3DW.%2BM.%26aulast%3DZhou%26aufirst%3DL.%2BL.%26aulast%3DLu%26aufirst%3DQ.%2BN.%26aulast%3DHe%26aufirst%3DW.%26atitle%3DMelatonin%2520induces%2520apoptosis%2520of%2520colorectal%2520cancer%2520cells%2520through%2520HDAC4%2520nuclear%2520import%2520mediated%2520by%2520CaMKII%2520inactivation%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D58%26spage%3D429%26epage%3D438%26doi%3D10.1111%2Fjpi.12226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauchy, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wren, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belancio, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blask, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S. M.</span></span> <span> </span><span class="NLM_article-title">Doxorubicin resistance in breast cancer is driven by light at night-induced disruption of the circadian melatonin signal</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1111/jpi.12239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25857269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslensrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=60-69&author=S.+L.+Xiangauthor=R.+T.+Dauchyauthor=A.+Hauchauthor=L.+L.+Maoauthor=L.+Yuanauthor=M.+A.+Wrenauthor=V.+P.+Belancioauthor=D.+Mondalauthor=T.+Fraschauthor=D.+E.+Blaskauthor=S.+M.+Hill&title=Doxorubicin+resistance+in+breast+cancer+is+driven+by+light+at+night-induced+disruption+of+the+circadian+melatonin+signal&doi=10.1111%2Fjpi.12239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Doxorubicin resistance in breast cancer is driven by light at night-induced disruption of the circadian melatonin signal</span></div><div class="casAuthors">Xiang, Shulin; Dauchy, Robert T.; Hauch, Adam; Mao, Lulu; Yuan, Lin; Wren, Melissa A.; Belancio, Victoria P.; Mondal, Debasis; Frasch, Tripp; Blask, David E.; Hill, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-69</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Chemotherapeutic resistance, particularly to doxorubicin (Dox), represents a major impediment to successfully treating breast cancer and is linked to elevated tumor metab. and tumor over-expression and/or activation of various families of receptor- and non-receptor-assocd. tyrosine kinases.  Disruption of circadian time structure and suppression of nocturnal melatonin prodn. by dim light exposure at night (dLEN), as occurs with shift work, and/or disturbed sleep-wake cycles, is assocd. with a significantly increased risk of an array of diseases, including breast cancer.  Melatonin inhibits human breast cancer growth via mechanisms that include the suppression of tumor metab. and inhibition of expression or phospho-activation of the receptor kinases AKT and ERK1/2 and various other kinases and transcription factors.  We demonstrate in tissue-isolated estrogen receptor alpha-pos. (ERα+) MCF-7 human breast cancer xenografts, grown in nude rats maintained on a light/dark cycle of LD 12:12 in which dLEN is present during the dark phase (suppressed endogenous nocturnal melatonin), a significant shortening of tumor latency-to-onset, increased tumor metab. and growth, and complete intrinsic resistance to Dox therapy.  Conversely, a LD 12:12 dLEN environment incorporating nocturnal melatonin replacement resulted in significantly lengthened tumor latency-to-onset, tumor regression, suppression of nighttime tumor metab., and kinase and transcription factor phosphorylation, while Dox sensitivity was completely restored.  Melatonin acts as both a tumor metabolic inhibitor and circadian-regulated kinase inhibitor to reestablish the sensitivity of breast tumors to Dox and drive tumor regression, indicating that dLEN-induced circadian disruption of nocturnal melatonin prodn. contributes to a complete loss of tumor sensitivity to Dox chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM6CZtxM6IO7Vg90H21EOLACvtfcHk0lgWAJ8Ru2Y-QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslensrk%253D&md5=bdbc62e23ad721390984935c19f954a4</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12239%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DS.%2BL.%26aulast%3DDauchy%26aufirst%3DR.%2BT.%26aulast%3DHauch%26aufirst%3DA.%26aulast%3DMao%26aufirst%3DL.%2BL.%26aulast%3DYuan%26aufirst%3DL.%26aulast%3DWren%26aufirst%3DM.%2BA.%26aulast%3DBelancio%26aufirst%3DV.%2BP.%26aulast%3DMondal%26aufirst%3DD.%26aulast%3DFrasch%26aufirst%3DT.%26aulast%3DBlask%26aufirst%3DD.%2BE.%26aulast%3DHill%26aufirst%3DS.%2BM.%26atitle%3DDoxorubicin%2520resistance%2520in%2520breast%2520cancer%2520is%2520driven%2520by%2520light%2520at%2520night-induced%2520disruption%2520of%2520the%2520circadian%2520melatonin%2520signal%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D59%26spage%3D60%26epage%3D69%26doi%3D10.1111%2Fjpi.12239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, G.</span></span> <span> </span><span class="NLM_article-title">Melatonin as a treatment for gastrointestinal cancer: a review</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1111/jpi.12227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25752643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=375-387&author=Z.+L.+Xinauthor=S.+Jiangauthor=P.+Jiangauthor=X.+L.+Yanauthor=C.+X.+Fanauthor=S.+Y.+Diauthor=G.+L.+Wuauthor=Y.+Yangauthor=R.+J.+Reiterauthor=G.+Ji&title=Melatonin+as+a+treatment+for+gastrointestinal+cancer%3A+a+review&doi=10.1111%2Fjpi.12227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin as a treatment for gastrointestinal cancer: a review</span></div><div class="casAuthors">Xin, Zhenlong; Jiang, Shuai; Jiang, Peng; Yan, Xiaolong; Fan, Chongxi; Di, Shouyin; Wu, Guiling; Yang, Yang; Reiter, Russel J.; Ji, Gang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-387</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Gastrointestinal cancer is a disease that affects the population worldwide with high morbidity and mortality.  Melatonin, an endogenously produced mol., may provide a defense against a variety of cancer types.  In particular, the ability of melatonin to inhibit gastrointestinal cancer is substantial.  In this review, we first clarify the relationship between the disruption of the melatonin rhythm and gastrointestinal cancer (based on epidemiol. surveys and animal and human studies) and summarize the preventive effect of melatonin on carcinogenesis.  Thereafter, the mechanisms through which melatonin exerts its anti-gastrointestinal cancer actions are explained, including inhibition of proliferation, invasion, metastasis, and angiogenesis, and promotion of apoptosis and cancer immunity.  Moreover, we discuss the drug synergy effects and the role of melatonin receptors involved in the growth-inhibitory effects on gastrointestinal cancer.  Taken together, the information compiled here serves as a comprehensive ref. for the anti-gastrointestinal cancer actions of melatonin that have been identified to date and will hopefully aid in the design of further exptl. and clin. studies and increase the awareness of melatonin as a therapeutic agent in cancers of the gastrointestinal tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHJNmf8c1j7rVg90H21EOLACvtfcHk0lgWAJ8Ru2Y-QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb8%253D&md5=708b3eedfa5d45583901675aa3aa6628</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12227%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DZ.%2BL.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DX.%2BL.%26aulast%3DFan%26aufirst%3DC.%2BX.%26aulast%3DDi%26aufirst%3DS.%2BY.%26aulast%3DWu%26aufirst%3DG.%2BL.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26aulast%3DJi%26aufirst%3DG.%26atitle%3DMelatonin%2520as%2520a%2520treatment%2520for%2520gastrointestinal%2520cancer%253A%2520a%2520review%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D58%26spage%3D375%26epage%3D387%26doi%3D10.1111%2Fjpi.12227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Melatonin: a potential therapeutic option for breast cancer</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2020.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.tem.2020.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32893084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1CktLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=859-871&author=X.+Y.+Kongauthor=R.+Gaoauthor=Z.+Z.+Wangauthor=X.+Y.+Wangauthor=Y.+Fangauthor=J.+D.+Gaoauthor=R.+J.+Reiterauthor=J.+Wang&title=Melatonin%3A+a+potential+therapeutic+option+for+breast+cancer&doi=10.1016%2Fj.tem.2020.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin As A Potential Therapeutic Option for Breast Cancer</span></div><div class="casAuthors">Kong, Xiangyi; Gao, Ran; Wang, Zhongzhao; Wang, Xiangyu; Fang, Yi; Gao, Jidong; Reiter, Russel J.; Wang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">859-871</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Melatonin has significant inhibitory effects in numerous cancers, esp. breast cancer.  In estrogen receptor (ER)-pos. human breast cancer, the oncostatic actions of melatonin are mainly achieved by suppressing ER mRNA expression and ER transcriptional activity via the MT1 receptor.  Melatonin also regulates the transactivation of nuclear receptors, estrogen-metabolizing enzymes, and the expression of related genes.  Furthermore, melatonin suppresses tumor aerobic glycolysis, crit. cell-signaling pathways relevant to cell proliferation, survival, metastasis, and overcomes drug resistance.  Studies in animal and human models indicate that disruption of the circadian nocturnal melatonin signal promotes the growth, metab., and signaling of human breast cancer, resulting in resistance to hormone therapy and chemotherapy, which may be reversed by melatonin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn-7V7hTSjA7Vg90H21EOLACvtfcHk0lgWAJ8Ru2Y-QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1CktLjI&md5=a4f7a9c322df69a02ec72d9f6032160c</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2020.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2020.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%2BY.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DZ.%2BZ.%26aulast%3DWang%26aufirst%3DX.%2BY.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DJ.%2BD.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DMelatonin%253A%2520a%2520potential%2520therapeutic%2520option%2520for%2520breast%2520cancer%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2020%26volume%3D31%26spage%3D859%26epage%3D871%26doi%3D10.1016%2Fj.tem.2020.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farhood, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goradel, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortezaee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanlarkhani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahebkar, A.</span></span> <span> </span><span class="NLM_article-title">Melatonin and cancer: from the promotion of genomic stability to use in cancer treatment</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">5613</span>– <span class="NLM_lpage">5627</span>, <span class="refDoi"> DOI: 10.1002/jcp.27391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fjcp.27391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30238978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGlt7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2019&pages=5613-5627&author=B.+Farhoodauthor=N.+H.+Goradelauthor=K.+Mortezaeeauthor=N.+Khanlarkhaniauthor=M.+Najafiauthor=A.+Sahebkar&title=Melatonin+and+cancer%3A+from+the+promotion+of+genomic+stability+to+use+in+cancer+treatment&doi=10.1002%2Fjcp.27391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin and cancer: From the promotion of genomic stability to use in cancer treatment</span></div><div class="casAuthors">Farhood, Bagher; Goradel, Nasser Hashemi; Mortezaee, Keywan; Khanlarkhani, Neda; Najafi, Masoud; Sahebkar, Amirhossein</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5613-5627</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cancer remains among the most challenging human diseases.  Several lines of evidence suggest that carcinogenesis is a complex process that is initiated by DNA damage.  Exposure to clastogenic agents such as heavy metals, ionizing radiation (IR), and chemotherapy drugs may cause chronic mutations in the genomic material, leading to a phenomenon named genomic instability.  Evidence suggests that genomic instability is responsible for cancer incidence after exposure to carcinogenic agents, and increases the risk of secondary cancers following treatment with radiotherapy or chemotherapy.  Melatonin as the main product of the pineal gland is a promising hormone for preventing cancer and improving cancer treatment.  Melatonin can directly neutralize toxic free radicals more efficiently compared with other classical antioxidants.  In addn., melatonin is able to regulate the redn./oxidn. (redox) system in stress conditions.  Through regulation of mitochondrial nction and inhibition of pro-oxidant enzymes, melatonin suppresses chronic oxidative stress.  Moreover, melatonin potently stimulates DNA damage responses that increase the tolerance of normal tissues to toxic effect of IR and may reduce the risk of genomic instability in patients who undergo radiotherapy.  Through these mechanisms, melatonin attenuates several side effects of radiotherapy and chemotherapy.  Interestingly, melatonin has shown some synergistic properties with IR and chemotherapy, which is distinct from classical antioxidants that are mainly used for the alleviation of adverse events of radiotherapy and chemotherapy.  In this review, we describe the anticarcinogenic effects of melatonin and also its possible application in clin. oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro3KIsdYB6mrVg90H21EOLACvtfcHk0lh6pzg83KeNKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslGlt7fE&md5=7965ad9b4f7bdf2058dee7683a406ee7</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fjcp.27391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.27391%26sid%3Dliteratum%253Aachs%26aulast%3DFarhood%26aufirst%3DB.%26aulast%3DGoradel%26aufirst%3DN.%2BH.%26aulast%3DMortezaee%26aufirst%3DK.%26aulast%3DKhanlarkhani%26aufirst%3DN.%26aulast%3DNajafi%26aufirst%3DM.%26aulast%3DSahebkar%26aufirst%3DA.%26atitle%3DMelatonin%2520and%2520cancer%253A%2520from%2520the%2520promotion%2520of%2520genomic%2520stability%2520to%2520use%2520in%2520cancer%2520treatment%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D2019%26volume%3D234%26spage%3D5613%26epage%3D5627%26doi%3D10.1002%2Fjcp.27391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maroufi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vahedian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rashidi, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akbarzadeh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahedi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouremamali, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isazadeh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taefehshokr, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajazimian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seraji, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nouri, M.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer stem cells by melatonin: effective therapy for cancer treatment</span>. <i>Pathol., Res. Pract.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>216</i></span>,  <span class="NLM_fpage">152919</span>– <span class="NLM_lpage">152926</span>, <span class="refDoi"> DOI: 10.1016/j.prp.2020.152919</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.prp.2020.152919" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32171553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvFChsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=216&publication_year=2020&pages=152919-152926&author=N.+F.+Maroufiauthor=V.+Vahedianauthor=S.+Hematiauthor=M.+R.+Rashidiauthor=M.+Akbarzadehauthor=M.+Zahediauthor=F.+Pouremamaliauthor=A.+Isazadehauthor=S.+Taefehshokrauthor=S.+Hajazimianauthor=N.+Serajiauthor=M.+Nouri&title=Targeting+cancer+stem+cells+by+melatonin%3A+effective+therapy+for+cancer+treatment&doi=10.1016%2Fj.prp.2020.152919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer stem cells by melatonin: Effective therapy for cancer treatment</span></div><div class="casAuthors">Maroufi, Nazila Fathi; Vahedian, Vahid; Hemati, Saideh; Rashidi, Mohammad-Reza; Akbarzadeh, Maryam; Zahedi, Mohammadsaeid; Pouremamali, Farhad; Isazadeh, Alireza; Taefehshokr, Sina; Hajazimian, Saba; Seraji, Nikou; Nouri, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Pathology, Research and Practice</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">216</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">152919</span>CODEN:
                <span class="NLM_cas:coden">PARPDS</span>;
        ISSN:<span class="NLM_cas:issn">0344-0338</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Melatonin is a physiol. hormone produced by the pineal gland.  In recent decades, enormous investigations showed that melatonin can prompt apoptosis in cancer cells and inhibit tumor metastasis and angiogenesis in variety of malignancies such as ovarian, melanoma, colon, and breast cancer; therefore, its possible therapeutic usage in cancer treatment was confirmed.  CSCs, which has received much attention from researchers in past decades, are major challenges in the treatment of cancer.  Because CSCs are resistant to chemotherapeutic drugs and cause recurrence of cancer and also have the ability to be regenerated; they can cause serious problems in the treatment of various cancers.  For these reasons, the researchers are trying to find a soln. to destroy these cells within the tumor mass.  In recent years, the effect of melatonin on CSCs has been investigated in some cancers.  Given the importance of CSCs in the process of cancer treatment, this article reviewed the studies conducted on the effect of melatonin on CSCs as a soln. to the problems caused by CSCs in the treatment of various cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_fmDcMvz0t7Vg90H21EOLACvtfcHk0lh6pzg83KeNKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvFChsL0%253D&md5=0621003b2d1100c020602169959fe669</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.prp.2020.152919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.prp.2020.152919%26sid%3Dliteratum%253Aachs%26aulast%3DMaroufi%26aufirst%3DN.%2BF.%26aulast%3DVahedian%26aufirst%3DV.%26aulast%3DHemati%26aufirst%3DS.%26aulast%3DRashidi%26aufirst%3DM.%2BR.%26aulast%3DAkbarzadeh%26aufirst%3DM.%26aulast%3DZahedi%26aufirst%3DM.%26aulast%3DPouremamali%26aufirst%3DF.%26aulast%3DIsazadeh%26aufirst%3DA.%26aulast%3DTaefehshokr%26aufirst%3DS.%26aulast%3DHajazimian%26aufirst%3DS.%26aulast%3DSeraji%26aufirst%3DN.%26aulast%3DNouri%26aufirst%3DM.%26atitle%3DTargeting%2520cancer%2520stem%2520cells%2520by%2520melatonin%253A%2520effective%2520therapy%2520for%2520cancer%2520treatment%26jtitle%3DPathol.%252C%2520Res.%2520Pract.%26date%3D2020%26volume%3D216%26spage%3D152919%26epage%3D152926%26doi%3D10.1016%2Fj.prp.2020.152919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamtaji, O. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirhosseini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnamfar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asemi, Z.</span></span> <span> </span><span class="NLM_article-title">Melatonin and pancreatic cancer: current knowledge and future perspectives</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">5372</span>– <span class="NLM_lpage">5378</span>, <span class="refDoi"> DOI: 10.1002/jcp.27372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fjcp.27372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30229898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslCjtbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2019&pages=5372-5378&author=O.+R.+Tamtajiauthor=N.+Mirhosseiniauthor=R.+J.+Reiterauthor=M.+Behnamfarauthor=Z.+Asemi&title=Melatonin+and+pancreatic+cancer%3A+current+knowledge+and+future+perspectives&doi=10.1002%2Fjcp.27372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin and pancreatic cancer: Current knowledge and future perspectives</span></div><div class="casAuthors">Tamtaji, Omid Reza; Mirhosseini, Naghmeh; Reiter, Russel J.; Behnamfar, Morteza; Asemi, Zatollah</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5372-5378</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Pancreatic cancer has a high mortality rate due to the absence of early symptoms and subsequent late diagnosis; addnl., pancreatic cancer has a high resistance to radio- and chemotherapy.  Multiple inflammatory pathways are involved in the pathophysiol. of pancreatic cancer.  Melatonin an indoleamine produced in the pineal gland mediated and receptor-independent action is the pancreas and other where has both receptors.  Melatonin is a potent antioxidant and tissue protector against inflammation and oxidative stress.  In vivo and in vitro studies have shown that melatonin supplementation is an appropriate therapeutic approach for pancreatic cancer.  Melatonin may be an effective apoptosis inducer in cancer cells through regulation of a large no. of mol. pathways including oxidative stress, heat shock proteins, and vascular endothelial growth factor.  Limited clin. studies, however, have evaluated the role of melatonin in pancreatic cancer.  This review summarizes what is known regarding the effects of melatonin on pancreatic cancer and the mechanisms involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGwcyvLulZM7Vg90H21EOLACvtfcHk0lh6pzg83KeNKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslCjtbnM&md5=27fe8cc751f9d3f174e1f751a6999f21</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fjcp.27372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.27372%26sid%3Dliteratum%253Aachs%26aulast%3DTamtaji%26aufirst%3DO.%2BR.%26aulast%3DMirhosseini%26aufirst%3DN.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26aulast%3DBehnamfar%26aufirst%3DM.%26aulast%3DAsemi%26aufirst%3DZ.%26atitle%3DMelatonin%2520and%2520pancreatic%2520cancer%253A%2520current%2520knowledge%2520and%2520future%2520perspectives%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D2019%26volume%3D234%26spage%3D5372%26epage%3D5378%26doi%3D10.1002%2Fjcp.27372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shieh, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sytwu, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. W.</span></span> <span> </span><span class="NLM_article-title">Melatonin exerts anti-oral cancer effect via suppressing LSD1 in patient-derived tumor xenograft models</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">33756</span>– <span class="NLM_lpage">33769</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.16808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.18632%2Foncotarget.16808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28422711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BC1cvptFCisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=33756-33769&author=C.+Y.+Yangauthor=C.+K.+Linauthor=C.+H.+Tsaoauthor=C.+C.+Hsiehauthor=G.+J.+Linauthor=K.+H.+Maauthor=Y.+S.+Shiehauthor=H.+K.+Sytwuauthor=Y.+W.+Chen&title=Melatonin+exerts+anti-oral+cancer+effect+via+suppressing+LSD1+in+patient-derived+tumor+xenograft+models&doi=10.18632%2Foncotarget.16808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin exerts anti-oral cancer effect via suppressing LSD1 in patient-derived tumor xenograft models</span></div><div class="casAuthors">Yang Cheng-Yu; Sytwu Huey-Kang; Chen Yuan-Wu; Yang Cheng-Yu; Shieh Yi-Shing; Chen Yuan-Wu; Lin Chih-Kung; Tsao Chang-Huei; Sytwu Huey-Kang; Tsao Chang-Huei; Hsieh Cheng-Chih; Lin Gu-Jiun; Ma Kuo-Hsing; Chen Yuan-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">33756-33769</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrant activation of histone lysine-specific demethylase (LSD1) increases tumorigenicity; hence, LSD1 is considered a therapeutic target for various human cancers.  Although melatonin, an endogenously produced molecule, may defend against various cancers, the precise mechanism involved in its anti-oral cancer effect remains unclear.  Patient-derived tumor xenograft (PDTX) models are preclinical models that can more accurately reflect human tumor biology compared with cell line xenograft models.  Here, we evaluated the anticancer activity of melatonin by using LSD1-overexpressing oral cancer PDTX models.  By assessing oral squamous cell carcinoma (OSCC) tissue arrays through immunohistochemistry, we examined whether aberrant LSD1 overexpression in OSCC is associated with poor prognosis.  We also evaluated the action mechanism of melatonin against OSCC with lymphatic metastases by using the PDTX models.  Our results indicated that melatonin, at pharmacological concentrations, significantly suppresses cell proliferation in a dose- and time-dependent manner.  The observed suppression of proliferation was accompanied by the melatonin-mediated inhibition of LSD1 in oral cancer PDTXs and oral cancer cell lines.  In conclusion, we determined that the beneficial effects of melatonin in reducing oral cancer cell proliferation are associated with reduced LSD1 expression in vivo and in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSR9ND6-Hc_YBjv2lLSLOxFfW6udTcc2eYO_AF4id6lirntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvptFCisA%253D%253D&md5=812cc98de3dc08051125329329a3a8d2</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.16808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.16808%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BK.%26aulast%3DTsao%26aufirst%3DC.%2BH.%26aulast%3DHsieh%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DG.%2BJ.%26aulast%3DMa%26aufirst%3DK.%2BH.%26aulast%3DShieh%26aufirst%3DY.%2BS.%26aulast%3DSytwu%26aufirst%3DH.%2BK.%26aulast%3DChen%26aufirst%3DY.%2BW.%26atitle%3DMelatonin%2520exerts%2520anti-oral%2520cancer%2520effect%2520via%2520suppressing%2520LSD1%2520in%2520patient-derived%2520tumor%2520xenograft%2520models%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D33756%26epage%3D33769%26doi%3D10.18632%2Foncotarget.16808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xi, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2018.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29524666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkt1Grtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=7-16&author=J.+Y.+Xiauthor=S.+Y.+Xuauthor=L.+L.+Zhangauthor=X.+Y.+Biauthor=Y.+S.+Renauthor=Y.+C.+Liuauthor=Y.+Q.+Guauthor=Y.+G.+Xuauthor=F.+Lanauthor=X.+M.+Zha&title=Design%2C+synthesis+and+biological+activity+of+4-%284-benzyloxy%29phenoxypiperidines+as+selective+and+reversible+LSD1+inhibitors&doi=10.1016%2Fj.bioorg.2018.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors</span></div><div class="casAuthors">Xi, Jiayue; Xu, Siyuan; Zhang, Lulu; Bi, Xueyuan; Ren, Yanshen; Liu, Yu-Chih; Gu, Yueqing; Xu, Yungen; Lan, Fei; Zha, Xiaoming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-16</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays a vital role in epigenetic regulation of gene activation and repression in several human cancers and is recognized as a promising antitumor therapeutic target.  In this paper, a series of 4-(4-benzyloxy)phenoxypiperidines were synthesized and evaluated.  Among the tested compds., compd. 10d (I) exhibited the potent and reversible inhibitory activity against LSD1 in vitro (IC50 = 4 μM).  Mol. docking was conducted to predict its binding mode.  Furthermore, 10d displayed it could inhibit migration of HCT-116 colon cancer cells and A549 lung cancer cells.  Taken together, 10d deserves further investigation as a hit-to-lead for the treatment of LSD1 assocd. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4X0tCTXSso7Vg90H21EOLACvtfcHk0lh5XE7SbkIjhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkt1Grtr0%253D&md5=fb872c048079936b4d47edc4bf25cbdd</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DXi%26aufirst%3DJ.%2BY.%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DZhang%26aufirst%3DL.%2BL.%26aulast%3DBi%26aufirst%3DX.%2BY.%26aulast%3DRen%26aufirst%3DY.%2BS.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DGu%26aufirst%3DY.%2BQ.%26aulast%3DXu%26aufirst%3DY.%2BG.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DZha%26aufirst%3DX.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520activity%2520of%25204-%25284-benzyloxy%2529phenoxypiperidines%2520as%2520selective%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D78%26spage%3D7%26epage%3D16%26doi%3D10.1016%2Fj.bioorg.2018.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Ligand-based design, synthesis and biological evaluation of xanthine derivatives as LSD1/KDM1A inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2018.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30472603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12ltLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2019&pages=555-567&author=Q.+S.+Maauthor=Y.+Yaoauthor=Y.+C.+Zhengauthor=S.+Fengauthor=J.+Changauthor=B.+Yuauthor=H.+M.+Liu&title=Ligand-based+design%2C+synthesis+and+biological+evaluation+of+xanthine+derivatives+as+LSD1%2FKDM1A+inhibitors&doi=10.1016%2Fj.ejmech.2018.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-based design, synthesis and biological evaluation of xanthine derivatives as LSD1/KDM1A inhibitors</span></div><div class="casAuthors">Ma, Qi-Sheng; Yao, Yongfang; Zheng, Yi-Chao; Feng, Siqi; Chang, Junbiao; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">555-567</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1, KDM1A) has been recognized as an important epigenetic target for disease treatment.  To date, a large no. of LSD1 inhibitors have been developed, some of which are currently being evaluated in clin. trials for the treatment of cancers, virus infection, and neurodegenerative diseases.  In this paper, we for the first time reported the ligand-based design of fragment-like xanthine derivs. as LSD1 inhibitors, of which compd. 4 possessed acceptable pharmacol. inhibition against LSD1 (IC50 = 6.45 μM) and favorable fragment-like nature, and therefore could be used as a promising template to design new LSD1 inhibitors.  Interestingly, compds. 6c and 6i strongly suppressed growth of MGC-803 cells partly dependent on their LSD1 inhibition, and were also found to be able to inhibit BRD4 and IDO1.  The docking studies were performed to rationalize the biochem. potency against LSD1 and to explain the obsd. activity discrepancy.  The proof-of-concept work may provide an example for other natural ligand-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNwIGUA6safrVg90H21EOLACvtfcHk0lh5XE7SbkIjhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12ltLvO&md5=3b4bd4668111aad864a366d55ab6c777</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DQ.%2BS.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DLigand-based%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520xanthine%2520derivatives%2520as%2520LSD1%252FKDM1A%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D162%26spage%3D555%26epage%3D567%26doi%3D10.1016%2Fj.ejmech.2018.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Furan carbonylhydrazones-derived elicitors that induce the resistance of rice to the brown planthopper Nilaparvata lugens</span>. <i>Phytochem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1016/j.phytol.2018.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.phytol.2018.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1SmsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=184-189&author=X.+Chenauthor=X.+R.+Heauthor=W.+W.+Wangauthor=P.+Y.+Zhouauthor=Y.+G.+Louauthor=J.+Wu&title=Furan+carbonylhydrazones-derived+elicitors+that+induce+the+resistance+of+rice+to+the+brown+planthopper+Nilaparvata+lugens&doi=10.1016%2Fj.phytol.2018.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Furan carbonylhydrazones-derived elicitors that induce the resistance of rice to the brown planthopper Nilaparvata lugens</span></div><div class="casAuthors">Chen, Xia; He, Xingrui; Wang, Wanwan; Zhou, Pengyong; Lou, Yonggen; Wu, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">184-189</span>CODEN:
                <span class="NLM_cas:coden">PLHEBK</span>;
        ISSN:<span class="NLM_cas:issn">1874-3900</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">To obtain new elicitors that could induce plant defense against pests, a series of furan carbonylhydrazones were synthesized.  Phenotypic high-throughput screening of β-glucuronidase (GUS) activity was used to identify elicitors with strong activity.  Integrated bioassays were conducted to acquire chem. elicitors that induce resistance in rice to the brown planthopper Nilaparvata lugens and they are not direct toxic to the pest.  Our results demonstrate that these compds. have the potential to serve as new kinds of environment-friendly plant protection agents in rice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3EUTv63xn1rVg90H21EOLACvtfcHk0liAAueaOE0XJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1SmsLzF&md5=82372e1573a137e0f017afc2a29d0277</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.phytol.2018.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytol.2018.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%2BR.%26aulast%3DWang%26aufirst%3DW.%2BW.%26aulast%3DZhou%26aufirst%3DP.%2BY.%26aulast%3DLou%26aufirst%3DY.%2BG.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DFuran%2520carbonylhydrazones-derived%2520elicitors%2520that%2520induce%2520the%2520resistance%2520of%2520rice%2520to%2520the%2520brown%2520planthopper%2520Nilaparvata%2520lugens%26jtitle%3DPhytochem.%2520Lett.%26date%3D2018%26volume%3D26%26spage%3D184%26epage%3D189%26doi%3D10.1016%2Fj.phytol.2018.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Finding new elicitors that induce resistance in rice to the white-backed planthopper sogatella furcifera</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5601</span>– <span class="NLM_lpage">5603</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2015.10.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=26508551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegu7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=5601-5603&author=X.+R.+Heauthor=Z.+N.+Yuauthor=S.+J.+Jiangauthor=P.+Z.+Zhangauthor=Z.+C.+Shangauthor=Y.+G.+Louauthor=J.+Wu&title=Finding+new+elicitors+that+induce+resistance+in+rice+to+the+white-backed+planthopper+sogatella+furcifera&doi=10.1016%2Fj.bmcl.2015.10.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Finding new elicitors that induce resistance in rice to the white-backed planthopper Sogatella furcifera</span></div><div class="casAuthors">He, Xingrui; Yu, Zhaonan; Jiang, Shaojie; Zhang, Peizhi; Shang, Zhicai; Lou, Yonggen; Wu, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5601-5603</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we report a new way to identify chem. elicitors that induce resistance in rice to herbivores.  Using this method, by quantifying the induction of chems. for GUS activity in a specific screening system that we established previously, 5 candidate elicitors were selected from the 29 designed and synthesized phenoxyalkanoic acid derivs.  Bioassays confirmed that these candidate elicitors could induce plant defense and then repel feeding of white-backed planthopper Sogatella furcifera.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8WPKM2PNM7Vg90H21EOLACvtfcHk0liAAueaOE0XJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegu7bO&md5=72683849ec089ea049906f74daeac615</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.041%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DX.%2BR.%26aulast%3DYu%26aufirst%3DZ.%2BN.%26aulast%3DJiang%26aufirst%3DS.%2BJ.%26aulast%3DZhang%26aufirst%3DP.%2BZ.%26aulast%3DShang%26aufirst%3DZ.%2BC.%26aulast%3DLou%26aufirst%3DY.%2BG.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DFinding%2520new%2520elicitors%2520that%2520induce%2520resistance%2520in%2520rice%2520to%2520the%2520white-backed%2520planthopper%2520sogatella%2520furcifera%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D5601%26epage%3D5603%26doi%3D10.1016%2Fj.bmcl.2015.10.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X. Y.</span></span> <span> </span><span class="NLM_article-title">4-Hydroxy-3-methylbenzofuran-2-carbohydrazones as novel LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127109</span>– <span class="NLM_lpage">127114</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2020.127109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32201021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1Cgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127109-127114&author=X.+R.+Heauthor=Y.+Gaoauthor=Z.+Huiauthor=G.+D.+Shenauthor=S.+Wangauthor=T.+Xieauthor=X.+Y.+Ye&title=4-Hydroxy-3-methylbenzofuran-2-carbohydrazones+as+novel+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2020.127109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">4-Hydroxy-3-methylbenzofuran-2-carbohydrazones as novel LSD1 inhibitors</span></div><div class="casAuthors">He, Xingrui; Gao, Yuan; Hui, Zi; Shen, Guodong; Wang, Shuo; Xie, Tian; Ye, Xiang-Yang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">127109</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1 or KDM1A) is a potential therapeutic target in oncol. due to its overexpression in various human tumors.  The authors report herein a new class of benzofuran acylhydrazones as potent LSD1 inhibitors.  Among the 31 compds. prepd., 14 compds. exhibited excellent LSD1 inhibitory activity with IC50 values ranging from 7.2 to 68.8 nM.  In cellular assays, several compds. inhibited the proliferations of various cancer cell lines, including PC-3, MCG-<803≥, U87 MG, PANC-1, HT-29 and MCF-7.  This opens up the opportunity for further optimization and study of this class compds. for potential cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWwiTYTNJDRbVg90H21EOLACvtfcHk0liAAueaOE0XJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1Cgs7g%253D&md5=aef250e0011154d944a3aae164f14619</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127109%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DX.%2BR.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DHui%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DG.%2BD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DX.%2BY.%26atitle%3D4-Hydroxy-3-methylbenzofuran-2-carbohydrazones%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127109%26epage%3D127114%26doi%3D10.1016%2Fj.bmcl.2020.127109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amigoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legnaghi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romussi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thaler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultaggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrugno, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessanti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagarri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carettoni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iuzzolino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vianello, P.</span></span> <span> </span><span class="NLM_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as new reversible inhibitors of histone lysine demethylase KDM1A/LSD1. Part 1: High-throughput screening and preliminary exploration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1673</span>– <span class="NLM_lpage">1692</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisVOqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1673-1692&author=L.+Sartoriauthor=C.+Mercurioauthor=F.+Amigoniauthor=A.+Cappaauthor=G.+Fagaauthor=R.+Fattoriauthor=E.+Legnaghiauthor=G.+Ciossaniauthor=A.+Matteviauthor=G.+Meroniauthor=L.+Morettiauthor=V.+Cecatielloauthor=S.+Pasqualatoauthor=A.+Romussiauthor=F.+Thalerauthor=P.+Trifiroauthor=M.+Villaauthor=S.+Vultaggioauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=E.+Zagarriauthor=D.+Carettoniauthor=L.+Iuzzolinoauthor=M.+Varasiauthor=P.+Vianello&title=Thieno%5B3%2C2-b%5Dpyrrole-5-carboxamides+as+new+reversible+inhibitors+of+histone+lysine+demethylase+KDM1A%2FLSD1.+Part+1%3A+High-throughput+screening+and+preliminary+exploration&doi=10.1021%2Facs.jmedchem.6b01018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration</span></div><div class="casAuthors">Sartori, Luca; Mercurio, Ciro; Amigoni, Federica; Cappa, Anna; Faga, Giovanni; Fattori, Raimondo; Legnaghi, Elena; Ciossani, Giuseppe; Mattevi, Andrea; Meroni, Giuseppe; Moretti, Loris; Cecatiello, Valentina; Pasqualato, Sebastiano; Romussi, Alessia; Thaler, Florian; Trifiro, Paolo; Villa, Manuela; Vultaggio, Stefania; Botrugno, Oronza A.; Dessanti, Paola; Minucci, Saverio; Zagarri, Elisa; Carettoni, Daniele; Iuzzolino, Lucia; Varasi, Mario; Vianello, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1673-1692</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 KDM1A (LSD1) is one regulator of histone methylation and it is increasingly recognized as a potential therapeutic target in oncol.  The authors report on a high-throughput screening campaign performed on KDM1A/CoREST, using a time resolved fluorescence resonance energy transfer (TR-FRET) technol., to identify reversible inhibitors.  The screening led to 115 hits for which the authors detd. biochem. IC50, thus identifying 4 chem. series.  After data anal., the authors have prioritized the chem. series of N-phenyl-4H-thieno[3,2-b]pyrrole-5-carboxamide for which the authors obtained x-ray structures of the most potent hit (compd. 19, IC50 = 2.9 μM) in complex with the enzyme.  Initial expansion of this chem. class, both modifying core structure and decorating benzamide moiety, was directed towards the definition of the moieties responsible for the interaction with the enzyme.  Preliminary optimization brought to compd. 90 (4-methyl-N-[3-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]- thieno[3,2-b]pyrrole-5-carboxamide) which inhibited the enzyme with a submicromolar IC50 (0.162 μM), capable to inhibit the target in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3SWpFkenrJLVg90H21EOLACvtfcHk0liEV7u-g2umeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisVOqt7o%253D&md5=e3ad614cffd844ea9ca1da1747f6aa2d</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26aulast%3DSartori%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DAmigoni%26aufirst%3DF.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DFaga%26aufirst%3DG.%26aulast%3DFattori%26aufirst%3DR.%26aulast%3DLegnaghi%26aufirst%3DE.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DPasqualato%26aufirst%3DS.%26aulast%3DRomussi%26aufirst%3DA.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVultaggio%26aufirst%3DS.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DZagarri%26aufirst%3DE.%26aulast%3DCarettoni%26aufirst%3DD.%26aulast%3DIuzzolino%26aufirst%3DL.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DVianello%26aufirst%3DP.%26atitle%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2520as%2520new%2520reversible%2520inhibitors%2520of%2520histone%2520lysine%2520demethylase%2520KDM1A%252FLSD1.%2520Part%25201%253A%2520High-throughput%2520screening%2520and%2520preliminary%2520exploration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1673%26epage%3D1692%26doi%3D10.1021%2Facs.jmedchem.6b01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vianello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amigoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagá, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legnaghi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romussi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thaler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifiró, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrugno, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessanti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultaggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagarrí, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, C.</span></span> <span> </span><span class="NLM_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as new reversible inhibitors of histone lysine demethylase KDM1A/LSD1. Part 2: Structure-based drug design and structure-activity relationship</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1715</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisVOkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1693-1715&author=P.+Vianelloauthor=L.+Sartoriauthor=F.+Amigoniauthor=A.+Cappaauthor=G.+Fag%C3%A1author=R.+Fattoriauthor=E.+Legnaghiauthor=G.+Ciossaniauthor=A.+Matteviauthor=G.+Meroniauthor=L.+Morettiauthor=V.+Cecatielloauthor=S.+Pasqualatoauthor=A.+Romussiauthor=F.+Thalerauthor=P.+Trifir%C3%B3author=M.+Villaauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=S.+Vultaggioauthor=E.+Zagarr%C3%ADauthor=M.+Varasiauthor=C.+Mercurio&title=Thieno%5B3%2C2-b%5Dpyrrole-5-carboxamides+as+new+reversible+inhibitors+of+histone+lysine+demethylase+KDM1A%2FLSD1.+Part+2%3A+Structure-based+drug+design+and+structure-activity+relationship&doi=10.1021%2Facs.jmedchem.6b01019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure-Activity Relationship</span></div><div class="casAuthors">Vianello, Paola; Sartori, Luca; Amigoni, Federica; Cappa, Anna; Faga, Giovanni; Fattori, Raimondo; Legnaghi, Elena; Ciossani, Giuseppe; Mattevi, Andrea; Meroni, Giuseppe; Moretti, Loris; Cecatiello, Valentina; Pasqualato, Sebastiano; Romussi, Alessia; Thaler, Florian; Trifiro, Paolo; Villa, Manuela; Botrugno, Oronza A.; Dessanti, Paola; Minucci, Saverio; Vultaggio, Stefania; Zagarri, Elisa; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1693-1715</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1).  KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development of novel cancer therapeutics.  The authors have previously described the identification of thieno[3,2-b]pyrrole-5-carboxamides, as novel reversible inhibitors of KDM1A, whose preliminary exploration resulted in compd. (I) with biochem. IC50 = 160 nM.  The authors now report the structure-guided optimization of this chem. series, based on multiple ligand/KDM1A-CoRest co-crystal structures, which led to several extremely potent inhibitors.  In particular, compds. N-[3-(methoxymethyl)-2-[[4-[[(3R)-pyrrolidin-3-yl]- methoxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-b]-pyrrole-5-carboxamide hydrochloride (46), 4-ethyl-N-[3-(methoxymethyl)-2-[[4-[[(3R)-pyrrolidin-3-yl]-methoxy]phenoxy]methyl]phenyl]thieno[3,2-b]pyrrole-5-carboxamide (49) and N-[3-(ethoxymethyl)-2-[[4-[[(3R)-pyrrolidin-3-yl]methoxy]-phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-b]pyrrole-5-carboxamide hydrochloride (50) showed single digit nanomolar IC50 values for in vitro inhibition of KDM1A, with high selectivity in secondary assays.  In THP-1 cells these compds. transcriptionally affected the expression of genes regulated by KDM1A, such as CD14, CD11b, and CD86.  Moreover, 49 and 50 showed a remarkable anti-clonogenic cell growth effect on MLL-AF9 human leukemia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmOBI9faVjBrVg90H21EOLACvtfcHk0liEV7u-g2umeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisVOkur8%253D&md5=c4713516ee7b832817c386717c0bd71f</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DSartori%26aufirst%3DL.%26aulast%3DAmigoni%26aufirst%3DF.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DFag%25C3%25A1%26aufirst%3DG.%26aulast%3DFattori%26aufirst%3DR.%26aulast%3DLegnaghi%26aufirst%3DE.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DPasqualato%26aufirst%3DS.%26aulast%3DRomussi%26aufirst%3DA.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTrifir%25C3%25B3%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DVultaggio%26aufirst%3DS.%26aulast%3DZagarr%25C3%25AD%26aufirst%3DE.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2520as%2520new%2520reversible%2520inhibitors%2520of%2520histone%2520lysine%2520demethylase%2520KDM1A%252FLSD1.%2520Part%25202%253A%2520Structure-based%2520drug%2520design%2520and%2520structure-activity%2520relationship%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1693%26epage%3D1715%26doi%3D10.1021%2Facs.jmedchem.6b01019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romussi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vianello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambillasca, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cera, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dal Zuffo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagà, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifirò, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultaggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dondio, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, C. W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of reversible inhibitors of KDM1A efficacious in acute myeloid leukemia models</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">754</span>– <span class="NLM_lpage">759</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00604</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00604" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFyisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=754-759&author=A.+Romussiauthor=A.+Cappaauthor=P.+Vianelloauthor=S.+Brambillascaauthor=M.+R.+Ceraauthor=R.+Dal+Zuffoauthor=G.+Fag%C3%A0author=R.+Fattoriauthor=L.+Morettiauthor=P.+Trifir%C3%B2author=M.+Villaauthor=S.+Vultaggioauthor=V.+Cecatielloauthor=S.+Pasqualatoauthor=G.+Dondioauthor=C.+W.+E.+Soauthor=S.+Minucciauthor=L.+Sartoriauthor=M.+Varasiauthor=C.+Mercurio&title=Discovery+of+reversible+inhibitors+of+KDM1A+efficacious+in+acute+myeloid+leukemia+models&doi=10.1021%2Facsmedchemlett.9b00604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Reversible Inhibitors of KDM1A Efficacious in Acute Myeloid Leukemia Models</span></div><div class="casAuthors">Romussi, Alessia; Cappa, Anna; Vianello, Paola; Brambillasca, Silvia; Cera, Maria Rosaria; Dal Zuffo, Roberto; Faga, Giovanni; Fattori, Raimondo; Moretti, Loris; Trifiro, Paolo; Villa, Manuela; Vultaggio, Stefania; Cecatiello, Valentina; Pasqualato, Sebastiano; Dondio, Giulio; So, Chi Wai Eric; Minucci, Saverio; Sartori, Luca; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">754-759</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1 or KDM1A) is a FAD-dependent enzyme that acts as a transcription corepressor or coactivator by regulating the methylation status of histone H3 lysines K4 and K9, resp.  KDM1A represents an attractive target for cancer therapy.  While, in the past, the main medicinal chem. strategy toward KDM1A inhibition was based on the optimization of ligands that irreversibly bind the FAD cofactor within the enzyme catalytic site, we and others have also identified reversible inhibitors.  Herein we reported the discovery of 5-imidazolylthieno[3,2-b]pyrroles, a new series of KDM1A inhibitors endowed with picomolar inhibitory potency, active in cells and efficacious after oral administration in murine leukemia models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiYpo0iHtNZLVg90H21EOLACvtfcHk0lixhCM31hi6jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFyisL0%253D&md5=638d1b2f8653b1fa53f8d6278e297553</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00604%26sid%3Dliteratum%253Aachs%26aulast%3DRomussi%26aufirst%3DA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBrambillasca%26aufirst%3DS.%26aulast%3DCera%26aufirst%3DM.%2BR.%26aulast%3DDal%2BZuffo%26aufirst%3DR.%26aulast%3DFag%25C3%25A0%26aufirst%3DG.%26aulast%3DFattori%26aufirst%3DR.%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DTrifir%25C3%25B2%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVultaggio%26aufirst%3DS.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DPasqualato%26aufirst%3DS.%26aulast%3DDondio%26aufirst%3DG.%26aulast%3DSo%26aufirst%3DC.%2BW.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DSartori%26aufirst%3DL.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520reversible%2520inhibitors%2520of%2520KDM1A%2520efficacious%2520in%2520acute%2520myeloid%2520leukemia%2520models%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D754%26epage%3D759%26doi%3D10.1021%2Facsmedchemlett.9b00604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazeldine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachaiyappan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbergs, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotarski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Low molecular weight amidoximes that act as potent inhibitors of lysine-specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7378</span>– <span class="NLM_lpage">7391</span>, <span class="refDoi"> DOI: 10.1021/jm3002845</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3002845" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKnsrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7378-7391&author=S.+Hazeldineauthor=B.+Pachaiyappanauthor=N.+Steinbergsauthor=S.+Nowotarskiauthor=A.+S.+Hansonauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=Low+molecular+weight+amidoximes+that+act+as+potent+inhibitors+of+lysine-specific+demethylase+1&doi=10.1021%2Fjm3002845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Low Molecular Weight Amidoximes that Act as Potent Inhibitors of Lysine-Specific Demethylase 1</span></div><div class="casAuthors">Hazeldine, Stuart; Pachaiyappan, Boobalan; Steinbergs, Nora; Nowotarski, Shannon; Hanson, Allison S.; Casero, Robert A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7378-7391</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently discovered enzyme lysine-specific demethylase 1 (LSD1) plays an important role in the epigenetic control of gene expression, and aberrant gene silencing secondary to LSD1 dysregulation is thought to contribute to the development of cancer.  We reported that (bis)guanidines, (bis)biguanides, and their urea- and thiourea isosteres are potent inhibitors of LSD1 and induce the re-expression of aberrantly silenced tumor suppressor genes in tumor cells in vitro.  We now report a series of small mol. amidoximes that are moderate inhibitors of recombinant LSD1 but that produce dramatic changes in methylation at the histone 3 lysine 4 (H3K4) chromatin mark, a specific target of LSD1, in Calu-6 lung carcinoma cells.  In addn., these analogs increase cellular levels of secreted frizzle-related protein (SFRP) 2, H-cadherin (HCAD), and the transcription factor GATA4.  These compds. represent leads for an important new series of drug-like epigenetic modulators with the potential for use as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVxPPqT5KXUrVg90H21EOLACvtfcHk0lixhCM31hi6jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKnsrvJ&md5=e3a9936ea336fe333cc1cbd0056c11c2</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fjm3002845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3002845%26sid%3Dliteratum%253Aachs%26aulast%3DHazeldine%26aufirst%3DS.%26aulast%3DPachaiyappan%26aufirst%3DB.%26aulast%3DSteinbergs%26aufirst%3DN.%26aulast%3DNowotarski%26aufirst%3DS.%26aulast%3DHanson%26aufirst%3DA.%2BS.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DLow%2520molecular%2520weight%2520amidoximes%2520that%2520act%2520as%2520potent%2520inhibitors%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7378%26epage%3D7391%26doi%3D10.1021%2Fjm3002845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F. J.</span></span> <span> </span><span class="NLM_article-title">Natural polyphenols inhibit lysine-specific demethylase-1 in vitro</span>. <i>J. Biochem. Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">63</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23662249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BC2srntVelsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=56-63&author=A.+Abdullaauthor=X.+P.+Zhaoauthor=F.+J.+Yang&title=Natural+polyphenols+inhibit+lysine-specific+demethylase-1+in+vitro"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Polyphenols Inhibit Lysine-Specific Demethylase-1 in vitro</span></div><div class="casAuthors">Abdulla Arian; Zhao Xiaoping; Yang Fajun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biochemical and pharmacological research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-63</span>
        ISSN:<span class="NLM_cas:issn">2168-8761</span>.
    </div><div class="casAbstract">Natural polyphenols, such as resveratrol, have beneficial functions on major human diseases such as cancer, diabetes, and cardiovascular disease.  Besides acting as antioxidants, some of these polyphenols can also target proteins to modulate specific biological pathways.  The lysine-specific histone demethylase LSD1 plays important roles in cell growth, differentiation and nutrient metabolism.  Here, we studied the effect of natural polyphenols resveratrol, curcumin, quercetin and analogs on LSD1.  Using in vitro LSD1 enzymatic assays, we show that resveratrol, curcumin and quercetin displayed a potent inhibitory effect on the LSD1 activity and were more potent than the known LSD1 inhibitor trans-2-phenylcyclopropylamine (TCP).  The new function of resveratrol, curcumin and quercetin is independent of their antioxidant properties, as other antioxidants had no effect on LSD1 under the similar conditions.  In C2C12 fibroblasts, resveratrol and curcumin can efficiently inhibit myogenic expression and differentiation, for which LSD1 is required.  Thus, our study has identified LSD1 as a novel target of bioactive natural compounds, such as resveratrol, curcumin and quercetin, and such finding suggests that LSD1 inhibition can at least partially contribute to some of the previously observed beneficial effects of these compounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTetSe7hlvX8kHdcurJClFcfW6udTcc2eacb7WL_hovhbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srntVelsg%253D%253D&md5=4a0007d63d999f4f20d4830b781ab1ed</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbdulla%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%2BP.%26aulast%3DYang%26aufirst%3DF.%2BJ.%26atitle%3DNatural%2520polyphenols%2520inhibit%2520lysine-specific%2520demethylase-1%2520in%2520vitro%26jtitle%3DJ.%2520Biochem.%2520Pharmacol.%2520Res.%26date%3D2013%26volume%3D1%26spage%3D56%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of resveratrol derivatives as novel LSD1 inhibitors: design, synthesis and their biological evaluation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2016.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27888721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGqsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2017&pages=246-258&author=Y.+C.+Duanauthor=Y.+Y.+Guanauthor=X.+Y.+Zhaiauthor=L.+N.+Dingauthor=W.+P.+Qinauthor=D.+D.+Shenauthor=X.+Q.+Liuauthor=X.+D.+Sunauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Discovery+of+resveratrol+derivatives+as+novel+LSD1+inhibitors%3A+design%2C+synthesis+and+their+biological+evaluation&doi=10.1016%2Fj.ejmech.2016.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of resveratrol derivatives as novel LSD1 inhibitors: Design, synthesis and their biological evaluation</span></div><div class="casAuthors">Duan, Ying-Chao; Guan, Yuan-Yuan; Zhai, Xiao-Yu; Ding, Li-Na; Qin, Wen-Ping; Shen, Dan-Dan; Liu, Xue-Qi; Sun, Xu-Dong; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">246-258</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Inhibition of lysine-specific demethylase 1 (LSD1) has recently emerged as an attractive therapeutic target for treating cancer and other diseases.  As a continuity of our ongoing effort to identify novel small-mol. LSD1-inhibitors, we designed and synthesized a series of resveratrol derivs., which were shown to be potent inhibitors of LSD1.  Among them, compds. I and II displayed the most potent LSD1-inhibitory activities in enzyme assays, with IC50 values of 121 nM and 123 nM, resp.  Biochem. study and docking anal. indicated that compds. I and II were reversible LSD1 inhibitors.  High content anal. showed that I and II induced a dose-dependent increase of dimethylated Lys4 of histone H3 and had no impact on the expression of LSD1 in MGC-803 cells.  Furthermore, I or II could remarkably increase the mRNA level of CD86, a surrogate cellular biomarker for LSD1 activity, in MGC-803 cells, suggesting that they are likely to exhibit LSD1-inhibitory activities intracellularly.  These findings should encourage further modification of these compds. to produce more potent LSD1 inhibitors with potential anticancer activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWLdv3p7rUK7Vg90H21EOLACvtfcHk0lheSBQ2r1GXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGqsrbN&md5=812cb448167dda2f2a2f69b269a4c594</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DGuan%26aufirst%3DY.%2BY.%26aulast%3DZhai%26aufirst%3DX.%2BY.%26aulast%3DDing%26aufirst%3DL.%2BN.%26aulast%3DQin%26aufirst%3DW.%2BP.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DLiu%26aufirst%3DX.%2BQ.%26aulast%3DSun%26aufirst%3DX.%2BD.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520resveratrol%2520derivatives%2520as%2520novel%2520LSD1%2520inhibitors%253A%2520design%252C%2520synthesis%2520and%2520their%2520biological%2520evaluation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D126%26spage%3D246%26epage%3D258%26doi%3D10.1016%2Fj.ejmech.2016.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">6000</span>– <span class="NLM_lpage">6014</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.10.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2018.10.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30448189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1WltrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=6000-6014&author=Y.+C.+Duanauthor=W.+P.+Qinauthor=F.+Z.+Suoauthor=X.+Y.+Zhaiauthor=Y.+Y.+Guanauthor=X.+J.+Wangauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Design%2C+synthesis+and+in+vitro+evaluation+of+stilbene+derivatives+as+novel+LSD1+inhibitors+for+AML+therapy&doi=10.1016%2Fj.bmc.2018.10.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy</span></div><div class="casAuthors">Duan, Yingchao; Qin, Wenping; Suo, Fengzhi; Zhai, Xiaoyu; Guan, Yuanyuan; Wang, Xiaojuan; Zheng, Yichao; Liu, Hongmin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">6000-6014</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">LSD1 is implicated in a no. of malignancies and has emerged as an exciting target.  As part of our sustained efforts to develop novel reversible LSD1 inhibitors for epigenetic therapy of cancers, in this study, we reported a series of stilbene derivs. and evaluated their LSD1 inhibitory activities, obtaining several compds. as potent LSD1 inhibitors with IC50 values in submicromolar range.  Enzyme kinetics studies and SPR assay suggested that compd. 8c, the most active LSD1 inhibitor (IC50 = 283 nM), potently inhibited LSD1 in a reversible and FAD competitive manner.  Consistent with the kinetics data, mol. docking showed that compd. 8c can be well docked into the FAD binding site of LSD1.  Flow cytometry anal. showed that compd. 8c was capable of up-regulating the expression of the surrogate cellular biomarker CD86 in THP-1 human leukemia cells, suggesting the ability to block LSD1 activity in cells.  Compd. 8c showed good inhibition against THP-1 and MOLM-13 cells with IC50 values of 5.76 and 8.34 μM, resp.  Moreover, compd. 8c significantly inhibited colony formation of THP-1 cells dose dependently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCQHvTfU4lSbVg90H21EOLACvtfcHk0lheSBQ2r1GXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1WltrjJ&md5=989826c422d025071b5533a8be86edae</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.10.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.10.037%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DQin%26aufirst%3DW.%2BP.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DZhai%26aufirst%3DX.%2BY.%26aulast%3DGuan%26aufirst%3DY.%2BY.%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520in%2520vitro%2520evaluation%2520of%2520stilbene%2520derivatives%2520as%2520novel%2520LSD1%2520inhibitors%2520for%2520AML%2520therapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D6000%26epage%3D6014%26doi%3D10.1016%2Fj.bmc.2018.10.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M. S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of curcumin analogues as novel LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">126683</span>– <span class="NLM_lpage">126688</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2019.126683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31627991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2gsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=126683-126688&author=J.+M.+Wangauthor=X.+Y.+Zhangauthor=J.+K.+Yanauthor=W.+Liauthor=Q.+W.+Jiangauthor=X.+R.+Wangauthor=D.+M.+Zhaoauthor=M.+S.+Cheng&title=Design%2C+synthesis+and+biological+evaluation+of+curcumin+analogues+as+novel+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2019.126683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of curcumin analogues as novel LSD1 inhibitors</span></div><div class="casAuthors">Wang, Jiming; Zhang, Xiangyu; Yan, Jiangkun; Li, Wei; Jiang, Qinwen; Wang, Xinran; Zhao, Dongmei; Cheng, Maosheng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">126683</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) was the first discovered histone demethylase.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development, and thus, it is an attractive mol. target for the development of novel cancer therapeutics.  In this study, we worked on the structural optimization of natural products and identified 30 novel LSD1 inhibitors.  Utilizing a structure-based drug design strategy, we designed and synthesized a series of curcumin analogs that were shown to be potent LSD1 inhibitors in the enzyme assay.  Compd. WB07 (I) displayed the most potent LSD1 inhibitory activity, with an IC50 value of 0.8μM.  Moreover, WA20 (II) showed an anticlonogenic effect on A549 cells with an IC50 value of 4.4μM.  Mol. docking simulations were also carried out, and the results indicated that the inhibitors bound to the protein active site located around the key residues of Asp555 and Asp556.  These findings suggested that compds. II and I are the first curcumin analog-based LSD1 inhibitors with remarkable A549 suppressive activity, providing a novel scaffold for the development of LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTnlsmI6uw3LVg90H21EOLACvtfcHk0lheSBQ2r1GXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2gsrzF&md5=2c6688ce5e03fcaee6f0634423523f41</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126683%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DZhang%26aufirst%3DX.%2BY.%26aulast%3DYan%26aufirst%3DJ.%2BK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DQ.%2BW.%26aulast%3DWang%26aufirst%3DX.%2BR.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520curcumin%2520analogues%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D126683%26epage%3D126688%26doi%3D10.1016%2Fj.bmcl.2019.126683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span> <span> </span><span class="NLM_article-title">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4871</span>– <span class="NLM_lpage">4880</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.08.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2018.08.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30153955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=4871-4880&author=S.+Y.+Xuauthor=C.+Zhouauthor=R.+F.+Liuauthor=Q.+H.+Zhuauthor=Y.+G.+Xuauthor=F.+Lanauthor=X.+M.+Zha&title=Optimization+of+5-arylidene+barbiturates+as+potent%2C+selective%2C+reversible+LSD1+inhibitors+for+the+treatment+of+acute+promyelocytic+leukemia&doi=10.1016%2Fj.bmc.2018.08.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia</span></div><div class="casAuthors">Xu, Siyuan; Zhou, Chen; Liu, Rongfeng; Zhu, Qihua; Xu, Yungen; Lan, Fei; Zha, Xiaoming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4871-4880</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) is overexpressed in diverse hematol. disorders and recognized as a promising target for blood medicines.  In this study, mol. docking-based virtual screening united with bioevaluation was utilized to identify novel skeleton of 5-arylidene barbiturate as small-mol. inhibitors of LSD1.  Among the synthesized derivs., I exhibited reversible and potent inhibition (IC50 = 0.41 μM) and high selectivity over the MAO-A and MAO-B.  Notably, I strongly induced differentiation effect on acute promyelocytic leukemia NB4 cell line and distinctly escalated the methylation level on histone 3 lysine 4 (H3K4).  Our findings indicate that 5-arylidene barbiturate may represent a new skeleton of LSD1 inhibitors and I deserve as a promising agent for the further research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbUIvf5kGjSbVg90H21EOLACvtfcHk0lheSBQ2r1GXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtr3F&md5=27a2e2ea365f97dba3279fee15d65a2e</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.08.026%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DZhu%26aufirst%3DQ.%2BH.%26aulast%3DXu%26aufirst%3DY.%2BG.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DZha%26aufirst%3DX.%2BM.%26atitle%3DOptimization%2520of%25205-arylidene%2520barbiturates%2520as%2520potent%252C%2520selective%252C%2520reversible%2520LSD1%2520inhibitors%2520for%2520the%2520treatment%2520of%2520acute%2520promyelocytic%2520leukemia%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D4871%26epage%3D4880%26doi%3D10.1016%2Fj.bmc.2018.08.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of orally active chalcones as histone lysine specific demethylase 1 inhibitors for the treatment of leukaemia</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">217</span>, <span class="refDoi"> DOI: 10.1080/14756366.2020.1852556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1080%2F14756366.2020.1852556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=33307878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFyqurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2021&pages=207-217&author=Y.+Liauthor=Y.+Sunauthor=Y.+Zhouauthor=X.+Y.+Liauthor=H.+Zhangauthor=G.+J.+Zhang&title=Discovery+of+orally+active+chalcones+as+histone+lysine+specific+demethylase+1+inhibitors+for+the+treatment+of+leukaemia&doi=10.1080%2F14756366.2020.1852556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of orally active chalcones as histone lysine specific demethylase 1 inhibitors for the treatment of leukaemia</span></div><div class="casAuthors">Li, Yang; Sun, Ying; Zhou, Yang; Li, Xinyang; Zhang, Huan; Zhang, Guojun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-217</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) has emerged as an attractive mol. target for the discovery of potently anticancer drugs to treat leukemia.  In this study, a series of novel chalcone derivs. were designed, synthesized and evaluated for their inhibitory activities against LSD1 in vitro.  Among all these compds., displayed the best LSD1 inhibitory activity with an IC50 value of 0.14 μM.  In the cellular level, compd. can induce the accumulation of H3K9me1/2 and inhibit cell proliferation by inactivating LSD1.  It exhibited the potent antiproliferative activity with IC50 values of 1.10 μM, 3.64 μM, 3.85 μM, 1.87 μM, 0.87 μM and 2.73 μM against HAL-01, KE-37, P30-OHK, SUP-B15, MOLT-4 and LC4-1 cells, resp.  Importantly, compd. significantly suppressed MOLT-4 xenograft tumor growth in vivo, indicating its great potential as an orally bioavailable candidate for leukemia therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLsxSVulKy4LVg90H21EOLACvtfcHk0lgg3SBbRTZ7EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFyqurzL&md5=673f155e29c88f16e77fb8da756b0a47</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1080%2F14756366.2020.1852556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14756366.2020.1852556%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%2BY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DG.%2BJ.%26atitle%3DDiscovery%2520of%2520orally%2520active%2520chalcones%2520as%2520histone%2520lysine%2520specific%2520demethylase%25201%2520inhibitors%2520for%2520the%2520treatment%2520of%2520leukaemia%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2021%26volume%3D36%26spage%3D207%26epage%3D217%26doi%3D10.1080%2F14756366.2020.1852556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sorna, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theisen, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bearss, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span> <span> </span><span class="NLM_article-title">High-throughput virtual screening identifies novel N’-(1-phenylethylidene)-benzohydrazides as potent, specific, and reversible LSD1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9496</span>– <span class="NLM_lpage">9508</span>, <span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-throughput+virtual+screening+identifies+novel+N%E2%80%99-%281-phenylethylidene%29-benzohydrazides+as+potent%2C+specific%2C+and+reversible+LSD1+inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0lgg3SBbRTZ7EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520identifies%2520novel%2520N%25E2%2580%2599-%25281-phenylethylidene%2529-benzohydrazides%2520as%2520potent%252C%2520specific%252C%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soldi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh Halder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thode, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drenner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaadige, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel Ampanattu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez Del Villar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, W. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span> <span> </span><span class="NLM_article-title">The novel reversible LSD1 inhibitor SP-2577 promotes anti-tumor immunity in SWItch/Sucrose-NonFermentable (SWI/SNF) complex mutated ovarian cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e0235705</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0235705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1371%2Fjournal.pone.0235705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32649682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVejurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&author=R.+Soldiauthor=T.+Ghosh+Halderauthor=A.+Westonauthor=T.+Thodeauthor=K.+Drennerauthor=R.+Lewisauthor=M.+R.+Kaadigeauthor=S.+Srivastavaauthor=S.+Daniel+Ampanattuauthor=R.+Rodriguez+Del+Villarauthor=J.+Langauthor=H.+Vankayalapatiauthor=B.+Weissmanauthor=J.+M.+Trentauthor=W.+P.+D.+Hendricksauthor=S.+Sharma&title=The+novel+reversible+LSD1+inhibitor+SP-2577+promotes+anti-tumor+immunity+in+SWItch%2FSucrose-NonFermentable+%28SWI%2FSNF%29+complex+mutated+ovarian+cancer&doi=10.1371%2Fjournal.pone.0235705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">The novel reversible LSD1 inhibitor SP-2577 promotes anti-tumor immunity in SWItch/Sucrose-NonFermentable (SWI/SNF) complex mutated ovarian cancer</span></div><div class="casAuthors">Soldi, Raffaella; Ghosh Halder, Tithi; Weston, Alexis; Thode, Trason; Drenner, Kevin; Lewis, Rhonda; Kaadige, Mohan R.; Srivastava, Shreyesi; Daniel Ampanattu, Sherin; Rodriguez del Villar, Ryan; Lang, Jessica; Vankayalapati, Hariprasad; Weissman, Bernard; Trent, Jeffrey M.; Hendricks, William P. D.; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0235705</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Mutations of the SWI/SNF chromatin remodeling complex occur in 20% of all human cancers, including ovarian cancer.  Approx. half of ovarian clear cell carcinomas carry mutations in the SWI/SNF subunit ARID1A, while small cell carcinoma of the ovary hypercalcemic type presents with inactivating mutations of the SWI/SNF ATPase SMARCA4 alongside epigenetic silencing of the ATPase SMARCA2.  Loss of these ATPases disrupts SWI/SNF chromatin remodeling activity and may also interfere with the function of other histone-modifying enzymes that assoc. with or are dependent on SWI/SNF activity.  One such enzyme is lysine-specific histone demethylase 1 (LSD1/KDM1A), which regulates the chromatin landscape and gene expression by demethylating proteins such as histone H3.  Cross-cancer anal. of the TCGA database shows that LSD1 is highly expressed in SWI/SNF-mutated tumors.  SCCOHT and OCCC cell lines have shown sensitivity to the reversible LSD1 inhibitor SP-2577, suggesting that SWI/SNF-deficient ovarian cancers are dependent on LSD1 activity.  In this study, we investigated the ability of SP-2577 to promote anti-tumor immunity and T-cell infiltration in SCCOHT and OCCC cell lines.  We found that SP-2577 stimulated IFN-dependent anti-tumor immunity in SCCOHT and promoted the expression of PD-L1 in both SCCOHT and OCCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzji-N7T50nrVg90H21EOLACvtfcHk0lgg3SBbRTZ7EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVejurvM&md5=4f39f8b246f8e6f2c732ac6f81a3a450</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0235705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0235705%26sid%3Dliteratum%253Aachs%26aulast%3DSoldi%26aufirst%3DR.%26aulast%3DGhosh%2BHalder%26aufirst%3DT.%26aulast%3DWeston%26aufirst%3DA.%26aulast%3DThode%26aufirst%3DT.%26aulast%3DDrenner%26aufirst%3DK.%26aulast%3DLewis%26aufirst%3DR.%26aulast%3DKaadige%26aufirst%3DM.%2BR.%26aulast%3DSrivastava%26aufirst%3DS.%26aulast%3DDaniel%2BAmpanattu%26aufirst%3DS.%26aulast%3DRodriguez%2BDel%2BVillar%26aufirst%3DR.%26aulast%3DLang%26aufirst%3DJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DWeissman%26aufirst%3DB.%26aulast%3DTrent%26aufirst%3DJ.%2BM.%26aulast%3DHendricks%26aufirst%3DW.%2BP.%2BD.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DThe%2520novel%2520reversible%2520LSD1%2520inhibitor%2520SP-2577%2520promotes%2520anti-tumor%2520immunity%2520in%2520SWItch%252FSucrose-NonFermentable%2520%2528SWI%252FSNF%2529%2520complex%2520mutated%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2020%26volume%3D15%26doi%3D10.1371%2Fjournal.pone.0235705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harancher, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packard, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLuca, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dembowski, J. A.</span></span> <span> </span><span class="NLM_article-title">Antiviral properties of the LSD1 inhibitor SP-2509</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">e00974</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1128/JVI.00974-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1128%2FJVI.00974-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32699090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitV2jsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2020&pages=e00974-20&author=M.+R.+Harancherauthor=J.+E.+Packardauthor=S.+P.+Cowanauthor=N.+A.+DeLucaauthor=J.+A.+Dembowski&title=Antiviral+properties+of+the+LSD1+inhibitor+SP-2509&doi=10.1128%2FJVI.00974-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Antiviral properties of the LSD1 inhibitor SP-2509</span></div><div class="casAuthors">Harancher, Mitchell R.; Packard, Jessica E.; Cowan, Shane P.; DeLuca, Neal A.; Dembowski, Jill A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">e00974</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) targets cellular proteins, including histone H3, p53, E2F, and Dnmt1, and is involved in the regulation of gene expression, DNA replication, the cell cycle, and the DNA damage response.  LSD1 catalyzes demethylation of histone H3K9 assocd. with herpes simplex virus 1 (HSV-1) immediate early (IE) promoters and is necessary for IE gene expression, viral DNA replication, and reactivation from latency.  We previously found that LSD1 assocs. with HSV-1 replication forks and replicating viral DNA, suggesting that it may play a direct role in viral replication or coupled processes.  We investigated the effects of the LSD1 inhibitor SP-2509 on the HSV-1 life cycle.  Unlike previously investigated LSD1 inhibitors tranylcypromine (TCP) and OG-L002, which covalently attach to the LSD1 cofactor FAD (FAD) to inhibit demethylase activity, SP-2509 has previously been shown to inhibit LSD1 protein-protein interactions.  We found that SP-2509 does not inhibit HSV-1 IE gene expression or transcription factor and RNA polymerase II (Pol II) assocn. with viral DNA prior to the onset of replication.  However, SP-2509 does inhibit viral DNA replication, late gene expression, and virus prodn.  We used EdC labeling of nascent viral DNA to image aberrant viral replication compartments that form in the presence of SP-2509.  Treatment resulted in the formation of small replication foci that colocalize with replication proteins but are defective for Pol II recruitment.  Taken together, these data highlight a potential new role for LSD1 in the regulation of HSV-1 DNA replication and gene expression after the onset of DNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBYhNfWFB6uLVg90H21EOLACvtfcHk0lgd_gbdSCCVwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitV2jsb3F&md5=84e266762880c90ff0ffdd95cdd853bc</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1128%2FJVI.00974-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00974-20%26sid%3Dliteratum%253Aachs%26aulast%3DHarancher%26aufirst%3DM.%2BR.%26aulast%3DPackard%26aufirst%3DJ.%2BE.%26aulast%3DCowan%26aufirst%3DS.%2BP.%26aulast%3DDeLuca%26aufirst%3DN.%2BA.%26aulast%3DDembowski%26aufirst%3DJ.%2BA.%26atitle%3DAntiviral%2520properties%2520of%2520the%2520LSD1%2520inhibitor%2520SP-2509%26jtitle%3DJ.%2520Virol.%26date%3D2020%26volume%3D94%26spage%3De00974%26epage%3D20%26doi%3D10.1128%2FJVI.00974-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reed, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascarenhas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chawla, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wages, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenehjem, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiesteban, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DuBois, S. G.</span></span> <span> </span><span class="NLM_article-title">A phase I/II clinical trial of the reversible LSD1 inhibitor, seclidemstat, in patients with relapsed/refractory Ewing sarcoma</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">TPS11567</span>, <span class="refDoi"> DOI: 10.1200/JCO.2020.38.15_suppl.TPS11567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1200%2FJCO.2020.38.15_suppl.TPS11567" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2020&pages=TPS11567&author=D.+R.+Reedauthor=L.+Mascarenhasauthor=P.+A.+Meyersauthor=S.+P.+Chawlaauthor=D.+J.+Harrisonauthor=B.+Settyauthor=J.+Mettsauthor=D.+S.+Wagesauthor=D.+D.+Stenehjemauthor=D.+Y.+Santiestebanauthor=S.+G.+DuBois&title=A+phase+I%2FII+clinical+trial+of+the+reversible+LSD1+inhibitor%2C+seclidemstat%2C+in+patients+with+relapsed%2Frefractory+Ewing+sarcoma&doi=10.1200%2FJCO.2020.38.15_suppl.TPS11567"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1200%2FJCO.2020.38.15_suppl.TPS11567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2020.38.15_suppl.TPS11567%26sid%3Dliteratum%253Aachs%26aulast%3DReed%26aufirst%3DD.%2BR.%26aulast%3DMascarenhas%26aufirst%3DL.%26aulast%3DMeyers%26aufirst%3DP.%2BA.%26aulast%3DChawla%26aufirst%3DS.%2BP.%26aulast%3DHarrison%26aufirst%3DD.%2BJ.%26aulast%3DSetty%26aufirst%3DB.%26aulast%3DMetts%26aufirst%3DJ.%26aulast%3DWages%26aufirst%3DD.%2BS.%26aulast%3DStenehjem%26aufirst%3DD.%2BD.%26aulast%3DSantiesteban%26aufirst%3DD.%2BY.%26aulast%3DDuBois%26aufirst%3DS.%2BG.%26atitle%3DA%2520phase%2520I%252FII%2520clinical%2520trial%2520of%2520the%2520reversible%2520LSD1%2520inhibitor%252C%2520seclidemstat%252C%2520in%2520patients%2520with%2520relapsed%252Frefractory%2520Ewing%2520sarcoma%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2020%26volume%3D38%26spage%3DTPS11567%26doi%3D10.1200%2FJCO.2020.38.15_suppl.TPS11567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. L.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel (E)-N’-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides as potent LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4552</span>– <span class="NLM_lpage">4557</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.06.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2015.06.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27524309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOhtrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4552-4557&author=Y.+Zhouauthor=Y.+Liauthor=W.+J.+Wangauthor=P.+Xiangauthor=X.+M.+Luoauthor=L.+Yangauthor=S.+Y.+Yangauthor=Y.+L.+Zhao&title=Synthesis+and+biological+evaluation+of+novel+%28E%29-N%E2%80%99-%282%2C3-dihydro-1H-inden-1-ylidene%29+benzohydrazides+as+potent+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2015.06.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of novel (E)-N'-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides as potent LSD1 inhibitors</span></div><div class="casAuthors">Zhou, Yang; Li, Yan; Wang, Wen-Jing; Xiang, Pu; Luo, Xin-Mei; Yang, Li; Yang, Sheng-Yong; Zhao, Ying-Lan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4552-4557</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is recognized as an attractive therapeutic target in multiple malignancies.  A series of novel (E)-N'-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides were synthesized and biol. evaluated for their potential LSD1 inhibitory effect.  Among them, compds. (E)-N'-(7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide a and (E)-N'-(4-chloro-7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide showed the most potent LSD1 inhibitory activity with IC50 values of 1.4 and 1.7 nM, resp., which were ∼10 times more potent compared with (E)-N-(1-(5-chloro-2-hydroxyphenyl) ethylidene)-3-(morpholinosulf-only) benzohydrazide.  Compds. (E)-N'-(7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide a and (E)-N'-(4-chloro-7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide also exhibited marked anti-proliferation activities against cancer cell lines that highly expressed LSD1.  These results suggest that these optimized compds. might be served as promising LSD1 inhibitors against cancer, which merit further study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6SXQSBWFZcLVg90H21EOLACvtfcHk0lgd_gbdSCCVwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOhtrfL&md5=f0469735243cb2b9b569cd4fdeba9a85</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.06.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.06.054%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%2BJ.%26aulast%3DXiang%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DX.%2BM.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%2BY.%26aulast%3DZhao%26aufirst%3DY.%2BL.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520%2528E%2529-N%25E2%2580%2599-%25282%252C3-dihydro-1H-inden-1-ylidene%2529%2520benzohydrazides%2520as%2520potent%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4552%26epage%3D4557%26doi%3D10.1016%2Fj.bmcl.2015.06.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xi, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological activity of 3-oxoamino-benzenesulfonamides as selective and reversible LSD1 inhibitors</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2017.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2017.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28460360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFCjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2017&pages=182-189&author=J.+Y.+Xiauthor=S.+Y.+Xuauthor=L.+M.+Wuauthor=T.+F.+Maauthor=R.+F.+Liuauthor=Y.+C.+Liuauthor=D.+W.+Dengauthor=Y.+Q.+Guauthor=J.+P.+Zhouauthor=F.+Lanauthor=X.+M.+Zha&title=Design%2C+synthesis+and+biological+activity+of+3-oxoamino-benzenesulfonamides+as+selective+and+reversible+LSD1+inhibitors&doi=10.1016%2Fj.bioorg.2017.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological activity of 3-oxoamino-benzenesulfonamides as selective and reversible LSD1 inhibitors</span></div><div class="casAuthors">Xi, Jiayue; Xu, Siyuan; Wu, Liming; Ma, Tianfang; Liu, Rongfeng; Liu, Yu-Chih; Deng, Dawei; Gu, Yueqing; Zhou, Jinpei; Lan, Fei; Zha, Xiaoming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">182-189</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) is a flavin-dependent amine oxidase that selectively removes one or two Me groups from H3 at Lys4 and is recognized as a promising therapeutic target for cancer and other diseases.  Here, a series of 3-oxoamino-benzenesulfonamides were synthesized and evaluated for their inhibitory activity against LSD1.  Compds. I and II showed the most potent inhibition with the IC50 values of 9.5 and 6.9 μM, resp.  Furthermore, the LSD1 inhibition of I and II were reversible and selective.  Docking study presented the possible binding mode between 7b, 7h and the LSD1 active site.  Taken together, 3-oxoamino-benzenesulfonamides may represent a new class of reversible LSD1 inhibitors and 7b and 7h were two hit compds. deserved further structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsoVXX7-ETcrVg90H21EOLACvtfcHk0lh-EOdAjC6Ehg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFCjtbg%253D&md5=1ded182ebd4f36de601a0d2d7dfcb93d</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2017.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2017.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DXi%26aufirst%3DJ.%2BY.%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DWu%26aufirst%3DL.%2BM.%26aulast%3DMa%26aufirst%3DT.%2BF.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDeng%26aufirst%3DD.%2BW.%26aulast%3DGu%26aufirst%3DY.%2BQ.%26aulast%3DZhou%26aufirst%3DJ.%2BP.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DZha%26aufirst%3DX.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520activity%2520of%25203-oxoamino-benzenesulfonamides%2520as%2520selective%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Chem.%26date%3D2017%26volume%3D72%26spage%3D182%26epage%3D189%26doi%3D10.1016%2Fj.bioorg.2017.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological activity of N-(3-substituted-phenyl)benzenesulfonamides as selective and reversible LSD1 inhibitors</span>. <i>Med. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2822</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1007/s00044-016-1706-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00044-016-1706-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCgsLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=2822-2831&author=X.+M.+Zhaauthor=L.+M.+Wuauthor=S.+Y.+Xuauthor=F.+X.+Zouauthor=J.+X.+Xiauthor=T.+F.+Maauthor=R.+F.+Liuauthor=Y.+C.+Liuauthor=D.+W.+Dengauthor=Y.+Q.+Guauthor=J.+P.+Zhouauthor=F.+Lan&title=Design%2C+synthesis+and+biological+activity+of+N-%283-substituted-phenyl%29benzenesulfonamides+as+selective+and+reversible+LSD1+inhibitors&doi=10.1007%2Fs00044-016-1706-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological activity of N-(3-substituted-phenyl)benzenesulfonamides as selective and reversible LSD1 inhibitors</span></div><div class="casAuthors">Zha, Xiaoming; Wu, Liming; Xu, Siyuan; Zou, Fangxia; Xi, Jiayue; Ma, Tianfang; Liu, Rongfeng; Liu, Yu-Chih; Deng, Dawei; Gu, Yueqing; Zhou, Jinpei; Lan, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2822-2831</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 plays a crucial role in regulating histone methylation at residues K4 and K9 on histone H3 and over-expresses in a variety of cancers.  Here the authors designed, synthesized and evaluated a series of N-(3-substituted-phenyl)benzenesulfonamides as reversible lysine specific demethylase 1 inhibitors.  All the compds. exhibited lysine specific demethylase 1 inhibition with the half maximal inhibitory concn. (IC50) values between 7.5 and 68 μM.  Three most active compds. including I displayed only modest effect on FAD-dependent enzymes monoamine oxidases A and B, and the reversibility of lysine specific demethylase 1 inhibition were confirmed.  Mol. docking was also carried out to predict the binding mode of I into the active site of lysine specific demethylase 1.  Taken together, N-(3-substituted-phenyl)benzenesulfonamides including I represent a new class of selective and reversible lysine specific demethylase 1 inhibitors with pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouiqDa8FKmmbVg90H21EOLACvtfcHk0lh-EOdAjC6Ehg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCgsLbP&md5=252a51da6c0983e8774887f8bb6ae284</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1007%2Fs00044-016-1706-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-016-1706-8%26sid%3Dliteratum%253Aachs%26aulast%3DZha%26aufirst%3DX.%2BM.%26aulast%3DWu%26aufirst%3DL.%2BM.%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DZou%26aufirst%3DF.%2BX.%26aulast%3DXi%26aufirst%3DJ.%2BX.%26aulast%3DMa%26aufirst%3DT.%2BF.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDeng%26aufirst%3DD.%2BW.%26aulast%3DGu%26aufirst%3DY.%2BQ.%26aulast%3DZhou%26aufirst%3DJ.%2BP.%26aulast%3DLan%26aufirst%3DF.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520activity%2520of%2520N-%25283-substituted-phenyl%2529benzenesulfonamides%2520as%2520selective%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2016%26volume%3D25%26spage%3D2822%26epage%3D2831%26doi%3D10.1007%2Fs00044-016-1706-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellaisamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span> <span> </span><span class="NLM_article-title">A rhodium(III)-based inhibitor of lysine-specific histone demethylase 1 as an epigenetic modulator in prostate cancer cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2597</span>– <span class="NLM_lpage">2603</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVajtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2597-2603&author=C.+Yangauthor=W.+Wangauthor=J.+X.+Liangauthor=G.+Liauthor=K.+Vellaisamyauthor=C.+Y.+Wongauthor=D.+L.+Maauthor=C.+H.+Leung&title=A+rhodium%28III%29-based+inhibitor+of+lysine-specific+histone+demethylase+1+as+an+epigenetic+modulator+in+prostate+cancer+cells&doi=10.1021%2Facs.jmedchem.7b00133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">A Rhodium(III)-Based Inhibitor of Lysine-Specific Histone Demethylase 1 as an Epigenetic Modulator in Prostate Cancer Cells</span></div><div class="casAuthors">Yang, Chao; Wang, Wanhe; Liang, Jia-Xin; Li, Guodong; Vellaisamy, Kasipandi; Wong, Chun-Yuen; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2597-2603</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report herein a novel rhodium(III) complex 1 (I) as a new LSD1 targeting agent and epigenetic modulator.  Complex 1 disrupted the interaction of LSD1-H3K4me2 in human prostate carcinoma cells, and enhanced the amplification of p21, FOXA2 and BMP2 gene promoters.  Complex 1 was selective for LSD1 over other histone demethylases, such as KDM2b, KDM7 and MAO activities, and also showed anti-proliferative activity towards human cancer cells.  To date, complex 1 is the first metal-based inhibitor of LSD1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobbBaU5SOfBrVg90H21EOLACvtfcHk0lh-EOdAjC6Ehg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVajtr8%253D&md5=6c9c9729423cf36a645c0f77ccf186b9</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00133%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DJ.%2BX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DVellaisamy%26aufirst%3DK.%26aulast%3DWong%26aufirst%3DC.%2BY.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DA%2520rhodium%2528III%2529-based%2520inhibitor%2520of%2520lysine-specific%2520histone%2520demethylase%25201%2520as%2520an%2520epigenetic%2520modulator%2520in%2520prostate%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2597%26epage%3D2603%26doi%3D10.1021%2Facs.jmedchem.7b00133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G. Q.</span></span> <span> </span><span class="NLM_article-title">Discovery of vanadium complexes bearing tridentate schiff base ligands as novel LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">683</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2018.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30611616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksl2isw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=681-683&author=L.+P.+Luauthor=J.+H.+Liuauthor=S.+H.+Cenauthor=Y.+L.+Jiangauthor=G.+Q.+Hu&title=Discovery+of+vanadium+complexes+bearing+tridentate+schiff+base+ligands+as+novel+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2018.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of vanadium complexes bearing tridentate schiff base ligands as novel LSD1 inhibitors</span></div><div class="casAuthors">Lu, Ling-Pan; Liu, Jin-Hai; Cen, Shi-Hong; Jiang, Ya-Ling; Hu, Guo-Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">681-683</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase (LSD1) plays a pivotal role in epigenetic modulation of gene expression.  Aberrant expression of LSD1 was assocd. with the progress and oncogenesis of multiple human cancers.  Herein, the authors report the preliminary anti-LSD1 evaluation of the synthetic vanadium (V) complexes.  Among them, complex 2 showed a moderate inhibitory effect against LSD1 with IC50 value of 19.0 μM, as well as good selectivity over MAO-A/B.  Complex 2 is the first vanadium based LSD1 inhibitor, which provides a novel scaffold for the development of LSD1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWDIJ04kf_8LVg90H21EOLACvtfcHk0lh_cA1k_WUDog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksl2isw%253D%253D&md5=7b4fa22c0dd01d42ef867dfc88e1d36b</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DL.%2BP.%26aulast%3DLiu%26aufirst%3DJ.%2BH.%26aulast%3DCen%26aufirst%3DS.%2BH.%26aulast%3DJiang%26aufirst%3DY.%2BL.%26aulast%3DHu%26aufirst%3DG.%2BQ.%26atitle%3DDiscovery%2520of%2520vanadium%2520complexes%2520bearing%2520tridentate%2520schiff%2520base%2520ligands%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D681%26epage%3D683%26doi%3D10.1016%2Fj.bmcl.2018.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremberg, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cartic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turlais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D.</span></span> <span> </span><span class="NLM_article-title">Development of (4-cyanophenyl)glycine derivatives as reversible inhibitors of lysine specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7984</span>– <span class="NLM_lpage">7999</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00462</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00462" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKqsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7984-7999&author=D.+P.+Mouldauthor=C.+Alliauthor=U.+Brembergauthor=S.+Carticauthor=A.+M.+Jordanauthor=M.+Geitmannauthor=A.+Maiques-Diazauthor=A.+E.+McGonagleauthor=T.+C.+P.+Somervailleauthor=G.+J.+Spencerauthor=F.+Turlaisauthor=D.+Ogilvie&title=Development+of+%284-cyanophenyl%29glycine+derivatives+as+reversible+inhibitors+of+lysine+specific+demethylase+1&doi=10.1021%2Facs.jmedchem.7b00462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Development of (4-Cyanophenyl)glycine Derivatives as Reversible Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Mould, Daniel P.; Alli, Cristina; Bremberg, Ulf; Cartic, Sharon; Jordan, Allan M.; Geitmann, Matthis; Maiques-Diaz, Alba; McGonagle, Alison E.; Somervaille, Tim C. P.; Spencer, Gary J.; Turlais, Fabrice; Ogilvie, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7984-7999</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of lysine specific demethylase 1 (LSD1) has been shown to induce the differentiation of leukemia stem cells in acute myeloid leukemia (AML).  Irreversible inhibitors developed from the nonspecific inhibitor tranylcypromine have entered clin. trials; however, the development of effective reversible inhibitors has proved more challenging.  Herein, we describe our efforts to identify reversible inhibitors of LSD1 from a high throughput screen and subsequent in silico modeling approaches.  From a single hit (12) validated by biochem. and biophys. assays, we describe our efforts to develop acyclic scaffold-hops from GSK-690 (1).  A further scaffold modification to a (4-cyanophenyl)glycinamide (e.g., 29a) led to the development of compd. 32, with a Kd value of 32 nM and an EC50 value of 0.67 μM in a surrogate cellular biomarker assay.  Moreover, this deriv. does not display the same level of hERG liability as obsd. with 1 and represents a promising lead for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5NyXkmZPOqrVg90H21EOLACvtfcHk0lh_cA1k_WUDog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKqsrrJ&md5=af673e7b3a2815aa09fe7264a3e4247b</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00462%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DAlli%26aufirst%3DC.%26aulast%3DBremberg%26aufirst%3DU.%26aulast%3DCartic%26aufirst%3DS.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DTurlais%26aufirst%3DF.%26aulast%3DOgilvie%26aufirst%3DD.%26atitle%3DDevelopment%2520of%2520%25284-cyanophenyl%2529glycine%2520derivatives%2520as%2520reversible%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7984%26epage%3D7999%26doi%3D10.1021%2Facs.jmedchem.7b00462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. B.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel dual-acting modulators targeting both estrogen receptor alpha (ERalpha) and lysine-specific demethylase 1 (LSD1) for treatment of breast cancer</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">112281</span>– <span class="NLM_lpage">112285</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2020.112281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32283297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntVOisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2020&pages=112281-112285&author=M.+Heauthor=W.+Ningauthor=Z.+Huauthor=J.+Huangauthor=C.+Dongauthor=H.+B.+Zhou&title=Design%2C+synthesis+and+biological+evaluation+of+novel+dual-acting+modulators+targeting+both+estrogen+receptor+alpha+%28ERalpha%29+and+lysine-specific+demethylase+1+%28LSD1%29+for+treatment+of+breast+cancer&doi=10.1016%2Fj.ejmech.2020.112281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel dual-acting modulators targeting both estrogen receptor α (ERα) and lysine-specific demethylase 1 (LSD1) for treatment of breast cancer</span></div><div class="casAuthors">He, Ming; Ning, Wentao; Hu, Zhiye; Huang, Jian; Dong, Chune; Zhou, Hai-Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112281</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Breast cancer is a multi-factor disease, thus more and more drug combination therapies are applied in the treatment.  However, there are undeniable disadvantages in drug combination therapy.  Therefore, the development of new dual-targeting drugs has become a new strategy.  In this study, we have developed a series of dual-acting agents targeting both estrogen receptor α (ERα) and histone demethylase based on a privileged OBHS pharmacophore scaffold developed previously by our lab.  These novel OBHS-LSD1i conjugates showed excellent ERα binding affinity and selectivity, and exhibited potent inhibitory activity against lysine specific demethylase 1 (LSD1).  Several conjugates showed higher antiproliferative efficacy in MCF-7 breast cancer cell line compared to 4-hydroxytamoxifen in vitro.  Among them, the best compd. 11g displayed potent inhibitory activity against LSD1 and MCF-7 cells with IC50 values of 1.55 μM and 8.79 μM, resp.  Flow cytometry anal. of apoptosis of 11g suggested that the effect of this type compds. on MCF-7 cells is partly caused by inducing apoptosis.  Moreover, the mol. docking of 11g with ERα and the active site of LSD1/CoREST complex provides practical way for understanding the dual mechanism actions of this kind of compds. with the targets.  As such, these compds. have shown potential to become promising leads for the development of highly efficient dual-acting modulators for breast cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLGygeo7gyc7Vg90H21EOLACvtfcHk0lh_cA1k_WUDog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntVOisbk%253D&md5=baa26f620cad01fbe06964b00dedd351</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112281%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26aulast%3DNing%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DH.%2BB.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520dual-acting%2520modulators%2520targeting%2520both%2520estrogen%2520receptor%2520alpha%2520%2528ERalpha%2529%2520and%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520for%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D195%26spage%3D112281%26epage%3D112285%26doi%3D10.1016%2Fj.ejmech.2020.112281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L. Y.</span></span> <span> </span><span class="NLM_article-title">Bioactivity evaluation of natural product alpha-mangostin as a novel xanthone-based lysine-specific demethylase 1 inhibitor to against tumor metastasis</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2017.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2017.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29274582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWltr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2018&pages=415-419&author=C.+Hanauthor=Z.+R.+Liauthor=J.+Q.+Houauthor=Z.+Wangauthor=D.+Q.+Xuauthor=G.+M.+Xueauthor=L.+Y.+Kong&title=Bioactivity+evaluation+of+natural+product+alpha-mangostin+as+a+novel+xanthone-based+lysine-specific+demethylase+1+inhibitor+to+against+tumor+metastasis&doi=10.1016%2Fj.bioorg.2017.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivity evaluation of natural product α-mangostin as a novel xanthone-based lysine-specific demethylase 1 inhibitor to against tumor metastasis</span></div><div class="casAuthors">Han, Chao; Li, Zhongrui; Hou, Jiqin; Wang, Zhen; Xu, Dingqiao; Xue, Guimin; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">415-419</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), which has been reported to be overexpressed in several human cancers, has recently emerged as an attractive therapeutic target for treating cancer.  To date, almost all the developed LSD1 inhibitors are chemo-synthesized mols., while α-mangostin is first characterized as xanthone-based natural inhibitor in the current study with IC50 values of 2.81±0.44μM.  Bioactivity study and docking anal. indicated that α-mangostin could inhibit MDA-MB-231 cells migration and evasion through inhibit intracellular LSD1 activity.  These findings provides new mol. skeleton for LSD1 inhibitor study and should encourage further modification of α-mangostin to produce more potent LSD1 inhibitors with potential anticancer activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPGpOCbxAuNLVg90H21EOLACvtfcHk0ljteYAiQIOEuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWltr3J&md5=27679988f1a7c98127c71c3799734c5f</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2017.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2017.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DHou%26aufirst%3DJ.%2BQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DD.%2BQ.%26aulast%3DXue%26aufirst%3DG.%2BM.%26aulast%3DKong%26aufirst%3DL.%2BY.%26atitle%3DBioactivity%2520evaluation%2520of%2520natural%2520product%2520alpha-mangostin%2520as%2520a%2520novel%2520xanthone-based%2520lysine-specific%2520demethylase%25201%2520inhibitor%2520to%2520against%2520tumor%2520metastasis%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D76%26spage%3D415%26epage%3D419%26doi%3D10.1016%2Fj.bioorg.2017.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Natural protoberberine alkaloids, identified as potent selective LSD1 inhibitors, induce AML cell differentiation</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">103648</span>– <span class="NLM_lpage">103655</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.103648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2020.103648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32065882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFajsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2020&pages=103648-103655&author=Z.+R.+Liauthor=F.+Z.+Suoauthor=Y.+J.+Guoauthor=H.+F.+Chengauthor=S.+H.+Niuauthor=D.+D.+Shenauthor=L.+J.+Zhaoauthor=Z.+Z.+Liuauthor=M.+Maaauthor=B.+Yuauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Natural+protoberberine+alkaloids%2C+identified+as+potent+selective+LSD1+inhibitors%2C+induce+AML+cell+differentiation&doi=10.1016%2Fj.bioorg.2020.103648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Natural protoberberine alkaloids, identified as potent selective LSD1 inhibitors, induce AML cell differentiation</span></div><div class="casAuthors">Li, Zhong-Rui; Suo, Feng-Zhi; Guo, Yan-Jia; Cheng, Hai-Fang; Niu, Sheng-Hui; Shen, Dan-Dan; Zhao, Li-Juan; Liu, Zhen-Zhen; Maa, Mamun; Yu, Bin; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103648</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Natural protoberberine alkaloids were first identified and characterized as potent, selective and cellular active lysine specific demethylase 1 (LSD1) inhibitors.  Due to our study, isoquinoline-based tetracyclic scaffold was identified as the key structural element for their anti-LSD1 activity, subtle changes of substituents attached to the core structure led to dramatic changes of the activity.  Among these protoberberine alkaloids, epiberberine potently inhibited LSD1 (IC50 = 0.14 ± 0.01μM) and was highly selective to LSD1 over MAO-A/B.  Furthermore, epiberberine could induce the expression of CD86, CD11b and CD14 in THP-1 and HL-60 cells, confirming its cellular activity of inducing acute myeloid leukemia (AML) cells differentiation.  Moreover, epiberberine prolonged the survival of THP-1 cells bearing mice and inhibited the growth of AML cells in vivo without obvious global toxicity.  These findings give the potential application of epiberberine in AML treatment, and the isoquinoline-based tetracyclic scaffold could be used for further development of LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3QhTkrzh9qrVg90H21EOLACvtfcHk0ljteYAiQIOEuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFajsr0%253D&md5=4868d9e25f504a93a3c79b01dce5ccc1</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.103648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.103648%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DGuo%26aufirst%3DY.%2BJ.%26aulast%3DCheng%26aufirst%3DH.%2BF.%26aulast%3DNiu%26aufirst%3DS.%2BH.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DLiu%26aufirst%3DZ.%2BZ.%26aulast%3DMaa%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DNatural%2520protoberberine%2520alkaloids%252C%2520identified%2520as%2520potent%2520selective%2520LSD1%2520inhibitors%252C%2520induce%2520AML%2520cell%2520differentiation%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D97%26spage%3D103648%26epage%3D103655%26doi%3D10.1016%2Fj.bioorg.2020.103648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cang, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Capsaicin: a “hot” KDM1A/LSD1 inhibitor from peppers</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">104161</span>– <span class="NLM_lpage">104165</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.104161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2020.104161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32889380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslynsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2020&pages=104161-104165&author=G.+Jiaauthor=S.+D.+Cangauthor=P.+Z.+Maauthor=Z.+Y.+Song&title=Capsaicin%3A+a+%E2%80%9Chot%E2%80%9D+KDM1A%2FLSD1+inhibitor+from+peppers&doi=10.1016%2Fj.bioorg.2020.104161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Capsaicin: A "hot" KDM1A/LSD1 inhibitor from peppers</span></div><div class="casAuthors">Jia, Gang; Cang, Shundong; Ma, Peizhi; Song, Zhiyu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104161</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Capsaicinoids are plant secondary metabolites, and capsaicin is the main principal that responsible to the pungency of chili peppers, with widely application as food additive.  In our study, capsaicin was characterized as lysine specific demethylase 1A (KDM1A/LSD1) inhibitor with IC50 of 0.6 ± 0.0421 μM in biochem. level, and can bind KDM1A recombinant directly and reversibly.  Further cellular study confirmed that capsaicin can bind and inhibit KDM1A in gastric cancer cell line BGC-823 and further inhibit cell invasion and migration by reversing epithelial-mesenchymal transition (EMT).  In sum, our findings identified KDM1A as a target of capsaicin and reveals capsaicin as a modifier of histone methylation for the first time, which may provide a new skeleton for further optimization of KDM1A inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv0ax5ObfkzLVg90H21EOLACvtfcHk0ljteYAiQIOEuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslynsL%252FE&md5=6695d3eabad21b8ef51ec6515c0eae9e</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.104161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.104161%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DG.%26aulast%3DCang%26aufirst%3DS.%2BD.%26aulast%3DMa%26aufirst%3DP.%2BZ.%26aulast%3DSong%26aufirst%3DZ.%2BY.%26atitle%3DCapsaicin%253A%2520a%2520%25E2%2580%259Chot%25E2%2580%259D%2520KDM1A%252FLSD1%2520inhibitor%2520from%2520peppers%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D103%26spage%3D104161%26epage%3D104165%26doi%3D10.1016%2Fj.bioorg.2020.104161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">TCPs: privileged scaffolds for identifying potent LSD1 inhibitors for cancer therapy</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.2217/epi-2015-0002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2217%2Fepi-2015-0002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27102879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XotVSrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=651-666&author=Y.+C.+Zhengauthor=B.+Yuauthor=Z.+S.+Chenauthor=Y.+Liuauthor=H.+M.+Liu&title=TCPs%3A+privileged+scaffolds+for+identifying+potent+LSD1+inhibitors+for+cancer+therapy&doi=10.2217%2Fepi-2015-0002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">TCPs: privileged scaffolds for identifying potent LSD1 inhibitors for cancer therapy</span></div><div class="casAuthors">Zheng, Yi-Chao; Yu, Bin; Chen, Zhe-Sheng; Liu, Ying; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">651-666</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Since the first lysine-specific demethylase (KDM), lysine-specific demethylase 1 (LSD1), was characterized in 2004, several families of KDMs have been identified.  LSD1 can specifically demethylate H3K4me1/2, H3K9me1/2 as well as some nonhistone substrates.  It has been demonstrated to be an oncogene as well as a drug target.  Hence, tens of small-mol. LSD1 inhibitors have been designed, synthesized and applied for cancer treatment.  However, the two LSD1 inhibitors that have been advanced into early phase clin. trials are trans-2-phenylcyclopropylamine (TCP) derivs., which indicate that TCP is a druggable scaffold for LSD1 inhibitor.  Here, we review the design, synthesis and properties of reported TCP-based LSD1 inhibitors as well as their biol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQtSkXNPl_7LVg90H21EOLACvtfcHk0lgiMu_Brv8x3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotVSrsrs%253D&md5=6f8021101929c931a6769dce538010c2</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.2217%2Fepi-2015-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2015-0002%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DZ.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTCPs%253A%2520privileged%2520scaffolds%2520for%2520identifying%2520potent%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D8%26spage%3D651%26epage%3D666%26doi%3D10.2217%2Fepi-2015-0002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Viegas-Junior, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danuello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Silva
Bolzani, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barreiro, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraga, C. A.</span></span> <span> </span><span class="NLM_article-title">Molecular hybridization: a useful tool in the design of new drug prototypes</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1829</span>– <span class="NLM_lpage">1852</span>, <span class="refDoi"> DOI: 10.2174/092986707781058805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2174%2F092986707781058805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17627520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvFanuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1829-1852&author=C.+Viegas-Juniorauthor=A.+Danuelloauthor=V.+da+Silva%0ABolzaniauthor=E.+J.+Barreiroauthor=C.+A.+Fraga&title=Molecular+hybridization%3A+a+useful+tool+in+the+design+of+new+drug+prototypes&doi=10.2174%2F092986707781058805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular hybridization: a useful tool in the design of new drug prototypes</span></div><div class="casAuthors">Viegas-Junior, Claudio; Danuello, Amanda; da Silva Bolzani, Vanderlan; Barreiro, Eliezer J.; Fraga, Carlos Alberto Manssour</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1829-1852</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Mol. hybridization is a new concept in drug design and development based on the combination of pharmacophoric moieties of different bioactive substances to produce a new hybrid compd. with improved affinity and efficacy, when compared to the parent drugs.  Addnl., this strategy can result in compds. presenting modified selectivity profile, different and/or dual modes of action and reduced undesired side effects.  So, in this paper, we described several examples of different strategies for drug design, discovery and pharmacomodulation focused on new innovative hybrid compds. presenting analgesic, anti-inflammatory, platelet anti-aggregating, anti-infectious, anticancer, cardio- and neuroactive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvbzA71FPqLVg90H21EOLACvtfcHk0lgiMu_Brv8x3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvFanuro%253D&md5=7d3a16e3deeacb735bc0357fc6a99c37</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.2174%2F092986707781058805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986707781058805%26sid%3Dliteratum%253Aachs%26aulast%3DViegas-Junior%26aufirst%3DC.%26aulast%3DDanuello%26aufirst%3DA.%26aulast%3Dda%2BSilva%2BBolzani%26aufirst%3DV.%26aulast%3DBarreiro%26aufirst%3DE.%2BJ.%26aulast%3DFraga%26aufirst%3DC.%2BA.%26atitle%3DMolecular%2520hybridization%253A%2520a%2520useful%2520tool%2520in%2520the%2520design%2520of%2520new%2520drug%2520prototypes%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2007%26volume%3D14%26spage%3D1829%26epage%3D1852%26doi%3D10.2174%2F092986707781058805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sampath Kumar, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsogoeva, S. B.</span></span> <span> </span><span class="NLM_article-title">Structural hybridization as a facile approach to new drug candidates</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127514</span>– <span class="NLM_lpage">127528</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2020.127514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32860980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFGqurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127514-127528&author=H.+M.+Sampath+Kumarauthor=L.+Herrmannauthor=S.+B.+Tsogoeva&title=Structural+hybridization+as+a+facile+approach+to+new+drug+candidates&doi=10.1016%2Fj.bmcl.2020.127514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Structural hybridization as a facile approach to new drug candidates</span></div><div class="casAuthors">Sampath Kumar, Halmuthur M.; Herrmann, Lars; Tsogoeva, Svetlana B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">127514</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Structural hybridization of preclinically and clin. validated pharmacol. active mols. has emerged as a promising tool to develop new generations of safe and highly efficient drug candidates against various diseases including microbial infections, virus infections and cancer.  Strategies of drug-drug combinations have been adopted to generate hybrid conjugates of many clin. used drugs, designed to address inherent problems assocd. with these drugs.  Thus, the design of hybrids was aimed to achieve higher efficacy through possible multi-target interactions, selective delivery of the drug to the site of action with the aim to improve bioavailability, alleviate toxicity and circumvent drug resistances.  In this review article, we summarize the progress made in recent years in the rapidly growing field of drug discovery, focusing on the rationality of the hybrid design with particular emphasis on the linker architecture, which plays a crucial role in the overall success of a hybrid drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4eHnPN7UP0rVg90H21EOLACvtfcHk0lgiMu_Brv8x3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFGqurjK&md5=b8ad2e17f676eec2b81835e43b6212c0</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127514%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%2BKumar%26aufirst%3DH.%2BM.%26aulast%3DHerrmann%26aufirst%3DL.%26aulast%3DTsogoeva%26aufirst%3DS.%2BB.%26atitle%3DStructural%2520hybridization%2520as%2520a%2520facile%2520approach%2520to%2520new%2520drug%2520candidates%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127514%26epage%3D127528%26doi%3D10.1016%2Fj.bmcl.2020.127514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hailan  Huang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel benzofuran derivatives as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113501. <a href="https://doi.org/10.1016/j.ejmech.2021.113501" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113501</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113501%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bbenzofuran%252Bderivatives%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26volume%3D220%26spage%3D113501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure and function of LSD1. (A) Cocrystal structure of LSD1/CoREST complex (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>). (B) FAD-dependent LSD1 mediated demethylation of methylated lysine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. LSD1 inhibitors in clinical trials. (A) Structure of TCP-based irreversible LSD1 inhibitors in clinical trials. (B) Structure of reversible LSD1 inhibitors in clinical trials.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Polyamine derivatives. (A) Structure and properties of amidinoguanidinium derivatives as LSD1 inhibitors. (B) Structure and properties of thiourea derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Polypeptide derivatives. (A) Structure and properties of linear peptides based LSD1 inhibitors. (B) Structure and properties of cyclic peptides based LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Six-membered heterocyclic compounds. (A) Structure and properties of pyridine derivatives as LSD1 inhibitors. (B) Structure and properties of pyrimidine derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Triazole derivatives. (A) Structure and properties of 3,5-diamino-1,2,4-triazole derivatives as LSD1 inhibitors. (B) Structure and properties of triazole-dithiocarbamate derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Other five-membered heterocyclic compounds. (A–E) Structure and properties of oxazole derivatives (A), aminothiazole derivatives (B), pyrazole derivatives (C), imidazole derivatives (D), and pyrrolidine derivatives (E) as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. 6,6-Fused heterocyclic compounds. (A–C) Structure and properties of flavones derivatives (A), quinazoline derivatives (B), and tetrahydroquinoline derivatives (C) as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Triazole-fused pyrimidines. (A) Structure and properties of [1,2,3]triazolo[4,5-<i>d</i>]pyrimidine derivatives as LSD1 inhibitors. (B) Structure and properties of [1,2,4]triazolo[1,5-<i>a</i>]pyrimidine derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Other 6,5-fused heterocyclic compounds. (A–C) Structure and properties of indole derivatives (A), xanthine derivatives (B), and 2,3-benzofuran derivatives (C) as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. 5,5-Fused heterocyclic compounds. (A) Structure and properties of thieno[3,2-<i>b</i>]pyrrole-5-carboxamides derivatives as LSD1 inhibitors. (B) Structure and properties of 5-imidazolyl-thieno[3,2-<i>b</i>]pyrroles derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Styrene derivatives. (A) Structure and properties of stilbene-amidoxime hybrids as LSD1 inhibitors. (B) Structure and properties of cinnamic acid derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. 3-Oxoamino-benzenesulfonamide derivatives as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Metal complexes. (A) Structure and properties of rhodium(III) complex as LSD1 inhibitor. (B) Structure and properties of vanadium complexes as LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/medium/jm0c02176_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Other reversible LSD1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c02176/20210305/images/large/jm0c02176_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02176&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 120 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lj0nJuGC2mv0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">Advances toward LSD1 inhibitors for cancer therapy</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1242</span>, <span class="refDoi"> DOI: 10.4155/fmc-2017-0068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.4155%2Ffmc-2017-0068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28722477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1227-1242&author=X.+L.+Fuauthor=P.+Zhangauthor=B.+Yu&title=Advances+toward+LSD1+inhibitors+for+cancer+therapy&doi=10.4155%2Ffmc-2017-0068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Advances toward LSD1 inhibitors for cancer therapy</span></div><div class="casAuthors">Fu, Xiaoli; Zhang, Peng; Yu, Bin</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1227-1242</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  LSD1 has become an important biol. validated epigenetic target for cancer therapy since its identification in 2004.  LSD1 mediates many cellular signaling pathways and is involved in the initiation and development of cancers.  Aberrant overexpression of LSD1 has been obsd. in different types of cancers, and inactivation by small mols. suppresses cancer cell differentiation, proliferation, invasion and migration.  To date, a large no. of LSD1 inhibitors have been reported, RG6016, GSK-2879552, INCB059872, IMG-7289 and CC-90011 are currently undergoing clin. assessment for the treatment of acute myeloid leukemia, small-cell lung cancer, etc.  In this review, we briefly highlight recent advances of LSD1 inhibitors mainly covering the literatures from 2015 to 2017 and tentatively propose our perspectives on the design of new LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlJAzt51S4crVg90H21EOLACvtfcHk0lj0nJuGC2mv0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qsLbK&md5=182f10eb7a2aea7b5c13f50f84f44c20</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DX.%2BL.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DAdvances%2520toward%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D1227%26epage%3D1242%26doi%3D10.4155%2Ffmc-2017-0068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stavropoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelz, A.</span></span> <span> </span><span class="NLM_article-title">Crystal structure and mechanism of human lysine-specific demethylase-1</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">626</span>– <span class="NLM_lpage">632</span>, <span class="refDoi"> DOI: 10.1038/nsmb1113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnsmb1113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=16799558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=626-632&author=P.+Stavropoulosauthor=G.+Blobelauthor=A.+Hoelz&title=Crystal+structure+and+mechanism+of+human+lysine-specific+demethylase-1&doi=10.1038%2Fnsmb1113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and mechanism of human lysine-specific demethylase-1</span></div><div class="casAuthors">Stavropoulos, Pete; Blobel, Guenter; Hoelz, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">626-632</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The reversible methylation of specific Lys residues in histone tails is crucial in epigenetic gene regulation.  LSD1, the 1st known lysine-specific demethylase, selectively removes mono-Me and di-Me, but not tri-Me modifications of Lys-4 or Lys-9 in histone-3.  Here, the authors present the crystal structure of LSD1 at 2.9 Å resoln.  LSD1 was found to form a highly asym., closely packed domain structure from which a long helical 'tower' domain protruded.  The active site cavity was spacious enough to accommodate several residues of the histone tail substrate, but did not appear capable of recognizing the different methylation states of the substrate, lysine.  This supported the hypothesis that trimethylated lysine was chem. rather than sterically discriminated.  The authors presented a biochem. anal. of LSD1 mutants that identified crucial residues in the active site cavity and showed the importance of the SWIRM and tower domains for catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG8Vp2AlsWQLVg90H21EOLACvtfcHk0li4AzGTKo-OMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D&md5=34cbc6f0936b642a8668d1d76abd1e0e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1113%26sid%3Dliteratum%253Aachs%26aulast%3DStavropoulos%26aufirst%3DP.%26aulast%3DBlobel%26aufirst%3DG.%26aulast%3DHoelz%26aufirst%3DA.%26atitle%3DCrystal%2520structure%2520and%2520mechanism%2520of%2520human%2520lysine-specific%2520demethylase-1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D626%26epage%3D632%26doi%3D10.1038%2Fnsmb1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yennawar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the LSD1/CoREST histone demethylase bound to its nucleosome substrate</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">914</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.molcel.2020.04.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32396821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptlegsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=903-914&author=S.+A.+Kimauthor=J.+Zhuauthor=N.+Yennawarauthor=P.+Eekauthor=S.+Tan&title=Crystal+structure+of+the+LSD1%2FCoREST+histone+demethylase+bound+to+its+nucleosome+substrate&doi=10.1016%2Fj.molcel.2020.04.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the LSD1/CoREST Histone Demethylase Bound to Its Nucleosome Substrate</span></div><div class="casAuthors">Kim, Sang-Ah; Zhu, Jiang; Yennawar, Neela; Eek, Priit; Tan, Song</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">903-914.e4</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">LSD1 (lysine specific demethylase; also known as KDM1A), the first histone demethylase discovered, regulates cell-fate detn. and is overexpressed in multiple cancers.  LSD1 demethylates histone H3 Lys4, an epigenetic mark for active genes, but requires the CoREST repressor to act on nucleosome substrates.  To understand how an accessory subunit (CoREST) enables a chromatin enzyme (LSD1) to function on a nucleosome and not just histones, we have detd. the crystal structure of the LSD1/CoREST complex bound to a 191-bp nucleosome.  We find that the LSD1 catalytic domain binds extranucleosomal DNA and is unexpectedly positioned 100 Å away from the nucleosome core.  CoREST makes crit. contacts with both histone and DNA components of the nucleosome, explaining its essential function in demethylating nucleosome substrates.  Our studies also show that the LSD1(K661A) frequently used as a catalytically inactive mutant in vivo (based on in vitro peptide studies) actually retains substantial H3K4 demethylase activity on nucleosome substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH-RwffJwsB7Vg90H21EOLACvtfcHk0li4AzGTKo-OMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptlegsr0%253D&md5=6604db71a7ad1eb0ca25ecddd96ca8f5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.019%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BA.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DYennawar%26aufirst%3DN.%26aulast%3DEek%26aufirst%3DP.%26aulast%3DTan%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520LSD1%252FCoREST%2520histone%2520demethylase%2520bound%2520to%2520its%2520nucleosome%2520substrate%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D903%26epage%3D914%26doi%3D10.1016%2Fj.molcel.2020.04.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikuchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higuchi, T.</span></span> <span> </span><span class="NLM_article-title">Development and crystallographic evaluation of histone H3 peptide with N-terminal serine substitution as a potent inhibitor of lysine-specific demethylase 1</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2617</span>– <span class="NLM_lpage">2624</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2017.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28336409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVSltbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2617-2624&author=Y.+Amanoauthor=M.+Kikuchiauthor=S.+Satoauthor=S.+Yokoyamaauthor=T.+Umeharaauthor=N.+Umezawaauthor=T.+Higuchi&title=Development+and+crystallographic+evaluation+of+histone+H3+peptide+with+N-terminal+serine+substitution+as+a+potent+inhibitor+of+lysine-specific+demethylase+1&doi=10.1016%2Fj.bmc.2017.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Development and crystallographic evaluation of histone H3 peptide with N-terminal serine substitution as a potent inhibitor of lysine-specific demethylase 1</span></div><div class="casAuthors">Amano, Yuichi; Kikuchi, Masaki; Sato, Shin; Yokoyama, Shigeyuki; Umehara, Takashi; Umezawa, Naoki; Higuchi, Tsunehiko</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2617-2624</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1A) is a flavoenzyme demethylase, which removes mono- and di-Me groups from histone H3 Lys4 (H3K4) or Lys9 (H3K9) in complexes with several nuclear proteins.  Since LSD1 is implicated in the tumorigenesis and progression of various cancers, LSD1-specific inhibitors are considered as potential anti-cancer agents.  A modified H3 peptide with substitution of Lys4 to Met [H3K4M] is already known to be a potent competitive inhibitor of LSD1.  In this study, we synthesized a series of H3K4M peptide derivs. and evaluated their LSD1-inhibitory activities in vitro.  We found that substitutions of the N-terminal amino acid with amino acids having a larger side chain were generally not tolerated, but substitution of Ala1 to Ser unexpectedly resulted in more potent inhibitory activity toward LSD1.  X-ray crystallog. anal. of H3K4M derivs. bound to the LSD1·CoREST complex revealed the presence of addnl. hydrogen bonding between the N-terminal Ser residue of the H3 peptide deriv. and LSD1.  The present structural and biochem. findings will be helpful for obtaining more potent peptidic inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSLYWQH97b27Vg90H21EOLACvtfcHk0li4AzGTKo-OMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVSltbc%253D&md5=c988cd6c93b3c0dc4bd371374cd2d2df</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DAmano%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DHiguchi%26aufirst%3DT.%26atitle%3DDevelopment%2520and%2520crystallographic%2520evaluation%2520of%2520histone%2520H3%2520peptide%2520with%2520N-terminal%2520serine%2520substitution%2520as%2520a%2520potent%2520inhibitor%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2617%26epage%3D2624%26doi%3D10.1016%2Fj.bmc.2017.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">953</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+J.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liZizadq1HWcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BJ.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, A. H. F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüle, R.</span></span> <span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>437</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0liZizadq1HWcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray-Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The re-expression of the epigenetically silenced e-cadherin gene by a polyamine analogue lysine-specific demethylase-1 (LSD1) inhibitor in human acute myeloid leukemia cell lines</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">594</span>, <span class="refDoi"> DOI: 10.1007/s00726-013-1485-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00726-013-1485-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23508577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjt1Srt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=585-594&author=T.+Murray-Stewartauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=The+re-expression+of+the+epigenetically+silenced+e-cadherin+gene+by+a+polyamine+analogue+lysine-specific+demethylase-1+%28LSD1%29+inhibitor+in+human+acute+myeloid+leukemia+cell+lines&doi=10.1007%2Fs00726-013-1485-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The re-expression of the epigenetically silenced e-cadherin gene by a polyamine analogue lysine-specific demethylase-1 (LSD1) inhibitor in human acute myeloid leukemia cell lines</span></div><div class="casAuthors">Murray-Stewart, Tracy; Woster, Patrick M.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">585-594</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag GmbH</span>)
        </div><div class="casAbstract">Aberrant epigenetic silencing of tumor suppressor genes is a common feature obsd. during the transformation process of many cancers, including those of hematol. origin.  Histone modifications, including acetylation, phosphorylation, and methylation, collaborate with DNA CpG island methylation to regulate gene expression.  The dynamic process of histone methylation is the latest of these epigenetic modifications to be described, and the identification and characterization of LSD1 as a demethylase of lysine 4 of histone H3 (H3K4) has confirmed that both the enzyme and the modified histone play important roles as regulators of gene expression.  LSD1 activity contributes to the suppression of gene expression by demethylating promoter-region mono- and dimethyl-H3K4 histone marks that are assocd. with active gene expression.  As most post-translational modifications are reversible, the enzymes involved in the modification of histones have become targets for chemotherapeutic intervention.  In this study, we examd. the effects of the polyamine analog LSD1 inhibitor 2d (1,15-bis{N5-[3,3-(diphenyl)propyl]-N1-biguanido}-4,12-diazapentadecane) in human acute myeloid leukemia (AML) cell lines.  In each line studied, 2d evoked cytotoxicity and inhibited LSD1 activity, as evidenced by increases in the global levels of mono- and di-methylated H3K4 proteins.  Global increases in other chromatin modifications were also obsd. following exposure to 2d, suggesting a broad response to this compd. with respect to chromatin regulation.  On a gene-specific level, treatment with 2d resulted in the re-expression of e-cadherin, a tumor suppressor gene frequently silenced by epigenetic modification in AML.  Quant. chromatin immunopptn. anal. of the e-cadherin promoter further confirmed that this re-expression was concurrent with changes in both active and repressive histone marks that were consistent with LSD1 inhibition.  As hematol. malignancies have demonstrated promising clin. responses to agents targeting epigenetic silencing, this polyamine analog LSD1 inhibitor presents an exciting new avenue for the development of novel therapeutic agents for the treatment of AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpENhX8f3UG3bVg90H21EOLACvtfcHk0liZizadq1HWcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjt1Srt7s%253D&md5=01861b27c9f87be5e1915cb3dc3a9978</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs00726-013-1485-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-013-1485-1%26sid%3Dliteratum%253Aachs%26aulast%3DMurray-Stewart%26aufirst%3DT.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DThe%2520re-expression%2520of%2520the%2520epigenetically%2520silenced%2520e-cadherin%2520gene%2520by%2520a%2520polyamine%2520analogue%2520lysine-specific%2520demethylase-1%2520%2528LSD1%2529%2520inhibitor%2520in%2520human%2520acute%2520myeloid%2520leukemia%2520cell%2520lines%26jtitle%3DAmino%2520Acids%26date%3D2014%26volume%3D46%26spage%3D585%26epage%3D594%26doi%3D10.1007%2Fs00726-013-1485-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">8023</span>– <span class="NLM_lpage">8028</span>, <span class="refDoi"> DOI: 10.1073/pnas.0700720104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1073%2Fpnas.0700720104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17463086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+Murray+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes&doi=10.1073%2Fpnas.0700720104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span></div><div class="casAuthors">Huang, Yi; Greene, Eriko; Stewart, Tracy Murray; Goodwin, Andrew C.; Baylin, Stephen B.; Woster, Patrick M.; Casero, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8023-8028</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Epigenetic chromatin modification is a major regulator of eukaryotic gene expression, and aberrant epigenetic silencing of gene expression contributes to tumorigenesis.  Histone modifications include acetylation, phosphorylation, and methylation, resulting in a combination of histone marks known collectively as the histone code.  The chromatin marks at a given promoter det., in part, whether specific promoters are in an open/active conformation or closed/repressed conformation.  Dimethyl-lysine 4 histone H3 (H3K4me2) is a transcription-activating chromatin mark at gene promoters, and demethylation of this mark by the lysine-specific demethylase 1 (LSD1), a homolog of polyamine oxidases, may broadly repress gene expression.  We now report that novel biguanide and bisguanidine polyamine analogs are potent inhibitors of LSD1.  These analogs inhibit LSD1 in human colon carcinoma cells and affect a reexpression of multiple, aberrantly silenced genes important in the development of colon cancer, including members of the secreted frizzle-related proteins (SFRPs) and the GATA family of transcription factors.  Furthermore, we demonstrate by chromatin immunopptn. anal. that the reexpression is concurrent with increased H3K4me2 and acetyl-H3K9 marks, decreased H3K9me1 and H3K9me2 repressive marks.  We thus define important new agents for reversing aberrant repression of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC96zJoYEK67Vg90H21EOLACvtfcHk0liXjngb0HOZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D&md5=4c7863ecb038e3e10c5774cba2c28a45</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700720104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700720104%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMurray%2BStewart%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028%26doi%3D10.1073%2Fpnas.0700720104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. P.</span></span> <span> </span><span class="NLM_article-title">LSD1 demethylates histone and non-histone proteins</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.4161/epi.4.3.8443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.4161%2Fepi.4.3.8443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=19395867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlWntbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=129-132&author=T.+B.+Nicholsonauthor=T.+P.+Chen&title=LSD1+demethylates+histone+and+non-histone+proteins&doi=10.4161%2Fepi.4.3.8443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates histone and non-histone proteins</span></div><div class="casAuthors">Nicholson, Thomas B.; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">EPIGB6</span>;
        ISSN:<span class="NLM_cas:issn">1559-2294</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  One of the key breakthroughs in the epigenetics/chromatin field in the last several years was the identification of enzymes capable of removing the Me group from methylated lysines in histone proteins.  Lysine-specific demethylase 1 (LSD1) was the first such enzyme identified, which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  LSD1 is essential for mammalian development and likely involved in many biol. processes.  Recent studies show that LSD1 demethylates p53 and DNA methyltransferase 1 (Dnmt1) and regulates their cellular functions, indicating that LSD1 fulfills its biol. functions by directly acting on both histone and non-histone proteins.  LSD1 contains several defined domains and assocs. with a no. of protein complexes.  Interacting partners of LSD1 may play key roles in detg./modulating the activity and specificity of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3kUnCrwJ2H7Vg90H21EOLACvtfcHk0liXjngb0HOZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlWntbs%253D&md5=3bb2c45af1a5b8f1e0149d518409faba</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4161%2Fepi.4.3.8443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.4.3.8443%26sid%3Dliteratum%253Aachs%26aulast%3DNicholson%26aufirst%3DT.%2BB.%26aulast%3DChen%26aufirst%3DT.%2BP.%26atitle%3DLSD1%2520demethylates%2520histone%2520and%2520non-histone%2520proteins%26jtitle%3DEpigenetics%26date%3D2009%26volume%3D4%26spage%3D129%26epage%3D132%26doi%3D10.4161%2Fepi.4.3.8443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hevi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurash, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gay, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T. P.</span></span> <span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+T.+Sunauthor=H.+Changauthor=G.+L.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+P.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0liXjngb0HOZVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%2BT.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%2BL.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%2BP.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubalek, F.</span></span> <span> </span><span class="NLM_article-title">Structure and mechanism of monoamine oxidase</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1983</span>– <span class="NLM_lpage">1993</span>, <span class="refDoi"> DOI: 10.2174/0929867043364784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2174%2F0929867043364784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=15279562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslWmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1983-1993&author=D.+E.+Edmondsonauthor=A.+Matteviauthor=C.+Bindaauthor=M.+Liauthor=F.+Hubalek&title=Structure+and+mechanism+of+monoamine+oxidase&doi=10.2174%2F0929867043364784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of monoamine oxidase</span></div><div class="casAuthors">Edmondson, D. E.; Mattevi, A.; Binda, C.; Li, M.; Hubalek, F.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1983-1993</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Monoamine oxidases A and B (MAO A and MAO B) are mitochondrial outer membrane-bound flavoproteins that catalyze the oxidative deamination of neurotransmitters and biogenic amines.  A no. of mechanism-based inhibitors have been developed for clin. use as antidepressants and as neuroprotective drugs.  To facilitate the development of more effective and specific inhibitors, a detailed understanding of the structures and catalytic mechanisms of these enzymes is required.  The recent development of high level expression systems for producing recombinant human liver MAO A and MAO B in Pichia pastoris has facilitated the detn. of the 3-dimensional crystal structures of MAO B (up to 1.7 Å resoln.) in complex with different reversible (isatin, 1,4-diphenyl-2-butene) and irreversible inhibitors (pargyline, N-(2-aminoethyl)-p-chlorobenzamide, and trans-2-phenylcyclopropylamine).  The binding of substrates or inhibitors to MAO B involves an initial negotiation of a protein loop occurring near the surface of the membrane and 2 hydrophobic cavities, an "entrance" cavity, and an "active site" cavity.  These 2 cavities can either be sep. or in a fused state depending on the conformation of the Ile-199 side-chain, which appears to function as a gate.  The amine function of the bound substrate approaches the re face of the bent and "puckered" covalent FAD through an "arom. cage" formed by 2 Tyr residues that are perpendicular to the plane of the flavin ring.  No amino acid residues that could function as acids or bases are found near the catalytic site.  The existing structural data on MAO B support previous QSAR results and are also supportive of a proposed polar nucleophilic mechanism for MAO A and B catalysis rather than the alternatively proposed single electron transfer mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaQv9eLs73u7Vg90H21EOLACvtfcHk0lhsx8yr-fuFZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslWmu7s%253D&md5=270a408e70f5a136f49489ff7b5fd482</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.2174%2F0929867043364784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867043364784%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DHubalek%26aufirst%3DF.%26atitle%3DStructure%2520and%2520mechanism%2520of%2520monoamine%2520oxidase%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2004%26volume%3D11%26spage%3D1983%26epage%3D1993%26doi%3D10.2174%2F0929867043364784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karytinos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Profumo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaglioli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">A novel mammalian flavin-dependent histone demethylase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">17775</span>– <span class="NLM_lpage">17782</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.003087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1074%2Fjbc.M109.003087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=19407342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=17775-17782&author=A.+Karytinosauthor=F.+Fornerisauthor=A.+Profumoauthor=G.+Ciossaniauthor=E.+Battaglioliauthor=C.+Bindaauthor=A.+Mattevi&title=A+novel+mammalian+flavin-dependent+histone+demethylase&doi=10.1074%2Fjbc.M109.003087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Mammalian Flavin-dependent Histone Demethylase</span></div><div class="casAuthors">Karytinos, Aristotele; Forneris, Federico; Profumo, Antonella; Ciossani, Giuseppe; Battaglioli, Elena; Binda, Claudia; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17775-17782</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Methylation of Lys residues on histone proteins is a well known and extensively characterized epigenetic mark.  The recent discovery of lysine-specific demethylase 1 (LSD1) demonstrated that lysine methylation can be dynamically controlled.  Among the histone demethylases so far identified, LSD1 has the unique feature of functioning through a flavin-dependent amine oxidn. reaction.  Data base anal. reveals that mammalian genomes contain a gene (AOF1, for amine-oxidase flavin-contg. domain 1) that is homologous to the LSD1-coding gene.  Here, we demonstrate that the protein encoded by AOF1 represents a second mammalian flavin-dependent histone demethylase, named LSD2.  The new demethylase is strictly specific for mono- and dimethylated Lys4 of histone H3, recognizes a long stretch of the H3 N-terminal tail, senses the presence of addnl. epigenetic marks on the histone substrate, and is covalently inhibited by tranylcypromine.  As opposed to LSD1, LSD2 does not form a biochem. stable complex with the C-terminal domain of the corepressor protein CoREST.  Furthermore, LSD2 contains a CW-type zinc finger motif with potential zinc-binding sites that are not present in LSD1.  We conclude that mammalian LSD2 represents a new flavin-dependent H3-Lys4 demethylase that features substrate specificity properties highly similar to those of LSD1 but is very likely to be part of chromatin-remodeling complexes that are distinct from those involving LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjSPhtflZQU7Vg90H21EOLACvtfcHk0lhsx8yr-fuFZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnsVWksb8%253D&md5=4fbe32f2274de75915e2308781a214d1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.003087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.003087%26sid%3Dliteratum%253Aachs%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DProfumo%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520novel%2520mammalian%2520flavin-dependent%2520histone%2520demethylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D17775%26epage%3D17782%26doi%3D10.1074%2Fjbc.M109.003087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span> <span> </span><span class="NLM_article-title">Pharmacological inhibition of LSD1 for cancer treatment</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">3194</span>– <span class="NLM_lpage">3213</span>, <span class="refDoi"> DOI: 10.3390/molecules23123194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.3390%2Fmolecules23123194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXms1yrtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=3194-3213&author=G.+J.+Yangauthor=P.+M.+Leiauthor=S.+Y.+Wongauthor=D.+L.+Maauthor=C.+H.+Leung&title=Pharmacological+inhibition+of+LSD1+for+cancer+treatment&doi=10.3390%2Fmolecules23123194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological inhibition of LSD1 for cancer treatment</span></div><div class="casAuthors">Yang, Guan-Jun; Lei, Pui-Man; Wong, Suk-Yu; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3194/1-3194/20</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Lysine-specific demethylase 1A (LSD1, also named KDM1A) is a demethylase that can remove Me groups from histones H3K4me1/2 and H3K9me1/2.  It is aberrantly expressed in many cancers, where it impedes differentiation and contributes to cancer cell proliferation, cell metastasis and invasiveness, and is assocd. with inferior prognosis.  Pharmacol. inhibition of LSD1 has been reported to significantly attenuate tumor progression in vitro and in vivo in a range of solid tumors and acute myeloid leukemia.  This review will present the structural aspects of LSD1, its role in carcinogenesis, a comparison of currently available approaches for screening LSD1 inhibitors, a classification of LSD1 inhibitors, and its potential as a drug target in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSEBy-HsQHlrVg90H21EOLACvtfcHk0lhsx8yr-fuFZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXms1yrtb8%253D&md5=87fe2ac11e363e75ac0ef22ff6412d24</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23123194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23123194%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DG.%2BJ.%26aulast%3DLei%26aufirst%3DP.%2BM.%26aulast%3DWong%26aufirst%3DS.%2BY.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DPharmacological%2520inhibition%2520of%2520LSD1%2520for%2520cancer%2520treatment%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D3194%26epage%3D3213%26doi%3D10.3390%2Fmolecules23123194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Pharmacoepigenetics of LSD1 inhibitors in cancer</span>. <i>Pharmacoepigenetics</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">523</span>– <span class="NLM_lpage">530</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-813939-4.00014-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2FB978-0-12-813939-4.00014-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=523-530&author=B.+Yuauthor=H.+M.+Liu&title=Pharmacoepigenetics+of+LSD1+inhibitors+in+cancer&doi=10.1016%2FB978-0-12-813939-4.00014-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-813939-4.00014-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-813939-4.00014-0%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DPharmacoepigenetics%2520of%2520LSD1%2520inhibitors%2520in%2520cancer%26jtitle%3DPharmacoepigenetics%26date%3D2019%26volume%3D10%26spage%3D523%26epage%3D530%26doi%3D10.1016%2FB978-0-12-813939-4.00014-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Majello, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacca, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amente, S.</span></span> <span> </span><span class="NLM_article-title">Expanding the role of the histone lysine-specific demethylase LSD1 in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">324</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.3390/cancers11030324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.3390%2Fcancers11030324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlKntLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=324-338&author=B.+Majelloauthor=F.+Goriniauthor=C.+D.+Saccaauthor=S.+Amente&title=Expanding+the+role+of+the+histone+lysine-specific+demethylase+LSD1+in+cancer&doi=10.3390%2Fcancers11030324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the role of the histone Lysine-specific demethylase LSD1 in cancer</span></div><div class="casAuthors">Majello, Barbara; Gorini, Francesca; Sacca, Carmen Daniela; Amente, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">324</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Studies of alterations in histone methylation in cancer have led to the identification of histone methyltransferases and demethylases as novel targets for therapy.  Lysine-specific demethylase 1 (LSD1, also known as KDM1A), demethylates H3K4me1/2, or H3K9me1/2 in a context-dependent manner.  In addn. to the well-studied role of LSD1 in the epigenetic regulation of histone methylation changes, LSD1 regulates the methylation dynamic of several non-histone proteins and participates in the assembly of different long noncoding RNA (lncRNA_ complexes.  LSD1 is highly expressed in various cancers, playing a pivotal role in different cancer-related processes.  Here, we summarized recent findings on the role of LSD1 in the regulation of different biol. processes in cancer cells through dynamic methylation of non-histone proteins and phys. assocn. with dedicated lncRNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHjEJqn28EirVg90H21EOLACvtfcHk0lhsx8yr-fuFZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlKntLjF&md5=d3aede81a2c84173b2efeb5a997a8e6d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.3390%2Fcancers11030324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11030324%26sid%3Dliteratum%253Aachs%26aulast%3DMajello%26aufirst%3DB.%26aulast%3DGorini%26aufirst%3DF.%26aulast%3DSacca%26aufirst%3DC.%2BD.%26aulast%3DAmente%26aufirst%3DS.%26atitle%3DExpanding%2520the%2520role%2520of%2520the%2520histone%2520lysine-specific%2520demethylase%2520LSD1%2520in%2520cancer%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D324%26epage%3D338%26doi%3D10.3390%2Fcancers11030324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Targeting LSD1 for acute myeloid leukemia (AML) treatment</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">105335</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2020.105335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.phrs.2020.105335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=33285227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3MXit12qsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2021&pages=105335&author=S.+J.+Zhangauthor=M.+H.+Liuauthor=Y.+F.+Yaoauthor=B.+Yuauthor=H.+M.+Liu&title=Targeting+LSD1+for+acute+myeloid+leukemia+%28AML%29+treatment&doi=10.1016%2Fj.phrs.2020.105335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting LSD1 for acute myeloid leukemia (AML) treatment</span></div><div class="casAuthors">Zhang, Shujing; Liu, Menghan; Yao, Yongfang; Yu, Bin; Liu, Hongmin</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105335</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Targeted therapy for acute myeloid leukemia (AML) is an effective strategy, but currently there are very limited therapeutic targets for AML treatment.  Histone lysine specific demethylase 1 (LSD1) is highly expressed in many cancers, impedes the differentiation of cancer cells, promotes the proliferation, metastasis and invasion of cancer cells, and is assocd. with poor prognosis.  Targeting LSD1 has been recognized as a promising strategy for AML treatment in recent years.  Based on these features, in the , we discussed the main epigenetic drugs targeting LSD1 for AML therapy.  Thus, this focuses on the progress of LSD1 inhibitors in AML treatment, particularly those such as tranylcypromine (TCP), ORY-1001, GSK2879552, and IMG-7289 in clin. trials.  These inhibitors provide novel scaffolds for designing new LSD1 inhibitors.  Besides, combined therapies of LSD1 inhibitors with other drugs for AML treatment are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbIlySiOvBHLVg90H21EOLACvtfcHk0lgvkEFQKeS0sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXit12qsr8%253D&md5=ce58f5ef8cd0e2cc605d0f18a1ab30d0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2020.105335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2020.105335%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%2BJ.%26aulast%3DLiu%26aufirst%3DM.%2BH.%26aulast%3DYao%26aufirst%3DY.%2BF.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTargeting%2520LSD1%2520for%2520acute%2520myeloid%2520leukemia%2520%2528AML%2529%2520treatment%26jtitle%3DPharmacol.%2520Res.%26date%3D2021%26volume%3D164%26spage%3D105335%26doi%3D10.1016%2Fj.phrs.2020.105335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stazi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwergel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span> <span> </span><span class="NLM_article-title">LSD1 inhibitors: a patent review (2010–2015)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1517/13543776.2016.1165209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1517%2F13543776.2016.1165209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27019002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvFGqs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=565-580&author=G.+Staziauthor=C.+Zwergelauthor=S.+Valenteauthor=A.+Mai&title=LSD1+inhibitors%3A+a+patent+review+%282010%E2%80%932015%29&doi=10.1517%2F13543776.2016.1165209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibitors: a patent review (2010-2015)</span></div><div class="casAuthors">Stazi, Giulia; Zwergel, Clemens; Valente, Sergio; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">565-580</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Lysine demethylase 1 (LSD1) plays an important role in mediating the expression of genes involved in cancer and non-cancer diseases such as viral infections, cardiovascular and neurodegenerative disorders.  It is involved in a no. of processes from adipogenesis to cell adhesion to viral latency, regulating several cell pathways related with proliferation, development, and cell cycle control.  Numerous chem. entities have been studied in recent years and some of them entered the clin. arena.  This review summarizes recent efforts in the drug development of LSD1 inhibitors reported in the patent literature covering the 2010-2015 period, including their potential use as therapeutics in cancerous, neurol., inflammatory, cardiovascular, and viral diseases.  The development of novel potent and selective LSD1 inhibitors is ongoing, in order to improve their potency and selectivity against specific types of cancer or non-cancer diseases.  More in-depth studies are required to assess the role of LSD1 inhibitors in the expression of LSD1 target genes, for a better assessment of the biochem. underlying their efficacy, and to provide evidence for any possible side effects.  Furthermore, an interesting therapeutic approach is the use of LSD1 inhibitors in conjunction with other epidrugs to combine their therapeutic potential, leading to innovative, personalized treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCf5uleaXD27Vg90H21EOLACvtfcHk0lgvkEFQKeS0sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvFGqs7w%253D&md5=557242114797bc59b35e9f66cec893bf</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1517%2F13543776.2016.1165209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2016.1165209%26sid%3Dliteratum%253Aachs%26aulast%3DStazi%26aufirst%3DG.%26aulast%3DZwergel%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DLSD1%2520inhibitors%253A%2520a%2520patent%2520review%2520%25282010%25E2%2580%25932015%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2016%26volume%3D26%26spage%3D565%26epage%3D580%26doi%3D10.1517%2F13543776.2016.1165209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, O.</span></span> <span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1038/nm.3101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnm.3101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23416702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=291-294&author=L.+Shiauthor=S.+Cuiauthor=J.+D.+Engelauthor=O.+Tanabe&title=Lysine-specific+demethylase+1+is+a+therapeutic+target+for+fetal+hemoglobin+induction&doi=10.1038%2Fnm.3101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is a therapeutic target for fetal hemoglobin induction</span></div><div class="casAuthors">Shi, Lihong; Cui, Shuaiying; Engel, James D.; Tanabe, Osamu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">291-294</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Enhanced fetal γ-globin synthesis alleviates symptoms of β-globinopathies such as sickle cell disease and β-thalassemia, but current γ-globin-inducing drugs offer limited beneficial effects.  We show here that lysine-specific demethylase 1 (LSD1) inhibition by RNAi in human erythroid cells or by the monoamine oxidase inhibitor tranylcypromine in human erythroid cells or β-type globin-transgenic mice enhances γ-globin expression.  LSD1 is thus a promising therapeutic target for γ-globin induction, and tranylcypromine may serve as a lead compd. for the development of a new γ-globin inducer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPuLk6KrTQ-rVg90H21EOLACvtfcHk0lgvkEFQKeS0sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Wrurs%253D&md5=3747256331d40eb16b2ce54a85df1761</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnm.3101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3101%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DS.%26aulast%3DEngel%26aufirst%3DJ.%2BD.%26aulast%3DTanabe%26aufirst%3DO.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520a%2520therapeutic%2520target%2520for%2520fetal%2520hemoglobin%2520induction%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D291%26epage%3D294%26doi%3D10.1038%2Fnm.3101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, P.</span></span> <span> </span><span class="NLM_article-title">Medicinal chemistry insights in the discovery of novel LSD1 inhibitors</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1396</span>, <span class="refDoi"> DOI: 10.2217/epi.15.86</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2217%2Fepi.15.86" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=26646727" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVejtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1379-1396&author=X.+Wangauthor=B.+Huangauthor=T.+Suzukiauthor=X.+Liuauthor=P.+Zhan&title=Medicinal+chemistry+insights+in+the+discovery+of+novel+LSD1+inhibitors&doi=10.2217%2Fepi.15.86"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Medicinal chemistry insights in the discovery of novel LSD1 inhibitors</span></div><div class="casAuthors">Wang, Xueshun; Huang, Boshi; Suzuki, Takayoshi; Liu, Xinyong; Zhan, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1379-1396</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">LSD1 is an epigenetic modulator assocd. with transcriptional regulation of genes involved in a broad spectrum of key cellular processes, and its activity is often altered under pathol. conditions.  LSD1 inhibitors are considered to be candidates for therapy of cancer, viral diseases and neurodegeneration.  Many LSD1 inhibitors with various scaffolds have been disclosed, and a few potent mols. are in different stages of clin. development.  In this review, we summarize recent biol. findings on the roles of LSD1 and the current understanding of the clin. significance of LSD1, and focus on the medicinal chem. strategies used in the design and development of LSD1 inhibitors as drug-like epigenetic modulators since 2012, including a brief consideration of structure-activity relationships.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDkOV2H-u3CrVg90H21EOLACvtfcHk0lj48A3lN79P5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVejtbrL&md5=da13cfc3e8ec09e86a00d5e87e700f6f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.2217%2Fepi.15.86&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.15.86%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DZhan%26aufirst%3DP.%26atitle%3DMedicinal%2520chemistry%2520insights%2520in%2520the%2520discovery%2520of%2520novel%2520LSD1%2520inhibitors%26jtitle%3DEpigenomics%26date%3D2015%26volume%3D7%26spage%3D1379%26epage%3D1396%26doi%3D10.2217%2Fepi.15.86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span> <span> </span><span class="NLM_article-title">Drug design concepts for LSD1-selective inhibitors</span>. <i>Chem. Rec.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1782</span>– <span class="NLM_lpage">1791</span>, <span class="refDoi"> DOI: 10.1002/tcr.201810031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Ftcr.201810031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30277644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslSntrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=1782-1791&author=Y.+Otaauthor=T.+Suzuki&title=Drug+design+concepts+for+LSD1-selective+inhibitors&doi=10.1002%2Ftcr.201810031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Design Concepts for LSD1-Selective Inhibitors</span></div><div class="casAuthors">Ota, Yosuke; Suzuki, Takayoshi</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Record</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1782-1791</span>CODEN:
                <span class="NLM_cas:coden">CRHEAK</span>;
        ISSN:<span class="NLM_cas:issn">1528-0691</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is one of the flavin-dependent oxidases and is involved in many cellular processes by controlling the methylation of histone H3.  Recently, it has been reported that LSD1 is assocd. with several diseases such as cancer, metabolic disorders, and psychiatric diseases.  Thus, LSD1 is an attractive mol. target for the treatment of these diseases, and its inhibitors are predicted as therapeutic agents.  Although a variety of LSD1 inhibitors have been reported to date, many of them show insufficient activities and selectivity toward LSD1.  Meanwhile, we identified several LSD1-selective inhibitors using target-guided synthesis strategies based on our original ideas.  Our LSD1 inhibitors show not only potent LSD1-selective inhibitory activities, but also unique bioactivities both in vitro and in vivo.  This account highlights our drug design concepts for and identification of LSD1-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVzDiBBp_k5bVg90H21EOLACvtfcHk0lj48A3lN79P5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslSntrbO&md5=c87dd1d25218fb7f06e302087c61834a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Ftcr.201810031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Ftcr.201810031%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DY.%26aulast%3DSuzuki%26aufirst%3DT.%26atitle%3DDrug%2520design%2520concepts%2520for%2520LSD1-selective%2520inhibitors%26jtitle%3DChem.%2520Rec.%26date%3D2018%26volume%3D18%26spage%3D1782%26epage%3D1791%26doi%3D10.1002%2Ftcr.201810031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+U.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lj48A3lN79P5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, Q. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">Natural products as LSD1 inhibitors for cancer therapy</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.apsb.2020.06.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=Y.+Fangauthor=C.+Yangauthor=Z.+Q.+Yuauthor=X.+C.+Liauthor=Q.+C.+Muauthor=G.+C.+Liaoauthor=B.+Yu&title=Natural+products+as+LSD1+inhibitors+for+cancer+therapy&doi=10.1016%2Fj.apsb.2020.06.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DZ.%2BQ.%26aulast%3DLi%26aufirst%3DX.%2BC.%26aulast%3DMu%26aufirst%3DQ.%2BC.%26aulast%3DLiao%26aufirst%3DG.%2BC.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DNatural%2520products%2520as%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26doi%3D10.1016%2Fj.apsb.2020.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span> <span> </span><span class="NLM_article-title">LSD1/KDM1A inhibitors in clinical trials: advances and prospects</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1186/s13045-019-0811-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1186%2Fs13045-019-0811-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31801559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2019&pages=129-142&author=Y.+Fangauthor=G.+C.+Liaoauthor=B.+Yu&title=LSD1%2FKDM1A+inhibitors+in+clinical+trials%3A+advances+and+prospects&doi=10.1186%2Fs13045-019-0811-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1/KDM1A inhibitors in clinical trials: advances and prospects</span></div><div class="casAuthors">Fang Yuan; Liao Guochao; Yu Bin; Yu Bin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hematology & oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone demethylase LSD1 plays key roles during carcinogenesis, targeting LSD1 is becoming an emerging option for the treatment of cancers.  Numerous LSD1 inhibitors have been reported to date, some of them such as TCP, ORY-1001, GSK-2879552, IMG-7289, INCB059872, CC-90011, and ORY-2001 currently undergo clinical assessment for cancer therapy, particularly for small lung cancer cells (SCLC) and acute myeloid leukemia (AML).  This review is to provide a comprehensive overview of LSD1 inhibitors in clinical trials including molecular mechanistic studies, clinical efficacy, adverse drug reactions, and PD/PK studies and offer prospects in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQbk3-5OqEjmWlSCSgBhwJFfW6udTcc2ebZQD40YCEszbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfmsFylsg%253D%253D&md5=112299f15281667341a55dea57f6c83b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DG.%2BC.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DLSD1%252FKDM1A%2520inhibitors%2520in%2520clinical%2520trials%253A%2520advances%2520and%2520prospects%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2019%26volume%3D12%26spage%3D129%26epage%3D142%26doi%3D10.1186%2Fs13045-019-0811-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Tranylcypromine based LSD1 inhibitor: summary and prospective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14197</span>– <span class="NLM_lpage">14215</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14197-14215&author=X.+J.+Daiauthor=Y.+Liuauthor=X.+P.+Xiongauthor=L.+P.+Xueauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Tranylcypromine+based+LSD1+inhibitor%3A+summary+and+prospective&doi=10.1021%2Facs.jmedchem.0c00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective</span></div><div class="casAuthors">Dai, Xing-Jie; Liu, Ying; Xiong, Xiao-Peng; Xue, Lei-Peng; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14197-14215</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1/KDM1A) has become an important and promising anticancer target since it was first identified in 2004 and specially demethylates lysine residues of histone H3K4me1/2 and H3K9me1/2.  LSD1 is ubiquitously overexpressed in diverse cancers, and abrogation of LSD1 results in inhibition of proliferation, invasion, and migration in cancer cells.  Over the past decade, a no. of biol. active small-mol. LSD1 inhibitors have been developed.  To date, six trans-2-phenylcyclopropylamine (TCP)-based LSD1 inhibitors (including TCP, ORY-1001, GSK-2879552, INCB059872, IMG-7289, and ORY-2001) that covalently bind to the FAD (FAD) within the LSD1 catalytic cavity have already entered into clin. trials.  Here, we provide an overview about the structures, activities, and structure-activity relationship (SAR) of TCP-based LSD1 inhibitors that mainly covers the literature from 2008 to date.  The opportunities, challenges, and future research directions in this emerging and promising field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqasvgaOVZu7Vg90H21EOLACvtfcHk0lji_hb3XJIAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVeru7vO&md5=88638bc10c8c090b3df3a0d2f2c020a8</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DX.%2BP.%26aulast%3DXue%26aufirst%3DL.%2BP.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTranylcypromine%2520based%2520LSD1%2520inhibitor%253A%2520summary%2520and%2520prospective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14197%26epage%3D14215%26doi%3D10.1021%2Facs.jmedchem.0c00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span> </span><span class="NLM_article-title">The LSD1 inhibitor iadademstat is active in acute
myeloid leukemia</span>.  <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume">10</span>,  <span class="NLM_fpage">OF4</span>.<span class="refDoi"> DOI: 10.1158/2159-8290.CD-RW2020-155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1158%2F2159-8290.CD-RW2020-155" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+LSD1+inhibitor+iadademstat+is+active+in+acute%0Amyeloid+leukemia.+Cancer+Discovery+2020%2C+10%2C+OF4.+10.1158%2F2159-8290.CD-RW2020-155"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-RW2020-155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-RW2020-155%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520LSD1%2520inhibitor%2520iadademstat%2520is%2520active%2520in%2520acute%250Amyeloid%2520leukemia%26jtitle%3DCancer%2520Discovery%26date%3D2020%26volume%3D10%26spage%3DOF4%26doi%3D10.1158%2F2159-8290.CD-RW2020-155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascaro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirapu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estiarte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunardi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guibourt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lufino, M. M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willekens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valls, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fyfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro-Palomino, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buesa, C.</span></span> <span> </span><span class="NLM_article-title">ORY-1001, a potent and selective covalent KDM1A inhibitor, for the treatment of acute leukemia</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">495</span>– <span class="NLM_lpage">511</span>, e12 <span class="refDoi"> DOI: 10.1016/j.ccell.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ccell.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29502954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=495-511&author=T.+Maesauthor=C.+Mascaroauthor=I.+Tirapuauthor=A.+Estiarteauthor=F.+Ciceriauthor=S.+Lunardiauthor=N.+Guibourtauthor=A.+Perdonesauthor=M.+M.+P.+Lufinoauthor=T.+C.+P.+Somervailleauthor=D.+H.+Wisemanauthor=C.+Duyauthor=A.+Melnickauthor=C.+Willekensauthor=A.+Ortegaauthor=M.+Martinellauthor=N.+Vallsauthor=G.+Kurzauthor=M.+Fyfeauthor=J.+C.+Castro-Palominoauthor=C.+Buesa&title=ORY-1001%2C+a+potent+and+selective+covalent+KDM1A+inhibitor%2C+for+the+treatment+of+acute+leukemia&doi=10.1016%2Fj.ccell.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">ORY-1001, a Potent and Selective Covalent KDM1A Inhibitor, for the Treatment of Acute Leukemia</span></div><div class="casAuthors">Maes, Tamara; Mascaro, Cristina; Tirapu, Inigo; Estiarte, Angels; Ciceri, Filippo; Lunardi, Serena; Guibourt, Nathalie; Perdones, Alvaro; Lufino, Michele M. P.; Somervaille, Tim C. P.; Wiseman, Dan H.; Duy, Cihangir; Melnick, Ari; Willekens, Christophe; Ortega, Alberto; Martinell, Marc; Valls, Nuria; Kurz, Guido; Fyfe, Matthew; Castro-Palomino, Julio Cesar; Buesa, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">495-511.e12</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lysine-specific demethylase KDM1A is a key regulator of stem cell potential in acute myeloid leukemia (AML).  ORY-1001 is a highly potent and selective KDM1A inhibitor that induces H3K4me2 accumulation on KDM1A target genes, blast differentiation, and redn. of leukemic stem cell capacity in AML.  ORY-1001 exhibits potent synergy with std.-of-care drugs and selective epigenetic inhibitors, reduces growth of an AML xenograft model, and extends survival in a mouse PDX (patient-derived xenograft) model of T cell acute leukemia.  Surrogate pharmacodynamic biomarkers developed based on expression changes in leukemia cell lines were translated to samples from patients treated with ORY-1001.  ORY-1001 is a selective KDM1A inhibitor in clin. trials and is currently being evaluated in patients with leukemia and solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrb2QFk3s8-eLVg90H21EOLACvtfcHk0lji_hb3XJIAVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjsl2ktL8%253D&md5=68f69ade1b0110f3e0adf0173326e9b7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DMaes%26aufirst%3DT.%26aulast%3DMascaro%26aufirst%3DC.%26aulast%3DTirapu%26aufirst%3DI.%26aulast%3DEstiarte%26aufirst%3DA.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DLunardi%26aufirst%3DS.%26aulast%3DGuibourt%26aufirst%3DN.%26aulast%3DPerdones%26aufirst%3DA.%26aulast%3DLufino%26aufirst%3DM.%2BM.%2BP.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DDuy%26aufirst%3DC.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DWillekens%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DMartinell%26aufirst%3DM.%26aulast%3DValls%26aufirst%3DN.%26aulast%3DKurz%26aufirst%3DG.%26aulast%3DFyfe%26aufirst%3DM.%26aulast%3DCastro-Palomino%26aufirst%3DJ.%2BC.%26aulast%3DBuesa%26aufirst%3DC.%26atitle%3DORY-1001%252C%2520a%2520potent%2520and%2520selective%2520covalent%2520KDM1A%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520acute%2520leukemia%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D495%26epage%3D511%26doi%3D10.1016%2Fj.ccell.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levant, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keck, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. M.</span></span> <span> </span><span class="NLM_article-title">Tranylcypromine substituted <i>cis</i>-hydroxycyclobutylnaphthamides as potent and selective dopamine D<sub>3</sub> receptor antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">4962</span>– <span class="NLM_lpage">4968</span>, <span class="refDoi"> DOI: 10.1021/jm401798r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401798r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4962-4968&author=J.+Chenauthor=B.+Levantauthor=C.+Jiangauthor=T.+M.+Keckauthor=A.+H.+Newmanauthor=S.+M.+Wang&title=Tranylcypromine+substituted+cis-hydroxycyclobutylnaphthamides+as+potent+and+selective+dopamine+D3+receptor+antagonists&doi=10.1021%2Fjm401798r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Tranylcypromine Substituted cis-Hydroxycyclobutylnaphthamides as Potent and Selective Dopamine D3 Receptor Antagonists</span></div><div class="casAuthors">Chen, Jianyong; Levant, Beth; Jiang, Cheng; Keck, Thomas M.; Newman, Amy Hauck; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4962-4968</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report a class of potent and selective dopamine D3 receptor antagonists based upon tranylcypromine.  Although tranylcypromine has a low affinity for the rat D3 receptor (Ki = 12.8 μM), our efforts have yielded (1R,2S)-11 (CJ-1882), which has Ki values of 2.7 and 2.8 nM at the rat and human dopamine D3 receptors, resp., and displays resp. selectivities of >10000-fold and 223-fold over the rat and human D2 receptors.  Evaluation in a β-arrestin functional assay showed that (1R,2S)-11 is a potent and competitive antagonist at the human D3 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowWudWdAfWgLVg90H21EOLACvtfcHk0lhjGzIpQEhaOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXosVOnurg%253D&md5=e2ddf0711d00103be281cb043b19ec3a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401798r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401798r%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLevant%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DKeck%26aufirst%3DT.%2BM.%26aulast%3DNewman%26aufirst%3DA.%2BH.%26aulast%3DWang%26aufirst%3DS.%2BM.%26atitle%3DTranylcypromine%2520substituted%2520cis-hydroxycyclobutylnaphthamides%2520as%2520potent%2520and%2520selective%2520dopamine%2520D3%2520receptor%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4962%26epage%3D4968%26doi%3D10.1021%2Fjm401798r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prusevich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Givens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieniewicz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsiao, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adejola, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taverna, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tackett, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratan, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, O. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feinberg, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">A selective phenelzine analogue inhibitor of histone demethylase LSD1</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1284</span>– <span class="NLM_lpage">1293</span>, <span class="refDoi"> DOI: 10.1021/cb500018s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500018s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1284-1293&author=P.+Prusevichauthor=J.+H.+Kalinauthor=S.+A.+Mingauthor=M.+Bassoauthor=J.+Givensauthor=X.+Liauthor=J.+Huauthor=M.+S.+Taylorauthor=A.+M.+Cieniewiczauthor=P.+Y.+Hsiaoauthor=R.+Huangauthor=H.+Robersonauthor=N.+Adejolaauthor=L.+B.+Averyauthor=R.+A.+Caseroauthor=S.+D.+Tavernaauthor=J.+Qianauthor=A.+J.+Tackettauthor=R.+R.+Ratanauthor=O.+G.+McDonaldauthor=A.+P.+Feinbergauthor=P.+A.+Cole&title=A+selective+phenelzine+analogue+inhibitor+of+histone+demethylase+LSD1&doi=10.1021%2Fcb500018s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span></div><div class="casAuthors">Prusevich, Polina; Kalin, Jay H.; Ming, Shonoi A.; Basso, Manuela; Givens, Jeffrey; Li, Xin; Hu, Jianfei; Taylor, Martin S.; Cieniewicz, Anne M.; Hsiao, Po-Yuan; Huang, Rong; Roberson, Heather; Adejola, Nkosi; Avery, Lindsay B.; Casero, Robert A., Jr.; Taverna, Sean D.; Qian, Jiang; Tackett, Alan J.; Ratan, Rajiv R.; McDonald, Oliver G.; Feinberg, Andrew P.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1284-1293</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme that oxidatively cleaves Me groups from monomethyl and di-Me Lys 4 of histone H3 (H3K4Me1, H3K4Me2) and can contribute to gene silencing.  This study describes the design and synthesis of analogs of a monoamine oxidase antidepressant, phenelzine, and their LSD1 inhibitory properties.  A novel phenelzine analog (bizine) contg. a phenyl-butyrylamide appendage was shown to be a potent LSD1 inhibitor in vitro and was selective vs. monoamine oxidases A/B and the LSD1 homolog, LSD2.  Bizine was found to be effective at modulating bulk histone methylation in cancer cells, and ChIP-seq expts. revealed a statistically significant overlap in the H3K4 methylation pattern of genes affected by bizine and those altered in LSD1-/- cells.  Treatment of two cancer cell lines, LNCaP and H460, with bizine conferred a redn. in proliferation rate, and bizine showed additive to synergistic effects on cell growth when used in combination with two out of five HDAC inhibitors tested.  Moreover, neurons exposed to oxidative stress were protected by the presence of bizine, suggesting potential applications in neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Z1w5uKT7qbVg90H21EOLACvtfcHk0lhjGzIpQEhaOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D&md5=36c9f3fb9b28579fa3cad41052cf335a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fcb500018s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500018s%26sid%3Dliteratum%253Aachs%26aulast%3DPrusevich%26aufirst%3DP.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DMing%26aufirst%3DS.%2BA.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DGivens%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCieniewicz%26aufirst%3DA.%2BM.%26aulast%3DHsiao%26aufirst%3DP.%2BY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DRoberson%26aufirst%3DH.%26aulast%3DAdejola%26aufirst%3DN.%26aulast%3DAvery%26aufirst%3DL.%2BB.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DMcDonald%26aufirst%3DO.%2BG.%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520selective%2520phenelzine%2520analogue%2520inhibitor%2520of%2520histone%2520demethylase%2520LSD1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1284%26epage%3D1293%26doi%3D10.1021%2Fcb500018s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pippel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz-Fincke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yiu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Nonpeptidic propargylamines as inhibitors of lysine specific demethylase 1 (LSD1) with cellular activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7334</span>– <span class="NLM_lpage">7342</span>, <span class="refDoi"> DOI: 10.1021/jm400792m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400792m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7334-7342&author=M.+L.+Schmittauthor=A.+T.+Hauserauthor=L.+Carlinoauthor=M.+Pippelauthor=J.+Schulz-Finckeauthor=E.+Metzgerauthor=D.+Willmannauthor=T.+Yiuauthor=M.+Bartonauthor=R.+Schuleauthor=W.+Sipplauthor=M.+Jung&title=Nonpeptidic+propargylamines+as+inhibitors+of+lysine+specific+demethylase+1+%28LSD1%29+with+cellular+activity&doi=10.1021%2Fjm400792m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Nonpeptidic Propargylamines as Inhibitors of Lysine Specific Demethylase 1 (LSD1) with Cellular Activity</span></div><div class="casAuthors">Schmitt, Martin L.; Hauser, Alexander-Thomas; Carlino, Luca; Pippel, Martin; Schulz-Fincke, Johannes; Metzger, Eric; Willmann, Dominica; Yiu, Teresa; Barton, Michelle; Schuele, Roland; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7334-7342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine demethylases play an important role in epigenetic regulation and thus in the development of diseases like cancer or neurodegenerative disorders.  As the lysine specific demethylase 1 (LSD1/KDM1) has been strongly connected to androgen and estrogen dependent gene expression, it serves as a promising target for the therapy of hormone dependent cancer.  Here, we report on the discovery of new small mol. inhibitors of LSD1 contg. a propargylamine warhead, starting out from lysine contg. substrate analogs.  On the basis of these substrate mimicking inhibitors, we were able to increase potency by a combination of similarity-based virtual screening and subsequent synthetic optimization resulting in more druglike LSD1 inhibitors that led to histone hypermethylation in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGRr4rUPfsLVg90H21EOLACvtfcHk0ljaG74NW8RSsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO&md5=d4de914dec37f011a7b9488ec539a395</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm400792m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400792m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSchulz-Fincke%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DYiu%26aufirst%3DT.%26aulast%3DBarton%26aufirst%3DM.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DNonpeptidic%2520propargylamines%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520with%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7334%26epage%3D7342%26doi%3D10.1021%2Fjm400792m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span> <span> </span><span class="NLM_article-title">Reversible inhibitors of LSD1 as therapeutic agents in acute myeloid leukemia: clinical significance and progress to date</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">586</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+inhibitors+of+LSD1+as+therapeutic+agents+in+acute+myeloid+leukemia%3A+clinical+significance+and+progress+to+date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0ljaG74NW8RSsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520inhibitors%2520of%2520LSD1%2520as%2520therapeutic%2520agents%2520in%2520acute%2520myeloid%2520leukemia%253A%2520clinical%2520significance%2520and%2520progress%2520to%2520date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bianchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polticelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ascenzi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Federico, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariottini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cona, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of polyamine and spermine oxidases by polyamine analogues</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1123</span>, <span class="refDoi"> DOI: 10.1111/j.1742-4658.2006.05137.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fj.1742-4658.2006.05137.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=16519678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsVCmt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=2006&pages=1115-1123&author=M.+Bianchiauthor=F.+Polticelliauthor=P.+Ascenziauthor=M.+Bottaauthor=R.+Federicoauthor=P.+Mariottiniauthor=A.+Cona&title=Inhibition+of+polyamine+and+spermine+oxidases+by+polyamine+analogues&doi=10.1111%2Fj.1742-4658.2006.05137.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of polyamine and spermine oxidases by polyamine analogues</span></div><div class="casAuthors">Bianchi, Marzia; Polticelli, Fabio; Ascenzi, Paolo; Botta, Maurizio; Federico, Rodolfo; Mariottini, Paolo; Cona, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1115-1123</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Polyamine oxidase (PAO) and spermine oxidase (SMO) are involved in the catabolism of polyamines - basic regulators of cell growth and proliferation.  The discovery of selective inhibitors of PAO and SMO represents an important tool in studying the involvement of these enzymes in polyamine homeostasis and a starting point for the development of novel antineoplastic drugs.  Here, a comparative study on murine PAO (mPAO) and SMO (mSMO) inhibition by the polyamine analogs 1,8-diaminooctane, 1,12-diaminododecane, N-prenylagmatine (G3), guazatine and N,N1-bis(2,3-butadienyl)-1,4-butanediamine (MDL72527) is reported.  Interestingly, 1,12-Diaminododecane and G3 behave as specific inhibitors of mPAO, values of Ki for mPAO inhibition being lower than those for mSMO inactivation by several orders of magnitude.  The anal. of mol. models of mPAO and mSMO indicates a significant redn. of the hydrophobic pocket located in maize PAO (MPAO) at the wider catalytic tunnel opening.  This observation provides a rationale to explain the lower affinity displayed by G3, guazatine and MDL72527 for mPAO and mSMO as compared to MPAO.  The different behavior displayed by 1,12-diaminododecane towards mPAO and mSMO reveals the occurrence of basic differences in the ligand binding mode of the two enzymes, the first enzyme interacting mainly with substrate secondary amino groups and the second one with substrate primary amino groups.  Thus, the data reported here provide the basis for the development of novel and selective inhibitors able to discriminate between mammalian SMO and PAO activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqrXK8OAU1drVg90H21EOLACvtfcHk0ljaG74NW8RSsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsVCmt7w%253D&md5=370a7eff98ab45bec57d14c1839cad93</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2006.05137.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2006.05137.x%26sid%3Dliteratum%253Aachs%26aulast%3DBianchi%26aufirst%3DM.%26aulast%3DPolticelli%26aufirst%3DF.%26aulast%3DAscenzi%26aufirst%3DP.%26aulast%3DBotta%26aufirst%3DM.%26aulast%3DFederico%26aufirst%3DR.%26aulast%3DMariottini%26aufirst%3DP.%26aulast%3DCona%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520polyamine%2520and%2520spermine%2520oxidases%2520by%2520polyamine%2520analogues%26jtitle%3DFEBS%2520J.%26date%3D2006%26volume%3D273%26spage%3D1115%26epage%3D1123%26doi%3D10.1111%2Fj.1742-4658.2006.05137.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">8023</span>– <span class="NLM_lpage">8028</span>, <span class="refDoi"> DOI: 10.1073/pnas.0700720104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1073%2Fpnas.0700720104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17463086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+Murray+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes&doi=10.1073%2Fpnas.0700720104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span></div><div class="casAuthors">Huang, Yi; Greene, Eriko; Stewart, Tracy Murray; Goodwin, Andrew C.; Baylin, Stephen B.; Woster, Patrick M.; Casero, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8023-8028</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Epigenetic chromatin modification is a major regulator of eukaryotic gene expression, and aberrant epigenetic silencing of gene expression contributes to tumorigenesis.  Histone modifications include acetylation, phosphorylation, and methylation, resulting in a combination of histone marks known collectively as the histone code.  The chromatin marks at a given promoter det., in part, whether specific promoters are in an open/active conformation or closed/repressed conformation.  Dimethyl-lysine 4 histone H3 (H3K4me2) is a transcription-activating chromatin mark at gene promoters, and demethylation of this mark by the lysine-specific demethylase 1 (LSD1), a homolog of polyamine oxidases, may broadly repress gene expression.  We now report that novel biguanide and bisguanidine polyamine analogs are potent inhibitors of LSD1.  These analogs inhibit LSD1 in human colon carcinoma cells and affect a reexpression of multiple, aberrantly silenced genes important in the development of colon cancer, including members of the secreted frizzle-related proteins (SFRPs) and the GATA family of transcription factors.  Furthermore, we demonstrate by chromatin immunopptn. anal. that the reexpression is concurrent with increased H3K4me2 and acetyl-H3K9 marks, decreased H3K9me1 and H3K9me2 repressive marks.  We thus define important new agents for reversing aberrant repression of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC96zJoYEK67Vg90H21EOLACvtfcHk0ljaG74NW8RSsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D&md5=4c7863ecb038e3e10c5774cba2c28a45</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700720104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700720104%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMurray%2BStewart%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028%26doi%3D10.1073%2Fpnas.0700720104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marton, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Polyamine analogs modulate gene expression by inhibiting lysine-specific demethylase 1 (LSD1) and altering chromatin structure in human breast cancer cells</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">887</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1007/s00726-011-1004-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00726-011-1004-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=21805138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtlyqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2012&pages=887-898&author=Q.+Zhuauthor=Y.+Huangauthor=L.+J.+Martonauthor=P.+M.+Wosterauthor=N.+E.+Davidsonauthor=R.+A.+Casero&title=Polyamine+analogs+modulate+gene+expression+by+inhibiting+lysine-specific+demethylase+1+%28LSD1%29+and+altering+chromatin+structure+in+human+breast+cancer+cells&doi=10.1007%2Fs00726-011-1004-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Polyamine analogs modulate gene expression by inhibiting lysine-specific demethylase 1 (LSD1) and altering chromatin structure in human breast cancer cells</span></div><div class="casAuthors">Zhu, Qingsong; Huang, Yi; Marton, Laurence J.; Woster, Patrick M.; Davidson, Nancy E.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">887-898</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">SpringerWienNewYork</span>)
        </div><div class="casAbstract">Aberrant epigenetic repression of gene expression has been implicated in most cancers, including breast cancer.  The nuclear amine oxidase, lysine-specific demethylase 1 (LSD1) has the ability to broadly repress gene expression by removing the activating mono- and di-methylation marks at the lysine 4 residue of histone 3 (H3K4me1 and me2).  Addnl., LSD1 is highly expressed in estrogen receptor α neg. (ER-) breast cancer cells.  Since epigenetic marks are reversible, they make attractive therapeutic targets.  Here we examine the effects of polyamine analog inhibitors of LSD1 on gene expression, with the goal of targeting LSD1 as a therapeutic modality in the treatment of breast cancer.  Exposure of the ER-neg. human breast cancer cells, MDA-MB-231 to the LSD1 inhibitors, 2d or PG11144, significantly increases global H3K4me1 and H3K4me2, and alters gene expression.  Array anal. indicated that 98 (75 up and 23 down) and 477 (237 up and 240 down) genes changed expression by at least 1.5-fold or greater after treatment with 2d and PG11144, resp.  The expression of 12 up-regulated genes by 2d and 14 up-regulated genes by PG11144 was validated by quant. RT-PCR.  Quant. chromatin immunopptn. (ChIP) anal. demonstrated that up-regulated gene expression by polyamine analogs is assocd. with increase of the active histone marks H3K4me1, H3K4me2 and H3K9act, and decrease of the repressive histone marks H3K9me2 and H3K27me3, in the promoter regions of the relevant target genes.  These data indicate that the pharmacol. inhibition of LSD1 can effectively alter gene expression and that this therapeutic strategy has potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3fWUWScc-bVg90H21EOLACvtfcHk0lgHNhpJ77ubOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtlyqtbo%253D&md5=3d99339a2158d2a16e270250c4b44426</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2Fs00726-011-1004-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-011-1004-1%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DMarton%26aufirst%3DL.%2BJ.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DDavidson%26aufirst%3DN.%2BE.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DPolyamine%2520analogs%2520modulate%2520gene%2520expression%2520by%2520inhibiting%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520and%2520altering%2520chromatin%2520structure%2520in%2520human%2520breast%2520cancer%2520cells%26jtitle%3DAmino%2520Acids%26date%3D2012%26volume%3D42%26spage%3D887%26epage%3D898%26doi%3D10.1007%2Fs00726-011-1004-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span> <span> </span><span class="NLM_article-title">Novel histone demethylase LSD1 inhibitors selectively target cancer cells with pluripotent stem cell properties</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">7238</span>– <span class="NLM_lpage">7249</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-0896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1158%2F0008-5472.CAN-11-0896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=21975933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFCmsb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=7238-7249&author=J.+Wangauthor=F.+Luauthor=Q.+Renauthor=H.+Sunauthor=Z.+S.+Xuauthor=R.+F.+Lanauthor=Y.+Q.+Liuauthor=D.+Wardauthor=J.+M.+Quanauthor=T.+Yeauthor=H.+Zhang&title=Novel+histone+demethylase+LSD1+inhibitors+selectively+target+cancer+cells+with+pluripotent+stem+cell+properties&doi=10.1158%2F0008-5472.CAN-11-0896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Histone Demethylase LSD1 Inhibitors Selectively Target Cancer Cells with Pluripotent Stem Cell Properties</span></div><div class="casAuthors">Wang, Jing; Lu, Fei; Ren, Qi; Sun, Hong; Xu, Zhengshuang; Lan, Rongfeng; Liu, Yuqing; Ward, David; Quan, Junmin; Ye, Tao; Zhang, Hui</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7238-7249</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone modification dets. epigenetic patterns of gene expression with methylation of histone H3 at lysine 4 (H3K4) often assocd. with active promoters.  LSD1/KDM1 is a histone demethylase that suppresses gene expression by converting dimethylated H3K4 to mono- and unmethylated H3K4.  LSD1 is essential for metazoan development, but its pathophysiol. functions in cancer remain mainly uncharacterized.  In this study, we developed specific bioactive small inhibitors of LSD1 that enhance H3K4 methylation and derepress epigenetically suppressed genes in vivo.  Strikingly, these compds. inhibited the proliferation of pluripotent cancer cells including teratocarcinoma, embryonic carcinoma, and seminoma or embryonic stem cells that express the stem cell markers Oct4 and Sox2 while displaying min. growth-inhibitory effects on non-pluripotent cancer or normal somatic cells.  RNA interference-mediated knockdown of LSD1 expression phenocopied these effects, confirming the specificity of small mols. and further establishing the high degree of sensitivity and selectivity of pluripotent cancer cells to LSD1 ablation.  In support of these results, we found that LSD1 protein level is highly elevated in pluripotent cancer cells and in human testicular seminoma tissues that express Oct4.  Using these novel chem. inhibitors as probes, our findings establish LSD1 and histone H3K4 methylation as essential cancer-selective epigenetic targets in cancer cells that have pluripotent stem cell properties.  Cancer Res; 71(23); 7238-49.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojMPbxSB7ahbVg90H21EOLACvtfcHk0lgHNhpJ77ubOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFCmsb%252FF&md5=742f5d8239623f8ebe17f968f152a2a0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-0896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-0896%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DF.%26aulast%3DRen%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%2BS.%26aulast%3DLan%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BQ.%26aulast%3DWard%26aufirst%3DD.%26aulast%3DQuan%26aufirst%3DJ.%2BM.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26atitle%3DNovel%2520histone%2520demethylase%2520LSD1%2520inhibitors%2520selectively%2520target%2520cancer%2520cells%2520with%2520pluripotent%2520stem%2520cell%2520properties%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D7238%26epage%3D7249%26doi%3D10.1158%2F0008-5472.CAN-11-0896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. C.</span></span> <span> </span><span class="NLM_article-title">CBB1003, a lysine-specific demethylase 1 inhibitor, suppresses colorectal cancer cells growth through down-regulation of leucine-rich repeat-containing G-protein-coupled receptor 5 expression</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1007/s00432-014-1782-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00432-014-1782-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25060070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1aitrzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2015&pages=11-21&author=H.+C.+Hsuauthor=Y.+S.+Liuauthor=K.+C.+Tsengauthor=T.+S.+Yangauthor=C.+Y.+Yehauthor=J.+F.+Youauthor=H.+Y.+Hungauthor=S.+J.+Chenauthor=H.+C.+Chen&title=CBB1003%2C+a+lysine-specific+demethylase+1+inhibitor%2C+suppresses+colorectal+cancer+cells+growth+through+down-regulation+of+leucine-rich+repeat-containing+G-protein-coupled+receptor+5+expression&doi=10.1007%2Fs00432-014-1782-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">CBB1003, a lysine-specific demethylase 1 inhibitor, suppresses colorectal cancer cells growth through down-regulation of leucine-rich repeat-containing G-protein-coupled receptor 5 expression</span></div><div class="casAuthors">Hsu, Hung-Chih; Liu, Yi-Shiuan; Tseng, Kai-Chi; Yang, Tsai-Sheng; Yeh, Chien-Yuh; You, Jeng-Fu; Hung, Hsin-Yuan; Chen, Shu-Jen; Chen, Hua-Chien</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer Research and Clinical Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-21</span>CODEN:
                <span class="NLM_cas:coden">JCROD7</span>;
        ISSN:<span class="NLM_cas:issn">0171-5216</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Lysine-specific demethylase 1 (LSD1) was highly expressed in several malignancies and had been implicated in pathol. processes of cancer cells.  However, the role of LSD1 in colorectal cancer (CRC) carcinogenesis, prognosis and treatment remains uncharacterized.  Methods: In this study, we examd. LSD1 expression in paraffin-embedded CRC specimens from 295 patients, including 65 patients with paired samples of colorectal carcinoma, adjacent adenoma and normal colorectal tissues.  Using an LSD1 inhibitor, CBB1003, as a probe, we studied the assocn. between LSD1 and leucine-rich repeat-contg. G-protein-coupled receptor 5 (LGR5), a CRC stem cell marker involved in carcinogenesis.  The anti-tumor effects of CBB1003 on CRC cells were also examd.  Results: LSD1 expression was significantly elevated in colorectal tumor tissues compared with adjacent adenoma and normal colorectal tissues (P < 0.001), and LSD1 levels were significantly correlated with an advanced AJCC T stage (P = 0.012) and distant metastasis (P = 0.004).  CBB1003 inhibited CRC cell proliferation and colony formation.  In cultured CRC cells, inhibiting LSD1 activity by CBB1003 caused a decrease in LGR5 levels while overexpression of LGR5 reduced CBB1003-induced cell death.  We also obsd. the inactivation of β-catenin/TCF signaling after CBB1003 treatment, consistent with the pos. correlations among LSD1, LGR5, β-catenin and c-Myc expression in human colon tumor and adenoma tissues.  Conclusion: Our result suggested that LSD1 overexpression promotes CRC development and that the LSD1 inhibitor inhibits CRC cell growth by down-regulating LGR5 levels and inactivates the Wnt/β-catenin pathway.  Thus, LSD1 and its inhibitor might provide a new target or a useful strategy for therapy of CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ21LenCyhXbVg90H21EOLACvtfcHk0lgHNhpJ77ubOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1aitrzM&md5=75c69d5e807d2d587103b9a8761077b7</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1007%2Fs00432-014-1782-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-014-1782-4%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DH.%2BC.%26aulast%3DLiu%26aufirst%3DY.%2BS.%26aulast%3DTseng%26aufirst%3DK.%2BC.%26aulast%3DYang%26aufirst%3DT.%2BS.%26aulast%3DYeh%26aufirst%3DC.%2BY.%26aulast%3DYou%26aufirst%3DJ.%2BF.%26aulast%3DHung%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DChen%26aufirst%3DH.%2BC.%26atitle%3DCBB1003%252C%2520a%2520lysine-specific%2520demethylase%25201%2520inhibitor%252C%2520suppresses%2520colorectal%2520cancer%2520cells%2520growth%2520through%2520down-regulation%2520of%2520leucine-rich%2520repeat-containing%2520G-protein-coupled%2520receptor%25205%2520expression%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2015%26volume%3D141%26spage%3D11%26epage%3D21%26doi%3D10.1007%2Fs00432-014-1782-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowotarski, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachaiyappan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holshouser, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutz, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Structure-activity study for (bis)ureidopropyl- and (bis)thioureidopropyldiamine LSD1 inhibitors with 3–5-3 and 3–6-3 carbon backbone architectures</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.01.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2015.01.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25725609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=1601-1612&author=S.+L.+Nowotarskiauthor=B.+Pachaiyappanauthor=S.+L.+Holshouserauthor=C.+J.+Kutzauthor=Y.+Liauthor=Y.+Huangauthor=S.+K.+Sharmaauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=Structure-activity+study+for+%28bis%29ureidopropyl-+and+%28bis%29thioureidopropyldiamine+LSD1+inhibitors+with+3%E2%80%935-3+and+3%E2%80%936-3+carbon+backbone+architectures&doi=10.1016%2Fj.bmc.2015.01.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity study for (bis)ureidopropyl- and (bis)thioureidopropyldiamine LSD1 inhibitors with 3-5-3 and 3-6-3 carbon backbone architectures</span></div><div class="casAuthors">Nowotarski, Shannon L.; Pachaiyappan, Boobalan; Holshouser, Steven L.; Kutz, Craig J.; Li, Youxuan; Huang, Yi; Sharma, Shiv K.; Casero, Robert A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1601-1612</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Methylation at specific histone lysine residues is a crit. posttranslational modification that alters chromatin architecture, and dysregulated lysine methylation/demethylation is assocd. with the silencing of tumor suppressor genes.  The enzyme lysine-specific demethylase 1 (LSD1) complexed to specific transcription factors catalyzes the oxidative demethylation of mono- and dimethyllysine 4 of histone H3 (H3K4me and H3K4me2, resp.).  The authors have previously reported potent (bis)urea and (bis)thiourea LSD1 inhibitors that increase cellular levels of H3K4me and H3K4me2, promote the reexpression of silenced tumor suppressor genes and suppress tumor growth in vitro.  Here the authors report the design addnl. (bis)urea and (bis)thiourea LSD1 inhibitors that feature 3-5-3 or 3-6-3 carbon backbone architectures.  Three of these compds. displayed single-digit IC50 values in a recombinant LSD1 assay.  In addn., compd. I exhibited an IC50 of 4.2 μM against the Calu-6 human lung adenocarcinoma line, and 4.8 μM against the MCF7 breast tumor cell line, in an MTS cell viability assay.  Following treatment with I and its analogs, Calu-6 cells exhibited a significant increase in the mRNA expression for the silenced tumor suppressor genes SFRP2, HCAD and p16, and modest increases in GATA4 message.  The compds. described in this paper represent the most potent epigenetic modulators in this series, and have potential for use as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc-GvXmyP2JbVg90H21EOLACvtfcHk0lh0ihVSF5GzTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrY%253D&md5=84aca2f083c3f4df2d66818c08129152</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.01.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.01.049%26sid%3Dliteratum%253Aachs%26aulast%3DNowotarski%26aufirst%3DS.%2BL.%26aulast%3DPachaiyappan%26aufirst%3DB.%26aulast%3DHolshouser%26aufirst%3DS.%2BL.%26aulast%3DKutz%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DSharma%26aufirst%3DS.%2BK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DStructure-activity%2520study%2520for%2520%2528bis%2529ureidopropyl-%2520and%2520%2528bis%2529thioureidopropyldiamine%2520LSD1%2520inhibitors%2520with%25203%25E2%2580%25935-3%2520and%25203%25E2%2580%25936-3%2520carbon%2520backbone%2520architectures%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D1601%26epage%3D1612%26doi%3D10.1016%2Fj.bmc.2015.01.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tortorici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrello, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tardugno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarelli, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellore, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crabb, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packham, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1677</span>– <span class="NLM_lpage">1682</span>, <span class="refDoi"> DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged α-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0lhxXhat7swQRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How proteins bind macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+proteins+bind+macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0licyciRIZgv1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520proteins%2520bind%2520macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">How beyond rule of 5 drugs and clinical candidates bind to their targets</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">2312</span>– <span class="NLM_lpage">2327</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01286</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01286" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=2312-2327&author=B.+C.+Doakauthor=J.+Zhengauthor=D.+Dobritzschauthor=J.+Kihlberg&title=How+beyond+rule+of+5+drugs+and+clinical+candidates+bind+to+their+targets&doi=10.1021%2Facs.jmedchem.5b01286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">How Beyond Rule of 5 Drugs and Clinical Candidates Bind to Their Targets</span></div><div class="casAuthors">Doak, Bradley C.; Zheng, Jie; Dobritzsch, Doreen; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2327</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To improve discovery of drugs for difficult targets, the opportunities of chem. space beyond the rule of 5 (bRo5) were examd. by retrospective anal. of a comprehensive set of structures for complexes between drugs and clin. candidates and their targets.  The anal. illustrates the potential of compds. far beyond rule of 5 space to modulate novel and difficult target classes that have large, flat, and groove-shaped binding sites.  However, ligand efficiencies are significantly reduced for flat- and groove-shape binding sites, suggesting that adjustments of how to use such metrics are required.  Ligands bRo5 appear to benefit from an appropriate balance between rigidity and flexibility to bind with sufficient affinity to their targets, with macrocycles and nonmacrocycles being found to have similar flexibility.  However, macrocycles were more disk- and spherelike, which may contribute to their superior binding to flat sites, while rigidification of nonmacrocycles lead to rodlike ligands that bind well to groove-shaped binding sites.  These insights should contribute to altering perceptions of what targets are considered "druggable" and provide support for drug design in beyond rule of 5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZD-h9VodiPLVg90H21EOLACvtfcHk0li4szNIput16Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Cgu77F&md5=a417b605d1a3ffbc7ee84706401f01f4</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01286%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DHow%2520beyond%2520rule%2520of%25205%2520drugs%2520and%2520clinical%2520candidates%2520bind%2520to%2520their%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D2312%26epage%3D2327%26doi%3D10.1021%2Facs.jmedchem.5b01286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaglioli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of LSD1-CoREST selectivity in histone H3 recognition</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">20070</span>– <span class="NLM_lpage">20074</span>, <span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+basis+of+LSD1-CoREST+selectivity+in+histone+H3+recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0li4szNIput16Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520LSD1-CoREST%2520selectivity%2520in%2520histone%2520H3%2520recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasinghe, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel cyclic peptide inhibitors of lysine-specific demethylase 1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1021/ml4002997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4002997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslCrsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=29-33&author=I.+R.+Kumarasingheauthor=P.+M.+Woster&title=Synthesis+and+evaluation+of+novel+cyclic+peptide+inhibitors+of+lysine-specific+demethylase+1&doi=10.1021%2Fml4002997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Novel Cyclic Peptide Inhibitors of Lysine-Specific Demethylase 1</span></div><div class="casAuthors">Kumarasinghe, Isuru R.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-33</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) selectively removes Me groups from mono- and dimethylated histone 3 lysine 4 (H3K4), resulting in gene silencing.  LSD1 is overexpressed in many human cancers, resulting in aberrant silencing of tumor suppressor genes.  Thus, LSD1 is a validated target for the discovery of antitumor agents.  Using a ligand-based approach, the authors designed and synthesized a series of cyclic and linear peptides that are effective inhibitors of LSD1.  Linear peptide 7 and cyclic peptide 9 inhibited LSD1 in vitro by 91 and 94%, resp., at a concn. of 10 μM.  Compd. 9 was a potent LSD1 inhibitor (IC50 = 2.1 μM; Ki = 385 nM) and had moderate antitumor activity in the MCF-7 and Calu-6 cell lines in vitro.  Importantly, 9 is significantly more stable to hydrolysis in rat plasma than the linear analog 7.  The cyclic peptides described herein represent important lead structures in the search for inhibitors of flavin-dependent histone demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGZKg9n5W93bVg90H21EOLACvtfcHk0li4szNIput16Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslCrsr3F&md5=65810b4100bf4fb4fb10c204e77f88a0</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fml4002997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4002997%26sid%3Dliteratum%253Aachs%26aulast%3DKumarasinghe%26aufirst%3DI.%2BR.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520cyclic%2520peptide%2520inhibitors%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D29%26epage%3D33%26doi%3D10.1021%2Fml4002997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasinghe, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Cyclic peptide inhibitors of lysine-specific demethylase 1 with improved potency identified by alanine scanning mutagenesis</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2018.01.098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29459279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1yrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2018&pages=210-220&author=I.+R.+Kumarasingheauthor=P.+M.+Woster&title=Cyclic+peptide+inhibitors+of+lysine-specific+demethylase+1+with+improved+potency+identified+by+alanine+scanning+mutagenesis&doi=10.1016%2Fj.ejmech.2018.01.098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic by peptide inhibitors of lysine-specific demethylase 1 with improved potency identified by alanine scanning mutagenesis</span></div><div class="casAuthors">Kumarasinghe, Isuru R.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">210-220</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is a chromatin-remodeling enzyme that plays an important role in cancer.  Over-expression of LSD1 decreases methylation at histone 3 lysine 4, and aberrantly silences tumor suppressor genes.  Inhibitors of LSD1 have been designed as chem. probes and potential antitumor agents.  The authors recently reported the cyclic peptide 9 (H-ARTM-c[K5TARK9E10]TGGK14APRKQLA-NH2), which potently and reversibly inhibits LSD1 (IC50 2.1 μM; Ki 385nM).  Systematic alanine mutagenesis of 9 revealed residues that are crit. for LSD1 inhibition, and these mutated peptides were evaluated as LSD1 inhibitors.  Alanine substitution at positions 2, 3, 4, 6 and 11-17 preserved inhibition, while substitution of alanine at positions 8 and 9 resulted in complete loss of activity.  Cyclic mutant peptides 11 (H-ARAM-c[K5TARKE10]TGGK14APRKQLA-OH) and 16 (H-ARTM-c[K5TARKE10]AGG-KAPRKQLA-OH) produced the greatest LSD1 inhibition, and 11, 16, 27 and 28 increased global H3K4me2 in K562cells.  In addn., 16, 27 and 28 promoted significant increases in H3K4me2 levels at the promoter sites of the genes IGFBP2 and FEZ1.  Data from these LSD1 inhibitors will aid in the design of peptidomimetics with improved stability and pharmacokinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMbcw-dQpAy7Vg90H21EOLACvtfcHk0liXkJAd62nCcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1yrsLg%253D&md5=931ec70bf2eb602bc64fbdc3b8b68ae3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.098%26sid%3Dliteratum%253Aachs%26aulast%3DKumarasinghe%26aufirst%3DI.%2BR.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DCyclic%2520peptide%2520inhibitors%2520of%2520lysine-specific%2520demethylase%25201%2520with%2520improved%2520potency%2520identified%2520by%2520alanine%2520scanning%2520mutagenesis%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D148%26spage%3D210%26epage%3D220%26doi%3D10.1016%2Fj.ejmech.2018.01.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talibov, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebastiano, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobritzsch, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danielson, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic peptides uncover a novel binding mode for reversible inhibitors of LSD1</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3979</span>– <span class="NLM_lpage">3995</span>, <span class="refDoi"> DOI: 10.1021/acsomega.9b03493</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.9b03493" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFansLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=3979-3995&author=J.+Yangauthor=V.+O.+Talibovauthor=S.+Peintnerauthor=C.+Rheeauthor=V.+Poongavanamauthor=M.+Geitmannauthor=M.+R.+Sebastianoauthor=B.+Simonauthor=J.+Hennigauthor=D.+Dobritzschauthor=U.+H.+Danielsonauthor=J.+Kihlberg&title=Macrocyclic+peptides+uncover+a+novel+binding+mode+for+reversible+inhibitors+of+LSD1&doi=10.1021%2Facsomega.9b03493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides Uncover a Novel Binding Mode for Reversible Inhibitors of LSD1</span></div><div class="casAuthors">Yang, Jie; Talibov, Vladimir O.; Peintner, Stefan; Rhee, Claire; Poongavanam, Vasanthanathan; Geitmann, Matthis; Sebastiano, Matteo Rossi; Simon, Bernd; Hennig, Janosch; Dobritzsch, Doreen; Danielson, U. Helena; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3979-3995</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme which regulates the methylation of Lys4 of histone 3 (H3) and is overexpressed in certain cancers.  We used structures of H3 substrate analogs bound to LSD1 to design macrocyclic peptide inhibitors of LSD1.  A linear, Lys4 to Met-substituted, 11-mer (4) was identified as the shortest peptide distinctly interacting with LSD1.  It was evolved into macrocycle 31, which was >40 fold more potent (Ki = 2.3μM) than 4.  Linear and macrocyclic peptides exhibited unexpected differences in structure-activity relationships for interactions with LSD1, indicating that they bind LSD1 differently.  This was confirmed by the crystal structure of 31 in complex with LSD1-CoREST1, which revealed a novel binding mode at the outer rim of the LSD1 active site and without a direct interaction with FAD.  NMR spectroscopy of 31 suggests that macrocyclization restricts its soln. ensemble to conformations that include the one in the cryst. complex.  Our results provide a solid basis for the design of optimized reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSVg23mFeHAbVg90H21EOLACvtfcHk0liXkJAd62nCcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFansLc%253D&md5=34987d78346262625897765eefc198cf</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facsomega.9b03493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.9b03493%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTalibov%26aufirst%3DV.%2BO.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DRhee%26aufirst%3DC.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DSebastiano%26aufirst%3DM.%2BR.%26aulast%3DSimon%26aufirst%3DB.%26aulast%3DHennig%26aufirst%3DJ.%26aulast%3DDobritzsch%26aufirst%3DD.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520peptides%2520uncover%2520a%2520novel%2520binding%2520mode%2520for%2520reversible%2520inhibitors%2520of%2520LSD1%26jtitle%3DACS%2520Omega%26date%3D2020%26volume%3D5%26spage%3D3979%26epage%3D3995%26doi%3D10.1021%2Facsomega.9b03493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Speranzini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrocco, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forgione, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forneris, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehdipour, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Polymyxins and quinazolines are LSD1/KDM1A inhibitors with unusual structural features</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1601017</span> <span class="refDoi"> DOI: 10.1126/sciadv.1601017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1126%2Fsciadv.1601017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27626075" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltVWisL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=V.+Speranziniauthor=D.+Rotiliauthor=G.+Ciossaniauthor=S.+Pilottoauthor=B.+Marroccoauthor=M.+Forgioneauthor=A.+Lucidiauthor=F.+Fornerisauthor=P.+Mehdipourauthor=S.+Velankarauthor=A.+Maiauthor=A.+Mattevi&title=Polymyxins+and+quinazolines+are+LSD1%2FKDM1A+inhibitors+with+unusual+structural+features&doi=10.1126%2Fsciadv.1601017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Polymyxins and quinazolines are LSD1/KDM1A inhibitors with unusual structural features</span></div><div class="casAuthors">Speranzini, Valentina; Rotili, Dante; Ciossani, Giuseppe; Pilotto, Simona; Marrocco, Biagina; Forgione, Mariantonietta; Lucidi, Alessia; Forneris, Federico; Mehdipour, Parinaz; Velankar, Sameer; Mai, Antonello; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1601017/1-e1601017/8</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Because of its involvement in the progression of several malignant tumors, the histone lysine-specific demethylase1 (LSD1) has become a prominent drug target in modern medicinal chem. research.  We report on the discoveryof two classes of noncovalent inhibitors displaying unique structural features.  The antibiotics polymyxins bind atthe entrance of the substrate cleft, where their highly charged cyclic moiety interacts with a cluster of pos. charged amino acids.  The same site is occupied by quinazoline-based compds., which were found to inhibit theenzyme through a most peculiar mode because they form a pile of five to seven mols. that obstruct accessto the active center.  These data significantly indicate unpredictable strategies for the development of epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs4C4ugZSy5bVg90H21EOLACvtfcHk0lgsgn899x7zqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltVWisL8%253D&md5=1bb63c0077a5f83a3d270933314b7d9d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1601017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1601017%26sid%3Dliteratum%253Aachs%26aulast%3DSperanzini%26aufirst%3DV.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DForgione%26aufirst%3DM.%26aulast%3DLucidi%26aufirst%3DA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DMehdipour%26aufirst%3DP.%26aulast%3DVelankar%26aufirst%3DS.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DPolymyxins%2520and%2520quinazolines%2520are%2520LSD1%252FKDM1A%2520inhibitors%2520with%2520unusual%2520structural%2520features%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1601017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span> <span> </span><span class="NLM_article-title">Drugging the cancer epigenome</span>.  <i>Proceedings of the 104th Annual Meeting of the American Association for Cancer Research</i>; <span class="NLM_publisher-name">American Association for Cancer Research</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=D.+Dhanak&title=Drugging+the+cancer+epigenome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DDrugging%2520the%2520cancer%2520epigenome%26jtitle%3DProceedings%2520of%2520the%2520104th%2520Annual%2520Meeting%2520of%2520the%2520American%2520Association%2520for%2520Cancer%2520Research%26pub%3DAmerican%2520Association%2520for%2520Cancer%2520Research%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitchin, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockerill, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairweather, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1513</span>– <span class="NLM_lpage">1522</span>, <span class="refDoi"> DOI: 10.1039/c3md00226h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2Fc3md00226h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1513-1522&author=J.+R.+Hitchinauthor=J.+Blaggauthor=R.+Burkeauthor=S.+Burnsauthor=M.+J.+Cockerillauthor=E.+E.+Fairweatherauthor=C.+Huttonauthor=A.+M.+Jordanauthor=C.+McAndrewauthor=A.+Mirzaauthor=D.+Mouldauthor=G.+J.+Thomsonauthor=I.+Waddellauthor=D.+J.+Ogilvie&title=Development+and+evaluation+of+selective%2C+reversible+LSD1+inhibitors+derived+from+fragments&doi=10.1039%2Fc3md00226h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span></div><div class="casAuthors">Hitchin, James R.; Blagg, Julian; Burke, Rosemary; Burns, Samantha; Cockerill, Mark J.; Fairweather, Emma E.; Hutton, Colin; Jordan, Allan M.; McAndrew, Craig; Mirza, Amin; Mould, Daniel; Thomson, Graeme J.; Waddell, Ian; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1513-1522</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of aminothiazoles have been developed as reversible inhibitors of lysine specific demethylase 1 (LSD1) through the expansion of a hit derived from a high concn. biochem. fragment based screen of 2466 compds.  The potency of the initial fragment hit was increased 32-fold through synthesis, with one series of compds. showing clear structure-activity relationships and inhibitory activities in the range of 7 to 187 μM in a biochem. assay.  This series also showed selectivity against the related FAD-dependent enzyme mono-amine oxidase A (MAO-A).  Although a wide range of irreversible inhibitors of LSD1 have been reported with activities in the low nanomolar range, this work represents one of the first reported examples of a reversible small mol. inhibitor of LSD1 with clear SAR and selectivity against MAO-A, and could provide a platform for the development of more potent reversible inhibitors.  Herein, we also report the use of a recently developed cell-based assay for profiling LSD1 inhibitors, and present results on our own compds. as well as a selection of recently described reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUgrL8DOIuDbVg90H21EOLACvtfcHk0lhQYkd_7nCosw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK&md5=bec441bbb94f970eab94b96026db7fbe</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2Fc3md00226h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00226h%26sid%3Dliteratum%253Aachs%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DBurns%26aufirst%3DS.%26aulast%3DCockerill%26aufirst%3DM.%2BJ.%26aulast%3DFairweather%26aufirst%3DE.%2BE.%26aulast%3DHutton%26aufirst%3DC.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DMould%26aufirst%3DD.%26aulast%3DThomson%26aufirst%3DG.%2BJ.%26aulast%3DWaddell%26aufirst%3DI.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520selective%252C%2520reversible%2520LSD1%2520inhibitors%2520derived%2520from%2520fragments%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D1513%26epage%3D1522%26doi%3D10.1039%2Fc3md00226h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y. C.</span></span> <span> </span><span class="NLM_article-title">3-(Piperidin-4-ylmethoxy)pyridine containing compounds are potent inhibitors of lysine specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=253-263&author=F.+R.+Wuauthor=C.+Zhouauthor=Y.+Yaoauthor=L.+P.+Weiauthor=Z.+Z.+Fengauthor=L.+S.+Dengauthor=Y.+C.+Song&title=3-%28Piperidin-4-ylmethoxy%29pyridine+containing+compounds+are+potent+inhibitors+of+lysine+specific+demethylase+1&doi=10.1021%2Facs.jmedchem.5b01361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Wu, Fangrui; Zhou, Chao; Yao, Yuan; Wei, Liping; Feng, Zizhen; Deng, Lisheng; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-263</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation.  Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4).  LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression.  We report the design, synthesis, and structure-activity relationships of 3-(piperidin-4-ylmethoxy)pyridine contg. compds. as potent LSD1 inhibitors with Ki values as low as 29 nM.  These compds. exhibited high selectivity (>160×) against related monoamine oxidase A and B.  Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode.  The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKAqQdCeodaLVg90H21EOLACvtfcHk0lhQYkd_7nCosw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylu77N&md5=2c74e06ed3c755826d25917d02cb845f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%2BR.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%2BP.%26aulast%3DFeng%26aufirst%3DZ.%2BZ.%26aulast%3DDeng%26aufirst%3DL.%2BS.%26aulast%3DSong%26aufirst%3DY.%2BC.%26atitle%3D3-%2528Piperidin-4-ylmethoxy%2529pyridine%2520containing%2520compounds%2520are%2520potent%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D253%26epage%3D263%26doi%3D10.1021%2Facs.jmedchem.5b01361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of LSD1 in complex with 4-[5-(Piperidin-4-ylmethoxy)-2-(<i>p</i>-tolyl)pyridin-3-yl]benzonitrile</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1538</span>– <span class="NLM_lpage">1545</span>, <span class="refDoi"> DOI: 10.3390/molecules23071538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.3390%2Fmolecules23071538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGht7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1538-1545&author=H.+Niwaauthor=S.+Satoauthor=T.+Hashimotoauthor=K.+Matsunoauthor=T.+Umehara&title=Crystal+structure+of+LSD1+in+complex+with+4-%5B5-%28Piperidin-4-ylmethoxy%29-2-%28p-tolyl%29pyridin-3-yl%5Dbenzonitrile&doi=10.3390%2Fmolecules23071538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of LSD1 in complex with 4-[5-(piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile</span></div><div class="casAuthors">Niwa, Hideaki; Sato, Shin; Hashimoto, Tomoko; Matsuno, Kenji; Umehara, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1538/1-1538/9</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Because lysine-specific demethylase 1 (LSD1) regulates the maintenance of cancer stem cell properties, small-mol. inhibitors of LSD1 are expected to be useful for the treatment of several cancers.  Reversible inhibitors of LSD1 with submicromolar inhibitory potency have recently been reported, but their exact binding modes are poorly understood.  In this study, we synthesized a recently reported reversible inhibitor, 4-[5-(piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile, which bears a 4-piperidinylmethoxy group, a 4-methylphenyl group, and a 4-cyanophenyl group on a pyridine ring, and detd. the crystal structure of LSD1 in complex with this inhibitor at 2.96 Å.  We obsd. strong electron d. for the compd., showing that its cyano group forms a hydrogen bond with Lys661, which is a crit. residue in the lysine demethylation reaction located deep in the catalytic center of LSD1.  The piperidine ring interacts with the side chains of Asp555 and Asn540 in two conformations, and the 4-methylphenyl group is bound in a hydrophobic pocket in the catalytic center.  Our elucidation of the binding mode of this compd. can be expected to facilitate the rational design of more-potent reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5WNfouAObSLVg90H21EOLACvtfcHk0lh47jkBDwwqXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGht7zF&md5=774ef15eacf2facce6aa485a877b31a9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23071538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23071538%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DMatsuno%26aufirst%3DK.%26aulast%3DUmehara%26aufirst%3DT.%26atitle%3DCrystal%2520structure%2520of%2520LSD1%2520in%2520complex%2520with%25204-%255B5-%2528Piperidin-4-ylmethoxy%2529-2-%2528p-tolyl%2529pyridin-3-yl%255Dbenzonitrile%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1538%26epage%3D1545%26doi%3D10.3390%2Fmolecules23071538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and structure-activity relationship of novel LSD1 inhibitors based on pyrimidine-thiourea hybrids as potent, orally active antitumor agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1705</span>– <span class="NLM_lpage">1716</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlyksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1705-1716&author=L.+Y.+Maauthor=Y.+C.+Zhengauthor=S.+Q.+Wangauthor=B.+Wangauthor=Z.+R.+Wangauthor=L.+P.+Pangauthor=M.+Zhangauthor=J.+W.+Wangauthor=L.+Dingauthor=J.+Liauthor=C.+Wangauthor=B.+Huauthor=Y.+Liuauthor=X.+D.+Zhangauthor=J.+J.+Wangauthor=Z.+J.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=Design%2C+synthesis%2C+and+structure-activity+relationship+of+novel+LSD1+inhibitors+based+on+pyrimidine-thiourea+hybrids+as+potent%2C+orally+active+antitumor+agents&doi=10.1021%2Facs.jmedchem.5b00037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Structure-Activity Relationship of Novel LSD1 Inhibitors Based on Pyrimidine-Thiourea Hybrids As Potent, Orally Active Antitumor Agents</span></div><div class="casAuthors">Ma, Li-Ying; Zheng, Yi-Chao; Wang, Sai-Qi; Wang, Bo; Wang, Zhi-Ru; Pang, Lu-Ping; Zhang, Miao; Wang, Jun-Wei; Ding, Lina; Li, Juan; Wang, Cong; Hu, Biao; Liu, Ying; Zhang, Xiao-Dan; Wang, Jia-Jia; Wang, Zhi-Jian; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1705-1716</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) was reported to be overexpressed in several human cancers and recognized as a promising anticancer drug target.  In the current study, we designed and synthesized a novel series of pyrimidine-thiourea hybrids and evaluated their potential LSD1 inhibitory effect.  One of the compds., 6b, contg. a terminal alkyne appendage, was shown to be the most potent and selective LSD1 inhibitor in vitro and exhibited strong cytotoxicity against LSD1 overexpressed gastric cancer cells.  Compd. 6b also showed marked inhibition of cell migration and invasion as well as significant in vivo tumor suppressing and antimetastasis role, without significant side effects by oral administration.  Our findings indicate that the pyrimidine-thiourea-based LSD1 inactivator may serve as a leading compd. targeting LSD1 overexpressed cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy8SIAhdqGGbVg90H21EOLACvtfcHk0lh47jkBDwwqXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlyksbw%253D&md5=7256792e23b75e5f26bac47dfa773b11</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00037%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DS.%2BQ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DPang%26aufirst%3DL.%2BP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%2BD.%26aulast%3DWang%26aufirst%3DJ.%2BJ.%26aulast%3DWang%26aufirst%3DZ.%2BJ.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520structure-activity%2520relationship%2520of%2520novel%2520LSD1%2520inhibitors%2520based%2520on%2520pyrimidine-thiourea%2520hybrids%2520as%2520potent%252C%2520orally%2520active%2520antitumor%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1705%26epage%3D1716%26doi%3D10.1021%2Facs.jmedchem.5b00037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Identification of osimertinib (AZD9291) as a lysine specific demethylase 1 inhibitor</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.11.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2018.11.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30502627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGjsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2019&pages=164-169&author=Z.+R.+Liauthor=F.+Z.+Suoauthor=B.+Huauthor=Y.+J.+Guoauthor=D.+J.+Fuauthor=B.+Yuauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Identification+of+osimertinib+%28AZD9291%29+as+a+lysine+specific+demethylase+1+inhibitor&doi=10.1016%2Fj.bioorg.2018.11.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of osimertinib (AZD9291) as a lysine specific demethylase 1 inhibitor</span></div><div class="casAuthors">Li, Zhong-Rui; Suo, Feng-Zhi; Hu, Bo; Guo, Yan-Jia; Fu, Dong-Jun; Yu, Bin; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">164-169</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Osimertinib (AZD9291) is a third-generation epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor (TKI) that has been approved for the treatment of EGFR-mutated non-small cell lung cancer (NSCLC).  In this study, osimertinib was characterized as a LSD1 inhibitor for the first time with an IC50 of 3.98 ± 0.3 μM and showed LSD1 inhibitory effect at cellular level.  These findings provide new mol. skeleton for dual inhibitor for LSD1 and EGFR.  Osimertinib could serve as a lead compd. for further development for anti-NSCLC drug discovery with dual targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxIcxf303_PLVg90H21EOLACvtfcHk0lh47jkBDwwqXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGjsLzK&md5=88eae871919618a7a03a57d01096fd87</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.11.018%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DGuo%26aufirst%3DY.%2BJ.%26aulast%3DFu%26aufirst%3DD.%2BJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DIdentification%2520of%2520osimertinib%2520%2528AZD9291%2529%2520as%2520a%2520lysine%2520specific%2520demethylase%25201%2520inhibitor%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D84%26spage%3D164%26epage%3D169%26doi%3D10.1016%2Fj.bioorg.2018.11.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonanno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zulato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attili, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Bianco, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zago, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frega, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amadori, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarneri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conte, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indraccolo, S.</span></span> <span> </span><span class="NLM_article-title">Liquid biopsy as tool to monitor and predict clinical benefit from chemotherapy (CT) and immunotherapy (IT) in advanced non-small cell lung cancer (aNSCLC): A prospective study</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">649</span>– <span class="NLM_lpage">669</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdy303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1093%2Fannonc%2Fmdy303" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=649-669&author=L.+Bonannoauthor=E.+Zulatoauthor=I.+Attiliauthor=A.+Pavanauthor=P.+Del+Biancoauthor=G.+Nardoauthor=M.+Verzaauthor=L.+Pasqualiniauthor=G.+Paselloauthor=G.+Zagoauthor=S.+Fregaauthor=M.+Fassanauthor=F.+Calabreseauthor=A.+Amadoriauthor=V.+Guarneriauthor=P.+F.+Conteauthor=S.+Indraccolo&title=Liquid+biopsy+as+tool+to+monitor+and+predict+clinical+benefit+from+chemotherapy+%28CT%29+and+immunotherapy+%28IT%29+in+advanced+non-small+cell+lung+cancer+%28aNSCLC%29%3A+A+prospective+study&doi=10.1093%2Fannonc%2Fmdy303"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdy303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdy303%26sid%3Dliteratum%253Aachs%26aulast%3DBonanno%26aufirst%3DL.%26aulast%3DZulato%26aufirst%3DE.%26aulast%3DAttili%26aufirst%3DI.%26aulast%3DPavan%26aufirst%3DA.%26aulast%3DDel%2BBianco%26aufirst%3DP.%26aulast%3DNardo%26aufirst%3DG.%26aulast%3DVerza%26aufirst%3DM.%26aulast%3DPasqualini%26aufirst%3DL.%26aulast%3DPasello%26aufirst%3DG.%26aulast%3DZago%26aufirst%3DG.%26aulast%3DFrega%26aufirst%3DS.%26aulast%3DFassan%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DF.%26aulast%3DAmadori%26aufirst%3DA.%26aulast%3DGuarneri%26aufirst%3DV.%26aulast%3DConte%26aufirst%3DP.%2BF.%26aulast%3DIndraccolo%26aufirst%3DS.%26atitle%3DLiquid%2520biopsy%2520as%2520tool%2520to%2520monitor%2520and%2520predict%2520clinical%2520benefit%2520from%2520chemotherapy%2520%2528CT%2529%2520and%2520immunotherapy%2520%2528IT%2529%2520in%2520advanced%2520non-small%2520cell%2520lung%2520cancer%2520%2528aNSCLC%2529%253A%2520A%2520prospective%2520study%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2018%26volume%3D29%26spage%3D649%26epage%3D669%26doi%3D10.1093%2Fannonc%2Fmdy303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollebecque, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isambert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Romano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baudin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filvaroff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolova, Z.</span></span> <span> </span><span class="NLM_article-title">Phase I study of CC-90011 in patients with advanced solid tumors and relapsed/refractory non-Hodgkin lymphoma (R/R NHL)</span>. <i>Annals of Oncology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">I4</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdz029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1093%2Fannonc%2Fmdz029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=I4&author=A.+Hollebecqueauthor=J.+S.+de+Bonoauthor=R.+Plummerauthor=N.+Isambertauthor=P.+Martin-Romanoauthor=E.+Baudinauthor=S.+Moraauthor=E.+Filvaroffauthor=M.+Lambaauthor=Z.+Nikolova&title=Phase+I+study+of+CC-90011+in+patients+with+advanced+solid+tumors+and+relapsed%2Frefractory+non-Hodgkin+lymphoma+%28R%2FR+NHL%29&doi=10.1093%2Fannonc%2Fmdz029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdz029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdz029%26sid%3Dliteratum%253Aachs%26aulast%3DHollebecque%26aufirst%3DA.%26aulast%3Dde%2BBono%26aufirst%3DJ.%2BS.%26aulast%3DPlummer%26aufirst%3DR.%26aulast%3DIsambert%26aufirst%3DN.%26aulast%3DMartin-Romano%26aufirst%3DP.%26aulast%3DBaudin%26aufirst%3DE.%26aulast%3DMora%26aufirst%3DS.%26aulast%3DFilvaroff%26aufirst%3DE.%26aulast%3DLamba%26aufirst%3DM.%26aulast%3DNikolova%26aufirst%3DZ.%26atitle%3DPhase%2520I%2520study%2520of%2520CC-90011%2520in%2520patients%2520with%2520advanced%2520solid%2520tumors%2520and%2520relapsed%252Frefractory%2520non-Hodgkin%2520lymphoma%2520%2528R%252FR%2520NHL%2529%26jtitle%3DAnnals%2520of%2520Oncology%26date%3D2019%26volume%3D30%26spage%3DI4%26doi%3D10.1093%2Fannonc%2Fmdz029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, N. Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stansfield, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawton, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosfield, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreilein, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares-Greco, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veal, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of CC-90011: a potent and selective reversible inhibitor of lysine specific demethylase 1 (LSD1)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14522</span>– <span class="NLM_lpage">15529</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00978</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00978" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVSqsr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14522-15529&author=T.+Kanouniauthor=C.+Severinauthor=R.+W.+Choauthor=N.+Y.+Y.+Yuenauthor=J.+Xuauthor=L.+Shiauthor=C.+Laiauthor=J.+R.+Del+Rosarioauthor=R.+K.+Stansfieldauthor=L.+N.+Lawtonauthor=D.+Hosfieldauthor=S.+O%E2%80%99Connellauthor=M.+M.+Kreileinauthor=P.+Tavares-Grecoauthor=Z.+Nieauthor=S.+W.+Kaldorauthor=J.+M.+Vealauthor=J.+A.+Staffordauthor=Y.+K.+Chen&title=Discovery+of+CC-90011%3A+a+potent+and+selective+reversible+inhibitor+of+lysine+specific+demethylase+1+%28LSD1%29&doi=10.1021%2Facs.jmedchem.0c00978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CC-90011: A Potent and Selective Reversible Inhibitor of Lysine Specific Demethylase 1 (LSD1)</span></div><div class="casAuthors">Kanouni, Toufike; Severin, Christophe; Cho, Robert W.; Yuen, Natalie Y.-Y.; Xu, Jiangchun; Shi, Lihong; Lai, Chon; Del Rosario, Joselyn R.; Stansfield, Ryan K.; Lawton, Lee N.; Hosfield, David; O'Connell, Shawn; Kreilein, Matt M.; Tavares-Greco, Paula; Nie, Zhe; Kaldor, Stephen W.; Veal, James M.; Stafford, Jeffrey A.; Chen, Young K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14522-14529</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone demethylase LSDl (KDMlA) belongs to the FAD (FAD) dependent family of monoamine oxidases and is vital in regulation of mammalian biol.  Dysregulation and overexpression of LSD1 are hallmarks of a no. of human diseases, particularly cancers that are characterized as morphol. poorly differentiated.  As such, inhibitors of LSD1 have potential to be beneficial as a cancer therapy.  The most clin. advanced inhibitors of LSDl are covalent inhibitors derived from tranylcypromine (TCP).  Herein, we report the discovery of a novel series of reversible and selective LSDl inhibitors.  Exploration of structure-activity relationships (SARs) and optimization of ADME properties resulted in the identification of clin. candidate CC-90011.  CC-90011 exhibits potent on-target induction of cellular differentiation in acute myeloid leukemia (AML) and small cell lung cancer (SCLC) cell lines, and antitumor efficacy in patient-derived xenograft (PDX) SCLC models.  CC-90011 is currently in phase 2 trials in patients with first line, extensive stage SCLC (ClinicalTrials.gov identifier: NCT03850067).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6pM7y3zzSNbVg90H21EOLACvtfcHk0lijiawIqv_O3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVSqsr%252FF&md5=cd3fdea62cf5448237fa341747c59884</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00978%26sid%3Dliteratum%253Aachs%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DSeverin%26aufirst%3DC.%26aulast%3DCho%26aufirst%3DR.%2BW.%26aulast%3DYuen%26aufirst%3DN.%2BY.%2BY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DDel%2BRosario%26aufirst%3DJ.%2BR.%26aulast%3DStansfield%26aufirst%3DR.%2BK.%26aulast%3DLawton%26aufirst%3DL.%2BN.%26aulast%3DHosfield%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DS.%26aulast%3DKreilein%26aufirst%3DM.%2BM.%26aulast%3DTavares-Greco%26aufirst%3DP.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DKaldor%26aufirst%3DS.%2BW.%26aulast%3DVeal%26aufirst%3DJ.%2BM.%26aulast%3DStafford%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DY.%2BK.%26atitle%3DDiscovery%2520of%2520CC-90011%253A%2520a%2520potent%2520and%2520selective%2520reversible%2520inhibitor%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14522%26epage%3D15529%26doi%3D10.1021%2Facs.jmedchem.0c00978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niwa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umehara, T.</span></span> <span> </span><span class="NLM_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1080/15592294.2017.1290032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1080%2F15592294.2017.1290032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28277979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=340-352&author=H.+Niwaauthor=T.+Umehara&title=Structural+insight+into+inhibitors+of+flavin+adenine+dinucleotide-dependent+lysine+demethylases&doi=10.1080%2F15592294.2017.1290032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases</span></div><div class="casAuthors">Niwa Hideaki; Umehara Takashi; Umehara Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-352</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Until 2004, many researchers believed that protein methylation in eukaryotic cells was an irreversible reaction.  However, the discovery of lysine-specific demethylase 1 in 2004 drastically changed this view and the concept of chromatin regulation.  Since then, the enzymes responsible for lysine demethylation and their cellular substrates, biological significance, and selective regulation have become major research topics in epigenetics and chromatin biology.  Many cell-permeable inhibitors for lysine demethylases have been developed, including both target-specific and nonspecific inhibitors.  Structural understanding of how these inhibitors bind to lysine demethylases is crucial both for validation of the inhibitors as chemical probes and for the rational design of more potent, target-specific inhibitors.  This review focuses on published small-molecule inhibitors targeted at the two flavin adenine dinucleotide-dependent lysine demethylases, lysine-specific demethylases 1 and 2, and how the inhibitors interact with the tertiary structures of the enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyKG85Hfra9xjv2lLSLOxFfW6udTcc2eadLGKAAWPNLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czks1GitA%253D%253D&md5=31f93c2cc7909c4197f5892f7f04af9d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1080%2F15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DUmehara%26aufirst%3DT.%26atitle%3DStructural%2520insight%2520into%2520inhibitors%2520of%2520flavin%2520adenine%2520dinucleotide-dependent%2520lysine%2520demethylases%26jtitle%3DEpigenetics%26date%3D2017%26volume%3D12%26spage%3D340%26epage%3D352%26doi%3D10.1080%2F15592294.2017.1290032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e25444</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0025444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1371%2Fjournal.pone.0025444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=21984927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=X.+Kongauthor=S.+Ouyangauthor=Z.+Liangauthor=J.+Luauthor=L.+Chenauthor=B.+Shenauthor=D.+Liauthor=M.+Zhengauthor=K.+K.+Liauthor=C.+Luoauthor=H.+Jiang&title=Catalytic+mechanism+investigation+of+lysine-specific+demethylase+1+%28LSD1%29%3A+a+computational+study&doi=10.1371%2Fjournal.pone.0025444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic mechanism investigation of lysine-specific demethylase 1 (LSD1): a computational study</span></div><div class="casAuthors">Kong, Xiangqian; Ouyang, Sisheng; Liang, Zhongjie; Lu, Junyan; Chen, Liang; Shen, Bairong; Li, Donghai; Zheng, Mingyue; Li, Keqin Kathy; Luo, Cheng; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25444</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), the first identified histone demethylase, is a flavin-dependent amine oxidase which specifically demethylates mono- or di-methylated H3K4 and H3K9 via a redox process.  It participates in a broad spectrum of biol. processes and is of high importance in cell proliferation, adipogenesis, spermatogenesis, chromosome segregation and embryonic development.  To date, as a potential drug target for discovering anti-tumor drugs, the medical significance of LSD1 has been greatly appreciated.  However, the catalytic mechanism for the rate-limiting reductive half-reaction in demethylation remains controversial.  By employing a combined computational approach including mol. modeling, mol. dynamics (MD) simulations and quantum mechanics/mol. mechanics (QM/MM) calcns., the catalytic mechanism of dimethylated H3K4 demethylation by LSD1 was characterized in details.  The three-dimensional (3D) model of the complex was composed of LSD1, CoREST, and histone substrate.  A 30-ns MD simulation of the model highlights the pivotal role of the conserved Tyr761 and lysine-water-flavin motif in properly orienting FAD with respect to substrate.  The synergy of the two factors effectively stabilizes the catalytic environment and facilitated the demethylation reaction.  On the basis of the reasonable consistence between simulation results and available mutagenesis data, QM/MM strategy was further employed to probe the catalytic mechanism of the reductive half-reaction in demethylation.  The characteristics of the demethylation pathway detd. by the potential energy surface and charge distribution anal. indicates that this reaction belongs to the direct hydride transfer mechanism.  Our study provides insights into the LSD1 mechanism of reductive half-reaction in demethylation and has important implications for the discovery of regulators against LSD1 enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOIm_wJ2koFrVg90H21EOLACvtfcHk0lhq503ZLfd7Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlGmt7rP&md5=7ccdcdcf6e8770ea0d77c1cc80f79da8</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025444%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DCatalytic%2520mechanism%2520investigation%2520of%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%253A%2520a%2520computational%2520study%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0025444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Exploration of 5-cyano-6-phenylpyrimidin derivatives containing an 1,2,3-triazole moiety as potent FAD-based LSD1 inhibitors</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1658</span>– <span class="NLM_lpage">1668</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.apsb.2020.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=33088686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnslSrsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=1658-1668&author=L.+Y.+Maauthor=H.+J.+Wangauthor=Y.+H.+Youauthor=C.+Y.+Maauthor=Y.+J.+Liuauthor=F.+F.+Yangauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Exploration+of+5-cyano-6-phenylpyrimidin+derivatives+containing+an+1%2C2%2C3-triazole+moiety+as+potent+FAD-based+LSD1+inhibitors&doi=10.1016%2Fj.apsb.2020.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of 5-cyano-6-phenylpyrimidin derivatives containing an 1,2,3-triazole moiety as potent FAD-based LSD1 inhibitors</span></div><div class="casAuthors">Ma, Liying; Wang, Haojie; You, Yinghua; Ma, Chaoya; Liu, Yuejiao; Yang, Feifei; Zheng, Yichao; Liu, Hongmin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Pharmaceutica Sinica B</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1658-1668</span>CODEN:
                <span class="NLM_cas:coden">APSBCW</span>;
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) has become a potential therapeutic target for the treatment of cancer.  Discovery and develop novel and potent LSD1 inhibitors is a challenge, although several of them have already entered into clin. trials.  Herein, for the first time, we reported the discovery of a series of 5-cyano-6-phenylpyrimidine derivs. as LSD1 inhibitors using FAD (FAD) similarity-based designing strategy, of which compd. 14q was finally identified to repress LSD1 with IC50 = 183 nmol/L.  Docking anal. suggested that compd. 14q fitted well into the FAD-binding pocket.  Further mechanism studies showed that compd. 14q may inhibit LSD1 activity competitively by occupying the FAD binding sites of LSD1 and inhibit cell migration and invasion by reversing epithelial to mesenchymal transition (EMT).  Overall, these findings showed that compd. 14q is a suitable candidate for further development of novel FAD similarity-based LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGalgCfnOafrVg90H21EOLACvtfcHk0lhq503ZLfd7Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnslSrsg%253D%253D&md5=fb61dc6d62dd0920a3dfa005ddfdf7b2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DWang%26aufirst%3DH.%2BJ.%26aulast%3DYou%26aufirst%3DY.%2BH.%26aulast%3DMa%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DY.%2BJ.%26aulast%3DYang%26aufirst%3DF.%2BF.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DExploration%2520of%25205-cyano-6-phenylpyrimidin%2520derivatives%2520containing%2520an%25201%252C2%252C3-triazole%2520moiety%2520as%2520potent%2520FAD-based%2520LSD1%2520inhibitors%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26volume%3D10%26spage%3D1658%26epage%3D1668%26doi%3D10.1016%2Fj.apsb.2020.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kutz, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holshouser, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrow, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1863</span>– <span class="NLM_lpage">1870</span>, <span class="refDoi"> DOI: 10.1039/C4MD00283K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2FC4MD00283K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25580204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1863-1870&author=C.+J.+Kutzauthor=S.+L.+Holshouserauthor=E.+A.+Marrowauthor=P.+M.+Woster&title=3%2C5-Diamino-1%2C2%2C4-triazoles+as+a+novel+scaffold+for+potent%2C+reversible+LSD1+%28KDM1A%29+inhibitors&doi=10.1039%2FC4MD00283K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span></div><div class="casAuthors">Kutz, Craig J.; Holshouser, Steven L.; Marrow, Ethan A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1863-1870</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The chromatin remodeling amine oxidase lysine-specific demethylase 1 (LSD1) has become an attractive target for the design of specific inhibitors with therapeutic potential.  We, and others, have described LSD1 inhibitors that have potential as antitumor agents.  Many of the currently known LSD1 inhibitors are poor drug candidates, or are structurally based on the tranylcypromine backbone, thus increasing the potential for off-target effects mediated by other amine oxidases.  We now describe a series of potent LSD1 inhibitors based on a novel 1,2,4-triazole scaffold; these inhibitors show a high degree of specificity for LSD1 in vitro, and cause increases in cellular histone 3 dimethyllysine 4 (H3K4me2), a gene transcription activating mark.  Importantly, these inhibitors exhibit low toxicity towards mammalian cells in vitro, and thus they may show utility in the treatment of epigenetically-based diseases where cell death is not a desired endpoint.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXpwx29ujiWbVg90H21EOLACvtfcHk0lhq503ZLfd7Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N&md5=3a6314468bdb665cb30ae86505e5eb15</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1039%2FC4MD00283K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00283K%26sid%3Dliteratum%253Aachs%26aulast%3DKutz%26aufirst%3DC.%2BJ.%26aulast%3DHolshouser%26aufirst%3DS.%2BL.%26aulast%3DMarrow%26aufirst%3DE.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D3%252C5-Diamino-1%252C2%252C4-triazoles%2520as%2520a%2520novel%2520scaffold%2520for%2520potent%252C%2520reversible%2520LSD1%2520%2528KDM1A%2529%2520inhibitors%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1863%26epage%3D1870%26doi%3D10.1039%2FC4MD00283K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holshouser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunworth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray-Stewart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Dual inhibitors of LSD1 and spermine oxidase</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">778</span>– <span class="NLM_lpage">790</span>, <span class="refDoi"> DOI: 10.1039/C8MD00610E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2FC8MD00610E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31191868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXis1Gqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=778-790&author=S.+Holshouserauthor=M.+Dunworthauthor=T.+Murray-Stewartauthor=Y.+K.+Petersonauthor=P.+Burgerauthor=J.+Kirkpatrickauthor=H.+H.+Chenauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=Dual+inhibitors+of+LSD1+and+spermine+oxidase&doi=10.1039%2FC8MD00610E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Dual inhibitors of LSD1 and spermine oxidase</span></div><div class="casAuthors">Holshouser, Steven; Dunworth, Matthew; Murray-Stewart, Tracy; Peterson, Yuri K.; Burger, Pieter; Kirkpatrick, Joy; Chen, Huan-Huan; Casero, Robert A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">778-790</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We have previously described the synthesis and evaluation of 3,5-diamino-1,2,4-triazole analogs as inhibitors of the flavin-dependent histone demethylase LSD1.  These compds. are potent inhibitors of LSD1 without activity against monoamine oxidases A and B, and promote the elevation of H3K4me2 levels in tumor cells in vitro.  We now report that the cytotoxicity of these analogs in pancreatic tumor cells correlates with the overexpression of LSD1 in each tumor type.  In addn., we show that a subset of these 3,5-diamino-1,2,4-triazole analogs inhibit a related flavin-dependent oxidase, the polyamine catabolic enzyme spermine oxidase (SMOX) in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruCgjp3cbY2rVg90H21EOLACvtfcHk0lhY3AeffRwXhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXis1Gqs70%253D&md5=b81307aef3440a1dc7ad431a33b6ca89</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2FC8MD00610E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8MD00610E%26sid%3Dliteratum%253Aachs%26aulast%3DHolshouser%26aufirst%3DS.%26aulast%3DDunworth%26aufirst%3DM.%26aulast%3DMurray-Stewart%26aufirst%3DT.%26aulast%3DPeterson%26aufirst%3DY.%2BK.%26aulast%3DBurger%26aufirst%3DP.%26aulast%3DKirkpatrick%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%2BH.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DDual%2520inhibitors%2520of%2520LSD1%2520and%2520spermine%2520oxidase%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D778%26epage%3D790%26doi%3D10.1039%2FC8MD00610E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mobley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Triazole-dithiocarbamate based selective lysine specific demethylase 1 (LSD1) inactivators inhibit gastric cancer cell growth, invasion, and migration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8543</span>– <span class="NLM_lpage">8560</span>, <span class="refDoi"> DOI: 10.1021/jm401002r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401002r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8543-8560&author=Y.+C.+Zhengauthor=Y.+C.+Duanauthor=J.+L.+Maauthor=R.+M.+Xuauthor=X.+Ziauthor=W.+L.+Lvauthor=M.+M.+Wangauthor=X.+W.+Yeauthor=S.+Zhuauthor=D.+Mobleyauthor=Y.+Y.+Zhuauthor=J.+W.+Wangauthor=J.+F.+Liauthor=Z.+R.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=Triazole-dithiocarbamate+based+selective+lysine+specific+demethylase+1+%28LSD1%29+inactivators+inhibit+gastric+cancer+cell+growth%2C+invasion%2C+and+migration&doi=10.1021%2Fjm401002r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Triazole-Dithiocarbamate Based Selective Lysine Specific Demethylase 1 (LSD1) Inactivators Inhibit Gastric Cancer Cell Growth, Invasion, and Migration</span></div><div class="casAuthors">Zheng, Yi-Chao; Duan, Ying-Chao; Ma, Jin-Lian; Xu, Rui-Min; Zi, Xiaolin; Lv, Wen-Lei; Wang, Meng-Meng; Ye, Xian-Wei; Zhu, Shun; Mobley, David; Zhu, Yan-Yan; Wang, Jun-Wei; Li, Jin-Feng; Wang, Zhi-Ru; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8543-8560</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1), the first identified histone demethylase, plays an important role in epigenetic regulation of gene activation and repression.  The up-regulated LSD1's expression has been reported in several malignant tumors.  In the current study, we designed and synthesized five series of 1,2,3-triazole-dithiocarbamate hybrids and screened their inhibitory activity toward LSD1.  We found that some of these compds., esp. compd. (I), exhibited the most specific and robust inhibition of LSD1.  Interestingly, compd. I also showed potent and selective cytotoxicity against LSD1 overexpressing gastric cancer cell lines MGC-803 and HGC-27, as well as marked inhibition of cell migration and invasion, compared to 2-PCPA.  Furthermore, compd. I effectively reduced the tumor growth bared by human gastric cancer cells in vivo with no signs of adverse side effects.  These findings suggested that compd. I deserves further investigation as a lead compd. in the treatment of LSD1 overexpressing gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjGdOvultbebVg90H21EOLACvtfcHk0lhY3AeffRwXhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL&md5=4e4594271ff6b62237ff17a87d6f04d8</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm401002r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401002r%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DZi%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DM.%2BM.%26aulast%3DYe%26aufirst%3DX.%2BW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DMobley%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%2BY.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DLi%26aufirst%3DJ.%2BF.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTriazole-dithiocarbamate%2520based%2520selective%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520inactivators%2520inhibit%2520gastric%2520cancer%2520cell%2520growth%252C%2520invasion%252C%2520and%2520migration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8543%26epage%3D8560%26doi%3D10.1021%2Fjm401002r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietrancosta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laras, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbette, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maigret, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quéléver, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatsubo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, J. L.</span></span> <span> </span><span class="NLM_article-title">Naphthyl and coumarinyl biarylpiperazine derivatives as highly potent human beta-secretase inhibitors. Design, synthesis, and enzymatic BACE-1 and cell assays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4275</span>– <span class="NLM_lpage">4285</span>, <span class="refDoi"> DOI: 10.1021/jm0602864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0602864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFent70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=4275-4285&author=C.+Garinoauthor=T.+Tomitaauthor=N.+Pietrancostaauthor=Y.+Larasauthor=R.+Rosasauthor=G.+Herbetteauthor=B.+Maigretauthor=G.+Qu%C3%A9l%C3%A9verauthor=T.+Iwatsuboauthor=J.+L.+Kraus&title=Naphthyl+and+coumarinyl+biarylpiperazine+derivatives+as+highly+potent+human+beta-secretase+inhibitors.+Design%2C+synthesis%2C+and+enzymatic+BACE-1+and+cell+assays&doi=10.1021%2Fjm0602864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Naphthyl and coumarinyl biarylpiperazine derivatives as highly potent human β-secretase inhibitors. design, synthesis, and enzymatic BACE-1 and cell assays</span></div><div class="casAuthors">Garino, Cedrik; Tomita, Taisuke; Pietrancosta, Nicolas; Laras, Younes; Rosas, Roselyne; Herbette, Gaetan; Maigret, Bernard; Quelever, Gilles; Iwatsubo, Takeshi; Kraus, Jean-Louis</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4275-4285</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Twenty novel β-secretase inhibitors contg. biarylpiperazine moieties belonging to naphthyl and coumarinyl series were designed for their potential use in Alzheimer's disease therapy.  Enzymic and cell-based assays have been carried out.  The biol. results clearly demonstrate that specific substituents located at the N4-position of the piperazine ring result in excellent in vitro inhibitory potency (IC50 values ranging between 40 and 70 nM).  Variable temp. NMR and modeling studies are consistent with the obtained biol. data, since these studies confirmed that introduction at the N4-position of the piperazine ring allows productive interactions within the BACE-1 active site, which appear to be determinative for high BACE-1 inhibitory activity.  These results are of particular interest since some of the new analogs belonging to the naphthyl series are almost one log more active than the best inhibitor of the similar family recently reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUbFH1mOoKlrVg90H21EOLACvtfcHk0lhY3AeffRwXhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFent70%253D&md5=8a29e41e8ae98ef9dab01d4e94bef291</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm0602864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0602864%26sid%3Dliteratum%253Aachs%26aulast%3DGarino%26aufirst%3DC.%26aulast%3DTomita%26aufirst%3DT.%26aulast%3DPietrancosta%26aufirst%3DN.%26aulast%3DLaras%26aufirst%3DY.%26aulast%3DRosas%26aufirst%3DR.%26aulast%3DHerbette%26aufirst%3DG.%26aulast%3DMaigret%26aufirst%3DB.%26aulast%3DQu%25C3%25A9l%25C3%25A9ver%26aufirst%3DG.%26aulast%3DIwatsubo%26aufirst%3DT.%26aulast%3DKraus%26aufirst%3DJ.%2BL.%26atitle%3DNaphthyl%2520and%2520coumarinyl%2520biarylpiperazine%2520derivatives%2520as%2520highly%2520potent%2520human%2520beta-secretase%2520inhibitors.%2520Design%252C%2520synthesis%252C%2520and%2520enzymatic%2520BACE-1%2520and%2520cell%2520assays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D4275%26epage%3D4285%26doi%3D10.1021%2Fjm0602864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of coumarin-1,2,3-triazole–dithiocarbamate hybrids as potent LSD1 inhibitors</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1039/C4MD00031E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2FC4MD00031E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2cXms1Wqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=650-654&author=X.+W.+Yeauthor=Y.+C.+Zhengauthor=Y.+C.+Duanauthor=M.+M.+Wangauthor=B.+Yuauthor=J.+L.+Renauthor=J.+L.+Maauthor=E.+Zhangauthor=H.+M.+Liu&title=Synthesis+and+biological+evaluation+of+coumarin-1%2C2%2C3-triazole%E2%80%93dithiocarbamate+hybrids+as+potent+LSD1+inhibitors&doi=10.1039%2FC4MD00031E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of coumarin-1,2,3-triazole-dithiocarbamate hybrids as potent LSD1 inhibitors</span></div><div class="casAuthors">Ye, Xian-Wei; Zheng, Yi-Chao; Duan, Ying-Chao; Wang, Meng-Meng; Yu, Bin; Ren, Jing-Li; Ma, Jin-Lian; Zhang, En; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">650-654</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of coumarin-1,2,3-triazole-dithiocarbamate hybrids I [R1 = 7-Cl, 6-Cl, 7-NH2, 5,7-(HO)2, etc.] and II (R2 = H, 7-OH) were designed, synthesized and evaluated for their inhibitory activity towards lysine specific demethylase 1 (LSD1).  Compds. I [R1 = 7-Cl, H, 7-NH2, 7-CH3, 5,7-(OH)2, 7,8-(OH)2, 7-OH, 7-OCH3] presented potent activity against lysine specific demethylase 1.  Among them, compd. I (R1 = 7-OH) (III) showed potent and reversible inhibition against lysine specific demethylase 1 with an IC50 value of 0.39 μM, which was 74-fold more potent than that of tranylcypromine (2-PCPA).  Besides, compd. III displayed excellent selectivity against lysine specific demethylase 1 without inhibition against monoamine oxidases (MAOs) A and B.  Further investigation revealed that compd. III was active at both recombinant and cell levels by upregulating the expression of H3K4me1, H3K4me2 and H3K9me2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4HUPEXzC-R7Vg90H21EOLACvtfcHk0li-4pZVQLSgJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXms1Wqs78%253D&md5=c29af9f1f1c649032d4bec605bd92e8e</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1039%2FC4MD00031E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00031E%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DX.%2BW.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DWang%26aufirst%3DM.%2BM.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DRen%26aufirst%3DJ.%2BL.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520coumarin-1%252C2%252C3-triazole%25E2%2580%2593dithiocarbamate%2520hybrids%2520as%2520potent%2520LSD1%2520inhibitors%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D650%26epage%3D654%26doi%3D10.1039%2FC4MD00031E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dulla, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirla, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathore, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deora, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavela, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddika, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatti, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of 3-amino/guanidine substituted phenyl oxazoles as a novel class of LSD1 inhibitors with anti-proliferative properties</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">3103</span>– <span class="NLM_lpage">3107</span>, <span class="refDoi"> DOI: 10.1039/c3ob40217g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1039%2Fc3ob40217g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23575971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtlCgtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=3103-3107&author=B.+Dullaauthor=K.+T.+Kirlaauthor=V.+Rathoreauthor=G.+S.+Deoraauthor=S.+Kavelaauthor=S.+Maddikaauthor=K.+Chattiauthor=O.+Reiserauthor=J.+Iqbalauthor=M.+Pal&title=Synthesis+and+evaluation+of+3-amino%2Fguanidine+substituted+phenyl+oxazoles+as+a+novel+class+of+LSD1+inhibitors+with+anti-proliferative+properties&doi=10.1039%2Fc3ob40217g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of 3-amino/guanidine substituted phenyl oxazoles as a novel class of LSD1 inhibitors with anti-proliferative properties</span></div><div class="casAuthors">Dulla, Balakrishna; Kirla, Krishna Tulasi; Rathore, Vandana; Deora, Girdhar Singh; Kavela, Sridhar; Maddika, Subbareddy; Chatti, Kiranam; Reiser, Oliver; Iqbal, Javed; Pal, Manojit</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3103-3107</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A series of functionalized Ph oxazole derivs. was designed, synthesized and screened in vitro for their activities against LSD1 and for effects on viability of cervical and breast cancer cells, and in vivo for effects using zebrafish embryos.  These compds. are likely to act via multiple epigenetic mechanisms specific to cancer cells including LSD1 inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNO6bAB1J-vLVg90H21EOLACvtfcHk0li-4pZVQLSgJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtlCgtLo%253D&md5=25274d1a41ee948934915c9ca004a884</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2Fc3ob40217g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob40217g%26sid%3Dliteratum%253Aachs%26aulast%3DDulla%26aufirst%3DB.%26aulast%3DKirla%26aufirst%3DK.%2BT.%26aulast%3DRathore%26aufirst%3DV.%26aulast%3DDeora%26aufirst%3DG.%2BS.%26aulast%3DKavela%26aufirst%3DS.%26aulast%3DMaddika%26aufirst%3DS.%26aulast%3DChatti%26aufirst%3DK.%26aulast%3DReiser%26aufirst%3DO.%26aulast%3DIqbal%26aufirst%3DJ.%26aulast%3DPal%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%25203-amino%252Fguanidine%2520substituted%2520phenyl%2520oxazoles%2520as%2520a%2520novel%2520class%2520of%2520LSD1%2520inhibitors%2520with%2520anti-proliferative%2520properties%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D3103%26epage%3D3107%26doi%3D10.1039%2Fc3ob40217g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. K.</span>; <span class="NLM_string-name">Kanouni, T.</span>; <span class="NLM_string-name">Kaldor, S. W.</span>; <span class="NLM_string-name">Stafford, J. A.</span>; <span class="NLM_string-name">Veal, J. M.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of lysine specific demethylase-1</span>. WO <span class="NLM_patent">2015168466 A1</span>, November 5, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Y.+K.+Chen&author=T.+Kanouni&author=S.+W.+Kaldor&author=J.+A.+Stafford&author=J.+M.+Veal&title=Inhibitors+of+lysine+specific+demethylase-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%2BK.%26atitle%3DInhibitors%2520of%2520lysine%2520specific%2520demethylase-1%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremberg, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D.</span></span> <span> </span><span class="NLM_article-title">Development of 5-hydroxypyrazole derivatives as reversible inhibitors of lysine specific demethylase 1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3190</span>– <span class="NLM_lpage">3195</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2017.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28545974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlWmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3190-3195&author=D.+P.+Mouldauthor=U.+Brembergauthor=A.+M.+Jordanauthor=M.+Geitmannauthor=A.+Maiques-Diazauthor=A.+E.+McGonagleauthor=H.+F.+Smallauthor=T.+C.+P.+Somervailleauthor=D.+Ogilvie&title=Development+of+5-hydroxypyrazole+derivatives+as+reversible+inhibitors+of+lysine+specific+demethylase+1&doi=10.1016%2Fj.bmcl.2017.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Development of 5-hydroxypyrazole derivatives as reversible inhibitors of lysine specific demethylase 1</span></div><div class="casAuthors">Mould, Daniel P.; Bremberg, Ulf; Jordan, Allan M.; Geitmann, Matthis; Maiques-Diaz, Alba; McGonagle, Alison E.; Small, Helen F.; Somervaille, Tim C. P.; Ogilvie, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3190-3195</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of reversible inhibitors of lysine specific demethylase 1 (LSD1) with a 5-hydroxypyrazole scaffold have been developed from compd. 7, which was identified from the patent literature.  Surface plasmon resonance (SPR) and biochem. anal. showed it to be a reversible LSD1 inhibitor with an IC50 value of 0.23 μM.  Optimization of this compd. by rational design afforded compds. with Kd values of <10 nM.  In human THP-1 cells, these compds. were found to upregulate the expression of the surrogate cellular biomarker CD86.  Compd. 11p was found to have moderate oral bioavailability in mice suggesting its potential for use as an in vivo tool compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6uEfijQ4G3rVg90H21EOLACvtfcHk0li-4pZVQLSgJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlWmt7k%253D&md5=2b4d825ef43c843b1436137a64d6caaf</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DBremberg%26aufirst%3DU.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSmall%26aufirst%3DH.%2BF.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DOgilvie%26aufirst%3DD.%26atitle%3DDevelopment%2520of%25205-hydroxypyrazole%2520derivatives%2520as%2520reversible%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3190%26epage%3D3195%26doi%3D10.1016%2Fj.bmcl.2017.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stansfield, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veal, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stafford, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design and discovery of potent and selective lysine-specific demethylase 1 (LSD1) inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2018.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30409536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFanur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=103-106&author=Z.+Nieauthor=L.+Shiauthor=C.+Laiauthor=C.+Severinauthor=J.+Xuauthor=J.+R.+Del+Rosarioauthor=R.+K.+Stansfieldauthor=R.+W.+Choauthor=T.+Kanouniauthor=J.+M.+Vealauthor=J.+A.+Staffordauthor=Y.+K.+Chen&title=Structure-based+design+and+discovery+of+potent+and+selective+lysine-specific+demethylase+1+%28LSD1%29+inhibitors&doi=10.1016%2Fj.bmcl.2018.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design and discovery of potent and selective lysine-specific demethylase 1 (LSD1) inhibitors</span></div><div class="casAuthors">Nie, Zhe; Shi, Lihong; Lai, Chon; Severin, Christophe; Xu, Jiangchun; Del Rosario, Joselyn R.; Stansfield, Ryan K.; Cho, Robert W.; Kanouni, Toufike; Veal, James M.; Stafford, Jeffrey A.; Chen, Young K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">103-106</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The histone demethylase LSD1 is a key enzyme in the epigenetic regulation of gene transcription.  Here we present our efforts to discover small mol. reversible inhibitors of LSD1 as an attractive approach to treat hematol. malignancies and certain solid tumors.  Using structure-based drug design, we designed and synthesized a novel series of heteroarom. imidazole inhibitors that demonstrate potent inhibition of the demethylase activity and low nanomolar cell-based activity.  This novel LSD1 inhibitor series was further optimized by attenuating the hERG inhibition and improving oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJSb7MdQzvUrVg90H21EOLACvtfcHk0lgxOov4CKlbTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFanur3M&md5=5185217eb3df25eebda16800bccf4419</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DSeverin%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDel%2BRosario%26aufirst%3DJ.%2BR.%26aulast%3DStansfield%26aufirst%3DR.%2BK.%26aulast%3DCho%26aufirst%3DR.%2BW.%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DVeal%26aufirst%3DJ.%2BM.%26aulast%3DStafford%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DY.%2BK.%26atitle%3DStructure-based%2520design%2520and%2520discovery%2520of%2520potent%2520and%2520selective%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D103%26epage%3D106%26doi%3D10.1016%2Fj.bmcl.2018.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremberg, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span> <span> </span><span class="NLM_article-title">Development and evaluation of 4-(pyrrolidin-3-yl)benzonitrile derivatives as inhibitors of lysine specific demethylase 1</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4755</span>– <span class="NLM_lpage">4759</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2017.08.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28927796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWgurjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=4755-4759&author=D.+P.+Mouldauthor=U.+Brembergauthor=A.+M.+Jordanauthor=M.+Geitmannauthor=A.+E.+McGonagleauthor=T.+C.+P.+Somervailleauthor=G.+J.+Spencerauthor=D.+J.+Ogilvie&title=Development+and+evaluation+of+4-%28pyrrolidin-3-yl%29benzonitrile+derivatives+as+inhibitors+of+lysine+specific+demethylase+1&doi=10.1016%2Fj.bmcl.2017.08.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of 4-(pyrrolidin-3-yl)benzonitrile derivatives as inhibitors of lysine specific demethylase 1</span></div><div class="casAuthors">Mould, Daniel P.; Bremberg, Ulf; Jordan, Allan M.; Geitmann, Matthis; McGonagle, Alison E.; Somervaille, Tim C. P.; Spencer, Gary J.; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4755-4759</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As part of our ongoing efforts to develop reversible inhibitors of LSD1, we identified a series of 4-(pyrrolidin-3-yl)benzonitrile derivs. that act as successful scaffold-hops of the literature inhibitor GSK-690.  The most active compd., I, demonstrated a Kd value of 22 nM and a biochem. IC50 of 57 nM.  In addn., this compd. displayed improved selectivity over the hERG ion channel compared to GSK-690, and no activity against the related enzymes MAO-A and B.  In human THP-1 acute myeloid leukemia cells, I was found to increase the expression of the surrogate cellular biomarker CD86.  This work further demonstrates the versatility of scaffold-hopping as a method to develop structurally diverse, potent inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriYqPorIJKgLVg90H21EOLACvtfcHk0lgxOov4CKlbTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWgurjF&md5=a2cf57c70017d8b08dd9d6c23fd5b0c4</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DBremberg%26aufirst%3DU.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%25204-%2528pyrrolidin-3-yl%2529benzonitrile%2520derivatives%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D4755%26epage%3D4759%26doi%3D10.1016%2Fj.bmcl.2017.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jandausch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forne, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imhof, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirfel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span> <span> </span><span class="NLM_article-title">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">2704</span>– <span class="NLM_lpage">2709</span>, <span class="refDoi"> DOI: 10.1002/ijc.27555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fijc.27555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=22447389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2704-2709&author=D.+Willmannauthor=S.+Limauthor=S.+Wetzelauthor=E.+Metzgerauthor=A.+Jandauschauthor=W.+Wilkauthor=M.+Jungauthor=I.+Forneauthor=A.+Imhofauthor=A.+Janzerauthor=J.+Kirfelauthor=H.+Waldmannauthor=R.+Schuleauthor=R.+Buettner&title=Impairment+of+prostate+cancer+cell+growth+by+a+selective+and+reversible+lysine-specific+demethylase+1+inhibitor&doi=10.1002%2Fijc.27555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span></div><div class="casAuthors">Willmann, Dominica; Lim, Soyoung; Wetzel, Stefan; Metzger, Eric; Jandausch, Anett; Wilk, Wolfram; Jung, Manfred; Forne, Ignasi; Imhof, Axel; Janzer, Andreas; Kirfel, Jutta; Waldmann, Herbert; Schuele, Roland; Buettner, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2704-2709</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Post-translational modifications of histones by chromatin modifying enzymes regulate chromatin structure and gene expression.  As deregulation of histone modifications contributes to cancer progression, inhibition of chromatin modifying enzymes such as histone demethylases is an attractive therapeutic strategy to impair cancer growth.  Lysine-specific demethylase 1 (LSD1) removes mono- and di-Me marks from lysine 4 or 9 of histone H3.  LSD1 in assocn. with the androgen receptor (AR) controls androgen-dependent gene expression and prostate tumor cell proliferation, thus highlighting LSD1 as a drug target.  By combining protein structure similarity clustering and in vitro screening, we identified Namoline, a γ-pyrone, as a novel, selective and reversible LSD1 inhibitor.  Namoline blocks LSD1 demethylase activity in vitro and in vivo.  Inhibition of LSD1 by Namoline leads to silencing of AR-regulated gene expression and severely impairs androgen-dependent proliferation in vitro and in vivo.  Thus, Namoline is a novel promising starting compd. for the development of therapeutics to treat androgen-dependent prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKxYi0f1zoOrVg90H21EOLACvtfcHk0lgxOov4CKlbTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D&md5=3104057823357b199b635e1a22071b2a</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1002%2Fijc.27555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27555%26sid%3Dliteratum%253Aachs%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DJandausch%26aufirst%3DA.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DForne%26aufirst%3DI.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DImpairment%2520of%2520prostate%2520cancer%2520cell%2520growth%2520by%2520a%2520selective%2520and%2520reversible%2520lysine-specific%2520demethylase%25201%2520inhibitor%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26spage%3D2704%26epage%3D2709%26doi%3D10.1002%2Fijc.27555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Baicalin, a natural LSD1 inhibitor</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2016.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2016.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27814566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOqsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=129-131&author=Y.+C.+Zhengauthor=D.+D.+Shenauthor=M.+Renauthor=X.+Q.+Liuauthor=Z.+R.+Wangauthor=Y.+Liuauthor=Q.+N.+Zhangauthor=L.+J.+Zhaoauthor=L.+J.+Zhaoauthor=J.+L.+Maauthor=B.+Yuauthor=H.+M.+Liu&title=Baicalin%2C+a+natural+LSD1+inhibitor&doi=10.1016%2Fj.bioorg.2016.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Baicalin, a natural LSD1 inhibitor</span></div><div class="casAuthors">Zheng, Yi-Chao; Shen, Dan-Dan; Ren, Meng; Liu, Xue-Qi; Wang, Zhi-Ru; Liu, Ying; Zhang, Qian-Na; Zhao, Li-Juan; Zhao, Li-Jie; Ma, Jin-Lian; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">129-131</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Baicalin is one of the active ingredients in the skullcap, with a variety of pharmacol. effects, such as blood pressure redn., sedation, liver-protection, gallbladder-protection, antibacteria, and anti-inflammation.  In the study, baicalin was first characterized as a LSD1 inhibitor with an IC50 of 3.01 μM and showed strong LSD1 inhibitory effect in cells.  Baicalin may serve as a template for designing flavone-based LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsRTkjAeKcWbVg90H21EOLACvtfcHk0ljxEgBJ34zPyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOqsbnK&md5=0fef2e4a12b491f203d24f4a23823b80</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2016.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2016.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DRen%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BQ.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DQ.%2BN.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DBaicalin%252C%2520a%2520natural%2520LSD1%2520inhibitor%26jtitle%3DBioorg.%2520Chem.%26date%3D2016%26volume%3D69%26spage%3D129%26epage%3D131%26doi%3D10.1016%2Fj.bioorg.2016.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span> <span> </span><span class="NLM_article-title">Bioactivity-guided cut countercurrent chromatography for isolation of lysine-specific demethylase 1 inhibitors from Scutellaria baicalensis Georgi</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1016</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.aca.2018.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.aca.2018.01.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29534805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1016&publication_year=2018&pages=59-68&author=C.+Hanauthor=S.+Wangauthor=Z.+Liauthor=C.+Chenauthor=J.+Houauthor=D.+Xuauthor=R.+Wangauthor=Y.+Linauthor=J.+Luoauthor=L.+Kong&title=Bioactivity-guided+cut+countercurrent+chromatography+for+isolation+of+lysine-specific+demethylase+1+inhibitors+from+Scutellaria+baicalensis+Georgi&doi=10.1016%2Fj.aca.2018.01.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivity-guided cut countercurrent chromatography for isolation of lysine-specific demethylase 1 inhibitors from Scutellaria baicalensis Georgi</span></div><div class="casAuthors">Han, Chao; Wang, Shanshan; Li, Zhongrui; Chen, Chen; Hou, Jiqin; Xu, Dingqiao; Wang, Ruizhi; Lin, Yaolan; Luo, Jianguang; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1016</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Countercurrent chromatog. (CCC) has gradually become a widely used method for preparative sepn. of bioactive natural mols.  These mols. generally contain distinct scaffolds and characteristics that cannot be readily isolated from plants.  While one-dimensional CCC is typically used for the initial purifn. with insufficiently resolved peaks after locating bioactive components, two-dimensional (2D) or multi-dimensional CCC strategies are employed to improve the resoln. of peaks.  However, these methods usually present certain disadvantages, such as complicated procedures and increased time consumption, exptl. costs, and equipment requirements.  Here, a bioactivity-guided cut CCC strategy was established to isolate lysine-specific demethylase 1 (LSD1) inhibitors from Scutellaria baicalensis Georgi.  Gradient-elution CCC coupled with real-time detection of LSD1 inhibition by the collected fractions was developed.  Next, an online-storage recycling CCC mode was designed to enable the active fractions to be stored in coils, and these active fractions were further sepd. to obtain pure compds. by using sequential recycling elution.  In this strategy, active fractions are first identified, and then pure LSD1 inhibitors are isolated in the 2D CCC mode through continuous sepn. on a single instrument.  By using our bioactivity-guided cut CCC strategy, we successfully isolated six natural LSD1 inhibitors from S. baicalensis Georgi, five of which were identified for the first time as natural LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHMApE4g-b17Vg90H21EOLACvtfcHk0ljxEgBJ34zPyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaqsbs%253D&md5=862344f018da2cf3a708e3476e5e5d6b</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2018.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2018.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DL.%26atitle%3DBioactivity-guided%2520cut%2520countercurrent%2520chromatography%2520for%2520isolation%2520of%2520lysine-specific%2520demethylase%25201%2520inhibitors%2520from%2520Scutellaria%2520baicalensis%2520Georgi%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2018%26volume%3D1016%26spage%3D59%26epage%3D68%26doi%3D10.1016%2Fj.aca.2018.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span> <span> </span><span class="NLM_article-title">Flavone-based natural product agents as new lysine-specific demethylase 1 inhibitors exhibiting cytotoxicity against breast cancer cells in vitro</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">370</span>– <span class="NLM_lpage">374</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2018.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30552007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyksL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=370-374&author=X.+Xuauthor=W.+Pengauthor=C.+Liuauthor=S.+Liauthor=J.+Leiauthor=Z.+Wangauthor=L.+Kongauthor=C.+Han&title=Flavone-based+natural+product+agents+as+new+lysine-specific+demethylase+1+inhibitors+exhibiting+cytotoxicity+against+breast+cancer+cells+in+vitro&doi=10.1016%2Fj.bmc.2018.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Flavone-based natural product agents as new lysine-specific demethylase 1 inhibitors exhibiting cytotoxicity against breast cancer cells in vitro</span></div><div class="casAuthors">Xu, Xiao; Peng, Wenhui; Liu, Cuiyun; Li, Sixuan; Lei, Jiali; Wang, Zhen; Kong, Lingyi; Han, Chao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">370-374</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) has recently emerged as a therapeutic target for cancer.  However, almost all LSD1 inhibitors developed to date are chemo-synthesized mols.  In this study, the LSD1 inhibitory activity of 12 natural flavones, including four aglycons and their corresponding monoglycosides and diglucosides, was evaluated.  Based on the structure-activity relationships, LSD1 inhibition activity was greater for flavonoid monoglycosides than their aglycons lacking the sugar moiety.  The effects of isoquercitrin, which exhibited optimal LSD1 inhibitory activity, on cancer cell properties were evaluated.  Isoquercitrin induced the expression of key proteins in the mitochondrial-mediated apoptosis pathway and caused apoptosis in LSD1-overexpressing MDA-MB-231 cells via the inhibition of LSD1.  These findings suggest that natural LSD1 inhibitors, and particularly isoquercitrin, are promising for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoslK-6yseLw7Vg90H21EOLACvtfcHk0ljxEgBJ34zPyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyksL7M&md5=2b21cf4cc599cd1a8ae85d7460754cf5</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DKong%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DC.%26atitle%3DFlavone-based%2520natural%2520product%2520agents%2520as%2520new%2520lysine-specific%2520demethylase%25201%2520inhibitors%2520exhibiting%2520cytotoxicity%2520against%2520breast%2520cancer%2520cells%2520in%2520vitro%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2019%26volume%3D27%26spage%3D370%26epage%3D374%26doi%3D10.1016%2Fj.bmc.2018.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Q. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Q.</span></span> <span> </span><span class="NLM_article-title">Identification and biological evaluation of natural product Biochanin A</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">103674</span>– <span class="NLM_lpage">103679</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.103674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2020.103674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32097796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjvVKjs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2020&pages=103674-103679&author=L.+Wangauthor=L.+Z.+Liauthor=Q.+X.+Hanauthor=X.+F.+Wangauthor=D.+Zhaoauthor=J.+Q.+Liu&title=Identification+and+biological+evaluation+of+natural+product+Biochanin+A&doi=10.1016%2Fj.bioorg.2020.103674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and biological evaluation of natural product Biochanin A</span></div><div class="casAuthors">Wang, Lei; Li, Lingzhao; Han, Quanxiang; Wang, Xiaofang; Zhao, Di; Liu, Junqi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103674</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Natural products have shown promise for epigenetic modulations and thus are therapeutically potential for cancer prevention and treatment.  In this work, we report the identification of natural product Biochanin A as a new LSD1 inhibitor and further biol. evaluation in gastric MGC-803 cells.  Biochanin A effectively and reversibly inhibited LSD1 (IC50 = 2.95μM) and showed selective inhibition to LSD1 over MAO-A/B.  In gastric MGC-803 cells, Biochanin A induced accumulation of H3K4me1/2, inhibited cell growth moderately (IC50 = 6.77μM), but was less toxic to normal GES-1 (IC50 > 32μM).  Mechanistic studies showed that Biochanin A suppressed colony formation, cell apoptosis, and migration of MGC-803 cells.  This study may help to elucidate the anticancer mechanisms of Biochanin A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnWmRkIH8HF7Vg90H21EOLACvtfcHk0liuLRjBKoZGyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjvVKjs7c%253D&md5=b0ff6f75858691d6b21d6a4b6d6f3a4e</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.103674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.103674%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%2BZ.%26aulast%3DHan%26aufirst%3DQ.%2BX.%26aulast%3DWang%26aufirst%3DX.%2BF.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26atitle%3DIdentification%2520and%2520biological%2520evaluation%2520of%2520natural%2520product%2520Biochanin%2520A%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D97%26spage%3D103674%26epage%3D103679%26doi%3D10.1016%2Fj.bioorg.2020.103674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M. S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">112243</span>– <span class="NLM_lpage">112262</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2020.112243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32229389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmtFOmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2020&pages=112243-112262&author=X.+R.+Wangauthor=C.+Zhangauthor=X.+Y.+Zhangauthor=J.+K.+Yanauthor=J.+M.+Wangauthor=Q.+W.+Jiangauthor=L.+Y.+Zhaoauthor=D.+M.+Zhaoauthor=M.+S.+Cheng&title=Design%2C+synthesis+and+biological+evaluation+of+tetrahydroquinoline-based+reversible+LSD1+inhibitors&doi=10.1016%2Fj.ejmech.2020.112243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors</span></div><div class="casAuthors">Wang, Xinran; Zhang, Cai; Zhang, Xiangyu; Yan, Jiangkun; Wang, Jiming; Jiang, Qinwen; Zhao, Liyu; Zhao, Dongmei; Cheng, Maosheng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112243</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The targeted regulation of LSD1, which is highly expressed in a variety of tumor cells, is a promising cancer therapy strategy.  Several LSD1 inhibitors are currently under clin. evaluation, and most of these inhibitors are irreversible.  Here, the design, synthesis and biochem. evaluation of novel tetrahydroquinoline-based reversible LSD1 inhibitors has been reported.  Compds. (I) and (II), which are selective to LSD1 over MAO-A/B, exhibit excellent LSD1 inhibition at the mol. levels with IC50= 55 nM and 540 nM, resp.  The classic Lineweaver-Burk plots revealed that compd. I could reversibly bind the LSD1 enzyme in a noncompetitive manner.  Mol. docking was used to reveal the potential binding-mode of the compds. and interpret the structure-activity relationships.  Furthermore, compds. I and II significantly inhibited proliferation (IC50= 1.13μM and 1.15μM, resp.) and induced apoptosis in MGC-803 cells with high expression of LSD1.  Compd. IIshowed acceptable liver microsomal stability.  Meanwhile, II did not appear to inhibit CYPs at 10μM in vitro.  Remarkably, the oral administration of compd. II can inhibit the growth of MGC-803 xenograft tumors without significant side effects.  Author findings suggest that tetrahydroquinoline-based LSD1 inhibitors deserve further investigation for the treatment of LSD1 overexpressing cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouUACUV9d8o7Vg90H21EOLACvtfcHk0liuLRjBKoZGyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmtFOmurg%253D&md5=83fc46cdb95e5833d0c17fd6b0709ebd</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112243%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%2BR.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%2BY.%26aulast%3DYan%26aufirst%3DJ.%2BK.%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DJiang%26aufirst%3DQ.%2BW.%26aulast%3DZhao%26aufirst%3DL.%2BY.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520tetrahydroquinoline-based%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D194%26spage%3D112243%26epage%3D112262%26doi%3D10.1016%2Fj.ejmech.2020.112243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of [1,2,3]triazolo[4,5-d]pyrimidine derivatives as novel LSD1 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00423</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00423" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjslyju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=384-389&author=Z.+H.+Liauthor=X.+Q.+Liuauthor=P.+F.+Gengauthor=F.+Z.+Suoauthor=J.+L.+Maauthor=B.+Yuauthor=T.+Q.+Zhaoauthor=Z.+Q.+Zhouauthor=C.+X.+Huangauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Discovery+of+%5B1%2C2%2C3%5Dtriazolo%5B4%2C5-d%5Dpyrimidine+derivatives+as+novel+LSD1+inhibitors&doi=10.1021%2Facsmedchemlett.6b00423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of [1,2,3]Triazolo[4,5-d]pyrimidine Derivatives as Novel LSD1 Inhibitors</span></div><div class="casAuthors">Li, Zhong-Hua; Liu, Xue-Qi; Geng, Peng-Fei; Suo, Feng-Zhi; Ma, Jin-Lian; Yu, Bin; Zhao, Tao-Qian; Zhou, Zhao-Qing; Huang, Chen-Xi; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">384-389</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays a pivotal role in regulating the lysine methylation.  The aberrant overexpression of LSD1 was reported to be involved in the progression of certain human malignant tumors.  Abrogation of LSD1 with RNAi or small mol. inhibitors may lead to the inhibition of cancer proliferation and migration.  Herein, a series of [1,2,3]triazolo[4,5-d]pyrimidine derivs. were synthesized and evaluated for their LSD1 inhibitory effects.  The structure-activity relation studies (SARs) were conducted by exploring three regions of this scaffold, leading to the discovery of compd. 27 as potent LSD1 inhibitor (IC50 = 0.564 μM).  Compd. 27 was identified as a reversible LSD1 inhibitor and showed certain selectivity to LSD1 over monoamine oxidase A/B (MAO-A/B).  When MGC-803 cells were treated with compd. 27, the activity of LSD1 can be significantly inhibited, and the cell migration ability was also suppressed.  Docking studies indicated that the hydrogen interaction between the nitrogen atom in the pyridine ring and Met332 could be responsible for the improved activity of 2-thiopyridine series.  The [1,2,3]triazolo[4,5-d]pyrimidine scaffold can be used as the template for designing new LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeTW4tbCivwLVg90H21EOLACvtfcHk0liuLRjBKoZGyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjslyju74%253D&md5=dc8bb933eebb850e165663d6de2b8270</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00423%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BH.%26aulast%3DLiu%26aufirst%3DX.%2BQ.%26aulast%3DGeng%26aufirst%3DP.%2BF.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DT.%2BQ.%26aulast%3DZhou%26aufirst%3DZ.%2BQ.%26aulast%3DHuang%26aufirst%3DC.%2BX.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520%255B1%252C2%252C3%255Dtriazolo%255B4%252C5-d%255Dpyrimidine%2520derivatives%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D384%26epage%3D389%26doi%3D10.1021%2Facsmedchemlett.6b00423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Development of the triazole-fused pyrimidine derivatives as highly potent and reversible inhibitors of histone lysine specific demethylase 1 (LSD1/KDM1A)</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2019.01.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.apsb.2019.01.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31384539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BB3MvltFyitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=794-808&author=Z.+H.+Liauthor=L.+N.+Dingauthor=Z.+R.+Liauthor=Z.+Z.+Wangauthor=F.+Z.+Suoauthor=D.+D.+Shenauthor=T.+Q.+Zhaoauthor=X.+D.+Sunauthor=J.+W.+Wangauthor=Y.+Liuauthor=L.+Y.+Maauthor=B.+Zhaoauthor=P.+F.+Gengauthor=B.+Yuauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Development+of+the+triazole-fused+pyrimidine+derivatives+as+highly+potent+and+reversible+inhibitors+of+histone+lysine+specific+demethylase+1+%28LSD1%2FKDM1A%29&doi=10.1016%2Fj.apsb.2019.01.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the triazole-fused pyrimidine derivatives as highly potent and reversible inhibitors of histone lysine specific demethylase 1 (LSD1/KDM1A)</span></div><div class="casAuthors">Li Zhonghua; Ding Lina; Li Zhongrui; Wang Zhizheng; Suo Fengzhi; Shen Dandan; Zhao Taoqian; Sun Xudong; Wang Junwei; Liu Ying; Ma Liying; Zhao Bing; Geng Pengfei; Yu Bin; Zheng Yichao; Liu Hongmin; Li Zhonghua; Ding Lina; Li Zhongrui; Wang Zhizheng; Suo Fengzhi; Shen Dandan; Zhao Taoqian; Sun Xudong; Wang Junwei; Liu Ying; Ma Liying; Zhao Bing; Geng Pengfei; Yu Bin; Zheng Yichao; Liu Hongmin; Li Zhonghua; Ding Lina; Li Zhongrui; Wang Zhizheng; Suo Fengzhi; Shen Dandan; Zhao Taoqian; Sun Xudong; Wang Junwei; Liu Ying; Ma Liying; Zhao Bing; Geng Pengfei; Yu Bin; Zheng Yichao; Liu Hongmin; Yu Bin</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">794-808</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) has been recognized as an important modulator in post-translational process in epigenetics.  Dysregulation of LSD1 has been implicated in the development of various cancers.  Herein, we report the discovery of the hit compound 8a (IC50 = 3.93 μmol/L) and further medicinal chemistry efforts, leading to the generation of compound 15u (IC50 = 49 nmol/L, and Ki = 16 nmol/L), which inhibited LSD1 reversibly and competitively with H3K4me2, and was selective to LSD1 over MAO-A/B.  Docking studies were performed to rationalize the potency of compound 15u.  Compound 15u also showed strong antiproliferative activity against four leukemia cell lines (OCL-AML3, K562, THP-1 and U937) as well as the lymphoma cell line Raji with the IC50 values of 1.79, 1.30, 0.45, 1.22 and 1.40 μmol/L, respectively.  In THP-1 cell line, 15u significantly inhibited colony formation and caused remarkable morphological changes.  Compound 15u induced expression of CD86 and CD11b in THP-1 cells, confirming its cellular activity and ability of inducing differentiation.  The findings further indicate that targeting LSD1 is a promising strategy for AML treatment, the triazole-fused pyrimidine derivatives are new scaffolds for the development of LSD1/KDM1A inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT87t7BYwXnlAqdek-gHSrIfW6udTcc2eal4XyuXM7WD7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvltFyitA%253D%253D&md5=9c4f8acdb5e0d95edc99b6e0cab947f2</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2019.01.001%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BH.%26aulast%3DDing%26aufirst%3DL.%2BN.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DWang%26aufirst%3DZ.%2BZ.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DZhao%26aufirst%3DT.%2BQ.%26aulast%3DSun%26aufirst%3DX.%2BD.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DGeng%26aufirst%3DP.%2BF.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDevelopment%2520of%2520the%2520triazole-fused%2520pyrimidine%2520derivatives%2520as%2520highly%2520potent%2520and%2520reversible%2520inhibitors%2520of%2520histone%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2019%26volume%3D9%26spage%3D794%26epage%3D808%26doi%3D10.1016%2Fj.apsb.2019.01.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. Q.</span></span> <span> </span><span class="NLM_article-title">[1,2,3]Triazolo[4,5-d]pyrimidine derivatives incorporating (thio)urea moiety as a novel scaffold for LSD1 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>187</i></span>,  <span class="NLM_fpage">111989</span>– <span class="NLM_lpage">111997</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2019.111989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31881456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2020&pages=111989-111997&author=Z.+H.+Liauthor=J.+L.+Maauthor=G.+Z.+Liuauthor=X.+H.+Zhangauthor=T.+T.+Qinauthor=W.+H.+Renauthor=T.+Q.+Zhaoauthor=X.+H.+Chenauthor=Z.+Q.+Zhang&title=%5B1%2C2%2C3%5DTriazolo%5B4%2C5-d%5Dpyrimidine+derivatives+incorporating+%28thio%29urea+moiety+as+a+novel+scaffold+for+LSD1+inhibitors&doi=10.1016%2Fj.ejmech.2019.111989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">[1,2,3]Triazolo[4,5-d]pyrimidine derivatives incorporating (thio)urea moiety as a novel scaffold for LSD1 inhibitors</span></div><div class="casAuthors">Li, Zhong-Hua; Ma, Jin-Lian; Liu, Gai-Zhi; Zhang, Xin-Hui; Qin, Ting-Ting; Ren, Wei-Hong; Zhao, Tao-Qian; Chen, Xiao-Hui; Zhang, Zhen-Qiang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">187</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111989</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an essential role in maintaining a balanced methylation status at histone tails.  Overexpression of LSD1 has been involved in the development of a variety of human diseases, including cancers.  Herein, on the basis of our previously developed LSD1 inhibitors, two series of new [1,2,3]triazolo[4,5-d]pyrimidine derivs. incorporating (thio)urea moiety were designed and evaluated for their LSD1 inhibitory abilities, leading to a novel chem. class of LSD1 inhibitors.  Among them, compd. 31 was found to moderately inhibit LSD1 activity, as well as increase the expression of H3K4me2 at the cellular level.  This compd. also showed good selectivity against MAO-A/-B, and a panel of kinases such as CDK and BTK.  Besides, the MTT assay suggested that the selected compds. could inhibit the proliferation of LSD1-overexpressed cancer cells.  Although this class of compds. only showed moderate anti-LSD1 activity in the micromolar range, this work presents a novel chemotype of LSD1 inhibitors with good enzyme selectivity as well as cellular LSD1 inhibitory activity, and could provide a useful template for the development of more potent LSD1 inhibitors for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ2CKPL-inPrVg90H21EOLACvtfcHk0ljncFq4EUNAOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVyrurfM&md5=a62bf474d109b25ad9a517cccb6d0db2</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111989%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BH.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DLiu%26aufirst%3DG.%2BZ.%26aulast%3DZhang%26aufirst%3DX.%2BH.%26aulast%3DQin%26aufirst%3DT.%2BT.%26aulast%3DRen%26aufirst%3DW.%2BH.%26aulast%3DZhao%26aufirst%3DT.%2BQ.%26aulast%3DChen%26aufirst%3DX.%2BH.%26aulast%3DZhang%26aufirst%3DZ.%2BQ.%26atitle%3D%255B1%252C2%252C3%255DTriazolo%255B4%252C5-d%255Dpyrimidine%2520derivatives%2520incorporating%2520%2528thio%2529urea%2520moiety%2520as%2520a%2520novel%2520scaffold%2520for%2520LSD1%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D187%26spage%3D111989%26epage%3D111997%26doi%3D10.1016%2Fj.ejmech.2019.111989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-a]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">951</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2016.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27769034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=940-951&author=S.+Wangauthor=L.+J.+Zhaoauthor=Y.+C.+Zhengauthor=D.+D.+Shenauthor=E.+F.+Miaoauthor=X.+P.+Qiaoauthor=L.+J.+Zhaoauthor=Y.+Liuauthor=R.+Huangauthor=B.+Yuauthor=H.+M.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+%5B1%2C2%2C4%5Dtriazolo%5B1%2C5-a%5Dpyrimidines+as+potent+lysine+specific+demethylase+1+%28LSD1%2FKDM1A%29+inhibitors&doi=10.1016%2Fj.ejmech.2016.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of [1,2,4]triazolo[1,5-a]pyrimidines as potent lysine specific demethylase 1 (LSD1/KDM1A) inhibitors</span></div><div class="casAuthors">Wang, Shuai; Zhao, Li-Jie; Zheng, Yi-Chao; Shen, Dan-Dan; Miao, Er-Fei; Qiao, Xue-Peng; Zhao, Li-Juan; Liu, Ying; Huang, Ruilei; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">940-951</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A new series of [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1 inhibitors was designed, synthesized, and further evaluated for cytotoxicity against MGC-803, EC109, A549 and PC-9 cells as well as the ability of inhibiting LSD1.  Some of these compds. showed potent inhibition toward LSD1 and selectively inhibited growth of A549 and PC-9 cells.  Compd. I potently inhibited growth of PC-9 cells (IC50 = 0.59 μM), about 4-fold more potent than 5-FU.  Further SAR studies led to the identification of compds. I and II, which had good growth inhibition against all the tested cancer cell lines and were much more potent than 5-FU and GSK2879552.  Besides, compds. III-V inhibited LSD1 potently (IC50 = 0.154, 1.19 and 0.557 μM, resp.).  Docking studies revealed that compd. III formed arene-H interactions with Val333 and hydrogen bonds with surrounding Ala331, Met332, and Ala539 residues.  Compd. III significantly inhibited migration of A549 and PC-9 cells in a concn.-dependent manner, but had different effect on the expression of E-cadherin and N-cadherin.  The [1,2,4]triazolo[1,5-a]pyrimidine scaffold may serve as a starting point for developing potent LSD1 inhibitors for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0jqW-eLBQObVg90H21EOLACvtfcHk0ljncFq4EUNAOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWju7rJ&md5=d022678613486890faacfef9c163492b</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DMiao%26aufirst%3DE.%2BF.%26aulast%3DQiao%26aufirst%3DX.%2BP.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520%255B1%252C2%252C4%255Dtriazolo%255B1%252C5-a%255Dpyrimidines%2520as%2520potent%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%252FKDM1A%2529%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D125%26spage%3D940%26epage%3D951%26doi%3D10.1016%2Fj.ejmech.2016.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.02.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2019.02.039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30780087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Wkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=388-401&author=S.+Wangauthor=Z.+R.+Liauthor=F.+Z.+Suoauthor=X.+H.+Yuanauthor=B.+Yuauthor=H.+M.+Liu&title=Synthesis%2C+structure-activity+relationship+studies+and+biological+characterization+of+new+%5B1%2C2%2C4%5Dtriazolo%5B1%2C5-a%5Dpyrimidine-based+LSD1%2FKDM1A+inhibitors&doi=10.1016%2Fj.ejmech.2019.02.039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors</span></div><div class="casAuthors">Wang, Shuai; Li, Zhong-Rui; Suo, Feng-Zhi; Yuan, Xiao-Han; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">388-401</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The design, synthesis and biochem. characterization of [1,2,4]triazolo[1,5-a]pyrimidine derivs. I [R1 = H, [(2-bromophenyl)methyl]sulfanyl, prop-2-en-1-ylsulfanyl, [(1H-1,3-benzodiazol-2-ylmethyl)sulfanyl], etc.; R2 = H, Me, (CH2)4CH3; R3 = Me, Et, C6H5; R2, R3 = -(CH2)3-; R4 = H, C6H5, [4-(4-methylpiperazin-1-yl)phenyl], etc.] as new LSD1 inhibitors have been reported.  Of these compds., compd. I [R1 = (1H-1,3-benzodiazol-2-ylsulfanyl)methyl; R2 = H; R3 = Me; R4 = [4-(4-methylpiperazin-1-yl)phenyl]] (II) inhibited LSD1 in a reversible manner (IC50 = 1.72 μM) and showed selectivity to LSD1 over MAO-A/B.  Besides, compd. II displayed FAD-competitive binding to LSD1.  Interestingly, compd. II did not inhibit horseradish peroxidase (HRP) and quench H2O2, thus excluding the possibility that LSD1 inhibition by compd. II was due to the HRP inhibition and consumption of H2O2.  In LSD1 overexpressed A549 cells, compd. II concn.-dependently induced accumulation of H3K4me1/me2 and H3K9me2 and showed cellular target engagement to LSD1.  Addnl., compd. II significantly inhibited migration of A549 cells in a concn.-dependent manner, further western blot anal. showed that compd. II increased expression levels of epithelial cell markers E-Cadherin and Claudin-1, down-regulated mesenchymal cell marker N-Cadherin and the upstream transcription factors Snail and Slug.  Docking studies were also performed to rationalize the potency of compd. II toward LSD1.  To conclude, the [1,2,4]triazolo[1,5-a]pyrimidine I could serve as a promising scaffold for the development of new LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFGjGReI1_PLVg90H21EOLACvtfcHk0ljxwBEpkh8Sew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Wkurk%253D&md5=a5ce8c3d2239535b47bf93cf20c9c5c3</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DYuan%26aufirst%3DX.%2BH.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DSynthesis%252C%2520structure-activity%2520relationship%2520studies%2520and%2520biological%2520characterization%2520of%2520new%2520%255B1%252C2%252C4%255Dtriazolo%255B1%252C5-a%255Dpyrimidine-based%2520LSD1%252FKDM1A%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401%26doi%3D10.1016%2Fj.ejmech.2019.02.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Experience-based discovery (EBD) of aryl hydrazines as new scaffolds for the development of LSD1/KDM1A inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2019.01.075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30739825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivFygtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2019&pages=432-444&author=Z.+R.+Liauthor=S.+Wangauthor=L.+Yangauthor=X.+H.+Yuanauthor=F.+Z.+Suoauthor=B.+Yuauthor=H.+M.+Liu&title=Experience-based+discovery+%28EBD%29+of+aryl+hydrazines+as+new+scaffolds+for+the+development+of+LSD1%2FKDM1A+inhibitors&doi=10.1016%2Fj.ejmech.2019.01.075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Experience-based discovery (EBD) of aryl hydrazines as new scaffolds for the development of LSD1/KDM1A inhibitors</span></div><div class="casAuthors">Li, Zhong-Rui; Wang, Shuai; Yang, Linlin; Yuan, Xiao-Han; Suo, Feng-Zhi; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">432-444</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Phenelzine was first employed to design new aryl hydrazine-based LSD1 inhibitors based on the experience-based discovery (EBD) strategy.  Among these compds., D8 potently inhibited LSD1 (IC50 = 882.30 nM) in a reversible manner.  Compd. D8 was selective to LSD1 over MAO-A/B and showed H3K4me2 competitive binding to LSD1.  The interaction between H3K4me2 and LSD1 was also confirmed by the Co-IP assay.  In LSD1 overexpressed A549 cells, compd. D8 dose-dependently induced accumulation of LSD1 substrates H3K4me1/2 and H3K9me1/2, showed cellular target engagement to LSD1 and significantly inhibited cell migration of A549 cells.  Docking studies suggested that compd. D8 occupied the peptide binding region and therefore blocked the access of the peptide substrate to the FAD, finally leading to the demethylase activity inhibition of LSD1.  The findings indicate that aryl hydrazines are new scaffolds for the design of LSD1 inhibitors, the identification of D8 provides further evidence for our previously proposed general principle that fused heterocycles with an amine group are potentially active toward LSD1 by competitive binding to LSD1 with H3 peptide substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGH1kwCayTurVg90H21EOLACvtfcHk0ljxwBEpkh8Sew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivFygtL4%253D&md5=18d5b4e7e51f6a911f0e07bb5ed955be</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.075%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYuan%26aufirst%3DX.%2BH.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DExperience-based%2520discovery%2520%2528EBD%2529%2520of%2520aryl%2520hydrazines%2520as%2520new%2520scaffolds%2520for%2520the%2520development%2520of%2520LSD1%252FKDM1A%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D166%26spage%3D432%26epage%3D444%26doi%3D10.1016%2Fj.ejmech.2019.01.075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coto-Montes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boga, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vriend, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span> <span> </span><span class="NLM_article-title">Melatonin: an ancient molecule that makes oxygen metabolically tolerable</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">403</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.1111/jpi.12267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=26272235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyju7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=403-419&author=L.+C.+Manchesterauthor=A.+Coto-Montesauthor=J.+A.+Bogaauthor=L.+P.+Andersenauthor=Z.+Zhouauthor=A.+Galanoauthor=J.+Vriendauthor=D.+X.+Tanauthor=R.+J.+Reiter&title=Melatonin%3A+an+ancient+molecule+that+makes+oxygen+metabolically+tolerable&doi=10.1111%2Fjpi.12267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin: an ancient molecule that makes oxygen metabolically tolerable</span></div><div class="casAuthors">Manchester, Lucien C.; Coto-Montes, Ana; Boga, Jose Antonio; Andersen, Lars Peter H.; Zhou, Zhou; Galano, Annia; Vriend, Jerry; Tan, Dun-Xian; Reiter, Russel J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">403-419</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Melatonin is remarkably functionally diverse with actions as a free radical scavenger and antioxidant, circadian rhythm regulator, anti-inflammatory and immunoregulating mol., and as an oncostatic agent.  We hypothesize that the initial and primary function of melatonin in photosynthetic cyanobacteria, which appeared on Earth 3.5-3.2 billion years ago, was as an antioxidant.  The evolution of melatonin as an antioxidant by this organism was necessary as photosynthesis is assocd. with the generation of toxic-free radicals.  The other secondary functions of melatonin came about much later in evolution.  We also surmise that mitochondria and chloroplasts may be primary sites of melatonin synthesis in all eukaryotic cells that possess these organelles.  This prediction is made on the basis that mitochondria and chloroplasts of eukaryotes developed from purple nonsulfur bacteria (which also produce melatonin) and cyanobacteria when they were engulfed by early eukaryotes.  Thus, we speculate that the melatonin-synthesizing actions of the engulfed bacteria were retained when these organelles became mitochondria and chloroplasts, resp.  That mitochondria are likely sites of melatonin formation is supported by the observation that this organelle contains high levels of melatonin that are not impacted by blood melatonin concns.  Melatonin has a remarkable array of means by which it thwarts oxidative damage.  It, as well as its metabolites, is differentially effective in scavenging a variety of reactive oxygen and reactive nitrogen species.  Moreover, melatonin and its metabolites modulate a large no. of antioxidative and pro-oxidative enzymes, leading to a redn. in oxidative damage.  The actions of melatonin on radical metabolizing/producing enzymes may be mediated by the Keap1-Nrf2-ARE pathway.  Beyond its direct free radical scavenging and indirect antioxidant effects, melatonin has a variety of physiol. and metabolic advantages that may enhance its ability to limit oxidative stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosyCdaNdqJYLVg90H21EOLACvtfcHk0ljxwBEpkh8Sew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyju7fI&md5=b086220e459573991456ef35b91d7cc6</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12267%26sid%3Dliteratum%253Aachs%26aulast%3DManchester%26aufirst%3DL.%2BC.%26aulast%3DCoto-Montes%26aufirst%3DA.%26aulast%3DBoga%26aufirst%3DJ.%2BA.%26aulast%3DAndersen%26aufirst%3DL.%2BP.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DGalano%26aufirst%3DA.%26aulast%3DVriend%26aufirst%3DJ.%26aulast%3DTan%26aufirst%3DD.%2BX.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26atitle%3DMelatonin%253A%2520an%2520ancient%2520molecule%2520that%2520makes%2520oxygen%2520metabolically%2520tolerable%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D59%26spage%3D403%26epage%3D419%26doi%3D10.1111%2Fjpi.12267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Q. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span> <span> </span><span class="NLM_article-title">Melatonin induces apoptosis of colorectal cancer cells through HDAC4 nuclear import mediated by CaMKII inactivation</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">438</span>, <span class="refDoi"> DOI: 10.1111/jpi.12226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25752481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=429-438&author=J.+Y.+Weiauthor=W.+M.+Liauthor=L.+L.+Zhouauthor=Q.+N.+Luauthor=W.+He&title=Melatonin+induces+apoptosis+of+colorectal+cancer+cells+through+HDAC4+nuclear+import+mediated+by+CaMKII+inactivation&doi=10.1111%2Fjpi.12226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin induces apoptosis of colorectal cancer cells through HDAC4 nuclear import mediated by CaMKII inactivation</span></div><div class="casAuthors">Wei, Jia-Yi; Li, Wan-Ming; Zhou, Lin-Lin; Lu, Qiu-Nan; He, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">429-438</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Melatonin induces apoptosis in many different cancer cell lines, including colorectal cancer.  However, the precise mechanisms involved remain largely unresolved.  In this study, we provide evidence to reveal a new mechanism by which melatonin induces apoptosis of colorectal cancer LoVo cells.  Melatonin at pharmacol. concns. significantly suppressed cell proliferation and induced apoptosis in a dose-dependent manner.  The obsd. apoptosis was accompanied by the melatonin-induced dephosphorylation and nuclear import of histone deacetylase 4 (HDAC4).  Pretreatment with a HDAC4-specific siRNA effectively attenuated the melatonin-induced apoptosis, indicating that nuclear localization of HDAC4 is required for melatonin-induced apoptosis.  Moreover, constitutively active Ca2+/calmodulin-dependent protein kinase II alpha (CaMKIIα) abrogated the melatonin-induced HDAC4 nuclear import and apoptosis of LoVo cells.  Furthermore, melatonin decreased H3 acetylation on bcl-2 promoter, leading to a redn. of bcl-2 expression, whereas constitutively active CaMKIIα(T286D) or HDAC4-specific siRNA abrogated the effect of melatonin.  In conclusion, the present study provides evidence that melatonin-induced apoptosis in colorectal cancer LoVo cells largely depends on the nuclear import of HDAC4 and subsequent H3 deacetylation via the inactivation of CaMKIIα.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqZKAF9ddSo7Vg90H21EOLACvtfcHk0ljxwBEpkh8Sew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb4%253D&md5=90bcaac43d977d9e8203437697b72c1b</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12226%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%2BY.%26aulast%3DLi%26aufirst%3DW.%2BM.%26aulast%3DZhou%26aufirst%3DL.%2BL.%26aulast%3DLu%26aufirst%3DQ.%2BN.%26aulast%3DHe%26aufirst%3DW.%26atitle%3DMelatonin%2520induces%2520apoptosis%2520of%2520colorectal%2520cancer%2520cells%2520through%2520HDAC4%2520nuclear%2520import%2520mediated%2520by%2520CaMKII%2520inactivation%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D58%26spage%3D429%26epage%3D438%26doi%3D10.1111%2Fjpi.12226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauchy, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wren, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belancio, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blask, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S. M.</span></span> <span> </span><span class="NLM_article-title">Doxorubicin resistance in breast cancer is driven by light at night-induced disruption of the circadian melatonin signal</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1111/jpi.12239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12239" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25857269" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslensrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=60-69&author=S.+L.+Xiangauthor=R.+T.+Dauchyauthor=A.+Hauchauthor=L.+L.+Maoauthor=L.+Yuanauthor=M.+A.+Wrenauthor=V.+P.+Belancioauthor=D.+Mondalauthor=T.+Fraschauthor=D.+E.+Blaskauthor=S.+M.+Hill&title=Doxorubicin+resistance+in+breast+cancer+is+driven+by+light+at+night-induced+disruption+of+the+circadian+melatonin+signal&doi=10.1111%2Fjpi.12239"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Doxorubicin resistance in breast cancer is driven by light at night-induced disruption of the circadian melatonin signal</span></div><div class="casAuthors">Xiang, Shulin; Dauchy, Robert T.; Hauch, Adam; Mao, Lulu; Yuan, Lin; Wren, Melissa A.; Belancio, Victoria P.; Mondal, Debasis; Frasch, Tripp; Blask, David E.; Hill, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-69</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Chemotherapeutic resistance, particularly to doxorubicin (Dox), represents a major impediment to successfully treating breast cancer and is linked to elevated tumor metab. and tumor over-expression and/or activation of various families of receptor- and non-receptor-assocd. tyrosine kinases.  Disruption of circadian time structure and suppression of nocturnal melatonin prodn. by dim light exposure at night (dLEN), as occurs with shift work, and/or disturbed sleep-wake cycles, is assocd. with a significantly increased risk of an array of diseases, including breast cancer.  Melatonin inhibits human breast cancer growth via mechanisms that include the suppression of tumor metab. and inhibition of expression or phospho-activation of the receptor kinases AKT and ERK1/2 and various other kinases and transcription factors.  We demonstrate in tissue-isolated estrogen receptor alpha-pos. (ERα+) MCF-7 human breast cancer xenografts, grown in nude rats maintained on a light/dark cycle of LD 12:12 in which dLEN is present during the dark phase (suppressed endogenous nocturnal melatonin), a significant shortening of tumor latency-to-onset, increased tumor metab. and growth, and complete intrinsic resistance to Dox therapy.  Conversely, a LD 12:12 dLEN environment incorporating nocturnal melatonin replacement resulted in significantly lengthened tumor latency-to-onset, tumor regression, suppression of nighttime tumor metab., and kinase and transcription factor phosphorylation, while Dox sensitivity was completely restored.  Melatonin acts as both a tumor metabolic inhibitor and circadian-regulated kinase inhibitor to reestablish the sensitivity of breast tumors to Dox and drive tumor regression, indicating that dLEN-induced circadian disruption of nocturnal melatonin prodn. contributes to a complete loss of tumor sensitivity to Dox chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoM6CZtxM6IO7Vg90H21EOLACvtfcHk0libBCkuan9_eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslensrk%253D&md5=bdbc62e23ad721390984935c19f954a4</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12239%26sid%3Dliteratum%253Aachs%26aulast%3DXiang%26aufirst%3DS.%2BL.%26aulast%3DDauchy%26aufirst%3DR.%2BT.%26aulast%3DHauch%26aufirst%3DA.%26aulast%3DMao%26aufirst%3DL.%2BL.%26aulast%3DYuan%26aufirst%3DL.%26aulast%3DWren%26aufirst%3DM.%2BA.%26aulast%3DBelancio%26aufirst%3DV.%2BP.%26aulast%3DMondal%26aufirst%3DD.%26aulast%3DFrasch%26aufirst%3DT.%26aulast%3DBlask%26aufirst%3DD.%2BE.%26aulast%3DHill%26aufirst%3DS.%2BM.%26atitle%3DDoxorubicin%2520resistance%2520in%2520breast%2520cancer%2520is%2520driven%2520by%2520light%2520at%2520night-induced%2520disruption%2520of%2520the%2520circadian%2520melatonin%2520signal%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D59%26spage%3D60%26epage%3D69%26doi%3D10.1111%2Fjpi.12239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, G.</span></span> <span> </span><span class="NLM_article-title">Melatonin as a treatment for gastrointestinal cancer: a review</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1111/jpi.12227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1111%2Fjpi.12227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=25752643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=375-387&author=Z.+L.+Xinauthor=S.+Jiangauthor=P.+Jiangauthor=X.+L.+Yanauthor=C.+X.+Fanauthor=S.+Y.+Diauthor=G.+L.+Wuauthor=Y.+Yangauthor=R.+J.+Reiterauthor=G.+Ji&title=Melatonin+as+a+treatment+for+gastrointestinal+cancer%3A+a+review&doi=10.1111%2Fjpi.12227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin as a treatment for gastrointestinal cancer: a review</span></div><div class="casAuthors">Xin, Zhenlong; Jiang, Shuai; Jiang, Peng; Yan, Xiaolong; Fan, Chongxi; Di, Shouyin; Wu, Guiling; Yang, Yang; Reiter, Russel J.; Ji, Gang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-387</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Gastrointestinal cancer is a disease that affects the population worldwide with high morbidity and mortality.  Melatonin, an endogenously produced mol., may provide a defense against a variety of cancer types.  In particular, the ability of melatonin to inhibit gastrointestinal cancer is substantial.  In this review, we first clarify the relationship between the disruption of the melatonin rhythm and gastrointestinal cancer (based on epidemiol. surveys and animal and human studies) and summarize the preventive effect of melatonin on carcinogenesis.  Thereafter, the mechanisms through which melatonin exerts its anti-gastrointestinal cancer actions are explained, including inhibition of proliferation, invasion, metastasis, and angiogenesis, and promotion of apoptosis and cancer immunity.  Moreover, we discuss the drug synergy effects and the role of melatonin receptors involved in the growth-inhibitory effects on gastrointestinal cancer.  Taken together, the information compiled here serves as a comprehensive ref. for the anti-gastrointestinal cancer actions of melatonin that have been identified to date and will hopefully aid in the design of further exptl. and clin. studies and increase the awareness of melatonin as a therapeutic agent in cancers of the gastrointestinal tract.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHJNmf8c1j7rVg90H21EOLACvtfcHk0libBCkuan9_eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFahsb8%253D&md5=708b3eedfa5d45583901675aa3aa6628</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12227%26sid%3Dliteratum%253Aachs%26aulast%3DXin%26aufirst%3DZ.%2BL.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DX.%2BL.%26aulast%3DFan%26aufirst%3DC.%2BX.%26aulast%3DDi%26aufirst%3DS.%2BY.%26aulast%3DWu%26aufirst%3DG.%2BL.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26aulast%3DJi%26aufirst%3DG.%26atitle%3DMelatonin%2520as%2520a%2520treatment%2520for%2520gastrointestinal%2520cancer%253A%2520a%2520review%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2015%26volume%3D58%26spage%3D375%26epage%3D387%26doi%3D10.1111%2Fjpi.12227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Melatonin: a potential therapeutic option for breast cancer</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">859</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2020.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.tem.2020.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32893084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1CktLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=859-871&author=X.+Y.+Kongauthor=R.+Gaoauthor=Z.+Z.+Wangauthor=X.+Y.+Wangauthor=Y.+Fangauthor=J.+D.+Gaoauthor=R.+J.+Reiterauthor=J.+Wang&title=Melatonin%3A+a+potential+therapeutic+option+for+breast+cancer&doi=10.1016%2Fj.tem.2020.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin As A Potential Therapeutic Option for Breast Cancer</span></div><div class="casAuthors">Kong, Xiangyi; Gao, Ran; Wang, Zhongzhao; Wang, Xiangyu; Fang, Yi; Gao, Jidong; Reiter, Russel J.; Wang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">859-871</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Melatonin has significant inhibitory effects in numerous cancers, esp. breast cancer.  In estrogen receptor (ER)-pos. human breast cancer, the oncostatic actions of melatonin are mainly achieved by suppressing ER mRNA expression and ER transcriptional activity via the MT1 receptor.  Melatonin also regulates the transactivation of nuclear receptors, estrogen-metabolizing enzymes, and the expression of related genes.  Furthermore, melatonin suppresses tumor aerobic glycolysis, crit. cell-signaling pathways relevant to cell proliferation, survival, metastasis, and overcomes drug resistance.  Studies in animal and human models indicate that disruption of the circadian nocturnal melatonin signal promotes the growth, metab., and signaling of human breast cancer, resulting in resistance to hormone therapy and chemotherapy, which may be reversed by melatonin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn-7V7hTSjA7Vg90H21EOLACvtfcHk0libBCkuan9_eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1CktLjI&md5=a4f7a9c322df69a02ec72d9f6032160c</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2020.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2020.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%2BY.%26aulast%3DGao%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DZ.%2BZ.%26aulast%3DWang%26aufirst%3DX.%2BY.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DJ.%2BD.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DMelatonin%253A%2520a%2520potential%2520therapeutic%2520option%2520for%2520breast%2520cancer%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2020%26volume%3D31%26spage%3D859%26epage%3D871%26doi%3D10.1016%2Fj.tem.2020.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farhood, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goradel, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortezaee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanlarkhani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahebkar, A.</span></span> <span> </span><span class="NLM_article-title">Melatonin and cancer: from the promotion of genomic stability to use in cancer treatment</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">5613</span>– <span class="NLM_lpage">5627</span>, <span class="refDoi"> DOI: 10.1002/jcp.27391</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fjcp.27391" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30238978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGlt7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2019&pages=5613-5627&author=B.+Farhoodauthor=N.+H.+Goradelauthor=K.+Mortezaeeauthor=N.+Khanlarkhaniauthor=M.+Najafiauthor=A.+Sahebkar&title=Melatonin+and+cancer%3A+from+the+promotion+of+genomic+stability+to+use+in+cancer+treatment&doi=10.1002%2Fjcp.27391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin and cancer: From the promotion of genomic stability to use in cancer treatment</span></div><div class="casAuthors">Farhood, Bagher; Goradel, Nasser Hashemi; Mortezaee, Keywan; Khanlarkhani, Neda; Najafi, Masoud; Sahebkar, Amirhossein</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5613-5627</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cancer remains among the most challenging human diseases.  Several lines of evidence suggest that carcinogenesis is a complex process that is initiated by DNA damage.  Exposure to clastogenic agents such as heavy metals, ionizing radiation (IR), and chemotherapy drugs may cause chronic mutations in the genomic material, leading to a phenomenon named genomic instability.  Evidence suggests that genomic instability is responsible for cancer incidence after exposure to carcinogenic agents, and increases the risk of secondary cancers following treatment with radiotherapy or chemotherapy.  Melatonin as the main product of the pineal gland is a promising hormone for preventing cancer and improving cancer treatment.  Melatonin can directly neutralize toxic free radicals more efficiently compared with other classical antioxidants.  In addn., melatonin is able to regulate the redn./oxidn. (redox) system in stress conditions.  Through regulation of mitochondrial nction and inhibition of pro-oxidant enzymes, melatonin suppresses chronic oxidative stress.  Moreover, melatonin potently stimulates DNA damage responses that increase the tolerance of normal tissues to toxic effect of IR and may reduce the risk of genomic instability in patients who undergo radiotherapy.  Through these mechanisms, melatonin attenuates several side effects of radiotherapy and chemotherapy.  Interestingly, melatonin has shown some synergistic properties with IR and chemotherapy, which is distinct from classical antioxidants that are mainly used for the alleviation of adverse events of radiotherapy and chemotherapy.  In this review, we describe the anticarcinogenic effects of melatonin and also its possible application in clin. oncol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro3KIsdYB6mrVg90H21EOLACvtfcHk0lj7d7YGwm0gIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslGlt7fE&md5=7965ad9b4f7bdf2058dee7683a406ee7</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fjcp.27391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.27391%26sid%3Dliteratum%253Aachs%26aulast%3DFarhood%26aufirst%3DB.%26aulast%3DGoradel%26aufirst%3DN.%2BH.%26aulast%3DMortezaee%26aufirst%3DK.%26aulast%3DKhanlarkhani%26aufirst%3DN.%26aulast%3DNajafi%26aufirst%3DM.%26aulast%3DSahebkar%26aufirst%3DA.%26atitle%3DMelatonin%2520and%2520cancer%253A%2520from%2520the%2520promotion%2520of%2520genomic%2520stability%2520to%2520use%2520in%2520cancer%2520treatment%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D2019%26volume%3D234%26spage%3D5613%26epage%3D5627%26doi%3D10.1002%2Fjcp.27391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maroufi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vahedian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hemati, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rashidi, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akbarzadeh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahedi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouremamali, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isazadeh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taefehshokr, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajazimian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seraji, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nouri, M.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer stem cells by melatonin: effective therapy for cancer treatment</span>. <i>Pathol., Res. Pract.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>216</i></span>,  <span class="NLM_fpage">152919</span>– <span class="NLM_lpage">152926</span>, <span class="refDoi"> DOI: 10.1016/j.prp.2020.152919</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.prp.2020.152919" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32171553" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvFChsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=216&publication_year=2020&pages=152919-152926&author=N.+F.+Maroufiauthor=V.+Vahedianauthor=S.+Hematiauthor=M.+R.+Rashidiauthor=M.+Akbarzadehauthor=M.+Zahediauthor=F.+Pouremamaliauthor=A.+Isazadehauthor=S.+Taefehshokrauthor=S.+Hajazimianauthor=N.+Serajiauthor=M.+Nouri&title=Targeting+cancer+stem+cells+by+melatonin%3A+effective+therapy+for+cancer+treatment&doi=10.1016%2Fj.prp.2020.152919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer stem cells by melatonin: Effective therapy for cancer treatment</span></div><div class="casAuthors">Maroufi, Nazila Fathi; Vahedian, Vahid; Hemati, Saideh; Rashidi, Mohammad-Reza; Akbarzadeh, Maryam; Zahedi, Mohammadsaeid; Pouremamali, Farhad; Isazadeh, Alireza; Taefehshokr, Sina; Hajazimian, Saba; Seraji, Nikou; Nouri, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Pathology, Research and Practice</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">216</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">152919</span>CODEN:
                <span class="NLM_cas:coden">PARPDS</span>;
        ISSN:<span class="NLM_cas:issn">0344-0338</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Melatonin is a physiol. hormone produced by the pineal gland.  In recent decades, enormous investigations showed that melatonin can prompt apoptosis in cancer cells and inhibit tumor metastasis and angiogenesis in variety of malignancies such as ovarian, melanoma, colon, and breast cancer; therefore, its possible therapeutic usage in cancer treatment was confirmed.  CSCs, which has received much attention from researchers in past decades, are major challenges in the treatment of cancer.  Because CSCs are resistant to chemotherapeutic drugs and cause recurrence of cancer and also have the ability to be regenerated; they can cause serious problems in the treatment of various cancers.  For these reasons, the researchers are trying to find a soln. to destroy these cells within the tumor mass.  In recent years, the effect of melatonin on CSCs has been investigated in some cancers.  Given the importance of CSCs in the process of cancer treatment, this article reviewed the studies conducted on the effect of melatonin on CSCs as a soln. to the problems caused by CSCs in the treatment of various cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_fmDcMvz0t7Vg90H21EOLACvtfcHk0lj7d7YGwm0gIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvFChsL0%253D&md5=0621003b2d1100c020602169959fe669</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.prp.2020.152919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.prp.2020.152919%26sid%3Dliteratum%253Aachs%26aulast%3DMaroufi%26aufirst%3DN.%2BF.%26aulast%3DVahedian%26aufirst%3DV.%26aulast%3DHemati%26aufirst%3DS.%26aulast%3DRashidi%26aufirst%3DM.%2BR.%26aulast%3DAkbarzadeh%26aufirst%3DM.%26aulast%3DZahedi%26aufirst%3DM.%26aulast%3DPouremamali%26aufirst%3DF.%26aulast%3DIsazadeh%26aufirst%3DA.%26aulast%3DTaefehshokr%26aufirst%3DS.%26aulast%3DHajazimian%26aufirst%3DS.%26aulast%3DSeraji%26aufirst%3DN.%26aulast%3DNouri%26aufirst%3DM.%26atitle%3DTargeting%2520cancer%2520stem%2520cells%2520by%2520melatonin%253A%2520effective%2520therapy%2520for%2520cancer%2520treatment%26jtitle%3DPathol.%252C%2520Res.%2520Pract.%26date%3D2020%26volume%3D216%26spage%3D152919%26epage%3D152926%26doi%3D10.1016%2Fj.prp.2020.152919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tamtaji, O. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirhosseini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnamfar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asemi, Z.</span></span> <span> </span><span class="NLM_article-title">Melatonin and pancreatic cancer: current knowledge and future perspectives</span>. <i>J. Cell. Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">5372</span>– <span class="NLM_lpage">5378</span>, <span class="refDoi"> DOI: 10.1002/jcp.27372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1002%2Fjcp.27372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30229898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslCjtbnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2019&pages=5372-5378&author=O.+R.+Tamtajiauthor=N.+Mirhosseiniauthor=R.+J.+Reiterauthor=M.+Behnamfarauthor=Z.+Asemi&title=Melatonin+and+pancreatic+cancer%3A+current+knowledge+and+future+perspectives&doi=10.1002%2Fjcp.27372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin and pancreatic cancer: Current knowledge and future perspectives</span></div><div class="casAuthors">Tamtaji, Omid Reza; Mirhosseini, Naghmeh; Reiter, Russel J.; Behnamfar, Morteza; Asemi, Zatollah</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Physiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5372-5378</span>CODEN:
                <span class="NLM_cas:coden">JCLLAX</span>;
        ISSN:<span class="NLM_cas:issn">0021-9541</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Pancreatic cancer has a high mortality rate due to the absence of early symptoms and subsequent late diagnosis; addnl., pancreatic cancer has a high resistance to radio- and chemotherapy.  Multiple inflammatory pathways are involved in the pathophysiol. of pancreatic cancer.  Melatonin an indoleamine produced in the pineal gland mediated and receptor-independent action is the pancreas and other where has both receptors.  Melatonin is a potent antioxidant and tissue protector against inflammation and oxidative stress.  In vivo and in vitro studies have shown that melatonin supplementation is an appropriate therapeutic approach for pancreatic cancer.  Melatonin may be an effective apoptosis inducer in cancer cells through regulation of a large no. of mol. pathways including oxidative stress, heat shock proteins, and vascular endothelial growth factor.  Limited clin. studies, however, have evaluated the role of melatonin in pancreatic cancer.  This review summarizes what is known regarding the effects of melatonin on pancreatic cancer and the mechanisms involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGwcyvLulZM7Vg90H21EOLACvtfcHk0lj7d7YGwm0gIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslCjtbnM&md5=27fe8cc751f9d3f174e1f751a6999f21</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1002%2Fjcp.27372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcp.27372%26sid%3Dliteratum%253Aachs%26aulast%3DTamtaji%26aufirst%3DO.%2BR.%26aulast%3DMirhosseini%26aufirst%3DN.%26aulast%3DReiter%26aufirst%3DR.%2BJ.%26aulast%3DBehnamfar%26aufirst%3DM.%26aulast%3DAsemi%26aufirst%3DZ.%26atitle%3DMelatonin%2520and%2520pancreatic%2520cancer%253A%2520current%2520knowledge%2520and%2520future%2520perspectives%26jtitle%3DJ.%2520Cell.%2520Physiol.%26date%3D2019%26volume%3D234%26spage%3D5372%26epage%3D5378%26doi%3D10.1002%2Fjcp.27372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shieh, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sytwu, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. W.</span></span> <span> </span><span class="NLM_article-title">Melatonin exerts anti-oral cancer effect via suppressing LSD1 in patient-derived tumor xenograft models</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">33756</span>– <span class="NLM_lpage">33769</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.16808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.18632%2Foncotarget.16808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28422711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BC1cvptFCisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=33756-33769&author=C.+Y.+Yangauthor=C.+K.+Linauthor=C.+H.+Tsaoauthor=C.+C.+Hsiehauthor=G.+J.+Linauthor=K.+H.+Maauthor=Y.+S.+Shiehauthor=H.+K.+Sytwuauthor=Y.+W.+Chen&title=Melatonin+exerts+anti-oral+cancer+effect+via+suppressing+LSD1+in+patient-derived+tumor+xenograft+models&doi=10.18632%2Foncotarget.16808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Melatonin exerts anti-oral cancer effect via suppressing LSD1 in patient-derived tumor xenograft models</span></div><div class="casAuthors">Yang Cheng-Yu; Sytwu Huey-Kang; Chen Yuan-Wu; Yang Cheng-Yu; Shieh Yi-Shing; Chen Yuan-Wu; Lin Chih-Kung; Tsao Chang-Huei; Sytwu Huey-Kang; Tsao Chang-Huei; Hsieh Cheng-Chih; Lin Gu-Jiun; Ma Kuo-Hsing; Chen Yuan-Wu</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">33756-33769</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aberrant activation of histone lysine-specific demethylase (LSD1) increases tumorigenicity; hence, LSD1 is considered a therapeutic target for various human cancers.  Although melatonin, an endogenously produced molecule, may defend against various cancers, the precise mechanism involved in its anti-oral cancer effect remains unclear.  Patient-derived tumor xenograft (PDTX) models are preclinical models that can more accurately reflect human tumor biology compared with cell line xenograft models.  Here, we evaluated the anticancer activity of melatonin by using LSD1-overexpressing oral cancer PDTX models.  By assessing oral squamous cell carcinoma (OSCC) tissue arrays through immunohistochemistry, we examined whether aberrant LSD1 overexpression in OSCC is associated with poor prognosis.  We also evaluated the action mechanism of melatonin against OSCC with lymphatic metastases by using the PDTX models.  Our results indicated that melatonin, at pharmacological concentrations, significantly suppresses cell proliferation in a dose- and time-dependent manner.  The observed suppression of proliferation was accompanied by the melatonin-mediated inhibition of LSD1 in oral cancer PDTXs and oral cancer cell lines.  In conclusion, we determined that the beneficial effects of melatonin in reducing oral cancer cell proliferation are associated with reduced LSD1 expression in vivo and in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSR9ND6-Hc_YBjv2lLSLOxFfW6udTcc2eaL7Tj8is3Tk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvptFCisA%253D%253D&md5=812cc98de3dc08051125329329a3a8d2</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.16808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.16808%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLin%26aufirst%3DC.%2BK.%26aulast%3DTsao%26aufirst%3DC.%2BH.%26aulast%3DHsieh%26aufirst%3DC.%2BC.%26aulast%3DLin%26aufirst%3DG.%2BJ.%26aulast%3DMa%26aufirst%3DK.%2BH.%26aulast%3DShieh%26aufirst%3DY.%2BS.%26aulast%3DSytwu%26aufirst%3DH.%2BK.%26aulast%3DChen%26aufirst%3DY.%2BW.%26atitle%3DMelatonin%2520exerts%2520anti-oral%2520cancer%2520effect%2520via%2520suppressing%2520LSD1%2520in%2520patient-derived%2520tumor%2520xenograft%2520models%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D33756%26epage%3D33769%26doi%3D10.18632%2Foncotarget.16808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xi, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2018.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29524666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkt1Grtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=7-16&author=J.+Y.+Xiauthor=S.+Y.+Xuauthor=L.+L.+Zhangauthor=X.+Y.+Biauthor=Y.+S.+Renauthor=Y.+C.+Liuauthor=Y.+Q.+Guauthor=Y.+G.+Xuauthor=F.+Lanauthor=X.+M.+Zha&title=Design%2C+synthesis+and+biological+activity+of+4-%284-benzyloxy%29phenoxypiperidines+as+selective+and+reversible+LSD1+inhibitors&doi=10.1016%2Fj.bioorg.2018.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors</span></div><div class="casAuthors">Xi, Jiayue; Xu, Siyuan; Zhang, Lulu; Bi, Xueyuan; Ren, Yanshen; Liu, Yu-Chih; Gu, Yueqing; Xu, Yungen; Lan, Fei; Zha, Xiaoming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7-16</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays a vital role in epigenetic regulation of gene activation and repression in several human cancers and is recognized as a promising antitumor therapeutic target.  In this paper, a series of 4-(4-benzyloxy)phenoxypiperidines were synthesized and evaluated.  Among the tested compds., compd. 10d (I) exhibited the potent and reversible inhibitory activity against LSD1 in vitro (IC50 = 4 μM).  Mol. docking was conducted to predict its binding mode.  Furthermore, 10d displayed it could inhibit migration of HCT-116 colon cancer cells and A549 lung cancer cells.  Taken together, 10d deserves further investigation as a hit-to-lead for the treatment of LSD1 assocd. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4X0tCTXSso7Vg90H21EOLACvtfcHk0ljAZ4gu5HKLvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkt1Grtr0%253D&md5=fb872c048079936b4d47edc4bf25cbdd</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DXi%26aufirst%3DJ.%2BY.%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DZhang%26aufirst%3DL.%2BL.%26aulast%3DBi%26aufirst%3DX.%2BY.%26aulast%3DRen%26aufirst%3DY.%2BS.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DGu%26aufirst%3DY.%2BQ.%26aulast%3DXu%26aufirst%3DY.%2BG.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DZha%26aufirst%3DX.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520activity%2520of%25204-%25284-benzyloxy%2529phenoxypiperidines%2520as%2520selective%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D78%26spage%3D7%26epage%3D16%26doi%3D10.1016%2Fj.bioorg.2018.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Ligand-based design, synthesis and biological evaluation of xanthine derivatives as LSD1/KDM1A inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2018.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30472603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit12ltLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2019&pages=555-567&author=Q.+S.+Maauthor=Y.+Yaoauthor=Y.+C.+Zhengauthor=S.+Fengauthor=J.+Changauthor=B.+Yuauthor=H.+M.+Liu&title=Ligand-based+design%2C+synthesis+and+biological+evaluation+of+xanthine+derivatives+as+LSD1%2FKDM1A+inhibitors&doi=10.1016%2Fj.ejmech.2018.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-based design, synthesis and biological evaluation of xanthine derivatives as LSD1/KDM1A inhibitors</span></div><div class="casAuthors">Ma, Qi-Sheng; Yao, Yongfang; Zheng, Yi-Chao; Feng, Siqi; Chang, Junbiao; Yu, Bin; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">555-567</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1, KDM1A) has been recognized as an important epigenetic target for disease treatment.  To date, a large no. of LSD1 inhibitors have been developed, some of which are currently being evaluated in clin. trials for the treatment of cancers, virus infection, and neurodegenerative diseases.  In this paper, we for the first time reported the ligand-based design of fragment-like xanthine derivs. as LSD1 inhibitors, of which compd. 4 possessed acceptable pharmacol. inhibition against LSD1 (IC50 = 6.45 μM) and favorable fragment-like nature, and therefore could be used as a promising template to design new LSD1 inhibitors.  Interestingly, compds. 6c and 6i strongly suppressed growth of MGC-803 cells partly dependent on their LSD1 inhibition, and were also found to be able to inhibit BRD4 and IDO1.  The docking studies were performed to rationalize the biochem. potency against LSD1 and to explain the obsd. activity discrepancy.  The proof-of-concept work may provide an example for other natural ligand-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNwIGUA6safrVg90H21EOLACvtfcHk0ljAZ4gu5HKLvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit12ltLvO&md5=3b4bd4668111aad864a366d55ab6c777</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DQ.%2BS.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DLigand-based%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520xanthine%2520derivatives%2520as%2520LSD1%252FKDM1A%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D162%26spage%3D555%26epage%3D567%26doi%3D10.1016%2Fj.ejmech.2018.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Furan carbonylhydrazones-derived elicitors that induce the resistance of rice to the brown planthopper Nilaparvata lugens</span>. <i>Phytochem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">184</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1016/j.phytol.2018.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.phytol.2018.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1SmsLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=184-189&author=X.+Chenauthor=X.+R.+Heauthor=W.+W.+Wangauthor=P.+Y.+Zhouauthor=Y.+G.+Louauthor=J.+Wu&title=Furan+carbonylhydrazones-derived+elicitors+that+induce+the+resistance+of+rice+to+the+brown+planthopper+Nilaparvata+lugens&doi=10.1016%2Fj.phytol.2018.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Furan carbonylhydrazones-derived elicitors that induce the resistance of rice to the brown planthopper Nilaparvata lugens</span></div><div class="casAuthors">Chen, Xia; He, Xingrui; Wang, Wanwan; Zhou, Pengyong; Lou, Yonggen; Wu, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Phytochemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">184-189</span>CODEN:
                <span class="NLM_cas:coden">PLHEBK</span>;
        ISSN:<span class="NLM_cas:issn">1874-3900</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">To obtain new elicitors that could induce plant defense against pests, a series of furan carbonylhydrazones were synthesized.  Phenotypic high-throughput screening of β-glucuronidase (GUS) activity was used to identify elicitors with strong activity.  Integrated bioassays were conducted to acquire chem. elicitors that induce resistance in rice to the brown planthopper Nilaparvata lugens and they are not direct toxic to the pest.  Our results demonstrate that these compds. have the potential to serve as new kinds of environment-friendly plant protection agents in rice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3EUTv63xn1rVg90H21EOLACvtfcHk0lhkcNRjjnLv7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1SmsLzF&md5=82372e1573a137e0f017afc2a29d0277</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.phytol.2018.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phytol.2018.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%2BR.%26aulast%3DWang%26aufirst%3DW.%2BW.%26aulast%3DZhou%26aufirst%3DP.%2BY.%26aulast%3DLou%26aufirst%3DY.%2BG.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DFuran%2520carbonylhydrazones-derived%2520elicitors%2520that%2520induce%2520the%2520resistance%2520of%2520rice%2520to%2520the%2520brown%2520planthopper%2520Nilaparvata%2520lugens%26jtitle%3DPhytochem.%2520Lett.%26date%3D2018%26volume%3D26%26spage%3D184%26epage%3D189%26doi%3D10.1016%2Fj.phytol.2018.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span> <span> </span><span class="NLM_article-title">Finding new elicitors that induce resistance in rice to the white-backed planthopper sogatella furcifera</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5601</span>– <span class="NLM_lpage">5603</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2015.10.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=26508551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslegu7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=5601-5603&author=X.+R.+Heauthor=Z.+N.+Yuauthor=S.+J.+Jiangauthor=P.+Z.+Zhangauthor=Z.+C.+Shangauthor=Y.+G.+Louauthor=J.+Wu&title=Finding+new+elicitors+that+induce+resistance+in+rice+to+the+white-backed+planthopper+sogatella+furcifera&doi=10.1016%2Fj.bmcl.2015.10.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Finding new elicitors that induce resistance in rice to the white-backed planthopper Sogatella furcifera</span></div><div class="casAuthors">He, Xingrui; Yu, Zhaonan; Jiang, Shaojie; Zhang, Peizhi; Shang, Zhicai; Lou, Yonggen; Wu, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5601-5603</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Herein we report a new way to identify chem. elicitors that induce resistance in rice to herbivores.  Using this method, by quantifying the induction of chems. for GUS activity in a specific screening system that we established previously, 5 candidate elicitors were selected from the 29 designed and synthesized phenoxyalkanoic acid derivs.  Bioassays confirmed that these candidate elicitors could induce plant defense and then repel feeding of white-backed planthopper Sogatella furcifera.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8WPKM2PNM7Vg90H21EOLACvtfcHk0lhkcNRjjnLv7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslegu7bO&md5=72683849ec089ea049906f74daeac615</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.041%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DX.%2BR.%26aulast%3DYu%26aufirst%3DZ.%2BN.%26aulast%3DJiang%26aufirst%3DS.%2BJ.%26aulast%3DZhang%26aufirst%3DP.%2BZ.%26aulast%3DShang%26aufirst%3DZ.%2BC.%26aulast%3DLou%26aufirst%3DY.%2BG.%26aulast%3DWu%26aufirst%3DJ.%26atitle%3DFinding%2520new%2520elicitors%2520that%2520induce%2520resistance%2520in%2520rice%2520to%2520the%2520white-backed%2520planthopper%2520sogatella%2520furcifera%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D5601%26epage%3D5603%26doi%3D10.1016%2Fj.bmcl.2015.10.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X. Y.</span></span> <span> </span><span class="NLM_article-title">4-Hydroxy-3-methylbenzofuran-2-carbohydrazones as novel LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127109</span>– <span class="NLM_lpage">127114</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2020.127109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32201021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1Cgs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127109-127114&author=X.+R.+Heauthor=Y.+Gaoauthor=Z.+Huiauthor=G.+D.+Shenauthor=S.+Wangauthor=T.+Xieauthor=X.+Y.+Ye&title=4-Hydroxy-3-methylbenzofuran-2-carbohydrazones+as+novel+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2020.127109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">4-Hydroxy-3-methylbenzofuran-2-carbohydrazones as novel LSD1 inhibitors</span></div><div class="casAuthors">He, Xingrui; Gao, Yuan; Hui, Zi; Shen, Guodong; Wang, Shuo; Xie, Tian; Ye, Xiang-Yang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">127109</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1 or KDM1A) is a potential therapeutic target in oncol. due to its overexpression in various human tumors.  The authors report herein a new class of benzofuran acylhydrazones as potent LSD1 inhibitors.  Among the 31 compds. prepd., 14 compds. exhibited excellent LSD1 inhibitory activity with IC50 values ranging from 7.2 to 68.8 nM.  In cellular assays, several compds. inhibited the proliferations of various cancer cell lines, including PC-3, MCG-<803≥, U87 MG, PANC-1, HT-29 and MCF-7.  This opens up the opportunity for further optimization and study of this class compds. for potential cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWwiTYTNJDRbVg90H21EOLACvtfcHk0lhkcNRjjnLv7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1Cgs7g%253D&md5=aef250e0011154d944a3aae164f14619</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127109%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DX.%2BR.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DHui%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DG.%2BD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DX.%2BY.%26atitle%3D4-Hydroxy-3-methylbenzofuran-2-carbohydrazones%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127109%26epage%3D127114%26doi%3D10.1016%2Fj.bmcl.2020.127109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amigoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legnaghi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romussi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thaler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultaggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrugno, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessanti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagarri, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carettoni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iuzzolino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vianello, P.</span></span> <span> </span><span class="NLM_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as new reversible inhibitors of histone lysine demethylase KDM1A/LSD1. Part 1: High-throughput screening and preliminary exploration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1673</span>– <span class="NLM_lpage">1692</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisVOqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1673-1692&author=L.+Sartoriauthor=C.+Mercurioauthor=F.+Amigoniauthor=A.+Cappaauthor=G.+Fagaauthor=R.+Fattoriauthor=E.+Legnaghiauthor=G.+Ciossaniauthor=A.+Matteviauthor=G.+Meroniauthor=L.+Morettiauthor=V.+Cecatielloauthor=S.+Pasqualatoauthor=A.+Romussiauthor=F.+Thalerauthor=P.+Trifiroauthor=M.+Villaauthor=S.+Vultaggioauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=E.+Zagarriauthor=D.+Carettoniauthor=L.+Iuzzolinoauthor=M.+Varasiauthor=P.+Vianello&title=Thieno%5B3%2C2-b%5Dpyrrole-5-carboxamides+as+new+reversible+inhibitors+of+histone+lysine+demethylase+KDM1A%2FLSD1.+Part+1%3A+High-throughput+screening+and+preliminary+exploration&doi=10.1021%2Facs.jmedchem.6b01018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration</span></div><div class="casAuthors">Sartori, Luca; Mercurio, Ciro; Amigoni, Federica; Cappa, Anna; Faga, Giovanni; Fattori, Raimondo; Legnaghi, Elena; Ciossani, Giuseppe; Mattevi, Andrea; Meroni, Giuseppe; Moretti, Loris; Cecatiello, Valentina; Pasqualato, Sebastiano; Romussi, Alessia; Thaler, Florian; Trifiro, Paolo; Villa, Manuela; Vultaggio, Stefania; Botrugno, Oronza A.; Dessanti, Paola; Minucci, Saverio; Zagarri, Elisa; Carettoni, Daniele; Iuzzolino, Lucia; Varasi, Mario; Vianello, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1673-1692</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 KDM1A (LSD1) is one regulator of histone methylation and it is increasingly recognized as a potential therapeutic target in oncol.  The authors report on a high-throughput screening campaign performed on KDM1A/CoREST, using a time resolved fluorescence resonance energy transfer (TR-FRET) technol., to identify reversible inhibitors.  The screening led to 115 hits for which the authors detd. biochem. IC50, thus identifying 4 chem. series.  After data anal., the authors have prioritized the chem. series of N-phenyl-4H-thieno[3,2-b]pyrrole-5-carboxamide for which the authors obtained x-ray structures of the most potent hit (compd. 19, IC50 = 2.9 μM) in complex with the enzyme.  Initial expansion of this chem. class, both modifying core structure and decorating benzamide moiety, was directed towards the definition of the moieties responsible for the interaction with the enzyme.  Preliminary optimization brought to compd. 90 (4-methyl-N-[3-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]- thieno[3,2-b]pyrrole-5-carboxamide) which inhibited the enzyme with a submicromolar IC50 (0.162 μM), capable to inhibit the target in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3SWpFkenrJLVg90H21EOLACvtfcHk0lj7qsUXCZbDSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisVOqt7o%253D&md5=e3ad614cffd844ea9ca1da1747f6aa2d</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26aulast%3DSartori%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DC.%26aulast%3DAmigoni%26aufirst%3DF.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DFaga%26aufirst%3DG.%26aulast%3DFattori%26aufirst%3DR.%26aulast%3DLegnaghi%26aufirst%3DE.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DPasqualato%26aufirst%3DS.%26aulast%3DRomussi%26aufirst%3DA.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTrifiro%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVultaggio%26aufirst%3DS.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DZagarri%26aufirst%3DE.%26aulast%3DCarettoni%26aufirst%3DD.%26aulast%3DIuzzolino%26aufirst%3DL.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DVianello%26aufirst%3DP.%26atitle%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2520as%2520new%2520reversible%2520inhibitors%2520of%2520histone%2520lysine%2520demethylase%2520KDM1A%252FLSD1.%2520Part%25201%253A%2520High-throughput%2520screening%2520and%2520preliminary%2520exploration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1673%26epage%3D1692%26doi%3D10.1021%2Facs.jmedchem.6b01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vianello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amigoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagá, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legnaghi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciossani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meroni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romussi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thaler, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifiró, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botrugno, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessanti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultaggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagarrí, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, C.</span></span> <span> </span><span class="NLM_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as new reversible inhibitors of histone lysine demethylase KDM1A/LSD1. Part 2: Structure-based drug design and structure-activity relationship</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1693</span>– <span class="NLM_lpage">1715</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisVOkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1693-1715&author=P.+Vianelloauthor=L.+Sartoriauthor=F.+Amigoniauthor=A.+Cappaauthor=G.+Fag%C3%A1author=R.+Fattoriauthor=E.+Legnaghiauthor=G.+Ciossaniauthor=A.+Matteviauthor=G.+Meroniauthor=L.+Morettiauthor=V.+Cecatielloauthor=S.+Pasqualatoauthor=A.+Romussiauthor=F.+Thalerauthor=P.+Trifir%C3%B3author=M.+Villaauthor=O.+A.+Botrugnoauthor=P.+Dessantiauthor=S.+Minucciauthor=S.+Vultaggioauthor=E.+Zagarr%C3%ADauthor=M.+Varasiauthor=C.+Mercurio&title=Thieno%5B3%2C2-b%5Dpyrrole-5-carboxamides+as+new+reversible+inhibitors+of+histone+lysine+demethylase+KDM1A%2FLSD1.+Part+2%3A+Structure-based+drug+design+and+structure-activity+relationship&doi=10.1021%2Facs.jmedchem.6b01019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure-Activity Relationship</span></div><div class="casAuthors">Vianello, Paola; Sartori, Luca; Amigoni, Federica; Cappa, Anna; Faga, Giovanni; Fattori, Raimondo; Legnaghi, Elena; Ciossani, Giuseppe; Mattevi, Andrea; Meroni, Giuseppe; Moretti, Loris; Cecatiello, Valentina; Pasqualato, Sebastiano; Romussi, Alessia; Thaler, Florian; Trifiro, Paolo; Villa, Manuela; Botrugno, Oronza A.; Dessanti, Paola; Minucci, Saverio; Vultaggio, Stefania; Zagarri, Elisa; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1693-1715</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The balance of methylation levels at histone H3 lysine 4 (H3K4) is regulated by KDM1A (LSD1).  KDM1A is overexpressed in several tumor types, thus representing an emerging target for the development of novel cancer therapeutics.  The authors have previously described the identification of thieno[3,2-b]pyrrole-5-carboxamides, as novel reversible inhibitors of KDM1A, whose preliminary exploration resulted in compd. (I) with biochem. IC50 = 160 nM.  The authors now report the structure-guided optimization of this chem. series, based on multiple ligand/KDM1A-CoRest co-crystal structures, which led to several extremely potent inhibitors.  In particular, compds. N-[3-(methoxymethyl)-2-[[4-[[(3R)-pyrrolidin-3-yl]- methoxy]phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-b]-pyrrole-5-carboxamide hydrochloride (46), 4-ethyl-N-[3-(methoxymethyl)-2-[[4-[[(3R)-pyrrolidin-3-yl]-methoxy]phenoxy]methyl]phenyl]thieno[3,2-b]pyrrole-5-carboxamide (49) and N-[3-(ethoxymethyl)-2-[[4-[[(3R)-pyrrolidin-3-yl]methoxy]-phenoxy]methyl]phenyl]-4-methyl-thieno[3,2-b]pyrrole-5-carboxamide hydrochloride (50) showed single digit nanomolar IC50 values for in vitro inhibition of KDM1A, with high selectivity in secondary assays.  In THP-1 cells these compds. transcriptionally affected the expression of genes regulated by KDM1A, such as CD14, CD11b, and CD86.  Moreover, 49 and 50 showed a remarkable anti-clonogenic cell growth effect on MLL-AF9 human leukemia cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmOBI9faVjBrVg90H21EOLACvtfcHk0lj7qsUXCZbDSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisVOkur8%253D&md5=c4713516ee7b832817c386717c0bd71f</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DSartori%26aufirst%3DL.%26aulast%3DAmigoni%26aufirst%3DF.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DFag%25C3%25A1%26aufirst%3DG.%26aulast%3DFattori%26aufirst%3DR.%26aulast%3DLegnaghi%26aufirst%3DE.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DPasqualato%26aufirst%3DS.%26aulast%3DRomussi%26aufirst%3DA.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTrifir%25C3%25B3%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DVultaggio%26aufirst%3DS.%26aulast%3DZagarr%25C3%25AD%26aufirst%3DE.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2520as%2520new%2520reversible%2520inhibitors%2520of%2520histone%2520lysine%2520demethylase%2520KDM1A%252FLSD1.%2520Part%25202%253A%2520Structure-based%2520drug%2520design%2520and%2520structure-activity%2520relationship%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1693%26epage%3D1715%26doi%3D10.1021%2Facs.jmedchem.6b01019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romussi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cappa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vianello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brambillasca, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cera, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dal Zuffo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fagà, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifirò, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultaggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecatiello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dondio, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">So, C. W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of reversible inhibitors of KDM1A efficacious in acute myeloid leukemia models</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">754</span>– <span class="NLM_lpage">759</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00604</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00604" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivFyisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=754-759&author=A.+Romussiauthor=A.+Cappaauthor=P.+Vianelloauthor=S.+Brambillascaauthor=M.+R.+Ceraauthor=R.+Dal+Zuffoauthor=G.+Fag%C3%A0author=R.+Fattoriauthor=L.+Morettiauthor=P.+Trifir%C3%B2author=M.+Villaauthor=S.+Vultaggioauthor=V.+Cecatielloauthor=S.+Pasqualatoauthor=G.+Dondioauthor=C.+W.+E.+Soauthor=S.+Minucciauthor=L.+Sartoriauthor=M.+Varasiauthor=C.+Mercurio&title=Discovery+of+reversible+inhibitors+of+KDM1A+efficacious+in+acute+myeloid+leukemia+models&doi=10.1021%2Facsmedchemlett.9b00604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Reversible Inhibitors of KDM1A Efficacious in Acute Myeloid Leukemia Models</span></div><div class="casAuthors">Romussi, Alessia; Cappa, Anna; Vianello, Paola; Brambillasca, Silvia; Cera, Maria Rosaria; Dal Zuffo, Roberto; Faga, Giovanni; Fattori, Raimondo; Moretti, Loris; Trifiro, Paolo; Villa, Manuela; Vultaggio, Stefania; Cecatiello, Valentina; Pasqualato, Sebastiano; Dondio, Giulio; So, Chi Wai Eric; Minucci, Saverio; Sartori, Luca; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">754-759</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1 or KDM1A) is a FAD-dependent enzyme that acts as a transcription corepressor or coactivator by regulating the methylation status of histone H3 lysines K4 and K9, resp.  KDM1A represents an attractive target for cancer therapy.  While, in the past, the main medicinal chem. strategy toward KDM1A inhibition was based on the optimization of ligands that irreversibly bind the FAD cofactor within the enzyme catalytic site, we and others have also identified reversible inhibitors.  Herein we reported the discovery of 5-imidazolylthieno[3,2-b]pyrroles, a new series of KDM1A inhibitors endowed with picomolar inhibitory potency, active in cells and efficacious after oral administration in murine leukemia models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiYpo0iHtNZLVg90H21EOLACvtfcHk0lj7qsUXCZbDSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivFyisL0%253D&md5=638d1b2f8653b1fa53f8d6278e297553</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00604%26sid%3Dliteratum%253Aachs%26aulast%3DRomussi%26aufirst%3DA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBrambillasca%26aufirst%3DS.%26aulast%3DCera%26aufirst%3DM.%2BR.%26aulast%3DDal%2BZuffo%26aufirst%3DR.%26aulast%3DFag%25C3%25A0%26aufirst%3DG.%26aulast%3DFattori%26aufirst%3DR.%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DTrifir%25C3%25B2%26aufirst%3DP.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVultaggio%26aufirst%3DS.%26aulast%3DCecatiello%26aufirst%3DV.%26aulast%3DPasqualato%26aufirst%3DS.%26aulast%3DDondio%26aufirst%3DG.%26aulast%3DSo%26aufirst%3DC.%2BW.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DSartori%26aufirst%3DL.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520reversible%2520inhibitors%2520of%2520KDM1A%2520efficacious%2520in%2520acute%2520myeloid%2520leukemia%2520models%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D754%26epage%3D759%26doi%3D10.1021%2Facsmedchemlett.9b00604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazeldine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachaiyappan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbergs, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotarski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woster, P. M.</span></span> <span> </span><span class="NLM_article-title">Low molecular weight amidoximes that act as potent inhibitors of lysine-specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7378</span>– <span class="NLM_lpage">7391</span>, <span class="refDoi"> DOI: 10.1021/jm3002845</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3002845" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKnsrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7378-7391&author=S.+Hazeldineauthor=B.+Pachaiyappanauthor=N.+Steinbergsauthor=S.+Nowotarskiauthor=A.+S.+Hansonauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=Low+molecular+weight+amidoximes+that+act+as+potent+inhibitors+of+lysine-specific+demethylase+1&doi=10.1021%2Fjm3002845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Low Molecular Weight Amidoximes that Act as Potent Inhibitors of Lysine-Specific Demethylase 1</span></div><div class="casAuthors">Hazeldine, Stuart; Pachaiyappan, Boobalan; Steinbergs, Nora; Nowotarski, Shannon; Hanson, Allison S.; Casero, Robert A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7378-7391</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently discovered enzyme lysine-specific demethylase 1 (LSD1) plays an important role in the epigenetic control of gene expression, and aberrant gene silencing secondary to LSD1 dysregulation is thought to contribute to the development of cancer.  We reported that (bis)guanidines, (bis)biguanides, and their urea- and thiourea isosteres are potent inhibitors of LSD1 and induce the re-expression of aberrantly silenced tumor suppressor genes in tumor cells in vitro.  We now report a series of small mol. amidoximes that are moderate inhibitors of recombinant LSD1 but that produce dramatic changes in methylation at the histone 3 lysine 4 (H3K4) chromatin mark, a specific target of LSD1, in Calu-6 lung carcinoma cells.  In addn., these analogs increase cellular levels of secreted frizzle-related protein (SFRP) 2, H-cadherin (HCAD), and the transcription factor GATA4.  These compds. represent leads for an important new series of drug-like epigenetic modulators with the potential for use as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVxPPqT5KXUrVg90H21EOLACvtfcHk0lgcZcc2y8pufQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKnsrvJ&md5=e3a9936ea336fe333cc1cbd0056c11c2</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fjm3002845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3002845%26sid%3Dliteratum%253Aachs%26aulast%3DHazeldine%26aufirst%3DS.%26aulast%3DPachaiyappan%26aufirst%3DB.%26aulast%3DSteinbergs%26aufirst%3DN.%26aulast%3DNowotarski%26aufirst%3DS.%26aulast%3DHanson%26aufirst%3DA.%2BS.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3DLow%2520molecular%2520weight%2520amidoximes%2520that%2520act%2520as%2520potent%2520inhibitors%2520of%2520lysine-specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7378%26epage%3D7391%26doi%3D10.1021%2Fjm3002845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F. J.</span></span> <span> </span><span class="NLM_article-title">Natural polyphenols inhibit lysine-specific demethylase-1 in vitro</span>. <i>J. Biochem. Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">63</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=23662249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A280%3ADC%252BC2srntVelsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2013&pages=56-63&author=A.+Abdullaauthor=X.+P.+Zhaoauthor=F.+J.+Yang&title=Natural+polyphenols+inhibit+lysine-specific+demethylase-1+in+vitro"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Polyphenols Inhibit Lysine-Specific Demethylase-1 in vitro</span></div><div class="casAuthors">Abdulla Arian; Zhao Xiaoping; Yang Fajun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biochemical and pharmacological research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-63</span>
        ISSN:<span class="NLM_cas:issn">2168-8761</span>.
    </div><div class="casAbstract">Natural polyphenols, such as resveratrol, have beneficial functions on major human diseases such as cancer, diabetes, and cardiovascular disease.  Besides acting as antioxidants, some of these polyphenols can also target proteins to modulate specific biological pathways.  The lysine-specific histone demethylase LSD1 plays important roles in cell growth, differentiation and nutrient metabolism.  Here, we studied the effect of natural polyphenols resveratrol, curcumin, quercetin and analogs on LSD1.  Using in vitro LSD1 enzymatic assays, we show that resveratrol, curcumin and quercetin displayed a potent inhibitory effect on the LSD1 activity and were more potent than the known LSD1 inhibitor trans-2-phenylcyclopropylamine (TCP).  The new function of resveratrol, curcumin and quercetin is independent of their antioxidant properties, as other antioxidants had no effect on LSD1 under the similar conditions.  In C2C12 fibroblasts, resveratrol and curcumin can efficiently inhibit myogenic expression and differentiation, for which LSD1 is required.  Thus, our study has identified LSD1 as a novel target of bioactive natural compounds, such as resveratrol, curcumin and quercetin, and such finding suggests that LSD1 inhibition can at least partially contribute to some of the previously observed beneficial effects of these compounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTetSe7hlvX8kHdcurJClFcfW6udTcc2eb82ogFzaozZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srntVelsg%253D%253D&md5=4a0007d63d999f4f20d4830b781ab1ed</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbdulla%26aufirst%3DA.%26aulast%3DZhao%26aufirst%3DX.%2BP.%26aulast%3DYang%26aufirst%3DF.%2BJ.%26atitle%3DNatural%2520polyphenols%2520inhibit%2520lysine-specific%2520demethylase-1%2520in%2520vitro%26jtitle%3DJ.%2520Biochem.%2520Pharmacol.%2520Res.%26date%3D2013%26volume%3D1%26spage%3D56%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of resveratrol derivatives as novel LSD1 inhibitors: design, synthesis and their biological evaluation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">246</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2016.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27888721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGqsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2017&pages=246-258&author=Y.+C.+Duanauthor=Y.+Y.+Guanauthor=X.+Y.+Zhaiauthor=L.+N.+Dingauthor=W.+P.+Qinauthor=D.+D.+Shenauthor=X.+Q.+Liuauthor=X.+D.+Sunauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Discovery+of+resveratrol+derivatives+as+novel+LSD1+inhibitors%3A+design%2C+synthesis+and+their+biological+evaluation&doi=10.1016%2Fj.ejmech.2016.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of resveratrol derivatives as novel LSD1 inhibitors: Design, synthesis and their biological evaluation</span></div><div class="casAuthors">Duan, Ying-Chao; Guan, Yuan-Yuan; Zhai, Xiao-Yu; Ding, Li-Na; Qin, Wen-Ping; Shen, Dan-Dan; Liu, Xue-Qi; Sun, Xu-Dong; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">246-258</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Inhibition of lysine-specific demethylase 1 (LSD1) has recently emerged as an attractive therapeutic target for treating cancer and other diseases.  As a continuity of our ongoing effort to identify novel small-mol. LSD1-inhibitors, we designed and synthesized a series of resveratrol derivs., which were shown to be potent inhibitors of LSD1.  Among them, compds. I and II displayed the most potent LSD1-inhibitory activities in enzyme assays, with IC50 values of 121 nM and 123 nM, resp.  Biochem. study and docking anal. indicated that compds. I and II were reversible LSD1 inhibitors.  High content anal. showed that I and II induced a dose-dependent increase of dimethylated Lys4 of histone H3 and had no impact on the expression of LSD1 in MGC-803 cells.  Furthermore, I or II could remarkably increase the mRNA level of CD86, a surrogate cellular biomarker for LSD1 activity, in MGC-803 cells, suggesting that they are likely to exhibit LSD1-inhibitory activities intracellularly.  These findings should encourage further modification of these compds. to produce more potent LSD1 inhibitors with potential anticancer activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWLdv3p7rUK7Vg90H21EOLACvtfcHk0lgcZcc2y8pufQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGqsrbN&md5=812cb448167dda2f2a2f69b269a4c594</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DGuan%26aufirst%3DY.%2BY.%26aulast%3DZhai%26aufirst%3DX.%2BY.%26aulast%3DDing%26aufirst%3DL.%2BN.%26aulast%3DQin%26aufirst%3DW.%2BP.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DLiu%26aufirst%3DX.%2BQ.%26aulast%3DSun%26aufirst%3DX.%2BD.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520resveratrol%2520derivatives%2520as%2520novel%2520LSD1%2520inhibitors%253A%2520design%252C%2520synthesis%2520and%2520their%2520biological%2520evaluation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D126%26spage%3D246%26epage%3D258%26doi%3D10.1016%2Fj.ejmech.2016.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">6000</span>– <span class="NLM_lpage">6014</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.10.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2018.10.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30448189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1WltrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=6000-6014&author=Y.+C.+Duanauthor=W.+P.+Qinauthor=F.+Z.+Suoauthor=X.+Y.+Zhaiauthor=Y.+Y.+Guanauthor=X.+J.+Wangauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Design%2C+synthesis+and+in+vitro+evaluation+of+stilbene+derivatives+as+novel+LSD1+inhibitors+for+AML+therapy&doi=10.1016%2Fj.bmc.2018.10.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy</span></div><div class="casAuthors">Duan, Yingchao; Qin, Wenping; Suo, Fengzhi; Zhai, Xiaoyu; Guan, Yuanyuan; Wang, Xiaojuan; Zheng, Yichao; Liu, Hongmin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">6000-6014</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">LSD1 is implicated in a no. of malignancies and has emerged as an exciting target.  As part of our sustained efforts to develop novel reversible LSD1 inhibitors for epigenetic therapy of cancers, in this study, we reported a series of stilbene derivs. and evaluated their LSD1 inhibitory activities, obtaining several compds. as potent LSD1 inhibitors with IC50 values in submicromolar range.  Enzyme kinetics studies and SPR assay suggested that compd. 8c, the most active LSD1 inhibitor (IC50 = 283 nM), potently inhibited LSD1 in a reversible and FAD competitive manner.  Consistent with the kinetics data, mol. docking showed that compd. 8c can be well docked into the FAD binding site of LSD1.  Flow cytometry anal. showed that compd. 8c was capable of up-regulating the expression of the surrogate cellular biomarker CD86 in THP-1 human leukemia cells, suggesting the ability to block LSD1 activity in cells.  Compd. 8c showed good inhibition against THP-1 and MOLM-13 cells with IC50 values of 5.76 and 8.34 μM, resp.  Moreover, compd. 8c significantly inhibited colony formation of THP-1 cells dose dependently.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCQHvTfU4lSbVg90H21EOLACvtfcHk0lgRtuECk5e41w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1WltrjJ&md5=989826c422d025071b5533a8be86edae</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.10.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.10.037%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DQin%26aufirst%3DW.%2BP.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DZhai%26aufirst%3DX.%2BY.%26aulast%3DGuan%26aufirst%3DY.%2BY.%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520in%2520vitro%2520evaluation%2520of%2520stilbene%2520derivatives%2520as%2520novel%2520LSD1%2520inhibitors%2520for%2520AML%2520therapy%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D6000%26epage%3D6014%26doi%3D10.1016%2Fj.bmc.2018.10.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M. S.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of curcumin analogues as novel LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">126683</span>– <span class="NLM_lpage">126688</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.126683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2019.126683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=31627991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2gsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=126683-126688&author=J.+M.+Wangauthor=X.+Y.+Zhangauthor=J.+K.+Yanauthor=W.+Liauthor=Q.+W.+Jiangauthor=X.+R.+Wangauthor=D.+M.+Zhaoauthor=M.+S.+Cheng&title=Design%2C+synthesis+and+biological+evaluation+of+curcumin+analogues+as+novel+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2019.126683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of curcumin analogues as novel LSD1 inhibitors</span></div><div class="casAuthors">Wang, Jiming; Zhang, Xiangyu; Yan, Jiangkun; Li, Wei; Jiang, Qinwen; Wang, Xinran; Zhao, Dongmei; Cheng, Maosheng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">126683</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) was the first discovered histone demethylase.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development, and thus, it is an attractive mol. target for the development of novel cancer therapeutics.  In this study, we worked on the structural optimization of natural products and identified 30 novel LSD1 inhibitors.  Utilizing a structure-based drug design strategy, we designed and synthesized a series of curcumin analogs that were shown to be potent LSD1 inhibitors in the enzyme assay.  Compd. WB07 (I) displayed the most potent LSD1 inhibitory activity, with an IC50 value of 0.8μM.  Moreover, WA20 (II) showed an anticlonogenic effect on A549 cells with an IC50 value of 4.4μM.  Mol. docking simulations were also carried out, and the results indicated that the inhibitors bound to the protein active site located around the key residues of Asp555 and Asp556.  These findings suggested that compds. II and I are the first curcumin analog-based LSD1 inhibitors with remarkable A549 suppressive activity, providing a novel scaffold for the development of LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTnlsmI6uw3LVg90H21EOLACvtfcHk0lgRtuECk5e41w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2gsrzF&md5=2c6688ce5e03fcaee6f0634423523f41</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126683%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DZhang%26aufirst%3DX.%2BY.%26aulast%3DYan%26aufirst%3DJ.%2BK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DQ.%2BW.%26aulast%3DWang%26aufirst%3DX.%2BR.%26aulast%3DZhao%26aufirst%3DD.%2BM.%26aulast%3DCheng%26aufirst%3DM.%2BS.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520curcumin%2520analogues%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D126683%26epage%3D126688%26doi%3D10.1016%2Fj.bmcl.2019.126683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span> <span> </span><span class="NLM_article-title">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4871</span>– <span class="NLM_lpage">4880</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.08.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmc.2018.08.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30153955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=4871-4880&author=S.+Y.+Xuauthor=C.+Zhouauthor=R.+F.+Liuauthor=Q.+H.+Zhuauthor=Y.+G.+Xuauthor=F.+Lanauthor=X.+M.+Zha&title=Optimization+of+5-arylidene+barbiturates+as+potent%2C+selective%2C+reversible+LSD1+inhibitors+for+the+treatment+of+acute+promyelocytic+leukemia&doi=10.1016%2Fj.bmc.2018.08.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia</span></div><div class="casAuthors">Xu, Siyuan; Zhou, Chen; Liu, Rongfeng; Zhu, Qihua; Xu, Yungen; Lan, Fei; Zha, Xiaoming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4871-4880</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) is overexpressed in diverse hematol. disorders and recognized as a promising target for blood medicines.  In this study, mol. docking-based virtual screening united with bioevaluation was utilized to identify novel skeleton of 5-arylidene barbiturate as small-mol. inhibitors of LSD1.  Among the synthesized derivs., I exhibited reversible and potent inhibition (IC50 = 0.41 μM) and high selectivity over the MAO-A and MAO-B.  Notably, I strongly induced differentiation effect on acute promyelocytic leukemia NB4 cell line and distinctly escalated the methylation level on histone 3 lysine 4 (H3K4).  Our findings indicate that 5-arylidene barbiturate may represent a new skeleton of LSD1 inhibitors and I deserve as a promising agent for the further research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbUIvf5kGjSbVg90H21EOLACvtfcHk0lgRtuECk5e41w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2jtr3F&md5=27a2e2ea365f97dba3279fee15d65a2e</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.08.026%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DZhu%26aufirst%3DQ.%2BH.%26aulast%3DXu%26aufirst%3DY.%2BG.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DZha%26aufirst%3DX.%2BM.%26atitle%3DOptimization%2520of%25205-arylidene%2520barbiturates%2520as%2520potent%252C%2520selective%252C%2520reversible%2520LSD1%2520inhibitors%2520for%2520the%2520treatment%2520of%2520acute%2520promyelocytic%2520leukemia%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D4871%26epage%3D4880%26doi%3D10.1016%2Fj.bmc.2018.08.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of orally active chalcones as histone lysine specific demethylase 1 inhibitors for the treatment of leukaemia</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">217</span>, <span class="refDoi"> DOI: 10.1080/14756366.2020.1852556</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1080%2F14756366.2020.1852556" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=33307878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFyqurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2021&pages=207-217&author=Y.+Liauthor=Y.+Sunauthor=Y.+Zhouauthor=X.+Y.+Liauthor=H.+Zhangauthor=G.+J.+Zhang&title=Discovery+of+orally+active+chalcones+as+histone+lysine+specific+demethylase+1+inhibitors+for+the+treatment+of+leukaemia&doi=10.1080%2F14756366.2020.1852556"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of orally active chalcones as histone lysine specific demethylase 1 inhibitors for the treatment of leukaemia</span></div><div class="casAuthors">Li, Yang; Sun, Ying; Zhou, Yang; Li, Xinyang; Zhang, Huan; Zhang, Guojun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">207-217</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Histone lysine specific demethylase 1 (LSD1) has emerged as an attractive mol. target for the discovery of potently anticancer drugs to treat leukemia.  In this study, a series of novel chalcone derivs. were designed, synthesized and evaluated for their inhibitory activities against LSD1 in vitro.  Among all these compds., displayed the best LSD1 inhibitory activity with an IC50 value of 0.14 μM.  In the cellular level, compd. can induce the accumulation of H3K9me1/2 and inhibit cell proliferation by inactivating LSD1.  It exhibited the potent antiproliferative activity with IC50 values of 1.10 μM, 3.64 μM, 3.85 μM, 1.87 μM, 0.87 μM and 2.73 μM against HAL-01, KE-37, P30-OHK, SUP-B15, MOLT-4 and LC4-1 cells, resp.  Importantly, compd. significantly suppressed MOLT-4 xenograft tumor growth in vivo, indicating its great potential as an orally bioavailable candidate for leukemia therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLsxSVulKy4LVg90H21EOLACvtfcHk0lhWmPJq4mTNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFyqurzL&md5=673f155e29c88f16e77fb8da756b0a47</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1080%2F14756366.2020.1852556&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14756366.2020.1852556%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%2BY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DG.%2BJ.%26atitle%3DDiscovery%2520of%2520orally%2520active%2520chalcones%2520as%2520histone%2520lysine%2520specific%2520demethylase%25201%2520inhibitors%2520for%2520the%2520treatment%2520of%2520leukaemia%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2021%26volume%3D36%26spage%3D207%26epage%3D217%26doi%3D10.1080%2F14756366.2020.1852556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sorna, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theisen, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bearss, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span> <span> </span><span class="NLM_article-title">High-throughput virtual screening identifies novel N’-(1-phenylethylidene)-benzohydrazides as potent, specific, and reversible LSD1 inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9496</span>– <span class="NLM_lpage">9508</span>, <span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-throughput+virtual+screening+identifies+novel+N%E2%80%99-%281-phenylethylidene%29-benzohydrazides+as+potent%2C+specific%2C+and+reversible+LSD1+inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0lhWmPJq4mTNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520identifies%2520novel%2520N%25E2%2580%2599-%25281-phenylethylidene%2529-benzohydrazides%2520as%2520potent%252C%2520specific%252C%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soldi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh Halder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thode, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drenner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaadige, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel Ampanattu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez Del Villar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, W. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span> <span> </span><span class="NLM_article-title">The novel reversible LSD1 inhibitor SP-2577 promotes anti-tumor immunity in SWItch/Sucrose-NonFermentable (SWI/SNF) complex mutated ovarian cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e0235705</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0235705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1371%2Fjournal.pone.0235705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32649682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVejurvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&author=R.+Soldiauthor=T.+Ghosh+Halderauthor=A.+Westonauthor=T.+Thodeauthor=K.+Drennerauthor=R.+Lewisauthor=M.+R.+Kaadigeauthor=S.+Srivastavaauthor=S.+Daniel+Ampanattuauthor=R.+Rodriguez+Del+Villarauthor=J.+Langauthor=H.+Vankayalapatiauthor=B.+Weissmanauthor=J.+M.+Trentauthor=W.+P.+D.+Hendricksauthor=S.+Sharma&title=The+novel+reversible+LSD1+inhibitor+SP-2577+promotes+anti-tumor+immunity+in+SWItch%2FSucrose-NonFermentable+%28SWI%2FSNF%29+complex+mutated+ovarian+cancer&doi=10.1371%2Fjournal.pone.0235705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">The novel reversible LSD1 inhibitor SP-2577 promotes anti-tumor immunity in SWItch/Sucrose-NonFermentable (SWI/SNF) complex mutated ovarian cancer</span></div><div class="casAuthors">Soldi, Raffaella; Ghosh Halder, Tithi; Weston, Alexis; Thode, Trason; Drenner, Kevin; Lewis, Rhonda; Kaadige, Mohan R.; Srivastava, Shreyesi; Daniel Ampanattu, Sherin; Rodriguez del Villar, Ryan; Lang, Jessica; Vankayalapati, Hariprasad; Weissman, Bernard; Trent, Jeffrey M.; Hendricks, William P. D.; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0235705</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Mutations of the SWI/SNF chromatin remodeling complex occur in 20% of all human cancers, including ovarian cancer.  Approx. half of ovarian clear cell carcinomas carry mutations in the SWI/SNF subunit ARID1A, while small cell carcinoma of the ovary hypercalcemic type presents with inactivating mutations of the SWI/SNF ATPase SMARCA4 alongside epigenetic silencing of the ATPase SMARCA2.  Loss of these ATPases disrupts SWI/SNF chromatin remodeling activity and may also interfere with the function of other histone-modifying enzymes that assoc. with or are dependent on SWI/SNF activity.  One such enzyme is lysine-specific histone demethylase 1 (LSD1/KDM1A), which regulates the chromatin landscape and gene expression by demethylating proteins such as histone H3.  Cross-cancer anal. of the TCGA database shows that LSD1 is highly expressed in SWI/SNF-mutated tumors.  SCCOHT and OCCC cell lines have shown sensitivity to the reversible LSD1 inhibitor SP-2577, suggesting that SWI/SNF-deficient ovarian cancers are dependent on LSD1 activity.  In this study, we investigated the ability of SP-2577 to promote anti-tumor immunity and T-cell infiltration in SCCOHT and OCCC cell lines.  We found that SP-2577 stimulated IFN-dependent anti-tumor immunity in SCCOHT and promoted the expression of PD-L1 in both SCCOHT and OCCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzji-N7T50nrVg90H21EOLACvtfcHk0lhWmPJq4mTNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVejurvM&md5=4f39f8b246f8e6f2c732ac6f81a3a450</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0235705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0235705%26sid%3Dliteratum%253Aachs%26aulast%3DSoldi%26aufirst%3DR.%26aulast%3DGhosh%2BHalder%26aufirst%3DT.%26aulast%3DWeston%26aufirst%3DA.%26aulast%3DThode%26aufirst%3DT.%26aulast%3DDrenner%26aufirst%3DK.%26aulast%3DLewis%26aufirst%3DR.%26aulast%3DKaadige%26aufirst%3DM.%2BR.%26aulast%3DSrivastava%26aufirst%3DS.%26aulast%3DDaniel%2BAmpanattu%26aufirst%3DS.%26aulast%3DRodriguez%2BDel%2BVillar%26aufirst%3DR.%26aulast%3DLang%26aufirst%3DJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DWeissman%26aufirst%3DB.%26aulast%3DTrent%26aufirst%3DJ.%2BM.%26aulast%3DHendricks%26aufirst%3DW.%2BP.%2BD.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DThe%2520novel%2520reversible%2520LSD1%2520inhibitor%2520SP-2577%2520promotes%2520anti-tumor%2520immunity%2520in%2520SWItch%252FSucrose-NonFermentable%2520%2528SWI%252FSNF%2529%2520complex%2520mutated%2520ovarian%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2020%26volume%3D15%26doi%3D10.1371%2Fjournal.pone.0235705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harancher, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packard, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLuca, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dembowski, J. A.</span></span> <span> </span><span class="NLM_article-title">Antiviral properties of the LSD1 inhibitor SP-2509</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">e00974</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1128/JVI.00974-20</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1128%2FJVI.00974-20" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32699090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitV2jsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2020&pages=e00974-20&author=M.+R.+Harancherauthor=J.+E.+Packardauthor=S.+P.+Cowanauthor=N.+A.+DeLucaauthor=J.+A.+Dembowski&title=Antiviral+properties+of+the+LSD1+inhibitor+SP-2509&doi=10.1128%2FJVI.00974-20"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Antiviral properties of the LSD1 inhibitor SP-2509</span></div><div class="casAuthors">Harancher, Mitchell R.; Packard, Jessica E.; Cowan, Shane P.; DeLuca, Neal A.; Dembowski, Jill A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">e00974</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) targets cellular proteins, including histone H3, p53, E2F, and Dnmt1, and is involved in the regulation of gene expression, DNA replication, the cell cycle, and the DNA damage response.  LSD1 catalyzes demethylation of histone H3K9 assocd. with herpes simplex virus 1 (HSV-1) immediate early (IE) promoters and is necessary for IE gene expression, viral DNA replication, and reactivation from latency.  We previously found that LSD1 assocs. with HSV-1 replication forks and replicating viral DNA, suggesting that it may play a direct role in viral replication or coupled processes.  We investigated the effects of the LSD1 inhibitor SP-2509 on the HSV-1 life cycle.  Unlike previously investigated LSD1 inhibitors tranylcypromine (TCP) and OG-L002, which covalently attach to the LSD1 cofactor FAD (FAD) to inhibit demethylase activity, SP-2509 has previously been shown to inhibit LSD1 protein-protein interactions.  We found that SP-2509 does not inhibit HSV-1 IE gene expression or transcription factor and RNA polymerase II (Pol II) assocn. with viral DNA prior to the onset of replication.  However, SP-2509 does inhibit viral DNA replication, late gene expression, and virus prodn.  We used EdC labeling of nascent viral DNA to image aberrant viral replication compartments that form in the presence of SP-2509.  Treatment resulted in the formation of small replication foci that colocalize with replication proteins but are defective for Pol II recruitment.  Taken together, these data highlight a potential new role for LSD1 in the regulation of HSV-1 DNA replication and gene expression after the onset of DNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBYhNfWFB6uLVg90H21EOLACvtfcHk0ljrKCw3c4GwxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitV2jsb3F&md5=84e266762880c90ff0ffdd95cdd853bc</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1128%2FJVI.00974-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00974-20%26sid%3Dliteratum%253Aachs%26aulast%3DHarancher%26aufirst%3DM.%2BR.%26aulast%3DPackard%26aufirst%3DJ.%2BE.%26aulast%3DCowan%26aufirst%3DS.%2BP.%26aulast%3DDeLuca%26aufirst%3DN.%2BA.%26aulast%3DDembowski%26aufirst%3DJ.%2BA.%26atitle%3DAntiviral%2520properties%2520of%2520the%2520LSD1%2520inhibitor%2520SP-2509%26jtitle%3DJ.%2520Virol.%26date%3D2020%26volume%3D94%26spage%3De00974%26epage%3D20%26doi%3D10.1128%2FJVI.00974-20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reed, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mascarenhas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chawla, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setty, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wages, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenehjem, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiesteban, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DuBois, S. G.</span></span> <span> </span><span class="NLM_article-title">A phase I/II clinical trial of the reversible LSD1 inhibitor, seclidemstat, in patients with relapsed/refractory Ewing sarcoma</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">TPS11567</span>, <span class="refDoi"> DOI: 10.1200/JCO.2020.38.15_suppl.TPS11567</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1200%2FJCO.2020.38.15_suppl.TPS11567" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2020&pages=TPS11567&author=D.+R.+Reedauthor=L.+Mascarenhasauthor=P.+A.+Meyersauthor=S.+P.+Chawlaauthor=D.+J.+Harrisonauthor=B.+Settyauthor=J.+Mettsauthor=D.+S.+Wagesauthor=D.+D.+Stenehjemauthor=D.+Y.+Santiestebanauthor=S.+G.+DuBois&title=A+phase+I%2FII+clinical+trial+of+the+reversible+LSD1+inhibitor%2C+seclidemstat%2C+in+patients+with+relapsed%2Frefractory+Ewing+sarcoma&doi=10.1200%2FJCO.2020.38.15_suppl.TPS11567"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1200%2FJCO.2020.38.15_suppl.TPS11567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2020.38.15_suppl.TPS11567%26sid%3Dliteratum%253Aachs%26aulast%3DReed%26aufirst%3DD.%2BR.%26aulast%3DMascarenhas%26aufirst%3DL.%26aulast%3DMeyers%26aufirst%3DP.%2BA.%26aulast%3DChawla%26aufirst%3DS.%2BP.%26aulast%3DHarrison%26aufirst%3DD.%2BJ.%26aulast%3DSetty%26aufirst%3DB.%26aulast%3DMetts%26aufirst%3DJ.%26aulast%3DWages%26aufirst%3DD.%2BS.%26aulast%3DStenehjem%26aufirst%3DD.%2BD.%26aulast%3DSantiesteban%26aufirst%3DD.%2BY.%26aulast%3DDuBois%26aufirst%3DS.%2BG.%26atitle%3DA%2520phase%2520I%252FII%2520clinical%2520trial%2520of%2520the%2520reversible%2520LSD1%2520inhibitor%252C%2520seclidemstat%252C%2520in%2520patients%2520with%2520relapsed%252Frefractory%2520Ewing%2520sarcoma%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2020%26volume%3D38%26spage%3DTPS11567%26doi%3D10.1200%2FJCO.2020.38.15_suppl.TPS11567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y. L.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel (E)-N’-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides as potent LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4552</span>– <span class="NLM_lpage">4557</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.06.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2015.06.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27524309" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOhtrfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4552-4557&author=Y.+Zhouauthor=Y.+Liauthor=W.+J.+Wangauthor=P.+Xiangauthor=X.+M.+Luoauthor=L.+Yangauthor=S.+Y.+Yangauthor=Y.+L.+Zhao&title=Synthesis+and+biological+evaluation+of+novel+%28E%29-N%E2%80%99-%282%2C3-dihydro-1H-inden-1-ylidene%29+benzohydrazides+as+potent+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2015.06.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of novel (E)-N'-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides as potent LSD1 inhibitors</span></div><div class="casAuthors">Zhou, Yang; Li, Yan; Wang, Wen-Jing; Xiang, Pu; Luo, Xin-Mei; Yang, Li; Yang, Sheng-Yong; Zhao, Ying-Lan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4552-4557</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is recognized as an attractive therapeutic target in multiple malignancies.  A series of novel (E)-N'-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides were synthesized and biol. evaluated for their potential LSD1 inhibitory effect.  Among them, compds. (E)-N'-(7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide a and (E)-N'-(4-chloro-7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide showed the most potent LSD1 inhibitory activity with IC50 values of 1.4 and 1.7 nM, resp., which were ∼10 times more potent compared with (E)-N-(1-(5-chloro-2-hydroxyphenyl) ethylidene)-3-(morpholinosulf-only) benzohydrazide.  Compds. (E)-N'-(7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide a and (E)-N'-(4-chloro-7-hydroxy-2,3-dihydro-1H-inden-1-ylidene)-3-(morpholinosulfonyl)benzohydrazide also exhibited marked anti-proliferation activities against cancer cell lines that highly expressed LSD1.  These results suggest that these optimized compds. might be served as promising LSD1 inhibitors against cancer, which merit further study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6SXQSBWFZcLVg90H21EOLACvtfcHk0ljrKCw3c4GwxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOhtrfL&md5=f0469735243cb2b9b569cd4fdeba9a85</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.06.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.06.054%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%2BJ.%26aulast%3DXiang%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DX.%2BM.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DS.%2BY.%26aulast%3DZhao%26aufirst%3DY.%2BL.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520%2528E%2529-N%25E2%2580%2599-%25282%252C3-dihydro-1H-inden-1-ylidene%2529%2520benzohydrazides%2520as%2520potent%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4552%26epage%3D4557%26doi%3D10.1016%2Fj.bmcl.2015.06.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xi, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological activity of 3-oxoamino-benzenesulfonamides as selective and reversible LSD1 inhibitors</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2017.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2017.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=28460360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFCjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2017&pages=182-189&author=J.+Y.+Xiauthor=S.+Y.+Xuauthor=L.+M.+Wuauthor=T.+F.+Maauthor=R.+F.+Liuauthor=Y.+C.+Liuauthor=D.+W.+Dengauthor=Y.+Q.+Guauthor=J.+P.+Zhouauthor=F.+Lanauthor=X.+M.+Zha&title=Design%2C+synthesis+and+biological+activity+of+3-oxoamino-benzenesulfonamides+as+selective+and+reversible+LSD1+inhibitors&doi=10.1016%2Fj.bioorg.2017.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological activity of 3-oxoamino-benzenesulfonamides as selective and reversible LSD1 inhibitors</span></div><div class="casAuthors">Xi, Jiayue; Xu, Siyuan; Wu, Liming; Ma, Tianfang; Liu, Rongfeng; Liu, Yu-Chih; Deng, Dawei; Gu, Yueqing; Zhou, Jinpei; Lan, Fei; Zha, Xiaoming</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">182-189</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) is a flavin-dependent amine oxidase that selectively removes one or two Me groups from H3 at Lys4 and is recognized as a promising therapeutic target for cancer and other diseases.  Here, a series of 3-oxoamino-benzenesulfonamides were synthesized and evaluated for their inhibitory activity against LSD1.  Compds. I and II showed the most potent inhibition with the IC50 values of 9.5 and 6.9 μM, resp.  Furthermore, the LSD1 inhibition of I and II were reversible and selective.  Docking study presented the possible binding mode between 7b, 7h and the LSD1 active site.  Taken together, 3-oxoamino-benzenesulfonamides may represent a new class of reversible LSD1 inhibitors and 7b and 7h were two hit compds. deserved further structural optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsoVXX7-ETcrVg90H21EOLACvtfcHk0ljrKCw3c4GwxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFCjtbg%253D&md5=1ded182ebd4f36de601a0d2d7dfcb93d</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2017.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2017.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DXi%26aufirst%3DJ.%2BY.%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DWu%26aufirst%3DL.%2BM.%26aulast%3DMa%26aufirst%3DT.%2BF.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDeng%26aufirst%3DD.%2BW.%26aulast%3DGu%26aufirst%3DY.%2BQ.%26aulast%3DZhou%26aufirst%3DJ.%2BP.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DZha%26aufirst%3DX.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520activity%2520of%25203-oxoamino-benzenesulfonamides%2520as%2520selective%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Chem.%26date%3D2017%26volume%3D72%26spage%3D182%26epage%3D189%26doi%3D10.1016%2Fj.bioorg.2017.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zha, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, F. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological activity of N-(3-substituted-phenyl)benzenesulfonamides as selective and reversible LSD1 inhibitors</span>. <i>Med. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2822</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1007/s00044-016-1706-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1007%2Fs00044-016-1706-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVCgsLbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2016&pages=2822-2831&author=X.+M.+Zhaauthor=L.+M.+Wuauthor=S.+Y.+Xuauthor=F.+X.+Zouauthor=J.+X.+Xiauthor=T.+F.+Maauthor=R.+F.+Liuauthor=Y.+C.+Liuauthor=D.+W.+Dengauthor=Y.+Q.+Guauthor=J.+P.+Zhouauthor=F.+Lan&title=Design%2C+synthesis+and+biological+activity+of+N-%283-substituted-phenyl%29benzenesulfonamides+as+selective+and+reversible+LSD1+inhibitors&doi=10.1007%2Fs00044-016-1706-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological activity of N-(3-substituted-phenyl)benzenesulfonamides as selective and reversible LSD1 inhibitors</span></div><div class="casAuthors">Zha, Xiaoming; Wu, Liming; Xu, Siyuan; Zou, Fangxia; Xi, Jiayue; Ma, Tianfang; Liu, Rongfeng; Liu, Yu-Chih; Deng, Dawei; Gu, Yueqing; Zhou, Jinpei; Lan, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2822-2831</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 plays a crucial role in regulating histone methylation at residues K4 and K9 on histone H3 and over-expresses in a variety of cancers.  Here the authors designed, synthesized and evaluated a series of N-(3-substituted-phenyl)benzenesulfonamides as reversible lysine specific demethylase 1 inhibitors.  All the compds. exhibited lysine specific demethylase 1 inhibition with the half maximal inhibitory concn. (IC50) values between 7.5 and 68 μM.  Three most active compds. including I displayed only modest effect on FAD-dependent enzymes monoamine oxidases A and B, and the reversibility of lysine specific demethylase 1 inhibition were confirmed.  Mol. docking was also carried out to predict the binding mode of I into the active site of lysine specific demethylase 1.  Taken together, N-(3-substituted-phenyl)benzenesulfonamides including I represent a new class of selective and reversible lysine specific demethylase 1 inhibitors with pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouiqDa8FKmmbVg90H21EOLACvtfcHk0ljiVljVzE8EhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVCgsLbP&md5=252a51da6c0983e8774887f8bb6ae284</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1007%2Fs00044-016-1706-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-016-1706-8%26sid%3Dliteratum%253Aachs%26aulast%3DZha%26aufirst%3DX.%2BM.%26aulast%3DWu%26aufirst%3DL.%2BM.%26aulast%3DXu%26aufirst%3DS.%2BY.%26aulast%3DZou%26aufirst%3DF.%2BX.%26aulast%3DXi%26aufirst%3DJ.%2BX.%26aulast%3DMa%26aufirst%3DT.%2BF.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDeng%26aufirst%3DD.%2BW.%26aulast%3DGu%26aufirst%3DY.%2BQ.%26aulast%3DZhou%26aufirst%3DJ.%2BP.%26aulast%3DLan%26aufirst%3DF.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520activity%2520of%2520N-%25283-substituted-phenyl%2529benzenesulfonamides%2520as%2520selective%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2016%26volume%3D25%26spage%3D2822%26epage%3D2831%26doi%3D10.1007%2Fs00044-016-1706-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellaisamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span> <span> </span><span class="NLM_article-title">A rhodium(III)-based inhibitor of lysine-specific histone demethylase 1 as an epigenetic modulator in prostate cancer cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2597</span>– <span class="NLM_lpage">2603</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00133</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00133" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVajtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2597-2603&author=C.+Yangauthor=W.+Wangauthor=J.+X.+Liangauthor=G.+Liauthor=K.+Vellaisamyauthor=C.+Y.+Wongauthor=D.+L.+Maauthor=C.+H.+Leung&title=A+rhodium%28III%29-based+inhibitor+of+lysine-specific+histone+demethylase+1+as+an+epigenetic+modulator+in+prostate+cancer+cells&doi=10.1021%2Facs.jmedchem.7b00133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">A Rhodium(III)-Based Inhibitor of Lysine-Specific Histone Demethylase 1 as an Epigenetic Modulator in Prostate Cancer Cells</span></div><div class="casAuthors">Yang, Chao; Wang, Wanhe; Liang, Jia-Xin; Li, Guodong; Vellaisamy, Kasipandi; Wong, Chun-Yuen; Ma, Dik-Lung; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2597-2603</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report herein a novel rhodium(III) complex 1 (I) as a new LSD1 targeting agent and epigenetic modulator.  Complex 1 disrupted the interaction of LSD1-H3K4me2 in human prostate carcinoma cells, and enhanced the amplification of p21, FOXA2 and BMP2 gene promoters.  Complex 1 was selective for LSD1 over other histone demethylases, such as KDM2b, KDM7 and MAO activities, and also showed anti-proliferative activity towards human cancer cells.  To date, complex 1 is the first metal-based inhibitor of LSD1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobbBaU5SOfBrVg90H21EOLACvtfcHk0ljiVljVzE8EhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVajtr8%253D&md5=6c9c9729423cf36a645c0f77ccf186b9</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00133%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DJ.%2BX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DVellaisamy%26aufirst%3DK.%26aulast%3DWong%26aufirst%3DC.%2BY.%26aulast%3DMa%26aufirst%3DD.%2BL.%26aulast%3DLeung%26aufirst%3DC.%2BH.%26atitle%3DA%2520rhodium%2528III%2529-based%2520inhibitor%2520of%2520lysine-specific%2520histone%2520demethylase%25201%2520as%2520an%2520epigenetic%2520modulator%2520in%2520prostate%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2597%26epage%3D2603%26doi%3D10.1021%2Facs.jmedchem.7b00133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, G. Q.</span></span> <span> </span><span class="NLM_article-title">Discovery of vanadium complexes bearing tridentate schiff base ligands as novel LSD1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">683</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2018.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=30611616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksl2isw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=681-683&author=L.+P.+Luauthor=J.+H.+Liuauthor=S.+H.+Cenauthor=Y.+L.+Jiangauthor=G.+Q.+Hu&title=Discovery+of+vanadium+complexes+bearing+tridentate+schiff+base+ligands+as+novel+LSD1+inhibitors&doi=10.1016%2Fj.bmcl.2018.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of vanadium complexes bearing tridentate schiff base ligands as novel LSD1 inhibitors</span></div><div class="casAuthors">Lu, Ling-Pan; Liu, Jin-Hai; Cen, Shi-Hong; Jiang, Ya-Ling; Hu, Guo-Qiang</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">681-683</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine specific demethylase (LSD1) plays a pivotal role in epigenetic modulation of gene expression.  Aberrant expression of LSD1 was assocd. with the progress and oncogenesis of multiple human cancers.  Herein, the authors report the preliminary anti-LSD1 evaluation of the synthetic vanadium (V) complexes.  Among them, complex 2 showed a moderate inhibitory effect against LSD1 with IC50 value of 19.0 μM, as well as good selectivity over MAO-A/B.  Complex 2 is the first vanadium based LSD1 inhibitor, which provides a novel scaffold for the development of LSD1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWDIJ04kf_8LVg90H21EOLACvtfcHk0ljiVljVzE8EhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksl2isw%253D%253D&md5=7b4fa22c0dd01d42ef867dfc88e1d36b</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DL.%2BP.%26aulast%3DLiu%26aufirst%3DJ.%2BH.%26aulast%3DCen%26aufirst%3DS.%2BH.%26aulast%3DJiang%26aufirst%3DY.%2BL.%26aulast%3DHu%26aufirst%3DG.%2BQ.%26atitle%3DDiscovery%2520of%2520vanadium%2520complexes%2520bearing%2520tridentate%2520schiff%2520base%2520ligands%2520as%2520novel%2520LSD1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D681%26epage%3D683%26doi%3D10.1016%2Fj.bmcl.2018.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mould, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bremberg, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cartic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geitmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGonagle, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somervaille, T. C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turlais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D.</span></span> <span> </span><span class="NLM_article-title">Development of (4-cyanophenyl)glycine derivatives as reversible inhibitors of lysine specific demethylase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7984</span>– <span class="NLM_lpage">7999</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00462</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00462" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKqsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7984-7999&author=D.+P.+Mouldauthor=C.+Alliauthor=U.+Brembergauthor=S.+Carticauthor=A.+M.+Jordanauthor=M.+Geitmannauthor=A.+Maiques-Diazauthor=A.+E.+McGonagleauthor=T.+C.+P.+Somervailleauthor=G.+J.+Spencerauthor=F.+Turlaisauthor=D.+Ogilvie&title=Development+of+%284-cyanophenyl%29glycine+derivatives+as+reversible+inhibitors+of+lysine+specific+demethylase+1&doi=10.1021%2Facs.jmedchem.7b00462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Development of (4-Cyanophenyl)glycine Derivatives as Reversible Inhibitors of Lysine Specific Demethylase 1</span></div><div class="casAuthors">Mould, Daniel P.; Alli, Cristina; Bremberg, Ulf; Cartic, Sharon; Jordan, Allan M.; Geitmann, Matthis; Maiques-Diaz, Alba; McGonagle, Alison E.; Somervaille, Tim C. P.; Spencer, Gary J.; Turlais, Fabrice; Ogilvie, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7984-7999</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of lysine specific demethylase 1 (LSD1) has been shown to induce the differentiation of leukemia stem cells in acute myeloid leukemia (AML).  Irreversible inhibitors developed from the nonspecific inhibitor tranylcypromine have entered clin. trials; however, the development of effective reversible inhibitors has proved more challenging.  Herein, we describe our efforts to identify reversible inhibitors of LSD1 from a high throughput screen and subsequent in silico modeling approaches.  From a single hit (12) validated by biochem. and biophys. assays, we describe our efforts to develop acyclic scaffold-hops from GSK-690 (1).  A further scaffold modification to a (4-cyanophenyl)glycinamide (e.g., 29a) led to the development of compd. 32, with a Kd value of 32 nM and an EC50 value of 0.67 μM in a surrogate cellular biomarker assay.  Moreover, this deriv. does not display the same level of hERG liability as obsd. with 1 and represents a promising lead for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5NyXkmZPOqrVg90H21EOLACvtfcHk0lhCE5kUrE1cyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKqsrrJ&md5=af673e7b3a2815aa09fe7264a3e4247b</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00462%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DAlli%26aufirst%3DC.%26aulast%3DBremberg%26aufirst%3DU.%26aulast%3DCartic%26aufirst%3DS.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DGeitmann%26aufirst%3DM.%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DTurlais%26aufirst%3DF.%26aulast%3DOgilvie%26aufirst%3DD.%26atitle%3DDevelopment%2520of%2520%25284-cyanophenyl%2529glycine%2520derivatives%2520as%2520reversible%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7984%26epage%3D7999%26doi%3D10.1021%2Facs.jmedchem.7b00462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. B.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel dual-acting modulators targeting both estrogen receptor alpha (ERalpha) and lysine-specific demethylase 1 (LSD1) for treatment of breast cancer</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">112281</span>– <span class="NLM_lpage">112285</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.ejmech.2020.112281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32283297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXntVOisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2020&pages=112281-112285&author=M.+Heauthor=W.+Ningauthor=Z.+Huauthor=J.+Huangauthor=C.+Dongauthor=H.+B.+Zhou&title=Design%2C+synthesis+and+biological+evaluation+of+novel+dual-acting+modulators+targeting+both+estrogen+receptor+alpha+%28ERalpha%29+and+lysine-specific+demethylase+1+%28LSD1%29+for+treatment+of+breast+cancer&doi=10.1016%2Fj.ejmech.2020.112281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel dual-acting modulators targeting both estrogen receptor α (ERα) and lysine-specific demethylase 1 (LSD1) for treatment of breast cancer</span></div><div class="casAuthors">He, Ming; Ning, Wentao; Hu, Zhiye; Huang, Jian; Dong, Chune; Zhou, Hai-Bing</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112281</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Breast cancer is a multi-factor disease, thus more and more drug combination therapies are applied in the treatment.  However, there are undeniable disadvantages in drug combination therapy.  Therefore, the development of new dual-targeting drugs has become a new strategy.  In this study, we have developed a series of dual-acting agents targeting both estrogen receptor α (ERα) and histone demethylase based on a privileged OBHS pharmacophore scaffold developed previously by our lab.  These novel OBHS-LSD1i conjugates showed excellent ERα binding affinity and selectivity, and exhibited potent inhibitory activity against lysine specific demethylase 1 (LSD1).  Several conjugates showed higher antiproliferative efficacy in MCF-7 breast cancer cell line compared to 4-hydroxytamoxifen in vitro.  Among them, the best compd. 11g displayed potent inhibitory activity against LSD1 and MCF-7 cells with IC50 values of 1.55 μM and 8.79 μM, resp.  Flow cytometry anal. of apoptosis of 11g suggested that the effect of this type compds. on MCF-7 cells is partly caused by inducing apoptosis.  Moreover, the mol. docking of 11g with ERα and the active site of LSD1/CoREST complex provides practical way for understanding the dual mechanism actions of this kind of compds. with the targets.  As such, these compds. have shown potential to become promising leads for the development of highly efficient dual-acting modulators for breast cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLGygeo7gyc7Vg90H21EOLACvtfcHk0lhCE5kUrE1cyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXntVOisbk%253D&md5=baa26f620cad01fbe06964b00dedd351</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112281%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DM.%26aulast%3DNing%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DH.%2BB.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520dual-acting%2520modulators%2520targeting%2520both%2520estrogen%2520receptor%2520alpha%2520%2528ERalpha%2529%2520and%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520for%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D195%26spage%3D112281%26epage%3D112285%26doi%3D10.1016%2Fj.ejmech.2020.112281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L. Y.</span></span> <span> </span><span class="NLM_article-title">Bioactivity evaluation of natural product alpha-mangostin as a novel xanthone-based lysine-specific demethylase 1 inhibitor to against tumor metastasis</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">419</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2017.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2017.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=29274582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWltr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2018&pages=415-419&author=C.+Hanauthor=Z.+R.+Liauthor=J.+Q.+Houauthor=Z.+Wangauthor=D.+Q.+Xuauthor=G.+M.+Xueauthor=L.+Y.+Kong&title=Bioactivity+evaluation+of+natural+product+alpha-mangostin+as+a+novel+xanthone-based+lysine-specific+demethylase+1+inhibitor+to+against+tumor+metastasis&doi=10.1016%2Fj.bioorg.2017.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactivity evaluation of natural product α-mangostin as a novel xanthone-based lysine-specific demethylase 1 inhibitor to against tumor metastasis</span></div><div class="casAuthors">Han, Chao; Li, Zhongrui; Hou, Jiqin; Wang, Zhen; Xu, Dingqiao; Xue, Guimin; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">415-419</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1), which has been reported to be overexpressed in several human cancers, has recently emerged as an attractive therapeutic target for treating cancer.  To date, almost all the developed LSD1 inhibitors are chemo-synthesized mols., while α-mangostin is first characterized as xanthone-based natural inhibitor in the current study with IC50 values of 2.81±0.44μM.  Bioactivity study and docking anal. indicated that α-mangostin could inhibit MDA-MB-231 cells migration and evasion through inhibit intracellular LSD1 activity.  These findings provides new mol. skeleton for LSD1 inhibitor study and should encourage further modification of α-mangostin to produce more potent LSD1 inhibitors with potential anticancer activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPGpOCbxAuNLVg90H21EOLACvtfcHk0lhCE5kUrE1cyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWltr3J&md5=27679988f1a7c98127c71c3799734c5f</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2017.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2017.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DHou%26aufirst%3DJ.%2BQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DD.%2BQ.%26aulast%3DXue%26aufirst%3DG.%2BM.%26aulast%3DKong%26aufirst%3DL.%2BY.%26atitle%3DBioactivity%2520evaluation%2520of%2520natural%2520product%2520alpha-mangostin%2520as%2520a%2520novel%2520xanthone-based%2520lysine-specific%2520demethylase%25201%2520inhibitor%2520to%2520against%2520tumor%2520metastasis%26jtitle%3DBioorg.%2520Chem.%26date%3D2018%26volume%3D76%26spage%3D415%26epage%3D419%26doi%3D10.1016%2Fj.bioorg.2017.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suo, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">Natural protoberberine alkaloids, identified as potent selective LSD1 inhibitors, induce AML cell differentiation</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">103648</span>– <span class="NLM_lpage">103655</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.103648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2020.103648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32065882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFajsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2020&pages=103648-103655&author=Z.+R.+Liauthor=F.+Z.+Suoauthor=Y.+J.+Guoauthor=H.+F.+Chengauthor=S.+H.+Niuauthor=D.+D.+Shenauthor=L.+J.+Zhaoauthor=Z.+Z.+Liuauthor=M.+Maaauthor=B.+Yuauthor=Y.+C.+Zhengauthor=H.+M.+Liu&title=Natural+protoberberine+alkaloids%2C+identified+as+potent+selective+LSD1+inhibitors%2C+induce+AML+cell+differentiation&doi=10.1016%2Fj.bioorg.2020.103648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Natural protoberberine alkaloids, identified as potent selective LSD1 inhibitors, induce AML cell differentiation</span></div><div class="casAuthors">Li, Zhong-Rui; Suo, Feng-Zhi; Guo, Yan-Jia; Cheng, Hai-Fang; Niu, Sheng-Hui; Shen, Dan-Dan; Zhao, Li-Juan; Liu, Zhen-Zhen; Maa, Mamun; Yu, Bin; Zheng, Yi-Chao; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">103648</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Natural protoberberine alkaloids were first identified and characterized as potent, selective and cellular active lysine specific demethylase 1 (LSD1) inhibitors.  Due to our study, isoquinoline-based tetracyclic scaffold was identified as the key structural element for their anti-LSD1 activity, subtle changes of substituents attached to the core structure led to dramatic changes of the activity.  Among these protoberberine alkaloids, epiberberine potently inhibited LSD1 (IC50 = 0.14 ± 0.01μM) and was highly selective to LSD1 over MAO-A/B.  Furthermore, epiberberine could induce the expression of CD86, CD11b and CD14 in THP-1 and HL-60 cells, confirming its cellular activity of inducing acute myeloid leukemia (AML) cells differentiation.  Moreover, epiberberine prolonged the survival of THP-1 cells bearing mice and inhibited the growth of AML cells in vivo without obvious global toxicity.  These findings give the potential application of epiberberine in AML treatment, and the isoquinoline-based tetracyclic scaffold could be used for further development of LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3QhTkrzh9qrVg90H21EOLACvtfcHk0ljCIS0YLAWh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFajsr0%253D&md5=4868d9e25f504a93a3c79b01dce5ccc1</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.103648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.103648%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BR.%26aulast%3DSuo%26aufirst%3DF.%2BZ.%26aulast%3DGuo%26aufirst%3DY.%2BJ.%26aulast%3DCheng%26aufirst%3DH.%2BF.%26aulast%3DNiu%26aufirst%3DS.%2BH.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DZhao%26aufirst%3DL.%2BJ.%26aulast%3DLiu%26aufirst%3DZ.%2BZ.%26aulast%3DMaa%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DNatural%2520protoberberine%2520alkaloids%252C%2520identified%2520as%2520potent%2520selective%2520LSD1%2520inhibitors%252C%2520induce%2520AML%2520cell%2520differentiation%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D97%26spage%3D103648%26epage%3D103655%26doi%3D10.1016%2Fj.bioorg.2020.103648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cang, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, P. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Capsaicin: a “hot” KDM1A/LSD1 inhibitor from peppers</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">104161</span>– <span class="NLM_lpage">104165</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2020.104161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bioorg.2020.104161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32889380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslynsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2020&pages=104161-104165&author=G.+Jiaauthor=S.+D.+Cangauthor=P.+Z.+Maauthor=Z.+Y.+Song&title=Capsaicin%3A+a+%E2%80%9Chot%E2%80%9D+KDM1A%2FLSD1+inhibitor+from+peppers&doi=10.1016%2Fj.bioorg.2020.104161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Capsaicin: A "hot" KDM1A/LSD1 inhibitor from peppers</span></div><div class="casAuthors">Jia, Gang; Cang, Shundong; Ma, Peizhi; Song, Zhiyu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104161</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Capsaicinoids are plant secondary metabolites, and capsaicin is the main principal that responsible to the pungency of chili peppers, with widely application as food additive.  In our study, capsaicin was characterized as lysine specific demethylase 1A (KDM1A/LSD1) inhibitor with IC50 of 0.6 ± 0.0421 μM in biochem. level, and can bind KDM1A recombinant directly and reversibly.  Further cellular study confirmed that capsaicin can bind and inhibit KDM1A in gastric cancer cell line BGC-823 and further inhibit cell invasion and migration by reversing epithelial-mesenchymal transition (EMT).  In sum, our findings identified KDM1A as a target of capsaicin and reveals capsaicin as a modifier of histone methylation for the first time, which may provide a new skeleton for further optimization of KDM1A inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv0ax5ObfkzLVg90H21EOLACvtfcHk0ljCIS0YLAWh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslynsL%252FE&md5=6695d3eabad21b8ef51ec6515c0eae9e</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2020.104161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2020.104161%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DG.%26aulast%3DCang%26aufirst%3DS.%2BD.%26aulast%3DMa%26aufirst%3DP.%2BZ.%26aulast%3DSong%26aufirst%3DZ.%2BY.%26atitle%3DCapsaicin%253A%2520a%2520%25E2%2580%259Chot%25E2%2580%259D%2520KDM1A%252FLSD1%2520inhibitor%2520from%2520peppers%26jtitle%3DBioorg.%2520Chem.%26date%3D2020%26volume%3D103%26spage%3D104161%26epage%3D104165%26doi%3D10.1016%2Fj.bioorg.2020.104161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">TCPs: privileged scaffolds for identifying potent LSD1 inhibitors for cancer therapy</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">651</span>– <span class="NLM_lpage">666</span>, <span class="refDoi"> DOI: 10.2217/epi-2015-0002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2217%2Fepi-2015-0002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=27102879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BC28XotVSrsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=651-666&author=Y.+C.+Zhengauthor=B.+Yuauthor=Z.+S.+Chenauthor=Y.+Liuauthor=H.+M.+Liu&title=TCPs%3A+privileged+scaffolds+for+identifying+potent+LSD1+inhibitors+for+cancer+therapy&doi=10.2217%2Fepi-2015-0002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">TCPs: privileged scaffolds for identifying potent LSD1 inhibitors for cancer therapy</span></div><div class="casAuthors">Zheng, Yi-Chao; Yu, Bin; Chen, Zhe-Sheng; Liu, Ying; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">651-666</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Since the first lysine-specific demethylase (KDM), lysine-specific demethylase 1 (LSD1), was characterized in 2004, several families of KDMs have been identified.  LSD1 can specifically demethylate H3K4me1/2, H3K9me1/2 as well as some nonhistone substrates.  It has been demonstrated to be an oncogene as well as a drug target.  Hence, tens of small-mol. LSD1 inhibitors have been designed, synthesized and applied for cancer treatment.  However, the two LSD1 inhibitors that have been advanced into early phase clin. trials are trans-2-phenylcyclopropylamine (TCP) derivs., which indicate that TCP is a druggable scaffold for LSD1 inhibitor.  Here, we review the design, synthesis and properties of reported TCP-based LSD1 inhibitors as well as their biol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQtSkXNPl_7LVg90H21EOLACvtfcHk0ljCIS0YLAWh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotVSrsrs%253D&md5=6f8021101929c931a6769dce538010c2</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.2217%2Fepi-2015-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi-2015-0002%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DZ.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTCPs%253A%2520privileged%2520scaffolds%2520for%2520identifying%2520potent%2520LSD1%2520inhibitors%2520for%2520cancer%2520therapy%26jtitle%3DEpigenomics%26date%3D2016%26volume%3D8%26spage%3D651%26epage%3D666%26doi%3D10.2217%2Fepi-2015-0002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Viegas-Junior, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danuello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Silva
Bolzani, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barreiro, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraga, C. A.</span></span> <span> </span><span class="NLM_article-title">Molecular hybridization: a useful tool in the design of new drug prototypes</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1829</span>– <span class="NLM_lpage">1852</span>, <span class="refDoi"> DOI: 10.2174/092986707781058805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.2174%2F092986707781058805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=17627520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvFanuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=1829-1852&author=C.+Viegas-Juniorauthor=A.+Danuelloauthor=V.+da+Silva%0ABolzaniauthor=E.+J.+Barreiroauthor=C.+A.+Fraga&title=Molecular+hybridization%3A+a+useful+tool+in+the+design+of+new+drug+prototypes&doi=10.2174%2F092986707781058805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular hybridization: a useful tool in the design of new drug prototypes</span></div><div class="casAuthors">Viegas-Junior, Claudio; Danuello, Amanda; da Silva Bolzani, Vanderlan; Barreiro, Eliezer J.; Fraga, Carlos Alberto Manssour</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1829-1852</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Mol. hybridization is a new concept in drug design and development based on the combination of pharmacophoric moieties of different bioactive substances to produce a new hybrid compd. with improved affinity and efficacy, when compared to the parent drugs.  Addnl., this strategy can result in compds. presenting modified selectivity profile, different and/or dual modes of action and reduced undesired side effects.  So, in this paper, we described several examples of different strategies for drug design, discovery and pharmacomodulation focused on new innovative hybrid compds. presenting analgesic, anti-inflammatory, platelet anti-aggregating, anti-infectious, anticancer, cardio- and neuroactive properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAvbzA71FPqLVg90H21EOLACvtfcHk0ljCIS0YLAWh6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvFanuro%253D&md5=7d3a16e3deeacb735bc0357fc6a99c37</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.2174%2F092986707781058805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986707781058805%26sid%3Dliteratum%253Aachs%26aulast%3DViegas-Junior%26aufirst%3DC.%26aulast%3DDanuello%26aufirst%3DA.%26aulast%3Dda%2BSilva%2BBolzani%26aufirst%3DV.%26aulast%3DBarreiro%26aufirst%3DE.%2BJ.%26aulast%3DFraga%26aufirst%3DC.%2BA.%26atitle%3DMolecular%2520hybridization%253A%2520a%2520useful%2520tool%2520in%2520the%2520design%2520of%2520new%2520drug%2520prototypes%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2007%26volume%3D14%26spage%3D1829%26epage%3D1852%26doi%3D10.2174%2F092986707781058805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sampath Kumar, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrmann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsogoeva, S. B.</span></span> <span> </span><span class="NLM_article-title">Structural hybridization as a facile approach to new drug candidates</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127514</span>– <span class="NLM_lpage">127528</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=10.1016%2Fj.bmcl.2020.127514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=32860980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvFGqurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127514-127528&author=H.+M.+Sampath+Kumarauthor=L.+Herrmannauthor=S.+B.+Tsogoeva&title=Structural+hybridization+as+a+facile+approach+to+new+drug+candidates&doi=10.1016%2Fj.bmcl.2020.127514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Structural hybridization as a facile approach to new drug candidates</span></div><div class="casAuthors">Sampath Kumar, Halmuthur M.; Herrmann, Lars; Tsogoeva, Svetlana B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">127514</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Structural hybridization of preclinically and clin. validated pharmacol. active mols. has emerged as a promising tool to develop new generations of safe and highly efficient drug candidates against various diseases including microbial infections, virus infections and cancer.  Strategies of drug-drug combinations have been adopted to generate hybrid conjugates of many clin. used drugs, designed to address inherent problems assocd. with these drugs.  Thus, the design of hybrids was aimed to achieve higher efficacy through possible multi-target interactions, selective delivery of the drug to the site of action with the aim to improve bioavailability, alleviate toxicity and circumvent drug resistances.  In this review article, we summarize the progress made in recent years in the rapidly growing field of drug discovery, focusing on the rationality of the hybrid design with particular emphasis on the linker architecture, which plays a crucial role in the overall success of a hybrid drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4eHnPN7UP0rVg90H21EOLACvtfcHk0lg80rZStjT02A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvFGqurjK&md5=b8ad2e17f676eec2b81835e43b6212c0</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127514%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%2BKumar%26aufirst%3DH.%2BM.%26aulast%3DHerrmann%26aufirst%3DL.%26aulast%3DTsogoeva%26aufirst%3DS.%2BB.%26atitle%3DStructural%2520hybridization%2520as%2520a%2520facile%2520approach%2520to%2520new%2520drug%2520candidates%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127514%26epage%3D127528%26doi%3D10.1016%2Fj.bmcl.2020.127514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D','PDB','2V1D'); return false;">PDB: 2V1D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZMT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZMT','PDB','3ZMT'); return false;">PDB: 3ZMT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S35" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S35','PDB','6S35'); return false;">PDB: 6S35</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3F','PDB','5L3F'); return false;">PDB: 5L3F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3G','PDB','5L3G'); return false;">PDB: 5L3G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5YJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5YJB','PDB','5YJB'); return false;">PDB: 5YJB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6W4K','PDB','6W4K'); return false;">PDB: 6W4K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L3E','PDB','5L3E'); return false;">PDB: 5L3E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN','PDB','5LGN'); return false;">PDB: 5LGN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LHI','PDB','5LHI'); return false;">PDB: 5LHI</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02176&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02176%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02176" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bd44f8493d36","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
