
<html lang="en"     class="pb-page"  data-request-id="8730739a-4a02-499b-8bb0-fa27ad5db135"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00292;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-11"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of (1H-Pyrazolo[3,4-c]pyridin-5-yl)sulfonamide Analogues as Hepatitis B Virus Capsid Assembly Modulators by Conformation Constraint" /></meta><meta name="dc.Creator" content="Chunting  Wang" /></meta><meta name="dc.Creator" content="Yameng  Pei" /></meta><meta name="dc.Creator" content="Lin  Wang" /></meta><meta name="dc.Creator" content="Shuo  Li" /></meta><meta name="dc.Creator" content="Chao  Jiang" /></meta><meta name="dc.Creator" content="Xu  Tan" /></meta><meta name="dc.Creator" content="Yi  Dong" /></meta><meta name="dc.Creator" content="Ye  Xiang" /></meta><meta name="dc.Creator" content="Yao  Ma" /></meta><meta name="dc.Creator" content="Gang  Liu" /></meta><meta name="dc.Description" content="Hepatitis B virus (HBV) capsid assembly modulators (CAMs) have been suggested to be effective anti-HBV agents in both preclinical and clinical studies. In addition to blocking HBV replication, CAMs..." /></meta><meta name="Description" content="Hepatitis B virus (HBV) capsid assembly modulators (CAMs) have been suggested to be effective anti-HBV agents in both preclinical and clinical studies. In addition to blocking HBV replication, CAMs..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 18, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00292" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00292" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00292" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00292" /></link>
        
    
    

<title>Discovery of (1H-Pyrazolo[3,4-c]pyridin-5-yl)sulfonamide Analogues as Hepatitis B Virus Capsid Assembly Modulators by Conformation Constraint | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00292" /></meta><meta property="og:title" content="Discovery of (1H-Pyrazolo[3,4-c]pyridin-5-yl)sulfonamide Analogues as Hepatitis B Virus Capsid Assembly Modulators by Conformation Constraint" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0027.jpeg" /></meta><meta property="og:description" content="Hepatitis B virus (HBV) capsid assembly modulators (CAMs) have been suggested to be effective anti-HBV agents in both preclinical and clinical studies. In addition to blocking HBV replication, CAMs could reduce the formation of covalently closed circular DNA (cccDNA), which accounts for the persistence of HBV infection. Here, we describe the discovery of (1H-indazole-5-yl)sulfonamides and (1H-pyrazolo[3,4-c]pyridin-5-yl)sulfonamides as new CAM chemotypes by constraining the conformation of the sulfamoylbenzamide derivatives. Lead optimization resulted in compound 56 with an EC50 value of 0.034 μM and good metabolic stability in mouse liver microsomes. To increase the solubility, the amino acid prodrug (65) and its citric acid salt (67) were prepared. Compound 67 dose dependently inhibited HBV replication in a hydrodynamic injection-based mouse model of HBV infection, while 56 did not show in vivo anti-HBV activity, likely owing to its suboptimal solubility. This class of compounds may serve as a starting point to develop novel anti-HBV drugs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00292"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00292">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00292&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00292&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00292&amp;href=/doi/10.1021/acs.jmedchem.0c00292" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 6066-6089</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00197" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00295" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of (1<i>H</i>-Pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide Analogues as Hepatitis B Virus Capsid Assembly Modulators by Conformation Constraint</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Chunting Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunting Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunting++Wang">Chunting Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yameng Pei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yameng Pei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Tsinghua University, Beijing 100084, China</div><div class="loa-info-affiliations-info">Tsinghua-Peking Center for Life Sciences, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yameng++Pei">Yameng Pei</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lin Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lin Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beijing Advanced Innovation Center for Structural Biology, Collaborative Innovation Center for Diagnosis and Treatment of Infectious Diseases, Center for Global Health and Infectious Diseases, Department of Basic Medical Sciences, School of Medicine, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lin++Wang">Lin Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuo Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuo Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, School of Pharmaceutical Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuo++Li">Shuo Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, School of Pharmaceutical Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao++Jiang">Chao Jiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xu Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xu Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, School of Pharmaceutical Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University, Beijing 100084, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xu++Tan">Xu Tan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Materia Medica, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing 100050, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi++Dong">Yi Dong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7844-7206" title="Orcid link">http://orcid.org/0000-0001-7844-7206</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ye Xiang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ye Xiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Beijing Advanced Innovation Center for Structural Biology, Beijing Frontier Research Center for Biological Structure, Center for Global Health and Infectious Diseases, Department of Basic Medical Sciences, School of Medicine, Tsinghua University, Beijing 100084, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#176e6f7e767970577a767e7b3963647e79707f627639727362397479"><span class="__cf_email__" data-cfemail="7f0607161e11183f121e1613510b0c161118170a1e511a1b0a511c11">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ye++Xiang">Ye Xiang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yao Ma</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yao Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Materia Medica, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing 100050, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c2afa3bba3ad82abafafeca3a1eca1ac"><span class="__cf_email__" data-cfemail="a1ccc0d8c0cee1c8cccc8fc0c28fc2cf">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yao++Ma">Yao Ma</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Gang Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Sciences, Tsinghua University, Beijing 100084, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6007010e070c09155257201413090e070815014e0504154e030e"><span class="__cf_email__" data-cfemail="e087818e878c8995d2d7a09493898e87889581ce858495ce838e">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gang++Liu">Gang Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5549-5686" title="Orcid link">http://orcid.org/0000-0001-5549-5686</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00292&amp;href=/doi/10.1021%2Facs.jmedchem.0c00292" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 6066–6089</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 18, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 February 2020</li><li><span class="item_label"><b>Published</b> online</span>18 May 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 June 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00292" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00292</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6066%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChunting%2BWang%252C%2BYameng%2BPei%252C%2BLin%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D11%26contentID%3Dacs.jmedchem.0c00292%26title%3DDiscovery%2Bof%2B%25281H-Pyrazolo%255B3%252C4-c%255Dpyridin-5-yl%2529sulfonamide%2BAnalogues%2Bas%2BHepatitis%2BB%2BVirus%2BCapsid%2BAssembly%2BModulators%2Bby%2BConformation%2BConstraint%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6089%26publicationDate%3DJune%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00292"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1397</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00292" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of (1H-Pyrazolo[3,4-c]pyridin-5-yl)sulfonamide Analogues as Hepatitis B Virus Capsid Assembly Modulators by Conformation Constraint&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chunting&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yameng&quot;,&quot;last_name&quot;:&quot;Pei&quot;},{&quot;first_name&quot;:&quot;Lin&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Shuo&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Chao&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Xu&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Yi&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Ye&quot;,&quot;last_name&quot;:&quot;Xiang&quot;},{&quot;first_name&quot;:&quot;Yao&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Gang&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;6066-6089&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00292&quot;},&quot;abstract&quot;:&quot;Hepatitis B virus (HBV) capsid assembly modulators (CAMs) have been suggested to be effective anti-HBV agents in both preclinical and clinical studies. In addition to blocking HBV replication, CAMs could reduce the formation of covalently closed circular DNA (cccDNA), which accounts for the persistence of HBV infection. Here, we describe the discovery of (1H-indazole-5-yl)sulfonamides and (1H-pyrazolo[3,4-c]pyridin-5-yl)sulfonamides as new CAM chemotypes by constraining the conformation of the sulfamoylbenzamide derivatives. Lead optimization resulted in compound 56 with an EC50 value of 0.034 μM and good metabolic stability in mouse liver microsomes. To increase the solubility, the amino acid prodrug (65) and its citric acid salt (67) were prepared. Compound 67 dose dependently inhibited HBV replication in a hydrodynamic injection-based mouse model of HBV infection, while 56 did not show in vivo anti-HBV activity, likely owing to its suboptimal solubility. This class of compounds may serve as a starting poin&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00292&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00292" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00292&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00292" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00292&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00292" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00292&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00292&amp;href=/doi/10.1021/acs.jmedchem.0c00292" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00292" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00292" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00292%26sid%3Dliteratum%253Aachs%26pmid%3D32421339%26genre%3Darticle%26aulast%3DWang%26date%3D2020%26atitle%3DDiscovery%2Bof%2B%25281H-Pyrazolo%255B3%252C4-c%255Dpyridin-5-yl%2529sulfonamide%2BAnalogues%2Bas%2BHepatitis%2BB%2BVirus%2BCapsid%2BAssembly%2BModulators%2Bby%2BConformation%2BConstraint%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D11%26spage%3D6066%26epage%3D6089%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290659" title="DNA replication">DNA replication</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291454" title="Solubility">Solubility</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/jmcmar.2020.63.issue-11/20200611/jmcmar.2020.63.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Hepatitis B virus (HBV) capsid assembly modulators (CAMs) have been suggested to be effective anti-HBV agents in both preclinical and clinical studies. In addition to blocking HBV replication, CAMs could reduce the formation of covalently closed circular DNA (cccDNA), which accounts for the persistence of HBV infection. Here, we describe the discovery of (1<i>H</i>-indazole-5-yl)sulfonamides and (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamides as new CAM chemotypes by constraining the conformation of the sulfamoylbenzamide derivatives. Lead optimization resulted in compound <b>56</b> with an EC<sub>50</sub> value of 0.034 μM and good metabolic stability in mouse liver microsomes. To increase the solubility, the amino acid prodrug (<b>65</b>) and its citric acid salt (<b>67</b>) were prepared. Compound <b>67</b> dose dependently inhibited HBV replication in a hydrodynamic injection-based mouse model of HBV infection, while <b>56</b> did not show <i>in vivo</i> anti-HBV activity, likely owing to its suboptimal solubility. This class of compounds may serve as a starting point to develop novel anti-HBV drugs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Chronic hepatitis B (CHB), caused by infection of the hepatitis B virus (HBV), is one of the major infectious diseases in the world, leading to approximately 887 000 deaths in 2015.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Although the worldwide application of efficient HBV vaccines has greatly decreased morbidity, approximately 257 million people still live with CHB. CHB tremendously increases the risk of developing cirrhosis and hepatic carcinoma, which are the leading causes of mortality for CHB patients.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Two classes of drugs, nucleos(t)ide analogues (NAs) and interferon (IFN), have been approved for the treatment of CHB. For most patients, entecavir and two prodrugs of tenofovir, namely, tenofovir disoproxil fumarate and tenofovir alafenamide, are recommended as the first-line treatment because of their high potency, good safety profile, and very low probability of developing resistance after long-term treatment.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> However, these NAs only inhibit the replication of HBV and cannot disturb its persistence in hepatocytes. HBV surface antigen (HBsAg) loss, which is considered as a functional cure, is only achieved in less than 10% of patients after five years of treatment.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The major reason for HBV persistence is the formation of covalently closed circular DNA (cccDNA) in the nucleus of host cells, which is very stable and acts as a template for HBV transcription.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> IFN has been reported to decrease the amount of cccDNA<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and result in HBsAg seroclearance more frequently than NAs with a 58% cure rate in genotype A infection.<a onclick="showRef(event, 'ref7 ref2'); return false;" href="javascript:void(0);" class="ref ref7 ref2">(7,2)</a> However, the severe side effects and limited efficacy to other genotypes of HBV infection have hindered its use in clinics.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Capsid assembly modulators (CAMs) have recently been proven to be effective for blocking HBV replication in preclinical and clinical studies.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The HBV capsid comprises 120 core protein (Cp) dimers arranged with T=4 icosahedral symmetry.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Core nucleation is the rate limiting step of the self-assembly,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> during which three core protein dimers interact with HBV polymerase and pregenomic RNA (pgRNA) to form a precursor complex.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Then, the Cp dimers are rapidly added to this precursor and interact with each other through hydrophobic interactions to form the capsid. The weak interaction between the subunits enables error correction in the capsid assembly, which is important for correct assembly.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> The HBV genome is synthesized only in the properly formed capsid <i>via</i> reverse transcription using pgRNA as the template.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In addition to enabling reverse transcription, capsid assembly or core proteins regulate almost every step in the HBV life cycle, e.g., transporting the HBV genome to the nucleus and epigenetic regulation of HBV gene expression,<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> making capsid assembly one of the most attractive targets in anti-HBV drug discovery.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">The weak interaction between Cps also provides opportunities for regulation of the capsid assembly process by small molecules. CAMs, through kinetically and thermodynamically modulating capsid assembly, could disturb the proper formation of capsids and regulate the distribution and accessibility of Cps,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> which affords additional therapeutic benefits besides blocking HBV replication. Several CAMs have been reported to inhibit the <i>de novo</i> establishment of cccDNA by modulating the stability of the capsid.<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24">(20−24)</a> Long-term treatment with CAMs could inhibit the expression of HBV e antigen (HBeAg) and HBsAg, and the clearance of the latter has been considered a functional cure of CHB.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Thus, CAMs could potentially increase the functional cure rate for CHB by modulating multiple steps in the HBV life cycle.</div><div class="NLM_p">Several chemotypes of CAMs have been discovered, among which phenylpropenamides (PPAs, e.g., AT130),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> sulfamoylbenzamide (SBA, e.g., DVR-23),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and heteroarylpyrimidines (HAPs, e.g., BAY41-4109)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) are the most studied. Although they each have different chemical structures, all these CAMs bind to the same binding pocket at the Cp dimer–dimer interface.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a> They act as molecular glues to enhance the interaction between the Cps, thus accelerating the self-assembly of Cps and decreasing the formation of functional capsids. However, DVR-23 and AT130 promote the formation of morphologically normal but empty capsids (type II CAMs, CAM-II), while BAY41-4109 misdirects core proteins to form aberrant structures (type I CAMs, CAM-I). This indicates that the difference in the interaction with Cp accounts for the varying effects of CAMs. Both CAM-II (JNJ-6379, Phase II trial; AB-423, Phase I trial) and CAM-I (GLS4, Phase II trial) have been evaluated in clinical trials.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> However, no compounds have been approved. The discovery of new chemotypes of CAMs would provide additional candidates, promoting the development of CAMs for new CHB therapeutics. In addition, different chemotypes could differentially modulate the kinetics or thermodynamics of capsid assembly,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> providing variable effects against HBV compared with present CAMs. Here, we describe the discovery of (1<i>H</i>-indazole-5-yl)sulfonamide and (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide as new chemotypes of CAMs by constraining the conformation of SBAs. The (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide analogues <b>56</b> and <b>59</b> are less cytotoxic and approximately 10-fold more potent than SBA analogues with good metabolic stability in mouse liver microsomes. The citric acid salt (<b>67</b>) of <i>N</i>,<i>N</i>-dimethylglysine prodrug of <b>56</b> has good solubility at various pHs, and dose dependently inhibits <i>in vivo</i> HBV replication in a hydrodynamic injection-based mouse model.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of CAMs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57002" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57002" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">1.  Compound Design and Optimization</h3><div class="NLM_p">Sulfamoylbenzamide (SBA) was recently discovered as a new CAM chemotype from high-throughput screening.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Different from heteroarylpyrimidines (HAPs), SBA treatment induced the formation of morphologically normal capsids, which is characteristic of type II CAMs.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> SBAs bind to the same pocket as HAP at the Cp dimer–dimer interface (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). The halogen-substituted aniline moiety occupies a hydrophobic pocket. The amide forms hydrogen bonds with W102 and T128 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B),<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> which are essential for the inhibitory activity.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The sulfonamide moiety was exposed to the solvent and can tolerate various substituents.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> In addition to SBA and HAP, the PPA compound AT-130<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and several other recently discovered CAMs<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> have been reported to bind to the same pocket, indicating that various chemical scaffolds can be accommodated in this pocket. In the active conformation of SBA, the oxygen atom of the amide and carbon <b>A</b> in the central phenyl are close in space, as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A. To constrain this conformation, we considered whether C═O and carbon <b>A</b> could be cyclized to obtain a new scaffold in which the N atom in the aromatic heterocycle was designed to interact with W102 by hydrogen bonding in place of the C═O (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Various rings, such as phenyl or heterocycles, could be included in the ring B position, which enables structural diversity in lead discovery and optimization. To prove our idea, compounds with indazole or quinazoline scaffold were synthesized. Quinazoline derivatives did not inhibit HBV replication. Compound <b>1</b> with indazole scaffold (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) inhibited extracellular HBV DNA in HepAD38 cells with an EC<sub>50</sub> value of 1.28 μM. We proposed that the 2-N atom at indazole scaffold could form a proper hydrogen bond with W102 of HBV core protein. In addition, the 1-N atom provided an additional site for structure optimization, which distinguishes indazole from other scaffolds. The 1<i>H</i>-indazole-5-sulfonamide compounds were once reported as anti-HBV agents in a patent,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> which increases our confidence in the design concept. However, the structure-activity relationship (SAR) and mechanism of this class of compounds have yet to be elucidated.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of novel CAMs through conformation constraint. (A) Crystal structure of Cps binding with SBA-R01 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T2P">5T2P</a>). (B) Rational design of CAMs. (C) Merging of SBA-R01 and HAP-R01 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>) in the binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Anti-HBV Activity and Cytotoxicity of 1<i>H</i>-Indazole-5-sulfonamide Analogues</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0015.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0016.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The concentration of 50% inhibition for HBV DNA in HepAD38 cell supernatant.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">The concentration that kills 50% of HepAD38 cells.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Selectivity index, SI = CC<sub>50</sub>/EC<sub>50</sub>.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">CC<sub>50</sub> in HepG2 cells.</p></div></div><div></div></div><div class="NLM_p">The effect of substitution at the 1-position was first explored (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). <i>N</i>-methylation resulted in compound <b>2</b> with similar anti-HBV activity (EC<sub>50</sub> = 1.38 μM). However, the selectivity index (SI) of compound <b>2</b> increased compared with that of <b>1</b>. <i>N</i>-cyclopropanation greatly increased the potency compared to that of <b>1</b> by approximately 5-fold (<b>3</b>, EC<sub>50</sub> = 0.19 μM). The increase in activity may result from the occupation of a small hydrophobic pocket, which is close to carbon <b>A</b> and occupied by the 2-substituent of HAP derivatives (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a><i>N</i>-methylation of the aniline moiety (<b>4</b>) and replacement of the aniline with phenol (<b>5</b>), which prevents the hydrogen bond interaction with T128, completely abolished the activity.</div><div class="NLM_p">Although potent inhibition activity was achieved, the SI of these derivatives was relatively low (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Interestingly, replacement of 3-chloro-4-fluoroaniline moiety with 3,4,5-trifluoroaniline greatly decreased cytotoxicity (<b>6</b>, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). To detect the effect of compound <b>6</b> on capsid assembly, electron microscopy (EM) and size exclusion chromatography (SEC) analyses were performed. The <i>N</i>-terminal domain of the core protein (aa 1-149, 5 μM) was incubated with <b>6</b> (10 μM) in a buffer containing 150 mM NaCl. EM images showed that more capsids were formed after treatment with <b>6</b> compared with DMSO (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Two peaks of the DMSO treated core protein appeared in the SEC analysis (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C), indicating the coexistence of the capsids (left peak, elution volume: 1.08 mL, early fraction) and the core protein dimers (right peak, elution volume: 1.97 mL, late fraction).The ratio between the left and right peak area under the curve (AUC) (capsid peak AUC/core protein dimers peak AUC) was used to evaluate the ability of compound <b>6</b> to accelerate the capsid assembly. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D, the SEC peak AUC ratio of compound <b>6</b> was 1.43 while that of DMSO was 0.59, indicating that compound <b>6</b> promoted more core protein dimers to assemble into capsids than DMSO did, consistent with the EM results. The morphology and size of the capsid formed after treatment with <b>6</b> was the same as that of the normal capsid, which was characteristic of type II CAMs.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>6</b> and its effect on capsid assembly. (A) Structure of compound <b>6</b>, (B and C) Compound <b>6</b> accelerated capsid assembly as detected by EM (B) and SEC (C). (D) Calculation of the peak AUC and SEC peak AUC ratio in C. Cp149 protein (5 μM) was incubated with <b>6</b> (10 μM) in a buffer containing 150 mM NaCl and 50 mM HEPES (pH 7.4) at 37 °C overnight prior to EM or SEC detection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To explore the structural diversity of ring B (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), we optimized the scaffold by fluorine scan and <i>N</i>-walking strategy, which is widely applied in lead optimization.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> In this article, fluorination and <i>N</i>-replacement were performed on the three positions of the indazole scaffold (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <b>7</b>–<b>12</b>). Compounds <b>8</b> and <b>11</b> were comparable in potency and cytotoxicity to <b>6</b>, with EC<sub>50</sub> values of 0.13 and 0.24 μM, respectively, while modification at the other positions was not tolerable. Compounds <b>8</b> and <b>11</b> were thus considered proper starting points for structure optimization as the potential improvement in drug-like property. Due to its facile synthesis compared to <b>8</b> and <b>11</b>, indazole analogue <b>6</b> was used as a template for the SAR study. The SAR information was then applied to <b>8</b> and <b>11</b>, as we proposed that modification of the scaffold should not influence the interaction between the other parts of the molecule and Cp.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Anti-HBV Activity and Cytotoxicity of the Compounds Generated by Scaffold Modification</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0017.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0018.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">R<sup>4</sup> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) was supposed to bind to the hydrophobic pocket at Cp dimer interface as the halogen-substituted phenyl in SBA analogues does. Thus, the steric match, which is determined by the substituents on the phenyl group, is vital to the interaction between R<sup>4</sup> and this hydrophobic pocket.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Removal of the fluorine at the <i>meta</i> position (<b>13</b>, EC<sub>50</sub> = 0.22 μM, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) was tolerable and maintained potency compared with <b>6</b>. The 4-fluorophenyl group is a common fragment in several chemotypes of CAMs.<a onclick="showRef(event, 'ref26 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref26 ref32 ref33 ref34">(26,32−34)</a> In this case, replacement of the <i>p</i>-fluorine with a larger group, such as chlorine (<b>15</b>) and methyl (<b>14</b>), led to a 10-fold decrease or even a loss in activity, probably due to steric hindrance. Removal of the <i>para</i> fluorine (<b>16</b>) also resulted in diminished activity compared with <b>13</b>. These data suggested that the <i>para</i> fluorine-substituted aniline is essential for ligand–target interactions. Introducing chlorine and methyl groups at the 2-position reduced the activity by more than 5-fold (<b>17</b> and <b>18</b>). The addition of another fluorine or chlorine at the 3-position, which was expected to enhance hydrophobic interactions, did not significantly influence the anti-HBV activity (<b>19</b> in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <b>3</b> in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), suggesting that a 3-fluorine or 3-chlorine group did not cause steric hindrance and was not perfectly matched with the target. Inspired by the above result, we replaced the 3-substituents with a larger group, such as methyl, leading to <b>20</b>, which was approximately 10-fold more potent than <b>13</b> with an EC<sub>50</sub> value of 0.032 μM.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Effect of Aniline Substituents on Anti-HBV Activity and Cytotoxicity</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0019.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0020.gif" alt="" id="fx6" /></img><div></div></div><div class="NLM_p">Although Cl and CH<sub>3</sub> are isosteres with a similar size, the van der Waals radius of CH<sub>3</sub> (2 Å) is slightly larger than Cl (1.75 Å). As F, an electron-withdrawing atom like Cl, significantly contributed to the increase in potency, we proposed that the lower potency of <b>3</b> compared to that of <b>20</b> is partly, if not all, due to the smaller volume of Cl than CH<sub>3</sub> instead of the electron-withdrawing effect of Cl. Replacing the phenyl with a pyridinyl (<b>22</b>, <b>23</b>, and <b>24</b>) or introducing a polar group (CN in <b>21</b>) resulted in a loss of activity, consistent with the hydrophobic nature of the pocket accommodating the aniline moiety. To increase the flexibility of the compound, a methylene was inserted between the phenyl and the NH (<b>25</b> and <b>26</b>). The benzyl amine fragment was also present in other SBA analogues.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, benzyl amine substituents were not tolerable in the indazole scaffold (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Aliphatic substituents also resulted in a great decrease in potency (<b>27</b> and <b>28</b>). Although <b>20</b> was the most potent compound obtained from the optimization of R<sup>4</sup>, it was somewhat more cytotoxic than the others.</div><div class="NLM_p">To reduce the log <i>P</i> value of the compounds, the cyclopropane was replaced with trimethylene oxide (<b>29</b>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). This led to a loss in activity, suggesting the requirement of hydrophobic substituents at R<sup>1</sup>. Replacement of the cyclopropane with a smaller (ethyl, <b>30</b>) or larger (isopropyl in <b>31</b> or cyclobutyl in <b>32</b>) group significantly diminished the anti-HBV activity with EC<sub>50</sub> values >1 μM. As the sulfonamide moiety was supposed to bind in the solvent-exposed region and interact with the target through a water-mediated network,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> fragments containing hydrophilic groups, which enable the interaction with water, were included. Similar anti-HBV activity but varied cytotoxicity was achieved with various substitutions at the R<sup>3</sup> position. Replacement of the 4-hydroxyl piperidine with morpholine led to compound <b>33</b> with similar potency (EC<sub>50</sub> = 0.43 μM). Moving the hydroxyl group from the 4-position to the 3-position of piperidine did not affect the activity (EC<sub>50</sub> = 0.25 μM) but greatly increased the cytotoxicity (<b>34</b>). Carboxylic acid at R<sup>3</sup> was detrimental to anti-HBV activity (<b>35</b>). Replacement of the piperidine with pyrrolidine (<b>36</b>) or insertion of one carbon atom between the hydroxyl and piperidine (<b>37</b>) reduced the potency by 10-fold and increased the cytotoxicity. Other groups containing oxygen at the 4-position of piperidine (<b>38</b> and <b>39</b>) were well-tolerated.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Effects of R<sup>1</sup> and R<sup>3</sup> on Anti-HBV Activity and Cytotoxicity</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0021.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0022.gif" alt="" id="fx8" /></img><div></div></div><div class="NLM_p">Considering both HBV inhibition activity and cytotoxicity, we combined the optimal substituents from <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Tables <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. As described above, the introduction of a 3-methyl group on the aniline moiety (<b>20</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) greatly improved the anti-viral activity. We also found that different substitutes on the sulfonamide moiety resulted in variations in cytotoxicity. Compound <b>40</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), designed by combining 3-methyl-4-fluorophenyl at the R<sup>4</sup> position with morpholinyl at the R<sup>3</sup> position, inhibited HBV replication with an EC<sub>50</sub> value of 0.057 μM, without cytotoxicity (CC<sub>50</sub> > 100 μM). The introduction of these two substitutes into the scaffold of <b>11</b> but not <b>8</b> improved the anti-viral activity by 5-fold without increasing cytotoxicity (<b>42</b> and <b>41</b>, <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Compound <b>42</b> inhibited HBV replication with an EC<sub>50</sub> value of 0.042 μM, without cytotoxicity (CC<sub>50</sub> > 100 μM). Therefore, compounds <b>40</b> and <b>42</b> were selected for further optimization.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Anti-HBV Activity and Cytotoxicity of Compounds Generated by Combining the Optimal Substituents with Different Scaffolds</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0023.gif" alt="" id="fx9" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center" char=".">EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">HLM/MLM clint (μL/min/mg)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">0.032</td><td class="colsep0 rowsep0" align="left">11.0</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>40</b></td><td class="colsep0 rowsep0" align="char" char=".">0.057</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">41.0/26.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>41</b></td><td class="colsep0 rowsep0" align="char" char=".">0.21</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>42</b></td><td class="colsep0 rowsep0" align="char" char=".">0.042</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">38.0/53.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">The intrinsic clearance of compounds by human or mouse liver microsome (HLM/MLM).</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">ND, not determined.</p></div></div></div><div class="NLM_p">The metabolic stability in liver microsomes of <b>40</b> and <b>42</b> was low (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). To improve the metabolic stability, we replaced the methyl group at R<sup>4</sup> in <b>40</b> and <b>42</b>, which is prone to be oxidized by CYP450,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> with a F (<b>43</b>–<b>45</b>, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), CF<sub>3</sub> (<b>46</b>–<b>48</b>), or cyclopropanyl group (<b>49</b>). These modifications resulted in a 5- to 100-fold decrease in potency compared with <b>40</b> and <b>42</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). This was most likely because of the improper steric match of these groups with Cp. The cyclopropanyl group at the 1-position is another site prone to metabolism through N-dealkylation mediated by CYP450.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> We replaced cyclopropanyl with fluorine-substituted methyl or ethyl groups, which are less liable to CYP450-mediated oxidation due to the electron-withdrawing effects of fluorine and the higher strength of the C–F bond.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Although less potent than <b>40</b>, compounds <b>50</b>–<b>52</b> inhibited HBV replication with submicromolar EC<sub>50</sub> values. However, difluoromethyl substitution (<b>53</b>) significantly decreased the anti-HBV activity, probably because the strong electron-withdrawing effect of the difluoromethyl group reduced the hydrogen bond acceptor ability of the N atom at the 2-position. All of these attempts to block metabolism sites were discontinued due to the accompanying decreases in potency. Another popular approach to improve metabolic stability is to reduce the lipophilicity of the compound.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> Replacing the morpholine with its one-carbon bridged analogue (<b>54</b>), which has been reported to have lower lipophilicity,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> reduced the clogP by 0.85 units compared to <b>40</b>. Compound <b>54</b> inhibited HBV replication with an EC<sub>50</sub> value of 0.045 μM, similar to that of <b>40</b>. However, the cytotoxicity greatly increased compared to <b>40</b> (CC<sub>50</sub> 19.0 vs > 100 μM). The 3-methyl group at R<sup>4</sup> in compounds <b>40</b> and <b>42</b> was found to significantly contribute to the increased anti-HBV activity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). We intended to investigate the contribution of cyclopropane in <b>40</b>. The removal of cyclopropane resulted in a 1.07-unit reduction in clogP (<b>55</b>, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Compound <b>55</b> inhibited HBV replication at an EC<sub>50</sub> of 0.25 μM, which is less potent than <b>40</b>, indicating that both the cyclopropane and 3-methyl groups significantly contribute to the anti-HBV activity. Optimization of the solvent-exposed sulfonamide group was another approach to reduce the lipophilicity without significantly decreasing potency. Unfortunately, most analogues of <b>40</b> with various substituents at the R<sup>3</sup> position were cytotoxic with CC<sub>50</sub> values <30 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). Compound <b>56</b>, designed by replacing the morpholine in compound <b>42</b> with the more hydrophilic 4-hydroxypiperidine, inhibited HBV replication with an EC<sub>50</sub> value of 0.042 μM, without cytotoxicity at 100 μM in HepAD38 cells. Interestingly, the metabolic stability of <b>56</b> in mouse liver microsomes (MLMs) was significantly improved compared to <b>42</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). To block metabolism at the benzyl site, we replaced methyl with a difluoromethyl (<b>57</b>–<b>60</b>), a less lipophilic bioisostere of the methyl group.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The anti-viral activity of these compounds decreased slightly. Indazole derivatives <b>57</b> and <b>58</b> were found to be more cytotoxic than the 1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives <b>59</b> and <b>60</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>), consistent with the above result of the matched pair compounds <b>20</b> and <b>56</b>. Although replacement of methyl with difluoromethyl did not reduce the lipophilicity, we observed that the metabolic stability of <b>59</b> increased compared with <b>56</b> (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Structural Modifications to Improve the Metabolic Stability</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0024.gif" alt="" id="fx10" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0025.gif" alt="" id="fx11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Calculated by the ACD/Percepta platform.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Mouse Liver Microsome Metabolism of <b>56</b> and <b>59</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center" char="±">EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">CC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">MLM clint (μL/min/mg)<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>56</b></td><td class="colsep0 rowsep0" align="char" char="±">0.034 ± 0.012</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="left">10.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>59</b></td><td class="colsep0 rowsep0" align="char" char="±">0.071 ± 0.035</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="left">6.90</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">The intrinsic clearance of compounds by mouse liver microsome (MLM).</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">2.  <i>In Vitro</i> Effects of Compounds <b>56</b> and <b>59</b> in HepAD38 and HepG2.2.15 Cells</h3><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, compounds <b>56</b> and <b>59</b> accelerated the capsid assembly and behaved as type II CAMs. To clarify their effect on the HBV life cycle, we treated HepAD38 and HepG2.2.15 cells with <b>56</b>, <b>59</b>, positive control CAMs (AT130 and GLS4) and Lamivudine (3TC) at 2 μM for 4 days. At the extracellular level, all these compounds inhibited HBV replication in both cell lines, reducing extracellular HBV DNA by approximately 90% at 2 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). None of the compounds, including the positive controls, could inhibit the secretion of HBsAg in either cell line. The inhibition of HBsAg secretion by CAMs was only observed in the <i>de novo</i> infection models after long-term treatment,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> resulting from the inhibition of <i>de novo</i> synthesis of cccDNA. The secretion of HBeAg by HepAD38 cells was reduced by approximately 50% after treating with 4 CAMs. However, only GLS4 could weakly decrease HBeAg in HepG2.2.15 cells (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Inhibition of HBeAg was most probably due to the decrease in intracellular RNA accumulation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C) or a direct effect on HBeAg by CAMs, which could inhibit HBeAg secretion by inducing precore protein self-assembly,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> as CAMs did not inhibit cccDNA in such cell lines after short-term treatment.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <i>In vitro</i> effects of compounds <b>56</b> and <b>59</b> in HepAD38 and HepG2.2.15 cells. (A) EM analysis of the effects on capsid assembly. (B and C) Changes of HBV extracellular and intracellular parameters in HepAD38 and HepG2.2.15 cells after treatment with AT130, 3TC, GLS4, <b>56</b>, and <b>59</b> at 2 μM. Cells were treated with compounds or DMSO for 4 days. The medium was replaced every 2 days. DNA and RNA were quantified with qPCR or qRT-PCR. HBeAg and HBsAg were quantified by ELISA (VC, vehicle control). The results are presented as the mean ± SD. Statistics were determined by two-tailed <i>t</i> test. *, <i>P</i> < 0.05; **, <i>P</i> < 0.01; ***, <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">CAMs inhibit reverse transcription by reducing the encapsulation of pgRNA into the capsid. Thus, intracellular DNA levels in both cell lines were significantly reduced after treatment with CAMs (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). AT130, <b>56</b>, and <b>59</b> decreased intracellular total RNA in HepAD38 cells by 50%. GLS4 also induced a weak but significant decrease in the total intracellular RNA in HepAD38 cells. We proposed that the reduction of intracellular RNA may be due to the increased exposure to the cytoplasm as reduced encapsulation into the capsid. However, more details about which RNA decreased are needed to fully understand this phenomenon. The reverse transcriptase (RT) inhibitor 3TC slightly induced intracellular RNA accumulation due to the inhibition of reverse transcription. In HepG2.2.15 cells, only <b>56</b> and <b>59</b> could significantly reduce intracellular RNA, probably due to the different properties of each cell type.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">3.  Compounds <b>56</b> and <b>59</b> Suppressed Viral Replication of Both Lamivudine-Resistant and Multidrug-Resistant HBV Strains</h3><div class="NLM_p">As CAMs have different action mechanisms with NAs, they should be effective against HBV that is resistant to RT inhibitors. We then tested the effects of the compounds on HBV DNA in a lamivudine/entecavir (3TC/ETV)-resistant cell line (HepG2.A64) containing rtL180M+rtT184L+rtM204 V mutations and a multidrug-resistant (MDR) cell line (HepG2.1403F) containing rtL180M+rtM204 V+rtS202G+ rtN236T mutations.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> As shown in <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>, compounds <b>56</b> and <b>59</b> inhibited HBV replication in the HepG2.A64, HepG2.1403F, HepG2.2.15, and HepAD38 cell lines with EC<sub>50</sub> values well below 0.1 μM. Both compounds did not influence the viability of HepG2 cells with CC<sub>50</sub> values >100 μM. Although 3TC and ETV potently inhibited HBV replication in HepG2.2.15 and HepAD38 cells, they did not show a significant inhibitory effect even at 10 μM in HepG2.A64 and HepG2.1403F cells. The activity of tenofovir (TDF) on the resistant strains was also more than 100-fold weaker than that on HepG2.2.15 and HepAD38 cells. These results further proved the potential of CAMs in CHB therapy.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Anti-HBV Activity in Wild Type (HepAD38 and HepG2.2.15 Cells), Lamivudine-Resistant (HepG2.A64 Cells) and Multidrug-Resistant (HepG2.1403F cells) HBV Strains</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">HepG2.2.15</th><th class="colsep0 rowsep0" align="center">HepAD38</th><th class="colsep0 rowsep0" align="center">HepG2.A64</th><th class="colsep0 rowsep0" align="center">HepG2.1403F</th><th class="colsep0 rowsep0" align="center">HepG2</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">EC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" align="center">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3TC</td><td class="colsep0 rowsep0" align="left">0.0038 ± 0.0033</td><td class="colsep0 rowsep0" align="left">0.070 ± 0.019</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TDF</td><td class="colsep0 rowsep0" align="left">6.4 × 10<sup>–05</sup> ± 7.62 × 10<sup>–06</sup></td><td class="colsep0 rowsep0" align="left">0.010 ± 0.0017</td><td class="colsep0 rowsep0" align="left">0.84 ± 0.36</td><td class="colsep0 rowsep0" align="left">2.52 ± 1.04</td><td class="colsep0 rowsep0" align="left">15.04 ± 3.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ETV</td><td class="colsep0 rowsep0" align="left">2.2 × 10<sup>–06</sup> ± 3.71 × 10<sup>–07</sup></td><td class="colsep0 rowsep0" align="left">0.011 ± 0.011</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">42.07 ± 2.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">0.0035 ± 0.0012</td><td class="colsep0 rowsep0" align="left">0.034 ± 0.012</td><td class="colsep0 rowsep0" align="left">0.00025 ± 3.06 × 10<sup>–05</sup></td><td class="colsep0 rowsep0" align="left">0.00033 ± 1.5 × 10<sup>–04</sup></td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">0.042 ± 0.010</td><td class="colsep0 rowsep0" align="left">0.071 ± 0.035</td><td class="colsep0 rowsep0" align="left">0.0022 ± 0.00087</td><td class="colsep0 rowsep0" align="left">0.013 ± 0.0032</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AB-423</td><td class="colsep0 rowsep0" align="left">0.28 ± 0.074</td><td class="colsep0 rowsep0" align="left">0.31 ± 0.092</td><td class="colsep0 rowsep0" align="left">0.048 ± 0.028</td><td class="colsep0 rowsep0" align="left">0.074 ± 0.011</td><td class="colsep0 rowsep0" align="left">39.85 ± 5.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GLS4</td><td class="colsep0 rowsep0" align="left">0.0066 ± 0.0020</td><td class="colsep0 rowsep0" align="left">0.013 ± 0.0045</td><td class="colsep0 rowsep0" align="left">0.0063 ± 0.0017</td><td class="colsep0 rowsep0" align="left">0.042 ± 0.012</td><td class="colsep0 rowsep0" align="left">49.2 ± 1.30</td></tr></tbody></table></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">4.  Design of Amino Acid Prodrugs to Improve Anti-HBV Activity <i>in Vivo</i></h3><div class="NLM_p">The rigid and planar structure together with a high fraction of sp2 carbons led to the poor solubility of compound <b>56</b> and its analogues (as shown in <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). Salt formation is an efficient method to increase compound solubility.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> However, <b>56</b> and the related analogues are weak bases, which are not ideal for salt formation. Thus, amino acid prodrugs were designed, whose amino group was utilized for salt formation (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>). Several amino acids, including <i>N</i>,<i>N</i>-dimethylglycine, <span class="smallcaps smallerCapital">l</span>-valine, <span class="smallcaps smallerCapital">l</span>-alanine, and glycine, were attached to the hydroxyl group of <b>56</b> through an ester bond. Different α substituents were selected to modulate the stability of the prodrug. To evaluate the stability of these prodrugs, the percent of remaining compound after incubation in plasma and different buffers was determined by HPLC. Except for <b>62</b>, all the other prodrugs were rapidly hydrolyzed in plasma (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>). The steric hindrance of the <i>i</i>-Pr in <b>62</b> may account for the resistance to hydrolysis. However, a significant portion of compounds <b>63</b> and <b>64</b> was hydrolyzed in the aqueous buffers, indicating that they are unstable in the digestive tract. Compound <b>61</b> was chosen for salt screening to increase solubility.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Stability of the Prodrugs in Plasma and Buffers<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0026.gif" alt="" id="fx12" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">percent remaining</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center">R′</th><th class="colsep0 rowsep0" align="center">R″</th><th class="colsep0 rowsep0" align="center">half-life in plasma</th><th class="colsep0 rowsep0" align="center">pH 7.4</th><th class="colsep0 rowsep0" align="center">pH 6.8</th><th class="colsep0 rowsep0" align="center">pH 5.8</th><th class="colsep0 rowsep0" align="center">0.1 M HCl</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">47.2 min</td><td class="colsep0 rowsep0" align="left">stable<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"><a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">stable</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="left"><i>i</i>-Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">18 h</td><td class="colsep0 rowsep0" align="left">stable</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">31.2 min</td><td class="colsep0 rowsep0" align="left">55.6%<a class="ref internalNav" href="#t9fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">80.2%</td><td class="colsep0 rowsep0" align="left">85.7%</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">77.5 min</td><td class="colsep0 rowsep0" align="left">74.2%</td><td class="colsep0 rowsep0" align="left">67.6%</td><td class="colsep0 rowsep0" align="left">88.3%</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">Compounds were incubated in buffer at 37 °C.</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">No hydrolysis after 2 h at 37 °C.</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">Not detected.</p></div><div class="footnote" id="t9fn4"><sup><sup>d</sup></sup><p class="last">Percent remaining of the compound after 3 h.</p></div></div></div><div class="NLM_p">Introduction of the amino acid moiety indeed improved the solubility in water (<b>65</b>, <named-content content-type="anchor" rid="tbl10" type="simple"></named-content><a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>), probably due to the increased fraction of sp3 carbons and the basicity. The TFA salt further increased the solubility (<b>61</b>, <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). However, the solubility of <b>61</b> in pH 6.8 and 7.4 buffers was very low, which indicates poor solubility in the digestive tract. To increase the solubility at various pH values, other salts with acetic acid, citric acid, maleic acid, methanesulfonic acid, and hydrochloric acid were prepared (<b>66</b>–<b>70</b>, <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). As excess salt in the detection samples would change the pH of the dissolvent, 1 mg of compound in 100 μL of buffer was used to determine the solubility (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). The more acidic acids used in the salt led to increased solubility (<b>66</b> vs <b>68</b> and <b>69</b>). Hydrophilic groups on the acid also increased the solubility (<b>67</b> vs <b>70</b>). The solubility decreased with increasing pH (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>). The citric acid salt <b>66</b> was selected as the acceptable solubility in various buffers and a low degree of hygroscopicity.</div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. Solubility of the Prodrugs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0014.gif" alt="" id="fx13" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">solubility (mg/mL)<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">compound</th><th class="colsep0 rowsep0" align="center">acid</th><th class="colsep0 rowsep0" align="center">water</th><th class="colsep0 rowsep0" align="center">pH 5.8</th><th class="colsep0 rowsep0" align="center">pH 6.8</th><th class="colsep0 rowsep0" align="center">pH 7.4</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>56</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>61</b></td><td class="colsep0 rowsep0" align="left">TFA</td><td class="colsep0 rowsep0" align="left">1.51</td><td class="colsep0 rowsep0" align="left">0.29</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>65</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.71</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>66</b></td><td class="colsep0 rowsep0" align="left">AcOH</td><td class="colsep0 rowsep0" align="left">1.11</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>67</b></td><td class="colsep0 rowsep0" align="left">citric Acid</td><td class="colsep0 rowsep0" align="left">8.13</td><td class="colsep0 rowsep0" align="left">7.60</td><td class="colsep0 rowsep0" align="left">6.65</td><td class="colsep0 rowsep0" align="left">0.031</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>68</b></td><td class="colsep0 rowsep0" align="left">(MeSO<sub>3</sub>H)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">7.90</td><td class="colsep0 rowsep0" align="left">7.08</td><td class="colsep0 rowsep0" align="left">7.69</td><td class="colsep0 rowsep0" align="left">0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>69</b></td><td class="colsep0 rowsep0" align="left">HCl</td><td class="colsep0 rowsep0" align="left">6.90</td><td class="colsep0 rowsep0" align="left">6.75</td><td class="colsep0 rowsep0" align="left">0.040</td><td class="colsep0 rowsep0" align="left">0.016</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>70</b></td><td class="colsep0 rowsep0" align="left">maleic acid</td><td class="colsep0 rowsep0" align="left">4.20</td><td class="colsep0 rowsep0" align="left">7.93</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">ND</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup><sup>a</sup></sup><p class="last">Solubility in water or 0.067 M phosphate buffer. ND, not detectable, below the detection limit.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">5.  Anti-HBV Activity of <b>56</b> and <b>67</b> in the Hydrodynamic Injection (HDI) HBV Mouse Model</h3><div class="NLM_p">To evaluate the anti-HBV activity of <b>56</b> and <b>67</b>, a hydrodynamic injection-based mouse model of HBV infection was used. In this mouse model, the HBV genome was transiently transduced into the hepatocytes of C57BL/6 mice under high pressure conditions, and inhibition of HBV replication could be evaluated during the transient time of about 7 days.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Although this animal model cannot mimic the full cycle of HBV infection, it was commonly used to evaluate the inhibition of HBV replication and liver exposure of the compounds. Compounds <b>56</b> and <b>67</b> were chosen for <i>in vivo</i> anti-HBV evaluation in HDI model due to the higher potency and metabolic stability of <b>56</b> and the increased solubility of <b>67</b>. Oral administrations once a day at doses of 50 and 100 mg/kg were performed. Once daily oral dose of 100 mg/kg AB-423 and 0.03 mg/kg ETV were used as positive controls. However, neither <b>56</b> nor <b>67</b> as well as AB-423 significantly inhibited HBV replication in mice (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>). In the preclinical study, 100 mg/kg AB-423 was orally administrated twice daily for <i>in vivo</i> anti-HBV evaluation. Thus, we adopted the administration frequency of twice a day, to elucidate the therapeutic potential of this class of CAMs. After 7 days of treatment, 100 mg/kg <b>67</b> achieved significant inhibition of both serum (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) and liver (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) HBV DNA levels compared to vehicle control, whereas 50 mg/kg <b>67</b> only achieved weak inhibition of liver HBV DNA levels. Both doses of <b>56</b> did not lead to a significant anti-HBV effect (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), which probably resulted from poor absorption due to poor solubility.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Anti-HBV activity in an HDI HBV mouse model. (A) Serum HBV DNA copies after hydrodynamic injection. (B) Liver HBV DNA on the seventh day after hydrodynamic injection. <i>N</i> = 5; *, <i>P</i> < 0.05; **, <i>P</i> < 0.01; ***, <i>P</i> < 0.001; ****, <i>P</i> < 0.0001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The dose-dependent inhibition of <i>in vivo</i> HBV replication inspired us to further evaluate the <i>in vivo</i> anti-HBV activity of <b>67</b>. As the molecular weight of <b>67</b> is nearly twice that of AB-423, we increased the dose to 200 mg/kg. However, no significant improvement in the inhibition of HBV replication was observed (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf" class="ext-link">Supporting Information Figure S2</a>). This may result from a limited absorption rate, induction of the expression of metabolic enzymes, etc. Blood and liver concentrations of <b>56</b> after oral administration of <b>56</b> and its prodrug <b>67</b> were investigated. It was indicated that <b>56</b> was quickly eliminated in mice with a clearance of 0.14 L/h and a short half-life time of 0.63 h (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>). Oral bioavailability of <b>56</b> was calculated as 12%. The blood concentration (1.19 μM) of <b>56</b> reached maximum (<i>C</i><sub>max</sub>) at 30 min after administration of 100 mg/kg parent compound <b>56</b> (<a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>). At the dose of 100 mg/kg, oral bioavailability of <b>67</b> reached 87.6% (<a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>), much higher than that of parent compound <b>56</b> (12%). At the dose of 200 mg/kg, the oral bioavailability of <b>67</b> was increased to 132%, which might be resulted from a phenomenon of saturated metabolic enzyme.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The blood concentration of <b>56</b> reached <i>C</i><sub>max</sub> values of 3.73 and 8.84 μM at 1 h after administrating <b>67</b> with the doses of 100 and 200 mg/kg, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>). It was noted that a very low level of the prodrug <b>67</b> in blood of mice was observed (data not shown). Considering the fact that the half-life time of <b>67</b> in mice plasma was 47 min, we speculated that <b>67</b> could be hydrolyzed by additional enzymes in the small intestine and liver besides plasma.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0006.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Pharmacokinetics of compounds <b>56</b> and <b>67</b> in male C57BL/6 mice. (A) Plasma concentration of <b>56</b> after administration of <b>56</b> and its prodrug <b>67</b>. (B) Liver concentration of <b>56</b> and its prodrug (<i>N</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl11"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 11. Pharmacokinetics of <b>56</b> and <b>67</b> in Mice</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="center" /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col><col align="center" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">dose</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">CL (L/h)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>plasma</sub> (μg·h/L)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>Liver</sub> (nmol·h/g)</th><th class="colsep0 rowsep0" align="center" char=".">blood <i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">blood <i>C</i><sub>max</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">liver <i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">liver <i>C</i><sub>max</sub> (nmol/g)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>56</b> i.v. 20 mg/kg</td><td class="colsep0 rowsep0" align="center" char=".">0.63</td><td class="colsep0 rowsep0" align="center" char=".">0.14</td><td class="colsep0 rowsep0" align="center" char=".">2597</td><td class="colsep0 rowsep0" align="center" char=".">62</td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>56</b> p.o. 100 mg/kg</td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char=".">1580</td><td class="colsep0 rowsep0" align="center" char=".">142</td><td class="colsep0 rowsep0" align="center" char=".">0.5</td><td class="colsep0 rowsep0" align="center" char=".">1.19</td><td class="colsep0 rowsep0" align="center" char=".">0.5</td><td class="colsep0 rowsep0" align="center" char=".">32</td><td class="colsep0 rowsep0" align="center" char=".">12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>67</b> p.o. 100 mg/kg</td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char=".">7013.2</td><td class="colsep0 rowsep0" align="center" char=".">306</td><td class="colsep0 rowsep0" align="center" char=".">1.0</td><td class="colsep0 rowsep0" align="center" char=".">3.73</td><td class="colsep0 rowsep0" align="center" char=".">2</td><td class="colsep0 rowsep0" align="center" char=".">40</td><td class="colsep0 rowsep0" align="center" char=".">87.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center"><b>67</b> p.o. 200 mg/kg</td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char="."> </td><td class="colsep0 rowsep0" align="center" char=".">21106</td><td class="colsep0 rowsep0" align="center" char=".">482</td><td class="colsep0 rowsep0" align="center" char=".">1.0</td><td class="colsep0 rowsep0" align="center" char=".">8.84</td><td class="colsep0 rowsep0" align="center" char=".">4</td><td class="colsep0 rowsep0" align="center" char=".">59</td><td class="colsep0 rowsep0" align="center" char=".">132</td></tr></tbody></table></div></div><div class="NLM_p last">Parent compound <b>56</b> distributed fast to the liver after either intravascular or oral administration of <b>56</b> or prodrug <b>67</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). More than 10 nmol/g in the liver was detectable at 5 min after oral administration of <b>56</b> or <b>67</b>. The liver <i>C</i><sub>max</sub> of <b>56</b> reached 32 nmol/g at 0.5 h (<i>T</i><sub>max</sub>) after oral administration of <b>56</b>, corresponding with the <i>T</i><sub>max</sub> of <b>56</b> in blood. Fortunately, the liver <i>C</i><sub>max</sub> of <b>56</b> after oral administration of prodrug <b>67</b> with doses of 100 and 200 mg/kg reached 40 nmol/g at 2 h and 59 nmol/g at 4 h, respectively, which were much higher than that after oral administration of parent compound <b>56</b> (<a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>). Correspondingly, the liver exposure of <b>56</b> achieved by orally administrating 100 mg/kg <b>67</b> was 2-fold higher than that achieved by orally administrating 100 mg/kg <b>56</b>. Unexpectedly, 1.5-fold increase in liver exposure of <b>56</b> was observed at the dose of 200 mg/kg <b>67</b> comparing with that at 100 mg/kg (<a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>). One possibility was that a high concentration of the compound induced the metabolic enzyme overexpression.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> These results proved the advantage of amino acid prodrug in improving the exposure and <i>in vivo</i> anti-HBV efficacy of (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide derivative <b>56</b>.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">6.  Molecule Modeling</h3><div class="NLM_p">To gain insight to the interaction between <b>56</b> and HBV core protein, molecule modeling was performed. We carried out docking on the basis of the cocrystal structure of HAP-R01 and core protein (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>). The results showed that compound <b>56</b> binds to the core protein dimer–dimer interface with a similar interaction mode as that of SBAs and HAPs (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The modeling results show the formation of a hydrogen bond between 2-N and W102. The aniline moiety pointed to a hydrophobic pocket formed by the two core protein dimers, which is occupied by a halogen-substituted phenyl group of SBA or HAP derivatives (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B,C). The fluorine atom and methyl group of <b>56</b> were in tight contacts with the core protein, consistent with the SAR study. The sulfonamide moiety pointed to the solvent exposing region. The oxygen of sulfonamide formed another hydrogen bond with a water molecule (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A). The cyclopropanyl group of <b>56</b> bound to the pocket where the thiazolyl group of HAP-R01 binds (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C), which should contribute to the increased potency of <b>56</b>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0007.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Predicted binding mode of <b>56</b>. (A) Docking of <b>56</b> with HBV core protein crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>). (B) Merging of <b>56</b> and NVR 3-778 at the binding site. (C) Merging of <b>56</b> and HAP-R01 at the binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The preparation of (1<i>H</i>-indazol-5-yl)sulfonamide derivatives began with the cross coupling of benzyl mercaptan and 5-bromo-1<i>H</i>-indazol analogues (<b>A</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), whose preparation is described in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf" class="ext-link">Supporting Information</a>, using a reported method.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Oxidative chlorination<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> and reaction with the corresponding amines afforded sulfonamide derivatives (<b>C</b>). Intermediate <b>D</b> was prepared by bromination of <b>C</b> using NBS or 1,3-dibromo-5,5-dimethylhydantoin in a mixture of acetonitrile and acetic acid at room temperature or 40 °C. The target compounds (<b>E</b>) were synthesized by Buchwald–Hartwig cross coupling under the catalysis of Pd(OAc)<sub>2</sub> and xantphos (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0009.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route for the Indazole Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnSH (1.5 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5% mol), xantphos (10% mol), DIPEA (2 equiv), 1,4-dioxane, 90 °C; (b) i. 1,3-dibromo-5,5-dimethylhydantoin (2 equiv), −30 °C, CH<sub>3</sub>CN, H<sub>2</sub>O, AcOH; ii. 4-hydroxypiperidine (2 equiv), Et<sub>3</sub>N (3 equiv), DCM, room temperature; (c) NBS (1.5 equiv), or 1,3-dibromo-5,5-dimethylhydantoin, CH<sub>3</sub>CN/AcOH; (d) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; (e) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; ii. LiOH (3 equiv), THF, H<sub>2</sub>O, room temperature.</p></p></figure><div class="NLM_p">The preparation of <b>1</b> began with the cross coupling of benzyl mercaptan and 5-bromo-1<i>H</i>-indazol, followed by oxidative chlorination, coupling with 4-hydroxypiperidine, and bromination as described above. Compound <b>75</b> was afforded by protecting the hydroxyl group and 1-position N-atom using acetyl and tetrahydropyran, respectively. After cross coupling with 3-chloro-4-fluoroaniline and deprotection, <b>1</b> was obtained (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnSH (1.5 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5% mol), xantphos (10% mol), DIPEA (2 equiv), 1,4-dioxane, 90 °C, 92%; (b) i. 5,5-dimethylhydantoin (2 equiv), −30 °C, CH<sub>3</sub>CN, H<sub>2</sub>O, AcOH; ii. 4-hydroxypiperidine (2 equiv), Et<sub>3</sub>N (3 equiv), DCM, room temperature, 41%; (c) NBS (1.1 equiv), CHCl<sub>3</sub>, 60 °C; (d) i. DHP (2 equiv), TsOH H<sub>2</sub>O (0.3 equiv), EA, reflux; ii. Ac<sub>2</sub>O (1 equiv), 86% for three steps; (e) i. 3-chloro-4-fluoroaniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 110 °C; ii. LiOH (3 equiv), THF, H<sub>2</sub>O, rt, 89%; (f) HCl, EtOH, room temperature, quantitative.</p></p></figure><div class="NLM_p">Synthesis of 1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridine derivatives started from the reduction of the nitro group in <b>77</b> using Raney Ni and hydrogen, followed by acylation of the amino group. Then, <b>79</b> reacted with isoamyl nitrite to afford <b>80</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The target compounds were prepared following the protocol described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route of 1<i>H</i>-Pyrazolo[3,4-<i>c</i>]pyridine Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Raney Ni, H<sub>2</sub>, methanol, room temperaturet, quantitative; (b) Ac<sub>2</sub>O (3 equiv), PhCH<sub>3</sub>, 100 °C, 85%; (c) isoamyl nitrite (2.5 equiv), Ac<sub>2</sub>O (2 equiv), KOAc (2 equiv), 18-crown-6 (0.1 equiv), PhCH<sub>3</sub>, 80 °C, then NaOH (4 equiv), room temperature, 63%; (d) BnSH (1.5 equiv), Pd<sub>2</sub> (dba)<sub>3</sub> (5% mol), xantphos (5% mol), DIPEA (2 equiv), 1,4-dioxane, 90 °C, 87%; (e) cyclopropylboronic acid (2 equiv), Cu(OAc)<sub>2</sub> (1 equiv), 2,2′-bipyridine (1 equiv), Na<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,2-DCE, 70 °C, 51%; (f) i. 5,5-dimethylhydantoin (2 equiv), −15 °C, CH<sub>3</sub>CN, H<sub>2</sub>O (1% v/v), AcOH (1% v/v); ii. 4-hydroxypiperidine (for <b>83a</b>, 2 equiv) or morpholine (for <b>83b</b>, 2 equiv), Et<sub>3</sub>N (3 equiv), DCM, room temperature; (g) for <b>84a</b> i. Ac<sub>2</sub>O (5 equiv), pyridine (4 equiv), ethyl acetate, reflux; ii. NBS (1.5 equiv), TFA/AcOH (1:1), 50 °C, 50% for 2 steps; (h) for <b>84b</b> 1,3-dibromo-5,5-dimethylhydantoin (1.5 equiv), CH<sub>3</sub>CN/AcOH (1:1), 50 °C, 59% for 2 steps; (i) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; (j) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; ii. LiOH (3 equiv), THF, H<sub>2</sub>O, room temperature.</p></p></figure><div class="NLM_p">Compound <b>5</b> was prepared by the CuI-catalyzed cross coupling of 3,4,5-trifluorophenol and <b>85</b>, which was synthesized by iodination of 1-((1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-yl acetate (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf" class="ext-link">Supporting Information</a>) using NIS in the mixture of CH<sub>3</sub>CN and H<sub>2</sub>SO<sub>4</sub>. Compounds <b>27</b> and <b>28</b> were also synthesized using CuI-catalyzed cross-coupling between <b>85</b> and cyclohexylamine or cyclopentylamine (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).</div><figure id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0012.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>5</b>, <b>27</b>, and <b>28</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Four amino acids were selected for the preparation of prodrugs. <i>N</i>,<i>N</i>-Dimethylglycine or the <i>N</i>-Boc-<span class="smallcaps smallerCapital">l</span>-amino acid were attached to <b>56</b> through an ester bond under DCC and DMAP conditions. Compounds <b>61</b>–<b>64</b> were afforded by removing the Boc group in <b>86</b> and <b>87</b> using TFA. The other salts <b>66</b>–<b>70</b> were prepared by mixing <b>65</b> with the corresponding acid in the proper solvent (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>).</div><figure id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0013.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of the Prodrugs and Preparation of the Salt Form<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amino acid (3 equiv), DCC (3 equiv), DMAP (1.2 equiv), DCM, 40 °C, ∼90%; (b) for <b>61</b>–<b>64</b>, TFA/DCM (1/1), room temperature; for <b>66</b>–<b>70</b>, the corresponding acid, DCM or ethyl acetate.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31042" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31042" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">HBV genome is a 3.2 kbp relaxed circular DNA, expressing only seven proteins. To compensate for the limited number of viral proteins, each HBV protein plays multiple essential roles in the HBV life cycle.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The core protein could self-assemble to form the shell of HBV and enable the synthesis of the viral genome <i>via</i> reverse transcription in the capsid. In addition, Cps could bind to CpG island 2 to positively regulate HBV transcription by maintaining the hypomethylation of CpG island 2.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Capsids containing a mature HBV genome are able to release the viral genome into the nucleus<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> in the initial infection and amplification of the cccDNA pool, which is vital to the establishment and persistence of chronic infection. Thus, targeting the multifunctional core protein and capsid represents a promising strategy for HBV therapy.</div><div class="NLM_p">Sustained and prolonged suppression of HBV is required to prevent the progression of the disease. However, current therapeutics only inhibit virus replication and seldom achieve complete elimination of HBV. Lifelong treatment is usually required to suppress HBV replication. cccDNA, which is the template of transcription, is the major cause of HBV persistence. New therapies, especially those that could clear cccDNA, are needed to shorten the treatment period and improve the outcome. Various CAM chemotypes have been reported to inhibit the formation of cccDNA.<a onclick="showRef(event, 'ref20 ref21 ref23'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref23">(20,21,23)</a> However, the inhibition of cccDNA was achieved only when CAM treatment began before or together with the infection, meaning that the already established cccDNA in cells is not influenced by CAMs. In addition, a much higher concentration of compound is needed to inhibit cccDNA formation than to block HBV replication. CAMs have been reported to directly target the mature capsid.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> This interaction between CAMs and capsids may disturb the morphology and disassembly of the capsid, which is important for the importing of the genome into the nucleus.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> To evaluate the effect on cccDNA, the establishment of an HBV infection model is required, which is ongoing in our lab.</div><div class="NLM_p">As proven in both preclinical and clinical studies, CAMs, which could interfere with the HBV life cycle in multiple steps, have great potential to be developed into new anti-HBV therapies. However, no CAMs have currently been approved. Through constraining the conformation of SBAs, we designed indazole as a new chemotype of CAMs. The indazole analogue <b>6</b> accelerates the formation of morphologically normal (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) but empty capsids as SBAs do. In the subsequent SAR study, we discovered (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) analogue <b>56</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) through scaffold hopping and substituents modification (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The lipophilicity of (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide analogues is lower than that of indazoles, due to the replacement of the phenyl with a pyridinyl. Matched molecular pair comparison (<b>46</b>, <b>57</b>, and <b>58</b> vs <b>48</b>, <b>59</b>, and <b>60</b>, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) revealed that the cytotoxicity of (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamides is much lower than that of indazoles. Compound <b>56</b> potently inhibited HBV replication in HepAD38 cells with an EC<sub>50</sub> of 0.034 μM, approximately 10-fold lower than that of SBAs. However, the poor solubility of <b>56</b> diminished its <i>in vivo</i> anti-HBV activity (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Compound <b>67</b> is the citric acid salt of the prodrug of <b>56</b>. It has good aqueous solubility and is hydrolyzed quickly to release <b>56</b> in plasma. The <i>in vivo</i> exposure of <b>56</b> after oral administration of <b>67</b> was much higher compared to the oral administration of <b>56</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Compound <b>67</b> dose dependently inhibited <i>in vivo</i> HBV replication in an HDI HBV mouse model. These results highlight (1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonamide as a new chemotype of CAMs and provide a starting point to develop new therapeutics for HBV infection.</div><figure id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0008.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Summary of the structure–activity relationship (SAR).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Chemistry Method</h3><div class="NLM_p last">All the reactions were stirred magnetically unless otherwise noted. Reagents and solvents were purchased from commercial suppliers and used directly unless otherwise specified. Reactions were monitored using the Waters UPLC–MS system equipped with an autosampler, a PDA detector, and a SQ mass detector. The eluent was the mixture of acetonitrile and water containing 0.05% (v/v) HCOOH. TLC was performed using thin layer chromatography silica gel plate (HSGF254) from Yantai Institute of chemical industry and was visualized with UV light. The column chromatography was performed with silica gel 60 (200–300 mesh) from Qingdao Haiyang Chemical Factory. <sup>1</sup>H and <sup>13</sup>C NMR spectra were collected using a Bruker Advance 400 MHz spectrometer. Chemical shifts are given in parts per million (ppm), and coupling constants (J values) are reported in hertz (Hz). Multiplicities are described using the following abbreviations: s (singlet), d (doublet), dd (double doublet), dt (double triplet), t (triplet), q (quartet), br s (broad singlet), and m (multiplet). Accurate molecular weight and chemical formula were determined using HRMS, which was conducted by Agilent LC/MSD TOF. The ion source was electrospray ionization (ESI). All compounds submitted for biological test were >95% purity detected by an Agilent HPLC system equipped with an autosampler and a PDA detector. The eluent was the mixture of acetonitrile and water containing 0.05% (v/v) HCOOH.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Synthesis of the Target Compound 1-((3-((3-Chloro-4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>1</b>)</h3><div class="NLM_p last">The mixture of <b>75</b> (0.27 mmol), 3-chloro-5-fluoroaniline (0.4 mmol), Pd(OAc)<sub>2</sub> (0.014 mmol), xantphos (0.027 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (0.54 mmol) in 1.5 mL of 1,4-dioxane was stirred at 110 °C under an argon atmosphere for 3 h. After that, the solution was diluted with DCM and filtered. The filtrate was concentrated under a vacuum and dissolved in THF (2 mL) and H<sub>2</sub>O (2 mL). To the mixture was added LiOH (0.84 mmol). The mixture was stirred at room temperature for 1 h and extracted with ethyl acetate (15 mL). The organic layer was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified with silico column chromatography (petro ether/ethyl acetate, v/v, 1/1) to afford 122 mg of <b>76</b>, 89% yield. The mixture of <b>76</b> in ethanol (5 mL) and hydrochloric acid (1 mL) was stirred at room temperature for 30 min, neutralized with 2N NaOH to pH 8–9, and filtered. The solid was washed with methanol to afford compound <b>1</b> with quantitative yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.63 (s, 1H), 9.53 (s, 1H), 8.56 (s, 1H), 8.11 (s, 1H), 7.92–7.45 (m, 3H), 7.35 (t, <i>J</i> = 9.3 Hz, 1H), 4.65 (s, 1H), 3.49 (s, 1H), 3.16 (s, 2H), 2.71 (s, 2H), 1.74 (s, 2H), 1.44 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.2 (d, <i>J</i> = 237.9 Hz), 146.2, 141.7, 140.0, 126.0, 125.6, 122.4, 119.5 (d, <i>J</i> = 18.1 Hz), 117.4 (d, <i>J</i> = 21.6 Hz), 117.0, 116.5 (d, <i>J</i> = 6.1 Hz), 113.8, 110.8, 64.2, 43.7, 33.3. HRMS (ESI) calculated for C<sub>18</sub>H<sub>19</sub>ClFN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 425.0845, found 425.0831.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Procedure for Synthesis of the Title Compounds</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Method I (for Compounds <b>2</b>–<b>26</b>, <b>29</b>–<b>32</b>, <b>34</b>–<b>37</b>, <b>44</b>, <b>47</b>, <b>49</b>, <b>56</b>, <b>57</b>, and <b>59</b>)</h4><div class="NLM_p last">The mixture of the 3-bromoindazole derivatives (0.1 mmol), the corresponding aniline (0.2 mmol), Pd(OAc)<sub>2</sub> (0.01 mmol), xantphos (0.01 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (0.2 mmol) in 1.5 mL of 1,4-dioxane was stirred at 130 °C under an argon atmosphere for 3 h. After that, the solution was diluted with DCM and filtered. The filtrate was concentrated under a vacuum and dissolved in THF (2 mL) and H<sub>2</sub>O (2 mL). To the mixture was added LiOH (0.5 mmol). The mixture was stirred at room temperature for 1 h and extracted with ethyl acetate (15 mL). The organic layer was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified with column chromatography on silico gel.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Method II (for Compounds <b>33</b>, <b>38</b>, <b>40</b>–<b>42</b>, <b>43</b>, <b>45</b>, <b>46</b>, <b>48</b>, <b>50</b>–<b>55</b>, <b>58</b>, and <b>60</b>)</h4><div class="NLM_p last">The mixture of the 3-bromoindazole derivatives (0.1 mmol), the corresponding aniline (0.2 mmol), Pd(OAc)<sub>2</sub> (0.01 mmol), xantphos (0.01 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (0.2 mmol) in 1.5 mL of 1,4-dioxane was stirred at 130 °C under an argon atmosphere for 3 h. The insoluble matters were removed by filtration, and the filtrate was concentrated under a vacuum. The residue was purified with column chromatography on silico gel.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1-((3-((3-Chloro-4-fluorophenyl)amino)-1-methyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>2</b>)</h4><div class="NLM_p last">25% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.56 (s, 1H), 8.56 (d, <i>J</i> = 1.5 Hz, 1H), 8.04 (dd, <i>J</i> = 6.6, 2.7 Hz, 1H), 7.76–7.65 (m, 2H), 7.60 (ddd, <i>J</i> = 9.1, 4.1, 2.7 Hz, 1H), 7.37 (t, <i>J</i> = 9.1 Hz, 1H), 4.65 (s, 1H), 3.99 (s, 3H), 3.51 (dt, <i>J</i> = 7.9, 4.0 Hz, 1H), 3.23–3.10 (m, 2H), 2.72 (ddd, <i>J</i> = 11.8, 8.5, 3.4 Hz, 2H), 1.76 (ddd, <i>J</i> = 13.1, 6.9, 3.5 Hz, 2H), 1.44 (dtd, <i>J</i> = 12.2, 8.0, 3.5 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.9 (d, <i>J</i> = 238.4 Hz), 144.9, 140.8, 139.3 (d, <i>J</i> = 2.3 Hz), 125.5, 125.1, 122.1, 119.1 (d, <i>J</i> = 18.1 Hz), 117.0 (d, <i>J</i> = 21.7 Hz), 116.7, 116.1 (d, <i>J</i> = 6.5 Hz), 113.7, 109.7, 63.8, 43.3, 35.4, 32.9. HRMS (ESI) calculated for C<sub>19</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>ClFS [M + H]<sup>+</sup> 439.0986, found 439.1001.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1-((3-((3-Chloro-4-fluorophenyl)amino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>3</b>)</h4><div class="NLM_p last">39% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.57 (d, <i>J</i> = 1.9 Hz, 1H), 8.55 (s, 1H), 8.01 (dd, <i>J</i> = 5.8, 3.0 Hz, 1H), 7.72 (s, 2H), 7.62 (ddt, <i>J</i> = 9.0, 4.5, 2.3 Hz, 1H), 7.37 (td, <i>J</i> = 9.1, 1.9 Hz, 1H), 4.66 (t, <i>J</i> = 2.7 Hz, 1H), 3.76–3.61 (m, 1H), 3.52 (dt, <i>J</i> = 9.1, 4.5 Hz, 1H), 3.23–3.07 (m, 2H), 2.73 (t, <i>J</i> = 9.8 Hz, 2H), 1.83–1.68 (m, 2H), 1.53–1.37 (m, 3H), 1.18–1.09 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.4 (d, <i>J</i> = 238.5 Hz), 145.1, 142.0, 139.7 (d, <i>J</i> = 2.5 Hz), 126.3, 126.3, 122.6, 119.6 (d, <i>J</i> = 18.2 Hz), 117.5, 117.3, 116.6 (d, <i>J</i> = 6.5 Hz), 115.0, 110.4, 64.2, 43.7, 33.4, 29.6, 6.8. HRMS (ESI) calculated for C<sub>21</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>ClFS [M + H]<sup>+</sup> 465.1143, found 465.1158.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 1-((3-((3-Chloro-4-fluorophenyl)(methyl)amino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl) sulfonyl) piperidin-4-ol (<b>4</b>)</h4><div class="NLM_p last">10% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.81 (d, <i>J</i> = 8.9 Hz, 1H), 7.66 (dd, <i>J</i> = 8.9, 1.6 Hz, 1H), 7.38 (t, <i>J</i> = 9.0 Hz, 1H), 7.31 (dd, <i>J</i> = 6.6, 2.8 Hz, 1H), 7.23–7.10 (m, 2H), 4.65 (d, <i>J</i> = 3.8 Hz, 1H), 3.72 (p, <i>J</i> = 6.0 Hz, 1H), 3.48 (dd, <i>J</i> = 10.0, 6.0 Hz, 1H), 3.44 (s, 3H), 3.01 (dd, <i>J</i> = 11.2, 5.3 Hz, 2H), 2.63–2.53 (m, 2H), 1.79–1.66 (m, 2H), 1.42 (dtd, <i>J</i> = 11.9, 8.0, 3.5 Hz, 2H), 1.16 (d, <i>J</i> = 3.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.4 (d, <i>J</i> = 238.5 Hz), 145.1, 142.0, 139.7 (d, <i>J</i> = 2.5 Hz), 126.3, 126.3, 122.6, 119.6 (d, <i>J</i> = 18.2 Hz), 117.5, 117.3, 116.6 (d, <i>J</i> = 6.5 Hz), 115.0, 110.4, 64.2, 43.7, 33.4, 29.6, 6.8. HRMS (ESI) calculated for C<sub>22</sub>H<sub>25</sub>O<sub>3</sub>N<sub>4</sub>ClFS [M + H]<sup>+</sup> 479.1298, found 479.1314.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1-((3-(3-Chloro-4-fluorophenoxy)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>5</b>)</h4><div class="NLM_p last">To the solution of <b>85</b> (28 mg, 0.057 mmol) in 1,4-dioxane was added CuI (1.0 mg, 0.0057 mmol), <i>N</i>,<i>N</i>-dimethylglycine (1.5 mg, 0.014 mmol), and Cs<sub>2</sub>CO<sub>3</sub> (46 mg, 0.14 mmol). The mixture was stirred at 120 °C under an argon atmosphere for 20 h. The insoluble matter was removed through filtration, and filtrate was concentrated under a vacuum. The residue was dissolved in THF (2 mL) and H<sub>2</sub>O (2 mL). To the mixture was added LiOH (0.5 mmol). The mixture was stirred at room temperature for 1 h and extracted with ethyl acetate (15 mL). The organic layer was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified with column chromatography on C18 silico gel to afford compound <b>5</b>. Yield: 40%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 8.9 Hz, 1H), 7.83 (d, <i>J</i> = 1.5 Hz, 1H), 7.77 (dd, <i>J</i> = 8.9, 1.7 Hz, 1H), 7.64 (dd, <i>J</i> = 6.2, 3.0 Hz, 1H), 7.49 (t, <i>J</i> = 9.0 Hz, 1H), 7.36 (dt, <i>J</i> = 9.1, 3.5 Hz, 1H), 4.65 (s, 1H), 3.72 (tt, <i>J</i> = 6.8, 4.1 Hz, 1H), 3.49 (dq, <i>J</i> = 7.7, 4.0 Hz, 1H), 3.11 (ddd, <i>J</i> = 11.2, 7.1, 3.7 Hz, 2H), 2.69 (ddd, <i>J</i> = 11.7, 8.3, 3.4 Hz, 2H), 1.73 (ddt, <i>J</i> = 13.9, 7.2, 3.5 Hz, 2H), 1.42 (dtd, <i>J</i> = 12.2, 8.1, 3.7 Hz, 2H), 1.11 (dq, <i>J</i> = 7.7, 2.5 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.5 (d, <i>J</i> = 243.2 Hz), 153.0, 152.1 (d, <i>J</i> = 2.9 Hz), 143.2, 128.4, 126.3, 121.2, 120.9, 120.5 (d, <i>J</i> = 19.6 Hz), 119.7 (d, <i>J</i> = 7.3 Hz), 118.1 (d, <i>J</i> = 23.1 Hz), 112.4, 111.8, 64.1, 43.6, 33.3, 29.9, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>21</sub>O<sub>4</sub>N<sub>3</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 468.1184, found 468.1199.</div></div><div id="sec5_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 1-((1-Cyclopropyl-3-((3,4,5-trifluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>6</b>)</h4><div class="NLM_p last">52% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 8.52 (s, 1H), 7.74 (d, <i>J</i> = 1.8 Hz, 2H), 7.57 (dt, <i>J</i> = 10.9, 5.0 Hz, 2H), 4.66 (d, <i>J</i> = 3.9 Hz, 1H), 3.69 (ddd, <i>J</i> = 10.6, 6.9, 4.3 Hz, 1H), 3.51 (tq, <i>J</i> = 7.5, 3.7 Hz, 1H), 3.29–3.09 (m, 2H), 2.73 (ddd, <i>J</i> = 11.6, 8.4, 3.4 Hz, 2H), 1.76 (ddt, <i>J</i> = 13.9, 7.2, 3.6 Hz, 2H), 1.45 (dtd, <i>J</i> = 12.0, 8.0, 3.6 Hz, 2H), 1.24–1.09 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 150.8 (ddd, <i>J</i> = 242.6, 9.9, 6.1 Hz), 144.7, 141.9, 139.3–138.3 (m), 132.7 (dt, <i>J</i> = 239.8, 16.1 Hz), 126.5, 126.3, 122.4, 114.8, 110.5, 100.4 (d, <i>J</i> = 24.7 Hz), 64.2, 43.6, 33.3, 29.6, 6.8. HRMS (ESI) calculated for C<sub>21</sub>H<sub>22</sub>O<sub>3</sub>N<sub>4</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 467.1339, found 467.1359.</div></div><div id="sec5_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 1-((1-Cyclopropyl-7-fluoro-3-((3,4,5-trifluorophenyl)amino)-1H-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>7</b>)</h4><div class="NLM_p last">31% yield, <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.75 (s, 1H), 8.34 (s, 1H), 7.58–7.44 (m, 3H), 4.68 (d, <i>J</i> = 3.9 Hz, 1H), 3.83 (tt, <i>J</i> = 7.2, 3.7 Hz, 1H), 3.51 (tq, <i>J</i> = 7.6, 3.6 Hz, 1H), 3.19 (ddd, <i>J</i> = 10.6, 6.2, 3.4 Hz, 2H), 2.75 (ddd, <i>J</i> = 11.8, 8.3, 3.4 Hz, 2H), 1.76 (ddt, <i>J</i> = 13.8, 7.3, 3.6 Hz, 2H), 1.45 (dtd, <i>J</i> = 12.3, 8.3, 3.7 Hz, 2H), 1.25–1.19 (m, 2H), 1.15–1.09 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.8 (ddd, <i>J</i> = 242.8, 10.0, 6.0 Hz), 147.5 (d, <i>J</i> = 252.4 Hz), 144.9 (d, <i>J</i> = 2.2 Hz), 138.6–138.2 (m), 134.4–131.5 (m), 131.4 (d, <i>J</i> = 13.6 Hz), 127.0 (d, <i>J</i> = 4.4 Hz), 118.6 (d, <i>J</i> = 6.0 Hz), 118.4 (d, <i>J</i> = 3.7 Hz), 111.4 (d, <i>J</i> = 20.7 Hz), 100.5 (d, <i>J</i> = 24.9 Hz), 64.2, 43.8, 33.4, 32.0 (d, <i>J</i> = 2.6 Hz), 7.8 (d, <i>J</i> = 3.0 Hz). HRMS (ESI) calculated for C<sub>21</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 485.1265, found 485.1244.</div></div><div id="sec5_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 1-((1-Cyclopropyl-6-fluoro-3-((3,4,5-trifluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>8</b>)</h4><div class="NLM_p last">21% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.81 (s, 1H), 8.59 (d, <i>J</i> = 6.6 Hz, 1H), 7.60 (d, <i>J</i> = 11.2 Hz, 1H), 7.53 (dd, <i>J</i> = 11.0, 6.2 Hz, 2H), 4.71 (d, <i>J</i> = 4.0 Hz, 1H), 3.66 (p, <i>J</i> = 5.3 Hz, 1H), 3.62–3.51 (m, 1H), 2.89 (ddd, <i>J</i> = 12.0, 8.3, 3.2 Hz, 2H), 1.76 (ddt, <i>J</i> = 13.4, 7.1, 3.5 Hz, 2H), 1.43 (dtd, <i>J</i> = 12.5, 8.3, 3.7 Hz, 2H), 1.13 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.3 (d, <i>J</i> = 251.3 Hz), 150.8 (ddd, <i>J</i> = 243.0, 10.1, 6.4 Hz), 144.9, 142.4 (d, <i>J</i> = 12.5 Hz), 138.5 (t, <i>J</i> = 12.6 Hz), 132.8 (d, <i>J</i> = 240.9 Hz), 125.6 (d, <i>J</i> = 3.1 Hz), 117.5 (d, <i>J</i> = 19.1 Hz), 111.5, 100.5 (d, <i>J</i> = 24.4 Hz), 97.5 (d, <i>J</i> = 27.7 Hz), 64.5, 43.3, 33.7, 29.8, 6.8. HRMS (ESI) calculated for C<sub>21</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 485.1265, found 485.1243.</div></div><div id="sec5_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 1-((1-Cyclopropyl-4-fluoro-3-((3,4,5-trifluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>9</b>)</h4><div class="NLM_p last">12% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.06 (s, 1H), 7.68 (dd, <i>J</i> = 8.9, 6.2 Hz, 1H), 7.59 (dd, <i>J</i> = 11.2, 6.3 Hz, 2H), 7.55 (d, <i>J</i> = 8.9 Hz, 1H), 4.71 (d, <i>J</i> = 4.0 Hz, 1H), 3.73 (tt, <i>J</i> = 7.0, 4.0 Hz, 1H), 3.56 (dq, <i>J</i> = 8.1, 4.1 Hz, 1H), 3.28 (dt, <i>J</i> = 11.3, 4.7 Hz, 2H), 2.88 (ddd, <i>J</i> = 11.9, 8.8, 3.2 Hz, 2H), 1.82–1.71 (m, 2H), 1.45 (dtd, <i>J</i> = 12.3, 8.1, 3.7 Hz, 2H), 1.20–1.11 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.3 (d, <i>J</i> = 263.6 Hz), 152.0–149.0 (m), 145.4 (d, <i>J</i> = 8.7 Hz), 142.5 (d, <i>J</i> = 2.2 Hz), 139.0, 129.3, 113.4 (d, <i>J</i> = 11.3 Hz), 106.8 (d, <i>J</i> = 3.9 Hz), 105.1 (d, <i>J</i> = 20.1 Hz), 101.1 (d, <i>J</i> = 24.9 Hz), 64.3, 43.5, 33.5, 29.9, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 485.1265, found 485.1243.</div></div><div id="sec5_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 1-((1-Cyclopropyl-3-((3,4,5-trifluorophenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>b</i>]pyridin-5-yl)sulfonyl)piperidin-4-ol (<b>10</b>)</h4><div class="NLM_p last">27% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.91 (s, 1H), 8.82 (d, <i>J</i> = 2.1 Hz, 1H), 8.80 (d, <i>J</i> = 2.1 Hz, 1H), 7.49 (dd, <i>J</i> = 10.8, 6.2 Hz, 2H), 4.68 (d, <i>J</i> = 4.0 Hz, 1H), 3.82 (tt, <i>J</i> = 7.3, 3.7 Hz, 1H), 3.51 (tq, <i>J</i> = 7.5, 3.6 Hz, 1H), 3.20 (ddd, <i>J</i> = 13.1, 6.1, 3.0 Hz, 2H), 2.76 (ddd, <i>J</i> = 11.8, 8.6, 3.3 Hz, 2H), 1.77 (ddt, <i>J</i> = 13.5, 6.6, 3.5 Hz, 2H), 1.45 (dtd, <i>J</i> = 12.3, 8.0, 3.5 Hz, 2H), 1.25–1.18 (m, 2H), 1.18–1.10 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.1, 150.8 (ddd, <i>J</i> = 243.0, 10.0, 5.8 Hz), 148.6, 143.6, 138.2 (td, <i>J</i> = 12.3, 2.8 Hz), 133.0 (dt, <i>J</i> = 240.5, 15.9 Hz), 131.5, 123.8, 107.3, 100.7 (d, <i>J</i> = 24.3 Hz), 64.2, 43.7, 33.4, 29.2, 6.6. HRMS (ESI) calculated for C<sub>20</sub>H<sub>21</sub>O<sub>3</sub>N<sub>5</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 468.1312, found 468.1291.</div></div><div id="sec5_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 1-((1-Cyclopropyl-3-((3,4,5-trifluorophenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-ol (<b>11</b>)</h4><div class="NLM_p last">47% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.86 (s, 1H), 9.15 (s, 1H), 8.62 (s, 1H), 7.48 (dd, <i>J</i> = 10.9, 6.1 Hz, 2H), 4.69 (d, <i>J</i> = 3.9 Hz, 1H), 3.84 (dq, <i>J</i> = 6.9, 3.5, 3.0 Hz, 1H), 3.63–3.48 (m, 2H), 3.41 (dt, <i>J</i> = 11.4, 4.5 Hz, 2H), 2.94 (ddd, <i>J</i> = 12.2, 8.8, 3.4 Hz, 2H), 1.83–1.67 (m, 2H), 1.52–1.33 (m, 2H), 1.23–1.16 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.8 (ddd, <i>J</i> = 242.6, 10.0, 5.8 Hz), 144.6, 143.9, 138.9–137.9 (m), 137.5, 134.8, 132.9 (d, <i>J</i> = 240.7 Hz), 118.8, 116.4, 100.5 (d, <i>J</i> = 24.7 Hz), 64.8, 44.2, 33.8, 30.3, 7.0. HRMS (ESI) calculated for C<sub>20</sub>H<sub>21</sub>O<sub>3</sub>N<sub>5</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 468.1312, found 468.1289.</div></div><div id="sec5_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 1-((1-Cyclopropyl-3-((3,4,5-trifluorophenyl)amino)-1<i>H</i>-pyrazolo[4,3-<i>b</i>]pyridin-5-yl)sulfonyl)piperidin-4-ol (<b>12</b>)</h4><div class="NLM_p last">21% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.67 (s, 1H), 8.25 (d, <i>J</i> = 8.8 Hz, 1H), 7.94 (d, <i>J</i> = 8.8 Hz, 1H), 7.72 (dd, <i>J</i> = 11.2, 6.2 Hz, 2H), 4.69 (d, <i>J</i> = 4.0 Hz, 1H), 3.81–3.67 (m, 1H), 3.59–3.46 (m, 1H), 3.41 (dd, <i>J</i> = 12.4, 5.6 Hz, 2H), 2.95 (ddd, <i>J</i> = 12.2, 8.8, 3.3 Hz, 2H), 1.73 (ddt, <i>J</i> = 13.6, 7.0, 3.6 Hz, 2H), 1.40 (dtd, <i>J</i> = 12.6, 8.6, 3.8 Hz, 2H), 1.15 (d, <i>J</i> = 5.2 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.7–148.8 (m), 148.3, 143.6, 138.2 (t, <i>J</i> = 12.5 Hz), 134.0, 133.9–131.1 (m), 131.9, 120.8, 119.2, 100.7 (d, <i>J</i> = 24.7 Hz), 64.2, 43.6, 33.3, 29.6, 6.2. HRMS (ESI) calculated for C<sub>20</sub>H<sub>21</sub>O<sub>3</sub>N<sub>5</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 468.1312, found 468.1291.</div></div><div id="sec5_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 1-((1-Cyclopropyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>13</b>)</h4><div class="NLM_p last">11% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.39 (s, 1H), 8.58 (d, <i>J</i> = 1.3 Hz, 1H), 7.79–7.72 (m, 2H), 7.70 (d, <i>J</i> = 1.2 Hz, 2H), 7.21–7.13 (m, 2H), 4.69 (s, 1H), 3.65 (p, <i>J</i> = 5.3 Hz, 1H), 3.51 (tt, <i>J</i> = 7.6, 3.6 Hz, 1H), 3.18–3.11 (m, 2H), 2.72 (ddd, <i>J</i> = 11.8, 8.4, 3.4 Hz, 2H), 1.76 (ddt, <i>J</i> = 13.5, 6.8, 2.9 Hz, 2H), 1.45 (dtd, <i>J</i> = 12.0, 7.8, 3.6 Hz, 2H), 1.13 (d, <i>J</i> = 5.2 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.7 Hz), 145.7, 142.1, 139.0 (d, <i>J</i> = 2.1 Hz), 126.1 (d, <i>J</i> = 26.0 Hz), 122.8, 117.8 (d, <i>J</i> = 7.4 Hz), 115.9, 115.6, 115.1, 110.2, 64.2, 43.7, 33.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 431.1548, found 431.1528.</div></div><div id="sec5_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-((1-Cyclopropyl-3-(p-tolylamino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>14</b>)</h4><div class="NLM_p last">49% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.24 (s, 1H), 8.60 (t, <i>J</i> = 1.3 Hz, 1H), 7.72–7.67 (m, 2H), 7.67–7.61 (m, 2H), 7.12 (d, <i>J</i> = 8.3 Hz, 2H), 4.68 (d, <i>J</i> = 3.3 Hz, 1H), 3.64 (p, <i>J</i> = 5.3 Hz, 1H), 3.52 (d, <i>J</i> = 8.6 Hz, 1H), 3.16 (ddd, <i>J</i> = 11.1, 6.9, 3.7 Hz, 2H), 2.72 (ddd, <i>J</i> = 11.8, 8.4, 3.4 Hz, 2H), 2.26 (s, 3H), 1.76 (ddt, <i>J</i> = 13.5, 6.8, 2.9 Hz, 2H), 1.46 (dtd, <i>J</i> = 12.0, 7.9, 3.5 Hz, 2H), 1.13 (d, <i>J</i> = 5.2 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 145.8, 142.1, 140.1, 129.7, 128.7, 126.1, 125.8, 122.9, 116.6, 115.3, 110.08, 64.3, 43.7, 33.4, 29.6, 20.8, 6.9. HRMS (ESI) calculated for C<sub>23</sub>H<sub>27</sub>O<sub>3</sub>N<sub>4</sub>S [M + H]<sup>+</sup> 427.1798, found 427.1779.</div></div><div id="sec5_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 1-((3-((4-Chlorophenyl)amino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>15</b>)</h4><div class="NLM_p last">36% yield. <sup>1</sup>H NMR (400 MHz, DMSO): δ 9.51 (s, 1H), 8.58 (s, 1H), 7.74 (dd, <i>J</i> = 14.7, 10.6 Hz, 4H), 7.45–7.30 (m, 2H), 4.66 (d, <i>J</i> = 3.6 Hz, 1H), 3.72–3.59 (m, 1H), 3.51 (s, 1H), 3.15 (s, 2H), 2.71 (d, <i>J</i> = 8.1 Hz, 2H), 1.75 (s, 2H), 1.53–1.38 (m, 2H), 1.13 (d, <i>J</i> = 4.6 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 145.2, 142.0, 141.3, 129.1, 126.2, 126.0, 123.4, 122.7, 118.0, 115.2, 110.3, 64.2, 43.7, 33.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>ClS [M + H]<sup>+</sup> 447.1252, found 447.1232.</div></div><div id="sec5_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 1-((1-Cyclopropyl-3-(phenylamino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>16</b>)</h4><div class="NLM_p last">31% yield. <sup>1</sup>H NMR (400 MHz, DMSO): δ 9.34 (s, 1H), 8.61 (s, 1H), 7.83–7.64 (m, 4H), 7.32 (t, <i>J</i> = 7.9 Hz, 2H), 6.87 (t, <i>J</i> = 7.3 Hz, 1H), 4.66 (d, <i>J</i> = 3.8 Hz, 1H), 3.66 (dt, <i>J</i> = 10.4, 5.3 Hz, 1H), 3.59–3.45 (m, 1H), 3.16 (s, 2H), 2.72 (t, <i>J</i> = 8.3 Hz, 2H), 1.76 (dd, <i>J</i> = 12.3, 3.4 Hz, 2H), 1.46 (dd, <i>J</i> = 8.2, 3.8 Hz, 2H), 1.14 (d, <i>J</i> = 5.2 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 145.6, 142.5, 142.0, 129.3, 126.1, 125.8, 122.8, 120.1, 116.5, 115.3, 110.2, 64.2, 43.7, 33.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>25</sub>O<sub>3</sub>N<sub>4</sub>S [M + H]<sup>+</sup> 413.1642, found 413.1623.</div></div><div id="sec5_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 1-((1-Cyclopropyl-3-((4-fluoro-2-methylphenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>17</b>)</h4><div class="NLM_p last">20% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.41 (s, 1H), 8.16 (s, 1H), 7.81–7.61 (m, 3H), 7.07 (dd, <i>J</i> = 9.7, 2.9 Hz, 1H), 7.02–6.93 (m, 1H), 4.65 (d, <i>J</i> = 3.8 Hz, 1H), 3.65–3.54 (m, 1H), 3.54–3.45 (m, 1H), 3.14 (td, <i>J</i> = 11.1, 10.7, 5.0 Hz, 2H), 2.78–2.64 (m, 2H), 2.32 (s, 3H), 1.85–1.66 (m, 2H), 1.54–1.36 (m, 2H), 1.14–0.97 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.8 (d, <i>J</i> = 237.9 Hz), 146.7, 142.6, 137.0 (d, <i>J</i> = 2.5 Hz), 131.6 (d, <i>J</i> = 7.7 Hz), 125.9, 123.1, 122.0 (d, <i>J</i> = 8.3 Hz), 117.2 (d, <i>J</i> = 22.0 Hz), 115.1, 112.9 (d, <i>J</i> = 21.6 Hz), 110.3, 64.2, 43.6, 33.4, 29.4, 18.7, 6.8. HRMS (ESI) calculated for C<sub>22</sub>H<sub>26</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 445.1704, found 445.1684.</div></div><div id="sec5_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 1-((3-((2-Chloro-4-fluorophenyl)amino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>18</b>)</h4><div class="NLM_p last">43% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.57 (s, 1H), 8.54 (d, <i>J</i> = 1.5 Hz, 1H), 8.03 (dd, <i>J</i> = 9.2, 5.6 Hz, 1H), 7.80–7.64 (m, 2H), 7.47 (dd, <i>J</i> = 8.5, 3.0 Hz, 1H), 7.22 (td, <i>J</i> = 8.6, 3.0 Hz, 1H), 4.66 (d, <i>J</i> = 3.8 Hz, 1H), 3.71–3.60 (m, 1H), 3.53 (tq, <i>J</i> = 7.6, 3.7 Hz, 1H), 3.16 (ddt, <i>J</i> = 11.2, 7.3, 4.5 Hz, 2H), 2.74 (ddd, <i>J</i> = 11.8, 8.5, 3.4 Hz, 2H), 1.86–1.68 (m, 2H), 1.46 (dtd, <i>J</i> = 12.1, 8.0, 3.6 Hz, 2H), 1.15–1.05 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.7 (d, <i>J</i> = 240.9 Hz), 145.4, 142.5, 136.0 (d, <i>J</i> = 3.0 Hz), 126.4, 126.0, 123.4 (d, <i>J</i> = 10.5 Hz), 123.3, 121.8 (d, <i>J</i> = 8.2 Hz), 117.0 (d, <i>J</i> = 25.7 Hz), 115.3, 115.0 (d, <i>J</i> = 21.8 Hz), 110.4, 64.2, 43.7, 33.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>ClFS [M + H]<sup>+</sup> 465.1158, found 465.1139.</div></div><div id="sec5_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 1-((1-Cyclopropyl-3-((3,4-difluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>19</b>)</h4><div class="NLM_p last">11% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.63 (s, 1H), 8.56 (d, <i>J</i> = 1.2 Hz, 1H), 7.96–7.82 (m, 1H), 7.71 (s, 2H), 7.48–7.31 (m, 2H), 4.66 (s, 1H), 3.76–3.60 (m, 1H), 3.50 (s, 1H), 3.14 (dt, <i>J</i> = 11.2, 4.0 Hz, 2H), 2.72 (ddd, <i>J</i> = 11.7, 8.4, 3.4 Hz, 2H), 1.82–1.67 (m, 2H), 1.44 (dtd, <i>J</i> = 12.1, 8.1, 3.7 Hz, 2H), 1.18–1.05 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.4–146.8 (m), 145.2, 143.4 (dd, <i>J</i> = 237.0, 12.4 Hz), 142.0, 139.6 (dd, <i>J</i> = 9.7, 2.2 Hz), 126.3, 126.2, 122.6, 118.0 (d, <i>J</i> = 17.7 Hz), 115.0, 112.6 (dd, <i>J</i> = 5.3, 2.8 Hz), 110.4, 105.1 (d, <i>J</i> = 22.3 Hz), 64.2, 43.7, 33.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>F<sub>2</sub>S [M + H]<sup>+</sup> 449.1453, found 449.1432.</div></div><div id="sec5_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>20</b>)</h4><div class="NLM_p last">9% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.27 (s, 1H), 8.56 (s, 1H), 7.68 (s, 2H), 7.63 (dt, <i>J</i> = 8.2, 3.6 Hz, 1H), 7.53 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.09 (t, <i>J</i> = 9.2 Hz, 1H), 4.66 (s, 1H), 3.64 (p, <i>J</i> = 5.3 Hz, 1H), 3.50 (tt, <i>J</i> = 7.9, 3.6 Hz, 1H), 3.22–3.07 (m, 2H), 2.71 (ddd, <i>J</i> = 11.9, 8.6, 3.5 Hz, 2H), 2.24 (s, 3H), 1.74 (ddd, <i>J</i> = 13.9, 7.3, 3.6 Hz, 2H), 1.44 (dtd, <i>J</i> = 12.3, 8.2, 3.6 Hz, 2H), 1.12 (d, <i>J</i> = 5.2 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.2 (d, <i>J</i> = 234.8 Hz), 145.7, 142.1, 138.7 (d, <i>J</i> = 2.2 Hz), 126.0 (d, <i>J</i> = 29.0 Hz), 124.5 (d, <i>J</i> = 18.0 Hz), 122.8, 119.2 (d, <i>J</i> = 4.0 Hz), 115.5, 115.2 (d, <i>J</i> = 4.1 Hz), 115.2, 115.2, 110.2, 64.2, 43.7, 33.4, 29.6, 15.1 (d, <i>J</i> = 2.9 Hz), 6.9. HRMS (ESI) calculated for C<sub>22</sub>H<sub>26</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 445.1704, found 445.1686.</div></div><div id="sec5_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 5-((1-Cyclopropyl-5-((4-hydroxypiperidin-1-yl)sulfonyl)-1<i>H</i>-indazol-3-yl)amino)-2-fluorobenzonitrile (<b>21</b>)</h4><div class="NLM_p last">46% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.75 (s, 1H), 8.54 (s, 1H), 8.18 (dd, <i>J</i> = 5.5, 2.9 Hz, 1H), 8.02–7.87 (m, 1H), 7.72 (s, 2H), 7.48 (t, <i>J</i> = 9.1 Hz, 1H), 4.64 (d, <i>J</i> = 3.8 Hz, 1H), 3.68 (p, <i>J</i> = 5.4 Hz, 1H), 3.51 (tt, <i>J</i> = 7.7, 3.6 Hz, 1H), 3.15 (tt, <i>J</i> = 7.0, 3.2 Hz, 2H), 2.73 (ddd, <i>J</i> = 11.8, 8.5, 3.5 Hz, 2H), 1.84–1.66 (m, 2H), 1.45 (dtd, <i>J</i> = 12.3, 8.2, 3.7 Hz, 2H), 1.15 (d, <i>J</i> = 6.4 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 248.2 Hz), 144.9, 142.0, 139.6 (d, <i>J</i> = 2.2 Hz), 126.5, 126.4, 123.5 (d, <i>J</i> = 7.5 Hz), 122.5, 119.1, 117.6 (d, <i>J</i> = 20.5 Hz), 115.0, 114.9, 110.5, 100.2 (d, <i>J</i> = 16.0 Hz), 64.2, 43.7, 33.4, 29.7, 6.9. HRMS (ESI) calculated for C<sub>22</sub>H<sub>23</sub>O<sub>3</sub>N<sub>5</sub>FS [M + H]<sup>+</sup> 456.1500, found 456.1477.</div></div><div id="sec5_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 1-((1-Cyclopropyl-3-((2-methylpyridin-4-yl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>22</b>)</h4><div class="NLM_p last">32% yield. <sup>1</sup>H NMR (400 MHz, DMSO): δ 9.82 (s, 1H), 8.59 (s, 1H), 8.23 (d, <i>J</i> = 5.7 Hz, 1H), 7.74 (q, <i>J</i> = 8.9 Hz, 2H), 7.50 (d, <i>J</i> = 4.3 Hz, 1H), 7.44 (s, 1H), 4.71 (s, 1H), 3.72 (dt, <i>J</i> = 10.2, 5.2 Hz, 1H), 3.51 (s, 1H), 3.15 (s, 2H), 2.72 (t, <i>J</i> = 8.4 Hz, 2H), 2.42 (s, 3H), 1.83–1.68 (m, 2H), 1.45 (d, <i>J</i> = 8.4 Hz, 2H), 1.16 (d, <i>J</i> = 4.8 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.4, 149.9, 148.6, 144.4, 142.0, 126.5, 126.2, 122.7, 115.2, 110.6, 109.9, 108.7, 64.2, 43.7, 33.4, 29.7, 25.0, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>26</sub>O<sub>3</sub>N<sub>5</sub>S [M + H]<sup>+</sup> 428.1751, found 428.1736.</div></div><div id="sec5_3_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 1-((1-Cyclopropyl-3-((5-fluoropyridin-2-yl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>23</b>)</h4><div class="NLM_p last">70% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.19 (s, 1H), 8.71 (d, <i>J</i> = 1.2 Hz, 1H), 8.23 (d, <i>J</i> = 3.0 Hz, 1H), 8.05 (dd, <i>J</i> = 9.2, 3.9 Hz, 1H), 7.78–7.66 (m, 3H), 4.65 (d, <i>J</i> = 3.8 Hz, 1H), 3.69 (tt, <i>J</i> = 6.6, 4.5 Hz, 1H), 3.52 (tq, <i>J</i> = 7.4, 3.6 Hz, 1H), 3.18–3.10 (m, 2H), 2.73 (ddd, <i>J</i> = 11.8, 8.4, 3.4 Hz, 2H), 1.76 (ddt, <i>J</i> = 14.0, 7.2, 3.5 Hz, 2H), 1.46 (dtd, <i>J</i> = 11.9, 7.9, 3.6 Hz, 2H), 1.19–1.07 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.5 (d, <i>J</i> = 243.3 Hz), 151.5, 144.2, 142.2, 135.2 (d, <i>J</i> = 24.7 Hz), 126.5, 126.1, 125.9 (d, <i>J</i> = 19.9 Hz), 123.7, 115.4, 111.1 (d, <i>J</i> = 4.0 Hz), 110.3, 64.3, 43.7, 33.4, 29.6, 6.8. HRMS (ESI) calculated for C<sub>20</sub>H<sub>23</sub>O<sub>3</sub>N<sub>5</sub>FS [M + H]<sup>+</sup> 432.1500, found 432.1484.</div></div><div id="sec5_3_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 1-((1-Cyclopropyl-3-((5-fluoro-4-methylpyridin-2-yl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>24</b>)</h4><div class="NLM_p last">43% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.02 (s, 1H), 8.66 (s, 1H), 8.10 (s, 1H), 7.86 (d, <i>J</i> = 5.6 Hz, 1H), 7.78–7.66 (m, 2H), 4.64 (d, <i>J</i> = 3.8 Hz, 1H), 3.71 (td, <i>J</i> = 6.3, 3.2 Hz, 1H), 3.52 (tq, <i>J</i> = 7.6, 3.7 Hz, 1H), 3.16 (ddt, <i>J</i> = 11.2, 7.2, 4.2 Hz, 2H), 2.73 (ddd, <i>J</i> = 11.7, 8.2, 3.3 Hz, 2H), 2.31 (s, 3H), 1.76 (ddt, <i>J</i> = 14.0, 7.2, 3.5 Hz, 2H), 1.46 (dtd, <i>J</i> = 12.1, 8.1, 3.6 Hz, 2H), 1.19–1.07 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.9 (d, <i>J</i> = 242.1 Hz), 151.4 (d, <i>J</i> = 1.9 Hz), 144.2, 142.3, 135.9 (d, <i>J</i> = 16.1 Hz), 134.5 (d, <i>J</i> = 25.7 Hz), 126.5, 126.0, 123.8, 115.5, 112.1, 110.2, 64.3, 43.7, 33.4, 29.7, 15.0 (d, <i>J</i> = 2.8 Hz), 6.8. HRMS (ESI) calculated for C<sub>21</sub>H<sub>25</sub>O<sub>3</sub>N<sub>5</sub>FS [M + H]<sup>+</sup> 446.1657, found 446.1638.</div></div><div id="sec5_3_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 1-((3-(Benzylamino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>25</b>)</h4><div class="NLM_p last">28% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.34 (d, <i>J</i> = 1.6 Hz, 1H), 7.65–7.50 (m, 2H), 7.42 (d, <i>J</i> = 7.2 Hz, 2H), 7.34 (t, <i>J</i> = 7.4 Hz, 2H), 7.25 (t, <i>J</i> = 7.3 Hz, 1H), 7.07 (t, <i>J</i> = 5.8 Hz, 1H), 4.66 (s, 1H), 4.45 (d, <i>J</i> = 5.8 Hz, 2H), 3.48 (qt, <i>J</i> = 6.8, 3.8 Hz, 2H), 3.11 (ddd, <i>J</i> = 11.2, 6.9, 3.7 Hz, 2H), 2.66 (ddd, <i>J</i> = 11.8, 8.5, 3.4 Hz, 2H), 1.73 (ddt, <i>J</i> = 13.9, 7.2, 3.5 Hz, 2H), 1.43 (dtd, <i>J</i> = 12.2, 8.2, 3.7 Hz, 2H), 1.03 (dt, <i>J</i> = 8.3, 2.8 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.4, 143.2, 140.4, 128.6, 128.3, 127.2, 125.84, 124.9, 123.1, 114.5, 109.7, 64.3, 47.0, 43.7, 33.4, 29.2. HRMS (ESI) calculated for C<sub>22</sub>H<sub>27</sub>O<sub>3</sub>N<sub>4</sub>S [M + H]<sup>+</sup> 427.1798, found 427.1786.</div></div><div id="sec5_3_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 1-((1-Cyclopropyl-3-((4-fluorobenzyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>26</b>)</h4><div class="NLM_p last">16% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.31 (s, 1H), 7.64–7.57 (m, 1H), 7.55 (d, <i>J</i> = 8.9 Hz, 1H), 7.45 (dd, <i>J</i> = 8.4, 5.6 Hz, 2H), 7.16 (t, <i>J</i> = 8.7 Hz, 2H), 7.07 (t, <i>J</i> = 5.9 Hz, 1H), 4.64 (d, <i>J</i> = 3.8 Hz, 1H), 4.42 (d, <i>J</i> = 5.7 Hz, 2H), 3.61–3.42 (m, 2H), 3.10 (t, <i>J</i> = 8.8 Hz, 2H), 2.75–2.59 (m, 2H), 1.84–1.64 (m, 2H), 1.53–1.32 (m, 2H), 1.02 (t, <i>J</i> = 6.4 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.6 (d, <i>J</i> = 242.0 Hz), 150.3, 143.2, 136.6 (d, <i>J</i> = 2.9 Hz), 130.2 (d, <i>J</i> = 8.0 Hz), 125.9, 124.9, 123.0, 115.3 (d, <i>J</i> = 21.2 Hz), 114.5, 109.8, 64.3, 46.3, 43.7, 33.4, 29.2, 6.7. HRMS (ESI) calculated for C<sub>22</sub>H<sub>26</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 445.1704, found 445.1687.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> General Procedure for Synthesis of <b>27</b> and <b>28</b></h3><div class="NLM_p">The mixture of <b>85</b> (49 mg, 0.1 mmol), the corresponding aniline (0.15 mmol), CuI (1.9 mg, 0.01 mmol), <span class="smallcaps smallerCapital">d</span>-proline (1.0 mg, 0.01 mmol), and K<sub>3</sub>PO<sub>3</sub> (63 mg, 0.2 mmol) in 1.5 mL of DMSO was stirred at 90 °C under an argon atmosphere. After that, the solution was diluted ethyl acetate and washed with water for twice. The organic phase was concentrated under a vacuum and dissolved in THF (2 mL) and H<sub>2</sub>O (2 mL). To the mixture was added LiOH (0.5 mmol). The mixture was stirred at room temperature for 1 h and extracted with ethyl acetate (15 mL). The organic layer was washed with brine, dried with anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified with column chromatography on C18 silico gel.</div><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 1-((3-(Cyclohexylamino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>27</b>)</h4><div class="NLM_p last">17% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.34 (d, <i>J</i> = 1.6 Hz, 1H), 7.58 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 7.51 (d, <i>J</i> = 8.8 Hz, 1H), 6.40 (d, <i>J</i> = 7.3 Hz, 1H), 4.64 (d, <i>J</i> = 3.8 Hz, 1H), 3.51 (qt, <i>J</i> = 7.3, 4.2 Hz, 2H), 3.44 (tt, <i>J</i> = 6.9, 3.8 Hz, 1H), 3.12 (ddd, <i>J</i> = 11.1, 7.1, 3.8 Hz, 2H), 2.68 (ddd, <i>J</i> = 11.7, 8.4, 3.4 Hz, 2H), 2.11–2.01 (m, 2H), 1.80–1.70 (m, 4H), 1.61 (dt, <i>J</i> = 12.7, 3.8 Hz, 1H), 1.45 (ddt, <i>J</i> = 12.7, 8.3, 4.0 Hz, 2H), 1.37–1.22 (m, 5H), 1.09–0.98 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.8, 143.0, 125.8, 124.5, 123.1, 114.8, 109.5, 64.3, 51.6, 43.7, 33.4, 33.0, 29.2, 26.2, 25.1, 6.8.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 1-((3-(Cyclopentylamino)-1-cyclopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>28</b>)</h4><div class="NLM_p last">57% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.32 (d, <i>J</i> = 1.5 Hz, 1H), 7.58 (dd, <i>J</i> = 8.9, 1.7 Hz, 1H), 7.52 (d, <i>J</i> = 8.8 Hz, 1H), 6.51 (d, <i>J</i> = 6.4 Hz, 1H), 4.64 (d, <i>J</i> = 3.6 Hz, 1H), 3.97 (p, <i>J</i> = 6.3 Hz, 1H), 3.49 (dtt, <i>J</i> = 16.8, 6.8, 3.8 Hz, 2H), 3.12 (ddd, <i>J</i> = 11.1, 7.1, 3.7 Hz, 2H), 2.68 (ddd, <i>J</i> = 11.7, 8.4, 3.4 Hz, 2H), 2.05–1.90 (m, 2H), 1.80–1.65 (m, 4H), 1.63–1.50 (m, 4H), 1.44 (dtd, <i>J</i> = 12.1, 8.0, 3.5 Hz, 2H), 1.03 (t, <i>J</i> = 5.1 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.3, 143.0, 125.7, 124.6, 123.1, 114.8, 109.5, 64.3, 54.5, 43.7, 33.38, 32.9, 29.2, 24.0, 6.8. HRMS (ESI) calculated for C<sub>20</sub>H<sub>29</sub>O<sub>3</sub>N<sub>4</sub>S [M + H]<sup>+</sup> 405.1955, found 405.1939.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 1-((3-((4-Fluorophenyl)amino)-1-(oxetan-3-yl)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>29</b>)</h4><div class="NLM_p last">7% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.49 (s, 1H), 8.61 (s, 1H), 7.84 (dd, <i>J</i> = 8.9, 4.8 Hz, 2H), 7.75 (d, <i>J</i> = 8.9 Hz, 1H), 7.69 (d, <i>J</i> = 9.0 Hz, 1H), 7.21 (t, <i>J</i> = 8.7 Hz, 2H), 6.04 (p, <i>J</i> = 7.2 Hz, 1H), 5.12 (t, <i>J</i> = 6.4 Hz, 2H), 4.97 (t, <i>J</i> = 7.0 Hz, 2H), 4.63 (d, <i>J</i> = 3.8 Hz, 1H), 3.50 (dt, <i>J</i> = 8.2, 3.9 Hz, 1H), 3.16 (dt, <i>J</i> = 11.2, 4.6 Hz, 2H), 2.72 (ddd, <i>J</i> = 12.0, 8.9, 3.3 Hz, 2H), 1.83–1.67 (m, 2H), 1.44 (dtd, <i>J</i> = 12.5, 8.4, 3.7 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.6 (d, <i>J</i> = 236.2 Hz), 146.6, 141.1, 138.9, 126.2, 126.1, 122.8, 117.9 (d, <i>J</i> = 7.3 Hz), 115.8 (d, <i>J</i> = 22.1 Hz), 115.0, 110.1, 76.9, 64.2, 51.4, 43.7, 33.4. HRMS (ESI) calculated for C<sub>21</sub>H<sub>24</sub>O<sub>4</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 447.1497, found 447.1476.</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-((1-Ethyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>30</b>)</h4><div class="NLM_p last">48% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 8.59 (d, <i>J</i> = 1.6 Hz, 1H), 7.81–7.73 (m, 2H), 7.72–7.62 (m, 2H), 7.16 (t, <i>J</i> = 8.9 Hz, 2H), 4.66 (d, <i>J</i> = 3.9 Hz, 1H), 4.36 (q, <i>J</i> = 7.1 Hz, 2H), 3.52 (tt, <i>J</i> = 7.8, 3.9 Hz, 1H), 3.18 (ddd, <i>J</i> = 11.1, 7.0, 3.7 Hz, 2H), 2.73 (ddd, <i>J</i> = 11.8, 8.6, 3.4 Hz, 2H), 1.76 (ddt, <i>J</i> = 14.0, 7.3, 3.6 Hz, 2H), 1.52–1.36 (m, 5H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.4 (d, <i>J</i> = 235.7 Hz), 146.0, 140.5, 139.1 (d, <i>J</i> = 2.2 Hz), 125.9, 125.3, 122.9, 117.7 (d, <i>J</i> = 7.3 Hz), 115.7 (d, <i>J</i> = 22.0 Hz), 114.4, 109.8, 64.3, 43.8, 43.3, 33.4, 15.1. HRMS (ESI) calculated for C<sub>20</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 419.1548, found 419.1530.</div></div><div id="sec5_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-((3-((4-Fluorophenyl)amino)-1-isopropyl-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>31</b>)</h4><div class="NLM_p last">44% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 8.58 (d, <i>J</i> = 1.6 Hz, 1H), 7.82–7.74 (m, 2H), 7.72 (d, <i>J</i> = 9.0 Hz, 1H), 7.64 (dd, <i>J</i> = 8.9, 1.7 Hz, 1H), 7.17 (t, <i>J</i> = 8.9 Hz, 2H), 4.92 (p, <i>J</i> = 6.5 Hz, 1H), 4.66 (d, <i>J</i> = 3.9 Hz, 1H), 3.52 (tt, <i>J</i> = 7.6, 3.7 Hz, 1H), 3.17 (ddd, <i>J</i> = 11.1, 7.0, 3.8 Hz, 2H), 2.73 (ddd, <i>J</i> = 11.7, 8.5, 3.4 Hz, 2H), 1.77 (ddt, <i>J</i> = 13.7, 7.1, 3.2 Hz, 2H), 1.54–1.41 (m, 8H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.4 (d, <i>J</i> = 235.4 Hz), 145.7, 140.0, 139.2 (d, <i>J</i> = 1.9 Hz), 125.7, 125.2, 122.84, 117.6 (d, <i>J</i> = 7.4 Hz), 115.7 (d, <i>J</i> = 22.2 Hz), 114.4, 109.8, 64.3, 49.6, 43.8, 33.4, 22.3. HRMS (ESI) calculated for C<sub>21</sub>H<sub>26</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 433.1704, found 433.1687.</div></div><div id="sec5_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 1-((1-Cyclobutyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>32</b>)</h4><div class="NLM_p last">41% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.42 (d, <i>J</i> = 2.6 Hz, 1H), 8.57 (s, 1H), 7.81 (dt, <i>J</i> = 8.0, 3.6 Hz, 2H), 7.72 (dd, <i>J</i> = 9.0, 2.6 Hz, 1H), 7.64 (d, <i>J</i> = 8.5 Hz, 1H), 7.19 (td, <i>J</i> = 9.0, 2.7 Hz, 2H), 5.22 (p, <i>J</i> = 9.0 Hz, 1H), 4.66 (t, <i>J</i> = 3.3 Hz, 1H), 3.49 (tt, <i>J</i> = 7.4, 3.8 Hz, 1H), 3.24–3.08 (m, 2H), 2.69 (q, <i>J</i> = 9.5, 8.6 Hz, 4H), 2.47–2.34 (m, 2H), 1.95–1.80 (m, 2H), 1.75 (ddd, <i>J</i> = 13.6, 7.0, 3.5 Hz, 2H), 1.45 (ddt, <i>J</i> = 16.7, 11.9, 5.6 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.6 Hz), 146.0, 140.2, 139.1 (d, <i>J</i> = 1.9 Hz), 125.9, 125.55, 122.8, 117.7 (d, <i>J</i> = 7.4 Hz), 115.8 (d, <i>J</i> = 22.1 Hz), 114.6, 111.0, 64.3, 51.7, 43.8, 33.4, 29.9, 14.9. HRMS (ESI) calculated for C<sub>22</sub>H<sub>26</sub>O<sub>3</sub>N<sub>4</sub>S [M + H]<sup>+</sup> 445.1704, found 445.1684.</div></div><div id="sec5_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 1-Cyclopropyl-N-(4-fluorophenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>33</b>)</h4><div class="NLM_p last">24% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.39 (s, 1H), 8.60 (d, <i>J</i> = 1.4 Hz, 1H), 7.75 (dd, <i>J</i> = 9.1, 4.7 Hz, 2H), 7.73–7.67 (m, 2H), 7.17 (t, <i>J</i> = 8.9 Hz, 2H), 3.74–3.59 (m, 5H), 2.97–2.80 (m, 4H), 1.14 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.7 Hz), 145.7, 142.2, 138.9 (d, <i>J</i> = 2.0 Hz), 126.2, 124.8, 123.2, 117.8 (d, <i>J</i> = 7.4 Hz), 115.8 (d, <i>J</i> = 22.0 Hz), 115.2, 110.4, 65.7, 46.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>20</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 417.1391, found 417.1375.</div></div><div id="sec5_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 1-((1-Cyclopropyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-3-ol (<b>34</b>)</h4><div class="NLM_p last">28% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 8.58 (d, <i>J</i> = 1.2 Hz, 1H), 7.80–7.72 (m, 2H), 7.70 (d, <i>J</i> = 1.7 Hz, 2H), 7.24–7.10 (m, 2H), 4.98 (d, <i>J</i> = 4.8 Hz, 1H), 3.65 (p, <i>J</i> = 5.3 Hz, 1H), 3.55 (tq, <i>J</i> = 8.8, 4.1 Hz, 1H), 3.46 (dd, <i>J</i> = 10.9, 4.2 Hz, 1H), 3.30 (s, 1H), 2.31 (td, <i>J</i> = 11.1, 2.5 Hz, 1H), 2.18–2.07 (m, 1H), 1.81–1.65 (m, 2H), 1.47 (qd, <i>J</i> = 11.7, 10.6, 3.7 Hz, 1H), 1.13 (d, <i>J</i> = 5.3 Hz, 4H), 1.11–1.00 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.5 Hz), 145.6, 142.1, 139.0 (d, <i>J</i> = 2.0 Hz), 126.1, 126.0, 122.7, 117.8 (d, <i>J</i> = 7.3 Hz), 115.7 (d, <i>J</i> = 22.1 Hz), 115.1, 110.3, 65.5, 53.1, 46.3, 32.4, 29.6, 22.7, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 431.1548, found 431.1534.</div></div><div id="sec5_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-((1-Cyclopropyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)pyrrolidine-3-carboxylic acid (<b>35</b>)</h4><div class="NLM_p last">43% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.52 (s, 1H), 9.37 (s, 1H), 8.66 (s, 1H), 7.77 (dt, <i>J</i> = 8.9, 5.7 Hz, 3H), 7.69 (d, <i>J</i> = 8.9 Hz, 1H), 7.17 (t, <i>J</i> = 8.8 Hz, 2H), 3.65 (p, <i>J</i> = 5.3 Hz, 1H), 3.43–3.28 (m, 2H), 3.28–3.12 (m, 2H), 2.94 (p, <i>J</i> = 7.3 Hz, 1H), 1.97 (dd, <i>J</i> = 13.0, 7.1 Hz, 1H), 1.88 (td, <i>J</i> = 14.6, 12.8, 7.9 Hz, 1H), 1.14 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.3, 156.5 (d, <i>J</i> = 235.7 Hz), 145.7, 142.1, 139.0 (d, <i>J</i> = 2.1 Hz), 126.2, 126.0, 122.9, 117.8 (d, <i>J</i> = 7.5 Hz), 115.7 (d, <i>J</i> = 22.0 Hz), 115.1, 110.3, 50.2, 47.9, 42.5, 29.6, 28.4, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>22</sub>O<sub>4</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 445.1340, found 445.1323.</div></div><div id="sec5_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 1-((1-Cyclopropyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)pyrrolidin-3-ol (<b>36</b>)</h4><div class="NLM_p last">26% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.41 (s, 1H), 8.64 (s, 1H), 7.85–7.71 (m, 3H), 7.67 (d, <i>J</i> = 8.8 Hz, 1H), 7.17 (t, <i>J</i> = 8.7 Hz, 2H), 4.91 (s, 1H), 4.14 (s, 1H), 3.64 (p, <i>J</i> = 5.2 Hz, 1H), 3.25 (ddd, <i>J</i> = 15.8, 9.8, 6.2 Hz, 3H), 3.01 (dd, <i>J</i> = 10.4, 2.4 Hz, 1H), 1.72 (dtd, <i>J</i> = 13.3, 8.6, 4.9 Hz, 1H), 1.61 (ddt, <i>J</i> = 13.3, 7.0, 3.6 Hz, 1H), 1.13 (d, <i>J</i> = 5.4 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.5 Hz), 145.7, 142.1, 139.0 (d, <i>J</i> = 2.2 Hz), 126.7, 126.2, 122.6, 117.8 (d, <i>J</i> = 7.4 Hz), 115.7 (d, <i>J</i> = 22.2 Hz), 115.1, 110.2, 69.4, 56.3, 46.6, 34.1, 29.6, 6.8. HRMS (ESI) calculated for C<sub>20</sub>H<sub>22</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 417.1391, found 417.1370.</div></div><div id="sec5_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (1-((1-Cyclopropyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-yl)methanol (<b>37</b>)</h4><div class="NLM_p last">25% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.39 (s, 1H), 8.57 (d, <i>J</i> = 1.4 Hz, 1H), 7.84–7.70 (m, 3H), 7.68 (d, <i>J</i> = 1.2 Hz, 2H), 7.16 (t, <i>J</i> = 8.9 Hz, 2H), 4.49 (t, <i>J</i> = 5.1 Hz, 1H), 3.75–3.58 (m, 3H), 3.27–3.10 (m, 2H), 2.17 (td, <i>J</i> = 11.7, 2.4 Hz, 2H), 1.70 (dd, <i>J</i> = 12.9, 3.3 Hz, 2H), 1.20–1.07 (m, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.9 Hz), 145.7, 142.1, 139.0 (d, <i>J</i> = 2.2 Hz), 126.1 (d, <i>J</i> = 8.7 Hz), 122.8, 117.8 (d, <i>J</i> = 7.3 Hz), 115.8, 115.6, 115.1, 110.2, 65.7, 46.4, 37.7, 29.6, 28.3, 6.9. HRMS (ESI) calculated for C<sub>22</sub>H<sub>26</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 445.1704, found 445.1688.</div></div><div id="sec5_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 5-((1,4-Dioxa-8-azaspiro[4.5]decan-8-yl)sulfonyl)-1-cyclopropyl-<i>N</i>-(4-fluorophenyl)-1<i>H</i>-indazol-3-amine (<b>38</b>)</h4><div class="NLM_p last">25% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.36 (s, 1H), 8.60 (s, 1H), 7.75 (dd, <i>J</i> = 8.9, 4.8 Hz, 2H), 7.71 (s, 2H), 7.17 (t, <i>J</i> = 8.8 Hz, 2H), 3.78 (s, 4H), 3.65 (dq, <i>J</i> = 6.8, 4.5, 3.5 Hz, 1H), 3.00 (t, <i>J</i> = 5.5 Hz, 4H), 1.69 (t, <i>J</i> = 5.6 Hz, 4H), 1.20–1.05 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.5 (d, <i>J</i> = 235.9 Hz), 145.7, 142.09, 139.0 (d, <i>J</i> = 2.1 Hz), 126.1, 125.6, 122.8, 117.8 (d, <i>J</i> = 7.4 Hz), 115.8 (d, <i>J</i> = 22.2 Hz), 115.1, 110.3, 105.6, 64.2, 45.0, 34.1, 29.6, 6.8. HRMS (ESI) calculated for C<sub>23</sub>H<sub>26</sub>O<sub>4</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 473.1653, found 473.1636.</div></div><div id="sec5_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 1-((1-Cyclopropyl-3-((4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-one (<b>39</b>)</h4><div class="NLM_p last">To the solution of <b>38</b> (0.05 mmol, 23 mg) in 1 mL of acetone was added HCl (concd, 1 mL). The mixture was stirred at room temperature. Upon completion, acetone was evaporated under a vacuum. The residue was neutralized with 1 N NaOH. The product was collected by filtration and dried under a vacuum at 40 °C. 80% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.40 (s, 1H), 8.67 (s, 1H), 7.82–7.66 (m, 4H), 7.16 (t, <i>J</i> = 8.7 Hz, 2H), 3.66 (p, <i>J</i> = 5.4 Hz, 1H), 3.33 (d, <i>J</i> = 6.6 Hz, 4H), 2.44 (t, <i>J</i> = 6.1 Hz, 4H), 1.19–1.07 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 206.0, 156.50 (d, <i>J</i> = 235.6 Hz), 145.7, 142.2, 139.0 (d, <i>J</i> = 2.1 Hz), 126.3, 126.0, 122.9, 117.8 (d, <i>J</i> = 7.4 Hz), 115.7 (d, <i>J</i> = 22.2 Hz), 115.2, 110.4, 45.6, 29.6, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>22</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 429.1391, found 429.1376.</div></div><div id="sec5_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 1-Cyclopropyl-<i>N</i>-(4-fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>40</b>)</h4><div class="NLM_p last">58% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.29 (s, 1H), 8.59 (d, <i>J</i> = 1.5 Hz, 1H), 7.78–7.67 (m, 2H), 7.67–7.60 (m, 1H), 7.54 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.10 (t, <i>J</i> = 9.2 Hz, 1H), 3.75–3.59 (m, 5H), 2.87 (dd, <i>J</i> = 5.6, 3.4 Hz, 4H), 2.25 (s, 3H), 1.14 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.2 (d, <i>J</i> = 235.1 Hz), 145.7, 142.2, 138.6 (d, <i>J</i> = 2.2 Hz), 126.2, 124.7, 124.5 (d, <i>J</i> = 18.1 Hz), 123.2, 119.2 (d, <i>J</i> = 4.0 Hz), 110.3, 65.7, 46.4, 29.6, 15.1 (d, <i>J</i> = 3.0 Hz), 6.8. HRMS (ESI) calculated for C<sub>21</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 431.1548, found 431.1530.</div></div><div id="sec5_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 1-Cyclopropyl-6-fluoro-<i>N</i>-(4-fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>41</b>)</h4><div class="NLM_p last">56% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.36 (s, 1H), 8.66 (d, <i>J</i> = 6.7 Hz, 1H), 7.61 (ddd, <i>J</i> = 8.9, 4.4, 2.9 Hz, 1H), 7.55 (d, <i>J</i> = 11.3 Hz, 1H), 7.52 (dd, <i>J</i> = 6.8, 2.8 Hz, 1H), 7.08 (t, <i>J</i> = 9.2 Hz, 1H), 3.77–3.55 (m, 5H), 3.03 (dd, <i>J</i> = 5.7, 3.5 Hz, 4H), 2.24 (d, <i>J</i> = 1.8 Hz, 3H), 1.11 (td, <i>J</i> = 5.2, 4.6, 3.1 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.7 (d, <i>J</i> = 250.6 Hz), 154.8 (d, <i>J</i> = 235.3 Hz), 145.5, 142.2 (d, <i>J</i> = 12.5 Hz), 138.0 (d, <i>J</i> = 2.2 Hz), 125.9 (d, <i>J</i> = 3.2 Hz), 124.0 (d, <i>J</i> = 18.1 Hz), 118.8 (d, <i>J</i> = 4.0 Hz), 115.1 (d, <i>J</i> = 11.7 Hz), 114.9 (d, <i>J</i> = 6.2 Hz), 114.8 (d, <i>J</i> = 2.1 Hz), 111.6, 96.8 (d, <i>J</i> = 27.7 Hz), 65.6, 45.6, 29.2, 14.6 (d, <i>J</i> = 3.0 Hz), 6.3. HRMS (ESI) calculated for C<sub>21</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>F<sub>2</sub>S [M + H]<sup>+</sup> 449.1453, found 449.1438.</div></div><div id="sec5_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 1-Cyclopropyl-<i>N</i>-(4-fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-3-amine (<b>42</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.44 (s, 1H), 9.13 (s, 1H), 8.73 (s, 1H), 7.61 (dt, <i>J</i> = 8.4, 3.6 Hz, 1H), 7.52 (dd, <i>J</i> = 6.8, 2.8 Hz, 1H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 3.83 (tt, <i>J</i> = 7.0, 3.8 Hz, 1H), 3.64 (t, <i>J</i> = 4.7 Hz, 4H), 3.13 (t, <i>J</i> = 4.7 Hz, 4H), 2.26 (d, <i>J</i> = 1.8 Hz, 3H), 1.31–1.12 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.4 (d, <i>J</i> = 235.5 Hz), 145.6, 142.6, 138.3 (d, <i>J</i> = 2.2 Hz), 137.7, 134.7, 124.6 (d, <i>J</i> = 18.0 Hz), 119.3 (d, <i>J</i> = 4.2 Hz), 119.0, 117.4, 115.7–115.4 (m), 115.3, 66.1, 46.9, 30.2, 15.1 (d, <i>J</i> = 2.9 Hz), 7.0. HRMS (ESI) calculated for C<sub>20</sub>H<sub>23</sub>O<sub>3</sub>N<sub>5</sub>FS [M + H]<sup>+</sup> 432.1500, found 432.1488.</div></div><div id="sec5_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 1-Cyclopropyl-<i>N</i>-(3,4-difluorophenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>43</b>)</h4><div class="NLM_p last">9% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.63 (s, 1H), 8.57 (s, 1H), 7.89 (dd, <i>J</i> = 13.8, 7.1 Hz, 1H), 7.72 (q, <i>J</i> = 8.9 Hz, 2H), 7.45–7.33 (m, 2H), 3.66 (dt, <i>J</i> = 15.6, 5.0 Hz, 5H), 2.87 (t, <i>J</i> = 4.5 Hz, 4H), 1.15 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.7 (dd, <i>J</i> = 241.7, 12.9 Hz), 145.2, 143.5 (dd, <i>J</i> = 237.1, 12.9 Hz), 142.1, 139.6 (dd, <i>J</i> = 9.6, 2.2 Hz), 126.3, 125.1, 123.0, 118.0 (d, <i>J</i> = 17.8 Hz), 115.0, 112.6 (dd, <i>J</i> = 5.6, 2.9 Hz), 110.5, 105.1 (d, <i>J</i> = 22.1 Hz), 65.7, 46.4, 29.6, 6.9. HRMS (ESI) calculated for C<sub>20</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>F<sub>2</sub>S [M + H]<sup>+</sup> 435.1297, found 435.1282.</div></div><div id="sec5_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 1-((1-Cyclopropyl-3-((3,4-difluorophenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-ol (<b>44</b>)</h4><div class="NLM_p last">47% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.59 (s, 1H), 8.64 (s, 1H), 7.97–7.81 (m, 2H), 7.74 (d, <i>J</i> = 8.9 Hz, 1H), 7.46–7.31 (m, 2H), 4.47 (s, 2H), 3.76–3.63 (m, 2H), 3.61 (d, <i>J</i> = 7.5 Hz, 1H), 3.23 (d, <i>J</i> = 10.0 Hz, 1H), 3.12 (d, <i>J</i> = 9.9 Hz, 1H), 1.57 (d, <i>J</i> = 9.8 Hz, 1H), 1.15 (d, <i>J</i> = 6.3 Hz, 4H), 0.97 (d, <i>J</i> = 10.0 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.7 (dd, <i>J</i> = 241.8, 13.0 Hz), 145.3, 143.5 (dd, <i>J</i> = 237.1, 12.8 Hz), 142.0, 139.6 (dd, <i>J</i> = 9.5, 2.2 Hz), 128.3, 126.0, 122.6, 117.9 (d, <i>J</i> = 17.6 Hz), 114.9, 112.6 (dd, <i>J</i> = 5.6, 3.0 Hz), 110.7, 105.0 (d, <i>J</i> = 22.1 Hz), 76.1, 73.6, 60.3, 56.2, 35.4, 29.6, 6.8. HRMS (ESI) calculated for C<sub>20</sub>H<sub>22</sub>O<sub>3</sub>N<sub>2</sub>F<sub>2</sub>S [M + H]<sup>+</sup> 450.1406, found 450.1388.</div></div><div id="sec5_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 1-Cyclopropyl-N-(3,4-difluorophenyl)-5-(morpholinosulfonyl)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-3-amine (<b>45</b>)</h4><div class="NLM_p last">30% yield. <sup>1</sup>H NMR (400 MHz, DMSO): δ 9.75 (s, 1H), 9.17 (s, 1H), 8.71 (s, 1H), 7.85 (dd, <i>J</i> = 12.8, 6.6 Hz, 1H), 7.41 (dd, <i>J</i> = 17.8, 7.7 Hz, 2H), 3.85 (d, <i>J</i> = 3.7 Hz, 1H), 3.64 (s, 4H), 3.13 (s, 4H), 1.34–1.08 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 149.7 (dd, <i>J</i> = 242.0, 13.3 Hz), 145.1, 143.7 (dd, <i>J</i> = 237.7, 12.8 Hz), 142.8, 139.2 (dd, <i>J</i> = 9.5, 2.1 Hz), 137.6, 134.9, 118.8, 118.1 (d, <i>J</i> = 17.8 Hz), 117.2, 112.7 (dd, <i>J</i> = 5.7, 3.0 Hz), 105.1 (d, <i>J</i> = 22.2 Hz), 66.1, 46.8, 30.3, 7.0. HRMS (ESI) calculated for C<sub>19</sub>H<sub>20</sub>O<sub>3</sub>N<sub>5</sub>F<sub>2</sub>S [M + H]<sup>+</sup> 436.1249, found 436.1234.</div></div><div id="sec5_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 1-Cyclopropyl-<i>N</i>-(4-fluoro-3-(trifluoromethyl)phenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>46</b>)</h4><div class="NLM_p last">68% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.77 (s, 1H), 8.58 (d, <i>J</i> = 1.5 Hz, 1H), 8.18 (dd, <i>J</i> = 6.3, 2.8 Hz, 1H), 8.01 (dt, <i>J</i> = 8.9, 3.6 Hz, 1H), 7.77 (d, <i>J</i> = 8.9 Hz, 1H), 7.72 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 7.49 (t, <i>J</i> = 9.8 Hz, 1H), 3.73 (p, <i>J</i> = 5.3 Hz, 1H), 3.65 (dd, <i>J</i> = 5.8, 3.6 Hz, 4H), 2.97–2.80 (m, 4H), 1.15 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.6 (dd, <i>J</i> = 244.9, 2.6 Hz), 145.1, 142.1, 139.2 (d, <i>J</i> = 2.3 Hz), 126.3, 125.2, 123.3 (q, <i>J</i> = 271.8 Hz), 123.0, 121.8 (d, <i>J</i> = 7.7 Hz), 118.1 (d, <i>J</i> = 21.5 Hz), 116.9 (qd, <i>J</i> = 31.8, 13.1 Hz), 115.0, 113.8 (q, <i>J</i> = 5.0 Hz), 110.5, 65.7, 46.4, 29.7, 6.7. HRMS (ESI) calculated for C<sub>21</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 485.1265, found 485.1247.</div></div><div id="sec5_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 1-((1-Cyclopropyl-3-((4-fluoro-3-(trifluoromethyl)phenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>47</b>)</h4><div class="NLM_p last">56% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.74 (s, 1H), 8.56 (s, 1H), 8.17 (dd, <i>J</i> = 6.3, 2.8 Hz, 1H), 8.01 (dt, <i>J</i> = 9.1, 3.6 Hz, 1H), 7.81–7.69 (m, 2H), 7.49 (t, <i>J</i> = 9.8 Hz, 1H), 4.65 (d, <i>J</i> = 3.9 Hz, 1H), 3.72 (p, <i>J</i> = 5.3 Hz, 1H), 3.51 (dh, <i>J</i> = 7.8, 3.7 Hz, 1H), 3.25–3.07 (m, 2H), 2.74 (ddd, <i>J</i> = 11.8, 8.5, 3.4 Hz, 2H), 1.76 (ddt, <i>J</i> = 10.6, 6.9, 3.5 Hz, 2H), 1.45 (dtd, <i>J</i> = 11.9, 7.9, 3.5 Hz, 2H), 1.15 (d, <i>J</i> = 5.3 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.3–150.9 (m), 145.0, 142.0, 139.2 (d, <i>J</i> = 2.2 Hz), 126.3 (d, <i>J</i> = 3.3 Hz), 123.3 (q, <i>J</i> = 271.8 Hz), 122.6, 121.7 (d, <i>J</i> = 7.5 Hz), 118.2, 118.0, 116.9 (qd, <i>J</i> = 32.0, 13.0 Hz), 115.0, 113.8 (q, <i>J</i> = 5.3, 4.8 Hz), 110.4, 64.2, 43.7, 33.4, 29.7, 6.7. HRMS (ESI) calculated for C<sub>22</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 499.1421, found 499.1403.</div></div><div id="sec5_4_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 1-Cyclopropyl-<i>N</i>-(4-fluoro-3-(trifluoromethyl)phenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>48</b>)</h4><div class="NLM_p last">37% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.91 (s, 1H), 9.19 (d, <i>J</i> = 1.1 Hz, 1H), 8.70 (d, <i>J</i> = 1.1 Hz, 1H), 8.15 (dd, <i>J</i> = 6.2, 2.9 Hz, 1H), 7.96 (dt, <i>J</i> = 9.0, 3.6 Hz, 1H), 7.51 (t, <i>J</i> = 9.8 Hz, 1H), 3.94–3.83 (m, 1H), 3.64 (dd, <i>J</i> = 5.9, 3.5 Hz, 4H), 3.25–3.00 (m, 4H), 1.21 (ddt, <i>J</i> = 12.7, 9.3, 5.0 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.8 (d, <i>J</i> = 245.9 Hz), 145.0, 142.9, 138.8 (d, <i>J</i> = 2.5 Hz), 137.6, 135.0, 128.9–123.6 (m), 121.9 (d, <i>J</i> = 8.5 Hz), 118.8, 118.3 (d, <i>J</i> = 21.3 Hz), 117.1, 117.2–116.6 (m), 113.8 (d, <i>J</i> = 5.2 Hz), 66.1, 46.8, 30.3, 6.8. HRMS (ESI) calculated for C<sub>20</sub>H<sub>20</sub>O<sub>3</sub>N<sub>5</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 486.1218, found 486.1201.</div></div><div id="sec5_4_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 1-((1-Cyclopropyl-3-((3-cyclopropyl-4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>49</b>)</h4><div class="NLM_p last">47% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.26 (s, 1H), 8.55 (s, 1H), 7.70 (s, 2H), 7.62–7.52 (m, 1H), 7.31 (dd, <i>J</i> = 6.8, 2.8 Hz, 1H), 7.09 (t, <i>J</i> = 9.5 Hz, 1H), 4.65 (d, <i>J</i> = 3.8 Hz, 1H), 3.75–3.61 (m, 1H), 3.51 (tt, <i>J</i> = 7.6, 3.9 Hz, 1H), 3.15 (ddd, <i>J</i> = 11.2, 6.7, 3.5 Hz, 2H), 2.72 (dd, <i>J</i> = 20.2, 3.4 Hz, 2H), 2.07 (dp, <i>J</i> = 8.5, 5.1 Hz, 1H), 1.75 (ddd, <i>J</i> = 11.3, 7.4, 3.6 Hz, 2H), 1.45 (dtd, <i>J</i> = 12.3, 8.2, 3.6 Hz, 2H), 1.12 (d, <i>J</i> = 2.3 Hz, 4H), 1.03 (dt, <i>J</i> = 8.7, 3.2 Hz, 2H), 0.78–0.66 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.6 (d, <i>J</i> = 235.3 Hz), 145.6, 142.0, 139.0 (d, <i>J</i> = 2.0 Hz), 130.4 (d, <i>J</i> = 15.3 Hz), 126.2, 125.8, 122.7, 115.4 (d, <i>J</i> = 23.1 Hz), 115.1, 114.5 (d, <i>J</i> = 7.5 Hz), 113.4 (d, <i>J</i> = 3.4 Hz), 110.1, 64.2, 43.7, 33.4, 29.6, 8.9 (d, <i>J</i> = 4.7 Hz), 8.5, 6.7. HRMS (ESI) calculated for C<sub>24</sub>H<sub>28</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 471.1861, found 471.1846.</div></div><div id="sec5_4_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>N</i>-(4-Fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1-(2,2,2-trifluoroethyl)-1<i>H</i>-indazol-3-amine (<b>50</b>)</h4><div class="NLM_p last">51% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.44 (s, 1H), 8.68 (s, 1H), 7.89 (d, <i>J</i> = 8.8 Hz, 1H), 7.78 (d, <i>J</i> = 9.0 Hz, 1H), 7.65 (dt, <i>J</i> = 8.2, 3.6 Hz, 1H), 7.59 (d, <i>J</i> = 6.9 Hz, 1H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 5.39 (q, <i>J</i> = 9.0 Hz, 2H), 3.66 (t, <i>J</i> = 4.6 Hz, 4H), 2.90 (t, <i>J</i> = 4.6 Hz, 4H), 2.26 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.4 (d, <i>J</i> = 235.5 Hz), 147.2, 142.6, 138.3 (d, <i>J</i> = 2.2 Hz), 126.9, 126.2 (q, <i>J</i> = 281.5 Hz), 125.7, 124.6 (d, <i>J</i> = 18.0 Hz), 123.2, 119.5 (d, <i>J</i> = 4.0 Hz), 115.6, 115.5, 115.4 (d, <i>J</i> = 16.7 Hz), 110.4, 65.8, 49.2 (q, <i>J</i> = 32.8 Hz), 46.4, 15.1 (d, <i>J</i> = 3.0 Hz). HRMS (ESI) calculated for C<sub>20</sub>H<sub>21</sub>O<sub>3</sub>N<sub>4</sub>F<sub>4</sub>S [M + H]<sup>+</sup> 473.1265, found 473.1249.</div></div><div id="sec5_4_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 1-(2,2-Difluoroethyl)-<i>N</i>-(4-fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>51</b>)</h4><div class="NLM_p last">57% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.39 (s, 1H), 8.64 (s, 1H), 7.80 (d, <i>J</i> = 8.9 Hz, 1H), 7.71 (d, <i>J</i> = 9.0 Hz, 1H), 7.68–7.61 (m, 1H), 7.58 (d, <i>J</i> = 6.6 Hz, 1H), 7.09 (t, <i>J</i> = 9.2 Hz, 1H), 6.48 (t, <i>J</i> = 54.9 Hz, 1H), 4.86 (t, <i>J</i> = 14.8 Hz, 2H), 3.64 (t, <i>J</i> = 4.4 Hz, 4H), 2.87 (s, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.3 (d, <i>J</i> = 235.4 Hz), 146.8, 142.4, 138.4 (d, <i>J</i> = 2.2 Hz), 126.5, 125.1, 124.6 (d, <i>J</i> = 18.0 Hz), 123.1, 119.4 (d, <i>J</i> = 4.1 Hz), 115.3 (d, <i>J</i> = 14.7 Hz), 115.3 (d, <i>J</i> = 35.9 Hz), 115.0 (t, <i>J</i> = 241.8 Hz), 110.5, 65.8, 50.3 (t, <i>J</i> = 26.0 Hz), 46.4, 15.1 (d, <i>J</i> = 2.9 Hz). HRMS (ESI) calculated for C<sub>20</sub>H<sub>22</sub>O<sub>3</sub>N<sub>4</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 455.1359, found 455.1341.</div></div><div id="sec5_4_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>N</i>-(4-Fluoro-3-methylphenyl)-1-(2-fluoroethyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>52</b>)</h4><div class="NLM_p last">48% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.36 (s, 1H), 8.63 (s, 1H), 7.74 (d, <i>J</i> = 8.9 Hz, 1H), 7.66 (dd, <i>J</i> = 14.4, 6.3 Hz, 2H), 7.59 (dd, <i>J</i> = 6.9, 2.7 Hz, 1H), 7.09 (t, <i>J</i> = 9.2 Hz, 1H), 4.86 (dt, <i>J</i> = 47.3, 4.7 Hz, 2H), 4.68 (dt, <i>J</i> = 27.7, 4.8 Hz, 2H), 3.64 (d, <i>J</i> = 4.6 Hz, 4H), 2.89 (d, <i>J</i> = 4.6 Hz, 4H), 2.26 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.2 (d, <i>J</i> = 235.1 Hz), 146.5, 142.0, 138.6 (d, <i>J</i> = 2.2 Hz), 126.1, 124.5 (d, <i>J</i> = 18.1 Hz), 124.5, 123.2, 119.2 (d, <i>J</i> = 4.2 Hz), 115.4 (d, <i>J</i> = 13.0 Hz), 115.3 (d, <i>J</i> = 2.5 Hz), 114.8, 110.24, 82.6 (d, <i>J</i> = 167.7 Hz), 65.8, 49.0 (d, <i>J</i> = 19.9 Hz), 46.4, 15.1 (d, <i>J</i> = 2.9 Hz). HRMS (ESI) calculated for C<sub>20</sub>H<sub>23</sub>O<sub>3</sub>N<sub>4</sub>F<sub>2</sub>S [M + H]<sup>+</sup> 437.1453, found 437.1436.</div></div><div id="sec5_4_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 1-(Difluoromethyl)-<i>N</i>-(4-fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>53</b>)</h4><div class="NLM_p last">39% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.11 (s, 1H), 8.76 (s, 1H), 8.23 (t, <i>J</i> = 58.7 Hz, 1H), 7.68 (d, <i>J</i> = 1.4 Hz, 1H), 7.29 (dd, <i>J</i> = 7.0, 2.5 Hz, 1H), 7.25–7.12 (m, 2H), 6.87 (d, <i>J</i> = 1.4 Hz, 1H), 3.76–3.53 (m, 4H), 2.92 (t, <i>J</i> = 4.7 Hz, 4H), 2.23 (d, <i>J</i> = 1.9 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.1 (d, <i>J</i> = 238.8 Hz), 144.1, 137.4 (d, <i>J</i> = 2.8 Hz), 136.8, 131.3, 127.4, 125.3 (d, <i>J</i> = 18.2 Hz), 124.9 (d, <i>J</i> = 4.8 Hz), 121.2, 121.1 (d, <i>J</i> = 7.8 Hz), 116.0 (d, <i>J</i> = 23.1 Hz), 113.2, 111.2 (t, <i>J</i> = 252.6 Hz), 100.7, 65.9, 46.4, 14.8 (d, <i>J</i> = 2.9 Hz). HRMS (ESI) calculated for C<sub>19</sub>H<sub>20</sub>O<sub>3</sub>N<sub>4</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 441.1203, found 441.1185.</div></div><div id="sec5_4_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> 5-(((1<i>S</i>,4<i>S</i>)-2-Oxa-5-azabicyclo[2.2.1]heptan-5-yl)sulfonyl)-1-cyclopropyl-<i>N</i>-(4-fluoro-3-methylphenyl)-1<i>H</i>-indazol-3-amine (<b>54</b>)</h4><div class="NLM_p last">56% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.28 (s, 1H), 8.67 (d, <i>J</i> = 1.6 Hz, 1H), 7.82 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 7.69 (d, <i>J</i> = 8.8 Hz, 1H), 7.64 (ddd, <i>J</i> = 8.9, 4.5, 3.0 Hz, 1H), 7.54 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.10 (t, <i>J</i> = 9.2 Hz, 1H), 4.46 (d, <i>J</i> = 3.0 Hz, 2H), 3.72–3.63 (m, 2H), 3.60 (dd, <i>J</i> = 7.6, 1.8 Hz, 1H), 3.23 (d, <i>J</i> = 10.0 Hz, 1H), 3.12 (dd, <i>J</i> = 10.0, 1.6 Hz, 1H), 2.25 (d, <i>J</i> = 1.9 Hz, 3H), 1.62–1.49 (m, 1H), 1.19–1.07 (m, 4H), 0.96 (dd, <i>J</i> = 10.1, 2.3 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.17 (d, <i>J</i> = 235.1 Hz), 145.75, 142.05, 138.63 (d, <i>J</i> = 2.4 Hz), 127.89, 125.91, 124.49 (d, <i>J</i> = 18.0 Hz), 122.70, 119.16 (d, <i>J</i> = 4.1 Hz), 115.39 (d, <i>J</i> = 22.9 Hz), 115.18 (d, <i>J</i> = 7.1 Hz), 115.13, 110.53, 76.09, 73.61, 60.29, 56.24, 35.34, 29.60, 15.12 (d, <i>J</i> = 2.9 Hz), 6.82. HRMS (ESI) calculated for C<sub>22</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 443.1548, found 443.1530.</div></div><div id="sec5_4_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> <i>N</i>-(4-fluoro-3-methylphenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>55</b>)</h4><div class="NLM_p last">46% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.54 (s, 1H), 9.25 (s, 1H), 8.61 (s, 1H), 7.78–7.40 (m, 4H), 7.07 (t, <i>J</i> = 9.3 Hz, 1H), 3.65 (s, 5H), 2.88 (s, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.0 (d, <i>J</i> = 234.5 Hz), 146.9, 141.92, 139.0, 125.9, 124.4 (d, <i>J</i> = 17.8 Hz), 124.2, 123.1, 119.0 (d, <i>J</i> = 3.9 Hz), 115.4 (d, <i>J</i> = 22.2 Hz), 115.2, 114.1, 110.7, 65.8, 46.4, 15.1 (d, <i>J</i> = 3.1 Hz). HRMS (ESI) calculated for C<sub>18</sub>H<sub>20</sub>O<sub>3</sub>N<sub>4</sub>FS [M + H]<sup>+</sup> 391.1235, found 391.1217.</div></div><div id="sec5_4_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-ol (<b>56</b>)</h4><div class="NLM_p last">64% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.43 (s, 1H), 9.12 (s, 1H), 8.71 (s, 1H), 7.61 (dt, <i>J</i> = 8.3, 3.6 Hz, 1H), 7.51 (dd, <i>J</i> = 6.8, 2.8 Hz, 1H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 4.70 (d, <i>J</i> = 3.9 Hz, 1H), 3.82 (dq, <i>J</i> = 7.1, 3.5 Hz, 1H), 3.55 (tt, <i>J</i> = 8.1, 4.6 Hz, 1H), 3.48–3.37 (m, 2H), 2.94 (ddd, <i>J</i> = 12.2, 8.9, 3.4 Hz, 2H), 2.38–2.16 (m, 3H), 1.83–1.68 (m, 2H), 1.42 (dtd, <i>J</i> = 12.6, 8.6, 3.7 Hz, 2H), 1.28–1.13 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 155.3 (d, <i>J</i> = 235.5 Hz), 145.6, 143.5, 138.3 (d, <i>J</i> = 2.3 Hz), 137.6, 134.5, 124.6 (d, <i>J</i> = 18.2 Hz), 119.3 (d, <i>J</i> = 4.3 Hz), 119.1, 116.9, 115.6, 115.4, 115.3 (d, <i>J</i> = 7.2 Hz), 64.8, 44.2, 33.8, 30.2, 15.1 (d, <i>J</i> = 3.0 Hz), 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>25</sub>O<sub>3</sub>N<sub>5</sub>FS [M + H]<sup>+</sup> 446.1657, found 446.1639.</div></div><div id="sec5_4_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 1-((1-Cyclopropyl-3-((3-(difluoromethyl)-4-fluorophenyl)amino)-1<i>H</i>-indazol-5-yl)sulfonyl)piperidin-4-ol (<b>57</b>)</h4><div class="NLM_p last">61% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (s, 1H), 8.57 (s, 1H), 7.96 (td, <i>J</i> = 8.3, 3.7 Hz, 2H), 7.72 (s, 1H), 7.37 (d, <i>J</i> = 10.3 Hz, 1H), 7.23 (t, 1H), 4.67 (d, <i>J</i> = 4.0 Hz, 1H), 3.69 (q, <i>J</i> = 5.3 Hz, 1H), 3.51 (dt, <i>J</i> = 7.9, 3.8 Hz, 1H), 3.16 (p, <i>J</i> = 4.7 Hz, 2H), 2.72 (ddd, <i>J</i> = 11.8, 8.3, 3.8 Hz, 2H), 1.86–1.61 (m, 2H), 1.45 (dp, <i>J</i> = 12.5, 4.2 Hz, 2H), 1.14 (d, <i>J</i> = 5.2 Hz, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.7 (dt, <i>J</i> = 242.3, 5.6 Hz), 145.2, 142.0, 139.2 (d, <i>J</i> = 2.2 Hz), 126.3, 126.2, 122.6, 121.6 (td, <i>J</i> = 22.7, 13.5 Hz), 120.1 (d, <i>J</i> = 7.3 Hz), 117.0 (d, <i>J</i> = 21.1 Hz), 115.1, 114.0 (dt, <i>J</i> = 6.4, 3.5 Hz), 112.3 (td, <i>J</i> = 236.0, 3.8 Hz), 110.3, 64.2, 43.7, 33.4, 29.6, 6.8. HRMS (ESI) calculated for C<sub>22</sub>H<sub>24</sub>O<sub>3</sub>N<sub>4</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 481.1516, found 481.1499.</div></div><div id="sec5_4_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 1-Cyclopropyl-<i>N</i>-(3-(difluoromethyl)-4-fluorophenyl)-5-(morpholinosulfonyl)-1<i>H</i>-indazol-3-amine (<b>58</b>)</h4><div class="NLM_p last">58% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.63 (s, 1H), 8.58 (d, <i>J</i> = 1.7 Hz, 1H), 8.04–7.88 (m, 2H), 7.81–7.66 (m, 2H), 7.40–7.09 (m, 2H), 3.71 (p, <i>J</i> = 5.4 Hz, 1H), 3.68–3.58 (m, 4H), 2.91–2.82 (m, 4H), 1.20–1.08 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.7 (dt, <i>J</i> = 242.3, 5.7 Hz), 145.3, 142.1, 139.2 (d, <i>J</i> = 2.3 Hz), 126.3, 125.0, 123.1, 121.6 (td, <i>J</i> = 22.7, 13.5 Hz), 120.2 (d, <i>J</i> = 7.6 Hz), 117.0 (d, <i>J</i> = 21.3 Hz), 115.1, 114.0 (t, <i>J</i> = 5.7 Hz), 112.2 (td, <i>J</i> = 235.9, 3.8 Hz), 110.5, 65.75, 46.4, 29.7, 6.8. HRMS (ESI) calculated for C<sub>18</sub>H<sub>18</sub>O<sub>3</sub>N<sub>4</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 467.1359, found 467.1343.</div></div><div id="sec5_4_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 1-((1-Cyclopropyl-3-((3-(difluoromethyl)-4-fluorophenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-ol (<b>59</b>)</h4><div class="NLM_p last">55% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.74 (s, 1H), 9.15 (d, <i>J</i> = 1.1 Hz, 1H), 8.68 (d, <i>J</i> = 1.1 Hz, 1H), 7.94 (dd, <i>J</i> = 6.2, 2.8 Hz, 1H), 7.89 (dt, <i>J</i> = 8.0, 3.7 Hz, 1H), 7.42–7.35 (m, 1H), 7.34 (s, 1H), 4.68 (d, <i>J</i> = 3.9 Hz, 1H), 3.92–3.81 (m, 1H), 3.54 (tt, <i>J</i> = 8.0, 4.0 Hz, 1H), 3.46–3.36 (m, 2H), 2.93 (ddd, <i>J</i> = 12.3, 8.9, 3.4 Hz, 2H), 1.74 (dp, <i>J</i> = 13.3, 3.4 Hz, 2H), 1.41 (dtd, <i>J</i> = 12.4, 8.1, 3.5 Hz, 2H), 1.28–1.16 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.8 (d, <i>J</i> = 243.0 Hz), 145.2, 143.6, 138.8 (d, <i>J</i> = 2.3 Hz), 137.6, 134.7, 122.3–121.2 (m), 120.3 (d, <i>J</i> = 7.5 Hz), 118.9, 117.2 (d, <i>J</i> = 21.3 Hz), 116.7, 114.1 (d, <i>J</i> = 5.6 Hz), 114.9–109.5 (m), 64.8, 44.2, 33.8, 30.3, 6.9. HRMS (ESI) calculated for C<sub>21</sub>H<sub>23</sub>O<sub>3</sub>N<sub>5</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 482.1468, found 482.1453.</div></div><div id="sec5_4_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 1-Cyclopropyl-<i>N</i>-(3-(difluoromethyl)-4-fluorophenyl)-5-(morpholinosulfonyl)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-3-amine (<b>60</b>)</h4><div class="NLM_p last">56% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.78 (s, 1H), 9.17 (s, 1H), 8.71 (s, 1H), 7.95 (d, <i>J</i> = 5.7 Hz, 1H), 7.93–7.84 (m, 1H), 7.38 (d, <i>J</i> = 8.2 Hz, 1H), 7.36–7.09 (m, 1H), 3.86 (q, <i>J</i> = 4.7, 3.5 Hz, 1H), 3.63 (t, <i>J</i> = 4.4 Hz, 4H), 3.13 (t, <i>J</i> = 4.5 Hz, 4H), 1.34–1.09 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.6 (td), 145.2, 142.8, 138.8 (d, <i>J</i> = 2.2 Hz), 137.7, 134.8, 121.7 (td, <i>J</i> = 22.6, 13.2 Hz), 120.3 (d, <i>J</i> = 7.5 Hz), 118.9, 117.2, 117.0, 114.1 (d, <i>J</i> = 4.3 Hz), 112.2 (td, <i>J</i> = 236.1, 3.9 Hz), 66.1, 46.8, 30.3, 6.9. HRMS (ESI) calculated for C<sub>20</sub>H<sub>22</sub>O<sub>3</sub>N<sub>5</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 468.1312, found 468.1299.</div></div><div id="sec5_4_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate 2,2,2-trifluoroacetate (<b>61</b>)</h4><div class="NLM_p last">To the solution of <b>65</b> (30 mg) in 1 mL of DCM was added 1 mL of TFA. The solvent was evaporated under a vacuum. Ethyl ether was added to the residue. Precipitate was collected through filtration to afford <b>61</b> as a yellow solid in quantitative yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.06 (s, 1H), 9.46 (s, 1H), 9.13 (s, 1H), 8.74 (s, 1H), 7.61 (dt, <i>J</i> = 8.7, 3.5 Hz, 1H), 7.52 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.12 (t, <i>J</i> = 9.2 Hz, 1H), 4.94 (dq, <i>J</i> = 7.9, 4.0 Hz, 1H), 4.12 (s, 2H), 3.84 (tt, <i>J</i> = 7.0, 3.9 Hz, 1H), 3.45–3.36 (m, 2H), 3.18 (ddd, <i>J</i> = 12.1, 8.2, 3.6 Hz, 2H), 2.79 (s, 6H), 2.26 (d, <i>J</i> = 1.8 Hz, 3H), 2.01–1.85 (m, 2H), 1.68 (q, <i>J</i> = 9.4 Hz, 2H), 1.26–1.13 (m, 4H).</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> General Procedure for the Preparation of <b>62</b>–<b>64</b></h3><div class="NLM_p">To the solution of <b>56</b> (0.1 mmol, 44 mg) in 2 mL of DCM was added DCC (0.3 mmol, 62 mg), DMAP (0.12 mmol, 15 mg), and <i>N</i>-Boc-<span class="smallcaps smallerCapital">l</span>-amino acid or <i>N</i>,<i>N</i>-dimethyl glycine (0.3 mmol). The mixture was stirred for 3 h at 40 °C. After cooling to room temperature, the precipitate was removed through filtration. The filtrate was concentrated and purified through column chromatography on silico gel. To the solution of the above product in 1 mL of DCM was added 1 mL of TFA. The solvent was evaporated under a vacuum. Ethyl ether was added to the residue. Precipitate was collected through filtration to afford <b>62</b>–<b>64</b> as yellow solids in quantitative yield.</div><div id="sec5_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl <span class="smallcaps smallerCapital">l</span>-valinate 2,2,2-trifluoroacetate (<b>62</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.45 (s, 1H), 9.10 (d, <i>J</i> = 1.1 Hz, 1H), 8.73 (d, <i>J</i> = 1.1 Hz, 1H), 8.30 (s, 3H), 7.60 (dt, <i>J</i> = 8.7, 3.6 Hz, 1H), 7.50 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 4.93 (tt, <i>J</i> = 7.3, 3.6 Hz, 1H), 3.92–3.86 (m, 1H), 3.82 (dq, <i>J</i> = 7.0, 4.2, 3.5 Hz, 1H), 3.37 (ddt, <i>J</i> = 14.0, 6.9, 3.8 Hz, 2H), 3.28–3.12 (m, 2H), 2.25 (d, <i>J</i> = 1.8 Hz, 3H), 2.14–1.98 (m, 1H), 1.91 (s, 2H), 1.75–1.57 (m, 2H), 1.29–1.13 (m, 4H), 0.89 (t, <i>J</i> = 6.4 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.50, 158.82 (q, <i>J</i> = 32.5 Hz), 155.35 (d, <i>J</i> = 235.7 Hz), 145.67, 143.33, 138.28, 137.65, 134.45, 124.63 (d, <i>J</i> = 18.1 Hz), 119.27 (d, <i>J</i> = 4.1 Hz), 119.06, 117.25 (d, <i>J</i> = 297.5 Hz), 117.03, 115.47 (d, <i>J</i> = 23.4 Hz), 115.28, 70.68, 57.70, 43.55, 40.42, 30.18, 29.93, 29.86, 29.80, 18.65, 17.81, 15.09 (d, <i>J</i> = 3.0 Hz), 6.93.</div></div><div id="sec5_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl <span class="smallcaps smallerCapital">l</span>-alaninate 2,2,2-trifluoroacetate (<b>63</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.45 (s, 1H), 9.11 (s, 1H), 8.73 (s, 1H), 8.28 (s, 3H), 7.60 (dt, <i>J</i> = 7.7, 3.4 Hz, 1H), 7.50 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 4.88 (tt, <i>J</i> = 6.8, 3.4 Hz, 1H), 4.06 (q, <i>J</i> = 7.0 Hz, 1H), 3.82 (tt, <i>J</i> = 6.9, 4.0 Hz, 1H), 3.53–3.25 (m, 2H), 3.28–3.08 (m, 2H), 2.25 (s, 3H), 1.89 (td, <i>J</i> = 8.8, 4.6 Hz, 2H), 1.67 (p, <i>J</i> = 7.5, 6.7 Hz, 2H), 1.32 (d, <i>J</i> = 7.2 Hz, 3H), 1.27–1.13 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.65, 158.76 (q, <i>J</i> = 31.6 Hz), 155.35 (d, <i>J</i> = 235.6 Hz), 145.66, 143.47, 138.29, 137.66, 134.52, 124.63 (d, <i>J</i> = 18.1 Hz), 119.27 (d, <i>J</i> = 4.1 Hz), 119.05, 116.95, 115.48 (d, <i>J</i> = 23.6 Hz), 115.28, 70.55, 48.37, 43.49, 30.19, 29.87, 29.82, 16.12, 15.10 (d, <i>J</i> = 3.0 Hz), 6.94.</div></div><div id="sec5_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl glycinate 2,2,2-trifluoroacetate (<b>64</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 1H), 9.12 (d, <i>J</i> = 1.1 Hz, 1H), 8.73 (d, <i>J</i> = 1.1 Hz, 1H), 8.17 (s, 3H), 7.61 (dt, <i>J</i> = 8.7, 3.6 Hz, 1H), 7.51 (dd, <i>J</i> = 6.9, 2.8 Hz, 1H), 7.12 (t, <i>J</i> = 9.2 Hz, 1H), 4.90 (td, <i>J</i> = 7.7, 3.9 Hz, 1H), 3.84 (dq, <i>J</i> = 6.7, 3.4, 3.0 Hz, 1H), 3.79 (s, 2H), 3.48–3.39 (m, 2H), 3.15 (ddd, <i>J</i> = 12.3, 8.3, 3.7 Hz, 2H), 2.25 (d, <i>J</i> = 1.9 Hz, 3H), 1.98–1.85 (m, 2H), 1.65 (dtd, <i>J</i> = 12.1, 7.7, 3.6 Hz, 2H), 1.21 (dddd, <i>J</i> = 12.0, 9.5, 6.7, 3.0 Hz, 4H).</div></div><div id="sec5_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate (<b>65</b>)</h4><div class="NLM_p last">To the solution of <b>56</b> (0.1 mmol, 44 mg) in 2 mL of DCM was added DCC (0.3 mmol, 62 mg), DMAP (0.12 mmol, 15 mg), and <i>N</i>,<i>N</i>-dimethyl glycine (0.3 mmol). The mixture was stirred for 3 h at 40 °C. After cooling to room temperature, the precipitate was removed through filtration. The filtrate was concentrated and purified through column chromatography on C18 silico gel afford the product as a yellow solid with 90% yield. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>): δ 8.43 (dd, <i>J</i> = 4.0, 1.1 Hz, 1H), 7.45 (dt, <i>J</i> = 8.9, 3.6 Hz, 1H), 7.39 (dd, <i>J</i> = 6.6, 3.0 Hz, 1H), 6.98 (t, <i>J</i> = 9.0 Hz, 1H), 6.95 (s, 1H), 4.86 (tt, <i>J</i> = 7.9, 3.8 Hz, 1H), 3.63 (tq, <i>J</i> = 11.6, 3.5 Hz, 3H), 3.18 (ddd, <i>J</i> = 12.4, 8.6, 3.5 Hz, 2H), 3.12 (s, 2H), 2.37–2.25 (m, 9H), 1.97–1.86 (m, 2H), 1.77–1.68 (m, 2H), 1.33–1.21 (m, 4H). <sup>13</sup>C NMR (101 MHz, chloroform-<i>d</i>): δ 169.75, 156.32 (d, <i>J</i> = 238.4 Hz), 145.63, 143.61, 137.82, 136.93 (d, <i>J</i> = 2.5 Hz), 133.72, 125.19 (d, <i>J</i> = 18.4 Hz), 119.96 (d, <i>J</i> = 4.4 Hz), 119.41, 116.19, 115.69 (d, <i>J</i> = 7.4 Hz), 115.15 (d, <i>J</i> = 23.4 Hz), 68.95, 60.21, 45.17, 44.07, 30.23, 29.79, 14.89 (d, <i>J</i> = 3.3 Hz), 6.95. HRMS (ESI) calculated for C<sub>26</sub>H<sub>31</sub>O<sub>4</sub>N<sub>5</sub>F<sub>3</sub>S [M + H]<sup>+</sup> 531.2184, found 531.2169.</div></div><div id="sec5_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate acetate (<b>66</b>)</h4><div class="NLM_p last">To the solution of <b>65</b> (25 mg) in 1 mL of DCM was added 1 mL of AcOH. The solvent was evaporated under a vacuum. Ethyl ether was added to the residue. Precipitate was collected through filtration to afford <b>66</b> as a yellow solid in quantitative yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.93 (s, 2H), 9.44 (d, <i>J</i> = 3.2 Hz, 1H), 9.12 (d, <i>J</i> = 3.2 Hz, 1H), 8.71 (d, <i>J</i> = 3.2 Hz, 1H), 7.60 (dd, <i>J</i> = 8.6, 4.3 Hz, 1H), 7.55–7.44 (m, 1H), 7.11 (td, <i>J</i> = 9.3, 3.3 Hz, 1H), 4.78 (s, 1H), 3.82 (dq, <i>J</i> = 7.5, 3.8 Hz, 1H), 3.21–3.00 (m, 4H), 2.25 (s, 3H), 2.20 (d, <i>J</i> = 3.2 Hz, 6H), 1.98–1.78 (m, 5H), 1.60 (s, 2H), 1.19 (t, <i>J</i> = 7.6 Hz, 4H).</div></div><div id="sec5_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate 2-hydroxypropane-1,2,3-tricarboxylate (<b>67</b>)</h4><div class="NLM_p last">A solution of citric acid (230 mg, 1.2 mmol) in ethyl acetate and methanol was added to the solution of <b>65</b> (530 mg, 1 mmol) in ethyl acetate dropwise under stirring. The mixture was stirred at room temperature overnight. The mixture was filtered. The cake was washed with ethyl acetate and dried under a vacuum to provide the <b>67</b> as a yellow solid. Yield 91%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.99 (s, 2H), 9.43 (s, 1H), 9.12 (s, 1H), 8.71 (s, 1H), 7.60 (dd, <i>J</i> = 8.3, 4.1 Hz, 1H), 7.50 (dd, <i>J</i> = 6.8, 2.8 Hz, 1H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 4.82 (dt, <i>J</i> = 8.0, 4.4 Hz, 1H), 3.82 (tt, <i>J</i> = 6.9, 3.8 Hz, 1H), 3.46–3.31 (m, 4H), 3.13 (ddd, <i>J</i> = 12.3, 8.5, 3.6 Hz, 2H), 2.70 (d, <i>J</i> = 15.3 Hz, 2H), 2.60 (d, <i>J</i> = 15.3 Hz, 2H), 2.34 (s, 6H), 2.25 (s, 3H), 1.94–1.82 (m, 2H), 1.70–1.54 (m, 2H), 1.24–1.09 (m, 4H).</div></div><div id="sec5_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate methanesulfonate (<b>68</b>)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.84 (s, 1H), 9.45 (s, 1H), 9.12 (s, 1H), 8.73 (s, 1H), 7.68–7.24 (m, 3H), 7.11 (t, <i>J</i> = 9.2 Hz, 1H), 5.01–4.84 (m, 1H), 4.15 (d, <i>J</i> = 3.8 Hz, 2H), 3.83 (tt, <i>J</i> = 7.0, 4.0 Hz, 1H), 3.38 (q, <i>J</i> = 7.2, 6.8 Hz, 2H), 3.17 (t, <i>J</i> = 9.2 Hz, 2H), 2.80 (d, <i>J</i> = 3.3 Hz, 6H), 2.35 (s, 6H), 2.25 (s, 3H), 1.92 (t, <i>J</i> = 9.9 Hz, 2H), 1.69 (s, 2H), 1.27–1.13 (m, 4H).</div></div><div id="sec5_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate hydrochloride (<b>69</b>)</h4><div class="NLM_p last">The solution of <b>65</b> in ethyl acetate was bubbled with HCl gas, which is produced by H<sub>2</sub>SO<sub>4</sub> and NaCl, until a precipitate appeared. The mixture was filtered, and the cake was washed with ethyl acetate. The cake was dried under a vacuum to afford <b>69</b> as a wine red solid. Yield 49%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.55 (s, 1H), 9.59 (s, 1H), 9.12 (d, <i>J</i> = 3.3 Hz, 1H), 8.79 (d, <i>J</i> = 3.4 Hz, 1H), 7.69–7.58 (m, 1H), 7.58–7.49 (m, 1H), 7.11 (td, <i>J</i> = 9.4, 3.4 Hz, 1H), 4.93 (s, 1H), 4.15 (s, 2H), 3.90–3.77 (m, 1H), 3.39 (t, <i>J</i> = 9.6 Hz, 2H), 3.19 (t, <i>J</i> = 9.6 Hz, 2H), 2.79 (s, 6H), 2.25 (s, 3H), 1.93 (d, <i>J</i> = 12.8 Hz, 2H), 1.68 (d, <i>J</i> = 11.3 Hz, 2H), 1.27–1.13 (m, 4H).</div></div><div id="sec5_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 1-((1-Cyclopropyl-3-((4-fluoro-3-methylphenyl)amino)-1<i>H</i>-pyrazolo[3,4-<i>c</i>]pyridin-5-yl)sulfonyl)piperidin-4-yl dimethylglycinate maleate (<b>70</b>)</h4><div class="NLM_p last">A solution of citric acid (14 mg, 0.12 mmol) in ethyl acetate was added to the solution of <b>65</b> (53 mg, 0.1 mmol) in ethyl acetate dropwise under stirring. The mixture was stirred at room temperature overnight. The mixture was filtered. The cake was washed with ethyl acetate and dried under a vacuum to provide 46 mg of <b>70</b> as a yellow solid in 71% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 1H), 9.13 (s, 1H), 8.73 (s, 1H), 7.60 (dt, <i>J</i> = 8.4, 3.6 Hz, 1H), 7.51 (dd, <i>J</i> = 6.9, 2.9 Hz, 1H), 7.12 (t, <i>J</i> = 9.2 Hz, 1H), 6.06 (s, 2H), 4.92 (tt, <i>J</i> = 7.7, 3.8 Hz, 1H), 4.00 (s, 2H), 3.84 (tt, <i>J</i> = 6.9, 3.9 Hz, 1H), 3.64–3.23 (m, 2H), 3.17 (ddd, <i>J</i> = 12.3, 8.2, 3.8 Hz, 2H), 2.71 (s, 6H), 2.32–2.18 (m, 3H), 1.93 (ddt, <i>J</i> = 11.8, 7.6, 3.7 Hz, 2H), 1.67 (dtd, <i>J</i> = 12.3, 7.9, 3.8 Hz, 2H), 1.20 (t, <i>J</i> = 6.6 Hz, 4H).</div></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> Plasma Stability</h3><div class="NLM_p last">Two microliters of stock solution of compound (50 mM in DMSO) was diluted to 100 μL with the plasma, which was obtained from the mouse fundus vein after centrifugation at 13 000 rpm for 10 min. The mixture was incubated at 37 °C under shaking and quenched with 300 μL of acetonitrile at different time points. After centrifugation at 10 000 rpm for 10 min, the supernatant was detected in the HPLC system. The half-life time was determined on the basis of the peak area.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> Solubility</h3><div class="NLM_p last">About 1 mg of compound was added into 100 μL of medium. The mixture was kept at 25 °C under shaking overnight. After filtration with 0.22 μm microporous filer, the filtrate was detected in the HPLC. The amount of the dissolved compound was determined by the ratio of the peak area.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i107">Biological Method</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28278" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28278" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> Assay for Anti-HBV Activity</h3><div class="NLM_p last">HepAD38 cells were seeded into 96-well culture plates at approximately 1 × 10<sup>4</sup> cells/well in a volume of 100 μL and treated with compounds or DMSO for 6 days after cells adhered to the well. Medium containing compounds or DMSO were replaced every 3 days. After 6 days, 60 μL of cell supernatant was transferred to an eight strip PCR tube using a multichannel pipettor. The samples were heated for 15 min at 95 °C in a PCR amplifier and then centrifuged at 4000 rpm for 10 min. HBV DNA was detected using q-PCR.</div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> Quantitative PCR (q-PCR)</h3><div class="NLM_p last">Twenty microliter reaction systems contained 10 μL of TaqMan Gene Expression Master Mix (Applied Biosystems, 4369016), 2 μL of forward and reverse primers (10 μM), 0.25 μL of probe, 5.75 μL of dd H<sub>2</sub>O, and 2 μL of treated samples. PCR conditions: 95 °C for 10 min, 95 °C for 15 s, and 60 °C for 1 min for 40 cycles in a real-time 7500 machine. HBV forward primer, 5′-CCAAATGCCCCTATCCTATCA-3′; and HBV reverse primer, 5′-GAGGCGAGGGAGTTCTTCTTCTA-3′; HBV probe, 5′-CGGAAACTACTGTTGTTAGACGACGAGGCAG-3′. Pzac-1.2HBV plasmid was used as the standard substance to draw the standard curve, and 80 ng/mL plasmid corresponded to a 10<sup>13</sup> HBV DNA load. The standard curve contained eight points from 10<sup>11</sup> to 10<sup>4</sup> with a 10-fold dilution. For anti-HBV EC<sub>50</sub>, the data for each well was calculated using the following formula: 100% × (1 – 2 <sup>–(C<sub>T</sub> samples – C<sub>T</sub> control)</sup>), and analyzed using a 3- or 4-parameter curve fitting algorithm in GraphPad Prism.</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> Intracellular DNA and RNA Detection</h3><div class="NLM_p last">The comparative C<sub>T</sub> (ΔΔC<sub>T</sub>) quantitation method was used in quantitative PCR (qPCR) or quantitative reverse transcription (qRT-PCR) detection. For intracellular DNA, PCR reaction systems were the same as above. In addition, GAPDH was used as a housekeeping gene, and 20 μL reaction systems included 10 μL of TaqMan Gene Expression Master Mix (Applied Biosystems, 4369016), 1 μL of primer mix (ThermoFisher, Hs02758991_g1), 2 μL of DNA samples, and 7 μL of dd H<sub>2</sub>O. For intracellular RNA detection, 20 μL reaction systems consisted of 10 μL of Power SYBR Green PCR Master Mix (Applied Biosystems, 4367659), 2 μL of forward and reverse primers (10 μM), 2 μL of DNA samples, and 6 μL of ddH<sub>2</sub>O. GAPDH forward primer, 5′-ACCCACTCCTCCACCTTTG-3′; and reverse primer, 5′-CTGTAGCCAAATTCGTTGTCAT-3′. The data for each well was calculated using the following formula: 100% × 2<sup>–((C<sub>T</sub> sample – C<sub>T</sub> sample GAPDH) – (C<sub>T</sub> control – C<sub>T</sub> control GAPDH))</sup>, and was analyzed in GraphPad Prism.</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> SEC and EM Analysis</h3><div class="NLM_p last">A recombinant HBV core protein (Cp149) was used in SEC and EM studies. Cp149 proteins (5 μM) were incubated with 10 μM compounds in a buffer containing 150 mM NaCl and 50 mM HEPES (pH 7.4) at 37 °C overnight prior to EM and SEC analysis. Samples were adsorbed to freshly glow discharged carbon coated grids for 30 s. The water was removed by filter paper. The grids were stained in fresh 1% (wt/vol) uranyl acetate acid for 1 min. After absorbing excess water by filter paper and drying, the grids were examined with a transmission electron microscope. For SEC, samples were analyzed by an Äkta purifier equipped with a size exclusion column (GE Healthcare, Superdex 200 Increase 5/150 GL (micro)). The eluent was a pH 7.5 buffer containing 100 mM Tris, 100 mM NaCl, 17g/L sucrose. The capsids or Cps were detected using a UV detector at 280 nm, and the ratio of area under the curve (AUC) was calculated to assess capsid formation induced by compounds.</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> Enzyme-Linked Immunosorbent Assay</h3><div class="NLM_p last">HepAD38 or HepG2.2.15 cell culture supernatant was diluted 3-fold for HBeAg detection (Kehua, 20123400740) and 5-fold for HBsAg detection (Kehua, S10910113) according to manufacturer’s instructions. Enzyme-linked immunosorbent assay (ELISA) results are presented as 100% × S/VC (S = sample OD value and VC = vehicle control OD value).</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> <i>In Vivo</i> Efficacy in the Hydrodynamic-Injection (HDI) Mouse Model</h3><div class="NLM_p">Male C57BL/6 mice (6–8 weeks old) were purchased from Vital River Experimental Animal Co., Ltd. (Beijing, China). All mice were maintained under specific pathogen-free conditions in the animal research center laboratory of Tsinghua University. All animal protocols were approved by the Institute of Animal Care and Use Committee (IACUC) of Tsinghua University, and studies were conducted in compliance with institutional and national guidelines.</div><div class="NLM_p last">All the mice were weighed and hydrodynamically injected with 8 μg of pAAV2-HBV1.3HBV plasmid (genotype D) in a volume of phosphate buffer solution (PBS) to 10% of a mouse’s body weight (e.g., 2.0 mL for mouse of 20 g) within 5–7 s through a tail vein. From days 1 to 7, the mice in all the groups were orally dosed with the blank vehicle, 0.05 mg/kg entecavir, 100 mg/kg AB-423, 50 and 100 mg/kg compound <b>56</b>, and 50 and 100 mg/kg compound <b>67</b> at the indicated frequency. Serum and liver samples were collected at the indicated time points for HBV-DNA quantification by real-time PCR.</div></div><div id="sec6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> Pharmacokinetics in Mouse</h3><div class="NLM_p last">Male C57BL/6 mice (6–8 weeks old) (purchased from Vital River Experimental Animal Co., Ltd. (Beijing, China)) were used for the pharmacokinetic study. Mice (<i>N</i> = 3) were administered with <b>56</b> (iv 20 mg/kg and po 100 mg/kg) and <b>67</b> (po 100 and 200 mg/kg). Compound <b>56</b> for iv administration was dissolved in normal saline containing 5% (v/v) DMSO and 5% (v/v) Tween 80. Compound <b>56</b> for po administration was dissolved in normal saline containing 5% (v/v) DMSO and 0.5% CMC. Compound <b>67</b> was dissolved in normal saline containing 5% (v/v) DMSO. Blood samples were collected from the saphenous vein at 5 min, 10 min, 20 min, 30 min, 1 h, 2 h, 4 h, 8 h, 12 h, and 24 h into heparin-containing microcentrifuge tubes. The blood samples were immediately centrifuged at 4 °C, and the plasma was stored at −80 °C. Liver was collected and stored at −80 °C. The compound concentration was determined by HPLC using an internal standard method.</div></div><div id="sec6_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i115" class="anchor-spacer"></div><h3 class="article-section__title" id="_i115"> Molecule Modeling</h3><div class="NLM_p last">Molecule modeling of compound <b>56</b> and HBV core protein crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>) was performed with Schrödinger (2017-3) software.</div></div><div id="sec6_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> Interference Compounds Examination</h3><div class="NLM_p last">All these active compounds were screened with the publicly available filters (<a href="http://www.cbligand.org/PAINS/" class="extLink">http://www.cbligand.org/PAINS/</a>, <a href="http://advisor.docking.org" class="extLink">http://advisor.docking.org</a>) and did not preserve the potential to be PAINS or Aggregators.<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i117"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00292" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05164" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05164" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00292?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00292</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tables of anti-HBV activity and cytotoxicity of the compounds and purity of tested compounds, figures of <i>in vivo</i> anti-HBV efficiency, intracellular RNA level, <sup>1</sup>H and <sup>13</sup>C NMR spectra, and HPLC spectra, and discussions of procedures of synthesis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf">jm0c00292_si_001.pdf (9.27 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00292" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61010" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61010" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ye Xiang</span> - <span class="hlFld-Affiliation affiliation">Beijing
Advanced Innovation Center for Structural Biology, Beijing Frontier
Research Center for Biological Structure, Center for Global Health
and Infectious Diseases, Department of Basic Medical Sciences, School
of Medicine, Tsinghua University, Beijing 100084, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ec9594858d828bac818d8580c2989f85828b84998dc2898899c28f82"><span class="__cf_email__" data-cfemail="572e2f3e363930173a363e3b7923243e39303f223679323322793439">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yao Ma</span> - <span class="hlFld-Affiliation affiliation">Institute
of Materia Medica, Chinese Academy of Medical
Sciences & Peking Union Medical College, Beijing 100050, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0865697169674861656526696b266b66"><span class="__cf_email__" data-cfemail="375a564e5658775e5a5a195654195459">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gang Liu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Tsinghua University, Beijing 100084, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5549-5686" title="Orcid link">http://orcid.org/0000-0001-5549-5686</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5433353a33383d2166631420273d3a333c21357a3130217a373a"><span class="__cf_email__" data-cfemail="56313738313a3f2364611622253f38313e233778333223783538">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunting Wang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yameng Pei</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Sciences, Tsinghua University, Beijing 100084, China</span>; 
    <span class="hlFld-Affiliation affiliation">Tsinghua-Peking
Center for Life Sciences, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lin Wang</span> - <span class="hlFld-Affiliation affiliation">Beijing
Advanced Innovation Center for Structural Biology, Collaborative Innovation
Center for Diagnosis and Treatment of Infectious Diseases, Center
for Global Health and Infectious Diseases, Department of Basic Medical
Sciences, School of Medicine, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuo Li</span> - <span class="hlFld-Affiliation affiliation">Beijing
Advanced Innovation Center for Structural Biology, Beijing Frontier
Research Center for Biological Structure, School of Pharmaceutical
Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao Jiang</span> - <span class="hlFld-Affiliation affiliation">Beijing
Advanced Innovation Center for Structural Biology, Beijing Frontier
Research Center for Biological Structure, School of Pharmaceutical
Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xu Tan</span> - <span class="hlFld-Affiliation affiliation">Beijing
Advanced Innovation Center for Structural Biology, Beijing Frontier
Research Center for Biological Structure, School of Pharmaceutical
Sciences, Center for Infectious Disease Research, School of Medicine, Tsinghua University, Beijing 100084, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi Dong</span> - <span class="hlFld-Affiliation affiliation">Institute
of Materia Medica, Chinese Academy of Medical
Sciences & Peking Union Medical College, Beijing 100050, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7844-7206" title="Orcid link">http://orcid.org/0000-0001-7844-7206</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.W. and Y.P. contributed equally to this work. G.L., Y.M., and Y.X. designed this study. C.W. performed the synthesis of the compounds, evaluation of the solubility, and determination of the stability in plasma and pharmacokinetics. Y.P. performed the biological test. L.W. performed the EM and SEC analyses. C.J. and X.T. provided pAAV2-HBV1.3HBV plasmid. S.L performed the HBV infection in Huh7-NTCP cells. Y.X. designed and guided the EM and SEC analyses. G.L., C.W., and Y.P. wrote the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e9554-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i119">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03233" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03233" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was financially supported by the National Natural Science Foundation of China (81803358), the Beijing Municipal Natural Science Foundation (7202095) of China, and Postdoctoral Foundation of Tsinghua-Peking Center for Life Sciences. The authors would like to thank the Chinese PLA General Hospital for providing HepG2.A64 and HepG2.1403F cells.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">3TC</td><td class="NLM_def"><p class="first last">Lamivudine</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism and excretion</p></td></tr><tr><td class="NLM_term">CAMs</td><td class="NLM_def"><p class="first last">capsid assembly modulators</p></td></tr><tr><td class="NLM_term">cccDNA</td><td class="NLM_def"><p class="first last">covalently closed circular DNA</p></td></tr><tr><td class="NLM_term">CHB</td><td class="NLM_def"><p class="first last">chronic hepatitis B</p></td></tr><tr><td class="NLM_term">Cp</td><td class="NLM_def"><p class="first last">core protein</p></td></tr><tr><td class="NLM_term">DCC</td><td class="NLM_def"><p class="first last">dicyclohexylcarbodiimide</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">EM</td><td class="NLM_def"><p class="first last">electron microscopy</p></td></tr><tr><td class="NLM_term">HAP</td><td class="NLM_def"><p class="first last">heteroarylpyrimidines</p></td></tr><tr><td class="NLM_term">HBeAg</td><td class="NLM_def"><p class="first last">HBV e antigen</p></td></tr><tr><td class="NLM_term">HBsAg</td><td class="NLM_def"><p class="first last">HBV surface antigen</p></td></tr><tr><td class="NLM_term">HBV</td><td class="NLM_def"><p class="first last">Hepatitis B virus</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high resolution mass spectrum</p></td></tr><tr><td class="NLM_term">Hz</td><td class="NLM_def"><p class="first last">hertz</p></td></tr><tr><td class="NLM_term">IFN</td><td class="NLM_def"><p class="first last">interferon</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsome</p></td></tr><tr><td class="NLM_term">NAs</td><td class="NLM_def"><p class="first last">nucleos(t)ide analogues</p></td></tr><tr><td class="NLM_term">NBS</td><td class="NLM_def"><p class="first last"><i>N</i>-bromosuccinimide</p></td></tr><tr><td class="NLM_term">PPA</td><td class="NLM_def"><p class="first last">phenylpropenamides</p></td></tr><tr><td class="NLM_term">ppm</td><td class="NLM_def"><p class="first last">parts per million</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">reverse transcription</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure activity relationship</p></td></tr><tr><td class="NLM_term">SBA</td><td class="NLM_def"><p class="first last">sulfamoylbenzamide</p></td></tr><tr><td class="NLM_term">SEC</td><td class="NLM_def"><p class="first last">size exclusion chromatography</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i121">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03723" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03723" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span> </span><span class="NLM_article-title">Hepatitis
B</span>. <a href="https://www.who.int/news-room/fact-sheets/detail/hepatitis-b" class="extLink">https://www.who.int/news-room/fact-sheets/detail/hepatitis-b</a>, <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Hepatitis%0AB.+https%3A%2F%2Fwww.who.int%2Fnews-room%2Ffact-sheets%2Fdetail%2Fhepatitis-b%2C+World+Health+Organization%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DHepatitis%250AB%26pub%3DWorld%2520Health%2520Organization%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seto, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlotsky, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, M. F.</span></span> <span> </span><span class="NLM_article-title">Chronic hepatitis B virus infection</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>392</i></span>,  <span class="NLM_fpage">2313</span>– <span class="NLM_lpage">2324</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(18)31865-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS0140-6736%2818%2931865-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=30496122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A280%3ADC%252BB3crmslGhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2018&pages=2313-2324&author=W.+K.+Setoauthor=Y.+R.+Loauthor=J.+M.+Pawlotskyauthor=M.+F.+Yuen&title=Chronic+hepatitis+B+virus+infection&doi=10.1016%2FS0140-6736%2818%2931865-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chronic hepatitis B virus infection</span></div><div class="casAuthors">Seto Wai-Kay; Lo Ying-Ru; Pawlotsky Jean-Michel; Yuen Man-Fung</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">392</span>
        (<span class="NLM_cas:issue">10161</span>),
    <span class="NLM_cas:pages">2313-2324</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chronic hepatitis B virus infection is a global public health threat that causes considerable liver-related morbidity and mortality.  It is acquired at birth or later via person-to-person transmission.  Vaccination effectively prevents infection and chronic hepatitis B virus carriage.  In chronically infected patients, an elevated serum hepatitis B virus DNA concentration is the main risk factor for disease progression, although there are other clinical and viral parameters that influence disease outcomes.  In addition to liver biochemistry, virological markers, and abdominal ultrasonography, non-invasive assessment of liver fibrosis is emerging as an important assessment modality.  Long-term nucleos(t)ide-analogue therapy is safe and well tolerated, achieves potent viral suppression, and reduces the incidence of liver-related complications.  However, a need to optimise management remains.  Promising novel therapies are at the developmental stage.  With current vaccines, therapies, and an emphasis on improving linkage to care, WHO's goal of eliminating hepatitis B virus as a global health threat by 2030 is achievable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCvI0MsROihkewzGPswiWOfW6udTcc2eaM5mVC1AEelbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3crmslGhug%253D%253D&md5=e252e1c05793f99a1328c9d969bc5ba3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2818%2931865-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252818%252931865-8%26sid%3Dliteratum%253Aachs%26aulast%3DSeto%26aufirst%3DW.%2BK.%26aulast%3DLo%26aufirst%3DY.%2BR.%26aulast%3DPawlotsky%26aufirst%3DJ.%2BM.%26aulast%3DYuen%26aufirst%3DM.%2BF.%26atitle%3DChronic%2520hepatitis%2520B%2520virus%2520infection%26jtitle%3DLancet%26date%3D2018%26volume%3D392%26spage%3D2313%26epage%3D2324%26doi%3D10.1016%2FS0140-6736%2818%2931865-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoulim, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span> <span> </span><span class="NLM_article-title">Antiviral therapies and prospects for a cure of chronic hepatitis B</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a021501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1101%2Fcshperspect.a021501" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=21&author=F.+Zoulimauthor=D.+Durantel&title=Antiviral+therapies+and+prospects+for+a+cure+of+chronic+hepatitis+B&doi=10.1101%2Fcshperspect.a021501"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a021501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a021501%26sid%3Dliteratum%253Aachs%26aulast%3DZoulim%26aufirst%3DF.%26aulast%3DDurantel%26aufirst%3DD.%26atitle%3DAntiviral%2520therapies%2520and%2520prospects%2520for%2520a%2520cure%2520of%2520chronic%2520hepatitis%2520B%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2015%26volume%3D5%26spage%3D21%26doi%3D10.1101%2Fcshperspect.a021501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tuttleman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pourcel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summers, J.</span></span> <span> </span><span class="NLM_article-title">Formation of the pool of covalently closed circular viral-DNA in hepadnavirus-infected cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(86)90602-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2F0092-8674%2886%2990602-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=3768961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaL2sXhtlynug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1986&pages=451-460&author=J.+S.+Tuttlemanauthor=C.+Pourcelauthor=J.+Summers&title=Formation+of+the+pool+of+covalently+closed+circular+viral-DNA+in+hepadnavirus-infected+cells&doi=10.1016%2F0092-8674%2886%2990602-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of the pool of covalently closed circular viral DNA in hepadnavirus-infected cells</span></div><div class="casAuthors">Tuttleman, Jan S.; Pourcel, Christine; Summers, Jesse</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">451-60</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Covalently closed circular (CCC) double-stranded DNA believed to be the transcriptional template for duck hepatitis B virus (DHBV) is amplified in aging primary cultures of hepatocytes from congenitally infected ducklings.  Anal. of 5-bromodeoxyuridine-labeled heavy/light CCC DNA shows that the relaxed circular DNA synthesized in the cytoplasm by reverse transcription is the predominant precursor to the amplified pool of nuclear viral CCC DNA.  In vitro infection of uninfected hepatocyte cultures with DHBV demonstrates that a similar 50-fold amplification of CCC DNA occurs during an early stage in the infection before virus prodn.  This amplification allows the establishment of a pool of transcriptional templates in the cell without the need for semiconservative replication or multiple rounds of infection.  This process may account for the ability of hepadnavirus-infected cells persistently to produce virus particles in the absence of stable integration of viral DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeIiXXyDXnM7Vg90H21EOLACvtfcHk0liMAkzIRe50Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhtlynug%253D%253D&md5=4401425e1e3253a086d6c5c2366d0f64</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2886%2990602-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252886%252990602-1%26sid%3Dliteratum%253Aachs%26aulast%3DTuttleman%26aufirst%3DJ.%2BS.%26aulast%3DPourcel%26aufirst%3DC.%26aulast%3DSummers%26aufirst%3DJ.%26atitle%3DFormation%2520of%2520the%2520pool%2520of%2520covalently%2520closed%2520circular%2520viral-DNA%2520in%2520hepadnavirus-infected%2520cells%26jtitle%3DCell%26date%3D1986%26volume%3D47%26spage%3D451%26epage%3D460%26doi%3D10.1016%2F0092-8674%2886%2990602-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Werle-Lapostolle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locarnini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wursthorn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trepo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney IV, W. E.</span></span> <span> </span><span class="NLM_article-title">Persistence of cccDNA during the natural history of chronic hepatitis B and decline during adefovir dipivoxil therapy</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1758</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2004.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1053%2Fj.gastro.2004.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=15188170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsFaqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=1750-1758&author=B.+Werle-Lapostolleauthor=S.+Bowdenauthor=S.+Locarniniauthor=K.+Wursthornauthor=J.+Petersenauthor=G.+Lauauthor=C.+Trepoauthor=P.+Marcellinauthor=Z.+Goodmanauthor=W.+E.+Delaney+IV&title=Persistence+of+cccDNA+during+the+natural+history+of+chronic+hepatitis+B+and+decline+during+adefovir+dipivoxil+therapy&doi=10.1053%2Fj.gastro.2004.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Persistence of cccDNA during the natural history of chronic hepatitis B and decline during adefovir dipivoxil therapy</span></div><div class="casAuthors">Werle-Lapostolle, Bettina; Bowden, Scott; Locarnini, Stephen; Wursthorn, Karsten; Petersen, Jorg; Lau, George; Trepo, Christian; Marcellin, Patrick; Goodman, Zachary; Delaney, William E., IV; Xiong, Shelly; Brosgart, Carol L.; Chen, Shan-Shan; Gibbs, Craig S.; Zoulim, Fabien</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1758</span>CODEN:
                <span class="NLM_cas:coden">GASTAB</span>;
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Background & Aims: Hepatitis B virus (HBV) covalently closed circular DNA (cccDNA) is a unique episomal replicative intermediate responsible for persistent infection of hepatocytes.  Tech. constraints have hampered the direct study of cccDNA maintenance and clearance mechanisms in patients.  The aim of this study was to develop a sensitive and specific assay for quantifying cccDNA in biopsy samples from chronic hepatitis B patients during different natural history phases and in patients undergoing antiviral therapy.  Methods: Intrahepatic cccDNA levels were quantified by a specific real-time PCR assay.  Ninety-eight liver biopsy samples from patients in the major phases of the natural history of chronic hepatitis B and 32 pairs of samples from patients receiving adefovir dipivoxil (ADV) therapy were assessed.  Results: cccDNA was detected, at levels ranging over 3 orders of magnitude, in patients in different phases of the natural history of chronic hepatitis B.  CccDNA levels were strongly correlated with levels of total intracellular HBV DNA and serum HBV DNA.  Forty-eight weeks of ADV therapy resulted in a significant 0.8 log decrease in cccDNA copies/cell.  Changes in cccDNA were correlated with a similar redn. in serum HBsAg titer but not with a decrease in the no. of HBV antigen-pos. cells during ADV treatment.  Conclusions: cccDNA persists throughout the natural history of chronic hepatitis B, even in patients with serol. evidence of viral clearance.  Long-term ADV therapy significantly decreased cccDNA levels by a primarily noncytolytic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4L4TAyOGVxrVg90H21EOLACvtfcHk0liMAkzIRe50Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsFaqs78%253D&md5=3a390a5d3f59436b73007ba73caac1a9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2004.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2004.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DWerle-Lapostolle%26aufirst%3DB.%26aulast%3DBowden%26aufirst%3DS.%26aulast%3DLocarnini%26aufirst%3DS.%26aulast%3DWursthorn%26aufirst%3DK.%26aulast%3DPetersen%26aufirst%3DJ.%26aulast%3DLau%26aufirst%3DG.%26aulast%3DTrepo%26aufirst%3DC.%26aulast%3DMarcellin%26aufirst%3DP.%26aulast%3DGoodman%26aufirst%3DZ.%26aulast%3DDelaney%2BIV%26aufirst%3DW.%2BE.%26atitle%3DPersistence%2520of%2520cccDNA%2520during%2520the%2520natural%2520history%2520of%2520chronic%2520hepatitis%2520B%2520and%2520decline%2520during%2520adefovir%2520dipivoxil%2520therapy%26jtitle%3DGastroenterology%26date%3D2004%26volume%3D126%26spage%3D1750%26epage%3D1758%26doi%3D10.1053%2Fj.gastro.2004.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucifora, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprinzl, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppensteiner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makowska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remouchamps, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thasler, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browning, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dejardin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heikenwalder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protzer, U.</span></span> <span> </span><span class="NLM_article-title">Specific and nonhepatotoxic degradation of nuclear hepatitis B virus cccDNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1228</span>, <span class="refDoi"> DOI: 10.1126/science.1243462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1126%2Fscience.1243462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=24557838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjvVygsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=1221-1228&author=J.+Luciforaauthor=Y.+C.+Xiaauthor=F.+Reisingerauthor=K.+Zhangauthor=D.+Stadlerauthor=X.+M.+Chengauthor=M.+F.+Sprinzlauthor=H.+Koppensteinerauthor=Z.+Makowskaauthor=T.+Volzauthor=C.+Remouchampsauthor=W.+M.+Chouauthor=W.+E.+Thaslerauthor=N.+Huserauthor=D.+Durantelauthor=T.+J.+Liangauthor=C.+Munkauthor=M.+H.+Heimauthor=J.+L.+Browningauthor=E.+Dejardinauthor=M.+Dandriauthor=M.+Schindlerauthor=M.+Heikenwalderauthor=U.+Protzer&title=Specific+and+nonhepatotoxic+degradation+of+nuclear+hepatitis+B+virus+cccDNA&doi=10.1126%2Fscience.1243462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Specific and Nonhepatotoxic Degradation of Nuclear Hepatitis B Virus cccDNA</span></div><div class="casAuthors">Lucifora, Julie; Xia, Yuchen; Reisinger, Florian; Zhang, Ke; Stadler, Daniela; Cheng, Xiaoming; Sprinzl, Martin F.; Koppensteiner, Herwig; Makowska, Zuzanna; Volz, Tassilo; Remouchamps, Caroline; Chou, Wen-Min; Thasler, Wolfgang E.; Hueser, Norbert; Durantel, David; Liang, T. Jake; Muenk, Carsten; Heim, Markus H.; Browning, Jeffrey L.; Dejardin, Emmanuel; Dandri, Maura; Schindler, Michael; Heikenwalder, Mathias; Protzer, Ulrike</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6176</span>),
    <span class="NLM_cas:pages">1221-1228</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Current antiviral agents can control but not eliminate hepatitis B virus (HBV), because HBV establishes a stable nuclear covalently closed circular DNA (cccDNA).  Interferon-α treatment can clear HBV but is limited by systemic side effects.  The authors describe how interferon-α can induce specific degrdn. of the nuclear viral DNA without hepatotoxicity and propose lymphotoxin-β receptor activation as a therapeutic alternative.  Interferon-α and lymphotoxin-β receptor activation up-regulated APOBEC3A and APOBEC3B cytidine deaminases, resp., in HBV-infected cells, primary hepatocytes, and human liver needle biopsies.  HBV core protein mediated the interaction with nuclear cccDNA, resulting in cytidine deamination, apurinic/apyrimidinic site formation, and finally cccDNA degrdn. that prevented HBV reactivation.  Genomic DNA was not affected.  Thus, inducing nuclear deaminases-for example, by lymphotoxin-β receptor activation-allows the development of new therapeutics that, in combination with existing antivirals, may cure hepatitis B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrONjt8Ppl67rVg90H21EOLACvtfcHk0limkbXHmc2QTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjvVygsrY%253D&md5=18855f5a52c98d6167a7a8468d55e9ce</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.1243462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1243462%26sid%3Dliteratum%253Aachs%26aulast%3DLucifora%26aufirst%3DJ.%26aulast%3DXia%26aufirst%3DY.%2BC.%26aulast%3DReisinger%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DStadler%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DX.%2BM.%26aulast%3DSprinzl%26aufirst%3DM.%2BF.%26aulast%3DKoppensteiner%26aufirst%3DH.%26aulast%3DMakowska%26aufirst%3DZ.%26aulast%3DVolz%26aufirst%3DT.%26aulast%3DRemouchamps%26aufirst%3DC.%26aulast%3DChou%26aufirst%3DW.%2BM.%26aulast%3DThasler%26aufirst%3DW.%2BE.%26aulast%3DHuser%26aufirst%3DN.%26aulast%3DDurantel%26aufirst%3DD.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DMunk%26aufirst%3DC.%26aulast%3DHeim%26aufirst%3DM.%2BH.%26aulast%3DBrowning%26aufirst%3DJ.%2BL.%26aulast%3DDejardin%26aufirst%3DE.%26aulast%3DDandri%26aufirst%3DM.%26aulast%3DSchindler%26aufirst%3DM.%26aulast%3DHeikenwalder%26aufirst%3DM.%26aulast%3DProtzer%26aufirst%3DU.%26atitle%3DSpecific%2520and%2520nonhepatotoxic%2520degradation%2520of%2520nuclear%2520hepatitis%2520B%2520virus%2520cccDNA%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D1221%26epage%3D1228%26doi%3D10.1126%2Fscience.1243462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, G. K.K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piratvisuth, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongsawat, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooksley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gane, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fried, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flisiak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCloud, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pluck, N.</span></span> <span> </span><span class="NLM_article-title">Peginterferon alfa-2a, lamivudine, and the combination for HBeAg-positive chronic hepatitis B</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">2682</span>– <span class="NLM_lpage">2695</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa043470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1056%2FNEJMoa043470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=15987917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1SktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=2682-2695&author=G.+K.K.+Lauauthor=T.+Piratvisuthauthor=K.+X.+Luoauthor=P.+Marcellinauthor=S.+Thongsawatauthor=G.+Cooksleyauthor=E.+Ganeauthor=M.+W.+Friedauthor=W.+C.+Chowauthor=S.+W.+Paikauthor=W.+Y.+Changauthor=T.+Bergauthor=R.+Flisiakauthor=P.+McCloudauthor=N.+Pluck&title=Peginterferon+alfa-2a%2C+lamivudine%2C+and+the+combination+for+HBeAg-positive+chronic+hepatitis+B&doi=10.1056%2FNEJMoa043470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Peginterferon alfa-2a, lamivudine, and the combination for HBeAg-positive chronic hepatitis B</span></div><div class="casAuthors">Lau, George K. K.; Piratvisuth, Teerha; Luo, Kang Xian; Marcellin, Patrick; Thongswat, Satawat; Cooksley, Graham; Gane, Edward; Fried, Michael W.; Chow, Wan Cheng; Paik, Sueng Woon; Chang, Wen Yu; Berg, Thomas; Flisiak, Robert; McCloud, Philip; Pluck, Nigel; Balan, V.; Baruch, Y.; Boyer, N.; Box, T.; Burak, K.; Chao, Y.-C.; Cheinquer, H.; Chung, K.-W.; Chung, Y.-H.; Chutaputti, A.; Fawaz, K.; Feinman, V.; Girgrah, N.; Gish, R.; Gitlin, N.; Goeser, T.; Goncales, F., Jr.; Guan, R.; Haeussinger, D.; Halota, W.; Han, K,-H.; Heim, M.; Horban, A.; Jia, J.-D.; Jin, R.; Jung, M.-C.; Lai, M.-Y.; Lee, A.; Lee, S.-D.; Lei, B.-J.; Liaw, Y.-F.; Lok, A.; Lu, Z.-M.; Luengrojanakul, P.; Lurie, Y.; Mahachai, V.; Manns, M.; Martin, P.; Parana, R.; Pawlowska, M.; Schmidt, W.; Sette, H., Jr.; Smith, C.; Trepo, C.; Tsai, N.; Tung, B.; Tur-Kaspa, R.; Wan, M.-B.; Wang, Q.-H.; Xu, D.-Z.; Yao, G.-B.; Yao, J.-L.; Yin, Y.-K.; Yu, Y.; Zhang, H.-F.; Zhao, Y.-R.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">2682-2695</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Current treatments for chronic hepatitis B are suboptimal.  In the search for improved therapies, we compared the efficacy and safety of pegylated interferon alfa plus lamivudine, pegylated interferon alfa without lamivudine, and lamivudine alone for the treatment of hepatitis B e antigen (HBeAg)-pos. chronic hepatitis B.  A total of 814 patients with HBeAg-pos. chronic hepatitis B received either peginterferon alfa-2a (180 μg once weekly) plus oral placebo, peginterferon alfa-2a plus lamivudine (100 mg daily), or lamivudine alone.  The majority of patients in the study were Asian (87 %).  Most patients were infected with hepatitis B virus (HBV) genotype B or C.  Patients were treated for 48 wk and followed for an addnl. 24 wk.  After 24 wk of follow-up, significantly more patients who received peginterferon alfa-2a monotherapy or peginterferon alfa-2a plus lamivudine than those who received lamivudine monotherapy had HBeAg seroconversion (32 % vs. 19 % [P < 0.001] and 27 % vs. 19 % [P = 0.02], resp.) or HBV DNA levels below 100,000 copies per mL (32 % vs. 22 % [P = 0.01] and 34 % vs. 22 % [P = 0.003], resp.).  Sixteen patients receiving peginterferon alfa-2a (alone or in combination) had hepatitis B surface antigen (HBsAg) seroconversion, as compared with 0 in the group receiving lamivudine alone (P = 0.001).  The most common adverse events were those known to occur with therapies based on interferon alfa.  Serious adverse events occurred in 4 %, 6 %, and 2 % of patients receiving peginterferon alfa-2a monotherapy, combination therapy, and lamivudine monotherapy, resp.  Two patients receiving lamivudine monotherapy had irreversible liver failure after the cessation of treatment - one underwent liver transplantation, and the other died.  In patients with HBeAg-pos. chronic hepatitis B, peginterferon alfa-2a offers superior efficacy over lamivudine, on the basis of HBeAg seroconversion, HBV DNA suppression, and HBsAg seroconversion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosvuGLPwiN87Vg90H21EOLACvtfcHk0lhZjYVvCrocfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1SktL8%253D&md5=aac0f7d1a47cb9d5cc2e81060d8250cf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa043470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa043470%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DG.%2BK.K.%26aulast%3DPiratvisuth%26aufirst%3DT.%26aulast%3DLuo%26aufirst%3DK.%2BX.%26aulast%3DMarcellin%26aufirst%3DP.%26aulast%3DThongsawat%26aufirst%3DS.%26aulast%3DCooksley%26aufirst%3DG.%26aulast%3DGane%26aufirst%3DE.%26aulast%3DFried%26aufirst%3DM.%2BW.%26aulast%3DChow%26aufirst%3DW.%2BC.%26aulast%3DPaik%26aufirst%3DS.%2BW.%26aulast%3DChang%26aufirst%3DW.%2BY.%26aulast%3DBerg%26aufirst%3DT.%26aulast%3DFlisiak%26aufirst%3DR.%26aulast%3DMcCloud%26aufirst%3DP.%26aulast%3DPluck%26aufirst%3DN.%26atitle%3DPeginterferon%2520alfa-2a%252C%2520lamivudine%252C%2520and%2520the%2520combination%2520for%2520HBeAg-positive%2520chronic%2520hepatitis%2520B%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D352%26spage%3D2682%26epage%3D2695%26doi%3D10.1056%2FNEJMoa043470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span> <span> </span><span class="NLM_article-title">Treatment of chronic hepatitis B virus infection using small molecule modulators of nucleocapsid assembly: recent advances and perspectives</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">724</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00337</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00337" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1akt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=713-724&author=L.+Yangauthor=F.+Liuauthor=X.+Tongauthor=D.+Hoffmannauthor=J.+Zuoauthor=M.+Lu&title=Treatment+of+chronic+hepatitis+B+virus+infection+using+small+molecule+modulators+of+nucleocapsid+assembly%3A+recent+advances+and+perspectives&doi=10.1021%2Facsinfecdis.8b00337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of Chronic Hepatitis B Virus Infection Using Small Molecule Modulators of Nucleocapsid Assembly: Recent Advances and Perspectives</span></div><div class="casAuthors">Yang, Li; Liu, Feifei; Tong, Xiankun; Hoffmann, Daniel; Zuo, Jianping; Lu, Mengji</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">713-724</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  On the basis of the recent advance of basic research on mol. biol. of hepatitis B virus (HBV) infection, novel antiviral drugs targeting various steps of the HBV life cycle have been developed in recent years.  HBV nucleocapsid assembly is now recognized as a hot target for anti-HBV drug development.  Structural and functional anal. of HBV nucleocapsid allowed rational design and improvement of small mols. with the ability to interact with the components of HBV nucleocapsid and modulate the viral nucleocapsid assembly process.  Prototypes of small mol. modulators targeting HBV nucleocapsid assembly are being preclinically tested or have moved forward in clin. trials, with promising results.  This Review summarizes the recent advances in the approach to develop antiviral drugs based on the modulation of HBV nucleocapsid assembly.  The antiviral mechanisms of small mol. modulators beyond the capsid formation and the potential implications will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvT6WVN92udLVg90H21EOLACvtfcHk0lhZjYVvCrocfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1akt7o%253D&md5=f602f5c7f6e9af69ac2f777ca82efe28</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00337%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DTong%26aufirst%3DX.%26aulast%3DHoffmann%26aufirst%3DD.%26aulast%3DZuo%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DM.%26atitle%3DTreatment%2520of%2520chronic%2520hepatitis%2520B%2520virus%2520infection%2520using%2520small%2520molecule%2520modulators%2520of%2520nucleocapsid%2520assembly%253A%2520recent%2520advances%2520and%2520perspectives%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D713%26epage%3D724%26doi%3D10.1021%2Facsinfecdis.8b00337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowther, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of the human hepatitis B virus capsid</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)80009-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS1097-2765%2801%2980009-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10394365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaK1MXktlOktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=771-780&author=S.+A.+Wynneauthor=R.+A.+Crowtherauthor=A.+G.+W.+Leslie&title=The+crystal+structure+of+the+human+hepatitis+B+virus+capsid&doi=10.1016%2FS1097-2765%2801%2980009-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the human hepatitis B virus capsid</span></div><div class="casAuthors">Wynne, S. A.; Crowther, R. A.; Leslie, A. G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">771-780</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Hepatitis B is a small enveloped DNA virus that poses a major hazard to human health.  The crystal structure of the T = 4 capsid has been solved at 3.3 A resoln., revealing a largely helical protein fold that is unusual for icosahedral viruses.  The monomer fold is stabilized by a hydrophobic core that is highly conserved among human viral variants.  Assocn. of two amphipathic α-helical hairpins results in formation of a dimer with a four-helix bundle as the major central feature.  The capsid is assembled from dimers via interactions involving a highly conserved region near the C terminus of the truncated protein used for crystn.  The major immunodominant region lies at the tips of the α-helical hairpins that form spikes on the capsid surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhKm_c-2LrbrVg90H21EOLACvtfcHk0lhOFeokkFdEFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktlOktbs%253D&md5=af1db5a088f6df2a5c5284a61b75aff6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2980009-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252980009-5%26sid%3Dliteratum%253Aachs%26aulast%3DWynne%26aufirst%3DS.%2BA.%26aulast%3DCrowther%26aufirst%3DR.%2BA.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DThe%2520crystal%2520structure%2520of%2520the%2520human%2520hepatitis%2520B%2520virus%2520capsid%26jtitle%3DMol.%2520Cell%26date%3D1999%26volume%3D3%26spage%3D771%26epage%3D780%26doi%3D10.1016%2FS1097-2765%2801%2980009-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">To build a virus capsid-an equilibrium-model of the self-assembly of polyhedral protein complex</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>241</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1994.1473</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1006%2Fjmbi.1994.1473" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=8051707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFeiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=241&publication_year=1994&pages=59-67&author=A.+Zlotnick&title=To+build+a+virus+capsid-an+equilibrium-model+of+the+self-assembly+of+polyhedral+protein+complex&doi=10.1006%2Fjmbi.1994.1473"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">To build a virus capsid. An equilibrium model of the self assembly of polyhedral protein complexes</span></div><div class="casAuthors">Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">241</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-67</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The capsids of spherical (icosahedral) viruses are constructed of multiples of 60 subunits.  The question of how these polymers assemble is basic to understanding the viral life cycle.  A formalism describing virus assembly as an equil. between coat protein subunits, assembly intermediates and intact virus is presented.  This equil. model of virus assembly is consistent with exptl. observations of virus assembly.  At equil., either intact virus or free subunits are dominant species, assembly intermediates are predicted to be found only in trace concns.  The concn. of assembled virus at equil. is expected to be extremely concn.-dependent and resemble a highly cooperative reaction although the model does not explicitly include cooperativity.  For statistical assembly of a polyhedron, a nucleus is not necessarily required and polymn. can proceed through a cascade of biomol. reactions rather than a single higher order reaction.  Thus, kinetics of assembly do not necessarily show the extreme concn. dependence typical of nucleated protein polymn.  Modest intersubunit interaction energies result in a very stable capsid; consequently, a small change in this interaction energy can result in a considerable change in the capsid-subunit equil.  Some possible effects of nucleation and protein-nucleic acid interactions on virus assembly and capsid morphol. are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1NPGa8HJx77Vg90H21EOLACvtfcHk0lhOFeokkFdEFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFeiu7w%253D&md5=0e9870afe01e712992b591285e6693b1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1994.1473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1994.1473%26sid%3Dliteratum%253Aachs%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DTo%2520build%2520a%2520virus%2520capsid-an%2520equilibrium-model%2520of%2520the%2520self-assembly%2520of%2520polyhedral%2520protein%2520complex%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D241%26spage%3D59%26epage%3D67%26doi%3D10.1006%2Fjmbi.1994.1473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stray, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Punna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">A heteroaryldihydropyrimidine activates and can misdirect hepatitis B virus capsid assembly</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">8138</span>– <span class="NLM_lpage">8143</span>, <span class="refDoi"> DOI: 10.1073/pnas.0409732102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.0409732102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=15928089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsV2msbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8138-8143&author=S.+J.+Strayauthor=C.+R.+Bourneauthor=S.+Punnaauthor=W.+G.+Lewisauthor=M.+G.+Finnauthor=A.+Zlotnick&title=A+heteroaryldihydropyrimidine+activates+and+can+misdirect+hepatitis+B+virus+capsid+assembly&doi=10.1073%2Fpnas.0409732102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A heteroaryldihydropyrimidine activates and can misdirect hepatitis B virus capsid assembly</span></div><div class="casAuthors">Stray, Stephen J.; Bourne, Christina R.; Punna, Sreenivas; Lewis, Warren G.; Finn, M. G.; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8138-8143</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Heteroaryldihydropyrimidines (HAPs) are a new class of antivirals inhibiting prodn. of hepatitis B virus (HBV) virions in tissue culture.  Here, we examine the effect of a representative HAP mol., Me 4-(2-chloro-4-fluorophenyl)-6-methyl-2-(pyridin-2-yl)-1,4-dihydropyrimidine-5-carboxylate (HAP-1), on the in vitro assembly of HBV capsid protein (Cp).  HAP-1 enhances the rate and extent of Cp assembly over a broad concn. range.  Aberrant particles, dominated by hexagonal arrays of Cp, were obsd. from assembly reactions with high HAP-1 concns.  HAP-1 also led to dissocn. of metastable HBV capsids, overcoming a kinetic barrier to dissocn. by scavenging Cp and redirecting its assembly into hexamer-rich structures.  Thus, HAP drugs act as allosteric effectors that induce an assembly-active state and, at high concn., preferentially stabilize noncapsid polymers of Cp.  HAP compds. may have multiple effects in vivo stemming from inappropriate assembly of Cp.  These results show that activating and deregulating virus assembly may be a powerful general approach for antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5IO3o_Hqhk7Vg90H21EOLACvtfcHk0lh7ca9RoDyHCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsV2msbc%253D&md5=30abe296011a5e771047ce4fdc0343a8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409732102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409732102%26sid%3Dliteratum%253Aachs%26aulast%3DStray%26aufirst%3DS.%2BJ.%26aulast%3DBourne%26aufirst%3DC.%2BR.%26aulast%3DPunna%26aufirst%3DS.%26aulast%3DLewis%26aufirst%3DW.%2BG.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DA%2520heteroaryldihydropyrimidine%2520activates%2520and%2520can%2520misdirect%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D8138%26epage%3D8143%26doi%3D10.1073%2Fpnas.0409732102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingfield, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endres, D.</span></span> <span> </span><span class="NLM_article-title">A theoretical model successfully identifies features of hepatitis B virus capsid assembly</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">14644</span>– <span class="NLM_lpage">14652</span>, <span class="refDoi"> DOI: 10.1021/bi991611a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi991611a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFaqurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=14644-14652&author=A.+Zlotnickauthor=J.+M.+Johnsonauthor=P.+W.+Wingfieldauthor=S.+J.+Stahlauthor=D.+Endres&title=A+theoretical+model+successfully+identifies+features+of+hepatitis+B+virus+capsid+assembly&doi=10.1021%2Fbi991611a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A theoretical model successfully identifies features of hepatitis B virus capsid assembly</span></div><div class="casAuthors">Zlotnick, Adam; Johnson, Jennifer M.; Wingfield, Paul W.; Stahl, Stephen J.; Endres, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14644-14652</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The capsids of most spherical viruses are icosahedral, an arrangement of multiples of 60 subunits.  Though it is a salient point in the life cycle of any virus, the phys. chem. of virus capsid assembly is poorly understood.  We have developed general models of capsid assembly that describe the process in terms of a cascade of low order assocn. reactions.  The models predict sigmoidal assembly kinetics, where intermediates approach a low steady state concn. for the greater part of the reaction.  Features of the overall reaction can be identified on the basis of the concn. dependence of assembly.  In simulations, and on the basis of our understanding of the models, we find that nucleus size and the order of subsequent "elongation" reactions are reflected in the concn. dependence of the extent of the reaction and the rate of the fast phase, resp.  The reaction kinetics deduced for our models of virus assembly can be related to the assembly of any "spherical" polymer.  Using light scattering and size exclusion chromatog., we obsd. polymn. of assembly domain dimers of hepatitis B virus (HBV) capsid protein.  Empty capsids assemble at a rate that is a function of protein concn. and ionic strength.  The kinetics of capsid formation were sigmoidal, where the rate of the fast phase had second-power concn. dependence.  The extent of assembly had third-power concn. dependence.  Simulations based on the models recapitulated the concn. dependences obsd. for HBV capsid assembly.  These results strongly suggest that in vitro HBV assembly is nucleated by a trimer of dimers and proceeds by the addn. of individual dimeric subunits.  On the basis of this mechanism, we suggest that HBV capsid assembly could be an important target for antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyHwRNPTccU7Vg90H21EOLACvtfcHk0lh7ca9RoDyHCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFaqurk%253D&md5=a79ca5f7ecbfca3af0687cfad18c26e5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi991611a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi991611a%26sid%3Dliteratum%253Aachs%26aulast%3DZlotnick%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DJ.%2BM.%26aulast%3DWingfield%26aufirst%3DP.%2BW.%26aulast%3DStahl%26aufirst%3DS.%2BJ.%26aulast%3DEndres%26aufirst%3DD.%26atitle%3DA%2520theoretical%2520model%2520successfully%2520identifies%2520features%2520of%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D14644%26epage%3D14652%26doi%3D10.1021%2Fbi991611a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ceres, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">Weak protein-protein interactions are sufficient to drive assembly of hepatitis B virus capsids</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">11525</span>– <span class="NLM_lpage">11531</span>, <span class="refDoi"> DOI: 10.1021/bi0261645</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0261645" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsFGms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=11525-11531&author=P.+Ceresauthor=A.+Zlotnick&title=Weak+protein-protein+interactions+are+sufficient+to+drive+assembly+of+hepatitis+B+virus+capsids&doi=10.1021%2Fbi0261645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Weak Protein-Protein Interactions Are Sufficient To Drive Assembly of Hepatitis B Virus Capsids</span></div><div class="casAuthors">Ceres, Pablo; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">11525-11531</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is an enveloped DNA virus with a spherical capsid (or core).  The capsid is constructed from 120 copies of the homodimeric capsid protein arranged with T = 4 icosahedral symmetry.  We examd. in vitro assembly of purified E. coli expressed HBV capsid protein.  After equilibration, concns. of capsid and dimer were evaluated by size exclusion chromatog.  The extent of assembly increased as temp. and ionic strength increased.  The concn. dependence of capsid assembly conformed to the equil. expression: Kcapsid = [capsid]/[dimer]120.  Given the known geometry for HBV capsids and dimers, the per capsid assembly energy was partitioned into energy per subunit-subunit contact.  We were able to make three major conclusions. (i) Weak interactions (from -2.9 kcal/mol at 21° in low salt to -4.4 kcal/mol at 37° in high salt) at each intersubunit contact result in a globally stable capsid; weak intersubunit interactions may be the basis for the phenomenon of capsid breathing. (ii) HBV assembly is characterized by pos. enthalpy and entropy.  The reaction is entropy-driven, consistent with the largely hydrophobic contacts found in the crystal structure. (iii) Increasing NaCl concn. increases the magnitude of free energy, enthalpy, and entropy, as if ionic strength were increasing the amt. of hydrophobic surface buried by assembly.  This last point leads us to suggest that salt acts by inducing a conformational change in the dimer from an assembly-inactive form to an assembly-active form.  This model of conformational change linked to assembly is consistent with immunol. differences between dimer and capsid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0k_uhx3AKtLVg90H21EOLACvtfcHk0lh7ca9RoDyHCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsFGms7w%253D&md5=00756d19244fe9b7d7e12747a8ef401d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi0261645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0261645%26sid%3Dliteratum%253Aachs%26aulast%3DCeres%26aufirst%3DP.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DWeak%2520protein-protein%2520interactions%2520are%2520sufficient%2520to%2520drive%2520assembly%2520of%2520hepatitis%2520B%2520virus%2520capsids%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D11525%26epage%3D11531%26doi%3D10.1021%2Fbi0261645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lutomski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyktey, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierson, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarrold, M. F.</span></span> <span> </span><span class="NLM_article-title">Hepatitis B virus capsid completion occurs through error correction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">16932</span>– <span class="NLM_lpage">16938</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09932</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09932" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslylurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=16932-16938&author=C.+A.+Lutomskiauthor=N.+A.+Lykteyauthor=Z.+C.+Zhaoauthor=E.+E.+Piersonauthor=A.+Zlotnickauthor=M.+F.+Jarrold&title=Hepatitis+B+virus+capsid+completion+occurs+through+error+correction&doi=10.1021%2Fjacs.7b09932"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B Virus Capsid Completion Occurs through Error Correction</span></div><div class="casAuthors">Lutomski, Corinne A.; Lyktey, Nicholas A.; Zhao, Zhongchao; Pierson, Elizabeth E.; Zlotnick, Adam; Jarrold, Martin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">16932-16938</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding capsid assembly is important because of its role in virus lifecycles and in applications to drug discovery and nanomaterial development.  Many virus capsids are icosahedral, and assembly is thought to occur by the sequential addn. of capsid protein subunits to a nucleus, with the final step completing the icosahedron.  Almost nothing is known about the final (completion) step because the techniques usually used to study capsid assembly lack the resoln.  In this work, charge detection mass spectrometry (CDMS) has been used to track the assembly of the T = 4 hepatitis B virus (HBV) capsid in real time.  The initial assembly reaction occurs rapidly, on the time scale expected from low resoln. measurements.  However, CDMS shows that many of the particles generated in this process are defective and overgrown, contg. more than the 120 capsid protein dimers needed to form a perfect T = 4 icosahedron.  The defective and overgrown capsids self-correct over time to the mass expected for a perfect T = 4 capsid.  Thus, completion is a distinct phase in the assembly reaction.  Capsid completion does not necessarily occur by inserting the last building block into an incomplete, but otherwise perfect icosahedron.  The initial assembly reaction can be predominently imperfect, and completion involves the slow correction of the accumulated errors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRb3DXlFTAt7Vg90H21EOLACvtfcHk0lji6l3SSSjU9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslylurfJ&md5=779a0fafdd729b170e607c9ac36958bd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09932%26sid%3Dliteratum%253Aachs%26aulast%3DLutomski%26aufirst%3DC.%2BA.%26aulast%3DLyktey%26aufirst%3DN.%2BA.%26aulast%3DZhao%26aufirst%3DZ.%2BC.%26aulast%3DPierson%26aufirst%3DE.%2BE.%26aulast%3DZlotnick%26aufirst%3DA.%26aulast%3DJarrold%26aufirst%3DM.%2BF.%26atitle%3DHepatitis%2520B%2520virus%2520capsid%2520completion%2520occurs%2520through%2520error%2520correction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D16932%26epage%3D16938%26doi%3D10.1021%2Fjacs.7b09932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pionek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unchwaniwala, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeb, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">The interface between hepatitis B virus capsid proteins affects self-assembly, pregenomic RNA packaging, and reverse transcription</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">3275</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1128/JVI.03545-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FJVI.03545-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=25568211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktF2gtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=3275-3284&author=Z.+N.+Tanauthor=K.+Pionekauthor=N.+Unchwaniwalaauthor=M.+L.+Maguireauthor=D.+D.+Loebauthor=A.+Zlotnick&title=The+interface+between+hepatitis+B+virus+capsid+proteins+affects+self-assembly%2C+pregenomic+RNA+packaging%2C+and+reverse+transcription&doi=10.1128%2FJVI.03545-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The interface between hepatitis B virus capsid proteins affects self-assembly, pregenomic RNA packaging, and reverse transcription</span></div><div class="casAuthors">Tan, Zhenning; Pionek, Karolyn; Unchwaniwala, Nuruddin; Maguire, Megan L.; Loeb, Daniel D.; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3275-3284</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) capsid proteins (Cps) assemble around the pregenomic RNA (pgRNA) and viral reverse transcriptase (P). pgRNA is then reverse transcribed to double-stranded DNA (dsDNA) within the capsid.  The Cp assembly domain, which forms the shell of the capsid, regulates assembly kinetics and capsid stability.  The Cp, via its nucleic acid-binding C-terminal domain, also affects nucleic acid organization.  We hypothesize that the structure of the capsid may also have a direct effect on nucleic acid processing.  Using structure-guided design, we made a series of mutations at the interface between Cp subunits that change capsid assembly kinetics and thermodn. in a predictable manner.  Assembly in cell culture mirrored in vitro activity.  However, all of these mutations led to defects in pgRNA packaging.  The amt. of first-strand DNA synthesized was roughly proportional to the amt. of RNA packaged.  However, the synthesis of second-strand DNA, which requires two template switches, was not supported by any of the substitutions.  These data demonstrate that the HBV capsid is far more than an inert container, as mutations in the assembly domain, distant from packaged nucleic acid, affect reverse transcription.  We suggest that capsid mol. motion plays a role in regulating genome replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoot8_H5tbmwLVg90H21EOLACvtfcHk0lji6l3SSSjU9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktF2gtrY%253D&md5=cbb94c6befce1b30e4600b65c0523290</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.03545-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03545-14%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DZ.%2BN.%26aulast%3DPionek%26aufirst%3DK.%26aulast%3DUnchwaniwala%26aufirst%3DN.%26aulast%3DMaguire%26aufirst%3DM.%2BL.%26aulast%3DLoeb%26aufirst%3DD.%2BD.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DThe%2520interface%2520between%2520hepatitis%2520B%2520virus%2520capsid%2520proteins%2520affects%2520self-assembly%252C%2520pregenomic%2520RNA%2520packaging%252C%2520and%2520reverse%2520transcription%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D3275%26epage%3D3284%26doi%3D10.1128%2FJVI.03545-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span> <span> </span><span class="NLM_article-title">HBc binds to the CpG islands of HBV cccDNA and promotes an epigenetic permissive state</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">720</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.4161/epi.6.6.15815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.4161%2Fepi.6.6.15815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=21546797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=720-726&author=Y.-H.+Guoauthor=Y.-N.+Liauthor=J.-R.+Zhaoauthor=J.+Zhangauthor=Z.+Yan&title=HBc+binds+to+the+CpG+islands+of+HBV+cccDNA+and+promotes+an+epigenetic+permissive+state&doi=10.4161%2Fepi.6.6.15815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">HBc binds to the CpG islands of HBV cccDNA and promotes an epigenetic permissive state</span></div><div class="casAuthors">Guo, Yan-Hai; Li, Yong-Nian; Zhao, Jin-Rong; Zhang, Ju; Yan, Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">720-726</span>CODEN:
                <span class="NLM_cas:coden">EPIGB6</span>;
        ISSN:<span class="NLM_cas:issn">1559-2294</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) covalently closed circular DNA (cccDNA) is the template for the transcription of HBV.  HBV core protein (HBc or HBcAg) is a main component of the HBV cccDNA minichromosome.  However, the function of HBc in cccDNA is not fully understood.  In light of recent findings that HBV cccDNA may be regulated epigenetically, we analyzed the binding of HBc to cccDNA and the impact of HBc on cccDNA epigenetic profile in the liver biopsy samples of 22 patients with chronic Hepatitis B (CHB).  We found that HBc binding to HBV cccDNA occurred preferentially at CpG island 2, an important region for the regulation of HBV transcription.  Furthermore, the relative abundances of HBc binding to CpG island 2 were pos. correlated with the ratios of relaxed circular DNA to cccDNA and the levels of serum HBV DNA in those patients.  Interestingly, the relative abundances of HBc binding to CpG island 2 were assocd. with the binding of CREB binding protein (CBP) and with hypomethylation in CpG island 2 of HBV cccDNA minichromosomes.  However, relatively higher amts. of HBc binding to CpG island 2 of cccDNA were accompanied by lower amts. of HDAC1 binding.  Multivariate anal. revealed that the abundances of HBc binding to CpG island 2 of cccDNA and pos. HBeAg were independent factors assocd. with the replication of HBV (p = 0.001 for both).  Apparently, HBc is a pos. regulator of HBV transcription and replication, maintaining the permissive epigenetic state in the crit. region of the HBV cccDNA minichromosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO9wmAiX25lrVg90H21EOLACvtfcHk0lib8VsV1ZhJPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gs7w%253D&md5=3057fac0de5c9a054f5ace2967f52a93</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Fepi.6.6.15815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.6.6.15815%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DY.-H.%26aulast%3DLi%26aufirst%3DY.-N.%26aulast%3DZhao%26aufirst%3DJ.-R.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DZ.%26atitle%3DHBc%2520binds%2520to%2520the%2520CpG%2520islands%2520of%2520HBV%2520cccDNA%2520and%2520promotes%2520an%2520epigenetic%2520permissive%2520state%26jtitle%3DEpigenetics%26date%3D2011%26volume%3D6%26spage%3D720%26epage%3D726%26doi%3D10.4161%2Fepi.6.6.15815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewellyn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S.</span></span> <span> </span><span class="NLM_article-title">Core protein: a pleiotropic keystone in the HBV lifecycle</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2015.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.antiviral.2015.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=26129969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2ltLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2015&pages=82-93&author=A.+Zlotnickauthor=B.+Venkatakrishnanauthor=Z.+N.+Tanauthor=E.+Lewellynauthor=W.+Turnerauthor=S.+Francis&title=Core+protein%3A+a+pleiotropic+keystone+in+the+HBV+lifecycle&doi=10.1016%2Fj.antiviral.2015.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Core protein: A pleiotropic keystone in the HBV lifecycle</span></div><div class="casAuthors">Zlotnick, Adam; Venkatakrishnan, Balasubramanian; Tan, Zhenning; Lewellyn, Eric; Turner, William; Francis, Samson</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-93</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Hepatitis B Virus (HBV) is a small virus whose genome has only four open reading frames.  We argue that the simplicity of the virion correlates with a complexity of functions for viral proteins.  We focus on the HBV core protein (Cp), a small (183 residue) protein that self-assembles to form the viral capsid.  However, its functions are a little more complicated than that.  In an infected cell Cp modulates almost every step of the viral lifecycle.  Cp is bound to nuclear viral DNA and affects its epigenetics.  Cp correlates with RNA specificity.  Cp assembles specifically on a reverse transcriptase-viral RNA complex or, apparently, nothing at all.  Indeed Cp has been one of the model systems for investigation of virus self-assembly.  Cp participates in regulation of reverse transcription.  Cp signals completion of reverse transcription to support virus secretion.  Cp carries both nuclear localization signals and HBV surface antigen (HBsAg) binding sites; both of these functions appear to be regulated by contents of the capsid.  Cp can be targeted by antivirals - while self-assembly is the most accessible of Cp activities, we argue that it makes sense to engage the broader spectrum of Cp function.  This article forms part of a symposium in Antiviral Research on "From the discovery of the Australia antigen to the development of new curative therapies for hepatitis B: an unfinished story.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyafN5HJZSQLVg90H21EOLACvtfcHk0lib8VsV1ZhJPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2ltLvM&md5=e1bd58b97219cb54136dbb2db2fadb70</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DZlotnick%26aufirst%3DA.%26aulast%3DVenkatakrishnan%26aufirst%3DB.%26aulast%3DTan%26aufirst%3DZ.%2BN.%26aulast%3DLewellyn%26aufirst%3DE.%26aulast%3DTurner%26aufirst%3DW.%26aulast%3DFrancis%26aufirst%3DS.%26atitle%3DCore%2520protein%253A%2520a%2520pleiotropic%2520keystone%2520in%2520the%2520HBV%2520lifecycle%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D121%26spage%3D82%26epage%3D93%26doi%3D10.1016%2Fj.antiviral.2015.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule therapeutics for treatment of chronic HBV infection</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.7b00144</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.7b00144" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=257-277&author=S.+Fengauthor=L.+Gaoauthor=X.+C.+Hanauthor=T.+S.+Huauthor=Y.+M.+Huauthor=H.+X.+Liuauthor=A.+W.+Thomasauthor=Z.+P.+Yanauthor=S.+Yangauthor=J.+A.+T.+Youngauthor=H.+Y.+Yunauthor=W.+Zhuauthor=H.+C.+Shen&title=Discovery+of+small+molecule+therapeutics+for+treatment+of+chronic+HBV+infection&doi=10.1021%2Facsinfecdis.7b00144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Therapeutics for Treatment of Chronic HBV Infection</span></div><div class="casAuthors">Feng, Song; Gao, Lu; Han, Xingchun; Hu, Taishan; Hu, Yimin; Liu, Haixia; Thomas, Andrew W.; Yan, Zhipeng; Yang, Song; Young, John A. T.; Yun, Hongying; Zhu, Wei; Shen, Hong C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-277</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The chronic infection of hepatitis B virus (HBV) inflicts 250 million people worldwide representing a major public health threat.  A significant sub-population of chronically infected HBV patients eventually develop cirrhosis and hepatocellular carcinoma (HCC) which aggravate the issues caused by hepatitis alone.  Unfortunately, none of the current std. therapies for chronic Hepatitis B (CHB) result in a satisfactory clin. cure rate.  Driven by a highly unmet medical need for novel and more effective treatments of HBV infection, multiple pharmaceutical companies and research institutions have been engaged in drug discovery and development to improve the CHB functional cure rate, which is defined by sustainable viral suppression and HBsAg clearance after a finite treatment.  This review summarizes the recent significant advances in the discovery and development of novel small mol. anti-HBV compds., highlighted by representative structures, mechanism of action (MoA), and biol. activities.  Lastly, it is believed that improved CHB functional cure rate may be accomplished via the combination of mols. with distinct MoAs.  Thus some of the mols. may evolve into key components of a suitable combination therapy leading to superior outcome of clin. efficacy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1zsMqu16n17Vg90H21EOLACvtfcHk0lj2zStScYPGdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D&md5=45e6f4740904bd92d82193455a2d97ac</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.7b00144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.7b00144%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%2BC.%26aulast%3DHu%26aufirst%3DT.%2BS.%26aulast%3DHu%26aufirst%3DY.%2BM.%26aulast%3DLiu%26aufirst%3DH.%2BX.%26aulast%3DThomas%26aufirst%3DA.%2BW.%26aulast%3DYan%26aufirst%3DZ.%2BP.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DYun%26aufirst%3DH.%2BY.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%2BC.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520therapeutics%2520for%2520treatment%2520of%2520chronic%2520HBV%2520infection%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2018%26volume%3D4%26spage%3D257%26epage%3D277%26doi%3D10.1021%2Facsinfecdis.7b00144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corcuera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, A.</span></span> <span> </span><span class="NLM_article-title">Novel non-heteroarylpyrimidine (HAP) capsid assembly modifiers have a different mode of action from HAPs in vitro</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">135</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2018.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.antiviral.2018.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=30031759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFequr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2018&pages=135-142&author=A.+Corcueraauthor=K.+Stolleauthor=S.+Hillmerauthor=S.+Seitzauthor=J.+Y.+Leeauthor=R.+Bartenschlagerauthor=A.+Birkmannauthor=A.+Urban&title=Novel+non-heteroarylpyrimidine+%28HAP%29+capsid+assembly+modifiers+have+a+different+mode+of+action+from+HAPs+in+vitro&doi=10.1016%2Fj.antiviral.2018.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Novel non-heteroarylpyrimidine (HAP) capsid assembly modifiers have a different mode of action from HAPs in vitro</span></div><div class="casAuthors">Corcuera, Angelica; Stolle, Katharina; Hillmer, Stefan; Seitz, Stefan; Lee, Ji-Young; Bartenschlager, Ralf; Birkmann, Alexander; Urban, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">135-142</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">One of the most promising viral targets in current hepatitis B virus (HBV) drug development is the core protein due to its multiple roles in the viral life cycle.  Here we investigated the differences in the mode of action and antiviral activity of representatives of six different capsid assembly modifier (CAM) scaffolds: three from the well-characterized scaffolds heteroarylpyrimidine (HAP), sulfamoylbenzamide (SBA), and phenylpropenamide (PPA), and three from novel scaffolds glyoxamide-pyrrolamide (GPA), pyrazolyl-thiazole (PT), and dibenzo-thiazepin-2-one (DBT).  The target activity and antiviral efficacy of the different CAMs were tested in biochem. and cellular assays.  Anal. size exclusion chromatog. and transmission electron microscopy showed that only the HAP compd. induced formation of aberrant non-capsid structures (class II mode of action), while the remaining CAMs did not affect capsid gross morphol. (class I mode of action).  Intracellular lysates from the HepAD38 cell line, inducibly replicating HBV, showed no redn. in the quantities of intracellular core protein or capsid after treatment with SBA, PPA, GPA, PT, or DBT compds.; however HAP-treatment led to a profound decrease in both.  Addnl., immunofluorescence staining of compd.-treated HepAD38 cells showed that all non-HAP CAMs led to a shift in the equil. of HBV core antigen (HBcAg) towards complete cytoplasmic staining, while the HAP induced accumulation of HBcAg aggregates in the nucleus.  Our study demonstrates that the novel scaffolds GPA, PT, and DBT exhibit class I modes of action, alike SBA and PPA, whereas HAP remains the only scaffold belonging to class II inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuxr1oBqjkrVg90H21EOLACvtfcHk0lj2zStScYPGdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFequr%252FJ&md5=25cd40330f90d3dccdefe80daa51cffd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2018.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2018.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DCorcuera%26aufirst%3DA.%26aulast%3DStolle%26aufirst%3DK.%26aulast%3DHillmer%26aufirst%3DS.%26aulast%3DSeitz%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DBirkmann%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DA.%26atitle%3DNovel%2520non-heteroarylpyrimidine%2520%2528HAP%2529%2520capsid%2520assembly%2520modifiers%2520have%2520a%2520different%2520mode%2520of%2520action%2520from%2520HAPs%2520in%2520vitro%26jtitle%3DAntiviral%2520Res.%26date%3D2018%26volume%3D158%26spage%3D135%26epage%3D142%26doi%3D10.1016%2Fj.antiviral.2018.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehertogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergauwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Damme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostmans, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauwels, F.</span></span> <span> </span><span class="NLM_article-title">Capsid assembly modulators have a dual mechanism of action in primary human hepatocytes infected with hepatitis B virus</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">e00560</span>, <span class="refDoi"> DOI: 10.1128/AAC.00560-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00560-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28584155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVSktbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=e00560&author=J.+M.+Berkeauthor=P.+Dehertoghauthor=K.+Vergauwenauthor=E.+Van+Dammeauthor=W.+Mostmansauthor=K.+Vandyckauthor=F.+Pauwels&title=Capsid+assembly+modulators+have+a+dual+mechanism+of+action+in+primary+human+hepatocytes+infected+with+hepatitis+B+virus&doi=10.1128%2FAAC.00560-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Capsid assembly modulators have a dual mechanism of action in primary human hepatocytes infected with hepatitis B virus</span></div><div class="casAuthors">Berke, Jan Martin; Dehertogh, Pascale; Vergauwen, Karen; Van Damme, Ellen; Mostmans, Wendy; Vandyck, Koen; Pauwels, Frederik</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e00560-17/14</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) capsid assembly is a crit. step in the propagation of the virus and is mediated by the core protein.  Due to its multiple functions in the viral life cycle, core became an attractive target for new antiviral therapies.  Capsid assembly modulators (CAMs) accelerate the kinetics of capsid assembly and prevent encapsidation of the polymerase-pregenomic RNA (Pol-pgRNA) complex, thereby blocking viral replication.  CAM JNJ-632 is a novel and potent inhibitor of HBV replication in vitro across genotypes A to D.  It induces the formation of morphol. intact viral capsids, as demonstrated by size exclusion chromatog. and electron microscopy studies.  Antiviral profiling in primary human hepatocytes revealed that CAMs prevented formation of covalently closed circular DNA in a dose-dependent fashion when the compd. was added together with the viral inoculum, whereas nucleos(t)ide analogs (NAs) did not.  This protective effect translated into a dose-dependent redn. of intracellular HBV RNA levels as well as reduced HBe/cAg and HBsAg levels in the cell culture supernatant.  The same observation was made with another CAM (BAY41-4109), suggesting that mechanistic rather than compd.-specific effects play a role.  Our data show that CAMs have a dual mechanism of action, inhibiting early and late steps of the viral life cycle.  These effects clearly differentiate CAMs from NAs and may translate into higher functional cure rates in a clin. setting when given alone or in combination with the current std. of care.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdf2H1OpRSlrVg90H21EOLACvtfcHk0lga8lL-kffvXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVSktbbO&md5=665812f18beb49225110302e9afbd980</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FAAC.00560-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00560-17%26sid%3Dliteratum%253Aachs%26aulast%3DBerke%26aufirst%3DJ.%2BM.%26aulast%3DDehertogh%26aufirst%3DP.%26aulast%3DVergauwen%26aufirst%3DK.%26aulast%3DVan%2BDamme%26aufirst%3DE.%26aulast%3DMostmans%26aufirst%3DW.%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DPauwels%26aufirst%3DF.%26atitle%3DCapsid%2520assembly%2520modulators%2520have%2520a%2520dual%2520mechanism%2520of%2520action%2520in%2520primary%2520human%2520hepatocytes%2520infected%2520with%2520hepatitis%2520B%2520virus%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26spage%3De00560%26doi%3D10.1128%2FAAC.00560-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahlali, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergauwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauwels, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoulim, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span> <span> </span><span class="NLM_article-title">Novel potent capsid assembly modulators regulate multiple steps of the hepatitis B virus life cycle</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">15</span>, <span class="refDoi"> DOI: 10.1128/AAC.00835-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00835-18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=15&author=T.+Lahlaliauthor=J.+M.+Berkeauthor=K.+Vergauwenauthor=A.+Focaauthor=K.+Vandyckauthor=F.+Pauwelsauthor=F.+Zoulimauthor=D.+Durantel&title=Novel+potent+capsid+assembly+modulators+regulate+multiple+steps+of+the+hepatitis+B+virus+life+cycle&doi=10.1128%2FAAC.00835-18"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FAAC.00835-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00835-18%26sid%3Dliteratum%253Aachs%26aulast%3DLahlali%26aufirst%3DT.%26aulast%3DBerke%26aufirst%3DJ.%2BM.%26aulast%3DVergauwen%26aufirst%3DK.%26aulast%3DFoca%26aufirst%3DA.%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DPauwels%26aufirst%3DF.%26aulast%3DZoulim%26aufirst%3DF.%26aulast%3DDurantel%26aufirst%3DD.%26atitle%3DNovel%2520potent%2520capsid%2520assembly%2520modulators%2520regulate%2520multiple%2520steps%2520of%2520the%2520hepatitis%2520B%2520virus%2520life%2520cycle%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2018%26volume%3D62%26spage%3D15%26doi%3D10.1128%2FAAC.00835-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lam, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp, K.</span></span> <span> </span><span class="NLM_article-title">Hepatitis B virus capsid assembly modulators, but not nucleoside analogs, inhibit the production of extracellular pregenomic RNA and spliced RNA variants</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">e00680</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1128/AAC.00680-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00680-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28559265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Wltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=e00680-17&author=A.+M.+Lamauthor=S.+P.+Renauthor=C.+Espirituauthor=M.+Kellyauthor=V.+Lauauthor=L.+J.+Zhengauthor=G.+D.+Hartmanauthor=O.+A.+Floresauthor=K.+Klumpp&title=Hepatitis+B+virus+capsid+assembly+modulators%2C+but+not+nucleoside+analogs%2C+inhibit+the+production+of+extracellular+pregenomic+RNA+and+spliced+RNA+variants&doi=10.1128%2FAAC.00680-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B virus capsid assembly modulators, but not nucleoside analogs, inhibit the production of extracellular pregenomic RNA and spliced RNA variants</span></div><div class="casAuthors">Lam, Angela M.; Ren, Suping; Espiritu, Christine; Kelly, Mollie; Lau, Vincent; Zheng, Lingjie; Hartman, George D.; Flores, Osvaldo A.; Klumpp, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e00680-17/1-e00680-17/14</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The hepatitis B virus (HBV) core protein serves multiple essential functions in the viral life cycle, and antiviral agents that target the core protein are being developed.  Capsid assembly modulators (CAMs) are compds. that target core and misdirect capsid assembly, resulting in the suppression of HBV replication and virion prodn.  Besides HBV DNA, circulating HBV RNA has been detected in patient serum and can be assocd. with the treatment response.  Here the authors studied the effect of HBV CAMs on the prodn. of extracellular HBV RNA using infected HepaRG cells and primary human hepatocytes.  Representative compds. from the sulfonamide carboxamide and heteroaryldihydropyrimidine series of CAMs were evaluated and compared to nucleos(t)ide analogs as inhibitors of the viral polymerase.  The results showed that CAMs blocked extracellular HBV RNA with efficiencies similar to those with which they blocked pregenomic RNA (pgRNA) encapsidation, HBV DNA replication, and Dane particle prodn.  Nucleos(t)ide analogs inhibited viral replication and virion prodn. but not encapsidation or prodn. of extracellular HBV RNA.  Profiling of HBV RNA from both culture supernatants and patient serum showed that extracellular viral RNA consisted of pgRNA and spliced pgRNA variants with an internal deletion(s) but still retained the sequences at both the 5' and 3' ends.  Similar variants were detected in the supernatants of infected cells with and without nucleos(t)ide analog treatment.  Overall, the data demonstrate that HBV CAMs represent direct antiviral agents with a profile differentiated from that of nucleos(t)ide analogs, including the inhibition of extracellular pgRNA and spliced pgRNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJzKwT-Zo87rVg90H21EOLACvtfcHk0lga8lL-kffvXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Wltrw%253D&md5=8bfa2b78fa1da04f49b5f5e0d1fbdc77</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FAAC.00680-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00680-17%26sid%3Dliteratum%253Aachs%26aulast%3DLam%26aufirst%3DA.%2BM.%26aulast%3DRen%26aufirst%3DS.%2BP.%26aulast%3DEspiritu%26aufirst%3DC.%26aulast%3DKelly%26aufirst%3DM.%26aulast%3DLau%26aufirst%3DV.%26aulast%3DZheng%26aufirst%3DL.%2BJ.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26aulast%3DFlores%26aufirst%3DO.%2BA.%26aulast%3DKlumpp%26aufirst%3DK.%26atitle%3DHepatitis%2520B%2520virus%2520capsid%2520assembly%2520modulators%252C%2520but%2520not%2520nucleoside%2520analogs%252C%2520inhibit%2520the%2520production%2520of%2520extracellular%2520pregenomic%2520RNA%2520and%2520spliced%2520RNA%2520variants%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26spage%3De00680%26epage%3D17%26doi%3D10.1128%2FAAC.00680-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardzinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobarrubias, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuconati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harasym, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadhim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kultgen, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A. H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Q. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majeski, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClintock, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rijnbrand, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snead, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuer, H. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stever, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sofia, M. J.</span></span> <span> </span><span class="NLM_article-title">Preclinical profile of AB-423, an inhibitor of hepatitis B virus pregenomic RNA encapsidation</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">22</span>, <span class="refDoi"> DOI: 10.1128/AAC.00082-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00082-18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=22&author=N.+Maniauthor=A.+G.+Coleauthor=J.+R.+Phelpsauthor=A.+Ardzinskiauthor=K.+D.+Cobarrubiasauthor=A.+Cuconatiauthor=B.+D.+Dorseyauthor=E.+Evangelistaauthor=K.+Fanauthor=F.+Guoauthor=H.+T.+Guoauthor=J.+T.+Guoauthor=T.+O.+Harasymauthor=S.+Kadhimauthor=S.+G.+Kultgenauthor=A.+C.+H.+Leeauthor=A.+H.+L.+Liauthor=Q.+X.+Longauthor=S.+A.+Majeskiauthor=R.+C.+Maoauthor=K.+D.+McClintockauthor=S.+P.+Reidauthor=R.+Rijnbrandauthor=N.+M.+Sneadauthor=H.+M.+M.+Steuerauthor=K.+Steverauthor=S.+Tangauthor=X.+H.+Wangauthor=Q.+Zhaoauthor=M.+J.+Sofia&title=Preclinical+profile+of+AB-423%2C+an+inhibitor+of+hepatitis+B+virus+pregenomic+RNA+encapsidation&doi=10.1128%2FAAC.00082-18"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FAAC.00082-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00082-18%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DN.%26aulast%3DCole%26aufirst%3DA.%2BG.%26aulast%3DPhelps%26aufirst%3DJ.%2BR.%26aulast%3DArdzinski%26aufirst%3DA.%26aulast%3DCobarrubias%26aufirst%3DK.%2BD.%26aulast%3DCuconati%26aufirst%3DA.%26aulast%3DDorsey%26aufirst%3DB.%2BD.%26aulast%3DEvangelista%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DK.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DH.%2BT.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26aulast%3DHarasym%26aufirst%3DT.%2BO.%26aulast%3DKadhim%26aufirst%3DS.%26aulast%3DKultgen%26aufirst%3DS.%2BG.%26aulast%3DLee%26aufirst%3DA.%2BC.%2BH.%26aulast%3DLi%26aufirst%3DA.%2BH.%2BL.%26aulast%3DLong%26aufirst%3DQ.%2BX.%26aulast%3DMajeski%26aufirst%3DS.%2BA.%26aulast%3DMao%26aufirst%3DR.%2BC.%26aulast%3DMcClintock%26aufirst%3DK.%2BD.%26aulast%3DReid%26aufirst%3DS.%2BP.%26aulast%3DRijnbrand%26aufirst%3DR.%26aulast%3DSnead%26aufirst%3DN.%2BM.%26aulast%3DSteuer%26aufirst%3DH.%2BM.%2BM.%26aulast%3DStever%26aufirst%3DK.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DSofia%26aufirst%3DM.%2BJ.%26atitle%3DPreclinical%2520profile%2520of%2520AB-423%252C%2520an%2520inhibitor%2520of%2520hepatitis%2520B%2520virus%2520pregenomic%2520RNA%2520encapsidation%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2018%26volume%3D62%26spage%3D22%26doi%3D10.1128%2FAAC.00082-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bester, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blossey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacherl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondran, F. W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protzer, U.</span></span> <span> </span><span class="NLM_article-title">A new role for capsid assembly modulators to target mature hepatitis B virus capsids and prevent virus infection</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">13</span>, <span class="refDoi"> DOI: 10.1128/AAC.01440-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.01440-19" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2019&pages=13&author=C.+Koauthor=R.+Besterauthor=X.+Zhouauthor=Z.+H.+Xuauthor=C.+Blosseyauthor=J.+Sacherlauthor=F.+W.+R.+Vondranauthor=L.+Gaoauthor=U.+Protzer&title=A+new+role+for+capsid+assembly+modulators+to+target+mature+hepatitis+B+virus+capsids+and+prevent+virus+infection&doi=10.1128%2FAAC.01440-19"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FAAC.01440-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01440-19%26sid%3Dliteratum%253Aachs%26aulast%3DKo%26aufirst%3DC.%26aulast%3DBester%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%2BH.%26aulast%3DBlossey%26aufirst%3DC.%26aulast%3DSacherl%26aufirst%3DJ.%26aulast%3DVondran%26aufirst%3DF.%2BW.%2BR.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DProtzer%26aufirst%3DU.%26atitle%3DA%2520new%2520role%2520for%2520capsid%2520assembly%2520modulators%2520to%2520target%2520mature%2520hepatitis%2520B%2520virus%2520capsids%2520and%2520prevent%2520virus%2520infection%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D64%26spage%3D13%26doi%3D10.1128%2FAAC.01440-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perni, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladner, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaifert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. W.</span></span> <span> </span><span class="NLM_article-title">Phenylpropenamide derivatives as inhibitors of hepatitis B virus replication</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2687</span>– <span class="NLM_lpage">2690</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(00)00544-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS0960-894X%2800%2900544-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=11128652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotlantbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=2687-2690&author=R.+B.+Perniauthor=S.+C.+Conwayauthor=S.+K.+Ladnerauthor=K.+Zaifertauthor=M.+J.+Ottoauthor=R.+W.+King&title=Phenylpropenamide+derivatives+as+inhibitors+of+hepatitis+B+virus+replication&doi=10.1016%2FS0960-894X%2800%2900544-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylpropenamide derivatives as inhibitors of Hepatitis B virus replication</span></div><div class="casAuthors">Perni, R. B.; Conway, S. C.; Ladner, S. K.; Zaifert, K.; Otto, M. J.; King, R. W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2687-2690</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A non-nucleoside class of compds. that inhibits the replication of hepatitis B virus (HBV) in cell culture has been discovered.  A series of substituted analogs of phenylpropenamide has been prepd. and evaluated in the HepAD38 cellular assay.  Structure-activity relationships of this series are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcZcKEPcrqkbVg90H21EOLACvtfcHk0ljQeSPnxaEO9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotlantbw%253D&md5=4c3ee48f05dc8bee0724ba928d89f8a1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2800%2900544-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252800%252900544-8%26sid%3Dliteratum%253Aachs%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DConway%26aufirst%3DS.%2BC.%26aulast%3DLadner%26aufirst%3DS.%2BK.%26aulast%3DZaifert%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DM.%2BJ.%26aulast%3DKing%26aufirst%3DR.%2BW.%26atitle%3DPhenylpropenamide%2520derivatives%2520as%2520inhibitors%2520of%2520hepatitis%2520B%2520virus%2520replication%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D2687%26epage%3D2690%26doi%3D10.1016%2FS0960-894X%2800%2900544-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campagna, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuconati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span> <span> </span><span class="NLM_article-title">Sulfamoylbenzamide derivatives inhibit the assembly of hepatitis B virus nucleocapsids</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">6931</span>– <span class="NLM_lpage">6942</span>, <span class="refDoi"> DOI: 10.1128/JVI.00582-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FJVI.00582-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=23576513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlKrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=6931-6942&author=M.+R.+Campagnaauthor=F.+Liuauthor=R.+C.+Maoauthor=C.+Millsauthor=D.+W.+Caiauthor=F.+Guoauthor=X.+S.+Zhaoauthor=H.+Yeauthor=A.+Cuconatiauthor=H.+T.+Guoauthor=J.+H.+Changauthor=X.+D.+Xuauthor=T.+M.+Blockauthor=J.+T.+Guo&title=Sulfamoylbenzamide+derivatives+inhibit+the+assembly+of+hepatitis+B+virus+nucleocapsids&doi=10.1128%2FJVI.00582-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfamoylbenzamide derivatives inhibit the assembly of hepatitis B virus nucleocapsids</span></div><div class="casAuthors">Campagna, Matthew R.; Liu, Fei; Mao, Richeng; Mills, Courtney; Cai, Dawei; Guo, Fang; Zhao, Xuesen; Ye, Hong; Cuconati, Andrea; Guo, Haitao; Chang, Jinhong; Xu, Xiaodong; Block, Timothy M.; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6931-6942</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection, a serious public health problem leading to cirrhosis and hepatocellular carcinoma, is currently treated with either pegylated alpha interferon (pegIFN-α) or one of the five nucleos(t)ide analog viral DNA polymerase inhibitors.  However, neither pegIFN-α nor nucleos(t)ide analogs are capable of reliably curing the viral infection.  In order to develop novel antiviral drugs against HBV, we established a cell-based screening assay by using an immortalized mouse hepatocyte-derived stable cell line supporting a high level of HBV replication in a tetracycline-inducible manner.  Screening of a library consisting of 26,900 small mols. led to the discovery of a series of sulfamoylbenzamide (SBA) derivs. that significantly reduced the amt. of cytoplasmic HBV DNA.  Structure-activity relationship studies have thus far identified a group of fluorine-substituted SBAs with submicromolar antiviral activity against HBV in human hepatoma cells.  Mechanistic analyses reveal that the compds. dose dependently inhibit the formation of pregenomic RNA (pgRNA)-contg. nucleocapsids of HBV but not other animal hepadnaviruses, such as woodchuck hepatitis virus (WHV) and duck hepatitis B virus (DHBV).  Moreover, heterologous genetic complementation studies of capsid protein, DNA polymerase, and pgRNA between HBV and WHV suggest that HBV capsid protein confers sensitivity to the SBAs.  In summary, SBAs represent a novel chem. entity with superior activity and a unique antiviral mechanism and are thus warranted for further development as novel antiviral therapeutics for the treatment of chronic hepatitis B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNUsTkQKVBQbVg90H21EOLACvtfcHk0lials7wR0f2yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlKrsr0%253D&md5=80cd9d05d889ecf1bd2f6c28f7946bcf</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FJVI.00582-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00582-13%26sid%3Dliteratum%253Aachs%26aulast%3DCampagna%26aufirst%3DM.%2BR.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DMao%26aufirst%3DR.%2BC.%26aulast%3DMills%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DD.%2BW.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DX.%2BS.%26aulast%3DYe%26aufirst%3DH.%26aulast%3DCuconati%26aufirst%3DA.%26aulast%3DGuo%26aufirst%3DH.%2BT.%26aulast%3DChang%26aufirst%3DJ.%2BH.%26aulast%3DXu%26aufirst%3DX.%2BD.%26aulast%3DBlock%26aufirst%3DT.%2BM.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26atitle%3DSulfamoylbenzamide%2520derivatives%2520inhibit%2520the%2520assembly%2520of%2520hepatitis%2520B%2520virus%2520nucleocapsids%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D6931%26epage%3D6942%26doi%3D10.1128%2FJVI.00582-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlemmer, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagelschuer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paessens, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deres, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niewoehner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoltefuss, J. D. I.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Inhibition of human hepatitis B virus (HBV) by a novel non-nucleosidic compound in a transgenic mouse model</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/S0166-3542(01)00216-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS0166-3542%2801%2900216-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=12062392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtVShsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2002&pages=69-78&author=O.+Weberauthor=K.+H.+Schlemmerauthor=E.+Hartmannauthor=I.+Hagelschuerauthor=A.+D.+Paessensauthor=E.+Graefauthor=K.+Deresauthor=S.+Goldmannauthor=U.+Niewoehnerauthor=J.+D.+I.+Stoltefuss&title=Inhibition+of+human+hepatitis+B+virus+%28HBV%29+by+a+novel+non-nucleosidic+compound+in+a+transgenic+mouse+model&doi=10.1016%2FS0166-3542%2801%2900216-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human hepatitis B virus (HBV) by a novel non-nucleosidic compound in a transgenic mouse model</span></div><div class="casAuthors">Weber, O.; Schlemmer, K.-H.; Hartmann, E.; Hagelschuer, Ina; Paessens, A.; Graef, E.; Deres, K.; Goldmann, S.; Niewoehner, U.; Stoltefuss, J.; Haebich, D.; Ruebsamen-Waigmann, H.; Wohlfeil, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">69-78</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">BAY 41-4109 is a member of a class of heteroaryl-pyrimidines that was recently identified as potent inhibitors of human hepatitis B virus (HBV) replication.  We have investigated the antiviral activity of BAY 41-4109 (Me (R)-4-(2-chloro-4-fluorophenyl)-2-(3,5-difluoro-2-pyridinyl)-6-methyl-1,4-dihydro-pyrimidine-5-carboxylate) in HBV-transgenic mice (Tg [HBV1.3 fsX-3'5']).  Bay 41-4109 was administered per os using different schedules (b.i.d. or t.i.d. for up to 28 days) and dosages ranging from 3 to 30 mg/kg.  The compd. reduced viral DNA in the liver and in the plasma dose-dependently with efficacy comparable to 3TC.  In contrast to 3TC-treated mice, we found a redn. of cytoplasmic hepatitis B virus core antigen (HBcAg) in liver sections of BAY 41-4109-treated mice, which indicated a different mode of action.  Pharmacokinetic studies in mice have shown rapid absorption, a bioavailability of 30% and dose-proportional plasma concns.  We conclude that BAY 41-4109 is a new anti-HBV drug candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAj4XpaXvzGrVg90H21EOLACvtfcHk0lgsVQq4kOb3Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtVShsLg%253D&md5=bae520cd731096858f86f4ce28674231</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0166-3542%2801%2900216-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0166-3542%252801%252900216-9%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DSchlemmer%26aufirst%3DK.%2BH.%26aulast%3DHartmann%26aufirst%3DE.%26aulast%3DHagelschuer%26aufirst%3DI.%26aulast%3DPaessens%26aufirst%3DA.%2BD.%26aulast%3DGraef%26aufirst%3DE.%26aulast%3DDeres%26aufirst%3DK.%26aulast%3DGoldmann%26aufirst%3DS.%26aulast%3DNiewoehner%26aufirst%3DU.%26aulast%3DStoltefuss%26aufirst%3DJ.%2BD.%2BI.%26atitle%3DInhibition%2520of%2520human%2520hepatitis%2520B%2520virus%2520%2528HBV%2529%2520by%2520a%2520novel%2520non-nucleosidic%2520compound%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DAntiviral%2520Res.%26date%3D2002%26volume%3D54%26spage%3D69%26epage%3D78%26doi%3D10.1016%2FS0166-3542%2801%2900216-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katen, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">Assembly-directed antivirals differentially bind quasiequivalent pockets to modify hepatitis B virus capsid tertiary and quaternary structure</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1406</span>– <span class="NLM_lpage">1416</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.str.2013.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=23871485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCgs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1406-1416&author=S.+P.+Katenauthor=Z.+N.+Tanauthor=S.+R.+Chirapuauthor=M.+G.+Finnauthor=A.+Zlotnick&title=Assembly-directed+antivirals+differentially+bind+quasiequivalent+pockets+to+modify+hepatitis+B+virus+capsid+tertiary+and+quaternary+structure&doi=10.1016%2Fj.str.2013.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Assembly-Directed Antivirals Differentially Bind Quasiequivalent Pockets to Modify Hepatitis B Virus Capsid Tertiary and Quaternary Structure</span></div><div class="casAuthors">Katen, Sarah P.; Tan, Zhenning; Chirapu, Srinivas Reddy; Finn, M. G.; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1406-1416</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is a major cause of liver disease.  Assembly of the HBV capsid is a crit. step in virus prodn. and an attractive target for new antiviral therapies.  We detd. the structure of HBV capsid in complex with AT-130, a member of the phenylpropenamide family of assembly effectors.  AT-130 causes tertiary and quaternary structural changes but does not disrupt capsid structure.  AT-130 binds a hydrophobic pocket that also accommodates the previously characterized heteroaryldihydropyrimidine compds. but favors a unique quasiequivalent location on the capsid surface.  Thus, this pocket is a promiscuous drug-binding site and a likely target for different assembly effectors with a broad range of mechanisms of activity.  That AT-130 successfully decreases virus prodn. by increasing capsid assembly rate without disrupting capsid structure delineates a paradigm in antiviral design, that disrupting reaction timing is a viable strategy for assembly effectors of HBV and other viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZur3DCpnS3LVg90H21EOLACvtfcHk0lgsVQq4kOb3Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCgs7fF&md5=bb515fd504fb69f82755409c4c8f2079</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DKaten%26aufirst%3DS.%2BP.%26aulast%3DTan%26aufirst%3DZ.%2BN.%26aulast%3DChirapu%26aufirst%3DS.%2BR.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DAssembly-directed%2520antivirals%2520differentially%2520bind%2520quasiequivalent%2520pockets%2520to%2520modify%2520hepatitis%2520B%2520virus%2520capsid%2520tertiary%2520and%2520quaternary%2520structure%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1406%26epage%3D1416%26doi%3D10.1016%2Fj.str.2013.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baydo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abendroth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Efimov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, O. A.</span></span> <span> </span><span class="NLM_article-title">High-resolution crystal structure of a hepatitis B virus replication inhibitor bound to the viral core protein</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">15196</span>– <span class="NLM_lpage">15201</span>, <span class="refDoi"> DOI: 10.1073/pnas.1513803112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.1513803112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=26598693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWqsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15196-15201&author=K.+Klumppauthor=A.+M.+Lamauthor=C.+Lukacsauthor=R.+Vogelauthor=S.+P.+Renauthor=C.+Espirituauthor=R.+Baydoauthor=K.+Atkinsauthor=J.+Abendrothauthor=G.+C.+Liaoauthor=A.+Efimovauthor=G.+Hartmanauthor=O.+A.+Flores&title=High-resolution+crystal+structure+of+a+hepatitis+B+virus+replication+inhibitor+bound+to+the+viral+core+protein&doi=10.1073%2Fpnas.1513803112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution crystal structure of a hepatitis B virus replication inhibitor bound to the viral core protein</span></div><div class="casAuthors">Klumpp, Klaus; Lam, Angela M.; Lukacs, Christine; Vogel, Robert; Ren, Suping; Espiritu, Christine; Baydo, Ruth; Atkins, Kateri; Abendroth, Jan; Liao, Guochun; Efimov, Andrey; Hartman, George; Flores, Osvaldo A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">15196-15201</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The hepatitis B virus (HBV) core protein is essential for HBV replication and an important target for antiviral drug discovery.  We report the first, to our knowledge, high-resoln. crystal structure of an antiviral compd. bound to the HBV core protein.  The compd. NVR-010-001-E2 can induce assembly of the HBV core wild-type and Y132A mutant proteins and thermostabilize the proteins with a Tm increase of more than 10 °C.  NVR-010-001-E2 binds at the dimer-dimer interface of the core proteins, forms a new interaction surface promoting protein-protein interaction, induces protein assembly, and increases stability.  The impact of naturally occurring core protein mutations on antiviral activity correlates with NVR-010-001-E2 binding interactions detd. by crystallog.  The crystal structure provides understanding of a drug efficacy mechanism related to the induction and stabilization of protein-protein interactions and enables structure-guided design to improve antiviral potency and drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYxV-odPonGbVg90H21EOLACvtfcHk0lgsVQq4kOb3Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWqsrbK&md5=bf487dda7165d1637fad0ebab8f46142</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1513803112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1513803112%26sid%3Dliteratum%253Aachs%26aulast%3DKlumpp%26aufirst%3DK.%26aulast%3DLam%26aufirst%3DA.%2BM.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DVogel%26aufirst%3DR.%26aulast%3DRen%26aufirst%3DS.%2BP.%26aulast%3DEspiritu%26aufirst%3DC.%26aulast%3DBaydo%26aufirst%3DR.%26aulast%3DAtkins%26aufirst%3DK.%26aulast%3DAbendroth%26aufirst%3DJ.%26aulast%3DLiao%26aufirst%3DG.%2BC.%26aulast%3DEfimov%26aufirst%3DA.%26aulast%3DHartman%26aufirst%3DG.%26aulast%3DFlores%26aufirst%3DO.%2BA.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520a%2520hepatitis%2520B%2520virus%2520replication%2520inhibitor%2520bound%2520to%2520the%2520viral%2520core%2520protein%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15196%26epage%3D15201%26doi%3D10.1073%2Fpnas.1513803112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Alcalde, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najera, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span> <span> </span><span class="NLM_article-title">Heteroaryldihydropyrimidine (HAP) and sulfamoylbenzamide (SBA) inhibit hepatitis B virus replication by different molecular mechanisms</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42374</span>, <span class="refDoi"> DOI: 10.1038/srep42374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1038%2Fsrep42374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28205569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislOiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42374&author=Z.+Zhouauthor=T.+S.+Huauthor=X.+Zhouauthor=S.+Wildumauthor=F.+Garcia-Alcaldeauthor=Z.+H.+Xuauthor=D.+Z.+Wuauthor=Y.+Maoauthor=X.+J.+Tianauthor=Y.+Zhouauthor=F.+Shenauthor=Z.+S.+Zhangauthor=G.+Z.+Tangauthor=I.+Najeraauthor=G.+Yangauthor=H.+C.+Shenauthor=J.+A.+T.+Youngauthor=N.+Qin&title=Heteroaryldihydropyrimidine+%28HAP%29+and+sulfamoylbenzamide+%28SBA%29+inhibit+hepatitis+B+virus+replication+by+different+molecular+mechanisms&doi=10.1038%2Fsrep42374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Heteroaryldihydropyrimidine (HAP) and Sulfamoylbenzamide (SBA) Inhibit Hepatitis B Virus Replication by Different Molecular Mechanisms</span></div><div class="casAuthors">Zhou, Zheng; Hu, Taishan; Zhou, Xue; Wildum, Steffen; Garcia-Alcalde, Fernando; Xu, Zhiheng; Wu, Daitze; Mao, Yi; Tian, Xiaojun; Zhou, Yuan; Shen, Fang; Zhang, Zhisen; Tang, Guozhi; Najera, Isabel; Yang, Guang; Shen, Hong C.; Young, John A. T.; Qin, Ning</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42374</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Heteroaryldihydropyrimidine (HAP) and sulfamoylbenzamide (SBA) are promising non-nucleos(t)ide HBV replication inhibitors.  HAPs are known to promote core protein mis-assembly, but the mol. mechanism of abnormal assembly is still elusive.  Likewise, the assembly status of core protein induced by SBA remains unknown.  Here we show that SBA, unlike HAP, does not promote core protein mis-assembly.  Interestingly, two ref. compds. HAP_R01 and SBA_R01 bind to the same pocket at the dimer-dimer interface in the crystal structures of core protein Y132A hexamer.  The striking difference lies in a unique hydrophobic subpocket that is occupied by the thiazole group of HAP_R01, but is unperturbed by SBA_R01.  Photoaffinity labeling confirms the HAP_R01 binding pose at the dimer-dimer interface on capsid and suggests a new mechanism of HAP-induced mis-assembly.  Based on the common features in crystal structures we predict that T33 mutations generate similar susceptibility changes to both compds.  In contrast, mutations at positions in close contact with HAP-specific groups (P25A, P25S, or V124F) only reduce susceptibility to HAP_R01, but not to SBA_R01.  Thus, HAP and SBA are likely to have distinctive resistance profiles.  Notably, P25S and V124F substitutions exist in low-abundance quasispecies in treatment-naive patients, suggesting potential clin. relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw-qU6eO7rCrVg90H21EOLACvtfcHk0lgQLtDxaBWM-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislOiurs%253D&md5=1e401e3133194ae374435df0d0cfa637</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsrep42374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42374%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DT.%2BS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DWildum%26aufirst%3DS.%26aulast%3DGarcia-Alcalde%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DZ.%2BH.%26aulast%3DWu%26aufirst%3DD.%2BZ.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DX.%2BJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DZ.%2BS.%26aulast%3DTang%26aufirst%3DG.%2BZ.%26aulast%3DNajera%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DQin%26aufirst%3DN.%26atitle%3DHeteroaryldihydropyrimidine%2520%2528HAP%2529%2520and%2520sulfamoylbenzamide%2520%2528SBA%2529%2520inhibit%2520hepatitis%2520B%2520virus%2520replication%2520by%2520different%2520molecular%2520mechanisms%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42374%26doi%3D10.1038%2Fsrep42374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okazaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinoda, W.</span></span> <span> </span><span class="NLM_article-title">Heteroaryldihydropyrimidines alter capsid assembly by adjusting the binding affinity and pattern of the hepatitis B virus core protein</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5104</span>– <span class="NLM_lpage">5110</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1WktrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=5104-5110&author=H.+Liuauthor=S.+Okazakiauthor=W.+Shinoda&title=Heteroaryldihydropyrimidines+alter+capsid+assembly+by+adjusting+the+binding+affinity+and+pattern+of+the+hepatitis+B+virus+core+protein&doi=10.1021%2Facs.jcim.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Heteroaryldihydropyrimidines Alter Capsid Assembly By Adjusting the Binding Affinity and Pattern of the Hepatitis B Virus Core Protein</span></div><div class="casAuthors">Liu, Huihui; Okazaki, Susumu; Shinoda, Wataru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5104-5110</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) infections are a major global health concern, for which heteroaryldihydropyrimidines (HAPs) have been developed.  HAPs accelerate and/or result in aberrant capsid assembly; however, their effect on the assembly mechanism is unknown.  This study aimed to compare the effects of three representative HAPs on core protein dimer assembly through mol. dynamics simulations and free energy calcns.  Mol. docking and equil. simulations showed that different HAPs bind at the same binding site and are involved in different interactions.  The obsd. conformational changes in HAPs deter the calcn. of binding affinity.  Herein, the reduced free energy perturbation/Hamiltonian replica exchange mol. dynamics method was used to enhance sampling during binding affinity calcns., indicating consistency between the binding free energies of HAPs and pEC50.  Furthermore, binding pattern anal. revealed that the tetramer could sample flat structures after binding HAPs.  The present results suggest a mechanism wherein HAPs accelerate capsid assembly by increasing the binding affinity of dimers, leading to aberrant assembly by altering the binding orientation of dimers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc9UKC5UgIArVg90H21EOLACvtfcHk0lgQLtDxaBWM-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1WktrvP&md5=0871ef3341b009a15504cae14cd03938</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DOkazaki%26aufirst%3DS.%26aulast%3DShinoda%26aufirst%3DW.%26atitle%3DHeteroaryldihydropyrimidines%2520alter%2520capsid%2520assembly%2520by%2520adjusting%2520the%2520binding%2520affinity%2520and%2520pattern%2520of%2520the%2520hepatitis%2520B%2520virus%2520core%2520protein%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D5104%26epage%3D5110%26doi%3D10.1021%2Facs.jcim.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sari, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozturk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, O. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassit, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amblard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinazi, R. F.</span></span> <span> </span><span class="NLM_article-title">Synthesis of sulfamoylbenzamide derivatives as HBV capsid assembly effector</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.ejmech.2017.06.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28688280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=407-421&author=O.+Sariauthor=S.+Boucleauthor=B.+D.+Coxauthor=T.+Ozturkauthor=O.+I.+Russellauthor=L.+Bassitauthor=F.+Amblardauthor=R.+F.+Schinazi&title=Synthesis+of+sulfamoylbenzamide+derivatives+as+HBV+capsid+assembly+effector&doi=10.1016%2Fj.ejmech.2017.06.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of sulfamoylbenzamide derivatives as HBV capsid assembly effectors</span></div><div class="casAuthors">Sari, Ozkan; Boucle, Sebastien; Cox, Bryan D.; Ozturk, Tugba; Russell, Olivia Ollinger; Bassit, Leda; Amblard, Franck; Schinazi, Raymond F.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">407-421</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The synthesis of novel series of sulfamoylbenzamides, e.g. I [R1 = F, R2 = H; R1R2 = (CH2)2, (CH2)3; X = R3R4NSO2, R5SO2NH; R3 = cyclopentyl, cyclopentyloxy, MeSO2, PhCHMe, etc., R4 = H; R3R4 = CH2CF2CH2CH2, C(O)(CH2)3, etc.; R5 = cyclopropyl, cyclopentyl, cyclohexyl] as HBV capsid assembly effector is reported.  The structure was divided into five parts which were independently modified as part of the lead optimization.  All the synthesized compds. were evaluated for their anti-HBV activity and toxicity in human hepatocytes, lymphocytes and other cells.  Addnl., their effect on HBV cccDNA formation in an HBeAg reporter cell-based assay was assessed.  Among the 27 compds. reported, several analogs exhibited submicromolar activities and significant redn. of HBeAg secretion.  Selected compds. were studied under neg.-stain electron microscopy for their ability to disrupt the HBV capsid formation.  Structures were modeled into a binding site recently identified in the HBV capsid protein for similar mols. to rationalize the structure-activity relationships for this family of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUa_fzgDe9WrVg90H21EOLACvtfcHk0lgQLtDxaBWM-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvI&md5=cc0de429427455fac4cebd76c51119ed</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.062%26sid%3Dliteratum%253Aachs%26aulast%3DSari%26aufirst%3DO.%26aulast%3DBoucle%26aufirst%3DS.%26aulast%3DCox%26aufirst%3DB.%2BD.%26aulast%3DOzturk%26aufirst%3DT.%26aulast%3DRussell%26aufirst%3DO.%2BI.%26aulast%3DBassit%26aufirst%3DL.%26aulast%3DAmblard%26aufirst%3DF.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26atitle%3DSynthesis%2520of%2520sulfamoylbenzamide%2520derivatives%2520as%2520HBV%2520capsid%2520assembly%2520effector%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D407%26epage%3D421%26doi%3D10.1016%2Fj.ejmech.2017.06.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rombouts, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoops, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergauwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehertogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raboisson, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of <i>N</i>-phenyl-3-sulfamoyl-benzamide derivatives as capsid assembly modulators inhibiting hepatitis B virus (HBV)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6247</span>– <span class="NLM_lpage">6260</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6247-6260&author=K.+Vandyckauthor=G.+Romboutsauthor=B.+Stoopsauthor=A.+Tahriauthor=A.+Vosauthor=W.+Verschuerenauthor=Y.+M.+Wuauthor=J.+M.+Yangauthor=F.+L.+Houauthor=B.+Huangauthor=K.+Vergauwenauthor=P.+Dehertoghauthor=J.+M.+Berkeauthor=P.+Raboisson&title=Synthesis+and+evaluation+of+N-phenyl-3-sulfamoyl-benzamide+derivatives+as+capsid+assembly+modulators+inhibiting+hepatitis+B+virus+%28HBV%29&doi=10.1021%2Facs.jmedchem.8b00654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of N-Phenyl-3-sulfamoyl-benzamide Derivatives as Capsid Assembly Modulators Inhibiting Hepatitis B Virus (HBV)</span></div><div class="casAuthors">Vandyck, Koen; Rombouts, Geert; Stoops, Bart; Tahri, Abdellah; Vos, Ann; Verschueren, Wim; Wu, Yiming; Yang, Jingmei; Hou, Fuliang; Huang, Bing; Vergauwen, Karen; Dehertogh, Pascale; Berke, Jan Martin; Raboisson, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6247-6260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol. induced hepatitis B virus (HBV) capsid assembly modulation is considered an attractive approach for new antiviral therapies against HBV.  Here we describe efforts toward the discovery of a HBV capsid assembly modulator in a hit-to-lead optimization, resulting in JNJ-632, a tool compd. used to further profile the mode of action.  Administration of JNJ-632 (54) in HBV genotype D infected chimeric mice resulted in a 2.77 log redn. of the HBV DNA viral load.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKzPnHUf4R5LVg90H21EOLACvtfcHk0ljhTR9vuqUi2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsrfF&md5=b877a4732f8dc2141c60fba79f27aa52</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00654%26sid%3Dliteratum%253Aachs%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DRombouts%26aufirst%3DG.%26aulast%3DStoops%26aufirst%3DB.%26aulast%3DTahri%26aufirst%3DA.%26aulast%3DVos%26aufirst%3DA.%26aulast%3DVerschueren%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DHou%26aufirst%3DF.%2BL.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DVergauwen%26aufirst%3DK.%26aulast%3DDehertogh%26aufirst%3DP.%26aulast%3DBerke%26aufirst%3DJ.%2BM.%26aulast%3DRaboisson%26aufirst%3DP.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520N-phenyl-3-sulfamoyl-benzamide%2520derivatives%2520as%2520capsid%2520assembly%2520modulators%2520inhibiting%2520hepatitis%2520B%2520virus%2520%2528HBV%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6247%26epage%3D6260%26doi%3D10.1021%2Facs.jmedchem.8b00654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of new hepatitis B virus capsid assembly modulators by an optimal high-throughput cell-based assay</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">778</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.9b00030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.9b00030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivV2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=778-787&author=Y.+M.+Peiauthor=C.+T.+Wangauthor=H.+J.+Benauthor=L.+Wangauthor=Y.+Maauthor=Q.+Y.+Maauthor=Y.+Xiangauthor=L.+Q.+Zhangauthor=G.+Liu&title=Discovery+of+new+hepatitis+B+virus+capsid+assembly+modulators+by+an+optimal+high-throughput+cell-based+assay&doi=10.1021%2Facsinfecdis.9b00030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New Hepatitis B Virus Capsid Assembly Modulators by an Optimal High-Throughput Cell-Based Assay</span></div><div class="casAuthors">Pei, Yameng; Wang, Chunting; Ben, Haijing; Wang, Lei; Ma, Yao; Ma, Qingyan; Xiang, Ye; Zhang, Linqi; Liu, Gang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">778-787</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, a simple and effective high-throughput screening (HTS) assay was developed to identify anti-HBV compds. by using a HepAD38 luciferase reporter (HepAD38-luc) cell line that can effectively exclude the false pos. hit compds. targeted on the tetracycline off (tet-off) regulation system.  Through screening inhouse chem. libraries, N-phenylpiperidine-3-carboxamide derivs., represented by 1 and 2, were identified, while the other false pos. hits (i.e., quinoxaline (3) and benzothiazin (4) derivs.) were simultaneously excluded.  Compds. 1 and 2 exhibit strong inhibitory activity against HBV replication in both HepAD38 and HepG2.2.15 cells.  Further studies revealed that 1 and 2 reduced extracellular HBV DNA, HBeAg, and intracellular HBV intermediates, including total DNA, RNA, and precore RNA of HBV.  Size-exclusion chromatog. (SEC) and electron microscopy (EM) investigations demonstrated that 1 and 2 remarkably induced the formation of morphol. intact capsids and accelerated the dynamics of capsid assembly, suggesting that both 1 and 2 were type I capsid assembly modulators (CAMs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovmkgieeKSfrVg90H21EOLACvtfcHk0ljhTR9vuqUi2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivV2qu7w%253D&md5=770a2f4bb84f92426260d1031724319c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.9b00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.9b00030%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DY.%2BM.%26aulast%3DWang%26aufirst%3DC.%2BT.%26aulast%3DBen%26aufirst%3DH.%2BJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%2BY.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DL.%2BQ.%26aulast%3DLiu%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520new%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%2520modulators%2520by%2520an%2520optimal%2520high-throughput%2520cell-based%2520assay%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D778%26epage%3D787%26doi%3D10.1021%2Facsinfecdis.9b00030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-T.</span></span> <span> </span><span class="NLM_article-title">Discovery and mechanistic study of benzamide derivatives that modulate hepatitis B virus capsid assembly</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">e00519</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1128/JVI.00519-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FJVI.00519-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28566379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvVKh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2017&pages=e00519-17&author=S.+Wuauthor=Q.+Zhaoauthor=P.+Zhangauthor=J.+Kulpauthor=L.+Huauthor=N.+Hwangauthor=J.+Zhangauthor=T.+M.+Blockauthor=X.+Xuauthor=Y.+Duauthor=J.+Changauthor=J.-T.+Guo&title=Discovery+and+mechanistic+study+of+benzamide+derivatives+that+modulate+hepatitis+B+virus+capsid+assembly&doi=10.1128%2FJVI.00519-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and mechanistic study of benzamide derivatives that modulate hepatitis B virus capsid assembly</span></div><div class="casAuthors">Wu, Shuo; Zhao, Qiong; Zhang, Pinghu; Kulp, John; Hu, Lydia; Hwang, Nicky; Zhang, Jiming; Block, Timothy M.; Xu, Xiaodong; Du, Yanming; Chang, Jinhong; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">e00519-17/1-e00519-17/18</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection is a global public health problem.  Although the currently approved medications can reliably reduce the viral load and prevent the progression of liver diseases, they fail to cure the viral infection.  In an effort toward discovery of novel antiviral agents against HBV, a group of benzamide (BA) derivs. that significantly reduced the amt. of cytoplasmic HBV DNA were discovered.  The initial lead optimization efforts identified two BA derivs. with improved antiviral activity for further mechanistic studies.  Interestingly, similar to our previously reported sulfamoylbenzamides (SBAs), the BAs promote the formation of empty capsids through specific interaction with HBV core protein but not other viral and host cellular components.  Genetic evidence suggested that both SBAs and BAs inhibited HBV nucleocapsid assembly by binding to the heteroaryldihydropyrimidine (HAP) pocket between core protein dimer-dimer interfaces.  However, unlike SBAs, BA compds. uniquely induced the formation of empty capsids that migrated more slowly in native agarose gel electrophoresis from A36V mutant than from the wild-type core protein.  Moreover, we showed that the assembly of chimeric capsids from wild-type and drug-resistant core proteins was susceptible to multiple capsid assembly modulators.  Hence, HBV core protein is a dominant antiviral target that may suppress the selection of drug-resistant viruses during core protein-targeting antiviral therapy.  Our studies thus indicate that BAs are a chem. and mechanistically unique type of HBV capsid assembly modulators and warranted for further development as antiviral agents against HBV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLV9R8pGFhULVg90H21EOLACvtfcHk0ljDc4lVsZd6Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvVKh&md5=e1fae3fad652ba4c65d84d55ed9aa751</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1128%2FJVI.00519-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00519-17%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DKulp%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DHwang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBlock%26aufirst%3DT.%2BM.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.-T.%26atitle%3DDiscovery%2520and%2520mechanistic%2520study%2520of%2520benzamide%2520derivatives%2520that%2520modulate%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%26jtitle%3DJ.%2520Virol.%26date%3D2017%26volume%3D91%26spage%3De00519%26epage%3D17%26doi%3D10.1128%2FJVI.00519-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span>; <span class="NLM_string-name">Verschueren, W. G.</span>; <span class="NLM_string-name">Raboisson, P. J. -M. B.</span></span> <span> </span><span class="NLM_article-title">Fused bicyclic sulfamoyl derivatives and the use as medicaments for the treatment of hepatitis B</span>. PCT Int. Appl. <span class="NLM_patent">WO 2014033167</span>, Mar 6, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=K.+Vandyck&author=W.+G.+Verschueren&author=P.+J.+-M.+B.+Raboisson&title=Fused+bicyclic+sulfamoyl+derivatives+and+the+use+as+medicaments+for+the+treatment+of+hepatitis+B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVandyck%26aufirst%3DK.%26atitle%3DFused%2520bicyclic%2520sulfamoyl%2520derivatives%2520and%2520the%2520use%2520as%2520medicaments%2520for%2520the%2520treatment%2520of%2520hepatitis%2520B%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gillis, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eastman, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Applications of fluorine in medicinal chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">8315</span>– <span class="NLM_lpage">8359</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8315-8359&author=E.+P.+Gillisauthor=K.+J.+Eastmanauthor=M.+D.+Hillauthor=D.+J.+Donnellyauthor=N.+A.+Meanwell&title=Applications+of+fluorine+in+medicinal+chemistry&doi=10.1021%2Facs.jmedchem.5b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of Fluorine in Medicinal Chemistry</span></div><div class="casAuthors">Gillis, Eric P.; Eastman, Kyle J.; Hill, Matthew D.; Donnelly, David J.; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8315-8359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with meta-anal. The role of fluorine in drug design and development is expanding rapidly as we learn more about the unique properties assocd. with this unusual element and how to deploy it with greater sophistication.  The judicious introduction of fluorine into a mol. can productively influence conformation, pKa, intrinsic potency, membrane permeability, metabolic pathways, and pharmacokinetic properties.  In addn., 18F has been established as a useful positron emitting isotope for use with in vivo imaging technol. that potentially has extensive application in drug discovery and development, often limited only by convenient synthetic accessibility to labeled compds.  The wide ranging applications of fluorine in drug design are providing a strong stimulus for the development of new synthetic methodologies that allow more facile access to a wide range of fluorinated compds.  In this review, we provide an update on the effects of the strategic incorporation of fluorine in drug mols. and applications in positron emission tomog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8fUUtkPB4L7Vg90H21EOLACvtfcHk0ljDc4lVsZd6Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK&md5=9995829a94a8c0b8d9fb0d21bdfd5a1d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00258%26sid%3Dliteratum%253Aachs%26aulast%3DGillis%26aufirst%3DE.%2BP.%26aulast%3DEastman%26aufirst%3DK.%2BJ.%26aulast%3DHill%26aufirst%3DM.%2BD.%26aulast%3DDonnelly%26aufirst%3DD.%2BJ.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DApplications%2520of%2520fluorine%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8315%26epage%3D8359%26doi%3D10.1021%2Facs.jmedchem.5b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, S. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span> <span> </span><span class="NLM_article-title">The impact of aromatic ring count on compound developability: further insights by examining carbo- and hetero-aromatic and -aliphatic ring types</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.drudis.2010.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=21129497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVyqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=164-171&author=T.+J.+Ritchieauthor=S.+J.+F.+Macdonaldauthor=R.+J.+Youngauthor=S.+D.+Pickett&title=The+impact+of+aromatic+ring+count+on+compound+developability%3A+further+insights+by+examining+carbo-+and+hetero-aromatic+and+-aliphatic+ring+types&doi=10.1016%2Fj.drudis.2010.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of aromatic ring count on compound developability: further insights by examining carbo- and hetero-aromatic and -aliphatic ring types</span></div><div class="casAuthors">Ritchie, Timothy J.; MacDonald, Simon J. F.; Young, Robert J.; Pickett, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3/4</span>),
    <span class="NLM_cas:pages">164-171</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The impact of carboarom., heteroarom., carboaliph. and heteroaliph. ring counts and fused arom. ring count on several developability measures (soly., lipophilicity, protein binding, P 450 inhibition and hERG binding) is the topic for this review article.  Recent results indicate that increasing ring counts have detrimental effects on developability in the order carboaroms. » heteroaroms. > carboaliphatics > heteroaliphatics, with heteroaliphatics exerting a beneficial effect in many cases.  Increasing arom. ring count exerts effects on several developability parameters that are lipophilicity- and size-independent, and fused arom. systems have a beneficial effect relative to their nonfused counterparts.  Increasing arom. ring count has a detrimental effect on human bioavailability parameters, and heteroarom. ring count (but not other ring counts) has increased over time in marketed oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJYPC7hkoyv7Vg90H21EOLACvtfcHk0ljDc4lVsZd6Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVyqsr0%253D&md5=8e2d0e4e499d2f07d66ca2f385defb2d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DRitchie%26aufirst%3DT.%2BJ.%26aulast%3DMacdonald%26aufirst%3DS.%2BJ.%2BF.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26atitle%3DThe%2520impact%2520of%2520aromatic%2520ring%2520count%2520on%2520compound%2520developability%253A%2520further%2520insights%2520by%2520examining%2520carbo-%2520and%2520hetero-aromatic%2520and%2520-aliphatic%2520ring%2520types%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D164%26epage%3D171%26doi%3D10.1016%2Fj.drudis.2010.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, L.</span></span> <i>The role of functional groups in drug-receptor interaction: in the practice of medicinal chemistry</i>, <span class="NLM_edition">3</span>rd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermuth, C.-G.</span></span>, Ed.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">480</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=464-480&author=L.+Schaefferauthor=C.-G.+Wermuth&title=The+role+of+functional+groups+in+drug-receptor+interaction%3A+in+the+practice+of+medicinal+chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchaeffer%26aufirst%3DL.%26btitle%3DThe%2520role%2520of%2520functional%2520groups%2520in%2520drug-receptor%2520interaction%253A%2520in%2520the%2520practice%2520of%2520medicinal%2520chemistry%26aulast%3DWermuth%26aufirst%3DC.-G.%26pub%3DElsevier%26date%3D2008%26spage%3D464%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic efficiency as an important metric in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+efficiency+as+an+important+metric+in+drug+design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0lhhNWVtjxCFHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520efficiency%2520as%2520an%2520important%2520metric%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarcsay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyiri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span> <span> </span><span class="NLM_article-title">Impact of lipophilic efficiency on compound quality</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.1021/jm201388p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201388p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVKjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1252-1260&author=A.+Tarcsayauthor=K.+Nyiriauthor=G.+M.+Keseru&title=Impact+of+lipophilic+efficiency+on+compound+quality&doi=10.1021%2Fjm201388p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Lipophilic Efficiency on Compound Quality</span></div><div class="casAuthors">Tarcsay, Akos; Nyiri, Kinga; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1252-1260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lipophilic efficiency indexes such as LLE and LELP were suggested to support balanced optimization of potency and ADMET profile.  Here we investigated the performance of LLE and LELP on multiple data sets representing different stages of drug discovery including fragment and HTS hits and leads, development candidates, phase II compds., and launched drugs.  Analyzing their impact on ADME and safety properties and binding thermodn., we found that both LLE and LELP help identifying better quality compds.  LLE is sensible for the development stages but does not prefer fragment-type hits, while LELP has an advantage for this class of compds. and discriminates preferred starting points effectively.  Both LLE and LELP have significant impact on ADME and safety profiles; however, LELP outperforms LLE in risk assessment at least on the present data set.  On the basis of the results reported here, monitoring lipophilic efficiency metrics could contribute significantly to compd. quality and might improve the output of medicinal chem. programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxB8xs6-5DL7Vg90H21EOLACvtfcHk0lhhNWVtjxCFHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktVKjsQ%253D%253D&md5=2536abef781f52806013016971c78bcb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm201388p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201388p%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DNyiri%26aufirst%3DK.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DImpact%2520of%2520lipophilic%2520efficiency%2520on%2520compound%2520quality%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1252%26epage%3D1260%26doi%3D10.1021%2Fjm201388p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cumming, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span> <span> </span><span class="NLM_article-title">Lowering lipophilicity by adding carbon: one-carbon bridges of morpholines and piperazines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8934</span>– <span class="NLM_lpage">8943</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8934-8943&author=S.+L.+Degorceauthor=M.+S.+Bodnarchukauthor=I.+A.+Cummingauthor=J.+S.+Scott&title=Lowering+lipophilicity+by+adding+carbon%3A+one-carbon+bridges+of+morpholines+and+piperazines&doi=10.1021%2Facs.jmedchem.8b01148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Lowering Lipophilicity by Adding Carbon: One-Carbon Bridges of Morpholines and Piperazines</span></div><div class="casAuthors">Degorce, Sebastien L.; Bodnarchuk, Michael S.; Cumming, Iain A.; Scott, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8934-8943</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, we report our investigation of a phenomenon by which bridging morpholines across the ring with one-carbon tethers leads to a counterintuitive redn. in lipophilicity.  This effect was also found to occur in piperazines and piperidines and lowered the measured log D7.4 of the bridged mols. by as much as -0.8 relative to their unbridged counterparts.  As lowering lipophilicity without introducing addnl. heteroatoms can be desirable, we believe this potentially provides a useful tactic to improve the drug-like properties of mols. contg. morpholine-, piperazine-, and piperidine-like motifs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhZJB3SYRKD7Vg90H21EOLACvtfcHk0lhhNWVtjxCFHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK&md5=27f30b8220ea767d6ef1e492f3fa1aba</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01148%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DCumming%26aufirst%3DI.%2BA.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26atitle%3DLowering%2520lipophilicity%2520by%2520adding%2520carbon%253A%2520one-carbon%2520bridges%2520of%2520morpholines%2520and%2520piperazines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8934%26epage%3D8943%26doi%3D10.1021%2Facs.jmedchem.8b01148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zafrani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sod-Moriah, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeffet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berliner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katalan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madmon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershonov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saphier, S.</span></span> <span> </span><span class="NLM_article-title">CF2H, a functional group-dependent hydrogen-bond donor: is it a more or less lipophilic bioisostere of OH, SH, and CH3?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5628</span>– <span class="NLM_lpage">5637</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00604</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00604" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVOksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5628-5637&author=Y.+Zafraniauthor=G.+Sod-Moriahauthor=D.+Yeffetauthor=A.+Berlinerauthor=D.+Amirauthor=D.+Marcianoauthor=S.+Eliasauthor=S.+Katalanauthor=N.+Ashkenaziauthor=M.+Madmonauthor=E.+Gershonovauthor=S.+Saphier&title=CF2H%2C+a+functional+group-dependent+hydrogen-bond+donor%3A+is+it+a+more+or+less+lipophilic+bioisostere+of+OH%2C+SH%2C+and+CH3%3F&doi=10.1021%2Facs.jmedchem.9b00604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">CF2H, a Functional Group-Dependent Hydrogen-Bond Donor: Is It a More or Less Lipophilic Bioisostere of OH, SH, and CH3?</span></div><div class="casAuthors">Zafrani, Yossi; Sod-Moriah, Gali; Yeffet, Dina; Berliner, Anat; Amir, Dafna; Marciano, Daniele; Elias, Shlomi; Katalan, Shahaf; Ashkenazi, Nissan; Madmon, Moran; Gershonov, Eytan; Saphier, Sigal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5628-5637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effects of the CF2H moiety on H-bond (HB) acidity and lipophilicity of various compds., when attached directly to an arom. ring or to other functions like alkyls, ethers/thioethers, or electron-withdrawing groups, are discussed.  It was found that the CF2H group acts as a HB donor with a strong dependence on the attached functional group (A = 0.035-0.165).  Regarding lipophilicity, the CF2H group may act as a more lipophilic bioisostere of OH but as a similar or less lipophilic bioisostere of SH and CH3, resp., when attached to Ar or alkyl.  In addn., the lipophilicity of ethers, sulfoxides, and sulfones is dramatically increased upon CH3/CF2H exchange at the α position.  Interestingly, this exchange significantly affects not only the polarity and the vol. of the solutes but also their HB-accepting ability, the main factors influencing log Poct.  Accordingly, this study may be helpful in the rational design of drugs contg. this moiety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoib73JwC9F-bVg90H21EOLACvtfcHk0liAxdMQ3nYxfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVOksrw%253D&md5=cef6c35ca2138d3e18d79d41a1661c3c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00604%26sid%3Dliteratum%253Aachs%26aulast%3DZafrani%26aufirst%3DY.%26aulast%3DSod-Moriah%26aufirst%3DG.%26aulast%3DYeffet%26aufirst%3DD.%26aulast%3DBerliner%26aufirst%3DA.%26aulast%3DAmir%26aufirst%3DD.%26aulast%3DMarciano%26aufirst%3DD.%26aulast%3DElias%26aufirst%3DS.%26aulast%3DKatalan%26aufirst%3DS.%26aulast%3DAshkenazi%26aufirst%3DN.%26aulast%3DMadmon%26aufirst%3DM.%26aulast%3DGershonov%26aufirst%3DE.%26aulast%3DSaphier%26aufirst%3DS.%26atitle%3DCF2H%252C%2520a%2520functional%2520group-dependent%2520hydrogen-bond%2520donor%253A%2520is%2520it%2520a%2520more%2520or%2520less%2520lipophilic%2520bioisostere%2520of%2520OH%252C%2520SH%252C%2520and%2520CH3%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5628%26epage%3D5637%26doi%3D10.1021%2Facs.jmedchem.9b00604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span> <span> </span><span class="NLM_article-title">Direct inhibition of hepatitis B e antigen by core protein allosteric modulator</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1002/hep.30514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1002%2Fhep.30514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=30664279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yis7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2019&pages=11-24&author=Z.+P.+Yanauthor=D.+Wuauthor=H.+Huauthor=J.+Zengauthor=X.+Yuauthor=Z.+H.+Xuauthor=Z.+Zhouauthor=X.+Zhouauthor=G.+Yangauthor=J.+A.+T.+Youngauthor=L.+Gao&title=Direct+inhibition+of+hepatitis+B+e+antigen+by+core+protein+allosteric+modulator&doi=10.1002%2Fhep.30514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct Inhibition of Hepatitis B e Antigen by Core Protein Allosteric Modulator</span></div><div class="casAuthors">Yan, Zhipeng; Wu, Daitze; Hu, Hui; Zeng, Jing; Yu, Xin; Xu, Zhiheng; Zhou, Zheng; Zhou, Xue; Yang, Guang; Young, John A. T.; Gao, Lu</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-24</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hepatitis B e antigen (HBeAg) is an important immunomodulator for promoting host immune tolerance during chronic hepatitis B (CHB) infection.  In patients with CHB, HBeAg loss and seroconversion represent partial immune control of CHB infection and are regarded as valuable endpoints.  However, the current approved treatments have only a limited efficacy in achieving HBeAg seroconversion in HBeAg-pos. patients.  Hepatitis B virus (HBV) core protein has been recognized as an attractive antiviral target, and two classes of core protein allosteric modulator (CpAM) have been discovered: the phenylpropenamides (PPAs) and the heteroaryldihydropyrimidines (HAPs).  However, their differentiation and potential therapeutic benefit beyond HBV DNA inhibition remain to be seen.  Here, we show that in contrast to PPA series compd. AT-130, a HAP CpAM, HAP_R01, reduced HBeAg levels in multiple in vitro and in vivo HBV exptl. models.  Mechanistically, we found that HAP_R01 treatment caused the misassembly of capsids formed by purified HBeAg in vitro.  In addn., HAP_R01 directly reduces HBeAg levels by inducing intracellular precore protein misassembly and aggregation.  Using a HAP_R01-resistant mutant, we found that HAP_R01-mediated HBeAg and core protein redns. were mediated through the same mechanism.  Furthermore, HAP_R01 treatment substantially reduced serum HBeAg levels in an HBV mouse model.  Conclusion: Unlike PPA series compd. AT-130, HAP_R01 not only inhibits HBV DNA levels but also directly reduces HBeAg through induction of its misassembly.  HAP_R01, as well as other similar CpAMs, has the potential to achieve higher anti-HBeAg seroconversion rates than currently approved therapies for patients with CHB.  Our findings also provide guidance for dose selection when designing clin. trials with mols. from HAP series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptmHAW9ju4QrVg90H21EOLACvtfcHk0liAxdMQ3nYxfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yis7fO&md5=d400c27579525d3309ce818480deba6e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fhep.30514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.30514%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DZ.%2BP.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%2BH.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DGao%26aufirst%3DL.%26atitle%3DDirect%2520inhibition%2520of%2520hepatitis%2520B%2520e%2520antigen%2520by%2520core%2520protein%2520allosteric%2520modulator%26jtitle%3DHepatology%26date%3D2019%26volume%3D70%26spage%3D11%26epage%3D24%26doi%3D10.1002%2Fhep.30514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span> <span> </span><span class="NLM_article-title">Establishment of a stable expression and replication cell line transfected with HBV genotype C prevailed in China</span>. <i>Med. Pharm. J. Chin.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1428</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=1425-1428&author=L.+Wangauthor=W.+Liuauthor=Y.+Liuauthor=D.+Jiauthor=L.+Siauthor=Y.+Zhongauthor=D.+Xu&title=Establishment+of+a+stable+expression+and+replication+cell+line+transfected+with+HBV+genotype+C+prevailed+in+China"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DD.%26aulast%3DSi%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DD.%26atitle%3DEstablishment%2520of%2520a%2520stable%2520expression%2520and%2520replication%2520cell%2520line%2520transfected%2520with%2520HBV%2520genotype%2520C%2520prevailed%2520in%2520China%26jtitle%3DMed.%2520Pharm.%2520J.%2520Chin.%26date%3D2010%26volume%3D35%26spage%3D1425%26epage%3D1428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serajuddin, A. T. M.</span></span> <span> </span><span class="NLM_article-title">Salt formation to improve drug solubility</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">616</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2007.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.addr.2007.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=17619064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=603-616&author=A.+T.+M.+Serajuddin&title=Salt+formation+to+improve+drug+solubility&doi=10.1016%2Fj.addr.2007.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Salt formation to improve drug solubility</span></div><div class="casAuthors">Serajuddin, Abu T. M.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">603-616</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Salt formation is the most common and effective method of increasing soly. and dissoln. rates of acidic and basic drugs.  In this article, physicochem. principles of salt soly. are presented, with special ref. to the influence of pH-soly. profiles of acidic and basic drugs on salt formation and dissoln.  Non-ideality of salt soly. due to self-assocn. in soln. is also discussed.  Whether certain acidic or basic drugs would form salts and, if salts are formed, how easily they would dissoc. back into their free acid or base forms depend on interrelationships of several factors, such as S0 (intrinsic soly.), pH, pKa, Ksp (soly. product) and pHmax (pH of max. soly.).  The interrelationships of these factors are elaborated and their influence on salt screening and the selection of optimal salt forms for development are discussed.  Factors influencing salt dissoln. under various pH conditions, and esp. in reactive media and in presence of excess common ions, are discussed, with practical ref. to the development of solid dosage forms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZdRvmSUWvN7Vg90H21EOLACvtfcHk0ljaDIPhCVLycg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrc%253D&md5=6e77d1385622af6973a6d763746ad0db</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DSerajuddin%26aufirst%3DA.%2BT.%2BM.%26atitle%3DSalt%2520formation%2520to%2520improve%2520drug%2520solubility%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2007%26volume%3D59%26spage%3D603%26epage%3D616%26doi%3D10.1016%2Fj.addr.2007.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althage, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chisari, F. V.</span></span> <span> </span><span class="NLM_article-title">Hydrodynamic injection of viral DNA: A mouse model of acute hepatitis B virus infection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">13825</span>– <span class="NLM_lpage">13830</span>, <span class="refDoi"> DOI: 10.1073/pnas.202398599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.202398599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=12374864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=13825-13830&author=P.+L.+Yangauthor=A.+Althageauthor=J.+Chungauthor=F.+V.+Chisari&title=Hydrodynamic+injection+of+viral+DNA%3A+A+mouse+model+of+acute+hepatitis+B+virus+infection&doi=10.1073%2Fpnas.202398599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrodynamic injection of viral DNA: a mouse model of acute hepatitis B virus infection</span></div><div class="casAuthors">Yang, Priscilla L.; Althage, Alana; Chung, Josan; Chisari, Francis V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">13825-13830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is a prototype for liver-specific pathogens in which the failure of the immune system to mount an effective response leads to chronic infection.  The authors' understanding of the immune response to HBV is incomplete, largely due to the narrow host restriction of this pathogen and the limitations of existing exptl. models.  The authors have developed a murine model for studying human HBV replication, immunogenicity, and control.  After transfection of hepatocytes in vivo with a replication-competent, over-length, linear HBV genome, viral antigens and replicative intermediates were synthesized and virus was secreted into the blood.  Viral antigens disappeared from the blood as early as 7 days after transfection, coincident with the appearance of antiviral antibodies.  HBV transcripts and replicative intermediates disappeared from the liver by day 15, after the appearance of antiviral CD8 + T cells.  In contrast, the virus persisted for at least 81 days after transfection of NOD/Scid mice, which lack functional T cells, B cells, and natural killer (NK) cells.  Thus, the outcome of hydrodynamic transfection of HBV depends on the host immune response, as it is during a natural infection.  The methods the authors describe will allow the examn. of viral dynamics in a tightly controlled in vivo system, the application of mutagenesis methods to the study of the HBV life cycle in vivo, and the dissection of the immune response to HBV using genetically modified mice whose immunoregulatory and immune effector functions have been deleted or overexpressed.  In addn., this methodol. represents a prototype for the study of other known and to-be-discovered liver-specific pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo98nNA_w5YxrVg90H21EOLACvtfcHk0ljaDIPhCVLycg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D&md5=0c0f40c386fe47f6c57c413fccdaf3e5</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202398599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202398599%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DAlthage%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DChisari%26aufirst%3DF.%2BV.%26atitle%3DHydrodynamic%2520injection%2520of%2520viral%2520DNA%253A%2520A%2520mouse%2520model%2520of%2520acute%2520hepatitis%2520B%2520virus%2520infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D13825%26epage%3D13830%26doi%3D10.1073%2Fpnas.202398599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C.</span></span> <i>Drug metabolism and pharmacokinetics quick guide.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.1007/978-1-4419-5629-3_2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1007%2F978-1-4419-5629-3_2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Khojasteh%2C+S.+C.%3B+Wong%2C+H.%3B+Hop%2C+C.+Drug+metabolism+and+pharmacokinetics+quick+guide.+2011%2C+17%2C+10.1007%2F978-1-4419-5629-3_2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-5629-3_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-5629-3_2%26sid%3Dliteratum%253Aachs%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DWong%26aufirst%3DH.%26aulast%3DHop%26aufirst%3DC.%26jtitle%3DDrug%2520metabolism%2520and%2520pharmacokinetics%2520quick%2520guide.%26date%3D2011%26spage%3D17%26doi%3D10.1007%2F978-1-4419-5629-3_2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sinz, M. W.</span></span> <span> </span><span class="NLM_article-title">Avoiding PXR and CAR activation and CYP3A4 enzyme induction</span>. <i>Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1007/7355_2013_24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1007%2F7355_2013_24" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=159-190&author=M.+W.+Sinz&title=Avoiding+PXR+and+CAR+activation+and+CYP3A4+enzyme+induction&doi=10.1007%2F7355_2013_24"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1007%2F7355_2013_24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F7355_2013_24%26sid%3Dliteratum%253Aachs%26aulast%3DSinz%26aufirst%3DM.%2BW.%26atitle%3DAvoiding%2520PXR%2520and%2520CAR%2520activation%2520and%2520CYP3A4%2520enzyme%2520induction%26jtitle%3DTop.%2520Med.%2520Chem.%26date%3D2013%26volume%3D9%26spage%3D159%26epage%3D190%26doi%3D10.1007%2F7355_2013_24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mase, T.</span></span> <span> </span><span class="NLM_article-title">A general palladium-catalyzed coupling of aryl bromides/triflates and thiols</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4587</span>– <span class="NLM_lpage">4590</span>, <span class="refDoi"> DOI: 10.1021/ol047996t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol047996t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptVartbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=4587-4590&author=T.+Itohauthor=T.+Mase&title=A+general+palladium-catalyzed+coupling+of+aryl+bromides%2Ftriflates+and+thiols&doi=10.1021%2Fol047996t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A general palladium-catalyzed coupling of aryl bromides/triflates and thiols</span></div><div class="casAuthors">Itoh, Takahiro; Mase, Toshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4587-4590</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient palladium-catalyzed carbon-sulfur bond formation reaction of aryl bromides, triflates, and activated aryl chloride have been developed.  The reaction showed tolerance to a wide variety of aryl thiols and alkyl thiols which gave aryl sulfides, e.g., I, in good yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ47wzYjppQ7Vg90H21EOLACvtfcHk0licl5syqu6vsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptVartbg%253D&md5=a58e03f1934099065cf2500c276d89a1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fol047996t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol047996t%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DMase%26aufirst%3DT.%26atitle%3DA%2520general%2520palladium-catalyzed%2520coupling%2520of%2520aryl%2520bromides%252Ftriflates%2520and%2520thiols%26jtitle%3DOrg.%2520Lett.%26date%3D2004%26volume%3D6%26spage%3D4587%26epage%3D4590%26doi%3D10.1021%2Fol047996t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christesen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, Y. Y.</span></span> <span> </span><span class="NLM_article-title">A simple and highly effective oxidative chlorination protocol for the preparation of arenesulfonyl chlorides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">418</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2009.11.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.tetlet.2009.11.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=418-421&author=Y.+M.+Puauthor=A.+Christesenauthor=Y.+Y.+Ku&title=A+simple+and+highly+effective+oxidative+chlorination+protocol+for+the+preparation+of+arenesulfonyl+chlorides&doi=10.1016%2Fj.tetlet.2009.11.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A simple and highly effective oxidative chlorination protocol for the preparation of arenesulfonyl chlorides</span></div><div class="casAuthors">Pu, Yu-Ming; Christesen, Alan; Ku, Yi-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">418-421</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">2,4-Dichloro-5,5-dimethylhydantoin (DCDMH) was found to be a mild and efficient reagent for the direct oxidative conversion of sulfur compds. to the corresponding arenesulfonyl chlorides in good to excellent yields through oxidative chlorination.  The method is suitable for many types of sulfur substrates (thiols, disulfides, and benzylic sulfides).  The overall process is simple, practical, and it provides convenient access to a variety of aryl or heteroarylsulfonyl chlorides.  The mild reaction conditions and the broad substrate scope render this method attractive and complementary to existing syntheses of aryl or heteroarylsulfonyl chlorides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAa01G0iIlebVg90H21EOLACvtfcHk0licl5syqu6vsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO&md5=99e2389d65e31bc57a2a3c740b59df98</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2009.11.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2009.11.047%26sid%3Dliteratum%253Aachs%26aulast%3DPu%26aufirst%3DY.%2BM.%26aulast%3DChristesen%26aufirst%3DA.%26aulast%3DKu%26aufirst%3DY.%2BY.%26atitle%3DA%2520simple%2520and%2520highly%2520effective%2520oxidative%2520chlorination%2520protocol%2520for%2520the%2520preparation%2520of%2520arenesulfonyl%2520chlorides%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26spage%3D418%26epage%3D421%26doi%3D10.1016%2Fj.tetlet.2009.11.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlachou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pante, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helenius, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kann, M.</span></span> <span> </span><span class="NLM_article-title">Nuclear import of hepatitis B virus capsids and release of the viral genome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">9849</span>– <span class="NLM_lpage">9854</span>, <span class="refDoi"> DOI: 10.1073/pnas.1730940100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.1730940100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=12909718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvVejtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=9849-9854&author=B.+Rabeauthor=A.+Vlachouauthor=N.+Panteauthor=A.+Heleniusauthor=M.+Kann&title=Nuclear+import+of+hepatitis+B+virus+capsids+and+release+of+the+viral+genome&doi=10.1073%2Fpnas.1730940100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear import of hepatitis B virus capsids and release of the viral genome</span></div><div class="casAuthors">Rabe, Birgit; Vlachou, Angelika; Pante, Nelly; Helenius, Ari; Kann, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9849-9854</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">While studying the import of the hepatitis B virus genome into the nucleus of permeabilized tissue culture cells, we found that viral capsids were imported in intact form through the nuclear pore into the nuclear basket.  Import depended on phosphorylation of the capsid protein and was mediated by the cellular transport receptors importin α and β.  Virus-derived capsids that contained the mature viral genome were able to release the viral DNA and capsid protein into the nucleoplasm.  The uncoating reaction was independent of Ran, a GTP-binding enzyme responsible for dissocg. other imported cargoes from the inner face of the nuclear pore.  Immature capsids that did not contain the mature viral genome reached the basket but did not release capsid proteins nor immature genomes into the nucleoplasm.  The different fate of mature and immature capsids after passing the nuclear pore indicates that the outcome of a nuclear import event may be regulated within the nuclear basket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp7BEG_HbaGLVg90H21EOLACvtfcHk0ljYaNSAFfOBOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvVejtbg%253D&md5=9b27d82ed78f3d24d7926919ea199f0c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1730940100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1730940100%26sid%3Dliteratum%253Aachs%26aulast%3DRabe%26aufirst%3DB.%26aulast%3DVlachou%26aufirst%3DA.%26aulast%3DPante%26aufirst%3DN.%26aulast%3DHelenius%26aufirst%3DA.%26aulast%3DKann%26aufirst%3DM.%26atitle%3DNuclear%2520import%2520of%2520hepatitis%2520B%2520virus%2520capsids%2520and%2520release%2520of%2520the%2520viral%2520genome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D9849%26epage%3D9854%26doi%3D10.1073%2Fpnas.1730940100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljYaNSAFfOBOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torosyan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebart, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumanian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">An aggregation advisor for ligand discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7076</span>– <span class="NLM_lpage">7087</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01105</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01105" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7076-7087&author=J.+J.+Irwinauthor=D.+Duanauthor=H.+Torosyanauthor=A.+K.+Doakauthor=K.+T.+Ziebartauthor=T.+Sterlingauthor=G.+Tumanianauthor=B.+K.+Shoichet&title=An+aggregation+advisor+for+ligand+discovery&doi=10.1021%2Facs.jmedchem.5b01105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">An Aggregation Advisor for Ligand Discovery</span></div><div class="casAuthors">Irwin, John J.; Duan, Da; Torosyan, Hayarpi; Doak, Allison K.; Ziebart, Kristin T.; Sterling, Teague; Tumanian, Gurgen; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7076-7087</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation of org. mols. is the dominant mechanism for artifactual inhibition of proteins, and controls against it are widely deployed.  Notwithstanding an increasingly detailed understanding of this phenomenon, a method to reliably predict aggregation has remained elusive.  Correspondingly, active mols. that act via aggregation continue to be found in early discovery campaigns and remain common in the literature.  Over the past decade, over 12 thousand aggregating org. mols. have been identified, potentially enabling a precedent-based approach to match known aggregators with new mols. that may be expected to aggregate and lead to artifacts.  We investigate an approach that uses lipophilicity, affinity, and similarity to known aggregators to advise on the likelihood that a candidate compd. is an aggregator.  In prospective exptl. testing, five of seven new mols. with Tanimoto coeffs. (Tc's) between 0.95 and 0.99 to known aggregators aggregated at relevant concns.  Ten of 19 with Tc's between 0.94 and 0.90 and three of seven with Tc's between 0.89 and 0.85 also aggregated.  Another three of the predicted compds. aggregated at higher concns.  This method finds that 61 827 or 5.1% of the ligands acting in the 0.1 to 10 μM range in the medicinal chem. literature are at least 85% similar to a known aggregator with these phys. properties and may aggregate at relevant concns.  Intriguingly, only 0.73% of all drug-like com. available compds. resemble the known aggregators, suggesting that colloidal aggregators are enriched in the literature.  As a percentage of the literature, aggregator-like compds. have increased 9-fold since 1995, partly reflecting the advent of high-throughput and virtual screens against mol. targets.  Emerging from this study is an aggregator advisor database and tool (http://advisor.bkslab.org), free to the community, that may help distinguish between fruitful and artifactual screening hits acting by this mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTKGJSfIz30bVg90H21EOLACvtfcHk0ljYaNSAFfOBOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM&md5=d3112f9f3a57a2ac61a4b32f31778b1d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01105%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DDuan%26aufirst%3DD.%26aulast%3DTorosyan%26aufirst%3DH.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DZiebart%26aufirst%3DK.%2BT.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DTumanian%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DAn%2520aggregation%2520advisor%2520for%2520ligand%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7076%26epage%3D7087%26doi%3D10.1021%2Facs.jmedchem.5b01105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arif  Mermer</span>, <span class="hlFld-ContribAuthor ">Turgut  Keles</span>, <span class="hlFld-ContribAuthor ">Yakup  Sirin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent studies of nitrogen containing heterocyclic compounds as novel antiviral agents: A review. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>114 </em>, 105076. <a href="https://doi.org/10.1016/j.bioorg.2021.105076" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105076</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105076%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DRecent%252Bstudies%252Bof%252Bnitrogen%252Bcontaining%252Bheterocyclic%252Bcompounds%252Bas%252Bnovel%252Bantiviral%252Bagents%25253A%252BA%252Breview%26aulast%3DMermer%26aufirst%3DArif%26date%3D2021%26volume%3D114%26spage%3D105076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sameera  Senaweera</span>, <span class="hlFld-ContribAuthor ">Haijuan  Du</span>, <span class="hlFld-ContribAuthor ">Huanchun  Zhang</span>, <span class="hlFld-ContribAuthor ">Karen A.  Kirby</span>, <span class="hlFld-ContribAuthor ">Philip R.  Tedbury</span>, <span class="hlFld-ContribAuthor ">Jiashu  Xie</span>, <span class="hlFld-ContribAuthor ">Stefan G.  Sarafianos</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of New Small Molecule Hits as Hepatitis B Virus Capsid Assembly Modulators: Structure and Pharmacophore-Based Approaches. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (5)
                                     , 770. <a href="https://doi.org/10.3390/v13050770" title="DOI URL">https://doi.org/10.3390/v13050770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13050770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13050770%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DDiscovery%252Bof%252BNew%252BSmall%252BMolecule%252BHits%252Bas%252BHepatitis%252BB%252BVirus%252BCapsid%252BAssembly%252BModulators%25253A%252BStructure%252Band%252BPharmacophore-Based%252BApproaches%26aulast%3DSenaweera%26aufirst%3DSameera%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D5%26spage%3D770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yujie  Ren</span>, <span class="hlFld-ContribAuthor ">Yue  Ma</span>, <span class="hlFld-ContribAuthor ">Srinivasulu  Cherukupalli</span>, <span class="hlFld-ContribAuthor ">John E.  Tavis</span>, <span class="hlFld-ContribAuthor ">Luis  Menéndez-Arias</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and optimization of benzenesulfonamides-based hepatitis B virus capsid modulators via contemporary medicinal chemistry strategies. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112714. <a href="https://doi.org/10.1016/j.ejmech.2020.112714" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112714</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112714%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Boptimization%252Bof%252Bbenzenesulfonamides-based%252Bhepatitis%252BB%252Bvirus%252Bcapsid%252Bmodulators%252Bvia%252Bcontemporary%252Bmedicinal%252Bchemistry%252Bstrategies%26aulast%3DRen%26aufirst%3DYujie%26date%3D2020%26volume%3D206%26spage%3D112714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0027.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0027.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of CAMs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of novel CAMs through conformation constraint. (A) Crystal structure of Cps binding with SBA-R01 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T2P">5T2P</a>). (B) Rational design of CAMs. (C) Merging of SBA-R01 and HAP-R01 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>) in the binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>6</b> and its effect on capsid assembly. (A) Structure of compound <b>6</b>, (B and C) Compound <b>6</b> accelerated capsid assembly as detected by EM (B) and SEC (C). (D) Calculation of the peak AUC and SEC peak AUC ratio in C. Cp149 protein (5 μM) was incubated with <b>6</b> (10 μM) in a buffer containing 150 mM NaCl and 50 mM HEPES (pH 7.4) at 37 °C overnight prior to EM or SEC detection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <i>In vitro</i> effects of compounds <b>56</b> and <b>59</b> in HepAD38 and HepG2.2.15 cells. (A) EM analysis of the effects on capsid assembly. (B and C) Changes of HBV extracellular and intracellular parameters in HepAD38 and HepG2.2.15 cells after treatment with AT130, 3TC, GLS4, <b>56</b>, and <b>59</b> at 2 μM. Cells were treated with compounds or DMSO for 4 days. The medium was replaced every 2 days. DNA and RNA were quantified with qPCR or qRT-PCR. HBeAg and HBsAg were quantified by ELISA (VC, vehicle control). The results are presented as the mean ± SD. Statistics were determined by two-tailed <i>t</i> test. *, <i>P</i> < 0.05; **, <i>P</i> < 0.01; ***, <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Anti-HBV activity in an HDI HBV mouse model. (A) Serum HBV DNA copies after hydrodynamic injection. (B) Liver HBV DNA on the seventh day after hydrodynamic injection. <i>N</i> = 5; *, <i>P</i> < 0.05; **, <i>P</i> < 0.01; ***, <i>P</i> < 0.001; ****, <i>P</i> < 0.0001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0006.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Pharmacokinetics of compounds <b>56</b> and <b>67</b> in male C57BL/6 mice. (A) Plasma concentration of <b>56</b> after administration of <b>56</b> and its prodrug <b>67</b>. (B) Liver concentration of <b>56</b> and its prodrug (<i>N</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0007.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Predicted binding mode of <b>56</b>. (A) Docking of <b>56</b> with HBV core protein crystal structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>). (B) Merging of <b>56</b> and NVR 3-778 at the binding site. (C) Merging of <b>56</b> and HAP-R01 at the binding site.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0009.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route for the Indazole Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnSH (1.5 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5% mol), xantphos (10% mol), DIPEA (2 equiv), 1,4-dioxane, 90 °C; (b) i. 1,3-dibromo-5,5-dimethylhydantoin (2 equiv), −30 °C, CH<sub>3</sub>CN, H<sub>2</sub>O, AcOH; ii. 4-hydroxypiperidine (2 equiv), Et<sub>3</sub>N (3 equiv), DCM, room temperature; (c) NBS (1.5 equiv), or 1,3-dibromo-5,5-dimethylhydantoin, CH<sub>3</sub>CN/AcOH; (d) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; (e) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; ii. LiOH (3 equiv), THF, H<sub>2</sub>O, room temperature.</p></p></figure><figure data-id="sch2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compound <b>1</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnSH (1.5 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (5% mol), xantphos (10% mol), DIPEA (2 equiv), 1,4-dioxane, 90 °C, 92%; (b) i. 5,5-dimethylhydantoin (2 equiv), −30 °C, CH<sub>3</sub>CN, H<sub>2</sub>O, AcOH; ii. 4-hydroxypiperidine (2 equiv), Et<sub>3</sub>N (3 equiv), DCM, room temperature, 41%; (c) NBS (1.1 equiv), CHCl<sub>3</sub>, 60 °C; (d) i. DHP (2 equiv), TsOH H<sub>2</sub>O (0.3 equiv), EA, reflux; ii. Ac<sub>2</sub>O (1 equiv), 86% for three steps; (e) i. 3-chloro-4-fluoroaniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 110 °C; ii. LiOH (3 equiv), THF, H<sub>2</sub>O, rt, 89%; (f) HCl, EtOH, room temperature, quantitative.</p></p></figure><figure data-id="sch3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route of 1<i>H</i>-Pyrazolo[3,4-<i>c</i>]pyridine Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Raney Ni, H<sub>2</sub>, methanol, room temperaturet, quantitative; (b) Ac<sub>2</sub>O (3 equiv), PhCH<sub>3</sub>, 100 °C, 85%; (c) isoamyl nitrite (2.5 equiv), Ac<sub>2</sub>O (2 equiv), KOAc (2 equiv), 18-crown-6 (0.1 equiv), PhCH<sub>3</sub>, 80 °C, then NaOH (4 equiv), room temperature, 63%; (d) BnSH (1.5 equiv), Pd<sub>2</sub> (dba)<sub>3</sub> (5% mol), xantphos (5% mol), DIPEA (2 equiv), 1,4-dioxane, 90 °C, 87%; (e) cyclopropylboronic acid (2 equiv), Cu(OAc)<sub>2</sub> (1 equiv), 2,2′-bipyridine (1 equiv), Na<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,2-DCE, 70 °C, 51%; (f) i. 5,5-dimethylhydantoin (2 equiv), −15 °C, CH<sub>3</sub>CN, H<sub>2</sub>O (1% v/v), AcOH (1% v/v); ii. 4-hydroxypiperidine (for <b>83a</b>, 2 equiv) or morpholine (for <b>83b</b>, 2 equiv), Et<sub>3</sub>N (3 equiv), DCM, room temperature; (g) for <b>84a</b> i. Ac<sub>2</sub>O (5 equiv), pyridine (4 equiv), ethyl acetate, reflux; ii. NBS (1.5 equiv), TFA/AcOH (1:1), 50 °C, 50% for 2 steps; (h) for <b>84b</b> 1,3-dibromo-5,5-dimethylhydantoin (1.5 equiv), CH<sub>3</sub>CN/AcOH (1:1), 50 °C, 59% for 2 steps; (i) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; (j) corresponding aniline (1.5 equiv), Pd(OAc)<sub>2</sub> (5% mol), xantphos (10% mol), Cs<sub>2</sub>CO<sub>3</sub> (2 equiv), 1,4-dioxane, 130 °C; ii. LiOH (3 equiv), THF, H<sub>2</sub>O, room temperature.</p></p></figure><figure data-id="sch4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0012.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>5</b>, <b>27</b>, and <b>28</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0013.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of the Prodrugs and Preparation of the Salt Form<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) amino acid (3 equiv), DCC (3 equiv), DMAP (1.2 equiv), DCM, 40 °C, ∼90%; (b) for <b>61</b>–<b>64</b>, TFA/DCM (1/1), room temperature; for <b>66</b>–<b>70</b>, the corresponding acid, DCM or ethyl acetate.</p></p></figure><figure data-id="fig8" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/medium/jm0c00292_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0008.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Summary of the structure–activity relationship (SAR).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00292/20200604/images/large/jm0c00292_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00292&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i121">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16050" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16050" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span> </span><span class="NLM_article-title">Hepatitis
B</span>. <a href="https://www.who.int/news-room/fact-sheets/detail/hepatitis-b" class="extLink">https://www.who.int/news-room/fact-sheets/detail/hepatitis-b</a>, <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Hepatitis%0AB.+https%3A%2F%2Fwww.who.int%2Fnews-room%2Ffact-sheets%2Fdetail%2Fhepatitis-b%2C+World+Health+Organization%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DHepatitis%250AB%26pub%3DWorld%2520Health%2520Organization%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seto, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlotsky, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, M. F.</span></span> <span> </span><span class="NLM_article-title">Chronic hepatitis B virus infection</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>392</i></span>,  <span class="NLM_fpage">2313</span>– <span class="NLM_lpage">2324</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(18)31865-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS0140-6736%2818%2931865-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=30496122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A280%3ADC%252BB3crmslGhug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=392&publication_year=2018&pages=2313-2324&author=W.+K.+Setoauthor=Y.+R.+Loauthor=J.+M.+Pawlotskyauthor=M.+F.+Yuen&title=Chronic+hepatitis+B+virus+infection&doi=10.1016%2FS0140-6736%2818%2931865-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Chronic hepatitis B virus infection</span></div><div class="casAuthors">Seto Wai-Kay; Lo Ying-Ru; Pawlotsky Jean-Michel; Yuen Man-Fung</div><div class="citationInfo"><span class="NLM_cas:title">Lancet (London, England)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">392</span>
        (<span class="NLM_cas:issue">10161</span>),
    <span class="NLM_cas:pages">2313-2324</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Chronic hepatitis B virus infection is a global public health threat that causes considerable liver-related morbidity and mortality.  It is acquired at birth or later via person-to-person transmission.  Vaccination effectively prevents infection and chronic hepatitis B virus carriage.  In chronically infected patients, an elevated serum hepatitis B virus DNA concentration is the main risk factor for disease progression, although there are other clinical and viral parameters that influence disease outcomes.  In addition to liver biochemistry, virological markers, and abdominal ultrasonography, non-invasive assessment of liver fibrosis is emerging as an important assessment modality.  Long-term nucleos(t)ide-analogue therapy is safe and well tolerated, achieves potent viral suppression, and reduces the incidence of liver-related complications.  However, a need to optimise management remains.  Promising novel therapies are at the developmental stage.  With current vaccines, therapies, and an emphasis on improving linkage to care, WHO's goal of eliminating hepatitis B virus as a global health threat by 2030 is achievable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSCvI0MsROihkewzGPswiWOfW6udTcc2eb4B_Zzp4JtsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3crmslGhug%253D%253D&md5=e252e1c05793f99a1328c9d969bc5ba3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2818%2931865-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252818%252931865-8%26sid%3Dliteratum%253Aachs%26aulast%3DSeto%26aufirst%3DW.%2BK.%26aulast%3DLo%26aufirst%3DY.%2BR.%26aulast%3DPawlotsky%26aufirst%3DJ.%2BM.%26aulast%3DYuen%26aufirst%3DM.%2BF.%26atitle%3DChronic%2520hepatitis%2520B%2520virus%2520infection%26jtitle%3DLancet%26date%3D2018%26volume%3D392%26spage%3D2313%26epage%3D2324%26doi%3D10.1016%2FS0140-6736%2818%2931865-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoulim, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span> <span> </span><span class="NLM_article-title">Antiviral therapies and prospects for a cure of chronic hepatitis B</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">21</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a021501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1101%2Fcshperspect.a021501" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=21&author=F.+Zoulimauthor=D.+Durantel&title=Antiviral+therapies+and+prospects+for+a+cure+of+chronic+hepatitis+B&doi=10.1101%2Fcshperspect.a021501"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a021501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a021501%26sid%3Dliteratum%253Aachs%26aulast%3DZoulim%26aufirst%3DF.%26aulast%3DDurantel%26aufirst%3DD.%26atitle%3DAntiviral%2520therapies%2520and%2520prospects%2520for%2520a%2520cure%2520of%2520chronic%2520hepatitis%2520B%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2015%26volume%3D5%26spage%3D21%26doi%3D10.1101%2Fcshperspect.a021501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tuttleman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pourcel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summers, J.</span></span> <span> </span><span class="NLM_article-title">Formation of the pool of covalently closed circular viral-DNA in hepadnavirus-infected cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(86)90602-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2F0092-8674%2886%2990602-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=3768961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaL2sXhtlynug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1986&pages=451-460&author=J.+S.+Tuttlemanauthor=C.+Pourcelauthor=J.+Summers&title=Formation+of+the+pool+of+covalently+closed+circular+viral-DNA+in+hepadnavirus-infected+cells&doi=10.1016%2F0092-8674%2886%2990602-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of the pool of covalently closed circular viral DNA in hepadnavirus-infected cells</span></div><div class="casAuthors">Tuttleman, Jan S.; Pourcel, Christine; Summers, Jesse</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">451-60</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Covalently closed circular (CCC) double-stranded DNA believed to be the transcriptional template for duck hepatitis B virus (DHBV) is amplified in aging primary cultures of hepatocytes from congenitally infected ducklings.  Anal. of 5-bromodeoxyuridine-labeled heavy/light CCC DNA shows that the relaxed circular DNA synthesized in the cytoplasm by reverse transcription is the predominant precursor to the amplified pool of nuclear viral CCC DNA.  In vitro infection of uninfected hepatocyte cultures with DHBV demonstrates that a similar 50-fold amplification of CCC DNA occurs during an early stage in the infection before virus prodn.  This amplification allows the establishment of a pool of transcriptional templates in the cell without the need for semiconservative replication or multiple rounds of infection.  This process may account for the ability of hepadnavirus-infected cells persistently to produce virus particles in the absence of stable integration of viral DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeIiXXyDXnM7Vg90H21EOLACvtfcHk0liJOC1feHdxvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhtlynug%253D%253D&md5=4401425e1e3253a086d6c5c2366d0f64</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2886%2990602-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252886%252990602-1%26sid%3Dliteratum%253Aachs%26aulast%3DTuttleman%26aufirst%3DJ.%2BS.%26aulast%3DPourcel%26aufirst%3DC.%26aulast%3DSummers%26aufirst%3DJ.%26atitle%3DFormation%2520of%2520the%2520pool%2520of%2520covalently%2520closed%2520circular%2520viral-DNA%2520in%2520hepadnavirus-infected%2520cells%26jtitle%3DCell%26date%3D1986%26volume%3D47%26spage%3D451%26epage%3D460%26doi%3D10.1016%2F0092-8674%2886%2990602-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Werle-Lapostolle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locarnini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wursthorn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trepo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney IV, W. E.</span></span> <span> </span><span class="NLM_article-title">Persistence of cccDNA during the natural history of chronic hepatitis B and decline during adefovir dipivoxil therapy</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1758</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2004.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1053%2Fj.gastro.2004.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=15188170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsFaqs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=1750-1758&author=B.+Werle-Lapostolleauthor=S.+Bowdenauthor=S.+Locarniniauthor=K.+Wursthornauthor=J.+Petersenauthor=G.+Lauauthor=C.+Trepoauthor=P.+Marcellinauthor=Z.+Goodmanauthor=W.+E.+Delaney+IV&title=Persistence+of+cccDNA+during+the+natural+history+of+chronic+hepatitis+B+and+decline+during+adefovir+dipivoxil+therapy&doi=10.1053%2Fj.gastro.2004.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Persistence of cccDNA during the natural history of chronic hepatitis B and decline during adefovir dipivoxil therapy</span></div><div class="casAuthors">Werle-Lapostolle, Bettina; Bowden, Scott; Locarnini, Stephen; Wursthorn, Karsten; Petersen, Jorg; Lau, George; Trepo, Christian; Marcellin, Patrick; Goodman, Zachary; Delaney, William E., IV; Xiong, Shelly; Brosgart, Carol L.; Chen, Shan-Shan; Gibbs, Craig S.; Zoulim, Fabien</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1758</span>CODEN:
                <span class="NLM_cas:coden">GASTAB</span>;
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Background & Aims: Hepatitis B virus (HBV) covalently closed circular DNA (cccDNA) is a unique episomal replicative intermediate responsible for persistent infection of hepatocytes.  Tech. constraints have hampered the direct study of cccDNA maintenance and clearance mechanisms in patients.  The aim of this study was to develop a sensitive and specific assay for quantifying cccDNA in biopsy samples from chronic hepatitis B patients during different natural history phases and in patients undergoing antiviral therapy.  Methods: Intrahepatic cccDNA levels were quantified by a specific real-time PCR assay.  Ninety-eight liver biopsy samples from patients in the major phases of the natural history of chronic hepatitis B and 32 pairs of samples from patients receiving adefovir dipivoxil (ADV) therapy were assessed.  Results: cccDNA was detected, at levels ranging over 3 orders of magnitude, in patients in different phases of the natural history of chronic hepatitis B.  CccDNA levels were strongly correlated with levels of total intracellular HBV DNA and serum HBV DNA.  Forty-eight weeks of ADV therapy resulted in a significant 0.8 log decrease in cccDNA copies/cell.  Changes in cccDNA were correlated with a similar redn. in serum HBsAg titer but not with a decrease in the no. of HBV antigen-pos. cells during ADV treatment.  Conclusions: cccDNA persists throughout the natural history of chronic hepatitis B, even in patients with serol. evidence of viral clearance.  Long-term ADV therapy significantly decreased cccDNA levels by a primarily noncytolytic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4L4TAyOGVxrVg90H21EOLACvtfcHk0liJOC1feHdxvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsFaqs78%253D&md5=3a390a5d3f59436b73007ba73caac1a9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2004.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2004.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DWerle-Lapostolle%26aufirst%3DB.%26aulast%3DBowden%26aufirst%3DS.%26aulast%3DLocarnini%26aufirst%3DS.%26aulast%3DWursthorn%26aufirst%3DK.%26aulast%3DPetersen%26aufirst%3DJ.%26aulast%3DLau%26aufirst%3DG.%26aulast%3DTrepo%26aufirst%3DC.%26aulast%3DMarcellin%26aufirst%3DP.%26aulast%3DGoodman%26aufirst%3DZ.%26aulast%3DDelaney%2BIV%26aufirst%3DW.%2BE.%26atitle%3DPersistence%2520of%2520cccDNA%2520during%2520the%2520natural%2520history%2520of%2520chronic%2520hepatitis%2520B%2520and%2520decline%2520during%2520adefovir%2520dipivoxil%2520therapy%26jtitle%3DGastroenterology%26date%3D2004%26volume%3D126%26spage%3D1750%26epage%3D1758%26doi%3D10.1053%2Fj.gastro.2004.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucifora, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisinger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprinzl, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppensteiner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makowska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remouchamps, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thasler, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huser, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browning, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dejardin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schindler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heikenwalder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protzer, U.</span></span> <span> </span><span class="NLM_article-title">Specific and nonhepatotoxic degradation of nuclear hepatitis B virus cccDNA</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1228</span>, <span class="refDoi"> DOI: 10.1126/science.1243462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1126%2Fscience.1243462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=24557838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjvVygsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2014&pages=1221-1228&author=J.+Luciforaauthor=Y.+C.+Xiaauthor=F.+Reisingerauthor=K.+Zhangauthor=D.+Stadlerauthor=X.+M.+Chengauthor=M.+F.+Sprinzlauthor=H.+Koppensteinerauthor=Z.+Makowskaauthor=T.+Volzauthor=C.+Remouchampsauthor=W.+M.+Chouauthor=W.+E.+Thaslerauthor=N.+Huserauthor=D.+Durantelauthor=T.+J.+Liangauthor=C.+Munkauthor=M.+H.+Heimauthor=J.+L.+Browningauthor=E.+Dejardinauthor=M.+Dandriauthor=M.+Schindlerauthor=M.+Heikenwalderauthor=U.+Protzer&title=Specific+and+nonhepatotoxic+degradation+of+nuclear+hepatitis+B+virus+cccDNA&doi=10.1126%2Fscience.1243462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Specific and Nonhepatotoxic Degradation of Nuclear Hepatitis B Virus cccDNA</span></div><div class="casAuthors">Lucifora, Julie; Xia, Yuchen; Reisinger, Florian; Zhang, Ke; Stadler, Daniela; Cheng, Xiaoming; Sprinzl, Martin F.; Koppensteiner, Herwig; Makowska, Zuzanna; Volz, Tassilo; Remouchamps, Caroline; Chou, Wen-Min; Thasler, Wolfgang E.; Hueser, Norbert; Durantel, David; Liang, T. Jake; Muenk, Carsten; Heim, Markus H.; Browning, Jeffrey L.; Dejardin, Emmanuel; Dandri, Maura; Schindler, Michael; Heikenwalder, Mathias; Protzer, Ulrike</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">6176</span>),
    <span class="NLM_cas:pages">1221-1228</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Current antiviral agents can control but not eliminate hepatitis B virus (HBV), because HBV establishes a stable nuclear covalently closed circular DNA (cccDNA).  Interferon-α treatment can clear HBV but is limited by systemic side effects.  The authors describe how interferon-α can induce specific degrdn. of the nuclear viral DNA without hepatotoxicity and propose lymphotoxin-β receptor activation as a therapeutic alternative.  Interferon-α and lymphotoxin-β receptor activation up-regulated APOBEC3A and APOBEC3B cytidine deaminases, resp., in HBV-infected cells, primary hepatocytes, and human liver needle biopsies.  HBV core protein mediated the interaction with nuclear cccDNA, resulting in cytidine deamination, apurinic/apyrimidinic site formation, and finally cccDNA degrdn. that prevented HBV reactivation.  Genomic DNA was not affected.  Thus, inducing nuclear deaminases-for example, by lymphotoxin-β receptor activation-allows the development of new therapeutics that, in combination with existing antivirals, may cure hepatitis B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrONjt8Ppl67rVg90H21EOLACvtfcHk0liJOC1feHdxvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjvVygsrY%253D&md5=18855f5a52c98d6167a7a8468d55e9ce</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.1243462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1243462%26sid%3Dliteratum%253Aachs%26aulast%3DLucifora%26aufirst%3DJ.%26aulast%3DXia%26aufirst%3DY.%2BC.%26aulast%3DReisinger%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DStadler%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DX.%2BM.%26aulast%3DSprinzl%26aufirst%3DM.%2BF.%26aulast%3DKoppensteiner%26aufirst%3DH.%26aulast%3DMakowska%26aufirst%3DZ.%26aulast%3DVolz%26aufirst%3DT.%26aulast%3DRemouchamps%26aufirst%3DC.%26aulast%3DChou%26aufirst%3DW.%2BM.%26aulast%3DThasler%26aufirst%3DW.%2BE.%26aulast%3DHuser%26aufirst%3DN.%26aulast%3DDurantel%26aufirst%3DD.%26aulast%3DLiang%26aufirst%3DT.%2BJ.%26aulast%3DMunk%26aufirst%3DC.%26aulast%3DHeim%26aufirst%3DM.%2BH.%26aulast%3DBrowning%26aufirst%3DJ.%2BL.%26aulast%3DDejardin%26aufirst%3DE.%26aulast%3DDandri%26aufirst%3DM.%26aulast%3DSchindler%26aufirst%3DM.%26aulast%3DHeikenwalder%26aufirst%3DM.%26aulast%3DProtzer%26aufirst%3DU.%26atitle%3DSpecific%2520and%2520nonhepatotoxic%2520degradation%2520of%2520nuclear%2520hepatitis%2520B%2520virus%2520cccDNA%26jtitle%3DScience%26date%3D2014%26volume%3D343%26spage%3D1221%26epage%3D1228%26doi%3D10.1126%2Fscience.1243462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, G. K.K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piratvisuth, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongsawat, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooksley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gane, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fried, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flisiak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCloud, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pluck, N.</span></span> <span> </span><span class="NLM_article-title">Peginterferon alfa-2a, lamivudine, and the combination for HBeAg-positive chronic hepatitis B</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">2682</span>– <span class="NLM_lpage">2695</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa043470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1056%2FNEJMoa043470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=15987917" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1SktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=2682-2695&author=G.+K.K.+Lauauthor=T.+Piratvisuthauthor=K.+X.+Luoauthor=P.+Marcellinauthor=S.+Thongsawatauthor=G.+Cooksleyauthor=E.+Ganeauthor=M.+W.+Friedauthor=W.+C.+Chowauthor=S.+W.+Paikauthor=W.+Y.+Changauthor=T.+Bergauthor=R.+Flisiakauthor=P.+McCloudauthor=N.+Pluck&title=Peginterferon+alfa-2a%2C+lamivudine%2C+and+the+combination+for+HBeAg-positive+chronic+hepatitis+B&doi=10.1056%2FNEJMoa043470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Peginterferon alfa-2a, lamivudine, and the combination for HBeAg-positive chronic hepatitis B</span></div><div class="casAuthors">Lau, George K. K.; Piratvisuth, Teerha; Luo, Kang Xian; Marcellin, Patrick; Thongswat, Satawat; Cooksley, Graham; Gane, Edward; Fried, Michael W.; Chow, Wan Cheng; Paik, Sueng Woon; Chang, Wen Yu; Berg, Thomas; Flisiak, Robert; McCloud, Philip; Pluck, Nigel; Balan, V.; Baruch, Y.; Boyer, N.; Box, T.; Burak, K.; Chao, Y.-C.; Cheinquer, H.; Chung, K.-W.; Chung, Y.-H.; Chutaputti, A.; Fawaz, K.; Feinman, V.; Girgrah, N.; Gish, R.; Gitlin, N.; Goeser, T.; Goncales, F., Jr.; Guan, R.; Haeussinger, D.; Halota, W.; Han, K,-H.; Heim, M.; Horban, A.; Jia, J.-D.; Jin, R.; Jung, M.-C.; Lai, M.-Y.; Lee, A.; Lee, S.-D.; Lei, B.-J.; Liaw, Y.-F.; Lok, A.; Lu, Z.-M.; Luengrojanakul, P.; Lurie, Y.; Mahachai, V.; Manns, M.; Martin, P.; Parana, R.; Pawlowska, M.; Schmidt, W.; Sette, H., Jr.; Smith, C.; Trepo, C.; Tsai, N.; Tung, B.; Tur-Kaspa, R.; Wan, M.-B.; Wang, Q.-H.; Xu, D.-Z.; Yao, G.-B.; Yao, J.-L.; Yin, Y.-K.; Yu, Y.; Zhang, H.-F.; Zhao, Y.-R.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">2682-2695</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Current treatments for chronic hepatitis B are suboptimal.  In the search for improved therapies, we compared the efficacy and safety of pegylated interferon alfa plus lamivudine, pegylated interferon alfa without lamivudine, and lamivudine alone for the treatment of hepatitis B e antigen (HBeAg)-pos. chronic hepatitis B.  A total of 814 patients with HBeAg-pos. chronic hepatitis B received either peginterferon alfa-2a (180 μg once weekly) plus oral placebo, peginterferon alfa-2a plus lamivudine (100 mg daily), or lamivudine alone.  The majority of patients in the study were Asian (87 %).  Most patients were infected with hepatitis B virus (HBV) genotype B or C.  Patients were treated for 48 wk and followed for an addnl. 24 wk.  After 24 wk of follow-up, significantly more patients who received peginterferon alfa-2a monotherapy or peginterferon alfa-2a plus lamivudine than those who received lamivudine monotherapy had HBeAg seroconversion (32 % vs. 19 % [P < 0.001] and 27 % vs. 19 % [P = 0.02], resp.) or HBV DNA levels below 100,000 copies per mL (32 % vs. 22 % [P = 0.01] and 34 % vs. 22 % [P = 0.003], resp.).  Sixteen patients receiving peginterferon alfa-2a (alone or in combination) had hepatitis B surface antigen (HBsAg) seroconversion, as compared with 0 in the group receiving lamivudine alone (P = 0.001).  The most common adverse events were those known to occur with therapies based on interferon alfa.  Serious adverse events occurred in 4 %, 6 %, and 2 % of patients receiving peginterferon alfa-2a monotherapy, combination therapy, and lamivudine monotherapy, resp.  Two patients receiving lamivudine monotherapy had irreversible liver failure after the cessation of treatment - one underwent liver transplantation, and the other died.  In patients with HBeAg-pos. chronic hepatitis B, peginterferon alfa-2a offers superior efficacy over lamivudine, on the basis of HBeAg seroconversion, HBV DNA suppression, and HBsAg seroconversion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosvuGLPwiN87Vg90H21EOLACvtfcHk0ljaJOF1NCpMCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1SktL8%253D&md5=aac0f7d1a47cb9d5cc2e81060d8250cf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa043470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa043470%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DG.%2BK.K.%26aulast%3DPiratvisuth%26aufirst%3DT.%26aulast%3DLuo%26aufirst%3DK.%2BX.%26aulast%3DMarcellin%26aufirst%3DP.%26aulast%3DThongsawat%26aufirst%3DS.%26aulast%3DCooksley%26aufirst%3DG.%26aulast%3DGane%26aufirst%3DE.%26aulast%3DFried%26aufirst%3DM.%2BW.%26aulast%3DChow%26aufirst%3DW.%2BC.%26aulast%3DPaik%26aufirst%3DS.%2BW.%26aulast%3DChang%26aufirst%3DW.%2BY.%26aulast%3DBerg%26aufirst%3DT.%26aulast%3DFlisiak%26aufirst%3DR.%26aulast%3DMcCloud%26aufirst%3DP.%26aulast%3DPluck%26aufirst%3DN.%26atitle%3DPeginterferon%2520alfa-2a%252C%2520lamivudine%252C%2520and%2520the%2520combination%2520for%2520HBeAg-positive%2520chronic%2520hepatitis%2520B%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2005%26volume%3D352%26spage%3D2682%26epage%3D2695%26doi%3D10.1056%2FNEJMoa043470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span> <span> </span><span class="NLM_article-title">Treatment of chronic hepatitis B virus infection using small molecule modulators of nucleocapsid assembly: recent advances and perspectives</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">724</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.8b00337</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.8b00337" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXls1akt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=713-724&author=L.+Yangauthor=F.+Liuauthor=X.+Tongauthor=D.+Hoffmannauthor=J.+Zuoauthor=M.+Lu&title=Treatment+of+chronic+hepatitis+B+virus+infection+using+small+molecule+modulators+of+nucleocapsid+assembly%3A+recent+advances+and+perspectives&doi=10.1021%2Facsinfecdis.8b00337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of Chronic Hepatitis B Virus Infection Using Small Molecule Modulators of Nucleocapsid Assembly: Recent Advances and Perspectives</span></div><div class="casAuthors">Yang, Li; Liu, Feifei; Tong, Xiankun; Hoffmann, Daniel; Zuo, Jianping; Lu, Mengji</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">713-724</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  On the basis of the recent advance of basic research on mol. biol. of hepatitis B virus (HBV) infection, novel antiviral drugs targeting various steps of the HBV life cycle have been developed in recent years.  HBV nucleocapsid assembly is now recognized as a hot target for anti-HBV drug development.  Structural and functional anal. of HBV nucleocapsid allowed rational design and improvement of small mols. with the ability to interact with the components of HBV nucleocapsid and modulate the viral nucleocapsid assembly process.  Prototypes of small mol. modulators targeting HBV nucleocapsid assembly are being preclinically tested or have moved forward in clin. trials, with promising results.  This Review summarizes the recent advances in the approach to develop antiviral drugs based on the modulation of HBV nucleocapsid assembly.  The antiviral mechanisms of small mol. modulators beyond the capsid formation and the potential implications will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvT6WVN92udLVg90H21EOLACvtfcHk0ljNrjAJ35YokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXls1akt7o%253D&md5=f602f5c7f6e9af69ac2f777ca82efe28</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.8b00337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.8b00337%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DTong%26aufirst%3DX.%26aulast%3DHoffmann%26aufirst%3DD.%26aulast%3DZuo%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DM.%26atitle%3DTreatment%2520of%2520chronic%2520hepatitis%2520B%2520virus%2520infection%2520using%2520small%2520molecule%2520modulators%2520of%2520nucleocapsid%2520assembly%253A%2520recent%2520advances%2520and%2520perspectives%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D713%26epage%3D724%26doi%3D10.1021%2Facsinfecdis.8b00337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wynne, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowther, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of the human hepatitis B virus capsid</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">771</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)80009-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS1097-2765%2801%2980009-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10394365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaK1MXktlOktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1999&pages=771-780&author=S.+A.+Wynneauthor=R.+A.+Crowtherauthor=A.+G.+W.+Leslie&title=The+crystal+structure+of+the+human+hepatitis+B+virus+capsid&doi=10.1016%2FS1097-2765%2801%2980009-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the human hepatitis B virus capsid</span></div><div class="casAuthors">Wynne, S. A.; Crowther, R. A.; Leslie, A. G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">771-780</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Hepatitis B is a small enveloped DNA virus that poses a major hazard to human health.  The crystal structure of the T = 4 capsid has been solved at 3.3 A resoln., revealing a largely helical protein fold that is unusual for icosahedral viruses.  The monomer fold is stabilized by a hydrophobic core that is highly conserved among human viral variants.  Assocn. of two amphipathic α-helical hairpins results in formation of a dimer with a four-helix bundle as the major central feature.  The capsid is assembled from dimers via interactions involving a highly conserved region near the C terminus of the truncated protein used for crystn.  The major immunodominant region lies at the tips of the α-helical hairpins that form spikes on the capsid surface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhKm_c-2LrbrVg90H21EOLACvtfcHk0ljNrjAJ35YokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktlOktbs%253D&md5=af1db5a088f6df2a5c5284a61b75aff6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2980009-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252980009-5%26sid%3Dliteratum%253Aachs%26aulast%3DWynne%26aufirst%3DS.%2BA.%26aulast%3DCrowther%26aufirst%3DR.%2BA.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DThe%2520crystal%2520structure%2520of%2520the%2520human%2520hepatitis%2520B%2520virus%2520capsid%26jtitle%3DMol.%2520Cell%26date%3D1999%26volume%3D3%26spage%3D771%26epage%3D780%26doi%3D10.1016%2FS1097-2765%2801%2980009-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">To build a virus capsid-an equilibrium-model of the self-assembly of polyhedral protein complex</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>241</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1994.1473</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1006%2Fjmbi.1994.1473" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=8051707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaK2cXlsFeiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=241&publication_year=1994&pages=59-67&author=A.+Zlotnick&title=To+build+a+virus+capsid-an+equilibrium-model+of+the+self-assembly+of+polyhedral+protein+complex&doi=10.1006%2Fjmbi.1994.1473"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">To build a virus capsid. An equilibrium model of the self assembly of polyhedral protein complexes</span></div><div class="casAuthors">Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">241</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-67</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The capsids of spherical (icosahedral) viruses are constructed of multiples of 60 subunits.  The question of how these polymers assemble is basic to understanding the viral life cycle.  A formalism describing virus assembly as an equil. between coat protein subunits, assembly intermediates and intact virus is presented.  This equil. model of virus assembly is consistent with exptl. observations of virus assembly.  At equil., either intact virus or free subunits are dominant species, assembly intermediates are predicted to be found only in trace concns.  The concn. of assembled virus at equil. is expected to be extremely concn.-dependent and resemble a highly cooperative reaction although the model does not explicitly include cooperativity.  For statistical assembly of a polyhedron, a nucleus is not necessarily required and polymn. can proceed through a cascade of biomol. reactions rather than a single higher order reaction.  Thus, kinetics of assembly do not necessarily show the extreme concn. dependence typical of nucleated protein polymn.  Modest intersubunit interaction energies result in a very stable capsid; consequently, a small change in this interaction energy can result in a considerable change in the capsid-subunit equil.  Some possible effects of nucleation and protein-nucleic acid interactions on virus assembly and capsid morphol. are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1NPGa8HJx77Vg90H21EOLACvtfcHk0ljNrjAJ35YokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlsFeiu7w%253D&md5=0e9870afe01e712992b591285e6693b1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1994.1473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1994.1473%26sid%3Dliteratum%253Aachs%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DTo%2520build%2520a%2520virus%2520capsid-an%2520equilibrium-model%2520of%2520the%2520self-assembly%2520of%2520polyhedral%2520protein%2520complex%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D241%26spage%3D59%26epage%3D67%26doi%3D10.1006%2Fjmbi.1994.1473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stray, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Punna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">A heteroaryldihydropyrimidine activates and can misdirect hepatitis B virus capsid assembly</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">8138</span>– <span class="NLM_lpage">8143</span>, <span class="refDoi"> DOI: 10.1073/pnas.0409732102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.0409732102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=15928089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlsV2msbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8138-8143&author=S.+J.+Strayauthor=C.+R.+Bourneauthor=S.+Punnaauthor=W.+G.+Lewisauthor=M.+G.+Finnauthor=A.+Zlotnick&title=A+heteroaryldihydropyrimidine+activates+and+can+misdirect+hepatitis+B+virus+capsid+assembly&doi=10.1073%2Fpnas.0409732102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A heteroaryldihydropyrimidine activates and can misdirect hepatitis B virus capsid assembly</span></div><div class="casAuthors">Stray, Stephen J.; Bourne, Christina R.; Punna, Sreenivas; Lewis, Warren G.; Finn, M. G.; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8138-8143</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Heteroaryldihydropyrimidines (HAPs) are a new class of antivirals inhibiting prodn. of hepatitis B virus (HBV) virions in tissue culture.  Here, we examine the effect of a representative HAP mol., Me 4-(2-chloro-4-fluorophenyl)-6-methyl-2-(pyridin-2-yl)-1,4-dihydropyrimidine-5-carboxylate (HAP-1), on the in vitro assembly of HBV capsid protein (Cp).  HAP-1 enhances the rate and extent of Cp assembly over a broad concn. range.  Aberrant particles, dominated by hexagonal arrays of Cp, were obsd. from assembly reactions with high HAP-1 concns.  HAP-1 also led to dissocn. of metastable HBV capsids, overcoming a kinetic barrier to dissocn. by scavenging Cp and redirecting its assembly into hexamer-rich structures.  Thus, HAP drugs act as allosteric effectors that induce an assembly-active state and, at high concn., preferentially stabilize noncapsid polymers of Cp.  HAP compds. may have multiple effects in vivo stemming from inappropriate assembly of Cp.  These results show that activating and deregulating virus assembly may be a powerful general approach for antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5IO3o_Hqhk7Vg90H21EOLACvtfcHk0ljEmsqESIkBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlsV2msbc%253D&md5=30abe296011a5e771047ce4fdc0343a8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0409732102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0409732102%26sid%3Dliteratum%253Aachs%26aulast%3DStray%26aufirst%3DS.%2BJ.%26aulast%3DBourne%26aufirst%3DC.%2BR.%26aulast%3DPunna%26aufirst%3DS.%26aulast%3DLewis%26aufirst%3DW.%2BG.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DA%2520heteroaryldihydropyrimidine%2520activates%2520and%2520can%2520misdirect%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D8138%26epage%3D8143%26doi%3D10.1073%2Fpnas.0409732102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingfield, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endres, D.</span></span> <span> </span><span class="NLM_article-title">A theoretical model successfully identifies features of hepatitis B virus capsid assembly</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">14644</span>– <span class="NLM_lpage">14652</span>, <span class="refDoi"> DOI: 10.1021/bi991611a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi991611a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADyaK1MXmsFaqurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=14644-14652&author=A.+Zlotnickauthor=J.+M.+Johnsonauthor=P.+W.+Wingfieldauthor=S.+J.+Stahlauthor=D.+Endres&title=A+theoretical+model+successfully+identifies+features+of+hepatitis+B+virus+capsid+assembly&doi=10.1021%2Fbi991611a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A theoretical model successfully identifies features of hepatitis B virus capsid assembly</span></div><div class="casAuthors">Zlotnick, Adam; Johnson, Jennifer M.; Wingfield, Paul W.; Stahl, Stephen J.; Endres, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14644-14652</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The capsids of most spherical viruses are icosahedral, an arrangement of multiples of 60 subunits.  Though it is a salient point in the life cycle of any virus, the phys. chem. of virus capsid assembly is poorly understood.  We have developed general models of capsid assembly that describe the process in terms of a cascade of low order assocn. reactions.  The models predict sigmoidal assembly kinetics, where intermediates approach a low steady state concn. for the greater part of the reaction.  Features of the overall reaction can be identified on the basis of the concn. dependence of assembly.  In simulations, and on the basis of our understanding of the models, we find that nucleus size and the order of subsequent "elongation" reactions are reflected in the concn. dependence of the extent of the reaction and the rate of the fast phase, resp.  The reaction kinetics deduced for our models of virus assembly can be related to the assembly of any "spherical" polymer.  Using light scattering and size exclusion chromatog., we obsd. polymn. of assembly domain dimers of hepatitis B virus (HBV) capsid protein.  Empty capsids assemble at a rate that is a function of protein concn. and ionic strength.  The kinetics of capsid formation were sigmoidal, where the rate of the fast phase had second-power concn. dependence.  The extent of assembly had third-power concn. dependence.  Simulations based on the models recapitulated the concn. dependences obsd. for HBV capsid assembly.  These results strongly suggest that in vitro HBV assembly is nucleated by a trimer of dimers and proceeds by the addn. of individual dimeric subunits.  On the basis of this mechanism, we suggest that HBV capsid assembly could be an important target for antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyHwRNPTccU7Vg90H21EOLACvtfcHk0ljEmsqESIkBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmsFaqurk%253D&md5=a79ca5f7ecbfca3af0687cfad18c26e5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi991611a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi991611a%26sid%3Dliteratum%253Aachs%26aulast%3DZlotnick%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DJ.%2BM.%26aulast%3DWingfield%26aufirst%3DP.%2BW.%26aulast%3DStahl%26aufirst%3DS.%2BJ.%26aulast%3DEndres%26aufirst%3DD.%26atitle%3DA%2520theoretical%2520model%2520successfully%2520identifies%2520features%2520of%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D14644%26epage%3D14652%26doi%3D10.1021%2Fbi991611a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ceres, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">Weak protein-protein interactions are sufficient to drive assembly of hepatitis B virus capsids</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">11525</span>– <span class="NLM_lpage">11531</span>, <span class="refDoi"> DOI: 10.1021/bi0261645</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0261645" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD38XmsFGms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=11525-11531&author=P.+Ceresauthor=A.+Zlotnick&title=Weak+protein-protein+interactions+are+sufficient+to+drive+assembly+of+hepatitis+B+virus+capsids&doi=10.1021%2Fbi0261645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Weak Protein-Protein Interactions Are Sufficient To Drive Assembly of Hepatitis B Virus Capsids</span></div><div class="casAuthors">Ceres, Pablo; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">11525-11531</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is an enveloped DNA virus with a spherical capsid (or core).  The capsid is constructed from 120 copies of the homodimeric capsid protein arranged with T = 4 icosahedral symmetry.  We examd. in vitro assembly of purified E. coli expressed HBV capsid protein.  After equilibration, concns. of capsid and dimer were evaluated by size exclusion chromatog.  The extent of assembly increased as temp. and ionic strength increased.  The concn. dependence of capsid assembly conformed to the equil. expression: Kcapsid = [capsid]/[dimer]120.  Given the known geometry for HBV capsids and dimers, the per capsid assembly energy was partitioned into energy per subunit-subunit contact.  We were able to make three major conclusions. (i) Weak interactions (from -2.9 kcal/mol at 21° in low salt to -4.4 kcal/mol at 37° in high salt) at each intersubunit contact result in a globally stable capsid; weak intersubunit interactions may be the basis for the phenomenon of capsid breathing. (ii) HBV assembly is characterized by pos. enthalpy and entropy.  The reaction is entropy-driven, consistent with the largely hydrophobic contacts found in the crystal structure. (iii) Increasing NaCl concn. increases the magnitude of free energy, enthalpy, and entropy, as if ionic strength were increasing the amt. of hydrophobic surface buried by assembly.  This last point leads us to suggest that salt acts by inducing a conformational change in the dimer from an assembly-inactive form to an assembly-active form.  This model of conformational change linked to assembly is consistent with immunol. differences between dimer and capsid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0k_uhx3AKtLVg90H21EOLACvtfcHk0ljEmsqESIkBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmsFGms7w%253D&md5=00756d19244fe9b7d7e12747a8ef401d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi0261645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0261645%26sid%3Dliteratum%253Aachs%26aulast%3DCeres%26aufirst%3DP.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DWeak%2520protein-protein%2520interactions%2520are%2520sufficient%2520to%2520drive%2520assembly%2520of%2520hepatitis%2520B%2520virus%2520capsids%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D11525%26epage%3D11531%26doi%3D10.1021%2Fbi0261645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lutomski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyktey, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierson, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarrold, M. F.</span></span> <span> </span><span class="NLM_article-title">Hepatitis B virus capsid completion occurs through error correction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">16932</span>– <span class="NLM_lpage">16938</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09932</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09932" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslylurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=16932-16938&author=C.+A.+Lutomskiauthor=N.+A.+Lykteyauthor=Z.+C.+Zhaoauthor=E.+E.+Piersonauthor=A.+Zlotnickauthor=M.+F.+Jarrold&title=Hepatitis+B+virus+capsid+completion+occurs+through+error+correction&doi=10.1021%2Fjacs.7b09932"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B Virus Capsid Completion Occurs through Error Correction</span></div><div class="casAuthors">Lutomski, Corinne A.; Lyktey, Nicholas A.; Zhao, Zhongchao; Pierson, Elizabeth E.; Zlotnick, Adam; Jarrold, Martin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">16932-16938</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding capsid assembly is important because of its role in virus lifecycles and in applications to drug discovery and nanomaterial development.  Many virus capsids are icosahedral, and assembly is thought to occur by the sequential addn. of capsid protein subunits to a nucleus, with the final step completing the icosahedron.  Almost nothing is known about the final (completion) step because the techniques usually used to study capsid assembly lack the resoln.  In this work, charge detection mass spectrometry (CDMS) has been used to track the assembly of the T = 4 hepatitis B virus (HBV) capsid in real time.  The initial assembly reaction occurs rapidly, on the time scale expected from low resoln. measurements.  However, CDMS shows that many of the particles generated in this process are defective and overgrown, contg. more than the 120 capsid protein dimers needed to form a perfect T = 4 icosahedron.  The defective and overgrown capsids self-correct over time to the mass expected for a perfect T = 4 capsid.  Thus, completion is a distinct phase in the assembly reaction.  Capsid completion does not necessarily occur by inserting the last building block into an incomplete, but otherwise perfect icosahedron.  The initial assembly reaction can be predominently imperfect, and completion involves the slow correction of the accumulated errors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRb3DXlFTAt7Vg90H21EOLACvtfcHk0ljEmsqESIkBQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslylurfJ&md5=779a0fafdd729b170e607c9ac36958bd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09932%26sid%3Dliteratum%253Aachs%26aulast%3DLutomski%26aufirst%3DC.%2BA.%26aulast%3DLyktey%26aufirst%3DN.%2BA.%26aulast%3DZhao%26aufirst%3DZ.%2BC.%26aulast%3DPierson%26aufirst%3DE.%2BE.%26aulast%3DZlotnick%26aufirst%3DA.%26aulast%3DJarrold%26aufirst%3DM.%2BF.%26atitle%3DHepatitis%2520B%2520virus%2520capsid%2520completion%2520occurs%2520through%2520error%2520correction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D16932%26epage%3D16938%26doi%3D10.1021%2Fjacs.7b09932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pionek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unchwaniwala, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loeb, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">The interface between hepatitis B virus capsid proteins affects self-assembly, pregenomic RNA packaging, and reverse transcription</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">3275</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1128/JVI.03545-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FJVI.03545-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=25568211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktF2gtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2015&pages=3275-3284&author=Z.+N.+Tanauthor=K.+Pionekauthor=N.+Unchwaniwalaauthor=M.+L.+Maguireauthor=D.+D.+Loebauthor=A.+Zlotnick&title=The+interface+between+hepatitis+B+virus+capsid+proteins+affects+self-assembly%2C+pregenomic+RNA+packaging%2C+and+reverse+transcription&doi=10.1128%2FJVI.03545-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The interface between hepatitis B virus capsid proteins affects self-assembly, pregenomic RNA packaging, and reverse transcription</span></div><div class="casAuthors">Tan, Zhenning; Pionek, Karolyn; Unchwaniwala, Nuruddin; Maguire, Megan L.; Loeb, Daniel D.; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3275-3284</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) capsid proteins (Cps) assemble around the pregenomic RNA (pgRNA) and viral reverse transcriptase (P). pgRNA is then reverse transcribed to double-stranded DNA (dsDNA) within the capsid.  The Cp assembly domain, which forms the shell of the capsid, regulates assembly kinetics and capsid stability.  The Cp, via its nucleic acid-binding C-terminal domain, also affects nucleic acid organization.  We hypothesize that the structure of the capsid may also have a direct effect on nucleic acid processing.  Using structure-guided design, we made a series of mutations at the interface between Cp subunits that change capsid assembly kinetics and thermodn. in a predictable manner.  Assembly in cell culture mirrored in vitro activity.  However, all of these mutations led to defects in pgRNA packaging.  The amt. of first-strand DNA synthesized was roughly proportional to the amt. of RNA packaged.  However, the synthesis of second-strand DNA, which requires two template switches, was not supported by any of the substitutions.  These data demonstrate that the HBV capsid is far more than an inert container, as mutations in the assembly domain, distant from packaged nucleic acid, affect reverse transcription.  We suggest that capsid mol. motion plays a role in regulating genome replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoot8_H5tbmwLVg90H21EOLACvtfcHk0ljTvLwlTevLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktF2gtrY%253D&md5=cbb94c6befce1b30e4600b65c0523290</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.03545-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.03545-14%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DZ.%2BN.%26aulast%3DPionek%26aufirst%3DK.%26aulast%3DUnchwaniwala%26aufirst%3DN.%26aulast%3DMaguire%26aufirst%3DM.%2BL.%26aulast%3DLoeb%26aufirst%3DD.%2BD.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DThe%2520interface%2520between%2520hepatitis%2520B%2520virus%2520capsid%2520proteins%2520affects%2520self-assembly%252C%2520pregenomic%2520RNA%2520packaging%252C%2520and%2520reverse%2520transcription%26jtitle%3DJ.%2520Virol.%26date%3D2015%26volume%3D89%26spage%3D3275%26epage%3D3284%26doi%3D10.1128%2FJVI.03545-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span> <span> </span><span class="NLM_article-title">HBc binds to the CpG islands of HBV cccDNA and promotes an epigenetic permissive state</span>. <i>Epigenetics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">720</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.4161/epi.6.6.15815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.4161%2Fepi.6.6.15815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=21546797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2gs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=720-726&author=Y.-H.+Guoauthor=Y.-N.+Liauthor=J.-R.+Zhaoauthor=J.+Zhangauthor=Z.+Yan&title=HBc+binds+to+the+CpG+islands+of+HBV+cccDNA+and+promotes+an+epigenetic+permissive+state&doi=10.4161%2Fepi.6.6.15815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">HBc binds to the CpG islands of HBV cccDNA and promotes an epigenetic permissive state</span></div><div class="casAuthors">Guo, Yan-Hai; Li, Yong-Nian; Zhao, Jin-Rong; Zhang, Ju; Yan, Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">720-726</span>CODEN:
                <span class="NLM_cas:coden">EPIGB6</span>;
        ISSN:<span class="NLM_cas:issn">1559-2294</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) covalently closed circular DNA (cccDNA) is the template for the transcription of HBV.  HBV core protein (HBc or HBcAg) is a main component of the HBV cccDNA minichromosome.  However, the function of HBc in cccDNA is not fully understood.  In light of recent findings that HBV cccDNA may be regulated epigenetically, we analyzed the binding of HBc to cccDNA and the impact of HBc on cccDNA epigenetic profile in the liver biopsy samples of 22 patients with chronic Hepatitis B (CHB).  We found that HBc binding to HBV cccDNA occurred preferentially at CpG island 2, an important region for the regulation of HBV transcription.  Furthermore, the relative abundances of HBc binding to CpG island 2 were pos. correlated with the ratios of relaxed circular DNA to cccDNA and the levels of serum HBV DNA in those patients.  Interestingly, the relative abundances of HBc binding to CpG island 2 were assocd. with the binding of CREB binding protein (CBP) and with hypomethylation in CpG island 2 of HBV cccDNA minichromosomes.  However, relatively higher amts. of HBc binding to CpG island 2 of cccDNA were accompanied by lower amts. of HDAC1 binding.  Multivariate anal. revealed that the abundances of HBc binding to CpG island 2 of cccDNA and pos. HBeAg were independent factors assocd. with the replication of HBV (p = 0.001 for both).  Apparently, HBc is a pos. regulator of HBV transcription and replication, maintaining the permissive epigenetic state in the crit. region of the HBV cccDNA minichromosomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO9wmAiX25lrVg90H21EOLACvtfcHk0ljTvLwlTevLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2gs7w%253D&md5=3057fac0de5c9a054f5ace2967f52a93</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Fepi.6.6.15815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.6.6.15815%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DY.-H.%26aulast%3DLi%26aufirst%3DY.-N.%26aulast%3DZhao%26aufirst%3DJ.-R.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DZ.%26atitle%3DHBc%2520binds%2520to%2520the%2520CpG%2520islands%2520of%2520HBV%2520cccDNA%2520and%2520promotes%2520an%2520epigenetic%2520permissive%2520state%26jtitle%3DEpigenetics%26date%3D2011%26volume%3D6%26spage%3D720%26epage%3D726%26doi%3D10.4161%2Fepi.6.6.15815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatakrishnan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewellyn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S.</span></span> <span> </span><span class="NLM_article-title">Core protein: a pleiotropic keystone in the HBV lifecycle</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">93</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2015.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.antiviral.2015.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=26129969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtV2ltLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2015&pages=82-93&author=A.+Zlotnickauthor=B.+Venkatakrishnanauthor=Z.+N.+Tanauthor=E.+Lewellynauthor=W.+Turnerauthor=S.+Francis&title=Core+protein%3A+a+pleiotropic+keystone+in+the+HBV+lifecycle&doi=10.1016%2Fj.antiviral.2015.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Core protein: A pleiotropic keystone in the HBV lifecycle</span></div><div class="casAuthors">Zlotnick, Adam; Venkatakrishnan, Balasubramanian; Tan, Zhenning; Lewellyn, Eric; Turner, William; Francis, Samson</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-93</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Hepatitis B Virus (HBV) is a small virus whose genome has only four open reading frames.  We argue that the simplicity of the virion correlates with a complexity of functions for viral proteins.  We focus on the HBV core protein (Cp), a small (183 residue) protein that self-assembles to form the viral capsid.  However, its functions are a little more complicated than that.  In an infected cell Cp modulates almost every step of the viral lifecycle.  Cp is bound to nuclear viral DNA and affects its epigenetics.  Cp correlates with RNA specificity.  Cp assembles specifically on a reverse transcriptase-viral RNA complex or, apparently, nothing at all.  Indeed Cp has been one of the model systems for investigation of virus self-assembly.  Cp participates in regulation of reverse transcription.  Cp signals completion of reverse transcription to support virus secretion.  Cp carries both nuclear localization signals and HBV surface antigen (HBsAg) binding sites; both of these functions appear to be regulated by contents of the capsid.  Cp can be targeted by antivirals - while self-assembly is the most accessible of Cp activities, we argue that it makes sense to engage the broader spectrum of Cp function.  This article forms part of a symposium in Antiviral Research on "From the discovery of the Australia antigen to the development of new curative therapies for hepatitis B: an unfinished story.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyafN5HJZSQLVg90H21EOLACvtfcHk0ljTvLwlTevLjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2ltLvM&md5=e1bd58b97219cb54136dbb2db2fadb70</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2015.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2015.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DZlotnick%26aufirst%3DA.%26aulast%3DVenkatakrishnan%26aufirst%3DB.%26aulast%3DTan%26aufirst%3DZ.%2BN.%26aulast%3DLewellyn%26aufirst%3DE.%26aulast%3DTurner%26aufirst%3DW.%26aulast%3DFrancis%26aufirst%3DS.%26atitle%3DCore%2520protein%253A%2520a%2520pleiotropic%2520keystone%2520in%2520the%2520HBV%2520lifecycle%26jtitle%3DAntiviral%2520Res.%26date%3D2015%26volume%3D121%26spage%3D82%26epage%3D93%26doi%3D10.1016%2Fj.antiviral.2015.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule therapeutics for treatment of chronic HBV infection</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.7b00144</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.7b00144" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=257-277&author=S.+Fengauthor=L.+Gaoauthor=X.+C.+Hanauthor=T.+S.+Huauthor=Y.+M.+Huauthor=H.+X.+Liuauthor=A.+W.+Thomasauthor=Z.+P.+Yanauthor=S.+Yangauthor=J.+A.+T.+Youngauthor=H.+Y.+Yunauthor=W.+Zhuauthor=H.+C.+Shen&title=Discovery+of+small+molecule+therapeutics+for+treatment+of+chronic+HBV+infection&doi=10.1021%2Facsinfecdis.7b00144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Therapeutics for Treatment of Chronic HBV Infection</span></div><div class="casAuthors">Feng, Song; Gao, Lu; Han, Xingchun; Hu, Taishan; Hu, Yimin; Liu, Haixia; Thomas, Andrew W.; Yan, Zhipeng; Yang, Song; Young, John A. T.; Yun, Hongying; Zhu, Wei; Shen, Hong C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-277</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The chronic infection of hepatitis B virus (HBV) inflicts 250 million people worldwide representing a major public health threat.  A significant sub-population of chronically infected HBV patients eventually develop cirrhosis and hepatocellular carcinoma (HCC) which aggravate the issues caused by hepatitis alone.  Unfortunately, none of the current std. therapies for chronic Hepatitis B (CHB) result in a satisfactory clin. cure rate.  Driven by a highly unmet medical need for novel and more effective treatments of HBV infection, multiple pharmaceutical companies and research institutions have been engaged in drug discovery and development to improve the CHB functional cure rate, which is defined by sustainable viral suppression and HBsAg clearance after a finite treatment.  This review summarizes the recent significant advances in the discovery and development of novel small mol. anti-HBV compds., highlighted by representative structures, mechanism of action (MoA), and biol. activities.  Lastly, it is believed that improved CHB functional cure rate may be accomplished via the combination of mols. with distinct MoAs.  Thus some of the mols. may evolve into key components of a suitable combination therapy leading to superior outcome of clin. efficacy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1zsMqu16n17Vg90H21EOLACvtfcHk0lj8UK0xNJrE_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D&md5=45e6f4740904bd92d82193455a2d97ac</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.7b00144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.7b00144%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%2BC.%26aulast%3DHu%26aufirst%3DT.%2BS.%26aulast%3DHu%26aufirst%3DY.%2BM.%26aulast%3DLiu%26aufirst%3DH.%2BX.%26aulast%3DThomas%26aufirst%3DA.%2BW.%26aulast%3DYan%26aufirst%3DZ.%2BP.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DYun%26aufirst%3DH.%2BY.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%2BC.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520therapeutics%2520for%2520treatment%2520of%2520chronic%2520HBV%2520infection%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2018%26volume%3D4%26spage%3D257%26epage%3D277%26doi%3D10.1021%2Facsinfecdis.7b00144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Corcuera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartenschlager, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birkmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, A.</span></span> <span> </span><span class="NLM_article-title">Novel non-heteroarylpyrimidine (HAP) capsid assembly modifiers have a different mode of action from HAPs in vitro</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">135</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2018.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.antiviral.2018.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=30031759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFequr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2018&pages=135-142&author=A.+Corcueraauthor=K.+Stolleauthor=S.+Hillmerauthor=S.+Seitzauthor=J.+Y.+Leeauthor=R.+Bartenschlagerauthor=A.+Birkmannauthor=A.+Urban&title=Novel+non-heteroarylpyrimidine+%28HAP%29+capsid+assembly+modifiers+have+a+different+mode+of+action+from+HAPs+in+vitro&doi=10.1016%2Fj.antiviral.2018.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Novel non-heteroarylpyrimidine (HAP) capsid assembly modifiers have a different mode of action from HAPs in vitro</span></div><div class="casAuthors">Corcuera, Angelica; Stolle, Katharina; Hillmer, Stefan; Seitz, Stefan; Lee, Ji-Young; Bartenschlager, Ralf; Birkmann, Alexander; Urban, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">135-142</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">One of the most promising viral targets in current hepatitis B virus (HBV) drug development is the core protein due to its multiple roles in the viral life cycle.  Here we investigated the differences in the mode of action and antiviral activity of representatives of six different capsid assembly modifier (CAM) scaffolds: three from the well-characterized scaffolds heteroarylpyrimidine (HAP), sulfamoylbenzamide (SBA), and phenylpropenamide (PPA), and three from novel scaffolds glyoxamide-pyrrolamide (GPA), pyrazolyl-thiazole (PT), and dibenzo-thiazepin-2-one (DBT).  The target activity and antiviral efficacy of the different CAMs were tested in biochem. and cellular assays.  Anal. size exclusion chromatog. and transmission electron microscopy showed that only the HAP compd. induced formation of aberrant non-capsid structures (class II mode of action), while the remaining CAMs did not affect capsid gross morphol. (class I mode of action).  Intracellular lysates from the HepAD38 cell line, inducibly replicating HBV, showed no redn. in the quantities of intracellular core protein or capsid after treatment with SBA, PPA, GPA, PT, or DBT compds.; however HAP-treatment led to a profound decrease in both.  Addnl., immunofluorescence staining of compd.-treated HepAD38 cells showed that all non-HAP CAMs led to a shift in the equil. of HBV core antigen (HBcAg) towards complete cytoplasmic staining, while the HAP induced accumulation of HBcAg aggregates in the nucleus.  Our study demonstrates that the novel scaffolds GPA, PT, and DBT exhibit class I modes of action, alike SBA and PPA, whereas HAP remains the only scaffold belonging to class II inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIuxr1oBqjkrVg90H21EOLACvtfcHk0lj8UK0xNJrE_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFequr%252FJ&md5=25cd40330f90d3dccdefe80daa51cffd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2018.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2018.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DCorcuera%26aufirst%3DA.%26aulast%3DStolle%26aufirst%3DK.%26aulast%3DHillmer%26aufirst%3DS.%26aulast%3DSeitz%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DBartenschlager%26aufirst%3DR.%26aulast%3DBirkmann%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DA.%26atitle%3DNovel%2520non-heteroarylpyrimidine%2520%2528HAP%2529%2520capsid%2520assembly%2520modifiers%2520have%2520a%2520different%2520mode%2520of%2520action%2520from%2520HAPs%2520in%2520vitro%26jtitle%3DAntiviral%2520Res.%26date%3D2018%26volume%3D158%26spage%3D135%26epage%3D142%26doi%3D10.1016%2Fj.antiviral.2018.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehertogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergauwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Damme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostmans, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauwels, F.</span></span> <span> </span><span class="NLM_article-title">Capsid assembly modulators have a dual mechanism of action in primary human hepatocytes infected with hepatitis B virus</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">e00560</span>, <span class="refDoi"> DOI: 10.1128/AAC.00560-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00560-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28584155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVSktbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=e00560&author=J.+M.+Berkeauthor=P.+Dehertoghauthor=K.+Vergauwenauthor=E.+Van+Dammeauthor=W.+Mostmansauthor=K.+Vandyckauthor=F.+Pauwels&title=Capsid+assembly+modulators+have+a+dual+mechanism+of+action+in+primary+human+hepatocytes+infected+with+hepatitis+B+virus&doi=10.1128%2FAAC.00560-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Capsid assembly modulators have a dual mechanism of action in primary human hepatocytes infected with hepatitis B virus</span></div><div class="casAuthors">Berke, Jan Martin; Dehertogh, Pascale; Vergauwen, Karen; Van Damme, Ellen; Mostmans, Wendy; Vandyck, Koen; Pauwels, Frederik</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e00560-17/14</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) capsid assembly is a crit. step in the propagation of the virus and is mediated by the core protein.  Due to its multiple functions in the viral life cycle, core became an attractive target for new antiviral therapies.  Capsid assembly modulators (CAMs) accelerate the kinetics of capsid assembly and prevent encapsidation of the polymerase-pregenomic RNA (Pol-pgRNA) complex, thereby blocking viral replication.  CAM JNJ-632 is a novel and potent inhibitor of HBV replication in vitro across genotypes A to D.  It induces the formation of morphol. intact viral capsids, as demonstrated by size exclusion chromatog. and electron microscopy studies.  Antiviral profiling in primary human hepatocytes revealed that CAMs prevented formation of covalently closed circular DNA in a dose-dependent fashion when the compd. was added together with the viral inoculum, whereas nucleos(t)ide analogs (NAs) did not.  This protective effect translated into a dose-dependent redn. of intracellular HBV RNA levels as well as reduced HBe/cAg and HBsAg levels in the cell culture supernatant.  The same observation was made with another CAM (BAY41-4109), suggesting that mechanistic rather than compd.-specific effects play a role.  Our data show that CAMs have a dual mechanism of action, inhibiting early and late steps of the viral life cycle.  These effects clearly differentiate CAMs from NAs and may translate into higher functional cure rates in a clin. setting when given alone or in combination with the current std. of care.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdf2H1OpRSlrVg90H21EOLACvtfcHk0lj8UK0xNJrE_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVSktbbO&md5=665812f18beb49225110302e9afbd980</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FAAC.00560-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00560-17%26sid%3Dliteratum%253Aachs%26aulast%3DBerke%26aufirst%3DJ.%2BM.%26aulast%3DDehertogh%26aufirst%3DP.%26aulast%3DVergauwen%26aufirst%3DK.%26aulast%3DVan%2BDamme%26aufirst%3DE.%26aulast%3DMostmans%26aufirst%3DW.%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DPauwels%26aufirst%3DF.%26atitle%3DCapsid%2520assembly%2520modulators%2520have%2520a%2520dual%2520mechanism%2520of%2520action%2520in%2520primary%2520human%2520hepatocytes%2520infected%2520with%2520hepatitis%2520B%2520virus%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26spage%3De00560%26doi%3D10.1128%2FAAC.00560-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lahlali, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergauwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauwels, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoulim, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span> <span> </span><span class="NLM_article-title">Novel potent capsid assembly modulators regulate multiple steps of the hepatitis B virus life cycle</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">15</span>, <span class="refDoi"> DOI: 10.1128/AAC.00835-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00835-18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=15&author=T.+Lahlaliauthor=J.+M.+Berkeauthor=K.+Vergauwenauthor=A.+Focaauthor=K.+Vandyckauthor=F.+Pauwelsauthor=F.+Zoulimauthor=D.+Durantel&title=Novel+potent+capsid+assembly+modulators+regulate+multiple+steps+of+the+hepatitis+B+virus+life+cycle&doi=10.1128%2FAAC.00835-18"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FAAC.00835-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00835-18%26sid%3Dliteratum%253Aachs%26aulast%3DLahlali%26aufirst%3DT.%26aulast%3DBerke%26aufirst%3DJ.%2BM.%26aulast%3DVergauwen%26aufirst%3DK.%26aulast%3DFoca%26aufirst%3DA.%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DPauwels%26aufirst%3DF.%26aulast%3DZoulim%26aufirst%3DF.%26aulast%3DDurantel%26aufirst%3DD.%26atitle%3DNovel%2520potent%2520capsid%2520assembly%2520modulators%2520regulate%2520multiple%2520steps%2520of%2520the%2520hepatitis%2520B%2520virus%2520life%2520cycle%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2018%26volume%3D62%26spage%3D15%26doi%3D10.1128%2FAAC.00835-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lam, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp, K.</span></span> <span> </span><span class="NLM_article-title">Hepatitis B virus capsid assembly modulators, but not nucleoside analogs, inhibit the production of extracellular pregenomic RNA and spliced RNA variants</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">e00680</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1128/AAC.00680-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00680-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28559265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Wltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=e00680-17&author=A.+M.+Lamauthor=S.+P.+Renauthor=C.+Espirituauthor=M.+Kellyauthor=V.+Lauauthor=L.+J.+Zhengauthor=G.+D.+Hartmanauthor=O.+A.+Floresauthor=K.+Klumpp&title=Hepatitis+B+virus+capsid+assembly+modulators%2C+but+not+nucleoside+analogs%2C+inhibit+the+production+of+extracellular+pregenomic+RNA+and+spliced+RNA+variants&doi=10.1128%2FAAC.00680-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis B virus capsid assembly modulators, but not nucleoside analogs, inhibit the production of extracellular pregenomic RNA and spliced RNA variants</span></div><div class="casAuthors">Lam, Angela M.; Ren, Suping; Espiritu, Christine; Kelly, Mollie; Lau, Vincent; Zheng, Lingjie; Hartman, George D.; Flores, Osvaldo A.; Klumpp, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e00680-17/1-e00680-17/14</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The hepatitis B virus (HBV) core protein serves multiple essential functions in the viral life cycle, and antiviral agents that target the core protein are being developed.  Capsid assembly modulators (CAMs) are compds. that target core and misdirect capsid assembly, resulting in the suppression of HBV replication and virion prodn.  Besides HBV DNA, circulating HBV RNA has been detected in patient serum and can be assocd. with the treatment response.  Here the authors studied the effect of HBV CAMs on the prodn. of extracellular HBV RNA using infected HepaRG cells and primary human hepatocytes.  Representative compds. from the sulfonamide carboxamide and heteroaryldihydropyrimidine series of CAMs were evaluated and compared to nucleos(t)ide analogs as inhibitors of the viral polymerase.  The results showed that CAMs blocked extracellular HBV RNA with efficiencies similar to those with which they blocked pregenomic RNA (pgRNA) encapsidation, HBV DNA replication, and Dane particle prodn.  Nucleos(t)ide analogs inhibited viral replication and virion prodn. but not encapsidation or prodn. of extracellular HBV RNA.  Profiling of HBV RNA from both culture supernatants and patient serum showed that extracellular viral RNA consisted of pgRNA and spliced pgRNA variants with an internal deletion(s) but still retained the sequences at both the 5' and 3' ends.  Similar variants were detected in the supernatants of infected cells with and without nucleos(t)ide analog treatment.  Overall, the data demonstrate that HBV CAMs represent direct antiviral agents with a profile differentiated from that of nucleos(t)ide analogs, including the inhibition of extracellular pgRNA and spliced pgRNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJzKwT-Zo87rVg90H21EOLACvtfcHk0lh7WETdXXmJ3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Wltrw%253D&md5=8bfa2b78fa1da04f49b5f5e0d1fbdc77</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FAAC.00680-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00680-17%26sid%3Dliteratum%253Aachs%26aulast%3DLam%26aufirst%3DA.%2BM.%26aulast%3DRen%26aufirst%3DS.%2BP.%26aulast%3DEspiritu%26aufirst%3DC.%26aulast%3DKelly%26aufirst%3DM.%26aulast%3DLau%26aufirst%3DV.%26aulast%3DZheng%26aufirst%3DL.%2BJ.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26aulast%3DFlores%26aufirst%3DO.%2BA.%26aulast%3DKlumpp%26aufirst%3DK.%26atitle%3DHepatitis%2520B%2520virus%2520capsid%2520assembly%2520modulators%252C%2520but%2520not%2520nucleoside%2520analogs%252C%2520inhibit%2520the%2520production%2520of%2520extracellular%2520pregenomic%2520RNA%2520and%2520spliced%2520RNA%2520variants%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2017%26volume%3D61%26spage%3De00680%26epage%3D17%26doi%3D10.1128%2FAAC.00680-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelps, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardzinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cobarrubias, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuconati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsey, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harasym, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadhim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kultgen, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A. H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Q. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majeski, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClintock, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rijnbrand, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snead, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuer, H. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stever, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sofia, M. J.</span></span> <span> </span><span class="NLM_article-title">Preclinical profile of AB-423, an inhibitor of hepatitis B virus pregenomic RNA encapsidation</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">22</span>, <span class="refDoi"> DOI: 10.1128/AAC.00082-18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.00082-18" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&pages=22&author=N.+Maniauthor=A.+G.+Coleauthor=J.+R.+Phelpsauthor=A.+Ardzinskiauthor=K.+D.+Cobarrubiasauthor=A.+Cuconatiauthor=B.+D.+Dorseyauthor=E.+Evangelistaauthor=K.+Fanauthor=F.+Guoauthor=H.+T.+Guoauthor=J.+T.+Guoauthor=T.+O.+Harasymauthor=S.+Kadhimauthor=S.+G.+Kultgenauthor=A.+C.+H.+Leeauthor=A.+H.+L.+Liauthor=Q.+X.+Longauthor=S.+A.+Majeskiauthor=R.+C.+Maoauthor=K.+D.+McClintockauthor=S.+P.+Reidauthor=R.+Rijnbrandauthor=N.+M.+Sneadauthor=H.+M.+M.+Steuerauthor=K.+Steverauthor=S.+Tangauthor=X.+H.+Wangauthor=Q.+Zhaoauthor=M.+J.+Sofia&title=Preclinical+profile+of+AB-423%2C+an+inhibitor+of+hepatitis+B+virus+pregenomic+RNA+encapsidation&doi=10.1128%2FAAC.00082-18"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FAAC.00082-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00082-18%26sid%3Dliteratum%253Aachs%26aulast%3DMani%26aufirst%3DN.%26aulast%3DCole%26aufirst%3DA.%2BG.%26aulast%3DPhelps%26aufirst%3DJ.%2BR.%26aulast%3DArdzinski%26aufirst%3DA.%26aulast%3DCobarrubias%26aufirst%3DK.%2BD.%26aulast%3DCuconati%26aufirst%3DA.%26aulast%3DDorsey%26aufirst%3DB.%2BD.%26aulast%3DEvangelista%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DK.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DH.%2BT.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26aulast%3DHarasym%26aufirst%3DT.%2BO.%26aulast%3DKadhim%26aufirst%3DS.%26aulast%3DKultgen%26aufirst%3DS.%2BG.%26aulast%3DLee%26aufirst%3DA.%2BC.%2BH.%26aulast%3DLi%26aufirst%3DA.%2BH.%2BL.%26aulast%3DLong%26aufirst%3DQ.%2BX.%26aulast%3DMajeski%26aufirst%3DS.%2BA.%26aulast%3DMao%26aufirst%3DR.%2BC.%26aulast%3DMcClintock%26aufirst%3DK.%2BD.%26aulast%3DReid%26aufirst%3DS.%2BP.%26aulast%3DRijnbrand%26aufirst%3DR.%26aulast%3DSnead%26aufirst%3DN.%2BM.%26aulast%3DSteuer%26aufirst%3DH.%2BM.%2BM.%26aulast%3DStever%26aufirst%3DK.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DSofia%26aufirst%3DM.%2BJ.%26atitle%3DPreclinical%2520profile%2520of%2520AB-423%252C%2520an%2520inhibitor%2520of%2520hepatitis%2520B%2520virus%2520pregenomic%2520RNA%2520encapsidation%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2018%26volume%3D62%26spage%3D22%26doi%3D10.1128%2FAAC.00082-18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bester, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blossey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacherl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondran, F. W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Protzer, U.</span></span> <span> </span><span class="NLM_article-title">A new role for capsid assembly modulators to target mature hepatitis B virus capsids and prevent virus infection</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">13</span>, <span class="refDoi"> DOI: 10.1128/AAC.01440-19</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FAAC.01440-19" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2019&pages=13&author=C.+Koauthor=R.+Besterauthor=X.+Zhouauthor=Z.+H.+Xuauthor=C.+Blosseyauthor=J.+Sacherlauthor=F.+W.+R.+Vondranauthor=L.+Gaoauthor=U.+Protzer&title=A+new+role+for+capsid+assembly+modulators+to+target+mature+hepatitis+B+virus+capsids+and+prevent+virus+infection&doi=10.1128%2FAAC.01440-19"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1128%2FAAC.01440-19&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.01440-19%26sid%3Dliteratum%253Aachs%26aulast%3DKo%26aufirst%3DC.%26aulast%3DBester%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%2BH.%26aulast%3DBlossey%26aufirst%3DC.%26aulast%3DSacherl%26aufirst%3DJ.%26aulast%3DVondran%26aufirst%3DF.%2BW.%2BR.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DProtzer%26aufirst%3DU.%26atitle%3DA%2520new%2520role%2520for%2520capsid%2520assembly%2520modulators%2520to%2520target%2520mature%2520hepatitis%2520B%2520virus%2520capsids%2520and%2520prevent%2520virus%2520infection%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2019%26volume%3D64%26spage%3D13%26doi%3D10.1128%2FAAC.01440-19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perni, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladner, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaifert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. W.</span></span> <span> </span><span class="NLM_article-title">Phenylpropenamide derivatives as inhibitors of hepatitis B virus replication</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2687</span>– <span class="NLM_lpage">2690</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(00)00544-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS0960-894X%2800%2900544-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=11128652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotlantbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=2687-2690&author=R.+B.+Perniauthor=S.+C.+Conwayauthor=S.+K.+Ladnerauthor=K.+Zaifertauthor=M.+J.+Ottoauthor=R.+W.+King&title=Phenylpropenamide+derivatives+as+inhibitors+of+hepatitis+B+virus+replication&doi=10.1016%2FS0960-894X%2800%2900544-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylpropenamide derivatives as inhibitors of Hepatitis B virus replication</span></div><div class="casAuthors">Perni, R. B.; Conway, S. C.; Ladner, S. K.; Zaifert, K.; Otto, M. J.; King, R. W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2687-2690</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A non-nucleoside class of compds. that inhibits the replication of hepatitis B virus (HBV) in cell culture has been discovered.  A series of substituted analogs of phenylpropenamide has been prepd. and evaluated in the HepAD38 cellular assay.  Structure-activity relationships of this series are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcZcKEPcrqkbVg90H21EOLACvtfcHk0lhcVmiOIJ3kkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotlantbw%253D&md5=4c3ee48f05dc8bee0724ba928d89f8a1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2800%2900544-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252800%252900544-8%26sid%3Dliteratum%253Aachs%26aulast%3DPerni%26aufirst%3DR.%2BB.%26aulast%3DConway%26aufirst%3DS.%2BC.%26aulast%3DLadner%26aufirst%3DS.%2BK.%26aulast%3DZaifert%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DM.%2BJ.%26aulast%3DKing%26aufirst%3DR.%2BW.%26atitle%3DPhenylpropenamide%2520derivatives%2520as%2520inhibitors%2520of%2520hepatitis%2520B%2520virus%2520replication%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D2687%26epage%3D2690%26doi%3D10.1016%2FS0960-894X%2800%2900544-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campagna, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuconati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. T.</span></span> <span> </span><span class="NLM_article-title">Sulfamoylbenzamide derivatives inhibit the assembly of hepatitis B virus nucleocapsids</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">6931</span>– <span class="NLM_lpage">6942</span>, <span class="refDoi"> DOI: 10.1128/JVI.00582-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FJVI.00582-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=23576513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlKrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2013&pages=6931-6942&author=M.+R.+Campagnaauthor=F.+Liuauthor=R.+C.+Maoauthor=C.+Millsauthor=D.+W.+Caiauthor=F.+Guoauthor=X.+S.+Zhaoauthor=H.+Yeauthor=A.+Cuconatiauthor=H.+T.+Guoauthor=J.+H.+Changauthor=X.+D.+Xuauthor=T.+M.+Blockauthor=J.+T.+Guo&title=Sulfamoylbenzamide+derivatives+inhibit+the+assembly+of+hepatitis+B+virus+nucleocapsids&doi=10.1128%2FJVI.00582-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfamoylbenzamide derivatives inhibit the assembly of hepatitis B virus nucleocapsids</span></div><div class="casAuthors">Campagna, Matthew R.; Liu, Fei; Mao, Richeng; Mills, Courtney; Cai, Dawei; Guo, Fang; Zhao, Xuesen; Ye, Hong; Cuconati, Andrea; Guo, Haitao; Chang, Jinhong; Xu, Xiaodong; Block, Timothy M.; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6931-6942</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection, a serious public health problem leading to cirrhosis and hepatocellular carcinoma, is currently treated with either pegylated alpha interferon (pegIFN-α) or one of the five nucleos(t)ide analog viral DNA polymerase inhibitors.  However, neither pegIFN-α nor nucleos(t)ide analogs are capable of reliably curing the viral infection.  In order to develop novel antiviral drugs against HBV, we established a cell-based screening assay by using an immortalized mouse hepatocyte-derived stable cell line supporting a high level of HBV replication in a tetracycline-inducible manner.  Screening of a library consisting of 26,900 small mols. led to the discovery of a series of sulfamoylbenzamide (SBA) derivs. that significantly reduced the amt. of cytoplasmic HBV DNA.  Structure-activity relationship studies have thus far identified a group of fluorine-substituted SBAs with submicromolar antiviral activity against HBV in human hepatoma cells.  Mechanistic analyses reveal that the compds. dose dependently inhibit the formation of pregenomic RNA (pgRNA)-contg. nucleocapsids of HBV but not other animal hepadnaviruses, such as woodchuck hepatitis virus (WHV) and duck hepatitis B virus (DHBV).  Moreover, heterologous genetic complementation studies of capsid protein, DNA polymerase, and pgRNA between HBV and WHV suggest that HBV capsid protein confers sensitivity to the SBAs.  In summary, SBAs represent a novel chem. entity with superior activity and a unique antiviral mechanism and are thus warranted for further development as novel antiviral therapeutics for the treatment of chronic hepatitis B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNUsTkQKVBQbVg90H21EOLACvtfcHk0livP7lPXseccw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlKrsr0%253D&md5=80cd9d05d889ecf1bd2f6c28f7946bcf</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FJVI.00582-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00582-13%26sid%3Dliteratum%253Aachs%26aulast%3DCampagna%26aufirst%3DM.%2BR.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DMao%26aufirst%3DR.%2BC.%26aulast%3DMills%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DD.%2BW.%26aulast%3DGuo%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DX.%2BS.%26aulast%3DYe%26aufirst%3DH.%26aulast%3DCuconati%26aufirst%3DA.%26aulast%3DGuo%26aufirst%3DH.%2BT.%26aulast%3DChang%26aufirst%3DJ.%2BH.%26aulast%3DXu%26aufirst%3DX.%2BD.%26aulast%3DBlock%26aufirst%3DT.%2BM.%26aulast%3DGuo%26aufirst%3DJ.%2BT.%26atitle%3DSulfamoylbenzamide%2520derivatives%2520inhibit%2520the%2520assembly%2520of%2520hepatitis%2520B%2520virus%2520nucleocapsids%26jtitle%3DJ.%2520Virol.%26date%3D2013%26volume%3D87%26spage%3D6931%26epage%3D6942%26doi%3D10.1128%2FJVI.00582-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlemmer, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagelschuer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paessens, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deres, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niewoehner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoltefuss, J. D. I.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Inhibition of human hepatitis B virus (HBV) by a novel non-nucleosidic compound in a transgenic mouse model</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1016/S0166-3542(01)00216-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2FS0166-3542%2801%2900216-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=12062392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtVShsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2002&pages=69-78&author=O.+Weberauthor=K.+H.+Schlemmerauthor=E.+Hartmannauthor=I.+Hagelschuerauthor=A.+D.+Paessensauthor=E.+Graefauthor=K.+Deresauthor=S.+Goldmannauthor=U.+Niewoehnerauthor=J.+D.+I.+Stoltefuss&title=Inhibition+of+human+hepatitis+B+virus+%28HBV%29+by+a+novel+non-nucleosidic+compound+in+a+transgenic+mouse+model&doi=10.1016%2FS0166-3542%2801%2900216-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human hepatitis B virus (HBV) by a novel non-nucleosidic compound in a transgenic mouse model</span></div><div class="casAuthors">Weber, O.; Schlemmer, K.-H.; Hartmann, E.; Hagelschuer, Ina; Paessens, A.; Graef, E.; Deres, K.; Goldmann, S.; Niewoehner, U.; Stoltefuss, J.; Haebich, D.; Ruebsamen-Waigmann, H.; Wohlfeil, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">69-78</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">BAY 41-4109 is a member of a class of heteroaryl-pyrimidines that was recently identified as potent inhibitors of human hepatitis B virus (HBV) replication.  We have investigated the antiviral activity of BAY 41-4109 (Me (R)-4-(2-chloro-4-fluorophenyl)-2-(3,5-difluoro-2-pyridinyl)-6-methyl-1,4-dihydro-pyrimidine-5-carboxylate) in HBV-transgenic mice (Tg [HBV1.3 fsX-3'5']).  Bay 41-4109 was administered per os using different schedules (b.i.d. or t.i.d. for up to 28 days) and dosages ranging from 3 to 30 mg/kg.  The compd. reduced viral DNA in the liver and in the plasma dose-dependently with efficacy comparable to 3TC.  In contrast to 3TC-treated mice, we found a redn. of cytoplasmic hepatitis B virus core antigen (HBcAg) in liver sections of BAY 41-4109-treated mice, which indicated a different mode of action.  Pharmacokinetic studies in mice have shown rapid absorption, a bioavailability of 30% and dose-proportional plasma concns.  We conclude that BAY 41-4109 is a new anti-HBV drug candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAj4XpaXvzGrVg90H21EOLACvtfcHk0livP7lPXseccw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtVShsLg%253D&md5=bae520cd731096858f86f4ce28674231</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0166-3542%2801%2900216-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0166-3542%252801%252900216-9%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DSchlemmer%26aufirst%3DK.%2BH.%26aulast%3DHartmann%26aufirst%3DE.%26aulast%3DHagelschuer%26aufirst%3DI.%26aulast%3DPaessens%26aufirst%3DA.%2BD.%26aulast%3DGraef%26aufirst%3DE.%26aulast%3DDeres%26aufirst%3DK.%26aulast%3DGoldmann%26aufirst%3DS.%26aulast%3DNiewoehner%26aufirst%3DU.%26aulast%3DStoltefuss%26aufirst%3DJ.%2BD.%2BI.%26atitle%3DInhibition%2520of%2520human%2520hepatitis%2520B%2520virus%2520%2528HBV%2529%2520by%2520a%2520novel%2520non-nucleosidic%2520compound%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DAntiviral%2520Res.%26date%3D2002%26volume%3D54%26spage%3D69%26epage%3D78%26doi%3D10.1016%2FS0166-3542%2801%2900216-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katen, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zlotnick, A.</span></span> <span> </span><span class="NLM_article-title">Assembly-directed antivirals differentially bind quasiequivalent pockets to modify hepatitis B virus capsid tertiary and quaternary structure</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1406</span>– <span class="NLM_lpage">1416</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.06.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.str.2013.06.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=23871485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCgs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1406-1416&author=S.+P.+Katenauthor=Z.+N.+Tanauthor=S.+R.+Chirapuauthor=M.+G.+Finnauthor=A.+Zlotnick&title=Assembly-directed+antivirals+differentially+bind+quasiequivalent+pockets+to+modify+hepatitis+B+virus+capsid+tertiary+and+quaternary+structure&doi=10.1016%2Fj.str.2013.06.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Assembly-Directed Antivirals Differentially Bind Quasiequivalent Pockets to Modify Hepatitis B Virus Capsid Tertiary and Quaternary Structure</span></div><div class="casAuthors">Katen, Sarah P.; Tan, Zhenning; Chirapu, Srinivas Reddy; Finn, M. G.; Zlotnick, Adam</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1406-1416</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is a major cause of liver disease.  Assembly of the HBV capsid is a crit. step in virus prodn. and an attractive target for new antiviral therapies.  We detd. the structure of HBV capsid in complex with AT-130, a member of the phenylpropenamide family of assembly effectors.  AT-130 causes tertiary and quaternary structural changes but does not disrupt capsid structure.  AT-130 binds a hydrophobic pocket that also accommodates the previously characterized heteroaryldihydropyrimidine compds. but favors a unique quasiequivalent location on the capsid surface.  Thus, this pocket is a promiscuous drug-binding site and a likely target for different assembly effectors with a broad range of mechanisms of activity.  That AT-130 successfully decreases virus prodn. by increasing capsid assembly rate without disrupting capsid structure delineates a paradigm in antiviral design, that disrupting reaction timing is a viable strategy for assembly effectors of HBV and other viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZur3DCpnS3LVg90H21EOLACvtfcHk0lgVFPUqnbOsMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCgs7fF&md5=bb515fd504fb69f82755409c4c8f2079</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.06.013%26sid%3Dliteratum%253Aachs%26aulast%3DKaten%26aufirst%3DS.%2BP.%26aulast%3DTan%26aufirst%3DZ.%2BN.%26aulast%3DChirapu%26aufirst%3DS.%2BR.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DZlotnick%26aufirst%3DA.%26atitle%3DAssembly-directed%2520antivirals%2520differentially%2520bind%2520quasiequivalent%2520pockets%2520to%2520modify%2520hepatitis%2520B%2520virus%2520capsid%2520tertiary%2520and%2520quaternary%2520structure%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1406%26epage%3D1416%26doi%3D10.1016%2Fj.str.2013.06.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baydo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abendroth, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Efimov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, O. A.</span></span> <span> </span><span class="NLM_article-title">High-resolution crystal structure of a hepatitis B virus replication inhibitor bound to the viral core protein</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">15196</span>– <span class="NLM_lpage">15201</span>, <span class="refDoi"> DOI: 10.1073/pnas.1513803112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.1513803112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=26598693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWqsrbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15196-15201&author=K.+Klumppauthor=A.+M.+Lamauthor=C.+Lukacsauthor=R.+Vogelauthor=S.+P.+Renauthor=C.+Espirituauthor=R.+Baydoauthor=K.+Atkinsauthor=J.+Abendrothauthor=G.+C.+Liaoauthor=A.+Efimovauthor=G.+Hartmanauthor=O.+A.+Flores&title=High-resolution+crystal+structure+of+a+hepatitis+B+virus+replication+inhibitor+bound+to+the+viral+core+protein&doi=10.1073%2Fpnas.1513803112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution crystal structure of a hepatitis B virus replication inhibitor bound to the viral core protein</span></div><div class="casAuthors">Klumpp, Klaus; Lam, Angela M.; Lukacs, Christine; Vogel, Robert; Ren, Suping; Espiritu, Christine; Baydo, Ruth; Atkins, Kateri; Abendroth, Jan; Liao, Guochun; Efimov, Andrey; Hartman, George; Flores, Osvaldo A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">15196-15201</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The hepatitis B virus (HBV) core protein is essential for HBV replication and an important target for antiviral drug discovery.  We report the first, to our knowledge, high-resoln. crystal structure of an antiviral compd. bound to the HBV core protein.  The compd. NVR-010-001-E2 can induce assembly of the HBV core wild-type and Y132A mutant proteins and thermostabilize the proteins with a Tm increase of more than 10 °C.  NVR-010-001-E2 binds at the dimer-dimer interface of the core proteins, forms a new interaction surface promoting protein-protein interaction, induces protein assembly, and increases stability.  The impact of naturally occurring core protein mutations on antiviral activity correlates with NVR-010-001-E2 binding interactions detd. by crystallog.  The crystal structure provides understanding of a drug efficacy mechanism related to the induction and stabilization of protein-protein interactions and enables structure-guided design to improve antiviral potency and drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYxV-odPonGbVg90H21EOLACvtfcHk0lgVFPUqnbOsMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWqsrbK&md5=bf487dda7165d1637fad0ebab8f46142</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1513803112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1513803112%26sid%3Dliteratum%253Aachs%26aulast%3DKlumpp%26aufirst%3DK.%26aulast%3DLam%26aufirst%3DA.%2BM.%26aulast%3DLukacs%26aufirst%3DC.%26aulast%3DVogel%26aufirst%3DR.%26aulast%3DRen%26aufirst%3DS.%2BP.%26aulast%3DEspiritu%26aufirst%3DC.%26aulast%3DBaydo%26aufirst%3DR.%26aulast%3DAtkins%26aufirst%3DK.%26aulast%3DAbendroth%26aufirst%3DJ.%26aulast%3DLiao%26aufirst%3DG.%2BC.%26aulast%3DEfimov%26aufirst%3DA.%26aulast%3DHartman%26aufirst%3DG.%26aulast%3DFlores%26aufirst%3DO.%2BA.%26atitle%3DHigh-resolution%2520crystal%2520structure%2520of%2520a%2520hepatitis%2520B%2520virus%2520replication%2520inhibitor%2520bound%2520to%2520the%2520viral%2520core%2520protein%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15196%26epage%3D15201%26doi%3D10.1073%2Fpnas.1513803112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildum, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Alcalde, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, G. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najera, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span> <span> </span><span class="NLM_article-title">Heteroaryldihydropyrimidine (HAP) and sulfamoylbenzamide (SBA) inhibit hepatitis B virus replication by different molecular mechanisms</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">42374</span>, <span class="refDoi"> DOI: 10.1038/srep42374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1038%2Fsrep42374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28205569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislOiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=42374&author=Z.+Zhouauthor=T.+S.+Huauthor=X.+Zhouauthor=S.+Wildumauthor=F.+Garcia-Alcaldeauthor=Z.+H.+Xuauthor=D.+Z.+Wuauthor=Y.+Maoauthor=X.+J.+Tianauthor=Y.+Zhouauthor=F.+Shenauthor=Z.+S.+Zhangauthor=G.+Z.+Tangauthor=I.+Najeraauthor=G.+Yangauthor=H.+C.+Shenauthor=J.+A.+T.+Youngauthor=N.+Qin&title=Heteroaryldihydropyrimidine+%28HAP%29+and+sulfamoylbenzamide+%28SBA%29+inhibit+hepatitis+B+virus+replication+by+different+molecular+mechanisms&doi=10.1038%2Fsrep42374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Heteroaryldihydropyrimidine (HAP) and Sulfamoylbenzamide (SBA) Inhibit Hepatitis B Virus Replication by Different Molecular Mechanisms</span></div><div class="casAuthors">Zhou, Zheng; Hu, Taishan; Zhou, Xue; Wildum, Steffen; Garcia-Alcalde, Fernando; Xu, Zhiheng; Wu, Daitze; Mao, Yi; Tian, Xiaojun; Zhou, Yuan; Shen, Fang; Zhang, Zhisen; Tang, Guozhi; Najera, Isabel; Yang, Guang; Shen, Hong C.; Young, John A. T.; Qin, Ning</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42374</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Heteroaryldihydropyrimidine (HAP) and sulfamoylbenzamide (SBA) are promising non-nucleos(t)ide HBV replication inhibitors.  HAPs are known to promote core protein mis-assembly, but the mol. mechanism of abnormal assembly is still elusive.  Likewise, the assembly status of core protein induced by SBA remains unknown.  Here we show that SBA, unlike HAP, does not promote core protein mis-assembly.  Interestingly, two ref. compds. HAP_R01 and SBA_R01 bind to the same pocket at the dimer-dimer interface in the crystal structures of core protein Y132A hexamer.  The striking difference lies in a unique hydrophobic subpocket that is occupied by the thiazole group of HAP_R01, but is unperturbed by SBA_R01.  Photoaffinity labeling confirms the HAP_R01 binding pose at the dimer-dimer interface on capsid and suggests a new mechanism of HAP-induced mis-assembly.  Based on the common features in crystal structures we predict that T33 mutations generate similar susceptibility changes to both compds.  In contrast, mutations at positions in close contact with HAP-specific groups (P25A, P25S, or V124F) only reduce susceptibility to HAP_R01, but not to SBA_R01.  Thus, HAP and SBA are likely to have distinctive resistance profiles.  Notably, P25S and V124F substitutions exist in low-abundance quasispecies in treatment-naive patients, suggesting potential clin. relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw-qU6eO7rCrVg90H21EOLACvtfcHk0lhav4vWxEsC8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislOiurs%253D&md5=1e401e3133194ae374435df0d0cfa637</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fsrep42374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep42374%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DT.%2BS.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DWildum%26aufirst%3DS.%26aulast%3DGarcia-Alcalde%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DZ.%2BH.%26aulast%3DWu%26aufirst%3DD.%2BZ.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTian%26aufirst%3DX.%2BJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DZ.%2BS.%26aulast%3DTang%26aufirst%3DG.%2BZ.%26aulast%3DNajera%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DQin%26aufirst%3DN.%26atitle%3DHeteroaryldihydropyrimidine%2520%2528HAP%2529%2520and%2520sulfamoylbenzamide%2520%2528SBA%2529%2520inhibit%2520hepatitis%2520B%2520virus%2520replication%2520by%2520different%2520molecular%2520mechanisms%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D42374%26doi%3D10.1038%2Fsrep42374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okazaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinoda, W.</span></span> <span> </span><span class="NLM_article-title">Heteroaryldihydropyrimidines alter capsid assembly by adjusting the binding affinity and pattern of the hepatitis B virus core protein</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5104</span>– <span class="NLM_lpage">5110</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1WktrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=5104-5110&author=H.+Liuauthor=S.+Okazakiauthor=W.+Shinoda&title=Heteroaryldihydropyrimidines+alter+capsid+assembly+by+adjusting+the+binding+affinity+and+pattern+of+the+hepatitis+B+virus+core+protein&doi=10.1021%2Facs.jcim.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Heteroaryldihydropyrimidines Alter Capsid Assembly By Adjusting the Binding Affinity and Pattern of the Hepatitis B Virus Core Protein</span></div><div class="casAuthors">Liu, Huihui; Okazaki, Susumu; Shinoda, Wataru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5104-5110</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) infections are a major global health concern, for which heteroaryldihydropyrimidines (HAPs) have been developed.  HAPs accelerate and/or result in aberrant capsid assembly; however, their effect on the assembly mechanism is unknown.  This study aimed to compare the effects of three representative HAPs on core protein dimer assembly through mol. dynamics simulations and free energy calcns.  Mol. docking and equil. simulations showed that different HAPs bind at the same binding site and are involved in different interactions.  The obsd. conformational changes in HAPs deter the calcn. of binding affinity.  Herein, the reduced free energy perturbation/Hamiltonian replica exchange mol. dynamics method was used to enhance sampling during binding affinity calcns., indicating consistency between the binding free energies of HAPs and pEC50.  Furthermore, binding pattern anal. revealed that the tetramer could sample flat structures after binding HAPs.  The present results suggest a mechanism wherein HAPs accelerate capsid assembly by increasing the binding affinity of dimers, leading to aberrant assembly by altering the binding orientation of dimers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc9UKC5UgIArVg90H21EOLACvtfcHk0lhav4vWxEsC8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1WktrvP&md5=0871ef3341b009a15504cae14cd03938</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DOkazaki%26aufirst%3DS.%26aulast%3DShinoda%26aufirst%3DW.%26atitle%3DHeteroaryldihydropyrimidines%2520alter%2520capsid%2520assembly%2520by%2520adjusting%2520the%2520binding%2520affinity%2520and%2520pattern%2520of%2520the%2520hepatitis%2520B%2520virus%2520core%2520protein%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D5104%26epage%3D5110%26doi%3D10.1021%2Facs.jcim.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sari, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozturk, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, O. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassit, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amblard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schinazi, R. F.</span></span> <span> </span><span class="NLM_article-title">Synthesis of sulfamoylbenzamide derivatives as HBV capsid assembly effector</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.ejmech.2017.06.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28688280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=407-421&author=O.+Sariauthor=S.+Boucleauthor=B.+D.+Coxauthor=T.+Ozturkauthor=O.+I.+Russellauthor=L.+Bassitauthor=F.+Amblardauthor=R.+F.+Schinazi&title=Synthesis+of+sulfamoylbenzamide+derivatives+as+HBV+capsid+assembly+effector&doi=10.1016%2Fj.ejmech.2017.06.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of sulfamoylbenzamide derivatives as HBV capsid assembly effectors</span></div><div class="casAuthors">Sari, Ozkan; Boucle, Sebastien; Cox, Bryan D.; Ozturk, Tugba; Russell, Olivia Ollinger; Bassit, Leda; Amblard, Franck; Schinazi, Raymond F.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">407-421</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The synthesis of novel series of sulfamoylbenzamides, e.g. I [R1 = F, R2 = H; R1R2 = (CH2)2, (CH2)3; X = R3R4NSO2, R5SO2NH; R3 = cyclopentyl, cyclopentyloxy, MeSO2, PhCHMe, etc., R4 = H; R3R4 = CH2CF2CH2CH2, C(O)(CH2)3, etc.; R5 = cyclopropyl, cyclopentyl, cyclohexyl] as HBV capsid assembly effector is reported.  The structure was divided into five parts which were independently modified as part of the lead optimization.  All the synthesized compds. were evaluated for their anti-HBV activity and toxicity in human hepatocytes, lymphocytes and other cells.  Addnl., their effect on HBV cccDNA formation in an HBeAg reporter cell-based assay was assessed.  Among the 27 compds. reported, several analogs exhibited submicromolar activities and significant redn. of HBeAg secretion.  Selected compds. were studied under neg.-stain electron microscopy for their ability to disrupt the HBV capsid formation.  Structures were modeled into a binding site recently identified in the HBV capsid protein for similar mols. to rationalize the structure-activity relationships for this family of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUa_fzgDe9WrVg90H21EOLACvtfcHk0liUBpKKOrUdWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvI&md5=cc0de429427455fac4cebd76c51119ed</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.062%26sid%3Dliteratum%253Aachs%26aulast%3DSari%26aufirst%3DO.%26aulast%3DBoucle%26aufirst%3DS.%26aulast%3DCox%26aufirst%3DB.%2BD.%26aulast%3DOzturk%26aufirst%3DT.%26aulast%3DRussell%26aufirst%3DO.%2BI.%26aulast%3DBassit%26aufirst%3DL.%26aulast%3DAmblard%26aufirst%3DF.%26aulast%3DSchinazi%26aufirst%3DR.%2BF.%26atitle%3DSynthesis%2520of%2520sulfamoylbenzamide%2520derivatives%2520as%2520HBV%2520capsid%2520assembly%2520effector%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D407%26epage%3D421%26doi%3D10.1016%2Fj.ejmech.2017.06.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rombouts, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoops, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschueren, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergauwen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehertogh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berke, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raboisson, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of <i>N</i>-phenyl-3-sulfamoyl-benzamide derivatives as capsid assembly modulators inhibiting hepatitis B virus (HBV)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6247</span>– <span class="NLM_lpage">6260</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6247-6260&author=K.+Vandyckauthor=G.+Romboutsauthor=B.+Stoopsauthor=A.+Tahriauthor=A.+Vosauthor=W.+Verschuerenauthor=Y.+M.+Wuauthor=J.+M.+Yangauthor=F.+L.+Houauthor=B.+Huangauthor=K.+Vergauwenauthor=P.+Dehertoghauthor=J.+M.+Berkeauthor=P.+Raboisson&title=Synthesis+and+evaluation+of+N-phenyl-3-sulfamoyl-benzamide+derivatives+as+capsid+assembly+modulators+inhibiting+hepatitis+B+virus+%28HBV%29&doi=10.1021%2Facs.jmedchem.8b00654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of N-Phenyl-3-sulfamoyl-benzamide Derivatives as Capsid Assembly Modulators Inhibiting Hepatitis B Virus (HBV)</span></div><div class="casAuthors">Vandyck, Koen; Rombouts, Geert; Stoops, Bart; Tahri, Abdellah; Vos, Ann; Verschueren, Wim; Wu, Yiming; Yang, Jingmei; Hou, Fuliang; Huang, Bing; Vergauwen, Karen; Dehertogh, Pascale; Berke, Jan Martin; Raboisson, Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6247-6260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol. induced hepatitis B virus (HBV) capsid assembly modulation is considered an attractive approach for new antiviral therapies against HBV.  Here we describe efforts toward the discovery of a HBV capsid assembly modulator in a hit-to-lead optimization, resulting in JNJ-632, a tool compd. used to further profile the mode of action.  Administration of JNJ-632 (54) in HBV genotype D infected chimeric mice resulted in a 2.77 log redn. of the HBV DNA viral load.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKzPnHUf4R5LVg90H21EOLACvtfcHk0liUBpKKOrUdWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFCmsrfF&md5=b877a4732f8dc2141c60fba79f27aa52</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00654%26sid%3Dliteratum%253Aachs%26aulast%3DVandyck%26aufirst%3DK.%26aulast%3DRombouts%26aufirst%3DG.%26aulast%3DStoops%26aufirst%3DB.%26aulast%3DTahri%26aufirst%3DA.%26aulast%3DVos%26aufirst%3DA.%26aulast%3DVerschueren%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%2BM.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DHou%26aufirst%3DF.%2BL.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DVergauwen%26aufirst%3DK.%26aulast%3DDehertogh%26aufirst%3DP.%26aulast%3DBerke%26aufirst%3DJ.%2BM.%26aulast%3DRaboisson%26aufirst%3DP.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520N-phenyl-3-sulfamoyl-benzamide%2520derivatives%2520as%2520capsid%2520assembly%2520modulators%2520inhibiting%2520hepatitis%2520B%2520virus%2520%2528HBV%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6247%26epage%3D6260%26doi%3D10.1021%2Facs.jmedchem.8b00654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of new hepatitis B virus capsid assembly modulators by an optimal high-throughput cell-based assay</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">778</span>– <span class="NLM_lpage">787</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.9b00030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.9b00030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivV2qu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=778-787&author=Y.+M.+Peiauthor=C.+T.+Wangauthor=H.+J.+Benauthor=L.+Wangauthor=Y.+Maauthor=Q.+Y.+Maauthor=Y.+Xiangauthor=L.+Q.+Zhangauthor=G.+Liu&title=Discovery+of+new+hepatitis+B+virus+capsid+assembly+modulators+by+an+optimal+high-throughput+cell-based+assay&doi=10.1021%2Facsinfecdis.9b00030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of New Hepatitis B Virus Capsid Assembly Modulators by an Optimal High-Throughput Cell-Based Assay</span></div><div class="casAuthors">Pei, Yameng; Wang, Chunting; Ben, Haijing; Wang, Lei; Ma, Yao; Ma, Qingyan; Xiang, Ye; Zhang, Linqi; Liu, Gang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">778-787</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, a simple and effective high-throughput screening (HTS) assay was developed to identify anti-HBV compds. by using a HepAD38 luciferase reporter (HepAD38-luc) cell line that can effectively exclude the false pos. hit compds. targeted on the tetracycline off (tet-off) regulation system.  Through screening inhouse chem. libraries, N-phenylpiperidine-3-carboxamide derivs., represented by 1 and 2, were identified, while the other false pos. hits (i.e., quinoxaline (3) and benzothiazin (4) derivs.) were simultaneously excluded.  Compds. 1 and 2 exhibit strong inhibitory activity against HBV replication in both HepAD38 and HepG2.2.15 cells.  Further studies revealed that 1 and 2 reduced extracellular HBV DNA, HBeAg, and intracellular HBV intermediates, including total DNA, RNA, and precore RNA of HBV.  Size-exclusion chromatog. (SEC) and electron microscopy (EM) investigations demonstrated that 1 and 2 remarkably induced the formation of morphol. intact capsids and accelerated the dynamics of capsid assembly, suggesting that both 1 and 2 were type I capsid assembly modulators (CAMs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovmkgieeKSfrVg90H21EOLACvtfcHk0ljX3YTJueTTSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivV2qu7w%253D&md5=770a2f4bb84f92426260d1031724319c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.9b00030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.9b00030%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DY.%2BM.%26aulast%3DWang%26aufirst%3DC.%2BT.%26aulast%3DBen%26aufirst%3DH.%2BJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%2BY.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DL.%2BQ.%26aulast%3DLiu%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520new%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%2520modulators%2520by%2520an%2520optimal%2520high-throughput%2520cell-based%2520assay%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2019%26volume%3D5%26spage%3D778%26epage%3D787%26doi%3D10.1021%2Facsinfecdis.9b00030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-T.</span></span> <span> </span><span class="NLM_article-title">Discovery and mechanistic study of benzamide derivatives that modulate hepatitis B virus capsid assembly</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">e00519</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1128/JVI.00519-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1128%2FJVI.00519-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=28566379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvVKh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2017&pages=e00519-17&author=S.+Wuauthor=Q.+Zhaoauthor=P.+Zhangauthor=J.+Kulpauthor=L.+Huauthor=N.+Hwangauthor=J.+Zhangauthor=T.+M.+Blockauthor=X.+Xuauthor=Y.+Duauthor=J.+Changauthor=J.-T.+Guo&title=Discovery+and+mechanistic+study+of+benzamide+derivatives+that+modulate+hepatitis+B+virus+capsid+assembly&doi=10.1128%2FJVI.00519-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and mechanistic study of benzamide derivatives that modulate hepatitis B virus capsid assembly</span></div><div class="casAuthors">Wu, Shuo; Zhao, Qiong; Zhang, Pinghu; Kulp, John; Hu, Lydia; Hwang, Nicky; Zhang, Jiming; Block, Timothy M.; Xu, Xiaodong; Du, Yanming; Chang, Jinhong; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">e00519-17/1-e00519-17/18</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection is a global public health problem.  Although the currently approved medications can reliably reduce the viral load and prevent the progression of liver diseases, they fail to cure the viral infection.  In an effort toward discovery of novel antiviral agents against HBV, a group of benzamide (BA) derivs. that significantly reduced the amt. of cytoplasmic HBV DNA were discovered.  The initial lead optimization efforts identified two BA derivs. with improved antiviral activity for further mechanistic studies.  Interestingly, similar to our previously reported sulfamoylbenzamides (SBAs), the BAs promote the formation of empty capsids through specific interaction with HBV core protein but not other viral and host cellular components.  Genetic evidence suggested that both SBAs and BAs inhibited HBV nucleocapsid assembly by binding to the heteroaryldihydropyrimidine (HAP) pocket between core protein dimer-dimer interfaces.  However, unlike SBAs, BA compds. uniquely induced the formation of empty capsids that migrated more slowly in native agarose gel electrophoresis from A36V mutant than from the wild-type core protein.  Moreover, we showed that the assembly of chimeric capsids from wild-type and drug-resistant core proteins was susceptible to multiple capsid assembly modulators.  Hence, HBV core protein is a dominant antiviral target that may suppress the selection of drug-resistant viruses during core protein-targeting antiviral therapy.  Our studies thus indicate that BAs are a chem. and mechanistically unique type of HBV capsid assembly modulators and warranted for further development as antiviral agents against HBV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLV9R8pGFhULVg90H21EOLACvtfcHk0ljX3YTJueTTSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvVKh&md5=e1fae3fad652ba4c65d84d55ed9aa751</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1128%2FJVI.00519-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00519-17%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DKulp%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DHwang%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBlock%26aufirst%3DT.%2BM.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.-T.%26atitle%3DDiscovery%2520and%2520mechanistic%2520study%2520of%2520benzamide%2520derivatives%2520that%2520modulate%2520hepatitis%2520B%2520virus%2520capsid%2520assembly%26jtitle%3DJ.%2520Virol.%26date%3D2017%26volume%3D91%26spage%3De00519%26epage%3D17%26doi%3D10.1128%2FJVI.00519-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandyck, K.</span>; <span class="NLM_string-name">Verschueren, W. G.</span>; <span class="NLM_string-name">Raboisson, P. J. -M. B.</span></span> <span> </span><span class="NLM_article-title">Fused bicyclic sulfamoyl derivatives and the use as medicaments for the treatment of hepatitis B</span>. PCT Int. Appl. <span class="NLM_patent">WO 2014033167</span>, Mar 6, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=K.+Vandyck&author=W.+G.+Verschueren&author=P.+J.+-M.+B.+Raboisson&title=Fused+bicyclic+sulfamoyl+derivatives+and+the+use+as+medicaments+for+the+treatment+of+hepatitis+B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVandyck%26aufirst%3DK.%26atitle%3DFused%2520bicyclic%2520sulfamoyl%2520derivatives%2520and%2520the%2520use%2520as%2520medicaments%2520for%2520the%2520treatment%2520of%2520hepatitis%2520B%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gillis, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eastman, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Applications of fluorine in medicinal chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">8315</span>– <span class="NLM_lpage">8359</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8315-8359&author=E.+P.+Gillisauthor=K.+J.+Eastmanauthor=M.+D.+Hillauthor=D.+J.+Donnellyauthor=N.+A.+Meanwell&title=Applications+of+fluorine+in+medicinal+chemistry&doi=10.1021%2Facs.jmedchem.5b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of Fluorine in Medicinal Chemistry</span></div><div class="casAuthors">Gillis, Eric P.; Eastman, Kyle J.; Hill, Matthew D.; Donnelly, David J.; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8315-8359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with meta-anal. The role of fluorine in drug design and development is expanding rapidly as we learn more about the unique properties assocd. with this unusual element and how to deploy it with greater sophistication.  The judicious introduction of fluorine into a mol. can productively influence conformation, pKa, intrinsic potency, membrane permeability, metabolic pathways, and pharmacokinetic properties.  In addn., 18F has been established as a useful positron emitting isotope for use with in vivo imaging technol. that potentially has extensive application in drug discovery and development, often limited only by convenient synthetic accessibility to labeled compds.  The wide ranging applications of fluorine in drug design are providing a strong stimulus for the development of new synthetic methodologies that allow more facile access to a wide range of fluorinated compds.  In this review, we provide an update on the effects of the strategic incorporation of fluorine in drug mols. and applications in positron emission tomog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8fUUtkPB4L7Vg90H21EOLACvtfcHk0lhrjgNIHsxddQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1ajs7%252FK&md5=9995829a94a8c0b8d9fb0d21bdfd5a1d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00258%26sid%3Dliteratum%253Aachs%26aulast%3DGillis%26aufirst%3DE.%2BP.%26aulast%3DEastman%26aufirst%3DK.%2BJ.%26aulast%3DHill%26aufirst%3DM.%2BD.%26aulast%3DDonnelly%26aufirst%3DD.%2BJ.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DApplications%2520of%2520fluorine%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8315%26epage%3D8359%26doi%3D10.1021%2Facs.jmedchem.5b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, S. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickett, S. D.</span></span> <span> </span><span class="NLM_article-title">The impact of aromatic ring count on compound developability: further insights by examining carbo- and hetero-aromatic and -aliphatic ring types</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">164</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2010.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.drudis.2010.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=21129497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVyqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=164-171&author=T.+J.+Ritchieauthor=S.+J.+F.+Macdonaldauthor=R.+J.+Youngauthor=S.+D.+Pickett&title=The+impact+of+aromatic+ring+count+on+compound+developability%3A+further+insights+by+examining+carbo-+and+hetero-aromatic+and+-aliphatic+ring+types&doi=10.1016%2Fj.drudis.2010.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of aromatic ring count on compound developability: further insights by examining carbo- and hetero-aromatic and -aliphatic ring types</span></div><div class="casAuthors">Ritchie, Timothy J.; MacDonald, Simon J. F.; Young, Robert J.; Pickett, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3/4</span>),
    <span class="NLM_cas:pages">164-171</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The impact of carboarom., heteroarom., carboaliph. and heteroaliph. ring counts and fused arom. ring count on several developability measures (soly., lipophilicity, protein binding, P 450 inhibition and hERG binding) is the topic for this review article.  Recent results indicate that increasing ring counts have detrimental effects on developability in the order carboaroms. » heteroaroms. > carboaliphatics > heteroaliphatics, with heteroaliphatics exerting a beneficial effect in many cases.  Increasing arom. ring count exerts effects on several developability parameters that are lipophilicity- and size-independent, and fused arom. systems have a beneficial effect relative to their nonfused counterparts.  Increasing arom. ring count has a detrimental effect on human bioavailability parameters, and heteroarom. ring count (but not other ring counts) has increased over time in marketed oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJYPC7hkoyv7Vg90H21EOLACvtfcHk0lhxHaco-1xgXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVyqsr0%253D&md5=8e2d0e4e499d2f07d66ca2f385defb2d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2010.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2010.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DRitchie%26aufirst%3DT.%2BJ.%26aulast%3DMacdonald%26aufirst%3DS.%2BJ.%2BF.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DPickett%26aufirst%3DS.%2BD.%26atitle%3DThe%2520impact%2520of%2520aromatic%2520ring%2520count%2520on%2520compound%2520developability%253A%2520further%2520insights%2520by%2520examining%2520carbo-%2520and%2520hetero-aromatic%2520and%2520-aliphatic%2520ring%2520types%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D164%26epage%3D171%26doi%3D10.1016%2Fj.drudis.2010.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, L.</span></span> <i>The role of functional groups in drug-receptor interaction: in the practice of medicinal chemistry</i>, <span class="NLM_edition">3</span>rd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Wermuth, C.-G.</span></span>, Ed.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">480</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=464-480&author=L.+Schaefferauthor=C.-G.+Wermuth&title=The+role+of+functional+groups+in+drug-receptor+interaction%3A+in+the+practice+of+medicinal+chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSchaeffer%26aufirst%3DL.%26btitle%3DThe%2520role%2520of%2520functional%2520groups%2520in%2520drug-receptor%2520interaction%253A%2520in%2520the%2520practice%2520of%2520medicinal%2520chemistry%26aulast%3DWermuth%26aufirst%3DC.-G.%26pub%3DElsevier%26date%3D2008%26spage%3D464%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic efficiency as an important metric in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+efficiency+as+an+important+metric+in+drug+design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0lhxHaco-1xgXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520efficiency%2520as%2520an%2520important%2520metric%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarcsay, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyiri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keseru, G. M.</span></span> <span> </span><span class="NLM_article-title">Impact of lipophilic efficiency on compound quality</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.1021/jm201388p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201388p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVKjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1252-1260&author=A.+Tarcsayauthor=K.+Nyiriauthor=G.+M.+Keseru&title=Impact+of+lipophilic+efficiency+on+compound+quality&doi=10.1021%2Fjm201388p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Lipophilic Efficiency on Compound Quality</span></div><div class="casAuthors">Tarcsay, Akos; Nyiri, Kinga; Keseru, Gyorgy M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1252-1260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lipophilic efficiency indexes such as LLE and LELP were suggested to support balanced optimization of potency and ADMET profile.  Here we investigated the performance of LLE and LELP on multiple data sets representing different stages of drug discovery including fragment and HTS hits and leads, development candidates, phase II compds., and launched drugs.  Analyzing their impact on ADME and safety properties and binding thermodn., we found that both LLE and LELP help identifying better quality compds.  LLE is sensible for the development stages but does not prefer fragment-type hits, while LELP has an advantage for this class of compds. and discriminates preferred starting points effectively.  Both LLE and LELP have significant impact on ADME and safety profiles; however, LELP outperforms LLE in risk assessment at least on the present data set.  On the basis of the results reported here, monitoring lipophilic efficiency metrics could contribute significantly to compd. quality and might improve the output of medicinal chem. programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxB8xs6-5DL7Vg90H21EOLACvtfcHk0lg_rjgd2Imo9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktVKjsQ%253D%253D&md5=2536abef781f52806013016971c78bcb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm201388p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201388p%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3DA.%26aulast%3DNyiri%26aufirst%3DK.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DImpact%2520of%2520lipophilic%2520efficiency%2520on%2520compound%2520quality%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1252%26epage%3D1260%26doi%3D10.1021%2Fjm201388p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degorce, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodnarchuk, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cumming, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. S.</span></span> <span> </span><span class="NLM_article-title">Lowering lipophilicity by adding carbon: one-carbon bridges of morpholines and piperazines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8934</span>– <span class="NLM_lpage">8943</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01148</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01148" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8934-8943&author=S.+L.+Degorceauthor=M.+S.+Bodnarchukauthor=I.+A.+Cummingauthor=J.+S.+Scott&title=Lowering+lipophilicity+by+adding+carbon%3A+one-carbon+bridges+of+morpholines+and+piperazines&doi=10.1021%2Facs.jmedchem.8b01148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Lowering Lipophilicity by Adding Carbon: One-Carbon Bridges of Morpholines and Piperazines</span></div><div class="casAuthors">Degorce, Sebastien L.; Bodnarchuk, Michael S.; Cumming, Iain A.; Scott, James S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8934-8943</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this article, we report our investigation of a phenomenon by which bridging morpholines across the ring with one-carbon tethers leads to a counterintuitive redn. in lipophilicity.  This effect was also found to occur in piperazines and piperidines and lowered the measured log D7.4 of the bridged mols. by as much as -0.8 relative to their unbridged counterparts.  As lowering lipophilicity without introducing addnl. heteroatoms can be desirable, we believe this potentially provides a useful tactic to improve the drug-like properties of mols. contg. morpholine-, piperazine-, and piperidine-like motifs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhZJB3SYRKD7Vg90H21EOLACvtfcHk0lg_rjgd2Imo9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KhtbzK&md5=27f30b8220ea767d6ef1e492f3fa1aba</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01148%26sid%3Dliteratum%253Aachs%26aulast%3DDegorce%26aufirst%3DS.%2BL.%26aulast%3DBodnarchuk%26aufirst%3DM.%2BS.%26aulast%3DCumming%26aufirst%3DI.%2BA.%26aulast%3DScott%26aufirst%3DJ.%2BS.%26atitle%3DLowering%2520lipophilicity%2520by%2520adding%2520carbon%253A%2520one-carbon%2520bridges%2520of%2520morpholines%2520and%2520piperazines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8934%26epage%3D8943%26doi%3D10.1021%2Facs.jmedchem.8b01148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zafrani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sod-Moriah, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeffet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berliner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katalan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashkenazi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madmon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershonov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saphier, S.</span></span> <span> </span><span class="NLM_article-title">CF2H, a functional group-dependent hydrogen-bond donor: is it a more or less lipophilic bioisostere of OH, SH, and CH3?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5628</span>– <span class="NLM_lpage">5637</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00604</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00604" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvVOksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5628-5637&author=Y.+Zafraniauthor=G.+Sod-Moriahauthor=D.+Yeffetauthor=A.+Berlinerauthor=D.+Amirauthor=D.+Marcianoauthor=S.+Eliasauthor=S.+Katalanauthor=N.+Ashkenaziauthor=M.+Madmonauthor=E.+Gershonovauthor=S.+Saphier&title=CF2H%2C+a+functional+group-dependent+hydrogen-bond+donor%3A+is+it+a+more+or+less+lipophilic+bioisostere+of+OH%2C+SH%2C+and+CH3%3F&doi=10.1021%2Facs.jmedchem.9b00604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">CF2H, a Functional Group-Dependent Hydrogen-Bond Donor: Is It a More or Less Lipophilic Bioisostere of OH, SH, and CH3?</span></div><div class="casAuthors">Zafrani, Yossi; Sod-Moriah, Gali; Yeffet, Dina; Berliner, Anat; Amir, Dafna; Marciano, Daniele; Elias, Shlomi; Katalan, Shahaf; Ashkenazi, Nissan; Madmon, Moran; Gershonov, Eytan; Saphier, Sigal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5628-5637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effects of the CF2H moiety on H-bond (HB) acidity and lipophilicity of various compds., when attached directly to an arom. ring or to other functions like alkyls, ethers/thioethers, or electron-withdrawing groups, are discussed.  It was found that the CF2H group acts as a HB donor with a strong dependence on the attached functional group (A = 0.035-0.165).  Regarding lipophilicity, the CF2H group may act as a more lipophilic bioisostere of OH but as a similar or less lipophilic bioisostere of SH and CH3, resp., when attached to Ar or alkyl.  In addn., the lipophilicity of ethers, sulfoxides, and sulfones is dramatically increased upon CH3/CF2H exchange at the α position.  Interestingly, this exchange significantly affects not only the polarity and the vol. of the solutes but also their HB-accepting ability, the main factors influencing log Poct.  Accordingly, this study may be helpful in the rational design of drugs contg. this moiety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoib73JwC9F-bVg90H21EOLACvtfcHk0ljT_GbhMzFYSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvVOksrw%253D&md5=cef6c35ca2138d3e18d79d41a1661c3c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00604%26sid%3Dliteratum%253Aachs%26aulast%3DZafrani%26aufirst%3DY.%26aulast%3DSod-Moriah%26aufirst%3DG.%26aulast%3DYeffet%26aufirst%3DD.%26aulast%3DBerliner%26aufirst%3DA.%26aulast%3DAmir%26aufirst%3DD.%26aulast%3DMarciano%26aufirst%3DD.%26aulast%3DElias%26aufirst%3DS.%26aulast%3DKatalan%26aufirst%3DS.%26aulast%3DAshkenazi%26aufirst%3DN.%26aulast%3DMadmon%26aufirst%3DM.%26aulast%3DGershonov%26aufirst%3DE.%26aulast%3DSaphier%26aufirst%3DS.%26atitle%3DCF2H%252C%2520a%2520functional%2520group-dependent%2520hydrogen-bond%2520donor%253A%2520is%2520it%2520a%2520more%2520or%2520less%2520lipophilic%2520bioisostere%2520of%2520OH%252C%2520SH%252C%2520and%2520CH3%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5628%26epage%3D5637%26doi%3D10.1021%2Facs.jmedchem.9b00604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span> <span> </span><span class="NLM_article-title">Direct inhibition of hepatitis B e antigen by core protein allosteric modulator</span>. <i>Hepatology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1002/hep.30514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1002%2Fhep.30514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=30664279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yis7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2019&pages=11-24&author=Z.+P.+Yanauthor=D.+Wuauthor=H.+Huauthor=J.+Zengauthor=X.+Yuauthor=Z.+H.+Xuauthor=Z.+Zhouauthor=X.+Zhouauthor=G.+Yangauthor=J.+A.+T.+Youngauthor=L.+Gao&title=Direct+inhibition+of+hepatitis+B+e+antigen+by+core+protein+allosteric+modulator&doi=10.1002%2Fhep.30514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Direct Inhibition of Hepatitis B e Antigen by Core Protein Allosteric Modulator</span></div><div class="casAuthors">Yan, Zhipeng; Wu, Daitze; Hu, Hui; Zeng, Jing; Yu, Xin; Xu, Zhiheng; Zhou, Zheng; Zhou, Xue; Yang, Guang; Young, John A. T.; Gao, Lu</div><div class="citationInfo"><span class="NLM_cas:title">Hepatology (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-24</span>CODEN:
                <span class="NLM_cas:coden">HPTLD9</span>;
        ISSN:<span class="NLM_cas:issn">0270-9139</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Hepatitis B e antigen (HBeAg) is an important immunomodulator for promoting host immune tolerance during chronic hepatitis B (CHB) infection.  In patients with CHB, HBeAg loss and seroconversion represent partial immune control of CHB infection and are regarded as valuable endpoints.  However, the current approved treatments have only a limited efficacy in achieving HBeAg seroconversion in HBeAg-pos. patients.  Hepatitis B virus (HBV) core protein has been recognized as an attractive antiviral target, and two classes of core protein allosteric modulator (CpAM) have been discovered: the phenylpropenamides (PPAs) and the heteroaryldihydropyrimidines (HAPs).  However, their differentiation and potential therapeutic benefit beyond HBV DNA inhibition remain to be seen.  Here, we show that in contrast to PPA series compd. AT-130, a HAP CpAM, HAP_R01, reduced HBeAg levels in multiple in vitro and in vivo HBV exptl. models.  Mechanistically, we found that HAP_R01 treatment caused the misassembly of capsids formed by purified HBeAg in vitro.  In addn., HAP_R01 directly reduces HBeAg levels by inducing intracellular precore protein misassembly and aggregation.  Using a HAP_R01-resistant mutant, we found that HAP_R01-mediated HBeAg and core protein redns. were mediated through the same mechanism.  Furthermore, HAP_R01 treatment substantially reduced serum HBeAg levels in an HBV mouse model.  Conclusion: Unlike PPA series compd. AT-130, HAP_R01 not only inhibits HBV DNA levels but also directly reduces HBeAg through induction of its misassembly.  HAP_R01, as well as other similar CpAMs, has the potential to achieve higher anti-HBeAg seroconversion rates than currently approved therapies for patients with CHB.  Our findings also provide guidance for dose selection when designing clin. trials with mols. from HAP series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptmHAW9ju4QrVg90H21EOLACvtfcHk0ljT_GbhMzFYSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yis7fO&md5=d400c27579525d3309ce818480deba6e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1002%2Fhep.30514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fhep.30514%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DZ.%2BP.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%2BH.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DGao%26aufirst%3DL.%26atitle%3DDirect%2520inhibition%2520of%2520hepatitis%2520B%2520e%2520antigen%2520by%2520core%2520protein%2520allosteric%2520modulator%26jtitle%3DHepatology%26date%3D2019%26volume%3D70%26spage%3D11%26epage%3D24%26doi%3D10.1002%2Fhep.30514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, D.</span></span> <span> </span><span class="NLM_article-title">Establishment of a stable expression and replication cell line transfected with HBV genotype C prevailed in China</span>. <i>Med. Pharm. J. Chin.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1428</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2010&pages=1425-1428&author=L.+Wangauthor=W.+Liuauthor=Y.+Liuauthor=D.+Jiauthor=L.+Siauthor=Y.+Zhongauthor=D.+Xu&title=Establishment+of+a+stable+expression+and+replication+cell+line+transfected+with+HBV+genotype+C+prevailed+in+China"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DD.%26aulast%3DSi%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DD.%26atitle%3DEstablishment%2520of%2520a%2520stable%2520expression%2520and%2520replication%2520cell%2520line%2520transfected%2520with%2520HBV%2520genotype%2520C%2520prevailed%2520in%2520China%26jtitle%3DMed.%2520Pharm.%2520J.%2520Chin.%26date%3D2010%26volume%3D35%26spage%3D1425%26epage%3D1428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serajuddin, A. T. M.</span></span> <span> </span><span class="NLM_article-title">Salt formation to improve drug solubility</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">616</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2007.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.addr.2007.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=17619064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2007&pages=603-616&author=A.+T.+M.+Serajuddin&title=Salt+formation+to+improve+drug+solubility&doi=10.1016%2Fj.addr.2007.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Salt formation to improve drug solubility</span></div><div class="casAuthors">Serajuddin, Abu T. M.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">603-616</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Salt formation is the most common and effective method of increasing soly. and dissoln. rates of acidic and basic drugs.  In this article, physicochem. principles of salt soly. are presented, with special ref. to the influence of pH-soly. profiles of acidic and basic drugs on salt formation and dissoln.  Non-ideality of salt soly. due to self-assocn. in soln. is also discussed.  Whether certain acidic or basic drugs would form salts and, if salts are formed, how easily they would dissoc. back into their free acid or base forms depend on interrelationships of several factors, such as S0 (intrinsic soly.), pH, pKa, Ksp (soly. product) and pHmax (pH of max. soly.).  The interrelationships of these factors are elaborated and their influence on salt screening and the selection of optimal salt forms for development are discussed.  Factors influencing salt dissoln. under various pH conditions, and esp. in reactive media and in presence of excess common ions, are discussed, with practical ref. to the development of solid dosage forms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZdRvmSUWvN7Vg90H21EOLACvtfcHk0lgCGJPJyjbWnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpsFGksrc%253D&md5=6e77d1385622af6973a6d763746ad0db</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2007.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2007.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DSerajuddin%26aufirst%3DA.%2BT.%2BM.%26atitle%3DSalt%2520formation%2520to%2520improve%2520drug%2520solubility%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2007%26volume%3D59%26spage%3D603%26epage%3D616%26doi%3D10.1016%2Fj.addr.2007.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althage, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chisari, F. V.</span></span> <span> </span><span class="NLM_article-title">Hydrodynamic injection of viral DNA: A mouse model of acute hepatitis B virus infection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">13825</span>– <span class="NLM_lpage">13830</span>, <span class="refDoi"> DOI: 10.1073/pnas.202398599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.202398599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=12374864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=13825-13830&author=P.+L.+Yangauthor=A.+Althageauthor=J.+Chungauthor=F.+V.+Chisari&title=Hydrodynamic+injection+of+viral+DNA%3A+A+mouse+model+of+acute+hepatitis+B+virus+infection&doi=10.1073%2Fpnas.202398599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrodynamic injection of viral DNA: a mouse model of acute hepatitis B virus infection</span></div><div class="casAuthors">Yang, Priscilla L.; Althage, Alana; Chung, Josan; Chisari, Francis V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">13825-13830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is a prototype for liver-specific pathogens in which the failure of the immune system to mount an effective response leads to chronic infection.  The authors' understanding of the immune response to HBV is incomplete, largely due to the narrow host restriction of this pathogen and the limitations of existing exptl. models.  The authors have developed a murine model for studying human HBV replication, immunogenicity, and control.  After transfection of hepatocytes in vivo with a replication-competent, over-length, linear HBV genome, viral antigens and replicative intermediates were synthesized and virus was secreted into the blood.  Viral antigens disappeared from the blood as early as 7 days after transfection, coincident with the appearance of antiviral antibodies.  HBV transcripts and replicative intermediates disappeared from the liver by day 15, after the appearance of antiviral CD8 + T cells.  In contrast, the virus persisted for at least 81 days after transfection of NOD/Scid mice, which lack functional T cells, B cells, and natural killer (NK) cells.  Thus, the outcome of hydrodynamic transfection of HBV depends on the host immune response, as it is during a natural infection.  The methods the authors describe will allow the examn. of viral dynamics in a tightly controlled in vivo system, the application of mutagenesis methods to the study of the HBV life cycle in vivo, and the dissection of the immune response to HBV using genetically modified mice whose immunoregulatory and immune effector functions have been deleted or overexpressed.  In addn., this methodol. represents a prototype for the study of other known and to-be-discovered liver-specific pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo98nNA_w5YxrVg90H21EOLACvtfcHk0lgCGJPJyjbWnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D&md5=0c0f40c386fe47f6c57c413fccdaf3e5</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202398599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202398599%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DAlthage%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DChisari%26aufirst%3DF.%2BV.%26atitle%3DHydrodynamic%2520injection%2520of%2520viral%2520DNA%253A%2520A%2520mouse%2520model%2520of%2520acute%2520hepatitis%2520B%2520virus%2520infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D13825%26epage%3D13830%26doi%3D10.1073%2Fpnas.202398599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khojasteh, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C.</span></span> <i>Drug metabolism and pharmacokinetics quick guide.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_fpage">17</span>, <span class="refDoi"> DOI: 10.1007/978-1-4419-5629-3_2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1007%2F978-1-4419-5629-3_2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Khojasteh%2C+S.+C.%3B+Wong%2C+H.%3B+Hop%2C+C.+Drug+metabolism+and+pharmacokinetics+quick+guide.+2011%2C+17%2C+10.1007%2F978-1-4419-5629-3_2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-5629-3_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-5629-3_2%26sid%3Dliteratum%253Aachs%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DWong%26aufirst%3DH.%26aulast%3DHop%26aufirst%3DC.%26jtitle%3DDrug%2520metabolism%2520and%2520pharmacokinetics%2520quick%2520guide.%26date%3D2011%26spage%3D17%26doi%3D10.1007%2F978-1-4419-5629-3_2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sinz, M. W.</span></span> <span> </span><span class="NLM_article-title">Avoiding PXR and CAR activation and CYP3A4 enzyme induction</span>. <i>Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1007/7355_2013_24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1007%2F7355_2013_24" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=159-190&author=M.+W.+Sinz&title=Avoiding+PXR+and+CAR+activation+and+CYP3A4+enzyme+induction&doi=10.1007%2F7355_2013_24"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1007%2F7355_2013_24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F7355_2013_24%26sid%3Dliteratum%253Aachs%26aulast%3DSinz%26aufirst%3DM.%2BW.%26atitle%3DAvoiding%2520PXR%2520and%2520CAR%2520activation%2520and%2520CYP3A4%2520enzyme%2520induction%26jtitle%3DTop.%2520Med.%2520Chem.%26date%3D2013%26volume%3D9%26spage%3D159%26epage%3D190%26doi%3D10.1007%2F7355_2013_24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mase, T.</span></span> <span> </span><span class="NLM_article-title">A general palladium-catalyzed coupling of aryl bromides/triflates and thiols</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4587</span>– <span class="NLM_lpage">4590</span>, <span class="refDoi"> DOI: 10.1021/ol047996t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol047996t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptVartbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=4587-4590&author=T.+Itohauthor=T.+Mase&title=A+general+palladium-catalyzed+coupling+of+aryl+bromides%2Ftriflates+and+thiols&doi=10.1021%2Fol047996t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">A general palladium-catalyzed coupling of aryl bromides/triflates and thiols</span></div><div class="casAuthors">Itoh, Takahiro; Mase, Toshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4587-4590</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient palladium-catalyzed carbon-sulfur bond formation reaction of aryl bromides, triflates, and activated aryl chloride have been developed.  The reaction showed tolerance to a wide variety of aryl thiols and alkyl thiols which gave aryl sulfides, e.g., I, in good yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ47wzYjppQ7Vg90H21EOLACvtfcHk0lgx-W-k70XjwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXptVartbg%253D&md5=a58e03f1934099065cf2500c276d89a1</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fol047996t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol047996t%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DMase%26aufirst%3DT.%26atitle%3DA%2520general%2520palladium-catalyzed%2520coupling%2520of%2520aryl%2520bromides%252Ftriflates%2520and%2520thiols%26jtitle%3DOrg.%2520Lett.%26date%3D2004%26volume%3D6%26spage%3D4587%26epage%3D4590%26doi%3D10.1021%2Fol047996t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pu, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christesen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ku, Y. Y.</span></span> <span> </span><span class="NLM_article-title">A simple and highly effective oxidative chlorination protocol for the preparation of arenesulfonyl chlorides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">418</span>– <span class="NLM_lpage">421</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2009.11.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1016%2Fj.tetlet.2009.11.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=418-421&author=Y.+M.+Puauthor=A.+Christesenauthor=Y.+Y.+Ku&title=A+simple+and+highly+effective+oxidative+chlorination+protocol+for+the+preparation+of+arenesulfonyl+chlorides&doi=10.1016%2Fj.tetlet.2009.11.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">A simple and highly effective oxidative chlorination protocol for the preparation of arenesulfonyl chlorides</span></div><div class="casAuthors">Pu, Yu-Ming; Christesen, Alan; Ku, Yi-Yin</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">418-421</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">2,4-Dichloro-5,5-dimethylhydantoin (DCDMH) was found to be a mild and efficient reagent for the direct oxidative conversion of sulfur compds. to the corresponding arenesulfonyl chlorides in good to excellent yields through oxidative chlorination.  The method is suitable for many types of sulfur substrates (thiols, disulfides, and benzylic sulfides).  The overall process is simple, practical, and it provides convenient access to a variety of aryl or heteroarylsulfonyl chlorides.  The mild reaction conditions and the broad substrate scope render this method attractive and complementary to existing syntheses of aryl or heteroarylsulfonyl chlorides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAa01G0iIlebVg90H21EOLACvtfcHk0lgx-W-k70XjwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGmsrvO&md5=99e2389d65e31bc57a2a3c740b59df98</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2009.11.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2009.11.047%26sid%3Dliteratum%253Aachs%26aulast%3DPu%26aufirst%3DY.%2BM.%26aulast%3DChristesen%26aufirst%3DA.%26aulast%3DKu%26aufirst%3DY.%2BY.%26atitle%3DA%2520simple%2520and%2520highly%2520effective%2520oxidative%2520chlorination%2520protocol%2520for%2520the%2520preparation%2520of%2520arenesulfonyl%2520chlorides%26jtitle%3DTetrahedron%2520Lett.%26date%3D2010%26volume%3D51%26spage%3D418%26epage%3D421%26doi%3D10.1016%2Fj.tetlet.2009.11.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabe, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlachou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pante, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helenius, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kann, M.</span></span> <span> </span><span class="NLM_article-title">Nuclear import of hepatitis B virus capsids and release of the viral genome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">9849</span>– <span class="NLM_lpage">9854</span>, <span class="refDoi"> DOI: 10.1073/pnas.1730940100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=10.1073%2Fpnas.1730940100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=12909718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvVejtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=9849-9854&author=B.+Rabeauthor=A.+Vlachouauthor=N.+Panteauthor=A.+Heleniusauthor=M.+Kann&title=Nuclear+import+of+hepatitis+B+virus+capsids+and+release+of+the+viral+genome&doi=10.1073%2Fpnas.1730940100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear import of hepatitis B virus capsids and release of the viral genome</span></div><div class="casAuthors">Rabe, Birgit; Vlachou, Angelika; Pante, Nelly; Helenius, Ari; Kann, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9849-9854</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">While studying the import of the hepatitis B virus genome into the nucleus of permeabilized tissue culture cells, we found that viral capsids were imported in intact form through the nuclear pore into the nuclear basket.  Import depended on phosphorylation of the capsid protein and was mediated by the cellular transport receptors importin α and β.  Virus-derived capsids that contained the mature viral genome were able to release the viral DNA and capsid protein into the nucleoplasm.  The uncoating reaction was independent of Ran, a GTP-binding enzyme responsible for dissocg. other imported cargoes from the inner face of the nuclear pore.  Immature capsids that did not contain the mature viral genome reached the basket but did not release capsid proteins nor immature genomes into the nucleoplasm.  The different fate of mature and immature capsids after passing the nuclear pore indicates that the outcome of a nuclear import event may be regulated within the nuclear basket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp7BEG_HbaGLVg90H21EOLACvtfcHk0lhp-GFW438DPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvVejtbg%253D&md5=9b27d82ed78f3d24d7926919ea199f0c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1730940100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1730940100%26sid%3Dliteratum%253Aachs%26aulast%3DRabe%26aufirst%3DB.%26aulast%3DVlachou%26aufirst%3DA.%26aulast%3DPante%26aufirst%3DN.%26aulast%3DHelenius%26aufirst%3DA.%26aulast%3DKann%26aufirst%3DM.%26atitle%3DNuclear%2520import%2520of%2520hepatitis%2520B%2520virus%2520capsids%2520and%2520release%2520of%2520the%2520viral%2520genome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D9849%26epage%3D9854%26doi%3D10.1073%2Fpnas.1730940100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhp-GFW438DPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torosyan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebart, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumanian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">An aggregation advisor for ligand discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7076</span>– <span class="NLM_lpage">7087</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01105</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01105" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7076-7087&author=J.+J.+Irwinauthor=D.+Duanauthor=H.+Torosyanauthor=A.+K.+Doakauthor=K.+T.+Ziebartauthor=T.+Sterlingauthor=G.+Tumanianauthor=B.+K.+Shoichet&title=An+aggregation+advisor+for+ligand+discovery&doi=10.1021%2Facs.jmedchem.5b01105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">An Aggregation Advisor for Ligand Discovery</span></div><div class="casAuthors">Irwin, John J.; Duan, Da; Torosyan, Hayarpi; Doak, Allison K.; Ziebart, Kristin T.; Sterling, Teague; Tumanian, Gurgen; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7076-7087</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation of org. mols. is the dominant mechanism for artifactual inhibition of proteins, and controls against it are widely deployed.  Notwithstanding an increasingly detailed understanding of this phenomenon, a method to reliably predict aggregation has remained elusive.  Correspondingly, active mols. that act via aggregation continue to be found in early discovery campaigns and remain common in the literature.  Over the past decade, over 12 thousand aggregating org. mols. have been identified, potentially enabling a precedent-based approach to match known aggregators with new mols. that may be expected to aggregate and lead to artifacts.  We investigate an approach that uses lipophilicity, affinity, and similarity to known aggregators to advise on the likelihood that a candidate compd. is an aggregator.  In prospective exptl. testing, five of seven new mols. with Tanimoto coeffs. (Tc's) between 0.95 and 0.99 to known aggregators aggregated at relevant concns.  Ten of 19 with Tc's between 0.94 and 0.90 and three of seven with Tc's between 0.89 and 0.85 also aggregated.  Another three of the predicted compds. aggregated at higher concns.  This method finds that 61 827 or 5.1% of the ligands acting in the 0.1 to 10 μM range in the medicinal chem. literature are at least 85% similar to a known aggregator with these phys. properties and may aggregate at relevant concns.  Intriguingly, only 0.73% of all drug-like com. available compds. resemble the known aggregators, suggesting that colloidal aggregators are enriched in the literature.  As a percentage of the literature, aggregator-like compds. have increased 9-fold since 1995, partly reflecting the advent of high-throughput and virtual screens against mol. targets.  Emerging from this study is an aggregator advisor database and tool (http://advisor.bkslab.org), free to the community, that may help distinguish between fruitful and artifactual screening hits acting by this mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTKGJSfIz30bVg90H21EOLACvtfcHk0lgNxUgCs6pnkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM&md5=d3112f9f3a57a2ac61a4b32f31778b1d</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01105%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DDuan%26aufirst%3DD.%26aulast%3DTorosyan%26aufirst%3DH.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DZiebart%26aufirst%3DK.%2BT.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DTumanian%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DAn%2520aggregation%2520advisor%2520for%2520ligand%2520discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7076%26epage%3D7087%26doi%3D10.1021%2Facs.jmedchem.5b01105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T2P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T2P','PDB','5T2P'); return false;">PDB: 5T2P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE','PDB','5WRE'); return false;">PDB: 5WRE</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i117"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00292">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_13457"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00292?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00292</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tables of anti-HBV activity and cytotoxicity of the compounds and purity of tested compounds, figures of <i>in vivo</i> anti-HBV efficiency, intracellular RNA level, <sup>1</sup>H and <sup>13</sup>C NMR spectra, and HPLC spectra, and discussions of procedures of synthesis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00292/suppl_file/jm0c00292_si_001.pdf">jm0c00292_si_001.pdf (9.27 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00292&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00292%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-11" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00292" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996cc81b80195f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
