
<html lang="en"     class="pb-page"  data-request-id="e4bd61c4-ef9a-4083-84ef-44ecd4bc1054"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-12;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b00204;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease" /></meta><meta name="dc.Creator" content="Mariama  Jaiteh" /></meta><meta name="dc.Creator" content="Alexey  Zeifman" /></meta><meta name="dc.Creator" content="Marcus  Saarinen" /></meta><meta name="dc.Creator" content="Per  Svenningsson" /></meta><meta name="dc.Creator" content="Jose  Bréa" /></meta><meta name="dc.Creator" content="Maria Isabel  Loza" /></meta><meta name="dc.Creator" content="Jens  Carlsson" /></meta><meta name="dc.Description" content="Modulation of multiple biological targets with a single drug can lead to synergistic therapeutic effects and has been demonstrated to be essential for efficient treatment of CNS disorders. However,..." /></meta><meta name="Description" content="Modulation of multiple biological targets with a single drug can lead to synergistic therapeutic effects and has been demonstrated to be essential for efficient treatment of CNS disorders. However,..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 24, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00204" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00204" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00204" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00204" /></link>
        
    
    

<title>Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00204" /></meta><meta property="og:title" content="Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0008.jpeg" /></meta><meta property="og:description" content="Modulation of multiple biological targets with a single drug can lead to synergistic therapeutic effects and has been demonstrated to be essential for efficient treatment of CNS disorders. However, rational design of compounds that interact with several targets is very challenging. Here, we demonstrate that structure-based virtual screening can guide the discovery of multi-target ligands of unrelated proteins relevant for Parkinson’s disease. A library with 5.4 million molecules was docked to crystal structures of the A2A adenosine receptor (A2AAR) and monoamine oxidase B (MAO-B). Twenty-four compounds that were among the highest ranked for both binding sites were evaluated experimentally, resulting in the discovery of four dual-target ligands. The most potent compound was an A2AAR antagonist with nanomolar affinity (Ki = 19 nM) and inhibited MAO-B with an IC50 of 100 nM. Optimization guided by the predicted binding modes led to the identification of a second potent dual-target scaffold. The two discovered scaffolds were shown to counteract 6-hydroxydopamine-induced neurotoxicity in dopaminergic neuronal-like SH-SY5Y cells. Structure-based screening can hence be used to identify ligands with specific polypharmacological profiles, providing new avenues for drug development against complex diseases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00204"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00204">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00204&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00204&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00204&amp;href=/doi/10.1021/acs.jmedchem.8b00204" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 5269-5278</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00198" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00250" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Mariama Jaiteh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mariama Jaiteh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Science for Life Laboratory, Department of Cell and Molecular Biology, Uppsala University, BMC Box 596, SE-751 24 Uppsala, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mariama++Jaiteh">Mariama Jaiteh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexey Zeifman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexey Zeifman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Science for Life Laboratory, Department of Cell and Molecular Biology, Uppsala University, BMC Box 596, SE-751 24 Uppsala, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexey++Zeifman">Alexey Zeifman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcus Saarinen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcus Saarinen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center of Molecular Medicine, Department of Physiology and Pharmacology, Karolinska Institute, SE-171 77 Stockholm, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcus++Saarinen">Marcus Saarinen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Per Svenningsson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Per Svenningsson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center of Molecular Medicine, Department of Physiology and Pharmacology, Karolinska Institute, SE-171 77 Stockholm, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Per++Svenningsson">Per Svenningsson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jose Bréa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jose Bréa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">USEF Screening Platform-BioFarma Research Group, Centre for Research in Molecular Medicine and Chronic Diseases, University of Santiago de Compostela, 15706 Santiago de Compostela, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jose++Br%C3%A9a">Jose Bréa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maria Isabel Loza</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maria Isabel Loza</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">USEF Screening Platform-BioFarma Research Group, Centre for Research in Molecular Medicine and Chronic Diseases, University of Santiago de Compostela, 15706 Santiago de Compostela, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maria+Isabel++Loza">Maria Isabel Loza</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jens Carlsson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jens Carlsson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Science for Life Laboratory, Department of Cell and Molecular Biology, Uppsala University, BMC Box 596, SE-751 24 Uppsala, Sweden</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#a4cec1cad78ac7c5d6c8d7d7cbcae4cdc7c98ad1d18ad7c1"><span class="__cf_email__" data-cfemail="c1aba4afb2efa2a0b3adb2b2aeaf81a8a2acefb4b4efb2a4">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jens++Carlsson">Jens Carlsson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4623-2977" title="Orcid link">http://orcid.org/0000-0003-4623-2977</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00204&amp;href=/doi/10.1021%2Facs.jmedchem.8b00204" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 5269–5278</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 24, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 February 2018</li><li><span class="item_label"><b>Published</b> online</span>24 May 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 June 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00204" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00204</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00204"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2949</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">23</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00204" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mariama&quot;,&quot;last_name&quot;:&quot;Jaiteh&quot;},{&quot;first_name&quot;:&quot;Alexey&quot;,&quot;last_name&quot;:&quot;Zeifman&quot;},{&quot;first_name&quot;:&quot;Marcus&quot;,&quot;last_name&quot;:&quot;Saarinen&quot;},{&quot;first_name&quot;:&quot;Per&quot;,&quot;last_name&quot;:&quot;Svenningsson&quot;},{&quot;first_name&quot;:&quot;Jose&quot;,&quot;last_name&quot;:&quot;Bréa&quot;},{&quot;first_name&quot;:&quot;Maria&quot;,&quot;last_name&quot;:&quot;Isabel Loza&quot;},{&quot;first_name&quot;:&quot;Jens&quot;,&quot;last_name&quot;:&quot;Carlsson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;5269-5278&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00204&quot;},&quot;abstract&quot;:&quot;Modulation of multiple biological targets with a single drug can lead to synergistic therapeutic effects and has been demonstrated to be essential for efficient treatment of CNS disorders. However, rational design of compounds that interact with several targets is very challenging. Here, we demonstrate that structure-based virtual screening can guide the discovery of multi-target ligands of unrelated proteins relevant for Parkinson’s disease. A library with 5.4 million molecules was docked to crystal structures of the A2A adenosine receptor (A2AAR) and monoamine oxidase B (MAO-B). Twenty-four compounds that were among the highest ranked for both binding sites were evaluated experimentally, resulting in the discovery of four dual-target ligands. The most potent compound was an A2AAR antagonist with nanomolar affinity (Ki = 19 nM) and inhibited MAO-B with an IC50 of 100 nM. Optimization guided by the predicted binding modes led to the identification of a second potent dual-target scaffold. The two discovered sc&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00204&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00204" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00204&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00204" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00204&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00204" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00204&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00204&amp;href=/doi/10.1021/acs.jmedchem.8b00204" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00204" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00204" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00204%26sid%3Dliteratum%253Aachs%26pmid%3D29792714%26genre%3Darticle%26aulast%3DJaiteh%26date%3D2018%26atitle%3DDocking%2BScreens%2Bfor%2BDual%2BInhibitors%2Bof%2BDisparate%2BDrug%2BTargets%2Bfor%2BParkinson%25E2%2580%2599s%2BDisease%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D12%26spage%3D5269%26epage%3D5278%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/jmcmar.2018.61.issue-12/20180628/jmcmar.2018.61.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Modulation of multiple biological targets with a single drug can lead to synergistic therapeutic effects and has been demonstrated to be essential for efficient treatment of CNS disorders. However, rational design of compounds that interact with several targets is very challenging. Here, we demonstrate that structure-based virtual screening can guide the discovery of multi-target ligands of unrelated proteins relevant for Parkinson’s disease. A library with 5.4 million molecules was docked to crystal structures of the A<sub>2A</sub> adenosine receptor (A<sub>2A</sub>AR) and monoamine oxidase B (MAO-B). Twenty-four compounds that were among the highest ranked for both binding sites were evaluated experimentally, resulting in the discovery of four dual-target ligands. The most potent compound was an A<sub>2A</sub>AR antagonist with nanomolar affinity (<i>K</i><sub>i</sub> = 19 nM) and inhibited MAO-B with an IC<sub>50</sub> of 100 nM. Optimization guided by the predicted binding modes led to the identification of a second potent dual-target scaffold. The two discovered scaffolds were shown to counteract 6-hydroxydopamine-induced neurotoxicity in dopaminergic neuronal-like SH-SY5Y cells. Structure-based screening can hence be used to identify ligands with specific polypharmacological profiles, providing new avenues for drug development against complex diseases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Since the lock-and-key and receptor concepts were introduced more than a century ago,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> drug development has increasingly been focusing on identifying agents that modulate the activity of a single target. The belief that the magic bullet, a compound with high affinity and selectivity for a specific protein, has high potential as a drug candidate is now dominating the pharmaceutical industry. However, the “one target–one drug” paradigm neglects the fact that many diseases are multifactorial and efficient treatment in such cases will require modulation of several proteins. For example, it is now widely accepted that the efficacy of antipsychotic drugs can be ascribed to interactions with multiple members from the G protein-coupled receptor (GPCR) family.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Compounds that mediate their effects via several targets, which has been coined polypharmacology, show improved efficacy compared to single-target drugs by acting synergistically and avoid side effects associated with combination therapy.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">The realization that the therapeutic effects of several medications rely on polypharmacology<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> has sparked interest in applying this strategy to neurological and neuropsychiatric disorders for which traditional drug development approaches have fallen short.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> For example, the recently approved antiparkinson drug safinamide inhibits monoamine oxidase B as well as sodium and calcium channels, which may contribute to its unique properties.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The vast majority of the known multi-target drugs has been discovered by serendipity, and given the major efforts involved in the generation of single-target lead candidates, development of compounds with dual activity has been anticipated to be very difficult. Rational design of polypharmacology has mainly involved proteins that are either closely related or recognize similar biogenic molecules, e.g., kinases or aminergic GPCRs, but has rarely been accomplished for disparate targets.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> Access to atomic resolution information for proteins relevant for the same indication gives the opportunity to design multi-target ligands based on the binding site structures.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> A structure-based approach should enable identification of drugs to unrelated targets by taking advantage of common binding site features that are not apparent from protein sequence or ligand similarity.</div><div class="NLM_p">Two targets relevant for drug development against Parkinson’s disease (PD) were selected to explore the possibility to discover dual-target ligands using structure-based virtual screening. PD is characterized by a progressive loss of dopaminergic neurons, which results in symptoms such as trembling, stiffness, and bradykinesia. To overcome the decreased dopamine levels in the brain, treatment of PD mainly relies on the dopamine precursor levodopa. However, more efficient drugs are urgently needed due to side effects such as dyskinesia and the gradual loss of levodopa efficacy.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Alternative targets for development of antiparkinson drugs include non-dopaminergic members of the GPCR superfamily and enzymes involved in the degradation of monoamine neurotransmitters.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> From these two different target classes, we focused on identifying lead candidates with dual-target activity at the A<sub>2A</sub> adenosine receptor (A<sub>2A</sub>AR) and monoamine oxidase B (MAO-B). The idea of developing dual-target A<sub>2A</sub>AR/MAO-B inhibitors emerged from the serendipitous discovery that the A<sub>2A</sub> antagonist 8-(3-chlorostyryl)caffeine (CSC) also inhibited MAO-B.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Although CSC showed promising neuroprotective effects in experimental PD models,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> it has been considered to be undesirable as starting point for drug development due to low solubility and sensitivity to light-induced degradation.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Novel dual-target A<sub>2A</sub>AR/MAO-B ligands are hence needed to further assess the potential of the multi-target approach in treatment of PD.</div><div class="NLM_p">A dual A<sub>2A</sub>AR/MAO-B inhibitor would exert the symptomatic and neuroprotective effects of A<sub>2A</sub>AR antagonism<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> combined with the advantages of sustained dopaminergic signaling due to inhibition of MAO-B.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> As the targets are involved in different biochemical pathways, additive or even synergistic therapeutic effects could be expected for dual-target compounds.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> However, considering the dissimilar neurotransmitters recognized by the two proteins and the disparate binding sites revealed by atomic resolution crystal structures (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), design of dual-target compounds should be very challenging. In the present work, molecular docking screens of a commercial chemical library against crystal structures of the A<sub>2A</sub>AR and MAO-B binding sites were carried out to identify dual-target ligands. Twenty-four compounds that were top-ranked in the A<sub>2A</sub>AR and MAO-B screens were assayed experimentally. The screening results enabled us to assess the prospects of designing multi-target ligands using a structure-based approach and to discover starting points for development of a novel class of antiparkinson drugs.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Targets for development of dual-target antiparkinson agents. (a) Crystal structures of the A<sub>2A</sub>AR (gray) and MAO-B (green) are shown as cartoons together with 2D representations of adenosine and dopamine. (b) 2D representation of CSC, a dual-target A<sub>2A</sub>AR and MAO-B inhibitor.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Docking Screens for Dual-Target Ligands</h3><div class="NLM_p">In order to identify dual-target ligands of the A<sub>2A</sub>AR and MAO-B, molecular docking screens against the two binding sites were carried out. Sets of known ligands<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> together with property-matched decoys<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> were first docked against available crystal structures to benchmark the binding sites for virtual screening, which was quantified using the adjusted logAUC (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S1</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In this step, no particular focus was put on the limited set of known dual-target ligands<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> to avoid introducing bias in the prospective screen toward these two scaffolds. Crystal structures of the A<sub>2A</sub>AR in an inactive conformation, which is the state relevant for development of antiparkinson drugs,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> were considered. An A<sub>2A</sub>AR structure that strongly enriched known ligands over decoys (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S1a</a>) and had a relatively open binding site conformation was selected (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PWH">3PWH</a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>) to increase the probability of identifying compounds that could be accommodated by several targets. The MAO-B structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V61">2V61</a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>) was determined in complex with a noncovalent inhibitor and also showed excellent ligand enrichment (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S1b</a>). To assess potential similarities between the A<sub>2A</sub>AR and MAO-B, the crystal structures and known ligands of the targets were compared. Binding site similarity was assessed using ProBIS,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> an algorithm designed to identify common structural elements of proteins. No statistically significant similarity between the binding sites was detected by this method (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S2</a>). Ligand 2D similarity was quantified by calculating the Tanimoto similarity coefficient (<i>T</i><sub>c</sub>) for all pairs of A<sub>2A</sub>AR (3898 compounds) and MAO-B (1671 compounds) ligands from the ChEMBL database<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S2</a>). 99.9% of the <i>T</i><sub>c</sub> values were <0.30, which suggested that the A<sub>2A</sub>AR and MAO-B recognize vastly different ligand chemotypes.</div><div class="NLM_p last">Two chemical libraries containing 0.8 million fragment-like compounds (molecular weight of <250 Da) and 4.6 million lead-like compounds (250 Da < molecular weight < 350 Da) from the ZINC database<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> were docked separately to the A<sub>2A</sub>AR and MAO-B structures using DOCK3.6.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> On the basis of previous prospective docking screens against GPCRs,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> the fragment library was likely to give a higher hit rate but at the expense of ligand potency. Conversely, the lead-like compounds were less likely to fit in both binding sites, but hits could be expected to have higher affinity as larger molecules can form more interactions. Several thousand orientations were sampled in the binding sites for each of the 5.4 million molecules with the proteins kept rigid, resulting in billions of predicted complexes that were evaluated using the DOCK3.6 scoring function. The compounds in the fragment- and lead-like libraries were first ranked based on their docking energies<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> for the A<sub>2A</sub>AR and MAO-B. A consensus score was then calculated for each compound as the sum of the ranks from the two screens. In order to identify dual target ligands, the 500 compounds with the lowest consensus scores were inspected visually. In the compound selection step, energy terms neglected by the docking scoring function, commercial availability, and the novelty of the predicted ligands were also considered, as described previously.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> A set of 24 compounds (<b>1</b>–<b>24</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S3</a>), comprising 13 fragment- and 11 lead-like compounds, was finally prioritized for experimental evaluation. All of the selected compounds had negative docking energies for both targets, indicating favorable interactions between the binding sites and the predicted ligands. The ranks of the selected fragment-like compounds ranged from 34 to 1273 for the A<sub>2A</sub>AR and from 10 to 2664 for MAO-B. The predicted lead-like compounds had ranks from 465 to 11760 for the A<sub>2A</sub>AR and from 422 to 14642 for MAO-B. All the selected compounds were hence ranked among the top 0.3% of the screened chemical libraries.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Experimental Evaluation of Docking Predictions</h3><div class="NLM_p">The 24 predicted ligands were evaluated experimentally at both the A<sub>2A</sub>AR and MAO-B. For the A<sub>2A</sub>AR, six compounds showed significant (>60%) displacement of radioligand at 30 μM, corresponding to a screening hit rate of 25%. Full dose–response curves were obtained for these ligands, and their <i>K</i><sub>i</sub> values ranged from 19 to 7100 nM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S4</a>). In the MAO-B enzyme assay, 12 compounds showed >70% inhibition at 30 μM, corresponding to a hit rate of 50%, and the IC<sub>50</sub> values of these inhibitors ranged from 61 to 8700 nM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S4</a>). Of the 14 compounds that showed activity for at least one of the targets, seven originated from the fragment-like library and seven were from the set of lead-like compounds. Promising starting points for development of single-target drugs hence emerged from both the fragment and lead-like libraries. As expected, the lead-like compounds were, on average, more potent than the fragment-like, in particular for the A<sub>2A</sub>AR. On the other hand, the ligand efficiencies (LE, corresponding to the binding energy per atom<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>) were typically higher for the fragment-like compounds. The virtual screening results are summarized in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overview of the structure-based screen for dual-target A<sub>2A</sub>AR/MAO-B ligands. The ZINC lead-like and fragment libraries were screened against A<sub>2A</sub>AR (gray) and MAO-B (green) crystal structures. Twenty-four top-ranked molecules were selected for experimental evaluation. Twelve and six compounds showed activity at MAO-B and A<sub>2A</sub>AR, respectively, and four of these displayed dual-activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S4</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Experimental Data for Identified Dual-Target Ligands</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0006.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Consensus rank from docking screens of a fragment (compound <b>1</b>) or lead-like (compounds <b>2</b>–<b>4</b>) library from the ZINC database. The ranks for the individual targets are shown in parentheses (A<sub>2A</sub>AR/MAO-B).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last"><i>K</i><sub>i</sub> value expressed as the mean ± SEM from three independent experiments performed in duplicate or triplicate.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">IC<sub>50</sub> value expressed as the mean ± SEM from three independent experiments performed in duplicate or triplicate.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">The maximal Tanimoto coefficient (ECFP4) when compared with all compounds with dual-activity at the A<sub>2A</sub>AR and MAO-B. The maximal Tanimoto coefficient (ECFP4) for all known ligands is shown in parentheses (A<sub>2A</sub>AR/MAO-B). 2D structures of the most similar compounds are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S5</a>.</p></div></div><div></div></div><div class="NLM_p">Among the 14 experimentally confirmed ligands, four compounds (<b>1</b>–<b>4</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) displayed activity at both the A<sub>2A</sub>AR and MAO-B, corresponding to a dual-target hit rate of 17%. Compounds <b>2</b> and <b>3</b> belonged to the same scaffold and the most potent of these (<b>3</b>) had a <i>K</i><sub>i</sub> of 19 nM at the A<sub>2A</sub>AR and inhibited MAO-B with an IC<sub>50</sub> of 100 nM. The two remaining compounds (<b>1</b> and <b>4</b>) had micromolar activities at both targets. Of these, compound <b>1</b> (<i>K</i><sub>i</sub> = 2700 nM at the A<sub>2A</sub>AR and IC<sub>50</sub> = 5000 nM for MAO-B) was considered to be a more promising starting point for optimization as it was fragment-sized and hence had higher LE values. The predicted binding modes of compounds <b>1</b> and <b>3</b> are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. The ligands were deeply buried in the A<sub>2A</sub>AR binding site and formed hydrogen bonds with Asn253<sup>6.55</sup> (superscripts represent Ballesteros–Weinstein residue numbering for GPCRs<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a>). In the MAO-B binding pocket, both compounds <b>1</b> and <b>3</b> formed hydrogen bonds with Tyr326 and were anchored in a hydrophobic pocket created by Phe103, Ile316, and Phe168. To assess the novelty of the discovered dual-target ligands, we calculated the Tanimoto similarity of the compounds to the two previously identified scaffolds with activity at both targets and all known ligands of the A<sub>2A</sub>AR or MAO-B (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S5</a>) from the ChEMBL database.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Compounds <b>1</b>–<b>4</b> had maximal Tanimoto similarity coefficients (<i>T</i><sub>c</sub>) ranging from 0.18 to 0.34 for the dual-target ligands, which is indicative of novel scaffolds.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The maximal <i>T</i><sub>c</sub> values for all known ligands were higher (0.25–0.50) but remained low for either the A<sub>2A</sub>AR or MAO-B in each case, demonstrating that dual-target activity would not have been expected from 2D similarity.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Predicted binding modes of two dual-target ligands: the docking poses of compounds <b>1</b> (a, b) and <b>3</b> (c, d) in the A<sub>2A</sub>AR (gray cartoon, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PWH">3PWH</a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>) and MAO-B binding sites (green cartoon, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V61">2V61</a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>). Key binding site residues and the ligands are shown in sticks. Hydrogen bonds are shown as black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure–Activity Relationships of Dual-Target Ligands</h3><div class="NLM_p">Exploration of structure–activity relationships for the two scaffolds represented by compounds <b>1</b> and <b>3</b> was guided by docking of commercially available analogs and structure-based design (compounds <b>1a</b>–<b>i</b> and <b>3a</b>–<b>k</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S6</a>). A set of commercially available analogs that were predicted to have the same binding mode as compound <b>1</b> was first selected for experimental evaluation, but none of these showed significant affinity for the A<sub>2A</sub>AR (<b>1c</b>–<b>i</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S7</a>). Based on the model of compound <b>1</b> in complex with the A<sub>2A</sub>AR, substituents filling the small subpocket in the vicinity of the 7-position of the benzimidazole ring could yield large increases of affinity, as demonstrated by previous structure-based drug design efforts.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> Two custom-synthesized compounds were evaluated experimentally based on this prediction (<b>1a</b>,<b>b</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The chlorine (<b>1a)</b> and methoxy (<b>1b)</b> substituted compounds had 27- and 5-fold better A<sub>2A</sub>AR binding affinities than compound <b>1</b>, respectively. These compounds were then evaluated in the MAO-B assay to assess dual-target activity. Whereas compound <b>1b</b> (IC<sub>50</sub> = 7000 nM) showed no improvement over compound <b>1</b>, compound <b>1a</b> was a submicromolar inhibitor of both MAO-B (IC<sub>50</sub> = 410 nM) and A<sub>2A</sub>AR (<i>K</i><sub>i</sub> = 99 nM). A representative dose–response curve for compound <b>1a</b> is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S3</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Experimental Data for Analogs of Compounds <b>1</b> and <b>3</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0007.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>i</sub> value expressed as the mean ± SEM from three independent experiments performed in duplicate or triplicate.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> value expressed as the mean ± SEM from three independent experiments performed in duplicate or triplicate. Inactive compounds (>10 000 nM) were tested in one experiment performed in triplicate.</p></div></div><div></div></div><div class="NLM_p last">Compounds <b>2</b> and <b>3</b> belonged to the same scaffold, and the most potent of these (<b>3</b>) had an affinity of 19 nM for the A<sub>2A</sub>AR and inhibited MAO-B with an IC<sub>50</sub> value of 100 nM. A total of nine analogs (<b>3a</b>–<b>i</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S8</a>) were assayed. Two commercially available compounds with substituents on the benzyl ring (<b>3a</b>,<b>b</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), which were predicted to occupy the top part of A<sub>2A</sub>AR binding site and a buried pocket in MAO-B, retained activity at both targets. Commercially available and custom-synthesized analogs devoid of the ester moiety, which may be metabolized in vivo, were also explored by replacing this group with amide groups (<b>3c</b>–<b>i</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S8</a>). As the ester moiety was facing the solvent in the predicted binding mode for the A<sub>2A</sub>AR, these analogs were predicted to maintain high activity. Binding assays confirmed that the four compounds were A<sub>2A</sub>AR ligands, but none of them inhibited MAO-B (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This result could be explained by the fact that the ester moiety was predicted to bind in a narrow channel connecting the two major subpockets of the MAO-B binding site, which makes it challenging to introduce new substituents in this region (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). As an additional control, we confirmed that the activity of compound <b>3</b> was not mediated by its metabolized form by evaluating the two products of hydrolysis, and these (<b>3j</b> and <b>3k</b>) were inactive at both targets (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S8</a>). To summarize, many of the tested analogs retained activity only at one of the targets and compound <b>3</b> was still the most potent compound in the series. A representative dose–response curve for compound <b>3</b> is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S3</a>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Functional Assays and Selectivity</h3><div class="NLM_p">Functional assays measuring cyclic AMP accumulation mediated by activation of the A<sub>2A</sub>AR in response to the agonist 5′-<i>N</i>-ethylcarboxamidoadenosine (NECA) in the presence of <b>1a</b> and <b>3</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S4</a>) confirmed that these compounds were antagonists, which is the desired efficacy in treatment of PD.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> To evaluate if compounds <b>1a</b> and <b>3</b> were noncovalent inhibitors of MAO-B, a reactivation experiment was carried out. MAO-B was first preincubated in the presence of substrate and either compounds <b>1a</b>, <b>3</b>, or the irreversible inhibitor tranylcypromine (<i>trans</i>-(±)-2-phenylcyclopropanamine).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> An excess of substrate was then added, which should displace reversible inhibitors, whereas no effect should be observed for an irreversible mechanism of action. The measured fluorescence increased for compounds <b>1a</b> and <b>3</b>, as expected for reversible inhibition, whereas only a small response was observed for the irreversible control inhibitor (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p">The selectivity profiles of compounds <b>1a</b> and <b>3</b> were further probed by evaluating binding affinities for the A<sub>1</sub>-, A<sub>2B</sub>-, and A<sub>3</sub>AR (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Compound <b>1a</b> was 10- and 2-fold selective for the A<sub>2A</sub>AR over the A<sub>1</sub>- and A<sub>2B</sub>AR, respectively, whereas compound <b>3</b> showed 12- and 10-fold selectivity over the same subtypes. Both compounds <b>1a</b> and <b>3</b> showed weak affinity for the A<sub>3</sub>AR subtype (K<sub>i</sub> > 10000 nM). Similarly, specificity for MAO-B over MAO-A was also confirmed for compounds <b>1a</b> and <b>3</b>. Compound <b>1a</b> did not show any significant inhibition of MAO-A at 10 μM, whereas compound <b>3</b> had an IC<sub>50</sub> = 4700 nM, resulting in 47-fold selectivity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).<named-content content-type="anchor" rid="tbl3" type="simple"></named-content></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Activity of Compounds <b>1a</b> and <b>3</b> at A<sub>1</sub>-, A<sub>2B</sub>-, and A<sub>3</sub>ARs and MAO-A</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">A<sub>1</sub>AR</th><th class="colsep0 rowsep0" align="center">A<sub>2B</sub>AR</th><th class="colsep0 rowsep0" align="center">A<sub>3</sub>AR</th><th class="colsep0 rowsep0" align="center">MAO-A</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">(<i>K</i><sub>i</sub>/nM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">(<i>K</i><sub>i</sub>/nM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">(%)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">(IC<sub>50</sub>/nM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1a</b></td><td class="colsep0 rowsep0" align="left">1000 ± 260</td><td class="colsep0 rowsep0" align="left">230 ± 20</td><td class="colsep0 rowsep0" align="left">50 ± 6.0%</td><td class="colsep0 rowsep0" align="left">>10000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">220 ± 80</td><td class="colsep0 rowsep0" align="left">190 ± 8.5</td><td class="colsep0 rowsep0" align="left">44 ± 6.5%</td><td class="colsep0 rowsep0" align="left">4700 ± 97</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Percent of displacement at 10 μM or <i>K</i><sub>i</sub> value expressed as the mean ± SEM from three independent experiments performed in duplicate or triplicate.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> value expressed as the mean ± SEM from three independent experiments performed in duplicate or triplicate. Inactive compounds (>10 000 nM) were tested in one experiment performed in triplicate.</p></div></div></div><div class="NLM_p last">Potential concerns in development of multi-target ligands are that the screening hits may be promiscuous scaffolds and that the activity may be due to assay interference or inhibition by colloidal aggregation. However, none of the discovered dual-target ligands contained any of the substructures identified by Baell et al. as common among frequent-hitters (pan-assay interference compounds, PAINS).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In the case of MAO-B, we also specifically controlled for assay interference by carrying out the experiments in the absence of the target protein with added product, which did not result in significant activity for compound <b>1a</b> or <b>3</b>. Furthermore, the MAO-B assays for compounds <b>1a</b> and <b>3</b> were performed in the presence and absence of detergent (Triton X-100, 0.01%) to control for artifactual inhibition due to colloidal aggregation.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In the case of aggregating compounds, activity will be diminished by Triton X-100, but no significant changes in the IC<sub>50</sub> values were observed for <b>1</b> or <b>3a</b> in the presence of detergent. Finally, we found that the commercially available <named-content content-type="anchor" rid="fig4" type="simple"></named-content>compounds <b>1</b> and <b>3</b> had been evaluated experimentally in a large number of screens deposited in the PubChem bioassay database and showed no activity in the vast majority of these (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S9</a>).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Compound <b>1</b> did not show any significant effect in 710 assays and was annotated as active in 39 assays, whereas compound <b>3</b> was inactive in in 766 screens and was only active in five cases. A more detailed analysis of the PubChem data revealed that compound <b>1</b> showed >50% activity at 10 μM for 16 different targets (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S10</a>), whereas no targets remained for compound <b>3</b> at this concentration.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Evaluation of Compounds <b>1a</b> and <b>3</b> in a Cell Viability Assay</h3><div class="NLM_p">Evaluation of a cytoprotective effect exerted by compounds <b>1a</b>, <b>3</b>, and CSC on differentiated dopaminergic neuronal-like SH-SY5Y cells was carried out using the resazurin assay. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, 20 μM 6-OHDA resulted in a 37 ± 1.3% reduction in cell viability and was chosen for evaluating compound impact. Pretreatment with 0.15 μM of both compounds <b>1a</b> and <b>3</b> as well as CSC caused statistically significant (<i>F</i><sub>4,115</sub> = 116.4; <i>p</i> < 0.0001) counteraction of the 6-OHDA induced cytotoxicity. Compounds <b>1a</b> and <b>3</b> reduced cytotoxicity by 14 ± 2.1% and 13 ± 2.2%, respectively. The previously discovered dual-target ligand CSC (A<sub>2A</sub>AR, <i>K</i><sub>i</sub> = 38 nM; MAO-B, IC<sub>50</sub> = 18 nM)<a onclick="showRef(event, 'ref14 ref24'); return false;" href="javascript:void(0);" class="ref ref14 ref24">(14,24)</a> resulted in an equally potent protection (13 ± 2.0%). There was no difference in the protective effects of the studied compounds.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Viability assessment on human dopaminergic neuronal-like cells treated with 6-OHDA toxin. Cells were treated with 0.15 μM <b>1a</b>, <b>3</b>, or CSC for 3 h prior to addition of 20 μM 6-OHDA. After 24 h, resazurin was added for another 2 h, whereupon cell viability was assayed. Data are shown as the mean ± SEM and were analyzed using one-way ANOVA with Newman–Keuls multiple comparison test: (###) <i>p</i> < 0.001 versus control and (∗∗∗) <i>p</i> < 0.001 versus 6-OHDA alone.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Three main results emerged from our structure-guided screens for dual-target inhibitors of a GPCR and enzyme relevant for development of antiparkinson drugs. First, four dual-target ligands were identified among the 14 screening hits that were confirmed active for either the A<sub>2A</sub>AR or MAO-B. Second, several of the dual-target compounds were potent. The most promising scaffold had astonishing nanomolar activity at both the A<sub>2A</sub>AR and MAO-B. The structural models also guided <named-content content-type="anchor" rid="fig5" type="simple"></named-content>improvement of a weak hit from the virtual screen to yield a second dual-target scaffold with submicromolar potencies. Third, two dual-target compounds showed protective effects in an in vitro PD model.</div><div class="NLM_p">Structure-based virtual screening has successfully discovered ligands of both GPCRs and enzymes,<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> but prediction of compounds with multi-target action involving disparate targets has rarely been accomplished. The high docking hit rates (25% and 50%, respectively) are in line with results obtained in previous virtual screening studies against either the A<sub>2A</sub>AR or MAO-B.<a onclick="showRef(event, 'ref20 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref20 ref43 ref44 ref45">(20,43−45)</a> The challenges involved in identifying dual-target ligands were reflected by the fact that only 17% of the tested compounds showed activity at both the A<sub>2A</sub>AR and MAO-B, which is lower than the hit rates obtained previously in docking screens against the A<sub>2A</sub>AR.<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> Whereas the hit rate for dual-target ligands was close to 4-fold lower than for the single targets, we were surprised to find that the potencies of the hits were not affected. Remarkably, compound <b>3</b> (<i>K</i><sub>i</sub> = 19 nM) not only is among the highest affinity A<sub>2A</sub>AR ligands to emerge from a virtual screen but also was very potent at MAO-B (IC<sub>50</sub> = 100 nM). The fidelity of the predicted binding mode for compound <b>3</b> was further supported by a crystal structure of the A<sub>2A</sub>AR in complex with a related ligand, which was published during the preparation of this manuscript (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> A question that arises is how two proteins can recognize the same compound despite lack of sequence and structural similarity.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Analysis of the docking results revealed some similarities among the top-ranked compounds and the known dual-target inhibitor CSC (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). For example, the dual-target scaffolds represented by compounds <b>1a</b> and <b>3</b> both contained two aromatic groups connected by polar moieties. The aromatic groups occupied hydrophobic subpockets at the ends of the extended binding sites, and the polar moieties were stabilized by hydrogen bonds to a side chain positioned centrally in both pockets (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Interestingly, CSC (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which differs from both compounds <b>1a</b> and <b>3</b> by 2D similarity, has a similar shape and composition of aromatic and polar groups. Despite the differences in sequence and structure, the docking scoring function was thus able to distinguish similarities in the binding site shape and polarity, supporting the use of structure-based virtual screening as a tool for identifying multi-target ligands of disparate targets.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the predicted binding mode of compound <b>3</b> to a crystal structure of a related antagonist in complex with the A<sub>2A</sub>AR. The dual-target ligand <b>3</b> is shown as sticks with orange carbon atoms. The cocrystallized antagonist (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UIG">5UIG</a><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>) is depicted in sticks with gray carbon atoms. The A<sub>2A</sub>AR is depicted as gray cartoons. Hydrogen bonds are shown as black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The potential of polypharmacology in treatment of complex diseases has led to increasing interest in rational design of multi-target ligands.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Such efforts have mainly focused on either optimizing a single-target scaffold for activity at a second target or combining the pharmacophore features of two single-target ligands into one compound.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The former strategy has mainly been successful for targets that are related or recognize similar biogenic molecules. An elegant example is the computer-aided optimization of an acetylcholinesterase inhibitor for activity at aminergic GPCRs using 2D similarity methods by Besnard et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> However, the general applicability of this approach is limited in that it requires access to training sets with known ligands of each target and depends on selecting a suitable starting point for optimization. This is illustrated by the fact that medicinal chemistry efforts to obtain dual-target A<sub>2A</sub>AR/MAO-B ligands based on an A<sub>2A</sub>AR scaffold (4<i>H</i>-3,1-benzothiazin-4-one) were successful,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> but optimization of phthalimide inhibitors of MAO-B for activity at the A<sub>2A</sub>AR failed.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> For unrelated targets, two ligand scaffolds can be linked to obtain a single compound with dual activity. For example, Jörg et al. designed dual-target ligands of the A<sub>2A</sub>AR and D<sub>2</sub> dopamine receptor by connecting the D<sub>2</sub> agonist ropinirole to an A<sub>2A</sub> antagonist via a chemical linker<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> whereas De Simone et al. constructed compounds for both the D<sub>3</sub> dopamine receptor and fatty acid amide hydrolase by combining two pharmacophores.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Active compounds were identified in both studies, but the combination of two existing ligands led to high molecular weight compounds with properties that may not be compatible with oral availability and blood–brain barrier penetration. In contrast, the structure-based virtual screening strategy explored in this work was not constrained to the limited chemical space spanned by known scaffolds. High-throughput docking allowed us to carry out an unbiased screen of several million compounds, leading to the discovery of four dual-target ligands. The most potent scaffolds were reversible MAO-B inhibitors and antagonized the A<sub>2A</sub>AR, which are the desired functional properties in development of PD drugs. The lack of activity of the two scaffolds in high-throughput screens against other proteins and controls made for assay interference and colloidal aggregation suggested that the dual-target activity was unlikely to be due to promiscuous or artifactual inhibition. Moreover, the inhibitors not only did have low molecular weight and good selectivity properties but also showed cytoprotective effects in dopaminergic neuronal-like cells. Compounds <b>1</b> and <b>3a</b> hence provide excellent starting points for development of dual-target A<sub>2A</sub>AR/MAO-B leads that could be evaluated in vivo for antiparkinson activity.</div><div class="NLM_p last">A few caveats of our approach should be mentioned. First, both the A<sub>2A</sub>AR and MAO-B have been demonstrated to recognize diverse scaffolds and bind drug-like compounds with high affinity. The high docking hit-rates obtained in this work may not be directly transferrable to less druggable targets. Second, access to multiple high-resolution crystal structures and large sets of known ligands for both targets contributed to the successful selection of compounds for experimental testing, but such wealth of information is not available in all drug discovery projects. Finally, it should be emphasized that the complexity of ligand design increases dramatically when a single scaffold is optimized for multi-target activity. In agreement with this concern and previous studies,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> we found that closely related analogs of our dual-target leads with activity for one target could be completely inactive for the other. Considering the challenges involved in optimization of dual-target compounds, access to atomic resolution information for the target binding sites will be crucial for lead development. In summary, our results demonstrate that molecular docking screening can guide discovery of ligands with specific polypharmacological profiles, which can contribute to development of drugs against complex diseases with improved efficacy and less side effects.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Molecular Docking Screens</h3><div class="NLM_p">The molecular docking calculations were carried out using DOCK3.6<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (<a href="http://dock.compbio.ucsf.edu/DOCK3.6/" class="extLink">http://dock.compbio.ucsf.edu/DOCK3.6/</a>) against a crystal structure of the A<sub>2A</sub>AR in complex with an antagonist (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PWH">3PWH</a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>) and of MAO-B in complex with an inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V61">2V61</a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>). The A<sub>2A</sub>AR structure was prepared by removing non-protein atoms, and thermostabilizing mutations were modified to correspond to the wild type sequence. Side chain rotamers for modified amino acids were selected based on other high-resolution crystal structures of the A<sub>2A</sub>AR. In the case of MAO-B, all protein residues, the FAD cofactor, and two crystallographic waters (residues 1239 and 1304) were retained in the docking calculations. The protonation states of ionizable side chains of residues Asp, Glu, Lys, and Arg were set according to their most probable state at pH 7. The protonation states of histidines in the binding sites were assigned based on the local hydrogen-bonding network. In the case of the A<sub>2A</sub>AR, the side chains of His250<sup>6.52</sup> and His278<sup>7.42</sup> were protonated at the Nε and Nδ positions, respectively.</div><div class="NLM_p">DOCK3.6 uses a flexible-ligand sampling algorithm that defines the binding site based on a set of matching spheres.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Forty-five matching spheres were used, and these were derived based on the cocrystallized ligands. The extent of ligand sampling was defined by the bin size, bin size overlap, and distance tolerance, and these parameters were set to 0.4 Å, 0.1 Å, and 1.5 Å, respectively, for both the docked molecules and binding sites matching spheres. Ligand conformations passing a steric filter were scored using the DOCK3.6 physics-based scoring function. The binding energy was estimated as the sum of the van der Waals and electrostatic energies, corrected for ligand desolvation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The protein atoms were described using a united atom version of the AMBER force field.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The dipole moment of the Asn253 side chain in the A<sub>2A</sub>AR was increased to favor hydrogen bonding to this residue, as described previously.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Force field parameters for the FAD cofactor in the MAO-B binding site were derived using the generalized AMBER force field.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The program CHEMGRID<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> was used to generate the van der Waals grids,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and the electrostatic potential maps of the binding sites were obtained using the program Delphi.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> The desolvation energy of a docked compound was obtained by scaling its transfer free energy between solvents with dielectric constants of 78 and 2 with a factor that reflects the degree of burial in the binding site.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> For the best scoring conformation of each docked molecule, 100 steps of rigid-body minimization were carried out.</div><div class="NLM_p last">A chemical library containing 5.4 million compounds was obtained from the ZINC database.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Known ligands<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> and DUD-E decoys<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> were prepared for docking using the ZINC database protocol.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> All tested compounds were sourced from commercial vendors (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S3</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">S6</a>). Purity (>95%) was confirmed by LC/MS for compounds <b>1</b>–<b>4</b>, <b>1a</b>, <b>1b</b>, <b>3a</b>, and <b>3b</b>. The selected molecules were also screened for substructures present in pan-assay interference compounds (PAINS) and did not contain any of these motifs.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> 2D Ligand and Binding Site Similarity Calculations</h3><div class="NLM_p last">Tanimoto similarity coefficients (<i>T</i><sub>c</sub>) were calculated with ECFP4 fingerprints using Screenmd (JChem version 15.10.12, ChemAxon, 2015). The maximal <i>T</i><sub>c</sub> value between each compound and the previously discovered dual-target A<sub>2A</sub>AR/MAO-B ligands<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> or all known ligands of the targets in the ChEMBL database<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (activity <10 μM) was calculated. Binding site similarity was quantified using the ProBis Web server<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> based on the A<sub>2A</sub>AR and MAO-B crystal structures used in the virtual screen. The binding sites were defined as all residues within 7 Å of the cocrystallized ligands.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Binding Assays for the Adenosine Receptors</h3><div class="NLM_p">A<sub>2A</sub>AR competition binding experiments were carried out in a multiscreen GF/C 96-well plate (Millipore, Madrid, Spain) pretreated with binding buffer (Tris-HCl 50 mM, EDTA 1 mM, MgCl<sub>2</sub> 10 mM, 2 U/mL adenosine deaminase, pH = 7.4). An amount of 5 μg of membranes from HeLa-A<sub>2A</sub> cell line was incubated with 3 nM [<sup>3</sup>H]ZM241385 (4-[2-[[7-amino-2-(furan-2-yl)-[1,2,4]triazolo[1,5-<i>a</i>][1,3,5]triazin-5-yl]amino]ethyl]-2-tritiophenol) (50 Ci/mmol, 1 mCi/ml, ARC-ITISA 0884) and the studied compound at 25 °C for 30 min and then filtered and washed four times with 250 μL of wash buffer (Tris-HCl 50 mM, EDTA 1 mM, MgCl<sub>2</sub> 10 mM, pH = 7.4), before measuring in a microplate β scintillation counter (Microbeta Trilux, PerkinElmer, Madrid, Spain). Nonspecific binding was determined in the presence of 50 μM NECA (Sigma E2387).</div><div class="NLM_p">A<sub>1</sub>AR competition binding experiments were carried out in a multiscreen GF/C 96-well plate (Millipore, Madrid, Spain) pretreated with binding buffer (Hepes 20 mM, NaCl 100 mM, MgCl<sub>2</sub> 10 mM, 2 U/mL adenosine deaminase, pH = 7.4). An amount of 5 μg of membranes from the Euroscreen hA<sub>1</sub> cell line was incubated with 1 nM [<sup>3</sup>H]DPCPX (8-cyclopentyl-1,3-bis(1,3-ditritiopropyl)-7<i>H</i>-purine-2,6-dione) (120 Ci/mmol, 1 mCi/mL, PerkinElmer NET974001MC) and the studied compound at 25 °C for 60 min and then filtered and washed four times with 250 μL of wash buffer (Hepes 20 mM, NaCl 100 mM, MgCl<sub>2</sub> 10 mM, pH = 7.4), before measuring in a microplate β scintillation counter (Microbeta Trilux, PerkinElmer, Madrid, Spain). Nonspecific binding was determined in the presence of 10 μM R-PIA (Sigma P4532).</div><div class="NLM_p">A<sub>2B</sub>AR competition binding experiments were carried out in a polypropilene 96-well plate. An amount of 20 μg of membranes from Euroscreen hA<sub>2B</sub> cell line was incubated with 25 nM [<sup>3</sup>H]DPCPX (164 Ci/mmol, 1 mCi/ml, PerkinElmer NET974001MC) and the studied compound at 25 °C for 30 min and then filtered and washed four times with 250 μL of wash buffer (Tris-HCl 50 mM, EDTA 1 mM, MgCl<sub>2</sub> 5 mM, pH = 6.5), before measuring in a microplate β scintillation counter (Microbeta Trilux, PerkinElmer, Madrid, Spain). Nonspecific binding was determined in the presence of 1000 μM NECA (Sigma E2387).</div><div class="NLM_p">A<sub>3</sub>AR competition binding experiments were carried out in a multiscreen GF/B 96-well plate (Millipore, Madrid, Spain) pretreated with binding buffer (Tris-HCl 50 mM, EDTA 1 mM, MgCl<sub>2</sub> 5 mM, 2 U/mL adenosine deaminase, pH = 7.4). An amount of 70 μg of membranes from HeLa-A<sub>3</sub> cell line was incubated with 10 nM [<sup>3</sup>H]NECA (27.6 Ci/mmol, 1 mCi/ml, PerkinElmer NET811250UC) and the studied compound at 25 °C for 180 min and then filtered and washed six times with 250 μL of wash buffer (Tris-HCl, 50 mM, pH = 7.4), before measuring in a microplate β scintillation counter (Microbeta Trilux, PerkinElmer, Madrid, Spain). Nonspecific binding was determined in the presence of 100 μM R-PIA (Sigma P4532).</div><div class="NLM_p last">Nonlinear fitting of the concentration–response curves was carried out by using the GraphPad Prism 4.0 software (San Diego, CA, USA) by applying a four-parameters logistic equation for deriving IC<sub>50</sub> values. <i>K</i><sub>i</sub> values were calculated using the Cheng–Prusoff equation. All <i>K</i><sub>i</sub> values are reported as the mean ± SEM.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Functional Assays for the A<sub>2A</sub>AR</h3><div class="NLM_p last">Functional experiments were carried out in a CHO cell line transfected with human A<sub>2A</sub> receptors by measuring coupling to the G<sub>s</sub> signaling pathway. Twenty-four hours before the assay, 10<sup>4</sup> cells/well were seeded in a 96-well culture plate (Falcon 353072). The cells were washed with wash buffer (Dulbecco’s modified Eagle’s medium (DMEM) nutrient mixture F-12 ham (Sigma D8062), 25 mM Hepes; pH = 7.4). Wash buffer was replaced by incubation buffer (DMEM nutrient mixture F-12 ham (Sigma D8062), 25 mM Hepes, 20 μM Rolipram; pH = 7.4). The tested compounds were added and incubated at 37 °C for 15 min. After incubation, NECA (Sigma E2387) was added in several concentrations and incubated at 37 °C for 15 min. After incubation, the amount of cAMP was determined using cAMP Biotrak enzyme immunoassay (EIA) system kit (GE Healthcare RPN225).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> MAO-B and MAO-A Assays</h3><div class="NLM_p last">The 24 compounds selected from the from molecular docking screens were initially tested at MAO-B as described previously (IC<sub>50</sub> values in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S4</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compounds <b>1</b> and <b>3</b> and analogs thereof were evaluated using a slightly modified protocol (IC<sub>50</sub> values in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S8</a>). The human recombinant MAO-A and MAO-B enzyme expressed in baculovirus infected BTI insect cells were purchased from Sigma-Aldrich (84 U/mg, catalog no. M7316 and 71 U/mg, catalog no. M 7441, respectively). A volume of 5 μL (2-fold concentration) of human MAO (final 1 U/mL for MAO-B and final 0.5 U/mL for MAO-A) was delivered to the reaction plate (or well) containing the base reaction buffer (50 mM Tris-HCL, pH = 7.5, 0.05% CHAPS, and 1% DMSO). Compounds in concentrations ranging from 10<sup>–9</sup> to 10<sup>–4</sup> in 100% DMSO were added to the enzyme mixture by acoustic technology (Echo 550; nanoliter range). Samples were preincubated for 30 min at room temperature. A mixture of 5 μL (2-fold concentration) of tyramine (10 μM) with buffer was added to reaction wells, except in the no substrate wells, to initiate the reaction, and samples were incubated for 60 min at room temperature. For the detection step, 10 μL (2-fold concentration) of a premixture of horseradish peroxidase (final 0.1 U/mL) and Amplex Red reagent (final 10 μM) was added to the reaction mixture and fluorescence measurements of hydrogen peroxide and consequently of resorufin were performed using EnVision fluorescence reader (excitation 535 nm and emission 590 nm) over 30 min. In order to control for assay interference, the experiments were carried out in the absence of enzyme but with H<sub>2</sub>O<sub>2</sub> added (mimicking assay product). The MAO-B assays for compounds <b>1a</b> and <b>3</b> were also performed in the presence of Triton X-100 (0.01%) to rule out inhibition due to colloidal aggregation. All data were analyzed using the GraphPad Prism 4.0 software (San Diego, CA, USA). All IC<sub>50</sub> values are reported as the mean ± SEM.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Cell Viability Measurement</h3><div class="NLM_p last">Reagents for cell culture were acquired through Life Technologies unless otherwise stated. Human SH-SY5Y cells were grown in DMEM supplemented with 10% fetal bovine serum (FBS), 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, 100 μg/mL penicillin and streptomycin. Cells were maintained at 37 °C with 95% humidified air and 5% CO<sub>2</sub>. Cells were grown to a confluency of 70–80% and then seeded into 96-well plates (Corning) at a density of 2.5 × 10<sup>4</sup> cells/well with phenol-red free DMEM, in 1% FBS, 200 μM <span class="smallcaps smallerCapital">l</span>-glutamine, 100 μg/mL penicillin–streptomycin and with 10 μM retinoic acid (Sigma) for 72 h to stimulate differentiation into dopaminergic neuronal-like cells. Cells were then pretreated with the compounds diluted in DMSO (0.1% v/v, Sigma) for 3 h before the addition of 20 μM 6-OHDA (Sigma) for 24 h. 6-OHDA was freshly prepared and diluted in 0.9% sodium chloride (Fresenius-Kabi) and 0.007% ascorbic acid (Sigma). In the cell viability measurements, resazurin (0.02 mg/mL final concentration, Sigma) was added to each well 24 h after 6-OHDA exposure and further incubated for 2 h. Afterward, fluorescence intensity was examined at an excitation of 540 nm and an emission of 590 nm (Spark, Tecan).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00204" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47845" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47845" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00204" class="ext-link">10.1021/acs.jmedchem.8b00204</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Tables S1–S10 and Figures S1–S5 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Predicted binding modes of dual-target inhibitors <b>1</b>–<b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Predicted binding modes of dual-target inhibitors <b>1</b>–<b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_004.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf">jm8b00204_si_001.pdf (3.61 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_002.csv">jm8b00204_si_002.csv (2.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_003.pdb">jm8b00204_si_003.pdb (187.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_004.pdb">jm8b00204_si_004.pdb (326.62 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00204" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jens Carlsson</span> - <span class="hlFld-Affiliation affiliation">Science
for Life Laboratory, Department of Cell and Molecular Biology, Uppsala University, BMC Box 596, SE-751 24 Uppsala, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4623-2977" title="Orcid link">http://orcid.org/0000-0003-4623-2977</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8fe5eae1fca1eceefde3fcfce0e1cfe6ece2a1fafaa1fcea"><span class="__cf_email__" data-cfemail="6309060d104d0002110f10100c0d230a000e4d16164d1006">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mariama Jaiteh</span> - <span class="hlFld-Affiliation affiliation">Science
for Life Laboratory, Department of Cell and Molecular Biology, Uppsala University, BMC Box 596, SE-751 24 Uppsala, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexey Zeifman</span> - <span class="hlFld-Affiliation affiliation">Science
for Life Laboratory, Department of Cell and Molecular Biology, Uppsala University, BMC Box 596, SE-751 24 Uppsala, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcus Saarinen</span> - <span class="hlFld-Affiliation affiliation">Center
of Molecular Medicine, Department of Physiology and Pharmacology, Karolinska Institute, SE-171 77 Stockholm, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Per Svenningsson</span> - <span class="hlFld-Affiliation affiliation">Center
of Molecular Medicine, Department of Physiology and Pharmacology, Karolinska Institute, SE-171 77 Stockholm, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jose Bréa</span> - <span class="hlFld-Affiliation affiliation">USEF
Screening Platform-BioFarma Research Group, Centre for Research in
Molecular Medicine and Chronic Diseases, University of Santiago de Compostela, 15706 Santiago de Compostela, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maria Isabel Loza</span> - <span class="hlFld-Affiliation affiliation">USEF
Screening Platform-BioFarma Research Group, Centre for Research in
Molecular Medicine and Chronic Diseases, University of Santiago de Compostela, 15706 Santiago de Compostela, Spain</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.J. and A.Z. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d47e1916-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81755" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81755" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This project has received funding from the European Research Council (ERC) under the European Union’s Horizon 2020 Research and Innovation Programme (Grant Agreement 715052). The work was also supported by grants from the Swedish Research Council (Grants 2013-5708 and 2017-4676) and the Science for Life Laboratory to J.C. Computational resources were provided by the Swedish National Infrastructure for Computing (SNIC). We thank OpenEye Scientific Software for the use of OEChem and OMEGA at no cost.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">adenosine receptor</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoamine oxidase</p></td></tr><tr><td class="NLM_term">logAUC</td><td class="NLM_def"><p class="first last">logarithm of area under the curve</p></td></tr><tr><td class="NLM_term">CSC</td><td class="NLM_def"><p class="first last">8-(3-chlorostyryl)caffeine</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>c</sub></td><td class="NLM_def"><p class="first last">Tanimoto similarity coefficient</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">6-OHDA</td><td class="NLM_def"><p class="first last">6-hydroxydopamine</p></td></tr><tr><td class="NLM_term">ECFP4</td><td class="NLM_def"><p class="first last">extended chemical fingerprints up to four atoms</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle medium</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37340" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37340" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rang, H. P.</span></span> <span> </span><span class="NLM_article-title">The Receptor Concept: Pharmacology’s Big Idea</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">S9</span>– <span class="NLM_lpage">S16</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0706457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fsj.bjp.0706457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=16402126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XisVClsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2006&pages=S9-S16&author=H.+P.+Rang&title=The+Receptor+Concept%3A+Pharmacology%E2%80%99s+Big+Idea&doi=10.1038%2Fsj.bjp.0706457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The receptor concept: pharmacology's big idea</span></div><div class="casAuthors">Rang, H. P.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S9-S16</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. signaling is the main mechanism by which biol. function is controlled at all levels, from the single cell to the whole organism.  Chem. recognition is the function of receptors, which, in addn. to recognizing endogenous chem. signals, are also the target of many important exptl. and therapeutic drugs.  Receptors, therefore, lie at the heart of pharmacol.  This article describes the way in which the receptor concept originated early in the 20th century, and evolved through a highly innovative stage of quant. theory based on chem. kinetics, to the point where receptors were first isolated and later cloned, until we now have a virtually complete catalog of all the receptors present in the genome.  Studies on signal transduction are revealing great complexity in the events linking ligand binding to the physiol. or therapeutic response.  Though some simple quant. rules of 'receptor theory' are still useful, the current emphasis is on unraveling the pathways that link receptors to responses, and it will be some time before we know enough about them to embark on the next phase of 'receptor theory'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1enUcNFx4g7Vg90H21EOLACvtfcHk0libm0f5af5OcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisVClsA%253D%253D&md5=2e28f786f60570b75f4bd0d937a151df</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0706457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0706457%26sid%3Dliteratum%253Aachs%26aulast%3DRang%26aufirst%3DH.%2BP.%26atitle%3DThe%2520Receptor%2520Concept%253A%2520Pharmacology%25E2%2580%2599s%2520Big%2520Idea%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2006%26volume%3D147%26spage%3DS9%26epage%3DS16%26doi%3D10.1038%2Fsj.bjp.0706457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheffler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span> <span> </span><span class="NLM_article-title">Magic Shotguns versus Magic Bullets: Selectively Non-Selective Drugs for Mood Disorders and Schizophrenia</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1038/nrd1346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd1346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=15060530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1Gkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=353-359&author=B.+L.+Rothauthor=D.+J.+Shefflerauthor=W.+K.+Kroeze&title=Magic+Shotguns+versus+Magic+Bullets%3A+Selectively+Non-Selective+Drugs+for+Mood+Disorders+and+Schizophrenia&doi=10.1038%2Fnrd1346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Magic shotguns versus magic bullets: selectively non-selective drugs for mood disorders and schizophrenia</span></div><div class="casAuthors">Roth, Bryan L.; Sheffler, Douglas J.; Kroeze, Wesley K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">353-359</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Most common central nervous system disorders - such as depression, bipolar disorder, and schizophrenia - seem to be polygenic in origin, and the most effective medications have exceedingly complex pharmacologies.  Attempts to develop more effective treatments for diseases such as schizophrenia and depression by discovering drugs selective for single mol. targets (i.e., 'magic bullets') have, not surprisingly, been largely unsuccessful.  Here the authors propose that designing selectively non-selective drugs (i.e., 'magic shotguns') that interact with several mol. targets will lead to new and more effective medications for a variety of central nervous system disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnSsuUXzmbirVg90H21EOLACvtfcHk0libm0f5af5OcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1Gkt78%253D&md5=77196583acccca65457fec7a67218463</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd1346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1346%26sid%3Dliteratum%253Aachs%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSheffler%26aufirst%3DD.%2BJ.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26atitle%3DMagic%2520Shotguns%2520versus%2520Magic%2520Bullets%253A%2520Selectively%2520Non-Selective%2520Drugs%2520for%2520Mood%2520Disorders%2520and%2520Schizophrenia%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D353%26epage%3D359%26doi%3D10.1038%2Fnrd1346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lip8MJUqimR2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mìnarini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumiatti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchiorre, C.</span></span> <span> </span><span class="NLM_article-title">Multi-Target-Directed Ligands to Combat Neurodegenerative Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1021/jm7009364</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7009364" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=347-372&author=A.+Cavalliauthor=M.+L.+Bolognesiauthor=A.+M%C3%ACnariniauthor=M.+Rosiniauthor=V.+Tumiattiauthor=M.+Recanatiniauthor=C.+Melchiorre&title=Multi-Target-Directed+Ligands+to+Combat+Neurodegenerative+Diseases&doi=10.1021%2Fjm7009364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target-Directed Ligands to Combat Neurodegenerative Diseases</span></div><div class="casAuthors">Cavalli, Andrea; Bolognesi, Maria Laura; Minarini, Anna; Rosini, Michela; Tumiatti, Vincenzo; Recanatini, Maurizio; Melchiorre, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-372</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors review the investigations reported in the past decade that illustrate the ability of medicinal chemists to design multi-target-directed ligands (MTDLs) to confront neurodegenerative diseases.  The authors briefly illustrate the multifactorial nature of these diseases before discussing the possible development of MTDLs in light of the known mol. bases of the main neurodegenerative pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYY0I0UHEir7Vg90H21EOLACvtfcHk0lip8MJUqimR2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D&md5=c682235151489316d043cf97f78e56f3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm7009364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7009364%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DM%25C3%25ACnarini%26aufirst%3DA.%26aulast%3DRosini%26aufirst%3DM.%26aulast%3DTumiatti%26aufirst%3DV.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DMulti-Target-Directed%2520Ligands%2520to%2520Combat%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D347%26epage%3D372%26doi%3D10.1021%2Fjm7009364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhillon, S.</span></span> <span> </span><span class="NLM_article-title">Safinamide: A Review in Parkinson’s Disease</span>. <i>CNS Drugs</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">176</span>, <span class="refDoi"> DOI: 10.1007/s40263-017-0408-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1007%2Fs40263-017-0408-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=28110399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=169-176&author=H.+A.+Blairauthor=S.+Dhillon&title=Safinamide%3A+A+Review+in+Parkinson%E2%80%99s+Disease&doi=10.1007%2Fs40263-017-0408-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Safinamide: A Review in Parkinson's Disease</span></div><div class="casAuthors">Blair, Hannah A.; Dhillon, Sohita</div><div class="citationInfo"><span class="NLM_cas:title">CNS Drugs</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-176</span>CODEN:
                <span class="NLM_cas:coden">CNDREF</span>;
        ISSN:<span class="NLM_cas:issn">1172-7047</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Safinamide (Xadago) is an orally active, selective, reversible monoamine oxidase-B inhibitor with both dopaminergic and non-dopaminergic (glutamatergic) properties.  In the EU, safinamide is approved for the treatment of mid- to late-stage fluctuating Parkinson's disease (PD) as add-on therapy to a stable dose of levodopa alone or in combination with other PD medications.  Safinamide 50-100 mg/day administered as a fixed or flexible dose significantly increased daily 'on' time without dyskinesia (primary endpoint) in patients with mid- to late-stage PD with motor fluctuations in 24-wk, placebo-controlled clin. trials.  Other outcomes, including motor function, overall clin. status and health-related quality of life, were also generally improved with safinamide.  Furthermore, in an 18-mo extension of one study, although dyskinesia (primary endpoint) was not significantly improved with safinamide relative to placebo, treatment benefits in other outcomes were generally sustained over 24 mo of treatment.  Safinamide was generally well tolerated in clin. trials; dyskinesia was the most common adverse event.  Although further studies are needed, including comparative and long-term studies, current evidence indicates that safinamide extends the treatment options available for use as add-on therapy to levodopa and other PD medications in patients with mid- to late-stage PD experiencing motor fluctuations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl8UyUX-7XrVg90H21EOLACvtfcHk0lip8MJUqimR2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeisrc%253D&md5=7adc3525326e5274c2f9fac22d39cb28</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs40263-017-0408-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40263-017-0408-1%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DH.%2BA.%26aulast%3DDhillon%26aufirst%3DS.%26atitle%3DSafinamide%253A%2520A%2520Review%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DCNS%2520Drugs%26date%3D2017%26volume%3D31%26spage%3D169%26epage%3D176%26doi%3D10.1007%2Fs40263-017-0408-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein
Herenbrink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span> <span> </span><span class="NLM_article-title">Proof of Concept Study for Designed Multiple Ligands Targeting the Dopamine D<sub>2</sub>, Serotonin 5-HT<sub>2A</sub>, and Muscarinic M<sub>1</sub> Acetylcholine Receptors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1555</span>, <span class="refDoi"> DOI: 10.1021/jm5013243</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5013243" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVygug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1550-1555&author=M.+Szaboauthor=H.+D.+Limauthor=C.+Klein%0AHerenbrinkauthor=A.+Christopoulosauthor=J.+R.+Laneauthor=B.+Capuano&title=Proof+of+Concept+Study+for+Designed+Multiple+Ligands+Targeting+the+Dopamine+D2%2C+Serotonin+5-HT2A%2C+and+Muscarinic+M1+Acetylcholine+Receptors&doi=10.1021%2Fjm5013243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Proof of Concept Study for Designed Multiple Ligands Targeting the Dopamine D2, Serotonin 5-HT2A, and Muscarinic M1 Acetylcholine Receptors</span></div><div class="casAuthors">Szabo, Monika; Lim, Herman D.; Klein Herenbrink, Carmen; Christopoulos, Arthur; Lane, J. Robert; Capuano, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1550-1555</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein the authors describe the hybridization of a benzoxazinone M1 scaffold with D2 privileged structures derived from putative and clin. relevant antipsychotics to develop designed multiple ligands.  The M1 mAChR is an attractive target for the cognitive deficits in key CNS disorders.  Moreover, activity at D2 and 5-HT2A receptors has proven useful for antipsychotic efficacy.  The authors identified 9 which retained functional activity at the target M1 mAChR and D2R and demonstrated high affinity for the 5-HT2AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJqxxyfgdbsbVg90H21EOLACvtfcHk0liAVVtRS71OlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVygug%253D%253D&md5=9aebec84dcc8ce782a4ef4d53cf773e0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm5013243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5013243%26sid%3Dliteratum%253Aachs%26aulast%3DSzabo%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DH.%2BD.%26aulast%3DKlein%2BHerenbrink%26aufirst%3DC.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DCapuano%26aufirst%3DB.%26atitle%3DProof%2520of%2520Concept%2520Study%2520for%2520Designed%2520Multiple%2520Ligands%2520Targeting%2520the%2520Dopamine%2520D2%252C%2520Serotonin%25205-HT2A%252C%2520and%2520Muscarinic%2520M1%2520Acetylcholine%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1550%26epage%3D1555%26doi%3D10.1021%2Fjm5013243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated Design of Ligands to Polypharmacological Profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+Design+of+Ligands+to+Polypharmacological+Profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0liAVVtRS71OlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520Design%2520of%2520Ligands%2520to%2520Polypharmacological%2520Profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the Cancer Kinome through Polypharmacology</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1038/nrc2787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrc2787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20094047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=130-137&author=Z.+A.+Knightauthor=H.+Linauthor=K.+M.+Shokat&title=Targeting+the+Cancer+Kinome+through+Polypharmacology&doi=10.1038%2Fnrc2787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer kinome through polypharmacology</span></div><div class="casAuthors">Knight, Zachary A.; Lin, Henry; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-137</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Kinase inhibitors are the largest class of new cancer drugs.  However, it is already apparent that most tumors can escape from the inhibition of any single kinase.  If it is necessary to inhibit multiple kinases, how do we choose which ones.  In this Opinion article, we discuss some of the strategies that are currently being used to identify new therapeutic combinations of kinase targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkDv2FeeH58rVg90H21EOLACvtfcHk0liUgrrmtz-e4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D&md5=9aaa314e54b43889e147a9f11ac77072</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc2787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2787%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DTargeting%2520the%2520Cancer%2520Kinome%2520through%2520Polypharmacology%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D130%26epage%3D137%26doi%3D10.1038%2Fnrc2787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span> <span> </span><span class="NLM_article-title">Computational Studies to Predict or Explain G Protein Coupled Receptor Polypharmacology</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.tips.2014.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=25458540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCktL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=658-663&author=K.+A.+Jacobsonauthor=S.+Costanziauthor=S.+Paoletta&title=Computational+Studies+to+Predict+or+Explain+G+Protein+Coupled+Receptor+Polypharmacology&doi=10.1016%2Fj.tips.2014.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Computational studies to predict or explain G protein coupled receptor polypharmacology</span></div><div class="casAuthors">Jacobson, Kenneth A.; Costanzi, Stefano; Paoletta, Silvia</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">658-663</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Since G protein-coupled receptors (GPCRs) belong to a very large superfamily of evolutionarily related receptors (>800 members in humans), and due to the rapid progress on their structural biol., they are ideal candidates for polypharmacol. studies.  Broad screening and bioinformatics/chemoinformatics have been applied to understanding off-target effects of GPCR ligands.  It is now feasible to approach the question of GPCR polypharmacol. using mol. modeling and the available X-ray GPCR structures.  As an example, large and sterically constrained adenosine derivs. (potent adenosine receptor ligands with low conformational freedom and multiple extended substituents) were screened for binding at diverse receptors.  Unanticipated off-target interactions, including at biogenic amine receptors, were then modeled using a structure-based approach to provide a consistent understanding of recognition.  A conserved Asp in TM3 changed its role from counterion for biogenic amines to characteristic H-bonding to adenosine.  The same systematic approach could potentially be applied to many GPCRs or other receptors using other sets of congeneric ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfRo5bhR-c57Vg90H21EOLACvtfcHk0liUgrrmtz-e4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCktL3E&md5=e3490f7573c45c59760cb5e598b3cd9b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCostanzi%26aufirst%3DS.%26aulast%3DPaoletta%26aufirst%3DS.%26atitle%3DComputational%2520Studies%2520to%2520Predict%2520or%2520Explain%2520G%2520Protein%2520Coupled%2520Receptor%2520Polypharmacology%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D658%26epage%3D663%26doi%3D10.1016%2Fj.tips.2014.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Can We Rationally Design Promiscuous Drugs?</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.sbi.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=16442279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=127-136&author=A.+L.+Hopkinsauthor=J.+S.+Masonauthor=J.+P.+Overington&title=Can+We+Rationally+Design+Promiscuous+Drugs%3F&doi=10.1016%2Fj.sbi.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Can we rationally design promiscuous drugs?</span></div><div class="casAuthors">Hopkins, Andrew L.; Mason, Jonathan S.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-136</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design is now used widely in modern medicinal chem.  The application of structural biol. to medicinal chem. has heralded the 'rational drug design' vision of discovering exquisitely selective ligands.  However, recent advances in post-genomic biol. are indicating that polypharmacol. may be a necessary trait for the efficacy of many drugs, therefore questioning the 'one drug, one target' assumption of current rational drug design.  By combining advances in chemoinformatics and structural biol., it might be possible to rationally design the next generation of promiscuous drugs with polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQWdh60x9AHrVg90H21EOLACvtfcHk0liUgrrmtz-e4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D&md5=8f9e95a41976b8e3ac790f2fff2c406f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DCan%2520We%2520Rationally%2520Design%2520Promiscuous%2520Drugs%253F%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D127%26epage%3D136%26doi%3D10.1016%2Fj.sbi.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lees, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revesz, T.</span></span> <span> </span><span class="NLM_article-title">Parkinson’s Disease</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">2055</span>– <span class="NLM_lpage">2066</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(09)60492-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2FS0140-6736%2809%2960492-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=19524782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ans7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2009&pages=2055-2066&author=A.+J.+Leesauthor=J.+Hardyauthor=T.+Revesz&title=Parkinson%E2%80%99s+Disease&doi=10.1016%2FS0140-6736%2809%2960492-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Parkinson's disease</span></div><div class="casAuthors">Lees, Andrew J.; Hardy, John; Revesz, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">9680</span>),
    <span class="NLM_cas:pages">2055-2066</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: Parkinson's disease is a common progressive bradykinetic disorder that can be accurately diagnosed.  It is characterised by the presence of severe pars-compacta nigral-cell loss, and accumulation of aggregated α-synuclein in specific brain stem, spinal cord, and cortical regions.  The main known risk factor is age.  Susceptibility genes including α-synuclein, leucine rich repeat kinase 2 (LRRK-2), and glucocerebrosidase (GBA) have shown that genetic predisposition is another important causal factor.  Dopamine replacement therapy considerably reduces motor handicap, and effective treatment of assocd. depression, pain, constipation, and nocturnal difficulties can improve quality of life.  Embryonic stem cells and gene therapy are promising research therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjOcLAU8_Xq7Vg90H21EOLACvtfcHk0lgojiOtLmstnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ans7w%253D&md5=ae632b0ac91f4e5ee017bf2bf21f5926</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2809%2960492-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252809%252960492-X%26sid%3Dliteratum%253Aachs%26aulast%3DLees%26aufirst%3DA.%2BJ.%26aulast%3DHardy%26aufirst%3DJ.%26aulast%3DRevesz%26aufirst%3DT.%26atitle%3DParkinson%25E2%2580%2599s%2520Disease%26jtitle%3DLancet%26date%3D2009%26volume%3D373%26spage%3D2055%26epage%3D2066%26doi%3D10.1016%2FS0140-6736%2809%2960492-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeso, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rascol, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tison, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezard, E.</span></span> <span> </span><span class="NLM_article-title">Priorities in Parkinson’s Disease Research</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">393</span>, <span class="refDoi"> DOI: 10.1038/nrd3430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd3430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21532567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVGltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=377-393&author=W.+G.+Meissnerauthor=M.+Frasierauthor=T.+Gasserauthor=C.+G.+Goetzauthor=A.+Lozanoauthor=P.+Picciniauthor=J.+A.+Obesoauthor=O.+Rascolauthor=A.+Schapiraauthor=V.+Voonauthor=D.+M.+Weinerauthor=F.+Tisonauthor=E.+Bezard&title=Priorities+in+Parkinson%E2%80%99s+Disease+Research&doi=10.1038%2Fnrd3430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Priorities in Parkinson's disease research</span></div><div class="casAuthors">Meissner, Wassilios G.; Frasier, Mark; Gasser, Thomas; Goetz, Christopher G.; Lozano, Andres; Piccini, Paola; Obeso, Jose A.; Rascol, Olivier; Schapira, Anthony; Voon, Valerie; Weiner, David M.; Tison, Francois; Bezard, Erwan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">377-393</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The loss of dopaminergic neurons in the substantia nigra pars compacta leads to the characteristic motor symptoms of Parkinson's disease: bradykinesia, rigidity and resting tremors.  Although these symptoms can be improved using currently available dopamine replacement strategies, there is still a need to improve current strategies of treating these symptoms, together with a need to alleviate non-motor symptoms of the disease.  Moreover, treatments that provide neuroprotection and/or disease-modifying effects remain an urgent unmet clin. need.  This Review describes the most promising biol. targets and therapeutic agents that are currently being assessed to address these treatment goals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFgF5KqF7ZzrVg90H21EOLACvtfcHk0lgojiOtLmstnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVGltr4%253D&md5=d792624cf69d3a56537eb8aa00c5b963</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3430%26sid%3Dliteratum%253Aachs%26aulast%3DMeissner%26aufirst%3DW.%2BG.%26aulast%3DFrasier%26aufirst%3DM.%26aulast%3DGasser%26aufirst%3DT.%26aulast%3DGoetz%26aufirst%3DC.%2BG.%26aulast%3DLozano%26aufirst%3DA.%26aulast%3DPiccini%26aufirst%3DP.%26aulast%3DObeso%26aufirst%3DJ.%2BA.%26aulast%3DRascol%26aufirst%3DO.%26aulast%3DSchapira%26aufirst%3DA.%26aulast%3DVoon%26aufirst%3DV.%26aulast%3DWeiner%26aufirst%3DD.%2BM.%26aulast%3DTison%26aufirst%3DF.%26aulast%3DBezard%26aufirst%3DE.%26atitle%3DPriorities%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%2520Research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D377%26epage%3D393%26doi%3D10.1038%2Fnrd3430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, A. H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotchie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collingridge, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hengerer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Novère, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeso, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarzschild, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spampinato, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidai, G.</span></span> <span> </span><span class="NLM_article-title">Novel Pharmacological Targets for the Treatment of Parkinson’s Disease</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1038/nrd2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=17016425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVajsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=845-854&author=A.+H.+V.+Schapiraauthor=E.+Bezardauthor=J.+Brotchieauthor=F.+Calonauthor=G.+L.+Collingridgeauthor=B.+Fergerauthor=B.+Hengererauthor=E.+Hirschauthor=P.+Jennerauthor=N.+Le+Nov%C3%A8reauthor=J.+A.+Obesoauthor=M.+A.+Schwarzschildauthor=U.+Spampinatoauthor=G.+Davidai&title=Novel+Pharmacological+Targets+for+the+Treatment+of+Parkinson%E2%80%99s+Disease&doi=10.1038%2Fnrd2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Novel pharmacological targets for the treatment of Parkinson's disease</span></div><div class="casAuthors">Schapira, Anthony H. V.; Bezard, Erwan; Brotchie, Jonathan; Calon, Frederic; Collingridge, Graham L.; Ferger, Borris; Hengerer, Bastian; Hirsch, Etienne; Jenner, Peter; Le Novere, Nicolas; Obeso, Jose A.; Schwarzschild, Michael A.; Spampinato, Umberto; Davidai, Giora</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dopamine deficiency, caused by the degeneration of nigrostriatal dopaminergic neurons, is the cause of the major clin. motor symptoms of Parkinson's disease.  These symptoms can be treated successfully with a range of drugs that include levodopa, inhibitors of the enzymic breakdown of levodopa and dopamine agonists delivered by oral, s.c., transcutaneous, i.v. or intra-duodenal routes.  However, Parkinson's disease involves degeneration of non-dopaminergic neurons and the treatment of the resulting predominantly non-motor features remains a challenge.  This review describes the important recent advances that underlie the development of novel dopaminergic and non-dopaminergic drugs for Parkinson's disease, and also for the motor complications that arise from the use of existing therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR-UyAD07eK7Vg90H21EOLACvtfcHk0lgojiOtLmstnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVajsbnE&md5=be3cf2f5a3523bf11b274c501677a9f6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2087%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DA.%2BH.%2BV.%26aulast%3DBezard%26aufirst%3DE.%26aulast%3DBrotchie%26aufirst%3DJ.%26aulast%3DCalon%26aufirst%3DF.%26aulast%3DCollingridge%26aufirst%3DG.%2BL.%26aulast%3DFerger%26aufirst%3DB.%26aulast%3DHengerer%26aufirst%3DB.%26aulast%3DHirsch%26aufirst%3DE.%26aulast%3DJenner%26aufirst%3DP.%26aulast%3DLe%2BNov%25C3%25A8re%26aufirst%3DN.%26aulast%3DObeso%26aufirst%3DJ.%2BA.%26aulast%3DSchwarzschild%26aufirst%3DM.%2BA.%26aulast%3DSpampinato%26aufirst%3DU.%26aulast%3DDavidai%26aufirst%3DG.%26atitle%3DNovel%2520Pharmacological%2520Targets%2520for%2520the%2520Treatment%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D845%26epage%3D854%26doi%3D10.1038%2Fnrd2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzer, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Schyf, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castagnoli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonsalla, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castagnoli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarzschild, M. A.</span></span> <span> </span><span class="NLM_article-title">8-(3-Chlorostyryl) Caffeine May Attenuate MPTP Neurotoxicity through Dual Actions of Monoamine Oxidase Inhibition and A<sub>2A</sub> Receptor Antagonism</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">36040</span>– <span class="NLM_lpage">36044</span>, <span class="refDoi"> DOI: 10.1074/jbc.M206830200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1074%2Fjbc.M206830200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=12130655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVCjs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=36040-36044&author=J.+F.+Chenauthor=S.+Steynauthor=R.+Staalauthor=J.+P.+Petzerauthor=K.+Xuauthor=C.+J.+Van+Der+Schyfauthor=K.+Castagnoliauthor=P.+K.+Sonsallaauthor=N.+Castagnoliauthor=M.+A.+Schwarzschild&title=8-%283-Chlorostyryl%29+Caffeine+May+Attenuate+MPTP+Neurotoxicity+through+Dual+Actions+of+Monoamine+Oxidase+Inhibition+and+A2A+Receptor+Antagonism&doi=10.1074%2Fjbc.M206830200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">8-(3-Chlorostyryl)caffeine May Attenuate MPTP Neurotoxicity through Dual Actions of Monoamine Oxidase Inhibition and A2A Receptor Antagonism</span></div><div class="casAuthors">Chen, Jiang-Fan; Steyn, Salome; Staal, Roland; Petzer, Jacobus P.; Xu, Kui; Van der Schyf, Cornelis J.; Castagnoli, Kay; Sonsalla, Patricia K.; Castagnoli, Neal, Jr.; Schwarzschild, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">36040-36044</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Caffeine and more specific antagonists of the adenosine A2A receptor recently have been found to be neuroprotective in the MPTP (1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine) model of Parkinson's disease.  Here we show that 8-(3-chlorostyryl)caffeine (CSC), a specific A2A antagonist closely related to caffeine, also attenuates MPTP-induced neurotoxicity.  Because the neurotoxicity of MPTP relies on its oxidative metab. to the mitochondrial toxin MPP+, we investigated the actions of CSC on striatal MPTP metab. in vivo.  CSC elevated striatal levels of MPTP but lowered levels of the oxidative intermediate MPDP+ and of MPP+, suggesting that CSC blocks the conversion of MPTP to MPDP+ in vivo.  In assessing the direct effects of CSC and A2A receptors on monoamine oxidase (MAO) activity, we found that CSC potently and specifically inhibited mouse brain mitochondrial MAO-B activity in vitro with a Ki value of 100 nm, whereas caffeine and another relatively specific A2A antagonist produced little or no inhibition.  The A2A receptor independence of MAO-B inhibition by CSC was further supported by the similarity of brain MAO activities derived from A2A receptor knockout and wild-type mice and was confirmed by demonstrating potent inhibition of A2A receptor knockout-derived MAO-B by CSC.  Together, these data indicate that CSC possesses dual actions of MAO-B inhibition and A2A receptor antagonism, a unique combination suggesting a new class of compds. with the potential for enhanced neuroprotective properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL8ESRi-__yrVg90H21EOLACvtfcHk0liMioJeMYEl1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVCjs7Y%253D&md5=27154cad8b81538a593504f7ca2f42ec</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206830200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206830200%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%2BF.%26aulast%3DSteyn%26aufirst%3DS.%26aulast%3DStaal%26aufirst%3DR.%26aulast%3DPetzer%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DVan%2BDer%2BSchyf%26aufirst%3DC.%2BJ.%26aulast%3DCastagnoli%26aufirst%3DK.%26aulast%3DSonsalla%26aufirst%3DP.%2BK.%26aulast%3DCastagnoli%26aufirst%3DN.%26aulast%3DSchwarzschild%26aufirst%3DM.%2BA.%26atitle%3D8-%25283-Chlorostyryl%2529%2520Caffeine%2520May%2520Attenuate%2520MPTP%2520Neurotoxicity%2520through%2520Dual%2520Actions%2520of%2520Monoamine%2520Oxidase%2520Inhibition%2520and%2520A2A%2520Receptor%2520Antagonism%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D36040%26epage%3D36044%26doi%3D10.1074%2Fjbc.M206830200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hockemeyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbiel, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Multigram-Scale Syntheses, Stability, and Photoreactions of A<sub>2A</sub> Adenosine Receptor Antagonists with 8-Styrylxanthine Structure: Potential Drugs for Parkinson’s Disease</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">3308</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1021/jo0358574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo0358574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFarsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2004&pages=3308-3318&author=J.+Hockemeyerauthor=J.+C.+Burbielauthor=C.+E.+M%C3%BCller&title=Multigram-Scale+Syntheses%2C+Stability%2C+and+Photoreactions+of+A2A+Adenosine+Receptor+Antagonists+with+8-Styrylxanthine+Structure%3A+Potential+Drugs+for+Parkinson%E2%80%99s+Disease&doi=10.1021%2Fjo0358574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Multigram-Scale Syntheses, Stability, and Photoreactions of A2A Adenosine Receptor Antagonists with 8-Styrylxanthine Structure: Potential Drugs for Parkinson's Disease</span></div><div class="casAuthors">Hockemeyer, Joerg; Burbiel, Joachim C.; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3308-3318</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The improved multigram-scale syntheses of the important 8-styrylxanthine A2A adenosine receptor antagonist MSX-2 [I, R = CH2C≡CH, R1 = (CH2)3OH, R2 = H], its water-sol. prodrug MXS-3 [I, R = CH2C≡CH, R1 = (CH2)3OP(O)(OH)2, R2 = H], and KW-6002 [I, R = R1 = Et, R2 = OMe] are described.  N-Alkylation reactions at different positions of the uracil ring were optimized.  Two different methods for xanthine formation from 6-amino-5-cinnamoylaminouracil precursors were investigated: (a) the elimination of water by alk. catalysis and (b) hexamethyldisilazane as a condensing agent; the latter was found to be superior.  The photosensitivity of the 8-styrylxanthines was studied.  The (E)-configurated stryrylxanthine MSX-2 isomerized in dild. soln., and the resulting (Z)-isomer was isolated and characterized.  Furthermore, solid 8-styrylxanthines can dimerize upon exposure to daylight or irradn. with UV light.  The resulting cyclobutane derivs. with head-to-tail (syn) configuration exhibited a considerably lower A2A adenosine receptor affinity than their parent compds.  The dimerization product of MSX-2 was a moderately potent nonselective A1 and A2A antagonist (Ki(A1) = 273 nM, Ki (A2A) = 175 nM) while the dimer of KW-6002 was inactive at A1 and only weakly active at A2A adenosine receptors (Ki = 1.57 μM).  The light sensitivity of 8-styrylxanthine derivs., not only in soln., but also in the solid state, has to be considered when using those compds. as pharmacol. tools or drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH59mwpE6WRLVg90H21EOLACvtfcHk0liMioJeMYEl1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFarsro%253D&md5=b76c488fd1df3d5c31b3911bf1b27d6b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjo0358574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo0358574%26sid%3Dliteratum%253Aachs%26aulast%3DHockemeyer%26aufirst%3DJ.%26aulast%3DBurbiel%26aufirst%3DJ.%2BC.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DMultigram-Scale%2520Syntheses%252C%2520Stability%252C%2520and%2520Photoreactions%2520of%2520A2A%2520Adenosine%2520Receptor%2520Antagonists%2520with%25208-Styrylxanthine%2520Structure%253A%2520Potential%2520Drugs%2520for%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2004%26volume%3D69%26spage%3D3308%26epage%3D3318%26doi%3D10.1021%2Fjo0358574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qurishi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diekmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padgett, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, J. W.</span></span> <span> </span><span class="NLM_article-title">Imidazo[2,1-I ]Purin-5-Ones and Related Tricyclic Water-Soluble Purine Derivatives: Potent A<sub>2A</sub>- and A<sub>3</sub>-Adenosine Receptor Antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">3440</span>– <span class="NLM_lpage">3450</span>, <span class="refDoi"> DOI: 10.1021/jm011093d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm011093d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD38XkvVGhsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3440-3450&author=C.+E.+M%C3%BCllerauthor=M.+Thorandauthor=R.+Qurishiauthor=M.+Diekmannauthor=K.+A.+Jacobsonauthor=W.+L.+Padgettauthor=J.+W.+Daly&title=Imidazo%5B2%2C1-I+%5DPurin-5-Ones+and+Related+Tricyclic+Water-Soluble+Purine+Derivatives%3A+Potent+A2A-+and+A3-Adenosine+Receptor+Antagonists&doi=10.1021%2Fjm011093d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Imidazo[2,1-i]purin-5-ones and Related Tricyclic Water-Soluble Purine Derivatives: Potent A2A- and A3-Adenosine Receptor Antagonists</span></div><div class="casAuthors">Mueller, Christa E.; Thorand, Mark; Qurishi, Ramatullah; Diekmann, Martina; Jacobson, Kenneth A.; Padgett, William L.; Daly, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3440-3450</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of tricyclic imidazo[2,1-i]purinones I [R1, R4 = H, Me; R2 = H, Ph, (E)-PhCH:CH; R3 = H, Me, PhCH2; R5 = H, Me, Et] and ring-enlarged analogs, e.g. II, derived from xanthine derivs. were prepd. as adenosine receptor (AR) antagonists.  In comparison with xanthines, these tricyclic compds. exhibited increased water soly. due to a basic nitrogen atom, which can be protonated under physiol. conditions.  A new capillary electrophoresis method was developed for the detn. of the enantiomeric purity of selected chiral products using native and modified β-cyclodextrins as chiral discriminators.  The compds. were investigated in radioligand binding assays at rat brain A1 and A2A ARs.  Selected I were addnl. investigated in radioligand binding assays at human recombinant A3 ARs and in functional studies (adenylate cyclase assays) at A1 ARs of rat fat cell membranes, A2A ARs of rat PC 12 cell membranes, and mouse A2B ARs of NIH 3T3 cell membranes, and showed the structure-activity relationships similar to those of the corresponding xanthine derivs.  The 2-styrylimidazopurinones I [R1 = H, Me; R2 = (E)-PhCH:CH; R3 = Me; R4 = H, R5 = Et] were less potent at A2A ARs as compared to 8-styrylxanthine derivs.  The most potent compd. at A2A ARs was (S)-I [R1 = R3 = Me, R2 = (E)-PhCH:CH, R4 = H, R5 = Et; (III)] exhibiting a Ki value of 424 nM at rat A2A ARs.  III was also highly selective for A2A receptors vs A1 and A3 ARs; however, the selectivity vs A2B ARs was low.  Among the 1-unsubstituted (2-phenyl)imidazopurinones, the most potent A3 antagonist was (R)-I (R1 = R4 = H, R2 = Ph, R3 = Me, R5 = Et) exhibiting a Ki value of 2.3 nM and high selectivity for A3 receptors vs all other AR subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqea1ltPz9137Vg90H21EOLACvtfcHk0liMioJeMYEl1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XkvVGhsrs%253D&md5=893182fdc78beec54214cf712536a134</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm011093d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm011093d%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DThorand%26aufirst%3DM.%26aulast%3DQurishi%26aufirst%3DR.%26aulast%3DDiekmann%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DPadgett%26aufirst%3DW.%2BL.%26aulast%3DDaly%26aufirst%3DJ.%2BW.%26atitle%3DImidazo%255B2%252C1-I%2520%255DPurin-5-Ones%2520and%2520Related%2520Tricyclic%2520Water-Soluble%2520Purine%2520Derivatives%253A%2520Potent%2520A2A-%2520and%2520A3-Adenosine%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3440%26epage%3D3450%26doi%3D10.1021%2Fjm011093d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armentero, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferré, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanciego, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, R.</span></span> <span> </span><span class="NLM_article-title">Past, Present and Future of A<sub>2A</sub> Adenosine Receptor Antagonists in the Therapy of Parkinson’s Disease</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">280</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.pharmthera.2011.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21810444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtl2ntrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2011&pages=280-299&author=M.+T.+Armenteroauthor=A.+Pinnaauthor=S.+Ferr%C3%A9author=J.+L.+Lanciegoauthor=C.+E.+M%C3%BCllerauthor=R.+Franco&title=Past%2C+Present+and+Future+of+A2A+Adenosine+Receptor+Antagonists+in+the+Therapy+of+Parkinson%E2%80%99s+Disease&doi=10.1016%2Fj.pharmthera.2011.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Past, present and future of A2A adenosine receptor antagonists in the therapy of Parkinson's disease</span></div><div class="casAuthors">Armentero, Marie Therese; Pinna, Annalisa; Ferre, Sergi; Lanciego, Jose Luis; Mueller, Christa E.; Franco, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">280-299</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Several selective antagonists for adenosine A2A receptors (A2AR) are currently under evaluation in clin. trials (phases I to III) to treat Parkinson's disease, and they will probably soon reach the market.  The usefulness of these antagonists has been deduced from studies demonstrating functional interactions between dopamine D2 and adenosine A2A receptors in the basal ganglia.  At present it is believed that A2AR antagonists can be used in combination with the dopamine precursor L-DOPA to minimize the motor symptoms of Parkinson's patients.  However, a considerable body of data indicates that in addn. to ameliorating motor symptoms, adenosine A2AR antagonists may also prevent neurodegeneration.  Despite these promising indications, one further issue must be considered in order to develop fully optimized antiparkinsonian drug therapy, namely the existence of (hetero)dimers/oligomers of G protein-coupled receptors, a topic that is currently the focus of intense debate within the scientific community.  Dopamine D2 receptors (D2Rs) expressed in the striatum are known to form heteromers with A2A adenosine receptors.  Thus, the development of heteromer-specific A2A receptor antagonists represents a promising strategy for the identification of more selective and safer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2pOYdmdfQMLVg90H21EOLACvtfcHk0ljEMk-1UcTACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtl2ntrzP&md5=39fa871338516f7574deb7845029e624</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DArmentero%26aufirst%3DM.%2BT.%26aulast%3DPinna%26aufirst%3DA.%26aulast%3DFerr%25C3%25A9%26aufirst%3DS.%26aulast%3DLanciego%26aufirst%3DJ.%2BL.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DFranco%26aufirst%3DR.%26atitle%3DPast%252C%2520Present%2520and%2520Future%2520of%2520A2A%2520Adenosine%2520Receptor%2520Antagonists%2520in%2520the%2520Therapy%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DPharmacol.%2520Ther.%26date%3D2011%26volume%3D132%26spage%3D280%26epage%3D299%26doi%3D10.1016%2Fj.pharmthera.2011.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riederer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laux, G.</span></span> <span> </span><span class="NLM_article-title">MAO-Inhibitors in Parkinson’s Disease</span>. <i>Exp. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.5607/en.2011.20.1.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.5607%2Fen.2011.20.1.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=22110357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FlsFGlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=1-17&author=P.+Riedererauthor=G.+Laux&title=MAO-Inhibitors+in+Parkinson%E2%80%99s+Disease&doi=10.5607%2Fen.2011.20.1.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">MAO-inhibitors in Parkinson's Disease</span></div><div class="casAuthors">Riederer Peter; Laux Gerd</div><div class="citationInfo"><span class="NLM_cas:title">Experimental neurobiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Monoamine oxidase inhibitors (MAO-I) belong to the earliest drugs tried in Parkinson's disease (PD).  They have been used with or without levodopa (L-DOPA).  Non-selective MAO-I due to their side-effect/adverse reaction profile, like tranylcypromine have limited use in the treatment of depression in PD, while selective, reversible MAO-A inhibitors are recommended due to their easier clinical handling.  For the treatment of akinesia and motor fluctuations selective irreversible MAO-B inhibitors selegiline and rasagiline are recommended.  They are safe and well tolerated at the recommended daily doses.  Their main differences are related to (1) metabolism, (2) interaction with CYP-enzymes and (3) quantitative properties at the molecular biological/genetic level.  Rasagiline is more potent in clinical practise and has a hypothesis driven more favourable side effect/adverse reaction profile due to its metabolism to aminoindan.  Both selegiline and rasagiline have a neuroprotective and neurorestaurative potential.  A head-to head clinical trial would be of utmost interest from both the clinical outcome and a hypothesis-driven point of view.  Selegiline is available as tablet and melting tablet for PD and as transdermal selegiline for depression, while rasagiline is marketed as tablet for PD.  In general, the clinical use of MAO-I nowadays is underestimated.  There should be more efforts to evaluate their clinical potency as antidepressants and antidementive drugs in addition to the final proof of their disease-modifying potential.  In line with this are recent innovative developments of MAO-I plus inhibition of acetylcholine esterase for Alzheimer's disease as well as combined MAO-I and iron chelation for PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLFR15fhEdC_csfMJQC1ZdfW6udTcc2ebSWGNSb38J2bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FlsFGlsw%253D%253D&md5=83bd71bf598fe9e625291fa7c98ac0fd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.5607%2Fen.2011.20.1.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5607%252Fen.2011.20.1.1%26sid%3Dliteratum%253Aachs%26aulast%3DRiederer%26aufirst%3DP.%26aulast%3DLaux%26aufirst%3DG.%26atitle%3DMAO-Inhibitors%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DExp.%2520Neurobiol.%26date%3D2011%26volume%3D20%26spage%3D1%26epage%3D17%26doi%3D10.5607%2Fen.2011.20.1.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Bioactivity Database: An Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D1083</span>– <span class="NLM_lpage">D1090</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+Bioactivity+Database%3A+An+Update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0ljEMk-1UcTACg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520Bioactivity%2520Database%253A%2520An%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Molecular Docking Screening Using Agonist-Bound GPCR Structures: Probing the A<sub>2A</sub> Adenosine Receptor</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1021/ci500639g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500639g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=550-563&author=D.+Rodr%C3%ADguezauthor=Z.+G.+Gaoauthor=S.+M.+Mossauthor=K.+A.+Jacobsonauthor=J.+Carlsson&title=Molecular+Docking+Screening+Using+Agonist-Bound+GPCR+Structures%3A+Probing+the+A2A+Adenosine+Receptor&doi=10.1021%2Fci500639g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screening Using Agonist-Bound GPCR Structures: Probing the A2A Adenosine Receptor</span></div><div class="casAuthors">Rodriguez, David; Gao, Zhang-Guo; Moss, Steven M.; Jacobson, Kenneth A.; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">550-563</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Crystal structures of G protein-coupled receptors (GPCRs) have recently revealed the mol. basis of ligand binding and activation, which has provided exciting opportunities for structure-based drug design.  The A2A adenosine receptor (A2AAR) is a promising therapeutic target for cardiovascular diseases, but progress in this area is limited by the lack of novel agonist scaffolds.  The authors carried out docking screens of 6.7 million com. available mols. against active-like conformations of the A2AAR to investigate whether these structures could guide the discovery of agonists.  Nine out of the 20 predicted agonists were confirmed to be A2AAR ligands, but none of these activated the ARs.  The difficulties in discovering AR agonists using structure-based methods originated from limited at.-level understanding of the activation mechanism and a chem. bias toward antagonists in the screened library.  In particular, the compn. of the screened library was found to strongly reduce the likelihood of identifying AR agonists, which reflected the high ligand complexity required for receptor activation.  Extension of this anal. to other pharmaceutically relevant GPCRs suggested that library screening may not be suitable for targets requiring a complex receptor-ligand interaction network.  The authors' results provide specific directions for the future development of novel A2AAR agonists and general strategies for structure-based drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp89cADSkVJDLVg90H21EOLACvtfcHk0liFfyGFxb0ZHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D&md5=0faec5c3d810a9ef32e5f374b6f9b447</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci500639g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500639g%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DMoss%26aufirst%3DS.%2BM.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DMolecular%2520Docking%2520Screening%2520Using%2520Agonist-Bound%2520GPCR%2520Structures%253A%2520Probing%2520the%2520A2A%2520Adenosine%2520Receptor%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D550%26epage%3D563%26doi%3D10.1021%2Fci500639g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M.
M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carchia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6582</span>– <span class="NLM_lpage">6594</span>, <span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&author=M.%0AM.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+Useful+Decoys%2C+Enhanced+%28DUD-E%29%3A+Better+Ligands+and+Decoys+for+Better+Benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0liFfyGFxb0ZHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520Useful%2520Decoys%252C%2520Enhanced%2520%2528DUD-E%2529%253A%2520Better%2520Ligands%2520and%2520Decoys%2520for%2520Better%2520Benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1561</span>– <span class="NLM_lpage">1573</span>, <span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+Context-Dependent+Ligand+Desolvation+in+Molecular+Docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0liq1lL7tdqDRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520Context-Dependent%2520Ligand%2520Desolvation%2520in%2520Molecular%2520Docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rivara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piersanti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartoccini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamantini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pala, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccioni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borsini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarzia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minetti, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis of (E)-8-(3-Chlorostyryl)caffeine Analogues Leading to 9-Deazaxanthine Derivatives as Dual A<sub>2A</sub> antagonists/MAO-B Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/jm301686s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301686s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVarug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1247-1261&author=S.+Rivaraauthor=G.+Piersantiauthor=F.+Bartocciniauthor=G.+Diamantiniauthor=D.+Palaauthor=T.+Riccioniauthor=M.+A.+Stasiauthor=W.+Cabriauthor=F.+Borsiniauthor=M.+Morauthor=G.+Tarziaauthor=P.+Minetti&title=Synthesis+of+%28E%29-8-%283-Chlorostyryl%29caffeine+Analogues+Leading+to+9-Deazaxanthine+Derivatives+as+Dual+A2A+antagonists%2FMAO-B+Inhibitors&doi=10.1021%2Fjm301686s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of (E)-8-(3-Chlorostyryl)caffeine Analogues Leading to 9-Deazaxanthine Derivatives as Dual A2A Antagonists/MAO-B Inhibitors</span></div><div class="casAuthors">Rivara, Silvia; Piersanti, Giovanni; Bartoccini, Francesca; Diamantini, Giuseppe; Pala, Daniele; Riccioni, Teresa; Stasi, Maria Antonietta; Cabri, Walter; Borsini, Franco; Mor, Marco; Tarzia, Giorgio; Minetti, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic modification of the caffeinyl core and substituents of the ref. compd. (E)-8-(3-chlorostyryl)caffeine led to the 9-deazaxanthine deriv. (E)-6-(4-chlorostyryl)-1,3,5,-trimethyl-1H-pyrrolo[3,2-d]pyrimidine-2,4-(3H,5H)-dione (I), which acts as a dual human A2a antagonist/MAO-B inhibitor (Ki(A2A) = 260 nM; IC50(MAO-B) = 200 nM; IC50(MAO-A) = 10 μM) and dose dependently counteracts haloperidol-induced catalepsy in mice from 30 mg/kg by the oral route.  The compd. is the best balanced A2A antagonist/MAO-B inhibitor reported to date, and it could be considered as a new lead in the field of anti-Parkinson's agents.  A no. of analogs of I were synthesized and qual. SARs are discussed.  Two analogs of I, namely II and III, inhibit MAO-B with IC50 of 68 and 48 nM, resp., being 5-7-fold more potent than the prototypical MAO-B inhibitor deprenyl (IC50 = 334 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCISycjnohtLVg90H21EOLACvtfcHk0liq1lL7tdqDRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVarug%253D%253D&md5=d03678b8923f861c8da65501f05f8e7e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm301686s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301686s%26sid%3Dliteratum%253Aachs%26aulast%3DRivara%26aufirst%3DS.%26aulast%3DPiersanti%26aufirst%3DG.%26aulast%3DBartoccini%26aufirst%3DF.%26aulast%3DDiamantini%26aufirst%3DG.%26aulast%3DPala%26aufirst%3DD.%26aulast%3DRiccioni%26aufirst%3DT.%26aulast%3DStasi%26aufirst%3DM.%2BA.%26aulast%3DCabri%26aufirst%3DW.%26aulast%3DBorsini%26aufirst%3DF.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DTarzia%26aufirst%3DG.%26aulast%3DMinetti%26aufirst%3DP.%26atitle%3DSynthesis%2520of%2520%2528E%2529-8-%25283-Chlorostyryl%2529caffeine%2520Analogues%2520Leading%2520to%25209-Deazaxanthine%2520Derivatives%2520as%2520Dual%2520A2A%2520antagonists%252FMAO-B%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Fjm301686s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stößel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küppers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gütschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Dual Targeting of Adenosine A<sub>2A</sub> Receptors and Monoamine Oxidase B by 4 H-3,1-Benzothiazin-4-Ones</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4580</span>– <span class="NLM_lpage">4596</span>, <span class="refDoi"> DOI: 10.1021/jm400336x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400336x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BC3srpsFOqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4580-4596&author=A.+St%C3%B6%C3%9Felauthor=M.+Schlenkauthor=S.+Hinzauthor=P.+K%C3%BCppersauthor=J.+Heerauthor=M.+G%C3%BCtschowauthor=C.+E.+M%C3%BCller&title=Dual+Targeting+of+Adenosine+A2A+Receptors+and+Monoamine+Oxidase+B+by+4+H-3%2C1-Benzothiazin-4-Ones&doi=10.1021%2Fjm400336x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting of adenosine A(2A) receptors and monoamine oxidase B by 4H-3,1-benzothiazin-4-ones</span></div><div class="casAuthors">Stossel Anne; Schlenk Miriam; Hinz Sonja; Kuppers Petra; Heer Jag; Gutschow Michael; Muller Christa E</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4580-96</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Blockade of A2A adenosine receptors (A2AARs) and inhibition of monoamine oxidase B (MAO-B) in the brain are considered attractive strategies for the treatment of neurodegenerative diseases such as Parkinson's disease (PD).  In the present study, benzothiazinones, e.g., 2-(3-chlorophenoxy)-N-(4-oxo-4H-3,1-benzothiazin-2-yl)acetamide (13), were identified as a novel class of potent MAO-B inhibitors (IC50 human MAO-B: 1.63 nM).  Benzothiazinones with large substituents in the 2-position, e.g., methoxycinnamoylamino, phenylbutyrylamino, or chlorobenzylpiperazinylbenzamido residues (14, 17, 27, and 28), showed high affinity and selectivity for A2AARs (Ki human A2AAR: 39.5-69.5 nM).  By optimizing benzothiazinones for both targets, the first potent, dual-acting A2AAR/MAO-B inhibitors with a nonxanthine structure were developed.  The best derivative was N-(4-oxo-4H-3,1-benzothiazin-2-yl)-4-phenylbutanamide (17, Ki human A2A, 39.5 nM; IC50 human MAO-B, 34.9 nM; selective versus other AR subtypes and MAO-A), which inhibited A2AAR-induced cAMP accumulation and showed competitive, reversible MAO-B inhibition.  The new compounds may be useful tools for validating the A2AAR/MAO-B dual target approach in PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRoeXj8jrdZwDMsgLhIoICsfW6udTcc2eYRQJ1ZhElU-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srpsFOqug%253D%253D&md5=3f5ec3d9dcfd282e2cf26f5efc7e8e86</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400336x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400336x%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25B6%25C3%259Fel%26aufirst%3DA.%26aulast%3DSchlenk%26aufirst%3DM.%26aulast%3DHinz%26aufirst%3DS.%26aulast%3DK%25C3%25BCppers%26aufirst%3DP.%26aulast%3DHeer%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DDual%2520Targeting%2520of%2520Adenosine%2520A2A%2520Receptors%2520and%2520Monoamine%2520Oxidase%2520B%2520by%25204%2520H-3%252C1-Benzothiazin-4-Ones%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4580%26epage%3D4596%26doi%3D10.1021%2Fjm400336x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akkari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunschweiger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küppers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radjainia, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hockemeyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drabczyńska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieć-Kononowicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">1,3-Dialkyl-Substituted tetrahydropyrimido[1,2-F]purine-2,4-Diones as Multiple Target Drugs for the Potential Treatment of Neurodegenerative Diseases</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7435</span>– <span class="NLM_lpage">7452</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.09.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bmc.2013.09.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24139167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CrsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7435-7452&author=P.+Kochauthor=R.+Akkariauthor=A.+Brunschweigerauthor=T.+Borrmannauthor=M.+Schlenkauthor=P.+K%C3%BCppersauthor=M.+K%C3%B6seauthor=H.+Radjainiaauthor=J.+Hockemeyerauthor=A.+Drabczy%C5%84skaauthor=K.+Kie%C4%87-Kononowiczauthor=C.+E.+M%C3%BCller&title=1%2C3-Dialkyl-Substituted+tetrahydropyrimido%5B1%2C2-F%5Dpurine-2%2C4-Diones+as+Multiple+Target+Drugs+for+the+Potential+Treatment+of+Neurodegenerative+Diseases&doi=10.1016%2Fj.bmc.2013.09.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">1,3-Dialkyl-substituted tetrahydropyrimido[1,2-f]purine-2,4-diones as multiple target drugs for the potential treatment of neurodegenerative diseases</span></div><div class="casAuthors">Koch, Pierre; Akkari, Rhalid; Brunschweiger, Andreas; Borrmann, Thomas; Schlenk, Miriam; Kueppers, Petra; Koese, Meryem; Radjainia, Hamid; Hockemeyer, Joerg; Drabczynska, Anna; Kiec-Kononowicz, Katarzyna; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7435-7452</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Adenosine receptors and monoamine oxidases are drug targets for neurodegenerative diseases such as Parkinson's and Alzheimer's disease.  In the present study the authors prepd. a library of 55 mostly novel tetrahydropyrimido[2,1-f]purinediones I [R1 = Me, Et, n-Pr, CH≡CCH2; R2 = Me, Et, n-Pr; R3 = PhCH2, 4-FC6H4, 2-thienyl, etc.].  A synthetic strategy to obtain 3-propargyl-substituted tetrahydropyrimido[2,1-f]purinedione derivs. was developed.  The new compds. were evaluated for their interaction with all four adenosine receptor subtypes and for their ability to inhibit monoamine oxidases (MAO).  Introduction of mono- or di-chloro-substituted Ph, benzyl or phenethyl residues at N9 of the 1,3-di-Me series led to the discovery of a novel class of potent MAO-B inhibitors, the most potent compd. being I [R1 = Me; R2 = Me; R3 = 3,4-Cl2C6H3CH2] 9-(3,4-dichlorobenzyl)-1,3-dimethyl-6,7,8,9-tetrahydropyrimido[1,2-f]purine-2,4(1H,3H)-dione (IC50 human MAO-B: 0.0629 μM), which displayed high selectivity vs. the other investigated targets.  Potent dually active A1/A2A adenosine receptor antagonists were identified, for example, 9-benzyl-1-methyl-3-propargyl-6,7,8,9-tetrahydropyrimido[1,2-f]purine-2,4(1H,3H)dione I [R1 = CH≡CCH2; R2 = Me; R3 = PhCH2] (Ki, human receptors, A1: 0.249 μM, A2A: 0.253 μM).  Several compds. showed triple-target inhibition, the best compd. being I [R1 = CH≡CCH2; R2 = Me; R3 = 3-MeOC6H4CH2] (KiA1: 0.605 μM, KiA2A: 0.417 μM, IC50 MAO-B: 1.80 μM).  Compds. inhibiting several different targets involved in neurodegeneration may exhibit additive or even synergistic effects in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPERWbTreIyLVg90H21EOLACvtfcHk0ljWIAqSlr9BTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CrsbfK&md5=27b1f8581d9f8c96d80d93a1117e6008</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.09.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.09.044%26sid%3Dliteratum%253Aachs%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DAkkari%26aufirst%3DR.%26aulast%3DBrunschweiger%26aufirst%3DA.%26aulast%3DBorrmann%26aufirst%3DT.%26aulast%3DSchlenk%26aufirst%3DM.%26aulast%3DK%25C3%25BCppers%26aufirst%3DP.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DRadjainia%26aufirst%3DH.%26aulast%3DHockemeyer%26aufirst%3DJ.%26aulast%3DDrabczy%25C5%2584ska%26aufirst%3DA.%26aulast%3DKie%25C4%2587-Kononowicz%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3D1%252C3-Dialkyl-Substituted%2520tetrahydropyrimido%255B1%252C2-F%255Dpurine-2%252C4-Diones%2520as%2520Multiple%2520Target%2520Drugs%2520for%2520the%2520Potential%2520Treatment%2520of%2520Neurodegenerative%2520Diseases%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7435%26epage%3D7452%26doi%3D10.1016%2Fj.bmc.2013.09.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langgård, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrøder, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreilgaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boffey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang-Andersen, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR Studies of Analogues of 4-(3,3-Dimethyl-Butyrylamino)-3,5-Difluoro-N-Thiazol-2-Yl-Benzamide (Lu AA41063) as Adenosine A<sub>2A</sub> Receptor Ligands with Improved Aqueous Solubility</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1212</span>– <span class="NLM_lpage">1216</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.01.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bmcl.2015.01.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=25701253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1212-1216&author=G.+K.+Mikkelsenauthor=M.+Langg%C3%A5rdauthor=T.+J.+Schr%C3%B8derauthor=M.+Kreilgaardauthor=E.+B.+J%C3%B8rgensenauthor=G.+Brandtauthor=Y.+Griffonauthor=R.+Boffeyauthor=B.+Bang-Andersen&title=Synthesis+and+SAR+Studies+of+Analogues+of+4-%283%2C3-Dimethyl-Butyrylamino%29-3%2C5-Difluoro-N-Thiazol-2-Yl-Benzamide+%28Lu+AA41063%29+as+Adenosine+A2A+Receptor+Ligands+with+Improved+Aqueous+Solubility&doi=10.1016%2Fj.bmcl.2015.01.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR studies of analogues of 4-(3,3-dimethylbutyrylamino)-3,5-difluoro-N-thiazol-2-yl-benzamide (Lu AA41063) as adenosine A2A receptor ligands with improved aqueous solubility</span></div><div class="casAuthors">Mikkelsen, Gitte Kobberoee; Langgaard, Morten; Schroeder, Tenna Juul; Kreilgaard, Mads; Joergensen, Erling B.; Brandt, Guillaume; Griffon, Yann; Boffey, Ray; Bang-Andersen, Benny</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1212-1216</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An adenosine A2A receptor antagonist may be useful for the treatment of Parkinson's disease.  Synthesis and structure-activity studies starting from 4-(3,3-dimethylbutyrylamino)-3,5-difluoro-N-thiazol-2-ylbenzamide (Lu AA41063) led to a novel series of human (h) A2A receptor antagonists with improved aq. soly.  Compd. I was identified as a key representative from the series, displaying submicromolar hA2A receptor affinity and excellent aq. soly.  Compd. I also displayed good in vitro pharmacokinetic properties and is considered a good starting point for further lead optimization toward hA2A receptor antagonists with improved druggability properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp481kuz7w0qrVg90H21EOLACvtfcHk0ljWIAqSlr9BTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrc%253D&md5=b9e9970f6fbccd21ecfb7bb720a43758</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.01.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.01.062%26sid%3Dliteratum%253Aachs%26aulast%3DMikkelsen%26aufirst%3DG.%2BK.%26aulast%3DLangg%25C3%25A5rd%26aufirst%3DM.%26aulast%3DSchr%25C3%25B8der%26aufirst%3DT.%2BJ.%26aulast%3DKreilgaard%26aufirst%3DM.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DE.%2BB.%26aulast%3DBrandt%26aufirst%3DG.%26aulast%3DGriffon%26aufirst%3DY.%26aulast%3DBoffey%26aufirst%3DR.%26aulast%3DBang-Andersen%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520SAR%2520Studies%2520of%2520Analogues%2520of%25204-%25283%252C3-Dimethyl-Butyrylamino%2529-3%252C5-Difluoro-N-Thiazol-2-Yl-Benzamide%2520%2528Lu%2520AA41063%2529%2520as%2520Adenosine%2520A2A%2520Receptor%2520Ligands%2520with%2520Improved%2520Aqueous%2520Solubility%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1212%26epage%3D1216%26doi%3D10.1016%2Fj.bmcl.2015.01.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenstein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurrell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the Adenosine A<sub>2A</sub> Receptor in Complex with ZM241385 and the Xanthines XAC and Caffeine</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1283</span>– <span class="NLM_lpage">1293</span>, <span class="refDoi"> DOI: 10.1016/j.str.2011.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.str.2011.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21885291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1283-1293&author=A.+S.+Dor%C3%A9author=N.+Robertsonauthor=J.+C.+Erreyauthor=I.+Ngauthor=K.+Hollensteinauthor=B.+Tehanauthor=E.+Hurrellauthor=K.+Bennettauthor=M.+Congreveauthor=F.+Magnaniauthor=C.+G.+Tateauthor=M.+Weirauthor=F.+H.+Marshall&title=Structure+of+the+Adenosine+A2A+Receptor+in+Complex+with+ZM241385+and+the+Xanthines+XAC+and+Caffeine&doi=10.1016%2Fj.str.2011.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Adenosine A2A Receptor in Complex with ZM241385 and the Xanthines XAC and Caffeine</span></div><div class="casAuthors">Dore, Andrew S.; Robertson, Nathan; Errey, James C.; Ng, Irene; Hollenstein, Kaspar; Tehan, Ben; Hurrell, Edward; Bennett, Kirstie; Congreve, Miles; Magnani, Francesca; Tate, Christopher G.; Weir, Malcolm; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1283-1293</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylxanthines, including caffeine and theophylline, are among the most widely consumed stimulant drugs in the world.  These effects are mediated primarily via blockade of adenosine receptors.  Xanthine analogs with improved properties have been developed as potential treatments for diseases such as Parkinson's disease.  Here we report the structures of a thermostabilized adenosine A2A receptor in complex with the xanthines xanthine amine congener and caffeine, as well as the A2A selective inverse agonist ZM241385.  The receptor is crystd. in the inactive state conformation as defined by the presence of a salt bridge known as the ionic lock.  The complete third intracellular loop, responsible for G protein coupling, is visible consisting of extended helixes 5 and 6.  The structures provide new insight into the features that define the ligand binding pocket of the adenosine receptor for ligands of diverse chemotypes as well as the cytoplasmic regions that interact with signal transduction proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWsexwFu6LD7Vg90H21EOLACvtfcHk0lhVs8vlP6QRzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fE&md5=a25db14aa70c2b42b9c3eb98c3c16bae</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DRobertson%26aufirst%3DN.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DNg%26aufirst%3DI.%26aulast%3DHollenstein%26aufirst%3DK.%26aulast%3DTehan%26aufirst%3DB.%26aulast%3DHurrell%26aufirst%3DE.%26aulast%3DBennett%26aufirst%3DK.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMagnani%26aufirst%3DF.%26aulast%3DTate%26aufirst%3DC.%2BG.%26aulast%3DWeir%26aufirst%3DM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DStructure%2520of%2520the%2520Adenosine%2520A2A%2520Receptor%2520in%2520Complex%2520with%2520ZM241385%2520and%2520the%2520Xanthines%2520XAC%2520and%2520Caffeine%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1283%26epage%3D1293%26doi%3D10.1016%2Fj.str.2011.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caccia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvati, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Structures of Human Monoamine Oxidase B Complexes with Selective Noncovalent Inhibitors: Safinamide and Coumarin Analogs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5848</span>– <span class="NLM_lpage">5852</span>, <span class="refDoi"> DOI: 10.1021/jm070677y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070677y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWgurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5848-5852&author=C.+Bindaauthor=J.+Wangauthor=L.+Pisaniauthor=C.+Cacciaauthor=A.+Carottiauthor=P.+Salvatiauthor=D.+E.+Edmondsonauthor=A.+Mattevi&title=Structures+of+Human+Monoamine+Oxidase+B+Complexes+with+Selective+Noncovalent+Inhibitors%3A+Safinamide+and+Coumarin+Analogs&doi=10.1021%2Fjm070677y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of Human Monoamine Oxidase B Complexes with Selective Noncovalent Inhibitors: Safinamide and Coumarin Analogs</span></div><div class="casAuthors">Binda, Claudia; Wang, Jin; Pisani, Leonardo; Caccia, Carla; Carotti, Angelo; Salvati, Patricia; Edmondson, Dale E.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5848-5852</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structures of human monoamine oxidase B (MAO B) in complex with safinamide and two coumarin derivs., all sharing a common benzyloxy substituent, were detd. by x-ray crystallog.  These compds. competitively inhibit MAO B with Ki values in the 0.1-0.5 μM range that are 30-700-fold lower than those obsd. with MAO A.  The inhibitors bind noncovalently to MAO B, occupying both the entrance and the substrate cavities and showing a similarly oriented benzyloxy substituent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBPIubufo89bVg90H21EOLACvtfcHk0lhVs8vlP6QRzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWgurjI&md5=448f65696f4870e2301f46a6f015e35c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm070677y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070677y%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPisani%26aufirst%3DL.%26aulast%3DCaccia%26aufirst%3DC.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DSalvati%26aufirst%3DP.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructures%2520of%2520Human%2520Monoamine%2520Oxidase%2520B%2520Complexes%2520with%2520Selective%2520Noncovalent%2520Inhibitors%253A%2520Safinamide%2520and%2520Coumarin%2520Analogs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5848%26epage%3D5852%26doi%3D10.1021%2Fjm070677y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS Algorithm for Detection of Structurally Similar Protein Binding Sites by Local Structural Alignment</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1168</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btq100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fbioinformatics%2Fbtq100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20305268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1160-1168&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+Algorithm+for+Detection+of+Structurally+Similar+Protein+Binding+Sites+by+Local+Structural+Alignment&doi=10.1093%2Fbioinformatics%2Fbtq100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1160-1168</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exploitation of locally similar 3D patterns of physicochem. properties on the surface of a protein for detection of binding sites that may lack sequence and global structural conservation.  An algorithm, ProBiS is described that detects structurally similar sites on protein surfaces by local surface structure alignment.  It compares the query protein to members of a database of protein 3D structures and detects with sub-residue precision, structurally similar sites as patterns of physicochem. properties on the protein surface.  Using an efficient max. clique algorithm, the program identifies proteins that share local structural similarities with the query protein and generates structure-based alignments of these proteins with the query.  Structural similarity scores are calcd. for the query protein's surface residues, and are expressed as different colors on the query protein surface.  The algorithm has been used successfully for the detection of protein-protein, protein-small ligand and protein-DNA binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproKvL-3-CSLVg90H21EOLACvtfcHk0lhVs8vlP6QRzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D&md5=7ff0eecdf0ea5f7d9b82fd721ca88156</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq100%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520Algorithm%2520for%2520Detection%2520of%2520Structurally%2520Similar%2520Protein%2520Binding%2520Sites%2520by%2520Local%2520Structural%2520Alignment%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26spage%3D1160%26epage%3D1168%26doi%3D10.1093%2Fbioinformatics%2Fbtq100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">ZINC – A Free Database of Commercially Available Compounds for Virtual Screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+%E2%80%93+A+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eaQyK_upW1OULntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520%25E2%2580%2593%2520A%2520Free%2520Database%2520of%2520Commercially%2520Available%2520Compounds%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plückthun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kummer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Ligand Discovery for a Peptide-Binding GPCR by Structure-Based Screening of Fragment- and Lead-Like Chemical Libraries</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">735</span>– <span class="NLM_lpage">745</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFOitbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=735-745&author=A.+Ranganathanauthor=P.+Heineauthor=A.+Rudlingauthor=A.+Pl%C3%BCckthunauthor=L.+Kummerauthor=J.+Carlsson&title=Ligand+Discovery+for+a+Peptide-Binding+GPCR+by+Structure-Based+Screening+of+Fragment-+and+Lead-Like+Chemical+Libraries&doi=10.1021%2Facschembio.6b00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Discovery for a Peptide-Binding GPCR by Structure-Based Screening of Fragment- and Lead-Like Chemical Libraries</span></div><div class="casAuthors">Ranganathan, Anirudh; Heine, Philipp; Rudling, Axel; Pluckthun, Andreas; Kummer, Lutz; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">735-745</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide-recognizing G protein-coupled receptors (GPCRs) are promising therapeutic targets, but often resist drug discovery efforts.  Detn. of crystal structures for peptide-binding GPCRs has provided opportunities to explore structure-based methods in lead development.  Mol. docking screens of two chem. libraries, contg. either fragment- or lead-like compds., against a Neurotensin receptor 1 crystal structure allowed for a comparison between different drug development strategies for peptide-binding GPCRs.  A total of 2.3 million mols. were screened computationally and 25 fragments and 27 leads that were top-ranked in each library were selected for exptl. evaluation.  Of these, eight fragments and five leads were confirmed as ligands by surface plasmon resonance.  The hit rate for the fragment screen (32%) was thus higher than for the lead-like library (19%), but the affinities of the fragments were ∼100-fold lower.  Both screens returned unique scaffolds and demonstrated that a crystal structure of a stabilized peptide-binding GPCR can guide the discovery of small-mol. agonists.  The complementary advantages of exploring fragment and lead-like chem. space suggest that these strategies should be applied synergistically in structure-based screens against challenging GPCR targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMYKg8sFuRtbVg90H21EOLACvtfcHk0ljvVoaEQXshfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFOitbfK&md5=b1b72cda63503ab45ad0bd57cd70943f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00646%26sid%3Dliteratum%253Aachs%26aulast%3DRanganathan%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DP.%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DPl%25C3%25BCckthun%26aufirst%3DA.%26aulast%3DKummer%26aufirst%3DL.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DLigand%2520Discovery%2520for%2520a%2520Peptide-Binding%2520GPCR%2520by%2520Structure-Based%2520Screening%2520of%2520Fragment-%2520and%2520Lead-Like%2520Chemical%2520Libraries%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D735%26epage%3D745%26doi%3D10.1021%2Facschembio.6b00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Ligand Discovery from a Dopamine D<sub>3</sub> Receptor Homology Model and Crystal Structure</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+Discovery+from+a+Dopamine+D3+Receptor+Homology+Model+and+Crystal+Structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0ljvVoaEQXshfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520Discovery%2520from%2520a%2520Dopamine%2520D3%2520Receptor%2520Homology%2520Model%2520and%2520Crystal%2520Structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The Role of Ligand Efficiency Metrics in Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+Role+of+Ligand+Efficiency+Metrics+in+Drug+Discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0ljvVoaEQXshfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520Role%2520of%2520Ligand%2520Efficiency%2520Metrics%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballesteros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span> <span> </span><span class="NLM_article-title">Integrated Methods for the Construction of Three-Dimensional Models and Computational Probing of Structure-Function Relations in G Protein-Coupled Receptors</span>. <i>Methods Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1016/S1043-9471(05)80049-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1995&pages=366-428&author=J.+A.+Ballesterosauthor=H.+Weinstein&title=Integrated+Methods+for+the+Construction+of+Three-Dimensional+Models+and+Computational+Probing+of+Structure-Function+Relations+in+G+Protein-Coupled+Receptors&doi=10.1016%2FS1043-9471%2805%2980049-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span></div><div class="casAuthors">Ballesteros, Juan A.; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Neurosciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">366-428</span>CODEN:
                <span class="NLM_cas:coden">MENEE5</span>;
        ISSN:<span class="NLM_cas:issn">1043-9471</span>.
    </div><div class="casAbstract">A review, with 135 refs., on approaches that can be used to resolve the apparent ambiguities that burden the pharmacol. testing of G protein-coupled receptor (GPCR) models, based on the integration of structural information about the receptor, about mutants, and about the changes induced by ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfENoRspqa7LVg90H21EOLACvtfcHk0lh8fX4VzzORFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D&md5=d3f99e9d07cea245a770f3f099bc8cc7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3DIntegrated%2520Methods%2520for%2520the%2520Construction%2520of%2520Three-Dimensional%2520Models%2520and%2520Computational%2520Probing%2520of%2520Structure-Function%2520Relations%2520in%2520G%2520Protein-Coupled%2520Receptors%26jtitle%3DMethods%2520Neurosci.%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428%26doi%3D10.1016%2FS1043-9471%2805%2980049-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Similarity-Potency Trees: A Method to Search for SAR Information in Compound Data Sets and Derive SAR Rules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1395</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1021/ci100197b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100197b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1395-1409&author=M.+Wawerauthor=J.+Bajorath&title=Similarity-Potency+Trees%3A+A+Method+to+Search+for+SAR+Information+in+Compound+Data+Sets+and+Derive+SAR+Rules&doi=10.1021%2Fci100197b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-Potency Trees: A Method to Search for SAR Information in Compound Data Sets and Derive SAR Rules</span></div><div class="casAuthors">Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1395-1409</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An intuitive and generally applicable anal. method, termed similarity-potency tree (SPT), is introduced to mine structure-activity relation (SAR) information in compd. data sets of any source.  Only compd. potency values and nearest-neighbor similarity relationships are considered.  Rather than analyzing a data set as a whole, in part overlapping compd. neighborhoods are systematically generated and represented as SPTs.  This local anal. scheme simplifies the evaluation of SAR information and SPTs of high SAR information content are easily identified.  By inspecting only a limited no. of compd. neighborhoods, it is also straightforward to det. whether data sets contain only little or no interpretable SAR information.  Interactive anal. of SPTs is facilitated by reading the trees in two directions, which makes it possible to ext. SAR rules, if available, in a consistent manner.  The simplicity and interpretability of the data structure and the ease of calcn. are characteristic features of this approach.  We apply the methodol. to high-throughput screening and lead optimization data sets, compare the approach to std. clustering techniques, illustrate how SAR rules are derived, and provide some practical guidance how to best utilize the methodol.  The SPT program is made freely available to the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocIJW9gu2adrVg90H21EOLACvtfcHk0lh8fX4VzzORFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D&md5=7d4609c7ca3a695c0658233ca7867616</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci100197b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100197b%26sid%3Dliteratum%253Aachs%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSimilarity-Potency%2520Trees%253A%2520A%2520Method%2520to%2520Search%2520for%2520SAR%2520Information%2520in%2520Compound%2520Data%2520Sets%2520and%2520Derive%2520SAR%2520Rules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1395%26epage%3D1409%26doi%3D10.1021%2Fci100197b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurrell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Drug Design of Chromone Antagonists of the Adenosine A<sub>2A</sub> Receptor</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1039/C3MD00338H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1039%2FC3MD00338H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXms1WqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=571-575&author=S.+P.+Andrewsauthor=J.+S.+Masonauthor=E.+Hurrellauthor=M.+Congreve&title=Structure-Based+Drug+Design+of+Chromone+Antagonists+of+the+Adenosine+A2A+Receptor&doi=10.1039%2FC3MD00338H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design of chromone antagonists of the adenosine A2A receptor</span></div><div class="casAuthors">Andrews, Stephen P.; Mason, Jonathan S.; Hurrell, Edward; Congreve, Miles</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">571-575</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The structure-guided optimization of a hit series of chromone derivs., previously identified using virtual screening of homol. models of the adenosine A2A receptor, has led to the discovery of potent, selective and ligand efficient antagonists.  Lipophilic hotspots and calcd. water networks were modelled within the receptor binding site to facilitate rational ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplTU8yhstYi7Vg90H21EOLACvtfcHk0lh8fX4VzzORFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXms1WqsLw%253D&md5=ab3c3f180fe8f5f3a87f54035f92593d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC3MD00338H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00338H%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHurrell%26aufirst%3DE.%26aulast%3DCongreve%26aufirst%3DM.%26atitle%3DStructure-Based%2520Drug%2520Design%2520of%2520Chromone%2520Antagonists%2520of%2520the%2520Adenosine%2520A2A%2520Receptor%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D571%26epage%3D575%26doi%3D10.1039%2FC3MD00338H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matricon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambertucci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezzati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiteh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dal Ben, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Fragment Optimization for GPCRs by Molecular Dynamics Free Energy Calculations: Probing Druggable Subpockets of the A<sub>2A</sub> Adenosine Receptor Binding Site</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6398</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-04905-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fs41598-017-04905-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=28743961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BC1cfhtFKktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=6398&author=P.+Matriconauthor=A.+Ranganathanauthor=E.+Warnickauthor=Z.-G.+Gaoauthor=A.+Rudlingauthor=C.+Lambertucciauthor=G.+Marucciauthor=A.+Ezzatiauthor=M.+Jaitehauthor=D.+Dal+Benauthor=K.+A.+Jacobsonauthor=J.+Carlsson&title=Fragment+Optimization+for+GPCRs+by+Molecular+Dynamics+Free+Energy+Calculations%3A+Probing+Druggable+Subpockets+of+the+A2A+Adenosine+Receptor+Binding+Site&doi=10.1038%2Fs41598-017-04905-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment optimization for GPCRs by molecular dynamics free energy calculations: Probing druggable subpockets of the A 2A adenosine receptor binding site</span></div><div class="casAuthors">Matricon Pierre; Jaiteh Mariama; Carlsson Jens; Ranganathan Anirudh; Rudling Axel; Ezzati Aitakin; Warnick Eugene; Gao Zhan-Guo; Jacobson Kenneth A; Lambertucci Catia; Marucci Gabriella; Dal Ben Diego</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6398</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based lead discovery is becoming an increasingly popular strategy for drug discovery.  Fragment screening identifies weakly binding compounds that require optimization to become high-affinity leads.  As design of leads from fragments is challenging, reliable computational methods to guide optimization would be invaluable.  We evaluated using molecular dynamics simulations and the free energy perturbation method (MD/FEP) in fragment optimization for the A2A adenosine receptor, a pharmaceutically relevant G protein-coupled receptor.  Optimization of fragments exploring two binding site subpockets was probed by calculating relative binding affinities for 23 adenine derivatives, resulting in strong agreement with experimental data (R(2) = 0.78).  The predictive power of MD/FEP was significantly better than that of an empirical scoring function.  We also demonstrated the potential of the MD/FEP to assess multiple binding modes and to tailor the thermodynamic profile of ligands during optimization.  Finally, MD/FEP was applied prospectively to optimize three nonpurine fragments, and predictions for 12 compounds were evaluated experimentally.  The direction of the change in binding affinity was correctly predicted in a majority of the cases, and agreement with experiment could be improved with rigorous parameter derivation.  The results suggest that MD/FEP will become a powerful tool in structure-driven optimization of fragments to lead candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS_3RY8BzBzWXWEEzDrymxmfW6udTcc2eahLAP2pgA0lbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfhtFKktQ%253D%253D&md5=05f0210d0cbe5f4cd58677b8747dffa0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-04905-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-04905-0%26sid%3Dliteratum%253Aachs%26aulast%3DMatricon%26aufirst%3DP.%26aulast%3DRanganathan%26aufirst%3DA.%26aulast%3DWarnick%26aufirst%3DE.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DLambertucci%26aufirst%3DC.%26aulast%3DMarucci%26aufirst%3DG.%26aulast%3DEzzati%26aufirst%3DA.%26aulast%3DJaiteh%26aufirst%3DM.%26aulast%3DDal%2BBen%26aufirst%3DD.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DFragment%2520Optimization%2520for%2520GPCRs%2520by%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Calculations%253A%2520Probing%2520Druggable%2520Subpockets%2520of%2520the%2520A2A%2520Adenosine%2520Receptor%2520Binding%2520Site%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D6398%26doi%3D10.1038%2Fs41598-017-04905-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liXoMAC_fXjQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">A Detergent-Based Assay for the Detection of Promiscuous Inhibitors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnprot.2006.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=17191086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=550-553&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=A+Detergent-Based+Assay+for+the+Detection+of+Promiscuous+Inhibitors&doi=10.1038%2Fnprot.2006.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A detergent-based assay for the detection of promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">550-553</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At micromolar concns., many small mols. self-assoc. into colloidal aggregates that non-specifically inhibit enzymes and other proteins.  Here the authors describe a protocol for identifying aggregate-based inhibitors and distinguishing them from small mols. that inhibit via specific mechanisms.  As a convenient proxy for promiscuous, aggregate-based inhibition, the authors monitor inhibition of β-lactamase in the absence and presence of detergent.  Inhibition that is attenuated in the presence of detergent is characteristic of an aggregate-based mechanism.  In the 96-well-format assay described here, about 200 mols. can be tested, in duplicate, per h for detergent-dependent sensitivity.  Furthermore, the authors also describe simple expts. that can offer addnl. confirmation of aggregate-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGQSBvMgM-7Vg90H21EOLACvtfcHk0liXoMAC_fXjQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO&md5=8aa7562b5f69c896af7e133ecc35e6de</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.77%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520Detergent-Based%2520Assay%2520for%2520the%2520Detection%2520of%2520Promiscuous%2520Inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D550%26epage%3D553%26doi%3D10.1038%2Fnprot.2006.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">PubChem BioAssay: 2017 Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D955</span>– <span class="NLM_lpage">D963</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fnar%2Fgkw1118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=27899599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D955-D963&author=Y.+Wangauthor=S.+H.+Bryantauthor=T.+Chengauthor=J.+Wangauthor=A.+Gindulyteauthor=B.+A.+Shoemakerauthor=P.+A.+Thiessenauthor=S.+Heauthor=J.+Zhang&title=PubChem+BioAssay%3A+2017+Update&doi=10.1093%2Fnar%2Fgkw1118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem BioAssay: 2017 update</span></div><div class="casAuthors">Wang, Yanli; Bryant, Stephen H.; Cheng, Tiejun; Wang, Jiyao; Gindulyte, Asta; Shoemaker, Benjamin A.; Thiessen, Paul A.; He, Siqian; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D955-D963</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem's BioAssay database (https://pubchem.ncbi.nlm.nih.gov) has served as a public repository for small-mol. and RNAi screening data since 2004 providing open access of its data content to the community.  PubChem accepts data submission from worldwide researchers at academia, industry and government agencies.  PubChem also collaborates with other chem. biol. database stakeholders with data exchange.  With over a decade's development effort, it becomes an important information resource supporting drug discovery and chem. biol. research.  To facilitate data discovery, PubChem is integrated with all other databases at NCBI.  In this work, we provide an update for the Pub- Chem BioAssay database describing several recent development including added sources of research data, redesigned BioAssay record page, new BioAssay classification browser and new features in the Upload system facilitating data sharing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptFOpprG7GhLVg90H21EOLACvtfcHk0lg16Y46sYN_zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhsbw%253D&md5=4043316b6be487277f360fb4cc2beabb</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1118%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DPubChem%2520BioAssay%253A%25202017%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD955%26epage%3DD963%26doi%3D10.1093%2Fnar%2Fgkw1118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Docking Screens for Novel Ligands Conferring New Biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4103</span>– <span class="NLM_lpage">4120</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b02008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b02008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Siu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4103-4120&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Docking+Screens+for+Novel+Ligands+Conferring+New+Biology&doi=10.1021%2Facs.jmedchem.5b02008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Screens for Novel Ligands Conferring New Biology</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4103-4120</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is now plausible to dock libraries of 10 million mols. against targets over several days or weeks.  When the mols. screened are com. available, they may be rapidly tested to find new leads.  Although docking retains important liabilities (it cannot calc. affinities accurately nor even reliably rank order high-scoring mols.), it can often can distinguish likely from unlikely ligands, often with hit rates above 10%.  Here we summarize the improvements in libraries, target quality, and methods that have supported these advances, and the open access resources that make docking accessible.  Recent docking screens for new ligands are sketched, as are the binding, crystallog., and in vivo assays that support them.  Like any technique, controls are crucial, and key exptl. ones are reviewed.  With such controls, docking campaigns can find ligands with new chemotypes, often revealing the new biol. that may be docking's greatest impact over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCmaLVTnkkoLVg90H21EOLACvtfcHk0lg16Y46sYN_zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Siu7o%253D&md5=5cdfdd9f2d6b521398b151adb3ef145a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520Screens%2520for%2520Novel%2520Ligands%2520Conferring%2520New%2520Biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4103%26epage%3D4120%26doi%3D10.1021%2Facs.jmedchem.5b02008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of GPCR Ligands by Molecular Docking Screening: Novel Opportunities Provided by Crystal Structures</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2484</span>– <span class="NLM_lpage">2503</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150701112853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.2174%2F1568026615666150701112853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=26126906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFansrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=2484-2503&author=D.+Rodr%C3%ADguezauthor=A.+Ranganathanauthor=J.+Carlsson&title=Discovery+of+GPCR+Ligands+by+Molecular+Docking+Screening%3A+Novel+Opportunities+Provided+by+Crystal+Structures&doi=10.2174%2F1568026615666150701112853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of GPCR Ligands by Molecular Docking Screening: Novel Opportunities Provided by Crystal Structures</span></div><div class="casAuthors">Rodriguez, David; Ranganathan, Anirudh; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2484-2503</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) constitute the largest group of human membrane proteins and have received significant attention in drug discovery for their important roles in physiol. processes.  Drug development for GPCRs has been remarkably successful and several of the most profitable pharmaceuticals on the market target members of this superfamily.  Breakthroughs in structural biol. for GPCRs have revealed how their binding sites recognize extracellular mols. at the at. level.  High-resoln. crystal structures of GPCR-drug complexes capturing different receptor conformations are now available, which have provided insights into how ligands stabilize different functional states.  Recently, the basis for subtype selectivity and novel allosteric binding sites has also been revealed by crystal structures.  These accomplishments provide exciting opportunities to identify novel GPCR ligands using <i>in silico</i> structure-based methods such as mol. docking.  Increased computational power now enables docking screens of large chem. libraries to identify mols. that complement GPCR binding sites, which may provide possibilities to identify ligands with tailored pharmacol. properties.  This review focuses on prospective docking screens against GPCRs and how this technique can be used to identify lead candidates with specific signaling or selectivity profiles.  The current state of this field suggests that mol. docking, in combination with further understanding of GPCR signaling, will play an important role in future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxcu4b1g3uerVg90H21EOLACvtfcHk0lg16Y46sYN_zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFansrjN&md5=31a75abcc24bf453ad155903a7454d3e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150701112853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150701112853%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DRanganathan%26aufirst%3DA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520GPCR%2520Ligands%2520by%2520Molecular%2520Docking%2520Screening%253A%2520Novel%2520Opportunities%2520Provided%2520by%2520Crystal%2520Structures%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2015%26volume%3D15%26spage%3D2484%26epage%3D2503%26doi%3D10.2174%2F1568026615666150701112853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of A<sub>2A</sub> Adenosine Receptor Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3748</span>– <span class="NLM_lpage">3755</span>, <span class="refDoi"> DOI: 10.1021/jm100240h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.-G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-Based+Discovery+of+A2A+Adenosine+Receptor+Ligands&doi=10.1021%2Fjm100240h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0lia48JIOqaR_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-Based%2520Discovery%2520of%2520A2A%2520Adenosine%2520Receptor%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755%26doi%3D10.1021%2Fjm100240h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaakola, V.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of Novel Chemotypes for Adenosine A<sub>2A</sub> Receptor Antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1799</span>– <span class="NLM_lpage">1809</span>, <span class="refDoi"> DOI: 10.1021/jm901647p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901647p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlKqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1799-1809&author=V.+Katritchauthor=V.-P.+Jaakolaauthor=J.+R.+Laneauthor=J.+Linauthor=A.+P.+IJzermanauthor=M.+Yeagerauthor=I.+Kufarevaauthor=R.+C.+Stevensauthor=R.+Abagyan&title=Structure-Based+Discovery+of+Novel+Chemotypes+for+Adenosine+A2A+Receptor+Antagonists&doi=10.1021%2Fjm901647p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of Novel Chemotypes for Adenosine A2A Receptor Antagonists</span></div><div class="casAuthors">Katritch, Vsevolod; Jaakola, Veli-Pekka; Lane, J. Robert; Lin, Judy; IJzerman, Adriaan P.; Yeager, Mark; Kufareva, Irina; Stevens, Raymond C.; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1799-1809</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent progress in crystallog. of G-protein coupled receptors opens an unprecedented venue for structure-based GPCR drug discovery.  To test efficiency of the structure-based approach, we performed mol. docking and virtual ligand screening (VLS) of more than 4 million com. available "drug-like" and "lead-like" compds. against the A2AAR 2.6 Å resoln. crystal structure.  Out of 56 high ranking compds. tested in A2AAR binding assays, 23 showed affinities under 10 μM, 11 of those had sub-μM affinities and two compds. had affinities under 60 nM.  The identified hits represent at least 9 different chem. scaffolds and are characterized by very high ligand efficiency (0.3-0.5 kcal/mol per heavy atom).  Significant A2AAR antagonist activities were confirmed for 10 out of 13 ligands tested in functional assays.  High success rate, novelty, and diversity of the chem. scaffolds and strong ligand efficiency of the A2AAR antagonists identified in this study suggest practical applicability of receptor-based VLS in GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX-YAwoEVzEbVg90H21EOLACvtfcHk0lia48JIOqaR_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlKqsw%253D%253D&md5=5a54df43f6edd20d83e7e5942e2f9811</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm901647p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901647p%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DJaakola%26aufirst%3DV.-P.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DYeager%26aufirst%3DM.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Novel%2520Chemotypes%2520for%2520Adenosine%2520A2A%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1799%26epage%3D1809%26doi%3D10.1021%2Fjm901647p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geldenhuys, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvesh, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Schyf, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, R. T.</span></span> <span> </span><span class="NLM_article-title">Identification of Novel Monoamine Oxidase B Inhibitors by Structure-Based Virtual Screening</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5295</span>– <span class="NLM_lpage">5298</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.06.128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bmcl.2010.06.128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20650633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVehtr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=5295-5298&author=W.+J.+Geldenhuysauthor=A.+S.+Darveshauthor=M.+O.+Funkauthor=C.+J.+Van+der+Schyfauthor=R.+T.+Carroll&title=Identification+of+Novel+Monoamine+Oxidase+B+Inhibitors+by+Structure-Based+Virtual+Screening&doi=10.1016%2Fj.bmcl.2010.06.128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel monoamine oxidase B inhibitors by structure-based virtual screening</span></div><div class="casAuthors">Geldenhuys, Werner J.; Darvesh, Altaf S.; Funk, Max O.; Van der Schyf, Cornelis J.; Carroll, Richard T.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5295-5298</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Parkinson's disease is a severe debilitating neurodegenerative disorder.  Recently, it was shown that the peroxisome proliferating-activator receptor-γ agonist pioglitazone protected mice from 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine neurotoxicity due to its ability to inhibit monoamine oxidase B (MAO-B).  Docking studies were initiated to investigate pioglitazone's interactions within the substrate cavity of MAO-B.  Modeling studies indicated that the thiazolidinedione (TZD) moiety was a likely candidate for its specificity to MAO-B.  To explore this potential novel MAO-B scaffold, we performed a structure-based virtual screen to identify addnl. MAO-B inhibitors.  Our search identified eight novel compds. contg. the TZD-moiety that allowed for a limited study to identify structural requirements for binding to MAO-B.  Inhibition assays identified two TZDs (A6355 and L136662) which were found to inhibit recombinant human MAO-B with IC50 values of 82 and 195 nM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppUeMyr_qAULVg90H21EOLACvtfcHk0lia48JIOqaR_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVehtr3K&md5=27589110233cc8394f51afd1f7c0f204</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.06.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.06.128%26sid%3Dliteratum%253Aachs%26aulast%3DGeldenhuys%26aufirst%3DW.%2BJ.%26aulast%3DDarvesh%26aufirst%3DA.%2BS.%26aulast%3DFunk%26aufirst%3DM.%2BO.%26aulast%3DVan%2Bder%2BSchyf%26aufirst%3DC.%2BJ.%26aulast%3DCarroll%26aufirst%3DR.%2BT.%26atitle%3DIdentification%2520of%2520Novel%2520Monoamine%2520Oxidase%2520B%2520Inhibitors%2520by%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D5295%26epage%3D5298%26doi%3D10.1016%2Fj.bmcl.2010.06.128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachhawat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M. L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sands, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Adenosine A<sub>2A</sub> Receptor Bound to an Antagonist Reveals a Potential Allosteric Pocket</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2066</span>– <span class="NLM_lpage">2071</span>, <span class="refDoi"> DOI: 10.1073/pnas.1621423114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1073%2Fpnas.1621423114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=28167788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFCksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=2066-2071&author=B.+Sunauthor=P.+Bachhawatauthor=M.+L.-H.+Chuauthor=M.+Woodauthor=T.+Ceskaauthor=Z.+A.+Sandsauthor=J.+Mercierauthor=F.+Lebonauthor=T.+S.+Kobilkaauthor=B.+K.+Kobilka&title=Crystal+Structure+of+the+Adenosine+A2A+Receptor+Bound+to+an+Antagonist+Reveals+a+Potential+Allosteric+Pocket&doi=10.1073%2Fpnas.1621423114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the adenosine A2A receptor bound to an antagonist reveals a potential allosteric pocket</span></div><div class="casAuthors">Sun, Bingfa; Bachhawat, Priti; Chu, Matthew Ling-Hon; Wood, Martyn; Ceska, Tom; Sands, Zara A.; Mercier, Joel; Lebon, Florence; Kobilka, Tong Sun; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2066-2071</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The adenosine A2A receptor (A2AR) has long been implicated in cardiovascular disorders.  As more selective A2AR ligands are being identified, its roles in other disorders, such as Parkinson's disease, are starting to emerge, and A2AR antagonists are important drug candidates for non-dopaminergic anti-Parkinson treatment.  Here, the authors report the crystal structure of A2A receptor bound to compd. 1, a novel A2AR/N-methyl-D-aspartate (NMDA) receptor subtype 2B (NR2B; NMDA receptor 2B) dual antagonist and potential anti-Parkinson candidate compd., at 3.5 Å resoln.  The A2A receptor with a cytochrome b562-RIL (BRIL) fusion (A2AR-BRIL) in intracellular loop 3 (ICL3) was crystd. in detergent micelles using vapor-phase diffusion.  Whereas A2AR-BRIL bound to antagonist ZM241385 was previously been crystd. in the lipidic cubic phase (LCP), structural differences in the compd. 1-bound A2AR-BRIL prevented formation of the lattice obsd. with the ZM241385-bound receptor.  The crystals grew with a type II crystal lattice in contrast to the typical type I packing seen from membrane protein structures crystd. in LCP.  Compd. 1 bound in a position that overlapped with the native ligand, adenosine, but its methoxyphenyl group extended to an exosite not previously obsd. in other A2AR structures.  Structural anal. revealed that compd. 1 binding resulted in the unique conformations of 2 Tyr residues (Tyr91.35 and Tyr2717.36) which are crit. for the formation of the exosite.  The structure revealed insights into antagonist binding that were not obsd. in other A2AR structures, highlighting flexibility in the binding pocket that may facilitate the development of A2AR-selective compds. for the treatment of Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5SraoNhZQ0bVg90H21EOLACvtfcHk0lh3K3y8OFpajw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFCksb8%253D&md5=58ac85e930066ef7a6788c4d7f3a72b3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1621423114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1621423114%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DB.%26aulast%3DBachhawat%26aufirst%3DP.%26aulast%3DChu%26aufirst%3DM.%2BL.-H.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DCeska%26aufirst%3DT.%26aulast%3DSands%26aufirst%3DZ.%2BA.%26aulast%3DMercier%26aufirst%3DJ.%26aulast%3DLebon%26aufirst%3DF.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Adenosine%2520A2A%2520Receptor%2520Bound%2520to%2520an%2520Antagonist%2520Reveals%2520a%2520Potential%2520Allosteric%2520Pocket%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D2066%26epage%3D2071%26doi%3D10.1073%2Fpnas.1621423114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barelier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">The Recognition of Identical Ligands by Unrelated Proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2772</span>– <span class="NLM_lpage">2784</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2772-2784&author=S.+Barelierauthor=T.+Sterlingauthor=M.+J.+O%E2%80%99Mearaauthor=B.+K.+Shoichet&title=The+Recognition+of+Identical+Ligands+by+Unrelated+Proteins&doi=10.1021%2Facschembio.5b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The recognition of identical ligands by unrelated proteins</span></div><div class="casAuthors">Barelier, Sarah; Sterling, Teague; O'Meara, Matthew J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2772-2784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of drugs and reagents to off-targets is well-known.  Whereas many off-targets are related to the primary target by sequence and fold, many ligands bind to unrelated pairs of proteins, and these are harder to anticipate.  If the binding site in the off-target can be related to that of the primary target, this challenge resolves into aligning the 2 pockets.  However, other cases are possible: the ligand might interact with entirely different residues and environments in the off-target, or wholly different ligand atoms may be implicated in the 2 complexes.  To investigate these scenarios at at. resoln., the structures of 59 ligands in 116 complexes (62 pairs in total), where the protein pairs were unrelated by fold but bound an identical ligand, were examd.  In almost half of the pairs, the ligand interacted with unrelated residues in the 2 proteins (29 pairs), and in 14 of the pairs wholly different ligand moieties were implicated in each complex.  Even in those 19 pairs of complexes that presented similar environments to the ligand, ligand superposition rarely resulted in the overlap of related residues.  There appears to be no single pattern-matching "code" for identifying binding sites in unrelated proteins that bind identical ligands, although modeling suggests that there might be a limited no. of different patterns that suffice to recognize different ligand functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1_lQJDNMCrVg90H21EOLACvtfcHk0lh3K3y8OFpajw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J&md5=9d6ac31c2a58bc85f77e7eb34649c195</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00683%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520Recognition%2520of%2520Identical%2520Ligands%2520by%2520Unrelated%2520Proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2772%26epage%3D2784%26doi%3D10.1021%2Facschembio.5b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span></span> <span> </span><span class="NLM_article-title">From Magic Bullets to Designed Multiple Ligands</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">651</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03163-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2FS1359-6446%2804%2903163-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=15279847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVSktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=641-651&author=R.+Morphyauthor=C.+Kayauthor=Z.+Rankovic&title=From+Magic+Bullets+to+Designed+Multiple+Ligands&doi=10.1016%2FS1359-6446%2804%2903163-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">From magic bullets to designed multiple ligands</span></div><div class="casAuthors">Morphy, Richard; Kay, Corinne; Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">641-651</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  A major trend in medicinal chem. in the coming years will be the rational design of ligands that are capable of modulating multiple disease-relevant targets in a specific manner.  Increasingly, it is being recognized that a balanced modulation of several targets can provide a superior therapeutic effect and side effect profile compared to the action of a selective ligand.  Rational approaches in which structural features from selective ligands are combined have produced designed multiple ligands that span a wide variety of targets and target classes.  A key challenge in the design of multiple ligands is attaining a balanced activity at each target of interest while simultaneously achieving a wider selectivity and a suitable pharmacokinetic profile.  An anal. of literature examples reveals trends and insights that might help medicinal chemists discover the next generation of these types of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcsGpfzZok5rVg90H21EOLACvtfcHk0lh3K3y8OFpajw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVSktb8%253D&md5=cd2491baad707043dbee4f59ccad9e1d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903163-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903163-0%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26aulast%3DKay%26aufirst%3DC.%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DFrom%2520Magic%2520Bullets%2520to%2520Designed%2520Multiple%2520Ligands%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D641%26epage%3D651%26doi%3D10.1016%2FS1359-6446%2804%2903163-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Der Walt, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terre’Blanche, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzer, J. P.</span></span> <span> </span><span class="NLM_article-title">The Adenosine Receptor Affinities and Monoamine Oxidase B Inhibitory Properties of Sulfanylphthalimide Analogues</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">123</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2015.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bioorg.2015.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=25746740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1aqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=117-123&author=M.+M.+Van%0ADer+Waltauthor=G.+Terre%E2%80%99Blancheauthor=A.+Petzerauthor=J.+P.+Petzer&title=The+Adenosine+Receptor+Affinities+and+Monoamine+Oxidase+B+Inhibitory+Properties+of+Sulfanylphthalimide+Analogues&doi=10.1016%2Fj.bioorg.2015.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The adenosine receptor affinities and monoamine oxidase B inhibitory properties of sulfanylphthalimide analogues</span></div><div class="casAuthors">Van der Walt, Mietha M.; Terre'Blanche, Gisella; Petzer, Anel; Petzer, Jacobus P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117-123</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Based on a report that sulfanylphthalimides are highly potent monoamine oxidase (MAO) B selective inhibitors, the present study examines the adenosine receptor affinities and MAO-B inhibitory properties of a series of 4- and 5-sulfanylphthalimide analogs.  Since adenosine antagonists (A1 and A2A subtypes) and MAO-B inhibitors are considered agents for the therapy of neurodegenerative disorders such as Parkinson's disease and Alzheimer's disease, dual-target-directed drugs that antagonize adenosine receptors and inhibit MAO-B may have enhanced therapeutic value.  The results document that the sulfanylphthalimide analogs are selective for the adenosine A1 receptor over the A2A receptor subtype, with a no. of compds. also possessing MAO-B inhibitory properties.  Among the compds. evaluated, 5-[(4-methoxybenzyl)sulfanyl]phthalimide was found to possess the highest binding affinity to adenosine A1 receptors with a Ki value of 0.369 μM.  This compd. is reported to also inhibit MAO-B with an IC50 value of 0.020 μM.  Such dual-target-directed compds. may act synergistic in the treatment of Parkinson's disease: antagonism of the A1 receptor may facilitate dopamine release, while MAO-B inhibition may reduce dopamine metab.  Addnl., dual-target-directed compds. may find therapeutic value in Alzheimer's disease: antagonism of the A1 receptor may be beneficial in the treatment of cognitive dysfunction, while MAO-B inhibition may exhibit neuroprotective properties.  In neurol. diseases, such as Parkinson's disease and Alzheimer's disease, dual-target-directed drugs are expected to be advantageous over single-target treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQQY_dzBBNoLVg90H21EOLACvtfcHk0ljip3PaOtXT3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1aqsr0%253D&md5=20db87d9be6476d2eb5ab492f7fae1fb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2015.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2015.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BWalt%26aufirst%3DM.%2BM.%26aulast%3DTerre%25E2%2580%2599Blanche%26aufirst%3DG.%26aulast%3DPetzer%26aufirst%3DA.%26aulast%3DPetzer%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520Adenosine%2520Receptor%2520Affinities%2520and%2520Monoamine%2520Oxidase%2520B%2520Inhibitory%2520Properties%2520of%2520Sulfanylphthalimide%2520Analogues%26jtitle%3DBioorg.%2520Chem.%26date%3D2015%26volume%3D59%26spage%3D117%26epage%3D123%26doi%3D10.1016%2Fj.bioorg.2015.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jörg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Pharmacological Evaluation of Dual Acting Ligands Targeting the Adenosine A<sub>2A</sub> and Dopamine D<sub>2</sub> Receptors for the Potential Treatment of Parkinson’s Disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">718</span>– <span class="NLM_lpage">738</span>, <span class="refDoi"> DOI: 10.1021/jm501254d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501254d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGmsLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=718-738&author=M.+J%C3%B6rgauthor=L.+T.+Mayauthor=F.+S.+Makauthor=K.+C.+K.+Leeauthor=N.+D.+Millerauthor=P.+J.+Scammellsauthor=B.+Capuano&title=Synthesis+and+Pharmacological+Evaluation+of+Dual+Acting+Ligands+Targeting+the+Adenosine+A2A+and+Dopamine+D2+Receptors+for+the+Potential+Treatment+of+Parkinson%E2%80%99s+Disease&doi=10.1021%2Fjm501254d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Pharmacological Evaluation of Dual Acting Ligands Targeting the Adenosine A2A and Dopamine D2 Receptors for the Potential Treatment of Parkinson's Disease</span></div><div class="casAuthors">Jorg, Manuela; May, Lauren T.; Mak, Frankie S.; Lee, Kiew Ching K.; Miller, Neil D.; Scammells, Peter J.; Capuano, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">718-738</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A relatively new strategy in drug discovery is the development of dual acting ligands.  These mols. are potentially able to interact at two orthosteric binding sites of a heterodimer simultaneously, possibly resulting in enhanced subtype selectivity, higher affinity, enhanced or modified physiol. response, and reduced reliance on multiple drug administration regimens.  In this study, we have successfully synthesized a series of classical heterobivalent ligands as well as a series of more integrated and "drug-like" dual acting mols., incorporating ropinirole as a dopamine D2 receptor agonist and ZM 241385 as an adenosine A2A receptor antagonist.  The best compds. of our series maintained the potency of the original pharmacophores at both receptors (adenosine A2A and dopamine D2).  In addn., the integrated dual acting ligands also showed promising results in preliminary blood-brain barrier permeability tests, whereas the classical heterobivalent ligands are potentially more suited as pharmacol. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqChLZoS1UseLVg90H21EOLACvtfcHk0ljip3PaOtXT3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGmsLnL&md5=65ab4bdbcf04d6a2be666e48acc52a62</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm501254d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501254d%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25B6rg%26aufirst%3DM.%26aulast%3DMay%26aufirst%3DL.%2BT.%26aulast%3DMak%26aufirst%3DF.%2BS.%26aulast%3DLee%26aufirst%3DK.%2BC.%2BK.%26aulast%3DMiller%26aufirst%3DN.%2BD.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DCapuano%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520Evaluation%2520of%2520Dual%2520Acting%2520Ligands%2520Targeting%2520the%2520Adenosine%2520A2A%2520and%2520Dopamine%2520D2%2520Receptors%2520for%2520the%2520Potential%2520Treatment%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D718%26epage%3D738%26doi%3D10.1021%2Fjm501254d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Simone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarozzo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandiera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span> <span> </span><span class="NLM_article-title">Applying a Multitarget Rational Drug Design Strategy: The First Set of Modulators with Potent and Balanced Activity toward Dopamine D<sub>3</sub> Receptor and Fatty Acid Amide Hydrolase</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">4904</span>– <span class="NLM_lpage">4907</span>, <span class="refDoi"> DOI: 10.1039/C4CC00967C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1039%2FC4CC00967C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24691497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlersb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=4904-4907&author=A.+De%0ASimoneauthor=G.+F.+Rudaauthor=C.+Albaniauthor=G.+Tarozzoauthor=T.+Bandieraauthor=D.+Piomelliauthor=A.+Cavalliauthor=G.+Bottegoni&title=Applying+a+Multitarget+Rational+Drug+Design+Strategy%3A+The+First+Set+of+Modulators+with+Potent+and+Balanced+Activity+toward+Dopamine+D3+Receptor+and+Fatty+Acid+Amide+Hydrolase&doi=10.1039%2FC4CC00967C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Applying a multitarget rational drug design strategy: the first set of modulators with potent and balanced activity toward dopamine D3 receptor and fatty acid amide hydrolase</span></div><div class="casAuthors">De Simone, Alessio; Ruda, Gian Filippo; Albani, Clara; Tarozzo, Glauco; Bandiera, Tiziano; Piomelli, Daniele; Cavalli, Andrea; Bottegoni, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">4904-4907</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Combining computer-assisted drug design and synthetic efforts, we generated compds. with potent and balanced activities toward both D3 dopamine receptor and fatty acid amide hydrolase (FAAH) enzyme.  By concurrently modulating these targets, our compds. hold great potential toward exerting a disease-modifying effect on nicotine addiction and other forms of compulsive behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnUQcdwExHDLVg90H21EOLACvtfcHk0ljip3PaOtXT3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlersb0%253D&md5=24c31533c27a2c6e356af93edeb5156a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2FC4CC00967C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC00967C%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BSimone%26aufirst%3DA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DAlbani%26aufirst%3DC.%26aulast%3DTarozzo%26aufirst%3DG.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DApplying%2520a%2520Multitarget%2520Rational%2520Drug%2520Design%2520Strategy%253A%2520The%2520First%2520Set%2520of%2520Modulators%2520with%2520Potent%2520and%2520Balanced%2520Activity%2520toward%2520Dopamine%2520D3%2520Receptor%2520and%2520Fatty%2520Acid%2520Amide%2520Hydrolase%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D4904%26epage%3D4907%26doi%3D10.1039%2FC4CC00967C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lorber, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Hierarchical Docking of Databases of Multiple Ligand Conformations</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.2174/1568026054637683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.2174%2F1568026054637683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=16101414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=739-749&author=D.+M.+Lorberauthor=B.+K.+Shoichet&title=Hierarchical+Docking+of+Databases+of+Multiple+Ligand+Conformations&doi=10.2174%2F1568026054637683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical docking of databases of multiple ligand conformations</span></div><div class="casAuthors">Lorber, David M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">739-749</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Ligand flexibility is an important problem in mol. docking and virtual screening.  To address this challenge, we investigate a hierarchical pre-organization of multiple conformations of small mols.  Such organization of pre-calcd. conformations removes the exploration of ligand conformational space from the docking calcn. and allows for concise representation of what can be thousands of conformations.  The hierarchy also recognizes and prunes incompatible conformations early in the calcn., eliminating redundant calcns. of fit.  We investigate the method by docking the MDL Drug Data Report (MDDR), an annotated database of 100,000 mols., into apo and holo forms of 7 unrelated targets.  This annotated database allows us to track the ranking of tens to hundreds of annotated ligands in each of the docking systems.  The binding sites and database are prepd. in an automated fashion in an attempt to remove some human bias from the calcns.  Many thousands of explicit and implicit ligand conformations may be docked in calcns. not much longer than required for single conformer docking.  As long as internal energies are not considered, recombination with the hierarchy is additive as the no. of degrees of freedom is increased.  Mols. with even millions of conformations can be docked in a few minutes on a single desktop computer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ_iMnNjm9LVg90H21EOLACvtfcHk0ljip3PaOtXT3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D&md5=25c833b5608e5faedc1af3ad38fdfa06</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.2174%2F1568026054637683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054637683%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DHierarchical%2520Docking%2520of%2520Databases%2520of%2520Multiple%2520Ligand%2520Conformations%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D739%26epage%3D749%26doi%3D10.2174%2F1568026054637683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alagona, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Profeta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, P.</span></span> <span> </span><span class="NLM_article-title">A New Force Field for Molecular Mechanical Simulation of Nucleic Acids and Proteins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">765</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1021/ja00315a051</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00315a051" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaL2cXmsVCgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1984&pages=765-784&author=S.+J.+Weinerauthor=P.+A.+Kollmanauthor=D.+A.+Caseauthor=U.+C.+Singhauthor=C.+Ghioauthor=G.+Alagonaauthor=S.+Profetaauthor=P.+Weiner&title=A+New+Force+Field+for+Molecular+Mechanical+Simulation+of+Nucleic+Acids+and+Proteins&doi=10.1021%2Fja00315a051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A new force field for molecular mechanical simulation of nucleic acids and proteins</span></div><div class="casAuthors">Weiner, Scott J.; Kollman, Peter A.; Case, David A.; Singh, U. Chandra; Ghio, Caterina; Alagona, Guliano; Profeta, Salvatore, Jr.; Weiner, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">765-84</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A force field was developed for simulation of nucleic acids and proteins.  The approach began by obtaining equil. bond lengths and angles from microwave, neutron diffraction, and prior mol. mech. calcns., torsional consts. from microwave, NMR, and mol. mech. studies, nonbonded parameters from crystal packing calcns., and at. charges from the fit of a partial charge model to electrostatic potentials calcd. by ab initio quantum mech. theory.  The parameters then were refined with mol. mech. studies on the structures and energies of model compds.  For nucleic acids, Me Et ether, THF, deoxyadenosine, di-Me phosphate, 9-methylguanine-1-methylcytosine H-bonded complex, 9-methyladenine-1-methylthymine H-bonded complex, and 1,3-dimethyluracil base-stacked dimer were studied.  Bond, angle, torsional, nonbonded, and H-bond parameters were varied to optimize the agreement between calcd. and exptl. values for sugar pucker energies and structures, vibrational frequencies of di-Me phosphate and THF, and energies for base pairing and base stacking.  For proteins, Φ,ψ maps of glycyl and alanyl dipeptides, H-bonding interactions involving the various protein polar groups, and energy refinement calcns. on insulin were considered.  Unlike the models for H bonding involving N and O electron donors, an adequate description of S-H bonding required explicit inclusion of lone pairs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp26DDzh1ctfrVg90H21EOLACvtfcHk0li2guXzlXWvYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmsVCgug%253D%253D&md5=c4aa27feec7e2c4d34001be10a6cb8e9</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja00315a051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00315a051%26sid%3Dliteratum%253Aachs%26aulast%3DWeiner%26aufirst%3DS.%2BJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26aulast%3DGhio%26aufirst%3DC.%26aulast%3DAlagona%26aufirst%3DG.%26aulast%3DProfeta%26aufirst%3DS.%26aulast%3DWeiner%26aufirst%3DP.%26atitle%3DA%2520New%2520Force%2520Field%2520for%2520Molecular%2520Mechanical%2520Simulation%2520of%2520Nucleic%2520Acids%2520and%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1984%26volume%3D106%26spage%3D765%26epage%3D784%26doi%3D10.1021%2Fja00315a051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and Testing of a General Amber Force Field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+M.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+Testing+of+a+General+Amber+Force+Field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0li2guXzlXWvYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520Testing%2520of%2520a%2520General%2520Amber%2520Force%2520Field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span> <span> </span><span class="NLM_article-title">Automated Docking with Grid-based Energy Evaluation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1002/jcc.540130412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1002%2Fjcc.540130412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=505-524&author=E.+C.+Mengauthor=B.+K.+Shoichetauthor=I.+D.+Kuntz&title=Automated+Docking+with+Grid-based+Energy+Evaluation&doi=10.1002%2Fjcc.540130412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking with grid-based energy evaluation</span></div><div class="casAuthors">Meng, Elaine C.; Shoichet, Brian K.; Kuntz, Irwin D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-24</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The ability to generate feasible binding orientations of a small mol. within a site of known structure is important for ligand design.  The authors present a method that combines a rapid, geometric docking algorithm with the evaluation of mol. mechanics interaction energies.  The computational costs of evaluation are minimal because the authors precalc. the receptor-dependent terms in the potential function at points on a three-dimensional grid.  In four test cases where the components of crystallog. detd. complexes are redocked, the "force field" score correctly identifies the family of orientations closest to the exptl. binding geometry.  Scoring functions that consider only steric factors or only electrostatic factors are less successful.  The force field function will play an important role in efforts to search databases for potential lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKAw7Zj_QFSbVg90H21EOLACvtfcHk0li2guXzlXWvYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D&md5=f472fffa0c9a61652b4c20e4dbbba69e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130412%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DAutomated%2520Docking%2520with%2520Grid-based%2520Energy%2520Evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D505%26epage%3D524%26doi%3D10.1002%2Fjcc.540130412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">A Rapid Finite Difference Algorithm, Utilizing Successive Over-relaxation to Solve the Poisson–Boltzmann Equation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1002/jcc.540120405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1002%2Fjcc.540120405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaK3MXisFCgtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1991&pages=435-445&author=A.+Nichollsauthor=B.+Honig&title=A+Rapid+Finite+Difference+Algorithm%2C+Utilizing+Successive+Over-relaxation+to+Solve+the+Poisson%E2%80%93Boltzmann+Equation&doi=10.1002%2Fjcc.540120405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid finite difference algorithm, utilizing successive over-relaxation to solve the Poisson-Boltzmann equation</span></div><div class="casAuthors">Nicholls, Anthony; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-45</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient algorithm is presented for the numerical soln. of the Poisson-boltzmann equation by the finite difference method of successive over-relaxation.  Improvements include the rapid estn. of the optimum relaxation parameter, redn. in no. of operations per iteration, and vector-oriented array mapping.  The algorithm has been incorporated into the electrostatic program DelPhi, reducing the required computing time by between one and two orders of magnitude.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLJ8dypOKbsrVg90H21EOLACvtfcHk0lhPYw4vljADsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisFCgtr0%253D&md5=28ac444a000ad2f801ceea2237f4bc2e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540120405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540120405%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DA%2520Rapid%2520Finite%2520Difference%2520Algorithm%252C%2520Utilizing%2520Successive%2520Over-relaxation%2520to%2520Solve%2520the%2520Poisson%25E2%2580%2593Boltzmann%2520Equation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1991%26volume%3D12%26spage%3D435%26epage%3D445%26doi%3D10.1002%2Fjcc.540120405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS: A Web Server for Detection of Structurally Similar Protein Binding Sites</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W436</span>– <span class="NLM_lpage">W440</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fnar%2Fgkq479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20504855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W436-W440&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS%3A+A+Web+Server+for+Detection+of+Structurally+Similar+Protein+Binding+Sites&doi=10.1093%2Fnar%2Fgkq479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS: a web server for detection of structurally similar protein binding sites</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W436-W440</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A web server, ProBiS, freely available at http://probis.cmm.ki.si, is presented.  This provides access to the program ProBiS (Protein Binding Sites), which detects protein binding sites based on local structural alignments.  Detailed instructions and user guidelines for use of ProBiS are available at the server under HELP' and selected examples are provided under EXAMPLES'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_Hf9BhPvSrrVg90H21EOLACvtfcHk0lhPYw4vljADsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWjsrc%253D&md5=6b53ed57c159be18c05a5e2f72995983</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq479%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%253A%2520A%2520Web%2520Server%2520for%2520Detection%2520of%2520Structurally%2520Similar%2520Protein%2520Binding%2520Sites%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW436%26epage%3DW440%26doi%3D10.1093%2Fnar%2Fgkq479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 23 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jingxiao Bao, Xiao He, <span class="NLM_string-name hlFld-ContribAuthor">John Z. H. Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">DeepBSP—a Machine Learning Method for Accurate Prediction of Protein–Ligand Docking Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2231-2240. <a href="https://doi.org/10.1021/acs.jcim.1c00334" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00334</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00334%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDeepBSP%2525E2%252580%252594a%252BMachine%252BLearning%252BMethod%252Bfor%252BAccurate%252BPrediction%252Bof%252BProtein%2525E2%252580%252593Ligand%252BDocking%252BStructures%26aulast%3DBao%26aufirst%3DJingxiao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D20032021%26date%3D12052021%26volume%3D61%26issue%3D5%26spage%3D2231%26epage%3D2240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Haiying Dong, Jing Zhang, Hua Rong, Xiaojie Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Miaoxian Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Paeoniflorin and Plycyrrhetinic Acid Synergistically Alleviate MPP+/MPTP-Induced Oxidative Stress through Nrf2-Dependent Glutathione Biosynthesis Mechanisms. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (7)
                                     , 1100-1111. <a href="https://doi.org/10.1021/acschemneuro.0c00544" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00544</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00544%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DPaeoniflorin%252Band%252BPlycyrrhetinic%252BAcid%252BSynergistically%252BAlleviate%252BMPP%25252B%25252FMPTP-Induced%252BOxidative%252BStress%252Bthrough%252BNrf2-Dependent%252BGlutathione%252BBiosynthesis%252BMechanisms%26aulast%3DDong%26aufirst%3DHaiying%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D19082020%26date%3D09032021%26date%3D16032021%26volume%3D12%26issue%3D7%26spage%3D1100%26epage%3D1111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pricila  Pflüger</span>, <span class="hlFld-ContribAuthor ">Patrícia  Pereira</span>, <span class="hlFld-ContribAuthor ">María I.  Loza</span>, <span class="hlFld-ContribAuthor ">José  Brea</span>, <span class="hlFld-ContribAuthor ">Dolores  Viña</span>, <span class="hlFld-ContribAuthor ">Amit  Kumar</span>, <span class="hlFld-ContribAuthor ">José A.  Fontenla</span>. </span><span class="cited-content_cbyCitation_article-title">Gamma-decanolactone: Preliminary evaluation as potential antiparkinsonian drug. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmacology</span><span> <strong>2021,</strong> <em>906 </em>, 174276. <a href="https://doi.org/10.1016/j.ejphar.2021.174276" title="DOI URL">https://doi.org/10.1016/j.ejphar.2021.174276</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejphar.2021.174276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejphar.2021.174276%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmacology%26atitle%3DGamma-decanolactone%25253A%252BPreliminary%252Bevaluation%252Bas%252Bpotential%252Bantiparkinsonian%252Bdrug%26aulast%3DPfl%25C3%25BCger%26aufirst%3DPricila%26date%3D2021%26volume%3D906%26spage%3D174276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria João  Matos</span>. </span><span class="cited-content_cbyCitation_article-title">Multitarget therapeutic approaches for Alzheimer's and Parkinson's diseases: an opportunity or an illusion?. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (15)
                                     , 1301-1309. <a href="https://doi.org/10.4155/fmc-2021-0119" title="DOI URL">https://doi.org/10.4155/fmc-2021-0119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0119%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DMultitarget%252Btherapeutic%252Bapproaches%252Bfor%252BAlzheimer%252527s%252Band%252BParkinson%252527s%252Bdiseases%25253A%252Ban%252Bopportunity%252Bor%252Ban%252Billusion%25253F%26aulast%3DMatos%26aufirst%3DMaria%2BJo%25C3%25A3o%26date%3D2021%26volume%3D13%26issue%3D15%26spage%3D1301%26epage%3D1309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ioannis  Mantas</span>, <span class="hlFld-ContribAuthor ">Marcus  Saarinen</span>, <span class="hlFld-ContribAuthor ">Zhi-Qing David  Xu</span>, <span class="hlFld-ContribAuthor ">Per  Svenningsson</span>. </span><span class="cited-content_cbyCitation_article-title">Update on GPCR-based targets for the development of novel antidepressants. </span><span class="cited-content_cbyCitation_journal-name">Molecular Psychiatry</span><span> <strong>2021,</strong> <em>392 </em><a href="https://doi.org/10.1038/s41380-021-01040-1" title="DOI URL">https://doi.org/10.1038/s41380-021-01040-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41380-021-01040-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41380-021-01040-1%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Psychiatry%26atitle%3DUpdate%252Bon%252BGPCR-based%252Btargets%252Bfor%252Bthe%252Bdevelopment%252Bof%252Bnovel%252Bantidepressants%26aulast%3DMantas%26aufirst%3DIoannis%26date%3D2021%26date%3D2021%26volume%3D392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Denis  Schmidt</span>, <span class="hlFld-ContribAuthor ">Magdalena M.  Scharf</span>, <span class="hlFld-ContribAuthor ">Dominique  Sydow</span>, <span class="hlFld-ContribAuthor ">Eva  Aßmann</span>, <span class="hlFld-ContribAuthor ">Maria  Martí-Solano</span>, <span class="hlFld-ContribAuthor ">Marina  Keul</span>, <span class="hlFld-ContribAuthor ">Andrea  Volkamer</span>, <span class="hlFld-ContribAuthor ">Peter  Kolb</span>. </span><span class="cited-content_cbyCitation_article-title">Analyzing Kinase Similarity in Small Molecule and Protein Structural Space to Explore the Limits of Multi-Target Screening. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (3)
                                     , 629. <a href="https://doi.org/10.3390/molecules26030629" title="DOI URL">https://doi.org/10.3390/molecules26030629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26030629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26030629%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DAnalyzing%252BKinase%252BSimilarity%252Bin%252BSmall%252BMolecule%252Band%252BProtein%252BStructural%252BSpace%252Bto%252BExplore%252Bthe%252BLimits%252Bof%252BMulti-Target%252BScreening%26aulast%3DSchmidt%26aufirst%3DDenis%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D3%26spage%3D629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bassam M.  Ayoub</span>, <span class="hlFld-ContribAuthor ">Haidy E.  Michel</span>, <span class="hlFld-ContribAuthor ">Shereen  Mowaka</span>, <span class="hlFld-ContribAuthor ">Moataz S.  Hendy</span>, <span class="hlFld-ContribAuthor ">Mariam M.  Tadros</span>. </span><span class="cited-content_cbyCitation_article-title">Repurposing of Omarigliptin as a Neuroprotective Agent Based on Docking with A2A Adenosine and AChE Receptors, Brain GLP-1 Response and Its Brain/Plasma Concentration Ratio after 28 Days Multiple Doses in Rats Using LC-MS/MS. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 889. <a href="https://doi.org/10.3390/molecules26040889" title="DOI URL">https://doi.org/10.3390/molecules26040889</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26040889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26040889%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRepurposing%252Bof%252BOmarigliptin%252Bas%252Ba%252BNeuroprotective%252BAgent%252BBased%252Bon%252BDocking%252Bwith%252BA2A%252BAdenosine%252Band%252BAChE%252BReceptors%25252C%252BBrain%252BGLP-1%252BResponse%252Band%252BIts%252BBrain%25252FPlasma%252BConcentration%252BRatio%252Bafter%252B28%252BDays%252BMultiple%252BDoses%252Bin%252BRats%252BUsing%252BLC-MS%25252FMS%26aulast%3DAyoub%26aufirst%3DBassam%2BM.%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bertil B.  Fredholm</span>, <span class="hlFld-ContribAuthor ">Per  Svenningsson</span>. </span><span class="cited-content_cbyCitation_article-title">Why target brain adenosine receptors? A historical perspective. </span><span class="cited-content_cbyCitation_journal-name">Parkinsonism & Related Disorders</span><span> <strong>2020,</strong> <em>80 </em>, S3-S6. <a href="https://doi.org/10.1016/j.parkreldis.2020.09.027" title="DOI URL">https://doi.org/10.1016/j.parkreldis.2020.09.027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.parkreldis.2020.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.parkreldis.2020.09.027%26sid%3Dliteratum%253Aachs%26jtitle%3DParkinsonism%2520%2526%2520Related%2520Disorders%26atitle%3DWhy%252Btarget%252Bbrain%252Badenosine%252Breceptors%25253F%252BA%252Bhistorical%252Bperspective%26aulast%3DFredholm%26aufirst%3DBertil%2BB.%26date%3D2020%26volume%3D80%26spage%3DS3%26epage%3DS6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongguang  Ma</span>, <span class="hlFld-ContribAuthor ">Boshi  Huang</span>, <span class="hlFld-ContribAuthor ">Yan  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in multitarget-directed ligands targeting G-protein-coupled receptors. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1682-1692. <a href="https://doi.org/10.1016/j.drudis.2020.07.004" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.07.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.07.004%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DRecent%252Badvances%252Bin%252Bmultitarget-directed%252Bligands%252Btargeting%252BG-protein-coupled%252Breceptors%26aulast%3DMa%26aufirst%3DHongguang%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1682%26epage%3D1692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soha  Parsafar</span>, <span class="hlFld-ContribAuthor ">Zahra  Nayeri</span>, <span class="hlFld-ContribAuthor ">Farhang  Aliakbari</span>, <span class="hlFld-ContribAuthor ">Farshad  Shahi</span>, <span class="hlFld-ContribAuthor ">Mehdi  Mohammadi</span>, <span class="hlFld-ContribAuthor ">Dina  Morshedi</span>. </span><span class="cited-content_cbyCitation_article-title">Multiple neuroprotective features of Scutellaria pinnatifida–derived small molecule. </span><span class="cited-content_cbyCitation_journal-name">Heliyon</span><span> <strong>2020,</strong> <em>6 </em>
                                    (8)
                                     , e04737. <a href="https://doi.org/10.1016/j.heliyon.2020.e04737" title="DOI URL">https://doi.org/10.1016/j.heliyon.2020.e04737</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.heliyon.2020.e04737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.heliyon.2020.e04737%26sid%3Dliteratum%253Aachs%26jtitle%3DHeliyon%26atitle%3DMultiple%252Bneuroprotective%252Bfeatures%252Bof%252BScutellaria%252Bpinnatifida%2525E2%252580%252593derived%252Bsmall%252Bmolecule%26aulast%3DParsafar%26aufirst%3DSoha%26date%3D2020%26volume%3D6%26issue%3D8%26spage%3De04737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Can  Chen</span>, <span class="hlFld-ContribAuthor ">Changpeng  Zou</span>, <span class="hlFld-ContribAuthor ">Lin  Li</span>, <span class="hlFld-ContribAuthor ">Haidong  Yu</span>, <span class="hlFld-ContribAuthor ">Jixin  Zhu</span>, <span class="hlFld-ContribAuthor ">Jinhua  Liu</span>, <span class="hlFld-ContribAuthor ">Wei  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Blue and green emission-transformed fluorescent copolymer: Specific detection of levodopa of anti-Parkinson drug in human serum. </span><span class="cited-content_cbyCitation_journal-name">Talanta</span><span> <strong>2020,</strong> <em>214 </em>, 120817. <a href="https://doi.org/10.1016/j.talanta.2020.120817" title="DOI URL">https://doi.org/10.1016/j.talanta.2020.120817</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.talanta.2020.120817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.talanta.2020.120817%26sid%3Dliteratum%253Aachs%26jtitle%3DTalanta%26atitle%3DBlue%252Band%252Bgreen%252Bemission-transformed%252Bfluorescent%252Bcopolymer%25253A%252BSpecific%252Bdetection%252Bof%252Blevodopa%252Bof%252Banti-Parkinson%252Bdrug%252Bin%252Bhuman%252Bserum%26aulast%3DChen%26aufirst%3DCan%26date%3D2020%26volume%3D214%26spage%3D120817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>, <span class="hlFld-ContribAuthor ">Zhan‐Guo  Gao</span>, <span class="hlFld-ContribAuthor ">Pierre  Matricon</span>, <span class="hlFld-ContribAuthor ">Matthew T.  Eddy</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Adenosine A
              2A
              receptor antagonists: from caffeine to selective non‐xanthines. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2020,</strong> <em>176 </em><a href="https://doi.org/10.1111/bph.15103" title="DOI URL">https://doi.org/10.1111/bph.15103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bph.15103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbph.15103%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DAdenosine%252BA%252B2A%252Breceptor%252Bantagonists%25253A%252Bfrom%252Bcaffeine%252Bto%252Bselective%252Bnon%2525E2%252580%252590xanthines%26aulast%3DJacobson%26aufirst%3DKenneth%2BA.%26date%3D2020%26date%3D2020%26volume%3D176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamil J.  Kuder</span>, <span class="hlFld-ContribAuthor ">Michał  Załuski</span>, <span class="hlFld-ContribAuthor ">Jakub  Schabikowski</span>, <span class="hlFld-ContribAuthor ">Gniewomir  Latacz</span>, <span class="hlFld-ContribAuthor ">Agnieszka  Olejarz‐Maciej</span>, <span class="hlFld-ContribAuthor ">Piotr  Jaśko</span>, <span class="hlFld-ContribAuthor ">Agata  Doroz‐Płonka</span>, <span class="hlFld-ContribAuthor ">Andreas  Brockmann</span>, <span class="hlFld-ContribAuthor ">Christa E.  Müller</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Kieć‐Kononowicz</span>. </span><span class="cited-content_cbyCitation_article-title">Novel, Dual Target‐Directed Annelated Xanthine Derivatives Acting on Adenosine Receptors and Monoamine Oxidase B. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (9)
                                     , 772-786. <a href="https://doi.org/10.1002/cmdc.201900717" title="DOI URL">https://doi.org/10.1002/cmdc.201900717</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900717%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DNovel%25252C%252BDual%252BTarget%2525E2%252580%252590Directed%252BAnnelated%252BXanthine%252BDerivatives%252BActing%252Bon%252BAdenosine%252BReceptors%252Band%252BMonoamine%252BOxidase%252BB%26aulast%3DKuder%26aufirst%3DKamil%2BJ.%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D9%26spage%3D772%26epage%3D786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Lückmann</span>, <span class="hlFld-ContribAuthor ">Mette  Trauelsen</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Frimurer</span>, <span class="hlFld-ContribAuthor ">Thue W.  Schwartz</span>. </span><span class="cited-content_cbyCitation_article-title">Structural basis for GPCR signaling by small polar versus large lipid metabolites—discovery of non-metabolite ligands. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Cell Biology</span><span> <strong>2020,</strong> <em>63 </em>, 38-48. <a href="https://doi.org/10.1016/j.ceb.2019.12.005" title="DOI URL">https://doi.org/10.1016/j.ceb.2019.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ceb.2019.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ceb.2019.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Cell%2520Biology%26atitle%3DStructural%252Bbasis%252Bfor%252BGPCR%252Bsignaling%252Bby%252Bsmall%252Bpolar%252Bversus%252Blarge%2525C2%2525A0lipid%252Bmetabolites%2525E2%252580%252594discovery%252Bof%252Bnon-metabolite%252Bligands%26aulast%3DL%25C3%25BCckmann%26aufirst%3DMichael%26date%3D2020%26volume%3D63%26spage%3D38%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan  Tao</span>, <span class="hlFld-ContribAuthor ">Yukun  Huang</span>, <span class="hlFld-ContribAuthor ">Chong  Wang</span>, <span class="hlFld-ContribAuthor ">Fang  Chen</span>, <span class="hlFld-ContribAuthor ">Lingling  Yang</span>, <span class="hlFld-ContribAuthor ">Li  Ling</span>, <span class="hlFld-ContribAuthor ">Zhenming  Che</span>, <span class="hlFld-ContribAuthor ">Xianggui  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in molecular docking technology applied in food science: a review. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Food Science & Technology</span><span> <strong>2020,</strong> <em>55 </em>
                                    (1)
                                     , 33-45. <a href="https://doi.org/10.1111/ijfs.14325" title="DOI URL">https://doi.org/10.1111/ijfs.14325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/ijfs.14325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fijfs.14325%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Food%2520Science%2520%2526%2520Technology%26atitle%3DRecent%252Bdevelopments%252Bin%252Bmolecular%252Bdocking%252Btechnology%252Bapplied%252Bin%252Bfood%252Bscience%25253A%252Ba%252Breview%26aulast%3DTao%26aufirst%3DXuan%26date%3D2020%26date%3D2019%26volume%3D55%26issue%3D1%26spage%3D33%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Omar H.A.  Al-Attraqchi</span>, <span class="hlFld-ContribAuthor ">Mahesh  Attimarad</span>, <span class="hlFld-ContribAuthor ">Katharigatta N.  Venugopala</span>, <span class="hlFld-ContribAuthor ">Anroop  Nair</span>, <span class="hlFld-ContribAuthor ">Noor H.A.  Al-Attraqchi</span>. </span><span class="cited-content_cbyCitation_article-title">Adenosine A2A Receptor as a Potential Drug Target - Current Status and Future Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Current Pharmaceutical Design</span><span> <strong>2019,</strong> <em>25 </em>
                                    (25)
                                     , 2716-2740. <a href="https://doi.org/10.2174/1381612825666190716113444" title="DOI URL">https://doi.org/10.2174/1381612825666190716113444</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1381612825666190716113444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1381612825666190716113444%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Pharmaceutical%2520Design%26atitle%3DAdenosine%252BA2A%252BReceptor%252Bas%252Ba%252BPotential%252BDrug%252BTarget%252B-%252BCurrent%252BStatus%252Band%252BFuture%252BPerspectives%26aulast%3DAl-Attraqchi%26aufirst%3DOmar%2BH.A.%26date%3D2019%26volume%3D25%26issue%3D25%26spage%3D2716%26epage%3D2740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Widya  Dwi Aryati</span>, <span class="hlFld-ContribAuthor ">Nabilah  Nurtika Salamah</span>, <span class="hlFld-ContribAuthor ">Rezi  Riadhi Syahdi</span>, <span class="hlFld-ContribAuthor ">Arry  Yanuar</span>. </span><span class="cited-content_cbyCitation_article-title">The Role and Development of the Antagonist of Adenosine A
              2A
              in Parkinson’s Disease. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,<a href="https://doi.org/10.5772/intechopen.84272" title="DOI URL">https://doi.org/10.5772/intechopen.84272</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5772/intechopen.84272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5772%2Fintechopen.84272%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BRole%252Band%252BDevelopment%252Bof%252Bthe%252BAntagonist%252Bof%252BAdenosine%252BA%252B2A%252Bin%252BParkinson%2525E2%252580%252599s%252BDisease%26aulast%3DDwi%2BAryati%26aufirst%3DWidya%26date%3D2019%26date%3D2019%26pub%3DIntechOpen%26atitle%3DNeuroprotection%26aulast%3DChuen-Chung%2BChang%26aufirst%3DRaymond%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivia  Slater</span>, <span class="hlFld-ContribAuthor ">Maria  Kontoyianni</span>. </span><span class="cited-content_cbyCitation_article-title">The compromise of virtual screening and its impact on drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (7)
                                     , 619-637. <a href="https://doi.org/10.1080/17460441.2019.1604677" title="DOI URL">https://doi.org/10.1080/17460441.2019.1604677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1604677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1604677%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bcompromise%252Bof%252Bvirtual%252Bscreening%252Band%252Bits%252Bimpact%252Bon%252Bdrug%252Bdiscovery%26aulast%3DSlater%26aufirst%3DOlivia%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D7%26spage%3D619%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qunlin  Li</span>, <span class="hlFld-ContribAuthor ">Cuihua  Zhang</span>, <span class="hlFld-ContribAuthor ">Yujie  Ren</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular modeling technology studies of novel pyrazoylethylbenzamide derivatives as selective orexin receptor 1 antagonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of the Taiwan Institute of Chemical Engineers</span><span> <strong>2019,</strong> <em>100 </em>, 1-17. <a href="https://doi.org/10.1016/j.jtice.2019.03.018" title="DOI URL">https://doi.org/10.1016/j.jtice.2019.03.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jtice.2019.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jtice.2019.03.018%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520Taiwan%2520Institute%2520of%2520Chemical%2520Engineers%26atitle%3DMolecular%252Bmodeling%252Btechnology%252Bstudies%252Bof%252Bnovel%252Bpyrazoylethylbenzamide%252Bderivatives%252Bas%252Bselective%252Borexin%252Breceptor%252B1%252Bantagonists%26aulast%3DLi%26aufirst%3DQunlin%26date%3D2019%26volume%3D100%26spage%3D1%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Su Hui  Seong</span>, <span class="hlFld-ContribAuthor ">Pradeep  Paudel</span>, <span class="hlFld-ContribAuthor ">Jeong-Wook  Choi</span>, <span class="hlFld-ContribAuthor ">Dong Hyun  Ahn</span>, <span class="hlFld-ContribAuthor ">Taek-Jeong  Nam</span>, <span class="hlFld-ContribAuthor ">Hyun Ah  Jung</span>, <span class="hlFld-ContribAuthor ">Jae Sue  Choi</span>. </span><span class="cited-content_cbyCitation_article-title">Probing Multi-Target Action of Phlorotannins as New Monoamine Oxidase Inhibitors and Dopaminergic Receptor Modulators with the Potential for Treatment of Neuronal Disorders. </span><span class="cited-content_cbyCitation_journal-name">Marine Drugs</span><span> <strong>2019,</strong> <em>17 </em>
                                    (6)
                                     , 377. <a href="https://doi.org/10.3390/md17060377" title="DOI URL">https://doi.org/10.3390/md17060377</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/md17060377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmd17060377%26sid%3Dliteratum%253Aachs%26jtitle%3DMarine%2520Drugs%26atitle%3DProbing%252BMulti-Target%252BAction%252Bof%252BPhlorotannins%252Bas%252BNew%252BMonoamine%252BOxidase%252BInhibitors%252Band%252BDopaminergic%252BReceptor%252BModulators%252Bwith%252Bthe%252BPotential%252Bfor%252BTreatment%252Bof%252BNeuronal%252BDisorders%26aulast%3DSeong%26aufirst%3DSu%2BHui%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D6%26spage%3D377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed  Benchekroun</span>, <span class="hlFld-ContribAuthor ">Samuele  Maramai</span>. </span><span class="cited-content_cbyCitation_article-title">Multitarget-directed ligands for neurodegenerative diseases: real opportunity or blurry mirage?. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2019,</strong> <em>11 </em>
                                    (4)
                                     , 261-263. <a href="https://doi.org/10.4155/fmc-2018-0249" title="DOI URL">https://doi.org/10.4155/fmc-2018-0249</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0249%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DMultitarget-directed%252Bligands%252Bfor%252Bneurodegenerative%252Bdiseases%25253A%252Breal%252Bopportunity%252Bor%252Bblurry%252Bmirage%25253F%26aulast%3DBenchekroun%26aufirst%3DMohamed%26date%3D2019%26volume%3D11%26issue%3D4%26spage%3D261%26epage%3D263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Pinzi</span>, <span class="hlFld-ContribAuthor ">Fabiana  Caporuscio</span>, <span class="hlFld-ContribAuthor ">Giulio  Rastelli</span>. </span><span class="cited-content_cbyCitation_article-title">Selection of protein conformations for structure-based polypharmacology studies. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (11)
                                     , 1889-1896. <a href="https://doi.org/10.1016/j.drudis.2018.08.007" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DSelection%252Bof%252Bprotein%252Bconformations%252Bfor%252Bstructure-based%252Bpolypharmacology%252Bstudies%26aulast%3DPinzi%26aufirst%3DLuca%26date%3D2018%26volume%3D23%26issue%3D11%26spage%3D1889%26epage%3D1896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Justyna  Kalinowska-Tłuścik</span>, <span class="hlFld-ContribAuthor ">Agata  Piaskowska</span>, <span class="hlFld-ContribAuthor ">Marcin  Kołaczkowski</span>. </span><span class="cited-content_cbyCitation_article-title">Multifunctional arylsulfonamide derivatives with 5-HT
              6
              /5-HT
              7
              receptor antagonistic activity: a structural study. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section C Structural Chemistry</span><span> <strong>2018,</strong> <em>74 </em>
                                    (11)
                                     , 1477-1486. <a href="https://doi.org/10.1107/S2053229618013748" title="DOI URL">https://doi.org/10.1107/S2053229618013748</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2053229618013748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2053229618013748%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520C%2520Structural%2520Chemistry%26atitle%3DMultifunctional%252Barylsulfonamide%252Bderivatives%252Bwith%252B5-HT%252B6%252B%25252F5-HT%252B7%252Breceptor%252Bantagonistic%252Bactivity%25253A%252Ba%252Bstructural%252Bstudy%26aulast%3DKalinowska-T%25C5%2582u%25C5%259Bcik%26aufirst%3DJustyna%26date%3D2018%26date%3D2018%26volume%3D74%26issue%3D11%26spage%3D1477%26epage%3D1486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Targets for development of dual-target antiparkinson agents. (a) Crystal structures of the A<sub>2A</sub>AR (gray) and MAO-B (green) are shown as cartoons together with 2D representations of adenosine and dopamine. (b) 2D representation of CSC, a dual-target A<sub>2A</sub>AR and MAO-B inhibitor.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overview of the structure-based screen for dual-target A<sub>2A</sub>AR/MAO-B ligands. The ZINC lead-like and fragment libraries were screened against A<sub>2A</sub>AR (gray) and MAO-B (green) crystal structures. Twenty-four top-ranked molecules were selected for experimental evaluation. Twelve and six compounds showed activity at MAO-B and A<sub>2A</sub>AR, respectively, and four of these displayed dual-activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf" class="ext-link">Table S4</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Predicted binding modes of two dual-target ligands: the docking poses of compounds <b>1</b> (a, b) and <b>3</b> (c, d) in the A<sub>2A</sub>AR (gray cartoon, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PWH">3PWH</a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>) and MAO-B binding sites (green cartoon, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V61">2V61</a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>). Key binding site residues and the ligands are shown in sticks. Hydrogen bonds are shown as black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Viability assessment on human dopaminergic neuronal-like cells treated with 6-OHDA toxin. Cells were treated with 0.15 μM <b>1a</b>, <b>3</b>, or CSC for 3 h prior to addition of 20 μM 6-OHDA. After 24 h, resazurin was added for another 2 h, whereupon cell viability was assayed. Data are shown as the mean ± SEM and were analyzed using one-way ANOVA with Newman–Keuls multiple comparison test: (###) <i>p</i> < 0.001 versus control and (∗∗∗) <i>p</i> < 0.001 versus 6-OHDA alone.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/medium/jm-2018-00204n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the predicted binding mode of compound <b>3</b> to a crystal structure of a related antagonist in complex with the A<sub>2A</sub>AR. The dual-target ligand <b>3</b> is shown as sticks with orange carbon atoms. The cocrystallized antagonist (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UIG">5UIG</a><a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>) is depicted in sticks with gray carbon atoms. The A<sub>2A</sub>AR is depicted as gray cartoons. Hydrogen bonds are shown as black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-12/acs.jmedchem.8b00204/20190828/images/large/jm-2018-00204n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00204&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i24">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22661" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22661" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rang, H. P.</span></span> <span> </span><span class="NLM_article-title">The Receptor Concept: Pharmacology’s Big Idea</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">S9</span>– <span class="NLM_lpage">S16</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0706457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fsj.bjp.0706457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=16402126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XisVClsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2006&pages=S9-S16&author=H.+P.+Rang&title=The+Receptor+Concept%3A+Pharmacology%E2%80%99s+Big+Idea&doi=10.1038%2Fsj.bjp.0706457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The receptor concept: pharmacology's big idea</span></div><div class="casAuthors">Rang, H. P.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S9-S16</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chem. signaling is the main mechanism by which biol. function is controlled at all levels, from the single cell to the whole organism.  Chem. recognition is the function of receptors, which, in addn. to recognizing endogenous chem. signals, are also the target of many important exptl. and therapeutic drugs.  Receptors, therefore, lie at the heart of pharmacol.  This article describes the way in which the receptor concept originated early in the 20th century, and evolved through a highly innovative stage of quant. theory based on chem. kinetics, to the point where receptors were first isolated and later cloned, until we now have a virtually complete catalog of all the receptors present in the genome.  Studies on signal transduction are revealing great complexity in the events linking ligand binding to the physiol. or therapeutic response.  Though some simple quant. rules of 'receptor theory' are still useful, the current emphasis is on unraveling the pathways that link receptors to responses, and it will be some time before we know enough about them to embark on the next phase of 'receptor theory'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1enUcNFx4g7Vg90H21EOLACvtfcHk0lgzi2gMV2iIiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisVClsA%253D%253D&md5=2e28f786f60570b75f4bd0d937a151df</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0706457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0706457%26sid%3Dliteratum%253Aachs%26aulast%3DRang%26aufirst%3DH.%2BP.%26atitle%3DThe%2520Receptor%2520Concept%253A%2520Pharmacology%25E2%2580%2599s%2520Big%2520Idea%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2006%26volume%3D147%26spage%3DS9%26epage%3DS16%26doi%3D10.1038%2Fsj.bjp.0706457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheffler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroeze, W. K.</span></span> <span> </span><span class="NLM_article-title">Magic Shotguns versus Magic Bullets: Selectively Non-Selective Drugs for Mood Disorders and Schizophrenia</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1038/nrd1346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd1346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=15060530" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1Gkt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=353-359&author=B.+L.+Rothauthor=D.+J.+Shefflerauthor=W.+K.+Kroeze&title=Magic+Shotguns+versus+Magic+Bullets%3A+Selectively+Non-Selective+Drugs+for+Mood+Disorders+and+Schizophrenia&doi=10.1038%2Fnrd1346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Magic shotguns versus magic bullets: selectively non-selective drugs for mood disorders and schizophrenia</span></div><div class="casAuthors">Roth, Bryan L.; Sheffler, Douglas J.; Kroeze, Wesley K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">353-359</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Most common central nervous system disorders - such as depression, bipolar disorder, and schizophrenia - seem to be polygenic in origin, and the most effective medications have exceedingly complex pharmacologies.  Attempts to develop more effective treatments for diseases such as schizophrenia and depression by discovering drugs selective for single mol. targets (i.e., 'magic bullets') have, not surprisingly, been largely unsuccessful.  Here the authors propose that designing selectively non-selective drugs (i.e., 'magic shotguns') that interact with several mol. targets will lead to new and more effective medications for a variety of central nervous system disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnSsuUXzmbirVg90H21EOLACvtfcHk0lgzi2gMV2iIiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXis1Gkt78%253D&md5=77196583acccca65457fec7a67218463</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd1346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1346%26sid%3Dliteratum%253Aachs%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DSheffler%26aufirst%3DD.%2BJ.%26aulast%3DKroeze%26aufirst%3DW.%2BK.%26atitle%3DMagic%2520Shotguns%2520versus%2520Magic%2520Bullets%253A%2520Selectively%2520Non-Selective%2520Drugs%2520for%2520Mood%2520Disorders%2520and%2520Schizophrenia%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D353%26epage%3D359%26doi%3D10.1038%2Fnrd1346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lgzi2gMV2iIiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mìnarini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumiatti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchiorre, C.</span></span> <span> </span><span class="NLM_article-title">Multi-Target-Directed Ligands to Combat Neurodegenerative Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1021/jm7009364</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7009364" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=347-372&author=A.+Cavalliauthor=M.+L.+Bolognesiauthor=A.+M%C3%ACnariniauthor=M.+Rosiniauthor=V.+Tumiattiauthor=M.+Recanatiniauthor=C.+Melchiorre&title=Multi-Target-Directed+Ligands+to+Combat+Neurodegenerative+Diseases&doi=10.1021%2Fjm7009364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target-Directed Ligands to Combat Neurodegenerative Diseases</span></div><div class="casAuthors">Cavalli, Andrea; Bolognesi, Maria Laura; Minarini, Anna; Rosini, Michela; Tumiatti, Vincenzo; Recanatini, Maurizio; Melchiorre, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-372</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors review the investigations reported in the past decade that illustrate the ability of medicinal chemists to design multi-target-directed ligands (MTDLs) to confront neurodegenerative diseases.  The authors briefly illustrate the multifactorial nature of these diseases before discussing the possible development of MTDLs in light of the known mol. bases of the main neurodegenerative pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYY0I0UHEir7Vg90H21EOLACvtfcHk0lj6GgcFrArbBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D&md5=c682235151489316d043cf97f78e56f3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm7009364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7009364%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DM%25C3%25ACnarini%26aufirst%3DA.%26aulast%3DRosini%26aufirst%3DM.%26aulast%3DTumiatti%26aufirst%3DV.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DMulti-Target-Directed%2520Ligands%2520to%2520Combat%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D347%26epage%3D372%26doi%3D10.1021%2Fjm7009364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhillon, S.</span></span> <span> </span><span class="NLM_article-title">Safinamide: A Review in Parkinson’s Disease</span>. <i>CNS Drugs</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">176</span>, <span class="refDoi"> DOI: 10.1007/s40263-017-0408-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1007%2Fs40263-017-0408-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=28110399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVeisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=169-176&author=H.+A.+Blairauthor=S.+Dhillon&title=Safinamide%3A+A+Review+in+Parkinson%E2%80%99s+Disease&doi=10.1007%2Fs40263-017-0408-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Safinamide: A Review in Parkinson's Disease</span></div><div class="casAuthors">Blair, Hannah A.; Dhillon, Sohita</div><div class="citationInfo"><span class="NLM_cas:title">CNS Drugs</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-176</span>CODEN:
                <span class="NLM_cas:coden">CNDREF</span>;
        ISSN:<span class="NLM_cas:issn">1172-7047</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Safinamide (Xadago) is an orally active, selective, reversible monoamine oxidase-B inhibitor with both dopaminergic and non-dopaminergic (glutamatergic) properties.  In the EU, safinamide is approved for the treatment of mid- to late-stage fluctuating Parkinson's disease (PD) as add-on therapy to a stable dose of levodopa alone or in combination with other PD medications.  Safinamide 50-100 mg/day administered as a fixed or flexible dose significantly increased daily 'on' time without dyskinesia (primary endpoint) in patients with mid- to late-stage PD with motor fluctuations in 24-wk, placebo-controlled clin. trials.  Other outcomes, including motor function, overall clin. status and health-related quality of life, were also generally improved with safinamide.  Furthermore, in an 18-mo extension of one study, although dyskinesia (primary endpoint) was not significantly improved with safinamide relative to placebo, treatment benefits in other outcomes were generally sustained over 24 mo of treatment.  Safinamide was generally well tolerated in clin. trials; dyskinesia was the most common adverse event.  Although further studies are needed, including comparative and long-term studies, current evidence indicates that safinamide extends the treatment options available for use as add-on therapy to levodopa and other PD medications in patients with mid- to late-stage PD experiencing motor fluctuations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl8UyUX-7XrVg90H21EOLACvtfcHk0lj6GgcFrArbBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVeisrc%253D&md5=7adc3525326e5274c2f9fac22d39cb28</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs40263-017-0408-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40263-017-0408-1%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DH.%2BA.%26aulast%3DDhillon%26aufirst%3DS.%26atitle%3DSafinamide%253A%2520A%2520Review%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DCNS%2520Drugs%26date%3D2017%26volume%3D31%26spage%3D169%26epage%3D176%26doi%3D10.1007%2Fs40263-017-0408-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein
Herenbrink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span> <span> </span><span class="NLM_article-title">Proof of Concept Study for Designed Multiple Ligands Targeting the Dopamine D<sub>2</sub>, Serotonin 5-HT<sub>2A</sub>, and Muscarinic M<sub>1</sub> Acetylcholine Receptors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1550</span>– <span class="NLM_lpage">1555</span>, <span class="refDoi"> DOI: 10.1021/jm5013243</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5013243" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVygug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1550-1555&author=M.+Szaboauthor=H.+D.+Limauthor=C.+Klein%0AHerenbrinkauthor=A.+Christopoulosauthor=J.+R.+Laneauthor=B.+Capuano&title=Proof+of+Concept+Study+for+Designed+Multiple+Ligands+Targeting+the+Dopamine+D2%2C+Serotonin+5-HT2A%2C+and+Muscarinic+M1+Acetylcholine+Receptors&doi=10.1021%2Fjm5013243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Proof of Concept Study for Designed Multiple Ligands Targeting the Dopamine D2, Serotonin 5-HT2A, and Muscarinic M1 Acetylcholine Receptors</span></div><div class="casAuthors">Szabo, Monika; Lim, Herman D.; Klein Herenbrink, Carmen; Christopoulos, Arthur; Lane, J. Robert; Capuano, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1550-1555</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein the authors describe the hybridization of a benzoxazinone M1 scaffold with D2 privileged structures derived from putative and clin. relevant antipsychotics to develop designed multiple ligands.  The M1 mAChR is an attractive target for the cognitive deficits in key CNS disorders.  Moreover, activity at D2 and 5-HT2A receptors has proven useful for antipsychotic efficacy.  The authors identified 9 which retained functional activity at the target M1 mAChR and D2R and demonstrated high affinity for the 5-HT2AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJqxxyfgdbsbVg90H21EOLACvtfcHk0lj6GgcFrArbBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVygug%253D%253D&md5=9aebec84dcc8ce782a4ef4d53cf773e0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm5013243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5013243%26sid%3Dliteratum%253Aachs%26aulast%3DSzabo%26aufirst%3DM.%26aulast%3DLim%26aufirst%3DH.%2BD.%26aulast%3DKlein%2BHerenbrink%26aufirst%3DC.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DCapuano%26aufirst%3DB.%26atitle%3DProof%2520of%2520Concept%2520Study%2520for%2520Designed%2520Multiple%2520Ligands%2520Targeting%2520the%2520Dopamine%2520D2%252C%2520Serotonin%25205-HT2A%252C%2520and%2520Muscarinic%2520M1%2520Acetylcholine%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1550%26epage%3D1555%26doi%3D10.1021%2Fjm5013243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abecassis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguiz, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norval, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeons, F. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stojanovski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidah, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Constam, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetsel, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Automated Design of Ligands to Polypharmacological Profiles</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1038/nature11691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnature11691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=23235874" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=215-220&author=J.+Besnardauthor=G.+F.+Rudaauthor=V.+Setolaauthor=K.+Abecassisauthor=R.+M.+Rodriguizauthor=X.-P.+Huangauthor=S.+Norvalauthor=M.+F.+Sassanoauthor=A.+I.+Shinauthor=L.+A.+Websterauthor=F.+R.+C.+Simeonsauthor=L.+Stojanovskiauthor=A.+Pratauthor=N.+G.+Seidahauthor=D.+B.+Constamauthor=G.+R.+Bickertonauthor=K.+D.+Readauthor=W.+C.+Wetselauthor=I.+H.+Gilbertauthor=B.+L.+Rothauthor=A.+L.+Hopkins&title=Automated+Design+of+Ligands+to+Polypharmacological+Profiles&doi=10.1038%2Fnature11691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Automated design of ligands to polypharmacological profiles</span></div><div class="casAuthors">Besnard, Jeremy; Ruda, Gian Filippo; Setola, Vincent; Abecassis, Keren; Rodriguiz, Ramona M.; Huang, Xi-Ping; Norval, Suzanne; Sassano, Maria F.; Shin, Antony I.; Webster, Lauren A.; Simeons, Frederick R. C.; Stojanovski, Laste; Prat, Annik; Seidah, Nabil G.; Constam, Daniel B.; Bickerton, G. Richard; Read, Kevin D.; Wetsel, William C.; Gilbert, Ian H.; Roth, Bryan L.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7428</span>),
    <span class="NLM_cas:pages">215-220</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. efficacy and safety of a drug is detd. by its activity profile across many proteins in the proteome.  However, designing drugs with a specific multi-target profile is both complex and difficult.  Therefore methods to design drugs rationally a priori against profiles of several proteins would have immense value in drug discovery.  Here is described a new approach for the automated design of ligands against profiles of multiple drug targets.  The method is demonstrated by the evolution of an approved acetylcholinesterase inhibitor drug donepezil (I) into brain-penetrable ligands with either specific polypharmacol. or exquisite selectivity profiles for G-protein-coupled receptors.  Overall, 800 ligand-target predictions of prospectively designed ligands were tested exptl., of which 75% were confirmed to be correct.  Target engagement in vivo is also demonstrated.  The approach can be a useful source of drug leads when multi-target profiles are required to achieve either selectivity over other drug targets or a desired polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu5bwrBD3tbVg90H21EOLACvtfcHk0lhfuy9s-xKcPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN&md5=ace6cced250cf968d327b482a5bf1798</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature11691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11691%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DAbecassis%26aufirst%3DK.%26aulast%3DRodriguiz%26aufirst%3DR.%2BM.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DNorval%26aufirst%3DS.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DShin%26aufirst%3DA.%2BI.%26aulast%3DWebster%26aufirst%3DL.%2BA.%26aulast%3DSimeons%26aufirst%3DF.%2BR.%2BC.%26aulast%3DStojanovski%26aufirst%3DL.%26aulast%3DPrat%26aufirst%3DA.%26aulast%3DSeidah%26aufirst%3DN.%2BG.%26aulast%3DConstam%26aufirst%3DD.%2BB.%26aulast%3DBickerton%26aufirst%3DG.%2BR.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DWetsel%26aufirst%3DW.%2BC.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DAutomated%2520Design%2520of%2520Ligands%2520to%2520Polypharmacological%2520Profiles%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D215%26epage%3D220%26doi%3D10.1038%2Fnature11691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the Cancer Kinome through Polypharmacology</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1038/nrc2787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrc2787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20094047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=130-137&author=Z.+A.+Knightauthor=H.+Linauthor=K.+M.+Shokat&title=Targeting+the+Cancer+Kinome+through+Polypharmacology&doi=10.1038%2Fnrc2787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer kinome through polypharmacology</span></div><div class="casAuthors">Knight, Zachary A.; Lin, Henry; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-137</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Kinase inhibitors are the largest class of new cancer drugs.  However, it is already apparent that most tumors can escape from the inhibition of any single kinase.  If it is necessary to inhibit multiple kinases, how do we choose which ones.  In this Opinion article, we discuss some of the strategies that are currently being used to identify new therapeutic combinations of kinase targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkDv2FeeH58rVg90H21EOLACvtfcHk0lhfuy9s-xKcPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D&md5=9aaa314e54b43889e147a9f11ac77072</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc2787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2787%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DTargeting%2520the%2520Cancer%2520Kinome%2520through%2520Polypharmacology%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D130%26epage%3D137%26doi%3D10.1038%2Fnrc2787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paoletta, S.</span></span> <span> </span><span class="NLM_article-title">Computational Studies to Predict or Explain G Protein Coupled Receptor Polypharmacology</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2014.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.tips.2014.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=25458540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCktL3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=658-663&author=K.+A.+Jacobsonauthor=S.+Costanziauthor=S.+Paoletta&title=Computational+Studies+to+Predict+or+Explain+G+Protein+Coupled+Receptor+Polypharmacology&doi=10.1016%2Fj.tips.2014.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Computational studies to predict or explain G protein coupled receptor polypharmacology</span></div><div class="casAuthors">Jacobson, Kenneth A.; Costanzi, Stefano; Paoletta, Silvia</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">658-663</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Since G protein-coupled receptors (GPCRs) belong to a very large superfamily of evolutionarily related receptors (>800 members in humans), and due to the rapid progress on their structural biol., they are ideal candidates for polypharmacol. studies.  Broad screening and bioinformatics/chemoinformatics have been applied to understanding off-target effects of GPCR ligands.  It is now feasible to approach the question of GPCR polypharmacol. using mol. modeling and the available X-ray GPCR structures.  As an example, large and sterically constrained adenosine derivs. (potent adenosine receptor ligands with low conformational freedom and multiple extended substituents) were screened for binding at diverse receptors.  Unanticipated off-target interactions, including at biogenic amine receptors, were then modeled using a structure-based approach to provide a consistent understanding of recognition.  A conserved Asp in TM3 changed its role from counterion for biogenic amines to characteristic H-bonding to adenosine.  The same systematic approach could potentially be applied to many GPCRs or other receptors using other sets of congeneric ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfRo5bhR-c57Vg90H21EOLACvtfcHk0li6JwPvrvkC_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCktL3E&md5=e3490f7573c45c59760cb5e598b3cd9b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCostanzi%26aufirst%3DS.%26aulast%3DPaoletta%26aufirst%3DS.%26atitle%3DComputational%2520Studies%2520to%2520Predict%2520or%2520Explain%2520G%2520Protein%2520Coupled%2520Receptor%2520Polypharmacology%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D658%26epage%3D663%26doi%3D10.1016%2Fj.tips.2014.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Can We Rationally Design Promiscuous Drugs?</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.sbi.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=16442279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=127-136&author=A.+L.+Hopkinsauthor=J.+S.+Masonauthor=J.+P.+Overington&title=Can+We+Rationally+Design+Promiscuous+Drugs%3F&doi=10.1016%2Fj.sbi.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Can we rationally design promiscuous drugs?</span></div><div class="casAuthors">Hopkins, Andrew L.; Mason, Jonathan S.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-136</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design is now used widely in modern medicinal chem.  The application of structural biol. to medicinal chem. has heralded the 'rational drug design' vision of discovering exquisitely selective ligands.  However, recent advances in post-genomic biol. are indicating that polypharmacol. may be a necessary trait for the efficacy of many drugs, therefore questioning the 'one drug, one target' assumption of current rational drug design.  By combining advances in chemoinformatics and structural biol., it might be possible to rationally design the next generation of promiscuous drugs with polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQWdh60x9AHrVg90H21EOLACvtfcHk0li6JwPvrvkC_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D&md5=8f9e95a41976b8e3ac790f2fff2c406f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DCan%2520We%2520Rationally%2520Design%2520Promiscuous%2520Drugs%253F%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D127%26epage%3D136%26doi%3D10.1016%2Fj.sbi.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lees, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revesz, T.</span></span> <span> </span><span class="NLM_article-title">Parkinson’s Disease</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>373</i></span>,  <span class="NLM_fpage">2055</span>– <span class="NLM_lpage">2066</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(09)60492-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2FS0140-6736%2809%2960492-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=19524782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1ans7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=373&publication_year=2009&pages=2055-2066&author=A.+J.+Leesauthor=J.+Hardyauthor=T.+Revesz&title=Parkinson%E2%80%99s+Disease&doi=10.1016%2FS0140-6736%2809%2960492-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Parkinson's disease</span></div><div class="casAuthors">Lees, Andrew J.; Hardy, John; Revesz, Tamas</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">373</span>
        (<span class="NLM_cas:issue">9680</span>),
    <span class="NLM_cas:pages">2055-2066</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Summary: Parkinson's disease is a common progressive bradykinetic disorder that can be accurately diagnosed.  It is characterised by the presence of severe pars-compacta nigral-cell loss, and accumulation of aggregated α-synuclein in specific brain stem, spinal cord, and cortical regions.  The main known risk factor is age.  Susceptibility genes including α-synuclein, leucine rich repeat kinase 2 (LRRK-2), and glucocerebrosidase (GBA) have shown that genetic predisposition is another important causal factor.  Dopamine replacement therapy considerably reduces motor handicap, and effective treatment of assocd. depression, pain, constipation, and nocturnal difficulties can improve quality of life.  Embryonic stem cells and gene therapy are promising research therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjOcLAU8_Xq7Vg90H21EOLACvtfcHk0li6JwPvrvkC_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1ans7w%253D&md5=ae632b0ac91f4e5ee017bf2bf21f5926</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2809%2960492-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252809%252960492-X%26sid%3Dliteratum%253Aachs%26aulast%3DLees%26aufirst%3DA.%2BJ.%26aulast%3DHardy%26aufirst%3DJ.%26aulast%3DRevesz%26aufirst%3DT.%26atitle%3DParkinson%25E2%2580%2599s%2520Disease%26jtitle%3DLancet%26date%3D2009%26volume%3D373%26spage%3D2055%26epage%3D2066%26doi%3D10.1016%2FS0140-6736%2809%2960492-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccini, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeso, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rascol, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tison, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezard, E.</span></span> <span> </span><span class="NLM_article-title">Priorities in Parkinson’s Disease Research</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">393</span>, <span class="refDoi"> DOI: 10.1038/nrd3430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd3430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21532567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVGltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=377-393&author=W.+G.+Meissnerauthor=M.+Frasierauthor=T.+Gasserauthor=C.+G.+Goetzauthor=A.+Lozanoauthor=P.+Picciniauthor=J.+A.+Obesoauthor=O.+Rascolauthor=A.+Schapiraauthor=V.+Voonauthor=D.+M.+Weinerauthor=F.+Tisonauthor=E.+Bezard&title=Priorities+in+Parkinson%E2%80%99s+Disease+Research&doi=10.1038%2Fnrd3430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Priorities in Parkinson's disease research</span></div><div class="casAuthors">Meissner, Wassilios G.; Frasier, Mark; Gasser, Thomas; Goetz, Christopher G.; Lozano, Andres; Piccini, Paola; Obeso, Jose A.; Rascol, Olivier; Schapira, Anthony; Voon, Valerie; Weiner, David M.; Tison, Francois; Bezard, Erwan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">377-393</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The loss of dopaminergic neurons in the substantia nigra pars compacta leads to the characteristic motor symptoms of Parkinson's disease: bradykinesia, rigidity and resting tremors.  Although these symptoms can be improved using currently available dopamine replacement strategies, there is still a need to improve current strategies of treating these symptoms, together with a need to alleviate non-motor symptoms of the disease.  Moreover, treatments that provide neuroprotection and/or disease-modifying effects remain an urgent unmet clin. need.  This Review describes the most promising biol. targets and therapeutic agents that are currently being assessed to address these treatment goals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFgF5KqF7ZzrVg90H21EOLACvtfcHk0ljUrx3WKQ9_sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVGltr4%253D&md5=d792624cf69d3a56537eb8aa00c5b963</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnrd3430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3430%26sid%3Dliteratum%253Aachs%26aulast%3DMeissner%26aufirst%3DW.%2BG.%26aulast%3DFrasier%26aufirst%3DM.%26aulast%3DGasser%26aufirst%3DT.%26aulast%3DGoetz%26aufirst%3DC.%2BG.%26aulast%3DLozano%26aufirst%3DA.%26aulast%3DPiccini%26aufirst%3DP.%26aulast%3DObeso%26aufirst%3DJ.%2BA.%26aulast%3DRascol%26aufirst%3DO.%26aulast%3DSchapira%26aufirst%3DA.%26aulast%3DVoon%26aufirst%3DV.%26aulast%3DWeiner%26aufirst%3DD.%2BM.%26aulast%3DTison%26aufirst%3DF.%26aulast%3DBezard%26aufirst%3DE.%26atitle%3DPriorities%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%2520Research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D377%26epage%3D393%26doi%3D10.1038%2Fnrd3430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, A. H. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotchie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collingridge, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hengerer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Novère, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obeso, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarzschild, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spampinato, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidai, G.</span></span> <span> </span><span class="NLM_article-title">Novel Pharmacological Targets for the Treatment of Parkinson’s Disease</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1038/nrd2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=17016425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVajsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=845-854&author=A.+H.+V.+Schapiraauthor=E.+Bezardauthor=J.+Brotchieauthor=F.+Calonauthor=G.+L.+Collingridgeauthor=B.+Fergerauthor=B.+Hengererauthor=E.+Hirschauthor=P.+Jennerauthor=N.+Le+Nov%C3%A8reauthor=J.+A.+Obesoauthor=M.+A.+Schwarzschildauthor=U.+Spampinatoauthor=G.+Davidai&title=Novel+Pharmacological+Targets+for+the+Treatment+of+Parkinson%E2%80%99s+Disease&doi=10.1038%2Fnrd2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Novel pharmacological targets for the treatment of Parkinson's disease</span></div><div class="casAuthors">Schapira, Anthony H. V.; Bezard, Erwan; Brotchie, Jonathan; Calon, Frederic; Collingridge, Graham L.; Ferger, Borris; Hengerer, Bastian; Hirsch, Etienne; Jenner, Peter; Le Novere, Nicolas; Obeso, Jose A.; Schwarzschild, Michael A.; Spampinato, Umberto; Davidai, Giora</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">845-854</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dopamine deficiency, caused by the degeneration of nigrostriatal dopaminergic neurons, is the cause of the major clin. motor symptoms of Parkinson's disease.  These symptoms can be treated successfully with a range of drugs that include levodopa, inhibitors of the enzymic breakdown of levodopa and dopamine agonists delivered by oral, s.c., transcutaneous, i.v. or intra-duodenal routes.  However, Parkinson's disease involves degeneration of non-dopaminergic neurons and the treatment of the resulting predominantly non-motor features remains a challenge.  This review describes the important recent advances that underlie the development of novel dopaminergic and non-dopaminergic drugs for Parkinson's disease, and also for the motor complications that arise from the use of existing therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR-UyAD07eK7Vg90H21EOLACvtfcHk0ljUrx3WKQ9_sA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVajsbnE&md5=be3cf2f5a3523bf11b274c501677a9f6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2087%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DA.%2BH.%2BV.%26aulast%3DBezard%26aufirst%3DE.%26aulast%3DBrotchie%26aufirst%3DJ.%26aulast%3DCalon%26aufirst%3DF.%26aulast%3DCollingridge%26aufirst%3DG.%2BL.%26aulast%3DFerger%26aufirst%3DB.%26aulast%3DHengerer%26aufirst%3DB.%26aulast%3DHirsch%26aufirst%3DE.%26aulast%3DJenner%26aufirst%3DP.%26aulast%3DLe%2BNov%25C3%25A8re%26aufirst%3DN.%26aulast%3DObeso%26aufirst%3DJ.%2BA.%26aulast%3DSchwarzschild%26aufirst%3DM.%2BA.%26aulast%3DSpampinato%26aufirst%3DU.%26aulast%3DDavidai%26aufirst%3DG.%26atitle%3DNovel%2520Pharmacological%2520Targets%2520for%2520the%2520Treatment%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D845%26epage%3D854%26doi%3D10.1038%2Fnrd2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzer, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Der Schyf, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castagnoli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonsalla, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castagnoli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarzschild, M. A.</span></span> <span> </span><span class="NLM_article-title">8-(3-Chlorostyryl) Caffeine May Attenuate MPTP Neurotoxicity through Dual Actions of Monoamine Oxidase Inhibition and A<sub>2A</sub> Receptor Antagonism</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">36040</span>– <span class="NLM_lpage">36044</span>, <span class="refDoi"> DOI: 10.1074/jbc.M206830200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1074%2Fjbc.M206830200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=12130655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsVCjs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=36040-36044&author=J.+F.+Chenauthor=S.+Steynauthor=R.+Staalauthor=J.+P.+Petzerauthor=K.+Xuauthor=C.+J.+Van+Der+Schyfauthor=K.+Castagnoliauthor=P.+K.+Sonsallaauthor=N.+Castagnoliauthor=M.+A.+Schwarzschild&title=8-%283-Chlorostyryl%29+Caffeine+May+Attenuate+MPTP+Neurotoxicity+through+Dual+Actions+of+Monoamine+Oxidase+Inhibition+and+A2A+Receptor+Antagonism&doi=10.1074%2Fjbc.M206830200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">8-(3-Chlorostyryl)caffeine May Attenuate MPTP Neurotoxicity through Dual Actions of Monoamine Oxidase Inhibition and A2A Receptor Antagonism</span></div><div class="casAuthors">Chen, Jiang-Fan; Steyn, Salome; Staal, Roland; Petzer, Jacobus P.; Xu, Kui; Van der Schyf, Cornelis J.; Castagnoli, Kay; Sonsalla, Patricia K.; Castagnoli, Neal, Jr.; Schwarzschild, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">36040-36044</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Caffeine and more specific antagonists of the adenosine A2A receptor recently have been found to be neuroprotective in the MPTP (1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine) model of Parkinson's disease.  Here we show that 8-(3-chlorostyryl)caffeine (CSC), a specific A2A antagonist closely related to caffeine, also attenuates MPTP-induced neurotoxicity.  Because the neurotoxicity of MPTP relies on its oxidative metab. to the mitochondrial toxin MPP+, we investigated the actions of CSC on striatal MPTP metab. in vivo.  CSC elevated striatal levels of MPTP but lowered levels of the oxidative intermediate MPDP+ and of MPP+, suggesting that CSC blocks the conversion of MPTP to MPDP+ in vivo.  In assessing the direct effects of CSC and A2A receptors on monoamine oxidase (MAO) activity, we found that CSC potently and specifically inhibited mouse brain mitochondrial MAO-B activity in vitro with a Ki value of 100 nm, whereas caffeine and another relatively specific A2A antagonist produced little or no inhibition.  The A2A receptor independence of MAO-B inhibition by CSC was further supported by the similarity of brain MAO activities derived from A2A receptor knockout and wild-type mice and was confirmed by demonstrating potent inhibition of A2A receptor knockout-derived MAO-B by CSC.  Together, these data indicate that CSC possesses dual actions of MAO-B inhibition and A2A receptor antagonism, a unique combination suggesting a new class of compds. with the potential for enhanced neuroprotective properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL8ESRi-__yrVg90H21EOLACvtfcHk0lizG-vPT0MtRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsVCjs7Y%253D&md5=27154cad8b81538a593504f7ca2f42ec</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M206830200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M206830200%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%2BF.%26aulast%3DSteyn%26aufirst%3DS.%26aulast%3DStaal%26aufirst%3DR.%26aulast%3DPetzer%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DVan%2BDer%2BSchyf%26aufirst%3DC.%2BJ.%26aulast%3DCastagnoli%26aufirst%3DK.%26aulast%3DSonsalla%26aufirst%3DP.%2BK.%26aulast%3DCastagnoli%26aufirst%3DN.%26aulast%3DSchwarzschild%26aufirst%3DM.%2BA.%26atitle%3D8-%25283-Chlorostyryl%2529%2520Caffeine%2520May%2520Attenuate%2520MPTP%2520Neurotoxicity%2520through%2520Dual%2520Actions%2520of%2520Monoamine%2520Oxidase%2520Inhibition%2520and%2520A2A%2520Receptor%2520Antagonism%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D36040%26epage%3D36044%26doi%3D10.1074%2Fjbc.M206830200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hockemeyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbiel, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Multigram-Scale Syntheses, Stability, and Photoreactions of A<sub>2A</sub> Adenosine Receptor Antagonists with 8-Styrylxanthine Structure: Potential Drugs for Parkinson’s Disease</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">3308</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1021/jo0358574</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo0358574" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFarsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2004&pages=3308-3318&author=J.+Hockemeyerauthor=J.+C.+Burbielauthor=C.+E.+M%C3%BCller&title=Multigram-Scale+Syntheses%2C+Stability%2C+and+Photoreactions+of+A2A+Adenosine+Receptor+Antagonists+with+8-Styrylxanthine+Structure%3A+Potential+Drugs+for+Parkinson%E2%80%99s+Disease&doi=10.1021%2Fjo0358574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Multigram-Scale Syntheses, Stability, and Photoreactions of A2A Adenosine Receptor Antagonists with 8-Styrylxanthine Structure: Potential Drugs for Parkinson's Disease</span></div><div class="casAuthors">Hockemeyer, Joerg; Burbiel, Joachim C.; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3308-3318</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The improved multigram-scale syntheses of the important 8-styrylxanthine A2A adenosine receptor antagonist MSX-2 [I, R = CH2C≡CH, R1 = (CH2)3OH, R2 = H], its water-sol. prodrug MXS-3 [I, R = CH2C≡CH, R1 = (CH2)3OP(O)(OH)2, R2 = H], and KW-6002 [I, R = R1 = Et, R2 = OMe] are described.  N-Alkylation reactions at different positions of the uracil ring were optimized.  Two different methods for xanthine formation from 6-amino-5-cinnamoylaminouracil precursors were investigated: (a) the elimination of water by alk. catalysis and (b) hexamethyldisilazane as a condensing agent; the latter was found to be superior.  The photosensitivity of the 8-styrylxanthines was studied.  The (E)-configurated stryrylxanthine MSX-2 isomerized in dild. soln., and the resulting (Z)-isomer was isolated and characterized.  Furthermore, solid 8-styrylxanthines can dimerize upon exposure to daylight or irradn. with UV light.  The resulting cyclobutane derivs. with head-to-tail (syn) configuration exhibited a considerably lower A2A adenosine receptor affinity than their parent compds.  The dimerization product of MSX-2 was a moderately potent nonselective A1 and A2A antagonist (Ki(A1) = 273 nM, Ki (A2A) = 175 nM) while the dimer of KW-6002 was inactive at A1 and only weakly active at A2A adenosine receptors (Ki = 1.57 μM).  The light sensitivity of 8-styrylxanthine derivs., not only in soln., but also in the solid state, has to be considered when using those compds. as pharmacol. tools or drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH59mwpE6WRLVg90H21EOLACvtfcHk0lizG-vPT0MtRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFarsro%253D&md5=b76c488fd1df3d5c31b3911bf1b27d6b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjo0358574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo0358574%26sid%3Dliteratum%253Aachs%26aulast%3DHockemeyer%26aufirst%3DJ.%26aulast%3DBurbiel%26aufirst%3DJ.%2BC.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DMultigram-Scale%2520Syntheses%252C%2520Stability%252C%2520and%2520Photoreactions%2520of%2520A2A%2520Adenosine%2520Receptor%2520Antagonists%2520with%25208-Styrylxanthine%2520Structure%253A%2520Potential%2520Drugs%2520for%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2004%26volume%3D69%26spage%3D3308%26epage%3D3318%26doi%3D10.1021%2Fjo0358574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qurishi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diekmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padgett, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, J. W.</span></span> <span> </span><span class="NLM_article-title">Imidazo[2,1-I ]Purin-5-Ones and Related Tricyclic Water-Soluble Purine Derivatives: Potent A<sub>2A</sub>- and A<sub>3</sub>-Adenosine Receptor Antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">3440</span>– <span class="NLM_lpage">3450</span>, <span class="refDoi"> DOI: 10.1021/jm011093d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm011093d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD38XkvVGhsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3440-3450&author=C.+E.+M%C3%BCllerauthor=M.+Thorandauthor=R.+Qurishiauthor=M.+Diekmannauthor=K.+A.+Jacobsonauthor=W.+L.+Padgettauthor=J.+W.+Daly&title=Imidazo%5B2%2C1-I+%5DPurin-5-Ones+and+Related+Tricyclic+Water-Soluble+Purine+Derivatives%3A+Potent+A2A-+and+A3-Adenosine+Receptor+Antagonists&doi=10.1021%2Fjm011093d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Imidazo[2,1-i]purin-5-ones and Related Tricyclic Water-Soluble Purine Derivatives: Potent A2A- and A3-Adenosine Receptor Antagonists</span></div><div class="casAuthors">Mueller, Christa E.; Thorand, Mark; Qurishi, Ramatullah; Diekmann, Martina; Jacobson, Kenneth A.; Padgett, William L.; Daly, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3440-3450</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of tricyclic imidazo[2,1-i]purinones I [R1, R4 = H, Me; R2 = H, Ph, (E)-PhCH:CH; R3 = H, Me, PhCH2; R5 = H, Me, Et] and ring-enlarged analogs, e.g. II, derived from xanthine derivs. were prepd. as adenosine receptor (AR) antagonists.  In comparison with xanthines, these tricyclic compds. exhibited increased water soly. due to a basic nitrogen atom, which can be protonated under physiol. conditions.  A new capillary electrophoresis method was developed for the detn. of the enantiomeric purity of selected chiral products using native and modified β-cyclodextrins as chiral discriminators.  The compds. were investigated in radioligand binding assays at rat brain A1 and A2A ARs.  Selected I were addnl. investigated in radioligand binding assays at human recombinant A3 ARs and in functional studies (adenylate cyclase assays) at A1 ARs of rat fat cell membranes, A2A ARs of rat PC 12 cell membranes, and mouse A2B ARs of NIH 3T3 cell membranes, and showed the structure-activity relationships similar to those of the corresponding xanthine derivs.  The 2-styrylimidazopurinones I [R1 = H, Me; R2 = (E)-PhCH:CH; R3 = Me; R4 = H, R5 = Et] were less potent at A2A ARs as compared to 8-styrylxanthine derivs.  The most potent compd. at A2A ARs was (S)-I [R1 = R3 = Me, R2 = (E)-PhCH:CH, R4 = H, R5 = Et; (III)] exhibiting a Ki value of 424 nM at rat A2A ARs.  III was also highly selective for A2A receptors vs A1 and A3 ARs; however, the selectivity vs A2B ARs was low.  Among the 1-unsubstituted (2-phenyl)imidazopurinones, the most potent A3 antagonist was (R)-I (R1 = R4 = H, R2 = Ph, R3 = Me, R5 = Et) exhibiting a Ki value of 2.3 nM and high selectivity for A3 receptors vs all other AR subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqea1ltPz9137Vg90H21EOLACvtfcHk0lizG-vPT0MtRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XkvVGhsrs%253D&md5=893182fdc78beec54214cf712536a134</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm011093d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm011093d%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DThorand%26aufirst%3DM.%26aulast%3DQurishi%26aufirst%3DR.%26aulast%3DDiekmann%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DPadgett%26aufirst%3DW.%2BL.%26aulast%3DDaly%26aufirst%3DJ.%2BW.%26atitle%3DImidazo%255B2%252C1-I%2520%255DPurin-5-Ones%2520and%2520Related%2520Tricyclic%2520Water-Soluble%2520Purine%2520Derivatives%253A%2520Potent%2520A2A-%2520and%2520A3-Adenosine%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3440%26epage%3D3450%26doi%3D10.1021%2Fjm011093d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Armentero, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferré, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanciego, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, R.</span></span> <span> </span><span class="NLM_article-title">Past, Present and Future of A<sub>2A</sub> Adenosine Receptor Antagonists in the Therapy of Parkinson’s Disease</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">280</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2011.07.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.pharmthera.2011.07.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21810444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtl2ntrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2011&pages=280-299&author=M.+T.+Armenteroauthor=A.+Pinnaauthor=S.+Ferr%C3%A9author=J.+L.+Lanciegoauthor=C.+E.+M%C3%BCllerauthor=R.+Franco&title=Past%2C+Present+and+Future+of+A2A+Adenosine+Receptor+Antagonists+in+the+Therapy+of+Parkinson%E2%80%99s+Disease&doi=10.1016%2Fj.pharmthera.2011.07.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Past, present and future of A2A adenosine receptor antagonists in the therapy of Parkinson's disease</span></div><div class="casAuthors">Armentero, Marie Therese; Pinna, Annalisa; Ferre, Sergi; Lanciego, Jose Luis; Mueller, Christa E.; Franco, Rafael</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">280-299</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Several selective antagonists for adenosine A2A receptors (A2AR) are currently under evaluation in clin. trials (phases I to III) to treat Parkinson's disease, and they will probably soon reach the market.  The usefulness of these antagonists has been deduced from studies demonstrating functional interactions between dopamine D2 and adenosine A2A receptors in the basal ganglia.  At present it is believed that A2AR antagonists can be used in combination with the dopamine precursor L-DOPA to minimize the motor symptoms of Parkinson's patients.  However, a considerable body of data indicates that in addn. to ameliorating motor symptoms, adenosine A2AR antagonists may also prevent neurodegeneration.  Despite these promising indications, one further issue must be considered in order to develop fully optimized antiparkinsonian drug therapy, namely the existence of (hetero)dimers/oligomers of G protein-coupled receptors, a topic that is currently the focus of intense debate within the scientific community.  Dopamine D2 receptors (D2Rs) expressed in the striatum are known to form heteromers with A2A adenosine receptors.  Thus, the development of heteromer-specific A2A receptor antagonists represents a promising strategy for the identification of more selective and safer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2pOYdmdfQMLVg90H21EOLACvtfcHk0liY_vp3O309bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtl2ntrzP&md5=39fa871338516f7574deb7845029e624</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2011.07.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2011.07.004%26sid%3Dliteratum%253Aachs%26aulast%3DArmentero%26aufirst%3DM.%2BT.%26aulast%3DPinna%26aufirst%3DA.%26aulast%3DFerr%25C3%25A9%26aufirst%3DS.%26aulast%3DLanciego%26aufirst%3DJ.%2BL.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26aulast%3DFranco%26aufirst%3DR.%26atitle%3DPast%252C%2520Present%2520and%2520Future%2520of%2520A2A%2520Adenosine%2520Receptor%2520Antagonists%2520in%2520the%2520Therapy%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DPharmacol.%2520Ther.%26date%3D2011%26volume%3D132%26spage%3D280%26epage%3D299%26doi%3D10.1016%2Fj.pharmthera.2011.07.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riederer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laux, G.</span></span> <span> </span><span class="NLM_article-title">MAO-Inhibitors in Parkinson’s Disease</span>. <i>Exp. Neurobiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.5607/en.2011.20.1.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.5607%2Fen.2011.20.1.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=22110357" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FlsFGlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=1-17&author=P.+Riedererauthor=G.+Laux&title=MAO-Inhibitors+in+Parkinson%E2%80%99s+Disease&doi=10.5607%2Fen.2011.20.1.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">MAO-inhibitors in Parkinson's Disease</span></div><div class="casAuthors">Riederer Peter; Laux Gerd</div><div class="citationInfo"><span class="NLM_cas:title">Experimental neurobiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Monoamine oxidase inhibitors (MAO-I) belong to the earliest drugs tried in Parkinson's disease (PD).  They have been used with or without levodopa (L-DOPA).  Non-selective MAO-I due to their side-effect/adverse reaction profile, like tranylcypromine have limited use in the treatment of depression in PD, while selective, reversible MAO-A inhibitors are recommended due to their easier clinical handling.  For the treatment of akinesia and motor fluctuations selective irreversible MAO-B inhibitors selegiline and rasagiline are recommended.  They are safe and well tolerated at the recommended daily doses.  Their main differences are related to (1) metabolism, (2) interaction with CYP-enzymes and (3) quantitative properties at the molecular biological/genetic level.  Rasagiline is more potent in clinical practise and has a hypothesis driven more favourable side effect/adverse reaction profile due to its metabolism to aminoindan.  Both selegiline and rasagiline have a neuroprotective and neurorestaurative potential.  A head-to head clinical trial would be of utmost interest from both the clinical outcome and a hypothesis-driven point of view.  Selegiline is available as tablet and melting tablet for PD and as transdermal selegiline for depression, while rasagiline is marketed as tablet for PD.  In general, the clinical use of MAO-I nowadays is underestimated.  There should be more efforts to evaluate their clinical potency as antidepressants and antidementive drugs in addition to the final proof of their disease-modifying potential.  In line with this are recent innovative developments of MAO-I plus inhibition of acetylcholine esterase for Alzheimer's disease as well as combined MAO-I and iron chelation for PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLFR15fhEdC_csfMJQC1ZdfW6udTcc2eal6-dP9EyUsrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FlsFGlsw%253D%253D&md5=83bd71bf598fe9e625291fa7c98ac0fd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.5607%2Fen.2011.20.1.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5607%252Fen.2011.20.1.1%26sid%3Dliteratum%253Aachs%26aulast%3DRiederer%26aufirst%3DP.%26aulast%3DLaux%26aufirst%3DG.%26atitle%3DMAO-Inhibitors%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DExp.%2520Neurobiol.%26date%3D2011%26volume%3D20%26spage%3D1%26epage%3D17%26doi%3D10.5607%2Fen.2011.20.1.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellis, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krüger, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Light, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlinchey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The ChEMBL Bioactivity Database: An Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D1083</span>– <span class="NLM_lpage">D1090</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fnar%2Fgkt1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24214965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D1083-D1090&author=A.+P.+Bentoauthor=A.+Gaultonauthor=A.+Herseyauthor=L.+J.+Bellisauthor=J.+Chambersauthor=M.+Daviesauthor=F.+A.+Kr%C3%BCgerauthor=Y.+Lightauthor=L.+Makauthor=S.+McGlincheyauthor=M.+Nowotkaauthor=G.+Papadatosauthor=R.+Santosauthor=J.+P.+Overington&title=The+ChEMBL+Bioactivity+Database%3A+An+Update&doi=10.1093%2Fnar%2Fgkt1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The ChEMBL bioactivity database: an update</span></div><div class="casAuthors">Bento, A. Patricia; Gaulton, Anna; Hersey, Anne; Bellis, Louisa J.; Chambers, Jon; Davies, Mark; Krueger, Felix A.; Light, Yvonne; Mak, Lora; McGlinchey, Shaun; Nowotka, Michal; Papadatos, George; Santos, Rita; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1083-D1090</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an open large-scale bioactivity database (https://www.ebi.ac.uk/chembl), previously described in the 2012 Nucleic Acids Research Database Issue.  Since then, a variety of new data sources and improvements in functionality have contributed to the growth and utility of the resource.  In particular, more comprehensive tracking of compds. from research stages through clin. development to market is provided through the inclusion of data from United States Adopted Name applications; a new richer data model for representing drug targets has been developed; and a no. of methods have been put in place to allow users to more easily identify reliable data.  Finally, access to ChEMBL is now available via a new Resource Description Framework format, in addn. to the web-based interface, data downloads and web services.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXb5k1IvJNaLVg90H21EOLACvtfcHk0liY_vp3O309bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslWl&md5=31b832d03d56ea3065d7aa29618362bc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1031%26sid%3Dliteratum%253Aachs%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DKr%25C3%25BCger%26aufirst%3DF.%2BA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMak%26aufirst%3DL.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520ChEMBL%2520Bioactivity%2520Database%253A%2520An%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD1083%26epage%3DD1090%26doi%3D10.1093%2Fnar%2Fgkt1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Molecular Docking Screening Using Agonist-Bound GPCR Structures: Probing the A<sub>2A</sub> Adenosine Receptor</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">563</span>, <span class="refDoi"> DOI: 10.1021/ci500639g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500639g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=550-563&author=D.+Rodr%C3%ADguezauthor=Z.+G.+Gaoauthor=S.+M.+Mossauthor=K.+A.+Jacobsonauthor=J.+Carlsson&title=Molecular+Docking+Screening+Using+Agonist-Bound+GPCR+Structures%3A+Probing+the+A2A+Adenosine+Receptor&doi=10.1021%2Fci500639g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screening Using Agonist-Bound GPCR Structures: Probing the A2A Adenosine Receptor</span></div><div class="casAuthors">Rodriguez, David; Gao, Zhang-Guo; Moss, Steven M.; Jacobson, Kenneth A.; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">550-563</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Crystal structures of G protein-coupled receptors (GPCRs) have recently revealed the mol. basis of ligand binding and activation, which has provided exciting opportunities for structure-based drug design.  The A2A adenosine receptor (A2AAR) is a promising therapeutic target for cardiovascular diseases, but progress in this area is limited by the lack of novel agonist scaffolds.  The authors carried out docking screens of 6.7 million com. available mols. against active-like conformations of the A2AAR to investigate whether these structures could guide the discovery of agonists.  Nine out of the 20 predicted agonists were confirmed to be A2AAR ligands, but none of these activated the ARs.  The difficulties in discovering AR agonists using structure-based methods originated from limited at.-level understanding of the activation mechanism and a chem. bias toward antagonists in the screened library.  In particular, the compn. of the screened library was found to strongly reduce the likelihood of identifying AR agonists, which reflected the high ligand complexity required for receptor activation.  Extension of this anal. to other pharmaceutically relevant GPCRs suggested that library screening may not be suitable for targets requiring a complex receptor-ligand interaction network.  The authors' results provide specific directions for the future development of novel A2AAR agonists and general strategies for structure-based drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp89cADSkVJDLVg90H21EOLACvtfcHk0linzqSjBYQ0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVWgtbs%253D&md5=0faec5c3d810a9ef32e5f374b6f9b447</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci500639g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500639g%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DMoss%26aufirst%3DS.%2BM.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DMolecular%2520Docking%2520Screening%2520Using%2520Agonist-Bound%2520GPCR%2520Structures%253A%2520Probing%2520the%2520A2A%2520Adenosine%2520Receptor%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D550%26epage%3D563%26doi%3D10.1021%2Fci500639g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M.
M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carchia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6582</span>– <span class="NLM_lpage">6594</span>, <span class="refDoi"> DOI: 10.1021/jm300687e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300687e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6582-6594&author=M.%0AM.+Mysingerauthor=M.+Carchiaauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Directory+of+Useful+Decoys%2C+Enhanced+%28DUD-E%29%3A+Better+Ligands+and+Decoys+for+Better+Benchmarking&doi=10.1021%2Fjm300687e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></div><div class="casAuthors">Mysinger, Michael M.; Carchia, Michael; Irwin, John. J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6582-6594</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A key metric to assess mol. docking remains ligand enrichment against challenging decoys.  Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged.  Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC.  To ensure chemotype diversity, we cluster each target's ligands by their Bemis-Murcko at. frameworks.  We add net charge to the matched physicochem. properties and include only the most dissimilar decoys, by topol., from the ligands.  An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands.  We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6.  The complete DUD-E benchmarking set is freely available at http://dude.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowquULXHTtJLVg90H21EOLACvtfcHk0linzqSjBYQ0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFaku7c%253D&md5=23ba4049662b388a880d4a48dde3769c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DCarchia%26aufirst%3DM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDirectory%2520of%2520Useful%2520Decoys%252C%2520Enhanced%2520%2528DUD-E%2529%253A%2520Better%2520Ligands%2520and%2520Decoys%2520for%2520Better%2520Benchmarking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6582%26epage%3D6594%26doi%3D10.1021%2Fjm300687e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mysinger, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1561</span>– <span class="NLM_lpage">1573</span>, <span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+Context-Dependent+Ligand+Desolvation+in+Molecular+Docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0linzqSjBYQ0hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520Context-Dependent%2520Ligand%2520Desolvation%2520in%2520Molecular%2520Docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rivara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piersanti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartoccini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamantini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pala, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccioni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasi, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borsini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarzia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minetti, P.</span></span> <span> </span><span class="NLM_article-title">Synthesis of (E)-8-(3-Chlorostyryl)caffeine Analogues Leading to 9-Deazaxanthine Derivatives as Dual A<sub>2A</sub> antagonists/MAO-B Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/jm301686s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301686s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVarug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1247-1261&author=S.+Rivaraauthor=G.+Piersantiauthor=F.+Bartocciniauthor=G.+Diamantiniauthor=D.+Palaauthor=T.+Riccioniauthor=M.+A.+Stasiauthor=W.+Cabriauthor=F.+Borsiniauthor=M.+Morauthor=G.+Tarziaauthor=P.+Minetti&title=Synthesis+of+%28E%29-8-%283-Chlorostyryl%29caffeine+Analogues+Leading+to+9-Deazaxanthine+Derivatives+as+Dual+A2A+antagonists%2FMAO-B+Inhibitors&doi=10.1021%2Fjm301686s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of (E)-8-(3-Chlorostyryl)caffeine Analogues Leading to 9-Deazaxanthine Derivatives as Dual A2A Antagonists/MAO-B Inhibitors</span></div><div class="casAuthors">Rivara, Silvia; Piersanti, Giovanni; Bartoccini, Francesca; Diamantini, Giuseppe; Pala, Daniele; Riccioni, Teresa; Stasi, Maria Antonietta; Cabri, Walter; Borsini, Franco; Mor, Marco; Tarzia, Giorgio; Minetti, Patrizia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A systematic modification of the caffeinyl core and substituents of the ref. compd. (E)-8-(3-chlorostyryl)caffeine led to the 9-deazaxanthine deriv. (E)-6-(4-chlorostyryl)-1,3,5,-trimethyl-1H-pyrrolo[3,2-d]pyrimidine-2,4-(3H,5H)-dione (I), which acts as a dual human A2a antagonist/MAO-B inhibitor (Ki(A2A) = 260 nM; IC50(MAO-B) = 200 nM; IC50(MAO-A) = 10 μM) and dose dependently counteracts haloperidol-induced catalepsy in mice from 30 mg/kg by the oral route.  The compd. is the best balanced A2A antagonist/MAO-B inhibitor reported to date, and it could be considered as a new lead in the field of anti-Parkinson's agents.  A no. of analogs of I were synthesized and qual. SARs are discussed.  Two analogs of I, namely II and III, inhibit MAO-B with IC50 of 68 and 48 nM, resp., being 5-7-fold more potent than the prototypical MAO-B inhibitor deprenyl (IC50 = 334 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCISycjnohtLVg90H21EOLACvtfcHk0lhND-qfqcCehw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVarug%253D%253D&md5=d03678b8923f861c8da65501f05f8e7e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm301686s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301686s%26sid%3Dliteratum%253Aachs%26aulast%3DRivara%26aufirst%3DS.%26aulast%3DPiersanti%26aufirst%3DG.%26aulast%3DBartoccini%26aufirst%3DF.%26aulast%3DDiamantini%26aufirst%3DG.%26aulast%3DPala%26aufirst%3DD.%26aulast%3DRiccioni%26aufirst%3DT.%26aulast%3DStasi%26aufirst%3DM.%2BA.%26aulast%3DCabri%26aufirst%3DW.%26aulast%3DBorsini%26aufirst%3DF.%26aulast%3DMor%26aufirst%3DM.%26aulast%3DTarzia%26aufirst%3DG.%26aulast%3DMinetti%26aufirst%3DP.%26atitle%3DSynthesis%2520of%2520%2528E%2529-8-%25283-Chlorostyryl%2529caffeine%2520Analogues%2520Leading%2520to%25209-Deazaxanthine%2520Derivatives%2520as%2520Dual%2520A2A%2520antagonists%252FMAO-B%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Fjm301686s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stößel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küppers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gütschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">Dual Targeting of Adenosine A<sub>2A</sub> Receptors and Monoamine Oxidase B by 4 H-3,1-Benzothiazin-4-Ones</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4580</span>– <span class="NLM_lpage">4596</span>, <span class="refDoi"> DOI: 10.1021/jm400336x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400336x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BC3srpsFOqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=4580-4596&author=A.+St%C3%B6%C3%9Felauthor=M.+Schlenkauthor=S.+Hinzauthor=P.+K%C3%BCppersauthor=J.+Heerauthor=M.+G%C3%BCtschowauthor=C.+E.+M%C3%BCller&title=Dual+Targeting+of+Adenosine+A2A+Receptors+and+Monoamine+Oxidase+B+by+4+H-3%2C1-Benzothiazin-4-Ones&doi=10.1021%2Fjm400336x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting of adenosine A(2A) receptors and monoamine oxidase B by 4H-3,1-benzothiazin-4-ones</span></div><div class="casAuthors">Stossel Anne; Schlenk Miriam; Hinz Sonja; Kuppers Petra; Heer Jag; Gutschow Michael; Muller Christa E</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4580-96</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Blockade of A2A adenosine receptors (A2AARs) and inhibition of monoamine oxidase B (MAO-B) in the brain are considered attractive strategies for the treatment of neurodegenerative diseases such as Parkinson's disease (PD).  In the present study, benzothiazinones, e.g., 2-(3-chlorophenoxy)-N-(4-oxo-4H-3,1-benzothiazin-2-yl)acetamide (13), were identified as a novel class of potent MAO-B inhibitors (IC50 human MAO-B: 1.63 nM).  Benzothiazinones with large substituents in the 2-position, e.g., methoxycinnamoylamino, phenylbutyrylamino, or chlorobenzylpiperazinylbenzamido residues (14, 17, 27, and 28), showed high affinity and selectivity for A2AARs (Ki human A2AAR: 39.5-69.5 nM).  By optimizing benzothiazinones for both targets, the first potent, dual-acting A2AAR/MAO-B inhibitors with a nonxanthine structure were developed.  The best derivative was N-(4-oxo-4H-3,1-benzothiazin-2-yl)-4-phenylbutanamide (17, Ki human A2A, 39.5 nM; IC50 human MAO-B, 34.9 nM; selective versus other AR subtypes and MAO-A), which inhibited A2AAR-induced cAMP accumulation and showed competitive, reversible MAO-B inhibition.  The new compounds may be useful tools for validating the A2AAR/MAO-B dual target approach in PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRoeXj8jrdZwDMsgLhIoICsfW6udTcc2eZQYZaJMBc_Y7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srpsFOqug%253D%253D&md5=3f5ec3d9dcfd282e2cf26f5efc7e8e86</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400336x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400336x%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25B6%25C3%259Fel%26aufirst%3DA.%26aulast%3DSchlenk%26aufirst%3DM.%26aulast%3DHinz%26aufirst%3DS.%26aulast%3DK%25C3%25BCppers%26aufirst%3DP.%26aulast%3DHeer%26aufirst%3DJ.%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3DDual%2520Targeting%2520of%2520Adenosine%2520A2A%2520Receptors%2520and%2520Monoamine%2520Oxidase%2520B%2520by%25204%2520H-3%252C1-Benzothiazin-4-Ones%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D4580%26epage%3D4596%26doi%3D10.1021%2Fjm400336x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akkari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunschweiger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Küppers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Köse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radjainia, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hockemeyer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drabczyńska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieć-Kononowicz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, C. E.</span></span> <span> </span><span class="NLM_article-title">1,3-Dialkyl-Substituted tetrahydropyrimido[1,2-F]purine-2,4-Diones as Multiple Target Drugs for the Potential Treatment of Neurodegenerative Diseases</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7435</span>– <span class="NLM_lpage">7452</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.09.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bmc.2013.09.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24139167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CrsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7435-7452&author=P.+Kochauthor=R.+Akkariauthor=A.+Brunschweigerauthor=T.+Borrmannauthor=M.+Schlenkauthor=P.+K%C3%BCppersauthor=M.+K%C3%B6seauthor=H.+Radjainiaauthor=J.+Hockemeyerauthor=A.+Drabczy%C5%84skaauthor=K.+Kie%C4%87-Kononowiczauthor=C.+E.+M%C3%BCller&title=1%2C3-Dialkyl-Substituted+tetrahydropyrimido%5B1%2C2-F%5Dpurine-2%2C4-Diones+as+Multiple+Target+Drugs+for+the+Potential+Treatment+of+Neurodegenerative+Diseases&doi=10.1016%2Fj.bmc.2013.09.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">1,3-Dialkyl-substituted tetrahydropyrimido[1,2-f]purine-2,4-diones as multiple target drugs for the potential treatment of neurodegenerative diseases</span></div><div class="casAuthors">Koch, Pierre; Akkari, Rhalid; Brunschweiger, Andreas; Borrmann, Thomas; Schlenk, Miriam; Kueppers, Petra; Koese, Meryem; Radjainia, Hamid; Hockemeyer, Joerg; Drabczynska, Anna; Kiec-Kononowicz, Katarzyna; Mueller, Christa E.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7435-7452</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Adenosine receptors and monoamine oxidases are drug targets for neurodegenerative diseases such as Parkinson's and Alzheimer's disease.  In the present study the authors prepd. a library of 55 mostly novel tetrahydropyrimido[2,1-f]purinediones I [R1 = Me, Et, n-Pr, CH≡CCH2; R2 = Me, Et, n-Pr; R3 = PhCH2, 4-FC6H4, 2-thienyl, etc.].  A synthetic strategy to obtain 3-propargyl-substituted tetrahydropyrimido[2,1-f]purinedione derivs. was developed.  The new compds. were evaluated for their interaction with all four adenosine receptor subtypes and for their ability to inhibit monoamine oxidases (MAO).  Introduction of mono- or di-chloro-substituted Ph, benzyl or phenethyl residues at N9 of the 1,3-di-Me series led to the discovery of a novel class of potent MAO-B inhibitors, the most potent compd. being I [R1 = Me; R2 = Me; R3 = 3,4-Cl2C6H3CH2] 9-(3,4-dichlorobenzyl)-1,3-dimethyl-6,7,8,9-tetrahydropyrimido[1,2-f]purine-2,4(1H,3H)-dione (IC50 human MAO-B: 0.0629 μM), which displayed high selectivity vs. the other investigated targets.  Potent dually active A1/A2A adenosine receptor antagonists were identified, for example, 9-benzyl-1-methyl-3-propargyl-6,7,8,9-tetrahydropyrimido[1,2-f]purine-2,4(1H,3H)dione I [R1 = CH≡CCH2; R2 = Me; R3 = PhCH2] (Ki, human receptors, A1: 0.249 μM, A2A: 0.253 μM).  Several compds. showed triple-target inhibition, the best compd. being I [R1 = CH≡CCH2; R2 = Me; R3 = 3-MeOC6H4CH2] (KiA1: 0.605 μM, KiA2A: 0.417 μM, IC50 MAO-B: 1.80 μM).  Compds. inhibiting several different targets involved in neurodegeneration may exhibit additive or even synergistic effects in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPERWbTreIyLVg90H21EOLACvtfcHk0ljJNnvpumBhjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CrsbfK&md5=27b1f8581d9f8c96d80d93a1117e6008</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.09.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.09.044%26sid%3Dliteratum%253Aachs%26aulast%3DKoch%26aufirst%3DP.%26aulast%3DAkkari%26aufirst%3DR.%26aulast%3DBrunschweiger%26aufirst%3DA.%26aulast%3DBorrmann%26aufirst%3DT.%26aulast%3DSchlenk%26aufirst%3DM.%26aulast%3DK%25C3%25BCppers%26aufirst%3DP.%26aulast%3DK%25C3%25B6se%26aufirst%3DM.%26aulast%3DRadjainia%26aufirst%3DH.%26aulast%3DHockemeyer%26aufirst%3DJ.%26aulast%3DDrabczy%25C5%2584ska%26aufirst%3DA.%26aulast%3DKie%25C4%2587-Kononowicz%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DC.%2BE.%26atitle%3D1%252C3-Dialkyl-Substituted%2520tetrahydropyrimido%255B1%252C2-F%255Dpurine-2%252C4-Diones%2520as%2520Multiple%2520Target%2520Drugs%2520for%2520the%2520Potential%2520Treatment%2520of%2520Neurodegenerative%2520Diseases%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7435%26epage%3D7452%26doi%3D10.1016%2Fj.bmc.2013.09.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mikkelsen, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langgård, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrøder, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreilgaard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jørgensen, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffon, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boffey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang-Andersen, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR Studies of Analogues of 4-(3,3-Dimethyl-Butyrylamino)-3,5-Difluoro-N-Thiazol-2-Yl-Benzamide (Lu AA41063) as Adenosine A<sub>2A</sub> Receptor Ligands with Improved Aqueous Solubility</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1212</span>– <span class="NLM_lpage">1216</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.01.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bmcl.2015.01.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=25701253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1212-1216&author=G.+K.+Mikkelsenauthor=M.+Langg%C3%A5rdauthor=T.+J.+Schr%C3%B8derauthor=M.+Kreilgaardauthor=E.+B.+J%C3%B8rgensenauthor=G.+Brandtauthor=Y.+Griffonauthor=R.+Boffeyauthor=B.+Bang-Andersen&title=Synthesis+and+SAR+Studies+of+Analogues+of+4-%283%2C3-Dimethyl-Butyrylamino%29-3%2C5-Difluoro-N-Thiazol-2-Yl-Benzamide+%28Lu+AA41063%29+as+Adenosine+A2A+Receptor+Ligands+with+Improved+Aqueous+Solubility&doi=10.1016%2Fj.bmcl.2015.01.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR studies of analogues of 4-(3,3-dimethylbutyrylamino)-3,5-difluoro-N-thiazol-2-yl-benzamide (Lu AA41063) as adenosine A2A receptor ligands with improved aqueous solubility</span></div><div class="casAuthors">Mikkelsen, Gitte Kobberoee; Langgaard, Morten; Schroeder, Tenna Juul; Kreilgaard, Mads; Joergensen, Erling B.; Brandt, Guillaume; Griffon, Yann; Boffey, Ray; Bang-Andersen, Benny</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1212-1216</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An adenosine A2A receptor antagonist may be useful for the treatment of Parkinson's disease.  Synthesis and structure-activity studies starting from 4-(3,3-dimethylbutyrylamino)-3,5-difluoro-N-thiazol-2-ylbenzamide (Lu AA41063) led to a novel series of human (h) A2A receptor antagonists with improved aq. soly.  Compd. I was identified as a key representative from the series, displaying submicromolar hA2A receptor affinity and excellent aq. soly.  Compd. I also displayed good in vitro pharmacokinetic properties and is considered a good starting point for further lead optimization toward hA2A receptor antagonists with improved druggability properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp481kuz7w0qrVg90H21EOLACvtfcHk0ljJNnvpumBhjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXisVSnsrc%253D&md5=b9e9970f6fbccd21ecfb7bb720a43758</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.01.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.01.062%26sid%3Dliteratum%253Aachs%26aulast%3DMikkelsen%26aufirst%3DG.%2BK.%26aulast%3DLangg%25C3%25A5rd%26aufirst%3DM.%26aulast%3DSchr%25C3%25B8der%26aufirst%3DT.%2BJ.%26aulast%3DKreilgaard%26aufirst%3DM.%26aulast%3DJ%25C3%25B8rgensen%26aufirst%3DE.%2BB.%26aulast%3DBrandt%26aufirst%3DG.%26aulast%3DGriffon%26aufirst%3DY.%26aulast%3DBoffey%26aufirst%3DR.%26aulast%3DBang-Andersen%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520SAR%2520Studies%2520of%2520Analogues%2520of%25204-%25283%252C3-Dimethyl-Butyrylamino%2529-3%252C5-Difluoro-N-Thiazol-2-Yl-Benzamide%2520%2528Lu%2520AA41063%2529%2520as%2520Adenosine%2520A2A%2520Receptor%2520Ligands%2520with%2520Improved%2520Aqueous%2520Solubility%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1212%26epage%3D1216%26doi%3D10.1016%2Fj.bmcl.2015.01.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenstein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurrell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the Adenosine A<sub>2A</sub> Receptor in Complex with ZM241385 and the Xanthines XAC and Caffeine</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1283</span>– <span class="NLM_lpage">1293</span>, <span class="refDoi"> DOI: 10.1016/j.str.2011.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.str.2011.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21885291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=1283-1293&author=A.+S.+Dor%C3%A9author=N.+Robertsonauthor=J.+C.+Erreyauthor=I.+Ngauthor=K.+Hollensteinauthor=B.+Tehanauthor=E.+Hurrellauthor=K.+Bennettauthor=M.+Congreveauthor=F.+Magnaniauthor=C.+G.+Tateauthor=M.+Weirauthor=F.+H.+Marshall&title=Structure+of+the+Adenosine+A2A+Receptor+in+Complex+with+ZM241385+and+the+Xanthines+XAC+and+Caffeine&doi=10.1016%2Fj.str.2011.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Adenosine A2A Receptor in Complex with ZM241385 and the Xanthines XAC and Caffeine</span></div><div class="casAuthors">Dore, Andrew S.; Robertson, Nathan; Errey, James C.; Ng, Irene; Hollenstein, Kaspar; Tehan, Ben; Hurrell, Edward; Bennett, Kirstie; Congreve, Miles; Magnani, Francesca; Tate, Christopher G.; Weir, Malcolm; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1283-1293</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylxanthines, including caffeine and theophylline, are among the most widely consumed stimulant drugs in the world.  These effects are mediated primarily via blockade of adenosine receptors.  Xanthine analogs with improved properties have been developed as potential treatments for diseases such as Parkinson's disease.  Here we report the structures of a thermostabilized adenosine A2A receptor in complex with the xanthines xanthine amine congener and caffeine, as well as the A2A selective inverse agonist ZM241385.  The receptor is crystd. in the inactive state conformation as defined by the presence of a salt bridge known as the ionic lock.  The complete third intracellular loop, responsible for G protein coupling, is visible consisting of extended helixes 5 and 6.  The structures provide new insight into the features that define the ligand binding pocket of the adenosine receptor for ligands of diverse chemotypes as well as the cytoplasmic regions that interact with signal transduction proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWsexwFu6LD7Vg90H21EOLACvtfcHk0ljJNnvpumBhjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaru7fE&md5=a25db14aa70c2b42b9c3eb98c3c16bae</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2011.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2011.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DRobertson%26aufirst%3DN.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DNg%26aufirst%3DI.%26aulast%3DHollenstein%26aufirst%3DK.%26aulast%3DTehan%26aufirst%3DB.%26aulast%3DHurrell%26aufirst%3DE.%26aulast%3DBennett%26aufirst%3DK.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMagnani%26aufirst%3DF.%26aulast%3DTate%26aufirst%3DC.%2BG.%26aulast%3DWeir%26aufirst%3DM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DStructure%2520of%2520the%2520Adenosine%2520A2A%2520Receptor%2520in%2520Complex%2520with%2520ZM241385%2520and%2520the%2520Xanthines%2520XAC%2520and%2520Caffeine%26jtitle%3DStructure%26date%3D2011%26volume%3D19%26spage%3D1283%26epage%3D1293%26doi%3D10.1016%2Fj.str.2011.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Binda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caccia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvati, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattevi, A.</span></span> <span> </span><span class="NLM_article-title">Structures of Human Monoamine Oxidase B Complexes with Selective Noncovalent Inhibitors: Safinamide and Coumarin Analogs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5848</span>– <span class="NLM_lpage">5852</span>, <span class="refDoi"> DOI: 10.1021/jm070677y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070677y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFWgurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5848-5852&author=C.+Bindaauthor=J.+Wangauthor=L.+Pisaniauthor=C.+Cacciaauthor=A.+Carottiauthor=P.+Salvatiauthor=D.+E.+Edmondsonauthor=A.+Mattevi&title=Structures+of+Human+Monoamine+Oxidase+B+Complexes+with+Selective+Noncovalent+Inhibitors%3A+Safinamide+and+Coumarin+Analogs&doi=10.1021%2Fjm070677y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of Human Monoamine Oxidase B Complexes with Selective Noncovalent Inhibitors: Safinamide and Coumarin Analogs</span></div><div class="casAuthors">Binda, Claudia; Wang, Jin; Pisani, Leonardo; Caccia, Carla; Carotti, Angelo; Salvati, Patricia; Edmondson, Dale E.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5848-5852</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structures of human monoamine oxidase B (MAO B) in complex with safinamide and two coumarin derivs., all sharing a common benzyloxy substituent, were detd. by x-ray crystallog.  These compds. competitively inhibit MAO B with Ki values in the 0.1-0.5 μM range that are 30-700-fold lower than those obsd. with MAO A.  The inhibitors bind noncovalently to MAO B, occupying both the entrance and the substrate cavities and showing a similarly oriented benzyloxy substituent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBPIubufo89bVg90H21EOLACvtfcHk0lgTtcAF-51h5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFWgurjI&md5=448f65696f4870e2301f46a6f015e35c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm070677y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070677y%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DPisani%26aufirst%3DL.%26aulast%3DCaccia%26aufirst%3DC.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DSalvati%26aufirst%3DP.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructures%2520of%2520Human%2520Monoamine%2520Oxidase%2520B%2520Complexes%2520with%2520Selective%2520Noncovalent%2520Inhibitors%253A%2520Safinamide%2520and%2520Coumarin%2520Analogs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5848%26epage%3D5852%26doi%3D10.1021%2Fjm070677y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS Algorithm for Detection of Structurally Similar Protein Binding Sites by Local Structural Alignment</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1168</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btq100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fbioinformatics%2Fbtq100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20305268" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2010&pages=1160-1168&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS+Algorithm+for+Detection+of+Structurally+Similar+Protein+Binding+Sites+by+Local+Structural+Alignment&doi=10.1093%2Fbioinformatics%2Fbtq100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS algorithm for detection of structurally similar protein binding sites by local structural alignment</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1160-1168</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Exploitation of locally similar 3D patterns of physicochem. properties on the surface of a protein for detection of binding sites that may lack sequence and global structural conservation.  An algorithm, ProBiS is described that detects structurally similar sites on protein surfaces by local surface structure alignment.  It compares the query protein to members of a database of protein 3D structures and detects with sub-residue precision, structurally similar sites as patterns of physicochem. properties on the protein surface.  Using an efficient max. clique algorithm, the program identifies proteins that share local structural similarities with the query protein and generates structure-based alignments of these proteins with the query.  Structural similarity scores are calcd. for the query protein's surface residues, and are expressed as different colors on the query protein surface.  The algorithm has been used successfully for the detection of protein-protein, protein-small ligand and protein-DNA binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGproKvL-3-CSLVg90H21EOLACvtfcHk0lgTtcAF-51h5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1equ74%253D&md5=7ff0eecdf0ea5f7d9b82fd721ca88156</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtq100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtq100%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%2520Algorithm%2520for%2520Detection%2520of%2520Structurally%2520Similar%2520Protein%2520Binding%2520Sites%2520by%2520Local%2520Structural%2520Alignment%26jtitle%3DBioinformatics%26date%3D2010%26volume%3D26%26spage%3D1160%26epage%3D1168%26doi%3D10.1093%2Fbioinformatics%2Fbtq100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">ZINC – A Free Database of Commercially Available Compounds for Virtual Screening</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1021/ci049714+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC+%E2%80%93+A+Free+Database+of+Commercially+Available+Compounds+for+Virtual+Screening&doi=10.1021%2Fci049714%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC--a free database of commercially available compounds for virtual screening</span></div><div class="casAuthors">Irwin John J; Shoichet Brian K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-82</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">A critical barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compounds.  We have therefore prepared a library of 727,842 molecules, each with 3D structure, using catalogs of compounds from vendors (the size of this library continues to grow).  The molecules have been assigned biologically relevant protonation states and are annotated with properties such as molecular weight, calculated LogP, and number of rotatable bonds.  Each molecule in the library contains vendor and purchasing information and is ready for docking using a number of popular docking programs.  Within certain limits, the molecules are prepared in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the molecules.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a molecular drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own molecules by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQVi_-excgTX-_Q9URG6Dx8fW6udTcc2eboJyvicfE43bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2M%252FktVKrtA%253D%253D&md5=9f2e3ec06d53276884fc17b9aad47e61</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%2520%25E2%2580%2593%2520A%2520Free%2520Database%2520of%2520Commercially%2520Available%2520Compounds%2520for%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182%26doi%3D10.1021%2Fci049714%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plückthun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kummer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Ligand Discovery for a Peptide-Binding GPCR by Structure-Based Screening of Fragment- and Lead-Like Chemical Libraries</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">735</span>– <span class="NLM_lpage">745</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFOitbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=735-745&author=A.+Ranganathanauthor=P.+Heineauthor=A.+Rudlingauthor=A.+Pl%C3%BCckthunauthor=L.+Kummerauthor=J.+Carlsson&title=Ligand+Discovery+for+a+Peptide-Binding+GPCR+by+Structure-Based+Screening+of+Fragment-+and+Lead-Like+Chemical+Libraries&doi=10.1021%2Facschembio.6b00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Discovery for a Peptide-Binding GPCR by Structure-Based Screening of Fragment- and Lead-Like Chemical Libraries</span></div><div class="casAuthors">Ranganathan, Anirudh; Heine, Philipp; Rudling, Axel; Pluckthun, Andreas; Kummer, Lutz; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">735-745</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide-recognizing G protein-coupled receptors (GPCRs) are promising therapeutic targets, but often resist drug discovery efforts.  Detn. of crystal structures for peptide-binding GPCRs has provided opportunities to explore structure-based methods in lead development.  Mol. docking screens of two chem. libraries, contg. either fragment- or lead-like compds., against a Neurotensin receptor 1 crystal structure allowed for a comparison between different drug development strategies for peptide-binding GPCRs.  A total of 2.3 million mols. were screened computationally and 25 fragments and 27 leads that were top-ranked in each library were selected for exptl. evaluation.  Of these, eight fragments and five leads were confirmed as ligands by surface plasmon resonance.  The hit rate for the fragment screen (32%) was thus higher than for the lead-like library (19%), but the affinities of the fragments were ∼100-fold lower.  Both screens returned unique scaffolds and demonstrated that a crystal structure of a stabilized peptide-binding GPCR can guide the discovery of small-mol. agonists.  The complementary advantages of exploring fragment and lead-like chem. space suggest that these strategies should be applied synergistically in structure-based screens against challenging GPCR targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMYKg8sFuRtbVg90H21EOLACvtfcHk0lhFzoQdG64gQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFOitbfK&md5=b1b72cda63503ab45ad0bd57cd70943f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00646%26sid%3Dliteratum%253Aachs%26aulast%3DRanganathan%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DP.%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DPl%25C3%25BCckthun%26aufirst%3DA.%26aulast%3DKummer%26aufirst%3DL.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DLigand%2520Discovery%2520for%2520a%2520Peptide-Binding%2520GPCR%2520by%2520Structure-Based%2520Screening%2520of%2520Fragment-%2520and%2520Lead-Like%2520Chemical%2520Libraries%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D735%26epage%3D745%26doi%3D10.1021%2Facschembio.6b00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setola, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlessinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Ligand Discovery from a Dopamine D<sub>3</sub> Receptor Homology Model and Crystal Structure</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1038/nchembio.662</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+Discovery+from+a+Dopamine+D3+Receptor+Homology+Model+and+Crystal+Structure&doi=10.1038%2Fnchembio.662"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lhFzoQdG64gQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520Discovery%2520from%2520a%2520Dopamine%2520D3%2520Receptor%2520Homology%2520Model%2520and%2520Crystal%2520Structure%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778%26doi%3D10.1038%2Fnchembio.662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The Role of Ligand Efficiency Metrics in Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+Role+of+Ligand+Efficiency+Metrics+in+Drug+Discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lhFzoQdG64gQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520Role%2520of%2520Ligand%2520Efficiency%2520Metrics%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ballesteros, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstein, H.</span></span> <span> </span><span class="NLM_article-title">Integrated Methods for the Construction of Three-Dimensional Models and Computational Probing of Structure-Function Relations in G Protein-Coupled Receptors</span>. <i>Methods Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">428</span>, <span class="refDoi"> DOI: 10.1016/S1043-9471(05)80049-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2FS1043-9471%2805%2980049-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1995&pages=366-428&author=J.+A.+Ballesterosauthor=H.+Weinstein&title=Integrated+Methods+for+the+Construction+of+Three-Dimensional+Models+and+Computational+Probing+of+Structure-Function+Relations+in+G+Protein-Coupled+Receptors&doi=10.1016%2FS1043-9471%2805%2980049-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated methods for the construction of three-dimensional models and computational probing of structure-function relations in G protein-coupled receptors</span></div><div class="casAuthors">Ballesteros, Juan A.; Weinstein, Harel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Neurosciences</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">366-428</span>CODEN:
                <span class="NLM_cas:coden">MENEE5</span>;
        ISSN:<span class="NLM_cas:issn">1043-9471</span>.
    </div><div class="casAbstract">A review, with 135 refs., on approaches that can be used to resolve the apparent ambiguities that burden the pharmacol. testing of G protein-coupled receptor (GPCR) models, based on the integration of structural information about the receptor, about mutants, and about the changes induced by ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfENoRspqa7LVg90H21EOLACvtfcHk0lj7kJlN4EqWCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1Kmtrk%253D&md5=d3f99e9d07cea245a770f3f099bc8cc7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS1043-9471%2805%2980049-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1043-9471%252805%252980049-7%26sid%3Dliteratum%253Aachs%26aulast%3DBallesteros%26aufirst%3DJ.%2BA.%26aulast%3DWeinstein%26aufirst%3DH.%26atitle%3DIntegrated%2520Methods%2520for%2520the%2520Construction%2520of%2520Three-Dimensional%2520Models%2520and%2520Computational%2520Probing%2520of%2520Structure-Function%2520Relations%2520in%2520G%2520Protein-Coupled%2520Receptors%26jtitle%3DMethods%2520Neurosci.%26date%3D1995%26volume%3D25%26spage%3D366%26epage%3D428%26doi%3D10.1016%2FS1043-9471%2805%2980049-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Similarity-Potency Trees: A Method to Search for SAR Information in Compound Data Sets and Derive SAR Rules</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1395</span>– <span class="NLM_lpage">1409</span>, <span class="refDoi"> DOI: 10.1021/ci100197b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100197b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1395-1409&author=M.+Wawerauthor=J.+Bajorath&title=Similarity-Potency+Trees%3A+A+Method+to+Search+for+SAR+Information+in+Compound+Data+Sets+and+Derive+SAR+Rules&doi=10.1021%2Fci100197b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity-Potency Trees: A Method to Search for SAR Information in Compound Data Sets and Derive SAR Rules</span></div><div class="casAuthors">Wawer, Mathias; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1395-1409</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An intuitive and generally applicable anal. method, termed similarity-potency tree (SPT), is introduced to mine structure-activity relation (SAR) information in compd. data sets of any source.  Only compd. potency values and nearest-neighbor similarity relationships are considered.  Rather than analyzing a data set as a whole, in part overlapping compd. neighborhoods are systematically generated and represented as SPTs.  This local anal. scheme simplifies the evaluation of SAR information and SPTs of high SAR information content are easily identified.  By inspecting only a limited no. of compd. neighborhoods, it is also straightforward to det. whether data sets contain only little or no interpretable SAR information.  Interactive anal. of SPTs is facilitated by reading the trees in two directions, which makes it possible to ext. SAR rules, if available, in a consistent manner.  The simplicity and interpretability of the data structure and the ease of calcn. are characteristic features of this approach.  We apply the methodol. to high-throughput screening and lead optimization data sets, compare the approach to std. clustering techniques, illustrate how SAR rules are derived, and provide some practical guidance how to best utilize the methodol.  The SPT program is made freely available to the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocIJW9gu2adrVg90H21EOLACvtfcHk0lj7kJlN4EqWCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCiurc%253D&md5=7d4609c7ca3a695c0658233ca7867616</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci100197b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100197b%26sid%3Dliteratum%253Aachs%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSimilarity-Potency%2520Trees%253A%2520A%2520Method%2520to%2520Search%2520for%2520SAR%2520Information%2520in%2520Compound%2520Data%2520Sets%2520and%2520Derive%2520SAR%2520Rules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1395%26epage%3D1409%26doi%3D10.1021%2Fci100197b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurrell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Drug Design of Chromone Antagonists of the Adenosine A<sub>2A</sub> Receptor</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">571</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1039/C3MD00338H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1039%2FC3MD00338H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXms1WqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=571-575&author=S.+P.+Andrewsauthor=J.+S.+Masonauthor=E.+Hurrellauthor=M.+Congreve&title=Structure-Based+Drug+Design+of+Chromone+Antagonists+of+the+Adenosine+A2A+Receptor&doi=10.1039%2FC3MD00338H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design of chromone antagonists of the adenosine A2A receptor</span></div><div class="casAuthors">Andrews, Stephen P.; Mason, Jonathan S.; Hurrell, Edward; Congreve, Miles</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">571-575</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The structure-guided optimization of a hit series of chromone derivs., previously identified using virtual screening of homol. models of the adenosine A2A receptor, has led to the discovery of potent, selective and ligand efficient antagonists.  Lipophilic hotspots and calcd. water networks were modelled within the receptor binding site to facilitate rational ligand design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplTU8yhstYi7Vg90H21EOLACvtfcHk0lj7kJlN4EqWCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXms1WqsLw%253D&md5=ab3c3f180fe8f5f3a87f54035f92593d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC3MD00338H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC3MD00338H%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHurrell%26aufirst%3DE.%26aulast%3DCongreve%26aufirst%3DM.%26atitle%3DStructure-Based%2520Drug%2520Design%2520of%2520Chromone%2520Antagonists%2520of%2520the%2520Adenosine%2520A2A%2520Receptor%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D571%26epage%3D575%26doi%3D10.1039%2FC3MD00338H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matricon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warnick, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambertucci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezzati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaiteh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dal Ben, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Fragment Optimization for GPCRs by Molecular Dynamics Free Energy Calculations: Probing Druggable Subpockets of the A<sub>2A</sub> Adenosine Receptor Binding Site</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">6398</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-04905-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fs41598-017-04905-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=28743961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A280%3ADC%252BC1cfhtFKktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=6398&author=P.+Matriconauthor=A.+Ranganathanauthor=E.+Warnickauthor=Z.-G.+Gaoauthor=A.+Rudlingauthor=C.+Lambertucciauthor=G.+Marucciauthor=A.+Ezzatiauthor=M.+Jaitehauthor=D.+Dal+Benauthor=K.+A.+Jacobsonauthor=J.+Carlsson&title=Fragment+Optimization+for+GPCRs+by+Molecular+Dynamics+Free+Energy+Calculations%3A+Probing+Druggable+Subpockets+of+the+A2A+Adenosine+Receptor+Binding+Site&doi=10.1038%2Fs41598-017-04905-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment optimization for GPCRs by molecular dynamics free energy calculations: Probing druggable subpockets of the A 2A adenosine receptor binding site</span></div><div class="casAuthors">Matricon Pierre; Jaiteh Mariama; Carlsson Jens; Ranganathan Anirudh; Rudling Axel; Ezzati Aitakin; Warnick Eugene; Gao Zhan-Guo; Jacobson Kenneth A; Lambertucci Catia; Marucci Gabriella; Dal Ben Diego</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6398</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based lead discovery is becoming an increasingly popular strategy for drug discovery.  Fragment screening identifies weakly binding compounds that require optimization to become high-affinity leads.  As design of leads from fragments is challenging, reliable computational methods to guide optimization would be invaluable.  We evaluated using molecular dynamics simulations and the free energy perturbation method (MD/FEP) in fragment optimization for the A2A adenosine receptor, a pharmaceutically relevant G protein-coupled receptor.  Optimization of fragments exploring two binding site subpockets was probed by calculating relative binding affinities for 23 adenine derivatives, resulting in strong agreement with experimental data (R(2) = 0.78).  The predictive power of MD/FEP was significantly better than that of an empirical scoring function.  We also demonstrated the potential of the MD/FEP to assess multiple binding modes and to tailor the thermodynamic profile of ligands during optimization.  Finally, MD/FEP was applied prospectively to optimize three nonpurine fragments, and predictions for 12 compounds were evaluated experimentally.  The direction of the change in binding affinity was correctly predicted in a majority of the cases, and agreement with experiment could be improved with rigorous parameter derivation.  The results suggest that MD/FEP will become a powerful tool in structure-driven optimization of fragments to lead candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS_3RY8BzBzWXWEEzDrymxmfW6udTcc2eaR3jzv9fLldbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfhtFKktQ%253D%253D&md5=05f0210d0cbe5f4cd58677b8747dffa0</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-04905-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-04905-0%26sid%3Dliteratum%253Aachs%26aulast%3DMatricon%26aufirst%3DP.%26aulast%3DRanganathan%26aufirst%3DA.%26aulast%3DWarnick%26aufirst%3DE.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DRudling%26aufirst%3DA.%26aulast%3DLambertucci%26aufirst%3DC.%26aulast%3DMarucci%26aufirst%3DG.%26aulast%3DEzzati%26aufirst%3DA.%26aulast%3DJaiteh%26aufirst%3DM.%26aulast%3DDal%2BBen%26aufirst%3DD.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DFragment%2520Optimization%2520for%2520GPCRs%2520by%2520Molecular%2520Dynamics%2520Free%2520Energy%2520Calculations%253A%2520Probing%2520Druggable%2520Subpockets%2520of%2520the%2520A2A%2520Adenosine%2520Receptor%2520Binding%2520Site%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D6398%26doi%3D10.1038%2Fs41598-017-04905-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgN_UQSwlZHvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">A Detergent-Based Assay for the Detection of Promiscuous Inhibitors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">550</span>– <span class="NLM_lpage">553</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1038%2Fnprot.2006.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=17191086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=550-553&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=A+Detergent-Based+Assay+for+the+Detection+of+Promiscuous+Inhibitors&doi=10.1038%2Fnprot.2006.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A detergent-based assay for the detection of promiscuous inhibitors</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">550-553</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">At micromolar concns., many small mols. self-assoc. into colloidal aggregates that non-specifically inhibit enzymes and other proteins.  Here the authors describe a protocol for identifying aggregate-based inhibitors and distinguishing them from small mols. that inhibit via specific mechanisms.  As a convenient proxy for promiscuous, aggregate-based inhibition, the authors monitor inhibition of β-lactamase in the absence and presence of detergent.  Inhibition that is attenuated in the presence of detergent is characteristic of an aggregate-based mechanism.  In the 96-well-format assay described here, about 200 mols. can be tested, in duplicate, per h for detergent-dependent sensitivity.  Furthermore, the authors also describe simple expts. that can offer addnl. confirmation of aggregate-based inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplGQSBvMgM-7Vg90H21EOLACvtfcHk0lgN_UQSwlZHvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitLjO&md5=8aa7562b5f69c896af7e133ecc35e6de</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.77%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520Detergent-Based%2520Assay%2520for%2520the%2520Detection%2520of%2520Promiscuous%2520Inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D550%26epage%3D553%26doi%3D10.1038%2Fnprot.2006.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">PubChem BioAssay: 2017 Update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D955</span>– <span class="NLM_lpage">D963</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fnar%2Fgkw1118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=27899599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D955-D963&author=Y.+Wangauthor=S.+H.+Bryantauthor=T.+Chengauthor=J.+Wangauthor=A.+Gindulyteauthor=B.+A.+Shoemakerauthor=P.+A.+Thiessenauthor=S.+Heauthor=J.+Zhang&title=PubChem+BioAssay%3A+2017+Update&doi=10.1093%2Fnar%2Fgkw1118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem BioAssay: 2017 update</span></div><div class="casAuthors">Wang, Yanli; Bryant, Stephen H.; Cheng, Tiejun; Wang, Jiyao; Gindulyte, Asta; Shoemaker, Benjamin A.; Thiessen, Paul A.; He, Siqian; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D955-D963</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem's BioAssay database (https://pubchem.ncbi.nlm.nih.gov) has served as a public repository for small-mol. and RNAi screening data since 2004 providing open access of its data content to the community.  PubChem accepts data submission from worldwide researchers at academia, industry and government agencies.  PubChem also collaborates with other chem. biol. database stakeholders with data exchange.  With over a decade's development effort, it becomes an important information resource supporting drug discovery and chem. biol. research.  To facilitate data discovery, PubChem is integrated with all other databases at NCBI.  In this work, we provide an update for the Pub- Chem BioAssay database describing several recent development including added sources of research data, redesigned BioAssay record page, new BioAssay classification browser and new features in the Upload system facilitating data sharing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptFOpprG7GhLVg90H21EOLACvtfcHk0lgN_UQSwlZHvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhsbw%253D&md5=4043316b6be487277f360fb4cc2beabb</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1118%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DPubChem%2520BioAssay%253A%25202017%2520Update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD955%26epage%3DD963%26doi%3D10.1093%2Fnar%2Fgkw1118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Docking Screens for Novel Ligands Conferring New Biology</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4103</span>– <span class="NLM_lpage">4120</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b02008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b02008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Siu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4103-4120&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Docking+Screens+for+Novel+Ligands+Conferring+New+Biology&doi=10.1021%2Facs.jmedchem.5b02008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Screens for Novel Ligands Conferring New Biology</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4103-4120</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is now plausible to dock libraries of 10 million mols. against targets over several days or weeks.  When the mols. screened are com. available, they may be rapidly tested to find new leads.  Although docking retains important liabilities (it cannot calc. affinities accurately nor even reliably rank order high-scoring mols.), it can often can distinguish likely from unlikely ligands, often with hit rates above 10%.  Here we summarize the improvements in libraries, target quality, and methods that have supported these advances, and the open access resources that make docking accessible.  Recent docking screens for new ligands are sketched, as are the binding, crystallog., and in vivo assays that support them.  Like any technique, controls are crucial, and key exptl. ones are reviewed.  With such controls, docking campaigns can find ligands with new chemotypes, often revealing the new biol. that may be docking's greatest impact over the next few years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCmaLVTnkkoLVg90H21EOLACvtfcHk0likpp8FRjcRaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Siu7o%253D&md5=5cdfdd9f2d6b521398b151adb3ef145a</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520Screens%2520for%2520Novel%2520Ligands%2520Conferring%2520New%2520Biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4103%26epage%3D4120%26doi%3D10.1021%2Facs.jmedchem.5b02008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of GPCR Ligands by Molecular Docking Screening: Novel Opportunities Provided by Crystal Structures</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2484</span>– <span class="NLM_lpage">2503</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150701112853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.2174%2F1568026615666150701112853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=26126906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFansrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=2484-2503&author=D.+Rodr%C3%ADguezauthor=A.+Ranganathanauthor=J.+Carlsson&title=Discovery+of+GPCR+Ligands+by+Molecular+Docking+Screening%3A+Novel+Opportunities+Provided+by+Crystal+Structures&doi=10.2174%2F1568026615666150701112853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of GPCR Ligands by Molecular Docking Screening: Novel Opportunities Provided by Crystal Structures</span></div><div class="casAuthors">Rodriguez, David; Ranganathan, Anirudh; Carlsson, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2484-2503</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) constitute the largest group of human membrane proteins and have received significant attention in drug discovery for their important roles in physiol. processes.  Drug development for GPCRs has been remarkably successful and several of the most profitable pharmaceuticals on the market target members of this superfamily.  Breakthroughs in structural biol. for GPCRs have revealed how their binding sites recognize extracellular mols. at the at. level.  High-resoln. crystal structures of GPCR-drug complexes capturing different receptor conformations are now available, which have provided insights into how ligands stabilize different functional states.  Recently, the basis for subtype selectivity and novel allosteric binding sites has also been revealed by crystal structures.  These accomplishments provide exciting opportunities to identify novel GPCR ligands using <i>in silico</i> structure-based methods such as mol. docking.  Increased computational power now enables docking screens of large chem. libraries to identify mols. that complement GPCR binding sites, which may provide possibilities to identify ligands with tailored pharmacol. properties.  This review focuses on prospective docking screens against GPCRs and how this technique can be used to identify lead candidates with specific signaling or selectivity profiles.  The current state of this field suggests that mol. docking, in combination with further understanding of GPCR signaling, will play an important role in future drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxcu4b1g3uerVg90H21EOLACvtfcHk0likpp8FRjcRaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFansrjN&md5=31a75abcc24bf453ad155903a7454d3e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150701112853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150701112853%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DD.%26aulast%3DRanganathan%26aufirst%3DA.%26aulast%3DCarlsson%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520GPCR%2520Ligands%2520by%2520Molecular%2520Docking%2520Screening%253A%2520Novel%2520Opportunities%2520Provided%2520by%2520Crystal%2520Structures%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2015%26volume%3D15%26spage%3D2484%26epage%3D2503%26doi%3D10.2174%2F1568026615666150701112853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, K. A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of A<sub>2A</sub> Adenosine Receptor Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">3748</span>– <span class="NLM_lpage">3755</span>, <span class="refDoi"> DOI: 10.1021/jm100240h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.-G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-Based+Discovery+of+A2A+Adenosine+Receptor+Ligands&doi=10.1021%2Fjm100240h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0likpp8FRjcRaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.-G.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-Based%2520Discovery%2520of%2520A2A%2520Adenosine%2520Receptor%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755%26doi%3D10.1021%2Fjm100240h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaakola, V.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufareva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of Novel Chemotypes for Adenosine A<sub>2A</sub> Receptor Antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1799</span>– <span class="NLM_lpage">1809</span>, <span class="refDoi"> DOI: 10.1021/jm901647p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901647p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptlKqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1799-1809&author=V.+Katritchauthor=V.-P.+Jaakolaauthor=J.+R.+Laneauthor=J.+Linauthor=A.+P.+IJzermanauthor=M.+Yeagerauthor=I.+Kufarevaauthor=R.+C.+Stevensauthor=R.+Abagyan&title=Structure-Based+Discovery+of+Novel+Chemotypes+for+Adenosine+A2A+Receptor+Antagonists&doi=10.1021%2Fjm901647p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of Novel Chemotypes for Adenosine A2A Receptor Antagonists</span></div><div class="casAuthors">Katritch, Vsevolod; Jaakola, Veli-Pekka; Lane, J. Robert; Lin, Judy; IJzerman, Adriaan P.; Yeager, Mark; Kufareva, Irina; Stevens, Raymond C.; Abagyan, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1799-1809</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent progress in crystallog. of G-protein coupled receptors opens an unprecedented venue for structure-based GPCR drug discovery.  To test efficiency of the structure-based approach, we performed mol. docking and virtual ligand screening (VLS) of more than 4 million com. available "drug-like" and "lead-like" compds. against the A2AAR 2.6 Å resoln. crystal structure.  Out of 56 high ranking compds. tested in A2AAR binding assays, 23 showed affinities under 10 μM, 11 of those had sub-μM affinities and two compds. had affinities under 60 nM.  The identified hits represent at least 9 different chem. scaffolds and are characterized by very high ligand efficiency (0.3-0.5 kcal/mol per heavy atom).  Significant A2AAR antagonist activities were confirmed for 10 out of 13 ligands tested in functional assays.  High success rate, novelty, and diversity of the chem. scaffolds and strong ligand efficiency of the A2AAR antagonists identified in this study suggest practical applicability of receptor-based VLS in GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX-YAwoEVzEbVg90H21EOLACvtfcHk0ljvW8tRKg9wXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptlKqsw%253D%253D&md5=5a54df43f6edd20d83e7e5942e2f9811</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm901647p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901647p%26sid%3Dliteratum%253Aachs%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DJaakola%26aufirst%3DV.-P.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DYeager%26aufirst%3DM.%26aulast%3DKufareva%26aufirst%3DI.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Novel%2520Chemotypes%2520for%2520Adenosine%2520A2A%2520Receptor%2520Antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1799%26epage%3D1809%26doi%3D10.1021%2Fjm901647p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geldenhuys, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvesh, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Schyf, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, R. T.</span></span> <span> </span><span class="NLM_article-title">Identification of Novel Monoamine Oxidase B Inhibitors by Structure-Based Virtual Screening</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5295</span>– <span class="NLM_lpage">5298</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.06.128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bmcl.2010.06.128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20650633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVehtr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=5295-5298&author=W.+J.+Geldenhuysauthor=A.+S.+Darveshauthor=M.+O.+Funkauthor=C.+J.+Van+der+Schyfauthor=R.+T.+Carroll&title=Identification+of+Novel+Monoamine+Oxidase+B+Inhibitors+by+Structure-Based+Virtual+Screening&doi=10.1016%2Fj.bmcl.2010.06.128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel monoamine oxidase B inhibitors by structure-based virtual screening</span></div><div class="casAuthors">Geldenhuys, Werner J.; Darvesh, Altaf S.; Funk, Max O.; Van der Schyf, Cornelis J.; Carroll, Richard T.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5295-5298</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Parkinson's disease is a severe debilitating neurodegenerative disorder.  Recently, it was shown that the peroxisome proliferating-activator receptor-γ agonist pioglitazone protected mice from 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine neurotoxicity due to its ability to inhibit monoamine oxidase B (MAO-B).  Docking studies were initiated to investigate pioglitazone's interactions within the substrate cavity of MAO-B.  Modeling studies indicated that the thiazolidinedione (TZD) moiety was a likely candidate for its specificity to MAO-B.  To explore this potential novel MAO-B scaffold, we performed a structure-based virtual screen to identify addnl. MAO-B inhibitors.  Our search identified eight novel compds. contg. the TZD-moiety that allowed for a limited study to identify structural requirements for binding to MAO-B.  Inhibition assays identified two TZDs (A6355 and L136662) which were found to inhibit recombinant human MAO-B with IC50 values of 82 and 195 nM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppUeMyr_qAULVg90H21EOLACvtfcHk0ljvW8tRKg9wXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVehtr3K&md5=27589110233cc8394f51afd1f7c0f204</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.06.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.06.128%26sid%3Dliteratum%253Aachs%26aulast%3DGeldenhuys%26aufirst%3DW.%2BJ.%26aulast%3DDarvesh%26aufirst%3DA.%2BS.%26aulast%3DFunk%26aufirst%3DM.%2BO.%26aulast%3DVan%2Bder%2BSchyf%26aufirst%3DC.%2BJ.%26aulast%3DCarroll%26aufirst%3DR.%2BT.%26atitle%3DIdentification%2520of%2520Novel%2520Monoamine%2520Oxidase%2520B%2520Inhibitors%2520by%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D5295%26epage%3D5298%26doi%3D10.1016%2Fj.bmcl.2010.06.128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachhawat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M. L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceska, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sands, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Adenosine A<sub>2A</sub> Receptor Bound to an Antagonist Reveals a Potential Allosteric Pocket</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2066</span>– <span class="NLM_lpage">2071</span>, <span class="refDoi"> DOI: 10.1073/pnas.1621423114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1073%2Fpnas.1621423114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=28167788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitFCksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=2066-2071&author=B.+Sunauthor=P.+Bachhawatauthor=M.+L.-H.+Chuauthor=M.+Woodauthor=T.+Ceskaauthor=Z.+A.+Sandsauthor=J.+Mercierauthor=F.+Lebonauthor=T.+S.+Kobilkaauthor=B.+K.+Kobilka&title=Crystal+Structure+of+the+Adenosine+A2A+Receptor+Bound+to+an+Antagonist+Reveals+a+Potential+Allosteric+Pocket&doi=10.1073%2Fpnas.1621423114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the adenosine A2A receptor bound to an antagonist reveals a potential allosteric pocket</span></div><div class="casAuthors">Sun, Bingfa; Bachhawat, Priti; Chu, Matthew Ling-Hon; Wood, Martyn; Ceska, Tom; Sands, Zara A.; Mercier, Joel; Lebon, Florence; Kobilka, Tong Sun; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2066-2071</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The adenosine A2A receptor (A2AR) has long been implicated in cardiovascular disorders.  As more selective A2AR ligands are being identified, its roles in other disorders, such as Parkinson's disease, are starting to emerge, and A2AR antagonists are important drug candidates for non-dopaminergic anti-Parkinson treatment.  Here, the authors report the crystal structure of A2A receptor bound to compd. 1, a novel A2AR/N-methyl-D-aspartate (NMDA) receptor subtype 2B (NR2B; NMDA receptor 2B) dual antagonist and potential anti-Parkinson candidate compd., at 3.5 Å resoln.  The A2A receptor with a cytochrome b562-RIL (BRIL) fusion (A2AR-BRIL) in intracellular loop 3 (ICL3) was crystd. in detergent micelles using vapor-phase diffusion.  Whereas A2AR-BRIL bound to antagonist ZM241385 was previously been crystd. in the lipidic cubic phase (LCP), structural differences in the compd. 1-bound A2AR-BRIL prevented formation of the lattice obsd. with the ZM241385-bound receptor.  The crystals grew with a type II crystal lattice in contrast to the typical type I packing seen from membrane protein structures crystd. in LCP.  Compd. 1 bound in a position that overlapped with the native ligand, adenosine, but its methoxyphenyl group extended to an exosite not previously obsd. in other A2AR structures.  Structural anal. revealed that compd. 1 binding resulted in the unique conformations of 2 Tyr residues (Tyr91.35 and Tyr2717.36) which are crit. for the formation of the exosite.  The structure revealed insights into antagonist binding that were not obsd. in other A2AR structures, highlighting flexibility in the binding pocket that may facilitate the development of A2AR-selective compds. for the treatment of Parkinson's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5SraoNhZQ0bVg90H21EOLACvtfcHk0lgiLkDFZOc-Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitFCksb8%253D&md5=58ac85e930066ef7a6788c4d7f3a72b3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1621423114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1621423114%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DB.%26aulast%3DBachhawat%26aufirst%3DP.%26aulast%3DChu%26aufirst%3DM.%2BL.-H.%26aulast%3DWood%26aufirst%3DM.%26aulast%3DCeska%26aufirst%3DT.%26aulast%3DSands%26aufirst%3DZ.%2BA.%26aulast%3DMercier%26aufirst%3DJ.%26aulast%3DLebon%26aufirst%3DF.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Adenosine%2520A2A%2520Receptor%2520Bound%2520to%2520an%2520Antagonist%2520Reveals%2520a%2520Potential%2520Allosteric%2520Pocket%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D2066%26epage%3D2071%26doi%3D10.1073%2Fpnas.1621423114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barelier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterling, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">The Recognition of Identical Ligands by Unrelated Proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2772</span>– <span class="NLM_lpage">2784</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00683</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00683" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=2772-2784&author=S.+Barelierauthor=T.+Sterlingauthor=M.+J.+O%E2%80%99Mearaauthor=B.+K.+Shoichet&title=The+Recognition+of+Identical+Ligands+by+Unrelated+Proteins&doi=10.1021%2Facschembio.5b00683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The recognition of identical ligands by unrelated proteins</span></div><div class="casAuthors">Barelier, Sarah; Sterling, Teague; O'Meara, Matthew J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2772-2784</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding of drugs and reagents to off-targets is well-known.  Whereas many off-targets are related to the primary target by sequence and fold, many ligands bind to unrelated pairs of proteins, and these are harder to anticipate.  If the binding site in the off-target can be related to that of the primary target, this challenge resolves into aligning the 2 pockets.  However, other cases are possible: the ligand might interact with entirely different residues and environments in the off-target, or wholly different ligand atoms may be implicated in the 2 complexes.  To investigate these scenarios at at. resoln., the structures of 59 ligands in 116 complexes (62 pairs in total), where the protein pairs were unrelated by fold but bound an identical ligand, were examd.  In almost half of the pairs, the ligand interacted with unrelated residues in the 2 proteins (29 pairs), and in 14 of the pairs wholly different ligand moieties were implicated in each complex.  Even in those 19 pairs of complexes that presented similar environments to the ligand, ligand superposition rarely resulted in the overlap of related residues.  There appears to be no single pattern-matching "code" for identifying binding sites in unrelated proteins that bind identical ligands, although modeling suggests that there might be a limited no. of different patterns that suffice to recognize different ligand functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1_lQJDNMCrVg90H21EOLACvtfcHk0lgiLkDFZOc-Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFyitr7J&md5=9d6ac31c2a58bc85f77e7eb34649c195</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00683%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DThe%2520Recognition%2520of%2520Identical%2520Ligands%2520by%2520Unrelated%2520Proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D2772%26epage%3D2784%26doi%3D10.1021%2Facschembio.5b00683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span></span> <span> </span><span class="NLM_article-title">From Magic Bullets to Designed Multiple Ligands</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">641</span>– <span class="NLM_lpage">651</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03163-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2FS1359-6446%2804%2903163-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=15279847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVSktb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=641-651&author=R.+Morphyauthor=C.+Kayauthor=Z.+Rankovic&title=From+Magic+Bullets+to+Designed+Multiple+Ligands&doi=10.1016%2FS1359-6446%2804%2903163-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">From magic bullets to designed multiple ligands</span></div><div class="casAuthors">Morphy, Richard; Kay, Corinne; Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">641-651</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  A major trend in medicinal chem. in the coming years will be the rational design of ligands that are capable of modulating multiple disease-relevant targets in a specific manner.  Increasingly, it is being recognized that a balanced modulation of several targets can provide a superior therapeutic effect and side effect profile compared to the action of a selective ligand.  Rational approaches in which structural features from selective ligands are combined have produced designed multiple ligands that span a wide variety of targets and target classes.  A key challenge in the design of multiple ligands is attaining a balanced activity at each target of interest while simultaneously achieving a wider selectivity and a suitable pharmacokinetic profile.  An anal. of literature examples reveals trends and insights that might help medicinal chemists discover the next generation of these types of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcsGpfzZok5rVg90H21EOLACvtfcHk0lgiLkDFZOc-Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVSktb8%253D&md5=cd2491baad707043dbee4f59ccad9e1d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903163-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903163-0%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26aulast%3DKay%26aufirst%3DC.%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DFrom%2520Magic%2520Bullets%2520to%2520Designed%2520Multiple%2520Ligands%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D641%26epage%3D651%26doi%3D10.1016%2FS1359-6446%2804%2903163-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Der Walt, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terre’Blanche, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzer, J. P.</span></span> <span> </span><span class="NLM_article-title">The Adenosine Receptor Affinities and Monoamine Oxidase B Inhibitory Properties of Sulfanylphthalimide Analogues</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">117</span>– <span class="NLM_lpage">123</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2015.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1016%2Fj.bioorg.2015.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=25746740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1aqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=117-123&author=M.+M.+Van%0ADer+Waltauthor=G.+Terre%E2%80%99Blancheauthor=A.+Petzerauthor=J.+P.+Petzer&title=The+Adenosine+Receptor+Affinities+and+Monoamine+Oxidase+B+Inhibitory+Properties+of+Sulfanylphthalimide+Analogues&doi=10.1016%2Fj.bioorg.2015.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The adenosine receptor affinities and monoamine oxidase B inhibitory properties of sulfanylphthalimide analogues</span></div><div class="casAuthors">Van der Walt, Mietha M.; Terre'Blanche, Gisella; Petzer, Anel; Petzer, Jacobus P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">117-123</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Based on a report that sulfanylphthalimides are highly potent monoamine oxidase (MAO) B selective inhibitors, the present study examines the adenosine receptor affinities and MAO-B inhibitory properties of a series of 4- and 5-sulfanylphthalimide analogs.  Since adenosine antagonists (A1 and A2A subtypes) and MAO-B inhibitors are considered agents for the therapy of neurodegenerative disorders such as Parkinson's disease and Alzheimer's disease, dual-target-directed drugs that antagonize adenosine receptors and inhibit MAO-B may have enhanced therapeutic value.  The results document that the sulfanylphthalimide analogs are selective for the adenosine A1 receptor over the A2A receptor subtype, with a no. of compds. also possessing MAO-B inhibitory properties.  Among the compds. evaluated, 5-[(4-methoxybenzyl)sulfanyl]phthalimide was found to possess the highest binding affinity to adenosine A1 receptors with a Ki value of 0.369 μM.  This compd. is reported to also inhibit MAO-B with an IC50 value of 0.020 μM.  Such dual-target-directed compds. may act synergistic in the treatment of Parkinson's disease: antagonism of the A1 receptor may facilitate dopamine release, while MAO-B inhibition may reduce dopamine metab.  Addnl., dual-target-directed compds. may find therapeutic value in Alzheimer's disease: antagonism of the A1 receptor may be beneficial in the treatment of cognitive dysfunction, while MAO-B inhibition may exhibit neuroprotective properties.  In neurol. diseases, such as Parkinson's disease and Alzheimer's disease, dual-target-directed drugs are expected to be advantageous over single-target treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQQY_dzBBNoLVg90H21EOLACvtfcHk0lgiLkDFZOc-Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1aqsr0%253D&md5=20db87d9be6476d2eb5ab492f7fae1fb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2015.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2015.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BWalt%26aufirst%3DM.%2BM.%26aulast%3DTerre%25E2%2580%2599Blanche%26aufirst%3DG.%26aulast%3DPetzer%26aufirst%3DA.%26aulast%3DPetzer%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520Adenosine%2520Receptor%2520Affinities%2520and%2520Monoamine%2520Oxidase%2520B%2520Inhibitory%2520Properties%2520of%2520Sulfanylphthalimide%2520Analogues%26jtitle%3DBioorg.%2520Chem.%26date%3D2015%26volume%3D59%26spage%3D117%26epage%3D123%26doi%3D10.1016%2Fj.bioorg.2015.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jörg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuano, B.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Pharmacological Evaluation of Dual Acting Ligands Targeting the Adenosine A<sub>2A</sub> and Dopamine D<sub>2</sub> Receptors for the Potential Treatment of Parkinson’s Disease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">718</span>– <span class="NLM_lpage">738</span>, <span class="refDoi"> DOI: 10.1021/jm501254d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501254d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGmsLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=718-738&author=M.+J%C3%B6rgauthor=L.+T.+Mayauthor=F.+S.+Makauthor=K.+C.+K.+Leeauthor=N.+D.+Millerauthor=P.+J.+Scammellsauthor=B.+Capuano&title=Synthesis+and+Pharmacological+Evaluation+of+Dual+Acting+Ligands+Targeting+the+Adenosine+A2A+and+Dopamine+D2+Receptors+for+the+Potential+Treatment+of+Parkinson%E2%80%99s+Disease&doi=10.1021%2Fjm501254d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Pharmacological Evaluation of Dual Acting Ligands Targeting the Adenosine A2A and Dopamine D2 Receptors for the Potential Treatment of Parkinson's Disease</span></div><div class="casAuthors">Jorg, Manuela; May, Lauren T.; Mak, Frankie S.; Lee, Kiew Ching K.; Miller, Neil D.; Scammells, Peter J.; Capuano, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">718-738</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A relatively new strategy in drug discovery is the development of dual acting ligands.  These mols. are potentially able to interact at two orthosteric binding sites of a heterodimer simultaneously, possibly resulting in enhanced subtype selectivity, higher affinity, enhanced or modified physiol. response, and reduced reliance on multiple drug administration regimens.  In this study, we have successfully synthesized a series of classical heterobivalent ligands as well as a series of more integrated and "drug-like" dual acting mols., incorporating ropinirole as a dopamine D2 receptor agonist and ZM 241385 as an adenosine A2A receptor antagonist.  The best compds. of our series maintained the potency of the original pharmacophores at both receptors (adenosine A2A and dopamine D2).  In addn., the integrated dual acting ligands also showed promising results in preliminary blood-brain barrier permeability tests, whereas the classical heterobivalent ligands are potentially more suited as pharmacol. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqChLZoS1UseLVg90H21EOLACvtfcHk0lg89FgEcFvrow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGmsLnL&md5=65ab4bdbcf04d6a2be666e48acc52a62</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fjm501254d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501254d%26sid%3Dliteratum%253Aachs%26aulast%3DJ%25C3%25B6rg%26aufirst%3DM.%26aulast%3DMay%26aufirst%3DL.%2BT.%26aulast%3DMak%26aufirst%3DF.%2BS.%26aulast%3DLee%26aufirst%3DK.%2BC.%2BK.%26aulast%3DMiller%26aufirst%3DN.%2BD.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DCapuano%26aufirst%3DB.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520Evaluation%2520of%2520Dual%2520Acting%2520Ligands%2520Targeting%2520the%2520Adenosine%2520A2A%2520and%2520Dopamine%2520D2%2520Receptors%2520for%2520the%2520Potential%2520Treatment%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D718%26epage%3D738%26doi%3D10.1021%2Fjm501254d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Simone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarozzo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandiera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piomelli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span> <span> </span><span class="NLM_article-title">Applying a Multitarget Rational Drug Design Strategy: The First Set of Modulators with Potent and Balanced Activity toward Dopamine D<sub>3</sub> Receptor and Fatty Acid Amide Hydrolase</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">4904</span>– <span class="NLM_lpage">4907</span>, <span class="refDoi"> DOI: 10.1039/C4CC00967C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1039%2FC4CC00967C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=24691497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlersb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=4904-4907&author=A.+De%0ASimoneauthor=G.+F.+Rudaauthor=C.+Albaniauthor=G.+Tarozzoauthor=T.+Bandieraauthor=D.+Piomelliauthor=A.+Cavalliauthor=G.+Bottegoni&title=Applying+a+Multitarget+Rational+Drug+Design+Strategy%3A+The+First+Set+of+Modulators+with+Potent+and+Balanced+Activity+toward+Dopamine+D3+Receptor+and+Fatty+Acid+Amide+Hydrolase&doi=10.1039%2FC4CC00967C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Applying a multitarget rational drug design strategy: the first set of modulators with potent and balanced activity toward dopamine D3 receptor and fatty acid amide hydrolase</span></div><div class="casAuthors">De Simone, Alessio; Ruda, Gian Filippo; Albani, Clara; Tarozzo, Glauco; Bandiera, Tiziano; Piomelli, Daniele; Cavalli, Andrea; Bottegoni, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">4904-4907</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Combining computer-assisted drug design and synthetic efforts, we generated compds. with potent and balanced activities toward both D3 dopamine receptor and fatty acid amide hydrolase (FAAH) enzyme.  By concurrently modulating these targets, our compds. hold great potential toward exerting a disease-modifying effect on nicotine addiction and other forms of compulsive behavior.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnUQcdwExHDLVg90H21EOLACvtfcHk0lg89FgEcFvrow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlersb0%253D&md5=24c31533c27a2c6e356af93edeb5156a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1039%2FC4CC00967C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CC00967C%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BSimone%26aufirst%3DA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DAlbani%26aufirst%3DC.%26aulast%3DTarozzo%26aufirst%3DG.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DPiomelli%26aufirst%3DD.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBottegoni%26aufirst%3DG.%26atitle%3DApplying%2520a%2520Multitarget%2520Rational%2520Drug%2520Design%2520Strategy%253A%2520The%2520First%2520Set%2520of%2520Modulators%2520with%2520Potent%2520and%2520Balanced%2520Activity%2520toward%2520Dopamine%2520D3%2520Receptor%2520and%2520Fatty%2520Acid%2520Amide%2520Hydrolase%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D4904%26epage%3D4907%26doi%3D10.1039%2FC4CC00967C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lorber, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Hierarchical Docking of Databases of Multiple Ligand Conformations</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.2174/1568026054637683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.2174%2F1568026054637683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=16101414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=739-749&author=D.+M.+Lorberauthor=B.+K.+Shoichet&title=Hierarchical+Docking+of+Databases+of+Multiple+Ligand+Conformations&doi=10.2174%2F1568026054637683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Hierarchical docking of databases of multiple ligand conformations</span></div><div class="casAuthors">Lorber, David M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">739-749</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Ligand flexibility is an important problem in mol. docking and virtual screening.  To address this challenge, we investigate a hierarchical pre-organization of multiple conformations of small mols.  Such organization of pre-calcd. conformations removes the exploration of ligand conformational space from the docking calcn. and allows for concise representation of what can be thousands of conformations.  The hierarchy also recognizes and prunes incompatible conformations early in the calcn., eliminating redundant calcns. of fit.  We investigate the method by docking the MDL Drug Data Report (MDDR), an annotated database of 100,000 mols., into apo and holo forms of 7 unrelated targets.  This annotated database allows us to track the ranking of tens to hundreds of annotated ligands in each of the docking systems.  The binding sites and database are prepd. in an automated fashion in an attempt to remove some human bias from the calcns.  Many thousands of explicit and implicit ligand conformations may be docked in calcns. not much longer than required for single conformer docking.  As long as internal energies are not considered, recombination with the hierarchy is additive as the no. of degrees of freedom is increased.  Mols. with even millions of conformations can be docked in a few minutes on a single desktop computer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ_iMnNjm9LVg90H21EOLACvtfcHk0lg89FgEcFvrow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsFGht74%253D&md5=25c833b5608e5faedc1af3ad38fdfa06</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.2174%2F1568026054637683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026054637683%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DHierarchical%2520Docking%2520of%2520Databases%2520of%2520Multiple%2520Ligand%2520Conformations%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2005%26volume%3D5%26spage%3D739%26epage%3D749%26doi%3D10.2174%2F1568026054637683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alagona, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Profeta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiner, P.</span></span> <span> </span><span class="NLM_article-title">A New Force Field for Molecular Mechanical Simulation of Nucleic Acids and Proteins</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">765</span>– <span class="NLM_lpage">784</span>, <span class="refDoi"> DOI: 10.1021/ja00315a051</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00315a051" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaL2cXmsVCgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=1984&pages=765-784&author=S.+J.+Weinerauthor=P.+A.+Kollmanauthor=D.+A.+Caseauthor=U.+C.+Singhauthor=C.+Ghioauthor=G.+Alagonaauthor=S.+Profetaauthor=P.+Weiner&title=A+New+Force+Field+for+Molecular+Mechanical+Simulation+of+Nucleic+Acids+and+Proteins&doi=10.1021%2Fja00315a051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">A new force field for molecular mechanical simulation of nucleic acids and proteins</span></div><div class="casAuthors">Weiner, Scott J.; Kollman, Peter A.; Case, David A.; Singh, U. Chandra; Ghio, Caterina; Alagona, Guliano; Profeta, Salvatore, Jr.; Weiner, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">765-84</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A force field was developed for simulation of nucleic acids and proteins.  The approach began by obtaining equil. bond lengths and angles from microwave, neutron diffraction, and prior mol. mech. calcns., torsional consts. from microwave, NMR, and mol. mech. studies, nonbonded parameters from crystal packing calcns., and at. charges from the fit of a partial charge model to electrostatic potentials calcd. by ab initio quantum mech. theory.  The parameters then were refined with mol. mech. studies on the structures and energies of model compds.  For nucleic acids, Me Et ether, THF, deoxyadenosine, di-Me phosphate, 9-methylguanine-1-methylcytosine H-bonded complex, 9-methyladenine-1-methylthymine H-bonded complex, and 1,3-dimethyluracil base-stacked dimer were studied.  Bond, angle, torsional, nonbonded, and H-bond parameters were varied to optimize the agreement between calcd. and exptl. values for sugar pucker energies and structures, vibrational frequencies of di-Me phosphate and THF, and energies for base pairing and base stacking.  For proteins, Φ,ψ maps of glycyl and alanyl dipeptides, H-bonding interactions involving the various protein polar groups, and energy refinement calcns. on insulin were considered.  Unlike the models for H bonding involving N and O electron donors, an adequate description of S-H bonding required explicit inclusion of lone pairs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp26DDzh1ctfrVg90H21EOLACvtfcHk0lheXCVG8266tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmsVCgug%253D%253D&md5=c4aa27feec7e2c4d34001be10a6cb8e9</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fja00315a051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00315a051%26sid%3Dliteratum%253Aachs%26aulast%3DWeiner%26aufirst%3DS.%2BJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DSingh%26aufirst%3DU.%2BC.%26aulast%3DGhio%26aufirst%3DC.%26aulast%3DAlagona%26aufirst%3DG.%26aulast%3DProfeta%26aufirst%3DS.%26aulast%3DWeiner%26aufirst%3DP.%26atitle%3DA%2520New%2520Force%2520Field%2520for%2520Molecular%2520Mechanical%2520Simulation%2520of%2520Nucleic%2520Acids%2520and%2520Proteins%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1984%26volume%3D106%26spage%3D765%26epage%3D784%26doi%3D10.1021%2Fja00315a051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and Testing of a General Amber Force Field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+M.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+Testing+of+a+General+Amber+Force+Field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0lheXCVG8266tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BM.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520Testing%2520of%2520a%2520General%2520Amber%2520Force%2520Field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, I. D.</span></span> <span> </span><span class="NLM_article-title">Automated Docking with Grid-based Energy Evaluation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1002/jcc.540130412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1002%2Fjcc.540130412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=505-524&author=E.+C.+Mengauthor=B.+K.+Shoichetauthor=I.+D.+Kuntz&title=Automated+Docking+with+Grid-based+Energy+Evaluation&doi=10.1002%2Fjcc.540130412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking with grid-based energy evaluation</span></div><div class="casAuthors">Meng, Elaine C.; Shoichet, Brian K.; Kuntz, Irwin D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-24</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The ability to generate feasible binding orientations of a small mol. within a site of known structure is important for ligand design.  The authors present a method that combines a rapid, geometric docking algorithm with the evaluation of mol. mechanics interaction energies.  The computational costs of evaluation are minimal because the authors precalc. the receptor-dependent terms in the potential function at points on a three-dimensional grid.  In four test cases where the components of crystallog. detd. complexes are redocked, the "force field" score correctly identifies the family of orientations closest to the exptl. binding geometry.  Scoring functions that consider only steric factors or only electrostatic factors are less successful.  The force field function will play an important role in efforts to search databases for potential lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKAw7Zj_QFSbVg90H21EOLACvtfcHk0lheXCVG8266tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D&md5=f472fffa0c9a61652b4c20e4dbbba69e</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130412%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DAutomated%2520Docking%2520with%2520Grid-based%2520Energy%2520Evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D505%26epage%3D524%26doi%3D10.1002%2Fjcc.540130412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span> <span> </span><span class="NLM_article-title">A Rapid Finite Difference Algorithm, Utilizing Successive Over-relaxation to Solve the Poisson–Boltzmann Equation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1002/jcc.540120405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1002%2Fjcc.540120405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADyaK3MXisFCgtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1991&pages=435-445&author=A.+Nichollsauthor=B.+Honig&title=A+Rapid+Finite+Difference+Algorithm%2C+Utilizing+Successive+Over-relaxation+to+Solve+the+Poisson%E2%80%93Boltzmann+Equation&doi=10.1002%2Fjcc.540120405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid finite difference algorithm, utilizing successive over-relaxation to solve the Poisson-Boltzmann equation</span></div><div class="casAuthors">Nicholls, Anthony; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-45</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient algorithm is presented for the numerical soln. of the Poisson-boltzmann equation by the finite difference method of successive over-relaxation.  Improvements include the rapid estn. of the optimum relaxation parameter, redn. in no. of operations per iteration, and vector-oriented array mapping.  The algorithm has been incorporated into the electrostatic program DelPhi, reducing the required computing time by between one and two orders of magnitude.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLJ8dypOKbsrVg90H21EOLACvtfcHk0ljotOXrugftMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisFCgtr0%253D&md5=28ac444a000ad2f801ceea2237f4bc2e</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540120405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540120405%26sid%3Dliteratum%253Aachs%26aulast%3DNicholls%26aufirst%3DA.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DA%2520Rapid%2520Finite%2520Difference%2520Algorithm%252C%2520Utilizing%2520Successive%2520Over-relaxation%2520to%2520Solve%2520the%2520Poisson%25E2%2580%2593Boltzmann%2520Equation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1991%26volume%3D12%26spage%3D435%26epage%3D445%26doi%3D10.1002%2Fjcc.540120405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janežič, D.</span></span> <span> </span><span class="NLM_article-title">ProBiS: A Web Server for Detection of Structurally Similar Protein Binding Sites</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">W436</span>– <span class="NLM_lpage">W440</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=10.1093%2Fnar%2Fgkq479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=20504855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;key=1%3ACAS%3A528%3ADC%252BC3cXotVWjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=W436-W440&author=J.+Koncauthor=D.+Jane%C5%BEi%C4%8D&title=ProBiS%3A+A+Web+Server+for+Detection+of+Structurally+Similar+Protein+Binding+Sites&doi=10.1093%2Fnar%2Fgkq479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">ProBiS: a web server for detection of structurally similar protein binding sites</span></div><div class="casAuthors">Konc, Janez; Janezic, Dusanka</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W436-W440</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A web server, ProBiS, freely available at http://probis.cmm.ki.si, is presented.  This provides access to the program ProBiS (Protein Binding Sites), which detects protein binding sites based on local structural alignments.  Detailed instructions and user guidelines for use of ProBiS are available at the server under HELP' and selected examples are provided under EXAMPLES'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_Hf9BhPvSrrVg90H21EOLACvtfcHk0ljotOXrugftMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXotVWjsrc%253D&md5=6b53ed57c159be18c05a5e2f72995983</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq479%26sid%3Dliteratum%253Aachs%26aulast%3DKonc%26aufirst%3DJ.%26aulast%3DJane%25C5%25BEi%25C4%258D%26aufirst%3DD.%26atitle%3DProBiS%253A%2520A%2520Web%2520Server%2520for%2520Detection%2520of%2520Structurally%2520Similar%2520Protein%2520Binding%2520Sites%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DW436%26epage%3DW440%26doi%3D10.1093%2Fnar%2Fgkq479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PWH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PWH','PDB','3PWH'); return false;">PDB: 3PWH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V61" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V61','PDB','2V61'); return false;">PDB: 2V61</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UIG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5UIG','PDB','5UIG'); return false;">PDB: 5UIG</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00204">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31588"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00204">10.1021/acs.jmedchem.8b00204</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Tables S1–S10 and Figures S1–S5 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_002.csv">CSV</a>)</p></li><li><p class="inline">Predicted binding modes of dual-target inhibitors <b>1</b>–<b>4</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Predicted binding modes of dual-target inhibitors <b>1</b>–<b>4</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_004.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_001.pdf">jm8b00204_si_001.pdf (3.61 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_002.csv">jm8b00204_si_002.csv (2.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_003.pdb">jm8b00204_si_003.pdb (187.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00204/suppl_file/jm8b00204_si_004.pdb">jm8b00204_si_004.pdb (326.62 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00204&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-12%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00204%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00204" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991fa30d4c3ce8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
